# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# D. Type of Test:

Reverse transcription, quantitative, polymerase chain reaction (qPCR) based nucleic acid amplification

# E. Applicant:

MolecularMD

# F. Proprietary and Established Names:

Trade Name: MolecularMD MRDx $^ \mathrm { \textregistered }$ BCR-ABL Test Common Name: BCR-ABL RT-qPCR Test

G. Regulatory Information: 1. Regulation section:

21 CFR 866.6060

2. Classification: Class II

3. Product code: OYX, BCR/ABL1 monitoring test

4. Panel: 88 – Pathology

# H. Intended Use:

1. Indications for Use

The MolecularMD MRDx BCR-ABL Test is an in vitro diagnostic test for the quantitative detection of BCR-ABL1 transcripts (e13a2/b2a2 and/or e14a2/b3a2) and the ABL1 endogenous control mRNA in peripheral blood specimens from patients previously diagnosed with t(9:22) positive chronic myeloid leukemia (CML). The ratio of BCR-ABL1 to ABL1 is calculated and reported on the WHO International Scale. The test utilizes quantitative, real-time reverse transcription polymerase chain reaction performed on the Applied Biosystems 7500 Fast Dx instrument.

The MolecularMD MRDx BCR-ABL Test is intended to measure BCR-ABL mRNA transcript levels in patients diagnosed with t(9;22) positive CML during monitoring of treatment with Tyrosine Kinase Inhibitors (TKIs).

The device is also intended to be used in the serial monitoring for BCR-ABL mRNA transcript levels as an aid in identifying CML patients in the chronic phase being treated with nilotinib who may be candidates for treatment discontinuation and for monitoring of treatment-free remission.

The test does not differentiate between e13a2 or e14a2 fusion transcripts and does not monitor other rare fusion transcripts resulting from t(9;22). The test is not intended for the diagnosis of CML.

2. Special conditions for use statement(s): For in vitro diagnostic use only. For prescription use only.

3. Special instrument requirements:

Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument using System Sequence Detection Software v.1.4.1.

# I. Device Description:

The MolecularMD MRDx BCR-ABL Test supports blood samples collected in either $\mathrm { K } _ { 2 } \mathrm { E D T A }$ blood tubes or PAXgene Blood RNA Tubes. The EDTA tubes are processed using the Maxwell CSC RNA Blood Kit and the semi-automated Maxwell CSC Instrument. The PAXgene Blood RNA tubes are processed by a manual extraction procedure using the PAXgene Blood RNA Kit. The MolecularMD MRDx BCR-ABL Test kit includes reagents sufficient for 26 samples. A total of 17 patient samples can be tested per plate. A description of the reagents provided with the kit are described below in Table 1.

Table 1: Components of the MRDx BCR-ABL Test kit   

<table><tr><td rowspan=1 colspan=1>Item Name</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Use</td></tr><tr><td rowspan=1 colspan=1>MRDx BCR-ABLPrimer/Probe</td><td rowspan=1 colspan=1>(1 vial x 40 μL/vial)</td><td rowspan=1 colspan=1>For PCR amplification anddetection of target BCR-ABLfusion sequences.</td></tr><tr><td rowspan=1 colspan=1>MRDx ABLPrimer/Probe</td><td rowspan=1 colspan=1>(1 vial x 40 µL/vial)</td><td rowspan=1 colspan=1>For PCR amplification anddetection of target endogenousABL sequences.</td></tr><tr><td rowspan=1 colspan=1>MRDx Nuclease-FreeWater</td><td rowspan=1 colspan=1>(2 vials x 1.8 mL/vial)</td><td rowspan=1 colspan=1>Adjust reaction volume.</td></tr><tr><td rowspan=1 colspan=1>MRDx Master Mix (2X)</td><td rowspan=1 colspan=1>(3 vials x 850 µL/vial)2X reaction mix containing: RTenzyme, hot start PCR enzyme,RNase inhibitor, buffers,nucleotides, Rox passivereference dye, blue qPCR dye,MgCl2, and stabilizers.</td><td rowspan=1 colspan=1>For PCR amplification anddetection of target sequences.</td></tr><tr><td rowspan=1 colspan=1>MRDx RNA ControlMMR</td><td rowspan=1 colspan=1>Targeted to 0.10% (IS) BCR-ABL/ABL; MR3.0</td><td rowspan=2 colspan=1>RNA from human cellsformulated from a mixture ofBCR-ABL negative and BCR-ABL e14a2 positive RNA in abackground mix of carrier RNAand stabilizers. Controls used toensure the RT and PCR stepsperformed properly based onexpected MR value.</td></tr><tr><td rowspan=1 colspan=1>MRDx RNA ControlMR 4.5</td><td rowspan=1 colspan=1>Targeted to 0.0032% (IS) BCR-ABL/ABL; MR4.5</td></tr><tr><td rowspan=1 colspan=1>MRDx BCR-ABL RNACalibrator - A-H</td><td rowspan=1 colspan=1>Synthetic RNA calibrators instabilization matrix</td><td rowspan=1 colspan=1>To generate two calibrationcurves (one for BCR-ABL andone for ABL) analyzed with everytest run to allow for accuratequantitation of the transcripts incontrols and patient samples.</td></tr></table>

# RNA Calibrators:

A set of calibrators containing in vitro transcribed BCR-ABL and ABL RNA targets representing a dynamic range of $1 0 ^ { 5 }$ is provided as part of the test kit. Two calibration curves (one for BCR-ABL and one for ABL) are analyzed with every test run to allow for quantitation of the two transcripts in patient samples. The BCR-ABL and ABL levels for each calibrator is listed in Table 2 below.

Table 2: MRDx BCR-ABL Test Calibrators   

<table><tr><td rowspan=1 colspan=1>Calibrator</td><td rowspan=1 colspan=1>BCR-ABL</td><td rowspan=1 colspan=1>ABL</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>600,000 copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>300,000copies/10 μL</td><td rowspan=1 colspan=1>300,000copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>30,000 copies/10 μL</td><td rowspan=1 colspan=1>30,000copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>3,000copies/10 μL</td><td rowspan=1 colspan=1>3,000copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>300copies/10 μL</td><td rowspan=1 colspan=1>300copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>30copies/10 μL</td><td rowspan=1 colspan=1>30copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>10copies/10 μL</td><td rowspan=1 colspan=1>10copies/10 μL</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>3 copies/10 μL</td><td rowspan=1 colspan=1></td></tr></table>

Software:   
The MRDx BCR-ABL Test Software is used to analyze all test results. This software is provided with the MRDx BCR-ABL Test.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Asuragen QuantideX qPCR BCR-ABL IS Kit

2. Predicate K number(s): DEN160003

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Applied Biosystems 7500 Fast Dx Real Time PCRInstrument</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=1>BCR-ABL1 fusion transcripts (e13a2/b2a2 and/ore14a2/b3a2) and the ABL1 endogenous controlmRNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>MeasurementType</td><td rowspan=1 colspan=1>Quantitative</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Principle ofAssay</td><td rowspan=1 colspan=1>Reverse transcription, quantitative, polymerasechain reaction (qPCR) based nucleic acidamplification</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Traceability/Standard</td><td rowspan=1 colspan=1>1st WHO International Genetic Reference Panel forquantitation of BCR-ABL translocation by RQ-PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Report Units</td><td rowspan=1 colspan=1>Both % IS and Molecular Response (MR)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TestLimitation</td><td rowspan=1 colspan=1>The test does not differentiate between e13a2 ore14a2 fusion transcripts and does not monitorother rare fusion transcripts resulting from t(9;22).The test is not intended for the diagnosis of CML.</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=2>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=3 colspan=1>Intended Use</td><td rowspan=3 colspan=2>The MolecularMD MRDx BCR-ABL Test isan in vitro diagnostic test for the quantitativedetection of BCR-ABL1 transcripts(e13a2/b2a2 and/or e14a2/b3a2) and the ABL1endogenous control mRNA in peripheral bloodspecimens from patients previously diagnosedwith t(9:22) positive chronic myeloid leukemia(CML). The ratio of BCR-ABL1 to ABL1 iscalculated and reported on the WHOInternational Scale. The test utilizesquantitative, real-time reverse transcriptionpolymerase chain reaction performed on theApplied Biosystems 7500 Fast Dx instrument.The MolecularMD MRDx BCR-ABL Test isintended to measure BCR-ABL mRNAtranscript levels in patients diagnosed witht(9;22) positive CML during monitoring oftreatment with Tyrosine Kinase Inhibitors(TKIs). The device is also intended to be usedin the serial monitoring for BCR-ABL mRNAtranscript levels as an aid in identifying CMLpatients in the chronic phase being treated withnilotinib who may be candidates for treatmentdiscontinuation and for monitoring oftreatment-free remission. The test does notdifferentiate between e13a2 or e14a2 fusiontranscripts and does not monitor other rarefusion transcripts resulting from t(9;22). Thetest is not intended for the diagnosis of CML.</td><td rowspan=3 colspan=1>The QuantideX® qPCRBCR-ABL IS Kit is an invitro nucleic acidamplification test for thequantitation of BCR-ABL1and ABL1 transcripts in totalRNA from whole blood ofdiagnosed t(9;22) positiveChronic Myeloid Leukemia(CML) patients expressingBCR-ABL1 fusiontranscripts type e13a2 and/ore14a2. The QuantideX qPCRBCR-ABL IS Kit is a reversetranscription-quantitativePCR performed on theApplied Biosystems 7500Fast Dx Real-Time PCRInstrument and is intended tomeasure BCR-ABL1 toABL1, expressed as a logmolecular reduction (MRvalue) from a baseline of100% on the InternationalScale, in t(9;22) positiveCML patients duringmonitoring of treatment withTyrosine Kinase Inhibitors(TKIs).</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BCR-ABL IS Kit is a reverse</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>QualityControls</td><td rowspan=1 colspan=2>RNA MMR (MR3.0)RNA MR4.5Nuclease-Free Water  NTC</td><td rowspan=1 colspan=1>RNA High (MR 1.5)RNA Low (MR 3.5)ABL armored RNARNA Negative</td></tr><tr><td rowspan=1 colspan=1>Calibrators</td><td rowspan=1 colspan=2>8 vials total, representing 7 concentrations eachof BCR-ABL and ABL transcripts.</td><td rowspan=1 colspan=1>Four levels formulated toMR1.0, 2.0, 3.0, 4.0.</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=2>RNA from whole blood (PAXgene BloodRNA Tubes or EDTA)</td><td rowspan=1 colspan=1>RNA from whole blood(EDTA)</td></tr><tr><td rowspan=1 colspan=1>RNA InputAmount</td><td rowspan=1 colspan=2>2 µg PAXgene tubes and 1μg EDTA tubes</td><td rowspan=1 colspan=1>RNA input range of 1 to 5μg</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

· Guidance for Industry and FDA Staff: Guidance for the Content of Premarket Submission for Software Contained in Medical Devices

General Principles of Software Validation; Final Guidance for Industry and FDA Staff Guidance for Industry, FDA Reviewers and Compliance on Off-the-Shelf Software Use in Medical Devices Content of Premarket Submissions for Management of Cybersecurity in Medical Devices; Guidance for Industry and FDA Staff CLSI EP05-A3, Evaluation of Precision Performance of Quantitative Measurement Methods CLSI EP06-A, Evaluation of the Linearity of Quantitative Measurement Procedures, A Statistical Approach · CLSI EP07-A2, Interference Testing in Clinical Chemistry · CLSI EP25-A, Evaluation of Stability on In Vitro Diagnostic Reagents

# L. Test Principle:

The MolecularMD MRDx $^ \mathrm { \textregistered }$ BCR-ABL Test is a quantitative, real-time polymerase chain reaction test that provides quantitation of BCR-ABL1 (hereafter BCR-ABL) transcript e13a2/b2a2 or e14a2/b3a2 and ABL1 (hereafter ABL) transcript levels in RNA extracted from peripheral blood samples collected from CML patients. Peripheral blood is collected in either EDTA or PAXgene tubes. Each collection tube type requires a specified RNA extraction method and RNA input amount. The detailed extraction protocol and RNA quantity requirements are provided in the device package insert and instructions for use. Total RNA is extracted from peripheral blood and serves as the template for RT-qPCR. The test is performed using a one-step RT-qPCR protocol wherein the reverse transcription and quantitative, realtime PCR reactions are performed in the same well. BCR-ABL and ABL amplicons are generated and detected in real-time using TaqMan $^ \mathrm { \textregistered }$ MGB probes. For RNA extracted from EDTA tubes, a total of $4 \mu \mathrm { g }$ of RNA is required for each patient sample:1 $\mu \mathrm { g }$ of extracted RNA is used in the MRDx test in each of 4 wells, 2 wells for BCR-ABL and 2 wells for ABL. For RNA extracted from PAXgene tubes, total of $8 ~ { \mu \mathrm { g } }$ of RNA is required for each patient sample: $2 \mu \mathrm { g }$ of extracted RNA is used in the MRDx test in each of 4 wells, two2 wells for BCR-ABL quantitation and 2 wells for ABL quantitation.

The MolecularMD MRDx BCR-ABL Test is performed on the Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument. The instrument integrates a thermal cycler, a fluorometer and application specific software. The instrument houses the thermal cycler and the fluorometer, while the application software is installed on a computer that is attached to the instrument. The ABI 7500 Fast Dx instrument software v1.4.1 calculates the BCR-ABL and ABL copy numbers using a standard curve generated with calibrators on each individual plate.

Quantitation is achieved using RNA calibration standards and linear regression analysis provided by the ABI 7500 Fast Dx PCR Instrument Software. BCR-ABL transcript levels are measured in relation to the ABL transcript as an endogenous reference. The data are exported as a CSV file for further analysis in the MRDx BCRABL Test Software. The BCR-ABL/ABL ratio is calculated and converted to the International Scale by the MRDx $^ \mathrm { \textregistered }$ BCR-ABL Test Software.

The BCR-ABL/ABL $\%$ IS for patient sample results is calculated using the conversion factor for the MRDx BCR-ABL Test in addition to validating each MRDx BCR-ABL Test result against the run acceptance criteria and sample acceptance criteria.

Calibrators are analyzed for both BCR-ABL and ABL in every run simultaneously with patient samples. RNA controls provided at MMR (MR3.0) and MR4.5 allow for on-plate verification of test accuracy and reproducibility over this critical transcript range. The integrated conversion factor provides test results on the International Scale (IS) harmonized to the WHO.

# Interpretation of Results:

Test results are reported in both IS scale and the corresponding molecular response (MR) value. The BCR-ABL/ABL ratio percent IS result and MR value is calculated by the MRDx BCR-ABL Test Software using the following equations.

$$
{ \frac { B C R - A B L } { A B L } } \ \% \ S = \left( { \frac { M e a n \ B C R - A B L C o p y \ N u m b e r } { M e a n \ A B L C o p y \ N u m b e r } } \right) * 1 0 0 * C o n v e r s i o n \ F a c t o r ( C o m p o p ) .
$$

$$
M R x . x = l o g _ { 1 0 } \left( \frac { 1 0 0 } { \% J 5 } \right) = l o g _ { 1 0 } ( 1 0 0 ) - l o g _ { 1 0 } ( \% I S ) = 2 - l o g _ { 1 0 } ( \% I S )
$$

MR Values with the corresponding IS values are show in the table below:

<table><tr><td rowspan=1 colspan=1>MR</td><td rowspan=1 colspan=1>IS (%)</td></tr><tr><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>0.032</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.01</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.0032</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>0.001</td></tr></table>

The assay reporting is shown in the table below. The assay will only report quantified results for BCR-ABL levels within MR4.5 to MR1.0 range.

<table><tr><td rowspan=1 colspan=1>Collection Tube</td><td rowspan=1 colspan=1>BCR-ABL% (MR)</td><td rowspan=1 colspan=1>MRDx Result</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>1-4.5</td><td rowspan=1 colspan=1>BCR-ABL detected</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>4.5-5.3</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>&gt;5.3</td><td rowspan=1 colspan=1>BCR-ABL not detected</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>1-4.5</td><td rowspan=1 colspan=1>BCR-ABL detected</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>4.5-4.7</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>&gt;4.7</td><td rowspan=1 colspan=1>BCR-ABL not detected</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a) Precision-Repeatability:

Contrived patient samples targeting MR3.0, MR4.0, and MR4.5 for e13a2 and e14a2 BCR-ABL transcripts were created using CML patient RNA and nondiseased subject RNA extracted from both PAXgene Blood RNA tubes and EDTA blood collection tubes. The MR3.0, MR4.0, and MR4.5 RNA samples were each tested in 17 replicates on individual MRDx BCR-ABL Test plates.

For EDTA Blood RNA, all levels for both transcripts passed the acceptance criteria of $\leq 0 . 2 5$ SD $\log _ { 1 0 }$ . The highest standard deviation was observed for the e14a2 BCR-ABL transcript at MR4.5 $\mathrm { ( S D = 0 . 0 8 7 ) }$ ).

For PAXgene Blood RNA, all levels for both transcripts passed the acceptance criteria $\mathrm { \Delta B D } \leq 0 . 2 5$ SD log10). The highest standard deviation was observed for the e14a2 BCR-ABL transcript at MR4.5 $\mathrm { S D } { = } 0 . 1 0 $ ).

The repeatability results are shown in Table 3A below.

Table 3A: Repeatability   

<table><tr><td colspan="1" rowspan="2">TubeType</td><td colspan="1" rowspan="2">TranscriptLevel</td><td colspan="1" rowspan="2">SampleSize (N)</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">BCR-ABL%IS</td><td colspan="1" rowspan="1">BCR-ABL%IS</td><td colspan="1" rowspan="1">BCR-ABL%IS</td><td colspan="1" rowspan="1">MR</td><td colspan="1" rowspan="1">MR</td><td colspan="1" rowspan="1">MR</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e13a2MR3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.0087</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e13a2MR4</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.017</td><td colspan="1" rowspan="1">0.0026</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e13a2MR4.5</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.0062</td><td colspan="1" rowspan="1">0.0017</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">0.099</td><td colspan="1" rowspan="1">2.4</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e14a2MR3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.0092</td><td colspan="1" rowspan="1">8.1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e14a2MR4</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.021</td><td colspan="1" rowspan="1">0.0035</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">3.7</td><td colspan="1" rowspan="1">0.078</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1">PAXgene</td><td colspan="1" rowspan="1">e14a2MR4.5</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.0051</td><td colspan="1" rowspan="1">0.0012</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">2.4</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e13a2MR3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.0069</td><td colspan="1" rowspan="1">5.9</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e13a2MR4</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.011</td><td colspan="1" rowspan="1">0.0012</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">1.5</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e13a2MR4.5</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.0034</td><td colspan="1" rowspan="1">0.00055</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">0.079</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e14a2MR3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.012</td><td colspan="1" rowspan="1">9.7</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e14a2MR4</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.013</td><td colspan="1" rowspan="1">0.0012</td><td colspan="1" rowspan="1">9.2</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">EDTA</td><td colspan="1" rowspan="1">e14a2MR4.5</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.0038</td><td colspan="1" rowspan="1">0.00071</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">0.087</td><td colspan="1" rowspan="1">2</td></tr></table>

Table 3B: Precision- MRDx RNA Controls   

<table><tr><td rowspan=2 colspan=1>RNAControlLot</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=4>RNA Control-MMR</td><td rowspan=1 colspan=4>RNA Control-MR4.5</td></tr><tr><td rowspan=1 colspan=1>MeanMR</td><td rowspan=1 colspan=1>SDMR</td><td rowspan=1 colspan=1>Mean% IS</td><td rowspan=1 colspan=1>SD% IS</td><td rowspan=1 colspan=1>MeanMR</td><td rowspan=1 colspan=1>SDMR</td><td rowspan=1 colspan=1>Mean% IS</td><td rowspan=1 colspan=1>SD% IS</td></tr><tr><td rowspan=1 colspan=1>1751</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.99</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.013</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.0041</td><td rowspan=1 colspan=1>0.0013</td></tr><tr><td rowspan=1 colspan=1>1848</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>0.083</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>4.43</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.004</td><td rowspan=1 colspan=1>0.0012</td></tr><tr><td rowspan=1 colspan=1>1863</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.95</td><td rowspan=1 colspan=1>0.061</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.014</td><td rowspan=1 colspan=1>4.42</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.0041</td><td rowspan=1 colspan=1>0.0013</td></tr><tr><td rowspan=1 colspan=1>1863</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.89</td><td rowspan=1 colspan=1>0.081</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.0041</td><td rowspan=1 colspan=1>0.0011</td></tr><tr><td rowspan=1 colspan=1>1900</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.96</td><td rowspan=1 colspan=1>0.084</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>4.44</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.0037</td><td rowspan=1 colspan=1>0.001</td></tr><tr><td rowspan=1 colspan=1>1906</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>0.072</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>4.41</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0041</td><td rowspan=1 colspan=1>0.001</td></tr></table>

# b) Precision -Assay Controls

To evaluate the precision of the RNA controls, six lots of the RNA controls were tested with twenty replicates per lot. The acceptance criteria were that the accuracy (Mean MR) is within $\pm 0 . 2 5$ of target MR value, and the precision (SD) are less than 0.125 for MMR and less than 0.25 for MR4.5 across all six lots. The RNA control precision data are shown in Table 3B above. All acceptance criteria were met, supporting the precision conclusion of the assay RNA controls.

# c) Precision -Reproducibility

i. Reproducibility – EDTA tube

Two (2) series of samples were created with blood drawn into EDTA tubes by separately diluting pooled e13a2 and pooled e14a2 high positive CML patient samples into blood from non-diseased subjects. Five (5) levels were created with targeted BCR-ABL/ABL $\%$ IS ratios from $10 \%$ IS to $0 . 0 0 3 2 \%$ IS (MR1.0, 2.0, 3.0, 4.0 and 4.5). Three (3) testing sites with 2 instruments and 2 operators were used. Each day at each site, each operator ran 2 plates, 1 on each instrument. Each plate had 1 replicate of the e13a2 five-member panel and one replicate of the e14a2 five-member panel. For each of 2 lots, 5 runs were performed. A total of 15 measurements were made per instrument, per operator. A total of 60 replicates (15 measurements per instrument per site $_ { \textrm { X 2 } }$ instruments $_ { \textrm { X 2 } }$ operators $\scriptstyle = 6 0$ ) were run at each site for each BCR-ABL transcript type.

The acceptance criteria were that the total precision expressed in SD of the MR value for each sample across sites, lots, instruments, operators, and days must be less than 0.25. The acceptance criteria were met, with the highest SD of 0.14 observed with e14a2 transcript at MR4.5. The data support the conclusion that the assay with EDTA collection tube is reproducible across MR1.0-MR4.5 levels. The results are shown in Table 4A, 4B and Table 5 below.

Tables 4A: Reproducibility (between sites, operators, days) -EDTA   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean(MR)</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Between-Day(Run)</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2MR1.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.027</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0077</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.0046</td><td rowspan=1 colspan=1>0.46</td></tr><tr><td rowspan=1 colspan=1>e14a2MR2.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e14a2MR3.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.015</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e14a2MR4.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1>e14a2MR4.5</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.023</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2MR1.0</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2MR2.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.024</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.0086</td><td rowspan=1 colspan=1>0.38</td></tr><tr><td rowspan=1 colspan=1>e13a2MR3.0</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>0.0070</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2MR4.0</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.019</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2MR4.5</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Tables 4B: Reproducibility (between instruments, lots) -EDTA   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean(MR)</td><td rowspan=1 colspan=2>Between-Instruments</td><td rowspan=1 colspan=2>Between-Lots</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2 L1</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.078</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=1 colspan=1>e14a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.011</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e14a2 L3</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0065</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>e14a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0061</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>e14a2 L5</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L1</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.010</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.031</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>4.9</td></tr><tr><td rowspan=1 colspan=1>e13a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.0023</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.033</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.064</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>e13a2 L3</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.010</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.019</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.096</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 L5</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>2.9</td></tr></table>

Table 5: Within-Laboratory Precision – EDTA   

<table><tr><td rowspan=1 colspan=7>Within-Laboratory Precision</td></tr><tr><td rowspan=1 colspan=1>BCR-ABLTranscript</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanMR</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.059</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.058</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.092</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.098</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.066</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.053</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.056</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.055</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.056</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.089</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.092</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.097</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>2.9</td></tr></table>

# ii. Reproducibility - PAXgene tube

For PAXgene Blood RNA tubes, 2 precision panels (1 each for e13a2 and e14a2e transcript) were created using pooled high positive CML patient samples diluted non-diseased (BCR-ABL negative) samples. Five (5) levels were created for each panel with targeted BCRABL/ABL $\%$ IS ratios ranging from $10 \%$ IS to $0 . 0 0 3 2 \%$ IS (MR1.0, 2.0, 3.0, 4.0 and 4.5).

Reproducibility was evaluated at 1 testing site with 3 operators and 1 instrument per operator. Precision panel samples were extracted with 3 extraction reagent lots and evaluated using 3 MRDx BCR-ABL Test kit lots over 5 non-consecutive days for a total of 15 replicates. A total of 90 replicates were generated for each panel member for each BCR-ABL

transcript type. The acceptance criteria were that the total withinlaboratory precision expressed in SD (MR value) for each must be less than 0.25. The reproducibility acceptance criteria were met for all the variables tested, with the highest variability was observed with e14a2 transcript at MR4.5 level (Total $\mathrm { S D } = 0 . 2 4 $ ). The results (summarized in Table 6 below) support the conclusion that the assay is reproducible using PAXgene collection tube across MR1.0-MR4.5 levels.

Table 6: Reproducibility– PAXgene   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean(MR)</td><td rowspan=1 colspan=2>Between Operator(Instrument)</td><td rowspan=1 colspan=2>Between Day(Run)</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Total(Within-Laboratory)</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2 MR1.0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.0074</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>0.034</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.095</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>e14a2 MR2.0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.038</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.067</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>e14a2 MR3.0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.063</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.090</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>e14a2 MR4.0</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.082</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.058</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>e14a2 MR4.5</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0.078</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.069</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>e13a2 MR1.0</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.032</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>e13a2 MR2.0</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.014</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>5.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 MR3.0</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 MR4.0</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 MR4.5</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.070</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>4.3</td></tr></table>

# d) Linearity/Reportable range:

Linearity was estimated by testing 2 dilution series (1 for e13a2 and 1 for e14a2 transcript) for each blood collection tube type (the EDTA and PAXgene tube). Each dilution series had 10 BCR-ABL target levels and 5 replicates at each level to evaluate the assay linearity across the target range from $1 5 \%$ IS to $0 . 0 0 1 0 \%$ IS (MR0.8 to MR5.0).

The linear regression analyses were performed for first, second and third order polynomials Per CLSI EP6-A. The data was considered linear if the non-linear coefficients from the second and third order polynomials were insignificant (p $> 0 . 0 5 )$ . When significant non-linear coefficients were observed, the Degree of Nonlinearity was calculated per CLSI EP6-A. The linear regression curves for both transcripts in the two collection tube types are shown in Figure 1A and 1B below. The estimated regression intercepts and slopes from the linear model are shown in Table 7.

Transcript e13a2 was linear from MR 0.93 to MR 5.1 with a maximum SD of 0.22 using EDTA tubes and was linear from MR 0.78 to MR 4.8 with a maximum SD of 0.13 using PAXgene tubes. Transcript e14a2 was linear from MR 0.99 to MR 5.0 with a maximum SD of 0.26 using EDTA tubes and was linear from MR 0.93 to MR 4.9 with a maximum SD of 0.31 using PAXgene tubes. The data support the assay’s linearity across the indicated detection range.

![](images/120517ebb41c992146651df1f25cc6a2be22fa7a308c98aea8d6b824df275212.jpg)  
Figure 1A: Linearity- EDTA Tube

Table 7: Regression Coefficients from Linear Model   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Transcript</td><td rowspan=1 colspan=2>Intercept</td><td rowspan=1 colspan=2>Slope</td></tr><tr><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=2 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>-0.045</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.017</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.026</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>0.008</td></tr><tr><td rowspan=2 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>-0.016</td><td rowspan=1 colspan=1>0.037</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.011</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.059</td><td rowspan=1 colspan=1>0.046</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.015</td></tr></table>

The lower and upper limits of linearity for BCR-ABL/ABL $\%$ IS and MR values for each tube type and BCR-ABL transcript were shown in Table 8 below. The linear range was truncated by the LoQ to determine assay range. The data support the conclusion that the assay linearity across the reportable range of MR1.0 to MR4.5 (shown in Table 9 below).

Table 8: Assay Linear Range   

<table><tr><td rowspan=1 colspan=1>Tube Type</td><td rowspan=1 colspan=1>BCR-ABLTranscript</td><td rowspan=1 colspan=1>Lower Limit(% IS)</td><td rowspan=1 colspan=1>Lower Limit(MR)</td><td rowspan=1 colspan=1>Upper Limit(% IS)</td><td rowspan=1 colspan=1>Upper Limit(MR)</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.00080</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.93</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>0.0010</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.0017</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>0.0012</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.93</td></tr></table>

Table 9: Assay Report Range   

<table><tr><td rowspan=1 colspan=1>Tube Type</td><td rowspan=1 colspan=1>Linear RangeBCR-ABL/ABL % IS</td><td rowspan=1 colspan=1>Linear RangeMolecular Response</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>10% to 0.0032%</td><td rowspan=1 colspan=1>MR1.0 to MR4.5</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>10% to 0.0032%</td><td rowspan=1 colspan=1>MR1.0 to MR4.5</td></tr></table>

e) Detection Limit:

i. Limit of Blank (LoB):

The LoB was determined by testing 30 BCR-ABL negative samples. Samples were tested in duplicate using 3 assay kit lots for a total of 180 replicates. Out of the 180 measurements, 177 had no measurable BCRABL values. Three $( 1 . 7 \% )$ had measurements well below the LoD of the test which were reported as undetected.

# ii. Limit of Detection (LoD):

The LoD was determined using 6 CML patient specimens (3 specimens each for e13a2 and e14a2 transcripts). RNA of each sample was extracted with both PAXgene Blood RNA Tubes and EDTA blood collection tubes and serial diluted into RNA extracted from non-diseased subject blood to create five level panels. Sufficient RNA for each panel was made to allow for the testing of 60 total replicates (20 replicates x 3 lots) of each level in the panel. There were 1725 and 1595 valid EDTA and PAXgene samples tested respectively.

The LoD was calculated using the parametric analysis as described in CLSI EP17-A2. Using EDTA blood collection tubes, the LoDs were estimated to be $0 . 0 0 0 1 9 \%$ IS (MR5.7) for e13a2 and $0 . 0 0 0 2 9 \%$ IS (MR5.5) for e14a2. Using PAXgene Blood RNA Tubes, the LoDs were estimated to be $0 . 0 0 0 3 9 \%$ IS (MR5.4) for $\mathrm { e } 1 3 \mathrm { a } 2$ and $0 . 0 0 0 2 9 \%$ IS (MR 5.5) for e14a2. The final LoD was determined to be $0 . 0 0 0 2 9 \%$ IS (MR5.5) for EDTA blood collection tubes and $0 . 0 0 0 3 9 \%$ IS (MR5.4) for PAXgene Blood RNA Tubes for both transcripts.

# iii.Limit of Quantification (LoQ):

The same samples in the LoD study were used to establish LoQ. The average and standard deviation for the BCR-ABL/ABL $\%$ IS values and MR values were calculated for the replicates for each level for both extraction methods, BCR-ABL transcripts, and all 3 lots. The $\log _ { 1 0 }$ bias between the nominal values and the MRDx BCR-ABL Test for each level was calculated by subtracting the average measured BCR-ABL/ABL $\%$ IS $\log _ { 1 0 }$ for each level from the BCR-ABL/ABL $\%$ IS $\log _ { 1 0 }$ nominal value. The Total Error (TE) was calculated using the following equation: Total Error $= | \mathrm { B i a s } | + 2 \mathrm { ~ x ~ } \mathrm { S }$ D.

The results were evaluated against the acceptance criteria of Total Error≤ $0 . 5 \log _ { 1 0 }$ . The LoQ for EDTA blood collection tubes was $0 . 0 0 1 6 \%$ IS (MR4.8). The LoQ for PAXgene Blood RNA Tubes was $0 . 0 0 2 5 \%$ IS (MR4.6).

The MRDx BCR-ABL Test Software limits the LoQ and quantitated results to $0 . 0 0 3 2 \%$ IS (MR4.5) for both blood tube types.

f) Analytical specificity:

i. Interfering Substances:

For endogenous blood substance interference testing, CML samples at 2 levels (MR3.0 and MR4.5) extracted from both EDTA and PAXgene collection tubes were spiked with all the following test substances:

Hemoglobin $2 \mathrm { m g / m L }$ Bilirubin $3 4 2 \mu \mathrm { m o l / L }$ Triglycerides $2 8 8 \mathrm { m g / d L }$

When compared to test results from samples spiked with matrix control (i.e., cleared serum and water), no clinically significant difference was observed from the samples spiked with the potential interference substances, although reduced RNA yield during extraction was observed in some samples. The largest difference between the endogenous substance spike and the matrix control spike for PAXgene Blood RNA Tubes was - $- 0 . 0 8 0 \log _ { 1 0 }$ ( $9 5 \%$ CI: -0.32 to 0.16) at MR4.5. The largest difference between the exogenous substance spiked group and the matrix control group for EDTA blood collection tubes was $0 . 0 6 0 \log _ { 1 0 }$ ( $9 5 \%$ CI: -0.042 to 0.16) at MR4.5.

For exogenous interference testing, extraction kit reagents used in the RNA extraction processes for PAXgene Blood RNA tubes and EDTA blood collection tubes were examined to determine if any interference from the blood tubes and RNA extraction processes were observed with the MRDx BCR-ABL Test. Ten replicates of the MR3.0 and MR4.5 RNA controls were spiked and tested together with ten replicates of the MR3.0 and MR4.5 RNA Controls spiked with water as a matrix control sample. No clinically significant difference from the matrix control samples was observed. The largest difference between the exogenous substance spiked group and the matrix control group for PAXgene Blood RNA Tubes was $0 . 0 4 0 \log _ { 1 0 }$ ( $9 5 \%$ CI: -0.010 to 0.090) at MR3.0. The largest difference between the exogenous substance spiked group and the matrix control group for EDTA blood collection tubes was $0 . 0 1 0 \log _ { 1 0 }$ $9 5 \%$ CI: -0.022 to 0.22) at MR4.5.

The data support the conclusion that none of the potential interferants evaluated had any significant interference with the assay.

# ii. Primer Specificity:

RNA from 6 CML patient samples (3 samples each for the e13a2 and e14a2 transcript) were extracted and used as template for the MRDx BCRABL Test. The resulting PCR products for both BCR-ABL and ABL were sub-cloned into a cloning vector and ten10 colonies were sequenced. The sequencing results were used to create a consensus sequence to compare to the established Genbank sequence for ABL and both BCR-ABL transcripts. The results matched the Genbank sequence with agreement across all bases demonstrating that the MRDx primers and probes amplified the intended targets for all samples tested.

Cross reactivity was further accessed by testing ABL2 in vitro transcribed RNA and RNA from an e19a2 positive cell line as template. No cross reactivity was observed with the ABL2 IVT RNA. As expected, cross reactivity was present for the e19a2 BCR-ABL transcript as it contained the binding sites for the primers and probe in BCR exon 13 and ABL exon 2. The cross reactivity of the MRDx BCR-ABL Test with the e19a2 BCRABL transcript was mitigated by a precaution statement in the labeling that the test is only indicated for use with CML patients diagnosed with the e13a2 and/or e14a2 BCR-ABL transcript and should not be used with samples expressing other atypical fusion transcripts.

# iii. Specimen Carryover Contamination:

The carryover contamination study was performed by testing high positive samples (near MR1.0) alternately juxtaposed with negative samples in both extraction rack setup as well as MRDx plate setup for both PAXgene Blood RNA Tubes and EDTA blood collection tubes.

The incidence of cross-contamination with PAXgene Blood RNA tubes of samples was determined using 30 replicates across 5 extraction racks and 5 MRDx BCR-ABL Test plates. The incidence of cross-contamination with EDTA blood collection tubes was determined using the Maxwell CSC RNA Blood Kit with 40 replicates across 5 extraction racks and 5 MRDx BCR-ABL test plates.

No negative samples for either extraction method had a BCR-ABL/ABL $\%$ IS value that was detectable above the LoD of the assay. The data support the conclusion that the test generates no significant carryover between wells on the plate.

g) Traceability:

The assay calibrators are traceable to the First $( 1 ^ { \mathrm { s t } } )$ WHO International Genetic Reference Panel based on the method described in the $I ^ { s t }$ WHO International Genetic Reference Panel for quantitation of BCR-ABL translocation by RQ-PCR, Instruction for Use. On 5 different days, a different WHO primary panel comprised of four levels (A-D) of freeze-dried materials was reconstituted and extracted following the assay protocol. The reconstituted/extracted WHO reference samples (200 ng RNA input/well) and the MolecularMD WHO BCR-ABL Reference Panel Secondary Standards (1 $\mu \mathrm { g }$ RNA input/well) were tested using the MRDx BCR-ABL Test over 20 days for each of three MRDx BCR-ABL Test kit lots.

Deming Regression analysis were performed between the expected and observed MR values for each lot. The conversion factor was calculated for each lot individually and was 1.1 for all three lots (A, B and C), showing the consistency of conversion factor across multiple lots. Data from all three lots showed high traceability to WHO reference standards (see Figure 2 and Table 10 below), supporting the conclusion that the assay calibrators traceability to the WHO reference panel has been established.

![](images/ee0529bf51006075d2cedbf808a25bf5eb48a66e292c1a7c61cd556047984fd2.jpg)  
Figure 2: Deming Regression for WHO Standards by Lot

Table 10: Regression Statistics for Each Lot   

<table><tr><td rowspan=1 colspan=1>Metrics</td><td rowspan=1 colspan=1>Lot A</td><td rowspan=1 colspan=1>Lot B</td><td rowspan=1 colspan=1>Lot C</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>95% Lower CI (slope)</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.98</td></tr><tr><td rowspan=1 colspan=1>95% Upper CI (slope)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Y-Intercept</td><td rowspan=1 colspan=1>-0.048</td><td rowspan=1 colspan=1>-0.034</td><td rowspan=1 colspan=1>-0.057</td><td rowspan=1 colspan=1>-0.046</td></tr><tr><td rowspan=1 colspan=1>95% Lower CI(intercept)</td><td rowspan=1 colspan=1>-0.065</td><td rowspan=1 colspan=1>-0.057</td><td rowspan=1 colspan=1>-0.082</td><td rowspan=1 colspan=1>-0.059</td></tr><tr><td rowspan=1 colspan=1>95% Upper CI(intercept)</td><td rowspan=1 colspan=1>-0.030</td><td rowspan=1 colspan=1>-0.010</td><td rowspan=1 colspan=1>-0.033</td><td rowspan=1 colspan=1>-0.034</td></tr><tr><td rowspan=1 colspan=1>Correlation</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td></tr></table>

# h) Stability:

# i. Kit Real-Time Stability:

The real-time stability of the MRDx BCR-ABL Test was evaluated at two storage temperatures (-30 to $- 1 5 \mathrm { { } ^ { \circ } C }$ and -80 to $- 6 5 ^ { \circ } \mathrm { C } ,$ ) at periodic intervals with three lots. Six kits from each lot for each time point were tested using the final product release testing procedure. After use, the kits were frozen, thawed, and refrozen without testing to achieve an additional freeze/thaw cycle (Cycle 2). These six kits were then openvial tested at one month (Test 2 and Cycle 3), and two months (Test 3 and Cycle 4) post-reconstitution using the WHO BCR-ABL Reference Panel Secondary Standards to ensure the kits were stable for three kit uses with three freeze/thaw cycles with 2 month stability. Kits were stored at $- 3 0$ to $- 1 5 \mathrm { { } ^ { \circ } C }$ after initial use for open-vial stability, regardless the original storage condition.

![](images/90856ac552e2821421beff81449e4d914505f4c28704616a201fef0fb0d4dcdc.jpg)

For each time point, the MRDx kit was tested for performance of the calibration curve criteria, the control results, and the WHO secondary standards. Passing test results data exist for at least 16 months for 3 lots of MRDx BCR-ABL Test stored at the two separate temperatures.

The data support a shelf life of 15 months at both storage temperatures (-80 to ${ } _ { - 6 5 } { } ^ { \circ } \mathrm { C }$ and -30 to $- 1 5 ^ { \circ } \mathrm { C }$ ) and an open vial stability of 3 freezethaw cycles and up to two 2 months following initial use.

# ii. Kit In-Use Stability (Freeze-thaw):

This study is to demonstrate the MRDx BCR-ABL Test kit stability during PCR plate preparation and the post-plate-assembly PCR reaction at 2 to $8 \mathrm { ^ \circ C }$ before evaluation on the ABI 7500 Fast Dx instrument.

Both standard and extreme conditions were tested. The standard condition used a newly opened kit immediately after thawing to setup a PCR plate. The PCR plate was processed immediately after setup on the ABI 7500 Fast Dx instrument. The extreme condition used a kit that was thawed and refrozen twice to simulate the third use of the kit. After the reagents were thawed, components were stored at 2 to $8 \mathrm { ^ \circ C }$ for 8 hours, and then the PCR plate was assembled and placed at 2 to $8 \mathrm { ^ \circ C }$ for 1.5 hours before evaluation on the ABI 7500 Fast Dx instrument. Two (2) kits were run for each condition on each of 2 days with 2 sets of calibration curves, RNA controls, and samples (total $\scriptstyle \mathrm { n = } 8$ per condition).

The acceptance criteria were that the mean of the test samples $\mathrm { w a s } \pm 0 . 5$ MR of the control. The acceptance criteria were met for both standard and extreme conditions. The differences between the standard condition and the extreme condition were less than $0 . 2 5 \log _ { 1 0 }$ for all levels of samples tested. The largest difference observed between conditions was -0.063 ( $9 5 \%$ CI: -0.24 to 0.12) for the MR4.5 sample. These results support the stability conclusion that after 2 freeze-thaw cycles, the

reagent is stable at 2 to $8 \mathrm { { } ^ { \circ } C }$ for up to eight hours before PCR reagent assembly and up to 1.5 hours after PCR plate assembly.

# iii. Kit Shipping Stability:

In this study, the MRDx BCR-ABL Test kits were evaluated with multiple types of containers and shipping conditions. The mock shipping conditions included both stress temperatures and extended duration. The shipping conditions evaluated the minimum and maximum payload. Maximum load was the worst case regarding the amount of dry ice per shipping condition. The testing conditions were executed sequentially as follows:

· $2 2 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C }$ for 4 hours · $3 5 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C }$ for 6 hours · $3 0 ^ { \circ } C \pm 3 ^ { \circ } C$ for 56 hours $3 5 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C }$ for 6 hours

The results of this study demonstrated that the Polar Tech TC229 (P/N 049-0128) and BioFreeze 22L-133Hr (P/N 049-0129) shipping containers can maintain an internal temperature below ${ } _ { - 6 5 } { } ^ { \circ } \mathrm { C }$ for all load conditions during a 72-hour temperature stress test that simulated typical shipping conditions, and the MRDx BCR-ABL Test kits remained at their recommended storage conditions of -80 to ${ } _ { - 6 5 } \mathrm { \Omega } ^ { \circ } \mathrm { C }$ .

Functional testing using samples at MR3.0-MR4.5 also showed that the kits performance after shipping stress met equivalency acceptance criteria when compared to unstressed control kits. These results support the conclusion that the assay kit is stable following shipment of up to 72 hours on dry ice.

# iv. Specimen Stability:

For PAXgene Blood RNA Tube, samples at three levels (MR3.0, MR4.0, and MR4.5) were created, stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ and tested in quadruplicate at 0, 24, 48, and 60 hours. An additional series of samples was created, stored at $- 3 0$ to ${ } _ { - 1 5 } { } ^ { \circ } \mathrm { C }$ , and tested in quadruplicate at 0, 14, 28, and 30 days. For EDTA collection tube, 17 CML patient specimens covering the clinical decision range were collected and stored at 2 to $8 \mathrm { { } ^ { \circ } C }$ and tested in quadruplicate at 0, 24, 48, and 60 hours. All tested samples passed the acceptance criteria of $< 0 . 2 5 \mathrm { l o g } _ { 1 0 }$ difference compared to the mean BCR-ABL/ABL IS $\log _ { 1 0 }$ ratio results at the T0.

These results support the following stability conclusion:

· Specimens collected in PAXgene Blood RNA Tube are stable for at least 48 hours at 2 to $8 \mathrm { ^ \circ C }$ and at least 28 days at -30 to $- 1 5 \mathrm { { } ^ { \circ } C }$ . Specimens collected in EDTA blood tubes are stable for at least 48 hours at 2 to $8 \mathrm { { } ^ { \circ } C }$ .

# 2. Method Comparison Study:

The accuracy of detecting BCR-ABL transcript levels by MRDx assay was further evaluated by comparing to a second reference method, a validated BCRABL reverse transcription droplet digital PCR (ddPCR) assay. The study analyzed a total of 217 samples of blood from CML patients ranging from $10 \%$ BCRABL/ABL IS (MR1) to $0 . 0 0 1 \%$ BCR-ABL/ABL IS (MR5.0) drawn into PAXgene Blood RNA tubes and extracted using the PAXgene Blood RNA Kit as well as samples drawn into EDTA blood collection tubes and extracted using the Maxwell CSC RNA Blood Purification Kit with the Maxwell CSC instrument. A total of 119 evaluable EDTA samples and 98 evaluable PAXgene samples were tested with both tests.

A Deming regression was applied and $9 5 \%$ confidence intervals of the slope and y-intercept were calculated. The predicted systematic difference between the two methods was calculated at the clinical decision points for treatment-free remission monitoring (MR3.0, MR4.0, and MR4.5) with $9 5 \%$ confidence intervals. High concordance between MRDx test results and the reference ddPCR test results were observed for both EDTA and PAXgene tube types (see Figure 3, Table 11 and Table 12 below), supporting the detection accuracy of the device.

![](images/7e3b140852b1b7ca70ceb13fe220bc6878690a798ec357ac0b63ec42d26276b4.jpg)  
Figure 3: Deming Regression Plot (MRDx vs. ddPCR)

Table 11: Deming Regression (MRDx vs. ddPCR)   

<table><tr><td rowspan=1 colspan=1>Blood TubeType</td><td rowspan=1 colspan=1>SampleSize</td><td rowspan=1 colspan=1>Deming Slope</td><td rowspan=1 colspan=1>DemingY-Intercept</td><td rowspan=1 colspan=1>PearsonCoefficient</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0.9695% CI [0.94, 0.99]</td><td rowspan=1 colspan=1>0.07795% CI[[0.0027, 0.17]</td><td rowspan=1 colspan=1>0.984</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>0.9895% CI [0.95, 1.0]</td><td rowspan=1 colspan=1>-0.02195% CI [-0.11, 0.056]</td><td rowspan=1 colspan=1>0.985</td></tr></table>

Table 12: Predicted Difference at Clinical Decision Points   

<table><tr><td rowspan=1 colspan=1>Tube Type</td><td rowspan=1 colspan=1>Predicted DifferenceMR3 (MMR)</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.0</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.5</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>-0.03195% CI, [-0.053, -0.0049]</td><td rowspan=1 colspan=1>-0.06795% CI [-0.10, -0.031]</td><td rowspan=1 colspan=1>-0.08595% CI [-0.13, -0.039]</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>-0.08895% CI [-0.11, -0.062]</td><td rowspan=1 colspan=1>-0.1195% CI [-0.15, -0.071]</td><td rowspan=1 colspan=1>-0.1295% CI [-0.17, -0.071]</td></tr></table>

# 3. Clinical Performance:

To support the indications for use, 2 clinical studies were conducted to demonstrate the effectiveness of the device in identifying patients with chronic phase (CP) CML eligible for nilotinib (TASIGNA $\textsuperscript { \textregistered }$ ) treatment discontinuation and continuous monitoring of patients off treatment for potential loss of remission.

# Study 1: Treatment discontinuation in newly diagnosed $\mathbf { P h } +$ CML-CP patients who have achieved a sustained molecular response (MR4.5)

The ENESTfreedom (Evaluating Nilotinib Efficacy and Safety in clinical Trialsfreedom) study (NCT01784068) is an open-label, multicenter, single-arm study, where 215 adult patients with $\mathrm { P h } +$ CML-CP treated with nilotinib in first-line for $\geq 2$ years who achieved MR4.5 as measured with the MolecularMD MRDx™ BCR-ABL Test were enrolled to continue nilotinib treatment for an additional 52 weeks (nilotinib consolidation phase).

Of the 215 patients, 190 patients $( 8 8 . 4 \% )$ entered the “Treatment-free Remission” (TFR) phase after achieving a sustained molecular response (MR4.5) during the consolidation phase, defined by the following criteria:

The 4 last quarterly assessments (taken every 12 weeks) were at least MR4 (BCR-ABL/ABL   
$\leq 0 . 0 1 \%$ IS), and maintained for 1 year   
The last assessment being MR4.5 ( $\mathrm { B C R - A B L / A B L } \leq 0 . 0 0 3 2 \%$ IS) No more than two assessments falling between MR4 and MR4.5 ( $0 . 0 0 3 2 \%$ IS $<$ BCR-ABL/ABL $\leq 0 . 0 1 \%$ IS).

The median age of patients who entered the TFR phase was 55 years, $4 9 . 5 \%$ were females, and $2 1 . 1 \%$ of the patients were $\ge 6 5$ years of age. BCR-ABL levels were monitored every 4 weeks during the first 48 weeks of the TFR phase. Monitoring frequency was intensified to every 2 weeks upon the loss of MR4.0. Biweekly monitoring ended at one of the following time points:

Loss of MMR requiring patient to re-initiate nilotinib treatment When the BCR-ABL levels returned to a range between MR4.0 and MR4.5 When the BCR-ABL levels remained lower than MMR for 4 consecutive measurements (8 weeks from initial loss of MR4.0).

Any patient with loss of MMR during the TFR phase re-initiated nilotinib treatment at $3 0 0 \mathrm { m g }$ twice daily or at a reduced dose level of $4 0 0 \mathrm { m g }$ once daily if required from the perspective of tolerance, within 5 weeks after the collection date of the blood sample demonstrating loss of MMR. Patients who required reinitiation of nilotinib treatment were monitored for BCR-ABL levels every 4 weeks for the first 24 weeks and then every 12 weeks thereafter in patients who regained MMR.

Efficacy was based on the 96-week analysis data cut-off date, by which time, 91 patients $( 4 7 . 9 \% )$ discontinued from the TFR phase due to loss of MMR, and 1 $( 0 . 5 \% )$ , 1 $( 0 . 5 \% )$ , 1 $( 0 . 5 \% )$ and 3 patients $( 1 . 6 \% )$ due to death from unknown cause, physician decision, lost to follow-up and subject decision, respectively. Among the 91 patients who discontinued the TFR phase due to loss of MMR, 88 patients restarted nilotinib treatment and 3 patients permanently discontinued from the study.

By the 96-week data cut-off, of the 88 patients who restarted treatment due to loss of MMR in the TFR phase, 87 patients $( 9 8 . 9 \% )$ patients regained MMR (one patient discontinued study permanently due to subject decision after 7.1 weeks of retreatment without regaining MMR) and 81 patients $( 9 2 . 0 \% )$ regained MR4.5 by the time of the cut-off date. The cumulative rate of MMR and MR4.5 regained at 24 weeks since treatment re initiation was $9 7 . 7 \%$ (86/88 patients) and $8 6 . 4 \%$ (76/88 patients), respectively. The efficacy results for ENESTfreedom study are summarized in Table 13.

Table 13: Efficacy Results for ENESTfreedom   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Patients in TFR phaseat the specified time point</td><td rowspan=1 colspan=1>Loss of MMR2 by the specifiedtime point</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>24 weeks</td><td rowspan=1 colspan=1>62.1%</td><td rowspan=1 colspan=1>(54.8, 69.0)</td><td rowspan=1 colspan=1>35.8%</td></tr><tr><td rowspan=1 colspan=1>48 weeks</td><td rowspan=1 colspan=1>51.6%</td><td rowspan=1 colspan=1>(44.2, 58.9)</td><td rowspan=1 colspan=1>45.8%</td></tr><tr><td rowspan=1 colspan=1>96 weeks</td><td rowspan=1 colspan=1>48.9%</td><td rowspan=1 colspan=1>(41.6, 56.3)</td><td rowspan=1 colspan=1>47.9%</td></tr></table>

1 Patients in MMR at the specified time point in the TFR phase 2 Based on the time to event (loss of MMR) data during the TFR phase

Among the 190 patients in the TFR phase, 98 patients had a treatment-free survival (TFS) event (defined as discontinuation from TFR phase due to any reason, loss of MMR, death due to any cause, progression to AP/BC up to the end of TFR phase, or re-initiation of treatment due to any cause in the study) by the 96-week cut-off date. The Kaplan-Meier estimate of treatment-free survival for is shown in Figure 4.

![](images/ea9c44cd915f63eb86757cf64cdf6322059be26d6c4b7a389563c52771cc0bcb.jpg)  
Figure 4: Kaplan-Meier estimate of treatment-free survival after start of TFR (Full Analysis Set ENESTfreedom)   
Time since TFR (weeks)

-190  Ve 120 :70 99 : 91 95 : 95 75 : 97 8 : 97 0 : 98

1For a given time point, the points on the dashed curves represent the $9 5 \%$ confidence limits for the associated KM estimate on the solid curve.

2By the time of the 96-week data cut-off date, one single patient lost MMR at week 120, at the time when only 8 patients were considered at risk. This explains the artificial drop at the end of the curve.

# Study 2: Treatment discontinuation in $\mathbf { P h } +$ CML-CP patients who have achieved a sustained molecular response (MR4.5) on nilotinib following prior imatinib therapy

The ENESTop (Evaluating Nilotinib Efficacy and Safety in clinical Trials-STop) study (NCT01698905) is an open-label, multicenter, single-arm study, where 163 adult patients with $\mathrm { P h } +$ CML-CP taking tyrosine kinase inhibitors (TKIs) for $\geq 3$ years (imatinib as initial TKI therapy for more than 4 weeks without documented MR4.5 on imatinib at the time of switch to nilotinib, then switched to nilotinib for at least 2 years), and who achieved MR4.5 on nilotinib treatment as measured with the MolecularMD MRDx™ BCR-ABL Test were enrolled to continue nilotinib treatment for an additional 52 weeks (nilotinib consolidation phase). Of the 163 patients, 126 patients $( 7 7 . 3 \% )$ entered the TFR phase after achieving a sustained molecular response (MR4.5) during the consolidation phase, defined by the following criterion:

The 4 last quarterly assessments (taken every 12 weeks) showed no confirmed loss of MR4.5 (BCR-ABL/ABL $\leq 0 . 0 0 3 2 \%$ IS) during 1 year.

The median age of patients who entered the TFR phase was 56 years, $5 5 . 6 \%$ were females, and $2 7 . 8 \%$ of the patients were $\ge 6 5$ years of age. The median actual dose intensity during the 52-week nilotinib consolidation phase was $7 7 1 . 8 \mathrm { m g / d a y }$ with $5 2 . 4 \%$ , $2 9 . 4 \%$ , $0 . 8 \%$ , $1 6 . 7 \%$ and $0 . 8 \%$ of patients receiving a daily nilotinib dose of $8 0 0 \mathrm { m g }$ , $6 0 0 \mathrm { m g }$ , 450mg, $4 0 0 \mathrm { m g }$ and $3 0 0 \mathrm { m g }$ just before entry into the TFR phase, respectively.

Patients who entered the TFR phase but experienced two consecutive measurements of BCR-ABL/ABL $> 0 . 0 1 \%$ IS were considered having a confirmed loss of MR4.0, triggering re-initiation of nilotinib treatment. Patients with loss of MMR in the TFR phase immediately restarted nilotinib treatment without confirmation. All patients who restarted nilotinib therapy had BCR-ABL transcript levels monitored every 4 weeks for the first 24 weeks, then once every 12 weeks.

Efficacy was based on the 96-week analysis data cut-off date, by which time, 61 patients $( 4 8 . 4 \% )$ had discontinued from the TFR phase: 58 patients $( 4 6 . 0 \% )$ due to loss of MMR or confirm loss of MR4.0, 2 patients $( 1 . 6 \% )$ due to subject/guardian decision and one patient $( 0 . 8 \% )$ due to pregnancy. Among the 58 patients who discontinued from the TFR phase due to confirmed loss of MR4.0 or loss of MMR, 56 patients restarted nilotinib therapy and 2 patients permanently discontinued from the study.

By the 96-week data cut-off, of the 56 patients who restarted nilotinib treatment due to confirmed loss of MR4.0 or loss of MMR in the TFR phase, 52 patients $( 9 2 . 9 \% )$ regained MR4.0 and MR4.5; 4 patients $( 7 . 1 \% )$ did not regain MR4.0 by the time of the cut-off date. The cumulative rate of MR4 and MR4.5 regained by 48-weeks since treatment re-initiation, was $9 2 . 9 \%$ (52/56 patients) and $9 1 . 1 \%$ (51/56 patients), respectively. The efficacy results for ENESTop study are summarized in Table 14.

Table 14: Efficacy Results for ENESTop   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Patients in TFR phaseat the specified time point</td><td rowspan=1 colspan=1>Loss of MMR or confirmed loss ofMR4² by the specified time point</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>(%)</td></tr><tr><td rowspan=1 colspan=1>24 weeks</td><td rowspan=1 colspan=1>60.3%</td><td rowspan=1 colspan=1>(51.2, 68.9)</td><td rowspan=1 colspan=1>38.9%</td></tr><tr><td rowspan=1 colspan=1>48 weeks</td><td rowspan=1 colspan=1>57.9%</td><td rowspan=1 colspan=1>(48.8, 66.7)</td><td rowspan=1 colspan=1>41.3%</td></tr><tr><td rowspan=1 colspan=1>96 weeks</td><td rowspan=1 colspan=1>53.2%</td><td rowspan=1 colspan=1>(44.1, 62.1)</td><td rowspan=1 colspan=1>43.7%</td></tr></table>

1 Patients without loss of MMR or confirmed loss of MR4 by specified time point of TFR phase 2 Based on the time to event (loss of MMR or confirmed loss of MR4) data during the TFR phase

Among the 126 patients in the TFR phase, 61 patients $( 4 8 . 4 \% )$ had a treatmentfree survival (TFS) event (defined as discontinuation from TFR phase due to any reason, loss of MMR, confirmed loss of MR4, death due to any cause, progression to AP/BC up to the end of TFR phase, or re-initiation of treatment due to any cause in the study) on or before the 96-month cut-off date. The Kaplan-Meier estimate of treatment-free survival for the ENESTop study is shown in Figure 5.

![](images/37f97eb19d028262ced25862c49188fe20e6f9f4b26a833d333b528cd80f8a23.jpg)  
Figure 5: Kaplan-Meier estimate of treatment-free survival after start of TFR (Full Analysis Set ENESTop)

At-risk : Events Time since TFR (weeks) 126:0 107:19 76:50 74:52 73:53 72:54 71:55 70:56 55:58 32:60 13:61 1:61 0:61 1For a given time point, the points on the dashed curves represent the $9 5 \%$ confidence limits for the associated KM estimate on the solid curve.

Refer to the TASIGNA label available at Drugs $@$ FDA for more information.

# N. Instrument Name:

Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument

# O. System Description:

1. Modes of Operation:

The Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument using System Sequence Detection Software v1.4.1 is designed to accept 96-well microtiter plates. The instrument outputs quantitative results based on the measured fluorescence signals.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No ___

3. Calibration and Quality Controls Refer to Device Description and Traceability sections above.

# P. Other Supportive Performance Characteristics Data

1. PAXgene-EDTA Tube Equivalency:

A blood matrix equivalency study using 100 CML patient paired blood samples was conducted to demonstrate that MRDx BCR-ABL Test results from RNA samples extracted from PAXgene Blood RNA Tubes are the same as from EDTA blood collection tubes. At least 4 PAXgene Blood RNA Tubes and two $6 \mathrm { m L }$ EDTA blood collection tubes were collected for each subject. RNA was extracted from the PAXgene Blood RNA Tubes using the PAXgene Blood RNA Kit. RNA from the EDTA blood collection tubes was extracted using the Maxwell CSC Blood RNA Kit according to the device’s Instructions For Use.

For all paired samples that were above the LoQ of the test, a Deming regression was applied and $9 5 \%$ confidence intervals of the slope and y-intercept were calculated. The predicted systematic difference between the two methods was calculated at the clinical decision points (MR3.0, MR4.0, and MR4.5) with $9 5 \%$ confidence intervals. The data demonstrated that no clinically significant bias at the clinical decision points was observed in the MR values from MRDx assay results between the two blood collection tube types. Results are shown in Table 15 and Table 16 below.

Table 15: Deming Regression (EDTA vs. PAXgene)   

<table><tr><td rowspan=1 colspan=1>SampleSize</td><td rowspan=1 colspan=1>Deming Slope</td><td rowspan=1 colspan=1>DemingY-Intercept</td><td rowspan=1 colspan=1>DemingPearson Coefficient</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.0295% CI [1.00, 1.05]</td><td rowspan=1 colspan=1>-0.11595% CI [-0.202, -0.042]</td><td rowspan=1 colspan=1>0.992</td></tr></table>

Table 16: Predicted Difference at Clinical Decision Points   

<table><tr><td rowspan=1 colspan=1>Predicted Difference(MMR/MR3)</td><td rowspan=1 colspan=1>Predicted DifferenceMR4</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.5</td></tr><tr><td rowspan=1 colspan=1>-0.043395% CI [-0.0687, -0.0175]</td><td rowspan=1 colspan=1>-0.019295% CI [-0.0564, 0.0238]</td><td rowspan=1 colspan=1>-0.0072195% CI [-0.0549, 0.0462]</td></tr></table>

# 2. CTA-IVD MRDx Equivalency

During the early stage of consolidation phase for the 2 clinical studies described above in Section M.3, a prototype (CTA) of the MRDx assay was used. Samples at later stage of the consolidation phase and all samples in TFR phase of the studies were tested using the final (IVD) version of the MRDx assay.

To demonstrate equivalency between the prototype CTA and IVD MRDx test, over 100 paired CML specimens across the assay reportable (MR) range were tested using both CTA and IVD MRDx assays. This study was performed for both EDTA and PAXgene collection tubes. Deming regression analysis demonstrated a high concordance between the CTA and IVD MRDx test results for both collection tube types across the assay reportable range (shown in Figure 6, Table 17 and Table 18 below), supporting the conclusion that the CTA and IVD version of the MRDx tests are equivalent.

![](images/69bcefdb5a64d8847a63ac3c5f473118d448556b8bd930842603279ae128cd99.jpg)  
Figure 6: Deming Regression Plot (CTA vs. IVD results)

Table 17: Deming Regression Results (CTA vs. IVD)   

<table><tr><td rowspan=1 colspan=1>SampleTypes</td><td rowspan=1 colspan=1>SampleSize</td><td rowspan=1 colspan=1>Deming Slope(MR)</td><td rowspan=1 colspan=1>Deming Y-Intercept(MR)</td><td rowspan=1 colspan=1>PearsonCoefficient (MR)</td></tr><tr><td rowspan=1 colspan=1>1 μg EDTARNA input</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>0.99895% CI (0.989, 1.01)</td><td rowspan=1 colspan=1>0.025195% CI (0.000187, 0.0509)</td><td rowspan=1 colspan=1>0.999</td></tr><tr><td rowspan=1 colspan=1>2 μg PAXgeneRNA input</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0.99195% CI (0.982, 1.00)</td><td rowspan=1 colspan=1>0.038095% CI (0.00993, 0.0663)</td><td rowspan=1 colspan=1>0.999</td></tr></table>

Table 18: Predicted Difference at Clinical Decision Points (CTA vs. IVD)   

<table><tr><td rowspan=1 colspan=1>Tube Type</td><td rowspan=1 colspan=1>Predicted Difference(MR3/MMR)</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.0</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.5</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>0.017695% CI [0.00954, 0.0257]</td><td rowspan=1 colspan=1>0.015295% CI [0.0041, 0.0263]</td><td rowspan=1 colspan=1>0.013995% CI [-0.00042, 0.0282]</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>0.012295% CI [0.00268, 0.0221]</td><td rowspan=1 colspan=1>0.0035695% CI [-0.0119, 0.019]</td><td rowspan=1 colspan=1>-0.00073895% CI [-0.0205, 0.0198]</td></tr></table>

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.