MolecularMD Corporation   
Kevin Hawkins   
Director, Quality and Regulatory Affairs   
1341 SW Custer Drive   
Portland, OR 97219

Re: K173492 Trade/Device Name: MolecularMD MRDx BCR-ABL Test Regulation Number: 21 CFR 866.6060 Regulation Name: BCR-ABL Quantitation Test Regulatory Class: Class II Product Code: OYX Dated: November 9, 2017 Received: November 13, 2017

Dear Kevin Hawkins:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice   
(https://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/) and CDRH Learn   
(http://www.fda.gov/Training/CDRHLearn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (http://www.fda.gov/DICE) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

# Reena Philip -S

Reena Philip, Ph.D.   
Director   
Division of Molecular Genetics and Pathology Office of In Vitro Diagnostics   
and Radiological Health   
Center for Devices and Radiological Health

Enclosure

Device Name MRDx $^ \mathrm { \textregistered }$ BCR-ABL Test

Indications for Use (Describe)

The MolecularMD MRDx BCR-ABL Test is an in vitro diagnostic test for the quantitative detection of BCR-ABL1 trancripts (13a2/b2a2 and/ore14a2/b3a2)and theABL1endogenous control mRNAin peripheral blood specimes from pats previusly diagose with t(9:22) positiv croicmyeloid leukemia(CML). Theratif BCR-ABL1 to ABL is an ralT  - v polymerase chain reaction performed on the Applied Biosystems 7500 Fast Dx instrument.

The MolecularMD MRDx BCR-ABL Test is intended to measure BCR-ABL mRNA transcript levels in patients diagnosed with t(9;22) positive CML during monitoring of treatment with Tyrosine Kinase Inhibitors (TKIs).

The device is also intended to be used in the serial monitoring for BCR-ABL mRNA transcript levels as an aid in identifying ML patients in the chronic phase being treated with nilotinib who may be candidatesor treatent discontinuation and for monitoring of treatment-free remission.

Th resulting from t(9;22). The test is not intended for the diagnosis of CML.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

Turenotioatinatveo time to reviw instructions, search existig data sources,gather and maintainhe dataneeded and cplee and review the colection of information. Send comments regarding this burden estimate or anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 510(k) SUMMARY

# I. COMPANY AND CONTACT INFORMATION

Company Name: Address:

Telephone: Fax: Contact Person:

Date of Summary:

MolecularMD Corporation   
1341 SW Custer Drive   
Portland, OR 97219   
503.459.4974   
503.459.4976   
Kevin Hawkins   
Director, Quality and Regulatory Affairs   
20 December 2017

# II. DEVICE IDENTIFICATION

Trade (Proprietary) Name:   
Common (Usual) Name:   
Classification Name:   
Product Code:   
Regulation Number:   
Regulatory Classification:   
Panel:   
MolecularMD MRDx® BCR-ABL Test   
BCR-ABL RT-qPCR Test   
BCR-ABL Quantitation Test   
OYX   
866.6060   
Class II   
88 - Pathology

# III. PREDICATE DEVICE

QuantideX® qPCR BCR-ABL IS Kit

# IV. DEVICE DESCRIPTION

# A. Principle of the Procedure - Test Methodology

The MolecularMD ${ \mathsf { M R D x } } ^ { \otimes }$ BCR-ABL Test is a quantitative, real-time polymerase chain reaction test that provides quantitation of BCR-ABL1 (hereafter BCR-ABL) transcript e13a2/b2a2 or e14a2/b3a2 and ABL1 (hereafter ABL) transcript levels in RNA extracted from peripheral blood samples collected from CML patients. Peripheral blood is collected in either EDTA or PAXgene Blood RNA Tubes. Each collection tube type requires a specified RNA extraction method and RNA input amount. The detailed extraction protocol and RNA quantity requirements are provided in the device package insert and instructions for use. Total RNA is extracted from peripheral blood and serves as the template for RT-qPCR. The test is performed using a onestep RT-qPCR protocol wherein the reverse transcription and quantitative, real-time PCR reactions are performed in the same well. BCR-ABL and ABL amplicons are generated and detected in real-time using TaqMan® MGB probes. For RNA extracted from EDTA tubes, a total of $4 \mu \ g$ of RNA is required for each patient sample: $1 \mu \ g$ of extracted RNA is used in the MRDx BCR-ABL Test in each of 4 wells, 2 wells for BCR-ABL quantitation and 2 wells for ABL quantitation. For RNA extracted from PAXgene tubes, a total of $8 \mu \ g$ of RNA is required for each patient sample: $2 \mu \ g$ of extracted RNA is used in the MRDx BCR-ABL Test in each of 4 wells, 2 wells for BCR-ABL quantitation and 2 wells for ABL quantitation.

The MolecularMD MRDx BCR-ABL Test is performed on the Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument. The instrument integrates a thermal cycler, a fluorometer and application specific software. The instrument houses the thermal cycler and the fluorometer, while the application software is installed on a computer that is attached to the instrument. The ABI 7500 Fast Dx instrument software v1.4.1 calculates the BCR-ABL and ABL copy numbers using a standard curve generated with calibrators on each individual plate.

Quantitation is achieved using RNA calibration standards and linear regression analysis provided by the ABI 7500 Fast Dx PCR Instrument Software. BCR-ABL transcript levels are measured in relation to the ABL transcript as an endogenous reference. The data are exported as a CSV file for further analysis in the ${ \mathsf { M R D x } } ^ { \otimes }$ BCR-ABL Test Software. The BCR-ABL/ABL ratio is calculated and converted to the International Scale by the MRDx BCR-ABL Test Software.

The MRDx BCR-ABL Test Software is used to analyze all test results. This software, provided with the MRDx BCR-ABL Test, is used to calculate the BCR-ABL/ABL $\%$ IS and MR value for patient sample using the conversion factor for the MRDx BCR-ABL Test after validating each MRDx BCR-ABL Test result against the run acceptance criteria and sample acceptance criteria.

Calibrators are analyzed for both BCR-ABL and ABL in every run simultaneously with patient samples. RNA controls provided at MMR (MR3.0) and MR4.5 allow for on-plate verification of test accuracy and reproducibility over this critical transcript range. The integrated conversion factor provides test results on the International Scale (IS) harmonized to the WHO.

# Interpretation of Results

Teresults are eporte in bot I ale and the corresonig molecular esn (MR) value. The BCR-ABL/ABL ratio percent IS result and MR value is calculated by the MRDx BCR-ABL Test Software using the following equations.

$$
{ \frac { B C R - A B L } { A B L } } \ \% \ S = \left( { \frac { M e a n \ B C R - A B L C o p y \ N u m b e r } { M e a n \ A B L C o p y \ N u m b e r } } \right) * 1 0 0 * C o n v e r s i o n \ F a c t o r ( C o m p o p ) .
$$

$$
M R x . x = l o g _ { 1 0 } \left( \frac { 1 0 0 } { \% J 5 } \right) = l o g _ { 1 0 } ( 1 0 0 ) - l o g _ { 1 0 } ( \% I S ) = 2 - l o g _ { 1 0 } ( \% I S )
$$

MR Values with the corresponding IS values are show in the table below:

<table><tr><td rowspan=1 colspan=1>MR</td><td rowspan=1 colspan=1>IS (%)</td></tr><tr><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>0.032</td></tr><tr><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.010</td></tr><tr><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.0032</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>0.0010</td></tr></table>

The assay reporting is shown in the table below. The assay will only report quantified results for BCR-ABL levels within the range of MR4.5 to MR1.0.

<table><tr><td rowspan=1 colspan=1>Collection Tube</td><td rowspan=1 colspan=1>BCR-ABL% (MR)</td><td rowspan=1 colspan=1>MRDx Result</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>&lt;1.0 (&gt;10% IS)</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>1.0-4.5</td><td rowspan=1 colspan=1>BCR-ABL detected</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>4.5-5.3</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>&gt;5.3 (&lt;0.00050% IS)</td><td rowspan=1 colspan=1>BCR-ABL not detected</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>&lt;1.0 (&gt;10% IS)</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>1.0-4.5</td><td rowspan=1 colspan=1>BCR-ABL detected</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>4.5-4.7</td><td rowspan=1 colspan=1>BCR-ABL detected, not quantifiable</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>&gt;4.7 (&lt;0.0020% IS)</td><td rowspan=1 colspan=1>BCR-ABL not detected</td></tr></table>

# B. Specimen Type

Two sample collection tube types and RNA extraction methods are specified in the package insert. The first, K2EDTA blood tubes (hereafter referred to simply as EDTA), are processed using the Maxwell CSC RNA Blood Kit (Promega Cat. No. AS1410) and semi-automated Maxwell CSC Instrument (Promega Cat. No. AS4000 or AS6000). The instructions for use for the MRDx BCRABL Test provide specific instructions for use of $5 \mathsf { m l }$ of blood versus the $2 . 5 ~ \mathsf { m L }$ specified in the Maxwell manual. The increased sample volume ensures the yield of RNA required is available for testing. A total of $4 \mu \ g$ of RNA is required for each patient sample. One $\mu \ g$ of extracted RNA is used in the MRDx BCR-ABL Test in each of 4 wells, 2 wells for BCR-ABL quantitation and 2 wells for ABL quantitation.

In addition to testing EDTA blood samples, the test may be used with blood collected into PAXgene Blood RNA Tubes (PreAnalytiX Cat. No. 762165) and a manual extraction procedure using the PAXgene Blood RNA Kit (QIAGEN Cat. No. 762164). The extraction procedure is performed per a MolecularMD-specified procedure utilizing four PAXgene Blood RNA Tubes to yield the required quantity of RNA. A total of $8 \mu \ g$ of RNA is required for each patient sample. Two $\mu \mathrm { g }$ of extracted RNA is used in the MRDx BCR-ABL Test in each of 4 wells, 2 wells for BCRABL quantitation and 2 wells for ABL quantitation.

# C. Kit Contents

The MolecularMD MRDx BCR-ABL Test includes the following components for testing RNA extracted from samples:

# Reagents:

• MRDx Master Mix (2X) (3 vials x 850 µL/vial)   
• MRDx BCR-ABL Primer/Probe, concentrated (1 vial x 40 µL/vial)   
C MRDx ABL Primer/Probe, concentrated (1 vial x 40 µL/vial)   
Nuclease-Free Water (2 vials x 1.8 mL/vial)

# IVT RNA Calibrators:

A set of calibrators containing in vitro transcribed BCR-ABL and ABL targets representing a dynamic range of ${ 1 0 ^ { 5 } }$ is provided as part of the test kit. Two calibration curves (one for BCR-ABL and one for ABL) are analyzed with every test run in order to allow for quantitation of the two transcripts in patient samples. The BCR-ABL and ABL levels for each calibrator are listed in the table below.

MRDx RNA Calibrators   

<table><tr><td rowspan=1 colspan=1>Label</td><td rowspan=1 colspan=1>BCR-ABL</td><td rowspan=1 colspan=1>ABL</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>600,000 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>300,000 copies/10 μL</td><td rowspan=1 colspan=1>300,000 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>30,000 copies/10 µL</td><td rowspan=1 colspan=1>30,000 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>3,000 copies/10 μL</td><td rowspan=1 colspan=1>3,000 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>300 copies/10 μL</td><td rowspan=1 colspan=1>300 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>30 copies/10 μL</td><td rowspan=1 colspan=1>30 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>10 copies/10 μL</td><td rowspan=1 colspan=1>10 copies/10 μL</td><td rowspan=1 colspan=1>80 μL/vial</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>3 copies/10 μL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>80 μL/vial</td></tr></table>

# RNA Controls:

MMR (MR3.0) and MR4.5 RNA controls are formulated from a mixture of BCR-ABL negative HL60 $( \mathsf { A T C C } ^ { \otimes } \mathsf { C C l } . 2 4 0 ^ { \mathsf { T M } } )$ and BCR-ABL e14a2 positive K-562 (ATCC® CCL-243T™M) RNA in a background mix of carrier RNA and stabilizers to allow for long-term stability. The MMR RNA Control is targeted to $0 . 1 0 \%$ IS BCR-ABL/ABL. The MR4.5 RNA Control is targeted to $0 . 0 0 3 2 \%$ IS BCR-ABL/ABL.

BCR-ABL RNA Control - MMR (10X, 1 vial x 14 µL/vial) − BCR-ABL RNA Control - MR4.5 (10X, 1 vial x 14 µL/vial)

# D. Hardware and Software

The MolecularMD MRDx BCR-ABL Test is performed on the Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument. The instrument integrates a thermal cycler, a fluorometer and application specific software. The instrument houses the thermal cycler and the fluorometer, while the application software is installed on a computer that is attached to the instrument. The ABI 7500 Fast Dx instrument software v1.4.1 calculates the BCR-ABL and ABL copy numbers using a standard curve generated with calibrators on each individual plate. The data are exported as a CSV file for further analysis in the MRDx BCR-ABL Test Software.

The MRDx BCR-ABL Test Software is used to analyze all test results. This software, provided with the MRDx BCR-ABL Test, is used to calculate the BCR-ABL/ABL $\%$ IS for patient sample results using the conversion factor for the MRDx BCR-ABL Test in addition to validating each MRDx BCR-ABL Test result against the run acceptance criteria and sample acceptance criteria.

# V. INDICATIONS FOR USE

The MolecularMD MRDx BCR-ABL Test is an in vitro diagnostic test for the quantitative detection of BCR-ABL1 transcripts (e13a2/b2a2 and/or e14a2/b3a2) and the ABL1 endogenous control mRNA in peripheral blood specimens from patients previously diagnosed with t(9;22) positive chronic myeloid leukemia (CML). The ratio of BCR-ABL1 to ABL1 is calculated and reported on the WHO International Scale. The test utilizes quantitative, real-time reverse transcription polymerase chain reaction performed on the Applied Biosystems 7500 Fast Dx instrument.

The MolecularMD MRDx BCR-ABL Test is intended to measure BCR-ABL mRNA transcript levels in patients diagnosed with t(9;22) positive CML during monitoring of treatment with Tyrosine Kinase Inhibitors (TKIs).

The device is also intended to be used in the serial monitoring for BCR-ABL mRNA transcript levels as an aid in identifying CML patients in the chronic phase being treated with nilotinib who may be candidates for treatment discontinuation and for monitoring of treatment-free remission.

The test does not differentiate between e13a2 or e14a2 fusion transcripts and does not monitor other rare fusion transcripts resulting from t(9;22). The test is not intended for the diagnosis of CML.

# VI. SUMMARY COMPARISON OF TECHNOLGICAL CHARACTERISTICS

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>TechnologicalCharacteristic</td><td rowspan=1 colspan=1>Candidate DeviceMRDx® BCR-ABL Test</td><td rowspan=1 colspan=1>Predicate DeviceQuantideX® qPCRBCR-ABL IS Kit</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>Applied Biosystems 7500 Fast Dx Real TimePCR Instrument</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Measurand</td><td rowspan=1 colspan=1>BCR-ABL1 fusion transcripts (e13a2/b2a2a n d / or e14a2/b3a2) and the ABL1endogenous control mRNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Measurement Type</td><td rowspan=1 colspan=1>Quantitative</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Principle of Assay</td><td rowspan=1 colspan=1>Reverse transcription, quantitative,polymerase chain reaction (qPCR) basednucleic acid amplification</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Traceability/Standard</td><td rowspan=1 colspan=1>1st WHO International Genetic ReferencePanel for quantitation of BCR-ABLtranslocation by RQ-PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Report Units</td><td rowspan=1 colspan=1>Both % IS and Molecular Response (MR)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test Limitation</td><td rowspan=1 colspan=1>The test does not differentiate betweene13a2 or e14a2 fusion transcripts anddoes not monitor other rare fusiontranscripts resulting from t(9;22). Thetest is not intended for the diagnosis ofCML.</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">TechnologicalCharacteristic</td><td colspan="1" rowspan="1">Candidate DeviceMRDx® BCR-ABL Test</td><td colspan="1" rowspan="1">Predicate DeviceQuantideX® qPCR BCR-ABL ISKit</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The MolecularMD MRDx BCR-ABL Test is anin vitro diagnostic test for the quantitativedetection of BCR-ABL1 transcripts(e13a2/b2a2 and/or e14a2/b3a2) and theABL1 endogenous control mRNA inperipheral blood specimens from patientspreviously diagnosed with t(9;22) positivechronic myeloid leukemia (CML). The ratioof BCR-ABL1 to ABL1 is calculated andreported on the WHO International Scale.</td><td colspan="1" rowspan="1">The QuantideX® qPCR BCR-ABL IS Kit is an in vitronucleic acid amplificationtest for the quantitation ofBCR-ABL1 and ABL1transcripts in total RNA fromwhole blood of diagnosedt(9;22) positive ChronicMyeloid Leukemia (CML)patients expressing BCR-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The test utilizes quantitative, real-timereverse transcription polymerase chainreaction performed on the AppliedBiosystems 7500 Fast Dx instrument.The MolecularMD MRDx BCR-ABL Test isintended to measure BCR-ABL mRNAtranscript levels in patients diagnosed witht(9;22) positive CML during monitoring oftreatment with Tyrosine Kinase Inhibitors(TKIs). The device is also intended to beused in the serial monitoring for BCR-ABLmRNA transcript levels as an aid inidentifying CML patients in the chronicphase being treated with nilotinib who maybe candidates for treatmentdiscontinuation and for monitoring oftreatment-free remission. The test does notdifferentiate between e13a2 or e14a2fusion transcripts and does not monitorother rare fusion transcripts resulting fromt(9;22). The test is not intended for thediagnosis of CML.</td><td colspan="1" rowspan="1">ABL1 fusion transcripts typee13a2 and/or e14a2. TheQuantideX qPCR BCR-ABL ISKit is a reverse transcription-quantitative PCR performedon the Applied Biosystems7500 Fast Dx Real-Time PCRInstrument and is intendedto measure BCR-ABL1 toABL1, expressed as a logmolecular reduction (MRvalue) from a baseline of100% on the InternationalScale, in t(9;22) positive CMLpatients during monitoringof treatment with TyrosineKinase Inhibitors (TKIs).</td></tr><tr><td colspan="1" rowspan="1">QualityControls</td><td colspan="1" rowspan="1">RNA MMR (MR3.0)RNA MR4.5Nuclease-Free Water - NTC</td><td colspan="1" rowspan="1">RNA High (MR 1.5)RNA Low (MR 3.5)ABL armored RNARNA Negative</td></tr><tr><td colspan="1" rowspan="1">Calibrators</td><td colspan="1" rowspan="1">8 vials total, representing 7 concentrationseach of BCR-ABL and ABL transcripts.</td><td colspan="1" rowspan="1">Four levels formulated toMR1.0, 2.0, 3.0, 4.0.</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">RNA from whole blood (PAXgene BloodRNA Tubes or EDTA)</td><td colspan="1" rowspan="1">RNA from whole blood(EDTA)</td></tr><tr><td colspan="1" rowspan="1">RNA InputAmount</td><td colspan="1" rowspan="1">2 µg PAXgene tubes and 1µg EDTA tubes</td><td colspan="1" rowspan="1">RNA input range of 1 to 5μg</td></tr></table>

# VII. PERFORMANCE CHARACTERISTICS

A. Pre-Clinical Analytical Studies

# 1. Traceability to the International Scale

The assay calibrators are traceable to the First $( 1 ^ { \mathfrak { s t } } )$ WHO International Genetic Reference Panel based on the method described in the $\boldsymbol { \mathit { 1 } } ^ { s t }$ WHO International Genetic Reference Panel for quantitation of BCR-ABL translocation by RQ-PCR, Instruction for Use. On 5 different days, a different WHO primary panel comprised of four levels (A-D) of freeze-dried materials was reconstituted and extracted following the assay protocol. The reconstituted/extracted WHO reference samples (200 ng RNA input/well) and the MolecularMD WHO BCR-ABL Reference Panel Secondary Standards ${ 1 \mu \ g }$ RNA input/well) were tested using the MRDx BCR-ABL Test over 20 days for each of 3 MRDx BCR-ABL Test kit lots.

Deming Regression analyses were performed comparing the expected and observed MR values for each lot. The conversion factor was calculated for each lot individually and was 1.1 for all three lots (A, B and C), showing the consistency of conversion factor across multiple lots. Data from all three lots showed high traceability to WHO reference standards supporting the conclusion that the assay calibrators to the WHO reference panel has been established. See the Deming Regression plots and statistics below.

# Deming Regression for WHO Standards by Lot

![](images/8c4f540c14dab3eaba75b46fa48928885a4ccf766cfd0e6b8c6fcd879795468e.jpg)

Regression Statistics for Each Lot   

<table><tr><td rowspan=1 colspan=1>Metrics</td><td rowspan=1 colspan=1>Lot A</td><td rowspan=1 colspan=1>Lot B</td><td rowspan=1 colspan=1>Lot C</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>95% Lower CI (slope)</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.98</td></tr><tr><td rowspan=1 colspan=1>95% Upper CI (slope)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Y-Intercept</td><td rowspan=1 colspan=1>-0.048</td><td rowspan=1 colspan=1>-0.034</td><td rowspan=1 colspan=1>-0.057</td><td rowspan=1 colspan=1>-0.046</td></tr><tr><td rowspan=1 colspan=1>95% Lower Cl(intercept)</td><td rowspan=1 colspan=1>-0.065</td><td rowspan=1 colspan=1>-0.057</td><td rowspan=1 colspan=1>-0.082</td><td rowspan=1 colspan=1>-0.059</td></tr><tr><td rowspan=1 colspan=1>95% Upper Cl(intercept)</td><td rowspan=1 colspan=1>-0.030</td><td rowspan=1 colspan=1>-0.010</td><td rowspan=1 colspan=1>-0.033</td><td rowspan=1 colspan=1>-0.034</td></tr><tr><td rowspan=1 colspan=1>Correlation</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.0</td></tr></table>

# 2. Correlation to a Reference Method (Accuracy)

The accuracy of detecting BCR-ABL transcript levels by the MRDx BCR-ABL Test was further evaluated by comparing to a second reference method, a validated BCR-ABL reverse transcription droplet digital PCR (ddPCR) assay. The study analyzed a total of 217 samples of blood from CML patients ranging from $10 \%$ BCR-ABL/ABL IS (MR1.0) to $0 . 0 0 1 \%$ BCR-ABL/ABL IS (MR5.0) drawn into PAXgene Blood RNA Tubes and extracted using the PAXgene Blood RNA Kit as well as samples drawn into EDTA blood collection tubes and extracted using the Maxwell CSC RNA Blood Purification Kit with the Maxwell CSC instrument. A total of 119 evaluable EDTA samples and 98 evaluable PAXgene samples were tested with both tests.

A Deming regression was applied and $9 5 \%$ confidence intervals of the slope and y-intercept were calculated. The predicted systematic difference between the two methods was calculated at the clinical decision points for treatment-free remission monitoring (MR3.0, MR4.0, and MR4.5) with $9 5 \%$ confidence intervals. High concordance between MRDx BCR-ABL Test results and the reference ddPCR test results were observed for both EDTA and PAXgene tube types (see below), supporting the detection accuracy of the device.

![](images/50063bc259181fbd6bdbd5508d1cf9f0e4410ae1b65afaaac651b073d9670577.jpg)  
Deming Regression Plots (MRDx vs. ddPCR)

Deming Regression Statistics (MRDx vs. ddPCR)   

<table><tr><td rowspan=1 colspan=1>Blood TubeType</td><td rowspan=1 colspan=1>SampleSize</td><td rowspan=1 colspan=1>Deming Slope</td><td rowspan=1 colspan=1>DemingY-Intercept</td><td rowspan=1 colspan=1>PearsonCoefficient</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0.9695% CI [0.94, 0.99]</td><td rowspan=1 colspan=1>0.07795% CI [0.0027, 0.17]</td><td rowspan=1 colspan=1>0.984</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>0.9895% CI [0.95, 1.0]</td><td rowspan=1 colspan=1>-0.02195% CI [-0.11, 0.056]</td><td rowspan=1 colspan=1>0.985</td></tr></table>

Predicted Difference at Clinical Decision Points   

<table><tr><td rowspan=1 colspan=1>Blood TubeType</td><td rowspan=1 colspan=1>Predicted Difference MR3(MMR)</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.0</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.5</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>-0.03195% CI, [-0.053,-0.0049]</td><td rowspan=1 colspan=1>-0.06795% CI [-0.10,-0.031]</td><td rowspan=1 colspan=1>-0.08595% CI [-0.13,-0.039]</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>-0.08895% CI [-0.11, -0.062]</td><td rowspan=1 colspan=1>-0.1195% CI [-0.15,-0.071]</td><td rowspan=1 colspan=1>-0.1295% CI [-0.17, -0.071]</td></tr></table>

# 3. Detection Capability

# Limit of Blank (LoB):

The LoB was determined by testing 30 BCR-ABL negative samples. Samples were tested in duplicate using 3 assay kit lots for a total of 180 replicates. Out of the 180 measurements, 177 had no measurable BCR-ABL values. Three $( 1 . 7 \% )$ had measurements well below the LoD of the test which were reported as undetected.

# Limit of Detection (LoD):

The LoD was determined using 6 CML patient specimens (3 specimens each for e13a2 and e14a2 transcripts). RNA of each sample was extracted with both PAXgene Blood RNA Tubes and EDTA blood collection tubes and serial diluted into RNA extracted from non-diseased subject blood to create five level panels. Sufficient RNA for each panel was made to allow for the testing of 60 total replicates (20 replicates x 3 lots) of each level in the panel. There were 1725 and 1595 valid EDTA and PAXgene samples tested respectively.

The LoD was calculated using the parametric analysis as described in CLSI EP17-A2. Using EDTA blood collection tubes, the LoD values were estimated to be $0 . 0 0 0 1 9 \%$ IS (MR5.7) for e13a2 and $0 . 0 0 0 2 9 \%$ IS (MR5.5) for e14a2. Using PAXgene Blood RNA Tubes, the LoD values were estimated to be $0 . 0 0 0 3 9 \%$ IS (MR5.4) for e13a2 and $0 . 0 0 0 2 9 \%$ IS (MR 5.5) for e14a2. The final LoD was determined to be $0 . 0 0 0 2 9 \%$ IS (MR5.5) for EDTA blood collection tubes and $0 . 0 0 0 3 9 \%$ IS (MR5.4) for PAXgene Blood RNA Tubes for both transcripts.

# Limit of Quantitation (LoQ):

The same LoD study samples were used to establish LoQ. The average and standard deviation for the BCR-ABL/ABL $\%$ IS values and MR values were calculated for the replicates for each level for all extraction methods, BCR-ABL transcripts, and all 3 lots. The log1o bias between the nominal values and the MRDx BCR-ABL Test for each level was calculated by subtracting the average measured BCR-ABL/ABL $\%$ IS log1o for each level from the BCR-ABL/ABL $\%$ IS log10 nominal value. The Total Error (TE) was calculated using the following equation: Total Error $= \left| \mathsf { B i a s } \right| + 2 \times \mathsf { S D }$

The results were evaluated against the acceptance criteria of Total Error $\leq 0 . 5 \log _ { 1 0 }$ The LoQ for EDTA blood collection tubes was $0 . 0 0 1 6 \%$ IS (MR4.8). The LoQ for PAXgene Blood RNA Tubes was $0 . 0 0 2 5 \%$ IS (MR4.6).

The MRDx BCR-ABL Test Software limits the LoQ and quantitated results to $0 . 0 0 3 2 \%$ IS (MR4.5) for both blood tube types.

# 4. Analytical Specificity - Interfering Substances

For endogenous blood substance interference testing, CML samples at 2 levels (MR3.0 and MR4.5) extracted from both EDTA and PAXgene collection tubes were spiked with all the following test substances to the concentrations noted:

Hemoglobin $2 \ : \mathrm { m g / m L }$ • Bilirubin $3 4 2 \mu \mathrm { m o l } .$ /L Triglycerides $2 8 8 ~ \mathrm { m g / d L }$

When compared to test results from samples spiked with matrix control (i.e., cleared serum and water), no clinically significant difference was observed from the samples spiked with the potential interference substances, although reduced RNA yield during extraction was observed in some samples. The largest difference between the endogenous substance spike and the matrix control spike for PAXgene Blood RNA Tubes was $- 0 . 0 8 0 \log _ { 1 0 }$ $9 5 \%$ Cl: -0.32 to 0.16) at MR4.5. The largest difference between the endogenous substance spiked group and the matrix control group for EDTA blood collection tubes was $0 . 0 6 0 \log _ { 1 0 }$ $9 5 \%$ CI: -0.042 to 0.16) at MR4.5.

For exogenous interference testing, extraction kit reagents used in the RNA extraction processes for PAXgene Blood RNA Tubes and EDTA blood collection tubes were examined to determine if any interference from the blood tubes and RNA extraction processes were observed with the MRDx BCR-ABL Test. Ten replicates of the MR3.0 and MR4.5 RNA controls were spiked and tested together with ten replicates of the MR3.0 and MR4.5 RNA Controls spiked with water as a matrix control sample. No clinically significant difference from the matrix control samples was observed. The largest difference between the exogenous substance spiked group and the matrix control group for PAXgene Blood RNA Tubes was $0 . 0 4 0 \log _ { 1 0 }$ $9 5 \%$ Cl: -0.010 to 0.090) at MR3.0. The largest difference between the exogenous substance spiked group and the matrix control group for EDTA blood collection tubes was $0 . 0 1 0 \log _ { 1 0 }$ $9 5 \%$ CI: -0.022 to 0.22) at MR4.5.

The data support the conclusion that none of the potential interferents evaluated had any significant interference with the assay.

# 5. Analytical Specificity - Primer Specificity

RNA from six CML patient samples (three with the e13a2 transcript and three with the e14a2 transcript) was extracted and used as template for the MRDx BCR-ABL Test. The resulting PCR products for both BCR-ABL and ABL were sub-cloned into a cloning vector and ten colonies were sequenced. The sequencing results were used to create a consensus sequence to compare to the established Genbank sequence for ABL and both BCR-ABL transcripts. The results matched the Genbank sequence with agreement across all bases demonstrating that the MRDx primers and probes amplified the intended targets for all samples tested.

Cross reactivity was further assessed by testing ABL2 in vitro transcribed RNA and RNA from an e19a2 positive cell line as template. No cross reactivity was observed with the ABL2 IVT RNA. As expected, cross reactivity was present for the e19a2 BCR-ABL transcript as it contains the binding sites for the primers and probe in BCR exon 13 and ABL exon 2. The cross reactivity of the MRDx BCR-ABL Test with the e19a2 BCR-ABL transcript was mitigated by a precaution statement in the labeling that the test is only indicated for use with CML patients diagnosed with the e13a2 and/or e14a2 BCR-ABL transcript and should not be used with patients confirmed to express atypical transcripts.

# 6. Analytical Specificity - Specimen Cross Contamination (Carryover)

The carryover contamination study was performed by testing high positive samples (near MR1.0) alternately juxtaposed with negative samples in both extraction rack setup as well as MRDx plate setup for both PAXgene Blood RNA Tubes and EDTA blood collection tubes.

The incidence of cross-contamination with PAXgene Blood RNA tubes of samples was determined using 30 replicates across 5 extraction racks and 5 MRDx BCR-ABL Test plates. The incidence of cross-contamination with EDTA blood collection tubes was determined using the Maxwell CSC RNA Blood Kit with 40 replicates across 5 extraction racks and 5 MRDx BCR-ABL Test plates.

No negative samples $( 0 \% 1 5 )$ for either extraction method had a BCR-ABL/ABL % IS value that was detectable above the LoD of the assay. The data support the conclusion that the test generates no significant carryover between wells on the plate.

# 7. Precision - Repeatability

Contrived patient samples targeting MR3.0, MR4.0, and MR4.5 for e13a2 and e14a2 BCR-ABL transcripts were created using CML patient RNA and non-diseased subject RNA extracted from both PAXgene Blood RNA tubes and EDTA blood collection tubes. The MR3.0, MR4.0, and MR4.5 RNA samples were each tested in 17 replicates on individual MRDx BCR-ABL Test plates. See the repeatability results in the table below.

For EDTA Blood RNA, all levels for both transcripts passed the acceptance criteria of $\leq 0 . 2 5$ SD $\mathsf { l o g } _ { 1 0 }$ . The highest standard deviation was observed for the e14a2 BCR-ABL transcript at MR4.5 $\left. { \mathsf { S D } } = 0 . 0 8 7 \right.$ .

For PAXgene Blood RNA, allevels for both transcripts passed the acceptance criteria $( \mathsf { S D } \le 0 . 2 5$ SD $\mathsf { l o g } _ { 1 0 } \mathsf { ) }$ . The highest standard deviation was observed for the e14a2 BCR-ABL transcript at MR4.5 $\mathsf { S D } { = } 0 . 1 0 )$ .

Repeatability Results   

<table><tr><td rowspan=2 colspan=1>TubeType</td><td rowspan=2 colspan=1>TranscriptLevel</td><td rowspan=2 colspan=1>SampleSize (N)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>BCR-ABL%IS</td><td rowspan=1 colspan=1>BCR-ABL%IS</td><td rowspan=1 colspan=1>BCR-ABL%IS</td><td rowspan=1 colspan=1>MR</td><td rowspan=1 colspan=1>MR</td><td rowspan=1 colspan=1>MR</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e13a2 MR3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.0087</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e13a2 MR4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.0026</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e13a2 MR4.5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.0062</td><td rowspan=1 colspan=1>0.0017</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>0.099</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2 MR3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.0092</td><td rowspan=1 colspan=1>8.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2 MR4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>0.0035</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.078</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2 MR4.5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.0051</td><td rowspan=1 colspan=1>0.0012</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e13a2 MR3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.0069</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e13a2 MR4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.011</td><td rowspan=1 colspan=1>0.0012</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e13a2 MR4.5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.0034</td><td rowspan=1 colspan=1>0.00055</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.079</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2 MR3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2 MR4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.013</td><td rowspan=1 colspan=1>0.0012</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2 MR4.5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.0038</td><td rowspan=1 colspan=1>0.00071</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.087</td><td rowspan=1 colspan=1>2</td></tr></table>

# 8. Precision - Reproducibility

Two series of samples were created with blood drawn into EDTA tubes by separately diluting pooled e13a2 and pooled e14a2 positive CML patient samples with high BCR-ABL/ABL ratios into blood from non-diseased subjects. Five levels were created with targeted BCR-ABL/ABL $\%$ IS ratios from $10 \%$ IS to $0 . 0 0 3 2 \%$ IS. Three testing sites with two $7 5 0 0 \mathrm { D } \mathsf { x }$ instruments and two operators each were used. Each day at each site, each operator ran two plates, one on each instrument. Each plate had one replicate of the e13a2 5 member panel and one replicate of the e14a2 five member panel. For each of three lots, five runs were performed. Fifteen measurements were made per instrument, per operator. A total of 60 replicates (15 measurements per instrument per site x 2 instruments $\times 2$ operatorg $\mathtt { \Omega } = 6 0$ ) were run at each site for each BCR-ABL transcript type. The total number of tests run across transcript types and panel members was 1800 (60 replicates x 3 sites x 2 transcripts x 5 panel members $= 1 8 0 0 )$ .

Reproducibility Results - EDTA   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanMRValue</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=3>Between-Operator</td><td rowspan=1 colspan=3>Between-Day(Run)</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=2>%CV</td><td rowspan=1 colspan=2>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2 L1</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.027</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0077</td><td rowspan=1 colspan=2>0.78</td><td rowspan=1 colspan=2>0.0046</td><td rowspan=1 colspan=1>0.46</td></tr><tr><td rowspan=1 colspan=1>e14a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=2>1.1</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e14a2 L3</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.015</td><td rowspan=1 colspan=2>0.50</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e14a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0.018</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1>e14a2 L5</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.023</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L1</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=2>1.5</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.024</td><td rowspan=1 colspan=2>1.1</td><td rowspan=1 colspan=2>0.0086</td><td rowspan=1 colspan=1>0.38</td></tr><tr><td rowspan=1 colspan=1>e13a2 L3</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>0.0070</td><td rowspan=1 colspan=2>0.23</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.019</td><td rowspan=1 colspan=2>0.48</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L5</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanMRValue</td><td rowspan=1 colspan=2>Between-Instruments</td><td rowspan=1 colspan=3>Between-Lot</td><td></td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=2>SD</td><td rowspan=1 colspan=2>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2 L1</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0.078</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=1 colspan=1>e14a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0.011</td><td rowspan=1 colspan=2>0.54</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e14a2 L3</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0.0065</td><td rowspan=1 colspan=2>0.21</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>e14a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0.0061</td><td rowspan=1 colspan=2>0.15</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>e14a2 L5</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L1</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.010</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=2>0.031</td><td rowspan=1 colspan=2>2.5</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>4.9</td></tr><tr><td rowspan=1 colspan=1>e13a2 L2</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.0023</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=2>0.033</td><td rowspan=1 colspan=2>1.5</td><td rowspan=1 colspan=1>0.064</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>e13a2 L3</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0.010</td><td rowspan=1 colspan=2>0.35</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>e13a2 L4</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.019</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=2>0.012</td><td rowspan=1 colspan=2>0.31</td><td rowspan=1 colspan=1>0.096</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 L5</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>2.9</td></tr></table>

Within-Laboratory Precision - EDTA   

<table><tr><td colspan="7" rowspan="1">Within-Laboratory Precision</td></tr><tr><td colspan="1" rowspan="1">BCR-ABL Transcript</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Mean MR</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">5.0</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">0.035</td><td colspan="1" rowspan="1">3.6</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">0.054</td><td colspan="1" rowspan="1">2.6</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">0.045</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">0.044</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">0.059</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">0.058</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">0.057</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">0.092</td><td colspan="1" rowspan="1">2.3</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">0.098</td><td colspan="1" rowspan="1">2.4</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">3.1</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">2.6</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">3.0</td></tr><tr><td colspan="1" rowspan="1">e14a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">2.9</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.054</td><td colspan="1" rowspan="1">4.3</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.057</td><td colspan="1" rowspan="1">4.7</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.054</td><td colspan="1" rowspan="1">4.4</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">0.066</td><td colspan="1" rowspan="1">2.9</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">0.057</td><td colspan="1" rowspan="1">2.5</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">0.053</td><td colspan="1" rowspan="1">2.3</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">0.056</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">0.055</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">0.056</td><td colspan="1" rowspan="1">1.9</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">0.089</td><td colspan="1" rowspan="1">2.3</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">0.092</td><td colspan="1" rowspan="1">2.4</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">0.097</td><td colspan="1" rowspan="1">2.5</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">2.5</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">3.0</td></tr><tr><td colspan="1" rowspan="1">e13a2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">2.9</td></tr></table>

For PAXgene Blood RNA tubes, two series of samples were created with blood drawn into PAXgene RNA Blood tubes by separately diluting pooled e13a2 and pooled e14a2 positive CML patient blood samples with high BCR-ABL/ABL ratios into blood from non-diseased subjects. Five levels were created with targeted BCR-ABL/ABL $\%$ IS ratios ranging from $10 \%$ IS to $0 . 0 0 3 2 \%$ IS. One testing site used three operators and each operator used one 7500 Fast DX instrument exclusively. Two sample panels were extracted with three lots of extraction reagents and evaluated using three lots of MRDx BCR-ABL Test reagents over five non-consecutive days for a total of 15 replicates. A total of 90 measurements were made for each panel member for each BCR-ABL transcript type and RNA input amount. A total of 900 tests were completed for variables including operator, instrument, day, and lot. The results for the reproducibility study are provided in the following table.

Reproducibility Results - PAXgene   

<table><tr><td rowspan=2 colspan=1>Level</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>MeanMRValue</td><td rowspan=1 colspan=2>Between Operator(Instrument)</td><td rowspan=1 colspan=2>Between Day(Run)</td><td rowspan=1 colspan=2>Between Lot</td><td rowspan=1 colspan=2>Total (Within-Laboratory)</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>e14a2 L1</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>0.0074</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>0.034</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.095</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>e14a2 L2</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.038</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.067</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>e14a2 L3</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.063</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.090</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=1 colspan=1>e14a2 L4</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.082</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.058</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>e14a2 L5</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0.078</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.069</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>e13a2 L1</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.032</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>e13a2 L2</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.014</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>5.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 L3</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 L4</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>e13a2 L5</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.070</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>4.3</td></tr></table>

# 9. Linear Range - Assay Range

Linearity was estimated by testing two dilution series (one for e13a2 and one for e14a2 transcript) for each blood collection tube type (EDTA and PAXgene). Each dilution series had 10 BCR-ABL target levels and 5 replicates at each level to evaluate the assay linearity across the target range from $1 5 \%$ IS to $0 . 0 0 1 0 \%$ IS (MR0.8 to MR5.0).

The linear regression analyses were performed for first, second and third order polynomials per CLSI EP6-A. The data were considered linear if the non-linear coefficients from the second and third order polynomials were insignificant $\left( \mathsf { p } > 0 . 0 5 \right)$ . When significant non-linear coefficients were observed, the Degree of Nonlinearity was calculated per CLSI EP6-A. The linear regression curves for both transcripts in the two collection tube types are shown in below followed by the estimated regression intercepts and slopes from the linear model.

![](images/baaf2447fed61a04c5af4e94da7a04b00f8228da0b9ff46178a4a9b887bf183f.jpg)

![](images/05d32a3c21520fffe3d31d07b7212ad839eb5afb6078f335bf08a25671781373.jpg)

Regression Coefficients from Linear Model   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Transcript</td><td rowspan=1 colspan=2>Intercept</td><td rowspan=1 colspan=2>Slope</td></tr><tr><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Estimate</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=2 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>-0.045</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.017</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.026</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>0.008</td></tr><tr><td rowspan=2 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>-0.016</td><td rowspan=1 colspan=1>0.037</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.011</td></tr><tr><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.059</td><td rowspan=1 colspan=1>0.046</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>0.015</td></tr></table>

Transcript e13a2 was linear from MR0.93 to MR5.1 with a maximum SD of 0.22 using EDTA tubes and was linear from MR0.78 to MR4.8 with a maximum SD of 0.13 using PAXgene tubes. Transcript e14a2 was linear from MR0.99 to MR5.0 with a maximum SD of 0.26 using EDTA tubes and was linear from MR0.93 to MR4.9 with a maximum SD of 0.31 using PAXgene tubes. The data support the assay's linearity across the indicated detection range.

Linear Range   

<table><tr><td rowspan=1 colspan=1>TubeType</td><td rowspan=1 colspan=1>BCR-ABLTranscript</td><td rowspan=1 colspan=1>NominalLower LimitBCR-ABL/ABL% IS</td><td rowspan=1 colspan=1>NominalLower LimitMolecularResponse</td><td rowspan=1 colspan=1>NominalUpper LimitBCR-ABL/ABL% IS</td><td rowspan=1 colspan=1>Nominal UpperLimitMolecularResponse</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.00080</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.93</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>0.0010</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.99</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e13a2</td><td rowspan=1 colspan=1>0.0017</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.78</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>e14a2</td><td rowspan=1 colspan=1>0.0012</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.93</td></tr></table>

The linear range was truncated by the LoQ to determine the assay range. The data support the conclusion that the assay is linear across the reportable range of MR1.0 to MR4.5. The assay range for the MRDx BCR-ABL Test for each blood collection tube type are:

Assay Range   

<table><tr><td rowspan=1 colspan=1>Tube Type</td><td rowspan=1 colspan=1>Linear RangeBCR-ABL/ABL % IS</td><td rowspan=1 colspan=1>Linear RangeMolecular Response</td></tr><tr><td rowspan=1 colspan=1>EDTA</td><td rowspan=1 colspan=1>10% to 0.0032%</td><td rowspan=1 colspan=1>MR1.0 to MR4.5</td></tr><tr><td rowspan=1 colspan=1>PAXgene</td><td rowspan=1 colspan=1>10% to 0.0032%</td><td rowspan=1 colspan=1>MR1.0 to MR4.5</td></tr></table>

# 10. Stability Studies

Kit Real-Time and Freeze-Thaw Stability:

The real-time stability of the MRDx BCR-ABL Test was evaluated at two storage temperatures (-30 to $- 1 5 ^ { \circ } \mathsf { C }$ and $\mathtt { - 8 0 }$ to $- 6 5 ^ { \circ } \mathsf { C } )$ at periodic intervals with three lots. Six kits from each lot for each time point were tested using the final product release testing procedure. After use, the kits were frozen, thawed, and refrozen without testing to achieve an additional freeze/thaw cycle (Cycle 2). These six kits were then open-vial tested at one month (Test 2 and Cycle 3), and two months (Test 3 and Cycle 4) post-reconstitution using the WHO BCR-ABL Reference Panel Secondary Standards to ensure the kits were stable for three kit uses with three freeze/thaw cycles with 2 month stability. Kits were stored at -30 to $\scriptscriptstyle - 1 5 ^ { \circ } \mathsf C$ after initial use for open-vial stability, regardless the original storage condition.

![](images/6c29c6cf1443e86ca3121e11817e784e15430331e6edc0b4a7caf60e5f75a1b2.jpg)

For each time point, the MRDx kit was tested for performance of the calibration curve criteria, the control results, and the WHO secondary standards. Passing test results data exist for at least 16 months for 3 lots of MRDx BCR-ABL Test stored at the two separate temperatures.

The data support a shelf life of 15 months at both storage temperatures (-30 to $- 1 5 ^ { \circ } \mathsf { C }$ and -80 to $- 6 5 ^ { \circ } \mathsf { C } )$ and an open vial stability of 3 freeze-thaw cycles and up to 2 months following initial use.

Reagent Preparation and Reaction Intermediate Stability (In Use):

This study demonstrated the MRDx BCR-ABL Test reagent stability during PCR plate preparation and the post-plate-assembly PCR reaction stability at 2 to ${ 8 ^ { \circ } C }$ before evaluation on the ABI 7500 Fast Dx instrument.

Both standard and extreme conditions were tested. The standard condition used a newly opened kit immediately after thawing to setup a PCR plate. The PCR plate was processed immediately after setup on the ABl 7500 Fast $\mathsf { D } \times$ . The extreme condition used a kit that was thawed and refrozen twice to simulate the third use of the kit. After the reagents were thawed, components were stored at 2 to $8 ^ { \circ } C$ for eight hours, and then the PCR plate was assembled and placed at 2 to ${ 8 ^ { \circ } C }$ for 1.5 hours before evaluation on the ABI 7500 Fast Dx instrument. Two kits were run for each condition on each of 2 days with 2 sets of calibration curves, RNA controls, and samples (total $\scriptstyle 1 = 8$ per condition).

The acceptance criteria were that the mean of the test samples was $\pm 0 . 5$ MR of the control. The acceptance criteria were met for the standard and the extreme conditions. The differences between the standard condition and the extreme condition were less than $0 . 2 5 \log _ { 1 0 }$ for all levels of samples tested. The largest difference observed between conditions was -0.063 $( 9 5 \%$ Cl: -0.24 to 0.12) for the MR4.5 sample. These results support the stability conclusion that after 2 freeze-thaw cycles, the reagent is stable at 2 to ${ 8 ^ { \circ } C }$ for up to eight hours before PCR reagent assembly and up to 1.5 hours after PCR plate assembly.

# Specimen Stability:

For PAXgene Blood RNA Tubes, samples at three levels (MR3.0, MR4.0, and MR4.5) were created, stored at 2 to $8 \%$ and tested in quadruplicate at 0, 24, 48, and 60 hours. An additional series of samples was created, stored at -30 to $- 1 5 ^ { \circ } \mathsf C$ , and tested in quadruplicate at 0, 14, 28, and 30 days. For EDTA blood collection tubes, 17 CML patient specimens covering the clinical decision range were collected and stored at 2 to $8 \%$ and tested in quadruplicate at 0, 24, 48, and 60 hours.

All tested samples passed the acceptance criteria of $< 0 . 2 5 \log _ { 1 0 }$ difference compared to the mean BCR-ABL/ABL IS log1o ratio results at ${ \sf T } _ { 0 }$ These results support the following stability conclusion:

Specimens collected in PAXgene Blood RNA Tubes are stable for at least 48 hours at 2 to $8 ^ { \circ } C$ and at least 28 days at -30 to $- 1 5 ^ { \circ } \mathsf { C }$ Specimens collected in EDTA blood tubes are stable for at least 48 hours at 2 to ${ } ^ { 8 ^ { \circ } } \mathsf { C }$

# Shipping Stability:

In this study, the MRDx BCR-ABL Test kits were evaluated with multiple types of containers and shipping conditions. The mock shipping conditions included both stress temperatures and extended duration. The shipping conditions evaluated the minimum and maximum payload for both types of shipping boxes. Maximum load was the worst case regarding the amount of dry ice per shipping condition. The testing conditions were executed sequentially as follows:

$2 2 ^ { \circ } \mathsf { C } \pm 3 ^ { \circ } \mathsf { C }$ for 4 hours • $3 5 ^ { \circ } C \pm 3 ^ { \circ } C$ for 6 hours • $3 0 ^ { \circ } \mathsf { C } \pm 3 ^ { \circ } \mathsf { C }$ for 56 hours $3 5 ^ { \circ } C \pm 3 ^ { \circ } C$ for 6 hours

The results of this study demonstrated that the Polar Tech TC229 (P/N 049-0128) and BioFreeze 22L-133Hr (P/N 049-0129) shipping containers can maintain an internal temperature below $- 6 5 ^ { \circ } \mathsf { C }$ for all load conditions during a 72 hour temperature stress test that simulated typical shipping conditions. During this shipping test, the MRDx BCR-ABL Test kits remained at storage conditions of -80 to $\scriptscriptstyle - 6 5 ^ { \circ } C$

Functional testing using samples across the assay range also showed that the kit performance after shipping stress met equivalency acceptance criteria when compared to unstressed control kits. These results support the conclusion that the assay kit is stable following shipment of up to 72 hours on dry ice.

# 11. PAXgene-EDTA Tube Equivalency

A blood matrix equivalency study using 100 CML patient paired blood samples was conducted to demonstrate that MRDx BCR-ABL Test results from RNA samples extracted from PAXgene Blood RNA Tubes are the same as from EDTA blood collection tubes. At least 4 PAXgene Blood RNA Tubes and two 6 mL EDTA blood collection tubes were collected for each subject. RNA was extracted from the PAXgene Blood RNA Tubes using the PAXgene Blood RNA Kit. RNA from the EDTA blood collection tubes was extracted using the Maxwell CSC Blood RNA Kit.

For all paired samples that were above the LoQ of the test, a Deming regression was applied and $9 5 \%$ confidence intervals of the slope and y-intercept were calculated. The predicted systematic difference between the two methods was calculated at the clinical decision points (MR3.0, MR4.0, and MR4.5) with $9 5 \%$ confidence intervals. The data demonstrated that no clinically significant bias at the clinical decision points was observed in the MR values from MRDx BCR-ABL Test results between the two blood collection tube types. Results are shown below.

Deming Regression (EDTA vs. PAXgene)   

<table><tr><td rowspan=1 colspan=1>SampleSize</td><td rowspan=1 colspan=1>Deming Slope</td><td rowspan=1 colspan=1>DemingY-Intercept</td><td rowspan=1 colspan=1>DemingPearson Coefficient</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.0295% CI [1.00, 1.05]</td><td rowspan=1 colspan=1>-0.11595% CI [-0.202, -0.042]</td><td rowspan=1 colspan=1>0.992</td></tr></table>

Predicted Difference at Clinical Decision Points   

<table><tr><td rowspan=1 colspan=1>Predicted Difference(MMR/MR3)</td><td rowspan=1 colspan=1>Predicted DifferenceMR4</td><td rowspan=1 colspan=1>Predicted DifferenceMR4.5</td></tr><tr><td rowspan=1 colspan=1>-0.043395% CI [-0.0687, -0.0175]</td><td rowspan=1 colspan=1>-0.019295% CI [-0.0564, 0.0238]</td><td rowspan=1 colspan=1>-0.0072195% CI [-0.0549, 0.0462]</td></tr></table>

# B. Clinical Studies

The clinical validity of the MRDx BCR-ABL Test was demonstrated in two clinical trials, ENESTfreedom (CAMN10712201) and ENESTop (CAMN107A2408), for efficacy of identifying CMLCP patients on nilotinib treatment, who may be eligible to enter and maintain treatment free remission (TFR), and monitoring for potential loss of remission after treatment discontinuation. The study summary is described below, please refer to Tasigna® drug labeling at Drugs@FDA for more details.

# Treatment discontinuation in newly diagnosed $\mathsf { P h } +$ CML-CP patients who have achieved a sustained molecular response (MR4.5)

The ENESTfreedom study is an open-label, multicenter, single-arm study, where 215 adult patients with $\mathsf { P h } + \mathsf { C M L - C P }$ treated with nilotinib in first-line for $\geq 2$ years who achieved MR4.5 as measured with the MolecularMD MRDx BCR-ABL Test were enrolled to continue nilotinib treatment for an additional 52 weeks (nilotinib consolidation phase).

Of the 215 patients, 190 patients $( 8 8 . 4 \% )$ entered the "Treatment-Free Remission" (TFR) phase after achieving a sustained molecular response (MR4.5) during the consolidation phase, defined by the following criteria:

The 4 last quarterly assessments by MRDx BCR-ABL Test (taken every 12 weeks) were at least MR4.0 (BCR-ABL/ABL ≤ 0.01% IS), and maintained for 1 year   
• The last assessment being MR4.5 (BCR-ABL/ABL ≤ 0.0032% IS) No more than two assessments falling between MR4.0 and MR4.5 (0.0032% IS < BCR$\mathsf { A B L } / \mathsf { A B L } \leq 0 . 0 1 \% 1 \mathsf { S } )$

The median age of patients who entered the TFR phase was 55 years, $4 9 . 5 \%$ were females, and $2 1 . 1 \%$ of the patients were $\ge 6 5$ years of age. BCR-ABL levels were monitored every 4 weeks during the first 48 weeks of the TFR phase. Monitoring frequency was intensified to every 2 weeks upon the loss of MR4.0. Biweekly monitoring ended at one of the following time points:

Loss of MMR requiring patient to re-initiate nilotinib treatment   
• When the BCR-ABL levels returned to a range between MR4.0 and MR4.5   
When the BCR-ABL levels remained lower than MMR for 4 consecutive measurements (8 weeks from initial loss of MR4.0).

Any patient with loss of MMR during the TFR phase re-initiated nilotinib treatment at $3 0 0 ~ \mathsf { m g }$ twice daily or at a reduced dose level of $4 0 0 ~ \mathsf { m g }$ once daily if required from the perspective of tolerance, within 5 weeks after the collection date of the blood sample demonstrating loss of MMR. Patients who required re-initiation of nilotinib treatment were monitored in the nilotinib treatment reinitiation (NTRIl) phase for BCR-ABL levels every 4 weeks for the first 24 weeks and then every 12 weeks thereafter in patients who regained MMR.

Efficacy was based on the 96-week analysis data cut-off date, by which time, 91 patients $( 4 7 . 9 \% )$ discontinued from the TFR phase due to loss of MMR, and 1 (0.5%), 1 (0.5%), 1 (0.5%)

510(k) Summary

and 3 patients $( 1 . 6 \% )$ due to death from unknown cause, physician decision, lost to follow-up, and subject decision, respectively. Among the 91 patients who discontinued the TFR phase due to loss of MMR, 88 patients restarted nilotinib treatment and 3 patients permanently discontinued from the study. The efficacy results for ENESTfreedom study are summarized in the table below.

Efficacy Results for ENESTfreedom Study   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>Patients who entered TFR phase (Full Analysis Set, N=190)</td></tr><tr><td rowspan=1 colspan=2>Patients in TFR phase</td><td rowspan=1 colspan=1>Discontinuations dueto loss of MMR</td><td rowspan=1 colspan=1>Patients in NTRIphase</td></tr><tr><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td></tr><tr><td rowspan=1 colspan=1>24 weeks</td><td rowspan=1 colspan=1>118 (62.1%)</td><td rowspan=1 colspan=1>(54.8, 69.0)</td><td rowspan=1 colspan=1>70 (36.8%)</td><td rowspan=1 colspan=1>69 (36.3%)</td></tr><tr><td rowspan=1 colspan=1>48 weeks</td><td rowspan=1 colspan=1>98 (51.6%)</td><td rowspan=1 colspan=1>(44.2, 58.9)</td><td rowspan=1 colspan=1>88 (46.3%)</td><td rowspan=1 colspan=1>86 (45.3%)</td></tr><tr><td rowspan=1 colspan=1>96 weeks</td><td rowspan=1 colspan=1>93 (48.9%)</td><td rowspan=1 colspan=1>(41.6, 56.3)</td><td rowspan=1 colspan=1>91 (47.9%)</td><td rowspan=1 colspan=1>88 (46.3%)</td></tr></table>

Of the 88 patients who restarted treatment due to loss of MMR in the TFR phase, 87 patients $( 9 8 . 9 \% )$ patients regained MMR (one patient discontinued study permanently due to subject decision after 7.1 weeks of retreatment without regaining MMR) and 81 patients $( 9 2 . 0 \% )$ regained MR4.5 by the time of the cut-off date. The cumulative rate of MMR and MR4.5 regained at 24 weeks since treatment reinitiation was $9 7 . 7 \%$ (86/88 patients) and $8 6 . 4 \%$ (76/88 patients), respectively.

Among the 190 patients in the TFR phase, 98 patients had a treatment-free survival (TFS) event (defined as discontinuation from TFR phase due to any reason, loss of MMR, death due to any cause, progression to AP/BC up to the end of TFR phase, or re-initiation of treatment due to any cause in the study) by the 96-week cut-off date.

# 2.Treatment discontinuation in Ph+ CML-CP patients who have achieved a sustained molecular response (MR4.5) onnilotinib following prior imatinib therapy

Study ENESTop (NCT01698905) is an open-label, multicenter, single-arm study, where 163 adult patients with $\mathsf { P h } + \mathsf { C M L - C P }$ taking tyrosine kinase inhibitors (TKls) for $\geq 3$ years (imatinib as initial TKI therapy for more than 4 weeks without documented MR4.5 on imatinib at the time of switch to nilotinib, then switched to nilotinib for at least 2 years), and who achieved MR4.5 on nilotinib treatment as measured with the MolecularMD MRDx BCR-ABL Test were enrolled to continue nilotinib treatment for an additional 52 weeks (nilotinib consolidation phase). Of the 163 patients, 126 patients $( 7 7 . 3 \% )$ entered the TFR phase after achieving a sustained molecular response (MR4.5) during the consolidation phase, defined by the following criterion:

The 4 last quarterly assessments by the MRDx BCR-ABL Test (taken every 12 weeks) showed no confirmed loss of MR4.5 (BCR-ABL/ABL ≤ 0.0032% IS) during one year.

The median age of patients who entered the TFR phase was 56 years, $5 5 . 6 \%$ were females, and $2 7 . 8 \%$ of the patients were $\ge 6 5$ years of age. The median actual dose intensity during the 52- week nilotinib consolidation phase was 771.8 mg/day with $5 2 . 4 \%$ , $2 9 . 4 \%$ , $0 . 8 \%$ , $1 6 . 7 \%$ and $0 . 8 \%$ of patients receiving a daily nilotinib dose of $8 0 0 ~ \mathsf { m g } ,$ $6 0 0 ~ \mathsf { m g } ,$ , 450mg, 400mg and 300mg just before entry into the TFR phase, respectively.

Patients who entered the TFR phase but experienced two consecutive measurements of BCR$\mathsf { A B L } / \mathsf { A B L } > 0 . 0 1 \% | \mathsf { \Omega } | \mathsf { \Omega }$ S were considered having a confirmed loss of MR4.0, triggering re-initiation of nilotinib treatment. Patients with loss of MMR in the TFR phase immediately restarted nilotinib treatment without confirmation. All patients who restarted nilotinib therapy had BCR-ABL transcript levels monitored every 4 weeks for the first 24 weeks, then once every 12 weeks.

Efficacy was based on the 96-week analysis data cut-off date, by which time, 61 patients $( 4 8 . 4 \% )$ had discontinued from the TFR phase: 58 patients $( 4 6 . 0 \% )$ due to loss of MMR or confirmed loss of MR4.0, 2 patients $( 1 . 6 \% )$ due to subject/guardian decision and one patient $( 0 . 8 \% )$ due to pregnancy. Among the 58 patients who discontinued from the TFR phase due to confirmed loss of MR4.0 or loss of MMR, 56 patients restarted nilotinib therapy and 2 patients permanently discontinued from the study. The efficacy results are summarized in the table below.

Efficacy Results for ENESTop Study   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>Patients who entered TFR phase (Full Analysis Set, N=126)</td></tr><tr><td rowspan=1 colspan=2>Patients in TFR phase</td><td rowspan=1 colspan=1>Discontinuations due toloss of MMR or confirmedloss of MR4</td><td rowspan=1 colspan=1>Patients inNTRI phase</td></tr><tr><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>n (%)</td><td rowspan=1 colspan=1>n (%)</td></tr><tr><td rowspan=1 colspan=1>24 weeks</td><td rowspan=1 colspan=1>76 (60.3%)</td><td rowspan=1 colspan=1>(51.2, 68.9)</td><td rowspan=1 colspan=1>49 (38.9%)</td><td rowspan=1 colspan=1>48 (38.1%)</td></tr><tr><td rowspan=1 colspan=1>48 weeks</td><td rowspan=1 colspan=1>73 (57.9%)</td><td rowspan=1 colspan=1>(48.8, 66.7)</td><td rowspan=1 colspan=1>53 (42.1%)</td><td rowspan=1 colspan=1>51 (40.5%)</td></tr><tr><td rowspan=1 colspan=1>96 weeks</td><td rowspan=1 colspan=1>67 (53.2%)</td><td rowspan=1 colspan=1>(44.1, 62.1)</td><td rowspan=1 colspan=1>58 (46.0%)</td><td rowspan=1 colspan=1>56 (44.4%)</td></tr></table>

Of the 56 patients who restarted nilotinib treatment due to confirmed loss of MR4.0 or loss of MMR in the TFR phase, 52 patients $( 9 2 . 9 \% )$ regained MR4.0 and MR4.5 and 4 patients $( 7 . 1 \% )$ did not regain MR4.0 by the time of the cut-off date. The cumulative rate of MR4.0 and MR4.5 regained by 48 weeks since treatment reinitiation, was $9 2 . 9 \%$ (52/56 patients) and $9 1 . 1 \%$ (51/56 patients), respectively.

Among the 126 patients in the TFR phase, 61 patients $( 4 8 . 4 \% )$ had a treatment-free survival (TFS) event (defined as discontinuation from TFR phase due to any reason, loss of MMR, confirmed loss of MR4.0, death due to any cause, progression to AP/BC up to the end of TFR phase, or reinitiation of treatment due to any cause in the study) on or before the 96-month cut-off date.

# C. Conclusions Drawn from Preclinical and Clinical Studies

The effectiveness and clinical benefit of the ${ \mathsf { M R D x } } ^ { \otimes }$ BCR-ABL Test was demonstrated in two prospective clinical trials utilizing the monitoring of BCR-ABL mRNA transcript levels in patients diagnosed with CML as an aid in identifying patients being treated with nilotinib as candidates for initiating treatment-free remission. Further, the monitoring of patients in treatment-free remission successfully met the intended use of identifying patients requiring reinitiation of nilotinib treatment. Analytical performance studies provide assurance of the accuracy and reliability of the quantitation of BCR-ABL transcript levels.

The data from these studies support the reasonable assurance of safety and effectiveness of the ${ \mathsf { M R D x } } ^ { \otimes }$ BCR-ABL Test when used in accordance with the indications for use.