NantHealth, Inc.   
Aleece Nolasco   
Vice President, Regulatory Affairs 9920 Jefferson Blvd   
Culver City, CA 90232

Re: K190661 Trade/Device Name: Omics Core Regulation Number: 21 CFR 866.6080 Regulation Name: Next generation sequencing based tumor profiling test Regulatory Class: Class II Product Code: PZM Dated: September 26, 2019 Received: September 27, 2019

Dear Aleece Nolasco:

This letter corrects our substantially equivalent letter of November 9, 2019.

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's

requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Donna M. Roscoe -S

Donna Roscoe, Ph.D.   
Chief, Molecular Genetics Branch   
Division of Molecular Genetics and Pathology OHT7: Office of In Vitro Diagnostics   
and Radiological Health   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

<table><tr><td></td><td>DEPARTMENT OF HEALTH AND HUMAN SERVICES Food and Drug Administration Indications for Use</td><td>Form Approved: OMB No. 0910-0120 Expiration Date: 06/30/2020 See PRA Statement below.</td></tr><tr><td>510(k) Number (if known) K190661</td><td></td><td></td></tr><tr><td>Device Name</td><td></td><td></td></tr><tr><td>Omics Core</td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr></table>

Indications for Use (Describe)

The Omics Core assay is a qualitative in vitro diagnostic test that uses targeted next generation sequencing of formalin-fixed paraffinembedded tumor tissue matched with normal specimens from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multi gene panel. The test is intended to provide information on somatic mutations (point mutations and small insertions and deletions) and tumor mutational burden (TMB) for use by qualified health care professionals in accordance with professional guidelines, and is not conclusive or prescriptive for labeled use of any specific therapeutic product. Omics Core is a single-site assay performed at NantHealth, Inc.

# CONTINUE ON A SEPARATE PAGE IF NEEDED.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# TABLE OF CONTENTS

510(K) SUMMARY 4

1. SUBMITTER .   
2. DEVICE .   
3. PREDICATE DEVICE.   
4. DEVICE DESCRIPTION 5   
4.1. Test Principle ..   
4.2. Equipment, Supplies and Reagents ..   
4.3. Omics Core Workflow ..   
4.4. Indications For Use . .8   
5. COMPARISON OF TECHNOLOGICAL CHARACTERISTICS   
WITH THE PREDICATE DEVICE .8   
6. PERFORMANCE DATA 11   
7. SUMMARY . 14

# LIST OF TABLES

Table 1: Equipment Supplies, Reagents, and Software ..   
Table 2: Comparison of Omics Core with Predicate Device ..   
Table 3: Omics Core Performance Specifications ...

# LIST OF FIGURES

Figure 1: Laboratory Workflow .   
Figure 2: Bioinformatics Workflow .

# 510(K) SUMMARY

This summary of $5 1 0 ( \mathrm { k } )$ safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

# 1. SUBMITTER

NantHealth, Inc. 9920 Jefferson Blvd. Culver City, CA 90232

Contact Person:   
Aleece Nolasco   
Vice President, Regulatory Affairs   
Phone: 310-405-7549   
Email: Aleece.Nolasco@NantHealth.com   
Date prepared: October 21, 2019

# 2. DEVICE

<table><tr><td rowspan=1 colspan=1>Name of Device</td><td rowspan=1 colspan=1>Omics Core</td></tr><tr><td rowspan=1 colspan=1>Common Name</td><td rowspan=1 colspan=1>Next Generation Sequencing Tumor Profiling Test</td></tr><tr><td rowspan=1 colspan=1>Classification Name</td><td rowspan=1 colspan=1>Next Generation Sequencing Based Tumor Profiling Test(21CFR 866.6080)</td></tr><tr><td rowspan=1 colspan=1>Regulatory Class</td><td rowspan=1 colspan=1>Ⅱ</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>PZM</td></tr></table>

# 3. PREDICATE DEVICE

MSK-IMPACT (Integrated Mutation Profiling Of Actionable Cancer Targets): A Hybridization-Capture Based Next Generation Sequencing Assay, DEN170058

This predicate device has not been subject to a design-related recall.

No reference devices were used in this submission.

# 4. DEVICE DESCRIPTION

# 4.1. Test Principle

The NantHealth Omics Core assay is a custom targeted whole exome sequencing platform, utilizing solution-phase exon capture and sequencing, to report somatic alterations (point mutations, small insertions and deletions) in 468 genes and sequencing of 19,396 protein-coding genes (whole exome) to determine overall tumor mutation burden in tumor specimens.

Genomic DNA is extracted from both a tumor and a patient-matched normal control sample. Sequence libraries are prepared through a series of enzymatic steps including shearing of double-stranded DNA, end repair, A-base addition, ligation of barcoded sequence adaptors, and low cycle PCR amplification. Single barcoded sequence libraries are captured using the biotinylated probes. Captured DNA fragments are then pooled and sequenced on an Illumina NovaSeq 6000 as paired-end reads. Sequence reads are then aligned to the reference human genome. Somatic alterations are identified in the tumor DNA by direct comparison to the matched normal DNA.

# 4.2. Equipment, Supplies and Reagents

Table 1 summarizes some of the main equipment, supplies, reagents, and software used by the Omics Core assay.

All reagents, materials and equipment needed to perform the assay are qualified by NantHealth.

Table 1: Equipment Supplies, Reagents, and Software   

<table><tr><td rowspan=1 colspan=1>Library Prep</td><td rowspan=1 colspan=1>Kapa Hyper Prep Kit</td></tr><tr><td rowspan=1 colspan=1>Hybridization</td><td rowspan=1 colspan=1>IDT xGen® Exome Research Panel v1.0</td></tr><tr><td rowspan=1 colspan=1>Sequencer Chemistry</td><td rowspan=1 colspan=1>2-Channel SBS Chemistry</td></tr><tr><td rowspan=1 colspan=1>Sequencer</td><td rowspan=1 colspan=1>Illumina NovaSeq 6000</td></tr><tr><td rowspan=1 colspan=1>Bioinformatics</td><td rowspan=1 colspan=1>Omics Core software</td></tr></table>

# 4.3. Omics Core Workflow

The following figures provide an illustrated overview of the flow of the Omics Core testing process from sample receipt, through testing, to the report. Detailed descriptions of each aspect of the assay are provided in subsequent sections.

The laboratory workflow for Omics Core is illustrated in Figure 1.

The bioinformatics workflow for Omics Core is illustrated in Figure 2.

![](images/2c8b669296831e59f3d82a5c12fa424bec7e279d1e4f8fcba6c02f84a0d01b9f.jpg)  
Figure 1: Laboratory Workflow

![](images/41d27fac30425e000ad2d69173647a7b0068845bfcc5b2ad34e5b2e9a0a9d841.jpg)  
Figure 2: Bioinformatics Workflow

# 4.4. Indications For Use

The Omics Core assay is a qualitative in vitro diagnostic test that uses targeted next generation sequencing of formalin-fixed paraffin-embedded tumor tissue matched with normal specimens from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multi gene panel. The test is intended to provide information on somatic mutations (point mutations and small insertions and deletions) and tumor mutational burden (TMB) for use by qualified health care professionals in accordance with professional guidelines, and is not conclusive or prescriptive for labeled use of any specific therapeutic product. Omics Core is a single-site assay performed at NantHealth, Inc.

The Indications For Use statement for Omics Core is not identical to the predicate device. Omics Core is intended to provide information on tumor mutational burden (TMB) and not intended to provide information on microsatellite instability. Omics Core reports TMB as additional information on the tumor tissue. TMB is reported via two metrics: (1) total number of somatic non-synonymous exonic variants within the 19,396 genes (whole exome) surveyed and (2) a calculation of mutation rate by counting all somatic, synonymous and non-synonymous variants detected in gene coding regions and dividing by the approximate size of the exome $\left( 3 3 . 7 \mathrm { M b } \right)$ . TMB is reported as mutations per megabase (mut/Mb) unit. Mutations included in the calculation of TMB must be present at $5 \%$ allele frequency or greater after direct comparison between tumor and matched normal DNA. TMB is reported as rate (not as High/Low) and as a mutation within the category of “Cancer Mutations with Potential Clinical Significance” to be signed off by the Medical Director. This difference does not constitute a new Indications For Use nor raises different issues of safety and effectiveness.

# 5. COMPARISON OF TECHNOLOGICAL CHARACTERISTICS WITH THE PREDICATE DEVICE

The primary technological characteristics and indications for use of the Omics Core assay are substantially equivalent to MSK-IMPACT. A substantial equivalence table comparing overall similarities and differences between Omics Core and MSK-IMPACT is provided in Table 2.

Table 2: Comparison of Omics Core with Predicate Device   

<table><tr><td colspan="1" rowspan="1">Characteristics</td><td colspan="1" rowspan="1">Predicate Device:MSK-IMPACT (DEN170058)</td><td colspan="1" rowspan="1">Subject Device:Omics Core</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Indications ForUse</td><td colspan="1" rowspan="1">The MSK-IMPACT assay is aqualitative in vitro diagnostic testthat uses targeted next generationsequencing of formalin-fixedparaffin-embedded tumor tissuematched with normal specimensfrom patients with solid malignantneoplasms to detect tumor genealterations in a broad multi genepanel. The test is intended toprovide information on somaticmutations (point mutations andsmall insertions and deletions) andmicrosatellite instability for use byqualified health care professionalsin accordance with professionalguidelines, and is not conclusiveor prescriptive for labeled use ofany specific therapeutic product.MSK-IMPACT is a single-siteassay performed at MemorialSloan Kettering Cancer Center.</td><td colspan="1" rowspan="1">The Omics Core assay is aqualitative in vitro diagnostic testthat uses targeted next generationsequencing of formalin-fixedparaffin-embedded tumor tissuematched with normal specimensfrom patients with solid malignantneoplasms to detect tumor genealterations in a broad multi genepanel. The test is intended toprovide information on somaticmutations (point mutations andsmall insertions and deletions) andtumor mutational burden (TMB)for use by qualified health careprofessionals in accordance withprofessional guidelines, and is notconclusive or prescriptive forlabeled use of any specifictherapeutic product. Omics Core isa single-site assay performed atNantHealth, Inc.</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Hybrid Capture</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Formalin-fixed, paraffin-embedded (FFPE) tumor tissuematched with normal specimensfrom patients with solid malignantneoplasms</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target Population</td><td colspan="1" rowspan="1">Patients with solid malignantneoplasms</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Genes on Panel forReporting SNVsand Indels</td><td colspan="1" rowspan="1">468</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Environment</td><td colspan="1" rowspan="1">Single-site assay (performed atMemorial Sloan Kettering CancerCenter</td><td colspan="1" rowspan="1">Same. Single-site assay(performed at NantHealth, Inc.)</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Matched normalPositive controlNegative controlNo template control (NTC)</td><td colspan="1" rowspan="1">Omics Core uses the same types ofcontrols.</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Predicate Device:MSK-IMPACT (DEN170058)</td><td colspan="1" rowspan="1">Subject Device:Omics Core</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Black List</td><td colspan="1" rowspan="1">73 exons</td><td colspan="1" rowspan="1">203 exons</td></tr><tr><td colspan="1" rowspan="1">Variant types</td><td colspan="1" rowspan="1">Intended to provide informationon somatic mutations (pointmutations and small insertionsand deletions), andmicrosatellite instability</td><td colspan="1" rowspan="1">Same except Omics Core providesinformation on tumor mutationalburden (TMB) and notmicrosatellite instability (MSI).</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Illumina HiSeq® 2500Sequencing System</td><td colspan="1" rowspan="1">Illumina NovaSeqTM 6000Sequencing System</td></tr><tr><td colspan="1" rowspan="1">Determination ofPipeline Thresholds</td><td colspan="1" rowspan="1">•  Based on &gt;200X targetcoverage,•  100X for ≥ 98% targetexons,hotspot mutation callingthreshold (mutationcoverage (DP) ≥ 20, mutantreads (AD) ≥ 8, mutationfrequency (VF) ≥ 2%, andnon-hotspot mutationthreshold (DP ≥ 20,AD ≥ 10, VF ≥ 5%).</td><td colspan="1" rowspan="1">Based on ≥ 500X targetcoverage,≥ 100X for 95% of targetexons, and a mutation callingthreshold of allele frequency(AF) ≥ 2% with Conf &gt; 15 andheuristic filters,•  The minimum read depth forvariants in the Omics Coreassay are allele depth (AD) ≥ 2and overall depth (DP) ≥ 4.</td></tr><tr><td colspan="1" rowspan="1">Assay cut-off</td><td colspan="1" rowspan="1">MSK-IMPACT does not reportmutations below 2% for knownhotspot mutations and 5% fornon-hotspot mutations.</td><td colspan="1" rowspan="1">Omics Core does not reportmutations below 2% for allmutations.</td></tr><tr><td colspan="1" rowspan="1">Clinical EvidenceCurationOncopanel results arereported under one ofthese two categories:CancerMutations withEvidence ofClinicalSignificance" or"CancerMutations withPotential ClinicalSignificance."</td><td colspan="1" rowspan="1">Classification criteria weredeveloped by MSK using thein-house OncoKB database.OncoKB undergoes periodicupdates through the review ofnew information by a panel ofexperts</td><td colspan="1" rowspan="1">Classification criteria weredeveloped by NantHealth, Inc.NantHealth periodically updatesOmics Core through the review ofnew information available.</td></tr></table>

# 6. PERFORMANCE DATA

Table 3 summarizes the performance data that are provided in support of the substantial equivalence determination. Performance specifications were established following special controls outlined for next generation sequencing based tumor profiling test (21 CFR 866.6080).

Table 3: Omics Core Performance Specifications   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">Performance Specifications</td></tr><tr><td colspan="1" rowspan="1">Specimens</td><td colspan="1" rowspan="1">The specimens accepted for testing are formalin-fixed, paraffin-embedded (FFPE) tumor tissue blocks and two vials of wholeblood for the normal comparator. If a tumor block is not available,slides containing tumor sections are acceptable. The FFPE blocksand slides are prepared per industry standards by the requestingmedical facility.Tumor sample requirements are 5-20 unstained sections,10 microns thick. Tumor sections must have more than 10% oftumor cells; however, sections containing &gt;20% viable tumor arepreferred for testing.</td></tr><tr><td colspan="1" rowspan="1">Pre-Analytical</td><td colspan="1" rowspan="1">DNA samples are normalized to yield 50-300ng input in 60μlprior to shearing.</td></tr><tr><td>Characteristic Precision</td><td>Performance Specifications</td></tr><tr><td rowspan="8"></td><td>Precision was assessed using 12 FFPE clinical tumor samples and 1 commercial cell line. A total of 530 variants were identified in the samples that included 483 SNVs, 32 deletions, and 15 insertions.</td></tr><tr><td>Precision  Panel-Wide Reproducibility: The overall positive call rate for all variants analyzed across the</td></tr><tr><td>12 FFPE clinical samples and one commercial cell line was 2607/2650, or 98.4% (97.8-98.8% CI). Three-hundred and ninety- eight (398) of 521 (76%) mutations in the clinical specimens had %CV ≤ 10%; 91 of 521 (17%) were between 10 and 20%; and 32 of 521 (6%) were &gt; 20%.</td></tr><tr><td>Precision  Per Specimen: Per specimen variant analysis for 12 clinical specimens and a</td></tr><tr><td>commercial cell line demonstrated consistent repeatability and reproducibility by showing 100% concordance for 511 out of 530 unique mutations, or 96.4% (94.5%-97.8% CI).</td></tr><tr><td>Precision  Well Characterized Reference Material: Analysis of the well characterized sample demonstrated consistent</td></tr><tr><td>repeatability and reproducibility by achieving an overall positive call rate of 99.86% (99.75%-99.93% CI).</td></tr><tr><td>Precision  TMB: TMB precision analysis was based on 12 clinical samples and 1 commercial cell line. The tumor purity for these samples ranged from 25% to 90%. The per sample TMB analysis demonstrated</td></tr><tr><td>Analytical Sensitivity - Limit of Detection 10 to 20%, demonstrated consistent repeatability and reproducibility as all samples evaluated had a %CV &lt; 10%. The</td><td>repeatable and reproducible TMB rates with a %CV &lt;10% for all 13 samples. LoD  SNVs and Indels: The results from 13 FFPE clinical samples demonstrated the ability to detect and reliably call each variant class at the 5% mutant allele frequency with a success rate of ≥ 95%, with SNVs called at 96.7% (82.8%-99.9% CI), insertions at 100% (83.2%- 100.0% CI) and deletions at 100.0% (78.2%-100.0% CI). LoD - TMB: Ten (10) FFPE tumor specimens, with tumor purities ranging from</td></tr><tr><td>Characteristic</td><td>Performance Specifications</td></tr><tr><td rowspan="5">Accuracy  Comparison To Orthogonal Method</td><td rowspan="2">Accuracy  SNVs and Indels: Accuracy was assessed by comparing Omics Core results to results obtained from the orthogonal method in a total of 401 FFPE tumor samples representing mutations covering 2,634 SNVs, 125 small insertions and 313 small deletions.</td></tr><tr><td>Omics Core successfully detected mutations in all 401 samples assessed, representing accuracy of 100% (99.08-100% CI). • 2634 unique SNVs demonstrated PPA of 99.76% (99.50-99.90% CI), PPV of 99.93% (99.75-99.99% CI) 125 unique small insertions demonstrated PPA of 100% (97.20-100.00% CI), PPV of 100% (97.20-100.00% CI) 313 unique small deletions demonstrated a PPA of</td></tr><tr><td colspan="3">99.71% (98.38-99.99% CI), PPV of 99.71% (98.38- 99.99% CI)</td></tr><tr><td colspan="3">Accuracy - TMB: TMB accuracy was assessed by comparing the TMB rates generated by Omics Core and the orthogonal method. A linear regression demonstrated high correlation between the two methods</td></tr><tr><td colspan="3">(R2 = 0.9899). Accuracy  Supplemental An analysis of 220 positive mutations and 12,612 wildtype calls Method Comparisons demonstrated a PPA of 99.54% (97.48-99.99%) and NPA of Study for Wildtype Calls 99.99% (99.99-100%).</td></tr><tr><td>Traceability</td><td colspan="3">Omics Core assay is not traceable to any known standard. The assay uses matched normal whole blood as a matched normal control, a No Template Control (NTC), and positive and negative controls to monitor the ongoing performance of the assay.</td></tr><tr><td>Stability</td><td colspan="3">Reagent stability is based on manufacturer expiration dating and supported by NantHealth verification. Stability of the reagent is monitored through the use of consistent controls.</td></tr><tr><td>Expected Values</td><td colspan="3">The Omics Core assay does not use calibrators; however, the verification of mutant allele frequency is maintained by analysis of a positive control with expected allele frequencies.</td></tr><tr><td>Analytical Specificity</td><td colspan="3">High analytical specificity is maintained by paired tumor/matched normal sequencing, and established during assay optimization. Interference is minimized with pre-analytic steps. Invalid rates in historical testing from &gt;2,000 samples support that any</td></tr><tr><td>Assay Cut-Off</td><td colspan="3">interference from any challenging tissues is minimized. Omics Core does not report mutations below 2% MAF. Mutations included in the calculation of TMB must be present at 5% allele frequency or greater.</td></tr><tr><td colspan="1" rowspan="1">Clinical Performance</td><td colspan="1" rowspan="1">The genes in the panel include those that play a role in cancerpathogenesis and tumor suppression, or for clinical or mechanisticinformation of relevance in the management of cancer patients.The assay reports mutations under two categories: "CancerMutations with Evidence of Clinical Significance" and "CancerMutations with Potential Clinical Significance" consistent with theintended use clinical settings. Mutations with evidence of clinicalsignificance are represented in professional guidelines asestablished by consensus opinion of experts in the healthcarecommunity.</td></tr><tr><td colspan="1" rowspan="1">Software Verification andValidation Testing</td><td colspan="1" rowspan="1">Software level of concern: MODERATEVerification and validation testing conducted as per FDA'sGuidance for Industry and FDA Staff, "Guidance for the Contentof Premarket Submissions for Software Contained in MedicalDevices" and "Content of Premarket Submissions forManagement of Cybersecurity in Medical Devices"</td></tr></table>

# 7. SUMMARY

The submitted information in this 510(k) notification supports the Indications For Use for Omics Core and demonstrates that the Omics Core assay is as safe and effective as the predicate device and therefore supports a substantial equivalence conclusion.