Personal Genome Diagnostics Jennifer Dickey, Ph.D., RAC VP, Regulatory and Quality 2809 Boston Street, Suite 503 Baltimore, Maryland 21224

Re: K192063 Trade/Device Name: $\mathrm { P G D x ^ { T M } }$ elio tissue complete Regulation Number: 21 CFR 866.6080 Regulation Name: Next generation sequencing based tumor profiling test Regulatory Class: Class II Product Code: PZM Dated: July 31, 2019 Received: August 1, 2019

Dear Jennifer Dickey:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part

801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Donna Roscoe, Ph.D.   
Chief   
Division of Molecular Genetics and Pathology   
OHT7: Office of In Vitro Diagnostics and Radiological Health Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K192063

Device Name PGDx elio™ tissue complete

Indications for Use (Describe)

The PGDx elio™ tissue complete assay is a qualitative in vitro diagnostic device that uses targeted next generation sequencing of DNA isolated from formalin-fixed, paraffin-embedded tumor tissue from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multi-gene panel.

PGDx elio tissue complete is intended to provide tumor mutation profiling information on somatic alterations (SNVs, small insertions and deletions, one amplification and four translocations), microsatellite instability (MSI) and tumor mutation burden (TMB) for use by qualified healthcare professionals in accordance with professional guidelines in oncology for previously diagnosed cancer patients, and is not conclusive or prescriptive for labeled use of any specific therapeutic product.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

The burden time for this collection of information is estimated to average 79 hours per response, including the time to review instructions, search existing data sources, gather and maintain the data needed and complete and review the collection of information. Send comments regarding this burden estimate or any other aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

“An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number.”

# 510(k) Summary

Submission Date: April 24, 2019

# Submitter Information:

Submitted By: Personal Genome Diagnostics Inc. 2809 Boston Street, Suite 503 Baltimore, MD 21224

Contact Person: Jennifer S. Dickey PhD, RAC Vice President, Regulatory & Quality Personal Genome Diagnostics Tel: (443) 602-8833 Email: jdickey@pgdx.com

A. Proprietary and Established Names PGDx elio™ tissue complete

B. 510(k) number

K192063

# C. Measurand

Somatic single nucleotide variants, insertions and deletions, select amplifications and translocations, microsatellite instability and tumor mutation burden in human genomic DNA obtained from formalin-fixed, paraffin-embedded tumor tissue.

# D. Regulatory Information

1. Regulation section   
21 CFR 866.6080

2. Classifications Class II

3. Product Code PZM

# E. Indications for Use

# 1. Indications for Use

The PGDx elio™ tissue complete assay is a qualitative in vitro diagnostic device that uses targeted next generation sequencing of DNA isolated from formalin-fixed, paraffin

PGDx elio™ tissue complete

embedded tumor tissue from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multi-gene panel.

PGDx elio tissue complete is intended to provide tumor mutation profiling information on somatic alterations (SNVs, small insertions and deletions, one amplification and four translocations), microsatellite instability (MSI) and tumor mutation burden (TMB) for use by qualified healthcare professionals in accordance with professional guidelines in oncology for previously diagnosed cancer patients, and is not conclusive or prescriptive for labeled use of any specific therapeutic product.

2. Special conditions for use statement(s): For prescription use. For in vitro diagnostic use.

3. Special Instrument Requirements NextSeq $\textsuperscript { \textregistered }$ 550Dx (qualified by PGDx)

F. Substantial Equivalence Information:

1. Predicate device name(s) MSK-IMPACT

2. Predicate 510(k) Number DEN170058

# 3. Comparison with predicate

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PGDx elio tissue complete</td><td rowspan=1 colspan=1>MSK-IMPACT(Predicate Device)</td></tr><tr><td rowspan=1 colspan=1>K Number</td><td rowspan=1 colspan=1>K192063</td><td rowspan=1 colspan=1>DEN170058</td></tr><tr><td rowspan=1 colspan=3>SIMILARITIES</td></tr><tr><td rowspan=1 colspan=1>AssayIntended Use/ Indicationsfor Use</td><td rowspan=1 colspan=1>The PGDx elioTM tissue completeassay is a qualitative in vitrodiagnostic device that uses targetednext generation sequencing of DNAisolated from formalin-fixed,paraffin-embedded tumor tissuefrom patients with solid malignantneoplasms to detect tumor gene</td><td rowspan=1 colspan=1>The MSK-IMPACT assay is aqualitative in vitro diagnostic testthat uses targeted next generationsequencing of formalin-fixedparaffin-embedded tumor tissuematched with normal specimensfrom patients with solid malignantneoplasms to detect tumor genealterations in a broad multi genepanel. The test is intended to provide</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PGDx elio tissue complete</td><td rowspan=1 colspan=1>MSK-IMPACT(Predicate Device)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>alterations in a broad multi-genepanel.PGDx elio tissue complete isintended to provide tumor mutationprofiling information on somaticalterations (SNVs, small insertionsand deletions, one amplificationand four translocations),microsatellite instability (MSI) andtumor mutation burden (TMB) foruse by qualified healthcareprofessionals in accordance withprofessional guidelines in oncologyfor previously diagnosed cancerpatients, and is not conclusive orprescriptive for labeled use of anyspecific therapeutic product.</td><td rowspan=1 colspan=1>information on somatic mutations(point mutations and small insertionsand deletions) and microsatelliteinstability for use by qualified healthcare professionals in accordancewith professional guidelines, and isnot conclusive or prescriptive forlabeled use of any specifictherapeutic product. MSK-IMPACTis a single-site assay performed atMemorial Sloan Kettering CancerCenter.</td></tr><tr><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>PZM</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>21 CFR 866.6080</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Method</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>FFPE tumor tissue from cancerpatients with solid malignantneoplasms</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TargetPopulation</td><td rowspan=1 colspan=1>Previously diagnosed cancerpatients with solid malignantneoplasms</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Mode ofMeasurement</td><td rowspan=1 colspan=1>PCR and Next GenerationSequencing (hybrid capturemethodology)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Calibrators</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Same</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PGDx elio tissue complete</td><td rowspan=1 colspan=1>MSK-IMPACT(Predicate Device)</td></tr><tr><td rowspan=1 colspan=1>DNA Input</td><td rowspan=1 colspan=1>100 ng</td><td rowspan=1 colspan=1>100-250 ng</td></tr><tr><td rowspan=1 colspan=3>DIFFERENCES</td></tr><tr><td rowspan=1 colspan=1>TestEnvironment</td><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1>Single-site assay (performed atMemorial Sloan Kettering CancerCenter)</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Positive control, negative control,normalized to database of commongermline SNPs</td><td rowspan=1 colspan=1>Matched normal, positive controland negative control</td></tr><tr><td rowspan=1 colspan=1>Assay Target</td><td rowspan=1 colspan=1>•  SNVs and indels in 505genes•  MSI•  Amplification in ERBB2•  Translocations in ALK,RET, NTRK2, and NTRK3•  TMB</td><td rowspan=1 colspan=1>SNVs and indels in 468genes• MSI</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>NextSeq 550Dx (qualified byPGDx)</td><td rowspan=1 colspan=1>HiSeq 2500 Sequencer (qualified byMSK)</td></tr></table>

# G. Summary and Explanation

# 4. Product Description

PGDx elio tissue complete is an in vitro diagnostic assay that uses NGS to detect tumor gene alterations in genomic DNA isolated from formalin-fixed, paraffin-embedded (FFPE) tumor tissue from a variety of tumor types, using a targeted panel (505 genes). The assay takes less than 7 days from DNA to report and provides information on single nucleotide variants (SNVs) in a range of GC content and genomic contexts, insertion/ deletions (indels), 1 amplification as well as 4 translocations. It also identifies microsatellite instability based on select mononucleotide tracts and signatures of sequence mutations. The PGDx elio tissue complete assay utilizes a ${ \sim } 1 . 3$ Mb region of interest (ROI) to calculate tumor mutation burden (TMB). Figure 1.1 describes components of the assay. A complete list of components, equipment and materials can be found in Part 2 (User Guide) of the PGDx elio tissue complete Manual (MN-ETC-03). The panel gene list is provided in Table 1.1.

PGDx elio™ tissue complete

# PGDx elio™M tissue complete: Assay Components

![](images/29253efad7ec340fadb0484dc7f4b871b1aba03f5b60cac7a3347c2a78bc95d9.jpg)  
Figure 1.1 PGDx elio tissue complete assay components. PGDx provided components include reagent kits, software for data analysis, and a server.

Table 1.1   

<table><tr><td rowspan=1 colspan=8>PGDx elio tissue complete Gene List‡</td></tr><tr><td rowspan=1 colspan=1>ABL1</td><td rowspan=1 colspan=1>ABL2</td><td rowspan=1 colspan=1>ACVR1</td><td rowspan=1 colspan=1>ACVR1B</td><td rowspan=1 colspan=1>ADORA2A</td><td rowspan=1 colspan=1>AKT1</td><td rowspan=1 colspan=1>AKT2</td><td rowspan=1 colspan=1>AKT3</td></tr><tr><td rowspan=1 colspan=1>ALK*</td><td rowspan=1 colspan=1>ALOX12B</td><td rowspan=1 colspan=1>AMER1</td><td rowspan=1 colspan=1>APC</td><td rowspan=1 colspan=1>AR</td><td rowspan=1 colspan=1>ARAF</td><td rowspan=1 colspan=1>ARFRP1</td><td rowspan=1 colspan=1>ARID1A</td></tr><tr><td rowspan=1 colspan=1>ARID1B</td><td rowspan=1 colspan=1>ARID2</td><td rowspan=1 colspan=1>ARID5B</td><td rowspan=1 colspan=1>ASXL1</td><td rowspan=1 colspan=1>ASXL2</td><td rowspan=1 colspan=1>ATM</td><td rowspan=1 colspan=1>ATR</td><td rowspan=1 colspan=1>ATRX</td></tr><tr><td rowspan=1 colspan=1>AURKA</td><td rowspan=1 colspan=1>AURKB</td><td rowspan=1 colspan=1>AXIN1</td><td rowspan=1 colspan=1>AXIN2</td><td rowspan=1 colspan=1>AXL</td><td rowspan=1 colspan=1>B2M</td><td rowspan=1 colspan=1>BAP1</td><td rowspan=1 colspan=1>BARD1</td></tr><tr><td rowspan=1 colspan=1>BBC3</td><td rowspan=1 colspan=1>BCL2</td><td rowspan=1 colspan=1>BCL2L1</td><td rowspan=1 colspan=1>BCL2L11</td><td rowspan=1 colspan=1>BCL2L2</td><td rowspan=1 colspan=1>BCL6</td><td rowspan=1 colspan=1>BCOR</td><td rowspan=1 colspan=1>BCORL1</td></tr><tr><td rowspan=1 colspan=1>BCR</td><td rowspan=1 colspan=1>BIRC2</td><td rowspan=1 colspan=1>BLM</td><td rowspan=1 colspan=1>BMPR1A</td><td rowspan=1 colspan=1>BRAF</td><td rowspan=1 colspan=1>BRCA1</td><td rowspan=1 colspan=1>BRCA2</td><td rowspan=1 colspan=1>BRD4</td></tr><tr><td rowspan=1 colspan=1>BRIP1</td><td rowspan=1 colspan=1>BTG1</td><td rowspan=1 colspan=1>BTG2</td><td rowspan=1 colspan=1>BTK</td><td rowspan=1 colspan=1>BUB1B</td><td rowspan=1 colspan=1>C11ORF30</td><td rowspan=1 colspan=1>CALR</td><td rowspan=1 colspan=1>CARD11</td></tr><tr><td rowspan=1 colspan=1>CASP8</td><td rowspan=1 colspan=1>CBFB</td><td rowspan=1 colspan=1>CBL</td><td rowspan=1 colspan=1>CCND1</td><td rowspan=1 colspan=1>CCND2</td><td rowspan=1 colspan=1>CCND3</td><td rowspan=1 colspan=1>CCNE1</td><td rowspan=1 colspan=1>CD22</td></tr><tr><td rowspan=1 colspan=1>CD274</td><td rowspan=1 colspan=1>CD276</td><td rowspan=1 colspan=1>CD70</td><td rowspan=1 colspan=1>CD79A</td><td rowspan=1 colspan=1>CD79B</td><td rowspan=1 colspan=1>CDC73</td><td rowspan=1 colspan=1>CDH1</td><td rowspan=1 colspan=1>CDK12</td></tr><tr><td rowspan=1 colspan=1>CDK4</td><td rowspan=1 colspan=1>CDK6</td><td rowspan=1 colspan=1>CDK8</td><td rowspan=1 colspan=1>CDKN1A</td><td rowspan=1 colspan=1>CDKN1B</td><td rowspan=1 colspan=1>CDKN1C</td><td rowspan=1 colspan=1>CDKN2A</td><td rowspan=1 colspan=1>CDKN2B</td></tr><tr><td rowspan=1 colspan=1>CDKN2C</td><td rowspan=1 colspan=1>CEBPA</td><td rowspan=1 colspan=1>CHD2</td><td rowspan=1 colspan=1>CHD4</td><td rowspan=1 colspan=1>CHEK1</td><td rowspan=1 colspan=1>CHEK2</td><td rowspan=1 colspan=1>CIC</td><td rowspan=1 colspan=1>CREBBP</td></tr><tr><td rowspan=1 colspan=1>CRKL</td><td rowspan=1 colspan=1>CSF1</td><td rowspan=1 colspan=1>CSF1R</td><td rowspan=1 colspan=1>CSF2</td><td rowspan=1 colspan=1>CSF3</td><td rowspan=1 colspan=1>CSF3R</td><td rowspan=1 colspan=1>CTCF</td><td rowspan=1 colspan=1>CTLA4</td></tr><tr><td rowspan=1 colspan=1>CTNNA1</td><td rowspan=1 colspan=1>CTNNB1</td><td rowspan=1 colspan=1>CUL3</td><td rowspan=1 colspan=1>CUL4A</td><td rowspan=1 colspan=1>CXCR2</td><td rowspan=1 colspan=1>CXCR4</td><td rowspan=1 colspan=1>CYLD</td><td rowspan=1 colspan=1>CYP17A1</td></tr><tr><td rowspan=1 colspan=1>DAXX</td><td rowspan=1 colspan=1>DCUN1D1</td><td rowspan=1 colspan=1>DDB2</td><td rowspan=1 colspan=1>DDR1</td><td rowspan=1 colspan=1>DDR2</td><td rowspan=1 colspan=1>DICER1</td><td rowspan=1 colspan=1>DIS3</td><td rowspan=1 colspan=1>DNMT1</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>DNMT3A</td><td rowspan=1 colspan=1>DNMT3B</td><td rowspan=1 colspan=1>DOT1L</td><td rowspan=1 colspan=1>E2F3</td><td rowspan=1 colspan=1>EED</td><td rowspan=1 colspan=1>EGFL7</td><td rowspan=1 colspan=1>EGFR</td><td rowspan=1 colspan=1>EIF1AX</td></tr><tr><td rowspan=1 colspan=1>EP300</td><td rowspan=1 colspan=1>EPAS1</td><td rowspan=1 colspan=1>EPCAM</td><td rowspan=1 colspan=1>EPHA2</td><td rowspan=1 colspan=1>EPHA3</td><td rowspan=1 colspan=1>EPHA5</td><td rowspan=1 colspan=1>EPHA7</td><td rowspan=1 colspan=1>EPHB1</td></tr><tr><td rowspan=1 colspan=1>EPHB4</td><td rowspan=1 colspan=1>ERBB2#</td><td rowspan=1 colspan=1>ERBB3</td><td rowspan=1 colspan=1>ERBB4</td><td rowspan=1 colspan=1>ERCC1</td><td rowspan=1 colspan=1>ERCC2</td><td rowspan=1 colspan=1>ERCC3</td><td rowspan=1 colspan=1>ERCC4</td></tr><tr><td rowspan=1 colspan=1>ERCC5</td><td rowspan=1 colspan=1>ERCC6</td><td rowspan=1 colspan=1>ERCC8</td><td rowspan=1 colspan=1>ERG</td><td rowspan=1 colspan=1>ERRFI1</td><td rowspan=1 colspan=1>ESR1</td><td rowspan=1 colspan=1>ETV1</td><td rowspan=1 colspan=1>ETV4</td></tr><tr><td rowspan=1 colspan=1>ETV5</td><td rowspan=1 colspan=1>ETV6</td><td rowspan=1 colspan=1>EWSR1</td><td rowspan=1 colspan=1>EXT1</td><td rowspan=1 colspan=1>EXT2</td><td rowspan=1 colspan=1>EZH2</td><td rowspan=1 colspan=1>FAM175A</td><td rowspan=1 colspan=1>FAM46C</td></tr><tr><td rowspan=1 colspan=1>FANCA</td><td rowspan=1 colspan=1>FANCB</td><td rowspan=1 colspan=1>FANCC</td><td rowspan=1 colspan=1>FANCD2</td><td rowspan=1 colspan=1>FANCE</td><td rowspan=1 colspan=1>FANCF</td><td rowspan=1 colspan=1>FANCG</td><td rowspan=1 colspan=1>FANCI</td></tr><tr><td rowspan=1 colspan=1>FANCL</td><td rowspan=1 colspan=1>FANCM</td><td rowspan=1 colspan=1>FAS</td><td rowspan=1 colspan=1>FAT1</td><td rowspan=1 colspan=1>FBXW7</td><td rowspan=1 colspan=1>FGF10</td><td rowspan=1 colspan=1>FGF12</td><td rowspan=1 colspan=1>FGF14</td></tr><tr><td rowspan=1 colspan=1>FGF19</td><td rowspan=1 colspan=1>FGF23</td><td rowspan=1 colspan=1>FGF3</td><td rowspan=1 colspan=1>FGF4</td><td rowspan=1 colspan=1>FGF6</td><td rowspan=1 colspan=1>FGFR1</td><td rowspan=1 colspan=1>FGFR2</td><td rowspan=1 colspan=1>FGFR3</td></tr><tr><td rowspan=1 colspan=1>FGFR4</td><td rowspan=1 colspan=1>FH</td><td rowspan=1 colspan=1>FLCN</td><td rowspan=1 colspan=1>FLT1</td><td rowspan=1 colspan=1>FLT3</td><td rowspan=1 colspan=1>FLT4</td><td rowspan=1 colspan=1>FOXA1</td><td rowspan=1 colspan=1>FOXL2</td></tr><tr><td rowspan=1 colspan=1>FOXP1</td><td rowspan=1 colspan=1>FRS2</td><td rowspan=1 colspan=1>FUBP1</td><td rowspan=1 colspan=1>GABRA6</td><td rowspan=1 colspan=1>GATA1</td><td rowspan=1 colspan=1>GATA2</td><td rowspan=1 colspan=1>GATA3</td><td rowspan=1 colspan=1>GATA4</td></tr><tr><td rowspan=1 colspan=1>GATA6</td><td rowspan=1 colspan=1>GID4</td><td rowspan=1 colspan=1>GLI1</td><td rowspan=1 colspan=1>GNA11</td><td rowspan=1 colspan=1>GNA13</td><td rowspan=1 colspan=1>GNAQ</td><td rowspan=1 colspan=1>GNAS</td><td rowspan=1 colspan=1>GPC3</td></tr><tr><td rowspan=1 colspan=1>GPR124</td><td rowspan=1 colspan=1>GREM1</td><td rowspan=1 colspan=1>GRIN2A</td><td rowspan=1 colspan=1>GRM3</td><td rowspan=1 colspan=1>GSK3B</td><td rowspan=1 colspan=1>H3F3A</td><td rowspan=1 colspan=1>H3F3B</td><td rowspan=1 colspan=1>H3F3C</td></tr><tr><td rowspan=1 colspan=1>HDAC1</td><td rowspan=1 colspan=1>HDAC2</td><td rowspan=1 colspan=1>HDAC6</td><td rowspan=1 colspan=1>HGF</td><td rowspan=1 colspan=1>HIST1H1C</td><td rowspan=1 colspan=1>HIST1H2BD</td><td rowspan=1 colspan=1>HIST1H3B</td><td rowspan=1 colspan=1>HNF1A</td></tr><tr><td rowspan=1 colspan=1>HRAS</td><td rowspan=1 colspan=1>HSD3B1</td><td rowspan=1 colspan=1>HSP90AA1</td><td rowspan=1 colspan=1>HSP90AB1</td><td rowspan=1 colspan=1>ICOSLG</td><td rowspan=1 colspan=1>ID3</td><td rowspan=1 colspan=1>IDH1</td><td rowspan=1 colspan=1>IDH2</td></tr><tr><td rowspan=1 colspan=1>IFNGR1</td><td rowspan=1 colspan=1>IGF1</td><td rowspan=1 colspan=1>IGF1R</td><td rowspan=1 colspan=1>IGF2</td><td rowspan=1 colspan=1>IGF2R</td><td rowspan=1 colspan=1>IKBKE</td><td rowspan=1 colspan=1>IKZF1</td><td rowspan=1 colspan=1>IL10</td></tr><tr><td rowspan=1 colspan=1>IL7R</td><td rowspan=1 colspan=1>INHBA</td><td rowspan=1 colspan=1>INPP4A</td><td rowspan=1 colspan=1>INPP4B</td><td rowspan=1 colspan=1>INSR</td><td rowspan=1 colspan=1>IRF2</td><td rowspan=1 colspan=1>IRF4</td><td rowspan=1 colspan=1>IRS1</td></tr><tr><td rowspan=1 colspan=1>IRS2</td><td rowspan=1 colspan=1>JAK1</td><td rowspan=1 colspan=1>JAK2</td><td rowspan=1 colspan=1>JAK3</td><td rowspan=1 colspan=1>JUN</td><td rowspan=1 colspan=1>KAT6A</td><td rowspan=1 colspan=1>KDM5A</td><td rowspan=1 colspan=1>KDM5C</td></tr><tr><td rowspan=1 colspan=1>KDM6A</td><td rowspan=1 colspan=1>KDR</td><td rowspan=1 colspan=1>KEAP1</td><td rowspan=1 colspan=1>KEL</td><td rowspan=1 colspan=1>KIT</td><td rowspan=1 colspan=1>KLF4</td><td rowspan=1 colspan=1>KLHL6</td><td rowspan=1 colspan=1>KMT2A</td></tr><tr><td rowspan=1 colspan=1>KMT2C</td><td rowspan=1 colspan=1>KMT2D</td><td rowspan=1 colspan=1>KRAS</td><td rowspan=1 colspan=1>LATS1</td><td rowspan=1 colspan=1>LATS2</td><td rowspan=1 colspan=1>LMO1</td><td rowspan=1 colspan=1>LRP1B</td><td rowspan=1 colspan=1>LTK</td></tr><tr><td rowspan=1 colspan=1>LYN</td><td rowspan=1 colspan=1>LZTR1</td><td rowspan=1 colspan=1>MAF</td><td rowspan=1 colspan=1>MAGI2</td><td rowspan=1 colspan=1>MAML1</td><td rowspan=1 colspan=1>MAP2K1</td><td rowspan=1 colspan=1>MAP2K2</td><td rowspan=1 colspan=1>MAP2K4</td></tr><tr><td rowspan=1 colspan=1>MAP3K1</td><td rowspan=1 colspan=1>MAP3K13</td><td rowspan=1 colspan=1>MAPK1</td><td rowspan=1 colspan=1>MAX</td><td rowspan=1 colspan=1>MCL1</td><td rowspan=1 colspan=1>MDC1</td><td rowspan=1 colspan=1>MDM2</td><td rowspan=1 colspan=1>MDM4</td></tr><tr><td rowspan=1 colspan=1>MED12</td><td rowspan=1 colspan=1>MEF2B</td><td rowspan=1 colspan=1>MEN1</td><td rowspan=1 colspan=1>MERTK</td><td rowspan=1 colspan=1>MET</td><td rowspan=1 colspan=1>MITF</td><td rowspan=1 colspan=1>MKNK1</td><td rowspan=1 colspan=1>MLH1</td></tr><tr><td rowspan=1 colspan=1>MLH3</td><td rowspan=1 colspan=1>MPL</td><td rowspan=1 colspan=1>MRE11A</td><td rowspan=1 colspan=1>MSH2</td><td rowspan=1 colspan=1>MSH3</td><td rowspan=1 colspan=1>MSH6</td><td rowspan=1 colspan=1>MST1R</td><td rowspan=1 colspan=1>MTAP</td></tr><tr><td rowspan=1 colspan=1>MTOR</td><td rowspan=1 colspan=1>MUTYH</td><td rowspan=1 colspan=1>MYB</td><td rowspan=1 colspan=1>MYC</td><td rowspan=1 colspan=1>MYCL</td><td rowspan=1 colspan=1>MYCN</td><td rowspan=1 colspan=1>MYD88</td><td rowspan=1 colspan=1>MYOD1</td></tr><tr><td rowspan=1 colspan=1>NBN</td><td rowspan=1 colspan=1>NCOA3</td><td rowspan=1 colspan=1>NCOR1</td><td rowspan=1 colspan=1>NF1</td><td rowspan=1 colspan=1>NF2</td><td rowspan=1 colspan=1>NFE2L2</td><td rowspan=1 colspan=1>NFKBIA</td><td rowspan=1 colspan=1>NKX2-1</td></tr><tr><td rowspan=1 colspan=1>NKX3-1</td><td rowspan=1 colspan=1>NOTCH1</td><td rowspan=1 colspan=1>NOTCH2</td><td rowspan=1 colspan=1>NOTCH3</td><td rowspan=1 colspan=1>NOTCH4</td><td rowspan=1 colspan=1>NPM1</td><td rowspan=1 colspan=1>NRAS</td><td rowspan=1 colspan=1>NSD1</td></tr><tr><td rowspan=1 colspan=1>NT5C2</td><td rowspan=1 colspan=1>NTRK1</td><td rowspan=1 colspan=1>NTRK2*</td><td rowspan=1 colspan=1>NTRK3*</td><td rowspan=1 colspan=1>NUP93</td><td rowspan=1 colspan=1>NUTM1</td><td rowspan=1 colspan=1>PAK1</td><td rowspan=1 colspan=1>PAK3</td></tr><tr><td rowspan=1 colspan=1>PAK7</td><td rowspan=1 colspan=1>PALB2</td><td rowspan=1 colspan=1>PARK2</td><td rowspan=1 colspan=1>PARP1</td><td rowspan=1 colspan=1>PARP2</td><td rowspan=1 colspan=1>PARP3</td><td rowspan=1 colspan=1>PAX5</td><td rowspan=1 colspan=1>PAX8</td></tr><tr><td rowspan=1 colspan=1>PBRM1</td><td rowspan=1 colspan=1>PDCD1</td><td rowspan=1 colspan=1>PDCD1LG2</td><td rowspan=1 colspan=1>PDGFRA</td><td rowspan=1 colspan=1>PDGFRB</td><td rowspan=1 colspan=1>PDK1</td><td rowspan=1 colspan=1>PDPK1</td><td rowspan=1 colspan=1>PHOX2B</td></tr><tr><td rowspan=1 colspan=1>PIK3C2B</td><td rowspan=1 colspan=1>PIK3C2G</td><td rowspan=1 colspan=1>PIK3C3</td><td rowspan=1 colspan=1>PIK3CA</td><td rowspan=1 colspan=1>PIK3CB</td><td rowspan=1 colspan=1>PIK3CD</td><td rowspan=1 colspan=1>PIK3CG</td><td rowspan=1 colspan=1>PIK3R1</td></tr><tr><td rowspan=1 colspan=1>PIK3R2</td><td rowspan=1 colspan=1>PIK3R3</td><td rowspan=1 colspan=1>PIM1</td><td rowspan=1 colspan=1>PLCG2</td><td rowspan=1 colspan=1>PLK2</td><td rowspan=1 colspan=1>PMAIP1</td><td rowspan=1 colspan=1>PMS1</td><td rowspan=1 colspan=1>PMS2</td></tr><tr><td rowspan=1 colspan=1>PNRC1</td><td rowspan=1 colspan=1>POLD1</td><td rowspan=1 colspan=1>POLE</td><td rowspan=1 colspan=1>POLH</td><td rowspan=1 colspan=1>POT1</td><td rowspan=1 colspan=1>PPARG</td><td rowspan=1 colspan=1>PPP2R1A</td><td rowspan=1 colspan=1>PPP2R2A</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>PRDM1</td><td rowspan=1 colspan=1>PREX2</td><td rowspan=1 colspan=1>PRKAR1A</td><td rowspan=1 colspan=1>PRKCI</td><td rowspan=1 colspan=1>PRKDC</td><td rowspan=1 colspan=1>PRSS1</td><td rowspan=1 colspan=1>PRSS8</td><td rowspan=1 colspan=1>PTCH1</td></tr><tr><td rowspan=1 colspan=1>PTEN</td><td rowspan=1 colspan=1>PTK2</td><td rowspan=1 colspan=1>PTPN11</td><td rowspan=1 colspan=1>PTPRD</td><td rowspan=1 colspan=1>PTPRO</td><td rowspan=1 colspan=1>PTPRS</td><td rowspan=1 colspan=1>PTPRT</td><td rowspan=1 colspan=1>QKI</td></tr><tr><td rowspan=1 colspan=1>RAC1</td><td rowspan=1 colspan=1>RAD21</td><td rowspan=1 colspan=1>RAD50</td><td rowspan=1 colspan=1>RAD51</td><td rowspan=1 colspan=1>RAD51B</td><td rowspan=1 colspan=1>RAD51C</td><td rowspan=1 colspan=1>RAD51D</td><td rowspan=1 colspan=1>RAD52</td></tr><tr><td rowspan=1 colspan=1>RAD54B</td><td rowspan=1 colspan=1>RAD54L</td><td rowspan=1 colspan=1>RAF1</td><td rowspan=1 colspan=1>RANBP2</td><td rowspan=1 colspan=1>RARA</td><td rowspan=1 colspan=1>RASA1</td><td rowspan=1 colspan=1>RB1</td><td rowspan=1 colspan=1>RBM10</td></tr><tr><td rowspan=1 colspan=1>RECQL4</td><td rowspan=1 colspan=1>REL</td><td rowspan=1 colspan=1>RET*</td><td rowspan=1 colspan=1>RFWD2</td><td rowspan=1 colspan=1>RHOA</td><td rowspan=1 colspan=1>RICTOR</td><td rowspan=1 colspan=1>RIT1</td><td rowspan=1 colspan=1>RNF43</td></tr><tr><td rowspan=1 colspan=1>ROS1</td><td rowspan=1 colspan=1>RPA1</td><td rowspan=1 colspan=1>RPS6KA4</td><td rowspan=1 colspan=1>RPS6KB2</td><td rowspan=1 colspan=1>RPTOR</td><td rowspan=1 colspan=1>RUNX1</td><td rowspan=1 colspan=1>RUNX1T1</td><td rowspan=1 colspan=1>RYBP</td></tr><tr><td rowspan=1 colspan=1>SBDS</td><td rowspan=1 colspan=1>SDHA</td><td rowspan=1 colspan=1>SDHAF2</td><td rowspan=1 colspan=1>SDHB</td><td rowspan=1 colspan=1>SDHC</td><td rowspan=1 colspan=1>SDHD</td><td rowspan=1 colspan=1>SETD2</td><td rowspan=1 colspan=1>SF3B1</td></tr><tr><td rowspan=1 colspan=1>SGK1</td><td rowspan=1 colspan=1>SH2D1A</td><td rowspan=1 colspan=1>SHQ1</td><td rowspan=1 colspan=1>SLIT2</td><td rowspan=1 colspan=1>SLX4</td><td rowspan=1 colspan=1>SMAD2</td><td rowspan=1 colspan=1>SMAD3</td><td rowspan=1 colspan=1>SMAD4</td></tr><tr><td rowspan=1 colspan=1>SMARCA4</td><td rowspan=1 colspan=1>SMARCB1</td><td rowspan=1 colspan=1>SMARCD1</td><td rowspan=1 colspan=1>SMO</td><td rowspan=1 colspan=1>SNCAIP</td><td rowspan=1 colspan=1>SOCS1</td><td rowspan=1 colspan=1>SOX10</td><td rowspan=1 colspan=1>SOX17</td></tr><tr><td rowspan=1 colspan=1>SOX2</td><td rowspan=1 colspan=1>SOX9</td><td rowspan=1 colspan=1>SPEN</td><td rowspan=1 colspan=1>SPOP</td><td rowspan=1 colspan=1>SPTA1</td><td rowspan=1 colspan=1>SRC</td><td rowspan=1 colspan=1>STAG2</td><td rowspan=1 colspan=1>STAT3</td></tr><tr><td rowspan=1 colspan=1>STAT4</td><td rowspan=1 colspan=1>STK11</td><td rowspan=1 colspan=1>STK40</td><td rowspan=1 colspan=1>SUFU</td><td rowspan=1 colspan=1>SUZ12</td><td rowspan=1 colspan=1>SYK</td><td rowspan=1 colspan=1>TAF1</td><td rowspan=1 colspan=1>TBX3</td></tr><tr><td rowspan=1 colspan=1>TEK</td><td rowspan=1 colspan=1>TERC</td><td rowspan=1 colspan=1>TERT</td><td rowspan=1 colspan=1>TET1</td><td rowspan=1 colspan=1>TET2</td><td rowspan=1 colspan=1>TGFBR1</td><td rowspan=1 colspan=1>TGFBR2</td><td rowspan=1 colspan=1>TIPARP</td></tr><tr><td rowspan=1 colspan=1>TLR4</td><td rowspan=1 colspan=1>TLR7</td><td rowspan=1 colspan=1>TLR8</td><td rowspan=1 colspan=1>TLR9</td><td rowspan=1 colspan=1>TMEM127</td><td rowspan=1 colspan=1>TMPRSS2</td><td rowspan=1 colspan=1>TNFAIP3</td><td rowspan=1 colspan=1>TNFRSF14</td></tr><tr><td rowspan=1 colspan=1>TOP1</td><td rowspan=1 colspan=1>TOP2A</td><td rowspan=1 colspan=1>TP53</td><td rowspan=1 colspan=1>TP53BP1</td><td rowspan=1 colspan=1>TP63</td><td rowspan=1 colspan=1>TRAF7</td><td rowspan=1 colspan=1>TSC1</td><td rowspan=1 colspan=1>TSC2</td></tr><tr><td rowspan=1 colspan=1>TSHR</td><td rowspan=1 colspan=1>TYRO3</td><td rowspan=1 colspan=1>U2AF1</td><td rowspan=1 colspan=1>VEGFA</td><td rowspan=1 colspan=1>VHL</td><td rowspan=1 colspan=1>VTCN1</td><td rowspan=1 colspan=1>WAS</td><td rowspan=1 colspan=1>WEE1</td></tr><tr><td rowspan=1 colspan=1>WHSC1</td><td rowspan=1 colspan=1>WHSC1L1</td><td rowspan=1 colspan=1>WISP3</td><td rowspan=1 colspan=1>WRN</td><td rowspan=1 colspan=1>WT1</td><td rowspan=1 colspan=1>XIAP</td><td rowspan=1 colspan=1>XPA</td><td rowspan=1 colspan=1>XPC</td></tr><tr><td rowspan=1 colspan=1>XPO1</td><td rowspan=1 colspan=1>XRCC1</td><td rowspan=1 colspan=1>XRCC2</td><td rowspan=1 colspan=1>XRCC3</td><td rowspan=1 colspan=1>YAP1</td><td rowspan=1 colspan=1>YES1</td><td rowspan=1 colspan=1>ZBTB2</td><td rowspan=1 colspan=1>ZNF217</td></tr><tr><td rowspan=1 colspan=1>ZNF703</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

‡ - All genes listed contribute to TMB score and SNV/indel reporting; \* - Translocations reported for this gene; # - Amplifications reported for this gene

# 5. Sample Preparation

The PGDx elio tissue complete assay requires genomic DNA isolated from FFPE tissue specimens using commercially available DNA extraction methods. The assay is validated for use with DNA recovered from tissue with a minimum of $20 \%$ viable tumor nuclei. If less than $100 \%$ of the tissue section contains $\ge 2 0 \%$ tumor purity, the tissue should be macro-dissected to select as much viable tumor as possible and minimize the amount of adjacent non-tumor tissue. The recommended DNA input for the assay is $1 0 0 ~ \mathrm { n g }$ at a minimum concentration of $1 \mathrm { n g / \mu L }$ ; results can be obtained with inputs down to $5 0 \mathrm { n g }$ .

# 6. Library Preparation

The PGDx elio tissue complete assay workflow begins with genomic DNA. Genomic DNA is quantified using a fluorometer. DNA molecules are mechanically sheared to a target size of 200 bp and subjected to a magnetic bead purification step to remove smaller fragments and perform an exchange of buffer.

PGDx elio™ tissue complete

Fragmented DNA is end-repaired, phosphorylated, and adenylated. Indexed adapters are then ligated to the A-tailed DNA molecules. Unincorporated adapters and reagents are removed by magnetic bead purification. Adapter-ligated DNA is enriched by PCR amplification. Primer dimers and residual reagents are removed by magnetic bead purification. Library quality is assessed using a DNA fragment analyzer prior to hybrid capture.

# 7. Hybrid Capture NGS

The adapter-ligated library is hybridized with biotinylated RNA library baits, and targeted regions are captured using magnetic streptavidin coated beads. Captured libraries are purified to remove baits and incompletely hybridized DNA fragments. Captured libraries are enriched by PCR amplification. Primer dimers and residual reagents are removed by magnetic bead purification. Final library quality is assessed using a DNA fragment analyzer prior to sequencing.

# 8. Sequencing

Sample libraries are quantified and normalized into a sequencing pool of up to 15 samples and the external control. Pooled sample libraries are fluorometrically quantified, loaded on a sequencing flow cell and sequenced using a NextSeq $\textsuperscript { \textregistered }$ 550Dx instrument which has been pre-qualified by PGDx.

# 9. Data Analysis

Sequence data is processed using the PGDx elio platform software. The software contains a user interface that tracks sample status from sequencing through analysis and reporting. Users configure sequencing runs, and an automated pipeline of software for bioinformatic analysis identifies and reports genomic alterations. After processing, the software generates FASTQ files containing sequences and quality scores for each sample. The FASTQ files are then aligned to a reference genome to generate BAM files, which are processed for variant calling of different alteration types (SNVs, indels, amplifications, translocations, and MSI). SNVs and indels are then used to determine TMB scores reported as mutations per megabase.

# 10. Controls

i. Negative Control: A no template control (NTC) can be processed to serve as a negative control to validate the acceptability of all the test samples processed through library preparation and capture steps by testing for sample or reagent contamination. The NTC is not included on the sequencing run.

ii. Positive Control: An external control that is provided in the PGDx elio tissue complete assay reagent kit consists of cell line derived-DNA with multiple verified sequence mutations. The external control is processed from library preparation through sequencing to serve as an end to end control to demonstrate assay

PGDx elio™ tissue complete

performance. The external control is checked for quality during library preparation and after sequencing. Failure of the external control to meet the pre-defined quality metrics will result in all test samples on the run being reported as ‘No result.’

# 11. Results Reporting

PGDx elio tissue complete reports SNVs and indels in protein coding regions across all genes in the panel. In addition, amplifications are reported for ERBB2 as well as translocations for ALK, RET, NTRK2, and NTRK3. The assay also reports on 2 genomic signatures, MSI and TMB.

The variants listed in the section Variants with Evidence of Clinical Significance are determined based on the selected tumor type. Only variants clinically associated with the tested tumor type will appear in the Variants with Evidence of Clinical Significance section. Any remaining detected variants will appear as the Variants with Potential Clinical Significance. A qualified healthcare professional can select the appropriate tumor type, and depending on the tumor type selected, variants will be reported as Variants with Evidence of Clinical Significance by PGDx elio tissue complete. Any variants clinically associated with tumor types other than the one selected will be reported in the section labeled 'Variants with Potential Clinical Significance. A list of all genes is provided in Appendix A.

SNVs and indels results are also presemted in terms of somatic hotspot or non-hotspot. PGDx defines hotspots as $> 2 5$ exact hits in COSMIC version 72. A lower minimum MAF is used when reporting these hotspot mutations.

Table 1: Summary of PGDx elio tissue complete Quality Control Metrics Postsequencing   

<table><tr><td rowspan=1 colspan=1>Quality Metric</td><td rowspan=1 colspan=1>Level ofQualification</td><td rowspan=1 colspan=1>Passing Criteria</td></tr><tr><td rowspan=1 colspan=1>Cluster Density</td><td rowspan=1 colspan=1>Batch-level</td><td rowspan=1 colspan=1>Sequencer Cluster Density ≥ 130</td></tr><tr><td rowspan=1 colspan=1>Q30 Reads</td><td rowspan=1 colspan=1>Batch-level</td><td rowspan=1 colspan=1>%Q30 (Read1 and Read4) ≥ 80%%Q30 (Read2 and Read3) ≥ 85%</td></tr><tr><td rowspan=1 colspan=1>ExternalControl</td><td rowspan=1 colspan=1>Batch-level</td><td rowspan=1 colspan=1>All expected sequence mutations are detected andpasses all other quality criteria</td></tr><tr><td rowspan=1 colspan=1>Percent RegionsCovered</td><td rowspan=1 colspan=1>Sample-level</td><td rowspan=1 colspan=1>≥ 90% exons with &gt; 100x Median Distinct Coverage</td></tr><tr><td rowspan=1 colspan=1>Percent ReadsIdentified</td><td rowspan=1 colspan=1>Sample-level</td><td rowspan=1 colspan=1>Percent Reads Identified 15%-35%</td></tr><tr><td rowspan=1 colspan=1>ContaminationQC</td><td rowspan=1 colspan=1>Sample-level</td><td rowspan=1 colspan=1>Estimated contamination levels &lt; 2%</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>Quality Metric</td><td rowspan=1 colspan=1>Level ofQualification</td><td rowspan=1 colspan=1>Passing Criteria</td></tr><tr><td rowspan=1 colspan=1>Select SNVsand Indels withEvidence ofClinicalSignificance</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Mutant reads ≥ 4MAF &gt; 0.4%</td></tr><tr><td rowspan=1 colspan=1>Hotspot SNVsand Indels</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Mutant reads ≥ 4MAF &gt; 2%</td></tr><tr><td rowspan=1 colspan=1>Non-hotspotSNVs</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Mutant reads ≥ 6MAF with lower bound 95% CI ≥ 5%</td></tr><tr><td rowspan=1 colspan=1>Non-hotspotIndels</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Mutant reads ≥ 6MAF &gt; 5%</td></tr><tr><td rowspan=1 colspan=1>HomopolymerIndels</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Homopolymer regions &lt; 5 bp orHomopolymer regions ≥ 5 bp with MAF ≥ 12%</td></tr><tr><td rowspan=1 colspan=1>ERBB2Amplifications</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Fold change ≥ 2.5 in ≥ 25% regions covered</td></tr><tr><td rowspan=1 colspan=1>Translocations(ALK, NTRK2,NTRK3 andRET)</td><td rowspan=1 colspan=1>Analyte-level</td><td rowspan=1 colspan=1>Fusion reads ≥ 3</td></tr></table>

# 12. Analytical Studies

# 13. Specificity

i. Limit of Blank (LoB)

Non-cancerous FFPE tissues were assessed for analytical specificity to confirm the reporting thresholds and quality metrics minimize false positives. Two reference standards from National Institute of Standards and Technology (NIST), NA24531 and NA24385 were evaluated by PGDx elio tissue complete for variants reported at the $1 0 0 \mathrm { n g }$ DNA input. Specificity was observed at $100 \%$ with no unverified mutations reported across 5 replicates for each standard. Unique test cases from normal FFPE samples were processed with the recommended $1 0 0 \mathrm { n g \ D N A }$ input across 2 different lots of the PGDx elio tissue complete assay kit. For Variants with Evidence of Clinical Significance, the rate of false positives is $< 0 . 1 \%$ while the false positive rate for hotspot SNVs is $< 3 . 2 \%$ $\mathrm { n } { = } 2 / 6 3$ ). For MSI-H, the false positive rate is $< 1 . 6 \%$ .

ii. Cross Reactivity

PGDx elio™ tissue complete

An in silico cross-reactivity analysis was performed to evaluate specificity of capture baits based on sequence identity of target baits to the human reference genome. Bait sequences corresponding to gene regions of clinical significance demonstrated specificity to target regions based on mapping quality or unique sequence identity to the human reference genome. Any bait that had some sequence identity to regions other than the target region of interest was further investigated and were shown to be specific. This study demonstrated that the oligonucleotide baits in the PGDx elio tissue complete assay are specific to the target regions of DNA intended to be reported.

# 14. Sensitivity

i. Limit of Detection (LoD) – SNVs and Indels

The LoD was assessed by variant type and is defined as the lowest MAF at which $2 9 5 \%$ of replicates are detected. The LoD was evaluated 2 ways, through a dilution series using cell lines to determine the LoD and by confirmation with clinical specimens. Ten unique clinical cases were selected for SNVs, insertions and deletions and diluted with normal DNA derived from FFPE tissues. Each unique clinical case was tested with 10 replicates across 2 kit lots $\mathrm { \Pi } ^ { \prime } \mathrm { n } ^ { = 2 0 }$ per specimen) for a total of 200 observations (Error! Reference source not found.). Data was aggregated across 2 reagent kit lots when possible, otherwise the lot with the higher MAF LoD is displayed.

Additional evaluations of analytical sensitivity performance used dilution series of FFPE clinical specimens. The positive call rates were assessed for a total of 11 SNVs, 3 insertions, and 5 deletions from 5 clinical FFPE specimens with 5 replicates per dilution level. A range of $5 . 9 \mathrm { - } 1 2 . 6 \%$ MAF was observed using the lowest average MAF where the positive call rates was $\ge 9 5 \%$ .

Cell lines were used to establish the LoD MAF range for 451 SNVs and 31 indels across the panel. A total of 150 observations were generated (3 samples with 10 replicates at 5 dilution levels). The established ranges were then confirmed with $a \geq 9 5 \%$ call rate with FFPE clinical cases on a per variant level (Error! Reference source not found.) and for the entire panel (Table 1.2). A summary of the LoD by variant type across all replicates is shown in the table below.

Table 1.2 Analytical Sensitivity (LoD MAF) for Representative SNVs and Indels   

<table><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>MAF Range</td><td rowspan=1 colspan=1>Cell Line VariantsEvaluated toEstablish MAFRange</td><td rowspan=1 colspan=1>Number of Variants inClinical Cases in theEstablished Range</td></tr><tr><td rowspan=1 colspan=1>Hotspot SNVs</td><td rowspan=1 colspan=1>3.1% to 5.4%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-hotspot SNVs</td><td rowspan=1 colspan=1>6.3% to 17.8%</td><td rowspan=1 colspan=1>443</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=1>Indels at homopolymercontext1</td><td rowspan=1 colspan=1>13.7% to 17.5%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td>Indels at non-homopolymer context</td><td>6.1% to 10.9%</td><td>19</td><td>4</td></tr></table>

1 Greater than or equal to 5 bp repeat ii. LoD – ERBB2, ALK, RET, NTRK2, NTRK3 and MSI

Analytical sensitivity of ERBB2, ALK, RET, NTRK3, and MSI was confirmed by testing 7 clinical FFPE cases diluted with normal FFPE DNA to achieve targeted detection levels. Each unique case was confirmed at $\ge 9 5 \%$ call rate at 1 tumor purity level, with 10 replicates per kit lot, across 2 unique lots for translocations and amplifications. For MSI-H, 3 cases were confirmed at 1 tumor purity level with 10 replicates each. Results summarized in Table 1.3 indicate that the assay is sensitive in detecting specific translocations, amplifications and MSI-H.

Table 1.3 Analytical Sensitivity (LoD Tumor Purity) – Translocations, Amplifications and MSI   

<table><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>LoD Tumor Purity</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>18.1%</td></tr><tr><td rowspan=1 colspan=1>ERBB2 amplifications</td><td rowspan=1 colspan=1>4.4%</td></tr><tr><td rowspan=1 colspan=1>ALK translocations1</td><td rowspan=1 colspan=1>5.6%</td></tr><tr><td rowspan=1 colspan=1>NTRK2 translocations</td><td rowspan=1 colspan=1>30%2</td></tr><tr><td rowspan=1 colspan=1>NTRK3 translocations</td><td rowspan=1 colspan=1>11.5%</td></tr><tr><td rowspan=1 colspan=1>RET translocations</td><td rowspan=1 colspan=1>12.8%</td></tr></table>

1 The enrolled ALK case was evaluated with only 17 total replicates due to insufficient DNA quantity.

2 In silico down sampling suggests LoD $3 \%$ .

iii. TMB and Tumor Purity

The minimum tumor purity requirement for input into PGDx elio tissue complete is $20 \%$ . The minimum tumor purity required for robust reporting of TMB scores by PGDx elio tissue complete was established using 8 clinical FFPE cases. Samples 1-5 were serially diluted across 3 levels with 5 replicates per level and 1 level with 3 replicates (18 total), sample 6 was serially diluted across 5 levels at 10 replicates per level (50 total), and samples 7-8 were serially diluted across 5 levels with 5 replicates per level (25 total, 10 of which had a tumor purity $\geq 1 5 \%$ ). The total number of replicates per sample and the $\% \mathrm { C V }$ of all replicates with $\geq 1 5 \%$ tumor purity are shown below in Table 1.4. Together these data show PGDx elio tissue complete TMB performance across tumor purities at or above $1 5 \%$ . Figure 1.2 demonstrates the consistency in TMB as a function of tumor purity for each of the specimens assessed across a range of TMB Muts/Mb scores.

PGDx elio™ tissue complete iv. DNA Extraction

Table 1.4 TMB Precision for Samples $\ge 1 5 \%$ Tumor Purity   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Reference UndilutedTMB Score</td><td rowspan=1 colspan=1>CV of Replicates ≥ 15%Tumor Purity</td><td rowspan=1 colspan=1>Number of Replicates 15% Tumor Purity</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>12.5%</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>50.8</td><td rowspan=1 colspan=1>6.3%</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>64.7</td><td rowspan=1 colspan=1>15.9%</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>8.0%</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>455.4</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>6.0%</td><td rowspan=1 colspan=1>10</td></tr></table>

![](images/ff16af896eb7244a72329444d2a5a30c9526215dfabf47dfd30e5fc17a9ef519.jpg)  
Figure 1.2 Linearity of TMB score with tumor purity in PGDx elio tissue complete. The tumor purity is shown on the $\mathbf { X }$ -axis and the mean TMB score of the replicates at a specific tumor purity is shown on the y-axis.

PGDx elio tissue complete is compatible with genomic DNA extracted from FFPE samples using any appropriate commercially available FFPE extraction method. Samples (3 FFPE specimens and 1 cell line) were extracted in duplicate by 2 operators using 3 different methods, equaling 48 total samples and processed in duplicate for a total of 96 observations to assess concordance. Data for all variant types, including BRAF V600

PGDx elio™ tissue complete

SNV, ALK and NTRK3 translocations and ERBB2 amplification, were aggregated and Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) are presented in Table 1.5. Method 2 (bead-based) and Method 3 (automated) were compared to the reference Method 1 (column-based). The overall pass rate for FFPE samples was $9 3 . 1 \%$ (67/72). The $\% \mathrm { C V }$ for TMB in assessed cases was $< 1 2 . 5 \%$ . All DNA extraction methods yielded concordant results in variant calls with PGDx elio tissue complete.

Table 1.5 DNA Extraction Methods Compared to Reference   

<table><tr><td rowspan=1 colspan=1>DNA Extraction Method</td><td rowspan=1 colspan=1>Concordance in Variant Calls with Method 1 (n/N) (2-sided 95%CII</td></tr><tr><td rowspan=2 colspan=1>Method 2</td><td rowspan=1 colspan=1>PPA - 97.8% (673/688) (96.4%, 98.7%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (71535481/71535520) (99.9%, 100%)</td></tr><tr><td rowspan=2 colspan=1>Method 3</td><td rowspan=1 colspan=1>PPA - 97.5% (624/640) (96.0%, 98.5%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (51416128/51416155) (99.9%, 100%)</td></tr></table>

v. DNA Input

The recommended DNA input for PGDx elio tissue complete is $1 0 0 ~ \mathrm { n g }$ . To evaluate assay performance across a range of DNA inputs, 4 unique FFPE samples with known variants were prepared in triplicate at 10, 25, 50, 100, and $2 0 0 \mathrm { n g }$ DNA input levels. The variant calls for these samples were compared to the respective reference DNA input of $1 0 0 ~ \mathrm { n g }$ for each case to assess concordance. Table 1.6 describes PPA and NPA for each input level where aggregated variants were analyzed, including SNVs, indels, amplifications, translocations, and MSI. For TMB, the mean absolute percent error rate of 10, 25, 50 and $2 0 0 \mathrm { n g }$ DNA input compared to $1 0 0 ~ \mathrm { n g }$ were $1 1 . 8 \%$ , $3 . 3 \%$ , $4 . 4 \%$ and $1 . 8 \%$ , respectively. These data indicate the assay is robust around the recommended 100 ng DNA input.

Table 1.6 DNA Input Compared to $\mathbf { 1 0 0 \ n g }$ Reference   

<table><tr><td rowspan=1 colspan=1>DNA Input</td><td rowspan=1 colspan=1>Variant Call Concordance (n/N) (2-sided 95% CI)</td></tr><tr><td rowspan=2 colspan=1>10 ng</td><td rowspan=1 colspan=1>PPA - 92.2% (177/192) (87.5%, 95.2%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (26825815/26825826) (99.9%, 100%)</td></tr><tr><td rowspan=2 colspan=1>25 ng</td><td rowspan=1 colspan=1>PPA - 94.8% (182/192) (90.7%, 97.1%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (26825815/26825826) (99.9%, 100%)</td></tr><tr><td rowspan=2 colspan=1>50 ng</td><td rowspan=1 colspan=1>PPA - 96.9% (186/192) (93.4%, 98.6%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (26825822/26825826) (99.9%, 100%)</td></tr><tr><td rowspan=2 colspan=1>200 ng</td><td rowspan=1 colspan=1>PPA - 97.4% (187/192) (94.0%, 98.9%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (26825818/26825826) (99.9%, 100%)</td></tr></table>

vi. Sample Carryover and Cross-contamination

PGDx elio™ tissue complete

Cross-contamination (contamination from one sample to another within the same batch) and sample carryover (contamination from a previous sequencing run when using the same instrument) were assessed by evaluating false positive and false negative variant calls in 29 FFPE samples. Seven of the 29 cases had known positive variants, the remaining samples were known negative samples. All FFPE samples were assessed across 2 batches to test for contamination within and between runs. In batch 1, a checkerboard pattern within a 96-well plate was created by alternating the samples with representative positive variants and known negative samples. Batch 2 contained known negative samples and was pooled and sequenced directly after completion of batch 1 sequencing, following standard instrument cleaning procedures. No positive variant results were observed in known negative samples tested. Therefore, the PGDx elio tissue complete assay workflow presents minimal risk for contamination.

# 15. Interference (Endogenous and Exogenous)

i. Interfering Substances (Exogenous)

The impact of exogenous interfering substances on the performance of the PGDx elio tissue complete assay was assessed by processing DNA from FPPE samples tested in the presence of each interfering substance at varying amounts (Table 1.7). The samples were evaluated for concordance of variant calls when compared to samples processed without the interfering substances. Replicates for 5 test cases were analyzed for 8 experimental and 2 baseline conditions. Analysis of all variant types tested (SNVs, indels, translocations, amplifications and MSI) showed high PPA $( > 9 7 . 2 \% )$ and NPA $( > 9 9 . 9 \% )$ for all variants. The TMB mean absolute percent error (MAPE) ranged from $0 \%$ to $6 . 0 \%$ across conditions. The results show minimal risk to assay performance from interfering exogenous substances.

Table 1.7 Exogenous Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Amount in Excess of StandardConditions</td></tr><tr><td rowspan=1 colspan=1>Proteinase K</td><td rowspan=1 colspan=1>2X and 3X</td></tr><tr><td rowspan=1 colspan=1>Indexed adapters</td><td rowspan=1 colspan=1>15% and 30%</td></tr><tr><td rowspan=1 colspan=1>Melanin</td><td rowspan=1 colspan=1>0.2 µg/mL and 1.6 µg/mL</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td><td rowspan=1 colspan=1>2.5% and 5%</td></tr></table>

# ii. Endogenous Interference

The impact of necrosis and FFPE block age on the performance of PGDx elio tissue complete was evaluated by assessing the first pass and overall pass rates of samples processed in the accuracy study (see Accuracy section below). Of 521 samples enrolled for accuracy, 448 were evaluated for necrosis over a range of $0 - 7 5 \%$ , and 378 were evaluated for age of block over a range of 0-253 months. The data indicated there is no correlation between necrosis and pass rate. A logistic regression analysis was performed to establish the probability of sample pass/fail according to FFPE block age as shown in Figure 1.3. There is a correlation between overall pass rate and the age of block; as the age of block increases, the overall pass rate decreases. The probability of samples passing from blocks aged roughly 175 months (or 14.5 years) is approximately $7 5 \%$ . The results show minimal risk to assay performance from interfering endogenous factors.

![](images/b4a897ac3991d25569e0346f4644bcf08513158cb87f3338bb06a0a9e3a7bde7.jpg)  
Figure 1.3 Logistic Regression Graphs of First Pass and Overall Acceptability vs FFPE Block Age. The orange line represents a regression line, and orange shading represents the $9 5 \%$ confidence interval. The blue dots represent individual samples assessed.

# 16. Assay Acceptance Rates

Multiple factors can influence overall robustness and performance of complex molecular tests, including pre-analytical factors and overall sample quality. If key in-process or automated data quality metrics are not met, PGDx elio tissue complete supports repeating samples through the workflow. Performance throughout verification and validation of the device was tracked and a summary of the rates for first pass (no repeat) and overall pass (allowing for a single repeat) are presented below.

# i. Overall Clinical FFPE Sample Acceptance Rate

Data were aggregated for unique clinical cases from ${ > } 4 0$ tumor types assessed during verification and validation of PGDx elio tissue complete. Resulting pass rates for clinical samples are presented in Table 1.8. The data indicate that results are obtained on a high percentage of FFPE samples after processing through the PGDx elio tissue complete assay.

Table 1.8 PGDx elio tissue complete Acceptability Rates   

<table><tr><td>First Pass Rate (n/N) (2-sided 95% CI)  Overall Pass Rate (n/N) (2-sided 95% CI)</td><td></td></tr><tr><td>81.8% (2352/2874) (80.4%, 83.2%)</td><td>92.9% (2671/2874) (91.9%, 93.8%)</td></tr></table>

# ii. Pan-Tumor Type/Tissue Comparability

PGDx elio™ tissue complete

Invalid rates for the different tumor types assessed in the analytical accuracy study are provided in Table 1.9 below.

Table 1.9 Specimen Invalid Rates for $\mathbf { \tau } > 1 8$ FFPE Tumor Types   

<table><tr><td rowspan=1 colspan=1>Tumor Type</td><td rowspan=1 colspan=1>Passing Samples</td><td rowspan=1 colspan=1>Total Samples</td><td rowspan=1 colspan=1>Invalid Rate (%)</td></tr><tr><td rowspan=1 colspan=1>Bladder</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14.3</td></tr><tr><td rowspan=1 colspan=1>Brain</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Breast</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>16.7</td></tr><tr><td rowspan=1 colspan=1>Colorectal</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>6.2</td></tr><tr><td rowspan=1 colspan=1>Endometrial</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Gastric</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>19.4</td></tr><tr><td rowspan=1 colspan=1>Glioma</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Head and Neck</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>16.7</td></tr><tr><td rowspan=1 colspan=1>Lung - NOS1</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>5.9</td></tr><tr><td rowspan=1 colspan=1>Melanoma</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>5.6</td></tr><tr><td rowspan=1 colspan=1>NOS1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>NSCLC1</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>7.6</td></tr><tr><td rowspan=1 colspan=1>Other2</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>Ovarian</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=1 colspan=1>Pediatric Glioma</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Prostate</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12.5</td></tr><tr><td rowspan=1 colspan=1>Skin</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Triple Negative Breast</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>479</td><td rowspan=1 colspan=1>521</td><td rowspan=1 colspan=1>8.1</td></tr></table>

1NOS: not otherwise specified; NSCLC: non-small cell lung cancer. 2Other $[ \mathfrak { n } \leq 3$ cases per tumor type): cervical, cholangiocarcinoma, gallbladder, pancreatic, rhabdomyosarcoma, trachea, esophageal, fallopian tube, liver, mediastinum, peritoneal, renal, and thyroid.

# 17. Accuracy – Concordance to Orthogonal Methods

In order to demonstrate the accuracy of PGDx elio tissue complete as a tumor profiling device, a study was performed with 582 samples that had both PGDx elio tissue complete data and orthogonal data. Due to the rarity of specific genetic variants in solid tumor FFPE samples, most samples selected for this study were pre-screened, resulting in enrichment of certain variants relative to real-world clinical prevalence. Data were

PGDx elio™ tissue complete

aggregated at the variant level for SNVs, insertions, and deletions, gene level for amplifications and translocations, and case level for MSI and TMB.

The results summarized in Table 1.10 indicate that the assay accurately detects SNVs and indels.

Table 1.10 Accuracy – SNVs and Indels   

<table><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>OrthogonalMethod(s)</td><td rowspan=1 colspan=1>Performance (n/N) (2-sided 95% CI)</td></tr><tr><td rowspan=2 colspan=1>SNVs with Evidenceof ClinicalSignificance</td><td rowspan=2 colspan=1>2 NGStargetedpanels</td><td rowspan=1 colspan=1>PPA - 97.2% (35/36) (85.8%, 99.5%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (3994/3996) (99.8%, 99.9%)</td></tr><tr><td rowspan=2 colspan=1>Hotspot SNVs</td><td rowspan=2 colspan=1>2 NGStargetedpanels andPCR</td><td rowspan=1 colspan=1>PPA - 97.1% (132/136) (92.7%, 98.9%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (35845/35850) (99.9%,99.9%)</td></tr><tr><td rowspan=2 colspan=1>Non-hotspot SNVs</td><td rowspan=2 colspan=1>2 NGStargetedpanels</td><td rowspan=1 colspan=1>PPA - 85.1% (516/606) (82.1%, 87.8%)</td></tr><tr><td rowspan=1 colspan=1>NPA  99.9% (178513452/178513618)(99.9%, 99.9%)</td></tr><tr><td rowspan=2 colspan=1>SNVs with PotentialClinical Significance</td><td rowspan=2 colspan=1>2 NGStargetedpanels</td><td rowspan=1 colspan=1>PPA  86.4% (591/684) (83.6%, 88.8%)</td></tr><tr><td rowspan=1 colspan=1>NPA  99.9% (178513372/178513540)(99.9%, 99.9%)</td></tr><tr><td rowspan=2 colspan=1>Hotspot deletions</td><td rowspan=2 colspan=1>2 NGStargetedpanels andPCR</td><td rowspan=1 colspan=1>PPA - 100% (20/20) (84.5%, 100%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (2064/2067) (99.6%, 99.9%)</td></tr><tr><td rowspan=2 colspan=1>Hotspot insertions</td><td rowspan=2 colspan=1>2 NGStargetedpanels</td><td rowspan=1 colspan=1>PPA - 100% (1/1) (20.7%, 100%)</td></tr><tr><td rowspan=1 colspan=1>NPA  100% (2015/2015) (99.8%, 100%)</td></tr><tr><td rowspan=2 colspan=1>Non-hotspot indels</td><td rowspan=2 colspan=1>NGStargetedpanel</td><td rowspan=1 colspan=1>PPA - 81.4% (79/97) (72.6%, 87.9%)</td></tr><tr><td rowspan=1 colspan=1>NPA  99.9% (67104842/67104857)(99.9%, 99.9%)</td></tr><tr><td rowspan=2 colspan=1>Non-hotspotinsertions</td><td rowspan=2 colspan=1>NGStargetedpanel</td><td rowspan=1 colspan=1>PPA - 80.8% (21/26) (62.1%, 91.5%)</td></tr><tr><td rowspan=1 colspan=1>NPA  99.9% (67104926/67104928)</td></tr><tr><td rowspan=2 colspan=1>Non-hotspotdeletions</td><td rowspan=2 colspan=1>NGStargetedpanel</td><td rowspan=1 colspan=1>PPA - 81.7% (58/71) (71.2%, 89.0%)</td></tr><tr><td rowspan=1 colspan=1>NPA - 99.9% (67104870/67104883)</td></tr><tr><td rowspan=2 colspan=1>Insertions withPotential ClinicalSignificance</td><td rowspan=2 colspan=1>NGStargetedpanel</td><td rowspan=1 colspan=1>PPA - 80.8% (21/26) (62.1%, 91.5%)</td></tr><tr><td rowspan=1 colspan=1>NPA  99.9% (67497962/67497964)(99.9%, 99.9%)</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan="2">Deletions with Potential Clinical Significance</td><td rowspan="2">NGS targeted panel</td><td colspan="2">PPA  82.7% (62/75) (72.6%, 89.6%)</td></tr><tr><td>(99.9%, 99.9%)</td><td>NPA - 99.9% (67497902/67497915)</td></tr></table>

# a) Accuracy for ERBB2 Amplifications

Concordance for ERBB2 Amplifications was assessed by comparing PGDx elio tissue complete to ERBB2 FISH (Table 1.11 and Table 1.12). The PPA was $7 5 . 0 \%$ $9 5 \%$ CI: $6 2 . 3 \%$ , $8 4 . 5 \%$ ) for all cases and $8 7 . 0 \%$ $9 5 \%$ CI: $7 4 . 3 \%$ , $9 3 . 9 \%$ when excluding borderline FISH cases (defined as HER2/CEP17 ratio between 1.5 and 2.5). The NPA was $9 6 . 7 \%$ $9 5 \%$ CI: $9 0 . 8 \%$ , $9 8 . 9 \%$ for all cases and $9 5 . 9 \%$ ( $9 5 \%$ CI: $8 8 . 7 \%$ , $9 8 . 6 \%$ when excluding borderline FISH cases.

Table 1.11 Summary of Concordance between PGDx elio tissue complete and ERBB2 FISH Including Borderline Cases   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>ERBB2 FISH</td></tr><tr><td rowspan=1 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>ERBB2 Positive</td><td rowspan=1 colspan=1>ERBB2 Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>ERBB2 Positive</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>ERBB2 Negative</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>147</td></tr></table>

Table 1.12 Summary of Concordance between PGDx elio tissue complete and ERBB2 FISH Excluding Borderline Cases   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>ERBB2 FISH</td></tr><tr><td rowspan=1 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>ERBB2 Positive</td><td rowspan=1 colspan=1>ERBB2 Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>ERBB2 Positive</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>ERBB2 Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>120</td></tr></table>

# b) Accuracy for ALK Translocations

Concordance for ALK translocations was assessed by comparing PGDx elio tissue complete to ALK FISH (Table 1.13). The PPA was $9 2 . 9 \%$ ( $9 5 \%$ CI: $6 8 . 5 \%$ , $9 8 . 7 \%$ ), and NPA was $9 8 . 2 \%$ $9 5 \%$ CI: $9 0 . 7 \%$ , $9 9 . 7 \%$ ).

Table 1.13 Summary of Concordance between PGDx elio tissue complete and ALK

PGDx elio™ tissue complete

FISH   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>ALK FISH</td></tr><tr><td rowspan=1 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>ALK Positive</td><td rowspan=1 colspan=1>ALK Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>ALK Positive</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>ALK Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>71</td></tr></table>

ALK translocations were additionally assessed in silico due to limited availability of clinical cases close to the ALK FISH equivocal zone $10 \% { - } 5 0 \%$ rearrangement positive nuclei). A total of 410 observations were generated for ALK by down-sampling 10 clinical samples from analytical accuracy to 4 tumor purity dilution levels with 10 replicates per level, to mimic samples in the FISH equivocal zone. For example, if the undiluted sample had a FISH score of $50 \%$ from analytical accuracy, the sample was diluted with wild type reads by a factor of 0.8 to get to a $40 \%$ positive nuclei FISH score. These data demonstrate an $8 8 \%$ positive call rate at $20 \%$ positive nuclei by FISH (Table 1.14).

Table 1.14 In silico Analysis of ALK Translocation Borderline Performance   

<table><tr><td rowspan=1 colspan=1>FISH (%PositiveNuclei)</td><td rowspan=1 colspan=1>FISHCall</td><td rowspan=1 colspan=1>PGDx elio tissue complete Positive Call Rate (%) (n/N)(95% CI)</td></tr><tr><td rowspan=1 colspan=1>50 - 88</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>100% (10/10) (72%, 100%)</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>98% (98/100) (93%, 99%)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>+$</td><td rowspan=1 colspan=1>95% (95/100) (89%, 98%)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>88% (88/100) (80%, 93%)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>80% (80/100) (71%, 87%)</td></tr></table>

# c) Accuracy for RET Translocations

Concordance for RET translocations was assessed by comparing PGDx elio tissue complete to RET FISH (Table 1.15). The PPA was $5 5 . 6 \%$ ( $9 5 \%$ CI: $2 6 . 7 \%$ , $8 1 . 1 \%$ ), and NPA was $100 \%$ ( $9 5 \%$ CI: $8 2 . 4 \%$ , $100 \%$ ).

Table 1.15 Summary of Concordance between PGDx elio tissue complete and RET FISH   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>RET FISH</td></tr><tr><td rowspan=1 colspan=1>PGDx elio</td><td rowspan=1 colspan=1>RET Positive</td><td rowspan=1 colspan=1>RET Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>RET Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>RET Negative</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>22</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td>Total</td><td>9</td><td>18</td><td>27</td></tr><tr><td></td><td></td><td></td><td></td></tr></table>

1No read data supporting RET translocations was found in the raw data for the discrepant cases. In addition to the clinical samples processed to assess RET translocations, 3 RET translocation-positive cell lines were also tested with PGDx elio tissue complete. All 3 cell lines were positive for a fusion either by a validated assay performed by the cell line provider, or via literature. PGDx elio tissue complete detected all 3 fusions in these cell lines.

ii. Accuracy – TMB

The PGDx elio tissue complete assay reports a TMB score comprised of sequence mutations detected across the entire coding region of interest per sample. The ability of PGDx elio tissue complete to accurately identify TMB in multiple solid tissue FFPE tumor types was assessed by comparing to matched tumor-normal whole exome sequencing results. Across 8 tumor types (non-small cell lung carcinoma (NSCLC), melanoma, renal, bladder, endometrial, triple negative breast, head and neck, lung-NOS (not otherwise specified)), 118 cases were enrolled covering a dynamic range of 1.5- 118.5 Muts/Mb. Of those, 31 fell below the established LoB $\leq 7 . 2$ Muts/Mb). The Spearman correlation coefficient was used to determine the relationship between the 2 assays. Assessment of all 118 cases resulted in a Spearman correlation coefficient of 0.903. The results in Figure 1.4 show strong concordance between PGDx elio tissue complete TMB scores and tumor-normal whole exome sequencing.

![](images/494a6109e3e8ca71ac89dcf6a47c657298553d9eb247cb811104ea7671daffb4.jpg)  
Figure 1.4 PGDx elio tissue complete TMB score vs. Matched Tumor-Normal Exome Sequencing.

iii. Accuracy – MSI

PGDx elio™ tissue complete

Table 1.16 contains MSI performance for all 283 samples including PGDx elio tissue complete and PCR failures and indeterminates. This cohort is further divided into performance for colorectal (CRC) and endometrial cases (Table 1.17) and non-CRC and non-endometrial cases (Table 1.18). MSI accuracy was assessed in 18 tumor types: ampulla (1), bladder (7), breast (21), colorectal (66), endometrial (18), esophagus (1), fallopian tube (1), gall bladder (1), gastric (40), lung (39), kidney (3), omentum (1), ovarian (2), prostate (4), sarcoma (3), skin (8), thyroid (2), and cancer of unknown primary (5).

Table 1.16 MSI Performance for All Cases   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>MSI PCR</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>183</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>283</td></tr><tr><td></td><td></td><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>Excluding failed/indeterminatespecimens with 95%CI</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=4>98.8% (79/80) (93.3%, 99.8%)</td></tr><tr><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=4>99.3% (142/143) (96.1%, 99.9%)</td></tr><tr><td rowspan=1 colspan=2>PPV</td><td rowspan=1 colspan=4>98.8% (79/80) (93.3%, 99.8%)</td></tr><tr><td rowspan=1 colspan=2>NPV</td><td rowspan=1 colspan=4>99.3% (142/143) (96.1%, 99.9%)</td></tr><tr><td rowspan=4 colspan=1>Accounting forfailed/ indeterminatespecimens with 95%CI</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=4>94.0% (79/84) (86.8%, 97.4%)</td></tr><tr><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=4>77.6% (142/183) (71.0%, 83.0%)</td></tr><tr><td rowspan=1 colspan=2>PPV</td><td rowspan=1 colspan=4>97.5% (79/81) (91.4%, 99.3%)</td></tr><tr><td rowspan=1 colspan=2>NPV</td><td rowspan=1 colspan=4>95.9% (142/148) (91.4%, 98.1%)</td></tr></table>

1This case was MSI-H by PGDx elio, and Promega PCR gave an “Indeterminate” resu 2These 5 cases did not have matching normal DNA to test via Promega PCR.

Table 1.17 MSI Performance for CRC and Endometrial Cases   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>MSI PCR</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>Excluding failed/indeterminate</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=4>100% (51/51) (93.0%, 100%)</td></tr><tr><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=4>100% (33/33) (89.6%, 100%)</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=2 colspan=1>specimens with 95%CI</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>100% (51/51) (93.0%, 100%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>100% (33/33) (89.6%,100%)</td></tr><tr><td rowspan=4 colspan=1>Accounting forfailed/indeterminatespecimens with 95%CI</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>100% (51/51) (93.0%,100%)</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>100% (33/33) (89.6%,100%</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>100% (51/51) (93.0%, 100%)</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>100% (33/33) (89.6%, 100%</td></tr></table>

Table 1.18 MSI Performance for Non-CRC and Non-Endometrial Cases   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>MSI PCR</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>Indeterminate</td></tr><tr><td rowspan=3 colspan=1>PGDx elio tissuecomplete</td><td rowspan=1 colspan=1>MSI</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>52$</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>Failed</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>199</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>Excluding failed/indeterminatespecimens with 95%CI</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=4>96.6% (28/29) (82.8%, 99.4%)</td></tr><tr><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=4>99.1% (109/110) (95.0%, 99.8%)</td></tr><tr><td rowspan=1 colspan=2>PPV</td><td rowspan=1 colspan=4>96.6% (28/29) (82.8%, 99.4%)</td></tr><tr><td rowspan=1 colspan=2>NPV</td><td rowspan=1 colspan=4>99.1% (109/110) (95.0%, 99.8%)</td></tr><tr><td rowspan=4 colspan=1>Accounting forfailed/ indeterminatespecimens with 95%CI</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=4>84.8% (28/33) (69.1%, 93.4%)</td></tr><tr><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=4>72.7% (109/150) (65.0%, 79.2%)</td></tr><tr><td rowspan=1 colspan=2>PPV</td><td rowspan=1 colspan=4>93.3% (28/30) (78.7%, 98.2%)</td></tr><tr><td rowspan=1 colspan=2>NPV</td><td rowspan=1 colspan=4>94.8% (109/115) (89.1%, 97.6%)</td></tr></table>

1This case was MSI-H by PGDx elio, and Promega PCR gave an “Indeterminate” result. 2These 5 cases did not have matching normal DNA to test via Promega PCR. iv. Method Comparison Study for Wild Type Calls

A study was conducted to assess accuracy for 75 hotspot loci within 20 genes. A total of 112 specimens were tested, and the accuracy of PGDx elio tissue complete at all 75 positions was compared to 2 orthogonal methods (42 samples using 1 method, and 70 using a second method). Within the 112 specimens, there were 112 mutations across samples and 8,283 wild type calls. Overall variant-level concordance (PPA and NPA) was $9 6 . 4 \%$ and $9 9 . 9 \%$ respectively with two-sided $9 5 \%$ confidence intervals of $( 9 1 . 1 \%$ , $9 9 . 0 \%$ ) for mutations (PPA), and $9 9 . 9 \%$ , $9 9 . 9 \%$ ) for wild type locations (NPA).

# 18. Reproducibility

# i. Interlaboratory Reproducibility

Interlaboratory reproducibility of the PGDx elio tissue complete assay was assessed across 3 different sites, using DNA extracted from 13 FFPE tissue specimens and 1 cell line. Together these 14 samples represented a range of SNVs and indels, ERBB2 amplifications, ALK, RET, and NTRK3 translocations, MSI, and TMB. Each of the 14 samples was tested in duplicate by 2 different operators on 12 sequencing runs across 3 non-consecutive days at each of the 3 independent laboratory sites using a single kit lot. The first pass rate was $9 0 . 3 \%$ (455/504) and the overall pass rate of the study was $9 8 . 2 \%$ (495/504) allowing a maximum of 1 round of repeat testing. Reproducibility was assessed 3 ways; (1) agreement for each positive variant detected across all replicates is reported (Positive call rate), and (2) Average Positive Agreement (APA) and Average Negative Agreement (ANA) and (3) modal analysis was used for per specimen reproducibility.

The positive call rate across all variants was $8 6 . 2 \%$ , while for SNVs, insertions, and deletions it was $8 8 . 8 \%$ , $8 2 . 8 \%$ , and $8 0 . 5 \%$ , respectively. Table 1.19 shows the positive call rate stratified by variant type and allele fraction.

Table 1.19 Interlaboratory Reproducibility Call Rates   

<table><tr><td rowspan=1 colspan=1>MutationType</td><td rowspan=1 colspan=1>MAFThreshold</td><td rowspan=1 colspan=1>Positive Call Rate AmongAll Observed Mutations</td><td rowspan=1 colspan=1>Total UniqueVariants</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>MAF≥0</td><td rowspan=1 colspan=1>86.2% (14493/16813)</td><td rowspan=1 colspan=1>474</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>MAF≥5</td><td rowspan=1 colspan=1>88.0%(14483/16458)</td><td rowspan=1 colspan=1>464</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>MAF≥8</td><td rowspan=1 colspan=1>91.9%(13921/15146)</td><td rowspan=1 colspan=1>427</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>MAF≥10</td><td rowspan=1 colspan=1>93.1%(13404/14400)</td><td rowspan=1 colspan=1>406</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>MAF≥15</td><td rowspan=1 colspan=1>96.4%(12387/12846)</td><td rowspan=1 colspan=1>362</td></tr><tr><td rowspan=1 colspan=1>All SNVs</td><td rowspan=1 colspan=1>MAF≥0</td><td rowspan=1 colspan=1>88.4%(10549/11937)</td><td rowspan=1 colspan=1>337</td></tr><tr><td rowspan=1 colspan=1>All SNVs</td><td rowspan=1 colspan=1>MAF≥5</td><td rowspan=1 colspan=1>91.0%(10539/11582)</td><td rowspan=1 colspan=1>327</td></tr><tr><td rowspan=1 colspan=1>All SNVs</td><td rowspan=1 colspan=1>MAF≥8</td><td rowspan=1 colspan=1>95.7%(10070/10519)</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1>All SNVs</td><td rowspan=1 colspan=1>MAF≥10</td><td rowspan=1 colspan=1>97.7%(9618/9845)</td><td rowspan=1 colspan=1>278</td></tr><tr><td rowspan=1 colspan=1>All SNVs</td><td rowspan=1 colspan=1>MAF≥15</td><td rowspan=1 colspan=1>97.8%(8773/8966)</td><td rowspan=1 colspan=1>253</td></tr><tr><td rowspan=1 colspan=1>All Insertions</td><td rowspan=1 colspan=1>MAF≥0</td><td rowspan=1 colspan=1>82.8%b (649/784)</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>All Insertions</td><td rowspan=1 colspan=1>MAF≥5</td><td rowspan=1 colspan=1>82.8%(649/784)</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>All Insertions</td><td rowspan=1 colspan=1>MAF≥8</td><td rowspan=1 colspan=1>86.9%(619/712)</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>All Insertions</td><td rowspan=1 colspan=1>MAF≥10</td><td rowspan=1 colspan=1>86.9%(619/712)</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>All Insertions</td><td rowspan=1 colspan=1>MAF≥15</td><td rowspan=1 colspan=1>95.9%(614/640)</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>All Deletions</td><td rowspan=1 colspan=1>MAF≥0</td><td rowspan=1 colspan=1>80.5%(3295/4092)</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>All Deletions</td><td rowspan=1 colspan=1>MAF≥5</td><td rowspan=1 colspan=1>80.5%(3295/4092)</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>All Deletions</td><td rowspan=1 colspan=1>MAF≥8</td><td rowspan=1 colspan=1>82.6%b (3232/3915)</td><td rowspan=1 colspan=1>110</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>MutationType</td><td rowspan=1 colspan=1>MAFThreshold</td><td rowspan=1 colspan=1>Positive Call Rate AmongAll Observed Mutations</td><td rowspan=1 colspan=1>Total UniqueVariants</td></tr><tr><td rowspan=1 colspan=1>All Deletions</td><td rowspan=1 colspan=1>MAF≥10</td><td rowspan=1 colspan=1>82.4%(3167/3843)</td><td rowspan=1 colspan=1>108</td></tr><tr><td rowspan=1 colspan=1>All Deletions</td><td rowspan=1 colspan=1>MAF≥15</td><td rowspan=1 colspan=1>92.6%(3000/3240)</td><td rowspan=1 colspan=1>91</td></tr></table>

Agreement was also assessed by evaluating the APA and ANA, which assess the degree of agreement for variants based on the average result (Table 1.20) APA and ANA across all 3 sites were $> 9 2 \%$ for all variant types tested.

Reproducibility was also assessed for independent variables (site, operator, day and within-run). No differences across sources of imprecision were observed.

Table 1.20 Interlaboratory Reproducibility Performance   

<table><tr><td rowspan=1 colspan=1>Alteration Type</td><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Overall (CI)</td><td rowspan=1 colspan=1>Alteration Type</td><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Overall (CI)</td></tr><tr><td rowspan=2 colspan=1>MSI</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>99.1% (98.7%,99.4%)</td><td rowspan=2 colspan=1>ERBB2amplifications</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>100% (99.3%,100%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.3% (99.0%,99.5%)</td><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>100% (100%,100%)</td></tr><tr><td rowspan=2 colspan=1>SNVs</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>97.8% (97.7%,97.9%)</td><td rowspan=2 colspan=1>ALKtranslocations</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>98.6% (97.7%,99.1%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.8% (99.6%,99.9%)</td></tr><tr><td rowspan=2 colspan=1>Insertions</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>95.6% (95.2%,96.0%)</td><td rowspan=1 colspan=1>NTRK3translocations</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>92.7% (90.4%,94.5%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.4% (99.2%,99.5%)</td></tr><tr><td rowspan=2 colspan=1>Deletions</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>94.4% (94.2%,94.6%)</td><td rowspan=1 colspan=1>RETtranslocations</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>98.7% (97.8%,99.2%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.8% (99.6%,99.9%)</td></tr><tr><td rowspan=1 colspan=1>TMB</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=3></td></tr></table>

The modal positive and negative call rates for sequence mutations (SNVs and indels) in each specimen are summarized in Table 1.21.

Table 1.21 Modal Call Rates for Interlaboratory Reproducibility   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Total UniqueMutationsDetectedAcross AllReplicates</td><td rowspan=1 colspan=1>Modal Positive Call Rate1 (n/N)(two-sided 95% CI)</td><td rowspan=1 colspan=1>Modal Negative Call Rate2(n/N) (two-sided 95% CI)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>99.6% (251/252) (97.8%,99.9%)</td><td rowspan=1 colspan=1>97.2% (105/108) (92.2%,99.1%)</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100% (315/315) (98.8%,,100%)</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100% (216/216) (98.3%, ,100%)</td><td rowspan=1 colspan=1>97.2% (35/36) (85.8%,99.5%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>99.5% (1462/1470) (98.9%,99.7%)</td><td rowspan=1 colspan=1>91.4% (32/35) (77.6%,97.0%)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>98.9%6 (639/646) (97.8%,99.5%)</td><td rowspan=1 colspan=1>88.2% (30/34) (73.4%,95.3%)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>97.8% (678/693) (96.5%,98.7%)</td><td rowspan=1 colspan=1>95.2% (157/165) (90.7%,97.5%)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>96.4% (1991/2065) (95.5%,97.1%)</td><td rowspan=1 colspan=1>88.7% (683/770) (86.3%,90.8%)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>97.8% (2710/2772) (97.1%,98.3%)</td><td rowspan=1 colspan=1>80.3% (318/396) (76.1%,83.9%)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>99.0% (998/1008) (98.2%,99.5%)</td><td rowspan=1 colspan=1>97.2% )70/72) (90.4%,99.2%)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>96.1% (2907/3024) (95.4%,96.8%)</td><td rowspan=1 colspan=1>83.1% (299/360) (78.8%,86.6%)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>99.4% (1109/1116) (98.7%,99.7%)</td><td rowspan=1 colspan=1>97.2% (70/72) (90.4%,99.2%)</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100% (216/216) (98.3%, 100%)</td><td rowspan=1 colspan=1>93.5% (101/108) (87.2%,96.8%)</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>96.8% (732/756) (95.3%,97.9%)</td><td rowspan=1 colspan=1>88.0% (95/108) (80.5%,92.8%)</td></tr></table>

1Positive call rate was calculated based on variants with majority call detected as positive. 2 Negative call rate was calculated based on variants detected at least once, but with majority or equal call as negative. For all other locations, the negative call rates are $100 \%$ . 3 Specimen 2 was selected for presence of ALK translocation and had no detected SNVs or indels.

Precision of MSI was evaluated across 8 MSS and 6 MSI-H samples with a range of MSI scores evaluated by PGDx elio tissue complete with positive call rates provided in Table 1.22.

Table 1.22 MSI Performance in the Interlaboratory Reproducibility Study   

<table><tr><td rowspan=1 colspan=1>CaseNo.</td><td rowspan=1 colspan=1>ModalStatus</td><td rowspan=1 colspan=1>TotalReplicates</td><td rowspan=1 colspan=1>Mean MSIScoreValue</td><td rowspan=1 colspan=1>MSI ScoreRange</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Positive Call Rate(95% CI)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>(4.4, 19.1)</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>(5.8, 20.4)</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>100% (90.1%,100%</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=1 colspan=1>CaseNo.</td><td rowspan=1 colspan=1>ModalStatus</td><td rowspan=1 colspan=1>TotalReplicates</td><td rowspan=1 colspan=1>Mean MSIScoreValue</td><td rowspan=1 colspan=1>MSI ScoreRange</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Positive Call Rate(95% CI)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>(7.2, 20.2)</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>100% (90.1%,100%)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>(3.9, 19.8)</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>209.7</td><td rowspan=1 colspan=1>(203.7, 216.5)</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>100% (90.1%,100%)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>(4.7, 16.0)</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>100% (89.9%,100%)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>-21.9</td><td rowspan=1 colspan=1>(-29.9, -13.2)</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>-22.8</td><td rowspan=1 colspan=1>100% (89.6%,100%)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>223.5</td><td rowspan=1 colspan=1>(213.1, 236.3)</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>100% (90.1%,100%)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>271.6</td><td rowspan=1 colspan=1>(261.6, 287.2)</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>77.5</td><td rowspan=1 colspan=1>(62.6, 102.5)</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>219.0</td><td rowspan=1 colspan=1>(212.6, 224.0)</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>-56.1</td><td rowspan=1 colspan=1>(-61.5, -41.5)</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>-7.3</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>(10.4, 25.1)</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>100% (90.4%,100%)</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>49.3</td><td rowspan=1 colspan=1>(36.7, 61.7)</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>94.4% (81.9%,98.5%)</td></tr></table>

Precision of TMB was evaluated across 11 samples (with TMB scores above TMB LoB of 7.2 Muts/Mb) with a range of TMB across site, operator, and day provided in Figure 1.7.

![](images/3d72ba68c310ba46d6c51ab5604ea4b75c489b0da709aa800206d8dc05bfd8d8.jpg)  
Figure 1.5 TMB Performance in the Interlaboratory Reproducibility Study by Site, Operator, and Day

Lot to Lot Precision

Performance of PGDx elio tissue complete was assessed across 3 unique kit lots by determining concordance of variant calls in FFPE tissue samples. The 3 unique kit lots were utilized to process 5 test cases in triplicate for a total of 45 observations. All batches were sequenced on the same instrument. The overall pass rate of the study was $100 \%$ (45/45). Table 1.23 lists the APA and ANA used to assess lot to lot performance. APA for all variants is $> 8 6 \%$ , and $\% \mathrm { C V }$ for TMB analyses is $< 1 0 \%$ .

Table 1.23 Lot to Lot Precision   

<table><tr><td rowspan=1 colspan=1>Variant Type</td><td rowspan=1 colspan=1>Performance</td><td rowspan=1 colspan=1>Between Lot 1 &amp; 2</td><td rowspan=1 colspan=1>Between Lot 1 &amp; 3</td><td rowspan=1 colspan=1>Between Lot 2 &amp; 3</td></tr><tr><td rowspan=2 colspan=1>Variants withEvidence ofClinicalSignificance</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>98.7% (93%, 99.8%)</td><td rowspan=1 colspan=1>96.1% (89.2%,98.7%)</td><td rowspan=1 colspan=1>97.4% (91.1%,99.3%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.6%, 100%)</td><td rowspan=1 colspan=1>99.8% (99.4%,99.9%)</td><td rowspan=1 colspan=1>99.9% (99.5%,100%)</td></tr><tr><td rowspan=2 colspan=1>MSI</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>100% (75.8%, 100%)</td><td rowspan=1 colspan=1>100% (75.8%, 100%)</td><td rowspan=1 colspan=1>100% (75.8%, 100%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>100% (82.4%, 100%)</td><td rowspan=1 colspan=1>100% (82.4%, 100%)</td><td rowspan=1 colspan=1>100% (82.4%, 100%)</td></tr></table>

PGDx elio™ tissue complete

<table><tr><td rowspan=2 colspan=1>SNVs</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>92.1% (90.8%,93.2%)</td><td rowspan=1 colspan=1>91.9% (90.6%,93.0%)</td><td rowspan=1 colspan=1>91.9% (90.7%,93.0%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%, 100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td></tr><tr><td rowspan=2 colspan=1>Insertions</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>88.9% (80.2%,94.0%)</td><td rowspan=1 colspan=1>88.9% (80.2%,94.0%)</td><td rowspan=1 colspan=1>87.2% (78%, 92.9%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%, 100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td></tr><tr><td rowspan=2 colspan=1>Deletions</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>86.2% (82.6%,89.1%)</td><td rowspan=1 colspan=1>89.8% (86.7%,92.2%)</td><td rowspan=1 colspan=1>87.3% (83.9%,90.0%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>99.9% (99.9%, 100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td><td rowspan=1 colspan=1>99.9% (99.9%,100%)</td></tr><tr><td rowspan=2 colspan=1>ERBB2amplification</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>100% (86.2%,100%)</td><td rowspan=1 colspan=1>100% (86.2%, 100%)</td><td rowspan=1 colspan=1>100% (86.2%, 100%)</td></tr><tr><td rowspan=2 colspan=1>ALKtranslocation</td><td rowspan=1 colspan=1>APA</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td><td rowspan=1 colspan=1>100% (61.0%, 100%)</td></tr><tr><td rowspan=1 colspan=1>ANA</td><td rowspan=1 colspan=1>100% (96.7%, 100%)</td><td rowspan=1 colspan=1>100% (96.7%, 100%)</td><td rowspan=1 colspan=1>100% (96.7%, 100%)</td></tr><tr><td rowspan=1 colspan=1>TMB</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>7.9%</td><td rowspan=1 colspan=1>7.1%</td></tr></table>

# H. Conclusions

The submitted information in this premarket notification the nonclinical and clinical tests that demonstrate that the device is as safe, as effective, and performs substantially equivalent to the legally marketed predicated device and supports a substantial equivalence decision