# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k062028   
B. Purpose for Submission: New Device   
C. Measurand: Cystic Fibrosis   
D. Type of Test: Multiplex PCR followed by capillary electrophoresis and OLA reflex testing.   
E. Applicant: Celera Diagnostics   
F. Proprietary and Established Names: Cystic Fibrosis Genotyping Assay   
G. Regulatory Information: 1. Regulation section: 21 CFR 866.5900, CFTR (cystic fibrosis transmembrane conductance regulator) gene mutation detection system 2. Classification: Class II 3. Product code: NUA, System, test, CFTR (cystic fibrosis transmembrane conductance regulator) gene mutation detection system 4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): The Cystic Fibrosis Genotyping Assay is a qualitative in vitro diagnostic device used to genotype a panel of mutations and variants in the cystic fibrosis transmembrane conductance regulator (CFTR) gene in genomic deoxyribonucleic acid (DNA) isolated from human whole blood specimens. The panel includes mutations and variants recommended by the American College of Medical Genetics (ACMG, 2004) and the American College of Obstetricians and Gynecologists (ACOG, 2005) plus additional multiethnic mutations. The Cystic Fibrosis Genotyping Assay provides information intended to be used for carrier testing in adults of reproductive age, as an aid in newborn screening and in confirmatory diagnostic testing of newborns and children. This test is not indicated for use in fetal diagnostic or pre-implantation testing. This test is also not indicated for stand-alone diagnostic purposes.

2. Indication(s) for use: Same as Intended Use   
3. Special conditions for use statement(s): For Prescription Use only   
4. Special instrument requirements: CEGA- $. 1 6 ^ { \mathrm { T M } }$ Instrument System

I. Device Description: The Celera Cystic Fibrosis Genotyping Assay (CF GT) system is comprised of three reagent modules, the CEGA- $1 6 ^ { \mathbf { T M } }$ Instrument, computer workstation, and CF Configuration Disk v1.0 (GeneMapper $\textsuperscript { \textregistered }$ Software v3.5.3 Upgrade File - CF Analysis Settings and Parameters). The three reagent modules consist of:

CF Setup: Two tubes each of CF Sample Diluent and CFTR Wild Type control. CF PCR: One tube of CF PCR Reagent containing oligonucleotides, DNA polymerase, deoxy-nucleotides, and uracil N-glycosylase. CF OLA: One tube each of OLA enzyme solution, OLA ROX size standards, synthetic oligonucleotide probes for the 30 normal and 32 mutant CFTR alleles (CF OLA Core); synthetic oligonucleotide probes for the 5T, 7T, and 9T variants of intron 8 (IVS-8) (CF 5/7/9T Reflex OLA); and synthetic oligonucleotide probes for the I506V, I507V, and F508C polymorphisms in exon 10 (CF Exon 10 Reflex OLA).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Tag-It™ Cystic Fibrosis Kit   
2. Predicate 510(k) number(s): k043011   
3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Cystic Fibrosis GenotypingAssay is a qualitative in vitrodiagnostic device used to genotypea panel of mutations and variants inthe cystic fibrosis transmembraneconductance regulator (CFTR) genein genomic deoxyribonucleic acid(DNA) isolated from human wholeblood specimens. The panelincludes mutations and variantsrecommended by the AmericanCollege of Medical Genetics(ACMG, 2004) and the AmericanCollege of Obstetricians andGynecologists (ACOG, 2005) plusadditional multiethnic mutations.The Cystic Fibrosis GenotypingAssay provides informationintended to be used for carriertesting in adults of reproductiveage, as an aid in newborn screeningand in confirmatory diagnostictesting in newborns and children.</td><td colspan="1" rowspan="1">The Tag-IT Cystic FibrosisKit is a device used tosimultaneously detect andidentify a panel ofmutations and variants inthe cystic fibrosistransmembraneconductance regulator(CFTR) gene in humanblood specimens. The panelincludes mutations andvariants currentlyrecommended by theAmerican College ofMedical Genetics andAmerican College ofObstetricians andGynecologists(ACMG/ACOG), plussome of the worlds mostcommon and NorthAmerican-prevalentmutations. The Tag-ItCystic Fibrosis Kit is aqualitative genotyping testwhich provides informationintended to be used forcarrier testing in adults of</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">reproductive age, as an aidin newborn screening, andin confirmatory diagnostictesting in newborns andchildren.</td></tr><tr><td colspan="1" rowspan="1">Contra-indications</td><td colspan="1" rowspan="1">This device is not intended for usein fetal diagnostic or pre-implantation testing. This test isalso not indicated for stand-alonediagnostic purposes.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Genomic DNA (gDNA) fromhuman whole blood</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Mutations andpolymorphismsdetected</td><td rowspan=1 colspan=1>32 mutations (23 ACMG + 9additional)/4 polymorphisms</td><td rowspan=1 colspan=1>39 mutations (23 ACMG + 16additional)/4 polymorphisms</td></tr><tr><td rowspan=1 colspan=1>Test methodology</td><td rowspan=1 colspan=1>Multiplex PCR followed byDNA sequencing and OLAreflex testing</td><td rowspan=1 colspan=1>Multiplex PCR followed bymultiplex allele specificprimer extension forgenotyping, hybridized tomultiplexed fluorescingmicroparticles, detected byflow cytometry</td></tr><tr><td rowspan=1 colspan=1>InstrumentSystems</td><td rowspan=1 colspan=1>CEGA-16TM Instrument System</td><td rowspan=1 colspan=1>Luminex 100 IS (IntegratedSystem)</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>CEGA-16TM InstrumentSoftware with supplied settingsand parameters</td><td rowspan=1 colspan=1>Tag-IT Data AnalysisSoftware TDAS CF-I</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Negative:CF Sample DiluentPositive (amplification):CFTR Wild Type ControlPositive (mutation):States user can use acommercially obtained or in-house developed sample canbe used as a mutation positivecontrol</td><td rowspan=1 colspan=1>Negative:ddH2O ControlPositive (amplification &amp;mutation):Recommend user usegDNA similar to sampletype with ΔF508 mutation</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

American College of Medical Genetics (ACMG) / American College of Obstetricians and Gynecologists (ACOG) • 2001, 2002, 2004 ACMG Technical Standards and Guidelines for CFTR mutation testing.

• ACMG 2004 Standards and Guidelines for Clinical Genetics Laboratories   
DA Guidances • Class II Special Controls Guidance Document - CFTR Gene Mutation Detection Systems Medical Device Use-Safety: Incorporating Human Factors Engineering into Risk Management Guidance for Industry - Cybersecurity for Networked Medical Devices Containing Off-the-Shelf (OTS) Software Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices - Guidance for Industry and FDA Staff Guidance for Off-the-Shelf Software Use in Medical Devices; Final Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Draft Guidance for Industry and FDA Reviewers

# CLSI Guidelines

• Molecular Diagnostic Methods for Genetic Diseases; Approved Guideline (MM1- A) • Protocol for Evaluation of Qualitative Test Performance; Approved Guideline (EP12- A) Interference Testing in Clinical Chemistry; Approved Guideline (EP 7-A)

ISO Guidelines • Medical devices - Application of risk management to medical devices (14971:2000) • Stability Testing of In Vitro Diagnostic Reagents (13640)

# L. Test Principle:

The Celera CF GT Assay is based on five major processes: sample preparation, polymerase chain reaction (PCR), oligonucleotide ligation assay (OLA), detection and electrophoretic analysis of OLA fragments, data analysis. It uses DNA fragment analysis for determining the carrier status of a defined panel of CF mutations. Purified gDNA is prepared from whole blood by standard purification methods. The gDNA is amplified in a multiplex PCR reaction with 16 pairs of PCR primers and DNA polymerase followed by an OLA). Allele-specific OLA probes hybridize to the respective normal and become ligated with fluorescent-labeled common probes upon successful annealing by the ligase enzyme. The OLA probes are varied in length due to the addition of inert mobility modifiers. The ligated, fluorescentlabeled DNA fragments are separated on the CEGA- $1 6 ^ { \mathrm { T M } }$ Instrument, by capillary electrophoresis. Detection is based on size and fluorescent label. The ligation products are then identified and genotyped by analysis with the GeneMapper $^ \mathrm { \textregistered }$ Software v3.5.3.

The CF GT Assay kit also contains Reflex OLA reagents for the detection of the polythymidine (Poly-T) 5/7/9T variants in IVS-8 of the CFTR gene and for the detection of the I506V, I507V, and F508C polymorphisms in Exon 10 of the CFTR gene. The CFTR R117H mutation, along with 5T of the 5/7/9T variant in IVS-8 on the same allele (in cis), has been shown to be associated with a variable phenotype when associated with another CF mutation. As a result, reflex testing for the 5T variant with the CF 5/7/9T Reflex OLA assay is recommended when the R117H mutation is detected in the core assay. The CF Exon 10 Reflex OLA assay is used to verify a homozygous deletion of the F508 or I507 codon and to exclude a potential false-positive result due to interference by certain non-CF causing polymorphisms at codons 506, 507, and 508. The CF Exon 10 Reflex OLA assay will distinguish between a true homozygous delF508 or delI507 from a sample containing one delF508 or delI507 allele plus the benign polymorphism of I506V, I507V or F508C. The same software contained on the configuration disk is used to report reflex testing genotyping

information.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: A single-blinded study was conducted at three external test sites. Each test site had one set of instruments, two operators, and three lots of reagents. One lot of reagents was used each day for three days. Each site also processed the same set of samples. The samples tested included gDNA obtained from frozen whole blood and gDNA from commercially available frozen cell line pellets. The expected genotype call for every unique sample was confirmed by DNA bi-directional sequencing. The 144 reproducibility study samples consisted of three (3) replicates each of forty-eight (48) unique gDNA samples derived from frozen whole blood $( \mathtt { n } = 2 5 )$ ), and frozen cell line pellets $( \mathtt { n } = 2 3$ ). The frozen whole blood samples contained homozygous normal alleles for each of the targeted loci within the Core OLA Assay, whereas the frozen cell line specimens contained either a single heterozygous mutation, a single homozygous mutation, or compound heterozygous mutations for CF. Each of three test sites received sufficient aliquots of each test panel member to conduct three independent assays (3 assays $\textbf { x } 1 4 4$ samples/assay $= 4 3 2$ results per site). Each assay was performed with a different reagent lot $( \mathtt { n } = 3 )$ ) on a different day. There were a total of 1,296 test results for each mutation (48 samples $\mathbf { \Delta } \mathbf { X } \ 2 7$ replicates). Reflex testing was performed as indicated per results of the Core OLA Assay and in accordance with ACMG guidelines.

# Reproducibility Results

Statistical analysis of all 1,296 results within the sample set indicates $100 \%$ agreement between the Core OLA Assay and sequencing, with a one-sided $9 5 \%$ lower confidence limit equal to $9 9 . 8 \%$ . Based on results of the Core OLA Assay, two samples required reflex testing. One sample had a homozygous delF508 call by the Core OLA Assay. Both sequencing and Exon 10 reflex testing for all replicates $\mathbf { \tilde { n } = }$ 9) confirmed the homozygous delF508 call. The Core OLA Assay result for the other sample contained an R117H mutation, therefore, reflex testing for variants in the Poly-T tract on IVS-8 (i.e., 5/7/9T) was conducted. Specifically, the $5 \mathrm { T }$ and 9T variant alleles were identified in this sample. Both sequencing and reflex testing results were in $100 \%$ agreement for all calls.

Results of the study indicate that the CF GT Assay has a reproducibility of $100 \%$ , with zero (0) missed calls, as shown in the Reproducibility Table below.

Table 1. Reproducibility of the CF GT assay study $( > 9 9 . 1 \%$ : site-to-site, lot-to-lot, operator-to-operator).   

<table><tr><td colspan="1" rowspan="1">Mutation(s)</td><td colspan="1" rowspan="1"># SampleReplicatesper site</td><td colspan="1" rowspan="1">Total #SampleReplicatesPer Sitea</td><td colspan="1" rowspan="1"># AssayCalls PerEach Siteb</td><td colspan="1" rowspan="1"># AssayCalls Per   MissedEach      Callsdmutation</td></tr><tr><td colspan="1" rowspan="1">delF508 (homozygous)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">279e</td><td colspan="1" rowspan="1">432         0</td></tr><tr><td colspan="1" rowspan="1">3120+1G&gt;A; 621+1G&gt;T</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">delF508; R553X</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">3659delC; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">delI507</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">Mutation(s)</td><td colspan="1" rowspan="1"># SampleReplicatesper site</td><td colspan="1" rowspan="1">Total #SampleReplicatesPer Sitea</td><td colspan="1" rowspan="1"># AssayCalls PerEach Siteb</td><td colspan="1" rowspan="1"># AssayCalls Per    MissedEach      Callsdmutation</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T; 711+1G&gt;T</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T; G85E</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">A455E; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">delF508; R560T</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">G542X; G542X</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A; 2789+5G&gt;A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">3849+10kb C&gt;T (homozygous)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;A</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">G551D; R347P</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">R117H; 5T/9T; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">288</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">2184delA; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">1898 +1G&gt;A; delF508</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">432          0</td></tr><tr><td colspan="1" rowspan="1">WT (wild type)</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">225</td><td colspan="1" rowspan="1">6750</td><td colspan="1" rowspan="1">432          0</td></tr></table>

a. 3 replicate x 3 assay lots b. 30 loci calls per sample $_ { \textrm { X 3 } }$ replicates $_ { \textrm { X 3 } }$ assay lots c. 48 samples x 3 replicates/assay3 assays d. Missed Calls are defined as discrepant results relative to the sequencing result. e. An additional 9 calls per site (1 call/sample $_ { \textrm { X 3 } }$ replicates $_ { \textrm { X 3 } }$ lots) were generated as a result of Exon 10 reflex testing. f. An additional 18 calls per site (2 calls/sample $_ { \textrm { X 3 } }$ replicates $_ { \textrm { X 3 } }$ lots) were generated as a result of 5/7/9T reflex testing. g. 25 individual WT samples were used $2 5 \times 3 { = } 7 5$ ).

Exon 10 reflex testing detected all polymorphisms as “present” or “not present,” and 5/7/9T reflex testing detected all variants as “present” or “not present,” with a reproducibility of $100 \%$ . These results are limited to two samples and 81 test results for each sample (27 sample replicates $_ { \textrm { X 3 } }$ tests each). The following table summarizes the precision results for the two reflex assays.

Table 2. Reproducibility of Exon 10 and 5/7/9T Reflex Testing.   

<table><tr><td rowspan="2">Reflex Assay</td><td colspan="3">Percent Correct Calls Made by Exon 10 or 5/7/9T Reflex Testing (%)</td></tr><tr><td>Site to Site 3 Sites, n = 27 results per site</td><td>Operator to Operator 6 Operators, (n = 9 results for 3 operators, n = 18 results, for 3 operators)</td><td>Lot to Lot 3 Lots, n = 27 results per lot</td></tr><tr><td>5/7/9T</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Exon 10</td><td>100</td><td>100</td><td>100</td></tr></table>

An overview of the entire reproducibility study results by mutation is provided in Table 3, below.

Table 3. Reproducibility Study by the Mutation Table   

<table><tr><td rowspan=3 colspan=1>Genotype by DNASequencing</td><td rowspan=3 colspan=1># SamplesReplicatesper site(A, B, C)</td><td rowspan=1 colspan=6># CF GT Calls Before Repeat Testing (Based on InitialResults</td><td rowspan=1 colspan=4># CF GT Calls After RepeatTesting (30 calls per samplex 3 replicates x 3 lots)</td><td rowspan=3 colspan=1>Total #CorrectCalls</td><td rowspan=3 colspan=1>Agree-ment(%)</td></tr><tr><td rowspan=1 colspan=3>Correct Calls</td><td rowspan=2 colspan=1>DelayedCallsa</td><td rowspan=2 colspan=1>AgreementInitiaaReesults)</td><td rowspan=2 colspan=1>Repeatsc</td><td rowspan=1 colspan=3>Correct Calls</td><td rowspan=2 colspan=1>Missed(Incorrect)Calls</td></tr><tr><td rowspan=1 colspan=1>SiteA</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>SiteA</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2184delA/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A/2789+5G&gt;A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A/621+1G&gt;T</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3659delC/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3849+10kbC&gt;T (homo)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T/711+1G&gt;T</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T/G85E</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>A455E /delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>delF508/delF508d</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>249d}$</td><td rowspan=1 colspan=1>276</td><td rowspan=1 colspan=1>159d</td><td rowspan=1 colspan=1>60d</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>93$</td><td rowspan=1 colspan=1>279d</td><td rowspan=1 colspan=1>279$</td><td rowspan=1 colspan=1>279$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>837d</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>del1507</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G542X/G542X</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G551D/R347P</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R117H (5T/9T)/delF508d</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>258$</td><td rowspan=1 colspan=1>282$</td><td rowspan=1 colspan=1>138$</td><td rowspan=1 colspan=1>90}$</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>96}$</td><td rowspan=1 colspan=1>288d</td><td rowspan=1 colspan=1>288}$</td><td rowspan=1 colspan=1>288d$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>864}$</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R553X/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R560T/delF508</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>270</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Wild Type</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>5,430</td><td rowspan=1 colspan=1>6,660</td><td rowspan=1 colspan=1>3,420</td><td rowspan=1 colspan=1>2,820</td><td rowspan=1 colspan=1>90.5</td><td rowspan=1 colspan=1>1,920</td><td rowspan=1 colspan=1>6,750</td><td rowspan=1 colspan=1>6,750</td><td rowspan=1 colspan=1>6,750</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20,250</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Overall Testing</td><td rowspan=1 colspan=1>432</td><td rowspan=1 colspan=1>11,007</td><td rowspan=1 colspan=1>12,858</td><td rowspan=1 colspan=1>6,957</td><td rowspan=1 colspan=1>4,230</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>3,909</td><td rowspan=1 colspan=3>38,961</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38,961</td><td rowspan=1 colspan=1>100</td></tr></table>

a. Re-injection of same OLA reaction. Represents cases where users chose to reinject individual samples as well as entire or partial plates (usually based on convenience). In cases where plates were reinjected, initial, correct genotype calls did not change with repeat testing.   
b. Results reflect percent agreement with sequencing prior to repeat testing.   
c. Repeat from DNA/PCR. Repeat testing was performed due to a poor reaction or PCR failure, GeneMapper quality flags indicating questionable or invalid data.   
d. Reflex testing included. Exon 10: Nine additional calls per site (1 call per sample $_ { \textrm { X 3 } }$ replicates x 3 lots). 5/7/9/T: Eighteen additional calls per site (2 calls per sample $_ { \textrm { X 3 } }$ replicates $_ { \textrm { X 3 } }$ lots).   
b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): The reference method is bi-directional sequencing. Assay Controls: The assay contains positive and negative controls to assure proper test results.

d. Detection limit:

The package insert recommends that the concentration of gDNA should be between $5 { \mathrm { - } } 1 5 ~ \mathrm { { n g / \mu L } }$ . Development studies tested up to $1 0 0 ~ \mathrm { n g }$ of input DNA and no off-scale peaks were ever observed. Artifact peaks have been observed when excessive amount $( \geq 1 0 0 { \mathrm { n g } } )$ of input DNA is used however the peaks always appear outside the defined marker bins. These peaks are not labeled, and therefore do not affect the genotype call.

e. Analytical specificity:

Commonly occurring endogenous substances were tested for their potential to interfere with the performance of the CF Genotyping Assay in detecting mutations. Potential interfering substances were added to whole blood prior to sample

preparation process for purified gDNA. Results indicate that these four substances at the concentrations tested did not interfere with the CF GT Assay: hemoglobin (600 to $8 2 0 \mathrm { m g / d L } )$ , bilirubin $( 2 0 \mathrm { m g } / \mathrm { d L } )$ ), triglycerides ${ 5 0 0 } \mathrm { m g } / \mathrm { d L } )$ , and protein $( 8 ~ \mathrm { g } / \mathrm { d L } )$ .

$f .$ Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device:

All genotyping calls made by the CF GT Assay on all samples were compared to those made by bi-directional dideoxy terminal DNA sequencing. The study was conducted using 163 unique samples representing archived purified gDNA from blood (51), frozen whole blood (98), fresh whole blood (6), and commercially available frozen cell line pellets (8). Most of the samples $( 6 5 \% )$ contained a single, heterozygous core mutation. Full segmentation of the sample set is summarized in the following table:

Table 4. CF GT panel.   

<table><tr><td rowspan=1 colspan=1>Sample Segments</td><td rowspan=1 colspan=1>Number of Unique Samples</td></tr><tr><td rowspan=1 colspan=1>One Core CF Mutation</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>Homozygotes</td><td rowspan=1 colspan=1>3 (all delF508)</td></tr><tr><td rowspan=1 colspan=1>Heterozygotes</td><td rowspan=1 colspan=1>106</td></tr><tr><td rowspan=1 colspan=1>Two Core CF Mutations</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>Exon 10 Polymorphismswithout Core CF Mutation*</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Normal for all targeted loci</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Total =</td><td rowspan=1 colspan=1>163</td></tr></table>

\*Included to assess the assay’s ability to detect the polymorphism when present.

The study requirement was to a minimum of 5 samples per mutation. In order to achieve the target minimum of 5 samples per mutation, additional independent aliquots were generated from 38 of the unique samples. Thus, the complete sample set consisted of 201 samples (i.e., 163 unique $+ 3 8$ duplicates). An overview of the mutation, polymorphism, and variant distribution within each of the complete sample set is provided in Table 5.

Table 6 summarizes the CF GT Assay results from the core OLA assay relative to the sequencing results. For each sample, analysis by both methods confirmed that normal alleles were present and detected for all loci excluding the loci that contained the expected mutations.

<table><tr><td rowspan=1 colspan=11>Table 5.Mutation, Polymorphism and Variant Distribution of samples:</td></tr><tr><td rowspan=1 colspan=1>Mutations &amp;P/Vsa$</td><td rowspan=1 colspan=1>#Indepen-dentClinicalSamples</td><td rowspan=1 colspan=1>#Indepen-dent CellLinesb</td><td rowspan=1 colspan=1># UniqueSampleswithMutation</td><td rowspan=1 colspan=1>Subset with a SecondMutation</td><td rowspan=1 colspan=1>Subset withCore + P/V</td><td rowspan=1 colspan=1># UniqueSaampleswith P/Vonly</td><td rowspan=1 colspan=1>AdditionalIndependentAliquots fromthe UniqueSammple Setd</td><td rowspan=1 colspan=1>Total Sampleswith Mutation</td><td rowspan=1 colspan=1>TotalSampleswiith P/V</td><td rowspan=1 colspan=1>TotalReplicatesTTestede</td></tr><tr><td rowspan=1 colspan=1>1078delT</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2 of5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2183AA&gt;G</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1 of 1 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2184delA</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>3659delC</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (G542X))</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>3849+ 10kbC&gt;T</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>3876delA</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>3905insT</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>394delTT</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (711+1G&gt;T)1 of 5 (delF508)1 of 5 (G85E)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (621+1G&gt;T)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>A455E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=2 colspan=1>delF508</td><td rowspan=2 colspan=1>22</td><td rowspan=2 colspan=1>4</td><td rowspan=2 colspan=1>26</td><td rowspan=2 colspan=1>2 of 26 (1717-1G&gt;A)1 of 26 (2183AA&gt;G)1 of 26 (2184delA)1 of 26(2789+5G&gt;A)1 of 26 (621+1G&gt;T)1 of 26 (R117H+5T)1 of 26 (A455E)3 of 26 (G542X)2 of 26 (G551D)1 of 26 (N1303K)2 of 26 (R553X)1 of 26 (R560T)1 of 26 (S549N)1 of 26 (W1282X)</td><td rowspan=1 colspan=1>2 (I506V)1 (1507V)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>132</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>132</td></tr><tr><td rowspan=1 colspan=1>del1507</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (3659delC)3 of 5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2 of 5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (621+1G&gt;T)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 of 5 (delF508)</td><td rowspan=1 colspan=1>2 (5T/7T)3 (7T/7T)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R347H</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (R553X)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R347P</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R553X</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2 of 4 (delF508)1 of 4 (R347H)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>R560T</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 of 4 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>S549N</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1 of 3 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>S549R</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>V520F</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1 of 6 (delF508)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>5T</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2 (R117H)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>7T</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5(R117H)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>9T</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1506V</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2 (delF508)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>1507V</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1 (delF508)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>F508C</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td></tr></table>

a. $\mathrm { P } = \mathrm { E x o n } ~ 1 0$ polymorphisms (F508C, I506V, and I507V); $\mathrm { V } = \mathrm { I V S } { - } 8$ variants (5T, 7T, and 9T) b. 8 total cell lines, 6 compound heterozygotes c. Samples that contained an Exon 10 polymorphism $\mathrm { w } / \mathrm { o }$ a corresponding core mutation (of either homozygous delF508 or delI507). d. Independent aliquots of individual, unique samples that were used to achieve the target minimum of 4 samples/mutation. e. Total Replicates Tested $=$ [Total Samples with Mutation] $\mathbf { X }$ [2 replicates/sample] x [2 test sites] f. $\mathrm { { N / A } = }$ not applicable

Table 6. Method comparison of Core OLA results.   

<table><tr><td rowspan=2 colspan=1>Target Locusby Mutations</td><td rowspan=1 colspan=2>804 Sample Replicate Test Results Per CoreOLA Assay(201 Samples/Set x 2 Sets/Site x 2 Sites)</td><td rowspan=1 colspan=1>Agreement</td></tr><tr><td rowspan=1 colspan=1># Sample Replicateswith a MutationIdentified at theTarget Locus</td><td rowspan=1 colspan=1># Sample Replicateswith a Normal/WildType Allele Identifiedat the Target Locus</td><td rowspan=1 colspan=1>(%)</td></tr><tr><td rowspan=1 colspan=1>1078delT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2183AA&gt;G</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2184delA</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3659delC</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3849+10kbC&gt;T</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3876delA</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>3905insT</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>394delTT</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>A455E</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>delF508</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>792*</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>delI507</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R347H</td><td rowspan=1 colspan=1>20</td><td rowspan=2 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R347P</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R553X</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R560T</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S549N</td><td rowspan=1 colspan=1>20</td><td rowspan=2 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>S549R</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>V520F</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>100</td></tr></table>

\*Three samples that contained the delF508 mutation were homozygous mutants. Since these three samples did not contain any normal/wild type alleles at the F508 locus, the number of sample replicates with a wild type allele is lower [804 – (3 samples/set $_ { \textrm { X 2 } }$ sets/site $\textbf { x } 2$ sites) $= 7 9 2 \hat { 1 }$ ].

Analysis of results from the two reflex OLA assays indicated nearly $100 \%$ agreement with sequencing, with the exception of one sample replicate that produced a 5T/7T/9T genotype call per the 5/7/9T OLA Reflex Assay. Sequencing indicated that only the 5T and 7T alleles were present. The results from the two reflex OLA assays are summarized in Table 7:

Table 7. Concordance between CF GT assay and sequencing.   

<table><tr><td rowspan=2 colspan=1>P/Va</td><td rowspan=1 colspan=3>Test Results (Reflex OLA Assays)</td><td rowspan=2 colspan=1>Agreement(%)</td></tr><tr><td rowspan=1 colspan=1># Replicates withP/V Detected</td><td rowspan=1 colspan=1># Replicates notmeeting ReflexTesting Requirementsb</td><td rowspan=1 colspan=1># Replicates withNormal/WT Calls</td></tr><tr><td rowspan=1 colspan=1>5T</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7T</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>9T</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>$\frac{}$</td><td rowspan=1 colspan=1>80 (4/5)</td></tr><tr><td rowspan=1 colspan=1>F508C</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>I506V</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>I507V</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td></tr></table>

a. $\mathrm { P } = \mathrm { E x o n } ~ 1 0$ polymorphisms (F508C, I506V, and I507V); $\mathrm { V } = \mathrm { I V S } { - } 8$ variants (5T, 7T and 9T)   
b. These are counted as correct results since in all cases the Core OLA Assay correctly made a wild type (normal)call in the presence of heterozygous F508C, I506V or I507V polymorphism (as indicated by sequencing).Reflex testing was not performed for these samples per the clinical trial protocol, which included a reflex testing algorithm based on ACMG recommendations   
c. $\mathrm { { N / A = } }$ not applicable   
d. The 5/7/9T OLA Reflex Assay generated a 5T/7T/9T call for one replicate of one sample, which did not match the reference sequencing call (5T/7T).   
e. The Exon 10 OLA Reflex Assay is not designed to detect the normal allele F508. In the presence of deletion mutations, the I506 and I507 normal alleles are not seen by the Exon 10 OLA Reflex Assay in most cases; thus, Exon 10 results must always be analyzed with results from the Core OLA Assay.

All 201 samples, classified below into one of four genotypic categories, were tested twice for the presence of 32 mutant alleles, 30 normal alleles, 3 Exon 10 polymorphisms, and 3 Poly-T variants by each of the two independent sites:

• 25: normal DNA with none of the CFTR mutations within the test panel   
• 140: contained one mutant allele within the core panel   
• 35: contained either two mutant alleles within the core panel or one mutant allele $^ +$ one polymorphism/variant   
• 1: contained two mutant alleles within the core panel $^ +$ one variant

The results for the Core OLA Assay were compared to sequencing with respect to the mutations and normal alleles that the core assay is designed to identify in accordance with ACMG guidelines. When reflex testing was indicated by the Core OLA Assay, the results for the polymorphisms or variants identified by the reflex testing were compared with sequencing. The CF GT Assay uses a testing algorithm based on the ACMG guidelines; therefore, testing for polymorphisms within Exon 10 and variants within the Poly-T tract of IVS-8 is only warranted when specific results are observed in the Core OLA Assay. Percent agreement between the CF GT Assay and sequencing is summarized in the following table for all 24,954 genotype calls within the sample set.

Table 8. Concordance of the combined Core and Reflex OLA assays.   

<table><tr><td rowspan=2 colspan=1>SequencingResult</td><td rowspan=1 colspan=4>Test Call Result (Core + Reflex OLA Assays)</td><td rowspan=2 colspan=1>Total (%)</td></tr><tr><td rowspan=1 colspan=1>Normal/WT</td><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>Homozygous</td><td rowspan=1 colspan=1>Concordant</td></tr><tr><td rowspan=1 colspan=1>Normal/WT</td><td rowspan=1 colspan=1>24,14524,146a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24,145a</td><td rowspan=1 colspan=1>99.996</td></tr><tr><td rowspan=1 colspan=1>Heterozygous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Homozygous</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24,14524,146a</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>24,953</td><td rowspan=1 colspan=1>99.996b</td></tr></table>

a. The 5/7/9T Reflex OLA Assay erroneously identified the 9T variant in one replicate of sample 6043.Sequencing confirmed that this sample contained a heterozygous 5T variant and the 7T normal allele, which were also detected by the 5/7/9T Reflex OLA Assay (along with 9T).   
b. The one-sided $9 5 \%$ lower confidence limit $= 9 9 . 9 8 \%$ .

# System Failure Rate

A total of 24,954 genotype calls was generated by the CF GT Assay during the accuracy study, and all but one call matched the sequencing result (failure rate $=$ $0 . 0 0 4 \%$ ). The single incorrect genotype call was associated with the 5/7/9T Reflex OLA Assay result from a single replicate of sample 6043. The reflex assay erroneously indicated the presence of a 9T variant. One 5T and one 7T variant were also detected; thus, the genotype call was 5T/7T/9T. Sequencing confirmed that the sample contained a heterozygous 5T variant and the 7T normal allele. All other replicates of this sample produced the correct 5T/7T result.

# Retest Rate

Of the 804 sample replicates tested, a total of 20 $( 2 . 5 \% )$ required retest due to a poor reaction or PCR failure, or GeneMapper quality flags that indicated that the sample should be repeated when data results were considered questionable or invalid. For samples that did not initially report a genotype result, the correct result was reported upon repeat testing.

Additionally, injection failures on the genetic analyzer were observed for 53 samples $( 6 . 6 \% )$ . The injections were repeated using the same sample-specific electrophoretic mixes that were prepared for the original injection. Therefore, the genotype call was classified as ‘delayed’ since repeating the assay was not required. In some cases, the user chose to limit the reinjection to the individual affected samples. In other cases, all samples within the capillary run or entire plate were reinjected, based on convenience. Reinjections were usually performed immediately after the previous run and were always performed within 48 hours of the addition of Hi-Di™ Formamide to the OLA sample. Upon reinjection, valid results were obtained for each sample. In cases where partial or entire plates were reinjected, the initial, correct genotype calls did not change upon reinjection. Table 9 summarizes comparison results before and after retesting.

Table 9. Clinical trial method comparison study (by mutation) before and after retesting.   

<table><tr><td colspan="1" rowspan="3">Genotype by DNASequencing</td><td colspan="2" rowspan="1"># UniqueSamplesand/orReplicates</td><td colspan="5" rowspan="1">Number CF GT Calls Before Repeat Testing (Based onInitial Results</td><td colspan="3" rowspan="1">Number of CF GT CallsAfter Repeat Testing (30calls per sample x numberof replicates)a</td><td colspan="1" rowspan="3">TotalNumberofCorrectCalls</td><td colspan="1" rowspan="3">Agreement(%)</td></tr><tr><td colspan="1" rowspan="2">SiteA</td><td colspan="1" rowspan="2">SiteB</td><td colspan="2" rowspan="1">Correct Calls</td><td colspan="1" rowspan="2">DelayedDall ls</td><td colspan="1" rowspan="2">Agreement(InitialResults)c</td><td colspan="1" rowspan="2">Repeatsd</td><td colspan="2" rowspan="1">Correct Calls</td><td colspan="1" rowspan="2">MissedCalls</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1078delT</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;A</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;A/delF508</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1898+1G&gt;A</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">2183AA&gt;G/delF508</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">2184delA</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">2184 del A/delF508</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="3">Genotype by DNASequencing</td><td colspan="2" rowspan="1"># UniqueSamplesand/orReplicates</td><td colspan="5" rowspan="1">Number CF GT Calls Before Repeat Testing (Based onInitial Results)</td><td colspan="3" rowspan="1">Number of CF GT CallsAfter Repeat Testing (30calls per sample x numberof replicates)a</td><td colspan="1" rowspan="3">TotalNumberofCorrectCalls</td><td colspan="1" rowspan="3">Agreement(%)</td></tr><tr><td colspan="1" rowspan="2">SiteA</td><td colspan="1" rowspan="2">SiteB</td><td colspan="2" rowspan="1">Correct Calls</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Agreement</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Correct Calls</td><td colspan="1" rowspan="2">MissedCalls</td></tr><tr><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">DelayedCallsb</td><td colspan="1" rowspan="1">(InitialResults)c</td><td colspan="1" rowspan="1">Repeatsd</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">Site B</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">3120+1G&gt;A</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">3849+10kbC&gt;T</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">3876delA</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">330</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">720</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">3905insT</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">394delTT</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T/711+1G&gt;T</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">711+1G&gt;T</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">A455E</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">A455E/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">delF508/delF508e</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">186°</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">372</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">del1507</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">960</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">F508C</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G542X</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G542X/3659delC</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G542X/delF508</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G551D/delF508</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G85E</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">G85E/621+1G&gt;T</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">I506V</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">I506V/delF508</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">156</td><td colspan="1" rowspan="1">155</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">372</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">1507V</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">I507V/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">124</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">N1303K/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R117H/7T</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">126</td><td colspan="1" rowspan="1">125</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">372</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R117H/5T/7T</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">127</td><td colspan="1" rowspan="1">99.2</td></tr><tr><td colspan="1" rowspan="1">R117H/5T/9T/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R347H</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">93.8</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R347H/R553X</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R347P</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R553X</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R553X/delF508</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R560T</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">360</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">R560T/delF508</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S549N</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">240</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">480</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S549N/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">75.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">S549R</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">95.0</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">V520F</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">270</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">W1282X/delF508</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Wild Type</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">1,440</td><td colspan="1" rowspan="1">1,290</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">94.0</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">1,500</td><td colspan="1" rowspan="1">1,500</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3,000</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Overall Testing</td><td colspan="1" rowspan="1">402</td><td colspan="1" rowspan="1">402</td><td colspan="1" rowspan="1">11,128</td><td colspan="1" rowspan="1">10,825</td><td colspan="1" rowspan="1">1,590</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">632</td><td colspan="1" rowspan="1">12,088</td><td colspan="1" rowspan="1">12,087</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">24,175</td><td colspan="1" rowspan="1">99.996</td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="3" rowspan="1">23,543</td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a. Two lots were used in this study.   
b. Reinjection of same OLA reaction used for the first injection. Represents cases where users chose to reinject individual samples as well as entire or partial plates (usually based on convenience). In cases where partial or entire plates were reinjected, the initial, correct genotype calls did not change upon reinjection.   
c. Results reflect percent agreement with sequencing prior to repeat testing.

d. Repeat from DNA/PCR. Repeat testing was performed due to a poor reaction, PCR failure, or GeneMapper quality flags indicating questionable or invalid data.

e. Reflex testing included. Exon 10: Additional calls per site (1 call per sample x number of replicates $_ { \textrm { X 2 } }$ lots). 5/7/9T: Additional calls per site (1 or 2 variant calls per sample x number of replicates x 2 lots).

f. Initial reflex testing made an erroneous call for the 9T variant for 1 of 4 replicates of sample C6043; correct calls were made for the 5/7T variants in this sample. Operator did not perform follow-up testing for this sample, therefore, it remains a missed call.

b. Matrix comparison: Not applicable. This test is only for use with human whole blood collected using EDTA as the anticoagulant.

3. Clinical studies:

a. Clinical Sensitivity: The clinical sensitivity can be estimated based on the published studies of mutation frequencies in various ethnicities.   
$b$ . Clinical specificity: The clinical specificity can be estimated based on published literature and based on the results of analytical studies described in this submission.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

Cystic Fibrosis (CF) is the most common autosomal recessive disorder in the Caucasian population, with an incidence of approximately 1 in 3,200 live births. The incidence of CF in other ethnic groups varies, as seen in the following table.

Table 10. Incidence of Cystic Fibrosis in different Ethnic Groups   

<table><tr><td rowspan=1 colspan=1>Ethnic Group</td><td rowspan=1 colspan=1>Incidence of Cystic Fibrosis</td></tr><tr><td rowspan=1 colspan=1>North AmericanCaucasian</td><td rowspan=1 colspan=1>1 in 3200</td></tr><tr><td rowspan=1 colspan=1>Ashkenazi Jewish</td><td rowspan=1 colspan=1>1 in 3300</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>1 in 9500</td></tr><tr><td rowspan=1 colspan=1>African American</td><td rowspan=1 colspan=1>1 in 15 300</td></tr><tr><td rowspan=1 colspan=1>Asian American</td><td rowspan=1 colspan=1>1 in 32 100</td></tr><tr><td rowspan=1 colspan=1>Native American (Pueblo)</td><td rowspan=1 colspan=1>1 in 3970</td></tr><tr><td rowspan=1 colspan=1>Native American (Zuni)</td><td rowspan=1 colspan=1>1 in 1347</td></tr></table>

Table 11A. CFTR mutation frequency among individuals with clinically diagnosed cystic fibrosis by racial/ethnic group and in a pan-ethnic U.S. population.   

<table><tr><td colspan="1" rowspan="2">CFTR CoreMutationsrecommended by the2004 &amp; 2006ACMG guidelines</td><td colspan="6" rowspan="1">Mutation frequencies among individuals with clinically diagnosed cystic fibrosis (%)</td></tr><tr><td colspan="1" rowspan="1">Non-HispanicCaucasian</td><td colspan="1" rowspan="1">HispanicAmerican</td><td colspan="1" rowspan="1">AfricanAmerican</td><td colspan="1" rowspan="1">AsianAmerican</td><td colspan="1" rowspan="1">AshkenaziJewish</td><td colspan="1" rowspan="1">Pan-EthnicPopulation</td></tr><tr><td colspan="1" rowspan="1">R553X</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.81</td><td colspan="1" rowspan="1">2.32</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">1.21</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">2.25</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">3.15</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">1.93</td></tr><tr><td colspan="1" rowspan="1">del1507</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">1.87</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.90</td></tr><tr><td colspan="1" rowspan="1">delF508</td><td colspan="1" rowspan="1">72.42</td><td colspan="1" rowspan="1">54.38</td><td colspan="1" rowspan="1">44.07</td><td colspan="1" rowspan="1">38.95</td><td colspan="1" rowspan="1">31.41</td><td colspan="1" rowspan="1">66.31</td></tr><tr><td colspan="1" rowspan="1">1717-1G→A</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.44</td></tr><tr><td colspan="1" rowspan="1">G542X</td><td colspan="1" rowspan="1">2.28</td><td colspan="1" rowspan="1">5.10</td><td colspan="1" rowspan="1">1.45</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">7.55</td><td colspan="1" rowspan="1">2.64</td></tr><tr><td colspan="1" rowspan="1">R560T</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">3120+1G→A</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">9.57</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">A455E</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">R117H</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="1">2184delA</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.15</td></tr><tr><td colspan="1" rowspan="1">2789+5G→A</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.38</td></tr><tr><td colspan="1" rowspan="1">1898+1G→A</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.13</td></tr><tr><td colspan="1" rowspan="1">621+1G→T</td><td colspan="1" rowspan="1">1.57</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">1.30</td></tr><tr><td colspan="1" rowspan="1">711+1G→T</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.35</td></tr><tr><td colspan="1" rowspan="1">G85E</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">R347P</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.36</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">1.50</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">45.92</td><td colspan="1" rowspan="1">2.20</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">1.78</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="1">3849+10kbC→T</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">1.57</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">5.31</td><td colspan="1" rowspan="1">4.77</td><td colspan="1" rowspan="1">0.85</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">1.66</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">2.78</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.28</td></tr></table>

Table 11B. Frequency of CFTR Mutations not in ACMG Panel.   

<table><tr><td rowspan=1 colspan=1>Mutation</td><td rowspan=1 colspan=1>Frequency of Mutations in Specific Populations</td></tr><tr><td rowspan=1 colspan=1>S549N</td><td rowspan=1 colspan=1>Multi-ethnic mutation: Belarus 0.5%, UK 0.5%, Mexico 1.9%, AfricanAmerican 0.7%</td></tr><tr><td rowspan=1 colspan=1>S549R</td><td rowspan=1 colspan=1>Arabic Lineage: United Arab Emirates 61% Morocco: (Jewish) 6%</td></tr><tr><td rowspan=1 colspan=1>V520F</td><td rowspan=1 colspan=1>UK 0.3%</td></tr><tr><td rowspan=1 colspan=1>3876delA</td><td rowspan=1 colspan=1>4th most common CFTR mutation found in California Hispanic population1.7%</td></tr><tr><td rowspan=1 colspan=1>394delTT</td><td rowspan=1 colspan=1>Nordic mutation prevalent in Scandinavia and Baltic: Sweden &gt;7%,Finland &gt;28%, Estonia 13% Als0, second most common mutation in whitepopulation in South Africa</td></tr><tr><td rowspan=1 colspan=1>2183AA→G</td><td rowspan=1 colspan=1>Multi-ethnic mutation; Italy (NE) 9.3%, Austria (Tyrol) 2.4%, Turkey2.5%, other countries 0.5 to 1.5%</td></tr><tr><td rowspan=1 colspan=1>3905insT</td><td rowspan=1 colspan=1>Swiss mutation 4.8%; also in Amish, Acadian, and Mennonites</td></tr><tr><td rowspan=1 colspan=1>R347H</td><td rowspan=1 colspan=1>Multi-national mutation: Turkey 3.6%, France (Brittany) 0.8%</td></tr><tr><td rowspan=1 colspan=1>1078delT</td><td rowspan=1 colspan=1>Mutation supports genetic diversity of multi-ethnic populations that may beunderserved by the ACMG panel alone. 0.09% per ACMG guidelines(Hispanic American) 0.06% per Strom Analysis 0.31% per Sugarmananalysis (Hispanic population)</td></tr></table>

# N. Instrument Name:

CEGA- $1 6 ^ { \mathbf { T M } }$ Instrument; other names: CF Configuration Disk v1.0 (GeneMapper $\textsuperscript { \textregistered }$ Software v3.5.3 Upgrade File - CF Analysis Settings and Parameters) for use with CF GT assay.

# O. System Descriptions:

1. Modes of Operation: CEGA- $1 6 ^ { \mathbf { T M } }$ Instrument operates in batch mode.

2. Software:

The operator software is installed on an off-the-shelf Windows PC running Windows 2000. Access to the software is first controlled by secure Windows logon and then by Login name/password identification in the operator software.

The CEGA- $1 6 ^ { \mathrm { T M } }$ Instrument computer workstation is supplied with the instrument. Its minimum configuration consists of the following:

• Dell™ Precision workstation 340 with: - 2.4 GHz Inte $^ \mathrm { \textregistered }$ Pentium $^ \mathrm { \textregistered }$ IV processor - 1 GB of RAM - Two 120 GB hard drives - CD-ROM Drive

The Software Configuration:

• Microsoft $^ \mathrm { \textregistered }$ Windows $^ \mathrm { \textregistered }$ 2000 Professional Operating System, Service Pack 4   
• Data Collection v2.0   
• GeneMapper $^ \mathrm { \textregistered }$ Software v3.5 - One user kit   
• CF Configuration Disk v1.0 - GeneMapper $\textsuperscript { \textregistered }$ Software v3.5.3 Upgrade File - CF Analysis Settings and Parameters

There are three default profiles (Administrator, Scientist, and Technician). Changes or edits are tracked through the Audit Map function and the users are instructed not to disable this feature. The CF Configuration Disk version 1.0 is provided by Celera and installed by Abbott Molecular. It consists of a patch to GeneMapper v3.5 which upgrades this software package to GeneMapper v3.5.3 to incorporate the necessary IVD labeling changes and includes minor bug fixes. The CF Configuration disk includes the GeneMapper Software v3.5.3 Upgrade File along with CF Analysis Settings and Parameters that customize the GeneMapper Software for use with the CF Assay. Data on the OLA fragments is collected with the Data Collection Software v2.0. Only the fragment analysis function/capability of the CEGA- $1 6 ^ { \mathrm { T M } }$ Instrument was reviewed. Sequence analysis software is not used with the CF GT assay and were not reviewed. The system runs as a stand alone including an attached printer, or can be networked to a central printer.

Detailed use of the software is described in the Cystic Fibrosis Genotyping Assay Instrument and Operator’s manuals, provided separately by the manufacturer. The ligated, fluorescent-labeled DNA fragments are separated on the CEGA- $1 6 ^ { \mathrm { T M } }$ Instrument by capillary electrophoresis. Detection is based on size, as compared to the fragments generated from the CFTR Wild Type Control (allelic ladder), and fluorescent label. The ligation products are then identified and genotyped by analysis with the GeneMapper $\textsuperscript { \textregistered }$ Software v3.5.3 using a combination of an analysis method, panel, and size standard files selected during creation of the plate record in the Plate Manager window of the Data Collection software prior to starting a run, based on the assay type to be run (i.e., Core, 5/7/9T Reflex or Exon 10 Reflex panel). Choice of the wrong method or panel for the assay type will result in no data. The user can re-run samples however if this occurs with the correct choices. Sample name or identifiers are entered to identify the location of the samples on the plate. Samples are also identified by sample type (Allelic Ladder, Negative Control, or Sample).

The Plate Record is linked to the Run Scheduler through either scanning the plate barcode or through a search function and dropdown list. The user is instructed to review the run using the Run View feature to verify the run is scheduled correctly. Data analysis can be performed automatically by enabling the autoanalysis feature using Autoanalysis

Manager. The GeneMapper software must be closed for the Autoanalysis Manager to function, if not, sample data is stored in a queue until GeneMapper is closed. The user is warned not to perform data analysis or run other software during a run.

GeneMapper v3.5.3 analysis functions read the output data (.fsa file extension) from the Data Collection Software. Samples are sized based on size standards added to each sample during preparation to be analyzed, and overlays intervals (bins) in which peaks are expected, grey bins are for normal alleles associated with the marker and pink for mutant alleles, then labels peaks falling within the expected intervals as either normal or mutant.

GeneMapper $\textsuperscript { \textregistered }$ Software v3.5.3 provides information that allows the user to assess the quality of the data used to conduct the genotype call analysis. GeneMapper v3.5.3 ensures that the data received after Autoanalysis have at least one or two alleles (normal or mutant) per locus, have been sized correctly produce expected peak heights and shapes, and have a normal baseline. To evaluate these parameters, GeneMapper v3.5.3 analyzes the data according to CF specific, preset parameters that are defined within the configuration file. The data quality results are expressed in the form of flags associated with Process Quality Values (PQVs), which include Genotype Quality (GQ) and Sample Genotype Quality (SGQ). The software qualifies results using a summary of signal quality based on all Quality Values as either Good & Acceptable, Compromised (but potentially usable), and Unacceptable. The CF GT assay does not support the editing of Quality Values within the analysis software therefore samples associated with a red Sample Genotype Quality flag (Unacceptable) must be rejected. Results for samples associated with a yellow SGQ flag (compromised) may be accepted by the user following manual review of all data, which includes all sample and marker PQVs, as well as contents of the electropherogram plot(s). Justification for the acceptance and subsequent release of results with a yellow SGQ flag is required. The software will provide printouts of the detected genotypes, electropherograms, and quality values.

# Results

Every sample must have at least two alleles for each marker and each marker must have at least one peak in the electropherogram. The following calls are possible based on the results.

Table 12. Possible calls for a given result.   

<table><tr><td rowspan=1 colspan=1>If the marker is associatedwith:</td><td rowspan=1 colspan=1>Then the allele call is:</td></tr><tr><td rowspan=1 colspan=1>No peaks</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>One peak in a gray bin</td><td rowspan=1 colspan=1>Homozygous normal</td></tr><tr><td rowspan=1 colspan=1>One peak in a pink bin</td><td rowspan=1 colspan=1>Homozygous mutant</td></tr><tr><td rowspan=1 colspan=1>One peak in a gray bin + onepeak in a pink bin</td><td rowspan=1 colspan=1>Heterozygous carrier of the mutant allele associated withthe pink bin</td></tr><tr><td rowspan=1 colspan=1>One peak in each of two pinkbins</td><td rowspan=1 colspan=1>Heterozygous carrier of the mutant alleles associated withthe pink bins</td></tr></table>

# Views

A number of views are available to visual the data. The GeneMapper v3.5.3 can be used to view the sample data in both raw and analyzed forms. Operators may adjust the size using the resize pane and/or zoom functions within the GeneMapper software to enhance viewing of the electropherogram within the display screen. Operators can also control the number of individual electropherograms displayed on the screen using the pane function within the toolbar.

# Printing

For the CF GT Assay, the Configuration Disk v1.0 includes a file that configures GeneMapper in a manner that yields a customized results printout. Each printout contains information for a single sample. The electropherogram section of the printout depicts three panes for the CF Core OLA assay results, one for each set of dye-labeled probes within the OLA reaction (FAM-blue, HEX-green and TAMRA-black). The Genotypes Table within the printout identifies the alleles detected for each marker, as well as each marker’s PQV information.

Table 13. The information contained in each results printout.   

<table><tr><td rowspan=1 colspan=1>GeneMapper Version:</td><td rowspan=1 colspan=1>3.5.3</td></tr><tr><td rowspan=1 colspan=1>Sample File:</td><td rowspan=1 colspan=1>The .fsa file used to generate the printout</td></tr><tr><td rowspan=1 colspan=1>Sample Name:</td><td rowspan=1 colspan=1>The sample identifier assigned by the user within theDataCollection software</td></tr><tr><td rowspan=1 colspan=1>Panel:</td><td rowspan=1 colspan=1>CF IVD Core Panel, 5/7/9T Panel, or Exon 10 Panel</td></tr><tr><td rowspan=1 colspan=1>Date and Time Stamp:</td><td rowspan=1 colspan=1>Date and time the printout was printed</td></tr><tr><td rowspan=1 colspan=1>Intended Use Statement:</td><td rowspan=1 colspan=1>For In Vitro Diagnostic Use</td></tr><tr><td rowspan=1 colspan=1></td><td></td></tr><tr><td rowspan=1 colspan=1>Printed by:</td><td rowspan=1 colspan=1>Account name of the individual logged into GeneMapperwhen the printout is printed</td></tr><tr><td rowspan=1 colspan=1>Pagination:</td><td rowspan=1 colspan=1>Page number and the total number of pages in the printout</td></tr></table>

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification:

Users must fill in Batch Information by providing a unique batch Name, Description and Creator. Users have to enter appropriate patient information, i.e. number of samples, and sample IDs.

4. Specimen Sampling and Handling:

Genomic DNA is extracted from whole blood samples by the laboratory’s validated method. The extracted DNA sample is the starting point for the CF GT Assay system.

5. Calibration:

Prior to use, both a spatial and spectral calibration must be performed on the instrument as described in the CEGA- $1 6 ^ { \mathrm { T M } }$ Instrument manual. A spatial calibration must be performed after each time the user installs or replaces a capillary array, temporarily removes the capillary array from the detection block, or moves the instrument. The spectral calibration must be performed whenever a new dye set is used on the instrument; whenever a service engineer realigns/replaces the laser, optics, or CCD camera; if a decrease in spectral separation (i.e., called pull-up and/or pull-down peaks) is observed; or whenever an array is changed and one or more capillaries show low signal strength and/or pull-up peaks.

6. Quality Control:

The CF GT Assay includes two reagents that are used as controls to monitor assay performance: CFTR Wild Type Control and CF Sample Diluent. Both the CFTR Wild Type Control and CF Sample Diluent must be processed with each plate for each OLA probing test procedure (i.e., a plate with Core, 5/7/9T Reflex, and Exon 10 Reflex samples must have 3 sets of controls).

CFTR Wild Type Control contains normal alleles for all 30 of the targeted loci within the CFTR gene. It serves as a positive control for both amplification and detection of the normal CFTR alleles (allelic ladder control).

A No Template Control (NTC) is consisting of the CF Sample Diluent, which is a buffered solution that does not contain DNA, is used for both the amplification and data analysis steps. This reagent must be processed with each batch of samples from PCR through data analysis. The observation of labeled peaks for the No Template Control may indicate amplicon cross-contamination.

Optional CF Mutation Positive Control is suggested to the users as recommended in section CF 3.2.3.4.2 of the 2006 ACMG Technical Standards and Guidelines for CFTR Mutation Testing. This is a user supplied control which can consist of samples that have CF mutations that are externally sourced (e.g., reference cell lines), developed in-house (e.g., plasmid), or previously identified patient samples can be used as a mutation positive control. If such controls are used, identify them as samples and manually verify that the results are correct.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above: None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10, Labeling for In Vitro Diagnostic Products.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.