# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

E. Applicant: Third Wave Technology, Inc.

F. Proprietary and Established Names: InPlex™ CF Molecular Test

# G. Regulatory Information:

3. Product code: NUA, System, test, CFTR

4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): InPlex™ CF Molecular Test is an in vitro diagnostic device used to simultaneously detect and identify a panel of mutations and variants in the cystic fibrosis transmembrane conductance regulator (CFTR) gene in genomic DNA samples isolated from human peripheral whole blood specimens. The panel includes mutations and variants recommended by the 2004 American College of Medical Genetics (ACMG). The InPlex™ CF Molecular Test is a qualitative genotyping test that provides information intended to be used for cystic fibrosis carrier screening as recommended by ACMG and the 2005 American College of Obstetricians and Gynecologists (ACOG) for adults of reproductive age, as an aid in newborn screening for cystic fibrosis, and in confirmatory diagnostic testing for cystic fibrosis in newborns and children. The test is not indicated for use in fetal diagnostic or pre-implantation testing. This test is also not indicated for stand-alone diagnostic purposes and results should be used in conjunction with other available laboratory and clinical information.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For Prescription Use Only

4. Special instrument requirements:

Multi-well fluorometer capable of: Multi-Labeling Measurement  Measurement  Measurement Parameters 1 2 Read mode: Top Top Excitation wavelength: 485/20 nm 562/10 nm Emission wavelength: 535/25 nm 635/35 nm Gain (Manual): 48 – 102 90 – 152 Number of flashes: 10 10 Integration time: $\leq 2 0 ~ \mu \mathrm { s }$ $\leq 2 0 ~ \mu \mathrm { s }$

# I. Device Description:

InPlex™ CF Molecular Test is comprised of:

12 InPlex™ CF Micro-fluidic Cards -12 cards 2 vials each of Amplification Primer Mix, Amplification Buffer, Amplification Enzyme, DNA Reaction Buffer, and Cleavase® Enzyme 12 InPlex™ Card Holders CD-ROM containing InPlex™ CF Molecular Test Call Reporting Software and Call Reporting Software User Manual (provided with the first order/shipment)

# J. Substantial Equivalence Information:

1. Predicate device name(s): Tag-It™ Cystic Fibrosis Kit   
2. Predicate K number(s): k043011   
3. Comparison with predicate:

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Device</td><td>Predicate</td></tr><tr><td>Intended Use/Indications for Use</td><td>InPlexTM CF Molecular Test is an in vitro diagnostic device used to simultaneously detect and identify a panel of mutations and variants in the cystic fibrosis transmembrane conductance regulator (CFTR) gene in genomic DNA samples isolated from human peripheral whole blood specimens. The panel includes mutations and variants recommended by the 2004 American College of Medical Genetics (ACMG). The InPlexTM CF Molecular Test is a qualitative genotyping test that provides information intended to be used for cystic fibrosis carrier screening as recommended by ACMG and the 2005 American College of Obstetricians and Gynecologists (ACOG) for adults of reproductive age, as an aid in newborn screening for cystic fibrosis, and in confirmatory diagnostic testing for cystic fibrosis in newborns and children.</td><td>Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The test is not indicated for use in fetal diagnostic or pre-implantation testing. This test is also not indicated forstand-alone diagnostic purposes and results should beused in conjunction with other available laboratory andclinical information.</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Genomic DNA (gDNA) isolated from whole blood</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Referencemethod</td><td colspan="1" rowspan="1">Bi-directional sequencing</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Mutations andvariantsdetected</td><td rowspan=1 colspan=1>23 mutations and 4 variants(polymorphisms)recommended by ACMG(2004)/ACOG (2005).</td><td rowspan=1 colspan=1>23 mutation panel recommended byACMG/ACOG, and 16 additionalmutations associated with CFphenotypes in Caucasian Americans,Hispanic Americans and AfricanAmericans and 4 polymorphisms</td></tr><tr><td rowspan=1 colspan=1>Methodology</td><td rowspan=1 colspan=1>Multiplex PCR, isothermicprimer/probe hybridizationand signal amplification inmicrofluidic card</td><td rowspan=1 colspan=1>Multiplex PCR, allele specificprimer extension and Luminexbead/anti-tag/tag primer association</td></tr><tr><td rowspan=1 colspan=1>Detection</td><td rowspan=1 colspan=1>Fluorometric resonanceenergy transfer (FRET)</td><td rowspan=1 colspan=1>Flow cytometry</td></tr><tr><td rowspan=1 colspan=1>Instruments</td><td rowspan=1 colspan=1>Multi-well fluorometer</td><td rowspan=1 colspan=1>Luminex 100 IS (Integrated System)</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>InPlexTM CF Molecular TestCall Reporting Software</td><td rowspan=1 colspan=1>Tag-ItTM Data Analysis Software</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

American College of Medical Genetics (ACMG) / American College of Obstetricians and Gynecologists - 2001, 2002, 2004 ACMG Technical Standards and Guidelines for CFTR mutation   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: CFTR Gene Mutation Detection Systems

# L. Test Principle:

InPlex™ CF Molecular Test amplifies specific regions of the CFTR gene in genomic DNA extracted from human whole peripheral blood. Each amplified DNA sample is subsequently mixed with Cleavase® enzyme and buffer then added to a loading port on an InPlex™ micro-fluidic card. Each InPlex™ card contains eight sample-loading ports, each connected to 48 independent reaction chambers. Twenty-eight of these reaction chambers contain dried assay mixes specific for reporting the 23 ACMG/ACOG recommended CFTR mutations and variants. The remaining chambers consist of a “No Invader® Control,” an independent Quality Control, and several unused chambers.

After an InPlex™ card is loaded; the channels are mechanically sealed using a microfluidic card sealer, isolating each individual reaction chamber from all other chambers. The card is then incubated to allow individual Invader® reactions to occur. Following incubation, the card is read in a multi-well fluorometer and the raw signal data are imported into the InPlex™ CF Molecular Test Call Reporting Software for final result analysis. IVS-8 5T, 7T, and 9T results are only displayed for R117H positive (Het or Mut) samples. The nonsense F508C variant is assessed only for the interpretation of delI507 and delF508 mutations. The $2 1 8 3 \mathrm { A A } > \mathrm { G }$ mutation is assessed only for the interpretation of the 2184delA mutation and not displayed.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility study was performed in two phases, a proficiency phase, and a performance phase. The proficiency phase was used to ensure each site had the required expertise in the methodology to ensure that meaningful data could be generated and calculated during the performance phase. For the Proficiency testing, two technicians at each of three investigative sites ran the same CF panel in duplicate. Data generated during this phase of the study was to demonstrate technician proficiency only. Reproducibility testing for the performance phase was conducted using 23 samples containing mutations representing the ACMG recommended panel. Each site ran the same samples in duplicate by each technician on each of five (5) non-consecutive days.

Table 1. Summary of Agreement Data for All Three Sites   

<table><tr><td></td><td>Number of</td><td>Number of</td><td>%</td><td></td></tr><tr><td>Analysis</td><td>Comparisons</td><td>Agreements</td><td>Agreement</td><td>95% LCB1</td></tr><tr><td>Within operator/Within day</td><td>15,870</td><td>15,869</td><td>99.994%</td><td>99.984%</td></tr><tr><td>Between days/Within operator</td><td>126,960</td><td>126,944</td><td>99.987%</td><td>99.982%</td></tr><tr><td>Between operators/within site</td><td>158,700</td><td>158,680</td><td>99.987%</td><td>99.982%</td></tr><tr><td>Between Sites Agreement</td><td>634,800</td><td>634,720</td><td>99.987%</td><td>99.985%</td></tr><tr><td colspan="5">1-sided lower 95% Confidence Limit</td></tr></table>

Lot-to-Lot Equivalency: Equivalency between three lots of InPlex™ CF Molecular Test kits was evaluated. A panel of 23 CFTR gDNA samples was tested in singlicate with each lot. Five hundred twenty-nine (529) genotype calls were generated for each lot (23 samples x 23 assays). All genotype calls were in $100 \%$ $( 9 9 . 4 \%$ , $9 5 \%$ LCB) agreement to pre-characterized gDNA genotypes for each lot tested.

$b$ . Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods): The reference method was bi-directional sequencing.

Assay Controls: The assay contains a No DNA Control which is used to indicate if contamination occurred during the amplification step. It is recommended that the user run CFTR mutation positive control(s) as per

ACMG Standards and Technical Guideline recommendations to insure proper test results.

Stability: On-going real-time stability studies are being performed on 3 lots each of InPlex™ micro-fluidic cards and the InPlex™ CF molecular test when stored under pre-defined conditions. A panel of seven CFTR gDNA samples and eight control samples comprised of pooled amplicons that cover the mutations in the InPlex™ CF Molecular Test, providing 345 calls (15 samples x 23 assays), were used to assess the performance. The samples were tested in duplicate with each lot for each storage condition at each time point. Three lots of the InPlex™ Reagents are being maintained at the recommended storage temperature of $( - 2 0 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C } )$ . The InPlex™ micro-fluidic cards are stored at the recommended temperature $( - 2 0 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C } )$ , substandard temperature $( 4 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C } )$ , and ship-stressed conditions $3 7 ^ { \circ } \mathrm { C } { \pm } 1 ^ { \circ } \mathrm { C }$ for 48 hours, then room temperature $( 2 5 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C } )$ ). Results to date, indicate $100 \%$ agreement with initial results $( \mathrm { T } { = } 0 )$ ). Stability studies are ongoing for which results from future time points will be used to extend shelf-life dating.

Freeze-Thaw: A study was performed to establish the tolerance of the InPlex™ CF Molecular Test to various freeze-thaw cycles. The InPlex™ CF Molecular Test kit were subjected to 2, 4, 6, 8, 10 and 12 freeze-thaw cycles (test points) followed by functional testing of the product with control samples. Each freezethaw cycle consisted of freezing at $- 2 0 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C }$ for at least 24 hours and thawing at room temperature for 30 minutes. For the multiple freeze-thaw cycles the cards were returned to $- 2 0 ^ { \circ } \mathrm { C } \pm 3 ^ { \circ } \mathrm { C }$ storage after 30 minutes at room temperature. An overall percent agreement of $100 \%$ $9 9 . 9 2 \%$ , 1-sided lower $9 5 \%$ Confidence Limit) as compared to DNA sequencing for all freeze-thaw cycles tested. Eight or fewer freeze-thaw cycles for the $\mathrm { I n P l e x ^ { T M } }$ Molecular Test are recommended to ensure the InPlex™ CF Molecular Test maintains the ability to generate accurate genotype calls.

d. Detection limit:

The package insert recommends that the concentration of gDNA should be 15 ${ \mathrm { g / \mu L } }$ . An input genomic DNA concentration range was evaluated with the InPlex™ CF Molecular Test. A panel of eight gDNA samples were prepared and tested at eight concentrations ranging from $1 \mathrm { n g / \mu L }$ to $1 5 0 ~ \mathrm { { n g / \mu L } }$ (total input DNA range of $5 { \mathrm { - } } 7 5 0 \ \mathrm { n g / }$ reaction). Genotype call results for all eight characterized samples for each of the 23 alleles were assessed for percent agreement at each concentration. The lower limit of detection was defined as the lowest DNA concentration in which a $9 9 \%$ or greater concordance with DNA sequencing was observed.

Based on these results, a $5 \mathrm { n g / \mu L }$ DNA concentration (input DNA of $2 5 \mathrm { n g }$ ) provided a percent agreement of $100 \%$ $9 8 . 4 \%$ , $9 5 \%$ LCB) with DNA sequencing qualifying it as the lower limit of detection. At the highest DNA concentration tested, $1 5 0 ~ \mathrm { { n g / \mu L } }$ (input DNA of $7 5 0 \mathrm { n g }$ ), a $9 9 . 5 \%$ percent agreement was obtained $( 9 8 . 6 \%$ , $9 5 \%$ LCB). The remaining DNA concentrations tested, 10, 20, 50, and $1 0 0 \mathrm { n g / \mu l }$ (input DNA of 50, 100, 250 and $5 0 0 \mathrm { n g } )$ all obtained a 100 percent agreement $( 9 8 . 4 \%$ , $9 5 \%$ LCB) with expected results.

e. Analytical specificity:

The following interfering substances were added separately to aliquots of 8 different CF positive whole blood samples, prior to extraction, and compared to an untreated sample. The following substances were added to the final calculated concentration indicated: bilirubin $( 1 0 \mathrm { m g / d L } )$ , triglycerides (250 $\mathrm { m g / d L ) }$ , and potassium EDTA $( 2 . 1 6 \mathrm { m g / m l } )$ ). Qiagen® Buffer AW2 and hemoglobin were added to pre-extracted gDNA to a concentration of ${ \sim } 5 \%$ and $0 . 0 1 \%$ respectively. No interference was observed by the potential interferants when compared to the untreated sample or bi-directional sequencing. $f .$ Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device: Accuracy and repeat rate of the InPlex™ CF Molecular Test were determined by comparing InPlex™ CF Molecular Test genotyping results from gDNA isolated from 123 samples, representing a total of $1 4 4 \mathrm { C F }$ mutations and variants were compared to bi-directional sequencing. The sample panel consisted of 96 clinical samples $\mathrm { \underline { { g D N A } } }$ samples isolated from whole peripheral blood) and 27 gDNA samples isolated from commercially available cell lines. The 123 samples used in this study were only extracted one time and tested. The extracted DNA was diluted in nuclease-free ultra pure water to concentrations that ranged from $1 0 \mathrm { n g / \mu L }$ to $2 5 \mathrm { n g / \mu L }$ . Aliquots were then sent for bi-directional DNA sequence analysis and tested with the InPlex™ CF Molecular Test. Twenty-one samples contained mutations not covered by the InPlex™ CF Molecular Test (13 WT/non-ACMG heterozygotes and 9 ACMG/non-ACMG compound heterozygotes), including two $2 1 8 3 \mathrm { A A } > \mathrm { G }$ used to accurately assess the 2184delA mutation, are reported as Wild Type by the assay. Accuracy results were compared to DNA sequencing for these samples, all mutations were Wild Type (Normal) for the 23 mutations included in the product. Table 2 summarizes the DNA sequencing based genotype results for the 123 test samples.

A total of 23 CFTR mutations were tested in this study, with the IVS8- 5T/7T/9T variant being reported for R117H positive samples only. Genotype calls were compared between the DNA sequencing results and the InPlex™ CF Molecular Test results for the calculation of overall agreement. In addition, positive and negative agreement for each mutation was calculated (Table 3a). The repeat rate $( 0 . 8 \% )$ was determined by the number of samples that generated an invalid genotype call for one or more mutations with the InPlex™ CF Molecular Test on the first attempt. The no-call rate was calculated as the number of invalid genotype calls of the total number of calls.

Table 2: Mutation content of Accuracy Study Samples   

<table><tr><td>TWT InPlex CF Mutation Panel</td><td>Pan-Ethnic Mutation Freq.</td><td>Whole blood</td><td>Genomic DNA1</td><td>Cell Line</td><td>Total Clinical Samples</td><td>Total Study</td></tr><tr><td>delta F 508</td><td>66.31</td><td>19</td><td>10</td><td>9</td><td>29</td><td>Samples 38</td></tr><tr><td>G542X</td><td>2.64</td><td>1</td><td>4</td><td>1</td><td>5</td><td>6</td></tr><tr><td>W1282X</td><td>2.2</td><td>2</td><td>4</td><td>0</td><td>6</td><td>6</td></tr><tr><td>G551D</td><td>1.93</td><td>0</td><td>5</td><td>2</td><td>5</td><td>7</td></tr><tr><td>621+1G&gt;T</td><td>1.3</td><td>0</td><td>4</td><td>4</td><td>4</td><td>8</td></tr><tr><td>N1303K</td><td>1.27</td><td>3</td><td>2</td><td>1</td><td>5</td><td>6</td></tr><tr><td>R553X</td><td>1.21</td><td>2</td><td>2</td><td>1</td><td>4</td><td>5</td></tr><tr><td>delta I507</td><td>0.9</td><td>0</td><td>2</td><td>1</td><td>2</td><td>3</td></tr><tr><td>3849+10kbC&gt;T</td><td>0.85</td><td>3</td><td>2</td><td>1</td><td>5</td><td>6</td></tr><tr><td>3120+1G&gt;A</td><td>0.86</td><td>1</td><td>1</td><td>1</td><td>2</td><td>3</td></tr><tr><td>R117H</td><td>0.54</td><td>3</td><td>2</td><td>0</td><td>5</td><td>5</td></tr><tr><td>1717-1G&gt;A</td><td>0.44</td><td>3</td><td>2</td><td>1</td><td>5</td><td>6</td></tr><tr><td>2789+5G&gt;A</td><td>0.38</td><td>1</td><td>2</td><td>1</td><td>3</td><td>4</td></tr><tr><td>R347P</td><td>0.36</td><td>1</td><td>2</td><td>1</td><td>3</td><td>4</td></tr><tr><td>711+1G&gt;T</td><td>0.35</td><td>0</td><td>2</td><td>1</td><td>2</td><td>3</td></tr><tr><td>R334W</td><td>0.37</td><td>1</td><td>1</td><td>1</td><td>2</td><td>3</td></tr><tr><td>R560T</td><td>0.3</td><td>1</td><td>2</td><td>1</td><td>3</td><td>4</td></tr><tr><td>R1162X</td><td>0.3</td><td>1</td><td>2</td><td>1</td><td>3</td><td>4</td></tr><tr><td>3659delC</td><td>0.28</td><td>2</td><td>2</td><td>1</td><td>4</td><td>5</td></tr><tr><td>A455E</td><td>0.26</td><td>0</td><td>3</td><td>1</td><td>3</td><td>4</td></tr><tr><td>G85E</td><td>0.26</td><td>0</td><td>2</td><td>1</td><td>2</td><td>3</td></tr><tr><td>2184delA</td><td>0.15</td><td>0</td><td>1</td><td>1</td><td>1</td><td>2</td></tr><tr><td>1898+1G&gt;A R117H/7T (4 x 7T/7T, 1 x</td><td>0.13</td><td>0</td><td>2</td><td>1</td><td>2</td><td>3</td></tr><tr><td>5T/7T)</td><td></td><td>3</td><td>2</td><td>0</td><td>5</td><td>5</td></tr><tr><td colspan="2">R117H/5T (5T/7T)</td><td>1</td><td>0</td><td>0</td><td>1</td><td>1</td></tr><tr><td colspan="2">TOTAL</td><td>48</td><td>63</td><td>33</td><td>111</td><td>144</td></tr><tr><td>2183AA&gt;G2</td><td></td><td>1</td><td>0</td><td>1</td><td>0</td><td>0</td></tr><tr><td>non-ACMG/WT3</td><td></td><td>6</td><td>3</td><td>4</td><td>9</td><td>13</td></tr><tr><td>ACMG/non-ACMG3</td><td></td><td>2</td><td>3</td><td>3</td><td>5</td><td>8</td></tr><tr><td colspan="2">1 Archived samples extracted from blood</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="8">2 Assessed for purposes of accurate cal of 2184delT mutation only</td></tr><tr><td colspan="8">3 Includes 1 sample with the 2183AA&gt;G mutation</td></tr></table>

Table 3a. Call Comparison of Invader InPlex CFTR assay to bi-directional sequencing.   

<table><tr><td rowspan=2 colspan=1>Mutation</td><td rowspan=2 colspan=1>Calls perutation</td><td rowspan=1 colspan=2>SequencingCalls</td><td rowspan=1 colspan=3>InPlex Calls</td><td rowspan=1 colspan=3>Agreements (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Indet</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>delta F 508</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.2% (97.9%)</td><td rowspan=1 colspan=1>98.8% (96.9%)</td><td rowspan=1 colspan=1>100% (92.8%)</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (65.2%)</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.4%)</td><td rowspan=1 colspan=1>100% (68.8%)</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>R553X</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (54.9%)</td></tr><tr><td rowspan=1 colspan=1>delta I507</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>3849+10kbC&gt;T</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (54.9%)</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (47.3%)</td></tr><tr><td rowspan=1 colspan=1>R347P</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (47.3%)</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>R560T</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (47.3%)</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (47.3%)</td></tr><tr><td rowspan=1 colspan=1>3659delC</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (54.9%)</td></tr><tr><td rowspan=1 colspan=1>A455E</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (47.3%)</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>2184delA</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (22.4%)</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (36.8%)</td></tr><tr><td rowspan=1 colspan=1>IVS-8 5T/7T/9TVariant ()</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100% (97.6%)</td><td rowspan=1 colspan=1>100% (97.5%)</td><td rowspan=1 colspan=1>100% (60.7%)</td></tr><tr><td rowspan=1 colspan=1>Total calls</td><td rowspan=1 colspan=1>2952</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>2808</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>2807</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>99.96% (99.9%)</td><td rowspan=1 colspan=1>99.96% (99.9%)</td><td rowspan=1 colspan=1>100% (97.4%)</td></tr><tr><td rowspan=1 colspan=10>   allele while &quot;Negative&quot; samples are regarded as having only the 7T and/or 9T allele.(*)                                                                                                             CE L</td></tr></table>

$^ { * } )$ In the initial study, four (4) samples confirmed by sequencing as 7T/9T, were called 7T/7T by the CF Inplex Molecular Test. Upon retesting all four (4) samples gave the correct 7T/9T result once per day for five consecutive days.

The IVS8-5T/7T/9T genotyping accuracy was assessed by the manufacturer on all the 123 samples used in the Method Comparison study. The IVS-8 results for four non-R117H samples were miscalled as 7T/7T instead of 7T/9T according to bi-directional sequencing resulting in an initial miscall rate of $0 . 1 4 \%$ (Table 3b).

The four miscalled samples were tested in a reproducibility study once a day for 5 days in conjunction with two 7T/7T and two 7T/9T samples, which were called correctly in the initial study. Upon retesting all four (4) miscalled samples gave the correct 7T/9T result on each of the five consecutive days (Table 4). The root cause of the miscalls was determined to potentially be due to the InPlex cards failing to be rotated during incubation resulting in temperature inconsistencies (e.g., hot spots) near the heat source causing the 9T/7T ratio to fall below the cut-off for the 9T/7T call. The 7T/7T miscall was reproduced in one of the four samples experimentally.

Table 3b: Accuracy of the InPlex CF Molecular Test - IVS8-5T, 7T, 9T.   

<table><tr><td rowspan=2 colspan=1>IVS-85T/7T/9TVariant</td><td rowspan=2 colspan=1>Calls perMutation</td><td rowspan=1 colspan=2>SequencingCalls</td><td rowspan=1 colspan=3>InPlex Calls</td><td rowspan=1 colspan=3>Agreements(95% CI)</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>FalseNeg</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Neg.</td><td rowspan=1 colspan=1>Pos.</td></tr><tr><td rowspan=1 colspan=1>5T</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.6%)</td><td rowspan=1 colspan=1>100%(97.5%)</td><td rowspan=1 colspan=1>100%(60.7%)</td></tr><tr><td rowspan=1 colspan=1>7T (or No 5T)</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(97.6%)</td><td rowspan=1 colspan=1>100%(65.2%)</td><td rowspan=1 colspan=1>100%(97.5%)</td></tr><tr><td rowspan=1 colspan=1>9T (or No 5T)</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>4*</td><td rowspan=1 colspan=1>96.74%(94.1%)</td><td rowspan=1 colspan=1>100%(95.1%)</td><td rowspan=1 colspan=1>93.65%(88.7%)</td></tr><tr><td rowspan=1 colspan=10>*In the initial study, our ) samples confirmed by sequencing s 7T/T, were called 7T/7T by the F InPlexMolecular Test. Upon retesting all four (4) samples gave the correct 7T/9T result once per day for fiveconsecutive days.</td></tr></table>

Table 4. Reproducibility of 7T/9T miscalled samples.   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Orig.Genotypecall</td><td rowspan=1 colspan=1>Genotype(by BDS)</td><td rowspan=1 colspan=1>Agreementwith BDS(%)</td><td rowspan=1 colspan=1>Repeat result(1 run/day x 5days)</td><td rowspan=1 colspan=1>Agreementwith BDS (%)after repeat</td></tr><tr><td rowspan=1 colspan=1>TWT1</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT4</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT33</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT45</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT C1</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT C2</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT C3</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TWT C4</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>100%</td></tr></table>

b. Matrix comparison:

Not applicable. This test is only for use with human whole blood collected using EDTA as the anticoagulant.

3. Clinical studies:

a. Clinical Sensitivity: The clinical sensitivity can be estimated based on the published studies of mutation frequencies in various ethnicities.   
$b$ . Clinical specificity: The clinical specificity can be estimated based on published literature and based on the results of analytical studies described in this submission.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off:

Not applicable. 5. Expected values/Reference range: Cystic Fibrosis (CF) is the most common autosomal recessive disorder in the Caucasian population, with an incidence of approximately 1 in 3,200 live births. The incidence of CF in other ethnic groups varies, as seen in the following table.

Table 5. Incidence of Cystic Fibrosis in different Ethnic Groups   

<table><tr><td rowspan=1 colspan=1>Ethnic Group</td><td rowspan=1 colspan=1>Incidence of CF</td></tr><tr><td rowspan=1 colspan=1>North American Caucasian</td><td rowspan=1 colspan=1>1 in 3200</td></tr><tr><td rowspan=1 colspan=1>Ashkenazi Jewish</td><td rowspan=1 colspan=1>1 in 3300</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>1 in 9500</td></tr><tr><td rowspan=1 colspan=1>African American</td><td rowspan=1 colspan=1>1 in 15 300</td></tr><tr><td rowspan=1 colspan=1>Asian American</td><td rowspan=1 colspan=1>1 in 32 100</td></tr><tr><td rowspan=1 colspan=1>Native American (Pueblo)</td><td rowspan=1 colspan=1>1 in 3970</td></tr><tr><td rowspan=1 colspan=1>Native American (Zuni)</td><td rowspan=1 colspan=1>1 in 1347</td></tr></table>

Table 6. CFTR mutation frequency among individuals with clinically diagnosed cystic fibrosis by racial/ethnic group and in a pan-ethnic U.S. population.   

<table><tr><td colspan="7">Mutation frequencies among individuals with clinically diagnosed cystic</td></tr><tr><td>2004 ACMG</td><td colspan="6">fibrosis (%)</td></tr><tr><td>recommended CFTR Core</td><td>Non- Hispanic</td><td>Hispanic</td><td>African</td><td>Asian</td><td>Ashkenazi</td><td>Pan-Ethnic</td></tr><tr><td>Mutations</td><td>Caucasian</td><td>American</td><td>American</td><td>American</td><td>Jewish</td><td>Population</td></tr><tr><td>delF508</td><td>72.42</td><td>54.38</td><td>44.07</td><td>38.95</td><td>31.41</td><td>66.31</td></tr><tr><td>G542X</td><td>2.28</td><td>5.1</td><td>1.45</td><td>0</td><td>7.55</td><td>2.64</td></tr><tr><td>W1282X</td><td>1.5</td><td>0.63</td><td>0.24</td><td>0</td><td>45.92</td><td>2.2</td></tr><tr><td>G551D</td><td>2.25</td><td>0.56</td><td>1.21</td><td>3.15</td><td>0.22</td><td>1.93</td></tr><tr><td>621+1G&gt;T</td><td>1.57</td><td>0.26</td><td>1.11</td><td>0</td><td>0</td><td>1.3</td></tr><tr><td>N1303K</td><td>1.27</td><td>1.66</td><td>0.35</td><td>0.76</td><td>2.78</td><td>1.27</td></tr><tr><td>R553X</td><td>0.87</td><td>2.81</td><td>2.32</td><td>0.76</td><td>0</td><td>1.21</td></tr><tr><td>delI507</td><td>0.88</td><td>0.68</td><td>1.87</td><td>0</td><td>0.22</td><td>0.9</td></tr><tr><td>3120+1G&gt;A</td><td>0.08</td><td>0.16</td><td>9.57</td><td>0</td><td>0.1</td><td>0.86</td></tr><tr><td>3849+10kbC&gt;T</td><td>0.58</td><td>1.57</td><td>0.17</td><td>5.31</td><td>4.77</td><td>0.85</td></tr><tr><td>R117H</td><td>0.7</td><td>0.11</td><td>0.06</td><td>0</td><td>0</td><td>0.54</td></tr><tr><td>1717-1G&gt;A</td><td>0.48</td><td>0.27</td><td>0.37</td><td>0</td><td>0.67</td><td>0.44</td></tr><tr><td>2789+5G&gt;A</td><td>0.48</td><td>0.16</td><td>0</td><td>0</td><td>0.1</td><td>0.38</td></tr><tr><td>R334W</td><td>0.14</td><td>1.78</td><td>0.49</td><td>0</td><td>0</td><td>0.37</td></tr><tr><td>R347P</td><td>0.45</td><td>0.16</td><td>0.06</td><td>0</td><td>0</td><td>0.36</td></tr><tr><td>711+1G&gt;T</td><td>0.43</td><td>0.23</td><td>0</td><td>0</td><td>0.1</td><td>0.35</td></tr><tr><td>R560T</td><td>0.38</td><td>0</td><td>0.17</td><td>0</td><td>0</td><td>0.3</td></tr><tr><td>R1162X</td><td>0.23</td><td>0.58</td><td>0.66</td><td>0</td><td>0</td><td>0.3</td></tr><tr><td>3659delC</td><td>0.34</td><td>0.13</td><td>0.06</td><td>0</td><td>0</td><td>0.28</td></tr><tr><td>A455E</td><td>0.34</td><td>0.05</td><td>0</td><td>0</td><td>0</td><td>0.26</td></tr><tr><td>G85E</td><td>0.29</td><td>0.23</td><td>0.12</td><td>0</td><td>0</td><td>0.26</td></tr><tr><td>2184delA</td><td>0.17</td><td>0.16</td><td>0.05</td><td>0</td><td>0.1</td><td>0.15</td></tr><tr><td>1898+1G&gt;A</td><td>0.16</td><td>0.05</td><td>0.06</td><td>0</td><td>0.1</td><td>0.13</td></tr></table>

# N. Software:

The InPlex™ CF CRS is an Excel $^ \mathrm { \textregistered }$ template created and validated using Microsoft Excel $^ \mathrm { \textregistered }$ 2000. Excel $^ \mathrm { \textregistered }$ formulas, functions, and Microsoft $^ \mathrm { \textregistered }$ Visual Basic $^ \mathrm { \textregistered }$ 6.0 (VBA) were used to create the software. Four worksheets are visible to the user: Mix Worksheet, Sample Placement, Executive Summary, and Sample Summary. The Mix Worksheet allows the user to calculate the volume of reagents needed for an individual run. The user may enter component lot numbers and expiration dates, operator name, run ID, and run date. All cells are locked except those where the user can enter information. The worksheet is password protected. The sample placement worksheet allows the user to label the locations of each sample on each card within a run. The Sample and Executive summaries display the results for each sample on a card for each mutation and a condensed display for the run, respectively. Results of the IVS-8 variants are only displayed if the sample is heterozygous or homozygous for the R117H mutation. The F508C variant and $2 1 8 3 \mathrm { A A } > \mathrm { G }$ mutation are assessed for all samples for the purposes of correctly calling the delF508, delI507, and 2184delA mutations, respectively, but not displayed. The software displays an “Invalid” result for any sample failing to meet the quality controls for result validity (e.g., equivocal, low signal, or increase gain).

# O. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# P. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.