# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k070675   
B. Purpose for Submission: Modifications of the cleared device by changing the specimen type from fresh frozen tissue to fresh tissue stored in a specific RNA preservative solution and XPrint software v1.33 to v1.40.   
C. Measurand: 70 gene expression profile   
D. Type of Test: Expression microarray Test service performed in a single laboratory in Agendia’s Amsterdam facility.   
E. Applicant: Agendia BV   
F. Proprietary and Established Names: MammaPrint $^ \mathrm { \textregistered }$

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.6040 Gene expression profiling test system for breast cancer prognosis   
2. Classification: Class II   
3. Product code: NYI, Classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer   
4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): MammaPrint $^ \mathrm { \textregistered }$ is a qualitative in vitro diagnostic test service, performed in a single laboratory, using the gene expression profile of fresh breast cancer tissue samples to assess a patients' risk for distant metastasis.

The test is performed for breast cancer patients who are less than 61 years old, with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The MammaPrint $^ \mathrm { \textregistered }$ result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s):

For prescription use only   
MammaPrint $^ \mathrm { \textregistered }$ is not intended for diagnosis, or to predict or detect response to therapy, or to help select the optimal therapy for patients.

4. Special instrument requirements:

Agilent 2100 Bioanalyzer: Serial number DE54700497 en DE24802382   
Agilent DNA microarray scanner: Serial number us22502555

Note: The scanner and bioanalyzer are components of this assay and are cleared only for this assay and not for any other application. In addition, clearance is only limited to the bioanalyzer and scanner with the serial numbers as specified above.

# I. Device Description:

The MammaPrint $^ \mathrm { \textregistered }$ test is performed and provided as a service by Agendia Laboratory. The test is a microarray based gene expression analysis of RNA extracted from breast tumor tissue. The test is a custom-designed array chip manufactured by Agilent Technologies using the Agilent oligonucleotide microarray platform which assesses the mRNA expression of the 70 genes in triplicate. The MammaPrint $^ \mathrm { \textregistered }$ microarray features eight 1900-feature subarrays per glass slide which can each be individually hybridized. Per subarray 232 reporter genes are printed in triplicate, including the 70 genes which make up the MammaPrint $^ \mathrm { \textregistered }$ prognostic profile. Each subarray additionally includes 915 normalization genes and 289 spots for hybridization and printing quality control.

The analysis is based on several processes: isolation of RNA from fresh tumor tissue sections, DNAse treatment of isolated RNA, linear amplification and labeling of DNAse treated RNA, cRNA purification, hybridization of the cRNA to the MammaPrint $^ \mathrm { \textregistered }$ microaray, scanning the MammaPrint $^ \mathrm { \textregistered }$ microarray and data acquisition (feature extraction), calculation and determination of the risk of recurrence in breast cancer patients.

The MammaPrint $^ \mathrm { \textregistered }$ analysis is designed to determine the gene activity of specific genes in a tissue sample compared to a reference standard. The result is an expression profile, or fingerprint, of the sample. The correlation of the sample expression profile to a template (the mean expression profile of 44 tumors with a known good clinical outcome) is calculated and the molecular profile of the sample is determined (Low Risk, High Risk, Low Risk Borderline, High Risk Borderline).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Agendia BV’s MammaPrint $^ \mathrm { \textregistered }$   
2. Predicate 510(k) number(s): k062694   
3. Comparison with predicate: The device is the same as the predicate, except for the sample collection method. The previous recommended sample collection method was to freeze the sample immediately after collection and then ship fresh frozen to Agendia. The current method uses RNAretain which is a non-toxic tissue storage reagent to stabilize and preserve cellular RNA and therefore, eliminates the need to freeze the samples immediately after harvesting. Samples are transported in RNAretain to Agendia BV for MammaPrint testing. Upon arrival at Agendia, samples are snap-frozen and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

K. Standard/Guidance Document Referenced (if applicable): None

The MammaPrint $^ \mathrm { \textregistered }$ service is a microarray based gene expression analysis of a tumor. Refer

to k062694 for detailed description.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

i. Reproducibility of multiple isolations starting from tissue sample collected in RNAretain: In order to determine the reproducibility of the MammaPrint $^ \mathrm { \textregistered }$ device process from tissue processing to the end result, five previously analyzed tumor samples (one borderline, two high risk and two low risk) were isolated in duplicate. Over multiple days, the ten isolations from five tumors were processed according to standard MammaPrint $^ \mathrm { \textregistered }$ protocols.

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Originalindex</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Index from firstisolation</td><td rowspan=1 colspan=1>Index fromsecond isolation</td></tr><tr><td rowspan=1 colspan=1>S1</td><td rowspan=1 colspan=1>0.376</td><td rowspan=1 colspan=1>High risk (borderline)</td><td rowspan=1 colspan=1>0.254</td><td rowspan=1 colspan=1>0.374</td></tr><tr><td rowspan=1 colspan=1>S2</td><td rowspan=1 colspan=1>0.608</td><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>0.564</td><td rowspan=1 colspan=1>0.553</td></tr><tr><td rowspan=1 colspan=1>S3</td><td rowspan=1 colspan=1>0.659</td><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>0.639</td><td rowspan=1 colspan=1>0.680</td></tr><tr><td rowspan=1 colspan=1>S4</td><td rowspan=1 colspan=1>-0.105</td><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>0.068</td><td rowspan=1 colspan=1>0.067</td></tr><tr><td rowspan=1 colspan=1>S5</td><td rowspan=1 colspan=1>-0.305</td><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>-0.171</td><td rowspan=1 colspan=1>-0.337</td></tr></table>

No statistically significant difference in MammaPrint $^ \mathrm { \textregistered }$ risk group assignment or MammaPrint $^ \mathrm { { \textregistered } }$ index between the two separate RNA isolations was observed.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Same as previous submission.

Stability of MammaPrint outcomes in RNAretain

To determine if shipment in RNAretain affects stability of MammaPrint outcomes, an experiment was performed to determine MammaPrint variance in a tumor of which RNAretain and frozen tissue was available.

A tumor was selected from which both immediately snap-frozen and RNAretain preserved sections were available. Both sections of the tumor had a similar tumor cell percentage and similar RNA quality. Both samples were labeled 5 times and hybridized on MammaPrint microarrays according to standard protocols. MammaPrint Indices were compared to determine if samples shipped in RNAretain have a greater stability than tumor sections which are immediately stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ after excision.

Results showed that the incorporation of both Cy5 and Cy3 were well above the Agendia quality control minimum threshold of 3.0. Incorporation was observed to be significantly higher for samples shipped in RNAretain. (Unpaired T-tests, $\mathrm { p { = } } 0 . 0 1 8$ and $\mathrm { p { = } } 0 . 0 0 1$ respectively). The variance in MammaPrint indices was smaller for samples that were stored in RNAretain compared to the samples that were frozen immediately (Stdev 0.022 vs. 0.042). An unpaired T-test of the MammaPrint index revealed no significant difference in the actual MammaPrint indices for the RNAretain and frozen samples $( \mathrm { p } { = } 0 . 2 4 )$ ). Based on these experiments, the stability in MammaPrint index is greater in samples stored in RNAretain then in samples that were immediately frozen. The difference in MammaPrint index between RNAretain and frozen tissue $( \Delta 0 . 0 2 7 )$ was within the previously determined acceptable limit of index variation.

d. Detection limit: Same as previous submission.   
e. Analytical specificity: Same as previous submission.   
f. Assay cut-off: Using Feature Extraction 8.5 and XPrint v1.40 software, the classification threshold is set at 0.415 (See 2c).

2. Comparison studies:

a. Method comparison with predicate device: The samples for this study were collected in 2003 as a pilot study for the Dutch Raster clinical trial sponsored by the Dutch Health Insurance Council where tissue would be shipped in RNAretain from 20 hospitals. One set consisted of 33 breast tumor samples of which one part of the sample was immediately snap-frozen in liquid nitrogen and stored at $- 7 0 \mathrm { { ^ \circ C } }$ , another part was stored in RNAretain for 3 to 5 days at room temperature and subsequently removed from the preservation solution, snap frozen and stored at $- 7 0 \mathrm { { \circ C } }$ . Another set comprised of 18 tumors of which two parts were available for research that were immediately snap frozen and stored at $- 7 0 \mathrm { { \circ C } }$ . RNA isolation and DNAse treatment were performed in this same period. HE stained section were re-examined by a pathologist to confirm invasive ductal carcinoma and sufficient tumor cell content. All samples were hybridized on MammaPrint microarrays, and passed all sample, labeling and hybridization QCs. Analysis was performed using Feature Extraction version 8.5 and XPrint version 1.40.

Results of MammaPrint indices of paired RNAretain and frozen samples are shown in Figure 1A. The median difference between the RNAretain preserved and the snapfrozen is 0.070. The Pearson correlation (0.94) and regression analysis indicate a high similarity $\mathrm { ~ R } ^ { 2 } = 0 . 9 0 )$ .

![](images/ef57cfe8921befba7e684bab84c9cf8fe9cdf55ee4321f13ffad04565e447995.jpg)  
Figure 1: Comparison of MammaPrint Indices between two samplings of the same tumor. A RNAretain-frozen; B frozen-frozen.

This finding is similar to the results of a series of tumors of which two frozen samples were available and were collected in the same time period (Figure 1B). The median difference in MammaPrint Index was 0.105. A comparison of the differences in both series (RNAretain-frozen vs. frozen-frozen) showed no significant difference (t-test, $p { = } 0 . 5 7 )$ indicating no variation is introduced by RNAretain.

![](images/61c214c3a17475dcd065cd52ed41f3d09efb530864627fa3629e5fa92e9ba189.jpg)  
Figure 2: A Passing and Bablok regression analysis on the RNAretain vs. frozen samples

b. Matrix comparison: As described above.

c. Comparison of XPrint software v1.40 and v1.33 Due to the update of the Feature Extraction (FE) software from version 7.5 to 8.5, the MammaPrint analysis software XPrint was updated from v1.33 to v1.40. To validate this new combination of FE v.8.5/XPrint v1.40 software, 504 samples from the intended use population were analyzed with both versions of the FE and XPrint software. Regression analysis showed that the samples analyzed with FE v8.5 had on average a slightly higher correlation to the XPrint v1.40 than previously observed with FE v7.5 and XPrint v1.33. The regression equation is shown below: XPrint v1.40 Index $= 1 . 0 6 7 5$ (XPrint v.1.33 Index) – 0.0116.

The classification threshold for the FE v8.5/XPrint v1.40 combination was set at 0.415 instead of 0.40 after recalibration of the MammaPrint index using the 78 samples from the 2002 Nature study. The new classification threshold was then validated using 307 samples from the TransBig study and results were analyzed in relation to time to distant metastasis (see $4 \mathrm { x } 4$ and 2x2 tables below).

<table><tr><td rowspan=1 colspan=7>Poor (&lt;5 year)</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=5>FE7.5/XPrint 1.33</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>FE8.5/XPrint1.40</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>High Risk Borderline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Low Risk Borderline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>57</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Good (&gt;5 year)</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=5>FE7.5/XPrint 1.33</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>FE8.5/XPrint1.40</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>High Risk Borderline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Low Risk Borderline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>250</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Poor (&lt;5 year)</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FE7.5/XPrint 1.33</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>FE8.5/XPrint 1.40</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>57</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Good (&gt;5 year)</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FE7.5/XPrint 1.33</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>FE8.5/XPrint 1.40</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>137</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>250</td></tr></table>

3. Clinical studies:

Same as previous submission.

a. Clinical Sensitivity:

Same as previous submission.

b. Clinical specificity: Same as previous submission.   
c. Other clinical supportive data (when a. and b. are not applicable): Same as previous submission.

4. Clinical cut-off: Same as Assay cut-off.

5. Expected values/Reference range: Same as previous submission.

N. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

O. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.