# 10(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k090901   
B. Purpose for Submission: New Device   
C. Measurand: Cystic Fibrosis Transmembrane Conductance Regulator (CFTR) Gene in human blood specimens   
D. Type of Test: Multiplex PCR amplification and exonuclease digestion, followed by genotyping by hybridization and electrochemical detection   
E. Applicant: Osmetech Molecular Diagnostics   
F. Proprietary and Established Names: eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test   
G. Regulatory Information: 1. Regulation section: 21 CFR $\$ 866.5900- C F \mathrm { T R }$ (cystic fibrosis transmembrane conductance regulator) gene mutation detection system 21CFR $\ S 8 6 2 . 2 5 7 0 -$ Instrument for Clinical Multiplex Test Systems 2. Classification: Class II 3. Product code: NUA; System, CFTR (cystic fibrosis transmembrane conductance regulator) gene mutation detection system NSU; Instrumentation for Clinical Multiplex Test Systems 4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): The eSensor $^ \mathrm { \textregistered }$ CF Genotyping Test is an in vitro diagnostic device used to simultaneously detect and identify a panel of mutations and variants in the cystic fibrosis transmembrane conductance regulator (CFTR) gene in genomic DNA samples isolated from human peripheral whole blood specimens. The panel includes mutations and variants recommended by the 2004 American College of Medical Genetics (ACMG). The eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test is a qualitative genotyping test that provides information intended to be used for cystic fibrosis carrier screening as recommended by ACMG and the 2005 American College of Obstetricians and Gynecologists (ACOG) for adults of reproductive age, as an aid in newborn screening for cystic fibrosis, and in confirmatory diagnostic testing for cystic fibrosis in newborns and children. The test is not indicated for use in fetal diagnostic or pre-implantation testing. This test is also not indicated for stand-alone diagnostic purposes and results should be used in conjunction with other available laboratory and clinical information.

2. Indication(s) for use:

Same as Intended Use.

3. Special conditions for use statement(s):

For professional use only.

The eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test does not identify the I506V, I507V, and F508C polymorphisms, thus in the case of $\Delta \mathrm { F } 5 0 8$ homozygosity, reflex testing by bi-directional DNA sequencing is recommended

The eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test does not identify all possible mutations in the CFTR gene, for example E60X, I148T, 1078delT, V520F, 2143delT, 3199del6, D1152H, 3876delA, 2183AA>G, R560K, R117L, R347H, G551S, $7 1 1 + 5 \mathrm { G } { > } \mathrm { A }$ , 394delTT,and 3905insT. A negative result for all the mutations in this panel does not necessarily indicate that the individual does not carry mutations in CFTR gene (carrier or affected status).

The mutations included in this test represent $58 0 \%$ of the mutations carried by Caucasian American adults.

4. Special instrument requirements: eSensor $^ \mathrm { \textregistered }$ XT-8 Instrument system (k073720)

# I. Device Description:

The eSensor $^ \mathrm { \textregistered }$ CF Genotyping Test consists of 3 foil bags (each containing 8 test cartridges) and one vial of each of the following components: CF Genotyping test PCR mix $( 1 1 0 0 \mu \mathrm { L } )$ , Taq polymerase $( 8 0 \mu \mathrm { L } )$ , exonuclease $( 1 4 5 \mu \mathrm { L } )$ ), exonuclease dilution buffer $( 9 0 0 \mu \mathrm { L } )$ , CF genotyping test signal buffer $( 2 5 0 0 ~ \mu \mathrm { L } )$ , XT Buffer-1 $( 3 5 0 ~ \mu \mathrm { L } )$ , and XT Buffer-2 $( 7 0 0 \mu \mathrm { L } )$ .

# J. Substantial Equivalence Information:

1. Predicate device name(s): eSensor $^ \mathrm { \textregistered }$ Cystic Fibrosis Carrier Detection System   
2. Predicate 510(k) number(s): k051435 and k060543   
3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Cystic Fibrosis gene mutations</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Genomic DNA isolated from human EDTAwhole blood</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Number ofmutations/variantsdetected</td><td colspan="1" rowspan="1">23 mutations [ACMG (2004)/ACOG (2005)]recommended; IVS-8 polymorphisms(5T/7T/9T)</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Methodology</td><td colspan="1" rowspan="1">Multiplex PCR, exonuclease digestion,hybridization and electrochemical detection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Alternating current voltammetry</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reference Method</td><td colspan="1" rowspan="1">Bi-directional sequencing</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/Indicationsfor Use</td><td colspan="1" rowspan="1">The eSensor® CF GenotypingTest is an in vitro diagnosticdevice used to simultaneouslydetect and identify a panel ofmutations and variants in thecystic fibrosis transmembraneconductance regulator (CFTR)gene in genomic DNA samplesisolated from human peripheralwhole blood specimens. The panelincludes mutations and variantsrecommended by the 2004American College of MedicalGenetics (ACMG). The eSensor®CF Genotyping Test is aqualitative genotyping test thatprovides information intended tobe used for cystic fibrosis carrierscreening as recommended byACMG and the 2005 AmericanCollege of Obstetricians andGynecologists (ACOG) for adultsof reproductive age, as an aid innewborn screening for cysticfibrosis, and in confirmatorydiagnostic testing for cysticfibrosis in newborns and children.The test is not indicated for use infetal diagnostic or preimplantationtesting. This test is also notindicated for stand-alonediagnostic purposes and resultsshould be used in conjunction withother available laboratory andclinical information.</td><td colspan="1" rowspan="1">The eSensor® Cystic FibrosisCarrier Detection (CFCD)System is a device for thedetection of carrier status forcystic fibrosis for all adultcouples contemplatingpregnancy, regardless ofethnicity. It is a qualitativegenotyping assay thatsimultaneously detects mutationscurrently recommended by theAmerican College of MedicalGenetics and American Collegeof Obstetricians and Gynecol-ogists (ACMG/ ACOG). TheeSensor® CFCD System is notindicated for prenatal screeningor to establish a diagnosis forcystic fibrosis, and is for Rx onlyprofessional use within theconfines of a licensed laboratory,as defined by the ClinicalLaboratory ImprovementAmendments (CLIA) of 1988.</td></tr><tr><td colspan="1" rowspan="1">Instrumentsystem</td><td colspan="1" rowspan="1">eSensor® XT-8 Instrument</td><td colspan="1" rowspan="1">eSensor® Instrument Model4800</td></tr><tr><td colspan="1" rowspan="1">Software</td><td colspan="1" rowspan="1">eSensor® XT-8 Instrumentsoftware with Assay AnalysisModule (AAM). CF Genotypingtest AAM v. 1.0.1</td><td colspan="1" rowspan="1">eSensorTM DNA Detection SystemApplication software</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CFTR Gene Mutation Detection Systems - Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document

# L. Test Principle:

The OMD eSensor $^ \mathrm { \textregistered }$ CF Genotyping Test is a multiplex microarray-based genotyping test system. It is based on the principles of competitive DNA hybridization using a sandwich assay format, wherein a single-stranded target binds concurrently to sequence-specific solution-phase signal probe and solid-phase electrode-bound capture probe. The test employs polymerase chain reaction amplification, exonuclease digestion and hybridization of target DNA. In the process, the double stranded PCR amplicons are digested with exonuclease to generate single stranded DNA suitable for hybridization. Hybridization occurs in the eSensor $\textsuperscript { \textregistered }$ XT-8 Cartridge (described below) where the single-stranded target DNA is mixed with a hybridization solution containing labeled signal probes.

During hybridization, the target DNA binds to a complementary, single-stranded capture probe immobilized on the working electrode surface. Single-stranded signal probes (labeled with electrochemically active ferrocenes) bind to the target adjacent to the capture probe. When inserted into the eSensor $\textsuperscript { \textregistered }$ XT-8 instrument (described below), simultaneous hybridization of target to signal probes and capture probe is detected by alternating current voltammetry (ACV). Each pair of working electrodes on the array contains a different capture probe, and sequential analysis of each electrode allows genotyping of multiple mutations or polymorphisms.

M. Performance Characteristics:

1. Analytical performance: a. Precision/Reproducibility: The reproducibility study panel consisted of twenty-two (22) genomic DNA (gDNA) samples obtained from commercially available cultured cell lines, representing the 23 recommended ACOG/ACMG panel mutations and the 5/7/9T polymorphism. For practical considerations, the 22 samples were divided into 2 sets of 11 samples, so that the sample set was run in duplicate to evaluated intra-assay reproducibility using a single kit. Only one lot of reagents was used for this study. A total of 1320 samples (30,360 genotype calls) were tested, consisting of sixty (60) replicates (3 sites X 2 operators X 5 days X 2 replicates) for each of the 22 gDNA samples. DNA concentrations ranged from 6.43 to $1 2 . 7 3 ~ \mathrm { n g / \mu L }$ (32.2-63.7 ng/sample), which is within the $1 0 { - } 5 0 0 ~ \mathrm { n g / }$ sample range recommended in the package insert. $\mathsf { A } _ { 2 6 0 / 2 8 0 }$ ratios ranged from 1.52-2.84. Two additional runs were performed for any test which gave a “no-call” result. No miscalls were observed on first pass or after repeat testing. Thirty-six samples generated no-calls on the first pass were called successfully upon repeat testing. The data were evaluated after first-pass results and following additional runs for no-calls. All samples gave $100 \%$ agreement with DNA sequencing after repeat testing. There were 36/1320 $( 2 . 7 \% )$ per sample first pass no-calls and zero miscalls. The data were also evaluated after first-pass results on a per SNP basis and the correct call rate was 1980/2040, $9 7 \%$ agreement $( 9 6 . 3 7 \%$ , $9 5 \% \mathrm { L C B } )$ for Positive (Mutation) Calls and 27552/28320 and $9 7 \%$ agreement $( 9 7 . 1 2 \%$ , $9 5 \%$ LCB) for Negative (Wild-Type) Calls.

Among 36 no-calls, three were attributed to poor PCR amplification, four were attributed to an increased amount of mutant signal relative to WT, nine due to cartridge pumping, nine due to XT-8 instrument slot pumping, one due to invalid controls and three were attributed to operator error in set up of the PCR and seven were attributed to operator error in set-up of the exonuclease reaction (6 due to lack of addition of exonuclease and one due to insufficient mixing). All no-call samples gave correct calls after additional testing. The reproducibility results by sample, by genotype, and site/operator are shown in Tables 1, 2, and 3, respectively.

Table 1. Summary of Reproducibility Results by Sample Genotype and Site.   

<table><tr><td rowspan=3 colspan=1>Sample Genotype bySequencing</td><td rowspan=3 colspan=1>5/7/9T</td><td rowspan=3 colspan=1>#Replicatestested ateach site</td><td rowspan=1 colspan=4>Number of CF Sample CallsBefore Repeat Testing</td><td rowspan=1 colspan=4>Number of CF Sample CallsAfter Repeat Testing</td></tr><tr><td rowspan=1 colspan=3>Correct calls</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=3>Correct calls</td><td rowspan=2 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>SiteB</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A/ΔF508</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2184delA/ΔF508</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A/621+1G&gt;T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A/2789+5G&gt;A</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3659delC/ΔF508</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>3849+10KbC&gt;T/ΔF508</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T/G85E</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T/621+1G&gt;T</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A455E/621+1G&gt;T</td><td rowspan=1 colspan=1>9T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>ΔI507</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>G551D/R347P</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R117H/ΔF508</td><td rowspan=1 colspan=1>5T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R553X/G551D</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R560T/ΔF508</td><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>5T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>WT</td><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>R117H/ΔF508</td><td rowspan=1 colspan=1>5T/9T</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Totals</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>1304</td><td rowspan=1 colspan=1>1314</td><td rowspan=1 colspan=1>1306</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>0</td></tr></table>

Table 2. Summary of Reproducibility Results by Genotype and Site.   

<table><tr><td colspan="1" rowspan="4">Genotype bysequencing</td><td colspan="1" rowspan="3">#Repli-cates</td><td colspan="3" rowspan="1"># CF GT calls be</td><td colspan="3" rowspan="1"></td><td colspan="6" rowspan="1"># CF GT Calls after repeat testing</td></tr><tr><td colspan="3" rowspan="2">Correct Calls per site</td><td colspan="1" rowspan="2"># No</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="2">95%</td><td colspan="3" rowspan="2"># samples beingrepeated per site</td><td colspan="1" rowspan="2">CorrectCalls</td><td colspan="1" rowspan="2">%Agree</td><td colspan="1" rowspan="3">95%LCB</td></tr><tr><td colspan="1" rowspan="1">Agree</td></tr><tr><td colspan="1" rowspan="1">per site</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">Calls</td><td colspan="1" rowspan="1">-ment</td><td colspan="1" rowspan="1">LCB</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">per site</td><td colspan="1" rowspan="1">-ment</td></tr><tr><td colspan="1" rowspan="1">delta F 508</td><td colspan="1" rowspan="1">140</td><td colspan="1" rowspan="1">135</td><td colspan="1" rowspan="1">138</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">94.84</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">140</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.94</td></tr><tr><td colspan="1" rowspan="1">G542X</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">97.50</td><td colspan="1" rowspan="1">93.67</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.32</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">97.08</td><td colspan="1" rowspan="1">94.59</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">98.15</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R553X</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.33</td><td colspan="1" rowspan="1">92.34</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">delta I507</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">95.13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">3849+10kbC&gt;T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">95.00</td><td colspan="1" rowspan="1">87.58</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">3120+1G&gt;T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R117H</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">99.17</td><td colspan="1" rowspan="1">96.11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96.32</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">95.00</td><td colspan="1" rowspan="1">87.58</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R347P</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">711+1G&gt;T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.33</td><td colspan="1" rowspan="1">92.34</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R560T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.33</td><td colspan="1" rowspan="1">92.34</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">98.33</td><td colspan="1" rowspan="1">92.34</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">A455E</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">G85E</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">2184delA</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">93.33</td><td colspan="1" rowspan="1">85.39</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">1898+1G&gt;A</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">95.00</td><td colspan="1" rowspan="1">87.58</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">IVS8-5T</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">98.33</td><td colspan="1" rowspan="1">95.75</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">97.53</td></tr><tr><td colspan="1" rowspan="1">IVS8-7T</td><td colspan="1" rowspan="1">380</td><td colspan="1" rowspan="1">365</td><td colspan="1" rowspan="1">375</td><td colspan="1" rowspan="1">368</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">97.19</td><td colspan="1" rowspan="1">96.25</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">380</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99.61</td></tr><tr><td colspan="1" rowspan="1">IVS8-9T</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">96.67</td><td colspan="1" rowspan="1">89.88</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1">I507V</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">I506V</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">F508C</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Wild Type</td><td colspan="1" rowspan="1">9440</td><td colspan="1" rowspan="1">9100</td><td colspan="1" rowspan="1">9311</td><td colspan="1" rowspan="1">9141</td><td colspan="1" rowspan="1">768</td><td colspan="1" rowspan="1">97.29</td><td colspan="1" rowspan="1">97.12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9440</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">92.78</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total w/ PolyT</td><td colspan="1" rowspan="1">10580</td><td colspan="1" rowspan="1">10195</td><td colspan="1" rowspan="1">9468</td><td colspan="1" rowspan="1">10063</td><td colspan="1" rowspan="1">865</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">10580</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total w/o PolyT</td><td colspan="1" rowspan="1">10120</td><td colspan="1" rowspan="1">9752</td><td colspan="1" rowspan="1">9468</td><td colspan="1" rowspan="1">9618</td><td colspan="1" rowspan="1">828</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">10120</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

Table 3. Summary of Reproducibility Results by Site and Operator.   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>Samplestested</td><td rowspan=1 colspan=1>First passcorrect calls</td><td rowspan=1 colspan=1>First passNo Calls</td><td rowspan=1 colspan=1>First pass %No-calls</td><td rowspan=1 colspan=1>Final Correctcalls</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>1284</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>1320</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Lot-to-Lot Reproducibility

The same genomic DNA samples covering all possible genotypes, but not including a wild type sample were tested using three different kit lots at a single site. A single nocall was observed with one lot but was resolved upon repeat testing. No miscalls were observed either before or after repeat testing. The data were evaluated after first-pass results and following additional run for a no-call. All samples gave $100 \%$ correct calls when compared with DNA sequencing.

# Genomic DNA Extraction Reproducibility

A total of 20 independent whole blood samples of different heterozygous genotypes (6 ΔF508 7T/9T, 1 R117H 7T/9T and 2 with 7T/7T, 1 G551D 5T/7T, and 8 WT 7T/7T and 2 with 5T/7T) were extracted by three commonly used extraction methods and tested using the eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test. The data were evaluated after first-pass results and following additional runs to resolve the six no-calls observed. After repeat testing, all samples gave $100 \%$ correct calls when compared with DNA sequencing.

b. Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods): The reference method is bi-directional sequencing. A minimum acceptance criteria for Phred scores was ${ > } 2 0$ , which represents a $9 9 \%$ confidence of accuracy.

# Assay Controls:

Each test contains internal positive and negative controls to assure proper functioning of the system: Failure of either control will be indicated on the summary line of the report as “Invalid Control(s)” and the identity of the failed control will be given in the test results section of the report. The genotyping test results will not be reported for any sample for which a positive or negative control failure occurs.

Positive Control: Each cartridge contains a capture probe that is complementary to a synthetic target DNA present in the hybridization mixture. The target also contains ferrocene label and thus generates an appropriate signal in the assay. The positive control is designed to detect a systematic failure of the hybridization and/or detection processes. A lack of signal for the positive control indicates a genotyping assay failure. If a correct signal is observed for the positive control, but one or more genotyping assays are invalid due to low signals, then a failure of DNA isolation or PCR amplification or Exonuclease digestion is indicated.

Negative Control: A negative control is present on each cartridge, consisting of a capture probe that does not hybridize to any sequence within the target DNA or signal probes. Signals on the negative control indicate an assay system failure.

DNA Contamination Monitor (DCM): It is recommended that a blank PCR using nuclease-free water instead of DNA template be performed and included with each PCR run. This DCM sample is then processed with the other samples in the run, and a DNA Contamination Monitor Report is reviewed for this sample. The severity of contamination is determined by the System, and reported as a Contamination Metric with a value between 0.0 and 1.0.

# Stability:

Six stability studies were performed which included Kit stability, open pouch, hybridization solution on cartridge, single and double stranded DNA, and reagent pack freeze/thaw stability. Real-time stability studies currently support a three month shelf life. The stability studies for the components and processes performed supports the stability claims shown in Table 4 below.

Table 4. Summary of Assay component stability.   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Shelf Life</td></tr><tr><td rowspan=1 colspan=1>Once opened cartridge Pouch*</td><td rowspan=1 colspan=1>Up to 14 days 30°C and humidity (&lt;15%)</td></tr><tr><td rowspan=1 colspan=1>Freeze-thaw stability of all reagents (reagents canbe thawed up to 3 times</td><td rowspan=1 colspan=1>Up to 1 month @-20°</td></tr><tr><td rowspan=1 colspan=1>Sample hybridization stability in cartridge loadingreservoir</td><td rowspan=1 colspan=1>Up to 8 hours@ 10-25°C</td></tr><tr><td rowspan=1 colspan=1>Double-stranded &amp; single-stranded (exonuclease-digested) amplicon</td><td rowspan=1 colspan=1>Up to 7 days @ 4°C or -20°</td></tr></table>

\*Open pouches should not be exposed to high humidity $( \sim 8 0 \% )$ to prevent possibility of miscalls.

# d. Detection limit:

Two genomic DNA samples of different genotypes (Wild Type and $\Delta \mathrm { F } 5 0 8$ heterozygous) were extracted from whole blood stored in EDTA and serially diluted and tested 20 times each at input amounts of 0.01, 0.1, 1, 10, 100, 500 and 1000 ng/PCR using the eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test. Absorbance ratios (260/280) were 1.88 and 1.89 from the Wild Type and ΔF508 heterozygous samples respectively. An additional run was performed for tests that gave a first pass no call result. The number of first pass correct calls for the 0.01 ng sample for the WT and $\Delta \mathrm { F } 5 0 8$ sample was 5 and 1, respectively. The $0 . 0 1 \mathrm { n g } \Delta \mathrm { F } 5 0 8$ sample provided 18 no calls and 1 miscall. The miscalled sample gave a 9T/9T call for a 9T/7T sample. No miscalls were observed with the WT sample.

All input amounts from 0.01 ng to $1 0 0 0 \mathrm { n g }$ for both samples gave equivalent final performance ( $100 \%$ Agreement with $9 8 . 7 6 \%$ , $9 5 \%$ LCB), $( 9 9 . 9 5 \%$ LCB on a per SNP basis). The limit of detection was established as the lowest concentration at which no-calls or miscalls were obtained after repeat testing. The lower detection limit was determined to be $0 . 1 { \mathrm { n g } }$ of purified DNA per reaction and the upper detection limit was determined to be $1 0 0 0 \mathrm { n g }$ of purified DNA per reaction. The recommended range of DNA input amounts for the eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test is from 10 to 500 ng. A summary of the results is shown in Table 5 below.

Table 5. Summary of Limit of Detection study results.   

<table><tr><td rowspan=3 colspan=1>TotalgDNAinput (ng)</td><td rowspan=1 colspan=4>WT</td><td rowspan=1 colspan=4>ΔF508 Heterozygous</td></tr><tr><td rowspan=1 colspan=2>First Pass Calls</td><td rowspan=1 colspan=2>Final calls afterrepeat testing</td><td rowspan=1 colspan=2>First Pass Calls</td><td rowspan=1 colspan=2>Final calls afterrepeat testing</td></tr><tr><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Correct</td><td rowspan=1 colspan=1>No Call</td></tr><tr><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td></tr></table>

e. Analytical specificity:

The following interfering substances were added separately to seven whole blood (4 $\Delta \mathrm { F } 5 0 8$ heterozygous, 2 R117H heterozygous, and 1 G551D heterozygous) samples at the concentrations indicated, and no effect was observed on yield of extracted DNA, multiplex amplification of CFTR gene sequences, or genotyping of mutations in the CF carrier screening panel: triglycerides $( 5 0 0 \mathrm { m g / d L } )$ ), conjugated bilirubin (30 $\mathrm { m g / d L ) }$ ), hemoglobin $( { \sim } 2 0 \ \mathrm { g / d L } )$ , and short draw EDTA (at a concentration equivalent to 5-fold higher than that provided in a standard EDTA blood collection tube).

The $2 1 8 3 \mathrm { A A } > \mathrm { G }$ and R117L were evaluated and demonstrated not to impact the results of the eSensor $^ \mathrm { \textregistered }$ CF Genotyping Test. When present in the same region as panel mutations, they may interfere with genotyping results. Two whole blood samples with the $2 1 8 3 \mathrm { A A } > \mathrm { G }$ mutation (N1303K/2183AA>G, and 2183AA $>$ G/WT) and 1 with the R117L (G85E/R117L) mutation were tested as part of the method comparison study and shown not to interfere with the calling of the 2184delA and R117H mutations. Two samples containing the benign variants near the ΔF508 mutation which consisted of a compound heterozygote sample $\Delta \mathrm { F } 5 0 8 / \mathrm { I } 5 0 7 \mathrm { V }$ and $3 8 4 9 + 1 0 \mathrm { K b }$ $\mathrm { C } >$ T/F508C were tested as part of the method comparison study and were called correctly. Both samples were correctly called as heterozygous for $\Delta \mathrm { F } 5 0 8$ and $3 8 4 9 + 1 0 \mathrm { K b } \mathrm { C } >$ $\mathrm { C } { > } \mathrm { T }$ , respectively.

f. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison with predicate device:

A total of 112 whole blood and previously banked samples isolated from whole blood were used in this study. There were 162 positive calls from the 112 samples which consisted of heterozygous, homozygous, and compound heterozygous for the panel mutations. One mutant sample each for R334W, $2 7 8 9 + 5 \mathrm { G } { > } \mathrm { A }$ , ΔF508 and 5T were included in this study.

Table 6. Method Comparison Samples by mutation.   

<table><tr><td colspan="1" rowspan="2">Genotype bysequencing</td><td colspan="1" rowspan="2">Pan-EthnicCFTRMutation Freq.</td><td colspan="2" rowspan="1"># Indep Clinical Samples</td><td colspan="2" rowspan="1">Minimum # Total Samples2</td></tr><tr><td colspan="1" rowspan="1">Required byFDA</td><td colspan="1" rowspan="1">Used inthis study</td><td colspan="1" rowspan="1">Required byFDA</td><td colspan="1" rowspan="1">Used in thisstudy</td></tr><tr><td colspan="1" rowspan="1">ΔF508</td><td colspan="1" rowspan="1">66.31</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">47</td></tr><tr><td colspan="1" rowspan="1">G542X</td><td colspan="1" rowspan="1">2.64</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">1.93</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">R553X</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">ΔI507</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">3120+1G&gt;A</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">3849+10kbC&gt;T</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">R117H</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;A</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">R347P</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">711+1G&gt;T</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">R560T</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">A455E</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">G85E</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">2184delA</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">1898+1G&gt;A</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">109</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">156</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Reflex Polymorphisms</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">IVS8-5T</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">IVS8-7T</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">102</td></tr><tr><td colspan="1" rowspan="1">IVS8-9T</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Polymorphisms not specifically genotyped</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">I507V</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">F508C</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="3" rowspan="1">Potentially interfering mutations not part of assay panel</td><td colspan="2" rowspan="1">ns not part of assay panel</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">2183AA&gt;G</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">R117L</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">4Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">112</td><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">162</td></tr></table>

1 Independent clinical samples consisted of whole blood samples or archived gDNA from whole blood obtained from different individuals. For compound Heterozygous samples, a sample is counted only once, based on fulfilling the per mutation sample requirement first. For number of independent samples for 5/7/9T, 5T includes 5T/5T, 5T/7T and 5T/9T; 7T includes 7T/7T and 7T/9T and 9T includes only 9T/9T samples

2 Samples include independent clinical samples or gDNA obtained from cultured cell lines. For total samples for 5/7/9T, 5T includes 5T/5T, 5T/7T and 5T/9T; 7T includes 5T/7T, 7T/7T and 7T/9T and 9T includes 5T/9T, 7T/9T and 9T/9T samples   
3 One(1) sample was compound HET for G85E and R117L, which is counted as an independent sample for G85E   
4 "Used in this study" sample total includes one(1) 5T/5T sample

Results were based on a total of 112 calls per mutation yielding 2688 total mutation calls in this study. All three samples with the non-ACMG panel mutations (potential interfering mutations) were correctly called as WT. The samples containing the nonACMG mutations consisted of heterozygous or compound heterozygotes for G85E/R117L, N1303K/2183AA>G, and 2183AA $>$ G/WT. All were correctly called as heterozygous and was correctly called as heterozygous for the mutation contained in the assay or as wild type. The two samples containing the benign variants near the polymorphisms not specifically genotyped, and the potentially interfering mutations not part of assay panel. I507V, F508C, $2 { \cal I } 8 3 { \cal A } { \cal A } > { \cal G } ,$ , and $R I I 7 L$ are non-panel polymorphism containing samples correctly called as WildType, which are not included in the grand total for calls. No samples with I506V were tested.

ΔF508 mutation, consisted of a compound heterozygote sample ΔF508/I507V and $3 8 4 9 + 1 0 \mathrm { K b }$ $\mathrm { C } >$ T/F508C. Both samples were correctly called as heterozygous for ΔF508 and $3 8 4 9 + 1 0 \mathrm { K b }$ $\mathrm { C } { > } \mathrm { T }$ , respectively. The resulting positive agreement with biTable 7. Clinical sample results compared to bi-directional sequencing.   

<table><tr><td rowspan=2 colspan=1>Genotype bysequencing</td><td rowspan=1 colspan=2>SequencingCalls</td><td rowspan=1 colspan=3>1st Pass CF GTCalls</td><td rowspan=2 colspan=1>%Agree-ment</td><td rowspan=2 colspan=1>95%LCB</td><td rowspan=1 colspan=3>FINAL CF GTCalls</td><td rowspan=2 colspan=1>%Agree-ment</td><td rowspan=2 colspan=1>95%LCB</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>NoCalls</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>NoCalls</td></tr><tr><td rowspan=1 colspan=1>ΔF508</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>90.3</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>93.8</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>85.7</td><td rowspan=1 colspan=1>47.9</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>65.2</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>60.7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>60.7</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>65.2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>65.2</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td></tr><tr><td rowspan=1 colspan=1>R553X</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>ΔI507</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36.8</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>3849+10kbC&gt;T</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54.9</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>R347P</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>R560T</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>68.8</td></tr><tr><td rowspan=1 colspan=1>3659delC</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>A455E</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>65.2</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>65.2</td></tr><tr><td rowspan=1 colspan=1>2184delA</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.47</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>Totals</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>2420</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>2420</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>2420</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Reflex Polymorphism</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>IVS-8 5T Variant ()</td><td rowspan=1 colspan=1>6$</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>6$r}$</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>60.7</td></tr><tr><td rowspan=1 colspan=1>IVS-8 7T Variant (§)</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>92.6</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97.1</td></tr><tr><td rowspan=1 colspan=1>IVS-8 9T Variant (§)</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>89.3</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.8</td></tr><tr><td rowspan=1 colspan=7>Polymorphisms not specifically genotyped</td><td rowspan=1 colspan=6></td></tr><tr><td rowspan=1 colspan=1>1507V</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>F508C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=9>Potentially interfering mutations not part of assay panel</td><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=1>2183AA&gt;G</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22.4</td></tr><tr><td rowspan=1 colspan=1>R117L</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>Grand Total</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>2526</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>2526</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>162</td><td rowspan=1 colspan=1>2526</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.1</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>V5T elin</td><td rowspan=1 colspan=1>ieal al</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>anle Or</td><td rowspan=1 colspan=1>e aided 0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n eenfi</td><td rowspan=1 colspan=1>lenee b</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

$\nmid$ indicates that there is one 5T/5T clinical gDNA sample, One-sided lower confidence bound) $\ S$ For the purpose of the IVS8-5T/7T/9T Variant, "Positive" samples are regarded as those that have at least one copy of the 5T allele while "Negative" samples are regarded as having only the 7T and/or 9T allele The number of positive sequencing calls is greater than the number of independent samples due to the inclusion of compound heterozygous samples. The Grand Total consists of the total number of sequencing calls for mutations, the 1 5T/5T sample included,

directional sequencing, including variant and non-ACMG mutations was $9 7 . 5 \%$ $( 9 4 . 3 \% \mathrm { L C B } _ { 9 5 \% } )$ for first pass and $100 \%$ $( 9 8 . 1 \% \mathrm { L C B } _ { 9 5 \% } )$ after a maximum of 2 repeat tests on the no-call samples. The total number of IVS-8 sample calls are shown by genotype and from the method comparison study before and after repeat testing (single repeat test allowed) are shown in Tables 8a and 8b, below.

Table 8a. Total number of clinical IVS-8 samples.   

<table><tr><td rowspan=1 colspan=1>Genotypebysequencing</td><td rowspan=1 colspan=1># ofIndependentsamples</td><td rowspan=1 colspan=3>Per Sample GT Calls</td><td rowspan=1 colspan=3>Total GT Calls</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5T</td><td rowspan=1 colspan=1>7T</td><td rowspan=1 colspan=1>9T</td><td rowspan=1 colspan=1>5T</td><td rowspan=1 colspan=1>7T</td><td rowspan=1 colspan=1>9T</td></tr><tr><td rowspan=1 colspan=1>5T/5T</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5T/7T</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>5T/9T</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>9T/9T</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Totals</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>70</td></tr></table>

Table 8b. Clinical IVS-8 sample results compared to bi-directional sequencing.   

<table><tr><td rowspan=2 colspan=1>Genotypebysequencing</td><td rowspan=2 colspan=1># ofsamples(gDNA)</td><td rowspan=1 colspan=5>Number of CF GT Calls Before RepeatTesting</td><td rowspan=1 colspan=5>Number of CF GT Calls After Repeat Testing</td></tr><tr><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoCalls</td><td rowspan=1 colspan=1>MissedCalls</td><td rowspan=1 colspan=1>%Agree-ment</td><td rowspan=1 colspan=1>95%LCB</td><td rowspan=1 colspan=1>Correctcalls</td><td rowspan=1 colspan=1># NoCalls</td><td rowspan=1 colspan=1>MissedCalls</td><td rowspan=1 colspan=1>% Agree-ment</td><td rowspan=1 colspan=1>95%LCB</td></tr><tr><td rowspan=1 colspan=1>5T/5T</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>5T/7T</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>47.3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>47.3</td></tr><tr><td rowspan=1 colspan=1>5T/9T</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>7T/7T</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>92.2</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>92.2</td></tr><tr><td rowspan=1 colspan=1>7T/9T</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>95.1</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>95.2</td></tr><tr><td rowspan=1 colspan=1>9T/9T</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>68.8</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>68.8</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>93.4</td></tr></table>

$b$ . Matrix comparison: Not applicable. This test is only for use with human whole blood collected using EDTA as the anticoagulant.

3. Clinical studies:

a. Clinical Sensitivity: The clinical sensitivity can be estimated based on the published studies of mutation frequencies in various ethnicities and based on the results of analytical studies described in this submission.

$b$ . Clinical specificity:

The clinical specificity can be estimated based on published literature and based on the results of analytical studies described in this submission.

c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.   
5. Expected values/Reference range: Cystic Fibrosis (CF) is the most common autosomal recessive disorder in the Caucasian population, with an incidence of approximately 1 in 3,200 live births. The incidence of CF in other ethnic groups varies, as seen in the following tables.

Table 9. Incidence of Cystic Fibrosis in different Ethnic Groups.   

<table><tr><td rowspan=1 colspan=1>Ethnic Group</td><td rowspan=1 colspan=1>Incidence of CF</td></tr><tr><td rowspan=1 colspan=1>North American Caucasian</td><td rowspan=1 colspan=1>1 in 3200</td></tr><tr><td rowspan=1 colspan=1>Ashkenazi Jewish</td><td rowspan=1 colspan=1>1 in 3300</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>1 in 9500</td></tr><tr><td rowspan=1 colspan=1>African American</td><td rowspan=1 colspan=1>1 in 15 300</td></tr><tr><td rowspan=1 colspan=1>Asian American</td><td rowspan=1 colspan=1>1 in 32 100</td></tr><tr><td rowspan=1 colspan=1>Native American (Pueblo)</td><td rowspan=1 colspan=1>1 in 3970</td></tr><tr><td rowspan=1 colspan=1>Native American (Zuni)</td><td rowspan=1 colspan=1>1 in 1347</td></tr></table>

Table 10. CFTR mutation frequency among individuals with clinically diagnosed cystic fibrosis by racial/ethnic group and in a pan-ethnic U.S. population.   

<table><tr><td colspan="1" rowspan="2">2004 ACMG/2005 ACOGrecommendedCFTR CoreMutations</td><td colspan="6" rowspan="1">Mutation frequencies among individuals with clinically diagnosed cysticfibrosis (%)</td></tr><tr><td colspan="1" rowspan="1">Non-HispanicCaucasian</td><td colspan="1" rowspan="1">HispanicAmerican</td><td colspan="1" rowspan="1">AfricanAmerican</td><td colspan="1" rowspan="1">AsianAmerican</td><td colspan="1" rowspan="1">AshkenaziJewish</td><td colspan="1" rowspan="1">Pan-EthnicPopulation</td></tr><tr><td colspan="1" rowspan="1">delF508</td><td colspan="1" rowspan="1">72.42</td><td colspan="1" rowspan="1">54.38</td><td colspan="1" rowspan="1">44.07</td><td colspan="1" rowspan="1">38.95</td><td colspan="1" rowspan="1">31.41</td><td colspan="1" rowspan="1">66.31</td></tr><tr><td colspan="1" rowspan="1">G542X</td><td colspan="1" rowspan="1">2.28</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">1.45</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7.55</td><td colspan="1" rowspan="1">2.64</td></tr><tr><td colspan="1" rowspan="1">W1282X</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">45.92</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">G551D</td><td colspan="1" rowspan="1">2.25</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.21</td><td colspan="1" rowspan="1">3.15</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">1.93</td></tr><tr><td colspan="1" rowspan="1">621+1G&gt;T</td><td colspan="1" rowspan="1">1.57</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">N1303K</td><td colspan="1" rowspan="1">1.27</td><td colspan="1" rowspan="1">1.66</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">2.78</td><td colspan="1" rowspan="1">1.27</td></tr><tr><td colspan="1" rowspan="1">R553X</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">2.81</td><td colspan="1" rowspan="1">2.32</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1.21</td></tr><tr><td colspan="1" rowspan="1">delI507</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">1.87</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.9</td></tr><tr><td colspan="1" rowspan="1">3120+1G&gt;A</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">9.57</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.86</td></tr><tr><td colspan="1" rowspan="1">3849+10kbC&gt;T</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">1.57</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">5.31</td><td colspan="1" rowspan="1">4.77</td><td colspan="1" rowspan="1">0.85</td></tr><tr><td colspan="1" rowspan="1">R117H</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.54</td></tr><tr><td colspan="1" rowspan="1">1717-1G&gt;A</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">0.44</td></tr><tr><td colspan="1" rowspan="1">2789+5G&gt;A</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.38</td></tr><tr><td colspan="1" rowspan="1">R334W</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">1.78</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="1">R347P</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.36</td></tr><tr><td colspan="1" rowspan="1">711+1G&gt;T</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.35</td></tr><tr><td colspan="1" rowspan="1">R560T</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">R1162X</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">3659delC</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.28</td></tr><tr><td colspan="1" rowspan="1">A455E</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">G85E</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.26</td></tr><tr><td colspan="1" rowspan="1">2184delA</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.15</td></tr><tr><td colspan="1" rowspan="1">1898+1G&gt;A</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.13</td></tr></table>

N. Instrument Name:

eSensor® XT-8 Instrument

# O. System Descriptions:

See k073720 decision summary for description of the eSensor® XT-8 instrument description, principles and mode of operation.

1. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

2. Specimen Identification:

The sponsor’s submitted software documentation demonstrated that the software design meets the stated requirements for this device and were verified and validated in part by testing the system with known wild type, mutant, and negative control samples determining that signal detection configurations result in correct call/results.

3. Specimen Sampling and Handling:

Sample identification is performed through entering in the sample ID and the barcode of the test cartridge to be used with the sample. The barcode of the cartridge is scanned as the sample is about to be run.

4. Calibration: See k073720.

5. Quality Control:

The eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test on the XT-8 Instrument uses a combination of electronic and molecular controls and validity criteria to verify the proper operation of the system and assure the delivery of accurate results. The eSensor $\textsuperscript { \textregistered }$ CF Genotyping Test provides a system of controls for proper instrument setup and function, preparation of reagents, handling of samples, and function of the cartridge and reagents. The criteria for control and test results comprise consistent and stringent test for reporting of results. Furthermore, results from control tests provide useful information to troubleshoot test failures and to identify the potential root cause(s) and corrective action(s). A defect in product or process occurring at any step of the process is expected to cause a failed control or a signal which does not meet the criteria for a valid test result. In any of these cases, no result is reported and further troubleshooting and/or a repeat of the test is performed.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.