# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k091960   
B. Purpose for Submission: New device   
C. Measurand: Early Growth Response 1 (EGR1) gene deletion   
D. Type of Test: Fluorescence in situ hybridization (FISH)   
E. Applicant: Abbott Molecular, Inc.   
F. Proprietary and Established Names: Vysis EGR1 FISH Probe Kit

# G. Regulatory Information:

2. Classification: Class II

3. Product code: OWK, Early growth response 1 (EGR1) FISH probe kit

4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): The Vysis EGR1 FISH Probe Kit is intended to detect deletion of the LSI EGR1 probe target on chromosome 5 in bone marrow specimens and to be used, in addition to cytogenetics, other biomarkers, morphology and other clinical information, at the time of acute myeloid leukemia (AML) diagnosis as an aid in determining prognosis. Deletion of chromosome 5q has been associated with an unfavorable prognosis in AML patients.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Fluorescence microscope equipped with appropriate excitation and emission filters.

# I. Device Description:

The Vysis EGR1 FISH Probe Kit consists of two DNA FISH probes and four general purpose reagents as follows:

1. Vysis LSI EGR1 SpectrumOrange and D5S23, D5S721 SpectrumGreen probes a. The SpectrumOrange-labeled LSI EGR1 probe is approximately $2 0 9 ~ \mathrm { k b }$ in length (chr5: 137682107-137890637: March 2006 Assembly; University of

California at Santa Cruz (UCSC) Human Genome Browser) is located at 5q31 and contains the complete EGR1 gene.

b. The SpectrumGreen-labeled LSI D5S23, D5S721 probe is approximately 561 kb in length (chr5:9450109-10011407: March 2006 Assembly; UCSC Human Genome Browser) is located at 5p15.2.

2. General Purpose Reagents

a. Vysis LSI/WCP Hybridization Buffer   
b. DAPI II Counterstain   
c. NP-40   
d. 20X SSC Salt

# J. Substantial Equivalence Information:

1. Predicate device name(s) and 510(k) number(s): Vysis Chronic Lymphocytic Leukemia (CLL) FISH Probe Kit (k100015)

2. Comparison with predicate:

Similarities   

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate (k100015)</td></tr><tr><td rowspan=1 colspan=1>IntendedUse</td><td rowspan=1 colspan=1>Determination of deletion status asan aid in determining prognosis</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>FISH</td><td rowspan=1 colspan=1>Same</td></tr></table>

Differences   

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate (k100015)</td></tr><tr><td rowspan=1 colspan=1>PatientPopulation</td><td rowspan=1 colspan=1>Acute myeloid leukemia (AML)patients</td><td rowspan=1 colspan=1>Chronic lymphocyticleukemia (CLL) patients</td></tr><tr><td rowspan=1 colspan=1>ProbeTargets</td><td rowspan=1 colspan=1>LSI EGR1</td><td rowspan=1 colspan=1>LSI TP53LSI ATMLSI D13S319CEP 12</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

American College of Medical Genetics, “Standards and Guidelines for Clinical Genetics Laboratories”, 2008

# L. Test Principle:

Deletion of chromosome 5q as detected by cytogenetics is a recurring abnormality in AML. The early growth response 1 (EGR1) gene is among the genes in a commonly deleted segment on chromosome band 5q31. The Vysis EGR1 FISH Probe Kit uses FISH DNA probe technology to determine the deletion status of probe targets in AML bone marrow specimens for LSI EGR1, D5S23 and D5S721 on chromosome 5.

Bone marrow specimens from AML patients are attached to microscope slides using standard cytogenetic procedures. The resulting specimen DNA is denatured to singlestranded form and subsequently allowed to hybridize with the probes of the EGR1 FISH Probe Kit. Following hybridization, the unbound probe is removed by a series of washes, and the nuclei are counterstained with DAPI, a DNA-specific stain that fluoresces blue. Hybridization of the LSI EGR1 SpectrumOrange and D5S23, D5S721 SpectrumGreen probes is viewed using a fluorescence microscope equipped with appropriate excitation and emission filters, allow visualization of the orange and

green fluorescent signals.

In a cell with normal copy numbers of the LSI EGR1 SpectrumOrange and D5S23, D5S721 SpectrumGreen probe targets, two SpectrumOrange signals (LSI EGR1) and two SpectrumGreen signals (LSI D5S23, D5S721) will be expected.

In a cell with the 5q deletion, one SpectrumOrange signal (LSI EGR1) and two SpectrumGreen signals (LSI D5S23, D5S721) will be expected. Enumeration of the orange LSI EGR1 and green LSI D5S23, D5S721 signals provide a mechanism for determining absolute copy number of the probe targets and the presence of the aberration of interest.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility: Precision and Reproducibility were tested at three sites on five different days. Two replicates of two normal specimens (patients that do not contain either -5 or 5q- based on prior karyotype or FISH results) and 4 abnormal specimens (2 low positive ( $1 2 . 0 \%$ of cells positive for 5q-) and 2 high positive $( \ge 4 5 . 0 \%$ of cells positive for 5q-)) were tested. Low positive specimens were targeted at two times the normal EGR1 cutoff of $6 . 0 \%$ . Positive specimens were obtained by mixing positive bone marrow cells with normal bone marrow cells to obtain the desired levels of abnormality. The mean and the standard deviation (SD) of the percentage of cells with the 1R2G signal pattern was calculated.

i. Precision:

Lot-to-Lot: Using the same specimens from the site-to-site study, four replicates of the 2 high positive, 2 low positive, and 2 normal specimens were tested with 3 different lots of probe at a single site. All replicates using the three probe lots for each of the 6 specimens produced agreement with the known status of the specimens.

Within-Day and Between Day: Using the same samples from the site-tosite study, two replicates of on the 2 high positive, 2 low positive, and 2 normal specimens were run at each site on the same day and between 5 non-consecutive days. Analysis of Variance results are presented below for all three sites combined:

<table><tr><td rowspan=2 colspan=1>Sample Type</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Meana</td><td rowspan=1 colspan=1>Within-DayComponent</td><td rowspan=1 colspan=1>Between-DayComponent</td></tr><tr><td rowspan=1 colspan=1>SDb</td><td rowspan=1 colspan=1>SDb</td></tr><tr><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>3.28</td><td rowspan=1 colspan=1>4.01</td></tr><tr><td rowspan=1 colspan=1>High Positive2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>5.56</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>LowPositive 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>3.82</td></tr><tr><td rowspan=1 colspan=1>Low Positive 2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>3.25</td><td rowspan=1 colspan=1>1.54</td></tr><tr><td rowspan=1 colspan=1>Normal 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Normal 2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>.66</td><td rowspan=1 colspan=1>1.42</td></tr></table>

a Percentage of cells with 1R2G signal pattern ${ } ^ { \mathrm { b } } \operatorname { S D } =$ standard deviation

ii. Reproducibility

Results showed $100 \%$ agreement among the test sites for the high and low positive samples and $9 8 \%$ agreement for the normal samples among the three testing sites. The following tables summarize results of the site analyses.

Site-to-Site Agreement   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Agree</td><td rowspan=1 colspan=1>Disagree</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>98</td></tr></table>

Between Site Analysis of Variance   

<table><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Meana</td><td rowspan=1 colspan=1>SDb</td></tr><tr><td rowspan=1 colspan=1>High Positive 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>5.44</td></tr><tr><td rowspan=1 colspan=1>High Positive 2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>0.74</td></tr><tr><td rowspan=1 colspan=1>Low Positive 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>1.03</td></tr><tr><td rowspan=1 colspan=1>Low Positive 2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Normal 1</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>.99</td></tr><tr><td rowspan=1 colspan=1>Normal 2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.5</td></tr></table>

a Percentage of cells with 1R2G signal pattern ${ } ^ { \mathrm { b } } \operatorname { S D } =$ standard deviation

$b$ . Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): i. Real-time stability: The functionality of the Vysis EGR1 FISH Probe kit components was evaluated based on subjective rating of the following attributes: signal intensity, target background, cross-hybridization, specificity and overall readability in three lots. The Vysis EGR1 FISH Probe Kit dating is determined by the component with the shortest expiration dating. The shelf-life of Vysis EGR1 FISH Probe Kit was determined to be 12 months.

<table><tr><td rowspan=1 colspan=1>Components</td><td rowspan=1 colspan=1>Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>Vysis LSI EGR1SpectrumOrange/D5S23, D5S721SpectrumGreen Probe</td><td rowspan=1 colspan=1>-20°C, protected from light</td></tr><tr><td rowspan=1 colspan=1>Vysis LSI/WCP HybridizationBuffer</td><td rowspan=1 colspan=1>-20°C, protected from light</td></tr><tr><td rowspan=1 colspan=1>DAPI II Counterstain</td><td rowspan=1 colspan=1>-20°C, protected from light</td></tr><tr><td rowspan=1 colspan=1>NP-40</td><td rowspan=1 colspan=1>-20°C ± 10°Cand roomtemperature (15-30°C)</td></tr><tr><td rowspan=1 colspan=1>20X SSC</td><td rowspan=1 colspan=1>-20°C ± 10°C and roomtemperature (15-30°C)</td></tr></table>

ii Freeze-thaw Stability: A series of 20 freeze-thaw cycles was performed on the probes, hybridization buffer and DAPI II counterstain. The functionality of these components was evaluated based on subjective rating of the following attributes: signal intensity, target background, cross-hybridization, specificity and overall readability in one lot. Twenty freeze-thaw cycles were found to be acceptable for this kit.

iii. Transport and Temperature Extreme Stability. The functionality of the Vysis EGR1 FISH Probe kit components was evaluated based on subjective rating of the following attributes: signal intensity, target background, cross-hybridization, specificity and overall readability in one lot. Components were removed from ${ } _ { - 2 0 } \mathrm { { \circ C } }$ and cycled for 48 hours on dry ice, 72 hours at $2 5 ^ { \circ } \mathrm { C }$ , 72 hours at $4 0 ^ { \circ } \mathrm { C }$ , and ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for 24 hours prior to testing. There was no change in device performance under these stress conditions.

iv. Post-hybridization Signal Stability. A single slide from each of three bone marrow specimens was tested at baseline and then stored at $- 2 0 ^ { \circ } \mathrm { C } \pm 1 0 ^ { \circ } \mathrm { C }$ while protected from light. The samples were tested at three time points (Day 7, Day 14, and Day 25) using the following attributes: nuclear morphology, background, signal intensity and overall readability in one lot. The post-hybridization signal stability was found to be three weeks.

d. Detection limit:

The analytical sensitivity of the Vysis LSI EGR1 SpectrumOrange D5S23, D5S721 SpectrumGreen probes was established using interphase nuclei prepared from 25 bone marrow specimens that were either karyotypically normal or 5p15 and 5q31 deletion-free. The orange and green signal patterns of nuclei for 25 specimens were evaluated by two technologists. Each technologist evaluated 100 nuclei per specimen for a total of 200 nuclei per specimen and 5000 scoreable nuclei from normal specimens. The analytical sensitivity was calculated as the percentage of scoreable interphase nuclei with the expected $2 \mathrm { r e d } / 2$ green signal pattern.

<table><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>Total Number ofNuclei Scored</td><td rowspan=1 colspan=1>Number of NucleiWith ExpectedSignal Pattern</td><td rowspan=1 colspan=1>Analytical Sensitivity(95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Vysis LSI EGR1/D5S23, D5S721</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>4979</td><td rowspan=1 colspan=1>99.6%(99.4% -99.7%)</td></tr></table>

e. Analytical specificity:

The analytical specificity of the Vysis LSI EGR1 SpectrumOrange and D5S23, D5S721 SpectrumGreen probes for their respective chromosome target loci was established using metaphase chromosomes prepared from peripheral blood cultures of five karyotypically normal males that were pooled prior to dropping on microscope slides. The hybridization location of each FISH signal on chromosomes of 100 consecutive metaphase nuclei was evaluated by one technologist for a total of 200 target loci. For each probe and sample, the number of metaphase chromosome FISH signals hybridized to the correct locus and the number of metaphase chromosome FISH signals hybridized to the incorrect locus were enumerated. The analytical specificity of each probe was calculated as the percentage of metaphase chromosome FISH signals hybridized to the correct locus as follows:

<table><tr><td rowspan=1 colspan=1>Probe</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Total No. ofMetaphaseChromosomeHybridized</td><td rowspan=1 colspan=1>No. ofMetaphaseChromosomeHybridized tothe CorrectTarget</td><td rowspan=1 colspan=1>AnalyticalSpecificity(95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Vysis LSI EGR1SpectrumOrange</td><td rowspan=1 colspan=1>5q31</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100%(98%- 100%)</td></tr><tr><td rowspan=1 colspan=1>Vysis LSI D5S23,D5S721SpectrumGreen</td><td rowspan=1 colspan=1>5p15.2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100%(98% - 100%)</td></tr></table>

f. Assay cut-off:

The normal cut-off value is defined as the maximum quantity of scoreable interphase nuclei with an abnormal signal pattern at which a specimen is considered normal for that signal pattern. The normal cut-off value is expressed in terms of a percentage of the actual number of nuclei with a specific abnormal FISH signal pattern per the standard number of nuclei tested.

The normal cut-off value for this assay is $6 \%$ or 12 1red/2green (1R2G) patterns per 200 scoreable interphase nuclei. Specimens exceeding 12 1R2G patterns per 200 scoreable nuclei are considered abnormal for deletion of the Vysis LSI EGR1 probe target. This $6 \%$ normal cut-off value was adopted from the publication Vance et al3 who utilized the Vysis LSI EGR1/D5S23,D5S721 probe set in a study that established agreement between cytogenetics and FISH in 183 bone marrow specimens.

In order to validate the $6 \%$ normal cut-off of the Vysis EGR1 FISH Probe Kit, the assay was performed on interphase nuclei from 25 bone marrow specimens from either karyotypically normal specimens or 5p15.2 and 5q31 deletion-free specimens. The signal patterns of 200 nuclei were evaluated by each of two technologists who evaluated 100 nuclei per specimen. Among the 25 normal specimens, none produced 1R2G signals at or above the $6 \%$ normal cut-off.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity: Not applicable.

b. Clinical specificity:

Not applicable.

c. Other clinical supportive data (when a. and b. are not applicable): The clinical utility of the cytogenetic detection of the deletion of chromosome 5q is correlated with reduced 5-year overall survival in studies by Byrd et al1 and Grimwade et al2 . The Byrd publication demonstrated the prognostic value in 86 patients with a $- 5 / 5 { \mathrm { q } } \cdot$ - abnormality by a median overall survival (OS) of 0.3 years compared to the median OS of 1.3 years associated with patients exhibiting a normal karyotype. Patients with a -5/5q- abnormality had a 5-year OS of $6 \%$ compared to a 5-year OS of $24 \%$ associated with patients exhibiting a normal karyotype. Patients with a -5/5q- abnormality also had a significantly lower complete remission (CR) rate of $31 \%$ than th e normal karyotype, which resulted in a CR of $6 8 \%$ with a $\mathrm { p } { < } 0 . 0 0 1$ . For del(5q), 42 patients had a 5-year OS of $5 \%$ , $9 5 \%$ CI $( 2 \mathrm { - } 1 3 \% )$ and median overall surviv al of 0.3 years.

Grimwade et al, showed prognostic value in 28 patients for 5q-. Cytogenetic abnormalities in the Medical Research Council (MRC) AML 10 clinical trial for del(5q) patients showed 5-year OS of $1 1 \%$ and CR of $5 7 \%$ in the adverse risk group. These values varied significantly from the no abnormality, or normal karyotype group, which had 5-year OS of $42 \%$ and CR of $8 8 \%$ $\mathrm { ( p { < } 0 . 0 0 1 ) }$ ).

The publication “Utility of interphase FISH to stratify patients into cytogenetic risk categories at diagnosis of AML in an Eastern Cooperative Oncology Group (ECOG) clinical trial $\left( \mathrm { E } 1 9 0 0 \right) ^ { \gamma }$ by Vance et $\mathrm { a l } ^ { 3 }$ , establishes linkage between cytogenetic results and the Vysis EGR1 FISH Probe kit. When 183 bone marrow specimens were compared to cytogenetic results at the $56 \%$ cut-off, there was overall agreement of $9 8 . 9 1 \%$ $( 9 5 \% \mathrm { C I } { \cdot } 9 6 . \% )$ - $9 9 . 7 0 \%$ ), negative percent agreement of $100 \%$ ( $9 5 \%$ CI- $9 7 . 8 3 \% - 1 0 0 . 0 0 \% )$ and positive percent agreement of $80 \%$ $9 5 \% \mathrm { C I } { - } 4 9 . 0 2 \% { - } 9 4 . 3 4 \% )$ . Results are presented below:

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Karyotype -5/del5q</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>FISH (1R2G-5q deletionsignal pattern)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2a,b</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>181</td></tr></table>

a Cytogenetic result was -5/del(5q). FISH signal pattern was $44 \%$ 1R1G (monosomy of chromosome 5). b Cytogenetic result was -5/del(5q). FISH signal pattern was $1 \%$ 1R2G. False negative results.

4. Clinical cut-off: Not applicable.   
5. Expected values/Reference range: Not applicable.

N. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

The submitted information in this premarket notification is complete and support substantial equivalence decision.