# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k101454   
B. Purpose for Submission: Adding two additional scanners, two bioanalyzers and one physical location for additional testing service   
C. Measurand: 70 gene expression profile   
D. Type of Test: Expression microarray Test service performed in Agendia’s two central laboratories: Amsterdam and Huntington Beach, CA.   
E. Applicant: Agendia BV   
F. Proprietary and Established Names: MammaPrint $^ \mathrm { \textregistered }$

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.6040 Gene expression profiling test system for breast cancer prognosis   
2. Classification: Class II   
3. Product code: NYI, Classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer   
4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): MammaPrint $^ \mathrm { \textregistered }$ is a qualitative in vitro diagnostic test service, performed in a central laboratory, using the gene expression profile of fresh breast cancer tissue samples to assess a patients' risk for distant metastasis (up to 10 years for patients less than 61 years old, up to 5 years for patients $\geq 6 1$ years).

The test is performed for breast cancer patients, with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The MammaPrint $^ \mathrm { \textregistered }$ result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s):

For prescription use only   
MammaPrint $^ \mathrm { \textregistered }$ is not intended for diagnosis, or to predict or detect response to therapy, or to help select the optimal therapy for patients.

4. Special instrument requirements:

Agilent 2100 Bioanalyzer: Serial number DE54700497, DE24802382, DE72901757, and   
DE72902383   
Agilent DNA microarray scanner: Serial numbers US22502555, US45103019,   
US810R3210, and US811R3213

Note: The scanners and Bio-analyzers are components of this assay and are cleared only for this assay and not for any other application. In addition, clearance is only limited to the bioanalyzers and scanners with the serial numbers as specified above.

# I. Device Description:

The MammaPrint $^ \mathrm { \textregistered }$ test is performed and provided as a service by Agendia’s two central Laboratories. The test is a microarray based gene expression analysis of RNA extracted from breast tumor tissue. The test is a custom-designed array chip manufactured by Agilent Technologies using the Agilent oligonucleotide microarray platform which assesses the mRNA expression of the 70 genes printed in nine-fold.

The analysis is based on several processes: isolation of RNA from fresh tumor tissue sections, DNAse treatment of isolated RNA, linear amplification and labeling of DNAse treated RNA, cRNA purification, hybridization of the cRNA to the MammaPrint $^ \mathrm { \textregistered }$ microarray, scanning the MammaPrint $^ \mathrm { { \textregistered } }$ microarray and data acquisition (feature extraction), calculation and determination of the risk of recurrence in breast cancer patients.

The MammaPrint $^ \mathrm { \textregistered }$ analysis is designed to determine the gene activity of specific genes in a tissue sample compared to a reference standard. The result is an expression profile, or fingerprint, of the sample. The correlation of the sample expression profile to a template (the mean expression profile of 44 tumors with a known good clinical outcome) is calculated and the molecular profile of the sample is determined (Low Risk, High Risk, Low Risk Borderline, High Risk Borderline).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Agendia BV’s MammaPrint $^ \mathrm { \textregistered }$   
2. Predicate 510(k) number(s): k062694, k070675, k080252, k081092   
3. Comparison with predicate: The device is the same as the predicate.

K. Standard/Guidance Document Referenced (if applicable): None

L. Test Principle: The MammaPrint $^ \mathrm { \textregistered }$ service is a microarray based gene expression analysis of breast tumor tissue. Refer to $\mathrm { k } 0 6 2 6 9 4$ for detailed description.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: Since the device is the same as the predicate device, please see the analytical performance data from k062694, k070675 and k080252. a. Precision/Reproducibility: Same as previous submission.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): Same as previous submission.   
d. Detection limit: Same as previous submission.   
e. Analytical specificity: Same as previous submission.   
f. Assay cut-off: Same as previous submission.

2. Comparison studies:

a. Method comparison with predicate device:

i. Comparison of new scanners to previously cleared scanners This study used a total of 100 samples (included samples with either high, low or borderline results). For each sample, 2 hybridizations are performed: straight and dye swap. $1 ^ { \mathrm { s t } }$ scan was generated on all slides using previously FDA cleared scanners (serial US45103019 and US22502555). 2nd scan was generated using new scanners (US810R3210 and US811R3213). Additionally, control samples LRC and HRC were included. MammaPrint indices were compared between both scans using Passing and Bablok regression analysis and a comparison of the variance per scanner. Results of the Passing and Bablok regression analysis are summarized in Tables 1 and 2 below. The $9 5 \%$ confidence interval of the slope contains the value 1 and of the intercept B contains the value 0, indicating the new and the FDA cleared scanners are equivalent.

Table 1: Results of Passing and Bablok regression analysis FDA cleared scanner (US22502555) vs. New scanner (US810R3210)   

<table><tr><td colspan="3">Equation: y = 0.000 + 1.0000 x</td></tr><tr><td>Intercept B</td><td></td><td>0.0000</td></tr><tr><td>95% CI</td><td></td><td>-0.0104 to 0.0006716</td></tr><tr><td>Slope A</td><td></td><td>1.0000</td></tr><tr><td>95% CI</td><td></td><td>0.99983to 1.0040</td></tr></table>

Table 2: Results of Passing and Bablok regression analysis FDA cleared scanner (US45103019) vs. New scanner (US811R3213)   

<table><tr><td colspan="3">equation: y = -0.000956 + 0.9980 x</td></tr><tr><td>Intercept B</td><td></td><td>-0.000956</td></tr><tr><td>95% CI</td><td></td><td>-0.0010 to -0.0003548</td></tr><tr><td>Slope A</td><td></td><td>0.9980</td></tr><tr><td>95% CI</td><td></td><td>0.9952 to 1.0000</td></tr></table>

Results showed the difference between the mean, median and standard deviation for all samples levels between both scanners were within the accepted variance of the predicate device of $1 . 9 6 ^ { * } 0 . 0 3 0$ .

# McNemars test on MammaPrint Outcome

In the $2 \mathbf { x } 2$ contingency tables, the outcome comparison is shown for the FDA cleared scanner and the new scanner (Table 3 and Table 4). The McNemars test shows that there is no difference in the marginal row and column frequencies $( \mathrm { p } { = } 1 . 0 )$ ).

Table 3: $2 \mathbf { x } 2$ contingency table of the MammaPrint Outcome comparison between the FDA cleared scanner (US22502555) and New scanner (US810R3210)   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>FDA cleared scanner: US22502555</td></tr><tr><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>New scanner: US810R3210</td><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>73</td></tr><tr><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>103</td></tr></table>

Table 4: 2x2 contingency table of MammaPrint Outcome comparison between FDA cleared scanner (US45103019) and New scanner (US811R3213)   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=3>FDA cleared scanner: US45103019</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>New scanner: US811R3213</td><td rowspan=1 colspan=2>High risk</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=2>Low risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>100</td></tr></table>

The Kappa Score indicates high agreement in MammaPrint Outcome between both scanners $\scriptstyle \prime \kappa = 0 . 9 7 4$ , $9 5 \% C I$ : 0.922 – 1.025).

# Investigation of MammaPrint Outcome switchers

# Between the FDA cleared (US22502555) and New scanner (US810R3210)

There is one sample out of the 103 that has switched in MammaPrint outcome $( 0 . 9 7 \% )$ ; from low risk in the FDA cleared scanner dataset to high risk in the new scanner dataset (table 4).

The sample with switching outcome has a MammaPrint Index in the borderline region (0.380-0.450) and is very close to the MammaPrint threshold (0.415) with both scanners (Table 5).

Table 5:   
Overview of samples switched between FDA cleared scanner (US22502555)   
and New scanner (US810R3210)

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FDA cleared scanner:US22502555</td><td rowspan=1 colspan=2>New scanner:US810R3210</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lab Number</td><td rowspan=1 colspan=1>MammaPrintIndex</td><td rowspan=1 colspan=1>Outcome</td><td rowspan=1 colspan=1>MammaPrintIndex</td><td rowspan=1 colspan=1>Outcome</td><td rowspan=1 colspan=1>differen</td></tr><tr><td rowspan=1 colspan=1>09001529I1A1L1</td><td rowspan=1 colspan=1>0.418</td><td rowspan=1 colspan=1>low risk</td><td rowspan=1 colspan=1>0.409</td><td rowspan=1 colspan=1>high risk</td><td rowspan=1 colspan=1>0.00</td></tr></table>

Samples that lie within the borderline region and that are close to the threshold are more likely to switch in outcome. In a diagnostic setting these samples will be performed in duplicate in order to obtain better outcome accuracy.

$$
{ \begin{array} { r l r l } & { { \mathrm { P P A } } = 1 } & & { { \begin{array} { r l } & { 9 5 \% \mathrm { C I } = ( 0 . 9 3 7 - 1 ) } \\ & { { \mathrm { N P A } } = 0 . 9 6 8 } \end{array} } } & & { { \begin{array} { r l } & { 9 5 \% \mathrm { C I } = ( 0 . 8 1 5 - 0 . 9 9 8 ) } \end{array} } } \end{array} }
$$

# Between the FDA cleared (US45103019) and New scanner (US811R3213)

As shown in the 2x2 contingency table, there are two samples switching in MammaPrint Outcome between both scanners. These samples switched from a low risk with the FDA cleared scanner to a high risk using the New scanner.

Table 6: MammaPrint results on switching samples between FDA cleared scanner (US45103019) and New scanner (US811R3213)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FDA cleared scanner US45103019</td><td rowspan=1 colspan=2>New scanner US811R3213</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>MammaPrint Index</td><td rowspan=1 colspan=1>verdict</td><td rowspan=1 colspan=1>MammaPrint Index</td><td rowspan=1 colspan=1>verdict</td></tr><tr><td rowspan=1 colspan=1>09006606I2A1L1</td><td rowspan=1 colspan=1>0.423</td><td rowspan=1 colspan=1>low risk</td><td rowspan=1 colspan=1>0.415</td><td rowspan=1 colspan=1>high risk</td></tr><tr><td rowspan=1 colspan=1>09005722I1A1L1</td><td rowspan=1 colspan=1>0.418</td><td rowspan=1 colspan=1>low risk</td><td rowspan=1 colspan=2>0.415          high risk</td></tr></table>

However, the samples that switched in Outcome are well within the borderline region $( 0 . 3 8 0 - 0 . 4 5 0 )$ . The MammaPrint result of the New scanner lies exactly on the MammaPrint threshold (0.415).

Since these samples are within the borderline region and extremely close to the threshold it is very likely to switch in MammaPrint Outcome. In a diagnostic setting these samples will be performed in duplicate in order to obtain better outcome accuracy.

$$
{ \begin{array} { r l r l } & { { \mathrm { P P A } } = 1 } & & { { \mathrm { 9 5 \% ~ C I } } = ( 0 . 9 3 9 - 1 ) } \\ & { { \mathrm { N P A } } = 0 . 9 2 3 } & & { { \mathrm { 9 5 \% ~ C I } } = ( 0 . 7 3 4 - 0 . 9 8 7 ) } \end{array} }
$$

# MammaPrint Stability over time

Both control samples LRC2 and HRC2 were also tested to show MammaPrint Stability overtime on the New scanner and FDA cleared scanner. For each control samples 20 measurements were generated. The mean and standard deviation were determined for LRC2 and HRC2 for both scanners (Tables 7 and 8).

Table 7: Mean and standard deviation of LRC2 and HRC2 generated using the FDA cleared and New scanner (Between the FDA cleared (US45103019) and New scanner (US811R3213))   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>FDA cleared scanner(US495103019)</td><td rowspan=1 colspan=2>New scanner(US811R3213)</td></tr><tr><td rowspan=1 colspan=1>LRC2</td><td rowspan=1 colspan=1>HRC2</td><td rowspan=1 colspan=1>LRC2</td><td rowspan=1 colspan=1>HRC2</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>0.783</td><td rowspan=1 colspan=1>-0.524</td><td rowspan=1 colspan=1>0.748</td><td rowspan=1 colspan=1>-0.523</td></tr><tr><td rowspan=1 colspan=1>Stdev</td><td rowspan=1 colspan=1>0.016</td><td rowspan=1 colspan=1>0.029</td><td rowspan=1 colspan=1>0.016</td><td rowspan=1 colspan=1>0.027</td></tr></table>

Table 8: Mean and standard deviation for LRC and HRC generated with New scanner (Between FDA cleared scanner (US22502555) and the new scanner (US810R3210))   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>FDA cleared scanner(US22502555)</td><td rowspan=1 colspan=2>New scanner(US810R3210)</td></tr><tr><td rowspan=1 colspan=1>LRC</td><td rowspan=1 colspan=1>HRC</td><td rowspan=1 colspan=1>LRC</td><td rowspan=1 colspan=1>HRC</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>0.796</td><td rowspan=1 colspan=1>-0.528</td><td rowspan=1 colspan=1>0.794</td><td rowspan=1 colspan=1>-0.526</td></tr><tr><td rowspan=1 colspan=1>Stdev</td><td rowspan=1 colspan=1>0.025</td><td rowspan=1 colspan=1>0.023</td><td rowspan=1 colspan=1>0.024</td><td rowspan=1 colspan=1>0.025</td></tr></table>

The standard deviation of MammaPrint Index determined for the New scanner is comparable to the FDA cleared scanner. This indicates that MammaPrint Stability generated with the New scanner is comparable to the FDA cleared scanner.

ii. Comparison of new bioanalyzers to previously cleared bioanalyzers A selection of about 60 samples that cover the complete RNA Integrity Number (RIN) measuring range was analyzed on previously FDA cleared Bio-analyzer (Serial nr DE54700497), as well as the new Bio-analyzer (DE72902383). A selection of 59 samples that cover the complete RIN measuring range was analyzed on the FDA cleared Bio-analyzer (ID035/Serial nr. DE24802382) as well as the new Bio-analyzer (ID 132/Serial nr. DE72901757). Depending on the distribution of the data a statistical test was performed to determine if there is a significant difference in RIN measurements between both Bio-analyzers.

The RIN measurements of the samples on both Bio-Analyzers were collected and the D’Agostino-Pearson test on the RIN differences of both analyzers showed a normal distribution $\mathrm { ( p { < } 0 . 0 0 0 1 }$ ). A Wilcoxon signed ranks test was used which showed that there was no significant difference in RIN measurements between the FDA cleared and New Bio-analyzers $\mathrm { ( p { = } 0 . 4 6 }$ for Serial nr. DE72901757 and $\mathrm { p } { = } 0 . 4 7$ for DE72902383, respectively).

iii. Comparison of two central laboratories Validation of MammaPrint at the European (Amsterdam) and U.S. (Huntington Beach, HB) central laboratories was performed in two parts.

Samples were selected based on sufficient tissue material available for sectioning and isolation at the US laboratory (Lab 2). These samples have previously shown to generate acceptable quality of RNA at the Amsterdam laboratory (Lab 1). After isolation the concentration and RNA quality (RIN) was assessed using the Bio-analyzers; all values have to meet the standard quality controls for MammaPrint $( \mathrm { R I N } { > } 7 )$ ).

Isolations were performed on three different days, twelve samples each day, in total 36 samples.

![](images/d4b4de087896ebdf74250a400756b7d5e467f7ce39e99c788e20972587171635.jpg)  
Figure 1: A box plot representing measured RINs of isolations performed at both locations.   
Box plot of measured RIN of isolations performed at Amsterdam (Lab1) and US Huntington Beach (Lab2)

This plot indicates high similarity in RIN and RNA quality between Lab1 and Lab2.

# PART 2: AMPLIFICATION/LABELING AND HYBRIDIZATION

For validation of the labeling, amplification and hybridization steps of MammaPrint at the US lab (Lab 2), RNA from 99 samples was used. All samples have been previously subjected to a diagnostic MammaPrint test at the Amsterdam Lab (Lab 1). Based on the Amsterdam result the following result distribution was selected:

High risk: $\mathtt { n } { = } 5 4$ $( 5 4 . 5 \% )$ Low risk: $\mathrm { n } { = } 3 8$ $( 3 8 . 3 \% )$ Borderline: $\mathrm { n } { = } 7$ $( 7 . 1 \% )$

RNA was amplified, labeled and hybridized according to standard MammaPrint protocols on FDA cleared MammaPrint Low (HD) 8-pack array.

Along with 99 samples, the standard control samples (Low Risk Control and High Risk Control) were also analyzed. To show MammaPrint stability over time and to determine variation in MammaPrint Index, LRC and HRC were analyzed on each labeling day and on additional days resulting in 20 data points per control sample.

Statistical analyses that have been performed on the data are;

• Passing and Bablok regression analysis   
• Bland & Altman analysis   
• McNEMARS TEST   
• Analysis on Control Pools: LRC AND HRC

Table 9: Results of Passing and Bablok regression analysis   

<table><tr><td rowspan=1 colspan=2>y = 0.005849 + 0.9913 x</td></tr><tr><td rowspan=1 colspan=1>Intercept A</td><td rowspan=1 colspan=1>0.005849</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>-0.002651 to 0.009136</td></tr><tr><td rowspan=1 colspan=1>Slope B</td><td rowspan=1 colspan=1>0.9913</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>0.9605 to 1.0192</td></tr><tr><td rowspan=1 colspan=1>Cusum test for linearity</td><td rowspan=1 colspan=1>No significant deviation from linearity (P&gt;0.10)</td></tr></table>

The $9 5 \%$ confidence interval of the slope contains the value 1 and $9 5 \%$ confidence interval of the intercept contains the value 0. These results show that there is a high similarity in MammaPrint Indices generated at Amsterdam Lab and HB lab.

Table 10: Comparison between Amsterdam and Huntington Beach labs, positive percent agreement (PPA) and negative percent agreement (NPA) along with $9 5 \%$ CI (VR-TR-083).   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>AMSTERDAM LAB</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>High riskborderline</td><td rowspan=1 colspan=1>Low riskborderline</td><td rowspan=1 colspan=1>Low risk</td></tr><tr><td rowspan=4 colspan=1>Newscanner:HB LAB</td><td rowspan=1 colspan=1>High risk</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>High riskborderline</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Low riskborderline</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Low risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>99</td></tr></table>

$\mathrm { P P A } = 5 4 / 5 7 = 0 . 9 4 7$ $\mathrm { N P A } = 3 9 / 4 2 { = } 0 . 9 2 9$

$\mathrm { P P A } = 0 . 9 4 7$ $\begin{array} { r } { 9 5 \% \mathrm { C I } = ( 0 . 8 4 5 - 0 . 9 8 6 ) } \\ { 9 5 \% \mathrm { C I } = ( 0 . 7 9 4 - 0 . 9 8 1 ) } \end{array}$ $\mathrm { N P A } = 0 . 9 2 9$

One sample with “High Risk Borderline” index in Amsterdam Lab had “Low Risk” index in on HB lab; and one sample with “Low Risk” in Amsterdam lab had “High Risk” index in HB lab.

After investigation, there are no indications that these switchers are related to hybridization quality or result of location. All results passed the QC model. Important note; in regular MammaPrint diagnostics, borderline region samples are performed in duplicate in order to increase accuracy. This was not performed in this validation.

The standard deviation of both control samples LRC and HRC were determined for the HB lab (Lab 2) and compared to the standard deviation at the Amsterdam Lab (Lab 1). The accepted difference between both standard deviations is 0.059 $( 1 . 9 6 ^ { * } 0 . 0 3 )$ .

The studies show that there is no significant difference in RNA quality of RIN measurement between Amsterdam (L1) and US lab (L2). Moreover when comparing MammaPrint Index and Outcome, it is concluded that there is no significant difference in MammaPrint Indices between European Dutch (L1) and US California (L2) lab. All results were within the predefined validation acceptance criteria.

b. Matrix comparison: Not applicable.

3. Clinical studies: Same as previous submissions. a. Clinical Sensitivity: Same as previous submission. b. Clinical specificity: Same as previous submission. c. Other clinical supportive data (when a. and b. are not applicable): Same as previous submission.

4. Clinical cut-off: Same as Assay cut-off.

5. Expected values/Reference range: Same as previous submission.

N. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.