# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k101185   
B. Purpose for Submission: New device   
C. Measurand: Total Prostate specific antigen (tPSA)   
D. Type of Test: Quantitative, Immuno-PCR (Polymerase Chain Reaction)   
E. Applicant: Iris Molecular Diagnostics   
F. Proprietary and Established Names: NADiA® ProsVue™

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.6040, Gene expression profiling test system for breast cancer prognosis   
2. Classification: Class-II   
3. Product code: OWM – Prostate-specific antigen (PSA) for prognostic, recurrence risk assessment of prostate cancers   
4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): NADiA® ProsVue™ is an in-vitro diagnostic assay for determining rate of change of serum total prostate specific antigen over a period of time (slope, $\boldsymbol { \mathrm { p g / m L } }$ per month). The NADiA® ProsVue™ assay is performed for patients having less than $0 . 1 ~ \mathrm { n g / m L }$ serum total PSA values (determined by standard-of-care assays that are FDA approved/cleared) in the first sample collected more than 6 weeks after radical prostatectomy. ProsVue™ slope is indicated for use as a prognostic marker in conjunction with clinical evaluation as an aid in identifying those patients at reduced risk for recurrence of prostate cancer for the eight year period following prostatectomy.

The NADiA® ProsVue™ assay is not intended for the diagnosis or for the monitoring of prostate cancer.

2. Indication(s) for use:

Same as Intended use.

3. Special conditions for use statement(s): For prescription use only. Patient sample collection and testing schedule requirements:

ProsVue™ results are calculated as the linear slope of three NADiA® ProsVue™ total PSA test results obtained on three serum samples collected between six weeks and 20 months post-radical prostatectomy (RP).   
• All three samples from a single patient must be tested in a single NADiA® ProsVue™ assay run.   
• No results are to be reported to the physician until the ProsVue™ slope has been calculated from the three data points. ProsVue™ slope (not the individual PSA concentrations) is reported to the physician. The first serum sample for the NADiA® ProsVue™ assay is collected at least six weeks after the date of radical prostatectomy, frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ . The first serum sample must have a PSA value of $< 0 . 1 \ \mathrm { n g / m L }$ by a standard-ofcare (that is FDA approved/cleared) total PSA assay. The NADiA® ProsVue™ assay should not be used for patients with total PSA values greater than 0.1 $\mathrm { { n g / m L } }$ in the first serum sample. The date of radical prostatectomy must be sent to the laboratory along with the first ProsVue™ sample. The date of sample collection must be sent to the laboratory with each sample.   
• The second serum sample collection date must be at least two months after the first sample collection date.   
• The third serum sample collection must be completed within 10 to 20 months (no sooner than 10 months) of the date of radical prostatectomy, and at least two months after the second sample.   
• The first and second serum samples are not tested by NADiA® ProsVue™ assay at the time of collection. Rather, the two samples are stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ until the third sample is available for testing. Frozen samples are stable for at least 20 months.   
• To ensure the proper time intervals for sample collection, dates of sample collection are entered into the “Date/1st Sample PSA Validator” module of the ProsVue™ Software. Refer to ProsVue™ Software Directions for Use.

4. Special instrument requirements:

NADiA® ProsVue™ is intended to be used in conjunction with Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument (k082562) and the ProsVue Software.

# I. Device Description:

NADiA® ProsVue™ is a two-site immunoassay utilizing an assay specific synthetic DNA sequence as a label with a PCR detection method. The kit is made up of:

ProsVue Antibody Reagent Set includes: (1) Target Capture Reagent (biotinylated monoclonal anti-PSA murine antibody attached to streptavidin-coated paramagnetic microparticles, buffers, salts, surfactant and $0 . 0 9 \%$ sodium azide), and (2) Reporter Antibody Reagent (monoclonal PSA-specific murine antibody labeled with reporter DNA sequence, bovine serum albumin, murine $\mathrm { I g G }$ , buffers, salts, surfactant and $0 . 0 9 \%$ sodium azide). • ProsVue PCR Reagent includes: Taq DNA polymerase, dNTP’s, buffer, salts, SYBR $^ \mathrm { \textregistered }$ Green I, primers specific to the reporter DNA, reference dye and surfactant. Calibrator Set includes: Purified human prostate specific antigen $90 \%$ PSA–ACT $^ +$ $10 \%$ Free PSA), Equine serum, stabilizer and $0 . 0 9 \%$ sodium azide.

• ProsVue Software: The software has been validated with Windows 7 and Excel 2010. Wash Solution, Sample Diluent, and a three-level assayed control set are provided separately.

# J. Substantial Equivalence Information:

1. Predicate device name(s) and 510(k) number(s): Agendia BV MammaPrint $^ \mathrm { \textregistered }$ (k062694)   
2. Comparison with predicate: Not applicable. Clearance is supported by clinical study

K. Standard/Guidance Document Referenced (if applicable): CLSI EP5-A2: Evaluation of Precision Performance of Quantitative Methods. CLSI EP6-A: Evaluation of the Linearity of Quantitative Measurement Procedures: A Statistical Approach. CLSI EP07-A2: Interference Testing in Clinical Chemistry. CLSI EP9-A2: Method Comparison and Bias Estimation Using Patient samples; Approved Guidelines – Second Edition. CLSI EP17-A: Protocols for Determination of Limits of Detection and Limits of Quantitation Guidance for Submission of Tumor Marker Premarket Notifications $( 5 1 0 ( \mathrm { k } ) ) \mathrm { s }$ to FDA”, September 19, 1996.

# L. Test Principle:

The NADiA® ProsVue™ assay is a dual-monoclonal nucleic acid detection immuno-PCR assay (NADiA) for the quantitative measurement of total PSA (tPSA) in human serum. The design of the assay combines the specificity of monoclonal antibody capture with the sensitivity of real-time PCR detection. It is a two-site immunoassay utilizing an assay specific synthetic DNA sequence as a label and detected by a PCR method. Calibrators, controls and samples react with a reagent containing a monoclonal PSA-specific antibody (Mab) labeled with an assay-specific double-stranded DNA sequence (reporter MabDNA conjugate). Then a reagent of paramagnetic microparticles coupled to a monoclonal antibody specific for another site on PSA is added and allowed to react to form an insoluble sandwich immune complex with PSA. After washing the particles to remove excess reporter Mab-DNA conjugate, a reagent containing a heat-stable polymerase, specific primers, nucleotides, and a fluorescent dye is added to the washed microparticles. An Applied Biosystems® (AB) 7500 Fast Dx Real-Time PCR instrument is utilized to detect the specifically bound DNA label by subjecting suspended particles to PCR conditions and monitoring the generations of amplicon in real time. Quantitation of PSA is achieved by monitoring the number of PCR thermocycles it takes to generate a predetermined fluorescent signal over baseline. This is accomplished by automatic instrumentation designed to monitor fluorescence intensity as a function of cycle number. The amount of DNA initially present in the sample is inversely proportional to the threshold cycle (Ct). PSA values of controls and samples are calculated in $\mathsf { p g / m L }$ from a calibrator dose-response plot. ProsVue slope is calculated using ProsVue Software from the calculated PSA concentrations of three patient samples collected between six weeks and 20 months post radical prostatectomy.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

i) Assay:

Precision testing was conducted by following CLSI EP5-A2. Samples were analyzed in duplicate determinations at two sites, by three operators using two AB 7500 Fast Dx instruments and two reagent lots. Forty assays were performed at the first site by two operators. Each operator performed one run per day, alternating between reagent lots, over a course of 20 days. At the second site, a single operator performed 2 runs a day, one on each reagent lot, for 5 days.

The test samples consisted of three levels of male serum pools. One of the serum samples was in the lower portion of the normal range of $< 4 ~ \mathrm { p g / m L }$ (Low sample), one approximately at the middle range of $\sim 2 4 ~ \mathrm { p g / m L }$ (Intermediate sample) and one sample was at the upper end of the assay range of $\mathrm { \sim 6 9 \ p g / m L }$ (High sample). The range of samples tested encompassed the analytical measuring range. Analysis of variance (ANOVA) was used to evaluate the data consistent with the recommendations of EP5-A2. In addition, MIVQUE0, ML, and REML models in PROC VARCOMP procedure in SAS V9.1 (SAS Institute, Cary NC) were tested; REML produced the most reliable estimates from the analysis. The two statistical analyses were performed to analyze contribution of components of variation to total variation: one for estimation of within-run precision (repeatability), between-runs (within-lab), and between-day components of variation (Table-1 below); and a second for estimation of within-run, between-run, and betweenlot components of variation (Table-2 below). The following results were obtained:

Table-1: Within-run, between-run, and between-day variation.   

<table><tr><td rowspan=2 colspan=1>Samples</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=2>Within-runvariation</td><td rowspan=1 colspan=2>Between-runvariation</td><td rowspan=1 colspan=2>Between-dayvariation</td><td rowspan=1 colspan=2>Totalvariation</td></tr><tr><td rowspan=1 colspan=1>(pg/mL)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Low Sample</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>15.2</td></tr><tr><td rowspan=1 colspan=1>Intermediate Sample</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>9.4</td></tr><tr><td rowspan=1 colspan=1>High sample</td><td rowspan=1 colspan=1>69.1</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>8.60</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>10.6</td></tr></table>

Components of variation estimated only include within-run, between-run, and between-day in this analysis.

Table-2: Within-run, between-run, and between-lot variation.   

<table><tr><td colspan="1" rowspan="2">Samples</td><td colspan="1" rowspan="2">Mean(pg/mL)</td><td colspan="2" rowspan="1">Within-runvariation</td><td colspan="2" rowspan="1">Between-runvariation</td><td colspan="2" rowspan="1">Between-lotvariation</td><td colspan="2" rowspan="1">Totalvariation</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">Low Sample</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">14.0</td></tr><tr><td colspan="1" rowspan="1">Intermediate Sample</td><td colspan="1" rowspan="1">24.1</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">6.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">6.5</td></tr><tr><td colspan="1" rowspan="1">High Sample</td><td colspan="1" rowspan="1">69.1</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">6.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">4.6</td><td colspan="1" rowspan="1">6.6</td></tr></table>

Components of variation estimated only include within-run, between-run, and between-lot in this analysis. It should be noted that in this analysis, a $0 . 0 \%$ CV was estimated for between-lot for the high sample tested. This result may occur due to the partial-factorial design or co-linearity. The between-lot imprecision was not more than $4 . 0 \%$ .

# ii) Reproducibility study:

The studies were done at 2 sites in accordance with CLSI EP05-A2. The operators were trained in the operation of the NADiA® ProsVue™ system prior to testing. Study was performed over 20 days utilizing 3 male serum pools consisted of a Low sample (in the lower portion of the normal range of $< 4 ~ \mathrm { p g / m L }$ ), an Intermediate sample (in the middle range of $\sim 2 4 ~ \mathrm { p g / m L }$ ) and a High sample (in the upper end of the assay range $\mathbf { 0 . 7 } \mathrm { { \sim } } 6 9 \mathrm { { p g / m L } }$ ). The two different sites provide side-by-side comparison of two different instruments. The range of samples tested encompassed the analytical measuring range. Analysis of variance (ANOVA) was used to evaluate the data consistent with the recommendations of EP5-A2. This experiment allowed a determination of precision for each site and a determination of between site precision as given in the Table-3 below.

Table-3: Between Sites Precision results:   

<table><tr><td rowspan=1 colspan=1>Site(s)</td><td rowspan=1 colspan=1>Days</td><td rowspan=1 colspan=1>Datapoints</td><td rowspan=1 colspan=1>Sample#</td><td rowspan=1 colspan=1>Mean(pg/mL)</td><td rowspan=1 colspan=1>Between-site%CV</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=3 colspan=1>20</td><td rowspan=3 colspan=1>80</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.96</td><td rowspan=6 colspan=1>b</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24.40</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>70.70</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=3 colspan=1>5</td><td rowspan=3 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.11</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>23.00</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>62.50</td></tr><tr><td rowspan=3 colspan=1>1 &amp; 2</td><td rowspan=3 colspan=1>51 +5</td><td rowspan=3 colspan=1>40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.55</td><td rowspan=1 colspan=1>13.70</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24.20</td><td rowspan=1 colspan=1>7.30</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>66.90</td><td rowspan=1 colspan=1>9.60</td></tr></table>

1 $5 + 5 = \mathrm { f i r s t } 5$ days at site 1 and entire 5 days at site 2.

# b. Linearity/assay reportable range:

i) Linearity: The linear range was determined according to CLSI EP06-A. Based on the results of the Limit of Detection Study, linearity testing was done on a range $20 \mathrm { - } 3 0 \%$ wider than the anticipated measuring interval of 0.65 to $1 0 0 \mathrm { p g / m L }$ . A 15-member panel was prepared by combining appropriate volumes of a High sample and a Low sample. The High Sample was prepared by pooling 31 post RP patient samples (22 with low PSA; 9 with high PSA), and the pool’s assay value was $1 5 2 . 0 \ : \mathrm { p g / m L }$ . The equine-based ProsVue Sample Diluent was used as the Low Sample. All samples were run in triplicate in the ProsVue assay, using the AB 7500 Fast Dx instrument. The linear regression analysis of the linearity data was performed as “Observed” (Y-axis) vs “Expected” (X-axis) and with constant CV (weighted least squares regression): the best fitted line was $\mathrm { y } { = } 1 . 0 8 \mathrm { x } { - } 0 . 0 6$ . Based on the results of linearity range testing and that from the Limit of Detection Study, the method has been demonstrated to be linear from 0.65 to $1 0 0 \mathrm { p g / m L }$ with deviation from linearity less than $24 \%$ .

# ii) Spiking and Dilution Recovery Studies:

a) Spiking Recovery: Varying amounts of WHO PSA Standard Reference Preparation 96/670 were added to 6 human post-RP serum samples with endogenous PSA levels ranging from 2.7 to $3 7 . 3 ~ \mathrm { p g / m L }$ . The amount of PSA that was added varied from 10 to $8 0 ~ \mathrm { p g / m L }$ When compared to the expected ProsVue values, as shown in the table below, percent recovery of measured values ranged from 87.5 to $1 1 9 . 2 \%$ .

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Spike(pg/mL)</td><td colspan="1" rowspan="1">Observed(pg/mL)</td><td colspan="1" rowspan="1">Expected(pg/mL)</td><td colspan="1" rowspan="1">Percentrecovery</td></tr><tr><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">5.2</td><td colspan="1" rowspan="1">5.2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">18.1</td><td colspan="1" rowspan="1">15.2</td><td colspan="1" rowspan="1">119.2</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">46.0</td><td colspan="1" rowspan="1">45.2</td><td colspan="1" rowspan="1">101.8</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">87.0</td><td colspan="1" rowspan="1">85.2</td><td colspan="1" rowspan="1">102.2</td></tr><tr><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">6.2</td><td colspan="1" rowspan="1">6.2</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">17.0</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">105.2</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">47.7</td><td colspan="1" rowspan="1">46.2</td><td colspan="1" rowspan="1">103.3</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">86.2</td><td colspan="1" rowspan="1">94.3</td></tr><tr><td colspan="1" rowspan="4">3</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">50.3</td><td colspan="1" rowspan="1">47.3</td><td colspan="1" rowspan="1">106.3</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">63.7</td><td colspan="1" rowspan="1">77.3</td><td colspan="1" rowspan="1">96.4</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">111.2</td><td colspan="1" rowspan="1">117.3</td><td colspan="1" rowspan="1">102.4</td></tr><tr><td colspan="1" rowspan="4">4</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">8.5</td><td colspan="1" rowspan="1">8.5</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">18.6</td><td colspan="1" rowspan="1">18.5</td><td colspan="1" rowspan="1">100.5</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">51.0</td><td colspan="1" rowspan="1">48.5</td><td colspan="1" rowspan="1">105.1</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">90.7</td><td colspan="1" rowspan="1">88.5</td><td colspan="1" rowspan="1">102.5</td></tr><tr><td colspan="1" rowspan="3">5</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">27.8</td><td colspan="1" rowspan="1">27.8</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">93.8</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">65.6</td><td colspan="1" rowspan="1">67.8</td><td colspan="1" rowspan="1">96.7</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">110.3</td><td colspan="1" rowspan="1">107.8</td><td colspan="1" rowspan="1">102.3</td></tr><tr><td colspan="1" rowspan="4">6</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">11.7</td><td colspan="1" rowspan="1">12.7</td><td colspan="1" rowspan="1">91.8</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">42.7</td><td colspan="1" rowspan="1">87.5</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">77.5</td><td colspan="1" rowspan="1">82.7</td><td colspan="1" rowspan="1">93.7</td></tr></table>

a) Dilution Recovery: Eight human samples with assayed ProsVue values in the range of 249.4 to $1 , 2 6 9 . 2 \ : \mathrm { p g / m L }$ were diluted 1:10, 1:20 and 1:40 with ProsVue Sample Diluent and assayed for recovery. As shown in the table below, the recoveries ranged from $9 2 . 5 \%$ to $1 1 5 . 3 \%$ .

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Dilutionfactor</td><td colspan="1" rowspan="1">Observed(pg/mL)</td><td colspan="1" rowspan="1">Expected(pg/mL)</td><td colspan="1" rowspan="1">Percentrecovery</td></tr><tr><td colspan="1" rowspan="4">1</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">249.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">265.3</td><td colspan="1" rowspan="1">249.4</td><td colspan="1" rowspan="1">106.4</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">261.8</td><td colspan="1" rowspan="1">249.4</td><td colspan="1" rowspan="1">105.0</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">271.8</td><td colspan="1" rowspan="1">249.4</td><td colspan="1" rowspan="1">109.0</td></tr><tr><td colspan="1" rowspan="4">2</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">1269.2</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">1188.7</td><td colspan="1" rowspan="1">1269.2</td><td colspan="1" rowspan="1">93.7</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">1305.1</td><td colspan="1" rowspan="1">1269.2</td><td colspan="1" rowspan="1">102.8</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">1238.5</td><td colspan="1" rowspan="1">1269.2</td><td colspan="1" rowspan="1">97.6</td></tr><tr><td colspan="1" rowspan="4">3</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">760.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">702.6</td><td colspan="1" rowspan="1">760.0</td><td colspan="1" rowspan="1">92.5</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">781.7</td><td colspan="1" rowspan="1">760.0</td><td colspan="1" rowspan="1">102.9</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">747.9</td><td colspan="1" rowspan="1">760.0</td><td colspan="1" rowspan="1">98.4</td></tr><tr><td colspan="1" rowspan="4">4</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">624.1</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">583.3</td><td colspan="1" rowspan="1">624.1</td><td colspan="1" rowspan="1">93.5</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">625.0</td><td colspan="1" rowspan="1">624.1</td><td colspan="1" rowspan="1">100.1</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">649.9</td><td colspan="1" rowspan="1">624.1</td><td colspan="1" rowspan="1">104.1</td></tr><tr><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">309.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">304.6</td><td colspan="1" rowspan="1">309.2</td><td colspan="1" rowspan="1">98.5</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">328.7</td><td colspan="1" rowspan="1">309.2</td><td colspan="1" rowspan="1">106.3</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">347.8</td><td colspan="1" rowspan="1">309.2</td><td colspan="1" rowspan="1">112.5</td></tr><tr><td colspan="1" rowspan="4">6</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">509.3</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">526.5</td><td colspan="1" rowspan="1">509.3</td><td colspan="1" rowspan="1">103.4</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">559.6</td><td colspan="1" rowspan="1">509.3</td><td colspan="1" rowspan="1">109.9</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">587.2</td><td colspan="1" rowspan="1">509.3</td><td colspan="1" rowspan="1">115.3</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">431.6</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">428.9</td><td colspan="1" rowspan="1">431.6</td><td colspan="1" rowspan="1">99.4</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">420.6</td><td colspan="1" rowspan="1">431.6</td><td colspan="1" rowspan="1">97.5</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">486.4</td><td colspan="1" rowspan="1">431.6</td><td colspan="1" rowspan="1">112.7</td></tr><tr><td colspan="1" rowspan="4">8</td><td colspan="1" rowspan="1">Neat</td><td colspan="1" rowspan="1">322.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">1:10</td><td colspan="1" rowspan="1">313.5</td><td colspan="1" rowspan="1">322.2</td><td colspan="1" rowspan="1">97.3</td></tr><tr><td colspan="1" rowspan="1">1:20</td><td colspan="1" rowspan="1">310.6</td><td colspan="1" rowspan="1">322.2</td><td colspan="1" rowspan="1">96.4</td></tr><tr><td colspan="1" rowspan="1">1:40</td><td colspan="1" rowspan="1">319.1</td><td colspan="1" rowspan="1">322.2</td><td colspan="1" rowspan="1">99.0</td></tr></table>

iii) High Dose Hook-effect:

High dose hook effect was evaluated up to a level of $5 0 , 0 0 0 \mathrm { p g / m L }$ . A sample with a PSA concentration of $5 0 , 0 0 0 \mathrm { p g / m L }$ was prepared by adding purified PSA (WHO PSA Standard Reference Preparation 96/670) to ProsVue Sample Diluent. Serial dilutions were made using Sample Diluent down to a level of $1 . 5 ~ \mathrm { p g / m L }$ , and the dilutions were tested in duplicate with the ProsVue assay using the AB 7500 Fast Dx Instrument. Based on this procedure, the data shows that the samples with values as high as $5 0 , 0 0 0 \mathrm { p g / m L }$ did not exhibit this effect.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

i) Traceability:

The NADiA® ProsVueTM Assay calibrators and controls are manufactured with the WHO PSA Standard Reference Preparation 96/670 ( $90 \%$ PSA–ACT $+ 1 0 \%$ free PSA).

ii) Calibrator and controls:

The Prosvue PSA Calibrator is an in-vitro diagnostic product intended to be used to calibrate the NADiA® ProsVueTM method. It is a frozen liquid product consists of two vials, packaged as two vials for each of three levels (A, B and C), $0 . 0 6 \mathrm { m L }$ per vial. Calibrators are prepared in off-clot equine serum that is collected from herds that are certified as disease-free. The serum is pooled and delipidated using fumed silica adsorbent. EDTA is added as a stabilizer, and the serum is heat-treated at $6 0 ^ { \circ } \mathrm { C }$ , followed by a stripping process using ProsVue Target Capture Reagent. The 3 calibrators provided (A, B and C) contain PSA levels of $1 0 0 ~ \mathrm { p g / m L }$ , $2 5 ~ \mathrm { p g / m L }$ and $5 \ : \mathrm { p g / m L }$ , respectively. A PSA calibration curve is generated by analyzing each calibrator in triplicate in each microplate run. The calibration range of ProsVue is from 5 to $1 0 0 \mathrm { p g / m L }$ PSA. Values are assigned to each lot of calibrator from using the NADiA® ProsVueTM System, through a WHO PSA Standard Reference Preparation 96/670 ( $90 \%$ PSA–ACT + 10 % free PSA).

Prosvue Controls are prepared in the same matrix that is used for the calibrators. ProsVue Control Levels 1, 2 and 3 are formulated at approximately 3, 20, and $8 0 ~ \mathrm { p g / m L }$ of purified human prostate specific antigen $90 \%$ PSA–ACT $+ 1 0 \%$ Free PSA).

iii) Kit Stability:

ProsVue reagent stability was evaluated for three lots of ProsVue reagents that were stored at the labeled storage temperatures and tested at multiple time points. The kit reagents and wash solution were tested as a kit using in-house controls as test samples. The expiration period was defined as the elapsed time up to the point (the first time point of three successive time points) when the mean control recovery from three consecutive time points fell outside the value-assigned control limits. Other components like ProsVue Controls and ProsVue Sample Diluent were tested for storage stability. The results of the ProsVue stability testing are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>StorageConditions</td><td rowspan=1 colspan=1>Shelf Life(Days)</td></tr><tr><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Calibrators</td><td rowspan=1 colspan=1>-30° to -10°C</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>PCR Reagent</td><td rowspan=1 colspan=1>-30°to -10°</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Control Level 1</td><td rowspan=1 colspan=1>-30°to -10°C</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Control Level 2</td><td rowspan=1 colspan=1>-30°to -10°C</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Control Level 3</td><td rowspan=1 colspan=1>-30°to -10°C</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Target Capture Reagent</td><td rowspan=1 colspan=1>2°to8°</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Reporter Antibody Reagent</td><td rowspan=1 colspan=1>2°to8°</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Wash Solution</td><td rowspan=1 colspan=1>15°to30°C</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>Sample Diluent</td><td rowspan=1 colspan=1>-30°to -10°C</td><td rowspan=1 colspan=1>41</td></tr></table>

# iii) Sample Stability:

To demonstrate that low-level PSA samples observed post-RP have a similar stability profile as those with values above $1 0 0 \mathrm { p g / m L }$ when stored at $- 7 0 ^ { \circ } \mathrm { C }$ , serum pools from post-RP patients containing PSA concentrations at ${ \sim } 7$ and $\mathrm { \sim } 1 5 0 ~ \mathrm { p g / m L }$ were tested in an accelerated stability study at 4, 20, 30 and $4 0 \mathrm { { } ^ { \circ } C }$ . Aliquots of each pool were assayed with NADiA $\mathrm { \stackrel { \triangledown } { \mathrm { P r o s V u e } } } ^ { \mathrm { T M } }$ at baseline and at serial time points over 15 days at each incubation temperature. Stability projections indicated that $9 9 . 6 \%$ and $9 9 . 7 \%$ of PSA would remain immunoreactive at 20 years of storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ based on the observed decomposition rates of the low and high serum pools, respectively. Further, the rates of PSA decomposition were not significantly different for the two PSA pools at any of the four temperatures tested, thus indicating that the stability of PSA is not related to its concentration.

# d. Detection limit:

Limit of blank (LOB), limit of detection (LOD), and limit of quantitation (LOQ) were determined in accordance with CLSI/NCCLS EP 17-A. Four low level PSA samples were prepared (0.55, 0.65, 0.75 and $1 . 0 \ : \mathrm { p g / m L }$ ) by adding PSA antigen (WHO International PSA Standard 96/670 (90:10)) to the ProsVue Sample Diluent. Sixty replicates of each sample and ProsVue Sample Diluent were run in the ProsVue assay.

LOB – the highest measurement result that is likely to be observed (with a probability [alpha] of $0 . 0 5 [ 5 \% ] )$ for a blank sample was $0 . 1 7 \mathrm { p g / m L }$

LOD – the lowest amount of analyte in a sample that can be detected with type I and II error rates set to $5 \%$ was observed as $0 . 2 7 ~ \mathrm { p g / m L }$ .

LOQ – the lowest actual amount of analyte in a sample that can be reliably detected and at which total error meets lab pre-specified requirements for accuracy $( 8 0 - 1 2 0 \% )$ ) and precision $( < 2 5 \% )$ was $0 . 6 5 ~ \mathrm { p g / m L }$ .

e. Analytical specificity:

i) Equimolarity:

To demonstrate the equimolarity of ProsVue (the assay recognizes free PSA and PSA-ACT equally well), five sets of samples with free PSA concentrations ranging from zero to 100 percent and total PSA concentrations of 5, 30, 70 and $1 0 0 \mathrm { p g / m L }$ were assayed using ProsVue. The following data with a recovery range of 88 to $1 1 3 . 2 \%$ demonstrate that ProsVue is equimolar.

<table><tr><td rowspan=2 colspan=1>Total PSA(pg/mL)</td><td rowspan=2 colspan=1>Percentfree PSA</td><td rowspan=2 colspan=1>PercentPSA-ACT</td><td rowspan=1 colspan=2>ProsVue (pg/mL)</td><td rowspan=2 colspan=1>%Recovery</td></tr><tr><td rowspan=1 colspan=1>Expected</td><td rowspan=1 colspan=1>Observed</td></tr><tr><td rowspan=5 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=5 colspan=1>4.2</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>113.2</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>108.3</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>104.1</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>113.2</td></tr><tr><td rowspan=5 colspan=1>30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=5 colspan=1>27.9</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>97.7</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25.2</td><td rowspan=1 colspan=1>90.3</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>94.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>100.4</td></tr><tr><td rowspan=5 colspan=1>70</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=5 colspan=1>65.2</td><td rowspan=1 colspan=1>61.3</td><td rowspan=1 colspan=1>94.1</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>59.5</td><td rowspan=1 colspan=1>91.4</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>65.2</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>61.5</td><td rowspan=1 colspan=1>94.4</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>62.9</td><td rowspan=1 colspan=1>96.5</td></tr><tr><td rowspan=5 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=5 colspan=1>96.5</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>98.1</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>84.9</td><td rowspan=1 colspan=1>88.0</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>94.8</td><td rowspan=1 colspan=1>98.2</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>105.1</td><td rowspan=1 colspan=1>108.9</td></tr></table>

# ii) Interference:

Interference testing was performed according to CLSI/NCCLS EP7-A2 to determine the effect of various endogenous and exogenous substances on the NADiA® ProsVueTM assays. For all interferents the percent bias was determined by testing a control sample without the interferent and comparing it to the value obtained from a test sample to which the potential interferent had been added.

a) Endogenous Substance Interference:

Interference testing in the ProsVue assay was done at two PSA concentrations of 3 and $5 0 \mathrm { p g / m L }$ . Test samples were prepared by spiking the potential interferent into serum. Elevated concentrations of blood constituents were added to serum samples containing PSA and assayed in quadruplicate with the ProsVue™ assay. Bias exceeding $10 \%$ was considered interference. Results showed no intereference from the endogenous substance are summarized in the table below.

<table><tr><td rowspan=1 colspan=2>Interference by blood constituents</td></tr><tr><td rowspan=1 colspan=1>Bilirubin, conjugated</td><td rowspan=1 colspan=1>30 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Cholesterol</td><td rowspan=1 colspan=1>500 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Creatinine</td><td rowspan=1 colspan=1>5.0 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>200 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Immunoglobulin G</td><td rowspan=1 colspan=1>6 g/dL</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>1000 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Urea</td><td rowspan=1 colspan=1>260 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Uric acid</td><td rowspan=1 colspan=1>23.5 mg/dL</td></tr></table>

# b) Exogenous Substance Interference:

Forty two exogenous substances were tested for interference including common over-the-counter drugs and cancer drugs. Unspiked and spiked samples were run in quadruplicate and the mean ProsVue concentrations were determined. Interference was significant if the mean PSA values of the unspiked and spiked samples differed by $> 1 0 \%$ . The substances added and the highest concentrations tested are listed in the table below. Inaccuracies (biases) due to these substances were less than $10 \%$ at PSA concentrations of 3 and $5 0 ~ \mathrm { p g / m L }$ .

<table><tr><td colspan="2" rowspan="1">Interference by drugs</td></tr><tr><td colspan="1" rowspan="1">10-hydroxynortriptyline</td><td colspan="1" rowspan="1">700 ng/mL</td></tr><tr><td colspan="1" rowspan="1">5-Fluorouracil</td><td colspan="1" rowspan="1">390 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Acetaminophen</td><td colspan="1" rowspan="1">200 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1">53 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ascorbic acid</td><td colspan="1" rowspan="1">60 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Biotin</td><td colspan="1" rowspan="1">50 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Caffeine</td><td colspan="1" rowspan="1">60 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Carbamazepine</td><td colspan="1" rowspan="1">30 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Chloramphenicol</td><td colspan="1" rowspan="1">50 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Cimetidine</td><td colspan="1" rowspan="1">20 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ciprofloxacin</td><td colspan="1" rowspan="1">10 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Cisplatin</td><td colspan="1" rowspan="1">12 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Cotinine</td><td colspan="1" rowspan="1">1.9 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Cyclophosphamide</td><td colspan="1" rowspan="1">375 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Dextran-40</td><td colspan="1" rowspan="1">60 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Digoxin</td><td colspan="1" rowspan="1">6.1 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Doxorubicin</td><td colspan="1" rowspan="1">240 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Erythromycin</td><td colspan="1" rowspan="1">60 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">4 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Ethosuximide</td><td colspan="1" rowspan="1">250 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Flutamide</td><td colspan="1" rowspan="1">500 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Furosemide</td><td colspan="1" rowspan="1">60 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Gentamicin</td><td colspan="1" rowspan="1">10 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Heparin sodium</td><td colspan="1" rowspan="1">3 U/mL</td></tr><tr><td colspan="1" rowspan="1">Ibuprofen</td><td colspan="1" rowspan="1">500 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Leuprolide acetate</td><td colspan="1" rowspan="1">200 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Lidocaine</td><td colspan="1" rowspan="1">12 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Lithium</td><td colspan="1" rowspan="1">22.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Methotrexate</td><td colspan="1" rowspan="1">910 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Paclitaxel</td><td colspan="1" rowspan="1">6.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Pamidronate</td><td colspan="1" rowspan="1">9 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Phenytoin</td><td colspan="1" rowspan="1">50 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Prednisone</td><td colspan="1" rowspan="1">300 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Primidone</td><td colspan="1" rowspan="1">40 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Prochlorperazine</td><td colspan="1" rowspan="1">1 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Salicylic acid</td><td colspan="1" rowspan="1">600 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Sulfamethoxazole</td><td colspan="1" rowspan="1">400 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Tamoxifen</td><td colspan="1" rowspan="1">1.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Trimethoprim</td><td colspan="1" rowspan="1">40 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Valproate sodium</td><td colspan="1" rowspan="1">500 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">100 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Vinorelbine</td><td colspan="1" rowspan="1">1.2 μg/mL</td></tr></table>

# c) HAMA interference:

Two studies utilizing different panels of HAMA-positive samples were performed wherein the samples were run in the standard ProsVue assay and in a version of the assay lacking the polyclonal murine IgG. The results demonstrated marked reduction of observed ProsVue PSA values in some samples. To address the uncertainty between true PSA values and HAMA interference, a model system was designed using a purified Donkey Anti-Mouse Antibody (DAMA) reagent. This reagent was run in various concentrations from 10 to $1 0 , 0 0 0 , 0 0 0 { \mathrm { p g / m L } }$ versus the standard ProsVue assay and the version of the assay lacking the polyclonal murine IgG. This model system indicated complete assay resistance to nonspecific cross-linking. In addition, three known HAMA positive samples were titrated into serum diluent aliquots containing PSA at approximately 5 or $5 0 ~ \mathrm { p g / m L }$ . Titration of the HAMA samples up to a 1:8 dilution demonstrated approximately $10 0 \pm 2 0 \%$ recovery at both the expected 5 and $5 0 ~ \mathrm { p g / m L }$ concentrations. Since there was no trend in observed PSA values at decreasing HAMA concentrations, it is inferred that no quantifiable interference occurred in the neat sample and that PSA values in the assay represent true PSA concentrations. ProsVue has additives in the reporter antibody solution to avoid heterophilic interference; however, care should be taken in evaluating results of patients suspected of having these antibodies.

# d) Method Cross-Reactivity:

Cross-reactivity with prostatic acid phosphatase (PAP) was determined in the ProsVue assay. Purified human prostate specific antigen ( $90 \%$ PSA– $\mathrm { A C T } + 1 0 \%$ Free PSA) at two concentrations (5 and $5 0 \mathrm { p g / m L }$ ) was spiked into ProsVue Sample Diluent and also spiked into a PAP solution at a concentration of $1 ~ { \mu \mathrm { g / m L } }$ . The two pairs of solutions were assayed in 8 replicates. Results showed that PAP cross-reactivity was $< 0 . 0 0 0 4 \%$ in the ProsVue assay at both PSA concentrations tested.

# f. Assay cut-off:

For the NADiA® ProsVueTM assays, the slope (pg serum PSA/mL per month) is used to determine if a significant change occurred in the post-prostatectomy patients. Slope is a rate of change in serum total PSA concentration over a period of month. ProsVue™ results are calculated as the linear slope of three NADiA® $\mathrm { P r o s V u e } ^ { \mathrm { T M } }$ total PSA test results obtained on three serum samples collected between six weeks and 20 months post-radical prostatectomy.

The ProsVue software determines the slope by linear regression using the least squares method in the following formula:

$$
\frac { \displaystyle \sum _ { i = 1 } ^ { 3 } ( x _ { i } - { \bar { x } } ) ( y _ { i } - { \bar { y } } ) } { \displaystyle \sum _ { i = 1 } ^ { 3 } ( x _ { i } - { \bar { x } } ) ^ { 2 } }
$$

Where $\mathbf { y } = \mathbf { p } \mathbf { g } / \mathbf { m } \mathbf { L }$ and $\mathbf { X } =$ days between ProsVue samples. The slope in $\mathrm { p g / m L / d a y }$ is then converted to $\mathrm { p g / m L / }$ month by multiplying by 30.4375 (average days/month). No slope is calculated if any value is less than $0 . 6 5 ~ \mathrm { p g / m L }$ . ProsVue linear slope at $\mathrm { a } < 2 . 0 \ p \mathrm { g } / \mathrm { m L } / 1$ month determine cut-off.

The ProsVue™ software identifies serum samples with invalid collection intervals and calculates the equation of the calibration line, PSA concentration $\mathrm { ( p g / m L ) }$ of patient samples and assay controls, patient ProsVue slope $\mathrm { ( p g / m L / m o n t h ) }$ , and patient risk category. ProsVue software generates a report that includes the ProsVue slope (in $\mathrm { p g / m L / }$ month) and whether that slope categorizes a patient as being at “reduced risk” or “not at reduced risk” of prostate cancer recurrence.

2. Comparison studies:.

a. Method comparison with predicate device: Not applicable - clearance is supported by clinical study results   
$b$ . Matrix comparison: Not applicable.

3. Clinical studies:

To evaluate the prognostic capability of ProsVue linear slope to identify prostate cancer patients at reduced risk of prostate cancer recurrence following radical prostatectomy (RP), 304 patients were selected using a case-cohort study design. Patients meeting the study entry criteria were acquired from four clinical sites where each patient’s clinical status (“Stable” or “Recurrence”) was documented. Samples used were from men age 41 to 79 years with documented prostate cancer treated via radical prostatectomy with curative intent. Caucasian $( 8 8 . 8 \% )$ , African-American $( 8 . 2 \% )$ , Asian $( 2 . 0 \% )$ and patients of unknown race $( 1 . 0 \% )$ achieving post-RP total PSA values $< 1 0 0 ~ \mathrm { p g / m L }$ by standard-of-care assays were followed up to 17.6 years post-prostatectomy with three serum samples per patient acquired at various time points during the 1.5 - 20 month ProsVue sampling period. The study cohort consisted of 64 patients with “Recurrence” and 240 “Stable” patients. Recurrent patients were followed a median of 4.7 years (range 1.2 – 15.3 years) until recurrence. “Recurrence” was documented by positive imaging, positive biopsy, or death due to prostate cancer. “Stable” patients were followed for a median of 11.0 years (range 8.0 – 17.6 years).

Figure 1: Kaplan-Meier plot of probability of stable classification versus years (Solid line indicates patients with ProsVue slope $\leq 2 ~ \mathrm { p g / m L / 1 }$ month $[ \mathrm { n } { = } 2 4 5 ]$ and dashed line indicates patients with ProsVue slope $> 2 ~ \mathrm { p g / m L }$ month $[ \mathrm { n } \mathrm { = } 5 9 ]$ ).

![](images/6772cb5265f020167a70575f05ac53d53e2ddeec8cda1e55f9bd01ec0d943579.jpg)

The Kaplan-Meier estimates of the 8-year probability of “Stable” were $9 5 . 9 \%$ $9 5 \%$ CI: $9 3 . 4 - 9 8 . 4 \%$ for the patients with ProsVue Slope $\leq 2 ~ \mathrm { p g / m L } / .$ month and $3 7 . 3 \%$ ( $9 5 \%$ CI: $2 5 . 0 - 4 9 . 6 \%$ for the patients with ProsVue Slope $> 2 ~ \mathrm { p g / m L / m c }$ nth. Median time to recurrence in the patients with ProsVue slope $\leq 2 ~ \mathrm { p g / m L / 1 }$ month was ${ > } 1 7 . 6$ years versus 4.8 years for patients with ProsVue slope $> 2 ~ \mathrm { p g / m L / m o n }$ th.

Further investigation with Cox proportional hazards regression analysis used to evaluate the association between ProsVue linear slope and prostate cancer recurrence status in a univariate analysis (can be interpreted as standalone performance) indicated that the patients with ProsVue Slope $\leq 2 ~ \mathrm { p g / m L / 1 }$ month were 18.3 times more likely to be “Stable” than those with ProsVue Slope $> 2 ~ \mathrm { p g / m L / 1 }$ month $\mathrm { ( H R = }$ 18.3 with $9 5 \%$ CI: 10.6 to 31.8; $\mathrm { p } { < } 0 . 0 0 0 1 $ ). Thus, ProsVue linear slope was a significant predictor of reduced risk for prostate cancer recurrence in this analysis (Table-4).

Table-4: Univariate Cox proportional hazards regression results for ProsVue Slope   

<table><tr><td rowspan=1 colspan=1>Term</td><td rowspan=1 colspan=1>HR</td><td rowspan=1 colspan=1>HR 95% CI</td><td rowspan=1 colspan=1>p-value</td></tr><tr><td rowspan=1 colspan=1>ProsVue linear slope(&gt; 2.0 pg/mL/month vs.≤ 2.0 pg/mL/month)</td><td rowspan=1 colspan=1>18.332</td><td rowspan=1 colspan=1>10.6 - 31.8</td><td rowspan=1 colspan=1>&lt;0.0001</td></tr></table>

Cox proportional hazards regression analysis adjusted for pre-specified covariates of preprostatectomy PSA value, pathologic disease stage and Gleason score was performed with the ProsVue slope term (Table-5). Of the covariates, only the Gleason score variable reached significance $( \mathbf { p } = 0 . 0 0 0 4 )$ . The ProsVue linear slope term was attenuated from the univariate analysis but remained a highly significant and independent predictor of risk for prostate cancer recurrence with a HR of 9.8 ( $9 5 \%$ CI: 5.4 to 17.8, $\mathsf { p } < 0 . 0 0 0 1$ ). The reduced hazard probably reflects that ProsVue slope may be related to the clinical factors but is clearly providing additional information.

Table-5: Multivariate Cox proportional hazards regression results for ProsVue Slope.   

<table><tr><td rowspan=1 colspan=1>Term</td><td rowspan=1 colspan=1>HR</td><td rowspan=1 colspan=1>HR 95% CI</td><td rowspan=1 colspan=1>p-value</td></tr><tr><td rowspan=1 colspan=1>ProsVue linear slope(&gt; 2.0 pg/mL/month vs.≤ 2.0 pg/mL/month)</td><td rowspan=1 colspan=1>9.824</td><td rowspan=1 colspan=1>5.4 - 17.8</td><td rowspan=1 colspan=1>&lt;0.0001</td></tr><tr><td rowspan=1 colspan=1>Pre-prostatectomy PSA value</td><td rowspan=1 colspan=1>1.006</td><td rowspan=1 colspan=1>0.98 - 1.03</td><td rowspan=1 colspan=1>0.6469</td></tr><tr><td rowspan=1 colspan=1>Pathologic disease stage</td><td rowspan=1 colspan=1>1.729</td><td rowspan=1 colspan=1>0.89 - 3.4</td><td rowspan=1 colspan=1>0.1052</td></tr><tr><td rowspan=1 colspan=1>Gleason score</td><td rowspan=1 colspan=1>5.389</td><td rowspan=1 colspan=1>2.1 - 13.8</td><td rowspan=1 colspan=1>0.0004</td></tr></table>

The probability of clinical status “Recurrence” for the subjects with Slope $> 2$ $\mathrm { p g / m L / }$ month (Positive Predictive Value, PPV) and probability to remain with clinical status “Stable” through at least 8 years for the subjects with slope $\leq 2$ $\mathrm { p g / m L / }$ month (Negative Predictive Value, NPV) were calculated from the clinical study results $\scriptstyle ( \mathrm { n } = 3 0 4 )$ ). Table-6 presents a $2 \times 2$ table of the ProsVue classification based on the slope outcome and to the clinical status of patients as “Stable” or ”Recurrence” in the following 8 years (row and column totals are also displayed).

Table-6: 2x2 table of ProsVue classification vs. Reference Clinical status $\scriptstyle ( \mathrm { N } = 3 0 4 )$   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Reference Clinical Status</td></tr><tr><td rowspan=1 colspan=1>Recurrence</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Totals</td></tr><tr><td rowspan=3 colspan=1>ProsVue</td><td rowspan=1 colspan=1>Slope &gt; 2 pg/mL/month</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>Slope ≤ 2 pg/mL/month</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>245</td></tr><tr><td rowspan=1 colspan=1>Totals</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>304</td></tr></table>

Among subjects who had “Recurrence” during 8 years of follow-up, $7 1 . 9 \%$ (46/64) subjects had slope $> 2 ~ \mathrm { p g / m L } /$ month and among the subjects who had “Stable” status through at least 8 years of follow-up, $9 4 . 6 \%$ (227/240) subjects had slope $\leq 2$ $\boldsymbol { \mathrm { p g / m L / } }$ month.

Positive likelihood ratio (PLR) was 13.3 with $9 5 \%$ CI: 7.7 - 23.0 and negative likelihood ratio (NLR) was 0.297 with $9 5 \%$ CI: 0.201 - 0.440.

In these data, probability of “Recurrence” regardless of the values of the ProsVue slope was $2 1 . 0 \%$ (64/304); PPV was $78 . 0 \%$ with $9 5 \%$ CI: $6 5 . 3 - 8 7 . 7 \%$ and NPV was $9 2 . 7 \%$ with $9 5 \%$ CI: $8 8 . 6 - 9 5 . 6 \%$ .

Because the estimates of PPV and NPV depend on the probability of “Recurrence” among all subjects (prevalence of clinical status ”Recurrence”), Table-7 presents estimates of PPV and NPV calculated for various levels of prevalence of ”Recurrence”.

Table-7: Estimates of PPV and NPV calculated for various levels of prevalence $( \% )$ of clinical recurrence.

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>59.7</td><td rowspan=1 colspan=1>96.8</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>70.1</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>76.9</td><td rowspan=1 colspan=1>93.1</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>81.6</td><td rowspan=1 colspan=1>91.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>85.1</td><td rowspan=1 colspan=1>88.7</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>87.8</td><td rowspan=1 colspan=1>86.2</td></tr></table>

A subgroup analysis was performed on data from 20 patients with at least one postRP PSA value $\geq 3 0 0 ~ \mathrm { p g / m L }$ , but who remained stable throughout at least 8 years of follow-up. ProsVue slope correctly classified 11 of the 20 patients as “at reduced risk for recurrence” (ProsVue slope $\leq 2 \mathrm { p g / m L / m o n t h } )$ .

4. Clinical cut-off:

Frozen serum from 104 post-prostatectomy patients were used to demonstrate clinically useful cut-off. A two part study assessed the value of the linear slope of post-prostatectomy ProsVue values (in $\mathsf { p g } / \mathrm { m L }$ ) versus time (in months) as a prognostic indicator of risk for clinical recurrence of prostate cancer. The indicator employs a $< 2 . 0 \ \mathrm { p g / m L } /$ month cutpoint. Values at or below this cutpoint were associated with reduced risk for prostate cancer recurrence – biochemically in the first part study and clinically defined by positive imaging, positive biopsy results or death in the second part study.

The ProsVue software categorizes patients as “at reduced risk for prostate cancer recurrence” or “not at reduced risk for prostate cancer recurrence” based on the ProsVue slope criteria listed in the table below. The ProsVue software reports results as ProsVue slope $\mathrm { ( p g / m L }$ per month) and categorization.

<table><tr><td rowspan=1 colspan=1>Risk Category</td><td rowspan=1 colspan=1>ProsVue Slope</td></tr><tr><td rowspan=1 colspan=1>At reduced risk for prostatecancer recurrence</td><td rowspan=1 colspan=1>Less than or equal to2.0 pg/mL per month</td></tr><tr><td rowspan=1 colspan=1>Not at reduced risk for prostatecancer recurrence</td><td rowspan=1 colspan=1>Greater than2.0 pg/mL per month</td></tr></table>

5. Expected values/Reference range:

Not Applicable.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.