# 510(k) Summary

# GENERAL INFORMATION

# Submitted by:

Illumina Inc. 5200 Illumina Way San Diego, CA 92122 858-202-4500 (phone)

# Company Contact:

Leanne M. Kiviharju Sr. Director, Regulatory Affairs 858-246-8811 (phone) lkiviharju@illumina.com

# Date Prepared:

November 18, 2013

# DEVICE IDENTIFICATION

# Assay:

Trade or Proprietary Name: Illumina MiSeqDx™ Cystic Fibrosis 139-Variant Assay

Assay Common Name: Classification Name:

Next generation sequencing cystic fibrosis test CFTR (cystic fibrosis transmembrane conductance regulator) gene mutation detection (21 CFR 866.5900, Product Code PFR)

# Predicate Device:

Luminex xTAG® Cystic Fibrosis 60 Kit v2 (k083845)

# DEVICE DESCRIPTION

The Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay consists of library preparation and sample indexing reagents, sequencing reagents and consumables, MiSeqDx instrument and data analysis software. Testing begins with genomic DNA from a peripheral whole blood sample. The genomic DNA is processed through the library preparation steps, which specifically amplifies the intended genomic regions of each sample while also adding the indexes for sample identification. Flow cell capture sequences are also added to the amplified products. The resulting sample libraries are then transferred into a MiSeqDx reagent cartridge which contains all of the reagents required for cluster generation and sequencing (Sequencing By Synthesis). The MiSeqDx Cartridge, MiSeqDx Flow Cell, and MiSeqDx SBS Solution (PR2) are then inserted into the MiSeqDx instrument, which performs cluster generation, sequencing and data analysis.

# INTENDED USE

# Illumina MiSeqDxTM Cystic Fibrosis 139-Variant Assay

The Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay is a qualitative in vitro diagnostic system used to simultaneously detect 139 clinically relevant cystic fibrosis disease-causing mutations and variants of the cystic fibrosis transmembrane conductance regulator (CFTR) gene in genomic DNA isolated from human peripheral whole blood specimens. The variants include those recommended in 2004 by the American College of Medical Genetics (ACMG) and in 2011 by the American College of Obstetricians and Gynecologists (ACOG). The test is intended for carrier screening in adults of reproductive age, in confirmatory diagnostic testing of newborns and children, and as an initial test to aid in the diagnosis of individuals with suspected cystic fibrosis. The results of this test are intended to be interpreted by a board-certified clinical molecular geneticist or equivalent and should be used in conjunction with other available laboratory and clinical information. This test is not indicated for use for newborn screening, fetal diagnostic testing, pre-implantation testing, or for stand-alone diagnostic purposes.

The test is intended to be used on the Illumina MiSeqDx™M Instrument.

# SUBSTANTIAL EQUIVALENCE

# MiSeqDx Cystic Fibrosis 139-Variant Assay

<table><tr><td rowspan=6 colspan=2>CharacteristicAssay NameIntended UseAssay typeVariants</td><td rowspan=1 colspan=2>Characteristic                         Illumina                               Luminex (K083845)</td></tr><tr><td rowspan=1 colspan=1>Assay Name</td><td rowspan=1 colspan=1>Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay</td><td rowspan=1 colspan=1>Luminex xTAC® CysticFibrosis 60 Kit v2</td></tr><tr><td rowspan=1 colspan=1>The Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay is a qualitative in vitrodiagnostic system used to simultaneouslydetect 139 clinically relevant cystic fibrosisdisease-causing mutations and variants ofthe cystic fibrosis transmembraneconductance regulator (CFTR) gene ingenomic DNA isolated from humanperipheral whole blood specimens. Thevariants include those recommended in 2004by the American College of Medical Genetics(ACMG) and in 2011 by the AmericanCollege of Obstetricians and Gynecologists(ACOG). The test is intended for carrierscreening in adults of reproductive age, inconfirmatory diagnostic testing of newbornsand children, and as an initial test to aid inthe diagnosis of individuals with suspectedcystic fibrosis. The results of this test areintended to be interpreted by a board-certified clinical molecular geneticist orequivalent and should be used inconjunction with other available laboratoryand clinical information. This test is notindicated for use for newborn screening.fetal diagnostic testing, pre-implantationtesting, or for stand-alone diagnosticpurposes.The test is intended to be used on theIllumina MiSeqDxTM instrument.</td><td rowspan=1 colspan=1>The xTAG® Cystic Fibrosis 60kit v2 is a device used tosimultaneously detect andidentify a panel of mutationsand variants in the cysticfibrosis transmembraneconductance regulator (CFTR)gene in human bloodspecimens. The panel includesmutations and variantscurrently recommended by theAmerican College of MedicalGenetics and AmericanCollege of Obstetricians andGynecologists (ACMG/ACOG)plus some of the world&#x27;s mostcommon and North Americanprevalent mutations. ThexTAG Cystic Fibrosis 60 kit v2is a qualitative genotyping testwhich provides informationintended to be used for carriertesting in adults ofreproductive age, as an aid innewborn screening, and inconfirmatory diagnostic testingin newborns and children.The kit is not indicated for usein fetal diagnostic or pre-implantation testing. The kit isalso not indicated for stand-alone diagnostic purposes.</td></tr><tr><td rowspan=1 colspan=1>Assay type</td><td rowspan=1 colspan=1>Next generation sequencing test</td><td rowspan=1 colspan=1>Qualitative nucleic acidmultiplex test</td></tr><tr><td rowspan=1 colspan=1>Variants</td><td rowspan=1 colspan=1>139 clinically relevant variants</td><td rowspan=1 colspan=1>60 CFTR mutations and 4</td></tr><tr><td></td><td></td></tr></table>

# illumina

<table><tr><td rowspan=1 colspan=3>Characteristic                         Illumina                                Luminex (K083845)</td></tr><tr><td rowspan=2 colspan=1>DetectedTechnology</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>variants (benignpolymorphisms)</td></tr><tr><td rowspan=1 colspan=1>Sequencing by Synthesis (SBS)</td><td rowspan=1 colspan=1>Multiplex PCR followed bymultiplex allele specific primerextension for genotyping,hybridized to multiplexfluorescent microparticles,detected by flow cytometry.</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Nucleic acid from EDTA anticoagulatedblood</td><td rowspan=1 colspan=1>Nucleic acid from whole bloodanticoagulated with eitherEDTA or citrate.</td></tr><tr><td rowspan=1 colspan=1>SamplePreparation</td><td rowspan=1 colspan=1>DNA extraction using validated laboratorymethod</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Contra-indications</td><td rowspan=1 colspan=1>Not indicated for newborn screening, fetaldiagnostic testing, pre-implantation testing,or for stand-alone diagnostic purposes.</td><td rowspan=1 colspan=1>The kit is not indicated for usein fetal diagnostic or pre-implantation testing. This kit isalso not indicated for stand-alone diagnostic purposes</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Positive and negative controls required, notsupplied</td><td rowspan=1 colspan=1>Negative controls required, notsupplied. Positive controlsrecommended, not supplied.</td></tr><tr><td rowspan=1 colspan=1>InstrumentSystem</td><td rowspan=1 colspan=1>MiSeqDx instrument</td><td rowspan=1 colspan=1>Luminex 100 or 200 IS</td></tr></table>

# PERFORMANCE CHARACTERISTICS

# Accuracy

Accuracy of the Illumina MiSeqDx Cystic Fibrosis 139-Variant assay was assessed by evaluating 500 samples representing a wide variety of CFTR variants from four separate sources. The primary source of accuracy data was a clinical accuracy study conducted using a panel of 366 samples. The majority $( n { = } 3 5 5 )$ of samples consisted of archived, anonymized clinical $\mathsf { g D N A }$ specimens isolated from human blood, the remaining 11 samples were obtained from commercially available cell line specimens.

Data from this study was supplemented with accuracy data from 68 cell line samples evaluated in the reproducibility study, 14 clinical samples from the extraction method evaluation analytical study, and 52 synthetic plasmid samples. The synthetic plasmids were designed to include the genomic context of the rare variants, and contained anywhere from 1 to 9 variants within the same construct. They were linearized, diluted to genomic DNA equivalent copy numbers, and blended with human genomic DNA samples of wild type genotype at equivalent copy numbers to mimic a heterozygous sample.

The genotyping results for 137 SNA/small InDel sites, including the PolyTG/Poly T region were compared to Sanger bi-directional sequence analysis. A PCR based assay was used as the reference method for the two large deletions in the panel. Each duplex PCR assay made use of 2 primer sets to discriminate between wild type, heterozygous, and homozygous genotypes. One of the primer sets was designed to flank the deletion breakpoints, whereas the other amplified a region internal to the deletion. The two products were detected by size separation on an agarose gel.

The assays were validated using a panel of 28 samples in all (22 samples for each deletion) consisting of cell line and blood derived genomic DNA samples, and synthetic plasmids which encompassed the WT, HET and HOM genotypes for each large deletion. The PCR assays were confirmed to have $1 0 0 \%$ specificity and reproducibility for all samples tested, by evaluation of PCR products on an agarose gel. The accuracy of the PCR assays was confirmed using Sanger Sequencing and found to be 100% for all sample

Accuracy was determined for each genotype through three statistical measures. Positive Agreement (PA) was calculated for each variant genotype by dividing the number of samples with agreeing variant calls by the total number of samples with that variant as identified by the reference methods. Negative Agreement (NA) was calculated across all wild type (WT) positions by dividing the number of concordant WT positions by the total number of WT positions as defined by the reference methods. Overall Agreement (OA) was calculated across all reported positions by dividing the number of concordant WT and variant positions by the total number of reported positions as determined by the reference methods.

The MiSeqDx Cystic Fibrosis 139-Variant Assay had a genotype-level PA of $100 \%$ The NA for all wild types was ${ > } 9 9 . 9 9 \%$ and the OA for all reported positions was ${ > } 9 9 . 9 9 \%$ . All test results were based on initial testing.

<table><tr><td rowspan=1 colspan=1>CFTR dele2, 3</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>495</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1001</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E60X</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=2>1            0</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=2>0         0</td><td rowspan=1 colspan=3>100          100          100</td></tr><tr><td rowspan=1 colspan=1>P67L</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=2>R75X            500</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>1            0</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>O</td><td rowspan=1 colspan=2>.0         100</td><td rowspan=1 colspan=2>100          100</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>492</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>394deITT</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>.I    3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0          100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>406-1G&gt;A</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E92X</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=2>100          100</td></tr><tr><td rowspan=1 colspan=1>D110H</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R117C</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>:   4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>481</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Y122X</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>.0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>488</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>.663delT</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G178R</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>P205S</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1    100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>L206W</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>491</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1078delT</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G330X</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan="2">Mutation (Common Name)</td><td rowspan="2">Total calls per mutation</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative calls (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2"># No Calls</td><td rowspan="2">Positive Agreement (%)</td><td rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement (%)</td></tr><tr><td>Clinical</td><td>Cell Line</td><td>Synthetic Samples</td></tr><tr><td>1336K</td><td>500</td><td>Samples 0</td><td>Samples 1</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100 1</td><td>100</td><td>100</td></tr><tr><td>1154insTC</td><td>500</td><td>0</td><td>1</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R347H</td><td>500</td><td>6</td><td>1</td><td>1</td><td>492 :</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R347P</td><td>500</td><td>3</td><td>2</td><td>0</td><td>495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R352Q</td><td>; 500</td><td>5</td><td>0</td><td>0</td><td>1 495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A455E</td><td>500</td><td>4</td><td>2</td><td>0</td><td>494</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S466X (C-&gt;G)</td><td>500</td><td>1</td><td>0</td><td>1</td><td>498</td><td>0</td><td>0 &quot;</td><td>100</td><td>100 :</td><td>100</td></tr><tr><td>1548delG</td><td>500</td><td>1</td><td>0</td><td>1</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q493X</td><td>500</td><td>4</td><td>2</td><td>0</td><td>494</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>107del</td><td>500</td><td>4</td><td>2</td><td>0</td><td>494</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>F508del</td><td>500</td><td>84</td><td>29</td><td>0</td><td>387</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1677delTA</td><td>500</td><td>1</td><td>0</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>V520F</td><td>500</td><td>2</td><td>0</td><td>0</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1717-1G&gt;A</td><td>500</td><td>4</td><td>1</td><td>0</td><td>495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>G542X</td><td>500</td><td>12</td><td>3</td><td>0</td><td>485</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S549N</td><td>500</td><td>2</td><td>2</td><td>1</td><td>495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S549R (c.1647T&gt;G)</td><td>500</td><td>3</td><td>1 :</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>G551D</td><td>500</td><td>8</td><td>3</td><td>0</td><td>489</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R553X</td><td>500</td><td>8</td><td>2</td><td>0</td><td>490</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A559T</td><td>500</td><td>4</td><td>0</td><td>1</td><td>495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R560T</td><td>500</td><td>6</td><td>1</td><td>0</td><td>493</td><td>0</td><td>0 ;</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1812-1 G-&gt;A</td><td>500</td><td>0</td><td>2</td><td>0</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr></table>

<table><tr><td>Mutation (Common Name)</td><td rowspan="2">Total calls per mutation</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative calls (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2"># No Calls</td><td rowspan="2">Positive Agreement (%)</td><td rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement (9%)</td></tr><tr><td></td><td>Clinical Samples</td><td>Cell Line Samples</td><td>Synthetic Samples</td></tr><tr><td>1898+1G&gt;A</td><td>500</td><td>2</td><td>1</td><td>0</td><td>497 4</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2143delT</td><td>500</td><td>·2</td><td>1</td><td>0</td><td>497</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2183AA&gt;G</td><td>500</td><td>3</td><td>1</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2184insA</td><td>500</td><td>3</td><td>0</td><td>1</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2184delA</td><td>500</td><td>1</td><td>1</td><td>0</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R709X</td><td>500</td><td>1</td><td>0</td><td>2</td><td>497</td><td>0</td><td>0</td><td>100 2</td><td>100</td><td>100</td></tr><tr><td>K710X</td><td>500 i- ;</td><td>3</td><td>0</td><td>0</td><td>497</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2307insA</td><td>500</td><td>3</td><td>0</td><td>2</td><td>495</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R764X</td><td>500</td><td>1</td><td>0</td><td>2</td><td>497</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>W846X</td><td>500</td><td>0</td><td>1</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2789+5G&gt;A</td><td>500</td><td>9</td><td>1</td><td>0</td><td>490</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q890X</td><td>500</td><td>1</td><td>0</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3120G&gt;A</td><td>500</td><td>1</td><td>0</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3120+1G&gt;A</td><td>500</td><td>7</td><td>1</td><td>0</td><td>492</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3272-26A&gt;G</td><td>500 </td><td>0</td><td>1.</td><td>0</td><td>499.</td><td>0</td><td>0</td><td>100</td><td>100 </td><td>100</td></tr><tr><td>R1066C</td><td>500</td><td>6</td><td>0</td><td>0</td><td>494</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R1066H</td><td>500</td><td>1</td><td>0</td><td>1</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>W1089X</td><td>500</td><td>4</td><td>0</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Y1092X (C&gt;A)</td><td>500</td><td>3</td><td>1</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>M1101K</td><td>500</td><td>2</td><td>2</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R1158X</td><td>500</td><td>7</td><td>t 1</td><td>0</td><td>492</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R1162X</td><td>500</td><td>5</td><td>1</td><td>0</td><td>494</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3659delC</td><td>500</td><td>4 .</td><td>1 i. &quot;</td><td>0 I</td><td>495 i</td><td>0 !-</td><td>0</td><td>100 I</td><td>100</td><td>1 100</td></tr></table>

<table><tr><td rowspan="2">Mutation (Common Name)</td><td rowspan="2">Total calls per</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative calls (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2"># No Calls</td><td rowspan="2">Positive Agreement (%)</td><td rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement (%)</td></tr><tr><td>Clinical Samples</td><td>Cell Line Samples</td><td>Synthetic Samples</td></tr><tr><td>S1196X</td><td>mutation 500</td><td>1 1</td><td>0</td><td>0</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3791delC</td><td>500</td><td>2</td><td>0</td><td>0</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3849+10kbC&gt;T</td><td>500</td><td>11</td><td>2</td><td>0</td><td>487</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3876delA</td><td>500</td><td>6</td><td>1</td><td>0</td><td>493</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S1251N</td><td>500</td><td>1</td><td>0</td><td>1</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3905insT</td><td>500 1</td><td>3</td><td>1</td><td>0</td><td>496</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>W1282X</td><td>500</td><td>9</td><td>1</td><td>0</td><td>490</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>N1303K</td><td>500</td><td>9</td><td>1</td><td>0</td><td>490</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>CFTRdele22,23</td><td>500</td><td>1</td><td>0</td><td>1</td><td>498</td><td>1</td><td>0</td><td>100</td><td>99.8</td><td>99.8</td></tr><tr><td>M1V</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q39X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0 1</td><td>100</td><td>100</td><td>100</td></tr><tr><td>405+1 G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>: 499</td><td>0</td><td>... 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>E92K</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q98X</td><td>S00 1</td><td>0 :</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>457TAT-&gt;G</td><td>500</td><td>0 !</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>574delA</td><td>.500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>711+3A&gt;G</td><td>500</td><td>0 i</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>711+5 G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>712-1 G-&gt;T</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>H199Y</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>.0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q220X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>852del22</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499 </td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr></table>

<table><tr><td rowspan="2">Mutation (Common Name)</td><td rowspan="2">Total calls per</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative calls (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2"># No Calls</td><td rowspan="2">Positive Agreement (%)</td><td rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement {%)</td></tr><tr><td>Clinical</td><td>Cell Line</td><td>Synthetic</td></tr><tr><td>T3381</td><td>mutation 500</td><td>Samples 0</td><td>Samples 0</td><td>Samples 1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S341P</td><td>500</td><td>0</td><td>.0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1213delT</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1248+1G&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1259insA</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>W401X</td><td>1 500</td><td>0</td><td>0 ;</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>(c.1202G&gt;A) W401X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>(c.1203G&gt;A) 1341+1G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1461ins4</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1525-1G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S466X (C-&gt;A)</td><td>S00</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>L467P</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S489X</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S492F</td><td>500</td><td>0</td><td>0 :</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q525X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1717-8G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S549R (c.1645A&gt;C)</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Q552X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R560K</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1811+1.6kb A-&gt;G</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>E585X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td rowspan="2">Mutation (Common Name)</td><td rowspan="2">Total calls per mutation</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative calls (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2">#No Calls</td><td rowspan="2">Positive Agreement (%)</td><td rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement (%)</td></tr><tr><td>Clinical</td><td>Cell Line</td><td>Synthetic Samples</td></tr><tr><td>1898+3A&gt;G</td><td>500</td><td>Samples 0</td><td>Samples 0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>L732X</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2347delG</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2585delT</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>E822X</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2622+1G&gt;A</td><td>500</td><td>0</td><td>0</td><td>2</td><td>498</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>E831X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>R851X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>2711delT</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>1927P</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>S945L</td><td>500</td><td>.0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3007delG</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>G970R</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>3121-1G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>L1065P</td><td>500</td><td>0</td><td>0</td><td>.1</td><td>499</td><td>: 0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>L1077P</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>$o^</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>Y1092X (C&gt;G)</td><td>500</td><td>:0</td><td>0</td><td>1</td><td>499.</td><td>0</td><td>:0.</td><td>100</td><td>100</td><td>100</td></tr><tr><td>E1104X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>W1204X</td><td>500</td><td>.0</td><td>0 1</td><td>1</td><td>499 .:</td><td>0</td><td>0</td><td>100</td><td>100.</td><td>100 5</td></tr><tr><td>(c.3611G&gt;A) W1204X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>(c.3612G&gt;A) G1244E</td><td>500</td><td>0</td><td>.0</td><td>= 1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr></table>

lllwn!   

<table><tr><td>Mutation (Common Name)</td><td>Total calls per</td><td colspan="3">Positive calls (Variants)</td><td rowspan="2">Negative cails (Wild Type)</td><td rowspan="2"># Miscalls</td><td rowspan="2"># No Cails</td><td rowspan="2">Positive Agreement (%)</td><td colspan="2" rowspan="2">Negative Agreement (%)</td><td rowspan="2">Overall Agreement (%)</td></tr><tr><td></td><td>mutation</td><td>Clinical Samples</td><td>Cell Line Samples</td><td>Synthetic Samples</td></tr><tr><td>4005+1G-&gt;A</td><td>500</td><td>0</td><td>0</td><td>1.</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>4016insT</td><td>500</td><td>0 :</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100 :</td><td>100</td><td>100</td></tr><tr><td>Q1313X</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>4209TGTT&gt;AA</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>4382delA</td><td>500</td><td>0</td><td>0</td><td>1</td><td>499</td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>PolyTG/PolyT</td><td>19</td><td>17</td><td>2</td><td>0</td><td>0</td><td>0</td><td>0</td><td>100</td><td>N/A</td><td>100</td></tr><tr><td>1506V</td><td>1</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>N/A</td><td>100</td><td>100</td></tr><tr><td>1507V*</td><td>1</td><td>0 I</td><td>0</td><td>0</td><td>i</td><td>0</td><td>0</td><td>N/A</td><td>100</td><td>100</td></tr><tr><td>F508C*</td><td>1</td><td>0</td><td>0</td><td>0</td><td>1</td><td>0</td><td>0</td><td>N/A</td><td>100</td><td>100</td></tr><tr><td>Total</td><td>67522</td><td></td><td>557</td><td></td><td>66965</td><td>1</td><td>0</td><td>100.00</td><td>&gt;99.99</td><td>&gt;99.99</td></tr></table>

D '77aian ) miphipa an # <   
nn8Aznialau sp mpupa cco7d au nalcil 1iodal ja8upc aul

JJJ T   
T p    G S  S      e   
  

<table><tr><td rowspan=2 colspan=1>Variant(CommonName)•</td><td rowspan=2 colspan=1>TotalCalls perVariant</td><td rowspan=1 colspan=3>Positive Calls (Variants)</td><td rowspan=2 colspan=1>NegativeCalls (WildType)</td><td rowspan=2 colspan=1>:Miscalls</td><td rowspan=2 colspan=1>= NoCalls</td><td rowspan=2 colspan=1>PositiveAgreement(%b)</td><td rowspan=2 colspan=1>NegativeAgreement(^0)</td><td rowspan=2 colspan=1>OverallAgreement(%0)</td></tr><tr><td rowspan=1 colspan=1>ClinicalSamples</td><td rowspan=1 colspan=1>Cell LineSamples</td><td rowspan=1 colspan=1>SyntheticSamples</td></tr><tr><td rowspan=1 colspan=1>1506V</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>493</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td rowspan=3 colspan=1>Variant(CommonName</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>TotalCalls perVariant</td><td rowspan=1 colspan=3>Positive Calls (Variants)</td><td rowspan=2 colspan=1>NegativeCalls (WildType)</td><td rowspan=2 colspan=1>:Miscalls</td><td rowspan=2 colspan=1>z NoCalls</td><td rowspan=2 colspan=1>PositiveAgreement(0}}</td><td rowspan=2 colspan=1>NegativeAgreement(a)</td><td rowspan=2 colspan=1>OverallAgreemen(%)</td></tr><tr><td rowspan=1 colspan=1>ClinicalSamples</td><td rowspan=1 colspan=1>Cell LineSamples</td><td rowspan=1 colspan=1>SyntheticSamples</td></tr><tr><td rowspan=1 colspan=1>1507V</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>499</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>F50SC</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Genotype.</td><td rowspan=1 colspan=1>ClinicalSamples</td><td rowspan=1 colspan=1>CellLineSamples</td><td rowspan=1 colspan=1>SyntheticSamples</td><td rowspan=1 colspan=1>:Miscalls</td><td rowspan=1 colspan=1>: NoCalls^</td><td rowspan=1 colspan=1>%0Accuracy</td></tr><tr><td rowspan=1 colspan=1>(TG)9(T)7/(TG)1)(T)7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>50.0</td></tr><tr><td rowspan=1 colspan=1>(TG)9(T)9/(TG)10(T)7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)9(T)9/(TG)11(T)7</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TC)9(T)9/(TG)11(T)9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)10(T)7/(TG)0(T)7</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0    •</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)7/(TC)0(T)9</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TC)0(T)7/(TG)11(T)5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)7/(TG)(T)7</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)7/(TG)12(T)5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)10(T)7/(TG)12(T)7</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90.9</td></tr></table>

<table><tr><td rowspan=1 colspan=1>(TG)|0(T)9/(TG)10(T)9</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)10(T)9/(TG)11(T)5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)9/(TG)H(T)7</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TC)10(T)9/(TG)11(T)9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)9/(TG)12(T)5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)0(T)9/(TG)12(T)7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>92.3</td></tr><tr><td rowspan=1 colspan=1>(TG)1(T)5/(TG))1(T)7</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>83.3</td></tr><tr><td rowspan=1 colspan=1>(TG)11(T)7/(TG)I(T)7</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)1(T)7/(TG)1I(T)9*</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>(TG)11(T)7/(TG)2(T)5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)13(T)7/(TG)2(T)7</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)11(T)9/(TG)12(T)7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>(TG)12(T)7/(TG)12(T)7</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total**</td><td rowspan=1 colspan=3>448</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.44</td></tr></table>

# illumina

# Reproducibility  139-Variant Assay

The reproducibility of the MiSeqDx Cystic Fibrosis System was determined through a blinded study using 3 trial sites and 2 operators at each site. Two well characterized panels of 46 samples each were tested by each of the operators at each site for a total of 810 calls per site. The panels contained a mix of genomic DNA from lymphoblastoid cell lines with known mutations in the CFTR gene as well as some leukocyte-depleted blood spiked with lymphoblastoid cell lines with known mutations in the CFTR gene. The blood samples were provided to allow incorporation of the extraction steps used to prepare gDNA that serves as the primary input for the assay workflow.

The sample pass rate, defined as the number of samples passing QC metrics on the first attempt, was $9 9 . 9 \%$ .

The genotype-level Positive Agreement for all variants was $9 9 . 7 7 \%$ The Negative Agreement for all WT positions was $9 9 . 8 8 \%$ and the Overall Agreement for all reported positions was $9 9 . 8 8 \%$ .All test results are based on initial testing. No repeat testing was done for the reproducibility study.

<table><tr><td></td><td></td><td>Sample Genotype</td><td rowspan="4">Variants</td><td rowspan="4">oe e se cee</td><td colspan="4">Positive Agreeing calls (Variants)</td><td rowspan="2">Negative Agreeing calls (Wild type) MWlstt # Nos ct #</td><td rowspan="2">(%) te n</td><td rowspan="2">(%) tem aen</td><td rowspan="2">(%) Na </td></tr><tr><td>Pennn</td><td>#adaen</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>A</td><td>A AX</td><td>O A</td><td>2 Eis</td><td></td><td></td></tr><tr><td>A 1 A</td><td>1 2</td><td>S549N (HET) 1812-1 G-</td><td>810 810</td><td>6 6</td><td>6 6 6 6</td><td>804 804 804</td><td>804 804 0 0 804 0 0</td><td>100 100 100 100</td><td>100 100</td></tr><tr><td>A</td><td>3</td><td>&gt;A(HET Q493X/F50</td><td>810</td><td>12</td><td>12 12</td><td>798 798</td><td>798 0</td><td>0 100</td><td>100 100</td></tr><tr><td>A</td><td>4*</td><td>8del(HET) F508del/21 84delA</td><td>810</td><td>12</td><td>12 12</td><td>797</td><td>1* 798 798 0</td><td>100 100</td><td>100</td></tr><tr><td>A</td><td>5^</td><td>(HET) Y122X/R115</td><td>810</td><td>12 10</td><td>12</td><td>798 665 798</td><td>0 135</td><td>94.44 94.44</td><td>94.44</td></tr><tr><td>A</td><td>6</td><td>8X (HET). F508del/21 83AA&gt;G</td><td>810</td><td>12 12</td><td>12</td><td>798 798</td><td>. ^ 798 0 0</td><td>100</td><td>100 100</td></tr><tr><td>A 1</td><td>7</td><td>(HET) R75X(HET)</td><td>810</td><td>6 6</td><td>6</td><td>804 804</td><td>804 0</td><td>0 100</td><td>100 100</td></tr><tr><td>A</td><td>8</td><td>1507del/F50 8del (HET)</td><td>810</td><td>12</td><td>12 12</td><td>798 798</td><td>798 0</td><td>0 100</td><td>100 100</td></tr><tr><td>A</td><td>g**</td><td>F508del/W1 282X(HET)</td><td>810</td><td>12 11</td><td>12</td><td>798 797</td><td>798 2* 0 *</td><td>97.22</td><td>99.96 99.92</td></tr><tr><td></td><td>A 10* *</td><td>F508del/32 72-26A&gt;G</td><td>810</td><td>12 11</td><td>12</td><td>798 797</td><td>798 2* 0 *</td><td>97.22</td><td>99.96 99.92</td></tr><tr><td>A</td><td>11</td><td>(HET) F508del/38 49+10kbC&gt;T</td><td></td><td></td><td></td><td>810 12 12 12 798 798 798</td><td>o 0</td><td>100</td><td>100 100</td></tr><tr><td>A</td><td>12</td><td>(HET) 621+1G&gt;T/3 120+1G&gt;A</td><td>810</td><td></td><td></td><td>12 12 12 798 798 798</td><td>0 0</td><td>100</td><td>100 100</td></tr><tr><td>1</td><td>13</td><td>(HET) E60X/F508d</td><td>810 12</td><td>12 :</td><td>12 798</td><td>798</td><td>798 0</td><td>0 100.</td><td>100 100</td></tr><tr><td>A</td><td>14</td><td>el (HET) M1101K</td><td>810 6</td><td>6</td><td>6 804</td><td>804</td><td>804 0</td><td>0 100</td><td>100 100</td></tr><tr><td>A</td><td>.15.</td><td>(HET) M1101K</td><td>810</td><td>6 6</td><td>6 804</td><td>804</td><td>804 0</td><td>0 100</td><td>100</td></tr><tr><td></td><td></td><td>(HOM)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>100</td></tr></table>

# illumina

<table><tr><td>A</td><td>16</td><td>F508del (HOM)</td><td>I506V, I507V, F508C</td><td></td><td>828</td><td>6</td><td>6</td><td>6 822</td><td>822</td><td>822</td><td></td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>A</td><td>17</td><td>F508del/36 59delC</td><td>not present</td><td></td><td>810</td><td>12</td><td></td><td>12 12:798</td><td></td><td>798.798</td><td></td><td>0 0</td><td></td><td>100.</td><td>100</td><td></td><td>100</td></tr><tr><td>A</td><td></td><td>18</td><td>(HET) R117H/F508 del (HET)</td><td>(TG)10(T) 9/(TG)12</td><td>816</td><td>18</td><td>18</td><td>18</td><td>798</td><td>798</td><td>798</td><td>0</td><td>0</td><td>100</td><td></td><td>100.0</td><td>100</td></tr><tr><td>A</td><td>19</td><td></td><td>(T)5 621+1G&gt;T/7 11+1G&gt;T</td><td></td><td>810</td><td>12</td><td>12</td><td>12</td><td>798</td><td>798 798</td><td></td><td>0</td><td>0</td><td>100</td><td>0 100</td><td></td><td>100 ..</td></tr><tr><td>A</td><td>20</td><td>G85E/621+1</td><td>(HET)</td><td>810</td><td></td><td>12</td><td>12</td><td>12 798</td><td></td><td>798 798</td><td></td><td>0 0</td><td></td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>21</td><td>G&gt;T (HET) A455E/F508</td><td></td><td>810</td><td></td><td>12</td><td>12</td><td>122</td><td>798</td><td>798 798</td><td></td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>22</td><td>del (HET</td><td>F508del/R5</td><td></td><td>810</td><td>12</td><td>12</td><td>12</td><td>798</td><td>,&#x27; 798</td><td>: 798</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>23</td><td>60T (HET) F508del/Y1 092X (C&gt;A)</td><td></td><td></td><td>8100</td><td>12</td><td>12</td><td>12</td><td>7798</td><td>798 798</td><td>0</td><td></td><td>0</td><td>100</td><td>100</td><td></td><td>.100</td></tr><tr><td>A</td><td>24</td><td>(HET) N1303K</td><td></td><td>810</td><td></td><td>6</td><td>6</td><td>6</td><td>804 804</td><td>804</td><td>0</td><td></td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>25</td><td>(HET) G542X _(HOM)</td><td></td><td>810</td><td></td><td>.6</td><td>6</td><td>6</td><td>804 804</td><td>804</td><td>0</td><td></td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>L A</td><td>26</td><td>G542X (HET)</td><td></td><td>810</td><td></td><td>6</td><td>6</td><td>6 804</td><td>804</td><td>804</td><td>0</td><td></td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>i A</td><td>27</td><td>G551D/RS5</td><td></td><td>810</td><td></td><td>12</td><td>12</td><td>12 798</td><td></td><td>798, 798</td><td>0</td><td></td><td>0</td><td>100</td><td></td><td>100</td><td>·100</td></tr><tr><td>A</td><td>28</td><td>3X (HET) 3849+10kbC</td><td></td><td>810</td><td></td><td>6</td><td>6</td><td>6 804</td><td></td><td>804 804</td><td>0</td><td></td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>[ A</td><td>29</td><td>&gt;T (HOM) WT</td><td></td><td>810</td><td></td><td>0</td><td>0</td><td>0</td><td>810</td><td>810&#x27;</td><td>810</td><td>0</td><td>0</td><td>N/A</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>30</td><td>F508del (HET)</td><td></td><td>810</td><td></td><td>6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>31</td><td>1717-1G&gt;A (HET)</td><td></td><td>810</td><td></td><td>66</td><td></td><td>*6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100 100</td></tr><tr><td>A</td><td>.32</td><td>R1162X (HET)</td><td></td><td>810</td><td></td><td>6 12</td><td>6 12</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>33</td><td>R347P/G55 1D (HET)</td><td></td><td>810</td><td></td><td></td><td></td><td>12</td><td>798</td><td>798</td><td>798</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>ame1</td></tr><tr><td>A</td><td>34</td><td>R334W (HET</td><td></td><td>810</td><td></td><td>6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>35</td><td>WT</td><td></td><td>810</td><td></td><td>0</td><td>0</td><td>0</td><td>810</td><td>810</td><td>810</td><td>0</td><td>&#x27;0</td><td>N/A</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>36</td><td>G85E (HET)</td><td></td><td>810</td><td>6</td><td></td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A L</td><td>37</td><td>1336K (HET)</td><td></td><td>810</td><td>6</td><td></td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>.0</td><td>100</td><td></td><td>100</td><td>100</td></tr><tr><td>A</td><td>38</td><td>WT</td><td></td><td>810</td><td>0</td><td></td><td>0</td><td>0</td><td>810</td><td>810</td><td>810</td><td>0</td><td>0</td><td>N/A</td><td></td><td>100</td><td>100</td></tr></table>

<table><tr><td>A</td><td>39</td><td>F508del/38 49+10kbC&gt;T</td><td>810</td><td>12 12 12</td><td>798</td><td>798 798</td><td>0</td><td>0</td><td>100</td><td></td><td>100 100</td></tr><tr><td>A</td><td>40</td><td>(HET) 621+1G&gt;T/3 120+1G&gt;A</td><td></td><td>810 12 12</td><td>12 798</td><td></td><td>798 798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A</td><td>41</td><td>(HET) F508del/36 59delC (HET)</td><td>,</td><td>810 12</td><td>12 12</td><td>798</td><td>798 798</td><td>0 0 .</td><td>100</td><td>100 •</td><td>100</td></tr><tr><td>A</td><td>42</td><td>R117H/F508 del (HET)</td><td>(TG)10(T) 9/(TG)12 (T)s</td><td>816</td><td>18 18 18</td><td>798</td><td>798 798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A</td><td>43</td><td>G85E/621+1 G&gt;T(HET)</td><td></td><td>810 12</td><td>12 12</td><td>798</td><td>798 798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A</td><td>44</td><td>A455E/FS08 del (HET)</td><td></td><td>810 12</td><td>12 12</td><td>798</td><td>798 798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A I</td><td>45</td><td>N1303K (HET)</td><td></td><td>810 6</td><td>6</td><td>6 804</td><td>804 804</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>A</td><td>46</td><td>G551D/R5S 3X (HET)</td><td></td><td>810 12</td><td>12</td><td>12 798</td><td>798 798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B `</td><td>·47</td><td>.2789+5G&gt;A (HOM)</td><td></td><td>810 6</td><td>6 12</td><td>6 804 12 798</td><td>804 804 798</td><td>0 0</td><td>100</td><td>100 .</td><td>100</td></tr><tr><td>m B</td><td>48</td><td>CFTR dele2, 3/F508del (HET)</td><td></td><td>810 12</td><td></td><td></td><td>798</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>49</td><td>F508del/18 98+1G&gt;A (HET)</td><td></td><td>810 12</td><td>12</td><td>12 798</td><td>798 798</td><td>0</td><td>0 100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>50 51</td><td>WT F508del/21</td><td>810</td><td>810 0 12</td><td>0 12</td><td>0 810 12 798</td><td>810 798 798</td><td>810 0 0</td><td>0 0</td><td>N/A 1000</td><td>100 100 100 100</td></tr><tr><td>B</td><td></td><td>43deiT (HET)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>B</td><td>52</td><td>3876delA (HET)</td><td>810</td><td>6</td><td>6</td><td>6 804</td><td>804</td><td>804 0</td><td>0</td><td>100 100</td><td>100</td></tr><tr><td>r B</td><td>53</td><td>3905insT (HET)</td><td>810</td><td>6</td><td>6</td><td>6 804</td><td>804</td><td>804 0</td><td>0</td><td>100</td><td>100 100</td></tr><tr><td>B</td><td>54</td><td>394delTT (HET)</td><td>810</td><td>6</td><td>6</td><td>6 804</td><td>804</td><td>804 0</td><td>0</td><td>100</td><td>100 100</td></tr><tr><td>B</td><td>55</td><td>F508del (HET)</td><td>810</td><td>6.</td><td>6</td><td>6 804</td><td>804</td><td>804 0</td><td>0</td><td>100</td><td>100 100</td></tr><tr><td>B</td><td>56</td><td>WT</td><td>810</td><td>0.</td><td>0 0</td><td>810</td><td>810</td><td>810 0</td><td>0</td><td>N/A</td><td>100 100</td></tr><tr><td>1</td><td>57</td><td>WT</td><td>810</td><td>0 6</td><td>0 0 6 6</td><td>810 804</td><td>810 804 804</td><td>810 0 0</td><td>0 . 0</td><td>N/A</td><td>100 100</td></tr><tr><td>B</td><td>58</td><td>F508del (HET)</td><td>810</td><td></td><td></td><td></td><td></td><td></td><td></td><td>100</td><td>100 100</td></tr><tr><td>B</td><td>59</td><td>WT</td><td>810</td><td>0</td><td>0 0</td><td>810</td><td>810</td><td>810 0</td><td>0</td><td>N/A</td><td>100 100</td></tr><tr><td>B</td><td>60</td><td>L206W (HET)</td><td>810</td><td>6</td><td>6 6</td><td>804</td><td>804</td><td>804 0</td><td>0</td><td>100</td><td>100 100</td></tr><tr><td>&gt; B</td><td>61</td><td>WT</td><td>810</td><td>0</td><td>0 0</td><td>810</td><td>810</td><td>810 0</td><td>0.</td><td>N/A</td><td>100 100</td></tr></table>

# illumina

<table><tr><td>B</td><td>62</td><td>G330X</td><td></td><td>810</td><td>6 6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>63</td><td>(HET) WT</td><td>810</td><td>0</td><td>0</td><td>0</td><td>810</td><td>810</td><td>810</td><td>0</td><td>0</td><td>N/A</td><td></td><td>100</td><td>100</td></tr><tr><td>B</td><td>64</td><td>R347H (HET)</td><td>810</td><td>6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>65</td><td>1078delT (HET)</td><td></td><td>810 6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100 .</td></tr><tr><td>B</td><td>66</td><td>G178R/F508 del(HET)</td><td></td><td>810 12</td><td>12</td><td>12</td><td>798</td><td>798</td><td>798</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>67</td><td>S549R</td><td></td><td>810 6</td><td>6</td><td>6</td><td></td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td></td><td></td><td>(c.1647T&gt;G) (HET)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>&#x27;</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>B</td><td>68 69</td><td>S549N (HET) W846X</td><td>810</td><td>810 6 6</td><td>6 6</td><td>6 6</td><td>804 804</td><td>804 804</td><td>804 804</td><td>0 0</td><td>0 0</td><td></td><td>100</td><td>100</td><td>100 100</td></tr><tr><td>B</td><td></td><td>(HET)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>100</td><td>100</td><td></td></tr><tr><td>B</td><td>70</td><td>WT E92X/F508d</td><td>810 810</td><td>0 12</td><td>0 12</td><td>0 12</td><td>810 798</td><td>810 798</td><td>810 798</td><td>0 0</td><td>0 0</td><td></td><td>N/A</td><td>100</td><td>100 100</td></tr><tr><td>B }</td><td>71 72</td><td>el(HET) 621+1G&gt;T/1</td><td>810</td><td>12</td><td>12</td><td>12</td><td>798</td><td>798</td><td>797</td><td>0</td><td>${</td><td></td><td>100</td><td>100 99.96</td><td>99.96</td></tr><tr><td>B</td><td></td><td>154insTC (HET)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>100</td><td></td><td></td></tr><tr><td>B</td><td>73</td><td>G542X (HET)</td><td></td><td>810 6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td></td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>74</td><td>F508del (HET)</td><td>810</td><td>6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>. B 2</td><td>75^</td><td>F508del (HET)</td><td>810</td><td>6</td><td>5</td><td>6</td><td>804</td><td>670</td><td>804</td><td>0</td><td>135 ^</td><td></td><td>94.44</td><td>94.44</td><td>94.44</td></tr><tr><td>B</td><td>76</td><td>F508del (HET)</td><td>810</td><td>6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td>0</td><td></td><td>100</td><td>100</td><td>100</td></tr><tr><td>: B ]</td><td>77</td><td>621+1G&gt;T/ A455E (HET)</td><td></td><td>810 12</td><td>12</td><td>12</td><td>798</td><td>798</td><td>798</td><td>0</td><td></td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>78</td><td>1812-1 G- &gt;A(HET</td><td></td><td>810 6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td>0</td><td></td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>79</td><td>WT</td><td></td><td>810 0</td><td>0 12</td><td>0 12</td><td>810 798</td><td>810 798</td><td>810 798</td><td></td><td>0</td><td>0</td><td>N/A</td><td>100</td><td>100</td></tr><tr><td>B</td><td>80</td><td>F508del/R5 53X (HET)</td><td></td><td>810 12</td><td>12</td><td></td><td></td><td>798</td><td>798</td><td></td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>81</td><td>F508del/G5 51D (HET)..</td><td></td><td>810</td><td>12 12 12</td><td>12 :. 12</td><td>798 798</td><td>798</td><td>798</td><td></td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B</td><td>82</td><td>R347P/F508 del (HET)</td><td></td><td>810</td><td>18 18</td><td>18</td><td>798</td><td>798</td><td>798</td><td></td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B i</td><td>83</td><td>R117H/F508 del (HET)</td><td>(TG)1O(T) 9/(TG)12</td><td>816</td><td></td><td></td><td></td><td></td><td></td><td></td><td>0</td><td>0.</td><td>100</td><td>100</td><td>100 i</td></tr><tr><td>i B</td><td>84</td><td>1507del</td><td>.(T)5</td><td>810</td><td>6 6</td><td>6</td><td>804</td><td>804</td><td>804</td><td></td><td>0</td><td>0</td><td>100</td><td>100</td><td>. 100</td></tr><tr><td>B</td><td>85</td><td>(HET) 2789+5G&gt;A</td><td></td><td>810 6</td><td>6</td><td>6</td><td>804</td><td>804</td><td>804</td><td></td><td>0</td><td>0</td><td>100</td><td>100</td><td>100</td></tr><tr><td></td><td></td><td>(HOM)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>.</td><td></td><td></td><td></td></tr></table>

# illumina

<table><tr><td>B</td><td>86*</td><td>CFTR dele2, 3/F508del (HET)</td><td>810</td><td>12 12</td><td>12 798</td><td>797 798</td><td>0 1&quot;</td><td>100</td><td>99.96</td><td>99.96</td></tr><tr><td>B</td><td>87</td><td>F508del/18 98+1G&gt;A (HET)</td><td>810 • 1</td><td>12 12 12</td><td></td><td>798 798 798</td><td>0 0</td><td>100 .</td><td>100</td><td>100 *</td></tr><tr><td>B</td><td>88 89.</td><td>WT F508del/21</td><td>810 810.</td><td>0 12</td><td>0 0 810 810 810 12 12 798 798 798</td><td></td><td>0 0 0 0</td><td>N/A 100</td><td>100 100</td><td>100 100</td></tr><tr><td>B 1</td><td></td><td>43delT (HET)</td><td>•</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>B</td><td>90</td><td>3905insT (HET)</td><td>810</td><td>6</td><td>6 6 804</td><td>804 804</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td>B : 5</td><td>91</td><td>394de!TT _(HET)_</td><td>810</td><td>6</td><td>6. 6 804 :</td><td>804 804</td><td>0 0</td><td>100 .</td><td>100</td><td>100</td></tr><tr><td>B</td><td>92</td><td>FS08del (HET)</td><td>810</td><td>6</td><td>6 6 804</td><td>804 804</td><td>0 0</td><td>100</td><td>100</td><td>100</td></tr><tr><td></td><td></td><td>Total</td><td>74556</td><td>209</td><td>221182</td><td></td><td>4 273</td><td>99.77</td><td>99.88</td><td>99.88</td></tr></table>

One replicate of samples 5 and 75 had a $0 \%$ c sample plate prior to library preparation. $* *$ insufficient coverage.

Table 4: Reproducibility Panel Variants   

<table><tr><td rowspan=1 colspan=1>Variation (CommonName</td><td rowspan=1 colspan=1>Variant Type</td><td rowspan=1 colspan=1>CFTR GeneRegion</td></tr><tr><td rowspan=1 colspan=1>polyTG/PolyT</td><td rowspan=1 colspan=1>Compound DIV*</td><td rowspan=1 colspan=1>Intron 9</td></tr><tr><td rowspan=1 colspan=1>2183AA&gt;G</td><td rowspan=1 colspan=1>Compound DIV*</td><td rowspan=1 colspan=1>Exon 14</td></tr><tr><td rowspan=1 colspan=1>CFTR dele2, 3</td><td rowspan=1 colspan=1>DEL</td><td rowspan=1 colspan=1>Intron1-Intron3</td></tr><tr><td rowspan=1 colspan=1>1154insTC</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>1507del</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 11</td></tr><tr><td rowspan=1 colspan=1>F508del</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 11</td></tr><tr><td rowspan=1 colspan=1>2143delT</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 14</td></tr><tr><td rowspan=1 colspan=1>3659delC</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 22</td></tr><tr><td rowspan=1 colspan=1>3876delA</td><td rowspan=1 colspan=1>DIV*</td><td rowspan=1 colspan=1>Exon 23</td></tr><tr><td rowspan=1 colspan=1>394delTT</td><td rowspan=1 colspan=1>DIV in homopolymericregion</td><td rowspan=1 colspan=1>Exon 3.</td></tr><tr><td rowspan=1 colspan=1>1078delT</td><td rowspan=1 colspan=1>DIV in homopolymericregion*</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>2184delA</td><td rowspan=1 colspan=1>DIV in homopolymericregion*</td><td rowspan=1 colspan=1>. Exon 14</td></tr><tr><td rowspan=1 colspan=1>3905insT</td><td rowspan=1 colspan=1>DIV in homopolymericregion*</td><td rowspan=1 colspan=1>Exon 23</td></tr><tr><td rowspan=1 colspan=1>E60X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 3</td></tr><tr><td rowspan=1 colspan=1>R75X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 3</td></tr><tr><td rowspan=1 colspan=1>G85E</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 3</td></tr><tr><td rowspan=1 colspan=1>E92X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 4</td></tr><tr><td rowspan=1 colspan=1>R117H</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 4</td></tr><tr><td rowspan=1 colspan=1>Y122X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 4</td></tr></table>

# illumina

<table><tr><td rowspan=1 colspan=1>Variation (CommonName</td><td rowspan=1 colspan=1>Variant Type</td><td rowspan=1 colspan=1>CFTR GeneRegion</td></tr><tr><td rowspan=1 colspan=1>621+1G&gt;T</td><td rowspan=1 colspan=1>SNV.</td><td rowspan=1 colspan=1>Intron 4</td></tr><tr><td rowspan=1 colspan=1>G178R</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 5</td></tr><tr><td rowspan=1 colspan=1>711+1G&gt;T</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 5</td></tr><tr><td rowspan=1 colspan=1>L206W</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 6</td></tr><tr><td rowspan=1 colspan=1>C330X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>R334W</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>I336K</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>R347P</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>R347H</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 8</td></tr><tr><td rowspan=1 colspan=1>A-455E</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 10</td></tr><tr><td rowspan=1 colspan=1>Q493X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 11</td></tr><tr><td rowspan=1 colspan=1>1717-1G&gt;A</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 11.</td></tr><tr><td rowspan=1 colspan=1>G542X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>S549N</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>S549R (c.1647T&gt;G)</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>G551D</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>R553X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>R560T</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 12</td></tr><tr><td rowspan=1 colspan=1>1812-1 G-&gt;A</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 12</td></tr><tr><td rowspan=1 colspan=1>1898+1G&gt;A</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 13</td></tr><tr><td rowspan=1 colspan=1>W846X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 15</td></tr><tr><td rowspan=1 colspan=1>2789+5G&gt;A</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 16</td></tr></table>

# illumina

<table><tr><td rowspan=1 colspan=1>Variation (ConmonName)</td><td rowspan=1 colspan=1>Variant Type</td><td rowspan=1 colspan=1>CFTR GeneRegion</td></tr><tr><td rowspan=1 colspan=1>3120+1G&gt;A                1</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Initron 18</td></tr><tr><td rowspan=1 colspan=1>3272-26A&gt;G</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 19</td></tr><tr><td rowspan=1 colspan=1>Y1092X (C&gt;A)</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 20</td></tr><tr><td rowspan=1 colspan=1>M1101K</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 20</td></tr><tr><td rowspan=1 colspan=1>R1158X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 22</td></tr><tr><td rowspan=1 colspan=1>R1162X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 22</td></tr><tr><td rowspan=1 colspan=1>3849+10kbC&gt;T</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Intron 22</td></tr><tr><td rowspan=1 colspan=1>W1282X</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 23</td></tr><tr><td rowspan=1 colspan=1>N1303K</td><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>Exon 24</td></tr></table>

# DNA Extraction

Three commonly used, commercially available extraction methods representing magnetic bead extraction, alcohol precipitation and silica filter column isolation methods, were evaluated using KEDTA anti-coagulated whole blood. A total of 14 unique blood samples were used in the study representing wild type and three mutant genotypes (3 samples with F508del, 1 sample with I506V, and 1 sample with D1 10H). The three DNA extraction methods were tested independently by 2 different operators who each performed 3 runs per extraction method. Each extraction was performed by each operator on different days. The DNA concentration and A260/A280 ratio of the extracted gDNA samples was determined using spectrophotometry. The total sample size for each extraction method in this study was 168 (14 samples $\textbf { x 2 }$ operators/extraction method x 3 runs/operator $\textbf { x } 2$ replicates/extracted gDNA sample).

<table><tr><td>Extraction Method</td><td>Number of samples tested</td><td>Call Rate</td><td>Accuracy</td><td>Sample First Pass Rate*</td></tr><tr><td>Alcohol Precipitation</td><td>168</td><td>100%</td><td>100%</td><td>100%</td></tr><tr><td>Silica Filter Column Isolation</td><td>168</td><td>100%</td><td>100%</td><td>100%</td></tr><tr><td>Magnetic Bead Extraction</td><td>168</td><td>100%</td><td>100%</td><td>100%</td></tr></table>

Percent of samples having call rate of $> 9 9 \%$ in first run

# DNA input

The DNA input range of the Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay was evaluated by performing a serial dilution study using 14 representative DNA samples containing 16 unique CF variants. Each sample was tested in duplicate at 9 DNA input levels ranging from $1 2 5 0 ~ \mathfrak { n g }$ to 1 ng ( $1 2 5 0 ~ \mathrm { n g }$ $5 0 0 ~ \mathfrak { n } \mathfrak { g }$ $2 5 0 ~ \mathrm { n g }$ , $1 0 0 ~ \mathfrak { n } \mathbf { g }$ , 50 ng, 25ng, 10 ng, $5 \ \mathrm { n g } ,$ and $1 \ \mathfrak { n g } .$ ). For determination of accuracy, sample genotypes were compared to bidirectional Sanger sequencing data and the deletions were compared to PCR assay. 1250 ng and $2 5 { \mathrm { n g } }$ were identified as the upper and lower bound for DNA input respectively as they had $2 9 5 \%$ sample first pass rate with no incorrect calls ( $100 \%$ accuracy and call rate).

DNA inputs of 1250 ng, 250 ng, and 100 ng were further tested with 4 representative DNA samples and 20 replicates per DNA input level for each sample $\mathrm { n } { = } 4 ^ { \ast } 2 0 { = } 8 0$ samples), while the lower bound of $2 5 { \mathfrak { n } } { \mathfrak { g } }$ was tested with 14 samples, 20 replicates for each sample $( \mathrm { n } = 1 4 ^ { \ast } 2 0 { = } 2 8 0$ samples). The accuracy and sample first pass rate was $100 \%$ . at all DNA input levels.

# illumina

# Interfering Substances

To assess the impact of interfering substances on the MiSeqDx Cystic Fibrosis 139- Variant Assay, the performance of the assay was evaluated in the presence and absence of potential interferents. Eight whole blood specimens were tested in the study including 3 CF positive samples with unique genotypes. Four endogenous interfering substances (bilirubin, cholesterol, hemoglobin, and triglycerides) were tested by spiking them into blood specimens prior to DNA extraction. The concentration limits for each substance is shown in the following table. Additionally, to assess interference resulting from blood collection (short draw), EDTA was spiked into blood samples, and to assess interference resulting from sample preparation, the final wash buffer from a silica filter column isolation method was added to purified genomic DNA.

The MiSeqDx Cystic Fibrosis 139-Variant Assay achieved $100 \%$ call rate for all samples tested, and $100 \%$ reproducibility in genotype calls between samples in the presence and absence of interfering substances.

To access the impact of multiplexing index primer interference, a cross-contamination study using two samples, each with unique homozygous genotypes at 4 different genomic positions, and two respective index primers was performed. No change in variant calling was observed with contamination levels $< 4 0 \%$ . The sample genotype became heterozygous when contamination levels were $\geq 4 0 \%$ .

No interference was observed from any of the endogenous or exogenous interferents.   

<table><tr><td>Test Substance</td><td>Total Number of Replicates</td><td>Concentration Tested in Blood (Upper Limit)</td><td>Concentration Tested in Blood (Lower Limit)</td><td>Call Rate</td></tr><tr><td>Bilirubin</td><td>16</td><td>684 μmol/L</td><td>137 μmol/L</td><td>100%</td></tr><tr><td>Cholesterol</td><td>16</td><td>13 mmol/L</td><td>2.6 mmol/L</td><td>100%</td></tr><tr><td>Hemoglobin</td><td>16</td><td>2 g/L</td><td>0.4 g/L</td><td>100%</td></tr><tr><td>Triglyceride</td><td>16</td><td>37 mmol/L</td><td>7.4 mmol/L</td><td>100%</td></tr><tr><td>EDTA</td><td>16</td><td>7.0 mg/mL</td><td>2.8 mg/mL</td><td>100%</td></tr></table>

# illumina

# Sample Indexing

Sample index primers are used in the assay to assign a unique barcode to each sample DNA, allowing the ability to pool multiple samples together into a single sequencing run. A total of 96 samples indexes were tested using 8 unique DNA samples to verify the ability of the assay to consistently make a genotyping call for a given sample across different indexing primer combinations. Each sample was tested with 12 different indexing primer combinations. Sample results were compared against bidirectional Sanger sequencing data for all positions/variants except the 2 large deletions, which were confirmed using a duplex PCR assay. Reproducibility and accuracy were $100 \%$ for all sample/index primer combinations.

# November 19, 2013

ILLUMINA, INC.  
LEANNE M. KIVIHARJU  
SENIOR DIRECTOR, REGULATORY AFFAIRS5200 ILLUMINA WAY  
SAN DIEGO CA 92122

Re: k124006 Trade/Device Name: Ilumina MiSeqDx Cystic Fibrosis 139-Variant Assay Regulation Number: 21 CFR 866.5900 Regulation Name: Cystic fibrosis transmembrane conductance regulator (CFTR) gene mutation detection system Regulatory Class: 11 Product Code: PFR Dated: November 18, 2013 Received: November 19. 2013

Dear Ms. Kiviharju:

We have reviewed your Section 5l0(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28. 1976, the enactment date of the Medical Device Amendments. or to devices that have been reclassified in accordance with the provisions of the Federal Food. Drug. and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may. therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labecling. and prohibitions against misbranding and adulteration. Please note: CDRH docs not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into cither class 11 (Special Controls) or class II (PMA). it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations. Title 21. Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination docs not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencics. You must comply with all the Act's requirements, including. but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809): medical device reporting (reporting of

Page 2—Ms. Kiviharju

medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulations (21 CFR Parts 801 and 809), please contact the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638 2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/dcfault.hum. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/lndustry/default.htm.

Sincerely yours,

# Reena Philip -S

Maria M. Chan, Ph.D.   
Director   
Division of Immunology and Hematology Devices Office of In Vitro Diagnostics and Radiological Health   
Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) k124006

Device Name Illumina MiSeqDx Cystic Fibrosis 139-Variant Assay

Indications for Use (Describe)

The Ilumina MiSeqDx Cystic Fibrosis 139-Variant Assay is a qualitative in vitro diagnostic system used to sansyelvn -cuuns nn fibrosis transmembrane conductance regulator (CFTR) gene in genomicDNA isolated from human peripheral whole blood specimens. The variants include those recommended in 2004 by the American College of Medical Genetics (ACMG) and in 2011 by the American College of Obstetricians and Gynecologists (ACOG)2. The test is intended for carrier screening in adults o reproductive age, in confirmatory diagnostic testing o The results of this test are intended to be interpreted by a board-certified clinical molecular enetics equivalent and should be used in conjunction with other available laboratory and clinical information.

Ts  is not indicatorusor ewbo enin tal dosc est, pre-plnatn estin stand-alone diagnostic purposes.

The test is intended to be used on the Illumina MiSeqDx instrument.