Nanostring Technologies   
Sylva Krizan, Ph.D.   
Regulatory Affairs Specialist 530 Fairview Ave N, Suite 2000 Seattle, WA 98109

Re: K130010 Trade/Device Name: Prosigna™ Breast Cancer Prognostic Gene Signature Assay Regulation Number: 21 CFR $\ S 8 6 6 . 6 0 4 0 $ Regulation Name: Gene expression profiling test system for breast cancer prognosis Regulatory Class: Class II Product Code: NYI, NSU Dated: August 9, 2013 Received: August 9, 2013

Dear Dr. Krizan:

This letter corrects our substantially equivalent letter of September 6, 2013.

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA’s issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act’s requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of

medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR 1000- 1050.

If you desire specific advice for your device on our labeling regulation (21 CFR Part Parts 801 and 809), please contact the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm. Also, please note the regulation entitled, sMisbranding by reference to premarket notifications (21CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH’s Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Act from the Division of Industry and Consumer Education at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely,

# Reena Philip -S

Reena Philip, Ph.D.   
Director   
Division of Molecular Genetics and Pathology   
Office of In Vitro Diagnostics and Radiological Health Center for Devices and Radiological Health

Enclosure

# 510(k) Number (if known): K130010

Device Name: Prosigna™ Breast Cancer Prognostic Gene Signature Assay

Indications for Use:

The Prosigna™ Breast Cancer Prognostic Gene Signature Assay is an in vitro diagnostic assay which is performed on the NanoString nCounter $\textsuperscript { \textregistered }$ Dx Analysis System using FFPE breast tumor tissue previously diagnosed as invasive breast carcinoma. This qualitative assay utilizes gene expression data, weighted together with clinical variables to generate a risk category and numerical score, to assess a patient’s risk of distant recurrence of disease.

The Prosigna Breast Cancer Prognostic Gene Signature Assay is indicated in female breast cancer patients who have undergone surgery in conjunction with locoregional treatment consistent with standard of care, either as:

1. A prognostic indicator for distant recurrence-free survival at 10 years in post-menopausal women with Hormone Receptor-Positive $\left( \mathrm { H R + } \right)$ , lymph node-negative, Stage I or II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors.

2. A prognostic indicator for distant recurrence-free survival at 10 years in post-menopausal women with Hormone Receptor-Positive $\left( \mathrm { H R + } \right)$ , lymph node-positive (1-3 positive nodes), Stage II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors. The device is not intended for patients with 4 or more positive nodes.

Special Conditions for Use: Prosigna is not intended for diagnosis, to predict or detect response to therapy, or to help select the optimal therapy for patients.

Prescription Use x_ AND/OR Over-The-Counter Use (Part 21 CFR 801 Subpart (21 CFR 801 Subpart C)

(PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ONANOTHER PAGE IF NEEDED)

# Concurrence of CDRH, Office of In Vitro Diagnostics and Radiological Health (OIR)

Division Sign-Off Office of In Vitro Diagnostics and Radiological Health

# 510(k) Summary: K130010

Applicant: NanoString Technologies, Inc.

Establishment Registration Number: 3006389928

# Contact person:

Sylva Krizan, Ph.D.   
Regulatory Affairs Specialist   
NanoString Technologies   
530 Fairview Avenue North, Suite 2000 Seattle, WA 98109

Phone: (206) 432-8854   
Fax: (206) 378-6288

Summary Date: September 6, 2013

# Device Name:

Trade name: Prosigna™ Breast Cancer Prognostic Gene Signature Assay Common Name: NanoString gene expression profiling test for breast cancer prognosis

Classification: 21 CFR $\ S$ 866.6040: Gene expression profiling test system for breast cancer prognosis

# Guidance Document:

Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis, issued on May 9, 2007

Product Code: NYI

# Indications for Use / Intended Use:

The Prosigna™ Breast Cancer Prognostic Gene Signature Assay is an in vitro diagnostic assay which is performed on the NanoString nCounter® Dx Analysis System using FFPE breast tumor tissue previously diagnosed as invasive breast carcinoma. This qualitative assay utilizes gene expression data, weighted together with clinical variables to generate a risk category and numerical score, to assess a patient’s risk of distant recurrence of disease.

The Prosigna Breast Cancer Prognostic Gene Signature Assay is indicated in female breast cancer patients who have undergone surgery in conjunction with locoregional treatment consistent with standard of care, either as:

1. A prognostic indicator for distant recurrence-free survival at 10 years in postmenopausal women with Hormone Receptor-Positive $\left( \mathsf { H R } + \right)$ , lymph node-negative, Stage I or II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors.

2. A prognostic indicator for distant recurrence-free survival at 10 years in postmenopausal women with Hormone Receptor-Positive $\left( \mathsf { H R } + \right)$ , lymph node-positive (1-3 positive nodes), Stage II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors. The device is not intended for patients with 4 or more positive nodes.

# Special Conditions for Use:

Prosigna™ is not intended for diagnosis, to predict or detect response to therapy, or to help select the optimal therapy for patients.

# Device Description:

Used together, the Prosigna™ Breast Cancer Prognostic Gene Signature Assay and nCounter Dx Analysis System are a nucleic acid hybridization, visualization and image analysis system based upon coded probes designed to detect the messenger RNA transcribed from 58 genes. The test input is purified RNA from FFPE breast tumor specimens which are acquired from surgical resection. The Prosigna assay uses gene-specific probe pairs that hybridize directly to the mRNA transcripts in solution. The nCounter Dx Analysis System delivers direct, multiplexed measurements of gene expression through digital readouts of the relative abundance of the mRNA transcripts. Specifications are included as part of the Prosigna Assay to control for sample quality, RNA quality, and process quality. Prosigna simultaneously measures the expression levels of 50 genes used in the PAM50 classification algorithm (Parker et al., 2009), 8 housekeeping genes used for signal normalization, 6 positive controls, and 8 negative controls in a single hybridization reaction, using nucleic acid probes designed specifically to those genes. The Prosigna assay utilizes prototypical expression profiles (centroids) which are associated with and define each of the four PAM50 molecular subtypes of breast cancer. The software algorithm produces a Prosigna Score (referred to as ROR Score or Risk of Recurrence Score in the literature (Dowsett et al., 2013)) based on the similarity of the expression profile to each PAM50 molecular subtype, as well as the gross pathological tumor size and a proliferation score computed from a subset of genes. Three risk categories (low, intermediate and high) were defined based on a study with over 1007 patient samples associating Prosigna score with longterm outcome.

The required components for the Prosigna Assay include the RNA Isolation kit (manufactured by Roche), Prosigna reagents (Reference Sample, CodeSet, Prep Pack, Cartridge(s) and Prep Plate) and the instruments that comprise the nCounter Dx Analysis System; the Prep Station and Digital Analyzer.

The test output is a patient specific report which includes a Prosigna score (0-100) and risk category (low/intermediate/high).

A number of pre-analytical and analytical studies were carried out with the Prosigna Assay to assess the precision, reproducibility, cutoff, sensitivity, specificity and robustness of the assay. Analytical studies also addressed specimen shipping and storage, reagent stability, RNA extraction specifications, tissue requirements, RNA input, cross-hybridization, crosscontamination and tissue interferents testing.

Technical validity was demonstrated in two multi-site (3 sites total), blinded and randomized studies which were designed to test variability across operators, sites, instruments, reagent lots, time, runs and sample position within a 10-sample cartridge. One study assessed reproducibility including pre-analytical factors with a total of 43 tissue samples (FFPE) and the other assessed assay precision with 5 pooled RNA samples. All reproducibility samples were within the intended use patient population indicated by Prosigna, and constituted a large range of Prosigna scores (across 94 Prosigna Score units).

The standard deviation (SD) of the Prosigna Score from the 5 pooled RNA samples was $< 1$ Prosigna Score unit across 3 sites, 3 reagent lots, and 108 measurements of each RNA sample. Using a linear regression and correlation analysis, the normalized gene expression from the 50 classifier genes was compared between the replicate tumor RNA hybridization measurements from all valid samples tested at each site. The average intercept, slope, and Pearson’s correlation (r) of the pair-wise comparisons are reported with the $95 \%$ confidence interval. At each site, the normalized gene expression between RNA replicates was highly correlative with slopes ranging from $0 . 9 8 - 1 . 0 0 $ , intercepts at 0, and r values of 0.99.

Pairwise correlation for Replicate RNA Hybridizations from tissue reproducibility study   

<table><tr><td rowspan=1 colspan=5>Pairwise Comparison of Replicate RNA Hybridizations</td></tr><tr><td rowspan=1 colspan=1>Comparison</td><td rowspan=1 colspan=1>PairwiseComparisons (n)</td><td rowspan=1 colspan=1>Intercept[95% CI]</td><td rowspan=1 colspan=1>Slope[95% CI]</td><td rowspan=1 colspan=1>r[95% CI]</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>0.00[-0.01 , 0]</td><td rowspan=1 colspan=1>0.99[0.99 , 1]</td><td rowspan=1 colspan=1>0.99[0.99 , 0.99]</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>-0.01[-0.01 , 0]</td><td rowspan=1 colspan=1>1.00[0.99 , 1.01]</td><td rowspan=1 colspan=1>0.99[0.99 , 0.99]</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>0.00[-0.01 , 0.01]</td><td rowspan=1 colspan=1>0.98[0.97 , 0.99]</td><td rowspan=1 colspan=1>0.99[0.99 , 0.99]</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0.00[-0.01 , 0.01]</td><td rowspan=1 colspan=1>0.99[0.99 , 1]</td><td rowspan=1 colspan=1>0.99[0.99 , 0.99]</td></tr></table>

Using a linear regression and correlation analysis, the normalized gene expression from the 50 classifier genes was also compared between the tissue replicates from all valid specimens tested at each site. The average intercept, slope, and Pearson’s correlation (r) of the pair-wise comparisons between sites are reported with the $95 \%$ confidence interval. The gene expression between tissue replicates was highly correlative between sites with slopes ranging from 0.97 – 1.00, intercepts at 0, and r values of 0.98 or greater.

Pairwise correlation for Replicate Tissues from tissue reproducibility study   

<table><tr><td rowspan=1 colspan=5>Pairwise Comparison of Tissue Replicates</td></tr><tr><td rowspan=1 colspan=1>Comparison</td><td rowspan=1 colspan=1>PairwiseComparisons (n)</td><td rowspan=1 colspan=1>Intercept[95% CI]</td><td rowspan=1 colspan=1>Slope[95% CI]</td><td rowspan=1 colspan=1>r[95% CI]</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>0.00[-0.01 , 0.01]</td><td rowspan=1 colspan=1>0.98[0.97 , 0.99]</td><td rowspan=1 colspan=1>0.98[0.98 , 0.98]</td></tr><tr><td rowspan=1 colspan=1>Site 1 vs. Site 2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0.00[-0.01 , 0.01]</td><td rowspan=1 colspan=1>0.97[0.95 , 0.98]</td><td rowspan=1 colspan=1>0.98[0.97 , 0.98]</td></tr><tr><td rowspan=1 colspan=1>Site 1 vs. Site 3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0.01[0 , 0.02]</td><td rowspan=1 colspan=1>1.00[0.98 , 1.01]</td><td rowspan=1 colspan=1>0.98[0.98 , 0.99]</td></tr><tr><td rowspan=1 colspan=1>Site 2 vs. Site 3</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>-0.01[-0.02 , 0]</td><td rowspan=1 colspan=1>0.99[0.97 , 1]</td><td rowspan=1 colspan=1>0.99[0.98 , 0.99]</td></tr></table>

The total variability using the sum of the tissue processing variability (including across sites and within tissue samples) as well as the total RNA Processing Variability from the RNA precision study (averaged across the five tested RNA samples) is summarized as a total standard deviation for tissue and RNA Processing of 2.9 Prosigna Score units. A standard deviation of 2.9 Prosigna Score units demonstrates that the Prosigna Assay can reliably measure a difference between two Prosigna Scores of 6.75 with $9 5 \%$ confidence.

Additionally, the concordance in categorical risk classifications across the 43 tissue samples in the Tissue Reproducibility study (node-negative and positive risk categories) between all sites was very high (average concordance greater than $90 \%$ ) .

Additional analyses of the gene expression from samples used in the validation studies shows that the gene expression inherent to the intrinsic biology of breast cancer is the primary factor in explaining the differences in expression in this patient population, which is independent of the patient’s nodal status. For further details, see Package Insert.

# Clinical Performance:

Prosigna’s clinical performance has been verified and validated in two large studies using retrospective tissue samples from 2485 patients within the Intended Use patient population. The first study, TransATAC, demonstrated that Prosigna Score was continuously related to Distant Recurrence-Free Survival (DRFS) at 10 years and was used to select the Prosigna Score cut-offs for low, intermediate, and high risk categories. The second study, ABCSG-8 replicated the result that Prosigna Score was continuously related to DRFS at 10 years and validated the risk group cut-offs. Both the TransATAC and ABCSG-8 study samples were independent from those samples used to train the Prosigna algorithm.

For the ABCSG-8 study, all samples were sent to, and all tests were performed at, an independent academic pathology laboratory. Of the 1,620 tissues available for testing in the ABCSG-8 study, 25 $( 1 . 5 \% )$ did not pass pre-defined pathology review criteria for adequate tumor, 73 of the 1595 tissue samples $( 4 . 6 \% )$ with viable invasive tissue did not pass pre-defined QC specifications for quantity and quality of extracted RNA, and 44 of the 1522 RNA samples $( 2 . 9 \% )$ failed the Prosigna assay QC specifications leaving a total of 1,478 $( 9 1 . 2 \% )$ available for analysis. Of the 1,478 patients available for analysis, 155 had distant recurrences and 194 had local or distant recurrence or death due to breast cancer. The median follow-up for the trial was 10 years.

The table below shows a summary of the primary analysis of the ABCSG-8 study using a Cox proportional hazards model in which (1) Prosigna Score was added to the clinical treatment score (CTS) as a continuous variable and (2) Prosigna Score was added to CTS using the predefined Prosigna Score-based risk groups. In both cases, a null model consisting of CTS alone was compared to an alternate model using a likelihood ratio (LR) test. The table shows the test statistic $( \Delta \mathsf { L R } \chi 2 = - 2 \mathsf { l n } ( \mathsf { L R } ) ) .$ , the critical value for the degrees of freedom for the $\mathtt { a } = 0 . 0 5$ test, and the $\mathsf { p }$ -value based on the $\tt X 2$ distribution.

Summary of Primary Analysis Testing from ABCSG-8 clinical validation study   

<table><tr><td rowspan=1 colspan=1>Null Model</td><td rowspan=1 colspan=1>Alternate Model</td><td rowspan=1 colspan=1>∆LR x^</td><td rowspan=1 colspan=1>x²{ Critical Value(Degrees of freedom)</td><td rowspan=1 colspan=1>$x²2 p-value</td></tr><tr><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>CTS + Prosigna Score</td><td rowspan=1 colspan=1>53.49</td><td rowspan=1 colspan=1>3.84 (df = 1)</td><td rowspan=1 colspan=1>p &lt; 0.0001</td></tr><tr><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>CTS + Risk Groups</td><td rowspan=1 colspan=1>34.12</td><td rowspan=1 colspan=1>5.99 (df = 2)</td><td rowspan=1 colspan=1>p &lt; 0.0001</td></tr></table>

$^ { * } \Delta \mathsf { L } \mathsf { R }$ is used to denote twice the difference of the log likelihoods when comparing two models, e.g., CTS and $\mathsf { C T S } + |$ Prosigna Score. The statistic has an approximate χ2 distribution.

CTS is an optimized combination of clinical and treatment variables (patient age, tumor grade, gross pathological tumor size, nodal status, and adjuvant therapy) which is a best-case approximation of how a physician may use these factors in treatment decisions. When adding Prosigna Score either as a continuous variable or using risk-groups, the Prosigna Score was shown to add significant prognostic information $( \mathsf { p } < 0 . 0 0 0 1 )$ for DRFS over and above that contained in the CTS score.

The table below shows the results of Cox modeling when CTS and the two or three Prosigna Score-based risk groups were included as covariates in the ABCSG-8 study, by nodal status.

Cox Regression Results for Pre-Defined Risk Groups in ABCSG-8 Clinical Validation Study   

<table><tr><td rowspan=2 colspan=1>NodeGroup*</td><td rowspan=2 colspan=1>Variable</td><td rowspan=2 colspan=1>Coefficient</td><td rowspan=2 colspan=1>P-value</td><td rowspan=1 colspan=3>Hazard Ratio</td></tr><tr><td rowspan=1 colspan=1>PointEstimate</td><td rowspan=1 colspan=1>Lower95% CL</td><td rowspan=1 colspan=1>Upper95% CL</td></tr><tr><td rowspan=3 colspan=1>NO</td><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>0.0013</td><td rowspan=1 colspan=1>2.01</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>3.08</td></tr><tr><td rowspan=1 colspan=1>Intermediate vs. LowProsigna Score</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.0015</td><td rowspan=1 colspan=1>2.60</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>4.70</td></tr><tr><td rowspan=1 colspan=1>High vs.LowProsigna score</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>&lt;0.0001</td><td rowspan=1 colspan=1>3.96</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>7.20</td></tr><tr><td rowspan=2 colspan=1>N1</td><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>0.0098</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>3.34</td></tr><tr><td rowspan=1 colspan=1>High vs. Low ProsignaScore</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>0.0002</td><td rowspan=1 colspan=1>4.22</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>9.00</td></tr></table>

\*N0: Node-negative, N1: Node-positive (1-3 nodes)

In the node-negative population, the hazard ratio for Intermediate vs. Low Prosigna Score is statistically significantly greater than 1 $9 5 \%$ confidence interval does not include 1) and that for High vs. Low Prosigna Score is statistically significantly greater than 2 $9 5 \%$ confidence interval does not include 2), i.e. the pre-defined Prosigna Score cutoffs separate the patients into three risk groups (low risk, intermediate risk, high risk) with statistically significantly different outcomes at 10 years.

In the node-positive population (1-3 nodes), the hazard ratio for High vs. Low Prosigna Score is statistically significantly greater than 2 $9 5 \%$ confidence interval does not include 2), i.e. the predefined Prosigna Score cutoffs separate the patients into two risk groups (low risk, high risk) with statistically significantly different outcomes at 10 years.

The cutoffs for the risk group classifications were defined based on the results of the TransATAC study:   

<table><tr><td>Risk Group</td><td>Risk of distant recurrence by 10 years</td><td>Prosigna Score Range for Node-Negative</td><td>Prosigna Score Range for Node-Positive (1-3 Nodes)</td></tr><tr><td>Low</td><td>&lt; 10%</td><td>0-40</td><td rowspan="2">0-40</td></tr><tr><td>Intermediate</td><td>10 - 20%</td><td>41-60</td></tr><tr><td>High</td><td>&gt; 20%</td><td>61-100</td><td>41-100</td></tr></table>

The following figures are the Kaplan-Meier curves showing the percent of patients without distant recurrence by risk-group through 10 years for all patients from the ABCSG-8 study, by nodal status.

![](images/f3b494ecdd8d9bee19088dde442ca9e1804734720dcf425acab0dbb4c49fbb35.jpg)

Summary: DRFS by Risk Group for Node-Negative Patients   

<table><tr><td rowspan=1 colspan=1>Risk Group</td><td rowspan=1 colspan=1>Number ofPatients (%)</td><td rowspan=1 colspan=1>Number of EventsThrough 10 Years</td><td rowspan=1 colspan=1>Estimated Percent WithoutRecurrence at 10 years [95% CI]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>487 (47%)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>96.6% [94.4% - 97.9%]</td></tr><tr><td rowspan=1 colspan=1>Intermediate</td><td rowspan=1 colspan=1>335 (32%)</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>90.4% [86.3% - 93.3%]</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>225 (21%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>84.3% [78.4% - 88.6%]</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,047 (100%)</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>EY</td></tr></table>

![](images/acb2263ba9c4e64b3ba6b122cc77ddd1a9229cfef2869e5c97550b33f88c31f4.jpg)

Summary: DRFS by Risk Group for Node-Positive (1-3 nodes) Patients   

<table><tr><td rowspan=1 colspan=1>Risk Group</td><td rowspan=1 colspan=1>Number of Patients (%)</td><td rowspan=1 colspan=1>Number of EventsThrough 10 Years</td><td rowspan=1 colspan=1>Estimated Percent Without DistantRecurrence at 10 years [95% CI]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>158 (41%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>94.2% [88.1%-97.2%]</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>224 (59%)</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>75.8% [68.9%-81.4%]</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>382 (100%)</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1></td></tr></table>

The Prosigna Score was demonstrated to add significant prognostic information over and above the standard clinical and treatment variables both when included as a continuous measure and when included using pre-defined risk groups. The low-risk groups (each of node-negative and node-positive patients) had 10-year DRFS well above $90 \%$ and was separated from the high-risk group by more than a $10 \%$ probability of recurrence at 10 years. The Prosigna Score (continuous and risk-group based) showed similar prognostic information in various subgroups.

A C-index analysis was used to evaluate the correlation between the Prosigna Score and the time to distant recurrence. The C-index analysis was restricted to comparing patient samples with Prosigna Scores that differed by only 5-10 Prosigna Score units. This analysis showed that there is statistically significant information in small changes in Prosigna Score $\scriptstyle ( { \mathsf { P } } < 0 . 0 5 )$ . Based on the analytical precision and reproducibility studies and the restricted C-index analysis of 5-10 ΔProsigna Score units, a difference of Prosigna Score of 7 is shown to be both a reliable measure of difference of the Prosigna test performance (statistically reproducible based on analytical studies), and of clinical utility (clinically meaningful based on restricted C-index analysis).

The analytical performance studies in combination with pre-analytical studies validate that the Prosigna assay is appropriate for use as a distributed gene expression profiling test system for breast cancer prognosis. The clinical studies demonstrate the validity of a risk classifier that includes High, Intermediate and Low risk groups (where indicated) as well as a continuous risk score that outputs an integer Prosigna Score of 0-100.

Predicate Device: Agendia, MammaPrint K062694, K081092

<table><tr><td colspan="3" rowspan="1">Substantial Equivalence Comparison Table</td></tr><tr><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">New Device (Prosigna™ BreastCancer Prognostic Gene SignatureAssay)</td><td colspan="1" rowspan="1">Predicate Device(MammaPrint, K062694,K081092)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Prosigna™ Breast CancerPrognostic Gene Signature Assay isan in vitro diagnostic assay which isperformed on the NanoStringnCounter® Dx Analysis System usingFFPE breast tumor tissue previouslydiagnosed as invasive breastcarcinoma. This qualitative assayutilizes gene expression data,weighted together with clinicalvariables to generate a risk categoryand numerical score, to assess apatient's risk of distant recurrence ofdisease.The   Prosigna   Breast   CancerPrognostic Gene Signature Assay isindicated in female breast cancerpatientswhohave undergonesurgery   in  conjunctionwithlocoregional treatment consistentwith standard of care, either as:1.      A prognostic indicator fordistant recurrence-free survival at10 years in post-menopausal womenwith Hormone Receptor-Positive(HR+), lymph node-negative, Stage Ior Il breast cancer to be treated withadjuvant endocrine therapy alone,when used in conjunction with otherclinicopathological factors.2.      A prognostic indicator for</td><td colspan="1" rowspan="1">K062694MammaPrint is a qualitative invitro diagnostic test service,performed in a singlelaboratory, using the geneexpression profile of freshfrozen breast cancer tissuesamples to assess a patients'risk for distant metastasis.The test is performed forbreast cancer patients whoare less than 61 years old,with Stage I or Stage IIdisease, with tumor size ≤5.0cm and lymph node negative.The MammaPrint result isindicated for use by physiciansas a prognostic marker only,along with otherclinicopathological factorsK081092MammaPrint is a qualitativein vitro diagnostic test service,performed in a singlelaboratory, using the geneexpression profile of freshbreast cancer tissue samplesto assess a patient's risk fordistant metastasis.The test is performed forbreast cancer patients, with</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">distant recurrence-free survival at10 years in post-menopausal womenwith Hormone Receptor-Positive(HR+), lymph node-positive (1-3positive nodes), Stage II breastcancer to be treated with adjuvantendocrine therapy alone, when usedin   conjunction   with   otherclinicopathological factors.   Thedevice is not intended for patientswith 4 or more positive nodes.</td><td colspan="1" rowspan="1">Stage I or Stage 1l disease,with tumor size &lt;= 5.0 cm andwho are lymph node negative.The MammaPrint result isindicated for use by physiciansas a prognostic marker only,along with otherclinicopathological factors.</td></tr><tr><td colspan="1" rowspan="1">Indications</td><td colspan="1" rowspan="1">Same as intended use</td><td colspan="1" rowspan="1">Same as intended use</td></tr><tr><td colspan="1" rowspan="1">Special conditions foruse statement(s)</td><td colspan="1" rowspan="1">For prescription use only.Prosigna™is not intended fordiagnosis, to predict or detectresponse to therapy, or to helpselect the optimal therapy forpatients</td><td colspan="1" rowspan="1">For prescription use only.MammaPrint® is not intendedfor diagnosis, or to predict ordetect response to therapy, orto help select the optimaltherapy for patients</td></tr><tr><td colspan="1" rowspan="1">Device Description</td><td colspan="1" rowspan="1">Prosigna™ Breast Cancer PrognosticGene Signature Assay and nCounterDx Analysis Platform; all elementscleared by FDA as a distributed testand platform</td><td colspan="1" rowspan="1">Microarray-based assayperformed as a service at asingle site; includesinstrumentation that iscleared for use at thedesignated site</td></tr><tr><td colspan="1" rowspan="1">Test Sample</td><td colspan="1" rowspan="1">FFPE tumor samples</td><td colspan="1" rowspan="1">Fresh frozen or freshpreserved tissue sections</td></tr><tr><td colspan="1" rowspan="1">Extraction/amplificationreagents/amplificationprocedures</td><td colspan="1" rowspan="1">No amplification required;procedure for processing FFPEtumor samples provided; includesRNA isolation, multiplexhybridization in solution, automatedpurification on a liquid handlingrobot and analysis on an automatedepifluorescence microscope</td><td colspan="1" rowspan="1">Amplification required; singlesite handles entire protocolstarting from tissue; includesRNA isolation, labelingamplification, microarrayhybridization and scanning</td></tr><tr><td colspan="1" rowspan="1">Validation population</td><td colspan="1" rowspan="1">Treatment arms from a randomizedtrial conducted in Europe;prospective retrospective studydesign</td><td colspan="1" rowspan="1">European cohort; literaturebased support</td></tr><tr><td colspan="1" rowspan="1">Method ComparisonResults</td><td colspan="1" rowspan="1">Not applicable</td><td colspan="1" rowspan="1">Not applicable</td></tr><tr><td colspan="1" rowspan="1">Clinical Studies</td><td colspan="1" rowspan="1">1478 patients evaluated resulting inoverall percentage without distantrecurrence at 10 years separated bythree risk groups in node-negativepatients (n=1047): low risk 96.6%(94.4%-97.9%), intermediate risk</td><td colspan="1" rowspan="1">302 patients evaluated(K062694) resulting in ametastasis-free survival byprofile at 10 yrs (for patientsless than 61 years old): lowrisk profile 90% (85-96%), high</td></tr><tr><td></td><td>90.4% (86.3%-93.3%), high risk 84.3% (78.4%-88.6%) and by two risk groups in node-positive (1-3 nodes) patients (n=382): low risk 94.2% (88.1%-97.2%), high risk 75.8% (68.9%-81.4%)</td><td>risk profile 71% (65-78%) (at 5 yrs: 95% (91%-99%) and 78% (72%-84%) respectively) 131 patients were evaluated (K081092) for 5 year survival. The study showed that the device could categorize risk of</td></tr><tr><td></td><td></td><td>metastatic disease within 5 years for patients ≥ 61 years with PPV = 0.22 (0.12-0.38) NPV = 0.93 (0.85-0.97)</td></tr></table>

Based on the Intended Use of the Prosigna Breast Cancer Prognostic Gene Signature Assay and the results of the performance and analytical studies provided in the 510(k), the Prosigna Breast Cancer Prognostic Gene Signature Assay is found to be Substantially Equivalent to the predicate device, Mammaprint.

# References:

Dowsett M. et al. on behalf of the ATAC and LATTE Trialists Group. Comparison of PAM50 Risk of Recurrence Score With Oncotype DX and IHC4 for Predicting Risk of Distant Recurrence After Endocrine Therapy. Journal of Clinical Oncology. ePub ahead of print July 1, 2013 as 10.1200/JCO.2012.46.1558

Parker J.S., et. al., Supervised Risk Predictor of Breast Cancer Based on Intrinsic Subtypes. Journal of Clinical Oncology, v27 No.8 (2009) 1160-1167