# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K130010   
B. Purpose for Submission: New device   
C. Measurand: 58 gene RNA expression profile   
D. Type of Test: Gene expression profile system based upon non-amplified RNA hybridization, visualization, and image analysis   
E. Applicant: NanoString Technologies   
F. Proprietary and Established Names: ProsignaTM Breast Cancer Prognostic Gene Signature Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR $\ S 8 6 6 . 6 0 4 0 $ Gene expression profiling test system for breast cancer prognosis

2. Classification: Class II

3. Product code: NYI, Classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer

4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s): The Prosigna™ Breast Cancer Prognostic Gene Signature Assay is an in vitro diagnostic assay which is performed on the NanoString nCounter $\textsuperscript { \textregistered }$ Dx Analysis System using FFPE breast tumor tissue previously diagnosed as invasive breast carcinoma. This qualitative assay utilizes gene expression data, weighted together with clinical variables to generate a risk category and numerical score, to assess a patient’s risk of distant recurrence of disease.

The Prosigna Breast Cancer Prognostic Gene Signature Assay is indicated in female breast cancer patients who have undergone surgery in conjunction with locoregional treatment consistent with standard of care, either as:

1. A prognostic indicator for distant recurrence‐free survival at 10 years in post‐menopausal women with hormone receptor‐positive $\left( \mathrm { H R + } \right)$ , lymph node‐negative, Stage I or II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors.

2. A prognostic indicator for distant recurrence‐free survival at 10 years in post‐menopausal women with hormone receptor‐positive $\left( \mathrm { H R + } \right)$ , lymph node‐positive (1‐3 positive nodes), Stage II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors. The device is not intended for patients with 4 or more positive nodes.

Special Conditions for Use: Prosigna is not intended for diagnosis, to predict or detect response to therapy, or to help select the optimal therapy for patients.

2. Indication(s) for use:

Same as intended use

3. Special conditions for use statement(s):

For Prescription Use Only

Prosigna™ Breast Cancer Prognostic Gene Signature Assay is not intended for diagnosis, or to predict or detect response to therapy, or to help select the optimal therapy for patients.

4. Special instrument requirements: nCounter Dx Analysis System

# I. Device Description:

The required components for the Prosigna™ Breast Cancer Prognostic Gene Signature Assay include the RNA Isolation kit (manufactured by Roche), Prosigna reagents (Reference Sample, CodeSet, Prep Pack, Cartridge(s) and Prep Plate) and the instruments that comprise the nCounter Dx Analysis System; the Prep Station and Digital Analyzer.

The assay requires microdissection of tumor from FFPE biopsies, isolation of RNA using a Roche RNA isolation kit, transfer of RNA to PCR tubes for hybridization before placing onto the prep station. Two sets of probes specific to each of 58 RNAs are added to the

hybridization reaction. These consist of biotin-labeled magnetic probes to purify the RNAs and capture them on the assay cartridge and fluorescent “barcode” probes to detect and quantify individual RNAs. The patient sample and probes are pipetted automatically into the Prosigna test cartridge by the Prep Station. The prep station uses magnetic bead capture and washing to remove excess RNA and un-hybridized probes. The isolated and hybridized RNA species are then bound via biotin on the capture probe randomly to streptavidin on the cartridge. The fluorescent molecules are then aligned on the cartridge by addition of an electric current. The cartridge is then transferred to the Digital Analyzer where the cartridge is scanned and digital analysis software is used to count the number of each RNA species present. The amount of each RNA is then put into a proprietary algorithm to produce a Prosigna score.

The test output is a patient specific report which includes a Prosigna score (0-100) and risk category (low/intermediate/high).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Agendia BV’s MammaPrint®

2. Predicate 510(k) number(s): k062694, k081092

3. Comparison with predicate:

Table 1: Comparison with Predicate   

<table><tr><td colspan="3" rowspan="1">Similarities and Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">New Device (ProsignaTMBreast Cancer PrognosticGene Signature Assay)</td><td colspan="1" rowspan="1">Predicate Device(MammaPrint, K062694,K070675)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The ProsignaTM BreastCancer Prognostic GeneSignature Assay is an invitro diagnostic assay whichis performed on theNanoString nCounter® DxAnalysis System usingFFPE breast tumor tissuepreviously diagnosed asinvasive breast carcinoma.This qualitative assayutilizes gene expressiondata, weighted together withclinical variables to generatea risk category and</td><td colspan="1" rowspan="1">MammaPrint is aqualitative in vitrodiagnostic test service,performed in a singlelaboratory, using the geneexpression profile of fresh-frozen breast cancer tissuesamples to assess apatients' risk for distantmetastasis.The test is performed forbreast cancer patients whoare less than 61 years old,with Stage I or Stage IIdisease, with tumor size</td></tr><tr><td>Item</td><td>Device</td><td>Predicate &lt;5.0 cm and lymph node</td></tr><tr><td></td><td>numerical score, to assess a patient's risk of distant recurrence of disease. The Prosigna Breast Cancer Prognostic Gene Signature Assay is indicated in female breast cancer patients who have undergone surgery in conjunction with locoregional treatment consistent with standard of care, either as: 1. A prognostic indicator for distant recurrence-free survival at 10 years in post-menopausal women with Hormone Receptor-Positive (HR+), lymph node-negative, Stage I or II breast cancer to be treated with adjuvant endocrine therapy alone, when used in conjunction with other clinicopathological factors. 2. A prognostic indicator for distant recurrence-free survival at 10 years in post-menopausal women with Hormone Receptor-Positive (HR+), lymph node-positive (1-3 positive nodes), Stage II breast cancer to be</td><td>negative. The MammaPrint result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">when used inconjunction with otherclinicopathologicalfactors. The device isnot intended for patientswith 4 or more positivenodes.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Special conditions foruse statement(s)</td><td colspan="1" rowspan="1">For prescription use only.Not intended for diagnosis,or to predict or detectresponse to therapy, or tohelp select the optimaltherapy for patients.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Device Description</td><td colspan="1" rowspan="1">ProsignaTM Breast CancerPrognostic Gene SignatureAssay of 58 RNAtranscripts on nCounter DxAnalysis Platform</td><td colspan="1" rowspan="1">Microarray-based assay of70 RNA transcriptsperformed as a service at asingle site</td></tr><tr><td colspan="1" rowspan="1">Test Sample</td><td colspan="1" rowspan="1">FFPE tumor samples</td><td colspan="1" rowspan="1">Fresh frozen or freshpreserved tissue sections</td></tr><tr><td colspan="1" rowspan="1">Extraction/amplificationreagents/amplificationprocedures</td><td colspan="1" rowspan="1">No amplification required;procedure for processingFFPE tumor samplesprovided; includes RNAisolation, multiplexhybridization in solution,automated purification on aliquid handling robot andanalysis on an automatedepifluorescence microscope</td><td colspan="1" rowspan="1">Amplification required;single site handles entireprotocol starting fromtissue; includes RNAisolation, labelingamplification, microarrayhybridization and scanning</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: “Gene Expression Profiling Test System for Breast Cancer Prognosis, issued on May 9, $2 0 0 7 '$

# L. Test Principle:

Used together, the Prosigna™ Breast Cancer Prognostic Gene Signature Assay and nCounter Dx Analysis System are a nucleic acid hybridization, visualization and image analysis system based upon coded probes designed to detect the messenger RNA transcribed from 58 genes. The test input is purified RNA from FFPE breast tumor specimens which are acquired from surgical resection. The Prosigna assay uses gene-specific probe pairs that hybridize directly to the mRNA transcripts in solution. The nCounter Dx Analysis System delivers direct,

multiplexed measurements of gene expression through digital readouts of the relative abundance of the mRNA transcripts.

Specifications are included as part of the Prosigna Assay to control for sample quality, RNA quality, and process quality. The Prosigna assay utilizes prototypical expression profiles (centroids) for breast cancer. Patients RNA signatures are categorized into one of four centroids (not reported) based upon how close their gene expression pattern is to each of the centroids. The software algorithm produces a Prosigna score based on the similarity of the expression profile to each centroid, as well as the pathological tumor size and a proliferation score computed from a subset of genes. Three risk categories (low, intermediate and high) were defined based on a study with over 1007 patient samples associating Prosigna score with long-term outcome, defined by distance recurrence free survival at 10 years (DRFS) (Table 2).

Table 2: Risk Classification Scoring Algorithm Using Prosigna Score   

<table><tr><td rowspan=1 colspan=1>Nodal Status</td><td rowspan=1 colspan=1>Prosigna Score Range</td><td rowspan=1 colspan=1>Risk Classification</td></tr><tr><td rowspan=3 colspan=1>Node-Negative</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-60</td><td rowspan=1 colspan=1>Intermediate</td></tr><tr><td rowspan=1 colspan=1>61-100</td><td rowspan=1 colspan=1>High</td></tr><tr><td rowspan=2 colspan=1>Node-Positive(1-3 nodes)</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-100</td><td rowspan=1 colspan=1>High</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

i. Precision/Reproducibility starting from RNA pools: Five pooled breast tumor RNA samples were generated from 43 archived FFPE breast tumor tissue samples containing viable invasive breast carcinoma to comprise a panel that represented each of the three risk classification groups (low, intermediate and high) (Tables 3).

Table 3: RNA pools used for RNA Reads Precision Study   

<table><tr><td rowspan=1 colspan=1>PAM50Signature</td><td rowspan=1 colspan=1>Prosigna Score FromPrecision Study</td><td rowspan=1 colspan=1>Tumor Size</td><td rowspan=1 colspan=1>Nodal Status</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>&gt;2cm</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>&gt;2cm</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>&gt;2cm</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>&gt;2cm</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=2 colspan=1>3*</td><td rowspan=2 colspan=1>65</td><td rowspan=1 colspan=1>≤2cm</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>&gt;2cm</td><td rowspan=1 colspan=1>Negative</td></tr></table>

\* As per the protocol, RNA from two different FFPE breast tumor blocks was pooled to reach the amount of RNA mass required to complete the RNA precision study.

Single use aliquots of each pooled breast tumor RNA sample were distributed to each of three testing sites along with 3 ProsignaTM Breast Cancer Prognostic Gene Signature Assay kit lots for testing. Each site completed 18 valid runs (9 runs by each of 2 operators with one run per day). Each sample was tested in duplicate at $2 5 0 \mathrm { n g }$ RNA for each run. There are 108 independent measurements for each breast tumor RNA sample. Variation of RNA reads was calculated and is summarized in Table 4.

Table 4: Variance Components by Panel Member (pooled RNA sample)   

<table><tr><td rowspan=2 colspan=1>MeanProsignaScore</td><td rowspan=1 colspan=5>VarianceComponent</td><td rowspan=2 colspan=1>TotalVariance</td><td rowspan=2 colspan=1>TotalSD</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>Run</td><td rowspan=1 colspan=1>Within-Run</td></tr><tr><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>0.010(2%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.134(30%)</td><td rowspan=1 colspan=1>0.296(67%)</td><td rowspan=1 colspan=1>0.44(100%)</td><td rowspan=1 colspan=1>0.66</td></tr><tr><td rowspan=1 colspan=1>55.0</td><td rowspan=1 colspan=1>0.105(18%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.046(8%)</td><td rowspan=1 colspan=1>0.426(74%)</td><td rowspan=1 colspan=1>0.576(100%)</td><td rowspan=1 colspan=1>0.76</td></tr><tr><td rowspan=1 colspan=1>55.4</td><td rowspan=1 colspan=1>0.059(20%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.046(15%)</td><td rowspan=1 colspan=1>0.194(65%)</td><td rowspan=1 colspan=1>0.299(100%)</td><td rowspan=1 colspan=1>0.55</td></tr><tr><td rowspan=1 colspan=1>64.8</td><td rowspan=1 colspan=1>0.119(21%)</td><td rowspan=1 colspan=1>0.014(2%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.064(11%)</td><td rowspan=1 colspan=1>0.380(66%)</td><td rowspan=1 colspan=1>0.576(100%)</td><td rowspan=1 colspan=1>0.76</td></tr><tr><td rowspan=1 colspan=1>76.2</td><td rowspan=1 colspan=1>0.165(37%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.000(0%)</td><td rowspan=1 colspan=1>0.277(63%)</td><td rowspan=1 colspan=1>0.442(100%)</td><td rowspan=1 colspan=1>0.66</td></tr></table>

ii. Precision/Reproducibility starting from tumor tissue

A set of 43 FFPE breast tumor specimens from hormone receptor positive breast cancer patients, including both node‐negative and node‐positive patients, were processed for RNA extraction and testing at each of the three sites. Three lots of the Roche FFPET RNA Isolation Kit (one per site) and a single lot of assay kit reagents were used. A single slide was input for RNA extraction when the tumor surface area measured $\geq 1 0 0 \mathrm { m m } ^ { 2 }$ , and 3 slides were input when the tumor surface measured $< 1 0 0 \mathrm { m m } ^ { 2 }$ with a minimum tumor surface area of $4 \mathrm { m m } ^ { 2 }$ required.

The calculated test results from the 43 specimens represent a wide range (94 units) of Prosigna scores, gene expression profiles, and all risk categories when applying the node‐negative or node‐positive cutoffs to all specimens (Table 5). Results for two samples were not obtained at all 3 sites and were excluded from all subsequent statistical analyses.

The study showed that RNA yields per slide mounted tissue section were highly correlated $( \mathrm { r } > 0 . 9 )$ when repeat isolations were performed from the same FFPE blocks across 3 sites, 3 operators, and 3 RNA extraction lots using the FFPE RNA extraction kit. The measured tumor surface area for 4/5 RNA isolation failures $\mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm { \mathrm \Delta } \mathrm  \mathrm \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm \mathrm { \Delta } \mathrm \mathrm \mathrm { \mathrm \Delta } \mathrm \mathrm  \mathrm \mathrm \mathrm { } \mathrm \mathrm \mathrm { \Delta } \mathrm \mathrm \mathrm { \Delta \Delta } \mathrm \mathrm \mathrm \mathrm \mathrm { \Delta \Delta } \mathrm \mathrm \mathrm \mathrm  \mathrm \mathrm \mathrm \mathrm { } \mathrm \mathrm \mathrm \mathrm \mathrm  \Delta \Delta \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm { \Delta } \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm  \Delta \Delta \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm  \Delta \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm  \Delta \mathrm \Delta \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm  \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm \mathrm $ , equaling less than $5 0 \mathrm { m m } ^ { 2 }$ total tissue by area input into the test. The mean RNA purity (A260/A280) at each site was close to a theoretically pure RNA measurement of 2.0 (2.0 at two sites and 1.9 at the other site) with a SD of 0.1, supporting the RNA isolation procedure and specifications for the Prosigna assay.

$100 \%$ of samples passing tissue review and RNA isolation specifications yielded passing results from the Prosigna Assay. Prosigna scores were reproducible from site to site with a total $\% \mathrm { C V } { = } 7 . 2 \%$ for the assay at the cutoff of 40 and a total $\% { \mathrm { C V } } { = } 4 . 8 \%$ at the cutoff of 60.

Table 5: Concordance of risk category between node positive and node negative samples in tissue reproducibility study.   

<table><tr><td rowspan="2">Comparison Type</td><td colspan="3">Pairwise Concordance</td></tr><tr><td>Site 1 vs. Site 2 (n=40)</td><td>Site 2 vs. Site 3 (n=41)</td><td>Site 1 vs. Site 3 (n=40)</td></tr><tr><td>Risk Category (Node Negative)</td><td>87.5% [73.9% -94.5%]</td><td>92.7% [80.6% -97.5%]</td><td>90% [76.9% - 96%]</td></tr><tr><td>Risk Category (Node Positive)</td><td>88.8% [75.4% -95.3%]</td><td>92.7% [80.6% -97.5%]</td><td>91.3% [78.5% -96.7%]</td></tr></table>

# b. Linearity/assay reportable range:

Linearity studies are not applicable to this type of device. The Prosigna scores associated with risk may be achieved by varying amounts of 50 gene expression products. Therefore a standard is not available for determining linear relationships.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

i. Controls

ProsignaTM Breast Cancer Prognostic Gene Signature Assay includes three pools of RNA as controls:

· One is a quantitation reference sample, containing known amounts of each of the 50 test genes and 8 housekeeping genes. This control is run independently of patient samples and used to compare each signal from the test sample.   
· A second is a series of 6 positive controls. These are added to each patient sample to guarantee that the sample has been prepared and hybridized properly.   
· The third is a series of 8 negative controls. These are added to the patient samples and are used to guarantee that no contamination of the patient RNA exists.

In addition, users are instructed to identify and maintain clinically relevant control materials for each of the risk categories.

# ii. Analyte Stability

The stability of the RNA analytes (50 gene signature) assessed by the Prosigna device were assessed prospectively. RNA was isolated periodically over a 12 month period from the same tumor samples. No significant change in assessment scores (delta $\leq 1$ unit on a 94 unit scale) was observed. This demonstrated that RNA was stable for approximately 9 months under recommended storage.

# d. Detection limit:

The geometric mean (geomean) of the housekeeping genes is used as a measure of the quality of RNA extracted from the FFPE tissue sample and then input into the NanoString Prosigna assay. A geomean cutoff is set to qualify input breast tumor RNA for further analysis and is calculated by the system software in order to determine if a detectable amount of RNA is present for this assay.

A reproducible result could be achieved with the Prosigna Assay when using minimum tissue requirements that included the following specifications:

Biopsy material used to isolate RNA could contain no less than $50 \%$ tumor (no more than $50 \%$ normal cells surrounding the tumor mass); · Tumor mass must contain at least $10 \%$ tumor cells (in presence of necrotic, inflammatory or DCIS components); · Tumor surface area on the H & E stained slide must $\mathsf { b e } \geq 4 \mathrm { m m } ^ { 2 }$ ; $\bullet$ Total surface area of tumor must be at least $2 4 \mathrm { m m } ^ { 2 }$ ( $4 \mathrm { m m } \mathrm { x } \ 6$ slides); · Minimum amount of isolated RNA should be $1 2 5 { \mathrm { n g } }$ ; · RNA concentration Optical Density at $2 6 0 \ : \mathrm { { n m } } \geq 1 2 . 5 \ : \mathrm { { n g / \mu L } }$ ; · RNA purity (OD 260/280 nm) Ratio: 1.7 – 2.3.

Use of less material resulted in either increased failure of RNA quality or in lower agreement rates on repeated runs.

# e. Analytical specificity:

Interference was detected from samples containing greater than $50 \%$ normal tissue in the entire sample. Interference was also detected from contaminating genomic DNA.

The presence of necrotic /hemorrhagic/DCIS contamination had minimal effect on Prosigna scores. RNA specifications and sample preparation instructions provided are adequate to exclude the presence of any effect from likely interfering substances.

# f. Assay cut-off:

The Prosigna score cut-offs were set separately for the Node-Negative population and the Node-Positive (1-3 nodes) population (Table 6).

Table 6: Risk Classification Scoring Algorithm Using Prosigna Score   

<table><tr><td rowspan=1 colspan=1>Nodal Status</td><td rowspan=1 colspan=1>Prosigna Score Range</td><td rowspan=1 colspan=1>Risk Classification</td></tr><tr><td rowspan=3 colspan=1>Node-Negative</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-60</td><td rowspan=1 colspan=1>Intermediate</td></tr><tr><td rowspan=1 colspan=1>61-100</td><td rowspan=1 colspan=1>High</td></tr><tr><td rowspan=2 colspan=1>Node-Positive (1-3 nodes)</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-100</td><td rowspan=1 colspan=1>High</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable. FFPE tissue is the only matrix indicated for this device.

3. Clinical studies:

Clinical samples were left over from the ABSCG trial. Patients in this trial were postmenopausal women with $\mathrm { H R + }$ , early stage breast cancer treated with 5 years of tamoxifen or 2 years of Tamoxifen followed by 3 years of Anastrozole (ABCSG8 trial), the distant recurrence-free survival over 10 years by risk categorization is represented for node negative patients or node positive patients with 1-3 nodes present. For patients with 4 or more nodes, the test is not indicated as these patients are all at high risk of recurrence.

The validation cohort represents the fraction of the evaluable ABCSG‐8 cohort for which tissue specimens could be collected from the retrospectively archived ABCSG tumor bank and for which informed consent could be obtained, or the patient was deceased. Patients who meet the eligibility criteria for the original trial were only excluded either because tissue was unavailable for NanoString's assay to be performed or the patient could not be re-consented. All samples with a tumor block and patient consent available were tested as part of this study, resulting in 1478 patient samples with clinical outcomes available for analysis.

The primary endpoint was distant recurrence‐free survival (DRFS). This was defined as the interval from diagnosis until distant recurrence or death due to breast cancer. Using all available patient samples, multivariate Cox proportional hazards (PH) models were fitted to evaluate the primary objective in sequential tests of Prosigna score. The model included the standard clinical covariates (age, tumor grade, gross tumor size, nodal status, adjuvant therapy). A Cox Proportional Hazards model was then fitted and a likelihood ratio test was used to test whether Prosigna score added statistically significant $( \mathfrak { a } = 0 . 0 5 )$ additional prognostic information. The primary analyses were repeated for different patient subsets (all, node‐ negative, node‐positive) and endpoint (DRFS).

# a. Clinical positive predictive value:

Positive predictive value (or absolute risk) is calculated using the data from the ABCSG-8 trial. Positive predictive value (PPV) is the probability that an event occurs (e.g. metastatic disease occurs within 10 years) given the device output for that patient is high risk.

Distant Recurrence within 10 years for Node-Negative Patients:

$$
\mathrm { P P V } = 1 5 . 7 \% [ 1 1 . 4 \% - 2 1 . 6 \% ]
$$

Distant Recurrence within 10 years for Node-positive (1-3 positive nodes) Patients:

$$
\mathrm { P P V } = 2 4 . 2 \% \left[ 1 8 . 6 \% - 3 1 . 1 \% \right]
$$

b. Clinical negative predictive value:

Negative predictive value is calculated using the data from the ABCSG-8 trial. Negative predictive value (NPV) (or 1- absolute risk) is the probability that an event does not occur (i.e. metastatic disease does not occur within 10 years) given the device output for that patient is low.

Distant Recurrence within 10 years for Node-Negative Patients:

$$
\mathrm { N P V } = 9 6 . 6 \% \left[ 9 4 . 4 \% - 9 7 . 9 \% \right]
$$

Distant Recurrence within 10 years for Node-positive (1-3 positive nodes) Patients:

$$
\mathrm { N P V } = 9 4 . 2 \% \left[ 8 8 . 1 \% - 9 7 . 2 \% \right]
$$

c. Other clinical supportive data (when a. and b. are not applicable):

ABCSG-8 Study

The ABCSG-8 trial was an independent European validation of the Prosigna™ Breast Cancer Prognostic Gene Signature Assay in post-menopausal women with $\mathrm { H R ^ { + } } .$ , early stage breast cancer treated with 5 years of tamoxifen or 2 years of tamoxifen followed

by 3 years of Anastrozole. The risk of distant-recurrence free survival was determined by correlating patient Prosigna scores with clinical outcomes. Results of the study are summarized below:

# i. Proportional Hazards for Prosigna Score

The table below (Table 7) shows a summary of the primary analysis of the ABCSG-8 study using a Cox proportional hazards model in which (1) Prosigna score was added to the clinical treatment score (CTS) as a continuous variable or (2) Prosigna score was added to CTS using the pre-defined Prosigna score-based risk groups. In both cases, a null model consisting of CTS alone was compared to an alternate model using a likelihood ratio (LR) test. The table shows the test statistic $( \Delta \mathrm { L R } \chi 2 = - 2 \mathrm { l n } ( \mathrm { L R } ) )$ , the critical value for the degrees of freedom for the $\mathfrak { a } = 0 . 0 5$ test, and the p-value based on the $\chi 2$ distribution.

Table 7: Summary of Primary Analysis Testing from ABCSG-8 clinical validation study   

<table><tr><td rowspan=1 colspan=1>NullModel</td><td rowspan=1 colspan=1>Alternate Model</td><td rowspan=1 colspan=1>ΔLR χ2</td><td rowspan=1 colspan=1>χ2 Critical Value(Degrees of freedom)</td><td rowspan=1 colspan=1>χ2 p-value</td></tr><tr><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>CTS + Prosignascore</td><td rowspan=1 colspan=1>53.49</td><td rowspan=1 colspan=1>3.84 (df = 1)</td><td rowspan=1 colspan=1>p &lt; 0.0001</td></tr><tr><td rowspan=1 colspan=1>CTS</td><td rowspan=1 colspan=1>CTS + Risk Groups</td><td rowspan=1 colspan=1>34.12</td><td rowspan=1 colspan=1>5.99 (df = 2)</td><td rowspan=1 colspan=1>p &lt; 0.0001</td></tr></table>

\*ΔLR is used to denote twice the difference of the log likelihoods when comparing two models, e.g., CTS and $\mathrm { C T S } + \mathrm { R O R }$ (Prosigna Score). The statistic has an approximate $\chi 2$ distribution.

CTS is an optimized combination of clinical and treatment variables (patient age, tumor grade, gross pathological tumor size, nodal status, and adjuvant therapy) which is a best-case approximation of how a physician may use these factors in treatment decisions. In all cases, age, treatment and grade are not significant when other variables are included in the model. Tumor stage was a significant predictor for the node-negative patients but not for the node-positive patients. When adding Prosigna score either as a continuous variable or using risk-groups, the Prosigna score was shown to add significant prognostic information $\mathrm { ( p < 0 . 0 0 0 1 }$ ) for DRFS as compared to the optimized combination of clinical and treatment variables.

ii. Risk Estimation

# 1) Risk estimation in node-negative patients

The results indicated that three clinical risk categories associated with each Prosigna score (Low, Intermediate, High), as set by the Trans-ATAC study were validated for node –negative patients in the ABCSG-8 patients. The following charts (Figure 1 and Table 8) show the Kaplan-Meier curves of the percent of Node negative patients without distant recurrence by risk-group through 10 years from the ABCSG-8 study.

![](images/64043674ebcbb47e06491acac7d518e97154a6e7e36e92d58f5afb97573c6365.jpg)  
Figure 1: DRFS by Risk Group for Node-Negative Patients

Table 8: Summary of DRFS by Risk Group for Node-Negative Patients   

<table><tr><td rowspan=1 colspan=1>Risk Group</td><td rowspan=1 colspan=1>Number ofPatients (%)</td><td rowspan=1 colspan=1>Number ofEventsThrough10 Years</td><td rowspan=1 colspan=1>Estimated PercentWithout Recurrenceat 10 years</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>487 (47%)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>96.6%[94.4% - 97.9%]</td></tr><tr><td rowspan=1 colspan=1>Intermediate</td><td rowspan=1 colspan=1>335 (32%)</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>90.4%[86.3% - 93.3%]</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>225 (21%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>84.3% [78.4% - 88.6%]</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1,047 (100%)</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1></td></tr></table>

# 2) Risk estimation in node-positive patients

The results indicated that both clinical risk categories associated with each Prosigna score (Low, High), as set by the Trans-ATAC study were validated for node –positive patients in the ABCSG-8 patients. The following charts (Figure 2 and Table 9) show the Kaplan-Meier curves of the percent of Node negative patients without distant recurrence by risk-group through 10 years from the ABCSG-8 study.

![](images/a236fe830a10adff1de96cf9b6a5a5151caefd60167df4fe2ff600b8ef96c824.jpg)  
Figure 2: DRFS by Risk Group for Node-Positive Patients

Table 9: Summary of DRFS by Risk Group for Node-Positive Patients   

<table><tr><td rowspan=1 colspan=1>RiskGroup</td><td rowspan=1 colspan=1>Number ofPatients (%)</td><td rowspan=1 colspan=1>Number ofEventsThrough 10Years</td><td rowspan=1 colspan=1>Estimated PercentWithout DistantRecurrence at 10 years[95% CI]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>158 (41%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>94.2%[88.1% - 97.2%]</td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>224 (59%)</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>75.8%[68.9% - 81.4%]</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>382 (100%)</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1></td></tr></table>

# iii. Results Distribution

Patients in the ABCSG-8 study, while having Prosigna scores across the measuring range of the assay, were not evenly distributed. Patient data was placed into bins consisting of 10 scoring levels and plotted against the data modeled from the whole ABCSG-8 based validation study.

1) Results distribution in node-negative patients

Table 10: Distribution of node-negative patients by 10‐unit ROR (Prosigna) score Range   

<table><tr><td rowspan=1 colspan=1>Prosignascore Range</td><td rowspan=1 colspan=1>Number of Patients</td><td rowspan=1 colspan=1>Percent ofPatients</td><td rowspan=1 colspan=1>10-year DR Risk(Empirical)</td></tr><tr><td rowspan=1 colspan=1>1-10</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>11-20</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>14.8%</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5.1%</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>183</td><td rowspan=1 colspan=1>17.5%</td><td rowspan=1 colspan=1>7.5%</td></tr><tr><td rowspan=1 colspan=1>51-60</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>14.5%</td><td rowspan=1 colspan=1>12.1%</td></tr><tr><td rowspan=1 colspan=1>61-70</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>15.0%</td></tr><tr><td rowspan=1 colspan=1>71-80</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>12.3%</td></tr><tr><td rowspan=1 colspan=1>81-90</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>26.1%</td></tr><tr><td rowspan=1 colspan=1>91-100</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>33.3%</td></tr><tr><td rowspan=1 colspan=1>totals</td><td rowspan=1 colspan=1>1,047</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td></tr></table>

![](images/74e82e49aa34103b87204d05e6b52beca2738074d9121c9155c8e28db6f87ea1.jpg)  
Figure 3: ABCSG-8 data plotted against Trans-ATAC model for nodenegative patients

2) Results distribution in node-positive patients

Table 11: Distribution of Node-Positive (1-3 nodes) patients by 10‐unit Prosigna Score Range   

<table><tr><td rowspan=1 colspan=1>Prosigna scoreRange</td><td rowspan=1 colspan=1>Number of PatientsPercent of Patients</td><td rowspan=1 colspan=1>Number of PatientsPercent of Patients</td><td rowspan=1 colspan=1>10-year DR Risk(Empirical)</td></tr><tr><td rowspan=1 colspan=1>1-10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>11-20</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>13.9%</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>17.8%</td><td rowspan=1 colspan=1>8.5%</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>14.9%</td><td rowspan=1 colspan=1>16.7%</td></tr><tr><td rowspan=1 colspan=1>51-60</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>18.6%</td><td rowspan=1 colspan=1>17.8%</td></tr><tr><td rowspan=1 colspan=1>61-70</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>11.0%</td><td rowspan=1 colspan=1>28.9%</td></tr><tr><td rowspan=1 colspan=1>71-80</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>39.5%</td></tr><tr><td rowspan=1 colspan=1>81-90</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>33.0%</td></tr><tr><td rowspan=1 colspan=1>91-100</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>33.3%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>382</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1></td></tr></table>

![](images/293f5dca21d0a128a5a0e15a325e6ce6a577ccc407e4b0af5dcc368b3df0655b.jpg)  
Figure 4: Binned ABCSG-8 data plotted against ABCSG-8 validation data model for node-positive patients

4. Clinical cut-off:

Same as assay cut-off

5. Expected values/Reference range:

Risk assessment is reported as Low Risk, Intermediate Risk, or High Risk for node negative patients or as Low Risk or High Risk for Node positive patients (see Table 12).

Table 12: Risk Classification Scoring Algorithm Using Prosigna Score   

<table><tr><td rowspan=1 colspan=1>Nodal Status</td><td rowspan=1 colspan=1>Prosigna Score Range</td><td rowspan=1 colspan=1>Risk Classification</td></tr><tr><td rowspan=3 colspan=1>Node-Negative</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-60</td><td rowspan=1 colspan=1>Intermediate</td></tr><tr><td rowspan=1 colspan=1>61-100</td><td rowspan=1 colspan=1>High</td></tr><tr><td rowspan=2 colspan=1>Node-Positive (1-3 nodes)</td><td rowspan=1 colspan=1>0-40</td><td rowspan=1 colspan=1>Low</td></tr><tr><td rowspan=1 colspan=1>41-100</td><td rowspan=1 colspan=1>High</td></tr></table>

# N. Instrument Name:

The nCounter Dx Analysis System consists of a liquid handling robot Prep Station 5s and an epifluorescent scanner Digital Analyzer 5s.

# O. System Descriptions:

1. Modes of Operation: Automated

2. Software:

The Digital Analyzer measures and sorts multiple signals (reporter probes bound to mRNA transcript) from the clinical sample to establish an indicator (Prosigna score and risk category) to aid in determining patient prognosis. The Prep Station automates posthybridization sample processing while the Digital Analyzer includes signal reading, raw data storage, data acquisition software and software to process the detected targets (algorithm).

The Software is a Visual $\mathrm { C } { + } { + }$ web-based application developed by Nanostring.

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __x_ _ or No

3. Specimen Identification:

Specimen identifying information is entered into a computer application manually.

4. Specimen Sampling and Handling:

Samples are handled individually until RNA is extracted from FFPE tissue. RNA samples are then handled in batches of 12 on the instrument.

5. Calibration:

Installation, calibration and preventative maintenance of instrumentation are performed by the instrument manufacturer. No user calibration required.

6. Quality Control:

Quality control includes testing of the mixed Reporter CodeSet and Capture ProbeSet for the following performance characteristics:

Signal level of the geometric mean of housekeeping gene probes   
Signal levels of each of the 50 classifier genes   
Background level of the negative controls   
Linearity of positive controls   
Probe cross-contamination

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.