# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k130268   
B. Purpose for Submission: To obtain clearance for a new device, BD ProbeTecTM Trichomonas vaginalis (TV) Qx Amplified DNA Assay   
C. Measurand: 138 base-pair region of Trichomonas vaginalis adhesion protein gene (AP65-1)   
D. Type of Test: Nucleic acid amplification assay   
E. Applicant: Becton, Dickinson and Company   
F. Proprietary and Established Names: BD ProbeTecTM Trichomonas vaginalis (TV) $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay TV Qx Assay

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3860

2. Classification: Class II

3. Product code: OUY - Trichomonas vaginalis nucleic acid amplification test system

4. Panel:

# 83 - Microbiology

# H. Intended Use:

1. Intended use:

The BD ProbeTec™ Trichomonas vaginalis (TV) $\mathrm { Q } ^ { x }$ Amplified DNA Assay, when tested with the BD Viper™ System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Trichomonas vaginalis DNA in clinician-collected female endocervical swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and female urine specimens. The assay is indicated for use with asymptomatic and symptomatic females to aid in the diagnosis of trichomoniasis.

2. Indications for use: Same as Intended Use

3. Special conditions for use statement: For prescription use only

4. Special instrument requirements: BD Viper System

# I. Device Description:

The BD ProbeTec Trichomonas vaginalis (TV) $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay (TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay) is based on the simultaneous amplification and detection of target DNA using amplification primers and fluorescently-labeled detector probes. The assay employs strand displacement amplification (SDA) technology in similar fashion to the previously cleared assays k081824, k081825, and k103798. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The assay is designed for use with the BD ProbeTec $\mathrm { Q } ^ { x }$ specimen collection and transport devices, applicable reagents, the BD Viper System, and BD $\mathrm { F O X ^ { T M } }$ Extraction Tubes. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, prewarmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of $T .$ . vaginalis DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

The TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay utilizes two specimen collection and transport kits. These collection kits were cleared for use with BD Viper System in extracted mode (k081824, and k081825)

· BD ProbeTec $\mathrm { Q } ^ { x }$ Collection Kit for Endocervical or Lesion Specimens Vaginal Specimen Transport for the BD ProbeTec $\mathrm { Q } ^ { x }$ Amplified DNA Assays

# J. Substantial Equivalence Information:

1. Predicate device name: APTIMA Trichomonas vaginalis Assay on the TIGRIS DTS System

2. Predicate 510(k) number: K102911

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">APTIMA TV Assay on theTIGRIS DTS System(K102911)</td><td colspan="1" rowspan="1">TV Qx Assay on the BDViper System</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The APTIMA Trichomonasvaginalis Assay is an in vitroqualitative nucleic acidamplification test (NAAT) for thedetection of ribosomal RNA(rRNA) from Trichomonasvaginalis to aid in the diagnosis oftrichomoniasis using the TIGRISDTS System.The assay may be used to test thefollowing specimens fromsymptomatic or asymptomaticwomen: clinician-collectedendocervical swabs, clinician-collected vaginal swabs, femaleurine specimens, and specimenscollected in PreservCyt solution.</td><td colspan="1" rowspan="1">The BD ProbeTecTMTrichomonas vaginalis (TV)Qx Amplified DNA Assay,when tested with the BDViperTM System in ExtractedMode, uses Strand DisplacementAmplification technology for thedirect, qualitative detection ofTrichomonas vaginalis DNA inclinician-collected femaleendocervical swab specimens,patient-collected vaginal swabspecimens (in a clinical setting),and female urine specimens. Theassay is indicated for use withasymptomatic and symptomaticfemales to aid in the diagnosis oftrichomoniasis.</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Molecular NAAT</td><td colspan="1" rowspan="1">Molecular NAAT</td></tr><tr><td>Instrument</td><td>TIGRIS DTS System</td><td>BD Viper System</td></tr><tr><td>Specimen Types</td><td>Female Urine (UTT2) Vaginal Swab Endocervical Swab PreservCyt solution</td><td>Female Urine (Neat) Vaginal Swab Endocervical Swab</td></tr></table>

# K. Standard/Guidance Document Referenced:

EP5-A2, 2004 – Evaluation of Precision Performance of Quantitative Measurement Methods, CLSI Approved Guideline

MM03-A2, 2006 – Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline

MM13-A, 2005 – Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods, CLSI Approved Guideline

EP17-A2, 2012 – Evaluation of Detection Capability for Clinical laboratory Measurement Procedures; CLSI Approved Guideline

EP07-A2, 2005 – Interference Testing in Clinical Chemistry, Approved Guideline

# L. Test Principle:

The BD ProbeTec Trichomonas vaginalis (TV) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on strand displacement amplification (SDA) technology. The TV $\mathrm { Q } ^ { \mathrm { x } }$ assay is designed to target unique sequences within the T. vaginalis genome, a 138 base pair region of AP65-1 Adhesion Protein gene.

All specimens undergo a pre-warm step in the BD Viper Lysing Heater to dissolve mucus which may be present in certain specimens and to homogenize the specimen. After cooling, the specimens are loaded onto the BD Viper System which then performs all the steps involved in extraction and amplification of target DNA, without further user intervention. The specimen is transferred to an Extraction Tube that contains ferric oxide particles in a dissolvable film and dried Extraction Control. A high pH is used to lyse the cells and liberate the DNA into solution. Acid is then added to lower the pH and induce a positive charge on the ferric oxide, which in turn binds the negatively charged DNA. The particles and bound DNA are then pulled to the sides of the Extraction Tube by magnets and the treated specimen is aspirated to waste. The particles are washed and a high pH Elution Buffer is added to recover the purified DNA. Finally, a Neutralization Buffer is used to bring the pH of the extracted solution to the optimum for amplification of the target. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming

Microwell to rehydrate the contents. The reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell that contains the enzymes (DNA polymerase and restriction endonuclease) for Strand Displacement Amplification. The amplification well is then sealed to prevent contamination and incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of T. vaginalis DNA is determined by calculating the peak fluorescence (MaxRFU) over the course of the amplification process and by comparing this measurement to a predetermined threshold value. The magnitude of the MaxRFU score is not indicative of the level of $T .$ . vaginalis DNA in the specimen.

# M. Performance Characteristics:

1. Analytical performance:

# a. Reproducibility

Reproducibility of TV $\mathrm { Q } ^ { x }$ Assay was evaluated at two external and one internal site on one BD Viper System per site. Panels were comprised of four levels of T. vaginalis (ATCC strain 30001) seeded into urine specimen matrix or vaginal matrix in $\boldsymbol { \mathrm { Q } } ^ { x }$ diluent. Each panel member was either left unspiked (negative), or was spiked with known amounts of T. vaginalis organism at 0.3X (high negative), 1X (low positive), and 3X LOD (moderate positive) for each respective specimen matrix. Samples in each panel were randomized and blinded to the user prior to shipment to the testing sites. Three replicates of each panel member were tested twice a day for five days on each BD Viper System in Extracted Mode.

The following table presents, for each panel member, MaxRFU data in terms of mean, standard deviation (SD), and coefficient of variation (CV) between sites, , between runs, within runs, and between days. Percent agreement with expected results is also shown. Samples with valid results were included in the analyses. The results indicate reproducible performance within and across all three sites for all panel members tested when compared to the expected outcome.

Summary of Reproducibility Data for TV $\mathbf { Q } ^ { x }$ Assay on the BD Viper System   

<table><tr><td colspan="5" rowspan="1"></td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Between RunWithin Day</td><td colspan="2" rowspan="1">Between DayWithin Site</td><td colspan="2" rowspan="1">Between Sites</td></tr><tr><td colspan="1" rowspan="1">Specimentype</td><td colspan="1" rowspan="1">Panel</td><td colspan="1" rowspan="1">Agreement</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="4">Vaginal</td><td colspan="1" rowspan="1">Zero</td><td colspan="1" rowspan="1">100.0%(89/89)</td><td colspan="1" rowspan="1">(95.9%,100.0%)</td><td colspan="1" rowspan="1">3.89</td><td colspan="1" rowspan="1">4.02</td><td colspan="1" rowspan="1">103.53</td><td colspan="1" rowspan="1">2.01</td><td colspan="1" rowspan="1">51.64</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">23.57</td><td colspan="1" rowspan="1">4.69</td><td colspan="1" rowspan="1">120.60</td></tr><tr><td colspan="1" rowspan="1">High Negative(0.3X LOD)</td><td colspan="1" rowspan="1">43.3%(39/90)</td><td colspan="1" rowspan="1">(33.6%,56%</td><td colspan="1" rowspan="1">795.66</td><td colspan="1" rowspan="1">814.68</td><td colspan="1" rowspan="1">102.39</td><td colspan="1" rowspan="1">136.97</td><td colspan="1" rowspan="1">17.21</td><td colspan="1" rowspan="1">287.34</td><td colspan="1" rowspan="1">36.11</td><td colspan="1" rowspan="1">190.15</td><td colspan="1" rowspan="1">23.90</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1X LOD)</td><td colspan="1" rowspan="1">96.7%(87/90)</td><td colspan="1" rowspan="1">(90.7%,98.9%)</td><td colspan="1" rowspan="1">1632.07</td><td colspan="1" rowspan="1">457.83</td><td colspan="1" rowspan="1">28.05</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">110.13</td><td colspan="1" rowspan="1">6.75</td></tr><tr><td colspan="1" rowspan="1">High Positive(3X LOD)</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">(95.9%,100.0%)</td><td colspan="1" rowspan="1">1756.68</td><td colspan="1" rowspan="1">297.78</td><td colspan="1" rowspan="1">16.95</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">104.51</td><td colspan="1" rowspan="1">5.95</td><td colspan="1" rowspan="1">260.45</td><td colspan="1" rowspan="1">14.83</td></tr><tr><td colspan="1" rowspan="2">UrineMatrix</td><td colspan="1" rowspan="1">Zero</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">(95.9%,100.0%)</td><td colspan="1" rowspan="1">8.70</td><td colspan="1" rowspan="1">11.33</td><td colspan="1" rowspan="1">130.18</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">8.29</td><td colspan="1" rowspan="1">95.33</td></tr><tr><td colspan="1" rowspan="1">High Negative(0.3X LOD)</td><td colspan="1" rowspan="1">39.3%(35/89)*</td><td colspan="1" rowspan="1">(29.8%,49.7%)</td><td colspan="1" rowspan="1">976.96</td><td colspan="1" rowspan="1">913.87</td><td colspan="1" rowspan="1">93.54</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">Low Positive(1X LOD)</td><td colspan="1" rowspan="1">92.1%(82/89)*</td><td colspan="1" rowspan="1">(84.6%,96.1%</td><td colspan="1" rowspan="1">1574.67</td><td colspan="1" rowspan="1">681.14</td><td colspan="1" rowspan="1">43.26</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">High Positive 3X LOD)</td><td colspan="1" rowspan="1">100.0%(90/90)</td><td colspan="1" rowspan="1">(95.9%,100.0%)</td><td colspan="1" rowspan="1">1822.52</td><td colspan="1" rowspan="1">364.46</td><td colspan="1" rowspan="1">20.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">172.88</td><td colspan="1" rowspan="1">9.49</td></tr></table>

\* Four non-reportable results were due to extraction control failures which caused a reduction in the full number of replicates at one test site.

# b. Precision

Precision of the TV $\mathrm { Q } ^ { \mathrm { x } }$ assay was evaluated at one site over multiple days, by four users testing three instruments across calibration cycles, and multiple organism spike levels. Testing occurred in four phases on each of the three BD Viper Systems. Each phase consisted of three days of testing with two runs per day on three BD Viper Systems. The study was designed to capture the potential variation across calibration cycles with each instrument. Usually, the BD Viper System is calibrated every 30 days. In this study, the calibration cycles were artificially condensed to allow a shorter real-time interval between test phases.

Accuracy and Precision Testing Sequence   

<table><tr><td>Duration</td><td>Part 1</td><td>Calibration</td><td>Part 2</td><td></td><td>Part 3</td><td>Calibration</td><td>Part 4</td></tr><tr><td>Duration</td><td>3 days</td><td></td><td>3 days</td><td></td><td>3 days</td><td></td><td>3 days</td></tr></table>

Twelve different panel members were tested in each run. Panel members consisted of specimen matrix unspiked or spiked with 0.3X, 1X, and 3X LoD. Testing was performed for 12 days in four phases with 360 observations per panel member.

<table><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Spike Level</td><td rowspan=1 colspan=1>TotalTested</td><td rowspan=1 colspan=1>NumberCorrect</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Q* Vaginal</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>359</td><td rowspan=1 colspan=1>99.72%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Q* Vaginal</td><td rowspan=1 colspan=1>0.3XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>236</td><td rowspan=1 colspan=1>65.56%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Q Vaginal</td><td rowspan=1 colspan=1>1XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>353</td><td rowspan=1 colspan=1>98.06%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Q Vaginal</td><td rowspan=1 colspan=1>3XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Urine matrix</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>359</td><td rowspan=1 colspan=1>99.72%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Urine matrix</td><td rowspan=1 colspan=1>0.3XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>72.22%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Urine matrix</td><td rowspan=1 colspan=1>1XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>97.50%</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Urine matrix</td><td rowspan=1 colspan=1>3XLOD</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>360</td><td rowspan=1 colspan=1>100.00%</td></tr></table>

c. Linearity/assay reportable range: Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods): Specimen Stability Studies:

Urine Specimen Stability:

Data to support the recommended storage and transport conditions for neat urine was generated with negative female urine specimens spiked with T. vaginalis strain 30001 at 3X LOD. Twenty-four assay replicates were tested for each condition (sample type/temperature/duration). The study results demonstrated the urine stability for the following conditions:

$2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 7 days; $3 0 \mathrm { { } ^ { \circ } C }$ for 18 and 24 hours; ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for up to 180 days.

Vaginal Dry and Expressed Swab Stability:

Pools of T. vaginalis negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for dry vaginal swabs and vaginal swabs expressed in $\mathrm { Q } ^ { \mathrm { x } }$ swab diluent. Pools were spiked with $T .$ . vaginalis strain ATCC 30001 at 3X LOD. Twenty four assay replicates were tested for each condition (sample type/temperature/duration). The data supported the stability for dry vaginal swab specimens at $2 - 8 ^ { \circ } \mathrm { C }$ for up to 14 days, at $3 0 \mathrm { { } ^ { \circ } C }$ for up to 3 days, or at ${ \bf - } 2 0 ^ { \circ } \mathrm { C }$ for up to 180 days and the stability of the expressed vaginal swab specimens at $2 { - } 8 ^ { \circ } \mathrm { C }$ or $3 0 \mathrm { { } ^ { \circ } C }$ for up to 30 days or at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for up to 180 days.

Endocervical Swab Specimen Stability:

Pools of negative endocervical swab matrix were spiked with $T .$ vaginalis strain ATCC 30001 at 3X LOD in this study that demonstrated the endocervical swab specimens were stable at $2 { - } 8 ^ { \circ } \mathrm { C }$ or $3 0 \mathrm { { } ^ { \circ } C }$ for up to 30 days or at ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ for up to 180 days. Twenty four assay replicates were tested for each condition (sample type/temperature/duration).

# Post Pre-Warm Specimen Stability

Pools of $T .$ . vaginalis negative neat urine matrix were used to support the storage claims for pre-warmed neat urine specimens. Pools of neat urine were spiked with $T .$ . vaginalis ATCC strain 30001 at 3X LoD. The neat urine pools were dispensed in 2 mL volumes into BD Viper specimen tubes. The specimen tubes were pre-warmed at $1 1 4 \mathrm { { } ^ { \circ } C }$ for 15 minutes and cooled for 15 minutes. After the pre-warm process, the specimen tubes were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 7 days, at $3 0 \mathrm { { } ^ { \circ } C }$ for up to 3 days, and at ${ - 2 0 ^ { \circ } \mathrm { C } }$ for up to 180 days. At each time point, samples were removed from storage and tested with the TV $\mathrm { Q } ^ { x }$ Assay on the BD Viper System in extracted mode. Twenty-four assay replicates were tested for each condition (sample type/temperature/duration) demonstrating that the urine specimens are stable at the above conditions after the pre-warm process.

Pools of $T .$ vaginalis negative vaginal and endocervical swab specimen matrices in $\mathrm { Q } ^ { x }$ Swab Diluent were used to support the storage stability claims for pre-warmed expressed vaginal and endocervical swab specimens. For both types of matrices, pools were spiked with $T .$ . vaginalis strain ATCC 30001 at 3X LOD and aliquoted into $2 \mathrm { m L }$ volumes in BD Viper specimen tubes. The tubes were pre-warmed at $1 1 4 \mathrm { { } ^ { \circ } C }$ for 15 minutes and cooled for 15 minutes. After the pre-warm process, the specimen tubes were stored either at $2 { - } 8 ^ { \circ } \mathrm { C }$ or $3 0 ^ { \circ } \mathrm { C }$ for up to 30 days, or at ${ - 2 0 ^ { \circ } \mathrm { C } }$ for

up to 180 days. At each time point, samples were removed from storage and evaluated with the TV $\mathrm { Q } ^ { x }$ Assay on the BD Viper System in extracted mode. Twenty four assay replicates were tested for each condition (sample   
type/temperature/duration). The results supported the post pre-warm stability for expressed vaginal and endocervical swab specimens under above mentioned   
conditions.

Quality Control Results and Acceptability:

External Controls:

The control set for the assay is provided separately. One Positive and one Negative Control must be included in each assay run and for each new reagent kit lot number. Controls must be positioned according to the BD Viper Instrument User’s Manual. The CT/GC/TV $\mathrm { Q } ^ { x }$ Positive Control will monitor for substantial reagent failure only. The CT/GC/TV $\mathrm { Q } ^ { x }$ Negative Control monitors for reagent and/or environmental contamination.

The CT/GC/TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Positive Control is comprised of recombinant plasmids that contain the SDA target regions for the CT $\mathrm { Q } ^ { x }$ , GC $\boldsymbol { \mathrm { Q } } ^ { x }$ and TV $\mathrm { Q } ^ { x }$ Assays. The plasmids are not necessarily representative of native target DNA detected by the assay (e.g., their overall length is shorter than that of the complete gene or genomic sequence), nor are the controls representative of the specimen matrices indicated for use with the assays on the BD Viper System in extracted mode. The TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Negative Control is comprised of the same milieu as the Positive Control but without the plasmid DNA. The Positive and Negative Control formulations are dried in separate $4 . 5 ~ \mathrm { m L }$ specimen tubes. A QC pair (Positive Control and Negative Control) must be logged in for each plate to be tested and for each reagent kit lot number.

The location of the microwells is shown in a color-coded plate layout screen on the LCD Monitor. The plus symbol $( + )$ within the microwell indicates the positive QC sample. The minus symbol (-) within the microwell indicates the negative QC sample.

The CT/GC/TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Positive Control and the CT/GC/TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Negative Control must test as positive and negative, respectively. If controls do not perform as expected, the run is considered invalid and results will not be reported by the instrument. If either of the controls does not provide the expected results, the entire run should be repeated using a new set of controls, new extraction tubes, new extraction trough, new lysis trough, and new microwells.

The Extraction Control (EC):

Each extraction tube contains a dried fluorescently labeled oligonucleotide that is incorporated into each tube. EC is labeled with a dye different that that used for detection of T. vaginalis specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is rehydrated by the BD Viper System upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the instrument and an automated algorithm is applied to both the EC and TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Assay specific signals to report results as positive, negative, or EC failure.

# Interpretation of Quality Control Results

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Tube ResultReport Symbol</td><td rowspan=1 colspan=1>TV QrMaxRFU</td><td rowspan=1 colspan=1>QCDisposition</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QrPositive Control</td><td rowspan=1 colspan=1>OK</td><td rowspan=1 colspan=1>≥125</td><td rowspan=1 colspan=1>QC Pass</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QPositive Control</td><td rowspan=1 colspan=1>®</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QrPositive Control</td><td rowspan=1 colspan=1>$ or Xor</td><td rowspan=1 colspan=1>Anyvalue</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QxNegative Control</td><td rowspan=1 colspan=1>OK</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>QC Pass</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QrNegative Control</td><td rowspan=1 colspan=1>®</td><td rowspan=1 colspan=1>≥125</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QNegative Control</td><td rowspan=1 colspan=1>$ or  or or</td><td rowspan=1 colspan=1>Anyvalue</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1>CT/GC/TV QrNegative Control</td><td rowspan=1 colspan=1>BQ</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>QC Failure</td></tr></table>

\* $=$ Fail, $=$ Extraction Transfer failure, $=$ Liquid Level failure, $=$ Extraction Control failure, $\pm \Updownarrow = \mathrm { E r r o r }$ , $\Breve { \mathbf { B } } \Breve { \mathbf { Q } } = \mathbf { R } \mathrm { O X }$ failure $\mathrm { R O X = }$ Sulforhodamine dye used to monitor EC performance.

# e. Detection limit:

The limit of detection (LOD) for the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay on BD Viper System was determined in specimen matrices (vaginal swab, endocervical swab, and urine) and in $\mathrm { Q } ^ { \mathrm { x } }$ swab diluent. Two strains of $T .$ . vaginalis, one Metronidazole-sensitive and one Metronidazole-resistant, were used at varying concentrations to create an LOD panel consisting of six target levels. For each target level 20 replicates were tested across three lots of assay and extraction reagents on three BD Viper Systems to yield a total of 60 replicates at each target level. The resulting MaxRFU values were analyzed to determine proportion positive at each level that was used to generate positivity curves from which each $9 5 \%$ LOD was calculated.

LOD was confirmed using 20 replicates at the estimated LOD for each matrix type with one lot of reagents and one BD Viper System for each strain of $T .$ . vaginalis.

The Limit of Detection (LOD) for the TV $\mathrm { Q } ^ { x }$ Assay with T. vaginalis ATCC strains 30001 and 50143 in BD $\boldsymbol { \mathrm { Q } } ^ { x }$ Swab Diluent when extracted on the BD Viper System was 54.5 and 55.5 trichomonads $/ \mathrm { m L }$ , respectively. The LODs for neat urine, vaginal and endocervical specimen matrices are presented in the Table below.

LOD Estimates for TV $\mathbf { Q } ^ { x }$ Assay   

<table><tr><td rowspan=1 colspan=1>Specimen type</td><td rowspan=1 colspan=1>ATCC Strain</td><td rowspan=1 colspan=1>LOD(TV/mL)</td></tr><tr><td rowspan=2 colspan=1>Neat Urine</td><td rowspan=1 colspan=1>30001</td><td rowspan=1 colspan=1>109.7</td></tr><tr><td rowspan=1 colspan=1>50143</td><td rowspan=1 colspan=1>108.2</td></tr><tr><td rowspan=3 colspan=1>Vaginal</td><td rowspan=1 colspan=1>30001</td><td rowspan=1 colspan=1>74.4</td></tr><tr><td rowspan=1 colspan=1>50143</td><td rowspan=1 colspan=1>88.4</td></tr><tr><td rowspan=1 colspan=1>30184*</td><td rowspan=1 colspan=1>152.8</td></tr><tr><td rowspan=2 colspan=1>Endocervical</td><td rowspan=1 colspan=1>30001</td><td rowspan=1 colspan=1>64.8</td></tr><tr><td rowspan=1 colspan=1>50143</td><td rowspan=1 colspan=1>76.2</td></tr></table>

\* LoD for an additional ATCC T. vaginalis strain was determined in vaginal swab specimen matrix. Please see the Inclusivity section below for additional information.

# f. Inclusivity:

Four ATCC Trichomonas vaginalis strains 30185, 30237, 50144, and 30184 were evaluated in the inclusivity study for TV $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ assay on BD Viper System in urine and vaginal swab specimen matrix. A total of 20 assay replicates were run for each strain and specimen matrix. One ATCC strain 30184 could not meet the $9 5 \%$ positivity criteria for vaginal matrix LoD confirmation; therefore, a full LoD testing was performed with this strain in vaginal swab matrix. The same panel that was used in the LoD study with six target levels was tested with 60 assay replicates for each level, using two lots of assay reagents, three lots of extractions reagents, and three BD Viper Systems. The resulting MaxRFU values were analyzed to determine proportion positive at each level that was used to generate positivity curves from which each $9 5 \%$ LOD was calculated. The TVQ LOD for strain 30184 in vaginal matrix was determined to be $1 5 2 . 8 \mathrm { ~ T V / m L }$ ( $9 5 \%$ CI: 122.9, 182.7).

# g. Analytical specificity:

DNA from 54 organisms was extracted on the BD Viper System and tested with the TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Assay. All potentially cross-reactive species were tested at approximately $1 \mathrm { x } 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and yeast, and for viruses/pathogens at $\mathrm { i } \mathrm { \dot { x } } 1 0 ^ { 6 }$ viral particles/mL or genomic RNA/DNA equivalents per mL, or organisms/mL (or at the highest concentration possible). One organism, Trichomonas tenax was identified as a cross-reactant at concentration above $1 . 0 \mathrm { x } 1 0 ^ { 4 }$ organisms/mL. This limitation is included in the labeling. No other organisms cross-reacted in the TV $\mathbf { Q } ^ { \mathrm { x } }$ Assay when tested on the BD Viper System.

Microorganisms tested for Potential Cross Reactivity   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>FinalConcentration</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Final Concentration</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>1.03 x 109 CFU/mL</td><td rowspan=1 colspan=1>HPV-18</td><td rowspan=1 colspan=1>6.13 x 108 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>6.23 x 107 CFU/mL</td><td rowspan=1 colspan=1>Human Immunodeficiency Virus (HIV-1)</td><td rowspan=1 colspan=1>1.00 x 108 vp/mL</td></tr><tr><td rowspan=1 colspan=1>Atopobium vaginae</td><td rowspan=1 colspan=1>5.70 x 107 CFU/mL</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>6.27 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>1.04 x 10 CFU/mL</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>1.37 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium bifidum</td><td rowspan=1 colspan=1>7.57 x 107 CFU/mL</td><td rowspan=1 colspan=1>Lactobacillus jensenii</td><td rowspan=1 colspan=1>4.00 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>2.87 x 107 CFU/mL</td><td rowspan=1 colspan=1>Lactobacillus vaginalis</td><td rowspan=1 colspan=1>6.10 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>8.50 x 106 CFU/mL</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>8.50 x 108 CFU/ml</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>2.05 x 107 CFU/mL</td><td rowspan=1 colspan=1>Mobiluncus curtisii</td><td rowspan=1 colspan=1>6.47 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>2.20 x 107 CFU/mL</td><td rowspan=1 colspan=1>Moraxella catarrhalis (Branhamella sp.)</td><td rowspan=1 colspan=1>1.93 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>7.00 x 106 CFU/mL</td><td rowspan=1 colspan=1>Mycobacterium smegmatis</td><td rowspan=1 colspan=1>1.23 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>3.50 x 106 EB/mL</td><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>1.00 x 107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Clostridium difficile</td><td rowspan=1 colspan=1>6.10 x 107 CFU/mL</td><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>1.00 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>5.63 x 107 CFU/mL</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>4.63 X 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium biovar1</td><td rowspan=1 colspan=1>1.16 x 108 CFU/mL</td><td rowspan=1 colspan=1>Pentatrichomonas hominis</td><td rowspan=1 colspan=1>3.39 x 105Organisms/mL</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>6.33 x 106 CFU/mL</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>2.37 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>8.23 x 108 CFU/mL</td><td rowspan=1 colspan=1>Prevotella bivia</td><td rowspan=1 colspan=1>7.03 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloaceae</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>7.97 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>5.13 x 108 CFU/mL</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.20 x 10 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>6.17 x 108 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus, non-protein A</td><td rowspan=1 colspan=1>4.14 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Fusobacterium nucleatum</td><td rowspan=1 colspan=1>3.00 x 106 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus aureus, protein-Approducing</td><td rowspan=1 colspan=1>4.80 x 109 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>4.07 x 108 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>1.80 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Haemophilus ducreyi</td><td rowspan=1 colspan=1>9.93 x 108 CFU/mL</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td><td rowspan=1 colspan=1>3.47 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus Type 1</td><td rowspan=1 colspan=1>1.00 x 107 vp/mL</td><td rowspan=1 colspan=1>Streptococcus pyogenes (Group A)</td><td rowspan=1 colspan=1>3.73 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus Type 2</td><td rowspan=1 colspan=1>1.00 x 107 vp/mL</td><td rowspan=1 colspan=1>Streptococcus agalactiae (Group B)</td><td rowspan=1 colspan=1>5.67 x 108 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>HPV-6</td><td rowspan=1 colspan=1>1.57 x 10° cells/mL</td><td rowspan=1 colspan=1>Trichomonas tenax</td><td rowspan=1 colspan=1>1.50 x 106Organisms/mL</td></tr><tr><td rowspan=1 colspan=1>HPV-11</td><td rowspan=1 colspan=1>4.87 x 108 cells/mL</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1>1.00 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>HPV-16</td><td rowspan=1 colspan=1>8.43 x 108 cells/mL</td><td rowspan=1 colspan=1>Veillonella parvula</td><td rowspan=1 colspan=1>7.47 x 108 CFU/mL</td></tr></table>

# h. Interference

The effect of endogenous and exogenous substances that may be present in urine and vaginal swab specimens on the sensitivity and specificity of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System was evaluated in an interference study.

The table below describes the substances that were individually spiked into urine and vaginal sample matrices. Testing included 20 replicates containing each potentially interfering substance alone as well as 20 replicates containing each substance spiked with T. vaginalis at 3X LoD in each matrix. ATCC strain 50143 or 30001 was used for spiking the urine or vaginal specimen matrix respectively. The study also included sample matrices unspiked or spiked with Trichomonas vaginalis without interfering substances as a control. The results are summarized in the table below.

Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine Matrix</td></tr><tr><td rowspan=1 colspan=1>No interferenceobserved at levelslisted</td><td rowspan=1 colspan=1>Whole Blood (≤ 60%)Seminal FluidMucusOver the counter vaginalproducts and contraceptivesHemorrhoidal creamPrescription vaginal treatmentsLeukocytes (1x106 cells/mL)Intravaginal hormones</td><td rowspan=1 colspan=1>Phenazopyridine HydrochlorideWhole Blood (≤ 1% v/v)Acidic urine (pH 5.0)Alkaline urine (pH 9.0)Horomone poolAnalgesic poolAntibioticsBilirubinMucusAlbumin (≤ 1mg/mL)GlucoseSemen (5% v/v)Over the counter deodorant sprayand powderLeukocytes (2.5x106 cells/mL)</td></tr><tr><td rowspan=1 colspan=1>May cause ExtractionControl (EC) failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>May cause falsenegative results</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr></table>

$\mathrm { V } / \mathrm { V } =$ volume/volume

# i. Assay cut-off:

The TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay is designed for and validated on the BD Viper System in Extracted Mode. The assay test results are automatically interpreted by the BD Viper System software. The presence or absence of $T .$ vaginalis DNA is determined by calculating the peak fluorescence (MaxRFU) over the course of the amplification process and by comparing this measurement to a predetermined threshold value. If the T. vaginalisspecific signal is greater than or equal to a threshold of 125 MaxRFU, the EC fluorescence is ignored by the algorithm. If the T. vaginalis-specific signal is less than a threshold of 125 MaxRFU, the EC fluorescence is utilized by the algorithm in the interpretation of the result.

Interpretation of Test Results for the TV $\mathbf { Q } ^ { \mathbf { x } }$ Assay   

<table><tr><td colspan="1" rowspan="1">TubeReportResult</td><td colspan="1" rowspan="1">TV QxMaxRFU</td><td colspan="1" rowspan="1">Report</td><td colspan="1" rowspan="1">Interpretation</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">≥125</td><td colspan="1" rowspan="1">T. vaginalis DNAdetected by SDA</td><td colspan="1" rowspan="1">Positive for T. vaginalis DNA.T. vaginalis organism viability and/orinfectivity cannot be inferred since targetDNA may persist in the absence of viableorganisms.</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">&lt;125</td><td colspan="1" rowspan="1">T. vaginalis DNA notdetected by SDA</td><td colspan="1" rowspan="1">Presumed negative for T. vaginalis DNA.A negative result does not preclude</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">T. vaginalis infection because results aredependent on adequate specimen collection,absence of inhibitors, and the presence ofsufficient DNA to be detected.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">x</td><td colspan="1" rowspan="1">&lt;125</td><td colspan="1" rowspan="1">Extraction ControlFailure. Repeat test frominitial specimen tube orobtain another specimenfor testing.</td><td colspan="1" rowspan="1">T. vaginalis, if present, is not detectable.</td><td colspan="1" rowspan="1">ExtractionControlFailure</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">Any value</td><td colspan="1" rowspan="1">Extraction TransferFailure. Repeat test frominitial specimen tube orobtain another specimenfor testing</td><td colspan="1" rowspan="1">T. vaginalis, if present, is not detectable.</td><td colspan="1" rowspan="1">ExtractionTransferFailure</td></tr><tr><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">Any value</td><td colspan="1" rowspan="1">Liquid Level Failure.Repeat test from initialspecimen tube or obtainanother specimen fortesting.</td><td colspan="1" rowspan="1">T. vaginalis, if present, is not detectable.</td><td colspan="1" rowspan="1">Liquid LevelFailure</td></tr><tr><td colspan="1" rowspan="1">→0</td><td colspan="1" rowspan="1">Any value</td><td colspan="1" rowspan="1">Error. Repeat test frominitial specimen tube orobtain another specimenfor testing.</td><td colspan="1" rowspan="1">T. vaginalis, if present, is not detectable.</td><td colspan="1" rowspan="1">Error</td></tr><tr><td colspan="1" rowspan="1">BQ</td><td colspan="1" rowspan="1">&lt;125</td><td colspan="1" rowspan="1">ROX Channel Failure.Repeat test from initialspecimen tube or obtainanother specimen fortesting.</td><td colspan="1" rowspan="1">T. vaginalis, if present, is not detectable.</td><td colspan="1" rowspan="1">ROX Failure</td></tr></table>

The performance of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay with the preliminary assay cutoff (125 MaxRFU) was evaluated in three phases at external clinical sites: a cutoff analysis (to verify the preliminary assay cutoff prior to the main clinical trial), a clinical trial (to validate the performance of the assays in different clinical settings), and a reproducibility study. These three phases confirmed that the preliminary cutoff values for assay was acceptable.

# j. Contamination and Carryover Study

An internal study was conducted to evaluate the risk of producing a false positive result with the TV $\mathrm { Q } ^ { x }$ Assay protocol in either the same run on the BD Viper System in Extracted Mode (within run cross-contamination) or in a subsequent run (between run carryover). Testing was conducted using negative and positive samples on three BD Viper Systems utilizing CTQ/GCQ/TVQ Panel Mode (login mode in which a single sample tube may be tested with the CT $\boldsymbol { \mathrm { Q } } ^ { x }$ , GC $\mathrm { Q } ^ { x } \mathrm { , }$ , and TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Assays on the BD Viper System) as the worst-case for carryover and crossover contamination rates. CTQ/GCQ/TVQ Panel Mode utilizes the most complex series of robotic arm movements and incorporates the longest instrument run time. This study utilized the most sensitive BD ProbeTec $\boldsymbol { \mathrm { Q } } ^ { \mathrm { X } }$ Amplified DNA Assay (CTQ Assay based on analytical LOD, K081824). Positive samples were seeded with a representative analyte $\mathrm { ( 1 0 ^ { 5 } C T E B s / m L ) }$ . The experimental series was comprised of 6 consecutive runs using alternating positive and negative samples arranged in a checkerboard pattern (at a positivity rate of $50 \%$ ) on each BD Viper System. The overall rate of contamination was ${ \leq } 0 . 4 9 \%$ (3/612) in TVQ mode and ${ \leq } 0 . 3 9 \%$ (3/807) in CTQ/GCQ/TVQ panel mode.

$k$ . Fresh vs. Frozen samples:

This study was done to verify that specimens which have been frozen prior to processing on the BD Viper System perform equivalently to those that are not frozen prior to processing. Urine and vaginal swab specimens expressed in $\mathbf { Q } ^ { \mathrm { x } }$ diluent, unspiked or spiked with $T .$ vaginalis at 3X LoD were tested either fresh or after storing at $- 2 0 ^ { \circ } \mathrm { C }$ for 7 days. Each fresh/frozen pair was considered as one set of sample and if the frozen result did not match the fresh then it was a “disagreement”. 44 such sets of fresh/frozen were tested at each T. vaginalis concentration for each specimen matrix.

Overall percent agreement between fresh and frozen specimens was calculated by sample type for the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay using the MaxRFU algorithm and a cutoff threshold of 125 MaxRFU.

Overall Agreement Between Combined (Positive and Negative) Fresh and Frozen Samples   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>TotalTested</td><td rowspan=1 colspan=1>Number inAgreement</td><td rowspan=1 colspan=1>%Agreement</td></tr><tr><td rowspan=2 colspan=1>Vaginal Matrix</td><td rowspan=1 colspan=1>TV Negative</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3x LOD(223.2 TV/mL)</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Neat Urine</td><td rowspan=1 colspan=1>TV Negative</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3x LOD(329.1 TV/mL)</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>95.5%</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not applicable

# 3. Clinical studies:

Performance characteristics for the BD TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay on BD Viper System were established in multi-center clinical study using prospectively collected specimens from consenting subjects during 2012. First void urine specimens between $2 0 - 6 0 ~ \mathrm { m L }$ , selfcollected vaginal swabs in a clinical setting, and clinician-collected endocervical swabs were obtained from 1222 symptomatic and asymptomatic female subjects attending family planning, OB/GYN, and sexually transmitted disease clinics at seven (7) geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they presented to the clinic with abnormal vaginal discharge, itching, dysuria, or odor as determined by the inclusion/exclusion criteria. The final data analysis included 1197 evaluable subjects. Exclusions from the data analysis were made due to specimens not collected, enrollment issues, transport errors, collection errors, shipping errors, processing errors, or BD Viper System operating errors. The final data analysis included 735 compliant results for the neat urine specimen type, 838 compliant results for the vaginal swab specimen type, and 995 compliant results for the endocervical swab specimen type.

All sensitivity and specificity calculations were based on the total number of TV $\mathrm { Q } ^ { x }$ Assay results for the neat urines, vaginal, and endocervical specimens as compared to the composite reference of the wet mount and the commercially available $T .$ . vaginalis culture test method. The subject was considered to be positive for $T .$ vaginalis if either the wet mount or $T .$ vaginalis culture result was positive. Subjects were considered negative for $T .$ . vaginalis if both of the reference methods were negative.

The following tables show the sensitivity, and specificity of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System for each specimen type by symptom status as well as sensitivity, specificity, prevalence, PPV, and NPV of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay by specimen type and collection site. Subjects were classified as symptomatic if symptoms were reported by the subject. Subjects were classified as asymptomatic if the subject did not report symptoms.

TV $\mathbf { Q } ^ { x }$ Assay Performance Compared to Composite Reference by Symptomatic Status   

<table><tr><td colspan="3" rowspan="1"></td><td colspan="4" rowspan="1">Performance Compared to Composite Reference</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Status</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">95% C.I.</td><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">95% C.I.</td></tr><tr><td colspan="1" rowspan="3">Neat Urine</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">289</td><td colspan="1" rowspan="1">93.1% (27/29)</td><td colspan="1" rowspan="1">(78.0%, 98.1%)</td><td colspan="1" rowspan="1">99.6% (259/260)</td><td colspan="1" rowspan="1">(97.9%, 99.9%)</td></tr><tr><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">446</td><td colspan="1" rowspan="1">96.4% (80/83)</td><td colspan="1" rowspan="1">(89.9%, 98.8%)</td><td colspan="1" rowspan="1">98.1% (356/363)</td><td colspan="1" rowspan="1">(96.1%, 99.1%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">735</td><td colspan="1" rowspan="1">95.5% (107/112)^</td><td colspan="1" rowspan="1">(90.0%, 98.1%)</td><td colspan="1" rowspan="1">98.7% (615/623)B</td><td colspan="1" rowspan="1">(97.5%, 99.3%)</td></tr><tr><td colspan="1" rowspan="3">Vaginal</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">343</td><td colspan="1" rowspan="1">93.5% (29/31)</td><td colspan="1" rowspan="1">(79.3%, 98.2%)</td><td colspan="1" rowspan="1">99.0% (309/312)</td><td colspan="1" rowspan="1">(97.2%, 99.7%)</td></tr><tr><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">495</td><td colspan="1" rowspan="1">100.0% (85/85)</td><td colspan="1" rowspan="1">(95.7%, 100.0%)</td><td colspan="1" rowspan="1">99.0% (406/410)</td><td colspan="1" rowspan="1">(97.5%, 99.6%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">838</td><td colspan="1" rowspan="1">98.3% (114/116)</td><td colspan="1" rowspan="1">(93.9%, 99.5%)</td><td colspan="1" rowspan="1">99.0% (715/722)c</td><td colspan="1" rowspan="1">(98.0%, 99.5%)</td></tr><tr><td colspan="1" rowspan="3">Endocervical</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">505</td><td colspan="1" rowspan="1">92.2% (47/51)</td><td colspan="1" rowspan="1">(81.5%, 96.9%)</td><td colspan="1" rowspan="1">99.1% (450/454)</td><td colspan="1" rowspan="1">(97.8%, 99.7%)</td></tr><tr><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">490</td><td colspan="1" rowspan="1">98.8% (82/83)</td><td colspan="1" rowspan="1">(93.5%, 99.8%)</td><td colspan="1" rowspan="1">99.8% (406/407)</td><td colspan="1" rowspan="1">(98.6%,10.0%)</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">995</td><td colspan="1" rowspan="1">96.3% (129/134)</td><td colspan="1" rowspan="1">(91.6%, 98.4%)</td><td colspan="1" rowspan="1">99.4% (856/861)D</td><td colspan="1" rowspan="1">(98.6%, 99.8%)</td></tr><tr><td colspan="1" rowspan="3">All SpecimenTypesCombined</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">1137</td><td colspan="1" rowspan="1">92.8% (103/111)</td><td colspan="1" rowspan="1">(86.4%, 96.3%)</td><td colspan="1" rowspan="1">99.2% (1018/1026)</td><td colspan="1" rowspan="1">(98.5%, 99.6%)</td></tr><tr><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">1431</td><td colspan="1" rowspan="1">98.4% (247/251)</td><td colspan="1" rowspan="1">(96.0%, 99.4%)</td><td colspan="1" rowspan="1">99.0% (1168/1180)</td><td colspan="1" rowspan="1">(98.2%, 99.4%)</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">2568</td><td colspan="1" rowspan="1">96.7% (350/362)</td><td colspan="1" rowspan="1">(94.3%, 98.1%)</td><td colspan="1" rowspan="1">99.1% (2186/2206)</td><td colspan="1" rowspan="1">(98.6%, 99.4%)</td></tr></table>

$\mathbf { A } =$ asymptomatic, C.I. $=$ confidence interval, $\mathbf { n } =$ number, $\mathbf { S } =$ symptomatic

TV $\mathbf { Q } ^ { x }$ Assay Performance Compared to Wet Mount and T. vaginalis Culture Composite Reference Result (by collection site)   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4>Performance Compared to Composite Reference</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>ClinicalSite</td><td rowspan=1 colspan=1>Prev</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>PPV%</td><td rowspan=1 colspan=1>NPV%</td></tr><tr><td rowspan=7 colspan=1>Neat Urine</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>29.1%</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>81.3%(13/16)</td><td rowspan=1 colspan=1>(57.0%,93.4%)</td><td rowspan=1 colspan=1>100.0%(39/39)</td><td rowspan=1 colspan=1>(91.0%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>27.4%</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>100.0%(26/26)</td><td rowspan=1 colspan=1>(87.1%,100.0%)</td><td rowspan=1 colspan=1>98.6%(68/69)</td><td rowspan=1 colspan=1>(92.2%,99.7%)</td><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14.4%</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>(74.2%,99.0%)</td><td rowspan=1 colspan=1>97.2%(104/107)</td><td rowspan=1 colspan=1>(92.1%,99.0%)</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>99.0%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>100.0%(4/4)</td><td rowspan=1 colspan=1>(51.0%,100.0%)</td><td rowspan=1 colspan=1>99.3%(137/138)</td><td rowspan=1 colspan=1>(96.0%,99.9%)</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>9.9%</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>100.0%(7/7)</td><td rowspan=1 colspan=1>(64.6%,100.0%)</td><td rowspan=1 colspan=1>98.4%(63/64)</td><td rowspan=1 colspan=1>(91.7%,99.7%)</td><td rowspan=1 colspan=1>87.5%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>100.0%(9/9)</td><td rowspan=1 colspan=1>(70.1%,100.0%)</td><td rowspan=1 colspan=1>100.0%(113/113)</td><td rowspan=1 colspan=1>(96.7%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>25.6%</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>96.9%(31/32)</td><td rowspan=1 colspan=1>(84.3%,99.4%)</td><td rowspan=1 colspan=1>97.8%(91/93)</td><td rowspan=1 colspan=1>(92.5%,99.4%)</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>98.9%</td></tr><tr><td rowspan=7 colspan=1>Vaginal</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>100.0%(16/16)</td><td rowspan=1 colspan=1>(80.6%,100.0%</td><td rowspan=1 colspan=1>100.0%(40/40)</td><td rowspan=1 colspan=1>(91.2%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>27.1%</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100.0%(26/26)</td><td rowspan=1 colspan=1>(87.1%,100.0%)</td><td rowspan=1 colspan=1>100.0%(70/70)</td><td rowspan=1 colspan=1>(94.8%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14.4%</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>94.4%(17/18)</td><td rowspan=1 colspan=1>(74.2%,99.0%)</td><td rowspan=1 colspan=1>98.1%(105/107)</td><td rowspan=1 colspan=1>(93.4%,99.5%)</td><td rowspan=1 colspan=1>89.5%</td><td rowspan=1 colspan=1>99.1%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>100.0%(4/4)</td><td rowspan=1 colspan=1>(51.0%,100.0%)</td><td rowspan=1 colspan=1>97.8%(135/138)</td><td rowspan=1 colspan=1>(93.8%,99.3%)</td><td rowspan=1 colspan=1>57.1%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>90.9%(10/11)</td><td rowspan=1 colspan=1>(62.3%,98.4%)</td><td rowspan=1 colspan=1>100.0%(158/158)</td><td rowspan=1 colspan=1>(97.6%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>100.0%(9/9)</td><td rowspan=1 colspan=1>(70.1%,100.0%)</td><td rowspan=1 colspan=1>100.0%(113/113)</td><td rowspan=1 colspan=1>(96.7%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>100.0%(32/32)</td><td rowspan=1 colspan=1>(89.3%,100.0%)</td><td rowspan=1 colspan=1>97.9%(94/96)</td><td rowspan=1 colspan=1>(92.7%,99.4%)</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

A Of the five neat urine Viper negative, composite reference positive, one was also negative by alternate NAAT test. B Of the eight neat Viper positive, composite reference negative, six were also positive by alternate NAAT test. C Of the seven vaginal Viper positive, composite reference negative, four were also positive by alternate NAAT test. D Of the two endocervical Viper positive, composite reference negative, both were also positive in at least one specimen tested by alternate NAAT test.

<table><tr><td rowspan=7 colspan=1>Endocervical</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>24.7%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>(74.2%,97.7%)</td><td rowspan=1 colspan=1>100.0%(73/73)</td><td rowspan=1 colspan=1>(95.0%,00.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.3%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>23.9%</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.5%,00.0%</td><td rowspan=1 colspan=1>100.0%(86/86)</td><td rowspan=1 colspan=1>(95.7%,10.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15.6%</td><td rowspan=1 colspan=1>154</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>(79.8%,99.3%)</td><td rowspan=1 colspan=1>100.0%(130/130)</td><td rowspan=1 colspan=1>(97.1%,10.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>100.0%(8/8)</td><td rowspan=1 colspan=1>(67.6%,10.0%</td><td rowspan=1 colspan=1>98.0%(201/205)</td><td rowspan=1 colspan=1>(95.1%,99.2%)</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.5%</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>81.8%(9/11)</td><td rowspan=1 colspan=1>(52.3%,94.9%</td><td rowspan=1 colspan=1>100.0%(159/159)</td><td rowspan=1 colspan=1>(97.6%,100.0%</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.8%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>7.4%</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>100.0%(9/9)</td><td rowspan=1 colspan=1>(70.1%,100.0%)</td><td rowspan=1 colspan=1>100.0%(112/112)</td><td rowspan=1 colspan=1>(96.7%,100.0%)</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>24.4%</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>100.0%(31/31)</td><td rowspan=1 colspan=1>(89.0%,100.0%)</td><td rowspan=1 colspan=1>99.0%(95/96)</td><td rowspan=1 colspan=1>(94.3%,99.8%)</td><td rowspan=1 colspan=1>96.9%</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

$\mathrm { { C . I . } = }$ confidence interval, $\mathbf { n } =$ number, NPV $=$ negative predictive value, $\mathrm { P P V = }$ positive predictive value, Prev $=$ prevalence, Spec $=$ specimen

The results obtained with the TV $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Assay during the clinical performance testing were analyzed to generate frequency distributions of MaxRFU values. The data summarized in the following three tables indicate that positive and negative results obtained with the TV $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Assay were well separated from the cutoff of 125 MaxRFU.

# MaxRFU Distribution for Neat Urine

![](images/6cc4c24caba80b72b6432ff7caad402befc66ddb29f44fda56c2068b5eeaedc2.jpg)

MaxRFU Distribution for Vaginal Swab Specimen

![](images/c21d4aa98ee6b4b7b1d265a1f907e65ba4fd15cacee53fda29ef9bfead2edbcd.jpg)  
There were no CT/GC/TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Positive Control Failures from 235 TV $\boldsymbol { \mathrm { Q } } ^ { x }$ runs. For the CT/GC/TV $\mathrm { Q } ^ { x }$ Negative Control, there was 1 CT/GC/TV $\boldsymbol { \mathrm { Q } } ^ { x }$ Negative control failure from the $2 3 5 \mathrm { T V } \mathrm { Q } ^ { x }$ runs. The CT/GC/TV $\mathrm { Q } ^ { x }$ Positive and Negative Control MaxRFU values observed in the clinical trial are shown in Table below.

# CT/GC/TV $\mathbf { Q } ^ { x }$ Control Information

<table><tr><td rowspan=2 colspan=1>Control</td><td rowspan=1 colspan=6>MaxRFU</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>5thPercentile</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>95thPercentile</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>0-56</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>724 - 2304</td><td rowspan=1 colspan=1>982</td><td rowspan=1 colspan=1>1419</td><td rowspan=1 colspan=1>1335</td><td rowspan=1 colspan=1>1960</td></tr></table>

$\mathbf { n } =$ number

A multi-site clinical trial was conducted to verify the pre-established cutoff value (threshold). The Cutoff Trial was designed to assess the performance of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay as defined by sensitivity and specificity relative to the reference methods (InPouch TV Culture and Wet Mount) when using the preliminary cutoff value. To evaluate the performance of the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay, clinical specimens were collected from female participants at clinical centers across the United States, according to established inclusion/exclusion criteria. For specimen testing during the Cutoff Trial with the TV Qx Assay, three BD Viper Systems were utilized at three external clinical sites. The MaxRFU values from the dataset were examined for distribution. Receiver operating curve (ROC) analysis was performed for the assay to verify that the current MaxRFU cutoff of 125 was appropriate for the TV $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System. ROC curves were created by specimen type (vaginal, neat urine, and endocervical) and overall. The data analysis showed that the current cutoff for positive results of $> 1 2 5$ is appropriate.

# 5. Expected values/Reference range:

A summary of the prevalence of $T .$ . vaginalis, by specimen type, as determined by the composite reference method during the clinical study (April 2012-August 2012) is shown in the following table.

TV $\mathbf { Q } ^ { x }$ Assay Prevalence by Specimen Type and Collection Site   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=8>Prevalence (%)(# positive/# tested)</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>Site 7</td></tr><tr><td rowspan=1 colspan=1>Neat Urine</td><td rowspan=1 colspan=1>15.2(112/735)</td><td rowspan=1 colspan=1>29.1(16/55)</td><td rowspan=1 colspan=1>27.4(26/95)</td><td rowspan=1 colspan=1>14.4(18/125)</td><td rowspan=1 colspan=1>2.8(4/142)</td><td rowspan=1 colspan=1>9.9(7/71)</td><td rowspan=1 colspan=1>7.4(9/122)</td><td rowspan=1 colspan=1>25.6(32/125)</td></tr><tr><td rowspan=1 colspan=1>Vaginal</td><td rowspan=1 colspan=1>13.8(116/838)</td><td rowspan=1 colspan=1>28.6(16/56)</td><td rowspan=1 colspan=1>27.1(26/96)</td><td rowspan=1 colspan=1>14.4(18/125)</td><td rowspan=1 colspan=1>2.8(4/142)</td><td rowspan=1 colspan=1>6.5(11/169)</td><td rowspan=1 colspan=1>7.4(9/122)</td><td rowspan=1 colspan=1>25.0(32/128)</td></tr><tr><td rowspan=1 colspan=1>Endocervical</td><td rowspan=1 colspan=1>13.5(134/995)</td><td rowspan=1 colspan=1>24.7(24/97)</td><td rowspan=1 colspan=1>23.9(27/13)</td><td rowspan=1 colspan=1>15.6(24/154)</td><td rowspan=1 colspan=1>3.8(8/213)</td><td rowspan=1 colspan=1>6.5(11/170)</td><td rowspan=1 colspan=1>7.4(9/121)</td><td rowspan=1 colspan=1>24.4(31/127)</td></tr></table>

# N. Instrument Name:

The BD Viper System

# O. System Descriptions:

1. Modes of Operation: Operation of the BD Viper System operating in extracted mode (BD Viper) remains the same as previously cleared (K081824/K081825/K103798).

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X or No

3. Specimen Identification: Barcode or key entered accession numbers and patient sample location identification

4. Specimen Sampling and Handling: Automated

5. Calibration:

Self-calibrated; the BD Viper instrument has 8 channels, one for each row of a 96-well plate. Each channel has slightly different optical characteristics. To minimize these differences, the reader has normalizers in each channel. After the normalizer is read, the readings are used by the Built-In-Test to verify that the detection system is in calibration. Any readings for a given normalizer that fall out of the expected range cause the system to report error conditions. The readers are recalibrated when a software upgrade is performed, when new Normalizers are logged in, and when System Parameters are cleared.

# 6. Quality Control:

In addition to the extraction and external controls described in Section M.1.d., the Specimen process controls which test the entire system may also be run in accordance with the requirements of appropriate accrediting organizations. For this purpose, known positive specimens can serve as controls by being processed and tested in conjunction with unknown specimens. Specimens used as processing controls must be stored, processed, and tested according to the package insert instructions. Specimen processing controls for $T .$ vaginalis may also be prepared in the laboratory using commercially available Gibson Laboratories Tri-Valent™ Swab Positive Control (Cat. # TVS-01).

P.O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered in the “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.