# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY AND INSTRUMENT COMBINATION TEMPLATE

# D. Type of Test:

Test performed in Agendia’s two central laboratories: Amsterdam, Netherlands; and Irvine, California, USA

F. Proprietary and Established Names: MammaPrint $\textsuperscript { \textregistered }$ FFPE

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.6040, Gene expression profiling test system for breast cancer prognosis

2. Classification: Class II

3. Product code: NYI, Classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer

4. Panel:

Immunology (82)

# H. Intended Use:

1. Intended use(s):

MammaPrint $\textsuperscript { \textregistered }$ FFPE is a qualitative in vitro diagnostic test, performed in a central laboratory, using the gene expression profile obtained from formalin-fixed paraffin embedded (FFPE) breast cancer tissue samples to assess a patient’s risk for distant metastasis within 5 years.

The test is performed for breast cancer patients, with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The MammaPrint $\textsuperscript { \textregistered }$ FFPE result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

2. Indication(s) for use: Same as the above intended use

3. Special conditions for use statement(s):

For prescription use only

MammaPrint $\textsuperscript { \textregistered }$ FFPE is not indicated as a standalone test to determine the outcome of disease, nor to suggest or infer an individual patient’s likely response to therapy. Results should be taken in the context of other relevant clinico-pathological factors and standard practice of medicine.

4. Special instrument requirements:

Agilent 2100 Bioanalyzer: Serial numbers DE54700497, DE24802382, DE72901757, and DE72902383.

Agilent DNA microarray scanner: Serial numbers US22502555, US810R3210, US45103019, and US811R3213

Note: The scanners and Bio-analyzers are components of this assay and are cleared only for this assay and not for any other application. In addition, clearance is only limited to the bioanalyzers and scanners with the serial numbers as specified above.

# I. Device Description:

The MammaPrint $\textsuperscript { \textregistered }$ FFPE test is performed at Agendia’s two central Laboratories, one located in Netherland and the other one in California, USA. The test is a microarray based gene expression analysis of RNA extracted from FFPE breast tumor tissue. The test is a custom-designed array chip manufactured by Agilent Technologies using the Agilent oligonucleotide microarray platform which assesses the mRNA expression of the 70 genes printed in nine-fold.

The MammaPrint $\textsuperscript { \textregistered }$ FFPE analysis is designed to determine the expression of specific genes in a tissue sample. The result is an expression profile, or “fingerprint”, of the sample. Using this expression profile, the MammaPrint $\textsuperscript { \textregistered }$ FFPE Index is calculated and the molecular prognosis profile of the sample is determined (Low Risk, High Risk). The genes and scoring algorithm for MammaPrint $\textsuperscript { \textregistered }$ FFPE are the same as those used for MammaPrint $\textsuperscript { \textregistered }$ , performed with fresh and fresh-frozen tissues.

# J. Substantial Equivalence Information:

1. Predicate device name(s): MammaPrint®

2. Predicate 510(k) number(s): K101454

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Name</td><td colspan="1" rowspan="1">MammaPrint® FFPE</td><td colspan="1" rowspan="1">MammaPrint®</td></tr><tr><td colspan="1" rowspan="1">Risk Classification</td><td colspan="1" rowspan="1">A numerical MammaPrint®Index (MPI) is generated and aresult of Low Risk, High Risk,Low Risk Borderline, or HighRisk Borderline is provided ina report to the ordering healthcare provider.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">MammaPrint® MicroarrayDensity</td><td colspan="1" rowspan="1">Analysis is performed usingAgendia designed HighDensity diagnosticMicroarrays manufacturedunder GMP by AgilentTechnologies.70 signature genes printed innine-fold</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Feature Extraction Software</td><td colspan="1" rowspan="1">Version 9.5 is used to analyzeintensities of the 70 genes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Qualitative in vitro diagnostictest</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>MammaPrint® FFPE</td><td rowspan=1 colspan=1>MammaPrint®</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>MammaPrint® FFPE is aqualitative in vitro diagnostictest, performed in a centrallaboratory, using the geneexpression profile obtainedfrom formalin-fixed paraffinembedded (FFPE) breastcancer tissue samples to assessa patient&#x27;s risk for distantmetastasis within 5 years.</td><td rowspan=1 colspan=1>MammaPrint® is a qualitativein vitro diagnostic testservice, performed in a centrallaboratory, using the geneexpression profile of freshbreast cancer tissue samplesto assess a patients&#x27; risk fordistant metastasis (up to 10years for patients less than 61years old, up to 5 years forpatients ≥ 61 years).</td></tr><tr><td rowspan=1 colspan=1>Test Sample</td><td rowspan=1 colspan=1>FFPE breast cancer tissuesamples</td><td rowspan=1 colspan=1>Fresh and fresh-frozen breastcancer tissue samples</td></tr><tr><td rowspan=1 colspan=1>Analyte Detected on Chip</td><td rowspan=1 colspan=1>Labeled cDNA</td><td rowspan=1 colspan=1>Labeled cRNA</td></tr><tr><td rowspan=1 colspan=1>Pre-analytical SamplePreparation</td><td rowspan=1 colspan=1>Procedure for FFPE tumorsamples</td><td rowspan=1 colspan=1>Procedure for fresh frozentumor samples</td></tr><tr><td rowspan=1 colspan=1>XPrint Analysis Software</td><td rowspan=1 colspan=1>XPrint version 2.24This updated version includesthe module for calculatingMammaPrint® Index (MPI) onFFPE.</td><td rowspan=1 colspan=1>XPrint version 2.0.2</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. CLSI EP5-A2: Evaluation of Precision Performance of Quantitative Measurement Methods; Clinical and Laboratory Standards Institute; 2004.   
2. Guidance for Industry and FDA Staff, Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, 2007.   
3. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis, 2007

# L. Test Principle:

The MammaPrint $\textsuperscript { \textregistered }$ FFPE is a microarray based gene expression analysis of breast tumor tissue. The analysis is based on several processes: isolation of RNA from FFPE breast cancer tissue sections; elimination of gDNA, reverse transcription of RNA resulting in cDNA; amplification and labeling of the cDNA; hybridization of the amplified and labeled cDNA to the diagnostic microarray; washing and scanning the diagnostic microarray and data acquisition (feature extraction); calculation and determination of the risk of recurrence.

Briefly, the amplified and labeled cDNA is hybridized to slides at $6 0 ^ { 0 } \mathrm { C }$ for 17 hours in a rotation oven. By hybridization only complementary cDNA will bind to a 60-mer oligo on the array. For scanning the ManmmaPrint $\textsuperscript { \textregistered }$ FFPE microarray, an Agilent microarray scanner is used. The Agilent DNA microarray scanner is a 48-slide scanning system that can read $1 " \mathbf { x } 3 "$ glass slide microarrays. The result after scanning is a scan file (multi-page TIFF). This TIFF contains two pages, one page for each dye used. These are used by the feature extraction software.

Agilent Feature Extraction Software opens the multipage TIFF and combines those into one image which shows a pattern of differently colored spots. The Feature Extraction Software analyses the scan file (TIFF) by determining the intensities of the individual features, subtracting background signal, perform normalization, and calculate ratios, errors and pvalues for each spot. The feature extraction software uses the MamnmaPrint $\textsuperscript { \textregistered }$ FFPE microarray chip design file as a template in order to identify control features, normalization features and reporter features. The fluorescent intensity of the features is a measure for the activity of that particular gene.

Data analysis is performed according to a specific MamnmaPrint $\textsuperscript { \textregistered }$ FFPE algorithm (MammaPrint $\textsuperscript { \textregistered }$ Index, or MPI). The algorithm calculates the correlation of the sample expression profile to a template (the mean expression profile of 44 tumors with a known good clinical outcome) and determines the molecular profile of the sample. This algorithm is designed and programmed by Agendia and compiled into a standalone software program, "XPrint Analysis Software". The "X-Print Analysis Software" loads a data file (CSV) which is created by the laboratory technician by extracting specific information from the laboratory database. The "X-Print Analysis Software" reads the CSV file, opens the Feature Extraction Software data files (TXT), performs quality control checks, determines the sample expression profile, calculates the correlation of sample profile to the "Low Risk" template profile on a scale of -1.000 to $+ 1 . 0 0 0$ (MammaPrint $\textsuperscript { \textregistered }$ FFPE reportable range), compares the calculated correlation to a pre-defined cut-off value and determines the samples prognostic profile (i.e., Low Risk, High Risk, Low Risk Borderline, or High Risk Borderline).

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility of MammaPrint $\textsuperscript { \textregistered }$ FFPE

Thirty (30) samples of different risk categories (see table below) with sufficient amount of FFPE tissue were isolated 4 times by a single operator. Isolation 1 and 2 (I1 and I2) were performed on day 1, and isolation 3 and 4 (I3 and I4) were performed on day 2. After the isolations were performed the samples were processed further (amplification, labeling, and hybridization) on different days per isolation round; I1, I2, I3 and I4. This resulted in the generation of four results per tissue sample processed on different days. The results were then compared over four isolations to determine if there is a significant difference in indices between different isolations for MammaPrint $\textsuperscript { \textregistered }$ FFPE.

Table 1 Reproducibility Study – Sample Category and Distribution   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>PreferredDistribution</td><td rowspan=1 colspan=1>ActualDistribution</td><td rowspan=1 colspan=1>Specimens(N)</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>40%-50%</td><td rowspan=1 colspan=1>57 %</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>Maximum 5%</td><td rowspan=1 colspan=1>7 %</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>50%-60%</td><td rowspan=1 colspan=1>37 %</td><td rowspan=1 colspan=1>11</td></tr></table>

![](images/ad69f601009ccd980ee286ddc2b38516966acc48fe0900d2d6b793198593dddd.jpg)  
Figure 1 Reproducibility Results of MammaPrint $^ \mathrm { \textregistered }$ FFPE over Four Isolations

A repeated Measurements ANOVA was used to determine if there was a different in MammaPrint $\textsuperscript { \textregistered }$ Index (MPI) over all four isolations. The results show that there is no significant difference $\scriptstyle ( { \mathtt { p } } = 0 . 9 9 4 )$ ). A Cochran’s Q test was used to assess the difference in risk reporting outcome over the 4 isolations. The results of this test show that there is no significant difference in MammaPrint $\textsuperscript { \textregistered }$ FFPE outcome over the different isolations $( \mathtt { p } ^ { = } 0 . 2 9 0 )$ ).

# Repeatability of MammaPrint $\textsuperscript { \textregistered }$ FFPE

A Precision and Evaluation (P&E) experiment (CLSI, Evaluation of Precision Performance of Quantative Measurement Methods, EP5-A2) was performed to determine the Repeatability and Method Precision of MammaPrint $\textsuperscript { \textregistered }$ FFPE test. Three controls and four FFPE samples representing all test outcome levels were analyzed and run repeatedly consecutively over 20 days. RNA was used as starting material for each sample (no RNA pooling of multiple samples). Per day, one run was performed consisting of two replicates of each test sample.

![](images/8041fdb350e7e34966358f92aece86f518316a2f32e3bb108cf53671e286a769.jpg)  
Figure 2 MammaPrint $^ \mathrm { \textregistered }$ FFPE Repeatability – MPI Values Overtime

The standard deviation (Std. Dev.) and variance for MPI were calculated and summarized in table below.

Table 2 Repeatability of MammaPrint $^ \mathrm { \textregistered }$ FFPE - FFPE Samples   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Repeatability(Within-Run)</td><td rowspan=1 colspan=2>Method Precision(within-Laboratory)</td></tr><tr><td rowspan=1 colspan=1>Outcome Level</td><td rowspan=1 colspan=1>MPI(Std. Dev.)</td><td rowspan=1 colspan=1>Variance</td><td rowspan=1 colspan=1>MPI(Std. Dev.)</td><td rowspan=1 colspan=1>Variance</td></tr><tr><td rowspan=1 colspan=1>High Risk(I11003290)</td><td rowspan=1 colspan=1>0.036</td><td rowspan=1 colspan=1>0.0013</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>0.0019</td></tr><tr><td rowspan=1 colspan=1>High Risk(I11003347)</td><td rowspan=1 colspan=1>0.046</td><td rowspan=1 colspan=1>0.0021</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>0.0033</td></tr><tr><td rowspan=1 colspan=1>Low Risk(11003810)</td><td rowspan=1 colspan=1>0.042</td><td rowspan=1 colspan=1>0.0018</td><td rowspan=1 colspan=1>0.050</td><td rowspan=1 colspan=1>0.0025</td></tr><tr><td rowspan=1 colspan=1>Low RiskBorderline(11003860)</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.0024</td><td rowspan=1 colspan=1>0.066</td><td rowspan=1 colspan=1>0.0044</td></tr></table>

Table 3 Repeatability of MammaPrint $^ \mathrm { \textregistered }$ FFPE - Control Samples   

<table><tr><td rowspan=1 colspan=1>Outcome Level</td><td rowspan=1 colspan=1>MPI (Std. Dev.)</td><td rowspan=1 colspan=1>Variance</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>0.070</td><td rowspan=1 colspan=1>0.005</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>0.002</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>0.004</td></tr></table>

These results met the pre-defined acceptance criteria as described in the study report (std. dev. ${ \underline { { < 0 . 2 ) } } }$ . These results also met Product Specifications criteria for MammaPrint $\textsuperscript { \textregistered }$ FFPE [i.e., std. dev. $\leq 0 . 1 2 6$ (equals $7 \%$ of dynamic range (dynamic range ${ \mathrm { \varepsilon } } = 1 . 8$ )].

MammaPrint $\textsuperscript { \textregistered }$ FFPE specific control samples $\scriptstyle ( \mathtt { n } = 3 )$ ) were pooled, amplified, labeled and hybridized according to standard FFPE laboratory protocols in Agendia’s Amsterdam and Irvine laboratories. This was performed on a daily basis in order to obtain MammaPrint $\textsuperscript { \textregistered }$ FFPE results over time. Results were combined for analysis: 25 days of data from Amsterdam scanner serial number US810R3210, 7 days of data from Amsterdam scanner serial number US22502555, and 3 days of data from Irvine scanner serial number US811R3213. The duplicate results on one day were performed in separate experiments by different technicians in the lab. Therefore, on some days when only one technician was performing an experiment in the lab this will result in a single result for that day.

These results met Product Specifications criteria for MammaPrint $\textsuperscript { \textregistered }$ FFPE [i.e., std. dev. $\leq 0 . 1 2 6$ (equals $7 \%$ of dynamic range (dynamic range ${ \displaystyle \ E = 1 . 8 }$ )]. To establish stability of the control samples over time, the standard deviation of MPI values and corresponding regression analysis for each sample are provided in table below.

Table 4 Reproducibility of MammaPrint $^ \mathrm { \textregistered }$ FFPE Control Samples   

<table><tr><td rowspan=1 colspan=1>OutcomeLevel</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MPI(Std.Dev.)</td><td rowspan=1 colspan=1>Mini-mum</td><td rowspan=1 colspan=1>Maxi-mum</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>RegressionAnalysis</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>-1</td><td rowspan=1 colspan=1>-0.649</td><td rowspan=1 colspan=1>-0.855</td><td rowspan=1 colspan=1>0.351</td><td rowspan=1 colspan=1>-0.856</td><td rowspan=1 colspan=1>p=0.020</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0.072</td><td rowspan=1 colspan=1>-1</td><td rowspan=1 colspan=1>-0.817</td><td rowspan=1 colspan=1>-0.991</td><td rowspan=1 colspan=1>0.183</td><td rowspan=1 colspan=1>-0.968</td><td rowspan=1 colspan=1>p=0.009</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0.056</td><td rowspan=1 colspan=1>0.116</td><td rowspan=1 colspan=1>0.395</td><td rowspan=1 colspan=1>0.311</td><td rowspan=1 colspan=1>0.279</td><td rowspan=1 colspan=1>0.305</td><td rowspan=1 colspan=1>p=0.242</td></tr></table>

Inter-Laboratory Comparison of MammaPrint $\textsuperscript { \textregistered }$ FFPE between Amsterdam and Irvine Laboratories

FFPE samples $( \mathtt { n } \mathtt { = } 2 5$ , including 18 high risk and 7 low risk samples) were selected from which MammaPrint $\textsuperscript { \textregistered }$ FFPE results were previously generated using standard FFPE protocols in Product Support department Amsterdam. From these samples, sections were taken and processed from isolation onwards by the Diagnostic departments in both Agendia Irvine and Amsterdam according to standard protocols (i.e., 25 samples at each site). Diagnostic FFPE control samples $\mathrm { n } { = } 3$ for Amsterdam; $\mathrm { n } { = } 1$ to 3 for Irvine) were taken along with each amplification, labeling and hybridization run of these samples. After hybridization and washing the MammaPrint $\textsuperscript { \textregistered }$ FFPE arrays were scanned using Agilent scanner number US810R3210 in Amsterdam and US811R3213 in Irvine. The MammaPrint $\textsuperscript { \textregistered }$ FFPE results generated for these 25 samples at Irvine and Amsterdam locations were compared to the previously generated results. Identical $2 \mathbf { x } 2$ contingency tables of MammaPrint $\textsuperscript { \textregistered }$ FFPE outcome were generated for both sites.

Table 5 Inter-Laboratory Study Results - Comparison to Product SupportAmsterdam (R&D AMS Results)   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>R&amp;D Original Results</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>Diagnostics Operation Results(Amsterdam or Irvine)</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>25</td></tr></table>

Results from Amsterdam Diagnostics, the Pearson correlation coefficient for MPI was 0.929 between original versus diagnostics. Result for Irvine Diagnostics, the Pearson correlation coefficient for MPI was 0.976 between original versus diagnostics. Pearson correlations of MPI at both locations fall within the predefined validation criteria of above or equal to 0.8. Based on the $2 \mathbf { x } 2$ contingency table, the NPA is $100 \%$ ( $9 5 \% C \mathrm { I }$ : 64.6 to 100) and the PPA is $9 4 . 4 \%$ ( $9 5 \%$ CI: 74.2 to 99.0) for both laboratories.

Additional Passing and Bablok regression analysis and Bland-Altman plot were conducted to analyze differences of the MPI values between the two laboratories. Passing and Bablok regression analysis showed high similarities between the two laboratories, with the intercept close to zero and the slope close to 1. Bland-Altman plots demonstrated that the differences between the MPI values from either laboratory against the R&D original results are limited. In addition, magnitude of the differences in MPI reported for the two laboratories is comparable.

Table 6 Inter-Laboratory Study Results - Passing and Bablok Analysis   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Amsterdam</td><td rowspan=1 colspan=3>Irvine</td></tr><tr><td rowspan=1 colspan=1>Values</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>Values</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Intercept</td><td rowspan=1 colspan=1>0.035</td><td rowspan=1 colspan=1>-0.054</td><td rowspan=1 colspan=1>0.084</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>-0.029</td><td rowspan=1 colspan=1>0.097</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1.078</td><td rowspan=1 colspan=1>0.919</td><td rowspan=1 colspan=1>1.203</td><td rowspan=1 colspan=1>1.111</td><td rowspan=1 colspan=1>0.988</td><td rowspan=1 colspan=1>1.228</td></tr></table>

Validation for Use of Multiple Microarray Scanners in Amsterdam and Irvine

For validation of both microarray scanners in the Amsterdam central lab, 25 samples (i.e., 16 high risk, 8 low risk and 1 borderline samples) were scanned two times; first on the originally validated scanner, serial number US810R3210 and in addition using the scanner with serial number US22502555. For validation of both microarray scanners in the Irvine central lab, 27 samples (i.e., 13 high risk, 13 low risk and 1 borderline samples) were scanned two times; first on the originally validated scanner, serial number US811R3213 and in addition using the additional scanner with serial number US45103019. MPI were compared between both scanners at each central lab.

At both central lab testing sites, the comparison of MPI between both scanners met the predefined acceptance criteria, i.e., both arrays are considered in agreement when the Pearson correlation is above or equal to 0.8. The observed Pearson correlation was 1.0 for each site. Based on the $2 \mathbf { x } 2$ contingency table for the 2nd scanner in Amsterdam, the NPA is $100 \%$ ( $9 5 \%$ CI: 67.6 to 100) and the PPA is $100 \%$ ( $9 5 \% C \mathrm { I }$ : 81.6 to 99.0). Based on the $2 \mathbf { x } 2$ contingency table for the 2nd scanner in Irvine, the NPA is $100 \%$ ( $9 5 \%$ CI: 77.2 to 100) and the PPA is $100 \%$ ( $9 5 \% C \mathrm { I }$ : 78.5 to 100).

Table 7 Microarray Scanners Study Results at Amsterdam Testing Site   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Amsterdam Scanner 1 (US810R3210)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>AmsterdamScanner 2(US22502555)</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>25</td></tr></table>

Table 8 Microarray Scanners Study Results at Irvine Testing Site   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Irvine Scanner 1 (US811R3213)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IrvineScanner 2(US45103019)</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>27</td></tr></table>

Additional Passing and Bablok regression analysis and Bland-Altman plot were conducted to analyze differences of the MPI values between the scanners at each laboratory. Passing and Bablok regression analysis showed high similarities between the two scanners at each laboratory, with the intercept close to/equal to zero and the slope close to/equal to 1. Bland-Altman plots demonstrated that differences between the scanners within each site are limited, and that magnitude of the between-scanner differences is comparable for the two laboratories.

Table 9 Microarray Scanners Study Results - Passing and Bablok Analysis   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Amsterdam</td><td rowspan=1 colspan=3>Irvine</td></tr><tr><td rowspan=1 colspan=1>Values</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>Values</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Intercept</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-0.00787</td><td rowspan=1 colspan=1>0.00034</td><td rowspan=1 colspan=1>-0.00046</td><td rowspan=1 colspan=1>-0.00607</td><td rowspan=1 colspan=1>-0.00045</td></tr><tr><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.9921</td><td rowspan=1 colspan=1>1.0072</td><td rowspan=1 colspan=1>0.9954</td><td rowspan=1 colspan=1>0.9893</td><td rowspan=1 colspan=1>1.0028</td></tr></table>

# b. Linearity/assay reportable range:

Linearity is not applicable for this type of qualitative assay.

As reported in MammaPrin $\textsuperscript { \textregistered }$ , the correlation coefficient to the good profile for MPI is reported on a scale of -1.000 to $+ 1 . 0 0 0$ . In current submission, the mathematical shift in MammaPrint $\textsuperscript { \textregistered }$ cut-off from 0.415 to 0.0 resulted in the virtual range to be on a scale of -1.415 to $+ 0 . 5 8 5$ for fresh-frozen samples. For MammaPrint $\textsuperscript { \textregistered }$ FFPE

samples, the correlation coefficient to the good profile for MPI is reported on a scale of -1.000 to $+ 1 . 0 0 0$ .

For both MammaPrint $\textsuperscript { \textregistered }$ and MammaPrint $\textsuperscript { \textregistered }$ FFPE samples, the cut-off for MPI is set at $+ 0 . 0$ .

Table 10 MammaPrint $^ \mathrm { \textregistered }$ Index (MPI) Cut-Offs and Borderline Ranges   

<table><tr><td rowspan=1 colspan=1>MPI</td><td rowspan=1 colspan=1>MammaPrint®FFPE</td><td rowspan=1 colspan=1>MammaPrint®</td></tr><tr><td rowspan=1 colspan=1>Cut-off</td><td rowspan=1 colspan=1>+0.0</td><td rowspan=1 colspan=1>+0.0</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>≤ +0.0</td><td rowspan=1 colspan=1>≤+0.0</td></tr><tr><td rowspan=1 colspan=1>LowRisk</td><td rowspan=1 colspan=1>&gt; +0.0</td><td rowspan=1 colspan=1>&gt; +0.0</td></tr><tr><td rowspan=1 colspan=1>HighRisk Borderline</td><td rowspan=1 colspan=1>-0.0575 to 0.0</td><td rowspan=1 colspan=1>-0.0275 to 0.0</td></tr><tr><td rowspan=1 colspan=1>LowRiskBorderline</td><td rowspan=1 colspan=1>0.0 to +0.0575</td><td rowspan=1 colspan=1>0.0 to +0.0275</td></tr><tr><td rowspan=1 colspan=1>Borderline Range</td><td rowspan=1 colspan=1>0.115</td><td rowspan=1 colspan=1>0.055</td></tr></table>

The “borderline region” refers to a range of MPI values surrounding the clinical classification threshold. In this region the clinical classification result accuracy potentially falls below the overall predetermined analytical accuracy of $90 \%$ . This occurs as a result of technical variation in the performance of MammaPrint $\textsuperscript { \textregistered }$ and MammaPrint $\textsuperscript { \textregistered }$ FFPE, only when the MPI value is close to the classification threshold. The technical variation was determined using the median standard deviation of the control sample measurements in the precision and repeatability study. Samples outside this borderline region have $5 9 0 \%$ chance of being correctly classified. This technical variation (i.e., $> 1 0 \%$ chance of false classification for borderline samples) does not change with the mathematical shift of cut-off from 0.415 to 0 for MammaPrint $\textsuperscript { \textregistered }$ .

Samples that lie within the borderline region and that are close to the threshold are more likely to switch classes with repeated analyses. In a diagnostic setting these samples will be performed in duplicate in order to obtain better outcome accuracy. A borderline sample will be re-tested from RNA onwards going through the following steps a second time: synthesis, amplification, labeling, hybridization, and scanning and XPrint analysis. After two QC-passed results are generated for a borderline sample, the MammaPrint $\textsuperscript { \textregistered }$ Index of both results will be averaged, and the final risk classification will be made according to the mean value.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Control Materials

Positive controls are pooled RNA starting from amplification. Negative control is DNAse/RNAse free water.

MammaPrint $\textsuperscript { \textregistered }$ FFPE control pools are created from samples with known outcome and are available for at least one year. New control samples are created and checked

for stability in at least 20 measurements prior to diagnostic implementation and well in advance of depletion of the current control samples.

# Quality Controls

Quality of the RNA was assessed for every FFPE sample through the RNA Quality Assay, and the QC model on the array gives the final call of the quality of the amplification and hybridization.

Quality of the complete MammaPrint $\textsuperscript { \textregistered }$ FFPE is monitored from sample reception up to and including external reporting. Quality controls (QCs) are used to monitor the quality of the overall hybridization on the array referred to as Technical QCs as well as the MammaPrint $\textsuperscript { \textregistered }$ FFPE specific test readout referred to as Product Specific QCs. Combining both types of quality controls results in a total of 28 QCs.

# Device Stability

The chip is stable for at least one year. The Cy Dyes are stable for up to 3 months at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# d. Detection limit:

Minimum labeled cDNA Input Amount

For FFPE samples, isolated RNA is diluted to a final concentration of $2 5 \mathrm { { n g / \mu l } }$ prior to cDNA generation. Labeled cDNA is generated using $1 3 ~ \mu \mathrm { l }$ of cDNA at 100.9 $\mathrm { n g / \mu l }$ , which is equivalent to $1 3 1 1 . 7 \mathrm { n g }$ as the regular input. When the isolated RNA and cDNA cannot be diluted to the pre-specified final concentrations of $2 5 ~ \mathrm { { n g / \mu l } }$ , and $1 0 0 . 9 \mathrm { n g / \mu l }$ , respectively, the samples will not be analyzed and will be reported as “Quality Not Sufficient” (QNS).

A dilution study was performed to determine the minimum for input of labeled cDNA in hybridization of MammaPrint $\textsuperscript { \textregistered }$ FFPE. Total amount of labeled cDNA (ng) ranged from 0, 300, 900, 1500, 2100, 2700, and $3 3 0 0 { \mathrm { n g } }$ . The MPI showed stable results at 900, 1500, 2100, 2700, and $3 3 0 0 { \mathrm { n g } }$ . The lowest hybridization input $( 9 0 0 \mathrm { n g } )$ of cDNA with stable MPI results is below the regular input of $1 3 0 0 \mathrm { n g }$ .

# Percent $( \% )$ Tumor Content

A minimum of $30 \%$ tumor content in a sample was used in the method comparison study as well as in the RASTER study for MammaPrint $\textsuperscript { \textregistered }$ FFPE. Therefore, MammaPrint $\textsuperscript { \textregistered }$ FFPE will require a minimum of $30 \%$ tumor content in an H&E stained slide, as stated in Agendia’s “Specimens Sampling Instructions” insert for MammaPrint $\textsuperscript { \textregistered }$ FFPE.

# e. Analytical specificity:

The isolated RNA and amplified cDNA are quantified per SOPs using a spectrometer. The ratio of absorbance at $2 6 0 \mathrm { n m }$ and $2 8 0 \mathrm { n m }$ is recorded as a measure to assess the purity of RNA and cDNA, and included as part of QC to support the absence of interfering substances. Specifically, the 260/280 ratio for the first method comparison study dataset $\mathrm { ( n { = } 1 2 2 }$ ) ranged from 1.74 to 1.90. The 260/280 ratio for RASTER FFPE dataset $( \mathrm { n } { = } 3 4 5 )$ ranged from 1.6 to 2.41. RNA and cDNA specifications provided are adequate to exclude the presence of any effect from likely interfering substances.

f. Assay cut-off:

MPI cut-off is set at $+ 0 . 0$ for MammaPrint $\textsuperscript { \textregistered }$ FFPE.

2. Comparison studies:

# a. Method comparison with predicate device:

Assay development to correlate MPI between MammaPrint $\textsuperscript { \textregistered }$ (for use with fresh and fresh-frozen samples) and MammaPrint $\textsuperscript { \textregistered }$ FFPE was conducted using a training set of 125 samples to lock the MammaPrint $\textsuperscript { \textregistered }$ FFPE. The 125 samples with matched freshfrozen (FF) and FFPE sections were analyzed with MammaPrint $\textsuperscript { \textregistered }$ or MammaPrint $\textsuperscript { \textregistered }$ FFPE, respectively. The genes and algorithm used to calculate the MPI for MammaPrint $\textsuperscript { \textregistered }$ FFPE was unchanged from those used for MammaPrint $\textsuperscript { \textregistered }$ . The test results were used to determine calibration necessary for MammaPrint $\textsuperscript { \textregistered }$ FFPE, utilizing Passing and Bablok regression analysis with Analyse-it for Microsoft Excel (version 2.20).

The locked MammaPrint $\textsuperscript { \textregistered }$ FFPE was subsequently validated against two independent datasets, where breast cancer tumor samples with matched FF and FFPE were used in these studies. The FF section of the tumor was subjected to MammaPrint $\textsuperscript { \textregistered }$ (internal version US09.1 / EU09.1) and the FFPE sections were subjected to MammaPrint $\textsuperscript { \textregistered }$ FFPE (internal version US01.1 / EU01.1). The two validation datasets included breast cancer samples in the IU population (i.e., Stage I or Stage II disease, tumor size $\le 5 . 0$ cm and lymph node negative). Samples in the two validation datasets do not include any of the 125 training dataset samples.

# First Validation Dataset $\mathrm { \Delta _ { n = } }$ 122)

The first validation dataset used samples collected from two European sites $( \mathrm { n } { = } 4 5 )$ and from a U.S. site $( \mathrm { n } { = } 7 7 )$ , resulting a total of 122 independent samples for this dataset. Comparison of MPI and risk reporting outcomes between FF and FFPE is shown below, graphically for MPI and in a $4 \mathrm { x } 4$ table for the risk reporting outcomes.

![](images/b225e31c36765e2f17e91ebf41b0eeb21f2673464a05a0b89f7839c6aba6c0a2.jpg)  
Figure 3 First Validation Dataset $\scriptstyle ( \mathbf { n } = 1 2 2$ ) – MPI, FFPE vs. FF Comparison

Table 11 First Validation Dataset $\scriptstyle ( \mathbf { n } = 1 2 2 )$ ) – Outcomes, FFPE vs. FF Comparison   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>MammaPrint®</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>MammaPrint®FFPE</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>122</td></tr></table>

Assay performance was assessed by calculating the positive percent agreement (PPA) and negative percent agreement (NPA). Point estimate, number of samples (N), and the corresponding $9 5 \%$ confidence interval $9 5 \%$ CI) are listed below accordingly.

Table 12 First Validation Dataset – Agreement Analysis   
Second Validation Dataset $\scriptstyle ( \mathrm { n } = 3 4 5 $ , RASTER)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>First Validation Dataset (n=122)</td></tr><tr><td rowspan=1 colspan=1>Point Estimate</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>90.6%</td><td rowspan=1 colspan=1>48/53</td><td rowspan=1 colspan=1>79.8% - 95.9%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>88.4%</td><td rowspan=1 colspan=1>61/69</td><td rowspan=1 colspan=1>80.5% - 95.0%</td></tr><tr><td rowspan=1 colspan=1>OverallConcordance</td><td rowspan=1 colspan=1>89.3%</td><td rowspan=1 colspan=1>109/122</td><td rowspan=1 colspan=1>82.1% - 94.0%</td></tr></table>

The second validation dataset used samples collected from the RASTER study; see Section 3 Clinical Studies below for more details on RASTER study design. The second validation dataset included a total 345 samples where both FF tissue as well as an FFPE tissue counterpart was available from the original RASTER study that included 427 patients. Comparison of MPI and risk reporting outcomes between FF and FFPE is shown below, graphically for MPI and in a $4 \mathrm { x } 4$ table for the reporting outcomes.

![](images/9d96a7ca14142fbbc253c0a148adcae8e50e7b106ed8eb990b64e44086384b89.jpg)  
Figure 4 Second Validation Dataset $\scriptstyle ( \mathbf { n } + 3 4 5 )$ ) – MPI, FFPE vs. FF Comparison

Table 13 Second Validation Dataset $\scriptstyle ( \mathbf { n } + 3 4 5 )$ ) – Outcomes, FFPE vs. FF Comparison   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>MammaPrint®</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>MammaPrint®FFPE</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>High RiskBorderline</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Low RiskBorderline</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>345</td></tr></table>

Assay performance was assessed by calculating the positive percent agreement (PPA) and negative percent agreement (NPA). Point estimate, number of samples $( \mathrm { N } )$ , and the corresponding $9 5 \%$ confidence interval ( $9 5 \%$ CI) are listed below accordingly.

Table 14 Second Validation Dataset – Agreement Analysis   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Second Validation Dataset (n=345)</td></tr><tr><td rowspan=1 colspan=1>Point Estimate</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>86.6%</td><td rowspan=1 colspan=1>136/157</td><td rowspan=1 colspan=1>80.4% - 91.1%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>91.5%</td><td rowspan=1 colspan=1>172/188</td><td rowspan=1 colspan=1>86.7% - 94.7%</td></tr><tr><td rowspan=1 colspan=1>OverallConcordance</td><td rowspan=1 colspan=1>89.3%</td><td rowspan=1 colspan=1>308/345</td><td rowspan=1 colspan=1>85.4% - 92.2%</td></tr></table>

b. Matrix comparison: Not applicable.

3. Clinical studies: a. Clinical Sensitivity: Not applicable. b. Clinical specificity: Not applicable.

c. Other clinical supportive data (when a. and b. are not applicable):

The microarRAy prognoSTics in breast cancER (RASTER) study was conducted in 16 community hospitals in the Netherlands between 2004 and 2006. The primary aim of this multicenter observational study was to assess the feasibility of implementing the MammaPrint $\textsuperscript { \textregistered }$ 70-gene signature in a community-based setting and to study the clinical impact of the 70-gene signature test result on adjuvant systemic therapy (AST) decision making. For 427 patients enrolled in the RASTER, treatment decisions were based on standard guidelines, the MammaPrint $\textsuperscript { \textregistered }$ 70-gene signature, and doctors’ and patients’ preferences. In January 2013, the 5 year outcome results of the prospective observational RASTER study were published1 , which included the analysis of estimated five-year distant-recurrence free interval (DRFI).

To support clinical performance of MammaPrint $\textsuperscript { \textregistered }$ FFPE, MammaPrint $\textsuperscript { \textregistered }$ FFPE was also performed on FFPE tissue of the RASTER patients. Results from MammaPrint $\textsuperscript { \textregistered }$ Fresh and MammaPrint $\textsuperscript { \textregistered }$ FFPE were compared for the 345 paired fresh and FFPE samples with 5 year outcome data from the 427 RASTER patient samples. DRFI was the study endpoint as defined in the RASTER study. Specifically, DRFI measures the time until the diagnosis of distant metastasis or death from breast cancer. The local, regional and second primaries prior to Distant Metastasis are also ignored. Data on all other patients were censored on the date of the last follow-up visit or date of death. The RASTER samples included in this comparison $\scriptstyle ( \mathrm { n } = 3 4 5 )$ comply with the MammaPrint $\textsuperscript { \textregistered }$ FFPE intended use population.

Table 15 RASTER Study: 5-year DFRI for MammaPrint $^ \mathrm { \textregistered }$ and MammaPrint® FFPE   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>MammaPrint®</td><td rowspan=1 colspan=3>MammaPrint® FFPE</td></tr><tr><td rowspan=1 colspan=1>No Recurrencewithin 5 years</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Low Risk Signature</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>High Risk Signature</td><td rowspan=1 colspan=1>89.1</td><td rowspan=1 colspan=1>84.0</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>83.0</td><td rowspan=1 colspan=1>94.0</td></tr></table>

Not accounting for any covariates other than the patient’s MammaPrint $\textsuperscript { \textregistered }$ FFPE status, patients classified as ‘Low Risk’ by MammaPrint $\textsuperscript { \textregistered }$ FFPE (81 adjuvantly treated and 112 adjuvantly not treated), demonstrated a $2 . 3 \%$ ( $9 5 \%$ CI 0.1-4.5) chance of cancer recurrence within 5 years. Patients classified as ‘High Risk’ by MammaPrint $\textsuperscript { \textregistered }$ FFPE (135 adjuvantly treated and 17 adjuvantly not treated), demonstrated a $1 1 . 5 \%$ $( 9 5 \% \mathrm { C I } 6 . 0 – 1 7 . 0 )$ chance of cancer recurrence within 5 years. Kaplan-Meier curves showed a similar difference in DRFI between the Low and High Risk comparing MammaPrint $\textsuperscript { \textregistered }$ and MammaPrint $\textsuperscript { \textregistered }$ FFPE.

![](images/838aecef4d3a76ec37a0d048840f36d6e30f5a3170b7393ce4b887cafc12707b.jpg)  
Figure 5 RASTER Study: Kaplan-Meier Analysis of the 5-year DRFI for MammaPrint $^ \mathrm { \textregistered }$ and MammaPrint $^ \mathrm { \textregistered }$ FFPE

Prognostic assessment of MammaPrint $\textsuperscript { \textregistered }$ FFPE was further investigated using univariate and multivariate analyses. In the univariate analysis, a MammaPrint $\textsuperscript { \textregistered }$ FFPE High/Low Risk result is significantly associated with high/low risk for recurrence. Multivariate analysis of the 345 samples analyzed did not conclusively demonstrate prognostic significance for MammaPrint $\textsuperscript { \textregistered }$ FFPE beyond that of other clinico-pathological factors. This is attributable to the RASTER study design, in which MammaPrint $\textsuperscript { \textregistered }$ result was included along with all relevant clinico-pathological factors, and treatment decisions were guided by assessed prognostic risk and the standard of practice. In this real–world context, the overall cohort experienced a low event rate which, despite the favorable trend, diminishes independent contribution of MammaPrint $\textsuperscript { \textregistered }$ FFPE. Aggregate outcomes were broadly similar between study patients who received adjuvant chemotherapy and study patients who did not.

Table 16 Univariate analysis: DRFI at 5 years   

<table><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>HazardRatio</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>MammaPrint®FFPE</td><td rowspan=1 colspan=1>High vs. Low</td><td rowspan=1 colspan=1>0.002</td><td rowspan=1 colspan=1>5.443</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>16.28</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Age&lt;=50 vs.Age&gt;50</td><td rowspan=1 colspan=1>0.778</td><td rowspan=1 colspan=1>1.135</td><td rowspan=1 colspan=1>0.470</td><td rowspan=1 colspan=1>2.739</td></tr><tr><td rowspan=1 colspan=1>Tumor Size</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.003</td><td rowspan=1 colspan=1>1.047</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.08</td></tr><tr><td rowspan=3 colspan=1>Grade</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.008</td><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.430</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>20.18</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.675</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>66.33</td></tr><tr><td rowspan=1 colspan=1>ER</td><td rowspan=1 colspan=1>Positive vs.Negative</td><td rowspan=1 colspan=1>0.002</td><td rowspan=1 colspan=1>0.244</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.59</td></tr><tr><td rowspan=1 colspan=1>HER2</td><td rowspan=1 colspan=1>Positive vs.Negative</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>2.718</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>8.13</td></tr><tr><td rowspan=1 colspan=1>EndocrineTherapy (ET)</td><td rowspan=1 colspan=1>None vs. ET</td><td rowspan=1 colspan=1>0.207</td><td rowspan=1 colspan=1>0.540</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>1.41</td></tr></table>

Table 17 Multivariate analysis: DRFI at 5 years   

<table><tr><td rowspan=1 colspan=1>Variable</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>HazardRatio</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>MammaPrint®FFPE</td><td rowspan=1 colspan=1>High vs. Low</td><td rowspan=1 colspan=1>0.087</td><td rowspan=1 colspan=1>3.776</td><td rowspan=1 colspan=1>0.827</td><td rowspan=1 colspan=1>17.250</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Age&lt;=50 vs.Age&gt;50</td><td rowspan=1 colspan=1>0.250</td><td rowspan=1 colspan=1>1.708</td><td rowspan=1 colspan=1>0.686</td><td rowspan=1 colspan=1>4.253</td></tr><tr><td rowspan=1 colspan=1>Tumor Size</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.013</td><td rowspan=1 colspan=1>1.063</td><td rowspan=1 colspan=1>1.013</td><td rowspan=1 colspan=1>1.115</td></tr><tr><td rowspan=3 colspan=1>Grade</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.539</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.867</td><td rowspan=1 colspan=1>0.206</td><td rowspan=1 colspan=1>16.939</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3.623</td><td rowspan=1 colspan=1>0.285</td><td rowspan=1 colspan=1>46.108</td></tr><tr><td rowspan=1 colspan=1>ER</td><td rowspan=1 colspan=1>Positive vs.Negative</td><td rowspan=1 colspan=1>0.463</td><td rowspan=1 colspan=1>2.071</td><td rowspan=1 colspan=1>0.296</td><td rowspan=1 colspan=1>14.465</td></tr><tr><td rowspan=1 colspan=1>HER2</td><td rowspan=1 colspan=1>Positive vs.Negative</td><td rowspan=1 colspan=1>0.392</td><td rowspan=1 colspan=1>1.649</td><td rowspan=1 colspan=1>0.525</td><td rowspan=1 colspan=1>5.181</td></tr><tr><td rowspan=1 colspan=1>EndocrineTherapy (ET)</td><td rowspan=1 colspan=1>None vs. ET</td><td rowspan=1 colspan=1>0.133</td><td rowspan=1 colspan=1>0.270</td><td rowspan=1 colspan=1>0.049</td><td rowspan=1 colspan=1>1.488</td></tr></table>

4. Clinical cut-off:

Same as Assay cut-off. Risk assessment is reported as Low Risk $( \mathrm { M P I } > + 0 . 0 )$ ), High Risk $( \mathrm { M P I } \le + 0 . 0 )$ , High Risk Borderline or Low Risk Borderline. Borderline classification is provided when the MPI is between $+ 0 . 0 5 7 5$ and -0.0575.

5. Expected values/Reference range:

<table><tr><td rowspan=1 colspan=1>MammaPrint® FFPE Result</td><td rowspan=1 colspan=1>Expected Value/Range</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>≤ +0.0</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>&gt; +0.0</td></tr><tr><td rowspan=1 colspan=1>High Risk Borderline</td><td rowspan=1 colspan=1>-0.0575 to 0.0</td></tr><tr><td rowspan=1 colspan=1>LowRisk Borderline</td><td rowspan=1 colspan=1>0.0 to +0.0575</td></tr><tr><td rowspan=1 colspan=1>Reportable Range</td><td rowspan=1 colspan=1>-1.000 to +1.000</td></tr></table>

# N. Instrument Name:

See special instrument requirements above (H-4).

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes __√_ _ or No _

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No _√_

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __√_ _ or No

3. Specimen Identification:

Limited patient identification (e.g., name, date of birth and gender) and specimen information (e.g., collection date, test request date, report date and specimen type) are included as part of the MammaPrint $\textsuperscript { \textregistered }$ FFPE results reporting form to the ordering health care provider. The ordering health care provider is identified as “customer” on the MammaPrint $\textsuperscript { \textregistered }$ FFPE results reporting form.

4. Specimen Sampling and Handling:

Specimen Preparation Instruction is included as part of the MammaPrint $\textsuperscript { \textregistered }$ FFPE Sample Collection Kit.

5. Calibration:

MammaPrint $\textsuperscript { \textregistered }$ FFPE analysis does not require calibration except for standard laboratory equipment as used in MammaPrint $\textsuperscript { \textregistered }$ using FF, such as pipettes, scanners and etc..

6. Quality Control:

Quality of the complete MammaPrint $\textsuperscript { \textregistered }$ FFPE is monitored from sample reception up to and including external reporting. Quality controls were selected to monitor the quality of the overall hybridization on the array referred to as Technical QCs as well as the MammaPrint $\textsuperscript { \textregistered }$ FFPE specific test readout referred to as Product Specific QCs. Combining both types of quality controls results in a total of 28 QCs.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

None.

# Q. Proposed Labeling:

For health care provider ordering the MammaPrint $\textsuperscript { \textregistered }$ FFPE test, two labeling documentations listed below are provided in the sample collection kit.

Specimens Sampling Instructions Physician’s Brochure

Upon completion of the MammaPrint $\textsuperscript { \textregistered }$ FFPE test, the ordering health care provider will receive MammaPrint $\textsuperscript { \textregistered }$ FFPE results for the risk determination. There are four (4) different patient report forms, i.e., high risk, low risk, borderline high risk, and borderline low risk.

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.

The device is classified as Class II under regulation 21 CFR 862.6040 with special controls. The special control guidance document “Guidance for Industry and FDA Staff – Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis” is available at   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/uc m079163.htm.