# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K151565 B. Purpose for Submission:

The purpose of this submission is to show that the Cepheid® Xpert® Trichomonas vaginalis (TV) Assay on the Cepheid GeneXpert Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpertInfinity-80 systems) is substantially equivalent to the Gen-Probe APTIMA® Trichomonas vaginalis Assay on the PANTHER® System.

# C. Measurand:

Trichomonas vaginalis (TV) DNA

# D. Type of Test:

Real-time polymerase chain reaction (PCR)

# E. Applicant:

Cepheid

F. Proprietary and Established Names:

Proprietary Name: Xpert® TV

Common Names: Xpert TV Assay, Xpert Trichomonas Assay

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3860, Trichomonas vaginalis nucleic acid assay

2. Classification: Class II

3. Product code: OUY, OOI

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Cepheid Xpert TV Assay, performed on the GeneXper $\textsuperscript { \textregistered }$ Instrument Systems, is a qualitative in vitro diagnostic test for the detection of Trichomonas vaginalis genomic DNA. The test utilizes automated real-time polymerase chain reaction (PCR) to detect Trichomonas vaginalis genomic DNA. The Xpert TV Assay uses female urine specimens, endocervical swab specimens, or patient-collected vaginal swab specimens (collected in a clinical setting). The Xpert TV Assay is intended to aid in the diagnosis of trichomoniasis in symptomatic or asymptomatic individuals.

Ancillary Collection Kits:

Xpert Vaginal/Endocervical Specimen Collection Kit

The Cepheid® Xpert® Vaginal/Endocervical Specimen Collection Kit is designed to collect, preserve, and transport Chlamydia trachomatis, Neisseria gonorrhoeae, and Trichomonas vaginalis DNA in endocervical swab specimens (collected by a clinician) and patientcollected vaginal swab specimens (collected in a clinical setting) from symptomatic and asymptomatic women prior to analysis with the Xpert CT/NG Assay and the Xpert TV Assay.

Xpert Urine Specimen Collection Kit

The Cepheid® Xpert® Urine Specimen Collection Kit is designed to preserve and transport Chlamydia trachomatis, Neisseria gonorrhoeae, and Trichomonas vaginalis DNA in firstcatch urine specimens from symptomatic and asymptomatic individuals prior to analysis with the Xpert CT/NG Assay and the Xpert TV Assay. The Xpert Urine Specimen Collection Kit is intended for use with male (Xpert CT/NG Assay) and female (Xpert CT/NG Assay and Xpert TV Assay) urine.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: The Cepheid Xpert TV Assay uses PCR technology on the GeneXpert Instrument Systems, which extract, amplify, and detect the target DNA.

# I. Device Description:

The Xpert TV Assay is an automated real-time polymerase chain reaction (PCR) in vitro diagnostic test for qualitative detection of genomic DNA from Trichomonas vaginalis. The Xpert TV Assay is intended as an aid in the diagnosis of trichomoniasis.

The Xpert TV Assay is performed on the Cepheid GeneXpert $\textsuperscript { \textregistered }$ Instrument Systems (GeneXpert Dx, GeneXpert Infinity-48, GeneXpert Infinity-48s, and GeneXpert Infinity-80 systems). The GeneXpert Instrument System consists of an instrument, personal computer, and preloaded software for running tests and viewing the results. The GeneXpert Instrument System requires single-use, disposable cartridges (the Xpert TV cartridges) that hold the PCR reagents and host the PCR process. The cartridges are self-contained, so specimens never come into contact with the working parts of the instrument.

A Sample Processing Control (SPC), Sample Adequacy Control (SAC), and a Probe Check Control (PCC) are controls utilized by the GeneXpert Instrument System. The SPC is present to control for adequate processing of the target trichomonads and to monitor the presence of inhibitors in the real-time PCR reaction to reduce the possibility of false negative results. The SAC reagents detect the presence of a single copy human gene and monitor whether the specimen contains human cells. The PCC verifies reagent rehydration, real-time PCR tube filling in the cartridge, probe integrity, and dye stability.

The ancillary specimen collection kits for use with the Xpert TV Assay are the Cepheid Xpert Vaginal/Endocervical Specimen Collection Kit and the Cepheid Xpert Urine Specimen Collection Kit. The swab and/or urine specimens are collected from asymptomatic or symptomatic patients and placed into a specimen transport tube containing preservative. After transferring the specimen to the sample chamber of the Xpert TV cartridge, the user initiates a test, and places the cartridge into the GeneXpert Instrument System. Results are automatically generated by the instrument at the end of the process in a report that can be viewed and printed.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Gen-Probe® APTIMA Trichomonas vaginalis Assay on the PANTHER System.

2. Predicate 510(k) number(s): K122062

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>Subject Device: Cepheid Xpert TV Assay (K151565)</td><td>Predicate Device: Gen-Probe APTIMA Trichomonas vaginalis Assay (K122062)</td></tr><tr><td>Intended Use</td><td>The Cepheid Xpert TV Assay, performed on the GeneXpert® Instrument Systems, is a qualitative in vitro diagnostic test for the detection of Trichomonas vaginalis genomic DNA. The test utilizes automated real-time polymerase chain reaction (PCR) to detect Trichomonas vaginalis genomic DNA. The Xpert TV Assay uses female urine specimens, endocervical swab specimens, or patient-collected vaginal swab specimens (collected in a clinical setting). The Xpert TV Assay is intended to aid in the diagnosis of trichomoniasis in symptomatic or asymptomatic individuals. Ancillary Collection Kits: Xpert Vaginal/Endocervical Specimen Collection Kit The Cepheid® Xpert® Vaginal/Endocervical Specimen Collection Kit is designed to collect, preserve, and transport Chlamydia trachomatis, Neisseria gonorrhoeae, and Trichomonas vaginalis DNA in endocervical swab specimens (collected by a</td><td>The APTIMA Trichomonas vaginalis Assay is an in vitro qualitative nucleic acid amplification test (NAAT) for the detection of ribosomal RNA (rRNA) from Trichomonas vaginalis to aid in the diagnosis of trichomoniasis using the PANTHER System. The assay may be used to test the following specimens from symptomatic or asymptomatic women: clinician-collected endocervical swabs, clinician- collected vaginal swabs, and specimens collected in PreservCyt Solution.</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject Device:Cepheid Xpert TV Assay(K151565)</td><td colspan="1" rowspan="1">Predicate Device:Gen-Probe APTIMATrichomonas vaginalis Assay(K122062)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TV Assay.Xpert Urine Specimen CollectionKitThe Cepheid® Xpert® UrineSpecimen Collection Kit isdesigned to preserve and transportChlamydia trachomatis, Neisseriagonorrhoeae, and Trichomonasvaginalis DNA in first-catch urinespecimens from symptomatic andasymptomatic individuals prior toanalysis with the Xpert CT/NGAssay and the Xpert TV Assay.The Xpert Urine SpecimenCollection Kit is intended for usewith male (Xpert CT/NG Assay)and female (Xpert CT/NG Assayand Xpert TV Assay) urine.</td><td colspan="1" rowspan="1">b</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AssayResults</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same as predicate</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Endocervical SwabsFemale Urine</td><td colspan="1" rowspan="1">Endocervical SwabsFemale Urine</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject Device:Cepheid Xpert TV Assay(K151565)</td><td colspan="1" rowspan="1">Predicate Device:Gen-Probe APTIMATrichomonas vaginalis Assay(K122062)</td></tr><tr><td colspan="1" rowspan="1">AssayTarget</td><td colspan="1" rowspan="1">T. vaginalis genomic DNA</td><td colspan="1" rowspan="1">T. vaginalis ribosomal RNA</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Real-time polymerase chainreaction (PCR)</td><td colspan="1" rowspan="1">Transcription-mediatedamplification (TMA)</td></tr><tr><td colspan="1" rowspan="1">InstrumentSystem</td><td colspan="1" rowspan="1">Cepheid GeneXpert InstrumentSystem</td><td colspan="1" rowspan="1">PANTHER Instrument System</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Patient-Collected Vaginal Swabs</td><td colspan="1" rowspan="1">Clinician-Collected VaginalSwabs</td></tr></table>

K. Standard/Guidance Documents Referenced (if applicable): 1. FDA. Format for Traditional and Abbreviated 510(k)s. Guidance for Industry and FDA Staff; 2005. 2. FDA. Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Systems. Guidance for Industry and FDA Staff; 2005. 3. FDA. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices. Guidance for Industry and FDA Staff; 2005. 4. FDA. Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens That Are Not Individually Identifiable. Guidance for Industry and FDA Staff; 2006. 5. FDA. Off-The-Shelf Software Use in Medical Devices. Guidance for Industry and FDA Staff; 1999. 6. FDA. Cybersecurity for Networked Medical Devices Containing Off-the-Shelf (OTS) Software. Guidance for Industry and FDA Staff; 2005. 7. FDA. General Principles of Software Validation. Guidance for Industry and FDA Staff; 2002. 8. FDA. Content of Premarket Submissions for Management of Cybersecurity in Medical Devices. Guidance for Industry and FDA Staff; 2014. 9. FCC. Part 15 (Subparts A and B) and Part 18. 10. CLSI MM03-A2: Molecular Diagnostics Methods for Infectious Disease – Second Edition; 2006. 11. CLSI EP17-A2, Protocols for Determination of Limits of Detection and Limits of Quantitation, Approved Guideline – Second Edition; 2006. 12. CLSI EP7-A2, Interference Testing in Clinical Chemistry, Approved Guideline – Second Edition; 2004. 13. CLSI EP15-A2, User Verification of Performance for Precision and Trueness, Approved Guideline - Second Edition; 2006. 14. CLSI Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline – Second Edition; 2004. 15. ISO 10993: Biological Evaluation of Medical Devices –Parts 1 (2009),7 (2008), and 10; 2010. 16. ISO 11135-1: Sterilization of Health Care Products – Ethylene Oxide – Part 1; 2007. 17. ISO 11137-2: Sterilization of Health Care Products – Radiation – Part 2; 2013. 18. ASTM D4169: Standard Practice for Performance Testing of Shipping Containers and Systems; 2009. 19. EN 13640: Stability Testing of in Vitro Diagnostic Reagents; 2002. 20. EN 61326-1: Electrical Equipment for Measurement and Control and Laboratory Use – Parts 1, 2, and 6; 2006. 21. EN 61010: Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Parts 1 (2001) and 2 (2002). 22. EN 55011: Industrial, Scientific, and Medical Radio-Frequency Equipment – Electromagnetic Disturbance Characteristics; 2007. 23. NF EN ISO 10993-5: Biological Evaluation of Medical Devices – Part 5; 2010. 24. UNI EN 556-1: Sterilization of Medical Devices – Requirements for Medical Devices to be Designated “Sterile”; 2002

25. IEC 61010: Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Parts 1 (2001) and 2-101; 2002.   
26. CISPR 11: Industrial, Scientific, and Medical Equipment-Radio Frequency EquipmentElectromagnetic Disturbance Characteristics; 2004.   
27. CISPR 22: Information Technology Equipment-Radio Disturbance Characteristics – Limits and Methods of Measurement; 2006.   
28. CAN/CSA C22.2, No. 61010: Safety Requirements for Electrical Equipment for Measurement, Control, and Laboratory Use – Parts 1 and 2-101; 2004.   
29. EMC Directive 2004/108/EC.   
30. WEEE Directive 2002/96/EC.   
31. LVD Directive 2006/95/EC.

# L. Test Principle:

This is a nucleic acid based test using real-time PCR. T. vaginalis is detected through the use of real-time PCR to amplify and detect TV DNA. Genomic TV DNA is amplified and detected using a sequence-specific probe that is cleaved during PCR amplification, resulting in a signal that occurs when the fluorescent reporter dye is released from the quencher.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility

A panel of 8 specimens with varying concentrations of TV (ATCC 30001) was tested on twelve non-consecutive days by two different operators, at each of three sites (8 specimens $_ { \textrm { X 2 } }$ operators $_ { \textrm { X 2 } }$ runs per day x 12 days $_ { \textrm { X 3 } }$ sites $= 1 1 5 2$ measurements total). Two runs were performed per day by each of the 6 study operators (12 days x 2 runs per day $= 2 4$ runs per operator). Each site used a different GeneXpert Instrument (running the same analytical software) for the analysis. Site 1 used an Infinity-80 instrument. Sites 2 and 3 used GeneXpert Dx instruments. The specimen panel was prepared in clinical matrix (either pooled vaginal swabs or urine in their respective Cepheid transport reagent). The panel consisted of four concentration levels per matrix: a high negative sample with an analyte concentration below the limit of detection (LoD), an LoD sample with an analyte concentration of 1X LoD, a moderate positive sample with an analyte concentration of 3X LoD, and a negative sample which was comprised of matrix only. Three lots of Xpert TV Assay cartridges were used at each of the 3 testing sites, with each lot being used for 4 days of testing. One TV positive and one negative control were tested daily prior to testing the panel specimens (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). Results are presented in Table 1.

Table 1. Reproducibility of the 8 Member Specimen Panel   

<table><tr><td rowspan=2 colspan=1>Samplea</td><td rowspan=1 colspan=3>Site 1(Infinity-80)</td><td rowspan=1 colspan=3>Site 2(GeneXpert Dx)</td><td rowspan=1 colspan=3>Site 3(GeneXpert Dx)</td><td rowspan=2 colspan=1>TotalAgreementby Sample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Site</td></tr><tr><td rowspan=1 colspan=1>FS-Neg</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>FS-Mod Pos(~3X LOD;~6 cells/mL)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>FS-LoD~ 1X LOD;~2 cells/mL)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>87.5%(21/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>91.7%(44/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>95.8%(138/144)</td></tr><tr><td rowspan=1 colspan=1>FS-High Neg(below LoD;&lt; 2 cells/mL)</td><td rowspan=1 colspan=1>87.5%(21/24)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>81.3%(39/48)</td><td rowspan=1 colspan=1>66.7%(16/24)</td><td rowspan=1 colspan=1>79.2%(19/24)</td><td rowspan=1 colspan=1>72.9%35/48)</td><td rowspan=1 colspan=1>79.2%(19/24)</td><td rowspan=1 colspan=1>70.8%(17/24)</td><td rowspan=1 colspan=1>75.0%(36/48)</td><td rowspan=1 colspan=1>76.4%(110/144)</td></tr><tr><td rowspan=1 colspan=1>UR-Neg</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>UR-Mod Pos(3X LOD;~9 cells/mL)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(144/144)</td></tr><tr><td rowspan=1 colspan=1>UR-LoD(~1X LoD;3 cells/mL)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>83.3%(40/48)</td><td rowspan=1 colspan=1>83.3%(20/24)</td><td rowspan=1 colspan=1>91.3%(21/(/23))</td><td rowspan=1 colspan=1>87.2%41/47)</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>88.8%(127/143)</td></tr><tr><td rowspan=1 colspan=1>UR-High Negblow LoD;&lt; 3 cells/mL)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>75.0%(36/48)</td><td rowspan=1 colspan=1>70.8%(17/24)</td><td rowspan=1 colspan=1>54.2%(13/24)</td><td rowspan=1 colspan=1>62.5%(30/48)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>75.0%(18/24)</td><td rowspan=1 colspan=1>75.0%(36/48)</td><td rowspan=1 colspan=1>70.8%(102/144)</td></tr></table>

a. $\mathrm { F S } { = }$ female swab matrix; $\mathrm { U R } =$ urine matrix. b. One sample was indeterminate on initial testing and was retested.

The reproducibility of the Xpert TV Assay was also evaluated in terms of the fluorescent signal (expressed in Ct values) for each target detected. The mean, standard deviation (SD), and coefficient of variation (CV) between sites, between days, and between operators for each panel member are presented in Table 2.

Table 2. Reproducibility of the Fluorescent Signal   

<table><tr><td rowspan=2 colspan=1>Samplea</td><td rowspan=2 colspan=1>AssayChannel(Analyte)</td><td rowspan=2 colspan=1>Nb</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Residual</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>FS-Neg</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>67.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>FS-Mod Pos(~3X LoD;~6 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>79.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>FS-LoD(~1X LoD;~ 2 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>72.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>FS-High Neg(below LoD;&lt; 2 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>82.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>UR-Neg</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>63.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>UR-Mod Pos(~3X LoD;~9 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>22.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>67.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.9</td></tr><tr><td rowspan=1 colspan=1>UR-LoD(~1X LoD;~3 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>75.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>UR-High Negblow LoOD;&lt; 3 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>11.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>44.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.3</td></tr></table>

a. $\mathrm { F S } { = }$ female swab matrix; $\mathrm { U R } =$ urine matrix. b. Results with non-zero Ct values out of 144. c. $( \% )$ is contribution of variance component to overall CV.

The data presented in Tables 1 and 2 demonstrate good reproducibility for the Xpert TV Assay on the GeneXpert Instrument Systems. The assay shows $\ge 8 8 . 8 \%$ detection of TV in LoD samples diluted in pooled vaginal swab and urine matrix (detection in LoD samples is $9 3 . 1 \%$ if operator 1 from site 1 is excluded). Due to the relatively low LoD concentrations of TV analyte, it is believed that small variations in sample aliquots contributed to detection levels below the anticipated $9 5 \%$ .

Detection of TV was $100 \%$ for moderate positive samples diluted in pooled vaginal swab and urine matrix. Detection was $\leq 7 6 . 4 \%$ for the high negative samples, which is acceptable and typical for real-time PCR assays.

Agreement between all operators was evaluated using Fisher’s Exact test to assess operator performance. No statistically significant differences were observed for any of the specimens (including the Urine LoD sample; p-value $= 0 . 0 6 1$ ). Therefore, the operator performance was deemed acceptable. Detection was $\leq 7 6 . 4 \%$ for the high negative samples, which is acceptable and typical for real-time PCR assays.

# Instrument Precision

An in-house precision study was conducted to compare the performance of the GeneXpert Dx and the GeneXpert Infinity Instrument Systems. A panel of 8 specimens with varying concentrations of TV was tested on 12 different days by two operators. Each operator conducted four runs of the specimen panel per day on three different instruments (8 specimens x 12 days $_ { \textrm { X 2 } }$ operators $\mathtt { x 4 }$ runs per day x 3 instrument systems $= 2 3 0 4$ measurements total). The three instruments included in this study were the GeneXpert Dx, the Infinity-48, and the Infinity-80 (running the same analytical software). Three lots of Xpert TV Assay cartridges were used for the study, each lot used for 4 days of testing. One TV positive and one negative control were tested daily prior to testing the panel specimens (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). Results are summarized in Table 3.

Table 3. In-House Precision for the 8 Member Specimen Panel   

<table><tr><td rowspan=2 colspan=1>Samplea</td><td rowspan=1 colspan=3>GeneXpert Dx</td><td rowspan=1 colspan=3>Infinity-48</td><td rowspan=1 colspan=3>Infinity-80</td><td rowspan=2 colspan=1>% TotalAgreementbby Sample</td></tr><tr><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Inst</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Inst</td><td rowspan=1 colspan=1>Op 1</td><td rowspan=1 colspan=1>Op 2</td><td rowspan=1 colspan=1>Inst</td></tr><tr><td rowspan=1 colspan=1>FS-Neg</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>99.0%(95/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>99.7%(287/288)</td></tr><tr><td rowspan=1 colspan=1>FS-Mod Pos(~3X LoD;~6 cells/mL)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(288/288)</td></tr><tr><td rowspan=1 colspan=1>FS-LoD(~1X LoD;~ 2 cells/mL)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>87.5%(42/48)</td><td rowspan=1 colspan=1>90.6%(87/96)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>89.6%(43/48)</td><td rowspan=1 colspan=1>91.7%(88/96)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>89.6%(43/48)</td><td rowspan=1 colspan=1>92.7%(89/96)</td><td rowspan=1 colspan=1>91.7%(264/288)</td></tr><tr><td rowspan=1 colspan=1>FS-High Negbelow LoOD;&lt; 2 cells/mL)</td><td rowspan=1 colspan=1>74.5%(35/47)</td><td rowspan=1 colspan=1>75.0%(36/48)</td><td rowspan=1 colspan=1>74.7%(71/95)</td><td rowspan=1 colspan=1>77.1%(37/48)</td><td rowspan=1 colspan=1>75.0%(36/48)</td><td rowspan=1 colspan=1>76.0%(73/96)</td><td rowspan=1 colspan=1>83.3%(40/48)</td><td rowspan=1 colspan=1>68.8%(33/48)</td><td rowspan=1 colspan=1>76.0%(73/96)</td><td rowspan=1 colspan=1>75.6%(217/287)b</td></tr><tr><td rowspan=1 colspan=1>UR-Neg</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(47/47)</td><td rowspan=1 colspan=1>100%(95/95)</td><td rowspan=1 colspan=1>100%(287/287)b</td></tr><tr><td rowspan=1 colspan=1>UR-Mod Pos(3X LOD;9 cells/mL)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>100%(96/96)</td><td rowspan=1 colspan=1>100%(288/288)</td></tr><tr><td rowspan=1 colspan=1>UR-LoD(~1X LOD;~3 cells/mL)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=1>93.8%(90/96)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>89.6%(43/48)</td><td rowspan=1 colspan=1>92.7%(89/96)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>95.8%(92/96)</td><td rowspan=1 colspan=1>94.1%(271/288)</td></tr><tr><td rowspan=1 colspan=1>UR-High Negbelow LoD&lt; 3 cels/mL)</td><td rowspan=1 colspan=1>72.9%(35/48)</td><td rowspan=1 colspan=1>77.1%(37/48)</td><td rowspan=1 colspan=1>75.0%(72/96)</td><td rowspan=1 colspan=1>70.8%(34/48)</td><td rowspan=1 colspan=1>79.2%(38/48)</td><td rowspan=1 colspan=1>75.0%(72/96)</td><td rowspan=1 colspan=1>81.3%(39/48)</td><td rowspan=1 colspan=1>85.4%(41/48)</td><td rowspan=1 colspan=1>83.3%(80/96)</td><td rowspan=1 colspan=1>77.8%(224/288)</td></tr></table>

a. FS $=$ female swab matrix; $\mathrm { U R } =$ urine matrix. b. One FS-Low Pos and one UR-Neg sample indeterminate and not retested.

The precision of the Xpert TV Assay was also evaluated in terms of the fluorescent signal (expressed in Ct values) for each target detected. The mean, standard deviation (SD), and coefficient of variation (CV) between-instruments, between-lots, between-days, betweenoperators, and residual variability for each panel member are presented in Table 4.

Table 4. Precision of the Fluorescent Signal   

<table><tr><td rowspan=2 colspan=1>Samplea</td><td rowspan=2 colspan=1>AssayChannel(Analyte)</td><td rowspan=2 colspan=1>Nb</td><td rowspan=2 colspan=1>MeanCt</td><td rowspan=1 colspan=2>Between-Instrument</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Residual</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)c</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)c</td></tr><tr><td rowspan=1 colspan=1>FS-Neg</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>53.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>44.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1>FS-Mod Pos(~3X LoD;~6 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>73.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>FS-LoD(~1X LoD;~ 2 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>264</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.4</td></tr><tr><td rowspan=1 colspan=1>FS-High Neg below LoD;&lt; 2 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>217</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>UR-Neg</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>47.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>49.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>UR-Mod Pos(~3X LD;;~9 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>11.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>58.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>UR-LoD(~1X LoD;~3 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>271</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>66.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>UR-High Negbelow LoD;&lt; 3 cells/mL)</td><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>224</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>90.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.3</td></tr></table>

a. FS=female swab matrix; UR $\underline { { \underline { { \mathbf { \Pi } } } } }$ urine matrix. b. Results with non-zero Ct values out of 288. c. $( \% )$ is contribution of variance component to overall CV.

The data presented in Tables 3 and 4 demonstrated acceptable precision between instruments as it shows $100 \%$ detection for TV diluted in swab or urine matrix at the moderate positive level on all three instruments. Detection of samples at the assay LoD was $\geq 9 1 . 7 \%$ for both matrices. Due to the relatively low LoD concentrations of TV analyte, it is believed that small variations in sample aliquots contributed to detection levels below the anticipated $9 5 \%$ .

b. Linearity/assay reportable range: N/A c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# External Controls

The external positive control (ZeptoMetrix; Catalog # NATTVPOS-6MC) for the Xpert TV Assay consists of NATtrol inactivated and heat-treated cells of TV strain Z070 and human A549 cells. The external negative control (ZeptoMetrix; Catalog # NATTVNEG-6MC) consists of NATtrol inactivated cells of Neisseria gonorrhoeae strain Z017 and human A549 cells. Three lots of these controls were tested in replicates of six (positive external control) or four (negative external control). Of the 33 runs, all 18 runs with positive controls were reported as “TV DETECTED” by the Xpert TV Assay. Of the 12 runs with negative controls, 11 were reported as “TV NOT DETECTED” and one provided an indeterminate GeneXpert result (ERROR). To verify that N. gonorrhoeae cells were present in the three negative control lots tested, a single sample from each of the lots was taken tested in triplicate using the Xpert CT/NG Assay. All three runs were reported as “CT NOT DETECTED, NG DETECTED.” These results indicate that the external controls gave the expected results using the Xpert TV Assay.

# Internal Controls

The Xpert TV Assay includes the following internal controls: Sample Adequacy Control (SAC), Sample Processing Control (SPC), and the Probe Check Control (PCC).

1. Sample Adequacy Control (SAC): The SAC verifies that human DNA is present in the sample, indicating that sample collection was adequate. Test results are reported as “INVALID” if the SAC fails to meet the valid minimum or maximum Ct specification (range $9 { - } 4 5 \ \mathrm { C t }$ ). Examination of all SAC values obtained from samples tested in the clinical study indicated that, out of 5391 total runs for eligible specimens, two samples $( 0 . 0 3 7 \% )$ received a GeneXpert result of “INVALID.” Both were due to SAC failure. In addition, examination of the SAC Ct values of the 4744 samples that were negative relative to the patient infected status, indicated that none of the Ct values obtained from clinical samples with valid Xpert TV Assay results approached the SAC maximum cycle cut-off of Ct 45. This data indicates that the Ct range for the SAC is appropriate and that the SAC functions as an adequate control for sample adequacy.

2. Sample Processing Control (SPC): The SPC bead contains genomic DNA of Bacillus globigii that is included in each cartridge to verify adequate processing of the sample. The SPC is processed with the sample and verifies the effectiveness of each sample preparation step including; binding and elution of DNA, correct reaction tube filling, PCR components are present and functioning, and monitors the presence of potential inhibitor(s) in the real-time PCR assay. Test results are reported as “INVALID” if the SPC fails to meet the valid minimum or maximum Ct specification (range $9 { - } 4 5 \ \mathrm { C t }$ ). From 5391 eligible samples tested during the clinical study, zero “INVALID” GeneXpert results were due to an SPC failure alone and zero “INVALID” GeneXpert results were caused by failure of both the SAC and SPC. In addition, examination of the SPC Ct values of the 4744 samples that were negative relative to the patient infected status, indicated that none of the Ct values obtained from clinical samples with valid Xpert TV Assay results approached the SPC maximum cycle cut-off of Ct 45. This data indicates that the Ct range for the SPC is appropriate and that the SPC functions as an adequate control to monitor nucleic acid extraction and amplification.

3. Probe Check Control (PCC): After sample preparation, bead reconstitution, and reaction tube filling, but prior to thermal cycling, the GeneXpert Instrument System is programmed to perform a probe check on the amplification mixture. The PCC verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for the TV target or the SAC and SPC controls, a probe check error is reported and the test will not continue. If a probe check error is reported, the test may be repeated using a new sample, a new cartridge, and new reagents. During clinical testing, one test result was reported as “ERROR” due to a PCC failure out of 5391 total runs for eligible specimens $\left( 0 . 0 2 \% \right)$ .

# Shipping and Storage Stability

The conditions claimed for specimen transport and storage were evaluated for first-catch urine, endocervical swabs, and patient-collected vaginal swabs (collected in a clinical setting). Cultured and titered TV cells (strain ATCC 30001) were spiked into pooled negative urine or swab specimens (in their respective Cepheid transport reagent) at a concentration of 3X LoD. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GenXpert Dx (GX-IV and GX-XVI) Instrument. For the TV positive samples, replicates of 16 were tested at $\mathrm { \Delta t } = 0$ and replicates of 8 were tested at all other time and temperature combinations. For the TV negative samples (matrix only), replicates of 8 were tested at $\mathrm { \Delta t } = 0$ and replicates of 4 were tested at all other time and temperature combinations.

Pooled negative urine in Cepheid transport reagent was stored at $2 \mathrm { ^ \circ C }$ and $8 \mathrm { { } ^ { \circ } C }$ and tested at selected time points up to 61 days $\mathbf { \rho } _ { \mathbf { t } } = 0$ , 7, 14, 28, 42, and 61 days), or stored at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 \mathrm { { } ^ { \circ } C }$ and tested at selected time points up to 28 days $\mathrm { \Delta t } = \mathrm { \Delta 0 }$ , 7, 14, 21, and 28 days). Pooled unpreserved urine was stored at $2 \mathrm { ^ { \circ } C }$ and $8 ^ { \circ } \mathrm { C }$ , and tested at selected time points up to 7 days $( \mathrm { t } = 0 , 1 , 4$ , and 7 days), or stored at $1 5 \mathrm { { ^ \circ C } }$ and $3 0 \mathrm { { } ^ { \circ } C }$ , and tested at selected time points up to 7 hours ( $\mathrm { t } = 0 , 2 , 4$ , and 7 hours). Pooled vaginal or endocervical swabs stored in Cepheid transport reagent were stored at $2 \mathrm { ^ { \circ } C }$ and $3 0 ^ { \circ } \mathrm { C }$ and tested at selected time points up to 61 days $\mathrm { t } = 0$ , 14, 28, and 61 days).

The results of this study support the following claims for specimen shipping and storage:

Female urine in Xpert Urine Transport Reagent refrigerated at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 28 days or at 15- $3 0 \mathrm { { } ^ { \circ } C }$ (room temperature) for 14 days.   
Vaginal and endocervical swabs in Xpert Swab Transport Reagent at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ for 60 days.   
Female urine neat (no preservative) refrigerated at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 4 days or at $1 5 { - } 3 0 ^ { \circ } \mathrm { C }$ for 4 hours.

# Specimen Stability – Cartridge Hold Time Study

If an Xpert cartridge is loaded onto a fully loaded GeneXpert Infinity Instrument, samples may wait up to two hours before a GenXpert module becomes available. During this wait time, the assay target organisms and human cells may degrade or become unstable, resulting in a low positive (“NEGATIVE”) or invalid (SAC Ct not within the valid range) result.

The cartridge hold time study was conducted to determine the acceptable maximum hold time for Xpert TV Assay cartridges held on the GeneXpert Infinity System. Cultured and titered TV cells (strain ATCC 30001) and human A549 cells were spiked into pooled negative urine or vaginal swab specimens (in their respective Cepheid transport reagent) at a concentration of 3X LoD and as a 10X high positive cell mix, respectively. Samples loaded into Xpert TV cartridges were

tested after incubation for 1, 4, or 5 hours at ambient conditions (room temperature and ambient humidity), high humidity ( $2 5 ^ { \circ } \mathrm { C }$ and $7 5 \%$ relative humidity), and at high temperature $3 5 ^ { \circ } \mathrm { C }$ and ambient humidity). Replicates of 8 positive and 8 negative samples (human A549 cells and matrix only) were tested at each of the time and temperature combinations. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells, respectively). The data for this study was collected with the GenXpert Dx (GX-IV and GX-XVI) Instruments. Xpert TV Assay cartridges were held at three storage conditions (room temperature, $2 5 ^ { \circ } \mathrm { C } / 7 5 \%$ relative humidity, and $3 5 ^ { \circ } \mathrm { C }$ ) and tested at $\mathbf { t } = 0 , 1 , 4$ , and 5 hours. Positive and negative samples were tested in replicates of eight.

The results of this study support the following claims for on-board specimen storage:

Urine and swabs in Cepheid transport reagent loaded into Xpert TV cartridges can be stored on-board the GeneXpert Infinity for up to 4 hours prior to testing with the Xpert TV Assay.

d. Detection limit:

# Limit of Detection (LoD)

LoD was established using two TV strains, one metronidazole sensitive (ATCC 30001) and one metronidazole resistant (ATCC 30238) diluted into negative clinical matrix (either pooled vaginal swabs or urine in their respective Cepheid transport reagent). The LoD is defined as the lowest concentration (expressed as number of cells per milliliter; cells/ml) per sample that can be reproducibly distinguished from negative samples with $9 5 \%$ confidence or the lowest concentration at which 19 of 20 replicates were positive. Each strain was initially tested in a range finding study at 5 different concentrations in replicates of 20 per concentration of TV. Testing was performed with two lots of reagents across three testing days. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GenXpert Dx (GX-IV and GX-XVI) and the GeneXpert Infinity-80 Instruments. The results of the LoD range finding study are shown in Tables 5 to 8 below.

Table 5. Concentrations of TV ATCC 30001 Tested in Vaginal Swabs   

<table><tr><td rowspan=2 colspan=1>Reagent Lot</td><td rowspan=2 colspan=1>Cells/mL</td><td rowspan=2 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=3>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>SACa</td><td rowspan=1 colspan=1>SPCb</td></tr><tr><td rowspan=7 colspan=1>Reagent Lot1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>40.2</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12c</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>EY</td></tr><tr><td rowspan=6 colspan=1>Reagent Lot2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>22.2</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>34.0</td></tr></table>

a. SAC (Sample Adequacy Control). b. SPC (Sample Processing Control). c. 1 of 20 replicates was reported as "ERROR".The run was repeated resulting in a valid positive TV result.

Table 6. Concentrations of TV ATCC 30001 Tested in Vaginal Swabs   

<table><tr><td rowspan=2 colspan=1>Reagent Lot</td><td rowspan=2 colspan=1>Cells/mL</td><td rowspan=2 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=3>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>SAC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=7 colspan=1>Reagent Lot1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>21.6</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>21.6</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=6 colspan=1>Reagent Lot2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>13a</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>22.2</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>34.0</td></tr></table>

a of 20 replicates was reported as "ERROR".The run was repeated resulting in a valid positive TV result.

Table 7. Concentrations of TV ATCC 30001 Tested in Urine   

<table><tr><td rowspan=2 colspan=1>Reagent Lot</td><td rowspan=2 colspan=1>Cells/mL</td><td rowspan=2 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=3>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>SAC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=7 colspan=1>Reagent Lot1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>$$</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>39.2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=6 colspan=1>Reagent Lot2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr></table>

a.of 20 replicates were reported as "ERROR".The runs were repeated resulting in valid positive TV results.

Table 8. Concentrations of TV ATCC 30001 Tested in Vaginal Swabs   

<table><tr><td rowspan=2 colspan=1>Reagent Lot</td><td rowspan=2 colspan=1>Cells/mL</td><td rowspan=2 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=3>Mean Ct</td></tr><tr><td rowspan=1 colspan=1>TV</td><td rowspan=1 colspan=1>SAC</td><td rowspan=1 colspan=1>SPC</td></tr><tr><td rowspan=7 colspan=1>Reagent Lot1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>39.2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=6 colspan=1>Reagent Lot2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>33.9</td></tr></table>

For each TV strain, the LoD estimate selected for verification was the highest value determined from the two lots. Verification of the estimated LoD claim was performed on one reagent lot across three testing days using 20 replicates per strain. The LoD was determined empirically as the lowest concentration that had 20/20 positive results. In addition to the empirical determination of LoD, Cepheid also conducted Probit analyses and the results were identical. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in

Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GenXpert Dx (GX-IV and GX-XVI) and the GeneXpert Infinity-80 Instruments. The LoD point values for each strain tested are summarized in Table 9 below.

Table.9 LoD of Two TV Strains in Either Pooled Vaginal Swab or Urine Matrix   

<table><tr><td rowspan=2 colspan=1>Trichomonasvaginalisstrain and matrix</td><td rowspan=1 colspan=2>LoD Estimates byProbit Analysis(cells/mL)</td><td rowspan=2 colspan=1>VerifiedLoD(cells/mL)</td><td rowspan=2 colspan=1>Verification(Positives/20)</td><td rowspan=2 colspan=1>Mean TVCt</td><td rowspan=2 colspan=1>Mean SACCt</td><td rowspan=2 colspan=1>Mean SPCCt</td><td rowspan=2 colspan=1>LoDClaim(cells/mL)</td></tr><tr><td rowspan=1 colspan=1>ReagentLot 1</td><td rowspan=1 colspan=1>ReagentLot 2</td></tr><tr><td rowspan=1 colspan=1>ATCC 30001 inVaginal Swab</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>ATCC 30238 inVaginal Swab</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>ATCC 30001 inUrine</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>ATCC 30238 inUrine</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>2</td></tr></table>

The results of the LoD verification study indicate that the LoD for TV strains ATCC 30001 and ATCC 30238 in vaginal swab matrix is $2 \mathrm { c e l l s } / \mathrm { m L }$ and that the LoD for TV strain ATCC 30001 in urine matrix is 3 cells/mL. The claimed LoD for TV strain ATCC 30238 in urine matrix is 2 cells/mL.

# e. Analytical specificity:

# Inclusivity

An inclusivity study was conducted to test reactivity of the Xpert TV Assay with 17 strains of TV, tested at a concentration of 3X LoD. The TV isolates were obtained from ATCC. TV cells were diluted into negative clinical matrix (either pooled vaginal swabs or urine in their respective Cepheid transport reagent). One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GenXpert Infinity-80 Instrument. Results are summarized in Table 10.

Table 10. TV Strain Inclusivity for the Xpert TV Assay   

<table><tr><td colspan="1" rowspan="1">Isolate ATCC #</td><td colspan="1" rowspan="1">Isolation Source</td><td colspan="1" rowspan="1">Results VaginalSwab</td><td colspan="1" rowspan="1">Results Urine</td></tr><tr><td colspan="1" rowspan="1">30001</td><td colspan="1" rowspan="1">Vaginal exudate</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30184</td><td colspan="1" rowspan="1">Vaginal swab</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30187</td><td colspan="1" rowspan="1">Endocervical swab</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30188</td><td colspan="1" rowspan="1">Vagina</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30236</td><td colspan="1" rowspan="1">Endocervical swab</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30240</td><td colspan="1" rowspan="1">Vaginal pool</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30245</td><td colspan="1" rowspan="1">Vaginal andEndocervical material</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">30247</td><td colspan="1" rowspan="1">Vagina</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50138</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50139</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50141</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50143</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50147</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50167</td><td colspan="1" rowspan="1">Vagina</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">50183</td><td colspan="1" rowspan="1">Prostatic fluid</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">PRA-95</td><td colspan="1" rowspan="1">Vaginal exudate</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">PRA-98</td><td colspan="1" rowspan="1">Human</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr></table>

All $1 7 \mathrm { T V }$ strains tested above were detected by the Xpert TV Assay.

# Microbial Interference (Cross-Reactivity and Competitive Interference)

This study was conducted to evaluate interference in the Xpert TV Assay when non-target organisms are present at high concentrations. A panel of 124 microorganisms (5 viruses, 6 yeasts, 6 protozoans, and 107 bacteria) representing pathogens or flora commonly present in the urogenital system were evaluated. Each bacterial or fungal strain was tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ or greater or at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ genomes/mL. Viral strains were tested at $1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { U } / \mathrm { m L }$ or $1 0 ^ { 5 }$ genomes $/ \mathrm { m L }$ or greater. Protozoans were cultured in growth media, visually enumerated by light microscopy and tested at $1 \mathrm { \ x \ 1 0 ^ { 5 } c e l l s / m L }$ or greater or $1 0 ^ { 5 }$ genomes/mL. All microorganisms were tested in triplicate in the presence (competitive interference) and absence (cross-reactivity) of TV (ATCC 3001) at 3X LoD. All microorganisms tested were diluted into negative clinical matrix (either pooled vaginal swabs or urine in their respective Cepheid transport reagent).

One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GenXpert Dx (GX-IV) Instrument.

One organism, Trichomonas tenax, demonstrated cross-reactivity (result of TV DETECTED in the absence of TV) at $\mathrm { 1 ~ x ~ 1 0 ^ { 5 } c e l l s / m L }$ for the urine and vaginal swab matrix samples. This microorganism was subjected to repeat analysis at various other concentrations until a result of TV NOT DETECTED was obtained (at $\mathrm { 1 ~ x ~ i 0 ^ { 2 } c e l l s / m L } )$ . For the other 123 microorganisms, all TV positive samples remained positive and all TV negative samples remained negative, indicating that there was no interference or cross-reactivity with the results of the Xpert TV Assay for these microbes. Results are summarized in Tables 11 and 12 for urine and vaginal swab matrix, respectively.

Table 11. Microbial Interference in Pooled Urine   

<table><tr><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">ConcentrationTesteda</td><td colspan="2" rowspan="1">Xpert TV Assay Result</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveInterference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xerosis</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Actinomyces israelib</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Actinomyces pyogenes</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">5 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Atopobium vaginaeb</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilisb</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ureolyticus</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bifidobacteriumadolescentisb</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium bevi beve</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">1 x 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Branhamella catarrhalis</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium linens</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida albicanse</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosi</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalise</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Chromobacterium violaceum</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficileb</td><td colspan="1" rowspan="1">4x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringensb</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformanse</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">1 x 105d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirusf</td><td colspan="1" rowspan="1">5x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Deinococcus radiodurans</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Derxia gummosa</td><td colspan="1" rowspan="1">1 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">1 x 10</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Flavobacterium meningosepticum</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatumb</td><td colspan="1" rowspan="1">4x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">6 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">1 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">1  10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus If</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus If</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">HIV-1f</td><td colspan="1" rowspan="1">2 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Human pilloma virus 16</td><td colspan="1" rowspan="1">6 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Kingella dentrificans</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus jensonii</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus lactis</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">Competitive nterference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus vaginalis</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Leuconostoc paramensenteroides</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisib</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Moraxella lacunata</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium smegmatis</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria cinerea</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria dentrificans</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria flava</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria flavescens</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria perflava</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria polysaccharea</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria subflava</td><td colspan="1" rowspan="1">9 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Paracoccus denitrificans</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pentatrichomonis hominis</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Peptostreptooccus anus</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Prevotellla biviadb</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveI nerference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Rahnella aquatilis</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Rhodospirillum rubrum</td><td colspan="1" rowspan="1">1 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Salmonella minnesota</td><td colspan="1" rowspan="1">1x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">2 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguis</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptomyces griseinus</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1 x 103</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1x 10^{2</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma parvum</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr></table>

a. Tests ru $\mathrm { 1 } \geq \mathrm { 1 0 ^ { 6 } C F U / m L }$ for bacteria and fungi, $\overline { { 2 1 0 ^ { 6 } } }$ genomes/mL for yeast, $\overline { { \geq 1 0 ^ { 5 } \mathrm { U } / \mathrm { m L } } }$ or ${ \geq } 1 0 ^ { 5 }$ genomes/mL for viruses and ${ \geq } 1 0 ^ { 5 }$ cells/mL for protozoans.

b. Anaerobic organism   
c. Protozoan   
d. Genome equivalents tested (DNA)   
e. Fungal organism   
f. Virus

Table 12. Microbial Interference in Vaginal Swab Matrix   

<table><tr><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">ConcentrationTesteda</td><td colspan="2" rowspan="1">Xpert TV Assay Result</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveIn erference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xerosis</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffii</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Actinomyces israelib</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Actinomyces pyogenes</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Atopobium vaginae</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilisb</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ureolyticu</td><td colspan="1" rowspan="1">1x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bifidobacteriumadolescentisb</td><td colspan="1" rowspan="1">6x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Bifibacterium brevi ee</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">1 x 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Branhamella catarrhalis</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium linens</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida albicanse</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosi</td><td colspan="1" rowspan="1">3x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalise</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Chromobacterium violaceum</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficileb</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveInterference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neformanse</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">1 x 10</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">5x 10}$</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Deinococcus radiodurans</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Derxia gummosa</td><td colspan="1" rowspan="1">1 x 106 d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">1 x 10</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Flavobacteriummeningosepticum</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatu</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Giardia intestinalis</td><td colspan="1" rowspan="1">1 x 10</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">1 x 10d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus If</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus If</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">HIV-1f</td><td colspan="1" rowspan="1">2 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Human papilloma virus 16</td><td colspan="1" rowspan="1">6 x 105</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Kingella dentrificans</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">9 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">Competitivenerference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus jensonii</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus lactis</td><td colspan="1" rowspan="1">1 x 107</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus vaginalis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Leuconostocparamensenteroides</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisib</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Moraxella lacunata</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium smegmatis</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria cinerea</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria dentrificans</td><td colspan="1" rowspan="1">7 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria flava</td><td colspan="1" rowspan="1">9 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria flavescens</td><td colspan="1" rowspan="1">8 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">9x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria perflava</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria polysaccharea</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria sicca</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Neisseria subflava</td><td colspan="1" rowspan="1">9 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pantoea agglomerans</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Paracoccus denitrificans</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pentarichomonis hominis</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pepto</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">ConcentrationTested</td><td colspan="2" rowspan="1">Xpert TV Assay Result</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveIerference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococus productuss</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Prevotella biviab</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Propionibacterium acnesb</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">7x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Providencia stuartii</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">4x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Rahnella aquatilis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Rhodospirillum rubrum</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Salmonella minnesota</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">9 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">6 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus bovis</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans</td><td colspan="1" rowspan="1">5x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptococcus sanguis</td><td colspan="1" rowspan="1">2 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Streptomyces griseinus</td><td colspan="1" rowspan="1">4 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1 x 105</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1 x 103</td><td colspan="1" rowspan="1">TV DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Trichomonas tenax</td><td colspan="1" rowspan="1">1 x 10{2</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma parvum</td><td colspan="1" rowspan="1">1 x 106d</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="2">Microorganism</td><td colspan="1" rowspan="2">ConcentrationTesteda</td><td colspan="2" rowspan="1">Xpert TV Assay Result</td></tr><tr><td colspan="1" rowspan="1">Cross-Reactivity(- T. vaginalis)</td><td colspan="1" rowspan="1">CompetitiveInterference(+ T. vaginalis)</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">1 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">3 x 106</td><td colspan="1" rowspan="1">TV NOT DETECTED</td><td colspan="1" rowspan="1">TV DETECTED</td></tr></table>

a. Tests $\mathrm { \Delta r u n } \geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacteria and fungi, $\overline { { 2 1 0 ^ { 6 } } }$ genomes/mL for yeast, $\overline { { \geq 1 0 ^ { 5 } \mathrm { { U } / \mathrm { { m L } } } } }$ or $\overline { { 2 1 0 ^ { 5 } } }$ genomes $/ \mathrm { m L }$ for viruses and $\geq 1 0 ^ { 5 } \mathrm { c e l l s / m L }$ for protozoans. b. Anaerobic organism c. Protozoan d. Genome equivalents tested (DNA) e. Fungal organism f. Virus

Additional three microorganisms, Dientamoeba fragilis, Agrobacterium radiobacter, and Erwinia herbicola, were not available for direct testing. An in silico analysis was conducted using the Basic Local Alignment Search Tool (BLAST) to compare the Xpert TV Assay primer and probe sequences with all available sequences associated with these three microorganisms in the GenBank database. Available sequence data for $D .$ . fragilis was examined and showed a maximum of $7 \%$ homology to the Xpert TV primer and probe sequences. Available sequence data for $A$ . radiobacter was examined and showed a maximum of $3 8 \%$ homology to the Xpert TV primer and probe sequences. Available sequence data for $E$ . herbicola was examined and showed a maximum of $10 \%$ homology to the Xpert TV primer and probe sequences. Results are shown in Table 13.

Table 13. In silico Analysis for Microbial Interference   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Accession Number</td><td rowspan=1 colspan=1>% Homology</td></tr><tr><td rowspan=1 colspan=1>Dientamoeba fragilis</td><td rowspan=1 colspan=1>KC967121.1</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>Agrobacterium radiobacter</td><td rowspan=1 colspan=1>CP000629.1</td><td rowspan=1 colspan=1>38%</td></tr><tr><td rowspan=1 colspan=1>Erwinia herbicola</td><td rowspan=1 colspan=1>NG_035384.1</td><td rowspan=1 colspan=1>10%</td></tr></table>

# Potentially Interfering Substances

An interfering substances study was carried out to examine if substances or conditions that may be present in urine or vaginal swab specimens could affect the performance of the Xpert TV Assay. The panel consisted of endogenous substances such as blood, mucin, and leukocytes, and medications (prescription and over-the-counter) that could be used to treat urogenital conditions. Eight replicates were tested for each interfering substance. All substances were tested in the presence (interference) and absence (cross-reactivity) of TV (ATCC 3001) at 3X LoD.

Substances were diluted in negative clinical matrix (either pooled vaginal swabs or urine in their respective Cepheid transport reagent) as appropriate. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). The data for this study was collected with the GeneXpert Dx (GX-IV and GX-XVI) Instrument.

One substance, blood at $> 6 0 \%$ v/v demonstrated interference (result of TV NOT DETECTED in the presence of TV) in the vaginal swab matrix samples. This substance (blood) was subjected to repeat analysis at various other concentrations until a result of TV DETECTED was obtained $( 5 0 \% \mathrm { v / v } )$ . For the other conditions and substances tested, all TV positive samples remained positive and all TV negative samples remained negative, indicating that there was no interference or cross-reactivity with the results of the Xpert TV Assay for these microbes. Results are summarized in Tables 14 and 15 for urine and vaginal swab matrix, respectively.

Table 14. Potentially Interfering Substances in Urine Samples   

<table><tr><td rowspan=1 colspan=1>Class/Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>0.3% v/v, 1% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucus</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>0.8% w/v</td></tr><tr><td rowspan=4 colspan=1>Analgesics &amp;Antibiotics</td><td rowspan=1 colspan=1>Acetylsalicylic Acid 500mg</td><td rowspan=1 colspan=1>40 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>3.2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Azithromycin</td><td rowspan=1 colspan=1>1.8 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Doxycycline</td><td rowspan=1 colspan=1>3.6 mg/mL</td></tr><tr><td rowspan=2 colspan=1>C Deodorant &amp; Powders</td><td rowspan=1 colspan=1>PEG-20; PEG-32;PEG-20 Stearate</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Nanoxynol-9</td><td rowspan=1 colspan=1>0.25% w/v</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>BSA</td><td rowspan=1 colspan=1>10 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Glucose</td><td rowspan=1 colspan=1>Glucose</td><td rowspan=1 colspan=1>10 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>1 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Acidic Urine (pH 4.0)</td><td rowspan=1 colspan=1>Urine + N-Acetyl-L-Cysteine</td><td rowspan=1 colspan=1>pH 4.0</td></tr><tr><td rowspan=1 colspan=1>Alkaline Urine (pH 9.0)</td><td rowspan=1 colspan=1>Urine + Ammonium Citrate</td><td rowspan=1 colspan=1>pH 9.0</td></tr><tr><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>105 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Intravaginal Hormones</td><td rowspan=1 colspan=1>Progesterone; Estradiol</td><td rowspan=1 colspan=1>7 mg/mL Progesterone +0.07 mg/mL Beta Estradiol</td></tr></table>

Table 15. Potentially Interfering Substances in Swab Samples   

<table><tr><td colspan="1" rowspan="1">Class/Substance</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Blooda</td><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">10%, 50%, 60% v/v</td></tr><tr><td colspan="1" rowspan="1">Seminal Fluid</td><td colspan="1" rowspan="1">Seminal Fluid</td><td colspan="1" rowspan="1">5.0% v/v</td></tr><tr><td colspan="1" rowspan="1">Mucus</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">0.8% w/v</td></tr><tr><td colspan="1" rowspan="8">Over the counter (OTC)Vaginal Products;Contraceptives; Vaginaltreatments</td><td colspan="1" rowspan="1">Benzocaine 5%;Resorcinol 2%</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole 2%</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Miconazole Nitrate 2%</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Tioconazole</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">5% w/w Aciclovir</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Glycerin, Propylene glycol</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Glycerin; Carbomer</td><td colspan="1" rowspan="1">0.12% w/v</td></tr><tr><td colspan="1" rowspan="1">Glycerin, Hydroxyethyl cellulose</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Goldenseal 3X HPUS; Kreosotum12X HPUS</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Povidone-iodine 10%</td><td colspan="1" rowspan="1">0.25% v/v</td></tr><tr><td colspan="1" rowspan="1">Nonoxynol-9 12.5%</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Hemorrhoidal Cream</td><td colspan="1" rowspan="1">Glycerin 14%;Pramoxine HCl 1%</td><td colspan="1" rowspan="1">0.25% w/v</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">105 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Intravaginal Hormones</td><td colspan="1" rowspan="1">Progesterone; Estradiol</td><td colspan="1" rowspan="1">7 mg/mL Progesterone +0.07 mg/mL Beta Estradiol</td></tr></table>

a. In tests with substances diluted into pooled T. vaginalis-positive swab matrix, assay interference was observed in tests with blood at $60 \%$ v/v. No assay interference was observed in tests with blood at $50 \%$ v/v. This is addressed in the labeling Section 14, Limitations.

# Carry-Over

The purpose of the carry-over study was to uncover the presence of contamination in negative specimens due to carry-over of TV during nucleic acid extraction and amplification in the GeneXpert cartridge. The study was conducted over six days (not necessarily consecutive) on two modules using the same GeneXpert Dx Instrument (GX-IV). Twenty high positive samples were tested per module and 21 negative samples were tested per module. Two modules were used in the study for a total of 40 positive samples and 42 negative samples tested in the study. Negative samples consisted of pooled vaginal swab specimens in Cepheid Swab Transport Reagent and high positive samples consisted of1 $\mathrm { ~ X ~ } 1 \bar { 0 } ^ { 6 }$ cells/mL of TV ATCC 3001 in vaginal swab in Cepheid Swab Transport Reagent. High positive samples were alternated with negative samples. One TV positive and one negative control were tested daily (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively). All 42 replicates of negative sample were reported as “TV NOT DETECTED,” therefore, no evidence of carryover was observed.

# f. Assay cut-off:

Pre-clinical testing data generated with 309 prospectively collected clinical female samples (collected from 105 subjects, providing urine, endocervical swabs, and/or vaginal swab samples) was used to determine the valid minimum and maximum cycle threshold settings for TV. The results of this preliminary study indicated that the valid minimum cycle threshold setting for TV was 9 and the valid maximum cycle threshold setting for TV was 45.

These assay cut-offs were subsequently validated in the clinical study. For each specimen type, an analysis of receiver operator curves (ROCs) was conducted for the Xpert TV Assay results versus patient infected status (PIS) to confirm that the correct cut-off was chosen to optimize both sensitivity and specificity. The Youden’s Index and the Area Under the Curve (AUC) were used to make the final decision because the cut-off that maximizes the Youden’s index will give the optimal results in terms of sensitivity and specificity and the AUC gives an indication of how well the assay performs. In this study, all AUC statistics were greater than or equal to $9 8 \%$ and the Youden Index value was the highest at the chosen cut-off of $4 5 \mathrm { C t }$ .

The minimum and maximum Ct values observed for TV in the clinical study were 12 and 43.7, respectively. These values were within the valid minimum and maximum cycle threshold ranges described above. Taken together with the ROC analysis, these results indicate that the cut-off values selected for detection of TV with the Xpert Assay are appropriate.

2. Comparison studies:

# a. Method comparison with predicate device:

Method comparison was based on the results from the Cepheid Xpert TV Assay on the GeneXpert Instrument Systems compared to the patient infected status (PIS) obtained from an FDA cleared NAAT assay and TV culture for all female specimen types (endocervical swabs, patient-collected vaginal swabs, and urine). The testing description and data are listed below in the Clinical Studies section.

$b$ . Matrix comparison: N/A

3. Clinical studies:

# a. Clinical Sensitivity and Specificity

Specimen collection was conducted at 17 geographically diverse locations in the United States. Sites included (but were not limited to) OB/GYN, sexually transmitted disease (STD), and family planning clinics. Testing was conducted at 11 locations that represented the intended use sites for the Xpert TV Assay. For sites performing the Xpert TV Assay, positive and negative external controls (inactivated TV ATCC 30001 and human A549 cells in Cepheid Swab Transport Reagent, respectively) were tested with the assay on each day of testing. One testing site used a GeneXpert Infinity-80 Instrument. All other testing sites used the GeneXpert Dx Instrument (IV or XVI).

For each specimen type, results of the Xpert TV Assay were compared to a patient infected status (PIS) algorithm comprised of an FDA cleared NAAT test and culture. A specimen was called positive for PIS if either of the reference test results were positive. A specimen was called negative for PIS if both of the reference test results were negative. Specimens were collected in the following order: urine, vaginal swabs (patient-collected first then clinician-collected second), and endocervical swabs. Samples collected and tested on the Xpert TV Assay included firstcatch urine, patient-collected vaginal swabs (collected in a clinical setting), and endocervical swabs. Two clinician-collected vaginal swabs were collected and used for reference testing. Three endocervical swabs were collected (two for reference testing and one for testing with the Xpert TV Assay). Study subjects were instructed to collect first-catch urine which was used for reference testing and testing with the Xpert TV Assay. Urine was collected from study subjects who had not urinated at least one hour prior to specimen collection.

Study participants were included in the study if they could provide informed consent, were sexually active, and were $\geq 1 4$ years of age. Subjects were excluded from the study if they had been previously enrolled, received antimicrobial treatment within the previous 21 days, or had a history of hysterectomy (partial or complete removal of the cervix). The mean age of eligible study participants was 33.5 years $\mathrm { R a n g e } = 1 8$ to 78 years old). A total of 1867 eligible female subjects were enrolled in the study. Of the 5383 valid Xpert TV Assay results (1799 endocervical swabs, 1791 vaginal swabs, and 1793 urine), 85 had initial ERROR, INVALID, or NO RESULT outcomes ( $1 . 5 8 \%$ ; $9 5 \%$ CI 1.26-1.95). Upon retest, 77 of these specimens had valid results that were included in the final dataset used for the analyses and eight specimens had indeterminate results. These 8 specimens with indeterminate results were otherwise eligible specimens so were included in the calculation of indeterminate rate/assay success rate. The overall valid reporting rate for the assay was $9 9 . 9 \%$ (5383/5391). Specimens with discrepant results between the Xpert TV Assay and the PIS were analyzed by validated bi-directional Sanger sequencing and results are footnoted in Table 6 for informational purposes only.

For all three specimen types, there were no statistically significant differences between symptomatic and asymptomatic women for the performance of the Xpert TV Assay. Results from the clinical study are provided in Table 16.

Table 16. Clinical Study Results (Xpert TV Assay Results vs. PIS)   

<table><tr><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>Status</td><td rowspan=1 colspan=1>Total(n)</td><td rowspan=1 colspan=1>Sens</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Spec</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Prev(%)</td><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td></tr><tr><td rowspan=3 colspan=1>ES</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>685</td><td rowspan=1 colspan=1>100%(71/71)</td><td rowspan=1 colspan=1>94.9%-100%</td><td rowspan=1 colspan=1>98.5%(605/614)</td><td rowspan=1 colspan=1>97.2%-99.3%</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>88.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>1114</td><td rowspan=1 colspan=1>98.1%(104/106)</td><td rowspan=1 colspan=1>93.4%-99.8%</td><td rowspan=1 colspan=1>99.1%(999/1008)</td><td rowspan=1 colspan=1>98.3%-99.6%</td><td rowspan=1 colspan=1>9.5%</td><td rowspan=1 colspan=1>92.0%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1799</td><td rowspan=1 colspan=1>98.9%(175/177)a</td><td rowspan=1 colspan=1>96.0%-99.9%</td><td rowspan=1 colspan=1>98.9%(1604/1622)b</td><td rowspan=1 colspan=1>98.3%-99.3%</td><td rowspan=1 colspan=1>9.8%</td><td rowspan=1 colspan=1>90.7%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=3 colspan=1>PC-VS</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>682</td><td rowspan=1 colspan=1>98.6%(73/74)</td><td rowspan=1 colspan=1>92.7%-100%</td><td rowspan=1 colspan=1>99.5%(605/608)</td><td rowspan=1 colspan=1>98.6%-99.9%</td><td rowspan=1 colspan=1>10.9%</td><td rowspan=1 colspan=1>96.1%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>1109</td><td rowspan=1 colspan=1>95.0%(113/119)</td><td rowspan=1 colspan=1>89.3%-98.1%</td><td rowspan=1 colspan=1>99.6%(986/990)</td><td rowspan=1 colspan=1>99.0%-99.9%</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>96.6%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1791</td><td rowspan=1 colspan=1>96.4%(186/193)c</td><td rowspan=1 colspan=1>92.7%-98.5%</td><td rowspan=1 colspan=1>99.6%(1591/1598)d</td><td rowspan=1 colspan=1>99.1%-99.8%</td><td rowspan=1 colspan=1>10.8%</td><td rowspan=1 colspan=1>96.4%</td><td rowspan=1 colspan=1>99.6%</td></tr><tr><td rowspan=3 colspan=1>UR</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>688</td><td rowspan=1 colspan=1>98.6%(71/72)</td><td rowspan=1 colspan=1>92.5%-100%</td><td rowspan=1 colspan=1>99.8%(615/616)</td><td rowspan=1 colspan=1>99.1%-100%</td><td rowspan=1 colspan=1>10.5%</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>1105</td><td rowspan=1 colspan=1>98.2%(109/111)</td><td rowspan=1 colspan=1>93.6%-99.8%</td><td rowspan=1 colspan=1>99.6%(990/994)</td><td rowspan=1 colspan=1>99.0.%-99.9%</td><td rowspan=1 colspan=1>10.0%</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1793</td><td rowspan=1 colspan=1>98.4%(180/183)e</td><td rowspan=1 colspan=1>95.3%-99.7%</td><td rowspan=1 colspan=1>99.7%(1605/1610)f</td><td rowspan=1 colspan=1>99.3%-99.9%</td><td rowspan=1 colspan=1>10.2%</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>99.8%</td></tr></table>

$\mathbf { T P } { = }$ true positive, $\mathbf { F P } { = }$ false positive, TN=true negative, FN=false negative, ES endocervical swab, PC-VS=patient-collected vaginal swab, $\mathrm { U R } =$ urine

a. Testing results by sequencing: 1 of 2 FN was TV positive; 1 of 2 was TV negative.   
b. Testing results by sequencing: 8 of 18 FP were TV positive; 10 of 18 were TV negative.   
c. Testing results by sequencing: 3 of 7 FN were TV positive; 4 of 7 were TV negative.   
d. Testing results by sequencing: 5 of 7 FP were TV positive; 2 of 7 were TV negative.   
e. Testing results by sequencing: 3 of 3 FN were TV negative.   
f. Testing results by sequencing: 5 of 5 FP were TV negative.

4. Clinical cut-off: N/A

# 5. Expected values/Reference range:

During the clinical evaluation of the Xpert TV Assay, the observed prevalence of TV infection in females was $1 0 . 3 \%$ . The estimated positive predictive value (PPV) and negative predictive value (NPV) of the Xpert TV Assay across different hypothetical prevalence rates are shown for each specimen type in Table 17. These calculations are based on the overall estimated sensitivity and specificity observed for each specimen type during the Xpert TV multi-center clinical study (Table 16). The overall sensitivity and specificity for female urine (UR) were $9 8 . 4 \%$ and $9 9 . 7 \%$ , respectively. In patient-collected vaginal swab specimens (PC-VS), the overall sensitivity and specificity were $9 6 . 4 \%$ and $9 9 . 6 \%$ , respectively. For endocervical swabs (ES), the overall sensitivity and specificity were $9 8 . 9 \%$ and $9 8 . 9 \%$ , respectively.

Table 17. Hypothetical PPV and NPV of the Xpert TV Assay by Specimen Type   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Prevalence (%)</td><td rowspan=1 colspan=1>PPV (%)</td><td rowspan=1 colspan=1>NPV (%)</td></tr><tr><td rowspan=8 colspan=1>Female UR</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>76.2%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>86.6%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>94.3%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>97.2%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>99.7%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>99.6%</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>99.5%</td></tr><tr><td rowspan=8 colspan=1>PC-VS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>69.0%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>81.8%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>92.1%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>96.1%</td><td rowspan=1 colspan=1>99.6%</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>99.5%</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>98.2%</td><td rowspan=1 colspan=1>99.1%</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>98.8%</td></tr><tr><td rowspan=8 colspan=1>ES</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>47.4%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>64.5%</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>82.4%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>90.8%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>92.4%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>94.0%</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>99.7%</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>99.6%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.