# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYINSTRUMENT ONLY TEMPLATE

A. 510(k) Number: k170299   
B. Purpose for Submission: Expansion of the intended use of the device to include DNA and RNA from FFPE samples.   
C. Manufacturer and Instrument Name: Life Technologies Corporation Ion PGM Dx System   
D. Type of Test or Tests Performed: High-throughput DNA sequencing   
E. System Descriptions:

1. Device Description:

The Ion PGM Dx System is a high throughput DNA sequence analyzer for clinical use. The Ion PGM Dx System consists of the Ion OneTouch Dx Instrument, the Ion OneTouch ES Dx Instrument, the Ion OneTouch Rack Kit, the Ion PGM Dx Chip Minifuge, the Ion PGM Dx Sequencer, the Ion PGM Wireless Scanner, the DynaMag Dx Kit – Tube and Plate, the Ion Torrent Server and the Torrent Suite Dx Software. It is for use with the Ion PGM Dx Library Kit, the Ion OneTouch Dx Template Kit, the Ion PGM Dx Sequencing Kit, and the Ion $3 1 8 \mathrm { D x }$ Chip Kit. The end-user inputs extracted DNA to be sequenced and uses an established assay or provides the Analyte Specific Reagents (ASRs) to develop a sequencing assay that targets their sequence of interest.

2. Principles of Operation:

DNA from whole blood samples is isolated using a commercially available extraction method, or DNA and RNA from FFPE tissue samples are isolated and the RNA is reverse transcribed into cDNA.

The DNA and/or cDNA is made into amplicon libraries via polymerase chain reaction (PCR) to specifically amplify the regions intended for sequencing while also adding indexing sequences (barcodes) to the amplified products. Employing emulsion PCR, each sample library is templated onto Ion PGM Dx Ion Sphere Particles (ISPs) using the Ion

OneTouch Dx Instrument. Templated ISPs are enriched from non-templated ISPs and loaded onto an Ion $3 1 8 \mathrm { D x }$ Chip for sequencing on the Ion PGM Dx Sequencer. The sequencing reaction measures hydrogen ions that are generated during the incorporation of nucleotides into the nascent strand complementary to the template sequence. The resulting signal is translated into base calls which are assembled into reads, which are strings of nucleotide bases in the order found in the original library molecules. Reads are mapped to a reference sequence and mapped reads are assessed at specific nucleotide locations to identify variation from the sequence information in the reference sequence from which variant calls are determined.

3. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ____X____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes _ or No __ _X

4. Specimen Identification:

Sample barcode adapters are used in all runs to assign a unique nucleic acid barcode to each sample DNA allowing the ability to pool up to sixteen (16) barcoded sample libraries into a single sequencing run.

# 5. Specimen Sampling and Handling:

The Ion PGM Dx specimen is a library (or multiple pooled libraries) derived from genomic DNA extracted from peripheral whole blood or DNA or RNA from FFPE samples that is generated and sequenced through the following steps: nucleic acid is extracted from a sample, quantified and qualified, and used to make an indexed barcode library; the library is processed to remove remaining library preparation reagents (e.g. unused primers), normalized as necessary to ensure that each library is equally represented in the pooled samples, and templated onto beads; the templated beads are added onto a sequencing chip and sequenced.

# 6. Calibration:

There is no end-user calibration of the system. During the installation of the Ion PGM Dx System a company representative performs an Installation Qualification/ Operational Qualification/ Performance Qualification (IQ/OQ/PQ) protocol to ensure that software has been installed properly on all items (IQ), the instruments have been calibrated properly (OQ), and performance of the system is adequate (PQ). This includes an actual run on the system and uses all instruments included in the Ion PGM Dx System; the run comprises a control for the Ion OneTouch Dx Instrument, and another control for the Ion PGM Dx Sequencer. Two controls are needed in case one fails so the company representative can troubleshoot at which step the sequencing run is failing. These assessments can only be performed by a company representative and are required after major software updates. The system cannot be used by customers for any other runs until a PQ run has been successfully performed.

7. Quality Control:

A double-stranded 221 bp oligonucleotide (CF-1 control fragment) synthesized with the appropriate ends needed for amplification is spiked into every templating reaction setup. One region of the CF-1 oligonucleotide contains a unique sequence to differentiate it from library amplicons. CF-1 is used as a run quality control to ensure that templating and sequencing was performed correctly. A quality control check is run on results of CF1 sequencing and must be passed before results are released to a user.

8. Software:

FDA has reviewed applicant’s Hazard Analysis and Software Development processes for this line of product types:

Yes____X____ or No_

# F. Regulatory Information:

1. Regulation section:   
21 CFR 862.2265

2. Classification: Class II (special controls). The device is exempt from the premarket notification procedures in subpart E of part 807 of this chapter subject to 862.9.

3 Product code: PFF - High throughput DNA sequence analyzer

4. Panel: Toxicology (91)

# G. Intended Use:

1. Indication(s) for Use:

The Ion PGM Dx Instrument System is composed of a sequencing instrument that measures the hydrogen ions that are generated during the incorporation of nucleotides in the DNA sequencing reaction, and the ancillary instrumentation necessary for sample processing. This instrument system is used in conjunction with the instrument-specific Ion PGM Dx Library Kit, Ion OneTouch Dx Template Kit, Ion PGM Dx Sequencing Kit, and Ion $3 1 8 \mathrm { D x }$ Chip Kit, and data analysis software. The Ion PGM Dx Instrument System is intended for targeted sequencing of human genomic DNA (gDNA) from peripheral whole-blood samples and DNA and RNA extracted from formalin-fixed, paraffin-embedded (FFPE) samples. The Ion PGM Dx Instrument System is not intended for whole genome or de novo sequencing.

2. Special Conditions for Use Statement(s): For in vitro diagnostic use. For prescription use only.

The instrument was previously legally marketed for use with genomic DNA from whole blood samples under 21 CFR 862.2265; special conditions for use described here are specific to RNA and genomic DNA from FFPE samples only. Special conditions for use of the instrument with genomic DNA from whole blood are included in the device labeling.

Special Conditions statement for performance derived from a representative assay using RNA and genomic DNA from FFPE:

The Ion PGM Dx System has been validated to deliver the following using a representative assay:

Sequencing output $> 0 . 7$ gigabases Reads $> 3$ million Read length up to 141 base pairs

The system has been validated for the detection of single nucleotide variants, multinucleotide variants and up to 18 bp deletions.

§ The system is designed to deliver qualitative results.

As with any hybridization-based workflow, underlying polymorphisms or mutations in primer binding regions can affect the regions being sequenced and, consequently, the ability to make calls.

The minimal coverage required to call a SNV, MNV or deletion variant is $: \geq 3 4 7 \mathrm { x }$ . The minimal coverage required to call a fusion variant is $\geq 4 1 \mathrm { X }$ .

The Ion PGM Dx System can be used only with the Ion PGM Dx Library Kit, Ion OneTouch Dx Template Kit, Ion PGM Dx Sequencing Kit, and the Ion 318 Dx Chip Kit.

# H. Substantial Equivalence Information:

1. Predicate Device Name(s) and 510(k) numbers: Illumina MiSeqDx Platform, k123989 (DEN130011)

2. Comparison with Predicate Device:

<table><tr><td rowspan=1 colspan=3>Similarities and Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceIon PGM Dx System(k170299)</td><td rowspan=1 colspan=1>PredicateMiSeqDx Platform(k123989)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>For targeted sequencing ofhuman genomic DNA fromperipheral whole bloodsamples and for DNA andRNA extracted fromformalin-fixed, paraffin-embedded (FFPE) samples</td><td rowspan=1 colspan=1>For targeted sequencing ofhuman genomic DNA fromperipheral whole bloodsamples</td></tr><tr><td rowspan=1 colspan=1>Environment of Use</td><td rowspan=1 colspan=1>Clinical Laboratories</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>Combined functionssoftware for separate use inIVD and research use onlymodes</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Whole blood or formalinfixed paraffin embedded(FFPE)samples</td><td rowspan=1 colspan=1>Whole blood samples</td></tr><tr><td rowspan=1 colspan=1>Input Sample</td><td rowspan=1 colspan=1>Genomic DNA and cDNA</td><td rowspan=1 colspan=1>Genomic DNA</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Measurement of hydrogenions that are generatedduring the incorporation ofnucleotides during a DNAsequencing reaction</td><td rowspan=1 colspan=1>Measurement offluorescent signals fromlabeled nucleotidesincorporated during a DNAsequencing reaction</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>Combined functionssoftware for separate use inIVD and research use onlymodes</td><td rowspan=1 colspan=1>Combined functionssoftware for separate use inIVD and research use onlymodes</td></tr></table>

# I. Special Control/Guidance Document Referenced (if applicable):

November 19, 2013, De Novo Classification Order for k123989 (DEN130011)

# J. Performance Characteristics:

The instrument was previously legally marketed for use with genomic DNA from whole blood samples under 21 CFR 862.2265. The performance information provided here is specific to RNA and genomic DNA from FFPE samples only. Performance characteristics of the instrument with genomic DNA from whole blood are included in the device labeling.

The following table is a summary of the sample type distribution used to characterize instrument performance with the indicated variants in nucleic acid extracted from FFPE samples using a representative assay:

<table><tr><td rowspan=2 colspan=1>Type ofVariant</td><td rowspan=2 colspan=1>Number of variantsdetected by therepresentativeassay</td><td rowspan=1 colspan=3>Number of samples tested for detection bysample type</td></tr><tr><td rowspan=1 colspan=1>Plasmid/FFPESample Blend</td><td rowspan=1 colspan=1>FFPE CellLine orFFPE CellLineBlend</td><td rowspan=1 colspan=1>FFPEClinicalSample</td></tr><tr><td rowspan=1 colspan=1>MNV</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>SNV</td><td rowspan=1 colspan=1>326</td><td rowspan=1 colspan=1>329</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>3-bpdeletion</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>6-bpdeletion</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9-bpdeletion</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>12-bpdeletion</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>15-bpdeletion</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>18-bpdeletion</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr></table>

1. Analytical Performance:

a. Accuracy:

A study was performed to demonstrate positive percent agreement (PPA) and negative percent agreement (NPA) concordance between detection of somatic variants in FFPE tumor samples using a representative next-generation sequencing (NGS) assay and detection using validated reference methods. The following reference detection methods were used:

• A validated NGS assay to detect SNV and deletion hotspot variants to detect SNV and deletion hotspot variants

• FISH/IHC testing to detect RNA fusions

Variants detected by the representative assay for which the reference method testing failed and did not yield a valid result were not included in the calculations. Accuracy data was analyzed by the following: each variant location, bins (or categories of variants, i.e., RNA fusions, simple and complex single nucleotide variants (SNVs) and deletions), and each FFPE sample. The results are shown in the following tables:

# Positive Percent Agreement (PPA)

<table><tr><td rowspan=2 colspan=1>PPAMeasure</td><td rowspan=1 colspan=2>Excluding no calls</td><td rowspan=1 colspan=2>Including no calls</td></tr><tr><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>98.5% (195/198)</td><td rowspan=1 colspan=1>95.6%,99.7%</td><td rowspan=1 colspan=1>98.5% (195/198)</td><td rowspan=1 colspan=1>(95.6%,99.7%)</td></tr><tr><td rowspan=1 colspan=1>Bin</td><td rowspan=1 colspan=1>97.2% (176/181)</td><td rowspan=1 colspan=1>93.7%,99.1%</td><td rowspan=1 colspan=1>97.2% (176/181)</td><td rowspan=1 colspan=1>(93.7%,99.1%)</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>96.9% (158/163)</td><td rowspan=1 colspan=1>93.0%,99.0%</td><td rowspan=1 colspan=1>96.9% (158/163)</td><td rowspan=1 colspan=1>(93.0%,99.0%)</td></tr></table>

# Negative Percent Agreement (NPA)

<table><tr><td rowspan=2 colspan=1>NPAMeasure</td><td rowspan=1 colspan=2>Excluding no calls</td><td rowspan=1 colspan=2>Including no calls</td></tr><tr><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>100.0%(118,155/118,159)</td><td rowspan=1 colspan=1>99.99%,100.0%</td><td rowspan=1 colspan=1>96.8%(118,155/122,012)</td><td rowspan=1 colspan=1>(96.7%,96.9%)</td></tr><tr><td rowspan=1 colspan=1>Bin</td><td rowspan=1 colspan=1>99.8% (942/944)</td><td rowspan=1 colspan=1>99.2%,100.0%</td><td rowspan=1 colspan=1>70.0%(657/939)</td><td rowspan=1 colspan=1>(66.9%,72.9%)</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>98.4% (124/126)</td><td rowspan=1 colspan=1>94.4%,99.8%</td><td rowspan=1 colspan=1>23.4% (29/124)</td><td rowspan=1 colspan=1>(16.3%,31.8%)</td></tr></table>

# Overall Percent Agreement (OPA)

<table><tr><td rowspan=2 colspan=1>OPAMeasure</td><td rowspan=1 colspan=2>Excluding no calls</td><td rowspan=1 colspan=2>Including no calls</td></tr><tr><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percentagreement</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Variant</td><td rowspan=1 colspan=1>100.%(118,350/118,357)</td><td rowspan=1 colspan=1>99.99%,100.0%</td><td rowspan=1 colspan=1>96.8%(118,350/122,210)</td><td rowspan=1 colspan=1>(96.74%,96.94%)</td></tr><tr><td rowspan=1 colspan=1>Bin</td><td rowspan=1 colspan=1>99.4%(1,118/1,125)</td><td rowspan=1 colspan=1>98.72%,99.75%</td><td rowspan=1 colspan=1>74.4%(833/1,120)</td><td rowspan=1 colspan=1>(71.71%,76.91%)</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>97.6% (282/289)</td><td rowspan=1 colspan=1>95.07%,99.02%</td><td rowspan=1 colspan=1>65.2%(187/287)</td><td rowspan=1 colspan=1>(59.34%,70.66%)</td></tr></table>

# b. Precision/Reproducibility:

Two studies were conducted to assess reproducibility:

# Sample Reproducibility

Two wild-type (WT) and ten variant-positive samples were evaluated at 4 testing sites, each with 4 Ion PGM Dx systems and 4 operators, to determine the reproducibility and repeatability of detection for multiple variants using a representative assay.

Each sample was tested 8 times at each site, for a total of 32 replicates per sample, or 768 sample sequencing results (12 samples $\times 3 2$ replicates $\times 2$ library types (RNA and DNA)). The call rate, no call rate, positive call rate, negative call rate, and withinrun repeatability were computed at each variant location of interest.

Including no calls and excluding known positive variant locations, the negative call rate at each clinical variant location for all samples was $100 \%$ . Including no calls, all positive call rates from positive variant locations were ${ > } 8 4 \%$ .

Excluding no calls and combining data across all study samples, the estimate of repeatability was $100 \%$ for DNA variants and $8 7 . 5 \%$ for the RNA variants. The lower limit of the $9 5 \%$ two-sided confidence interval (CI) for repeatability exceeded $9 6 \%$ at all variant locations.

Including no calls from the data, the estimate of repeatability was $100 \%$ at 218 out of 605 variant locations, $9 4 { - } 9 9 . 9 \%$ at 186 out of 605 variant locations, and $7 1 . 6 { - } 9 3 . 9 \%$ at 184 out of 605 variant locations. Including no calls, the lower limit of the $9 5 \%$ twosided confidence interval for repeatability exceeded $6 4 . 6 \%$ at all variant locations. The lower limit of the $9 5 \%$ two-sided confidence interval (CI) for repeatability exceeded $9 6 \%$ at all variant locations.

Sample reproducibility results are shown in the two tables below (data from one study split into two tables):

<table><tr><td colspan="1" rowspan="2">Sadmee</td><td colspan="1" rowspan="2">tangAr</td><td colspan="1" rowspan="2">ArCo</td><td colspan="1" rowspan="2"># ofvalidsampleresults(N)</td><td colspan="1" rowspan="2"># ofpositivecalls(A)</td><td colspan="1" rowspan="2"># ofnegativecalls(B)</td><td colspan="1" rowspan="2"># ofNoCalls(C)</td><td colspan="2" rowspan="1">Positive call rate + 95%CI</td></tr><tr><td colspan="1" rowspan="1">Includingno calls(A/N)</td><td colspan="1" rowspan="1">Excludingno calls(A/(A+B))</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">COSM6223</td><td colspan="1" rowspan="1">EGRFExon19del</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">COSM763</td><td colspan="1" rowspan="1">PIK3C AE545K</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">ROS1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93.8%(79.2%,99.2%)</td><td colspan="1" rowspan="1">93.8%(79.2%,99.2%)</td></tr><tr><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">COSM6225</td><td colspan="1" rowspan="1">EGFRExon19del</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1">COSM476</td><td colspan="1" rowspan="1">BRAFV600E</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">COSM521</td><td colspan="1" rowspan="1">KRASG12D</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">93.8%(79.2%,99.2%)</td><td colspan="1" rowspan="1">100%(88.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">COSM29313</td><td colspan="1" rowspan="1">PIK3C AM1043I</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">93.8%(79.2%,99.2%)</td><td colspan="1" rowspan="1">100%(88.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">G</td><td colspan="1" rowspan="1">COSM6224</td><td colspan="1" rowspan="1">EGFRL858R</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">J</td><td colspan="1" rowspan="1">COSM87298</td><td colspan="1" rowspan="1">KRASQ61K</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,</td><td colspan="1" rowspan="1">100%(89.1%,</td></tr><tr><td colspan="1" rowspan="2">Saee</td><td colspan="1" rowspan="2">thhunArr!</td><td colspan="1" rowspan="2">arto</td><td colspan="1" rowspan="2"># ofvalidsampleresults(N)</td><td colspan="1" rowspan="2"># ofpositivecalls(A)</td><td colspan="1" rowspan="2"># ofnegativecalls(B)</td><td colspan="1" rowspan="2"># ofNoCalls(C)</td><td colspan="2" rowspan="1">Positive call rate + 95%CI</td></tr><tr><td colspan="1" rowspan="1">Includingno calls(A/N)</td><td colspan="1" rowspan="1">Excludingno calls(A/(A+B))</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%)</td><td colspan="1" rowspan="1">100%)</td></tr><tr><td colspan="1" rowspan="1">J</td><td colspan="1" rowspan="1">COSM172423</td><td colspan="1" rowspan="1">ERBB3V104M</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">K</td><td colspan="1" rowspan="1">COSM775</td><td colspan="1" rowspan="1">PIK3H1047R</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">96.7%(82.8%,99.9%)</td><td colspan="1" rowspan="1">100%(88.1%,100%)</td></tr><tr><td colspan="1" rowspan="1">M</td><td colspan="1" rowspan="1">COSM715</td><td colspan="1" rowspan="1">FGR3S249C</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td><td colspan="1" rowspan="1">100%(89.1%,100%)</td></tr><tr><td colspan="1" rowspan="2">Sdme</td><td colspan="1" rowspan="2">tongAr</td><td colspan="1" rowspan="2">Rrr tr</td><td colspan="1" rowspan="2"># ofvalidsampleresults(N)</td><td colspan="1" rowspan="2"># ofpositivecalls(A)</td><td colspan="1" rowspan="2"># ofnegativecallS(B)</td><td colspan="1" rowspan="2"># ofNoCalls(C)</td><td colspan="2" rowspan="1">Negative callrate + 95% CI</td><td colspan="2" rowspan="1">Within-runrepeatability +95% CI</td></tr><tr><td colspan="1" rowspan="1">Including nocalls(B/N)</td><td colspan="1" rowspan="1">Excluding nocalls(B/(A+B</td><td colspan="1" rowspan="1">Including nocalls</td><td colspan="1" rowspan="1">Excluding nocalls</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">COSM6223</td><td colspan="1" rowspan="1">EGRFExon19del</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">COSM763</td><td colspan="1" rowspan="1">PIK3CAE545K</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,10.9%</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">C</td><td colspan="1" rowspan="1">ROS1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6.3%(0.8%,20.8%)</td><td colspan="1" rowspan="1">6.3%(0.8%,20.8%</td><td colspan="1" rowspan="1">87.5%(61.7%,98.4%)</td><td colspan="1" rowspan="1">87.5%(61.7%,98.4%)</td></tr><tr><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">COSM6225</td><td colspan="1" rowspan="1">EGFRExon19del</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">E</td><td colspan="1" rowspan="1">COSM476</td><td colspan="1" rowspan="1">BRAF600E</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.%),100%)</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">COSM521</td><td colspan="1" rowspan="1">KRASG12D</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,11.6%</td><td colspan="1" rowspan="1">87.5%(61.7%,98.4%)</td><td colspan="1" rowspan="1">100%(76.8%,100%)</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">COSM29313</td><td colspan="1" rowspan="1">PIK3CAM1043I</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,11.6%)</td><td colspan="1" rowspan="1">87.5%(61.7%,98.4%)</td><td colspan="1" rowspan="1">100%(76.8%,100%)</td></tr><tr><td colspan="1" rowspan="1">G</td><td colspan="1" rowspan="1">COSM6224</td><td colspan="1" rowspan="1">EGFRL858R</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,</td><td colspan="1" rowspan="1">0%(0%,</td><td colspan="1" rowspan="1">100%(79.4%,</td><td colspan="1" rowspan="1">100%(79.4%,</td></tr><tr><td colspan="1" rowspan="2">Sdmee</td><td colspan="1" rowspan="2">iangnFrr</td><td colspan="1" rowspan="2">Arr r</td><td colspan="1" rowspan="2"># ofvalidsampleresults(N)</td><td colspan="1" rowspan="2"># ofpositivecalls(A)</td><td colspan="1" rowspan="2"># ofnegativecallS(B)</td><td colspan="1" rowspan="2"># ofNoCalls(C)</td><td colspan="2" rowspan="1">Negative callrate + 95% CI</td><td colspan="2" rowspan="1">Within-runrepeatability +95% CI</td></tr><tr><td colspan="1" rowspan="1">Including nocalls(B/N)</td><td colspan="1" rowspan="1">Excluding nocalls(B/(A+B)</td><td colspan="1" rowspan="1">Including nocalls</td><td colspan="1" rowspan="1">Excluding nocalls</td></tr><tr><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10.9%)</td><td colspan="1" rowspan="1">10.9%</td><td colspan="1" rowspan="1">100%)</td><td colspan="1" rowspan="1">100%)</td></tr><tr><td colspan="1" rowspan="1">J</td><td colspan="1" rowspan="1">COSM87298</td><td colspan="1" rowspan="1">KRASQ61K</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">J</td><td colspan="1" rowspan="1">COSM172423</td><td colspan="1" rowspan="1">ERBB3V104M</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr><tr><td colspan="1" rowspan="1">K</td><td colspan="1" rowspan="1">COSM775</td><td colspan="1" rowspan="1">PIK3H1047R</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0%(0%,11.6%)</td><td colspan="1" rowspan="1">0%(0%,11.9%)</td><td colspan="1" rowspan="1">93.3%(68.1%,99.8%)</td><td colspan="1" rowspan="1">100%(76.8%,100%)</td></tr><tr><td colspan="1" rowspan="1">M</td><td colspan="1" rowspan="1">COSM715</td><td colspan="1" rowspan="1">FGR3S249C</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%(0%,10.9%</td><td colspan="1" rowspan="1">0%(0%,10.9%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td><td colspan="1" rowspan="1">100%(79.4%,100%)</td></tr></table>

# Assay Reproducibility

The reproducibility and repeatability of a representative assay on the Ion PGM Dx System was evaluated for 30 representative variants from 18 DNA and 9 RNA samples. The study was designed to evaluate within-run precision performance (repeatability) and variability across sites, operators, and instrument platforms (reproducibility). Six of the 18 DNA samples were mixtures of plasmid and clinical DNA. Seven of the 12 deletion variants were represented by these plasmid blends. All other variant types were represented by clinical sample DNA.

Due to the large number of variants detected by the test and the rarity of some of the variants, a representative variant approach was used. Variants were selected in the following categories:

<table><tr><td rowspan=1 colspan=1>Variant Category</td><td rowspan=1 colspan=1>No. of Plasmid Blends No. of Clinical SamplesUsed</td><td rowspan=1 colspan=1>No. of Plasmid Blends No. of Clinical SamplesUsed</td></tr><tr><td rowspan=1 colspan=1>6-bp deletion</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>9-bp deletion</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>15-bp deletion</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>18-bp deletion</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Simple SNV</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Complex SNVs andMNPs**</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Fusion</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr></table>

\*Including SNVs in di- or tri-nucleotide repeat regions and SNVs in high-GC $( > 6 0 \% )$ or low-GC $( < 4 0 \% )$ content regions

Two of the $1 8 \mathrm { D N A }$ samples were WT at all locations, and the remaining 16 contained DNA from one or more DNA variants. One of the 9 RNA samples contained no fusion molecules, and the remaining 8 samples each contained RNA from an RNA variant. Each pre-extracted DNA or RNA sample was sequenced at 4 sites by 2 operators on 2 systems at each site.

At each site, operators were grouped into 2 pairs, with each pair being assigned to 2 instrument systems, and each pair being responsible for testing of 9 of the DNA samples and all 9 of the RNA samples. Samples were run in duplicate using 2 different reagent lots at 3 of the study sites and on all 3 reagent lots at 1 study site. The design resulted in a total of 72 test determinations per DNA sample (3 sites $\times 2$ lots $\times 2$ operators $\times 2$ systems $\times 2$ replicates $^ { + 1 }$ site $\times 3$ lots $\times 2$ operators $\times 2$ systems $\times 2$ replicates). Because there were half as many RNA samples as DNA samples, each RNA sample was tested twice as many times $\scriptstyle ( \mathrm { n = 1 4 4 } )$ ). In total, at least 1,296 sequencing reactions were performed, and all variant locations were assessed for each sample.

The reproducibility results are summarized in the following table:

<table><tr><td rowspan=2 colspan=1>Description</td><td rowspan=2 colspan=1>No. of variantsamples</td><td rowspan=1 colspan=2>Call rateexcluding nocalls</td><td rowspan=1 colspan=2>Call rateincluding nocalls</td><td rowspan=1 colspan=2>Call rateincluding no callsand invalids</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td></tr><tr><td rowspan=1 colspan=1>DNApositive variants(positive calls)</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>96.60%</td><td rowspan=1 colspan=1>97.10%</td><td rowspan=1 colspan=1>94.50%</td><td rowspan=1 colspan=1>95.80%</td><td rowspan=1 colspan=1>94.50%</td><td rowspan=1 colspan=1>95.80%</td></tr><tr><td rowspan=1 colspan=1>RNApositive variants(positive calls)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>94.80%</td><td rowspan=1 colspan=1>95.50%</td><td rowspan=1 colspan=1>94.80%</td><td rowspan=1 colspan=1>95.50%</td><td rowspan=1 colspan=1>94.80%</td><td rowspan=1 colspan=1>95.50%</td></tr><tr><td rowspan=1 colspan=1>WTDNAvariantlocations</td><td rowspan=1 colspan=1>872</td><td rowspan=1 colspan=1>96.10%</td><td rowspan=1 colspan=1>95.00%</td><td rowspan=1 colspan=1>96.10%</td><td rowspan=1 colspan=1>95.00%</td><td rowspan=1 colspan=1>90.70%</td><td rowspan=1 colspan=1>93.10%</td></tr><tr><td rowspan=1 colspan=1>WTRNAvariantlocations</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>99.30%</td><td rowspan=1 colspan=1>99.30%</td><td rowspan=1 colspan=1>99.30%</td><td rowspan=1 colspan=1>99.30%</td><td rowspan=1 colspan=1>99.30%</td><td rowspan=1 colspan=1>99.30%</td></tr></table>

Excluding no calls, the estimate of repeatability at each DNA variant location across all the samples was $2 9 8 . 8 \%$ ( $9 5 \%$ CI lower limit of $2 9 7 . 5 \%$ ). The coefficient of variation (CV) across all DNA clinical variants ranged from $9 . 8 \%$ to $3 9 \%$ . The highest CVs $( 2 4 . 9 \mathrm { - } 3 9 . 2 \% )$ were observed for one specific variant (BRAF V600E variant); the root cause for the higher percent CV for this sample was not definitively determined. The CVs for other specific variants (e.g., EGFR L858R variant and EGFR deletion variants) ranged from $9 . 8 \%$ to $1 1 . 3 \%$ , and $1 1 . 2 \%$ to $2 5 . 5 \%$ respectively.

Excluding no calls, the estimate of repeatability at each RNA clinical variant location was $9 4 . 4 \%$ . The CV across all RNA locations ranged from $72 \%$ to $78 \%$ .

c. Linearity:

Not applicable.

d. Carryover:

A study was performed to evaluate the potential for inter-run and intra-run sample carryover (cross-contamination). Eight FFPE cell line samples were evaluated to determine the percentage of false positive results caused by contamination from one sample to another within the same sequencing run and by contamination from a previous run on the same instrument system. Samples that were WT and mutant were tested in consecutive sequencing runs on the same instruments, and 5 DNA variant locations and 2 RNA variant locations that were expected to be WT for a sample were evaluated for contamination. Out of $1 0 0 \mathrm { D N A }$ and 80 RNA data points analyzed, no false positive results were reported in the DNA variants, and 1 false positive result was reported in an RNA variant. Therefore, the false-positive rate at the DNA variant locations was $0 \%$ (0/100) and the false-positive rate at the RNA variant locations was $1 . 2 5 \%$ (1/80).

# e. Interfering Substances:

Endogenous and Exogenous Substances

To assess the impact of endogenous and exogenous interfering substances on the instrument, a representative assay was evaluated in the presence and absence of potential interferents. Six potentially interfering substances were evaluated, including five potentially interfering substances used to extract DNA and RNA from FFPE tissue samples, in addition to hemoglobin, a potentially interfering endogenous substance. A total of 8 FFPE samples (1 WT and 7 mutants) with 6 replicates each were processed through the entire assay workflow, and the samples were spiked with additional concentrations or amounts of the listed substances at the relevant processing step, as shown in the table. Replicates of a control sample with no spikeins were also analyzed. The concordance between variant calls in samples with and without interfering substances was computed for each substance under investigation.

Amounts of the tested substances and the steps at which they were added during the assay are summarized in the table below:

<table><tr><td colspan="1" rowspan="1">Potential interferingsubstance</td><td colspan="1" rowspan="1">Step</td><td colspan="1" rowspan="1">Amount ofsubstance</td><td colspan="1" rowspan="1">Call Rate</td></tr><tr><td colspan="1" rowspan="1">Paraffin</td><td colspan="1" rowspan="1">At the deparaffinization step, extraparaffin was addedto the xylenebath that contained 250mL ofxylene.</td><td colspan="1" rowspan="1">4Xof normallyexpected levels</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Xylene</td><td colspan="1" rowspan="1">Extra xylene was addedinto theethanol bath that contained 250mLof ethanol.</td><td colspan="1" rowspan="1">6Xof normallyexpectedresidual volume</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">Extra ethanol was addedinto theProtease digestion step beforedigestion</td><td colspan="1" rowspan="1">4Xof normallyexpectedresidual volume</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">After deparaffinization,hemoglobin was addedto theDigestion Buffer used to pre-wetthe tissue section</td><td colspan="1" rowspan="1">4 mgmL</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Protease</td><td colspan="1" rowspan="1">Extra Protease was addedinto thereactionafter the digestion step andbefore column purification</td><td colspan="1" rowspan="1">&gt;10Xof expectedresidualProtease afterthe heat-killstep</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Washbuffer</td><td colspan="1" rowspan="1">Washbuffer used to isolate DNAand RNAfrom deparaffinized anddigested samples was addedinto analiquot of Dilution Solution, whichwas subsequently used to dilute theRNAand DNAto the appropriateconcentration before librarypreparation</td><td colspan="1" rowspan="1">1%owash buffer(equivalent to~10%owashbuffer carriedover into eluate)</td><td colspan="1" rowspan="1">′</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Tissue sections were processedusing the standard protocol, withoutthe addition of anypotentiallyinterfering substances.</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">−</td></tr></table>

No false positive or false negative results were observed with any of the tested exogenous interferents. With no calls excluded, for each potential interferent, the positive and negative concordance with the control condition across all samples was $100 \%$ , and the overall concordance with the control condition across all samples was $100 \%$ .

With no calls excluded, the results of testing with hemoglobin showed positive concordance with the control condition of $100 \%$ (only samples with a positive control condition were analyzed), negative concordance of $9 9 . 9 9 \%$ , and overall concordance of $9 9 . 9 9 \%$ . For one sample, 1 of 6 replicates of a control condition was a false negative. For another sample, 1 of 6 replicates of a hemoglobin condition was a false positive.

# Necrosis

It is currently unknown whether necrotic tissue in the region of interest in FFPE tissue samples interferes with sequencing of FFPE samples. Therefore, users should macrodissect highly necrotic areas or select alternate samples if possible.

2. Other Supportive Instrument Performance Data Not Covered Above:

# Tissue input study:

Sixty slide-mounted FFPE samples were analyzed to determine if samples extracted using the Ion Torrent Dx Total Nucleic Acid Isolation Kit yield DNA and RNA at the concentrations required by a representative assay when tissue input requirements are met. The test requires DNA at a concentration of ${ \geq } 0 . 8 3 ~ \mathrm { n g / \mu L }$ and RNA at a concentration of ${ \geq } 1 . 4 3 ~ \mathrm { n g / \mu L }$ .

Thirty resection samples with $2 0 \%$ tumor content were prepared without macrodissection, 15 resection samples with $< 2 0 \%$ to $2 1 0 \%$ tumor cell content were macrodissected, and 15 samples were collected by core needle biopsy (CNB). For the resection samples, $2 \times 5 ~ { \mu \mathrm { m } }$ sections were used per extraction. For CNBs, $9 \times 5 ~ { \mu \mathrm { m } }$ sections were used per extraction. DNA and RNA concentrations were determined using the Ion Torrent Dx DNA and RNA Quantification Kits, respectively. No sequencing was performed on the extracted samples.

Of the 60 samples tested, $9 8 . 3 \%$ (59/60) had a DNA concentration of $\geq 0 . 8 3 ~ \mathrm { n g / \mu L }$ and an RNA concentration of ${ \geq } 1 . 4 3 ~ \mathrm { n g / \mu L }$ . One CNB sample failed the minimum DNA and RNA concentration specifications, with values of $0 . 5 2 ~ \mathrm { n g / \mu L }$ and $1 . 2 3 ~ \mathrm { n g / \mu L }$ respectively.

# DNA input study:

To evaluate the amount of input FFPE-excted DNA and RNA required to reproducibly make accurate variant calls, eight cell-line samples were prepared as FFPE sections, and DNA and RNA were extracted and quantified from multiple sections from each cell line for blending and testing. Sample blends were prepared with known variants at various DNA and RNA input-level combinations within the range of $5 \mathrm { - } 1 5 ~ \mathrm { n g } .$ . The DNA and RNA blends had a target allele frequency of $1 5 \%$ for SNVs and deletions and target fusion reads of 300−600 for a specific variant (ROS1). A total of 540 individual DNA and RNA libraries were tested, including positive controls and NTC controls, with 6 replicate libraries each for DNA and RNA per test condition.

The study demonstrated a $100 \%$ positive variant call rate within the input range tested, supporting an input amount of $1 0 \mathrm { n g }$ each for DNA and RNA for a specific representative assay.

The negative variant call rate was $59 5 \%$ for all except 4 sample and DNA/RNA inputlevel combinations. All cases with a negative variant call rate of ${ < } 9 5 \%$ were due to no calls, 3 of which occurred with a DNA or RNA input amount of $5 \mathrm { n g }$ and 1 of which occurred in a single sample with DNA and RNA inputs of $1 0 \mathrm { n g }$ each. There were no false-positive calls.

Additionally, 4 clinical samples prepared as FFPE sections were tested: two samples containing DNA variants and two containing the specific CD74-ROS1 fusion variant. The DNA variant samples were paired with wild-type RNA from the same sample at various input combinations within the range of $5 - 1 5 ~ \mathrm { n g }$ , and the RNA variant samples were paired with wild-type DNA at input combinations within the same range.

The study demonstrated positive and negative call rates of $59 5 \%$ for the DNA variants at all input combinations, and $100 \%$ for one of the CD74-ROS1 fusion variants at all input combinations. The second CD74-ROS1 clinical sample showed $100 \%$ negative call rates for all test conditions, and $100 \%$ positive call rates except for Test Condition 4 $( 8 . 5 \mathrm { n g }$ RNA/15 ng DNA), where the call rate was $83 \%$ , and Test Condition 6 ( $1 5 \mathrm { n g } \mathrm { R N A } / 1 5 \mathrm { n g }$ DNA), where the call rate was $50 \%$ .

# Sample indexing study:

Sample barcode adapters are used in all runs to assign a unique nucleic acid barcode to each sample DNA, allowing the ability to pool up to sixteen (16) barcoded sample

libraries into a single Ion PGM Dx sequencing run. All 16 barcode adapters were tested with the SVA panel and genomic DNA samples (NA12878 and NA19240) to determine variant call (SNV and Indel) reproducibility between adapter sequences. Sample results were compared to reference database sequences for the NA12878 and NA19240 genomes at all variant and non-variant positions. Comparison of data from 60 runs consisting of 4 sequencing reactions (2 DNA samples $\times 2$ barcodes each) and 24 runs consisting of 16 sequencing reactions (2 DNA samples $\times 8$ barcodes each) indicate that the barcode adapter used and the number of samples pooled together had no effect on variant call reproducibility.

Indexing chemistry and software is identical for all libraries, independent of the type and source of nucleic acid. The previously conducted indexing study for indexing genomic DNA extracted from whole blood was therefore used to support indexing of FFPE RNA and DNA samples.

# K. Proposed Labeling:

Labeling satisfies the requirements of 21 CFR 809.10, 21 CFR 801.109, including an appropriate prescription statement as required by 21 CFR 801.109(b), and the special controls for this type of device.

# L. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.