# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

SERPINA1 gene variants from human blood specimens

# D. Type of Test:

Multiplex PCR followed by multiplex allele specific primer extension for genotyping, hybridized to multiplexed fluorescing microparticles, detected by a Luminex analyzer.

# E. Applicant:

# G. Regulatory Information:

1. Regulation section: 21 CFR $\ S$ 866.5130 Alpha-1-antitrypsin immunological test system

2. Classification: Class II

3. Product code: PZH, SERPINA1 Variant Detection System

4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use:

The Progenika A1AT genotyping kit is a qualitative, polymerase chain reaction (PCR) and hybridization-based in vitro diagnostic test to be used with the Luminex $2 0 0 ^ { \mathrm { T M } }$ instrument (with xPONENT® software) for the simultaneous detection and identification of 14 allelic variants and their associated alleles found in the Alpha-1 antitrypsin (A1AT) codifying gene SERPINA1.The test is intended for use with genomic DNA extracted from human whole blood samples collected as dry blood spots (DBS) or in K2 EDTA. The A1AT allelic variant genotypes and associated allele results, when used in conjunction with clinical findings and other laboratory tests, are intended as an aid in the diagnosis of individuals with A1AT deficiency (A1ATD).

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements:

The A1AT Genotyping Test is to be performed using the 384-well Veriti $^ \mathrm { \textregistered }$ Dx and 96-well Veriti $^ \mathrm { \textregistered }$ Dx thermal cyclers, and the Luminex 200 instrument.

The xPONENT 3.1 is a software application that is used to view and analyze genotype data obtained from Luminex 200. The A1AT Genotyping Test ANALYSIS SOFTWARE conducts control checks on the file, resulting in a final analytical genotype profile for each sample.

# I. Device Description:

The A1AT Genotyping Test is comprised of a single multiplex PCR reaction for amplification of target DNA sequences followed by hybridization to microspheres and extension of allele-specific oligonucleotide probes for determination of allelic variants.

The test is composed of four reagent components in sufficient quantity for either 48 or 192 tests and a CD containing the A1AT Genotyping Test ANALYSIS SOFTWARE.

1. The A1AT PCR Master Mix contains four primer pairs needed for the amplification of four genomic DNA fragments that contain the 14 allelic variants interrogated by the test. In addition to the primers, this kit component contains all the ingredients needed for a PCR reaction (nucleotides, biotin-dCTP, buffer) except the DNA Taq Polymerase that must be purchased independently.

2. The A1AT Beads Master Mix contains 28 individual oligonucleotide probes (two probes per tested allelic variant, one complementary to the common allele and another complementary to the variant allele) coupled to color coded beads (carboxylated microspheres from Luminex xMAP $^ \mathrm { \textregistered }$

technology) in addition to the tetramethylammonium chloride containing buffer needed to guarantee the correct performance of the hybridization.

3. The SAPE Dilution Buffer (phosphate buffer with Tween and ProClin 950) is used to dilute the SAPE component which contains a streptavidin-phycoerythrin conjugate required for the labeling and detection steps for the A1AT Genotyping Test.

4. The A1AT Genotyping Test Software component contains the kit instructions for use (Package Insert and ANALYSIS SOFTWARE User Manual), the A1AT Genotyping Test ANALYSIS SOFTWARE, and the A1AT Genotyping Test Luminex Template. This template file contains the Luminex protocol to run the A1AT Genotyping Test and includes the following information: the required temperature of the heater $( 5 2 ^ { \circ } \mathrm { C } )$ , the specific bead types to be tested, the association between the bead type and the specific probe (Probe Set and normal or variant), the allowed doublet discrimination window (DD gate) between 7000 ̶ 20000, the volume of sample to be taken from each well $( 9 0 \mu \mathrm { L } )$ , the minimum number of each bead type to be counted per well (80), and the maximum time allowed to analyze each well (90 seconds).

The end user of A1AT Genotyping Test assay is required to include one Negative Control (nuclease free molecular-grade water, not provided as part of A1AT Genotyping Test kit) to determine run validity. It must be included in each assay run to check for potential DNA contamination. It is also recommended, but not required, to include a Positive Control sample (DNA of known genotype, not provided in the kit) in each assay run.

# J. Substantial Equivalence Information:

1. Predicate device name and 510(k) number: SEBIA, Inc. HYDRAGEL 18 A1AT Isofocusing Kit, K063498

2. Comparison with predicate:

<table><tr><td colspan="3">Differences</td></tr><tr><td>Item</td><td>New Device A1AT Genotyping Test</td><td>Predicate HYDRAGEL 18 A1AT Isofocusing Kit</td></tr><tr><td>Intended Use</td><td>For the simultaneous detection and identification of 14 allelic variants and their associated alleles found in the Alpha-1 antitrypsin (A1AT) codifying gene SERPINA1. The test is intended for use with genomic DNA extracted from human whole blood samples collected as dry blood spots (DBS) or in K2 EDTA. The A1AT allelic variant genotypes and associated allele results, when used in conjunction with clinical findings and other laboratory tests, are intended as an</td><td>For the qualitative detection and identification of the different phenotypes of Alpha-1 antitrypsin (A1AT). Phenotyping results in conjunction with clinical findings and other laboratory assays aid in the diagnosis of Alpha-1 antitrypsin deficiency. The analysis is performed on human sera separated into electrophoretic patterns ready for qualitative analysis. The procedure includes isoelectrofocusing on agarose gel, performed on the semi-automatic</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">aid in the diagnosis of individuals withA1AT deficiency (A1ATD).</td><td colspan="1" rowspan="1">HYDRASYS system, followed byimmunofixation with anti-Alpha-1antitrypsin antiserum. The use of enzymelabeled anti-Alpha-1 antitrypsinantiserum enhanced the detection andidentification of the different phenotypes.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Human whole blood samples (DBS or K2EDTA tubes)</td><td colspan="1" rowspan="1">Human serum from whole blood samples</td></tr><tr><td colspan="1" rowspan="1">Assay format</td><td colspan="1" rowspan="1">Qualitative identification of A1AT allelesindicative of A1ATD.</td><td colspan="1" rowspan="1">Qualitative identification of A1ATphenotypes indicative of A1ATD.</td></tr><tr><td colspan="1" rowspan="1">Methodologyand Instrument</td><td colspan="1" rowspan="1">Multiplex PCR amplification andbiotinylation of DNA extracted from DBSor from human K2 EDTA anticoagulatedwhole blood. PCR products are denaturedand hybridized to oligonucleotide probescoupled to color-coded beads. HybridizedDNA is labeled with a fluorescentconjugate and resulting signal is detectedwith a Luminex 200 system. Results areinterpreted by A1AT Genotyping TestAnalysis Software to report the presence,absence or combination of the testedalleles.</td><td colspan="1" rowspan="1">Isoelectrofocusing on agarose gel,performed on the semiautomaticHYDRASYS system, followed byimmunofixation with anti-Alpha-1antitrypsin antiserum. The migrationpattern obtained for each sample isvisually evaluated and used to determinethe A1AT phenotype, as compared tocontrol sample of known A1ATphenotype.</td></tr></table>

# K. Standard/Guidance Document Referenced:

CLSI guideline EP05-A3, “Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline–Third Edition”

CLSI guideline EP07-A2, “Interference Testing in Clinical Chemistry; Approved Guideline–Second Edition”

CLSI guideline EP12-A2, “User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline–Second Edition”

CLSI guideline EP17-A2, “Evaluation of Detection Capability for Clinical. Laboratory Measurement Procedures; Approved Guideline–Second Edition”

CLSI guideline EP25-A, “Evaluation of Stability of In Vitro Diagnostic Reagents”

# L. Test Principle:

The A1AT Genotyping Test assay incorporates multiplex Polymerase Chain Reaction (PCR) and multiplex Allele Specific Primer Extension (ASPE) with Luminex’s Universal Tag sorting system on the Luminex 200 xMAP™ platform.

The assay consists of the following processing steps:

Firstly, target sequences in the DNA (previously extracted from human K2 EDTA anticoagulated whole blood or from dry blood spots) are amplified using specific primer pairs and biotinylated using the A1AT PCR Master Mix. As result of this multiplex PCR, four biotin-labeled DNA fragments that span the 14 allelic variants under analysis are obtained. A 384-well Veriti® Dx (Life Technologies®) thermal cycler is required for this step.

The PCR products are then denatured and hybridized to oligonucleotide probes coupled to specific color-coded beads. Each probe type is coupled to a specific bead identifiable by its specific spectral properties included in the A1AT Beads Master Mix reagent. Two probes coupled to two different beads are needed for the analysis of each allelic variant, one complementary to the sequence of the common allele and the other specific for the variant allele. Therefore, A1AT uses 28 color-coded beads, two color-coded beads per tested variant allele, to simultaneously genotype the allelic variants in one single suspension array. Each coupled bead is spectrally distinguishable from the other coupled beads. A 96-well Veriti Dx (Life Technologies) thermal cycler is required for this step.

Hybridized DNA is labeled with a fluorescent conjugate (SAPE, previously diluted in SAPE Dilution Buffer) with a high binding affinity for the biotin incorporated to the DNA in the amplification step. This fluorescent labeling step allows the detection and quantification of the hybridization signal by the Luminex System. This step is also performed using a 96-well Veriti Dx (Life Technologies) thermal cycler.

In the data acquisition step, the fluorescent signal associated with each specific bead and probe is detected with a Luminex 200 instrument, allowing the analysis of the mixture of different oligonucleotide probe-coupled beads one-by-one. The Luminex instrument contains two lasers: one identifies the color-coded beads, and the other detects the fluorescence emission intensity coming from the SAPE fluorescent conjugate attached to the biotinylated DNA that is bound to the probes on the surface of each bead. For each colored bead (i.e. each tested allele), the median fluorescence intensity (MFI) emitted by at least 80 beads is provided by the Luminex xPonent Software.

Finally, in the data analysis step, raw data from the Luminex System (.csv files containing the MFI value for each bead type) is processed with the A1AT Genotyping Test ANALYSIS SOFTWARE to determine the presence or absence $( + + , + / - , - / - )$ of each variant allele detected by the A1AT Genotyping Test.

The A1AT Genotyping Test is indicated to detect and report the presence or absence of 14 allelic variants found in the SERPINA1 gene.

<table><tr><td colspan="1" rowspan="1">AllelicVariant%</td><td colspan="1" rowspan="1">Associated allelicnames^</td><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Allelegenotyperesult</td><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Allelegenotyperesult</td><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Allelegenotyperesult</td></tr><tr><td colspan="1" rowspan="1">c.187C&gt;T</td><td colspan="1" rowspan="1">PI* I</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/C</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.194T&gt;C</td><td colspan="1" rowspan="1">PI* M procida</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.226_228delTTC@</td><td colspan="1" rowspan="1">PI* M malton,PI* M palermo,PI* M nichinan</td><td colspan="1" rowspan="1">delTTC/delTTC</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">delTTC/ TTC</td><td colspan="1" rowspan="1">+ -</td><td colspan="1" rowspan="1">TTC/TTC</td><td colspan="1" rowspan="1">− -</td></tr><tr><td colspan="1" rowspan="1">c.230C&gt;T</td><td colspan="1" rowspan="1">PI* S iiyama</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/c</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.552delC</td><td colspan="1" rowspan="1">PI* Q0 granite falls</td><td colspan="1" rowspan="1">delC/delC</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">delC/C</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-1-</td></tr><tr><td colspan="1" rowspan="1">c.646+1G&gt;T</td><td colspan="1" rowspan="1">PI* Q0 west</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/G</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">-1-</td></tr><tr><td colspan="1" rowspan="1">c.721A&gt;T</td><td colspan="1" rowspan="1">PI* Q0 bellingham</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/A</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.739C&gt;T</td><td colspan="1" rowspan="1">PI* F</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/A</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.839A&gt;T@</td><td colspan="1" rowspan="1">PI* P lowell,PI* P duarte,PI* Q0 cardiff,PI* Y barcelona</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/A</td><td colspan="1" rowspan="1">+ -</td><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">- -</td></tr><tr><td colspan="1" rowspan="1">c.863A&gt;T</td><td colspan="1" rowspan="1">PI* S}$</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/A</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.1096G&gt;A</td><td colspan="1" rowspan="1">PI* Z$</td><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">G/A</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.1130dupT@</td><td colspan="1" rowspan="1">PI* Q0 mattawa,PI* Q0* ourem</td><td colspan="1" rowspan="1">dupT/dupT</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">dupT/T</td><td colspan="1" rowspan="1">+ -</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">-/-</td></tr><tr><td colspan="1" rowspan="1">c.1158dupC@</td><td colspan="1" rowspan="1">PI* Q0 clayton,PI* Q0 saarbruecken</td><td colspan="1" rowspan="1">dupC/dupC</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">dupC/C</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-1-</td></tr><tr><td colspan="1" rowspan="1">c.1178C&gt;T</td><td colspan="1" rowspan="1">PI* M heerlen</td><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">+/+</td><td colspan="1" rowspan="1">T/C</td><td colspan="1" rowspan="1">+/-</td><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">-/-</td></tr></table>

<table><tr><td>Refers to the specific positions in RefSeq: NM_00112770.1of the National Center for Biotechnology Information (NCBI) reference human genome. veeoas T</td></tr><tr><td>mnaloee M a t </td></tr><tr><td>PI* Q0 in the case of a null allele (example: PI* Q0 granite falls). e  oiy h</td></tr><tr><td>heterozygosity; -/-, absence of the associated variant allele  *S PZ  *M/ S, PI* M/ Z PI* SZ, P*S/ S  </td></tr><tr><td>Z/PI* Z), cover more than 95% of variant alleles identified in the SERPINA1 gene worldwide. @u .  1 n1 debeb wmoan GyT ak</td></tr></table>

In addition to the allele genotype results, the A1AT Genotyping Test ANALYSIS SOFTWARE also provides Sample Result for each sample based on the specific combination of those alleles:

<table><tr><td rowspan=1 colspan=1>Sample Result</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>M/M</td><td rowspan=1 colspan=1>Common genotype. Samples with an absence (-/-) result for all theallelic variants evaluated</td></tr><tr><td rowspan=1 colspan=1>M/Present Variant allele</td><td rowspan=1 colspan=1>Samples with only one present result for a given associated variantallele in heterozygosity (+/-)</td></tr><tr><td rowspan=1 colspan=1>Present Variant allele/Present Variant allele</td><td rowspan=1 colspan=1>Samples with presence of one associated variant allele inhomozygosity (+/+)</td></tr><tr><td rowspan=1 colspan=1>Combination of thoseboth allelic variants</td><td rowspan=1 colspan=1>Samples with a present result in heterozygosity (+/-) for twodifferent associated variant alleles</td></tr><tr><td rowspan=1 colspan=1>Unknown^</td><td rowspan=1 colspan=1>Any result not indicative of a known variant combination</td></tr><tr><td rowspan=1 colspan=2>Unknown&quot; results correspond to a highly infrequent combination of genetic mutations that have not beenreported in the literature and that could give rise to more than two present (+) results for a given sample.</td></tr></table>

After allele genotype and Sample Result determination, the A1AT Genotyping Test ANALYSIS

SOFTWARE allows for the generation of pdf and/or .csv reports summarizing those results. Two different pdf reports can be generated: one report per sample (individual report) or one report per run or runs (global report).

MFI values that fall outside of the expected range of intensities are returned as “Invalid Test”. The proximity between the allelic variants $\mathrm { P I } ^ { * } \mathrm { M }$ malton (c.226_228delTTC) and $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama ( $\mathrm { c } . 2 3 0 \mathrm { C } { > } \mathrm { T } )$ , can lead to a decrease in the hybridization signal under the following conditions resulting in a ‘no-call’: When $\mathrm { P I } ^ { * } \mathrm { M }$ malton is homozygous (delTTC/ delTTC) then $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama cannot be detected. Conversely, when the genotype for allelic variant $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama is homozygous (T/T) then $\mathrm { P I } ^ { * } \mathrm { M }$ malton cannot be detected. The software generates an Invalid Test result for these particular cases as mentioned in the “Assay Limitations” of the labeling. The potential for hybridization interference is minimal because the possibility of having two disease associated variants in close proximity in the same sample is rare.

When $\mathrm { P I } ^ { * } \mathrm { M }$ malton is heterozygous, the normal genotype for $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama is correctly detected by the A1AT Genotyping Test. However, the A1AT Genotyping Test would not adequately genotype samples that are compound heterozygous for $\mathrm { P I } ^ { * } \mathrm { \mathbf { M } }$ malton and $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama (i.e. $+ / -$ for $\mathrm { P I } ^ { * } \mathrm { M }$ malton and $+ / -$ for $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama). Samples with mutations of both these allelic variants in the same allele have not been evaluated since such mutations have not been described in the literature. Based on theoretically expected results, the labeling indicates that “for a sample compound heterozygous for $\mathrm { P I } ^ { * } \mathrm { M }$ malton and $\mathrm { P I ^ { * } }$ S iiyama, a homozygous $+ / +$ result for either $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama or $\mathrm { P I } ^ { * } \mathrm { \mathbf { M } }$ malton is provided by A1AT Genotyping Test. Either Sample Result (M malton/S iiyama, M malton/ M malton or S iiyama/ S iiyama) is associated to a phenotype of severe A1AT deficiency”.

Results for a sample are deemed invalid (“Invalid Test”) when at least one of the allelic variants genotyping results cannot be called due to "Indeterminate genotype”, “Low signal intensity” or “Low bead count”. Any Invalid Test will be repeated once. Whether the repeat test yields a valid or invalid result, testing is considered complete.

# M. Performance Characteristics:

# 1. Analytical performance:

The results of all studies met the manufacturer’s pre-determined acceptance criteria.

a. Precision/Reproducibility:

Site-to-Site Reproducibility:

To evaluate total variability of the A1AT Genotyping Test, a reproducibility study was conducted at three sites (LifeShare Blood Centers and Progenika Inc. in the USA, and the Center for Diagnosis of Inherited Alpha1-antitrypsin Deficiency, University of Pavia, in Italy). At each site, two operators performed one run per day across three non-consecutive days (three runs per operator and six runs per site) over at least a twenty-day period, using one lot of Taq polymerase, one lot of DNA extraction reagents and one lot of A1AT Genotyping Test. Within a given run, the sample panel was tested in duplicate. The reproducibility of the DNA extraction steps for the DBS type of samples was also evaluated in this study. The sample panel consisted of: five whole blood samples of selected volunteer donors representing the $\mathrm { P I ^ { * } }$

M/PI\* S, PI\* M/PI\* Z, PI\* S/PI\* Z, $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes collected in DBS, 11 archived genomic DNA samples covering the heterozygous genotype of all 14 allelic variants included in the A1AT Genotyping Test, and a synthetic DNA sample (HET2) included to represent the presence of the M/M iiyama allelic variant not covered by the genomic samples. The 12 DNA samples were prepared at Progenika and shipped along with the 5 DBS samples to each site in randomized, blinded, panels.

For each genomic DNA sample and the synthetic DNA sample, a total of 36 replicates were obtained: three sites, two operators, three runs per operator and samples run in replicates of two; $3 \mathrm { ~ x ~ } 2 \mathrm { ~ x ~ } 3 \mathrm { ~ x ~ } 2 = 3 6 .$ . For the five DBS samples, the DNA extraction was performed at each site according to the two methods specified in the A1AT Genotyping Test package insert; hence, a total of 72 replicates of each DBS sample were obtained over the course of the study. The two methods are commercial lysis and neutralization solutions method (LN) and the in house-made BSA/Azide (BA) buffer method. The expected results for 16 of the 17 samples were confirmed by bidirectional sequencing at Progenika. For the HET2 sample, the sequencing results were provided by Integrated DNA Technologies (IDT, USA). The genotype call, expressed as presence or absence of associated allele for each allelic variant, and Sample Result, the combination of the alleles detected, for all samples and replicates were determined.

A summary of results of the reproducibility study and $\%$ agreement between sites by Sample Results is shown in the table below. Three sample replicates produced invalid test results due to processing errors related to improper sealing of the PCR plate (both replicates) and improper removal of the adhesive film from the plate (one replicate). Upon repeat testing, all alleles were called correctly as compared to bidirectional sequencing results. There were no invalid test results.

<table><tr><td colspan="3" rowspan="1">DNA Samples</td><td colspan="4" rowspan="1">Correct Sample Results</td></tr><tr><td colspan="1" rowspan="1">Types</td><td colspan="1" rowspan="1">Extractionmethods</td><td colspan="1" rowspan="1">Allelic VariantsTested</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Agreementacross sites</td></tr><tr><td colspan="1" rowspan="1">DBS1</td><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">PI* M/PI* S</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS2</td><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">PI* M/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS3</td><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">PI* S/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS4</td><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">PI* S/PI* S</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS5</td><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">PI* Z/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS1</td><td colspan="1" rowspan="1">BA</td><td colspan="1" rowspan="1">PI* M/PI* S</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS2</td><td colspan="1" rowspan="1">BA</td><td colspan="1" rowspan="1">PI* M/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS3</td><td colspan="1" rowspan="1">BA</td><td colspan="1" rowspan="1">PI* S/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS4</td><td colspan="1" rowspan="1">BA</td><td colspan="1" rowspan="1">PI* S/PI* S</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">DBS5</td><td colspan="1" rowspan="1">BA</td><td colspan="1" rowspan="1">PI* Z/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* F</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Q0 west</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* Q0 granitefalls/PI* Z</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Q0mattawa</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M malton/PI* M heerlen</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Q0bellingham</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* Z/PI* Q0clayton</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* Z/PI* P lowell</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* Z/PI* I</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Mmalton</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">gDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Mprocida</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="1" rowspan="1">sDNA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">PI* M/PI* Miiyama</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">12/12(100%)</td><td colspan="1" rowspan="1">36/36(100%)</td></tr><tr><td colspan="7" rowspan="1">gDNA: genomic DNA,sDNA: synthetic DNA, LN: commercial lysis and neutralization solutions method,</td></tr></table>

BA:home-made BSA/Azidebuffer method

Lot-to-Lot Variability:

To evaluate the reproducibility of the reagent lots for the A1AT Genotyping Test, a study was conducted by two operators who tested DNA samples from five patient specimens representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes in triplicate using three reagent lots and two instruments. Each operator performed one run per lot, alternating between two Luminex instruments, on six non-consecutive days over a period of at least 20 days. A total of 360 replicates were obtained: five samples x two replicates x three lots x six days x two operators; $5 \times 2 \times 3 \times 6 \times 2 = 3 6 0$ . Genotype calls were determined for all samples and replicates for all reagent lots and the percent of correct calls was calculated. None of the samples produced invalid test results that required repeating. There are no ‘no-call’ or unknown results. The percent of correct Sample Results for the $\mathrm { P I ^ { * } M / P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes is $100 \%$ for all reagent lots. For the $\mathrm { P I ^ { * } M / P I ^ { * } }$ S genotypes, one incorrect Sample Result is obtained (a M/M Sample Result was provided for an $\mathrm { P I ^ { * } M / P I ^ { * } \Sigma }$ sample), giving rise to an overall correct rate of $9 8 . 6 \%$ for Lot 3.

<table><tr><td rowspan=2 colspan=1>Allelic VariantsTested</td><td rowspan=1 colspan=4>Correct Sample Results</td></tr><tr><td rowspan=1 colspan=1>Reagent Lot 1</td><td rowspan=1 colspan=1>Reagent Lot 2</td><td rowspan=1 colspan=1>Reagent Lot 3</td><td rowspan=1 colspan=1>All Reagent Lots</td></tr><tr><td rowspan=1 colspan=1>PI* M/PI* S</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>23/24 (95.8%)</td><td rowspan=1 colspan=1>71/72 (98.6%)</td></tr><tr><td rowspan=1 colspan=1>PI* M/PI* Z</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>72/72 (100%)</td></tr><tr><td rowspan=1 colspan=1>PI* S/PI* Z</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>72/72 (100%)</td></tr><tr><td rowspan=1 colspan=1>PI* S/PI* S</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>24/24 (100%)</td><td rowspan=1 colspan=1>72/72 (100%)</td></tr><tr><td rowspan=1 colspan=1>PI* Z/PI* Z</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>72/72 (100%)</td></tr></table>

DNA Extraction Variability:

To assess the impact of different DNA extraction methods and types of sample collection matrix on the performance of the A1AT Genotyping Test, 12 donor samples collected both in DBS and in blood tubes with K2 EDTA anti-coagulant were analyzed. Two DNA extraction methods, the commercial lysis and neutralization solutions method and the home-made BSA/Azide buffer method, were tested for samples collected in DBS, and one DNA extraction, the QIAamp DNA Blood Mini Kit method, was tested for samples collected in K2 EDTA blood extraction tubes. Additionally, the effect of the use of different sample collection matrices on the performance of the test was evaluated using two different kinds of filter paper cards for DBS sample collection, the Whatman 903 Neonate Blood Collection Cards (K100682) and PerkinElmer 226 filter paper cards (K121864). The test samples include six wild-type (M/M) and six allelic variants representing $\mathrm { P I ^ { * } M / P I ^ { * } S }$ (two samples), $\mathrm { P I ^ { * } M / P I ^ { * } } Z$ (one sample), $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } } Z$ (one sample), $\mathrm { P I ^ { * } S / P I ^ { * } S }$ (one sample) and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ (one sample). The three extraction methods were tested using the same DNA polymerase enzyme lot and A1AT Genotyping Test lot by two operators across three non-consecutive days at a single site. DNA extraction for DBS was carried out from a $3 \mathrm { m m }$ (1/8 inch) punch per extraction and the extracted DNA, if not being processed immediately, was stored at $2 ~ 8 \%$ . Each operator performed one run with the extracted DNA per day in duplicate for a total of 12 replicates (3 extractions x 1 replicate x 1 run x 3 days $_ { \textrm { X 2 } }$ operators) per sample. The test samples were also analyzed by bi-directional sequencing. Variant calls were determined for all samples and replicates for all conditions and the percent of correct Sample Results calculated. There were no invalid runs and sample repetition was not needed in this study. There were also no incorrect calls as compared to bidirectional sequencing results and no ‘no-call’ or unknown results. The percent of correct Sample Results across all the samples, conditions and replicates was $100 \%$ .

<table><tr><td rowspan=3 colspan=1>DNA Extraction Methods</td><td rowspan=1 colspan=6>Correct Sample Results</td></tr><tr><td rowspan=1 colspan=3>Allelic Variants Tested</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>PI* M/PI*M(6 samples)</td><td rowspan=1 colspan=1>PI* M/PI*S(2 samples)</td><td rowspan=1 colspan=1>PI* M/PI*Z(1 sample)</td><td rowspan=1 colspan=1>PI* S/PI*Z(1 sample)</td><td rowspan=1 colspan=1>PI* S/PI*S(1 sample)</td><td rowspan=1 colspan=1>PI* Z/PI*Z(1 sample)</td></tr><tr><td rowspan=1 colspan=1>Whole blood samples collectedin DBS (Whatman 903) andextracted with the commerciallysis and neutralization solutionmethod</td><td rowspan=1 colspan=1>72/72(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Whole blood samples collectedin DBS (Whatman 903) andextracted with the home-madeBSA/Azide buffer method</td><td rowspan=1 colspan=1>72/72(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Whole blood samples collectedin DBS (PE226) and extractedwith the commercial lysis andneutralization solution method</td><td rowspan=1 colspan=1>72/72(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Whole blood samples collectedin DBS (PE226) and extractedwith the home-madeBSA/Azide buffer method</td><td rowspan=1 colspan=1>72/72(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr><tr><td rowspan=1 colspan=1>Whole blood samples collectedin K2 EDTA tube and extractedwith the QIAamp DNA BloodMini Kit.</td><td rowspan=1 colspan=1>72/72(100%)</td><td rowspan=1 colspan=1>24/24(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td><td rowspan=1 colspan=1>12/12(100%)</td></tr></table>

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Reagent stability:

The shelf-life stability of the A1AT Genotyping Test was evaluated using three lots of each set of reagents that were stored at $2 \mathrm { - } 8 \mathrm { \mathrm { ' C } }$ for 3, 6, 9, 10, 12, 13, 15 and 16 months after manufacturing. At each time point of evaluation, 20 replicates of each of the five genomic DNA samples representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } }$ $Z / \mathrm { P I } ^ { * } Z$ genotypes were assayed per reagent lot. Sample Results from each time point were compared against those obtained at time point $\mathrm { { T _ { 0 } } }$ . For all time points tested for each reagent lot, all sample replicates for each variant provided $100 \%$ correct Sample Results, as compared to the corresponding Sample Results from $\mathrm { { T _ { 0 } } }$ and bidirectional sequencing. Results demonstrated that the A1AT Genotyping Test is stable for at least 15 months when stored at $2 -$ $8 \mathrm { ^ \circ C }$ .

The open-vial (in-use) stability of the A1AT Genotyping Test was evaluated using sets of reagents from one production lot that had been opened for 3, 6, 7, 9 and 10 months and stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ . The impact of having the A1AT Beads Master Mix at room temperature during plate dispensing was also evaluated in this study by keeping this component at room temperature for at least 30 minutes before plate dispensing at each time point. At each time point, twenty replicates of each of the five genomic DNA samples representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M / P I ^ { * } }$ Z, $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes were assayed. The percent of correct Sample Results was $100 \%$ for all DNA samples and replicates at every time point as compared to bidirectional sequencing results. Results demonstrated that the A1AT Genotyping Test is stable up to 9 months after the reagent vials were first opened and that the A1AT Beads Master Mix component can be maintained at room temperature $( 1 8 - 2 5 ^ { \circ } \mathrm { C } )$ for up to 30 minutes without affecting assay performance.

# Specimen stability:

Genomic DNA from a total of $1 2 \mathrm { K } 2$ EDTA anticoagulated whole blood samples, representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes, was extracted after 10 to 43 days of storage at $2 8 \mathrm { { \% } }$ after collection using the QIAamp DNA Blood Mini Kit and tested with the A1AT Genotyping Test. Results demostratedthat all samples provided $100 \%$ correct Sample Results. Progenika claims that whole blood samples collected in K2 EDTA can be maintained up to 24 days at $2 8 \mathrm { { \% } }$

To evaluate the stability of DBS samples, five whole blood samples, representing the $\mathrm { P I ^ { * } }$ ${ \mathrm { M } } / { \mathrm { P I } ^ { * } } \mathrm { S }$ , $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes, were collected in DBS and stored at room temperature away from direct sunlight prior to DNA extraction using the two methods evaluated in the external reproducibility study. Results show that samples extracted from 36 to 202 days (or 1.2 months to 6.7 months) after collection provided $100 \%$ correct Sample Results. Progenika claims 180 days (6 months) of stability for the DBS samples maintained at room temperature away from direct sunlight.

# d. Detection limit:

A lower limit of detection (LLoD) study was performed with the aim of determining the lowest concentration of DNA at which $9 5 \%$ of the sample replicates resulted in correct sample results by using the Probit Approach described in CLSI EP17-A2. DNA samples from five patient specimens representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M / P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ , $\mathrm { P I ^ { * } S / P I ^ { * } : }$ S and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } }$ Z genotypes were each diluted at nine different DNA concentration levels (0.16, 0.1, 0.08, 0.04, 0.02, 0.0133, 0.0067, 0.0044 and $0 . 0 0 3 3 ~ \mathrm { n g / \mu L } )$ and tested by one operator 20 times using one thermocycler, one Luminex instrument and one polymerase enzyme lot. A total of two different reagent lots were used throughout the study.  For each reagent lot, a total of 20 replicates for each sample or a total of 100 replicates for all five samples at each DNA level were obtained. The rates of correct genotype calls and Sample Results were determined for all replicates per DNA input level and per reagent lot.

The percent of correct results are mathematically converted into cumulative normal probability units (probits) and fitted by a regression model versus their respective DNA input level, in order to calculate the hit rate of 0.95 to be reported as the LLoD of each reagent lot used in the study. The LLoD obtained was $0 . 0 2 2 8 \mathrm { n g / \mu L }$ $9 5 \%$ CI: 0.0183  0.0308) for reagent lot 1 and $0 . 0 3 1 0 \mathrm { n g / \mu L }$ $9 5 \%$ CI: 0.0242  0.0429) for reagent lot 2. Based on the probit analysis of correct Sample Results for each reagent lot, the A1AT Genotyping Test LLoD is $0 . 0 3 1 0 \mathrm { n g / \mu L }$ corresponding to a DNA input of $0 . 1 5 5 ~ \mathrm { n g }$ .

<table><tr><td rowspan=1 colspan=2>DNA Samples</td><td rowspan=1 colspan=2>Correct Sample Results</td></tr><tr><td rowspan=1 colspan=1>InputLevel (ng)</td><td rowspan=1 colspan=1>ConcentrationLevel (ng/μL)</td><td rowspan=1 colspan=1>Reagent Lot 1</td><td rowspan=1 colspan=1>Reagent Lot 2</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>100/100 (100%)</td><td rowspan=1 colspan=1>100/100 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>100/100 (100%)</td><td rowspan=1 colspan=1>100/100 (100%)</td></tr><tr><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>99/100 (99%)</td><td rowspan=1 colspan=1>98/100 (98%)</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>99/100 (99%)</td><td rowspan=1 colspan=1>94/100 (94%)</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>95/100 (95%)</td><td rowspan=1 colspan=1>94/100 (94%)</td></tr><tr><td rowspan=1 colspan=1>0.067</td><td rowspan=1 colspan=1>0.0133</td><td rowspan=1 colspan=1>86/100 (86%)</td><td rowspan=1 colspan=1>86/100 (86%)</td></tr><tr><td rowspan=1 colspan=1>0.033</td><td rowspan=1 colspan=1>0.0067</td><td rowspan=1 colspan=1>74/100 (74%)</td><td rowspan=1 colspan=1>69/100 (69%)</td></tr><tr><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.0044</td><td rowspan=1 colspan=1>56/100 (56%)</td><td rowspan=1 colspan=1>67/100 (67%)</td></tr><tr><td rowspan=1 colspan=1>0.0165</td><td rowspan=1 colspan=1>0.0033</td><td rowspan=1 colspan=1>52/100 (52%)</td><td rowspan=1 colspan=1>46/100 (46%)</td></tr></table>

An upper limit of detection (ULoD) study was carried performed using two reagent lots to determine the highest DNA concentration at which $9 5 \%$ of correct calls are obtained. A total of twenty replicates of each sample were tested with each reagent lot at two DNA concentration levels, $1 5 0 ~ \mathrm { { n g / \mu L } }$ and $2 0 0 ~ \mathrm { { n g / \mu L } }$ . Results demonstrated that that the ULoD of the A1AT Genotyping Test assay is above $2 0 0 ~ \mathrm { { n g / \mu L } }$ (an input of $1 0 0 0 \mathrm { n g }$ of DNA).

<table><tr><td rowspan=1 colspan=2>DNA Samples</td><td rowspan=1 colspan=2>Correct Sample Result</td></tr><tr><td rowspan=1 colspan=1>InputLevel (ng)</td><td rowspan=1 colspan=1>ConcentrationLevel (ng/μL)</td><td rowspan=1 colspan=1>Reagent Lot 1</td><td rowspan=1 colspan=1>Reagent Lot 2</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>100/100 (100%)</td><td rowspan=1 colspan=1>100/100 (100%)</td></tr><tr><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>100/100 (100%)</td><td rowspan=1 colspan=1>100/100 (100%)</td></tr></table>

# e. Analytical specificity:

# i. Endogenous Interference:

To evaluate how potentially interfering substances may impact the performance of the A1AT Genotyping Test, the effects of hemoglobin $( 5 0 0 \mathrm { m g / d L } )$ , unconjugated bilirubin (20 $\mathrm { m g / d L }$ dissolved in NaOH) and triglycerides $( 3 0 0 \mathrm { m g / d L } )$ ) were assessed. Five whole blood samples collected in blood tubes with K2 EDTA from selected volunteer donors representing the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ , $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } } Z$ , $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } } Z / \mathrm { P I ^ { * } } Z$ genotypes were evaluated. The substances were spiked into each K2 EDTA whole blood sample, and the spiked blood samples were then processed according to the standard procedure, along with the corresponding set of unspiked blood samples. In addition, the effect of an K2 EDTA short drawn was also examined by using an K2 EDTA tube with 1/5 of the total volume (a 5x final concentration of K2 EDTA in the sample). For K2 EDTA whole blood samples (spiked and unspiked), DNA extraction was carried out according to the QIAamp DNA Blood Mini Kit. For DBS samples, Whatman 903 cards were spotted

with the spiked and unspiked K2 EDTA whole blood and DNA extraction was carried out using the commercial lysis and neutralization solution method. DBS without K2 EDTA (spotted directly from whole blood) acted as a control for DBS with K2 EDTA blood samples. A total of five spiked and one unspiked conditions for the K2 EDTA whole blood samples and five spiked and two unspiked conditions for the DBS samples were tested. DNA was extracted in triplicate for each condition. The variant calls were compared across the spiked and unspiked samples to determine if the substances tested lead to alterations in the Sample Results. None of the samples produced invalid test results that required repeating. There are no ‘no-call’ or unknown results. The percent of correct Sample Results for all samples and replicates is $100 \%$ as compared to bidirectional sequencing. The results indicate that none of the potentially interfering substances evaluated affects assay performance at the concentrations tested.

# ii. Cross-reactivity:

To assure the specificity of the four primer pairs (forward and reverse) for PCR amplification and the 28 oligonucleotide probes for allele-specific primer extension reactions, BLAST analyses were conducted using the Primer-BLAST tool and the Standard Nucleotide BLAST tool to evaluate whether primer and probe sequences could hybridize to other regions in the human genome apart from the expected sequences. Non-specific amplicons are not predicted for any of the potentially cross-reactive primer pairs, in any of the A1AT Genotyping Test multiplex PCR reactions. All probes bind to DNA regions with no described variants or reported polymorphisms in the target population that could potentially interfere with assay results.

# iii. Cross-contamination:

A cross contamination study was performed with one lot of A1AT Genotyping Test to evaluate cross-contamination and/or carryover occurs between samples or between samples and the Negative Control during processing in 384-well PCR and 96-well hybridization plates.

For the Sample-to-Sample cross-contamination evaluation, replicates of genomic DNA samples representing the $\mathrm { P I ^ { * } M / P I ^ { * } } ;$ S, $\mathrm { P I ^ { * } }$ M/PI\* Z, PI\* S/PI\* Z, $\mathrm { P I ^ { * } S / P I ^ { * } S }$ and $\mathrm { P I ^ { * } }$ $Z / \mathrm { P I } ^ { * } Z$ genotypes were tested in the PCR plate, alternating between high $( 2 0 0 ~ \mathrm { n g / \mu L } )$ and low $( 0 . 1 \ \mathrm { n g / \mu L } )$ DNA input levels. None of the samples produced invalid test results that required repeating. There were no incorrect calls as compared to bidirectional sequencing results and no ‘no-call’ or unknown results. The percent of correct Sample Result calls for all five samples at low DNA concentration and at high DNA concentration was $100 \%$ as compared to bidirectional sequencing.

For the Sample-to-Negative Control cross-contamination evaluation, replicates of the same panel samples at the selected high DNA concentration $( 2 0 0 ~ \mathrm { n g / \mu L } )$ were alternated with Negative Control samples (DNase free water). The complete checkerboard pattern included 47 alternating pairs for a total of 94 reactions (47 high samples and 47 negative control samples). Results from these studies demonstrated that no carryover events occurred. The following precautionary note ‘Be careful in the dispensation of the DNA samples in the PCR step to avoid contamination of the adjacent well’ is included in the Package Insert.

f. Assay cut-off: Not applicable.

2. Comparison studies:

a. Method comparison:

Accuracy of the A1AT Genotyping Test was assessed by evaluating a sample panel representing all heterozygous and homozygous genotypes of each allelic variant and covering all the possible results provided by the assay. The sample panel consists of 66 clinical samples (12 clinical K2 EDTA anticoagulated blood samples and 54 archived clinical genomic DNA samples) and 46 genomic DNAs extracted from cell lines. The sample panel was supplemented with recombinant synthetic DNA samples representing all the homozygous genotypes for the A1AT Genotyping Test allelic variants. The allelic variant coverage of the sample panel is summarized in the table below.

<table><tr><td rowspan=3 colspan=1>Allelic Variant</td><td rowspan=3 colspan=1>Most CommonAssociated Allele</td><td rowspan=1 colspan=6># of Genomic DNASamples</td><td rowspan=2 colspan=3># ofSyntheticDNASamples</td><td rowspan=2 colspan=3># of AllDNASamples</td></tr><tr><td rowspan=1 colspan=3>Clinical</td><td rowspan=1 colspan=3>Cell lines</td></tr><tr><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/+</td></tr><tr><td rowspan=1 colspan=1>c.187C&gt;T</td><td rowspan=1 colspan=1>PI* I</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>c.194T&gt;C</td><td rowspan=1 colspan=1>PI* M procida</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>c.226_228 delTTC</td><td rowspan=1 colspan=1>PI* M malton</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>c.230C&gt;T</td><td rowspan=1 colspan=1>PI* S iiyama</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>c.552delC</td><td rowspan=1 colspan=1>PI* Q0granite falls</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.646+1G&gt;T</td><td rowspan=1 colspan=1>PI* Q0 west</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.721A&gt;T</td><td rowspan=1 colspan=1>PI* Q0 bellingham</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.739C&gt;T</td><td rowspan=1 colspan=1>PI* F</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.839A&gt;T</td><td rowspan=1 colspan=1>PI* P lowell</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.863A&gt;T</td><td rowspan=1 colspan=1>PI* S</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>c.1096G&gt;A</td><td rowspan=1 colspan=1>PI* Z</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>c.1130dupT</td><td rowspan=1 colspan=1>PI* Q0 mattawa</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>c.1158dupC</td><td rowspan=1 colspan=1>PI* Q0 clayton</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>c.1178C&gt;</td><td rowspan=1 colspan=1>PI* M heerlen</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td></tr></table>

As sequencing is a sensitive technique for A1AT variant detection and it is able to identify specific rare and null alleles, which are not detected by isoelectrofocusing techniques, results obtained with the A1AT Genotyping Test are compared to bi-directional Sanger sequencing instead of the HYDRAGEL 18 A1AT ISOFOCUSING kit. All the DNA samples were sequenced. For the synthetic DNA samples, the sequencing results were provided by the supplier (IDT). None of the samples produced invalid test results that required repeating. The concordance for both genotype $( + + , + / - , - / - )$ and Sample Results as shown in the tables below is $100 \%$ . There are no incorrect calls as compared to bidirectional sequencing results and no ‘no-call’ or unknown results. The study showed that A1AT Genotyping Test is able to correctly genotype all samples analyzed.

<table><tr><td rowspan=2 colspan=1>Allelic Variant</td><td rowspan=2 colspan=1>Most CommonAssociated Allele</td><td rowspan=1 colspan=2>Allelic Variant Present(+/- and +/+)</td><td rowspan=1 colspan=2>Allelic Variant Absent(-/-)</td><td rowspan=2 colspan=1>%Agreement</td></tr><tr><td rowspan=1 colspan=1>Bi-directionalSanger</td><td rowspan=1 colspan=1>A1ATGenotypingTest</td><td rowspan=1 colspan=1>Bi-directionalSanger</td><td rowspan=1 colspan=1>A1ATGenotypingTest</td></tr><tr><td rowspan=1 colspan=1>c.187C&gt;T</td><td rowspan=1 colspan=1>PI* I</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.194T&gt;C</td><td rowspan=1 colspan=1>PI* M procida</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.226_228delTTC</td><td rowspan=1 colspan=1>PI* M malton</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.230C&gt;T</td><td rowspan=1 colspan=1>PI* S iiyama</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.552delC</td><td rowspan=1 colspan=1>PI* Q0 granite falls</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.646+1G&gt;T</td><td rowspan=1 colspan=1>PI* Q0 west</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.721A&gt;T</td><td rowspan=1 colspan=1>PI* Q0 bellingham</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.739C&gt;T</td><td rowspan=1 colspan=1>PI* F</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.839A&gt;T</td><td rowspan=1 colspan=1>PI* P lowell</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.863A&gt;T</td><td rowspan=1 colspan=1>PI* S</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.1096G&gt;A</td><td rowspan=1 colspan=1>PI* Z</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.1130dupT</td><td rowspan=1 colspan=1>PI* Q0 mattawa</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.1158dupC</td><td rowspan=1 colspan=1>PI* Q0 clayton</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>c.1178C&gt;</td><td rowspan=1 colspan=1>PI* M heerlen</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td colspan="1" rowspan="1">ExpectedSample Results^</td><td colspan="1" rowspan="1">Bi-directionalSanger</td><td colspan="1" rowspan="1">A1ATGenotyping Test</td><td colspan="1" rowspan="1">% Agreement</td><td colspan="1" rowspan="1">LCL(95%CI)</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI* I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15.8 22.4</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI* M procida</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">39.847.3</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI* MI malton</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">59.065.2</td></tr><tr><td colspan="1" rowspan="1">PI* Z/PI*Q0granite falls</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.55.0</td></tr><tr><td colspan="1" rowspan="1">PI*M/PI*Q0 west</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15.822.4</td></tr><tr><td colspan="1" rowspan="1">PI*M/PI* F</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">39.847.3</td></tr><tr><td colspan="1" rowspan="1">PI*Z/PI* P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15.8 22.4</td></tr><tr><td colspan="1" rowspan="1">PI*M/PI* S</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">87.789.9</td></tr><tr><td colspan="1" rowspan="1">PI*M/PI* Z</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">78.281.9</td></tr><tr><td colspan="1" rowspan="1">PI*M/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15.8 22.4</td></tr><tr><td colspan="1" rowspan="1">PI*Z/PI* Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI* M heerlem</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* F/PI* S</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* F/PI* Z</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* I/PI* Q0 bellingham</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* S/PI* I</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* M malton/PI* M heerlen</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* M/PI* P lowell</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">29.2 36. 8</td></tr><tr><td colspan="1" rowspan="1">PI* S/PI* M procida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI* S/PI* Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="1" rowspan="1">PI*S/PI* Z</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">63.168.8</td></tr><tr><td colspan="1" rowspan="1">PI*Z/PI* M malton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.55.0</td></tr><tr><td colspan="1" rowspan="1">PI*Z/PI* M procida</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">29.236.8</td></tr><tr><td colspan="1" rowspan="1">PI*Z/PI* I</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">29.236.8</td></tr><tr><td colspan="1" rowspan="1">PI* S/PI* S</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">63.168.8</td></tr><tr><td colspan="1" rowspan="1">PI* Z/PI* Z</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">59.065.2</td></tr><tr><td colspan="1" rowspan="1">PI* Q0 mattawa /PI* Q0mattawa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.55.0</td></tr><tr><td colspan="1" rowspan="1">PI* Q0 clayton/PI* Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">15.8 22.4</td></tr><tr><td colspan="1" rowspan="1">PI* M heerlen/PI* M heerlen</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">2.5 5.0</td></tr><tr><td colspan="5" rowspan="1">^Synthetic DNA samples have been removed from this summary.LCL, Lower Confidence Level of the two-sided 95% Confidence interval for the Clopper-Pearson method</td></tr></table>

b. Matrix comparison:

Not required because analytical performance was demonstrated in both claimed matrices, K2 EDTA and DBS.

3. Clinical studies:

a. Clinical Sensitivity and Specificity: Not applicable

b. Other clinical supportive data:

The manufacturer provided literature support for the clinical interpretation of the identified genotypes. The literature references for the clinical relevance of each of the allelic variant included in A1AT Genotyping Test and their associated alleles are detailed in the following table. At least two bibliographic references are included per allelic variant.

<table><tr><td colspan="2" rowspan="1">Mutation Panel</td><td colspan="1" rowspan="2">A1AT ProteinActivity</td><td colspan="1" rowspan="2">Reference</td></tr><tr><td colspan="1" rowspan="1">A1ATAllele</td><td colspan="1" rowspan="1">AssociatedAllele</td></tr><tr><td colspan="1" rowspan="1">c.187C&gt;T</td><td colspan="1" rowspan="1">PI* I</td><td colspan="1" rowspan="1">Impairedsecretionandmild plasmadeficiency</td><td colspan="1" rowspan="1">Graham A, Kalsheker N, Newton C, Bamforth F, Powell S,Markham A. Molecular characterisation of three alpha-1-antitrypsin deficiency variants: proteinase inhibitor (Pi)nullcardiff (Asp256-&gt; Val); PiMmalton,(Phe51-&gt; deletion)and Pil (Arg39-&gt; Cys). Hum Genet. 1989. 84:5558.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Seri M, Magi B, Cellesi C, Olia PM, Renieri A, De MarchiM. Molecular characterization of the P and I variants ofalpha 1-antitrypsin. Int J Clin Lab Res.1992. 22:119121.</td></tr><tr><td colspan="1" rowspan="2">c.194T&gt;C</td><td colspan="1" rowspan="2">PI* Mprocida</td><td colspan="1" rowspan="2">Impairedsecretion(degradation)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Brantly M1, Nukiwa T, Crystal RG. Molecular basis ofalpha-1-antitrypsin deficiency. Am J Med. 1988. 84:1331.</td></tr><tr><td colspan="1" rowspan="1">Takahashi H, Nukiwa T, Satoh K, Ogushi F, Brantly M,Fells G, Stier L, Courtney M, Crystal RG. Characterizationof the gene and protein of the alpha 1-antitrypsin"deficiency" allele Mprocida. J Biol Chem. 1988.263:1552815534.</td></tr><tr><td colspan="1" rowspan="7">c.226_228delTTC</td><td colspan="1" rowspan="3">PI* Mmalton</td><td colspan="1" rowspan="3">Impairedsecretion(polymerization)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Cox, D. W. A new deficiency allele of alpha1-antitrypsin: PiMmalton. Pp. 375-378 in H. Peters, ed. Protides of thebiological fluids. Vol. 23. Pergammon, Oxford. Alpha-l-antitrypsin deficiency. In C. R. Scriver, A. L. Beaudet, W. S.Sly, and D. Valle, eds. The metabolic basis of inheriteddisease. 1976. 6th ed. McGraw-Hill, New York.</td></tr><tr><td colspan="1" rowspan="1">Curiel DT, Holmes MD, Okayama H, Brantly ML,Vogelmeier C, Travis WD, Stier LE, Perks WH, Crystal RG.Molecular basis of the liver and lung disease associated withthe alpha-1-antitrypsin deficiency allele Mmalton. J BiolChem. 1989. 264:1393813945.</td></tr><tr><td colspan="1" rowspan="1">Graham A, Kalsheker N, Newton C, Bamforth F, Powell S,Markham A. Molecular characterisation of three alpha-1-antitrypsin deficiency variants: proteinase inhibitor (Pi)nullcardiff (Asp256-&gt;Val); PiMmalton,(Phe51-&gt;deletion)and Pil (Arg39-&gt;Cys). Hum Genet. 1989. 84:5558.</td></tr><tr><td colspan="1" rowspan="2">PI* Mpalermo</td><td colspan="1" rowspan="2">Impairedsecretion(polymerization)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Faber J.P, Poller W, Weidinger S, Kirchgesser M, SchwaabR, Bidlingmaier F, Olek K. Identification and DNASequence Analysis of 15 New alpha1- Antitrypsin Variants,Including Two PI*QO Alleles and One Deficient PI*MAllele. Am .J. Hum. Genet. 1994. 55:11131121.</td></tr><tr><td colspan="1" rowspan="1">Lang T, Mühlbauer M, Strobelt M, Weidinger S, HadornHB. Alpha-1-antitrypsin deficiency in children: liver diseaseis not reflected by low serum levels of alpha-1-antitrypsin - astudy on 48 pediatric patients. Eur J Med Res. 2005. 10:509514.</td></tr><tr><td colspan="1" rowspan="2">PI* Mnichinan</td><td colspan="1" rowspan="2">Impairedsecretion(polymerization)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Nakamura H, Ogawa A, Hisano S, Fukuma M, Tachibana N,Tsuda K. J. A family with a new deficient variant of alpha-1-antitrypsin PiM(Nichinan): with special reference todiastase-resistant, periodic acid-Schiff positive globules inthe liver cells. Soc. Intern. Med. 1980. 69: 4754.</td></tr><tr><td colspan="1" rowspan="1">Matsunaga E, Shiokawa I, Nakamura H, Maruyamaj T,Tsuda K, Fukumaki Y. Molecular Analysis of the Gene of</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">the alpha 1-Antitrypsin Deficiency Variant,Mnichinan. Am. J. Hum. Genet. 1990. 46:602612</td></tr><tr><td colspan="1" rowspan="2">c.230C&gt;T</td><td colspan="1" rowspan="2">PI* Siiyama</td><td colspan="1" rowspan="2">Impairedsecretion(polymerization)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Takabe K, Seyama K, Shinada H, Nouchi T, Miyahara Y,Nukiwa T, Miyake K,Tsukimoto K, Ichioka M, Marumo F.A new variant of alpha-1-antitrypsin deficiency (Siiyama)associated with pulmonary emphysema. Intern Med. 1992.31:702707.</td></tr><tr><td colspan="1" rowspan="1">Seyama K. State of alpha1-antitrypsin deficiency in Japan.Respirology. 2001. 6 Suppl: S3538.</td></tr><tr><td colspan="1" rowspan="2">c.552delC</td><td colspan="1" rowspan="2">PI* Q0granite falls</td><td colspan="1" rowspan="2">UndetectablemRNA</td><td colspan="1" rowspan="1">Nukiwa T, Takahashi H, Brantly M, Courtney M, CrystalRG. Alpha 1-Antitrypsin Null Granite Falls, a nonexpressingalpha 1-antitrypsin gene associated with aframeshift to stopmutation in a coding exon. J Biol Chem. 1987. 262: 11999112004.</td></tr><tr><td colspan="1" rowspan="1">Holmes M, Curiel D, Brantly M, Crystal RG.Characterization of the intracellularmechanism causing thealpha-1-antitrypsin Nullgranite falls deficiency state. Am RevRespir Dis. 1989 140:16621667.</td></tr><tr><td colspan="1" rowspan="2">c.646+1G&gt;T</td><td colspan="1" rowspan="2">PI* Q0west</td><td colspan="1" rowspan="2">Truncatedprotein/Degradation</td><td colspan="1" rowspan="1">Laubach VE, Ryan WJ, Brantly M. Characterization of ahuman alpha 1-antitrypsin null allele involving aberrantmRNA splicing. Hum Mol Genet. 1993. 2:10011005.</td></tr><tr><td colspan="1" rowspan="1">Lee J and Brantly M. Molecular mechanisms of alpha1-antitrypsin null alleles. Respir Med. 2000. 94 (supplement c).S7-S11.</td></tr><tr><td colspan="1" rowspan="2">c.721A&gt;T</td><td colspan="1" rowspan="2">PI* Q0bellingham</td><td colspan="1" rowspan="2">UndetectablemRNA</td><td colspan="1" rowspan="1">Satoh K, Nukiwa T, Brantly M, Garver RI Jr, Hofker M,Courtney M, Crystal RG. Emphysema associated withcomplete absence of alphal-antitrypsin in serum and thehomozygous inheritance [corrected] of a stop codon in analpha 1-antitrypsin coding exon. Am J Hum Genet. 1988.42:7783.</td></tr><tr><td colspan="1" rowspan="1">Fregonese L, Stolk J, Frants RR, Veldhuisen B. Alpha-1antitrypsin Null mutations and severity of emphysema.Respir Med. 2008.102:876884.</td></tr><tr><td colspan="1" rowspan="2">c.739C&gt;T</td><td colspan="1" rowspan="2">PI* F</td><td colspan="1" rowspan="2">Impairedsecretionandmild plasmadeficiency</td><td colspan="1" rowspan="1">Okayama H, Brantly M, Holmes M, Crystal RG.Characterization of the molecular basis of the alpha 1-antitrypsin F allele. Am J Hum Genet. 1991. 48:11541158.</td></tr><tr><td colspan="1" rowspan="1">Ringenbach MR, Banta E, Snyder MR, Craig TJ, IshmaelFT. A challenging diagnosis of alpha-1-antitrypsindeficiency: identification of a patient with a novel F/Nullphenotype. Allergy Asthma Clin Immunol. 2011. 7:18.</td></tr><tr><td colspan="1" rowspan="1">c.839A&gt;T</td><td colspan="1" rowspan="1">PI* Plowell</td><td colspan="1" rowspan="1">Impairedsecretion(degradation)</td><td colspan="1" rowspan="1">Faber JP, Weidinger S, Goedde HW, Ole K. The deficientalpha-1-antitrypsin phenotype PI P is associated with an A-to-T transversion in exon III of the gene. Am J Hum Genet.</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Mutation Panel</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>A1AT ProteinActivity</td></tr><tr><td rowspan=1 colspan=1>A1ATAllele</td><td rowspan=1 colspan=2>AssociatedAllele</td><td></td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>andmild plasmadeficiency</td><td rowspan=1 colspan=1>1989. 45:161163.</td></tr><tr><td rowspan=1 colspan=1>Cook L, Burdon J, Brenton S, Janus ED, Knight K. Alpha-1-antitrypsin PLowell: a normally functioning variant presentin low concentration. Aust N Z J Med. 1995. 25:695697.</td></tr><tr><td rowspan=2 colspan=2>PI* Pduarte</td><td rowspan=2 colspan=1>Impairedsecretion(degradation)andmild plasmadeficiency</td><td rowspan=1 colspan=1>Lieberman J, Gaidulis L, Klotz SD. A new deficient variantof alpha1-antitrypsin (MDUARTE). Inability to detect theheterozygous state by antitrypsin phenotyping. Am RevRespir Dis. 1976. 113:3136.</td></tr><tr><td rowspan=1 colspan=1>Hildesheim J, Kinsley G, Bissell M, Pierce J. Geneticdiversity from a limited repertoire of mutations on differentcommon allelic backgrounds: alpha 1-antitrypsin Deficiencyvariant Pduarte. Hum Mutat. 1993. 2:221228.</td></tr><tr><td rowspan=2 colspan=2>PI* Q0cardiff</td><td rowspan=2 colspan=1>Truncatedprotein/Degradation</td><td rowspan=1 colspan=1>Bamforth F, Kalsheker N. Alpha1 antitrypsin deficiency dueto Pi null: clinical presentation and evidence for molecularheterogeneity. J Med Genet. 1988. 25: 8387.</td></tr><tr><td rowspan=1 colspan=1>Graham A, Kalsheker N, Newton C, Bamforth F, Powell S,Markham A. Molecular characterisation of three alpha-1-antitrypsin deficiency variants: proteinase inhibitor (Pi)nullcardiff (Asp256-&gt; Val); PiMmalton,(Phe51-&gt; deletion)and Pil (Arg39-&gt; Cys). Hum Genet. 1989. 84:5558.</td></tr><tr><td rowspan=2 colspan=2>PI* Ybarcelona</td><td rowspan=2 colspan=1>Impairedsecretion(degradation)andmild plasmadeficiency</td><td rowspan=1 colspan=1>Jardi R, Rodriguez F, Miravitlles M, Vidal R, Cotrina M,Quer J, Pascual C, Weidinger S. Identification and MolecularCharacterization of the New Alpha-I-Antitrypsin DeficientAllele PI Y barcelona (Asp256Val and Pro 39&#x27; His). HumanMutation. 1998. 12: 213.</td></tr><tr><td rowspan=1 colspan=1>Miravitlles M, Vilà S, Jardí R, de la Roza C, Rodríguez-FríasF, Vidal R. Emphysema due to alpha-antitrypsin deficiency:familial study of the YBARCELONA variant. Chest.2003.124:404406.</td></tr><tr><td rowspan=3 colspan=1>c.863A&gt;T</td><td rowspan=3 colspan=2>PI* S</td><td rowspan=3 colspan=1>Impairedsecretion(degradation)andmild plasmadeficiency</td><td rowspan=1 colspan=1>Yoshida A, Ewing C, Wessels M, Lieberman J, Gaidulis L.Molecular abnormality of PI S variant of human alpha1-antitrypsin. Am J Hum Genet. 1977. 29:233239.</td></tr><tr><td rowspan=1 colspan=1>Engh R, Löbermann H, Schneider M, Wiegand G, Huber R,Laurell CB. The S variant of human alpha 1-antitrypsin,structure and implications for function and metabolism.Protein Eng. 1989. 2:407415.</td></tr><tr><td rowspan=1 colspan=1>Turino GM, Barker AF, Brantly ML, Cohen AB, ConnellyRP, Crystal RG, Eden E, Schluchter MD, Stoller JK. Clinicalfeatures of individuals with PI*SZ phenotype of alpha 1-antitrypsin deficiency. Alpha 1-Antitrypsin DeficiencyRegistry Study Group. Am J Respir Crit Care Med. 1996.154:17181725.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PI* Z</td><td rowspan=1 colspan=1>Impaired</td><td rowspan=1 colspan=1>Ogushi F, Fells GA, Hubbard RC, Straus SD, Crystal RG. Z-</td></tr></table>

<table><tr><td colspan="2" rowspan="1">Mutation Panel</td><td colspan="1" rowspan="2">A1AT ProteinActivity</td><td colspan="1" rowspan="2">Reference</td></tr><tr><td colspan="1" rowspan="1">A1ATAllele</td><td colspan="1" rowspan="1">AssociatedAllele</td></tr><tr><td colspan="1" rowspan="3">c.1096G&gt;A</td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3">secretion(polymerization)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">type alpha 1-antitrypsin is less competent than M1-typealpha 1-antitrypsin as an inhibitor of neutrophil elastase. JClin Invest. 1987. 80:13661374.</td></tr><tr><td colspan="1" rowspan="1">Turino GM, Barker AF, Brantly ML, Cohen AB, ConnellyRP, Crystal RG, Eden E, Schluchter MD, Stoller JK. Clinicalfeatures of individuals with PI*SZ phenotype of alpha 1-antitrypsin deficiency. Alpha 1-Antitrypsin DeficiencyRegistry Study Group. Am J Respir Crit Care Med. 1996.154:17181725.</td></tr><tr><td colspan="1" rowspan="1">Chappell S, Hadzic N, Stockley R, Guetta-Baranes T,Morgan K, Kalsheker N. A polymorphism of the alpha1-antitrypsin gene represents a risk factor for liver disease.Hepatology. 2008. 47:127132.</td></tr><tr><td colspan="1" rowspan="4">c.1130dupT</td><td colspan="1" rowspan="2">PI* Q0mattawa</td><td colspan="1" rowspan="2">Truncatedprotein/Degradation</td><td colspan="1" rowspan="1">Curiel D, Brantly M, Curiel E, Stier L, Crystal RG. Alpha 1-antitrypsin deficiency caused by the alpha 1-antitrypsin Nullmattawa gene. An insertion mutation rendering the alpha 1-antitrypsin gene incapable of producing alpha 1-antitrypsin.J Clin Invest. 1989. 83:11441152.</td></tr><tr><td colspan="1" rowspan="1">Lara B, Martinez-Delgado B, Torres ML, Marín-Arguedas S,Bustamante A, Miravitlles M. Alpha-1-antitrypsin deficiencyassociated with the Mattawa variant. Arch Bronconeumol.2013. 49:548550.</td></tr><tr><td colspan="1" rowspan="2">PI* Q0ourem</td><td colspan="1" rowspan="2">Truncatedprotein/Degradation</td><td colspan="1" rowspan="1">Seixas S, Mendonca C, Costa F, Rocha J. alpha1-Antitrypsinnull alleles: evidence for the recurrence of the L353fsX376mutation and a novel G&gt;A transition in position +1of intronIC affecting normal mRNA splicing. Clin Genet. 2002. 62:175180.</td></tr><tr><td colspan="1" rowspan="1">Vas Rodrigues L, Costa F, Marques P, Mendonça C, RochaJ, Seixas S. Severe α-1antitrypsin deficiency caused byQ0(Ourém) allele: clinical features, haplotypecharacterization and history.V. Clin Genet. 2012. 81:462469.</td></tr><tr><td colspan="1" rowspan="3">c.1158dupC</td><td colspan="1" rowspan="2">PI* Q0cclayton</td><td colspan="1" rowspan="2">Truncatedprotein/Degradation</td><td colspan="1" rowspan="1">Brantly M, Lee JH, Hildesheim J, Uhm CS, Prakash UB,Staats BA, Crystal RG, Hildeshiem J. Alpha1-antitrypsingene mutation hot spot associated with the formation of aretained and degraded null variant. Am J Respir Cell MolBiol. 1997.16:225231.</td></tr><tr><td colspan="1" rowspan="1">Rodríguez F, Vila B, Homs M, Vidal R, Calpe JL, Jardi R.Diagnosis of Alpha-1 Antitrypsin Deficiency: Limitations ofRapid Diagnostic Laboratory Tests. ArchBronconeumol.2011. 47:415417.</td></tr><tr><td colspan="1" rowspan="1">PI* Q0saarbruecken</td><td colspan="1" rowspan="1">Truncatedprotein</td><td colspan="1" rowspan="1">Lee H and Brantly M. Molecular mechanisms of alpha1-antitrypsin null alleles. Respir Med. 2000. 94 Suppl C: S7</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">S11.</td></tr><tr><td colspan="1" rowspan="1">Lin L, Schmidt B, Teckman J, Perlmutter DH. A naturallyoccurring nonpolymerogenic mutant of alpha 1-antitrypsincharacterized by prolonged retention in the endoplasmicreticulum. J Biol Chem. 2001. 276:3389333898.</td></tr><tr><td colspan="1" rowspan="2">c.1178C&gt;T</td><td colspan="1" rowspan="2">PI* Mheerlen</td><td colspan="1" rowspan="2">Impairedsecretion(degradation)andsevere plasmadeficiency</td><td colspan="1" rowspan="1">Hofker MH, Nukiwa T, van Paassen HM, Nelen M, KrampsJA, Klasen EC, Frants RR, Crystal RG. A Pro-Leusubstitution in codon 369 of the alpha-1-antitrypsindeficiency variant PI MHeerlen. Hum Genet. 1989. 81:264268.</td></tr><tr><td colspan="1" rowspan="1">Poller W, Merklein F, Schneider-Rasp S, Haack A, FechnerH, Wang H, Anagnostopoulos I, Weidinger S. Molecularcharacterisation of the defective alphal-antitrypsin alleles PIPI*Mwurzburg (Pro369Ser), PI*M heerlen (Pro369Leu), andPI*Q0 Lisbon (Thr68Ile). Eur J Hum Genet. 1999. 7:321331.</td></tr></table>

4. Clinical cut-off:

Not applicable.

5. Expected values/Reference range:

The table below presents the prevalence of the alleles in the A1AT Genotyping Test based on the cited reference.

<table><tr><td colspan="2" rowspan="1">Mutation Panel</td><td colspan="8" rowspan="1">Mutation frequencies among individuals with clinically diagnosedA1AT(%)</td></tr><tr><td colspan="1" rowspan="2">AllelicVariant</td><td colspan="1" rowspan="2">Associatedallele</td><td colspan="2" rowspan="1">Caucasian</td><td colspan="2" rowspan="1">HispanicAmericans</td><td colspan="2" rowspan="1">AfricanAmericans</td><td colspan="2" rowspan="1">AsianAmericans</td></tr><tr><td colspan="1" rowspan="1">Freq.</td><td colspan="1" rowspan="1">Reference</td><td colspan="1" rowspan="1">Freq.</td><td colspan="1" rowspan="1">Reference</td><td colspan="1" rowspan="1">Freq.</td><td colspan="1" rowspan="1">Reference</td><td colspan="1" rowspan="1">Freq.</td><td colspan="1" rowspan="1">Reference</td></tr><tr><td colspan="1" rowspan="1">c.187C&gt;T</td><td colspan="1" rowspan="1">PI*I</td><td colspan="1" rowspan="1">&lt;10-3**(heterozygous)6x10-4#</td><td colspan="1" rowspan="1">Ferrarotti I etal. J MedGenet. 2005.42:282287#https://www.ncbi.nlm.nih.gov/projectPI* S/PI*SNP/snp_ref.cgi?rs=28931570</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.194T&gt;C</td><td colspan="1" rowspan="1">PI* M procida</td><td colspan="1" rowspan="1">&lt;10-3 **(heterozygous)</td><td colspan="1" rowspan="1">Zorzetto Met al. ClinChem. 2008.54:1331-1338</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.226_228delTTC</td><td colspan="1" rowspan="1">PI* M malton;PI* M palermo;PI* M nichinan</td><td colspan="1" rowspan="1">1.1 x 10-4( PI*M maltoncarriers)</td><td colspan="1" rowspan="1">Cox DW andBillingsleyGD. Am JHumGenet.1989.44:844854</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.230C&gt;T</td><td colspan="1" rowspan="1">PI* S iiyama</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.551_552delC</td><td colspan="1" rowspan="1">PI* Q0 granitefalls</td><td colspan="1" rowspan="1">&lt;10-3(carriers)</td><td colspan="1" rowspan="1">Curiel D etal. J ClinInvest 1989.83:11441152</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.647G&gt;T</td><td colspan="1" rowspan="1">PI* Q0 west</td><td colspan="1" rowspan="1">Detected onlyin USA</td><td colspan="1" rowspan="1">Brantly, M.Am. J.RespirCCritic.CareMed2009,179:A3506</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">c.721A&gt;T</td><td colspan="1" rowspan="1">PI* Q0bellingham</td><td colspan="1" rowspan="1">&lt;10-3**(heterozygous)</td><td colspan="1" rowspan="1">Fregonese Let al. RespirMed.2008.102:876-884</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.739C&gt;T</td><td colspan="1" rowspan="1">PI* F</td><td colspan="1" rowspan="1">&lt;10-3**(heterozygous)10-3#</td><td colspan="1" rowspan="1">Okayama Het al. Am JHumGenet.1991.48:11541158#https://www.ncbi.nlm.nihgov/projectPI* S/PI*SNP/snp_ref.cgi?rs=28929470</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.839A&gt;T</td><td colspan="1" rowspan="1">PI* P lowellPI* P duartePI* Q0 cardiffPI* Y barcelona</td><td colspan="1" rowspan="1">&lt;10-3**(PI* P lowellheterozygous)</td><td colspan="1" rowspan="1">Ferrarotti Iet al. J MedGenet. 200542:282287</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.863A&gt;T</td><td colspan="1" rowspan="1">PI* S</td><td colspan="1" rowspan="1">5-10%(carriers)0.0196#</td><td colspan="1" rowspan="1">de Serres Fand Blanco I.J Intern Med.2014.276:311-335#https://www.ncbi.nlm.nih</td><td colspan="1" rowspan="1">5%(carriers)</td><td colspan="1" rowspan="1">de SerresFJ, et al.ClinGenet.2003. 64:382397</td><td colspan="1" rowspan="1">1%(carriers)</td><td colspan="1" rowspan="1">de SerresFJ, et al.ClinGenet.2003. 64:382397</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">.gov/projectPI* /PI*SNP/snp_ref.cgi?rs=28929474</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.1096G&gt;A</td><td colspan="1" rowspan="1">PI* Z</td><td colspan="1" rowspan="1">1-3%(carriers)4x10-3#</td><td colspan="1" rowspan="1">de Serres Fand Blanco I.J Intern Med.2014. 276:311335#https://www.ncbi.nlm.nih.gov/projectPI* S/PI*SNP/snp_ref.cgi?rs=28929474</td><td colspan="1" rowspan="1">1%(carriers)</td><td colspan="1" rowspan="1">de SerresFJ, et al.ClinGenet.2003. 64:382397</td><td colspan="1" rowspan="1">&lt;1%(carriers)</td><td colspan="1" rowspan="1">de SerresFJ, et al. ClinGenet.2003.64:382397</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">c.1130_1131insT</td><td colspan="1" rowspan="1">PI* Q0 mattawaPI* Q0 ourem</td><td colspan="1" rowspan="1">&lt;10-3(PI Q0mattawa,carriers)</td><td colspan="1" rowspan="1">Curiel D et al.J Clin Invest1989. 83:1144-1152</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">→</td></tr><tr><td colspan="1" rowspan="1">c.1156_1157 insC</td><td colspan="1" rowspan="1">PI* Q0 claytonPI* Q0saarbruecken</td><td colspan="1" rowspan="1">&lt;10-3*(PI Q0 clayton,heterozygous</td><td colspan="1" rowspan="1">Ferrarotti I etal. J MedGenet. 2005.42:282287</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Detectedin JJapanandKorea</td><td colspan="1" rowspan="1">Dae-HyunKo, et al.Korean JLab Med.2011.31:294297Kuniaki S.Respirology2000. 5,S35-S38</td></tr><tr><td colspan="1" rowspan="1">c.1178C&gt;T</td><td colspan="1" rowspan="1">PI* M heerlen</td><td colspan="1" rowspan="1">&lt;10-3*(heterozygous)</td><td colspan="1" rowspan="1">Ferrarotti I etal. J MedGenet. 200542:282287</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="10" rowspan="1">ND: no data available. Data btained from the NCBI 1000 genome database**Data estimated according to the reported alpha-1-antitrypsin deficiency prevalence data.</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.