# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3860 - Trichomonas vaginalis nucleic acid assay

2. Classification: Class II

3. Product code:

OUY Trichomonas vaginalis Nucleic Acid Amplification Test System MKZ DNA Probe, Nucleic Acid Amplification, Chlamydia LSL DNA-Reagents, Neisseria

Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD MAX CTGCTV2 assay, performed on the BD MAX System, incorporates automated DNA extraction and real-time polymerase chain reaction (PCR) for the direct, qualitative detection of DNA from:

Chlamydia trachomatis (CT) • Neisseria gonorrhoeae (GC) Trichomonas vaginalis (TV)

The assay may be used for detection of CT, GC and/or TV DNA in patient- or cliniciancollected vaginal swab specimens (in a clinical setting), and male and female urine specimens. The assay may also be used for the detection of CT and GC DNA in endocervical swab and Liquid-Based Cytology (LBC) specimens in PreservCy $\textsuperscript { \textregistered }$ Solution using an aliquot that is removed prior to processing for the ThinPrepTM Pap test.

The assay is indicated for use with asymptomatic and symptomatic individuals to aid in the diagnosis of chlamydial urogenital disease, gonococcal urogenital disease and/or trichomoniasis.

Ancillary Collection kits:

The BD MAX 3-in-1 Swab Collection Kit is intended to be used in clinical settings according to the instructions provided for collection and transport of vaginal and endocervical swab specimens. This transport system is for use for testing with BD MAX products.

The BD MAX Urine Transport Kit is intended to be used in clinical settings according to the instructions provided for collection, preservation and transport of urine specimens. This transport system is for use for testing with the BD MAX products.

The BD MAX LBC Sample Buffer Tubes are intended to be used in clinical settings according to the instructions provided for the preservation and transport of Liquid-Based Cytology (LBC) specimens. This transport system is for use for testing with the BD MAX products.

2. Indication(s) for use:

Same as the Intended use above.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: BD MAX System

# I. Device Description:

The BD MAX System and the BD MAX CTGCTV 2 assay are comprised of an instrument with associated hardware and accessories, and a reagent kit. Each reagent kit contains a PCR cartridge with the master mix, Unitized Reagent Strips (URS), extraction reagent, and sample buffer tubes, sufficient for 24 tests.

Urogenital specimens are collected using proprietary collection kits (BD MAX Urine Transport Kit, the BD MAX LBC Sample Buffer Tubes or the BD MAX 3-in-1 Swab Collection Kit) and transported to a laboratory. The Unitized Reagent Strips needed for the run are placed in the sample rack and securely seated. Foil-sealed dried Extraction Tubes and PCR reagent tubes are snapped into the appropriate positions on each Unitized Reagent Strip. The PCR cartridge is placed in a designated place on the instrument and the specimens are loaded onto a sample rack. As the run is initiated, the reagents are rehydrated and the cells in the sample are lysed releasing DNA which is then bound to magnetic beads. After a washing step, the PCR master mix is added and the specific targets, if present, are amplified followed by detection of the generated fluorescent signal.

Specimens collected in PreservCyt solution (Liquid-based Cytology (LBC) specimens) require a pre-warming step prior to loading on the BD MAX instrument, using the BD PreWarm Heater. Following the pre-heating, a $0 . 5 ~ \mathrm { m L }$ aliquot of the specimen is transferred to the BD MAX LBC Sample Buffer Tube and loaded onto the instrument.

The instrument consists of a sample processing sub-system with a fluid handling robot, and the integrated readers, which perform thermal cycling and optical detection. The instrument automates sample preparation including target lysis, DNA extraction and concentration, reagent rehydration, and target nucleic acid amplification and detection using real-time PCR.

A Sample Processing Control is included in every Extraction Tube and serves to detect a reagent or a procedure failure during the extraction and thermal cycling steps. The probes used for the amplification are labeled with a fluorophore which generates fluorescence in the presence of complementary sequences. The BD MAX software interprets the emitted fluorescence to generate final qualitative results. The following table shows the result interpretation for the BD MAX CTGCTV2 assay.

# Result Interpretation

<table><tr><td rowspan=1 colspan=1>Assay ResultReported</td><td rowspan=1 colspan=1>Inte rpre tation of Re sults</td></tr><tr><td rowspan=1 colspan=1>CT POS</td><td rowspan=1 colspan=1>Chlamydia trachomatis DNA Detected</td></tr><tr><td rowspan=1 colspan=1>CT NEG</td><td rowspan=1 colspan=1>No Chlamydia trachomatis DNA Detected</td></tr><tr><td rowspan=1 colspan=1>CT UNR</td><td rowspan=1 colspan=1>Unresolved  inhibitory sample or reagent failure; no target orSample Processing Control amplification</td></tr><tr><td rowspan=1 colspan=1>GC POS</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae DNA Detected</td></tr><tr><td rowspan=1 colspan=1>GC NEG</td><td rowspan=1 colspan=1>No Neisseria gonorrhoeae DNA Detected</td></tr><tr><td rowspan=1 colspan=1>GC UNR</td><td rowspan=1 colspan=1>Unresolved  inhibitory sample or reagent failure; no target orSample Processing Control amplification</td></tr><tr><td rowspan=1 colspan=1>TV POS</td><td rowspan=1 colspan=1>Trichomonas vaginalis DNA Detected</td></tr><tr><td rowspan=1 colspan=1>TV NEG</td><td rowspan=1 colspan=1>No Trichomonas vaginalis DNA Detected</td></tr><tr><td rowspan=1 colspan=1>TV UNR</td><td rowspan=1 colspan=1>Unresolved  inhibitory sample or reagent failure; no target orSample Processing Control amplification</td></tr><tr><td rowspan=1 colspan=1>IND</td><td rowspan=1 colspan=1>Indeterminate result due to BD MAX System failure (withWarning or Error Codes)</td></tr><tr><td rowspan=1 colspan=1>INC</td><td rowspan=1 colspan=1>Incomplete Run (with Warning or Error Codes)</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD MAX CT/GC/TV

2. Predicate 510(k) number(s): K151589

3. Comparison with predicate:

# Similarities and Differences

<table><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="1" rowspan="1">BD MAX CTGCTV2K182692</td><td colspan="1" rowspan="1">BD MAX CTGCTVK151589</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.3860</td><td colspan="1" rowspan="1">866.3860</td></tr><tr><td colspan="1" rowspan="1">IntendedUse</td><td colspan="1" rowspan="1">The BD MAX CTGCTV2 assay,performed on the BD MAX System,incorporates automated DNA extractionand real-time polymerase chain reaction(PCR) for the direct, qualitative detectionof DNA from:•  Chlamydia trachomatis (CT)Neisseria gonorrhoeae (GC)Trichomonas vaginalis (TV)</td><td colspan="1" rowspan="1">The BD MAX CT/GC/TV assay, asperformed using the BD MAX Systemincorporates automated DNA extraction andreal-time polymerase chain reaction (PCR)for the direct, qualitative detection of DNAfrom Chlamydia trachomatis (CT),Neisseria gonorrhoeae (GC) and/orTrichomonas vaginalis (TV). The assay maybe used for detection of CT and/or GC DNAin male urine specimens, and the detection</td></tr><tr><td colspan="1" rowspan="1">Ite m</td><td colspan="3" rowspan="1">BD MAX CTGCTV2K182692</td><td colspan="3" rowspan="1">BD MAX CTGCTVK151589</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">The assay may be used for detection of CT,GC and/or TV DNA in patient- orclinician-collected vaginal swab specimens(in a clinical setting), and male and femaleurine specimens. The assay may also beused for the detection of CT and GC DNAin endocervical swab and Liquid-BasedCytology (LBC) specimens inPreservCyt® Solution using an aliquot thatis removed prior to processing for theThinPrep Pap test.The assay is indicated for use withasymptomatic and symptomatic individualsto aid in the diagnosis of chlamydialurogenital disease, gonococcal urogenitaldisease and/or trichomonias is.</td><td colspan="3" rowspan="1">of CT, GC and/or TV DNA in female urinespecimens, clinician-collected femaleendocervical swab specimens and patient-collected vaginal swab specimens (in aclinical setting). The assay is indicated foruse to aid in the diagnosis of chlamydialurogenital disease, gonococcal urogenitaldisease and/or trichomonias is inasymptomatic and symptomatic individuals.</td></tr><tr><td colspan="1" rowspan="1">AncillaryCollection/TransportKits</td><td colspan="3" rowspan="1">•Swab Sample Buffer Tube (2.0 mL)(same formulation as UVE SampleBufferUrine Sample Buffer Tube (0.5 mL)LBC Sample Buffer Tube (1.5 mL)</td><td colspan="3" rowspan="1">• UVE Sample Buffer Tube (1.5 mL),swab/urine</td></tr><tr><td colspan="1" rowspan="1">AssayResults</td><td colspan="3" rowspan="1">Qualitative</td><td colspan="3" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="3" rowspan="1">Chlamydia trachomatisNeisseria gonorrhoeaeTrichomonas vaginalis</td><td colspan="3" rowspan="1">Chlamydia trachomatisNeisseria gonorrhoeaeTrichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="3" rowspan="1">BD MAX System</td><td colspan="3" rowspan="1">BD MAX System</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="3" rowspan="1">Real-time PCR</td><td colspan="3" rowspan="1">Real-time PCR</td></tr><tr><td colspan="1" rowspan="1">Specimens</td><td colspan="3" rowspan="1">For CT and GC:Clinician-collected vaginal swabPatient-collected vaginal swabEndocervical swabPreservCyt LBCUrine (female and male)For TV:Clinician-collected vaginal swab,Patient-collected vaginal swabUrine (female and male)</td><td colspan="3" rowspan="1">For CT, GC and TV:• Endocervical swab,Patient-collected vaginal swab,Urine (female and male)</td></tr><tr><td colspan="1" rowspan="1">AssayControls</td><td colspan="3" rowspan="1">Sample Processing Control</td><td colspan="3" rowspan="1">Sample Processing Control</td></tr><tr><td colspan="1" rowspan="3">TargetDetection</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Dye</td><td colspan="1" rowspan="1">Channel</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">Dye</td><td colspan="1" rowspan="1">Channel</td></tr><tr><td colspan="1" rowspan="1">CT</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">CT</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">FAM</td></tr><tr><td colspan="1" rowspan="1">CT</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">CT</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">FAM</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">GC (GC1)</td><td colspan="1" rowspan="1">CFO</td><td colspan="1" rowspan="1">VIC</td><td colspan="1" rowspan="1">GC (GC1)</td><td colspan="1" rowspan="1">CFO</td><td colspan="1" rowspan="1">VIC</td></tr><tr><td colspan="1" rowspan="1">GC (GC2)</td><td colspan="1" rowspan="1">Q705</td><td colspan="1" rowspan="1">CY5.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">TV</td><td colspan="1" rowspan="1">Q670</td><td colspan="1" rowspan="1">CY5</td><td colspan="1" rowspan="1">TV</td><td colspan="1" rowspan="1">Q670</td><td colspan="1" rowspan="1">CY5</td></tr><tr><td colspan="1" rowspan="1">SamplePrep</td><td colspan="3" rowspan="1">1 swab added to Swab Sample BufferTube2.0 mL urine added to Urine SampleBuffer Tube•0.5 mL LBC added to LBC SampleBuffer Tube</td><td colspan="3" rowspan="1">1 swab added to UVE Sample BufferTube1.0 ml urine added to UVE SampleBuffer Tube</td></tr><tr><td colspan="1" rowspan="1">Pre-analytical</td><td colspan="3" rowspan="1">Prewarm only LBC samples prior toextraction</td><td colspan="3" rowspan="1">Prewarm all UVE samples prior toextraction</td></tr><tr><td colspan="1" rowspan="1">Extraction</td><td colspan="3" rowspan="1">Magnetic affinity beads with protease</td><td colspan="3" rowspan="1">Magnetic affinity beads with protease</td></tr><tr><td colspan="1" rowspan="1">On-boardlysis</td><td colspan="3" rowspan="1">On-board lysis of all specimens</td><td colspan="3" rowspan="1">No on-board lysis</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guideline: Nucleic Acid Amplification Assays for the Detection of Trichomonas vaginalis Guideline for Industry and Food and Drug Administration Staff, issued August 4, 2015

EP-07-A2, 2012 - Interference Testing in Clinical Chemistry; Approved Guideline

MM03-ED3, 2015 – Molecular Diagnostic Methods for Infectious Diseases, Approved Guideline

EP12-A2, 2008 – User protocol for Evaluation of Qualitative Test performance, Approved Guideline

M29-A4, 2014 – Protection of laboratory workers from occupationally acquired infections, Approved Guideline

# L. Test Principle:

The assay utilizes real-time polymerase chain reaction (PCR) for amplification of specific genetic sequences in the DNA of the target microorganisms. The nucleic acids from the target organisms are released during cell lysis and are captured on magnetic affinity beads. The beads are washed using Wash Buffer and the nucleic acids are then eluted by a combination of heat and pH. Eluted DNA is neutralized using Neutralization Buffer and transferred to the Master Mix to rehydrate the PCR reagents. After reconstitution, the BD MAX System dispenses a fixed volume of PCR-ready solution containing extracted nucleic acids into the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed by the system prior to initiating PCR in order to contain the amplification mixture, thus preventing evaporation and contamination.

The amplified DNA targets are detected using hydrolysis (TaqMan) probes, labeled at one end with a fluorescent reporter dye (fluorophore), and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect amplicons for target analytes and the Sample Processing Control in five different optical channels of the BD MAX System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The BD MAX System monitors these signals at the end of each cycle and interprets the data at the end of the reaction to provide qualitative test results for each analyte (i.e., positive or negative).

# M. Performance Characteristics:

# 1. Analytical performance:

a. Precision

Within-laboratory precision was evaluated for the BD MAX CTGCTV2 assay at one site with one lot of reagents. Testing was performed over 12 days, by two technologists, alternating each day to perform two runs (total of 24 runs). The test panel consisted of four samples prepared in pooled negative vaginal, urine, and PreservCyt specimen matrix. Each of the three organisms, Chlamydia trachomatis (Serovar D), Neisseria gonorrhoeae, and Trichomonas vaginalis, was spiked into the respective matrices at the following target concentrations relative to the established limit of detection (LoD):

• True negative (TN): no target (expected negative $100 \%$ of the time) • Low Positive (LP): 1.5x LoD (expected positive $9 5 \%$ to $100 \%$ of the time) High Negative (HN): below assay LoD (expected negative $5 \%$ to $9 5 \%$ of the time) • Moderate Positive (MP): 3x LoD (expected positive $100 \%$ of the time)

The following 10 panel members were prepared:

Precision Test Panel   

<table><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="3" rowspan="1">Target Level</td></tr><tr><td colspan="1" rowspan="1">CT</td><td colspan="1" rowspan="1">GC</td><td colspan="1" rowspan="1">TV</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">TN</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">MP</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">LP</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">HN</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">TN</td></tr></table>

Each sample was tested in duplicate in each run and one pair of positive and negative External Controls was included in each test run.

The within-laboratory qualitative and quantitative precision data by target are presented below. The mean values with the associated variance components (SD and $\% C V$ ) are based on ct.score values, the internal numeric instrument output used to determine the final assay result. The within-laboratory precision results for the BD MAX CTGCTV2 are shown below.

Overall Precision (Within Laboratory) Using One Lot of the BD MAX CTGCTV2   

<table><tr><td rowspan="5">Category</td><td colspan="9">Percent Agreement with Expected Results</td></tr><tr><td colspan="3">Chlamydia trachomatis</td><td colspan="3">Neisseria gonorrhoeae</td><td colspan="2">Trichomonas vaginalis</td></tr><tr><td colspan="3"></td><td colspan="3">(n), 95% CI</td><td colspan="2">(n), 95% CI</td></tr><tr><td>Swab</td><td>(n), 95% CI Urine</td><td>LBCc</td><td>Swab</td><td>Urine</td><td>LBCc</td><td>Swab</td><td>Urine</td></tr><tr><td rowspan="2">TNa</td><td rowspan="2">100% (336/336)</td><td rowspan="2">100% (336/336)</td><td rowspan="2">99.4%</td><td>100%</td><td>100%</td><td>100%</td><td>100%</td><td>100%</td></tr><tr><td>(336/336)</td><td>(336/336)</td><td>(336/336)</td><td></td><td>(336/336)</td></tr><tr><td rowspan="2"></td><td rowspan="2">98.9-100 31.3%</td><td rowspan="2">98.9-100</td><td rowspan="2">(334/336) 97.9-99.8</td><td rowspan="2">98.9-100</td><td rowspan="2">98.9-100</td><td rowspan="2">98.9-100</td><td rowspan="2">(336/336) 98.9-100</td><td rowspan="2">98.9-100</td></tr><tr><td></td></tr><tr><td rowspan="2">HNb</td><td rowspan="2">(15/48) 19.9-45.3</td><td rowspan="2">31.3% (15/48)</td><td rowspan="2">18.8% (9/48)</td><td rowspan="2">27.1% (13/48)</td><td rowspan="2">29.2%</td><td rowspan="2">16.7%</td><td rowspan="2">37.5% (18/48)</td><td rowspan="2">68.8% (33/48)</td></tr><tr><td>(14/48) (8/48)</td></tr><tr><td rowspan="2">LP</td><td rowspan="2">100% (48/48)</td><td rowspan="2">19.9-45.3 100%</td><td rowspan="2">10.2-31.9 100%</td><td rowspan="2">16.6-41.0 97.9%</td><td rowspan="2">18.2-43.2 100%</td><td rowspan="2">8.7-29.6 100%</td><td rowspan="2">25.2-51.6 97.9%</td><td rowspan="2">54.7-80.1 100%</td></tr><tr><td></td></tr><tr><td rowspan="2"></td><td rowspan="2">92.6-100 100%</td><td rowspan="2">(48/48) 92.6-100</td><td rowspan="2">(48/48) 92.6-100</td><td rowspan="2">(47/48) 89.1-99.6</td><td rowspan="2">(48/48) 92.6-100</td><td rowspan="2">(48/48) 92.6-100</td><td rowspan="2">(47/48) 89.1-99.6</td><td rowspan="2">(48/48) 92.6-100</td></tr><tr><td></td></tr><tr><td rowspan="2">MP</td><td rowspan="2">(48/48) 92.6-100</td><td rowspan="2">100% (48/48) 92.6-100</td><td rowspan="2">100% (48/48) 92.6-100</td><td rowspan="2">100% (48/48)</td><td rowspan="2">100% (48/48)</td><td rowspan="2">100% (48/48)</td><td rowspan="2">100% (48/48)</td><td rowspan="2">100% (48/48)</td></tr><tr><td>92.6-100 92.6-100 92.6-100</td></tr></table>

a For the True Negative (TN) category, the reported agreement indicates the percent of negative results. b For the High Negative (HN) category, the reported agreement indicates the percent of positive results. c PreservCyt Solution

Chlamydia trachomatis Within-laboratory Precision, Quantitative, by Matrix   

<table><tr><td colspan="1" rowspan="2">Matrix</td><td colspan="1" rowspan="2">Sample</td><td colspan="1" rowspan="2">Agree/N</td><td colspan="1" rowspan="2">Mean</td><td colspan="2" rowspan="1">Within Run</td><td colspan="2" rowspan="1">Between Run</td><td colspan="2" rowspan="1">BetweenDay</td><td colspan="2" rowspan="1">BetweenOperator</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">/o CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="1">Swab</td><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">15/48</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">6.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">15/48</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">2.1</td><td colspan="1" rowspan="1">5.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.1</td><td colspan="1" rowspan="1">5.5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">3.3</td></tr><tr><td colspan="1" rowspan="1">LBCa</td><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">9/48</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">3.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">5.8</td></tr><tr><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">/% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">3.6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.9</td></tr></table>

a PreserveCyt Solution

Neisseria gonorrhoeae Within-laboratory Precision, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>MatrixType</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>AgreeIN</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>Between Operator</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=9 colspan=1>GC1</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>13/48</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>14/48</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>8/48</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>9.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=9 colspan=1>GC2</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>13/48</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.3</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>14/48</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>8/48</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>9.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td></tr></table>

a PreservCyt Solution

Trichomonas vaginalis Within-laboratory Precision, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>MatrixType</td><td rowspan=2 colspan=1>Cat</td><td rowspan=2 colspan=1>AgreeN</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>18/48</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.7</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>33/48</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>7.4</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.9</td></tr></table>

Based on the evaluation of precision of the BD MAX CTGCTV2 assay system, conducted at one site, with one lot of reagents, the overall variability (for positive samples), across all matrices and across all analytical targets, expressed as $\% \mathrm { C V }$ , ranged from $3 . 8 \%$ to $9 . 5 \%$ for the High Negative samples; from $1 . 4 \%$ to $3 . 7 \%$ for the Low Positive samples; and from $1 . 5 \%$ to $3 . 3 \%$ for the Moderate Positive samples.

# b. Reproducibility

# Site-to-Site Reproducibility

A site-to-site reproducibility of the BD MAX CTGCTV2 assay was evaluated at three testing sites (two external and one internal), using one lot of reagents. Testing was performed on eight distinct days, by two technologists, alternating each day to perform two runs per day (16 runs per site, a total of 48 runs across the sites). Each site was provided the same panels as described for the Precision study, above. Each sample was tested in duplicate in each run and one pair of positive and negative External Controls was included in each test run.

The data was analyzed for percent agreement with the expected qualitative results. Additionally, quantitative analyses of the PCR parameters were performed for each target (for positive results only).

Site-to-Site Reproducibility, Qualitative, Across Sites   

<table><tr><td rowspan=1 colspan=9>Percent Agreement with Expected Results</td></tr><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Chlamydia trachomatis(n), 95% CI</td><td rowspan=1 colspan=3>Neisseria gonorrhoeae(n), 95% CI</td><td rowspan=1 colspan=2>Trichomonas vaginalis(n), 95% CI</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>LBCc</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>LBCc</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>99.6%(669/672)98.7-99.8</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>99.9%(671/672)99.2-100</td><td rowspan=1 colspan=1>99.9%(671/672)99.2-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td></tr><tr><td rowspan=1 colspan=1>HNb</td><td rowspan=1 colspan=1>20.8%(20/96)13.9-30.0</td><td rowspan=1 colspan=1>35.4%(34/96)26.6-45.4</td><td rowspan=1 colspan=1>21.9%(21/96)14.8-31.1</td><td rowspan=1 colspan=1>34.4%(33/96)25.6-44.3</td><td rowspan=1 colspan=1>28.1%(27/96)20.1-37.8</td><td rowspan=1 colspan=1>11.5%(11/96)6.5-19.4</td><td rowspan=1 colspan=1>37.5%(36/96)28.5-47.5</td><td rowspan=1 colspan=1>78.1%(75/96)68.9-85.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>97.9%(94/96)92.7-99.4</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>97.9%(94/96)92.7-99.4</td></tr></table>

a For the True Negative (TN) category, the reported agreement indicates the percent of negative results. b For the High Negative (HN) category, the reported agreement indicates the percent of positive results. c PreservCyt Solution

Chlamydia trachomatis Site-to-Site Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>MatrixType</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>AgreeiN</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>Between Day</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>BetweenSite</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>20/96</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>34/96</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.9</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>21/96</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr></table>

a PreservCyt Solution

Neisseria Gonorrhoeae Site-to-Site Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>WMk</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Agree7N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=9 colspan=1>GC1</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>33/96</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>6.4</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>27/96</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>11/96</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>94/96</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=9 colspan=1>GC2</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>33/96</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>27/96</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>11/96</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>94/96</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.2</td></tr></table>

a PreservCyt Solution

Trichomonas vaginalis Site-to-Site Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Agree/N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Run</td><td rowspan=1 colspan=2>Between Day</td><td rowspan=1 colspan=2>Between Operator</td><td rowspan=1 colspan=2>Between Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>36/96</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>75/96</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>94/96</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.6</td></tr></table>

Based on the evaluation of reproducibility of the BD MAX CTGCTV2 assay system, conducted at three sites, with one lot of reagents, the overall variability (for positive samples), across all matrices and across all analytical targets, expressed as $\%$ CV, ranged from $3 . 2 \%$ to $6 . 6 \%$ for the High Negative samples; from $2 . 1 \%$ to $4 . 0 \%$ for the Low Positive samples; and from $1 . 5 \%$ to $3 . 2 \%$ for the Moderate Positive samples.

# Lot-to-Lot Reproducibility

A lot-to-lot reproducibility was evaluated at one (internal) site using three lots of reagents. The testing was performed over eight days, by two technologists, alternating each day to perform two runs per day. Using the same test panels as described for the Precision study, above, there were 16 runs performed on each lot of reagents, for a total of 48 runs. Each sample was tested in duplicate in each run and one pair of positive and negative External Controls was included in each test run.

The data was analyzed for percent agreement with the expected qualitative results. Additionally, quantitative analyses of the PCR parameters were performed for each target (for positive results only).

Lot-to-Lot Reproducibility, by Organism, Qualitative   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=8>Percent Agreement with Expected Results</td></tr><tr><td rowspan=1 colspan=3>Chlamydia trachomatis</td><td rowspan=1 colspan=3>Neisseria gonorrhoeae</td><td rowspan=1 colspan=2>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>LBCc</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>LBCc</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td></tr><tr><td rowspan=1 colspan=1>TNa</td><td rowspan=1 colspan=1>99.4%(668/672)98.5-99.8</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>99.6%(669/672)98.7-99.8</td><td rowspan=1 colspan=1>99.9%(671/672)99.2-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td><td rowspan=1 colspan=1>100%(672/672)99.4-100</td></tr><tr><td rowspan=1 colspan=1>HNb</td><td rowspan=1 colspan=1>22.9%(22/96)15.6-32.3</td><td rowspan=1 colspan=1>37.5%(36/96)28.5-47.5</td><td rowspan=1 colspan=1>15.6%(15/96)9.7-24.2</td><td rowspan=1 colspan=1>24.0%(23/96)16.5-33.4</td><td rowspan=1 colspan=1>24.0%(23/96)16.5-33.4</td><td rowspan=1 colspan=1>10.4%(10/96)5.8-18.1</td><td rowspan=1 colspan=1>40.6%(39/96)31.3-50.6</td><td rowspan=1 colspan=1>63.5%(61/96)53.6-72.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>95.8%(92/96)89.8-98.4</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td><td rowspan=1 colspan=1>99.0%(95/96)94.3-99.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.-100</td><td rowspan=1 colspan=1>95.8%(92/96)89.8-98.4</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td><td rowspan=1 colspan=1>100%(96/96)96.2-100</td></tr></table>

a For the True Negative (TN) category, the reported agreement indicates the percent of negative results. b For the High Negative (HN) category, the reported agreement indicates the percent of positive results. c PreservCyt Solution

Chlamydia trachomatis Lot-to-Lot Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Agree7N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>Betweenperator</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>22/96</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>7.3</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>36/96</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>7.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>15/96</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>6.7</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.3</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.7</td></tr></table>

a PreservCyt Solution

Neisseria Gonorrhoeae Lot-to-Lot Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>Target</td><td rowspan=2 colspan=1>Matrix</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Agree/N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=9 colspan=1>GC1</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>23/96</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>6.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>23/96</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>92/96</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>92/96</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>10/96</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=9 colspan=1>GC2</td><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>23/96</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>23/96</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>92/96</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>92/96</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>LBCa</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>10/96</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.4</td></tr></table>

a PreservCyt Solution

Trichomonas vaginalis Lot-to-Lot Reproducibility, Quantitative, by Matrix   

<table><tr><td rowspan=2 colspan=1>MatrixType</td><td rowspan=2 colspan=1>Cat</td><td rowspan=2 colspan=1>Agree/N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>WithinRun</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenDay</td><td rowspan=1 colspan=2>BetweenOperator</td><td rowspan=1 colspan=2>BetweenLot</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>39/96</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>5.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>61/96</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>5.4</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>95/96</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td></tr></table>

Based on the evaluation of precision of the BD MAX CTGCTV2 assay system, conducted at one site, with three lots of reagents, the overall variability (for positive samples), across all matrices and across the three analytes, expressed as $\% C V$ , ranged from $5 . 3 \%$ to $1 2 . 1 \%$ for the High Negative samples; from $2 . 0 \%$ to $4 . 7 \%$ for the Low Positive samples; and from $2 . 3 \%$ to $3 . 2 \%$ for the Moderate Positive samples.

c. Linearity/assay reportable range:

Not applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls:

The BD MAX CTGCTV2 assay includes a Sample Processing Control (SPC) in each Extraction Tube which serves to monitor the DNA extraction and thermal cycling steps, as well as to detect reagent deterioration and the presence of inhibitory substances. If the Sample Processing Control result fails to meet the specific acceptance criteria written in the software, the result of the specimen will be reported as Unresolved; however, any positive (POS) assay result will be reported and no targets will be called NEG. An Unresolved result is indicative of specimen-associated inhibition or reagent failure. Any specimen reported as Unresolved must be retested.

External controls are not provided by the manufacturer. Laboratories must establish the number, type, and frequency of testing of control materials according to guidelines or requirements of local, provincial, state and federal and/or country regulations, or accreditation organizations in order to monitor the effectiveness of the entire analytical process. An External Positive Control is intended to monitor for substantial reagent failure. An External Negative Control is intended to detect reagent or environmental contamination (or carry-over) by target nucleic acids.

Commercially available control material or a previously characterized clinical sample known to be positive may be used as a positive control. The BD MAX CTGCTV2 Swab Buffer Tube without the addition of organism or a previously characterized sample known to be negative is recommended for use as an external negative control. External positive and negative controls were included in all analytical and clinical studies performed in support of this submission.

# Specimen Stability Studies:

Specimen stability (i.e., stability of the amplifiable DNA in the specimen) was evaluated for urine, vaginal swabs, and endocervical specimens collected in PreservCyt solution (LBC specimen). Test samples were prepared in pooled negative clinical matrix (i.e., swab, urine and LBC) by spiking with the target organisms to concentrations targeted at $2 \mathbf { x }$ LoD and processing them according to the directions in the collection kit appropriate for the matrix. Baseline testing was performed on the same day as the samples were prepared. Specimen stability was evaluated at $2 { - } 8 \mathrm { { ^ \circ C } }$ , $3 0 \mathrm { { ^ { o } C } }$ and $- 2 0 \mathrm { { ^ \circ C } }$ . All positive (spiked) samples were tested in 20 replicates at each time point. All negative samples were tested in four replicates at each time point. The data generated during the study support specimen stability at the following storage conditions:

Urine specimens, when transferred immediately to the MAX Urine SBT, are stable up to 21 days at $2 { - } 3 0 \mathrm { { ^ \circ C } }$ and then additional 30 days at $- 2 0 \mathrm { { ^ { \circ } C } }$ .

Endocervical and vaginal swab specimens, when transferred immediately to the MAX Swab SBT, are stable up to 21 days at $2 { - } 3 0 \mathrm { { ^ \circ C } }$ and then additional 30 days at $- 2 0 \mathrm { { ^ \circ C } }$ .

Neat PreservCyt specimens must be transferred to the MAX LBC SBT within 14 days of collection when stored at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ . PreservCyt in MAX LBC SBT prior or post-prewarm can be stored for up to 21 days at $2 { - } 3 0 ^ { \circ } \mathrm { C }$ and then an additional 30 days at $- 2 0 ^ { \circ } \mathrm { C }$ before testing.

# e. Detection limit:

# Analytical Sensitivity

The Limit of Detection (LoD) for the BD MAX CTGCTV2 assay was determined in urine, vaginal swab specimen matrix, and in PreservCyt liquid based cytology (LBC) specimen matrix. The study included two representative strains of each: CT (serovars D and H), GC, and TV organisms.

The CT, GC, and TV stock cultures were obtained from the ATCC either as lyophilized or frozen cultures. Each organism was re-grown in the appropriate media designated for that organism and quantified stocks were prepared for use in the study. CT EBs were enumerated by direct immunofluorescent staining using a labeled antibody directed toward the Major Outer Membrane Protein. The EBs were counted using a fluorescent microscope. GC was enumerated by preparing a suspension of cells and measuring the optical density to calculate the number of organisms/mL. TV was enumerated using a Neubauer hemocytometer under a light microscope to obtain the total count of organisms/mL

For each strain to be tested, pooled female urine, pooled vaginal swab matrix, and pooled PreservCyt samples (pre-screened negative for CT, GC and TV) were individually spiked with different concentrations of CT, GC or TV organisms. After determining a putative LoD (serial dilutions tested in 12 replicates, to determine the highest dilution (the lowest target level) yielding $100 \%$ positivity rate), a confirmation of LoD study was conducted in each matrix. Dilutions prepared at organism concentration levels below and above the putative LoD were tested with the BD MAX CTGCTV2 assay in 20 replicates. Confirmation of LoD was achieved when $\ge 9 5 \%$ positivity rate was observed, i.e., at least 19 out of 20 results per dilution were positive. Additional intermediate dilutions were tested when the observed results were $100 \%$ positive at one level and $< 8 5 \%$ at the next lower concentration. The testing was performed across nine BD MAX instruments using three lots of reagents. The confirmed Limit of Detection by serovar/strain and specimen type is shown below.

Limit of Detection of the BD MAX CTGCTV2 Assay, by Organism and Matrix   

<table><tr><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>Units</td></tr><tr><td rowspan=6 colspan=1>Chlamydiatrachomatis</td><td rowspan=3 colspan=1>Serovar H</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=3 colspan=1>Serovar D</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>EB/mL</td></tr><tr><td rowspan=6 colspan=1>Neisseriagonorrhoeae</td><td rowspan=3 colspan=1>ATCC 19424</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=3 colspan=1>ATCC 49226</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>Cells/mL</td></tr><tr><td rowspan=4 colspan=1>Trichomonasvaginalis</td><td rowspan=2 colspan=1>ATCC 30001</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>TV/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>TV/mL</td></tr><tr><td rowspan=2 colspan=1>ATCC 50143</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>TV/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>1.88</td><td rowspan=1 colspan=1>TV/mL</td></tr></table>

# Analytical Reactivity (Inclusivity)

The reactivity of the BD MAX CTGCTV2 assay was evaluated with additional clinically relevant and geographically diverse serovars and/or strains of Chlamydia trachomatis (CT), Neisseria gonorrhoeae (GC), and Trichomonas vaginalis (TV). The study included 13 serovars of Chlamydia trachomatis (A, B, C, E, vE, F, G, I, J, K, LGV1, LGV2, and LGV3), 30 strains of Neisseria gonorrhoeae and eight strains of Trichomonas vaginalis. Organism stocks were obtained from ATCC either as lyophilized or frozen cultures. Each organism was grown on/in the appropriate media designated for that organism and quantified. Each organism was evaluated in swab, urine, and PreservCyt matrix in 20 replicates at the pre-determined LoD (1x) concentration, using at least three reagent lots. All organisms, except four of the Neisseria gonorrhoeae strains, were positive in at least 19 out of $2 0 \left( \geq 9 5 \% \right)$ replicates tested. Three of the remaining four N. gonorrhoeae strains were titered to a 1.5x LoD concentration in vaginal matrix and retested. One additional strain was re-titered to a 1.5x LoD concentration in urine and retested. All four organisms were detected in at least 19 out of 20 replicates when tested at 1.5x LoD concentration. The results of the analytical reactity testing are shown below.

Analytical Reactivity for Chlamydia trachomatis   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Serovar</td><td rowspan=1 colspan=2>Swab</td><td rowspan=1 colspan=2>Urine</td><td rowspan=1 colspan=2>PreservCyt</td></tr><tr><td rowspan=1 colspan=1>EBs/mL</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>EBs/mL</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>EBs/mL</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=13 colspan=1>Chlamydiatrachomatis</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>LGV1</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>LGV2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>LGV3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>vE</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20/20</td></tr></table>

Analytical Reactivity for Neisseria gonorrhoeae   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>NumberStrains</td><td rowspan=1 colspan=2>Swab</td><td rowspan=1 colspan=2>Urine</td><td rowspan=1 colspan=2>PreservCyt</td></tr><tr><td rowspan=1 colspan=1>Cells/mL</td><td rowspan=1 colspan=1>% POS</td><td rowspan=1 colspan=1>Cells/mL</td><td rowspan=1 colspan=1>% POS</td><td rowspan=1 colspan=1>Cells/mL</td><td rowspan=1 colspan=1>% POS</td></tr><tr><td rowspan=5 colspan=1>Neisseriagonorrhoeae</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>≥95</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

Analytical Reactivity for Trichomonas vaginalis   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>ATCCStrain</td><td rowspan=1 colspan=2>Swab</td><td rowspan=1 colspan=2>Urine</td></tr><tr><td rowspan=1 colspan=1>TV/mL</td><td rowspan=1 colspan=1>% POS</td><td rowspan=1 colspan=1>TV/mL</td><td rowspan=1 colspan=1>% POS</td></tr><tr><td rowspan=8 colspan=1>Trichomonasvaginalis</td><td rowspan=1 colspan=1>30092</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30184</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30185</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30186</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30235</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30236</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30238</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>≥95</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>30240</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100</td></tr></table>

Cross-reactivity with Other Microorganisms

The analytical specificity of the MAX CTGCTV2 assay was evaluated with wellcharacterized isolates of microorganisms that are phylogenetically related or likely to be found in urogenital specimens. The bacterial strains, viruses, parasites, and yeast species commonly found in urogenital specimens were added to the Sample Buffer Tubes and tested in three replicates across two lots of assay reagents. Bacteria, parasites, and yeast were grown and quantified to achieve final test concentrations $\mathrm { o f } \geq 1 \mathrm { x l } 0 ^ { 6 }$ organisms/mL. Quantified viruses were obtained from commercial vendors and diluted to achieve final test concentration in the Sample Buffer Tube o $\tilde { \mathbf { \Gamma } } \geq 1 \mathbf { x } 1 0 ^ { 5 }$ pfu or genomic equivalents $/ \mathrm { m L }$ . All samples were negative for CT, GC, and TV in all three replicates tested, except for Pentatrichomonas hominis and Trichomonas tenax. Pentatrichomonas hominis (commensal of the large intestine) produced positive results for TV at a concentration $\geq 1 . 0 0 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ $\mathrm { T V / m L }$ ; Trichomonas tenax (commensal of the oral cavity) produced positive results for TV at a concentration $\geq 1 . 8 8 \ \mathrm { T V / m L }$ . Both organisms produced negative results for CT and GC (i.e., no cross-reactivity).

# Organisms Tested with the BD MAXCTGCTV2 Assay for Cross-reactivity

<table><tr><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Organis m</td><td colspan="1" rowspan="1">Organism</td></tr><tr><td colspan="1" rowspan="1">Achromobacter xerosis</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Neisseria mucosa (3)a</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">Escherichia vulneris</td><td colspan="1" rowspan="1">Neisseria perflava</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Fuseobacterium nucleatum</td><td colspan="1" rowspan="1">Neisseria polysaccharea</td></tr><tr><td colspan="1" rowspan="1">Actinomyces israelii</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Neisseria sicca(3)</td></tr><tr><td colspan="1" rowspan="1">Actinomyces pyogenes</td><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">Neisseria subflava (14)a</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">Paracoccus denitrificans</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophilia</td><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Pentatrichomonas hominis</td></tr><tr><td colspan="1" rowspan="1">Agrobacterium radiobacter</td><td colspan="1" rowspan="1">Herpes Simplex Virus I</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Herpes Simplex Virus II</td><td colspan="1" rowspan="1">Peptostreptococcus productus</td></tr><tr><td colspan="1" rowspan="1">Atopobium vaginae</td><td colspan="1" rowspan="1">HIV 1</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">HPV 16</td><td colspan="1" rowspan="1">Prevotella bivia</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Kingelladentrificans</td><td colspan="1" rowspan="1">Propionibacterium acnes</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ureoeolyticus</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium brevi</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Branhamella catarrhalis</td><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium linens</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">Pseudomonas putida</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Lactobacillus lactis</td><td colspan="1" rowspan="1">Rahnella aquatilis</td></tr><tr><td colspan="1" rowspan="1">Candidaalbicans</td><td colspan="1" rowspan="1">Lactobacillus vaginalis</td><td colspan="1" rowspan="1">Rhodospirilium rubrum</td></tr><tr><td colspan="1" rowspan="1">Candida gabralta</td><td colspan="1" rowspan="1">Legionella pneumophilia (2)a</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Leuconostoc paramensenteroides</td><td colspan="1" rowspan="1">Salmonella minnesota</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Salmonella typhimurium</td></tr><tr><td colspan="1" rowspan="1">Chlamydiophila pneumoniae</td><td colspan="1" rowspan="1">Micrococcus leutus</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Chlamydiophila psittaci (2)a</td><td colspan="1" rowspan="1">Mobiluncus curtisii</td><td colspan="1" rowspan="1">Staphylococcus aureus,non-protein A</td></tr><tr><td colspan="1" rowspan="1">Chromobacterium violaceum</td><td colspan="1" rowspan="1">Moraxellalacunata</td><td colspan="1" rowspan="1">Staphylococcus aureus, protein-Aproducing</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Mycobacterium smegmatis</td><td colspan="1" rowspan="1">Streptcoccus bovis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium biovarl</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">Streptococcus agalactiae (Group B)</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Neisseria cinerea(4)a</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Neisseria denitrificans</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Deinococcus radiodurans</td><td colspan="1" rowspan="1">Neisseria elongate (3) a</td><td colspan="1" rowspan="1">Streptococcus pyogenes (Group A)</td></tr><tr><td colspan="1" rowspan="1">Derxia gummosa</td><td colspan="1" rowspan="1">Neisseriaflava</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">Neisseria flavescens (2)a</td><td colspan="1" rowspan="1">Streptococcus sanguis</td></tr><tr><td colspan="1" rowspan="1">Elizabethkingia meningosepticum</td><td colspan="1" rowspan="1">Neisserialactamica (9) a</td><td colspan="1" rowspan="1">Streptomyces griseinus</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Neisseria meningitidis A</td><td colspan="1" rowspan="1">Trichomonas tenax</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Neisseria meningitidis B</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">Neisseria meningitidis C(4) a</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Neisseria meningitidis D</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Neisseria meningitidis W 135</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">Neisseria meningitidis Y</td></tr></table>

The number in parenthesis indicates the number of strains tested. \*Pentatrichomonas hominis produced positive results for TV at a concentration $\phantom { + } 1 { = } 1 . 0 0 \mathrm { x } ~ 1 0 ^ { 5 } \mathrm { T V / m L }$ ; and negative results for CT and GC; \*\*Trichomonas tenax produced positive results for TV at a concentration $\geq 1 . 8 8 \mathrm { T V / m L }$ and negative results for CT and GC

# Interference from Endogenous and Exogenous Substances

Potential interference/inhibition from endogenous and exogenous substances that may be present in urogenital specimens was evaluated by testing specimens contrived in negative matrix at low concentrations of the target analytes (CT, GC and TV), in the presence of each of the substances listed in the table below. Two serovars of CT, two strains of GC, and one strain of TV were spiked into urine, swab, and PreservCyt matrices at target concentrations of 3x LoD. The interferents tested were added to each of the samples containing the target analytes. Each sample was tested in triplicate, in accordance with the test procedure. Twenty six of the substances were evaluated in urine, 23 were tested in vaginal swab matrix, and 22 were tested in PreservCyt media. Each of the substances was titrated to determine the concentration at which no interference was observed. There was no interference observed up to the concentrations tested, as shown below.

Exogenous and Endogenous Substances Tested for Interference in Urine   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Norethindrone</td><td colspan="1" rowspan="1">16 ng/mL</td></tr><tr><td colspan="1" rowspan="1">17-α-Ethinylestradiol</td><td colspan="1" rowspan="1">0.96 ng/mL</td></tr><tr><td colspan="1" rowspan="1">4-Acetamidophenol</td><td colspan="1" rowspan="1">160 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Acetylsalicylic Acid</td><td colspan="1" rowspan="1">521.6 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Naproxen</td><td colspan="1" rowspan="1">400 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ibuprofen</td><td colspan="1" rowspan="1">400 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Human Serum Albumin</td><td colspan="1" rowspan="1">0.8 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Glucose</td><td colspan="1" rowspan="1">0.96 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Amoxicillin Trihydrate</td><td colspan="1" rowspan="1">60.16 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">96 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Subs tance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">Tetracycline Hydrochloride</td><td colspan="1" rowspan="1">12 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Azithromycin</td><td colspan="1" rowspan="1">9.6 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Ceftriaxone</td><td colspan="1" rowspan="1">648.8 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Sulfamethoxazole</td><td colspan="1" rowspan="1">320 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Trimethoprim</td><td colspan="1" rowspan="1">32 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Erythromycin</td><td colspan="1" rowspan="1">48 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucous (Bovine Cervical)</td><td colspan="1" rowspan="1">4% v/v</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">0.6% v/v a</td></tr><tr><td colspan="1" rowspan="1">Semen</td><td colspan="1" rowspan="1">4% v/v</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">2x106 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Phenazopyridine Hydrochloride</td><td colspan="1" rowspan="1">160 μg/mL</td></tr><tr><td colspan="1" rowspan="1">High pH (NaOH)</td><td colspan="1" rowspan="1">pH 9</td></tr><tr><td colspan="1" rowspan="1">Low pH (HCl)</td><td colspan="1" rowspan="1">pH 4</td></tr><tr><td colspan="1" rowspan="1">Bilirubin</td><td colspan="1" rowspan="1">0.16 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Feminine Deodorant Spray</td><td colspan="1" rowspan="1">0.68% v/v</td></tr><tr><td colspan="1" rowspan="1">Talcum Powder</td><td colspan="1" rowspan="1">2.64% v/v</td></tr></table>

a May interfere with the BD MAX CTGCTV2 assay when at concentrations higher than shown.

Endogenous and Exogenous Substances Tested for Interference in Swab Specimens   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">VCF Contraceptive Foam</td><td colspan="1" rowspan="1">15 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">VCF Contraceptive Film</td><td colspan="1" rowspan="1">0.5 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Conceptrol Contraceptive Gel</td><td colspan="1" rowspan="1">3 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Gyne-Lotrimin 3</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Monistat 3 Cream</td><td colspan="1" rowspan="1">0.1 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Tioconazole 1</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Vaginal Anti-Itch Cream</td><td colspan="1" rowspan="1">0.1 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Vaginal Lubricant Liquid - water based</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Preparation H Hemorrhoid Gel</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Antiviral (Zovirax - Acyclovir)</td><td colspan="1" rowspan="1">0.01 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Metronidazole Gel (AntiProtozoal)</td><td colspan="1" rowspan="1">1.25 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Replens (Vaginal Moisturizer)</td><td colspan="1" rowspan="1">0.1 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Douche</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Feminine Deodorant Spray</td><td colspan="1" rowspan="1">50 μL/mL</td></tr><tr><td colspan="1" rowspan="1">Progesterone</td><td colspan="1" rowspan="1">20 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Estradiol</td><td colspan="1" rowspan="1">1.2 ng/mL</td></tr><tr><td colspan="1" rowspan="1">Mucous (Bovine Cervical)</td><td colspan="1" rowspan="1">4.5% v/va</td></tr><tr><td colspan="1" rowspan="1">Semen</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">8 μL/mLa</td></tr><tr><td colspan="1" rowspan="1">Leukocytes</td><td colspan="1" rowspan="1">1x106 cells/mL</td></tr><tr><td colspan="1" rowspan="1">Aquagel</td><td colspan="1" rowspan="1">0.0345 mg/mLa</td></tr><tr><td colspan="1" rowspan="1">Subs tance</td><td colspan="1" rowspan="1">Concentration</td></tr><tr><td colspan="1" rowspan="1">McKesson Lubricating Jelly</td><td colspan="1" rowspan="1">0.445 mg/mLa</td></tr><tr><td colspan="1" rowspan="1">Surgilube</td><td colspan="1" rowspan="1">0.41mg/mL a</td></tr><tr><td colspan="1" rowspan="1">KY Lubricating Jelly</td><td colspan="1" rowspan="1">137.5mg/mL</td></tr></table>

a May interfere with the BD MAX CTGCTV2 assay when at concentrations higher than shown

Endogenous and Exogenous Substances Tested for Interference in PreservCyt LBC Specimens   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Vaginal Lubricant</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Douche</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Vaginal Deodorant Spray</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Progesterone</td><td rowspan=1 colspan=1>20 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Estradiol</td><td rowspan=1 colspan=1>1.2 ng/mL</td></tr><tr><td rowspan=1 colspan=1>Leukocytes</td><td rowspan=1 colspan=1>1 x 106 cells/mL</td></tr><tr><td rowspan=1 colspan=1>Semen</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Monistat 3</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Clotrimazole 7</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Tioconazole 1</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Vaginal contraceptive film</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Vaginal contraceptive foam</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Contraceptive gel</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Vaginal anti-itch Cream</td><td rowspan=1 colspan=1>1% v/va</td></tr><tr><td rowspan=1 colspan=1>Zovirax (Acyclovir) Cream</td><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>Metronidazole Gel 0.75%</td><td rowspan=1 colspan=1>0.01% v/va</td></tr><tr><td rowspan=1 colspan=1>Replens (Vaginal Moisturizer)</td><td rowspan=1 colspan=1>0.1% v/va</td></tr><tr><td rowspan=1 colspan=1>Mucous (Bovine Cervical)</td><td rowspan=1 colspan=1>1.9% v/va</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>0.5% v/va</td></tr><tr><td rowspan=1 colspan=1>Aquagel</td><td rowspan=1 colspan=1>0.276 mg/mLa</td></tr><tr><td rowspan=1 colspan=1>McKesson Lubricating Jelly</td><td rowspan=1 colspan=1>3.56 mg/mLa</td></tr><tr><td rowspan=1 colspan=1>Surgilube</td><td rowspan=1 colspan=1>328 mg/mLa</td></tr><tr><td rowspan=1 colspan=1>KY Lubricating Jelly</td><td rowspan=1 colspan=1>1,100 mg/mL</td></tr></table>

a May interfere with the BD MAX CTGCTV2 assay when at concentrations higher than shown.

# g. Assay cut-off:

For all analytes in the CTGCTV2 assay, the cutoff was determined using a training dataset and later validated by the clinical study. With the exception of GC2, the chemistry of the CTGCTV2 assay is identical to the previously cleared CT/GC/TV assay (K151589), and therefore presented a unique opportunity to use the previous clinical study as a training dataset for those analytes.

For each analyte, an acceptable threshold range was established for each sample type in the study, setting a minimum threshold at a point where no false positives were caused by system noise. System noise was evaluated subjectively, using expert analysis of individual signal curves to distinguish them from true PCR amplification. A maximum threshold was then set as the highest value with no new false negatives. Positive results are reported when any Ct score is calculated and, by definition, there are no Ct values for negative specimens (note: instrument reports a $^ { - 1 }$ for negative specimens).

However, since GC2 was not included in the CT/GC/TV study, the cutoff for that analyte was determined using a separate analytical dataset. Multiple GC LoD titration studies were performed and the data was evaluated over a range of Ct thresholds. The studies used contrived samples at varying concentrations including true negatives, high negatives (below LoD), low positives (at LoD), and moderate positives $\left( 2 { \cdot } 4 \mathrm { x } \ \mathrm { L o D } \right)$ in both vaginal and urine matrix. True negatives were $100 \%$ negative across the entire Ct threshold. As a result, thresholds were selected to maximize sensitivity.

The assay cutoff was then verified based on the results of the clinical study, using all samples with valid reference results (3756 for CT and for GC and 3735 for TV). The infected/noninfected status was determined by the agreed PIS algorithm for all specimens, except for urine, which was evaluated by comparison to a Composite Comparator Algorithm (CCA), consisting of three NAATs (with a 2 out of 3 rule).

# h. Competitive Inhibition:

A study was conducted to evaluate the performance of the assay in the presence of mixed infection, i.e., the potential inhibition of one of the targets by the presence of the second (or third) analyte. The urine, vaginal, and PreservCyt specimens were spiked with different combinations of high and low concentrations of the organisms to determine the impact, if any, of mixed infections on assay results. The Low concentration was prepared at 1.5X LoD (as shown below) and the high concentration was prepared at ${ \geq } 1 0 ^ { 6 }$ units/mL.

Target Concentration of the Test Samples   

<table><tr><td rowspan=1 colspan=1>Organis m</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>LowConce ntration</td><td rowspan=1 colspan=1>Units/mL</td></tr><tr><td rowspan=3 colspan=1>Chlamydiatrachomatis</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>1.88</td><td rowspan=3 colspan=1>EB/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=3 colspan=1>Neisseriagonnorhoeae</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>22.5</td><td rowspan=3 colspan=1>Cells/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=2 colspan=1>Trichomonasvaginalis</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>7.5</td><td rowspan=2 colspan=1>TV/mL</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>11.25</td></tr></table>

Each sample was tested in 20 replicates. All targets were detected in $2 0 / 2 0$ replicates, except for TV in vaginal swab matrix which was detected in 19/20 replicates (within the acceptance criteria of a $2 9 5 \%$ ).

Mixed Infection Study Results   

<table><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Low TargetOrganis m (1.5x LoD)</td><td rowspan=1 colspan=1>High Target Mix(≥106 units/mL)</td><td rowspan=1 colspan=1>Positive(for Low Target)</td></tr><tr><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>C. trachomatis</td><td rowspan=1 colspan=1>CT/GC</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>N. gonorrhoeae</td><td rowspan=1 colspan=1>CT/TV</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>T. vaginalis</td><td rowspan=1 colspan=1>GC/TV</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=3 colspan=1>Swab</td><td rowspan=1 colspan=1>C. trachomatis</td><td rowspan=1 colspan=1>CT/GC</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>N. gonorrhoeae</td><td rowspan=1 colspan=1>CT/TV</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>T. vaginalis</td><td rowspan=1 colspan=1>GC/TV</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>C. trachomatis</td><td rowspan=1 colspan=1>CT/GC</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>N. gonorrhoeae</td><td rowspan=1 colspan=1>CT/TV</td><td rowspan=1 colspan=1>20/20</td></tr></table>

The observed shift in the signal, expressed in Ct-score values, for all samples tested across the three matrices, ranged from -0.1 to 4.5 for CT, from $- 1 . 8$ to 5.8 for GC, and from 1.0 to 4.8 for TV.

# i. Carryover/Contamination:

A study was conducted to estimate the rate of carryover/cross-contamination for the MAX CTGCTV2 assay on the BD MAX System when processing specimens with a high concentration of an analyte. The study was performed using high positive samples alternated with negative samples. PreservCyt matrix was used as a representative specimen type in this study because of its high alcohol content and thus most likely to result in crosscontamination. The High Positive samples were prepared at a concentration $\geq 1 0 ^ { 6 } \mathrm { E B / m L }$ in the LBC Sample Buffer Tubes. Negative samples consisted of LBC Sample Buffer Tubes without target added.

Eighteen runs were conducted, each containing 12 replicates of the High Positive sample and 12 replicates of the negative sample (24 samples per run). The samples were loaded in an alternating fashion (positive, negative, positive, etc.) and were equally distributed across BD MAX sample racks for a total of 432 measurements. The testing was performed by two operators over two days, using 3 instruments in total.

Data was analyzed by comparing the proportion of false positive results observed in the negative samples. Among the 216 negative specimens analyzed there were two positive results, with the contamination rate estimate of $0 . 9 3 \%$ $9 5 \%$ CI: $0 . 2 5 \%$ ; $3 . 3 1 \%$ ).

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

# 3. Clinical studies:

Twelve geographically diverse clinical collection centers participated in this study. Two additional sites participated in the study and performed reference testing only. Clinical centers represented high and low prevalence settings and included sexually transmitted disease (STD) clinics, family planning clinics, research facilities and obstetrics and gynecology clinics (OB/GYN). Of the 12 collection sites, five sites enrolled both female and male subjects, six sites enrolled females only, and one site enrolled males only. Enrollment occurred between July 6, 2016 and June 2, 2017, during which a total of 2,547 female subjects and 1,159 male subjects were enrolled. Specimens were collected from asymptomatic and symptomatic subjects. The collected specimens were distributed and tested with the BD MAX CTGCTV2 assay across five clinical testing sites.

Male subjects provided a urine specimen which was aliquoted into four transport devices for testing with the BD MAX assay and three comparator assays.

A total of eight specimens were collected from each female and included one urine specimen (aliquoted into four transport devices), two patient-collected vaginal swabs (in a clinical setting), two clinician-collected vaginal swabs, two endocervical swabs, and one LBC specimen (in ThinPrep PreservCyt Solution).

Standard of care specimens were collected either separately or first within the specimen category prior to the remaining collection used in the investigational study.

Samples were prepared for BD MAX CTGCTV2 testing in accordance with the appropriate BD MAX specimen collection kit package insert instructions. Specimens intended for reference testing were prepared in accordance with the instructions for use in the appropriate specimen collection kit package insert.

Of the total subjects enrolled in the study, 11 females and 10 males were excluded due to enrollments issues. The remaining 2,536 female subjects and 1,149 male subjects provided specimens for testing with the BD MAX CTGCTV2 assay and the comparator assays. After specimen exclusion for reasons such as missing specimens, transport errors, collection errors, shipping errors, processing errors, missing reference test results or nonreportable BD MAX test results (Indeterminate, Unresolved or Invalid), the final data analysis of performance estimates, using evaluable specimens, included:

• For Chlamydia trachomatis, there were 2,508 patient-collected vaginal swabs, 2,502 clinician-collected vaginal swabs, 2,512 endocervical swabs, 2,469 PreservCyt LBC, 2,416 female urine and 1,140 male urine specimens; For Neisseria gonorrhoeae, there were 2,506 patient-collected vaginal swabs, 2,503 clinician-collected vaginal swabs, 2,511 endocervical swabs, 2,470 PreservCyt LBC, 2,419 female urine and 1,142 male urine specimens; For Trichomonas vaginalis, there were 1,742 patient-collected vaginal swabs, 1,732 clinician-collected vaginal swabs, 1,646 female urine and 1,141 male urine specimens.

a. In Females with Vaginal Swabs, Endocervical Swabs, and LBC Specimens

The clinical performance of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in females was calculated compared to a Patient Infected Status (PIS) based on testing urine and cervical specimens with two different FDA-cleared NAATs. Female subjects were designated as infected if at least one different reference NAAT was positive for urine and for the cervical specimen. For the purpose of data analysis, women who tested negative in swabs with both NAATs and positive with the two comparator NAATs in urine only, were considered non-infected (Negative PIS) when calculating the performance of the assay with vaginal or endocervical specimens.

The clinical sensitivity of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in female vaginal swabs (patient- and cliniciancollected vaginal swabs) and cervical specimens (endocervical swab and PreservCyt LBC) is shown below.

Clinical Performance for $C .$ trachomatis and $N _ { \bullet }$ gonorrhoeae in Females (Vaginal and Endocervical Specimens)   

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Chlamydia trachomatis</td><td rowspan=1 colspan=2>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td></tr><tr><td rowspan=12 colspan=1>Female</td><td rowspan=3 colspan=1>VaginalClinician</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>98.255/5690.6-99.7</td><td rowspan=1 colspan=1>98.81,076/1,08998-99.3</td><td rowspan=1 colspan=1>10015/1579.6-100</td><td rowspan=1 colspan=1>99.81,129/1,13199.4-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>98.671/7292.5-99.8</td><td rowspan=1 colspan=1>99.01,272/1,28598.3-99.4</td><td rowspan=1 colspan=1>96.427/2882.3-99.4</td><td rowspan=1 colspan=1>99.91,328/1,32999.6-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>98.4126/12894.5-99.6</td><td rowspan=1 colspan=1>98.92,348/2,37498.4-99.3</td><td rowspan=1 colspan=1>97.742/4387.9-99.6</td><td rowspan=1 colspan=1>99.92,457/2,46099.6-100</td></tr><tr><td rowspan=3 colspan=1>Vaginal Self</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>98.255/5690.6-99.7</td><td rowspan=1 colspan=1>98.91,077/1,08998.1-99.4</td><td rowspan=1 colspan=1>10015/1579.6-100</td><td rowspan=1 colspan=1>99.61,126/1,13099.1-99.9</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>98.671/7292.5-99.8</td><td rowspan=1 colspan=1>98.51,271/1,29197.6-99</td><td rowspan=1 colspan=1>10028/2887.9-100</td><td rowspan=1 colspan=1>1001,333/1,33399.7-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>98.4126/12894.5-99.6</td><td rowspan=1 colspan=1>98.72,348/2,38098.1-99</td><td rowspan=1 colspan=1>10043/4391.8-100</td><td rowspan=1 colspan=1>99.82,459/2,46399.6-99.9</td></tr><tr><td rowspan=3 colspan=1>Endo-cervical</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>96.454/5687.9-99</td><td rowspan=1 colspan=1>99.41,084/1,09098.8-99.7</td><td rowspan=1 colspan=1>10015/1579.6-100</td><td rowspan=1 colspan=1>99.91,131/1,13299.5-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>93.167/7284.8-97</td><td rowspan=1 colspan=1>99.11,282/1,29498.4-99.5</td><td rowspan=1 colspan=1>92.926/2877.4-98</td><td rowspan=1 colspan=1>1001,336/1,33699.7-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>94.5121/12889.1-97.3</td><td rowspan=1 colspan=1>99.22,366/2,38498.8-99.5</td><td rowspan=1 colspan=1>95.341/4384.5-98.7</td><td rowspan=1 colspan=1>1002,467/2,46899.8-100</td></tr><tr><td rowspan=3 colspan=1>LBCPreservCyt</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>92.650/5482.4-97.1</td><td rowspan=1 colspan=1>99.71,076/1,07999.2-99.9</td><td rowspan=1 colspan=1>10015/1579.6-100</td><td rowspan=1 colspan=1>99.91,118/1,11999.5-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>92.965/7084.3-96.9</td><td rowspan=1 colspan=1>99.81,264/1,26699.4-100</td><td rowspan=1 colspan=1>88.924/2771.9-96.1</td><td rowspan=1 colspan=1>1001,309/1,30999.7-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>92.7115/12486.8-96.1</td><td rowspan=1 colspan=1>99.82,340/2,34599.5-99.9</td><td rowspan=1 colspan=1>92.939/4281-97.5</td><td rowspan=1 colspan=1>1002,427/2,42899.8-100</td></tr></table>

Based on available data generated in the study, the clinical performance of the BD MAX CTGCTV2 assay for female vaginal swabs (patient- and clinician-collected vaginal swabs) and cervical specimens (endocervical swab and PreservCyt LBC) was also assessed against a PIS algorithm composed of vaginal swab and urine specimens. Comparative reference methods included two different FDA cleared molecular tests where female subjects were designated as infected if at least two different reference NAATs were positive for urine and vaginal specimens. Females who tested positive with the two comparator NAATs in urine only (negative in swabs with both NAATs), were considered non-infected

(Negative PIS) when calculating the performance of the assay with swab specimens.

The resulting clinical performance of the BD MAX CTGCTV2 when comparing to the vaginal/urine reference algorithm (PIS) for the detection of Chlamydia trachomatis infection in females was: $9 5 . 0 \%$ (132/139) sensitivity and $9 9 . 2 \%$ (2,340/2,359) specificity for cliniciancollected vaginal swabs, $9 7 . 9 \%$ (137/140) sensitivity and $9 9 . 2 \%$ (2,348/2,368) specificity for patient-collected vaginal swabs, $8 9 . 2 \%$ (124/139) sensitivity and $9 9 . 4 \%$ (2,354/2,368) specificity for endocervical swabs, and $8 5 . 8 \% ( 1 1 5 / 1 3 4 )$ sensitivity and $9 9 . 8 \%$ (2,326/2,330) specificity for PreservCyt LBC specimens.

The resulting clinical performance of the BD MAX CTGCTV2 with the vaginal/urine reference algorithm for the detection of Neisseria gonorrhoeae infection in females was: $100 \%$ (44/44) sensitivity and $100 \%$ (2,454/2,455) specificity for clinician-collected vaginal swabs, $9 7 . 8 \%$ (44/45) sensitivity and $9 9 . 9 \%$ (2,458/2,461) specificity for patient-collected vaginal swabs, $9 3 . 3 \%$ (42/45) sensitivity and $100 \%$ (2,461/2,461) specificity for endocervical swabs, and $9 0 . 9 \% ( 4 0 / 4 4 )$ sensitivity and $100 \%$ (2,421/2,421) specificity for PreservCyt LBC specimens.

# $b$ . In Male Urine

The clinical performance of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in male urine was calculated compared to a Patient Infected Status (PIS) based on testing urine specimens with up to three different FDA cleared NAATs, where male subjects were designated as infected if at least two out of three reference NAAT results were positive. Subjects were categorized as non-infected if at least two out of three reference NAAT results were negative.

The clinical sensitivity of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in male urine is shown below.

Clinical Performance for C. trachomatis and $N _ { \bullet }$ gonorrhoeae in Males (Urine Specimens)   

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Chlamydia trachomatis</td><td rowspan=1 colspan=2>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td></tr><tr><td rowspan=3 colspan=1>Male</td><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>97.371/7390.5-99.2</td><td rowspan=1 colspan=1>99.6667/67098.7-99.8</td><td rowspan=1 colspan=1>10012/1275.8-100</td><td rowspan=1 colspan=1>100732/73299.5-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>96.377/8089.5-98.7</td><td rowspan=1 colspan=1>99.1314/31797.3-99.7</td><td rowspan=1 colspan=1>99.1110/11195.1-99.8</td><td rowspan=1 colspan=1>99.7286/28798.1-99.9</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>96.7148/15392.6-98.6</td><td rowspan=1 colspan=1>99.4981/98798.7-99.7</td><td rowspan=1 colspan=1>99.2122/12395.5-99.9</td><td rowspan=1 colspan=1>99.91,018/1,01999.4-100</td></tr></table>

The clinical performance of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in female urine was calculated compared to a Composite Comparator Algorithm (CCA), where urine samples were tested with up to three reference NAATs to generate reference results for each analyte. Female subjects were designated as positive if at least two out of three reference NAAT results were positive. Subjects were categorized as negative if at least two out of three reference NAAT results were negative. The clinical performance of the BD MAX CTGCTV2 for the detection of Chlamydia trachomatis and Neisseria gonorrhoeae infections in female urine, expressed as positive percent agreement (PPA) and negative percent agreement (NPA) with the CCA, is shown below.

Clinical Performance for $C .$ trachomatis and $N _ { \bullet }$ gonorrhoeae in Females (Urine Specimens)   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Chlamydia trachomatis</td><td rowspan=1 colspan=2>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>PPAN95% CI</td><td rowspan=1 colspan=1>NPAN95% CI</td><td rowspan=1 colspan=1>PPAN95% CI</td><td rowspan=1 colspan=1>NPAN95% CI</td></tr><tr><td rowspan=3 colspan=1>Female Urine</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>98.151/5289.9-99.7</td><td rowspan=1 colspan=1>99.21,037/1,04598.5-99.6</td><td rowspan=1 colspan=1>10014/1478.5-100</td><td rowspan=1 colspan=1>99.91,085/1,08699.5-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>98.670/7192.4-99.8</td><td rowspan=1 colspan=1>99.41,241/1,24898.8-99.7</td><td rowspan=1 colspan=1>10025/2586.7-100</td><td rowspan=1 colspan=1>1001,294/1,29499.7-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>98.4121/12394.3-99.6</td><td rowspan=1 colspan=1>99.32,278/2,29398.9-99.6</td><td rowspan=1 colspan=1>10039/3991.0-100</td><td rowspan=1 colspan=1>1002,379/2,38099.8-100</td></tr></table>

Additionally, the results obtained testing urine specimens in females with the BD MAX CTGCTV2 assay were evaluated against the PIS based on cervical and urine algorithm. In the clinical study conducted for the BD MAX CTGCTV2 assay, $5 . 8 \%$ fewer Chlamydia trachomatis infections were detected in female urine than in clinician and self-collected vaginal swabs and $1 . 9 \%$ fewer Chlamydia trachomatis infections than in endocervical swabs. Similarly, $2 . 4 \%$ fewer Neisseria gonorrhoeae infections were detected in female urine than in clinician-collected vaginal swabs and $4 . 7 \%$ fewer Neisseria gonorrhoeae infections than in patient-collected vaginal swabs. There was no difference in the detection of Neisseria gonorrhoeae infections when comparing female urine and endocervical swabs.

# Clinical Performance of the BD MAX CTGCTV2 Assay for the Detection of Trichomonas vaginalis Infections

# a. In Females with Vaginal Swabs

The clinical performance of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infection in females, using vaginal swabs, was compared to a PIS based on testing vaginal specimens with two different FDA-cleared molecular tests, where one of the tests utilized two instrument platforms (for a total of three comparator test results). Female subjects were designated as infected (Positive PIS) when at least two of the three test results were positive and non-infected (Negative PIS) when at least two of the three test results were negative.

The clinical sensitivity of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infections in female vaginal swabs (patient- and clinician-collected vaginal swabs) is shown below.

Clinical performance for T. vaginalis in Females (Vaginal Specimens)   

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td></tr><tr><td rowspan=6 colspan=1>Female</td><td rowspan=3 colspan=1>VaginalClinician</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>98.566/6792.0-99.7</td><td rowspan=1 colspan=1>99.6687/69098.7-99.9</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>97.5116/11992.8-99.1</td><td rowspan=1 colspan=1>99.6853/85699.0-99.9</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>97.8182/18694.6-99.2</td><td rowspan=1 colspan=1>99.61540/154699.2-99.8</td></tr><tr><td rowspan=3 colspan=1>Vaginal Self</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>97.065/6789.8-99.2</td><td rowspan=1 colspan=1>99.1685/69198.1-99.6</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>98.4120/12294.2-99.5</td><td rowspan=1 colspan=1>99.2855/86298.3-99.6</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>97.9185/18994.7-99.2</td><td rowspan=1 colspan=1>99.21540/155398.6-99.5</td></tr></table>

# $b$ . In Male Urine

The clinical performance of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infections in male urine was calculated compared to a Patient Infected Status (PIS) based on testing urine specimens using up to three different FDA cleared NAATs where male subjects were designated as infected if at least two out of three reference NAAT results were positive. Subjects were categorized as non-infected if at least two out of three reference NAAT results were negative.

The clinical sensitivity of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infections in male urine is shown below.

Clinical Performance for T. vaginalis in Males (Urine Specimens)   

<table><tr><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>% SensN95% CI</td><td rowspan=1 colspan=1>% SpecN95% CI</td></tr><tr><td rowspan=3 colspan=1>Male</td><td rowspan=3 colspan=1>Urine</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>96.225/2681.1-99.3</td><td rowspan=1 colspan=1>99.6678/68198.7-99.9</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>10022/2285.1-100</td><td rowspan=1 colspan=1>100412/41299.1-100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>97.947/4889.1-99.6</td><td rowspan=1 colspan=1>99.71,090/1,09399.2-99.9</td></tr></table>

# c. In Female Urine

The clinical performance of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infection in female urine was calculated compared to a Composite Comparator Algorithm (CCA), where urine samples were tested with up to three reference NAATs to generate reference results. Female subjects were designated as positive if at least two out of three reference NAAT results were positive. Subjects were categorized as negative if at least two out of three reference NAAT results were negative. The clinical performance of the BD MAX CTGCTV2 for the detection of Trichomonas vaginalis infections in female urine, expressed as percent agreement with CCA, is shown below.

Clinical performance for T. vaginalis in Females (Urine Specimens)   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=2 colspan=1>SymptomStatus</td><td rowspan=1 colspan=2>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>PPAN95% CI</td><td rowspan=1 colspan=1>NPAN95% CI</td></tr><tr><td rowspan=3 colspan=1>Female Urine</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>10058/5893.8-100</td><td rowspan=1 colspan=1>99.8646/64799.1-100</td></tr><tr><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>100115/11596.8-100</td><td rowspan=1 colspan=1>99.4821/82698.6-99.7</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>100173/17397.8-100</td><td rowspan=1 colspan=1>99.61,467/1,47399.1-99.8</td></tr></table>

# Quality Control

External quality controls were tested during the course of the clinical study. One Positive and one Negative External Control (EC) were tested in each BD MAX run. The mean, SD, and $\% \mathrm { C V }$ were calculated for the Positive EC for each analyte at each site and overall and are presented below.

Summary of Positive EC Results from the Study (Based on Ct.score Values)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>CT</td><td rowspan=1 colspan=3>GC</td><td rowspan=1 colspan=3>TV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>% CV</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>8.16</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>5.46</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>4.34</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>3.41</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.64</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>4.40</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.27</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.74</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.59</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.83</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>698</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.79</td><td rowspan=1 colspan=1>6.61</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>3.84</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>3.65</td></tr></table>

# BD MAX CTGCTV2 Non-Reportable Results

Of the 13,655 specimens initially tested with the BD MAX CT/GC/TV, there were a total of 321 non-reportable results, for a total non-reportable rate of $2 . 3 5 \%$ $9 5 \%$ CI: $2 . 1 1 \% - 2 . 6 2 \% )$ . Among those, there were a total of 124 Unresolved results, 77 Indeterminate results, and 120 Incomplete results. Six of the samples that initially generated non-reportable results were not repeated. Of the 316 samples that were retested, 38 results remained non-reportable $( 0 . 2 8 \%$ , with $9 5 \%$ CI: $0 . 2 0 \% - 0 . 3 8 \%$ ). The tables below show the observed non-reportable results by specimen type.

Unresolved Rate by Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Initial Unresolved Rate</td><td rowspan=1 colspan=2>Final Unresolved Rate a</td></tr><tr><td rowspan=1 colspan=1>Percent (n)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent (n)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Vaginal Clinician-Collected</td><td rowspan=1 colspan=1>1.1% (28/2,517)</td><td rowspan=1 colspan=1>(0.8%, 1.6%)</td><td rowspan=1 colspan=1>0.3% (7/2,517)</td><td rowspan=1 colspan=1>(0.1%, 0.6%)</td></tr><tr><td rowspan=1 colspan=1>Vaginal Patient-Collected</td><td rowspan=1 colspan=1>1.3% (34/2,525)</td><td rowspan=1 colspan=1>(1.0%, 1.9%)</td><td rowspan=1 colspan=1>0.4% (10/2,525)</td><td rowspan=1 colspan=1>(0.2%, 0.7%)</td></tr><tr><td rowspan=1 colspan=1>Endocervical</td><td rowspan=1 colspan=1>0.9% (22/2,519)</td><td rowspan=1 colspan=1>(0.6%, 1.3%)</td><td rowspan=1 colspan=1>0.1% (3/2,516)</td><td rowspan=1 colspan=1>(0.0%, 0.3%)</td></tr><tr><td rowspan=1 colspan=1>LBC PreservCyt</td><td rowspan=1 colspan=1>0.2% (6/2,473)</td><td rowspan=1 colspan=1>(0.1%, 0.5%)</td><td rowspan=1 colspan=1>0.0% (0/2,471)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>0.9% (34/3,621)</td><td rowspan=1 colspan=1>(0.7%, 1.3%)</td><td rowspan=1 colspan=1>0.1% (3/3,620)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr></table>

a Based on the total number of samples that were available for retest

Indeterminate Rate by Specimen Type   

<table><tr><td colspan="1" rowspan="2">Specimen Type</td><td colspan="2" rowspan="1">Initial Indete rminate Rate</td><td colspan="2" rowspan="1">Final Indeterminate Rate a</td></tr><tr><td colspan="1" rowspan="1">Percent (n)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Percent (n)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Vaginal Clinician-Collected</td><td colspan="1" rowspan="1">0.8% (20/2,517)</td><td colspan="1" rowspan="1">(0.5%, 1.2%)</td><td colspan="1" rowspan="1">0.2% (6/2,517)</td><td colspan="1" rowspan="1">(0.1%, 0.5%)</td></tr><tr><td colspan="1" rowspan="1">Vaginal Patient-Collected</td><td colspan="1" rowspan="1">0.9% (23/2,525)</td><td colspan="1" rowspan="1">(0.6%, 1.4%)</td><td colspan="1" rowspan="1">0.1% (3/2,525)</td><td colspan="1" rowspan="1">(0.0%, 0.3%)</td></tr><tr><td colspan="1" rowspan="1">Endocervical</td><td colspan="1" rowspan="1">0.8% (19/2,519)</td><td colspan="1" rowspan="1">(0.5%, 1.2%)</td><td colspan="1" rowspan="1">0.1% (2/2,516)</td><td colspan="1" rowspan="1">(0.0%, 0.3%)</td></tr><tr><td colspan="1" rowspan="3">Specimen TypeLBC PreservCyt</td><td colspan="2" rowspan="1">Initial Indete rminate Rate</td><td colspan="2" rowspan="1">Final Indeterminate Rate a</td></tr><tr><td colspan="1" rowspan="1">Percent (n)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">Percent (n)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">0.2% (6/2,473)</td><td colspan="1" rowspan="1">(0.1%, 0.5%)</td><td colspan="1" rowspan="1">0.0% (0/2,471)</td><td colspan="1" rowspan="1">(0.0%, 0.2%)</td></tr><tr><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">0.2% (9/3,621)</td><td colspan="1" rowspan="1">(0.1%, 0.5%)</td><td colspan="1" rowspan="1">0.1% (3/3,620)</td><td colspan="1" rowspan="1">(0.0%, 0.2%)</td></tr></table>

a Based on the total number of samples that were available for retest

Incomplete Rate by Specimen Type   

<table><tr><td rowspan=2 colspan=1>Specimen Type</td><td rowspan=1 colspan=2>Initial Incomple te Rate</td><td rowspan=1 colspan=2>Final Incomplete Rate a</td></tr><tr><td rowspan=1 colspan=1>Percent (n)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Percent(n)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Vaginal Clinician-Collected</td><td rowspan=1 colspan=1>0.7% (17/2,517)</td><td rowspan=1 colspan=1>(0.4%, 1.1%)</td><td rowspan=1 colspan=1>0.0% (0/2,517)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Vaginal Patient-Collected</td><td rowspan=1 colspan=1>0.6% (14/2,525)</td><td rowspan=1 colspan=1>(0.3%, 0.9%)</td><td rowspan=1 colspan=1>0.0% (1/2,525)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Endocervical</td><td rowspan=1 colspan=1>0.6% (14/2,519)</td><td rowspan=1 colspan=1>(0.3%, 0.9%)</td><td rowspan=1 colspan=1>0.0% (0/2,516)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>LBC PreservCyt</td><td rowspan=1 colspan=1>1.2% (30/2,473)</td><td rowspan=1 colspan=1>(0.9%, 1.7%)</td><td rowspan=1 colspan=1>0.0% (0/2,471)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr><tr><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>1.2% (45/3,621)</td><td rowspan=1 colspan=1>(0.9%, 1.7%)</td><td rowspan=1 colspan=1>)0.0% (0/3,620) (0.0%, 0.1%)</td><td rowspan=1 colspan=1>)0.0% (0/3,620) (0.0%, 0.1%)</td></tr></table>

a Based on the total number of samples that were available for retest

# 4. Clinical cut-off:

Not applicable

# 5. Expected values/Reference range:

The prevalence of infection with CT and/or NG, and/or TV in patient populations depends on risk factors such as age, gender, the type of clinic, and the sensitivity of the test used to detect infections. The observed positivity rates of the BD MAX CTGCTV2 assay in females during the clinical study are shown below.

BD MAX CTGCTV2 Observed Positivity Rate in Females   

<table><tr><td colspan="1" rowspan="2">Site</td><td colspan="5" rowspan="1">Chlamydia trachomatis</td><td colspan="5" rowspan="1">Neisseria gonorrhoeae</td><td colspan="3" rowspan="1">Trichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">CCVSa</td><td colspan="1" rowspan="1">SCVSb</td><td colspan="1" rowspan="1">Endo</td><td colspan="1" rowspan="1">LBCc</td><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">CCVSa</td><td colspan="1" rowspan="1">SCVSb</td><td colspan="1" rowspan="1">Endo</td><td colspan="1" rowspan="1">LBCc</td><td colspan="1" rowspan="1">Urine</td><td colspan="1" rowspan="1">CCVSa</td><td colspan="1" rowspan="1">SCVSb</td><td colspan="1" rowspan="1">Urine</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3.4%10/294</td><td colspan="1" rowspan="1">3.4%10//297</td><td colspan="1" rowspan="1">3.3%10/299</td><td colspan="1" rowspan="1">2.7%8/298</td><td colspan="1" rowspan="1">3.1%9/295</td><td colspan="1" rowspan="1">0.3%1/294</td><td colspan="1" rowspan="1">0.7%2/297</td><td colspan="1" rowspan="1">0.3%1//299</td><td colspan="1" rowspan="1">0.3%1/298</td><td colspan="1" rowspan="1">0.3%1//295</td><td colspan="1" rowspan="1">23.1%68/295</td><td colspan="1" rowspan="1">23.7%1/299</td><td colspan="1" rowspan="1">23.4%69/295</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3.7%18/486</td><td colspan="1" rowspan="1">5.1%25/490</td><td colspan="1" rowspan="1">4.7%23/489</td><td colspan="1" rowspan="1">3.8%18/471</td><td colspan="1" rowspan="1">4.7%23/489</td><td colspan="1" rowspan="1">1.0%5/486</td><td colspan="1" rowspan="1">0.8%4/489</td><td colspan="1" rowspan="1">0.6%3/489</td><td colspan="1" rowspan="1">0.4%2/471</td><td colspan="1" rowspan="1">0.8%/489</td><td colspan="1" rowspan="1">10.7%52/486</td><td colspan="1" rowspan="1">12.0%5/490</td><td colspan="1" rowspan="1">11.6%57/491</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5.3%7/132</td><td colspan="1" rowspan="1">6.0%8/133</td><td colspan="1" rowspan="1">4.5%6/132</td><td colspan="1" rowspan="1">3.8%5/133</td><td colspan="1" rowspan="1">4.5%6/134</td><td colspan="1" rowspan="1">2.3%3/132</td><td colspan="1" rowspan="1">2.3%3/133</td><td colspan="1" rowspan="1">2.3%3/132</td><td colspan="1" rowspan="1">2.3%3/133</td><td colspan="1" rowspan="1">2.2%3/134</td><td colspan="1" rowspan="1">6.8%9/132</td><td colspan="1" rowspan="1">7.5%10/133</td><td colspan="1" rowspan="1">7.5%10/134</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">17.2%5/29</td><td colspan="1" rowspan="1">17.2%5/29</td><td colspan="1" rowspan="1">17.2%5//29</td><td colspan="1" rowspan="1">13.8%4/29</td><td colspan="1" rowspan="1">10.3%3/29</td><td colspan="1" rowspan="1">3.4%1/29</td><td colspan="1" rowspan="1">3.4%1/29</td><td colspan="1" rowspan="1">3.4%1/29</td><td colspan="1" rowspan="1">3.4%1/29</td><td colspan="1" rowspan="1">3.4%1//29</td><td colspan="1" rowspan="1">20.7%6//29</td><td colspan="1" rowspan="1">20.7%6/29</td><td colspan="1" rowspan="1">20.7%6//29</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7.1%13/12</td><td colspan="1" rowspan="1">7.1%13/182</td><td colspan="1" rowspan="1">5.5%10/182</td><td colspan="1" rowspan="1">6.0%11/182</td><td colspan="1" rowspan="1">7.7%14/182</td><td colspan="1" rowspan="1">3.3%6/182</td><td colspan="1" rowspan="1">3.3%6/182</td><td colspan="1" rowspan="1">2.7%5/182</td><td colspan="1" rowspan="1">2.7%5/182</td><td colspan="1" rowspan="1">2.7%5/182</td><td colspan="1" rowspan="1">14.8%27182</td><td colspan="1" rowspan="1">14.3%26/182</td><td colspan="1" rowspan="1">14.3%26/182</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">7.3%35/482</td><td colspan="1" rowspan="1">7.0%34/483</td><td colspan="1" rowspan="1">5.4%6/481</td><td colspan="1" rowspan="1">5.2%25/484</td><td colspan="1" rowspan="1">6.2%28/453</td><td colspan="1" rowspan="1">1.5%7/482</td><td colspan="1" rowspan="1">1.7%8/482</td><td colspan="1" rowspan="1">1.5%7/481</td><td colspan="1" rowspan="1">1.2%6/484</td><td colspan="1" rowspan="1">1.5%77/453</td><td colspan="1" rowspan="1">6.6%32/482</td><td colspan="1" rowspan="1">7.9%38/482</td><td colspan="1" rowspan="1">6.6%30/453</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">6.7%5/75</td><td colspan="1" rowspan="1">6.7%5/75</td><td colspan="1" rowspan="1">7.9%6/76</td><td colspan="1" rowspan="1">6.4%5/78</td><td colspan="1" rowspan="1">7.9%6/76</td><td colspan="1" rowspan="1">0.0%05</td><td colspan="1" rowspan="1">0.0%05</td><td colspan="1" rowspan="1">0.0%0/76</td><td colspan="1" rowspan="1">0.0%0/78</td><td colspan="1" rowspan="1">0.0%0/76</td><td colspan="1" rowspan="1">10.7%8/75</td><td colspan="1" rowspan="1">12.0%9/75</td><td colspan="1" rowspan="1">11.8%/76</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3.9%11/283</td><td colspan="1" rowspan="1">4.6%13/283</td><td colspan="1" rowspan="1">3.9%11/282</td><td colspan="1" rowspan="1">3.6% 0/276</td><td colspan="1" rowspan="1">3.7%10/272</td><td colspan="1" rowspan="1">0.0%0/283</td><td colspan="1" rowspan="1">0.0%0/282</td><td colspan="1" rowspan="1">0.0%0//282</td><td colspan="1" rowspan="1">0.0%0/276</td><td colspan="1" rowspan="1">0.0%0/272</td><td colspan="1" rowspan="1">1.4%4/283</td><td colspan="1" rowspan="1">1.1%3/282</td><td colspan="1" rowspan="1">1.1%3/272</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">12.8%3/257</td><td colspan="1" rowspan="1">11.6%30/259</td><td colspan="1" rowspan="1">11.2%29/259</td><td colspan="1" rowspan="1">10.6%25//235</td><td colspan="1" rowspan="1">10.9%2/256</td><td colspan="1" rowspan="1">6.2%16/257</td><td colspan="1" rowspan="1">6.6% 17/259</td><td colspan="1" rowspan="1">6.2%16/257</td><td colspan="1" rowspan="1">6.8%16/235</td><td colspan="1" rowspan="1">5.5%14/256</td><td colspan="1" rowspan="1">18.7%48/257</td><td colspan="1" rowspan="1">18.1%7/259</td><td colspan="1" rowspan="1">16.8%3/256</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">9.4%12/127</td><td colspan="1" rowspan="1">11.0%4/127</td><td colspan="1" rowspan="1">10.2%13/127</td><td colspan="1" rowspan="1">7.9%0/127</td><td colspan="1" rowspan="1">7.9%10/127</td><td colspan="1" rowspan="1">4.7%6/127</td><td colspan="1" rowspan="1">4.7%6/127</td><td colspan="1" rowspan="1">4.7%6/127</td><td colspan="1" rowspan="1">4.7%6/127</td><td colspan="1" rowspan="1">4.7%6/127</td><td colspan="1" rowspan="1">7.9% 10/127</td><td colspan="1" rowspan="1">7.9%10/127</td><td colspan="1" rowspan="1">7.9%10/127</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">2.5% 4/157</td><td colspan="1" rowspan="1">1.9%3/156</td><td colspan="1" rowspan="1">0.6%1/157</td><td colspan="1" rowspan="1">0.0%00/157</td><td colspan="1" rowspan="1">1.9%3//157</td><td colspan="1" rowspan="1">0.0%00/157</td><td colspan="1" rowspan="1">0.0%0/156</td><td colspan="1" rowspan="1">0.0%0/157</td><td colspan="1" rowspan="1">0.0%00/157</td><td colspan="1" rowspan="1">0.0%00/157</td><td colspan="1" rowspan="1">1.3%2//157</td><td colspan="1" rowspan="1">1.3%2/156</td><td colspan="1" rowspan="1">1.3%2/157</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">6.1%153/2504</td><td colspan="1" rowspan="1">6.4%160/2514</td><td colspan="1" rowspan="1">5.6%14/2513</td><td colspan="1" rowspan="1">4.9%121/2470</td><td colspan="1" rowspan="1">5.7%140/2470</td><td colspan="1" rowspan="1">1.8%45/2504</td><td colspan="1" rowspan="1">1.9% 47/2511</td><td colspan="1" rowspan="1">1.7%42/511</td><td colspan="1" rowspan="1">1.6%40/2470</td><td colspan="1" rowspan="1">1.7%411/24700</td><td colspan="1" rowspan="1">10.6%266/2505</td><td colspan="1" rowspan="1">11.2%281/2514</td><td colspan="1" rowspan="1">10.7%265/2472</td></tr></table>

a Clinician-collected vaginal swab b Self-collected vaginal swab c PreservCyt Solution

BD MAX CTGCTV2 Observed Positivity Rate in Males   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17.0%8/47</td><td rowspan=1 colspan=1>2.1%1/47</td><td rowspan=1 colspan=1>0.0%0/47</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18.9%28/148</td><td rowspan=1 colspan=1>23.6%35/148</td><td rowspan=1 colspan=1>2.0%3/148</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12.0%35/291</td><td rowspan=1 colspan=1>9.6%28/291</td><td rowspan=1 colspan=1>5.2%15/291</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10.0%36/359</td><td rowspan=1 colspan=1>7.2%26/359</td><td rowspan=1 colspan=1>3.3%12/359</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>13.3%13/98</td><td rowspan=1 colspan=1>4.1%4/98</td><td rowspan=1 colspan=1>5.1%5/98</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>17.6%35/199</td><td rowspan=1 colspan=1>14.6%29/199</td><td rowspan=1 colspan=1>7.5%15/199</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13.6%155/1,142</td><td rowspan=1 colspan=1>10.8%123/1,142</td><td rowspan=1 colspan=1>4.4%50/1,142</td></tr></table>

# 6. Positive and Negative Predictive Values

Hypothetical Positive Predictive Value (PPV) and Negative Predictive Value (NPV) were calculated for patient- and clinician-collected vaginal swabs, endocervical swabs, male urine and PreservCyt LBC specimens and are shown in Table 8. The calculations are based on observed sensitivity and specificity for each specimen type, as compared to the Patient Infected Status.

Hypothetical Predictive Values for the BD MAX CTGCTV2 Assay   

<table><tr><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>HypotheticalPrevalence</td><td rowspan=1 colspan=2>Chlamydia trachomatis</td><td rowspan=1 colspan=2>Neisseria gonorrhoeae</td><td rowspan=1 colspan=2>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>% PPV95% CI</td><td rowspan=1 colspan=1>% NPV95% CI</td><td rowspan=1 colspan=1>% PPV95% CI</td><td rowspan=1 colspan=1>% NPV95% CI</td><td rowspan=1 colspan=1>% PPV95% CI</td><td rowspan=1 colspan=1>% NPV95% CI</td></tr><tr><td rowspan=7 colspan=1>Vaginala</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>44.9%(36.3%, 53.9%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td><td rowspan=1 colspan=1>87.5%(73.8%, 96.1%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td><td rowspan=1 colspan=1>61.7%(47.3%, 76.4%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td></tr><tr><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>62.2%(53.5%, 70.3%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td><td rowspan=1 colspan=1>93.4%(85.1%, 98.0%)</td><td rowspan=1 colspan=1>100%(99.8%, 100%)</td><td rowspan=1 colspan=1>76.5%(64.5%, 86.7%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>80.9%(74.8%, 85.9%)</td><td rowspan=1 colspan=1>99.9%(99.7%, 100%)</td><td rowspan=1 colspan=1>97.3%(93.6%, 99.2%)</td><td rowspan=1 colspan=1>99.9%(99.6%, 100%)</td><td rowspan=1 colspan=1>89.4%(82.4%, 94.4%)</td><td rowspan=1 colspan=1>99.9%(99.7%, 100%)</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>90.0%(86.2%, 92.8%)</td><td rowspan=1 colspan=1>99.8%(99.4%, 100%)</td><td rowspan=1 colspan=1>98.7%(96.9%, 99.6%)</td><td rowspan=1 colspan=1>99.9%(99.1%, 100%)</td><td rowspan=1 colspan=1>94.7%(90.8%, 97.3%)</td><td rowspan=1 colspan=1>99.8%(99.4%, 99.9%)</td></tr><tr><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>93.4%(90.9%, 95.3%)</td><td rowspan=1 colspan=1>99.7%(99.0%, 99.9%)</td><td rowspan=1 colspan=1>99.2%(98.0%, 99.8%)</td><td rowspan=1 colspan=1>99.8%(98.5%, 100%)</td><td rowspan=1 colspan=1>96.6%(94.0%, 98.3%)</td><td rowspan=1 colspan=1>99.6%(99.1%, 99.9%)</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>95.3%(93.4%, 96.7%)</td><td rowspan=1 colspan=1>99.6%(98.6%, 99.9%)</td><td rowspan=1 colspan=1>99.4%(98.6%, 99.8%)</td><td rowspan=1 colspan=1>99.7%(97.9%, 100%)</td><td rowspan=1 colspan=1>97.6%(95.7%, 98.8%)</td><td rowspan=1 colspan=1>99.5%(98.7%, 99.8%)</td></tr><tr><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>96.4%(94.9%, 97.5%)</td><td rowspan=1 colspan=1>99.5%(98.2%, 99.9%)</td><td rowspan=1 colspan=1>99.6%(98.9%, 99.9%)</td><td rowspan=1 colspan=1>99.6%(97.3%, 100%)</td><td rowspan=1 colspan=1>98.2%(96.7%, 99.1%)</td><td rowspan=1 colspan=1>99.3%(98.2%, 99.7%)</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>55.8%(44.5%, 66.7%)</td><td rowspan=1 colspan=1>99.9%(99.9%, 100%)</td><td rowspan=1 colspan=1>96.0%(80.8%, 99.3%)</td><td rowspan=1 colspan=1>100%(99.8%, 100%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>71.9%(61.8%, 80.2%)</td><td rowspan=1 colspan=1>99.9%(99.8%, 99.9%)</td><td rowspan=1 colspan=1>98.0%(89.5%, 99.6%)</td><td rowspan=1 colspan=1>99.9%(99.7%, 100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>86.8%(80.7%, 91.2%)</td><td rowspan=1 colspan=1>99.7%(99.4%, 99.9%)</td><td rowspan=1 colspan=1>99.2%(95.6%, 99.9%)</td><td rowspan=1 colspan=1>99.8%(99.2%, 99.9%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=4 colspan=1>Endocervical</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>93.3%(89.8%, 95.7%)</td><td rowspan=1 colspan=1>99.4%(98.8%, 99.7%)</td><td rowspan=1 colspan=1>99.6%(97.9%, 99.9%)</td><td rowspan=1 colspan=1>99.5%(98.3%, 99.9%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>95.7%(93.3%, 97.2%)</td><td rowspan=1 colspan=1>99.0%(98.1%, 99.5%)</td><td rowspan=1 colspan=1>99.8%(98.7%, 100%)</td><td rowspan=1 colspan=1>99.2%(97.3%, 99.8%)</td><td rowspan=2 colspan=1>-</td><td rowspan=2 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>96.9%(95.2%, 98.0%)</td><td rowspan=1 colspan=1>98.6%(97.3%, 99.3%)</td><td rowspan=1 colspan=1>99.8%(99.0%, 100%)</td><td rowspan=1 colspan=1>98.9%(96.3%, 99.7%)</td></tr><tr><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>97.7%(96.4%, 98.5%)</td><td rowspan=1 colspan=1>98.2%(96.5%, 99.1%)</td><td rowspan=1 colspan=1>99.9%(99.3%, 100%)</td><td rowspan=1 colspan=1>98.5%(95.1%, 99.6%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=7 colspan=1>LBCPreservCyt</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>81.5%(65.2%, 91.1%)</td><td rowspan=1 colspan=1>99.9%(99.9%, 100%)</td><td rowspan=1 colspan=1>95.8%(80.1%, 99.2%)</td><td rowspan=1 colspan=1>99.9%(99.8%, 100%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>89.9%(79.1%, 95.4%)</td><td rowspan=1 colspan=1>99.9%(99.7%, 99.9%)</td><td rowspan=1 colspan=1>97.9%(89.0%, 99.6%)</td><td rowspan=1 colspan=1>99.9%(99.6%, 99.9%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>95.8%(90.7%, 98.2%)</td><td rowspan=1 colspan=1>99.6%(99.3%, 99.8%)</td><td rowspan=1 colspan=1>99.2%(95.4%, 99.9%)</td><td rowspan=1 colspan=1>99.6%(99.0%, 99.9%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>98.0%(95.4%, 99.1%)</td><td rowspan=1 colspan=1>99.2%(98.5%, 99.6%)</td><td rowspan=1 colspan=1>99.6%(97.8%, 99.9%)</td><td rowspan=1 colspan=1>99.2%(97.9%, 99.7%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>98.7%(97.0%, 99.4%)</td><td rowspan=1 colspan=1>98.7%(97.7%, 99.3%)</td><td rowspan=1 colspan=1>99.7%(98.6%, 100%)</td><td rowspan=1 colspan=1>98.8%(96.8%, 99.6%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>99.1%(97.9%, 99.6%)</td><td rowspan=1 colspan=1>98.2%(96.8%, 99.0%)</td><td rowspan=1 colspan=1>99.8%(99.0%, 100%)</td><td rowspan=1 colspan=1>98.2%(95.5%, 99.4%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>99.3%(98.4%, 99.7%)</td><td rowspan=1 colspan=1>97.6%(95.8%, 98.7%)</td><td rowspan=1 colspan=1>99.9%(99.3%, 100%)</td><td rowspan=1 colspan=1>97.7%(94.0%, 99.2%)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=7 colspan=1>MaleUrine</td><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>61.6%(42.5%, 77.8%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td><td rowspan=1 colspan=1>91.1%(64.4%, 98.3%)</td><td rowspan=1 colspan=1>100%(100%, 100%)</td><td rowspan=1 colspan=1>78.3%(55.1%, 91.4%)</td><td rowspan=1 colspan=1>100%(99.9%, 100%)</td></tr><tr><td rowspan=1 colspan=1>2%</td><td rowspan=1 colspan=1>76.5%(59.9%, 87.6%)</td><td rowspan=1 colspan=1>99.9%(99.8%, 100%)</td><td rowspan=1 colspan=1>95.4%(78.5%, 99.2%)</td><td rowspan=1 colspan=1>100.0%(99.9%, 100%)</td><td rowspan=1 colspan=1>87.9%(71.3%, 95.5%)</td><td rowspan=1 colspan=1>100%(99.8%, 100%)</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>89.3%(79.4%, 94.8%)</td><td rowspan=1 colspan=1>99.8%(99.6%, 99.9%)</td><td rowspan=1 colspan=1>98.2%(90.4%, 99.7%)</td><td rowspan=1 colspan=1>100%(99.8%, 100%)</td><td rowspan=1 colspan=1>94.9%(86.5%, 98.2%)</td><td rowspan=1 colspan=1>99.9%(99.4%, 100%)</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>94.6%(89.1%, 97.5%)</td><td rowspan=1 colspan=1>99.6%(99.2%, 99.8%)</td><td rowspan=1 colspan=1>99.1%(95.2%, 99.8%)</td><td rowspan=1 colspan=1>99.9%(99.5%, 100%)</td><td rowspan=1 colspan=1>97.5%(93.1%, 99.1%)</td><td rowspan=1 colspan=1>99.8%(98.8%, 100%)</td></tr><tr><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>96.6%(92.8%, 98.4%)</td><td rowspan=1 colspan=1>99.4%(98.7%, 99.8%)</td><td rowspan=1 colspan=1>99.4%(96.9%, 99.9%)</td><td rowspan=1 colspan=1>99.9%(99.2%, 100%)</td><td rowspan=1 colspan=1>98.4%(95.5%, 99.5%)</td><td rowspan=1 colspan=1>99.6%(98.1%, 99.9%)</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>97.5%(94.8%, 98.9%)</td><td rowspan=1 colspan=1>99.2%(98.2%, 99.6%)</td><td rowspan=1 colspan=1>99.6%(97.8%, 99.9%)</td><td rowspan=1 colspan=1>99.8%(98.9%, 100%)</td><td rowspan=1 colspan=1>98.9%(96.8%, 99.6%)</td><td rowspan=1 colspan=1>99.5%(97.3%, 99.9%)</td></tr><tr><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>98.1%(96.1%, 99.1%)</td><td rowspan=1 colspan=1>98.9%(97.6%, 99.5%)</td><td rowspan=1 colspan=1>99.7%(98.4%, 99.9%)</td><td rowspan=1 colspan=1>99.7%(98.5%, 100%)</td><td rowspan=1 colspan=1>99.2%(97.6%, 99.7%)</td><td rowspan=1 colspan=1>99.3%(96.5%, 99.9%)</td></tr></table>

a The sensitivity and specificity estimates for the patient-and clinician-collected vaginal swabs are similar; the PPV and NPV for vaginal swabs was calculated based on the averages of those estimates.

N. Instrument Name: BD MAX System

# O. System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes X or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes or No X

2. Software:

a. FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types: Yes X or No   
b. Level of Concern: Moderate   
c. Device Hazard Analysis: Acceptable

The hazards exclusively related to the CTGCTV2 ADF (e.g., inaccurate temperature, incorrect magnet position, incorrect volume transfer, etc.) are included in the BD MAX System hazard analysis (HA) (BALTRM-MAXINST-APH). All hazards were properly documented and mitigated.

# 3. Specimen Identification:

Specimens are labeled with a unique bar code and following the BD MAX Specimen Collection Kit package insert.

# 4. Specimen Sampling and Handling:

Specimens are handled through the BD MAX sample processing subsystem, which utilizes a fluid handling robot for sampling. The instrument system includes the BD Pre-Warm Heater (required for LBC specimens only), which is completely controlled by the BD MAX System and does not require user intervention to operate. The specimens are loaded onto the BD MAX System where automated processing and reporting of assay results are completed.

# 5. Calibration:

BD MAX does not require user calibration. All maintenance procedures, other than daily and weekly cleaning procedures, are performed by BD qualified personnel.

# 6. Quality Control:

The assay includes a Specimen Processing Control (SPC) that is present in every Extraction Tube. The SPC monitors DNA extraction steps, thermal cycling steps, reagent integrity and the presence of inhibitory substances. External Control materials are treated as if they were patient samples; the BD MAX software does not use QC results for the interpretation of test results. External controls are not provided by BD and must be sourced by the user.

It is recommended that one External Positive Control and one External Negative Control be run at least daily until adequate process validation is achieved on the BD MAX System in each laboratory setting.

The External Positive Control is intended to monitor for substantial reagent failure. The External Negative Control is intended to detect reagent or environmental contamination (or carry-over) by target nucleic acids.

P. Other Supportive Instrument Performance Characteristics Data Not Covered in the Performance Characteristics” Section above:

Not Applicable.

# Q. Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.