# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K192073   
B Applicant Helix OpCo, LLC

# C Proprietary and Established Names

The Helix Genetic Health Risk App for late-onset Alzheimer's disease

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PTA</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.5950 -Genetic Health RiskAssessment System</td><td rowspan=1 colspan=1>IM - Immunology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New Device

# B Measurand:

rs7412 SNP and rs429358 SNP in genomic DNA obtained from a human saliva sample

# C Type of Test:

The Helix Genetic Health Risk App for late-onset Alzheimer’s disease is intended to provide the late-onset Alzheimer’s disease risk report. The report is based on a qualitative genetic test for detecting single nucleotide polymorphisms (SNP), rs429358 and rs7412, and for reporting e2, e3 and e4 alleles in the APOE gene. The Helix Genetic Health Risk App is for over-the-counter.

# A Intended Use / Indications for Use:

The Helix Genetic Health Risk App (HRA) uses qualitative genotyping to detect clinically relevant variants in genomic DNA isolated from human saliva collected from individuals $\geq 1 8$ years with Oragene Dx OGD-610 for the purpose of reporting and interpreting Genetic Health Risks (GHR):

The Helix Genetic Health Risk App (HRA) for late-onset Alzheimer’s disease is indicated for reporting of the e2/e2, e2/e3, e3/e3, $\mathrm { e } 2 / \mathrm { e } 4$ , $\mathrm { e } 3 / \mathrm { e } 4$ and $\mathrm { e 4 / e 4 }$ genotypes in the APOE gene. The report describes if a person's genetic result is associated with an increased or decreased risk of developing late-onset Alzheimer’s disease. The e2 and ${ \tt e 4 }$ variants included in this report are found and have been studied in many ethnicities. Detailed risk estimates have been studied the most in people of European descent.

The Helix Genetic Health Risk App (HRA) is to be used with the Helix Laboratory Platform.

# B Special Conditions for Use Statement(s):

For over-the-counter (OTC) use. The customer must first opt-in to receive the late-onset Alzheimer’s disease risk report. The test is intended for users $\geq 1 8$ years old. This test is not intended to diagnose a disease, determine medical treatment or other medical intervention, or tell the user anything about their current state of health. Any diagnostic or treatment decisions must be based on confirmatory prescription testing and/or other information that a healthcare professional determines to be appropriate for the patient, such as additional clinical testing and other risk factors that may affect individual risk and health care.   
• This test is not a substitute for visits to a healthcare provider. It is recommended that you consult with a healthcare provider if you have any questions or concerns about your results.   
• Subject to meeting the limitations contained in the special controls under regulation 21 CFR 866.5950. The laboratory may not be able to process a user’s sample. The probability that the laboratory cannot process a sample can be up to $2 . 5 \%$ . A user’s race, ethnicity, age, and sex may affect how the genetic test results are interpreted.   
• The test does not detect all genetic variants associated with late-onset Alzheimer’s Disease (AD). The absence of a variant tested does not rule out the presence of other genetic variants that may be disease-related. The test does not describe a person's overall risk of developing Alzheimer’s Disease. In addition, other genetic and all non-genetic factors should be considered. Third-party application developers must discuss with FDA the usability of the original data generated on the Helix Laboratory Platform (HLP).

# C Special Instrument Requirements:

Helix Laboratory Platform

# IV Device/System Characteristics:

# A Device Description:

The HRA is an OTC and direct-to-consumer DNA genetic testing service. The HRA for lateonset Alzheimer’s disease reports the lifetime risk of developing Alzheimer’s disease at or above age 65 years based on six genotypes (e2/e2, e2/e3, e2/e4, $\mathrm { e } 3 / \mathrm { e } 3$ , $\mathrm { e } 3 / \mathrm { e } 4$ and $\mathrm { e 4 / e 4 }$ ) of the APOE gene combining qualitative genotyping data with descriptive information derived from peerreviewed published genetic research studies. A customer’s saliva is self-collected using the Oragene Dx OGD-610 (K192920) manufactured by DNA Genotek, Inc., which consists of a sealable collection tube containing a stabilizing buffer solution. Once the saliva sample is collected, it is shipped to the Clinical Laboratory Improvement Amendments (CLIA)-certified and College of American Pathologists (CAP)-accredited Helix Clinical Laboratory for testing.

DNA is isolated from the saliva and tested using Helix’s proprietary whole exome sequencing assay in the Helix clinical laboratory. The genomic DNA is processed and sequenced using next generation sequencing (NGS) reagents and instrumentation manufactured by Illumina. The sequencing data is analyzed using Helix’s proprietary software, where the genetic variants of interest are determined. All samples must pass Helix’s quality control metrics prior to analysis. Samples that do not pass quality thresholds undergo re-sequencing and/or sample re-collection.

The APOE genotypes are used to generate personalized reports for each user. These reports tell the user which genetic variant(s) has/have been detected in their sample and provide information on the risk of disease associated with the genetic variants. If no variant was detected, that information is also provided. The personalized reports are designed to present scientific concepts to users in an easy-to-understand format. The reports provide scientifically valid information about the risks associated with the presence of a particular genetic variant. The reports are designed to help users understand the meaning of their results and any appropriate actions that may be taken based on their results.

# B Principle of Operation:

DNA is isolated from a user’s saliva sample and tested on the HLP. The HLP detects the rs429358 and rs7412 variants in the APOE gene and determines each user’s APOE genotype. The HRA interprets and categorizes each user’s APOE genotype (e2/e2, e2/e3, e2/e4, e3/e3, $\mathrm { e } 3 / \mathrm { e } 4$ and $\mathrm { e 4 / e 4 }$ ) into risk groups (decreased, average, and increased) based on the estimated lifetime risk of developing late-onset Alzheimer’s disease.

# V Substantial Equivalence Information:

A Predicate Device Name(s): 23andMe PGS Genetic Health Risk Report for Late-onset Alzheimer’s Disease   
B Predicate 510(k) Number(s): DEN160026

# C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp;PredicateDevice(s):</td><td colspan="1" rowspan="1">DeviceK192073</td><td colspan="1" rowspan="1">PredicateDEN160026</td></tr><tr><td colspan="1" rowspan="1">Device TradeName</td><td colspan="1" rowspan="1">The Helix Genetic Health Risk Appfor late-onset Alzheimer's disease</td><td colspan="1" rowspan="1">23andMe PGS Genetic HealthRisk Report for Late-onsetAlzheimer's Disease</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">IntendedUse/IndicationsFor Use</td><td colspan="1" rowspan="1">The Helix Genetic Health Risk App(HRA) uses qualitative genotypingto detect clinically relevant variantsin genomic DNA isolated fromhuman saliva collected fromindividuals ≥18 years with OrageneDx OGD-610 for the purpose ofreporting and interpreting GeneticHealth Risks (GHR):The Helix Genetic Health Risk App(HRA) for late-onset Alzheimer'sdisease is indicated for reporting ofthe e2/e2, e2/e3, e3/e3, e2/e4, e3/e4and e4/e4 genotypes in the APOEgene. The report describes if aperson's genetic result is associatedwith an increased or decreased riskof developing late-onsetAlzheimer's disease. The e2 and e4variants included in this report arefound and have been studied inmany ethnicities. Detailed riskestimates have been studied themost in people of Europeandescent.</td><td colspan="1" rowspan="1">The 23andMe Personal GenomeService (PGS) Test usesqualitative genotyping to detectthe following clinically relevantvariants in genomic DNA isolatedfrom human saliva collected fromindividuals ≥18 years with theOragene Dx model OGD-500.001for the purpose of reporting andinterpreting Genetic Health Risks(GHR):The 23andMe PGS Genetic HealthRisk Report for Late-onsetAlzheimer's Disease is indicatedfor reporting of the ε4 variant inthe APOE gene. The reportdescribes if a person's geneticresult is associated with anincreased risk of developing Late-onset Alzheimer's Disease, but itdoes not describe a person'soverall risk of developingAlzheimer's Disease. The ε4variant included in this report isfound and has been studied inmany ethnicities. Detailed riskestimates have been studied the</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">The Helix Genetic Health Risk App(HRA) is to be used with the HelixLaboratory Platform.</td><td colspan="1" rowspan="1">most in people of Europeandescent.</td></tr><tr><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Class Ⅱ</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product code</td><td colspan="1" rowspan="1">PTA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.5950</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target population</td><td colspan="1" rowspan="1">≥ 18 years old</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofresults</td><td colspan="1" rowspan="1">For over the counter use (OTC).Specialized interpretation by aphysician not required</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Human factors</td><td colspan="1" rowspan="1">User comprehension testing</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Design</td><td colspan="1" rowspan="1">Software application that includesproduct information page, e-commerce (registration and orderDNA kit), secure login, downloadgenetic report</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample type</td><td colspan="1" rowspan="1">Saliva</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Variants detectedand allelesreported</td><td colspan="1" rowspan="1">APOE genotype (e2) is the result ofa 'T' nucleotide at the rs7412 SNPand a 'T' nucleotide at thers429358 SNP.APOE genotype (e3) is the result ofa 'C' nucleotide at the rs7412 SNPand a 'T' nucleotide at thers429358 SNP.APOE genotype (e4) is the result ofa 'C' nucleotide at the rs7412 SNPand a 'C' nucleotide at thers429358 SNP.</td><td colspan="1" rowspan="1">rs429358 SNP for APOE e4 allele</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Next Generation Sequencing</td><td colspan="1" rowspan="1">Microarray genotyping</td></tr><tr><td colspan="1" rowspan="1">Method andsequencingplatform</td><td colspan="1" rowspan="1">Whole exome sequencing isconducted using Helix LaboratoryPlatform (HLP) and pipelineanalysis using Helix BioinformaticsPipeline (FDA-cleared underDEN190035).</td><td colspan="1" rowspan="1">Magnetic bead DNA extraction isperformed using Tecan Evo.Genotyping and PCR analysis isconducted using a chip-basedmethod with Illumina Infinium'sBeadChip v4 assay, the IlluminaiScan System for detection and</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">analysis is facilitated with the useof GenomeStudio and Coregensoftware.</td></tr><tr><td colspan="1" rowspan="1">Specimencollection kit</td><td colspan="1" rowspan="1">DNA Genotek Inc., Oragene DxOGD-610</td><td colspan="1" rowspan="1">DNA Genotek Inc., Oragene DxOGD500.001</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Guidance for Industry and Food and Drug Administration Staff - Format for Traditional and Abbreviated 510(k)s (2019) • Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices (2005) General Principles of Software Validation Guidance for Industry and FDA Staff (2002) Off-The-Shelf Software Use in Medical Devices Guidance for Industry and Food and Drug Administration Staff (2019) Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (2014) Class II Special Control: 866.5950 - Code of Federal Regulations Title 21 Direct-to-Consumer test (https://www.fda.gov/medical-devices/vitro-diagnostics/directconsumer-tests)

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

All results presented below met the sponsor’s pre-defined acceptance criteria outlined in the Special Controls of 21 CFR 866.5950. Information regarding samples that failed quality control (FQC) was also evaluated and presented in each study below.

# 1. Precision / Reproducibility:

Reproducibility of the APOE genotype calls made by the HRA for late-onset Alzheimer’s disease was assessed by testing 24 samples that include six human Blymphocyte cell lines (6 cell lines hereafter) and 18 unique saliva-derived DNA samples) in two independent studies (Study 1 and Study 2). The APOE genotype calls for the in the well-characterized cell line samples were compared to known APOE genotypes.

Study 1 tested 24 samples (6 cell lines and 18 unique saliva-derived DNA samples) with up to 72 replicates (3 replicates / sample / library prep plate x 3 plates x 2 enrichments $_ { \textrm { X 4 } }$ independent runs of cBot and HiSeq instruments) for the APOE genotype calls. The test results are summarized below:

<table><tr><td>Cell Line</td><td>N</td><td>APOE Genotype</td><td>No. of replicates</td><td>No. of Correct Genotype Calls</td><td>No. of Incorrect Genotype Calls</td><td>No. of FQC</td><td>Percent of FQC (%)</td><td>Percent Correct Genotype Calls (%)</td></tr></table>

Study 1: Cell lines   

<table><tr><td rowspan=1 colspan=1>NA24385</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24149</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA12877</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24143</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24631</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

Study 1: Clinical Samples   

<table><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>276</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>504</td><td rowspan=1 colspan=1>484</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Study 2 tested 24 samples (6 cell lines and 18 unique saliva-derived DNA samples) with up to 54 replicates (3 replicates / sample / library prep plate x 3 plates $_ { \textrm { X 2 } }$ enrichments x 3 reagent lots) for the APOE genotype calls. The test results are summarized below:

<table><tr><td rowspan=1 colspan=1>Cell Line</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percent ofFQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=9>Study 2: Cell lines</td></tr><tr><td rowspan=1 colspan=1>NA24385</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24149</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA12878</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA12877</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24143</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

Stud 2: Clinical Samples   

<table><tr><td rowspan=1 colspan=1>Cell Line</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percent ofFQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=1>NA24631</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>11.1</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>376</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

The calls for concordance the APOE genotypes were analyzed in the clinical samples. Genotyping results produced $100 \%$ of replicates that were called correctly for all APOE genotypes.

2. Linearity:

Not applicable

# 3. Analytical Specificity / Interference:

Four studies were performed to determine the effects of substances found in saliva that may interfere with the performance of the HRA for late-onset Alzheimer’s disease. Per study protocol, for sequenced samples to be included in the data analyses (e.g., evaluable samples), the sample(s) must pass pre-defined QC thresholds.

a. Study 1 evaluated four endogenous proteins commonly found in saliva. Each of the following were added to saliva samples: alpha-amylase $( 3 9 5 \mathrm { U / m L } )$ ), hemoglobin $( 2 0 \mathrm { m g / m L } )$ , immunoglobulin A (IgA) $( 0 . 4 3 ~ \mathrm { m g / m L } )$ , and albumin $\left( 2 . 6 7 \mathrm { m g / m L } \right)$ . The study showed that these proteins did not affect test performance for saliva samples $\scriptstyle ( \mathtt { n } = 2 9 - 3 0$ evaluable samples across four endogenous proteins). The test results are shown below.

<table><tr><td colspan="1" rowspan="1">APOEGenotype</td><td colspan="1" rowspan="1">No. ofreplicates</td><td colspan="1" rowspan="1">No. ofCorrectGenotypeCalls</td><td colspan="1" rowspan="1">No. ofIncorrectGenotypeCalls</td><td colspan="1" rowspan="1">No. ofFQC</td><td colspan="1" rowspan="1">Percentof FQC(%)</td><td colspan="1" rowspan="1">PercentCorrectGenotype(%)</td></tr><tr><td colspan="1" rowspan="1">e2/e3</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">e2/e4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">e3/e3</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">e3/e4</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">e4/e4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td></tr></table>

b. In study 2, saliva samples were tested before and after (either immediately or 30 minutes after) exposure to one of four exogenous substances: eating food, drinking liquids, using mouthwash, or chewing gum. The study showed that exogenous substances did not interfere with test performance for saliva samples $( \mathrm { n } { = } 1 2 { - } 1 5$ evaluable samples across testing time for four exogenous substances. The test results are shown below.

<table><tr><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percentof FQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>N/A</td></tr></table>

c. Study 3 evaluated saliva samples tested at 60 minutes before smoking, immediately after smoking, and 30 minutes after smoking and showed that smoking did not interfere with test performance $\mathrm { \acute { n } } ^ { = 1 5 }$ evaluable samples across three smoking conditions). The test results are shown below.

<table><tr><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percentof FQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

d. In study 4, bacterial DNA from a commercial source (American Type Culture Collection) was added in various amounts into six cell line DNA samples (NA24385, NA24149, NA12878, NA12877, NA24143, and NA24631) to evaluate the effects of bacterial contamination in the test performance. This bacterial sample (ATCC MSA-1003) is comprised of 20 fully sequenced cultures that encompass a variety of characteristics including bacterial species found in mouse and oral cavity. The six cell lines were tested across five levels of bacterial content $( 0 \%$ , $10 \%$ , $20 \%$ , $30 \%$ , and $50 \%$ ). This study showed that microbial DNA did not interfere with test performance $\mathrm { n } { = } 1 1 { - } 1 8$ evaluable samples across five levels of bacterial content). Study 4 also tested various amounts of microbial and yeast DNA added to the saliva samples from three volunteers $0 \%$ , $10 \%$ Candida albicans, and $30 \%$ of ATCC MSA-1003). This study showed that yeast DNA did not interfere with test performance in clinical samples. The test results are shown below.

<table><tr><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percentof FQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotype Calls(%)</td></tr><tr><td rowspan=1 colspan=7>Cell lines</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=7>Clinical samples</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Refer to K192920 for pre-collection shelf-life stability of the collection device, stability of samples post-saliva collection, and freeze-thaw stability of samples stored in the Oragene Dx collection device.

# 6. Detection Limit:

The detection limit study evaluated the impact of different levels of DNA input on the performance of the HRA for late-onset Alzheimer’s disease. The study yielded concordant test results for all 19 saliva samples with known APOE genotypes when tested at sample DNA concentrations between 3.5 to $1 0 ~ \mathrm { n g / \mu L }$ , an input corresponding to a range of 35 to $1 0 0 ~ \mathrm { n g }$ of DNA in the library preparation.

<table><tr><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percent ofFQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotype Calls(%)</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

# B Comparison Studies:

# 1. Method Comparison:

Three method comparison studies were conducted to evaluate accuracy: the first study was conducted with human saliva samples with known genotypes and a second study was conducted with human cell line samples with known genotypes to determine the rates of correct APOE genotype calls. A third study was conducted with human saliva and/or human cell line samples with known variants in genes other than the APOE gene to determine the agreement of the genotype calls.

a) Accuracy study with human saliva samples:

Accuracy of the HRA for late-onset Alzheimer’s disease was evaluated by testing human saliva samples with known APOE genotypes. The presence of the two variants in the APOE gene was analyzed on the HLP and the genotyping results were compared to the known genotypes confirmed by Sanger sequencing (comparator). The accuracy for detecting the two variants in the APOE gene on the HLP (genotype call) using DNA isolated from human saliva samples $\scriptstyle ( \mathrm { n } = 9 9 )$ was $100 \%$ with a lower bound of $9 5 \%$ CI as $9 6 . 3 \%$ for all samples tested. The test results are shown in the table below.

<table><tr><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percent ofFQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>All</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

b) Accuracy study with human cell line samples:

Accuracy for detecting two variants, rs7412 and rs429358, in the APOE gene on the HLP was evaluated by using six cell lines with known APOE genotypes (e2/e3, e2/e4, e3/e3, $\mathrm { e } 3 / \mathrm { e } 4$ , $\mathrm { e } 3 / \mathrm { e } 4$ , and $\mathrm { e 4 / e 4 }$ ). The accuracy for detecting the two variants in the APOE gene was $100 \%$ for all samples tested.

<table><tr><td rowspan=1 colspan=1>Cell Line</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>APOEGenotype</td><td rowspan=1 colspan=1>No. ofreplicates</td><td rowspan=1 colspan=1>No. ofCorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofIncorrectGenotypeCalls</td><td rowspan=1 colspan=1>No. ofFQC</td><td rowspan=1 colspan=1>Percentof FQC(%)</td><td rowspan=1 colspan=1>PercentCorrectGenotypeCalls(%)</td></tr><tr><td rowspan=1 colspan=1>NA24385</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24149</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA12878</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA12877</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24143</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>NA24631</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr></table>

c) Accuracy study with 3,295 human saliva and cell line samples with known variants in genes other than APOE:

In addition to the two variants in the APOE gene, accuracy of the HRA for lateonset Alzheimer’s disease for detecting variants in the genes associated with other clinical conditions included in DEN160026 was evaluated on the HLP using specimens carrying unique genetic variants linked to the specific clinical conditions as listed in the table below:

<table><tr><td rowspan=1 colspan=1>Clinical condition</td><td rowspan=1 colspan=1>Gene</td><td rowspan=1 colspan=1>SNP</td></tr><tr><td rowspan=1 colspan=1>Hereditary Thrombophilia</td><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>rs6025</td></tr><tr><td rowspan=1 colspan=1>Hereditary Thrombophilia</td><td rowspan=1 colspan=1>F2</td><td rowspan=1 colspan=1>rs1799963</td></tr><tr><td rowspan=1 colspan=1>Alpha-1 Antitrypsin Deficiency</td><td rowspan=1 colspan=1>SERPINA1</td><td rowspan=1 colspan=1>rs28929474</td></tr><tr><td rowspan=1 colspan=1>Alpha-1 Antitrypsin Deficiency</td><td rowspan=1 colspan=1>SERPINA1</td><td rowspan=1 colspan=1>rs17580</td></tr><tr><td rowspan=1 colspan=1>Late-Onset Alzheimer&#x27;s Disease</td><td rowspan=1 colspan=1>APOE</td><td rowspan=1 colspan=1>rs429358</td></tr><tr><td rowspan=1 colspan=1>Parkinson&#x27;s Disease</td><td rowspan=1 colspan=1>LRRK2</td><td rowspan=1 colspan=1>rs34637584</td></tr><tr><td rowspan=1 colspan=1>Parkinson&#x27;s Disease; Gaucher Disease Type 1</td><td rowspan=1 colspan=1>GBA</td><td rowspan=1 colspan=1>rs76763715</td></tr><tr><td rowspan=1 colspan=1>Gaucher Disease Type 1</td><td rowspan=1 colspan=1>GBA</td><td rowspan=1 colspan=1>rs387906315</td></tr><tr><td rowspan=1 colspan=1>Gaucher Disease Type 1</td><td rowspan=1 colspan=1>GBA</td><td rowspan=1 colspan=1>rs80356769</td></tr><tr><td rowspan=1 colspan=1>Factor XI Deficiency</td><td rowspan=1 colspan=1>FXI</td><td rowspan=1 colspan=1>rs121965064</td></tr><tr><td rowspan=1 colspan=1>Factor XI Deficiency</td><td rowspan=1 colspan=1>FXI</td><td rowspan=1 colspan=1>rs121965063</td></tr><tr><td rowspan=1 colspan=1>Factor XI Deficiency</td><td rowspan=1 colspan=1>FXI</td><td rowspan=1 colspan=1>rs373297713</td></tr><tr><td rowspan=1 colspan=1>Celiac Disease</td><td rowspan=1 colspan=1>HLA-DQA1</td><td rowspan=1 colspan=1>rs2187668</td></tr><tr><td rowspan=1 colspan=1>Glucose-6-Phosphate-DehydrogenaseDeficiency</td><td rowspan=1 colspan=1>G6PD</td><td rowspan=1 colspan=1>rs1050828</td></tr><tr><td rowspan=1 colspan=1>Hereditary Hemochromatosis</td><td rowspan=1 colspan=1>HFE</td><td rowspan=1 colspan=1>rs1800562</td></tr><tr><td rowspan=1 colspan=1>Hereditary Hemochromatosis</td><td rowspan=1 colspan=1>HFE</td><td rowspan=1 colspan=1>rs1799945</td></tr><tr><td rowspan=1 colspan=1>Early-Onset Primary Dystonia</td><td rowspan=1 colspan=1>DYT1</td><td rowspan=1 colspan=1>rs724159981</td></tr></table>

The overall number of true positive, false positives, true negative and false negatives were analyzed for 17 variants in 11 genes. A total of 4,282 true positive and 51,731 true negative calls were reported. There were no false positive and no false negative results reported for the known variants tested and one no call was reported each for rs76763715 and rs429358.

2. Matrix Comparison: Not applicable

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

1) Risk of developing late-onset Alzheimer’s disease and APOE allele frequency

Late-onset Alzheimer’s disease is characterized by onset of the disease in people at or older than age 65 years. Many factors have been suggested to be associated with a decreased or increased risk of developing late-onset Alzheimer’s disease, including age, genetic variants, family history of dementia, and lifestyle. The association of the APOE e2 and ${ \tt e 4 }$ alleles with the modified risk of developing late-onset Alzheimer’s disease is one of the well-known genetic determinants across various ethnicities based on peer-reviewed scientific literature. The clinical performance and genetic variant detection by the HRA for late-onset Alzheimer’s disease are supported by peer-reviewed scientific literature. The APOE e2 allele is associated with a decreased risk to develop late-onset Alzheimer's disease at or above age 65 years, whereas the APOE e4 allele is associated with an increased risk to develop late-onset Alzheimer's disease at or above age 65 years. The HLP detects the two single nucleotide polymorphisms (SNPs), rs429358 and rs7412, that determine the status of the e2, e3, and ${ \tt e 4 }$ alleles of the APOE gene.

The different combinations of the APOE alleles (e2, e3 and e4) were grouped based on the risk of developing late-onset Alzheimer’s disease as follows:

a. Individuals with the APOE $\mathrm { e } 3 / \mathrm { e } 3$ genotype have an average risk of developing late-onset Alzheimer’s disease.

b. Individuals with the APOE $\mathrm { e } 2 / \mathrm { e } 2$ or $\mathrm { e } 2 / \mathrm { e } 3$ genotype have a decreased risk of developing late-onset Alzheimer’s disease compared to individuals with the APOE $\mathrm { e } 3 / \mathrm { e } 3$ genotype for the European, African, and South Asian descent.   
c. Individuals with the APOE $\mathrm { e } 2 / \mathrm { e } 4$ , $\mathrm { e } 3 / \mathrm { e } 4$ , or $\mathrm { e 4 / e 4 }$ genotype have an increased risk of developing late-onset Alzheimer’s disease compared to individuals with the APOE $\mathrm { e } 3 / \mathrm { e } 3$ genotype.   
d. Individuals who have none of these APOE genotypes will not receive information on their relative risk to develop late-onset Alzheimer’s disease due to the lack of studies in the scientific literature.

Three APOE alleles covered by this test are found in people of all ethnicities. The percentage of the population carrying each of the six possible APOE genotypes will vary from study to study based on the enrollment criteria of the genetic study. A summary of the frequency of each APOE genotype in different ethnicities collected from multiple published peer-reviewed scientific studies is presented in the table below:

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=6>Allele frequency (%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>Reference</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Decreased risk</td><td rowspan=1 colspan=1>Averagerisk</td><td rowspan=1 colspan=3>Increased risk</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Europeandescent</td><td rowspan=1 colspan=1>AD*</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>14.8</td><td rowspan=2 colspan=1>Farrer et al.,1997</td></tr><tr><td rowspan=1 colspan=1>controls</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>60.9</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=2 colspan=1>AfricanAmericandescent</td><td rowspan=1 colspan=1>AD</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>13.0</td><td rowspan=2 colspan=1>Murrell et al.,2006</td></tr><tr><td rowspan=1 colspan=1>controls</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>15.1</td><td rowspan=1 colspan=1>51.3</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=2 colspan=1>SouthAsiandescent</td><td rowspan=1 colspan=1>AD</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>44.8</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>5.3</td><td rowspan=2 colspan=1>Agarwal et al.,2014</td></tr><tr><td rowspan=1 colspan=1>controls</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>72.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>11.7</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>East Asian</td><td rowspan=1 colspan=1>AD</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>51.0</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>7.6</td><td rowspan=2 colspan=1>Liu et al., 2014</td></tr><tr><td rowspan=1 colspan=1>descent</td><td rowspan=1 colspan=1>controls</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>73.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>0.3</td></tr></table>

${ } ^ { * } \mathbf { A } \mathbf { D } =$ Alzheimer’s disease

Using the same references, the likelihood ratios (LR) for APOE e2, e3 and e4 combinations in different ethnicities, which represent an estimate of how the test results affects the chances of a condition was calculated as shown in the table below.

<table><tr><td rowspan=1 colspan=1>Ethnicity</td><td rowspan=1 colspan=1>Test result</td><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>LR</td><td rowspan=1 colspan=1>95% CI forLR</td><td rowspan=1 colspan=1>References</td><td rowspan=1 colspan=1>Study summary</td></tr><tr><td rowspan=6 colspan=1>Europeandescent</td><td rowspan=2 colspan=1>Decreasedrisk</td><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.1-0.5</td><td rowspan=6 colspan=1>Farrer et al.,1997</td><td rowspan=6 colspan=1>A meta-analysis of 5,930patients who met criteria forprobable or definiteAlzheimer&#x27;s disease and 8,607controls. Among studyparticipants, there were 5,107Alzheimer&#x27;s disease patientsfrom European descent, and6,262 controls from Europeandescent.</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.30.4</td></tr><tr><td rowspan=1 colspan=1>Average risk</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.570.62</td></tr><tr><td rowspan=3 colspan=1>Increasedrisk</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.8-1.3</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.82.1</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>6.8-10.0</td></tr><tr><td rowspan=6 colspan=1>AfricanAmericandescent</td><td rowspan=2 colspan=1>Decreasedrisk</td><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.042.7</td><td rowspan=6 colspan=1>Murrell etal., 2006</td><td rowspan=6 colspan=1>This study included 162individuals from Africandescent (African American)with Alzheimer&#x27;s disease and318 controls from Africandescent (African American).</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3-0.9</td></tr><tr><td rowspan=1 colspan=1>Average risk</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.6-0.9</td></tr><tr><td rowspan=3 colspan=1>Increasedrisk</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.42.3</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.22.1</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>1.9-7.6</td></tr><tr><td rowspan=6 colspan=1>South Asiandescent</td><td rowspan=2 colspan=1>Decreasedrisk</td><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.02-0.9</td><td rowspan=6 colspan=1>Agarwal etal., 2014</td><td rowspan=6 colspan=1>A meta-analysis of 417individuals from South Asiandescent with Alzheimer&#x27;sdisease and 651 controls fromSouth Asian descent.</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3-0.8</td></tr><tr><td rowspan=1 colspan=1>Average risk</td><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.6-0.7</td></tr><tr><td rowspan=3 colspan=1>Increasedrisk</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>2.0-8.3</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>2.54.1</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>2.314.0</td></tr><tr><td rowspan=6 colspan=1>East Asiandescent</td><td rowspan=3 colspan=1>Average risk</td><td rowspan=1 colspan=1>e2/e2</td><td rowspan=1 colspan=1>1.1*</td><td rowspan=1 colspan=1>0.5- 2.3</td><td rowspan=6 colspan=1>Liu et al.,2014</td><td rowspan=6 colspan=1>A meta-analysis of 1,576individuals from East Asiandescent with Alzheimer&#x27;sdisease and 1,741 controls fromEast Asian descent. (*limitednumber of e2/e2 samples usedfor analysis)</td></tr><tr><td rowspan=1 colspan=1>e2/e3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.5-0.8</td></tr><tr><td rowspan=1 colspan=1>e3/e3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.6-0.7</td></tr><tr><td rowspan=3 colspan=1>Increasedrisk</td><td rowspan=1 colspan=1>e2/e4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.23.0</td></tr><tr><td rowspan=1 colspan=1>e3/e4</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>2.22.9</td></tr><tr><td rowspan=1 colspan=1>e4/e4</td><td rowspan=1 colspan=1>25.6</td><td rowspan=1 colspan=1>10.562.6</td></tr></table>

2) User comprehension study

a. Saliva collection kit user study: Refer to K192920 for the saliva collection device (Oragene Dx OGD-610) instructions for use and to assess the ability of lay users to provide samples adequate for testing.

b. HRA user comprehension study:

# Objectives:

The user comprehension study was performed to assess user comprehension of the HRA reports. The study was performed with a demographically diverse group of study participants (e.g. age, gender, ethnicity, and education level) by asking questions for five comprehension categories (purpose of test, limitation, relevant ethnicities, meaning of results, and appropriate follow-up) of the HRA in the prepurchase page and the test reports in a controlled online setting.

# Methods:

Quota-based sampling per U.S. Census was employed to recruit a diverse set of study participants based on their demographic variables in age, gender, ethnicity, and education level. Study participants who enrolled into the user comprehension study were randomly assigned into one of four risk category groups (increased, average, decrease, and result not available). Study participants took the test on their personal desktop. Voice and screen recordings of each study participant are captured during the test. The user comprehension was tested through a two-step process. First, participants' understanding of genetics was tested prior to viewing the pre-purchase page and test reports. Second, participants were shown the pre-purchase page and the test reports and completed the comprehension assessment. A total of 303 study participants met the inclusion criteria and were enrolled in the study. Fifty-nine (59) participants were non-complete users and removed from final analysis due to one of the following reasons across four risk category groups.

Exited test: an automated reason generated by Usertesting.com when their software detects there was some type of technical difficulty that resulted in the participant leaving the study prior to completion.   
• Session Expired is an automated reason generated by Usertesting.com when their software detects that a participant does not start the study within 30 minutes.   
• Uploading Issue is an automated reason generated by Usertesting.com when their software detects that a participant had issues uploading the test answers to the testing site.

The number of non-complete users in each risk category group were as follows: 16 users $( 2 0 \% )$ in the increased risk group, 9 users $( 1 2 . 8 \% )$ in the average risk group, 12 users $( 1 6 . 4 \% )$ in the decreased risk group, and 22 users $( 2 6 . 5 \% )$ in the result not available group. There were no timed-out users in this study. A total of 244 study participants assigned to a risk category group across all risk categories (increased, average, decrease, and result not available) were included in the comprehension rate analysis. The completion rate was $100 \%$ for all 244 study participants.

# Results:

The overall comprehension rate per comprehension category ranged 90.6 to $9 8 . 4 \%$ across all risk categories. The comprehension rate within the risk category ranged from 85.2 to $100 \%$ per comprehension category. The user comprehension study results are shown in the table below.

<table><tr><td rowspan=1 colspan=1>Comprehensioncategory</td><td rowspan=1 colspan=4>Comprehension rates (%)within Risk category</td><td rowspan=1 colspan=1>Overallcomprehensionrate (%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>IncreasedRisk</td><td rowspan=1 colspan=1>AverageRisk</td><td rowspan=1 colspan=1>DecreasedRisk</td><td rowspan=1 colspan=1>Result notavailable</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Purpose of Test</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>98.4</td></tr><tr><td rowspan=1 colspan=1>Limitation</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>92.6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>95.7</td></tr><tr><td rowspan=1 colspan=1>Relevant Ethnicities</td><td rowspan=1 colspan=1>90.2</td><td rowspan=1 colspan=1>91.8</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>91.8</td><td rowspan=1 colspan=1>91.8</td></tr><tr><td rowspan=1 colspan=1>Meaning of Results</td><td rowspan=1 colspan=1>85.2</td><td rowspan=1 colspan=1>89.6</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>90.6</td></tr><tr><td rowspan=1 colspan=1>Appropriate Follow-up</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>95.9</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>97.3</td></tr></table>

The comprehension rates were also analyzed by age, gender, race, and education level for each comprehension category. The comprehension rates ranged 87– $100 \%$ across ages, $9 3 - 9 9 \%$ across genders, $7 9 - 1 0 0 \%$ across ethnicities, and 87– $100 \%$ across education levels as summarized in the table as below.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Purpose of Test</td><td rowspan=1 colspan=2>Limitations</td><td rowspan=1 colspan=2>Relevant Ethnicities</td><td rowspan=1 colspan=2>Meaning ofResults</td><td rowspan=1 colspan=2>Appropriate follow-up</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>correctresponses /totalresponses</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>correctresponses /totalresponses</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>correctresponses /totalresponses</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>correctresponses /totalresponses</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>correctresponses /totalresponses</td></tr><tr><td rowspan=1 colspan=11>Age (group)</td></tr><tr><td rowspan=1 colspan=1>18-24</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>74/80</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>35/40</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>75/80</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>78/80</td></tr><tr><td rowspan=1 colspan=1>25-34</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>43/43</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>81/86</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>41/43</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>85/85</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96/86</td></tr><tr><td rowspan=1 colspan=1>35-44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>82/84</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>39/42</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>81/84</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>84/84</td></tr><tr><td rowspan=1 colspan=1>45-54</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>86/88</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>39/44</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>84/88</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88/88</td></tr><tr><td rowspan=1 colspan=1>55-64</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35/36</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>70/72</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>33/36</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>68/72</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>70/72</td></tr><tr><td rowspan=1 colspan=1>65+</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>37/39</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>74/78</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>37/39</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>74/78</td></tr><tr><td rowspan=1 colspan=11>Gender (male or female)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>123/124</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>233/240</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>112/120</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>232/240</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>234/240</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>117/120</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>234/248</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>112/124</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>237/248</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>246/248</td></tr><tr><td rowspan=1 colspan=11>Race/Ethnicity</td></tr><tr><td rowspan=1 colspan=1>Caucasian</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>139/140</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>272/280</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>131/140</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>272/281</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>278/280</td></tr><tr><td rowspan=1 colspan=1>Black</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>32/34</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>63/68</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>27/34</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>62/68</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>64/68</td></tr><tr><td rowspan=1 colspan=1>Hispanic /Latino</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>40/41</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>77/82</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>39/41</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>77/81</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>80/82</td></tr><tr><td rowspan=1 colspan=1>All otherethnicities</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>55/58</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>27/29</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>58/58</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>58/58</td></tr><tr><td rowspan=1 colspan=11>Education Level</td></tr><tr><td rowspan=1 colspan=1>High School</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>83/84</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>158/168</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>76/84</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>158/168</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>166/168</td></tr><tr><td rowspan=1 colspan=1>SomeCollege</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>53/54</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>105/108</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>50/54</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>100/107</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>106/108</td></tr><tr><td rowspan=1 colspan=1>Collegeor equivalent</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>71/73</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>142/146</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>69/74</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>142/146</td></tr><tr><td rowspan=1 colspan=1>PostGraduate</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>62/66</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>29/33</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>67/69</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>66/66</td></tr></table>

# c. Frequently Asked Questions Material:

A Frequently Asked Questions (FAQ) section was developed and included in each Genetic Health Risk Report. The FAQ section was to provide users information to adequately understand the purpose, limitations, and the meaning of the results of the test, and was developed using methodology consistent with the Manufacturer's labeling design, identification of communication messages, and label comprehension. The concepts covered in the FAQ section include: the test results, the purpose of the test, limitations of the test, relevance of race and ethnicity on test results, the meaning of the result, other risks factors that contribute to disease, appropriate follow-up procedures, how the results of the test may affect the user's family and children, and links to resources that provide additional information. The questions included in the FAQ section for each of the tests report included the following:

• What does this test do?   
• What does this test not do?   
• My report says my result is $\mathrm { e } 2 / \mathrm { e } 2$ what does this mean about my late onset Alzheimer's disease risk?   
• What does "decreased risk" mean?   
• Based on my result, what are some things I could do?   
• The report says that estimates about risk are best studied in people of European descent. What if I am not of European descent?   
• How could these results affect my family? Where can I find references for this information? How private is my result?

Each Genetic Health Report has answers to the FAQs that are specific to the variant(s) and disease being reported, where applicable.

# d. User Opt-In page:

Due to the nature of late-onset Alzheimer’s disease, users are informed on the prepurchase page, which shows information on the user opt-in section prior to adding the HRA for late-onset Alzheimer’s disease to the (purchase) cart as indicated below.

Are you sure you want to view your Helix Genetic Health Risk App for late-onset Alzheimer's Disease test report?   
By purchasing this product, you are opting-in to receive personal health   
information for Alzheimer's disease alone. This test will indicate whether you are at an increased, decreased or average risk of developing late-onset Alzheimer's disease. Note that there are currently no genetic testing guidelines or treatment options available for Alzheimer's disease. Talk to a healthcare provider if you have any questions or concerns.

Before the report is generated, users are asked to opt-in for viewing the report as below.

• Are you sure you want to view your Alzheimer's Disease Gene Test report? • This test will indicate whether you are at an increased, decreased or average risk of developing Alzheimer's disease. Note that there are currently no genetic testing guidelines or treatment options available for Alzheimer's disease. Talk to a healthcare provider if you have any questions or concerns.

i. Yes, I want to view my report ii. No, I do not want to view my report iii. Ask me again later

# D Clinical Cut-off:

Not applicable

# E Expected Values/Reference Range

Not applicable

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.