# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

# C Proprietary and Established Names

AncestryDNA Factor V Leiden Genetic Health Risk Test

# D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PTA</td><td rowspan=1 colspan=1>Class I</td><td rowspan=1 colspan=1>21 CFR 866.5950 -Genetic Health RiskAssessment System</td><td rowspan=1 colspan=1>IM - Immunology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New Device

# B Measurand:

Factor V Leiden c.1601G>A variant in the F5 gene from a human saliva sample

# C Type of Test:

The AncestryDNA Factor V Leiden Genetic Health Risk Test for Hereditary Thrombophilia determines and interprets if a person has variants associated with a higher risk of developing harmful blood clots. The report is based on a qualitative genetic test for a single nucleotide polymorphism detection of Factor V Leiden variant in the F5 gene (rs6025).

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The AncestryDNA Factor V Leiden Genetic Health Risk Test for Hereditary Thrombophilia uses qualitative genotyping to detect clinically relevant variants in genomic DNA isolated from human saliva collected from individuals 18 years and older with the AncestryDNA Saliva Collection Kit for the purpose of reporting and interpreting Genetic Health Risks (GHR).

The AncestryDNA Factor V Leiden Genetic Health Risk Test for Hereditary Thrombophilia is indicated for reporting of the Factor V Leiden variant in the F5 gene. This report describes if a person has variants associated with a higher risk of developing harmful blood clots, but it does not describe a person's overall risk of developing harmful blood clots. This test is most relevant for people of European descent.

# C Special Conditions for Use Statement(s):

OTC - Over the Counter

This test is not a substitute for visits to a healthcare provider. It is recommended that you consult with a healthcare provider if you have any questions or concerns about your results or your current state of health.

The AncestryDNA Factor V Leiden Genetic Health Risk Test does not detect all genetic variants associated with Hereditary Thrombophilia. The absence of a variant tested does not rule out the presence of other genetic variants that may be disease-related.

The test is intended for users $\geq 1 8$ years old.

The test does not diagnose any specific health conditions. Results should not be used to make medical decisions.

The laboratory may not be able to process a user's sample. The probability that the laboratory cannot process a sample can be up to $3 \%$ .

A user's race, ethnicity, age, and other life style factors may affect how the genetic test results are interpreted.

Subject to meeting the limitations contained in the special controls under regulation 21 CFR 866.5950.

D Special Instrument Requirements:

The AncestryDNA Factor V Leiden Genetic Health Risk Test is to be performed using the Tecan Evo and Illumina iScan instruments.

GenomeStudio Software is a modular software application that is used to view and analyze genotypic data obtained from the Illumina iScan System and based on the cluster definition file. AncestryDNA Factor V Leiden Genetic Health Risk Test software conducts a variety of control checks on the file, resulting in a final genotype profile for each sample. These data are used to generate test reports on a user’s genotype and associated risk of disease.

# IV Device/System Characteristics:

# A Device Description:

The AncestryDNA Genetic Health Risk (GHR) Test for Factor V Leiden is intended for the detection of single nucleotide polymorphism (SNP) in the F5 gene associated with Hereditary Thrombophilia from human saliva collected in the AncestryDNA Saliva Collection Kit (SCK).

The user’s saliva is self-collected using the AncestryDNA SCK, which consists of a single-use sealable collection tube containing a stabilizing buffer solution. Once the sample is collected, it is shipped to one of two Clinical Laboratory Improvement Amendments (CLIA) certified laboratories for processing, testing and analysis. DNA is extracted, fragmented and tested with the AncestryDNA Factor V Leiden GHR Test, a multiplex assay using a customized genotyping chip and instrumentation manufactured by Illumina. The multiplex assay simultaneously tests for more than 500,000 variants, including those for the indication proposed herein.

The raw data is generated using the Illumina GenomeStudio software and then delivered to Ancestry Genomics for analysis using the proprietary AncestryDNA GHR Software. A genotype is determined for each tested variant. The results for the Factor V Leiden variant are used to generate personalized reports for users which provide information about the disease associated with the detected variant.

Personalized reports are generated for each user that provide results of the testing performed. These reports tell the user which variant(s) has/have been detected in their sample and provide information on the risk of disease associated with the variant(s). If no variant was detected, that information is also provided. The personalized reports are designed to present scientific concepts to users in an easy-to-understand format. The reports provide scientific information about the risks associated with the presence of a particular variant. The reports are designed to help users understand the meaning of their results and any appropriate actions that may be taken based on their results.

# B Principle of Operation:

The AncestryDNA Factor V Leiden GHR Test is performed by CLIA-certified laboratories using the BeadChip v10 assay (Illumina Infinium HumanOmniExpress-24 format chip) on the Illumina Infinium platform. Samples collected using the AncestryDNA Saliva Collection Kit are delivered to laboratories for testing and analysis. DNA from saliva is extracted and quantified. Samples

with DNA concentrations in the range of $1 . 5 3 ~ \mathrm { n g / \mu L } - 5 0 ~ \mathrm { n g / \mu L }$ are eligible for further processing. These samples are fragmented and captured on a bead array by hybridization to immobilized variant-specific primers, followed by extension with hapten-labeled nucleotides. The primers hybridize adjacent to the variants and are extended with a single nucleotide corresponding to the variant allele. The incorporated hapten-modified nucleotides are detected by adding fluorescently labeled antibodies in several steps to amplify the signals. The Tecan Evo and Illumina iScan instruments are used for extraction and processing of the DNA, and the BeadChip for scanning and quantification of the results. Genotypes are determined using the GenomeStudio software package and delivered to Ancestry Genomics for analysis using the AncestryDNA GHR software.

# C Instrument Description Information:

<table><tr><td rowspan=1 colspan=1>Modes of Operation</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device contain the ability to transmit data to a computer,webserver, or mobile device?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>□</td></tr><tr><td rowspan=1 colspan=1>Does the applicant&#x27;s device transmit data to a computer, webserver, or mobile deviceusing wireless transmission?</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Software</td></tr><tr><td rowspan=1 colspan=1>FDA has reviewed applicant&#x27;s Hazard Analysis and software development processesfor this line of product types.</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr></table>

1. Instrument Name:

Tecan Evo, Illumina iScan BeadChip scanner with GenomeStudio software

# 2. Specimen Identification:

Consumers must register their saliva collection kit, linking their saliva sample to a secure online account with a valid email address through a unique activation code, in order to use the test. The activation code is matched to records of kits shipped to consumers to ensure it is a valid kit. A timestamp of the user completing the entries to activate the kit is recorded.

# 3. Specimen Sampling and Handling:

Samples are collected using the AncestryDNA Saliva Collection Kit and delivered to one of two CLIA-certified laboratories for testing and analysis. The recommended volume of saliva is $1 \mathrm { m L }$ . Saliva is collected directly by the user spitting into the provided saliva collection tube via the pre-installed funnel. After providing saliva, the user is instructed to remove the funnel and screw on tightly the provided cap. Affixing the cap by screwing on releases the stabilization solution. The saliva sample can be immediately processed, transported, or stored for future use. Device and sample integrity are preserved during typical ambient transport and storage conditions for up to 12 months.

4. Calibration:

Calibration and calibration verification procedures are established to demonstrate continued accuracy of the test systems.

# 5. Quality Control:

The AncestryDNA Factor V Leiden Genetic Health Risk Test uses one control material, which serves as both the sample processing control and the reproducibility control. The control material is genotyped on the Illumina BeadChip according to routine standard procedures at the laboratory. Each new lot of the control is tested by comparison with reference BeadChip genotype results.

The sample processing control is run on every sample genotyping plate and the reproducibility control is run approximately once per week.

# V Substantial Equivalence Information:

A Predicate Device Name(s):

23andMe Personal Genome Service Test for Hereditary Thrombophilia

B Predicate 510(k) Number(s):

DEN160026

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K192944</td><td colspan="1" rowspan="1">DEN160026</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">AncestryDNA Factor VLeiden Genetic Health RiskTest</td><td colspan="1" rowspan="1">23andMe Personal GenomeService (PGS) Test</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicSimilarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The AncestryDNA Factor VLeiden Genetic Health RiskTest for HereditaryThrombophilia usesqualitative genotyping todetect clinically relevantvariants in genomic DNAisolated from human salivacollected from individuals 18years and older with theAncestryDNA SalivaCollection Kit for the purposeof reporting and interpretingGenetic Health Risks (GHR).</td><td colspan="1" rowspan="1">The 23andMe PersonalGenome Service (PGS) Testuses qualitative genotyping todetect the following clinicallyrelevant variants in genomicDNA isolated from humansaliva collected fromindividuals ≥ 18 years withthe Oragene Dx model OGD-500.001 for the purpose ofreporting and interpretingGenetic Health Risks (GHR):The 23andMe PGS Genetic</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">The AncestryDNA Factor VLeiden Genetic Health RiskTest for HereditaryThrombophilia is indicated forreporting of the Factor VLeiden variant in the F5 gene.This report describes if aperson has variants associatedwith a higher risk ofdeveloping harmful bloodclots, but it does not describea person's overall risk ofdeveloping harmful bloodclots. This test is mostrelevant for people ofEuropean descent.</td><td colspan="1" rowspan="1">Health Risk Report forHereditary Thrombophilia isindicated for reporting of theFactor V Leiden variant in theF gene, and the ProthrombinG20210A variant in the F2gene. This report describes ifa person has variantsassociated with a higher riskof developing harmful bloodblots, but it does not describea person's overall risk ofdeveloping harmful bloodclots. This test is mostrelevant for people ofEuropean descent.</td></tr><tr><td colspan="1" rowspan="1">Special InstrumentRequirements</td><td colspan="1" rowspan="1">The AncestryDNA Factor VLeiden Genetic Health RiskTest is to be performed usingthe Tecan Evo and IlluminaiScan instruments.GenomeStudio Software is amodular software applicationthat is used to view andanalyze genotypic dataobtained from the IlluminaiScan System and based onthe cluster definition file.AncestryDNA GHR softwareconducts a variety of controlchecks on the file, resulting ina final genotype profile foreach sample. These data areused to generate test reportson a user's genotype andassociated risk of disease.</td><td colspan="1" rowspan="1">The 23andMe PGS GeneticHealth Risk Tests forHereditary Thrombophilia isto be performed using theTecan Evo and Illumina iScaninstruments.GenomeStudio is a modularsoftware application that isused to view and analyzegenotypic data obtained fromthe iScan. Coregen softwareconducts a variety of controlchecks on the file, resulting ina final genotype profile foreach sample. These data areused to generate test reportson a user's genotype andassociated risk of disease.</td></tr><tr><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">Factor V Leiden c.1601G&gt;Avariant in the F5 gene</td><td colspan="1" rowspan="1">Factor V Leiden variant(rs6025, c.1601G&gt;A) in theF5 gene and ProthrombinG20210A</td></tr><tr><td colspan="1" rowspan="1">Type of Test</td><td colspan="1" rowspan="1">Qualitative in vitro moleculardiagnostic system</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation</td><td colspan="1" rowspan="1">DNA extraction from humansaliva</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Calibration</td><td colspan="1" rowspan="1">Calibration and calibrationverification procedures areestablished to demonstratecontinued accuracy of the testsystems.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">The test report describes if aperson has variants associatedwith a higher risk ofdeveloping harmful bloodclots. The report does notdescribe a person's overallrisk of developing harmfulblood clots.</td><td colspan="1" rowspan="1">The report provides results ofwhich variant(s) has/havebeen detected and providedinformation on the risk ofdisease associated with thevariant(s).</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristicDifferences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Sample CollectionDevice</td><td colspan="1" rowspan="1">AncestryDNA SalivaCollection Kit (K192947)</td><td colspan="1" rowspan="1">Oragene®·Dx CollectionDevice, model OGD-500.001(K141410)</td></tr><tr><td colspan="1" rowspan="1">Analytical Sensitivity</td><td colspan="1" rowspan="1">The performance requirementfor the AncestryDNA GHRTest has been set at aminimum of 1.53 ng/μL DNAand maximum of 50 ng/μLDNA.</td><td colspan="1" rowspan="1">The performance requirementfor the PGS Test has been setat a minimum of 15 ng/μLDNA and maximum of 50ng/μL DNA.</td></tr><tr><td colspan="1" rowspan="1">Interfering Mutations</td><td colspan="1" rowspan="1">The potential interferingmutations includers770011773, rs773367113and rs143663052. The impactof these mutations on theperformance of the assay hasnot been evaluated.</td><td colspan="1" rowspan="1">For Factor V Leiden thepotential interfering mutationsinclude rs760488939 andrs763859650. The impact ofthese mutations on theperformance of the assay hasnot been evaluated.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

Special Controls for Genetic Health Risk Assessment System as detailed in 21 CFR 866.5950 CLSI EP07-A2, Interference Testing in Clinical Chemistry; Approved Guideline – Second Edition   
• CLSI EP12-A2, User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline – Second Edition   
• CLSI EP17-A2, Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition   
• CLSI EP37-A1, Supplement Tables for Interference Testing in Clinical Laboratory; Approved Guideline – First Edition   
• CLSI EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline

# A Analytical Performance:

# 1. Precision/Reproducibility:

The purpose of the precision studies was to determine the imprecision of the AncestryDNA Factor V Leiden GHR Test under the following changed conditions: assay run, critical reagent lot, instrument, operator, day and site. Saliva samples from nine donors, three for each genotype (homogenous common – GG, heterozygous – GA and homozygous rare – AA), were collected with three lots of the Ancestry DNA Saliva Collection Kit. Genotypes of these samples were confirmed through bi-directional sequencing.

The precision studies were performed at two independent CLIA-certified testing laboratories (Lab 1 and Lab 2). Saliva samples were genotyped by the AncestryDNA Factor V Leiden GHR Test using three lots of critical reagents by six different operator teams (three per laboratory) on eight instrument combinations (four per laboratory) over multiple days. The precision study yielded $100 \%$ correct genotype calls for all samples with a valid call across multiple days, operator teams, instruments and reagent lot at both laboratory sites.

Information regarding samples that failed quality control (FQC) was also evaluated. Sample replicates that did not pass sample call rate (SCR) QC in the first run underwent second, and when eligible, third genotyping run per laboratory standard operating procedures (SOPs). The overall precision exceeded $9 9 \%$ point estimate. The results of the study are demonstrated in the tables below.

Summary of the within-run repeatability results at Lab 1   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Total Numberof Replicates</td><td rowspan=1 colspan=1>Number ofConcordantCalls</td><td rowspan=1 colspan=1>Number of&quot;No-Calls&quot;</td><td rowspan=1 colspan=1>Number ofCall RateQC Failures</td><td rowspan=1 colspan=1>FQC(%)</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr></table>

Summary of the within-laboratory testing results at Lab 1   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Total Numberof Replicates</td><td rowspan=1 colspan=1>Number ofConcordantCalls</td><td rowspan=1 colspan=1>Number ofNo-Calls</td><td rowspan=1 colspan=1>Number ofCall RateQC Failures</td><td rowspan=1 colspan=1>FQC(%)</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>537</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.56</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>540</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1620</td><td rowspan=1 colspan=1>1617</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.19</td></tr></table>

Summary of the within-laboratory testing results at Lab 2   

<table><tr><td colspan="1" rowspan="1">Genotype</td><td colspan="1" rowspan="1">Total Numberof Replicates</td><td colspan="1" rowspan="1">Number ofConcordantCalls</td><td colspan="1" rowspan="1">Number of"No-Calls"</td><td colspan="1" rowspan="1">Number ofCall RateQC Failures</td><td colspan="1" rowspan="1">FQC(%)</td></tr><tr><td colspan="1" rowspan="1">GG</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">GA</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">AA</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">540</td><td colspan="1" rowspan="1">540</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.00</td></tr></table>

Summary of the inter-laboratory reproducibility   

<table><tr><td rowspan=2 colspan=1>Genotype</td><td rowspan=1 colspan=2>TotalNumber ofReplicates</td><td rowspan=1 colspan=2>Number ofConcordantCalls</td><td rowspan=1 colspan=2>Number ofNo-Calls&quot;</td><td rowspan=1 colspan=2>Number ofCall RateQC Failures</td><td rowspan=1 colspan=2>FQC(%)</td></tr><tr><td rowspan=1 colspan=1>Lab 1</td><td rowspan=1 colspan=1>Lab 2</td><td rowspan=1 colspan=1>Lab 1</td><td rowspan=1 colspan=1>Lab 2</td><td rowspan=1 colspan=1>Lab 1</td><td rowspan=1 colspan=1>Lab 2</td><td rowspan=1 colspan=1>Lab 1</td><td rowspan=1 colspan=1>Lab 2</td><td rowspan=1 colspan=1>Lab 1</td><td rowspan=1 colspan=1>Lab 2</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr></table>

2. Linearity:

Not applicable

# 3. Analytical Specificity/Interference:

The study was to establish the analytical specificity of the AncestryDNA Factor V Leiden GHR Test from samples collected using the AncestryDNA Saliva Collection Kit (SCK) by evaluating the impact of potential endogenous interferents on assay performance. A series of studies were conducted to assess the effects of endogenous substances, exogenous substances, microbial substances and mutation on the AncestryDNA Factor V Leiden GHR Test.

# i. Endogenous Interference

Saliva samples from ten donors were collected using the AncestryDNA SCK and their genotypes were determined with bi-directional sequencing. The endogenous substances were individually spiked into saliva prior to DNA extraction and genotyping. Saliva spiked with PBS served as the control. The results indicated that the performance of the AncestryDNA Factor V Leiden GHR Test is not affected by salivary ${ \bf { d } } \cdot { \bf { \Lambda } }$ -amylase (395 $\mathrm { U } / \mathrm { m L }$ ), hemoglobin $( 2 0 \mathrm { m g / m L } )$ ), IgA ( $0 . 4 4 \mathrm { m g / m L } )$ ) and total protein (including 0.185 $\mathrm { m g / m L }$ salivary ${ \bf \mathfrak { d } }$ -amylase, $0 . 4 4 ~ \mathrm { m g / m L }$ IgA and $2 . 0 5 ~ \mathrm { { m g / m L } }$ human serum albumin).

# ii. Exogenous Interference

The AncestryDNA Factor V Leiden GHR Test requires the use of an FDA cleared saliva collection kit (K192947). The saliva collection kit includes Instructions for Use (IFU) instructing the user not to eat, drink, smoke, or chew gum for 30 minutes prior to collecting their saliva, thus minimizing the presence of interferents in the sample. The saliva collection IFU was tested for user comprehension and a paper version of the IFU is included in every collection kit. Should an interfering substance be present after DNA has been extracted and an insufficient concentration or quality of DNA is available, then the sample is managed per standard operating procedures that are pre-determined by the manufacturer.

iii. Microbial Interference

Microbial DNA from Staphylococcus epidermis, Streptococcus mutans, Lactobacillus casei, Actinomyces odontolyticus and Candida albicans were tested to determine their impact on the performance of the AncestryDNA Factor V Leiden GHR Test. DNA from each of the six human cell lines was spiked with two concentrations (low/normal: 2.8 $\mathrm { n g / \mu L }$ ; high: $1 2 . 5 ~ \mathrm { { n g / \mu L } }$ ) of the five different species of microbial DNA. The results indicate that there is no significant impact of common microbial interferents on the performance of the AncestryDNA Factor V Leiden GHR test in either low/normal or high concentrations.

# iv. Mutational Interference

An in silico analyses was performed to identify potential interfering variants within the 50 nucleotide probe-binding region downstream of the variant being detected. The genome aggregation database (gnomAD database) was utilized to search for mutations in this region. Only mutations that have been observed in more than one individual were evaluated. Three potential interfering mutations (rs770011773, rs773367113 and rs143663052) were identified for the AncestryDNA Factor V Leiden GHR Test. The impact of these mutations on the performance of the assay has not been evaluated.

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

The AncestryDNA Factor V Leiden GHR Test uses one control material, which serves as both the sample processing control and the reproducibility control. The control material is genotyped on the Illumina BeadChip according to the standard of procedures at the laboratory. Each new lot of the control is tested by comparison with reference BeadChip genotype results.

The stability studies are ongoing and current data support a shelf-life stability claim of one month and an in-use stability claim of 30 days when the control material is stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ .

# 6. Detection Limit:

i. Limit of Blank

The Limit of Blank (LoB) study was performed to incorporate measurement variability due to the noisy nature of UV-Vis spectroscopy and potentially erroneous fluorescence signal in the assay. A plate of 95 blanks containing $1 ~ \mathrm { m L }$ of molecular grade water and the standard volume of DNA stabilizing solution from the AncestryDNA Saliva Collection Kit was extracted, quantified and genotyped using three lots of critical reagents. LoB calculation was performed using the non-parametric rank method in accordance with CLSI EP17-A2. No blanks returned a sample call rate above $9 8 \%$ . The LoB was determined to be $1 . 0 0 4 ~ \mathrm { n g / \mu L }$ .

ii. Limit of Detection

The Limit of Detection (LoD) study was performed to establish the lowest concentration of DNA obtained from the AncestryDNA Saliva Collection Kit that is necessary when tested with the AncestryDNA Factor V Leiden GHR Test to successfully assign the correct variant. The study used both saliva samples from 15 donors (five for each genotype) and DNA from four cell lines. DNA from each sample was diluted to multiple concentrations and genotyped by the AncestryDNA Factor V Leiden GHR Test using three lots of critical reagents. To confirm the genotype call, each sample was also sequenced by bi-directional sequencing. The LoD was defined as the lowest DNA concentration at which at least $9 5 \%$ of samples yielded the correct call and statistically different from LoB. The LoD was determined to be $1 . 5 3 ~ \mathrm { n g / \mu L }$ . The manufacturer has also claimed the upper limit of DNA concentration of $5 0 ~ \mathrm { n g / \mu L }$ .

7. Assay Cut-Off: Not applicable

8. Accuracy (Instrument): Refer to Section B.1.i.

9. Carry-Over: Not applicable

# B Comparison Studies:

1. Comparison with Sanger Bi-directional Sequencing

i. Accuracy

The accuracy study was performed at one laboratory site to calculate the agreement of the genetic variant determination between the AncestryDNA Factor V Leiden GHR Test and results from bi-directional sequencing. Saliva samples were collected from 209 donors with known Factor V Leiden genotypes: 73 homozygous common (GG), 69 heterozygous (GA) and 67 homozygous rare (AA). Of the 200 samples initially genotyped, 11 samples failed the sample call rate (SCR) quality control (QC) and nine alternate samples were added to the same cohort, all of which passed SCR QC on the first run. All samples were genotyped using bi-directional sequencing to determine their true variant status. Genotyping results were compared between the GHR test and bi-directional sequencing to calculate percent agreement with the sequencing results considered to be “truth”. The percent agreement for each genotype, the overall percent agreement and confidence intervals were calculated based on a binomial distribution. Data analysis results did not include FQC samples. The results of the study are summarized in the tables below.

Distribution of the 11 FQC samples by genotypes   

<table><tr><td rowspan=1 colspan=1>Genotype</td><td rowspan=1 colspan=1>Total Numberof Samples</td><td rowspan=1 colspan=1>Numberof FQC</td><td rowspan=1 colspan=1>FQC (%)</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.5</td><td rowspan=1 colspan=1>1.5% to13.4%</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>0.9% to12.5%</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>1.6% to 14.2%</td></tr></table>

Comparison of the genotyping results between AncestryDNA Factor V Leiden GHR Test and bi-directional sequencing   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Bi-directional Sequencing Genotypes</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=5 colspan=1>AncestryDNAFactor VLeiden GHRTest</td><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>64</td></tr><tr><td rowspan=1 colspan=1>No call or invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FQC</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>209</td></tr></table>

Percent agreement and confidence intervals for AncestryDNA Factor V Leiden GHR Test genotypes   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Correct/Incorrect</td><td rowspan=1 colspan=1>NoCall</td><td rowspan=1 colspan=1>FQC</td><td rowspan=1 colspan=1>PercentAgreement (PA)</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>GG</td><td rowspan=1 colspan=1>69/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.8% to 100%</td></tr><tr><td rowspan=1 colspan=1>GA</td><td rowspan=1 colspan=1>65/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.5% to 100%</td></tr><tr><td rowspan=1 colspan=1>AA</td><td rowspan=1 colspan=1>64/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.4% to 100%</td></tr><tr><td rowspan=1 colspan=1>All Genotypes</td><td rowspan=1 colspan=1>198/0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2% to 100%</td></tr><tr><td rowspan=1 colspan=1>No call</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/198</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0% to 1.8%</td></tr></table>

As reported in published literature, factor $\mathrm { V }$ Leiden heterozygosity is present in $5 . 1 \%$ , $2 . 0 \%$ and $1 . 2 \%$ of Caucasians, Hispanics and African Americans, respectively. The frequencies of homozygosity for the above populations are 65, 10 and 4 per 100,000 individuals correspondingly. Technical (analytical) Positive Predictive Value (TPPV) of the AncestryDNA Factor V Leiden GHR Test results are $100 \%$ for both GA and AA.

2. Matrix Comparison: Not applicable

# C Clinical Studies:

# 1. Clinical Summary

Clinical performance for this test was assessed using published data and studies to demonstrate user comprehension of the labeling and test results.

The AncestryDNA Factor V Leiden GHR Test for Hereditary Thrombophilia is indicated for the detection of the Factor V Leiden variant (c $\therefore 1 6 0 1 \mathrm { G } > \mathrm { A } )$ in the F5 gene. This variant is associated with higher risk for developing harmful blood clots or venous thromboembolism (VTE; includes deep vein thrombosis and pulmonary embolism (Simone et al., 2013)1 . Odds ratios are available in the meta-analysis performed by Simone et al. (2013). Odds ratios adjusted for age and sex using logistic regression and $9 5 \%$ confidence intervals for each variant status are given below:

<table><tr><td rowspan=1 colspan=1>Factor V Leiden variant status</td><td rowspan=1 colspan=1>Factor V Leiden variant status Odds ratio (95% confidence interval)</td></tr><tr><td rowspan=1 colspan=1>One copy, heterozygote</td><td rowspan=1 colspan=1>4.22 (3.35 to 5.32)</td></tr><tr><td rowspan=1 colspan=1>Two copies, homozygote</td><td rowspan=1 colspan=1>5.45 (6.79 to 19.29)</td></tr></table>

To support test results, a meta-analysis study conducted by Simone et al. (2013) was used to calculate likelihood ratios (LR, an estimate of how the test result affects the chances of a condition). Post-test risk $\mathrm { ( R _ { p o s t } ) }$ was calculated from the LR and using a pre-test risk $\mathrm { ( R _ { p r e } ) }$ of $1 1 \%$ from the 2018 ACMG venous thromboembolism laboratory testing standard (Zhang, 2018)2 . LR and $9 5 \%$ confidence intervals calculated using a normal approximation are listed in the table below along with the post-test risk calculated using the relationship $\mathrm { R } _ { \mathrm { p o s t } } / ( 1 { - } \mathrm { R } _ { \mathrm { p o s t } } )$ $= \mathrm { L R } \left[ \mathrm { R _ { p r e } } / ( 1 { - } \mathrm { R _ { p r e } } ) \right]$ .

<table><tr><td rowspan=1 colspan=1>FVL variantcount (genotype)</td><td rowspan=1 colspan=1>Case/control distribution</td><td rowspan=1 colspan=1>LR</td><td rowspan=1 colspan=1>95%confidenceinterval</td><td rowspan=1 colspan=1>Post-testrisk</td></tr><tr><td rowspan=1 colspan=1>0 variant(homozygouscommon / wildtype)</td><td rowspan=1 colspan=1>1758 cases and 1201 controls among 2959Factor V Leiden carriers. 7323 cases and16312 controls among 23635 wild typeindividuals.</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>0.86 to 0.88</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>1 variant(heterozygous)</td><td rowspan=1 colspan=1>1758 cases and 1201 controls among 2959Factor V Leiden carriers. 7323 cases and16312 controls among 23635 wild typeindividuals.</td><td rowspan=1 colspan=1>2.82</td><td rowspan=1 colspan=1>2.64 to 3.02</td><td rowspan=1 colspan=1>26%</td></tr><tr><td rowspan=1 colspan=1>2 variants(homozygousrare)</td><td rowspan=1 colspan=1>92 cases and 24 controls among 116Factor V Leiden homozygous individuals.4524 cases and 11643 controls among16167 wild type individuals.</td><td rowspan=1 colspan=1>9.69</td><td rowspan=1 colspan=1>6.19 to 15.16</td><td rowspan=1 colspan=1>54%</td></tr></table>

# 2. Other Clinical Supportive Data

a. User Comprehension Study

Two user comprehension studies (Study 1 and Study 2) were conducted to assess user comprehension of the AncestryDNA Factor V Leiden GHR Test process and reports. The user comprehension studies were performed using a sampling of individuals that were demographically diverse in a controlled environment. Quota-based sampling was used to recruit study participants representative of the U.S. population according to education, age, sex/gender and race/ethnicity.

In Study 1, geographic diversity was addressed through participant recruitment and comprehension testing was conducted from each of the four U.S. census regions: South, Midwest, Northeast and West. Approximately 120 participants were tested at each location. These subjects were randomly assigned to each of the four study arms, testing comprehension of four GHR Test reports:

• 0 Variants Identified • 1 Variant Identified • 2 Variants Identified Result Not Determined

The study was conducted in person and each interview session was administered by a trained interviewer/moderator using a series of pre-defined questions. Participants were given a time limit from start of report review to completion of the comprehension study. After excluding several participants according to pre-defined exclusion criteria, the final 378 participants completed the survey talk for the four AncestryDNA Factor V Leiden GHR Test reports and were included in the endpoint analysis. The manufacturer determined comprehension accuracy rates for multiple core comprehension concepts. The comprehension assessment addressed the overall comprehension and the following core comprehension concepts: purpose of test, limitations of test, ethnic relevance, results of test, other risk factors and appropriate follow-up action.

In Study 2, geographically diverse participants were randomly assigned to each of two study arms, representing the two most challenging reports:

1 Variant Identified Result Not Determined

The study was conducted via live tele-video interview and each interview session was administered online by a trained interviewer/moderator using a series of pre-defined questions. After excluding participants per pre-defined exclusion criteria, a total of 213 individuals completed the study and were included in the endpoint analysis.

Both studies were performed on the different types of the GHR Test reports developed using representative samples of the materials below (supplemental materials):

• Education Module including definition of terms Pre-purchase page   
• Frequently Asked Questions, and Technical Details

The representative samples were developed per FDA guidance for medical device patient labeling and designed for readability no higher than a defined reading level using a Flesch-Kincaid Readability test. The representative samples were reviewed by a Certified Genetic Counselor to confirm that the materials tested accomplished the following:

Defined the target condition being tested and related symptoms   
• Explained the intended use and limitations of the test   
• Explained the relevant ethnicities in regard to the variant tested   
• Explained genetic health risks and relevance to the user’s ethnicity, and   
• Assessed participants’ ability to understand the following comprehension concepts: the test’s limitations, purpose, appropriate follow-up action, test results, ethnic relevance and other risk factors that may have an impact on the test results.

For both studies, the completion rate was $100 \%$ (378/378 for Study 1 and 213/213 for Study 2) for all subjects who appeared for their interviews in the study after passing the pre-defined exclusion criteria. The average comprehension rates per core comprehension concept were $9 0 . 7 \%$ to $9 7 . 4 \%$ for Study 1 and $9 0 . 9 \%$ to $9 9 . 1 \%$ for Study 2. The overall comprehension score across all GHR reports was $9 3 . 2 \%$ for Study 1 and $9 6 \%$ for Study 2.

Overall user comprehension rates for Factor V Leiden GHR Test reports in Study 1   

<table><tr><td rowspan=2 colspan=1>Core Concept</td><td rowspan=1 colspan=4>Comprehension Rates by GHR Report Type (%)</td><td rowspan=2 colspan=1>OverallComprehensionRates (%)</td></tr><tr><td rowspan=1 colspan=1>0 Variant</td><td rowspan=1 colspan=1>1 Variant</td><td rowspan=1 colspan=1>2 Variants</td><td rowspan=1 colspan=1>Result NotDetermined</td></tr><tr><td rowspan=1 colspan=1>AppropriateFollow-Up Action</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>97.4</td></tr><tr><td rowspan=1 colspan=1>EthnicityRelevance</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>95.5</td></tr><tr><td rowspan=1 colspan=1>Other RiskFactors</td><td rowspan=1 colspan=1>89.6</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>92.5</td><td rowspan=1 colspan=1>92.2</td><td rowspan=1 colspan=1>92.1</td></tr><tr><td rowspan=1 colspan=1>Limitations ofTest</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>88.9</td><td rowspan=1 colspan=1>91.5</td></tr><tr><td rowspan=1 colspan=1>Purpose of Test</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>92.7</td></tr><tr><td rowspan=1 colspan=1>Results of Test</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>83.7</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>91.1</td><td rowspan=1 colspan=1>90.7</td></tr><tr><td rowspan=1 colspan=1>Total Number ofReports</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>378</td></tr></table>

Overall user comprehension rates for Factor V Leiden GHR Test reports in Study 2   

<table><tr><td rowspan=2 colspan=1>Core Concept</td><td rowspan=1 colspan=2>Comprehension Rates by GHR ReportType (%)</td><td rowspan=1 colspan=1>Overall ComprehensionRates (%)</td></tr><tr><td rowspan=1 colspan=1>1 Variant</td><td rowspan=1 colspan=1>Result Not Determined</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Appropriate Follow-Up Action</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.5</td></tr><tr><td rowspan=1 colspan=1>Ethnic Relevance</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>97.7</td></tr><tr><td rowspan=1 colspan=1>Other Risk Factors</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>93.6</td><td rowspan=1 colspan=1>95.8</td></tr><tr><td rowspan=1 colspan=1>Limitations of Test</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>96.2</td></tr><tr><td rowspan=1 colspan=1>Purpose of Test</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>99.1</td></tr><tr><td rowspan=1 colspan=1>Results of Test</td><td rowspan=1 colspan=1>88.5</td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>90.9</td></tr><tr><td rowspan=1 colspan=1>Total Number ofReports</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>213</td></tr></table>

Comparison of pre- and post-test by core concept in Study 2   

<table><tr><td colspan="1" rowspan="1">Core Concept</td><td colspan="1" rowspan="1">Pre-test (%)</td><td colspan="1" rowspan="1">Post-test (%)</td><td colspan="1" rowspan="1">%Improvement</td><td colspan="1" rowspan="1">p-value</td></tr><tr><td colspan="1" rowspan="1">Purpose</td><td colspan="1" rowspan="1">89.0</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">10.1</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">Other Risk Factors</td><td colspan="1" rowspan="1">91.1</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">4.7</td><td colspan="1" rowspan="1">0.002</td></tr><tr><td colspan="1" rowspan="1">Ethnic Relevance</td><td colspan="1" rowspan="1">95.8</td><td colspan="1" rowspan="1">97.7</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">0.117</td></tr><tr><td colspan="1" rowspan="1">Limitations</td><td colspan="1" rowspan="1">87.7</td><td colspan="1" rowspan="1">96.2</td><td colspan="1" rowspan="1">8.5</td><td colspan="1" rowspan="1">&lt;0.001</td></tr></table>

# b. Frequently Asked Questions Material

The manufacturer has developed a Frequently Asked Question (FAQ) section for each Genetic Health Risk Test Report. The FAQ section was created to provide users information to adequately understand the purpose, limitations and the meaning of the results of the test. The FAQ section was developed using methodology consistent with the Manufacturer’s labeling design, identification of primary communication messages and label comprehension. The concepts covered in the FAQ section include: the test results, the purpose of the test, limitations of the test, relevance of race and ethnicity on test results, the meaning of the result, other risk factors that may affect the user’s family and children, and links to resources that provide additional information. Additionally, the FAQ section provide definitions for terminology found in Genetic Health Risk Reports that is used to describe risks associated with detected variants. The questions included in the FAQ section for the test include, but are not limit to:

• What is the purpose of this test?   
• What is a blood clot? What is thrombophilia? What can this test tell me?   
• What can this test not tell me?   
• What does it mean if I have an increased risk?   
• What does it mean if I don’t have an increased risk? Can other things increase my risk for an abnormal blood clot? Is factor V Leiden thrombophilia more common in certain ethnic groups? What should I do with my results? Where can I get more information?

Each Genetic Health Risk Test Report has answers to FAQ that are specific to the variant(s) and disease being reported, where applicable. The FAQ section is included in the Genetic Health Risk Test Report.

# D Clinical Cut-Off:

Not applicable

E Expected Values/Reference Range:

Not applicable

F Other Supportive Instrument Performance Characteristics Data:

i. Reagent Shelf-Life Stability

The purpose of the real-time stability study is to establish the shelf-life stability for the five critical reagents used in the AncestryDNA Factor V Leiden GHR Test. Three lots of critical reagents were assessed with DNA from three commercially available cell lines (one cell line for each genotype). The study is ongoing and current results support a shelf-life stability claim of 2 months when the critical reagents are stored under the following conditions:

<table><tr><td rowspan=1 colspan=1>Critical Reagent</td><td rowspan=1 colspan=1>Storage Temperature</td></tr><tr><td rowspan=1 colspan=1>MSM</td><td rowspan=1 colspan=1>-15°C to -25</td></tr><tr><td rowspan=1 colspan=1>FMS</td><td rowspan=1 colspan=1>-15°C to -25</td></tr><tr><td rowspan=1 colspan=1>BeadChip</td><td rowspan=1 colspan=1>2C to 8</td></tr><tr><td rowspan=1 colspan=1>X-Stain Plate</td><td rowspan=1 colspan=1>-15°C to -25°</td></tr><tr><td rowspan=1 colspan=1>EML</td><td rowspan=1 colspan=1>-15°C to -25</td></tr></table>

ii. Specimen Stability

Saliva samples for testing are collected with the AncestryDNA Saliva Collection Kit. Refer to K192947 for sample stability information as well as a description of the device and performance characteristics.

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.