Promega Corporation Pamela Swatkowski Regulatory Consultant 2800 Woods Hollow Rd Madison, Wisconsin 53711

Re: K200129 Trade/Device Name: OncoMate MSI Dx Analysis System Regulation Number: 21 CFR 864.1866 Regulation Name: Lynch Syndrome Test Systems Regulatory Class: Class II Product Code: PZJ Dated: November 30, 2020 Received: December 1, 2020

Dear Pamela Swatkowski:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR

803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Zivana Tezak, PhD   
Chief   
Molecular Genetics Branch   
Division of Molecular Genetics and Pathology   
OHT7: Office of In Vitro Diagnostics and Radiological Health   
Office of Product Evaluation and Quality   
Center for Devices and Radiological Health

Enclosure

Device Name OncoMate™M MSI Dx Analysis System

Indications for Use (Describe)   
Th OcoMateSI Dx Analysi Systes  qualitativeultplex polymerae hain action (CR) tet inte he eleidellie (BAT-5, BAT-26NR-21, NR- MONOmanoraltax pn-eFcls ts. The OncoMate™M MSI Dx Analysis System is for use with the Applied Biosystems $\textsuperscript { \textregistered }$ 3500Dx Genetic Analyzer and OncoMate™M MSI Dx Interpretive Software.

The OncoMate™ MSI Dx Analysis System is indicated in patients diagnosed with colorectal cancer (CRC) todetect would benefit from additional genetic testing to diagnose Lynch syndrome.

Resulrohe cMatS  Analys Syste hould bterprete byalhare proesinals with other clinical findings, family history, and other laboratory data.

The cinal peor thi dvi  uie reaent decisnor S h patits has ot bee etablihed.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

Turenotioatinatveo time to reviw instructions, search existig data sources,gather and maintainhe dataneeded and cplee and review the colection of information. Send comments regarding this burden estimate or anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 510(k) SUMMARY

# 1 COMPANY AND CONTACT INFORMATION

Company Name:

Promega Corporation

Address:

2800 Woods Hollow Rd Madison, WI 53711

Telephone:

608.274.4330

Contact Person:

Ron Wheeler Sr. Director, Quality and Regulatory Affairs

Date of Summary:

20 July 2021

# 2 DEVICE IDENTIFICATION

OncoMateTM MSI Dx Analysis System

OncoMateTM MSI Dx Analysis System

Lynch Syndrome Test System

Trade (Proprietary) Name:   
Common (Usual) Name:   
Classification Name:   
Product Code:   
Regulation Number:   
Regulatory Classification:   
Panel:

PZJ

21 CFR 864.1866

# Class II

88 – Pathology

# 3 PREDICATE DEVICE

VENTANA MMR IHC Panel

# 4 DEVICE DESCRIPTION

# 4.1 OncoMate™ MSI Dx Analysis System Overview

The OncoMate™ MSI Dx Analysis System assay encompasses a complete workflow for MSI determination, from DNA extraction to data analysis (Figure 1). DNA is extracted from FFPE colorectal tissue samples (normal and tumor from the same patient) using the Maxwell® CSC DNA FFPE Kit and Maxwell® CSC Instrument. Double-stranded DNA (dsDNA) is then quantified using a fluorescence-based dsDNA quantification system of the user’s choice. Next, amplification products are generated through multiplex PCR amplification of DNA microsatellite markers using the OncoMate™ MSI Dx Analysis System amplification kit. The PCR products are then mixed with Hi-Di™ Formamide and Size Standard 500 and heat-denatured. The resulting single-stranded DNA fragments are separated by size and detected via fluorescence using an Applied Biosystems® 3500Dx Genetic Analyzer. Following capillary electrophoresis, allele sizes from the CRC tumor DNA and the normal DNA are calculated and compared for each of the microsatellite markers using OncoMate™ MSI Dx Interpretive Software. If the length of two or more of the five mononucleotide-repeat marker alleles is changed by $2 2 . 7 5$ base pairs (bp), the tumor is classified as MSI-H; if the allele length is changed for only one marker, or if the difference in allele lengths at the five markers is ${ < } 2 . 7 5 \mathrm { b p }$ , the tumor is classified as Microsatellite Stable (MSS). The sizes of the Penta C and Penta D pentanucleotide-repeat marker alleles are compared as an identity check between the normal and tumor DNA samples.

![](images/26a8014cc761bef79bec5d368277daadd1d0af1b08b38ce2b72614406147e0c2.jpg)  
Figure 1: OncoMate™ MSI Dx Analysis System Workflow

# 4.2 Test Principle

The genomic DNA inside a cell must be replicated during each round of cell division to ensure new cells have a complete copy of the genetic information. DNA replication occurs with high fidelity (i.e., errors in copying the genomic DNA occur infrequently). The low error rate is attributed to high fidelity of replication and an efficient DNA repair system (1). If an error is made during replication, the DNA repair system (mismatch repair system) is normally able to excise the erroneous nucleotide(s) and insert the correct molecule(s). It has been observed that errors occur more frequently during replication of homopolymer regions (short tandem repeats) within the genome (2). A loss of function in the mismatch repair system allows errors to accumulate during replication, eventually leading to propagation of the incorrect sequence. The OncoMate™ MSI Dx Analysis System is used to examine the frequency of errors in five homopolymer (poly-A) microsatellite regions of the genome (NR-21, BAT-26, BAT-25, NR-24, and MONO-27). DNA replication errors that occur in these regions are an indicator of dysfunctional DNA repair. The replication errors result in an increase or decrease in the number of nucleotides in the locus that can be measured by PCR amplification of the regions followed by determination of the molecular weight of the resultant amplicon. Amplicons of the expected size indicate DNA repair is functioning normally. A change in the molecular weight is indicative of dysfunction. This assessment is performed by comparing DNA extracted from matched tumor and normal samples from the same individual.

# 4.3 Summary and Explanation

Lynch syndrome, also called hereditary nonpolyposis colorectal cancer (HNPCC), is an inherited disorder that increases the risk of many types of cancer, particularly cancers of the colon and rectum (3,4). Lynch syndrome accounts for approximately $3 \%$ of colorectal cancers and is caused by autosomal-dominant germline mutations in DNA mismatch repair (MMR) genes (2, 5, 3). These mutations impair cellular MMR function, such that mutations introduced during normal cellular DNA replication are not properly repaired. The accumulation of mutations may lead to cellular dysfunction and, eventually, cancer (6, 7, 8). Identification of individuals with Lynch syndrome offers an understanding of future cancer susceptibility and an opportunity for increased cancer surveillance. Family members of that individual also may undergo increased medical surveillance or testing for Lynch syndrome (3, 9). DNA sequencing across multiple regions of several MMR genes is the definitive diagnostic test for Lynch syndrome (4, 6).

Microsatellite instability testing cannot be used to diagnose Lynch syndrome. Instead, MSI testing is a rapid and cost-effective method to identify MMR deficiency in CRC tumor cells (4, 5, 9, 10). Microsatellites are short, DNA-repeat regions that are naturally prone to DNA replication errors that alter (typically shorten) their length. MSI is observed when MMR function is compromised, and errors made during DNA replication are not repaired. As a result, the length of microsatellite alleles may differ in MMRdeficient versus normal tissue samples (6, 10, 11, 12). CRC patients identified as MSI high (MSI-H) by MSI testing may have Lynch syndrome and are therefore candidates for DNA sequencing to determine whether they have germline mutations in MMR genes (2, 4, 5, 9). Many professional groups and institutions, including the National

Comprehensive Cancer Network (NCCN), endorse universal MMR or MSI testing in all patients with a personal history of colon or rectal cancer to determine which patients should have genetic testing for Lynch syndrome (9, 13, 14, 15, 16, 17).

# 4.4 System Components

# 4.4.1 System Overview

The OncoMate™ MSI Dx Analysis System is a fluorescent, multiplex PCR-based test to detect DNA sequence length changes in microsatellite regions of colorectal tumor cell DNA relative to the same regions from the patient’s normal cells. Microsatellites are short, DNA-repeat regions [e.g., (A)n, (CA)n, (AAT)n, (AGAT)n, (AAAAG)n,] that are distributed throughout the human genome and are prone to insertion and deletion copying errors during DNA replication.

The OncoMate™ MSI Dx Analysis System includes fluorophore-labeled primers for coamplification of seven microsatellite markers: five mononucleotide repeat markers (BAT25, BAT-26, NR-21, NR-24 and MONO-27) and two pentanucleotide repeat markers (Penta C and Penta D, Table 1). The OncoMate™ MSI Dx Analysis System amplification kit is intended for use with DNA isolated with the Maxwell® CSC DNA FFPE Kit (Cat.# AS1350) and quantified using dsDNA-binding dyes.

Table 1: Expected Amplified Size Ranges and Detection Channels for the Markers Included in the OncoMate™ MSI Dx Analysis System Amplification Kit.   

<table><tr><td>Mononucleotide Markers</td><td>Repeat Structure</td><td>Detection Channel</td><td>Amplified Size Range</td></tr><tr><td>BAT-26</td><td>A(26)</td><td>Blue</td><td>83 to 121bp</td></tr><tr><td>NR-21</td><td>A(21)</td><td>Green</td><td>83 to 108bp</td></tr><tr><td>BAT-25</td><td>A(25)</td><td>Green</td><td>110 to 132bp</td></tr><tr><td>MONO-27</td><td>A(27)</td><td>Green</td><td>134 to 168bp</td></tr><tr><td>NR-24</td><td>A(24)</td><td>Yellow (displayed black)</td><td>103 to 138bp</td></tr><tr><td>Pentanucleotide Markers</td><td>Repeat Structure</td><td>Detection Channel</td><td>Amplified Size Range</td></tr><tr><td>Penta D</td><td>AAAGA(217)</td><td>Blue</td><td>123 to 253bp</td></tr><tr><td>Penta C</td><td>AAAAC(4-17)</td><td>Yellow (displayed black)</td><td>140 to 228bp</td></tr></table>

OncoMate™ MSI Dx Analysis System amplification products are analyzed by capillary electrophoresis using the Applied Biosystems $\textsuperscript { \textregistered }$ $3 5 0 0 \mathrm { D x }$ Genetic Analyzer in ‘Diagnostic Mode’ using POP- $7 ^ { \textregistered } 3 5 0 0 \mathrm { { D x } }$ Series Polymer and a $3 5 0 0 \mathrm { D x }$ Series Capillary Array, $5 0 \mathrm { c m }$ .

During capillary electrophoresis, OncoMate™ MSI Dx Analysis System (Cat. #MD2140) amplification products are separated and analyzed alongside fluorescently labeled DNA fragments of known size, the Size Standard 500. Following capillary electrophoresis, the resulting DNA fragment data (.fsa files) are simultaneously imported and analyzed by the OncoMate™ MSI Dx Interpretive Software. During this process, data quality control (QC) checks are performed, and DNA fragments amplified from seven microsatellite regions are sized with reference to the size standard fragments using the Local Southern method (18).

005_510k Summary Original 510(k) Application

# 4.4.2 Materials Provided with the OncoMate™ MSI Dx Analysis System (Catalog Number MD2140)

The OncoMateTM MSI Dx Analysis System contains sufficient reagents to perform 100 reactions (50 paired reactions). The following materials are included within the reagent kit:

Table 2: Materials Provided with the OncoMateTM MSI Dx Analysis System   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Size</td><td rowspan=1 colspan=1>Catalog Number</td></tr><tr><td rowspan=1 colspan=1>OncoMateTMSI 5X Primer Mix</td><td rowspan=1 colspan=1>200μL</td><td rowspan=1 colspan=1>MD705A</td></tr><tr><td rowspan=1 colspan=1>OncoMateMSI 5X Master Mix</td><td rowspan=1 colspan=1>200μL</td><td rowspan=1 colspan=1>MD280A</td></tr><tr><td rowspan=1 colspan=1>2800M Control DNA, 10ng/μL</td><td rowspan=1 colspan=1>25μL</td><td rowspan=1 colspan=1>MD810A</td></tr><tr><td rowspan=1 colspan=1>Water, Amplification Grade</td><td rowspan=1 colspan=1>1.25mL</td><td rowspan=1 colspan=1>MD193A</td></tr><tr><td rowspan=1 colspan=1>Size Standard 500</td><td rowspan=1 colspan=1>100μL</td><td rowspan=1 colspan=1>MD500A</td></tr></table>

# 4.4.3 Materials Not Provided with the OncoMate™ MSI Dx Analysis System (Catalog Number MD2140)

# Instruments and Accessories

• OncoMateTM 5C Matrix Standard, (Cat.# MD4850)   
• Maxwell® CSC DNA FFPE Kit, (Cat.# AS1350)   
• Maxwell® CSC Instrument (Cat.# AS6000)   
• Fluorometer compatible with fluorescent-dye-based dsDNA quantification reagents   
• Applied Biosystems® $3 5 0 0 \mathrm { D x }$ Genetic Analyzer (Thermo Fisher Scientific Cat.#   
4461450)   
• Thermal cycler compatible with 96-well plates or reaction tubes   
• $3 5 0 0 \mathrm { D x }$ Capillary Array $5 0 \mathrm { c m }$ (Thermo Fisher Scientific Cat.# 4404684)   
• $3 5 0 0 \mathrm { D x }$ Series Septa 96-Well (Thermo Fisher Scientific Cat.# 4410700)   
• POP- $7 ^ { \textcircled { \mathtt { B } } }$ Performance Optimized Polymer $3 5 0 0 \mathrm { D x }$ Series (Thermo Fisher Scientific Cat.# 4393709 and 4393713)   
• Anode Buffer Container $3 5 0 0 \mathrm { D x }$ Series (Thermo Fisher Scientific Cat.# 4393925)   
• Cathode Buffer Container $3 5 0 0 \mathrm { D x }$ Series (Thermo Fisher Scientific Cat.# 4408258)   
• 3500 Dx Series Septa Cathode Buffer Container (Thermo Fisher Scientific Cat.#   
4410716)   
• Conditioning Reagent $3 5 0 0 \mathrm { D x }$ Series (Thermo Fisher Scientific Cat.# 4409543)

# Software

• OncoMate™ MSI Dx Assay Installer (Cat.# MD4150) • OncoMate™ MSI Dx Interpretive Software (Cat.# MD4140) • OncoMate™ MSI Dx License Key (Cat.# MD4141)

# 4.5 Controls

# 4.5.1 Spectral Calibration

During capillary electrophoresis, dye-labeled OncoMate™ MSI Dx Analysis System amplification products are separated and detected using the Applied Biosystems® 3500 Dx Genetic Analyzer. Prior to analysis, the Applied Biosystems® $3 5 0 0 \mathrm { D x }$ Genetic Analyzer is calibrated with matrix standards so that the fluorescent signals resulting from the set of specific dyes used in the assay can be distinguished. The OncoMate™ 5C Matrix Standard consists of DNA fragments labeled with five different fluorescent dyes (fluorescein, JOE, TMR-ET, CXR-ET and WEN) in one tube. The calibration is performed using the ‘OncoMate_MSI’ dye set, which is installed on the Applied Biosystems® $3 5 0 0 \mathrm { D x }$ Genetic Analyzer using the OncoMate™ MSI Dx Assay Installer. Once generated, the spectral calibration file is applied automatically during sample detection to account for the spectral overlap among the dyes and to separate the raw fluorescent signals into individual dye signals.

# 4.5.2 Positive and Negative Controls

Positive and Negative Controls are used within the system. The Positive Control is 2800M Control DNA $( 1 0 \mathrm { n g / \mu l } )$ which is DNA extracted from a microsatellite stable human cell line. The Negative Control is Water, Amplification Grade which is used as a reagent blank.

Positive and no-template ("negative") control amplification reactions using 2800M Control DNA and Water, Amplification Grade, respectively, must be analyzed concurrently with patient samples to verify assay performance. At least one 2800M Control DNA amplification reaction and one negative control amplification reaction must be completed for each plate (i.e., batch) of patient samples analyzed using the OncoMate™ MSI Dx Interpretive Software. The negative control reaction is analyzed to ensure that no unexpected amplification occurred in no-template reactions, which would indicate the presence of DNA contamination and lead to an Invalid assay result. The positive control reaction is analyzed to demonstrate that the amplification chemistry performed as expected.

# 4.5.3 Capillary Electrophoresis Standards

All analyzed samples and controls must contain Size Standard 500 (added prior to CE). Size Standard 500 contains a series of 21 DNA fragments of known lengths (60, 65, 80, 100, 120, 140, 160, 180, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 500bp), also referred to as a DNA ladder. Each fragment is labeled with WEN dye and is detected separately (as a fourth color, orange) in the presence of OncoMate™ MSI Dx Analysis System-amplified products using the Applied Biosystems $\textsuperscript { \textregistered }$ $3 5 0 0 \mathrm { D x }$ Genetic Analyzer. For each sample or control, amplified DNA fragments are sized with reference to the size standard fragments using the Local Southern method (18). The size standard controls for capillary-to-capillary variations in sizing precision during capillary electrophoresis and allows direct comparison of samples across the capillary electrophoresis run. Only the 60-base to 300-base fragments are analyzed for fragment sizing in the OncoMate™ MSI Dx Interpretive Software.

# 4.6 Results Reporting

The OncoMate™ MSI Dx Analysis System generates size (i.e., DNA fragment length) data for microsatellite regions amplified from matched normal and CRC tumor sample pairs. These data are analyzed using the OncoMate™ MSI Dx Interpretive Software to determine tumor sample MSI status.

The OncoMate™ MSI Dx Interpretive Software provides an automated interpretive result, either MSI-H or MSS, when no data QC issues are observed for a sample. Five mononucleotide-repeat markers (BAT-25, BAT-26, NR-21, NR-24 and MONO-27) and two pentanucleotide-repeat markers (Penta C and Penta D) are evaluated. The analysis of mononucleotide-repeat markers in paired normal and CRC tumor samples determines tumor MSI status. For each of the five mononucleotide-repeat markers, the smallest allele (in bp) identified is considered the allele of interest for subsequent comparisons. The size difference (bp) between the allele of interest in the normal and tumor samples is calculated to determine marker stability. The interpretive software defines marker instability as a 3bp change (implemented as $\ge 2 . 7 $ 5bp to account for the sizing precision of capillary electrophoresis). The interpretive software will score as stable novel alleles that are shifted less than 3bp from the normal reference allele as well as fragment profiles without a distinct novel allele. The OncoMate™ MSI Dx Interpretive Software provides an automated interpretive result, either MSI-H or MSS, when no data QC issues are observed for a sample. A tumor sample is interpreted as MSI-H when two or more markers are ‘Unstable’. A tumor sample is interpreted as MSS when fewer than two markers are interpreted as ‘Unstable’.

# 5 INDICATIONS FOR USE

The OncoMate™ MSI Dx Analysis System is a qualitative multiplex polymerase chain reaction (PCR) test intended to detect the deletion of mononucleotides in 5 microsatellite loci (BAT-25, BAT-26, NR-21, NR-24 and MONO-27) using matched tumor and normal DNA obtained from formalin fixed paraffin-embedded (FFPE) colorectal tissue sections. The OncoMateTM MSI Dx Analysis System is for use with the Applied Biosystems® 3500Dx Genetic Analyzer and OncoMate™ MSI Dx Interpretive Software.

The OncoMate™ MSI Dx Analysis System is indicated in patients diagnosed with colorectal cancer (CRC) to detect microsatellite instability (MSI) as an aid in the identification of probable Lynch syndrome to help identify patients that would benefit from additional genetic testing to diagnose Lynch syndrome.

Results from the OncoMate™ MSI Dx Analysis System should be interpreted by healthcare professionals in conjunction with other clinical findings, family history, and other laboratory data.

The clinical performance of this device to guide treatment decision for MSI high patients has not been established.

6 SUMMARY COMPARISON OF TECHNOLOGICAL CHARACTERISTICS Table 3: Substantial Equivalence Comparison   

<table><tr><td>Device &amp; Predicate Device(s): Device Trade</td><td>K200129</td><td>DEN170030</td></tr><tr><td>Name General Device Characteristic Similarities</td><td>Promega OncoMateTM MSI Dx Analysis System</td><td>Ventana MMR IHC panel</td></tr><tr><td>Intended Use/Indications For Use</td><td>The OncoMateTM MSI Dx Analysis System is a qualitative multiplex polymerase chain reaction (PCR) test intended to</td><td>The VENTANA MMR IHC Panel is a qualitative immunohistochemistry (IHC) test intended for use in the light microscopic assessment of mismatch repair (MMR) proteins (MLH1, PMS2, MSH2, and MSH6) and BRAF V600E</td></tr><tr><td></td><td>detect the deletion of mononucleotides in 5 microsatellite loci (BAT-25, BAT-26, NR-21, NR-24 and MONO-27) using matched tumor and normal DNA obtained from formalin fixed paraffin- embedded (FFPE) colorectal tissue sections. The OncoMateTM MSI Dx Analysis System is for use with the Applied Biosystems® 3500Dx Genetic Analyzer and OncoMateTM MSI Dx Interpretive Software. The OncoMateTM MSI Dx Analysis System is indicated in patients diagnosed with colorectal cancer (CRC) to detect</td><td>proteins in formalin-fixed, paraffin- embedded colorectal cancer (CRC) tissue sections. The OptiView DAB IHC Detection Kit is used with MLH1, MSH2, MSH6 and BRAF V600E, and the OptiView DAB IHC Detection Kit with OptiView amplification Kit is used for PMS2 detection. The VENTANA MMR IHC Panel is for use on the VENTANA BenchMark ULTRA instrument. The VENTANA MMR IHC Panel includes VENTANA anti-MLH1 (M1) Mouse Monoclonal Primary Antibody, VENTANA anti-PMS2 (A16-4)</td></tr><tr><td></td><td>microsatellite instability (MSI) as an aid in the identification of probable Lynch syndrome to help identify patients that would benefit from additional genetic testing to diagnose Lynch</td><td>Mouse Monoclonal Primary Antibody, VENTANA anti-MSH2 (G219-1129) Mouse Monoclonal Primary Antibody, VENTANA anti-MSH6 (SP93) Rabbit Monoclonal Primary Antibody, and VENTANA anti-BRAF V600E (VE1) Mouse Monoclonal Primary</td></tr><tr><td>syndrome. data.</td><td>Results from the OncoMateTM MSI Dx Analysis System should be interpreted by healthcare professionals in conjunction with other clinical findings, family history, and other laboratory</td><td>Antibody. The VENTANA MMR IHC Panel is indicated in patients diagnosed with colorectal cancer (CRC) to detect mismatch repair (MMR) proteins deficiency as an aid in the identification of probable Lynch syndrome and to detect BRAFV600E protein as an aid to differentiate</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">The clinical performance of thisdevice to guide treatmentdecision for MSI high patientshas not been established.</td><td colspan="1" rowspan="1">between sporadic CRC and probableLynch syndrome.Results from the Ventana MMR IHCPanel should be interpreted by aqualified pathologist inconjunction with histologicalexamination, relevant clinicalinformation, and proper controls.The clinical performance of this deviceto guide treatment of MMR deficientpatients has notbeen established.</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">FFPE colorectal cancer tissueand Normal tissue</td><td colspan="1" rowspan="1">FFPE colorectal cancer tissue</td></tr><tr><td colspan="1" rowspan="1">TargetPopulation</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Patients diagnosed with CRC</td></tr><tr><td colspan="1" rowspan="1">Performance</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Aids in the identification of probableLynch syndrome</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Positive (2800M) control DNANegative control</td><td colspan="1" rowspan="1">Positive controls - MMR antibodies,BRAF V600E mutated proteinNegative control</td></tr><tr><td colspan="1" rowspan="1">SpecialConditions forUse Statements</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Intended for in vitro diagnostic IVDusePrescription use only</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">PCR based microsatellitemeasurement in normal andtumor DNA</td><td colspan="1" rowspan="1">Immunohistochemistry based proteinexpression</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">5 mononucleotide tractsBAT25, BAT26, MONO27,NR21 and NR24</td><td colspan="1" rowspan="1">MMR proteins MLH1, MSH2, MSH6and PMS2 as well asBRAFV600 E protein</td></tr></table>

# 6.1 Differences Between the Device and Predicate

The assay targets and technology for assessing probable Lynch syndrome cases for the predicate and subject device are different, the two test systems assess the status for the mismatch repair deficiency in CRC patients albeit at protein level for the predicate and DNA genotype for the subject device. The clinical validation (the method comparison study) using CRC FFPE samples yielded comparable performance between the device and the predicate in identifying probable Lynch syndrome cases. Therefore, the differences do not affect the safety or effectiveness of the device when used as labeled.

# 6.2 Special Controls

The device meets all General Controls and Special Controls listed for regulation 21CFR 864.1866. Documentation and data supporting the Special Controls specific to this

product classification were documented in the submission and summarized in this document and the product label.

# 7 PERFORMANCE CHARACTERISTICS

All analytical studies followed the procedure outlined in the OncoMate™ MSI Dx Analysis System instructions, unless noted otherwise in the study results section.

# 7.1 Extraction

Suitability of the Maxwell $\textsuperscript { \textregistered }$ CSC DNA FFPE Kit using the Maxwell $\textsuperscript { \textregistered }$ CSC Instrument for DNA extraction was demonstrated by performing DNA extractions from FFPE curls $( 0 . 1 { - } 2 . 0 \mathrm { m m } ^ { 3 }$ tissue) obtained from four MSI-H, three MSS tumor samples and matched normal samples. The MSI-H samples were at $20 { - } 3 0 \%$ tumor content, and the MSS samples were at $20 \mathrm { - } 6 0 \%$ tumor content. DNA extraction was performed by each of two operators using three lots of the Maxwell $\textsuperscript { \textregistered }$ CSC DNA FFPE Kit. The DNA FFPE kit lot was alternated between operators (e.g., Operator 1 used Lot 1 on Day 1 and Operator 2 used Lot 2 on Day 1). Once isolated, the extracted DNA was quantified using the QuantiFluor $\textsuperscript { \textregistered }$ dsDNA System and amplified using the OncoMate™ MSI Dx Analysis System amplification kit. The amplified DNA was subjected to capillary electrophoresis using an Applied Biosystems $\textsuperscript { \textregistered }$ $3 5 0 0 \mathrm { D x }$ Genetic Analyzer and analyzed with the OncoMate™ MSI Dx Interpretive Software.

A total of 5 samples initially yielded Invalid results. After reinjection of all 5 samples, 1 sample was resolved. The remaining 4 samples were resolved by reamplification.

In the study, $9 6 . 4 \%$ (81/84) of the individual FFPE curls extracted produced results that were concordant with the predetermined MSI status. The $9 5 \%$ confidence intervals (CI) for percent correct and percent incorrect results were $8 9 . 9 - 9 9 . 3 \%$ and $0 . 7 \mathrm { - } 1 0 . 1 \%$ , respectively.

The study demonstrated that the Maxwell® CSC Instrument using the Maxwell® CSC DNA FFPE Kit for DNA extraction met extraction capabilities for use with the OncoMate™ MSI Dx Analysis System.

# 7.2 Normal Range and Cutoff

The OncoMate™ MSI Dx Analysis System is intended to measure changes in amplified fragment length. The Normal Range and Cutoff study was conducted to verify the system’s capability to resolve amplicons that differ by ${ \ge } 3$ base pairs. Two sets of seven synthetic DNA fragments (“resolution markers”) were analyzed during this study. These fragments consist of dye-labeled amplicons of known size that are separated by 1bp within each set, with the two sets designed to bracket the upper (Large) and lower (Small) ends of the amplicon size range of the MSI markers (83–168bp).

The resolution markers were subjected to capillary electrophoresis using an Applied Biosystems $\textsuperscript { \textregistered }$ $3 5 0 0 \mathrm { D x }$ Genetic Analyzer and analyzed with OncoMate™ MSI Dx Interpretive Software. Resolution markers were analyzed either mixed only with the Size Standard 500 or mixed separately with two MSS tumor samples and the Size Standard 500. While the OncoMate™ MSI Dx Interpretive Software was not designed to identify the resolution markers, it was critical to demonstrate the System software’s ability to determine fragment size and precision. The sizing precision of individual resolution fragments was characterized (Table 4), and size differences between all fragments separated by 3bp were calculated and averaged. Observed mean differences were compared with predicted values.

There were no invalid results or samples requiring reinjection or reamplification testing for this study.

Resolution fragments were sized precisely, with standard deviations ranging from $0 . 0 7 -$ 0.13bp. Mean absolute differences calculated for fragments separated by 3bp were also precise, ranging from 3.06–3.40bp and 3.05–3.35bp for observed and predicted values, respectively. Accordingly, the study met the objective of ${ \ge } 3$ base pair resolution and, in fact, demonstrated measurement precision of individual resolution markers (standard deviations ${ \underline { { < 0 . 1 3 } } }$ base pairs) that was sufficient to detect single-base-pair differences in size.

Table 4: Descriptive Statistics for Resolution Marker Base Pair Size   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=2>95%-CI</td></tr><tr><td rowspan=1 colspan=1>ResolutionMarker</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Minimum</td><td rowspan=1 colspan=1>Maximum</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Standarddeviation</td><td rowspan=1 colspan=1>LowerLimit</td><td rowspan=1 colspan=1>UpperLimit</td></tr><tr><td rowspan=1 colspan=1>Large</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>180.60</td><td rowspan=1 colspan=1>180.95</td><td rowspan=1 colspan=1>180.77</td><td rowspan=1 colspan=1>180.75</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>180.71</td><td rowspan=1 colspan=1>180.80</td></tr><tr><td rowspan=1 colspan=1>Large 2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>181.67</td><td rowspan=1 colspan=1>182.02</td><td rowspan=1 colspan=1>181.89</td><td rowspan=1 colspan=1>181.83</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>181.78</td><td rowspan=1 colspan=1>181.88</td></tr><tr><td rowspan=1 colspan=1>Large 3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>182.75</td><td rowspan=1 colspan=1>182.97</td><td rowspan=1 colspan=1>182.85</td><td rowspan=1 colspan=1>182.86</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>182.83</td><td rowspan=1 colspan=1>182.90</td></tr><tr><td rowspan=1 colspan=1>Large 4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>183.70</td><td rowspan=1 colspan=1>184.04</td><td rowspan=1 colspan=1>183.88</td><td rowspan=1 colspan=1>183.85</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>183.80</td><td rowspan=1 colspan=1>183.89</td></tr><tr><td rowspan=1 colspan=1>Large 5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>184.66</td><td rowspan=1 colspan=1>184.99</td><td rowspan=1 colspan=1>184.83</td><td rowspan=1 colspan=1>184.80</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>184.76</td><td rowspan=1 colspan=1>184.85</td></tr><tr><td rowspan=1 colspan=1>Large6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>185.73</td><td rowspan=1 colspan=1>186.06</td><td rowspan=1 colspan=1>185.90</td><td rowspan=1 colspan=1>185.88</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>185.83</td><td rowspan=1 colspan=1>185.93</td></tr><tr><td rowspan=1 colspan=1>Large7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>186.80</td><td rowspan=1 colspan=1>187.01</td><td rowspan=1 colspan=1>186.91</td><td rowspan=1 colspan=1>186.91</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>186.88</td><td rowspan=1 colspan=1>186.94</td></tr><tr><td rowspan=1 colspan=1>Small</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>84.29</td><td rowspan=1 colspan=1>84.71</td><td rowspan=1 colspan=1>84.62</td><td rowspan=1 colspan=1>84.57</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>84.51</td><td rowspan=1 colspan=1>84.63</td></tr><tr><td rowspan=1 colspan=1>Small 2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>85.52</td><td rowspan=1 colspan=1>85.95</td><td rowspan=1 colspan=1>85.83</td><td rowspan=1 colspan=1>85.79</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>85.72</td><td rowspan=1 colspan=1>85.85</td></tr><tr><td rowspan=1 colspan=1>Small 3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>86.66</td><td rowspan=1 colspan=1>87.14</td><td rowspan=1 colspan=1>86.95</td><td rowspan=1 colspan=1>86.92</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>86.86</td><td rowspan=1 colspan=1>86.98</td></tr><tr><td rowspan=1 colspan=1>Small 4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>87.71</td><td rowspan=1 colspan=1>88.15</td><td rowspan=1 colspan=1>87.99</td><td rowspan=1 colspan=1>87.97</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>87.91</td><td rowspan=1 colspan=1>88.03</td></tr><tr><td rowspan=1 colspan=1>Small 5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>88.73</td><td rowspan=1 colspan=1>89.18</td><td rowspan=1 colspan=1>89.04</td><td rowspan=1 colspan=1>89.02</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>88.96</td><td rowspan=1 colspan=1>89.07</td></tr><tr><td rowspan=1 colspan=1>Small 6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>89.96</td><td rowspan=1 colspan=1>90.40</td><td rowspan=1 colspan=1>90.25</td><td rowspan=1 colspan=1>90.21</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>90.15</td><td rowspan=1 colspan=1>90.27</td></tr><tr><td rowspan=1 colspan=1>Small7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>91.06</td><td rowspan=1 colspan=1>91.55</td><td rowspan=1 colspan=1>91.39</td><td rowspan=1 colspan=1>91.34</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>91.28</td><td rowspan=1 colspan=1>91.41</td></tr></table>

# 7.3 Limit of Blank

A Limit of Blank study was conducted to confirm a blank (an MSS sample in this study) did not produce positive MSI-H results. The study tested four known MSS samples across three amplification kit lots, two operators and 60 replicates of each MSS sample— a total of 1,440 test results.

005_510k Summary Original 510(k) Application

There were 2 samples for which the initial test result was invalid. These samples were reinjected per protocol, and after reinjection were resolved. No reamplification testing was required for this study.

All tests (1,440/1,440; $100 \%$ ) resulted in MSS final interpretive results. The 1,440 test results represent 7,200 mononucleotide locus allele calls. For the mononucleotide loci, $9 9 . 9 9 \%$ (7,199/7,200) of the marker stability calls were “Stable”. There was a single instance of one locus, NR-21, being called unstable. A single unstable locus results in an MSS final interpretive result, and the one unstable locus did not affect the final test result. In conclusion, the OncoMateTM MSI Dx Analysis System provides MSS results that are highly reproducible and were not affected by lot or operator.

# 7.4 Limit of Detection

The analytical sensitivity of the OncoMate™ MSI Dx Analysis System was determined using extracted DNA isolated from six MSI-H tumor and matched normal samples, as well as a titration series of the 2800M Control DNA. The 2800M Control DNA samples were treated as MSS samples for analysis using the OncoMate™ MSI Dx Interpretive Software. To create $20 \%$ tumor content (the minimum tumor content required), the tumor samples were mixed with the matched normal samples as needed. A subsequent study investigated higher tumor content. The MSI-H samples and the MSS samples were tested at 0.2, 0.5, 1.0, 2.0 and $2 . 5 \mathrm { n g }$ DNA per amplification reaction with 20 replicates for each sample and DNA amount (five MSI-H cases $\times 2 0$ replicates). To evaluate the influence of tumor content on limit of detection (LOD), one tumor sample was combined with the matched normal sample to simulate a tumor content of $5 \%$ , $10 \%$ , $1 5 \%$ and $20 \%$ .

Each sample in the study was tested on at least 1 instrument with two OncoMate MSI Dx Analysis system lots at 6 different concentrations by two operators (2 replicates/operator/ sample) and over 5 days for a total of 20 replicates per sample to identify the LOD. Because there were no failures based upon the amount of DNA tested, additional testing was necessary to identify the LOD. New sample dilutions were prepared to test a $0 . 1 \mathrm { n g }$ per reaction DNA input using samples with a $20 \%$ tumor content, as well as a 1ng per reaction DNA input using a sample containing $2 . 5 \%$ tumor.

A summary of the interpretive results for the MSI-H and MSS cases are summarized in Table 5 for the DNA input study, and for MSI-H cases in the tumor content study in Table 6. Table 7 displays a summary of reference result vs. interpretive result and locus status by sample ID. The results for all samples tested at 1ng DNA Input and $20 \%$ Tumor Content are shown. Table 8 displays a summary of reference result vs. interpretive result and locus status by sample ID. The results for all samples tested at 0.5, 1.0, and $2 . 0 \mathrm { n g }$ DNA input and 20 and $30 \%$ tumor content are shown in Table 8.

The LOD for the OncoMate™ MSI Dx Analysis System was determined across two dimensions: 1) the total amount of input DNA used for the assay and 2) the fraction of tumor DNA present in the sample.

The LOD for the OncoMate™ MSI Dx Analysis System was established at $30 \%$ tumor content based on concordance across all loci when using the recommended 1ng DNA 005_510k Summary   
Original 510(k) Application

input and samples where tumor content was adjusted by blending with DNA extracted from matched normal tissue.

Table 5: Interpretive Result Frequency for the DNA Concentration Study, by Reagent Lot   

<table><tr><td rowspan=2 colspan=2>Lot by FinalDNA Input (ng)</td><td rowspan=1 colspan=3>OncoMateTM InterpretiveResult (MSI-H  DilutedSample Set)</td><td rowspan=1 colspan=3>OncoMateTM InterpretiveResult (MSS - Diluted SampleSet)</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Lot 1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=6 colspan=1>Lot 2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=6 colspan=1>All</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr></table>

1 One No Call due to low allele peak height detected.

Table 6: Interpretive Result Frequency for the Tumor Concentration Study, by Reagent Lot   

<table><tr><td rowspan=2 colspan=2>Lot by Tumor content (%)</td><td rowspan=1 colspan=2>OncoMateTM Interpretive Result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>Lot 1</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=5 colspan=1>Lot 2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=5 colspan=1>All</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr></table>

Table 7: Summary of Reference Result vs. Interpretive Result and Locus Status with $9 5 \%$ Wilson-Score CI by Sample (1ng DNA Input and $2 0 \%$ Tumor Content)   

<table><tr><td rowspan=2 colspan=1>Sample(MSI-Status)</td><td rowspan=1 colspan=6>20% Tumor content</td></tr><tr><td rowspan=1 colspan=1>ngNAinput</td><td rowspan=1 colspan=1>NR21(n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-26 (n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-25 (n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>NR-24 (n/N)% Concordantto referenceresult (95%CI)</td><td rowspan=1 colspan=1>Mono-27 (n/N)% Concordant toreference result(95% CI)</td></tr><tr><td rowspan=1 colspan=1>2800M(MSS)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-066(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-076(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>17/40; 42.5%(28.5-57.8)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-079(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-081(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>36/40; 90%(77.0-96.0)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>39/40; 97.5(87.1-99.6)</td></tr><tr><td rowspan=1 colspan=1>CRC-084(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>6/40; 15%(7.1--29.1)</td><td rowspan=1 colspan=1>2/40; 5%(1.4-16.5)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-213(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr></table>

Table 8: Summary of Reference Result vs. Interpretive Result and Locus Status with $9 5 \%$ Wilson-Score CI by Sample (Supplemental Study, All DNA Inputs, 20 and $3 0 \%$ Tumor Content)   

<table><tr><td rowspan=1 colspan=1>Sample(All MSI-H)</td><td rowspan=1 colspan=1>%Tumorcontent</td><td rowspan=1 colspan=1>ngDNAinput</td><td rowspan=1 colspan=1>NR21 (n/N) %Concordant toreferenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-26 (n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-25 (n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>NR-24 (n/N)% Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>MONO-27 (n/N)% Concordant toreference result(95% CI)</td></tr><tr><td rowspan=3 colspan=1>CRC-066</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>19/20; 95%(76.4-99.1)</td><td rowspan=1 colspan=1>18/20; 90%(69.9-99.2)</td><td rowspan=1 colspan=1>17/20; 85%(64.0-94.8)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=3 colspan=1>CRC-066</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=3 colspan=1>CRC-079</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>17/20; 85%(64.0-94.8)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=3 colspan=1>CRC-079</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=3 colspan=1>CRC-084</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>2/20; 10%(2.8-30.1)</td><td rowspan=1 colspan=1>0/20; 0%(0-16.1)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>1/20; 5%(0.9--23.6)</td><td rowspan=1 colspan=1>0/20; 0%(0-16.1)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>0/20; 0%(0-16.1)</td><td rowspan=1 colspan=1>0/20; 0%(0-16.1)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=3 colspan=1>CRC-084</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>8/20; 40%(21.9-61.3)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>19/20; 95%(76.4-99.1)</td><td rowspan=1 colspan=1>0/20; 100%(0-16.1)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td><td rowspan=1 colspan=1>0/20; 100%(0-16.1)</td><td rowspan=1 colspan=1>20/20; 100%(83.9-100)</td></tr></table>

# 7.5 Analytical Specificity

Primer pairs for the seven OncoMate™ MSI Dx Analysis System markers were checked for target specificity using the publicly available BLASTn and Primer BLAST search tools on the US National Center for Biotechnology Information website

(https://ncbi.nlm.nih.gov/tools/primer-blast/, accessed 01/14/2020; https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM $\varprojlim \varprojlim 2$ blastn&PAGE_TYPE $\varprojlim .$ BlastSearch &LINK_LOC $\varprojlim 2$ blasthome, accessed 07/24/2021; Ye et al., 2012). The primers share $100 \%$ identity with their intended targets, and BLAST results predicted specific target amplification.

The OncoMateTM MSI Dx Analysis System primers were assessed for off-target homology with $2 8 0 \%$ identity. Seven of the 14 primers exhibited off-target matches with $2 8 0 \%$ identity. However, no off-target homology was identified that would interfere with assay interpretation.

In summary, the labeled primers are specific for the template of interest because of 3’ terminal mismatches and/or depressed annealing temperatures with off-target template. Also, by design, less-specific primers are unlabeled. Accordingly, non-specific product generated with the unlabeled primers would not be detected by the analytical system and would not interfere with the assay. In addition, the assay only considers fluorescent products within the range of 60 through 300bp. Therefore, any non-specific product observed outside of this range would not affect the assay results, and possible effects of non-specific amplification with labeled primers would be limited to the dye channel of the specific label. Finally, there were no reproducible, unexpected amplification artifacts observed during product development or during the analytical or clinical studies. These theoretical and empirical data support the Primer BLAST analysis showing target specificity of the primer pairs.

An analytical specificity study evaluated primer specificity of the OncoMate™ MSI Dx Analysis System and demonstrated compatibility of the system with different thermal cycler models.

The 2800M Control DNA was used, and replicates were treated as MSS samples for analysis using the OncoMate™ MSI Dx Interpretive Software. Samples were amplified in duplicate with the OncoMate™ MSI Dx Analysis System amplification kit using 1ng, 2ng or $4 \mathrm { n g }$ of DNA on each of three different thermal cycler models. The thermal cyclers (Applied Biosystems Veriti $\textsuperscript { \textregistered }$ 96-Well Thermal Cycler, Eppendorf MasterCycler $\textsuperscript { \textregistered }$ Nexus SX1 Thermal Cycler and BioRad C1000 Touch™ Thermal Cycler) all fall within the following required performance specifications:

Maximum Block Ramp Rate: $3 . 9 ^ { \circ } \mathrm { C } /$ second to $5 ^ { \circ } \mathrm { C } /$ second Temperature Accuracy: $\pm 0 . 2 5 ^ { \circ } \mathrm { C }$ (at ${ \geq } 9 0 ^ { \circ } \mathrm { C }$ ) Temperature Uniformity: ${ < } 0 . 5 ^ { \circ } \mathrm { C }$ (at ${ \geq } 9 0 ^ { \circ } \mathrm { C }$ ) Heated lid capable of reaching $1 0 3 { \mathrm { - } } 1 0 5 ^ { \circ } \mathrm { C }$

There were no invalid results or samples requiring reinjection or reamplification testing for this study.

There was $100 \%$ agreement (36/36) between the expected and observed MSS call for all samples. There were no artifacts observed that interfered with the system’s ability to provide the expected interpretive result when using different thermal cyclers.

# 7.6 Interfering Substances

A study was performed to establish the potential influence of interfering substances on the performance of the OncoMate™ MSI Dx Analysis System, specifically chaotropic salts, alcohol, proteinase K treatment time, necrotic tissue, hemoglobin, triglycerides and mucin. DNA was extracted from sample curls $\mathrm { 0 . 1 { - } } 2 \mathrm { m m } 3$ tissue). DNA extraction was performed for each sample at each condition tested using the Maxwell $\textsuperscript { \textregistered }$ CSC Instrument and Maxwel $\textsuperscript { \textregistered }$ CSC DNA FFPE Kit.

In the first series of experiments, lysates from four tumor and matched normal samples were spiked with hemoglobin $\mathrm { 2 m g / m l }$ final concentration), triglycerides ( $3 7 \mathrm { m M }$ final concentration) or mucin $\mathrm { { 1 m g / m l } }$ final concentration) prior to DNA extraction. Following analysis with the OncoMate™ MSI Dx Analysis System, all samples yielded the expected result.

Four tumor and matched normal samples (two replicates per sample for a total of 16 extractions) were incubated at $5 6 \mathrm { { } ^ { \circ } C }$ in the presence of proteinase K for 20 minutes, 30 minutes (recommended condition) or 40 minutes prior to purification. All samples yielded the expected result.

Twelve tumor samples with necrotic tissue ranging from $0 - 7 5 \%$ and matched normal samples were tested. All samples yielded the expected result.

To evaluate the impact of potential carry-over alcohol or guanidine salts from the DNA purification process, aliquots of the extracted DNA from tumor and matched normal samples with varying amounts of tissue necrosis $( 0 - 7 5 \% )$ were spiked with ethanol $5 \%$ final concentration), guanidine hydrochloride ( $5 0 \mu \mathrm { M }$ final concentration) or water prior to amplification and analysis. All samples yielded the expected result.

There were 13 Invalid results initially obtained across the Interfering Substances testing. All 13 samples were reinjected resulting in 5 samples being resolved. The remaining 8 samples were resolved after reamplification.

In conclusion, the OncoMate™ MSI Dx Analysis System showed no statistically significant impact on assay performance by the interfering substances, specifically chaotropic salts ( $5 0 \mu \mathrm { M }$ guanidine hydrochloride), ethanol $( 5 \% )$ , necrotic tissue $( 0 - 7 5 \% )$ , hemoglobin $( 2 \mathrm { m g / m l } )$ , triglycerides $( 3 7 \mathrm { m M } )$ , and mucin ( $\mathrm { { ( 1 m g / m l ) } }$ or proteinase K digestion time [20, 30 (standard) or 40 minutes] tested in this study (Table 9).

Table 9: OncoMate™ MSI Dx Analysis System Sample Treatment by Interpretative Result   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=4>Interpretive Result</td><td rowspan=2 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=2>MSI-H</td><td rowspan=1 colspan=2>MSS</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>Sample Treatment</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>50.0</td><td rowspan=2 colspan=1>8</td><td rowspan=2 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>20min at 56°C</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>30min at 56°C</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>40min at 56°C</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Ethanol spike</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Guanidine spike</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Necrosis</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>58.3</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>41.7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Triglycerides</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Water spike</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>51.7</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>48.3</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100</td></tr></table>

1 All samples returned the expected result (MSS or MSI-H) in interfering substances experiments.

# 7.7 Cross Contamination

The sample-to-sample cross-contamination in the OncoMate™ MSI Dx Analysis System was evaluated using extracted DNA from an MSI-H sample and an MSS sample, including both the tumor and its matched normal sample. The samples and reagent blanks were interspersed in a grid design across 96-well plates. In this plate layout, amplification, capillary electrophoresis and analysis were performed each day for a total of 10 days. Expected results included MSI-H, MSS and No Call test results. The No Call test result was expected for the reagent blanks. The results were not averaged, and any observed carryover was reported.

There was a total of 3 Invalid results initially obtained in the study. After reinjection all 3 sample results were resolved. No reamplification was performed for this study.

There was $100 \%$ concordance between the interpretative result and the expected results for the 470 samples analyzed, and no interference with data interpretation was observed. The study concluded the OncoMate™ MSI Dx Analysis System was not susceptible to sample-to-sample cross-contamination.

# 7.8 Reproducibility

Precision and reproducibility for the OncoMate™ MSI Dx Analysis System was evaluated across multiple sites, operators, runs, days, replicates and assay kit lots. Data were assessed for between-site, between-operator, between- run, between-day, within-run and between-lot repeatability and precision.

The test panel consisted of seven CRC samples (four MSI-H and three MSS cases), a negative amplification control (Water, Amplification Grade) and a positive amplification control (2800M Control DNA). Two of the MSI-H samples were adjusted to a $20 \%$ tumor burden by mixing DNA extracted from the tumor sample with matched normal DNA. The samples were blinded and distributed to the operators at each of the test sites. The panel of samples was tested by two operators located at each of three sites on three instruments (one at each site). Two external sites and one internal site served as the three test sites. Three reagent lots were used for two runs per day, on three nonconsecutive days.

The Positive Percent Agreements (PPAs) for MSI-H and Negative Percent Agreements (NPAs) for MSS interpretative results versus expected results (Table 10) were reproducible for site, operator, day, lot and run. The PPAs for site, operator, day, lot and run ranged from $8 9 . 6 - 9 7 . 9 \%$ and the NPAs ranged from $9 7 . 2 - 1 0 0 \%$ , demonstrating reproducibility for each factor tested. The overall reproducibility PPA ( $9 5 \%$ CI) and NPA $9 5 \%$ CI) were $9 5 . 5 \%$ $( 9 2 . 4 - 9 7 . 6 \% )$ and $9 9 . 5 \% \%$ $( 9 7 . 4 \mathrm { - } 1 0 0 \% )$ , respectively.

Table 11 provides the absolute and relative frequency of interpretive results by sample ID.

A summary of PPA with $9 5 \%$ Wilson-Score CI for interpretive results and locus status vs reference result is shown in Table 12.

A total of 13 results were initially Invalid during testing. All 13 samples were reinjected resulting in 8 samples that resolved and 5 samples that did not resolve. Upon reamplification 3 of the samples were resolved. A total of 2 samples remained Invalid after reamplification and are included in the agreement analysis below.

In conclusion, this study demonstrated that the OncoMateTM MSI Dx Analysis System and reagents are reproducible between and across sites, operators, days, lots, and runs.

Table 10: Summary of PPA and NPA for Interpretative Result versus Reference Result   

<table><tr><td rowspan=2 colspan=1>Factor</td><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>% (#/n)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>% (#/n)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>96.9 (93/96)</td><td rowspan=1 colspan=1>91.1 - 99.4</td><td rowspan=1 colspan=1>98.6 (71/72)</td><td rowspan=1 colspan=1>92.5 - 100</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.7 (89/96)</td><td rowspan=1 colspan=1>85.6-97.0</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.9 (93/96)</td><td rowspan=1 colspan=1>91.1 - 99.4</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr><tr><td rowspan=1 colspan=1>Operator 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9 (47/48)</td><td rowspan=1 colspan=1>88.9- 100</td><td rowspan=1 colspan=1>100 (36/36)</td><td rowspan=1 colspan=1>90.3 - 100</td></tr><tr><td rowspan=1 colspan=1>Operator 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8 (46/48)</td><td rowspan=1 colspan=1>85.8 - 99.5</td><td rowspan=1 colspan=1>97.2 (35/36)</td><td rowspan=1 colspan=1>85.5 - 99.9</td></tr><tr><td rowspan=1 colspan=1>Operator 3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>89.6 (43/48)</td><td rowspan=1 colspan=1>77.3 - 96.5</td><td rowspan=1 colspan=1>100 (36/36)</td><td rowspan=1 colspan=1>90.3 - 100</td></tr><tr><td rowspan=1 colspan=1>Operator 4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>95.8 (46/48)</td><td rowspan=1 colspan=1>85.8 - 99.5</td><td rowspan=1 colspan=1>100 (36/36)</td><td rowspan=1 colspan=1>90.3 - 100</td></tr><tr><td rowspan=1 colspan=1>Operator 5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>97.9 (47/48)</td><td rowspan=1 colspan=1>88.9 - 100</td><td rowspan=1 colspan=1>100 (36/36)</td><td rowspan=1 colspan=1>90.3 - 100</td></tr><tr><td rowspan=1 colspan=1>Operator 6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>95.8 (46/48)</td><td rowspan=1 colspan=1>85.8 - 99.5</td><td rowspan=1 colspan=1>100 (36/36)</td><td rowspan=1 colspan=1>90.3 - 100</td></tr><tr><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.8 (90/96)</td><td rowspan=1 colspan=1>86.9 - 97.7</td><td rowspan=1 colspan=1>98.6 (71/72)</td><td rowspan=1 colspan=1>92.5 - 100</td></tr><tr><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8 (92/96)</td><td rowspan=1 colspan=1>89.7 - 98.8</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr><tr><td rowspan=1 colspan=1>Day 3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>96.9 (93/96)</td><td rowspan=1 colspan=1>91.1 - 99.4</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr></table>

005_510k Summary Original 510(k) Application

<table><tr><td rowspan=2 colspan=1>Factor</td><td rowspan=2 colspan=1>Item</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>% (#/n)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>% (#/n)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Lot 1 *</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95.8 (92/96)</td><td rowspan=1 colspan=1>89.7 - 98.8</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr><tr><td rowspan=1 colspan=1>Lot 2 *</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94.8 (91/96)</td><td rowspan=1 colspan=1>88.3 - 98.3</td><td rowspan=1 colspan=1>98.6 (71/72)</td><td rowspan=1 colspan=1>92.5 - 100</td></tr><tr><td rowspan=1 colspan=1>Lot 3 *</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.8 (92/96)</td><td rowspan=1 colspan=1>89.7-98.8</td><td rowspan=1 colspan=1>100 (72/72)</td><td rowspan=1 colspan=1>95 - 100</td></tr><tr><td rowspan=1 colspan=1>Run A *</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>94.4 (136/144)</td><td rowspan=1 colspan=1>89.4- 97.6</td><td rowspan=1 colspan=1>100 (108/108)</td><td rowspan=1 colspan=1>96.6 - 100</td></tr><tr><td rowspan=1 colspan=1>Run B *</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>96.5 (139/144)</td><td rowspan=1 colspan=1>92.1−98.9</td><td rowspan=1 colspan=1>99.1 (107/108)</td><td rowspan=1 colspan=1>95.0 - 100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95.5 (275/288)</td><td rowspan=1 colspan=1>92.4  97.6</td><td rowspan=1 colspan=1>99.5 (215/216)</td><td rowspan=1 colspan=1>97.4 - 100</td></tr></table>

\*Not all lots tested in a run

Table 11: Absolute and Relative Frequencies for Interpretative Results by Sample ID   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=12>Reference Result</td><td rowspan=3 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>MSI-H</td><td rowspan=1 colspan=6>MSS</td></tr><tr><td rowspan=1 colspan=1>InterpretativeResult</td><td rowspan=1 colspan=2>MSI-H</td><td rowspan=1 colspan=2>MSS</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=2>MSI-H</td><td rowspan=1 colspan=2>MSS</td><td rowspan=1 colspan=2>Invalid</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>ASREP01</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>81.9</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP02</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP03</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP04</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP05</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP06</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>ASREP07</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>504</td></tr></table>

Table 12: Summary of PPA and $9 5 \%$ Wilson-Score CI for Interpretative Result and Locus Status   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>ReferenceMSIstatus</td><td rowspan=1 colspan=1>Agreement toreferencestatus (n/N)% PPA (95%CI</td><td rowspan=1 colspan=1>NR21(n/N)% Agreement(95% CI)</td><td rowspan=1 colspan=1>BAT-26% Agreement(95% CI)</td><td rowspan=1 colspan=1>BAT-25% Agreement(95% CI)</td><td rowspan=1 colspan=1>NR-24% Agreement(95% CI)</td><td rowspan=1 colspan=1>Mono-27% Agreement(95% CI)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_01</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>59/71; 83.1%(72.7-90.1)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>27/71; 38.0%(27.6-49.7)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>53/71; 74.6%(63.4-83.3)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_02</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_03</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_04</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_05</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_06</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>AS_REP_07</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9 100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr></table>

# 7.9 Shelf Life

Studies were conducted to determine the stability of the Promega OncoMate™ MSI Dx Analysis System and Promega OncoMate™ 5C Matrix Standard.

Prior to execution of the real time or open vial studies, lots of the reagent kits were subjected to freeze-thaw cycles by cycling between the recommended storage temperature of - $\cdot 1 0 ^ { \circ }$ to $- 3 0 ^ { \circ } \mathrm { C }$ and ambient temperature until thawed. The OncoMate™ MSI Dx Analysis System amplification kit reagents were subjected to five freeze-thaw cycles and OncoMate™ 5C Matrix Standard was subjected to one freeze-thaw cycle.

Following freeze-thaw, the reagent lots were then transferred into shipping containers at - $1 0 ^ { \circ }$ to $- 3 0 ^ { \circ } \mathrm { C }$ on dry ice and shipped overnight by aircraft to a remote site. The samples were returned to Promega and stored in the shipping containers for a total of 5 days with sufficient dry ice to maintain $- 1 0 ^ { \circ }$ to $- 3 0 ^ { \circ } \mathrm { C }$ . The reagents were then transferred to a $- 1 0 ^ { \circ }$ to $- 3 0 ^ { \circ } \mathrm { C }$ freezer until used for stability testing.

# 7.9.1 Open Vial Studies

# Open Vial Reagent Stability (OncoMate™ MSI Analysis System amplification kit)

The purpose of this study was to determine the open vial shelf life stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents at $^ { + 2 }$ to $1 0 ^ { \circ } \mathrm { C }$ in real time.

DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l }$ , and $1 0 \mu \mathrm { l }$ aliquots were provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots, two of the lots were subjected to freeze-thaw and shipment. Tests were conducted at time zero, followed by each lot being tested at four additional time points of 1, 2, 3, and 4 months. Two repeats were performed for each time point.

Across all time points a total of 17 results were initially called Invalid during testing. All 17 samples were reinjected, and 16 samples were resolved. The single remaining Invalid result was resolved upon reamplification.

The open vial shelf life stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents was determined to be four (4) months, supporting the intended claim of 3 months stability at $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ .

# Open Vial Reagent Stability (OncoMate™ 5C Matrix Standard)

The purpose of this study was to determine the open vial shelf life stability of the OncoMate™ 5C Matrix Standard at $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ in real time.

One OncoMate™ 5C Matrix Standard reagent lot, subjected to freeze-thaw and shipping, was tested at time zero, followed by testing at four additional time points of 1, 2, 3, and 4 months. Two repeats were performed for each time point.

The open vial short-term stability of diluted OncoMate™ 5C Matrix Standard was performed at time zero and 7 days using three lots of reagents. Two of the lots were subjected to freeze/thaw and shipping. The open vial short-term stability study was performed 7 days after the Matrix Standard was diluted to support a 6 day stability claim for the diluted Matrix Standard.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 ~ \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The open vial shelf life stability of the OncoMate™ 5C Matrix Standard was determined to be four (4) months, supporting the intended claim of 3-month stability at $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ .

The open vial stability of diluted Matrix was determined to be seven (7) days, supporting the intended claim of six (6) days at $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ . No difference was observed based on lot, regardless of whether the lot was subjected to freeze-thaw and shipping.

# 7.9.2 Real Time Studies

# Real Time Stability at $\mathbf { \nabla } _ { - 3 0 } \circ \mathbf { C }$ to $\mathbf { - 1 0 \% }$ (OncoMate™ MSI Analysis System amplification kit)

The purpose of this study was to determine the real time stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents stored at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ in real time. The purpose of these experiments was to confirm the reagents are stable after storage at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ .

DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l }$ and provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots in corrugate packaging and three reagents lots in paperboard packaging. Two lots in each package type were subjected to freeze-thaw and shipping, of the OncoMate™ MSI Dx Analysis System at time zero, followed by each lot being thawed and tested at 6, 13, 19, and 25 months. Two repeats were performed for each time point.

Across all time points a total of 7 results were called Invalid during testing. All 7 samples were reinjected, and 6 samples were resolved. The remaining sample was resolved upon reamplification.

005_510k Summary Original 510(k) Application

The Real-Time Long-Term Stability of the OncoMate™ MSI Dx System amplification kit reagents was determined to be 24 months at frozen storage in both types of packaging based on the data collected to date. No difference was observed based on OncoMate™ MSI Dx Analysis System amplification kit lot.

# Real Time Stability at $\mathbf { \nabla } _ { - 3 0 } \circ \mathbf { C }$ to $\mathbf { - 1 0 \% }$ followed by 4 months at $^ { + 2 }$ to ${ \bf 1 0 ^ { \circ } C }$ (OncoMate™ MSI Analysis System amplification kit)

DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l }$ and provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots in corrugate packaging and three reagents lots in paperboard packaging. Two lots in each package type were subjected to freeze-thaw and shipping of the OncoMate™ MSI Dx Analysis System amplification kit at time zero and after 4 months storage at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ . Another set of samples were stored at $- 3 0 ^ { \circ } \mathrm { C }$ to - $\cdot 1 0 ^ { \circ } \mathrm { C }$ for 0, 2, 9, 15, and 21 months. Each storage period at $- 3 0 ^ { \circ } \mathrm { C }$ to - $\mathrm { - 1 0 ^ { \circ } C }$ was followed by storage for 4 months at the customer in-use temperature of $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ . Two repeats were performed for each time point.

Across all time points a total of 10 results were called Invalid during testing. All 10 samples were reinjected, and 6 samples were resolved. The remaining 4 samples were resolved upon reamplification.

The Real Time Shelf Life Stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents was determined to be frozen storage for 20 months followed by refrigerated storage for four (4) months, for a total of 24 months based on the data collected to date. No difference was observed based on OncoMate™ MSI Dx Analysis System amplification kit lot.

# Real Time Stability at $\mathbf { \nabla } _ { - 3 0 } \circ \mathbf { C }$ to $\mathbf { - 1 0 \% }$ (OncoMate™ 5C Matrix Standard)

The purpose of this study was to determine the real time stability of the OncoMate™ 5C Matrix Standard stored at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ in real time.

Samples were tested by one operator and instrument using three reagent lots, two lots subjected to freeze-thaw and shipping, at time zero and after 4 months storage at $- 3 0 ^ { \circ } \mathrm { C }$ to - $\cdot 1 0 ^ { \circ } \mathrm { C }$ . Samples were tested after storage at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ for 0, 6, 13, 19, and 25 months. Two repeats were performed for each time point.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 ~ \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The Shelf Life Stability of the OncoMate™ 5C Matrix Standard was determined to 24 months for frozen storage based on the data collected to date. No difference was observed based on OncoMate 5C Matrix Standard lot.

# Real Time Stability at $\mathbf { \nabla } _ { - 3 0 } \circ \mathbf { C }$ to $\mathbf { - 1 0 \% }$ followed by 4 months at $\mathbf { + 2 ^ { \circ } C }$ to $\mathbf { 1 0 ^ { \circ } C }$ (OncoMate™ 5C Matrix Standard)

Another set of samples were stored at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ for 2, 9, 15, and 21 months. Each storage period at $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 0 ^ { \circ } \mathrm { C }$ was followed by storage for 4 months at the customer in-use temperature of $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ . Two repeats were performed for each time point.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 ~ \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The Real-Time Long-Term Stability of the OncoMate™ 5C Matrix Standard was determined to be frozen storage for 20 months followed by storage at $^ { + 2 }$ to $1 0 ^ { \circ } \mathrm { C }$ for four (4) months, for a total of 24 months based on the data collected to date. No difference was observed based on OncoMate™ 5C Matrix Standard lot.

# 7.10 Method Comparison Study

The primary objective of the method comparison study was to evaluate the accuracy and usability of the OncoMate™ MSI Dx Analysis System in identifying microsatellite instability in the clinical setting. Tumors from Lynch syndrome patients have a high likelihood of having an MSI-H status (4, 5, 19), and therefore MSI-H status can identify patients where further genetic testing for Lynch syndrome is recommended.

A method comparison was performed between the OncoMate™ MSI Dx Analysis System and the VENTANA MMR IHC Panel, the predicate device to identify Lynch syndrome candidates, which stains for the presence or absence of DNA mismatch repair proteins. A comparison of the OncoMate™ MSI Dx Analysis System result to germline Next Generation Sequencing for DNA mismatch repair genes (NGS MMR genes) was performed to confirm identification of Lynch syndrome patients.

The study was performed by testing a sequential series of colorectal cancer patient samples that were enriched with a second set of suspected Lynch syndrome samples, for a total of 154 cases. Sample curls, generated from FFPE tissue blocks, were provided to an external laboratory to perform immunohistochemistry. Extracted DNA aliquots of 154 matched CRC DNA samples of unknown MSI status were randomized then analyzed by the OncoMate™ MSI Dx Analysis System and NGS.

Immunohistochemistry was performed on all 154 tumor samples to determine protein expression of the MHL1, MSH2, MSH6, and PMS2 using the VENTANA MMR IHC Panel on the provided sample curls, per the VENTANA instructions for use and the laboratory’s Standard Operating Procedures. BRAF testing was only performed if the sample exhibited a loss of one of the mismatch repair genes. BRAF testing may provide evidence of the mechanism for the mismatch repair deficiency and therefore differentiate Lynch syndrome candidates from sporadic dMMR cases (4, 16, 20).

DNA from the 154 samples were provided to a reference laboratory for Next Generation Sequencing. These samples underwent DNA sequencing to determine the presence or absence of pathogenic mutations of the mismatch repair genes (MLH1, MSH2, MSH6 and PMS2) and BRAF exon 15. Upon germline testing, 18 Lynch syndrome cases were confirmed. Only pathogenic or likely pathogenic mutations listed in the ClinVar database (21) were accepted as confirmed Lynch syndrome cases.

A total of 2 samples tested using the OncoMateTM MSI Dx Analysis System yielded Invalid results during initial testing. Both samples were resolved upon reinjection.

7.10.1 Method Comparison: OncoMateTM MSI Dx Analysis System vs. IHC Results The comparison results and agreement analysis between the OncoMate™ MSI Dx Analysis System and the VENTANA IHC MMR Panel for the 154 samples are listed in Table 13. A total of 106 samples were scored as MSS using the OncoMate™ MSI Dx Analysis System and MMR Intact for all four MMR proteins using the VENTANA IHC MMR Panel. Forty-one samples exhibited a loss of IHC staining for at least one of the four MMR proteins (dMMR). Of these 45 samples, 44 samples were scored as MSI-H by the OncoMate™ MSI Dx Analysis System.

Three samples were scored as MSI-H by the OncoMateTM MSI Dx Analysis System but were scored as MMR Intact by IHC staining for all four MMR proteins.

The Positive Percent Agreement (PPA) was $9 7 . 8 \%$ and the Negative Percent Agreement (NPA) was $9 7 . 2 \%$ , with an Overall Percent Agreement (OPA) of $9 7 . 4 \%$ between the two methods. Additional comparison and agreement analysis data stratified by sequential and enrichment cohort can be found in Table 14 and Table 15.

The OncoMate™ MSI Dx Analysis System effectively identified tumors with MMR deficiency and shows strong agreement with the VENTANA MMR IHC panel.

Table 13: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. Ventana MMR IHC (All Samples)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>Ventana MMR IHC results</td></tr><tr><td rowspan=1 colspan=1>MMR Loss</td><td rowspan=1 colspan=1>MMR Intact</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>154</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>44/45</td><td rowspan=1 colspan=2>97.8</td><td rowspan=1 colspan=2>88.4  99.6</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/109</td><td rowspan=1 colspan=2>97.2</td><td rowspan=1 colspan=2>92.2 99.1</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>150/154</td><td rowspan=1 colspan=2>97.4</td><td rowspan=1 colspan=2>93.5 - 99.0</td></tr></table>

Table 14: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. Ventana MMR IHC (Sequential Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>Ventana MMR IHC results</td></tr><tr><td rowspan=1 colspan=1>MMR Loss</td><td rowspan=1 colspan=1>MMR Intact</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=2>95.8</td><td rowspan=1 colspan=2>79.8  99.3</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/106</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=2>96.5  100.0</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>129/130</td><td rowspan=1 colspan=2>99.2</td><td rowspan=1 colspan=2>95.8  99.9</td></tr></table>

Table 15: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. Ventana MMR IHC (Enrichment Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=4>Ventana MMR IHC results</td></tr><tr><td rowspan=1 colspan=1>MMR Loss</td><td rowspan=1 colspan=1>MMR Intact</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>84.5  100.0</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0 - 56.1</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>21/24</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>69.0 - 95.7</td></tr></table>

# 7.10.2 Method Comparison to NGS Mismatch Repair Gene Mutations Results

A total of 18 samples tested positive for Lynch syndrome, based on detection of a pathogenic or likely pathogenic mutation in one of the mismatch repair genes (MLH1, MSH2, MSH6 or PMS2) and no mutations in BRAF exon 15. The data for all samples are summarized in Table 16. Table 17 and Table 18 summarize the data for the enrichment cohort and sequential cohort, respectively.

Seventeen of the samples (17/18) tested MSI-H with the OncoMate™ MSI Dx Analysis System. One of the samples (1/18) tested MSS with the OncoMate™ MSI Dx Analysis System and exhibited no loss of MMR protein by IHC (Table 16). This sample is referenced as having a likely pathogenic mutation in the PMS2 gene on the ClinVar database (21). This single nucleotide polymorphism (rs267608153) results in a $\mathrm { c } . 9 0 3 \mathrm { G } { > } \mathrm { T }$ variant that likely results in a splicing defect (25). All 16 of the enrichment cohort samples tested MSI-H with the OncoMateTM MSI Dx Analysis System (Table 17).

The Positive Percent Agreement (PPA) was $9 4 . 4 \%$ and the Negative Percent Agreement (NPA) was $7 7 . 9 \%$ between the two methods for all samples (Table 16). The PPA was $100 \%$ between the two methods for the enrichment cohort (Table 17). The NPA is less informative than the PPA in a comparison of somatic microsatellite instability to germline mutations in MMR genes, since cases negative for germline, pathogenic Lynch syndrome mutations include MSI-H cases with sporadic, somatic causes for dMMR as well as MSS cases. Somatic mechanisms such as epigenetic silencing and biallelic somatic mutation can lead to dMMR and an MSI-H phenotype (26) without a germline MMR gene mutation. Table 18 provides the comparison and agreement analysis between the two methods for the sequential cohort.

Table 19 provides a comparison of all three testing methods and displays the results for each sample confirmed as Lynch. This table also provides the pathogenic germline mutation, ClinVar accession number, and clinical significance for each sample confirmed as Lynch.

In conclusion, the OncoMate™ MSI Dx Analysis System effectively identified tumors with confirmed germline MMR mutations indicative of Lynch syndrome.

Table 16: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. NGS MMR Sequencing (All Samples)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>154</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=2>94.4</td><td rowspan=1 colspan=2>74.2  99.0</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/136</td><td rowspan=1 colspan=2>77.9</td><td rowspan=1 colspan=2>70.3  84.1</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>123/154</td><td rowspan=1 colspan=2>79.9</td><td rowspan=1 colspan=2>72.8  85.4</td></tr></table>

Table 17: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. NGS MMR Sequencing (Enrichment Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=2>80.6 - 100</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=2>0.0</td><td rowspan=1 colspan=2>0.0  32.4</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>16/24</td><td rowspan=1 colspan=2>66.7</td><td rowspan=1 colspan=2>46.7  82.0</td></tr></table>

Table 18: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. NGS MMR Sequencing (Sequential Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=4>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>130</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>9.5 - 90.5</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/128</td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>75.3 - 88.4</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>107/130</td><td rowspan=1 colspan=1>82.3</td><td rowspan=1 colspan=1>74.8 - 87.9</td></tr></table>

Table 19: Lynch Syndrome Cases Confirmed During the Method Comparison Study   

<table><tr><td colspan="1" rowspan="1">UIDNumber</td><td colspan="1" rowspan="1">Cohort</td><td colspan="1" rowspan="1">OncoMateTMResult</td><td colspan="1" rowspan="1">IHCStatus</td><td colspan="1" rowspan="1">Pathogenicgermlinemutation</td><td colspan="1" rowspan="1">ClinVarAccessionNumber2</td><td colspan="1" rowspan="1">Clinicalsignificance2</td></tr><tr><td colspan="1" rowspan="1">P00010</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2)</td><td colspan="1" rowspan="1">MLH1rs63751642</td><td colspan="1" rowspan="1">VCV000090067.4</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00060</td><td colspan="1" rowspan="1">Sequential</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">MSH6rs63750955</td><td colspan="1" rowspan="1">VCV000008932.6</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00092</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(PMS2)</td><td colspan="1" rowspan="1">PMS2rs121434629</td><td colspan="1" rowspan="1">VCV000009245.12</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">P00096</td><td colspan="1" rowspan="1">Sequential</td><td colspan="1" rowspan="1">MSS</td><td colspan="1" rowspan="1">Intact</td><td colspan="1" rowspan="1">PMS2rs267608153</td><td colspan="1" rowspan="1">VCV000091377.9</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">P00109</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">MSH2rs63749932</td><td colspan="1" rowspan="1">VCV000036572.7</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00120</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2)</td><td colspan="1" rowspan="1">MLH1rs587779029</td><td colspan="1" rowspan="1">VCV000090295.1</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00125</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2</td><td colspan="1" rowspan="1">MLH1rs267607767</td><td colspan="1" rowspan="1">VCV000090291.7</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00126</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2</td><td colspan="1" rowspan="1">MLH1rs876658821</td><td colspan="1" rowspan="1">VCV000230876.3</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00131</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Intact</td><td colspan="1" rowspan="1">MSH6rs730881830</td><td colspan="1" rowspan="1">VCV000182683.3</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">P00146</td><td colspan="1" rowspan="1">Enrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">PMS2rs573125799</td><td colspan="1" rowspan="1">VCV000183716.9</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00153</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2</td><td colspan="1" rowspan="1">MLH1rs267607888</td><td colspan="1" rowspan="1">VCV000090046.3</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">P00158</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2</td><td colspan="1" rowspan="1">PMS2rs876661113</td><td colspan="1" rowspan="1">VCV000234604.9</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00162</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">PMS2rs587781317</td><td colspan="1" rowspan="1">VCV000140847.7</td><td colspan="1" rowspan="1">Likely pathogenic;Uncertainsignificance</td></tr><tr><td colspan="1" rowspan="1">P00164</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">MSH2rs63751044</td><td colspan="1" rowspan="1">VCV000091262.2</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00171</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MSH6)</td><td colspan="1" rowspan="1">MSH2rs63750302</td><td colspan="1" rowspan="1">VCV000090558.1</td><td colspan="1" rowspan="1">Pathogenic</td></tr><tr><td colspan="1" rowspan="1">P00172</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2)</td><td colspan="1" rowspan="1">MLH1rs267607888</td><td colspan="1" rowspan="1">VCV000090046.3</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">UIDNumber</td><td colspan="1" rowspan="1">Cohort</td><td colspan="1" rowspan="1">OncoMateTMResult</td><td colspan="1" rowspan="1">IHCStatus</td><td colspan="1" rowspan="1">Pathogenicgermlinemutation1</td><td colspan="1" rowspan="1">ClinVarAccessionNumber{2</td><td colspan="1" rowspan="1">Clinicalsignificance2</td></tr><tr><td colspan="1" rowspan="1">P00173</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MMLH1,PMS2</td><td colspan="1" rowspan="1">MLH1rs267607888</td><td colspan="1" rowspan="1">VCV000090046.3</td><td colspan="1" rowspan="1">Pathogenic/Likelypathogenic</td></tr><tr><td colspan="1" rowspan="1">P00174</td><td colspan="1" rowspan="1">SupplementalEnrichment</td><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">Loss(MLH1,PMS2)</td><td colspan="1" rowspan="1">MLH1rs267607723</td><td colspan="1" rowspan="1">VCV000090061.4</td><td colspan="1" rowspan="1">Pathogenic</td></tr></table>

1 Lynch syndrome status was confirmed by the detection of a “Pathogenic” or “Likely pathogenic” germline mutation. Germline mutations are those detected in the normal tissue sample. 2 National Center for Biotechnology Information. ClinVar; https://www.ncbi.nlm.nih.gov/clinvar/variation/VCV... (accessed Sept. 23, 2020).

# 7.11 Conclusion

The results of the nonclinical analytical and clinical performance studies summarized above demonstrate that the OncoMateTM MSI Dx Analysis System is substantially equivalent to the predicate device.

# 8 REFERENCES

1. Jascur, T. and Boland, C.R. (2006) Structure and function of the components of the human DNA mismatch repair system. Intl. J. Cancer 119, 2030-2035   
2. Hampel, H. et al. (2005) Screening for the Lynch syndrome (hereditary nonpolyposis colorectal cancer). N. Engl. J. Med. 352, 1851–60.   
3. Lynch, H.T. and Chapelle, A. (2003) Hereditary colorectal cancer. N. Engl. J. Med. 348, 919–32.   
4. NCCN Clinical Practice Guidelines in Oncology, Genetic/Familial High-Risk Assessment: Colorectal Version 3.2019.   
5. Hampel, H. et al. (2008) Feasibility of screening for Lynch syndrome among patients with colorectal cancer. J. Clin. Oncol. 26, 5783–8.   
6. Boland, C.R. and Goel, A. (2010) Microsatellite instability in colorectal cancer. Gastroenterology. 138, 2073–87.   
7. Le, D.T. et al. (2015) PD-1 blockade in tumors with mismatch-repair deficiency. N. Engl. J. Med. 372, 2509–20.   
8. Timmermann, B. et al. (2010) Somatic mutation profiles of MSI and MSS colorectal cancer identified by whole exome next generation sequencing and bioinformatics analysis. PLoS ONE. 5(12): e15661.   
9. Bacher, J.W. et al. (2004) Development of a fluorescent multiplex assay for detection of MSI-High tumors. Dis. Markers. 20, 237–50.

10. Berg, A.O. et al. (2009) Recommendations from the EGAPP Working Group: Genetic testing strategies in newly diagnosed individuals with colorectal cancer aimed at reducing morbidity and mortality from Lynch syndrome in relatives. Genet. Med. 11, 35–41.

11. Ionov Y. et al. (1993) Ubiquitous somatic mutations in simple repeated sequences reveal a new mechanism for colonic carcinogenesis. Nature. 363, 558–61.

12. Umar, A. et al. (2004) Revised Bethesda guidelines for hereditary nonpolyposis colorectal cancer (Lynch syndrome) and microsatellite instability. J. Natl. Cancer Inst. 96, 261–8.

13. Beamer, L.C. et al. (2012) Reflex immunohistochemistry and microsatellite instability testing of colorectal tumors for lynch syndrome among us cancer programs and follow-up of abnormal results. J. Clin. Oncol. 30, 1058–63.

14. Matloff, J. et al. (2013) Molecular tumor testing for Lynch syndrome in patients with colorectal cancer. J. Natl. Compr. Canc. Netw. 11, 1380–5.

15. NCCN Clinical Practice Guidelines in Oncology, Genetic/Familial High-Risk Assessment: Colorectal Version 3.2019.

16. Rubenstein, J.H. et al. (2015) American Gastroenterological Association Institute guideline on the diagnosis and management of Lynch syndrome. Gastroenterology. 149, 777–82.

17. Sepulveda, A.R. et al. (2017) Molecular biomarkers for the evaluation of colorectal cancer: Guideline from the American Society for Clinical Pathology, College of American Pathologists, Association for Molecular Pathology, and the American Society of Clinical Oncology. J. Clin. Oncol. 35, 1453–86.

18. Southern, E.M. (1979) Measurement of DNA length by gel electrophoresis. Anal. Biochem. 100, 319–23.

19. Boland, C. R. et al. (1998) A National Cancer Institute workshop on microsatellite instability for cancer detection and familial predisposition: Development of international criteria for the determination of microsatellite instability in colorectal cancer. Cancer Res. 58, 5248-57.

20. Boyle, T.A. et al. (2014) Summary of microsatellite instability test results from laboratories participating in proficiency surveys: proficiency survey results from 2005 to 2012. Arch. Pathol. Lab. Med. 138, 363-70.

21. Landrum, M.J. et al. (2018) ClinVar: Improving access to variant interpretations and supporting evidence. Nucleic Acids Res. 46, D1062–7.

22. Van Riel, E. et al. (2010) A novel pathogenic MLH1 missense mutation, $\mathrm { c . l 1 2 A > C }$ , p.Asn38His, in six families with Lynch syndrome. Hered. Cancer Clin. Pract. 8, 7.

23. Raevaara, T.E. et al. (2005) Functional significance and clinical phenotype of nontruncating mismatch repair variants of MLH1. Gastroenterology 129, 537–49

24. Wahlberg, S.S. et al. (2002) Evaluation of microsatellite instability and immunohistochemistry for the prediction of germ-line MSH2 and MLH1 mutations in hereditary nonpolyposis colon cancer families. Cancer Res. 62, 3485–92

25. Suerink, M. et al. (2016) The effect of genotypes and parent of origin on cancer risk and age of cancer development in PMS2 mutation carriers. Genet. Med. 18, 405–9.

26. Chen, W. et al. (2017) Molecular genetics of microsatellite-unstable colorectal cancer for pathologists. Diagn. Pathol. 12, 24