# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I Background Information:

A. 510(k) Number: K200129   
B. Applicant Promega Corporation   
C. Proprietary and Established Names: OncoMate MSI Dx Analysis System   
D. Regulatory Information

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PZJ</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 864.1866 -Lynch SyndromeTest Systems</td><td rowspan=1 colspan=1>88 Pathology</td></tr></table>

II Submission/Device Overview:

A. Purpose for Submission: New device

# B. Measurand:

Five mononucleotide repeat markers (microsatellite sites BAT-26, NR-21, BAT25, MONO-27 and NR-24) in matched tumor and normal DNA obtained from formalin fixed, paraffin-embedded (FFPE) colorectal tissue sections. Two additional repeat loci pentanucleotide repeat markers (Penta C and Penta D) are part of internal controls to match tumor/normal paired samples in the test.

# C. Type of Test:

Polymerase chain reaction (PCR)-based nucleic acid amplification followed by size resolution with capillary electrophoresis

# A. Indications for Use:

The OncoMate™ MSI Dx Analysis System is a qualitative multiplex polymerase chain reaction (PCR) test intended to detect the deletion of mononucleotides in 5 microsatellite loci (BAT-25, BAT-26, NR-21, NR-24 and MONO-27) using matched tumor and normal DNA obtained from formalin fixed, paraffin-embedded (FFPE) colorectal tissue sections. The OncoMateTM MSI Dx Analysis System is for use with the Applied Biosystems $\textsuperscript { \textregistered }$ 3500Dx Genetic Analyzer and OncoMate™ MSI Dx Interpretive Software.

The OncoMate™ MSI Dx Analysis System is indicated in patients diagnosed with colorectal cancer (CRC) to detect microsatellite instability (MSI) as an aid in the identification of probable Lynch syndrome to help identify patients that would benefit from additional genetic testing to diagnose Lynch syndrome.

Results from the OncoMate™ MSI Dx Analysis System should be interpreted by healthcare professionals in conjunction with other clinical findings, family history, and other laboratory data.

The clinical performance of this device to guide treatment decision for MSI high patients has not been established.

B. Special conditions for use statement(s): Rx - For prescription use. For in vitro diagnostic use.

# C. Special instrument requirements:

Applied Biosystems 3500Dx Genetic Analyzer (k191030), OncoMate MSI Dx Assay Installer, and OncoMate MSI Dx Interpretive Software.

IV Device/System Characteristics:

# A. Device Description:

1. Kit Contents:

The OncoMate MSI Dx Analysis System includes reagents sufficient for 100 reactions (50 paired reactions). The materials listed in Table 1a are included with the kit.

Table 1a. Components of the OncoMate MSI Dx Analysis   

<table><tr><td colspan="1" rowspan="1">Component</td><td colspan="1" rowspan="1">Content</td><td colspan="1" rowspan="1">Volume</td></tr><tr><td colspan="1" rowspan="1">5X Primer Mix</td><td colspan="1" rowspan="1">Fluorophore-labeled and unlabeled primersfor BAT-26, Penta D, NR-21, BAT-25,MONO-27, NR-24 and Penta C in abuffered solution</td><td colspan="1" rowspan="1">200 μL</td></tr><tr><td colspan="1" rowspan="1">5X Master Mix</td><td colspan="1" rowspan="1">GoTaq MDx Hot Start DNA Polymerase,dNTPs, magnesium chloride and salts in abuffered solution with stabilizers</td><td colspan="1" rowspan="1">200 μL</td></tr><tr><td colspan="1" rowspan="1">2800M Control DNA,10ng/μl</td><td colspan="1" rowspan="1">Cell-line derived male genomic DNAstandard in a buffered solution</td><td colspan="1" rowspan="1">25 μL</td></tr><tr><td colspan="1" rowspan="1">Water, Amplification Grade</td><td colspan="1" rowspan="1">Water</td><td colspan="1" rowspan="1">1.25 mL</td></tr><tr><td colspan="1" rowspan="1">Size Standard 500</td><td colspan="1" rowspan="1">Fluorophore-labeled DNA fragments in abuffered solution</td><td colspan="1" rowspan="1">100 μL</td></tr></table>

Additional reagents required but not provided with the kit:

• Maxwell CSC DNA FFPE Kit   
• OncoMate MSI Dx 5C Matrix Standard   
• Hi-Di Formamide $3 5 0 0 \mathrm { D x }$ Series   
• Fluorescent-dye-based dsDNA quantification reagents   
• Nuclease-Free Water

# 2. Specimen Requirement and Sample preparation:

The OncoMate MSI Dx Analysis System is intended for use with FFPE colorectal tissue samples collected from colorectal cancer patients. The test requires DNA extracted from colorectal cancer and matched normal tissue to be extracted and run in parallel to yield a test result. Tissue sections suitable for use in the assay contain sufficient nucleated cells and a tumor content $23 0 \%$ tumor cells, and range in volume from $0 . 1 \mathrm { m m } ^ { 3 }$ to $2 . 0 \mathrm { m m } ^ { 3 }$ .

DNA is extracted from FFPE tissue specimens using the Maxwell CSC system and Maxwell CSC FFPE DNA extraction kit. Tumor and matched normal tissue specimens are extracted in parallel. The minimum recommended DNA input for the test is 1ng of DNA in $2 \mathrm { u l }$ of volume for processing test material. DNA is quantified using a fluorescence-based DNA quantification system consisting of ds-DNA binding dye and a fluorometer.

# 3. DNA Amplification and Capillary electrophoresis:

Tumor and normal DNA from patient specimens are amplified with the fluorophore labeled primers for co-amplification of seven microsatellite markers; five mononucleotide repeat markers ((BAT-25, BAT-26, NR-21, NR-24 and MONO-27) and two pentanucleotide repeat markers (Penta C and Penta D,) in parallel. The expected amplified size ranges and detection channels for the markers included in the OncoMate MSI Dx Analysis System Amplification Kit. are listed in Table 1b. The specifications for thermal cyclers and the protocol for amplification are detailed in the OncoMate MSI Dx Analysis System product IFU.

Table 1b. Markers Included in the OncoMate MSI Dx Analysis System Amplification Kit.   

<table><tr><td rowspan=1 colspan=1>MononucleotideMarkers</td><td rowspan=1 colspan=1>Repeat Structure</td><td rowspan=1 colspan=1>Detection Channel</td><td rowspan=1 colspan=1>Amplified SizeRange</td></tr><tr><td rowspan=1 colspan=1>BAT-26</td><td rowspan=1 colspan=1>A(26)</td><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>83 to 121bp</td></tr><tr><td rowspan=1 colspan=1>NR-21</td><td rowspan=1 colspan=1>A(21)</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>83 to 108bp</td></tr><tr><td rowspan=1 colspan=1>BAT-25</td><td rowspan=1 colspan=1>A(25)</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>110 to 132bp</td></tr><tr><td rowspan=1 colspan=1>MONO-27</td><td rowspan=1 colspan=1>A(27)</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>134 to 168bp</td></tr><tr><td rowspan=1 colspan=1>NR-24</td><td rowspan=1 colspan=1>A(24)</td><td rowspan=1 colspan=1>Yellow (displayedblack)</td><td rowspan=1 colspan=1>103 to 138bp</td></tr><tr><td rowspan=1 colspan=1>PentanucleotideMarkers</td><td rowspan=1 colspan=1>Repeat Structure</td><td rowspan=1 colspan=1>Detection Channel</td><td rowspan=1 colspan=1>Amplified SizeRange</td></tr><tr><td rowspan=1 colspan=1>Penta D</td><td rowspan=1 colspan=1>AAAGA(217)</td><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>123 to 253bp</td></tr><tr><td rowspan=1 colspan=1>Penta C</td><td rowspan=1 colspan=1>AAAAC(4-17)</td><td rowspan=1 colspan=1>Yellow (displayedblack)</td><td rowspan=1 colspan=1>140 to 228bp</td></tr></table>

OncoMate MSI Dx Analysis System amplification products are analyzed by capillary electrophoresis using the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer in ‘Diagnostic Mode’ using POP-7 3500 Dx Series Polymer and a $3 5 0 0 \mathrm { D x }$ Series Capillary Array, $5 0 \mathrm { c m }$

During capillary electrophoresis, OncoMate MSI Dx Analysis System amplification products are separated and analyzed alongside fluorescently labeled DNA fragments of known size, the Size Standard 500.

# 4. Data Analysis:

The OncoMate MSI Dx Analysis System generates size data (i.e., DNA fragment length) for microsatellite regions amplified from matched normal and CRC tumor sample pairs. These data are analyzed using the OncoMate MSI Dx Interpretive Software to determine tumor sample MSI status. Following capillary electrophoresis, the resulting DNA fragment data (.fsa files) are simultaneously imported and analyzed by the OncoMate MSI Dx Interpretive Software. During this process, data quality control (QC) checks are performed, and DNA fragments amplified from seven microsatellite regions are sized with reference to the size standard fragments at each loci is analyzed by the software per the criteria described in Result Interpretation section below.

# 5. Controls:

# 1.Positive and Negative controls

The Positive Control is 2800M Control DNA $( 1 0 \mathrm { n g / \mu l } )$ which is DNA extracted from a microsatellite stable human cell line. The No-template Control is Water, Amplification Grade which is used as a reagent blank.

Each control is analyzed concurrently with patient samples to verify assay performance. At least one 2800M Control DNA amplification reaction and one notemplate control amplification reaction must be completed for each plate (i.e., batch) of patient samples analyzed using the OncoMate MSI Dx Interpretive Software. The no-template control reaction is analyzed to ensure that no unexpected amplification occurred in no-template reactions, which would indicate the presence of DNA contamination and lead to an Invalid assay result. The positive control reaction is analyzed to demonstrate that the amplification chemistry performed as expected.

# 2. Capillary Electrophoresis (CE) Standards

All analyzed samples and controls must contain Size Standard 500 (added prior to CE). Size Standard 500 contains a series of 21 DNA fragments of known lengths (60, 65, 80, 100, 120, 140, 160, 180, 200, 225, 250, 275, 300, 325, 350, 375, 400, 425, 450, 500bp), also referred to as a DNA ladder. Each fragment is labeled with WEN dye and is detected separately (as a fourth color, orange) in the presence of OncoMate MSI Dx Analysis System-amplified products using the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer. For each sample or control, amplified DNA fragments are sized with reference to the size standard fragments. The size standard controls for capillaryto-capillary variations in sizing precision during capillary electrophoresis and allows direct comparison of samples across the capillary electrophoresis run. Only the 60- base to 300-base fragments are analyzed for fragment sizing in the OncoMate MSI Dx Interpretive Software.

# 6. Result Reporting:

The OncoMate MSI Dx Interpretive Software provides an automated interpretive result, either microsatellite instability -high (MSI-H) or microsatellite stable (MSS. Five mononucleotide-repeat markers (BAT-25, BAT-26, NR-21, NR-24 and MONO-27) and two pentanucleotide-repeat markers (Penta C and Penta D) are evaluated. The analysis of mononucleotide-repeat markers in CRC tumor samples relative to the repeat lengths in normal tissue determines tumor MSI status. The size difference (bp) between the allele of interest in the normal and tumor samples is calculated to determine marker stability. The interpretive software defines marker instability as a 3bp change (implemented as ${ \ge } 2 . 7 5$ bp to account for the sizing imprecision of capillary electrophoresis). The interpretive software will score alleles that are shifted less than 3bp from the normal reference allele as stable. A tumor sample is interpreted as MSI-H when $\geq 2$ markers are ‘Unstable’. A tumor sample is interpreted as MSS when $< 2$ markers are interpreted as ‘Unstable’.

# B. Test Principle:

Lynch Syndrome is an inherited disorder caused by autosomal dominant germline mutations in the DNA mismatch repair (MMR) genes that increases the risk of developing many types of cancer. About $3 \%$ of colorectal cancer is caused by Lynch syndrome. The MMR gene products are responsible for repair of DNA replication errors and mutations to these genes result in accumulation of mutations in genomic DNA. Accumulation of DNA replication errors can be assessed in short repeat regions of the genome called microsatellites. Microsatellite instability testing can be used to identify those colorectal cancer patients who can benefit from confirmatory genetic testing for Lynch syndrome due to an MSI high (MSI-H) result.

The OncoMate MSI Dx Analysis System is a fluorescent, multiplex PCR-based test to detect DNA sequence length changes in microsatellite regions of colorectal tumor cell DNA relative to the same regions from the patient’s normal cells. Microsatellites are short, DNA-repeat regions that are distributed throughout the human genome and are prone to insertion and deletion copying errors during DNA replication.

The OncoMate MSI Dx Analysis System is used to examine the frequency of errors in five homopolymer (poly-A) microsatellite regions of the genome (NR-21, BAT-26, BAT-25, NR-24, and MONO-27). The replication errors result in an increase or decrease in the number of nucleotides in the locus that can be measured by PCR amplification of the regions followed by determination of the fragment length of the resultant amplicon. Amplicons of the expected size indicate DNA repair is functioning normally. A change in the fragment length is indicative of dysfunction. This assessment is performed by comparing DNA extracted from matched tumor and normal samples from the same individual.

If the length of two or more of the five mononucleotide-repeat marker alleles is changed by $2 2 . 7 5$ base pairs (bp), the tumor is classified as MSI-H; if the allele length is changed for only one marker, or if the difference in allele lengths at the five markers is ${ < } 2 . 7 5 \mathrm { b p }$ , the tumor is classified as Microsatellite Stable (MSS). The sizes of the Penta C and Penta D pentanucleotide-repeat marker alleles are compared as an identity check between the normal and tumor DNA samples.

# V Substantial Equivalence Information:

A. Predicate Device Name: Ventana MMR IHC panel B. Predicate 510(k) Number: DEN170030 C. Comparison with Predicate:

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K200129</td><td colspan="1" rowspan="1">DEN170030</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Promega MSI Oncotype Dx</td><td colspan="1" rowspan="1">Ventana MMR IHCpanel</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">As an aid in the identification ofprobable Lynch syndrome.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Limitation</td><td colspan="1" rowspan="1">The clinical performance of thisdevice to guide treatmentdecision for MSI high patients</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">has not been established.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen</td><td colspan="1" rowspan="1">FFPE Tumor Tissue</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Target Population</td><td colspan="1" rowspan="1">Patients diagnosed with CRC</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">External Positive and Negativecontrols</td><td colspan="1" rowspan="1">External Positive andNegative controls</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Differences</td></tr><tr><td colspan="1" rowspan="1">SupplementalBRAFV600E testing</td><td colspan="1" rowspan="1">Not applicable</td><td colspan="1" rowspan="1">As an aid todifferentiate betweensporadic CRC andprobable Lynchsyndrome.</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">PCR based microsatellitemeasurement in normal andtumor DNA</td><td colspan="1" rowspan="1">Immunohistochemistrybased proteinexpression</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">5 Mononucleotide tractsBAT25, BAT26, MONO27,NR21 and NR24</td><td colspan="1" rowspan="1">4 -Mismatch repair(MMR) proteins MLH1,MSH2, MSH6 andPMS2 as well asBRAFV600 E protein</td></tr><tr><td colspan="1" rowspan="1">Instruments</td><td colspan="1" rowspan="1">Applied BioSystems 3500 DxGenetic Analyzer</td><td colspan="1" rowspan="1">Ventana BenchmarkUltra stainer</td></tr><tr><td colspan="1" rowspan="1">Result Interpretation</td><td colspan="1" rowspan="1">OncoMate MSI Dx InterpretiveSoftware</td><td colspan="1" rowspan="1">Pathologist assessment</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">2800M Control DNA (10ng/μl)extracted from MSS human cellline and no template control</td><td colspan="1" rowspan="1">Pre-qualified CRCtissue expressing MMRproteins</td></tr><tr><td colspan="1" rowspan="1">Calibration</td><td colspan="1" rowspan="1">Size standards</td><td colspan="1" rowspan="1">Not applicable</td></tr></table>

# VI Standard/Guidance Document Referenced (if applicable):

CLSI Standard EP05-A3, Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline – Second Edition.

•CLSI EP12-A2: User Protocol for Evaluation of Qualitative Test Performance - Approved Guideline – Second Edition

CLSI Standard EP17-A2, Protocols for the Determination of Limits of Detection and Limits of Quantification, Approved Guideline – Second Edition.

# A. Analytical performance

# 1. Precision/Reproducibility Study

Precision of the OncoMate MSI test was assessed in DNA extracted from 7 CRC cases (4 MSI-H and 3 MSS cases) and the corresponding matched normal samples. Samples were randomized for blind evaluation by two (2) operators located at each of three (3) sites, on three (3) instruments (1 at each site). Two (2) replicates per sample were tested, using three (3) lots, two (2) runs per day, on three (3) non-consecutive days for a total of 36 replicates per sample. DNA from the 7 CRC cases and matched normal were extracted at the sponsor site with the Maxwell CSC DNA FFPE extraction kit on the Maxwell CSC instrument and quantitated with QuantiFluor dsDNA system according to instructions. Different thermocyclers were used at each site.

CRC specimens contained a minimum of at least $20 \%$ tumor cells and 1 ng DNA (tumor and Normal each) was used as sample input for the study. Two of the study samples were generated by mixing tumor and matched normal DNA to yield $20 \%$ tumor cell content for the sample. This was intended to represent challenging case with minimal required tumor content. One of the two challenging cases at $20 \%$ tumor content represented a borderline case with instability at 3 of the 5 loci (NR-21 and NR-24 were stable for this sample). The reference status of the markers for the samples used in the study are shown in table below

Table 2: Sample Characteristics for specimens enrolled in Precision study   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>NR-21</td><td rowspan=1 colspan=1>BAT-26</td><td rowspan=1 colspan=1>BAT-25</td><td rowspan=1 colspan=1>NR-24</td><td rowspan=1 colspan=1>Mono-27</td><td rowspan=1 colspan=1>Status</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>Unstable</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>MSS</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>MSS</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>Stable</td><td rowspan=1 colspan=1>MSS</td></tr></table>

Precision of the overall qualitative MSI-H or MSS result (i.e., interpretive result) and each allele was assessed. Positive percent agreement (PPA) and negative percent agreement (NPA) were calculated. NPA was based on MSS results. Point estimates and $9 5 \%$ Clopper-Pearson confidence intervals for PPA and NPA were provided for the following factors:

Between sites (one instrument within each site) Between operators within a site   
• Between days per operator   
Between lots within a day Within-run

The overall invalid rate for all the sample test results where batch QC passed was $3 . 4 \%$ (19/504). A total of 13 results were initially invalid during testing. All 13 samples were reinjected, resulting in 8 samples that resolved and 5 samples that did not resolve. Upon reamplification, three of the five unresolved samples were resolved. A total of two samples remained invalid after reamplification and are included in the agreement analysis below.After repeat testing of invalids and no calls $( 0 . 7 \% ; 4 / 5 0 4 )$ in accordance with the instructions for use, the final invalid rate for the testing was $0 . 4 \%$ (2/504).

The PPAs for MSI-H and NPAs for MSS interpretative results versus expected results (Table 3 & 4) were reproducible for site, operator, day, lot, and run. The PPAs for site, operator, day, lot, run and replicate ranged from 83.3 to $100 \%$ and the NPAs were all $100 \%$ , demonstrating reproducibility for each factor tested. The overall reproducibility (and $9 5 \%$ CI) for PPA and NPA were $9 5 . 8 \%$ (91.2 – 98.5) and $100 \%$ (96.6 -100), respectively. The slightly lower agreement rate of $8 3 . 1 \%$ (51/71) was observed for sample 1 (see table 4 below). This sample also showed $3 8 \%$ (27/71) agreement at BAT-25 and agreement of $7 4 . 6 \%$ (53/71) at MONO27 loci. This sample was tested at $20 \%$ tumor content (below the recommended $30 \%$ tumor content) to represent a challenging case and in reference testing showed instability at 3 of the 5 loci.

Table 3: Summary of PPA and NPA for Interpretative Result versus Reference Result   

<table><tr><td rowspan=1 colspan=1>Factor</td><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>% (#n)</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>% (#/n)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Site</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100% (48/48)</td><td rowspan=1 colspan=1>[92.6; 100]</td><td rowspan=1 colspan=1>100% (36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>91.7% (44/48)</td><td rowspan=1 colspan=1>[80.0; 97.7]</td><td rowspan=1 colspan=1>100% (36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>95.8% (46/48)</td><td rowspan=1 colspan=1>[85.7; 99.5]</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=6 colspan=1>Operator</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>[85.8;100]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>[85.8;100]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>83.3% (20/24)</td><td rowspan=1 colspan=1>[62.6; 95.3]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>100% (24/24)</td><td rowspan=1 colspan=1>[85.8; 100]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[78.9; 99.9]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[78.9; 99.9]</td><td rowspan=1 colspan=1>100% (18/18)</td><td rowspan=1 colspan=1>[81.5;100]</td></tr><tr><td rowspan=3 colspan=1>Day</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>93.8% (45/48)</td><td rowspan=1 colspan=1>[82.8; 98.7]</td><td rowspan=1 colspan=1>100% (36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8% (46/48)</td><td rowspan=1 colspan=1>[85.7; 99.5]</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>[88.9; 99.9]</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=3 colspan=1>Lot *</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>97.9% (47/48)</td><td rowspan=1 colspan=1>[88.9; 99.9]</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.8% (46/48)</td><td rowspan=1 colspan=1>[85.7; 99.5]</td><td rowspan=1 colspan=1>100% (36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>93.8% (45/48)</td><td rowspan=1 colspan=1>[82.8; 98.7]</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>[90.3;100]</td></tr><tr><td rowspan=2 colspan=1>Run *</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>95.8% (69/72)</td><td rowspan=1 colspan=1>[88.3; 99.1]</td><td rowspan=1 colspan=1>100% (54/54)</td><td rowspan=1 colspan=1>[93.4;100]</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>95.8% (69/72)</td><td rowspan=1 colspan=1>[88.3; 99.1]</td><td rowspan=1 colspan=1>100% (54/54)</td><td rowspan=1 colspan=1>[93.4;100]</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95.8% (138/144)</td><td rowspan=1 colspan=1>[91.2; 98.5]</td><td rowspan=1 colspan=1>100% (108/108)</td><td rowspan=1 colspan=1>[96.6;100]</td></tr></table>

Table 4: Precision per sample and loci   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>MSIStatus</td><td rowspan=1 colspan=1>Agreementto Reference(n/N);Percent PPA(95% CI)</td><td rowspan=1 colspan=1>NR-21(n/N);(95% CI)</td><td rowspan=1 colspan=1>BAT-26(n/N); (95%CI</td><td rowspan=1 colspan=1>BAT-25(n/N);(95% CI)</td><td rowspan=1 colspan=1>NR-24(n/N);(95% CI)</td><td rowspan=1 colspan=1>Mono-27(n/N);(95% CI)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>59/71; 83.1%(72.7-90.1)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>27/71; 38.0%(27.6 49.7)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>53/71; 74.6%(63.4-83.3)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>71/71; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td><td rowspan=1 colspan=1>72/72; 100%(94.9-100)</td></tr></table>

# 2. DNA Extraction:

The imprecision of the assay between extractions was evaluated. DNA extractions from FFPE curls $\left( 0 . 1 { - } 2 . 0 \mathrm { m m } ^ { 3 } \right.$ tissue) obtained from four MSI-H, three MSS tumor samples and matched normal samples were performed using the Maxwell CSC DNA FFPE Kit and the Maxwell CSC Instrument. The MSI-H samples were at $20 { - } 3 0 \%$ tumor content, and the MSS samples were at $20 \mathrm { - } 6 0 \%$ tumor content. Each sample was processed by 2 operators x 2 replicates $\textbf { X } 3$ days yielding 12 extractions from 7 CRC specimens (i.e., 84 independent extractions). These were run in duplicates to generate 24 expected results per sample or a total of 168 results. Once isolated, the extracted DNA was quantified using the QuantiFluor dsDNA System and run according to protocol.

A total of five (5) cases study samples initially yielded invalid results. After reinjection of all five samples, one sample was resolved. The remaining four (4) samples were resolved by reamplification.

In the study, $9 6 . 4 \%$ (81/84) of the individual FFPE curls extracted produced results that were concordant with the predetermined MSI status. The $9 5 \%$ confidence intervals (CI) for percent correct and percent incorrect results were $8 9 . 9 - 9 9 . 3 \%$ and $0 . 7 \mathrm { - } 1 0 . 1 \%$ , respectively.

The study demonstrated that the Maxwell $\textsuperscript { \textregistered }$ CSC Instrument using the Maxwell $\textsuperscript { \textregistered }$ CSC

DNA FFPE Kit for DNA extraction met extraction capabilities for use with the OncoMate™ MSI Dx Analysis System.

# 3. Normal Range and Cutoff:

Microsatellite instability at each of the five target loci is measured as change in fragment length in amplification product of matched tumor and normal DNA samples. A normal range study was conducted to assess the systems capability to resolve amplicons that differ by ${ > } 3$ base pairs (bp). Two (2) sets of 7 synthetic DNA fragments (“resolution markers”) consisting of dye-labeled amplicons of known size that are separated by 1bp within each set was used in the study to assess fragment resolution. The two sets were designed to bracket the upper (Large) and lower (Small) ends of the amplicon size range of MSI markers (83-168bp).

Resolution markers were analyzed either mixed only with the Size Standard 500 or mixed separately with two MSS tumor samples and the Size Standard 500 using the Applied Biosystems $3 5 0 0 \mathrm { D x }$ genetic analyzer and with OncoMate MSI Dx interpretive software.

The sizing precision of individual resolution fragments was characterized (Table 2), and size differences between all fragments separated by 3bp were calculated and averaged. Observed mean differences were compared with predicted values.

Table 5: Descriptive Statistics for Resolution Marker Base Pair Size   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=2>95%-CI</td></tr><tr><td rowspan=1 colspan=1>ResolutionMarker</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Minimum</td><td rowspan=1 colspan=1>Maximum</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Standarddeviation</td><td rowspan=1 colspan=1>LowerLimit</td><td rowspan=1 colspan=1>UpperLimit</td></tr><tr><td rowspan=1 colspan=1>Large</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>180.60</td><td rowspan=1 colspan=1>180.95</td><td rowspan=1 colspan=1>180.77</td><td rowspan=1 colspan=1>180.75</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>180.71</td><td rowspan=1 colspan=1>180.80</td></tr><tr><td rowspan=1 colspan=1>Large 2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>181.67</td><td rowspan=1 colspan=1>182.02</td><td rowspan=1 colspan=1>181.89</td><td rowspan=1 colspan=1>181.83</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>181.78</td><td rowspan=1 colspan=1>181.88</td></tr><tr><td rowspan=1 colspan=1>Large 3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>182.75</td><td rowspan=1 colspan=1>182.97</td><td rowspan=1 colspan=1>182.85</td><td rowspan=1 colspan=1>182.86</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>182.83</td><td rowspan=1 colspan=1>182.90</td></tr><tr><td rowspan=1 colspan=1>Large 4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>183.70</td><td rowspan=1 colspan=1>184.04</td><td rowspan=1 colspan=1>183.88</td><td rowspan=1 colspan=1>183.85</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>183.80</td><td rowspan=1 colspan=1>183.89</td></tr><tr><td rowspan=1 colspan=1>Large 5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>184.66</td><td rowspan=1 colspan=1>184.99</td><td rowspan=1 colspan=1>184.83</td><td rowspan=1 colspan=1>184.80</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>184.76</td><td rowspan=1 colspan=1>184.85</td></tr><tr><td rowspan=1 colspan=1>Large 6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>185.73</td><td rowspan=1 colspan=1>186.06</td><td rowspan=1 colspan=1>185.90</td><td rowspan=1 colspan=1>185.88</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>185.83</td><td rowspan=1 colspan=1>185.93</td></tr><tr><td rowspan=1 colspan=1>Large7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>186.80</td><td rowspan=1 colspan=1>187.01</td><td rowspan=1 colspan=1>186.91</td><td rowspan=1 colspan=1>186.91</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>186.88</td><td rowspan=1 colspan=1>186.94</td></tr><tr><td rowspan=1 colspan=1>Small</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>84.29</td><td rowspan=1 colspan=1>84.71</td><td rowspan=1 colspan=1>84.62</td><td rowspan=1 colspan=1>84.57</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>84.51</td><td rowspan=1 colspan=1>84.63</td></tr><tr><td rowspan=1 colspan=1>Small 2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>85.52</td><td rowspan=1 colspan=1>85.95</td><td rowspan=1 colspan=1>85.83</td><td rowspan=1 colspan=1>85.79</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>85.72</td><td rowspan=1 colspan=1>85.85</td></tr><tr><td rowspan=1 colspan=1>Small 3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>86.66</td><td rowspan=1 colspan=1>87.14</td><td rowspan=1 colspan=1>86.95</td><td rowspan=1 colspan=1>86.92</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>86.86</td><td rowspan=1 colspan=1>86.98</td></tr><tr><td rowspan=1 colspan=1>Small 4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>87.71</td><td rowspan=1 colspan=1>88.15</td><td rowspan=1 colspan=1>87.99</td><td rowspan=1 colspan=1>87.97</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>87.91</td><td rowspan=1 colspan=1>88.03</td></tr><tr><td rowspan=1 colspan=1>Small 5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>88.73</td><td rowspan=1 colspan=1>89.18</td><td rowspan=1 colspan=1>89.04</td><td rowspan=1 colspan=1>89.02</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>88.96</td><td rowspan=1 colspan=1>89.07</td></tr><tr><td rowspan=1 colspan=1>Small 6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>89.96</td><td rowspan=1 colspan=1>90.40</td><td rowspan=1 colspan=1>90.25</td><td rowspan=1 colspan=1>90.21</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>90.15</td><td rowspan=1 colspan=1>90.27</td></tr><tr><td rowspan=1 colspan=1>Small7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>91.06</td><td rowspan=1 colspan=1>91.55</td><td rowspan=1 colspan=1>91.39</td><td rowspan=1 colspan=1>91.34</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>91.28</td><td rowspan=1 colspan=1>91.41</td></tr></table>

Resolution fragments were sized with standard deviations ranging from 0.07–0.13bp. Mean absolute differences calculated for fragments separated by 3bp ranged from 3.06– 3.40bp and 3.05–3.35bp for observed and predicted values, respectively. Measurement precision of individual resolution markers (standard deviations ${ \underline { { < 0 . 1 3 } } }$ base pairs) was sufficient to detect single-base-pair differences in size. The study met the objective of demonstrating a ${ \ge } 3$ base pair resolution in the test system.

# 4. Analytical Sensitivity

# i. Limit of Detection:

The analytical sensitivity of the OncoMate MSI Dx Analysis System for detection of each marker when the marker is categorized as instable was evaluated in two studies. Due to the rarity of samples with 1, 2 or 3 instable markers (i.e., samples with results that would be considered near the clinical decision point), the LoD was taken as the lowest tumor proportion at which the minimum DNA input (1ng) yielded no more than one marker with less than $9 5 \%$ call rate.

Study 1: Analytical sensitivity of the assay was first determined using extracted DNA isolated from five MSI-H tumor and matched normal samples, as well as a titration series of the 2800M Control DNA. The 2800M Control DNA samples were treated as MSS samples for analysis using the OncoMate MSI Dx Interpretive Software. Contrived MSI-H specimens representing low tumor content were generating by blending tumor DNA and matched normal non-tumor DNA. The MSI-H samples and the MSS samples were tested at 0.2, 0.5, 1.0, 2.0 and $2 . 5 \mathrm { n g }$ DNA per amplification reaction with 20 replicates for each sample and DNA amount (five MSI-H cases $\times 2 0$ replicates). To evaluate the influence of tumor content on limit of detection (LOD), one tumor sample was combined with the matched normal sample to simulate a tumor content of $5 \%$ , $10 \%$ , $1 5 \%$ and $20 \%$ .

Each sample in the study was tested on at least 1 instrument with two OncoMate MSI Dx Analysis system lots at 6 different concentrations by two operators (2 replicates/ operator/ sample) and over 5 days for a total of 20 replicates per sample. to identify the LOD. New sample dilutions were prepared to test a 0.1ng per reaction DNA input using samples with a $20 \%$ tumor content, as well as a 1ng per reaction DNA input using a sample containing $2 . 5 \%$ tumor.

A combined lot analysis determined $9 9 . 5 \%$ (199/200) positivity at a DNA concentration of $0 . 1 \mathrm { n g / \mu l }$ $( 0 . 2 \mathrm { n g }$ input) for sample interpretive results. At the $0 . 1 \mathrm { n g / \mu l }$ DNA concentration, a No Call test result was obtained for one replicate out of the twenty replicates for that sample case with a single reagent lot (Table 6) due to low signal. While $100 \%$ positivity was noted at $0 . 2 5 \mathrm { n g / \mu l }$ of DNA using a $2 \mu \mu$ $( 0 . 5 \mathrm { n g } )$ input using the minimum $20 \%$ tumor content, test performance at individual loci was also assessed to determine LoD for each of the target loci as the study did not include challenging cases with instability at 2 or 3 of the five target loci. Overall the positivity rate was $59 5 \%$ for NR-21, BAT-26 and MONO-27 loci while at $80 \%$ and $60 \%$ for BAT-25 and NR-24 loci. An additional supplemental study was conducted to determine the DNA input and tumor content at which performance at these loci would be agree $59 5 \%$ for calling at each loci to represent test performance with challenging cases. The results are summarized in table 5 below.

Table 6: Analytical sensitivity of MSI-H Calls Based on DNA Input   

<table><tr><td rowspan=2 colspan=2>Lot by Final DNAInput (ng)</td><td rowspan=1 colspan=2>OncoMateTM InterpretiveResult</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Lot 1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=6 colspan=1>Lot 2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=6 colspan=1>All</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>199</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>200</td></tr></table>

1 One No Call due to low allele peak height detected.

Table 7: Analytical sensitivity of MSS Calls Based on DNA Input   

<table><tr><td rowspan=2 colspan=2>Lot by Final DNAInput (ng)</td><td rowspan=1 colspan=2>OncoMateTM InterpretiveResult</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>No Call</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=6 colspan=1>Lot 1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=6 colspan=1>Lot 2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=6 colspan=1>All</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr></table>

Table 8: Analytical sensitivity - Tumor Content study   

<table><tr><td rowspan=2 colspan=2>Lot by Tumorcontent (%)</td><td rowspan=1 colspan=2>OncoMateTM InterpretiveResult</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=5 colspan=1>Lot 1</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=5 colspan=1>Lot 2</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=5 colspan=1>All</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr></table>

Table 9: Summary of Reference Result vs. Interpretive Result and Marker Status with $9 5 \%$ Wilson-Score CI by Sample (1ng DNA Input and $2 0 \%$ Tumor Content).   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=6>20% Tumor content</td></tr><tr><td rowspan=1 colspan=1>ngDNAinput</td><td rowspan=1 colspan=1>NR21(n/N)%Concordantto referenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-26 (n/N)%Concordant toreferenceresult (95%CI</td><td rowspan=1 colspan=1>BAT-25 (n/N)%Concordantto referenceresult (95%CII</td><td rowspan=1 colspan=1>NR-24 (n/N)%Concordantto referenceresult (95%CII</td><td rowspan=1 colspan=1>Mono-27 (n/N)%Concordant toreference result(95% CI)</td></tr><tr><td rowspan=1 colspan=1>2800M(MSS)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-066(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-076(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>17/40; 42.5%(28.5-57.8)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-079(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-081(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>36/40; 90%(77.0-96.0)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>39/40; 97.5(87.1-99.6)</td></tr><tr><td rowspan=1 colspan=1>CRC-084(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>6/40; 15%(7.1--29.1)</td><td rowspan=1 colspan=1>2/40; 5%(1.4-16.5)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr><tr><td rowspan=1 colspan=1>CRC-213(MSI-H)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td><td rowspan=1 colspan=1>40/40; 100%(91.2-100)</td></tr></table>

Study 2: Because the $9 5 \%$ correct call rate was not achieved for 2 markers when $20 \%$ tumor content is used, a supplemental limit of detection study was conducted to resolve performance at the NR-24 and BAT-25 loci and assess the LoD for detection of instability at all of the five loci with challenging cases. This study included specimens with imperfect concordance at different target loci at minimal recommended assay input of $20 \%$ tumor content and $1 \mathrm { n g } \mathrm { D N A }$ input though reference testing at unblended samples ( $50 \%$ tumor content) indicated instability at all 5 loci for all of the 3 cases. These cases were considered to represent challenging cases and simulate cases with instability at 2 or 3 loci. Limit of detection for cases with instability in 2 or 3 of the target loci is defined as tumor content and input at which there is $59 5 \%$ concordance. Three MSI cases that were generated by blending the tumor DNA with matched normal DNA were tested with samples simulating tumor DNA content of 10, 20, 30 and $50 \%$ . . Each sample was tested using a single lot across multiple days on one instrument at three different DNA concentrations (0.5ng, $1 . 0 \mathrm { n g }$ and $2 \mathrm { n g }$ of DNA) and 20 replicates per concentration. Interpretive Result frequencies summed across all sample groups from the LoD Supplemental study with the MSI-H Blended and Diluted sample set for various DNA input at different tumor contents are shown in Tables 9 13. Sample conditions that fell below the $9 5 \%$ call rate are in bold.

Agreement rates to instability at each loci was variable (ranging from $0 \%$ at $10 \%$ tumor input over the range of DNA concentrations tested and therefore below the LoD for the test. Similarly, at $20 \%$ tumor input instability at precision at the BAT-25 and NR-24 loci was measured at $0 . 1 0 \%$ for all concentrations of DNA tested while MONO-27 instability was measured at $8 5 \%$ agreement for 2 other cases at lowest DNA input of $0 . 5 \mathrm { n g }$ . At $30 \%$ tumor input one of the cases showed agreement rate of $9 5 \%$ at BAT25 and $0 \%$ at NR-24 loci while only $50 \%$ tumor input produced $100 \%$ agreement for instability at all 5 loci for all 3 cases.

The LoD of the test was assessed at $30 \%$ tumor content and 1 ng DNA input based on $100 \%$ agreement at interpretive results for all cases tested as well as $59 5 \%$ agreement at 4 of the 5 loci. While imprecision is noted at 1 of 5 loci, the risk of false negative test result is considered to be mitigated given the redundancy in calling algorithm requiring at least 2 of 5 loci being unstable to call MSI status, and inclusion of the following limitation statement:

“For tumor samples exhibiting instability at a single loci (1/5 alleles unstable) assess the tumor content and examine electropherograms and consider a retest by enriching tumor content for the sample or orthogonal testing to rule out a false negative test result.”

Table 10 –Percent Positivity of the Marker Instability Results of the MSI-H –Blended and Diluted Sample Set by Sample Group and DNA Amount at $1 0 \%$ Tumor   

<table><tr><td colspan="1" rowspan="1">Group</td><td colspan="1" rowspan="1">Final[DNA] (ng)</td><td colspan="1" rowspan="1">NR-21</td><td colspan="1" rowspan="1">BAT-26</td><td colspan="1" rowspan="1">BAT-25</td><td colspan="1" rowspan="1">NR-24</td><td colspan="1" rowspan="1">MONO-27</td><td colspan="1" rowspan="1">MSI-H</td></tr><tr><td colspan="1" rowspan="3">CRC-066</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">25%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">25%</td><td colspan="1" rowspan="1">25%</td><td colspan="1" rowspan="1">30%</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">45%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">85%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">CRC-079</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">30%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">95%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="3">CRC-084</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">90%</td><td colspan="1" rowspan="1">85%</td></tr><tr><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr></table>

Table 11 –Percent Positivity of the Marker Instability Results of the MSI-H –Blended and Diluted Sample Set by Sample Group and DNA Amount at $2 0 \%$ Tumor   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Final[DNA] (ng)</td><td rowspan=1 colspan=1>NR-21</td><td rowspan=1 colspan=1>BAT-26</td><td rowspan=1 colspan=1>BAT-25</td><td rowspan=1 colspan=1>NR-24</td><td rowspan=1 colspan=1>MONO-27</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=3 colspan=1>CRC-066</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>90%</td><td rowspan=1 colspan=1>85%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-079</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-084</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 12 –Percent Positivity of the Marker Instability Results of the MSI-H –Blended and Diluted Sample Set by Sample Group and DNA Amount at $3 0 \%$ Tumor   
Table 13 –Percent Positivity of the Marker Instability Results of the MSI-H –Blended and   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Final [DNA](ng)</td><td rowspan=1 colspan=1>NR-21</td><td rowspan=1 colspan=1>BAT-26</td><td rowspan=1 colspan=1>BAT-25</td><td rowspan=1 colspan=1>NR-24</td><td rowspan=1 colspan=1>MONO-27</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=3 colspan=1>CRC-066</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-079</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-084</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

Diluted Sample Set by Sample Group and DNA Amount at $5 0 \%$ Tumor   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Final [DNA](ng)</td><td rowspan=1 colspan=1>NR-21</td><td rowspan=1 colspan=1>BAT-26</td><td rowspan=1 colspan=1>BAT-25</td><td rowspan=1 colspan=1>NR-24</td><td rowspan=1 colspan=1>MONO-27</td><td rowspan=1 colspan=1>MSI-H</td></tr><tr><td rowspan=3 colspan=1>CRC-066</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-079</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>CRC-084</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr></table>

# ii. Limit of Blank

A Limit of Blank study was conducted to confirm a blank (an MSS sample in this study) did not produce positive MSI-H results. The study tested four known MSS samples across three amplification kit lots, two operators and 60 replicates of each MSS sample—a total of 1,440 test results.

There were two samples for which the initial test result was invalid. These samples were reinjected per protocol, and after reinjection were resolved. No reamplification testing was required for this study.

All tests (1,440/1,440; $100 \%$ ) resulted in MSS final interpretive results. The 1,440 test results represent 7,200 mononucleotide locus allele calls. For the mononucleotide loci, $9 9 . 9 9 \%$ (7,199/7,200) of the marker stability calls were “Stable”. There was a single instance of one locus, NR-21, being called unstable. A single unstable locus results in an MSS final interpretive result, and the one unstable locus did not affect the final test result. In conclusion, the OncoMateTM MSI Dx Analysis System provides MSS results that are highly reproducible and were not affected by lot or operator.

5. Linearity/assay reportable range: Not applicable

# 6. Traceability (controls, calibrators, or methods):

OncoMate MSI Dx is not traceable to any known standard. Controls and quality metrics are described in the device description section.

# 7. Stability

Studies were conducted to determine the stability of the Promega OncoMate™ MSI Dx Analysis System and Promega OncoMate™ 5C Matrix Standard.

Prior to execution of the real time or open vial studies, lots of the reagent kits were subjected to freeze-thaw cycles by cycling between the recommended storage temperature of $- 1 0 \sqsubset$ to $- 3 0 \bigcirc \mathrm { C }$ and ambient temperature until thawed. The OncoMate™ MSI Dx Analysis System amplification kit reagents were subjected to five freeze-thaw cycles and OncoMate™ 5C Matrix Standard was subjected to one freeze-thaw cycle.

Following freeze-thaw, the reagent lots were then transferred into shipping containers at $- 1 0 ^ { 0 } \mathrm { C }$ to $\cdot 3 0 ^ { 0 } \mathrm { C }$ on dry ice and shipped overnight by aircraft to a remote site. The samples were returned to Promega and stored in the shipping containers for a total of 5 days with sufficient dry ice to maintain $- 1 0 ^ { 0 } \mathrm { C }$ to $- 3 0 ^ { 0 } \mathrm { C }$ . The reagents were then transferred to a - $- 1 0 ^ { 0 }$ to $- 3 0 ^ { 0 } \mathrm { C }$ freezer until used for stability testing

i. Real Time Stability study (long term storage and shelf life stability):   
Real Time Stability at $- 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ (OncoMate™ MSI Analysis System   
amplification kit)   
The purpose of this study was to determine the real time stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents stored at $- 3 0 \bigcirc \mathrm { C }$ to $- 1 0 \bigcirc \bigcirc \mathrm { C }$ in real time. The purpose of these experiments was to confirm the reagents are stable after storage at ${ \bar { - 3 0 ^ { \bar { 0 } } } } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ .

DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l }$ and provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots in corrugate packaging and three reagents lots in paperboard packaging. Two lots in each package type were subjected to freeze-thaw and shipping, of the OncoMate™ MSI Dx Analysis System at time zero, followed by each lot being thawed and tested at 6, 13, 19, and 25 months. Two repeats were performed for each time point.

Across all time points a total of 7 results were called Invalid during testing. All 7 samples were reinjected, and 6 samples were resolved. The remaining sample was resolved upon reamplification.

The Real-Time Long-Term Stability of the OncoMate™ MSI Dx System amplification kit reagents was determined to be 24 months at frozen storage in both types of packaging based on the data collected to date. No difference was observed based on OncoMate™ MSI Dx Analysis System amplification kit lot.

Real Time Stability at $\cdot 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ followed by 4 months at $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ (OncoMate™ MSI Analysis System amplification kit) DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l }$ and provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots in corrugate packaging and three reagents lots in paperboard packaging. Two lots in each package type were subjected to freeze-thaw and shipping of the OncoMate™ MSI Dx Analysis System amplification kit at time zero and after 4 months storage at $- 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ . Another set of samples were stored at $\cdot 3 0 ^ { 0 } \mathrm { C }$ to $\mathrm { - 1 0 ^ { 0 } C }$ for 0, 2, 9, 15, and 21 months. Each storage period at ${ } _ { - 3 0 { } ^ { 0 } \mathrm { C } }$ to $\mathrm { - 1 0 ^ { 0 } C }$ was followed by storage for 4 months at the customer in-use temperature of $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ . Two repeats were performed for each time point.

Across all time points a total of 10 results were called Invalid during testing. All 10 samples were reinjected, and 6 samples were resolved. The remaining 4 samples were resolved upon reamplification.

The Real Time Shelf Life Stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents was determined to be frozen storage for 20 months followed by refrigerated storage for four (4) months, for a total of 24 months based on the data collected to date. No difference was observed based on OncoMate™ MSI Dx Analysis System amplification kit lot.

Real Time Stability at $\cdot 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ (OncoMate™ 5C Matrix Standard) The purpose of this study was to determine the real time stability of the OncoMate™ 5C Matrix Standard stored at $- 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ in real time.

Samples were tested by one operator and instrument using three reagent lots, two lots subjected to freeze-thaw and shipping, at time zero and after 4 months storage at $- 3 0 ^ { 0 }$ C to $\mathrm { - 1 0 ^ { 0 } C }$ . Samples were tested after storage at ${ } _ { - 3 0 { } ^ { 0 } \mathrm { C } }$ to ${ } _ { - 1 0 } { } ^ { 0 } \mathrm { C }$ for 0, 6, 13, 19, and 25 months. Two repeats were performed for each time point.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The Shelf Life Stability of the OncoMate™ 5C Matrix Standard was determined to 24 months for frozen storage based on the data collected to date. No difference was observed based on OncoMate 5C Matrix Standard lot.

Real Time Stability at $\cdot 3 0 ^ { 0 } \mathrm { C }$ to $- 1 0 ^ { 0 } \mathrm { C }$ followed by 4 months at $+ 2 ^ { 0 } \mathrm { C }$ to $1 0 ^ { 0 } \mathrm { C }$ (OncoMate™ 5C Matrix Standard)

Another set of samples were stored at $\boldsymbol - 3 0 ^ { 0 } \boldsymbol { \mathrm { C } }$ to $\mathrm { - 1 0 ^ { 0 } C }$ for 2, 9, 15, and 21 months. Each storage period at ${ } _ { - 3 0 { } ^ { 0 } \mathrm { C } }$ to $\mathrm { - 1 0 ^ { 0 } C }$ was followed by storage for 4 months at the customer in-use temperature of $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ . Two repeats were performed for each

time point.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The Real-Time Long-Term Stability of the OncoMate™ 5C Matrix Standard was determined to be frozen storage for 20 months followed by storage at $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ for four (4) months, for a total of 24 months based on the data collected to date. No difference was observed based on OncoMate™ 5C Matrix Standard lot.

# ii. Open vial stability Studies:

The purpose of this study was to determine the open vial shelf life stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents at $^ { + 2 }$ to $1 0 ^ { \circ } \mathrm { C }$ in real time.

DNA was extracted from four MSI-High and four MSS CRC tumor samples with matched normal samples, quantitated, adjusted to $0 . 5 \mathrm { n g / \mu l } .$ , and $1 0 \mu \mathrm { l }$ aliquots were provided to the study operator. The study operator did not know the MSI status of the samples. Samples were tested by one operator and instrument using three reagent lots, two of the lots were subjected to freeze-thaw and shipment. Tests were conducted at time zero, followed by each lot being tested at four additional time points of 1, 2, 3, and 4 months. Two repeats were performed for each time point.

Across all time points a total of 17 results were initially called Invalid during testing. All 17 samples were reinjected, and 16 samples were resolved. The single remaining Invalid result was resolved upon reamplification.

The open vial shelf life stability of the OncoMate™ MSI Dx Analysis System amplification kit reagents was determined to be four (4) months, supporting the intended claim of 3 months stability at $^ { + 2 }$ to $1 0 ^ { 0 } \mathrm { C }$ .

Open Vial Reagent Stability (OncoMate™ 5C Matrix Standard) The purpose of this study was to determine the open vial shelf life stability of the OncoMate™ 5C Matrix Standard at $+ 2$ to $1 0 ^ { \circ } \mathrm { C }$ in real time.

One OncoMate™ 5C Matrix Standard reagent lot, subjected to freeze-thaw and shipping, was tested at time zero, followed by testing at four additional time points of 1, 2, 3, and 4 months. Two repeats were performed for each time point.

The open vial short-term stability of diluted OncoMate™ 5C Matrix Standard was performed at time zero and 7 days using three lots of reagents. Two of the lots were subjected to freeze/thaw and shipping. The open vial short-term stability study was performed 7 days after the Matrix Standard was diluted to support a 6 day stability claim for the diluted Matrix Standard.

The OncoMate™ 5C Matrix Standard was used to perform spectral calibration on the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer as described in the Technical Manual (TM542). The result of the spectral calibration is a multicomponent matrix, which is applied during sample detection to compensate for spectral overlap among the dyes and separate the raw fluorescent signals into individual dye signals. The Quality Value (Q value) for each capillary was measured by the instrument software. A Q score of $2 0 . 9 5$ is indicative of a successful spectral calibration.

The open vial shelf life stability of the OncoMate™ 5C Matrix Standard was determined to be four (4) months, supporting the intended claim of 3-month stability at $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ .

The open vial stability of diluted Matrix was determined to be seven (7) days, supporting the intended claim of six (6) days at $+ 2$ to $1 0 ^ { 0 } \mathrm { C }$ . No difference was observed based on lot, regardless of whether the lot was subjected to freeze-thaw and shipping.

# 8. Expected values:

The OncoMate MSI Dx Analysis System determines microsatellite instability status based on results generated for five mononucleotide repeat markers (BAT-25, BAT-26, NR-21, NR-24 and MONO-27). A tumor sample is interpreted as MSI-H when two or more markers are ‘Unstable’. A tumor sample is interpreted as MSS when fewer than two markers are interpreted as ‘Unstable’. A sample may be interpreted as ‘No Call’ or ‘Invalid’ due to specific QC failures. During the Method Comparison study (section C below), the most common observation for MSI-H and MSS samples was for all or none of the markers to be unstable (147/154 cases, $9 5 \%$ ), which is consistent with published literature.

# 9. Analytical specificity:

Primer pairs for the seven OncoMate MSI Dx Analysis System markers were checked for target specificity using the publicly available Primer BLAST search tool on the US National Center for Biotechnology Information website

(https://ncbi.nlm.nih.gov/tools/primer-blast/, accessed 01/14/2020; Ye et al., 2012). The primers share $100 \%$ identity with their intended targets. Analysis for off target homology showed that 7 of the 14 primers in the test system exhibited off target homology with $2 8 0 \%$ identity. 3 of the 7 off-target templates had lower annealing temperature than on target primers and 4 other off targets templates showed high homology to antisense template. For these 4 off target templates the sense primer would be unlabeled and result in products that are invisible to the detection system. Therefore, the identified off target homologies did not interfere with assay interpretation.

The in-silico analysis provided supports target specificity for the proposed primer pairs in the test.

# 10. Interference:

A study was performed to establish the potential influence of interfering substances on the performance of the OncoMate MSI Dx Analysis System, specifically chaotropic salts, alcohol, proteinase K treatment time, necrotic tissue, hemoglobin, triglycerides and mucin. DNA was extracted from sample curls $\mathrm { 0 . 1 { - } } 2 \mathrm { m m } 3$ tissue). DNA extraction was performed for each sample at each condition tested using the Maxwell CSC Instrument and Maxwell $\textsuperscript { \textregistered }$ CSC DNA FFPE Kit.

In the first series of experiments, lysates from four tumor and matched normal samples were spiked with hemoglobin $( 2 \mathrm { m g / m l }$ final concentration), triglycerides $3 7 \mathrm { m M }$ final concentration) or mucin $\mathrm { { 1 m g / m l } }$ final concentration) prior to DNA extraction. Following analysis with the OncoMate MSI Dx Analysis System, all samples yielded the expected result.

Four tumor and matched normal samples (two replicates per sample for a total of 16 extractions) were incubated at $5 6 \mathrm { { } ^ { \circ } C }$ in the presence of proteinase K for 20 minutes, 30 minutes (recommended condition) or 40 minutes prior to purification. All samples yielded the expected result.

Twelve tumor samples with necrotic tissue ranging from $0 - 7 5 \%$ and matched normal samples were tested. All samples yielded the expected result.

To evaluate the impact of potential carry-over alcohol or guanidine salts from the DNA purification process, aliquots of the extracted DNA from tumor and matched normal samples with varying amounts of tissue necrosis $( 0 - 7 5 \% )$ were spiked with ethanol $5 \%$ final concentration), guanidine hydrochloride ( $5 0 \mu \mathrm { M }$ final concentration) or water prior to amplification and analysis. All samples yielded the expected result.

There were 13 invalid results initially obtained across the interfering substances testing. All 13 samples were reinjected, resulting in five samples being resolved. The remaining eight samples were resolved after reamplification.

The OncoMate MSI Dx Analysis System showed no statistically significant impact on assay performance by the interfering substances, specifically chaotropic salts $( 5 0 \mu \mathrm { M }$ guanidine hydrochloride), ethanol $( 5 \% )$ , necrotic tissue $( 0 - 7 5 \% )$ , hemoglobin $( 2 \mathrm { m g / m l } )$ , triglycerides $( 3 7 \mathrm { m M } )$ , and mucin $\mathrm { ( 1 m g / m l ) }$ or proteinase K digestion time [20, 30 (standard) or 40 minutes] tested in this study (Table 12).

Table 14: OncoMate™ MSI Dx Analysis System Sample Treatment by Interpretative Result   

<table><tr><td colspan="1" rowspan="3"></td><td colspan="4" rowspan="1">Interpretive Result</td><td colspan="2" rowspan="2">Total</td></tr><tr><td colspan="2" rowspan="1">MSI-H</td><td colspan="2" rowspan="1">MSS</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">Sample Treatment</td><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">50.0</td><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">50.0</td><td colspan="1" rowspan="2">8</td><td colspan="1" rowspan="2">100</td></tr><tr><td colspan="1" rowspan="1">20min at 56°C</td></tr><tr><td colspan="1" rowspan="1">30min at 56°C</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">40min at 56°C</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Ethanol spike</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Guanidine spike</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Hemoglobin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Necrosis</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">58.3</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">41.7</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Triglycerides</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Water spike</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">51.7</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">48.3</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">100</td></tr></table>

1 All samples returned the expected result (MSS or MSI-H) in interfering substances experiments.

# 11. Cross Contamination

The sample-to-sample cross-contamination in the OncoMate™ MSI Dx Analysis System was evaluated using extracted DNA from an MSI-H sample and an MSS sample, including both the tumor and its matched normal sample. The samples and reagent blanks were interspersed in a grid design across 96-well plates. In this plate layout, amplification, capillary electrophoresis and analysis were performed each day for a total of 10 days. Expected results included MSI-H, MSS and No Call test results. The No Call test result was expected for the reagent blanks. The results were not averaged, and any observed carryover was reported.

There were a total of three invalid results initially obtained in the study. After reinjection, all three sample results wereresolved. No reamplification was performed for this study.

There was $100 \%$ concordance between the interpretative result and the expected results for the 470 samples analyzed, and no interference with data interpretation was observed. The study concluded the OncoMate MSI Dx Analysis System was not susceptible to sampleto-sample cross-contamination.

# 12. Assay cut-off:

See normal range study validating the cut off.

# B. Comparison studies:

See clinical performance section below.

# C. Clinical Performance:

The primary objective of the method comparison study was to evaluate the accuracy of the OncoMate MSI Dx Analysis System in identifying microsatellite instability in the clinical setting. A method comparison was performed between the OncoMate MSI Dx Analysis System and the predicate device which is an immunohistochemistry test for the presence or absence of DNA mismatch repair proteins. Accuracy of the OncoMate MSI Dx Analysis System result was also assessed by comparison to a validated next generation sequencing test for germline mutations to the DNA mismatch repair genes (NGS MMR genes) to confirm identification of Lynch syndrome patients.

The study was performed by testing 130 sequentially selected colorectal cancer patient samples that were enriched with a second set of 24 suspected Lynch syndrome samples, for a total of 154 cases. Sample curls, generated from FFPE tissue blocks, were provided to an external laboratory to perform immunohistochemistry. Extracted DNA aliquots of 154 matched CRC DNA samples of unknown MSI status were randomized then analyzed by the OncoMate MSI Dx Analysis System and by the analytically validated NGS test for MMR genes.

Immunohistochemistry was performed on all 154 tumor samples to determine protein expression of the MHL1, MSH2, MSH6, PMS2 and BRAF genes using the predicate on the provided sample curls, per the predicate instructions for use. BRAF testing was only performed if the sample exhibited a loss of one of the mismatch repair genes.

DNA from the 154 samples were provided to a reference laboratory for NGS with a validated test for MMR genes. These samples underwent DNA sequencing to determine the presence or absence of pathogenic mutations of the mismatch repair genes (MLH1, MSH2, MSH6 and PMS2) and BRAF exon 15. Upon germline testing, 18 Lynch syndrome cases were confirmed. Only pathogenic or likely pathogenic mutations listed in the ClinVar database were accepted as confirmed Lynch syndrome cases.

A total of two samples yielded invalid results during initial testing. Both samples were resolved upon reinjection.

# 1. Method Comparison: OncoMateTM MSI Dx Analysis System vs. IHC Results

The comparison results between the OncoMate MSI Dx Analysis System and the VENTANA IHC MMR Panel for the 154 samples are listed in Table 15. A total of 106 samples were scored as MSS using the OncoMate MSI Dx Analysis System and MMR Intact for all four MMR proteins using the predicate. Forty-one (41) samples exhibited a loss of IHC staining for at least one of the four MMR proteins (dMMR). Of these 45 samples, 44 samples were scored as MSI-H by the OncoMate MSI Dx Analysis System. Three samples were scored as MSI-H by the OncoMateTM MSI Dx Analysis System but were scored as MMR Intact by IHC staining for all four MMR proteins. The data are summarized in Table 16 and Table 17.

The Positive Percent Agreement (PPA) was $9 7 . 8 \%$ b $9 5 \%$ CI: $8 8 . 4 \% - 9 9 . 6 \%$ ) and the Negative Percent Agreement (NPA) was $9 7 . 2 \%$ $9 5 \%$ CI: $9 2 . 2 \% - 9 9 . 1 \% )$ . . The OncoMate MSI Dx Analysis System effectively identified tumors with MMR deficiency and shows agreement with the VENTANA MMR IHC panel.

Table 15: Comparison of OncoMate MSI Dx Analysis System Interpretative Results vs. predicate (All Samples)

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>Ventana MMR IHC results</td></tr><tr><td rowspan=1 colspan=1>MMR Loss</td><td rowspan=1 colspan=1>MMR Intact</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>154</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>44/45</td><td rowspan=1 colspan=2>97.8</td><td rowspan=1 colspan=2>88.4  99.6</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/109</td><td rowspan=1 colspan=2>97.2</td><td rowspan=1 colspan=2>92.2 99.1</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>150/154</td><td rowspan=1 colspan=2>97.4</td><td rowspan=1 colspan=2>93.5 - 99.0</td></tr></table>

Table 16: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. predicate (Sequential Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>Ventana MMR IHC results</td></tr><tr><td rowspan=1 colspan=1>MMR Loss</td><td rowspan=1 colspan=1>MMR Intact</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=2>95.8</td><td rowspan=1 colspan=2>79.8  99.3</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/106</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=2>96.5 - 100.0</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>129/130</td><td rowspan=1 colspan=2>99.2</td><td rowspan=1 colspan=2>95.8  99.9</td></tr></table>

Table 17: Comparison and Agreement Analysis of OncoMate™ MSI Dx Analysis System Interpretative Results vs. predicate (Enrichment Cohort)   

<table><tr><td colspan="1" rowspan="2">OncoMate MSI Dx</td><td colspan="5" rowspan="1">Ventana MMR IHC results</td></tr><tr><td colspan="1" rowspan="1">MMR Loss</td><td colspan="1" rowspan="1">MMR Intact</td><td colspan="2" rowspan="1">Invalid</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">MSI-H</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">MSS</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">3</td><td colspan="2" rowspan="1">0</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="6" rowspan="1">Agreement</td></tr><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">n/N</td><td colspan="2" rowspan="1">%</td><td colspan="2" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">21/21</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">84.5 - 100.0</td></tr><tr><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 56.1</td></tr><tr><td colspan="1" rowspan="1">OPA</td><td colspan="1" rowspan="1">21/24</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">69.0  95.7</td></tr></table>

# 2. Accuracy assessed by concordance to NGS Mismatch Repair Gene Mutations Results

A total of 18 samples tested positive for Lynch syndrome, based on detection of a pathogenic or likely pathogenic mutation in one of the mismatch repair genes (MLH1, MSH2, MSH6 or PMS2) and no mutations in BRAF exon 15. The data are summarized in Table 18, 19 and 20. Seventeen of the samples (17/18) tested MSI-H with the OncoMate MSI Dx Analysis System. One of the samples (1/18) tested MSS with the OncoMate MSI Dx Analysis System and exhibited no loss of MMR protein by IHC. This sample is referenced as having a likely pathogenic mutation in the PMS2 gene on the ClinVar database. This single nucleotide polymorphism (rs267608153) results in a $\mathrm { c } . 9 0 3 \mathrm { G } { > } \mathrm { T }$ variant that likely results in a splicing defect. All 16 of the enrichment cohort samples tested MSI-H with the OncoMateTM MSI Dx Analysis System.

The Positive Percent Agreement (PPA) was $9 4 . 4 \%$ and the Negative Percent Agreement (NPA) was $7 7 . 9 \%$ between the two methods for all samples. The PPA was $100 \%$ between the two methods for the enrichment cohort. The NPA is less informative than the PPA in a comparison of somatic microsatellite instability to germline mutations in MMR genes, since cases negative for germline, pathogenic Lynch syndrome mutations include MSI-H cases with sporadic, somatic causes for dMMR as well as MSS cases. Somatic mechanisms such as epigenetic silencing and biallelic somatic mutation can lead to dMMR and an MSI-H phenotype without a germline MMR gene mutation.

Table 18: Comparison of OncoMate MSI Dx Analysis System Interpretative Results vs. NGS Results for MMR Genes, All Samples   
Table 19: Comparison of OncoMate MSI Dx Analysis System Interpretative Results   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>154</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>17/18</td><td rowspan=1 colspan=2>94.4</td><td rowspan=1 colspan=2>74.2  99.0</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/136</td><td rowspan=1 colspan=2>77.9</td><td rowspan=1 colspan=2>70.3  84.1</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>123/154</td><td rowspan=1 colspan=2>79.9</td><td rowspan=1 colspan=2>72.8  85.4</td></tr></table>

vs. NGS Results for MMR Genes (Sequential Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>1/2</td><td rowspan=1 colspan=2>50.0</td><td rowspan=1 colspan=2>9.5 - 90.5</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>106/128</td><td rowspan=1 colspan=2>82.8</td><td rowspan=1 colspan=2>75.3  88.4</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>107/130</td><td rowspan=1 colspan=2>82.3</td><td rowspan=1 colspan=2>74.8  87.9</td></tr></table>

Table 20: Comparison of OncoMate MSI Dx Analysis System Interpretative Results vs. NGS Results for MMR Genes (Enrichment Cohort)   

<table><tr><td rowspan=2 colspan=1>OncoMate MSI Dx</td><td rowspan=1 colspan=5>DNA Sequencing Results</td></tr><tr><td rowspan=1 colspan=1>Pathogenic Mutation</td><td rowspan=1 colspan=1>No Pathogenic Mutation</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MSI-H</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>MSS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=6>Agreement</td></tr><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>n/N</td><td rowspan=1 colspan=2>%</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=2>100.0</td><td rowspan=1 colspan=2>80.6 - 100</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=2>0.0</td><td rowspan=1 colspan=2>0.0 - 32.4</td></tr><tr><td rowspan=1 colspan=1>OPA</td><td rowspan=1 colspan=1>16/24</td><td rowspan=1 colspan=2>66.7</td><td rowspan=1 colspan=2>46.7  82.0</td></tr></table>

# N. Instrument Name

Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer.

# O. System Descriptions:

# 1. Modes of Operation:

The OncoMate MSI test system is a PCR based technology to assess fragment length of target microsatellite site to assess Microsatellite instability as a measure of mismatch repair in colorectal cancer tissue.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes x or No

3. Level of Concern: Moderate

# 4. Specimen Handling:

Refer to Device Description section above.

# 5. Calibration and Quality Controls:

During capillary electrophoresis, dye-labeled OncoMate MSI Dx Analysis System amplification products are separated and detected using the Applied Biosystems® $3 5 0 0 \mathrm { D x }$ Genetic Analyzer. Prior to analysis, the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer is calibrated with matrix standards so that the fluorescent signals resulting from the set of specific dyes used in the assay can be distinguished. The OncoMate 5C Matrix Standard consists of DNA fragments labeled with five different fluorescent dyes (fluorescein, JOE, TMR-ET, CXR-ET and WEN) in one tube. The calibration is performed using the ‘OncoMate_MSI’ dye set, which is installed on the Applied Biosystems $3 5 0 0 \mathrm { D x }$ Genetic Analyzer using the OncoMate MSI Dx Assay Installer. Once generated, the spectral calibration file is applied automatically during sample detection to account for the spectral overlap among the dyes and to separate the raw fluorescent signals into individual dye signals.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered in the “Performance Characteristics” Section Above:

The OncoMate™ MSI Dx Analysis System was evaluated with three different thermal cycler models from three different manufacturers. The 2800M Control DNA was used, and replicates were treated as MSS samples for analysis using the OncoMate MSI Dx Interpretive Software. Samples were amplified in duplicate with the OncoMate™ MSI Dx Analysis System amplification kit using 1ng, 2ng or 4ng of DNA on each of three different thermal cycler models. The thermal cyclers all fall within the following required performance specifications:

• Maximum Block Ramp Rate: $3 . 9 ^ { \circ } \mathrm { C } /$ second to $5 ^ { \circ } \mathrm { C } /$ second • Temperature Accuracy: $\pm 0 . 2 5 ^ { \circ } \mathrm { C }$ (at ${ \geq } 9 0 ^ { \circ } \mathrm { C }$ ) • Temperature Uniformity: ${ \mathrm { < } } 0 . 5 { ^ { \circ } } \mathrm { C }$ (at ${ \geq } 9 0 ^ { \circ } \mathrm { C }$ ) Heated lid capable of reaching $1 0 3 { \mathrm { - } } 1 0 5 ^ { \circ } \mathrm { C }$

There was $100 \%$ agreement (36/36) between the expected and observed MSS call for all samples. There were no artifacts observed that interfered with the system’s ability to provide the expected interpretive result when using different thermal cyclers.

# Q. Proposed Labeling:

The labeling is sufficient, and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable, and the special controls for this device type.

# R. Patient Perspectives

This submission did not include specific information on patient perspectives for this device.

# S. Conclusion:

The submitted information in this premarket notification is complete and supports a substantially equivalence decision.