# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION MEMORANDUM ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K201902 B. Purpose for Submission:

The replacement of microarray C-Scanner (Part Number G2505) with D-Scanner (Part Number G5761AA) and the update of XPrint analysis software from version 2.24 to version 3.2.0.

C. Measurand: 70 gene expression profile

# D. Type of Test:

Expression microarray

Test performed in Agendia’s two central laboratories: Amsterdam, Netherlands; and Irvine, California, USA

# E. Applicant:

Agendia NV

F. Proprietary and Established Names: MammaPrint $\textsuperscript { \textregistered }$ FFPE

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.6040, Gene expression profiling test system for breast cancer prognosis

2. Classification: Class II

3. Product code:

NYI, Classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer

4. Panel: Immunology (82)

# H. Intended Use:

1. Intended use(s):

MammaPrint $\textsuperscript { \textregistered }$ FFPE is a qualitative in vitro diagnostic test, performed in a central laboratory, using the gene expression profile obtained from formalin-fixed paraffin embedded (FFPE) breast cancer tissue samples to assess a patient’s risk for distant metastasis within 5 years.

The test is performed for breast cancer patients, with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The MammaPrint $\textsuperscript { \textregistered }$ FFPE result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

2. Indication(s) for use: Same as the above intended use

3. Special conditions for use statement(s):

For prescription use only

MammaPrint $\textsuperscript { \textregistered }$ FFPE is not indicated as a standalone test to determine the outcome of disease, nor to suggest or infer an individual patient’s likely response to therapy. Results should be taken in the context of other relevant clinico-pathological factors and standard practice of medicine.

4. Special instrument requirements:

Agilent 2100 Bioanalyzer: Serial numbers DE54700497, DE24802382, DE72901757, and DE72902383.

Agilent DNA microarray D-scanner (Part Number G5761AA): Serial numbers SG18309119 and SG18449122.

Note: The scanners and Bio-analyzers are components of this assay and are cleared only for this assay and not for any other application. In addition, clearance is only limited to the bioanalyzers and scanners with the serial numbers as specified above.

# I. Device Description:

The MammaPrint $\textsuperscript { \textregistered }$ FFPE test is performed at Agendia’s two central Laboratories, one located in Netherland and the other one in California, USA. The test is a microarray based gene expression analysis of RNA extracted from FFPE breast tumor tissue. The test is a custom-designed array chip manufactured by Agilent Technologies using the Agilent oligonucleotide microarray platform which assesses the mRNA expression of the 70 genes printed in nine-fold.

The MammaPrint $\textsuperscript { \textregistered }$ FFPE analysis is designed to determine the expression of specific genes in a tissue sample. The result is an expression profile, or “fingerprint”, of the sample. Using this expression profile, the MammaPrint $\textsuperscript { \textregistered }$ FFPE Index is calculated and the molecular prognosis profile of the sample is determined (Low Risk, High Risk).

# J. Substantial Equivalence Information:

1. Predicate device name(s): MammaPrint $\textsuperscript { \textregistered }$ FFPE

2. Predicate 510(k) number(s): K141142

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K201902</td><td colspan="1" rowspan="1">K141142</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">MammaPrint® FFPE</td><td colspan="1" rowspan="1">MammaPrint® FFPE</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Similarities</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">MammaPrint® FFPE isa qualitative in vitrodiagnostic test,performed in a centrallaboratory, using thegene expression profileobtained from formalin-fixed paraffinembedded (FFPE)breast cancer tissuesamples to assess apatient's risk for distantmetastasis within 5years.The test is performed</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">for breast cancerpatients, with Stage I orStage II disease, withtumor size ≤ 5.0 cm andlymph node negative.The MammaPrint FFPEresult is indicated foruse by physicians as aprognostic marker only,along with otherclinicopathologicalfactors.</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">Common Name</td><td colspan="1" rowspan="1">Multivariate device forcancer prognosis</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.6040</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">NYI</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalCharacteristics,Performance, and Materials</td><td colspan="1" rowspan="1">The MammaPrintservice is a microarray-based gene expressionanalysis of a tumor. Theanalysis is based onscanning theMammaPrintmicroarray and dataacquisition (featureextraction), calculationand determination ofthe risk of recurrence inbreast cancer patients.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Classification and Cut-off</td><td colspan="1" rowspan="1">MammaPrint Indexprovides a High Risk orLow Risk result.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Clinicalperformance/Indication foruse</td><td colspan="1" rowspan="1">MammaPrint is a test toassess a patients' riskfor distant metastasis.The test is performedfor breast cancerpatients with Stage Iand II disease, with atumor size ≤5.0cm andlymph node negative.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Pre-analytical samplepreparation</td><td colspan="1" rowspan="1">Isolation of RNA fromformalin-fixed paraffinembedded (FFPE)tumor tissue sections,DNase treatment of</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">isolated RNA,amplification andpurification DNasetreated RNA resultingin cDNA, labeling andpurification ofamplified cDNA,hybridization of thecDNA to the diagnosticmicroarray.</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1">MammaPrint MicroarrayDensity</td><td colspan="1" rowspan="1">Analysis is performedusing Agendia-designedHigh Density diagnosticMicroarraysmanufactured underGMP by AgilentTechnologies. 70signature genes areprinted in nine-fold.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">XPrint analysis software forMammaPrint indexcalculation</td><td colspan="1" rowspan="1">Calculation ofMammaPrint indexbased on geneintensities of probesprinted in nine-fold.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Reporting</td><td colspan="1" rowspan="1">Agendia reportsMammaPrint expectedvalues and clinicalperformance asdescribed in Section Aof this DecisionSummary</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">General DeviceCharacteristic Differences</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Agilent scanner type</td><td colspan="1" rowspan="1">SureScan Dxmicroarray scanner, part#G5761AA</td><td colspan="1" rowspan="1">Microarray scanner,part #G2505</td></tr><tr><td colspan="1" rowspan="1">XPrint analysis software</td><td colspan="1" rowspan="1">XPrint version 3.2.0This updated versionwas developed to acceptdata generated by theSureScan Dx scanner,part number G5761AA.</td><td colspan="1" rowspan="1">XPrint version 2.24</td></tr><tr><td colspan="1" rowspan="1">Labeling - Results Reports</td><td colspan="1" rowspan="1">In addition to the lowrisk/high risk qualitativedevice output, the report</td><td colspan="1" rowspan="1">The report includes lowrisk/high risk qualitativedevice output with an</td></tr><tr><td></td><td>includes an arrow with MammaPrint Index score indicating "distance" of the result relative to the cut-off. The numerical result illustrates distance from the borderline region but should not be interpreted as a quantitative risk metric.</td><td>arrow indicating "distance" of the result relative to the cut-off.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. CLSI EP5-A2: Evaluation of Precision Performance of Quantitative Measurement Methods; Clinical and Laboratory Standards Institute; 2004.

2. Guidance for Industry and FDA Staff, Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, 2007.

3. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis, 2007

# L. Test Principle:

The MammaPrint $\textsuperscript { \textregistered }$ FFPE is a microarray based gene expression analysis of breast tumor tissue. The analysis is based on several processes: isolation of RNA from FFPE breast cancer tissue sections; elimination of gDNA, reverse transcription of RNA resulting in cDNA; amplification and labeling of the cDNA; hybridization of the amplified and labeled cDNA to the diagnostic microarray; washing and scanning the diagnostic microarray and data acquisition (feature extraction); calculation and determination of the risk of recurrence.

Briefly, the amplified and labeled cDNA is hybridized to slides at $6 0 ^ { 0 } \mathrm { C }$ for 17 hours in a rotation oven. By hybridization only complementary cDNA will bind to a 60-mer oligo on the array. For scanning the MammaPrint $\textsuperscript { \textregistered }$ FFPE microarray, an Agilent microarray DScanner (Part Number G5761AA) is used. The Agilent DNA microarray scanner is a 48-slide scanning system that can read $1 " \mathbf { x } 3 "$ glass slide microarrays. The result after scanning is a scan file (multi-page TIFF). This TIFF contains two pages, one page for each dye used. These are used by the feature extraction software.

Agilent Feature Extraction Software opens the multipage TIFF and combines those into one image which shows a pattern of differently colored spots. The Feature Extraction Software analyses the scan file (TIFF) by determining the intensities of the individual features, subtracting background signal, perform normalization, and calculate ratios, errors and pvalues for each spot. The feature extraction software uses the MammaPrint $\textsuperscript { \textregistered }$ FFPE microarray chip design file as a template in order to identify control features, normalization features and reporter features. The fluorescent intensity of the features is a measure for the activity of that particular gene.

Data analysis is performed according to a specific MammaPrint $\textsuperscript { \textregistered }$ FFPE algorithm (MammaPrint $\textsuperscript { \textregistered }$ Index, or MPI). The algorithm calculates the correlation of the sample expression profile to a template (the mean expression profile of 44 tumors with a known good clinical outcome) and determines the molecular profile of the sample. This algorithm is designed and programmed by Agendia and compiled into a standalone software program, XPrint Analysis Software. The X-Print Analysis Software version 3.2.0 loads a data file (CSV) which is created by the laboratory technician by extracting specific information from the laboratory database. The X-Print Analysis Software version 3.2.0 reads the CSV file, opens the Feature Extraction Software data files (TXT), performs quality control checks, determines the sample expression profile, calculates the correlation of sample profile to the "Low Risk" template profile on a scale of -1.000 to $+ 1 . 0 0 0$ (MammaPrint $\textsuperscript { \textregistered }$ FFPE reportable range), compares the calculated correlation to a pre-defined cut-off value and determines the samples prognostic profile (i.e., Low Risk, High Risk, Low Risk Borderline, or High Risk Borderline).

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

Refer to the Analytical Performance section in Decision Summary of K141142 for the complete performance characteristics summary.

# i. Precision/Reproducibility:

The precision assessment of both SureScan Dx D-scanners versus the predicate C-scanners was performed using four samples. These samples are routinely used as diagnostic controls for MammaPrint and included two low risk samples with one being around device cut-off and two high risk samples. The samples had a known MammaPrint result and covered all MammaPrint categorical results (i.e. high risk, low risk, border line). In order to assess the precision of both SureScan Dx D-scanners versus the predicate C-scanners, the variability of repeated measurements between the C- and D-scanners were compared using the F-test.

The results from both the SureScan Dx D-scanners and the predicate C-scanners are plotted over time (Figure 1).

![](images/3f7c0253f6713534f04d0505e70fc60900e6ed79396595f4bfbb05763a1ad00d.jpg)  
Figure 1: Over time MammaPrint measurements obtained for the four control samples collected on both SureScan $D x D .$ -scanners and the predicate $C \mathrm { \cdot }$ -scanners.

Table 1 shows the control sample measurements for each of the two SureScan Dx Dscanners, their related mean and standard deviation as well as the mean and standard deviation based on the predicate C-scanner measurement. The p-values for all control samples were above the significance level of 0.05, which shows that there is no statistically significant difference in precision between the $\mathrm { C } \mathrm { - }$ and D-scanners (Table 2).

Table 1: Mean and standard deviation for control samples per SureScan Dx (D-scanner) scanner compared to the C-scanners.   

<table><tr><td rowspan=2 colspan=1>SureScan Dx</td><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Type</td><td rowspan=2 colspan=1>n</td><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=2>Standard Deviation</td></tr><tr><td rowspan=1 colspan=1>C-scanner</td><td rowspan=1 colspan=1>D-scanner</td><td rowspan=1 colspan=1>C-scanner</td><td rowspan=1 colspan=1>D-scanner</td></tr><tr><td rowspan=4 colspan=1>SG18309119</td><td rowspan=1 colspan=1>PLEP3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0.457</td><td rowspan=1 colspan=1>0.458</td><td rowspan=1 colspan=1>0.040</td><td rowspan=1 colspan=1>0.039</td></tr><tr><td rowspan=1 colspan=1>PHHE2</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>-0.631</td><td rowspan=1 colspan=1>-0.632</td><td rowspan=1 colspan=1>0.040</td><td rowspan=1 colspan=1>0.039</td></tr><tr><td rowspan=1 colspan=1>PHTR2</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>-0.644</td><td rowspan=1 colspan=1>-0.642</td><td rowspan=1 colspan=1>0.059</td><td rowspan=1 colspan=1>0.059</td></tr><tr><td rowspan=1 colspan=1>PBCL2</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0.007</td><td rowspan=1 colspan=1>0.004</td><td rowspan=1 colspan=1>0.036</td><td rowspan=1 colspan=1>0.037</td></tr><tr><td rowspan=4 colspan=1>SG18449122</td><td rowspan=1 colspan=1>PLEP3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0.444</td><td rowspan=1 colspan=1>0.446</td><td rowspan=1 colspan=1>0.050</td><td rowspan=1 colspan=1>0.048</td></tr><tr><td rowspan=1 colspan=1>PHHE2</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>-0.639</td><td rowspan=1 colspan=1>-0.637</td><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>0.052</td></tr><tr><td rowspan=1 colspan=1>PHTR2</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>-0.618</td><td rowspan=1 colspan=1>-0.618</td><td rowspan=1 colspan=1>0.027</td><td rowspan=1 colspan=1>0.030</td></tr><tr><td rowspan=1 colspan=1>PBCL2</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0.025</td><td rowspan=1 colspan=1>0.026</td><td rowspan=1 colspan=1>0.048</td><td rowspan=1 colspan=1>0.048</td></tr></table>

Table 2: Assessment of control sample variances from SureScan $D x$ scanners versus the predicate $C \mathrm { . }$ -scanners.   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>PLEP3</td><td rowspan=1 colspan=1>PHTR2</td><td rowspan=1 colspan=1>PHHE2</td><td rowspan=1 colspan=1>PBCL2</td></tr><tr><td rowspan=1 colspan=1>SG18309119</td><td rowspan=1 colspan=1>0.916</td><td rowspan=1 colspan=1>0.992</td><td rowspan=1 colspan=1>0.965</td><td rowspan=1 colspan=1>0.954</td></tr><tr><td rowspan=1 colspan=1>SG18449122</td><td rowspan=1 colspan=1>0.914</td><td rowspan=1 colspan=1>0.731</td><td rowspan=1 colspan=1>0.965</td><td rowspan=1 colspan=1>0.943</td></tr><tr><td rowspan=1 colspan=1>Combined SureScanDx scanners</td><td rowspan=1 colspan=1>0.864</td><td rowspan=1 colspan=1>0.952</td><td rowspan=1 colspan=1>0.993</td><td rowspan=1 colspan=1>0.988</td></tr></table>

ii. Linearity:

Not applicable

iii. Analytical Specificity/Interference:

Same as previous submission.

iv. Assay Reportable Range:

Same as previous submission.

v. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Same as previous submission.

vi. Detection Limit:

Same as previous submission.

vii. Assay Cut-Off:

Same as previous submission.

viii. Accuracy (Instrument):

Same as previous submission.

ix. Carry-Over:

Same as previous submission.

# 2. Comparison studies:

i. Method Comparison with Predicate Device:

In the method comparison study, a set of 92 8-pack arrays was analyzed. These samples are regularly used by Agendia as part of diagnostics on the FDA-cleared C-scanners

(US45103019 and US811R3213). Subsequently, these were scanned a second time on the SureScan Dx D-scanners, SG18309119 and SG18449122. The text files generated as output were used to generate MammaPrint indices according to standard procedures. The samples included in this dataset covered the entire MammaPrint readout range (-1 to $+ 1$ ), including the borderline region ( -0.070 to $+ 0 . 0 7 0$ for single measurement). MammaPrint indices were compared between both $\mathrm { C } \mathrm { - }$ and D-scanners using Passing and Bablok regression. The MammaPrint categorical results (i.e., high-risk, low-risk) were compared using a 2x2 contingency table after which concordance, Negative Percent Agreement (NPA) and Positive Percent Agreement (PPA) were determined.

The Passing and Bablok regression comparing the MammaPrint indices between the Cscanners and SureScan Dx scanner SG18309119 was $\mathrm { y } { = } 0 . 0 0 + 1 . 0 0 \mathrm { x }$ ( $9 5 \%$ CI – slope: 1.000 $- 1 . 0 0 2$ , $9 5 \%$ CI intercept: -0.0002 to 0.000) (Figure 2A). The Passing and Bablok regression comparing the MammaPrint indices between the C-scanners and SureScan Dx scanner SG18449122 was $\mathrm { y } { = } 0 . 0 0 1 0 { + } 1 . 0 0 \mathrm { x }$ $9 5 \% C \mathrm { I } .$ – slope: 1.000 – 1.0021, $9 5 \% C \mathrm { I }$ intercept: 0.0014 to 0.001) (Figure 2B). The Passing and Bablok regression analysis for both SureScan Dx D-scanners had a slope that was equal to 1 with an intercept close or equal to zero, indicating a high concordance in MammaPrint index between the SureScan Dx Dscanners and the predicate C-scanners. Concordance, NPA and PPA of MammaPrint categorical results were identical between the two SureScan Dx D-scanners and the predicate C-scanners (Tables 2, 3). The lower bound of the $9 5 \%$ CI of both NPA and PPA were both above $8 5 \%$ . The overall concordance in MammaPrint categorical result between the predicate C-scanners and the SureScan Dx D-scanners, SG18309119 and SG18449122 was $9 9 . 7 \%$ and $100 \%$ , respectively. In this method comparison study of C-scanners and D-scanners, SG18309119 and SG18449122, the NPA was $100 \%$ $9 5 \% C \mathrm { I }$ , 98.1 – 100) and $100 \%$ $9 5 \% \mathrm { C I }$ , 97.1 – 100), the PPA was $9 9 . 3 \%$ $9 5 \% C \mathrm { I }$ : 96.0 – 99.9) and $100 \%$ ( $9 5 \% C \mathrm { I }$ : 97.5 - 100), respectively.

![](images/3849c5a78dc5c36404cff25b0dad2906a70643702e080eb7870a1e5ae2e29aed.jpg)

Figure 2: Comparison of MammaPrint indices between the two cleared C-scanners (x-axis) and the SureScan Dx scanners: A. SG18309119 (y-axis) and B. SG18449122 (y-axis).

Table 2: Comparison of MammaPrint outcomes between C-scanners and the $D$ -scanner SG18309119.   

<table><tr><td rowspan=2 colspan=2>MammaPrint Outcome</td><td rowspan=1 colspan=2>C-ScannerUS45103019 and US811R3213</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High-Risk</td><td rowspan=1 colspan=1>Low-Risk</td></tr><tr><td rowspan=2 colspan=1>D-ScannerSG18309119</td><td rowspan=1 colspan=1>High-Risk</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135</td></tr><tr><td rowspan=1 colspan=1>Low-Risk</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>339</td></tr></table>

Table 3: Comparison of MammaPrint outcomes between $C \mathrm { . }$ -scanners and the $D$ -scanner SG18449122.   

<table><tr><td rowspan=2 colspan=2>MammaPrint Outcome</td><td rowspan=1 colspan=2>C-ScannerUS45103019 and US811R3213</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High-Risk</td><td rowspan=1 colspan=1>Low-Risk</td></tr><tr><td rowspan=2 colspan=1>D-ScannerSG18449122</td><td rowspan=1 colspan=1>High-Risk</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=1>Low-Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>280</td></tr></table>

ii. Matrix Comparison: Not applicable

# 3. Clinical Studies:

i. Clinical Sensitivity: Not applicable.   
ii. Clinical specificity: Not applicable.   
iii. Other clinical supportive data (when a. and b. are not applicable):

To show a correlation between the MammaPrint index and clinical outcome based on the RASTER study results, the MammaPrint Index was divided into four pre-specified bins that cover the entire readout range from $+ 1$ to $^ { - 1 }$ .

Bin 1 ranges from $+ 1$ to $+ 0 . 3 6$ . The MammaPrint index threshold 0.36 was defined by Delahaye et al., and clinical utility was shown in the study by Esserman et al. where patients from the STO-3 clinical trial with a MammaPrint index above $+ 0 . 3 6$ had a very low recurrence even after 20 years (Delahaye et al., 2017; Esserman et al., 2017). Bin 2 is between $+ 0 . 3 6$ and 0. Bin 3 is between 0 and -0.57. Bin 4 ranges from a MammaPrint index - 0.57 to -1. The MammaPrint index threshold -0.57 was defined in the I-SPY 2 protocol (clincaltrials.gov, NCT01042379) as the median high risk score in I-SPY 1 study patients, and aimed to create the distinction High 1 and High 2 (or ultrahigh) which has since been used in the randomization of the I-SPY 2 trial (ClinicalTrials.gov NCT01042379; Park et al., 2016; Wolf et al., 2017; Nanda et al., 2020).

The observed risk within each bin as listed in Table 4 shows that the rate of events increases with a decreasing index. Bootstrapping was used for deriving a robust estimate of the $9 5 \%$ confidence interval for the proportion (Table 4, Figure 3A). Kaplan Meier analysis for the 5- year Distant Recurrence Risk (DR-risk) within each bin was performed (Table 4, Figure 3B). The analysis shows a trend between MammaPrint Index and the DR-risk as such with a decreasing index, an increasing DR-risk is observed.

Table 4: Review of 5-year Distant Recurrence Risk within RASTER study over 4 defined bins of MammaPrint Index   

<table><tr><td></td><td></td><td colspan="3"></td><td colspan="3">Observed 5-year Distant Recurrence Risk</td><td colspan="2">Kaplan Meier Analysis 5-year Distant Recurrence Risk</td></tr><tr><td>Bin</td><td>MPI range*</td><td>Number of Patients</td><td>Percentage of Patients</td><td>Number of Events</td><td>Percentage of Events</td><td>95% CI</td><td>5-yr DR risk (%)</td><td></td><td>95% CI</td></tr><tr><td>1</td><td>0.36&lt;MPI≤+1</td><td>37</td><td>10.7</td><td>0</td><td>0</td><td>--</td><td>--</td><td>0 --</td><td>--</td></tr><tr><td>2</td><td>0.00&lt;MPI≤0.36</td><td>142</td><td>41.2</td><td>2</td><td>1.4</td><td>0.0</td><td>3.6</td><td>1.7 0.0</td><td>4.1</td></tr><tr><td>3</td><td>-0.57&lt;MPI≤ 0.00</td><td>100</td><td>29.0</td><td>9</td><td>9.0</td><td>4.0</td><td>14.9</td><td>10.2 3.7</td><td>16.7</td></tr><tr><td>4</td><td>-1≤MPI≤-0.57</td><td>66</td><td>19.1</td><td>9</td><td>13.6</td><td>6.1</td><td>22.4</td><td>14.0 5.6</td><td>22.4</td></tr><tr><td></td><td></td><td>345</td><td>100.0</td><td>20</td><td>5.8</td><td>3.4 8.4</td><td></td><td></td><td></td></tr></table>

\* Actual MPI range: \~-1.2 to $+ 0 . 8 .$ . Reportable range -1 to $+ I$ (see K141142) (numbers ${ < } { - } I$ rounded to -1).

![](images/f83b84a393dd0ca9b433cf825ff8eb8f0dadd968204d2d94e54a238d09713dfa.jpg)  
Observed 5-year distant recurrence risk, RASTER series (data k141142)   
Figure 3: Recurrence risk in the RASTER study (K141142) A. Binned representation of MammaPrint index of the observed 5-year Distant Recurrence Risk, B. Binned representation of MammaPrint index of the 5-year Distant Recurrence Risk based on Kaplan Meier analysis.

Cox regression analysis was performed to show a correlation between MammaPrint index and risk of recurrence (Table 5). With each increase in MammaPrint index unit, there is a 0.224 (i.e., 4.5-fold) decrease in recurrence risk at 5 years. Separate analysis by risk groups is presented in Tables 6 and 7. Analysis for the high-risk group shows a 0.695 (i.e., 1.4-fold) decrease in recurrence risk at 5 years with each increase in MammaPrint index unit. Cox regression analyses for the low risk group is biased by the low number of events $( \mathrm { n } { = } 2 )$ which affects the Cox regression model and results as apparent from a large confidence interval. The two events had a MammaPrint index of 0.282 and 0.297, and are closely grouped together. In the trajectory from 0 to $+ 1$ , these two events result in an HR of 5.587, as the two events are close together and in the first part of the trajectory there are no events. Analyzing the model on the full MammaPrint range was not affected by low number of events.

Table 5: Cox Regression – MammaPrint index predicting ${ } ^ { 5 }$ -year DRFI, in the RASTER study $( n { = } 3 4 5 )$   

<table><tr><td>Parameter</td><td>p-value</td><td>HR</td><td>95% CI</td></tr><tr><td>MPI</td><td>0.001</td><td>0.224</td><td>0.092 0.543</td></tr></table>

Table 6: Cox regression - MammaPrint index predicting 5-year DRFI, in the RASTER study, MammaPrint High Risk patients $( n { = } I 6 6 )$   

<table><tr><td>Parameter</td><td>p-value</td><td>HR</td><td>95% CI</td></tr><tr><td>MPI</td><td>0.579</td><td>0.695</td><td>0.192 2.511</td></tr></table>

Table 7: Cox regression – MammaPrint index predicting 5-year DRFI, in the RASTER study – 5yr DRFI, MammaPrint Low Risk patients $( n { = } I 7 9 )$   

<table><tr><td>Parameter</td><td>p-value</td><td>HR</td><td>95% CI</td></tr><tr><td>MPI</td><td>0.691</td><td>5.587</td><td>0.001 26876</td></tr></table>

The categorical analysis and the Cox regression model demonstrate that there is a correlation between the change in the MammaPrint index and the clinical outcome for the RASTER study, thus supporting the inclusion of the MammaPrint index score in the assay report.

References:

ClinicalTrials.gov NCT01042379 I-SPY 2 TRIAL: Neoadjuvant and Personalized Adaptive Novel Agents to Treat Breast Cancer (I-SPY 2).

Delahaye, L. J. M. J. et al. (2017) ‘A breast cancer gene signature for indolent disease’, Breast Cancer Research and Treatment, 164(2), pp. 461–466. doi: 10.1007/s10549-017-4262-0.

Esserman, L. J. et al. (2017) ‘Use of molecular tools to identify patients with indolent breast cancers with ultralow risk over 2 decades’, JAMA Oncology, 3(11), pp. 1503–1510. doi: 10.1001/jamaoncol.2017.1261.

Nanda, R. et al. (2020) ‘Effect of Pembrolizumab Plus Neoadjuvant Chemotherapy on Pathologic Complete Response in Women with Early-Stage Breast Cancer: An Analysis of the Ongoing Phase 2 Adaptively Randomized I-SPY2 Trial’, JAMA Oncology, 6(5), pp. 676–684. doi: 10.1001/jamaoncol.2019.6650.

Park, J. W. et al. (2016) ‘Adaptive randomization of neratinib in early breast cancer’, New England Journal of Medicine, 375(1), pp. 11–22. doi: 10.1056/NEJMoa1513750.

Wolf, D. et al. (2017) ‘DNA repair deficiency biomarkers and the 70-gene ultra-high risk signature as predictors of veliparib/carboplatin response in the I-SPY breast cancer trial’, Nature PJ Breast, 3:31; doi: 10.1038/s41523-017-0025-7

# D. Clinical cut-off:

Same as previous submission.

# E. Expected Values/Reference Range:

Same as previous submission.

# F. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

i. Software Validation

No fundamental changes were implemented to the MammaPrint index software since the clearance of the predicate device. The current revision in XPrint software from version 2.24 to version 3.2.0 was made to recognize and accept data from the SureScan Dx scanner, part number G5761AA. There were also minor bug fixes, quality improvements, and usability improvements, as well as maintenance activities. However, the algorithm to calculate the MammaPrint index based on the intensities of the probes was the same as the predicate device. The changes were validated under Agendia’s quality systems management. The risk associated with the changes was determined to be Moderate, similar to the predicate device. Therefore, the new version of XPrint software does not raise any new safety or effectiveness concerns.

# N. Proposed Labeling:

For health care provider ordering the MammaPrint $\textsuperscript { \textregistered }$ test, two labeling documentations listed below are provided in the sample collection kit.

Specimens Sampling Instructions Physician’s Brochure

Upon completion of the MammaPrint $\textsuperscript { \textregistered }$ FFPE test, the ordering health care provider will receive MammaPrint $\textsuperscript { \textregistered }$ FFPE results for the risk determination. There are four (4) different patient report forms, i.e., high risk, low risk, borderline high risk, and borderline low risk.

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.

The device is classified as Class II under regulation 21 CFR 862.6040 with special controls. The special control guidance document “Guidance for Industry and FDA Staff – Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis” is available at