# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

I. Background Information

A. 510(k) Number K203245   
B. Applicant Nucleix, LTD.   
C. Proprietary and Established Names Bladder EpiCheck Test   
D. Regulatory Information

<table><tr><td rowspan=1 colspan=1>Product Code:</td><td rowspan=1 colspan=1>MMW</td></tr><tr><td rowspan=1 colspan=1>Device Class:</td><td rowspan=1 colspan=1>Class II</td></tr><tr><td rowspan=1 colspan=1>Classification Regulation:</td><td rowspan=1 colspan=1>21 CFR 866.6010 - Tumor-Associated AntigenImmunological Test System</td></tr><tr><td rowspan=1 colspan=1>Classification Panel:</td><td rowspan=1 colspan=1>Immunology</td></tr></table>

II. Submission/Device Overview

A. Purpose for submission New device.   
B. Measurand

Fifteen DNA methylation biomarkers associated with transitional cell carcinoma of the bladder, collected in DNA extracted from voided urine specimens obtained from patients previously diagnosed with non-muscle invasive bladder cancer (NMIBC)

C. Type of Test: Real-Time PCR (RT-PCR)

III. Intended Use/Indications for Use

A. Intended Use(s)

Bladder EpiCheck Kit is intended for the qualitative detection of DNA methylation patterns of 15 loci in human DNA that are associated with transitional cell carcinoma of the bladder. The test is performed on voided urine samples and run on the ABI 7500 Fast Dx Real-Time PCR system.

Bladder EpiCheck Kit is indicated for use as a non-invasive method to monitor for tumor recurrence in conjunction with cystoscopy in patients previously diagnosed with Non-Muscle Invasive Bladder Cancer.

# B. Special Conditions for Use Statement(s)

Rx - For prescription use For in vitro diagnostic use

C. Special Instrument Requirements

Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument, SDS Software version 1.4 (k082562)

IV. Device/System Characteristics

A. Device Description

# Kit Contents

The assay is comprised of 2 kits: the Bladder EpiCheck Test Kit (NX899090-02C) and Bladder EpiCheck DNA Extraction Kit (NX899090-01C).

The Bladder EpiCheck Test kit contains reagents for the digestion and the real-time PCR amplification assay. Reagents are sufficient for total of 60 reactions: 50 clinical samples and 10 controls required for each run of the Undigested Control (UC) and the No template control (NTC). All reagents of Bladder EpiCheck test kit should be stored at $- 2 0 ^ { \circ } \mathrm { C }$ . A description of the reagents provided in the kit is described in Table 1 below.

The Bladder EpiCheck DNA Extraction kit contains reagents and consumables for DNA extraction from the cell pellet of urine samples. The extraction kit processes sample sizes of up to $2 0 0 \mu \mathrm { L }$ , with a preparation time of 20–40 minutes. Reagents are sufficient for extraction of 50 samples. The extraction kit is stored at room temperature $1 5 \mathrm { - } 2 5 ^ { \circ } \mathrm { C }$ ; $5 9 { - } 7 7 ^ { \circ } \mathrm { F }$ ) with some components needed to be stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ $( 3 5 { - } 4 6 ^ { \circ } \mathrm { F } )$ upon arrival. A description of the reagents provided in the kit is described in Table 2 below.

Table 1. Components of the Bladder EpiCheck Kit   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Content</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - A</td><td rowspan=1 colspan=1>IR primers and probe, BE-1 primers and probe,BE-2 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - B</td><td rowspan=1 colspan=1>IR primers and probe, BE-3 primers and probe,BE-4 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - C</td><td rowspan=1 colspan=1>IR primers and probe, BE-5 primers and probe,BE-6 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - D</td><td rowspan=1 colspan=1>IR primers and probe, BE-7 primers and probe,</td><td rowspan=1 colspan=1>1100uL</td></tr></table>

Table 2. Components of the Bladder EpiCheck DNA Extraction Kit   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BE-8 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PCR Mix - E</td><td rowspan=1 colspan=1>IR primers and probe, BE-9 primers and probe,BE-10 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - F</td><td rowspan=1 colspan=1>IR primers and probe, DC primers andprobe,BE-11 primers and probe, DCR primersand probe, PCR buffer, dNTPs, filled inplastic tubes.</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - G</td><td rowspan=1 colspan=1>IR primers and probe, BE-12 primers and probe,BE-13 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>PCR Mix - H</td><td rowspan=1 colspan=1>IR primers and probe, BE-14 primers and probe,BE-15 primers and probe, PCR buffer, dNTPs, filled inplastic tubes</td><td rowspan=1 colspan=1>1100uL</td></tr><tr><td rowspan=1 colspan=1>Taq DNA Polymerase[5U/μL</td><td rowspan=1 colspan=1>Taq DNA Polymerase, Tris-HCl. KCl, EDTA, DTT, 50%Glycerol (v/v), 0.5% Tween®</td><td rowspan=1 colspan=1>6 x50uL</td></tr><tr><td rowspan=1 colspan=1>10X Digestion Buffer</td><td rowspan=1 colspan=1>Potassium acetate, Tris-acetate, Magnesium acetate, BSA</td><td rowspan=1 colspan=1>2 x1250uL</td></tr><tr><td rowspan=1 colspan=1>1X Dilution Buffer</td><td rowspan=1 colspan=1>Potassium acetate, Tris-acetate, Magnesium acetate, BSA</td><td rowspan=1 colspan=1>1500uL</td></tr><tr><td rowspan=1 colspan=1>Digestion Enzyme[10U/μL]</td><td rowspan=1 colspan=1>KCl, Tris-HCl, DTT, EDTA, 50% Glycerol (v/v), BSA</td><td rowspan=1 colspan=1>2 x220uL</td></tr><tr><td rowspan=1 colspan=1>Digestion ControlTemplate [100fg/μL]</td><td rowspan=1 colspan=1>Exogenous sequences DC and DCR dissolved in NEbuffer</td><td rowspan=1 colspan=1>150uL</td></tr><tr><td rowspan=1 colspan=1>NTC (No TemplateControl)</td><td rowspan=1 colspan=1>Ultra-pure water, DNA free</td><td rowspan=1 colspan=1>1600uL</td></tr><tr><td rowspan=1 colspan=1>UC (UndigestedControl) [2fg/μL]</td><td rowspan=1 colspan=1>17 exogenous sequences dissolved in NE buffer</td><td rowspan=1 colspan=1>1600uL</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Component</td><td colspan="1" rowspan="1">Content</td><td colspan="1" rowspan="1">Volume/Amount</td></tr><tr><td colspan="1" rowspan="1">Mini Spin Columnswith collection tube(2mL)</td><td colspan="1" rowspan="1">Plastic devices (columns) whichcontain a silica membrane and areassembled into 2 ml tubes</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Collection Tubes(2mL)</td><td colspan="1" rowspan="1">Plastic tubes</td><td colspan="1" rowspan="1">4 x 50</td></tr><tr><td colspan="1" rowspan="1">Safe-Lock tubes(1.5mL)</td><td colspan="1" rowspan="1">Plastic tubes with a safe-lock lid</td><td colspan="1" rowspan="1">75</td></tr><tr><td colspan="1" rowspan="1">Buffer NL</td><td colspan="1" rowspan="1">Aqueous solution which containschaotropic ions filled in plasticbottles</td><td colspan="1" rowspan="1">2 x 12mL</td></tr><tr><td colspan="1" rowspan="1">Buffer NW1(concentrate)</td><td colspan="1" rowspan="1">Aqueous solution which containschaotropic ions filled in plasticbottles</td><td colspan="1" rowspan="1">19mL</td></tr><tr><td colspan="1" rowspan="1">Buffer NW2(concentrate)</td><td colspan="1" rowspan="1">Aqueous buffered solution whichcontains sodium chloride andsodium azide as a preservativefilled in plastic bottles</td><td colspan="1" rowspan="1">13mL</td></tr><tr><td colspan="1" rowspan="1">Buffer NE</td><td colspan="1" rowspan="1">Aqueous buffered solution filled inplastic bottles</td><td colspan="1" rowspan="1">15mL</td></tr><tr><td colspan="1" rowspan="1">Protease (lyophilized)</td><td colspan="1" rowspan="1">Subtilisin enzyme which is filledand freeze-dried in glass vials</td><td colspan="1" rowspan="1">1 vial</td></tr><tr><td colspan="1" rowspan="1">Protease solvent</td><td colspan="1" rowspan="1">Aqueous solution filled in plasticbottles.</td><td colspan="1" rowspan="1">1.2mL</td></tr></table>

Materials required, but not provided:

Bladder EpiCheck software (P/N NX899090-03C).

# B. Principle of Operation

A voided urine specimen is centrifuged, and the cells (both normal and cancerous if present) are separated from the urine supernatant. DNA is then extracted from the cell pellet using the Bladder EpiCheck Extraction kit (P/N NX899090-01C). The extracted DNA is digested using a methylation-sensitive restriction enzyme mix, which cleaves DNA at specific recognition sequences if they are unmethylated. Methylated DNA is protected from enzymatic digestion and therefore remains intact.

# Real-Time PCR Amplification

DNA methylation refers to the covalent addition of methyl $\mathrm { ( C H } _ { 3 } )$ groups to the C5 position of the pyrimidine ring of cytosines, typically in a $\mathrm { C p G }$ dinucleotide, of which there are approximately 28 million in the human haploid genome. The covalent addition of a methyl group occurs generally in cytosine within $\mathrm { C p G }$ dinucleotides, which are concentrated in large clusters called CpG islands.

The Bladder EpiCheck Test consists of a panel of 15 novel DNA methylation biomarkers. The Bladder EpiCheck Test differentiates between methylated and non-methylated DNA using RealTime PCR amplification, creating a unique platform for methylation profiling of urine specimens towards the detection of bladder cancer recurrence in patients previously diagnosed with the disease. In cancer, tumor cells are characterized by genome-wide changes in methylation, with some CpG islands undergoing de novo methylation, and several non-CpG island regions becoming demethylated.

Using reagents provided in the Bladder EpiCheck Test kit, the digested DNA samples and test controls are mixed together with PCR primers and probe into 8 unique PCR multiplexes reactions and assayed using the ABI 7500 Fast Dx Real-Time PCR system. The panel of 15 biomarkers is amplified according to their residual levels of methylation. Real-time PCR amplification is performed in a 96-well plate, requires 2 assay-control samples (Undigested Control and No Template Control) and can test up to 10 patient samples. Each sample is analyzed in 8 wells, corresponding to a single column of the plate. In each well, 3-4 biomarkers are amplified. The pane of 15 biomarkers are used to determine whether a sample is bladder cancer positive or bladder

cancer negative. In addition, 3 internal controls (Internal Reference, Digestion Control, and Digestion Control Reference) are used to verify successful digestion and amplification processes.

# Data Analysis

The Bladder EpiCheck software (P/N NX899090-03C) accepts as an input (Source File) the \*.SDS output file of the ABI7500 DX real-time PCR machine, and outputs for each patient’s sample a qualitative diagnosis (positive/negative for recurrence of bladder cancer) based on the patient’s EpiScore. The software implements the Bladder EpiCheck algorithm, which starts from the raw fluorescence data of the real-time PCR run and ends in the diagnosis (positive/negative) of each patient. The software enables the operator, who is running the test, to view and export reports of each patient, as well as to view the raw data of the real-time PCR run. The Bladder EpiCheck software is installed on a stand-alone PC; it does not require web connection. The software setup installs 3rd-party specific templates, which contain all required data and steps for the PCR run on the ABI $7 5 0 0 \mathrm { D x }$ Real-time PCR machine. The user only needs to input the sample's IDs in the thirdparty software and upload the output file once the PCR run is completed. The Bladder EpiCheck software then parses the source file, and translates its content to a unified internal format, Nucleix Real-time Information XML (NRIX). Once in NRIX format, the Bladder EpiCheck algorithm processes the signals, while removing background noise and anomalies, and interpolates the distinct cycle data. The algorithm identifies the sample’s well location (controls and test) and validates their raw data and each calculation steps against predefined thresholds. Once the algorithm run is completed, the EpiScore and diagnosis are assigned to each sample unless an error occurred at any of the calculation steps. The software’s user interface is designed to clearly correlate between the sample’s location and their test results. A simple color-coded scheme separates between negative, positive, or invalid results and from clinical samples and kit control samples (UC and NTC). Per sample view is also available to see the sample's fluorescence signals, a sigmoid-like amplification curve for each marker. The patient's qualitative results and the overall run summary report can be viewed in the software’s user interface and exported as PDF file

# Controls

The following controls were designed to ensure the proper performance of the test:

IR (Internal Reference)

This is an internal control that does not contain a restriction site of the digestion enzyme, and therefore, is not affected by digestion. This marker is included in each well and serves as an internal control for the success of the amplification in that well, specifically that there was an adequate amount of DNA. If the amplification of the IR control fails in one or more of a sample’s wells, analysis of the sample will not be performed, and the software will provide an "invalid" indication.

DC (Digestion Control) and DCR (Digestion Control Reference)   
These are non-genomic internal controls that are located on exogenous DNA sequences. After DNA extraction, the DC and DCR fragments are added to each DNA sample and are used to verify that the digestion of the sample was successful. The DCR fragment does not contain a restriction site, while the DC fragment does contain a restriction site. If digestion is successful, the DCR marker is expected to be amplified successfully, while the DC marker is expected to be digested and therefore not be amplified (or be amplified with low efficiency).

NTC (No template control)

This control contains ultra-pure water (DNA free) to be used for the NTC reaction, which assesses potential contamination during assay setup.

# UC (Undigested control)

This is an external control that contains 17 exogenous sequences that represent the entire Bladder EpiCheck panel of biomarkers (except DCR which is added in the digestion step). These exogenous sequences are designed to be protected from restriction, and therefore, will result a high signal intensity at the PCR stage. The purpose of this sample is to verify that all primers and probes of the PCR mixes, as well as the instrument detectors, are working properly.

# Result Reporting

The Bladder EpiCheck Test Kit is used to calculate the associated methylation level for each of the 15 biomarkers based on the associated Cq (Cycle of quantitation, which is the PCR cycle at which the marker’s amplification curve reached a predefined signal threshold). These methylation levels are translated to marker scores that are based on reference methylation levels obtained from a large set of bladder cancer negative and positive patients. These 15 marker scores are then combined into a single number: the EpiScore, a number between 0 and 100 representing the overall methylation level of the sample at the panel biomarkers. The resulting EpiScore is then compared to a prespecified threshold; if it is above or equal to the threshold (60), the test is called “Positive”; otherwise, it is called “Negative”.

If the sample analysis is failed (e.g., insufficient DNA, incomplete digestion) or the analysis of the entire PCR plate failed (e.g., one of the plate controls failed), the output for the sample will be “Invalid”. In some cases of failure, the sample/plate run may be repeated a second time, and in other cases the user will be instructed to collect a new sample from the patient.

# C. Substantial Equivalence Information:

1. Predicate Device Name(s): UroVysion Bladder Cancer Recurrence Kit

2. Predicate 510(k) Number(s): k013785

3. Comparison with Predicate(s):

Similarities between the predicate and subject devices

<table><tr><td>Characteristic</td><td>Predicate UroVysion Bladder Cancer Recurrence Kit (k013785)</td><td>Subject Device Bladder EpiCheck (k203245)</td></tr><tr><td></td><td colspan="2">Similarities</td></tr><tr><td>Indications for Use</td><td>The UroVysion Bladder Cancer Recurrence kit is designed to detect aneuploidy for chromosomes 3, 7, 17 and loss of the 9p21 locus via</td><td>Bladder EpiCheck Kit is intended for the qualitative detection of DNA methylation patterns of 15 loci in human DNA that are associated with</td></tr><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">PredicateUroVysion Bladder CancerRecurrence Kit (k013785)</td><td colspan="1" rowspan="1">Subject DeviceBladder EpiCheck(k203245)</td></tr><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">fluorescence in situ hybridization(FISH) in urine specimens fromsubjects with transitional cellcarcinoma of the bladder. Resultsfrom the UroVysion Bladder CancerRecurrence Kit are intended for useas a noninvasive method formonitoring for tumor recurrence inconjunction with cystoscopy inpatients previously diagnosed withbladder cancer.</td><td colspan="1" rowspan="1">transitional cell carcinoma of thebladder. The test is performed onvoided urine samples and run on theABI 7500 Fast Dx Real-Time PCRsystem.Bladder EpiCheck Kit is indicated foruse as a non-invasive method tomonitor for tumor recurrence inconjunction with cystoscopy inpatients previously diagnosed withNon-Muscle Invasive Bladder Cancer.</td></tr><tr><td colspan="1" rowspan="1">Measurement Type</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Voided Urine</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample CollectionMethod</td><td colspan="1" rowspan="1">Non-invasive collection ofbiological samples delivered into anon-sterile plastic collectioncap/tube</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Special Conditionsfor Use</td><td colspan="1" rowspan="1">Prescription Only</td><td colspan="1" rowspan="1">Same</td></tr></table>

Differences between the predicate and subject devices   

<table><tr><td colspan="1" rowspan="1">Characteristic</td><td colspan="1" rowspan="1">PredicateUrovysion Bladder CancerRecurrence Kit (k013785)</td><td colspan="1" rowspan="1">Subject DeviceBladder EpiCheck(k203245)</td></tr><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Principle of theProcedure</td><td colspan="1" rowspan="1">Fluorescence In situ hybridization(FISH)</td><td colspan="1" rowspan="1">Real-Time PCR Reaction (RT-PCR)</td></tr><tr><td colspan="1" rowspan="1">Test Target</td><td colspan="1" rowspan="1">Chromosomal DNA</td><td colspan="1" rowspan="1">DNA Methylation Biomarkers</td></tr><tr><td colspan="1" rowspan="1">Extraction andAssayPreparation</td><td colspan="1" rowspan="1">Cells recovered from urine pellets arefixed on slides. DNA is denatured toits single stranded form andsubsequently allowed to hybridizewith the probes.</td><td colspan="1" rowspan="1">DNA is extracted from the cell pelletrecovered from urine.DNA is digested using a methylation-sensitive restriction enzyme, whichcleaves DNA at its recognitionsequence if it is unmethylated.</td></tr><tr><td colspan="1" rowspan="1">DetectionInstrument</td><td colspan="1" rowspan="1">Epi-fluorescence microscope</td><td colspan="1" rowspan="1">Applied Biosystems® 7500 Fast DxReal-Time PCR Instrument</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Control slides are run concurrentlywith patient slides</td><td colspan="1" rowspan="1">Three controls consisting of internalreference controls, one external</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">processing control, and a no templatecontrolControls are run concurrently withpatient samples</td></tr><tr><td colspan="1" rowspan="1">Analysis Software</td><td colspan="1" rowspan="1">Visual Interpretation of fluorescencesignal on FISH slides by labtechnician</td><td colspan="1" rowspan="1">Bladder EpiCheck Software</td></tr></table>

# D. Standards/Guidance Documents Referenced:

The following FDA guidance documents were consulted:

(1) Refuse to Accept Policy for $5 1 0 ( \mathrm { k } ) \mathrm { s }$ ; Guidance for Industry and Food and Drug Administration Staff (February 21, 2019);   
(2) Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices; Guidance for Industry and FDA Staff (May 11, 2005);   
(3) Format for Traditional and Abbreviated 510(k)s – Guidance for Industry and FDA Staff (August 12, 2005);   
(4) Appropriate Use of Voluntary Consensus Standards in Premarket Submissions for Medical Device   
(5) Interference Testing in Clinical Chemistry. CLSI EP07 3rd Edition   
(6) User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition. CLSI EP12-A2   
(7) Evaluation of Stability of In Vitro Diagnostic Reagents; Approved Guideline, A, 2009. CLSI EP25   
(8) Evaluation Of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline - Second Edition. CLSI EP17-A2   
(9) Evaluation Of Precision of Quantitative Measurement Procedures; Approved Guideline - Third Edition, 2014. CLSI EP05-A3   
(10) Medical Devices - Applications of Risk Management to Medical Devices. ISO 14971-1: 2012   
(11) Medical Devices – Quality Management Systems, Requirements for Regulatory Purposes. ISO 13485: 2016.   
(12) Clinical Investigation of Medical Devices for Human Subjects, Good Clinical Practice. ISO 14155: 2011   
(13) Medical Devices - Symbols To Be Used With Medical Device Labels, Labeling And Information To Be Supplied - Part 1: General Requirements, Version 1. ISO 15223: 2016

V. Performance Characteristics:

A. Analytical Performance

# 1. Precision/Reproducibility

Precision and reproducibility for the Bladder EpiCheck $\textsuperscript { \textregistered }$ Test were evaluated across 3 different studies: laboratory-to-laboratory, operator-to-operator/day-to-day, and lot-to-lot/instrument-toinstrument. Between-site, between-operator, between-run, between-day, within-run, and between-lot precision were assessed.

# Laboratory-to-Laboratory

A laboratory-to-laboratory repeatability and reproducibility study was performed to assess variation of Bladder EpiCheck when performed in 3 different laboratories by 2 operators at each laboratory site over 3 non-consecutive run days using contrived human DNA samples. Six (6) contrived human DNA samples were created by blending gDNA from different sample sources (commercially available gDNA, gDNA from T24 cell line, and gDNA derived from healthy donor urine pools) to represent different Bladder EpiCheck marker combinations and to represent four different EpiScore categories, including scores around the test cutoff (high negative and low positive). These 6 samples were tested in technical duplicates at 2 different inputs $( \mathrm { L o w } = 4 \mathrm { n g } / \mathrm { P C R }$ ; $\mathrm { H i g h } = 1 0 \mathrm { n g / P C R } )$ . The sample representation is shown in Table 3 below.

Table 3. Sample Representation for Contrived Specimens Used in Laboratory-Laboratory Precision Studies   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">gDNA source</td><td colspan="1" rowspan="1">Representation</td></tr><tr><td colspan="1" rowspan="1">NX#1</td><td colspan="1" rowspan="1">gDNA from T24 cellline spiked intogDNA derived fromhealthy donor urinepool (0.5% w/w)</td><td colspan="1" rowspan="1">Low Negative (LN)</td></tr><tr><td colspan="1" rowspan="1">NX#2</td><td colspan="1" rowspan="1">gDNA from T24 cellline spiked intogDNA derived fromhealthy donor urinepool (3% w/w)</td><td colspan="1" rowspan="1">High Negative (HN)</td></tr><tr><td colspan="1" rowspan="1">NX#3</td><td colspan="1" rowspan="1">gDNA from T24 cellline spiked intogDNA derived fromhealthy donor urinepool (20% w/w)</td><td colspan="1" rowspan="1">Low Positive (LP)</td></tr><tr><td colspan="1" rowspan="1">NX#4</td><td colspan="1" rowspan="1">gDNA from T24 cellline</td><td colspan="1" rowspan="1">High Positive (HP)</td></tr><tr><td colspan="1" rowspan="1">NX#5</td><td colspan="1" rowspan="1">Human highmethylationcommerciallyavailable gDNAspiked into humanlow methylationcommerciallyavailable gDNA(7.5% w/w)</td><td colspan="1" rowspan="1">Low Positive (LP)</td></tr><tr><td>NX#7</td><td>gDNA from a commercially available mix of human cancer tissue spiked into gDNA derived from healthy donor urine pool (14% w/w)</td><td>High Negative (HN)</td></tr></table>

A total number of 4 replicates were analyzed per panel member (24 replicates per operator per day) for a total of 144 measurements per laboratory (432 total measurements). Agreement between EpiCheck status and true status call (positive/negative) was calculated. Acceptance criteria was an overall agreement of $9 5 \%$ , with a positive/negative agreement (PPA/NPA) for each unique sample of $90 \%$ across all sites. The data demonstrated an overall performance of $9 9 . 3 1 \%$ , as well as an overall agreement above $90 \%$ for all 3 laboratory sites. Precision was also evaluated by EpiScore value, which further demonstrated the overall reproducibility of the Bladder EpiCheck assay.

Overall performance characteristics (agreement, PPA, and NPA) are shown in Tables 4, 5, and 6 below, with EpiScore results shown in Table 7. There were no control (plate) failures (0/54), and overall invalid rate was $0 . 2 \%$ (1/433; $9 5 \%$ CI $0 . 0 5 \%$ ; $1 . 0 3 \%$ ).

Table 4. Interlaboratory Reproducibility per Sample Presented as Overall and by Day   

<table><tr><td colspan="2" rowspan="3"></td><td colspan="8" rowspan="1">Bladder EpiCheck Result</td><td colspan="2" rowspan="3"></td></tr><tr><td colspan="6" rowspan="1">Timepoint (Day)</td><td colspan="2" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">3</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">NegativeSampleS</td><td colspan="1" rowspan="2">DNAInput</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">Invalid Rate</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="2">NX#1 /LN</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60 %;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60 %;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td><td colspan="1" rowspan="2">1.4%(1/73)</td><td colspan="1" rowspan="2">[0.31%;5.91%]</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td></tr><tr><td colspan="1" rowspan="2">NX#2 /HN</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">--</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td></tr><tr><td colspan="1" rowspan="2">NX#7 /HN</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">91.7%(11/12)</td><td colspan="1" rowspan="1">[69.88%;98.12%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">97.2%(35/36)</td><td colspan="1" rowspan="1">[88.46%;99.38%]</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">--</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">83.3%(10/12)</td><td colspan="1" rowspan="1">[60.08%;94.32%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%100%]</td><td colspan="1" rowspan="1">94.4%(34/36)</td><td colspan="1" rowspan="1">[84.53%;98.14%]</td></tr><tr><td colspan="2" rowspan="1">All Negatives</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[90.02%;95.83%]</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">98.6%(213/216)</td><td colspan="1" rowspan="1">[96.58%;99.44%]</td><td colspan="1" rowspan="1">0.5% (1/217)</td><td colspan="1" rowspan="1">[0.10%;2.04%]</td></tr><tr><td colspan="1" rowspan="2">PositiveSampleS</td><td colspan="1" rowspan="2">DNAInput</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">Invalid Rate</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="2">NX#3 /LP</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">--</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td></tr><tr><td colspan="1" rowspan="2">NX#4 /HP</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.%;100%]</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">--</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%; 10%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%; 100%]</td></tr><tr><td colspan="1" rowspan="2">NX#5 /LP</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%; 100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%; 100%]</td><td colspan="1" rowspan="2">0</td><td colspan="1" rowspan="2">--</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[96.38%; 100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%; 100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%; 100%]</td></tr><tr><td colspan="2" rowspan="1">All Positives</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%; 100%]</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%; 00%]</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%; 100%]</td><td colspan="1" rowspan="1">100%(216/216)</td><td colspan="1" rowspan="1">[98.76%; 00%]</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">--</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="2" rowspan="2">All Samples</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Invalid Rate</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="2" rowspan="1">Overall</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[98.16%; 00%]</td><td colspan="1" rowspan="1">97.9%(141/1444)</td><td colspan="1" rowspan="1">[97.92%;99.16%]</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[98.16%; 100%]</td><td colspan="1" rowspan="1">99.3%(429/432)</td><td colspan="1" rowspan="1">[98.28%99.72%]</td><td colspan="1" rowspan="1">0.2%(1/433)</td><td colspan="1" rowspan="1">[0.05%;1.03%</td></tr></table>

Table 5. Interlaboratory Reproducibility per Sample Presented as Overall and by Operator   

<table><tr><td colspan="1" rowspan="3">−</td><td colspan="14" rowspan="1">Bladder EpiCheck Result</td></tr><tr><td colspan="12" rowspan="1">Operator</td><td colspan="2" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">4</td><td colspan="2" rowspan="1">5</td><td colspan="2" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">NegativeSamples/Input</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">NX#1High</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100% (6/)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%6/6)</td><td colspan="1" rowspan="1">168.9100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%6/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">168.9100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;10%]</td></tr><tr><td colspan="1" rowspan="1">NX#1Low</td><td colspan="1" rowspan="1">100%(6/)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#2High</td><td colspan="1" rowspan="1">100% /6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;1000%1</td><td colspan="1" rowspan="1">100% /6)</td><td colspan="1" rowspan="1">[68.9%100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="2">NX#2Low</td><td colspan="1" rowspan="2">100%/6)</td><td colspan="1" rowspan="2">[68.9%;100%]</td><td colspan="1" rowspan="2">100%(6/6)</td><td colspan="1" rowspan="2">[68.9%;100%]</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="2">[68.9%;100%]</td><td colspan="1" rowspan="2">100%(6/6)</td><td colspan="1" rowspan="2">[68.9%;100%]</td><td colspan="1" rowspan="2">100%/6)</td><td colspan="1" rowspan="2">[68.9%;100%]</td><td colspan="1" rowspan="2">100%(6/6)</td><td colspan="1" rowspan="2">[68.9%;100%1</td><td colspan="1" rowspan="2">100%(36/36)</td><td colspan="1" rowspan="2">[93.0%;100%</td></tr><tr><td colspan="1" rowspan="1">/6)</td></tr><tr><td colspan="1" rowspan="1">NX#7Low</td><td colspan="1" rowspan="1">100%6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">83.3%(5/6)</td><td colspan="1" rowspan="1">[49.8%;96.19%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%100%]</td><td colspan="1" rowspan="1">97.2%(35/36)</td><td colspan="1" rowspan="1">[88.5%;99.38%]</td></tr><tr><td colspan="1" rowspan="1">NX#7Low</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">66.7%(4/6)</td><td colspan="1" rowspan="1">[34.7%;88.27%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%</td><td colspan="1" rowspan="1">94.4%(34/36)</td><td colspan="1" rowspan="1">[84.5%;98.14%]</td></tr><tr><td colspan="1" rowspan="1">AllNeg</td><td colspan="1" rowspan="1">100%(36//)6)</td><td colspan="1" rowspan="1">[93.0%;100%]</td><td colspan="1" rowspan="1">91.7%</td><td colspan="1" rowspan="1">[80.9%;</td><td colspan="1" rowspan="1">100%(36/ /)6)</td><td colspan="1" rowspan="1">[93.0%;</td><td colspan="1" rowspan="1">100%(6/)6)</td><td colspan="1" rowspan="1">[93.01%;100%]</td><td colspan="1" rowspan="1">100%(36//)6)</td><td colspan="1" rowspan="1">[93.0%;</td><td colspan="1" rowspan="1">100%(36//)6)</td><td colspan="1" rowspan="1">[93.0%</td><td colspan="1" rowspan="1">98.6%((213/216)</td><td colspan="1" rowspan="1">[96.6%;</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(33/36</td><td colspan="1" rowspan="1">96.62%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">99.44%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">cYanmaGYellow</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">PositiveSample/Input</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">NX#3High</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6//6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#3Low</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100% (/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#4High</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#4Low</td><td colspan="1" rowspan="1">100%(6/</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">168.9100%]</td><td colspan="1" rowspan="1">100%(/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">168:100%1</td><td colspan="1" rowspan="1">100%(3/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#5High</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.92%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.92%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(6//6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">NX#5Low</td><td colspan="1" rowspan="1">100%(6/</td><td colspan="1" rowspan="1">[68.92 %;100%]</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%]</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%/6)</td><td colspan="1" rowspan="1">[68.9%;100%1</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.0%;100%]</td></tr><tr><td colspan="1" rowspan="1">All Pos</td><td colspan="1" rowspan="1">100%(36/)6</td><td colspan="1" rowspan="1">[93.0%;100%]</td><td colspan="1" rowspan="1">100%(36//)6</td><td colspan="1" rowspan="1">[93.0%;100%]</td><td colspan="1" rowspan="1">100%((36/3)6</td><td colspan="1" rowspan="1">[93.0%100%]</td><td colspan="1" rowspan="1">100%( 36/3)6</td><td colspan="1" rowspan="1">[93.0%;100%]</td><td colspan="1" rowspan="1">100%(36/36</td><td colspan="1" rowspan="1">[93.0%;100%</td><td colspan="1" rowspan="1">100%(6/6/)6</td><td colspan="1" rowspan="1">[93.0%;100%</td><td colspan="1" rowspan="1">100%(216/216</td><td colspan="1" rowspan="1">[98.8%;1 0%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="2">Agreement</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">100% (72/72</td><td colspan="1" rowspan="1">[96.4%;100%]</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[90.0%;95.83%]</td><td colspan="1" rowspan="1">100%((72/72</td><td colspan="1" rowspan="1">[96.4%;100%]</td><td colspan="1" rowspan="1">100%((72/72</td><td colspan="1" rowspan="1">[96.4%;100%]</td><td colspan="1" rowspan="1">100%72/72</td><td colspan="1" rowspan="1">[96.4%;100%]</td><td colspan="1" rowspan="1">100%( 2/7)2</td><td colspan="1" rowspan="1">[96.4%;100%]</td><td colspan="1" rowspan="1">99.3%(429/4)2</td><td colspan="1" rowspan="1">[98.3B%99.72%]</td></tr></table>

Table 6. Interlaboratory Reproducibility per Sample Presented as Overall and by Laboratory   

<table><tr><td colspan="2" rowspan="3"></td><td colspan="8" rowspan="1">Bladder EpiCheck Result</td></tr><tr><td colspan="6" rowspan="1">Laboratory</td><td colspan="2" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">3</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">NegativeSamples</td><td colspan="1" rowspan="2">DNAInput</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="2">NX#1</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;00%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="2">NX#2</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;00%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;P10%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="2">NX#7</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">91.7%(11/12)</td><td colspan="1" rowspan="1">[69.88%;98.12%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">97.2%(35/36)</td><td colspan="1" rowspan="1">[88.46%;998%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">83.3%(10/12)</td><td colspan="1" rowspan="1">[60.08%;94.32%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;00%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">94.4%(34/36)</td><td colspan="1" rowspan="1">[84.53%;98.14%</td></tr><tr><td colspan="2" rowspan="1">All Negatives</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[90.02%;9583%]</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;00%]</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">98.6%(213/216)</td><td colspan="1" rowspan="1">[96.58%;9944%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">PositiveSamples</td><td colspan="1" rowspan="2">DNAInput</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="2">NX#3</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="2">NX#4</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;10 %</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%]</td></tr><tr><td colspan="1" rowspan="2">NX#5</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%]</td><td colspan="1" rowspan="1">100%(12/12)</td><td colspan="1" rowspan="1">[81.60%;100%</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">[93.01%;100%</td></tr><tr><td colspan="2" rowspan="1">All Positives</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%</td><td colspan="1" rowspan="1">100%(72/72)</td><td colspan="1" rowspan="1">[96.38%;100%]</td><td colspan="1" rowspan="1">100%(216/216)</td><td colspan="1" rowspan="1">[98.76%;100%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="2">All Samples</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="2" rowspan="1">Overall</td><td colspan="1" rowspan="1">97.9%(141/144)</td><td colspan="1" rowspan="1">[97.92%;99.16%]</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[98.16%;100%]</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[98.16%;100%]</td><td colspan="1" rowspan="1">99.3%(429/432)</td><td colspan="1" rowspan="1">[98.28%99.72%]</td></tr></table>

Table 7. Interlaboratory Reproducibility per Sample Presented as Bladder EpiScore Results by Laboratory   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=6>EpiScore</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Med</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>Laboratory</td><td rowspan=1 colspan=1>Specimen ID</td><td rowspan=1 colspan=6>EY</td></tr><tr><td rowspan=6 colspan=1>1</td><td rowspan=1 colspan=1>NX#1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>28.125</td><td rowspan=1 colspan=1>4.1000</td><td rowspan=1 colspan=1>20.000</td><td rowspan=1 colspan=1>28.500</td><td rowspan=1 colspan=1>36.000</td></tr><tr><td rowspan=1 colspan=1>NX#2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>52.042</td><td rowspan=1 colspan=1>5.9817</td><td rowspan=1 colspan=1>31.000</td><td rowspan=1 colspan=1>53.000</td><td rowspan=1 colspan=1>59.000</td></tr><tr><td rowspan=1 colspan=1>NX#3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>78.833</td><td rowspan=1 colspan=1>5.3541</td><td rowspan=1 colspan=1>64.000</td><td rowspan=1 colspan=1>79.000</td><td rowspan=1 colspan=1>91.000</td></tr><tr><td rowspan=1 colspan=1>NX#4</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>92.208</td><td rowspan=1 colspan=1>2.3587</td><td rowspan=1 colspan=1>86.000</td><td rowspan=1 colspan=1>93.000</td><td rowspan=1 colspan=1>96.000</td></tr><tr><td rowspan=1 colspan=1>NX#5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>74.917</td><td rowspan=1 colspan=1>3.0916</td><td rowspan=1 colspan=1>71.000</td><td rowspan=1 colspan=1>74.500</td><td rowspan=1 colspan=1>83.000</td></tr><tr><td rowspan=1 colspan=1>NX#7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>55.917</td><td rowspan=1 colspan=1>7.9669</td><td rowspan=1 colspan=1>41.000</td><td rowspan=1 colspan=1>55.000</td><td rowspan=1 colspan=1>78.000</td></tr><tr><td rowspan=6 colspan=1>2</td><td rowspan=1 colspan=1>NX#1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>27.125</td><td rowspan=1 colspan=1>4.5903</td><td rowspan=1 colspan=1>18.000</td><td rowspan=1 colspan=1>27.000</td><td rowspan=1 colspan=1>36.000</td></tr><tr><td rowspan=1 colspan=1>NX#2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>49.917</td><td rowspan=1 colspan=1>4.0316</td><td rowspan=1 colspan=1>45.000</td><td rowspan=1 colspan=1>49.500</td><td rowspan=1 colspan=1>58.000</td></tr><tr><td rowspan=1 colspan=1>NX#3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>73.667</td><td rowspan=1 colspan=1>4.1037</td><td rowspan=1 colspan=1>60.000</td><td rowspan=1 colspan=1>74.000</td><td rowspan=1 colspan=1>79.000</td></tr><tr><td rowspan=1 colspan=1>NX#4</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>91.042</td><td rowspan=1 colspan=1>1.8528</td><td rowspan=1 colspan=1>86.000</td><td rowspan=1 colspan=1>91.500</td><td rowspan=1 colspan=1>93.000</td></tr><tr><td rowspan=1 colspan=1>NX#5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>72.167</td><td rowspan=1 colspan=1>2.5481</td><td rowspan=1 colspan=1>64.000</td><td rowspan=1 colspan=1>73.000</td><td rowspan=1 colspan=1>76.000</td></tr><tr><td rowspan=1 colspan=1>NX#7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>50.417</td><td rowspan=1 colspan=1>6.1779</td><td rowspan=1 colspan=1>35.000</td><td rowspan=1 colspan=1>51.000</td><td rowspan=1 colspan=1>59.000</td></tr><tr><td rowspan=6 colspan=1>3</td><td rowspan=1 colspan=1>NX#1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>28.500</td><td rowspan=1 colspan=1>7.6158</td><td rowspan=1 colspan=1>18.000</td><td rowspan=1 colspan=1>27.500</td><td rowspan=1 colspan=1>59.000</td></tr><tr><td rowspan=1 colspan=1>NX#2</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>44.750</td><td rowspan=1 colspan=1>3.9037</td><td rowspan=1 colspan=1>36.000</td><td rowspan=1 colspan=1>44.500</td><td rowspan=1 colspan=1>59.000</td></tr><tr><td rowspan=1 colspan=1>NX#3</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>76.375</td><td rowspan=1 colspan=1>2.2030</td><td rowspan=1 colspan=1>69.000</td><td rowspan=1 colspan=1>76.500</td><td rowspan=1 colspan=1>79.000</td></tr><tr><td rowspan=1 colspan=1>NX#4</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>91.833</td><td rowspan=1 colspan=1>2.0990</td><td rowspan=1 colspan=1>83.000</td><td rowspan=1 colspan=1>92.000</td><td rowspan=1 colspan=1>94.000</td></tr><tr><td rowspan=1 colspan=1>NX#5</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>73.083</td><td rowspan=1 colspan=1>1.9092</td><td rowspan=1 colspan=1>70.000</td><td rowspan=1 colspan=1>73.000</td><td rowspan=1 colspan=1>79.000</td></tr><tr><td rowspan=1 colspan=1>NX#7</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>51.208</td><td rowspan=1 colspan=1>3.9780</td><td rowspan=1 colspan=1>43.000</td><td rowspan=1 colspan=1>51.000</td><td rowspan=1 colspan=1>59.000</td></tr></table>

Overall performance characteristics (agreement, PPA, and NPA) for all conditions tested are summarized in Table 8 below.

Table 8. Interlaboratory Reproducibility Performance (Study 1) Summarized by All Variables – by Day, Operator, Site, Sample Source, Sample Concentration, and EpiScore Value   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">−</td><td colspan="2" rowspan="1">OverallAgreement</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">% (n/N)</td><td colspan="1" rowspan="1">WilsonScore95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">WilsonScore95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">WilsonScore95%CI</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">99.31%(429/432)</td><td colspan="1" rowspan="1">[97.98%;99.76%]</td><td colspan="1" rowspan="1">98.61%(213/216)</td><td colspan="1" rowspan="1">[96.00%;99.53%]</td><td colspan="1" rowspan="1">100%(216/216)</td><td colspan="1" rowspan="1">[98.25%;100.0%]</td></tr><tr><td colspan="1" rowspan="3">Day</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[97.40%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">97.92%(141/144)</td><td colspan="1" rowspan="1">[94.05%;99.29%</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[88.5%;98.6%</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100%(144/144)</td><td colspan="1" rowspan="1">[97.40%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%</td></tr><tr><td colspan="1" rowspan="6">Operator</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.93%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">95.83%(69/72)</td><td colspan="1" rowspan="1">[88.45%;98.57%]</td><td colspan="1" rowspan="1">91.7%(33/36)</td><td colspan="1" rowspan="1">[79.1%;97.4%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.93%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.93%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;1000.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;10.0%]</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.93%;1000.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;10.0%]</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.93%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td><td colspan="1" rowspan="1">100.0%(36/36)</td><td colspan="1" rowspan="1">[90.6%;100.0%]</td></tr><tr><td colspan="1" rowspan="3">Laboratory</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">97.92%(141/144)</td><td colspan="1" rowspan="1">[94.05%;99.29%</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[88.5%;98.6%</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[97.40%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[97.40%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="2">SampleSource</td><td colspan="1" rowspan="1">T24</td><td colspan="1" rowspan="1">100.0%(288/288)</td><td colspan="1" rowspan="1">[98.68%;100.0%</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[97.4%;100.0%</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[97.4%;100.0%</td></tr><tr><td colspan="1" rowspan="1">CommercialDNA</td><td colspan="1" rowspan="1">97.92%(141/144)</td><td colspan="1" rowspan="1">[94.05%;99.29%]</td><td colspan="1" rowspan="1">95.8%(69/72)</td><td colspan="1" rowspan="1">[88.5%;98.6%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[94.9%;10.0%]</td></tr><tr><td colspan="1" rowspan="2">SampleConcentration</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">99.54%(215/216)</td><td colspan="1" rowspan="1">[97.42%;99.92%]</td><td colspan="1" rowspan="1">99.1%(107/108)</td><td colspan="1" rowspan="1">[95.2%;99.9%]</td><td colspan="1" rowspan="1">100.0%(108/108)</td><td colspan="1" rowspan="1">[96.7%;100.0%</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">99.07%(214/216)</td><td colspan="1" rowspan="1">[96.69%;99.7%</td><td colspan="1" rowspan="1">98.1%(106/108)</td><td colspan="1" rowspan="1">[93.9%;99.6%</td><td colspan="1" rowspan="1">100.0%(108/108)</td><td colspan="1" rowspan="1">[96.7%;1000%]</td></tr><tr><td colspan="1" rowspan="3">EpiScoreValue</td><td colspan="1" rowspan="1">LowNegative</td><td colspan="1" rowspan="1">100.0%(70/72)</td><td colspan="1" rowspan="1">[96.4%;100.0%]</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[96.4%;100.0%]</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">97.92%(141/144)</td><td colspan="1" rowspan="1">[94.1%;99.3%]</td><td colspan="1" rowspan="1">97.9%(141/144)</td><td colspan="1" rowspan="1">[94.1%;9.3%]</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[98.2%;100.0%]</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">100.0%(144/144)</td><td colspan="1" rowspan="1">[98.2%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">HighPositive</td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[96.4%;10%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0%(72/72)</td><td colspan="1" rowspan="1">[96.4%;10.0%]</td></tr></table>

An additional laboratory-to-laboratory reproducibility study, using clinical samples, was performed at 3 laboratories to determine precision between laboratories. Four clinical samples were used to represent Low Negative (LN; ES ${ \sim } 2 0$ ), High Negative $( \mathrm { H N } ; \mathrm { E S } { \sim } 4 5 )$ , Low Positive (HP; ES\~75), and High Positive (HP; ${ \mathrm { E S } } { \sim } 9 0 $ ) at 2 DNA input levels. One operator ran the test at each laboratory location on 3 different, non-consecutive workdays with an interval of at least 3 days between runs. A single reagent lot was used for this study. On each day of testing, each operator ran 4 unique clinical samples at 2 DNA concentration levels with the low DNA concentration representing a concentration close to the assay cut-off. A total of 12 replicates per sample per site were tested (48 per laboratory, 144 total measurements). Acceptance criteria was an overall (positive/negative) level of agreement of $9 5 \%$ . The data demonstrated an overall performance of $9 6 . 5 \%$ . Precision was also evaluated by EpiScore value, which further demonstrated the overall reproducibility of the Bladder EpiCheck assay. Overall performance characteristics (agreement, PPA, and NPA) are described in Table 9 and 10 below, with EpiScore results shown in Table 11. There were no control (plate) failures (0/18).

Table 9. Interlaboratory Reproducibility per Sample Presented as Overall and by Laboratory   

<table><tr><td rowspan=1 colspan=2>Caboratoy</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=8>Bladder EpiCheck Result</td></tr><tr><td rowspan=1 colspan=6>Laboratory</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>NegativeSample</td><td rowspan=2 colspan=1>DNAInput</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td></tr><tr><td rowspan=2 colspan=1>LN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;10%%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;10</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;P100%</td></tr><tr><td rowspan=2 colspan=1>HN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>83.3%(5/6)</td><td rowspan=1 colspan=1>[57.47%;94.87%</td><td rowspan=1 colspan=1>83.3%(5/6)</td><td rowspan=1 colspan=1>[57.47%;94.87%</td><td rowspan=1 colspan=1>88.9%(16/18)</td><td rowspan=1 colspan=1>[75.99%;95.29%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>66.7%(4/6)</td><td rowspan=1 colspan=1>[40.94%;85.23%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>83.3%(5/6)</td><td rowspan=1 colspan=1>[57.47%;94.87%</td><td rowspan=1 colspan=1>83.3%(15/18)</td><td rowspan=1 colspan=1>[69.42%;91.68%</td></tr><tr><td rowspan=1 colspan=2>All Negatives</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>[75.1%;98.2%</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[80.4%;99.7%</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>[75.1%;98.2%</td><td rowspan=1 colspan=1>93.1%(67/72)</td><td rowspan=1 colspan=1>[88.2%;96.0%</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>PositiveSample</td><td rowspan=2 colspan=1>DNAInput</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;00%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=2 colspan=1>HP</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;P100%]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=1 colspan=2>All Positives</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>[86.4%;10%</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>[86.4%;100%</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>[86.4%;10%</td><td rowspan=1 colspan=1>100%(72/72)</td><td rowspan=1 colspan=1>[97.8%;1 0%</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>cYanmaGellowb</td></tr><tr><td rowspan=2 colspan=2>All Samples</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td></tr><tr><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>90% CI</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n/N)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n/N)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n/N)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n/N)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>[90.4%;98.</td><td rowspan=1 colspan=1>97.9%(47/48)</td><td rowspan=1 colspan=1>[93.3%;9.4%]</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>[90.4%;98.%]</td><td rowspan=1 colspan=1>96.5%(139/144)</td><td rowspan=1 colspan=1>[94.0%;98.0%]</td></tr></table>

Table 10. Interlaboratory Reproducibility per Sample Presented as Overall and by Timepoint (Day)   

<table><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=8>Bladder EpiCheck Result</td><td rowspan=3 colspan=2></td></tr><tr><td rowspan=1 colspan=6>Timepoint (Day)</td><td rowspan=2 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=2>3</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>NegativeSSample</td><td rowspan=2 colspan=1>DNAInput</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=2>Invalid Rate</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td></tr><tr><td rowspan=2 colspan=1>LN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%100%]</td><td rowspan=2 colspan=1>0</td><td rowspan=2 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51 %;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=2 colspan=1>HN</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>66.7%(4/6)</td><td rowspan=1 colspan=1>[40.94%;85.23%]</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>88.9%(16/18)</td><td rowspan=1 colspan=1>[75.99%;95.29%]</td><td rowspan=2 colspan=1>5.3%(2/38)</td><td rowspan=2 colspan=1>$2.212.0%]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(4/4)</td><td rowspan=1 colspan=1>[70.89%;100%]</td><td rowspan=1 colspan=1>75%(6//8)</td><td rowspan=1 colspan=1>[52.37%;89.11%]</td><td rowspan=1 colspan=1>83.3%(55/6)</td><td rowspan=1 colspan=1>[57.47%;94.87%]</td><td rowspan=1 colspan=1>83.3%(15/18)</td><td rowspan=1 colspan=1>[69.42%;91.68%]</td></tr><tr><td rowspan=1 colspan=2>All Negatives</td><td rowspan=1 colspan=1>100%(22/22)</td><td rowspan=1 colspan=1>[85.3%;100%]</td><td rowspan=1 colspan=1>84.6%(22/26)</td><td rowspan=1 colspan=1>[67.6%;94.1%]</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[80.4%;99.7%]</td><td rowspan=1 colspan=1>93.1%(67/72)</td><td rowspan=1 colspan=1>[88.2%;96.0%]</td><td rowspan=1 colspan=1>2.7%(2/74)</td><td rowspan=1 colspan=1>0.16.3%]</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>PositiveSample</td><td rowspan=2 colspan=1>DNAInput</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=2>Invalid Rate</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td><td rowspan=2 colspan=1>0</td><td rowspan=2 colspan=1>--</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%</td></tr><tr><td rowspan=2 colspan=1>HP</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>78.51%;100%]</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%</td><td rowspan=1 colspan=1>100% (6/6)</td><td rowspan=1 colspan=1>[78.51 %;100%]</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%100%]</td><td rowspan=2 colspan=1>2.7%1/37)</td><td rowspan=2 colspan=1>$0.88.6%]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>[78.51%;100%]</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>[75.27%100%</td><td rowspan=1 colspan=1>100% (7/7)</td><td rowspan=1 colspan=1>[81.00%;100%]</td><td rowspan=1 colspan=1>100%(18/18)</td><td rowspan=1 colspan=1>[91.64%;100%]</td></tr><tr><td rowspan=1 colspan=2>All Positives</td><td rowspan=1 colspan=1>100%(224/24)</td><td rowspan=1 colspan=1>[86.4%;100%]</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>[85.9%;100%]</td><td rowspan=1 colspan=1>100%(25/25)</td><td rowspan=1 colspan=1>[86.9%;100%]</td><td rowspan=1 colspan=1>100%(72/72)</td><td rowspan=1 colspan=1>[97.8%;100%]</td><td rowspan=1 colspan=1>1.4%(1/73)</td><td rowspan=1 colspan=1>$0.4$4.5%]</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=2>AllSamples</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Invalid Rate</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90% CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>90%CI</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>[96.6%</td><td rowspan=1 colspan=1>01.80</td><td rowspan=1 colspan=1>[85.4%</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>[93.4%</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>[94.0%</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>[1.0</td></tr></table>

Table 11. Interlaboratory Reproducibility per Sample Presented as Bladder EpiScore Results by Laboratory   

<table><tr><td rowspan=2 colspan=1>Laboratory</td><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=6>Bladder EpiScore</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>98.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>76.6</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>70.0</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>82.0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>16.1</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>42.5</td><td rowspan=1 colspan=1>69.0</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>16.0</td></tr><tr><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>97.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>74.4</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>64.0</td><td rowspan=1 colspan=1>76.0</td><td rowspan=1 colspan=1>79.0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>66.0</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>7.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>30.0</td></tr><tr><td rowspan=4 colspan=1>3</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>96.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>97.0</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>75.0</td><td rowspan=1 colspan=1>77.5</td><td rowspan=1 colspan=1>82.0</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>46.0</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>50.5</td><td rowspan=1 colspan=1>68.0</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>16.3</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>23.0</td></tr></table>

Overall performance characteristics (agreement, PPA, and NPA) for all conditions tested are summarized in Table 12 below.

Table 12. Interlaboratory Reproducibility Performance (Study 2) Summarized by All Variables – by Laboratory, Day, Sample Concentration, and Specimen Type   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=7>Bladder EpiCheck</td></tr><tr><td rowspan=1 colspan=2>Overall Agreement</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=3>PPA</td></tr><tr><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>WilsonScore90% CI</td><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>WilsonScore90% CI</td><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=2>WilsonScore90% CI</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>96.5%(139/144)*</td><td rowspan=1 colspan=1>[94.0%;98.%]</td><td rowspan=1 colspan=1>93.1%(67/72)</td><td rowspan=1 colspan=1>[88.2%;96.0%]</td><td rowspan=1 colspan=2>100.0%(72/72)</td><td rowspan=1 colspan=1>[97.8%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>Day</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0%(46/46)</td><td rowspan=1 colspan=1>[96.6%;100.0%]</td><td rowspan=1 colspan=1>100.0%(22/22)</td><td rowspan=1 colspan=1>[85.3%;100.0%</td><td rowspan=1 colspan=1>100.0%(24/24)</td><td rowspan=1 colspan=2>[86.4%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>91.8%(45/49)</td><td rowspan=1 colspan=1>[85.4%;95.6%]</td><td rowspan=1 colspan=1>84.6%(22/26)</td><td rowspan=1 colspan=1>[67.6%;94.1%</td><td rowspan=1 colspan=1>100.0%(23/23)</td><td rowspan=1 colspan=2>[85.9%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.0%(48/49)</td><td rowspan=1 colspan=1>93.4%;[99.4%]</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[80.4%;99.7%</td><td rowspan=1 colspan=1>100.0%(25/25)</td><td rowspan=1 colspan=2>[86.9%;100.0%</td></tr><tr><td rowspan=1 colspan=1>Laboratory</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>[90.4%;98.3%</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>[75.1%;98.2%</td><td rowspan=1 colspan=1>100.0%(24/24)</td><td rowspan=1 colspan=2>[86.4%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>98.0%(47/48)</td><td rowspan=1 colspan=1>[93.3%;99.4%]</td><td rowspan=1 colspan=1>95.8%(23/24)</td><td rowspan=1 colspan=1>[80.4%;99.7%]</td><td rowspan=1 colspan=1>100.0%(24/24)</td><td rowspan=1 colspan=2>[86.4%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>95.8%(46/48)</td><td rowspan=1 colspan=1>[90.4%;98.3%]</td><td rowspan=1 colspan=1>91.7%(22/24)</td><td rowspan=1 colspan=1>[75.1%;98.2%]</td><td rowspan=1 colspan=1>100.0%(24/24)</td><td rowspan=1 colspan=2>[86.4%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>SampleConcentration</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td></tr></table>

<table><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>97.2%(70/72)</td><td rowspan=1 colspan=1>[93.5%;98.8%]</td><td rowspan=1 colspan=1>94.4%(34/36)</td><td rowspan=1 colspan=1>[82.7%;98.8%</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[90.6%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>95.8%(69/72)</td><td rowspan=1 colspan=1>[91.7%;98.0%</td><td rowspan=1 colspan=1>91.7%(33/36)</td><td rowspan=1 colspan=1>[79.1%;97.4%</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[90.6%;100.0%]</td></tr><tr><td rowspan=1 colspan=1>EpiScore Value</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Low Negative</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%]</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>86.1%(31/36)*</td><td rowspan=1 colspan=1>[77.1%;91.9%</td><td rowspan=1 colspan=1>86.1%(31/36)</td><td rowspan=1 colspan=1>[77.1%;91.9%]</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100.0%(36/36)</td><td rowspan=1 colspan=1>[95.6%;100%</td></tr></table>

\*All samples except the High Negative had $1 0 0 \%$ agreement with the Expected Result. However, these discordances occurred across all 3 sites uniformly and across both DNA concentrations, and this category of sample is incredibly rare $( \sim 4 . 5 \%$ of the sample population), so the risk of a false positive result would be ${ \sim } 1 \%$ .

# Operator-to-Operator/Day-to-Day

An operator-to-operator and day-to-day repeatability and reproducibility study was performed at single site to assess the intra-run (within-run), inter-run (between-run), and inter-operator (between- operator) variation of Bladder EpiCheck when performed by 2 different operators over 5 non-consecutive days. Healthy donor urine pools were spiked with bladder cancerderived cell line material at different input levels to formulate 4 test samples spanning the measurable range of EpiScore values (LN, HN, LP, and HP) tested in 5 replicates per sample (20 replicates per day per operator, for a total of 200 replicates). Acceptance criteria was an overall (positive/negative) level of agreement of $9 5 \%$ across all operators and days. The data demonstrated a $9 9 . 0 \%$ agreement, overall and across all operators and days. Precision was also evaluated by EpiScore value, which further demonstrated the overall reproducibility of the Bladder EpiCheck assay. Overall performance characteristics (agreement, PPA, and NPA) are summarized in Tables 13, 14, and 15 below, with EpiScore results shown in Table 16. There were no invalid results and 1 control (plate) failure out of 21 plates $1 / 2 1 = 4 . 8 \%$ $[ 9 5 \%$ CI $1 . 1 \%$ $1 8 . 8 \% ] _ { , }$ ).

Table 13. Within Site Precision per Sample Presented as Overall and by Day (5 Days)   

<table><tr><td colspan="1" rowspan="3">SampleCategory</td><td colspan="12" rowspan="1">Bladder EpiCheck Result</td></tr><tr><td colspan="10" rowspan="1">Timepoint (Day)</td><td colspan="2" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">1</td><td colspan="2" rowspan="1">2</td><td colspan="2" rowspan="1">3</td><td colspan="2" rowspan="1">4</td><td colspan="2" rowspan="1">5</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">NegativeSample</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(50/50)</td><td colspan="1" rowspan="1">[94.9%;[1100%]</td></tr><tr><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">[78.7%;</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">[78.7%;</td><td colspan="1" rowspan="1">90%(/ /))</td><td colspan="1" rowspan="1">[59.5%;</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">[78.7%;</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">[78.7%;</td><td colspan="1" rowspan="1">98%(49/50)</td><td colspan="1" rowspan="1">[91.5%;9.6%]</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(10/10)</td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1">(10/10)</td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">99.3%]</td><td colspan="1" rowspan="1">(10/10)</td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1">(10/10)</td><td colspan="1" rowspan="1">100%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">All Negatives</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%1</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">[76.4%;99.1%]</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%1</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">99%(9/100)</td><td colspan="1" rowspan="1">[94.6%;9.8%]</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">PositiveSample</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">90%(/ /))</td><td colspan="1" rowspan="1">[59.5%;99.3%]</td><td colspan="1" rowspan="1">98%(449/50))</td><td colspan="1" rowspan="1">[91.5%;9.6%]</td></tr><tr><td colspan="1" rowspan="1">HP</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(50/50)</td><td colspan="1" rowspan="1">[94.9%;00%]</td></tr><tr><td colspan="1" rowspan="1">All Positives</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100%]</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">[76.4%;99.1%]</td><td colspan="1" rowspan="1">99%(99/100)</td><td colspan="1" rowspan="1">[94.6%;99.8%</td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">—</td><td colspan="2" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td><td colspan="2" rowspan="1">Agreement</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">100%(40/40)</td><td colspan="1" rowspan="1">[91.2%;100%]</td><td colspan="1" rowspan="1">100%(40/40)</td><td colspan="1" rowspan="1">[91.2%;100%]</td><td colspan="1" rowspan="1">97.5%(39/40)</td><td colspan="1" rowspan="1">[87.1%;99.6%]</td><td colspan="1" rowspan="1">100%(40/40)</td><td colspan="1" rowspan="1">[91.2%;100%]</td><td colspan="1" rowspan="1">97.5%(39/40)</td><td colspan="1" rowspan="1">[87.1%;99.6%]</td><td colspan="1" rowspan="1">99%(198/200)</td><td colspan="1" rowspan="1">[96.4%;9.7%</td></tr></table>

Table 14. Within Site Precision per Sample Presented as Overall and by Operator 1 per Day (5 Days)   

<table><tr><td colspan="1" rowspan="3">SampleCategory</td><td colspan="12" rowspan="1">Bladder EpiCheck Result</td></tr><tr><td colspan="10" rowspan="1">Operator 1</td><td colspan="2" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">Timepoint1</td><td colspan="2" rowspan="1">Timepoint2</td><td colspan="2" rowspan="1">Timepoint3</td><td colspan="2" rowspan="1">Timepoint4</td><td colspan="2" rowspan="1">Timepoint5</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">NegativeSample</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">% (n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;1000%]</td><td colspan="1" rowspan="1">100%(25/25)</td><td colspan="1" rowspan="1">[86.7%;100%]</td></tr><tr><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%100%]</td><td colspan="1" rowspan="1">100%(25/25)</td><td colspan="1" rowspan="1">[86.7%; 100%]</td></tr><tr><td colspan="1" rowspan="2">All Negatives</td><td colspan="1" rowspan="2">100%(10/10)</td><td colspan="1" rowspan="2">[78.7%;100%]</td><td colspan="1" rowspan="2">100%(10/10)</td><td colspan="1" rowspan="2">[78.7%;1000%]</td><td colspan="1" rowspan="2">100%(10/10)</td><td colspan="1" rowspan="2">[78.7%;100%]</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="2">[78.7%;100%]</td><td colspan="1" rowspan="2">100%(10/10)</td><td colspan="1" rowspan="2">[78.7%;1000%]</td><td colspan="1" rowspan="2">100%(50/50)</td><td colspan="1" rowspan="2">[94.9%;100%]</td></tr><tr><td colspan="1" rowspan="1">(10/10)</td></tr><tr><td colspan="1" rowspan="2">PositiveSample</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">80%(4/5)</td><td colspan="1" rowspan="1">[37.6%;95.4%]</td><td colspan="1" rowspan="1">96%(24/25</td><td colspan="1" rowspan="1">[80.5%; 9.7%]</td></tr><tr><td colspan="1" rowspan="1">HP</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(25/25</td><td colspan="1" rowspan="1">[86.7%; 10%]</td></tr><tr><td colspan="1" rowspan="1">All Positives</td><td colspan="1" rowspan="1">100%(10/)0)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">90%(/ /10))</td><td colspan="1" rowspan="1">[59.5%;99.3%]</td><td colspan="1" rowspan="1">98%(49/50)</td><td colspan="1" rowspan="1">[91.5%;99.6%]</td></tr></table>

<table><tr><td rowspan=2 colspan=1>AllSamples</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td><td rowspan=1 colspan=2>Agreement</td></tr><tr><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>%(n/N)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>[83.9%100%]</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>[83.9%;100%]</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>[83.9%;100%]</td><td rowspan=1 colspan=1>100%(20/20)</td><td rowspan=1 colspan=1>[83.9%;1000%]</td><td rowspan=1 colspan=1>95%(19/20)</td><td rowspan=1 colspan=1>[76.4%;99.1%]</td><td rowspan=1 colspan=1>99%( 9/100)</td><td rowspan=1 colspan=1>[94.6%;9.8%]</td></tr></table>

Table 15. Within Site Precision per Sample Presented as – Overall and by Operator 2 per Day (5 Days)   

<table><tr><td colspan="1" rowspan="1">Day (5 Days)</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="3">SampleCategory</td><td colspan="13" rowspan="1">Bladder EpiCheck Result</td></tr><tr><td colspan="10" rowspan="1">Operator 2</td><td colspan="3" rowspan="2">Overall</td></tr><tr><td colspan="2" rowspan="1">Timepoint1</td><td colspan="2" rowspan="1">Timepoint2</td><td colspan="2" rowspan="1">Timepoint3</td><td colspan="2" rowspan="1">Timepoint4</td><td colspan="2" rowspan="1">Timepoint5</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">NegativeSample</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="2" rowspan="1">NPA</td><td colspan="3" rowspan="1">NPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="2" rowspan="1">% (n/N)</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LN</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="2" rowspan="1">100%(25/25)</td><td colspan="1" rowspan="1">[86.7%; 100%]</td></tr><tr><td colspan="1" rowspan="1">HN</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">80%(/5)</td><td colspan="1" rowspan="1">[37.6%;95.4%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="2" rowspan="1">96%(24/25)</td><td colspan="1" rowspan="1">[80.5%; 9.7%]</td></tr><tr><td colspan="1" rowspan="1">All Negatives</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">90%(9/10)</td><td colspan="1" rowspan="1">[59.5%;99.3%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="2" rowspan="1">98%(49/50)</td><td colspan="1" rowspan="1">[91.5%;9.6%]</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">PositiveSample</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="2" rowspan="1">PPA</td><td colspan="3" rowspan="1">PPA</td></tr><tr><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">%(n/N)</td><td colspan="2" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(25/25)</td><td colspan="1" rowspan="1">[86.7%;100%]</td></tr><tr><td colspan="1" rowspan="1">HP</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(5/5)</td><td colspan="1" rowspan="1">[56.6%;100%]</td><td colspan="1" rowspan="1">100%(25/25)</td><td colspan="1" rowspan="1">[86.7%;100%]</td></tr><tr><td colspan="1" rowspan="1">All Positives</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;10%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;1000%]</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">[78.7%;100%]</td><td colspan="1" rowspan="1">100%(50/50)</td><td colspan="1" rowspan="1">[94.9%;100%]</td></tr></table>

<table><tr><td rowspan="2">All Samples</td><td colspan="2">Agreement</td><td colspan="2">Agreement</td><td colspan="2">Agreement</td><td colspan="2">Agreement</td><td colspan="2">Agreement</td><td colspan="2">Agreement</td></tr><tr><td>% (n/N )</td><td>95% CI</td><td>% (n/N )</td><td>95% CI</td><td>% (n/N</td><td>95% CI</td><td>% (n/N</td><td>95% CI</td><td>% (n/N</td><td>95% CI</td><td>% (n/N)</td><td>95% CI</td></tr><tr><td rowspan="3">Overall</td><td>100 %</td><td>[83.9</td><td>100</td><td>[83.9</td><td>) 95%</td><td>[76.4 %;</td><td>) 100 %</td><td>[83.9 %;</td><td>) 100 %</td><td>[83.9 %;</td><td>99%</td><td rowspan="3">[94.6%; 9.8%</td></tr><tr><td></td><td>%;</td><td>%</td><td>%;</td><td>(19/2</td><td></td><td></td><td></td><td></td><td></td><td>( 9/)</td></tr><tr><td>(20/2 0)</td><td>10 %]</td><td>(20/2 0)</td><td>1000 %]</td><td>0)</td><td>99.1 %]</td><td>(20/2 0)</td><td>1000 %]</td><td>(20/2 0)</td><td>100 %]</td><td>00)</td></tr></table>

Table 16. Within Site Precision per Sample Presented as Bladder EpiScore Results – Overall and by Operator   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=6>EpiScore</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>Operator</td><td rowspan=1 colspan=1>Sample</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=1>1</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>46.80</td><td rowspan=1 colspan=1>5.377</td><td rowspan=1 colspan=1>33.00</td><td rowspan=1 colspan=1>48.00</td><td rowspan=1 colspan=1>55.00</td></tr><tr><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>92.96</td><td rowspan=1 colspan=1>0.611</td><td rowspan=1 colspan=1>92.00</td><td rowspan=1 colspan=1>93.00</td><td rowspan=1 colspan=1>94.00</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>15.36</td><td rowspan=1 colspan=1>1.655</td><td rowspan=1 colspan=1>13.00</td><td rowspan=1 colspan=1>15.00</td><td rowspan=1 colspan=1>19.00</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>67.80</td><td rowspan=1 colspan=1>4.010</td><td rowspan=1 colspan=1>59.00</td><td rowspan=1 colspan=1>69.00</td><td rowspan=1 colspan=1>74.00</td></tr><tr><td rowspan=4 colspan=1>2</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>50.80</td><td rowspan=1 colspan=1>3.096</td><td rowspan=1 colspan=1>44.00</td><td rowspan=1 colspan=1>51.00</td><td rowspan=1 colspan=1>60.00</td></tr><tr><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>92.88</td><td rowspan=1 colspan=1>0.440</td><td rowspan=1 colspan=1>92.00</td><td rowspan=1 colspan=1>93.00</td><td rowspan=1 colspan=1>94.00</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>14.60</td><td rowspan=1 colspan=1>1.225</td><td rowspan=1 colspan=1>12.00</td><td rowspan=1 colspan=1>15.00</td><td rowspan=1 colspan=1>17.00</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>68.44</td><td rowspan=1 colspan=1>2.181</td><td rowspan=1 colspan=1>64.00</td><td rowspan=1 colspan=1>69.00</td><td rowspan=1 colspan=1>72.00</td></tr><tr><td rowspan=4 colspan=1>Overall</td><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>48.80</td><td rowspan=1 colspan=1>4.789</td><td rowspan=1 colspan=1>33.00</td><td rowspan=1 colspan=1>50.00</td><td rowspan=1 colspan=1>60.00</td></tr><tr><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>92.92</td><td rowspan=1 colspan=1>0.528</td><td rowspan=1 colspan=1>92.00</td><td rowspan=1 colspan=1>93.00</td><td rowspan=1 colspan=1>94.00</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>14.98</td><td rowspan=1 colspan=1>1.491</td><td rowspan=1 colspan=1>12.00</td><td rowspan=1 colspan=1>15.00</td><td rowspan=1 colspan=1>19.00</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>68.12</td><td rowspan=1 colspan=1>3.211</td><td rowspan=1 colspan=1>59.00</td><td rowspan=1 colspan=1>69.00</td><td rowspan=1 colspan=1>74.00</td></tr></table>

Overall performance characteristics (agreement, PPA, and NPA) at single site are summarized per day and per operator in Table 17 below.

Table 17. Within Site Precision per Sample Summarized by All Variables – Overall and by Day, Operator, and Bladder EpiScore Results   

<table><tr><td rowspan="2"></td><td rowspan="2"></td><td colspan="2">Overall Agreement</td><td colspan="2">NPA</td><td colspan="2">PPA</td></tr><tr><td>% (n/N)</td><td>Wilson Score 95% CI</td><td>% (n/N)</td><td>Wilson Score 95% CI</td><td>% (n/N)</td><td>Wilson Score 95% CI</td></tr><tr><td>Overall</td><td></td><td>99.00% (198/200)</td><td>[96.43%; 99.73%</td><td>99.00% (99/100)</td><td>[94.55%; 99.82%</td><td>99.00% (99/100)</td><td>[94.55%; 99.82%</td></tr><tr><td colspan="1" rowspan="5">Day</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">[91.24%;100.0%</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">[91.24%;100.0%</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">97.50%(39/40)</td><td colspan="1" rowspan="1">[87.12%;99.56%]</td><td colspan="1" rowspan="1">95.0%(19/20)</td><td colspan="1" rowspan="1">[76.4%;99.1%]</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">100.0%(40/40)</td><td colspan="1" rowspan="1">[91.24%;100.0%</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">97.50%(39/40)</td><td colspan="1" rowspan="1">[87.12%;99.56%</td><td colspan="1" rowspan="1">100.0%(20/20)</td><td colspan="1" rowspan="1">[83.9%;100.0%]</td><td colspan="1" rowspan="1">95.0%(19/20)</td><td colspan="1" rowspan="1">[76.4%;99.1%]</td></tr><tr><td colspan="1" rowspan="2">Operator</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">99.00%(99/100)</td><td colspan="1" rowspan="1">[94.55%;99.82%</td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[93.1%;100.0%]</td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[90.0%;99.9%]</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">99.00%(99/100)</td><td colspan="1" rowspan="1">[94.55%;99.82%</td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[90.0%;99.9%</td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[93.1%;100.0%]</td></tr><tr><td colspan="1" rowspan="4">EpiScoreValue</td><td colspan="1" rowspan="1">LowNegative</td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HighNegative</td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[91.5%;99.6%]</td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[91.5%;99.6%]</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LowPositive</td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[91.5%;99.6%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">98.0%(49/50)</td><td colspan="1" rowspan="1">[91.5%;99.6%]</td></tr><tr><td colspan="1" rowspan="1">HighPositive</td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100.0%(50/50)</td><td colspan="1" rowspan="1">[94.9%;100.0%]</td></tr></table>

# Lot-to-Lot/Instrument-to-Instrument

A lot-to-lot and instrument-to-instrument repeatability and reproducibility study was performed at a single site to assess the variability of Bladder EpiCheck when performed using 3 unique kit lots and 3 unique PCR instruments. Healthy donor urine pools were spiked with bladder cancerderived cell line material at different DNA concentration levels to create 4 test samples spanning the range of EpiScore values (LN, HN, LP, and HP). A minimum of 70 replicates were tested for each sample. Acceptance criteria was an overall level of agreement (positive/negative) greater than $90 \%$ with $9 5 \%$ confidence. The data demonstrated and overall agreement of $1 0 0 . 0 \%$ . Precision was also evaluated by EpiScore value, which further demonstrated the overall reproducibility of the Bladder EpiCheck assay. Overall agreement is summarized in Table 18 below, with EpiScore results shown in Table 19.

Table 18. Within Site Precision – Overall and by Lot, Instrument, and EpiScore Value   

<table><tr><td colspan="1" rowspan="1">Condition</td><td colspan="1" rowspan="1">% (n/N)</td><td colspan="1" rowspan="1">Wilson Score 95% CI</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">100.0% (294/294)</td><td colspan="1" rowspan="1">99.09%; 100.0%</td></tr><tr><td colspan="3" rowspan="1">Lot</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0% (175/175)</td><td colspan="1" rowspan="1">98.48%;100.0%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0% (59/59)</td><td colspan="1" rowspan="1">95.62%;100.0%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0% (60/60)</td><td colspan="1" rowspan="1">95.69%;100.0%</td></tr><tr><td colspan="3" rowspan="1">Instrument</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0% (214/214)</td><td colspan="1" rowspan="1">98.75%; 100.0%</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0% (40/40)</td><td colspan="1" rowspan="1">93.66%; 100.0%</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0% (40/40)</td><td colspan="1" rowspan="1">93.66%; 100.0%</td></tr><tr><td colspan="3" rowspan="1">EpiScore Value</td></tr><tr><td colspan="1" rowspan="1">Low Negative</td><td colspan="1" rowspan="1">100.0% (74/74)</td><td colspan="1" rowspan="1">96.47%; 100.0%</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">100.0% (75/75)</td><td colspan="1" rowspan="1">96.52%;100.0%</td></tr><tr><td colspan="1" rowspan="1">Low Positive</td><td colspan="1" rowspan="1">100.0% (70/70)</td><td colspan="1" rowspan="1">96.28%;100.0%</td></tr><tr><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">100.0% (75/75)</td><td colspan="1" rowspan="1">96.52%;100.0%</td></tr></table>

Table 19. Within Site Precision per Sample Presented as Bladder EpiScore Results   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=6>EpiScore</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=6>EY</td></tr><tr><td rowspan=1 colspan=1>LN</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>17.74</td><td rowspan=1 colspan=1>3.91</td><td rowspan=1 colspan=1>12.00</td><td rowspan=1 colspan=1>17.00</td><td rowspan=1 colspan=1>32.00</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>34.01</td><td rowspan=1 colspan=1>9.43</td><td rowspan=1 colspan=1>10.00</td><td rowspan=1 colspan=1>35.00</td><td rowspan=1 colspan=1>54.00</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>83.63</td><td rowspan=1 colspan=1>3.32</td><td rowspan=1 colspan=1>67.00</td><td rowspan=1 colspan=1>84.00</td><td rowspan=1 colspan=1>90.00</td></tr><tr><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>91.77</td><td rowspan=1 colspan=1>3.63</td><td rowspan=1 colspan=1>75.00</td><td rowspan=1 colspan=1>92.00</td><td rowspan=1 colspan=1>97.00</td></tr></table>

# 2. DNA Extraction Efficiency

A total of 47 clinical urine specimens, collected from bladder cancer patients and healthy donors, were used for this study. Each specimen was pelleted, split into $1 0 \mathrm { m L }$ urine equivalents, and DNA was extracted from each split specimen using 4 Bladder EpiCheck Extraction lots. Performance of the Bladder EpiCheck kits was evaluated by invalid rate and Bladder EpiCheck test performance and tested in the Bladder EpiCheck test using one ABI 7500 Fast Dx RealTime PCR system and analyzed using Bladder EpiCheck software (Ver. 1.9.19). Sample Failure and Agreement between lots are shown in Tables 20 and 21, respectively. In summary, a total of 1 sample failed after extraction, leading to an overall sample failure rate of $2 . 1 \%$ , and study results showed an overall, positive, and negative agreement of $100 \%$ .

Table 20. Sample Failure Rate   

<table><tr><td rowspan=1 colspan=1>Number of Samples</td><td rowspan=1 colspan=1>Failed Samples</td><td rowspan=1 colspan=1>Failure Rate</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.1%</td></tr></table>

Table 21. Agreement Between Extraction Lots   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Number ofSamples</td><td rowspan=1 colspan=1>OverallAgreement</td><td rowspan=1 colspan=1>Wilson Score90% CI</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>100.0% (46/46)</td><td rowspan=1 colspan=1>[96.6%; 100.0%]</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100.0% (24/24)</td><td rowspan=1 colspan=1>[93.6%; 100.0%]</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>100.0% (22/22)</td><td rowspan=1 colspan=1>[93.1%; 100.0%]</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Lot#A1074800/Lot#A1074700</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>100.0% (9/9)</td><td rowspan=1 colspan=1>[84.6%; 100.0%]</td></tr><tr><td rowspan=1 colspan=1>Lot#C0314500/Lot#B0702100</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>100.0% (37/37)</td><td rowspan=1 colspan=1>[95.7%; 100.0%]</td></tr></table>

# 3. Clinical Cut-off

The Bladder EpiCheck test software translates the marker score of the 15 Bladder EpiCheck markers into an integrated score called EpiScore, ranging from 0-100. An EpiScore result of 0- 59 represents a negative test result for bladder cancer recurrence, and an EpiScore result of 60- 100 represents a positive test result for bladder cancer recurrence.

# 4. Analytical Sensitivity

# Limit of Blank

A Limit of Blank study was conducted to determine the residual levels of non-specific background signal in the absence of template DNA. Sixty (60) replicates of no-template control (without DNA) were tested across 2 reagent lots – a total of 120 test results. Mean $\mathrm { C q }$ results were 44.75 and 44.86 for Lots 1 and 2, respectively. Using a non-parametric approach, the resulting LoB was determined to be a Cq value equal to 45.

# Functional LoD (fLoD)

The functional Limit of Detection (fLoD) refers to the lowest total DNA that is required for the assay and was determined by testing DNA extracted from positive clinical specimens mixed with DNA extracted from pooled healthy individuals (negative clinical specimens) to create a clinical specimen with an EpiScore near the clinical cutoff (Low Positive). This sample was tested across a range of 5 concentrations (two-fold dilutions) around and below the assay cut-off $\mathrm { 1 . 2 \ n g / w e l l }$ , $0 . 6 5 ~ \mathrm { { n g / w e l l } }$ , $0 . 3 0 \mathrm { n g / w e l l }$ , $0 . 1 5 ~ \mathrm { n g / w e l l } .$ , and $0 . 0 7 5 ~ \mathrm { { n g / w e l l } ) }$ . A blank sample (0 $\mathrm { n g / w e l l } )$ was also tested. Each concentration was tested in replicates of 30 using one lot of reagents and one instrument – a total of 330 test measurements. The fLoD was determined from a PROBIT regression model as the concentration at which, with $9 5 \%$ probability and $90 \%$ CI, the measurement result yielded a “positive” classification. Hit Rate and PROBIT results are summarized in Tables 22 and 23, respectively. Using this approach, the estimated fLoD was determined to be $0 . 1 8 6 \mathrm { n g / }$ well ( $2 . 2 3 { \mathrm { n g } } /$ sample).

Table 22. Hit Rate   

<table><tr><td rowspan=2 colspan=1>DNA Conc(ng/PCR well)</td><td rowspan=2 colspan=1>Detection(n/N)</td><td rowspan=2 colspan=1>Hit Rate%</td><td rowspan=1 colspan=2>90% Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>86.7</td><td rowspan=1 colspan=1>76.8%</td><td rowspan=1 colspan=1>92.7%</td></tr><tr><td rowspan=1 colspan=1>0.075</td><td rowspan=1 colspan=1>17/30</td><td rowspan=1 colspan=1>56.7</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>67.6%</td></tr></table>

Table 23. Probability Estimated from the PROBIT Model   

<table><tr><td colspan="1" rowspan="2">Probability</td><td colspan="1" rowspan="2">Sample Type</td><td colspan="1" rowspan="2">Concentration(ng/well)</td><td colspan="2" rowspan="1">90% Confidence Intervals</td></tr><tr><td colspan="1" rowspan="1">Lower</td><td colspan="1" rowspan="1">Upper</td></tr><tr><td colspan="1" rowspan="2">0.95</td><td colspan="1" rowspan="1">Clinical</td><td colspan="1" rowspan="1">0.186</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">0.98</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0.202</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.98</td></tr><tr><td colspan="1" rowspan="2">0.75</td><td colspan="1" rowspan="1">Clinical</td><td colspan="1" rowspan="1">0.104</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0.081</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">0.85</td></tr><tr><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">Clinical</td><td colspan="1" rowspan="1">0.070</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.65</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0.043</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.72</td></tr><tr><td colspan="1" rowspan="2">0.25</td><td colspan="1" rowspan="1">Clinical</td><td colspan="1" rowspan="1">0.047</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.47</td></tr><tr><td colspan="1" rowspan="1">Contrived</td><td colspan="1" rowspan="1">0.023</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.59</td></tr></table>

# Tumor Limit of Detection $( t L o D )$

The Tumor Limit of Detection (tLoD) refers to the minimum amount of tumor DNA in the lowest total volume of DNA that can be detected and was determined by testing DNA extracted from bladder cancer positive patient specimens and mixing with DNA extracted from healthy urine donor samples in a ratio gradient of 7 positive DNA spike-in levels $50 \%$ , $30 \%$ , $20 \%$ , $1 5 \%$ , $12 \%$ , $8 \%$ , and $4 \%$ ). These samples were tested using 2 reagent lots in 30 replicates total. Samples with $100 \%$ and $0 \%$ positive DNA spike-ins were also tested, but only in 10 replicates total. For each sample type, positive and negative DNA pools were mixed at different ratios to create a series of samples ranging from 0 to $100 \%$ tumor DNA. The total DNA concentration was kept constant at $1 - 2 \mathbf { x }$ the functional LoD (fLoD). The tLoD was calculated from a PROBIT regression model as the concentration at which with $9 5 \%$ probability the measurement result yields a “positive” classification. Hit rate and PROBIT estimation results are summarized in Tables 24 and 25, respectively. Based on this model, the tLoD was determined to be $7 . 5 \%$ , which equates to a minimum amount of tumor DNA in a clinical sample of ${ \sim } 0 . 1 7 \mathrm { n g }$ .

Table 24. Hit Rate   

<table><tr><td rowspan=2 colspan=1>Tumor DNAFraction (%)</td><td rowspan=2 colspan=1>Hit Rate (n/N)</td><td rowspan=2 colspan=1>Hit Rate (%)</td><td rowspan=1 colspan=2>90% Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>85.9%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.8%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>89.5%</td><td rowspan=1 colspan=1>99.0%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>11/30</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>26.4%</td><td rowspan=1 colspan=1>48.4%</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>14.1%</td></tr></table>

Table 25. Probability Estimated from the PROBIT Model   

<table><tr><td rowspan=2 colspan=1>Probability</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=2 colspan=1>Tumor DNAFraction (%)</td><td rowspan=1 colspan=2>90% Confidence Limits</td></tr><tr><td rowspan=1 colspan=1>Lower</td><td rowspan=1 colspan=1>Upper</td></tr><tr><td rowspan=2 colspan=1>0.95</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>99.0%</td></tr><tr><td rowspan=1 colspan=1>Contrived</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>89.0%</td><td rowspan=1 colspan=1>98.0%</td></tr></table>

Methylation Limit of Detection (MLoD)   
The Methylation Limit of Detection (mLoD) refers to the minimal detectable level of methylation across the assay’s target loci and was determined using a mixture of methylated and unmethylated artificial plasma DNA to create a gradient of different levels of methylation. This study took place across 2 phases, performed using contrived DNA samples. The MLoD was determined to be $0 . 3 5 \%$ .

In Study 1 (Phase I), a preliminary mLoD was established using a single PCR mix (Mix A, which consisted of biomarkers BE-1 and BE-2), as it represented the lower end of the

sensitivity range of the assay. A gradient of 10 admixtures ( $10 \%$ , $5 \%$ , $2 . 5 \%$ , $1 \%$ , $0 . 5 \%$ , $0 . 2 5 \%$ , $0 . 1 \%$ , $0 . 0 5 \%$ , $0 . 0 2 5 \%$ , and $0 . 0 1 \%$ ), along with a $0 \%$ methylation sample, were tested using one reagent lot and between 16 and 60 replicates, depending on the methylation level – a total of 378 total measurements. Undigested $100 \%$ methylated UC was also run as a positive control. Hit rate and PROBIT results for both biomarkers are summarized in Tables 26 and 27, respectively. Based on interpolation from the PROBIT estimation, the higher of the two mLoD values was used as the preliminary mLoD: $0 . 3 4 8 \%$ .

Table 26. Hit Rate with Upper One-Sided $9 5 \%$ Confidence Interval – per Marker and Methylation $\%$ (Phase I)   

<table><tr><td rowspan=1 colspan=1>Marker</td><td rowspan=1 colspan=1>Methylation %</td><td rowspan=1 colspan=1>Hit Rate %(n/N)</td><td rowspan=1 colspan=1>Upper One-Sided 95% CL</td></tr><tr><td rowspan=10 colspan=1>BE-1</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>10.00% (3/30)</td><td rowspan=1 colspan=1>25.62%</td></tr><tr><td rowspan=1 colspan=1>0.025</td><td rowspan=1 colspan=1>10.00% (3/30)</td><td rowspan=1 colspan=1>25.62%</td></tr><tr><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>6.67% (4/60)</td><td rowspan=1 colspan=1>15.93%</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>40.00%(24/60)</td><td rowspan=1 colspan=1>52.63%</td></tr><tr><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>65.00%(39/60)</td><td rowspan=1 colspan=1>75.83%</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100.0%(16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100.0% (16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100.0% (16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=10 colspan=1>BE-2</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>10.00% (3/30)</td><td rowspan=1 colspan=1>25.62%</td></tr><tr><td rowspan=1 colspan=1>0.025</td><td rowspan=1 colspan=1>76.67%(23/30)</td><td rowspan=1 colspan=1>88.21%</td></tr><tr><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>78.33%(47/60)</td><td rowspan=1 colspan=1>86.88%</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>98.33%(59/60)</td><td rowspan=1 colspan=1>99.71%</td></tr><tr><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>98.33%(59/60)</td><td rowspan=1 colspan=1>99.71%</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0%(30/30)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>100.0% (16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100.0% (16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100.0% (16/16)</td><td rowspan=1 colspan=1>100.00%</td></tr></table>

Table 27. Probability Estimated from PROBIT Model   

<table><tr><td rowspan=1 colspan=1>Biomarker</td><td rowspan=1 colspan=1>Probability</td><td rowspan=1 colspan=1>Methylation%</td><td rowspan=1 colspan=1>95% FiducialLimits</td></tr><tr><td rowspan=1 colspan=1>BE-1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.34872</td><td rowspan=1 colspan=1>0.30102; 0.42287</td></tr><tr><td rowspan=1 colspan=1>BE-2</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.06681</td><td rowspan=1 colspan=1>0.05337; 0.09068</td></tr></table>

In Study 2 (Phase II), the preliminary mLoD value determined in Phase I, was confirmed for the other 13 biomarkers in the assay using a subset of the admixture gradients used in Phase I (3

methylation levels around the preliminary LoD: $0 . 0 5 \%$ , $0 . 1 \%$ , and $0 . 2 5 \%$ ) tested using one reagent lot and 60 replicates per methylation level. Undigested $100 \%$ methylated UC was also run as a positive control. Hit rate results for the other 13 biomarkers are summarized in Table 28 below. Based on the results, biomarkers BE-4 through BE-15 had hit rates of $1 0 0 . 0 \%$ at values below the preliminary mLoD. The one exception, BE-3, underwent a follow-up root analysis, and preparation of mix B (containing biomarkers BE-3 and BE-4) was repeated across the entire range of tested ratios from Phase II. Results from the repeat preparation showed acceptable hit rates below the preliminary mLoD.

Table 28. Hit Rate with Upper One-Sided $9 5 \%$ Confidence Limit – Per Marker and per Methylation $\%$ (Phase II)   

<table><tr><td colspan="1" rowspan="1">MctlyTaclon 70 (1 H</td><td colspan="1" rowspan="1"></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Marker</td><td colspan="1" rowspan="1">Methylation%</td><td colspan="1" rowspan="1">Hit Rate % (n/N)</td><td colspan="1" rowspan="1">Upper One-Sided 95% CL</td></tr><tr><td colspan="1" rowspan="3">BE-3</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">39.17% (47/120)</td><td colspan="1" rowspan="1">46.66%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">39.17%(47/120)</td><td colspan="1" rowspan="1">46.66%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">56.90%(66/116)</td><td colspan="1" rowspan="1">64.22%</td></tr><tr><td colspan="1" rowspan="3">BE-4</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">53.33%(32/60)</td><td colspan="1" rowspan="1">63.55%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">78.33% (47/60)</td><td colspan="1" rowspan="1">85.76%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">98.21%(55/56)</td><td colspan="1" rowspan="1">99.60%</td></tr><tr><td colspan="1" rowspan="3">BE-5</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">35.00% (21/60)</td><td colspan="1" rowspan="1">45.58%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">80.00% (48/60)</td><td colspan="1" rowspan="1">87.11%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">96.43%(54/56)</td><td colspan="1" rowspan="1">98.81%</td></tr><tr><td colspan="1" rowspan="3">BE-6</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">47.50%(57/120)</td><td colspan="1" rowspan="1">54.97%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">75.83%(91/120)</td><td colspan="1" rowspan="1">81.65%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">92.24%(107/116)</td><td colspan="1" rowspan="1">95.43%</td></tr><tr><td colspan="1" rowspan="3">BE-7</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">50.00% (30/60)</td><td colspan="1" rowspan="1">60.39%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">70.00% (42/60)</td><td colspan="1" rowspan="1">78.69%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">98.21% (55/56)</td><td colspan="1" rowspan="1">99.60%</td></tr><tr><td colspan="1" rowspan="3">BE-8</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">46.67%(28/60)</td><td colspan="1" rowspan="1">57.17%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">70.00% (42/60)</td><td colspan="1" rowspan="1">78.69%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">94.44% (51/54)</td><td colspan="1" rowspan="1">97.76%</td></tr><tr><td colspan="1" rowspan="3">BE-9</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">10.00% (6/60)</td><td colspan="1" rowspan="1">18.19%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">36.67%(22/60)</td><td colspan="1" rowspan="1">47.27%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">91.07%(51/56)</td><td colspan="1" rowspan="1">95.59%</td></tr><tr><td colspan="1" rowspan="3">BE-10</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">26.67%(16/60)</td><td colspan="1" rowspan="1">36.91%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">41.67%(25/60)</td><td colspan="1" rowspan="1">52.27%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">87.50% (49/56)</td><td colspan="1" rowspan="1">93.08%</td></tr><tr><td colspan="1" rowspan="3">BE-11</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">71.67% (41/60)</td><td colspan="1" rowspan="1">80.14%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">91.67% (55/60)</td><td colspan="1" rowspan="1">95.86%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">100.0% (55/55)</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="3">BE-12</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">48.33% (29/60)</td><td colspan="1" rowspan="1">58.79%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">88.33% (53/60)</td><td colspan="1" rowspan="1">93.55%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">100.0%(56/56)</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="3">BE-13</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">65.00% (39/60)</td><td colspan="1" rowspan="1">74.28%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">93.33% (56/60)</td><td colspan="1" rowspan="1">96.97%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">100.0% (56/56)</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="3">BE-14</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">50.85%(30/59)</td><td colspan="1" rowspan="1">61.28%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">73.33% (44/60)</td><td colspan="1" rowspan="1">81.57%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">100.0% (56/56)</td><td colspan="1" rowspan="1">100.00%</td></tr><tr><td colspan="1" rowspan="3">BE-15</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">48.28%(28/58)</td><td colspan="1" rowspan="1">58.90%</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">81.67% (49/60)</td><td colspan="1" rowspan="1">88.45%</td></tr><tr><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">96.43% (54/56)</td><td colspan="1" rowspan="1">98.81%</td></tr></table>

# 5. Linearity/Assay Reportable Range

Not applicable

# 6. Traceability (control, calibrators, or methods):

The Bladder EpiCheck test is not traceable to any known standard. Controls and quality metrics are described in the “Device Description” section.

# 7. Stability

# Real-Time (Shelf-Life) and In-Use

In-use and real-time stability testing of 3 lots of the Bladder EpiCheck kit stored at the recommended storage condition of $- 2 0 ^ { \circ } \mathrm { C } / 2 { - } 8 ^ { \circ } \mathrm { C } / 2 5 ^ { \circ } \mathrm { C }$ (depending on the reagents storage specifications) were conducted at six separate time points (baseline, 90 days, 180 days, 365 days, 440 days, and 540 days). Baseline for each lot was performed within 1 month of the kit manufacturing date and served as a reference for the subsequent time points for both in-use and real-time stability protocols. Each lot was tested using 4 contrived urine pools (T24 cell line DNA spiked into DNA extracted from healthy volunteer urine pools) representing the full range of EpiScore values (LN, HN, LP, and HP) tested in 7 replicates at each timepoint. For the inuse stability assessment, during each time point, all kits from the same lot $\scriptstyle ( \ n = 3 )$ underwent the same freeze/thaw and handling procedures. For real-time stability assessment, at each time point, “fresh” (unused) Extraction and test kits were used. Each “run” included the Undigested (UC) and Non-Template (NTC) Controls. For $\mathrm { T } { = } 0$ , a total of 84 samples were tested (4 sample pools tested in each of 3 kit lots in replicates of 7). For $\mathrm { T } { = } 1$ through $\mathrm { T } { = } 5$ , a total of 168 samples were testes (4 samples pools tested in each of $3 \mathrm { k i t }$ lots in replicates of 7 spanning both studies). Results showed that there was no significant change in performance when testing kits and controls up to 486 days. Overall agreement was $9 9 . 6 8 \%$ $[ 9 9 . 1 9 \%$ ; $9 9 . 8 7 \% ]$ ), thus supporting the stability claim of 486 days.

# Freeze-Thaw

Freeze/Thaw stability testing of three lots of the Bladder EpiCheck Test kit stored at the recommended storage condition of $- 2 0 \mathrm { { } ^ { \circ } C }$ was conducted at three separate test points $\mathrm { T } = 1$ , 8 and 12 cycles). Each lot was tested using 4 contrived urine pools (T24 cell line DNA spiked into DNA extracted from healthy volunteer urine pools) representing the full range of EpiScore values (LN, HN, LP, and HP) with 5 replicates per sample. Results showed no significant performance changes and low variability in EpiScore value between the 3 timepoints. The stability claim of 10 cycles was then confirmed in a supplementary study using 4 clinical samples representing the full range of EpiScore values (LN, HN, LP, and HP) with 5 replicates per sample over 2 timepoints (1 cycle and 12 cycles).

# Shipping

Twelve (12) Extraction Kit boxes and 12 Test kit boxes were used to evaluate the packing tolerance due to transportation. From the 12 boxes, two boxes were used to evaluate the functionality of the kits (i.e., extracting DNA and prepare the DNA for the PCR analysis) and to test the performance of the kit (quantitative and qualitative results) across a variety of simulated transportation conditions (changes in temperature for 24 hours, vibration, pressure, and drop tolerance). Simulated conditions and durations are listed in Table 29. In addition, a single control box that did not go through any transportation simulation was used as control. The performance of the kit was tested on urine samples collected from 8 subjects (4 healthy and 4 Bladder Cancer). Each sample was tested in five replicates by the kits underwent the transportation analysis $( \mathrm { n } { = } 4 0 )$ and in duplicate by the control kit $\mathrm { ( n = 1 6 ) }$ . Overall, Positive, and Negative agreement were $100 \%$ .

Table 29. Simulated Transport Conditions and Duration   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Temperature (°C)/Humidity(%)</td><td rowspan=1 colspan=1>ConditionDetails/Duration</td></tr><tr><td rowspan=1 colspan=1>Conditioning  Series I</td><td rowspan=1 colspan=1>50 ± 2 / 15 ± 5</td><td rowspan=1 colspan=1>Desert/24 hours</td></tr><tr><td rowspan=1 colspan=1>Drop Test  Series I</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Performed drop sequencesfrom pre-determinedheight based onweight/Not applicable</td></tr><tr><td rowspan=1 colspan=1>Vehicle Stacking</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Stacking of boxes duringtransportation/Notapplicable</td></tr><tr><td rowspan=1 colspan=1>Loose Load Vibration</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Transportation Vibrationtesting/1 hour</td></tr><tr><td rowspan=1 colspan=1>Conditioning  Series II</td><td rowspan=1 colspan=1>-30 ± 2 / N/A</td><td rowspan=1 colspan=1>Extreme Cold/24 hours</td></tr><tr><td rowspan=1 colspan=1>High Altitude</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Low Pressure/1 hour</td></tr><tr><td rowspan=1 colspan=1>Vehicle Vibration</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Random vibrationtesting/20-40 minutes peraxis</td></tr><tr><td rowspan=1 colspan=1>Conditioning  Series III</td><td rowspan=1 colspan=1>20 ± 2 / 90 ± 5</td><td rowspan=1 colspan=1>High Humidity/24 hours</td></tr><tr><td rowspan=1 colspan=1>Drop Test  Series III</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Performed drop sequencesfrom pre-determinedheight based onweight/Not applicable</td></tr></table>

# 8. Analytical Specificity

Digestion Restriction Efficiency

Digestion restriction efficiency was established by evaluating the digestion of all 15 Bladder EpiCheck markers using $100 \%$ unmethylated synthetic (plasmid) DNA tested across a series of dilution levels for a total of 30 replicates based on the relationship of the $\Delta \mathrm { C q }$ between the nondigestible IR marker ( $100 \%$ copies in the reaction) and digested targets (BE markers 1-15). Digestion efficiency for all 15 markers was shown to exceed $9 9 . 9 \%$ , with an average efficiency of $9 9 . 9 9 7 \%$ .

An additional study was performed to assess the impact of hemi-methylated molecules have on the enzymatic digestion step, as well as the contamination risk, in the Bladder EpiCheck assay protocol performed using synthetic DNA molecules of a representative locus (BE-14) containing all different permutations of the targeted cut site. Results showed that hemimethylated DNA does represent a digestion efficiency profile similar to unmethylated DNA; however, at best, this reflects a very small sub-population of target molecules, so the effect should be negligible.

# Digestion Restriction Efficiency – Exclusivity

Digestion restriction efficiency (Exclusivity) was established by evaluating the digestion of nonmethylated markers to avoid false positive results using a commercially available hypomethylated human genomic DNA sample $( < 5 \%$ average global methylation), which was added to the digestion reaction at an extreme excess (2.5ug/reaction), and tested in 10 replicates (with 6 replicates of a control reaction with no digestion enzyme to reflect the full amplification between test and control markers). The efficacy of the digestion was calculated per marker, by the $\Delta \mathrm { C q }$ between the digested (test) and non-digested (control) markers. The digestion efficiency was greater than $9 9 \%$ for 9 of the 15 biomarkers, ensuring that no non-specific signal is generated in the qPCR step.

A supplementary meta-analysis was conducted to demonstrate that each of the 15 Bladder EpiCheck markers can be digested at or above the $9 9 . 9 \%$ level. Raw data from 2,300 clinical specimens from US and European cohorts (including a wide range of methylation levels at all 15 loci) was analyzed, and max digestion was determined for each biomarker. The max digestion percentage was ${ > } 9 9 . 9 9 \%$ for all 15 biomarkers, thus further ensuring that no nonspecific signal is generated in the qPCR step.

# Primer Probe Specificity

In-Silico PCR analysis for each locus (Informative markers BE-1 to BE-15 and the Internal Reference (IR)) obtained a single hit using the forward and reverse primer sequences. Additionally, confirmatory PCR was performed using each primer pair, followed by gel electrophoresis. This showed only one product amplified per primer pair, this confirming the in-silico analysis.

# Interference

Performance of Bladder EpiCheck was evaluated with 27 potentially interfering substances (including common urine constituents, microbial contaminants, therapeutic agents, and

laboratory preservatives) tested across 3 contrived sample pools (2 negative pools from healthy individuals, and one positive pool of healthy urine spiked with bladder cancer cell-line material). Each substance was tested at 2 concentrations, with each test run in 3 replicates (18 total replicates for negative sample pools, 9 total replicates for positive sample pool). Any invalid or failed samples were repeated. Preliminary results (Table 30) suggested that uric acid and ampicillin were both potential interferents, so a dose response analysis was conducted, which showed that interference either did not repeat itself or was present at elevated levels that are not clinically meaningful. Therefore, no evidence of interference caused by the substances tested at clinically relevant physiological ranges exists.

Table 30. Distribution of EpiCheck Result – per Substance   

<table><tr><td colspan="1" rowspan="3"></td><td colspan="6" rowspan="1">EpiCheck Result</td></tr><tr><td colspan="2" rowspan="1">Invalid</td><td colspan="2" rowspan="1">Negative</td><td colspan="2" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">n/N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n/N</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n/N</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="1" rowspan="1">InterferingSubstance</td><td colspan="6" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acetaminophen</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Ascorbic Acid</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">BCG*</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6/6</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Bilirubin(conjugated)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Caffeine</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">C. albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Doxorubicin-HC1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">E. Coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Ehtanol</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Hemaglobin</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Mytomycin C</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Nicotine</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Nitrofurantion</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Norgenpreservative</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Phenazopyridine-HCl</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Protein IgG</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Red Blood Cells(RBC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Sodium Chloride(NaCl)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Thiotepa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Trimethoprin</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">White BloodCells (WBC)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">S. saprophyticus</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">S. epidermis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18/18</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Ampicillin</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">16/18</td><td colspan="1" rowspan="1">88.9%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">Uric Acid</td><td colspan="1" rowspan="1">6/18</td><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">12/18</td><td colspan="1" rowspan="1">66.7%</td><td colspan="1" rowspan="1">9/9</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

# 9. Specimen Stability

Thirty (30) Clinical specimens, collected from 15 bladder cancer patients and 15 normal donors across the full range of EpiScore values, were divided into 3 parts and used to evaluate the stability of fresh urine, urine pellet, and extracted DNA and digested DNA. Samples ranged from low negative through high positive. Each sample was tested immediately and after storage. Each clinical sample was divided into different aliquots and tested at 3 timepoints depending on the specimen type (Fresh urine: 0, 5, and 6 days; Pelleted urine: 0, 14, and 21 days; DNA: 0, 30, and 60 days) after storage at the recommended conditions. Agreement results are shown in Tables 31, 32, and 33 for urine, pelleted urine, and DNA, respectively. The results confirm a stability claim of 5 days for fresh urine at $2 { - } 8 ^ { \circ } \mathrm { C }$ , urine pellets for 19 days at $- 2 0 \mathrm { { ^ { o } C } }$ , and DNA for 30 days at $- 2 0 \mathrm { { ^ \circ C } }$ .

Two supplementary studies were conducted to assess the stability of digested DNA using residual clinical specimens stored for $> 1$ year (Study 1, $n { = } 2 5$ ) and clinical specimens stored for 30 days prior to testing (Study 2, $\scriptstyle \mathrm { n = 4 8 }$ ). Agreement results are shown in Table 34. The results showed an overall agreement of $100 \%$ for both sets of samples, confirming a frozen stability claim of 30 days for digested DNA.

Table 31. Bladder EpiCheck Results, Overall and by Timepoint – Fresh Urine   

<table><tr><td rowspan=2 colspan=1>Timepoint</td><td rowspan=1 colspan=2>EpiCheck Result</td></tr><tr><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>Wilson Score 95% CI</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>99.01% (100/101)</td><td rowspan=1 colspan=1>95.68%; 99.78%</td></tr><tr><td rowspan=1 colspan=1>0 Days (To)</td><td rowspan=1 colspan=1>100% (34/34)</td><td rowspan=1 colspan=1>92.63%; 100%</td></tr><tr><td rowspan=1 colspan=1>5 Days (T1)</td><td rowspan=1 colspan=1>100% (33/33)</td><td rowspan=1 colspan=1>92.42%; 100%</td></tr><tr><td rowspan=1 colspan=1>6 Days (T2)</td><td rowspan=1 colspan=1>97.06% (33/34)</td><td rowspan=1 colspan=1>87.84%; 99.34%</td></tr></table>

Table 32. Bladder EpiCheck Results, Overall and by Timepoint – Urine Pellet   

<table><tr><td rowspan=2 colspan=1>Timepoint</td><td rowspan=1 colspan=2>EpiCheck Result</td></tr><tr><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>Wilson Score 95%CI</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100.0% (90/90)</td><td rowspan=1 colspan=1>97.08%;100.0%</td></tr><tr><td rowspan=1 colspan=1>0 Days (To)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>91.73%;100.0%</td></tr><tr><td rowspan=1 colspan=1>14 Days (T1)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>91.73%;100.0%</td></tr><tr><td rowspan=1 colspan=1>21 Days (T2)</td><td rowspan=1 colspan=1>100.0% (30/30)</td><td rowspan=1 colspan=1>91.73%;100.0%</td></tr></table>

Table 33. Bladder EpiCheck Results, Overall and by Timepoint - Extracted DNA   

<table><tr><td colspan="1" rowspan="2">Timepoint</td><td colspan="2" rowspan="1">EpiCheck Result</td></tr><tr><td colspan="1" rowspan="1">% (n/N)</td><td colspan="1" rowspan="1">Wilson Score 95%CI</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">98.25% (112/114)</td><td colspan="1" rowspan="1">94.84%; 99.42%</td></tr><tr><td colspan="1" rowspan="1">0 Days (To)</td><td colspan="1" rowspan="1">100.0% (38/38)</td><td colspan="1" rowspan="1">93.35%; 100.0%</td></tr><tr><td colspan="1" rowspan="1">30 Days (T1)</td><td colspan="1" rowspan="1">100.0% (38/38)</td><td colspan="1" rowspan="1">93.35%; 100.0%</td></tr><tr><td colspan="1" rowspan="1">60 Days (T2)</td><td colspan="1" rowspan="1">94.87% (36/38)</td><td colspan="1" rowspan="1">85.28%; 98.24%</td></tr></table>

Table 34. Bladder EpiCheck versus True Status – Overall and by Timepoint   

<table><tr><td rowspan=1 colspan=1>Study</td><td rowspan=1 colspan=1>Timepoint</td><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>Wilson Score 95% CI</td></tr><tr><td rowspan=3 colspan=1>1</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100.0%(50/50)</td><td rowspan=1 colspan=1>94.87%; 100%</td></tr><tr><td rowspan=1 colspan=1>T = 0</td><td rowspan=1 colspan=1>100.0%(25/25)</td><td rowspan=1 colspan=1>90.23%; 100%</td></tr><tr><td rowspan=1 colspan=1>T &gt; 1 year</td><td rowspan=1 colspan=1>100.0%(25/25)</td><td rowspan=1 colspan=1>90.23%; 100%</td></tr><tr><td rowspan=3 colspan=1>2</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>100.0%(96/96)</td><td rowspan=1 colspan=1>97.3%; 100%</td></tr><tr><td rowspan=1 colspan=1>T= 0</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>97.4%; 100%</td></tr><tr><td rowspan=1 colspan=1>T &gt; 30 days</td><td rowspan=1 colspan=1>100.0%(48/48)</td><td rowspan=1 colspan=1>97.4%; 100.0%</td></tr></table>

# 10. Robustness

Robustness of the Bladder EpiCheck was evaluated using four (4) contrived specimens (T24 cell line DNA spiked into DNA extracted from healthy volunteer urine pools) representing the full range of EpiScore values (LN, HN, LP, and HP). Each sample was run in quintuplicates across a series of 13 conditions with to variations in the optimized PCR run conditions was evaluated under different assay conditions. The conditions deviated from the recommended condition specified in the Instructions for Use into either digestion or PCR stage including PCR denaturation and annealing temperatures, digestion temperature, and/or digestion time. Four clinical DNA sample pools, representing different EpiScore values, were analyzed in each condition. Overall agreement was shown to be $9 8 . 5 9 9 \%$ $9 6 . 7 7 \%$ ; $9 9 . 3 1 \% ]$ .

A supplementary study was conducted to further demonstrate the robustness of the Bladder EpiCheck assay using 4 clinical specimens (DNA extracted from clinical specimens adjusted using DNA extracted from healthy urine donor pools) that represent the full range of EpiScore values (LN, HN, LP, and HP). Each sample was tested in quintuplicates across 5 different assay conditions (including the control condition). Overall agreement was shown to be $9 9 . 3 \%$ $[ 9 6 . 9 \% ; 9 9 . 8 \% ]$ , thus confirming the robustness of the assay.

# B. Clinical Performance

# 1. Clinical Cutoff (Training and Feasibility Data)

The assay cut-off and proprietary EpiScore algorithm for the Bladder EpiCheck software were validated on a set of urine samples collected from control patients with a history of bladder cancer and bladder cancer positive patients confirmed by cystoscopy and pathology. The total sample size for the software algorithm development included 178 samples, which were collected in two separate sets. The first set of samples served for the cut-off definition $\scriptstyle ( 1 = 1 0 9$ , of which 40 were collected from control patients with a history of bladder cancer and 69 from Urothelial Cell Carcinoma (UCC) positive patients confirmed by pathology). The second sample set served as the validation of the cut-off $\scriptstyle ( n = 6 7$ , of which 51 were collected from control patients with a history of bladder cancer and 16 from UCC positive patients confirmed by pathology). The derived Bladder EpiCheck algorithm sensitivity and specificity compared to pathology outcome demonstrated a sensitivity of $94 \%$ for UCC recurrence and specificity of $84 \%$ .

# 2. Method Comparison

A multi-center, prospective, IRB-approved longitudinal study was conducted at 11 academic and urology specialty medical centers in the U.S. and Canada to establish the performance of Bladder EpiCheck as compared to the predicate device (UroVysion Bladder Cancer Recurrence Kit). Performance of the Bladder EpiCheck was also assessed against a “gold standard” (GS) derived using both clinical cytology and combined cystoscopy/pathology data.

A total of 674 subjects were enrolled in the study. Demographic and disease history for enrolled subjects are listed in Table 35. Eligible subjects included any male or female subject aged 22 and above, diagnosed with incident or recurrent UCC and undergoing surveillance for bladder cancer recurrence, who had urothelial cell carcinoma tumor resected within 12 months from baseline visit, and had a plan for cystoscopy surveillance. The study included up to 3 visits: baseline, and up to 2 additional surveillance visits, where voided urine specimens were collected. In total, voided urine specimens were collected from 583 subjects. For subjects who experienced a recurrence (as determined by positive cytology/pathology), the first positive visit was used (i.e., the visit at which the diagnosis of recurrence was established). For the nonrecurring subjects, the last negative visit was used for subjects with multiple surveillance visits.

Table 35. Demographic and Disease History for Enrolled Subjects   

<table><tr><td colspan="2" rowspan="1">Demographic</td></tr><tr><td colspan="2" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">Range (years)</td><td colspan="1" rowspan="1">38-93</td></tr><tr><td colspan="1" rowspan="1">Average (years)</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="2" rowspan="1">Sex (N/%)</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">531 (78.8%)</td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">140 (20.8)</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">3 (0.4%)</td></tr><tr><td colspan="2" rowspan="1">Ethnicity (N/%)</td></tr><tr><td colspan="1" rowspan="1">Not Hispanic or Latino</td><td colspan="1" rowspan="1">633 (93.9%)</td></tr><tr><td colspan="1" rowspan="1">Hispanic or Latino</td><td colspan="1" rowspan="1">34 (5.0%)</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">7 (1.0%)</td></tr><tr><td colspan="1" rowspan="1">Race (N/%)</td><td colspan="1" rowspan="1">N/%)</td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">616 (91.4%)</td></tr><tr><td colspan="1" rowspan="1">Black</td><td colspan="1" rowspan="1">24 (3.6%)</td></tr><tr><td colspan="1" rowspan="1">Asian</td><td colspan="1" rowspan="1">21 (3.1%)</td></tr><tr><td colspan="1" rowspan="1">American Indian or Alaska Native</td><td colspan="1" rowspan="1">5 (0.7%)</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">8 (1.2%)</td></tr><tr><td></td><td></td></tr><tr><td colspan="2" rowspan="1">Disease History</td></tr><tr><td colspan="2" rowspan="1">History of Recurrence (N/%)</td></tr><tr><td colspan="1" rowspan="1">Primary</td><td colspan="1" rowspan="1">355 (52.7%)</td></tr><tr><td colspan="1" rowspan="1">Recurrent</td><td colspan="1" rowspan="1">318 (47.2%)</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">1 (0.1%)</td></tr><tr><td colspan="2" rowspan="1">Grade History (N/%)</td></tr><tr><td colspan="1" rowspan="1">History of Low Grade/PUNLMP</td><td colspan="1" rowspan="1">253 (37.5%)</td></tr><tr><td colspan="1" rowspan="1">History of High Grade</td><td colspan="1" rowspan="1">418 (62.0%)</td></tr><tr><td colspan="1" rowspan="1">Not Reported</td><td colspan="1" rowspan="1">3 (0.4%)</td></tr><tr><td colspan="2" rowspan="1">Instillation Treatment (N/%)</td></tr><tr><td colspan="1" rowspan="1">&lt;90 Days from Last Instillation Treatment</td><td colspan="1" rowspan="1">264 (39.1%)</td></tr><tr><td colspan="1" rowspan="1">&gt;90 Days from Last Instillation Treatment</td><td colspan="1" rowspan="1">141 (20.9%)</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">8 (1.19%)</td></tr><tr><td colspan="1" rowspan="1">No Instillation Treatment</td><td colspan="1" rowspan="1">261 (38.7%)</td></tr></table>

DNA extracted from urine specimens was tested using the Bladder EpiCheck device and compared to the GS. Classification by the GS was as follows: a subject was considered positive if the interpretation for either cytology or the combined cystoscopy/pathology results were positive, and a subject was considered negative if both cytology and the combined cystoscopy/pathology results were negative. Equivocal cytology was also considered negative. A detailed description of how GS status was determined can be found in Table 36. In total, valid Bladder EpiCheck and GS results were obtained from 449 samples. In summary, the Bladder EpiCheck test performed with an overall accuracy of $7 8 . 8 \%$ $9 5 \%$ CI $[ 7 4 . 8 \%$ ; $8 2 . 4 \% ] $ ), a sensitivity of $6 6 . 7 \%$ $( [ 5 8 . 4 \% ; 7 4 . 0 \% ] )$ , and a specificity of $8 4 . 2 \%$ ( $[ 7 9 . 8 \%$ ; $8 7 . 9 \% ]$ ). Further details of the performance characteristics of the Bladder EpiCheck test compared to the established GS is found in Table 37. Additionally, performance of the Bladder EpiCheck test was evaluated in several subgroup analyses shown in Tables 38 (Cancer Grade) and 39 (Cancer Stage), whose results were comparable to the overall performance characteristics.

Table 36. Criteria Used to Determine Gold Standard (GS) Status   

<table><tr><td rowspan=1 colspan=1>Criteria</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=2>Cytology</td></tr><tr><td rowspan=1 colspan=1>Negative or Equivocal</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>Combined Cystoscopy/Pathology</td></tr><tr><td rowspan=1 colspan=1>Negative or Equivocal Cystoscopy;Negative, Missing, or Equivocal Pathology</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive Cystoscopy; Negative orEquivocal Pathology</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive, Negative, or EquivocalCystoscopy; Positive Pathology</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Final GS Result</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>Interpretation for either above criterion ispositive</td><td rowspan=1 colspan=1>GS Positive</td></tr><tr><td rowspan=1 colspan=1>Interpretation for both above criteria arenegative</td><td rowspan=1 colspan=1>GS Negative</td></tr></table>

Table 37. Bladder EpiCheck Performance vs. Gold Standard   

<table><tr><td colspan="1" rowspan="3"></td><td colspan="3" rowspan="1">Gold Standard Result</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Total GS</td></tr><tr><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">EpiCheck Result</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">262</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">308</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="2" rowspan="1">49</td><td colspan="2" rowspan="1">92</td><td colspan="1" rowspan="1">141</td></tr><tr><td colspan="1" rowspan="1">Total EpiCheck</td><td colspan="2" rowspan="1">311</td><td colspan="2" rowspan="1">138</td><td colspan="1" rowspan="1">449</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="2" rowspan="1">Metric</td><td colspan="2" rowspan="1">% (n/N)</td><td colspan="2" rowspan="1">95% CI</td></tr><tr><td colspan="2" rowspan="1">Accuracy</td><td colspan="2" rowspan="1">78.8% (354/449)</td><td colspan="2" rowspan="1">74.8%; 82.4%</td></tr><tr><td colspan="2" rowspan="1">Sensitivity</td><td colspan="2" rowspan="1">66.7% (92/138)</td><td colspan="2" rowspan="1">58.4%; 74.0%</td></tr><tr><td colspan="2" rowspan="1">Specificity</td><td colspan="2" rowspan="1">84.2% (262/311)</td><td colspan="2" rowspan="1">79.8%; 87.9%</td></tr><tr><td colspan="2" rowspan="1">PPV</td><td colspan="2" rowspan="1">65.3% (92/141)</td><td colspan="2" rowspan="1">57.1%; 72.6%</td></tr><tr><td colspan="2" rowspan="1">NPV</td><td colspan="2" rowspan="1">85.1% (262/308)</td><td colspan="2" rowspan="1">80.7%; 88.6%</td></tr></table>

Table 38. Bladder EpiCheck Sensitivity and NPV vs. Gold Standard by Grade   

<table><tr><td rowspan=2 colspan=1>Grade</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=2>Bladder EpiCheck Result</td></tr><tr><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>High Grade</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>76.6% (49/64)</td><td rowspan=1 colspan=1>64.9%; 85.3%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>95.1% (293/308)</td><td rowspan=1 colspan=1>92.1%; 97.0%</td></tr><tr><td rowspan=2 colspan=1>Low Grade</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>46.5% (20/43)</td><td rowspan=1 colspan=1>32.5%; 61.1%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>92.5% (285/308)</td><td rowspan=1 colspan=1>89.0%; 95.0%</td></tr><tr><td rowspan=2 colspan=1>No Grade</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>74.2% (23/31)</td><td rowspan=1 colspan=1>56.8%; 86.3%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>97.4% (300/308)</td><td rowspan=1 colspan=1>95.0%; 98.7%</td></tr></table>

Table 39. Bladder EpiCheck Performance vs. Gold Standard by Stage   

<table><tr><td rowspan=2 colspan=1>Stage</td><td rowspan=2 colspan=1>Metric</td><td rowspan=1 colspan=2>Bladder EpiCheck Result</td></tr><tr><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>CIS</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>76.9% (10/13)</td><td rowspan=1 colspan=1>49.7%; 91.8%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>99.0% (305/308)</td><td rowspan=1 colspan=1>97.2%; 99.7%</td></tr><tr><td rowspan=2 colspan=1>T1</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>63.6% (7/11)</td><td rowspan=1 colspan=1>35.4%; 84.8%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>98.1% (304/308)</td><td rowspan=1 colspan=1>96.7%; 99.5%</td></tr><tr><td rowspan=2 colspan=1>T2</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>33.3% (1/3)</td><td rowspan=1 colspan=1>6.2%; 79.2%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>99.4% (306/308)</td><td rowspan=1 colspan=1>97.7%; 99.8%</td></tr><tr><td rowspan=2 colspan=1>Ta</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>58.5% (38/65)</td><td rowspan=1 colspan=1>46.3%; 69.6%</td></tr><tr><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>91.2% (281/308)</td><td rowspan=1 colspan=1>87.6%; 93.9%</td></tr></table>

# 3. Comparison to UroVysion

DNA extracted from urine specimens was tested using the Bladder EpiCheck device and compared to the predicate device (UroVysion Bladder Cancer Recurrence Kit) with respect to the GS sample definition. In total, valid Bladder EpiCheck, UroVysion, and GS results were obtained from 352 samples. In summary, the Bladder EpiCheck was slightly more sensitive $( + 4 . 8 2 \% )$ and slightly less specific $( - 2 . 9 7 \% )$ than the predicate device; thus, the Bladder EpiCheck is substantially equivalent to the predicate device. Further details of the performance characteristics between the Bladder EpiCheck test and the predicate device are listed in Table 40. An additional analysis was performed by cancer grade (Table 41).

Table 40. Bladder EpiCheck and UroVysion vs. Gold Standard – Matched Cases   

<table><tr><td rowspan="2"></td><td colspan="3">Bladder EpiCheck Result</td></tr><tr><td>Negative</td><td>Positive</td><td></td></tr></table>

<table><tr><td rowspan=2 colspan=1>GoldStandardResult</td><td rowspan=1 colspan=2>UroVysion Result</td><td rowspan=1 colspan=2>UroVysion Result</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>269</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>352</td></tr><tr><td rowspan=1 colspan=2></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=2>Metric</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=3 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>EpiCheck</td><td rowspan=1 colspan=1>65.1% (54/83)</td><td rowspan=1 colspan=2>54.8%; 75.3%</td></tr><tr><td rowspan=1 colspan=1>Urovysion</td><td rowspan=1 colspan=1>60.2% (50/83)</td><td rowspan=1 colspan=2>49.7%; 70.8</td></tr><tr><td rowspan=1 colspan=1>Difference</td><td rowspan=1 colspan=1>4.82%</td><td rowspan=1 colspan=2>-5.7%; 15.3%</td></tr><tr><td rowspan=3 colspan=2>Specificity</td><td rowspan=1 colspan=1>EpiCheck</td><td rowspan=1 colspan=1>79.2% (213/269)</td><td rowspan=1 colspan=2>74.3%; 84.0%</td></tr><tr><td rowspan=1 colspan=1>Urovysion</td><td rowspan=1 colspan=1>82.2% (221/269)</td><td rowspan=1 colspan=2>77.6%; 86.7%</td></tr><tr><td rowspan=1 colspan=1>Difference</td><td rowspan=1 colspan=1>-2.97%</td><td rowspan=1 colspan=2>-7.8%; 1.9%</td></tr></table>

Table 41. Sensitivity of Bladder EpiCheck and UroVysion vs. Gold Standard by Grade – Matched Cases   

<table><tr><td rowspan=1 colspan=1>Metric</td><td rowspan=1 colspan=1>Grade</td><td rowspan=1 colspan=1>Test</td><td rowspan=1 colspan=1>% (n/N)</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>Sensitivity</td><td rowspan=3 colspan=1>High Grade</td><td rowspan=1 colspan=1>EpiCheck</td><td rowspan=1 colspan=1>75.0% (30/40)</td><td rowspan=1 colspan=1>61.6%;88.4%</td></tr><tr><td rowspan=1 colspan=1>UroVysion</td><td rowspan=1 colspan=1>70.0% (28/40)</td><td rowspan=1 colspan=1>55.8%;84.2%</td></tr><tr><td rowspan=1 colspan=1>Difference</td><td rowspan=1 colspan=1>5.00%</td><td rowspan=1 colspan=1>-10.4%; 20.4%</td></tr><tr><td rowspan=3 colspan=1>Low Grade</td><td rowspan=1 colspan=1>EpiCheck</td><td rowspan=1 colspan=1>41.7% (10/24)</td><td rowspan=1 colspan=1>21.9%; 61.4%</td></tr><tr><td rowspan=1 colspan=1>UroVysion</td><td rowspan=1 colspan=1>41.7% (10/24)</td><td rowspan=1 colspan=1>21.9%; .4%</td></tr><tr><td rowspan=1 colspan=1>Difference</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>-16.3%; 16.3%</td></tr></table>

# 4. Specificity in Urology Patients without Bladder Cancer

A multi-center, prospective study was conducted to establish the specificity of Bladder EpiCheck test in urology patients without prior history or clinical evidence of bladder cancer. One-hundred forty- seven (147) subjects aged 22 and above, visiting the urology clinic for any non-bladder cancer disorders. Subjects were excluded from the study if they had known current or prior diagnosis of both invasive and non-muscle invasive bladder cancer. The specificity of Bladder EpiCheck in this study population was $9 8 . 0 \%$ . Covariates analysis by sex, age group, ethnicity, race, smoking status, reason for visit at clinic, non-cancer genitourinary disease type, presence of hematuria, BPH, UTI and STD were performed. No statistical significance was found, indicating that there was no impact of any of the tested covariates on the applicability of the primary analysis results on the population tested.

# 5. Specificity – Cross-Reactivity with Other Cancers

The effect of potential cross-reactivity was assessed using banked remnant de-identified urine samples collected from patients diagnosed with different types of cancers, specifically cancers that are more likely to shed cancer cells into the bladder such as renal cell carcinoma (RCC) and prostate cancer as well as other solid and hematology cancer. One-hundred forty-seven urine samples were assessed. The most frequent cancer category was genitourinary, with 54 subjects $( 3 7 \% )$ including prostate and renal cell carcinoma, followed by gastrointestinal with 26 subjects $( 1 8 \% )$ ; 15 subjects $( 1 0 \% )$ had gynecological cancers and 10 subjects had hematological cancers $( 7 \% )$ . 42 subjects $( 2 9 \% )$ had other solid cancer types. The study population had a mean age of 65 years old (ranging from 27 to 89) with a similar rate of sexes. Nearly all subjects were nonsmokers or former smokers, with $12 \%$ current smokers. In this study, the specificity was found to be $9 5 \%$ (140/147: Prostate (3), RCC (2), Uterine (1), pancreas (1)) ( $9 5 \%$ confidence interval of $9 0 . 4 \% . 9 8 . 1 \%$ ), confirming the specificity of Bladder EpiCheck in patients with cancer other than bladder cancer.

# VI. Instrument Name:

Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument (k082562)

# System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless   
transmission?   
Yes or No ___X_

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of   
product types:   
Yes __ _X_ or No

3. Level of Concern: Moderate

4. Specimen Handling: Refer to Device Description section above.

5. Calibration and Quality Controls: Refer to Device Description section above.

# VII. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# VIII.Proposed Labeling:

The labeling is sufficient, and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable

# IX. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.