# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATIONDECISION SUMMARYASSAY AND INSTRUMENT

I Background Information:

A 510(k) Number K203035   
B Applicant PerkinElmer Inc   
C Proprietary and Established Names Eonis SCID-SMA kit   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PJI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.5980 -Newborn Screening Testfor Severe CombinedImmunodeficiencyDisorder (SCID)</td><td rowspan=1 colspan=1>IM - Immunology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission: New Device

# B Measurand:

T-cell receptor excision circles (TRECs) and Kappa-deleting element recombination circles (KRECs)

# C Type of Test:

Multiplex real-time polymerase chain reaction-based assay using target sequence specific primers and probes

# III Intended Use/Indications for Use:

A Intended Use(s): See Indications for Use below.

# B Indication(s) for Use:

The Eonis™ SCID-SMA kit is intended for the semi-quantitative determination of TREC (T-cell receptor excision circle) as an aid in screening newborns for Severe Combined Immunodeficiency (SCID) and for the semi-quantitative determination of KREC (Kappadeleting recombination excision circle) as an aid in screening newborns for X-linked agammaglobulinemia (XLA). The test is intended for DNA from blood specimens dried on a filter paper and for use on the QuantStudio™ Dx Real-Time PCR instrument.

This test is not intended for screening of SCID-like Syndromes, such as DiGeorge Syndrome, or Omenn Syndrome. It is also not intended to screen for less acute SCID syndromes such as leakySCID or variant SCID. The test is not indicated for screening B-cell deficiency disorders other than XLA, such as atypical XLA, or for screening of XLA carriers.

This test is not intended for use as a diagnostic test and a positive screening result should be followed by confirmatory testing.

# C Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

In vitro diagnostic use

# D Special Instrument Requirements:

QuantStudio™ Dx Real-Time PCR Instrument (ThermoFisher, k123955)

Liquid Handling Instruments: JANUS G3 Automated Workstation Extraction instrument (labeled for IVD use) or JANUS Extraction Mini instrument (labeled for IVD use) and the JANUS PCR Mastermix instrument (labeled for IVD use).

# IV Device/System Characteristics:

# A Device Description:

The Eonis SCID-SMA kit contains reagents to detect three biomarkers: TREC, KREC and exon 7 in the SMNI gene. Detection of SMN1 gene was submitted and reviewed in DEN200044. This Decision Summary only discusses the validation data in support of TREC and KREC.

The newborn screening workflow for the Eonis SCID-SMA kit includes:

• Two liquid handling platforms (one for DNA extraction and one for PCR master mix setup) QuantStudio™ Dx Real-Time PCR instrument (k123955) Eonis Analysis Software

Each Eonis SCID-SMA kit contains reagents for up to 384 reactions or 1152 reactions including kit controls. The kit contents are listed in Table 1. Materials required but not provided include the Eonis DNA Extraction Kit, Eonis Analysis Software, and consumables.

Table 1. Eonis SCID-SMA Kit content   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Quantity</td></tr><tr><td rowspan=1 colspan=1>SCID-SMA Kit Controls</td><td rowspan=1 colspan=1>2 filter paper cassettes containing 4 sets of driedblood spots for 384 reaction kit4 filter paper cassettes containing 8 sets of driedblood spots for 1152 reaction kit</td></tr><tr><td rowspan=1 colspan=2>C1 Analyte-negative (TREC/KREC/SMN1) control</td></tr><tr><td rowspan=1 colspan=2>C2 Low TREC/KREC and normal SMN1 control</td></tr><tr><td rowspan=1 colspan=1>C3 High TREC/KREC and normal SMN1 control</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>PCR Reagent 1</td><td rowspan=1 colspan=1>1 vial, 2.7 mL for 384 reaction kit3 vials, 2.7 mL for 1152 reaction kit</td></tr><tr><td rowspan=1 colspan=1>PCR Reagent 2</td><td rowspan=1 colspan=1>1 vial, 2.7 mL for 384 reaction kit3 vials, 2.7 mL for 1152 reaction kit</td></tr><tr><td rowspan=1 colspan=1>Lot-specificquality control certificate</td><td rowspan=1 colspan=1>1 pc</td></tr></table>

# B Principle of Operation:

The Eonis SCID-SMA kit is a multiplex real-time PCR based assay that uses target sequencespecific primers and TaqMan™ probes to amplify and detect TREC, KREC, and RPP30 in DNA extracted from newborn dried blood spot (DBS) using the Eonis DNA Extraction kit in a single PCR reaction. Each TaqMan probe has a unique dye linked to their terminal $5 '$ end, allowing for the simultaneous detection of the four targets, if present. The amount of each target present in the DNA is determined by the intensity of fluorescence emitted by each dye released from the degraded probe during amplification and detected by the QuantStudio Dx Real-Time PCR instrument. The instrument measures fluorescence signals and converts them into comparative quantitative readouts which are expressed as a function of cycle threshold $\mathrm { ( C t ) }$ values. RPP30 is used as an internal amplification control for DNA quality and to relatively quantitate TREC and KREC (copies/ $1 0 ^ { 5 }$ cells) using TREC ∆Ct and KREC ∆Ct. TREC and KREC concentrations are calculated using the following formulas to generate results expressed as copies/ $1 0 ^ { 5 }$ cells:

TREC: 2 X 2-(ΔCt TREC) x 117000   
KREC: 2 X 2-(ΔCt TREC) x 254000

The interpretation of results based on copy number is shown in Tables 2 and 3.

Table 2. TREC result interpretation matrix   

<table><tr><td rowspan=1 colspan=1>RPP30 Ct values</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TREC Ctvalues</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TREC conc.(copies/105 cells)</td><td rowspan=1 colspan=1>Resultinterpretation forTREC</td></tr><tr><td rowspan=1 colspan=1>Ct &lt; 15.0</td><td rowspan=1 colspan=4>Values not considered when RPP30 Ct &lt; 15.0</td><td rowspan=1 colspan=1>Invalid*</td></tr><tr><td rowspan=4 colspan=1>15.0 ≤ Ct ≤ 32.00</td><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>Ct &lt; 18.0</td><td rowspan=1 colspan=2>Values not consideredwhen TREC Ct &lt; 18.0</td><td rowspan=1 colspan=1>Invalid*</td></tr><tr><td rowspan=2 colspan=1>AND</td><td rowspan=2 colspan=1>Ct ≥ 18.0</td><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>TREC conc.≥CutoffTREC</td><td rowspan=1 colspan=1>Presumptive normal</td></tr><tr><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>TREC conc. &lt;CutoffTREC</td><td rowspan=1 colspan=1>Presumptive positive*</td></tr><tr><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>No Ct</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>Presumptive positive*</td></tr><tr><td rowspan=1 colspan=1>Ct &gt; 32.00or no Ct</td><td rowspan=1 colspan=4>Values not considered when RPP30 Ct &gt;32.00 or no Ct</td><td rowspan=1 colspan=1>Invalid*</td></tr></table>

\*Samples with invalid results should be repeated in singlicate, and samples with presumptive positive results should be repeat tested in duplicate. If one of the duplicate samples is positive, the result is presumptive positive.

Table 3. KREC result interpretation matrix   

<table><tr><td rowspan=1 colspan=1>RPP30 Ctvalues</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KREC Ctvalues</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>KREC conc.(copies/105 cells)</td><td rowspan=1 colspan=1>Resultinterpretation forKREC</td></tr><tr><td rowspan=1 colspan=1>Ct &lt; 15.0</td><td rowspan=1 colspan=4>Values not considered when RPP30 Ct &lt; 15.0</td><td rowspan=1 colspan=1>Invalid*</td></tr><tr><td rowspan=1 colspan=1>15.0 ≤ Ct≤ 32.00 AND</td><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>Ct &lt; 18.0</td><td rowspan=1 colspan=2>Values not consideredwhen TREC Ct &lt; 18.0</td><td rowspan=1 colspan=1>Invalid*</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>AND</td><td rowspan=2 colspan=1>Ct ≥ 18.0</td><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>KREC conc.≥CutoffKREC</td><td rowspan=1 colspan=1>Presumptive normal</td></tr><tr><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>KREC conc. &lt;CutoffKREC</td><td rowspan=1 colspan=1>Presumptive positive*</td></tr><tr><td rowspan=1 colspan=1>AND</td><td rowspan=1 colspan=1>No Ct</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>Presumptive positive*</td></tr><tr><td rowspan=1 colspan=1>Ct &gt; 32.00or no Ct</td><td rowspan=1 colspan=4>Values not considered when RPP30 Ct &gt;32.00 or no Ct</td><td rowspan=1 colspan=1>Invalid*</td></tr></table>

\*Samples with invalid results should be repeated in singlicate, and samples with presumptive positive results should be repeat tested in duplicate. If one of the duplicate samples is positive, the result is presumptive positive.

Result Reporting

Samples that result in values below 242 copies/ $1 0 ^ { 5 }$ cells are recommended to be reported as $" < 2 4 2$ copies/ $1 0 ^ { 5 }$ cells". Samples that result in values above 4316 copies/ $1 0 ^ { 5 }$ cells are recommended to be reported as $" > 4 3 2 0$ copies/ $1 0 ^ { 5 }$ cells" and are considered to be TREC screen negative.

Samples that result in values below 459 copies/ $1 0 ^ { 5 }$ cells are recommended to be reported as $" 4 5 9$ copies/ $1 0 ^ { 5 }$ cells". Samples that result in values above 24343 copies/ $1 0 ^ { 5 }$ cells are recommended to be reported as " $" > 2 4 3 0 0$ copies/ $1 0 ^ { 5 }$ cells" and are considered to be KREC screen negative.

# C Instrument Description Information:

1. Instrument Name:

QuantStudio Dx Real-Time PCR instrument (The information below is specific to the TREC   
and KREC analytes; refer to the decision summary for additional information regarding the   
features in this section K123955)   
JANUS G3 Automated Workstation (for IVD use) consisting of the JANUS Extraction   
Instrument and JANUS PCR Mastermix Instrument

2. Specimen Identification:

Specimens are barcoded and entered into the testing queue consistent with the laboratory procedures.

# 3. Specimen Sampling and Handling:

The required sample is a $3 . 2 \mathrm { m m }$ specimen punch from a dried blood spot (DBS) into a 96- well plate. DNA is extracted and purified using the Eonis DNA Extraction kit. The extraction can be completed manually or using an automated liquid handling platform, the JANUS G3 Workstation (IVD), for processing up to four 96-well plates simultaneously.

# 4. Calibration:

The device (Eonis SCID-SMA kit) calibrates the TREC results utilizing the internal control RPP30 gene (assumes two copies) and a mathematical algorithm and reports out the results in the unit of copies/ $1 0 ^ { 5 }$ cells.

5. Quality Control:

Quality control is addressed for each separately cleared specific assay to be run on the instrument. For the Eonis SCID-SMA assay, quality is maintained by the process control, used during sample processing and amplification in the assay. Additionally, there are external positive and negative controls available that are used in accordance with the assay instructions for use.

Controls are used throughout the entire procedure from sample preparation to PCR as a quality check for performance of the Eonis test. The Eonis SCID-SMA kit contains three multi-level dried blood spot controls: C1, C2 and C3. The DBS controls are filter paper cassettes with each control level spotted in duplicate.

C1 is an analyte-negative control and does not contain TREC, KRECanalytes.   
C2 and C3 are analyte-positive controls that have different levels of TREC and KREC analytes (lower level in C2 and a higher level in C3).   
All three kit controls have the same level of RPP30 analyte.   
Blank reaction wells without sample disks (NTC [no template control]) are used to monitor the absence of contamination. All controls are prepared from human whole blood with a hematocrit value of $48 \mathrm { - } 5 5 \%$ and contain purified salmon-sperm DNA and   
synthetic TREC, KREC, and RPP30 plasmid DNA.

Each Eonis run includes all four controls in duplicate in each 96-well DNA extraction plate. The newborn samples and DBS controls are punched into the plate, while the well reserved for the Blank control does not contain any punches. The Blank control undergoes the whole process, from DNA extraction to amplification, and the NTC sample consists of elution solution derived from the DNA extraction process.

The test result validation for each 96-well DNA extraction plate is based on the three kit DBS controls and the Blank reaction well. The Eonis Analysis software will identify each control by their designated locations and check their validity. The possible outcome for each control replicate is ‘Valid’ or ‘Invalid’. If one or more controls are invalid, the results of the clinical specimens processed in the same 96-well DNA extraction plate are reported as ‘Invalid’. For the Blank reaction well, both replicates must meet the acceptance criteria for the run to be valid. For C1-C3 DBS controls, the results of both replicates must meet the acceptance criteria for the run to be valid. The thresholds for valid controls are shown in Table 4.

Table 4. Acceptance limits for kit controls for TREC and KREC   

<table><tr><td colspan="1" rowspan="1">Kit Control Name</td><td colspan="1" rowspan="1">ParameterName</td><td colspan="1" rowspan="1">Lower Limit(2)</td><td colspan="1" rowspan="1">Upper Limit(≤ or No Ct)</td></tr><tr><td colspan="1" rowspan="3">NTC (Blank)</td><td colspan="1" rowspan="1">TREC Ct</td><td colspan="1" rowspan="1">32.65</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">KREC Ct</td><td colspan="1" rowspan="1">32.55</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">RPP30 Ct</td><td colspan="1" rowspan="1">28.35</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="3">C1</td><td colspan="1" rowspan="1">TREC Ct</td><td colspan="1" rowspan="1">32.65</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">KREC Ct</td><td colspan="1" rowspan="1">32.55</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">RPP30 Ct</td><td colspan="1" rowspan="1">15.00</td><td colspan="1" rowspan="1">28.00</td></tr><tr><td colspan="1" rowspan="5">C2</td><td colspan="1" rowspan="1">TREC Ct</td><td colspan="1" rowspan="1">18.00</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">KREC Ct</td><td colspan="1" rowspan="1">18.00</td><td colspan="1" rowspan="1">No Ct</td></tr><tr><td colspan="1" rowspan="1">RPP30 Ct</td><td colspan="1" rowspan="1">15.00</td><td colspan="1" rowspan="1">28.00</td></tr><tr><td colspan="1" rowspan="1">TREC ΔCt</td><td colspan="1" rowspan="1">6.10</td><td colspan="1" rowspan="1">13.90</td></tr><tr><td colspan="1" rowspan="1">KREC ΔCt</td><td colspan="1" rowspan="1">3.55</td><td colspan="1" rowspan="1">16.45</td></tr><tr><td colspan="1" rowspan="5">C3</td><td colspan="1" rowspan="1">TREC Ct</td><td colspan="1" rowspan="1">18.00</td><td colspan="1" rowspan="1">33.00</td></tr><tr><td colspan="1" rowspan="1">KREC Ct</td><td colspan="1" rowspan="1">18.00</td><td colspan="1" rowspan="1">34.00</td></tr><tr><td colspan="1" rowspan="1">RPP30 Ct</td><td colspan="1" rowspan="1">15.00</td><td colspan="1" rowspan="1">28.00</td></tr><tr><td colspan="1" rowspan="1">TREC ΔCt</td><td colspan="1" rowspan="1">1.10</td><td colspan="1" rowspan="1">8.90</td></tr><tr><td colspan="1" rowspan="1">KREC ΔCt</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">12.45</td></tr></table>

# V Substantial Equivalence Information:

A Predicate Device Name(s): PerkinElmer EnLite™ Neonatal TREC Kit B Predicate 510(k) Number(s): DEN140010 C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp; PredicateDevice(s):</td><td colspan="1" rowspan="1">K203035</td><td colspan="1" rowspan="1">DEN140010</td></tr><tr><td colspan="1" rowspan="1">Device Trade Name</td><td colspan="1" rowspan="1">Eonis SCID-SMA Kit</td><td colspan="1" rowspan="1">EnLite Neonatal TREC Kit</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">The EonisTM SCID-SMA kit isintended for the semi-quantitative determination ofTREC (T-cell receptor excisioncircle) as an aid in screeningnewborns for Severe CombinedImmunodeficiency (SCID).The test is intended for DNAfrom blood specimens dried ona filter paper and for use on theQuantStudioTM Dx Real-TimePCR instrument.This test is not intended forscreening of SCID-likeSyndromes, such as DiGeorgeSyndrome, or OmennSyndrome.</td><td colspan="1" rowspan="1">The EnLiteTM NeonatalTREC Kit is an in vitrodiagnostic device intendedfor the semi-quantitativedetermination of TREC (T-cell receptor excision circle)DNA in blood specimensdried on filter paper. Thetest is for use on theVICTORTM EnLiteinstrument. The test isindicated for use as an aid inscreening newborns forsevere combinedimmunodeficiency disorder(SCID). This test is notintended for use as adiagnostic test for screeningof SCID-like Syndromes,such as DiGeorge Syndrome,</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">It is also not intended to screenfor less acute SCID syndromessuch as leaky-SCID or variantSCID.This test is not intended for useas a diagnostic test and apositive screening result shouldbe followed by confirmatorytesting.</td><td colspan="1" rowspan="1">or Omenn Syndrome. It isalso not intended to screenfor less acute SCIDsyndromes such as leaky-SCID or variant SCID.</td></tr><tr><td colspan="1" rowspan="1">Specimen</td><td colspan="1" rowspan="1">Dried blood spot</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Stability/Shelf Life</td><td colspan="1" rowspan="1">The kit should be stored at -30to -16°C. Once opened, the kitshould be used within 14 days.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="2" rowspan="1">General Device Characteristic Differences</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Intended Use/IndicationsFor Use</td><td colspan="1" rowspan="1">For the semi-quantitativedetermination of KREC(Kappa-deleting recombinationexcision circle) as an aid inscreening newborns for X-linked agammaglobulinemia(XLA).The test is not indicated forscreening B-cell deficiencydisorders other than XLA, suchas atypical XLA, or forscreening of XLA carriers.</td><td colspan="1" rowspan="1">Not applicable</td></tr><tr><td colspan="1" rowspan="1">Test Methodology</td><td colspan="1" rowspan="1">Semi-quantitative, multiplexreal-time fluorescent-basedpolymerase chain reaction(PCR) based nucleic acidamplification and detection</td><td colspan="1" rowspan="1">Semi-quantitative,polymerase chain reaction(PCR) based nucleic acidamplification and time-resolved fluorescenceresonance energy transfer(TR-FRET) based detection</td></tr><tr><td colspan="1" rowspan="1">Units of Reporting</td><td colspan="1" rowspan="1">TREC results are calibratedusing the internal control geneand a mathematical algorithmand reports out the results in theunit of copies/105 cells.</td><td colspan="1" rowspan="1">TREC results are calibratedagainst a calibrator curve oneach plate and reports out theresults in the unit ofcopies/μL.</td></tr><tr><td colspan="1" rowspan="1">Instrument/Software</td><td colspan="1" rowspan="1">QuantStudio Dx Real-TimePCR instrument and Eonissoftware</td><td colspan="1" rowspan="1">VICTOR EnLite instrumentand EnLiteTM workstationsoftware</td></tr><tr><td colspan="1" rowspan="1">Reportable Range</td><td colspan="1" rowspan="1">TREC 242  4320 copies/105cells</td><td colspan="1" rowspan="1">TREC 29-473 copies/μLblood</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">KREC 459 - 24300 copies/105cells</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lower Limits of Measure</td><td colspan="1" rowspan="1">TREC LoB=0 copies/105 cells,LoD/LoQ= 242 copies/105 cellsKREC LoB=0 copies/105 cells,LoD/LoQ =459 copies/105 cells</td><td colspan="1" rowspan="1">TREC LoB=3 copies/ μLblood, LoD=20 copies/ μLblood, LoQ=29 copies/ μLblood</td></tr><tr><td colspan="1" rowspan="1">Calibrators / Standards</td><td colspan="1" rowspan="1">Calibration is based on internalreference (RPP30) in each welland manufacturer calibration foreach kit lot.</td><td colspan="1" rowspan="1">3 levels of DBS calibratorsprepared from porcine wholeblood spiked with TREC andbeta-actin (reference)plasmids, and a no-templateblank</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">3 levels of DBS controlsprepared from leucocyte-depleted human red blood cellsand TREC, KREC and RPP30plasmids spiked in, plus Notemplate control (NTC)</td><td colspan="1" rowspan="1">3 levels of DBS controlsprepared from porcine wholeblood, TREC and beta-actin(reference) plasmids spikedin.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

• CLSI EP05-A3: Evaluation of precision of quantitative measurement procedures; approved guideline-third edition   
• CLSI EP6-A: Evaluation of the linearity of quantitative measurement procedures: a statistical approach; approved guideline   
• CLSI EP07-A2: Interference testing in clinical chemistry: approved guideline-second edition   
• CLSI EP25-A: Evaluation of Stability of In Vitro Diagnostic Reagents; approved guideline   
• CLSI NBS01-A6: Blood Collection on Filter Paper for Newborn Screening Programs-sixth edition   
• CLSI MM13-A Collection Transport Preparation and Storage of Specimens for Molecular Methods; approved guideline   
• CLSI MM17 Verification and validation of multiplex nucleic acid assays-second edition   
• IEC 62304 Edition 1.1 2015-06 Medial device software-Software life cycle processes

VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

1. Precision/Reproducibility:

Study 1: Site-to-Site Reproducibility

The objective of this study was to characterize the within-laboratory precision and reproducibility of the Eonis test system across 3 study sites using a panel of samples with different TREC and KREC levels. At each study site, 2 operators performed 5 runs each, over 5 operating days, 1 run per day, and 5 specimen replicates tested per run. A total of 13 independent samples were tested at each of the 3 study sites using 1 kit lot for a total of 150 replicates per sample (3 sites $_ { \textrm { X 2 } }$ operators x 5 runs/days x 5 replicates per run $= 1 5 0$ ). The study used 1 lot of reagents. The samples were generated to represent a range of concentrations and were spotted onto filter paper, dried overnight, and stored at $- 3 0 ^ { \circ } \mathrm { C }$ to - $1 6 \mathrm { { ^ \circ C } }$ in a sealed bag with desiccant until use. A description of the samples is provided in Table 5.

Table 5. Descriptive summary samples used in the precision study   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">TREC</td><td colspan="1" rowspan="1">KREC</td><td colspan="1" rowspan="1">Preparation</td></tr><tr><td colspan="1" rowspan="1">Sample 1</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">TREC plasmid spiked into cord blood at 1500copies/µL with the hematocrit adjusted to 48.2(approx. Ct value 28.63)</td></tr><tr><td colspan="1" rowspan="1">Sample 2</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">KREC plasmid spiked into cord blood at 5000copies/ µL with the hematocrit adjusted to 48.2(approx. Ct value 28.11)</td></tr><tr><td colspan="1" rowspan="1">Sample 3</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Cord blood with hematocrit adjusted to 48.2</td></tr><tr><td colspan="1" rowspan="1">Sample 4</td><td colspan="1" rowspan="1">Low(Above cut-offf)</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Diluting 3mL of cord blood with hematocritadjusted to 48.2 into 2.25 mL of Sample 6 targetingnear but below 98% quantile (32.71) of TREC Ctdistribution of a population of presumptive normalNBS samples</td></tr><tr><td colspan="1" rowspan="1">Sample 5</td><td colspan="1" rowspan="1">Low(Below cut-off</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Diluting 857.1 µL of cord blood with hematocrit of48.32 in 2142.9 µL of Sample 6 (approx. Ct value33.07</td></tr><tr><td colspan="1" rowspan="1">Sample 6</td><td colspan="1" rowspan="1">Very low(Near 0)</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">KREC plasmid spiked into adult whole blood at2000 copies/μL with hematocrit adjusted to 50.7(approx. TREC Ct value 38.08; approx. KREC Ctvalue 30.56)</td></tr><tr><td colspan="1" rowspan="1">Sample 7</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Low(Abovecut-off)</td><td colspan="1" rowspan="1">Diluting 0.75 mL of cord blood with hematocritadjusted to 48.2 in 2.25 mL of Sample 8 (approx.KREC Ct value 31.96)</td></tr><tr><td colspan="1" rowspan="1">Sample 8</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Low(Belowcut-off)</td><td colspan="1" rowspan="1">TREC plasmid spiked into adult whole blood at 700copies/μL with hematocrit adjusted to 50.7 (approx.TREC value 30.01 and approx. KREC value 33.30)</td></tr><tr><td colspan="1" rowspan="1">Sample 9</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Very low(Near 0)</td><td colspan="1" rowspan="1">TREC plasmid spiked at 2000 copies/ µL withhematocrit adjusted to 49.5 (approx. TREC Ct value30.08 and approx. KREC Ct value 0)</td></tr><tr><td colspan="1" rowspan="1">Sample 10</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Pooled adult whole blood from 2 SMA carriers withhematocrit adjusted to 46.1</td></tr><tr><td colspan="1" rowspan="1">Sample 11</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">Normal</td><td colspan="1" rowspan="1">TREC plasmid at 700 copies/µL, KREC plasmid at1000 copies/μL and SMA Coriell cells at 15000cells/µL spiked into leukocyte depleted blood withhematocrit adjusted to 49.5</td></tr><tr><td colspan="1" rowspan="1">Sample 12</td><td colspan="1" rowspan="1">Low(Above cut-off</td><td colspan="1" rowspan="1">Low(Abovecut-off)</td><td colspan="1" rowspan="1">Diluting 4.4 mL of cord blood with hematocritadjusted to 48.2 into 6.6 mL of adult whole bloodwith hematocrit adjusted to 50.7 (approx. TREC Ctvalue 32.48 and approx. KREC Ct value 31.30)</td></tr><tr><td colspan="1" rowspan="1">Sample 13</td><td colspan="1" rowspan="1">Low(Below cut-off)</td><td colspan="1" rowspan="1">Low(Belowcut-off)</td><td colspan="1" rowspan="1">Adult blood hematocrit adjusted to 50.7 (approx.TREC Ct value 36.34 and approx. KREC Ct value33.89)</td></tr></table>

The values in the mean TREC and KREC copies/ $1 0 ^ { 5 }$ cells column are transformed from logarithmic (Ln) mean values, and therefore, they represent geometric means in the copies/ $1 0 ^ { 5 }$ cells scale. The analysis of variance approach was used to calculate the results presented as standard deviations (SDs) in the logarithmic (Ln) scale complemented with total coefficient of variation $( \% C V )$ in lognormal scale (Table 6 and Table 8). Summary mean, min and max copies/ $1 0 ^ { 5 }$ cells and SD and $\% C V$ results without log transformation for total imprecision are shown in Table 7 and Table 9.

Acceptance Criteria:

For TREC, the total assay precision within the measuring range must have a $\mathrm { S D } \le 0 . 9 \mathrm { L n }$ (copies $1 0 ^ { 5 }$ cells) where the measuring range starts at 250 copies/ $1 0 ^ { 5 }$ cells. For KREC, the total assay precision within the measuring range must have a $\mathrm { S D } \leq 1 . 4 9$ Ln (copies/ $1 0 ^ { 5 }$ cells).

Table 6. TREC reproducibility data pooled across three laboratories   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>GeometricMean(Copies105/cells)</td><td rowspan=1 colspan=1>MeanLn(Copies105/cells)</td><td rowspan=1 colspan=1>Repeata-bilityLn SD</td><td rowspan=1 colspan=1>BetweenRunLn SD</td><td rowspan=1 colspan=1>BetweenOperatorLn SD</td><td rowspan=1 colspan=1>BetweenSiteLn SD</td><td rowspan=1 colspan=1>TotalLnSD</td><td rowspan=1 colspan=1>Log-normalCV%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>3.14</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>151</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>3.53</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>180</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>5.19</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>90</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>5.70</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>912</td><td rowspan=1 colspan=1>6.82</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>936</td><td rowspan=1 colspan=1>6.84</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1026</td><td rowspan=1 colspan=1>6.93</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1478</td><td rowspan=1 colspan=1>7.30</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3013</td><td rowspan=1 colspan=1>8.01</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3290</td><td rowspan=1 colspan=1>8.10</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3867</td><td rowspan=1 colspan=1>8.26</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8225</td><td rowspan=1 colspan=1>9.01</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9000</td><td rowspan=1 colspan=1>9.11</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>27</td></tr></table>

Table 7. TREC total variation results without logarithmic transformation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>Calculations without logarithmic transformation</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>Min(Copies/105 cells)</td><td rowspan=1 colspan=1>Max(Copies/105 cells)</td><td rowspan=1 colspan=1>SD(Copies/105 cells)</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>308</td><td rowspan=1 colspan=1>55.3</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>796</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>346</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>971</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>952</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>1792</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>997</td><td rowspan=1 colspan=1>239</td><td rowspan=1 colspan=1>2563</td><td rowspan=1 colspan=1>349</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1076</td><td rowspan=1 colspan=1>497</td><td rowspan=1 colspan=1>2490</td><td rowspan=1 colspan=1>335</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1526</td><td rowspan=1 colspan=1>654</td><td rowspan=1 colspan=1>2411</td><td rowspan=1 colspan=1>370</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>3388</td><td rowspan=1 colspan=1>513</td><td rowspan=1 colspan=1>6401</td><td rowspan=1 colspan=1>767</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4076</td><td rowspan=1 colspan=1>2043</td><td rowspan=1 colspan=1>16934</td><td rowspan=1 colspan=1>1590</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4601</td><td rowspan=1 colspan=1>797</td><td rowspan=1 colspan=1>218936</td><td rowspan=1 colspan=1>17700</td><td rowspan=1 colspan=1>384*</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>8571</td><td rowspan=1 colspan=1>3233</td><td rowspan=1 colspan=1>14974</td><td rowspan=1 colspan=1>2420</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>9277</td><td rowspan=1 colspan=1>4236</td><td rowspan=1 colspan=1>16270</td><td rowspan=1 colspan=1>2320</td><td rowspan=1 colspan=1>25</td></tr></table>

\*Dataset for sample 8 has one high outlier (max value) affecting the variability estimate. Without the outlier, the estimated $C V \%$ is $41 \%$ and similar to other samples within the measuring range.

Table 8. KREC reproducibility data pooled across three laboratories   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>GeometricMean(Copies105/cells)</td><td rowspan=1 colspan=1>Mean Ln(Copies105/cells)</td><td rowspan=1 colspan=1>Repeata-bilityLn SD</td><td rowspan=1 colspan=1>BetweenRunLn SD</td><td rowspan=1 colspan=1>BetweenOperator Ln SD</td><td rowspan=1 colspan=1>BetweenSiteLn SD</td><td rowspan=1 colspan=1>TotalLn SD</td><td rowspan=1 colspan=1>Log-normalCV%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>5.85</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>89</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>763</td><td rowspan=1 colspan=1>6.64</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2770</td><td rowspan=1 colspan=1>7.93</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3849</td><td rowspan=1 colspan=1>8.26</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4037</td><td rowspan=1 colspan=1>8.30</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4359</td><td rowspan=1 colspan=1>8.38</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5007</td><td rowspan=1 colspan=1>8.52</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>9313</td><td rowspan=1 colspan=1>9.14</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9648</td><td rowspan=1 colspan=1>9.17</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>16531</td><td rowspan=1 colspan=1>9.71</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>34982</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>5.85</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>89</td></tr></table>

Table 9. KREC total variation results without logarithmic transformation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>Calculations without logarithmic transformation</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>Min(Copies/105 cells)</td><td rowspan=1 colspan=1>Max(Copies/105 cells)</td><td rowspan=1 colspan=1>SD(Copies/105 cells)</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>431</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1071</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>56</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>1925</td><td rowspan=1 colspan=1>349</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>862</td><td rowspan=1 colspan=1>183</td><td rowspan=1 colspan=1>2003</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>2877</td><td rowspan=1 colspan=1>1097</td><td rowspan=1 colspan=1>5232</td><td rowspan=1 colspan=1>792</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4025</td><td rowspan=1 colspan=1>1075</td><td rowspan=1 colspan=1>13447</td><td rowspan=1 colspan=1>1300</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4162</td><td rowspan=1 colspan=1>2241</td><td rowspan=1 colspan=1>6788</td><td rowspan=1 colspan=1>1030</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4457</td><td rowspan=1 colspan=1>2104</td><td rowspan=1 colspan=1>7746</td><td rowspan=1 colspan=1>960</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5457</td><td rowspan=1 colspan=1>1567</td><td rowspan=1 colspan=1>36630</td><td rowspan=1 colspan=1>3230</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>9508</td><td rowspan=1 colspan=1>4735</td><td rowspan=1 colspan=1>15856</td><td rowspan=1 colspan=1>1910</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>9915</td><td rowspan=1 colspan=1>1564</td><td rowspan=1 colspan=1>16967</td><td rowspan=1 colspan=1>2130</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>16955</td><td rowspan=1 colspan=1>8813</td><td rowspan=1 colspan=1>29034</td><td rowspan=1 colspan=1>3880</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>36370</td><td rowspan=1 colspan=1>16841</td><td rowspan=1 colspan=1>118413</td><td rowspan=1 colspan=1>11200</td><td rowspan=1 colspan=1>31</td></tr></table>

# Study 2: Lot-to-Lot Reproducibility

A study was performed internally (single site) using 3 sets of Eonis test systems, 2 operators, and 3 kit lots. The 13 samples (Table 5) were tested over 23 calendar days with 2 replicates per sample per plate in a total of 54 runs with a randomized plate map. The total number of measurements was 108 per sample.

The values in the mean TREC and KREC copies/ $1 0 ^ { 5 }$ cells column are transformed from logarithmic (Ln) mean values, and therefore, they represent geometric means in the copies $1 0 ^ { 5 }$ cells scale. The analysis of variance approach was used to calculate the results presented as SDs in the logarithmic (Ln) scale complemented with total $\% C V s$ in lognormal scale (Table 10 and Table 12). Summary mean, min and max copies/ $1 0 ^ { 5 }$ cells, and SD and $\% \mathrm { C V }$ results without log transformation for total imprecision are shown in Table 11 and Table 13.

Table 10. TREC precision data across three kit lots and three instruments   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">GeometricMean(Copies105/cells)</td><td colspan="1" rowspan="1">Mean Ln(Copies105/cels)</td><td colspan="1" rowspan="1">Repeata-biilityLn SD</td><td colspan="1" rowspan="1">BetweenRunLn SD</td><td colspan="1" rowspan="1">BetweenInstrumentLn SD</td><td colspan="1" rowspan="1">BetweenLotLn SD</td><td colspan="1" rowspan="1">TotalLn SD</td><td colspan="1" rowspan="1">Log-normalCV%</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">3.00</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">125</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">3.54</td><td colspan="1" rowspan="1">0.89</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">116</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">159</td><td colspan="1" rowspan="1">5.07</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">92</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">464</td><td colspan="1" rowspan="1">6.14</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">1022</td><td colspan="1" rowspan="1">6.93</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1130</td><td colspan="1" rowspan="1">7.03</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.02</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">GeometricMean(Copies105/cells)</td><td colspan="1" rowspan="1">Mean Ln(Copies10/cells)</td><td colspan="1" rowspan="1">Repeata-bilityLn SD</td><td colspan="1" rowspan="1">BetweenRunLn SD</td><td colspan="1" rowspan="1">BetweenInstrumentLn SD</td><td colspan="1" rowspan="1">BetweenLotLn SD</td><td colspan="1" rowspan="1">TotalLn SD</td><td colspan="1" rowspan="1">Log-normalCV%</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1176</td><td colspan="1" rowspan="1">7.07</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2165</td><td colspan="1" rowspan="1">7.68</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4105</td><td colspan="1" rowspan="1">8.32</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">4146</td><td colspan="1" rowspan="1">8.33</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">4866</td><td colspan="1" rowspan="1">8.49</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">8604</td><td colspan="1" rowspan="1">9.06</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">58</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">11048</td><td colspan="1" rowspan="1">9.31</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">31</td></tr></table>

Table 11. TREC total variation results without logarithmic transformation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>Calculations without logarithmic transformation</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>Min(Copies/105 cells)</td><td rowspan=1 colspan=1>Max(Copies/105 cells)</td><td rowspan=1 colspan=1>SD(Copies/105 cells)</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>41.4</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>479</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>508</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>1183</td><td rowspan=1 colspan=1>216</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>1094</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>2247</td><td rowspan=1 colspan=1>383</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>1206</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>2576</td><td rowspan=1 colspan=1>414</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>1259</td><td rowspan=1 colspan=1>422</td><td rowspan=1 colspan=1>3011</td><td rowspan=1 colspan=1>479</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>2263</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>3084</td><td rowspan=1 colspan=1>510</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>4263</td><td rowspan=1 colspan=1>2031</td><td rowspan=1 colspan=1>7418</td><td rowspan=1 colspan=1>1220</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>4407</td><td rowspan=1 colspan=1>1014</td><td rowspan=1 colspan=1>8206</td><td rowspan=1 colspan=1>1530</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>5754</td><td rowspan=1 colspan=1>2011</td><td rowspan=1 colspan=1>74368</td><td rowspan=1 colspan=1>7120</td><td rowspan=1 colspan=1>124</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>9540</td><td rowspan=1 colspan=1>471</td><td rowspan=1 colspan=1>19530</td><td rowspan=1 colspan=1>3760</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>11501</td><td rowspan=1 colspan=1>3218</td><td rowspan=1 colspan=1>19982</td><td rowspan=1 colspan=1>3170</td><td rowspan=1 colspan=1>28</td></tr></table>

Table 12. KREC precision data across three kit lots and three instruments   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">GeometricMean(Copies105/cells)</td><td colspan="1" rowspan="1">Mean Ln(Copies105/ells)</td><td colspan="1" rowspan="1">Repeata-bilityLn SD</td><td colspan="1" rowspan="1">BetweenRunLn SD</td><td colspan="1" rowspan="1">BetweenInstrumentLn SD</td><td colspan="1" rowspan="1">BetweenLotLn SD</td><td colspan="1" rowspan="1">TotalLn SD</td><td colspan="1" rowspan="1">Log-normalCV%</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">4.44</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.20</td><td colspan="1" rowspan="1">178</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">478</td><td colspan="1" rowspan="1">6.17</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">74</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">1033</td><td colspan="1" rowspan="1">6.94</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">48</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1075</td><td colspan="1" rowspan="1">6.98</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">52</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2416</td><td colspan="1" rowspan="1">7.79</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.01</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">46</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">3361</td><td colspan="1" rowspan="1">8.12</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">39</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">GeometricMean(Copies105/cells)</td><td colspan="1" rowspan="1">Mean Ln(Copies105/cells)</td><td colspan="1" rowspan="1">Repeata-bilityLn SD</td><td colspan="1" rowspan="1">BetweenRunLn SD</td><td colspan="1" rowspan="1">BetweenInstrumentLn SD</td><td colspan="1" rowspan="1">BetweenLotLn SD</td><td colspan="1" rowspan="1">TotalLn SD</td><td colspan="1" rowspan="1">Log-normalCV%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5271</td><td colspan="1" rowspan="1">8.57</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.04</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">5541</td><td colspan="1" rowspan="1">8.62</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">31</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">8691</td><td colspan="1" rowspan="1">9.07</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">11499</td><td colspan="1" rowspan="1">9.35</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">13494</td><td colspan="1" rowspan="1">9.51</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">13767</td><td colspan="1" rowspan="1">9.53</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.13</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">39735</td><td colspan="1" rowspan="1">10.6</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">33</td></tr></table>

Table 13. KREC total variation results without logarithmic transformation   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=5>Calculations without logarithmic transformation</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>Min(Copies/105 cells)</td><td rowspan=1 colspan=1>Max(Copies/105 cells)</td><td rowspan=1 colspan=1>SD(Copies/105 cells)</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>435</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>570</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>1396</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>1139</td><td rowspan=1 colspan=1>212</td><td rowspan=1 colspan=1>4188</td><td rowspan=1 colspan=1>546</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>1195</td><td rowspan=1 colspan=1>356</td><td rowspan=1 colspan=1>3180</td><td rowspan=1 colspan=1>538</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>2605</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>5513</td><td rowspan=1 colspan=1>898</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>3568</td><td rowspan=1 colspan=1>686</td><td rowspan=1 colspan=1>7096</td><td rowspan=1 colspan=1>1150</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>5560</td><td rowspan=1 colspan=1>474</td><td rowspan=1 colspan=1>10461</td><td rowspan=1 colspan=1>1620</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>5774</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>10031</td><td rowspan=1 colspan=1>1500</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>9308</td><td rowspan=1 colspan=1>3333</td><td rowspan=1 colspan=1>19079</td><td rowspan=1 colspan=1>3420</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>12212</td><td rowspan=1 colspan=1>2707</td><td rowspan=1 colspan=1>23846</td><td rowspan=1 colspan=1>4180</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>14103</td><td rowspan=1 colspan=1>5330</td><td rowspan=1 colspan=1>23372</td><td rowspan=1 colspan=1>4030</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>14492</td><td rowspan=1 colspan=1>5673</td><td rowspan=1 colspan=1>35070</td><td rowspan=1 colspan=1>4960</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>41717</td><td rowspan=1 colspan=1>15231</td><td rowspan=1 colspan=1>74089</td><td rowspan=1 colspan=1>12400</td><td rowspan=1 colspan=1>30</td></tr></table>

All 84 plates from both studies had results of the Eonis SCID-SMA Kit Controls within the acceptance limits. The data validity-check for each well resulted in a total of 15 out of 3354 well exclusions from the analysis (sample invalid rate is $0 . 4 \%$ ). The precision of the device control kits is shown in Table 14a and the internal control RPP30 gene in Table 14b below.

Table 14a. Descriptive statistics of device kit control results (Ct values) at one site   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">N CtN/A</td><td colspan="1" rowspan="1">CtMean</td><td colspan="1" rowspan="1">Ct SD</td><td colspan="1" rowspan="1">Ct CV%</td><td colspan="1" rowspan="1">Ct Min</td><td colspan="1" rowspan="1">Ct Max</td></tr><tr><td colspan="1" rowspan="1">KREC</td><td colspan="1" rowspan="1">C1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">1.68</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">34.6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29.5</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.87</td><td colspan="1" rowspan="1">28.8</td><td colspan="1" rowspan="1">31.1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1">KREC ΔCt</td><td colspan="1" rowspan="1">C1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10.1</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">5.22</td><td colspan="1" rowspan="1">8.98</td><td colspan="1" rowspan="1">11.2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6.07</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">7.10</td><td colspan="1" rowspan="1">5.48</td><td colspan="1" rowspan="1">7.20</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1">RPP30</td><td colspan="1" rowspan="1">C1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">22.2</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">2.23</td><td colspan="1" rowspan="1">21.5</td><td colspan="1" rowspan="1">23.1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21.7</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">1.69</td><td colspan="1" rowspan="1">21.1</td><td colspan="1" rowspan="1">22.4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21.9</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">2.35</td><td colspan="1" rowspan="1">21.1</td><td colspan="1" rowspan="1">23.4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">37.5</td></tr><tr><td colspan="1" rowspan="1">TREC</td><td colspan="1" rowspan="1">C1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">2.31</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">35.4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28.6</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">2.25</td><td colspan="1" rowspan="1">27.7</td><td colspan="1" rowspan="1">30.5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1">TREC ΔCt</td><td colspan="1" rowspan="1">C1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">2.31</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">35.4</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">C3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">28.6</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">2.25</td><td colspan="1" rowspan="1">27.7</td><td colspan="1" rowspan="1">30.5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NaN</td><td colspan="1" rowspan="1">NaN</td></tr></table>

N/A not applicable; $\mathrm { N a N } = \mathrm { N o t }$ a number (i.e., no Ct value generated)

Table 14b. RPP30 precision analysis on Ct values   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Mean</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRun</td><td rowspan=1 colspan=2>BetweenInstrument</td><td rowspan=1 colspan=2>WithinKit lot</td><td rowspan=1 colspan=2>Between Kitlot</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>4.9</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>3.9</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>4.9</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>5.6</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>25.6</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>5.0</td></tr></table>

Qualitative imprecision was determined in accordance with CLSI document EP12-A2 to classify the results, using the screening performance study cut-offs (262 copies $' 1 0 ^ { 5 }$ cells for TREC, 484 copies/ $1 0 ^ { 5 }$ cells for KREC). The C5-C95 interval was determined to be 79–626 copies/ $1 0 ^ { 5 }$ cells for TREC, and 189–1064 copies $1 0 ^ { 5 }$ cells for KREC. Concentrations outside of these intervals were consistently analyte negative (concentrations ${ < } C 5$ , equals to screen positive) or consistently analyte positive (concentrations ${ \mathrm { > C } } 9 5$ , equals to screen negative).

Linearity was determined with 1 kit lot and 1 operator. Three (3) sets of contrived samples were used for this evaluation and are described below. The samples were diluted to 9 levels with 4 replicates for each level. Samples were punched at 4 replicates using a DBS puncher and then transferred into a new 96-well plate in a randomized order and were extracted and run using JANUS automated liquid handlers. Acceptance criteria were that all analytes, TREC, KREC and RPP30, must have PCR efficiency between $80 \%$ and $120 \%$ with a coefficient of determination, denoted $R ^ { 2 } \geq 9 5 \%$ within their measuring ranges. The acceptance criteria for linearity for TREC and KREC concentrations were, on their measuring ranges, the maximal difference of the best fitting model from the linear model must be $\pm 2 5 \%$ (calculated in copies/ $1 0 ^ { 5 }$ cells).

# Sample Set 1: Estimation of PCR Efficiency for TREC, KREC and RPP30

The first set of samples were prepared by spiking leucocyte-depleted human red blood cells, which had the hematocrit adjusted to 51.9, with linearized TREC and KREC plasmids and cell lines covering the required measuring ranges (TREC: 52 to 7800 copies $ { \mu \mathrm { L } }$ blood; KREC: 67 to $1 0 , 1 0 0 \mathrm { c o p i e s / \mu L }$ blood; RPP30: 2670 to 400,000 copies $\mu \mathrm { L }$ blood, per spiked cell numbers), a semi-log linear regression line plot of Ct value vs. log of input nucleic acid in the unit of copies/ $\mu \mathrm { L }$ blood was first analyzed for each individual analyte (TREC, KREC and RPP30) and the corresponding coefficient of determination $( \mathbb { R } ^ { 2 } )$ of the linear regression of the mean $\mathrm { C t }$ values and PCR efficiency of each analyte were calculated. As shown in Table 15, all 3 analytes had $\mathrm { R } ^ { 2 } > 9 9 \%$ , and their estimates of PCR efficiency were between $80 \%$ and $120 \%$ .

Table 15. Estimate of $\mathbf { R } ^ { 2 }$ and PCR efficiency of TREC, KREC and RPP30   

<table><tr><td rowspan=1 colspan=1>Analyte Name</td><td rowspan=1 colspan=1>R2</td><td rowspan=1 colspan=1>PCR efficiency estimate</td><td rowspan=1 colspan=1>Lower95% CL</td><td rowspan=1 colspan=1>Upper95% CL</td></tr><tr><td rowspan=1 colspan=1>TREC</td><td rowspan=1 colspan=1>0.993</td><td rowspan=1 colspan=1>83.9%</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>92.6%</td></tr><tr><td rowspan=1 colspan=1>KREC</td><td rowspan=1 colspan=1>0.995</td><td rowspan=1 colspan=1>87.6%</td><td rowspan=1 colspan=1>81.2%</td><td rowspan=1 colspan=1>95.0%</td></tr><tr><td rowspan=1 colspan=1>RPP30</td><td rowspan=1 colspan=1>0.992</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>89.9%</td><td rowspan=1 colspan=1>112.3%</td></tr></table>

Sample Set 2 and 3: Estimation of Linearity of the TREC and KREC Assay Linearity was determined in accordance with CLSI document EP06-A2 and was designed to evaluate the linearity of TREC and KREC in the unit of copies/ $^ { \prime } 1 0 ^ { 5 }$ cells calculated using $\Delta \mathrm { C t }$ TREC and $\Delta \mathrm { C t }$ KREC for each dilution. A high concentration sample was prepared by resuspending WBC (White Blood Cells) pellets concentrated from the $6 0 \mathrm { m L }$ of human umbilical cord bloods into 1mL if human umbilical cord blood to achieve TREC and KREC Ct close to the lowest values (i.e., high concentrations) observed in a population of newborn specimens (28.34. for TREC and 26.91 for KREC). Samples were then diluted with leukocyte depleted whole blood into nine levels of serial dilutions down to a level close to their LoQs (242 copies/105 cells for TREC and 459 copies/105 cells for KREC). In addition, a third set of samples were created to provide additional information on the KREC linearity at lower KREC concentrations because the second set did not cover the measuring range. This low concentration sample set 3 was prepared by diluting umbilical cord blood with

leucocyte-depleted human red blood. HL-60 cells were spiked into the leucocyte-depleted blood to adjust the RPP30 Ct values to match the RPP30 Ct value in cord blood. Using the observed value for undiluted sample and the consecutive dilution factors, the expected values at copies/ $1 0 ^ { 5 }$ cells scale were calculated. Linearity of TREC and KREC was then determined by fitting regression of Ln Measured vs. Ln Expected value. Weights for regression fit were calculated from the observed variability of each level. Using the obtained equation, the predicted values at each dilution level and the deviations between measured and predicted values were calculated. The descriptive statistics of each linearity sample level are presented in Table 16 and Table 17. The data for KREC represents the combined datasets. The measured results and fitted models are shown in Figure 1 and 3 the relative deviations in Figures 2 and 4. For TREC, all points were within the study acceptance criteria of $\pm 2 5 \%$ , with highest relative deviation of - $. 1 4 . 3 \%$ . For KREC, all points were within the study acceptance criteria of $\pm 2 5 \%$ , with highest relative deviation of $1 7 . 3 \%$ .

Table 16. Descriptive statistics and calculated weights - TREC   

<table><tr><td rowspan=1 colspan=1>RC*</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Copies/105Mean</td><td rowspan=1 colspan=1>Ln CopiesMean</td><td rowspan=1 colspan=1>Copies/105SD</td><td rowspan=1 colspan=1>Ln CopiesSD</td><td rowspan=1 colspan=1>Weight</td><td rowspan=1 colspan=1>Copies/105Min</td><td rowspan=1 colspan=1>Copies/105Max</td></tr><tr><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4316</td><td rowspan=1 colspan=1>8.37</td><td rowspan=1 colspan=1>461</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>3790</td><td rowspan=1 colspan=1>4890</td></tr><tr><td rowspan=1 colspan=1>0.690</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2441</td><td rowspan=1 colspan=1>7.80</td><td rowspan=1 colspan=1>834</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>1430</td><td rowspan=1 colspan=1>3330</td></tr><tr><td rowspan=1 colspan=1>0.476</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1588</td><td rowspan=1 colspan=1>7.37</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1110</td><td rowspan=1 colspan=1>1910</td></tr><tr><td rowspan=1 colspan=1>0.328</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1353</td><td rowspan=1 colspan=1>7.21</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>1100</td><td rowspan=1 colspan=1>1710</td></tr><tr><td rowspan=1 colspan=1>0.226</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1287</td><td rowspan=1 colspan=1>7.16</td><td rowspan=1 colspan=1>260</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>999</td><td rowspan=1 colspan=1>1630</td></tr><tr><td rowspan=1 colspan=1>0.156</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>508</td><td rowspan=1 colspan=1>6.23</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>630</td></tr><tr><td rowspan=1 colspan=1>0.108</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>185</td><td rowspan=1 colspan=1>5.22</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>7.93</td><td rowspan=1 colspan=1>66.5</td><td rowspan=1 colspan=1>340</td></tr><tr><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>4.74</td><td rowspan=1 colspan=1>64.5</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>82.8</td><td rowspan=1 colspan=1>198</td></tr><tr><td rowspan=1 colspan=1>0.051</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>4.54</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>6.57</td><td rowspan=1 colspan=1>40.8</td><td rowspan=1 colspan=1>190</td></tr></table>

\*Relative concentration

![](images/2082b260a7015f60f9dc821363b959b6a059f9746e0d098ec68e9d4edd2f1a68.jpg)  
Figure 1. Fitted linear model TREC

![](images/33833c359ffc750f145e1e4579d2f4612bda4d9169cedae8c06ea1ffaeb84298.jpg)  
Figure 2. Relative deviations and the study limits – TREC

Table 17. Descriptive statistics and calculated weights - KREC   

<table><tr><td colspan="1" rowspan="1">RC*</td><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Hitrate</td><td colspan="1" rowspan="1">Copies/105cells Mean</td><td colspan="1" rowspan="1">LnCopiesMean</td><td colspan="1" rowspan="1">LnCopiesSD</td><td colspan="1" rowspan="1">Weight</td><td colspan="1" rowspan="1">Copies/105cells Min</td><td colspan="1" rowspan="1">Copies/105cells Max</td></tr><tr><td colspan="1" rowspan="1">1.000</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">24343</td><td colspan="1" rowspan="1">10.1</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">204</td><td colspan="1" rowspan="1">19700</td><td colspan="1" rowspan="1">26800</td></tr><tr><td colspan="1" rowspan="1">0.690</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">15835</td><td colspan="1" rowspan="1">9.67</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">12500</td><td colspan="1" rowspan="1">19500</td></tr><tr><td colspan="1" rowspan="1">0.476</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">3*</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">10405</td><td colspan="1" rowspan="1">9.25</td><td colspan="1" rowspan="1">0.03</td><td colspan="1" rowspan="1">3330</td><td colspan="1" rowspan="1">10100</td><td colspan="1" rowspan="1">10800</td></tr><tr><td colspan="1" rowspan="1">0.328</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">8519</td><td colspan="1" rowspan="1">9.05</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">1600</td><td colspan="1" rowspan="1">7990</td><td colspan="1" rowspan="1">9020</td></tr><tr><td colspan="1" rowspan="1">0.226</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6768</td><td colspan="1" rowspan="1">8.82</td><td colspan="1" rowspan="1">0.08</td><td colspan="1" rowspan="1">625</td><td colspan="1" rowspan="1">6290</td><td colspan="1" rowspan="1">7470</td></tr><tr><td colspan="1" rowspan="3">1.000</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6438</td><td colspan="1" rowspan="1">8.77</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">296</td><td colspan="1" rowspan="1">4170</td><td colspan="1" rowspan="1">11500</td></tr><tr><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5064</td><td colspan="1" rowspan="1">8.53</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">2380</td><td colspan="1" rowspan="1">8880</td></tr><tr><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3905</td><td colspan="1" rowspan="1">8.27</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">154</td><td colspan="1" rowspan="1">2500</td><td colspan="1" rowspan="1">8160</td></tr><tr><td colspan="1" rowspan="1">0.156</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3533</td><td colspan="1" rowspan="1">8.17</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">19.8</td><td colspan="1" rowspan="1">2170</td><td colspan="1" rowspan="1">5800</td></tr><tr><td colspan="1" rowspan="1">0.700</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3072</td><td colspan="1" rowspan="1">8.03</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">184</td><td colspan="1" rowspan="1">1440</td><td colspan="1" rowspan="1">5130</td></tr><tr><td colspan="1" rowspan="1">0.108</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2836</td><td colspan="1" rowspan="1">7.95</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">1720</td><td colspan="1" rowspan="1">3750</td></tr><tr><td colspan="1" rowspan="1">0.600</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2644</td><td colspan="1" rowspan="1">7.88</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">195</td><td colspan="1" rowspan="1">1500</td><td colspan="1" rowspan="1">4650</td></tr><tr><td colspan="1" rowspan="1">0.074</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">3*</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2490</td><td colspan="1" rowspan="1">7.82</td><td colspan="1" rowspan="1">0.16</td><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">2090</td><td colspan="1" rowspan="1">2790</td></tr><tr><td colspan="1" rowspan="1">0.500</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2080</td><td colspan="1" rowspan="1">7.64</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">567</td><td colspan="1" rowspan="1">4320</td></tr><tr><td colspan="1" rowspan="1">0.300</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1588</td><td colspan="1" rowspan="1">7.37</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">80.0</td><td colspan="1" rowspan="1">375</td><td colspan="1" rowspan="1">3540</td></tr><tr><td colspan="1" rowspan="1">0.051</td><td colspan="1" rowspan="1">Lin</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1249</td><td colspan="1" rowspan="1">7.13</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">29.2</td><td colspan="1" rowspan="1">780</td><td colspan="1" rowspan="1">1910</td></tr><tr><td colspan="1" rowspan="1">0.200</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">1188</td><td colspan="1" rowspan="1">7.08</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">147</td><td colspan="1" rowspan="1">619</td><td colspan="1" rowspan="1">2200</td></tr><tr><td colspan="1" rowspan="1">0.150</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">871</td><td colspan="1" rowspan="1">6.77</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">71.2</td><td colspan="1" rowspan="1">356</td><td colspan="1" rowspan="1">2850</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">804</td><td colspan="1" rowspan="1">6.69</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">48.8</td><td colspan="1" rowspan="1">82.9</td><td colspan="1" rowspan="1">1570</td></tr><tr><td colspan="1" rowspan="1">0.100</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">757</td><td colspan="1" rowspan="1">6.63</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">86.0</td><td colspan="1" rowspan="1">344</td><td colspan="1" rowspan="1">2970</td></tr><tr><td colspan="1" rowspan="2">0.0800.040</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">478</td><td colspan="1" rowspan="1">6.17</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">23.6</td><td colspan="1" rowspan="1">16.2</td><td colspan="1" rowspan="1">1580</td></tr><tr><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">305</td><td colspan="1" rowspan="1">5.72</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">13.2</td><td colspan="1" rowspan="1">54.9</td><td colspan="1" rowspan="1">1030</td></tr><tr><td colspan="1" rowspan="2">0.080</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">296</td><td colspan="1" rowspan="1">5.69</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">74.1</td><td colspan="1" rowspan="1">936</td></tr><tr><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">255</td><td colspan="1" rowspan="1">5.54</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">75.2</td><td colspan="1" rowspan="1">620</td></tr><tr><td colspan="1" rowspan="1">0.030</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">226</td><td colspan="1" rowspan="1">5.42</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">28.1</td><td colspan="1" rowspan="1">52.9</td><td colspan="1" rowspan="1">480</td></tr><tr><td colspan="1" rowspan="1">0.020</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">5.30</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">17.4</td><td colspan="1" rowspan="1">152</td><td colspan="1" rowspan="1">411</td></tr><tr><td colspan="1" rowspan="1">0.060</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">187</td><td colspan="1" rowspan="1">5.23</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">19.1</td><td colspan="1" rowspan="1">29.8</td><td colspan="1" rowspan="1">804</td></tr><tr><td colspan="1" rowspan="1">0.030</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">178</td><td colspan="1" rowspan="1">5.18</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">43.1</td><td colspan="1" rowspan="1">75.7</td><td colspan="1" rowspan="1">407</td></tr><tr><td colspan="1" rowspan="1">0.010</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">5.16</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">22.0</td><td colspan="1" rowspan="1">80.1</td><td colspan="1" rowspan="1">608</td></tr><tr><td colspan="1" rowspan="1">0.040</td><td colspan="1" rowspan="1">L3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0.85</td><td colspan="1" rowspan="1">166</td><td colspan="1" rowspan="1">5.11</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">16.0</td><td colspan="1" rowspan="1">10.5</td><td colspan="1" rowspan="1">392</td></tr><tr><td colspan="1" rowspan="1">0.020</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">5.02</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">16.0</td><td colspan="1" rowspan="1">22.2</td><td colspan="1" rowspan="1">502</td></tr><tr><td colspan="1" rowspan="1">0.010</td><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">147</td><td colspan="1" rowspan="1">4.99</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">77.9</td><td colspan="1" rowspan="1">61.7</td><td colspan="1" rowspan="1">190</td></tr><tr><td colspan="1" rowspan="1">0.030</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">4.86</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">30.8</td><td colspan="1" rowspan="1">44.3</td><td colspan="1" rowspan="1">466</td></tr><tr><td colspan="1" rowspan="1">0.010</td><td colspan="1" rowspan="1">L1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">4.76</td><td colspan="1" rowspan="1">0.56</td><td colspan="1" rowspan="1">9.57</td><td colspan="1" rowspan="1">61.8</td><td colspan="1" rowspan="1">166</td></tr></table>

\*Relative Concentration

![](images/1cbdf51e7cfc8df722ffc10953a1eb80123e6fafaf39c9b5daaebf1eefc2c902.jpg)  
Figure 3. Fitted linear model – KREC

![](images/961c212f322b68e5e07a56a8ed780e87d674e93bc4467ac170945c8e4f19cdd9.jpg)  
Figure 4. Relative deviations and the study limits – KREC

The TREC analyte is demonstrated to be linear from 94 copies $1 0 ^ { 5 }$ cells to 4316 copies/ $1 0 ^ { 5 }$ cells with observed maximum deviation of - $- 1 4 . 3 \%$ . The KREC analyte is demonstrated to be linear from 117 to 24343 copies $^ { \prime } 1 0 ^ { 5 }$ cells with observed maximum deviation of $1 7 . 3 \%$ .

# 3. Analytical Specificity

(i) Interference

The Eonis SCID-SMA kit was evaluated for interference from the following potential endogenous and exogenous sources:

Hemoglobin up to $2 0 0 { \mathrm { g / L } }$   
• Conjugated bilirubin up to $1 6 . 6 \ : \mathrm { m g / d L }$ (166mg/L)   
• Unconjugated bilirubin up to 10mg/dL $( 1 0 0 \mathrm { m g / L } )$ Intralipid up to $1 5 0 0 \mathrm { m g / d L }$ $1 5 \mathrm { m g / m L }$ )   
• Li-hep up to 7500 USP/dL Na Citrate up to $0 . 0 6 4 5 ~ \mathrm { m o l / L }$ EDTA up to $9 . 8 ~ \mathrm { m g / m L }$

The interference study was performed by 3 operators using 1 Eonis SCID-SMA Kit and 3 QuantStudio Dx Real-time PCR instruments. Three levels of TREC and KREC target DNA (normal, near cut-off and endogenous were tested in conjunction with each level of interferent (low and high). Blood samples were collected from healthy individuals and the hematocrit was adjusted to $40 \%$ corresponding to the hematocrit of neonates (except blood used in the hemoglobin sample series). Blood samples were then spiked with TREC and KREC plasmid DNA to obtain the desired concentration levels. Blood samples with the added potential interferent were used to prepare dried blood spots. The blood spots were dried overnight and stored at $- 8 0 ^ { \circ } \mathrm { C }$ . The specimens used are shown in Table 18.

Table 18. Samples used in the interference study   

<table><tr><td rowspan=1 colspan=1>Sample Name(Abbreviation)</td><td rowspan=1 colspan=1>TREC Level</td><td rowspan=1 colspan=1>KREC Level</td><td rowspan=1 colspan=1>InterferentSubstance Level</td></tr><tr><td rowspan=1 colspan=1>Normal Targets+ LowInterferent (NL)</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>Low (0%)</td></tr><tr><td rowspan=1 colspan=1>Normal Targets+ HighInterferent (NH)</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>High (100%)</td></tr><tr><td rowspan=1 colspan=1>Cut-off Targets + LowInterferent (CL)</td><td rowspan=1 colspan=1>Near Cut-off</td><td rowspan=1 colspan=1>Near Cut-off</td><td rowspan=1 colspan=1>Low (0%)</td></tr><tr><td rowspan=1 colspan=1>Cut-off Targets + HighInterferent (CH)</td><td rowspan=1 colspan=1>Near Cut-off</td><td rowspan=1 colspan=1>Near Cut-off</td><td rowspan=1 colspan=1>High (100%)</td></tr><tr><td rowspan=1 colspan=1>Endogenous Targets + LowInterferent (EL)</td><td rowspan=1 colspan=1>Endogenous</td><td rowspan=1 colspan=1>Endogenous</td><td rowspan=1 colspan=1>Low (0%)</td></tr><tr><td rowspan=1 colspan=1>Endogenous Targets + HighInterferent (EH)</td><td rowspan=1 colspan=1>Endogenous</td><td rowspan=1 colspan=1>Endogenous</td><td rowspan=1 colspan=1>High (100%)</td></tr></table>

The study generated a total of 544 sample results not including controls (7 interfering substances $_ { \textrm { X 2 } }$ interferent levels x 3 target DNA levels x 13 repeats). Acceptance criteria were based on observed imprecision in the precision studies. The results indicated there was no interference observed for any of the tested interferents. Hemoglobin at up to $2 0 0 \ \mathrm { g / L }$ , conjugated bilirubin at up to $1 6 . 6 \ : \mathrm { m g / d L }$ $\mathrm { ( = 1 6 6 ~ m g / L ) }$ , unconjugated bilirubin at up to 10 $\mathrm { m g / d L }$ $\scriptstyle ( = 1 0 0 \mathrm { m g / L } )$ , Intralipid at up to $1 5 0 0 \mathrm { m g / d L }$ $( 1 5 \mathrm { m g / m L } )$ , Li-heparin at up to 7500 USP/dL, Na-citrate at up to $0 . 0 6 4 5 \mathrm { m o l / L }$ , and EDTA at up to $9 . 8 ~ \mathrm { m g / m L }$ did not significantly interfere with the Eonis SCID-SMA kit.

# (ii) Primer and Probe Specificity

An in silico study was performed to assess the specificity of the primers and probes used in the Eonis assay. Primer and probe specificity was assessed by using the online Standard Nucleotide BLAST tool (Optimize for highly similar sequences, megablast) against the Human genomic plus transcript (Human $\mathrm { ~ G ~ } { + } \mathrm { ~ T ~ } )$ database published by the National Center for Biotechnology Information (NCBI).

The BLAST analysis confirmed that there were only hits with $100 \%$ homology to the T receptor alpha delta locus (TREC) on chromosome 14 along, the full length of the primer and probe sequences, and $100 \%$ homology to the immunoglobulin kappa locus (IGK) on chromosome 2, and to the ribonuclease $\mathrm { P }$ subunit $\mathsf { p } 3 0$ locus (RPP30) on chromosome 10. There was no significant homology to other genome sequences.

4. Assay Reportable Range: Refer to the Results interpretation section.

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

(i) Traceability

There are no recognized standards or reference material for TREC or KREC DNA. The kit contains controls employed in the determination of TREC and KREC values.

# (ii) Specimen Stability

The influence of storage time, temperature, and humidity was studied using DBS samples prepared with two brands of filter paper. Samples were prepared by mixing cell lines and red blood cell concentrate (mimicking a true positive sample for all 3 conditions), using cord blood (mimicking a true negative sample) or by mixing cell lines, red blood cell concentrate and cord blood for the near cut-off level samples.

After collection and preparation, the blood spots were dried overnight at ambient temperature $( + 1 9 - + 2 5 ^ { \circ } \mathrm { C } )$ . The dried blood spots were then stored at different storage conditions. Storage of specimens in an environment with elevated temperatures and humidity increases the risk of inaccurate TREC/KREC results in normal or near cut-off samples. The TREC/KREC results are shown in Figure 5a and Figure 5b.

![](images/aa80c703e81edc759133a3c9a729cda9a6982dc2bd1b28de1543ea015576cdba.jpg)

![](images/f0c37e8a53d3f07b314daccdd3d1ff0a084bd6e6fafe020b459b13e67135ccd7.jpg)

![](images/d0d0c34bef0481943789b9894239fc08e569fd2e474217889e66f02030cd16fa.jpg)

![](images/f9f7650cd720b7cded70ef6f9a6a8b665c81055c12b52eb729a436b1a8baa4d0.jpg)

![](images/f7d3bb27fdc75c512b045bfeeaa5d318de1f84c973a3d99868d31be722783735.jpg)  
Figure 5a. TREC copies across different storage conditions in samples mimicking true positive (Sample 1), true negative (Sample 4) and in-between (Sample 2 and Sample 3)

![](images/5146d00dc04473ab02cd511cdbf94eecea1624cc0b42b76da521e64b60757f59.jpg)  
Figure 5b. KREC copies across different storage conditions in samples mimicking true positive (Sample 1), true negative (Sample 4) and in-between (Sample 2 and Sample 3)

(iii) Reagent Stability:

Stability studies were conducted to demonstrate the real time, in-use, and on-board stability as well as transport shipping stress simulation of the Eonis DNA Extraction Kit components and the Eonis SCID-SMA kit. The tests were performed after a transport simulation treatment that simulates the worst-case conditions that can occur during packaging and shipping. When a transport simulation treatment was performed, all the kit components were used in the study. The stability testing was conducted in accordance with guideline EN ISO 23640 In vitro diagnostic medical devices – Evaluation of stability of in vitro diagnostic reagents (2015) and CLSI Guideline EP25-A Evaluation of Stability of In Vitro Diagnostic Reagents, Approved Guideline- Volume 29, No. 20 (2009).

The testing was performed at 7 months of the components’ shelf life. All Mean Ct values and variation (standard deviation) results fulfilled the acceptance criteria at all tested time points. At all tested time points, the results passed the data validity check. (Refer to DEN200044 for details of the study design.) The following stability conditions apply based on the results:

Eonis DNA Extraction Kit components (Wash Solution and Elution Solution) are stable for 14 days at $+ 1 9 - + 2 5 ^ { \circ } \mathrm { C }$ after first use.   
• Eonis DNA extraction Kit components are stable in troughs for 30 minutes at $+ 1 9$ - $+ 2 5 ^ { \circ } \mathrm { C }$ . Note: The Eonis DNA extraction Kit components are stable on board the JANUS Extraction instrument in troughs for 2 hours.   
• PCR Reagents 1 and 2 once thawed from $- 3 0 ^ { \circ } \mathrm { C }$ to $- 1 6 \mathrm { } ^ { \circ } \mathrm { C }$ , are stable for 14 days at $+ 2 { } ^ { \circ } \mathrm { C }$ to $+ 8 { } ^ { \circ } \mathrm { C }$ . PCR Reagents 1 and 2 are stable in their tubes uncapped on-board the JANUS PCR Mastermix instrument for 2 hours at $+ 1 9$ to $+ 2 5 ^ { \circ } \mathrm { C }$ .   
• The mixture of PCR Reagents 1 and 2 is stable for 15 minutes in a trough at $+ 1 9$ to $+ 2 5 ^ { \circ } \mathrm { C }$ . The mixture of PCR Reagents 1 and 2 is stable for 2 hours in a sealed plate protected from light.   
• The SCID-SMA Kit controls are stable for 14 days after first use when stored at - $3 0 \mathrm { { } ^ { \circ } C }$ to $- 1 6 ^ { \circ } \mathrm { C }$ in a sealed bag with desiccant.

The SCID-SMA Kit controls are stable on-board the JANUS Extraction instrument at $+ 1 9$ to $+ 2 5 ^ { \circ } \mathrm { C }$ for 4 hours after they are punched into 96-well plates. • Based on the results of the real-time and transport simulation stability study a shelf life 180 days (6 months) is claimed for all Eonis SCID-SMA kit components, when stored at the labeled storage temperature of $- 3 0 ^ { \circ } \mathrm { C } \mathrm { ~ -- } 1 6 ^ { \circ } \mathrm { C }$ .

# 6. Detection Limit:

(i) Limit of Blank (LoB)

The LoB is the highest measurement result that is likely to be observed for a blank sample that contains no analyte. The LoB study was performed using contrived analyte negative samples created by spiking a Coriell cell line into leukocyte depleted blood which was TREC and KREC negative. Five (5) individual contrived analyte-negative samples were prepared by spiking the Coriell cell line into 5 different lots of leukocyte-depleted human blood with the hematocrit adjusted to $40 \%$ . The study was performed by 1 operator using 2 sets of Instrument Systems and $2 \mathrm { k i t }$ lots run over 5 days for a total of 10 runs. Six replicates were tested in each run totaling 300 results (150 results/kit lot). For the RPP30 LoB, blank samples were used (no punch/no template DNA), 20 replicates were tested in each run totaling 200 results (100 results/kit lot). Each plate contained kit controls NTC, C1, C2 and C3, assayed in duplicate. Acceptance criteria were that the LoB of RPP30 must be zero; or any samples that have $\mathrm { R P P } 3 0 \mathrm { C t } \geq 3 2 . 0 0$ will be reported as ‘Invalid’ for TREC and KREC. The LoB of TREC and KREC must be no greater than the LoD.

For TREC and KREC, all of the replicates had no Ct value reported. For RPP30, there were two replicates having $\mathrm { C t }$ values reported at 34.49 and 36.23; however, as this would lead to an invalid result for the sample, the percentage of false-positive results for RPP30, defined as the percentage of replicates of the blank sample that had a valid RPP30 Ct value $\leq 3 2 . 0 0$ was also zero for both kit lots. The LoB of TREC is 0 copies $\mu \mathrm { L }$ blood, and the LoB of KREC is 0 copies/ $\mu \mathrm { L }$ blood.

# (ii) Limit of Detection (LoD) and Limit of Quantitation (LoQ)

LoD and LoQ were evaluated separately for TREC and KREC. The LoD and LoQ studies were performed using contrived samples prepared by diluting cord blood with adult leucocyte-depleted human red blood that had TREC and/or KREC levels that were undetectable by the Eonis SCID-SMA Kit. HL-60 cells were spiked into the adult leucocytedepleted blood to adjust the RPP30 Ct values to match the RPP30 Ct value in cord blood (within $1 \mathrm { C t }$ difference). Each cord blood sample was diluted into different levels in which at least the lowest three levels had yield hit rates within the range of 0.10 to 0.90 using the corresponding diluents. The samples were then spotted onto filter paper and dried overnight. For each analyte, TREC or KREC, the study was performed using 2 sets of instruments and 2 kit lots. One operator performed 2 runs per day, for 4 days, totaling 8 runs.

For each sample and each dilution, including negative samples (diluents), 5 replicates were tested in each run, totaling 960 results with 20 replicates per dilution per lot. The final dilutions used in the study and the corresponding number of positive results observed, total number of measurements and hit rates observed for TREC and KREC are summarized in Tables 19 and 20, respectively.

Table 19. TREC calculated hit rate percentages with sample concentrations in copies $/ 1 0 ^ { 5 }$ cells   

<table><tr><td rowspan=1 colspan=1>KitLot</td><td rowspan=1 colspan=1>CordBloodSample</td><td rowspan=1 colspan=1>SampleLevel</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Concentrationin copies/105cells</td><td rowspan=1 colspan=1>ObservedPositives</td><td rowspan=1 colspan=1>ObservedNegatives</td><td rowspan=1 colspan=1>HitRate%</td></tr><tr><td rowspan=28 colspan=1>1</td><td rowspan=9 colspan=1>L1</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>973</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>528</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>428</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>5.00</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>70%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>40%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=9 colspan=1>L2</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>2670</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>45%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>50%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>35%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=10 colspan=1>L3</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1525</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>365</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>194</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>65%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>30%</td></tr><tr><td rowspan=1 colspan=1>Level 8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>L1</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1141</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>395</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr></table>

Table 20. KREC calculated hit rate percentages with sample concentrations in copies/105 cells   

<table><tr><td rowspan=1 colspan=1>KitLot</td><td rowspan=1 colspan=3>CordBloodSample</td><td rowspan=1 colspan=1>SampleLevel</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Concentrationin copies/105cells</td><td rowspan=1 colspan=1>ObservedPositives</td><td rowspan=1 colspan=1>ObservedNegatives</td><td rowspan=1 colspan=1>HitRate%</td></tr><tr><td rowspan=27 colspan=1></td><td rowspan=7 colspan=3>′</td><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>596</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>5.00</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>65%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>2490</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1>−</td><td rowspan=2 colspan=1>Level 2</td><td rowspan=2 colspan=1>10.0</td><td rowspan=2 colspan=1>118</td><td rowspan=2 colspan=1>3</td><td rowspan=2 colspan=1>17</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=6 colspan=3>L2</td><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>60%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>70%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>45%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=10 colspan=3>L3</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>1510</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>321</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>136</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>70%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>45%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>35%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25%</td></tr><tr><td rowspan=1 colspan=1>Level 8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

<table><tr><td colspan="1" rowspan="1">KitLot</td><td colspan="1" rowspan="1">Cord BloodSample</td><td colspan="1" rowspan="1">SampleLevel</td><td colspan="1" rowspan="1">Dilution</td><td colspan="1" rowspan="1">Concentrationin copies/105cells</td><td colspan="1" rowspan="1">ObservedPositives</td><td colspan="1" rowspan="1">ObservedNegatives</td><td colspan="1" rowspan="1">Hit Rate%</td></tr><tr><td colspan="1" rowspan="3">1</td><td colspan="1" rowspan="3">L1</td><td colspan="1" rowspan="1">Cord Blood</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">3905</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 1</td><td colspan="1" rowspan="1">1.43</td><td colspan="1" rowspan="1">3072</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 2</td><td colspan="1" rowspan="1">1.67</td><td colspan="1" rowspan="1">2644</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="25"></td><td colspan="1" rowspan="6">b</td><td colspan="1" rowspan="1">Level 3</td><td colspan="1" rowspan="1">2.00</td><td colspan="1" rowspan="1">2080</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 4</td><td colspan="1" rowspan="1">5.00</td><td colspan="1" rowspan="1">1188</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 5</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">255</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="1">Level 6</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="1">Level 7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">15%</td></tr><tr><td colspan="1" rowspan="1">Diluent</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="9">L2</td><td colspan="1" rowspan="1">Cord Blood</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">6438</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 1</td><td colspan="1" rowspan="1">6.67</td><td colspan="1" rowspan="1">871</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 2</td><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">757</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 3</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">478</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 4</td><td colspan="1" rowspan="1">25.0</td><td colspan="1" rowspan="1">305</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">70%</td></tr><tr><td colspan="1" rowspan="1">Level 5</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">226</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="1">Level 6</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">151</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">65%</td></tr><tr><td colspan="1" rowspan="1">Level 7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">147</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">45%</td></tr><tr><td colspan="1" rowspan="1">Diluent</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="10">L3</td><td colspan="1" rowspan="1">Cord Blood</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">5064</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 1</td><td colspan="1" rowspan="1">3.33</td><td colspan="1" rowspan="1">1588</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 2</td><td colspan="1" rowspan="1">6.67</td><td colspan="1" rowspan="1">804</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 3</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">296</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">90%</td></tr><tr><td colspan="1" rowspan="1">Level 4</td><td colspan="1" rowspan="1">16.7</td><td colspan="1" rowspan="1">187</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">90%</td></tr><tr><td colspan="1" rowspan="1">Level 5</td><td colspan="1" rowspan="1">25.0</td><td colspan="1" rowspan="1">166</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">85%</td></tr><tr><td colspan="1" rowspan="1">Level 6</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">178</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">74%</td></tr><tr><td colspan="1" rowspan="1">Level 7</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">20%</td></tr><tr><td colspan="1" rowspan="1">Level 8</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">174</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">45%</td></tr><tr><td colspan="1" rowspan="1">Diluent</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="10">2</td><td colspan="1" rowspan="9">L1</td><td colspan="1" rowspan="1">Cord Blood</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">4064</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 1</td><td colspan="1" rowspan="1">1.43</td><td colspan="1" rowspan="1">4146</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 2</td><td colspan="1" rowspan="1">1.67</td><td colspan="1" rowspan="1">3828</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 3</td><td colspan="1" rowspan="1">2.00</td><td colspan="1" rowspan="1">2039</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 4</td><td colspan="1" rowspan="1">5.00</td><td colspan="1" rowspan="1">846</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Level 5</td><td colspan="1" rowspan="1">12.5</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">90%</td></tr><tr><td colspan="1" rowspan="1">Level 6</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">191</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">70%</td></tr><tr><td colspan="1" rowspan="1">Level 7</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">25%</td></tr><tr><td colspan="1" rowspan="1">Diluent</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">L2</td><td colspan="1" rowspan="1">Cord Blood</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">5825</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">100%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>KitLot</td><td rowspan=1 colspan=2>Cord BloodSample</td><td rowspan=1 colspan=1>SampleLevel</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Concentrationin copies/105cells</td><td rowspan=1 colspan=1>ObservedPositives</td><td rowspan=1 colspan=1>ObservedNegatives</td><td rowspan=1 colspan=1>Hit Rate%</td></tr><tr><td rowspan=18 colspan=1></td><td rowspan=8 colspan=2>b</td><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>1064</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>556</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>412</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>245</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>65%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>134</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>50%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=10 colspan=2>L3</td><td rowspan=1 colspan=1>Cord Blood</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>4915</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 1</td><td rowspan=1 colspan=1>3.33</td><td rowspan=1 colspan=1>1510</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Level 2</td><td rowspan=1 colspan=1>6.67</td><td rowspan=1 colspan=1>659</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Level 3</td><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>Level 4</td><td rowspan=1 colspan=1>16.7</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Level 5</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>Level 6</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>Level 7</td><td rowspan=1 colspan=1>50.0</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>Level 8</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>25%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr></table>

The LoD was calculated using Probit analysis at $9 5 \%$ probability for each reagent kit lot. Probit analysis in copies/ $1 0 ^ { 5 }$ cells unit was performed. The data collected from all 3 cord blood samples and their dilutions were pooled for the calculation. The results for both kit lots with $9 5 \%$ confidence intervals for TREC and KREC are summarized in Tables 21 and 22, respectively. The LoD estimate is the concentration value that Probit analysis gives an estimated hit rate of $9 5 \%$ .

Table 21. TREC LoD estimates based on Probit analysis with hit rate percentages and sample concentrations in copies $/ 1 0 ^ { 5 }$ cells unit   

<table><tr><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>LoD estimate(copies/105cells)</td><td rowspan=1 colspan=1>95% confidenceinterval</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>137 - 242</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>184 - 360</td></tr></table>

Table 22. KREC LoD estimates based on Probit analysis with hit rate percentages and sample concentrations in copies $/ 1 0 ^ { 5 }$ cells unit   

<table><tr><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>LoD estimate(copies/105cells)</td><td rowspan=1 colspan=1>95% confidenceinterval</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>371 - 669</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>292 - 439</td></tr></table>

The LoQ was evaluated as the functional sensitivity that represents the measurand concentration associated with a desired within-laboratory precision, as exemplified by CLSI guideline EP17-A2 Appendix D. Only the dilutions that yielded $100 \%$ hit rates were included in the LoQ calculation. For each qualified dilution, the mean and the SD of the concentrations in Ln (copies/ $1 0 ^ { 5 }$ cells) were calculated. A power function model (SD vs. the mean concentration) was used to fit the datasets for each reagent kit lot. The LoQ estimate for each reagent kit lot was determined as the predicted lowest concentration that has withinlaboratory precision equal to an $\mathrm { S D } \le 0 . 9 0 \mathrm { L n }$ (copies/ $1 0 ^ { 5 }$ cells) for TREC and $\leq 1 . 4 9 \mathrm { L n }$ (copies $1 0 ^ { 5 }$ cells) for KREC. The results are shown in Tables 23 and 24.

Table 23. TREC LoQ estimates based on conservative approach in copies $/ 1 0 ^ { 5 }$ cells   

<table><tr><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>LoD (copies/105 cells)</td><td rowspan=1 colspan=1>LoQ estimate(copies/105 cells)</td><td rowspan=1 colspan=1>Final LoQ (=LoD)(copies/105 cells)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>30</td><td rowspan=2 colspan=1>242</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>242</td><td rowspan=1 colspan=1>123</td></tr></table>

Table 24. KREC LoQ estimates based on conservative approach in copies $/ 1 0 ^ { 5 }$ cells   

<table><tr><td rowspan=1 colspan=1>Kit lot</td><td rowspan=1 colspan=1>LoD (copies/105 cells)</td><td rowspan=1 colspan=1>LoQ estimate(copies/105 cells)</td><td rowspan=1 colspan=1>Final LoQ (=LoD)(copies/105 cells)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>459</td><td rowspan=1 colspan=1>7</td><td rowspan=2 colspan=1>459</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>0</td></tr></table>

The following results were determined for LoD and LoQ of TREC and KREC:

$\mathrm { L o D } = \mathrm { L o Q }$ TREC: 242 copies/ $1 0 ^ { 5 }$ cells KREC: 459 copies $1 0 ^ { 5 }$ cells

7. Assay Cut-Off: Refer to the Clinical Cut-off in Section VII.D.

8. Accuracy (Instrument): Not applicable

A study was conducted to evaluate the potential for sample carryover to demonstrate control of contamination during testing of samples with the Eonis reagent kits. The carryover test was performed using 1 kit lot. The high-analyte positive sample was prepared by spiking plasmid DNAs of each analyte into leukocyte-depleted human blood at levels lower than 29.71 for TREC Ct, 29.22 for KREC Ct, and 22.31 for RPP30 Ct, corresponding to the $10 \%$ quantiles of a population of newborn specimens. The high analyte-positive sample and the analyte-negative sample were placed in alternating rows and columns to produce a checkerboard configuration. The observed carryover percentage was $0 \%$ for the three analytes.

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Not applicable

2. Matrix Comparison:

# Filter Paper Reproducibility

A filter paper reproducibility study was conducted using filter paper from 2 manufacturers in order to evaluate manufacturer difference, lot differences, as well as to assess homogeneity across dried blood spots (DBS). Between paper manufacturer and paper lots were evaluated using samples spotted on 3 lots of each filter paper brand.

The study was conducted using a subset of the samples that were used in the precision and 3- site reproducibility studies (although some at different concentrations) covering high, normal and low levels of each analyte (Table 5).

Sample 1 High TREC: Prepared by spiking TREC plasmid into cord blood at 2000 copies $ { / \mu \mathrm { L } }$ with hematocrit adjusted to 51.8 (approx. Ct value 28.19) Sample 2 High KREC: Prepared by spiking TREC plasmid into cord blood at 2000 copies $ { { } ^ { \prime } \mathrm { \mu L } }$ with hematocrit adjusted to 51.8 (approx. Ct value 28.65) Sample 3: Prepared by spotting cord blood with hematocrit adjusted to 51.8 onto 3 lots of each filter paper brand Sample 11: Prepared by spiking TREC plasmid at 700 copies/ $\mu \mathrm { L }$ , KREC plasmid at 1000 copies $\mu \mathrm { L }$ and Coriell cells at 15000 cells/ $\mu \mathrm { L }$ into leukocyte depleted blood with hematocrit adjusted to 49. • Sample 12 Low levels (above cut-off) TREC and KREC: Prepared by diluting $4 0 0 \mu \mathrm { L }$ of cord blood with hematocrit adjusted to 50.9 in $6 0 0 \mu \mathrm { L }$ of adult whole blood with hematocrit adjusted to 52.9 (approx. TREC Ct value 33.29 and approx. KREC Ct value 32.20) Sample 13 Low levels (below cut-off) TREC and KREC: Prepared by spotting adult whole blood with hematocrit adjusted to 52.9 onto 3 lots of each paper brand (approx. TREC Ct value no Ct and approx. KREC Ct value 3.44)

Each analyte was manually spotted on 3 filter paper lots at $8 5 ~ \mu \mathrm { L }$ per spot.

One operator performed 1 run per day for 5 days for a total of 5 runs and using 1 kit lot and 1 Eonis SCID-SMA test system. Six samples were prepared on 3 lots of each filter paper brand for a total of 36 conditions (6 samples x 3 lots $_ { \textrm { X 2 } }$ filter paper brands $= 3 6$ ). Each run consisted of 3 96-well extraction plates (2 randomized sample sets per plate plus 2 sets of kit controls) and 1 384-well PCR plate in each testing day.

To assess the homogeneity of the analytes across the DBS, 5 replicates of each condition were punched from 5 different locations of each spot in the following order: top, left, center, right, bottom. The study generated 900 results (25 replicates $\textrm { x 6 }$ samples $_ { \textrm { X 3 } }$ filter paper lots $_ { \textrm { X 2 } }$ filter paper brands) with 150 results per sample and 450 results per filter paper brand. The total variation lnSD results are compared to the product specification for total variation (“Acceptance criteria”) and are shown in Table 25. The total variation lognormal $C V \%$ results are presented in Table 26.

Table 25. TREC and KREC precision analysis with between variation estimates across paper manufacturers and across paper lots   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Meancopies/105 cells</td><td rowspan=1 colspan=1>Mean Lncopies/105cells</td><td rowspan=1 colspan=1>WithinPaper LotlnSD</td><td rowspan=1 colspan=1>BetweenPaperLot lnSD</td><td rowspan=1 colspan=1>BetweenManufacturerlnSD</td><td rowspan=1 colspan=1>TotallnSD</td><td rowspan=1 colspan=1>Specifi-cation</td></tr><tr><td rowspan=5 colspan=1>TREC</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>584</td><td rowspan=1 colspan=1>6.37</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>871</td><td rowspan=1 colspan=1>6.77</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1300</td><td rowspan=1 colspan=1>7.17</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4105</td><td rowspan=1 colspan=1>8.32</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>14328</td><td rowspan=1 colspan=1>9.57</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>000</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.90</td></tr><tr><td rowspan=6 colspan=1>KREC</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1064</td><td rowspan=1 colspan=1>6.97</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>000</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.49</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4403</td><td rowspan=1 colspan=1>8.39</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>1.49</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4817</td><td rowspan=1 colspan=1>8.48</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>1.49</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5486</td><td rowspan=1 colspan=1>8.61</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>1.49</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>11968</td><td rowspan=1 colspan=1>9.39</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>1.49</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>22026</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>000</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.49</td></tr></table>

Table 26. TREC and KREC precision analysis with between variation estimates across paper manufacturers   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Meancopies/105cells</td><td colspan="1" rowspan="1">WithinPaper LotLognormaCV%</td><td colspan="1" rowspan="1">BetweenPaperLognormalCV%</td><td colspan="1" rowspan="1">BetweenManufacturerLognormalCV%</td><td colspan="1" rowspan="1">TotalLognormalCV%</td></tr><tr><td colspan="1" rowspan="3">TREC</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">147</td><td colspan="1" rowspan="1">584</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">149</td><td colspan="1" rowspan="1">871</td><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">53</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">1300</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">4105</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">14328</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="6">KREC</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">1064</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">4403</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">4817</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">5486</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">22</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">11968</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">147</td><td colspan="1" rowspan="1">22026</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">28</td></tr></table>

ANOVA results in Table 27 below demonstrate that there were statistically significant differences between paper manufacturers (Sample #2/KREC) and between paper lots (Sample #11/TREC and KREC); however, the highest between lot variation was 0.06 (Sample #11/TREC and KREC), and the highest between manufacturer variation was 0.09 (Sample #2/KREC). Per study acceptance criteria, the variation between these samples would not be clinically significant. The data demonstrated the similarity in performance between two major brands of filter paper.

Table 27. TREC and KREC between-paper lot variation and between-paper manufacturer variation, and the corresponding p-values from ANOVA   

<table><tr><td rowspan=1 colspan=1>Analyte Sample</td><td rowspan=1 colspan=1>Analyte Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCopies(copies/105cells)</td><td rowspan=1 colspan=1>MeanLnCopies</td><td rowspan=1 colspan=1>BetweenPaperLot SD</td><td rowspan=1 colspan=1>P-value fortestingleveldifferencesbetweenpaper lots</td><td rowspan=1 colspan=1>BetweenManufacturerSD</td><td rowspan=1 colspan=1>P-value fortesting leveldifferencesbetweenmanufacturers</td></tr><tr><td rowspan=5 colspan=1>TREC</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>584</td><td rowspan=1 colspan=1>6.37</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.781</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.759</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>871</td><td rowspan=1 colspan=1>6.77</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.272</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.133</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1300</td><td rowspan=1 colspan=1>7.17</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.226</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.725</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4105</td><td rowspan=1 colspan=1>8.32</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.011</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.091</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>14328</td><td rowspan=1 colspan=1>9.57</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.776</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.291</td></tr><tr><td rowspan=6 colspan=1>KREC</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>1064</td><td rowspan=1 colspan=1>6.97</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.206</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.652</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4403</td><td rowspan=1 colspan=1>8.39</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.736</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.365</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>4817</td><td rowspan=1 colspan=1>8.48</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.063</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.622</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>5486</td><td rowspan=1 colspan=1>8.61</td><td rowspan=1 colspan=1>0.01</td><td rowspan=1 colspan=1>0.262</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.143</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>11968</td><td rowspan=1 colspan=1>9.39</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.009</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.291</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>22026</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.726</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.017</td></tr></table>

# C Clinical Studies:

(i) Specimens

A clinical study was conducted at one foreign testing site, the Statens Serum Institute (SSI) in Denmark. SSI is the same site that conducted the study for the PerkinElmer EnLite Neonatal TREC Assay (DEN140010). The majority of the DBS specimens tested for the clinical study (retrospective, anonymized residual [leftover] newborn specimens) were procured by SSI from the screened Danish population. Specimen collection was prospective-retrospective and enriched with confirmed positive specimens for SCID and XLA which came from two biobanks: Danish Neonatal Screening Biobank and the California Biobank. Additionally, 3 of the 4 confirmed positive samples for XLA were procured from the SSI. Confirmed positive specimens were also procured from the California Department of Public Health Genetic Disease Screening Program Biobank (CA), including SCID (all 17 specimens) and XLA (1 of 4 confirmed specimens). All samples were tested at SSI as part of the evaluation. Age matched control specimens were used and presumed to be normal and were archived just before and after the time period during which the confirmed positive specimens for each of the two disorders (SCID, XLA) were archived. These specimens were used as controls for storage conditions to ensure specimen integrity remained intact within the timeframe during which they were stored. Therefore, in addition to consecutive specimens, archived specimens with storage time matched to the storage time of positive specimens were included. Specimen inclusion/exclusion criteria were provided and included specimen age, and storage conditions.

# (ii) Inclusion/Exclusion criteria for assessment of specimens

The age of the newborns at the time of specimen collection was less than or equal to 7 days to mimic the expected intended use population and a typical sample collection age for first tier screening to enable early diagnosis. Routine newborn screening specimens originating from the study site were collected at least 5.5 years ago to be able to assess, based on medical records, that the subject had not been identified to present clinical signs attributable to SCID or XLA. The confirmed positive SCID or XLA specimens had been collected $\leq 1 7$ years before the start date of the study to avoid inclusion of sample stored for longer than that time period. Confirmed positive XLA samples were difficult to procure; therefore, samples violating the upper limit of the storage time or the limit for the age of the newborn at the time of sample collection were included.

Inclusion criteria included the following:

Age of the newborn at the time of specimen collection $\leq 7$ days Specimen storage conditions are known Specimen collected at least 5.5 years ago (from study start)

For specimens confirmed positive for SCID:

Specimen confirmed positive for typical SCID (documentation of confirmed diagnosis and/or documentation of flow cytometry result) Date of specimen collection is $\leq 1 7$ years from the start date of the study

For specimens confirmed positive for XLA:

Specimen confirmed positive for XLA (e.g., documentation of BTK gene mutation and/or flow cytometry result) Date of specimen collection is $\leq 1 7$ years from the start date of the study

In addition, for storage age-matched control specimens:

• Date of specimen collection is known Specimen collected just before or after the time period during which the confirmed positive specimens for SCID and/or XLA was collected

Exclusion criteria included the following:

The DBS is not of appropriate quality (i.e., DBS is scratched, abraded, show signs of incomplete saturation or supersaturation, dilution, discoloration, or contamination) Parents/guardians have opted out for use of the specimen beyond routine screening Repeat specimen from a newborn whose specimen is already included in the study Less than one disk with a diameter of $3 . 2 \mathrm { m m } \left( 1 / 8 \mathrm { i n c h } \right)$ of DBS available in the cutoff phase, and less than two disks available in the pivotal phase for routine specimens

In addition, for specimens confirmed positive for SCID:

Other than typical SCID, such as less acute SCID (e.g., leaky SCID, variant SCID) or SCID like syndromes (e.g., DiGeorge syndrome or Omenn syndrome) In addition, for specimens confirmed positive for XLA: • Atypical XLA or female carriers

(iii) Study Design:

A total of 6924 unique specimens were tested in the study: 3432 specimens were tested in the cut-off phase and 3484 specimens were tested in the pivotal phase (this number includes 17 SCID cases, 6 XLA cases and 52 SMA cases; SMA cases are deducted from the table – refer to DEN200044). The main study consisted of two phases. Specimens in the cut-off study were not retested in the pivotal study.

Cut-off Phase: Newborn specimens representative of samples of the target population at the study site were tested with the device to produce newborn population distribution data for TREC and KREC and to select clinical cut-off values for the pivotal study. The number of false positives for TREC and KREC were determined from the data using lower percentiles $0 . 3 \%$ , 0.5, and $1 \%$ and are shown in Table 28.

Table 28. Descriptive statistics for the copies (per $\mathbf { 1 0 5 }$ cells) for KREC and TREC and the percentile cut-offs used in the study   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=6>Copies / 105cells</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>0.3percentile</td><td rowspan=1 colspan=1>0.5percentile</td><td rowspan=1 colspan=1>1.0percentile</td></tr><tr><td rowspan=1 colspan=1>TREC</td><td rowspan=1 colspan=1>2520</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>9990</td><td rowspan=1 colspan=1>262</td><td rowspan=1 colspan=1>413</td><td rowspan=1 colspan=1>563</td></tr><tr><td rowspan=1 colspan=1>KREC</td><td rowspan=1 colspan=1>3590</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>20100</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>261</td><td rowspan=1 colspan=1>484</td></tr></table>

Pivotal Phase: Presumed unaffected newborn specimens representing the target population at the study site were supplemented with confirmed positive specimens collected from SCID and 6 XLA affected newborns. A total of 3432 unique samples as shown in Table 29 (after subtracting 52 SMA cases – refer to DEN200044), were assessed in the pivotal phase. After exclusion of the samples not fulfilling the inclusion/exclusion criteria, a total of 3383 samples were included in the final analysis. Demographic characteristics of the routine screening specimens and confirmed positive samples were collected. These included: age at the time of sample collection (mean 2.5 days); weight distribution of the newborns at the time of specimen collection $( 9 5 \% \geq 2 5 0 0 \mathrm { g } )$ ; gestational age $9 8 \% \geq 3 4$ weeks); and gender $( 5 0 . 6 \%$ male and $4 9 . 4 \%$ female).

Table 29. Summary of Pivotal Phase Specimens Assayed and Included in the Data Analysis   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>AssayedSamples</td><td rowspan=1 colspan=1>Excluded due tofailure to meetinclusion/exclusioncriteria</td><td rowspan=1 colspan=1>Excluded dueto InvalidResult</td><td rowspan=1 colspan=1>Samplesincluded infinal analysis</td></tr><tr><td rowspan=1 colspan=1>Routinescreeningspecimens</td><td rowspan=1 colspan=1>3044</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3018</td></tr><tr><td rowspan=1 colspan=1>SCID positives</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>XLA positives</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Storage timematchedcontrols</td><td rowspan=1 colspan=1>365</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>342</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3432</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3383</td></tr></table>

Confirmatory test results were used as the comparator for the confirmed positive SCID or XLA cases including the presence of BTK mutation for XLA. Determination of the true clinical status with regard to SCID has been previously described (DEN140010). The true clinical status with regard to XLA was assessed using data available in patient registries. The clinical assessment of the study subjects was obtained from their medical records to confirm that the subject, at 5.5 years of age (defined as 66 months) or younger, has not been identified with XLA (has no clinical signs attributable to XLA) and is apparently healthy. Danish hospital registry data were reviewed by filtering out subjects that had at least one of the 275 ICD10 diagnosis codes that were listed as potentially linked to XLA. By the age of 5.5 years, a subject with a XLA would have either:

• Been diagnosed with the XLA; or • Presented with serious medical complications suggestive of XLA as listed below:

Table 30. Symptoms and signs associated with XLA   

<table><tr><td>Age of onset</td><td>Symptoms, signs</td></tr><tr><td>0-2 years</td><td>A severe life-threatening bacterial infection such as sepsis, meningitis, cellulitis, or empyema</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Paucity of lymphoid tissue (small adenoids,tonsils,and lymph nodes on physical examination)</td></tr><tr><td colspan="1" rowspan="1">&lt; 5 years</td><td colspan="1" rowspan="1">Recurrent otitis, pneumonitis, sinusitis, andconjunctivitis</td></tr></table>

To establish the screening performance, the cut-offs for TREC and KREC were determined by calculating the TREC and KREC concentrations corresponding to the 0.3th and 1.0th population percentiles (262 copies/ $1 0 ^ { 5 }$ cells for TREC, 484 copies/ $1 0 ^ { 5 }$ cells for KREC) established in the cut-off study with an independent dataset and were applied in the pivotal study

Testing was in singlicate unless repeat testing was necessary due to an invalid result. The specimens having TREC and KREC levels below the cut-off values (i.e., potential positives) in the initial round of testing were re-tested in duplicate. Any invalids were also retested in singlicate per protocol. If either replicate was positive, the sample was considered positive. All 3 replicates were required to be negative for the sample to be negative. The final results (presumptive normal, presumptive positive, invalid result) were classified after the second round of testing according to the testing interpretation algorithm.

The Eonis SCID-SMA kits’ performance with each subjects’ sample was compared to the clinical data/outcome captured for that subject. Status of SCID patients were determined previously (DEN140010). A total of 18 subjects had screen positive results with initial KREC results below the cut-off (suggestive for XLA). After replicate testing, 9 subjects with repeat test KREC results below cut-off (one or both replicates below cut-off) remained. For all 18 subjects, the status was alive at 5.5 years after birth, and 6 subjects had at least one ICD10 code listed in the registry. Of these 6 subjects, none had multiple diagnosis that were indicative of XLA, nor did they have diagnosis that mandated further investigation. Thus, all were concluded to be unaffected for XLA (Table 31).

Table 31. Summary of the routine screening samples tested in the pivotal study   

<table><tr><td rowspan=1 colspan=1>Routine screening samples</td><td rowspan=1 colspan=1>TREC</td><td rowspan=1 colspan=1>KREC</td></tr><tr><td rowspan=1 colspan=1>Screened samples</td><td rowspan=1 colspan=1>3018</td><td rowspan=1 colspan=1>3018</td></tr><tr><td rowspan=1 colspan=1>Initialscreenpositive</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Initialscreennegative</td><td rowspan=1 colspan=1>3008</td><td rowspan=1 colspan=1>3000</td></tr><tr><td rowspan=1 colspan=1>Retest rate</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.6%</td></tr><tr><td rowspan=1 colspan=1>Finalscreen positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Final screen negative</td><td rowspan=1 colspan=1>3008</td><td rowspan=1 colspan=1>3004</td></tr><tr><td rowspan=1 colspan=1>False-positive rate</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.3%</td></tr></table>

The retest rate for the routine screening population was $0 . 9 \%$ with the study cut-offs for TREC and KREC. The results of the screening performance after retesting per protocol for SCID (TREC) below in Tables 32, along with the false positive rate and $9 5 \%$ confidence intervals (CI) in Table 33.

Table 32. Screening performance of TREC with the Eonis SCID-SMA kit   

<table><tr><td rowspan=2 colspan=2>TREC</td><td rowspan=1 colspan=2>Clinical Status</td><td rowspan=2 colspan=1>Total (%)</td></tr><tr><td rowspan=1 colspan=1>SCIDaffected (%)</td><td rowspan=1 colspan=1>Normal (%)</td></tr><tr><td rowspan=3 colspan=1>Screeningresult</td><td rowspan=1 colspan=1>Presumptive positive(%)</td><td rowspan=1 colspan=1>17 (100 %)</td><td rowspan=1 colspan=1>9 (0.3 %)</td><td rowspan=1 colspan=1>26 (0.9 %)</td></tr><tr><td rowspan=1 colspan=1>Presumptive normal(%)</td><td rowspan=1 colspan=1>0 (0.0 %)</td><td rowspan=1 colspan=1>3008 (99.7 %)</td><td rowspan=1 colspan=1>3008 (99.1%)</td></tr><tr><td rowspan=1 colspan=1>Total (%)</td><td rowspan=1 colspan=1>17 (100 %)</td><td rowspan=1 colspan=1>3017 (100 %)</td><td rowspan=1 colspan=1>3034 (100 %)</td></tr></table>

Table 33. SCID false positive rate observed in the pivotal trial after retest   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive %-agreement</td><td rowspan=1 colspan=1>Negative %-agreement</td><td rowspan=1 colspan=1>Overall %-agreement</td><td rowspan=1 colspan=1>Falsepositive rate</td><td rowspan=1 colspan=1>Falsenegativerate</td><td rowspan=1 colspan=1>Positivepredictivevalue</td><td rowspan=1 colspan=1>Negativepredictivevalue</td></tr><tr><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>99.7 %</td><td rowspan=1 colspan=1>99.7 %</td><td rowspan=1 colspan=1>0.3 %</td><td rowspan=1 colspan=1>0.0 %</td><td rowspan=1 colspan=1>0.6 %</td><td rowspan=1 colspan=1>100 %</td></tr><tr><td rowspan=1 colspan=1>Confidenceinterval</td><td rowspan=1 colspan=1>80.5 % - 100%</td><td rowspan=1 colspan=1>99.4 % -99.9 %</td><td rowspan=1 colspan=1>99.4 % -99.9 %</td><td rowspan=1 colspan=1>0.1 % - 0.6 %</td><td rowspan=1 colspan=1>0 % - 19.5%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

The results of the screening performance after retesting per protocol for XLA (KREC) is shown in Tables 34 and 35, along with the false positive rate after retest and $9 5 \%$ confidence intervals (CI) in Table 36.

Table 34. Screening performance of the Eonis SCID-SMA device for XLA after retesting. The screening performance is presented with $9 5 \%$ confidence intervals.   

<table><tr><td rowspan=2 colspan=2>KREC</td><td rowspan=1 colspan=2>Clinical Status</td><td rowspan=2 colspan=1>Total (%)</td></tr><tr><td rowspan=1 colspan=1>XLA affected(%)</td><td rowspan=1 colspan=1>Normal (%)</td></tr><tr><td rowspan=3 colspan=1>Screeningresult</td><td rowspan=1 colspan=1>Presumptive positive (%)</td><td rowspan=1 colspan=1>6 (100 %)</td><td rowspan=1 colspan=1>9* (0.3 %)</td><td rowspan=1 colspan=1>15 (0.5 %)</td></tr><tr><td rowspan=1 colspan=1>Presumptive normal (%)</td><td rowspan=1 colspan=1>0 (0.0 %)</td><td rowspan=1 colspan=1>3004 (99.7 %)</td><td rowspan=1 colspan=1>3004 (99.5 %)</td></tr><tr><td rowspan=1 colspan=1>Total (%)</td><td rowspan=1 colspan=1>6 (100 %)</td><td rowspan=1 colspan=1>3013 (100 %)</td><td rowspan=1 colspan=1>3019 (100 %)</td></tr></table>

\*Includes one false positive result from a female subject. In female population the false positive rate is estimated to be $0 . 1 \%$ .

Because XLA is a disease that affects only males, the results for KREC are shown for males in Table 35.

Table 35. Screening performance of the Eonis SCID-SMA device for XLA after retesting in males alone   

<table><tr><td rowspan=2 colspan=2>KREC(Male only)</td><td rowspan=1 colspan=2>Clinical Status</td><td rowspan=2 colspan=1>Total (%)</td></tr><tr><td rowspan=1 colspan=1>XLA affected(%)</td><td rowspan=1 colspan=1>Normal (%)</td></tr><tr><td rowspan=3 colspan=1>Screeningresult</td><td rowspan=1 colspan=1>Presumptive positive(%)</td><td rowspan=1 colspan=1>6 (100%)</td><td rowspan=1 colspan=1>8 (0.5%)</td><td rowspan=1 colspan=1>14 (0.9%)</td></tr><tr><td rowspan=1 colspan=1>Presumptive normal (%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1515 (99.5%)</td><td rowspan=1 colspan=1>1515 (99.1%)</td></tr><tr><td rowspan=1 colspan=1>Total (%)</td><td rowspan=1 colspan=1>6 (100%)</td><td rowspan=1 colspan=1>1523 (100%)</td><td rowspan=1 colspan=1>1529 (100%)</td></tr></table>

Table 36. XLA False positive rate observed in the trial after retest   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive %-agreement</td><td rowspan=1 colspan=1>Negative %-agreement</td><td rowspan=1 colspan=1>Overall %-agreement</td><td rowspan=1 colspan=1>Falsepositiverate</td><td rowspan=1 colspan=1>Falsenegativerate</td></tr><tr><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>100 %</td><td rowspan=1 colspan=1>99.4 %</td><td rowspan=1 colspan=1>99.4 %</td><td rowspan=1 colspan=1>0.6 %</td><td rowspan=1 colspan=1>0.0 %</td></tr><tr><td rowspan=1 colspan=1>Confidenceinterval</td><td rowspan=1 colspan=1>39.8 % - 100 %</td><td rowspan=1 colspan=1>99.1 % - 99.6 %</td><td rowspan=1 colspan=1>99.1 % - 99.6 %</td><td rowspan=1 colspan=1>0.4 % - 0.9 %</td><td rowspan=1 colspan=1>0 % - 60.2 %</td></tr></table>

# D Clinical Cut-Off:

TREC: 262 copies $^ { \prime } 1 0 ^ { 5 }$ cells KREC: 484 copies/ $^ { \prime } 1 0 ^ { 5 }$ cells

# E Expected Values/Reference Range:

The incidence of SCID is approximately 1 in 50,000 live births. 1 The incidence of XLA is approximately 1 in 190,000 live male births.2 The distribution of a newborn population was determined by analyzing 3341 newborn DBS from the Danish Newborn Screening Biobank (NBS-Biobank) with the Eonis SCID-SMA kit. The specimens were archived routine screening specimens. The TREC and KREC descriptive statistics (median and range of results observed) and the value at which 0.3, 0.5, or 1.0 percent of the specimens were below of the specimens are displayed in Table 28. Data separated by gender for KREC is shown in Table 37.

Table 37. Descriptive statistics and percentiles for KREC copies/105 cells by gender   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Gender</td><td rowspan=2 colspan=1>N</td><td rowspan=1 colspan=6>Copies/105 cells</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>0.3percentile</td><td rowspan=1 colspan=1>0.5percentile</td><td rowspan=1 colspan=1>1.0percentile</td></tr><tr><td rowspan=2 colspan=1>KREC</td><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>1699</td><td rowspan=1 colspan=1>3460</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>16200</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>433</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>1642</td><td rowspan=1 colspan=1>3740</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>20100</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>576</td></tr></table>

The frequency distributions $\scriptstyle ( \mathrm { N } = 3 3 4 1$ ) of TREC and KREC analyte are shown in Figures 6 and 7.

![](images/8298f3c7e310ea5c568f32568e938611e95377452962d70290c232ffecb2d7e9.jpg)  
Figure 6. TREC (copies $/ 1 0 ^ { 5 }$ cells) distribution of presumed normal newborn samples

![](images/a07c7f0f1122e11c02888943d8d5dc388881e841eb89568799a5f84f73f661a8.jpg)  
Figure 7. KREC (copies $/ 1 0 ^ { 5 }$ cells) distribution of presumed normal newborn samples.

# F Other Supportive Instrument Performance Characteristics Data:

DNA Extraction Equivalency

This study was conducted to demonstrate that Eonis test system has equivalent performance when the DNA extraction is performed either manually or by a different automated liquid handler and is designed based on the recommendation of CLSI Guideline EP05-A.

Two operators performed 3 Eonis extractions and PCR consolidations/day for 5 days using 1 kit lot, totaling 30 runs using 7 samples from the precision study (Table 5). Each day, an operator would run 1 extraction/PCR consolidation on the same JANUS Extraction/JANUS PCR Mastermix, 1 extraction/PCR consolidation on a second commercial liquid handler, and 1 manual extraction manual PCR setup/consolidation. The study generated a total of 1050 results (5 replicates x 7 samples x 3 extraction/PCR methods x 2 operators x 5 days) with 150 results per sample and 350 results per extraction/PCR method.

There were 45 replicates performed with the JANUS handler, 50 replicates for the second commercial liquid handler, and 50 replicates for the manual extraction process. The results for TREC and KREC, evaluated using variance analysis method, are presented in Table 38 and Table 39.

Table 38. DNA extraction equivalency for TREC   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>MeanLnCopies</td><td rowspan=1 colspan=1>RepeatabilityLn SD</td><td rowspan=1 colspan=1>BetweenExtractionMethod Ln SD</td><td rowspan=1 colspan=1>Total Ln SD</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>3.57</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.95</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>781</td><td rowspan=1 colspan=1>6.66</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.39</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>6.90</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>1188</td><td rowspan=1 colspan=1>7.08</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>1920</td><td rowspan=1 colspan=1>7.56</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.34</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>4964</td><td rowspan=1 colspan=1>8.51</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.27</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>13494</td><td rowspan=1 colspan=1>9.51</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.30</td></tr></table>

Table 39. DNA extraction equivalency for KREC   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean(Copies/105 cells)</td><td rowspan=1 colspan=1>MeanLnCopies</td><td rowspan=1 colspan=1>RepeatabilityLn SD</td><td rowspan=1 colspan=1>BetweenExtractionMethod LnSD</td><td rowspan=1 colspan=1>Total Ln SD</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>1097</td><td rowspan=1 colspan=1>7.00</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.42</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>3429</td><td rowspan=1 colspan=1>8.14</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>5768</td><td rowspan=1 colspan=1>8.66</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.29</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>6438</td><td rowspan=1 colspan=1>8.77</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.24</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>14618</td><td rowspan=1 colspan=1>9.59</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>17327</td><td rowspan=1 colspan=1>9.76</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.29</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>54176</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.34</td></tr></table>

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.