Agendia Inc.   
Marcelo Trevino   
Global VP, Regulatory Affairs & Quality Assurance 22 Morgan   
Irvine, CA 92618

Re: K210973 Trade/Device Name: MammaPrint FFPE NGS kit Regulation Number: 21 CFR 866.6040 Regulation Name: Gene Expression Profiling Test System For Breast Cancer Prognosis Regulatory Class: Class II Product Code: NYI Dated: March 31, 2021 Received: March 31, 2021

Dear Marcelo Trevino:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. Although this letter refers to your product as a device, please be aware that some cleared products may instead be combination products. The 510(k) Premarket Notification Database located at https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm identifies combination product submissions. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration. Please note: CDRH does not evaluate information related to contract liability warranties. We remind you, however, that device labeling must be truthful and not misleading.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801 and Part 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR

803) for devices or postmarketing safety reporting (21 CFR 4, Subpart B) for combination products (see https://www.fda.gov/combination-products/guidance-regulatory-information/postmarketing-safety-reportingcombination-products); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820) for devices or current good manufacturing practices (21 CFR 4, Subpart A) for combination products; and, if applicable, the electronic product radiation control provisions (Sections 531- 542 of the Act); 21 CFR 1000-1050.

Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to https://www.fda.gov/medical-devices/medical-device-safety/medical-device-reportingmdr-how-report-medical-device-problems.

For comprehensive regulatory information about medical devices and radiation-emitting products, including information about labeling regulations, please see Device Advice (https://www.fda.gov/medical  
devices/device-advice-comprehensive-regulatory-assistance) and CDRH Learn   
(https://www.fda.gov/training-and-continuing-education/cdrh-learn). Additionally, you may contact the Division of Industry and Consumer Education (DICE) to ask a question about a specific regulatory topic. See the DICE website (https://www.fda.gov/medical-devices/device-advice-comprehensive-regulatoryassistance/contact-us-division-industry-and-consumer-education-dice) for more information or contact DICE by email (DICE@fda.hhs.gov) or phone (1-800-638-2041 or 301-796-7100).

Sincerely,

Soma Ghosh, Ph.D.   
Chief   
Molecular Pathology and Cytology Branch Division of Molecular Genetics   
and Pathology   
OHT7: Office of In Vitro Diagnostics   
Office of Product Evaluation and Quality Center for Devices and Radiological Health

Enclosure

510(k) Number (if known) K210973

Indications for Use (Describe)

TeMG  liv l enrichment Next Generation Sequencing (NGS) technology for gene expression profiling of the70-gene MammaPrint rih assess a patient's risk to develop distant metastasis within 5 years and up to 10 years after diagnosis.

The M P G    r     a $\leq$ $5 . 0 \mathrm { c m }$ and ymhoegativThe resul dicte o us y hysicn   prooarkey, l other clinicopathological factors.

This section applies only to requirements of the Paperwork Reduction Act of 1995.

# \*DO NOT SEND YOUR COMPLETED FORM TO THE PRA STAFF EMAIL ADDRESS BELOW.\*

Turenotioatinatveo time to reviw instructions, search existig data sources,gather and maintainhe dataneeded and cplee and review the colection of information. Send comments regarding this burden estimate or anyother aspect of this information collection, including suggestions for reducing this burden, to:

Department of Health and Human Services Food and Drug Administration Office of Chief Information Officer Paperwork Reduction Act (PRA) Staff PRAStaff@fda.hhs.gov

"An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB number."

# 510 (k) Summary

# Submitter

Agendia Inc.   
22 Morgan   
Irvine, California 92618

# Contact Person

Email: marcelo.trevino@agendia.com

Telephone: $+ 1$ 310 409 7828

Contact person: Marcelo Trevino, Global VP Regulatory & Quality Assurance

# Date prepared

March 31, 2021

# Trade Name

MammaPrint® FFPE NGS kit

# Common Name

classifier, prognostic, recurrence risk assessment, RNA gene expression, breast cancer

# Classification Name

Gene expression profiling test system for breast cancer prognosis

Primary Predicate Device MammaPrint FFPE (K201902)

# Reference Predicate Device

MammaPrint FFPE (K141142)

# Device Description

The MammaPrint FFPE NGS kit is a sequencing-based gene expression analysis of a tumor. The analysis is based on several processes: isolation of RNA from FFPE breast cancer tissue sections; library preparation of RNA resulting in cDNA adapter-ligated sequences; enrichment of the 70 genes (capture step); sequencing of the enriched library in the flow cell and data acquisition; MammaPrint Index calculation and determination of the risk classification in breast cancer patients.

Data analysis is performed according to the MammaPrint FFPE NGS algorithm (resulting in MammaPrint Index or MPI). This algorithm was designed and programmed by Agendia and incorporated into a proprietary software program, which loads the FASTQ data file. The software loads the FASTQ data file, performs quality control checks and determines the molecular profile of the sample by calculating the MammaPrint index by determining the correlation of the sample’s 70 gene expression profile to the mean expression profiles of the risk templates of tumors with a known good and poor outcome.

# Indications for Use

The MammaPrint FFPE NGS kit is a qualitative in vitro diagnostic test for use by clinical laboratories using target enrichment Next Generation Sequencing (NGS) technology for gene expression profiling of the 70-gene MammaPrint Breast Cancer signature on formalin-fixed, paraffin-embedded (FFPE) breast cancer tissue samples. The test is used to assess a patient’s risk to develop distant metastasis within 5 years and up to 10 years after diagnosis.

The MammaPrint FFPE NGS kit is performed for breast cancer patients with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The test result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

# Comparison to Predicate Device

The primary predicate is Agendia’s MammaPrint device (K201902). The predicate device, with product code NYI, is regulated under 21CFR 866.6040. The modified MammaPrint device subject of this 510(k) is substantially equivalent to Agendia’s MammaPrint device covered under K201902. The MammaPrint FFPE NGS kit is indicated for the same use and has similar technological characteristics to Agendia’s previously cleared MammaPrint device, with the principal exception of the use of an NGS platform.

The MammaPrint FFPE NGS kit is processed in a decentralized setting in contrast to centralized processing of the predicate device. All other elements of the test – the 70-gene signature, the RNA isolation from FFPE tissue samples, the gene expression profile ‘fingerprint’ obtained from the Agendia proprietary analysis software and scoring algorithm – remain unchanged. The subject device assesses gene expression based on capture of the 70 MammaPrint genes using the MammaPrint NGS bait library that includes probes for these genes.

The predicate device is included as a reference device to demonstrate the scientifically equivalent results of technical and clinical studies. This reference device demonstrates that test samples ran on the device cleared are equivalent to the subject device.

# Performance Data (Bench)

# Analytical performance

MammaPrint analytical (non-clinical) performance characteristics investigated comprise of concordance of the subject device to the predicate based on testing at Agendia’s central laboratory and external sites, evaluation of detection limit, and repeatability/reproducibility and precision performance assessments.

# 1. Method comparison to predicate device – Study-1

A concordance experiment was performed in Agendia’s central laboratory to ensure that the MammaPrint FFPE NGS kit is comparable to its predicate device. For this assessment, a set of 155 samples that complied with the intended use population of MammaPrint were used. These samples were previously processed on MammaPrint FFPE microarray as part of routine diagnostics and covered the entire MammaPrint readout range with sufficient amount of samples representing the categorical results. The MammaPrint results for the NGS and microarray platforms were compared.

Since the NGS kit has three output categories (High risk, Borderline, and Low risk) and the comparator method has two output categories (High risk and Low risk), comparison of MammaPrint categorical result was based on two concordance analysis for High vs. Low risk.

In the first concordance analysis, samples with MPI scores $< 0 . 0 5 8$ (this also includes Borderline samples with MPI scores of between 0.058 and -0.058) were treated as High risk, that is, positive results in the concordance analysis. Samples with MPI scores ${ > } 0 . 0 5 8$ were treated as Low risk or negative results. In the second concordance analysis, samples with MPI scores $< - 0 . 0 5 8$ were treated as High risk /or positive results in the concordance analysis. Samples with MPI scores ${ \tt > - 0 . 0 5 8 }$ (this also includes Borderline samples with MPI scores of between 0.058 and -0.058) were treated as Low Risk or negative results.

For the first analysis, comparison of the test results showed an Overall Percent Concordance (OPA) of $9 7 . 4 2 \%$ $9 5 \%$ CI: 93.55, 98.99) with a Negative Percent Agreement (NPA) of $9 3 . 8 5 \%$ $9 5 \%$ CI: 85.22, 97.58) and a Positive Percent Agreement (PPA) of $1 0 0 . 0 0 \%$ ( $9 5 \%$ CI: 95.91, 100.00).

For the second analysis, comparison of the test results showed an OPA of $9 8 . 7 1 \%$ $9 5 \%$ CI: 95.42, 99.65) with a NPA of $1 0 0 . 0 0 \%$ $9 5 \%$ CI: 94.42, 100.00) and a PPA of $9 7 . 7 8 \%$ $9 5 \%$ CI: 92.26, 99.39)

# 2. Method Comparison to predicate device – Study-2

A comparison study using 303 samples was performed to determine the concordance of MammaPrint FFPE NGS kit processed at four external sites versus the predicate microarray test at Agendia using previously collected patient samples. Comparison of MammaPrint test results from between NGS kit and microarray was performed by the concordance analysis as described above. The analysis showed the OPA ranging from $9 1 . 0 9 \%$ to $9 2 . 4 1 \%$ and PPA values ranging from $9 1 . 3 9 \%$ to $9 9 . 3 4 \%$ . NPA values were lower, ranging from $8 2 . 8 9 \%$ to $9 3 . 4 2 \%$ . Borderline samples and samples close to the classification threshold (High Risk/ Borderline) impacted the concordance.

# 3. Test performance

# Repeatability of RNA isolation

Impact of the variability between RNA isolations of the same FFPE tissue on the performance of the MammaPrint FFPE NGS kit was assessed between two different isolations from the FFPE tumor block. Each block was sectioned twice: a first set of four sections for the first isolation and a second set of four sections for the second isolation. Subsequently, the RNA from both isolations were processed with the MammaPrint FFPE NGS kit to obtain MammaPrint results. The overall agreement of MammaPrint categorical results between isolation 1 and isolation 2 was $100 \%$ with the agreement for the three categorical results, High Risk, Low Risk and Borderline each $100 \%$ as well $9 5 \% C 1$ : High Risk: 86.2-100.0, Low Risk: 81.6-100.0, Borderline: 34.2 –100.0).

# Reproducibility of MammaPrint FFPE NGS kit Controls

The reproducibility of MammaPrint FFPE NGS kit was assessed by processing two control samples with known MammaPrint result at the four external sites. RNA from these two samples with known MammaPrint results were provided by Agendia to these sites, one sample (CTRL-HR) with High Risk result and the second one (CTRL-LR) with Low Risk result. Per site, there were two operators who processed this set of 2 samples across 6 NGS runs until 12 measurements were obtained. Additionally, there were three different lot numbers of the MammaPrint FFPE NGS kit included in this set up, each operator used each of the different lot numbers twice during these 6 different days. For each control sample 50 measurements were obtained. The agreement of MammaPrint categorical results for these two samples was also assessed as shown in Table 1. Both samples reach a $100 \%$ agreement.

Table 1: Agreement MammaPrint categorical result for CTRL-HR and CTRL-LR   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category(High or Low)</td><td rowspan=1 colspan=1>MeanMPI</td><td rowspan=1 colspan=1>Total # ofreplicates(N)</td><td rowspan=1 colspan=1>Agree</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% 2-sided scoreCI</td></tr><tr><td rowspan=1 colspan=1>1 - CTRL-HR</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.512</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>2 - CTRL-LR</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.359</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.2% - 100%</td></tr></table>

# Precision/Reproducibility Assessment-Study-1

A site-to-site reproducibility of the MammaPrint FFPE NGS kit was performed at four external sites in the United States . For this assessment, RNA of 3 samples with known MammaPrint results covering High Risk, Low Risk and Borderline were provided by Agendia to the four external US sites that were included in this study. Each site had two operators that processed a set of 3 samples in duplicate across 6 NGS runs until there were 12 measurements obtained. Additionally, there were three different lot numbers of the MammaPrint FFPE NGS kit included in this set up. The data generated within this study was used to assess precision on sites, operators, runs, and total reproducibility. Table 2 below shows the variances for each of the three samples based on the ANOVA model output.

Table 2: Overview of variability (in form of Standard Deviation (SD)) calculated for all three samples   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>Repeatability(SD)</td><td rowspan=1 colspan=1>Between-Run(SD)</td><td rowspan=1 colspan=1>Between-Operator(SD)</td><td rowspan=1 colspan=1>Between-Site(SD)</td><td rowspan=1 colspan=1>Reproducibility(SD)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-1.153</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.050</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.067</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-0.0125</td><td rowspan=1 colspan=1>0.055</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>0.095</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>0.436</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.039</td><td rowspan=1 colspan=1>0.063</td></tr></table>

Additionally, the agreement in MammaPrint categorical result was determined for each of the three samples, see Table 3.

Table 3: Agreement MammaPrint categorical result the three samples included on the precision study   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category (High,Low, Borderline)</td><td rowspan=1 colspan=1>Mean MPI</td><td rowspan=1 colspan=1>Total # ofreplicates(N)</td><td rowspan=1 colspan=1>AgreementN</td><td rowspan=1 colspan=1>Agreement (%)</td><td rowspan=1 colspan=1>95% 2-sided score Cl</td></tr><tr><td rowspan=1 colspan=1>PREC-BD</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>-0.013</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>72.0, 88.5</td></tr><tr><td rowspan=1 colspan=1>PREC-HR</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-1.154</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9, 100</td></tr><tr><td rowspan=1 colspan=1>PREC-LR</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.436</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9, 100</td></tr></table>

# Precision/Reproducibility Assessment-Study-2

In addition 8 samples with a MPI close to the MammaPrint Threshold were repeatedly processed (from RNA isolation to sequencing runs at three different sites. The samples were separately processed by two operators per site and each operators processed the samples twice. For both operators combined, this resulted in a total of four runs per site. Each run was performed on a different day to incorporate day-to-day variations. Per run, each sample was analyzed in duplicate, resulting in 8 replicate measurements per sample and site.

Table 4 below shows the variances for each of the eight samples based on the ANOVA model output.

Table 4: Overview of variability (in form of SD) calculated for all eight samples   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>Repeatability(SD)</td><td rowspan=1 colspan=1>Between-Run(SD)</td><td rowspan=1 colspan=1>Between-Operator(SD)</td><td rowspan=1 colspan=1>Between-Site(SD)</td><td rowspan=1 colspan=1>Reproducibility(SD)</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>-0.177</td><td rowspan=1 colspan=1>0.025</td><td rowspan=1 colspan=1>0.006</td><td rowspan=1 colspan=1>0.014</td><td rowspan=1 colspan=1>0.006</td><td rowspan=1 colspan=1>0.031</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>0.020</td><td rowspan=1 colspan=1>0.034</td><td rowspan=1 colspan=1>0.008</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.042</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.155</td><td rowspan=1 colspan=1>0.044</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.015</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.046</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.223</td><td rowspan=1 colspan=1>0.037</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.009</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>0.044</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.060</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.015</td><td rowspan=1 colspan=1>0.021</td><td rowspan=1 colspan=1>0.031</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.142</td><td rowspan=1 colspan=1>0.060</td><td rowspan=1 colspan=1>0.013</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.023</td><td rowspan=1 colspan=1>0.066</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>0.018</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.024</td><td rowspan=1 colspan=1>0.017</td><td rowspan=1 colspan=1>0.035</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.165</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.028</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.036</td></tr></table>

The agreement in the categorical results for the 8 samples is shown in Table 5. Most of the samples show a $100 \%$ concordance/agreement across all replicates. There are two samples with a $83 \%$ concordance/agreement and a third sample with an agreement of $58 \%$ .

Table 5: Agreement MammaPrint categorical result for 8 additional precision study samples   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category (Highor Low)</td><td rowspan=1 colspan=1>MeanMPI</td><td rowspan=1 colspan=1>Total # ofreplicates(N)</td><td rowspan=1 colspan=1>AgreementN</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% 2-sided scoreCI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.177</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.9, 100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>62.6, 95.3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.155</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.223</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.060</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>36.6, 77.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.142</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.165</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr></table>

# Detection Limit

The acceptable range of RNA input for the MammaPrint FFPE NGS kit was determined by evaluating FFPE breast cancer samples with DV200 “poor” quality (less than $50 \%$ ) and DV200 “good” (greater or equal to $50 \%$ ) samples separately. Eight (8) FFPE breast cancer samples with DV200 “poor” quality were evaluated with the RNA of each sample diluted to lower the RNA input for the MammaPrint FFPE NGS kit from 200 ng to 100 ng, 50 ng, 25 ng, and 12.5 ng and were processed repeatedly during several separate runs using the MammaPrint FFPE NGS kit. Samples with at least 100 ng RNA input and with DV200 values ranging from $3 5 \%$ to $49 \%$ had valid rates ranging from $80 \%$ to $100 \%$ , whereas the RNA input below 100 ng and with DV200 “poor” quality ranging from $3 5 \%$ to $49 \%$ had valid rates ranging from $0 \%$ to $100 \%$ .

An RNA input (ng) below and above the RNA quality specific recommendations in the instructions for use was performed on DV200 “poor” (less than $50 \%$ ) and DV200 “good” (greater or equal to $50 \%$ ) samples separately. The limit chosen for the above and below recommended RNA input are 200 ng for DV200 “poor” samples and 100 ng for better quality samples. For RNA input of DV200 “poor” samples, 40 replicates were tested for 200 ng and 100 ng RNA input, while 14 replicates for 300 ng RNA concentration. In DV200 “Standard” samples, 18 replicates were tested for 200 ng to 50 ng dilutions and 15 replicates for 25 ng RNA input. Samples with at least 100 ng RNA input and with DV200 values ranging from $3 5 \%$ to $80 \%$ had valid rates ranging from $80 \%$ to $100 \%$ , while samples with 200 ng RNA input and with DV200 values ranging from $3 5 \%$ to $80 \%$ had valid rates of $83 \%$ to $100 \%$ . Samples with RNA input values below 100 ng and with DV200 “poor” quality ranging from $3 5 \%$ to $49 \%$ had valid rates ranging from $0 \%$ to $100 \%$ .

The study supports the specifications for $\mathsf { D V } 2 0 0 \geq 7 0 \%$ above 200 nt for 100 ng, and $\mathsf { D V } 2 0 0 \geq 2 0 \%$ above 200 nt for 200 ng. For 150 ng the data supports DV200 $\yen 50 \%$ because at 100 ng RNA input with DV200 ranging from $3 5 \%$ to $40 \%$ the valid rate was at least $80 \%$ as observed from the two studies.

# Performance testing – clinical

The microarRAy prognoSTics in breast cancER (RASTER) study was conducted to prospectively evaluate the risk of breast cancer distant metastases using a gene-expression prognosis classifier as a risk estimation tool, in addition to clinicopathological factors. In this multicenter observational study, the feasibility of MammaPrint® 70-gene signature developed to predict the risk of breast cancer metastases was assessed in 16 community hospitals in the Netherlands between 2004 and 2006. The primary objective of this multicenter observational study was to assess the feasibility of implementing the 70-gene signature and to study the clinical impact of the 70-gene signature test result on adjuvant systemic therapy (AST) decision making. The secondary objective of the RASTER study was to assess the outcome of patients for whom a gene expression classifier was used to determine the need for adjuvant systemic treatment. A total of 427 patients were enrolled in the RASTER study with age 18–61 years old and had a histologically confirmed unilateral, unifocal, primary operable, invasive adenocarcinoma of the breast (cT1–3N0M0). After a protocol amendment in 2004, the study allowed only patients aged 55 or younger could enroll. The study exclusion criteria were a history of a malignancy (with exception of basal-cell carcinoma or cervical dysplasia) and neoadjuvant systemic treatment. The study assessed the primary endpoint of distant-recurrence free interval (DRFI) defined as a distant breast cancer recurrence or death from breast cancer and the secondary endpoint of breast cancer specific survival (BCSS) defined as mortality related to breast cancer.

Five years follow-up data were published which included the analysis of estimated five-year distant-recurrence free interval (DRFI) and confirmed the prognostic value of the MammaPrint Index (MP) published in the January 2013 issue of International Journal of Cancer. Survival data was updated in 2017 to determine 10-years median follow up for the distant-recurrence-free-interval (DRFI) probabilities.

To demonstrate the clinical validity of the MammaPrint FFPE NGS kit, samples from the prospective observational RASTER study were evaluated. RASTER samples were previously processed between 2004 and 2007 on the MammaPrint Fresh microarray (K101454), for which at the time a 5-year clinical follow-up was available for the RASTER study for 427 subjects. A subset of FFPE tissue samples from the RASTER study $\scriptstyle ( n = 3 4 5$ ) that was previously processed on the MammaPrint FFPE microarray as part of the clearance of the reference predicate device (K141142) was used for the validation of MammaPrint FFPE NGS kit. From the set of 345 samples, total RNA was available for 341 samples and 25 samples failed QC. Therefore, a total of 316 samples were successfully processed using the MammaPrint FFPE NGS kit and this dataset was used for survival analysis.

Clinical validation analysis was performed on the 316 samples that had NGS MammaPrint results based on a 10-year median follow-up of the observational RASTER study. $89 . 6 \%$ of the patients have 5-year follow-up data and $53 \%$ of the patients having 10-year follow-up data (median follow-up 10.27 years). The data was analyzed based on the three (3) categories, High Risk, Low Risk, and Borderline, determined by the two cut-offs, -0.058 and $+ 0 . 0 5 8 $ . Comparison of clinical performance within the RASTER study between MammaPrint FFPE NGS kit and the predicate device MammaPrint FFPE microarray was evaluated by the Kaplan-Meier graphs and 5-year and 10-year survival percentages. Kaplan-Meier plots suggest significant difference in survival curves among different risk groups for MammaPrint FFPE NGS kit, $\mathsf { p } { = } 0 . 0 0 1$ .

Results from MammaPrint FFPE NGS kit and MammaPrint FFPE microarray were compared for the 316 FFPE samples with 5- and 10-year outcome data from the 427 RASTER patient samples. DRFI was the study endpoint as defined in the RASTER study. Specifically, DRFI measures the time until the diagnosis of distant metastasis or death from breast cancer. Kaplan-Meier curves showed a similar difference in DRFI between the Low and High Risk comparing MammaPrint FFPE NGS kit and MammaPrint FFPE microarray.

![](images/a8db09853e4a29f281a467c3b1fe1e8e876911fa2d435500ab9e55d77b1c18a9.jpg)  
MammaPrint FFPE NGS

![](images/c422287c160a6a1055efbdce33a870c8f59985e86ab4eb1c1b006f5ec0de1de8.jpg)  
MammaPrint FFPE microarray

Table 6: RASTER Study: 5-year and 10-year DRFI Probabilities based on MammaPrint FFPE NGS Risk Categories   

<table><tr><td rowspan=1 colspan=1>Risk</td><td rowspan=1 colspan=1>#Patients</td><td rowspan=1 colspan=1>Events at 5Years</td><td rowspan=1 colspan=1>5-year DRFI(95% CI)</td><td rowspan=1 colspan=1>Events at 10Years</td><td rowspan=1 colspan=1>10-year DRFI (95% CI)</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1% (96.0-100)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>95.2% (91.8-98.8)</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.6% (83.2-100)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.6% (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>88.2% (82.8-94.0)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>82.0% (75.5-89.2)</td></tr></table>

The Low Risk patients classified by MammaPrint FFPE NGS kit demonstrated a $1 . 9 \% ( 9 5 \% \mathsf { C l } ; 0 { - } 4 . 0 \% )$ and $4 . 8 \%$ $9 5 \%$ CI $: 1 . 2 \cdot 8 . 2 \%$ ) chance of cancer recurrence within 5 years and 10 years respectively (Table 6). Patients classified as High Risk by MammaPrint FFPE NGS kit, demonstrated a $1 1 . 8 \%$ ( $9 5 \%$ CI: 6.0-17.2%) and $1 8 \%$ $9 5 \%$ CI: 10.8-24.5%) chance of cancer recurrence within 5 years and 10 years respectively. The Borderline samples for both 5 and 10 years had a DRFI of $9 2 . 6 \%$ . The results for the Borderline samples may have low reliability due to the small sample size.

The predicate, MammaPrint FFPE microarray showed $9 8 . 7 \%$ 5-year DRFI for Low Risk and $8 8 . 0 \%$ for High Risk. With the 10-year follow-up, DRFI estimate was $9 6 . 0 \%$ for Low Risk and $8 2 . 8 \%$ for High Risk. Thus, the results indicated that both devices show similar clinical performance based on the updated follow-up from the RASTER study.

The prognostic performance of MammaPrint FFPE NGS kit was further investigated using Cox proportional hazard regression analysis based on all follow-up data, 5-year follow-up data, and 10-year follow-up data respectively. The evaluated clinicopathological factors include adjuvant treatment status (i.e., whether a patient received or did not receive adjuvant treatment such as chemotherapy (CT) and/or endocrine therapy (ET)), age ( $\mathord { > } 5 0$ vs ≤50), tumor size, histological grade, estrogen receptor (ER) status (positive/negative), and human epidermal growth factor receptor 2 (HER2) status (positive/negative).

In both univariate and multivariate analysis, MammaPrint FFPE NGS High/Low Risk result is significantly associated with cancer recurrence based on all three follow up data (i.e., all follow-up, 5-year follow-up, and 10-year followup). For the multivariate analysis, MammaPrint FFPE NGS High/Low Risk result is significantly associated with cancer recurrence for the three follow up data with estimated hazard ratios of 7.31 $9 5 \%$ CI: 2.50-21.41), 12.55 $9 5 \%$ CI: 2.26-69.74), and 7.08 $9 5 \%$ CI: 2.23-22.49) respectively after adjusting for all other clinicopathological factors. No significant difference (p value $< 0 . 0 5$ ) was seen for Borderline group vs Low Risk group, which may be due to the small sample size for the Borderline group and the reliability of the results for this group may be low. The result support the prognostic significance for MammaPrint FFPE NGS kit beyond that of other clinicopathological factors.

# Conclusion

The MammaPrint FFPE NGS kit meets all product design requirements and applicable standards and embodies technological characteristics similar to the predicate device. The device has been shown to be substantially equivalent to the predicate device and is safe and effective.