# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K210973   
B Applicant Agendia, Inc.   
C Proprietary and Established Names MammaPrint® FFPE NGS kit   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>NYI</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.6040 - Gene Expression ProfilingTest System for Breast Cancer Prognosis</td><td rowspan=1 colspan=1>Immunology (82)</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New Device   
B Measurand: 70 gene expression profile   
C Type of Test: Next-Generation Tumor Profiling Test

III Intended Use:

A Indication(s) for Use:

The MammaPrint FFPE NGS kit is a qualitative in vitro diagnostic test for use by clinical laboratories using target enrichment Next Generation Sequencing (NGS) technology for gene expression profiling of the 70-gene MammaPrint Breast Cancer signature on formalin-fixed, paraffin-embedded (FFPE) breast cancer tissue samples. The test is used to assess a patient’s risk to develop distant metastasis within 5 years and up to 10 years after diagnosis.

The MammaPrint FFPE NGS kit is performed for breast cancer patients with Stage I or Stage II disease, with tumor size $\le 5 . 0$ cm and lymph node negative. The test result is indicated for use by physicians as a prognostic marker only, along with other clinicopathological factors.

# B Special Conditions for Use Statement(s):

Rx - For Prescription Use Only

For in vitro diagnostic use

MammaPrint® FFPE NGS kit is not indicated as a standalone test to determine the outcome of disease, nor to suggest or infer an individual patient’s likely response to therapy. Results should be taken in the context of other relevant clinico-pathological factors and standard practice of medicine.

# C Special Instrument Requirements:

Illumina MiSeqDx (qualified by Agendia)

# IV Device/System Characteristics:

# A Device Description:

# 1. Reagents

MammaPrint FFPE NGS kit is for use as part of a test system with the MiSeqDx and sequencing reagents. The MammaPrint FFPE NGS kit consists of 4 boxes with the components and the storage conditions are listed in Table 1. The MammaPrint FFPE NGS kit contains reagents for 16 reactions and sample indices to run up to 32 reactions (patient specimens and controls). A detailed list of required instruments, software, reagents, consumables, and storage conditions is described in the product labeling (MammaPrint FFPE NGS kit Package Insert).

Table 1. Reagents Provided   

<table><tr><td rowspan=1 colspan=3>MammaPrint NGS RNA Library Prep(Pre-PCR) Box 1 of 4]</td><td rowspan=1 colspan=1>-20°C</td></tr><tr><td rowspan=1 colspan=3>Component</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>304 μL</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1>140 μL400 μL80 μL320 μL80 μL80 μL60 μL800 μL</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Agendia NGS Uracil DNA Glycosylase (UDG)Agendia NGS Reverse PCR PrimerAgendia NGS Nuclease-Free Water</td><td rowspan=1 colspan=1>16 μL16 μL2.4 mL</td></tr><tr><td rowspan=1 colspan=1>MammaPrint NGS Target Enrichment (Post-PCR Box 1)[Box 2 of 4]</td><td rowspan=1 colspan=1>Room Temperature</td></tr><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>Agendia NGS Hyb 1Agendia NGS Hyb 2Agendia NGS Hyb 4Agendia NGS Binding BufferAgendia NGS Wash Buffer 1Agendia NGS Wash Buffer 2Agendia NGS Nuclease-Free WaterAgendia NGS Elution BufferAgendia NGS Neutralization BufferMammaPrint Package Insert</td><td rowspan=1 colspan=1>400 μL1.25 mL1.25 mL13.2 mL8 mL24 mL2.4 mL5.8 mL960 μL1x</td></tr><tr><td rowspan=1 colspan=1>MammaPrint NGS Target Enrichment (Post-PCR Box 2)[Box 3 of 4]</td><td rowspan=1 colspan=1>-20°C</td></tr><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>Agendia NGS Indexing Block 1Agendia NGS Block 2Agendia NGS Indexing Block 3Agendia NGS RNase BlockAgendia NGS Hyb 3Agendia NGS Post-Capture PCR PrimerAgendia NGS PCR Master MixAgendia NGS 8bp Index Plate</td><td rowspan=1 colspan=1>45 μL45 μL12 μL18 μL160 μL16 μL800 μL12 μL</td></tr><tr><td rowspan=1 colspan=1>MammaPrint NGS Panel[Box 4 of 4]</td><td rowspan=1 colspan=1>-80°C</td></tr><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>Agendia NGS Bait Library</td><td rowspan=1 colspan=1>36 μL</td></tr><tr><td rowspan=1 colspan=1>MammaPrint NGS Control Box</td><td rowspan=1 colspan=1>-80°C</td></tr><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Volume</td></tr><tr><td rowspan=1 colspan=1>Agendia NGS RNA Control Material (High Risk)</td><td rowspan=1 colspan=1>10 μL</td></tr><tr><td rowspan=1 colspan=1>Agendia NGS RNA Control Material (Low Risk)</td><td rowspan=1 colspan=1>10 μL</td></tr></table>

# 2. Materials Required but Not Provided

There are several reagents that are used in combination with the MammaPrint FFPE NGS kit that are not included in the kit. For a detailed list of required, but not provided reagents and consumables refer to MammaPrint FFPE NGS kit product labeling.

RNeasy DSP FFPE Kit from QIAGEN   
RNA purification reagents   
Sequencing Reagent Kit: The MammaPrint FFPE NGS kit is validated for use with the   
TG MiSeq Reagent Kit v3 Kit (150-cycle). If using additional MiSeq Reagents,   
MammaPrint FFPE NGS assay requires that only Agendia qualified lots of MiSeq   
reagents be used with the device. A list of MiSeq reagent lots that have been qualified by   
Agendia for use with MammaPrint FFPE NGS assays is available on the ADAPT-US

Portal. Reagents must only be used with the instructions for use contained in the package insert. The ADAPT-US software is designed to prevent the use of unqualified lots with the software.

# 3. ADAPT-US Portal and Software

The MammaPrint FFPE NGS kit analysis involves the use of Illumina MiSeq software, Agendia Data Analysis Pipeline Tool-US (ADAPT-US) as well as the MammaPrint data analysis software called RPrint_MP. The ADAPT-US is a customized software platform that provides a secure web-based portal and includes the MammaPrint analysis algorithm RPrint_MP. This analysis software, RPrint_MP is a custom software developed by Agendia. The proprietary Agendia Data Analysis Pipeline Tool –US (ADAPT-US) contains analysis and reporting software necessary for the MammaPrint FFPE NGS kit. The software is compatible with MiSeqDx instruments. A list of approved versions of MiSeq software and reagent lot numbers are available through the ADAPT-US Portal.

# 4. Instrument

The MammaPrint FFPE NGS kit is validated for use on the Illumina MiSeqDx instruments as part of a test system. MiSeqDx instruments must be qualified by an Agendia representative before use with the ADAPT-US platform software. Qualification establishes the instrument for use with the MammaPrint FFPE NGS kit only. Qualification is performed upon server installation and prior to use.

Other required equipment and the specifications for the specific equipment for use with the MammaPrint FFPE NGS kit are described in Table 2.

Table 2. Other Required Equipment, Not Provided   

<table><tr><td rowspan=1 colspan=2>Equipment</td><td rowspan=1 colspan=1>Notes</td></tr><tr><td rowspan=1 colspan=2>8-strip tube microcentrifuge</td><td rowspan=1 colspan=1>Any available</td></tr><tr><td rowspan=1 colspan=2>0.8 mL MIDI plates</td><td rowspan=1 colspan=1>Any available</td></tr><tr><td rowspan=1 colspan=2>Centrifuge</td><td rowspan=1 colspan=1>For 1.5 mL/0.5 mL tubes</td></tr><tr><td rowspan=1 colspan=2>Heat blocks</td><td rowspan=1 colspan=1>37°C for 0.8 mL MIDI plate</td></tr><tr><td rowspan=1 colspan=2>Heat blocks</td><td rowspan=1 colspan=1>65°C for 1.5/2 mL tube</td></tr><tr><td rowspan=1 colspan=2>Heat blocks</td><td rowspan=1 colspan=1>70°C for 0.5 mL tube</td></tr><tr><td rowspan=1 colspan=2>Magnetic stands</td><td rowspan=1 colspan=1>Fits 0.80 mL MIDI platesFits 0.2 mL 8-strip tubesFits 1.5/2 mL tubes</td></tr><tr><td rowspan=1 colspan=2>MiSeqDx system</td><td rowspan=1 colspan=1>Illumina, DX-410-1001</td></tr><tr><td rowspan=1 colspan=2>Multi-channel pipettes (optional)</td><td rowspan=1 colspan=1>1 μL − 1000 μL</td></tr><tr><td rowspan=1 colspan=2>Nucleic acid fragment analysis platform</td><td rowspan=1 colspan=1>To quantify and assess the percentage of RNAfragments &gt; 200 nucleotides (DV200 value) extractedfrom FFPE tissue samples. A nucleic acid fragmentanalysis platform is also used to confirm the sizedistribution of each amplified, target-enriched indexedlibrary.</td></tr><tr><td rowspan=1 colspan=2>Plate centrifuge</td><td rowspan=1 colspan=1>Fits 0.8 mL MIDI plates</td></tr><tr><td rowspan=1 colspan=1>Repeater pipettes (optional)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1 μL − 10 mL</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Equipment</td><td rowspan=1 colspan=1>Notes</td></tr><tr><td rowspan=1 colspan=1>Single channel pipettes</td><td rowspan=1 colspan=1>1 μL − 1000 μL</td></tr><tr><td rowspan=1 colspan=1>Thermal cycler</td><td rowspan=1 colspan=1>Configurable heated lid: ambient - 105°CTemperature range: 4°C - 105°C</td></tr><tr><td rowspan=1 colspan=1>Thermal mixer</td><td rowspan=1 colspan=1>27°C and 65°C1200-1400 rpmFits 0.2 mL 8-strip tubes</td></tr><tr><td rowspan=1 colspan=1>Timer</td><td rowspan=1 colspan=1>NIST traceable</td></tr><tr><td rowspan=1 colspan=1>Vacuum centrifuge</td><td rowspan=1 colspan=1>Temperature range: 15°C to 45°C</td></tr><tr><td rowspan=1 colspan=1>Vortex mixer</td><td rowspan=1 colspan=1>Any available</td></tr></table>

# 5. Sample Preparation

The MammaPrint FFPE NGS kit requires RNA isolated from FFPE tissue specimens. FFPE tumor block for each specimen to be processed are selected by using a tissue sample that contains the greatest amount of invasive carcinoma and is morphologically consistent with the submitted diagnosis.

One slide is used for hematoxylin and eosin (H&E) staining to determine the tumor cell percentage and the remaining slides, depending on the size of the tissue, can all or partly be used for the RNA isolation. The invasive tumor cell percentage must be at least $30 \%$ to obtain valid results. When needed and possible, a macro dissection can be performed to avoid large areas of in-situ carcinoma, necrosis, adipose tissue, stroma and/or hemorrhage as these will decrease the overall invasive tumor cell percentage.

# 6. RNA extraction

MammaPrint FFPE NGS kit requires RNA isolated from FFPE tissue using the RNeasy DSP FFPE Kit from QIAGEN. The quality of the total RNA is assessed by calculating the distribution value (DV) 200 metric, which represents the percentage of RNA fragments ${ > } 2 0 0$ nucleotides (nt). The recommended RNA input for MammaPrint FFPE NGS kit is $2 0 0 \mathrm { n g }$ for poor quality samples $( \mathrm { D V } 2 0 0 \geq 2 0 \%$ above 200 nt) and $1 0 0 ~ \mathrm { n g }$ for good quality samples $\mathrm { ( D V } 2 0 0 \geq 7 0 \%$ above 200 nt) with the total RNA recovered from tissue with a minimum $30 \%$ viable tumor nuclei. The extracted RNA is immediately used for library preparation.

# 7. Library Preparation

The MammaPrint FFPE NGS kit workflow begins with isolation of RNA from FFPE breast cancer tissue sections. FFPE total RNA is chemically fragmented to a target size of $2 0 0 \mathrm { n t }$ and bound to random primers. cDNA is synthesized, end-repaired, adenylated at the 3’end and indexed adapters are ligated. The cDNA adapter-ligated library is enriched by PCR amplification. The amplified adapter-ligated cDNA library is quantified using a suitable nucleic acid fragment analysis platform in the region of between 150 bp-550 bp prior to hybrid capture.

# 8. Hybridization Capture NGS

The adapter-ligated library is hybridized with biotinylated RNA library baits and targeted regions are captured using magnetic streptavidin coated beads. Captured DNA libraries are purified to remove baits and incompletely hybridized DNA fragments. Captured libraries are enriched by PCR amplification. Primer dimers and residual reagents are removed by magnetic bead purification. The quality and quantity of the amplified, target-enriched indexed libraries are

assessed using a suitable nucleic acid fragment analysis platform prior to sequencing. Samples must be within the linear range of the assay $( 5 { - } 5 0 0 ~ \mathrm { p g / \mu L } )$ and the fragment size distribution should be 150-700 bp.

# 9. Sequencing

cDNA adapter-ligated and captured libraries are pooled at equimolar concentrations before sequencing. Sample pools of 1, 2, or $4 \mathrm { n m o l / L }$ starting concentration are denatured, diluted, and loaded on a sequencing flow cell. Single-end sequencing is performed on the MiSeqDx instrument at the length of 150bp using the MiSeq reagent kit V3 (150 cycles) qualified by Agendia.

# 10. Data Analysis

MammaPrint analysis involves the use of MiSeqDx Operating Software. The MammaPrint results are based on the raw data files called FASTQ files that are generated by the MiSeqDx instrument. Generation of these raw data files is based on the workflow “Generate FASTQ” within the Local Run Manager (LRM). No other built-in workflow of the MiSeqDx software is used. Once the FASTQ files are generated, they are transferred to Agendia’s cloud-based analysis tool which generates the MammaPrint results.

The FASTQ files generated by the MiSeqDx system are processed by ADAPT-US, which is a high-performance and data security compliant cloud-based genomics analysis platform that delivers integrated data interpretation of samples processed with the MammaPrint NGS kit. ADAPT-US is the secure customer portal that is used to process FASTQ files against RPrint. The RPrint software generates an output file that is used by the ADAPT Report Generation software. The three technical components of ADAPT-US consist of:

The customer web interface o The web interface component allows customers to select and run sample FASTQ files. The customers can access the output reports created from the report generation components from this web interface.   
The computer system that stores and executes RPrint o This component takes FASTQ files as an input and runs them through Agendia’s RPrint software to produce the RPrint output file. RPrint custom software developed by Agendia is comprised of several dedicated algorithms to determine High Risk, Low Risk or Borderline for recurrence. The RPrint software contains the proprietary MammaPrint algorithms which analyze the FASTQ files provided by the MiSeqDx Sequencer.

The report generation software o The report generation component utilizes the RPrint output file to generate the MammaPrint Technical Reports.

# 11. Controls

Negative Control: A no template control (NTC) can be processed to serve as a negative control to validate the acceptability of all the test samples processed through library preparation and capture steps by testing for sample or reagent contamination. The NTC is expected to fail both the Library Prep and Capture QCs. Passing QCs for the NTC at these steps may indicate contamination.

Positive Control: Two types of positive controls, one that gives a High Risk result and another that gives a Low Risk result are provided in the MammaPrint FFPE NGS kit. One of each type must be included every time a batch of samples is processed. Those two controls are processed from library preparation through sequencing to serve as an end-toend control to demonstrate assay performance. The two controls must pass the quality control model and give MammaPrint Indices (MPI) that fall within the acceptable range for the controls. Failure of the controls to meet either condition will result in all test samples on the run being reported as ‘No Result’.

# 12. Result Reporting

A test result is considered valid only if the Overall Assessment field on the Technical Report says “Pass”. If any of the quality control metrics fail, the Overall Assessment will also indicate “Fail”. If the Overall Assessment says “Fail”, the Technical Report will show "Unable to provide result for this specimen" in the Test Results section. The testing laboratory may choose to retest the sample to see if the subsequent result will yield a valid test result.

The MammaPrint result is reported as Low Risk, High Risk and Borderline for risk of distant metastasis. The test result (Low Risk, High Risk, Borderline) of the sample is determined by calculating the MPI on a scale of -1.000 to $+ 1 . 0 0 0$ with two cut-offs, -0.058 and $+ 0 . 0 5 8$ . High Risk results are those results that are less than -0.058 whereas Low Risk results are those above $+ 0 . 0 5 8$ . If a FFPE sample’s MammaPrint Index (MPI) falls within -0.058 and $+ 0 . 0 5 8$ , then the test result is termed as “Borderline” as these represent a range of scores with low reliability for assessing risk of distant metastasis. The clinical significance of the Borderline region $- 0 . 0 5 8 \leq$ $\mathrm { M P I } \leq + 0 . 0 5 8 )$ is not established.

# 13. Quality Metrics

Quality metrics are assessed in analytical and post-sequencing level. Analytical quality assessment is conducted across the following categories:

FFPE total RNA: Quality Control (QC) assesses quality of the FFPE total RNA based on the DV200 metric.

Amplified, adaptor-ligated cDNA libraries: QC assesses quality (cDNA fragments must fall in the right size range, i.e., between 150 to $5 5 0 \mathrm { b p }$ ) and quantity (should be $2 0 0 \mathrm { n g }$ of cDNA library at a concentration of $5 8 . 8 ~ \mathrm { n g / \mu L }$ ) of the adaptor-ligated cDNA library.

Amplified, target-enriched indexed libraries: QC assesses quality (cDNA fragments must fall in the right size range, i.e., between 150 to $7 0 0 \mathrm { b p }$ ), quantity $( \mathrm { p g / \mu L } )$ and molarity (should be equal to or above $1 0 0 0 \mathrm { p m o l / L } )$ of the amplified, target-enriched indexed library.

The quality controls that assess quality at the level of the NGS run are outlined in the table below.

Table 3. Summary of MammaPrint FFPE NGS kit Post-Sequencing Quality Control Metrics   

<table><tr><td>Quality Metric</td><td>Level of Qualification</td></tr><tr><td>Positive Control  High Risk</td><td>Batch-level</td></tr><tr><td>Positive Control - Low Risk</td><td>Batch-level</td></tr><tr><td>Negative Control - (Water)</td><td>Batch-level</td></tr><tr><td>Total Read Counts (log2)</td><td>Sample-level</td></tr><tr><td>Percent Mapped</td><td>Sample-level</td></tr><tr><td>Percent On Target</td><td>Sample-level</td></tr><tr><td>Percent Q30</td><td>Sample-level</td></tr><tr><td>Additional NGS Run Quality Assessment</td><td>Sample-level</td></tr><tr><td>MammaPrint Total Counts</td><td>Sample-level</td></tr><tr><td>MammaPrint Median Counts</td><td>Sample-level</td></tr></table>

# B Principle of Operation:

The MammaPrint NGS FFPE kit consists of several processes: isolation of RNA from FFPE breast cancer tissue sections; library preparation of RNA resulting in cDNA adapter-ligated sequences; enrichment of the 70 genes (capture step); sequencing of the enriched library in the flow cell and data acquisition; MammaPrint Index (MPI) calculation and determination of the risk classification.

Data analysis is performed according to the specific MammaPrint FFPE algorithm, resulting in determination of the MPI. This algorithm is designed and programmed by Agendia and compiled into a proprietary standalone software program. The software loads the FASTQ data file, performs quality control checks and determines the molecular profile of the sample by calculating the MPI by determining the correlation of the sample 70 gene expression profile to the mean expression profiles of the risk templates of tumors with a known good and poor outcome. The MPI is reported on a scale of -1.000 to $+ 1 . 0 0 0$ (MammaPrint FFPE reportable range), compares the calculated correlation to pre-defined cut-off values and determines the sample’s prognostic profile. High Risk results are those results that are less than -0.058 whereas Low Risk results are those above $+ 0 . 0 5 8$ , except between -0.058 and $+ 0 . 0 5 8$ , which is the region/MPI for Borderline samples. The clinical significance of the Borderline region ( $- 0 . 0 5 8 \leq$ $\mathrm { M P I } \leq + 0 . 0 5 8 $ ) is not established.

# V Substantial Equivalence Information:

A Predicate Device Name(s): MammaPrint® FFPE   
B Predicate 510(k) Number(s): K201902

C Comparison with Predicate(s):   

<table><tr><td>Device &amp; Predicate Device:</td><td>K210973</td><td>K201902</td></tr><tr><td>Device Trade Name</td><td>MammaPrint® FFPE NGS kit General Device Characteristic: Similarities</td><td>MammaPrint® FFPE</td></tr><tr><td>Intended Use/Indications for Use</td><td>The MammaPrint FFPE NGS kit is a qualitative in vitro diagnostic test for use by clinical laboratories using</td><td>MammaPrint® FFPE is a qualitative in vitro diagnostic test, performed in a central laboratory, using the gene expression profile</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">gene MammaPrint Breast Cancersignature on formalin-fixed,paraffin-embedded (FFPE) breastcancer tissue samples. The test isused to assess a patient's risk todevelop distant metastasis within 5years and up to 10 years afterdiagnosis.The MammaPrint FFPE NGS kit isperformed for breast cancer patientswith Stage I or Stage II disease,with tumor size ≤ 5.0 cm and lymphnode negative. The test result isindicated for use by physicians as aprognostic marker only, along withother clinicopathological factors.</td><td colspan="1" rowspan="1">cancer tissue samples to assess apatient's risk for distant metastasiswithin 5 years.The test is performed for breastcancer patients, with Stage I orStage II disease, with tumor size ≤5.0 cm and lymph node negative.The MammaPrint® FFPE result isindicated for use by physicians as aprognostic marker only, along withother clinicopathological factors.</td></tr><tr><td colspan="1" rowspan="1">Clinicalperformance/Indication foruse</td><td colspan="1" rowspan="1">MammaPrint is a test to assess apatients' risk for distant metastasis.The test is performed for breastcancer patients with Stage I and IIdisease, with a tumor size ≤5.0 cmand lymph node negative.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">General Device Characteristic: Differences</td></tr><tr><td colspan="1" rowspan="1">Testing Environment</td><td colspan="1" rowspan="1">Kit</td><td colspan="1" rowspan="1">Performed in Agendia's twocentral laboratories: Amsterdam,Netherlands; and Irvine, California,USA</td></tr><tr><td colspan="1" rowspan="1">Technological Characteristicsand MammaPrint Platform</td><td colspan="1" rowspan="1">The MammaPrint® FFPE NGS kitis an NGS-based gene expressionanalysis of a tumor. The analysis isbased on sequencing of the enrichedlibrary in the flow cell on aMiSeqDx, data acquisition,calculation, and determination ofthe risk of recurrence in breastcancer patients.Analysis is performed usingAgendia designed NGS kitmanufactured under GMP byAgilent technologies.70 signature genes are included inthe Agendia designed bait library.</td><td colspan="1" rowspan="1">The MammaPrint service is amicroarray-based gene expressionanalysis of a tumor. The analysis isbased on scanning theMammaPrint microarray and dataacquisition (feature extraction),calculation and determination ofthe risk of recurrence in breastcancer patients.Analysis is performed usingAgendia designed High Densitydiagnostic Microarraysmanufactured under GMP byAgilent Technologies.70 signature genes printed in nine-fold.</td></tr><tr><td colspan="1" rowspan="1">Analysis software</td><td colspan="1" rowspan="1">Calculation of MammaPrint indexbased on sequence read counts forthe 70 genes measured by the test.</td><td colspan="1" rowspan="1">Calculation of MammaPrint indexbased on measurement ofintensities that result fromhybridization of fluorescentlylabeled cDNA sequences from the70 genes, measured by the test, tooligos on the microarray.</td></tr><tr><td colspan="1" rowspan="1">MammaPrint Index (MPI)and Cut-off</td><td colspan="1" rowspan="1">A numerical MammaPrint Index(MPI) is reported on a scale of -1.000 to +1.000, with two cut-offs, -0.058 and +0.058</td><td colspan="1" rowspan="1">A numerical MammaPrint Index(MPI) is reported on a scale of -1.000 to +1.000, with a single cut-off, +0.000</td></tr><tr><td colspan="1" rowspan="1">Reporting</td><td colspan="1" rowspan="1">MammaPrint result of Low Risk,High Risk, or Borderline isprovided in a report to the orderinghealth care provider.</td><td colspan="1" rowspan="1">MammaPrint result of Low Risk,High Risk, Low Risk Borderline,or High Risk Borderline isprovided in a report to the orderinghealth care provider.</td></tr><tr><td colspan="1" rowspan="1">Sample Processing</td><td colspan="1" rowspan="1">Isolation of RNA from formalin-fixed paraffin embedded (FFPE)tumor tissue sections, DNasetreatment of isolated RNA, librarypreparation of RNA resulting incDNA adapter-ligated sequences;enrichment of the 70 genes usinghybrid capture; sequencing of theenriched library in the flow cell on aMiSeqDx.</td><td colspan="1" rowspan="1">Isolation of RNA from formalin-fixed paraffin embedded (FFPE)tumor tissue sections, DNasetreatment of isolated RNA, linearamplification and labeling ofDNase treated RNA (fresh), cRNApurification, hybridization of thecRNA to the MammaPrintmicroarray (fresh), amplificationand purification DNase treatedRNA resulting in cDNA for FFPE,labeling and purification ofamplified cDNA for FFPE,hybridization of the cDNA to theMammaPrint microarray for FFPE.</td></tr><tr><td colspan="1" rowspan="1">Data Analysis Software</td><td colspan="1" rowspan="1">RPrint v1.4.1_MP</td><td colspan="1" rowspan="1">XPrint version 3.2.0</td></tr><tr><td colspan="1" rowspan="1">Instrument</td><td colspan="1" rowspan="1">Illumina MiSeqDx (Qualified byAgendia)</td><td colspan="1" rowspan="1">Agilent SureScan Dx microarrayscanner, part number G5761AA.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

1) EP17-A2: Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline 2nd Edition, Institute, CLSI - Clinical and Laboratory Standards, 2012.

2) EP05-A3: Evaluation of Precision of Quantitative Measurement Procedures; Approved Guidline-3rd Edition.

3) EP09c: Measurement Procedure Comparison and Bias Estimation Using Patient Samples, 3rd

Edition, Clinical and Laboratory Standards Institute, 2018.

4) Gene Expression Profiling Test System for Breast Cancer Prognosis - Class II Special Controls Guidance for Industry and FDA Staff: issued on: May 9, 2007.   
5) Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, Guidance for Industry and FDA Staff, issued on March 13, 2007.   
6) Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, Guidance for Industry and FDA Staff, issued on May 11, 2005.   
7) Cybersecurity concerns for Networked Medical Devices Containing Off-the-Shelf (OTS) Software: Guidance for Industry, issued on January 14, 2005.   
8) Content of Premarket Submissions for Management of cybersecurity concerns in Medical Devices – Guidance for Industry and Food and Drug Administration Staff, issued on Octobe 2, 2014.   
9) Postmarket Management of cybersecurity concerns in Medical Devices - Guidance for Industry and Food and Drug Administration Staff, issued on December 28, 2016.

# VII Performance Characteristics:

# A Analytical Performance:

Concordance between MammaPrint FFPE NGS kit run on MiSeq Research Use Only (RUO) instrument and run on MiSeqDx instrument qualified by Agendia.

The purpose of this study was to establish the concordance between the MammaPrint FFPE NGS kit run on the MiSeq RUO instrument and the MammaPrint FFPE NGS kit run on the MiSeqDx instrument qualified by Agendia to enable the use of MammaPrint FFPE NGS kit data generated from MiSeq RUO instruments. The MiSeqDx instrument qualification was performed at the Agendia laboratory in Irvine, USA. MiSeq RUO instruments were used initially for clinical validation and analytical performance studies including repeatability of RNA isolation and kit stability testing. In this study, RNA extracted from 50 FFPE samples were selected based on results obtained from MammaPrint FFPE NGS kit run on the MiSeq RUO instrument. These samples were subsequently analyzed using the MammaPrint FFPE NGS kit run on the MiSeqDx qualified by Agendia. The MammaPrint FFPE NGS kit results from both sequencers were compared to assess the agreement between the two sequencers. Two samples failed QC when tested with the MammaPrint FFPE NGS kit run on the MiSeqDx qualified by Agendia, and therefore provided invalid results. Agreement between the two sequencers was calculated for valid results generated by both sequencers. Since the MammaPrint FFPE NGS kit test reports three (3) categories, High Risk, Low Risk, and Borderline, around two cut-offs, -0.058 and $+ 0 . 0 5 8$ , agreements were evaluated for each category and presented in Table 4 below.

Table 4. Concordance Between MammaPrint FFPE NGS kit run on MiSeq RUO instrument and run on MiSeqDx instrument qualified by Agendia   

<table><tr><td rowspan=1 colspan=3>MiSeq RUO</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>Total</td></tr></table>

<table><tr><td rowspan=3 colspan=1>MiSeq-Dx</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=2>Agreement- High Risk (95% CI)</td><td rowspan=1 colspan=4>27/29 = 93.10% (78.04, 98.09)</td></tr><tr><td rowspan=1 colspan=2>Agreement Borderline (95% CI)</td><td rowspan=1 colspan=4>1/2 = 50% (9.40, 90.60)</td></tr><tr><td rowspan=1 colspan=2>Agreement- Low Risk (95% CI)</td><td rowspan=1 colspan=4>16/17 = 94.12% (73.02, 98.96)</td></tr><tr><td rowspan=1 colspan=2>Agreement- Overall (95% CI)</td><td rowspan=1 colspan=4>44/48 = 91.67% (80.02, 97.68)</td></tr></table>

The percent agreement for samples with High Risk and Low Risk categories was 93.10 ( $9 5 \%$ CI: 78.04, 98.09) and 94.12 $9 5 \%$ CI: 73.02, 98.96) respectively, demonstrating comparable performance of MammaPrint FFPE NGS kit between MiSeq RUO instrument and MiSeqDx instrument qualified by Agendia. While the percent agreement for Borderline cases was low, 50 ( $9 5 \%$ CI: 9.40, 90.60), this is likely due to the low reliability of the test for samples with MPI scores between -0.058 and $+ 0 . 0 5 8$ .

# 1. Precision/Reproducibility:

Two site-to-site reproducibility studies were conducted at multiple sites to demonstrate the precision and reproducibility of the MammaPrint FFPE NGS kit using RNA extracted from samples representing a range of MPI scores. In addition, the reproducibility of RNA from control samples was also assessed in a reproducibility study. An RNA isolation repeatability study was performed by conducting two independent RNA isolations from the same tissue block to demonstrate that MammaPrint results are reproducible between two isolations from the same tumor.

# Site-to-site Reproducibility Study-1

In Study-1, site-to-site reproducibility of the MammaPrint FFPE NGS kit was assessed across four external sites in the United States, using three samples. The three samples were generated by pooling of RNA representing the different MammaPrint categorical test results (High Risk, Low Risk, and Borderline). Per site, each sample was tested by two operators across six (6) runs in duplicate for a total of 24 measurements obtained per sample per site. In total, 96 replicates per sample across the four sites were evaluated. The study design included three (3) different MammaPrint FFPE NGS kit lots. A variance component analysis was used to estimate the standard deviation for repeatability, between-run, between-operator, between-site, and reproducibility for the study. These estimates were reported along with the number of observations for the mean of MammaPrint Index (MPI) values. Results by variance components and total variance are presented below for MPI. The column “N” included the number of data points that generated an MPI value. Across all variance components, the total standard deviation (SD) was ${ \underline { { - 0 . 0 9 5 } } }$ in all samples tested (Table 5).

Table 5: Site-to-Site Reproducibility Results Study 1 - Overall Mean, Standard Deviation (SD) for MPI – Repeatability, Between-Run, Between-Operator, Between-Site, and Reproducibility

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>Repeatability(SD)</td><td rowspan=1 colspan=1>Between-Run(SD)</td><td rowspan=1 colspan=1>Between-Operator(SD)</td><td rowspan=1 colspan=1>Between-Site(SD)</td><td rowspan=1 colspan=1>Reproducibility(SD)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-1.153</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.050</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.067</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-0.0125</td><td rowspan=1 colspan=1>0.055</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>0.095</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>0.436</td><td rowspan=1 colspan=1>0.045</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.022</td><td rowspan=1 colspan=1>0.039</td><td rowspan=1 colspan=1>0.063</td></tr></table>

In Table 6, results are summarized on the sample level agreements based on majority call along with the corresponding two-sided exact $9 5 \%$ confidence limit.

Table 6: Site-to-Site Reproducibility Results Study 1 – Sample Level Agreement   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category</td><td rowspan=1 colspan=1>Total # ofreplicates(N)</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>AgreementN</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95%2-sided score CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-0.013</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>72.0, 88.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>-1.154</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9, 100</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>0.436</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96.9, 100</td></tr></table>

Results of the sample level agreement were $100 \%$ for High Risk and Low Risk samples, and $8 1 \%$ for Borderline sample with the lower bound of the two-sided $9 5 \%$ score CI was $72 \%$ . The mean MPI score of this sample was -0.013, which was close to the cut-off of -0.058, thus explaining the disagreement of MPI score results near the threshold.

Results of this study demonstrate that for MPI scores well above and below the Borderline range of -0.058 and $+ 0 . 0 5 8$ , the precision is high. However, for samples with MPI scores near the two cut-offs, -0.058 and $+ 0 . 0 5 8$ precision may be lower as shown by this study.

# Site-to-site Reproducibility Study-2

A second site-to-site reproducibility study was conducted by testing an additional eight (8) samples that were specifically selected to represent MPI scores close to or within the Borderline category. The eight (8) samples were analyzed at three different sites, two external site and an internal Agendia site. At each site, the samples were separately processed by two operators. Each of the operators processed the samples twice and performed their own RNA extraction for both sequencing runs. For both operators combined, this resulted in a total of four runs per site. Each run was performed on a different day to incorporate day-to-day variations. Per run, each sample was analyzed in duplicate, resulting in 8 replicate measurements per sample and site for a total of 24 replicates per sample. The sequencing runs were performed with Agendia pre-qualified MiSeqDx instruments. A variance component analysis was used to estimate the standard deviation for repeatability, between-run, between-operator, between-site, and reproducibility for the study. Results are summarized in Table 7. Across all variance components, the total standard deviation was $\underline { { < } } 0 . 0 6 6$ in all samples tested.

Table 7: Site-to-Site Reproducibility Results Study 2 - Overall Mean, Standard Deviation (SD) for MPI – Repeatability, Between-Run, Between-Operator, Between-Site, and Reproducibility   

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">RiskCategory</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">MPIMean</td><td colspan="1" rowspan="1">Repeatability(SD)</td><td colspan="1" rowspan="1">Between-Run</td><td colspan="1" rowspan="1">Between-Operator</td><td colspan="1" rowspan="1">Between-Site</td><td colspan="1" rowspan="1">Reproducibility(SD)</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">High Risk</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">-0.177</td><td colspan="1" rowspan="1">0.025</td><td colspan="1" rowspan="1">0.006</td><td colspan="1" rowspan="1">0.014</td><td colspan="1" rowspan="1">0.006</td><td colspan="1" rowspan="1">0.031</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Low Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0.074</td><td colspan="1" rowspan="1">0.020</td><td colspan="1" rowspan="1">0.034</td><td colspan="1" rowspan="1">0.008</td><td colspan="1" rowspan="1">0.012</td><td colspan="1" rowspan="1">0.042</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Low Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0.155</td><td colspan="1" rowspan="1">0.044</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.015</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.046</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Low Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0.223</td><td colspan="1" rowspan="1">0.037</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.009</td><td colspan="1" rowspan="1">0.021</td><td colspan="1" rowspan="1">0.044</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">High Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">-0.060</td><td colspan="1" rowspan="1">0.017</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.015</td><td colspan="1" rowspan="1">0.021</td><td colspan="1" rowspan="1">0.031</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">High Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">-0.142</td><td colspan="1" rowspan="1">0.060</td><td colspan="1" rowspan="1">0.013</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.023</td><td colspan="1" rowspan="1">0.066</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Borderline</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0.012</td><td colspan="1" rowspan="1">0.018</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.024</td><td colspan="1" rowspan="1">0.017</td><td colspan="1" rowspan="1">0.035</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">High Risk</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">-0.165</td><td colspan="1" rowspan="1">0.022</td><td colspan="1" rowspan="1">0.028</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.036</td></tr></table>

In Table 8, results are summarized on the sample level agreements based on majority call along with the corresponding two-sided exact $9 5 \%$ confidence limit.

Table 8: Site-to-Site Reproducibility Results Study 2 – Sample Level Agreement   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Risk Category</td><td rowspan=1 colspan=1>Total # ofreplicates(N)</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>AgreementN</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95%2-sided score CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>-0.177</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>82.9, 100</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.074</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>62.6, 95.3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.155</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.223</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.060</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>36.6, 77.9</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.142</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.012</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>-0.165</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>88.3, 100</td></tr></table>

Of the eight samples, one sample failed QC at one of the sites and therefore only 16 replicates from the other two sites were used in the analysis.

In the variance analysis based on the ANOVA model output, the total standard deviation (SD) was $\underline { { < } } 0 . 0 6 6$ in all eight samples tested. In the concordance analysis, six samples produced $100 \%$ agreement across all replicates, while sample 2 and sample 5 had $8 3 \%$ and $53 \%$ agreement only. The lower concordance of those two samples was because the MPI scores were near the cutoff point of -0.058 and $+ 0 . 0 5 8$ .

Together the results of the second site-to-site reproducibility study demonstrates that MammaPrint FFPE NGS test results are reproducible when MPI scores are well above and below -0.058 and $+ 0 . 0 5 8$ cut-offs.

# Reproducibility of MammaPrint FFPE NGS kit Controls

The reproducibility of MammaPrint FFPE NGS kit was also assessed by processing two control samples with known MammaPrint results at the four external sites. RNA samples from these two controls, one control High Risk sample (CTRL-HR) and a second control Low Risk sample (CTRL-LR) were provided by Agendia to these sites. Per site, there were two operators who processed this set of 2 samples across 6 runs until 12 measurements were obtained per site. Additionally, there were three different lot numbers of the MammaPrint FFPE NGS kit included in this set up, each operator used each of the different lot numbers twice during these 6 different days. For each control sample 50 QC passed measurements were obtained. Five measurements failed QC for the two samples and therefore provided invalid results. A variance component analysis was used to estimate the standard deviation for repeatability, between-run, between

operator, between-site, and reproducibility for the study. Results are summarized in Table 9. Additionally, the MammaPrint FFPE results generated for these 2 control samples at the four sites were analyzed by calculating sample level concordance/agreement based on majority call (Table 10).

Table 9: Reproducibility of MammaPrint FFPE NGS kit Controls - Overall Mean, Standard Deviation (SD) for MPI – Repeatability, Between-Run, Between-Operator, Between-Site, and Reproducibility   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>RiskCategory</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MPIMean</td><td rowspan=1 colspan=1>Repeatability(SD)</td><td rowspan=1 colspan=1>BetweenRun</td><td rowspan=1 colspan=1>BetweenOperator</td><td rowspan=1 colspan=1>BetweenSite</td><td rowspan=1 colspan=1>Reproducibility(SD)</td></tr><tr><td rowspan=1 colspan=1>1 - CTRL-HR</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>-0.512</td><td rowspan=1 colspan=1>0.042</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.039</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.057</td></tr><tr><td rowspan=1 colspan=1>2 - CTRL-LR</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0.359</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.036</td><td rowspan=1 colspan=1>0.065</td></tr></table>

Table 10: Reproducibility of MammaPrint FFPE NGS kit Controls   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>RiskCategory</td><td rowspan=1 colspan=1>Mean MPI</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>AgreementN</td><td rowspan=1 colspan=1>Agreement(%)</td><td rowspan=1 colspan=1>95% 2-sided scoreCI</td></tr><tr><td rowspan=1 colspan=1>1 - CTRL-HR</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>-0.512</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.2, 100</td></tr><tr><td rowspan=1 colspan=1>2 - CTRL-LR</td><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0.359</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94.2, 100</td></tr></table>

The variant component analysis used 48 measurements and the standard deviation was $\underline { { < } } 0 . 0 6 5$ for the two control samples tested. For the analyses of the sample level agreement, there were in total 50 measurements, which included two additional measurements from two of the sites. The agreement of MammaPrint categorical results for these two samples were $100 \%$ for High Risk and Low Risk samples, demonstrating acceptable reproducibility for the two control samples.

# Repeatability of RNA isolation

The purpose of this study was to determine the variability between RNA isolations of the same FFPE tissue by comparing the MammaPrint-NGS results of two RNA isolations from each FFPE tissue sample. In this study, fifty-one (51) samples of high and Low Risk categories with enough FFPE tissue were isolated 2 times by a single operator, for a total of 3 operators, in 2 testing sites, one internal and one external sites through multiple testing days. After the isolations were performed the samples were processed further (Library preparation, Enrichment, and Sequencing) on different days per isolation. This resulted in the generation of two results per tissue sample processed on different days. Of the 51 selected samples, 8 did not pass the QC and therefore provided invalid results. Agreement between the two RNA isolations was calculated for valid test results and was found to be $100 \%$ for all three (3) categories, i.e., High Risk, Low Risk, and Borderline calls $9 5 \% \mathrm { C I }$ : High Risk: 86.2,100.0, Low Risk: 81.6,100.0, Borderline: 34.2, 100.0), demonstrating high concordance between isolations.

2. Linearity: Not applicable

# 3. Analytical Specificity/Interference:

# Interference

The impact of interfering substances on the performance of the MammaPrint FFPE NGS kit was assessed by processing RNA from FPPE samples tested in the presence of five substances, genomic DNA $( \mathrm { g D N A } )$ , proteinase K, actinomycin D, ethanol and NaOH, with each interfering substance at varying amounts. The first two substances, gDNA and proteinase K were used in the RNA isolation, while the other three substances, actinomycin D, ethanol and NaOH were used during the NGS processing of the samples. The impact of potential interference from gDNA, proteinase K, actinomycin D, and ethanol was tested by testing three (3) samples, one High Risk, one Low Risk and one Borderline measured in duplicate with a total of six (6) measurements with and without potential interferent.

To evaluate potential interferent from gDNA and proteinase K during RNA isolation, the samples were tested with $1 0 0 ~ \mathrm { n g }$ of gDNA, and $2 0 { , } 0 0 0 \mathrm { n g }$ proteinase K. Actinomycin D is used during the NGS process with a standard concentration of $4 \mu \mathrm { g / \mu L }$ . To assess the potential interference from actinomycin D, the three (3) samples were tested with $2 4 0 ~ \mathrm { { n g / \mu L } }$ and 480 $\mathrm { n g / \mu L }$ Actinomycin D. Ethanol in its standard setting $( 7 0 \% )$ , is used in multiple steps during the NGS process. For this assessment, the three samples were tested with an excess of $2 . 5 \%$ ethanol and an excess of $5 \%$ ethanol. Both concentrations were tested separately and in duplicate during the library preparation step and the capture preparation step, resulting in four (4) datasets of six (6) measurements each.

NaOH (0.2N) is used during the denaturation of pooled cDNA Library in the NGS process. For this assessment, the pooled cDNA library set containing ${ \mathfrak { n } } = 2 9$ samples was tested separately with 3 mM NaOH and $5 \ \mathrm { m M } \ \mathrm { N a O H }$ and measured in duplicate.

The impact of the substances was assessed by assessing QC pass/fail, comparison of MammaPrint index with regards to absolute differences between the results of the standard/original versus test conditions, i.e., presence of interferent, and concordance with the MammaPrint FFPE NGS results for standard/original versus test condition.

Except for NaOH tested at 5mM during denaturation of pooled cDNA Library, all samples with and without potential interferent passed QC, i.e., provided valid results. The absolute difference between MammaPrint indices of both conditions, with or without interferent were below the technical variance of the MammaPrint FFPE NGS kit. Further, concordance was $100 \%$ between MammaPrint FFPE NGS results obtained with or without potential interferent for all conditions evaluated except for excess of $3 \mathrm { m M N a O H }$ .

Excess of 3mM NaOH during the denaturation of pooled cDNA Library, resulted in agreement for high and Low Risk samples of $100 \%$ and $9 5 . 8 \%$ respectively. Further, two samples had slightly higher absolute differences in MPI. Excess of $5 \mathrm { m M N a O H }$ during the denaturation of pooled cDNA Library resulted in $1 7 . 2 \%$ of the measurements failed QC. However, the categorical MammaPrint FFPE NGS kit results were not impacted. According to the results, a limitation in the package insert was established to caution that excess of $3 \mathrm { m M N a O H }$ should not be used for the denaturation of pooled cDNA Library in the NGS process as it can lead to increased assay failure.

Further, RNA specification provided are designed to minimize the presence of any effect from interference of components such as hemorrhagic tissue, adipose tissue fibrosis material or necrotic tissue on the performance of the MammaPrint FFPE NGS kit.

# 4. Assay Reportable Range:

The MammaPrint Index is reported on a scale of -1.000 to $+ 1 . 0 0 0$ , with two cut-offs, -0.058 and $+ 0 . 0 5 8$ and determines the sample’s prognostic profile as Low Risk, High Risk, or Borderline. High Risk results are those results that are below -0.058 whereas Low Risk results are those above $+ 0 . 0 5 8$ , except between -0.058 and $+ 0 . 0 5 8$ , samples will be reported as Borderline.

# 5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

# a) Real-Time Stability of MammaPrint FFPE NGS kit

A real-time stability study was executed to determine the shelf life of the MammaPrint FFPE NGS kit. Three lots of NGS kits, which includes RNA Preparation Kit, Target Enrichment Kit, and the NGS Panel, and each kit is comprised of four boxes, were stored at its recommended storage conditions, and tested at discrete timepoints over the course of the study. In total eleven (11) clinical samples including High Risk, Low Risk and Borderline, were tested at three different time points, 2-3 months from Date of Manufacture (DOM), corresponding to baseline or time point zero, 12 months from DOM, time point 1 (T1), and 26 months from DOM, time point 2 (T2). For each measurement, the quality assessment, test results and test indices were recorded and reviewed for stability. The T0 measurement is used as the reference timepoint – all subsequent measurements were be compared with the T0 measurement. Data from T0, T1 and T2 time points were analyzed for Lot #1, Lot $\# 2$ , and Lot #3. For Lot #1, all measurements in timepoint T1 showed agreement with T0; however, in timepoint T2, one sample switched from a MammaPrint Low Risk result to a Borderline result as the MammaPrint index fell within $- 0 . 0 5 8 \mathrm { a n d } + 0 . 0 5 8 .$ . The difference in MammaPrint index between the T0 and the T2 for this sample was within the predefined allowable difference. Since the difference in MammaPrint Index for this Borderline sample was within the allowable difference, the switch in MammaPrint test outcome was accepted as Borderline call for the sample. Therefore, all timepoints for lot #1 demonstrated acceptable stability. For Lot #2, one High Risk sample at timepoint T1 showed an absolute difference in MammaPrint index between the T1 and $\mathrm { T 0 }$ results which exceeded the allowable difference, however because it demonstrated agreement with the T0 call, i.e., High Risk call, it was considered acceptable. For Lot #3 real-time stability studies, all samples passed T0, T1 and T2 time points. The results support a shelf life for the MammaPrint FFPE NGS kit of 25 months when stored according to the temperatures indicated on the label.

# b) Freeze-Thaw Stability

To demonstrate that the performance of MammaPrint FFPE NGS kit will remain stable under varied conditions, a freeze thaw assessment was performed using three lots of MammaPrint FFPE NGS kits for a total of six (6) cycles with assessments taken at three (3) time points. Three (3) samples, one High Risk, one Low Risk and one Borderline were tested after one freeze thaw cycle of the MammaPrint FFPE NGS kit (C0), after 3 freeze-thaw (F/T) cycles (C1) and after 6 F/T cycles (C2). For each cycle, the

components of the MammaPrint FFPE NGS kit were completely thawed and then placed back into their labeled storage conditions for at least 4 hours to ensure complete freezing. Results demonstrated that all samples passed QC after each testing, that sample results after $3 \ \mathrm { F } / \mathrm { T }$ cycles and $6 \mathrm { F / T }$ cycles were concordant with the MammaPrint FFPE NGS kit results at C0 and that MPI were within acceptable difference with C0. Based on the results, the MammaPrint FFPE NGS kit can be used for up to $5 \mathrm { F / T }$ cycles.

# c) Shipment Stability

MammaPrint FFPE NGS kit are shipped as small and medium packaging solutions, with each solution, there are two boxes – one for room temperature reagents and one for reagents that need to be kept cold (with dry ice), for all $- 8 0 ^ { \circ } \mathrm { C }$ and ${ } _ { - 2 0 } \mathrm { { } ^ { \circ } C }$ reagents. A shipping study was also performed to ensure that the MammaPrint FFPE NGS kit maintains the adequate level of performance after shipment. Within this study three packaging configurations were tested with a small number of kits (1 or 2), one for a medium number of kits (3 to 8), and one for a large number of kits (9 to 15). Controlled shipping simulations were conducted based on recognized standards and successively demonstrated that the packaging provided sufficient physical protection to its contents. All kits from the small and medium packaging solutions passed the inspection and were subsequently tested for performance assessment using samples that had been previously processed with accepted kits. For all samples and kits tested, the difference in MPI were within acceptable limits.

# d) Sample stability

The clinical validation utilized samples that were between 6 and 10 years old. The DV200 was used as an indication of RNA quality that is isolated from the block, which indicates tissue stability.

Out of the 345 RASTER samples used for MammaPrint microarray validation, 316 samples were successfully processed using the MammaPrint FFPE NGS kit. In the clinical evaluation, there was a relatively large proportion of samples with a lower DV200 value, however these samples generated an appropriate NGS result as demonstrated in the clinical validation study, see Section C below. In addition, several QC measures are included in the MammaPrint NGS kit to prevent erroneous results from being generated due to sample quality (e.g., unacceptable sample degradation). Following assessment of the starting material (tissue/RNA), a QC model that measures the quality of the sequencing (such as Percent Mapped, Percent on Target, and Total Read Count) is used. Additionally, MammaPrint-specific QCs included in this QC model (e.g., MammaPrint Total Counts and Median Counts) ensure quality of the MammaPrint result. Failure of the quality metrics is interpreted by the test as an indication of low tissue quality or low RNA quality.

# 6. Detection Limit:

Two studies were conducted to demonstrate the acceptable range of RNA input for MammaPrint FFPE NGS kit.

In the first study, eight (8) FFPE breast cancer samples with DV200 “poor” quality (less than $5 0 \%$ were evaluated. The selected samples cover the entire MammaPrint Index range, including the Borderline region. RNA of each sample was diluted to lower the RNA input for the

MammaPrint FFPE NGS kit from $2 0 0 \mathrm { n g }$ to $1 0 0 ~ \mathrm { n g }$ , $5 0 \mathrm { n g }$ , $2 5 { \mathrm { n g } }$ , and $1 2 . 5 \mathrm { n g }$ . For each RNA input level, all samples were processed repeatedly during several separate runs using the MammaPrint FFPE NGS kit. Samples that passed all lab Quality Controls (QCs) were subsequently sequenced. The eight (8) samples were repeatedly processed for each of the five RNA input levels resulting in a total of 32 replicates processed with $1 2 . 5 \mathrm { n g }$ RNA and 40 replicates processed for all other RNA input levels. QC passing rate was evaluated. The results of this study demonstrated that samples with at least $1 0 0 ~ \mathrm { n g }$ RNA input and with DV200 values ranging from $3 5 \%$ to $49 \%$ had valid rates ranging from $80 \%$ to $100 \%$ . Samples with RNA input values below $1 0 0 ~ \mathrm { n g }$ and with DV200 “poor” quality ranging from $3 5 \%$ to $4 9 \%$ had valid rates ranging from $0 \%$ to $100 \%$ .

To further assess whether RNA input (ng) below and above the RNA quality specific recommendations in the instructions for use influences the assay performance, an additional study was performed on DV200 “poor” (less than $50 \%$ ) and $\mathrm { D V } 2 0 0 \ " \mathrm { g o o d } ^ { \prime \prime }$ (greater or equal to $50 \%$ ) samples separately. The limit chosen for the above and below recommended RNA input are $2 0 0 \mathrm { n g }$ for DV200 “poor” samples and $1 0 0 ~ \mathrm { n g }$ for better quality samples. For DV200 “Poor” samples, $3 0 0 \mathrm { n g } , 2 0 0 \mathrm { n g }$ , and $1 0 0 \mathrm { n g } \mathrm { R N A }$ input were tested, while $2 0 0 \mathrm { n g }$ , 100, ng, $5 0 \mathrm { n g }$ , and $2 5 { \mathrm { n g } }$ RNA input were tested for DV200 “Standard” (greater or equal to $70 \%$ ) samples. The same eight samples used for the LoD study were selected for the DV200 “Poor” experiment. For RNA input of DV200 “Poor” samples, 40 replicates were tested for $2 0 0 \mathrm { n g }$ and $1 0 0 \mathrm { n g }$ RNA input, while 14 replicates for 300ng RNA concentration. For DV200 “Standard” experiment, 6 samples were selected in which 2 were from the middle category DV200 “Good to Medium”, based on their DV200 values. The 6 samples assessed contained equal numbers of High Risk and Low Risk samples and covers the entire MammaPrint Index range. In DV200 “Standard” samples, 18 replicates were tested for $2 0 0 \mathrm { n g }$ to $5 0 \mathrm { n g }$ dilutions and 15 replicates for 25ng RNA input. QC passing rate was evaluated. The results of this study demonstrated that samples with at least $1 0 0 ~ \mathrm { n g }$ RNA input and with DV200 values ranging from $3 5 \%$ to $80 \%$ had valid rates ranging from $80 \%$ to $100 \%$ , while samples with $2 0 0 \mathrm { n g }$ RNA input and with DV200 values ranging from $3 5 \%$ to $80 \%$ had valid rates of $8 3 \%$ to $100 \%$ . Samples with RNA input values below $1 0 0 ~ \mathrm { n g }$ and with DV200 “poor” quality ranging from $3 5 \%$ to $4 9 \%$ had valid rates ranging from $0 \%$ to $100 \%$ .

Together, the data supports the specifications for $\mathrm { D V } 2 0 0 { \geq } 7 0 \%$ above 200 nt for $1 0 0 ~ \mathrm { n g }$ , and $\mathrm { D V } 2 0 0 { \geq } 2 0 \%$ above 200 nt for $2 0 0 \mathrm { n g }$ . For $1 5 0 \mathrm { n g }$ the data supports $\mathrm { D V } 2 0 0 { \geq } 5 0 \%$ because at $1 0 0 \mathrm { n g } \mathrm { R N A }$ input with DV200 ranging from $3 5 \%$ to $40 \%$ the valid rate was at least $80 \%$ as observed from the two studies.

# 7. Assay Cut-Off:

MPI cut-off is set at -0.058 and $+ 0 . 0 5 8$ for MammaPrint FFPE NGS kit.

# B Comparison Studies:

Two method comparison studies were conducted at a single site (Study 1) and at multiple sites (Study 2) to demonstrate the agreement between MammaPrint FFPE NGS kit and the MammaPrint FFPE microarray test using RNA extracted from samples representing a range of MPI scores.

1. Method Comparison with Predicate Device – Study 1:

The concordance between the MammaPrint FFPE NGS kit and the MammaPrint FFPE microarray was assessed by testing a total of 155 samples. These samples complied with the intended use population of the MammaPrint FFPE NGS kit. These samples were previously processed on MammaPrint FFPE microarray as part of routine diagnostics and covered the entire MammaPrint readout range with sufficient number of samples representing High Risk, Low Risk and Borderline. None of these samples were used in the development of the MammaPrint NGS FFPE kit. Of the 155 samples, 90 were with High Risk MammaPrint Index while 65 were with Low Risk index, as determined by the microarray analysis using MammaPrint FFPE.

Since the NGS kit has three output categories (High Risk, Borderline, and Low Risk) and the comparator method has two output categories (High Risk and Low Risk), comparison of MammaPrint categorical results was performed by conducting two concordance analysis for High vs Low Risk.

In the first concordance analysis, samples with MPI scores ${ < } 0 . 0 5 8$ (this also includes Borderline samples with MPI scores of ${ < } 0 . 0 5 8$ and $_ { \mathrm { > - 0 . 0 5 8 } }$ ) were treated as High Risk, that is, positive results in the concordance analysis. Samples with MPI scores ${ > } 0 . 0 5 8$ were treated as Low Risk or negative results. In the second concordance analysis, samples with MPI scores $< - 0 . 0 5 8$ were treated as High Risk or positive results in the concordance analysis. Samples with MPI scores $> - 0 . 0 5 8$ (this also includes Borderline samples with MPI scores of ${ < } 0 . 0 5 8$ and $_ { \mathrm { > - 0 . 0 5 8 } }$ ) were treated as Low Risk or negative results.

For the first analysis, comparison of the test results showed an Overall Percent Concordance (OPA) of $9 7 . 4 2 \%$ ( $9 5 \%$ CI: 93.55, 98.99) with a Negative Percent Agreement (NPA) of $9 3 . 8 5 \%$ ( $9 5 \%$ CI: 85.22, 97.58) and a Positive Percent Agreement (PPA) of 100.00 ( $9 5 \%$ CI: 95.91, 100.00).

For the second analysis, comparison of the test results showed an OPA of $9 8 . 7 1 \ \%$ ( $9 5 \%$ CI: 95.42, 99.65) with a NPA of $1 0 0 . 0 0 \%$ ( $9 5 \%$ CI: 94.42, 100.00) and a PPA of 97.78 $9 5 \%$ C I: 92.26, 99.39).

Table 12: Comparison of MammaPrint test results between NGS kit and microarray for NPA and PPA calculation   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>MammaPrint FFPE Microarray</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td></tr><tr><td rowspan=3 colspan=1>MammaPrintFFPE NGS</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>155</td></tr><tr><td rowspan=1 colspan=5>Concordance using MPI score of +0.058 as cut-off (i.e., Borderline NGS = High Risk)</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=3>100% (95% CI: 95.91-100)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI)</td><td rowspan=1 colspan=3>93.85%(95% CI: 85.22-97.58)</td></tr><tr><td rowspan=1 colspan=1>OPA (95% CI)</td><td rowspan=1 colspan=3>97.42% (95% CI: 93.55-98.99)</td></tr><tr><td rowspan=1 colspan=2>Concordance using MPI score of -0.</td><td rowspan=1 colspan=3>Concordance using MPI score of -0.058 as cut-off (i.e., Borderline NGS = Low Risk)</td></tr><tr><td rowspan=3 colspan=1>Agreement(Total N=155)</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=3>97.78% (95% CI: 92.26-99.39)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI)</td><td rowspan=1 colspan=3>100%6 (95% CI: 94.42-100)</td></tr><tr><td rowspan=1 colspan=1>OPA (95% CI)</td><td rowspan=1 colspan=3>98.71% (95% CI: 95.42-99.65)</td></tr></table>

The Concordance, NPA and PPA of MammaPrint categorical results support agreement between the MammaPrint FFPE NGS kit and the predicate, MammaPrint FFPE microarray (Tables 12).

# 2. Method Comparison with Predicate Device Study-2

The comparability of the MammaPrint FFPE NGS kit and the MammaPrint microarray was assessed by evaluating 303 FFPE samples. The 303 samples were processed by the microarray test at Agendia. Then, the same samples were processed with the MammaPrint FFPE on the NGS platform at four external sites, i.e., 74 being processed at Site 1, 76 at Site 2, 78 at Site 3, and 75 at Site 4. The distribution of the categorical MammaPrint results analyzed on microarray varied slightly between the datasets selected by the four different sites (Table 13)

Table 13: Overview of the MammaPrint results of the dataset based on the microarray analysis   

<table><tr><td rowspan="2"></td><td rowspan="2">MammaPrint FFPE Microarray</td><td colspan="2">Distribution of results</td></tr><tr><td>Number of samples</td><td>%</td></tr><tr><td rowspan="2">Site 1</td><td rowspan="2">High Risk Low Risk</td><td>33</td><td>44.6</td></tr><tr><td>41</td><td>55.4</td></tr><tr><td rowspan="2">Site 2</td><td rowspan="2">High Risk Low Risk</td><td>46</td><td>60.5</td></tr><tr><td>30</td><td>39.5</td></tr><tr><td rowspan="2">Site 3</td><td rowspan="2">High Risk</td><td>42</td><td>53.8</td></tr><tr><td>36</td><td>46.2</td></tr><tr><td rowspan="2">Site 4</td><td rowspan="2">Low Risk High Risk Low Risk</td><td>30</td><td>40</td></tr><tr><td>45</td><td>60</td></tr><tr><td rowspan="2">Combined dataset</td><td rowspan="2">High Risk Low Risk</td><td rowspan="2">151</td><td rowspan="2">49.8</td></tr><tr><td>152</td></tr></table>

Comparison of MammaPrint test results from between NGS kit and microarray was performed by the concordance analysis as described above. Results are presented in Table 14 below.

Table 14: Comparison of MammaPrint test results between NGS kit and Microarray   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>MammaPrint FFPEMicroarray (Internal Site)</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>Low Risk</td></tr><tr><td rowspan=3 colspan=1>MammaPrintFFPE NGS(External Sites)</td><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>127</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>303</td></tr><tr><td rowspan=1 colspan=5>Concordance using MPI score of +0.058 as cut-off (i.e., Borderline NGS = High Risk)</td></tr><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=3>99.34% (95% CI: 96.35-99.88)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI)</td><td rowspan=1 colspan=3>82.89% (95% CI: 76.12-88.05)</td></tr><tr><td rowspan=1 colspan=1>OPA (95% CI)</td><td rowspan=1 colspan=3>91.09% (95% CI: 87.35-93.80)</td></tr><tr><td rowspan=1 colspan=5>Concordance using MPI score of -0.058 as cut-off (i.e., Borderline NGS = Low Risk)</td></tr><tr><td rowspan=2 colspan=1>Agreement(Total N=303)</td><td rowspan=1 colspan=1>PPA (95% CI)</td><td rowspan=1 colspan=3>91.39% (95% CI: 85.83-94.90)</td></tr><tr><td rowspan=1 colspan=1>NPA (95% CI)</td><td rowspan=1 colspan=3>93.42% (95% CI: 88.31-96.39)</td></tr></table>

<table><tr><td></td><td>OPA (95% CI)</td><td>92.41% 37-94.89)</td></tr></table>

The analysis showed PPA values ranging from $9 1 . 3 9 \%$ to $9 9 . 3 4 \%$ . NPA values were lower, ranging from $8 2 . 8 9 \%$ to $9 3 . 4 2 \%$ . Borderline samples and samples close to the classification threshold (High Risk/ Borderline) impacted the concordance results.

3. Matrix Comparison: Not Applicable

# C Clinical Studies:

1. Clinical Sensitivity Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

The microarRAy prognoSTics in breast cancER (RASTER) study was conducted to prospectively evaluate the risk of breast cancer distant metastases using a gene-expression prognosis classifier as a risk estimation tool, in addition to clinicopathological factors. In this multicenter observational study, the feasibility of MammaPrint® 70-gene signature developed to predict the risk of breast cancer metastases was assessed in 16 community hospitals in the Netherlands between 2004 and 2006. The primary objective of this multicenter observational study was to assess the feasibility of implementing the 70-gene signature and to study the clinical impact of the 70-gene signature test result on adjuvant systemic therapy (AST) decision making. The secondary objective of the RASTER study was to assess the outcome of patients for whom a gene expression classifier was used to determine the need for adjuvant systemic treatment. A total of 427 patients were enrolled in the RASTER study with age $1 8 -$ 61 years old and had a histologically confirmed unilateral, unifocal, primary operable, invasive adenocarcinoma of the breast (cT1–3N0M0). After a protocol amendment in 2004, the study allowed only patients aged 55 or younger could enroll. The study exclusion criteria were a history of a malignancy (with exception of basal-cell carcinoma or cervical dysplasia) and neoadjuvant systemic treatment. The study assessed the primary endpoint of distantrecurrence free interval (DRFI) defined as a distant breast cancer recurrence or death from breast cancer and the secondary endpoint of breast cancer specific survival (BCSS) defined as mortality related to breast cancer.

Five years follow-up data were published which included the analysis of estimated five-year distant-recurrence free interval (DRFI) and confirmed the prognostic value of the MammaPrint (MP) published in the January 2013 issue of International Journal of Cancer. Survival data was updated in 2017 to determine 10‐years median follow up for the distant‐ recurrence‐free‐interval (DRFI) probabilities.

To demonstrate the clinical validity of the MammaPrint FFPE NGS kit, samples from the prospective observational RASTER study were evaluated. RASTER samples were previously processed between 2004 and 2007 on the MammaPrint Fresh microarray (K101454), for

which a 5-year clinical follow-up was available for the RASTER study for 427 subjects. A subset of FFPE tissue samples from the RASTER study $\scriptstyle ( \ n = 3 4 5 )$ that was previously processed on the MammaPrint FFPE microarray as part of the clearance of the reference predicate device (K141142) was used for the validation of MammaPrint FFPE NGS kit. From the set of 345 samples, total RNA was available for 341 samples and 25 samples failed QC. Therefore, a total of 316 samples were successfully processed using the MammaPrint FFPE NGS kit and this dataset was used for survival analysis.

Clinical validation analysis was performed on the 316 samples that had NGS MammaPrint results based on a 10-year median follow-up of the observational RASTER study. $8 9 . 6 \%$ of the patients have 5-year follow-up data and $53 \%$ of the patients having 10-year follow-up data (median follow-up 10.27 years). The data was analyzed based on the three (3) categories, High Risk, Low Risk, and Borderline, determined by the two cut-offs, -0.058 and $+ 0 . 0 5 8$ . Comparison of clinical performance within the RASTER study between MammaPrint FFPE NGS kit and the predicate device MammaPrint FFPE microarray was evaluated by the Kaplan-Meier graphs and 5-year and 10-year survival percentages. KaplanMeier plots suggest significant difference in survival curves among different risk groups for MammaPrint FFPE NGS kit, $\mathrm { \tt p } { = } 0 . 0 0 1$ .

# Comparison of Kaplan-Meier graphs

![](images/66412559426151d1950d94722b0d9ee81f0ef78291044560ade5e31ce9633b86.jpg)

Results from MammaPrint® FFPE NGS kit and MammaPrint® FFPE microarray were compared for the 316 FFPE samples with 5- and 10-year outcome data from the 427 RASTER patient samples. DRFI was the study endpoint as defined in the RASTER study. Specifically, DRFI measures the time until the diagnosis of distant metastasis or death from breast cancer. Kaplan-Meier curves showed a similar difference in DRFI between the Low and High Risk comparing MammaPrint® FFPE NGS kit and MammaPrint® FFPE microarray.

Table 15: RASTER Study: 5-year and 10-year DRFI Probabilities based on MammaPrint FFPE NGS Risk Categories   

<table><tr><td rowspan=1 colspan=1>Risk</td><td rowspan=1 colspan=1>#Patients</td><td rowspan=1 colspan=1>Events at5 Years</td><td rowspan=1 colspan=1>5-year DRFI(95% CI)</td><td rowspan=1 colspan=1>Events at10 Years</td><td rowspan=1 colspan=1>10-year DRFI (95%CI</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>98.1% (96.0-100)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>95.2% (91.8-98.8)</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.6 % (83.2-100)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>92.6 % (83.2-100)</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>88.2% (82.8-94.0)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>82.0% (75.5-89.2)</td></tr></table>

The Low Risk patients classified by MammaPrint FFPE NGS kit demonstrated a $1 . 9 \%$ $9 5 \%$ CI: $0 { - } 4 . 0 \% )$ and $4 . 8 \%$ ( $9 5 \%$ CI: $1 . 2 \substack { - 8 . 2 \% } )$ chance of cancer recurrence within 5 years and 10 years respectively. Patients classified as High Risk by MammaPrint FFPE NGS kit, demonstrated a $1 1 . 8 \%$ ( $9 5 \%$ CI: $6 . 0 \substack { - 1 7 . 2 \% }$ and $1 8 \%$ ( $9 5 \%$ CI: $1 0 . 8 { - } 2 4 . 5 \% )$ chance of cancer recurrence within 5 years and 10 years respectively. The Borderline samples for both 5 and 10 years had a DRFI of $9 2 . 6 \%$ . The results for the Borderline samples may have low reliability due to the small sample size.

The predicate, MammaPrint FFPE microarray showed $9 8 . 7 \%$ 5-year DRFI for Low Risk and $8 8 . 0 \%$ for High Risk. With the 10-year follow-up, DRFI estimate was $9 6 . 0 \%$ for Low Risk and $8 2 . 8 \%$ for High Risk. Thus, the results indicated that both devices show similar clinical performance based on the updated follow-up from the RASTER study.

The prognostic performance of MammaPrint FFPE NSG Kit was further investigated using Cox proportional hazard regression analysis based on all follow-up data, 5-year follow-up data, and 10-year follow-up data respectively. The evaluated clinicopathological factors include adjuvant treatment status (i.e., whether a patient received or did not receive adjuvant treatment such as chemotherapy (CT) and/or endocrine therapy (ET)), age $( > 5 0 \ \mathrm { v s } \leq 5 0 )$ , tumor size, histological grade, estrogen receptor (ER) status (positive/negative), and human epidermal growth factor receptor 2 (HER2) status (positive/negative).

In both univariate and multivariate analysis, MammaPrint FFPE High/Low Risk result is significantly associated with cancer recurrence based on all three follow up data (i.e., all follow-up, 5-year follow-up, and 10-year follow-up). For the multivariate analysis, MammaPrint FFPE NGS High/Low Risk result is significantly associated with cancer recurrence for the three follow up data with estimated hazard ratios of 7.31 ( $9 5 \%$ CI: 2.50- 21.41), 12.55 ( $9 5 \%$ CI: 2.26-69.74), and 7.08 $9 5 \%$ CI: 2.23-22.49), respectively, after adjusting for all other clinicopathological factors. No significant difference (p value $< 0 . 0 5$ ) was seen for Borderline group vs Low Risk group, which may be due to the small sample size for the Borderline group and the reliability of the results for this group may be low. The result support the prognostic significance for MammaPrint FFPE NGS kit beyond that of other clinicopathological factors.

Reference: Drukker, C. A. et al. A prospective evaluation of a breast cancer prognosis signature in the observational RASTER study. Int. J. Cancer 133, 929–936 (2013).

4. Clinical Cut-Off:

Same as Assay cut-off

5. Expected Values/Reference Range:

<table><tr><td rowspan=1 colspan=1>MammaPrint FFPE NGS kit Result</td><td rowspan=1 colspan=1>Expected Value/Range</td></tr><tr><td rowspan=1 colspan=1>High Risk</td><td rowspan=1 colspan=1>&lt;-0.058</td></tr><tr><td rowspan=1 colspan=1>Low Risk</td><td rowspan=1 colspan=1>&gt; +0.058</td></tr><tr><td rowspan=1 colspan=1>Borderline</td><td rowspan=1 colspan=1>-0.058 ≤ MPI ≤ +0.058</td></tr><tr><td rowspan=1 colspan=1>Reportable Range</td><td rowspan=1 colspan=1>-1.000 to +1.000</td></tr></table>

# VIII Instrument Name:

Illumina MiSeqDx (qualified by Agendia)

# IX System Descriptions:

1. Modes of Operation:

Does the applicant’s device contain the ability to transmit data to a computer, webserver, or mobile device?

Yes ___X_____ or No

Does the applicant’s device transmit data to a computer, webserver, or mobile device using wireless transmission?

Yes ____X____ or No

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for   
this line of product types:   
Yes _ _X_ _ or No

# X Other Supportive Instrument Performance Characteristics Data Not Covered in the “Performance Characteristics” Section above:

Qualification of MiSeqDx instruments and Reagents

Agendia performs qualification of all MiSeqDx instruments that will be used by MammaPrintcertified partners in the testing of patient samples with the MammaPrint FFPE NGS kit as part of the partner certification process. For control of software versions installed on the MiSeqDx instruments, Agendia maintains a list of MiSeqDx Control Software (MCS) versions that are validated for use with the MammaPrint NGS kit and will update the list as new versions of MCS are released by Illumina and validated by Agendia. For updates that are determined to impact the test’s results or performance, validation of the software updates is performed to ensure the instrumentation and software continues to operate within specifications.

Reagents that are required and not provided with the MammaPrint NGS kit are qualified for use through Agendia’s Reagent Qualification procedures. Qualification of RUO reagents involves the testing performed on a specified minimum number of samples where results from previously cleared lots are available and MammaPrint index differences for paired results must be within predefined acceptable limits. The lot numbers of the approved lots of controlled reagents are available in the ADAPT-US system and the MammaPrint-Certified labs are required to select the lot numbers for each of the controlled reagents when analyzing the samples in ADAPT-US.

ADAPT-US platform has adequate control measures that ensure MammaPrint-Certified labs use Agendia-qualified reagent lots, MiSeqDx instruments, and versions of MiSeqDx software and thus by preventing the generation of an incorrect or inaccurate test result.

# XI Proposed Labeling:

The instructions for use for the MammaPrint FFPE NGS kit and ADAPT-US user guide are provided with each test kit.

There are three (3) different test reports, i.e., High Risk, Low Risk, and Borderline.

The labeling is sufficient, and it satisfies the requirements of 21 CFR Parts 801 and 809, as applicable, and the special controls for this device type.

# XII Patient Perspectives:

This submission did not include specific information on patient perspectives for this device.

# XIII Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.

The device is classified as Class II under regulation 21 CFR 862.6040 with special controls. The special control guidance document “Guidance for Industry and FDA Staff – Class II Special Controls Guidance Document: Gene Expression Profiling Test System for Breast Cancer Prognosis” is available at   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocuments/ucm0 79163.htm.