# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

# B Applicant

Progenika Biopharma S.A.

# C Proprietary and Established Names

AlphaID At Home Genetic Health Risk Service

D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>PTA</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.5950 - GeneticHealth Risk Assessment System</td><td rowspan=1 colspan=1>IM -Immunology</td></tr></table>

# II Submission/Device Overview:

A Purpose for Submission: New device

# B Measurand:

14 SERPINA1 gene variants in genomic DNA obtained from a human saliva sample

# C Type of Test:

The AlphaID At Home Genetic Health Risk Service determines if a person has variants associated with a higher risk of developing alpha-1 antitrypsin deficiency (AATD) associated lung or liver disease. The Service is intended to provide the AATD risk report. The report is based on a qualitative genetic test for detecting 14 genetic variants in the serpin peptidase

inhibitor class A member 1 (SERPINA1) gene: $\mathrm { P I } ^ { * } \mathrm { S }$ ; $\mathrm { P I } ^ { * } \mathrm { Z }$ ; $\mathrm { P I } ^ { * } \mathrm { I }$ ; $\mathrm { P I } ^ { * } \mathbf { M }$ procida; $\mathrm { P I } ^ { * } \mathbf { M }$ malton; $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ granite falls; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ west; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ bellingham; $\mathrm { P I ^ { * } F }$ ; $\mathrm { P I } ^ { * } \mathrm { P }$ lowell; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ mattawa; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ clayton, and $\mathrm { P I } ^ { * } \mathbf { M }$ heerlen. The AlphaID At Home Genetic Health Risk Service is for over-the-counter use.

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The AlphaID At Home Genetic Health Risk Service uses qualitative genotyping to detect clinically relevant genetic variants associated with alpha-1 antitrypsin deficiency (AATD) in genomic DNA isolated from human saliva collected from individuals $\geq 1 8$ years with ORAcollect·Dx OCD-100.014 for the purpose of reporting and interpreting Genetic Health Risks (GHR).

This Service is indicated for reporting 14 genetic variants in the SERPINA1 gene: $\mathrm { P I } ^ { * } \mathrm { S }$ ; $\mathrm { P I } ^ { * } \mathrm { Z }$ ; PI\*I; $\mathrm { P I } ^ { * } \mathbf { M }$ procida; $\mathrm { P I } ^ { * } \mathbf { M }$ malton; $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ granite falls; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ west; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ bellingham; $\mathrm { P I ^ { * } F }$ ; $\mathrm { P I } ^ { * } \mathrm { P }$ lowell; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ mattawa; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ clayton, and $\mathrm { P I } ^ { * } \mathbf { M }$ heerlen. The report describes if a person is at an increased or decreased risk of developing either lung and/or liver disease linked to AATD. The report does not describe a person´s overall risk of developing lung and/or liver disease. AATD is more common in persons of European descent.

# C Special Conditions for Use Statement(s):

• For over-the-counter (OTC) use.   
• The test is intended for users $\geq 1 8$ years old.   
• The test is not a substitute for an appointment with a healthcare professional. It is recommended that the user consults with a healthcare professional if the user has any questions or concerns about his/her results.   
• The test does not diagnose a disease or condition, determine medical treatment or other medical intervention, or tell the user anything about their current state of health. Only a healthcare professional can diagnose a disease or condition.   
• Any diagnostic or treatment decisions must be based on confirmatory prescription testing and/or other information that a healthcare professional determines to be appropriate for the patient, such as additional clinical testing and other risk factors that may affect individual risk and health care.   
• The test detects 14 variants in the SERPINA1 gene linked to AATD. These 14 variants explain $9 5 \%$ of AATD cases. The absence of a variant tested does not rule out the presence of other genetic variants that may be disease-related.   
• The test does not describe a person's overall risk of developing AATD. In addition, other   
genetic and all non-genetic factors should be considered   
• The laboratory may be unable to process every user’s sample. The probability that the laboratory cannot process a sample can be up to $0 . 5 \%$ . If this happens, the user will receive an

email notification. The user will also receive another AlphaID At Home Saliva Collection Kit to provide a new sample to the laboratory.

• The user´s race, ethnicity, age, and sex may affect how the genetic results are interpreted. • Subject to meeting limitations contained in the special controls under the regulation 21 CFR 866.5950.

# D Special Instrument Requirements:

The AlphaID At Home Genetic Health Risk Service is to be performed using 384-well Veriti Dx and 96-well Veriti Dx thermal cyclers, and the Luminex 200 instrument.

# IV Device/System Characteristics:

# A Device Description:

The AlphaID At Home Genetic Health Risk Service uses qualitative genotyping to detect clinically relevant genetic variants associated with alpha1-antitrypsin deficiency (AATD) in genomic DNA isolated from human saliva and provides a report describing if a person is at risk of developing either lung and/or liver disease linked to AATD. This Service is direct-toconsumer and intended for an over-the-counter (OTC) use.

The AlphaID At Home Genetic Health Risk Service is composed of:

(i) ORAcollect·Dx OCD-100.014 (cleared under K212745) is intended for use in the noninvasive collection of saliva samples.   
(ii) A1AT Genotyping Test (cleared under K192858) is intended for the genetic analysis and detection of genetic variants associated with alpha-1 antitrypsin deficiency (AATD) using the A1AT Genotyping Test ANALYSIS SOFTWARE, and   
(iii) AlphaID At Home Genetic Health Risk Service website and result portal software are intended to provide the contents and the procedure to order and use the OTC Service.

A consumer’s saliva is self-collected using ORAcollect·Dx OCD-100.014 manufactured by DNA Genotek, Inc which consists of a collection tube containing a stabilizing buffer solution. Once the sample is collected, it is shipped to a Clinical Laboratory Improvement Amendments (CLIA)-certified laboratory for processing.

Human DNA from the saliva sample is isolated, processed and analyzed with the A1AT Genotyping Test that is indicated for reporting 14 genetic variants in the SERPINA1 gene: $\mathrm { P I } ^ { * } \mathrm { S }$ ; $\mathrm { P I } ^ { * } \mathrm { Z }$ ; $\mathrm { P I } ^ { * } \mathrm { I }$ ; $\mathrm { P I } ^ { * } \mathbf { M }$ procida; $\mathrm { P I } ^ { * } \mathbf { M }$ malton; $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ granite falls; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ west; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ bellingham; PI\*F; $\mathrm { P I } ^ { * } \mathrm { P }$ lowell; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ mattawa; $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ clayton, and $\mathrm { P I } ^ { * } \mathbf { M }$ heerlen. Briefly, genomic DNA extracted from human saliva is amplified and biotinylated by multiplex polymerase chain reaction (PCR) and the resulting labeled PCR products are denatured and hybridized to allele-specific oligonucleotide probes coupled to color-coded microspheres. Hybridized DNA is labeled with a fluorescent conjugate and the resulting signal is detected with a Luminex 200 system. Raw fluorescence data is processed with the A1AT Genotyping Test ANALYSIS SOFTWARE to provide allelic variant genotypes, which are subsequently converted into associated alleles. Additionally, the software application also provides the type of

Genetic Health Risk Report associated with the identified alleles, which is subsequently used as the basis for the generation of personalized reports by the AlphaID At Home Genetic Health Risk Service website and result portal.

Depending on the specific variant combination detected, the AlphaID At Home Genetic Health Risk Service provides the individuals’ genetic health risk for developing lung and liver disease linked to AATD. Personalized reports, in an easy-to-understand format, are generated for each consumer that help to understand the meaning of their results and any appropriate actions that may be taken based on their results. In addition, every user before receiving report must undergo the educational module that helps to understand the AlphaID At Home Genetic Health Risk Service results report and explain how genetics can impact a person’s health risk for lung disease and/or liver disease linked to AATD. The AlphaID At Home Genetic Health Risk Service uses four (4) categories summarized in the table below to define risk. One of these risk categories has been assigned to each genetic result based on the information gathered from reported clinical cases for each genetic result. If any of the 14 variants tested by this Service is detected, one of the following four risk categories will be reported:

<table><tr><td rowspan=1 colspan=2>Risk Categories</td></tr><tr><td rowspan=1 colspan=1>Increased risk</td><td rowspan=1 colspan=1>Above 80% of people with this genetic result develop lung orliver disease during their lifetime.</td></tr><tr><td rowspan=1 colspan=1>Slightly increased risk</td><td rowspan=1 colspan=1>20-80% of people with this genetic result develop lung orliver disease during their lifetime.</td></tr><tr><td rowspan=1 colspan=1>Not likely at increased risk</td><td rowspan=1 colspan=1>Below 20% of people with this genetic result develop lung orliver disease during their lifetime.</td></tr><tr><td rowspan=1 colspan=1>Unknown risk</td><td rowspan=1 colspan=1>More clinical studies are needed to determine the risk levelassociated with this genetic result</td></tr></table>

When no variants are detected, the report shows ‘Not Likely at Risk’ for AATD. When the service is not able to provide a genetic result, the associated lung and liver disease risk categories cannot be determined, and another type of report will be generated: ‘Variant not determined’.

Based on the number of variants detected (0, 1, 2, or variant not determined) and specific lung and liver disease risk categorization reported, a total of 15 types of reports can be generated by the AlphaID At Home Genetic Health Risk Service, which are shown in the table below:

<table><tr><td colspan="1" rowspan="1">TypeofReport</td><td colspan="1" rowspan="1">Numberofvariants</td><td colspan="1" rowspan="1">Reported Lung DiseaseRisk Category</td><td colspan="1" rowspan="1">Reported Liver DiseaseRisk Category</td><td colspan="1" rowspan="1">Numberof geneticresults</td><td colspan="1" rowspan="1">Frequencyof geneticresults, %</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Not likely at risk for AATD</td><td colspan="1" rowspan="1">Not likely at risk for AATD</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">94.95</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4.23</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.002</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.002</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk(never-smokers)Slightly increased risk(ever-smokers)</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.780</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.40</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">&lt;0.00001</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0.017</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">&lt;0.00001</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">&lt;0.00001</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0.0016</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0.0001</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0.0001</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0.0069</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Variant notdetermined</td><td colspan="1" rowspan="1">Not determined risk</td><td colspan="1" rowspan="1">Not determined risk</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

# B Principle of Operation:

The AlphaID At Home Genetic Health Risk Service is performed by a CLIA-certified laboratory using the Alpha-1 antitrypsin (A1AT) Genotyping Test.

The A1AT Genotyping Test utilizes Luminex xMAP technology. Genomic DNA is extracted from human saliva samples collected as buccal swabs using ORAcollect·Dx OCD-100.014. Extracted DNA is amplified and biotinylated by multiplex PCR and the PCR products are denatured and hybridized to oligonucleotide probes coupled to color-coded beads. Hybridized DNA is labeled with fluorescent conjugate and the resulting signal is detected with a Luminex 200 system (with xPONENT software). Raw data obtained is processed with the A1AT Genotyping Test Analysis Software.

The A1AT Genotyping Test Analysis Software algorithm converts the allelic variant genotypes into associated alleles. Additionally, this software application also provides an associated template number for each sample, this is, a code that defines the type of Genetic Health Risk Report that is to be generated for each individual, depending on its combination of variants. This template number will be subsequently used as the basis for the generation of personalized reports by the AlphaID System.

# V Substantial Equivalence Information:

A Predicate Device Name(s):

23andMe Personal Genome Service (PGS) Genetic Health Risk Report for Alpha-1 Antitrypsin Deficiency

B Predicate 510(k) Number(s):

DEN160026

C Comparison with Predicate(s):

<table><tr><td colspan="1" rowspan="1">Device &amp;PredicateDevice(s):</td><td colspan="1" rowspan="1">DeviceK221420</td><td colspan="1" rowspan="1">PredicateDEN160026</td></tr><tr><td colspan="1" rowspan="1">Device TradeName</td><td colspan="1" rowspan="1">AlphaID At Home Genetic Health RiskService</td><td colspan="1" rowspan="1">23andMe Personal Genome Service(PGS) for A1AT</td></tr><tr><td colspan="3" rowspan="1">General Device Characteristic Similarities</td></tr><tr><td colspan="1" rowspan="1">Intended Use/Indications ForUse</td><td colspan="1" rowspan="1">The AlphaID At Home Genetic HealthRisk Service uses qualitativegenotyping to detect clinically relevantgenetic variants associated with alpha-1antitrypsin deficiency (AATD) ingenomic DNA isolated from humansaliva collected from individuals ≥ 18years with ORAcollect·Dx OCD-100.014 for the purpose of reporting andinterpreting Genetic Health Risks(GHR).The AlphaID At Home Genetic HealthRisk Service is indicated for reporting14 genetic variants in the SERPINA1gene: PI*S; PI*Z; PI*I; PI*M procida;PI*M malton; PI*S iiyama; PI*Q0granite falls; PI*Q0 west; PI*Q0bellingham; PI*F; PI*P lowell; PI*Q0mattawa; PI*Q0 clayton, and PI*Mheerlen. The report describes if a personis at an increased or decreased risk ofdeveloping either lung and/or liverdisease linked to AATD. The reportdoes not describe a person's overall riskof developing lung and/or liver disease.AATD is more common in persons ofEuropean descent.</td><td colspan="1" rowspan="1">The 23andMe Personal GenomeService (PGS) Test usesqualitative genotyping to detect thefollowing clinically relevant variantsin genomic DNA isolated fromhuman saliva collected fromindividuals ≥18 years with theOragene Dx model OGD-500.001 forthe purpose of reporting andinterpreting Genetic Health Risks(GHR).The 23andMe PGS Genetic HealthRisk Report for Alpha-1 AntitrypsinDeficiency is indicated for reportingof the PI*Z and PI*S variants in theSERPINA1 gene. This reportdescribes if a person has variantsassociated with AAT deficiency anda higher risk for lung or liver disease,but it does not describe a person'soverall risk of developing lung orliver disease. This test is mostrelevant for people of Europeandescent.</td></tr><tr><td colspan="1" rowspan="1">SpecialConditions forUseStatements</td><td colspan="1" rowspan="1">a. For over-the-counter (OTC) use.b. The test is intended for users ≥18years old.c. The test is not a substitute for anappointment with a healthcareprofessional. It is recommended thatthe user consult with a healthcareprofessional if the user have anyquestions or concerns about his/herresults.d. The test does not diagnose a diseaseor condition, determine medicaltreatment or other medicalintervention, or tell the user anything</td><td colspan="1" rowspan="1">a. For over-the-counter (OTC) use.b. This test is not a substitute forvisits to a healthcare provider. It isrecommended that you consultwith a healthcare provider if youhave any questions or concernsabout your results.c. The 23andMe PGS Genetic HealthRisk Tests for HereditaryThrombophilia, Alpha-1Antitrypsin Deficiency,Alzheimer's disease, Parkinson'sdisease, Gaucher Disease, FactorXI Deficiency, Celiac disease, and</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">about their current state of health.Only a healthcare professional candiagnose a disease or condition.e. Any diagnostic or treatment decisionsmust be based on confirmatoryprescription testing and/or otherinformation that a healthcareprofessional determines to beappropriate for the patient, such asadditional clinical testing and otherrisk factors that may affect individualrisk and health care.f. The test detects 14 variants in theSERPINA1 gene linked to AATD.These 14 variants explain 95% ofAATD cases. The absence of avariant tested does not rule out thepresence of other genetic variants thatmay be disease-related.g. The test does not describe a person'soverall risk of developing AATD. Inaddition, other genetic and all non-genetic factors should be consideredh. The laboratory may be unable toprocess every user's sample. Theprobability that the laboratory cannotprocess a sample can be up to 0.5%.If this happens, the user will receivean email notification. The user willalso receive another AlphaID AtHome Saliva Collection Kit toprovide a new sample to thelaboratory.i. The user's race, ethnicity, age, andsex may affect how the geneticresults are interpreted.j. Subject to meeting limitationscontained in the special controlsunder the regulation 21 CFR866.5950</td><td colspan="1" rowspan="1">Glucose-6- Phosphate-Dehydrogenase Deficiency, Early-Onset Primary Dystonia andHereditary Hemochromatosis donot detect all genetic variantsassociated with theaforementioned diseases. Theabsence of a variant tested doesnot rule out the presence of othergenetic variants that may bedisease related.d. The test is intended for users ≥ 18years old.e. The test does not diagnose anyspecific health conditions. Resultsshould not be used to makemedical decisions.f. The laboratory may not be able toprocess a user's sample. Theprobability that the laboratorycannot process a sample can be upto 7.6%.g. A user's race, ethnicity, age, andsex may affect how the genetic testresults are interpreted.h. Subject to meeting the limitationscontained in the special controlsunder regulation 21 CFR866.5950.</td></tr><tr><td colspan="1" rowspan="1">Type of Test</td><td colspan="1" rowspan="1">Qualitative genotyping for allelicvariant determination throughsingle nucleotide polymorphismdetection</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Design</td><td colspan="1" rowspan="1">Software application that includesproduct information page, e-commerce(registration and order DNA kit), securelogin, download genetic report</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Interpretationof results</td><td colspan="1" rowspan="1">For over-the-counter use (OTC).Specialized interpretation by aphysician not required</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Saliva</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Measurand</td><td colspan="1" rowspan="1">Genomic DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Comparisonwith SangerBidirectionalSequencing</td><td colspan="1" rowspan="1">Overall agreement for the 14 variantswas 100%.</td><td colspan="1" rowspan="1">Overall agreement for the twovariants was 100%.</td></tr><tr><td colspan="1" rowspan="1">Human factors</td><td colspan="1" rowspan="1">User Comprehension Study</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">General Device Characteristic Differences</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">SpecimenCollection Kit</td><td colspan="1" rowspan="1">DNA Genotek Inc., ORAcollect DxOCD-100.014</td><td colspan="1" rowspan="1">DNA Genotek Inc., Oragene Dxmodel OGD-500.001</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Polymerase Chain Reaction (PCR)amplification and hybridization basedLuminex technology</td><td colspan="1" rowspan="1">Multiplex customized genotypingchip (BeadChip v4 assay)</td></tr><tr><td colspan="1" rowspan="1">Method andSequencingPlatform</td><td colspan="1" rowspan="1">Multiplex polymerase chain reaction(PCR) and multiplex Allele SpecificPrimer Extension (ASPE) withLuminex's Universal Tag sortingsystem on the Luminex 200 xMAPplatform with xPONENT software andA1AT Genotyping Test Analysissoftware for the simultaneous detectionand identification of allelic variants andtheir associated alleles.</td><td colspan="1" rowspan="1">Magnetic bead DNA extraction isperformed using Tecan Evo.Genotyping and PCR analysis isconducted using a chip-based methodwith Illumina Infinium's BeadChipv4 assay, the Illumina iScan Systemfor detection and analysis isfacilitated with the use ofGenomeStudio and Coregensoftware.</td></tr><tr><td colspan="1" rowspan="1">SpecialInstrumentRequirements</td><td colspan="1" rowspan="1">Luminex 200 instrument (withxPONENT® software)The A1AT Genotyping TestANALYSIS SOFTWARE processesraw data from the Luminex System(csv. files containing the MFI value foreach bead type) to provide allelicvariant genotypes which aresubsequently converted into associatedalleles, and the type of Genetic HealthRisk Report associated with theidentified alleles</td><td colspan="1" rowspan="1">Tecan Evo and Illumina iScaninstrumentsGenomeStudio is a modular softwareapplication that is used to view andanalyze genotypic data obtained fromthe iScan.Coregen software conducts a varietyof control checks on thefile, resulting in a final genotypeprofile for each sample. Thesedata are used to generate test reportson a user's genotypeand associated risk of disease.</td></tr><tr><td colspan="1" rowspan="1">Allelic VariantsDetected</td><td colspan="1" rowspan="1">14 allelic variants in the SERPINA1gene: PI*I; PI*M procida; PI*Mmalton; PI*S iiyama; PI*Q0 granitefalls; PI*Q0 west; PI*Q0 bellingham;PI*F; PI*P lowell; PI*Q0 mattawa;</td><td colspan="1" rowspan="1">Two allelic variants in theSERPINA1 gene: P*Z and P*S</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PI*Q0 clayton, PI*M heerlen).</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AnalyticalSensitivity</td><td colspan="1" rowspan="1">The performance requirement for theA1AT Genotyping Test has been set at aminimum of 0.0215 ng/μL DNA.</td><td colspan="1" rowspan="1">The performance requirement for thePGS has been set at a minimum of 15ng/μL DNA and maximum of 50ng/μL DNA.</td></tr></table>

# VI Standards/Guidance Documents Referenced:

CLSI guideline EP07-A2, “Interference Testing in Clinical Chemistry; Approved Guideline– Second Edition”   
• Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices (2005) General Principles of Software Validation Guidance for Industry and FDA Staff (2002) Off-The-Shelf Software Use in Medical Devices Guidance for Industry and Food and Drug Administration Staff (2019)   
• Content of Premarket Submissions for Management of Cybersecurity in Medical Devices (2014)   
• ISO 14971 Third Edition 2019-12 Medical devices - Application of risk management to medical devices

# VII Performance Characteristics (if/when applicable):

# A Analytical Performance:

All results presented below met the sponsor’s pre-defined acceptance criteria outlined in the Special Controls of 21 CFR 866.5950. Information regarding samples that failed quality control (FQC) was also evaluated and presented in each study below.

# 1. Precision/Reproducibility:

Refer to K171868 for results of the reproducibility study that was conducted with a total of 17 samples. The reproducibility of the assay was evaluated at three sites, with two operators at each site performing one run per day across three non-consecutive days (three runs per operator or six runs per site). Within a given run, the sample panel was tested in duplicate. Concordance with the expected results was $100 \%$ at the three sites.

Refer to K192858 for DNA extraction variability in saliva samples that was conducted using three different extraction methods: QIAamp DNA Blood Mini Kit (Qiagen), Commercial lysis and neutralization solutions (Sigma) and QIAsymphony DNA Mini Kit (Qiagen).

Refer to K171868 and K211115 for results of the lot-to-lot imprecision study.

2. Linearity: Not applicable

3. Analytical Specificity/Interference:

i. Interference

Refer to K192858 for results of studies conducted to evaluate the potential interference of substances usually found in saliva samples.

ii. Cross-reactivity

Refer to K171868. An In Silico DNA sequence analysis of A1AT Genotyping Test primers and probes confirmed sequence specificity and the absence of cross-reactivity.

iii. Cross-contamination Refer to K171868

4. Assay Reportable Range: Not applicable

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

i. Reagent stability:

Refer to K192858. The claimed reagent stability is 24 months when stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and up to 9 months after the kit vials were first opened.

ii. Stability of collection device and specimens:

Saliva samples for testing are collected with the ORAcollect·Dx OCD-100.014 (customized version of ORAcollect·Dx OCD-100). Refer to K152464 for pre-collection shelf-life stability of the collection device, stability of samples post-saliva collection, and freeze-thaw stability of samples stored in the Oragene Dx collection device.

6. Detection Limit:

Refer to K211115 for results of the Lower Limit of Detection (LLoD) study that evaluated the impact of different levels of DNA input on test performance.

7. Assay Cut-Off: Not applicable

# B Comparison Studies:

1. Comparison with Sanger Bidirectional Sequencing:

A method comparison study was performed with saliva samples collected in ORAcollect·Dx OCD-100.014 (customized version of ORAcollect·Dx OCD-100) to assess the accuracy of the AlphaID At Home Genetic Health Risk Service to correctly detect the genetic variants. A total of 227 samples representing all genetic variants interrogated by the assay were analyzed and compared with bi-directional Sanger sequencing (reference method).

The percent agreements (PAs) per allelic variant from clinical samples and contrived samples (8 synthetic DNAs samples) were calculated separately. The PA per allelic variant between the two methods was $100 \%$ for both clinical samples and contrived samples. The percentage of “No call’ or “Invalid test” per allelic variant was also calculated and was $0 \%$ for both clinical samples and contrived samples.

The PAs of AlphaID At Home Genetic Health Risk Service genotype results with comparator results are summarized in the tables below.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="4" rowspan="1">PT"T (c.T87C&gt;1) Varlant ol the SERrINAT gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger Bidirectional Sequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (203/203) with 95% CI: 98.1%−100%PA (C/T |C/T) = 100% (15/15) with 95% CI: 79.6%−100%PA (T/T | T/T) = 100% (1/1) with 95% CI: 20.7%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger Bidirectional Sequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (4/4) with 95% CI: 51.0%−100%PA (C/T | C/T) = 100% (1/1) with 95% CI: 20.7%-100%PA (T/T | T/T) = 100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0% 2.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger Bidirectional Sequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">207</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">207</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="5" rowspan="1">PA (C/C C/C) = 100% (207/207) with 95% CI: 98.2%-100%PA (C/T | C/T) = 100% (16/16) with 95% CI: 80.6%−100%PA (T/T | T/T) = 100% (4/4) with 95% CI: 51.0%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%.</td></tr><tr><td>Minor allele frequency for PI*I SERPINA1 in the 1000 Genomes Project was 0.06%; technical (analytical) positive predictive values for AlphaID At Home Genetic Health Risk Service test result of C/T and T/T are 99.128% and 99.997% correspondingly</td></tr></table>

<table><tr><td rowspan=1 colspan=5>PI*M procida variant (c.194T&gt;C) of the SERPINA1 gene</td></tr><tr><td rowspan=2 colspan=1>ClinicalSamples</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=1 colspan=3>Sanger Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>T/T</td><td rowspan=1 colspan=1>T/C</td><td rowspan=1 colspan=1>C/C</td></tr><tr><td rowspan=4 colspan=1>AlphaID At HomeGenetic HealthRisk Service</td><td rowspan=1 colspan=1>C/C (homozygous common)</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>C/T (heterozygous)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>T/T (homozygous rare)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>&#x27;No call&#x27; or &#x27;Invalid test&#x27;</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total=219</td><td rowspan=1 colspan=1>211</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=5>PA (C/C | C/C) = 100% (211/211) with 95% CI: 98.2%-100%PA (C/T | C/T) = 100% (5/5) with 95% CI: 56.6%−100%PA (T/T | T/T) = 100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td rowspan=2 colspan=1>ContrivedSamples</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=1 colspan=3>Sanger Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>C/C</td><td rowspan=1 colspan=1>C/T</td><td rowspan=1 colspan=1>T/T</td></tr><tr><td rowspan=4 colspan=1>AlphaID At HomeGenetic HealthRisk Service</td><td rowspan=1 colspan=1>C/C (homozygous common)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>C/T (heterozygous)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>T/T (homozygous rare)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>&#x27;No call&#x27; or &#x27;Invalid test&#x27;</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total=8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=5>PA (C/C | C/C) = 100% (4/4) with 95% CI: 51.0%−100%PA (C/T | C/T) = 100% (1/1) with 95% CI: 20.7%−100%PA (T/T | T/T) =100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%-32.4%</td></tr><tr><td rowspan=2 colspan=1>AllSamples</td><td rowspan=2 colspan=1>Genotype</td><td rowspan=1 colspan=3>Sanger Bidirectional Sequencing</td></tr><tr><td rowspan=1 colspan=1>C/C</td><td rowspan=1 colspan=1>C/T</td><td rowspan=1 colspan=1>T/T</td></tr><tr><td rowspan=4 colspan=1>AlphaID At HomeGenetic HealthRisk Service</td><td rowspan=1 colspan=1>C/C (homozygous common)</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>C/T (heterozygous)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>T/T (homozygous rare)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>&#x27;No call&#x27; or &#x27;Invalid test&#x27;</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total=227</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=5>PA (C/C | C/C) = 100% (215/215) with 95% CI: 98.2%-100%PA (C/T | C/T) = 100% (6/6) with 95% CI: 61.0%-100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*M procida SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphalD At Home GeneticHealth Risk Service test result of C/T and T/T are 97.603% and 99.999% correspondingly</td></tr></table>

<table><tr><td colspan="2" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">TTC/TTC</td><td colspan="1" rowspan="1">TTC/delTTC</td><td colspan="1" rowspan="1">delTTC/delTTC</td></tr><tr><td colspan="2" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">TTC/TTC (homozygous common)</td><td colspan="1" rowspan="1">195</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TTC/ delTTC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">delTTC/ delTTC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=219</td><td colspan="1" rowspan="1">195</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="6" rowspan="1">PA (TTC/TTC | TTC/TTC) = 100% (195/195) with 95% CI: 98.1%-100%PA (TTC/delTTC | TTC/delTTC) = 100% (21/21) with 95% CI: 84.5%−100%PA (delTTC/delTTC | delTTC/delTTC) = 100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="2" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">TTC/TTC</td><td colspan="1" rowspan="1">TTC/delTTC</td><td colspan="1" rowspan="1">delTTC/delTTC</td></tr><tr><td colspan="2" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">TTC/TTC (homozygous common)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">ic</td><td colspan="1" rowspan="1">TTC/delTTC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">delTTC/delTTC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="6" rowspan="1">PA (TTC/TTC | TTC/TTC) = 100% (4/4) with 95% CI: 51.0%-100%PA (TTC/delTTC | TTC/delTTC) = 100% (1/1) with 95% CI: 20.7%−100%PA (delTTC/delTTC | delTTC/delTTC) =100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="2" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">TTC/TTC</td><td colspan="1" rowspan="1">TTC/delTTC</td><td colspan="1" rowspan="1">delTTC/delTTC</td></tr><tr><td colspan="2" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">TTC/TTC (homozygous common)</td><td colspan="1" rowspan="1">199</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TTC/ delTTC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">delTTC/ delTTC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=227</td><td colspan="1" rowspan="1">199</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="6" rowspan="1">PA (TTC/TTC | TTC/TTC) = 100% (199/199) with 95% CI: 98.1%-100%PA (TTC/delTTC | TTC/delTTC) = 100% (22/22) with 95% CI: 85.1%−100%PA (delTTC/delTTC | delTTC/delTTC) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*M malton SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphaID At Home GeneticHealth Risk Service test result of TTC/delTTC and delTTC/delTTC are 99.382% and99.996% correspondingly.</td></tr><tr><td colspan="6" rowspan="1">PI* S iiyama variant (c.230C&gt;T) of the SERPINA1 gene</td></tr><tr><td colspan="2" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="2" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">k</td><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=219</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="6" rowspan="1">PA (C/C | C/C) = 100% (217/217) with 95% CI: 98.3%-100%PA (C/T | C/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr></table>

<table><tr><td colspan="1" rowspan="2">Health RiskService</td><td colspan="1" rowspan="1">T7T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (217/217) with 95% CI: 98.3%-100%PA (C/T | C/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%−1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (4/4) with 95% CI: 51.0%−100%PA (C/T | C/T) = 100% (1/1) with 95% CI: 20.7%−100%PA (T/T | T/T) = 100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (221/221) with 95% CI: 98.3%−100%PA (C/T | C/T) = 100% (3/3) with 95% CI: 43.9%−100%PA (T/T | T/T) = 100% (3/3) with 95% CI: 43.9%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*S iiyama SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphaID At Home GeneticHealth Risk Service test result of C/T and T/T are 95.257% and 99.9995%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*Q0 granite falls (c.552delC) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/delC</td><td colspan="1" rowspan="1">delC/delC</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">216</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/ delC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">delC/delC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">216</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (216/216) with 95% CI: 98.3%-100%PA (C/delC | C/delC) = 100% (3/3) with 95% CI: 43.9%−100%PA (delC/delC | delC/delC) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/delC</td><td colspan="1" rowspan="1">delC/delC</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/delC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">delC/delC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = N/APA (C/delC | C/delC) = 100% (2/2) with 95% CI: 34.2%−100%PA (delC/delC | delC/delC) =100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/delC</td><td colspan="1" rowspan="1">delC/delC</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">216</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/ delC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">delC/delC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">216</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (216/216) with 95% CI: 98.3%−100%PA (C/delC | C/delC) = 100% (5/5) with 95% CI: 56.6%−100%PA (delC/delC | delC/delC) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%−1.7%Minor allele frequency for PI*Q0 granite falls SERPINA1 in the 1000 Genomes Projectwas &lt;0.001%; technical (analytical) positive predictive values for AlphaID At HomeGenetic Health Risk Service test result of C/delC and delC/delC are 97.124 and 99.999%correspondingly.</td></tr><tr><td colspan="6" rowspan="1">PI*Q0 west (646+1G&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="2" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="2" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">k</td><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=219</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="6" rowspan="1">PA (G/G | G/G) = 100% (215/215) with 95% CI: 98.2%-100%PA (G/T | G/T) = 100% (4/4) with 95% CI: 51.0%−100%PA (T/T | T/T) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr></table>

<table><tr><td colspan="1" rowspan="3">Health RiskService</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">PA (G/G | G/G) = 100% (215/215) with 95% CI: 98.2%−100%PA (G/T | G/T) = 100% (4/4) with 95% CI: 51.0%-100%PA (T/T | T/T) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (G/G | G/G) = N/APA (G/T | G/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) =100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (G/G | G/G) = 100% (215/215) with 95% CI: 98.2%-100%PA (G/T | G/T) = 100% (6/6) with 95% CI: 61.0%−100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 61.0% - 100%.Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%.Minor allele frequency for PI*Q0 west SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphaID At Home GeneticHealth Risk Service test result of G/T and T/T are 97.603% and 99.999%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*Q0 bellingham (c.721A&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">213</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">213</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (213/213) with 95% CI: 98.2%−100%PA (A/T | A/T) = 100% (5/5) with 95% CI: 56.6%−100%PA (T/T | T/T) = 100% (1/1) with 95% CI: 94.7%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%−1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = N/APA (A/T | A/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">213</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test"</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">213</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (213/213) with 95% CI: 98.2%-100%PA (A/T | A/T) = 100% (7/7) with 95% CI: 64.6%−100%PA (T/T | T/T) = 100% (7/7) with 95% CI: 64.6%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%−1.7%.Minor allele frequency for PI*Q0 bellingham SERPINA1 in the 1000 Genomes Projectwas &lt;0.001%; technical (analytical) positive predictive values for AlphalD At HomeGenetic Health Risk Service test result of A/T and T/T are 97.947% and 99.999%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*F (c.739C&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (194/194) with 95% CI: 98.1%-100%PA (C/T | C/T) = 100% (23/23) with 95% CI: 85.7%−100%PA (T/T | T/T) = 100% (2/2) with 95% CI: 34.2%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = N/APA (C/T | C/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (194/194) with 95% CI: 98.1%-100%PA (C/T | C/T) = 100% (25/25) with 95% CI: 86.7%−100%PA (T/T | T/T) = 100% (8/8) with 95% CI: 67.6%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*F SERPINA1 in the 1000 Genomes Project was 0.1%;technical (analytical) positive predictive values for AlphalD At Home Genetic HealthRisk Service test result of C/T and T/T are 99.464% and 99.996% correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*P lowell (c.839A&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">201</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">201</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (201/201) with 95% CI: 98.1%-100%PA (A/T | A/T) = 100% (14/14) with 95% CI: 78.5%−100%PA (T/T | T/T) = 100% (4/4) with 95% CI: 51.0%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = N/APA (A/T | A/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) =100% (6/6) with 95% CI: 61.0%-100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%.</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">201</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">TT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">201</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (201/201) with 95% CI: 98.1%-100%PA (A/T | A/T) = 100% (16/16) with 95% CI: 80.6%−100%PA (T/T | T/T) = 100% (10/10) with 95% CI: 72.2%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*P lowell SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphaID At Home GeneticHealth Risk Service test result of A/T and T/T are 99.464% and 99.996%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*S (c.863A&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (163/163) with 95% CI: 97.7%-100%PA (A/T | A/T) = 100% (36/36) with 95% CI: 90.4%−100%PA (T/T | T/T) = 100% (20/20) with 95% CI: 83.9%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = N/APA (A/T | A/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">A/A</td><td colspan="1" rowspan="1">A/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">A/A (homozygous common)</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">26</td></tr><tr><td colspan="5" rowspan="1">PA (A/A | A/A) = 100% (163/163) with 95% CI: 97.7%-100%PA (A/T | A/T) = 100% (38/38) with 95% CI: 90.8%−100%PA (T/T | T/T) = 100% (26/26) with 95% CI: 87.1%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%.Minor allele frequency for PI*S SERPINA1 in the 1000 Genomes Project was 2%;technical (analytical) positive predictive values for AlphalD At Home Genetic HealthRisk Service test result of A/T and T/T are 99.669% and 99.992% correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*Z (c.1096G&gt;A) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/A</td><td colspan="1" rowspan="1">A/A</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/A (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/A (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="5" rowspan="1">PA (G/G | GG) = 100% (167/167) with 95% CI: 97.8%-100%PA (G/A | G/A) = 100% (37/37) with 95% CI: 90.6%-100%PA (A/A | A/A) = 100% (15/15) with 95% CI: 79.6%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/A</td><td colspan="1" rowspan="1">A/A</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/A (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/A (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (G/G | G/G) = N/APA (G/A | G/A) = 100% (2/2) with 95% CI: 34.2%−100%PA (A/A | A/A) = 100% (6/6) with 95% CI: 61.0%-100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%.</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">G/G</td><td colspan="1" rowspan="1">G/A</td><td colspan="1" rowspan="1">A/A</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">G/G (homozygous common)</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">G/A (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">A/A (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">167</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="5" rowspan="1">PA(G/G | GG) = 100% (167/167) with 95% CI: 97.8%-100%PA (G/A | G/A) = 100% (39/39) with 95% CI: 91.0%−100%PA (A/A | A/A) = 100% (21/21) with 95% CI: 84.5%-100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%.Minor allele frequency for PI*Z SERPINA1 in the 1000 Genomes Project was 0.4%;technical (analytical) positive predictive values for AlphalD At Home Genetic HealthRisk Service test result of G/A and A/A are 99.680% and 99.992% correspondingly</td></tr><tr><td colspan="6" rowspan="1">PI*Q0 mattawa (c.1130dupT) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">ClinicalSamples</td><td colspan="2" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">T/dupT</td><td colspan="1" rowspan="1">dupT/dupT</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="2" rowspan="1">T/T (homozygous common)</td><td colspan="1" rowspan="1">212</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">T/dupT (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">dupT/dupT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="2" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=219</td><td colspan="1" rowspan="1">212</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="6" rowspan="1">PA (T/T | T/T) = 100% (212/212) with 95% CI: 98.2%−100%PA (T/dupT | T/dupT) = 100% (5/5) with 95% CI: 56.6%−100%PA (dupT/dupT | dupT/dupT) = 100% (2/2) with 95% CI: 34.2%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="2" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">T/dupT</td><td colspan="1" rowspan="1">dupT/dupT</td></tr><tr><td colspan="1" rowspan="4">AlphaID AtHome GeneticHealth RiskService</td><td colspan="2" rowspan="1">T/T (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">T/dupT (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">dupT/dupT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="2" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="6" rowspan="1">PA (T/T | T/T) = N/APA (T/dupT | T/dupT) = 100% (2/2) with 95% CI: 34.2%−100%PA (dupT/dupT | dupT/dupT) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="2" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">T/T</td><td colspan="1" rowspan="1">T/dupT</td><td colspan="1" rowspan="1">dupT/dupT</td></tr><tr><td colspan="2" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">T/T (homozygous common)</td><td colspan="1" rowspan="1">212</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/dupT (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">dupT/dupT (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="3" rowspan="1">Total=227</td><td colspan="1" rowspan="1">212</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="6" rowspan="1">PA (T/T | T/T) = 100% (212/212) with 95% CI: 98.2%−100%PA (T/dupT | T/dupT) = 100% (7/7) with 95% CI: 64.6%−100%PA (dupT/dupT | dupT/dupT) = 100% (8/8) with 95% CI: 67.6%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*Q0 mattawa SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphalD At Home GeneticHealth Risk Service test result of T/dupT and dupT/dupT are 97.947% and 99.999%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*Q0 clayton (c.1158dupC) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">Clinical Samples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">CC</td><td colspan="1" rowspan="1">C/dupC</td><td colspan="1" rowspan="1">dupC/dupC</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/dupC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">dupC/dupC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (217/217) with 95% CI: 98.3%−100%PA (C/dupC | C/dupC) = 100% (2/2) with 95% CI: 34.2%−100%PA (dupC/dupC | dupC/dupC) = N/APercent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">CC</td><td colspan="1" rowspan="1">C/dupC</td><td colspan="1" rowspan="1">dupC/dupC</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/dupC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">dupC/dupC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test"</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = N/APA (C/dupC | C/dupC) = 100% (2/2) with 95% CI: 34.2%−100%PA (dupC/dupC | dupC/dupC) =100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">CC</td><td colspan="1" rowspan="1">C/dupC</td><td colspan="1" rowspan="1">dupC/dupC</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/dupC (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">dupC/dupC (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">217</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (217/217) with 95% CI: 98.3%-100%PA (C/dupC | C/dupC) = 100% (4/4) with 95% CI: 51.0%−100%PA (dupC/dupC | dupC/dupC) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*Q0 clayton SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphalD At Home GeneticHealth Risk Service test result of C/dupC and dupC/dupC are 96.415% and 99.999%correspondingly.</td></tr><tr><td colspan="5" rowspan="1">PI*M heerlen (c.1178C&gt;T) variant of the SERPINA1 gene</td></tr><tr><td colspan="1" rowspan="2">Clinical Samples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=219</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (210/210) with 95% CI: 98.2%−100%PA (C/T | C/T) = 100% (5/5) with 95% CI: 56.6%−100%PA (T/T | T/T) = 100% (4/4) with 95% CI: 51.0%−100%Percent of no calls or invalid is 0.0% (0/219) with 95% CI: 0.0%1.7%</td></tr><tr><td colspan="1" rowspan="2">ContrivedSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">'no calls' or 'invalid"</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = N/APA (C/T | C/T) = 100% (2/2) with 95% CI: 34.2%−100%PA (T/T | T/T) = 100% (6/6) with 95% CI: 61.0%−100%Percent of no calls or invalid is 0.0% (0/8) with 95% CI: 0.0%32.4%</td></tr><tr><td colspan="1" rowspan="2">AllSamples</td><td colspan="1" rowspan="2">Genotype</td><td colspan="3" rowspan="1">Sanger BidirectionalSequencing</td></tr><tr><td colspan="1" rowspan="1">C/C</td><td colspan="1" rowspan="1">C/T</td><td colspan="1" rowspan="1">T/T</td></tr><tr><td colspan="1" rowspan="4">AlphaID At HomeGenetic HealthRisk Service</td><td colspan="1" rowspan="1">C/C (homozygous common)</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">C/T (heterozygous)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">T/T (homozygous rare)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">'No call' or 'Invalid test'</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">Total=227</td><td colspan="1" rowspan="1">210</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="5" rowspan="1">PA (C/C | C/C) = 100% (210/210) with 95% CI: 98.2%-100%PA (C/T | C/T) = 100% (7/7) with 95% CI: 64.6%−100%PA (T/T | T/T) = 100% (10/10) with 95% CI: 72.2%−100%Percent of no calls or invalid is 0.0% (0/227) with 95% CI: 0.0%1.7%Minor allele frequency for PI*M heerlen SERPINA1 in the 1000 Genomes Project was&lt;0.001%; technical (analytical) positive predictive values for AlphaID At Home GeneticHealth Risk Service test result of C/T and T/T are 97.947% and 99.999%correspondingly.</td></tr></table>

2. Matrix Comparison: Not applicable

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable

2. Clinical Specificity: Not applicable

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

a) Risk of developing lung and liver disease associated with AATD depends on specific variant combination detected

AATD is a genetic autosomal recessive disease with codominant expression (i.e., each of the two alleles present in each person contributes $50 \%$ on the total generated AAT protein), characterized by low levels of the AAT protein in the blood, that may lead to various health conditions. The most common diseases are lung and/or liver disease. AATD occurs in people of all ethnicities worldwide. 1 However, it is most common in people of European descent. AATD affects about 1 in 1,500 to 3,500 people of European descent. 2,3 Many individuals with AATD are likely undiagnosed, particularly people with a lung condition called chronic obstructive pulmonary disease (COPD). COPD can be caused by AATD; however, AATD is rarely diagnosed. Some people with AATD are misdiagnosed with asthma. 4 Two to three percent of patients with COPD in the United States are estimated to have AATD. 2 The percentage of patients with liver disease who have AATD is not known.

AATD is caused by modifications in the genetic sequence of the SERPINA1 gene, which give rise to different variants. More than 120 variants in the SERPINA1 gene have been identified. Some of these variants do not affect the production of AAT protein, but some others may differently affect the levels or functionality of the AAT protein and, therefore, each specific variant may be associated with different probabilities for the development of lung disease and/or liver disease.

The AlphaID At Home Genetic Health Risk Service is indicated for the detection of

14 variants in the SERPINA1 gene associated with AATD. These 14 variants explain $9 5 \%$ of AATD cases1 and are mainly found in European population, except for $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama, which was described in the Asian population. 5 The most common variants are $\mathrm { P I } ^ { * } \mathrm { S }$ and $\mathrm { P I } ^ { * } \mathrm { Z }$ . Published studies estimate frequency ranges of $5- 1 0 \%$ and $1 - 3 \%$ for $\mathrm { P I } ^ { * } \mathrm { S }$ and $\mathrm { P I } ^ { * } \mathrm { Z }$ , respectively, in the European population. 3 An analysis of the prevalence of $\mathrm { P I } ^ { * } \mathrm { S }$ and $\mathrm { P I } ^ { * } \mathrm { Z }$ amongst the five major ethnic subgroups in the United States has demonstrated that the highest risk for AATD is found in Caucasians, followed by Hispanics and Blacks, with the lowest prevalence amongst Mexican Americans and no risk amongst Asian. 3 The remaining 12 genetic variants tested are reported with a very low frequency in the population. 3

Considering that a given person can have a genetic result of zero, one or two copies of the variants detected, a total of 120 possible genotype combinations of these 14 variants can be reported by the AlphaID At Home Genetic Health Risk Service:

• 1 genetic result with 0 variants detected (wild-type homozygous),   
• 14 genetic results with 1 variant detected (heterozygous), and   
• 105 genetic results with 2 variants detected (rare homozygous or compound heterozygous).

In some rare circumstances, a “variant not determined” result could be reported, which means that the Service cannot determine if the user has any of the 14 tested variants in the SERPINA1 gene linked to AATD. This is an inconclusive genetic result. The presence of any other rare genetic variant in the DNA sequence can interfere with the genetic analysis, leading to the inability to determine the genetic result. In this case, the risk of developing lung and liver disease cannot be determined, and a different test/Service is recommended to the user (DNA sequencing of the SERPINA1 gene).

Depending on the specific variant combination detected, the AlphaID At Home Genetic Health Risk Service provides the individuals’ genetic health risk for developing lung and/or liver disease linked to AATD. These variants can cause mild, severe, or very severe plasma deficiency and can cause no accumulation or mild or severe accumulation of protein in the liver. Normal AAT levels $( 8 0 - 1 2 0 \% )$ are defined by serum AAT levels above $2 0 \mu \mathrm { m o l / L }$ (measured by nephelometry). 3 It is considered that severe AATD is defined by serum AAT levels below the lung protective threshold of $3 5 \%$ of the mean expected value $\mathrm { 5 0 m g / d l }$ or $1 1 ~ \mu \mathrm { m o l / L }$ measured by nephelometry, or $8 0 \mathrm { m g / d L }$ measured by radial immunodiffusion). 2,3 Very severe AATD is defined by no active AAT or AAT levels below $1 \%$ . AAT levels for the most frequent and most studies studied genetic results are reported in Stoller et al. $2 0 1 7 ^ { 1 }$ and provided in the table below.

<table><tr><td rowspan=2 colspan=1>AAT ProteinVariants</td><td rowspan=1 colspan=2>Serum AAT Levels</td><td rowspan=2 colspan=1>AAT ProteinReduction</td></tr><tr><td rowspan=1 colspan=1>True Level Mean,μmol/L(5th-95thpercentile)</td><td rowspan=1 colspan=1>Commercial StandardMedian, mg/dL(5th95th percentile)</td></tr><tr><td rowspan=1 colspan=1>PI*M/PI*M</td><td rowspan=1 colspan=1>33(20-53)</td><td rowspan=1 colspan=1>147(102254)</td><td rowspan=4 colspan=1>Normal-Mild</td></tr><tr><td rowspan=1 colspan=1>PI*M/PI*S</td><td rowspan=1 colspan=1>33(18-52)</td><td rowspan=1 colspan=1>125(86218)</td></tr><tr><td rowspan=1 colspan=1>PI*M/PI*Z</td><td rowspan=1 colspan=1>25.4(15-42)</td><td rowspan=1 colspan=1>90(62-151)</td></tr><tr><td rowspan=1 colspan=1>PI*S/PI*S</td><td rowspan=1 colspan=1>28(20-48)</td><td rowspan=1 colspan=1>95(43-154)</td></tr><tr><td rowspan=1 colspan=1>PI*S/PI*Z</td><td rowspan=1 colspan=1>16.5(10-23)</td><td rowspan=1 colspan=1>62(233108)</td><td rowspan=1 colspan=1>Mild severe</td></tr><tr><td rowspan=1 colspan=1>PI*Z/PI*Z</td><td rowspan=1 colspan=1>5.3(3.4-7)</td><td rowspan=1 colspan=1>≤29(≤29-52)</td><td rowspan=1 colspan=1>Severe</td></tr><tr><td rowspan=1 colspan=1>Null-Null*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Very severe</td></tr><tr><td rowspan=1 colspan=4>*Null variants include PI*Q0 granite falls, PI*Q0 west, PI*Q0 bellingham, PI*Q0mattawa and PI*Q0 clayton</td></tr></table>

The risk categorization is based on the reported clinical cases (published references) for each genetic result. If any of the 14 variants tested is detected, one of the four following risk categories will be reported:

1) Increased risk: There is an increased risk of developing lung or liver disease linked to AATD compared to the general population. The chance of developing lung or liver disease linked to AATD is higher than that of the general population. Above $80 \%$ of people with this genetic result develop lung or liver disease during their lifetime.

2) Slightly at Increased risk: There is a slightly increased risk of developing lung or liver disease linked to AATD compared to the general population. The chance of developing lung or liver disease linked to AATD is slightly higher than that of the general population. $20 \mathrm { - } 8 0 \%$ of people with this genetic result develop lung or liver disease during their lifetime.

3) Not likely at increased risk: There is average risk of developing lung or liver disease linked to AATD compared to the general population. The chance of developing lung or liver disease linked to AATD is similar to that of the general population. Below $20 \%$ of people with this genetic result develop lung or liver disease during their lifetime.

4) Unknown risk: The risk of developing lung or liver disease linked to AATD is not known due to the lack of reported clinical cases or inconclusive data. The chance of developing lung or liver disease linked to AATD is unknown. More clinical studies are needed to determine the risk level.

When No Variants are detected by this Service, the result will show ‘Not Likely at Risk for AATD’. It cannot rule out the possibility of having an extremely rare variant linked to AATD not tested by this Service. It is still possible to have a higher risk category of developing lung and/or liver disease linked to AATD.

These reported risk categories are based on the evidence in the scientific literature and are summarized below separately for lung disease and liver disease:

# a. Lung disease risk categorization

The AAT protein is made in the liver, released into the bloodstream, and enters the lungs, where it inactivates neutrophil elastase, thereby protecting the lung tissue from protease mediated damage. As such, the plasma levels of this protein have widely been described to be directly related to the amount that is reaching the lungs, and consequently, with its lung protection capability. $2 \%$ to $3 \%$ of patients with COPD in the United States are estimated to have AATD. 2 Symptomatic obstructive lung disease in AATD usually presents in patients in their 30s and 40s. Considerable variability in the time of onset of symptoms has been described. Although severe symptoms are most often seen in current or previous cigarette smokers, some smokers and many nonsmokers develop no symptoms at all. 2,3,6,7

The classic pulmonary presentation of AATD is a severe, early onset panacinar emphysema with a basilar predominance in adults. However, emphysema may also occur in a diffuse distribution or predominantly in the upper lobes. 2,3,6,7 In addition to emphysema, AATD is also characterized by chronic, progressive fatigue and other nonspecific respiratory symptoms. 3 Shortness of breath, medically known as dyspnea, is generally the prominent symptom, but chronic cough or wheezing may also occur, as early as age 18 years. 2,6 Pulmonary function testing show findings consistent with COPD; however, bronchodilator responsiveness may be seen and may be labelled as asthma. 7 The presence of episodic wheezing and dyspnea consistent with a diagnosis of asthma has been noted in AATD. 2 Recurring lung infections are also shown in subjects with AATD. 2 Bronchiectasis, with or without concomitant emphysema, is less common. 6,7 Although it is likely that AATD can promote the development of bronchial asthma and bronchiectasis, there is no reliable evidence as to whether AATD influences the frequency or gravity of these diseases. 3

The reported lung disease risk category has been determined based on the clinical outcome which includes chronic obstructive pulmonary disease (COPD), emphysema, chronic bronchitis, and bronchiectasis of the reported cases for each genetic result. The following general criteria summarized in the table below has been followed for the assignment of a risk category for each genetic result. The “Unknown risk” category is assigned to genotypes with $< 3$ reported clinical cases, as well those with ${ \ge } 3$ reported clinical cases but with a calculated risk confidence interval overlapping with the other two risk categories.

<table><tr><td rowspan=1 colspan=4>General Criteria for Lung Disease Risk Categorization</td></tr><tr><td rowspan=1 colspan=1>Number of reportedclinical cases</td><td rowspan=1 colspan=1>%Reported clinicalcases with lung disease</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Reported lung diseaserisk category*</td></tr><tr><td rowspan=1 colspan=1>n&lt;3</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=8 colspan=1>n≥3</td><td rowspan=2 colspan=1>&lt;20%</td><td rowspan=1 colspan=1>0-80%</td><td rowspan=1 colspan=1>Not Likely at increased risk</td></tr><tr><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=3 colspan=1>20-80%</td><td rowspan=1 colspan=1>0-80%</td><td rowspan=1 colspan=1>Not Likely at increased risk</td></tr><tr><td rowspan=1 colspan=1>20-100%</td><td rowspan=1 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=3 colspan=1>&gt;80%</td><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=1 colspan=1>20-100%</td><td rowspan=1 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>&gt;80%</td><td rowspan=1 colspan=1>Increased risk</td></tr><tr><td rowspan=1 colspan=4>NOTE: Exceptions to this general classification are:All carriers with at least one PI*M allele (except PI*M/PI*Z ever-smokers) will be reported as &#x27;Notlikely at increased risk&#x27;PI*Z/PI*Null and PI*Null/PI*Null combinations will be assigned as &#x27;Increased risk&#x27;. Null variants:PI*Q0 granite falls, PI*Q0 west, PI*Q0 bellingham, PI*Q0 mattawa and PI*Q0 clayton*The reporte risk category may beupdate if there s enough data to provideconfidence intervals that onot overlap between two consecutive risk categories</td></tr></table>

The lung disease risk categorization along with the $9 5 \%$ confidence intervals (CIs) of the risk estimates for the 119 genetic results with one or two variants detected are presented in the table below based on the number of reported clinical cases extracted from the referenced scientific literature and meta-analyses that analyze clinical cases with and without lung disease. Pending new clinical cases being published, the risk category is to be updated if there is enough data to provide confidence intervals that do not overlap between two consecutive risk categories. Since the intended population of the Service is $\geq 1 8$ years of age, only cases $( \ge 1 8$ years old) have been included in the analysis to estimate the risk, when the age date was described in the study. The exceptions to the general criteria for liver disease risk categorization are shaded in grey in the table below.

<table><tr><td colspan="1" rowspan="3">#</td><td colspan="6" rowspan="1">Lung Disease Risk Categorization</td></tr><tr><td colspan="1" rowspan="2">GeneticResults</td><td colspan="1" rowspan="1">Reported</td><td colspan="1" rowspan="2">ReportedClinicalCases withoutLung Disease</td><td colspan="1" rowspan="2">%ReportedClinicalCases withLung Disease1</td><td colspan="1" rowspan="2">95%CI1</td><td colspan="1" rowspan="2">Reported LungDisease RiskCategory</td></tr><tr><td colspan="1" rowspan="1">ClinicalCases withLung Disease</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Most freque</td><td colspan="2" rowspan="1">Most frequent² genetic results (#1- #6)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PI*M/PI*M</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">None of the 14 allelic variants interrogated by the A1AT Genotyping test are detected in the SERPINA1gene, but other variants could be present</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PI*M/</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">Never-smokers</td><td colspan="1" rowspan="1">155</td><td colspan="1" rowspan="1">1967</td><td colspan="1" rowspan="1">7.3</td><td colspan="1" rowspan="1">6.4 - 8.3</td><td colspan="1" rowspan="1">Not Likely atincreased risk</td></tr><tr><td colspan="5" rowspan="1">Dahl et al., 2002: 86 individuals with lung disease and 364 individuals without lungdisease; Seersholm et al., 2000: 47 individuals with lung disease and 1504 without lungdisease; Klayton et al., 1975: 13 individuals with lung disease and 14 without lungdisease; Matzen et al., 1977: 2 individuals with lung disease and 10 without lungdisease; Gulsvik et al., 1979: 7 individuals with lung disease and 44 without lungdisease; Chang-Yeung et al., 1978: 0 individuals with lung disease and 31 without lungdisease.</td></tr><tr><td colspan="1" rowspan="2">Ever-smokers</td><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">142</td><td colspan="1" rowspan="1">44.3</td><td colspan="1" rowspan="1">39.3 - 49.5</td><td colspan="1" rowspan="1">Slightlyincreased risk</td></tr><tr><td colspan="4" rowspan="1">Sørheim et al., 2010: 43 individuals with lung disease and 72 without lung diseaseMolloy et al., 2014: 64 individuals with lung disease and 78 without lung disease</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2">PI*M/PI*S</td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">590</td><td colspan="1" rowspan="1">14.6</td><td colspan="1" rowspan="1">12.5 - 17.0</td><td colspan="1" rowspan="1">Not Likely atincreased risk</td></tr><tr><td colspan="5" rowspan="1">Dahl et al., 2002: 73 individuals with lung disease and 383 without lung diseaseGulsvik et al., 1979: 14 individuals with lung disease and 46 without lung diseaseChang-Yeung et al., 1978: 7 individuals with lung disease and 82 without lung diseaseMatzen et al., 1977: 6 individuals with lung disease and 54 without lung diseaseOstrow et al., 1978: 1 individual with lung disease and 25 without lung disease</td></tr><tr><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="2">PI*S/PI*S</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">8.7</td><td colspan="1" rowspan="1">4.0 - 18.0</td><td colspan="1" rowspan="1">Not Likely atincreased risk</td></tr><tr><td colspan="5" rowspan="1">Sandford et al., 1999 2 individuals with lung disease and 0 without lung disease;Gulsvik et al., 1979 1 individual with lung disease and 0 without lung disease;Lieberman et al., 1986 1 individual with lung disease and 2 without lung disease;Arnaud et al., 1977 23 Individuals without lung disease; Dahl et al., 2002 0 individualswith lung disease and 12 without lung disease; Fagerhol et al., 1969 0 individuals withlung disease and 3 s without lung disease; Kueppers et al., 1977 0 individuals with lungdisease and 1 without lung disease; Lochon et al., 1978 0 individuals with lung diseaseand 1 without lung disease</td></tr><tr><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="2">PI*S/PI*Z</td><td colspan="1" rowspan="1">269</td><td colspan="1" rowspan="1">228</td><td colspan="1" rowspan="1">54.1</td><td colspan="1" rowspan="1">50.1 - 57.8</td><td colspan="1" rowspan="1">Slightlyincreased risk</td></tr><tr><td colspan="5" rowspan="1">McElvaney et al., 2020: 269 individuals with lung disease and 181 without lung diseaseBartmann et al., 1985: 18 individuals with lung disease and 1 without lung diseaseDahl et al., 2002: 4 individuals with lung disease and 10 without lung diseaseLieberman et al., 1986: 2 individuals with lung disease and 5 without lung diseaseAbboud et al., 1979: 1 individual with lung disease; Fagerhol et al., 1969: 1 individualwith lung disease and 1 individual without lung disease; Gulsvik et al., 1979: 1individual with lung disease and 3 individuals without lung disease</td></tr><tr><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="2">PI*Z/PI*Z</td><td colspan="1" rowspan="1">2775</td><td colspan="1" rowspan="1">630</td><td colspan="1" rowspan="1">81.5</td><td colspan="1" rowspan="1">80.4 - 82.6</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="5" rowspan="1">McElvaney et al., 2020: 2775 individuals with lung disease and 630 without lungdisease</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Less common genetic results (#7#120)</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">PI*S/PI*I</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Seri et al., 1992 1 individual without lung disease; Huang et al., 2017 1 individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">8</td><td colspan="1" rowspan="1">PI*F/PI*S</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Cook et al., 1996 1 individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">9</td><td colspan="1" rowspan="1">PI*I</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="1">PI*F/PI*I</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="4" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">11</td><td colspan="1" rowspan="1">PI*F/PI*F</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Sinden et al., 2014: 1 inc</td><td colspan="1" rowspan="1">ividual with lung d</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">12</td><td colspan="1" rowspan="1">PI*ZIPI*I</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">59.6 - 95.5</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Ferrarotti et al., 2005: 2 individuals with lung diseaseBaur &amp; Bencze 1987: 1 individual with lung diseaseCorda et al., 2006: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">13</td><td colspan="1" rowspan="1">PI*F/PI*Z</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Sinden et al., 2014: 5 individuals with lung disease and 1 without lung disease; Cockcroft et al., 1981: 3individuals with lung disease; Cook et al., 1996: 2 individuals with lung disease; Beckman et al. 1984: 1al wulal wuewith lung disease</td></tr><tr><td colspan="1" rowspan="2">14</td><td colspan="1" rowspan="1">PI*S/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">15</td><td colspan="1" rowspan="1">PI*S/PI*M malton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Figueira Gonçalves et a</td><td colspan="1" rowspan="1">, 2017: 1 individua</td><td colspan="1" rowspan="1">l with lung diseas</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">16</td><td colspan="1" rowspan="1">PI*S/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">17</td><td colspan="1" rowspan="1">PI*S/PI*P lowell</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Jardí et al., 1997: 1 indiv</td><td colspan="1" rowspan="1">idual without lung</td><td colspan="1" rowspan="1">disease; Balbi et a</td><td colspan="1" rowspan="1">., 2019 :1 indivic</td><td colspan="1" rowspan="1">ual with lung</td><td colspan="1" rowspan="1">canmealow</td></tr><tr><td colspan="1" rowspan="2">18</td><td colspan="1" rowspan="1">PI*S/PI*M heerlen</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Kramps et al., 1981: 1 individual with lung disease and 1 without lung disease</td></tr><tr><td colspan="1" rowspan="2">19</td><td colspan="1" rowspan="1">PI*I/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">20</td><td colspan="1" rowspan="1">PI*I/PI*M malton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">21</td><td colspan="1" rowspan="1">PI*I/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">22</td><td colspan="1" rowspan="1">PI*M/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">PI*I/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">24</td><td colspan="1" rowspan="1">PI*F/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">25</td><td colspan="1" rowspan="1">PI*F/PI*M malton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">26</td><td colspan="1" rowspan="1">PI*F/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">27</td><td colspan="1" rowspan="1">PI*F/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">28</td><td colspan="1" rowspan="1">PI*F/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">29</td><td colspan="1" rowspan="1">PI*S/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">30</td><td colspan="1" rowspan="1">PI*S/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">31</td><td colspan="1" rowspan="1">PI*S/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">32</td><td colspan="1" rowspan="1">PI*S/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">33</td><td colspan="1" rowspan="1">PI*S/PI*Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Rosenbaum et al., 2017: 1 individual with l</td><td colspan="2" rowspan="1">ng disease and 1 without lung dises</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">34</td><td colspan="1" rowspan="1">PI*I/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">35</td><td colspan="1" rowspan="1">PI*I/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">36</td><td colspan="1" rowspan="1">PI*I/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">37</td><td colspan="1" rowspan="1">PI*I/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">38</td><td colspan="1" rowspan="1">PI*I/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">39</td><td colspan="1" rowspan="1">PI*F/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">40</td><td colspan="1" rowspan="1">PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">41</td><td colspan="1" rowspan="1">PI*F/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">42</td><td colspan="1" rowspan="1">PI*F/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">43</td><td colspan="1" rowspan="1">PI*F/PI*Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0-100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Ringenbach et al., 2011:</td><td colspan="1" rowspan="1">1 individual with l</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">44</td><td colspan="1" rowspan="1">PI*Z/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">42.5-100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Ferrarotti et al., 2005: 1 individual with lung diseaseLonardo et al., 2002: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">45</td><td colspan="1" rowspan="1">PI*ZIPI*M malton</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">82.5 - 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">   0  Joiul wi uisae;All1viul wi u sal viual withlungisease;Suh-Lail alvial with uisease;FiguGonçalve2017: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">46</td><td colspan="1" rowspan="1">PI*ZIPI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">47</td><td colspan="1" rowspan="1">PPI*ZIPI*P lowell</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72.1 - 100.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Bamforth &amp; Kalsheker 1988: 4 individuals with lung disease; Esteves-Brandão et al., 2019: 1 individual withlu disease; Holmes e al., 199:individual with ung disease; Ferrarotti et al, 2005: with lungdisase</td></tr><tr><td colspan="1" rowspan="2">48</td><td colspan="1" rowspan="1">PI*ZIPI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">42.5 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Klaassen et al., 2001: 2</td><td colspan="1" rowspan="1">ndividuals with lun</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">49</td><td colspan="1" rowspan="1">PI*M procida/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">42.5 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Ferrarotti et al., 2005: 2</td><td colspan="1" rowspan="1">ndividuals with lur</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">50</td><td colspan="1" rowspan="1">PI*M malton/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">51</td><td colspan="1" rowspan="1">PI*M procida/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">52</td><td colspan="1" rowspan="1">PI*M procida/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">53</td><td colspan="1" rowspan="1">PI*M procida/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">PI*M malton/PI*M malton</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">79.2</td><td colspan="1" rowspan="1">63.0 - 89.5</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">′</td><td colspan="6" rowspan="1">al0ivil wihisaniut uiseFo al00iwith lung disease; Reid et al. 1987: 2 individuals with lung disease and 1 without lung disease; FigueiraGalves et al., 2017:individual with lung disease nd  witout lung disease; Curiel et al 189indvualiseBalc ivial wh unis; rnlvidallung disease; Joly et al., 2015: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">55</td><td colspan="1" rowspan="1">PI*M malton/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">56</td><td colspan="1" rowspan="1">PI*M malton/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">57</td><td colspan="1" rowspan="1">PI*M malton/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">58</td><td colspan="1" rowspan="1">PI*iiyama/PI*iiyama</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">52.6-100.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">l ivial wh unis La l 9iviual wh uniseseual., 1993: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">59</td><td colspan="1" rowspan="1">PI*P lowell/PI*iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">60</td><td colspan="1" rowspan="1">PI*M heerlen/PI*S iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">61</td><td colspan="1" rowspan="1">PI*P lowell/PI*P lowell</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Faber et al., 1989: 1 ind</td><td colspan="1" rowspan="1">vidual with lung di</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">62</td><td colspan="1" rowspan="1">PI*P lowell/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">63</td><td colspan="1" rowspan="1">PI*M heerlen/PI*M heerlen</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">64.9 - 100.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Fregonese et al., 2008: 4</td><td colspan="1" rowspan="1">individuals with lu</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">64</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">65</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">66</td><td colspan="1" rowspan="1">PI*I/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">67</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">68</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">PI*M malton/</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PI*Q0 granite falls</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">70</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">71</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">72</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 mattawa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50.0</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Lara et al., 2013: 1 indiv</td><td colspan="1" rowspan="1">idual with lung dise</td><td colspan="1" rowspan="1">ase; Balduyck et d</td><td colspan="1" rowspan="1">l., 2014: 1 indivi</td><td colspan="2" rowspan="1">dual without lung disease</td></tr><tr><td colspan="1" rowspan="2">73</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Rodriguez-Frias et al., 2</td><td colspan="1" rowspan="1">11: 1 individual w</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">74</td><td colspan="1" rowspan="1">PI*iiyama/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">75</td><td colspan="1" rowspan="1">PI*iiyama/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Cases not reported</td></tr><tr><td colspan="1" rowspan="2">76</td><td colspan="1" rowspan="1">PI*iiyama/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">77</td><td colspan="1" rowspan="1">PI*iiyama/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">78</td><td colspan="1" rowspan="1">PI*iiyama/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">42.5 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Ko et al., 2011: 1 indivic</td><td colspan="1" rowspan="1">ual with lung disea</td><td colspan="1" rowspan="1">se; Miyahara et al</td><td colspan="1" rowspan="1">,2001: 1 individu</td><td colspan="1" rowspan="1">al with lung di</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">79</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">80</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 granite falls</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Poller et al., 1999: 1 individual with lung disease</td><td colspan="2" rowspan="1">Poller et al., 1999: 1 individual with lung disease</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">81</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="3" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">82</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">83</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">84</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="2">86</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">87</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">88</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Brantly et al., 1997 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="6" rowspan="1">Assignment of 'Not likely at Increased risk' to all carriers with at least one PI*M allele(Exceptions to the general criteria for lung disease risk categorization are shaded)</td></tr><tr><td colspan="1" rowspan="2">89</td><td colspan="1" rowspan="1">PI*M/PI*I</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">14.3</td><td colspan="1" rowspan="1">6.6 - 28.3</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Zhuaaliyeval 1ividuals wihluisease; Dukal016:dividuals wi luisewithout clinical data;Arnaud et al.19783individuals without lung disease and 20 without clinicaldataBaur &amp; Bencze 1987: 1 individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">91</td><td colspan="1" rowspan="1">PI*M/PI*F</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">16.3</td><td colspan="1" rowspan="1">11.2 - 23.1</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Bean e al 18:5idiiduals withou u sease and7 wih un disease; Duk al 01:indvawith lung disease; Kwok et al., 2004: 5 individuals with lung disease; Tete-Benissan &amp; Gbeassor 2011: 74ividual without un disese; Cockcro etal 1983idividal ithout u seas; Cook al 191individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">94</td><td colspan="1" rowspan="1">PI*M/PI*M malton</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">2.4 - 14.1</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Corda et al., 2006: 2 individuals with lung disease and 4 without lung diseaseFigua Gonçalves e al. 0:individual with lung disease and 1without ung disease; Sproule et al 198:  eOl 0iv iu u;individuals without lung disease; Joly et al., 2015: 1 individual without lung disease; Canva et al., 2001: 1individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">90</td><td colspan="1" rowspan="1">PI*M/PI*M heerlen</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">11.5 - 52.1</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Kalsheker et al., 1992: 2 individuals with lung disease and 1 without lung disease; Rodriguez et al., 2002: 1individual with lung disease; Kramps et al. 1981:4 individuals without lung disease; Pollr et al, 1999: 3individuals without lung disease</td></tr><tr><td colspan="1" rowspan="2">92</td><td colspan="1" rowspan="1">PI*M/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Montealegre et al., 200e</td><td colspan="1" rowspan="1">: 2 individuals with</td><td colspan="1" rowspan="1">out lung disease</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">EY</td></tr><tr><td colspan="1" rowspan="2">93</td><td colspan="1" rowspan="1">PI*M/PI*S iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 40.4</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Takabe et al., 1992: 4 ir</td><td colspan="1" rowspan="1">dividuals without l</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">95</td><td colspan="1" rowspan="1">PI*M/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">11.1</td><td colspan="1" rowspan="1">3.7  28.6</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">S l19:idiviual wi lunisease;Denden al 00iviual with luiseae; Jr19712 individuals without lung disease; Cook et al 1995: 3 individuals without lung disease; Corda al2011: 1 individual without lung disease</td></tr><tr><td colspan="1" rowspan="2">96</td><td colspan="1" rowspan="1">PI*M/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 47.3</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Poller et al., 1999: 3 individuals without lung disease</td></tr><tr><td colspan="1" rowspan="2">97</td><td colspan="1" rowspan="1">PT"M/PI*Q0 west</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Laubach et al., 1993: 1 </td><td colspan="1" rowspan="1">ndividual with lung</td><td colspan="1" rowspan="1">disease</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">98</td><td colspan="1" rowspan="1">PI*M/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 47.3</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="3" rowspan="1">Cook et al., 1994: 3 individuals without lung disease</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">99</td><td colspan="1" rowspan="1">PI*M/PI*Q0 mattawa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">27.0 - 100.0</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Cox &amp; Levison 1988: 2 individuals with lung disease and 11 without lung diseaseLara et al., 2013: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">100</td><td colspan="1" rowspan="1">PPI*M/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Rodriguez-Frias et al., 2011:1 individual without lung disease</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="6" rowspan="1">Assignment of 'Increased risk' category to all carriers with PI*Z/PI*Null or PI*Nul/PI*Null(Exceptions to the general criteria for lung disease risk categorization are shaded)</td></tr><tr><td colspan="1" rowspan="2">101</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 granite falls</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">52.6 - 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Nukiwa et al., 1987: 1 individual with lung diseaseBalduyck et al., 2014: 1 individual with lung diseasePavii et al., 2019: 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">102</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 mattawa</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">52.6 - 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1">Cox &amp; Levinson 1988 3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">103</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 granite falls</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">59.6 - 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Hubbard et al., 1989: 2 individuals with lung diseaseBalbi et al., 2019: 1 individual with lung diseaseHolmes et al., 1989: 1 individual with lung disease.</td></tr><tr><td colspan="1" rowspan="2">104</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 bellingham</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">68.9 - 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Fregonese et al., 2008 3 individuals with lung disease.Satoh et al., 1988 1 individual with lung diseaseCook et al., 1994 1 individual with lung diseaseGarver et al., 1986 1 individual with lung disease</td></tr><tr><td colspan="1" rowspan="2">105</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="3" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">106</td><td colspan="3" rowspan="1">PI*ZI1                   0PI*Q0 bellingham</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 -100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="3" rowspan="1">Poller et al., 1990: 1 individual with lung disease</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">107</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 mattawa</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">27.0 -100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1">Balduyck et al., 2014: 1</td><td colspan="1" rowspan="1">individual with lung</td><td colspan="1" rowspan="1">cmYlow</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">108</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">109</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">110</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">111</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">112</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">PI*Q0 west/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">114</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">115</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">42.5 100.0</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Curiel et al.. 1989 2 ind</td><td colspan="1" rowspan="1">viduals with lung d</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">119</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Increased risk</td></tr><tr><td colspan="7" rowspan="1">Clinical cases not reported1 The percentage (%) and 95% confidence interval (C195%) of the reported clinical cases have been calculatedfollowing the score method described by Altman et al., 2000.2 The diff cis f the PI*S and PI*Z alels (PI*M/PI*S, PI*M/PI*Z, PI*S/PI*Z, PI*S/PI*S and PI*Z/PI*Z) cover more than 95% of variant alleles identified in the SERPINA1 gene worldwide.</td></tr></table>

Among the most frequent genetic combinations, the $\mathrm { P I ^ { * } M / P I ^ { * } S }$ and $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } S }$ genetic results do not appear to be associated with an increased risk for lung disease. 1-3 Individuals (especially never-smokers) with $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } Z }$ are not considered to be at risk for lung disease. 1 However, a subset of individuals with the $\mathrm { P I ^ { * } M / P I ^ { * } Z }$ genotype may experience accelerated lung destruction, especially if they are smokers. 2,8 Smokers with this genetic result have a slightly increased risk of developing lung disease linked to AATD. $\mathrm { P I ^ { * } M / P I ^ { * } Z }$ heterozygotes have a clear predisposition to chronic obstructive pulmonary disease (COPD), at least among smokers. 8 All the

AlphaID At Home Genetic Health Risk Service reports describe smoking as a nongenetic risk factor that can increase the risk of developing lung disease. Except for the $\mathrm { P I ^ { * } M P I ^ { * } } / Z$ genotype, a differential classification for ever-smokers (Slightly at increased risk) and non-smokers (Not likely at increased risk) has not been possible for other genotypes because information about the smoking history of every specific subject included in the studies was not available.

There is a $20 \%$ of individuals with $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ with AAT levels below the lung protective threshold $( { < } 1 1 \ \mu \mathrm { M / L } )$ , who are at risk for AATD related lung diseases. 1 Studies demonstrated that the odds ratio for COPD in individuals with this genetic result, compared with normal $\mathrm { P I ^ { * } M / P I ^ { * } M }$ individuals, was significantly increased at 3.3 ( $9 5 \%$ CI: $1 . 2 \mathrm { - } 8 . 6 \%$ ), showing that the $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ genotype is a significant risk factor for COPD. 9 Although individuals with this genetic result who do not smoke are considered at slightly at increased risk of developing lung disease, existing scientific evidence confirms that the effect of smoking in this specific variant combination is profound and will produce reduction in pulmonary function. It is generally accepted that individuals with $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ and serum AAT levels below the protective threshold value, have an increased risk of developing lung disease, especially if they smoke. 10 However, available data from ever-smokers indicate that smokers with $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ may be also assigned to the ‘Slightly at increased risk’ category, the same as the general risk category assigned for this genotype. Hutchinson et al., 1983 described that eight $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ index cases smokers developed emphysema out of the 12 total index cases smokers $6 6 . 7 \%$ with $9 5 \% \mathrm { C I }$ : $4 3 . 1 \mathrm { - } 8 4 . 1 \% )$ .10 Thus, even considering only smoker and index cases, there is no evidence supporting an “Increased risk” classification $( > 8 0 \%$ reported cases with lung disease) for this subgroup of $\mathrm { P I ^ { * } S } / \mathrm { P I ^ { * } Z }$ individuals.

The $\mathrm { P I } ^ { * } \mathrm { Z }$ variant in homozygosis is associated with an increased risk of lung disease. 2,3 Greater than $80 \%$ of people with this genetic result develop lung disease during their lifetime. 1

Exceptions to the criteria for lung disease risk categorization are applied to the following genetic results:

(i) carriers with at least one $\mathrm { P I } ^ { * } \mathbf { M }$ (normal) allele due to the limited information available up to date. Based on the clinical data from the carriers of the most common variants, as well as on the expected protein levels above the protective threshold for the subjects presenting at least one $\mathrm { P I } ^ { * } \mathbf { M }$ allele, it is more adequate to report all the carriers except for ever-smokers $\mathrm { P I ^ { * } M / P I ^ { * } Z }$ as “Not likely at increased risk” (refer to $\# 8 8 – 9 9$ in the ‘Lung Disease Risk Categorization’ table). Consistent with this risk categorization, for individuals who possess only one deficiency allele, the risk of developing any clinically significant symptoms is

either slight or negligible. 11 Data from carriers with enough clinical cases (refer to #88−94) also support this “Not likely at increased risk” classification.

(ii) carriers with the PI\*Z/PI\*Null and PI\*Null/PI\*Null combinations that cause severe and very severe plasma deficiency, respectively. The AAT protein levels for the $\mathrm { P I ^ { * } Z / P I ^ { * } N u l l }$ and PI\*Null/PI\*Null individuals are expected to be below the protective threshold. Both the $\mathrm { P I ^ { * } Z / P I ^ { * } N u l l }$ and PI\*Null/PI\*Null combinations are assigned as “Increased risk”. Data from combinations with enough clinical cases also support this “Increased risk” classification because $100 \%$ individuals with these genetic results develop lung disease, although the number of cases is not high enough (refer to #100−103 in the ‘Lung Disease Risk Categorization’ table).

# b. Liver disease risk categorization

In addition to pulmonary complications, AATD is also a risk factor for other serious illnesses, most commonly for liver disease. Liver disease associated with AATD is mainly caused by an impaired process of releasing the AAT protein into the bloodstream, which gives rise to its accumulation in the liver. The accumulation of inclusions of the mutant AAT protein triggers cycles of hepatocyte injury, cell death and compensatory proliferation. 2,3

The severity of the disease correlates with the magnitude of accumulation of the mutant AAT protein. It is known that the prevalence of chronic liver disease in AATD increases with age and that chronic liver disease may be asymptomatic for years. Liver disease linked to AATD may present clinically in a variety of ways: liver enzyme elevations, jaundice, hepatic fibrosis, chronic hepatitis, liver failure or liver cirrhosis, and in a low percentage of subject’s hepatocellular carcinoma. 2,3

The $\mathrm { P I } ^ { * } \mathrm { Z }$ , $\mathrm { P I } ^ { * } \mathbf { M }$ malton and $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama variants (i.e., severe accumulation alleles) cause severe accumulation of the mutant AAT protein in the liver whereas the $\mathrm { P I } ^ { * } \mathbf { M }$ procida, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ granite falls, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ west, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ bellingham, $\mathrm { P I ^ { * } F }$ , $\mathrm { P I } ^ { * } \mathrm { P }$ lowell, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ mattawa, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ clayton, and $\mathrm { P I } ^ { * } \mathbf { M }$ Heerlen variants (i.e., non-accumulation variants) are not associated with the accumulation of the mutant AAT protein in the liver. The mild accumulation alleles are $\mathrm { P I } ^ { * } \mathrm { I }$ and $\mathrm { P I } ^ { * } \mathrm { S }$ .

The $\mathrm { P I } ^ { * } \mathrm { Z }$ allele encodes an AAT protein that is able to bind to other AAT proteins, leading to AAT polymers formation, which form inclusion bodies inside the liver and are not secreted into the circulation. 2,3 Clinical studies with these affected individuals have consistently associated this variant with a slightly increased risk of developing liver disease. 2,3 Similarly, the $\mathrm { P I } ^ { * } \mathbf { M }$ malton variant is known to be significantly associated with hepatic disease due to intracellular accumulation of the mutant AAT protein. 13 $\mathrm { P I } ^ { * } \mathbf { M }$ malton homozygous individuals have also shown a tendency to form polymeric intrahepatic inclusions12 and available scientific data confirm association with liver disease in these individuals.2,3

The liver disease risk category reported by the AlphaID At Home Genetic Health Risk Service was determined based on the probability of each tested variant to be accumulated in the liver and clinical outcome of the reported cases for each genetic result. The following general criteria summarized in the table were followed for the assignment of a liver disease risk category for each genetic result. Pending new clinical cases being published, the risk category may be updated if there is enough data to provide confidence intervals that do not overlap between two consecutive risk categories.

<table><tr><td rowspan=1 colspan=1>Number of reportedclinical cases</td><td rowspan=1 colspan=1>%Reported clinicalcases with liver disease</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Reported liver diseaserisk category*</td></tr><tr><td rowspan=1 colspan=1>n&lt;3</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=8 colspan=1>n≥3</td><td rowspan=2 colspan=1>&lt;20%</td><td rowspan=1 colspan=1>0-80%</td><td rowspan=1 colspan=1>Not Likely at Increased risk</td></tr><tr><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=3 colspan=1>20-80%</td><td rowspan=1 colspan=1>0-80%</td><td rowspan=1 colspan=1>Not Likely at Increased risk</td></tr><tr><td rowspan=1 colspan=1>20-100%</td><td rowspan=1 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=3 colspan=1>&gt;80%</td><td rowspan=1 colspan=1>0-100%</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=1 colspan=1>20-100%</td><td rowspan=1 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>&gt;80%</td><td rowspan=1 colspan=1>Increased risk</td></tr><tr><td rowspan=1 colspan=4>NOTE: Exceptions to this general liver disease risk classification are:•Individuals with at least one PI*Z or PI*M malton severe accumulation allele and theircombinations will be reported as &quot;Slightly Increased risk&quot;•All the combinations including two non-accumulation alleles will be reported as &quot;Not likely atIncreased risk&quot;. Non-accumulation alleles: PI*M, PI*M procida, PI*Q0 granite falls, PI*Q0 west,PI*Q0 bellingham, PI*F, PI*P lowell, PI*Q0 mattawa, PI*Q0 clayton, and PI*M heerlen* The reported risk category may be updated if there is enough published data to provide confidenceintervals that do not overlap between two consecutive risk categories</td></tr></table>

The liver disease risk categorization for the 119 genetic results with one or two variants detected is presented in the table below along with the $9 5 \%$ CIs of the risk estimates based on the number of reported clinical cases with and without liver disease in the referenced scientific literature. Considering that the intended population is ${ \ge } 1 8$ years old and the phenotype of the liver disease in adults is different from that in children, all pediatric cases $( < 1 8$ years old) have been removed from the analysis performed to estimate the risk of liver disease when the patient age was included in the study.

<table><tr><td colspan="1" rowspan="2">#</td><td colspan="6" rowspan="1">Liver Disease Risk Categorization</td></tr><tr><td colspan="1" rowspan="1">GeneticResults</td><td colspan="1" rowspan="1">ReportedClinical CaseswithLiver Disease</td><td colspan="1" rowspan="1">ReportedClinical CaseswithoutLiver Disease</td><td colspan="1" rowspan="1">%ReportedClinical CaseswithLiver Disease1</td><td colspan="1" rowspan="1">95%CI1</td><td colspan="1" rowspan="1">ReportedLiver DiseaseRisk Category</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Most frequent genetic2 results (#1-#6)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PI*M/PI*M</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">None of the 14 allelic variants interrogated by the A1AT Genotyping test are detected in the SERPINA1 gene,but other variants could be present</td></tr><tr><td colspan="1" rowspan="2">2</td><td colspan="1" rowspan="1">PI*M/PI*S</td><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">1124</td><td colspan="1" rowspan="1">9.8</td><td colspan="1" rowspan="1">8.5 - 11.3</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Eigenrodt et al. 1997: 56 individuals with liver disease and 37individuals without liver disease; Grazid al 198:4 individuals with liver disease and 670 without liver disease; Arnaud et al.1977:185 individualswout liver disease; Bell et al 190individuals with liver disease and 16 without livrdisease; arsonet al., 1981: 12 individuals with liver disease and 116 without liver disease</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="1">PI*M/PI*Z</td><td colspan="1" rowspan="1">431</td><td colspan="1" rowspan="1">948</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">29.2 - 33.3</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Propst et al. 1992:114 individuals with liver disease and 14 without liver disease; Strnad et al., 2019: 90individuals with liver disease and 42 without liver disease; Schaefer et al. 2018 52 individuals with liverdisease and7 without liver disease; Eigenbrodt et al. 1997:50 individuals with liver disease and 25 withoutliver disease; Graziadei et al., 1998: 49 individuals with liver disease and 292 without liver diseaseBlenkinsopp &amp; Haffenden 1977: 15 individuals with liver disease and 4 without liver disease; Arnaud etal1977: 43 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">4</td><td colspan="1" rowspan="1">PI*S/PI*S</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">8.8</td><td colspan="1" rowspan="1">3.6 - 20.1</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Strad et al., 2019: 3 individuals with liver disease and 2 individuals without liver; Arnaud et al. 1977: 23iividuals without liver disease;Carlsont al 191:individuals without liver disease; Eigenbrod l1997: 2 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">5</td><td colspan="1" rowspan="1">PI*S/PI*Z</td><td colspan="1" rowspan="1">139</td><td colspan="1" rowspan="1">359</td><td colspan="1" rowspan="1">27.9</td><td colspan="1" rowspan="1">24.7 - 31.3</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">McElvaney et al., 2020:113 individuals with liver disease and 37 without liver disease; Strnad et al., 2019: 8individuals, 7 with liver disease and 1 without liver disease; Ferrarotti et al., 2005:4 individuals with liverdisease and17 without liverdisease; Eigenbrod et al 1997:3individuals with liverdisease; Graziade1998: 2 individuals with liver disease; Propst et al., 1992: 2 individuals with liver disease</td></tr><tr><td colspan="1" rowspan="2">6</td><td colspan="1" rowspan="1">PI*Z/PI*Z</td><td colspan="1" rowspan="1">1199</td><td colspan="1" rowspan="1">3238</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">25.9 - 28.1</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">McElvaney et al., 2020: 868 individuals with liver disease and 2537 individuals without; Hamesch et al,2019:197 individuals with liver disease and 357 individuals without liver disease; Schneider et al., 2020:77individuals with liver disease and232 individuals without livrdisease; Graziadei et al 199814individualsith liver disease; errarotti  al00:1dividuals with iver isease nd 83 without ive disease; z&amp; Eriksson 1996:13 individuals with liver disease and 18 individuals without liver disease; Eriksson et alndividuals withivrdisaseanidividuals without liverdisease; rops et al 9ndiviualwith iver disease and2 individuals without liver disease; Bell et al. 1990; 3 individuals with liver disease</td></tr><tr><td colspan="7" rowspan="1">Less common genetic results (#7-#120)</td></tr><tr><td colspan="7" rowspan="1">(i) Non-accumulation allele³ combinations</td></tr><tr><td colspan="1" rowspan="2">7</td><td colspan="1" rowspan="1">PI*I/PI*M</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0-10.1</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Araud et al., 1978: 23 individuals without liver disease; Baur &amp; Bencze 1987: 1 individual without liverdisease</td></tr><tr><td colspan="1" rowspan="2">8</td><td colspan="1" rowspan="1">PT*M/PI*M procida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">5.8 - 64.4</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Ferrarotti et al.200:individuals without liver isease; Balduycket al., 014:1individual with ciross</td></tr><tr><td colspan="1" rowspan="2">9</td><td colspan="1" rowspan="1">PI*M/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="1">PI*M/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">11</td><td colspan="1" rowspan="1">PI*M/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">12</td><td colspan="1" rowspan="1">PI*M/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0 - 3.4</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Kelly t al 1989:  individuals without liver disease; Tete-Benissan &amp; Gbeassor 2011: 7 individuals withoutliver disease</td></tr><tr><td colspan="1" rowspan="2">13</td><td colspan="1" rowspan="1">PI*M/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Corda et al., 2011: 1 in</td><td colspan="1" rowspan="1">dividual without l</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">14</td><td colspan="1" rowspan="1">PI*M/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 16.2</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Cox &amp; Levison 1988:1</td><td colspan="2" rowspan="1">3 individuals without liver disease;</td><td colspan="2" rowspan="1">Lara et al., 2013: 1 individual with</td><td colspan="1" rowspan="1">out liver disease</td></tr><tr><td colspan="1" rowspan="2">15</td><td colspan="1" rowspan="1">PI*M/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">16</td><td colspan="1" rowspan="1">PI*M/PI*M heerlen</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">12.1-87.9</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Balc al 0idividual with ive disease (holestasi; Ferrarotal 00:diviual witliver disease</td></tr><tr><td colspan="1" rowspan="2">17</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Cook et al., 1994: 1 inc</td><td colspan="1" rowspan="1">ividual without liv</td><td colspan="1" rowspan="1">er disease; Garver</td><td colspan="1" rowspan="1">et al., 1986: 1 ind</td><td colspan="1" rowspan="1">ividual without</td><td colspan="1" rowspan="1">liver disease</td></tr><tr><td colspan="1" rowspan="2">18</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYalow</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">19</td><td colspan="1" rowspan="1">PI*M heerlen/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0  73.0</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Kramps et al., 1981: 1</td><td colspan="1" rowspan="1">ndividual without</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">20</td><td colspan="1" rowspan="1">PI*P lowell/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">21</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.2</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Cox &amp; Levinson 1988:</td><td colspan="5" rowspan="1">Cox &amp; Levinson 1988: 2 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">22</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.2</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">PI*M procida/PI*M procida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Ferrarotti et al., 2005:</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">th liver disease an</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">24</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">25</td><td colspan="1" rowspan="1">PI*F/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Sinden et al., 2014: 1 i</td><td colspan="1" rowspan="1">ndividual without</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">26</td><td colspan="1" rowspan="1">PI*F/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">27</td><td colspan="1" rowspan="1">PI*F/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Ringenbach et al., 201</td><td colspan="1" rowspan="1">: 1 individual witl</td><td colspan="1" rowspan="1">nout liver disease</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">28</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Poller et al., 1999: 1 individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">29</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not likely atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">30</td><td colspan="1" rowspan="1">PI*F/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">31</td><td colspan="1" rowspan="1">PIF/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">32</td><td colspan="1" rowspan="1">PI*F/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1">ted</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">33</td><td colspan="1" rowspan="1">PI*F/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">34</td><td colspan="1" rowspan="1">PI*F/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">35</td><td colspan="1" rowspan="1">PI*F/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">36</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">37</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">38</td><td colspan="1" rowspan="1">PI*M heerlen/PI*QQ0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">39</td><td colspan="1" rowspan="1">PI*M procida/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">40</td><td colspan="1" rowspan="1">PI*M procida/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">41</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">42</td><td colspan="1" rowspan="1">PI*M procida/PI* Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">43</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">44</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">45</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">46</td><td colspan="1" rowspan="1">PI*P lowell/PI*Mherlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">47</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">E</td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">48</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">49</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">50</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">51</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">52</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">53</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">54</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">55</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">56</td><td colspan="1" rowspan="1">"Qo clayton/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">57</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">58</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanaGenYelowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">59</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI* Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmGlowb</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">60</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmaGenYelow</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">61</td><td colspan="1" rowspan="1">PI*Q0 west/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not Likely atIncreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="7" rowspan="1">(ii)Mild accumulation allele* combinations</td></tr><tr><td colspan="1" rowspan="2">62</td><td colspan="1" rowspan="1">PI*S/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">63</td><td colspan="1" rowspan="1">PI*S/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmGlwb</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">64</td><td colspan="1" rowspan="1">PI*S/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmlow</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">65</td><td colspan="1" rowspan="1">PI*S/PI*Q0 clayton</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">12.1-87.09</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Rosenbaum et al., 2017</td><td colspan="1" rowspan="1">:1 individual with</td><td colspan="1" rowspan="1">and 1 without live</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">66</td><td colspan="1" rowspan="1">PI*S/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYamlwb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">67</td><td colspan="1" rowspan="1">PI*S/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYaYnmaGlob</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">68</td><td colspan="1" rowspan="1">PI*S/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">69</td><td colspan="1" rowspan="1">PI*S/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">70</td><td colspan="1" rowspan="1">PI*S/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">PI*I/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">72</td><td colspan="1" rowspan="1">PI*I/PI*I</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">73</td><td colspan="1" rowspan="1">PI*I/PI*S</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">74</td><td colspan="1" rowspan="1">PI*I/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">75</td><td colspan="1" rowspan="1">PI*I/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">76</td><td colspan="1" rowspan="1">PI*I/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">77</td><td colspan="1" rowspan="1">PI*I/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">78</td><td colspan="1" rowspan="1">PI*I/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">79</td><td colspan="1" rowspan="1">PI*I/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">80</td><td colspan="1" rowspan="1">PI*I/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">81</td><td colspan="1" rowspan="1">PI*I/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="6" rowspan="1">(iii)_ Severe accumulation allele combinations</td></tr><tr><td colspan="1" rowspan="2">82</td><td colspan="1" rowspan="1">PI*S iiyama/PI*M</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">83</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Ko et al., 2011: 1 indiv</td><td colspan="1" rowspan="1">idual without liver</td><td colspan="1" rowspan="1">disease; Miyahar</td><td colspan="1" rowspan="1">et al., 2001: 1 in</td><td colspan="1" rowspan="1">dividual withou</td><td colspan="1" rowspan="1">liver disease</td></tr><tr><td colspan="1" rowspan="2">84</td><td colspan="1" rowspan="1">PI*S iiyama/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">85</td><td colspan="1" rowspan="1">PI*S iiyama/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not repor</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">86</td><td colspan="1" rowspan="1">PI*S iiyamalPI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">PI*S iiyama/</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PI*P lowell</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">88</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">89</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYalwb</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">90</td><td colspan="1" rowspan="1">PI*S iiyamalPI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td colspan="3" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">91</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYalb</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">92</td><td colspan="1" rowspan="1">PI*S iiyama/PI*S</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">93</td><td colspan="1" rowspan="1">PI*S iiyama/PI*I</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmeYelowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">94</td><td colspan="1" rowspan="1">PI*S iiyamalPI*S iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">Takabe et al., 1992: 1 i</td><td colspan="1" rowspan="1">ndividual without</td><td colspan="1" rowspan="1">liver disease; Yua</td><td colspan="1" rowspan="1">a et al., 1993: 1 ir</td><td colspan="1" rowspan="1">dividual withou</td><td colspan="1" rowspan="1">t liver disease</td></tr><tr><td colspan="1" rowspan="2">95</td><td colspan="1" rowspan="1">PI*M/PI*M malton</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">46.2</td><td colspan="1" rowspan="1">31.2 - 61.8</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Ot al 200ndividuls with ivrdiseaseand  without iver isease;Corda e al00: wih liverisease and3without liver disease; Callea et al 018: individuals with liver disease;Canva e al 001:1individual with liver disease; Janciauskiene et al., 2004:  individual with liver disease (hepatitis and AATaccumulation); Figueira Gonçalves et al., 2017: 9 individuals without liver disease; Joly et al., 2015: 1individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">96</td><td colspan="1" rowspan="1">PI*M malton/PI*M malton</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">31.0 - 69.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">O al 00:individuals with livriseasean6without iverdisease;allea al 018ndiviualsiviurl lval wi ivie; Joyvial wiivdisease; Janciauskiene et al., 2004: 1 individual with liver disease; Reid et al., 1987: 1 with liver disease;Figueira Gonçalves et al., 2017: 2 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">97</td><td colspan="1" rowspan="1">PI*M malton/PI*Z</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">2.3 - 34.8</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Ferrrotti  al 20: 3 individuals, 1 with liver disease n 2 without liver disease; Sproule et al183:4ividuals without livrdisease; Joly  al 01:ndividuals without iver disease; Figueir Gonçalvesal., 2017: 1 individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">98</td><td colspan="1" rowspan="1">PI*M malton/PI*I</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="2" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">99</td><td colspan="1" rowspan="1">PI*M malton/PI*S iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">100</td><td colspan="1" rowspan="1">PI*M malton/PI*S</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Figueira Gonçalves et al., 2017: 1 individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">101</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 57.5</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Lara et al., 2013: 1 ind</td><td colspan="1" rowspan="1">vidual without liv</td><td colspan="1" rowspan="1">er disease; Balduy</td><td colspan="1" rowspan="1">ck et al., 2014: 1 i</td><td colspan="2" rowspan="1">ndividual without liver disease</td></tr><tr><td colspan="1" rowspan="2">102</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">103</td><td colspan="1" rowspan="1">PI*M malton/PI*F</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYamlb</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">104</td><td colspan="1" rowspan="1">PI*M malton/PI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">105</td><td colspan="1" rowspan="1">PI*M malton/PI*M procida</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">106</td><td colspan="1" rowspan="1">PI*M malton/PI*P lowell</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">107</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">108</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1">cYanmeYellow</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">EY</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">109</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">110</td><td colspan="1" rowspan="1">PI*ZIPI*M heerlen</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">111</td><td colspan="1" rowspan="1">PI*ZIPI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">14.3 - 72.8</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Ferrarotti et al., 2005:1 individual with liver disease and 1 without liver disease; Bornhorst et al., 2007: 1individual with liver disease; Holmes et al., 1990: 1 individual without liver disease; Bamforth &amp; Kalsheker1988: 1 individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">112</td><td colspan="1" rowspan="1">PPIlPPI*I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">18.2-81.8</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Baur &amp; Bencze 1987: 1 individual with liver disease; Mahadeva et al., 1999: 1 individual with liver disease;Ferrarotti et al., 2005: 2 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">113</td><td colspan="1" rowspan="1">PI*ZIPI*S iiyama</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">114</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 granite falls</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="1" rowspan="1">Clinical cases not reported</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">115</td><td colspan="1" rowspan="1">PPI*IPI*F</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">8.7 - 54.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Frs l 0ndividual with livisease; Kelly  l9ividual with livisase; Snet al., 2014: 6 individuals without liver disease</td></tr><tr><td colspan="1" rowspan="2">116</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 clayton</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="3" rowspan="1">Clinical cases not reported</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="2">117</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 bellingham</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0 - 73.0</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Poller et al., 1990: 1 individual without liver disease</td></tr><tr><td colspan="1" rowspan="2">118</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 mattawa</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">119</td><td colspan="1" rowspan="1">PI*ZIPI*Q0 west</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Clinical cases not reported</td></tr><tr><td colspan="1" rowspan="2">120</td><td colspan="1" rowspan="1">PI*ZIPI*M procida</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">12.1 - 87.9</td><td colspan="1" rowspan="1">Slightly atincreased risk</td></tr><tr><td colspan="6" rowspan="1">Lonardoet al 200ndividual with liver disease;Ferarotti etl 00:individual without liver disase</td></tr><tr><td colspan="7" rowspan="1">1 The percentage (%) and 95% confidence interval (C195%) of the reported clinical cases have been calculatedfollowing the score method described by Altman et al., 2000.2 The diffet cmiis f the PI*S and PI*Z allele (PI*M/PI*S, PI*M/PI*Z, PI*S/PI*Z, PI*S/PI*S and PI*Z/PI*Z) cover more than 95% of variant alleles identified in the SERPINA1 gene worldwide3 Non-accumulation variants: PI*M, PI*M procida, PI*F, PI*P lowel, PI*M heerlen, PI*Q0 granite falls, PI*Q0west, PI*Q0 bellingham, PI*Q0 mattawa, and PI*Q0 clayton.4 Mild accumulation variants: PI*I and PI*S.5 Severe accumulation variants: PI*S iiyama, PI*M malton, and PI*Z.</td></tr></table>

Exceptions to this general liver disease risk classification are shaded in grey in the table above and are applied to the following genetic results:

# i) Any combinations of a non-accumulation allele:

It is well-established that $\mathrm { P I } ^ { * } \mathbf { M }$ procida, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ granite falls, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ west, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ bellingham, $\mathrm { P I ^ { * } F }$ , $\mathrm { P I } ^ { * } \mathrm { P }$ lowell, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ mattawa, $\mathrm { P I } ^ { * } \mathrm { Q 0 }$ clayton, and $\mathrm { P I } ^ { * } \mathbf { M }$ heerlen variants do not cause AAT protein accumulation in the liver. Therefore, all the combinations including two non-accumulation alleles (refer to $\# 6$ to 59 in the ‘Liver Disease Risk Categorization’ table) will be reported with as ‘Not likely at increased risk’ result for liver disease.

# ii) Any combination with the severe accumulation alleles PI\*Z and PI\*M malton:

Due to the high amount of scientific evidence showing the severe accumulation of the mutant AAT protein in the liver triggered by the $\mathrm { P I } ^ { * } \mathrm { Z }$ and $\mathrm { P I } ^ { * } \mathbf { M }$ malton variants, as well as on the clinical data obtained from the high number of $( i )$ homozygous individuals (i.e., $\mathrm { P I ^ { * } Z / P I ^ { * } Z }$ and $\mathrm { P I } ^ { * } \mathbf { M }$ malton/PI\*M malton; refer to $\# 5$ and $\# 9 5$ in the ‘Liver Disease Risk Categorization’ table) and (ii) carrier individuals of these two variants that have been studied $( \mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } } Z$ and $\mathrm { P I ^ { * } M / P I ^ { * } M }$ malton; refer to $\# 2$ and $\# 9 4$ in the same table), there is enough scientific evidence supporting the classification of the compound heterozygous $( \mathrm { P I ^ { * } Z / P I ^ { * } M }$ malton) and the genetic results with at least one severe accumulation allele, $\mathrm { P I } ^ { * } \mathrm { Z }$ or $\mathrm { P I } ^ { * } \mathbf { M }$ malton (refer to $\# 9 6 \mathrm { - } \# 1 1 9$ in the same table), to the ‘Slightly at Increased risk’ category even though the available clinical data up to date does not show the same calculated risk e.g., ‘Unknown risk’ for $\mathrm { P I ^ { * } Z / P I ^ { * } I }$ with $50 \%$ reported clinical cases of lung disease and $9 5 \%$ CI: $1 8 . 2 \% { - } 8 1 . 8 \%$ (refer to #111) and ‘Not likely at increased risk’ for $\mathrm { P I ^ { * } Z / P I ^ { * } F }$ with $2 5 \%$ reported clinical cases of lung disease and $9 5 \%$ CI: $8 . 7 \mathrm { - } 5 4 \%$ (refer to #114).

Although the $\mathrm { P I } ^ { * } \mathrm { S }$ iiyama variant has also been widely described as a severe accumulation variant, different combinations of this variant are reported as ‘Unknown risk’ (following the general criteria), since no enough clinical data have been published for its combinations except for $\mathrm { P I ^ { * } Z / P I ^ { * } S }$ iiyama compound heterozygous (refer to #112 in the ‘Liver Disease Risk Categorization’ table), which is reported as ‘Slightly at increased risk’ based on the established evidence of the $\mathrm { P I } ^ { * } \mathrm { Z }$ variant.

The risk classification of lung disease and liver disease for each of the 121 possible genetic results of the AlphaID At Home Genetic Health Risk Service including ‘Variants not Determined’ is summarized in the table below. The most frequent genetic results are shaded in the table below.

<table><tr><td colspan="1" rowspan="1">#</td><td colspan="1" rowspan="1">Genetic Result in the AlphaIDAt Home Genetic Health RiskService Report</td><td colspan="1" rowspan="1">Numberofvariants</td><td colspan="1" rowspan="1">Reported Lung DiseaseRisk Category</td><td colspan="1" rowspan="1">Reported Liver DiseaseRisk Category</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PI*M/PI*M</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">Not likely at risk for AATD</td><td colspan="1" rowspan="1">Not likely at risk for AATD</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">PI*S/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk</td></tr><tr><td colspan="1" rowspan="2">3</td><td colspan="1" rowspan="2">PI*Z/PI*M</td><td colspan="1" rowspan="2">1</td><td colspan="1" rowspan="1">Not likely at increased risk1(never-smokers)</td><td colspan="1" rowspan="2">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">Slightly increased risk(ever-smokers)</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">PI*S/PI*S</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Not likely at increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">PI*S/ PI*Z</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">PI*Z/PI*Z</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">PI*I/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">PI*M procida/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">PI*M malton/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">PI*S iiyama/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">PI*Q0 west/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">PI*F/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">PI*P lowell/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">PI*M heerlen/PI*M</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Not likely at increased risk1</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">PI*Z/PI*I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">PI*Z/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">PI*Z/ PI*M malton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">PI*Z/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">PI*F/PI*Z</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">PI*Z/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">PI*Z/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">PI*Z /PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">PI*S/PI*I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">PI*S/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">PI*S/PI*M malton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">PI*S/ PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">PI*S/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">PI*S/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">PI*S/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">PI*F/PI*S</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">PI*S/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">PI*S/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">PI*S/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">PI*S/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">PI*I/PI*I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">PI*I/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">PI*I/PI*M malton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">PI*I/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">PI*I/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">PI*I/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">PI*I/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">PI*F/PI*I</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">PI*I/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">PI*I/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">PI*I/ PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">PI*I/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">PI*F/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">PI*F/PI*M malton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">PI*F/ PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">PI*F/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">PI*F/ PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">PI*F/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">PI*F/PI*F</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">PI*F/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">PI*F/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">PI*F/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">PI*F/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">PI*M procida/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">PI*M malton/PI*M procida</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly increased risk³</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">PI*M procida/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">PI*M procida/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">PI*M procida/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">PI*M procida/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">PI*M malton/PI*M malton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">PI*M malton/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">PI*M malton/ PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">PI*M malton/ PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">PI*M malton/ PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">PI*M malton/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">PI*M malton/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Slightly at increased risk3</td></tr><tr><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">PI*S iiyama/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">PI*P lowell/ PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">PI*S iiyama/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">PI*M Heerlen/PI*S iiyama</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Unknown risk</td></tr><tr><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">"PI*Q0 bellingham/PI*Q0 granite falls"</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">PI*Q0 granite falls/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 granite falls</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">PI*Q0 west/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">PI*P Lowell/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 west</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">109</td><td colspan="1" rowspan="1">PI*Q0 bellingham/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 bellingham</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">PI*P lowell/PI*P lowell</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">112</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">PI*P lowell/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">114</td><td colspan="1" rowspan="1">PI*P lowell/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">115</td><td colspan="1" rowspan="1">PI*Q0 mattawa/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">PI*Q0 clayton/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk²</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 mattawa</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">PI*Q0 clayton/ PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Increased risk2</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">119</td><td colspan="1" rowspan="1">PI*M heerlen/PI*Q0 clayton</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Unknown risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">PI*M heerlen/PI*M heerlen</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Slightly at increased risk</td><td colspan="1" rowspan="1">Not likely at increased risk4</td></tr><tr><td colspan="1" rowspan="1">121</td><td colspan="1" rowspan="1">Variant not determined</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Not determined risk</td><td colspan="1" rowspan="1">Not determined risk</td></tr><tr><td colspan="4" rowspan="1">  P o </td><td colspan="1" rowspan="1">for lung disease risk even if</td></tr></table>

insufficient number of clinical cases have been reported.

<table><tr><td>2Z/P*Nul n *NulP*Nul  ve nuber  clinical cases have beenreported. Nu varints:PI*Q0 granite falls, PI*Q0 west, PI*Q0 bellngham, *Q0 mattawa and PI*Q0 clayton.</td></tr><tr><td>3Inividul  as PI* P*Mmal veulativnts nheiais e Sirvseil c avbe o ris evesbel cs av beeeporNolleM,P*M </td></tr></table>

# i. User Comprehension Study

a. Saliva collection kit user study:

Refer to K212745 for the saliva collection device (ORAcollect $\textsuperscript { \textregistered }$ ·Dx OCD-100.014) instructions for use and to assess the ability of lay users to provide samples adequate for testing.

b. AlphaID At Home GHR Service user comprehension study:

# Objectives:

The user comprehension study for the AlphaID At Home GHR Service was conducted in a demographically diverse U.S. population of naïve users (e.g., age, gender, ethnicity, and education level) to assess comprehension of the GHR reports through a survey that covers the six comprehension domains (purpose of the test, other risk factors, limitations of the test, relevance of ethnicity, results of the test, and appropriate next steps) in a controlled online setting.

# Study Methods:

The study was implemented as described below:

Step 1: Quota-based sampling per U.S. Census was employed to recruit a diverse set of study participants from urban and rural settings across a wide geography based on their demographic variables in age, gender, ethnicity, and education level. Each of the four U.S. Census geographic regions are adequately represented by participants: Northeast, Midwest, South and West.

Step 2: Study participants were asked to review and confirm their agreement to an informed consent form.

Step 3: Eligible study participants who had consented were passed to a Pre-Interview web-based survey which was self-administered and took approximately 10 minutes to complete a Pre-Test user comprehension questionnaire. At this stage, participants were randomly assigned into study arms which determined which type of report participants received.

The substantive exclusion criteria and operational exclusion criteria listed in the table below were applied at each step when necessary and detailed reasons were documented.

<table><tr><td colspan="2">Substantive Exclusion Criteria (criteria identified about potential users&#x27; background)</td></tr><tr><td>1</td><td>Indicated that they are not naive to the test subject of the study, in that they:</td></tr><tr><td rowspan="3"></td><td>Have had direct to consumer genetic testing themselves Have had research or healthcare-based genetic testing themselves (including</td></tr><tr><td>genetic testing for AATD) Have a personal diagnosis of AATD Have significant medical and/or genetic education</td></tr><tr><td>Have had experience working: as a healthcare provider where, genetic testing could have been involved (i.e., as a nurse, O physician, physician&#x27;s assistant, or genetic counselor)</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Have a personal diagnosis of AATDHave significant medical and/or genetic educationHave had experience working:Oas a healthcare provider where, genetic testing could have been involved (i.e., as a nurse,physician, physician&#x27;s assistant, or genetic counselor)in the field of human genetics.Have been a customer, employee, or study participant of Progenika, Grifols, or SoundRocket</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Indicated they would never be interested in using a direct-to-consumer genetic test</td></tr><tr><td rowspan=1 colspan=2>Operational Exclusion Criteria (criteria identified before or during study implementation)</td></tr><tr><td></td><td></td></tr><tr><td rowspan=1 colspan=2>An individual was considered a refusal if they did not consent or withdrew consent to participate inhe study.</td></tr></table>

Step 4: Study participants who completed the last step were invited to participate in a live televideo interview (LTI). The LTI was administered via a video conferencing platform at a time scheduled individually between the participant and a study

moderator. The interview was scheduled for no more than 60 minutes total. Each participant was required to review the Education Module for approximately 5−10 minutes).and then the GHR Service Report (Your Result section) for approximately 15−20 minutes. Each participant was informed regarding accessibility to Supplemental Materials consisting of the “Table of Risk Categories”, “Table of Variants”, “Package Insert”, and “References”.

User comprehension was tested through a two-step process. First, participants’ comprehension was tested prior to viewing the Educational Module and GHR Reports. The Pre-Test questionnaire included basic demographics (for inclusion/ exclusion criteria) and baseline comprehension questions that covered the following comprehension domains: test purpose [PURPOSE], test limitations [LIMITATIONS], race/ethnicity [ETHNICITY], and other factors that may have an impact on the test results [OTHER FACTORS]. Second, on completion of reviewing a representative sample of all the material that is included in these documents, participants were provided with the Post-Test questionnaire to complete (approximately 20 minutes). The Post-Test questionnaire included comprehension questions designed to measure all six comprehension domains: test results [RESULTS] and appropriate action [NEXT STEPS] in addition to PURPOSE, LIMITATIONS, ETHNICITY, OTHER FACTORS. The RESULTS and NEXT STEPS domains were only included in the Post-Test questionnaire because they represented specific results that participants could not know prior to viewing the GHR reports and related materials. Each comprehension domain included at least two comprehension questions.

Throughout the completion of the study tasks, the Moderator was able to help the participant navigate between tabs or with technical difficulties as needed but did not give any instructions or answer any questions about how to interpret the report or answer questions. Both the participant's face and screen actions were recorded during this interview. Time on-task (length of time to complete questionnaire) was also recorded for each participant. Operational exclusion criteria were applied at this step when necessary and detailed reasons were documented.

Of the 15 total possible GHR Reports that can be generated by the AlphaID At Home GHR Service, five were selected for the User Comprehension Study. The selected Reports summarized in the table below cover all the possibilities for number of variants and risk categories associated with both lung disease and liver disease that can be reported by the AlphaID At Home GHR Service:

• Number of variants detected in the SERPINA1 gene: a total of four different reports can be obtained: 0, 1, or 2 variants detected, and ‘Variant not determined’.

• Risk category: four different risk categories (‘Not likely at increased risk’, ‘Slightly increased risk’, ‘Increased risk', and ‘Unknown risk) can be provided by the test, which represent the risk for developing lung and /or liver disease. Additionally, a ‘Not determined risk’ result can also be provided under certain circumstances (Variant not determined).

<table><tr><td rowspan=2 colspan=1>b</td><td rowspan=2 colspan=1>GHR Report</td><td rowspan=2 colspan=1>Number ofVariantDetected</td><td rowspan=1 colspan=2>Risk Category for</td></tr><tr><td rowspan=1 colspan=1>Lung Disease</td><td rowspan=1 colspan=1>Liver Disease*</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PI*M/PI*M</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Not likely at risk</td><td rowspan=1 colspan=1>Not likely at risk</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>PI*M/PI*Z</td><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Not likely at increased risk(non-smokers)</td><td rowspan=2 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>Slightly increased risk(ever smokers)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>PI*Z/PI*Z</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Increased risk</td><td rowspan=1 colspan=1>Slightly increased risk</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>PI*S/PI*I</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Unknown risk</td><td rowspan=1 colspan=1>Unknown risk</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Variant notdetermined</td><td rowspan=1 colspan=1>Variant notdetermined</td><td rowspan=1 colspan=1>Risk not determined</td><td rowspan=1 colspan=1>Risk not determined</td></tr><tr><td rowspan=1 colspan=5>* None of the genetic results currently provided by the Service yield an &quot;increased risk&quot; for liverdisease. The Sponsor indicated that there is no evidence that users think of the risk of liver vs. lungdisease to be substantially differently in the context of the GHR report and, therefore, testing usercomprehension of the &quot;increased risk&quot; concept with lung disease would be sufficient to extrapolatethese results to the comprehension of the concept of &quot;increased risk&quot; of liver disease.</td></tr></table>

The study was set up to randomly assign each eligible study participant (refer to Step 3) to one of five study arms identified by the five pre-selected test report types, with the following number of participants included per study arm as summarized in the table below.

<table><tr><td rowspan=1 colspan=1>StudyArm</td><td rowspan=1 colspan=1>GHRReport</td><td rowspan=1 colspan=1>TotalParticipants</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PI*M/PI*M</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>PI*M/PI*Z</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>PI*Z/PI*Z</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>PI*S/PI*I</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Variant not determined</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>525</td></tr></table>

Each GHR Report includes the following sections: ‘Your Result’ (Result Report), ‘Scientific Details’, ‘Frequently Asked Questions’ (FAQs), and ‘Glossary of Terms’. The GHR Reports and Supplemental Materials were reviewed by a Certified Genetic Counselor (GC) to ensure that the materials sufficiently cover the relevant general and variant-specific concepts [comprehension domains]. Prior to recruitment for the user comprehension study, the test report materials (GHR Report, Education Module, and Supplemental Materials) with content targeted no more difficult than $8 ^ { \mathrm { t h } }$ grade reading level were evaluated for readability. The Educational Module and Supplemental Materials generated to support the test result report were also evaluated for readability. The Education Module, as well as all Results Reports, Scientific Details, FAQs, and Glossary reached the $8 ^ { \mathrm { t h } }$ grade readability target. The Supplemental Information section (which includes Table of Risk Categories, Table of Variants, and Package Insert) achieved a $1 0 ^ { \mathrm { t h } }$ grade readability score overall which was within the targeted readability maximum for the level of technical detail and language required in this document.

Of the 768 study participants engaged (consented) in this study, 57 were determined to be ineligible during the Pre-Interview web-based survey administration. Of those 711 eligible participants in the study, 186 were ultimately excluded after the PreInterview web-based survey for a variety of reasons. This includes 18 cases that failed to complete the pre-survey in full. Of the 693 respondents who completed the presurvey and scheduled a live televideo interview (LTI), 133 either failed to show up or cancelled their scheduled LTI. Of the 560 cases that were present at their scheduled LTI, 35 cases were excluded due to operational exclusion criteria applied with moderator judgment during the LTI. The most common operational exclusion criterion applied (in 26 of these 35 cases) was "failed to complete LTI due to inadequate technological capabilities”. The other reasons for exclusion include: (i) lacking abilities to read the report (i.e., forgot glasses, are legally blind, etc.), (ii) lacking abilities to use a computer or tablet to complete the basic tasks of viewing a browser window, navigating a mouse to click survey responses, etc., (iii) failure to follow verbal and/or written instructions during the study protocol, (iv) failure to review test report; (v) failure to follow study instructions, and (vi) failure to complete the live televideo interview (LTI) which includes the POST survey within a 60- minute session, (vii) Failure of facilities outside of control of moderator or participant (i.e., power outage, Internet down for moderator or participant, etc.).

# Results:

Comprehension scores were calculated using the total number of responses received that were correct over the total number of eligible responses received. The calculated proportion for each domain serves as the comprehension score for that domain. The target score for each comprehension domain was $90 \%$ across all reports. A mean comprehension rate was calculated across all comprehension question domains and all reports. To evaluate whether the comprehension scores had significantly improved statistically, the pre- and post-test scores in each domain, and overall, were compared by using paired sample t-test. Results summarized in the table below show that each comprehension domain achieved a minimum of $9 0 . 1 \%$ or higher user comprehension score in the pre-test questionnaire, and $9 4 . 0 \%$ or higher user comprehension score in the post-test questionnaire, across all reports. The overall comprehension scores were of $9 2 . 7 \%$ and $9 6 . 8 \%$ across all comprehension domains and reports, for the pre-test and post-test questionnaire, respectively, exceeding the study goals of $90 \%$ comprehension overall. Results of the pre- to post-test comparison by paired sample t-tests showed a statistically significant improvement in the user comprehension scores $\left( \mathbf { p } < 0 . 0 0 0 1 \right)$ between the pre-and post-test questionnaire for the Education Module, and the domains of ‘Purpose’, ‘Limitations’, and ‘Ethnicity’ as well as overall (all report domains). Statistical significance demonstrates that the increase in user comprehension score was a result of the labeling. The ‘Other Factors’ domain which was the highest Pre-Test score of all the domains showed no significant difference $( \mathbf { p } = 0 . 6 6 2 )$ ) because the score remained effectively the same from pre-test $( 9 8 . 0 \% )$ to post-test $( 9 7 . 7 \% )$ . The results showed that the GHR reports had excellent comprehension of the Service’s purpose, limitations, results, relevance of ethnicity, other factors that may impact test results, and appropriate next steps.

<table><tr><td rowspan=1 colspan=5>User Comprehension Pre- to Post-Test Scores for all GHR Reports by Domain</td></tr><tr><td rowspan=1 colspan=1>Domain Name</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Pre-Test(%)*</td><td rowspan=1 colspan=1>Post-Test(%)*</td><td rowspan=1 colspan=1>%Improvement</td></tr><tr><td rowspan=1 colspan=1>All ReportDomains</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=1 colspan=1>PurposeDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>90.1</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>7.1</td></tr><tr><td rowspan=1 colspan=1>Other FactorsDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>-0.3</td></tr><tr><td rowspan=1 colspan=1>LimitationsDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>90.6</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=1 colspan=1>EthnicityDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>4.5</td></tr><tr><td rowspan=1 colspan=1>ResultsDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Next StepsDomain</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=5>* The calculation (for each domain) is based on: Sum of all correct responses for all domainitems for all individuals / Total count of all questions asked for all domain items for allindividuals</td></tr></table>

When analyzed individually for each of the five GHR reports, the post-test comprehension scores for each domain are $9 1 . 7 \%$ or higher, result domain in the $\mathrm { P I ^ { * } M } / \mathrm { P I ^ { * } Z }$ report has a post-test score of 91.7.

<table><tr><td rowspan=1 colspan=7>Comprehension Rates (%) by GHR Report Type</td></tr><tr><td rowspan=1 colspan=1>DomainName</td><td rowspan=1 colspan=1>PI*M/PI*M</td><td rowspan=1 colspan=1>PI*M/PI*Z</td><td rowspan=1 colspan=1>PI*Z/PI*Z</td><td rowspan=1 colspan=1>PI*S/PI*I</td><td rowspan=1 colspan=1>Result NotDetermined</td><td rowspan=1 colspan=1>OverallComprehensionRates (%)</td></tr><tr><td rowspan=1 colspan=1>PurposeDomain</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>97.2</td></tr><tr><td rowspan=1 colspan=1>Other FactorsDomain</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>97.7</td></tr><tr><td rowspan=1 colspan=1>LimitationsDomain</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>97.2</td></tr><tr><td rowspan=1 colspan=1>EthnicityDomain</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>97.8</td></tr><tr><td rowspan=1 colspan=1>ResultsDomain</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>94.0</td></tr><tr><td rowspan=1 colspan=1>Next StepsDomain</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>99.1</td></tr></table>

The user comprehension scores were also analyzed by demographic category and results for the overall study (all five GHR reports combined) are presented in the table below. All demographic categories (including age, sex, race/ethnicity, education, and geographic region) scored $9 5 . 3 \%$ or higher on the post-test questionnaire overall.

<table><tr><td rowspan=1 colspan=5>User Comprehension Pre- and Post-Test Scores for all Reports byDemographic</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Pre-Test</td><td rowspan=1 colspan=2>Post-Test</td></tr><tr><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td></tr><tr><td rowspan=1 colspan=5>Overall n</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>525</td></tr><tr><td rowspan=1 colspan=5>Demographics</td></tr><tr><td rowspan=1 colspan=5>Age</td></tr><tr><td rowspan=1 colspan=1>18-39</td><td rowspan=1 colspan=1>92.7</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>203</td></tr><tr><td rowspan=1 colspan=1>40-59</td><td rowspan=1 colspan=1>92.1</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>172</td></tr><tr><td rowspan=1 colspan=1>60+</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=5>Sex</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>92.0</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>266</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>259</td></tr><tr><td rowspan=1 colspan=5>Race/Ethnicity</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>377</td></tr><tr><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>89.5</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>96.1</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=5>Education</td></tr><tr><td rowspan=1 colspan=1>HS* or Less</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>181</td></tr><tr><td rowspan=1 colspan=1>Some College</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Bachelor andHigher</td><td rowspan=1 colspan=1>96.1</td><td rowspan=1 colspan=1>176</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=5>Geographic Region</td></tr><tr><td rowspan=1 colspan=1>Midwest</td><td rowspan=1 colspan=1>90.8</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>116</td></tr><tr><td rowspan=1 colspan=1>Northeast</td><td rowspan=1 colspan=1>92.3</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>127</td></tr><tr><td rowspan=1 colspan=1>South</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=1>West</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=5>*HS: High School</td></tr></table>

The following table summarizes the user comprehension scores (post-test only) broken out by demographic category (age, sex, race/ethnicity, education level, and geographic region) per domain for the overall study (all five GHR reports combined). Every demographic category scored $9 1 . 5 \%$ or higher in every domain across all report types. Results showed that participant demographics were unrelated to user comprehension.

<table><tr><td rowspan=1 colspan=13>User Comprehension Post-Test Scores for all Reports by Demographic</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>PurposeDomain</td><td rowspan=1 colspan=2>FactorsDomain</td><td rowspan=1 colspan=2>LimitationsDomain</td><td rowspan=1 colspan=2>EthnicityDomain</td><td rowspan=1 colspan=2>ResultsDomain</td><td rowspan=1 colspan=2>Next StepsDomain</td></tr><tr><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Mean(%)</td><td rowspan=1 colspan=1>n</td></tr><tr><td rowspan=1 colspan=13>Overall n</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>525</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>525</td></tr><tr><td rowspan=1 colspan=13>Demographics</td></tr><tr><td rowspan=1 colspan=13>Age (group)</td></tr><tr><td rowspan=1 colspan=1>18-39</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>203</td></tr><tr><td rowspan=1 colspan=1>40-59</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>93.2</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>172</td></tr><tr><td rowspan=1 colspan=1>60+</td><td rowspan=1 colspan=1>96.7</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>91.5</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=8>Sex</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>93.0</td><td rowspan=1 colspan=1>266</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>266</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>95.1</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>259</td></tr><tr><td rowspan=1 colspan=13>Race</td></tr><tr><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>377</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>377</td></tr><tr><td rowspan=1 colspan=1>Black</td><td rowspan=1 colspan=1>95.8</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>95.1</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>94.4</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>91.9</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>All Other</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>94.8</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=13>Ethnicity</td></tr><tr><td rowspan=1 colspan=1>Hispanic</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>96.1</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>94.7</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=13>Education</td></tr><tr><td rowspan=1 colspan=1>High Schoolor Less</td><td rowspan=1 colspan=1>94.8</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>94.2</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>181</td></tr><tr><td rowspan=1 colspan=1>SomeCollege</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>93.6</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>168</td></tr><tr><td rowspan=1 colspan=1>Bachelors(4-year)</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Graduate/ProfessionalDegree</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>76</td></tr><tr><td rowspan=1 colspan=13>Geographic Region</td></tr><tr><td rowspan=1 colspan=1>Midwest</td><td rowspan=1 colspan=1>96.0</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>116</td></tr><tr><td rowspan=1 colspan=1>Northeast</td><td rowspan=1 colspan=1>96.3</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>96.1</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>96.9</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>127</td></tr><tr><td rowspan=1 colspan=1>South</td><td rowspan=1 colspan=1>98.4</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>93.7</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=1>West</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>93.8</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>120</td></tr></table>

In evaluating potential bias due to study participant nonresponse and exclusion based on eligibility in the Pre-Interview web-based survey or based on exclusion criteria applied after participants were in the study, the demographic make-up of the Completed (Total Participants) group was not different from that of the Target Population suggesting that the desired goal of reaching a diverse population of study participants that reflect the diversity of the U.S. on gender, race/ethnicity, education, and age was met. Even though ‘Education’ showed the largest difference between the Target Population and the final set of study participants, each education category (including high school or less) individually scored well above the $90 \%$ comprehension threshold both overall and on each domain. No significant differences between the categories of the Study Engaged (consented)

participants on ‘Age’, ‘Race/Ethnicity’, and ‘Geographic Region’ $\scriptstyle \mathbf { p } < 0 . 0 0 1 ,$ ) were noted. Though the impact of differential exclusion on ‘Sex’ $( \mathtt { p } ^ { = } 0 . 0 0 2 )$ and ‘Education’ $( \mathrm { p } { < } 0 . 0 0 1 )$ was statistically significant, the final set of study participants matched the target demographics. The differences in ‘Sex’ between the categories of Study Engaged participants did not impact user comprehension in this study as both males and females scored well above the $90 \%$ user comprehension threshold, both overall and in each domain. With ‘Education’, no statistical differences were found in user comprehension scores between participants with some college and those with a bachelor’s degree or higher. While the differences between those with high school/GED or less and those with a bachelor’s degree or higher $( \mathbf { p } { < } 0 . 0 0 1 )$ ) and those with high school/GED or less and those with some college $\scriptstyle \mathbf { p } = 0 . 0 1 8$ ) are significant, the span of $2 . 7 \%$ difference from the lowest to highest education category, all above the $9 5 \%$ threshold, is not meaningful. In sum, the demographic make-up of the potential bias in this study relating to nonresponse/exclusion is negligible and did not have a substantive impact on these study results.

<table><tr><td colspan="10" rowspan="1">Study Participants by Demographics</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">*TargetPopulation</td><td colspan="2" rowspan="1">StudyEngaged(Consented)</td><td colspan="2" rowspan="1">Excluded forEligibility inPre-Interview</td><td colspan="2" rowspan="1">Excluded forEligibility afterPre-Interview</td><td colspan="2" rowspan="1">Completed(TotalParticipants)</td></tr><tr><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">%</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">%</td></tr><tr><td colspan="10" rowspan="1">Overall n</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">768</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">186</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">525</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="10" rowspan="1">Demographics</td></tr><tr><td colspan="10" rowspan="1">Age</td></tr><tr><td colspan="1" rowspan="1">18-39**</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">304</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">39</td></tr><tr><td colspan="1" rowspan="1">40-59</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">255</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">172</td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">60+</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">150</td><td colspan="1" rowspan="1">28</td></tr><tr><td colspan="1" rowspan="1">Sex</td><td colspan="2" rowspan="1">cYanmaGlowb</td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">422</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">121</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">266</td><td colspan="1" rowspan="1">51</td></tr><tr><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">346</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">259</td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="1">Race/Ethnicity</td><td colspan="3" rowspan="1">cYanmaGellowb</td><td colspan="3" rowspan="1"></td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">538</td><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">377</td><td colspan="1" rowspan="1">72</td></tr><tr><td colspan="1" rowspan="1">Other</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">230</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">148</td><td colspan="1" rowspan="1">28</td></tr><tr><td colspan="1" rowspan="1">Education</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">High Schoolor Less</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">329</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">124</td><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">181</td><td colspan="1" rowspan="1">35</td></tr><tr><td colspan="1" rowspan="1">Some College</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">219</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">32</td></tr><tr><td colspan="1" rowspan="1">Bachelor andHigher</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">220</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">176</td><td colspan="1" rowspan="1">34</td></tr><tr><td colspan="1" rowspan="1">Geographic Region***</td><td colspan="1" rowspan="1">on***</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Midwest</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">172</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">Northeast</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">182</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">South</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">159</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">West</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">176</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">120</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="10" rowspan="1">* Based on 2019 Estimates from Census Factfinder** U.S. Census data is categorized using a lower limit of age 19. Inclusion criteria in this studyallowed any individual aged 18 or older to participate.</td></tr><tr><td></td></tr><tr><td colspan="2">***Geographic region was not sampled to match the U.S. population. Rather, it was monitored to ensure that we obtained approximately equal groups of participants from a wide geography within the</td></tr><tr><td>United States. As such, there are no target population figures for this category.</td></tr></table>

c. Frequently Asked Questions (FAQ) Material.

A FAQ section was developed and included in each GHR Service Report. The FAQ section was to provide users information to adequately understand the purpose of the test, limitations of the test, meaning of the results of the test, relevance of race and ethnicity on test results, how the results of the test may affect the user's family and children, other risks factors that contribute to disease and appropriate follow-up procedures. Each GHR Report has answers to the FAQs that are specific to the variant(s) of the SERPINA1 gene where applicable. The questions included in the FAQ section for the test include, but are not limit to:

<table><tr><td rowspan=1 colspan=1> What is alpha-1 antitrypsin deficiency (AATD)?</td></tr><tr><td rowspan=1 colspan=1>What is the Service for?</td></tr><tr><td rowspan=1 colspan=1>How accurate is the Service?</td></tr><tr><td rowspan=1 colspan=1>My report says 1 have &quot;2 variants&quot; in the SERPINA1 gene linked to AATD. How didI get these variants?*</td></tr><tr><td rowspan=1 colspan=1>My report says I have &quot;2 variants, Pl*Z and Pl*Z&quot;. What are some next steps I cantake?*</td></tr><tr><td rowspan=1 colspan=1>• What does &quot;at increased risk of developing lung disease linked to AATD&quot;compared to the general population&quot; mean?*</td></tr><tr><td rowspan=1 colspan=1>• What does &quot;at slightly increased risk of developing liver disease linked to AATDcompared to the general population&quot; mean?*</td></tr><tr><td rowspan=1 colspan=1>•Can AATD be treated?</td></tr><tr><td rowspan=1 colspan=1>•What are the limitations of the Service?</td></tr><tr><td rowspan=1 colspan=1>•How can your genetic result affect your family?*</td></tr><tr><td rowspan=1 colspan=1>•My report says AATD is most common in people of European descent. What if•I&#x27;m not of European descent?</td></tr><tr><td rowspan=1 colspan=1>Where can I learn more about AATD?</td></tr><tr><td rowspan=1 colspan=1>*These FAQs depend on the GHR report type</td></tr></table>

d. User Opt-In page: Not applicable

D Clinical Cut-Off: Not applicable   
E Expected Values/Reference Range: Not applicable

VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.