# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY

I Background Information:

A 510(k) Number K241220   
B Applicant Roche Diagnostics Operations   
C Proprietary and Established Names Tina-quant Lipoprotein(a) Gen.2 Molarity   
D Regulatory Information

<table><tr><td rowspan=1 colspan=1>ProductCode(s)</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>DFC</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.5600 - Low-Density LipoproteinImmunological TestSystem</td><td rowspan=1 colspan=1>IM - Immunology</td></tr></table>

II Submission/Device Overview:

A Purpose for Submission: New device   
B Measurand: Lipoprotein (a) [Lp(a)]   
C Type of Test: Quantitative immunoturbidimetric assay

# III Intended Use/Indications for Use:

A Intended Use(s):

See Indications for Use below.

# B Indication(s) for Use:

The Tina-quant Lipoprotein (a) Gen.2 Molarity assay is an in vitro test for the quantitative determination of lipoprotein (a) $\left[ \mathrm { L p } ( \mathrm { a } ) \right]$ in human serum and plasma on cobas c systems. The measurement of Lp(a) is useful in evaluating lipid metabolism disorders and assessing atherosclerotic cardiovascular disease risk, when used in conjunction with clinical evaluation and other lipoprotein tests.

C Special Conditions for Use Statement(s): Rx - For Prescription Use Only   
D Special Instrument Requirements: cobas c 503 analyzer

# IV Device/System Characteristics:

# A Device Description:

The Tina-quant Lipoprotein (a) Gen.2 Molarity assay consists of the following reagents:

• Glycine buffer $\mathrm { 1 7 0 m m o l / L }$ , $\mathrm { p H } 7 . 0 \}$ containing stabilizers, bovine serum albumin (BSA), $0 . 1 \%$ rabbit serum, and preservative.

• Latex particles coated with polyclonal rabbit anti-human $\operatorname { L p } ( { \mathsf { a } } )$ antibodies $( 0 . 5 \% )$ containing glycine buffer (170 mmol/L, pH 7.3), BSA, and preservative.

# B Principle of Operation:

The Tina-quant Lipoprotein (a) Gen.2 Molarity assay is a particle enhanced immunoturbidimetric assay. Human Lp(a) agglutinates with latex particles coated with anti-Lp(a) antibodies. The precipitate is determined turbidimetrically at $8 0 0 / 6 6 0 \mathrm { n m }$ .

# V Substantial Equivalence Information:

A Predicate Device Name(s): cobas c Tina-quant Lipoprotein (a) Gen.2 assay   
B Predicate 510(k) Number(s): K122722

C Comparison with Predicate(s):   

<table><tr><td rowspan=1 colspan=1>Device &amp; PredicateDevice(s):</td><td rowspan=1 colspan=1>K241220</td><td rowspan=1 colspan=1>K122722</td></tr><tr><td rowspan=1 colspan=1>Device Trade Name</td><td rowspan=1 colspan=1>Tina-quant Lipoprotein(a) Gen.2 Molarity</td><td rowspan=1 colspan=1>cobas c Tina-quantLipoprotein (a) Gen.2assay</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Similarities</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Intended for thequantitativedetermination of Lp(a)for the evaluation oflipid metabolismdisorders andassessment ofatheroscleroticcardiovascular disease,when used inconjunction withclinical evaluation andother lipoprotein tests.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay method</td><td rowspan=1 colspan=1>Particle enhancedimmunoturbidimetricassay</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Serum, plasma</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>General DeviceCharacteristic Differences</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lp(a) units of measurement</td><td rowspan=1 colspan=1>nmol/L</td><td rowspan=1 colspan=1>mg/dL</td></tr><tr><td rowspan=1 colspan=1>Analytical measuringinterval</td><td rowspan=1 colspan=1>7 to 240 nmol/L</td><td rowspan=1 colspan=1>6 to 80 mg/dL</td></tr><tr><td rowspan=1 colspan=1>Traceability</td><td rowspan=1 colspan=1>Standardized against theIFCC reference materialSRM2B</td><td rowspan=1 colspan=1>Standardized against aninternal referencematerial</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>cobas c 503</td><td rowspan=1 colspan=1>cobas c 501</td></tr></table>

# VI Standards/Guidance Documents Referenced:

• CLSI EP05-A3 (Reaffirmed: September 2019). Evaluation of Precision of Quantitative Measurement Procedures; Approved Guideline – Third Edition. CLSI EP06 $2 ^ { \mathrm { n d } }$ Edition. Evaluation of the Linearity of Quantitative Measurement Procedures. CLSI EP07 $3 ^ { \mathrm { r d } }$ Edition. Interference Testing in Clinical Chemistry. CLSI EP17-A2. Evaluation of Detection Capability for Clinical Laboratory Measurement Procedures; Approved Guideline – Second Edition.

# A Analytical Performance:

# 1. Precision/Reproducibility:

A precision study to assess repeatability and within-laboratory precision was conducted according to CLSI EP05-A3. Five human serum samples and two controls were run in replicates of two per run, with two runs per day, over 21 days on the cobas c 503 analyzer using three reagent lots. The three reagent lots performed similarly. Results from a representative lot are shown below.

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Mean Lp(a) (nmol/L)</td><td rowspan=1 colspan=2>Repeatability</td><td rowspan=1 colspan=2>Within-laboratory precision</td></tr><tr><td rowspan=1 colspan=1>SD (nmol/L)</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD (nmol/L)</td><td rowspan=1 colspan=1>CV (%)</td></tr><tr><td rowspan=1 colspan=1>Control 1</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.368</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.562</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>Control 2</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>0.389</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.997</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Serum 1</td><td rowspan=1 colspan=1>19.7</td><td rowspan=1 colspan=1>0.414</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.508</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>Serum 2</td><td rowspan=1 colspan=1>49.2</td><td rowspan=1 colspan=1>0.423</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.654</td><td rowspan=1 colspan=1>1.3</td></tr><tr><td rowspan=1 colspan=1>Serum 3</td><td rowspan=1 colspan=1>80.7</td><td rowspan=1 colspan=1>0.421</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.727</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Serum 4</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0.546</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.981</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Serum 5</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>0.577</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>0.5</td></tr></table>

An additional precision study was conducted per CLSI EP05-A3 using samples with known apolipoprotein a (apo(a)) isoform sizes (as determined by SDS agarose gel electrophoresis and Western blot analysis). Five human serum sample pools were tested in two replicates, two runs per day for $\geq 2 1$ days on the cobas c 503 analyzer using three lots of reagent.

Results support that precision is not dependent on apo(a) isoform size.

2. Linearity:

# Linearity studies

A linearity study was conducted to demonstrate that measurements across the claimed assay reportable range are linear. The study was conducted according to CLSI $\mathrm { E P 0 6 2 ^ { n d } }$ Edition. Dilution series were prepared from either a human serum sample or a human $\mathrm { K } _ { 3 }$ EDTA plasma sample containing a high concentration of Lp(a) diluted with $0 . 9 \%$ NaCl to create 15 levels for serum (ranging from ${ \sim } 3$ to $2 6 3 \mathrm { n m o l / L }$ ) and plasma (ranging from ${ \sim } 3$ to 249 nmol/L). The sets of dilutions were run on one cobas c 503 analyzer in one run using five replicates per level and three reagent lots for serum samples or one reagent lot for plasma samples. Linear regression was conducted for each dilution series and the deviation from linearity was calculated. The linear regression results are shown below.

<table><tr><td rowspan=1 colspan=1>Sample type, Lot</td><td rowspan=1 colspan=1>Linear regression equation</td><td rowspan=1 colspan=1>R^{2$</td></tr><tr><td rowspan=1 colspan=1>Serum, Lot 1</td><td rowspan=1 colspan=1>y= 1.044x</td><td rowspan=1 colspan=1>0.999</td></tr><tr><td rowspan=1 colspan=1>Serum, Lot 2</td><td rowspan=1 colspan=1>y = 1.022x</td><td rowspan=1 colspan=1>1.000</td></tr><tr><td rowspan=1 colspan=1>Serum, Lot 3</td><td rowspan=1 colspan=1>y=1.031x</td><td rowspan=1 colspan=1>0.999</td></tr><tr><td rowspan=1 colspan=1>Plasma, Lot 1</td><td rowspan=1 colspan=1>y= 1.020x</td><td rowspan=1 colspan=1>0.999</td></tr></table>

An additional linearity study was conducted according to CLSI $\mathrm { E P 0 6 2 ^ { n d } }$ Edition to assess linearity in serum samples with known apo(a) isoform sizes. A dilution series was prepared from a high Lp(a) concentration human serum (with small apo(a) isoform size of 19/20 kringle IV (KIV) repeats) and low Lp(a) concentration human serum (with large apo(a) isoform size of 35 KIV repeats) to create 15 levels (ranging from ${ \sim } 6$ to $2 5 2 \mathrm { n m o l / L } )$ . The dilution series was run on one cobas c 503 analyzer in one run using four replicates per level and one reagent lot. The linear regression results support that linearity is not dependent on apo(a) isoform size.

The linearity studies support the claimed measuring interval of $7 - 2 4 0 \ \mathrm { n m o l / L } .$

# Dilution study

A dilution study was conducted to validate the automatic rerun function on the cobas c 503 analyzer. Three native human serum samples with concentrations spanning the extended measuring interval (based on a validated Lp(a) ELISA assay that measures Lp(a) in nmol/L) were diluted manually and automatically (via the automatic rerun function) using a 1:3 dilution. Each sample was measured in replicates of three with the Tina-quant Lipoprotein (a) Gen.2 Molarity assay on the cobas c 503 analyzer. The results support the automatic rerun function on the cobas c 503 analyzer.

# 3. Analytical Specificity/Interference:

# Endogenous interference

An endogenous interference study was conducted to assess whether endogenous substances interfere with assay results. Two human serum pools with two different Lp(a) concentrations (low level ranged from 24.9 to $4 3 . 6 \mathrm { n m o l / L }$ ; high level ranged from 77.6 to $1 2 3 \mathrm { n m o l / L }$ ) were split into two parts with one part spiked with potential interferent (interference pool) and the other part spiked with solvent (dilution pool). A dilution series of 11 steps was prepared by mixing the interference pools and related dilution pools. Each dilution step was measured in 5 replicates according to CLSI EP07 $3 ^ { \mathrm { r d } }$ Edition and the mean value was used to calculate the difference between matched samples with or without interferent. The observed maximum difference was $< \pm 4 . 5 \ \mathrm { n m o l / L }$ for the low level Lp(a) sample and $< 7 . 5 \%$ for the high level Lp(a) sample at the following levels of interferent:

<table><tr><td rowspan=2 colspan=1>Substance</td><td rowspan=1 colspan=2>Highest level of substance at which no interference wasobserved (mg/dL unless otherwise indicated)</td></tr><tr><td rowspan=1 colspan=1>Low level Lp(a) sample</td><td rowspan=1 colspan=1>High level Lp(a) sample</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>7370</td><td rowspan=1 colspan=1>6880</td></tr><tr><td rowspan=1 colspan=1>Conjugated bilirubin</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Hemoglobin</td><td rowspan=1 colspan=1>1212</td><td rowspan=1 colspan=1>1200</td></tr><tr><td rowspan=1 colspan=1>Immunoglobulin G</td><td rowspan=1 colspan=1>6640</td><td rowspan=1 colspan=1>6700</td></tr><tr><td rowspan=1 colspan=1>Intralipid</td><td rowspan=1 colspan=1>2470</td><td rowspan=1 colspan=1>2229</td></tr><tr><td rowspan=1 colspan=1>Rheumatoid Factor</td><td rowspan=1 colspan=1>1253 IU/mL</td><td rowspan=1 colspan=1>1330 IU/mL</td></tr><tr><td rowspan=1 colspan=1>Unconjugated bilirubin</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>77</td></tr></table>

# Exogenous interference

An exogenous interference study was conducted to assess whether the presence of common drugs substances interferes with assay results. Two human serum pools with two different

Lp(a) concentrations (low level ranged from 38 to $4 7 \mathrm { n m o l / L }$ ; high level ranged from 115 to 149 nmol/L) were split into two parts with one part spiked with potential interferent and the other part spiked with solvent. Each part was measured in 5 replicates and the mean value was used to calculate the difference between matched samples with or without interferent. The observed maximum percent difference was $\leq 1 . 5 \%$ at the following levels of interferent:

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Highest level of substance atwhich no interference wasobserved (mg/dL)</td></tr><tr><td rowspan=1 colspan=1>N-Acetylcysteine</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Acetylsalicylic acid</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Ampicillin-Na</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>Ascorbic acid</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td rowspan=1 colspan=1>Cefoxitin</td><td rowspan=1 colspan=1>660</td></tr><tr><td rowspan=1 colspan=1>Doxycycline</td><td rowspan=1 colspan=1>1.8</td></tr><tr><td rowspan=1 colspan=1>Heparin</td><td rowspan=1 colspan=1>3300 IU/L</td></tr><tr><td rowspan=1 colspan=1>Levodopa</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>Methyldopa</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>12.3</td></tr><tr><td rowspan=1 colspan=1>Rifampicin</td><td rowspan=1 colspan=1>4.8</td></tr><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>15.6</td></tr><tr><td rowspan=1 colspan=1>Cyclosporine</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>21.9</td></tr><tr><td rowspan=1 colspan=1>Theophylline</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Phenylbutazone</td><td rowspan=1 colspan=1>32.1</td></tr></table>

# Cross-reactivity

A cross-reactivity study was conducted to assess whether substances structurally related to Lp(a) cross-react with the assay. Two human serum pools with two different Lp(a) concentrations (low level ranged from 41 to $4 9 \mathrm { n m o l / L } \mathrm { L p ( a ) }$ ; high level ranged from 112 to $1 1 7 { \mathrm { n m o l } } / \mathrm { L } \mathrm { L p } ( \mathrm { a } ) )$ were split into two parts with one part spiked with potential cross-reactant and the other part not spiked with cross reactant. Each part was measured in 5 replicates and the mean values were used to determine the absolute difference between matched samples with or without potential cross-reactant. Percent cross reactivity was then calculated using the following formula: ( $1 0 0 ^ { * }$ absolute difference) / cross-reactant concentration. No crossreactivity was detected for apolipoprotein B at $2 2 3 \mathrm { m g / d L }$ or plasminogen at $1 6 0 \mathrm { m g / d L }$ at either level of Lp(a).

# High dose hook effect

A high dose hook effect study was conducted to confirm that no false results will be reported due to a high dose hook effect. Two native human serum samples containing high concentrations of Lp(a) were diluted with $0 . 9 \%$ NaCl to create 16 levels. Each level was measured in triplicate on one cobas c 503 analyzer and three reagent lots of Tina-quant Lipoprotein (a) Gen.2 Molarity assay. The results support the labeling claim of no false results up to an Lp(a) concentration of $4 5 0 \mathrm { n m o l / L }$ .

4. Assay Reportable Range:

The reportable range of the assay is $7 - 2 4 0 \ \mathrm { n m o l / L } .$

5. Traceability, Stability, Expected Values (Controls, Calibrators, or Methods):

Traceability   
The Tina-quant Lipoprotein (a) Gen.2 Molarity assay is traceable to World Health Organization (WHO)/International Federation of Clinical Chemistry and Laboratory Medicine (IFCC) international reference material SRM2B.

# Sample stability

Serum with gel separator, lithium heparin plasma, K2EDTA plasma and K3EDTA plasma were collected from 50-56 donors. The samples were tested with the Tina-quant Lipoprotein (a) Gen.2 Molarity assay at baseline time zero and again after storage for several time points at different temperatures. The sample stability study results support the following labeling claims:

Stability in serum with gel separator, K2EDTA plasma and K3EDTA plasma:

5 days at $1 5 { - } 2 5 ^ { \circ } \mathrm { C }$ • 20 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ • 13 weeks at $- 2 0 ^ { \circ } \mathrm { C } \left( \pm 5 ^ { \circ } \mathrm { C } \right)$ 1 year at $- 7 0 ^ { \circ } \mathrm { C } \left( \pm 5 ^ { \circ } \mathrm { C } \right)$

Freeze only once.

Stability in lithium heparin plasma:

1 day at $1 5 . 2 5 ^ { \circ } \mathrm { C }$ • 5 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ • 9 weeks at $- 2 0 ^ { \circ } \mathbf { C } \left( \pm 5 ^ { \circ } \mathbf { C } \right)$ 1 year at $- 7 0 ^ { \circ } \mathrm { C } \left( \pm 5 ^ { \circ } \mathrm { C } \right)$

Freeze only once.

6. Detection Limit:

Limit of blank (LoB), limit of detection (LoD), and limit of quantitation (LoQ) studies were conducted based on CLSI EP17-A2.

For determination of LoB, one analyte-free saline $( 0 . 9 \% \mathrm { N a C l } )$ sample was measured with three reagent lots in six runs, with ten replicates per run, over three days on the cobas c 503 analyzer for a total of 60 determinations per lot. The LoB was determined as the $9 5 ^ { \mathrm { t h } }$ percentile of the 60 determinations.

For determination of LoD, five human serum samples with low Lp(a) concentrations were measured with three reagent lots in six runs, with two replicates per sample per run, over five days on the cobas c 503 analyzer for a total of 60 determinations per lot. The LoD was

determined as the lowest amount of analyte in a sample that can be detected with $9 5 \%$ probability.

For determination of LoQ, five human serum samples with low Lp(a) concentrations were measured with three reagent lots in five runs, with five replicates per sample per run, over five days on the cobas c 503 analyzer for a total of 25 determinations per sample per lot. The LoQ was determined as the lowest concentration of analyte that can be quantified with an intermediate precision of no more than $20 \%$ CV.

The results of the studies are shown below.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Lot 1(nmol/L)</td><td rowspan=1 colspan=1>Lot 2(nmol/L)</td><td rowspan=1 colspan=1>Lot 3(nmol/L)</td><td rowspan=1 colspan=1>Labeling claim(nmol/L)</td></tr><tr><td rowspan=1 colspan=1>LoB</td><td rowspan=1 colspan=1>0.326</td><td rowspan=1 colspan=1>0.473</td><td rowspan=1 colspan=1>0.267</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>0.713</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>1.51</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>LoQ</td><td rowspan=1 colspan=1>5.00</td><td rowspan=1 colspan=1>4.81</td><td rowspan=1 colspan=1>4.44</td><td rowspan=1 colspan=1>7</td></tr></table>

7. Assay Cut-Off:

Not applicable.

# B Comparison Studies:

1. Method Comparison with Predicate Device:

Method comparison with Lp(a) apo(a) size insensitive ELISA assay

A method comparison study was conducted with the Tina-quant Lipoprotein (a) Gen.2 Molarity assay on the cobas c 503 analyzer and an Lp(a) apo(a) size insensitive ELISA assay measuring Lp(a) in nmol/L. One hundred and twenty-six (126) native human serum samples were tested in singlicate on the candidate device reagent and cobas c 503 analyzer, as well as by the Lp(a) ELISA assay. Deming regression analysis was conducted. The results of the analysis, including slope with $9 5 \%$ confidence interval (CI), intercept with $9 5 \%$ CI, and correlation coefficient, are shown below.

<table><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sample range(nmol/L)</td><td rowspan=1 colspan=1>Slope(95% CI)</td><td rowspan=1 colspan=1>Intercept(95% CI) (nmol/L)</td><td rowspan=1 colspan=1>CorrelationCoefficient (r)</td></tr><tr><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>8.7 - 234</td><td rowspan=1 colspan=1>1.023(0.987, 1.058)</td><td rowspan=1 colspan=1>0.692(-0.981, 2.36)</td><td rowspan=1 colspan=1>0.992</td></tr></table>

2. Matrix Comparison:

A matrix comparison study was conducted to demonstrate equivalence between the sample types when used with the Tina-quant Lipoprotein (a) Gen.2 Molarity assay: serum, serum with gel separation, lithium heparin plasma, $\mathrm { K } _ { 2 } \mathrm { E D T A }$ plasma, and $\mathrm { K } _ { 3 } \mathrm { E D T A }$ plasma. Matched samples from up to 71 donors were tested on the cobas c 503 analyzer. PassingBablok regression analysis was conducted for each sample type relative to the reference serum values. The results of the analyses, including slope with $9 5 \%$ confidence interval (CI), intercept with $9 5 \%$ CI, and correlation coefficient, are shown below for each sample type comparison.

<table><tr><td rowspan=1 colspan=1>Matrix comparison</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sample range(nmol/L)</td><td rowspan=1 colspan=1>Slope(95% CI)</td><td rowspan=1 colspan=1>Intercept(95% CI)(nmol/L)</td><td rowspan=1 colspan=1>CorrelationCoefficient(r)</td></tr><tr><td rowspan=1 colspan=1>Serum with gel vs.serum</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>7.5 to 239</td><td rowspan=1 colspan=1>1.000(0.995, 1.003)</td><td rowspan=1 colspan=1>0(-0.079, 0.570)</td><td rowspan=1 colspan=1>1.000</td></tr><tr><td rowspan=1 colspan=1>Lithium heparinplasma vs. serum</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>7.5 to 239</td><td rowspan=1 colspan=1>0.981(0.974, 0.987)</td><td rowspan=1 colspan=1>0.286(-0.183, 0.855)</td><td rowspan=1 colspan=1>0.998</td></tr><tr><td rowspan=1 colspan=1>K2EDTA plasmavs. serum</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>7.5 to 233</td><td rowspan=1 colspan=1>0.972(0.967, 0.978)</td><td rowspan=1 colspan=1>0.134(-0.208, 0.566)</td><td rowspan=1 colspan=1>1.000</td></tr><tr><td rowspan=1 colspan=1>K3EDTAplasmaVS. serum</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>7.5 to 233</td><td rowspan=1 colspan=1>0.944(0.935, 0.952)</td><td rowspan=1 colspan=1>0.389(-0.044, 0.875)</td><td rowspan=1 colspan=1>1.000</td></tr></table>

# C Clinical Studies:

1. Clinical Sensitivity: Not applicable.

2. Clinical Specificity: Not applicable.

3. Other Clinical Supportive Data (When 1. and 2. Are Not Applicable):

The sponsor provided several studies showing associations between Lp(a) and ASCVD risk, including the UK Biobank study.1

In the UK Biobank study, which included 460,506 participants between the ages of 40 and 69 years, the relationship of Lp(a) concentrations to incident ASCVD was studied. ASCVD was defined as a composite of coronary artery disease (myocardial infarction and its acute complications, coronary artery bypass graft surgery, or percutaneous angioplasty/stent placement) and ischemic stroke (cerebral infarction due to thrombosis or cerebral atherosclerosis or cerebrovascular syndromes).

The event rate per 1000 person-years near 0 nmol/L Lp(a) was ${ \sim } 0 . 4 \%$ and the event rate increased to ${ \sim } 0 . 4 5 \%$ at $7 5 { \mathrm { ~ n m o l / L ~ L p ( a ) } }$ and ${ \sim } 0 . 5 \%$ at $1 2 5 { \mathrm { n m o l } } / \mathrm { L } \mathrm { L p } ( \mathrm { a }$ ) (the claimed thresholds for this device; see Figure 2A of Patel, et al. 2021). In addition, Cox proportional hazards regression model (using covariates of enrollment age, sex, race, and Lp(a) concentration) was performed. This analysis showed an adjusted hazard ratio (HR) of ${ \sim } 1 . 2$ at 75 nmol/L Lp(a) and ${ \sim } 1 . 4$ at 125 nmol/L Lp(a) (see Figure 2B of Patel, et al. 2021).

Incidence rates per 1000 person-years were also stratified by racial subgroup (White, Black, South Asian; see Supplemental Figure V, A of Patel, et al. 2021).

The event rate per 1000 person-years was ${ \sim } 0 . 4 \%$ for the White group near $0 \mathrm { n m o l / L }$ Lp(a) and increased to ${ \sim } 0 . 4 5 \%$ at 75 nmol/L Lp(a) and ${ \sim } 0 . 5 \%$ at 125 nmol/L Lp(a).

The event rate per 1000 person-years was ${ \sim } 0 . 3 5 \%$ for the Black group near 0 nmol/L Lp(a), 75 nmol/L Lp(a) and $1 2 5 \mathrm { n m o l / L } \mathrm { L p }$ (a). • The event rate per 1000 person-years was ${ \sim } 0 . 6 5 \%$ for the South Asian group near 0 nmol/L Lp(a) and increased to ${ \sim } 0 . 7 5 \%$ at 75 nmol/L Lp(a) and 125 nmol/L Lp(a).

The $9 5 \%$ confidence intervals were larger for the Black and South Asian groups. The authors of the study concluded that there is a linear risk gradient of Lp(a) across different races with standardized ASCVD risk found to be $1 1 \%$ higher for each increment of $5 0 \ \mathrm { n m o l / L }$ $\mathrm { { ( H R = } }$ 1.11 per $5 0 \mathrm { n m o l / L }$ $[ 9 5 \%$ CI, 1.10-1.12]), independent of adjustment for traditional risk factors, and with similar effect estimates in all race and ethnicity groups.

Similar associations between Lp(a) and ASCVD risk have also been demonstrated in multiple published clinical studies in the U.S., including the Dallas Heart Study2 . Information was provided to support that the associations between Lp(a) and ASCVD risk apply to the candidate assay.

The labeling states the following:

Scientific Statements on Use of Lp(a)

For an individual’s ASCVD risk assessment, according to the American societies (American Heart Association (AHA) and American College of Cardiology (ACC)), risk assessment of ASCVD should begin with a 10-year risk estimation which includes an individual’s age, race and other established risk factors.3,4 To determine a patient’s overall ASCVD risk, the patient’s baseline cardiovascular risk and Lp(a)-attributable risk should be considered together.

The Lp(a)-attributable ASCVD risk is independent of a patient’s baseline ASCVD risk, as Lp(a) can be regarded as a risk enhancer of a patient’s baseline ASCVD risk which is dependent on several risk factors including family history of premature ASCVD, primary hypercholesterolemia, metabolic syndrome, sex-specific risk enhancing factors, chronic inflammatory conditions, chronic kidney disease, diabetes, hypertension, age, smoking and highrisk ethnicities.5,6

The National Lipid Association $( 2 0 2 4 ) ^ { 7 }$ points to the continuous relationship between Lp(a) and ASCVD risk and provides guidance for a three-tiered risk stratification for Lp(a). The American College of Cardiology (ACC)/American Heart Association (AHA) Guideline on the Management of Blood Cholesterol (2018) and the ACC/AHA Guideline on the Primary Prevention of Cardiovascular Disease (2019) set an $\operatorname { L p } ( { \mathrm { a } } )$ threshold of $1 2 5 \mathrm { n m o l / L }$ above which individuals are to be considered at increased ASCVD risk.3,4 The table below shows NLArecommended Lp(a) thresholds, alongside the Lp(a)-attributable ASCVD risk classification5,8:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Lp(a)-attributable ASCVD risk</td></tr><tr><td rowspan=1 colspan=1>&lt;75 nmol/L</td><td rowspan=1 colspan=1>Low Lp(a)-attributable ASCVD risk</td></tr><tr><td rowspan=1 colspan=1>75 - &lt;125 nmol/L</td><td rowspan=1 colspan=1>Intermediate Lp(a)-attributable ASCVD risk</td></tr><tr><td rowspan=1 colspan=1>≥125 nmol/L</td><td rowspan=1 colspan=1>High Lp(a)-attributable ASCVD risk, with increasing risk with higher Lp(a)values</td></tr></table>

These thresholds are aligned with the Lp(a) Consensus Statement by the European Atherosclerosis Society, which uses UK Biobank data to derive the thresholds. The relationship of Lp(a) concentrations to incident ASCVD was studied in 460,506 participants between the ages of 40 and 69 years. ASCVD was defined as a composite of coronary artery disease (myocardial infarction and its acute complications, coronary artery bypass graft surgery, or percutaneous angioplasty/stent placement) and ischemic stroke (cerebral infarction due to thrombosis or cerebral atherosclerosis or cerebrovascular syndromes). The data showed a linear risk gradient of Lp(a) across different races with standardized ASCVD risk found to be $1 1 \%$ higher for each increment of $5 0 \mathrm { n m o l / L }$ $\mathrm { H R } = 1 . 1 1$ per $5 0 \mathrm { n m o l / L }$ $[ 9 5 \%$ CI, 1.10-1.12], independent of adjustment for traditional risk factors, and with similar effect estimates in all race and ethnicity groups (which included White, Black, and South Asian groups).1,5

The measured Lp(a) value of a patient’s sample can vary depending on the Lp(a) assay used and the units reported. The laboratory result report should therefore contain a statement on the Lp(a) assay used. Values determined in samples by different Lp(a) assays cannot be used interchangeably.

# References:

1. Patel AP, Wang M, Pirruccello JP, et al. Lp(a) (Lipoprotein[a]) Concentrations and Incident Atherosclerotic Cardiovascular Disease New Insights from a Large National Biobank. Arterioscler Thromb Vasc Biol 2021;41(1):465-74.   
2. Lee SR, Prasad A, Choi YS, Xing C, Clopton P, Witztum JL, Tsimikas S. LPA Gene, Ethnicity, and Cardiovascular Events. Circulation. 2017 Jan 17;135(3):251-263. doi: 10.1161/CIRCULATIONAHA.116.024611. Epub 2016 Nov 9. PMID: 27831500; PMCID: PMC5241172.   
3. Grundy SM, Stone NJ, Bailey AL, et al. 2018 AHA/ACC/AACVPR/AAPA/ABC/ACPM/ADA/AGS/APhA/ASPC/NLA/PCNA Guideline on the Management of Blood Cholesterol: Executive Summary: A Report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. J Am Coll Cardiol 2019;73(24):3168-209.   
4. Arnett DK, Blumenthal RS, Albert MA, et al. 2019 ACC/AHA Guideline on the Primary Prevention of Cardiovascular Disease: A Report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. Circulation. 2019 Sep 10;140(11):e596-e646.   
5. Kronenberg F, Mora S, Stroes ESG, et al. Lipoprotein(a) in atherosclerotic cardiovascular disease and aortic stenosis: A European Atherosclerosis Society consensus statement. Eur Heart J. 2022;43(39):3925-46.   
6. Wong ND, Budoff MJ, Ferdinand K, et al. Atherosclerotic cardiovascular disease risk assessment: An American Society for Preventive Cardiology clinical practice statement. Am J Prev Cardiol. 2022 Mar 15;10:100335.   
7. Koschinsky ML, Bajaj A, Boffa MB, et al. A focused update to the 2019 NLA scientific statement on use of lipoprotein(a) in clinical practice. J Clin Lipidol. 2024 Mar 29:S1933- 2874(24)00033-3. doi:10.1016/j.jacl.2024.03.001. Epub ahead of print. PMID: 38565461.   
8. Ciffone N, McNeal CJ, McGowan MP, et al. Lipoprotein(a): An important piece of the ASCVD risk factor puzzle across diverse populations. Am Heart J Plus. 2023 Nov 24;38:100350.

# D Clinical Cut-Off:

Not applicable.

# E Expected Values/Reference Range:

The labeling includes the following Expected values information:

Roche conducted a reference range study and measured Lp(a) levels among various subpopulations. The expected ranges were determined using samples from apparently healthy adults in the United States, with an equal representation of males and females. The median age of the participants was 46 years (IQR 33-57 years).

The values given are for information only and may vary from other published data. The reference ranges are not intended to be used as medical decision thresholds for cardiovascular risk.

Each laboratory should investigate the transferability of the expected values to its own patient population and if necessary determine its own reference ranges.

Lp(a) by Race   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>2.5th percentile(90%-CI)</td><td rowspan=1 colspan=1>97.5th percentile(90%-CI)</td><td rowspan=1 colspan=1>Unit</td></tr><tr><td rowspan=1 colspan=1>Caucasian/Whitea)</td><td rowspan=1 colspan=1>425</td><td rowspan=1 colspan=1>17.8</td><td rowspan=1 colspan=1>&lt;7 (n.a.)</td><td rowspan=1 colspan=1>259 (229; 265)</td><td rowspan=1 colspan=1>nmol/L</td></tr><tr><td rowspan=1 colspan=1>African-American/Black</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>66.2</td><td rowspan=1 colspan=1>&lt;7 (n.a.)</td><td rowspan=1 colspan=1>267 (207; 313)</td><td rowspan=1 colspan=1>nmol/L</td></tr><tr><td rowspan=1 colspan=1>Asian</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>&lt;7 (n.a.)</td><td rowspan=1 colspan=1>175 (166; 330)</td><td rowspan=1 colspan=1>nmol/L</td></tr></table>

a) Includes 4 subjects with unknown ethnicity.

Lp(a) by Ethnicity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>2.5thpercentile(90%-CI)</td><td rowspan=1 colspan=1>97.5th percentile(90%-CI)</td><td rowspan=1 colspan=1>Unit</td></tr><tr><td rowspan=1 colspan=1>Hispanic/Latino</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>&lt;7 (n.a.)</td><td rowspan=1 colspan=1>260 (162; 267)</td><td rowspan=1 colspan=1>nmol/L</td></tr><tr><td rowspan=1 colspan=1>Non-Hispanic/Non-Latino</td><td rowspan=1 colspan=1>311</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>&lt;7 (n.a.)</td><td rowspan=1 colspan=1>259 (229; 287)</td><td rowspan=1 colspan=1>nmol/L</td></tr></table>

# VIII Proposed Labeling:

The labeling supports the finding of substantial equivalence for this device.

# IX Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.