# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k051713   
B. Purpose for Submission: New test system and instrument   
C. Measurand: B. anthracis pX01 and pX02 DNA sequences   
D. Type of Test: DNA amplification with real-time fluorescent detection; qualitative result   
E. Applicant: Idaho Technology, Inc.   
F. Proprietary and Established Names: JBAIDS Anthrax Detection System/B. anthracis detection and presumptive identification using Nucleic Acid Amplification

# G. Regulatory Information:

1. Regulation section: Unclassified   
2. Classification: II (by panel recommendation)   
3. Product code: NHT, Assay, Nucleic Acid Amplification, Bacillus anthracis   
4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The JBAIDS Anthrax Detection System is a real-time polymerase chain reaction (PCR) test system intended for the qualitative in vitro diagnostic (IVD) detection of target DNA sequences on the pXO1 plasmid (Target 1) and the pXO2 plasmid (Target 2) from Bacillus anthracis. The system can be used to test human whole blood collected in sodium citrate from individuals suspected of having anthrax, positive blood cultures, and cultured organisms grown on blood agar plates. The JBAIDS Anthrax Target 2 assay is used as a supplementary test only after a positive result with the Target 1 Assay.

The JBAIDS Anthrax Target 1 and Target 2 Assays are run on the JBAIDS instrument using the Diagnostic Wizard.

Results are for the presumptive identification of $B$ . anthracis, in conjunction with culture and other laboratory tests. The following considerations also apply:

The diagnosis of anthrax infection must be made based on history, signs, symptoms, exposure likelihood, other laboratory evidence, in addition to the identification of pXO1 and pXO2 targets either from cultures or from direct blood specimens.   
The assays have not been evaluated with blood from individuals without clinical signs or symptoms who were presumed exposed and who subsequently developed anthrax (inhalation or other forms of the disease), or from individuals with any form of anthrax (inhalational, cutaneous, or gastrointestinal).   
The level of plasmid targets that would be present in blood from individuals with early systemic infection is unknown.   
The definitive identification of $B$ . anthracis from colony growth, liquid blood culture growth, or from blood specimens requires additional testing and confirmation procedures in consultation with public health or other authorities for whom reports are required.

The safety and effectiveness of other types of tests or sample types (not identified as “For in vitro diagnostic use”) have not been established.

# 2. Indication(s) for use:

The system can be used to test human whole blood collected in sodium citrate from individuals suspected of having anthrax, positive blood cultures, and cultured organisms grown on blood agar plates.

3. Special conditions for use statement(s):

Results are for the presumptive identification of $B$ . anthracis, in conjunction with culture and other laboratory tests. The following considerations also apply:

The diagnosis of anthrax infection must be made based on history, signs, symptoms, exposure likelihood, other laboratory evidence, in addition to the identification of pXO1 and $\mathsf { p X O 2 }$ targets either from cultures or from direct blood specimens. The assays have not been evaluated with blood from individuals without clinical signs or symptoms who were presumed exposed and who subsequently developed anthrax (inhalation or other forms of the disease), or from individuals with any form of anthrax (inhalational, cutaneous, or gastrointestinal). The level of plasmid targets that would be present in blood from individuals with early systemic infection is unknown. The definitive identification of $B$ . anthracis from colony growth, liquid blood culture growth, or from blood specimens requires additional testing and confirmation procedures in consultation with public health or other authorities for whom reports are required.

The safety and effectiveness of other types of tests or sample types (not identified as “For in vitro diagnostic use”) have not been established.

# 4. Special instrument requirements:

The JBAIDS Anthrax Target 1 and Target 2 Assays are run on the JBAIDS instrument using the Diagnostic Wizard and the JBAIDS software.

# I. Device Description:

The JBAIDS Anthrax Detection System is an integrated in vitro diagnostic (IVD) system composed of the JBAIDS instrument with laptop computer, software and two different freezedried sets of reagents (in one kit) for the detection of the pXO1 plasmid (Target 1) and the pXO2 plasmid (Target 2) from $B$ . anthracis.

The JBAIDS Anthrax Detection System is designed to detect virulent forms of $B$ . anthracis by detecting sequences within two plasmids, pXO1 and pXO2, that carry genes encoding for the organism’s two major virulence factors, the anthrax toxin complex and the capsular polypeptide. All samples are initially tested for Target 1, and only specimens that are positive are subsequently tested for Target 2. Both of these plasmids are required for virulence. Since nonplasmid containing $B .$ . anthracis have been isolated from the environment and these plasmids can be transferred to other bacteria, definitive identification of $B .$ . anthracis and discriminating it from other Bacillus species requires detection of specific chromosomal signatures and characteristic phenotypic features in consultation with the appropriate public health authorities.

The JBAIDS instrument, using PCR technology, is a portable thermocycler and real-time fluorimeter. The JBAIDS System Manual for the JBAIDS instrument and the software has a clearly defined workflow for diagnostic assays. The diagnostic workflow is accessed using the Diagnostic Wizard, which can only be used with legally marketed assays that are intended for in vitro diagnostic (IVD) use. When using the Diagnostic Wizard, the software ensures that the assays are conducted according to the product specifications, including the appropriate use of controls and application of the correct instrument settings. Data related to clinical assays are stored in a separate database used only for diagnostic test runs and results.

The JBAIDS Anthrax Detection Kit is specially designed for PCR in glass capillaries using the JBAIDS instrument and hydrolysis probes for detection of Target 1 and Target 2 DNA sequences. Amplicon is detected using a specific hydrolysis probe, which is a short oligonucleotide that hybridizes to an internal sequence of the amplified fragment during the annealing phase of the PCR cycle. This probe has the $_ 5 \cdot$ and $_ { 3 } ,$ ends labeled with a reporter dye and a quenching dye, respectively. When the probe hybridizes to the specific DNA target, the Taq polymerase enzyme replicating the target-specific DNA hydrolyzes the probe, separating the two fluorophores, thus allowing the reporter dye to fluoresce.

The characteristics of the amplification curves from the positive control (PC), negative control (NC), inhibition controls (IC) and from each unknown sample are analyzed using mathematical algorithms by the JBAIDS Software. Results are reported as Positive, Negative, Inhibited or Uncertain. When PCs or NCs are unacceptable, the test result for all samples in the JBAIDS run are considered Invalid and must be repeated.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Gamma Phage Lysis Assay 2. Predicate 510(k) number(s): k051794

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>presumptive identification ofB. anthracis</td><td rowspan=1 colspan=1>Aid in the identification ofB. anthracis</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Indications for use</td><td rowspan=1 colspan=1>Whole blood (sodium citrate)Liquid blood culture growthBacterial colonies on bloodagar</td><td rowspan=1 colspan=1>Bacterial colonies onblood agar</td></tr><tr><td rowspan=1 colspan=1>Principle of operation</td><td rowspan=1 colspan=1>Nucleic acid amplificationand fluorescent probedetection</td><td rowspan=1 colspan=1>Phage lysis</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>JBAIDS integratedthermocycler and fluorimeter</td><td rowspan=1 colspan=1>Visual observation</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): NA

# L. Test Principle:

DNA is initially extracted and purified from whole blood specimens using the Idaho Technology IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ FLOW Sample Purification Kit (or validated equivalent), and from blood culture and direct culture samples using the Idaho Technology IT $\pmb { I } \mathbf { - } \hat { 2 } \mathbf { - } \hat { 3 } ^ { \mathbf { T M } }$ SWIPE Sample Purification Kit (or validated equivalent). The purified sample is added to the freeze-dried reagent vial along with reconstitution buffer, and centrifuged into the tips of composite glass/plastic reaction capillaries. The JBAIDS Anthrax Detection Kit is specially designed for PCR in glass capillaries using the JBAIDS instrument and hydrolysis probes to detect $\mathsf { p X O 1 }$ plasmid (Target 1) and pXO2 plasmid (Target 2) DNA sequences. Amplicon is cumulatively detected using a specific hydrolysis probe, a short oligonucleotide that hybridizes to an internal sequence of the amplicon during the annealing phase of the PCR cycle. This probe has the $5 '$ and $_ { 3 } ,$ ends labeled with a reporter dye and a quenching dye, respectively. When the probe hybridizes to the specific DNA target, the Taq polymerase enzyme replicating the target-specific DNA hydrolyzes the probe, separating the two fluorophores, and allowing the reporter dye to fluoresce.

The characteristics of the amplification curves from the positive control (PC), negative control (NC), inhibition controls (IC) and from each unknown sample are analyzed by the JBAIDS Software, and results are reported as Positive, Negative, Inhibited or Uncertain. When PCs or NCs are unacceptable, the test result for all samples in the JBAIDS run are considered Invalid and must be repeated. PCR-based methods are well-established; approaches for software modeling of growth curves are not well-established. Genotypic characterization of B. anthracis is better understood than 10 years ago. However, culture-based methods remain the reference approach for detecting the organism in clinical samples.

Possible JBAIDS Results for Unknown Tests (either Target 1 or Target 2)   

<table><tr><td rowspan=1 colspan=1>Combined SampleResult Call</td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>A red Positive result indicates that the target wasidentified in both capillaries from the same sampleand all run controls were successful.Warning: False positives due to contaminationshould be considered when amplification curves areweak (late Cp (&gt;36) or &lt;50% fluorescence signalwhen compared to the PC).</td><td rowspan=1 colspan=1>If Target 1, test Target 2 from thesame purified sample.If Target 2, see next Table forInterpretation</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>A green Negative result indicates that the target wasnot identified in either capillary from the same sampleand all run controls were successful.</td><td rowspan=1 colspan=1>If Target 1 or Target 2, see nextTable for interpretation</td></tr><tr><td rowspan=1 colspan=1>Inhibited</td><td rowspan=1 colspan=1>A yellow Inhibited result indicates that at least one ICcapillary from a given sample showed PCR inhibition,and that the positive and negative controls weresuccessful.</td><td rowspan=1 colspan=1>Retest with the same assay using thesame purified sample and also usingadditional dilution procedure asdescribed in the QUALITYCONTROL Section Use retestresult to interpret.</td></tr><tr><td rowspan=1 colspan=1>Uncertain</td><td rowspan=1 colspan=1>A yellow Uncertain can occur:1. any time the results from the two capillaries fromone sample do not agree,2. when both capillaries have an amplification curvethat is not definitive.</td><td rowspan=1 colspan=1>Retest with the same assay using thesame purified sample.If retest is uncertain: Contactsupervisor or IT Technical Support</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>A yellow Invalid indicates that the positive and/ornegative control failed (in one or both capillaries). Allreactions for that assay in the run will be invalid andshould be repeated.</td><td rowspan=1 colspan=1>Retest with the same assay using thesame purified sample. Use retestresults if valid (positive or negative).If retest is invalid, contact supervisoror IT Technical Support</td></tr></table>

Interpreting Patient Results   

<table><tr><td rowspan=1 colspan=1>Sample ResultFor Target 1</td><td rowspan=1 colspan=1>Action</td><td rowspan=1 colspan=1>Follow-up Resultfor Target 2</td><td rowspan=1 colspan=1>Result Interpretation /Report*</td></tr><tr><td rowspan=1 colspan=1>Negative(both capillaries)</td><td rowspan=1 colspan=1>No further testing</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>B. anthracis pXO1 DNA not detected  colony or blood culture broth: Not B. anthracis; non-plasmid containing strain possible.if blood sample: B. anthracis bacteremia cannot beexcluded; blood cultures recommended.</td></tr><tr><td rowspan=2 colspan=1>Positive(both capillaries)</td><td rowspan=2 colspan=1>Run Target 2 assayon same purifiedsample.</td><td rowspan=1 colspan=1>Negative(both capillaries)</td><td rowspan=1 colspan=1>B. anthracis pXO1 DNA detected; pXO2 DNA not detected *A positive Target 1 result with a negative Target 2result can occur with infection due to rare virulentstrains of Bacillus cereus that contains the pXO1plasmid and can cause a fatal infection or a non-virulent for of B. anthracis, such as Sterne.If blood sample, a positive Target 1 result with anegative Target 2 result could also indicate infectionwith B. anthracis at or near the assays limit ofdetection.</td></tr><tr><td rowspan=1 colspan=1>Positive(both capillaries)</td><td rowspan=1 colspan=1>Both B. anthracis pXO1 and pXO2 DNA detected: B.anthracis likely. Follow-up per applicable Public Healthauthorities.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multi-center reproducibility study was assessed precision between-site using sample panels prepared by spiking either the whole blood or simulated blood culture matrix with inactivated $B$ anthracis Ames vegetative cells to represent a matrix blank (no organism), low, medium or high levels of organism. The samples were tested at 3 external sites and at Idaho Technology’s facility on 3 different days. One apparent false positive and one apparent false negative were recorded at one site and attributed to carousel misloading. Training and experience is required for performing the testing. The testing algorithm (Target 2 as a supplementary test) and requirement for use with blood culturing is designed to mitigate such errors.

Between-Site Reproducibility Results for Whole Blood and Blood Culture B. anthracis Target 1 and Target 2 Assays   

<table><tr><td colspan="1" rowspan="2">Matrix</td><td colspan="1" rowspan="2">SampleID</td><td colspan="5" rowspan="1">Target 1</td><td colspan="5" rowspan="1">Target 2</td></tr><tr><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">SiteIT</td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">SiteIT</td><td colspan="1" rowspan="1">Overall</td></tr><tr><td colspan="1" rowspan="3">WholeBlood</td><td colspan="1" rowspan="1">Matrixblank</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Matrixblank</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="7">′</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="6" rowspan="1">Total Agreement for Target 1</td><td colspan="1" rowspan="1">120/120</td><td colspan="4" rowspan="1">Total Agreement for Target 2</td><td colspan="1" rowspan="1">120/120</td></tr><tr><td colspan="1" rowspan="3">BloodCulture</td><td colspan="1" rowspan="1">Matrixblank</td><td colspan="1" rowspan="1">2/3 a</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">2/3 ^a</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">11/12</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">12/12</td></tr><tr><td colspan="6" rowspan="1">Total Agreement for Target 1</td><td colspan="1" rowspan="1">34/36</td><td colspan="4" rowspan="1">Total Agreement for Target 2</td><td colspan="1" rowspan="1">36/36</td></tr></table>

a Retesting of these samples yielded the expected test result. The original results are likely the result of a sample loading error.

# $b$ . Linearity/assay reportable range:

5-1000 Genomic equivalents/reaction (both Target 1 and 2); approximately 1,000-   
50,000 bacterial cells/mL

One genomic equivalent (GE) is defined as the molecular weiht of 1 genome of $B$ . anthracis DNA. Because the number of plasmids is variable between organisms, one GE is the total number of base pairs $( 5 . 2 \dot { 3 } \mathrm { ~ x ~ } 1 0 ^ { 6 } )$ ) that encompasses one copy of chromosomal DNA plus one copy each of the $\mathsf { p X } 0 1$ and $\mathsf { p X } 0 2$ plasmids or 5.78 femtograms DNA.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The following table provides information about the $\mathrm { C p }$ (crossing point) values obtained for positive amplification curves, including the PCs, during the clinical and selected non-clinical evaluations of this product. The Cp value is the approximate PCR cycle number when the signal for a positive amplification curve rises above the background fluorescence. Cp values are not assigned to negative amplification curves.

Cp Values for Positive Amplification Curves   

<table><tr><td colspan="1" rowspan="1">Sample or Control Type</td><td colspan="2" rowspan="1">Target 1Average    Range ofCp (SD)    Cp values</td><td colspan="2" rowspan="1">Target 2Average    Range ofCp (SD)   Cp values</td></tr><tr><td colspan="5" rowspan="1">Results of Controls used in Clinical Trials</td></tr><tr><td colspan="1" rowspan="1">Positive Controls</td><td colspan="1" rowspan="1">32.3 (0.5)n=152</td><td colspan="1" rowspan="1">31.5-34.0</td><td colspan="1" rowspan="1">32.3 (0.3)n=144</td><td colspan="1" rowspan="1">31.6-33.2</td></tr><tr><td colspan="1" rowspan="1">Inhibition Controlsa</td><td colspan="1" rowspan="1">32.1 (0.4)n=312</td><td colspan="1" rowspan="1">31.2-33.7</td><td colspan="1" rowspan="1">32.1 (0.4)n=311</td><td colspan="1" rowspan="1">30.7-32.9</td></tr><tr><td colspan="5" rowspan="1">Results of Samples containing B. anthracis</td></tr><tr><td colspan="1" rowspan="1">Whole Blood specimens spiked with live B.Anthracis at assay LOD levels (1000 CFU/mL)</td><td colspan="1" rowspan="1">34.2 (1.0)n= 136</td><td colspan="1" rowspan="1">31.8-38.5</td><td colspan="1" rowspan="1">34.9 (1.0)n= 136</td><td colspan="1" rowspan="1">32.5-37.7</td></tr><tr><td colspan="1" rowspan="1">B. Anthracis Positive Blood Cultures Samples</td><td colspan="1" rowspan="1">16.7 (0.9)n= 154</td><td colspan="1" rowspan="1">15.1-19.1</td><td colspan="1" rowspan="1">17.0 (0.8)n= 154</td><td colspan="1" rowspan="1">16.0-19.4</td></tr><tr><td colspan="1" rowspan="1">B. Anthracis Direct Culture Samples</td><td colspan="1" rowspan="1">20.3 (4.2)n=206</td><td colspan="1" rowspan="1">15.2-31.0</td><td colspan="1" rowspan="1">21.4(4.2)n=150</td><td colspan="1" rowspan="1">16.2-31.0</td></tr></table>

a These IC Cp values are representative of testing negative samples. The Cp values for IC associated with positive sample will be earlier (smaller number) due to the additional DNA added to the IC reactions.

The early $\mathrm { C p }$ values associated with culture samples (blood culture and direct culture) are indicative of the high concentration of $B$ . anthracis in these sample types.

# d. Detection limit:

The analytic sensitivity (limit of detection) of the test system was determined by spiking 68 whole blood samples (obtained from 68 different healthy donors) with live vegetative $B$ . anthracis at $1 , 0 0 0 \mathrm { C F U / m L }$ and testing them using both the Target 1 and Target 2 assays. Sixty seven (67) out of 68 spiked whole blood samples were successfully detected (positive for both assays) resulting in a limit of detection of $1 , 0 0 0 \mathrm { \ C F U / m L }$ of vegetative $B$ . anthracis cells in whole blood samples. The sensitivity of the test system using $B$ . anthracis spores was not evaluated.

# e. Analytical specificity:

Panels of well-characterized non-B. anthracis were tested using both the Target 1 and Target 2 assays. The results of the JBAIDS test were considered negative only if both the Target 1 and Target 2 assays were negative. This evaluation was performed with three different sample types, genomic DNA (DNA panel), organism grown on sheep blood agar plates (Direct culture panel) and positive blood cultures (Blood culture panel). The testing was performed as described in the previous section.

Panel Listing of non-virulent Bacillus anthracis Species Evaluated and JBAIDS Results for B. anthracis Target 1 and Target 2   

<table><tr><td rowspan=1 colspan=3>Nonvirulent Bacillus anthracis strains</td></tr><tr><td rowspan=1 colspan=1>DNA Panel</td><td rowspan=1 colspan=1>Direct Culture Panel</td><td rowspan=1 colspan=1>BloodCulturePanel</td></tr><tr><td rowspan=1 colspan=1>B. anthracis Delta Sterne (Target 1 -, Target 2 -)B. anthracis Delta NH-1 (Target 1 -, Target 2 +)B. anthracis Pasteur-like (Target 1 -, Target 2 +)(ATCC 4728)B. anthracis SPS 97.13.213 (Target 1 +, Target 2-)B. anthracis STI (Target 1 +, Target 2 -)B. anthracis Sterne (Target 1 +, Target 2 -)B. anthracis V770-NP1R (Target 1 +, Target 2 -)</td><td rowspan=1 colspan=1>B. anthracis Delta-Sterne (Target 1 -, Target 2 -)B. anthracis Delta-NH-1 (Target 1 -, Target 2 +)B. anthracis SPS97.13.213(Target 1 +, Target 2 -)B. anthracis STI (Target 1 +, Target 2 -)B. anthracis Sterne (Target 1 +, Target 2 -)B. anthracis V770-NP1R (Target 1 +, Target 2 -)B. anthracis Delta-Ames (Target 1 -, Target 2 +)B. anthracis Pasteur (Target 1 -, Target 2 +)</td><td rowspan=1 colspan=1>Notevaluated</td></tr></table>

Analytical Specificity Panel Results   

<table><tr><td colspan="1" rowspan="1">Organism Type</td><td colspan="1" rowspan="1">DNA Panel(# correct/# tested)</td><td colspan="1" rowspan="1">Direct Culture Panel(# correct/# tested)</td><td colspan="1" rowspan="1">Blood Culture Panel(# correct/# tested)</td></tr><tr><td colspan="1" rowspan="1">Nonvirulent B. anthracis</td><td colspan="1" rowspan="1">717^</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">Phylogenetically relatedorganisms (Nearest</td><td colspan="1" rowspan="1">25/25</td><td colspan="1" rowspan="1">22/25b</td><td colspan="1" rowspan="1">2/2</td></tr><tr><td colspan="1" rowspan="1">Neighbors)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clinically relevantunrelated organisms</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">12/12</td><td colspan="1" rowspan="1">10/10c</td></tr></table>

a One isolate of $B$ . anthracis Sterne was excluded from the analysis because it gave uncertain and negative test results with the Target 2 assay when it should have been negative. The mostly likely explanation for this result was that this DNA sample became contaminated with very low levels of DNA that reacted with the Target 2 assay.

b Three $B$ . cereus strains causing anthrax-like illness tested positive

For the direct culture and blood culture panels, the non-virulent $B$ . anthracis included in the test panel gave the expected microbiology results, including a positive gamma phage result, and also gave the expected JBAIDS result. In the direct culture panel, the three unexpected findings were $B _ { \cdot }$ . cereus 03BB102 which tested positive for both $B$ . anthracis Target 1 and Target 2, and $B$ . cereus 03BB87 and $B$ . cereus G9241, both of which tested positive with the Target 1 assay. All three of these $B$ . cereus organisms have been associated with anthrax-like illnesses and/or death.7,8 Follow-up testing using other pXO1 and $\mathsf { p X O 2 }$ real time PCR assays yielded results that were in agreement with the JBAIDS results.

f. Assay cut-off:

A data analysis module within assay-specific software applies mathematical modeling of expected amplification curve shapes to each individual capillary. When fixed thresholds with a quadratic formula or crossing point determination are matched, samples are called negative or positive. For undecided samples, an expert system approach is applied that uses filters to assess the fluorescence change around the crossing point.

2. Comparison studies: NA a. Method comparison with predicate device: NA b. Matrix comparison: NA

3. Clinical studies:

a. Clinical Sensitivity:

Due to the rare incidence of systemic anthrax disease, evaluating performance with blood specimens from infected patients was not possible. Testing with surrogate samples can be used to infer effectiveness with such samples, along with mitigating risk of a false negative by performing in conjunction with blood culture procedures.

b. Clinical specificity:

A multi-center clinical evaluation was conducted over a 4-month period and included 150 patients exhibiting symptoms consistent with systemic anthrax. Patients were selected from those who were hospitalized, exhibited the clinical definition of Systemic Inflammatory Response Syndrome (SIRS) and for whom a blood culture had been ordered. The study subjects ranged in age from 18-90 years (mean 48.5), and $56 \%$ were male. The study was conducted at three U.S. military medical installations in Texas, Washington DC, and Egypt. The clinical specificity of the JBAIDS Anthrax Detection System was assessed by comparing the JBAIDS results (positive/negative) to the results obtained by established culture and microbiology follow-up methods. With the evaluation of clinical blood specimens, no patients with $B .$ anthracis bacteremia were sampled during the study period. This is not unexpected for a rare human disease. Because sensitivity of the test system for $B .$ anthracis in blood samples from individuals with systemic anthrax cannot be determined, the intended use advises that the test system must be considered presumptive and done in conjunction with other microbiology testing particularly blood cultures. For inhibited, uncertain or invalid results were obtained with testing on the direct blood samples, following the repeat algorithm and use of the Target 2 assay as supplementary to the initial Target 1 assay, mitigated the potential for a false positive result being reported.

Table 1 - Summary of Clinical Testing (Initial Result)   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=10>JBAIDS Anthrax Detection Results (Initial)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=2>Inhibited</td><td rowspan=1 colspan=2>Uncertain5</td><td rowspan=1 colspan=2>Invalid</td><td rowspan=2 colspan=1>TotalSubjects</td></tr><tr><td rowspan=1 colspan=1>T1</td><td rowspan=1 colspan=1>T2</td><td rowspan=1 colspan=1>T1</td><td rowspan=1 colspan=1>T2</td><td rowspan=1 colspan=1>T1</td><td rowspan=1 colspan=1>T2</td><td rowspan=1 colspan=1>T1</td><td rowspan=1 colspan=1>T2</td><td rowspan=1 colspan=1>T1</td><td rowspan=1 colspan=1>T2</td></tr><tr><td rowspan=1 colspan=2>Patients - No Growthblood cultures result</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>116</td></tr><tr><td rowspan=1 colspan=2>Patients - Organismsrecovered fromstandard Bloodcultures1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=3 colspan=1>AllPatients</td><td rowspan=1 colspan=1>Site 12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Site 23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Site 34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>150</td></tr></table>

List of organisms recovered from blood culture and # : Coagulase Negative Staphylococcus (17); S. aureus (11); E. coli (5), Enterococcus (3); Staphylococcus epidermidis (2), P. acnes (1), Gemella morbillorum (1); Candida spp(2); Klebsiella pneumoniae (3), Strep spp (4); Enterobacter (1). 2 Blood culture technique at Site 1: BacT/ALERT 3D using BacT/ALERT SA (Aerobic) and BacT/ALERT SN (Anerobic),Bactec 9240 Plus (bioMerieux, Durham NC) held for 5 days at $3 5 ^ { \circ } \mathrm { C }$ . 3 Blood culture technique used at Site 2: BacTec Plus Aerobic F blood culture bottles (BD, Franklin Lake, NJ) held at $3 7 ^ { \circ } \mathrm { C }$ for 5 days, manual evaluation and subculture.

With recommended follow-up testing, all 150 samples included in the study gave the expected negative results for both the Target 1 and Target 2 assays and none of the blood cultures taken from the 150 patients grew $B$ . anthracis. The clinical specificity of the JBAIDS Anthrax Detection System is at least $9 8 \%$ (lower bound of $9 5 \%$ confidence interval).

c. Other clinical supportive data (when a. and b. are not applicable): NA

4. Clinical cut-off: NA

5. Expected values/Reference range:

Of the 150 whole blood sample tested during the clinical trial, 8 $( 5 . 3 \% )$ of the Target 1 and 7 $( 4 . 7 \% )$ of the Target 2 results were initially unresolved and required repeat testing in order to obtain a valid test result. The reasons for retesting are given in the following table. In all cases, retesting resulted in a valid Negative test result.

Reasons for Repeat Testing in Clinical Evaluation   

<table><tr><td rowspan=1 colspan=1>Reason for repeat testing</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Uncertain Test Result for Sample.</td><td rowspan=1 colspan=1>0.7%(1/150)</td><td rowspan=1 colspan=1>2.7%(4/150)</td><td rowspan=1 colspan=1>1.7%(5/300)</td></tr><tr><td rowspan=1 colspan=1>Instrument failure(one test run containing two samples).</td><td rowspan=1 colspan=1>1.3%(2/150)</td><td rowspan=1 colspan=1>1.3%(2/150)</td><td rowspan=1 colspan=1>1.3%(4/300)</td></tr><tr><td rowspan=1 colspan=1>Inhibited Test Result for Sample</td><td rowspan=1 colspan=1>0.7%(4/150)</td><td rowspan=1 colspan=1>0.7%(1/150)</td><td rowspan=1 colspan=1>0.7%(2/150)</td></tr><tr><td rowspan=1 colspan=1>Failure of the Positive Control</td><td rowspan=1 colspan=1>0.7%(1/150)</td><td rowspan=1 colspan=1>0%(0/150)</td><td rowspan=1 colspan=1>0.3%(1/300)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5.3%(8/150)</td><td rowspan=1 colspan=1>4.7%(7/150)</td><td rowspan=1 colspan=1>5%(15/300)</td></tr></table>

During the clinical evaluation there was one failure of the Target 1 PC ( $1 / 7 6$ or $1 . 3 \%$ ) and no failures of the Target 2 PC $( \mathrm { n } { = } 7 2 )$ ). All NCs were successful (76 Target $1 \ : \mathrm { N C s }$ and 72 Target 2 NCs). Of the 76 JBAIDS runs performed in the clinical trial, one $( 1 . 3 \% )$ had to be repeated due to an instrument failure.

N. Instrument Name: JBAIDS instrument

# O. System Descriptions:

1. Modes of Operation:

All sample testing is managed by the software embedded Diagnostic Wizard; up to 32 capillaries can be loaded into one carousel for a run.

2. Software: FDA has reviewed applicant’s Hazard Analysis and software development processes for

this line of product types: Yes _X___ or No

3. Specimen Identification:

Specimen information is a pre-defined protocol that guides a user through entering sample information, loading the JBAIDS carousel, and starting the testing sequence. Data, result call displays and generation of final reports is managed through the traceable database using the Diagnostic Wizard. Capillaries are identified by their position number in the carousel.

# 4. Specimen Sampling and Handling:

Blood specimens must be collected in tubes with sodium citrate anticoagulant. Other anticoagulants are not indicated and may contribute to unreliable results. DNA is initially extracted and purified from whole blood specimens using the Idaho Technology IT 1-2- $3 ^ { \mathrm { T M } }$ FLOW Sample Purification Kit (or validated equivalent), and from blood culture and direct culture samples using the Idaho Technology IT $\mathbf { \hbar } _ { I - 2 - 3 } \mathbf { \hbar } ^ { \mathrm { { T M } } }$ SWIPE Sample Purification Kit (or validated equivalent). Purified samples must be diluted prior to adding to reagent vials.

5. Calibration:

The fluorimeter is factory-calibrated. Internal self-check procedures are run with each startup. Cycling temperatures are monitored continuously during a run. The internal control and positive control must meet specific criteria for a successful run.

6. Quality Control:

The NC (a reagent blank) detects contamination from target specific amplicon, genomic DNA (as found in the PC and IC reagent vials) or organism. One NC (2 capillaries) must be included for each assay in a test run (Target 1 or Target 2). Both of the NC capillaries must be negative or the JBAIDS software will assign Invalid results to all the associated samples and the run will need to be repeated. Frequent or repeated failures of NCs may indicate significant contamination of the work area.

The PC (containing purified B. anthracis DNA) serves as an amplification and detection control. One PC (2 capillaries) must be included for each assay in a test run (Target 1 or Target 2). Both PC capillaries must be positive or the JBAIDS software will assign Invalid results to all the associated samples and the run will need to be repeated. In addition, the operator should manually evaluate the Crossing point $\displaystyle ( \mathrm { C p } )$ values for the PC capillaries. If the Cp for any of the PC capillaries is greater than 36, the run should be considered invalid. Failure of the PCs may indicate errors in sample set-up, degradation of the reagents or a malfunction of the JBAIDS instrument. If the IC capillaries in the same test run are Positive, then the failure is most likely caused by an isolated error with the set-up of the PC. If the IC capillaries are also Negative, possible causes for the failure are (1) a systematic error in sample set-up, (2) degradation of the reagents or (3) a malfunction of the JBAIDS instrument.

The IC demonstrates that the purified sample does not interfere with, or inhibit, the PCR reaction. Each purified sample is tested using an Unknown vial and an IC vial (for a total of 4 capillaries). If either of the IC amplification curves demonstrates inhibition then the result for that sample will be Inhibited. IC amplification curves are called Inhibited by the JBAIDS software if (1) the Cp value for the IC is significantly delayed (more than 3 cycles) compared to the amplification curve of the PC or (2) if the shape of the amplification curve demonstrates reduced PCR efficiency as evidenced by a flattening of the amplification curve. Each laboratory can trend Cp values for the Positive Control, and maintain records according to standard laboratory quality control practices. Any trend variations should be investigated.

A procedure to monitor the sample extraction/purification component of the test system can be used for training or for verification of user technique. Well characterized strains of $B$ . anthracis and other Bacillus species may be used as external negative and positive controls in accordance with guidelines or requirements of local, state and/or federal regulations or accrediting organizations. However, use of $B$ . anthracis strains containing either, or both, of the pXO1 and pXO2 plasmids is not feasible in many settings. Studies conducted at Idaho Technology demonstrate that the use of Baker’s Yeast may be considered as a surrogate control for monitoring extraction/purification of the FLOW and/or SWIPE components. Bakers Yeast is readily available, does not pose a biosafety risk and has a cell-wall, making it a suitable sample purification control for purification of samples containing B. anthracis. In addition, ITI provides a Training kit that contains freeze-dried reagents for detecting yeast DNA.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>Target 2</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>pX01</td><td rowspan=1 colspan=1>pX02</td></tr><tr><td rowspan=1 colspan=1>Amplicon length</td><td rowspan=1 colspan=1>110 bp</td><td rowspan=1 colspan=1>87 bp</td></tr><tr><td rowspan=1 colspan=1>Probe length</td><td rowspan=1 colspan=1>30 bp</td><td rowspan=1 colspan=1>29 bp</td></tr><tr><td rowspan=1 colspan=1>Fluorophore/Quencher</td><td rowspan=1 colspan=1>6-FAM/TAMRA</td><td rowspan=1 colspan=1>6-FAM/TAMRA</td></tr><tr><td rowspan=1 colspan=1>Inhibition Control</td><td rowspan=1 colspan=1>20 GE/rxn</td><td rowspan=1 colspan=1>20 GE/rxn</td></tr><tr><td rowspan=1 colspan=1>Detection Limit</td><td rowspan=1 colspan=1>5 GE/rxn (250 GE/mL)</td><td rowspan=1 colspan=1>5 GE/rxn (250 GE/mL)</td></tr><tr><td rowspan=1 colspan=1>PCR Efficiency</td><td rowspan=1 colspan=1>1.87+0.087</td><td rowspan=1 colspan=1>1.934+0.116</td></tr></table>

# Specifications for the JBAIDS Instrument

• Meets 810E US Military Transport Standard   
• Air, land and sea transportable   
• Carbon fiber outside case   
• Withstands one meter drop test   
• Resistant to temperature, humidity, shock and vibration. Air and water tight • 32 sample capacity   
• $5 \mathrm { - } 2 0 ~ \mu \mathrm { l }$ volume size   
• Composite glass/plastic reaction vessel loaded by centrifugation. • Three-color optics modules   
• Excitation wavelengths from $4 5 0 { \cdot } 4 9 0 \ \mathrm { n m }$ .   
• Emission wavelengths from 500-750 nm • Continuous (single sample) acquisition mode • Continuous scanning mode (frequency dependent on sample count) and programming

acquisition mode

• Range: Room temperature to $1 2 0 \mathrm { { } ^ { \circ } C }$ • Ramp Rate: $0 . 1 - 2 0 ~ ^ { \mathrm { o } } \mathrm { C } / \mathrm { s } \epsilon$ ec • 110/220 Volt auto-switching power supply unit. $\bullet$ 220/110 Volt, 50/60 HZ, 7.0/6.0 Amp, 650 Watt [ 8 Amp, 230 Volt , 3AG fuse required ]

• 10.5” (H) x 19.4” (W) x 14.3” (D) $\bullet$ 35.75 lbs. including instrument, laptop computer, laptop bag, cords, and DVD drive. Weight does not include microcentrifuge, tool kit, backpack, and startup kit.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.