# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

F. Proprietary and Established Names: IDI-VanR™ Assay

# G. Regulatory Information:

1. Regulation section: 866.1640

2. Classification: II

3. Product code: NIJ – System, test, genotypic detection, resistant markers

4. Panel:

# H. Intended Use:

# 1. Intended use(s):

The IDI-VanR $\textsuperscript { \textregistered }$ Assay is a qualitative in vitro test for the rapid detection of vancomycin-resistance (vanA and vanB) genes directly from rectal swabs. The IDI-vanR $\textsuperscript { \textregistered }$ Assay detects the presence of the vanA and vanB genes that can be associated with vancomycin-resistant enterococci (VRE). The assay is performed on an automated real-time PCR instrument with rectal swabs from patients at risk for VRE colonization. The IDI-VanR $\textsuperscript { \textregistered }$ Assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. Concomitant cultures are necessary to recover organisms for epidemiological typing, susceptibility testing and for further confirmatory identification. The IDIVanR $\textsuperscript { \textregistered }$ Assay is not intended to diagnose VRE infections nor to guide or monitor treatment for VRE infections.

# 2. Indication(s) for use:

The IDI-VanR $\textsuperscript { \textregistered }$ Assay is a qualitative in vitro test for the rapid detection of vancomycin-resistance (vanA and vanB) genes directly from rectal swabs. The IDI-vanR $\textsuperscript { \textregistered }$ Assay detects the presence of the vanA and vanB genes that can be associated with vancomycin-resistant enterococci (VRE). The assay is performed on an automated real-time PCR instrument with rectal swabs from patients at risk for VRE colonization. The IDI-VanR $\textsuperscript { \textregistered }$ Assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. Concomitant cultures are necessary to recover organisms for epidemiological typing, susceptibility testing and for further confirmatory identification. The IDIVanR $\textsuperscript { \textregistered }$ Assay is not intended to diagnose VRE infections nor to guide or monitor treatment for VRE infections.

3. Special conditions for use statement(s):

Samples must be collected using the Copan Transystem™ swabs. (Liquid Stuart, Liquid Amies, Amies Agar Gel without Charcoal, or Amies Agar Gel with Charcoal).

Prescription Use Only

4. Special instrument requirements:

Assay is performed with the SmartCycler $\textsuperscript { \textregistered }$ instrument

# I. Device Description:

The IDI-VanR ™ Assay is performed on rectal swabs with a procedure that includes specimen lysis, amplification of the vanA and vanB targets, and detection of fluorogenic target-specific hybridization probes. The amplification, detection and interpretation of the signals are done automatically by the Cepheid SmartCycler $\textsuperscript { \textregistered }$ instrument software. A positive and negative control is included in each IDI-vanR ™ Assay run. An Internal Control is also included in each assay tube. The inclusion of these three controls will monitor every step of the PCR procedure and the functionality of every reagent. External controls are recommended to monitor the cell lysis and DNA extraction steps. The procedure takes about 60 to 75 minutes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Bile Esculin Azide agar with $6 ~ { \mu \mathrm { g / m L } }$ vancomycin

2. Predicate 510(k) number(s): K972359

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Screening forvancomycin resistance</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Type of test</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>PCR amplification withdetection of fluorogenictarget-specifichybridization</td><td rowspan=1 colspan=1>Growth based phenotypicdetection</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Positive, negative andinternal controls are partof the assay. Specimenprocessing controls arerecommended whichinclude positive andnegative Enterococci spp.isolates</td><td rowspan=1 colspan=1>E. faecalis ATCC 51299- positive;E. faecalis ATCC 29212 negative</td></tr><tr><td rowspan=1 colspan=1>Mode of detection</td><td rowspan=1 colspan=1>Presence of vanA and orvanB genes</td><td rowspan=1 colspan=1>Growth or no growth inthe presence of 6 µg/mLvancomycin</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Rectal swab</td><td rowspan=1 colspan=1>Rectal swab or stool</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not Applicable

# L. Test Principle:

Following specimen lysis and vanA and vanB amplification, the DNA targets are detected with molecular beacons, a hairpin-forming single-stranded oligonucleotides labeled as tone end with a quencher and at the other end with a fluorescent reporter dye (fluorophore). In the absence of target, the fluorescence is quenched. In the presence of target, the hairpin structure opens upon beacon/target hybridization, resulting in emission of fluorescence. For the detection of vanA amplicons, the molecular beacon contains the fluorophore FAM at the $_ 5 \cdot$ end and the non-fluorescent quencher moiety DABCYL at the opposite end of the oligonucleotide. For the detection of the vanB amplicons, the molecular beacon contains the fluorophore Texas Red at the $_ 5 \cdot$ end and the quencher DABCYL at the $_ { 3 } ,$ end. Each beacontarget hybrid fluoresces at a wavelength characteristic of the fluorophore used in the particular molecular beacon. The amount of fluorescence depends on the amount of specific amplicons present at that time. The SmartCycler $^ \mathrm { \textregistered }$ software simultaneously monitors the fluorescence emitted by each beacon, interprets all data, and provides a final result at the end of the cycling program.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Ten samples containing negative matrix consisting of 3 low positive specimens and 5 high positive specimens and a negative and positive control were frozen and sent to three sites. Testing was performed in triplicate on three separate days at each site with three lots. Both vanA and vanB genes were included to ensure that all primers and probes of the IDI-vanR ™ Assay were assessed. The overall reproducibility was $> 9 5 \%$ with 3 invalid positive control results.

b. Linearity/assay reportable range: Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The table below describes internal controls that are part of the assay with the various parts controlled.

<table><tr><td rowspan=1 colspan=2>Step controlled</td><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>InternalControl</td></tr><tr><td rowspan=2 colspan=1>SpecimenPreparation</td><td rowspan=1 colspan=1>Cell lysis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>DNA extraction</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Inactivation of heat labile inhibitors</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=3 colspan=1>PCR</td><td rowspan=1 colspan=1>Denaturation step</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Primer/target annealing</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1>Elongation Step</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Probe/amplicon annealing</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=2>Presence of non heat labile PCR inhibitors</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td></tr><tr><td rowspan=1 colspan=2>Reagent contamination</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Environmental contamination with GBS amplicon</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td></tr></table>

Since Cell lysis and DNA extraction are not controlled, the Package Insert will carry a statement that the user may test a positive and negative strain as recommended or required by other regulations. The recommendations include the testing of ATCC $5 1 2 9 9 E .$ faecalis as a positive specimen processing control and for $E .$ . gallinarum ATCC 700425 as an external negative control.

# d. Detection limit:

Internal studies determined that the genomic limits of detection (LOD) for specific vanA and vanB are 5 and 10 DNA copies per reaction, respectively. The bacterial LOD for vanA and vanB are 1.6 and 1.4 CFU/reaction.

e. Analytical specificity:

Analytical specificity of the IDI-vanR ™ Assay was determined with wellcharacterized isolates of vancomycin-sensitive closely related genera, and other pathogenic and commensal flora found in the rectum and stools. These included Enterococcus spp. with the presence of other van genes (vanC, vanD, vanE, vanG). All provided a negative test result.

Potentially interfering substances were tested using materials that may be found in the rectum (petroleum jellies, creams, blood, suppositories). Blood showed a reduction in the $1 C \%$ Endpoint and hydrocortisone provided unresolved results when the swab was heavily covered in it.

# f. Assay cut-off:

Acceptance Criteria for the IDI- $. \mathrm { V a n } R ^ { \mathrm { T M } }$ were established during assay development and pre-validated at one external site with 216 specimens prior to the clinical trial. The acceptance criteria includes; Endpoint threshold, $2 ^ { \mathrm { n d } }$ derivative threshold, minimum cycle threshold, maximum cycle threshold, and $\%$ IC NC endpoint.

2. Comparison studies:

# a. Method comparison with predicate device:

A clinical study was conducted at 4 diverse sites from units with high VRE prevalence such as intensive care units. Each site performed routine VRE screening (culture on selective media [Bile Esculin azide agar with $6 ~ { \mu \mathrm { g / m L } }$ vancomycin (BEAV) plate) followed by phenotypic identification and detection of vancomycin and teicoplanin resistance]. An enriched culture method was also inoculated (Bile Esculin Azide broth with vancomycin 8 $\mu \mathrm { g } / \mathrm { m L } )$ and further cultured if the original culture was negative. A total of 968 rectal swab specimens were collected with one of the recommended swabs (refer to “Materials required but not provided”), screened for vancomycin-resistance with the reference culture method described above and with the IDI-VanR™ assay. Compared to the culture method, the IDI$\mathrm { V a n R } ^ { \mathrm { T M } }$ assay identified $9 7 . 3 \%$ of the positive specimens (by either culture technique), and $9 1 . 4 \%$ of the specimens negative (by both culture techniques). For the population tested, these results produced a negative predictive value of $9 9 . 6 \%$ and positive predictive value of $5 9 . 1 \%$ .

<table><tr><td rowspan="2" colspan="2"></td><td colspan="5">IDI-VANR ™M</td><td></td></tr><tr><td>VANA</td><td>VANB</td><td>VANA+B</td><td>NEGATIVE</td><td>UNRESOLVED</td><td>TOTAL</td></tr><tr><td rowspan="4">Reference Method</td><td>VanA</td><td>80</td><td>0</td><td>18</td><td>3</td><td>0</td><td>101</td></tr><tr><td>VanB</td><td>0</td><td>4</td><td>3</td><td>0</td><td>0</td><td>7</td></tr><tr><td>VanA + B</td><td>0</td><td>0</td><td>2</td><td>0</td><td>0</td><td>2</td></tr><tr><td>Negative</td><td>14</td><td>58</td><td>2</td><td>783</td><td>1</td><td>858</td></tr><tr><td></td><td>Total :</td><td>94</td><td>62</td><td>25</td><td>786</td><td>1</td><td>968</td></tr></table>

The IDI-vanR ™ “vanB “only column in the table above would be recommended for further confirmation according to the interpretation of this result. Of the 62 vanB only positives, 4 were culture vanB positive with the rest not positive fro VRE by culture.

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Clinical Sensitivity: Not Applicable

b. Clinical specificity: Not Applicable

c. Other clinical supportive data (when a. and b. are not applicable):

# Not Applicable

4. Clinical cut-off: Not Applicable

5. Expected values/Reference range:

The prevalence of VRE in the study population was $1 1 . 1 \%$ . The overall vanA prevalence was $1 0 . 6 \%$ and vanB was $0 . 9 \%$ .

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.