# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION ONLY TEMPLATE

A. 510(k) Number: K062948

B. Purpose for Submission: New instrument system

C. Measurand: Group B Streptococcus (S. agalactiae) DNA ( $_ 3 \cdot$ region adjacent to cfb gene)

D. Type of Test: Nucleic acid amplification assay system, automated

E. Applicant: Cepheid

F. Proprietary and Established Names: Cepheid Smart GBS™ Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3740 21 CFR 866.9 (c) (b), limitation to exemption

2. Classification: I (Not exempt)

3. Product code: NJR

4. Panel: 83 Microbiology

# H. Intended Use:

1. Intended use(s):

The Cepheid Smart GBS™ Assay performed on the Cepheid SmartCycler Dx System is a qualitative in vitro diagnostic test designed to detect Group B Streptococcus (GBS) DNA from vaginal/rectal swab specimens and Lim broth cultures. The test utilizes real-time polymerase chain reaction (PCR) for a unique gene-specific sequence amplification of Streptococcus agalactiae recovered from clinical samples and fluorogenic target specific hybridization for the detection of the amplified DNA.

2. Indication(s) for use:

The Smart $\mathrm { G B S ^ { T M } }$ Assay is intended for use as a method for rapid detection of GBS colonization in antepartum and intrapartum women.

3. Special conditions for use statement(s):

Prescription use only   
The use of the Smart GBS™ for intrapartum screening should not preclude the use of other strategies (e.g., antepartum testing). Intrapartum Smart GBS™ results are useful to identify candidates for intrapartum antibiotic prophylaxis when administration of intravenous antibiotics is not delayed pending results.   
The Smart GBS™ assay does not provide susceptibility results. Culture isolates are needed for performing susceptibility testing as recommended for penicillin-allergic women.

4. Special instrument requirements: Automated SmartCycler DX System (instrument, computer)

# I. Device Description:

The Smart GBS™ Assay contains the following reagents packaged in a test kit for detection of GBS DNA.

1. Sample Preparation Reagent   
2. Lysis Reagent   
3. Treatment Reagent   
4. Diluent Reagent   
5. Master Mix   
6. Positive Control   
7. Negative Control

The SmartCycler GBS™ Assay includes all reagents described above required for sample processing and for the simultaneous detection of the target GBS and an internal control (IC) to monitor the presence of inhibitors in the PCR assay in order to avoid falsenegative results. The GBS primers and probe detect a unique region of the S. agalactiae genome.

A vaginal/rectal swab is collected in Copan with Liquid Stuart Medium and transported to the laboratory for processing. Specimens can be prepared by either using the direct sample preparation or the enriched method. With the direct sample preparation, swabs are eluted, lysed, centrifuged, washed, and vortexed to release the chromosomal DNA. When using this method, testing can be completed in approximately 75 minutes where results can be viewed with growth curves. With the enriched sample preparation method, the swabs are placed in an enrichment broth after sample collection, and incubated overnight (18 to 24 hours) at $3 7 ^ { \circ } \mathrm { C }$ . Smart GBS™ can provide results the next day.

After extraction, an aliquot of the bacterial DNA is added to the reconstituted real-time PCR reagents. Dried PCR reagents (polymerase/inhibitor complex, primers for target and controls, Taqman probes for target and controls), dNTPs, internal DNA control are freeze-dried on beads. The resulting mixture is placed in a reaction tube and loaded into the SmartCycler Dx System for real-time PCR amplification and detection of target sequences.

For quality control, positive and negative controls are provided in the assay kit. One Positive Control and one Negative Control are processed for each assay run on the SmartCycler Dx System. The software automatically assigns the Positive Control to the second to last position, and the Negative Control to the last position.

External positive control monitors substantial reagent failure. The external negative control detects reagent or environmental contamination (or carry-over) by either S. agalactiae or GBS amplicons.

The internal control (IC) verifies functional PCR reagents and the absence of inhibition that would prevent PCR amplification. DNA for the IC is included in one of the reagent beads within the Smart GBS™ Assay. The response of the IC in the presence of inhibitors correlates with the response of target GBS in the presence of inhibitors.

The controls both complement and provide a level of redundancy. Both the IC and the external positive and negative run controls will react to degradation of enzyme because both require its activity. All will also respond to probe degradation. Only the IC will react to sample effects.

Before start of PCR reaction, the SmartCycler Dx System is programmed to perform an optical measurement or probe check on the optical channels associated with GBS target and Internal Control detection. The Probe Check control verifies reagent bead rehydration, appropriate tube filling, probe integrity, and dye stability. Probe Check is considered to PASS if optical measurements meet the validated acceptance criteria. If the Probe Check fails in either optical channel, the test will not continue.

The System Control Check for Temperature Control ensures that the SmartCycler Dx Instrument is operating within specification.

The Smart GBS™ assay is run on the automated Cepheid SmartCycler Dx System, also referred to as the SmartCycler II, (the same instrument system as the predicate device, IDI-Strep B Assay) consists of sixteen independently-controlled I-CORE (Intelligent Cooling/Heating Optical Reaction) modules or sites, each of which processes one sample at a time. Because the system controls the modules independently, different samples can be processed using different assay definitions in the same instrument. In addition, up to six instruments can be linked together to process up to 96 samples simultaneously.

During the amplification process, the I-CORE heater heats up and the fan cools down the reaction tube contents. Two optical blocks positioned within the I-CORE excite the dye molecules that make up the probes and detect the fluorescence emitted. By using probes labeled with different fluorescent dyes, up to four dyes can be amplified and detected simultaneously in a single reaction mixture. The optical system always collects data from all four channels. Because the emission spectra of fluorescent dyes can overlap, appropriate calibration and data analysis algorithms are used to separate the signal from each of the dyes.

The thermistors used to monitor the reaction chamber temperature are calibrated to $\pm 0 . 5 0 ^ { \circ } \mathrm { C }$ using National Institute of Standards and Technology (NIST)-traceable standards. The optical system is calibrated using standard concentrations of the individual unquenched dye-oligo standard to determine the spectral characteristics.

The instrument performs optics self-test before each run start to verify that the optical system is functioning properly.

To minimize instrument maintenance, the instrument is never exposed to any fluids. All fluids are completely contained within disposable single-use reaction tubes throughout all the amplification process. The PCR and detection reactants and products, including amplicons, are contained within the reaction tube to prevent cross contamination between patient specimens.

The software (version 1.7b) controls the operation of the I-CORE module, and collects, analyzes and interprets the acquired optical data. This software has been determined to be a moderate level of concern. This is the same software that is used with the predicate device, IDI-Strep B Assay.

# The Smart GBS™ Assay Kit

![](images/e1fb9d3bc5bd0fb4e70dd0b9e74e235dee86087814b06c37ac0a37eb541f4fd0.jpg)  
The SmartCycler Dx Instrument

![](images/10623617478bc0dd0f09e9fd0d88332bfadde1fbb1355a0aa1f2a109eb99af06.jpg)

# J. Substantial Equivalence Information:

1. Predicate device name(s): IDI-Strep B™ Assay

2. Predicate 510(k) number(s): K022504

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Rapid identification of GroupB Streptococcus</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Samples</td><td colspan="1" rowspan="1">vaginal/rectal swab specimensfrom ante-partum and intra-partum women</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Indications</td><td colspan="1" rowspan="1">Identification of GBScolonization</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Collection and TransportMedium</td><td colspan="1" rowspan="1">Copan with Liquid StuartMedium</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Sample Preparation</td><td colspan="1" rowspan="1">All reagents are provided inindividually packaged tubesfor manual sample preparation.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCR with I-CORE heating and coolingmodule. Detection:Fluorogenic target-specifichybridization</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Single use</td><td colspan="1" rowspan="1">Yes; single-use Smart GBSTMreagent beads, single-useSmart GBSTM reagent liquids,single-use CepheidSmartTubeTM reaction tubes</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Automated assay</td><td colspan="1" rowspan="1">Yes; amplification, detectionand result interpretation</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">External Run Controls</td><td colspan="1" rowspan="1">External positive and negativerun controls are required</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">External Sample ProcessingControls</td><td colspan="1" rowspan="1">Materials availablecommercially, but notrequired.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Nucleic acid amplification(DNA); real-time PCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">Cepheid SmartCycler DxSystem</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">DNA target sequence</td><td colspan="1" rowspan="1">3' region adjacent to cfb gene</td><td colspan="1" rowspan="1">cfb gene</td></tr><tr><td colspan="1" rowspan="1">Sample Processing</td><td colspan="1" rowspan="1">Direct Method: Thevaginal/rectal swab is placedin Sample Preparation Reagentand processed for real-timePCR amplification anddetectionEnriched Method: Thevaginal/rectal swab is placedinto Lim broth and incubatedovernight at 37°C, prior tobeing processed for real-timePCR amplification and</td><td colspan="1" rowspan="1">Direct Method only.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">detection.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Internal Assay and SystemControls</td><td colspan="1" rowspan="1">Internal Control; Probe Check(all optical channels). Failuresresult in single sample repeat.</td><td colspan="1" rowspan="1">Internal control; Site check (1optical channel). Same</td></tr><tr><td colspan="1" rowspan="1">Criteria for Cycle threshold(Ct) determination</td><td colspan="1" rowspan="1">Primary growth curve</td><td colspan="1" rowspan="1">ndderivative analysis</td></tr><tr><td colspan="1" rowspan="1">Probes</td><td colspan="1" rowspan="1">TaqMan® Probes</td><td colspan="1" rowspan="1">Molecular beacons</td></tr></table>

K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

The Cepheid Smart Cycler Dx System is an integrated nucleic acid amplification and detection instrument system based on the proprietary microprocessor-controlled ICORE $\textsuperscript { \textregistered }$ module.

During PCR cycling, optical signals from two sequence specific fluorescent probes within the reaction are monitored in real time. The probes correspond to the GBS target, and the Internal Control. In the presence of DNA, this resulting optical curve consists of a flat baseline section, a rapidly growing section, and a plateau section. The increase in fluorescence is proportional to the amount of amplicons generated. The values for Cycle threshold (Ct) and endpoint are used to determine the presence or absence of the GBS target and IC being detected.

All assay data and results are stored in a database.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A panel of specimens with varying concentrations of GBS and Lactobacillus acidophilus (negative) were tested in triplicate on 10 different days at each of the three sites (4 specimens $\times ~ 3 ~ \times ~ 1 0 ~ \mathrm { d a y s } \times 3$ sites). One lot of Smart GBS™ kits was used at each of the 3 testing sites, according to the Smart GBS™ procedure. Intersite and intrasite demonstrated ${ \tt > } 9 5 $ reproducibility.

The levels of cells in the preparation for precision evaluations are as follows:

1. Negative- Lactobacillus acidophilus ATCC $4 3 5 6 \sim 5 \mathrm { X } 1 0 ^ { 5 }$

2. Low Level – Streptococcus species (Group B, Type Ib) ATCC $1 2 4 0 1 { \sim } 5 \mathrm { X } 1 0 ^ { 4 }$

3. Moderate Level – Streptococcus species (Group B, Type Ib) ATCC 12401 \~5

4. High Level – Streptococcus species (Group B, Type Ib) ATCC 12401 \~5 X 10

Summary of Reproducibility Results   

<table><tr><td colspan="5">Specimen ID Site 1 Site 2 Site 3 Total Agreement Total %Agreement</td></tr><tr><td>Negative</td><td>30/30 30/30</td><td>29/30</td><td>89/90</td><td>98.9%</td></tr><tr><td>Weak Positive</td><td>29/30</td><td>30/30 30/30</td><td>89/90</td><td>98.9%</td></tr><tr><td>Positive</td><td>30/30</td><td>30/30 30/30</td><td>90/90</td><td>100%</td></tr><tr><td>Strong Positive</td><td>30/30</td><td>30/30 30/30</td><td>90/90</td><td>100%</td></tr><tr><td>Total Agreement</td><td></td><td> 119/120 120/120 119/120 358/360</td><td></td><td>99.4%</td></tr><tr><td>% Agreement</td><td>99.2%</td><td>100% 99.2%</td><td>99.4%</td><td></td></tr></table>

# b. Linearity/assay reportable range:

A stock concentration of GBS $\mathrm { 1 . 3 6 e 9 C F U / m L } )$ ) was serially diluted over 8 logs and loaded into each reaction tube with appropriate sample preparation reagents. Samples were tested in replicates of 3. Results demonstrated linearity of $( \mathbf { r } 2 = 0 . 9 9 9 1 $ ) GBS detection (FAM) as a function of GBS cell input over 7 logs. The reportable Ct range was 14.8 to 37.0 in this study. Results are summarized in the tables below.

![](images/f8828dfbf1636f281bb28127b706e468b52723313aa95603e119e1faff458e40.jpg)  
Dynamic Range (Concentration vs Ct) of Smart GBS™   
GBS SC Dynamic Range $( n = 3 )$

# Dynamic Range (Ct and EP Values) of Smart GBS™

![](images/0e0c567f0cec9585196d96189be8cfb0e1cd47a840c7b0edab973f34722ff718.jpg)

c. Traceability, Stability, Expected values (controls, calibrators, or methods): External positive and negative controls are required for each Smart GBS™ assay run.

Quality controls were performed on every test occasion with the SmartCycler producing acceptable QC results most of the time. Results of Cycle thresholds (Ct) were averaged and summarized in the tables below.

QC Results for Intrapartum/Antepartum Study   

<table><tr><td rowspan=1 colspan=1>Site No.</td><td rowspan=1 colspan=1>ExternalControl</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>*IC Ct(Mean)</td><td rowspan=1 colspan=1>IC Ct(SD)</td><td rowspan=1 colspan=1>IC CtAcceptableRange</td><td rowspan=1 colspan=1>GBS Ct(Mean)</td><td rowspan=1 colspan=1>GBSCt(SD)</td><td rowspan=1 colspan=1>GBS CtAcceptableRange</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>28  40</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.71</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>1.62</td><td rowspan=1 colspan=1>28  40</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>28 - 40</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>10 - 42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>10 - 42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>28 - 40</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>28 - 40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>1.59</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>35.09</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>10 - 42</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>34.55</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>28- 40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

\*Internal control (IC) verifies the performance of the PCR reagents and assesses the presence of sample inhibitors. The IC should be positive in a negative

sample and can be negative in a positive sample. The IC passes if it meets the validated acceptance criteria.

$\ddot { \mathbf { \eta } } ^ { * } \dot { \mathbf { \kappa } } \mathbf { N } / \mathbf { A } - \mathbf { N } \mathbf { o t }$ applicable (The GBS target for the Negative Control does not have a Ct within the valid range and/or does not have an endpoint above the endpoint minimum setting.)

QC Results for Antepartum Study   

<table><tr><td rowspan=1 colspan=1>Site No.</td><td rowspan=1 colspan=1>ExternalControl</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>*IC Ct(Mean)</td><td rowspan=1 colspan=1>IC Ct(SD)</td><td rowspan=1 colspan=1>IC CtAcceptableRange</td><td rowspan=1 colspan=1>GBS Ct(Mean)</td><td rowspan=1 colspan=1>GBSCt(SD)</td><td rowspan=1 colspan=1>GBS CtAcceptableRange</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.39</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>34.31</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>28 - 40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.39</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>34.12</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>10 - 42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.46</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.36</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>34.33</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>10-42</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>32.49</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>28- 40</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>10 - 42</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>28 - 40</td><td rowspan=1 colspan=1>**N/A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

\*Internal control (IC) verifies the performance of the PCR reagents and assesses the presence of sample inhibitors. The IC should be positive in a negative sample and can be negative in a positive sample. The IC passes if it meets the validated acceptance criteria.

$\ddot { \mathbf { \eta } } ^ { * } \dot { \mathbf { \kappa } } \mathbf { N } / \mathbf { A } - \mathbf { N } \mathbf { o t }$ applicable (The GBS target for the Negative Control does not have a Ct within the valid range and/or does not have an endpoint above the endpoint minimum setting.)

# Stability

Stability study was performed using one lot of Smart GBS™. Reagents beads being stored at $4 \mathrm { { ^ \circ C } }$ , $2 5 \mathrm { { ^ \circ C } }$ , $3 5 \mathrm { { ^ \circ C } }$ and $4 5 \mathrm { { ^ \circ C } }$ were evaluated at predefined time point intervals up to 24 months. Additionally assay reagent pouches being stored at $5 \mathrm { ^ \circ C }$ $+ / -$ - $3 ^ { \circ } \mathrm { C } )$ that have been opened and re-sealed to mimic use were tested at specific pre-defined time points.

Shelf-life for the Smart GBS™ is for a minimum of 6 months if the Smart GBS™ reagents are stored at room temperature $1 8 \mathrm { - } 2 5 ^ { \circ } \mathrm { C } )$ and for 12 months if the reagents are stored refrigerated at $4 \mathrm { { } ^ { \circ } C }$ $( 2 \small { - } 8 ^ { \circ } \mathrm { C } )$ .

# d. Detection limit:

The analytical sensitivity was determined using 11 S. agalactiae strains representing 9 GBS serotypes (Ia, Ib, II, III, IV, V, VII, and VIII). Dilutions were made to represent 250, 500, and 750 colony forming units (CFUs) per swab and tested in replicates of four. Each swab was processed using the Smart GBS™ (direct method). The lowest concentration of each subtype resulting in a positive result in all four replicates was 250 CFU/swab as shown in the table below.

# Limit of Detection Obtained for Each Serotype Tested

<table><tr><td>Serotype</td><td>CFU/Swab CFU/Reaction</td></tr><tr><td>ATCC 12973 (II) 250</td><td>4</td></tr><tr><td>Ia/c 250</td><td>4</td></tr><tr><td>Ib/c 500</td><td>8</td></tr><tr><td>I 250</td><td>4</td></tr><tr><td>III 250</td><td>4</td></tr><tr><td>IV 250</td><td>4</td></tr><tr><td>IVc 500</td><td>8</td></tr><tr><td>V 250</td><td>4</td></tr><tr><td>VI 500</td><td>8</td></tr><tr><td>VII 250</td><td>4</td></tr><tr><td>VIII 500</td><td>8</td></tr></table>

The Limit Of Detection (LOD) was evaluated in a separate study using groups of swabs $( \mathrm { n } { = } 2 0 )$ spiked with GBS Serotype II $( 7 5 ~ \mu \mathrm { L } / \mathrm { s w a b } )$ at 4 concentrations (100, 250, 500, and 750 CFU). The LOD was shown to be 750 CFU/swab (20/20 detected). Results are summarized in the chart below.

Smart GBS™ Limit of Detection Summary Results   

<table><tr><td rowspan=1 colspan=1>CFU per swab</td><td rowspan=1 colspan=1>Estimated CFU per rxn(8 μl lysate)</td><td rowspan=1 colspan=1># positive (n=20)</td></tr><tr><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>8.0</td><td rowspan=1 colspan=1>16/20</td></tr><tr><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>13/20</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>3/20</td></tr><tr><td rowspan=1 colspan=1>No GBS</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0/20</td></tr></table>

# e. Analytical specificity:

Purified DNA from 101 strains representing 28 Streptococci, 73 other species including strains phylogenetically related to Streptococcus agalactiae, other microflora (bacteria and yeasts) commonly found in vaginal/anal flora, and human DNA were tested. All strains were tested in triplicate at $1 . 5 \mathrm { n g }$ per $2 5 ~ \mu \mathrm { L }$ reaction (approximately 2e5 equivalent genome copies per reaction). None of the 28 Streptococcal isolates (nonGBS) tested positive. Of the remaining 73 strains, 4 (Enterococcus gallinarium, Staphylococcus simulans, Micrococcus luteus and Propionibacterium acnes) were weakly positive in one of six replicates.

# Interference Testing-Endogenous Substances

Thirteen samples of amniotic fluid contaminated with meconium, blood or human blood, human sera, urine, feces, and 2 types of lubricant were tested to evaluate the effects of potentially interfering endogenous substances that may be present in vaginal/rectal specimens on the Smart GBS™. All substances were tested undiluted or “neat” at concentrations close to saturation. Results showed that high concentrations of interfering substances, more than typically encountered in clinical specimens, did not interfere with the qualitative interpretation of results below the LOD (e.g., all GBS samples were still positive even though signals were reduced), except for feces. In the presence of feces, the IC signal was reduced with up to $94 \%$ GBS inhibition which

suggests that IC would likely fail in the presence of high level of feces.

Of the 774 patients in the final dataset, 235 specimen swabs were observed to contain at least one of the following pre-identified substances that could potentially interfere with the assay. The list of contaminants and the performance of the Smart GBS ™assay is shown in the table below.

Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Contaminants</td><td rowspan=1 colspan=1>No.of1Specimens</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>All contaminants</td><td rowspan=1 colspan=1>235</td><td rowspan=1 colspan=1>82.14</td><td rowspan=1 colspan=1>96.59</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>90.00%</td><td rowspan=1 colspan=1>96.67%</td></tr><tr><td rowspan=1 colspan=1>Feces</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>87.50%</td><td rowspan=1 colspan=1>94.29%</td></tr><tr><td rowspan=1 colspan=1>Mucous</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>70.59%</td><td rowspan=1 colspan=1>95.52%</td></tr><tr><td rowspan=1 colspan=1>Lubricant</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>60.00%</td><td rowspan=1 colspan=1>96.97%</td></tr><tr><td rowspan=1 colspan=1>2Meconium</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Amniotic Fluid</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>86.67%</td><td rowspan=1 colspan=1>98.25%</td></tr></table>

1Note: some specimens contain more than one of the above substances. 2Three specimens: 2 True Negative and 1 True Positive

f. Assay cut-off:

The following table demonstrates the acceptance criteria to distinguish a positive result from a negative result which is required for interpretation of data.

Acceptance Criteria for Smart GBS™ Assay   

<table><tr><td rowspan=2 colspan=1>AcceptanceCriteria</td><td rowspan=2 colspan=1>GBS Target(FAM)</td><td rowspan=2 colspan=1>InternalControl(TxRed)</td><td rowspan=1 colspan=2>Positive Control</td><td rowspan=1 colspan=2>Negative Control</td></tr><tr><td rowspan=1 colspan=1>(FAM)</td><td rowspan=1 colspan=1>(TxRed)</td><td rowspan=1 colspan=1>(FAM)</td><td rowspan=1 colspan=1>(TxRed)</td></tr><tr><td rowspan=1 colspan=1>ManualThreshold</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Min Endpoint</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Min Cyclethreshold</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Max Cyclethreshold</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Probe Check</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr></table>

The acceptance criteria for GBS target were determined using valid minimum and maximum GBS Ct settings derived from analytical data including simulated inhibitory studies (Ferric Ammonium Chloride (FAC) testing), GBS-positive endogenous inhibitory samples (ES), and pre-clinical data using the direct sample preparation method. Two kit lots are represented in the pre-clinical positives. Statistical representation of data is presented in the table below.

Statistical Representation of Data used to Determine Acceptance Criteria for the Target GBS   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Valid GBS Ct Range</td><td rowspan=1 colspan=1>FAC</td><td rowspan=1 colspan=1>Endogenoussubstances</td><td rowspan=1 colspan=1>Pre-Clinicalpositives(5 μL lysate)</td><td rowspan=1 colspan=1>Pre-Clinicalpositives(10 μL lysate)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n=49)</td><td rowspan=1 colspan=1>(n=38)</td><td rowspan=1 colspan=1>(n=56)</td><td rowspan=1 colspan=1>(n=57)</td></tr><tr><td rowspan=1 colspan=1>GBS</td><td rowspan=1 colspan=1>10-42</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>mean</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>31.2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>5.9</td></tr><tr><td rowspan=1 colspan=1>min</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1>18.9</td></tr><tr><td rowspan=1 colspan=1>max</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>41.9</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>40.5</td></tr></table>

The determination of acceptance criteria for the internal control were determined using valid minimum and maximum IC Cycle treshold $\mathrm { ( C t ) }$ settings derived from analytical data (GBS-negative bead release testing over 4 lots), simulated inhibitory studies (FAC testing), GBS-negative endogenous inhibitory samples (ES), and pre-clinical data using the direct method. Two kit lots are represented in the pre-clinical negatives. Statistical representation is detailed in the table below.

Statistical Representation of Data used to Determine Acceptance Criteria for the Internal Control (IC)   

<table><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>Valid IC Ct Range</td><td rowspan=1 colspan=1>Bead Release</td><td rowspan=1 colspan=1>FAC</td><td rowspan=1 colspan=1>Endogenoussubstances</td><td rowspan=1 colspan=1>Pre-Clinical negatives(10 μL lysate)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>(n=196)</td><td rowspan=1 colspan=1>(n=20)</td><td rowspan=1 colspan=1>(n=21)</td><td rowspan=1 colspan=1>(n=177)</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>28-40</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>mean</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>34.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>min</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>max</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>37.8</td></tr></table>

2. Comparison studies:

# a. Method comparison with predicate device:

Double swab sets were used for clinical evaluation, for a total of 4 swabs from each subject. For the first double swab collected, one swab was used for the culture procedure and the second swab for the Smart GBS™. For the second double swab, one swab was used for the Xpert GBS™ Assay and the other swab for the predicate nucleic acid amplification test (NAAT) assay. The predicate NAAT assay targets a sequence in the cfb gene and was previously FDA-cleared however results from this test were not used in performance estimations.

Direct Comparison of the Two PCR Tests from the Intrapartum/Antepartum Study   

<table><tr><td rowspan="2">2nd NAAT Neg</td><td rowspan="2" colspan="2">Culture Positive 2nd NAAT</td><td colspan="3">Culture Negative</td><td colspan="3">Culture ND</td><td rowspan="2"></td></tr><tr><td>2nd NAAT Pos</td><td>2nd NAAT Neg</td><td>Unresolved</td><td>2nd NAAT Pos</td><td>2nd NAAT Neg</td><td>Unresolved</td></tr><tr><td>137</td><td>13</td><td>1</td><td>12</td><td>9</td><td>0</td><td>0</td><td>0</td><td>0</td><td>172</td></tr><tr><td>13</td><td>21</td><td>0</td><td>13</td><td>554</td><td>1</td><td>0</td><td>5</td><td>0</td><td>607</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>0 150</td><td>0 34</td><td>0 1</td><td>2 27</td><td>12 575</td><td>0 1</td><td>0 0</td><td>0 5</td><td>0 0</td><td>14 793</td></tr></table>

Direct Comparison of the Two PCR Tests from the Antepartum (only) Study   

<table><tr><td rowspan="2">2nd NAAT</td><td rowspan="2" colspan="2">Culture Positive 2nd NAAT</td><td colspan="3">Culture Negative</td><td colspan="3">Culture ND</td><td rowspan="2"></td></tr><tr><td>2nd NAAT Pos</td><td>2nd NAAT Neg</td><td>Unresolved</td><td>2nd NAAT Pos</td><td>2nd NAAT</td><td>Unresolved</td></tr><tr><td>61</td><td>13</td><td>0</td><td>5</td><td>17</td><td>0</td><td>0</td><td>Neg 0</td><td>0</td><td>96</td></tr><tr><td>0</td><td>1</td><td>0</td><td>4</td><td>205</td><td>0</td><td>0</td><td>0</td><td>0</td><td>210</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>0 61</td><td>0 14</td><td>0 0</td><td>0 9</td><td>0 222</td><td>0 0</td><td>0 0</td><td>0 0</td><td>0 0</td><td>0 306</td></tr></table>

nd Performance of Smart GBS™ and 2 NAAT Test Relative to Culture from the Intrapartum/Antepartum Study   

<table><tr><td></td><td colspan="6">Smart GBSTM</td><td colspan="6">2nd NAAT</td></tr><tr><td>Category</td><td>Sensitivity</td><td>Low CI</td><td>Upper CI</td><td>Specificity</td><td>Lower CI</td><td>Upper CI</td><td>Sensitivity</td><td>Low CI</td><td>Upper cI</td><td>Specificity</td><td>Lower CI</td><td>Upper CI</td></tr><tr><td>Overall</td><td>81.6% (151/185)</td><td>75.3%</td><td>86.9%</td><td>96.4% (568/589)</td><td>94.6%</td><td>97.8%</td><td>81.5% (150/184)</td><td>75.2%</td><td>86.9%</td><td>95.5% (575/602)</td><td>93.5%</td><td>97.0%</td></tr><tr><td>Antepartum</td><td>78.3% (72/92)</td><td>68.4%</td><td>86.2%</td><td>95.6% (261/273)</td><td>92.5%</td><td>97.7%</td><td>79.4% (73/92)</td><td>69.6%</td><td>87.1%</td><td>96.1% (269/280)</td><td>93.1%</td><td>98.0%</td></tr><tr><td>Intrapartum</td><td>84.9% (79/93)</td><td>76.0%</td><td>91.5%</td><td>97.2% (307/316)</td><td>94.7%</td><td>98.7%</td><td>83.7% (77/92)</td><td>74.5%</td><td>90.6%</td><td>95.0% (306/322)</td><td>92.1%</td><td>97.1%</td></tr><tr><td>ROM 2</td><td>87.0% (20/23)</td><td>66.4%</td><td>97.2%</td><td>96.2% (101/105)</td><td>90.5%</td><td>99.0%</td><td>90.9% (20/22)</td><td>70.8%</td><td>98.9%</td><td>95.2% (100/105)</td><td>89.2%</td><td>98.4%</td></tr><tr><td>Not ROM 3</td><td>84.3% (59/70)</td><td>73.6%</td><td>91.9%</td><td>97.6% (206/211)</td><td>94.6%</td><td>99.2%</td><td>81.4% (57/70)</td><td>70.3%</td><td>89.7%</td><td>94.9% (206/217)</td><td>91.1%</td><td>97.4%</td></tr></table>

The 2  NAAT targets a sequence in the $c f b$ gene and was previously FDA-cleared. Subset of intrapartum women who had specimens collected after membrane rupture (rupture of membrane, ROM). 3 Subset of intrapartum women who had specimens collected before membrane rupture (ROM).

# nd Performance of Smart GBS™ and 2 NAAT Test Relative to Culture from the Antepartum (only) Study

<table><tr><td></td><td colspan="6">Smart GBSTM</td><td colspan="6">2nd NAAT</td></tr><tr><td>Category</td><td>Sensitivity</td><td>Low CI</td><td>Upper CI</td><td>Specificity</td><td>Lower CI</td><td>Upper CI</td><td>Sensitivity</td><td>Low CI</td><td>Upper CI</td><td>Specificity</td><td>Lower CI</td><td>Upper CI</td></tr><tr><td>Antepartum</td><td>98.7% (74/75)</td><td>92.8%</td><td>100.0%</td><td>90.5% (209/231</td><td>85.9%</td><td>93.9%</td><td>81.3% (61/75)</td><td>70.7%</td><td>89.4</td><td>96.1% (222/231)</td><td>92.7%</td><td>98.2%</td></tr></table>

The 2  NAAT targets a sequence in the $c f b$ gene and was previously FDA-cleared.

b. Matrix comparison: Not Applicable

3. Clinical studies:

Performance characteristics of the Smart GBS ™ Assay were determined in two multi-site studies. The first study was conducted at six institutions and included intrapartum and antepartum subjects using the direct method of inoculation. The second study included antepartum subjects only using the enriched method of inoculation and was conducted at 3 institutions with maternity services in the United States. Each institution had a culture-based or NAAT based screening program. Testing was done in clinical laboratories affiliated with each institution as well as the labor and delivery area.

To be enrolled in the intrapartum portion of the study, women had to provide written consent, be in labor, and have no contraindication to vaginal examination (for example, bleeding). To be enrolled in the antepartum portion of the studies, women had to provide written consent, be at 35-37 weeks gestation, and have no contraindication to vaginal examination (for example, bleeding). There was also no evidence of placenta previa, no urgent indication to proceed to delivery, and no antibiotic used in the week prior to admission for all subjects.

Vaginal/rectal specimens were collected from each of the 793 eligible subjects in the intrapartum/antepartum study and 307 eligible subjects in the antepartum study. In the intrapartum/antepartum study, specimens were collected using two sets of double-marked swabs. One of the swabs from the first set was used for culturing and the other swab was used for Smart GBS™ Assay testing by direct method on the SmartCycler Dx System. The second set of double-marked swabs was divided as follows: one swab was used in the Xpert GBS Assay on the GeneXpert Dx System; the other was used in the second NAAT assay. After use in these tests, each of these swabs was also placed into Lim culture broth, incubated, subcultured onto blood agar plate (BAP) and observed for GBS.

In the antepartum study, specimens were collected using one double-marked swab. One of the swabs was placed into Lim culture broth, incubated, subcultured onto blood agar plate (BAP) and observed for GBS. A portion of the Lim culture broth was tested by Smart GBS™ Assay using the enriched method on the

SmartCycler Dx System. The second swab was used in the secondary NAAT assay, as described above. After use in the second NAAT assay, the swab was also placed into Lim culture broth, incubated, subcultured onto blood agar plate (BAP) and observed for GBS.

The reference method used was the culture technique recommended in the 2002 CDC guidelines “Prevention of Perinatal Group B Streptococcal Disease: A Public Health Perspective” i.e., microbiological culture in selective broth medium (Lim broth, which is Todd-Hewitt broth supplemented with $1 5 ~ { \mu \mathrm { g / m l } }$ of nalidixic acid, and $1 0 ~ \mathrm { \mu g / m l }$ of colistin), followed by $1 8 { - } 2 4 \mathrm { h }$ incubation, and subculture onto BAP. Specific identification of colonies suggestive of GBS was done with slide agglutination tests.

# Overall Results

The performance characteristics of the Smart GBS™ Assay were evaluated from laboratory testing done in two clinical trials. In a trial of intrapartum and antepartum maternity patients, 774 specimens were analyzed (409 intrapartum and 365 antepartum). In a trial of antepartum only maternity patients, 306 specimens were evaluated. All subjects had culturing done (as described above) and most also had a second GBS NAAT performed. The second NAAT targets a sequence in the $c f b$ gene and was previously FDA-cleared; results from this test were not used in performance estimations.

Of the 793 eligible patients for the intrapartum/antepartum study, there were 723 reportable results with the Smart $\mathrm { G B S ^ { T M } }$ assay on the first attempt $( 9 1 . 2 \% )$ . Seventy samples did not have reportable results (37 invalids due to failed internal control and 33 tests due to failed external control in 4 invalid runs) however, 56 of these cases resolved upon repeat testing. Overall, 185 women had cultures positive for GBS from the single swab used for culture. The Smart GBS™ assay yielded 151 positive results initially $( 1 5 1 / 1 8 5 , 8 1 . 6 \% )$ . After repeat testing, the positive remained at $8 1 . 6 \%$ . Six hundred three women had negative cultures and 545 were negative initially with Smart GBS™ testing $( 9 0 . 4 \% )$ , and 568 after repeat testing $( 9 4 . 2 \% )$ .

The table below shows Smart GBS™ testing based on the positive and negative GBS culture findings for 774 subjects (14 subjects had no Smart $\mathrm { G B S ^ { T M } }$ results after 2 attempts and 5 cultures were overgrown or could not be otherwise interpreted). Summary of results are based upon after repeat-testing. There were 14 cases that remained unresolved even after repeat testing $( \mathtt { n } = 3 7 _ { . }$ ).

# Smart GBS™ Results and Estimated Performance by Patient Category from the Intrapartum/Antepartum Study

<table><tr><td rowspan=1 colspan=1>All Patients</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CulturePossitivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV 6[95% confidence]</td><td rowspan=1 colspan=1>NPV7[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>172(172)</td><td rowspan=1 colspan=1>151(151)</td><td rowspan=1 colspan=1>21(21)</td><td rowspan=4 colspan=1>81.6%[75.3%-86.9%]</td><td rowspan=4 colspan=1>96.3%[94.4%-97.7%]</td><td rowspan=4 colspan=1>87.8%[81.9%-92.3%]</td><td rowspan=4 colspan=1>94.1%[91.9%-95.9%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>602(579)</td><td rowspan=1 colspan=1>34(34)</td><td rowspan=1 colspan=1>568(545)</td></tr><tr><td rowspan=1 colspan=1>No Result²</td><td rowspan=1 colspan=1>14(37)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>14(37)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>1853</td><td rowspan=1 colspan=1>603</td></tr></table>

All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $8 1 . 6 \%$ , specificity is $9 6 . 4 \%$ , PPV is $8 7 . 8 \%$ , and NPV is $9 4 . 4 \%$ .

<table><tr><td rowspan=1 colspan=1>Ante-partum</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CultureositivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV 6[95% confidence]</td><td rowspan=1 colspan=1>NPV7[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>84(84)</td><td rowspan=1 colspan=1>72(72)</td><td rowspan=1 colspan=1>12(12)</td><td rowspan=4 colspan=1>78.3%[68.4%-86.2%]</td><td rowspan=4 colspan=1>95.4%[92.2%-97.6%]</td><td rowspan=4 colspan=1>85.7%[76.4%-92.4%]</td><td rowspan=4 colspan=1>92.6%[88.8%-95.4%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>281(221)</td><td rowspan=1 colspan=1>20(20)</td><td rowspan=1 colspan=1>261(251)</td></tr><tr><td rowspan=1 colspan=1>No Result2</td><td rowspan=1 colspan=1>7(17)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>7(20)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>372</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>280</td></tr></table>

All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $7 8 . 3 \%$ , specificity is $9 5 . 6 \%$ , PPV is $8 5 . 7 \%$ , and NPV is $9 2 . 9 \%$ .

<table><tr><td rowspan=1 colspan=1>Intra-partum</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CultureositivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV 6[95% confidence]</td><td rowspan=1 colspan=1>NPV7[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>88(88)</td><td rowspan=1 colspan=1>79(79)</td><td rowspan=1 colspan=1>9(9)</td><td rowspan=4 colspan=1>85.0%[76.0%-91.5%]</td><td rowspan=4 colspan=1>97.0%[94.4%-98.6%]</td><td rowspan=4 colspan=1>89.8%[81.5%-95.2%]</td><td rowspan=4 colspan=1>95.5%[92.5%-97.5%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>321(308)</td><td rowspan=1 colspan=1>14(14)</td><td rowspan=1 colspan=1>307(294)</td></tr><tr><td rowspan=1 colspan=1>No Result2</td><td rowspan=1 colspan=1>7(20)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>7(20)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>416</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>323</td></tr></table>

All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $8 4 . 9 \%$ , specificity is $9 7 . 2 \%$ , PPV is $8 9 . 8 \%$ , and NPV is $9 5 . 6 \%$ .

<table><tr><td rowspan=1 colspan=1>ROM4</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CulturePostivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV 6[95% confidence]</td><td rowspan=1 colspan=1>NPV7[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>24(24)</td><td rowspan=1 colspan=1>20(20)</td><td rowspan=1 colspan=1>4(4)</td><td rowspan=4 colspan=1>87.0%[66.4%-97.2%]</td><td rowspan=4 colspan=1>96.0%[90.1%-98.9%]</td><td rowspan=4 colspan=1>83.3%[62.6%-95.3%]</td><td rowspan=4 colspan=1>97.0%[91.4%-99.4%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>104(99)</td><td rowspan=1 colspan=1>3(3)</td><td rowspan=1 colspan=1>101(96)</td></tr><tr><td rowspan=1 colspan=1>No Result2</td><td rowspan=1 colspan=1>1(6)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>1(6)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>106</td></tr></table>

All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $8 7 . 0 \%$ , specificity is $9 6 . 2 \%$ , and NPV $9 7 . 1 \%$ .

<table><tr><td rowspan=1 colspan=1>No ROM 5</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CultureositivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV 6[95% confidence]</td><td rowspan=1 colspan=1>NPV7[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>64(64)</td><td rowspan=1 colspan=1>59(59)</td><td rowspan=1 colspan=1>5(5)</td><td rowspan=4 colspan=1>84.3%[73.6%-91.9%]</td><td rowspan=4 colspan=1>97.6%[94.4%-99.2%]</td><td rowspan=4 colspan=1>92.2%[82.7%-97.4%]</td><td rowspan=4 colspan=1>94.8%[90.8%-97.4%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>217(210)</td><td rowspan=1 colspan=1>11(11)</td><td rowspan=1 colspan=1>206(199)</td></tr><tr><td rowspan=1 colspan=1>No Result2</td><td rowspan=1 colspan=1>6(13)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>6(13)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>217</td></tr></table>

All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $8 4 . 3 \%$ , specificity is $9 7 . 6 \%$ , PPV is $9 2 . 2 \%$ , and NPV is $9 4 . 9 \%$ .

2 ‘No results’ from an Smart GBS test could be due to an invalid test because of failed internal control or   
invalid run, or system error.

Overall prevalence of GBS colonization as determined by culture is $2 3 . 5 \%$

The subset of intrapartum women who had specimens collected after membrane rupture (rupture of membrane, ROM).

The subset of intrapartum women who had specimens collected before membrane rupture (ROM). There would be no biological differences expected between these intrapartum specimens and those collected antepartum.

In the antepartum study, there were 307 eligible patients. Only 306 samples were analyzed since one sample was not provided. Of the 306 cases, the Smart GBS™ assays yielded 301 reportable results on the first attempt $( 9 8 . 4 \% )$ . There were 5 nonreportable results (2 were invalid due to failed internal control and 3 due to failed external control in 1 invalid run) however all of these cases resolved upon repeat testing. Overall, 75 women had positive culture for GBS from the single swab used for culture. The Smart GBS™ assay yielded 74 positive results initially $( 9 8 . 7 \% )$ which remained the same after repeat testing. Two hundred thirty one women had negative cultures and 207 were negative initially with Smart GBS™ testing $( 8 9 . 6 \% )$ , and 209 after repeat testing $( 9 0 . 5 \% )$ . The results of this clinical trial are summarized in the table below.

Smart GBS™ LB (Enriched Method) Results and Estimated Performance from the Antepartum (only) Study   

<table><tr><td rowspan=1 colspan=1>Ante-partum</td><td rowspan=1 colspan=1>Total N 1</td><td rowspan=1 colspan=1>CulturePositivePatients</td><td rowspan=1 colspan=1>CultureNegativePatients</td><td rowspan=1 colspan=1>Sensitivity[95% confidence]</td><td rowspan=1 colspan=1>Specificity[95% confidence]</td><td rowspan=1 colspan=1>PPV^3}$[95% confidence]</td><td rowspan=1 colspan=1>NPV4[95% confidence]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSPos</td><td rowspan=1 colspan=1>96(96)</td><td rowspan=1 colspan=1>74(74)</td><td rowspan=1 colspan=1>22(22)</td><td rowspan=4 colspan=1>98.7%[92.8%-100%]</td><td rowspan=4 colspan=1>90.4%[85.8%-93.9%]</td><td rowspan=4 colspan=1>77.1%[67.4%-85.1%]</td><td rowspan=4 colspan=1>99.5%[97.4%-0.00%]</td></tr><tr><td rowspan=1 colspan=1>Smart GBSNeg</td><td rowspan=1 colspan=1>210(208)</td><td rowspan=1 colspan=1>1(1)</td><td rowspan=1 colspan=1>209(207)</td></tr><tr><td rowspan=1 colspan=1>No Result2</td><td rowspan=1 colspan=1>0(2)</td><td rowspan=1 colspan=1>0(0)</td><td rowspan=1 colspan=1>0(2)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>306</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>231</td></tr></table>

Positive predictive value. Negative predictive value.   
All invalid results were repeated as described in the package insert. The values for repeated sensitivity is $9 8 . 7 \%$ , specificity is $9 0 . 5 \%$ , PPV is $7 7 . 1 \%$ , and NPV is $9 9 . 5 \%$ .

a. Clinical Sensitivity: Not Applicable

b. Clinical specificity:

Not Applicable c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Reported as Positive or Negative

5. Expected values/Reference range:

Positive - GBS target DNA is detected (presumptive for GBS colonization).   
External Positive and Negative Run Controls are valid. • Probe check results pass. The GBS target has a Ct within the valid range and endpoint above the endpoint minimum setting. IC is ignored since GBS amplification may compete with this control.

Negative - GBS target DNA is not detected (presumed not colonized with GBS), IC meets acceptance criteria. External Positive and Negative Run Controls are valid.

• Probe check results pass. The GBS target does not have a Ct within the valid range and/or does not have an endpoint above the endpoint minimum setting. The IC has a Ct within the valid range and endpoint above the endpoint minimum setting.

Invalid - Presence or absence of GBS DNA cannot be determined. One or both of the following may apply:

Individual patient sample - The Target result is negative and the IC failed. Invalid results are typically caused by inhibition of the PCR amplification of both the GBS template and the IC. External Positive Control and/or Negative Control - The control did not meet acceptance criteria. This invalidates the entire run.

ND - Presence or absence of GBS DNA cannot be determined. An I-CORE module error was detected during analysis of sample.

No Result - Presence or absence of GBS DNA cannot be determined. This result is reported if an insufficient number of cycles were completed, which may happen if the test was stopped by operator before the run was completed or a power failure occurred during the run.

When performing intrapartum testing; repeat testing may not be feasible and will depend on practices and policies within each facility. Coordination between clinicians and the testing laboratory is important to not delay administration of antibiotics while results are pending.

Repeat the test if one or more of the following occurs:

External positive and/or negative run controls have results of Invalid, ND, or No Result. Retest the entire run with new external positive and negative run controls.   
Individual samples (not the external positive or negative run controls) have results of Invalid, ND, or No Result. Retest only those samples with new external positive and negative run controls.

Approximately $10 { - } 3 0 \%$ of pregnant women are colonized with GBS in the vagina or rectum. GBS colonization can be transient, chronic or intermittent. Culture screening of both the vagina and rectum for GBS late in gestation during prenatal care can detect women who are likely to be colonized with GBS at the time of delivery. In various studies, sensitivities of $8 7 \%$ $8 3 - 9 2 \%$ CI) and $69 \%$ ( $5 7 - 7 9 \%$ CI) and specificities of $9 6 \%$ $( 9 5 - 9 8 \%$ CI) and $9 2 \%$ $89 - 9 4 \%$ CI) have been reported for late-prenatal cultures for identifying colonization status at delivery. During clinical evaluations for the Smart $\mathrm { G B S ^ { T M } }$ assay, $2 3 . 8 \%$ (260/1094) women were colonized with GBS by culture methods.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.