# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K063664   
B. Purpose for Submission: New device clearance   
C. Measurand:   
D. Type of Test: Transcription-Mediated Amplification (TMA), and Hybridization Protection Assay (HPA)   
E. Applicant: Gen-Probe, Inc   
F. Proprietary and Established Names: Gen-Probe APTIMA Assay for Neisseria gonorrhoeae (TIGRIS Application)

# G. Regulatory Information:

1. Regulation section: 866.3390

2. Classification: I

3. Product code: LSL

4. Panel: Microbiology

# H. Intended Use:

The APTIMA $\textsuperscript { \textregistered }$ Assay for Neisseria gonorrhoeae is a target amplification nucleic acid probe test that utilizes target capture for the in vitroqualitative detection of ribosomal RNA (rRNA) from Neisseriagonorrhoeae (GC) to aid in the diagnosis of gonococcal urogenital disease using the TIGRIS $\textsuperscript { \textregistered }$ DTS $\textsuperscript { \textregistered }$ Automated Analyzer or semi-automated instrumentation as specified. The assay may be used to test the

following specimens from symptomatic individuals: clinician-collected endocervical, vaginal and male urethral swab specimens; and female and male urine specimens. The assay may be used to test the following specimens from asymptomatic individuals: clinician-collected endocervical and vaginal swab specimens; patientcollected vaginal swab specimens1; and female and male urine specimens. The assay is also intended for use with the testing of gynecological specimens, from both symptomatic and asymptomatic patients, collected in the PreservCyt Solution and processed with the Cytyc ThinPrep 2000 System.

1Patient-collected vaginal swab specimens are an option for screening women when a pelvic exam is not otherwise indicated. The vaginal swab specimen collection kit is not for home use.

2. Indication(s) for use: NA

3. Special conditions for use statement(s): NA

4. Special instrument requirements:

Gen-Probe TIGRIS DTS

# I. Device Description:

The APTIMA CT Assay combines the technologies of target capture, TMA and HPA. Specimens are collected and transferred into their respective specimen transport tubes. The transport solution in these tubes releases the rRNA target and protects it from degradation during storage. When the APTIMA CT Assay is performed in the laboratory, the target rRNA molecule is isolated from the specimens by the use of a capture oligomer in a method called target capture; magnetic micro particles are another key feature of target capture. The capture oligomer contains a sequence complementary to a specific region of the target molecule as well as a string of deoxyadenosine residues. During the hybridization step, the sequence specific region of the capture oligomer binds to a specific region of the target molecule. The capture oligomer:target complex is then captured out of solution by decreasing the temperature of the reaction to room temperature. This temperature reduction allows hybridization to occur between the deoxyadenosine region on the capture oligomer and the poly-deoxythymidine molecules that are covalently attached to the magnetic particles. The micro particles, including the captured target molecule bound to them, are pulled to the side of the reaction vessel using magnets and the supernatant is aspirated. The particles are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the specimens are ready for amplification. Target amplification assays are based on the ability of complementary oligonucleotide primers to specifically anneal and allow enzymatic amplification of the target nucleic acid strands. The Gen-Probe TMA reaction replicates a specific region of the rRNA from CT via DNA intermediates. A unique set of primers is used for the target molecule. Detection of the rRNA

amplification product sequences (amplicon) is achieved using nucleic acid hybridization. A single-stranded chemiluminescent DNA probe, which is complementary to a region of the target amplicon, is labeled with an acridinium ester molecule. The labeled DNA probe combines with amplicon to form stable RNA:DNA hybrids. The Selection Reagent differentiates hybridized from unhybridized probe, eliminating the generation of signal from unhybridized probe. During the detection step, light emitted from the labeled RNA:DNA hybrids is measured as photon signals in a luminometer, and are reported as Relative Light Units (RLU).

The device is similar to the Gen-Probe Aptima Assay for Chlamydia trachomatis in that the target organism is $C .$ . trachomatis. The primary difference between the two assays is the platform on which the assays are performed. The TIGRIS fully automates all steps necessary to perform the ACT Assay from sample processing through amplification, detection, and data reduction.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Gen-Probe APTIMA Assay for Chlamydia trachomatis

2. Predicate 510(k) number(s): K053446

3. Comparison with predicate:

The reagents for the TIGRIS DTS APTIMA CT Assay are unchanged from the initial submission (K053446). No new QC test methods were developed and/or validated for any material or reagents since the QC test methods have been developed and/or validated for the ACT Assay on DTS Systems. There are no new materials that require further QC testing.

The Magellan system software is used for both the AC2 and ACT Assays, and since the initial clearance of the AC2 Assay on the TIGRIS DTS System, there have been no substantive changes to the hardware or system software components of the TIGRIS DTS System. This application supports modifications to the package insert for the TIGRIS DTS APTIMA CT Assay test system as a result of clinically validating all specimen types on the TIGRIS DTS System for the ACT Assay.

# K. Standard/Guidance Document Referenced (if applicable): NA

# L. Test Principle:

The APTIMA CT Assay combines the technologies of target capture, TMA and HPA. Specimens are collected and transferred into their respective specimen transport tubes. The transport solution in these tubes releases the rRNA target and protects it from degradation during storage. When the APTIMA CT Assay is performed in the laboratory, the target rRNA molecule is isolated from the specimens by the use of a capture oligomer in a method called target capture; magnetic micro particles are

another key feature of target capture. The capture oligomer contains a sequence complementary to a specific region of the target molecule as well as a string of deoxyadenosine residues. During the hybridization step, the sequence specific region of the capture oligomer binds to a specific region of the target molecule. The capture oligomer:target complex is then captured out of solution by decreasing the temperature of the reaction to room temperature. This temperature reduction allows hybridization to occur between the deoxyadenosine region on the capture oligomer and the poly-deoxythymidine molecules that are covalently attached to the magnetic particles. The micro particles, including the captured target molecule bound to them, are pulled to the side of the reaction vessel using magnets and the supernatant is aspirated. The particles are washed to remove residual specimen matrix that may contain amplification reaction inhibitors. After the target capture steps are completed, the specimens are ready for amplification. Target amplification assays are based on the ability of complementary oligonucleotide primers to specifically anneal and allow enzymatic amplification of the target nucleic acid strands. The Gen-Probe TMA reaction replicates a specific region of the 16S rRNA from CT via DNA intermediates. A unique set of primers is used for the target molecule. Detection of the rRNA amplification product sequences (amplicon) is achieved using nucleic acid hybridization. A single-stranded chemiluminescent DNA probe, which is complementary to a region of the target amplicon, is labeled with an acridinium ester molecule. The labeled DNA probe combines with amplicon to form stable RNA:DNA hybrids. The Selection Reagent differentiates hybridized from unhybridized probe, eliminating the generation of signal from unhybridized probe. During the detection step, light emitted from the labeled RNA:DNA hybrids is measured as photon signals in a luminometer, and are reported as Relative Light Units (RLU)

# M. Performance Characteristics (if/when applicable): See Attached

# DTS Systems Expected Values

# Prevalence

The prevalence of CT in patient populations depends on risk factors such as age, gender, the presence of symptoms, the type of clinic, and the test method. A summary of the prevalence of CT, by specimen type as determined by the APTIMA CT Assay is shown in Tables 1 and 1a for two multi-center clinical investigations by clinical site and overall.

<table><tr><td colspan="4">Table 1: Prevalence ofC. trachomatis by Clinical Site and Overall as Determined by APTIMA CT Assay Results</td></tr><tr><td colspan="4">% (#positive /</td></tr><tr><td>Site</td><td>MS MU</td><td>#tested) FS FU</td><td></td></tr><tr><td>1</td><td>27.0(68/252) 25. (</td><td>(63/252) 16.5 (38/230 17.0 (39/229) 19.2(42/219) 19.1(44/230)</td><td>PVS CVS</td></tr><tr><td rowspan="2">2</td><td>0</td><td>)</td><td></td></tr><tr><td>27.7(98/354) 26. (94/354) 35.0 (70/200 26.5 (53/200) 30.8(61/198) 33.0(66/200)</td><td></td><td></td></tr><tr><td>3</td><td>6 25.0(1/4) 25. (1/4)</td><td>) 11.4(13/114 8.8 (10/113) 10.8(12/111) 11.5(13/113)</td><td></td></tr></table>

0 )   
4 N/A N/A N/ N/A 11.6 (31/267 8.1 (22/271) 9.3 (25/268) 12.2(33/270) A )   
5 8.0 (16/200) 8.0 (16/200) 9.0 (18/199 7.5 (15/199) 8.0 (16/199) 10.1(20/199) )   
6 22.7 (69/304) 20. (61/305) 14.3 (42/294 13.2 (39/295) 15.2(44/290) 16.2(48/296) 0 )   
7 5.8 (12/207) 6.3 (13/207) 7.8 (8/102) 9.8 (10/102) 12.7(13/102) 8.8 (9/102)   
8 N/A N/A N/ N/A 8.2 (4/49) 6.1 (3/49) 12.5(6/48) 7.8 (4/51) A   
All 20.0 (264/132 18. (248/132 15.4 (224/14 13.1 (191/145 15.3(219/143 16.2(237/1461) 1) 8  2) 55) 8) 5)   
MS $=$ Male Urethral $\mathbf { M } \mathbf { U } = \mathbf { M } \mathrm { a l e }$ Urine; $\mathbf { F S } =$ Female Endocervical Swab; $\mathbf { F U } =$ Female Urine;   
Swab; $\mathbf { P } \mathbf { V } \mathbf { S } =$ Patient-Collected Vaginal   
Swab; CVS $=$ Clinician-Collected Vaginal Swab.

<table><tr><td>Table 1a: Prevalence of C. trachomatis by Clinical Site and Overall as</td></tr><tr><td>Determined by APTIMA CT Assay Results Using PreservCyt Solution</td></tr><tr><td>Liquid Pap Specimens</td></tr><tr><td>Site % (#positive / #tested)</td></tr><tr><td>1 17.0 (17/100)</td></tr><tr><td>2 3.2 (4/124)</td></tr><tr><td>3 7.4 (35/475) 4 4.2 (12/287)</td></tr><tr><td>5 5.4 (16/297)</td></tr><tr><td>6 5.5 (20/364)</td></tr><tr><td>All 6.3 (104/1647)</td></tr><tr><td></td></tr></table>

# Positive and Negative Predictive Values for Hypothetical Prevalence Rates in North America

The estimated positive and negative predictive values (PPV and NPV) for different hypothetical prevalence rates using the APTIMA CT Assay are shown in Table 2. These calculations are based on hypothetical prevalence rates and the overall sensitivity and specificity estimated from the patient infected status for three multi-center clinical investigations. The overall sensitivity and specificity for CT were $9 6 . 7 \%$ and $9 6 . 8 \%$ , respectively (Table 2). The actual PPV and NPV for clinician-collected endocervical, vaginal and male urethral swab, patient-collected vaginal swab, and male and female urine specimens are shown in Table 6 for each clinical site and overall. The actual PPV and NPV for PreservCyt Solution liquid Pap specimens are shown in Table 6a.

# Table 2: Positive and Negative Predictive Values for Hypothetical Prevalence Rates

<table><tr><td colspan="2">Hypothetical</td><td>Specificity (%)</td><td>PPV (%)</td><td>NPV (%)</td></tr><tr><td>Prevalence Rate (%) 1</td><td>Sensitivity (%) 96.7</td><td>96.8</td><td>23.5</td><td>100.0</td></tr><tr><td>2</td><td>96.7</td><td>96.8</td><td>38.3</td><td>99.9</td></tr><tr><td>5</td><td>96.7</td><td>96.8</td><td>61.6</td><td>99.8</td></tr><tr><td>10</td><td>96.7</td><td>96.8</td><td>77.2</td><td>99.6</td></tr><tr><td>15</td><td>96.7</td><td>96.8</td><td>84.3</td><td>99.4</td></tr><tr><td>20</td><td>96.7</td><td>96.8</td><td>88.4</td><td>99.2</td></tr><tr><td>25</td><td>96.7</td><td>96.8</td><td>91.0</td><td>98.9</td></tr><tr><td>30</td><td>96.7</td><td>96.8</td><td>92.9</td><td>98.6</td></tr></table>

# APTIMA CT Assay RLU Distribution

Figure 2 shows the RLU distribution for the APTIMA CT Assay for all specimen types in the clinical study except PreservCyt Solution liquid Pap specimens. Table 3 summarizes the RLU distribution for the total positive and total negative results, as well as the false positive and false negative results for each specimen type except PreservCyt Solution liquid Pap specimens relative to infected patient status. Across certain specimen types, there is a trend toward an increasing proportion of true positives as the RLU values increase.

![](images/bb77a765b3070f73a68608174622981f4bd6081ce6ab74c2f9c81c5454b03284.jpg)  
Figure 2. Frequency of RLU Distrbuton for the APTIMA cT Assay   
Table 3: APTIMA CT Assay RLU Distribution

<table><tr><td colspan="3"></td><td colspan="8">RLUs (x 1000)</td></tr><tr><td>0- &lt;10</td><td>10 - &lt;20</td><td>20 - &lt;30 30 - 40 - &lt;40</td><td>&lt;50</td><td>50 - &lt;100</td><td>100 - 1K - &lt;1K &lt;2K</td><td>&lt;3K</td><td>2K - 3K-</td><td>&lt;4K &lt;5K</td><td>4K - 5K - &lt;6K</td><td>&gt;6K</td></tr></table>

Total

<table><tr><td></td><td colspan="5"></td><td></td><td>50</td><td>22 14</td><td>16</td><td>18</td><td>92</td><td>1035</td></tr><tr><td colspan="5">Positives Total False</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="5"></td><td></td><td>43</td><td>17</td><td>7</td><td>11</td><td>10</td><td>25</td><td>126</td></tr><tr><td>Positives</td><td colspan="5"></td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>CVS</td><td colspan="5"></td><td>0</td><td>18</td><td>4</td><td>1 4</td><td>4</td><td>6</td><td>28</td></tr><tr><td>PVS</td><td colspan="4"></td><td>0</td><td>7</td><td>5</td><td>2</td><td>1</td><td>2</td><td>2</td><td>6</td></tr><tr><td>FS</td><td colspan="4"></td><td>0</td><td>9</td><td>2</td><td>3</td><td>2</td><td>2</td><td>5</td><td>26</td></tr><tr><td>MS</td><td colspan="4"></td><td>0</td><td>3</td><td>4</td><td>0</td><td>1</td><td>0</td><td>3</td><td>32</td></tr><tr><td>FU</td><td colspan="4"></td><td>0</td><td>5</td><td>2</td><td>0</td><td>1</td><td>0</td><td>6</td><td>12</td></tr><tr><td>MU</td><td colspan="4"></td><td>0</td><td>1</td><td>0</td><td>1</td><td>2</td><td>2</td><td>3</td><td>22</td></tr><tr><td>Total Negatives</td><td>6293 48</td><td></td><td>10</td><td>8</td><td>6 0</td><td colspan="4"></td><td></td><td></td><td></td></tr><tr><td>Total False</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Negatives</td><td>31</td><td>1</td><td>0</td><td>1</td><td>0 0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>CVS</td><td></td><td>0</td><td>0</td><td>1 0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>4 1</td><td>0</td><td>0</td><td>0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>PVS</td><td>3</td><td>0</td><td></td><td>0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>FS</td><td></td><td></td><td>0</td><td>0 0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>MS</td><td>4</td><td>1</td><td>0</td><td>0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>FU</td><td>10 9</td><td>0 0</td><td>0 0</td><td>0 0 0 0</td><td>0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="4">MU CVS = Clinician-Collected</td><td colspan="7">0</td></tr><tr><td colspan="3">Vaginal Swab;</td><td colspan="9">PVS = Asymptomatic Patient-Collected Vaginal Swab; FS = Male Urethral = Fema = Female Endocervical Swab; MS Swab; FU</td></tr></table>

# DTS Systems Clinical Performance Characteristics

See TIGRIS DTS System Clinical Specimen Agreement following the DTS Systems Analytical Performance Characteristics section for TIGRIS DTS System-specific clinical performance characteristics.

# Clinical Study Results

The performance characteristics of the APTIMA CT Assay were established in three multi-center clinical investigations conducted in North America. In the first clinical investigation, two studies were conducted. One, the clinical specimen study, established the sensitivity, specificity, and predictive values of the APTIMA CT Assay using clinician-collected endocervical, vaginal, and male urethral swab specimens, patientcollected vaginal swab specimens, and male and female urine specimens. The second study evaluated the precision of the APTIMA CT Assay when performed according to NCCLS Guidelines (16).

# Clinical Specimen Study: Endocervical Swab, Male Urethral Swab, Vaginal Swab, and Urine Specimens

Clinician-collected endocervical, vaginal and male urethral swab, patient-collected vaginal swab, and male and female urine specimens were collected from 2,787

symptomatic and asymptomatic, male and female subjects attending OB/GYN, sexually transmitted disease (STD), teen, and family planning clinics at eight geographically diverse clinical sites in North America. Subjects were classified as symptomatic if symptoms such as discharge, dysuria, and pelvic pain were reported by the subject. Subjects were classified as asymptomatic if the subject did not report symptoms. Of the 1,392 asymptomatic subjects enrolled in the study, 2 were less than 16 years of age, 237 were between the ages of 16 and 20, 423 were between the ages of 21 and 25, and 730 were greater than 25 years of age. Of the 1,395 symptomatic subjects enrolled in the study, 211 were between the ages of 16 and 20, 494 were between the ages of 21 and 25, and 690 were greater than 25 years of age. Three specimens were collected from each of the 1,322 eligible male subjects. Five specimens were collected from each of the 1,465 eligible female subjects. For male subjects, two randomized urethral swabs were collected followed by one urine specimen. For female subjects, one urine specimen was collected followed by one patient-collected vaginal swab, one clinician-collected vaginal swab, and two randomized endocervical swabs. APTIMA CT Assay and APTIMA Combo 2 Assay CT results were generated from the two vaginal swabs, one endocervical swab, one male urethral swab, and a male and female urine aliquot. The remaining endocervical swab, male urethral swab, and a male and female urine aliquot were tested using another commercially-available NAAT. Endocervical and male urethral swab specimens and male and female urine specimens tested in the APTIMA Combo 2 Assay and the other commercially available NAAT were used as the reference NAATs to determine infected status for each subject. Specimen testing was conducted either at the site of subject enrollment or at an external testing site.All performance calculations were based on the total number of APTIMA CT Assay results for endocervical, vaginal and male urethral swab, and male and female urine specimens compared to a patient infected status algorithm for each gender. In the algorithm, the designation of a subject as being infected or not infected with CT was based on endocervical swab and urine specimen results from the commercially-available APTIMA Combo 2 Assay and the other commercially-available NAAT. Subjects were considered infected with CT if two of the four endocervical swab and urine specimens tested positive in the APTIMA Combo 2 Assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 8,406 APTIMA CT Assay results were used to calculate sensitivity and specificity. Sensitivity and specificity for CT by gender, specimen type and symptom status are presented in Table 4. Table 6 shows the APTIMA CT Assay sensitivity, specificity, and predictive values compared to patient infected status for each clinical site and overall. Tables 7a-7d summarize the number of results from symptomatic and asymptomatic subjects designated as infected or non-infected with CT according to the patient infected status algorithm.Of the 2,787 subjects enrolled, there were 13 subjects with unknown CT patient infected status. Subjects were designated with an unknown patient infected status if results were missing that prevented conclusive determination of infected status. These subjects' results were not included in any performance calculations. Of the 8,452 APTIMA CT Assay results from the multi-center clinical study, there was a small percentage ( $8 , 0 . 0 9 \%$ of specimens that initially tested invalid for CT. Upon repeat testing, there were no equivocal or invalid results.

Table 4: Sensitivity and Specificity of the APTIMA CT Assay Relative to Patient Infected Status by Symptom Status and Overall   

<table><tr><td rowspan="2">Specimen</td><td rowspan="2">Symptom</td><td rowspan="2"></td><td colspan="6"></td><td rowspan="2">Sensitivity</td><td rowspan="2">Specificity</td></tr><tr><td>N</td><td>TP</td><td>FP</td><td>TN</td><td>FN</td></tr><tr><td rowspan="7">Male</td><td rowspan="3">Swab</td><td>Status</td><td></td><td></td><td></td><td></td><td></td><td>(95% C.I.)</td><td></td><td>(95% C.I.)</td></tr><tr><td>Symptomatic</td><td>576</td><td>131</td><td>23a 20b</td><td>418 634</td><td>4 1</td><td>97.0 (92.6 - 99.2) 98.9</td><td></td><td>94.8 (92.3 - 96.7)</td></tr><tr><td>Asymptomatic 745 All</td><td>1321</td><td>90 221</td><td>43c</td><td></td><td>5</td><td>(94.0 - 100) (94.9 - 99.3)</td><td></td><td>96.9 (95.3 - 98.1) 96.1 (94.7 - 97.1)</td></tr><tr><td rowspan="3">Urine</td><td></td><td></td><td></td><td></td><td>1052</td><td></td><td>97.8</td><td></td><td></td></tr><tr><td>Symptomatic</td><td>576</td><td>127</td><td>14d</td><td>427</td><td>8</td><td>94.1</td><td>(88.7 - 97.4)</td><td>96.8 (94.7 - 98.3)</td></tr><tr><td>Asymptomatic 746 All</td><td></td><td>90</td><td>17e 31f</td><td>638</td><td>1</td><td>98.9 (94.0 - 100)</td><td></td><td>97.4 (95.9 - 98.5)</td></tr><tr><td rowspan="6">Female</td><td rowspan="3">Swab</td><td></td><td>1322</td><td>217</td><td>1065</td><td>9</td><td>96.0</td><td>(92.6 - 98.2)</td><td>97.2 (96.0 - 98.1)</td><td></td></tr><tr><td>Symptomatic</td><td>807</td><td>114</td><td>28g 664</td><td>1</td><td>99.1</td><td>(95.3 - 100)</td><td></td><td>96.0 (94.2 - 97.3)</td></tr><tr><td>Asymptomatic 636</td><td></td><td>59</td><td>22h 553</td><td></td><td>2 96.7</td><td>(88.7 - 99.6)</td><td></td><td>96.2 (94.3 - 97.6)</td></tr><tr><td>All</td><td></td><td>1443 173</td><td>50i</td><td>1217</td><td>3</td><td>98.3</td><td>(95.1 - 99.6)</td><td></td><td>96.1 (94.8 - 97.1)</td></tr><tr><td rowspan="2">Urine</td><td>Symptomatic</td><td>809</td><td>107 13j</td><td>682</td><td>7</td><td>93.9</td><td>(87.8 - 97.5)</td><td></td><td>98.1 (96.8 - 99.0)</td></tr><tr><td>Asymptomatic 639 All</td><td>1448</td><td>58 165 261</td><td>13k 565</td><td></td><td>3 95.1</td><td>(86.3 - 99.0)</td><td></td><td>97.8 (96.2 - 98.8)</td></tr><tr><td rowspan="2">Patient-</td><td>Vaginal</td><td></td><td></td><td></td><td></td><td>1247</td><td>10 94.3</td><td>(89.7 - 97.2)</td><td></td><td>98.0 (97.0 - 98.7)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Collected</td><td>Swab</td><td>Asymptomatic 629</td><td></td><td>60</td><td>25m</td><td>543</td><td>1</td><td>98.4 (91.2 - 100)</td><td></td><td>95.6 (93.6 - 97.1)</td></tr><tr><td rowspan="2"></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Symptomatic</td><td>811</td><td>111 33n</td><td>663</td><td>4</td><td>96.5</td><td>(91.3 - 99.0)</td><td></td><td>95.3 (93.4 - 96.7)</td></tr><tr><td rowspan="2">Clinician-</td><td>Vaginal</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Asymptomatic 638</td><td></td><td>60</td><td>32Â° 545</td><td></td><td>1</td><td>98.4 (91.2 - 99.0)</td><td></td><td>94.5 (92.3 - 96.2)</td></tr><tr><td rowspan="2">Collected</td><td>Swab</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>All</td><td></td><td>1449</td><td>171</td><td>65P</td><td>1208</td><td>5</td><td>97.2 (93.5 - 99.1)</td><td></td><td>94.9 (93.5 - 96.0)</td></tr></table>

$\mathbf { N } =$ Negative; $\mathbf { T P } =$ True Positive; $\mathbf { F P } =$ False Positive; $\mathbf { T N } =$ True Negative; $\mathbf { F N } =$ False Negative. APTIMA Combo 2 Assay CT results: # positive results / # specimens tested a: 9/23; b: 14/20; c: 23/43; d: 6/14; e: 6/17; f: 12/31; g: 14/28; h:11/22; i: 25/50; j: 7/13; k: 5/ 13; l: 12/26; m: 15/25; n: 17/33; o: 15/32; p: 32/65.

# Clinical Specimen Study: PreservCyt Solution Liquid Pap

A prospective multi-center clinical study was conducted to evaluate the use of the PreservCyt Solution (a component of the ThinPrep 2000 System) as an alternative medium for gynecological specimens for the detection of CT by the APTIMA CT Assay. One thousand six hundred forty-seven (1,647) symptomatic and asymptomatic female subjects attending OB/GYN, family planning, public health, women's, and STD clinics were evaluated in the clinical study. Of the 1,647 evaluable subjects, 1,288 were asymptomatic subjects and 359 were symptomatic subjects. Subjects were enrolled from sites with CT prevalence that ranged from $2 . 8 \%$ to $1 4 . 0 \%$ . Two specimens were collected from each eligible subject: one PreservCyt Solution liquid Pap specimen and one endocervical swab specimen. PreservCyt Solution liquid Pap specimens were collected with the spatula/cyto-brush or a broom-like brush cervical sampling device. The distribution of cervical sampling devices is summarized in Table 5 by specimen collection site and overall. PreservCyt Solution liquid Pap specimens were processed in accordance with the ThinPrep 2000 Processor Operator's Manual and APTIMA Specimen Transfer Kit Package Insert. After processing the PreservCyt Solution liquid Pap specimen with the ThinPrep 2000 Processor, the specimen was transferred into the APTIMA Specimen Transfer Kit for testing with the APTIMA CT Assay. Sensitivity and specificity of the APTIMA CT Assay in PreservCyt Solution liquid Pap specimens were calculated by comparing results to a patient infected status algorithm. The algorithm included APTIMA Combo 2 Assay and APTIMA CT Assay results in endocervical swab specimens. Both reference NAATs were required to be positive to establish an infected patient status. At least one reference NAAT was required to be negative to establish a non-infected patient status. Table 7e summarizes the frequency of test outcomes for the two reference NAATs.

Table 5a shows the sensitivities and specificities of the APTIMA CT Assay by symptom status and overall. Overall sensitivity was $9 5 . 6 \%$ (86/90). In symptomatic and asymptomatic subjects, sensitivities were $9 6 . 7 \%$ (29/30) and $9 5 . 0 \%$ (57/60), respectively. Overall specificity was $9 8 . 8 \%$ (1539/1557). In symptomatic and asymptomatic subjects, specificities were $9 8 . 8 \%$ (325/329) and $9 8 . 9 \%$ (1214/1228), respectively. Table 6a shows the sensitivities and specificities of the APTIMA CT Assay by specimen collection site and overall. Sensitivities ranged from $9 2 . 9 \%$ to $100 \%$ . Specificities ranged from $9 6 . 5 \%$ to $100 \%$ .

Table 5: Distribution of Cervical Sampling Device Used for PreservCyt Solution Liquid Pap Specimens   
Clinical Collection Site   

<table><tr><td></td><td>1</td><td>2</td><td>3</td><td>4</td><td>5</td><td>6</td><td></td></tr><tr><td>Spatula/Cytobrush</td><td>0</td><td>124</td><td>475</td><td>287</td><td>57</td><td>364</td><td>1307</td></tr><tr><td>Broom-Type Device</td><td>100</td><td>0</td><td>0</td><td>0</td><td>240</td><td>0</td><td>340</td></tr></table>

Cervical Sampling Device Used

Total

# Table 5a: Sensitivity and Specificity of the APTIMA CT Assay Relative to Patient Infected Status by Symptom Status and

Solution Liquid Pap

<table><tr><td colspan="4">on Bdad T ap Specimens APTIMA CT</td><td colspan="2"></td></tr><tr><td>PreservCyt</td><td>+/+</td><td>+/-</td><td>-/+</td><td>Sensitivity (%) -/- (95% CI)</td><td>Specificity (%)</td></tr><tr><td>Solution result</td><td></td><td></td><td></td><td></td><td>(95% CI)</td></tr><tr><td>Positive</td><td>29</td><td>0</td><td>1 3</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>96.7 (29/30)</td><td>98.8 (325/329)</td></tr></table>

<table><tr><td rowspan="2">Symptomatic</td><td rowspan="2">Negative</td><td rowspan="2">1 3</td><td rowspan="2">3</td><td rowspan="2">319</td><td colspan="2"></td></tr><tr><td>(82.8 - 99.9)</td><td>(96.9  99.7)</td></tr><tr><td rowspan="4">Asymptomatic</td><td>Total</td><td>30</td><td>3</td><td>4</td><td>322</td><td></td></tr><tr><td>Positive</td><td>57</td><td>0</td><td>1</td><td>13</td><td>98.9 (1214/1228)</td></tr><tr><td>Negative</td><td>3</td><td>2</td><td>11</td><td>95.0 (57/60) 1201</td><td></td></tr><tr><td>Total</td><td>60</td><td></td><td></td><td>(86.1 - 99.0)</td><td>(98.1 99.4)</td></tr><tr><td rowspan="4">All</td><td>Positive</td><td>86</td><td>2 0</td><td>12 2</td><td>1214 16</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>95.6 (86/90)</td><td>98.8 (1539/1557)</td></tr><tr><td>Negative</td><td>4</td><td>5</td><td>14</td><td>1520</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>(89.0 - 98.8)</td><td>(98.2  99.3)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td>Total</td><td>90</td><td>5</td><td></td><td>1536</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td>16</td><td></td><td></td></tr></table>

$+ / + =$ Positive endocervical swab specimen result in the APTIMA COMBO 2 Assay/Positive endocervical swab specimen result in the APTIMA CT Assay   
$+ / - =$ Positive endocervical swab specimen result in the APTIMA COMBO 2 Assay/Negative endocervical swab specimen result in the APTIMA CT Assay   
$\mathbf { \partial } _ { - } / + = \mathrm { N }$ egative endocervical swab specimen result in the APTIMA COMBO 2 Assay/Positive endocervical swab specimen result in the APTIMA CT Assay   
$- / - =$ Negative endocervical swab specimen result in the APTIMA COMBO 2 Assay/Negative endocervical swab specimen result in the APTIMA CT Assay

Specimen

Table 6: Sensitivity, Specificity and Predictive Values of the APTIMA CT Assay Relative to Patient Infected Status by Clinical Site and Overall   

<table><tr><td colspan="10">Prev</td></tr><tr><td>Site N</td><td></td><td>TP</td><td>FP</td><td>TN</td><td>FN</td><td></td><td>Sensitivity (95% Specificity (95% PPV</td><td></td><td></td><td>NPV (%)</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td>C.I.) (%)</td><td>C.I.)</td><td></td><td>(%)</td><td></td></tr><tr><td>1</td><td>252</td><td>54</td><td>14</td><td>183 1</td><td></td><td>21.8 98.2</td><td>(90.3 - 100)92.9</td><td>(88.4 - 96.1)</td><td>79.4</td><td>99.5</td></tr><tr><td>2</td><td>354</td><td>83</td><td>15</td><td>252</td><td>4</td><td>24.6 95.4</td><td>(88.6 - 94.4</td><td>(90.9 - 96.8)</td><td>84.7</td><td>98.4</td></tr><tr><td>3</td><td>4</td><td>1</td><td>0</td><td>3</td><td>0</td><td>25.0 100</td><td>98.7) (2.5 - 100) 100</td><td>(29.2 -</td><td>100</td><td>100</td></tr><tr><td>4</td><td>N/A</td><td>N/AN/AN/A</td><td></td><td></td><td>N/A</td><td>N/A N/A</td><td>N/A</td><td>100) N/A N/A</td><td>N/A</td><td>N/A</td></tr><tr><td>5</td><td>200</td><td>12</td><td>4</td><td>184</td><td>0</td><td>6.0</td><td>100 (73.5 - 100)97.9</td><td>(94.6 -</td><td>75.0</td><td>100</td></tr><tr><td>6</td><td>304</td><td>59</td><td>10</td><td>235</td><td>0</td><td>19.4 100</td><td>(93.9 - 100)95.9</td><td>99.4) (92.6 -</td><td>85.5</td><td>100</td></tr><tr><td>7</td><td>207</td><td>12</td><td>0</td><td>195</td><td>0</td><td>5.8 100</td><td>(73.5 - 100)100</td><td>98.0) (98.1 -</td><td>100</td><td>100</td></tr><tr><td>8</td><td></td><td>N/A N/AN/AN/A</td><td></td><td></td><td></td><td>N/A N/A</td><td>N/A</td><td>100) N/A N/A</td><td>N/A</td><td>N/A</td></tr><tr><td>All</td><td>1321</td><td>221 43</td><td></td><td>1052</td><td>5</td><td>17.1</td><td>N/A 97.8 (94.9 - 99.3)</td><td>96.1 (94.7 - 97.1)</td><td>83.7</td><td>99.4</td></tr></table>

# Male

Urine

<table><tr><td>1</td><td>252</td><td>54</td><td>9</td><td>188</td><td>1</td><td>21.8 98.2</td><td></td><td>(90.3 - 100)95.4</td><td>(91.5 -</td><td>85.7</td><td>99.5</td></tr><tr><td>2</td><td>354</td><td>85</td><td>9</td><td>258</td><td>2</td><td>24.6 97.7</td><td>(91.9 - 99.7)</td><td>96.6</td><td>97.9) (93.7 -</td><td>90.4</td><td>99.2</td></tr><tr><td>3</td><td>4</td><td>1</td><td>0</td><td>3</td><td>0</td><td>25.0 100</td><td></td><td>(2.5 - 100) 100</td><td>98.4) (29.2 -</td><td>100</td><td>100</td></tr><tr><td>4</td><td>N/A</td><td></td><td></td><td>N/AN/AN/A</td><td></td><td></td><td>N/A N/A N/A</td><td>N/A N/A</td><td>100) N/A</td><td>N/A</td><td>N/A</td></tr><tr><td>5</td><td>200</td><td>12</td><td>4</td><td>184</td><td>0</td><td>6.0</td><td>100</td><td>(73.5 - 100)97.9</td><td>(94.6 - 99.4)</td><td>75.0</td><td>100</td></tr><tr><td>6</td><td>305</td><td>53</td><td>8</td><td>238</td><td>6</td><td></td><td>19.3 89.8</td><td>(79.2 - 96.7 96.2)</td><td>(93.7 - 98.6)</td><td>86.9</td><td>97.5</td></tr><tr><td>7</td><td>207</td><td>12</td><td>1</td><td>194</td><td>0</td><td>5.8</td><td>100</td><td>(73.5 - 100)99.5</td><td>(97.2 - 100)</td><td>92.3</td><td>100</td></tr><tr><td>8 All</td><td>N/A N/AN/AN/A 1322 217 31</td><td></td><td></td><td></td><td></td><td></td><td>N/A N/A N/A</td><td>N/A N/A (92.6 - 97.2</td><td>N/A (96.0 -</td><td>N/A 87.5</td><td>N/A</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td>1065 9</td><td></td><td>17.1 96.0</td><td>98.2)</td><td>98.1)</td><td></td><td>99.2</td></tr><tr><td>1</td><td>228</td><td>36</td><td>2</td><td>190</td><td>0</td><td></td><td>15.8 100</td><td>(90.3 - 100)99.0</td><td>(96.3 - 99.9)</td><td>94.7</td><td>100</td></tr><tr><td>2</td><td>198</td><td>52</td><td>18</td><td>128</td><td>0</td><td></td><td>26.3 100</td><td>(93.2 - 100)87.7</td><td>(81.2 - 92.5)</td><td>74.3</td><td>100</td></tr><tr><td>3</td><td>114</td><td>9</td><td>4</td><td>101</td><td>0</td><td>7.9</td><td>100</td><td>(66.4 - 100)96.2</td><td>(90.5 - 99.0)</td><td>69.2</td><td>100</td></tr><tr><td>4</td><td>260</td><td>19</td><td>11</td><td>229</td><td>1</td><td>7.7</td><td>95.0</td><td>(75.1 - 95.4 99.9)</td><td>(91.9 - 97.7)</td><td>63.3</td><td>99.6</td></tr><tr><td>5</td><td>199</td><td>13</td><td>5</td><td>181</td><td>0</td><td>6.5</td><td>100</td><td>(75.3 - 100)97.3</td><td>(93.8 - 99.1)</td><td>72.2</td><td>100</td></tr><tr><td>6</td><td>294</td><td>33</td><td>9</td><td>252</td><td>0</td><td></td><td>11.2 100</td><td>(89.4 - 100)96.6</td><td>(93.6 - 98.4)</td><td>78.6</td><td>100</td></tr><tr><td>7</td><td>102</td><td>8</td><td>0</td><td>92</td><td>2</td><td>9.8</td><td>80.0</td><td>(44.4 - 97.5)</td><td>100 (96.1 - 100)</td><td>100</td><td>97.9</td></tr><tr><td>8</td><td>48</td><td>3</td><td>1</td><td>44</td><td>0</td><td>6.3</td><td>100</td><td>(29.2 - 100)97.8</td><td>(88.2 - 99.9)</td><td>75.0</td><td>100</td></tr><tr><td>All</td><td></td><td>1443 173 50</td><td></td><td></td><td>1217 3</td><td></td><td>12.2 98.3</td><td>(95.1 - 99.6)</td><td>96.1 (94.8 - 97.1)</td><td>77.6</td><td>99.8</td></tr><tr><td>1</td><td>227</td><td>34</td><td>5</td><td></td><td></td><td></td><td></td><td>(85.1 -</td><td>97.4 (94.0 -</td><td></td><td>87.2 99.5</td></tr><tr><td>2</td><td>198</td><td></td><td></td><td>187</td><td>1</td><td></td><td>15.4 97.1 26.3 98.1</td><td>99.9) (89.7 - 100)98.6</td><td>99.1) (95.1 -</td><td></td><td>99.3</td></tr><tr><td></td><td></td><td>51</td><td>2</td><td>144</td><td>1</td><td></td><td></td><td>(66.4 - 100)99.0</td><td>99.8) (94.8 -</td><td>96.2 90.0</td><td></td></tr><tr><td>3</td><td>113</td><td>9</td><td>1</td><td>103</td><td>0</td><td>8.0</td><td>100</td><td></td><td>100)</td><td></td><td>100</td></tr><tr><td>4</td><td>265</td><td>18</td><td>4</td><td>241</td><td>2</td><td>7.5</td><td>90.0</td><td>(68.3 - 98.8)</td><td>98.4 (95.9 - 99.6)</td><td></td><td>81.8 99.2</td></tr></table>

# Swab

# Female

Table 6: Sensitivity, Specificity and Predictive Values of the APTIMA CT Assay Relative to Patient Infected Status by Clinical Site and Overall (Continued)   

<table><tr><td>5</td><td>199</td><td>11 4</td><td>182</td><td>2</td><td>6.5</td><td>84.6</td><td>(54.6 - 98.1)</td><td>97.8 (94.6 - 99.4)</td><td>73.3</td><td>98.9</td></tr><tr><td>6</td><td>295</td><td>29 10</td><td>252</td><td>4</td><td>11.2 87.9</td><td></td><td>(71.8 - 96.6)</td><td>96.2</td><td>(93.1 - 98.2)</td><td>74.4 98.4</td></tr><tr><td>7</td><td>102</td><td>10 0</td><td>92</td><td>0</td><td>9.8</td><td>100</td><td>(69.2 - 100)100</td><td></td><td>(96.1 - 100)</td><td>100 100</td></tr><tr><td>8</td><td>49</td><td>3 0</td><td>46</td><td>0</td><td>6.1</td><td>100</td><td>(29.2 - 100)100</td><td>100)</td><td>(92.3 -</td><td>100 100</td></tr><tr><td>All</td><td>1448</td><td>165 26</td><td></td><td>1247 10</td><td>12.1 94.3</td><td></td><td>(89.7 - 97.2)</td><td>98.0</td><td>(97.0 - 98.7)</td><td>86.4 99.2</td></tr></table>

Prev.   

<table><tr><td colspan="2">Specimen</td><td>Site N</td><td></td><td>TP</td><td rowspan="4">FP TN</td><td rowspan="4">FN</td><td rowspan="4">Prev.</td><td rowspan="4">C.I.)</td><td rowspan="4">Sensitivity (95% Specificity (95% PPV</td><td rowspan="4">C.I.)</td><td rowspan="4"></td><td rowspan="4">(%) 77.8</td><td rowspan="4">NPV 100</td></tr><tr><td rowspan="3"></td><td>1</td><td>70</td><td>14 4</td><td>52 0</td><td>(%) 20.0 100</td></tr><tr><td></td><td>46</td><td>4 29</td><td>0</td><td>(76.8 - 100)</td><td>92.9 (82.7 - 98.0) 87.9 (71.8 -</td></tr><tr><td></td><td>2</td><td>13</td><td></td><td>28.3 100</td><td>(75.3 - 100)</td></tr><tr><td></td><td>3</td><td>45</td><td>4</td><td>2</td><td>39</td><td>0</td><td>8.9</td><td>100</td><td>(39.8 - 100) (42.1 -</td><td>96.6) 95.1 (83.5 - 99.4 97.9</td><td>66.7</td><td>100</td></tr><tr><td rowspan="2">Patient-</td><td>4</td><td>152</td><td>6</td><td>3</td><td>142</td><td>1</td><td>4.6</td><td>85.7</td><td>99.6)</td><td>(94.1 - 99.666.7</td><td rowspan="2"></td><td rowspan="2">99.3</td></tr><tr><td>Vagina 1 5</td><td>130</td><td>7</td><td>3</td><td>120</td><td>0</td><td>5.4</td><td>100 (59.0 -</td><td>97.6 (93.0 -</td></tr><tr><td>Collected</td><td>Swab</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>100)</td><td></td><td>70.0 99.5)</td><td></td><td></td></tr><tr><td rowspan="5"></td><td>6</td><td>75</td><td>8</td><td>5</td><td>62</td><td>0</td><td></td><td>10.7 100</td><td>(63.1 - 100)</td><td>92.5</td><td>(83.4 - 97.5)</td><td>61.5</td><td>100</td></tr><tr><td>7</td><td>68</td><td>5</td><td>2</td><td>61</td><td>0</td><td></td><td>7.4 100</td><td>(47.8 - 100)</td><td>96.8 95.0</td><td>(89.0 - 99.6)</td><td>71.4</td><td>100</td></tr><tr><td>8</td><td>43</td><td>3</td><td>2</td><td>38</td><td></td><td>0</td><td>7.0</td><td>100 (29.2 - 100)</td><td></td><td>(83.1 - 99.4) (93.6 -</td><td>60.0 70.6</td><td>100</td></tr><tr><td></td><td>All</td><td>629</td><td>60</td><td>25</td><td>543</td><td>1</td><td>9.7</td><td>98.4</td><td>(91.2 - 100)</td><td>95.6 97.1)</td><td></td><td>99.8</td></tr><tr><td>1</td><td></td><td>228</td><td>36</td><td>8</td><td>184 130</td><td>0</td><td>15.8 100</td><td>(90.3 - 100)</td><td>95.8</td><td>(92.0 - 98.2) (82.8 -</td><td>81.8</td><td>100</td></tr><tr><td></td><td>2</td><td>198</td><td>50</td><td>16</td><td></td><td>2</td><td></td><td>26.3 96.2</td><td>(86.8 - 99.5)</td><td>89.0 96.2</td><td>93.6) (90.4 -</td><td>75.8</td><td>98.5</td></tr><tr><td></td><td>3</td><td>113</td><td>9</td><td>4</td><td></td><td>100</td><td>0</td><td>8.0</td><td>100 (66.4 - 100)</td><td></td><td>98.9)</td><td>69.2</td><td>100</td></tr><tr><td></td><td>4</td><td>263</td><td>18</td><td>14</td><td></td><td>229 2</td><td>7.6</td><td>90.0</td><td>(68.3 - 98.8)</td><td>94.2</td><td>(90.5 - 96.8)</td><td>56.3</td><td>99.1</td></tr><tr><td>Clinician-</td><td>Vagina 1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Collected</td><td>5</td><td>199</td><td>13</td><td>7</td><td></td><td>179 0</td><td>6.5</td><td>100</td><td>(75.3 - 100)</td><td>96.2</td><td>(92.4 - 98.5)</td><td>65.0</td><td>100</td></tr><tr><td rowspan="4"></td><td>Swab 6</td><td>296</td><td>33</td><td>15</td><td>248</td><td>0</td><td>11.1</td><td>100</td><td>(89.4 -</td><td>94.3</td><td>(90.8 -</td><td>68.8</td><td>100</td></tr><tr><td></td><td>102</td><td>9</td><td>0</td><td>92</td><td>1</td><td>9.8</td><td>90.0</td><td>100) (55.5 -</td><td>100</td><td>96.8) (96.1 -</td><td>100</td><td>98.9</td></tr><tr><td>7 8</td><td>50</td><td>3</td><td>1</td><td>46</td><td>0</td><td>6.0</td><td>100</td><td>99.7) (29.2 -</td><td>97.9</td><td>100) (88.7 -</td><td>75.0</td><td>100</td></tr><tr><td>All</td><td></td><td>1449 171</td><td>65</td><td></td><td>1208 5</td><td></td><td>12.1 97.2</td><td>100) (93.5 -</td><td>94.9</td><td>99.9) (93.5 -</td><td>72.5</td><td>99.6</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>99.1) = True Negative; FN = False</td><td></td><td>96.0)</td><td></td><td></td></tr><tr><td>Positive;</td><td>N = Negative; TP = True</td><td>FP = False</td><td></td><td>Positive; TN</td><td></td><td>Negative.</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr></table>

Table 6a: Sensitivity, Specificity and Predictive Values of the APTIMA CT Assay Relative to Patient Infected Status by Clinical Site and Overall for PreservCyt Solution Liquid Pap Specimens   

<table><tr><td rowspan="5">Site</td><td>APTIMA CT</td><td></td><td></td><td></td><td></td><td></td><td>Prev Sensitivity Specificity (%)</td><td></td><td></td><td></td></tr><tr><td>PreservCyt</td><td></td><td>+/+ +/- -/+ -/-</td><td></td><td></td><td></td><td></td><td></td><td>PPV</td><td>NPV (%)</td></tr><tr><td>Solution</td><td></td><td></td><td></td><td></td><td></td><td></td><td>(%) (%) (95% (95% C.I.)</td><td>(%)</td><td></td></tr><tr><td>Result</td><td></td><td></td><td></td><td></td><td></td><td>C.I.)</td><td></td><td></td><td></td></tr><tr><td>Positive</td><td>14</td><td>0</td><td>1</td><td>2</td><td></td><td></td><td>100 (14/14) 96.5 (83/86)</td><td></td><td></td></tr><tr><td rowspan="4">1</td><td>Negative</td><td>0</td><td>0</td><td>0</td><td>83</td><td>14.0</td><td>(76.8  100) (90.1  99.3)</td><td></td><td>82.4</td><td>100</td></tr><tr><td>Total</td><td>14</td><td>0</td><td>1</td><td>85</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Positive</td><td>4</td><td>0</td><td>0</td><td>0</td><td></td><td>100 (4/4)</td><td>100 (120/120)</td><td></td><td></td></tr><tr><td>Negative</td><td>0</td><td>0</td><td>2</td><td>118</td><td>3.2</td><td></td><td></td><td>100</td><td>100</td></tr><tr><td rowspan="3"></td><td>Total</td><td>4</td><td>0</td><td>2</td><td>118</td><td></td><td></td><td>(39.8  100) (97.0  100)</td><td></td><td></td></tr><tr><td>Positive</td><td>29</td><td>0</td><td>0</td><td>6</td><td></td><td>93.5 (29/31) 98.6 (438/444)</td><td></td><td></td><td></td></tr><tr><td>Negative</td><td>2</td><td>0</td><td>2</td><td>436</td><td>6.5</td><td></td><td></td><td>82.9</td><td>99.5</td></tr><tr><td>3</td><td>Total</td><td>31</td><td>0</td><td>2</td><td>442</td><td></td><td></td><td>(78.6  99.2) (97.1  99.5)</td><td></td><td></td></tr></table>

$+ / + =$ Positive endocervical swab specimen result in the APTIMA COMBO 2 Assay/Positive endocervical swab specimen result in the APTIMA CT Assay   
$+ / - =$ Positive endocervical swab specimen result in the APTIMA COMBO 2 Assay/Negative endocervical swab specimen result in the APTIMA CT Assay   
$\mathbf { \partial } _ { - } / + = \mathrm { N }$ egative endocervical swab specimen result in the APTIMA COMBO 2 Assay/Positive endocervical swab specimen result in the APTIMA CT Assay   
$- / - =$ Negative endocervical swab specimen result in the APTIMA COMBO 2 Assay/Negative endocervical swab specimen result in the APTIMA CT Assay

Table 7a: Male Urethral Swab and Urine Results from Subjects Infected or Non-Infected with C. trachomatis According to Patient Infected Status   

<table><tr><td rowspan="2"></td><td rowspan="2">Positive</td><td rowspan="2">8</td><td rowspan="2">0</td><td rowspan="2">0</td><td rowspan="2">4</td><td rowspan="2"></td><td colspan="3"></td></tr><tr><td>100 (8/8)</td><td>98.6 (275/279)</td><td></td></tr><tr><td rowspan="2">4</td><td>Negative</td><td>0</td><td>3</td><td>1</td><td>271</td><td>2.8</td><td></td><td></td><td>66.7</td><td>100</td></tr><tr><td>Total</td><td>8</td><td>3</td><td>1</td><td>275</td><td></td><td>(63.1 - 100) (96.4  99.6)</td><td></td><td></td><td></td></tr><tr><td rowspan="2">5</td><td>Positive</td><td>13</td><td>0</td><td>0</td><td>3</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Negative</td><td>1</td><td>1</td><td>4</td><td>275</td><td>4.7</td><td></td><td>92.9 (13/14) 98.9 (280/283)</td><td>81.3</td><td>99.6</td></tr><tr><td rowspan="3">6</td><td>Total</td><td>14</td><td>1</td><td>4</td><td>278</td><td></td><td>(66.1  99.8) (96.9  99.8)</td><td></td><td></td><td></td></tr><tr><td>Positive</td><td>18</td><td>0</td><td>1</td><td>1</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Negative</td><td>1</td><td>1</td><td>5</td><td>337</td><td>5.2</td><td></td><td>94.7 (18/19) 99.4 (343/345)</td><td>90.0</td><td>99.7</td></tr><tr><td rowspan="3"></td><td>Total</td><td></td><td></td><td></td><td></td><td></td><td></td><td>(74.0 - 99.9) (97.9  99.9)</td><td></td><td></td></tr><tr><td>Positive</td><td>19 86</td><td>1 0</td><td>6 2</td><td>338 16</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td>95.6 (86/90) 98.8</td><td></td><td>82.7</td><td></td></tr><tr><td rowspan="2">All</td><td>Negative 4 5</td><td></td><td></td><td>14</td><td></td><td>1520 5.5</td><td></td><td>(1539/1557)</td><td></td><td>99.7</td></tr><tr><td>Total</td><td>90 5</td><td></td><td></td><td></td><td>1536</td><td></td><td>(89.0  98.8) (98.2  99.3)</td><td></td><td></td></tr></table>

Patient Infected Status Infected Infected Infected Infected   

<table><tr><td>MS</td><td>MU</td><td>MS</td><td>MU</td><td>MS</td><td>MU</td><td>Sympt.</td><td>Asympt.</td><td>Total</td></tr><tr><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>96</td><td>68</td><td>164</td></tr><tr><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>-</td><td>5</td><td>1</td><td>6</td></tr><tr><td>+</td><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td>11</td><td>7</td><td>18</td></tr><tr><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>13</td><td>11</td><td>24</td></tr></table>

NAAT 1 (APTIMA Combo 2 Assay)

<table><tr><td>Infected</td><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td></td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td></td><td>+</td><td>-</td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>+</td><td>2</td><td>0</td><td>2</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td></td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td></td><td>+</td><td></td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td></td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>0</td><td>2</td><td>2</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td></td><td>+</td><td>3</td><td>1</td><td>4</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>0</td><td>1</td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td>+</td><td></td><td></td><td>+</td><td>+</td><td>4</td><td>4</td><td>8</td></tr><tr><td>Non-infected</td><td>+</td><td>+</td><td></td><td></td><td></td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td>+</td><td>1</td><td>4</td><td>5</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td></td><td>4</td><td>6</td><td>10</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td></td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td></td><td></td><td>3</td><td>0</td><td>3</td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td>+</td><td>0</td><td>2</td><td>2</td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td></td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td>+</td><td>+</td><td></td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>+</td><td></td><td></td><td>2</td><td>2</td><td>4</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td>+</td><td>1</td><td>1</td><td>2</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td></td><td>11</td><td>5</td><td>16</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>+</td><td>4</td><td>4</td><td>8</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td></td><td>403</td><td>618</td><td>1021</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>N/A</td><td></td><td>+</td><td>0</td><td>2</td><td>2</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>N/A</td><td></td><td></td><td>1</td><td>2</td><td>3</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>=</td><td></td><td></td><td>0</td><td>4</td><td>4</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2</td><td>0</td><td>2</td></tr><tr><td>Non-infected</td><td>N/A</td><td></td><td></td><td>-</td><td>N/A</td><td></td><td>0</td><td>1</td><td>1</td></tr></table>

Total 576 $\mathbf { N } / \mathbf { A } = \mathbf { \Gamma } _ { \mathbf { \Gamma } }$ Specimen not obtained or available for testing. The equal symbol $( = )$ represents equivocal or indeterminate on repeat testing. $\mathbf { M S } =$ Male Urethral Swab; $\mathbf { M } \mathbf { U } = \mathbf { M } \mathrm { a l e }$ Urine.

<table><tr><td colspan="5">Table 7b: Female Endocervical Swab and Urine Results from Subjects Infected C. trachomatis According</td></tr><tr><td colspan="5">or Non-Infected with Patient Infected Status</td></tr><tr><td colspan="5">NAAT 1 NAAT 2</td></tr><tr><td colspan="5"></td></tr><tr><td colspan="5">(APTIMA Combo 2</td></tr><tr><td colspan="5"></td></tr><tr><td colspan="5"></td></tr><tr><td>Patient</td><td>Assay)</td><td></td><td></td><td></td></tr><tr><td>Infected Status FS</td><td>FU</td><td>FS FU</td><td>FS FU Sympt. Asympt Total</td><td></td></tr></table>

<table><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>80</td><td>43</td><td>123</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>-</td><td>1</td><td>1</td><td>2</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td>10</td><td>5</td><td>15</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>9</td><td>3</td><td>12</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>+</td><td></td><td>1</td><td>4</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>-</td><td>MNNY</td><td>2</td><td>4</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td></td><td>+</td><td>+</td><td></td><td>0</td><td>2</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td></td><td>+</td><td>-</td><td></td><td>0</td><td>4</td></tr><tr><td>Infected</td><td>+</td><td></td><td>+</td><td></td><td>+</td><td>N/A</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Infected</td><td>-</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>0</td><td>1</td><td>1</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td>+</td><td>+</td><td>1</td><td>3</td><td>4</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td>-</td><td>+</td><td>1</td><td>2</td><td>3</td></tr><tr><td>Non-infected</td><td>+</td><td>+</td><td></td><td>-</td><td>+</td><td>+</td><td>1</td><td>2</td><td>3</td></tr><tr><td>Non-infected</td><td>+</td><td>+</td><td></td><td>N/A</td><td>+</td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td>+</td><td>0</td><td>2</td><td>2</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td></td><td>12</td><td>7</td><td>19</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td></td><td>-</td><td>0</td><td>1</td><td>1</td></tr><tr><td>Non-infected</td><td>-</td><td>+</td><td></td><td></td><td>+</td><td>+</td><td>1</td><td>0</td><td></td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td>+</td><td>4</td><td>3</td><td>NMHNMN</td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td>-</td><td>0</td><td>1</td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td>+</td><td></td><td></td><td></td><td>1</td><td>1</td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td></td><td>1</td><td>2</td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td>+</td><td>0</td><td>2</td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td>-</td><td>11</td><td>9</td><td>20</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>+</td><td>5</td><td>4</td><td>9</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>-</td><td>636</td><td>526</td><td>1162</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>N/A</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>N/A</td><td></td><td>-</td><td>2</td><td>3</td><td>5</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td></td><td>12</td><td>10</td><td>22</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td></td><td>1</td><td>1</td><td>22</td></tr><tr><td>Non-infected</td><td>-</td><td>N/A</td><td></td><td></td><td></td><td>N/A</td><td>1</td><td>1</td><td></td></tr><tr><td>Non-infected</td><td>N/A</td><td>-</td><td></td><td></td><td>N/A</td><td>-</td><td>5</td><td>4</td><td>9</td></tr><tr><td>Non-infected</td><td>=</td><td></td><td></td><td></td><td>+</td><td>+</td><td>1</td><td>0</td><td>1</td></tr><tr><td>Non-infected</td><td>=</td><td></td><td></td><td></td><td>+</td><td></td><td>1 9.12</td><td>0 .4.0</td><td>1 45</td></tr></table>

$\mathbf { N } / \mathbf { A } = \mathbf { S }$ pecimen not obtained or available for testing. The equal symbol $( = )$ represents equivocal or indeterminate on repeat testing.

$\mathbf { F S } =$ Female Endocervical $=$ Female Urine. Sympt. $=$ Swab; FU Symptomatic; Asympt

. $=$   
Asymptomat ic.

# Table 7c: Asymptomatic Patient-Collected Vaginal Swab Results from Subjects Infected or NonInfected with $C .$ trachomatis According to Patient Infected Status

# NAAT 1

# (APTIMA Combo 2 Assay)

# APTIMA CT

<table><tr><td colspan="3"></td><td colspan="2">NAAT 2</td><td colspan="2">Assay</td></tr><tr><td>Patient Infected Status</td><td>FS</td><td>FU</td><td>FS</td><td>FU</td><td>PVS</td><td>Total</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td>+</td><td>+</td><td>44</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>+</td><td>-</td><td>+</td><td>5</td></tr><tr><td>Infected</td><td>+</td><td>+</td><td>-</td><td>+</td><td>+</td><td>3</td></tr><tr><td>Infected</td><td>+</td><td>-</td><td>+</td><td>+</td><td>+</td><td>3</td></tr><tr><td>Infected</td><td></td><td>+</td><td>+</td><td>+</td><td>+</td><td>1</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td>+</td><td>4</td></tr><tr><td>Infected</td><td></td><td>+</td><td></td><td>+</td><td></td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td>+</td><td></td><td></td><td>+</td><td>2</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td>4</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td>+</td><td>1</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td></td><td>2</td></tr><tr><td>Non-infected</td><td>+</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td>+</td><td>MNN</td></tr><tr><td>Non-infected</td><td></td><td>+</td><td></td><td></td><td></td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td>+</td><td></td><td></td><td></td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>2</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td>5</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>+</td><td>10</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>15</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>500</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>N/A</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td>N/A</td><td>9</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>N/A</td><td>-</td><td>2</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>N/A</td><td>N/A</td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>=</td><td></td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>=</td><td></td><td>8</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td>=</td><td></td><td>1</td></tr><tr><td>Non-infected</td><td></td><td></td><td></td><td></td><td></td><td>1</td></tr><tr><td>Non-infected</td><td></td><td>N/A</td><td></td><td></td><td></td><td>1</td></tr><tr><td>Non-infected</td><td>N/A</td><td></td><td></td><td></td><td>+</td><td>1</td></tr><tr><td>Non-infected</td><td>N/A</td><td>-</td><td></td><td></td><td></td><td>3 6.40</td></tr></table>

Total   
$\mathbf { N } / \mathbf { A } = \mathbf { \underline { { S } } }$ Specimen not obtained or available for testing. The equal symbol $( = )$ represents equivocal or indeterminate on repeat testing.   
$\mathbf { F S } =$ Female Endocervical Swab; $\mathbf { F U } =$ $\mathbf { C V S } =$ Clinician-Collected   
Female Urine; Vaginal Swab;   
$\mathbf { P } \mathbf { V } \mathbf { S } =$ Asymptomatic Patient-Collected   
Vaginal Swab.

Table 7d: Clinician-Collected Vaginal Swab Results from Subjects Infected or Non- According Infected with C. trachomatis to Patient Infected Status   

<table><tr><td rowspan=1 colspan=14>NAAT 1APTIMACT(APTIMACombo 2NAAT 2            Assay      Symptom StatusAssay)PatientInfected             FS          FU   FS           FU   CVS        Sympt.         Asympt Totalâ¢Status</td></tr><tr><td rowspan=1 colspan=5>Infected              +            +</td><td rowspan=1 colspan=4>+             +</td><td rowspan=1 colspan=3>+             76</td><td rowspan=1 colspan=2>44       120</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=4>+             +</td><td rowspan=1 colspan=3>2</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=4>+             +</td><td rowspan=1 colspan=3>+             2</td><td rowspan=1 colspan=2>220</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=3>+            +</td><td rowspan=1 colspan=4>+             +</td><td rowspan=1 colspan=3>+             1</td><td rowspan=1 colspan=2>0        1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=3>+            +</td><td rowspan=1 colspan=4>+             -</td><td rowspan=1 colspan=3>+             8</td><td rowspan=1 colspan=2>5        13</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=4>+</td><td rowspan=1 colspan=3>1</td><td rowspan=1 colspan=2>0        1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>0        1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>=</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>0        1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>9</td><td rowspan=1 colspan=2>3        12</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=2>3        8</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td></td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>7</td><td rowspan=1 colspan=2>0        7</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=3>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>1        1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>4        5</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=4>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=4>+</td><td rowspan=1 colspan=2>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Non-infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=2>Non-infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=4>+</td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>30</td></tr><tr><td rowspan=1 colspan=2>Non-infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>4        7</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=3>-</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>1        1</td></tr><tr><td rowspan=1 colspan=2>Non-infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=3>-</td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2        4</td></tr><tr><td rowspan=1 colspan=2>Non-infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>3        8</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=5>--</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>0</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=5>--</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>5</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=3>--</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>|</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>1</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=3>|</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=2>2</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=1></td><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=2>16</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Non-infected</td><td rowspan=1 colspan=5></td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4>614             500     1114</td></tr></table>

Non-infected N/A 0 1 1 1   
Non-infected + 0 1   
Non-infected 13 9 22   
Non-infected N/A - 2 2 4   
Non-infected N/A - 0 1 1   
Non-infected = + 0 1 1   
Non-infected = - 12 8 20   
Non-infected = N/A 0 1 1   
Non-infected - - 1 1 2   
Non-infected - N/A - - - 0 1 1   
Non-infected - N/A - - N/A 1 0 1   
Non-infected N/A - - 0 1 1   
Non-infected N/A 5 3 8   
Non-infected $=$ 2 0 2   
Total 812 640 1452

$\mathbf { N } / \mathbf { A } =$ Specimen not obtained or available for testing. The equal symbol $( = )$ represents equivocal or indeterminate on repeat testing.

$\mathbf { F S } =$ Female Endocervical $=$ Female Urine; $\mathbf { C V S } =$ Clinician-Collected Vaginal Swab; FU Swab. Sympt. $=$ Symptomatic;

# Table 7e: PreservCyt Solution Liquid Pap Specimen Analysis for Patient Infected Status

Symptom Status

Endocervical Swab   
APTIMA APTIMA CT   
COMBO

Patient Infected Status

# Symptom Asymptomatic atic

2 Assay Assay   
Infected Positive Positive 30 60   
Non-Infected Negative Negative 322 1214   
Non-Infected Negative Positive 4 12   
Non-Infected Positive Negative 3 2   
Total 359 1288

# Precision Study

APTIMA CT Assay precision (i.e., reproducibility) was evaluated at two external clinical sites and at Gen-Probe. APTIMA CT Assay precision was evaluated across three APTIMA CT Assay kit lots, three study sites, six operators and 108 APTIMA CT Assay runs. Two operators at each of the three testing sites performed a total of six APTIMA CT Assay runs per kit lot for a total of 36 runs per kit lot. Each run was composed of a 12-member precision panel containing 0 to 2,000 fg/assay of CT rRNA. Reproducibility was established using spiked swab transport medium with rRNA. Reproducibility when testing swab and urine specimens containing target organism has not been determined. Table 8 presents the precision RLU data in terms of Mean, Standard Deviation, Coefficient of Variation (CV), and percent agreement with expected results for calculations of inter-site, inter-lot, inter-operator, inter-run, and intra-run variability.

Table 8: APTIMA CT Assay Precision Data   

<table><tr><td></td><td></td><td></td><td>Inter-Site</td><td>Inter-Lot</td><td></td><td>Inter- Operator</td><td>Inter-Run</td><td>Intra-Run</td><td></td></tr><tr><td>Concentration</td><td colspan="7">Mean % SD CV SD (RLUx10 Agrm(RLUx10 (% (RLUx10 (% (RLUx10(% (RLUx10(% (RLUx10(%)</td><td>CVSD CV SD</td><td></td><td>CV</td></tr><tr><td></td><td>00) N</td><td>t.</td><td>00)</td><td>) 00)</td><td></td><td>) 00)</td><td>) 00)</td><td>) 00)</td><td></td><td></td></tr><tr><td>Neg</td><td>5400.7</td><td>100 0.5</td><td></td><td>N/ 0.3</td><td></td><td>N/ 0.4</td><td>N/ 0</td><td>N/ 0.7</td><td></td><td>N/A</td></tr><tr><td>(0 fg/mL) Low</td><td></td><td></td><td></td><td>A</td><td></td><td>A</td><td>A</td><td>A</td><td></td><td></td></tr><tr><td>(12 fg/mL)</td><td>2167143.4</td><td></td><td>100 335.6</td><td></td><td>4.7 207.7</td><td>2.9 537.3</td><td></td><td>7.5 558.8</td><td>7.8 200.3</td><td>2.8</td></tr><tr><td>Mid (250 fg/mL)</td><td>1087084.9</td><td></td><td>100 275.1</td><td></td><td>3.9 159.5</td><td>2.3 546.3</td><td></td><td>7.7 578.2</td><td>8.2 162.2</td><td>2.3</td></tr><tr><td>Mid (2,500 fg/mL)</td><td>1086991.1</td><td></td><td>100 279.4</td><td></td><td>4.0 117.8</td><td>1.7 532.3</td><td></td><td>7.6 534.9</td><td>7.7 150.7</td><td>2.2</td></tr><tr><td>High (5,000-5,135 fg/mL)</td><td>7133.4 324</td><td></td><td>100 301.0</td><td></td><td>4.2 129.0</td><td>1.8 531.7</td><td></td><td>7.5 618.3</td><td>8.7 229.2</td><td>3.2</td></tr><tr><td colspan="9">SD = Standard Deviation; CV(%) = Percent Coefficient= Percent N/A = not applicable for of Variation; % Agrmt. Agreement. negative analyte.</td><td></td></tr><tr><td colspan="10">Note:Varibily m soe ctors may be umeialy negativewhi cncur  the vaiabiliy ue those factors is very small. When this occurs, the variability as measured with SD and %CV is set to zero (16). N/A = not</td><td></td></tr></table>

PreservCyt specimen intra-laboratory precision with the ACT Assay was determined by spiking PreservCyt vials with $2 0 \mathrm { C T }$ IFU per vial (0.1 IFU per reaction) and $1 0 0 \mathrm { C T }$ IFU per vial (0.5 IFU per reaction). Vials containing 1,000 CT IFU per vial (5 IFU per reaction) and unspiked PreservCyt vials were tested as positive and negative controls. Ten vials spiked at each IFU level and ten unspiked vials were divided between two operators. The operators vortexed the vials and then transferred 14 aliquots ( $\mathrm { 1 . 0 ~ m L }$ each) per vial into 14 APTIMA Transfer Tubes as per the APTIMA Specimen Transfer Kit package insert. The operators were blinded to the samples' titers. Each of the resulting Pap-STM samples was tested once in the ACT Assay. A total of five runs were performed over a five day period for 140 results at each IFU level. The results are summarized in Table 9.

<table><tr><td colspan="4">Table 9: APTIMA CT Assay Intra- Laboratory</td><td colspan="3">Precision Data for PreservCyt Mean Intra-</td><td colspan="2">Inter-Day</td><td colspan="3">Inter-</td></tr><tr><td>Panel</td><td>IFU/20m IFU/</td><td></td><td></td><td>%</td><td></td><td>Operator</td><td></td><td></td><td>Operator</td><td>Total</td></tr><tr><td>Member</td><td>L PreservCrxn</td><td>n</td><td>Agree d</td><td>RLU Agr</td><td>SD</td><td></td><td>SD</td><td>SD</td><td></td><td>SD</td></tr><tr><td></td><td>yt</td><td></td><td></td><td>mt.</td><td>(x1000)(x1000CV (%)</td><td></td><td>(x1000 CV (%) </td><td>(x100</td><td>CV (%)</td><td>(x1000CV (%)</td></tr><tr><td>A</td><td>20</td><td>0.1</td><td></td><td></td><td>)</td><td></td><td>)</td><td>0)</td><td>)</td><td></td></tr><tr><td>B</td><td></td><td></td><td>140 140</td><td>100</td><td>6501.7 734.8 11.3</td><td></td><td>0</td><td>0.0</td><td>546.9 8.4</td><td>916 14.1</td></tr><tr><td></td><td>100</td><td>0.5</td><td>140 138*</td><td>98.6</td><td>6337.7 1054.716.6</td><td></td><td>0 0.0</td><td></td><td>947.2 14.9</td><td>1417.622.4</td></tr><tr><td>C</td><td>1000</td><td>5</td><td>140 140</td><td>100</td><td>6521.9 909</td><td>13.9</td><td>247.1</td><td>3.8 393.9 6</td><td></td><td>1021 15.7</td></tr><tr><td>D</td><td>0</td><td>0</td><td>140 140</td><td>100 1.2</td><td>0.8</td><td>N/A</td><td>0</td><td>N/A 0.4</td><td>N/A 0.9</td><td>N/A</td></tr></table>

\* discordant results were one negative result and 1 equivocal result

Note: Variability from some factors may be numerically negative, which can occur if the variability due to those factors is very small. When this occurs,   
the variability as measured with SD and $\% \mathrm { C V }$ is set to $=$ not applicable for negative panel members. Operator $=$ zero (16). N/A Run. Samples with   
discordant results were included in the signal   
variability analysis.

# DTS Systems Analytical Performance Characteristics

See TIGRIS DTS System Analytical Performance Characteristics following the Figure TIGRIS DTS System Clinical Specimen Agreement section for TIGRIS DTS Systemspecific analytical performance characteristics.

# Analytical Sensitivity

C. trachomatis analytical sensitivity (limit of detection) was determined by directly comparing dilutions of CT organisms in cell culture and in the APTIMA CT assay. The analytical sensitivity claim for the assay is one Inclusion-Forming Unit (IFU) per assay (7.25 IFU/swab, $5 \ \mathrm { I F U / m L }$ urine, and $9 . 7 5 \ : \mathrm { I F U / m L }$ PreservCyt Solution liquid Pap) for all $1 5 \mathrm { C T }$ serovars. However, dilutions of less than one IFU/assay of all serovars tested positive.

# Analytical Specificity

A total of 154 culture isolates were evaluated using the APTIMA CT Assay. These isolates included 86 organisms that may be isolated from the urogenital tract and 68 additional organisms that represent a phylogenetic cross-section of organisms. The tested organisms included bacteria, fungi, yeast, parasites and viruses. All organisms except $C .$ . psittaci, C. pneumoniae, U. urealyticum and the viruses were tested at $1 . 0 \mathrm { ~ x ~ } 1 0 6$ cells/assay in KOVA-trol/Urine Transport Media and 60 organisms were tested in Swab

Transport Media. The Chlamydia and Neisseria organisms were tested in the PreservCyt Solution media. C. psittaci VR601 was tested at 8x104 cells/assay and $C .$ . psittaci VR125 was tested at $1 \mathrm { x } 1 0 5$ cells/assay. C. pneumoniaewas tested at $4 \mathrm { ~ x ~ } 1 0 3$ cells/assay and $U .$ . urealyticum was tested at $6 . 7 \times 1 0 6$ cells/assay. The viruses were tested as follows: (a) herpes simplex virus I: $2 . 5 \mathrm { ~ x ~ } 1 0 4$ TCID50/assay, (b) herpes simplex virus II: $6 . 0 \times 1 0 4$ TCID50/assay, (c) human papillomavirus 16: $2 . 9 \mathrm { ~ x ~ } 1 0 6$ DNA copies/assay and (d) cytomegalovirus: $4 . 8 \mathrm { ~ x ~ } 1 0 5$ cells/assay. The list of organisms tested is shown in Table 10.

# Table 10: Analytical Specificity Organism

Achromobacter xerosis Acinetobacter calcoaceticus

# Organism

Neisseria mucosa (3) Neisseria sicca (3)

Acinetobacter Iwoffi   
Actinomyces israelii   
Actinomyces pyogenes   
Aerococcus viridans   
Aeromonas hydrophila   
Agrobacterium radiobacter   
Alcaligenes faecalis   
Bacillus subtilis   
Bacteriodes fragilis   
Bacteriodes ureolyticus   
Bifidobacterium adolescentis   
Bifidobacterium brevi   
Branhamella catarrhalis   
Brevibacterium linens   
Campylobacter jejuni   
Candida albicans   
Candida glabrata   
Candida parapsilosis   
Organism   
Escherichia coli   
Flavobacterium   
meningosepticum   
Fusobacterium nucleatum   
Gardnerella vaginalis   
Gemella haemolysans   
Haemophilus ducreyi   
Haemophilus influenzae   
Herpes simplex virus I   
Herpes simplex virus II   
Human papilloma virus 16   
Kingella dentrificans   
Kingella kingae   
Klebsiella oxytoca   
Klebsiella pneumoniae   
Lactobacillus acidophilus   
Lactobacillus brevis   
Lactobacillus jensonii   
Lactobacillus lactis   
Legionella pneumophila (2)   
Leuconostoc   
paramensenteroides   
Listeria monocytogenes   
Micrococcus luteus   
Moraxella lacunata   
Moraxella osloensis   
Morganella morganii   
Mycobacterium smegmatis   
Mycoplasma genitalium   
Mycoplasma hominis   
N. meningitidis Serogroup A   
N. meningitidis Serogroup B   
N. meningitidis Serogroup   
(4)   
N. meningitidis Serogroup D   
N. meningitidis Serogroup Y   
N. meningitidis Serogroup   
Neisseria subflava (14)   
Neisseria perflava   
Neisseria polysaccharea   
Paracoccus denitrificans   
Peptostreptococcus anaerobius   
Peptostreptococcus productus   
Plesiomonas shigelloides   
Propionibacterium acnes   
Proteus mirabilis   
Proteus vulgaris   
Providencia stuartii   
Pseudomonas aeruginosa   
Pseudomonas fluorescens   
Pseudomonas putida   
Rahnella aquatilis   
Rhodospirillum rubrum   
Saccharomyces cerevisiae   
Salmonella minnesota

Candida tropicalis Chlamydia pneumoniae Chlamydia psittaci (2) Chromobacterium violaceum Citrobacter freundii Clostridium perfringens Corynebacterium genitalium Corynebacterium xerosis Cryptococcus neoformans Cytomegalovirus Deinococcus radiodurans

Salmonella typhimurium Serratia marcescens Staphylococcus saprophyticus Staphylococcus aureus Staphylococcus epidermidis Streptococcus agalactiae Streptococcus bovis Streptococcus mitis Streptococcus mutans Streptococcus pneumoniae Streptococcus pyogenes Streptococcus salivarius Streptococcus sanguis Streptomyces griseinus

Derxia gummosa Eikenella corrodens Enterobacter aerogenes

W135

Neisseria cinerea (4) Neisseria dentrificans Neisseria elongata (3) Neisseria flava Neisseria flavescens (2) Neisseria lactamica (9)

Trichomonas vaginalis Ureaplasma urealyticum Vibrio parahaemolyticus Yersinia enterocolitica

$( { \mathfrak { n } } ) =$ number of strains tested. All organisms tested produced a negative result in the APTIMA CT Assay.

# Interfering Substances

The following interfering substances were individually spiked into swab, PreservCyt liquid Pap and/or urine specimens: $10 \%$ blood, contraceptive jelly, spermicide, moisturizer, hemorrhoidal anesthetic, body oil, powder, anti-fungal cream, vaginal lubricants, feminine spray and leukocytes ( $\mathrm { 1 \times 1 0 6 \ c e l l s / m L }$ ). The following interfering substances were individually spiked into urine specimens: $30 \%$ blood, urine analytes, protein, glucose, ketones, bilirubin, nitrate, urobilinogen, $\mathrm { p H } 4$ (acidic), $\mathsf { p H } 9$ (alkaline), leukocytes (1 x 106 cells/mL), cellular debris, vitamins, minerals, acetaminophen, aspirin and ibuprofen. All were tested for potential assay interference in the absence and presence of CT at the estimated rRNA equivalent of 1 cell/assay (5 fg/assay). The rRNA equivalents were calculated based on the genome size and estimated DNA:RNA ratio/cell of each organism. No interference was observed with any of the tested substances. No inhibitors of amplification were observed in the APTIMA CT Assay.

# Recovery

Escherichia coli, Gardnerella vaginalis, Lactobacillus acidophilus, Bacteroides ureolyticus, and Staphylococcus epidermidis $^ \textrm { \scriptsize 1 x 1 0 8 }$ cells/assay) were added to samples containing the rRNA equivalent of approximately one CT IFU (5 fg). These additions did not interfere with the amplification and detection of CT rRNA using the APTIMA CT Assay.

# Swab and Urine Specimen Stability Studies

Data to support the recommended shipping and storage conditions for endocervical, urethral and vaginal swab samples were generated with pooled negative swab samples. Pooled samples were spiked with CT at a final concentration of 1 IFU per reaction. The spiked samples were held at -70oC, -20oC, 4oC, and 30oC. Samples were tested in duplicate at days 0, 20, 77, and 117. All test conditions were positive for CT at all times and temperatures.

Data to support the recommended shipping and storage conditions for urine samples were generated with female and male negative urine samples. The urine samples were spiked with CT at a final concentration of 10 IFU per reaction. Two sets of the spiked urine samples were held at $3 0 \mathrm { o C }$ for 24 hours prior to being added to the Urine Transport Media (UTM). The two sets of UTM samples then were held at $4 \mathrm { { } ^ { \circ } C }$ and $3 0 ^ { \circ } \mathrm { C }$ , and tested in triplicate at days 0, 1, 5, 20, and 35. All samples were positive for CT at all timepoints. The two sets of UTM samples were also tested after 116 days of storage at $- 2 0 \mathrm { o C }$ and - 70oC. All samples were positive for CT under both storage conditions. Liquid Pap Specimen Stability StudiesData to support the recommended shipping and storage conditions for PreservCyt Solution liquid Pap samples were generated with negative

processed and unprocessed liquid Pap samples. For the unprocessed samples, four pools of PreservCyt Solution samples were tested after being stored in the Cytyc PreservCyt Solution vial. Each specimen pool was spiked with 1-10 IFU CT/assay, held at $2 ^ { \circ } \mathrm { C }$ , $1 0 ^ { \circ } \mathrm { C }$ , and $3 0 ^ { \circ } \mathrm { C }$ , then tested at baseline and on days 5, 7, 8, 14, 18, 21, 25 and 36. All of the spiked samples were positive for CT at all times and temperatures.For the processed samples, four pools of PreservCyt Solution samples were used to determine processed specimen stability at $2 \mathrm { ^ { \circ } C }$ to $3 0 ^ { \circ } \mathrm { C }$ . Each negative sample pool was spiked with 1-10 IFU CT/assay, then tested at baseline. Prior to processing, the PreservCyt Solution samples were stored at $3 0 ^ { \circ } \mathrm { C }$ for seven (7) days to simulate the time lapse between sample collection, Pap processing and shipment to a microbiology testing lab. After seven days at $3 0 ^ { \circ } \mathrm { C }$ , $1 ~ \mathrm { m L }$ aliquots of each pool were transferred to an APTIMA Specimen Transfer Tube and tested at baseline before being placed at $2 ^ { \circ } \mathrm { C }$ , $1 0 ^ { \circ } \mathrm { C }$ , and $3 0 \mathrm { { } ^ { \circ } C }$ . The processed samples were then tested for 17 days stored at $3 0 \mathrm { { } ^ { \circ } C }$ and 36 days stored at $2 \mathrm { ^ { \circ } C }$ to $1 0 ^ { \circ } \mathrm { C }$ . All of the spiked samples were positive for CT at all times and temperatures.Data to support longer storage conditions were generated from four pools of negative processed PreservCyt Solution samples tested at below freezing temperatures. Each pool was spiked with 1-10 IFU CT/assay, then tested at baseline. Each pool was first placed at $3 0 \mathrm { { } ^ { \circ } C }$ for 14 days and then stored at $- 2 0 ^ { \circ } \mathrm { C }$ or $- 7 0 \mathrm { { } ^ { \circ } C }$ over the course of 106 days. All of the spiked samples were positive for CT at all times and temperatures.

# TIGRIS DTS System Clinical Specimen Agreement TIGRIS DTS System Agreement

Agreement between APTIMA CT Assay results generated on the fully automated TIGRIS DTS System and semi-automated DTS Systems was evaluated by testing endocervical swab, male urethral swab, male and female urine, vaginal swab, and PreservCyt liquid Pap specimens. Each of the clinical specimens was tested individually with the APTIMA CT Assay on both the TIGRIS DTS System and DTS Systems at GenProbe. The order of testing was not randomized. Specimens identified for inclusion were tested on the TIGRIS DTS System followed by testing on DTS Systems.

Clinical Specimen Agreement Study - Endocervical Swab, Male Urethral Swab, Female and Male Urine, Vaginal Swab, and PreservCyt liquid Pap Specimens Female and male subjects attending STD, family planning, and OB/GYN clinics from eight geographically diverse sites with low to high prevalence for CT contributed endocervical swab, male urethral swab, female and male urine, vaginal swab, and PreservCyt liquid Pap specimens. The specimens were transferred directly to Gen-Probe for testing while the PreservCyt liquid Pap specimens were processed at 2 cytopathology laboratories before being transferred. At Gen-Probe, endocervical swab, male urethral swab, female and male urine specimens were first screened with APTIMA Combo 2 Assay on the TIGRIS DTS System, and the vaginal swab and PreservCyt liquid Pap specimens were screened with APTIMA Combo 2 Assay on the DTS Systems. Specimens with final invalid or equivocal results were not selected in the APTIMA CT Clinical Specimen Agreement Study.Two hundred and five female swabs (87 endocervical and 118 vaginal), 120 male urethral swab, 98 female urine, 115 male urine, and 116 PreservCyt liquid Pap specimens with APTIMA Combo 2 Assay CT positive and negative results were selected for comparison testing between the TIGRIS DTS System and the DTS Systems for the APTIMA CT Assay. Specimens with initial invalid or equivocal results were retested using the same system on which the result was

generated. One female urine specimen had an initial equivocal result on the DTS Systems; when retested, the final result was valid. One male urine specimen had an initial invalid result on the TIGRIS DTS System; when retested, the final result was valid. One female urine specimen had an initial equivocal result on the TIGRIS DTS System; this specimen was retested, however, the specimen had expired, so the final result was equivocal. Table 11 shows the positive, negative, and overall agreements for all paired results for each specimen type by symptomatic status. Specimens are relatively imbalanced by symptomatic and asymptomatic status but overall agreements for symptomatic subjects were $9 8 . 5 \%$ (131/133) for female swabs (combined endocervical and vaginal swabs), $100 \%$ (60/60) for male urethral swab, $9 8 . 2 \%$ (55/56) for female urine specimens, $100 \%$ (60/60) for male urine specimens, and $100 \%$ (81/81) for PreservCyt liquid Pap specimens. For asymptomatic subjects, overall agreements were $100 \%$ for 72 female swabs, 60 male urethral swabs, 42 female urine, 55 male urine specimens, and 35 PreservCyt liquid Pap specimens, respectively. For âAllâ (symptomatic and asymptomatic combined) subjects, overall agreement was $9 9 . 0 \%$ (203/205) for female swab (combined endocervical and vaginal swabs), $100 \%$ (120/120) for male urethral swab, $9 9 . 0 \%$ (97/98) for female urine, $100 \%$ (115/115) for male urine, and $100 \%$ (116/116) for PreservCyt liquid Pap specimens. Due to the relatively smaller specimen number from asymptomatic subjects, these findings may not be generalizable to ACT-TIGRIS System testing with specimens from asymptomatic subjects. Refer to Tables 4 and 6 for APTIMA CT Assay sensitivity and specificity estimates from testing on the DTS Systems. Sensitivity and specificity of the APTIMA CT Assay when using the TIGRIS DTS System would be expected to be similar given the agreement findings.

Table 11: Clinical Specimen Agreement Study: Positive, Negative, and Overall Agreements by Symptom Status   

<table><tr><td colspan="2"></td><td rowspan="5"></td><td rowspan="2"></td><td rowspan="2">DTS+ DTS+ DTS- DTS-</td><td rowspan="2"></td><td rowspan="2"></td><td rowspan="2">Positive %</td><td rowspan="2"></td><td rowspan="2">Negative % Overall % Agreement Agreement Agreement</td></tr><tr><td>Symptom Specime Gender n n</td><td></td></tr><tr><td rowspan="3"></td><td></td><td>S+</td><td>S-</td><td>S+</td><td>TIGRI TIGRI TIGRI TIGRI S-</td><td></td><td></td><td></td></tr><tr><td></td><td>Female* 133</td><td>63</td><td>1</td><td>1</td><td>68</td><td>(95% CI) 98.4 (91.6-100)98.6 (92.2- 100)</td><td>(95% CI)</td><td>(95% CI) 98.5 (94.7-99.8)</td></tr><tr><td>Swab Male</td><td>60</td><td>42</td><td>0</td><td>0</td><td>18</td><td>100 (91.6-100) 100 (81.5-</td><td>100)</td><td>100 (94.0-100)</td></tr><tr><td rowspan="3">Sympt.</td><td></td><td>Female 56</td><td>33</td><td>0</td><td>11</td><td>22</td><td></td><td>100 (89.4-100) 95.7 (78.1- 99.9)</td><td>98.2 (90.4-100)</td></tr><tr><td>Urine</td><td>Male 60</td><td>41</td><td>0</td><td>0</td><td>19</td><td></td><td>100 (91.4-100) 100 (82.4- 100)</td><td>100 (94.0-100)</td></tr><tr><td>Preserv Female 81 39</td><td></td><td></td><td>0</td><td>0</td><td>42</td><td></td><td>100 (91.0-100) 100 (91.6-</td><td>100 (95.5-100)</td></tr></table>

Asympt.

<table><tr><td rowspan="2">Swab</td><td>Female* 72</td><td></td><td>41</td><td>0</td><td>0</td><td>31</td><td>100 (91.4-100) 100 (88.8- 100)</td><td>100 (95.0-100)</td></tr><tr><td>Male</td><td>60</td><td>23</td><td>0</td><td>0</td><td>37</td><td>100 (85.2-100) 100 (90.5- 100)</td><td>100 (94.0-100)</td></tr><tr><td rowspan="2">Urine</td><td>Female 42</td><td></td><td>23</td><td>0</td><td>0</td><td>19</td><td>100 (85.2-100) 100 (82.4- 100)</td><td>100 (91.6-100)</td></tr><tr><td>Male</td><td>55</td><td>20</td><td>0</td><td>0</td><td>35</td><td>100 (83.2-100) 100 (90.0- 100)</td><td>100 (93.5-100)</td></tr><tr><td>Cyt</td><td>Preserv Female 35</td><td></td><td>25</td><td>0</td><td>0</td><td>10</td><td>100 (86.3-100) 100 (69.2- 100)</td><td>100 (90.0-100)</td></tr><tr><td rowspan="2">Swab</td><td>Female* 205</td><td></td><td>104</td><td></td><td>1</td><td>99</td><td>99.0 (94.8-100)99.0 (94.6- 99.0 (96.5-99.9) 100)</td><td></td></tr><tr><td>Male</td><td>120</td><td>65</td><td>0</td><td>0</td><td>55</td><td>100 (94.5-100) 100 (93.5- 100)</td><td>100 (97.0-100)</td></tr><tr><td rowspan="2">Urine</td><td>Female 98</td><td></td><td>56</td><td>0</td><td>1</td><td>41</td><td>100 (93.6-100) 97.6 (87.4- 99.0 (94.4-100) 99.9)</td><td></td></tr><tr><td>Male</td><td>115</td><td>61</td><td>0</td><td>0</td><td>54</td><td>100 (94.1-100) 100 (93.4- 100)</td><td>100 (96.8-100)</td></tr><tr><td>Preserv Female 116 64 Cyt a positive result, &quot; a negative result, CI=</td><td></td><td></td><td></td><td>0</td><td>0</td><td>52</td><td>100 (94.4-100) 100 (93.2- 100)</td><td>100 (96.9-100)</td></tr></table>

All

$" + "$   
confidence interval   
\*Endocervical and Vaginal Swab samples   
combined   
1 Specimen had a final equivocal result on the TIGRIS   
DTS System

# Precision Study

The effect of several factors on the variability of APTIMA CT Assay performance on the TIGRIS DTS System was evaluated using 12-member STD reproducibility panels. Panel members contained 0 to 5,000 fg CT rRNA/assay. The panel included panel members with CT concentrations at the analytical sensitivity claim of 5 fg CT rRNA/assay.The panels were tested at one external testing site and at Gen-Probe using two APTIMA CT Assay reagent lots. At Gen-Probe, two operators each performed three valid worklists per reagent lot on each of two TIGRIS DTS System instruments. At the external testing site, two operators each performed three valid worklists per reagent lot on one TIGRIS DTS

System instrument. One worklist consisted of run controls and six 12-member panels. Reproducibility was determined by calculating the agreement between the final assay results and the expected outcome for each panel member. Reproducibility was also assessed by calculating the SD and coefficient of variation (CV) of signal with respect to sites, operators, lots, and worklists. CVs were not calculated for CT-negative panel members due to low signal values that could theoretically equal zero. Table 12 shows the reproducibility results. All APTIMA CT Assay results on the TIGRIS DTS System agreed with the expected results. CV values were less than or equal to $3 . 4 \%$ . These data indicate excellent reproducibility of the APTIMA CT Assay using the TIGRIS DTS System.

Table 12: TIGRIS DTS System Precision   

<table><tr><td>Data Conc</td><td colspan="3">Mean</td><td colspan="2">Inter-Site</td><td colspan="4">Inter-OperatorInter-Lot</td><td colspan="2">Inter- Worklist</td><td colspan="2">Intra-Worklist</td></tr><tr><td>(fg rRNA</td><td></td><td colspan="2">RLU</td><td colspan="2">SD1</td><td colspan="2">SD</td><td colspan="2">SD1</td><td colspan="2">SD</td><td colspan="2">SD</td></tr><tr><td rowspan="3">per assay)</td><td rowspan="3">n</td><td colspan="2" rowspan="3">(x1000) %</td><td colspan="2">(x1000) CV1</td><td colspan="2">(x1000) CV</td><td colspan="2">(x1000)CV</td><td colspan="2">(x1000 CV</td><td colspan="2">(x1000 CV (%)</td></tr><tr><td colspan="2"></td><td colspan="2"></td><td colspan="2"></td><td colspan="2"></td><td colspan="2"></td></tr><tr><td>Agrmt</td><td></td><td>(%)</td><td></td><td>(%)</td><td></td><td>(%)</td><td>) 0.2</td><td>(%)</td><td>)</td></tr><tr><td>0 5</td><td>863 432</td><td>2.9</td><td>100 100</td><td>1.4 32.0</td><td>N/A 0.5</td><td>0.3 217</td><td>N/A 3.1</td><td>0.0 63.7</td><td>N/A 0.9</td><td>174</td><td>N/A</td><td>2.2</td><td>N/A 2.9</td></tr><tr><td>50</td><td>4332</td><td>7041 7090</td><td>100</td><td>0.0</td><td>0.0</td><td>224</td><td>3.2</td><td>93.1</td><td>1.3</td><td>168</td><td>2.5 2.4</td><td>206 189</td><td>2.7</td></tr><tr><td>500</td><td>4313</td><td>7130</td><td>100</td><td>0.0</td><td>0.0</td><td>240</td><td>3.4</td><td>96.9</td><td>1.4</td><td>164</td><td>2.3</td><td>217</td><td></td></tr><tr><td>5,000</td><td>432</td><td>7152</td><td>100</td><td>0.0</td><td>0.0</td><td>208</td><td>2.9</td><td>85.7</td><td>1.2</td><td>179</td><td>2.5</td><td>211</td><td>3.0 3.0</td></tr><tr><td colspan="10">Agrmt = Agreement, Conc = Concentration, CV = Coefficient of Variation, N/A = Not Applicable for negative samples, RLU = Relative Light Units,</td><td colspan="4"></td></tr></table>

# TIGRIS DTS System Analytical Performance Characteristics

# Analytical Sensitivity Equivalence Study

Sensitivity panels in endocervical swab pool, vaginal specimen pool, urine specimen pool, and PreservCyt liquid Pap specimen pool were prepared at CT rRNA equivalent of 1 IFU per assay (7.25 IFU/swab and $5 \ \mathrm { I F U / m L }$ urine) and tested 60 replicates on the TIGRIS DTS System. Percent positivity $9 5 \%$ C.I.) on the TIGRIS DTS System for endocervical swab specimen was $100 \%$ (95.1 - 100), for vaginal swab specimen was $100 \%$ (95.1 - 100), for urine specimen was $100 \%$ (95.1 - 100), and for PreservCyt liquid Pap specimen was $100 \%$ (95.1 - 100).

# CT rRNA Spiked Clinical Panel Study

The CT rRNA spiked clinical panel study evaluated agreement between the two systems (TIGRIS DTS System and DTS Systems) using six Gen-Probe prepared CT clinical

panels spiked with 0 to 5,000 fg rRNA/assay of CT. The CT clinical panels were created from endocervical swab, vaginal swab, urethral swab, male urine, female urine, and PreservCyt liquid Pap specimens that had negative APTIMA CT results on the DTS Systems when tested at Gen-Probe. The negative specimens were pooled by specimen type, spiked or not spiked with CT rRNA and aliquotted as replicates of each panel member. Replicates of each of 6-panel members with different spiked rRNA levels were combined to create one clinical panel for each specimen type. Each panel contained a total of 132 replicates.Table 13 shows the percent agreement for each level of rRNA in the endocervical swab, vaginal swab, urethral swab, male urine, female urine, and PreservCyt liquid Pap panels, respectively, with expected CT results for the TIGRIS DTS System and for the DTS Systems. The concentration ranged from 1 log below to 3 logs above the 5 fg rRNA/assay for CT. Also shown in Table 13 are the overall percent agreements of the clinical panel study between the TIGRIS DTS System and DTS Systems.

Table 13: CT rRNA Spiked Clinical Panel Agreement Study   

<table><tr><td colspan="2">Specimen</td><td rowspan="2"></td><td colspan="2">Concentratio n</td><td colspan="2">TIGRIS % DTS %</td><td rowspan="2">Overall % Agreement</td></tr><tr><td></td><td>Panel Member</td><td>(fg</td><td>Replicates</td><td></td><td></td></tr><tr><td rowspan="12">Swab</td><td></td><td>0</td><td>rRNA/Assay)</td><td></td><td></td><td>t</td><td>Agreement Agreemenbetween TIGRIS</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td>and DTS (95%</td></tr><tr><td></td><td>No Target Very Low</td><td>0.5</td><td>12 30</td><td>100 100</td><td>100 100</td><td></td></tr><tr><td>EndocervicLow al</td><td></td><td>5</td><td>30</td><td>100</td><td>100</td><td>100 (97.2-100)</td></tr><tr><td></td><td>Medium</td><td>50</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td></td><td>High</td><td>5,000</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td rowspan="5">Vaginal</td><td>No Target</td><td>0</td><td>12</td><td>100</td><td>100</td><td></td></tr><tr><td>Very Low</td><td>0.5</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>Low</td><td>5</td><td>30</td><td>100</td><td>100</td><td>100 (97.2-100)</td></tr><tr><td>Medium</td><td>50</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>High</td><td>5,000</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td rowspan="6">Urethral</td><td>No Target</td><td>0</td><td>12</td><td>100</td><td>100</td><td></td></tr><tr><td>Very Low</td><td>0.5</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>Low</td><td>5</td><td>30</td><td>100</td><td>100</td><td>100 (97.2-100)</td></tr><tr><td>Medium</td><td>50</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>High</td><td>5,000</td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>No Target</td><td>0</td><td>12</td><td>91.7</td><td>100</td><td></td></tr><tr><td rowspan="2"></td><td rowspan="2">Male</td><td rowspan="2">Very Low Low Medium High</td><td rowspan="2">0.5 5 50 5,000</td><td>30 30 30</td><td>100 100 100</td><td rowspan="2">100 100 100</td><td rowspan="2">99.2 (95.9-100)</td></tr><tr><td>30</td><td>100</td></tr><tr><td rowspan="5">Urine Female</td><td>No Target</td><td>0</td><td></td><td>12</td><td></td><td>100</td><td></td></tr><tr><td></td><td>0.5</td><td></td><td></td><td>100</td><td>100 100</td><td></td></tr><tr><td>Very Low Low</td><td></td><td>30</td><td>100 100</td><td></td><td></td><td>100 (97.2-100)</td></tr><tr><td>Medium</td><td>5 50</td><td>30 30</td><td>100</td><td></td><td>100 100</td><td></td></tr><tr><td>High</td><td>5,000</td><td>30</td><td>100</td><td></td><td>100</td><td></td></tr><tr><td rowspan="5">PreservCyt liquid Pap</td><td>No Target</td><td>0</td><td></td><td>12</td><td>100</td><td>100</td><td></td></tr><tr><td>Very Low</td><td>0.5</td><td></td><td>30</td><td>100</td><td>100</td><td></td></tr><tr><td>Low</td><td>5</td><td>30</td><td>100</td><td></td><td>100</td><td>100 (97.2-100)</td></tr><tr><td>Medium</td><td>50</td><td>30</td><td>100</td><td></td><td>100</td><td></td></tr><tr><td>High</td><td>5,000</td><td>30</td><td>100</td><td></td><td>100</td><td></td></tr></table>

# Analytical Specificity Equivalence Study

For a nucleic acid amplification assay, analytical specificity with respect to individual organisms is largely determined by the chemistry of the assay (e.g. oligonucleotide sequences) rather than by the platform. Because the reagents for the APTIMA CT Assay are identical between the TIGRIS DTS System and the DTS Systems, analytical specificity experiments on the TIGRIS DTS System were designed to focus on the most challenging culture isolates. These organisms included those known to cross-react in other amplification assays. Twenty-four (24) culture isolates were selected from the panel of organisms in Table 10, including 3 organisms that are most closely related to CT. All of the organisms tested produced negative results on the TIGRIS DTS System.

# Interfering Substances Equivalence Study

Whole blood, a substance commonly found in urogenital specimens and known to interfere in some amplification assays, was used to establish that the TIGRIS DTS System tolerates similar levels of potentially interfering substances as does the DTS Systems. Fresh blood was added to clinical swab, vaginal swab, urine, and PreservCyt liquid Pap specimen pools, then tested for potential assay interference in the absence and presence of CT target at the estimated rRNA equivalent of one CT IFU/assay (5 fg/assay). The rRNA equivalents were calculated based on the genome size and estimated DNA:RNA ratio/cell of each organism. Specimens were tested on two TIGRIS DTS Systems. All samples containing target nucleic acid were positive when tested at a level of $10 \%$ blood in swab specimens, vaginal swab specimens, PreservCyt liquid Pap specimens, and $30 \%$ blood in urine specimens. All samples that did not contain target were negative for CT. These results indicate that at the levels tested, whole blood is unlikely to affect the CT result on the TIGRIS DTS System.

# Carryover Studies

To establish that the TIGRIS DTS System minimizes the risk of false positive results arising from carryover contamination, a study was conducted using spiked panels on three TIGRIS DTS Systems. The study used $20 \%$ high-target samples containing $1 \mathrm { ~ x ~ } 1 0 6$ fg CT rRNA/mL, which were randomly spaced amongst $80 \%$ negative samples

containing swab transport media. In the study, 576 high-target samples and 2,376 negative samples were tested across the three TIGRIS DTS Systems. Table 14 shows the overall carryover rate was averaged at $0 . 2 1 \%$ (5/2364). A total of 12 negative samples were reported as invalid and were excluded from the calculation. A separate analysis was conducted on a subset of the study population comprised of the negative samples that immediately followed a high-target positive. The carryover rate for this subset of the population was averaged at $0 . 4 7 \%$ (2/424). For false positives in this subset, the carryover rate ranged from $0 \%$ to $1 . 4 3 \%$ across the three TIGRIS DTS Systems. These results demonstrate that carryover contamination is minimized on the TIGRIS DTS System.

Table 14: Summary of Overall TIGRIS DTS System Carryover   

<table><tr><td rowspan="3">Instrument</td><td># Valid</td><td>Total # CT False</td><td>% CT False</td><td>Confidence Intervals</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>Negative Tests</td><td>Positive Results</td><td>Positive Results</td><td>(95% CI)</td></tr><tr><td>TIGRIS 1</td><td>789</td><td>2a</td><td>0.25</td><td>0.03 - 0.91</td></tr><tr><td>TIGRIS 2</td><td>783</td><td>3b</td><td>0.38</td><td>0.08 - 1.12</td></tr><tr><td>TIGRIS 3</td><td>792</td><td>0</td><td>0.00</td><td>0.00 - 0.38</td></tr><tr><td>All Instruments a. TIGRIS 1 had no false CT positive result directly following</td><td>2364</td><td>5</td><td>0.21</td><td>0.07 - 0.49</td></tr><tr><td colspan="5">a high-target positive b. TIGRIS 2 had two false CT positive results directly following a high-target positive c. TIGRIS 3 had no false CT positive result directly following</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.