# 510(k) SUMMARY

This summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1990 and 21 CFR 807.92.

The assigned 510(k) number is: K063261.

# TABLE OF CONTENTS

NAME OF DEVICE..

CLASSIFICATION. 2

INTENDED USE OF THE DEVICE . 2

DEVICE TO WHICH EQUIVALENCE IS CLAIMED.. .. 2

DESCRIPTION OF THE DEVICE.. 7

Device Description: miniMAG sample preparation.. 3 Device Description: EasyQ amplification and detection.. 3 Device Description: Controls.

# CLINICAL PERFORMANCE..

Clinical Study Design ... Agreement of EasyQ test with Cell Culture vs. Clinical Diagnosis.. 4 NucliSENS Enterovirus Test Results by Age and Gender. 6 Expected Values. 6

# PERFORMANCE CHARACTERISTICS OF THE ASSAY. 7

Reproducibility....

Analytical sensitivity: Limit of Detection (LoD).. .8

Analytical reactivity: Additional Enterovirus Serotypes. .9

Potentially Interfering Substances. .. 10

Analytical specificity: Potentially Interfering Substances. .10

Specimen Stability... .11

LIMITATIONS OF THE PROCEDURE. .. 11

REFERENCES.. 12

# Name of device

NucliseNS Easy $\boldsymbol { Q } ^ { ( \hat { 1 } ) }$ Enterovirus v1.1 Assay, including the following components:

NucliSENS Easy $Q ^ { ( \sqrt { 3 } ) }$ Enterovirus v1.1 NucliSENS Easy $\boldsymbol { \mathcal { Q } } ^ { \flat }$ Enterovirus Controls NucliSENS Easy $\alpha ^ { \mathrm { ( ) } }$ Analyzer NucliSenS Easy $\alpha ^ { \textregistered }$ Director Software 2.5 NuclisenS Easy $\boldsymbol { \mathcal { Q } } ^ { ( \Vec { \mathcal { P } } ) }$ Incubator NuclisENs miniMA $G ^ { \ddot { \textmd { B } } }$ NucliSENs® Lysis Buffer 48T NucliSENS® Magnetic Extraction Reagents

# Classification

Assay, enterovirus nucleic acid

# Intended Use of the Device

The NucliSens EasyQ Enterovirus v1.1 Assay is an in vitro nucleic acid amplification assay to be used in conjunction with the NucliSens EasyQ System for the qualitative detection of Enterovirus RNA in cerebral spinal fluid (CSF) specimens in patients with signs and symptoms of meningitis. This test, in conjunction with other laboratory results and clinical information, may be used as an aid in the presumptive laboratory diagnosis of enterovirus infection in pediatric patients with a clinical suspicion of aseptic meningitis or aseptic meningioencephalitis.

Negative results should be confirmed by cell culture.

Assay performance characteristics have not been established for adults, or for immunocompromised or immunosuppressed patients.

Caution: The results obtained with the NucliSens EasyQ Enterovirus v1.1 Assay should be used only as an adjunct to clinical observation and other information available to the physician. Positive results do not rule out other causes of meningitis, including bacteria, mycobacteria, other viruses (e.g. herpes family viruses, arboviruses, mumps virus, etc.) and fungi.

# Device to which equivalence is claimed

Cell Culture for Enterovirus Detection [pre-amendment test method; no 510(k) applies]

# Description of the Device

NucliSENS EasyQ® Enterovirus v1.1 is an in-vitro diagnostic assay which uses nucleic acid amplification combined with a simultaneous detection step to detect the presence of enteroviral RNA in eluates derived from cerebrospinal fluid or appropriate control material. The assay requires extracted nucleic acid as input material, and has been validated using nucleic acid eluates extracted from clinical specimens with NucliSENS® miniMAG ™. The amplification step is performed using the NucliSENS EasyQ System.

# Device Description: miniMAG sample preparation

NuclisENS $\textcircled{8}$ miniMAG™ utilizes NucliSENs® Lysis Buffer and magnetized silica beads to extract nucleic acids from lysed biological specimens, according to the operating principles initially described by Boom et al.' The end product of a NucliSENS® miniMAG™ extraction is an eluate containing total nucleic acid $( \mathsf { D N A } + \mathsf { R N A } )$ from the specimen.

# Device Description: EasyQ amplification and detection

The NucliSENS $\bigoplus$ easy $Q ^ { \top M }$ System utilizes Nucleic Acid Sequence-Based Amplification (NASBA) and detection of fluorescence from specific molecular beacons to signal the presence of target nucleic acid sequences. The NASBA reaction requires the use of specific reagents, including the enzymes, primers and probes which are components of the NucliSENS Easy® Enterovirus v1.1 assay. Reactions are performed in closed tubes in a NuclisENs EasyQ® Analyzer in which fluorescence is measured in real-time. NucliSENS EasyQ® Director Software, in combination with NucliSENS EasyQ Enterovirus v1.1 assay software, provides automated analysis of the resulting fluorescence signal curves and reporting of assay results.

# Device Description: Controls

The NucliSENS $\mathfrak { Q }$ easyQT System utilizes 3 types of controls:

Internal Control RNA, provided as a component of the NucliSENS EasyQ® Enterovirus v1.1 kit, is used to monitor nucleic acid extraction and subsequent amplification. The Enterovirus Internal Control RNA is co-extracted and co-amplified along with any Wild Type Enterovirus RNA present in the eluted nucleic acids. This Internal Control RNA differs from the Enterovirus Wild Type RNA by only a short nucleotide sequence designed to ensure similar amplification kinetics. Primer binding sites on the Internal Control RNA and Enterovirus RNA are identical, but the probe binding sequence differs, allowing differentially labeled molecular beacons (provided in the kit) to differentially detect the amplicons from wild-type Enteroviral RNA versus the Internal Control.

Negative and Positive Controls: NucliSENS $E a s y Q ^ { \beta ) }$ Enterovirus Controls are an external negative and external positive control (nuclease-free water and synthetic enteroviral RNA transcript, respectively) designed for use with the NucliSENS $E a s y Q ^ { \phi }$ Enterovirus v1.1 to monitor the performance of the assay. When used as directed, the positive control is present in the test at $\Im \times$ LoD $9 5 \%$ Limit of Detection) for this transcript. The Positive Control provides quality assurance for the amplification and detection procedures of the assay, for the extraction reagents, and for the NuclisENS $E a s y m A G ^ { ( ) }$ instrument operation. The Positive Control does not provide quality assurance for viral lysis or for the efficiency of RNA extraction from virions. NucliSENS Easy $\boldsymbol { Q } ^ { \mathfrak { G } }$ Enterovirus Controls are bioMérieux products which are sold separately from the NuclisENs Easy $Q ^ { \phi }$ Enterovirus v1.1 kit.

Lysis and Viral Extraction Controls: The NucliSENS Easy $Q ^ { ( j ) }$ Enterovirus Positive Control does not provide quality assurance for viral lysis or for the efficiency of RNA extraction from virions therefore we recommend the use of any one of 4 viral strains, available from ATCC, which have been shown to be appropriate for use as a control for lysis and extraction of virions in the NuclisENs $E \partial S y \partial ^ { \textregistered }$ Enterovirus v1.1 test, as described in the Instructions for Use provided with the NucliSENS Easy $Q ^ { \phi ) }$ Enterovirus v1.1 test.

# Clinical Performance

# Clinical Study Design

At six clinical sites, the NucliSENs Easy® Enterovirus v1.1 assay and cll culture (with identification by immunofluorescence [IFA] ) were used to evaluate cerebrospinal fluid (CSF) specimens from 520 pediatric subjects (0 to 21 years of age) who presented with signs and/or symptoms consistent with suspected aseptic meningitis (for example, fever, headache, stiff neck). Most $( 4 4 9 / 5 2 0 = 8 6 . 3 \% )$ CSF specimens were prospectively collected for this study and tested while fresh. However a minority (71/520 $= 1 3 . 7 \% )$ had been previously collected and stored at $- 7 0 \circ c$ prior to testing for this study. The six clinical sites were located in Ohio, Missouri, Texas (2 sites), Nebraska and California. Testing was performed at hospital laboratories associated with the testing sites in 2004-2005.

Bacteria gram stain, bacteria culture, and CSF profile (protein mg/dL, glucose mg/dL, white blood cell count, differential) were also collected for the overwhelming majority of patients. For many patients, serum glucose, white blood cell (WBC) count, complete blood cell (CBC) profile and CBC profile differential were also determined. At 5 of 6 clinical testing sites, the attending physicians ordered laboratory-developed tests for detection of enterovirus nucleic acid.

For each subject, the final diagnosis, as determined by the attending physicians based on signs and symptoms as well as laboratory testing results, was reported to the study investigator. Results from the NuclisENs Easy $Q ^ { \sqrt { S } }$ Enterovirus v1.1 were not provided to the attending physicians, and did not influence the final diagnosis.

# Agreement of EasyQ test with Cell Culture vs. Clinical Diagnosis

Results from the clinical study are presented below. Results from the NuclisENs $E \cos y Q ^ { \textregistered }$ Enterovirus v1.1 are compared against results from cell culture (coupled with immunofluorescent detection), and against the clinical diagnoses assigned by the attending physicians. Among 449 prospectively collected specimens, the NuclisENS $\pmb { \mathcal { E } } a s y Q ^ { ( \sharp ) }$ Enterovirus v1.1 displayed $80 . 9 \%$ sensitivity and $9 9 . 6 \%$ specificity versus clinical diagnosis. Additional data and analyses of the agreement between the NucliSENs Easy® Enterovirus v1.1, cell culture, and clinical diagnosis are presented in Table 1 and Table 2 below.

Table 1 Prospectively collected CSF specimens: EasyQ vs. Cell Culture and Clinical Diagnosis   

<table><tr><td>NucliSENS EasyQ Enterovirus v1.1</td><td>CSF Cell Culture + IFA</td><td>Total</td><td colspan="2">Clinical Diagnosis</td></tr><tr><td></td><td></td><td>Patients</td><td>Positive</td><td>Negative</td></tr><tr><td>+</td><td>+</td><td>114</td><td>114</td><td>0</td></tr><tr><td>+</td><td>-</td><td>35</td><td>34</td><td>1</td></tr><tr><td>1</td><td>+</td><td>18</td><td>17</td><td>1</td></tr><tr><td></td><td>-</td><td>282</td><td>18</td><td>264</td></tr><tr><td>Total</td><td></td><td>449</td><td>183</td><td>266</td></tr></table>

# Agreement of tests or of EasyQ/Clinical Diagnosis (Prospectively collected CSF specimens, $n { = } 4 4 9$ .

Sensitivity: EasyQ vs. Clinical Diagnosis (148/183) 80.9% 74.4 - 86.3% Specificity: EasyQ vs. Clinical Diagnosis (265/266) 99.6% 97.9 - 99.9% EasyQ Positive Predictive Value with Clinical Diagnosis (148/149) 99.3% 96.3 - 99.9% EasyQ Negative Predictive Value with Clinical Diagnosis (265/300) 88.3% 84.1 - 91.7% Positive Agreement of EasyQ and Cell Culture (114/132) 86.4% 79.3 - 91.7% Negative Agreement of EasyQ and Cell Culture (282/317) 89.0% 85.0 - 92.2% \*Confidence interval for the percent agreement noted in the previous column

Table 2 Retrospectively collected CSF specimens: EasyQ vs. Cell Culture and Clinical Diagnosis   

<table><tr><td rowspan="2">NucliSENS EasyQ Enterovirus v1.1</td><td rowspan="2">CSF Cell Culture + IFA</td><td rowspan="2">Total Patients</td><td colspan="2">Clinical Diagnosis</td></tr><tr><td>Positive</td><td>Negative</td></tr><tr><td>+</td><td>+</td><td>18</td><td>18</td><td>0</td></tr><tr><td>+</td><td></td><td>2</td><td>1</td><td>1</td></tr><tr><td>-</td><td>+</td><td>0</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td>51</td><td>0</td><td>51</td></tr><tr><td>Total</td><td></td><td>71</td><td>19</td><td>52</td></tr></table>

# Agreement of tests or of EasyQ/Clinical Diagnosis (Retrospectively collected CSF specimens, $\bf \Pi = 7 1 \cdot$

Sensitivity: EasyQ vs. Clinical Diagnosis (19/19) Specificity: EasyQ vs. Clinical Diagnosis (51/52) Positive Agreement of EasyQ and Cell Culture (18/18) Negative Agreement of EasyQ and Cell Culture (51/53) \*Confidence interval for the percent agreement noted in the previous columr

# NuclisENs Enterovirus Test Results by Age and Gender

The performance of the NucliSENS EasyQ Enterovirus v1.1 test as compared to clinical diagnosis was assessed for newborns, infants, children and adolescents. Results are shown in Table 3 below. Results for prospectively collected (fresh) specimens are presented separately from retrospectively collected (banked) specimens.

Table 3 Clinical Performance of Easy Enterovirus v1.1 by subject age groups   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Prospective (Fresh)</td><td rowspan=1 colspan=2>Retrospective (Banked)</td></tr><tr><td rowspan=1 colspan=1>Age Group*</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>Newborn(0-27 days)</td><td rowspan=1 colspan=1>80.0%(40/50)CI = 66.3-90.0%</td><td rowspan=1 colspan=1>100.0%(62/62)CI = 94.2-100.0%</td><td rowspan=1 colspan=1>100.0%(8/8)CI = 63.1-100.0%</td><td rowspan=1 colspan=1>94.7%(18/19)CI = 74.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>Infant/Toddler(28 days-23 months)</td><td rowspan=1 colspan=1>70.9%(61/86)CI = 60.1-80.2%</td><td rowspan=1 colspan=1>99.3%(143/144)CI = 96.2-99.9%</td><td rowspan=1 colspan=1>100.0%(10/10)CI = 69.2-100.0%</td><td rowspan=1 colspan=1>100.0%(21/21)CI = 83.9-100.0%</td></tr><tr><td rowspan=1 colspan=1>Child(2-11 years)</td><td rowspan=1 colspan=1>100.0%(37/37)CI = 90.5-100.0%</td><td rowspan=1 colspan=1>100.0%(46/46)CI = 92.3-100.0%</td><td rowspan=1 colspan=1>100.0%(1/1)C1 = 2.5-100.0%</td><td rowspan=1 colspan=1>100.0%(12/12)CI = 73.5-100.0%</td></tr><tr><td rowspan=1 colspan=1>Adolescent(12-21 years)</td><td rowspan=1 colspan=1>100.0%(10/10)CI = 69.2-100.0%</td><td rowspan=1 colspan=1>100.0%(14/14)CI = 76.8-100.0%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

\*Pediatric age groups as defined by ICH E11: International Conference on Hammonization, Guidance E11: Clinical Investigation of Medicinal Products in the Pediatric Population NOTE: Each data cell shows:

(1) $\%$ agreement between EasyQ and Clinical Diagnosis   
# specimens in agreement / total # positive by Clinical Diagnosis (for sensitivity) OR # specimens in agreement / total # negative by Clinical Diagnosis (for specificity)   
(3) $9 5 \%$ confidence interval

# Expected Values

Five-hundred twenty pediatric specimens from 6 clinical trial sites were tested with the NucliSENS EasyQ Enterovirus v1.1. Of these samples, 449 were prospectively collected clinical samples. Seventy-one specimens were previously collected at a single trial site. There was no statistically significant different between performance characteristics for prospectively versus retrospectively collected specimens. For each age and gender group listed, the number and percentage of specimens yielding positive results with the NucliSENS EasyQ Enterovirus v1.1 test are shown in Table 4 below.

Table 4 NucliSENS EasyQ Enterovirus v1.1 Test Results by Age and Gender   

<table><tr><td rowspan=2 colspan=1>Age Group*</td><td rowspan=2 colspan=1>Gender</td><td rowspan=1 colspan=2>Prospective</td><td rowspan=1 colspan=2>Retrospective</td></tr><tr><td rowspan=1 colspan=1>Positive n (%)</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive n (%)</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Newborn(0-27 days)</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>24 (41%)</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>5 (31%)</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>16 (30%)</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>4 (36%)</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Infant/Toddler(28 days-23 months)</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>34 (27%)</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>7 (35%)</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>28 (26%)</td><td rowspan=1 colspan=1>106</td><td rowspan=1 colspan=1>3 (27%)</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Child(2-11 years)</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>22 (45%)</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1 (8%)</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>15(44%)</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=2 colspan=1>Adolescent(12-21 years)</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>5 (45%)</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>5 (38%)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>both</td><td rowspan=1 colspan=1>149</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>71</td></tr></table>

\*Pediatric age groups as defined by ICH E11: International Conference on Harmonization, Guidance E11: Clinical Investigation of Medicinal Products in the Pediatric Population

# Performance Characteristics of the Assay

# Reproducibility

A reproducibility study was conducted using a panel with representative strains from four Enterovirus serotypes: Coxsackie A9, Coxsackie B5, Echovirus 30, and Enterovirus 71. Titered stocks were diluted to the estimated $9 5 \%$ Limit of Detection for each strain. Two additional panel members were prepared to detect reproducibility at lower hit rates, using using lower concentrations of Coxsackie A9 (these samples were 0.1 $T C l D _ { 5 0 }$ and 0.02 $\mathrm { T C l D } _ { 5 0 }$ , versus LoD $9 5 \% = 0 . 4 7 C 1 0 _ { 5 0 } )$ . Testing was performed at three sites. At each evaluation site, one investigator conducted the evaluation. Each panel member was tested in duplicate for four (4) days by each investigator using two reagent kit lots. Each of the 7 panel members was tested in duplicate in each of 4 days (one run per day) with 2 lots at each of 3 sites. Each of the controls was tested in singlicate in each day (one run per day) with 2 lots at each of 3 sites.

Fluorescence values in the EasyQ® system are not reported to users as part of the test output. However for this analysis, fluorescence values were retrieved from the instrument memory and were subjected to variance components analysis. For each panel member and for each control, variance components analysis was used to estimate total variability and variability due to site, lot, run within lot (test day), and (for panel members only) duplicates within run. For the sources of variability with estimates of zero for the standard deviation (SD) and zero for the coefficient of variation $( C V \% )$ , the zeros indicate instances of negative variance components. Variation due to these sources was negligible relative to the other sources of variation.

Variance components analysis indicated that variability within runs (i.e., between replicates for single specimens) was the major source of variability in this precision experiment. Variability between sites, between lots, and between test days contributed little additional variability to the total.

Table 5 below presents qualitative results from this experiment based on positive versus negative result output. The table also shows the mean fluorescence values obtained for testing with different panel specimens, and variance components analysis of the variability in these fluorescence values. Relative contributions to variability in

fluorescence signals from variability within runs, between test days, between lots and between sites are shown.

Reproducibility results: panel with four Enterovirus serotype strains   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>N</td><td rowspan=2 colspan=1>Totalositive</td><td rowspan=2 colspan=1>95%ci</td><td rowspan=2 colspan=1>MeanVValue*</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenDays</td><td rowspan=1 colspan=2>BetweenLots</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=1>Cox A9LoD 95%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>100%(48/48)</td><td rowspan=1 colspan=1>92.6-100%</td><td rowspan=1 colspan=1>3.56</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>16.8</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>7.3</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>29.7</td></tr><tr><td rowspan=1 colspan=1>Cox A9moderate</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>85%(41/48)</td><td rowspan=1 colspan=1>72.2-93.9%</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>18.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>12.6</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>33.3</td></tr><tr><td rowspan=1 colspan=1>Cox A9low</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>23%(11/48)</td><td rowspan=1 colspan=1>12.0-37.3%</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>27.0</td></tr><tr><td rowspan=1 colspan=1>Cox B5LoD 95%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>94%(45/48)</td><td rowspan=1 colspan=1>82.8-98.7%</td><td rowspan=1 colspan=1>2.81</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>37.1</td></tr><tr><td rowspan=1 colspan=1>Echo 30LOoD 95%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>98%(47/48)</td><td rowspan=1 colspan=1>88.9-99.9%</td><td rowspan=1 colspan=1>2.68</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>31.8</td></tr><tr><td rowspan=1 colspan=1>Entero 71LOD 95%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>88%(42/48)</td><td rowspan=1 colspan=1>74.8-95.3%</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>34.5</td></tr><tr><td rowspan=1 colspan=1>NegativePanel</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>0%(0/48)</td><td rowspan=1 colspan=1>0-7.4%</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0%(0/31)</td><td rowspan=1 colspan=1>0-11.2%</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>3.7</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>100%((331/31)</td><td rowspan=1 colspan=1>88.8- 100%</td><td rowspan=1 colspan=1>4.91</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>29.6</td></tr></table>

\*Mean background corrected fluorescencevalues (recorded on instrument but not reported in user outut) SD $=$ standard deviation $\complement \lor \% =$ coefficient of variation; 100xSD/mean ${ \sf C l } =$ confidence interval Cox A9 $\backsimeq$ Coxsackie A9 Cox A9 moderate $\mathbf { \delta } = \bar { \mathbf { 0 . 1 } }$ $\mathsf { T C l } \mathsf { D } _ { 5 0 }$ (versus $0 . 4 \mathrm { \ T C } | \mathsf { D } _ { 5 0 }$ at LoD $9 5 \%$ ) $\mathsf { C o x } \mathsf { A } \mathsf { 9 } \mathsf { l o w } = 0 . 0 2 \mathsf { T C } \mathsf { l D } _ { 5 0 }$ (versus 0.4 $\mathsf { T C I D } _ { 5 0 }$ at LoD $9 5 \%$ Cox $\boldsymbol { \mathsf { B 5 } } =$ Coxsackie B5 Echo $\mathfrak { z o } =$ Echovirus 30 Entero $7 1 =$ Enterovirus 71 $N / A =$ not applicable (negative and positive controls were run in singlicate)

# Analytical sensitivity: Limit of Detection (LoD)

One representative serotype from each of the four major serotype groups was evaluated with multiple tests over a concentration range (12 replicates per level) to derive an LoD value for $9 5 \%$ detection. This estimate was then used to conduct additional verification testing at the presumed LoD $9 5 \%$ value (20 replicates per level). LoD $9 5 \%$ values were calculated using Probit regression analysis.

The results of this study are summarized in Table 6.

Table 6 $95 \%$ LoD values for representative clinically relevant serotypes from each of the major serotype groups   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD 95 %(a)</td></tr><tr><td rowspan=1 colspan=1>Coxsackie A</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>P.B. (Bozek)</td><td rowspan=1 colspan=1>0.6 TCID50</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Faulkner</td><td rowspan=1 colspan=1>3 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Bastianni</td><td rowspan=1 colspan=1>46 pfu</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>BrCr</td><td rowspan=1 colspan=1>2 TCID50</td></tr></table>

(a) pfu $\equiv$ plaque forming unit, ${ \mathrm { T C } } | { \mathrm { D } } _ { 5 0 } =$ Tissue Culture Infectious Dose $50 \%$

For the remaining 18 serotypes, the LoD $9 5 \%$ values derived in a range-finding study were used as the basis for a more limited verification testing approach by testing two dilutions (20 replicates at each level) near the estimated LoD $9 5 \%$ value. The results of this study are summarized in Table 7.

Table 7 Estimated $95 \%$ LoD values for Clinically Relevant Enterovirus serotypes   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD 95 %(a)</td></tr><tr><td rowspan=1 colspan=1>Coxsackie A</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>G-10</td><td rowspan=1 colspan=1>0.3 TCID50</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Conn-5</td><td rowspan=1 colspan=1>717 pfu</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Ohio-1</td><td rowspan=1 colspan=1>2.6 pfu</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Nancy</td><td rowspan=1 colspan=1>2.5 pfu</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>J.V.B (Benschoten)</td><td rowspan=1 colspan=1>29.3 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Morrisey</td><td rowspan=1 colspan=1>10 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Pesascek</td><td rowspan=1 colspan=1>0.9 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Noyce</td><td rowspan=1 colspan=1>933 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>D&#x27;Amori</td><td rowspan=1 colspan=1>15.4 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Wallace</td><td rowspan=1 colspan=1>45 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Hill</td><td rowspan=1 colspan=1>0.3 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Gregory</td><td rowspan=1 colspan=1>40 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>DelCarmen</td><td rowspan=1 colspan=1>12.4 TCID50</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>TOW</td><td rowspan=1 colspan=1>0.2 TCID50</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Harrington</td><td rowspan=1 colspan=1>135 TCID50</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>CHHE-29</td><td rowspan=1 colspan=1>0.02 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>D3; Metcalf</td><td rowspan=1 colspan=1>0.01 pfu</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>JV-4</td><td rowspan=1 colspan=1>0.3 pfu</td></tr></table>

(a) pfu $=$ plaque forming unit, $T C | \mathsf { D } _ { 5 0 } =$ Tissue Culture Infectious Dose $50 \%$

# Analytical reactivity: Additional Enterovirus Serotypes

An additional study, summarized in Table 8 below, determined that the NucliSENs Easy $Q ^ { \phi }$ Enterovirus v1.1 assay was able to detect Enterovirus 68 and Polioviruses 1, 2, and 3 at the concentrations indicated below. The LoD's for these serotypes were not determined. This study also indicated that the NucliSENS Easy $\pmb { Q } ^ { ( \bar { \mathcal { B } } ) }$ Enterovirus $v 1 . 1$ assay was able to detect a number of other enterovirus strains, listed below. The concentrations of these additional strains was not known.

Table 8 Detection of Additional Enterovirus Serotypes   

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>TCIDso input</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Poliovirus</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Poliovirus</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Poliovirus</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Coxsackie</td><td rowspan=1 colspan=1>A2, A3, A4, A6, A7, A8, A10, A11, A12,A15, A17, A18, A20, A21, A24, B6</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>Echovirus</td><td rowspan=1 colspan=1>1, 2, 12, 15, 19, 20, 21, 24, 26, 27, 29,31, 32, 33</td><td rowspan=1 colspan=1>unknown</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>69,70</td><td rowspan=1 colspan=1>unknown</td></tr></table>

# Potentially Interfering Substances

A study was conducted with potentially interfering substances encountered in CSF. Substances tested were proteins (albumin, gammaglobulin), bilirubin, white blood cells, and whole blood.

The interferent was tested at different concentration levels with negative CSF. Each interferent was tested at a concentration above that normally seen in clinical cases of enteroviral meningitis. Specimens were spiked with Echovirus 30 virions (100 pfu, approx. 2x LoD) to mimic positive clinical specimens. All unspiked specimens gave valid negative results. The Enterovirus-spiked specimens consistently yielded valid positive results. Elevated levels of the tested substances did not interfere with the detection of Enterovirus RNA by the NucliSENS EasyQ® Enterovirus v1.1 assay even at the highest tested level.

# Analytical specificity: Potentially Interfering Substances

Cross-reactivity of various micro-organisms known to cause central nervous system infections was assessed. Each potential cross-reacting agent was tested with and without an Enterovirus spike (Echovirus 30 at 2-3x LoD; viral cross-reacting agents at $2 1 0 ^ { 3 } \tau \mathsf { C } | \mathsf { D } _ { 5 0 }$ ; bacterial cross-reacting agents at $e 1 0 ^ { 3 }$ cfu/mL ). Results indicated that the NucliSENS Easy $\cdot Q ^ { \circledast }$ Enterovirus v1.1 assay accurately detected the added Enteroviral RNA in the presence of the potential interfering substances, while yielding no false positives in unspiked specimens. Therefore the overall specificity in this study was $100 \%$ (i.e. 28 of 28 types of interfering substances). Microorganisms tested in this study were:

Haemophilus influenza type b Streptococcus pneumoniae Neisseria meningitides   
Escherichia coli   
Streptococcus agalactiae   
Klebsiella pneumoniae   
Staphylococcus aureus   
Enterobacter sp.   
Citrobacter sp. Proteus sp.   
Salmonella sp.   
Pseudomonas sp.   
Staphylococcus epidermidis Herpes Simplex Virus Type 1 Herpes Simplex Virus Type 2 Varicella Zoster   
Cytomegalovirus   
Rotavirus   
Rubella   
Parvovirus B19   
West Nile Virus   
Mumps   
Measles   
Cryptococcus neoformans   
Naegleria fowleri   
Rhinovirus   
Encephalomyocarditis virus

Because of the structural and genetic similarities between enteroviruses and rhinoviruses, cross-reactivity with Rhinoviruses was further assessed with 88 rhinovirus strains, including 13 strains at defined levels of viral input between 100-10,000 $\mathrm { \Delta T C | \mathrm { \Delta D _ { 5 0 } } }$ The NucliSENS Easy $Q ^ { \circledcirc }$ Enterovirus v1.1 assay cross-reacted with 4 of the tested Rhinovirus strains. However, human cerebrospinal fluid is not the normal viral reservoir for rhinoviruses, and rhinoviruses are not a recognized cause of meningitis.,

# Specimen Stability

Testing with the NucliSENS Easy $Q ^ { ( i ) }$ Enterovirus v1.1 assay demonstrated that for purposes of detection using this assay, Enterovirus in CSF samples is stable for at least 4 days at $2 - 8 ^ { \circ } C$ or for at least 2 months at $\leq - 7 0 ^ { \circ } C$ $+ 1 -$ Lysis Buffer) CSF specimens can be frozen and thawed (freeze/thaw cycle $- 7 0 ^ { \circ } C / 3 7 ^ { \circ } C$ up to two times without measurable loss of Enterovirus RNA.

# Limitations of the Procedure

Positive results do NOT rule out other non-EV causes of meningitis. Positive results should be interpreted in conjunction with other laboratory findings (e.g., CSF glucose, CSF Gram stains, CSF protein, CSF leukocytes, etc.), and clinical signs or symptoms. In rare instances, meningitis can be caused by co-infection of a viral and bacterial or other agent.

False positive test results are more likely when prevalence of disease due to enterovirus is low or non-existent in a community, or outside the enteroviral season.

A negative result does not preclude the possibility of the presence of Enterovirus RNA in a sample. For example, only $54 \%$ of the cell-culture positive specimens with Coxsackie B5 we eteced with the Nucli Easy® Enterovius v1.1 assay.

Cross-reactivity of this assay with human rhinoviruses does occur. However, human cerebrospinal fluid is not the normal viral reservoir for rhinoviruses, and rhinoviruses are not a recognized cause of meningitis.

NucliSENs EasyQ® Enterovirus v1.1 has been shown to detect numerous Enterovirus serotypes (Coxsackie A2, A3, A4, A6, A7, A8, A9, A10, A11, A12, A15, A16, A17, A18, A20, A21, A24; Coxsackie B1, B2, B3, B4, B5; Echovirus 1, 2, 3, 4, 5, 6, 7, 9, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 24, 25, 26, 27, 29, 30, 31, 32, 33; Enterovirus 68, 69, 70, 71; Poliovirus 1, 2, 3). The ability to detect other diverse Enterovirus serotypes has not been established.

eult ha oivi o n  l

The performance of the NucliSENS EasyQ® Enterovirus v1.1 assay was established using human cerebral spinal fluid. The use of the device with other specimen types (e.g., stool) has not been established.

Tu  asi u relation to patient symptoms and clinical history to establish a diagnosis.

Testing of clinical specimens showed an initial invalid rate of $9 \%$ . All invalid results were resolved using the instructions for retesting as outlined in the Instructions for Use.

# JUN 2 3 2008

Sandra Perreand   
Senior Director, North American Regulatory Affair bioMerieux, Inc.   
595 Anglum Road   
St. Louis, MO 63042

Re: k063261 Trade/Device Name: NucliSens EasyQ® Enterovirus v1.1 Regulation Number: 21 CFR 866.3225 Regulation Name: Nucleic acid amplification assay system, enterovirus (EV) RNA Regulatory Class: Class II Product Code: OAI Dated: April 30, 2008 Received: May 1, 2008

Dear Ms. Perreand:

We have reviewed your Section 5 10(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, god manufacturing practice, labeling, and prohibitions againstmisbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or class II (PMA), it may be subject to such additional controls.Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

Page 2 -

This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at 240-276-0450. Also, pleas note the regulation entitled, "Misbranding by reference t premarket notification" (1CFR Part 807.97). For questions regarding postmarket surveillance, please contact CDRH's Office of Surveillance and Biometric's (OSB's) Division of Postmarket Surveillance at 240-276-3474. For questions regarding the reporting of device adverse events (Medical Device Reporting (MDR)), please contact the Division of Surveillance Systems at 240-276-3464. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (240) 276-3150 or at its Internet address http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

![](images/0b16a835e1b2e071e8984a2c884a93b4c979d988821f3602f894395936184832.jpg)

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices Offce of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and   
Radiological Health

# INDICATIONS FOR USE STATEMENT

510(k) Number (if known): K063261

Device Name: NucliSENS EasyQ® Enterovirus v1.1

Indications For Use: The NucliSens EasyQ® Enterovirus v1.1 Assay is an in vitro nucleic acid amplification assay to be used in conjunction with the NucliSens EasyQ® System for the qualitative detection of Enterovirus RNA in cerebral spinal fluid (CSF) specimens in patients with signs and symptoms of meningitis. This test, in conjunction with other laboratory results and clinical information, may be used as an aid in the presumptive laboratory diagnosis of enterovirus infection in pediatric patients with a clinical suspicion of aseptic meningitis or aseptic meningioencephalitis.

Negative results should be confirmed by cell culture.

Assay performance characteristics have not been established for adults, or for immunocompromised or immunosuppressed patients.

Caution: The results obtained with the NucliSens EasyQ Enterovirus v1.1 Assay should be used only as an adjunct to clinical observation and other information available to the physician. Positive results do not rule out other causes of meningitis, including bacteria, mycobacteria, other viruses (e.g. herpes family viruses, arboviruses, mumps virus, etc) and fungi).

Prescription Use _X Over-The-Counter Use AND/OR (Part 21 CFR 801 Subpart D) (21 CFR 801 Subpart C)

Concurrence of CDRH, Office of Device Evaluation (ODE)

Division Sign-Off

Office of In Vitro Diagnostic Device Evaluation and Safety

510(k). k063261