# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION ONLY TEMPLATE

A. 510(k) Number: K071026

B. Purpose for Submission: New device, new claims

C. Measurand: SCCmec cassette (MRSA specific gene) at orfX junction, SA specific gene

D. Type of Test: Nucleic Acid Amplification Test, DNA, Staphylococcus aureus (SA) and Methicillinresistant Staphylococcus aureus (MRSA), qualitative

E. Applicant: BD Diagnostics (GeneOhm Sciences Canada Inc.)

F. Proprietary and Established Names: BD GeneOhm™ StaphSR Assay

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.1640

2. Classification: Class II

3. Product code: NQX

4. Panel: 83 Microbiology

# H. Intended Use:

1. Intended use(s):

The BD GeneOhm™ StaphSR Assay is a qualitative in vitro diagnostic test for the rapid detection of Staphylococcus aureus (SA) and methicillin-resistant Staphylococcus aureus (MRSA) directly from positive blood culture. The assay utilizes polymerase chain reaction (PCR) for the amplification of specific targets and fluorogenic target-specific hybridization probes for the real-time detection of

the amplified DNA.

2. Indication(s) for use:

The assay is performed on Gram positive cocci, identified by Gram stain, from positive blood cultures.

The BD GeneOhm™ StaphSR Assay is not intended to monitor treatment for MRSA/SA infections. Subculturing of positive blood cultures is necessary for further susceptibility testing.

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: Automated SmartCycle $\textsuperscript { \textregistered }$ System (instrument, software)

# I. Device Description:

The amplified DNA targets are detected with molecular beacon probes, hairpin-forming single stranded oligonucleotides labeled at one end with a quencher and at the other end with a fluorescent reporter dye (fluorophore). In the absence of target, the fluorescence is quenched. In the presence of target, the hairpin structure opens upon beacon/target hybridization, resulting in emission of fluorescence. For the detection of MRSA amplicon, the molecular beacon probe contains the fluorophore FAM at the $5 ^ { \circ }$ end and the non-fluorescent quencher moiety DABCYL at the opposite end of the oligonucleotide. For the detection of S. aureus amplicon, the molecular beacon probe is labeled with the fluorophore TexasRed at the $5 ^ { \circ }$ end and the quencher DABCYL at the $3 ^ { \circ }$ end. For the detection of the Internal Control (IC) amplicon, the molecular beacon probe contains the fluorophore TET at the $5 '$ end and the quencher DABCYL at the $3 ^ { \circ }$ end. Each beacon-target hybrid fluoresces at a wavelength characteristic of the fluorophore used in the particular molecular beacon probe. The amount of fluorescence at any given cycle, or following cycling, depends on the amount of specific amplicon present at that time. The SmartCycler $\textsuperscript { \textregistered }$ software simultaneously monitors the fluorescence emitted by each beacon probe, interprets all data, and provides a final result at the end of the cycling program. The whole procedure takes about 60-75 minutes.

The following controls were used for monitoring assay performance and included in the assay kit:

Internal control (IC) monitors the presence of inhibitory substances in the assay tube, also ensures that reaction conditions (temperature, time) of each step of the PCR in that tube are appropriate for the amplification reaction and that the reagents are functional. IC is also a reagent control. • Positive control (PC) is an assay run control. It is used in combination with the internal control to verify reagent and system functionality. Negative control (NC) is an assay run control, used to detect reagent or environmental contamination (or carry-over) by either S. aureus or MRSA DNA or amplicons.

BD Diagnostics BD GeneOhm™v StaphSR Assay PreMarket Notification

# 6.3. Outer Box Labels

6.3.1. 48 Test Format

![](images/8d7571a809fc9672361bd4cb7c50b3985c660cdc174ccb991afadbf4a0925a77.jpg)

# J. Substantial Equivalence Information:

1. Predicate device name(s): Remel Staphaurex Latex Agglutination BBL (BD) Oxacillin Screen Agar BD Phoenix Automated ID/AST System

2. Predicate 510(k) number(s): K851949 K863821 K020322, K023301

3. SA Comparison with predicate:

<table><tr><td rowspan=1 colspan=4>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD GeneOhmTStaphSR Assay</td><td rowspan=1 colspan=1>Agglutination Testfor SA</td><td rowspan=1 colspan=1>Automated test</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Detection of SA</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Single use</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Pos Control: SANeg Control: S.epidermidis</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=4>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD GeneOhmTStaphSR Assay</td><td rowspan=1 colspan=1>Agglutination Testfor SA</td><td rowspan=1 colspan=1>Automated test</td></tr><tr><td rowspan=1 colspan=1>Mode ofdetection</td><td rowspan=1 colspan=1>Nuc gene, specificfor SA</td><td rowspan=1 colspan=1>Clumping factorand protein A</td><td rowspan=1 colspan=1>Microbialutilization anddegradation ofspecific substrates</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Positive bloodculture</td><td rowspan=1 colspan=1>Staphylococcusspp</td><td rowspan=1 colspan=1>Gram positiveorganisms</td></tr><tr><td rowspan=1 colspan=1>Assay format</td><td rowspan=1 colspan=1>Amplication: PCRDetection:Fluorogenic target-specifichybridization</td><td rowspan=1 colspan=1>Agglutination withlatex particlessensitized withfibrinogen and IgG</td><td rowspan=1 colspan=1>Conventional,chromogenic andfluorogenicbiochemical tests</td></tr><tr><td rowspan=1 colspan=1>Interpretation oftest results</td><td rowspan=1 colspan=1>Diagnosticsoftware ofSmartCycler®</td><td rowspan=1 colspan=1>Visualinterpretation</td><td rowspan=1 colspan=1>Automated</td></tr></table>

MRSA Comparison with predicate:   

<table><tr><td rowspan=1 colspan=4>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD GeneOhmTMStaphSR Assay</td><td rowspan=1 colspan=1>Oxacillin ScreenAgar Test</td><td rowspan=1 colspan=1>Automated testfor resistance</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Detection ofMRSA</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Single use</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Pos Control:MRSANeg Control:MSSA</td><td rowspan=1 colspan=1>Same</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>BD GeneOhmTStaphSR Assay</td><td rowspan=1 colspan=1>Oxacillin ScreenAgar Test</td><td rowspan=1 colspan=1>Automated testfor resistance</td></tr><tr><td rowspan=1 colspan=1>Mode ofdetection formethicillinresistance</td><td rowspan=1 colspan=1>mecAgene,specific for MRSA</td><td rowspan=1 colspan=1>Growth on MuellerHinton Agar with4% NaCl and6μg/ml oxacillin</td><td rowspan=1 colspan=1>Redox indicatorfor the detection oforganism growthin the presence ofan antimicrobialagent</td></tr><tr><td rowspan=1 colspan=1>Sample type</td><td rowspan=1 colspan=1>Positive bloodculture</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Staphylococcusaureus</td></tr><tr><td rowspan=1 colspan=1>Assay format</td><td rowspan=1 colspan=1>Amplication: PCRDetection:Fluorogenic target-specifichybridization</td><td rowspan=1 colspan=1>Phenotypicdetection based ona 24 hr growth ofSA inoculated onmedia</td><td rowspan=1 colspan=1>AST panelscontaining MICtests for severalantimicrobialagents</td></tr><tr><td rowspan=1 colspan=1>Interpretation oftest results</td><td rowspan=1 colspan=1>Diagnosticsoftware ofSmartCycler®</td><td rowspan=1 colspan=1>Visualinterpretation</td><td rowspan=1 colspan=1>Automated</td></tr></table>

K. Standard/Guidance Document Referenced (if applicable): N/A

L. Test Principle:

In the BD GeneOhm™ StaphSR Assay $2 \mu \mathrm { L }$ of the positive blood culture are placed in the sample buffer. An aliquot is transferred to the lysis tube containing glass beads. Then cells are lysed through a combination of chemical and physical action (vortex in presence of beads and heat at $9 5 \pm 2 ^ { \circ } \mathrm { C }$ ). Without additional manipulation to purify the DNA present in the lysate, an aliquot of the lysate is added directly to the PCR master mix in a SmartCycler $\textsuperscript { \textregistered }$ reaction tube. The reaction tubes are placed in the SmartCycler $^ \mathrm { \textregistered }$ instrument. In specimens containing MRSA or SA, amplification of the targets [MRSA: a S. aureus specific target and a sequence near the insertion site of the Staphylococcal Cassette Chromosome mec (SCCmec); SA: another S. aureus specific sequence] occurs. Amplification of the IC, a DNA fragment not found in S. aureus or MRSA, also takes place unless PCR inhibitory substances are present.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility was assessed with three (3) lots of BD GeneOhm™ StaphSR Assay kit lots representing different Master Mix lots. The panel consisted of 12 tubes (labeled R1 to R12) containing $5 0 ~ \mu \mathrm { L }$ of a positive blood culture with either MRSA type ii, MSSA or a negative specimen (Staphylococcus epidermidis). Panel members were tested in triplicate on three (3) days at three (3) sites (12 specimens plus 2 kit controls tested x $3 \times 3$ days x 3 sites).

Inter-sites and across lots demonstrated $59 5 \%$ reproducibility.

Summary of Reproducibility Results by Lot across Sites   

<table><tr><td rowspan=1 colspan=1>Specimen ID</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Lot07T06016-48</td><td rowspan=1 colspan=1>Lot07T06017-48</td><td rowspan=1 colspan=1>Lot07T06022-48</td><td rowspan=1 colspan=1>Totalagreement</td><td rowspan=1 colspan=1>Total %agreement</td></tr><tr><td rowspan=1 colspan=1>R1 (negative)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R2 (negative)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R3 (MREJ ii)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R4 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-01</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R5 (MREJ ii)</td><td rowspan=1 colspan=1>5.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R6 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R7 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-03</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>26/26b</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R9</td><td rowspan=1 colspan=1>1.0 E-01</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R10</td><td rowspan=1 colspan=1>5.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R11</td><td rowspan=1 colspan=1>1.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R12</td><td rowspan=1 colspan=1>1.0 E-03</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>26/27c</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>26/27c</td><td rowspan=1 colspan=1>79/81</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total agreement</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>377/378</td><td rowspan=1 colspan=1>378/378</td><td rowspan=1 colspan=1>376/377</td><td rowspan=1 colspan=1>1131/1133</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=2>% of agreement</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1></td></tr></table>

Eighty (80) genome copy of purified genomic DNA of MREJ genotype ii. specimen was excluded. specimens are thoseobtaind from the repeat testing. However, for the controls, only the initial resuls were considered for the calculations.

Summary of Reproducibility Results by Site across Lots   

<table><tr><td rowspan=1 colspan=1>Specimen ID</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Sponsor</td><td rowspan=1 colspan=1>CCF</td><td rowspan=1 colspan=1>JHH</td><td rowspan=1 colspan=1>Totalagreement</td><td rowspan=1 colspan=1>Total %agreement</td></tr><tr><td rowspan=1 colspan=1>R1 (negative)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R2 (negative)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R3 (MREJ ii)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R4 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-01</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R5 (MREJ ii)</td><td rowspan=1 colspan=1>5.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R6 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R7 (MREJ ii)</td><td rowspan=1 colspan=1>1.0 E-03</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>26/26b</td><td rowspan=1 colspan=1>80/80</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R9</td><td rowspan=1 colspan=1>1.0 E-01</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R10</td><td rowspan=1 colspan=1>5.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R11</td><td rowspan=1 colspan=1>1.0 E-02</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>R12</td><td rowspan=1 colspan=1>1.0 E-03</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>25/27c</td><td rowspan=1 colspan=1>79/81</td><td rowspan=1 colspan=1>97.5%</td></tr><tr><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>81/81</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=2>Total agreement</td><td rowspan=1 colspan=1>378/378</td><td rowspan=1 colspan=1>378/378</td><td rowspan=1 colspan=1>375/377</td><td rowspan=1 colspan=1>1131/1133</td><td rowspan=1 colspan=1>99.8%</td></tr><tr><td rowspan=1 colspan=2>% of agreement</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1></td></tr></table>

Eighty (80) genome copy of purified genomic DNA of MREJ genotype ii p a al al obtained, the specimen was excluded. considered for the calculations.

b. Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Assay controls were performed satisfactorily at each site demonstrating acceptable QC results for the majority of runs. External specimen processing controls (S. aureus ATCC 43300 for MRSA, and S. aureus ATCC 25923) were also performed. Data demonstrated acceptable QC results for the majority of runs.

Additional positive control strains that represent MRSA MERJ iii and MERJ vii were recommended to test probes and primers that were not controlled directly in the assay.

# Assay Stability

Real-time and accelerated stability studies are ongoing for the BD GeneOhmTM StaphSR Assay when stored under $5 { \pm } 3 \mathrm { { ^ \circ C } }$ , $2 5 { \pm } 2 ^ { \circ } \mathrm { C }$ , $3 5 { \pm } 2 ^ { \circ } \mathrm { C }$ and $4 5 { \pm } 2 ^ { \circ } \mathrm { C }$ . Master Mix and Control DNA were evaluated at predefined time point intervals. Additionally, assay component pouches being stored at $5 { \pm } 3 \mathrm { { ^ \circ C } }$ that have been opened and resealed to mimic use were tested at specific predefined time points. The sealed pouches were stable for 12 weeks at $3 5 { \pm } 2 ^ { \circ } \mathrm { C }$ and $4 5 { \pm } 2 ^ { \circ } \mathrm { C }$ under accelerated storage conditions. The opened pouches were stable at $5 { \pm } 2 \mathrm { { ^ \circ C } }$ for four weeks.

The shelf life for the lysis kit (beads) was twelve months at $2 5 { \pm } 2 ^ { \circ } \mathrm { C }$ or at $5 { \pm } 3 \mathrm { { ^ \circ C } }$ . The Sample buffer was stable for twelve months in closed bags at 21- $2 5 \mathrm { { ^ \circ C } }$ and at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ in opened and closed bags.

# Specimen stability

Viability and amplifiability studies demonstrated that viable bacteria (by colony counts) and amplifiable DNA (by BD GeneOhm™ StaphSR Assay) were recoverable after a storage period of 18 hours for BacT/ALERT media, 30 hours for BACTEC media at $3 5 \mathrm { ^ \circ C }$ , and three days at room temperature $( 1 5 . 2 5 ^ { \circ } \mathrm { C } )$ .

A total of eight lysates (low, medium and high positive lysates), which were freeze-thawed for a minimum of two times, were tested. Results demonstrated that they were stable at $- 2 0 \mathrm { { ^ \circ C } }$ for at least two years.

# d. Detection limit:

# Bacterial Limit of Detection (LOD)

Broth serial dilutions of $1 0 ^ { 2 } , 1 0 ^ { 3 }$ , and $1 0 ^ { 4 }$ of $S .$ . aureus were prepared from TSB culture. Precise viable cell numbers were determined by plating $5 0 ~ \mu \mathrm { L }$ of the bacterial serial dilutions in triplicate on Columbia Blood agar plates for MSSA and on Müeller-Hinton $\mathrm { \ a g a r + 6 m g / L }$ of oxacillin for MRSA. After overnight incubation, the colony count was performed. For each dilution, an average of the three plates was calculated. Each bacterial dilution then was tested using the BD GeneOhmTM StaphSR Assay.

The final MRSA LOD is 9 CFU/reaction. The final bacterial LOD for MSSA is 1 CFU/reaction. The sponsor has chosen to claim a bacterial LOD of 10 CFU/reaction for both MRSA and MSSA.

# DNA LOD

Seven strains of MRSA and two strains of MSSA were plated on sheep blood agar and genomic DNA was extracted from the pure cultures. Each genomic DNA dilution then was tested. Results indicate a final LOD of 15 DNA copies/reaction for MRSA and a final LOD of 5 DNA copies/reaction for MSSA. However, the sponsor has chosen to claim a genomic LOD of 15 DNA copies/reaction for both MRSA and MSSA.

LOD values for the BD GeneOhm™ StaphSR™ Assay   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>BD GeneOhm™StaphSR_FAM channel</td><td rowspan=1 colspan=2>BD GeneOhm™ StaphSR TxR channel</td></tr><tr><td rowspan=1 colspan=1>cfu/reaction</td><td rowspan=1 colspan=1>DNA copies/reaction</td><td rowspan=1 colspan=1>cfu/reaction</td><td rowspan=1 colspan=1>DNA copies/reaction</td></tr><tr><td rowspan=1 colspan=1>type i</td><td rowspan=1 colspan=1>2.65</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type ii</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type ii mut16</td><td rowspan=1 colspan=1>6.58</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type iii</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type iv</td><td rowspan=1 colspan=1>2.69</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type v</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>type vii</td><td rowspan=1 colspan=1>8.52</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>SA 29213</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>SA 25923</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Final LOD range</td><td rowspan=1 colspan=1>0.35 to 8.52</td><td rowspan=1 colspan=1>2.5 to 15</td><td rowspan=1 colspan=1>0.46 to 1.17</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Mean LOD value</td><td rowspan=1 colspan=1>3.35</td><td rowspan=1 colspan=1>8.21</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>5</td></tr></table>

The High Target Concentration Interference study demonstrated that the presence of MSSA at a 500:1 (MSSA: MRSA) or 1000:1 ratio, the LOD for MREJ type ii mut16 and type iii was 30 copies/reaction instead of 15 copies/reaction.

Additionally, 100 MRSA (MREJ genotypes i, ii, iii, iv, v, vii) strains (representing 29 countries) and 199 MSSA strains (representing 18 countries) from well characterized clinical isolates or public collections were evaluated.

# e. Analytical specificity:

# Cross- reactivity Study:

Purified DNA or culture lysates from 99 strains representing 33 coagulasenegative Staphylococcus strains (representing 25 species), 66 nonStaphylococcal species, and human DNA were tested. The nonStaphylococcal species were from well characterized clinical isolates or public collections. They were commonly isolated bacterial pathogens (aerobes, anaerobes, Helicobacter spp, and Bordetella pertusis), yeast, Actinomyces spp, Chlamydia trachomatis, Mycobacterium gordonae and normal human skin/mucus flora.

All DNA were diluted with sample buffer to obtain approximately 1 ng $3 \mathrm { x } 1 0 ^ { 5 }$ CFU) of purified genomic DNA per PCR reaction. All assay tubes were analyzed by agarose gel electrophoresis to further assess if an amplicon had been produced.

The specificity with closely related organisms was $5 9 5 \%$

# f. Assay cut-off:

A multi-site investigational study was conducted on 701 (156 MRSA, 159 MSSA, 386 non S. aureus) positive blood cultures to establish the acceptance criteria. A total of 414 PC and $4 1 4 \mathrm { N C }$ were also conducted in the same

investigational study.

The following table demonstrates the acceptance criteria to distinguish a positive result from a negative result which is required for the interpretation of data.

Acceptance Criteria for BD GeneOhm™ StaphSR Assay   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>ClInical 8peclmens</td><td rowspan=1 colspan=2>Pooittive control</td><td rowspan=1 colspan=3>x Negative control</td></tr><tr><td rowspan=1 colspan=1>Acceptance criteria</td><td rowspan=1 colspan=1>As83y (FAM)</td><td rowspan=1 colspan=1>Assay (TxRd)</td><td rowspan=1 colspan=1>Internal control</td><td rowspan=1 colspan=1>A88ay (FAM)</td><td rowspan=1 colspan=1>Assay (TxRd)</td><td rowspan=1 colspan=1>A88ay (FAM)</td><td rowspan=1 colspan=1>As8ay (TxRd)</td><td rowspan=1 colspan=1>Internal control</td></tr><tr><td rowspan=1 colspan=1>Endpolnt threshold</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>2&quot;d darivative threehold</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>MinImum cycle threshold</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>MaxImum cycle threshold</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>% endpolnt</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

No: An assay Ct value between 15 and 45 ced with assay EP and SDP excding the speifd values invalidate a negative control.

2. Comparison studies:

# a. Method comparison with predicate device:

Culture was performed on positive blood bottles presenting bacterial growth of Gram positive cocci within 36 hours of being declared positive by the instrument. It consisted of an initial analysis on a blood agar plate after an overnight incubation. Presumptive colonies of S. aureus were confirmed with either a latex agglutination assay, tube coagulase testing, or an automated identification system. Confirmed colonies were tested for methicillin resistance using oxacillin screen agar, PBP2’ latex agglutination test, or an automated antimicrobial susceptibility testing system using appropriate panels.

Different testing algorithms were used at each site and percent agreement was used for data analysis.

$b$ . Matrix comparison: Not Applicable

3. Clinical studies:

A total of 1183 compliant positive blood bottles at five sites (three US and two Canadian) were tested for SA and MRSA with the standard culture method and the BD GeneOhm™ StaphSR Assay. Specimens were tested with the culture technique within 36 hours and with the StaphSR Assay within 72 hours of positivity of the blood bottle. The tables below demonstrated the performance of the assay.

The overall $\%$ agreement was $59 5 \%$ .

Performance obtained with BD GeneOhm™ StaphSR for MRSA (by investigational site) in comparison to the reference method.   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MRSA prevalence</td><td rowspan=1 colspan=1>MRSA Positive% Agreement(95% CI) 1</td><td rowspan=1 colspan=1>MRSA Negative% Agreement(95% CI) 1</td><td rowspan=1 colspan=1>Overall% Agreement</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>13.7% (61/446)</td><td rowspan=1 colspan=1>100%(61/61) (94.1%-100%)</td><td rowspan=1 colspan=1>98.7% (380/385) (97.0% - 99.6%)</td><td rowspan=1 colspan=1>98.9%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>18.0% (24/133)</td><td rowspan=1 colspan=1>100%(24/24) (85.8%-100%)</td><td rowspan=1 colspan=1>98.2% (107/109) (93.5% - 99.8%)</td><td rowspan=1 colspan=1>98.5%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9.1% (21/232)</td><td rowspan=1 colspan=1>100%(21/21) (83.9%-100%)</td><td rowspan=1 colspan=1>100.0% (211/211) (98.3% - 100.0%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>16.8% (48/286)</td><td rowspan=1 colspan=1>100% (48/48)(92.6%-100%)</td><td rowspan=1 colspan=1>98.3% (234/238) (95.8% - 99.5%)</td><td rowspan=1 colspan=1>98.6%</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>2.3% (2/86)</td><td rowspan=1 colspan=1>100%(2/2) (15.8%-100%)</td><td rowspan=1 colspan=1>100.0% (84/84) (95.7% - 100.0%)</td><td rowspan=1 colspan=1>100%</td></tr></table>

Binomial $9 5 \%$ exact confidence intervals.

Performance obtained with BD GeneOhm™ StaphSR for S. aureus (by investigational site) in comparison to the reference method.   

<table><tr><td rowspan=1 colspan=1>Investigatioal site</td><td rowspan=1 colspan=1>S. aureusprevalence</td><td rowspan=1 colspan=1>S. aureus Positive% Agreement(95% CI)1</td><td rowspan=1 colspan=1>S. aureus Negative% Agreement(95% CI) 1</td><td rowspan=1 colspan=1>Overall% Agreement</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>22.2% (99/446)</td><td rowspan=1 colspan=1>100.0% (99/99) (96.3%-100%)</td><td rowspan=1 colspan=1>100% (347/347)(98.9%-100%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>30.1% (40/133)</td><td rowspan=1 colspan=1>100% (40/40)(91.2%-100%)</td><td rowspan=1 colspan=1>98.9% (92/93) (94.2%-100%)</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>35.8% (83/232)</td><td rowspan=1 colspan=1>100% (83/83)( 95.7%-100%)</td><td rowspan=1 colspan=1>100% (149/149) (97.6%-100%)</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>29.7% (85/286)</td><td rowspan=1 colspan=1>98.8% (84/85) (93.6% - 100.0%)</td><td rowspan=1 colspan=1>96.5% (194/201) (93.0% - 98.6%)</td><td rowspan=1 colspan=1>97.2%</td></tr><tr><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>9.3% (8/86)</td><td rowspan=1 colspan=1>100% (8/8) (63.1% - 100.0%)</td><td rowspan=1 colspan=1>100% (78/78) (95.4%-100%)</td><td rowspan=1 colspan=1>100%</td></tr></table>

Binomial $9 5 \%$ exact confidence intervals.

a. Clinical Sensitivity: Not Applicable

$b$ . Clinical specificity: Not Applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Staphylococcus aureus is responsible for up to $2 5 \%$ of blood stream infections, among which, 26- $47 \%$ are caused by MRSA. When encountering prevalences that ranged from $2 . 3 \%$ to $3 5 . 6 \%$ , the assay performance ranged from $9 8 . 8 \%$ to $100 \%$ agreement when compared to the reference method.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirement of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.