# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

C. Measurand: Francisella Tularensis DNA sequences

# D. Type of Test:

DNA amplification with real-time polymerase chain reaction (PCR) in vitro diagnostic detection of target DNA sequences of Francisella tularensis (F. tularensis); qualitative result

# E. Applicant:

Idaho Technology, Inc.

# F. Proprietary and Established Names:

Joint Biological Agent Identification and Diagnostic System (JBAIDS) Tularemia Detection Kit; Real-time PCR amplification and detection system for targeted $F _ { \cdot }$ . tularensis DNA sequences

# G. Regulatory Information:

1. Regulation section: 866.3280

2. Classification: Class II

3. Product code: OEH

83 - Microbiology

# H. Intended Use:

1. Intended use(s):

The Joint Biological Agent Identification and Diagnostic System (JBAIDS) Tularemia Detection Kit is a real-time polymerase chain reaction (PCR) test system intended for the qualitative in vitro diagnostic (IVD) detection of target DNA sequences of Francisella tularensis. The system can be used to test human whole blood collected in sodium citrate or sputum collected aseptically from individuals greater than 18 years of age suspected of having tularemia. In addition, positive blood cultures and colonies may be tested. This assay is intended to aid in diagnosis of individuals presenting with signs and symptoms of pneumonic or typhoidal tularemia. It is not intended to aid in the diagnosis of glandular, ulceroglandular, oculoglandular, or oropharyngeal tularemia.

The JBAIDS Tularemia Detection Kit is run on the JBAIDS instrument using the Diagnostic Wizard. Results are for the presumptive identification of $F .$ . tularensis, in conjunction with culture and other laboratory tests.

The definitive identification of $F .$ . tularensis from colony growth, liquid blood culture, blood specimens, or sputum specimens requires additional testing and confirmation procedures in consultation with public health or other authorities for whom reports are required.

The diagnosis of tularemia infection must be made based on history, signs, symptoms, exposure likelihood, and other laboratory evidence, in addition to the identification of the target either from colonies, blood culture, whole blood specimens, or sputum specimens.

The JBAIDS Tularemia Detection Kit is intended for use by trained clinical laboratory personnel who have received specific training on the use of the JBAIDS Tularemia Detection kit. The level of F. tularensis that would be present in blood or sputum of individuals with early systemic or pneumonic infection is unknown. Due to the difficulty in obtaining clinical specimens, the assay was not evaluated with blood or sputum from individuals presenting with signs and symptoms of tularemia and who subsequently developed pneumonic or typhoidal tularemia.

# 2. Indication(s) for use:

The Joint Biological Agent Identification and Diagnostic System (JBAIDS) Tularemia Detection Kit is a real-time polymerase chain reaction (PCR) test system intended for the qualitative in vitro diagnostic (IVD) detection of target DNA sequences of Francisella tularensis. The system can be used to test human whole blood collected in sodium citrate or sputum collected aseptically from individuals greater than 18 years of age suspected of having tularemia. In addition, positive blood cultures and colonies may be tested. This assay is intended to aid in diagnosis of individuals presenting with signs and symptoms of pneumonic or typhoidal tularemia. It is not intended to aid in the diagnosis of glandular, ulceroglandular, oculoglandular, or oropharyngeal tularemia.

The JBAIDS Tularemia Detection Kit is run on the JBAIDS instrument using the Diagnostic Wizard. Results are for the presumptive identification of $F .$ . tularensis, in conjunction with culture and other laboratory tests.

The definitive identification of $F .$ . tularensis from colony growth, liquid blood culture, blood specimens, or sputum specimens requires additional testing and confirmation procedures in consultation with public health or other authorities for whom reports are required.

The diagnosis of tularemia infection must be made based on history, signs, symptoms, exposure likelihood, and other laboratory evidence, in addition to the identification of the target either from colonies, blood culture, whole blood specimens, or sputum specimens.

# 3. Special conditions for use statement(s):

The JBAIDS Tularemia Detection Kit is intended for use by trained clinical laboratory personnel who have received specific training on the use of the JBAIDS Tularemia Detection kit. The level of $F _ { \cdot }$ . tularensis that would be present in blood or sputum of individuals with early systemic or pneumonic infection is unknown. Due to the difficulty in obtaining clinical specimens, the assay was not evaluated with blood or sputum from individuals presenting with signs and symptoms of tularemia and who subsequently developed pneumonic or typhoidal tularemia.

4. Special instrument requirements: Not applicable.

# I. Device Description:

The JBAIDS instrument was developed by Idaho Technology to serve as a common platform for the rapid detection of $F _ { \cdot }$ . tularensis. The JBAIDS instrument and software were initially cleared for use with the JBAIDS Anthrax Detection Kit in the 510(k) submission K051713. The JBAIDS Tularemia Detection System is composed of the JBAIDS instrument and specific freeze-dried reagents for the detection of a gene target of the $F .$ . tularensis.

The JBAIDS Tularemia Detection Kit is specially designed for performing real-time PCR in glass capillaries using the JBAIDS instrument and JBAIDS software. Real-time PCR is accomplished with the use of a hydrolysis probe, which detects a specific sequence within the region amplified by the assay primers. The probe is labeled on one end with a fluorescent reporter molecule (6-FAM) and on the other end with a quencher (TAMRA). When the probe is intact, the quencher absorbs the light emitted by the reporter molecule.

During PCR, the intact probe hybridizes to the amplicon, and the exonuclease activity of Taq polymerase separates the fluorophore from the quencher, generating a fluorescent signal that is detected by the instrument. The fluorescent signal increases as additional template is amplified and more probe is hydrolyzed.

The primers and probes were selected based upon two basic criteria: 1) they detect all sequenced virulent isolates of $F .$ . tularensis, and 2) they do not detect sequences in other organisms.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

There are two predicate devices referenced to be substantially equivalent to the JBAIDS Tularemia Detection Kit.

SAS Francisella tularensis Antigens SAS Francisella tularensis Antiserum

2. Predicate 510(k) number(s):

• K952138   
• K952141

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceJBAIDS TularemiaDetection Kit</td><td rowspan=1 colspan=1>PredicateF. tularensis antigens(K952138) and antiserum(K952141) foragglutination tests</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative detection of F.tularensis DNA.</td><td rowspan=1 colspan=1>Qualitative and semi-quantitative detection ofantibodies to F. tularensis.</td></tr><tr><td rowspan=1 colspan=1>Indications for Use</td><td rowspan=1 colspan=1>Identification of F.tularensis individualssuspected of havingtyphoidal or pneumonictularemia.</td><td rowspan=1 colspan=1>Identification of F.tularensis antibodies inindividuals suspected ofhaving tularemia.</td></tr></table>

<table><tr><td>Differences</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>DeviceJBAIDS TularemiaDetection Kit</td><td rowspan=1 colspan=1>PredicateF. tularensis antigens(K952138) and antiserum(K952141) foragglutination tests</td></tr><tr><td rowspan=1 colspan=1>Technological Principles</td><td rowspan=1 colspan=1>Real-time PCR usinghydrolysis probes.</td><td rowspan=1 colspan=1>Agglutination of antibodiesdirected at F. tularensis.</td></tr><tr><td rowspan=1 colspan=1>Assay Target</td><td rowspan=1 colspan=1>DNA sequences unique toF. tularensis.</td><td rowspan=1 colspan=1>Antibodies against F.tularensis.</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Whole blood (collected in3.2% sodium citrate),sputum, blood culture, orbacterial colonies.</td><td rowspan=1 colspan=1>Serum obtained fromwhole blood.</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>JBAIDS instrument(K051713)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Time to Result</td><td rowspan=1 colspan=1>Less than 3 hours.</td><td rowspan=1 colspan=1>Less than 1 hr for slide-based test followed by lessthan 1 day for tube-basedtest.</td></tr><tr><td rowspan=1 colspan=1>Test Interpretation</td><td rowspan=1 colspan=1>Automated testinterpretation and reportgeneration.</td><td rowspan=1 colspan=1>Subjective interpretation byuser.</td></tr><tr><td rowspan=1 colspan=1>Physical Properties</td><td rowspan=1 colspan=1>Freeze dried reagents withreconstitution buffer andwater provided in kit.</td><td rowspan=1 colspan=1>Liquid reagents.</td></tr><tr><td rowspan=1 colspan=1>Storage and Shelf Life</td><td rowspan=1 colspan=1>1 year at room temperature(18-28°C).</td><td rowspan=1 colspan=1>Refrigerator temperature(2-8°C), manufacturerdefined expiration date.</td></tr></table>

The predicate device and the JBAIDS Tularemia Detection Kit have the same intended use. Both provide test results that aid in the diagnosis of tularemia when considered with other clinical and microbiological evidence, both test for $F .$ . tularensis infection directly from patient specimens, and both provide qualitative test results. While the basic intended use is the same for the predicate device and the JBAIDS Kit, the technological characteristics are quite different. The collected specimens were tested for $F .$ . tularensis using the JBAIDS Tularemia Detection Kit and the results compared to the gold standard culture method.

# K. Standard/Guidance Document Referenced (if applicable):

This 510(k) Premarket Notification was prepared and referenced the following guidance documents and recognized standards:

Nucleic Acid Based in Vitro Diagnostic Devices for Detection of Microbial Pathogens, FDA Guidance Document (DRAFT final: December 8, 2005)

User Protocol for Evaluation of Qualitative Test Performance, Clinical Laboratory   
Standards Institute (CLSI) Approved Guideline, EP12-A (August 2002)   
Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline,   
MM3-A (December 1995)   
Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A   
(December 2002)

# L. Test Principle:

The JBAIDS Tularemia Detection Kit uses real-time PCR with probes to detect $F$ . tularensis DNA in patient whole blood and sputum samples as well as positive blood cultures and colonies. The probe detects a specific sequence within the region amplified by the assay primers. The probe is labeled on one end with a fluorescent reporter molecule (6-FAM) and on the other end with a quencher (TAMRA). When the probe is intact, the quencher absorbs the light emitted by the reporter molecule. During PCR, the intact probe hybridizes to the amplicon, and the exonuclease activity of Taq polymerase separates the fluorophore from the quencher, generating a fluorescent signal that is detected by the instrument. The fluorescent signal increases as additional template is amplified and more probe is hydrolyzed. Temperature and fluorescence data are displayed in real-time. Each JBAIDS test is analyzed and assigned a final result by the Detector module of the JBAIDS software. Possible final results are positive, negative, inhibited, uncertain, or invalid. To assign a final test result, Detector first analyzes the data from each capillary independently before analyzing the sample duplicates together. Finally, the software assigns a final result, or a combined call, based on the results of the sample and all of its controls. Detector calls are characterized as follows:

Negative – A curve is called negative if its shape can be closely approximated by a line (or smooth curve), if it has low signal to noise, and if it shows little or no increase in fluorescence during the PCR. Positive – A curve is called positive if the fluorescence shows exponential growth out of the background, if it has high signal to noise, and if the slope of the exponential region of the amplification is consistent with a PCR efficiency of around two. Uncertain – For a small number of curves (about $0 . 2 \%$ ) no clear call is possible and the reaction is called uncertain.

Once each amplification curve is evaluated by Detector, the software evaluates the sample duplicates according to the following rules: only if both capillaries are called positive.

The final test result is a combined call (or meta-call) based on the results of the test sample and its associated controls. Failure of the Positive or Negative Control results in invalid results for all associated samples. Inhibition Controls with an inhibited result are further analyzed before a final result is assigned. An inhibited result can indicate that the patient sample contains substances capable of inhibiting the PCR reaction or that

competition for the primers has caused a failure of the Inhibition Control. Samples with inhibited, invalid, or uncertain results require follow-up testing in order to achieve a valid (positive or negative) result.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

Simulated test specimens were prepared by spiking the appropriate matrix with inactivated $F .$ . tularensis to achieve the desired test level concentrations. The table below shows the concentration of inactivated $F .$ . tularensis cells in each test level (for each matrix), the relationship between the test concentration and the limit of detection (LOD) for each matrix, and the number of replicate specimens assessed for each test level. All spiking was performed at Idaho Technology Inc. (ITI). Aliquots of each specimen were tested at three clinical laboratory test sites in different geographical locations. . Each site tested each specimen once on each of three days. This constituted the between-day and between-site evaluations. ITI tested each specimen twice on the first day, constituting the within-day (between-run) evaluation. Withinrun testing was also performed at ITI by testing an individual purified sample, representing each test level, seven times within the same JBAIDS run.

Specimen Panels Prepared for Reproducibility Testing   

<table><tr><td rowspan=2 colspan=1>Testlevel</td><td rowspan=1 colspan=3>Whole blood</td><td rowspan=1 colspan=3>Sputum</td><td rowspan=1 colspan=3>Blood culture</td></tr><tr><td rowspan=1 colspan=1>F.tularensisconc.(CFU/mL)</td><td rowspan=1 colspan=1>LODEquivalent</td><td rowspan=1 colspan=1>Number ofreplicates</td><td rowspan=1 colspan=1>F. tularensisconc.(CFU/mL)</td><td rowspan=1 colspan=1>LODEquivalent</td><td rowspan=1 colspan=1>Numberofreplicates</td><td rowspan=1 colspan=1>F.tularensisconc.(CFU/mL)</td><td rowspan=1 colspan=1>LODEquivalent</td><td rowspan=1 colspan=1>Number ofreplicates</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>2 X</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>2.5 X</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>17X</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>25,000</td><td rowspan=1 colspan=1>12.5 X</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>100,000</td><td rowspan=1 colspan=1>333 X</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100,000</td><td rowspan=1 colspan=1>50 X</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5,000,000</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3</td></tr></table>

To assess within-run reproducibility, one purified sample representing each of the concentration levels shown in the table above was tested seven times in one JBAIDS run. Within-run reproducibility was $100 \%$ for all matrices at all test levels. Additionally, the precision was high for all matrices when testing at the medium and high test levels, with a standard deviation (SD) of the $\mathrm { C p }$ that was less than 0.22 cycles. Within-day reproducibility was assessed by testing each matrix-specific specimen panel twice within one day. For whole blood, the within-day reproducibility was $9 6 . 4 \%$ (27 correct out of 28 total tests). For both sputum and blood culture, within-day reproducibility was $100 \%$ .

Between-day reproducibility was assessed by testing each matrix-specific specimen panel once each day over 3 separate days. Between-site reproducibility was assessed by performing the between-day evaluation at three test sites. The overall betweensite reproducibility was $9 9 . 2 \%$ for whole blood, $9 8 . 9 \%$ for sputum, and $100 \%$ for blood culture.

While the JBAIDS Tularemia Detection Kit is designed to provide qualitative results, an evaluation of precision was performed to estimate the overall variability of the test system. Testing precision was determined by analyzing the overall variation of the Cp values, represented by the standard deviation (SD). For whole blood and blood culture, the medium and high test levels had minimal variation, with the SD of the $\mathrm { C p }$ being less than one cycle. For sputum, the variability was higher than seen for the other matrices.

The Reproducibility and Precision studies were satisfactory for this type of assay.

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Positive, Negative and Inhibition controls were included in each run and gave expected results. If not, the runs failed or were called invalid and the run needed to be repeated.

# d. Detection limit:

Initial estimates for setting the Limitations of Detection (LOD) were performed by spiking whole blood or sputum specimens with a serial dilution of live $F .$ . tularensis. Whole blood specimens collected in $3 . 2 \%$ sodium citrate were spiked between 100 and $1 0 0 0 \mathrm { C F U / m L }$ . Sputum specimens were spiked between 330 and $3 3 3 0 \mathrm { C F U / m L }$ . Each specimen was purified using the appropriate sample purification kit, IT 1-2-3 QFLOWdna for whole blood and IT 1-2-3 VIBE for sputum specimens and the purified samples were tested with the $F$ . tularensis assay following standard procedures.

Based on the titration results, the LOD was estimated to be $3 0 0 \mathrm { C F U / m L }$ for whole blood and 2000 CFU/mL for sputum. Reactions were evaluated using the JBAIDS test result, the Cp, and a visual examination of the amplification curves. To confirm these LOD levels, 59 whole blood specimens were spiked at $3 0 0 \mathrm { C F U / m L }$ and 59 sputum specimens were spiked at $2 0 0 0 \mathrm { C F U / m L }$ with live $F .$ . tularensis. The specimens were processed and tested, and the frequency of positive results was determined. All 59 spiked whole blood specimens were successfully detected by the JBAIDS Tularemia Detection Kit, indicating that whole blood specimens containing organism concentrations as low as $3 0 0 \mathrm { C F U / m L }$ can be detected with $100 \%$ success ( $9 5 \%$ CI, $9 5 \% - 1 0 0 \%$ ). Fifty-seven of the 59 spiked sputum specimens were successfully detected, indicating that sputum specimens containing $F _ { \cdot }$ . tularensis at 2000 CFU/mL can be detected with $9 6 . 6 \%$ success ( $9 5 \%$ CI, $8 9 . 7 \mathrm { - } 1 0 0 \%$ ).

e. Analytical specificity:

The analytic inclusivity and exclusivity of the JBAIDS Tularemia Detection Kit was assessed by testing panels of well-characterized organisms. The panels were comprised of (1) strains of $F .$ . tularensis representing the known genetic and geographic distribution of the organism, (2) closely related organisms, and (3) unrelated organisms typically found in clinical specimens (refer to tables below). $F _ { \cdot }$ . tularensis strains were tested using colonies grown on culture plates as well as with purified DNA at a level equivalent to seven times the established nucleic acid LOD of the reference strain, $F .$ . tularensis Schu4. All non- $F _ { \cdot }$ . tularensis organisms were tested using only colonies grown on culture plates.

F. tularensis Inclusivity Test Panel   

<table><tr><td rowspan=1 colspan=1>Strain Designation</td><td rowspan=1 colspan=1>F. tularensis Strain and Type</td></tr><tr><td rowspan=1 colspan=1>Reference Strain</td><td rowspan=1 colspan=1>Subsp. Tularensis (SCHU4), Type A1</td></tr><tr><td rowspan=1 colspan=1>FRAN003 (UCC)</td><td rowspan=1 colspan=1>Subsp. novicida</td></tr><tr><td rowspan=1 colspan=1>FRAN004 (UCC)</td><td rowspan=1 colspan=1>Subsp. holarctica (LVS), Type B</td></tr><tr><td rowspan=1 colspan=1>FRAN012 (UCC)</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>FRAN016 (UCC)</td><td rowspan=1 colspan=1>Subsp. Tularensis (SCHU4), Type A1</td></tr><tr><td rowspan=1 colspan=1>FRAN029 (UCC)</td><td rowspan=1 colspan=1>Subsp. holarctica (425), Type B</td></tr><tr><td rowspan=1 colspan=1>FRAN007 (UCC)</td><td rowspan=1 colspan=1>Subsp. tularensis (DS88-R-160), Type A</td></tr><tr><td rowspan=1 colspan=1>FRAN008 (UCC)</td><td rowspan=1 colspan=1>Subsp. tularensis (DS87-R-135), Type A</td></tr><tr><td rowspan=1 colspan=1>NE-UNMC061598</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>NE-UNL091902 (B017)</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>WY-WSVL01 (B018)</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>WY-9868529 (BO19)</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>WY-00W4114 (BO20)</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>WY-96194280 (BO21)</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>WY-WSVL02 (BO22)</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>OK-00101504</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>OK-98041035 (BO23)</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>MS-304</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>NC-54558-01</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>NC-52797-99</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>NC-54559-01</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>Rabbit, no code</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>MO MS1349</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>AFIOH Feline</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>MO, no code, human</td><td rowspan=1 colspan=1>Subsp. holarctica, Type B</td></tr><tr><td rowspan=1 colspan=1>NE-BLGH100903</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>NC-48620-97</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr><tr><td rowspan=1 colspan=1>AFIOH UTPH 2173</td><td rowspan=1 colspan=1>Subsp. tularensis, Type A</td></tr></table>

# Exclusivity Panel

<table><tr><td>Organism</td></tr><tr><td>Francisella philomiragia (FOPA +, FTUL -)</td></tr><tr><td>Moraxella catarrhalis</td></tr><tr><td>Neisseria lactamica</td></tr><tr><td colspan="1" rowspan="1">Pasteurella multocida</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td></tr><tr><td colspan="1" rowspan="1">Enterobacter agglomerans</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Salmonella enteritidis</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Y. enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Y. enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Yersinia frederiksenii</td></tr><tr><td colspan="1" rowspan="1">Yersinia kristensenii</td></tr><tr><td colspan="1" rowspan="1">Y. kristensenii</td></tr><tr><td colspan="1" rowspan="1">Yersinia pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Y. pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Y. pseudotuberculosis</td></tr><tr><td colspan="1" rowspan="1">Yersinia ruckeri</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica serovar Typhi 14028</td></tr></table>

For all colonies evaluated, the final test result was compared to the expected test result based on microbiological identification methods, and the Cp value for positive samples was evaluated. Organisms in the inclusivity panel were expected to test positive and have Cp values around 20, while organisms in the exclusivity panel were expected to test negative. For the inclusivity panel testing using purified DNA samples, the test result and Cp value for each sample were evaluated. The LOD for each $F .$ . tularensis strain was considered equivalent to the reference strain, $F .$ . tularensis, if the test result was positive and the average Cp value for the strain was within 1.5 cycles of the average Cp value obtained for the reference strain.

When testing colonies, all 27 strains of $F .$ . tularensis were correctly identified by the JBAIDS Tularemia Detection Kit. The average $\mathrm { C p }$ values obtained for the different strains ranged from 19.18 to 24.69. In addition, all 27 strains gave the expected results. The probabilities ranged from $90 \%$ to $100 \%$ , indicating high confidence in the reference results.

Of the 24 organisms in the exclusivity panel, 23 gave the expected negative result. $F .$ . philomiragia tested positive with the JBAIDS Tularemia assay. Two additional isolates of $F .$ . philomiragia were tested and gave similar results. This result was not unexpected given the sequence similarities between $F .$ . tularensis and $F$ . philomiragia. The average Cp value when testing $F .$ . philomiragia was approximately 21 cycles later than the average Cp value obtained when testing $F .$ . tularensis colonies (42.5 compared to 21.2), indicating that the cross-reactivity is weak.

# Interfering Substances

The $F _ { \cdot }$ . tularensis assay was tested against a panel of potentially interfering substances. The concentration of each substance tested represented a relevant concentration in accordance with CLSI EP7-A Interference Testing in Clinical Chemistry Approved Guideline. All endogenous and exogenous substances typically found in whole blood, sputum, blood cultures, or colonies were spiked into the sample matrix and subjected to sample processing with the appropriate IT 1-2-3 Sample Purification Kit prior to testing with the JBAIDS Tularemia Detection Kit. The sample purification portion of the system is designed to isolate DNA and remove other impurities; therefore, sample processing should eliminate most potentially interfering substances from the purified sample. Because technique-specific substances can be encountered during reaction setup, all such substances were added to the purified samples immediately prior to reaction setup.

Once prepared, each sample was tested using an Inhibition Control vial and an Unknown vial. The substance was considered to be an interfering substance if the target assay was called negative or inhibited by the JBAIDS Software. A substance was considered to be potentially inhibitory if the Cp of the target assay or the IC assay was delayed by more than three cycles or if the Fmax of either assay was decreased by more than $50 \%$ compared to samples with no substance added.

The table below lists the potentially interfering substances that were tested.

List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>EndogenousSubstances</td><td rowspan=1 colspan=3>Exogenous Substances</td></tr><tr><td rowspan=1 colspan=1>HemoglobinAlbuminBilirubinTriglyceridesCholesterol (total)ImmunoglobulinsGlucose</td><td rowspan=1 colspan=1>AcetaminophenAmoxicillinAscorbic acidAspirinCefotaximeChloroquineCiprofloxacinDoxycyclineErythromycinGentamicin sulfate</td><td rowspan=1 colspan=1>IbuprofenNaproxen sodiumRifampinStreptomycinSulfamethoxazoleTetracyclineTobramycinTrimethoprim</td><td rowspan=1 colspan=1>Acid-citrate-dextroseCitrate (sodium)EDTAHeparinSodium polyanethol sulfonate(SPS)Albuterol (Salbutamol)Cromolyn sodiumFlunisolide (Flovent)N-acetylcysteineBlood culture mediaChocolate agar media</td></tr><tr><td rowspan=1 colspan=1>Solvents Used</td><td rowspan=1 colspan=3>Technique-specific Substances</td></tr><tr><td rowspan=1 colspan=1>AcetoneDMSOEthanolNH4OH</td><td rowspan=1 colspan=1>BleachDNAZapT (Ambion)Snap n&#x27; DigestTM (Scientific DeviceLaboratory)</td><td rowspan=1 colspan=1>ITI 1-2-3 kit Buffer 1ITI 1-2-3 kit Buffer1AITI 1-2-3 kit Buffer1BITI 1-2-3 kit Buffer1CITI 1-2-3 kit Buffer 2</td><td rowspan=1 colspan=1>QIAGEN Buffer ALQIAGEN Buffer AW1QIAGEN Buffer AW1 (w/oEtOH)QIAGEN Buffer AW2QIAGEN Buffer AW2 (w/0EtOH)</td></tr></table>

a These are solvents used to dissolve potentially interfering substances in preparation for testing.

Of all the substances tested, only those listed below showed evidence of inhibiting the $F .$ tularensis assay:

• SPS   
• Bleach   
• DNAZap   
• QIAGEN Buffer AL   
• QIAGEN Buffer AW1   
• QIAGEN Buffer AW1 (no ethanol added)   
• ITI Buffer 1A   
• ITI Buffer 1B   
• ITI Buffer 1C   
• ITI Buffer 1

Samples containing SPS have previously been found to inhibit PCR amplification, and, given the test results reported here, blood collection tubes containing this anticoagulant should not be used to collect specimens for use with the JBAIDS Tularemia Detection Kit. While blood culture media contains SPS, no inhibition was seen when testing blood culture samples. The 100-fold dilution included in the blood culture sample purification protocol likely decreases the SPS concentration in the sample below the inhibitory level. All other inhibitory substances were technique specific. Appropriate recommendations will be made in the product insert, Limitations of the Procedure section.

$f .$ Assay cut-off:

A data analysis module within assay-specific software applies mathematical modeling of expected amplification curve shapes to each individual capillary. When fixed thresholds with a quadratic formula or crossing point determination are matched, samples are called negative or positive. For undecided samples, an expert system approach is applied that uses filters to assess the fluorescence change around the crossing point.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison:

The studies were done in sputum, whole blood, cultured isolates and from positive blood culture bottles. Studies showed that there were differences in blood and sputum specimens.

3. Clinical studies:

a. Clinical Sensitivity: This assay was not evaluated with blood or sputum from individuals presenting with signs and symptoms of tularemia and who subsequently developed pneumonic or

typhoidal tularemia because of the low prevalence of tularemia in the US. Users may want to establish the clinical sensitivity of this test on prospectively collected clinical specimens.

# b. Clinical specificity:

A multicenter clinical study was conducted over a 7-month period on 169 patients exhibiting symptoms consistent with systemic tularemia infection. Patients were selected from those who were hospitalized, exhibited the clinical definition of Systemic Inflammatory Response Syndrome, and for whom a blood and/or sputum culture had been ordered. The study subjects ranged in age from 18–94 years (mean 43), and $5 0 . 6 \%$ were male. Blood samples from 149 subjects were collected and tested, while 45 sputum samples were collected and tested. For 25 subjects both blood and sputum specimens were tested.

The study was conducted at two U.S. sites and outside the U.S. The clinical specificity of the JBAIDS Tularemia Detection System was assessed by comparing the JBAIDS results (Positive/Negative) to the results obtained by established culture and microbiology follow-up methods.

Using the standard testing protocol, all 132 whole blood and 36 sputum samples gave the expected negative result with the $F .$ . tularensis assay. In addition, all culture results were negative for $F .$ . tularensis. Therefore, the clinical specificity of the JBAIDS Tularemia Detection Kit is at least $9 7 \%$ $( 9 7 - 1 0 0 \%$ with $9 5 \%$ confidence) when testing whole blood samples and $9 2 \%$ $9 2 \mathrm { - } 1 0 0 \%$ with $9 5 \%$ confidence) when testing sputum samples.

<table><tr><td colspan="8" rowspan="1">Table 6. Summary of Clinical Testing for Whole Blood and Sputum.</td></tr><tr><td colspan="8" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Inhibited</td><td colspan="1" rowspan="1">Uncertain</td><td colspan="1" rowspan="1">Removedfrom Study</td><td colspan="1" rowspan="1">TotalSubjects</td></tr><tr><td colspan="2" rowspan="1">Blood Culture—noorganism</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">129</td></tr><tr><td colspan="2" rowspan="1">Blood Culture—withorganisma</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">20</td></tr><tr><td colspan="1" rowspan="3">All Patients</td><td colspan="1" rowspan="1">Site 1b</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">30d</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="1">Site 2c</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">86c</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">Site 3b</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16f</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="2" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">132</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">149</td></tr><tr><td colspan="2" rowspan="1">Sputum Culture—noorganism</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">31</td></tr><tr><td colspan="2" rowspan="1">Sputum Culture—withorganism8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="3">All Patients</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">26h</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="2" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="8" rowspan="1">List of organisms recovered from blood culture and number: beta-hemolytic Streptococcus (n=1), Brucella sp.(nCoyeasp.=1), cagulasegativStylous (n=1),Eus p.(n=2, Esrichicol ,gram positivecoccus (n=1), Kocuriavarians (n=3), Micrococcus sp. (n=1), Salmonella enteria serovarTyphi (n=),Propionibacterium acnes (n=1), Staphylococcus epidermidis (n=2), Staphylococcus warner (n=1), Streptococcusconstellatus (n=1)Blood culture technique at Sites 1 and 3: BacT/ALERT® 3D using BacT/ALERT SA (aerobic) and BacT/ALERT SN(anaerobic), BACTECTM 9240 Plus held for 5 days at 35°C.Blcul eu us  S.BAC u eoi blculuebottes hel  7or devaluation and subculture.d Three samples required retesting to achieve the final result.wo samples required retesting to achieve the final result.To samples requied retesting to achieve the final esult.  lull  .eie (n=1), Haemophilu influena (n=1), Haemolystreptococcu (n=3), methicilln-reistant Saphyloccuaureus (n=5), Moraxella cataralis (n=1), Pseudomonas aeruginosa (n=1), Serratia marcescens (n=3), Staphylococcusepidermidis (n=1)   failed.</td></tr></table>

c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: Not applicable, qualitative test.

N. Instrument Name: JBAIDS instrument O. System Descriptions:

1. Modes of Operation:

All sample testing is managed by the software embedded Diagnostic Wizard; up to 32 capillaries can be loaded into one carousel for a run.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

Specimen information is a pre-defined protocol that guides a user through entering sample information, loading the JBAIDS carousel, and starting the testing sequence. Data, result call displays and generation of final reports is managed through the traceable database using the Diagnostic Wizard. Capillaries are identified by their position number in the carousel.

4. Specimen Sampling and Handling:

Blood specimens must be collected in tubes with sodium citrate anticoagulant. Other anticoagulants are not indicated and may contribute to unreliable results. DNA is initially extracted and purified from whole blood specimens using the Idaho Technology IT 1-2- 3TM FLOW Sample Purification Kit (or validated equivalent), and from blood culture and direct culture samples using the Idaho Technology IT 1-2-3TM SWIPE Sample Purification Kit (or validated equivalent). Purified samples must be diluted prior to adding to reagent vials.

5. Calibration:

The fluorimeter is factory-calibrated. Internal self-check procedures are run with each startup. Cycling temperatures are monitored continuously during a run. The internal control and positive control must meet specific criteria for a successful run.

6. Quality Control: Negative Control

The Negative Control (NC) is used to detect contamination from target-specific amplicon, genomic DNA {as found in the Positive Control (PC) vials}, or organism. One NC (resulting in two capillaries) must be included for each assay in a test run. Both of the NC capillaries must be Negative, or the JBAIDS software will assign Invalid results to all of the associated samples, and the run must be repeated. Frequent or repeated failures of NCs may indicate significant contamination of the work area.

# Positive Control

The PC serves as an amplification and detection control. One PC (resulting in two capillaries) must be included for each assay in a test run. Both of the PC capillaries must be Positive and have Cp results that are earlier than the assay’s specific cutoff value. If either capillary fails, the JBAIDS software will assign Invalid results to all of the associated samples, and the run must be repeated. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument. If the IC capillaries in the same test run are Positive, then the failure is most likely caused by an isolated error with the setup of the PC. If the IC capillaries are also Negative, possible causes for failure are 1) a systematic error in sample setup, 2) degradation of the reagents, or 3) a malfunction of the JBAIDS instrument.

# Inhibition Control

The IC is used to demonstrate that the purified sample does not interfere with, or inhibit, the PCR reaction. Each purified sample is tested using an Unknown vial and IC vial (for a total of four capillaries). If either of the IC amplification curves demonstrates inhibition and the target assay is negative, then the result for that sample will be Inhibited. The IC amplification curves are called Inhibited by the JBAIDS software if 1) the $\mathrm { C p }$ value for the IC exceeds the assay specific cutoff value or 2) if the shape of the amplification curve demonstrates reduced PCR efficiency, as evidenced by a flattening of the amplification curve.

# DNA Extraction Control

Well characterized strains of F. tularensis and non F. tularensis may be used as external NCs and PCs according to guidelines or requirements of local, state, and/or federal regulations or accrediting organizations. However, due to safety and regulatory requirements related to the distribution of F. tularensis and other select agents, this is not feasible in most user settings. To address the lack of appropriate control materials, Idaho Technology has developed a DNA Extraction Control Kit that can be used to ensure the proper functioning of the sample purification kits and procedures. The DNA Extraction Control kit is designed to work with all of Idaho Technology’s 1-2-3 Sample Purification Kits and can be used for both environmental and diagnostic applications. This kit can also be used for training or for verification of user technique.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Carryover and Cross-contamination

Sets of whole blood, sputum and blood culture specimens were spiked with inactivated $F$ tularensis at approximately $5 \mathrm { ~ x ~ } 1 0 6 \mathrm { C F U / m L }$ . This concentration of organism yields a very strong positive signal (an early $\mathrm { C p }$ with a consistent Fmax) when tested with the $F _ { \cdot }$ . tularensis target assay. For each matrix evaluated, three different operators tested three independent sample sets. The sample sets, consisting of seven negative samples and seven positive samples, were processed using the appropriate IT 1-2-3 Sample

Purification Kit. The purified samples were subsequently tested using the freeze-dried $F$ tularensis assay on the JBAIDS instrument.

For all three matrices, all negative reactions tested negative, indicating that no detectable carryover occurred during either sample purification or reaction set-up. Additionally, all positive samples tested positive with $\mathrm { C p }$ values indicative of high organism concentrations (whole blood, $2 4 . 2 3 { \scriptstyle \pm 0 . 6 6 }$ ; sputum, $2 3 . 7 6 { \pm } 0 . 3 9$ ; blood culture, $2 6 . 6 8 { \scriptstyle \pm 0 . 4 2 } )$ .

# Sample Transport and Storage

The evaluations were performed using simulated samples prepared by spiking live $F .$ . tularensis into whole blood and sputum specimens. For the assessment of colonies and blood-culture specimens, a panel of $F .$ . tularensis and non- $F .$ . tularensis isolates was used for the positive and negative samples, respectively.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The information submitted in this premarket notification is complete and supports a substantial equivalence decision.