# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

# D. Type of Test:

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay   
2. Classification: Class II   
3. Product code: OCC   
4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The ProFlu $+ \mathbf { T M }$ Assay is a multiplex Real Time RT-PCR in vitro diagnostic test for the rapid and qualitative detection and discrimination of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens obtained from symptomatic patients. This test is intended for use to aid in the differential diagnosis of Influenza A, Influenza B and RSV viral infections in humans and is not intended to detect Influenza C.

A negative test is presumptive and it is recommended these results be confirmed by cell culture. Negative results do not preclude influenza or RSV virus infection and should not be used as the sole basis for treatment or other management decisions.

Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infections with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use.   
3. Special conditions for use statement(s): For prescription use only   
4. Special instrument requirements: Cepheid SmartCycler II Real Time Instrument

# I. Device Description:

The ProFlu+ Assay enables detection and differentiation of Influenza A Virus, Influenza B Virus, Respiratory Syncytial Virus (Types A and B), and Internal Control.

An overview of the procedure is as follows:

1. Collect nasopharyngeal swab specimens from symptomatic patients using a polyester, rayon or nylon tipped swab and place into viral transport medium (refer to Materials Required but not Provided).   
2. Add an Internal Control (IC) to every sample to monitor for inhibitors present in the specimens.   
3. Perform isolation and purification of nucleic acids using the MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche).   
4. Add purified nucleic acids to ProFlu $^ +$ Supermix along with enzymes included in the ProFlu $^ +$ Detection Kit. The ProFlu $^ +$ Supermix contains oligonucleotide primers and target-specific oligonucleotide probes. The primers are complementary to highly conserved regions of genetic sequences for these respiratory viruses. The probes are dual-labeled with a reporter dye attached to the $_ 5 \cdot$ -end and a quencher dye attached to the $_ 3 \cdot$ -end (see Table below).

5. Perform reverse transcription of RNA into complementary DNA (cDNA) and subsequent amplification of DNA in a Cepheid SmartCycler ${ \mathfrak { P } } _ { \mathrm { I I } }$ instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProFlu $^ +$ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=2>Gene Targeted        Probe Fluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=2>Emission   InstrumentPeak      Channel</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>Polymerase</td><td rowspan=1 colspan=1>Cal Orange 560</td><td rowspan=1 colspan=1>540 nm</td><td rowspan=1 colspan=1>561 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=1>Polymerase</td><td rowspan=1 colspan=1>Cal Orange 560</td><td rowspan=1 colspan=1>540 nm</td><td rowspan=1 colspan=1>561 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>Non-structuralNS1 and NS2</td><td rowspan=1 colspan=1>Cal Red 610</td><td rowspan=1 colspan=1>595 nm</td><td rowspan=1 colspan=1>615 nm</td><td rowspan=1 colspan=1>Texas Red</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>667 nm</td><td rowspan=1 colspan=1>$Cy_\}$</td></tr></table>

# Materials Provided

$\mathrm { P r o F l u ^ { + } }$ Assay consists of two separate boxes: Detection Kit (Cat. # H44VK00) and Control Kit (Cat. # H44VK55).

Box 1: Detection Kit (Cat. # H44VK00)   

<table><tr><td rowspan=1 colspan=5>Reagents                            Description                         Quantity/   Cap    Cat. #Tube    Color</td><td rowspan=1 colspan=1>Reactions/Tube</td></tr><tr><td rowspan=1 colspan=1>ProFlu+Supermix</td><td rowspan=1 colspan=1>•Taq DNA polymerase, 5 oligonucleotide primer pairs5 oligonucleotide probes— Buffer containing dNTPs (dATP, dCTP,dGTP, dTTP),2 MgCl2 and stabilizers— Bovine serum albumin</td><td rowspan=1 colspan=1>1030 μL</td><td rowspan=1 colspan=1>Brown</td><td rowspan=1 colspan=1>HSM77</td><td rowspan=1 colspan=1>50(2 tubesprovided)</td></tr><tr><td rowspan=1 colspan=1>M-MLVReverseTranscriptase</td><td rowspan=1 colspan=1>—10U/μL</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>GLS26</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RNaseInhibitor</td><td rowspan=1 colspan=1>—40U/μL</td><td rowspan=1 colspan=1>100 μL</td><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>GLS27</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Internal RNAControl III</td><td rowspan=1 colspan=1>—Non-infectious in vitro transcribed RNA</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>GCT12</td><td rowspan=1 colspan=1>100</td></tr></table>

Box 2: Control Kit (Cat. # H44VK55)   

<table><tr><td rowspan=1 colspan=6>Reagents                            Description                         Quantit   Cap   Cat. Controly/ Tube  Color    #    s/ Tube</td></tr><tr><td rowspan=1 colspan=1>Influenza ARNA ControlIII</td><td rowspan=1 colspan=1>—Non-infectious in vitro transcribed RNA ofspecific viral sequences</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>HCT75</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Influenza BRNA ControlIII</td><td rowspan=1 colspan=1>2Non-infectious in vitro transcribed RNA ofspecific viral sequences</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>HCT76</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>RSV A RNAControl III</td><td rowspan=1 colspan=1>Non-infectious in vitro transcribed RNA ofspecific viral sequences</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>Purple</td><td rowspan=1 colspan=1>HCT77</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>RSV B RNAControl III</td><td rowspan=1 colspan=1>—Non-infectious in vitro transcribed RNA ofspecific viral sequences</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>Clear</td><td rowspan=1 colspan=1>HCT78</td><td rowspan=1 colspan=1>15</td></tr></table>

# Materials Required But Not Provided

# Plasticware and consumables

□ Polyester, rayon or nylon tipped nasopharyngeal swabs RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes Sterile RNase/DNase-free filter or positive displacement micropipettor tips   
MagNA Pure LC System Disposables (Reagent Tubs, Reaction Tips, Tip Trays, Cartridges)   
Cepheid PCR reaction tubes, $2 5 \mu \mathrm { L }$   
Parafilm $^ { \textregistered } \mathbf { M }$ or MagNA Pure LC Cartridge Seal

# Reagents

Roche MagNA Pure LC Total Nucleic Acid Isolation Kit (Roche Cat. # 03038505001) for 192 isolations Micro TestTM M4 Viral Transport Medium (Remel, Inc. Cat. # 12500) or BD Universal Viral Transport medium (UTM; Becton, Dickinson and Co. Cat. # 220220) Molecular Grade Water (RNase/DNase Free) Extraction Control (e.g. previously characterized positive sample)

# Equipment

– $7 0 ^ { \circ } \mathrm { C }$ Freezer Roche MagNA Pure LC System with software version 3.0.11   
Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { - } 1 0 0 0 \mu \mathrm { L }$ ) Mini-centrifuge with adapter for Cepheid Reaction Tubes   
Cepheid cooling block

# Interpretation of Specimen Results

The SmartCycler Dx software automatically determines the specimen results. The interpretation of the assay specimen results is as follows:

<table><tr><td rowspan=1 colspan=1>SampleID&#x27;</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>ICResult</td><td rowspan=1 colspan=1>Warning/ ErrorCode</td><td rowspan=1 colspan=1>Influenza AResult</td><td rowspan=1 colspan=1>RSVResult</td><td rowspan=1 colspan=1>Influenza BResult</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Influenza A, B and RSVnucleic acid not detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Influenza A nucleic aciddetected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>RSV nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Influenza B nucleic aciddetected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Influenza A and RSV nucleicacid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Influenza A and Influenza Bnucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>RSV and Influenza B nucleicacid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Influenza A, Influenza B andRSV nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Unresolved - PCR inhibitionor reagent failure</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>ND3</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>30792</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined  error code3079</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40983</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not determined  error code4098</td></tr></table>

1 Columns and data not used for interpretation are not included   
2 Error Code 3079: Warning/Error Code 3079 is periodically observed with Influenza A positives (Influenza A Positive Control, Influenza A positive NP swab samples). Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a $C t$ value $\geq I 3$ is reported in the Influenza A, RSV, and/or Influenza B Ct columns, the sample results can be recorded as POS for the specific analyte(s).   
3 An Invalid assay run will display Error Code 4098   
\* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High viral load can lead to reduced or absent Internal Control signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s): xTAG™ RVP (Respiratory Viral Panel) Common Name: Respiratory Viral Panel (RVP) Multiplex Nucleic Acid Detection Assay   
2. Predicate 510(k) number(s): k063765   
3. Comparison with predicate: Both assays detect Influenza A and B, and respiratory syncytial virus using

nucleic acid amplification techniques. Both assays use nasal pharyngeal swabs as the collection device and the MagNA Pure LC system for nucleic acid isolation. The detection system with both assays involves spectrophotometric detection. The assays differ in that the predicate also detects Influenza A subtypes H1 and H3, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Rhinovirus, and Adenovirus.

# K. Standard/Guidance Document Referenced (if applicable):

Special controls guidance documents will be promulgated Guidance on Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses (March 2006) Guidance on In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path (April 2006) Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) Draft Guidance on Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens (Dec 2005) – http://www.fda.gov/cdrh/oivd/guidance/1560.html Software Guidance for the content of premarket submissions for software contained in medical devices (May 2005) – http://www.fda.gov/cdrh/ode/guidance/337.html General Guidance on Software Validation (Jan 2002) – http://www.fda.gov/cdrh/comp/guidance/938.html CLSI EP17-A: Guidance for Protocols for Determination of Limits of Detection and Limits of Quantitations (Vol. 2, No. 34) (Oct 2004). CLSI MM13-A: Guidance for the Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (Vol. 25, No. 31) (Dec 2005). CLSI EP7-A2: Guidance for Interference Testing in Clinical Chemistry (Vol. 25, No.27 Second Ed) (Nov 2005). • CLSI EP12-A: Guidance for User Protocol for Evaluation of Qualitative Test Performance (Vol. 22, No. 14) (Sept 2002). CLSI MM6-A: Guidance for the Quantitative Molecular Methods for Infectious Diseases (Vol. 23, No.28) (Oct 2003). CLSI EP5-A2: Guidance for Evaluation of Precision Performance of Quantitative Measurement Methods (Vol. 24, No. 25 Second Ed.) (Aug 2004).

# L. Test Principle:

See I

# M. Performance Characteristics (if/when applicable):

# Expected Values

The prevalence of Influenza and RSV varies each year with epidemics occurring during the fall and winter months in the US. Variables that affect the rate of positivity observed in respiratory testing include: the efficiency and timing of specimen collection, handling and transport of the specimen, the time of year, age of the patient, and local disease prevalence. During the 2006-2007 U.S. respiratory season, the combined prevalence of Influenza A and Influenza B was $1 3 . 2 \%$ and in 2005-2006 the combined prevalence was $1 2 . 1 \% ^ { 9 }$ . The prevalence of RSV during the 2005-2006 season was $1 6 . 2 \% ^ { 1 0 }$ . In the 2007

ProFlu $^ +$ multi-center clinical study (samples collected between February and April), the prevalence as observed with culture of Influenza A was $1 5 . 8 \%$ , Influenza B was $5 . 4 \%$ and RSV was $4 . 2 \%$ . As influenza and RSV seasons overlap, dual positive infections can occur. During this study, culture and the ProFlu $^ +$ Assay each detected one Influenza A and RSV dual-positive (although not the same sample) and ProFlu+ detected one Influenza A and Influenza B dual-positive out of the 901 total samples included in the study. Because the incidence of a triple infection of Influenza A, Influenza B, and RSV is low, it is recommended that the samples undergo repeat testing if nucleic acids from all three analytes are detected in a single sample.

# Clinical Performance

Performance characteristics of the ProFlu+ Assay were established during a prospective study at 3 U.S. clinical laboratories and a retrospective study at 1 U.S. site during the 2006-2007 respiratory virus season (February – April). Samples used for this study were nasopharyngeal (NP) swab specimens that were collected for routine influenza or RSV testing by each site.

The reference method was rapid culture (shell vial) followed by direct fluorescent antibody (DFA) screening and identification.

A total of 891 NP swab samples were tested with the ProFlu $^ +$ Assay and by culture. Five (5) samples that initially gave unresolved results remained unresolved upon retesting with the ProFlu $^ +$ Assay and are not included in the analysis below. All 5 samples were culture negative.

A total of 23 samples were DFA Respiratory Virus Screen positive (screening reagent detects Influenza A and B, RSV, Parainfluenza 1, 2 and 3 and Adenovirus), but contained too few cells to obtain a specific positive identification. 21 of these 23 samples were also positive by the ProFlu $^ +$ Assay (9 Influenza A positive, 11 Influenza B positive and 1 RSV positive) and genetic sequencing analysis confirmed the identification of the specific virus. The other 2 DFA screen positive samples were negative by the ProFlu $^ +$ Assay and sequence analysis confirmed that they were negative for Influenza A, Influenza B and RSV; these 2 samples were considered true negatives. Discrepant analysis for samples where ProFlu+ Assay and culture results were in disagreement was performed using RT-PCR with virus specific primers obtained from literature followed by sequencing.

# Results from Prospective Study

Influenza A Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=2>Positive    Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Comments</td></tr><tr><td rowspan=2 colspan=1>PrASss</td><td rowspan=3 colspan=1>PositiveNegativeTotal</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>52 a</td><td rowspan=1 colspan=1>179</td><td rowspan=1 colspan=1>Sensitivity 100% (97.1% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>647</td><td rowspan=1 colspan=1>647</td><td rowspan=1 colspan=1>Specificity 92.6% (90.4% -94.3%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>699</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1></td></tr></table>

a Forty-three (43) samples positive for Influenza A by sequence analysis, 8 samples negative for Influenza A by sequence analysis, and 1 sample unavailable for sequence analysis.

# Influenza B Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=2>Positive    Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Comments</td></tr><tr><td rowspan=2 colspan=1>ProAsssE</td><td rowspan=3 colspan=1>PositiveNegativeTotal</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>11 a</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>Sensitivity 97.8% (88.7% -99.6%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>769</td><td rowspan=1 colspan=1>770</td><td rowspan=1 colspan=1>Specificity 98.6% (97.5% -99.2%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>780</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1></td></tr></table>

a Eleven (11) samples positive for Influenza $B$ by sequence analysis. b One (1) sample negative for Influenza $B$ by sequence analysis.

# RSV Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>20</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34 a</td><td rowspan=1 colspan=1>40 </td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>Sensitivity 89.5% (75.9% -95.8%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>748</td><td rowspan=1 colspan=1>752</td><td rowspan=1 colspan=1>Specificity 94.9% (93.2% -96.2%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1></td></tr></table>

a Thirty-four (34) samples positive for RSV by sequence analysis, 3 samples negative for RSV by sequence analysis, and 3 samples unavailable for sequence analysis. b One (1) sample positive for RSV by sequence analysis and 3 samples negative for RSV by sequence analysis.

# Results from Retrospective Study

Influenza A Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=2>Positive    Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Comments</td></tr><tr><td rowspan=2 colspan=1>PrASss</td><td rowspan=3 colspan=1>PositiveNegativeTotal</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Sensitivity 100% (56.6% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>Specificity 96.4% (87.7% -99.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1></td></tr></table>

a One (1) samples positive for Influenza A by sequence analysis and 1 sample negative for Influenza A by sequence analysis

# Influenza B Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=2>Positive    Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Comments</td></tr><tr><td rowspan=2 colspan=1>ProAsssE</td><td rowspan=3 colspan=1>PositiveNegativeTotal</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Sensitivity 89.5% (68.6% -97.1%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>Specificity 100% (91.4% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1></td></tr></table>

a Two (2) samples positive for Influenza $B$ by sequence analysis.

# RSV Comparison Results

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>P</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>−1</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Sensitivity 100% (85.7% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Specificity 97.3% (86.2% -99.5%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1></td></tr></table>

One sample positive for RSV by sequence analysis.

# Reproducibility

The reproducibility of the $\mathrm { P r o F l u ^ { + } }$ Assay was evaluated at 3 laboratory sites. Reproducibility was assessed using a panel of 10 simulated samples that included medium and low (near the assay limit of detection) Influenza A, Influenza B, or RSV positive and negative samples. Panels and controls were tested at each site by 2 operators for 5 days (10 samples and 5 controls X 2 operators X 5 days X 3 sites $= 4 5 0$ ). The overall percent agreement for the ProFlu $^ +$ Assay was $9 8 \%$ .

<table><tr><td rowspan=2 colspan=1>Panel MemberID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwith expectedresullt (%)</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Agreementwith expectedresult</td><td rowspan=1 colspan=1>AVECT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVECT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVECT</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Negative (2PanelMembers)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>19*/20</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>59/60(98%)</td><td rowspan=1 colspan=1>91% - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza ALow Positive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>5.3%</td><td rowspan=1 colspan=1>26/30(87%)</td><td rowspan=1 colspan=1>70% - 95%</td></tr><tr><td rowspan=1 colspan=1>Influenza AMediumPositive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza BLow Positive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza BMediumPositive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>RSV A Lowpositive</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>6.2%</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>24/30(80%)</td><td rowspan=1 colspan=1>63% - 90%</td></tr><tr><td rowspan=1 colspan=1>RSV A mediumpositive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>RSV B lowpositive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>RSV B mediumpositive</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza ARNA Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza BRNA Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>RSV A RNAControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>RSV B RNAControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>4.3%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>4.0%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>5.2%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30/30 (100%)</td><td rowspan=1 colspan=1>89% - 100%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreementAll</td><td rowspan=1 colspan=3>148/150 (99%)</td><td rowspan=1 colspan=3>147/150 (98%)</td><td rowspan=1 colspan=3>144/150 (96%)</td><td rowspan=1 colspan=1>439/450(98%)</td><td rowspan=1 colspan=1>96% - 99%</td></tr></table>

\* 1 negative sample Unresolved $\mathrm { T C } = \mathrm { F A I L }$ ). $\mathrm { C _ { T } }$ values for Influenza A, Influenza B and RSV were negative, however.

# Analytical Sensitivity

The analytical sensitivity (limit of detection or LoD) of the $\mathrm { P r o F l u ^ { + } }$ Assay was determined using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of 4 Influenza A (2 H1N1 and 2 H3N2), 2 Influenza B, 2 Respiratory Syncytial Virus Type A, and 2 Respiratory Syncytial Virus Type B strains serially diluted in nasopharyngeal clinical matrix. Each viral strain was extracted using the Roche MagNA Pure LC and tested in replicates of 20 per concentration of virus.

Analytical sensitivity (LoD) as defined as the lowest concentration at which $\ge 9 5 \%$ of all replicates tested positive, ranged from $1 0 ^ { 2 } - 1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>LoD Concentration</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Port Chalmers/1/73 (H3N2)</td><td rowspan=1 colspan=1>101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/CA/7/04 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Caledonia/12/99 (H1N1)</td><td rowspan=1 colspan=1>102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/WS/33 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Lee/40</td><td rowspan=1 colspan=1>101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Wisconsin/2/06</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus Type A Strain Long</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus Type A Strain A-2</td><td rowspan=1 colspan=1>101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus Type B Strain Wildtype B-1</td><td rowspan=1 colspan=1>10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus Type B Strain Wash/18537/62</td><td rowspan=1 colspan=1>101 TCID50/mL</td></tr></table>

# Reactivity

The reactivity of the ProFlu $^ +$ Assay was evaluated against multiple strains of Influenza A (H1N1, H3N2, and H5N1 subtypes), Influenza B, and Respiratory Syncytial Viruses. The panel consisted of 12 Influenza A subtype H1N1, 13 Influenza A subtype H3N2, 2 Influenza A subtype H5N1, 11 Influenza B, and 5 Respiratory Syncytial Virus strains. Each viral strain was extracted using the Roche MagNA Pure LC and tested in triplicate. All viral cultures of the panel were detected by the ProFlu $^ +$ Assay.

<table><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Influenza A(FAM)</td><td colspan="1" rowspan="1">RSV(TET)</td><td colspan="1" rowspan="1">Influenza B(Tex Red)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Fuijan/156/00 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hawaii/15/01 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Kentucky/2/06 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Jiangxi/160/05 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Henan/8/05 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Taiwan/42/06 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Virginia/1/06 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/2652/06 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/New Caledonia/12/99(H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/WS/33 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/PR/8 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brazil/1137/99 (H1N1)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Fuijan/411/02 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/New York/55/2004 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/07/04 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/512/05 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/218/06 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Anhui/1239/05 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/2831/05 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hiroshima/52/05 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Kentucky/03/06 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers/1/73 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Bahamas/2686/99 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brazil/02/99 (H3N2)</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Costa Rica/07/99 (H3N2)</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/HK 486 (H5N1)</td><td colspan="1" rowspan="1">0.14 ng/μL*</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/VN 1203 (H5N1)</td><td colspan="1" rowspan="1">0.27 ng/μL **</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/7/04</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hawaii/11/05</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Michigan/2/06</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin/2/06</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hawaii/33/04</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Ohio/1/05</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/2/06</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/St Petersburg/14/06</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Michigan/4/06</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Lee/40</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Malaysia/2506/04</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain A2</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wildtype B-1</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash/18537/62</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV B Strain 9320</td><td colspan="1" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr></table>

\*estimated concentration $\overline { { 2 . 8 \mathrm { ~ x ~ } 1 0 ^ { 5 } } }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ \*\*estimated concentration $7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$

NOTE: Although the ProFlu+ Assay has been shown to detect cultured avian influenza viruses, including avian Influenza A subtype H5N1 virus, the performance characteristics of this test with specimens from humans infected with H5N1 or other avian influenza viruses are unknown.

# Analytical Specificity

The analytical specificity of the ProFlu $^ +$ Assay was evaluated by testing a panel of 50 cultures consisting of 22 viral,

27 bacterial, and 1 yeast strain representing common respiratory pathogens or flora commonly present in nasopharynx. Bacteria and yeast were tested at concentrations of ${ 1 0 } ^ { 6 }$ to $1 0 ^ { 7 } \dot { \mathrm { C F U / m L } }$ . Viruses were tested at concentrations of ${ { 1 0 } ^ { 3 } }$ to $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Samples were extracted using the Roche MagNA Pure LC and tested in triplicate. Analytical specificity of the ProFlu $^ +$ Assay was $100 \%$ .

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="2" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Influenza A(FAM)</td><td colspan="1" rowspan="1">RSV(TET)</td><td colspan="1" rowspan="1">Influenza B(Tex Red)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/PortChalmers</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Wisconsin</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A/Strain Long</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus1/Adenoid 71</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoxsackievirusB5/10/2006</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 1Maclnytre strain</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 39</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">−10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="2" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella</td><td colspan="2" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Influenza A(FAM)</td><td colspan="1" rowspan="1">RSV(TET)</td><td colspan="1" rowspan="1">Influenza B(Tex Red)</td></tr><tr><td colspan="1" rowspan="1">bronchiseptica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydiatrachomatis</td><td colspan="1" rowspan="1">104TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionellapneumophila</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumintracellulare</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseriagonorrhoeae</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria menigitidis</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">10' CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillusplantarum</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr></table>

# Competitive Inhibition

Competitive inhibition of the ProFlu $^ +$ Assay was evaluated using simulated samples with varying concentrations of Influenza A Virus (LoD to 3 logs above LoD) and Respiratory Syncytial Virus A (LoD to 4 logs above LoD) in a single sample. Samples were extracted using the Roche MagNA Pure LC and tested in triplicate. The presence of both Influenza A Virus and Respiratory Syncytial Virus A at varying concentrations in a single sample had no effect on the analytical sensitivity (limit of detection or LoD) of the ProFlu $^ +$ Assay.

# Carry-over/Contamination

In an internal study there was no evidence of carry-over/cross contamination with the ProFlu $^ +$ Assay using the Roche MagNA Pure LC automated nucleic acid extraction instrument.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.