# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K080570   
B. Purpose for Submission: New device

# C. Measurand:

Influenza virus nucleic acids target sequences. Influenza types and subtypes detected: Influenza A, Influenza A/H1, Influenza A/H3, Influenza A/H5, and Influenza B

# D. Type of Test:

A panel of oligonucleotide primers and dual-labeled hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes for the qualitative detection and differentiation of influenza virus type and subtype target sequences in nasal and nasopharyngeal swabs using nucleic acid isolation, amplification, and detection on the ABI 7500 Fast Dx Real-Time PCR instrument.

# E. Applicant:

Centers for Disease Control and Prevention

# F. Proprietary and Established Names:

CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel.   
Common Name: rRT-PCR Flu Panel

G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>NXD, OEP, OCC, NSU</td><td>Class II</td><td>21 CFR 866.3332 Reagents for detection of specific novel influenza A viruses</td><td>Microbiology (83)</td></tr></table>

# H. Intended Use:

1. Intended use(s):

The Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel) is intended for use in real-time RT-PCR assays on an ABI 7500 Fast Dx Real-Time PCR instrument in conjunction with clinical and epidemiological information:

for qualitative detection of influenza virus type A or B in symptomatic patients from viral RNA in nasopharyngeal and/or nasal swab specimens,

for determination of the subtype of seasonal human influenza A virus, as seasonal A/H1 or A/H3, if present, from viral RNA in nasopharyngeal and/or nasal swab specimens,   
for presumptive identification of virus in patients who may be infected with influenza A subtype A/H5 (Asian lineage) from viral RNA in human respiratory specimens and viral culture in conjunction with clinical and epidemiological risk factors.   
to provide epidemiologic information for surveillance for influenza viruses.

Performance characteristics for influenza A were established when influenza A/H3 and A/H1 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

Testing with the influenza H5a and H5b primer and probe sets should not be performed unless the patient meets the most current U.S. Department of Health and Human Services (DHHS) clinical and epidemiologic criteria for testing suspect A/H5 specimens. The definitive identification of influenza A/H5 (Asian lineage) either directly from patient specimens or from virus cultures requires additional laboratory testing, along with clinical and epidemiological assessment in consultation with national influenza surveillance experts.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

All users, analysts, and any person reporting diagnostic results from use of this device should be trained to perform and interpret the results from this procedure by a CDC instructor or designee prior to use. CDC Influenza Division will limit the distribution of this device to only those users who have successfully completed training provided by CDC instructors or designees.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: ABI 7500 Fast Dx Real-Time PCR instrument

# I. Device Description:

The CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel) is a panel of oligonucleotide primers and dual-labeled hydrolysis (TaqMan®) probes designed to be utilized in real-time RT-PCR assays using the ABI 7500 Fast Dx Real-Time PCR instrument for detection and characterization of human influenza viral RNA. The rRT-PCR Flu Panel influenza A and B primer and probe sets target highly conserved regions of the matrix (M) and non-structural (NS) genes for universal detection of type A and type B influenza viruses. Influenza A subtyping primer and probe sets target highly conserved regions of subtype-specific hemagglutinin (HA) RNA to detect contemporary A/H1, A/H3, and A/H5 (Asian lineage) influenza viruses in humans.

The rRT-PCR Flu Panel also includes control materials.

Internal positive control, the human RNase P (RP) primer and probe set detects human RP and is used with each clinical specimen to indicate that adequate isolation of nucleic acid resulted from the extraction of the specimen. A positive RP assay result also ensures that common reagents and equipment are functioning properly, and demonstrates the absence of inhibitory substances.   
• A Human Specimen Control (HSC) is a noninfectious cultured human cell material that demonstrates successful recovery of RNA as well as extraction reagent integrity. The Seasonal Influenza Virus Control (SIVC) consists of three different influenza viruses representing influenza A/H1, A/H3, and influenza B viruses and cultured human cells. The SIVC demonstrates that the master mix and primer and probe sets for influenza A (InfA), influenza B (InfB), influenza A/H1 (H1), influenza A/H3 (H3), and RP are functioning properly.   
• The Influenza Virus A/H5N1 Positive Control (H5VC) consists of a genetically modified reassortant human influenza virus (Influenza A/Vietnam/1203/04 x PR/8/34, BSL2 category) and cultured human cells. The H5VC demonstrates that the master mix and primer and probe sets for InfA, influenza A/H5 (H5a, H5b), and RP are functioning properly.

The use of the CDC rRT-PCR Flu Panel requires ancillary reagents for which specific lots have been qualified by the CDC Influenza Division and incorporated in the CDC quality system. Any lots not specifically qualified by CDC for use with the rRT-PCR Flu Panel are not validated for use with this assay and may affect device performance. Lots which are qualified under this process are communicated through a Qualified Reagent Lot List. This list is maintained by CDC and made available to the end users of the CDC rRT-PCR Flu Panel by including it with the Product Insert as well as through the Product Support mechanism identified in the Product Insert.

The following is a list of ancillary reagents that are not supplied with the rRT-PCR Flu Panel and are included in CDC’s reagent qualification program:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Reagent</td><td colspan="1" rowspan="1">Quantity</td><td colspan="1" rowspan="1">Catalog No.</td></tr><tr><td colspan="1" rowspan="4">rRT-PCREnzymeMaster mixOptions</td><td colspan="1" rowspan="2">Invitrogen SuperScript™M IIIPlatinum®One-Step Quantitative RT-PCR System (without Rox)</td><td colspan="1" rowspan="1">100reactions</td><td colspan="1" rowspan="1">11732-020</td></tr><tr><td colspan="1" rowspan="1">500reactions</td><td colspan="1" rowspan="1">11732-088</td></tr><tr><td colspan="1" rowspan="2">Invitrogen SuperScript™M IIIPlatinumROne-Step Quantitative RT-PCR System (with Rox)</td><td colspan="1" rowspan="1">100reactions</td><td colspan="1" rowspan="1">11745-100</td></tr><tr><td colspan="1" rowspan="1">500reactions</td><td colspan="1" rowspan="1">11745-500</td></tr><tr><td colspan="1" rowspan="1">NucleicAcid</td><td colspan="1" rowspan="1">Qiagen QIAampRViral RNA MiniKit *</td><td colspan="1" rowspan="1">50extractions</td><td colspan="1" rowspan="1">52904</td></tr><tr><td colspan="1" rowspan="5">PurificationKit Options</td><td colspan="1" rowspan="1">(Qiagen Inc., Valencia, CA)</td><td colspan="1" rowspan="1">250extractions</td><td colspan="1" rowspan="1">52906</td></tr><tr><td colspan="1" rowspan="2">Qiagen RNeasy® Mini Kit *(Qiagen Inc., Valencia, CA)</td><td colspan="1" rowspan="1">50extractions</td><td colspan="1" rowspan="1">74104</td></tr><tr><td colspan="1" rowspan="1">250extractions</td><td colspan="1" rowspan="1">74106</td></tr><tr><td colspan="1" rowspan="1">Roche MagNA Pure Total NucleicAcid Kit *(Roche Applied Science, Indianapolis,IN)</td><td colspan="1" rowspan="1">192extractions</td><td colspan="1" rowspan="1">03038505001</td></tr><tr><td colspan="1" rowspan="1">Roche MagNA Pure LC RNAIsolation Kit II *(Roche Applied Science, Indianapolis,IN)</td><td colspan="1" rowspan="1">192extractions</td><td colspan="1" rowspan="1">03172627001</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): Influenza A/H5 (Asian lineage) Virus Real-time RT-PCR Primer and Probe Set and the Prodesse Multiplex RT-PCR ProFlu™ Plus Assay

2. Predicate K number(s): K060159 K073029

3. Comparison with predicate:

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate 1</td><td colspan="1" rowspan="1">Predicate 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">rRT-PCR Flu Panel</td><td colspan="1" rowspan="1">Influenza A/H5 (Asianlineage) Virus Real-time RT-PCR Primersand Probe Set(K060159)</td><td colspan="1" rowspan="1">Prodesse MultiplexRT-PCR ProFlu+TMAssay (K073029)</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Nasopharyngeal or nasalswab respiratoryspecimens, or virus culture</td><td colspan="1" rowspan="1">Human respiratoryspecimens or viruscultures</td><td colspan="1" rowspan="1">Nasopharyngealswab specimens</td></tr><tr><td colspan="1" rowspan="1">Technology</td><td colspan="1" rowspan="1">Real-time RT-PCR</td><td colspan="1" rowspan="1">Real-time RT-PCR</td><td colspan="1" rowspan="1">Real-time RT-PCR</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethod</td><td colspan="1" rowspan="1">QIAampViral RNAMini Kit, Qiagen Inc.Qiagen RNeasy® MiniKit, Qiagen, Inc.•MagNA Pure LC RNAIsolation Kit II, RocheApplied ScienceMagNA Pure TotalNucleic Acid Isolation Kit,Roche Applied Science</td><td colspan="1" rowspan="1">QIAampViral RNAMini Kit, Qiagen,Inc.RNeasy® Mini Kit,Qiagen, Inc.MagNA Pure LCTotal Nucleic AcidIsolation Kit, RocheApplied Science</td><td colspan="1" rowspan="1">MagNA Pure TotalNucleic AcidIsolation Kit, RocheApplied Science</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate 1</td><td colspan="1" rowspan="1">Predicate 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">rRT-PCR Flu Panel</td><td colspan="1" rowspan="1">Influenza A/H5 (Asianlineage) Virus Real-timeRT-PCR Primers andProbe Set (K060159)</td><td colspan="1" rowspan="1">Prodesse Multiplex RT-PCR ProFlu+TM Assay(K073029)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative in-vitrodetection of influenzaA/H1, A/H3, A/H5(Asian lineage), andinfluenza B viruses</td><td colspan="1" rowspan="1">Qualitative in vitrodetection of influenzaA/H5N1 (Asian lineage)virus</td><td colspan="1" rowspan="1">Detection/differentiationbetween influenza Avirus, influenza B virusand respiratory syncytialvirus (RSV)</td></tr><tr><td colspan="1" rowspan="1">OrganismDetected</td><td colspan="1" rowspan="1">Influenza A/H1, A/H3,A/H5 (Asian lineage),and influenza B viruses</td><td colspan="1" rowspan="1">Influenza A virus,subtype H5N1 (Asianlineage)</td><td colspan="1" rowspan="1">Influenza type A andtype B viruses, RSV</td></tr><tr><td colspan="1" rowspan="1">Enzyme MasterMix</td><td colspan="1" rowspan="1">InvitrogenSuperScript™M IIIPlatinum® One-StepQuantitative RT-PCRKits (with or withoutROX)</td><td colspan="1" rowspan="1">Qiagen QuantiTectTMProbe RT-PCR Kit,Qiagen, Inc.</td><td colspan="1" rowspan="1">Supplied in theProFlu+TM DetectionKit</td></tr><tr><td colspan="1" rowspan="1">RequiredInstrumentation</td><td colspan="1" rowspan="1">Applied Biosystems7500 Fast Dx Real-Time PCR Instrumentwith SDS softwareversion 1.4</td><td colspan="1" rowspan="1">•Roche LightCycler®Cepheid SmartCycler®Applied Biosystems7000 SequenceDetection SystemApplied BiosystemsPrism® 7700 SequenceDetection System</td><td colspan="1" rowspan="1">Cepheid SmartCycler®II instrument</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses - http://www.fda.gov/cdrh/oivd/guidance/1596.pdf. Guidance on In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path - http://www.fda.gov/cdrh/oivd/guidance/1594.pdf.   
Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses -   
http://www.fda.gov/cdrh/oivd/guidance/1638.pdf

# L. Test Principle:

The rRT-PCR Flu Panel includes a set of oligonucleotide primers and dual-labeled hydrolysis (Taqman®) probes to be used in real-time RT-PCR assays on an ABI 7500 Fast Dx Real-Time PCR instrument. The targeted regions of viral RNA are transcribed into complimentary DNA (cDNA) and amplified by the polymerase chain reaction (PCR). The fluorescently labeled probes anneal to amplified DNA fragments and the fluorescent signal intensity is monitored by the ABI 7500 Fast Dx instrument during each PCR cycle. Amplification of target is recorded as increase of fluorescence over time in comparison to background signal.

# Summary of Steps in rRT-PCR Flu Panel Assay:

![](images/906ca0ef72b938c810c066d1fea741ef02fb09a1a1b77e09246935100ad86271.jpg)

# Interpretation of Results:

Expected Performance of Controls Included in the rRT-PCR Flu Panel:   

<table><tr><td rowspan=1 colspan=1>ControlType</td><td rowspan=1 colspan=1>InternalControlName</td><td rowspan=1 colspan=1>Used toMonitor</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>H5a</td><td rowspan=1 colspan=1>H5b</td><td rowspan=1 colspan=1>ExpectedCtValues</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>SIVC</td><td rowspan=1 colspan=1>Substantialreagent failureincludingprimer andprobe integrity</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>≤37 Ct</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>H5VC</td><td rowspan=1 colspan=1>Substantialreagent failureincludingprimer andprobe integrity</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>≤37 Ct</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>Reagent and/orenvironmentalcontamination</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Nonedetected</td></tr><tr><td rowspan=1 colspan=1>Extraction</td><td rowspan=1 colspan=1>HSC</td><td rowspan=1 colspan=1>Failure in lysisand extractionprocedure</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>≤37 Ct</td></tr></table>

If the assay controls do not exhibit the expected performance as described, the assay may have been set up and/or executed improperly, or reagent or equipment malfunction could have occurred. Invalidate the run and re-test.

# Specimen Results and Interpretation

RNase P (Extraction Control)

All clinical samples should exhibit fluorescence growth curves in the RNase P reaction that cross the threshold line within 37 cycles, thus indicating the presence of the human RNase P gene. Failure to detect RNase P in any clinical specimens may indicate:

Improper extraction of nucleic acid from clinical materials resulting in loss of RNA   
and/or RNA degradation   
Absence of sufficient human cellular material due to poor collection or loss of   
specimen integrity   
Improper assay set up and execution   
Reagent or equipment malfunction

If the RP assay does not produce a positive result for human clinical specimens, interpret as follows:

If the InfA, InfB, and/or any subtype reactions are positive even in the presence of a negative RP, the influenza result should be considered valid. It is possible, that some samples may fail to exhibit RNase P growth curves due to low cell numbers in the original clinical sample. A negative RP signal does not preclude the presence of influenza virus RNA in a clinical specimen.

If all influenza markers AND RNase P are negative for the specimen, the assay is “inconclusive” for the specimen. If residual specimen is available, repeat the extraction procedure and repeat the test. If all markers remain negative after re-test, report the results as “inconclusive” and a new specimen should be collected if possible.

The RP assay may be negative when testing virus culture samples.

Influenza Markers (InfA, InfB, H1, H3, H5a, H5b)

When all controls exhibit the expected performance, a specimen is considered negative if influenza marker growth curves DO NOT cross the threshold line within 37 cycles and RNase P growth curve does cross the threshold line within 37 cycles.

When all controls exhibit the expected performance, a specimen is considered a “Presumptive Positive” for influenza if the influenza marker (InfA, InfB, H1, H3, H5a, and/or H5b) growth curve crosses the threshold line within 37 cycles. RP may be positive or negative as described above.

When all controls exhibit the expected performance and the growth curve for influenza A marker only crosses the threshold line within 37 cycles without any subtype detection (InfA - Influenza A positive without a subtype detected), the sample has potential for containing a novel and/or newly emerging influenza A virus. The extracted RNA from the specimen should be re-tested. If residual RNA is not available, re-extract RNA from residual specimen and re-test. If the re-tested sample is again positive for InfA only and all controls exhibit the expected performance, contact the CDC Influenza Division to coordinate transfer of the specimen to CDC for confirmatory testing.

When all controls exhibit the expected performance and none of the growth curves for the influenza markers or RP marker cross the threshold line within 37 cycles, the result is “Inconclusive”. The extracted RNA from the specimen should be re-tested. If residual RNA is not available, re-extract RNA from residual specimen and re-test. If the re-tested sample is negative for all markers and all controls exhibit the expected performance, the result is “Inconclusive.”

When all controls exhibit expected performance and growth curves for both H5a and H5b markers cross the threshold line within 37 cycles:

Report the specimen to be “Presumptive Positive for Influenza A/H5 (Asian lineage) Virus” and contact the CDC Influenza Division immediately to coordinate transfer of the specimen to CDC for additional testing.

When all controls exhibit the expected performance and growth curves for either H5a and/or H5b markers cross the threshold line within 37 cycles, extracted RNA from the specimen should be re-tested immediately. If residual RNA is not available, re-extract RNA from residual specimen and re-test. If all controls exhibit the expected performance on the repeated test and either H5a OR H5b reactions cross the threshold line within 37 cycles:

Report the specimen to be “Inconclusive for Influenza A/H5 (Asian lineage) virus” and contact the CDC Influenza Division immediately to coordinate transfer of the specimen to CDC for additional testing.

When all controls exhibit the expected performance and any of the following occurs, the test is “Invalid”:

growth curves for influenza A marker and influenza B marker cross the threshold line within 37 cycles, OR • growth curves for influenza A marker and two or more of the influenza A subtype markers cross the threshold line within 37 cycles, OR growth curves for any of the influenza A subtype markers (H1, H3, H5a, H5b) cross the threshold line within 37 cycles but the growth curve for the influenza A type marker (InfA) does not cross the threshold line within 37 cycles.

The extracted RNA from the specimen should be re-tested. If residual RNA is not available, re-extract RNA from residual specimen and re-test. If the repeat testing results remain the same as the original results, report the specimen as “Invalid.”

If the InfA, InfB, and/or any subtype reactions are positive and RP is negative, the influenza result should be considered valid.   
If a combination of results other than those in the chart is generated, the result is invalid. Repeat extraction and retest the sample. If repeat testing result is unchanged, report as “Invalid” result.

To refer the specimen to the CDC, the following shipping instructions should be followed:

Ship all specimens and related RNA overnight to CDC.   
Ship frozen specimens on dry ice and non-frozen specimens on cold packs. Ship extracted RNA on dry ice.   
Refer to the International Air Transport Association (IATA - www.iata.org) for requirements for shipment of human or potentially infectious biological specimens.   
Prior to shipping, notify the CDC Influenza Division (see contact information below) that you are sending specimens.   
Send all samples to the following recipient: Chief, Virus Surveillance and Diagnosis Branch Influenza Division Centers for Disease Control and Prevention c/o DASH, MS G-16 Attention: Dr. Stephen Lindstrom 1600 Clifton Rd., Atlanta, GA 30333 Phone: (404) 639-3387 or (404) 639-3591 Fax: (404) 639-2334

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The rRT-PCR Flu Panel reproducibility and precision studies were performed to 1) evaluate reproducibility of the assay at three laboratory sites using the ABI 7500 Fast Dx Real-Time PCR instruments and SDS software version 1.4 and 2) to compare the assay performance on the original ABI 7500 Fast Real-Time PCR instrument and the ABI 7500 Fast Dx Real-Time PCR instrument (bridging study).

1) The ABI 7500 Fast Dx Real-Time PCR instrument functionality was validated at each site by AB prior to the reproducibility assessment using a panel of nine (9) simulated samples including influenza A/H1N1, A/H3N2, A/H5N1 (reassortant), and Influenza B at two viral RNA concentrations each (a low viral RNA concentration and a 1:10 dilution of the same sample). The low viral RNA concentration generally was one log above the assay cutoff for all analytes, whereas the 1:10 dilution of the same sample approximated a sample at the assay cutoff. Simulated samples in the panel used in the reproducibility evaluation were:

Sample #1 Influenza A/H1N1 at a low viral RNA titer range Sample #2 Influenza A/H1N1 at a 1:10 dilution of sample #1 Sample #3 Influenza A/H3N2 at a low viral RNA titer range Sample #4 Influenza A/H3N2 at a 1:10 dilution of sample #3 Sample #5 Influenza A/H5N1 WT at a low viral RNA titer range Sample #6 Influenza A/H5N1 WT at a 1:10 dilution of sample #5 Sample #7 Influenza B Yamagata at a low viral RNA titer range Sample #8 Influenza B Yamagata at a 1:10 dilution of sample #7 Sample #9 Influenza Negative (uninfected A549 cells)

The panels and assay controls were tested at each site by two operators on five (5) different days within a 10-day period. Each participating clinical site tested one of the four following RNA purification methods to evaluate reproducibility of the CDC rRT-PCR Flu Panel on the validated ABI 7500 Fast Dx Real-Time PCR instruments: the automated MagNA Pure LC RNA Isolation Kit III (Roche Applied Science), the manual QIAGEN RNeasy® RNA extraction (QIAGEN Ltd.), the Roche MagNA Pure LC Automated Isolation System with Total RNA isolation method, and the QIAGEN QIAamp® Viral RNA manual extraction method. The manufacturer’s instructions for use provided in the package insert were followed. Results generated for each of the extraction methods are summarized in the Table below.

Reproducibility Study Summary for the CDC rRT-PCR Flu Panel on the ABI 7500   

<table><tr><td colspan="1" rowspan="2">Sample and AnalyteTested</td><td colspan="3" rowspan="1">Roche MagNA PureTNA</td><td colspan="3" rowspan="1">Qiagen QIAamp®RNA</td><td colspan="3" rowspan="1">Qiagen RNeasy®</td><td colspan="3" rowspan="1">Roche MagNA PureRNA</td><td colspan="1" rowspan="2">TotalAgreementw/ExpectedResults</td><td colspan="1" rowspan="2">95 %CI</td></tr><tr><td colspan="1" rowspan="1">AeenhWee Resct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Amnhwee Resct</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Reenhpepee /Resd</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Aeenhwee Rest</td><td colspan="1" rowspan="1">Avg.Ct</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low) InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.73</td><td colspan="1" rowspan="1">3.12</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.24</td><td colspan="1" rowspan="1">5.65</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.34</td><td colspan="1" rowspan="1">1.94</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.55</td><td colspan="1" rowspan="1">3.69</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low)H1</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.48</td><td colspan="1" rowspan="1">2.84</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.78</td><td colspan="1" rowspan="1">5.64</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.67</td><td colspan="1" rowspan="1">4.47</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.08</td><td colspan="1" rowspan="1">4.11</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 1 (low)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.62</td><td colspan="1" rowspan="1">4.00</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.08</td><td colspan="1" rowspan="1">5.63</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">27.67</td><td colspan="1" rowspan="1">3.28</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.22</td><td colspan="1" rowspan="1">3.13</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1)InfA</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.59</td><td colspan="1" rowspan="1">3.22</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.30</td><td colspan="1" rowspan="1">5.93</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">37.03</td><td colspan="1" rowspan="1">4.07</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.97</td><td colspan="1" rowspan="1">3.32</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1) H1</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">37.92</td><td colspan="1" rowspan="1">2.99</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.76</td><td colspan="1" rowspan="1">8.31</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.28</td><td colspan="1" rowspan="1">4.28</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.21</td><td colspan="1" rowspan="1">4.77</td><td colspan="1" rowspan="1">31/40</td><td colspan="1" rowspan="1">61.6 - 89.2</td></tr><tr><td colspan="1" rowspan="1">Sample 2(1:10 of Sample 1)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.51</td><td colspan="1" rowspan="1">3.80</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.13</td><td colspan="1" rowspan="1">9.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.46</td><td colspan="1" rowspan="1">2.47</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.65</td><td colspan="1" rowspan="1">2.12</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.39</td><td colspan="1" rowspan="1">3.74</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.32</td><td colspan="1" rowspan="1">7.09</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.01</td><td colspan="1" rowspan="1">7.16</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.29</td><td colspan="1" rowspan="1">1.82</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)H3</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.25</td><td colspan="1" rowspan="1">2.52</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.98</td><td colspan="1" rowspan="1">7.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.07</td><td colspan="1" rowspan="1">2.86</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">[33.28</td><td colspan="1" rowspan="1">1.71</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 3 (low)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.17</td><td colspan="1" rowspan="1">5.32</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.02</td><td colspan="1" rowspan="1">5.33</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.30</td><td colspan="1" rowspan="1">3.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.50</td><td colspan="1" rowspan="1">3.34</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1 10 of Sample 3)InfA</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.44</td><td colspan="1" rowspan="1">4.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">37.03</td><td colspan="1" rowspan="1">5.08</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">39.05</td><td colspan="1" rowspan="1">3.46</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.18</td><td colspan="1" rowspan="1">2.41</td><td colspan="1" rowspan="1">32/40</td><td colspan="1" rowspan="1">64.4 - 91.0</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1:10 of Sample 3) H3</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">38.82</td><td colspan="1" rowspan="1">3.34</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">39.90</td><td colspan="1" rowspan="1">6.14</td><td colspan="1" rowspan="1">6/10</td><td colspan="1" rowspan="1">38.01</td><td colspan="1" rowspan="1">4.81</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.82</td><td colspan="1" rowspan="1">3.49</td><td colspan="1" rowspan="1">29/40</td><td colspan="1" rowspan="1">56.1 - 85.4</td></tr><tr><td colspan="1" rowspan="1">Sample 4(1:10 of Sample 3)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.75</td><td colspan="1" rowspan="1">4.32</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.11</td><td colspan="1" rowspan="1">7.89</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.56</td><td colspan="1" rowspan="1">4.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.68</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)InfA</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.68</td><td colspan="1" rowspan="1">2.70</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">31.90</td><td colspan="1" rowspan="1">6.66</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.07</td><td colspan="1" rowspan="1">3.14</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.80</td><td colspan="1" rowspan="1">1.74</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)H5a</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.69</td><td colspan="1" rowspan="1">3.76</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.65</td><td colspan="1" rowspan="1">5.52</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.58</td><td colspan="1" rowspan="1">4.94</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.03</td><td colspan="1" rowspan="1">4.55</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)H5b</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">37.91</td><td colspan="1" rowspan="1">3.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.27</td><td colspan="1" rowspan="1">9.56</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">36.09</td><td colspan="1" rowspan="1">4.49</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.54</td><td colspan="1" rowspan="1">3.11</td><td colspan="1" rowspan="1">39/40</td><td colspan="1" rowspan="1">86.8 - 99.9</td></tr><tr><td colspan="1" rowspan="1">Sample 5 (low)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.44</td><td colspan="1" rowspan="1">4.05</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.16</td><td colspan="1" rowspan="1">7.18</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">27.97</td><td colspan="1" rowspan="1">4.84</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.46</td><td colspan="1" rowspan="1">2.78</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)InfA</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">36.90</td><td colspan="1" rowspan="1">6.60</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.84</td><td colspan="1" rowspan="1">4.89</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.10</td><td colspan="1" rowspan="1">5.15</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.54</td><td colspan="1" rowspan="1">2.61</td><td colspan="1" rowspan="1">36/40</td><td colspan="1" rowspan="1">76.3  97.2</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)H5aa</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">37.90</td><td colspan="1" rowspan="1">3.24</td><td colspan="1" rowspan="1">9/10</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">4.91</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">37.77</td><td colspan="1" rowspan="1">7.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.71</td><td colspan="1" rowspan="1">2.51</td><td colspan="1" rowspan="1">31/40</td><td colspan="1" rowspan="1">61.6- 89.2</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)H5b</td><td colspan="1" rowspan="1">4/10</td><td colspan="1" rowspan="1">39.29</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">5/10</td><td colspan="1" rowspan="1">39.43</td><td colspan="1" rowspan="1">3.74</td><td colspan="1" rowspan="1">7/10</td><td colspan="1" rowspan="1">38.74</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">35.71</td><td colspan="1" rowspan="1">5.30</td><td colspan="1" rowspan="1">27/40</td><td colspan="1" rowspan="1">50.9 - 81.4</td></tr><tr><td colspan="1" rowspan="1">Sample 6(1:10 of Sample 5)RNaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.05</td><td colspan="1" rowspan="1">3.58</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.12</td><td colspan="1" rowspan="1">3.39</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.59</td><td colspan="1" rowspan="1">4.37</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.66</td><td colspan="1" rowspan="1">3.45</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 7 (low)Inf B</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">34.10</td><td colspan="1" rowspan="1">4.25</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">33.49</td><td colspan="1" rowspan="1">9.12</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">34.70</td><td colspan="1" rowspan="1">7.70</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">32.83</td><td colspan="1" rowspan="1">3.61</td><td colspan="1" rowspan="1">38/40</td><td colspan="1" rowspan="1">83.1 - 99.4</td></tr><tr><td colspan="1" rowspan="1">Sample 7 (low)NaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.78</td><td colspan="1" rowspan="1">3.55</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.75</td><td colspan="1" rowspan="1">10.16</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.54</td><td colspan="1" rowspan="1">4.42</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.90</td><td colspan="1" rowspan="1">3.13</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 8(1:10 of Sample 7) InfB</td><td colspan="1" rowspan="1">1/10</td><td colspan="1" rowspan="1">36.76</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">5.77</td><td colspan="1" rowspan="1">3/10</td><td colspan="1" rowspan="1">38.95</td><td colspan="1" rowspan="1">ND*</td><td colspan="1" rowspan="1">8/10</td><td colspan="1" rowspan="1">37.11</td><td colspan="1" rowspan="1">5.81</td><td colspan="1" rowspan="1">20/40</td><td colspan="1" rowspan="1">33.8  66.2</td></tr><tr><td colspan="1" rowspan="1">Sample 8(1:10 of Sample 7)NaseP</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">29.48</td><td colspan="1" rowspan="1">4.46</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30.58</td><td colspan="1" rowspan="1">6.30</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.71</td><td colspan="1" rowspan="1">3.35</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">28.92</td><td colspan="1" rowspan="1">4.07</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">91.2100.0</td></tr><tr><td colspan="1" rowspan="1">Sample 9 Influenza (-)RNaseP</td><td colspan="9" rowspan="1">10/1029.79 3.17 10/10 30.08  5.91   10/10 28.27 4.83</td><td colspan="3" rowspan="1">10/10 28.86  3.68</td><td colspan="2" rowspan="1">40/40   91.2100.0</td></tr><tr><td colspan="15" rowspan="1">1161/1320 86.4189.6213 / 250 85.2 %      319 /330 96.7 %      303 / 33091.8 %      326 / 330 98.8 %88.0 %</td></tr></table>

2) Assay performance, when run on the original ABI 7500 Fast Real-Time PCR instrument and the ABI 7500 Fast Dx Real-Time PCR instrument, was compared utilizing an identical study protocol to that described in paragraph 1) above and using simulated samples for influenza A/H1N1, A/H3N2, A/H5N1 (reassortant), and Influenza B at a low viral RNA concentration, (generally one log above the assay cutoff). Although several panel members displayed, on average, statistically significant higher Ct values in the reproducibility study on the ABI 7500 Fast Dx Real-Time PCR instrument as compared to the original reproducibility study run on the ABI 7500 Fast Real-Time PCR instrument, all values were within clinically acceptable levels. The difference could be attributed to different operators and panel reagent lot-to-lot variability.

# b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods): Before analyzing the samples prepared using the CDC rRT-PCR Flu Panel on the ABI 7500 Fast Dx Real-Time PCR instrument, the ABI instrument must be prepared following the procedures described in the Package Insert for the CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel and the ABI 7500 Fast Dx Real-Time PCR instrument User Manual.

# Assay Controls

Quality Control requirements must be performed in conformance with local, state, and/or federal regulations or accreditation requirements and a laboratory’s standard quality control procedures. It is recommended that the user refer to CLSI document C24-A2.

# No Template Control (NTC)

The NTC consists of using nuclease-free water in the rRT-PCR reactions instead of RNA. The NTC reactions for all primer and probe sets should not exhibit fluorescence growth curves that cross the threshold line. If any of the NTC reactions exhibit a growth curve that crosses the cycle threshold, sample

contamination may have occurred. Invalidate the run and repeat the assay with stricter adherence to the guidelines.

# Seasonal Influenza Virus Control (SIVC)

The SIVC consists of three different influenza viruses representing influenza A/H1N1, A/H3N2, and influenza B and cultured human cells (A549). Purified RNA from the SIVC will yield a positive result with the following primer and probe sets: InfA, H1, H3, InfB, and RP.

# H5 Virus Control (H5VC)

The H5VC control consists of a reassortant human vaccine candidate virus (A/Vietnam/1203/04 x PR/8/34) that was generated by reverse genetics and cultured human cell (A549) material. Purified RNA from the H5VC will yield a positive result with the following primer and probe sets: InfA, H5a, H5b, and RP.

# Human Specimen Control (HSC) (Extraction)

The HSC control consists of noninfectious cultured human cell (A549) material. The HSC is used as a RNA extraction procedural control to demonstrate successful recovery of RNA as well as extraction reagent integrity. Purified RNA from the HSC should yield a positive result with the RP primer and probe set and negative results with all influenza specific markers

A negative amplification/detection control (NTC) and a positive control (SIVC and/or H5VC) should be included in each run. An HSC extraction control must proceed through nucleic acid isolation for each batch of specimens to be tested.

# d. Detection limit:

The Limit of Detection (LoD) was determined for each primer and probe set in the rRT-PCR Flu Panel. Ten-fold serial dilutions of two different influenza virus strains of each subtype were tested to identify an end-point for detection. RNA was extracted from each of the characterized viruses with the Qiagen QIAamp® Viral RNA Purification kit. The LoD for each primer and probe set (InfA, InfB, H1, H3, H5a, and H5b) was calculated to determine the lowest detectable concentration range of influenza virus $\mathrm { ( E I D _ { 5 0 } / m L ) }$ at which approximately $9 5 \%$ of all replicates tested positive. The lowest concentration of influenza virus detected was determined to be the end-point concentration where the type and subtype primer and probe sets had uniform detection. If the two end-points differed in concentration, the higher (or limiting) point was used.

Limit of Detection Summary   

<table><tr><td rowspan=1 colspan=1>InfluenzaVirus Tested</td><td rowspan=1 colspan=1>Influenza Strain Designation</td><td rowspan=1 colspan=1>Limit ofDetection(EID50/mL)</td></tr><tr><td rowspan=2 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>101.2</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/2001</td><td rowspan=1 colspan=1>101.5</td></tr><tr><td rowspan=2 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>A/New York/55/2004</td><td rowspan=1 colspan=1>102.2</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>101.2</td></tr><tr><td rowspan=2 colspan=1>A/H5N1</td><td rowspan=1 colspan=1>A/Vietnam/1203/2004×A/Puerto Rico/8/34reassortant(A/Vietnam/1203/2004 PR8-VNH5N1-PR8/CDC-RG)</td><td rowspan=1 colspan=1>101.0</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/01/2005×A/Puerto Rico/8/34 reassortant(A/Anhui/01/2005- PR8-IBCDC-RG5)</td><td rowspan=1 colspan=1>101.0</td></tr><tr><td rowspan=2 colspan=1>B</td><td rowspan=1 colspan=1>B/Florida/07/2004 (B/Victoria/2/87 genetic group)</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005 (B/Yamagata/16/88 genetic group)</td><td rowspan=1 colspan=1>100.5</td></tr></table>

# e. Analytical specificity:

Analytical specificity of rRT-PCR Flu Panel was evaluated with respect to 1) reactivity (inclusivity) with a number of geographically diverse influenza virus strains and 2) potential cross-reactivity with non-influenza pathogens associated with respiratory tract infections.

# 1) Reactivity/Inclusivity

Ten (10) influenza virus strains of A/H1N1, A/H3N2, and Influenza B were tested at low virus concentrations at or near the limit of detection $( 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ to demonstrate the flexibility of the primer and probe sets to detect multiple strains of influenza virus. There were 26 influenza A/H5N1 strains tested retrospectively from diverse geographic locations from suspect positive cases received at CDC. The rRT-PCR Flu Panel analytical specificity indicated $100 \%$ concordance with expected results for all primer and probe sets included in the device.

Analytical Reactivity Testing Results of the rRT-PCR Flu Panel with Human Seasonal Influenza Viruses A/H1, A/H3, and Influenza B.   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2">Strain Identification</td><td colspan="1" rowspan="2">StockTCID50mL</td><td colspan="6" rowspan="1">rRT-PCR Flu Panel Analyte Tested at10² TCID50/mL - (Avg Ct n=3)</td><td colspan="1" rowspan="2">Result</td></tr><tr><td colspan="1" rowspan="1">InfA</td><td colspan="1" rowspan="1">InfB</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">H5a</td><td colspan="1" rowspan="1">H5b</td></tr><tr><td colspan="1" rowspan="3">ThunnaaNIHA</td><td colspan="1" rowspan="1">A/JIANGXI/160/2005</td><td colspan="1" rowspan="1">105.6</td><td colspan="1" rowspan="1">+(25.25)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(25.57)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Island/03/2006</td><td colspan="1" rowspan="1">106.2</td><td colspan="1" rowspan="1">+(29.20)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(28.37)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Taiwan/42/2006</td><td colspan="1" rowspan="1">104.7</td><td colspan="1" rowspan="1">+(21.43)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(21.25)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">,</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="7"></td><td colspan="1" rowspan="1">A/Fukushima /141/2006</td><td colspan="1" rowspan="1">105.7</td><td colspan="1" rowspan="1">+(24.96)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(25.16)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1744/2007</td><td colspan="1" rowspan="1">105.3</td><td colspan="1" rowspan="1">+(22.56)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.13)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1729/2007</td><td colspan="1" rowspan="1">104.8</td><td colspan="1" rowspan="1">+(23.81)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.67)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1677/2007</td><td colspan="1" rowspan="1">105.7</td><td colspan="1" rowspan="1">+(25.55)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(25.22)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/949/2007</td><td colspan="1" rowspan="1">105.1</td><td colspan="1" rowspan="1">+(26.27)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(25.79)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Bangladesh/286/2007</td><td colspan="1" rowspan="1">106.1</td><td colspan="1" rowspan="1">+(27.94)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(27.84)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/2010/2007</td><td colspan="1" rowspan="1">105.1</td><td colspan="1" rowspan="1">+(24.27)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.54)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AH1</td></tr><tr><td colspan="1" rowspan="10">Ii/ Azu</td><td colspan="1" rowspan="1">A/Hawaii/08/2006</td><td colspan="1" rowspan="1">107.8</td><td colspan="1" rowspan="1">+(30.30)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(31.56)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/03/2007</td><td colspan="1" rowspan="1">108.2</td><td colspan="1" rowspan="1">+(36.75)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(35.82)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Henan/Jinshui/147/2007</td><td colspan="1" rowspan="1">108.1</td><td colspan="1" rowspan="1">+(25.52)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.72)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/2007</td><td colspan="1" rowspan="1">106.8</td><td colspan="1" rowspan="1">+(24.71)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.92)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/922/2007</td><td colspan="1" rowspan="1">107.8</td><td colspan="1" rowspan="1">+(24.07)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.08)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Afghanistan/2903/2008</td><td colspan="1" rowspan="1">105.0</td><td colspan="1" rowspan="1">+(26.46)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(26.51)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1938/2007</td><td colspan="1" rowspan="1">105.1</td><td colspan="1" rowspan="1">+(23.94)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.15)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1995/2007</td><td colspan="1" rowspan="1">104.3</td><td colspan="1" rowspan="1">+(21.69)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(21.56)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Anhui/1239/2005</td><td colspan="1" rowspan="1">108.1</td><td colspan="1" rowspan="1">+(23.02)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(22.13)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/1842/2007</td><td colspan="1" rowspan="1">106.1</td><td colspan="1" rowspan="1">+(24.14)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.31)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">AH3</td></tr><tr><td colspan="1" rowspan="10">ITz</td><td colspan="1" rowspan="1">B/Florida/04/2006</td><td colspan="1" rowspan="1">105.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.52)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Chongqing/Yongchuan/18/2007</td><td colspan="1" rowspan="1">107.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(28.71)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Florida/02/2006</td><td colspan="1" rowspan="1">106.0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.31)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Pennsylvania/05/2007</td><td colspan="1" rowspan="1">106.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.32)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Bangladesh/5972/2007</td><td colspan="1" rowspan="1">104.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(26.86)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Bangladesh/3461/2007</td><td colspan="1" rowspan="1">103.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(29.29)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Bangladesh/7110/2007</td><td colspan="1" rowspan="1">104.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(26.82)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Mexico/1819/2007</td><td colspan="1" rowspan="1">105.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(24.60)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Texas/17/2007</td><td colspan="1" rowspan="1">105.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.49)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr><tr><td colspan="1" rowspan="1">B/Texas/03/2007</td><td colspan="1" rowspan="1">105.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+(23.71)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Inf. B</td></tr></table>

Analytical Reactivity Testing Results of the rRT-PCR Flu Panel with Influenz A/H5N1Virus   

<table><tr><td rowspan=2 colspan=1>StrainIdentification</td><td rowspan=2 colspan=1>Species</td><td rowspan=2 colspan=1>SpecimenType</td><td rowspan=2 colspan=1>CountryofOrigin</td><td rowspan=2 colspan=1>Clade</td><td rowspan=1 colspan=6>rRT-PCR Flu Panel Analyte Tested</td><td rowspan=2 colspan=1>ExpectedResult</td></tr><tr><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>InfB</td><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>H5a</td><td rowspan=1 colspan=1>H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706688</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706690</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706691</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706692</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706693</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706694</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706695</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706696</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706697</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706698</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706699</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2007726139</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>throatswab</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2007726140</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>throatswab</td><td rowspan=1 colspan=1>Egypt</td><td rowspan=1 colspan=1>2.2.1</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008706700</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>egggrown</td><td rowspan=1 colspan=1>Pakistan</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#200732012</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#200732013</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#200732014</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2007730165</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>throatswab</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2007730166</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>trachealaspirate</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2007730167</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>throatswab</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008740525</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008740526</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008740527</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr><tr><td rowspan=1 colspan=1>CDC ref#2008740528</td><td rowspan=1 colspan=1>Human</td><td rowspan=1 colspan=1>tissueculture</td><td rowspan=1 colspan=1>Vietnam</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>+(1/1)</td><td rowspan=1 colspan=1>InfA,H5a, H5b</td></tr></table>

$^ { * 2 }$ samples were tested with InfA and H5a primer/probe sets only due to low sample volume. Both samples were positive for InfA and H5a, but were not included in the table above (CDC ref#2007700735 – China, and ref#2007700734 – China)

# Cross-Reactivity Evaluation

Analytical specificity (cross-reactivity) was evaluated by testing each primer/probe set within the device panel with nucleic acids extracted from 27 organisms (9 viruses, 17 bacteria, and 1 yeast) representing common respiratory pathogens or flora commonly present in specimens collected from the nasopharynx region. All the organisms were propagated, titered, and characterized to confirm identity prior to testing. The identity of the commensal respiratory bacteria was confirmed by 16S ribosomal RNA bi-directional sequencing. The identity of the non-influenza respiratory viruses was confirmed by bidirectional sequencing. Bacteria and yeast were tested at concentrations greater than or equal to $1 0 ^ { 6 } \mathrm { c f u / m L }$ . Non-influenza respiratory viruses were tested at concentrations greater than $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ with the exception of human parainfluenza type 2 which was tested at $1 0 ^ { 3 . 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ due to difficulty generating a high titer virus stock in culture and coronaviruses $\mathrm { C o V } 2 2 9 \mathrm { E }$ and CoV OC43 $( 3 1 . 6 \mathrm { n g / \mu l }$ and $5 0 . 4 ~ \mathrm { { n g / \mu l } }$ , respectively, of total RNA from viral culture).

Each sample (bacteria and virus) was extracted in parallel using both the Qiagen QIAamp® Viral RNA Mini Kit and the Qiagen RNeasy® Mini Kit. Each RNA sample was tested following the testing procedure previously described for the rRT-PCR Flu Panel and the Applied Biosystems 7500 Fast Real-Time PCR System. The dual extraction method was performed to demonstrate cross method equivalency and ensure no cross reactivity with the influenza primer and probe sets since negative results were expected for all samples.

Analytical Specificity (Cross-reactivity) Test Results with Common Noninfluenza Human Respiratory Pathogens   

<table><tr><td colspan="2" rowspan="1">Organism Tested</td><td colspan="8" rowspan="1">Referenced Primer &amp; Probe Set Positive (+) or Negative (-) forReactivity</td></tr><tr><td colspan="1" rowspan="1">CommonRespiratory Flora</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">cfu / ml</td><td colspan="1" rowspan="1">InfAset</td><td colspan="1" rowspan="1">Inf Bset</td><td colspan="1" rowspan="1">H1set</td><td colspan="1" rowspan="1">H3set</td><td colspan="1" rowspan="1">H5aset</td><td colspan="1" rowspan="1">H5bset</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">10 8.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Candida albicans (yeast)</td><td colspan="1" rowspan="1">2001-21-196</td><td colspan="1" rowspan="1">10 8.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">K12</td><td colspan="1" rowspan="1">10 9.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">M15709</td><td colspan="1" rowspan="1">10 6.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarium</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 8.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 10.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Moraxella cararrhalis</td><td colspan="1" rowspan="1">M15757</td><td colspan="1" rowspan="1">10 9.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis1</td><td colspan="1" rowspan="1">BCG</td><td colspan="1" rowspan="1">0.25 ng /μl</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">MI-29</td><td colspan="1" rowspan="1">10 7.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 8.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Neisseria miningitidis</td><td colspan="1" rowspan="1">M2578</td><td colspan="1" rowspan="1">10 7.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Psuedomonas aeruginosa</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 10.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 10.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">10 10.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">249-06(Thailand)</td><td colspan="1" rowspan="1">10 6.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyrogenes</td><td colspan="1" rowspan="1">7790-06</td><td colspan="1" rowspan="1">10 7.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">SS1672</td><td colspan="1" rowspan="1">10 8.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Common Non-InfluenzaRespiratory Viruses</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">TCID50 / ml</td><td colspan="1" rowspan="1">InfAset</td><td colspan="1" rowspan="1">Inf Bset</td><td colspan="1" rowspan="1">H1set</td><td colspan="1" rowspan="1">H3set</td><td colspan="1" rowspan="1">H5aset</td><td colspan="1" rowspan="1">H5bset</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Echo 6</td><td colspan="1" rowspan="1">10 6.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus, type 1</td><td colspan="1" rowspan="1">Ad.71</td><td colspan="1" rowspan="1">10 9.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus, type 7a</td><td colspan="1" rowspan="1">S-1058</td><td colspan="1" rowspan="1">10 7.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus virus1</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">50.4 ng /μl</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus virus1</td><td colspan="1" rowspan="1">299E</td><td colspan="1" rowspan="1">31.6 ng /μl</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus A</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">10 5.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza 2 virus</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">10 3.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza 3 virus</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">10 7.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial virus</td><td colspan="1" rowspan="1">CH93-18b</td><td colspan="1" rowspan="1">10 6.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Neg.</td></tr></table>

1 Organism quantitated by spectrophotometry $( \mathrm { n g / \mu l } )$

The study demonstrated that the primer and probe sets contained within the rRTPCR Flu Panel did not cross-react with any of the non-influenza respiratory pathogens or commensal organisms tested. The data demonstrated $100 \%$ concordance with the expected results.

# $f .$ Assay cut-off:

To assess the rRT-PCR Flu Panel assay cut-off, the data from the limit of detection study was analyzed. Data were plotted as virus concentration (log $\mathrm { T C I D } _ { 5 0 }$ ) versus cycle threshold (Ct) value at or below the limits of detection (in yellow) for each marker (e.g. Figure 18-1). The average $\mathrm { C t }$ value at the lower end of the range of the limit of detection plus two standards of deviation afforded the $\mathrm { C t }$ value cut-off for each of the markers in the rRT-PCR Flu Panel assay as shown by the red line $( \mathrm { I n f A } = 3 7$ ; $\mathrm { I n f B } = 3 5$ ; $\mathrm { H } 1 = 3 5$ ; $\mathrm { H } 3 = 3 6$ ; $\mathrm { H } 5 \mathrm { a } = 3 5$ ; $\mathrm { H } 5 \mathrm { b } = 3 5$ ). Particularly in the case of the InfA target (Figure 18-1); marked differences in detection between subtypes of influenza A are apparent. As the assay is designed for both seasonal and pandemic use, it is vital to detect any and all strains of influenza as the assay may detect novel influenza A strains. Therefore, the cut-off Ct value for the rRT-PCR assay was set at 37.

![](images/dd0e288808ea9de5045c45b9839134d08a74f28641e5a5a06544c1747735856b.jpg)  
Figure 18-1. Assay Cut-Off Plot for the InfA Primer and Probe Set when Compared to Three Influenza A Virus Subtypes.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable, performance of the assay was evaluated in comparison to the gold standard/reference method, viral culture followed by DFA and/or sequencing

$b$ . Matrix comparison:

Not applicable

3. Clinical studies:

Performance characteristics of the rRT-PCR Flu Panel were established during a prospective study at 4 U.S. state public health laboratories during the 2006-2007 respiratory virus season (February-April). A total of 439 (415 prospective seasonal specimens collected for routine influenza testing and 24 retrospective AH5 influenza samples) from nasal and nasopharyngeal swabs, or grown in culture in the case of influenza AH5 samples were used for this study.

The reference method was rapid culture (shell vial) followed by direct fluorescent antibody screening and identification. Specimens tested at the public health testing sites followed routine diagnostic influenza rapid culture protocols and the rRT-PCR Flu Panel protocol to demonstrate performance of the assay. Influenza AH5 and discrepant prospective seasonal influenza samples were further analyzed

by bi-directional sequencing.

a and b. Clinical Sensitivity and Specificity:

# Influenza A Comparison Results

<table><tr><td colspan="6">Virus Culture &quot;Gold Standard&quot; Results</td></tr><tr><td></td><td>Influenza</td><td>Influenza A Positive</td><td>Influenza A Negative</td><td>Total</td><td>Performance</td></tr><tr><td>Rr  P Lr</td><td>A Positive Influenza</td><td>142</td><td>21a</td><td>163</td><td>99.3 % Sensitivity (96.1%-99.9%) 95% CI</td></tr><tr><td></td><td>A Negative</td><td>1</td><td>251</td><td>252</td><td>92.3 % Specificity (88.5%-94.9%) 95% CI</td></tr><tr><td></td><td>Total</td><td>143</td><td>272</td><td>415</td><td></td></tr></table>

a 16 of the 21 samples that were influenza A positive by rRT-PCR and influenza A negative by virus culture were confirmed influenza A positive using bi-directional sequence analysis.

# Influenza H1 Comparison Results

<table><tr><td rowspan=5 colspan=1>InfluenzaR PN        A H1PositiveaandInfluenzaA H1NegativeTotal</td><td rowspan=1 colspan=4>Virus Culture &quot;Gold Standard&quot; Results</td></tr><tr><td rowspan=1 colspan=4>Influenza A  Influenza ATotal         PerformanceH1 Positive  H1 Negative</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>93.1 % Sensitivity(78.0%-98.1%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>384</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>99.5 % Specificity(98.1%-99.9%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1></td></tr></table>

$^ \mathrm { ~ a ~ } _ { 1 }$ of the 2 samples that were H1 positive by rRT-PCR and H1 negative by virus culture was confirmed H1 positive using bi-directional sequence analysis.

# Influenza H3 Comparison Results

<table><tr><td rowspan=5 colspan=1>InfluenzaR P N       A H3PositivePancInfluenzaA H3NegativeTotal</td><td rowspan=1 colspan=4>Virus Culture &quot;Gold Standard&quot; Results</td></tr><tr><td rowspan=1 colspan=1>Influenza AH3 Positive</td><td rowspan=1 colspan=1>InfluenzaA H3Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Performance</td></tr><tr><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>30a</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>100 % Sensitivity(96.2%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>287</td><td rowspan=1 colspan=1>90.5 % Specificity(86.7%-93.3%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>317</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1></td></tr></table>

a 28 of the 30 samples that were H3 positive by rRT-PCR and H3 negative by virus culture were confirmed H3 positive using bi-directional sequence analysis.

Influenza B Comparison Results   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>Virus Culture &quot;Gold Standard&quot; Results</td></tr><tr><td rowspan=1 colspan=1>Influenza BPositive</td><td rowspan=1 colspan=3>InfluenzaTotal           PerformanceB Negative</td></tr><tr><td rowspan=3 colspan=1>InfluenzaRr  P L   B PositiveInfluenzaB NegativeTotal</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>97.6 % Sensitivity(87.4%-99.6%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>367</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>98.1 % Specificity(96.2%-99.1%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>374</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1></td></tr></table>

$^ \mathrm { a } \ 6$ of the 7 samples that were influenza B positive by rRT-PCR and influenza B negative by virus culture were confirmed influenza B positive using bi-directional sequence analysis.

Clinical Sensitivity and Specificity Performance Summary - Prospective Specimen Testing   

<table><tr><td>Analyte Tested</td><td>Sensitivity % (95% CI)</td><td>Specificity % (95% CI)</td></tr><tr><td>InfA</td><td>99.3 (96.1 - 99.9)</td><td>92.3 (88.5 - 94.9)</td></tr><tr><td>InfB</td><td>97.6 (87.4 - 99.6)</td><td>98.1 (96.2 - 99.1)</td></tr><tr><td>H1</td><td>93.1 (78.0 - 98.1)</td><td>99.5 (98.1 - 99.9)</td></tr><tr><td>H3</td><td>100.0 (96.2 - 100.0)</td><td>90.5 (86.7 - 93.3)</td></tr></table>

Due to the absence of suspect cases within the United States, a formal prospective clinical evaluation of human and/or avian influenza A/H5N1 virus was not accomplished during the CDC Clinical Evaluation Study. Therefore, the performance specificity and detection capability of the H5a and H5b primer and probe sets in the rRT-PCR Flu Panel could not be evaluated at the U.S. state public health laboratories during the clinical study. Retrospective data from twenty–four (24) suspect A/H5N1 positive specimens from human cases received by the CDC Influenza Division were used to demonstrate clinical performance of the H5a and H5b components of the panel.

Retrospective Influenza A/H5N1 Data – Culture and Clinical Influenza Samples Combined H5a and H5b Analytes   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Reference Method Results</td></tr><tr><td rowspan=1 colspan=1>InfluenzaAH5Positive</td><td rowspan=1 colspan=1>InfluenzaAH5Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Performance</td></tr><tr><td rowspan=4 colspan=1>Rr  P LRES</td><td rowspan=1 colspan=1>Influenza H5a &amp; H5bPositive Clinical Specimen</td><td rowspan=1 colspan=1>5a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>100 % Percent PositiveAgreement(56.6% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Influenza H5a &amp; H5bPositive CulturedSpecimen</td><td rowspan=1 colspan=1>19a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>100 % Percent PositiveAgreement(83.2% -100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Prospective ClinicalSpecimen</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>100 % Percent NegativeAgreement(99.1%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>415</td><td rowspan=1 colspan=1>439</td><td rowspan=1 colspan=1></td></tr></table>

a Five clinical specimens and 19 specimens tested positive for influenza AH5 directly from the specimen source in the country of origin prior to receipt were included in the study. CDC Influenza Division tested grown virus (culture) from the country of origin and then propagated virus in culture for further confirmatory testing.

Performance Summary - Influenza A/H5N1 Testing with Prospective and Retrospective Specimens   

<table><tr><td></td><td>Analyte Tested</td><td>Percent Positive Agreement (%)</td></tr><tr><td>Influenza A/H5 Positive Clinical Specimens</td><td>H5a / H5b</td><td>100 % Percent Positive Agreement (56.6% -100%) 95% CI</td></tr><tr><td>Influenza A/H5 Positive Cultured Specimens</td><td>H5a / H5b</td><td>100 % Percent Positive Agreement (83.2% -100%) 95% CI</td></tr><tr><td>Prospective Clinical Specimens</td><td>H5a / H5b</td><td>100 % Percent Negative Agreement (99.1%-100%) 95% CI</td></tr></table>

4. Clinical cut-off:

Not applicable

Expected values/Reference range: During October 1, 2006--May 19, 2007, World Health Organization and National Respiratory and Enteric Virus Surveillance System (NREVSS) collaborating laboratories in the United States tested 179,268 respiratory specimens for influenza viruses; 23,753 $( 1 3 . 2 \% )$ were positive. Of these, 18,817 $( 7 9 . 2 \% )$ were influenza A viruses and 4,936 $( 2 0 . 8 \% )$ were influenza B viruses. Among the influenza A viruses, 6,280 $( 3 3 . 4 \% )$ were subtyped; 3,912 $( 6 2 . 3 \% )$ were influenza A/H1 viruses and 2,368 $( 3 7 . 7 \% )$ were influenza A/H3 viruses

(http://www.cdc.gov/mmwr/preview/mmwrhtmlml/mm5631a2.htm). In the rRTPCR Flu Panel multi-center prospective clinical study during the 2006-2007 influenza season, the prevalence as determined by virus culture was as follows: influenza A/H1 $( 7 . 0 \% )$ , influenza $\mathrm { A } / \mathrm { H } 3$ $( 2 3 . 6 \% )$ , and influenza B $( 9 . 9 \% )$ . No dual infections were found in this study. It is recommended that any sample that tests positive for more than one influenza virus be re-tested.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.