# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K081868   
B. Purpose for Submission: To obtain a $5 1 0 ( \mathrm { k } )$ clearance for a new device.   
C. Measurand: Leishmania species DNA and L. major DNA   
D. Type of Test: Real-time polymerase chain reaction assay

# E. Applicant:

Department of the Army   
U.S. Army Medical Research and Matériel Command   
1430Veterans Drive   
Fort Detrick, MD 21702-5012

F. Proprietary and Established Names: SMART Leish

# G. Regulatory Information:

1. Regulation section: 866.3870 Trypanosoma spp. serological reagents

2. Classification: Class I

3. Product code: OUZ

4. Panel: Microbiology 83

# H. Intended Use:

1. Intended use:

The SMART Leish is a qualitative diagnostic real-time PCR test for the rapid detection of Leishmania species and the identification of $L$ . major in skin lesion scrapings and punch biopsies from individuals suspected of having cutaneous leishmaniasis. The test utilizes real-time polymerase chain reaction assay on the Cepheid SmartCycler® II Dx to detect Leishmania species and $L$ . major.

The SMART Leish is intended for use only by trained laboratory personnel in Department of Defense laboratories. Clinical performance has not been established with strains other than $L$ . major.

2. Indications for use:

The SMART Leish is a qualitative diagnostic real-time PCR test for the rapid detection of Leishmania species and the identification of $L$ . major in skin lesion scrapings and punch biopsies from individuals suspected of having cutaneous leishmaniasis. The test utilizes real-time polymerase chain reaction assay on the Cepheid SmartCycler® II Dx to detect Leishmania species and $L$ . major.

The SMART Leish is intended for use only by trained laboratory personnel in Department of Defense laboratories. Clinical performance has not been established with strains other than $L$ . major.

3. Special conditions for use statement(s): The device is for prescription use only.

4. Special instrument requirements: Cepheid SmartCycler® II Dx

# I. Device Description:

The SMART Leish consists of an assay reagent kit and assay definition files for the polymerase chain reaction (PCR) instrument platform. The kit contains sufficient reagents, in lyophilized bead form, to qualitatively assay 50 clinical samples for both Leishmania genus and $L$ . major targets. Additional required accessories that are specified include the PCR instrument platform, a deoxyribonucleic acid (DNA) purification kit, and a positive extraction control.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Amizyme-Leishmania spp. Test Kit

2. Predicate 510(k) number(s):

K842526

3. Comparison with predicate:

# Similarities

<table><tr><td rowspan=1 colspan=1>Device Aspect</td><td rowspan=1 colspan=1>SMART Leish</td><td rowspan=1 colspan=1>Reference Device</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Detection of Leishmania spp.</td><td rowspan=1 colspan=1>Detection of Leishmania spp.</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Agent measured</td><td rowspan=1 colspan=1>Leishmania genus and L. majorspecific DNA</td><td rowspan=1 colspan=1>Leishmania spp.</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Qiagen QIAamp DNA Mini Kit</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>External  positive and negativecontrols for both the extractionprocess and detection assay runsInternal  positive control for PCRinhibitors in sample</td><td rowspan=1 colspan=1>External positive and negativecontrols</td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Instrument Platform</td><td rowspan=1 colspan=1>Cepheid SmartCycler</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Automation</td><td rowspan=1 colspan=1>Automated DNA amplificationcycling, probe detection, andinterpretation of results</td><td rowspan=1 colspan=1>Manual method</td></tr><tr><td rowspan=1 colspan=1>Sample Collection</td><td rowspan=1 colspan=1>Skin punch biopsy or scrapings,stored in ethanol</td><td rowspan=1 colspan=1>Serum</td></tr><tr><td rowspan=1 colspan=1>Endpoint detectionmethod</td><td rowspan=1 colspan=1>PCR amplification of DNAsequences unique to the organismsof interest with detection byhybridization probes incorporatingreporter dyes</td><td rowspan=1 colspan=1>Indirect Immunofluorescence(IFA) Fluorescent conjugatesecondary antibodies on aprepared slide</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

# L. Test Principle:

For the Leishmania assays, a tissue specimen (skin scraping or punch biopsy) from an individual suspected of being infected with Leishmania species or L. major is collected in $70 \%$ or $100 \%$ ethanol, and the DNA is extracted from the specimen using the Qiagen QIAamp DNA Mini Kit. An aliquot of this DNA is tested using the Leishmania genus assay, which will amplify a portion of DNA encoding the Leishmania species 16S ribosomal ribonucleic acid (rRNA) gene if present. Amplified targets are detected using a TaqMan® hybridization probe with 6-carboxyfluorescein (FAM) reporter dye and a Black Hole Quencher® (BHQ). This assay also contains a positive internal control consisting of a nonsensical, non-naturally

occurring DNA sequence, with Texas Red reporter dye and BHQ, used to detect evidence of PCR inhibition and confirm the integrity of assay reagents in negative specimens. If the sample tests positive for Leishmania species, another aliquot of DNA may be tested using the $L$ . major assay, which will amplify a portion of DNA encoding the GPI gene that is specific to $L$ . major. Amplified targets are detected using a TaqMan hybridization probe with FAM reporter dye and a BHQ. The DNA amplification, detection of fluorescence, and interpretation of signals are done automatically by the SmartCycler instrument for each assay. The thermocycling protocols take approximately 45–55 minutes.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Studies were conducted to evaluate the reproducibility of the SMART Leish assay. The precision of the Leishmania genus and L. major assays between laboratories was evaluated using purified L. major DNA and cultured L. major parasites. Additionally, between-laboratory precision of the assays was evaluated using mock human samples (punch biopsies from porcine tissue injected with L. major parasites).

Reproducibility Results for the Leishmania Genus Assay   

<table><tr><td colspan="1" rowspan="1">SampleType</td><td colspan="1" rowspan="1">SampleID</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Number ofReplicates(over 5days</td><td colspan="1" rowspan="1">(WRAIR)</td><td colspan="1" rowspan="1">(BAMC)</td><td colspan="1" rowspan="1">(MAMC)</td><td colspan="1" rowspan="1">TotalAgreement</td><td colspan="1" rowspan="1">% TotalAgreement</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">genomeequivalents/μL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">PurifiedDNA</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PositiveAgreementfor PurifiedDNA</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(parasites/extraction)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CulturedParasites</td><td colspan="1" rowspan="1">Low</td><td colspan="1" rowspan="1">1,000</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Medium</td><td colspan="1" rowspan="1">10,000</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">High</td><td colspan="1" rowspan="1">25,000</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PositiveAgreementforCulturedParasites</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(parasites/extraction)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MockHumanSamples</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">10/10</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">1.1x10^6</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">13/14</td><td colspan="1" rowspan="1">14/14</td><td colspan="1" rowspan="1">41/42</td><td colspan="1" rowspan="1">97.6%</td></tr></table>

# b. Linearity/assay reportable range:

The assay linearity of SMART Leish for Leishmania genus and Leishmania major was determined using eight different concentrations of purified L. major DNA tested in a 10-fold dilution series from $1 4 . 2 6 \mathrm { n g }$ to 0.001426 pg and $2 1 . 3 9 \mathrm { n g }$ to 0.002139 pg, respectively, DNA per reaction.

c. Traceability, Stability, Expected values (controls, calibrators, or methods): Not applicable

d. Detection limit:

In addition to the LOD determination, six different concentrations of purified L. major parasites were tested in a 10-fold dilution of $5 \mathrm { ~ X ~ } 1 0 ^ { 2 }$ parasites/mL (equivalent to $1 0 ^ { 7 }$ to $1 0 ^ { 2 }$ parasites/extraction) to determine the minimum number of L. major parasites that can be taken through the extraction and testing procedure and still give a positive result for the SMART Leish assays (extraction LOD). The following tables summarize the results of the assay LOD for the L. genus and L. major assays, and summarizes the results of the extraction LOD (eLOD) study.

Summary of the Measuring Range and LOD for the Leishmania genus and L. major Assays (Values are per assay reaction)   

<table><tr><td rowspan=1 colspan=2>Assay                        Linear Range1</td><td rowspan=1 colspan=1>Assay LOD1,2</td><td rowspan=1 colspan=1>LOD Equates to</td></tr><tr><td rowspan=1 colspan=1>Leishmania Genus Assay</td><td rowspan=1 colspan=1>0.014-14,000 pg</td><td rowspan=1 colspan=1>0.14 pg</td><td rowspan=1 colspan=1>4 genome copieswith ~200copies/genome oftarget gene</td></tr><tr><td rowspan=1 colspan=1>L. major Assay</td><td rowspan=1 colspan=1>0.213921,390 pg</td><td rowspan=1 colspan=1>2.1 pg</td><td rowspan=1 colspan=1>62 genome copies</td></tr></table>

1 Amount of L. major DNA in the PCR reaction. LOD refers to the minimum amount of L. major DNA that can be put into a SMART Leish PCR reaction and still result in a positive SMART Leish test greater than or equal to $9 5 \%$ of the time. $2 \text{‰}$ accuracy across different operators and instruments and from multiple L. major strains.

Summary of Assay eLOD for the Leishmania genus and L. major Assays Using Parasite Samples. Values are per extraction.

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>eLoD1                           LOD Equates to2</td></tr><tr><td rowspan=1 colspan=1>Leishmania GenusAssay</td><td rowspan=1 colspan=1>250 parasites</td><td rowspan=1 colspan=1>~5 genome copies atassay reaction stage</td></tr><tr><td rowspan=1 colspan=1>L. major Assay</td><td rowspan=1 colspan=1>1,000 parasites</td><td rowspan=1 colspan=1>~30 genome copies atassay reaction stage</td></tr></table>

$1 \%$ accuracy across different operators and days and from multiple L. major strains.

Extraction LOD refers to the minimum number of L. major parasites that can be taken through the Qiagen extraction procedure and still consistently result in a positive SMART Leish test greater than or equal to $9 5 \%$ of the time. Extraction LOD was determined across different operators, different days, and using multiple L. major strains.

# e. Analytical specificity:

A total of 121 DNA samples were evaluated in the specificity study. The DNA from 11 Leishmania major strains tested in this study gave robust positive results with the Leishmania major PCR assay. Twenty-five nonLeishmania major samples tested negative; hence the analytical specificity was $100 \%$ . The percentage of false positives was $0 \%$ , and the percentage of false negatives is $0 \%$ . In the Leishmania genus study, 85 of the 121 samples were non-target DNA samples. Eighty-five were negative, 2 false positives and 36 were positive. The analytical specificity of the Leishmania genus PCR assay was $9 8 . 3 \%$ , false positive rate was $2 . 4 \%$ and the false negative rate was $0 \%$ . See the following table for a list of organisms used in the study. Nucleic acids from non-target organisms were tested for both the Leishmania genus and Leishmania major assays. DNA samples with concentrations used in this study included purified DNA from bacteria (52; conc $= 5 0 - 4 6 { , } 2 5 0 \mathrm { p g / u L } )$ , fungi (11; co $\mathrm { \Delta \mathfrak { c } = 5 0 , 0 0 0 - 9 7 , 0 0 0 p g / u L }$ ), viruses $( 7 ; \mathrm { c o n c } = 5 , 9 5 0 - 3 0 , 9 9 0$ $\mathrm { p g / u L ) }$ , mammals (3; conc $= 2 0 \mathrm { p g / u L }$ (human, bovine, and murine)), human melanoma cell lines (3; conc $= 1 0 , 2 0 0 - 2 5 , 2 0 0 \mathrm { p g / u L } )$ , Leishmania major (11; conc $= 2 1 , 5 4 0 - 8 3 , 0 0 0 \mathrm { p g / u L } )$ ), 25 additional Leishmania species (not- $L$ . major) (conc $= 1 6 , 6 8 0 - 5 9 2 , 0 0 \mathrm { p g / u L ) }$ , and 9 trypanosomes representing 6 species (conc $= 1 2 , 7 6 0 - 9 5 , 0 0 0 \mathrm { p g / u L ) }$ .

List of Organisms Used in Specificity Study   

<table><tr><td>Bacteria Acinetobacter baumanni</td><td>Staphylococcus aureus (4) Staphylococcus hominis Stenotrophomonas maltophilia Streptococcus pyogenes (2) Streptococcus sp. (B) Streptococcus (F2)</td><td>Trichophyton mentagrophytes Trichophyton soudanense Arthroderma benhamiae Sporothrix schenckii</td></tr><tr><td>Bacillus anthracis (3) Bacillus cereus (2) Bacillus subtilis var.niger Bacillus thuringiensis</td><td></td><td>Microsporum canis</td></tr><tr><td>Brucella abortus Brucella melitiensis Brucella susi</td><td>Yesinia pestis Yersinia</td><td>Mammalian Bovine</td></tr><tr><td>Clostridium botulinum type A</td><td>Mycobacteria Mycobacteria abscessus</td><td>Human Murine</td></tr><tr><td>Clostridium botulinum type B</td><td>Mycobacteria fortuitum Mycobacteria marinum Mycobacteria spp Mycobacteria tuberculosis</td><td>Human melanoma cell line (3)</td></tr><tr><td>Clostridium perfringens (2) Clostridium sordelli</td><td>Mycobacteria ulcerans</td><td>Trypanosoma</td></tr><tr><td>Entrbacter aerogenes Enterococcus durans</td><td>Viruses Human Papilloma Virus</td><td>Trypanosoma cruzi (2)</td></tr></table>

Two strains of $C .$ . fasciculata were determined to be low-level cross-reactors based on false-positive results observed. The degree of cross-reactivity of the Leishmania genus assay for $C .$ . fasciculata can be considered low level or weak. No cross-reactors were determined for the $L$ . major assay through the study.

Supplementary in silico analysis was done comparing the Smart Leish primer and probe sequences to sequences from the following organisms that could potentially result in a clinical presentation similar to cutaneous leishmaniasis: Corynebacterium diphtheria (veldt sores), Mycobacteria (Tropical Ulcer), Mycobacterium tuberculosis (Lupus vulgaris), Treponema pallidum (Tertiary Syphilis), Treponema pertenue (Yaws), and Blastomyces (Blastomycosis). Each Smart Leish primer and probe contained at least four mismatches to those sequences. Based on these alignments, no cross-reactivity was predicted with the Smart Leish primers and probes.

f. Assay cut-off: Not applicable

2. Comparison studies:

a. Method comparison with predicate device: Clinical performance of the SMART Leish was compared to culture and microscopy.   
b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the SMART Leish assay was established in a multi-center, retrospective clinical study conducted at two military U.S. hospital sites. A total of 303 prospective specimens were collected from adults 18 years or older and evaluated in the SMART Leish assay and compared to culture and/or microscopy. Evaluated specimens were skin lesion scrapings and punch biopsies from individuals suspected of having cutaneous leishmaniasis. The patient population consisted entirely of military personnel who were at the time, or had previously been, deployed to Southwest Asia— primarily Afghanistan and Iraq. SMART Leish performance versus culture and microcopy, including $9 5 \%$ confidence intervals, is detailed below. For the BAMC study, DNA was extracted from specimens and Leishmania genus and $L$ . major assays were run. For the WRAIR study, Leishmania genus and $L$ . major assays were run on archived DNA samples that were previously extracted from clinical specimens. For both studies, the results were compared to “clinical truth” that had been established for each specimen or sample.

# Summary of Smart Leish WRAIR Studies for the

# Leishmania genus Assay

<table><tr><td colspan="2"></td><td colspan="3">Clinical Truth</td></tr><tr><td colspan="2">Smart Leish -</td><td>Positive</td><td>Negative</td><td>Total</td></tr><tr><td rowspan="2">Leishmania Genus Assay</td><td>Positive Negative</td><td>160</td><td>29</td><td>189</td></tr><tr><td></td><td>1</td><td>56</td><td>57</td></tr><tr><td></td><td>Total</td><td>161</td><td>85</td><td>246</td></tr><tr><td colspan="3">Sensitivity = 99.4% (96.1% - 99.9%)</td><td></td></tr><tr><td colspan="3">Specificity = 65.9% (54.7% - 75.6%)</td><td></td></tr></table>

Summary of Smart Leish WRAIR Studies for the Leishmania major Assay   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=3>Clinical TruthPositive       Negative          Total</td></tr><tr><td rowspan=2 colspan=1>Smart Leish - L.major</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=5>Sensitivity = 94.6% (86.0% - 98.3%)Specificity = 90.6% (81.8% - 95.6%)</td></tr></table>

Summary of Smart Leish BAMC Studies for the Leishmania genus Assay   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>Clinical TruthNegative          Total</td></tr><tr><td rowspan=2 colspan=1>Smart Leish LeishmaniaGenus</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=5>Sensitivity = 98.4% (90.5% - 99.9%)Specificity = 100% (19.8% - 100%)</td></tr></table>

Summary of Smart Leish BAMC Studies for the Leishmania major Assay   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Clinical Truth</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Smart Leish - L.major</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=5>Sensitivity = 100.0% (81.5% - 100.0%)Specificity = 100.0% (51.7% - 100.0%)</td></tr></table>

# Summary of SMART Leish Combined Studies for the Leishmania genus Assay

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>Clinical Truth</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>SMART Leish LeishmaniaGenus Assay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>252</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>312</td></tr><tr><td rowspan=1 colspan=5>*Sensitivity = (99.1% (96.5  99.8)*Specificity = (67% (55% - 76%)</td></tr></table>

Diagnosis of leishmaniasis by culture and slide techniques results in lower clinical sensitivity than PCR assays $4 6 \% 8 4 \%$ and $3 3 \% - 7 4 \%$ respectively). In this clinical data set there were 29 false positives generated for the Leishmania Genus assay. Twenty seven of these Genus false positives were confirmed to be Genus positive using sequencing methods. Clinical and specificity values were recalculated counting the 27 Genus false positives that were confirmed by sequencing to be Leishmania as true positives. These “adjusted” calculation results are as follows: Sensitivity Genus: $9 9 . 2 \%$ (96.9 - 99.9); Specificity Genus: $9 6 . 7 \%$ $( 8 7 . 5 - 9 9 . 9 )$ . \* Diagnosis of leishmaniasis by culture and slide techniques results in lower clinical sensitivity than PCR assays $4 6 \% 8 4 \%$ and $3 3 \% - 7 4 \%$ respectively)

# Summary of SMART Leish Combined Studies for the Leishmania major Assay

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Clinical Truth</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>SMART Leish L. majorAssay</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>92</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=2>4</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>87</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=2>96</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>187</td></tr><tr><td rowspan=1 colspan=6>*Sensitivity = = 95.8% (89.1 - 98.7)Specificity == 91.2% (82.9 - 95.9)</td></tr></table>

In this clinical data set there were eight false positives generated for the $L$ . major assay, of which three of were confirmed to be $L$ . major using sequencing methods. Clinical and specificity values were recalculated counting the 27 three $L$ . major false positives that were confirmed by sequencing to be $L$ . major as true positives. These “adjusted” calculation results are as follows: Sensitivity $L$ . major: $9 6 . 0 \%$ (89.4 - 98.7); Specificity $L$ . major: $9 4 . 3 \%$ (86.6 - 97.9).

$b$ . Clinical specificity: Not applicable

c. Other clinical supportive data (when a. and b. are not applicable): None

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the clinical studies conducted, for this study, on U.S. military personnel that served in the Middle East and Afghanistan, $5 7 \%$ of 2,783 samples tested using an in-house developed PCR assay in an accredited clinical laboratory were positive for Leishmania major. In the investigational study of the Smart Leish assay, $72 \%$ of 312 skin scrapings and punch biopsy specimens from U.S. service members that served in the Middle East and Afghanistan, and which had a skin lesion consistent with cutaneous leishmaniasis, tested positive compared to the gold standard of culture and/or microscopy.

# N. Instrument Name:

SmartCycler Dx System

# O. System Descriptions:

The SMART Leish consists of an assay reagent kit and assay definition files for the polymerase chain reaction (PCR) instrument platform. The kit contains sufficient reagents, in lyophilized bead form, to qualitatively assay 50 clinical samples for both Leishmania genus and L. major targets. Additional required accessories that are specified include the PCR instrument platform, a deoxyribonucleic acid (DNA) purification kit, and a positive extraction control. The device components and required accessories are listed as follows.

The SMART Leish Assay contains the following reagents packaged in a test kit for detection of Leishmania DNA.

• SmartMix™ HM Lyophilized PCR Master Mix • Leishmania Genus Primer and Probe Set • Leishmania major Primer and Probe Set • Leishmania Positive and Negative Control DNA Beads

Modes of Operation:

The assay is performed on the automated Cepheid SmartCycler Dx System. The SmartCycler Dx System is a real time thermal cycler used for identifying DNA or RNA from prepared samples. Each instrument contains 16 independently programmable I-CORE modules, each with one reaction site. Thermally optimized proprietary reaction tubes combined with the unique design of the ICORE modules allow very rapid cycling and faster amplification and detection. Up to six SmartCycler II processing blocks can be daisy-chained together,

allowing simultaneous analysis of up to 96 discrete samples.

The SmartCycler Dx System is suited to in vitro diagnostic assays that require automatic, repeated, rapid heating and cooling cycles for test samples, such as PCR and RT-PCR. Specific sequences can be detected using hybridization probes. The system has the capacity to store any number of runs, limited only by available disk space. All assay data and results are stored in a database.

# Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes _x or No   
The software controls the operation of the I-CORE module, and collects, analyzes and interprets the acquired optical data. This software has been determined to be a moderate level of concern.

3. Specimen Identification:

• User enters the sample identifiers under the “Sample ID Column” before starting the run. • User inserts each reaction tube into the assigned site of the SmartCycler.

4. Specimen Sampling and Handling:

• Collected specimens should be stored in $70 \% - 1 0 0 \%$ ethanol. During shipment, the samples can be kept at ambient temperature, $2 0 ^ { 0 } \mathrm { C } - 2 9 ^ { \overline { { 0 } } } \mathrm { C }$ . • Once received, samples should be stored at $2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C }$ until tested. Extracted DNA samples must be stored in the AE buffer provided in the Qiagen kit and may be stored at $2 ^ { \circ } \mathrm { C } { - } 8 ^ { \circ } \mathrm { C }$ for up to 3 days. DNA extracts in AE buffer may be stored for more than 3 days in a $- 2 0 ^ { \circ } \mathrm { C }$ freezer. Avoid multiple freeze–thaw cycles of the samples.

5. Calibration:

The thermistors used to monitor the reaction chamber temperature are calibrated to   
$\pm 0 . 5 0 ^ { \circ } \mathrm { C }$ using National Institute of Standards and Technology (NIST)-traceable standards. The optical system is calibrated using standard concentrations of the individual unquenched dye-oligo standard to determine the spectral   
characteristics.

The instrument performs optics self-test before each run start to verify that the optical system is functioning properly.

# 6. Quality Control:

Positive DNA controls are provided in the assay kit. The positive extraction control (Leishmania mexicana parasites) and negative controls (molecular biology grade water) are materials required but not supplied. One positive DNA Control and one negative DNA Control are processed for each assay run on the SmartCycler Dx System. The software automatically assigns the positive DNA Control to the second to last position, and the negative DNA control to the last position.

The positive DNA control monitors for reagent failure and operator error. The negative DNA control monitors reagent or environmental contamination (or carry-over). The positive extraction control monitors for extraction failure and operator error. The negative extraction control monitors for reagent contamination and environmental contamination in the extraction process.

The internal control (IC) verifies functional PCR reagents and the absence of inhibition   
that would prevent PCR amplification. DNA for the IC is included in one of the reagent   
beads within the Smart Leish Assay. The response of the IC in the presence of inhibitors correlates with the response of target DNA in the presence of inhibitors. Before start of PCR reaction, the SmartCycler Dx System is programmed to perform an   
optical measurement or probe check on the optical channels associated with Leishmania target and Internal Control detection. The Probe Check control verifies reagent bead   
rehydration, appropriate tube filling, probe integrity, and dye stability. Probe Check is   
considered to PASS if optical measurements meet the validated acceptance criteria. If the   
Probe Check fails in either optical channel, the test will not continue. The System Control Check for Temperature Control ensures that the SmartCycler Dx   
Instrument is operating within specification.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.