# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K081824

B. Purpose for Submission: To add the option for automated extraction of DNA from TM clinical specimens using the software accessory “BD ProbeTec Chlamydia trachomatis (CT) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay” to direct the BD Viper™ System.

C. Measurand: Chlamydia trachomatis DNA

D. Type of Test: Strand displacement nucleic acid amplification assay

E. Applicant: BD Diagnostic System

F. Proprietary and Established Names: BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>MKZ</td><td rowspan=1 colspan=1>Class I</td><td rowspan=1 colspan=1>21CFR 866.3120 Chlamydiaserological reagents</td><td rowspan=1 colspan=1>Microbiology(83)</td></tr></table>

# H. Intended Use:

1.) Intended use: The BD ProbeTecCT $\boldsymbol { \mathrm { Q } } ^ { X }$ Amplified DNA Assay, when tested with the BD ViperTM System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Chlamydia trachomatis DNA in clinician-collected female endocervical and male urethral swab specimens, patientcollected vaginal swab specimens (in a clinical setting), and male and female urine specimens. The assay is indicated for use with asymptomatic and symptomatic individuals to aid in the diagnosis of chlamydial urogenital disease.

2.) Indications for use: Same as intended use

3) Special conditions for use statement(s): NA

TM 4) Special instrument requirements: BD Viper  System

I. Device Description: The BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe (8, 9). The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermallycontrolled fluorescent readers. The presence or absence of $C .$ . trachomatis DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified $C .$ . trachomatis target DNA, a second fluorescently-labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the C. trachomatis-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and $C .$ . trachomatis-specific signals to report specimen results as positive, negative, or EC failure.

# J. Substantial Equivalence Information:

a) Predicate device name (s): BD ProbeTecET CT/GC Amplified DNA Assay APTIMA Combo 2 Assay   
b) Predicate Numbers (s): K984631, K003395

Comparison with predicate:

Comparison of Operating and Technological Characteristics: Assay   

<table><tr><td colspan="1" rowspan="1">BD ProbeTecCTQ Assay</td><td colspan="1" rowspan="1">BD ProbeTecCTQ Assay</td><td colspan="1" rowspan="1">BD ProbeTecET CT/GCAssay (K984361)</td></tr><tr><td colspan="1" rowspan="1">Amplification Technology</td><td colspan="1" rowspan="1">Same as BD ProbeTecETCT/GC (K984631)</td><td colspan="1" rowspan="1">Strand DisplacementAmplification</td></tr><tr><td colspan="3" rowspan="1">Priming Microwell</td></tr><tr><td colspan="1" rowspan="1">Primers</td><td colspan="1" rowspan="1">Alternate region of crypticCT plasmid</td><td colspan="1" rowspan="1">Region of cryptic CTplasmid</td></tr><tr><td colspan="1" rowspan="1">Detector</td><td colspan="1" rowspan="1">Linear DetectorFlourescein (fluorophore)•Dabcyl (quencher)</td><td colspan="1" rowspan="1">Hairpin DetectorFlourescein (fluorophore)•Rhodamine (quencher)</td></tr><tr><td colspan="1" rowspan="1">Nucleotides</td><td colspan="1" rowspan="1">4 of 4 nucleotides requiredfor SDA</td><td colspan="1" rowspan="1">1 of 4 nucleotides requiredfor SDA</td></tr><tr><td colspan="1" rowspan="1">Non-specific reagents andcofactors</td><td colspan="1" rowspan="1">Same as BD ProbeTecETCT/GC (K984631)</td><td colspan="1" rowspan="1">Buffering components,magnesium ions, salt andstabilizing reagents</td></tr><tr><td colspan="3" rowspan="1">Amplification Microwell</td></tr><tr><td colspan="1" rowspan="1">Restriction Enzyme</td><td colspan="1" rowspan="1">Same as BD ProbeTecETCT/GC (K984631)</td><td colspan="1" rowspan="1">BsoBI restriction enzyme</td></tr><tr><td colspan="1" rowspan="1">Polymerase</td><td colspan="1" rowspan="1">Same as BD ProbeTecETCT/GC (K984631)</td><td colspan="1" rowspan="1">Bst DNA polymerase</td></tr><tr><td colspan="1" rowspan="1">Nucleotides</td><td colspan="1" rowspan="1">0 of 4 nucleotides requiredfor SDA</td><td colspan="1" rowspan="1">3 of 4 nucleotides requiredfor SDA</td></tr><tr><td colspan="1" rowspan="1">Non-specific reagents andcofactors</td><td colspan="1" rowspan="1">Same as BD ProbeTecETCT/GC (K984631)</td><td colspan="1" rowspan="1">Buffering components,magnesium ions, salt andstabilizing reagents</td></tr><tr><td colspan="1" rowspan="1">Assay Buffer</td><td colspan="1" rowspan="1">Bicine-potassiumhydroxide-based</td><td colspan="1" rowspan="1">Potassium phosphate-based</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005. http://www.fda.gov/cdrh/ode/guidance/337.pdf

2. CLSI EP5-A2 “Evaluation of Precision Performance of Quantitative Measurement Methods”,

3. CLSI EP12-A “User Protocol for Evaluation of Qualitative Test Performance”.

# L. Test Principle:

See Device Description

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

Reproducibility of the BD Viper System using the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Simulated endocervical and urethral specimens contained a clean endocervical swab whereas the simulated urine and vaginal swab specimens did not. Un-inoculated swab diluent for the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in the following table.

Summary of Reproducibility Data on the BD Viper System for the CT $\mathbf { Q } ^ { x }$ Assay.   

<table><tr><td rowspan=1 colspan=6>cYanmaGYellowb</td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRuns WithinSite</td><td rowspan=1 colspan=2>Between Site</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>CTEB&#x27;s/mL</td><td rowspan=1 colspan=1>GCCells/mL</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MaxRFUMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=5 colspan=1>Endocervical/ Urethral</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5%(133/135)</td><td rowspan=1 colspan=1>(94.8-99.8%</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>233.0</td><td rowspan=1 colspan=1>778.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>113.4</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>2011.2</td><td rowspan=1 colspan=1>114.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>442.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>76.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-10.0%)</td><td rowspan=1 colspan=1>1991.9</td><td rowspan=1 colspan=1>118.0</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1954.8</td><td rowspan=1 colspan=1>169.4</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=5 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>542.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1999.8</td><td rowspan=1 colspan=1>131.8</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>442.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1995.2</td><td rowspan=1 colspan=1>125.8</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>2014.4</td><td rowspan=1 colspan=1>109.5</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr></table>

A second study was conducted internally to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LOD) of the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into $\mathrm { Q } ^ { \mathrm { x } }$ swab diluent at two different levels each of which was below the respective analytical LOD for the organisms (1:10, 1:100). These levels were selected to fall within the dynamic range of the analytical LOD curve of the assay. Fifteen replicates of each panel member were tested every day for five days across three BD Viper Systems. The data are summarized in the following table.

Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay.   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Dilutionof AnalyticalLOD</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>MaxRFUMean(Positive)</td><td rowspan=1 colspan=1>%Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>Max RFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>70.2(158/225)</td><td rowspan=1 colspan=1>(63.8, 76.1)</td><td rowspan=1 colspan=1>1794.2</td><td rowspan=1 colspan=1>29.8(67/225)</td><td rowspan=1 colspan=1>(23.9, 36.2)</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:100</td><td rowspan=2 colspan=1>10.2(23/225)64.4(145/225)</td><td rowspan=2 colspan=1>(6.6,14.9)(57.8, 70.7)</td><td rowspan=2 colspan=1>1643.81733.9</td><td rowspan=2 colspan=1>89.8(202/225)35.6(80/225)</td><td rowspan=2 colspan=1>(85.1, 93.4)(29.3, 42.2)</td><td rowspan=2 colspan=1>1.64.6</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:10</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>10.7(24/225)</td><td rowspan=1 colspan=1>(7.0, 15.5)</td><td rowspan=1 colspan=1>1666.6</td><td rowspan=1 colspan=1>89.3(201/225)</td><td rowspan=1 colspan=1>(84.5, 93.0)</td><td rowspan=1 colspan=1>2.4</td></tr></table>

# $b$ . Linearity/assay reportable range: NA

c. Traceability, Stability, Expected values (controls, calibrators, or methods): The recommended Positive, Negative and Internal Control material were tested a sufficient number of times with acceptable results on all testing days.

$d .$ . Detection limit: The Limits of Detection (LODs) for the CT $\mathrm { Q } ^ { \mathrm { X } }$ Assay with $C .$ . trachomatis serovar H in urine and swab specimens when extracted on the BD Viper System were determined to be $< 1 5$ CT elementary bodies (EB) per mL for neat and $\mathrm { Q } ^ { x }$ UPT urine and $< 3 0$ CT EB per mL for expressed vaginal and endocervical swab specimens. A correlation of EB to Inclusion-forming units (IFU) suggests that the CT $\mathrm { Q } ^ { \bf X }$ assay LODs with serovar $\mathrm { H }$ in urine and swab specimens correspond to $< 1$ IFU per mL (15). The CT $\mathrm { Q } ^ { \bf X }$ Assay on the BD Viper System in extracted mode was able to detect 16 isolates representing $1 5 \mathrm { { C T } }$ serovars (A, B, Ba, C, D, E $( 2 ) ^ { * }$ , F, G, H, I, J, K, LGV1, LGV2, and LGV3) with $\ge 9 5 \%$ proportion positive at a concentration of 15 EB per mL in CT/GC $\mathrm { Q } ^ { \bf X }$ Swab Diluent.

\* Testing with CT serovar E included the nvCT strain, a new variant with a deletion in the cryptic plasmid. (21)

# e. Analytical specificity:

1. Cross Reactivity: DNA from 141 organisms listed in the following table was extracted on the BD Viper System and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay. All potential cross-reactive species were tested at $> \mathrm { 1 x 1 0 } ^ { 8 }$ cells/mL except where noted. The CT $\mathrm { Q } ^ { x }$ Assay did not cross-react with any of the organisms tested.

Potential Cross-reacting Microorganisms   

<table><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus</td><td rowspan=1 colspan=1>Epstein Barr Virus***</td><td rowspan=1 colspan=1>Peptostreptococcus productus</td><td rowspan=1 colspan=1>Neisseria elongatasubsp.nitroreduscens (2)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Neisseria elongata</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Flavobacteriummeningosepticum</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Neisseria flava (4)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus***</td><td rowspan=1 colspan=1>Gardnerellavaginalis</td><td rowspan=1 colspan=1>Providenciastuartii</td><td rowspan=1 colspan=1>Neisseriaflavescens (4)</td></tr><tr><td rowspan=1 colspan=1>Aeromonashydrophilia</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Neisseriagonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis*</td><td rowspan=1 colspan=1>Haemophilusinfluenzae</td><td rowspan=1 colspan=1>Salmonellaminnesota</td><td rowspan=1 colspan=1>Neisseria lactamica(7)</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis*</td><td rowspan=1 colspan=1>Herpes SimplexViVus *</td><td rowspan=1 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>Neisseriameningitidis (12)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Humanpapillomavirus (16and 18*</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Neisseria mucosa(5)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>Neisseria perflava(8)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata*</td><td rowspan=1 colspan=1>Klebsiellapneumoniae</td><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>Neisseriapolysaccharea (2)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis*</td><td rowspan=1 colspan=1>Lactobacillusacidophilus*</td><td rowspan=1 colspan=1>Streptococcusmitis</td><td rowspan=1 colspan=1>Neisseria sicca (5)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae****</td><td rowspan=1 colspan=1>Lactobacillus brevis</td><td rowspan=1 colspan=1>Streptococcusmutans</td><td rowspan=1 colspan=1>Neisseria subflava(15)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia psittaci*</td><td rowspan=1 colspan=1>Lactobacillusjensenii*</td><td rowspan=1 colspan=1>Streptococcuspneumoniae*</td><td rowspan=1 colspan=1>Neisseria weaverii(3)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Listeriamonocytogenes</td><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Clostridiumperfringens</td><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>Streptomycesgriseus *</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumrenale</td><td rowspan=1 colspan=1>Moraxella lacunata*</td><td rowspan=1 colspan=1>Trichomonasvaginalis **</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans*</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>Veillonellaparvula</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Cytomegalovirus\*\* Morganella morganii Vibrio

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>Mycobacteriumgordonae</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Mycobacteriumsmegmatis</td><td rowspan=1 colspan=1>Branhamellacatarrhalis (5)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Peptostreptococcusanaerobius</td><td rowspan=1 colspan=1>Neisseria cinerea(2)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Peptostreptococcusasaccharolyticus</td><td rowspan=1 colspan=1>Neisseriaelongata subsp.glycolytica</td></tr></table>

(n) number of strains tested in the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay

\* Tested at $> 1 \mathrm { x } 1 0 ^ { 7 }$ cells or EB per mL; \*\*Tested at $> 1 \mathrm { x } 1 0 ^ { 6 }$ cells or viral particles per mL; \*\*\*Tested at $\geq 1 \mathrm { x } 1 0 ^ { 6 }$ genomic equivalents per mL;\*\*\*\* tested a $\mathbf { t } \geq 1 \dot { \mathbf { x } } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$

2. Interference: The performance of the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode was evaluated in the presence of potential interfering substances which may be encountered in swab and/or urine specimens. Potential interfering substances were spiked into $\mathrm { Q } ^ { x }$ UPT urine and vaginal swab specimen matrices in both the presence and the absence of CT elementary bodies (30 CT $\mathrm { E B / m L }$ in urine matrix and 90 CT EB/mL in swab matrix). Results are summarized in the following table.

GC Qx Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td></tr><tr><td rowspan=1 colspan=1>No InterferenceObserved</td><td rowspan=1 colspan=1>Blood (≤ 60%)Seminal FluidMucusOver The Counter vaginalproducts andcontraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 EB/mL C.trachomatis</td><td rowspan=1 colspan=1>Blood (≤1%)Seminal fluidMucusAntibioticsAnalgesicsOver The Counter deodorantsprays and powdersHormonesLeukocytesAlbumin &lt;1 mg/mLGlucoseAcidic urine (pH 4.0)Alkaline urine (pH 9.0)Bilirubin1x106 EB/mL C. trachomatisOrganisms associated withUrinary Tract Infections</td></tr><tr><td rowspan=1 colspan=1>May cause extractioncontrol (EC) failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not applicable</td></tr></table>

# g. Cross Contamination and Carryover

An internal study was conducted to evaluate the risk of producing a false positive result in either the same run on the BD Viper System in extracted mode (within run crosscontamination) or in a subsequent run (between run carryover). Testing was conducted using negative and positive samples on three BD Viper Systems. Negative samples consisted of CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent. Positive samples consisted of a representative analyte (at $1 0 ^ { 5 } \mathrm { C T E B / m L }$ spiked into CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent. The overall rate of cross-contamination (i.e., with alternating columns of positive and negative samples and a prevalence of $50 \%$ ) was $0 . 4 1 \%$ (9/2208). The overall rate of carryover contamination (i.e., carryover between successive runs when the prevalence was $50 \%$ in the previous run) was $0 . 3 6 \%$ (8/2208). Cross-contamination and carryover rates across the three BD Viper Systems are summarized in table.

Cross Contamination and Carryover Contamination   

<table><tr><td rowspan=2 colspan=1>AssayDispenseModeSelected</td><td rowspan=2 colspan=1>BDViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>DualAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.36</td></tr><tr><td rowspan=4 colspan=1>SingleAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate method: See below under Performance Characteristics   
b. Matrix comparison: NA

3. Clinical studies:

a. Clinical Sensitivity: NA b. Clinical specificity: NA

# Performance Characteristics

Clinician-collected endocervical and male urethral swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and male and female $\mathrm { Q } ^ { x }$ UPT and neat urine specimens were collected from 1059 female subjects and 479 male subjects attending OB/GYN, sexually transmitted disease (STD) and family planning clinics at seven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, urethral discharge, coital pain/difficulty/bleeding, testicular or scrotum pain/swelling, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms. Sixty five female subjects and 7 male subjects were excluded from the data analysis due to age requirement violations, antibiotic treatment in the last 21 days, opting to withdraw from the study after initially consenting, failure to obtain paired swab and urine specimens, urine quantity less than $2 0 ~ \mathrm { m L }$ , or transport and storage errors related to specimen collection. Therefore, the final data analysis included 994 compliant female subjects and 472 compliant male subjects.

Five specimens were collected from each of the 994 eligible female subjects. A urine specimen was collected and split into $\mathrm { Q } ^ { x }$ UPT, neat urine and the two reference urine specimen collection devices followed by a vaginal swab specimen and three randomized endocervical swab specimens. Up to four specimens were collected from each of the 472 eligible male subjects. Up to three randomized urethral swab specimens were collected followed by a urine specimen that was split into $\mathbf { Q } ^ { x } \mathbf { U } \mathbf { P } \mathrm { T }$ , neat urine and the two reference urine specimen collection devices. BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ assay results were generated from the $\mathrm { Q } ^ { x }$ UPT and neat urine specimens, the vaginal swab specimen, one endocervical swab specimen and one male urethral swab specimen. The remaining two endocervical swab specimens, up to two male urethral swab specimens, and the two reference urine specimens for each male and female subject were tested using two reference methods: the BD ProbeTec ET CT/AC assay and another commercially available NAAT (Nucleic Acid Amplification Test). Specimen testing was conducted either at the site of specimen collection or at a designated BD Viper testing site.

All performance calculations were based on the total number of BD ProbeTec CT $\mathrm { Q } ^ { x }$ assays results for endocervical, vaginal and male urethral swab specimens, and male and female $\mathrm { Q } ^ { x }$ UPT and neat urine specimens compared to a patient infected status (PIS) algorithm for each gender. In the algorithm, the designation of a subject as being infected with CT or not was based on endocervical swab and urine specimen results from the commercially available BD ProbeTec ET CT/AC assay and another commercially available NAAT. Subjects were considered infected with CT if two of the four endocervical swab and urine specimens (or two of the three or four urethral swab and urine specimens) tested positive in the BD ProbeTec ET CT/AC assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 5388 BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ assay results was used to calculate sensitivity and specificity. Sensitivity and specificity by specimen type and symptomatic status are presented in the table below.

Performance of the assay with endocervical swabs, patient collected vaginal swab specimens (in a clinical setting), female UPT and neat urine was assessed in the clinical study. Separate performance was calculated for specimens collected from pregnant females. Sensitivity compared to patient infected status for FS was $6 2 . 5 \%$ (5/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FV sensitivity was $7 5 \%$ (6/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FNU and FUPT sensitivity were $100 \%$ (8/8). Specificity was $9 4 . 7 \%$ (18/19) for FS, FV, FNU, and FUPT separately.

Tables below summarize the results from symptomatic and asymptomatic subjects designated as infected or non-infected with CT according to the PIS algorithm.

# CT $\mathbf { Q } ^ { x }$ Assay Performance Compared to Patient Infected Status (by specimen type and symptomatic status)

<table><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Symptomatic</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">95%C.I.</td><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">95%C.I.</td><td colspan="1" rowspan="1">PPV%</td><td colspan="1" rowspan="1">NPV%</td><td colspan="1" rowspan="1">ErrorInitial/Final</td></tr><tr><td colspan="1" rowspan="1">FS</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">450</td><td colspan="1" rowspan="1">93.0%(53/57)</td><td colspan="1" rowspan="1">(83.0%-98.1%</td><td colspan="1" rowspan="1">98.0%(385/393)</td><td colspan="1" rowspan="1">(96.0%99.1%)</td><td colspan="1" rowspan="1">86.9</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">2/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">543</td><td colspan="1" rowspan="1">89.7%(5258)</td><td colspan="1" rowspan="1">(78.8%-96.1%)</td><td colspan="1" rowspan="1">98.6%(478/485)</td><td colspan="1" rowspan="1">(97.0%-99.4%)</td><td colspan="1" rowspan="1">88.1</td><td colspan="1" rowspan="1">98.8</td><td colspan="1" rowspan="1">1/1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">993</td><td colspan="1" rowspan="1">91.3%(105/115)</td><td colspan="1" rowspan="1">(84.6%-95.8%)</td><td colspan="1" rowspan="1">98.3%(863/878)</td><td colspan="1" rowspan="1">(97.2%-99.0%</td><td colspan="1" rowspan="1">87.5</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">3/1</td></tr><tr><td colspan="1" rowspan="1">FV</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">449</td><td colspan="1" rowspan="1">98.2%(56/57)</td><td colspan="1" rowspan="1">(90.6%100.0%)</td><td colspan="1" rowspan="1">99.5%(390/392)</td><td colspan="1" rowspan="1">(98.2%99.9%)</td><td colspan="1" rowspan="1">96.6</td><td colspan="1" rowspan="1">99.7</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">544</td><td colspan="1" rowspan="1">94.8%(55/58)</td><td colspan="1" rowspan="1">(85.6%-98.9%)</td><td colspan="1" rowspan="1">99.0%(481/486)</td><td colspan="1" rowspan="1">(97.6%-99.7%</td><td colspan="1" rowspan="1">91.7</td><td colspan="1" rowspan="1">99.4</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">993</td><td colspan="1" rowspan="1">96.5%(111/115)</td><td colspan="1" rowspan="1">(91.3%-99.0%)</td><td colspan="1" rowspan="1">99.2%(871/878)</td><td colspan="1" rowspan="1">(98.4%-99.7%)</td><td colspan="1" rowspan="1">94.1</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">FN</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">450</td><td colspan="1" rowspan="1">93.0%(53/57)</td><td colspan="1" rowspan="1">(83.0%-98.1%)</td><td colspan="1" rowspan="1">100.0%(393/393)</td><td colspan="1" rowspan="1">(99.1%-100.0%)</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">543</td><td colspan="1" rowspan="1">93.1%(54/58)</td><td colspan="1" rowspan="1">(83.3%-98.1%)</td><td colspan="1" rowspan="1">99.0%(480/485)</td><td colspan="1" rowspan="1">(97.6%-99.7%)</td><td colspan="1" rowspan="1">91.5</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">993</td><td colspan="1" rowspan="1">93.0%(107/115)</td><td colspan="1" rowspan="1">(86.8%-96.9%)</td><td colspan="1" rowspan="1">99.4%(873/878)</td><td colspan="1" rowspan="1">(98.7%-99.8%)</td><td colspan="1" rowspan="1">95.5</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">FUPT</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">450</td><td colspan="1" rowspan="1">94.7%(54/57)</td><td colspan="1" rowspan="1">(85.4%-98.9%)</td><td colspan="1" rowspan="1">99.5%(391/393)</td><td colspan="1" rowspan="1">(98.2%-99.9%)</td><td colspan="1" rowspan="1">96.4</td><td colspan="1" rowspan="1">99.2</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">543</td><td colspan="1" rowspan="1">91.4%(53/58)</td><td colspan="1" rowspan="1">(81.0%97.1%)-</td><td colspan="1" rowspan="1">99.0%(480/485)</td><td colspan="1" rowspan="1">(97.6%-99.7%)</td><td colspan="1" rowspan="1">91.4</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">993</td><td colspan="1" rowspan="1">93.0%(107/115)</td><td colspan="1" rowspan="1">(86.8%-96.9%)</td><td colspan="1" rowspan="1">99.2%(871/878)</td><td colspan="1" rowspan="1">(98.4%-99.7%)</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">88.6%(31/35)</td><td colspan="1" rowspan="1">(73.3%-96.8%)</td><td colspan="1" rowspan="1">98.9%(178/180)</td><td colspan="1" rowspan="1">(96.0%-99.9%)</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">97.8</td><td colspan="1" rowspan="1">1/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">257</td><td colspan="1" rowspan="1">93.9%(62/66)</td><td colspan="1" rowspan="1">(85.2%-98.3%)</td><td colspan="1" rowspan="1">97.9%(187/191)</td><td colspan="1" rowspan="1">(94.7%-99.4%)</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">1/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">472</td><td colspan="1" rowspan="1">92.1%(93/101)</td><td colspan="1" rowspan="1">(85.0%-96.5%)</td><td colspan="1" rowspan="1">98.4%(365/371)</td><td colspan="1" rowspan="1">(96.5%-99.4%)</td><td colspan="1" rowspan="1">93.9</td><td colspan="1" rowspan="1">97.9</td><td colspan="1" rowspan="1">2/0</td></tr><tr><td colspan="1" rowspan="1">MN</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">100.0%(35/35)</td><td colspan="1" rowspan="1">(90.0%100.0%</td><td colspan="1" rowspan="1">98.9%(178/180)</td><td colspan="1" rowspan="1">(96.0%-99.9%)</td><td colspan="1" rowspan="1">94.6</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">257</td><td colspan="1" rowspan="1">97.0%(64/66)</td><td colspan="1" rowspan="1">(89.5%99.6%)</td><td colspan="1" rowspan="1">99.5%(190/191)</td><td colspan="1" rowspan="1">(97.1%-100.0%)</td><td colspan="1" rowspan="1">98.5</td><td colspan="1" rowspan="1">99.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">472</td><td colspan="1" rowspan="1">98.0%(99/101</td><td colspan="1" rowspan="1">(93.0%-99.8%)</td><td colspan="1" rowspan="1">99.2%(368/371)</td><td colspan="1" rowspan="1">(97.7%-99.8%)</td><td colspan="1" rowspan="1">97.1</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">MUPT</td><td colspan="1" rowspan="1">N</td><td colspan="1" rowspan="1">215</td><td colspan="1" rowspan="1">100.0%35//35)</td><td colspan="1" rowspan="1">(90.0%100.0%</td><td colspan="1" rowspan="1">98.9%(178/180)</td><td colspan="1" rowspan="1">(96.0%-99.9%)</td><td colspan="1" rowspan="1">94.6</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">257</td><td colspan="1" rowspan="1">97.0%(64/66)</td><td colspan="1" rowspan="1">(89.5%-99.6%)</td><td colspan="1" rowspan="1">97.4%(186/191)</td><td colspan="1" rowspan="1">(94.0%99.1%)</td><td colspan="1" rowspan="1">92.8</td><td colspan="1" rowspan="1">98.9</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">472</td><td colspan="1" rowspan="1">98.0%(9/101)</td><td colspan="1" rowspan="1">(93.0%-99.8%)</td><td colspan="1" rowspan="1">98.1%(364/371)</td><td colspan="1" rowspan="1">(96.2%-99.2%)</td><td colspan="1" rowspan="1">93.4</td><td colspan="1" rowspan="1">99.5</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5388</td><td colspan="1" rowspan="1">94.5%(721/763)</td><td colspan="1" rowspan="1">(92.6%-96.0%)</td><td colspan="1" rowspan="1">98.9%(4575/4625)</td><td colspan="1" rowspan="1">(98.6%-99.2%)</td><td colspan="1" rowspan="1">93.5</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">5/1</td></tr></table>

Note: For female subjects, infections localized to the endocervix or urethra have been reported in the literature (16- 20). Analyses were performed on the endocervical swab specimens and female UPT and neat urine specimens to further characterize the ten negative female endocervical swabs (105/115) and the eight negative female UPT and neat urine specimens (107/115).

• Of the 115 female subjects defined as positive by the PIS algorithm, ten had infections localized to the urethra as indicated by the urine reference result (i.e, BD ProbeTec ET CT/AC assay and other NAAT endocervical swab specimens were negative, BD ProbeTec ET CT/AC assay and other NAAT urine specimens were positive). The BD ProbeTec CT $\mathrm { Q } ^ { x }$ assay was negative for nine of the ten endocervical swab specimens from these subjects.   
o Of the 115 female subjects defined as positive by the PIS algorithm, three had infections localized to the endocervix as indicated by the endocervical reference result (i.e, BD ProbeTec ET CT/AC assay and other NAAT urine specimens were negative, BD ProbeTec ET CT/AC assay and other NAAT endocervical swab

specimens were positive.) T he BD ProbeTec CT $\mathrm { Q } ^ { x }$ assay was negative for UPT and neat urine for these three subjects.

CT $\mathbf { Q } ^ { x }$ Assay Performance Compared to Patient Infected Status (by clinical site).   

<table><tr><td>Specimen Type</td><td>Clinical Site</td><td>Prevalen ce</td><td>n</td><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td># CT (+) and GC (+)</td><td>PPV%</td><td>NPV%</td></tr><tr><td rowspan="8">FS1</td><td>1</td><td>16.1%</td><td>15 5</td><td>96.0% (24/25)</td><td>(79.6% - 99.9%)</td><td>96.2% (125/130)</td><td>(91.3% - 98.7%</td><td>5</td><td>82.8%</td><td>99.2%</td></tr><tr><td>2</td><td>11.0%</td><td>15 5</td><td>88.2% (15/17)</td><td>(63.6% - 8.5%)</td><td>98.6% (136/138)</td><td>(94.9% - 9.8%)</td><td>6</td><td>88.2%</td><td>98.6%</td></tr><tr><td>3</td><td>12.3%</td><td>73</td><td>88.9% (8/9)</td><td>(51.8% - 9.7%)</td><td>100.0% (64/64)</td><td>(94.4% - 100.0%)</td><td>2</td><td>100.0 %</td><td>98.5%</td></tr><tr><td>4</td><td>18.1%</td><td>10</td><td>89.5% (17/19)</td><td>(66.9% - 98.7%)</td><td>100.0% (86/86)</td><td>(95.8% - 100%)</td><td>6</td><td>100.0 %</td><td>97.7%</td></tr><tr><td>5</td><td>10.0%</td><td>70</td><td>100.0% (717)</td><td>(59.0% - 100%)</td><td>96.8% (61/63)</td><td>(89.0% - 99.%)</td><td>0</td><td>77.8%</td><td>100.0 %</td></tr><tr><td>6</td><td>8.5%</td><td>30</td><td>90.3% (28/31)</td><td>(74.2% - 980%)</td><td>98.5% (329/334)</td><td>(96.5% - 9.5%)</td><td>3</td><td>84.8%</td><td>99.1%</td></tr><tr><td>7</td><td>10.0%</td><td>70</td><td>85.7% (6/7)</td><td>(42.1%- 9.6%)</td><td>98.4% (62/63)</td><td>(91.5% - 100%)</td><td>0</td><td>85.7%</td><td>98.4%</td></tr><tr><td>1</td><td>16.1%</td><td>15 5</td><td>100.0% (25/25)</td><td>(86.3% - 100%)</td><td>97.7% (127/130)</td><td>(93.4% - 9.5%)</td><td>5</td><td>89.3%</td><td>100.0 %</td></tr><tr><td>2</td><td>11.0%</td><td>$1</td><td>100.0% 17/17)</td><td></td><td>(80.5% - 100%)</td><td>99.3% (137/138)</td><td>(96.0% - 100%)</td><td>6</td><td>94.4%</td><td>100.0 %</td></tr><tr><td>FV2</td><td>3</td><td>12.3%</td><td>73</td><td>77.8% (7/9)</td><td>(40.0% - 97.2%)</td><td>100.0% (64/64)</td><td>(94.4% - 100.0%)</td><td>2</td><td>100.0 %</td><td>97.0%</td></tr><tr><td>4</td><td>18.1%</td><td></td><td>10 5</td><td>94.7% (18/19)</td><td>(74.0% - 99.9%)</td><td>100.0% (86/86)</td><td>(95.8% - 100%)</td><td>6</td><td>100.0 %</td><td>98.9%</td></tr><tr><td>5</td><td>10.0%</td><td>70</td><td></td><td>100.0% (7/7)</td><td>(59.0% - 100%)</td><td>96.8% (61/63)</td><td>(89.0% - 99.6%)</td><td>0</td><td>77.8%</td><td>100.0 %</td></tr><tr><td>6</td><td>8.5%</td><td>$30ac6</td><td>96.8% (30/31)</td><td></td><td>(83.3% - 9.9%)</td><td>100.0% (334/334)</td><td>(98.9% - 100%)</td><td>3</td><td>100.0 %</td><td>99.7%</td></tr><tr><td>7</td><td>10.0%</td><td>70</td><td>100.0% (7/17)</td><td></td><td>(59.0% - 100%)</td><td>98.4% (62/63)</td><td>(91.5% - 100%)</td><td>0</td><td>87.5%</td><td>100.0 %</td></tr><tr><td>FNU³3</td><td>1</td><td>16.1%</td><td>$15 </td><td>92.0% (23/25)</td><td>(74.0% - 99.0%)</td><td>97.7% (127/130)</td><td>(93.4% - 99.5%)</td><td>5</td><td>88.5%</td><td>98.4%</td></tr><tr><td rowspan="8">Specimen Type</td><td>Clinical Site</td><td>Prevalen ce</td><td>n</td><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td># CT (+) and GC (+)</td><td>PPV%</td><td>NPV%</td></tr><tr><td>2</td><td>11.0%</td><td>15 5</td><td>82.4% 14/17)</td><td>(56.6% - 96.2%)</td><td>100.0% (138/138)</td><td>(97.4% - 100.0%)</td><td>6</td><td>100.0 %</td><td>97.9%</td></tr><tr><td>3</td><td>12.3%</td><td>73</td><td>88.9% (8/9)</td><td>(51.8% - 9.7%)</td><td>100.0% (64/64)</td><td>(94.4% - 100.0%)</td><td>2</td><td>100.0 %</td><td>98.5%</td></tr><tr><td>4</td><td>18.3%</td><td>10</td><td>100.0% (19/19)</td><td>(82.4% - 100%)</td><td>97.6% (83/85)</td><td>(91.8% - 9.7%)</td><td>6</td><td>90.5%</td><td>100.0 %</td></tr><tr><td>5</td><td>10.0%</td><td>70</td><td>100.0% (7/7)</td><td>(59.0% - 100.0%)</td><td>100.0% (63/63</td><td>(94.3% - 100.0%)</td><td>0</td><td>100.0 %</td><td>100.0 %</td></tr><tr><td>6</td><td>8.5%</td><td>36</td><td>93.5% (29/31</td><td>(78.6% - 99.2%)</td><td>100.0% (335/335)</td><td>(98.9% - 100%)</td><td>3</td><td>100.0 %</td><td>99.4%</td></tr><tr><td>7</td><td>10.0%</td><td>70</td><td>100.0% (7/7)</td><td>(59.0% - 100.0%)</td><td>100.0% (63/63)</td><td>(94.3% - 100.0%)</td><td>0</td><td>100.0 %</td><td>100.0 %</td></tr><tr><td>1</td><td>16.1%</td><td>$155</td><td>96.0% (24/25)</td><td>(79.6% - 9.9%)</td><td>97.7% (127/130)</td><td>(93.4% - 9.5%)</td><td>5</td><td>88.9%</td><td>99.2%</td></tr><tr><td rowspan="6">FUPT4</td><td>2</td><td>11.0%</td><td>15</td><td>82.4%</td><td>(56.6% - 96.2%</td><td>99.3% (137/138)</td><td>(96.0% - 100.0%)</td><td>6</td><td>93.3%</td><td>97.9%</td></tr><tr><td>3</td><td>12.3%</td><td>5 73</td><td>(14/17) 88.9% (8/9)</td><td>(51.8% - 99.7%)</td><td>100.0% (64/64)</td><td>(94.4% - 100.0%)</td><td>2</td><td>100.0 %</td><td>98.5%</td></tr><tr><td>4</td><td>18.3%</td><td>10</td><td>100.0%</td><td>(82.4% - 100.0%)</td><td>100.0% (85/85)</td><td>(95.8% - 100.0%)</td><td>6</td><td>100.0</td><td>100.0 %</td></tr><tr><td>5</td><td>10.0%</td><td>4 70</td><td>(19/19) 100.0% (717)</td><td>(59.0% - 100%)</td><td>100.0% (63/63)</td><td>(94.3% - 100%)</td><td>0</td><td>% 100.0</td><td>100.0</td></tr><tr><td>6</td><td>8.5%</td><td>36 6</td><td>93.5% (29/31)</td><td>(78.6% - 99.2%)</td><td>99.4% (333/335)</td><td>(97.9% - 99.9%)</td><td>3</td><td>% 93.5%</td><td>% 99.4%</td></tr><tr><td>7</td><td>10.0%</td><td>70</td><td>85.7% (6/7)</td><td>(42.1% - 99.6%)</td><td>98.4% (62/63)</td><td>(91.5% - 100.0%)</td><td>0</td><td>85.7%</td><td>98.4%</td></tr><tr><td rowspan="4">MS5$</td><td>1</td><td>24.1%</td><td>20</td><td>87.8% (43/49)</td><td>(75.2% - 95.4%)</td><td>98.1% (151/154)</td><td>(94.4% - 99.6%)</td><td>9</td><td>93.5%</td><td>96.2%</td></tr><tr><td>2</td><td>22.4%</td><td>3 76</td><td>100.0% (17/17)</td><td>(80.5% - 100%)</td><td>98.3% (58/59)</td><td>(90.9% - 100%)</td><td>10</td><td>94.4%</td><td>100.0 %</td></tr><tr><td>4</td><td>23.8%</td><td>10 1</td><td>95.8% (23/24)</td><td>(78.9% - 99.9%)</td><td>100.0% (77/77)</td><td>(95.3% - 100%)</td><td>11</td><td>100.0 %</td><td>98.7%</td></tr><tr><td>5</td><td>15.5%</td><td>71</td><td>90.9% (10/11)</td><td>(58.7% - 99.8%)</td><td>96.7% (58/60)</td><td>(88.5% - 9.6%)</td><td>3</td><td>83.3%</td><td>98.3%</td></tr><tr><td>Specimen Type</td><td>Clinical Site</td><td>Prevalen ce</td><td>n</td><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td># CT (+) and GC (+)</td><td>PPV%</td><td>NPV%</td></tr><tr><td></td><td>7</td><td>0.0%</td><td>21</td><td>NA</td><td>NA</td><td>100.0% (21/21)</td><td>(83.9% - 100%)</td><td>0</td><td>NA</td><td>NA</td></tr><tr><td>MNU6</td><td>1</td><td>24.1%</td><td>20 3</td><td>98.0% 48/9)</td><td>(89.1% - 9.9%)</td><td>99.4% (153/154)</td><td>(96.4% - 100%)</td><td>9</td><td>98.0%</td><td>99.4%</td></tr><tr><td></td><td>2</td><td>22.4%</td><td>76</td><td>100.0% (17/17)</td><td>(80.5% - 100%)</td><td>100.0% (59/59)</td><td>(93.9% - 100.0%)</td><td>10</td><td>100.0 %</td><td>100.0 %</td></tr></table>

# Analysis of CT Positive/Negative Specimens from Female Subjects Based on Patient Infected Status

<table><tr><td colspan="1" rowspan="1">−</td><td colspan="2" rowspan="1">NAAT 1</td><td colspan="2" rowspan="1">NAAT 2</td><td colspan="4" rowspan="1">BD ProbeTec CT Q* Amplified DNAAssay</td><td colspan="3" rowspan="2">Symptomatic Status</td></tr><tr><td colspan="1" rowspan="2">PT</td><td colspan="1" rowspan="2">EndocervicalSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">EndocervicalSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">QxEndocervicalSwab</td><td colspan="1" rowspan="2">QxVaginalSwab</td><td colspan="1" rowspan="2">NeatUrine</td><td colspan="1" rowspan="2">QxUPTUrine</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="17">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">87</td></tr><tr><td colspan="9" rowspan="1">Total PIS Positive</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">115</td></tr><tr><td colspan="1" rowspan="4">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">NAAT 1</td><td colspan="2" rowspan="1">NAAT 2</td><td colspan="4" rowspan="1">BD ProbeTec CT QX Amplified DNAAssay</td><td colspan="3" rowspan="2">Symptomatic Status</td></tr><tr><td colspan="1" rowspan="23">PT</td><td colspan="1" rowspan="2">EndocervicalSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">EndocervicalSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">QxEndocervicalSwab</td><td colspan="1" rowspan="2">Q*VaginalSwab</td><td colspan="1" rowspan="2">NeatUrine</td><td colspan="1" rowspan="2">Q^UPTUrine</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">s</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">LE</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">362</td><td colspan="1" rowspan="1">456</td><td colspan="1" rowspan="1">818</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="9" rowspan="1">Total PIS Negative</td><td colspan="1" rowspan="1">393</td><td colspan="1" rowspan="1">486</td><td colspan="1" rowspan="1">879</td></tr></table>

I Indeterminate LE Liquid Level Error

Analysis of CT Positive/Negative Specimens from Male Subjects Based on Patient Infected Status   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">NAAT 1</td><td colspan="2" rowspan="1">NAAT 2</td><td colspan="3" rowspan="1">BD ProbeTec CT QxAmplified DNA Assay</td><td colspan="3" rowspan="2">Symptomatic Status</td></tr><tr><td colspan="1" rowspan="2">PIS CT</td><td colspan="1" rowspan="2">UrethralSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">UrethralSwab</td><td colspan="1" rowspan="2">Urine</td><td colspan="1" rowspan="2">QxUrethralSwab</td><td colspan="1" rowspan="2">NeatUrine</td><td colspan="1" rowspan="2">QxUPTUrine</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">S</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="10">+</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">72</td></tr><tr><td colspan="8" rowspan="1">Total PIS Positive</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">101</td></tr><tr><td colspan="1" rowspan="16"></td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">157</td><td colspan="1" rowspan="1">146</td><td colspan="1" rowspan="1">303</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="8" rowspan="1">Total PIS Negative</td><td colspan="1" rowspan="1">180</td><td colspan="1" rowspan="1">191</td><td colspan="1" rowspan="1">371</td></tr></table>

# Neat and $\mathbf { Q } ^ { x }$ UPT Urine Stability

Pools of CT negative male and female urine specimens were used in analytical experiments to support the urine storage and transport stability claims.

For neat urine, pools were co-spiked with CT serovar H and GC strain ATCC 19424 at 45 EB per mL and 150 cells per mL, respectively. Neat urine specimens were stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 8, 24 or $3 0 \mathrm { h }$ ; or at $- 2 0 ^ { \circ } \mathrm { C }$ for 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

For $\mathrm { Q } ^ { x }$ UPT urine, pooled specimens were co-spiked with CT serovar H and GC strain ATCC 19424 at $4 5 \mathrm { E B }$ per mL and 150 cells per $\mathrm { m L }$ , respectively. The spiked urine specimen pools were then stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for $2 4 \mathrm { h }$ or $3 0 ^ { \circ } \mathrm { C }$ for $^ { 8 \mathrm { ~ h ~ } }$ prior to transfer into $\mathrm { Q } ^ { \mathrm { x } }$ UPT tubes. The $\mathrm { Q } ^ { x }$ UPT specimen pools were then stored either at $2 { - } 8 ^ { \mathrm { { \circ } } } \mathrm { C }$ for 14, 21 or 30 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 14, 21 or 30 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 60 days. At each time point $\mathrm { Q } ^ { x }$ UPT specimens were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Vaginal Dry and Expressed Swab Stability

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for dry vaginal swab specimens. Pools were cospiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively, when seeded onto swabs and expressed in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab

Diluent. Seeded dry swabs were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3, 7, or 14 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 3, 7 or 14 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 30 or 60 days. At each time point, dry swabs were removed from storage and expressed into $2 \mathrm { m L }$ of CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent and evaluated with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for expressed vaginal swab specimens. Pools were spiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively. The spiked swab matrix was stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \circ } \mathrm { C }$ for 30 or 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Endocervical and Urethral Swab Specimen Stability

Pools of CT negative endocervical swab matrix were used in analytical experiments to support the storage and transport stability claims for endocervical and urethral swab specimens. Pools of swab matrix were spiked with CT serovar H and GC strain ATCC 19424 at 90 EB per mL and 300 cells per mL, respectively. The pools were dispensed in $2 \mathrm { m L }$ volumes into BD sample tubes to simulate “wet” endocervical specimens and stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 30 or 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Post Pre-warm Specimen Stability

Pools of male and female CT negative neat urine were used in analytical experiments to support the storage stability claims for pre-warmed neat and $\mathrm { Q } ^ { x }$ UPT urine specimens. Pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at $4 5 \mathrm { E B }$ per mL and 150 cells per mL, respectively and either added to $\mathrm { Q } ^ { x }$ UPT tubes or left untreated as neat urine. Both specimen types were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ , and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, specimen tubes were stored at either 2- $8 ^ { \mathrm { o } } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 1, 3 or 7 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 30 days. At each time point samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal and endocervical swab specimen matrices in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent were used in analytical experiments to support the storage stability claims for pre-warmed expressed vaginal, endocervical, and male urethral swab specimens. For both types of matrix, pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at 90 EB per mL and 300 cells per $\mathrm { m L }$ , respectively and aliquotted into 2 mL volumes in BD specimen tubes. The tubes were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, the specimen tubes were stored either at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3 or 7 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 3 or 7 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30 days. At each time point, samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# 4. MaxRFU Frequency Distribution:

A total of 5388 CT $\mathrm { Q } ^ { x }$ Assay results was evaluated at seven geographically diverse clinical sites. A frequency distribution of the initial MaxRFU values for the CT $\mathrm { Q } ^ { x }$ assay is shown in Figure A. The distribution of MaxRFU values from CT $\mathrm { Q } ^ { x }$ true positive, true negative, false positive and false negative specimens (i.e. from those specimens that yielded results which were discordant with the patient infected status (PIS)) is shown in the table below.

![](images/a55dc3dfe1ce7dd3ebd7ae2fab15098b379f7e9d0fe99d94c7501471df62304f.jpg)  
Figure A: Frequency Distribution of MaxRFU for the CT $\boldsymbol { \mathsf { Q } } ^ { x }$ Assay.

CT $\mathbf { Q } ^ { x }$ MaxRFU Ranges for False Negative, False Positive, True Negative and True Positive Results   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MaxRFUrange</td><td colspan="1" rowspan="1">0-49</td><td colspan="1" rowspan="1">50-99</td><td colspan="1" rowspan="1">100-124</td><td colspan="1" rowspan="1">125-149</td><td colspan="1" rowspan="1">150-199</td><td colspan="1" rowspan="1">200-249</td><td colspan="1" rowspan="1">250-349</td><td colspan="1" rowspan="1">350-499</td><td colspan="1" rowspan="1">500-799</td><td colspan="1" rowspan="1">≥800</td></tr><tr><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4590</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">755</td></tr><tr><td colspan="1" rowspan="5">FN</td><td colspan="1" rowspan="1">FNU</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FS</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FUPT</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FV</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MNU</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">b</td><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MUPT</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="8">FP</td><td colspan="1" rowspan="1">FNU</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">FS</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">FUPT</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">FV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">MNU</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">MUPT</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="8">TN</td><td colspan="1" rowspan="1">FNU</td><td colspan="1" rowspan="1">868</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FS</td><td colspan="1" rowspan="1">857</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FUPT</td><td colspan="1" rowspan="1">866</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">FV</td><td colspan="1" rowspan="1">866</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MNU</td><td colspan="1" rowspan="1">368</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1">364</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MUPT</td><td colspan="1" rowspan="1">359</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">4548</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="8">TP</td><td colspan="1" rowspan="1">FNU</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">104</td></tr><tr><td colspan="1" rowspan="1">FS</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">104</td></tr><tr><td colspan="1" rowspan="1">FUPT</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">107</td></tr><tr><td colspan="1" rowspan="1">FV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">111</td></tr><tr><td colspan="1" rowspan="1">MNU</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="1" rowspan="1">MUPT</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">717</td></tr></table>

Controls: During the clinical evaluation, there were no CT $\mathrm { Q } ^ { x }$ positive control failures from $2 5 3 \mathrm { C T }$ $\mathrm { Q } ^ { x }$ plate runs. For the CT $\mathrm { Q } ^ { x }$ negative control, a failure was observed in 1 of 253 CT $\mathrm { Q } ^ { x }$ plate runs. The CT/GC $\mathrm { Q } ^ { x }$ positive and negative control MaxRFU values observed in the clinical trials are shown in the table below.

Distribution of MaxRFU Results for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay Positive and Negative Controls   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Max RFU</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>5thPercentile</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>95thPercentile</td></tr><tr><td rowspan=1 colspan=1>CT Qx NegativeControl</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>0 - 41</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>CT Q PositiveControl</td><td rowspan=1 colspan=1>253</td><td rowspan=1 colspan=1>629 - 2378</td><td rowspan=1 colspan=1>1597.7</td><td rowspan=1 colspan=1>1939.4</td><td rowspan=1 colspan=1>1968.8</td><td rowspan=1 colspan=1>2184.0</td></tr></table>

5. Expected values/Reference range:

A. Prevalence: The prevalence of positive $C .$ . trachomatis specimens in patient populations depends upon: clinic type, age, risk factors, gender, and test method. The prevalence observed with the CT $\mathrm { Q } ^ { x }$ Assay during a multi-center clinical trial ranged from $8 . 5 \%$ to $1 8 . 3 \%$ for female specimens and $0 \%$ to $2 4 . 1 \%$ for male specimens.

B. Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the CT $\mathrm { Q } ^ { x }$ Assay are shown in Table 5. These calculations are based on hypothetical prevalence and overall sensitivity and specificity (compared to the patient infected status) of $9 4 . 5 \%$ and $9 8 . 9 \%$ , respectively. In addition, PPV and NPV based on actual prevalence, sensitivity and specificity are shown in Tables 8 and 9. PPV was calculated using: (Sensitivity \* Prevalence) / (Sensitivity\*Prevalence $^ +$ (1 - Specificity) $^ *$ (1 - Prevalence)). NPV was calculated using: (Specificity \* (1 - Prevalence)/ (1-Sensitivity) \* Prevalence $^ +$ Specificity \* (1-Prevalence)).

CT Hypothetical Positive and Negative Predictive Values Compared to Patient Infected Status.   

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Sensitivity(%)</td><td rowspan=1 colspan=1>Specificity(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>64.1</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>98.6</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>97.7</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>96.4</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>94.7</td></tr></table>

# N. Instrument:

BD Viper™ System in extracted mode with the addition of the CTQ and GCQ Assays: The BD Viper System with the capability of automated nucleic acid extraction is the third generation of the BD Viper robotic platform for amplified DNA analysis. The system builds upon its predecessors, the BD Viper Instrument (K023955) and the BD Viper System (K052481).

Comparison of Operating and Technological Characteristics   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BD Viper System:Extracted mode of operation</td><td colspan="1" rowspan="1">BD Viper System Non-extracted mode ofoperation (K052481)</td></tr><tr><td colspan="1" rowspan="1">Thermal Module:Priming TemperaturePriming toAmplification TransferTemperatureAmplification</td><td colspan="1" rowspan="1">Same as BD Viper SystemK05248170°Same as BD Viper SystemK052481</td><td colspan="1" rowspan="1">70°•70°C to 52.5°C 52.5°C</td></tr><tr><td colspan="1" rowspan="1">Temperature</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Amplification PlateSealing</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481</td><td colspan="1" rowspan="1">Automated sealing ofamplification plate</td></tr><tr><td colspan="1" rowspan="1">Optical Module:Fluorescent readers</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Same as BD Viper SystemK052481•Extraction control read on ROX</td><td colspan="1" rowspan="1">Automatic amplificationplate transfer into residentreaders (2 readers)Target read on FAMchannelROX channel not used</td></tr><tr><td colspan="1" rowspan="1">Software Module</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Sample log-in</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Patient Sample LocationIdentification (optional)</td><td colspan="1" rowspan="1">Bar-code or key enteredaccession numbers</td></tr><tr><td colspan="1" rowspan="1">Algorithm</td><td colspan="1" rowspan="1">MaxRFU - Maximum(normalized) RelativeFluorescence Units</td><td colspan="1" rowspan="1">•MOTA - Method OtherThan Acceleration (areaunder the curve)</td></tr><tr><td colspan="1" rowspan="1">Menu</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Added: CTQ, GCQ andCTQ/GCQ</td><td colspan="1" rowspan="1">CT or CT/GC</td></tr><tr><td colspan="2" rowspan="1">Controls and System Checks</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Positive and negative runcontrolsAutomated control rehydrationExtraction controlSpecimen processing controlprocedure</td><td colspan="1" rowspan="1">Positive and negative runcontrolsManual controlrehydrationSpecimen processingcontrol procedure</td></tr><tr><td colspan="1" rowspan="1">System Checks</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481•Extraction reagent checks• Liquid Waste reservoir check</td><td colspan="1" rowspan="1">Consumables, Pipettor,Thermal, Optical, FluidVolumes</td></tr><tr><td colspan="1" rowspan="1">Waste and Waste Disposal</td><td colspan="1" rowspan="1">cYanlob</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Waste CompositionLiquid Waste DisposalContainerLiquid WasteNeutralizer</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481 and liquid waste(extraction reagents)•Modified waste disposal bottleNeutralization pouch for liquidwaste</td><td colspan="1" rowspan="1">Solid waste (pipette tips)Waste disposal bottleresident, not used except inaborted run conditionNA</td></tr></table>

# O. System Descriptions:

Viewing the BD Viper System from the perspective of assay workflow, the level of automation added to enable automated nucleic acid extraction on the existing BD Viper System includes the following:

(1) Chemical lysis of organisms in clinical specimens,   
(2) Chemical extraction and purification of DNA using paramagnetic particles facilitated by employment of an extractor block containing a movable magnet assembly;   
(3) Elution of extracted DNA into SDA-compatible buffer; and   
(4) Transfer of the eluate from the extraction tube to the assay priming microwells.

Beyond these additions to the existing BD Viper System’s workflow, the following processing functions are common to both systems (extracted and non-extracted):

(5) Priming microwell heat spike;   
(6) Transfer of sample from priming microwells to prewarmed amplification microwells located directly on the reader stage/heater;   
(7) Amplification microwell plate sealing and movement of the sealed amplification microwells into the fluorescent reader;   
(8) Amplification temperature control and fluorescent photodetection; and   
(9) Calculation and result interpretation.

Software: The FDA has reviewed applicant’s Hazard Analysis and Software development processes for the device

# P. Other Supportive Device and Instrument Information: NA

Q. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

R. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.