# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K081825

B. Purpose for Submission: To add the option for automated extraction of DNA from TM clinical specimens using the software accessory “BD ProbeTec Neisseria gonorrhoeae (GC) Q Amplified DNA Assay” to direct the BD Viper™ System.

C. Measurand: Neisseria gonorrhoeae DNA

D. Type of Test: Qualitative determination of Neisseria gonorrhoeae DNA using the Strand Displacement Amplification (SDA) technology

E. Applicant: BD Diagnostic System

TM F. Proprietary and Established Names: BD ProbeTec  Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>LSL</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21CFR 866.3390 Neisseria spp. directserological test reagents</td><td rowspan=1 colspan=1>Microbiology (83)</td></tr></table>

# H. Intended Use:

1.) Intended use: The BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay, when tested with the BD Viper™ System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Neisseria gonorrhoeae DNA in clinician-collected female endocervical and male urethral swab specimens, patientcollected vaginal swab specimens (in a clinical setting), and male and female urine specimens. The assay is indicated for use with asymptomatic and symptomatic female individuals and symptomatic male individuals to aid in the diagnosis of gonococcal urogenital disease.

2.) Indications for use: Same as intended use

3) Special conditions for use statement(s): For Prescription use only

TM 4) Special instrument requirements: BD Viper System with automated nucleic acid extraction mode

I. Device Description: The BD ProbeTec GC Q Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of N. gonorrhoeae DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified N. gonorrhoeae target DNA, a second labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the $N .$ . gonorrhoeae -specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is rehydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper System and an automated algorithm is applied to both the EC and $N .$ gonorrhoeae -specific signals to report results as positive, negative, or EC failure.

# J. Substantial Equivalence Information:

a) Predicate device name (s): BD ProbeTecTm ET CT/NG Amplified DNA Assays Gen-Probe Aptima Combo 2 Assay   
b) Predicate Numbers (s): K984631, K003395

Comparison with predicate:

Comparison of Operating and Technological Characteristics   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BD Viper System:Extracted mode of operation</td><td colspan="1" rowspan="1">BD Viper System Non-extracted mode of operation(K052481)</td></tr><tr><td colspan="1" rowspan="1">Thermal Module:Priming TemperaturePriming to AmplificationTransfer TemperatureAmplificationTemperature</td><td colspan="1" rowspan="1">Same as BD Viper SystemK05248170Same as BD Viper SystemK052481</td><td colspan="1" rowspan="1">70•70°C to 52.5°C52.5°C</td></tr><tr><td colspan="1" rowspan="1">Amplification PlateSealing</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481</td><td colspan="1" rowspan="1">Automated sealing ofamplification plate</td></tr><tr><td colspan="1" rowspan="2">Optical Module:Fluorescent readers</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Same as BD Viper SystemK052481Extraction control read onROX</td><td colspan="1" rowspan="1">Automatic amplificationplate transfer into residentreaders (2 readers)•Target read on FAM channelROX channel not used</td></tr><tr><td colspan="3" rowspan="1">Software Module</td></tr><tr><td colspan="1" rowspan="1">Sample log-in</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Patient Sample LocationIdentification (optional)</td><td colspan="1" rowspan="1">Bar-code or key enteredaccession numbers</td></tr><tr><td colspan="1" rowspan="1">Algorithm</td><td colspan="1" rowspan="1">•MaxRFU - Maximum(normalized) RelativeFluorescence Units</td><td colspan="1" rowspan="1">•MOTA - Method Other ThanAcceleration (area under thecurve</td></tr><tr><td colspan="1" rowspan="1">Menu</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481Added: CTQ, GCQ andCTQ/GCQ</td><td colspan="1" rowspan="1">•CT or CT/GC</td></tr><tr><td colspan="3" rowspan="1">Controls and System Checks</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Positive and negative runcontrols•Automated controlrehydrationExtraction control•Specimen processing controlprocedure</td><td colspan="1" rowspan="1">Positive and negative runcontrolsManual control rehydrationSpecimen processing controlprocedure</td></tr><tr><td colspan="1" rowspan="1">System Checks</td><td colspan="1" rowspan="1">Same as BD Viper SystemK052481•Extraction reagent checks• Liquid Waste reservoir check</td><td colspan="1" rowspan="1">Consumables, Pipettor,Thermal, Optical, FluidVolumes</td></tr><tr><td colspan="1" rowspan="1">Waste and Waste Disposal</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Waste CompositionLiquid Waste DisposalContainerLiquid Waste Neutralizer</td><td colspan="1" rowspan="1">•Same as BD Viper SystemK052481 and liquid waste(extraction reagents)Modified waste disposal bottleNeutralization pouch forliquid waste</td><td colspan="1" rowspan="1">Solid waste (pipette tips)Waste disposal bottleresident, not used except inaborted run conditionNA</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005. http://www.fda.gov/cdrh/ode/guidance/337.pdf

2. CLSI EP5-A2 “Evaluation of Precision Performance of Quantitative Measurement Methods”,

3. CLSI EP12-A “User Protocol for Evaluation of Qualitative Test Performance”.

# L. Test Principle:

The BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is designed for use with the BD ProbeTec Chlamydia trachomatis/Neisseria gonorrhoeae (CT/GC) $\mathrm { Q } ^ { \mathrm { x } }$ specimen collection and transport devices, applicable reagents, the BD Viper System and BD Fox Extraction. Specimens are collected and transported in their respective transport devices which preserve the integrity of the N. gonorrhoeae DNA over the specified ranges of temperature and time.

Urine and swab specimens undergo a pre-warm step in the BD Viper Lysing Heater to dissolve mucus and homogenize the specimen. After cooling, the specimens are loaded onto the BD Viper System which then performs all the steps involved in extraction and amplification of target DNA, without further user intervention. The specimen is transferred to an Extraction Tube that contains ferric oxide particles in a dissolvable film and dried Extraction Control. A high pH is used to lyse the bacterial cells and liberate their DNA into solution. Acid is then added to lower the pH and induce a positive charge on the ferric oxide, which in turn binds the negatively charged DNA. The particles and bound DNA are then pulled to the sides of the Extraction Tube by magnets and the treated specimen is aspirated to waste. The particles are washed and a high pH Elution Buffer is added to recover the purified DNA. Finally, a Neutralization Buffer is used to bring the pH of the extracted solution to the optimum for amplification of the target.

The BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermallycontrolled fluorescent readers. The presence or absence of N. gonorrhoeae DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified N. gonorrhoeae target DNA, a second fluorescently-labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the N. gonorrhoeae-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and N. gonorrhoeae-specific signals to report specimen results as positive, negative, or EC failure.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

Reproducibility of the BD Viper System using the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Un-inoculated swab diluent for the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay was used for the GC negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in the following table.

Summary of Reproducibility Data on the BD Viper System for the GC $\mathbf { Q } ^ { \mathbf { x } }$ Assay   

<table><tr><td rowspan=1 colspan=6></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRunsWithin Site</td><td rowspan=1 colspan=2>Between Site</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>CTEB&#x27;s/mL</td><td rowspan=1 colspan=1>GCCells/mL</td><td rowspan=1 colspan=1>%Correct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MaxRFUMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=5 colspan=1>Endocervical/ Urethral</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.3%(134/135)</td><td rowspan=1 colspan=1>(95.9%,00.0%</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>151.3</td><td rowspan=1 colspan=1>1096.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5%(133/135)</td><td rowspan=1 colspan=1>(94.8%,99.8%)</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>220.7</td><td rowspan=1 colspan=1>785.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>120.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,00.0%)</td><td rowspan=1 colspan=1>1859.5</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.%%</td><td rowspan=1 colspan=1>1847.3</td><td rowspan=1 colspan=1>117.6</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%</td><td rowspan=1 colspan=1>1855.9</td><td rowspan=1 colspan=1>119.4</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>42.2</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=5 colspan=1>Urine /Vaginal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.3%(134/135)</td><td rowspan=1 colspan=1>(95.9%,00.0%</td><td rowspan=1 colspan=1>15.7</td><td rowspan=1 colspan=1>162.3</td><td rowspan=1 colspan=1>1031.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%)</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>295.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>69.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>48.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,10.0%</td><td rowspan=1 colspan=1>1899.0</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,1000%</td><td rowspan=1 colspan=1>1884.2</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%</td><td rowspan=1 colspan=1>1867.2</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>1.0</td></tr></table>

A second study was conducted internally to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LoD) of the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay. A panel of simulated specimens was tested that comprised GC and CT organisms seeded into $\mathrm { Q } ^ { \mathrm { x } }$ swab diluent at two different levels each of which was below the respective analytical LoD for the organisms (1:10, 1:100). These levels were selected to fall within the dynamic range of the analytical LOD curve of the assay. Fifteen replicates of each panel member were tested every day for five days across three BD Viper Systems. The data are summarized in the table below.

# Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the GC $\mathbf { Q } ^ { \mathbf { x } }$ Assay

$b$ . Linearity/assay reportable range: NA   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Dilution ofAnalytical LOD</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>MaxRFUMean(Positive)</td><td rowspan=1 colspan=1>%Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>Max RFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>92.9(209/225)</td><td rowspan=1 colspan=1>(88.7, 95.9)</td><td rowspan=1 colspan=1>1324.6</td><td rowspan=1 colspan=1>7.1(16/225)</td><td rowspan=1 colspan=1>(4.1, 11.3)</td><td rowspan=1 colspan=1>41.4</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>30.7(69/225)</td><td rowspan=1 colspan=1>(24.7, 37.1)</td><td rowspan=1 colspan=1>835.9</td><td rowspan=1 colspan=1>69.3(156/225)</td><td rowspan=1 colspan=1>(62.9, 75.3)</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>90.7(204/225)</td><td rowspan=1 colspan=1>(86.1, 94.1)</td><td rowspan=1 colspan=1>1165.9</td><td rowspan=1 colspan=1>9.3(21/225)</td><td rowspan=1 colspan=1>(5.9, 13.9)</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>22.7(51/225)</td><td rowspan=1 colspan=1>(17.4, 28.7)</td><td rowspan=1 colspan=1>872.7</td><td rowspan=1 colspan=1>77.3(174/225)</td><td rowspan=1 colspan=1>(71.3, 82.6)</td><td rowspan=1 colspan=1>7.8</td></tr></table>

c. Traceability, Stability, Expected values (controls, calibrators, or methods): The recommended Positive, Negative and Internal Control material were tested a sufficient number of times with acceptable results on all testing days.

$d .$ . Detection limit: Endocervical swabs and vaginal swabs were used for LoD studies because these swabs contain higher concentration of protein, mucous, and cellular debris and are considered “worst case”. Urethral swabs were not tested for LoD studies.

Endocervical swabs pool - Minipool of 72 individual swab specimens in diluent showing GC negative results was divided into 6 for spiking with strain 19424 at 5 different target levels and 1 negative target level (unspiked).

Vaginal swabs pool - Minipool of 12 individual swab specimens in diluent showing GC negative results was divided into 6 for spiking with strain 19424 at 5 different target levels and 1 negative target level (unspiked).

Urine: Male and female urine specimens screened separately and equal volumes of negative specimens from male and female were pooled and were divided into 6 for spiking with strain 19424 at 5 different target levels and 1 negative target level (unspiked).

The Limits of Detection (LoDs) for the GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay with Neisseria gonorrhoeae strain ATCC 19424 in urine and swab specimens when extracted on the BD Viper System were determined to be $< 5 0$ cells per mL for neat and $\mathrm { Q } ^ { \mathrm { x } }$ UPT urine and $< 1 0 0$ GC cells per mL for expressed vaginal and endocervical swab specimens.

The GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode was able to detect 17 GC strains (ATCC 19424, 27628, 27629, 27630, 27632, 27633, 27631, 21823, 51803, 23051, 31407, 31953, 35201, 31397, 31151, 43785, 51804) with $\ge 9 5 \%$ proportion positive at a concentration of 50 cells per mL in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent.

1. Cross Reactivity: DNA from 141 organisms listed in the following table was extracted on the BD Viper System and tested with the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay. All potential cross-reactive species were tested at $> 1 \mathrm { x } 1 0 ^ { 8 }$ cells/mL except where noted. Two $N .$ cinerea and two $N .$ lactamica strains were shown to cross-react in the GC $\mathrm { Q } ^ { \mathrm { x } }$ assay.

# Potential Cross-reacting Microorganisms

<table><tr><td>Acinetobacter calcoaceticus</td><td>Enterococcus faecium</td><td>Peptostreptococcus asaccharolyticus</td><td>Neisseria elongata subsp. glycolytica</td></tr><tr><td rowspan="2">Acinetobacter lwoffi</td><td>Epstein Barr</td><td>Peptostreptococcus</td><td rowspan="2">Neisseria elongata subsp. nitroreduscens (2)</td></tr><tr><td>Virus***</td><td>productus</td></tr><tr><td>Actinomyces israelii</td><td>Escherichia coli</td><td>Plesiomonas shigelloides</td><td>Neisseria elongata</td></tr><tr><td>Adenovirus***</td><td>Flavobacterium meningosepticum</td><td>Propionibacterium acnes</td><td>Neisseria flava (4)</td></tr><tr><td>Aeromonas hydrophilia</td><td>Gardnerella</td><td>Providencia stuartii</td><td>Neisseria flavescens (4)</td></tr><tr><td>Alcaligenes</td><td>vaginalis Gemella</td><td>Pseudomonas</td><td>Neisseria lactamica</td></tr><tr><td>faecalis*</td><td>haemolysans Haemophilus</td><td>aeruginosa Salmonella</td><td>(7) Neisseria</td></tr><tr><td>Bacillus subtilis* Bacteroides</td><td>influenzae Herpes Simplex</td><td>minnesota Salmonella</td><td>meningitidis (12) Neisseria mucosa</td></tr><tr><td>fragilis</td><td>Virus ** Human</td><td>typhimurium</td><td>(5)</td></tr><tr><td>Candida albicans * Candida</td><td>papillomavirus (16 and 18)***</td><td>Staphylococcus aureus Staphylococcus</td><td>Neisseria perflava (8) Neisseria</td></tr><tr><td>glabrata* Candida</td><td>Kingella kingae Klebsiella</td><td>epidermidis Streptococcus</td><td>polysaccharea (2)</td></tr><tr><td>tropicalis* Chlamydia</td><td>pneumoniae Lactobacillus</td><td>agalactiae</td><td>Neisseria sicca (5) Neisseria subflava</td></tr><tr><td>trachomatis</td><td>acidophilus*</td><td>Streptococcus mitis</td><td>(15)</td></tr><tr><td>Chlamydia pneumoniae****</td><td>Lactobacillus brevis</td><td>Streptococcus</td><td>Neisseria weaverii (3)</td></tr><tr><td>Chlamydia</td><td>Lactobacillus</td><td>mutans Streptococcus</td><td></td></tr><tr><td>psittaci* Citrobacter</td><td>jensenii* Listeria</td><td>pneumoniae* Streptococcus</td><td></td></tr><tr><td>freundii</td><td>monocytogenes Mobiluncus</td><td>pyogenes</td><td></td></tr><tr><td>Clostridium</td><td>mulieris</td><td>Streptomyces</td><td></td></tr><tr><td>perfringens</td><td></td><td>griseus **</td><td></td></tr><tr><td>Corynebacterium</td><td>Moraxella</td><td>Trichomonas</td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td>lacunata*</td><td>vaginalis **</td><td></td></tr><tr><td>renale</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>Cryptococcus</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td>neoformans*</td><td>Moraxella osloensis</td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td>Veillonella parvula</td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td></tr></table>

Cytomegalovirus Morganella Vibrio

<table><tr><td rowspan=1 colspan=1>**</td><td rowspan=1 colspan=1>morganii</td><td rowspan=1 colspan=1>parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Edwardsiellatarda</td><td rowspan=1 colspan=1>Mycobacteriumgordonae</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>Mycobacteriumsmegmatis</td><td rowspan=1 colspan=1>Branhamellacatarrhalis (5)</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Peptostreptococcusanaerobius</td><td rowspan=1 colspan=1>Neisseria cinerea(2)</td></tr></table>

(n) number of strains tested in the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay \* Tested at $\mathrm { \varepsilon } > 1 \mathrm { x } 1 0 ^ { 7 }$ cells or $\mathrm { E B / m L }$ ; \*\*Tested at $> 1 \mathrm { x } 1 0 ^ { 6 }$ cells or viral particles per mL; \*\*\*Tested at $> \mathrm { { 1 x 1 0 } } ^ { 6 }$ genomic equivalents per mL;\*\*\*\* tested at $> 1 \mathrm { x i } 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } ^ { \bullet } / \mathrm { m L }$

2. Interference: The performance of the BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode was evaluated in the presence of potential interfering substances which may be encountered in swab and/or urine specimens. Potential interfering substances were spiked into UPT urine and vaginal swab specimen matrices in both the presence and the absence of GC organisms (150 GC cells/mL in urine matrix and 300 GC cells/mL in swab matrix). Results are summarized in the following table.

# GC $\mathbf { Q } ^ { \mathbf { x } }$ Interfering Substances

<table><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td></tr><tr><td rowspan=1 colspan=1>No InterferenceObserved</td><td rowspan=1 colspan=1>Blood (≤ 60%)Seminal FluidMucusOver The Counter vaginalproducts andcontraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 EB/mL Chlamydiatrachomatis</td><td rowspan=1 colspan=1>Blood (≤1%)Seminal fluidMucusAntibioticsAnalgesicsOver The Counter deodorantsprays and powdersHormonesLeukocytesAlbumin &lt;1 mg/mLGlucoseAcidic urine (pH 4.0)Alkaline urine (pH 9.0)Bilirubin1x106 EB/mL ChlamydiatrachomatisOrganisms associated withUrinary Tract Infections</td></tr><tr><td rowspan=1 colspan=1>May cause extractioncontrol (EC) failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not applicable</td></tr></table>

# g. Cross Contamination and Carryover

An internal study was conducted to evaluate the risk of producing a false positive result in either the same run on the BD Viper System in extracted mode (within run crosscontamination) or in a subsequent run (between run carryover). Testing was conducted using negative and positive samples on three BD Viper Systems. Negative samples consisted of CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent. Positive samples consisted of a representative analyte $\mathbf { ( 1 0 ^ { 5 } C T E B / m L ) }$ spiked into CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent. The overall rate of crosscontamination (i.e., with alternating columns of positive and negative samples and a prevalence of $50 \%$ ) was $0 . 4 1 \%$ (9/2208). The overall rate of carryover contamination (i.e., carryover between successive runs when the prevalence was $50 \%$ in the previous run) was $0 . 3 6 \%$ (8/2208). Cross-contamination and carryover rates across the three BD Viper Systems are summarized in the table.

Cross Contamination and Carryover Contamination   

<table><tr><td rowspan=2 colspan=1>AssayDispenseModeSelected</td><td rowspan=2 colspan=1>BDViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>DualAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.36</td></tr><tr><td rowspan=4 colspan=1>SingleAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate method: See below under Performance Characteristics   
b. Matrix comparison: NA

3. Clinical studies: a. Clinical Sensitivity: NA b. Clinical specificity: NA

# Performance Characteristics

Clinician-collected endocervical and male urethral swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and male and female $\mathrm { Q } ^ { \mathrm { x } }$ UPT and neat urine specimens were collected from 1059 female subjects and 479 male subjects attending OB/GYN, sexually transmitted disease (STD) and family planning clinics at seven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, urethral discharge, coital pain/difficulty/bleeding, testicular or scrotum pain/swelling, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms. Sixty five female subjects and 7 male subjects were excluded from the data analysis due to age requirement violations, antibiotic treatment in the last 21 days, opting to withdraw from the study after initially consenting, failure to obtain paired swab and urine specimens, urine quantity less than $2 0 ~ \mathrm { m L }$ , or transport and storage errors related to specimen collection. Therefore, the final data analysis included 994 compliant female subjects and 472 compliant male subjects.

Five specimens were collected from each of the 994 eligible female subjects. A urine specimen was collected and split into $\mathrm { Q } ^ { \mathrm { x } }$ UPT, neat urine and the two reference urine specimen collection devices followed by a vaginal swab specimen and three randomized endocervical swab specimens. Up to four specimens were collected from each of the 472 eligible male subjects. Up to three randomized urethral swab specimens were collected followed by a urine specimen that was split into $\mathrm { Q } ^ { \mathrm { x } } \mathrm { U P T }$ , neat urine and the two reference urine specimen collection devices. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ assay results were generated from the $\mathrm { Q } ^ { \mathrm { x } }$ UPT and neat urine specimens, the vaginal swab specimen, one endocervical swab specimen and one male urethral swab specimen. The remaining two endocervical swab specimens, up to two male urethral swab specimens, and the two reference urine specimens for each male and female subject were tested using two reference methods: the BD ProbeTec ET GC/AC assay and another commercially available NAAT (Nucleic Acid Amplification Test). Specimen testing was conducted either at the site of collection or at a designated BD Viper testing site.

All performance calculations were based on the total number of BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ assays results for endocervical, vaginal and male urethral swab specimens, and male and female $\mathrm { Q } ^ { \mathrm { x } }$ UPT and neat urine specimens compared to a patient infected status (PIS) algorithm for each gender. In the algorithm, the designation of a subject as being infected with GC or not was based on endocervical swab and urine specimen results from the commercially available BD ProbeTec ET GC/AC assay and the other commercially available NAAT. Subjects were considered infected with GC if two of the four endocervical swab and urine specimens (or two of the three or four urethral swab and urine specimens) tested positive in the BD ProbeTec ET GC/AC assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 5387 BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ assay results was used to calculate sensitivity and specificity. Sensitivity and specificity by specimen type and symptomatic status are presented in the table below.

Performance of the assay with endocervical swabs, patient collected vaginal swabs specimens (in a clinical setting), female UPT and neat urine was assessed in the clinical study. Separate performance was calculated for specimens collected from pregnant females. Sensitivity compared to patient infected status for FS, FV, FNU, and FUPT was $100 \%$ (3/3). In each case, specificity was $100 \%$ (24/24) for FS, FV, FNU, and FUPT separately.

GC $\mathbf { Q } ^ { \mathbf { x } }$ Assay Performance Compared to Patient Infected Status (by specimen type and symptomatic status)   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>PPV%</td><td rowspan=1 colspan=1>NPV%</td><td rowspan=1 colspan=1>ErrorInitial/Final</td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.0%-99.9%</td><td rowspan=1 colspan=1>99.5%421/423)</td><td rowspan=1 colspan=1>(98.3%-99.%%</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>3/0</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Y</td><td rowspan=2 colspan=1>542</td><td rowspan=2 colspan=1>100.0%(38/38)</td><td rowspan=2 colspan=1>(90.7% -100.0%)</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=2 colspan=1>(98.9% -100.0%)</td><td rowspan=2 colspan=1>97.4</td><td rowspan=2 colspan=1>100.0</td><td rowspan=2 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1>(503/504)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>98.5%(64/65)</td><td rowspan=1 colspan=1>(91.7% -100.0%)</td><td rowspan=1 colspan=1>99.7%(924/927)</td><td rowspan=1 colspan=1>(99.1% -99.9%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>5/2</td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.2%-100.0%</td><td rowspan=1 colspan=1>98.6%(416/422)</td><td rowspan=1 colspan=1>(96.9%-99.5%)</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>544</td><td rowspan=1 colspan=1>100.0%(38/38)</td><td rowspan=1 colspan=1>(90.7%-10.0%)</td><td rowspan=1 colspan=1>99.6%(504/506)</td><td rowspan=1 colspan=1>(98.6%-100.0%)</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>100.0%(65/65</td><td rowspan=1 colspan=1>(94.5%-100.0%</td><td rowspan=1 colspan=1>99.1%(920/928)</td><td rowspan=1 colspan=1>(98.3% -99.6%)</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>96.3%(26/27)</td><td rowspan=1 colspan=1>(81.0%-99.%</td><td rowspan=1 colspan=1>99.3%(420/423)</td><td rowspan=1 colspan=1>(97.9% -99.9%)</td><td rowspan=1 colspan=1>89.7</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>97.4%(37/38)</td><td rowspan=1 colspan=1>(86.2%-99.9%</td><td rowspan=1 colspan=1>99.6%503/505)</td><td rowspan=1 colspan=1>(98.6% -100.0%</td><td rowspan=1 colspan=1>94.9</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>96.9%(63/65)</td><td rowspan=1 colspan=1>(89.3% -99.6%</td><td rowspan=1 colspan=1>99.5%(923/928)</td><td rowspan=1 colspan=1>(98.7% -99.8%</td><td rowspan=1 colspan=1>92.6</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>100.0%(27/27)</td><td rowspan=1 colspan=1>(87.2%-100.0%</td><td rowspan=1 colspan=1>99.5%(421/423)</td><td rowspan=1 colspan=1>(98.3%-99.9%)</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>97.4%(37/38)</td><td rowspan=1 colspan=1>(86.2%-99.9%</td><td rowspan=1 colspan=1>99.8%(504/505)</td><td rowspan=1 colspan=1>(98.9% -100.0%</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>98.5%(64/65)</td><td rowspan=1 colspan=1>(91.7%-100.0%</td><td rowspan=1 colspan=1>99.7%(925/928)</td><td rowspan=1 colspan=1>(99.1%-99.9%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0%(7/7)</td><td rowspan=1 colspan=1>(59.0%-100.0%</td><td rowspan=1 colspan=1>100.0%(208/208)</td><td rowspan=1 colspan=1>(98.2% -1000%</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100.0%(100/00)</td><td rowspan=1 colspan=1>(96.4% -100.0%)</td><td rowspan=1 colspan=1>98.7%(155/157)</td><td rowspan=1 colspan=1>(95.5% -99.8%</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0%(107/107)</td><td rowspan=1 colspan=1>(96.6%-10.0%</td><td rowspan=1 colspan=1>99.5%(363/365)</td><td rowspan=1 colspan=1>(98.0% -99.9%</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1>MN</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0%(7/7)</td><td rowspan=1 colspan=1>(59.0%-100.0%)</td><td rowspan=1 colspan=1>100.0%(208/208)</td><td rowspan=1 colspan=1>(98.2%-100.%)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100.0%(100/100)</td><td rowspan=1 colspan=1>(96.4%-100.0%)</td><td rowspan=1 colspan=1>98.1%(154/157)</td><td rowspan=1 colspan=1>(94.5% -99.6%)</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0%(107/107)</td><td rowspan=1 colspan=1>(96.6% -100.0%</td><td rowspan=1 colspan=1>99.2%(362/365)</td><td rowspan=1 colspan=1>(97.6% -99.8%)</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0%(7/7)</td><td rowspan=1 colspan=1>(59.0%-100.0%)</td><td rowspan=1 colspan=1>99.5%(207/208)</td><td rowspan=1 colspan=1>(97.4% -1000%</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100.0%(100/100)</td><td rowspan=1 colspan=1>(96.4% -100.0%)</td><td rowspan=1 colspan=1>98.7%(155/157)</td><td rowspan=1 colspan=1>(95.5%-99.8%)</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0%(107/107)</td><td rowspan=1 colspan=1>(96.6%-100.0%)</td><td rowspan=1 colspan=1>99.2%(362/365)</td><td rowspan=1 colspan=1>(97.6% -99.8%)</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5387</td><td rowspan=1 colspan=1>99.3%(577/581)</td><td rowspan=1 colspan=1>(98.2%-99.8%)</td><td rowspan=1 colspan=1>99.4%(4779/4806)</td><td rowspan=1 colspan=1>(99.2% -99.6%</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>6/2</td></tr></table>

GC Qx Assay Performance Compared to Patient Infected Status (by clinical site)   

<table><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td># CT</td><td></td><td></td></tr><tr><td rowspan="2">Specimen Type</td><td rowspan="2">Site</td><td>Prevalence</td><td>n</td><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td>(+)</td><td>PPV%</td><td>NPV%</td></tr><tr><td></td><td></td><td>100.0%</td><td>(75.3%-</td><td>99.3%</td><td>(96.1% -</td><td>and GC (+)</td><td></td><td></td></tr><tr><td rowspan="3">FS1</td><td>1 2</td><td>8.4% 10.4%</td><td>155 154</td><td>(/13) 93.8%</td><td>100.0%) (69.8% -</td><td>(141/142) 99.3%</td><td>100.0%) (96.0% -</td><td>5 6</td><td>92.9% 93.7%</td><td>100.0% 99.3%</td></tr><tr><td>3</td><td>6.8%</td><td>73</td><td>(15/16) 100.0%</td><td>99.8% (47.8% -</td><td>(137/138) 98.5%</td><td>100.0%) (92.1% -</td><td></td><td></td><td>100.0%</td></tr><tr><td>4</td><td></td><td></td><td>(5/5) 100.0%</td><td>100.0%% (83.2% -</td><td>(67/68) 100.0%</td><td>100.0% (95.8%-</td><td>2</td><td>83.3%</td><td></td></tr><tr><td></td><td>5</td><td>19.0%</td><td>105 70</td><td>(20/20) 100.0%</td><td>1000.0%) (2.5%-</td><td>(85/85) 100.0%</td><td>100.0% (94.8%-</td><td>6</td><td>100.0%</td><td>100.0%</td></tr><tr><td rowspan="2"></td><td>6</td><td>1.4% 2.2%</td><td>365</td><td>(1/1) 100.0%</td><td>100.0% (63.1% -</td><td>(69/69) 100.0%</td><td>100.0%) (99.0%- 3</td><td>0</td><td>100.0% 100.0%</td><td>100.0% 100.0%</td></tr><tr><td>7</td><td></td><td>70</td><td>(8/8) 100.0%</td><td>100.0%) (15.8% -</td><td>(357/357) 100.0%</td><td>100.0%) (94.7%- 0</td><td></td><td>100.0%</td><td>100.0%</td></tr><tr><td rowspan="5">FV2</td><td>1</td><td>2.9%</td><td></td><td>(2/2) 100.0%</td><td>100.0% (75.3% -</td><td>(68/68) 99.3%</td><td>100.0% (96.1% -</td><td></td><td></td><td></td></tr><tr><td></td><td>8.4%</td><td>155</td><td>(1/13) 100.0%</td><td>100.% (79.4% -</td><td>(141/142) 97.1%</td><td>100.0%) (92.8% -</td><td>5</td><td>92.9%</td><td>100.0%</td></tr><tr><td>2</td><td>10.3%</td><td>155</td><td>(16/16)</td><td>100.0%)</td><td>(135/139)</td><td>99.2%</td><td>6</td><td>80.0%</td><td>100.0%</td></tr><tr><td>3</td><td>6.8%</td><td>73</td><td>100.0% (5/5)</td><td>(47.8% - 100.0%</td><td>100.0% (68/68)</td><td>(94.7% - 100.0%)</td><td>2</td><td>100.0%</td><td>100.0%</td></tr><tr><td>4</td><td>19.0%</td><td>105</td><td>100.0% (20/20)</td><td>(83.2% - 100.0%)</td><td>97.6% (83/85)</td><td>(91.8%- 99.7%)</td><td>6</td><td>90.9%</td><td>100.0%</td></tr><tr><td></td><td>5</td><td>1.4%</td><td>70</td><td>100.0% (1/1)</td><td>(2.5%- 100.0%</td><td>100.0% (69/69)</td><td>(94.8% - 100.0%%</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td rowspan="5">FNU3</td><td>6</td><td>2.2%</td><td>365</td><td>100.0% (8/8)</td><td>(63.1%- 100.0%</td><td>99.7% (356/357)</td><td>(98.4%- 00.0%)</td><td>3</td><td>88.9%</td><td>100.0%</td></tr><tr><td>7</td><td>2.9%</td><td>70</td><td>10.0% (2/2)</td><td>(15.8%- 100.0%</td><td>10.0% (68/68)</td><td>(94.7%- 100.0%</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td>1</td><td>8.4%</td><td>155</td><td>100.0% (13/13)</td><td>(75.3% - 100.%</td><td>98.6% (140/142)</td><td>( 95.0%- 99.8%</td><td>5</td><td>86.7%</td><td>100.0%</td></tr><tr><td>2</td><td>10.3%</td><td>155</td><td>93.8% (15/16)</td><td>(69.8% - 99.8%</td><td>97.8% (136/139)</td><td>(93.8%- 996%</td><td>6</td><td>83.3%</td><td>99.3%</td></tr><tr><td>3</td><td>6.8%</td><td>73</td><td>100.0% (5/5)</td><td>(47.8% - 100.0%</td><td>100.0% (68/68)</td><td>(94.7% - 2 100.0%)</td><td></td><td>100.0%</td><td>100.0%</td></tr><tr><td>4</td><td></td><td>19.2%</td><td>104</td><td>100.0% (20/20)</td><td>(83.2% - 100.0%)</td><td>100.0% (84/84)</td><td>(95.7% - 6 100.0%</td><td></td><td>100.0%</td><td>100.0%</td></tr><tr><td></td><td>5</td><td>1.4%</td><td>70</td><td>100.0% (1/1))</td><td>(2.5% - 1000%</td><td>100.0% (69/69)</td><td>(94.8% - 100.0%</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td>Specimen Type</td><td>Site</td><td>Prevalence</td><td>n</td><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td># CT (+) and GC (+)</td><td>PPV%</td><td>NPV%</td></tr><tr><td></td><td>6</td><td>2.2%</td><td>366</td><td>100.0% (8/8)</td><td>(63.1%- 100.0%%</td><td>100.0% (358/358)</td><td>(99.0% - 100%</td><td>3</td><td>100.0%</td><td>100.0%</td></tr><tr><td></td><td>7</td><td>2.9%</td><td>70</td><td>50.0% (1/2)</td><td>(1.3%- 98.7%</td><td>100.0% (68/68)</td><td>(94.7%- 100.0% (96.1%-</td><td>0</td><td>100.0%</td><td>98.6%</td></tr><tr><td>FUPT4</td><td>1</td><td>8.4%</td><td>155</td><td>100.0% (13/13) 93.8%</td><td>(75.3% - 100.0%) (69.8% -</td><td>99.3% (141/142) 99.3%</td><td>100.0% (96.1% -</td><td>5</td><td>92.9%</td><td>100.0%</td></tr><tr><td rowspan="5"></td><td>2</td><td>10.3%</td><td>155</td><td>(15/16) 100.0%</td><td>99.8% (47.8% -</td><td>(138/139) 100.0%</td><td>6 100.0%) (94.7% -</td><td>2</td><td>93.8%</td><td>99.3%</td></tr><tr><td>3</td><td>6.8%</td><td>73</td><td>(5/5)</td><td>100.0%)</td><td>(68/68)</td><td>100.0%)</td><td></td><td>100.0%</td><td>100.0%</td></tr><tr><td>4</td><td>19.2%</td><td>104</td><td>10.0% (20/20)</td><td>(83.2% - 100.0%</td><td>98.%% (83/84)</td><td>(93.5% - 100.0%)</td><td>6</td><td>95.2%</td><td>100.0%</td></tr><tr><td>5</td><td>1.4%</td><td>70</td><td>100.0% (1/1)</td><td>(2.5%- 100.0%)</td><td>100.0% (69/69)</td><td>(94.8% - 100.0%%</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td>6</td><td>2.2%</td><td>366</td><td>100.0%</td><td>(63.1%-</td><td>100.0% (358/358)</td><td>(99.0% - 100.0%</td><td>3</td><td>100.0%</td><td>100.0%</td></tr><tr><td></td><td>7</td><td>2.9%</td><td>70</td><td>(8/8) 100.0%</td><td>100.0% (15.8%-</td><td>100.0%</td><td>(94.7%- 100.0%)</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td>MSS</td><td>1</td><td>15.3%</td><td>203</td><td>(2/2 100.0% (31/31)</td><td>100.0%) (88.8%- 100.0%)</td><td>(68/68) 100.0% (172/172)</td><td>(97.9%- 100.0%)</td><td>9</td><td>100.0%</td><td>100.0%</td></tr><tr><td rowspan="4"></td><td>2</td><td>42.1%</td><td>76</td><td>100.0% (32/32)</td><td>(89.1%- 100.0%)</td><td>95.5% (42/44)</td><td>(84.5%- 99.4%</td><td>10</td><td>94.1%</td><td>100.0%</td></tr><tr><td>4</td><td>33.7%</td><td>101</td><td>100.0% (34/34)</td><td>(89.7% - 100.0%</td><td>100.0% (67/67)</td><td>(94.6% - 100.0%</td><td>11</td><td>100.0%</td><td>100.0%</td></tr><tr><td>5</td><td>12.7%</td><td>71</td><td>10.0% (9/9)</td><td>(66.4% - 100.0%)</td><td>100.0% (62/62)</td><td>(94.2% - 100.0%</td><td>3</td><td>100.0%</td><td>100.0%</td></tr><tr><td>7</td><td>4.8%</td><td>21</td><td>100.0% (1/1)</td><td>(2.5% - 100.0%%</td><td>100.0% (20/20)</td><td>(83.2%- 100.0%)</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td>MNU6</td><td>1</td><td>15.3%</td><td>203</td><td>100.0% (31/31)</td><td>(88.8%- 1000%</td><td>100.0% (172/172)</td><td>(97.9% - 100.0%%</td><td>9</td><td>100.0%</td><td>100.0%</td></tr><tr><td></td><td>2</td><td>42.1%</td><td>76</td><td>100.0% (32/32)</td><td>(89.1%- 100.0%)</td><td>95.5% (42/44)</td><td>(84.5%- 99.4%)</td><td>10</td><td>94.1%</td><td>100.0%</td></tr><tr><td></td><td>4</td><td>33.7%</td><td>101</td><td>100.0% (34/34)</td><td>(89.7% - 100.0%)</td><td>98.5% (66/67)</td><td>(92.0%- 100.0%)</td><td>11</td><td>97.1%</td><td>100.0%</td></tr><tr><td></td><td>5</td><td>12.7%</td><td>71</td><td>100.0% (9/9)</td><td>(66.4% - 00.0%</td><td>100.0% (62/62)</td><td>(94.2% - 100.0%)</td><td>3</td><td>100.0%</td><td>100.0%</td></tr><tr><td></td><td>7</td><td>4.8%</td><td>21</td><td>100.0% (1/1)</td><td>(2.5% - 100.0%)</td><td>100.0% (20/20)</td><td>(83.2%- 100.0%</td><td>0</td><td>100.0%</td><td>100.0%</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">Prevalence</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">95% C.I.</td><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">95% C.I.</td><td colspan="1" rowspan="1"># CT(+)andGC (+)</td><td colspan="1" rowspan="1">PPV%</td><td colspan="1" rowspan="1">NPV%</td></tr><tr><td colspan="1" rowspan="1">MUPT7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">15.3%</td><td colspan="1" rowspan="1">203</td><td colspan="1" rowspan="1">100.0%(31/31)</td><td colspan="1" rowspan="1">(88.8% -100.0%</td><td colspan="1" rowspan="1">99.4%(171/172)</td><td colspan="1" rowspan="1">(96.8% -100.0%)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">96.9%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">42.1%</td><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">100.0%(32/32)</td><td colspan="1" rowspan="1">(89.1%-100.0%)</td><td colspan="1" rowspan="1">97.7%(43/44)</td><td colspan="1" rowspan="1">(88.0% -99.9%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">97.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">33.7%</td><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">100.0%(34/34)</td><td colspan="1" rowspan="1">(89.7%-100.0%)</td><td colspan="1" rowspan="1">100.0%(67/67)</td><td colspan="1" rowspan="1">(94.6% -100.0%)</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">12.7%4.8%</td><td colspan="1" rowspan="1">7121</td><td colspan="1" rowspan="1">100.0%(9/9)100.0%(1/1)</td><td colspan="1" rowspan="1">(66.4%-10.0%)(2.5%-100.0%</td><td colspan="1" rowspan="1">98.4%(61/62)100.0%(20/20)</td><td colspan="1" rowspan="1">(91.3% -100.0%(83.2% -100.0%</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">90.0%100.0%</td><td colspan="1" rowspan="1">100.0%100.0%</td></tr></table>

1 22 of the 65 FS PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathbf { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $100 \%$ (22/22).   
$^ 2 2 2$ of the 65 FV PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $100 \%$ (22/22).   
$^ 3 2 2$ of the 65 FNU PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $9 5 . 5 \%$ (21/22).   
$^ 4 2 2$ of the 65 FUPT PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $9 5 . 5 \%$ (21/22).   
$^ { 5 } 3 3$ of the 107 MS PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $100 \%$ (33/33).   
$^ 6 3 3$ of the 107 MNU PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $100 \%$ (33/33).   
$^ 7 3 3$ of the 107 MUPT PIS positive subjects were co-infected with CT. BD ProbeTec GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay sensitivity was $100 \%$ (33/33).

# Analysis of GC Positive/Negative Specimens from Female Subjects Based on Patient Infected Status

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>NAAT 1</td><td rowspan=1 colspan=2>NAAT 2</td><td rowspan=1 colspan=4>BD ProbeTec GC Q* AmplifiedDNA Assay</td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=2 colspan=1>PISGC</td><td rowspan=2 colspan=1>Endocervical Swab</td><td rowspan=2 colspan=1>Urine</td><td rowspan=2 colspan=1>Endocervical Swab</td><td rowspan=2 colspan=1>Urine</td><td rowspan=2 colspan=1>QxEndocervical Swab</td><td rowspan=2 colspan=1>QxVaginalSwab</td><td rowspan=2 colspan=1>NeatUrine</td><td rowspan=2 colspan=1>QxUPTUrine</td><td rowspan=1 colspan=3>SymptomaticStatus</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=7 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=5>Total PIS Positive</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=21 colspan=1></td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>ET</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>LE</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>390</td><td rowspan=1 colspan=1>484</td><td rowspan=1 colspan=1>874</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=9>Total PIS Negative</td><td rowspan=1 colspan=1>423</td><td rowspan=1 colspan=1>506</td><td rowspan=1 colspan=1>929</td></tr></table>

I Indeterminate E Error LE Liquid Level Error ET Extraction Transfer Error

# Analysis of GC Positive/Negative Specimens from Male Subjects Based on Patient Infected Status

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>NAAT 1</td><td rowspan=1 colspan=2>NAAT 2</td><td rowspan=1 colspan=3>BD ProbeTec GC QxAmplified DNA Assay</td><td rowspan=2 colspan=3>Symptomatic Status</td></tr><tr><td rowspan=2 colspan=1>PISGC</td><td rowspan=2 colspan=1>UrethralSwab</td><td rowspan=2 colspan=1>Urine</td><td rowspan=2 colspan=1>Urethra1 Swab</td><td rowspan=2 colspan=1>Urine</td><td rowspan=2 colspan=1>QxUrethralSwab</td><td rowspan=2 colspan=1>NeatUrine</td><td rowspan=2 colspan=1>QxUPTUrine</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>+</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>Total P</td><td rowspan=1 colspan=1>Total PIS Positive</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=17 colspan=1></td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>304</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=8>Total PIS Negative</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>365</td></tr></table>

4. Clinical cut-off: Clinical cut-off was estimated by evaluating GCQ assay results obtained from a total of 696 unique subjects for a total of 4878 assay results. This dataset included both the initial and repeat assay results: specimens were tested and then retested, and both results were included in the analysis in order to generate a larger data pool for use in training the algorithm. The optimal cutoff for the GCQ Assay was located between 50 and 250 MaxRFU based on the analysis of this dataset. When evaluating assay sensitivity and specificity over this range of values across all specimen types, the assay sensitivity and specificity were relatively constant at approximately 92 and $9 9 \%$ , respectively. In order to guard against false positive results (protect specificity), the preliminary cutoff was set at 125 MaxRFU. These preliminary cutoff values were incorporated into the version of algorithm software that was utilized in the Cutoff Trial by the BD Viper System operating in extracted mode.

5. Expected values/Reference range:

A. Prevalence: The prevalence of positive N. gonorrhoeae specimens in patient populations depends upon: clinic type, age, risk factors, gender, and test method. The prevalence observed with the GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay during a multi-center clinical trial ranged from $1 . 4 \%$ to $1 9 . 1 \%$ for female specimens and $4 . 8 \%$ to $4 2 . 1 \%$ for male specimens.

B. Positive and Negative Predictive Value: Hypothetical positive and negative predictive values (PPV & NPV) for the GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay are shown in the following table. These calculations are based on hypothetical prevalence and overall sensitivity and specificity (compared to the patient infected status) of $9 9 . 3 \%$ and $9 9 . 4 \%$ , respectively. In addition, PPV and NPV based on actual prevalence, sensitivity and specificity are shown in the tables under performance characteristics section (See “GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay Performance Compared to Patient Infected Status (by specimen type and symptomatic status)” and “GC $\mathrm { Q } ^ { \mathrm { x } }$ Assay Performance Compared to Patient Infected Status (by clinical site).” PPV was calculated using: (Sensitivity \* Prevalence) / (Sensitivity\*Prevalence $\mathrm { ~ + ~ } ( 1 \mathrm { ~ - ~ }$ Specificity) \* (1 - Prevalence)). NPV was calculated using: (Specificity \* (1 - Prevalence)/ (1-Sensitivity) \* Prevalence $^ +$ Specificity \* (1-Prevalence)).

GC Hypothetical Positive and Negative Predictive Values Compared to Patient Infected Status   

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Sensitivity(%)</td><td rowspan=1 colspan=1>Specificity(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>2510</td><td rowspan=1 colspan=1>99.399.399.3</td><td rowspan=1 colspan=1>99.499.499.4</td><td rowspan=1 colspan=1>78.390.395.2</td><td rowspan=1 colspan=1>100.0100.099.9</td></tr><tr><td rowspan=2 colspan=1>203040</td><td rowspan=1 colspan=1>99.399.3</td><td rowspan=2 colspan=1>99.499.499.4</td><td rowspan=2 colspan=1>97.898.799.2</td><td rowspan=2 colspan=1>99.899.799.5</td></tr><tr><td rowspan=1 colspan=1>99.3</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>99.3</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>99.3</td></tr></table>

# N. Instrument:

BD Viper™ System in extracted mode with the addition of the CTQ and GCQ Assays: The BD Viper System with the capability of automated nucleic acid extraction is the third generation of the BD Viper robotic platform for amplified DNA analysis. The system builds upon its predecessors, the BD Viper Instrument (K023955) and the BD Viper System (K052481).

# O. System Descriptions:

Viewing the BD Viper System from the perspective of assay workflow, the level of automation added to enable automated nucleic acid extraction on the existing BD Viper System includes the following:

(1) chemical lysis of organisms in clinical specimens, (2) chemical extraction and purification of DNA using paramagnetic particles facilitated by employment of an extractor block containing a movable magnet assembly;

(3) elution of extracted DNA into SDA-compatible buffer; and (4) transfer of the eluate from the extraction tube to the assay priming microwells.

Beyond these additions to the existing BD Viper System’s workflow, the following processing functions are common to both systems (extracted and non-extracted):

(5) priming microwell heat spike;   
(6) transfer of sample from priming microwells to prewarmed amplification microwells located directly on the reader stage/heater;   
(7) amplification microwell plate sealing and movement of the sealed amplification microwells into the fluorescent reader;   
(8) amplification temperature control and fluorescent photodetection; and   
(9) calculation and result interpretation.

2. Software: The FDA has reviewed applicant’s Hazard Analysis and Software development processes for the device

# P. Other Supportive Device and Instrument Information: NA

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

R. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.