# BD ProbeTecTM Neisseria gonorrhoeae (GC) Qx Amplified DNA Assay

# Applicant

BD Diagnostic Systems 7 Loveton Circle Sparks, MD 21152

# Establishment Registration No.

1119779

# Contact Person

Kathryn Babka Carr, RAC tel. 410-316-4260 fax. 410-316-4041 Kathy Carr@bd.com

# Summary Date

December 1, 2008

Proprietary Name

BD ProbeTecTM Neisseria gonorrhoeae (GC) Q" Amplified DNA Assay

# Generic Name

DNA probe, nucleic acid amplification, Neisseria

Classification Classification Name Regulation Number Product Code

Class II   
Neisseria spp. direct serological test reagents   
866.3390   
LSL

Predicate Devices

BD ProbeTec ET CT/GC Amplified DNA Assay (K984631), APTIMA Combo 2 Assay (K003395)

# Device Description

The BD ProbeTec GC $\mathsf { Q } ^ { \mathsf { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper™ System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. Aftr a brie incubation, the reaction mixture is transferred to a coresponding pre-warmed Amplifcation Microwell which is sealed to prevent contamination and then incubated in one of the two thermallycontrolled fluorescent readers. The presence or absence of N. gonorrhoeae DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified N. gonorrhoede target DNA, a second labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the N. gonorrhoeae-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is rehydrated

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper System and an automated algorithm is applied to both the EC and N. gonorrhoeae-specific signals to report results as positive, negative, or EC failure.

# Intended Use

The BD ProbeTecTM Neisseria gonorrhoeae (GC) $Q ^ { r }$ Amplified DNA Assay, when tested with the BD Viper™M System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Neisseria gonorrhoeae DNA in clinician-collected female endocervical and male urethral swab specimens, patientcollected vaginal swab specimens (in a clinical setting), and male and female urine specimens (both UPT and Neat). The assay is indicated for use with asymptomatic and symptomatic females and symptomatic males to aid in the diagnosis of gonococal urogenital disease.

# Summarv and Principles of Operation

When used with the BD Viper System, the BD ProbeTec GC $\mathsf { Q } ^ { \mathrm { x } }$ Amplified DNA Assay (GC $\mathsf { Q } ^ { \mathsf { x } }$ Assay) involves automated extraction of DNA from clinical specimens through the chemical lysis of cells, followed by binding of DNA to para-magnetic particles, washing of the bound nucleic acid and elution in an amplification-compatible buffer. When present, N. gonorrhoeae DNA is then detected by Strand Displacement Amplification (SDA) of a speciic targt sequence in the presence of a fluorescently labeled detector probe.

# Analvtical Performance Characteristics

# Limit of Detection (Analytical Sensitivity)

The Limits of Detection (LODs) for the $\mathrm { G C } \ : \mathrm { Q } ^ { \mathrm { x } }$ Assay with Neisseria gonorrhoeae strain ATCC 19424 in urine and swab specimens when extracted on the BD Viper System were determined to be $\leq 5 0$ cells per mL for neat and UPT treated urine and $\leq 1 0 0$ GC cells per mL for expressed vaginal and endocervical swab specimens. The GC Q" Assay on the BD Viper System in extracted mode was able to detect 17 GC strains with $\ge 9 5 \%$ proportion positive at a concentration of 50 cells per mL in $\mathsf { Q } ^ { \mathsf { x } }$ Swab Diluent.

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

# Analytical Specificity

The 141 organisms listed in Table 1 were tested with the BD ProbeTec GC $Q ^ { r }$ Amplified DNA Assay on the BD Viper System. All potential cross-reactive species were tested a $\mathrm { \sf t } \geq \mathrm { \sf t x i 0 } ^ { \mathrm { \sf g } } \mathrm { c e l l s / m L }$ except where noted. Two N. cinerea and two N. lactamica strains were shown to cross-react with the GC $Q ^ { x }$ assay.

Table1: Potential'Gros-reaconteMicroorgansms: wn s:sT E:erm t: : o6  

<table><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Epstein Barr Virus ***</td><td rowspan=1 colspan=1>Peptostreptococcus productus</td><td rowspan=1 colspan=1>Neisseria elongata subsp.nitroreduscens (2)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Neisseria elongata</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Flavobacterium meningosepticum</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Neisseria flava (4)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus***</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>Neisseria flavescens (4)</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophilia</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis*</td><td rowspan=1 colspan=1>Haemophilus influen-ue</td><td rowspan=1 colspan=1>Salmonella minnesota</td><td rowspan=1 colspan=1>Neisseria lactamica (7)</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis*</td><td rowspan=1 colspan=1>Herpes Simplex Virus **</td><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>Neisseria meningitidis (12)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Human papillomavirus (16 and 18)***</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Neisseria mucosa (5)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Staphvlococcus epidermilis</td><td rowspan=1 colspan=1>Neisseria perflava (8)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata*</td><td rowspan=1 colspan=1>Klebsiella pneumoniue</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Neisseria polvsuccharea (2)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis*</td><td rowspan=1 colspan=1>Lactobacillus acidophilus*</td><td rowspan=1 colspan=1>Streplococcus mitis</td><td rowspan=1 colspan=1>Neisseria sicca(5)</td></tr><tr><td rowspan=1 colspan=1>Chlamvdia pneumoniue ****</td><td rowspan=1 colspan=1>Lactobacillus brevis</td><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>Neisseria subflava(15)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia psittaci*</td><td rowspan=1 colspan=1>Lactobacillus jensenii</td><td rowspan=1 colspan=1>Streptococcus pneumoniae*</td><td rowspan=1 colspan=1>Neisseria weaveri (3)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Streplococcus pyogenes</td><td rowspan=9 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Wobiluncus mulieris</td><td rowspan=1 colspan=1>Streplomyces griseus*</td></tr><tr><td rowspan=1 colspan=1>Corvnebacterium renale</td><td rowspan=1 colspan=1>Moraxella lucunata*</td><td rowspan=1 colspan=1>Trichomonas vaginalis*</td></tr><tr><td rowspan=1 colspan=1>Crptococcus neoformans*</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus**</td><td rowspan=1 colspan=1>Morganella morgunii</td><td rowspan=1 colspan=1>Vibrio parahaemolvticus</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella urda</td><td rowspan=1 colspan=1>Wvcobacterium gordonge</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Mvcobucterium smegmatis</td><td rowspan=1 colspan=1>Branhamella catarrhalis (51</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Peptostreplococcus anaerobius</td><td rowspan=1 colspan=1>Neisseria cinerea (2)</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Peptostreptococcus usaccharolyticus</td><td rowspan=1 colspan=1>Neisseria elongata ssgivcolvtica</td></tr></table>

(n) number of strains tested in the BD ProbeTec GC $Q ^ { x }$ Assay $\ast$ Tested at $> | \mathrm {  ~ x ~ } | 0 ^ { 7 } \dot { \mathrm {  ~ c ~ e ~ } } | ] _ { \mathrm { \scriptsize { s } } / \mathrm { \scriptsize { m L } } }$ $\star \star \mathsf { T }$ ested at $> | \mathbf { \boldsymbol { x } } \mathbf { \boldsymbol { \updownarrow 0 } } ^ { \dagger }$ c   pare pe T $\geq \{ \mathbf { x } \vert \boldsymbol { \Lambda } \} ^ { \ast }$ genomic equivalents per mL; $\hslash \nearrow \nearrow \star$ tested at $\geq \mathrm { \ell | \times \ell | 0 ^ { 5 } \mathrm { T C } } [ \mathrm { D } _ { : \upsilon } / \mathrm { m L }$

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

# Interfering Substances

Potential interfering substances which may be encountered in swab and/or urine specimens were extracted from urine and vaginal swab matrix in the absence and presence of GC target (150 GC cells/mL for urine and 300 GC cells/mL for swabs) and tested with the BD ProbeTec GC $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay on the BD Viper System. Results are summarized in Table 2.

Table 2: Interfering Substances   
Interpretation   

<table><tr><td rowspan=1 colspan=1>No InterferenceObserved</td><td rowspan=1 colspan=1>Blood (≤ 60%)Seminal FluidMucusOver The Counter vaginal products andcontraceptivesHemorrhoidal creamPrescription vaginal treatmentsLeukocytes (1x106 cells/mL)1x106 cells/mL Chlamydia trachomatis</td><td rowspan=1 colspan=1>Blood (1%)Seminal fluidMucusAntibioticsAnalgesicsOver The Counter deodorant spraysand powdersHormonesLeukocytesAlbumin &lt;1 mg/ml.GlucoseAcidic urine (pH 4.0)Alkaline urine (pH 9.0)BilirubinOrganisms associated with UrinaryTract Infections</td></tr><tr><td rowspan=1 colspan=1>May causeextraction control(EC) failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not observed</td></tr></table>

# Clinical Performance Characteristics

Clinician-collected endocervical and male urethral swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and male and female $\mathsf { Q } ^ { \mathrm { r } }$ UPT and neat urine specimens were collected from 1059 female subjects and 479 male subjects attending OB/GYN, sexually transmitted disease (STD) and family planning clinics at seven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, urethral discharge, coital pain/difficulty/bleeding, testicular or scrotum pain/swelling, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms. Sixty five female subjects and 7 male subjects were excluded from the data analysis due to age requirement violations, antibiotic treatment in the last 21 days, opting to withdraw from the study ater initially consenting, failure to obtain paired swab and urine specimens, urine quantity less than $2 0 ~ \mathrm { m L }$ , or

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Qx Amplified DNA Assay

transport and storage errors related to specimen collection. Therefore, the final data analysis included 994 compliant female subjects and 472 compliant male subjects.

Five specimens were collected from each of the 994 eligible female subjects. A urine specimen was collected and split into $\mathbf { Q } ^ { \pi }$ UPT, neat urine and the two reference urine specimen collection devices followed by a vaginal swab specimen and three randomized endocervical swab specimens. Up to four specimens were collected from each of the 472 eligible male subjects. Up to three randomized urethral swab specimens were collected followed by a urine specimen that was split into Q UPT, neat urine and the two reference urine specimen collection devices. BD ProbeTec GC Q" assay results were generated from the $Q ^ { \star }$ UPT and neat urine specimens, the vaginal swab specimen, one endocervical swab specimen and one male urethral swab specimen. The remaining two endocervical swab specimens, up to two male urethral swab specimens, and the two reference urine specimens for each male and female subject were tested using two reference methods: the BD ProbeTec ET GC/AC assay and another commercially available NAAT (Nucleic Acid Amplification Test). Specimen testing was conducted either at the site of collection or at a designated BD Viper testing site.

All performance calculations were based on the total number of BD ProbeTec GC Q" assays results for endocervical, vaginal and male urethral swab specimens, and male and female $\mathsf { Q } ^ { \mathsf { r } }$ UPT and neat urine specimens compared to a patient infected status (PIS) algorithm for each gender. In the algorithm, the designation of a subject as being infected with GC or not was based on endocervical swab and urine specimen results from the commercially available BD ProbeTec ET GC/AC assay and the other commercially available NAAT. Subjects were considered infected with GC if two of the four endocervical swab and urine specimens (or two of the three or four urethral swab and urine specimens) tested positive in the BD ProbeTec ET GC/AC assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 5387 BD ProbeTec GC Q" assay results was used to calculate sensitivity and specificity. Sensitivity and specificity by specimen type and symptomatic status are presented in Table 3.

Table 3: GC $\pmb { \mathbb { Q } } ^ { \pmb { \tau } }$ Assay Performance Compared to Patient Infected Status (by specimen type and symptomatic status)   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Symptomatic</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.IL</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>PPV%</td><td rowspan=1 colspan=1>NPV%</td><td rowspan=1 colspan=1>Err orni ntial/Final</td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>96.3% (26/27)</td><td rowspan=1 colspan=1>(81.0% - 99.9%)</td><td rowspan=1 colspan=1>99.5% (421/423)</td><td rowspan=1 colspan=1>(98.3% - 99.9%)</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>3/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>542</td><td rowspan=1 colspan=1>100.0% (38/38)</td><td rowspan=1 colspan=1>(90.7% - 100.0%)</td><td rowspan=1 colspan=1>99.8% (503/504)</td><td rowspan=1 colspan=1>(98.9% - 100.0%)</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>2/2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>992</td><td rowspan=1 colspan=1>98.5% (64/65)</td><td rowspan=1 colspan=1>(91.7% - 100.0%)</td><td rowspan=1 colspan=1>99.7% (924/927)</td><td rowspan=1 colspan=1>(99.1% - 99.9%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>5/2</td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>100.0% (27/27)</td><td rowspan=1 colspan=1>(87.2% - 100.0%)</td><td rowspan=1 colspan=1>98.6% (416/422)</td><td rowspan=1 colspan=1>(96.9% - 99.5%)</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>544</td><td rowspan=1 colspan=1>100.0% (38/38)</td><td rowspan=1 colspan=1>(90.7% - 100.0%)</td><td rowspan=1 colspan=1>99.6% (504/506)</td><td rowspan=1 colspan=1>(98.6% - 100.0%)</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>100.0% (65/65)</td><td rowspan=1 colspan=1>(94.5% - 100.0%)</td><td rowspan=1 colspan=1>99.1% (920/928)</td><td rowspan=1 colspan=1>(98.3% - 99.6%)</td><td rowspan=1 colspan=1>89.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>96.3% (26/27)</td><td rowspan=1 colspan=1>(81.0% - 99.9%)</td><td rowspan=1 colspan=1>99.3% (420/423)</td><td rowspan=1 colspan=1>(97.9% - 99.9%)</td><td rowspan=1 colspan=1>89.7</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>97.4% (37/38)</td><td rowspan=1 colspan=1>(86.2% - 99.9%)</td><td rowspan=1 colspan=1>99.6% (503/505)</td><td rowspan=1 colspan=1>(98.6% - 100.0%)</td><td rowspan=1 colspan=1>94.9</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>96.9% (63/65)</td><td rowspan=1 colspan=1>(89.3% - 99.6%)</td><td rowspan=1 colspan=1>99.5% (923/928)</td><td rowspan=1 colspan=1>(98.7% - 99.8%)</td><td rowspan=1 colspan=1>92.6</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>100.0% (27/27)</td><td rowspan=1 colspan=1>(87.2% - 100.0%)</td><td rowspan=1 colspan=1>99.5% (421/423)</td><td rowspan=1 colspan=1>(98.3% - 99.9%)</td><td rowspan=1 colspan=1>93.1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>97.4% (37/38)</td><td rowspan=1 colspan=1>(86.2% - 99.9%)</td><td rowspan=1 colspan=1>99.8% (504/505)</td><td rowspan=1 colspan=1>(98.9% - 100.0%)</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>98.5% (64/65)</td><td rowspan=1 colspan=1>(91.7% - 100.0%)</td><td rowspan=1 colspan=1>99.7% (925/928)</td><td rowspan=1 colspan=1>(99.1% - 99.9%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0% (7/7)</td><td rowspan=1 colspan=1>(59.0% - 100.0%)</td><td rowspan=1 colspan=1>100.0% (208/208)</td><td rowspan=1 colspan=1>(98.2% - 100.0%)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100 0% (100/100)</td><td rowspan=1 colspan=1>(96.4% - 100.0%)</td><td rowspan=1 colspan=1>98.7% (155/157)</td><td rowspan=1 colspan=1>(95.5% - 99.8%)</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0% (107/107)</td><td rowspan=1 colspan=1>(96.65% - 100.0%)</td><td rowspan=1 colspan=1>99.5% (363/365)</td><td rowspan=1 colspan=1>(98.0% - 99.9%)</td><td rowspan=1 colspan=1>98.2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1>MN</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0% (7/7)</td><td rowspan=1 colspan=1>(59.0% - 100.0%)</td><td rowspan=1 colspan=1>100.0% (208/208)</td><td rowspan=1 colspan=1>(98.2% - 100.0%)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100.0% (100/100)</td><td rowspan=1 colspan=1>(96.4% - 100.0%)</td><td rowspan=1 colspan=1>98.1% (154/157)</td><td rowspan=1 colspan=1>(94.5% - 99.6%)</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0% (107/107)</td><td rowspan=1 colspan=1>(96.6% - 100.0%)</td><td rowspan=1 colspan=1>99.2% (362/365)</td><td rowspan=1 colspan=1>(97.6% - 99.8%)</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0% (7/7)</td><td rowspan=1 colspan=1>(59.0% - 100.0%)</td><td rowspan=1 colspan=1>99.5% (207/208)</td><td rowspan=1 colspan=1>(97.4% - 100.0%)</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>100.0% (100/100)</td><td rowspan=1 colspan=1>(96.4% - 100.0%)</td><td rowspan=1 colspan=1>98.7% (155/157)</td><td rowspan=1 colspan=1>(95.5% - 99.8%)</td><td rowspan=1 colspan=1>98.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0% (107/107)</td><td rowspan=1 colspan=1>(96.6% - 100.0%)</td><td rowspan=1 colspan=1>99.2% (362/365)</td><td rowspan=1 colspan=1>(97.6% - 99.8%)</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5387</td><td rowspan=1 colspan=1>99.3% (577/581)</td><td rowspan=1 colspan=1>(98.2% - 99.8%)</td><td rowspan=1 colspan=1>99.4% (4779/4806)</td><td rowspan=1 colspan=1>(99.2% - 99.6%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>6/2</td></tr></table>

A Asymptomatic CI Confidence Interval FNU Female Neat Urine FS Female endocervical swab FUPT Female urine in ${ \sf { Q } } ^ { \ast }$ UPT FV Female vaginai swab MNU Male Neat Urine MS Male urethral swab MUPT Male urine in $\alpha ^ { \prime }$ UPT n number s Symptomatic

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

A total of ${ 5 3 8 7 } \thinspace \mathrm { G C } \thinspace \mathrm { Q } ^ { x }$ Assay results was evaluated at seven geographically diverse clinical sites. A frequency distribution of the initial MaxRFU values for the GC $Q ^ { r }$ Assay with an assay cutoff of 125 MaxRFU is shown in Figure A.

![](images/3d4a3631bb0fad6b2708a625edcce7b959f26aea0a654f56fa078143cc0fdf76.jpg)  
Figure A: Frequency Distribution of MaxRFU for the GC $\mathbf { Q } ^ { \star }$ Assay

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Qx Amplified DNA Assay

# Reproducibility

Reproducibility of the BD Viper System using the BD ProbeTec GC Q\* Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTec GC Q\* Assay. Simulated endocervical and urethral specimens contained a clean endocervical swab whereas the simulated urine and vaginal swab specimens did not. Uninoculated swab diluent for the BD ProbeTec GC $\bar { \mathsf { Q } } ^ { \tt x }$ Assay was used for the GC negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in Table 4.

# Table 4: Summary of Reproducibility Data on the BD Viper System for the GC Q" Assay

![](images/f2d134f63229deaebe8b1fffeebd39f770dcb3b9b6db95e092fb330281faedf6.jpg)

<table><tr><td rowspan=5 colspan=1>EndocervicallUrethral</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99 3%(134/135)</td><td rowspan=1 colspan=1>(95.9%,100.0%)</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>151.3</td><td rowspan=1 colspan=1>1096.3</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>4.3</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5%(133/135)</td><td rowspan=1 colspan=1>(94.8%,99.3%)</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>220.7</td><td rowspan=1 colspan=1>785.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>120.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%.1000%)</td><td rowspan=1 colspan=1>1859.5</td><td rowspan=1 colspan=1>94 1</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>00</td><td rowspan=1 colspan=1>192</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%)</td><td rowspan=1 colspan=1>1847.3</td><td rowspan=1 colspan=1>117.6</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>a0</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100 0%(135/135)</td><td rowspan=1 colspan=1>(97.3%.10.0%)</td><td rowspan=1 colspan=1>1855.9</td><td rowspan=1 colspan=1>119 4</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>42.2</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=5 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99.3%(134/135)</td><td rowspan=1 colspan=1>(95.9%,00.0%)</td><td rowspan=1 colspan=1>15.7</td><td rowspan=1 colspan=1>162.3</td><td rowspan=1 colspan=1>1031 1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>00</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%)</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>295.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>69 7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>48.3</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%100 0%)</td><td rowspan=1 colspan=1>1899.0</td><td rowspan=1 colspan=1>86.1</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>00</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100.0%)</td><td rowspan=1 colspan=1>1884.2</td><td rowspan=1 colspan=1>94.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3%,100%)</td><td rowspan=1 colspan=1>1867 2</td><td rowspan=1 colspan=1>877</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>19 2</td><td rowspan=1 colspan=1>1.0</td></tr></table>

A sond study was conducted interally to characterize the reproducibility of test results (i.e, proportion positive or negative) at target levels below the analytical Limit of Detection (LOD) of the BD ProbeTec GC $\hat { \mathbf { Q } } ^ { \mathrm { v } }$ Assay. A panel of simulated specimens was tested that comprised GC and CT organisms seeded into $\tilde { \mathrm { Q } } ^ { \ast }$ swab diluent at two different levels each of which was below the respective analytical LOD for the organisms (1:10, 1:100). These levels were selected to fall within the dynamic range of the analytical LOD curve of the assay. Fifteen replicates of each panel member were tested every day for five days across three BD Viper Systems. The data are summarized in Table 5.

# 510(k) Summary

# BD ProbeTec™ Neisseria gonorrhoeae (GC) Q Amplified DNA Assay

Table 15: Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the GC $\mathbf { Q } ^ { \star }$ Assay   

<table><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>92.9(209/225)</td><td rowspan=1 colspan=1>(88.7, 95.9)</td><td rowspan=1 colspan=1>1324.6</td><td rowspan=1 colspan=1>7.1(16/225)</td><td rowspan=1 colspan=1>(4.1, 11.3)</td><td rowspan=1 colspan=1>41.4</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>30.7(69/225)</td><td rowspan=1 colspan=1>(24.7,37.1)</td><td rowspan=1 colspan=1>835.9</td><td rowspan=1 colspan=1>69.3(156/225)</td><td rowspan=1 colspan=1>(62.9, 75.3)</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>90.7(204/225)</td><td rowspan=1 colspan=1>(86.1, 94.1)</td><td rowspan=1 colspan=1>1165.9</td><td rowspan=1 colspan=1>9.3(21/225)</td><td rowspan=1 colspan=1>(5.9, 13.9)</td><td rowspan=1 colspan=1>34.2</td></tr><tr><td rowspan=1 colspan=1>UrineVaginal</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>22.7(51/225)</td><td rowspan=1 colspan=1>(17.4,28.7)</td><td rowspan=1 colspan=1>872.7</td><td rowspan=1 colspan=1>77.3(174/225)</td><td rowspan=1 colspan=1>(71.3,82.6)</td><td rowspan=1 colspan=1>7.8</td></tr></table>

# Conclusions

The analytical and clinical study results for the BD ProbeTec Neisseria gonorrhoeae (GC) Q\* Amplified DNA Assay support the determination of substantial cquivalence in accordance with the intended use as stated in he pdc beng.

Ms. Kathryn Babka Carr   
Regulatory Affairs Specialist   
BD Diagnostics Systems   
Becton, Dickinson and Company   
7 Loveton Circle   
Sparks, MD 21152

# DEC 11 2008

Re: K081825 Trade/Device Name: BD ProbeTec™ Neisseria gonorrhoeae (GC) $Q ^ { X }$ Amplified DNA Assay Regulation Number: 21 CFR 866.3390 Regulation Name: Neisseria spp. direct serological test reagents Regulatory Class: Class II Product Code: LSL Dated: December 5, 2008 Received: December 9, 2008

Dear Ms. Babka Carr:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

I yur device s cass eebveinto either class I Special Contols)orclass I MA) it y e subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

Page 2 -

This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at 240-276-0450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). For questions regarding postmarket surveillance, please contact CDRH's Office of Surveillance and Biometric's (OSB's) Division of Postmarket Surveillance at 240-276-3474. For questions regarding the reporting of device adverse events (Medical Device Reporting (MDR)), please contact the Division of Surveillance Systems at 240-276-3464. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (240) 276-3150 or at its Internet address http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

Sall attoy

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices Office of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and   
Radiological Health

Enclosure

# Indications for Use

510(k) Number: K081825

Device Name: BD ProbeTecTM Neisseria gonorrhoeae (GC) $\boldsymbol { \mathrm { Q } } ^ { \mathrm { x } }$ Amplified DNA Assay

Indications For Use:

The BD ProbeTec Neisseria gonorrhoeae (GC) $\mathsf { Q } ^ { \boldsymbol { \mathsf { x } } }$ Amplified DNA Assay, when tested with the BD ViperTM System in extracted mode, uses Strand Displacement Amplification (SDA) technology for the direct, qualitative detection of Neisseria gonorrhoeae DNA in cliniciancollected female endocervical and male urethral swabs, patient-collected vaginal swab specimens (in a clinical setting), and female and male urine specimens. The assay is indicated for use with asymptomatic and symptomatic female individuals and symptomatic male individuals to aid in the diagnosis of gonococcal urogenital disease.

# (PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE IF NEEDED)

Concurrence of CDRH, Office of In Vitro Diagnostic Devices (OIVD)

![](images/7d80a7f56a1e48c9c33100d4e05e20e9d3cf6bae96d2736f3d8851f059a86d91.jpg)

Office of In Vitro Diagnostic Device   
Evaluation and Safety   
510(k)- K081825