# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K082688

# B. Purpose for Submission:

This is a new $5 1 0 \mathrm { k }$ application for a qualitative real-time reverse transcription-polymerase chain reaction (RT-PCR) assay used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version $1 . 7 6$ or 3.0a for the qualitative detection of human Metapneumovirus (hMPV) nucleic acids isolated and purified from nasopharyngeal swab (NP) specimens obtained from symptomatic patients. The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# C. Measurand:

Target RNA sequences for the highly conserved regions of the Nucleocapsid (N) gene for hMPV, and for the transcript derived from E. coli Bacteriophage MS2 A-protein gene (Internal Control).

# D. Type of Test:

Real-time reverse transcription-polymerase chain reaction (RT-PCR), qualitative determination of human Metapneumovirus (hMPV) in nasopharyngeal swabs using nucleic acid isolation (the isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux)), amplification and detection on the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a, which generates signals based on the acquisition of spectrofluorometric data.

# E. Applicant:

Prodesse Incorporated

F. Proprietary and Established Names:

Prodesse Pro hMPV $+$ Assay

# G. Regulatory Information:

1. Regulation section: 866.3980

2. Classification: Class II

3. Product code: OEM

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Pro $\mathrm { { h M P V + } }$ Assay is a Real Time RT-PCR in vitro diagnostic test for the qualitative detection of human Metapneumovirus (hMPV) nucleic acid isolated and purified from nasopharyngeal swab (NP) specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infection. This assay targets a highly conserved region of the Nucleocapsid gene of hMPV. The detection of hMPV nucleic acid from symptomatic patients aids in the diagnosis of human respiratory hMPV infection if used in conjunction with other clinical and laboratory findings. This test is not intended to differentiate the four genetic sub-lineages of hMPV.

Negative results do not preclude hMPV infection and should not be used as the sole basis for diagnosis, treatment or other management decisions.

2. Indication(s) for use: Same as Indication(s) for use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

To be used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version $1 . 7 6$ or 3.0a and either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# I. Device Description:

The Pro $\mathrm { { h M P V + } }$ Assay is a Taqman based Real Time RT-PCR Assay that enables detection of human Metapneumovirus and Internal Control.

An overview of the procedure is as follows:

1. Collect nasopharyngeal swab specimens from symptomatic patients using a polyester, rayon or nylon tipped swab and place into viral transport medium (not provided with kit).

2. Add an Internal Control (IC) to every sample to monitor for inhibitors present in the specimens.

3. Perform extraction and purification of nucleic acids using a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

4. Add purified nucleic acids to Pro hMPV $^ +$ Supermix along with enzymes included in the Assay kit. The Pro hMPV $^ +$ Supermix contains oligonucleotide primers and target-specific oligonucleotide probes. The primers and probe are complementary to highly conserved regions of the Nucleocapsid gene of hMPV. The probes are duallabeled with a reporter dye attached to the $5 ^ { \circ }$ -end and a quencher dye attached to the $3 ^ { \circ }$ -end (see table below).

5. Perform reverse transcription of RNA into complementary DNA (cDNA) and subsequent amplification of DNA in a Cepheid SmartCycler® II instrument. In this process, the probe anneals specifically to the cDNA template followed by primer extension and amplification. The $5 ' - 3 '$ exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

# Pro hMPV+ Assay Analyte Gene Targets and Probe Labels:

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Gene Targeted</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=1>EmissionPeak</td><td rowspan=1 colspan=1>InstrumentChannel</td></tr><tr><td rowspan=1 colspan=1>humanMetapneumovirus</td><td rowspan=1 colspan=1>Nucleocapsid</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>651nm</td><td rowspan=1 colspan=1>674nm</td><td rowspan=1 colspan=1>Cy5</td></tr></table>

Materials Provided (Pro hMPV+ Assay Kit (Cat. # H27VK00))   

<table><tr><td colspan="1" rowspan="1">Reagents</td><td colspan="1" rowspan="1">Description</td><td colspan="1" rowspan="1">Quantity/Tube</td><td colspan="1" rowspan="1">CapColor</td><td colspan="1" rowspan="1">Cat. #</td><td colspan="1" rowspan="1">Reactions/Tube</td></tr><tr><td colspan="1" rowspan="1">hMPV Mix II</td><td colspan="1" rowspan="1">•  Taq DNA polymerase•  2 oligonucleotide primer pairs•  2 oligonucleotide probes•  Buffer containing dNTPs (dATP,dCTP, dGTP, dTTP),•  MgCl2 and stabilizers</td><td colspan="1" rowspan="1">1030 μL</td><td colspan="1" rowspan="1">Brown</td><td colspan="1" rowspan="1">HSM47</td><td colspan="1" rowspan="1">50(2 tubesprovided)</td></tr><tr><td colspan="1" rowspan="1">M-MLV ReverseTranscriptase II</td><td colspan="1" rowspan="1">•  11.4 U/μL</td><td colspan="1" rowspan="1">30 μL</td><td colspan="1" rowspan="1">White</td><td colspan="1" rowspan="1">GLS32</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">RNase Inhibitor II</td><td colspan="1" rowspan="1">• 40 U/μL</td><td colspan="1" rowspan="1">120 μL</td><td colspan="1" rowspan="1">Green</td><td colspan="1" rowspan="1">GLS33</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">hMPV RNAControl III</td><td colspan="1" rowspan="1">•  Non-infectious in vitro transcribedRNA specific viral sequences</td><td colspan="1" rowspan="1">500 μL</td><td colspan="1" rowspan="1">Purple</td><td colspan="1" rowspan="1">HCT47</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">Internal RNAControl III</td><td colspan="1" rowspan="1">•   Non-infectious in vitro transcribedRNA</td><td colspan="1" rowspan="1">30 μL</td><td colspan="1" rowspan="1">Yellow</td><td colspan="1" rowspan="1">GCT12</td><td colspan="1" rowspan="1">100</td></tr></table>

# Materials Required But Not Provided

# Plasticware and consumables

• Polyester, rayon or nylon tipped nasopharyngeal swabs RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes Sterile RNase/DNase-free filter or positive displacement micropipettor tips MagNA Pure LC System Disposables (Reagent Tubs, Reaction Tips, Tip Trays, Cartridges) or easyMAG System Disposables (Sample Vessels and Tips) Biohit Pipette Tips for use with easyMAG System Greiner Break Four uncoated plates for use with easyMAG System Cepheid PCR reaction tubes, $2 5 ~ \mu \mathrm { L }$ Parafilm $^ { \mathfrak { B } } \mathbf { M }$ or MagNA Pure LC Cartridge Seals

# Reagents

Roche MagNA Pure LC Total Nucleic Acid Isolation Kit (Roche Cat. # 03038505001) for 192 isolations or bioMérieux NucliSENS easyMAG reagents (Buffer 1 Cat. # 280130, Buffer 2 Cat. # 280131, Buffer 3 Cat. # 280132, Magnetic Silica Cat. # 280133, and Lysis Buffer Cat. #. 280134)   
Micro TestTM M4 Viral Transport Medium (Remel, Inc. Cat. # 12500), Copan Mini UTM (Copan Diagnostics Inc. Cat. # 350C ), or BD Universal Viral Transport Medium (UTM; Becton, Dickinson and Co. Cat. # 220220)   
Molecular Grade Water (RNase/DNase Free)   
Extraction Control (e.g. previously characterized positive sample or negative sample spiked with a well characterized hMPV strain)

# Equipment

- $. 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ Freezer Roche MagNA Pure LC System with software version 3.0.11 or bioMérieux NucliSENS easyMAG System with Software version 1.0.1 or 2.0   
• Biohit multi-channel pipettor for use with easyMAG System   
• Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 \mu \mathrm { L }$ and $1 0 0 { - } 1 0 0 0 \mu \mathrm { L } )$ Mini-centrifuge with adapter for Cepheid Reaction Tubes   
• Cepheid cooling block

# Interpretation of Sample Results

The SmartCycler Dx software automatically determines the specimen results. The interpretation of

the assay specimen results is as follows:

<table><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>ICResult</td><td rowspan=1 colspan=1>WarningErrorCode</td><td rowspan=1 colspan=1>hMPVResult</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>hMPV nucleic acid not detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>hMPV nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Unresolved - PCR inhibition orreagent failure. Repeat testingfrom the purified nucleic acid orcollect and test a new sample.</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>30792</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined  error code 3079</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40983</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined — error code 4098</td></tr></table>

Columns and data not used for interpretation are not included Error Code 3079: Warning/Error Code 3079 is periodically observed with hMPV positive samples. Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a $C t$ value $\geq I 3$ is reported in the hMPV Ct column, the sample results can be recorded as POS for hMPV. An Invalid assay run will display Error Code 4098 \* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High viral load can lead to reduced or absent Internal Control signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ID-Tag Respiratory Virus Panel, Luminex Molecular Diagnostics, Inc.

2. Predicate K number(s): (K063765)

3. Comparison with predicates:

Both assays detect hMPV using nucleic acid amplification techniques. Both assays use nasal pharyngeal swabs as the collection device and the MagNA Pure LC system for nucleic acid isolation. In addition, the Pro hMPV+ Assay can also utilize the bioMérieux NucliSENS easyMAG System for nucleic acid extraction. The detection system with both assays involves spectrophotometric detection. The assays differ in that the predicate is a multiplex nucleic acid amplification and end-point detection assay also detects Influenza A, Influenza B, RSV, Influenza A subtypes H1 and H3, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Rhinovirus, and Adenovirus. The Pro hMPV $+$ assay is a real-time nucleic acid amplification and detection assay detects hMPV only.

# K. Standard/Guidance Document Referenced (if applicable):

Special controls guidance documents will be promulgated

Guidance on Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses (March 2006) – http://www.fda.gov/cdrh/oivd/guidance/1596.html Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) – http://www.fda.gov/cdrh/oivd/guidance/1588.html Draft Guidance on Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens (Dec 2005) – http://www.fda.gov/cdrh/oivd/guidance/1560.html Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Guidance for Industry and FDA Reviewers (March 2007) – http://www.fda.gov/cdrh/osb/guidance/1620.html Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff – http://www.fda.gov/cdrh/ode/guidance/1567.html Guidance for Off-the-Shelf Software Use in Medical Devices; Final (Sept 1999) – http://www.fda.gov/cdrh/ode/guidance/585.html Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (Feb 2008) – http://www.fda.gov/cdrh/oivd/guidance/1638.pdf Software Guidance for the content of premarket submissions for software contained in medical devices (May 2005) – http://www.fda.gov/cdrh/ode/guidance/337.html • General Guidance on Software Validation (Jan 2002) – http://www.fda.gov/cdrh/comp/guidance/938.html CLSI EP17-A; CLSI MM13-A; CLSI EP7-A2; CLSI EP12-A

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. The Pro hMPV+ Assay enables detection of hMPV and the Internal Control RNA and is based on three processes: nucleic acid isolation, reverse transcription and Real Time PCR amplification/detection. Human respiratory specimens (nasopharyngeal swabs) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. Each purified nucleic acid sample is added to the Pro hMPV+ Supermix (includes Taq polymerase) along with the appropriate enzymes (reverse transcriptase and RNase Inhibitor). The Pro hMPV+ Supermix contains oligonucleotide primers complementary to highly conserved regions of the Nucleocapsid gene for hMPV and targetspecific oligonucleotide probes dual-labeled with a reporter dye and a quencher dye. After initial reverse transcription of RNA into complementary DNA (cDNA), amplification proceeds during which the probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This generates an increase in fluorescent signal upon excitation from an LED light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time.

Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

An Inter-Laboratory Reproducibility study was conducted by extracting and testing a panel of 9 samples at three sites performed by two operators at each site for 5 days per operator (for a total of 10 runs per site). The Reproducibility Panel used in the study was prepared by spiking nasopharyngeal (NP) swab pools with cultured and titered stock solutions of human Metapneumovirus (hMPV) subtypes A2 (strain Iowa 14) and B2 (strain Iowa 8) at low positive (2x LoD), medium positive (10x LoD) or high negative $( 0 . 0 1 \mathrm { x } \ : \mathrm { L o D } )$ concentrations. Panel members were coded and randomly sorted to make 9- sample test panels and to ensure sample identification was unknown to the operators. The 9-sample test panels were stored at $\mathrm { \leq - 7 0 ^ { \mathrm { o } } C }$ and shipped to the sites frozen.

A set of ten 9-sample test panels (15 samples for each of the 6 reproducibility panel members x 6 reproducibility panel members $= 9 0$ total samples) was provided to each laboratory; one test panel was used for each of the 5 testing days per operator. Each operator thawed one 9-sample test panel per day, spiked the samples and a Negative Control with the Internal Control, performed nucleic acid extraction on the test panel samples using either the Roche MagNA Pure LC system (Clinical Trial Site #4) or the bioMérieux NucliSENS easyMAG (Site #1 and Site #2). An Extraction Control, noninfectious hMPV strain (A2 lineage) obtained from ZeptoMetrix Corp. (Buffalo, NY), was also included with each extraction run. Extracted nucleic acid was tested with the Pro hMPV $+$ Assay on the Cepheid SmartCycler II. The Positive hMPV RNA Control was also included with each test panel run. Each of two operators performed these activities on five days for a total of 10 reproducibility runs per site. A single lot of Pro $\mathrm { { h M P V + } }$ reagents was used. A total of 360 data points were included in the reproducibility study data analysis (12 samples and controls/run X 1 run/day/operator X 2 operators X 5 days X 3 sites $= 3 6 0$ ). The total percent agreement for the Pro hMPV+ Assay was $9 9 . 2 \%$ . The Pro $\mathrm { { h M P V + } }$ Assay is a qualitative assay based partially on numerical Cycle Threshold $\mathrm { ( C t ) }$ values. The overall $\mathrm { C t }$ value $\% \mathrm { C V }$ across all sites for all samples and controls ranged from $1 . 7 2 \%$ to $7 . 3 6 \%$ depending upon analyte type, target type, and concentration tested.

<table><tr><td rowspan=3 colspan=1></td><td rowspan=1 colspan=1>PanelMember ID</td><td rowspan=1 colspan=1>hMPVA2HighNegativea</td><td rowspan=1 colspan=1>hMPVA2LowPositive</td><td rowspan=1 colspan=1>hMPVA2Moder-atePositive</td><td rowspan=1 colspan=1>hMPVB2HighNegativea</td><td rowspan=1 colspan=1>hMPVB2LowPositive</td><td rowspan=1 colspan=1>hMPVB2Moder-atePositive</td><td rowspan=1 colspan=1>hMPVRNACon-trol</td><td rowspan=1 colspan=1>Nega-tiveCon-trola</td><td rowspan=1 colspan=1>ExtractionControlhMPVA2</td><td rowspan=3 colspan=1>Total %Agree-ment</td></tr><tr><td rowspan=2 colspan=1>Concen-tration</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>2 X LoD</td><td rowspan=1 colspan=1>10 XLoD</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>2 XLoD</td><td rowspan=1 colspan=1>10 XLoD</td><td rowspan=2 colspan=1>NA</td><td rowspan=2 colspan=1>NA</td><td rowspan=2 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>1 x 10TCID50/mL</td><td rowspan=1 colspan=1>2 x 102TCID50/mL</td><td rowspan=1 colspan=1>1 x 103TCID50/mL</td><td rowspan=1 colspan=1>1 x 10-1TCID50/mL</td><td rowspan=1 colspan=1>2 x 101TCID50/mL</td><td rowspan=1 colspan=1>1 x 102TCID50/mL</td></tr><tr><td rowspan=3 colspan=1>Site1</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>10/10(100)%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>33.1</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>2.84</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>2.20</td><td rowspan=1 colspan=1>1.68</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>2.73</td></tr><tr><td rowspan=3 colspan=1>Site2</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>15 /15(100%)</td><td rowspan=1 colspan=1>14/15(93.3%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>13/15(86.7%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>117/120(97.5%)</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>25.6</td><td rowspan=1 colspan=1>32.9</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>3.95</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>4.14</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>4.86</td></tr><tr><td rowspan=3 colspan=1>Site4</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>10/10(10)(%</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>120/120(100%)</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>28.8</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>3.80</td><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>3.08</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwithExpectedresult</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>44/45(97.8%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>43/45(95.6%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>30/30(100)%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>357/360(99.2%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>92.1% -100%</td><td rowspan=1 colspan=1>88.4% -99.6%</td><td rowspan=1 colspan=1>92.1% -100%</td><td rowspan=1 colspan=1>92.1% -100%</td><td rowspan=1 colspan=1>85.2% -98.8%</td><td rowspan=1 colspan=1>92.1% -100%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>88.6% -100%</td><td rowspan=1 colspan=1>97.6% -99.7%</td></tr><tr><td rowspan=1 colspan=1>OverallAverage CtVValue</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>27.63</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>31.6</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall %CV</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>3.73</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>3.29</td><td rowspan=1 colspan=1>3.97</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>3.49</td><td rowspan=1 colspan=1>7.36</td></tr><tr><td rowspan=1 colspan=12>aAverage Ct value calculated for the Internal Control (IC)</td></tr></table>

a Average Ct value calculated for the Internal Control (IC)

b. Linearity/assay reportable range: Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

The following controls are provided in the Pro hMPV $^ +$ Assay kit:

Positive Control (PC): The Pro $\mathrm { { h M P V + } }$ Assay kit contains an hMPV positive RNA control that is an RNA transcript of a plasmid containing the viral sequence of interest and is not intact virus particles. The PC does not go through nucleic acid extraction and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance. The PC is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc). As such, wide acceptance criteria were desired. The typical range for the PC in preliminary testing was in the low-mid $3 0 ^ { \circ } { \mathsf { s } } .$ . Therefore, the acceptable range was set to 25.0 to 40.0 in the Clinical Trial Protocol to allow for dilution variability (e.g. pipetting) of the PC by the user. Of a total of 69 PC tested by the Pro $\mathrm { { h M P V + } }$ Assay during the clinical trial, $100 \%$ (69/69) of these controls gave correct results. The average $\mathrm { C t }$ of the PCs tested is 31.9 (Min 29.4- Max 33.6), with 0.68 Standard Deviation (STDEV) and $2 . 1 3 \%$ CV. The pivotal clinical study data validated the pre-determined PC Ct acceptance range of 25.0 to 40.0.

Internal Control (IC): An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working. The typical range for the IC in preliminary testing was in the mid- ${ \cdot } 2 0 \mathrm { { ^ { \circ } s } }$ . To accommodate for dilution variability (e.g. pipetting) by the user in spiking the IC, and also accommodate slight inhibition, the range was set to 15.0 to 47.0 in the Clinical Trial Protocol. The average Ct of all eligible Pro $\mathrm { { h M P V + } }$ negative specimens is 26.98 (Min 23.90- Max 46.0), with 2.02 Standard Deviation (STDEV) and $7 . 4 8 \%$ CV. The pivotal clinical study data validated the pre-determined IC $\mathrm { C t }$ acceptance range of 15.0 to 47.0. A new IC Ct acceptable range of 17.0 to 40.0 was set based on Clinical Trial and (QC) Test Method Characterization data.

These RNA transcripts (PC and IC) were serially diluted in molecular grade water containing RNase Inhibitor at $1 \mathrm { U / \mu L }$ and made into the manufactured product included in the Pro $\mathrm { { h M P V + } }$ Assay kit. They are tested for final release by performing a prescribed dilution and using the $\mathrm { P r o } \ h \mathrm { M P V ^ { + } }$ Assay to ascertain the cycles at which these dilutions cross the threshold (cutoff). The specifications for final release are acceptable ranges of cycles at which the RNA control dilution can cross the threshold:

<table><tr><td rowspan=1 colspan=3>Kit Control Final Release Specifications</td></tr><tr><td rowspan=1 colspan=1>RNA Control</td><td rowspan=1 colspan=1>Lower SpecificationLimit (PCR cycles)</td><td rowspan=1 colspan=1>Upper SpecificationLimit (PCR cycles)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>26.76</td><td rowspan=1 colspan=1>38.67</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>21.83</td><td rowspan=1 colspan=1>32.67</td></tr></table>

The following controls are not provided in the Pro $\mathrm { { h M P V + } }$ Assay kit, but are required or recommended and described in the Pro hMPV $^ +$ Assay Instructions for Use :

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the Pro hMPV+ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination. Of a total of $8 8 \mathrm { N C }$ tested by the Pro hMPV $^ +$ Assay during the clinical trial, $100 \%$ (88/88) of these controls gave correct results. The average Ct of the IC in tested NC is 25.7 (Min 24.2 - Max 29.5), with 1.01 Standard Deviation (STDEV) and $3 . 9 3 \%$ CV.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a non-infectious hMPV strain (A2 lineage) was included with each extraction run as EC. Of a total of $8 7 \mathrm { E C }$ tested by the Pro $\mathrm { { h M P V + } }$ Assay during the clinical trial, $100 \%$ (87/87) of these controls gave correct results. The average $\mathrm { C t }$ of the EC is 32.7 ( $\mathrm { M i n } 2 7 . 9 - \mathrm { M a x } 3 7 . 3 )$ , with 2.04 Standard Deviation (STDEV) and $6 . 2 4 \%$ CV. The sponsor is recommending an extraction control in each nucleic acid extraction run to the end users in the package insert.

# Freeze/Thaw and Stability

Reagents and Controls stability

An Accelerated Stability study concluded that all Pro $\mathrm { { h M P V + } }$ reagents and controls (closed and open tubes) can be stored at $\mathrm { \le - 7 0 ^ { \mathrm { o } } C }$ for up to 18 months. A Freeze/Thaw Study of Pro $\mathrm { { h M P V + } }$ Supermix and enzymes included with the kit demonstrated that components can undergo up to 5 freeze-thaws and Controls (Positive and Internal Control) can undergo up to 2freeze-thaws. For the Controls, the Pro hMPV $+$ Assay Package Insert specifies that controls should not undergo more than 1 freeze-thaw cycle. Although internal studies demonstrated that up to 2 freeze-thaws of controls would not adversely affect performance, in an effort to mitigate risk it is recommended they not undergo more than 1 freeze-thaw.

A real time stability plan has been approved and is in the early stages of implementation. One master lot of reagents has been QC tested. The QC test acts as the initial time point for RT stability testing. The second time point has not been reached. Two additional master lots of product will be added to the real time stability study (per the plan) as they are added to the production schedule.

Specimen stability

The Pro hMPV $^ +$ Assay recommends that samples be stored refrigerated $( 2 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C } )$ for up to 72 hours prior to processing the samples. This recommendation is supported by a) The CLSI Guidance: Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (MM13) recommends that bronchioalveolar lavage samples be refrigerated for up to 72 hours or frozen (at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } ) }$ ) for longer periods prior to processing for molecular methods; b) A 72 hour timeframe is widely recommended and used (CLSI Guidance M41 Viral Culture) for viral culture. c) The viral transport medium that nasopharyngeal swab samples are collected in also serves to further

“preserve” and “stabilize” viruses, to maintain their viability for culture and NAAT. Furthermore, the average time to extraction was 37 hours during the clinical trial. The distribution of the times is tri-modal, reflecting the impact of working hours on the times to extraction. Site 4 has the longest average time to extraction (47 hours) while Site 1 has the shortest time (31 hours). There was no observed impact of times to extraction on hMPV or IC Ct values.

The Pro hMPV+ Assay also recommends that extracted nucleic acid samples can be stored at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ until RT-PCR testing, and they should be tested after no more than one freeze-thaw cycle. This recommendation is supported by a) Both CLSI guidance documents MM13 (Collection, Transport, Preparation and Storage of Specimens for Molecular Methods) and MM17 (Verification and Validation of Multiplex Nucleic Acid Assays) recommend storage of isolated RNA at $- 8 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ and limiting the number of freeze-thaw cycles before testing; b) A study that compared the extracted nucleic acid isolated from the Extraction Control (EC) during the clinical trial being tested “fresh” or after undergoing one freeze-thaw cycle was conducted. Sites 1 and 2 ran both “fresh” and “frozen” EC nucleic acids and t-test analysis demonstrated that there is no significant difference between the two conditions $( \mathtt { p } > 0 . 0 5$ for each site); c) A subset of hMPV positive samples from the clinical trial were re-run from the purified nucleic acid at the same time the reference assays were being set up. All of the samples in the subset were initially run as “fresh” nucleic acids and the repeat testing was indicative of 1 freeze-thaw cycle. Although a paired $t \cdot$ -test analysis indicates that the mean $\mathrm { C t }$ values of hMPV-positive sample nucleic acids that have undergone one freeze-thaw cycle are statistically different $( \mathrm { p } { < } 0 . 0 5 )$ than the mean $\mathrm { C t }$ values of “fresh” hMPV-positive sample nucleic acids, the mean $\mathrm { C t }$ difference is clinically insignificant (mean $= 0 . 2 2$ ; range $= 0 - 1 . 5 \mathrm { C t }$ ).

# d. Analytical Sensitivity (Detection limit):

# LoD studies using the Roche MagNA Pure LC

The analytical sensitivity (limit of detection or LoD) of the Pro $\mathrm { { h M P V + } }$ Assay was determined using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of 2 hMPV strains (hMPV A2 Iowa 14 and hMPV B2 Iowa 8) serially diluted in negative nasopharyngeal (NP) pools prepared from leftover NP swab clinical samples. Each viral strain was previously cultured and titered to determine its stock concentration in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Each viral stock was spiked into negative NP swab pools at concentrations of 1 log above, 1 log below, and at the estimated LoD for that virus (based on preliminary studies). Two dilution series were prepared for each viral stock by two operators for a total of 4 dilution series per virus. An Internal RNA Control (IC) was spiked into all dilution series samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Each dilution series was extracted on the Roche MagNA Pure LC and tested in quintuplicate reactions using two lots of Pro $\mathrm { { h M P V + } }$ reagents on four Cepheid SmartCycler II instruments. This resulted in a total of 20 data points per viral strain dilution. A negative control, which consisted of negative NP swab matrix spiked with IC was included with each set of dilution series. Nucleic acid isolation of the negative control was performed along with the viral strain dilution series. The negative control served to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The hMPV Positive RNA Control, non-infectious in vitro transcribed RNA of a specific viral sequence, was included with each RNA detection run to test for procedural errors (absence of reagent, instrument failure, etc.).

Analytical sensitivity was determined for each strain. The LoD was defined as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 19 out of 20 replicates were determined to be positive). The data are presented in the following table:

Analytical Sensitivity Results Combined (Lots #1 and #2)   

<table><tr><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageC</td><td rowspan=1 colspan=1>StandardDeviationCT</td><td rowspan=1 colspan=1>Min CT</td><td rowspan=1 colspan=1>Max CT</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=3 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>10{</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>38.2*</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>37.7*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=3 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36.5*</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>10-1</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0%</td></tr></table>

Data from Operators 1 and 2 pooled. ND – Not Detected. NA – Not Applicable $^ { * } \mathrm { N } < 2 0$ . Average based on number of replicates detected.

The sponsor chose to claim LoDs of $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and $1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for the hMPV A2 and B2 strains tested, respectively

# Extraction Equivalency Study

To evaluate the equivalency of nucleic acid extraction using the Roche MagNA Pure LC (used throughout Pro hMPV+ Assay development and Analytical Studies, Clinical Trials and Reproducibility) and the bioMérieux NucliSens easyMAG automated extraction instruments (used in Clinical Trials and Reproducibility) for use with the Pro $\mathrm { { h M P V + } }$ Assay, an extraction equivalency study was carried out to determine and compare the Limits of Detection (LoDs) for human Metapneumovirus (hMPV) on each automated extractor.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles.

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run.

Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads.

For the Extraction Equivalency LoD Study, single cultured and titered strains of hMPV A2 (Iowa 14 strain) and hMPV B2 (Iowa 8 strain) were spiked (along with an IC) into individual aliquots of negative nasopharyngeal swab (NP) matrix pools at concentrations of 1 log above, at, and 1 log below the previously determined LoD (see LoD studies using the Roche MagNA Pure LC). Each viral strain dilution was extracted in replicates of 10 on each automated extractor. An Extraction Control and a Negative Control (NP spiked with Internal Control - NPIC) were included in each run on each extractor. All samples of the same virus strain and same dilution were extracted on the same day in the same extraction run. All extracted nucleic acid samples were stored at $\leq$ - $- 7 0 \mathrm { { ^ \circ C } }$ until testing was performed.

Testing with the Pro hMPV $^ +$ Assay occurred within two weeks of extraction. The 10 extractions of each of the three concentrations from both extraction instruments (total of 60 samples) for a specific virus were tested using the Pro hMPV $^ +$ Assay (a single Pro $\mathrm { { h M P V + } }$ mastermix). Included with each run was the hMPV RNA Positive Control and the relevant NPICs from each automated extraction run for the specific samples being tested. A total of 10 data points were generated at each dilution per virus per extraction instrument. Results were analyzed and reported as specified in the Pro hMPV+ Instructions for Use. This study was limited to one operator, one extraction instrument (one easyMAG and one MagNA Pure LC), one lot of instrument specific extraction reagents per instrument (each automated extractor requires different extraction reagents), one lot of Pro hMPV+ reagents, and a single dilution series per virus.

LoDs for each virus were determined by $2 9 5 \%$ positivity at a specific concentration and are presented in the following table:

Extraction Equivalence LoD Study – Average $\mathbf { C _ { T } }$ Values for both MagNA Pure LC (MP) and easyMAG (EM)   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>ConcentrationTCID50/mL</td><td rowspan=1 colspan=2>Number of Positive Samples</td><td rowspan=1 colspan=2>Average CT</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>EM</td></tr><tr><td rowspan=3 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>1 x 103</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>1 x 102</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>30.0</td></tr><tr><td rowspan=1 colspan=1>1 x 101</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>32.1</td></tr><tr><td rowspan=3 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>1 x 102</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>25.2</td></tr><tr><td rowspan=1 colspan=1>1 x 10&#x27;</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>29.4</td></tr><tr><td rowspan=1 colspan=1>1 x 100</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>36.6*</td><td rowspan=1 colspan=1>33.7*</td></tr></table>

\* Average based on number of positives samples $( < 1 0 )$ .

A LoD of $1 0 ^ { 2 } \mathrm { \ T C I D } _ { 5 0 } / \mathrm { m L }$ for hMPV A2 was not achieved as expected (at $1 0 ^ { 1 }$ $\mathrm { T C I D 5 0 / m L \ 1 0 0 \% }$ of all replicates were still detected). However, average $\mathrm { C t }$ values were comparable at all concentration tested between both extraction systems and $100 \%$ of replicates at all concentrations were detected with both extraction systems.

This study demonstrated that the Roche MagNA Pure LC and the bioMérieux NucliSens easyMAG automated extraction instruments perform equivalently based on Analytical Sensitivity. Clinical trial and reproducibility studies also support the equivalence of both automated extraction systems.

# e. Analytical Reactivity:

The analytical reactivity of the Pro hMPV $^ +$ Assay was determined for the following additional hMPV viral strains:

<table><tr><td rowspan=1 colspan=1>Strains</td></tr><tr><td rowspan=1 colspan=1>human Metapneumovirus A1</td></tr><tr><td rowspan=1 colspan=1>human Metapneumovirus A2</td></tr><tr><td rowspan=1 colspan=1>human Metapneumovirus B1</td></tr><tr><td rowspan=1 colspan=1>human Metapneumovirus B2</td></tr></table>

The cultured and titered strains used to evaluate the reactivity of the Pro hMPV+ Assay were tested at a concentration of $1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All hMPV strains were obtained from ViroNovative (Rotterdam, Netherlands) and cultured and titered by the City of Milwaukee Health Department Laboratory. The Analytical Reactivity Panel was prepared by spiking the above listed viruses into aliquots from negative nasopharyngeal (NP) swab pools. An Internal RNA Control (IC) was spiked into each sample prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. A negative NP swab sample spiked with the Internal Control and an M4 Viral Transport Media-only sample spiked with the Internal Control were included in the Reactivity Panel and were extracted along with the Reactivity Panel samples. The negative controls served to monitor for contamination during the testing procedure. Samples were extracted on the Roche MagNA Pure LC and tested in triplicate PCR reactions using one lot of Pro hMPV+ reagents. The hMPV Positive RNA Control, non-infectious in vitro transcribed RNA of a specific viral sequence, was included with each run to test for procedural errors (absence of reagent, instrument failure, etc.).

The data are presented in the following table:

Analytical Reactivity Data   

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageC (n = 3)</td><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>Min CT</td><td rowspan=1 colspan=1>Max CT</td></tr><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>31.0</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>10|</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>31.2</td></tr><tr><td rowspan=1 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>31.1</td></tr></table>

The analytical specificity of the Pro $\mathrm { { h M P V + } }$ Assay was determined with 52 viral, bacterial and yeast (28 viruses, 23 bacteria and 1 yeast) strains of common respiratory pathogens or flora commonly present in the nasopharynx. All the viral strains were obtained from ATCC or University of Iowa and cultured and titered by Tricore Reference Laboratories (Albuquerque, NM). The majority of cultured and titered bacterial and yeast strains were obtained from MicroBioLogics, Inc. (St. Cloud, MN) or from ATCC and cultured and titered by the Clinical Microbiology Laboratories at Resurrection Medical Center (Chicago, IL) and Waukesha Memorial Hospital (Waukesha, WI). None of the facilities listed above were able to provide fresh cultured and titered stocks of $C .$ . pneumoniae or $C .$ . trachomatis. Additional laboratories (commercial and private) were contacted, but none were able to provide titered cultures at a reasonable cost and timeframe. Thus, ATCC frozen cultures were used and ATCC supplied titers were used for $C .$ . pneumoniae and $C .$ . trachomatis.

Aliquots of a negative nasopharyngeal (NP) swab pool matrix were spiked with the analytical specificity organisms at concentrations of $1 0 ^ { 2 } - 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } \mathrm { / m l }$ for viruses and $\mathrm { \dot { 1 } 0 ^ { 6 } - 7 . 4 { x } 1 0 ^ { 7 } \dot { C F } U / m l }$ for the bacterial and yeast strains. The Pro $\mathrm { { h M P V + } }$ Internal Control (IC) was spiked into all samples prior to nucleic acid isolation. A Negative Control, which consisted of the M4 viral transport media spiked with the IC, was included and nucleic acid isolation of the negative control was performed along with Analytical Specificity Panel samples. The Pro $\mathrm { { h M P V + } }$ Positive Control and was included with each detection run to test for global failure. The Analytical Specificity Panel samples and the Negative Control were extracted on the Roche MagNA Pure LC and tested in triplicate on a Cepheid SmartCycler II using one lot of Pro hMPV $^ +$ reagents.

The Pro hMPV+ Assay did not cross-react with any of the Analytical Specificity Panel samples tested. Detailed Analytical Specificity results are presented in the following table:

Analytical Specificity Results   

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">hMPVdetection</td></tr><tr><td colspan="1" rowspan="1">hMPV A2 (Iowa 14)</td><td colspan="1" rowspan="1">103TCID50/ml</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">hMPV B2 (Iowa 8)</td><td colspan="1" rowspan="1">102 TCID50/ml</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid 71</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B5/10/2006</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 39</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Strains</td><td colspan="2" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">hMPVdetection</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae*</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis*</td><td colspan="2" rowspan="1">104TCID50/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="2" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="2" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="2" rowspan="1">7.4x107 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="2" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="2" rowspan="1">3x107 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="2" rowspan="1">2x106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td></tr></table>

\* For C. pneumoniae and $C .$ . trachomatis ATCC frozen cultures and ATCC supplied titers were used due to the fact that other commercial or private entities were not able to provide titered cultures.

# g. Assay cut-off:

The “cutoff value” represents the fluorescent intensity signal (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

Cutoff Determination and Confirmation studies were conducted with two operators and two reagent lots. The reagent lots varied in raw material lots, personnel, and facility in which they were manufactured. Cutoff values (RFUs) were determined upon completion of the Cutoff Determination Study which included a training set of simulated samples. The training set of 66 samples contained two levels of each virus (hMPV subtype A1, A2, B1 and B2), a negative nasopharyngeal (NP) swab sample with and without Internal Control (NIC), a negative viral transport media control, and the hMPV Positive Control. The cutoff values were then verified in the Cutoff Confirmation Study against a set of clinical samples and controls ( $2 0 \ \mathrm { h M P V }$ positive samples and 24 negative samples; 16 NTC and 4 PC). The determined cutoff was confirmed if $2 9 0 \%$ sensitivity and specificity was attained for each channel with the above mentioned set of clinical samples and controls.

The cutoffs used in the Cutoff Determination and Confirmation Studies are presented in the following table:

Cutoffs used in Cut-off Determination and Confirmation Studies   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Preliminary Cut-offused in Cut-offDetermination Studya</td><td rowspan=1 colspan=1>Cut-off used in Cut-off ConfirmationStudy</td></tr><tr><td rowspan=2 colspan=1>EndPtThreshold</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>22</td></tr></table>

a Identified during product development

For both the FAM and Cy5 channels, the final threshold is set at the higher end of the cutoff range determined above and are shown in the following table:

<table><tr><td rowspan=1 colspan=13>Cutoff Determination - Data Analysis Summary</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=3>Range for CutoffThreshold</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Reactions</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>AveRFU</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>Distribution</td><td rowspan=1 colspan=1>ThresholdBased onNormalDistribution</td><td rowspan=1 colspan=1>Thres-holdBased onPercen-tiles</td><td rowspan=1 colspan=1>ThresholdBased onROC</td><td rowspan=1 colspan=2>Lowerlimitbasedon Neg</td><td rowspan=1 colspan=1>FinalThres-hold</td><td rowspan=1 colspan=1>Upper limitbasedon Pos</td></tr><tr><td rowspan=2 colspan=1>hMPV(FAM)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>535.41</td><td rowspan=1 colspan=1>380.15</td><td rowspan=1 colspan=1>Not normal</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>37</td><td rowspan=2 colspan=2>11</td><td rowspan=2 colspan=1>30</td><td rowspan=2 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>Normal</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=2 colspan=1>InternalRNAControl(Cy5)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Symmetric</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>46</td><td rowspan=2 colspan=2>5</td><td rowspan=2 colspan=1>22</td><td rowspan=2 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>Not normal</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr></table>

The hazard of an hMPV false positive is greater than a false negative so a higher threshold carries less risk for a patient. A false negative would probably not alter the clinical treatment of a patient presenting with upper respiratory symptoms, but a false positive might result in improper treatment and/or preclude the detection of a more serious pathogen.

Analysis settings and threshold settings were determined for FAM (30) and Cy5 (22) channels representing hMPV and IC detection, respectively. These settings produced $100 \%$ sensitivity and specificity in the verification set of samples used during Cutoff Confirmation and met acceptance criteria for the study. Therefore, the cutoff values are confirmed as the acceptable for use during the clinical trials.

# h. General Assay Analysis Settings:

<table><tr><td rowspan=1 colspan=3>Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=3>1.7b, Dx Software</td></tr><tr><td rowspan=1 colspan=1>Channel:</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>CY 5</td></tr><tr><td rowspan=1 colspan=1>Dye Name:</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>Q670</td></tr><tr><td rowspan=1 colspan=1>Target:</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Internal Control</td></tr><tr><td rowspan=1 colspan=1>Usage:</td><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Internal Control</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis:</td><td rowspan=1 colspan=1>Primary Curve</td><td rowspan=1 colspan=1>Primary Curve</td></tr><tr><td rowspan=1 colspan=1>Thresh Setting:</td><td rowspan=1 colspan=1>Manual</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>Manual Thresh:</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Auto Thresh:</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Auto Min Cycle:</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Auto Max Cycle:</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Valid Min Cycle:</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Valid Max Cycle:</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Backgrd Subtract:</td><td rowspan=1 colspan=1>On</td><td rowspan=1 colspan=1>On</td></tr><tr><td rowspan=1 colspan=1>Bkgnd Min Cycle</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Bkgnd Max Cycle</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Boxcar Avg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>EndPt Thresh</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>NC IC %</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>IC Delta</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

# i. Interfering Substances:

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors affected the performance of the Pro $\mathrm { { h M P V + } }$ assay. Blood, mucin, or medications (prescription and over-thecounter) for relief of congestion, sore throat, allergy and asthma symptoms were spiked into simulated hMPV positive NP samples. Two subtypes of hMPV (A2 and B2) were used and spiked into NP pools at either 2X Limit of Detection (LoD) or 10X LoD. Clinically relevant amounts of the potential inhibiting substances were added to spiked samples. An Internal Control (IC) was added to each sample. Nucleic acid from the samples was extracted with the Roche MagNA Pure LC. Pro hMPV+ was performed in triplicate reactions for each sample on the Cepheid SmartCycler II. Any substance in which detection of hMPV at 10X LoD was not observed was considered an inhibitor of PCR. A Negative Control, which consisted of M4 viral transport media spiked with an IC, was included with Interfering Substances Panel samples. The Pro hMPV $+$ Positive Control and was included with each detection run to test for global errors.

The following table shows the interfering substances used for this study. As described in the justification column, the concentrations spiked directly into samples ranged from approximately $2 \%$ to $1 5 \%$ of recommended dose for some substances, while others were tested at concentrations reported in scientific literature or in references from other IVD package inserts. The substances consisted of nasal sprays (liquid and powder), ingestible pills and lozenges, injectables, and endogenous substances (blood and mucin).

Interfering Substances Test Concentrations   

<table><tr><td rowspan=1 colspan=1>Substance Name</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td><td rowspan=1 colspan=1>Justification</td></tr><tr><td rowspan=1 colspan=1>Mucin (BovineSubmaxillarygland,type I-S)</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>60μg/ml</td><td rowspan=1 colspan=1>1000x maximum levelpresent in serum*</td></tr><tr><td rowspan=1 colspan=1>Blood (human), heparinanticoagulent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2% (volume/volume)</td><td rowspan=1 colspan=1>Other Respiratory IVD&#x27;sPackage Insert*.</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine®</td><td rowspan=1 colspan=1>Phenylephrine HCl</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of total recommendeddose (45μ1)*</td></tr><tr><td rowspan=1 colspan=1>Walgreens OriginalAnefrin Nasal Spray</td><td rowspan=1 colspan=1>OxymetazolineHydrochloride</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of total recommendeddose (45μ1)*</td></tr><tr><td rowspan=1 colspan=1>Zicam Homepathic Non-Drowsy Allergy Relief NoDrip Liquid Nasal gel</td><td rowspan=1 colspan=1>Luffa Operculata,Galphimia Glauca,HistaminumHydrochloricum,</td><td rowspan=1 colspan=1>5% (volume/volume)</td><td rowspan=1 colspan=1>10% of total recommendeddose (15μl)*</td></tr><tr><td rowspan=1 colspan=1>Walgreens Saline NasalSpray</td><td rowspan=1 colspan=1>Sodium chloride withpreservatives</td><td rowspan=1 colspan=1>15% (volume/volume) ofdose</td><td rowspan=1 colspan=1>10% of total recommendeddose (45μl)*</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic® Sore ThroatLozenges, Cherry</td><td rowspan=1 colspan=1>Oral anesthetic/analgesic</td><td rowspan=1 colspan=1>0.63mg/ml; 1/20 drop,crushed; activeingredients: 1.0mg/mlbenzocaine, 1.7mg/mlmenthol</td><td rowspan=1 colspan=1>5% of total dose*</td></tr><tr><td rowspan=1 colspan=1>Relenza®</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>3.3mg/ml;</td><td rowspan=1 colspan=1>10% of total spray dose*</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4.0μg/ml</td><td rowspan=1 colspan=1>10% of total recommendeddose*</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>6.6mg/ml</td><td rowspan=1 colspan=1>10% of total recommendeddose in Mupirocinointment*</td></tr><tr><td rowspan=1 colspan=1>Rebitol</td><td rowspan=1 colspan=1>Ribavirin</td><td rowspan=1 colspan=1>20mg/ml</td><td rowspan=1 colspan=1>10% of total recommendeddose*</td></tr><tr><td rowspan=1 colspan=1>TamiFlu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>25mg/ml</td><td rowspan=1 colspan=1>10% of total recommendeddose*</td></tr><tr><td rowspan=1 colspan=1>Beconase AQ®</td><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>5% volume/volume =14μg/ml</td><td rowspan=1 colspan=1>10% of total recommendeddose*</td></tr></table>

$^ { * } =$ Concentration tested is $10 \%$ of suggested dose into a $0 . 3 \mathrm { m l }$ sample, or otherwise stated in table above.

The Pro $\mathrm { { h M P V + } }$ Assay did not cross-react with any of the exogenous or endogenous interfering substances tested. Both concentrations (2X LoD and 10X LoD) of both strains of $\mathrm { { h M P V + } }$ (A2 and B2) were detected in triplicate reactions for all potential interfering substances. Interfering Substances Study results are presented in the following table:

Average hMPV $\mathbf { C _ { t } }$ according to lineage and concentration   

<table><tr><td rowspan=3 colspan=1>InterferingSubstance</td><td rowspan=1 colspan=4>A2 hMPV Ct value</td><td rowspan=1 colspan=4>B2 hMPV Ct value</td></tr><tr><td rowspan=1 colspan=2>10X LoD</td><td rowspan=1 colspan=2>2X LoD</td><td rowspan=1 colspan=2>10X LoD</td><td rowspan=1 colspan=2>2X LoD</td></tr><tr><td rowspan=1 colspan=1>Average Ct</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>AverageCt</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>AverageCt</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>AverageCt</td><td rowspan=1 colspan=1>Std Dev</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>Oxymetazoline HC1*</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>Luffa Operculata,Galphimia Glauca,HistaminumHydrochloricum*</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>26.7</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Sodium Chloride</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Benzocaine, menthol</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0.29</td></tr><tr><td rowspan=1 colspan=1>Tobromycin</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=1 colspan=1>Ribavrin</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>0.38</td></tr></table>

# $j$ . Internal Control Interference:

Competitive inhibition of the Pro $\mathrm { { h M P V + } }$ Assay due to the presence of the IC was assessed. Simulated samples were tested with and without IC to determine if the presence of the IC inhibited the reaction. Two sets of IC Interference Panel members were prepared by spiking serial dilutions of cultured and titered $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ strains of human Metapneumovirus (hMPV) subtype A2 and B2 into individual aliquots of negative nasopharyngeal (NP) swab pools at concentrations of 1 log below and at the Limit of Detection (LoD) (as determined in the Analytical Sensitivity Study). The IC was added to one set of IC Interference Panel samples prior to nucleic acid isolation; the second set did not contain the IC. Two negative controls were generated which consisted of negative NP swab matrix with or without the Internal Control. Nucleic acid isolation of the negative controls was performed along with the IC Interference Panel samples. The negative control served to monitor for contamination during the testing procedure or in the case of negative control spiked with IC, also served to monitor for the presence of inhibitors in the sample matrix. The hMPV Positive RNA Control was included with each RT-PCR run to test for global errors (absence of reagent, instrument failure, etc.). The Positive Control does not require nucleic acid isolation and was diluted just prior to set up of RT-PCR reactions. Nucleic acid from each sample was isolated in duplicate extractions on the Roche MagNA Pure LC. The purified nucleic acids from each duplicate extraction were combined $[ 1 0 0 \mu \mathrm { L }$ total volume) and tested with the Pro hMPV $^ +$ Assay in 10 replicate reactions using 1 lot of

Pro $\mathrm { { h M P V + } }$ reagents on the Cepheid Smartcycler II (Dx Software Version 1.7b). This resulted in a total of 10 data points for each viral strain at each concentration spiked with IC and 10 data points for each viral strain at each concentration without the IC.

Competitive inhibition of the IC was assessed by comparing the percent of samples detected at concentrations of 1 log below and at the LoD of each viral strain when spiked with or without the IC. Competitive inhibition was to be considered if the presence of the IC decreased the sensitivity (LoD) of the Pro hMPV $^ +$ Assay.

No competitive inhibition at the assay LoD was observed for detection of hMPV in the presence IC using the Pro hMP $\mathbf { V } +$ Assay.

# $k .$ . Carry-Over Contamination:

To evaluate the degree of carry-over/cross-contamination that occurs with the use of the Pro hMPV $^ +$ Assay in association with nucleic acid extraction on the Roche MagNA Pure LC and the bioMérieux NucliSens easyMAG instruments and PCR on the Cepheid SmartCycler II thermocycler, an internal Carry-Over study was carried out by testing simulated human Metapneumovirus (hMPV) high positive samples run in series alternating with hMPV high negative samples (i.e. “checkerboard” pattern). The high hMPV positive samples in this study represented the lower Cycle Threshold $\mathrm { ( C t ) }$ range (higher sample titer range) obtained in the Pro $\mathrm { { h M P V + } }$ clinical trials (lowest $\mathrm { C t } = 1 7 . 9$ ). To achieve this, the Extraction Control, an inactivated noninfectious A2 strain of hMPV, was spiked into negative nasopharyngeal (NP) swab matrix at approximately 6 logs above the organism’s limit of detection (LoD) along with an Internal Control (IC). The high negative samples contained the analyte at a concentration below the detection limit such that repeat testing of the sample would be negative approximately $9 5 \%$ of the time. High negative samples were generated by spiking hMPV into negative NP swab matrix at approximately 2 logs below the organism’s LoD. A negative control (viral transport medium) spiked with the IC was also included with each extraction run.

A total of 22 high positive and 22 high negative samples and 2 Negative Controls were generated to be run on the Roche MagNA Pure LC. A total of 11 high positive and 11 high negative samples and a Negative Controls were generated to be run on the bioMérieux NucliSens easyMAG. Total nucleic acid was extracted in a pattern of alternating high-positive and high-negative samples on each extraction instrument (Roche MagNA Pure LC and bioMérieux NucliSens easyMAG). Purified nucleic acid from each sample as well as the $\mathrm { { h M P V + } }$ Positive RNA Control were tested in single reactions using 1 lot of Pro hMPV $^ +$ reagents.

One high-negative sample showed potential hMPV contamination when extracted using the MagNA Pure extraction system. Contamination could have occurred during creation of the sample, during sample preparation for extraction, during extraction or when transferring the purified nucleic acid samples from the sample cartridge to microfuge tubes. (Note: since the high-negative samples include low amount of

hMPV that is detectable less than $5 \%$ of the time, the possibility of this sample actually being a sample that falls in the $5 \%$ category can not be ruled out). Correlation to expected results for the MagNA Pure and easyMAG across all samples was $9 5 . 7 \%$ (44/46) and $100 \%$ (23/23) respectively.

This Pro hMPV $^ +$ Carryover Study demonstrated that low level carry-over/cross contamination with the Pro hMPV+ Assay using either the Roche MagNA Pure LC or the bioMériuex NucliSens easyMAG automated nucleic acid extraction instruments and the Cepheid Smartcycler II could happen. Therefore, the following statements were included in the Limitations section to address the potential of crosscontamination when performing the assay: “The detection of viral nucleic acid is dependent upon proper specimen collection, handling, transportation, storage, and preparation (including extraction). Failure to observe proper procedures in any one of these steps can lead to incorrect results.” “There is a risk of false positive values resulting from cross-contamination by target organism or its nucleic acids.”

# l. Comparator Assays Analytical Validation Studies

The hMPV Comparator Assays are 2-Step RT-PCR assays which target conserved regions of the hMPV Nucleocapsid or Fusion gene. To validate the analytical performance of the hMPV Nucleocapsid (N) Reference Assay and the hMPV Fusion (F) Reference Assay used as the comparison methods to the Pro hMPV+ Assay for the Pro hMPV $+$ Clinical Study, the following analytical studies were independently performed: Analytical Sensitivity, Reactivity and Analytical Specificity.

# Analytical Sensitivity

Analytical Sensitivity was performed with viruses obtained from University of Iowa and were subsequently cultured and titered by ZeptoMetrix Corporation and TriCore Laboratories. Analytical Sensitivity was determined using two (A2 and B2) of the four (A1, A2, B1 and B2) sub-lineages of hMPV. Each virus was serially diluted in negative NP swab matrix and tested minimally at three concentrations: 1 log above, 1 log below and at the expected LoD (determined by Pro $\mathrm { { h M P V + } }$ Assay Analytical Sensitivity Study). Internal RNA Control (IC) was also spiked into samples. A single operator generated 2 dilution series for each lineage and each dilution series was extracted along with a corresponding negative $\mathrm { N P + }$ IC sample using the Roche MagNA Pure LC System. RT-PCR was performed in triplicate for each dilution series with both hMPV N and F Reference Assays. A total of 6 data points per virus concentration for both the hMPV N and F Reference Assays were generated. PCR products were analyzed using agarose gel electrophoresis and were determined hMPV positive if the band size was within an established range of the theoretical PCR target size for each assay and had an band intensity maximum (Imax) three standard deviations above the background (yielding a cutoff value of 29).

The LoD for hMPV A2 was $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ and the LoD for B2 was $1 0 ^ { 1 } ~ \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ respectively with both reference assays. Detailed analytical sensitivity data for the reference assays are presented in the following table:

<table><tr><td rowspan=1 colspan=6>A2 and B2 Analytical Sensitivity Results for each Reference Assay</td></tr><tr><td rowspan=1 colspan=1>Samples (Pooledby series)</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Fusion</td><td rowspan=1 colspan=1>Nucleocapsid</td><td rowspan=1 colspan=1>Fusion - Pooledtotal</td><td rowspan=1 colspan=1>NC - Pooledtotal</td></tr><tr><td rowspan=1 colspan=1>A2 102 Series 1</td><td rowspan=4 colspan=1>102 TCID50/ml</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=2 colspan=1>(6/6) 100%</td><td rowspan=2 colspan=1>(6/6) 100%</td></tr><tr><td rowspan=1 colspan=1>A2 10² Series 2</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td></tr><tr><td rowspan=1 colspan=1>B2 10 Series 1</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=2 colspan=1>(6/6) 100%</td><td rowspan=2 colspan=1>(6/6) 100%</td></tr><tr><td rowspan=1 colspan=1>B2 10 Series 2</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td></tr><tr><td rowspan=1 colspan=1>A2 10Series 1</td><td rowspan=4 colspan=1>101 TCID50/ml</td><td rowspan=1 colspan=1>2/3 - 67%</td><td rowspan=1 colspan=1>0/3 - 0%</td><td rowspan=2 colspan=1>(5/6) 83%</td><td rowspan=2 colspan=1>(1/6) 17%</td></tr><tr><td rowspan=1 colspan=1>A2 10 Series 2</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>1/3 - 33%</td></tr><tr><td rowspan=1 colspan=1>B2 10&#x27; Series 1</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=2 colspan=1>(6/6) 100%</td><td rowspan=2 colspan=1>(6/6) 100%</td></tr><tr><td rowspan=1 colspan=1>B2 10&#x27; Series 2</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td></tr><tr><td rowspan=1 colspan=1>A2 10P Series 1</td><td rowspan=4 colspan=1>100 TCID50/ml</td><td rowspan=1 colspan=1>0/3 - 0%</td><td rowspan=1 colspan=1>0/3 - 0%</td><td rowspan=2 colspan=1>(0/6) 0%</td><td rowspan=2 colspan=1>(0/6) 0%</td></tr><tr><td rowspan=1 colspan=1>A2 10° Series 2</td><td rowspan=1 colspan=1>0/3 - 0%</td><td rowspan=1 colspan=1>0/3 - 0%</td></tr><tr><td rowspan=1 colspan=1>B2 10RSeries 1</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=1 colspan=1>3/3 -100%</td><td rowspan=2 colspan=1>(5/6) 83%</td><td rowspan=2 colspan=1>(5/6) 83%</td></tr><tr><td rowspan=1 colspan=1>B2 10 Series 2</td><td rowspan=1 colspan=1>2/3 -100%</td><td rowspan=1 colspan=1>1/3 -100%</td></tr></table>

# Analytical reactivity

The analytical reactivity of the Reference Assays was evaluated with subtypes $\mathbf { A } _ { 1 }$ , $\mathbf { A } _ { 2 }$ , $\mathrm { B } _ { 1 }$ , and ${ \bf B } _ { 2 }$ clinical isolates of human Metapneumovirus that were cultured and titered by an independent laboratory and were not the same strains used in the Analytical Sensitivity studies. hMPV sub-lineages were tested at approximately $1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ . This study utilized the leftover extracted nucleic acid from the Pro hMPV $^ +$ Assay Reactivity Study (described in Section 5.1.2 of this submission), which was stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ from extraction until its use in this study. RT-PCR was performed in duplicate with the $_ \mathrm { N }$ and F Reference Assays for each sample and PCR products generated were analyzed using agarose gel electrophoresis. PCR products were determined hMPV positive if band size was within an established range of the theoretical PCR target size for each assay and were above the established cutoff intensity.

Each of the four hMPV strains was detected with both Reference Assays as shown in the following table:

Reference Assay Reactivity Summary   

<table><tr><td rowspan=1 colspan=1>Strain and Concentration</td><td rowspan=1 colspan=1>Nucleocapsid Ref Assay</td><td rowspan=1 colspan=1>Fusion Ref Assay</td></tr><tr><td rowspan=1 colspan=1>hMPV A1 101 TCID50/mL</td><td rowspan=1 colspan=1>+ (2/2 detected)</td><td rowspan=1 colspan=1>+ (2/2 detected)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2 10 TCID50/mL</td><td rowspan=1 colspan=1>+ (2/2 detected)</td><td rowspan=1 colspan=1>+ (2/2 detected)</td></tr><tr><td rowspan=1 colspan=1>hMPV B1 101 TCID50/mL</td><td rowspan=1 colspan=1>+ (2/2 detected)</td><td rowspan=1 colspan=1>+ (2/2 detected)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2 101 TCID50/mL</td><td rowspan=1 colspan=1>+ (2/2 detected)</td><td rowspan=1 colspan=1>+ (2/2 detected)</td></tr></table>

# Analytical Specificity

Analytical Specificity is defined as the ability of the assay to exclusively identify a target organism with no cross-reactivity with organisms that are closely related or cause similar disease syndromes as the targeted organism (hMPV) or with organisms present as normal flora in nasopharyngeal swab specimens. Analytical Specificity

testing for both N and F Reference Assays was performed using the panel of nucleic acid that was prepared for the Pro hMPV+ Assay Analytical Specificity Study. The Analytical Specificity panel included nucleic acids extracted from 28 viruses, 25 bacteria and one yeast that were cultured and titered by other reference laboratories. These nucleic acids were stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ since the Pro $\mathrm { { h M P V + } }$ Analytical Specificity testing. RT-PCR was performed with the $_ \mathrm { N }$ and F Reference Assays and PCR products generated were analyzed using agarose gel electrophoresis. PCR products were determined hMPV positive if band size was within an established range of the theoretical PCR target size for each assay and was above the established cutoff intensity.

Both Reference Assays demonstrated $100 \%$ specificity and data is presented in the follow table:

Reference Assay Analytical Specificity Summary   

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="2" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">N Assay</td><td colspan="1" rowspan="1">F Assay</td></tr><tr><td colspan="1" rowspan="1">hMPV A2 (Iowa 14)</td><td colspan="2" rowspan="1">103TCID50/ml</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">hMPV B2 (Iowa 8)</td><td colspan="2" rowspan="1">102 TCID50/ml</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid 71</td><td colspan="2" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="2" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B5/10/2006</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="2" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="2" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="2" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="2" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 39</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="2" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="2" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="2" rowspan="1">104TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="2" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="2" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="2" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">N Assay</td><td colspan="1" rowspan="1">F Assay</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">7.4x10' CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">107CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">3x107CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">2x10CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">106CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

Based on thorough analytical validations, the hMPV $\mathbf { N }$ and F Reference Assays are acceptable methods to be used as composite reference methods in determining “clinical diagnostic truth” for the Pro $\mathbf { h } \mathbf { M } \mathbf { P } \mathbf { V } +$ Clinical Trial. Both the N and F Reference Assays performed equivalently to the Pro hMP $\mathbf { V } +$ Assay for Analytical Sensitivity, Reactivity, and Analytical Specificity.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix Description and Comparison:

Analytical performance of the Pro hMPV+ Assay using three different viral transport media (VTM) (2 that were used for sample collection in the Pro hMPV+ clinical trials and one that was used for ProFlu+ clinical trials) was evaluated. Analytical sensitivity was determined using cultured and titered strains of human Metapneumovirus (hMPV) subtypes A2 (Iowa 14 strain) and B2 (Iowa 8 strain) spiked into each of the different VTM. The VTM were considered “equivalent” if the same Limits of Detection (LoD) were achieved for each.

The intended use of VTM is for the transport of clinical specimens containing viruses, chlamydiae, mycoplasma, and ureaplasma from the collection site to the laboratory for microbiological procedures. The three VTM evaluated in this study: M4 (Remel, Lenexa, KS), Copan Mini UTM (Copan, Murrieta, CA) and BD UTM (Becton Dickinson, Sparks, MD) are identical in composition with the exception that Copan UTM and BD UTM include L- Cysteine whereas M4 does not.

Cultured and titered human Metapneumovirus (hMPV) strains (subtypes A2 and B2) were diluted at one log above, at, and at one log below their previously determined limit of detection into each of three different VTM: M4, Copan UTM, and BD UTM to generate simulated positive samples $\mathrm { \Delta } \mathrm { n } = 8$ data points per virus per concentration). Samples were extracted using the Roche MagNA Pure LC and tested in triplicate PCR reactions on the Cepheid Smartcycler II using one lot of Pro hMPV $^ +$ reagents. An Internal Control (IC) was spiked into all VTM study samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Negative controls, consisting of each of the three different VTM spiked with IC were tested in duplicate in each run. Nucleic acid isolation of the negative controls was performed along with the VTM study samples. The negative controls serve to monitor for contamination or the presence of inhibitors. The hMPV RNA Positive Control, non-infectious in vitro transcribed RNA, was included with each run to test for global errors (absence of reagents, instrument failure, etc.). The Positive Control did not require nucleic acid isolation and was used just prior to set up of RT-PCR reactions.

Each of the three VTM performed “equivalently” at the Limit of Detection (LoD) as determined in a separate Analytical Sensitivity Study and average Ct values at each concentration tested were comparable among all three VTM. Detailed data are presented in the following table:

# VTM Study Data

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>CT ± Std Dev</td><td rowspan=1 colspan=1>% detection</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>27.4 ± 0.29</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>27.4 ± 0.43</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>27.3 ± 0.32</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>31.7 ± 0.33</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>102$</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>32.0 ± 0.40</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>10^2</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>31.6 ± 0.38</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>37.3 ± 0.70</td><td rowspan=1 colspan=1>75 (6/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>37.5 ± 2.0</td><td rowspan=1 colspan=1>88 (7/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>37.8 ± 0.75</td><td rowspan=1 colspan=1>62 (5/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>27.1 ± 0.20</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>27.0 ± 0.29</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>27.2 ± 0.21</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>31.6 ± 0.41</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>31.1 ± 0.29</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>31.6 ± 0.51</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>Remel</td><td rowspan=1 colspan=1>37.2 ± 1.21</td><td rowspan=1 colspan=1>100 (8/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>BD</td><td rowspan=1 colspan=1>36.2 ± 1.81</td><td rowspan=1 colspan=1>88 (7/8)</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>36.9 ± 0.77</td><td rowspan=1 colspan=1>100 (8/8)</td></tr></table>

Shaded cells represent the limit of detection (LoD) as determined during Analytical Sensitivity Study.

# All three VTM evaluated are compatible with the Pro hMPV $^ +$ Assay. The three VTM met the LoD that was previously established with Remel M4 in the Analytical Sensitivity Study.

3. Clinical studies:

# a. Prospective Clinical studies

Performance characteristics of the Pro $\mathrm { { h M P V + } }$ Assay were established during prospective studies at 4 U.S. clinical laboratories during the 2008 respiratory virus season (January - March). All specimens used in the study meeting the inclusion and exclusion criteria represented excess, remnants of nasopharyngeal (NP) swab specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study. Inclusion criteria included, but were not limited to: the specimen was from a symptomatic patient (for respiratory infection); the specimen was a NP swab in appropriate transport media (i.e. Remel M4 or Copan UTM viral transport medium); the specimen contained adequate volume for performing the Pro hMPV+ Assay; age and gender information were available; specimen was stored properly (refrigerated at $2 ^ { \circ } \mathrm { C } -$ $8 ^ { \circ } \mathrm { C }$ ); and initiation of the Pro hMPV $^ +$ assay (nucleic acid extraction) took place within 72 hours of sample collection.

Performance of the Pro hMPV $^ +$ Assay was assessed and compared to a predetermined algorithm that used composite reference methods. The composite reference methods consisted of two independent molecular (RT-PCR) tests for two separate gene targets of hMPV followed by bi-directional genetic sequencing. The two reference tests targeted the Nucleocapsid gene (different region of the gene than targeted by the Pro hMPV $^ +$ assay) and the Fusion gene. “True” hMPV RNA positives were considered as any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria, for one or both gene targets that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E-values. “True” hMPV RNA negatives were considered as any sample that was tested negative by both of the comparator methods. The E-values generated from the clinical trial range from a low of 1e-52 to a high of 1e-23. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone. (http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614). Therefore an E-Value ranging from 1e-23 to 1e-52 has a very low probability of occurring purely by chance.

A total of 1275 eligible NP swab samples (617 from female patients, 654 from male patients, and for 4 samples gender was not specified) were tested with the Pro hMPV $^ +$ Assay at the 4 clinical sites and by the composite reference methods at Prodesse. The NP swab specimens underwent nucleic acid extraction using either the Roche MagNA Pure Total Nucleic Acid Isolation Kit on the MagNA Pure LC Instrument (Site 4) or using the bioMérieux NucliSENS easyMAG instrument with Automated Magnetic Extraction Reagents (Sites 1, 2, 3). An Internal RNA Control (IC) was added to every NP swab sample. A Negative Control which consisted of viral transport medium and an Extraction Control, non-infectious hMPV strain (A2 lineage), were included with each extraction run. Nucleic acid detection was performed using the Pro hMPV $+$ Assay on the Cepheid SmartCycler II. Remaining nucleic acids were frozen at $\mathrm { - 7 0 ^ { \circ } C }$ , sent back to Prodesse and tested with the two independent reference RT-PCR assays. Samples that yielded a positive result by gel electrophoresis were sent for bi-directional sequencing.

The overall assay failure rate of the Pro hMPV+ Assay during the clinical trials due to failure of positive, negative or extraction controls was $0 \%$ (total of 65 runs across all 4 sites). On two occasions slight contamination of the Internal Control appeared in the Positive Control $\mathrm { C t } = 4 2 . 0$ (Site 1) and $\mathrm { C t } = 4 0 . 6$ (Site 3) and on two occasions in the Extraction Control $\mathrm { C t } = 4 2 . 7$ (Site 1) and $\mathrm { C t } = 3 9 . 8$ (Site 2)). As this does not affect the sample results, the runs were accepted as valid.

A total of $^ { 1 , 3 3 8 \mathrm { N P } }$ swab specimens were initially included for the clinical trial. 39 specimens (18 from Site 1, 11 from Site 2, none from Site 3, and 10 from Site 4) were excluded from the clinical study due to $> 7 2$ hours from time of collection to Pro hMPV+ Assay testing. 1 specimen (Site 4) was excluded due to failed NA extraction. Of the Pro $\mathrm { { h M P V + } }$ Assay run on all eligible specimens, $9 8 . 1 \%$ (1273/1298) of these specimens were successful on the first attempt (Site $1 ; 4 2 4 / 4 4 3 = 9 5 . 7 \%$ ; Site 2: $2 7 9 / 2 8 4 = 9 8 . 2 \%$ ; Site 3: $3 5 6 / 3 5 6 = 1 0 \%$ ; Site 4: $2 1 4 / 2 1 5 = 9 9 . 5 \%$ . The remaining 25 gave “Unresolved” results on the first attempt (19 from Site 1, 5 from Site 2, none from Site 3, and 1 from Site 4). Unresolved results occur when the sample is negative for both hMPV and the Internal Control, indicating potentially PCR-inhibiting samples. Of the 25 “Unresolved” specimens on the first attempt with sufficient sample for retest, only $8 . 0 \%$ (2/25) gave a valid “negative” result on the second attempt (1 from Site 2 and 1 from Site 4). The remaining 23 were “Unresolved” on the second attempt, therefore, were not eligible for inclusion in the data analysis. All 23 samples were tested negative by the composite reference methods.

Site poolability of performance data was determined based on the fact that similar Pro hMPV+ performance were obtained from the 4 clinical sites: Site 1: Sensitivity $9 0 . 0 \%$ ( $9 5 \%$ CI: $5 9 . 6 \% - 9 8 . 2 \%$ ); Specificity $9 9 . 8 \%$ ( $9 5 \%$ CI: $9 8 . 6 \% - 1 0 0 \%$ ) Site 2: Sensitivity $9 2 . 3 \%$ ( $9 5 \%$ CI: $7 5 . 9 \% - 9 7 . 9 \% )$ ); Specificity $100 \%$ ( $9 5 \%$ CI: $9 8 . 5 \% - 1 0 0 \% )$ . Site 3: Sensitivity $9 6 . 7 \%$ ( $9 5 \%$ CI: $8 3 . 8 \% - 9 9 . 4 \%$ ); Specificity $9 7 . 8 \%$ ( $9 5 \%$ CI: $9 5 . 6 \% - 9 9 . 0 \%$ ) Site 4: Sensitivity $100 \%$ $9 5 \%$ CI: $3 4 . 2 \% \mathrm { ~ - ~ } 1 0 0 \%$ ); Specificity $100 \%$ $9 5 \%$ CI: $9 8 . 2 \% - 1 0 0 \%$ )

The performance data from all study sites are presented in the following table:

<table><tr><td rowspan=2 colspan=1>Pro hMPV+ AssayResults</td><td rowspan=1 colspan=3>Composite Reference Methods Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1199</td><td rowspan=1 colspan=1>1203</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>1207</td><td rowspan=1 colspan=1>1275</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 94.1% (95% CI: 85.8% - 97.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.3% (95% CI: 98.7% - 99.7%)</td></tr></table>

The general demographic data for all eligible prospective specimens $\scriptstyle ( \mathrm { N } = 1 2 7 5$ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>617 (48.4%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>654 (51.3%)</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>4 (0.3%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>596 (46.7%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>254 (19.9%)</td></tr><tr><td rowspan=1 colspan=1>22- 59 years</td><td rowspan=1 colspan=1>219 (17.2%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>206 (16.2%)</td></tr><tr><td rowspan=1 colspan=1>Not Determined</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr></table>

b. Retrospective Clinical studies

Not applicable. c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

In the Pro $\mathrm { { h M P V + } }$ Assay clinical study, a total of 1275 eligible nasopharyngeal (NP) swab specimens were tested from four U.S. clinical laboratories across the United States during the 2008 respiratory virus season (January - March). The number and percentage of hMPV RNA positive cases by the Pro hMPV $^ +$ Assay, calculated by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Total N</td><td rowspan=1 colspan=1>hMPV RNA PositiveBy the Pro hMPV+Assay</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Number Positive</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>&lt; 1 year</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>8.4%</td></tr><tr><td rowspan=1 colspan=1>1-5 years</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10.0%</td></tr><tr><td rowspan=1 colspan=1>6-10 years</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6.1%</td></tr><tr><td rowspan=1 colspan=1>11-15 years</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5.3%</td></tr><tr><td rowspan=1 colspan=1>16-21 years</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>&gt; 21 years</td><td rowspan=1 colspan=1>425</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1275</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>5.6%</td></tr></table>

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a

MagNA Pure LC Instrument (Roche)

NucliSENS® easyMAG™ System (bioMérieux)

# O. System Descriptions:

# 1. Modes of Operation:

The Cepheid Smart Cycler II Real Time instrument with Dx software version 1.7b or 3.0a is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Four other nucleic acid amplification tests that use the Smart Cycler II instrument have received 510(k) clearance: Prodesse’s ProFlu $^ +$ Assay (K081030), IDIMRSA test (K033415), the IDI-Strep B Assay (K022504), and the Influenza A/H5 (Asian lineage) Virus Real-Time RT-PCR Primer and Probe Set (K060159). The Cepheid SmartCycler instrument is an integrated nucleic acid amplification and detection instrument system based on Cepheid’s proprietary microprocessor-controlled I-CORE module. For purified RNA samples, the SmartCycler instrument enables reverse-transcriptase (RT) to transcribe target viral RNA into cDNA, polymerase chain reaction (PCR) for the amplification of cDNA, and hybridization of fluorogenic targetspecific probes for the detection of the amplified cDNA.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles. Three other tests that use the Roche MagNA Pure LC system have received 510(k) clearance: Prodesse’s ProFlu $^ +$ Assay (K081030), Roche Factor V Leiden Kit (K033607) and the Roche Factor II (Prothrombin) G20210A Kit (K033612).

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads. Prodesse’s ProFlu $^ +$ Assay (K081030) that received Special 510(k) clearance also uses the easyMAG system for automated nucleic acid extraction.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The following controls are provided in the Pro hMPV $+$ Assay kit:

Positive Control (PC): The Pro hMPV $^ +$ Assay kit contains an hMPV positive RNA control that is an RNA transcript of a plasmid containing the viral sequence of interest and is not intact virus particles. The PC does not go through nucleic acid isolation and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance.

Internal Control (IC): An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working.

The following controls are not provided in the Pro hMPV $^ +$ Assay kit, but are required or recommended and described in the Pro hMPV $^ +$ Assay Instructions for Use:

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the Pro hMPV $^ +$ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a non-infectious hMPV strain (A2 lineage) obtained from ZeptoMetrix Corp. (Buffalo, NY), was included with each extraction run as EC. The sponsor is recommending this extraction control to the end users in the package insert.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.