# Nanosphere

MAY - 1 2009

# Summary of 510(k) Safety and Effectiveness

This summary of the 510(k) safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1900 and CFR 807.92.

# 510(k) number:

K083088

# Summary preparation date:

April 29, 2009

# Submitted by:

Nanosphere, Inc.   
4088 Commercial Avenue   
Northbrook, IL 60062   
Phone: 847-400-9000 Fax: 847-400-9199

# Contact:

Gregory W. Shipp, M.D.

Chief Medical Officer VP, Medical and Regulatory Affairs and Quality Assurance

# Proprietary names:

For instrument: Verigene System

For the assay: Verigene® Respiratory Virus Nucleic Acid Test

# Common names:

For the instrument: Bench-top molecular diagnostics workstation

For the assays:   
Respiratory panel   
Respiratory virus panel   
Respiratory viruses   
Influenza A assay   
Influenza B assay   
RSV assay   
Influenza A/B and RSV assay 847-40u-919

# Device descriptions:

The Verigene Respiratory Virus Nucleic Acid Test (VRNAT) is based on uniquely identifying virus-specific nucleic acids for Influenza A virus, Influenza B virus, and Respiratory Syncytial Virus (RSV). The VRNAT involves the following steps: (i) Sample Preparation: Isolation of the viral RNA from nasopharyngeal swab specimens obtained from symptomatic patients. Sample preparation is conducted on an automated sample isolation system (NucliSens® easyMAG™ System, bioMérieux); (ii) Multiplex RT-PCR procedure for the generation of virus-specific amplicons: A specified volume of the eluted genomic RNA from the sample preparation step is subjected to an RT-PCR target

amplification step; (ii) Verigene Test: Conducted on the Verigene® System for detection and identification of virus-specific amplicons.

The Verigene System consists of two instruments, the Verigene Processor and the Verigene Reader, and utilizes single-use, disposable Test Cartridges.

The Verigene Reader is a bench-top, free-standing instrument with a touch screen control panel and a wand-based barcode scanner. It utilizes a graphical user interface to guide the user through the process of ordering tests and reporting results. There are no serviceable parts and no user calibration is required. Interaction with the touch screen is minimized through barcode use. This instrument also serves as the reader of the hybridization substrate using optical detection.

The key functions of the Verigene Reader include:

$\circ$ Entry and tracking of specimen identification numbers via manual keyboard input or via barcode-reader wand.   
o Test selection for each specimen. - Automated transfer of specimen processing instructions on Test Cartridgespecific basis to linked Processor unit(s).   
$\circ$ Automated imaging and analysis of Test Cartridges.   
$\circ$ Results display.   
$^ \circ$ Results report generation.

The Verigene Processor performs the Verigene Test under the direction of the Verigene Reader. It has been designed to be simple and easy to use with minimal user interaction. It contains no fluids and has no user calibration requirements. There are four hybridization modules in each Verigene Processor and up to eight Verigene Processors may be connected to a single Verigene Reader. The modules within a Verigene Processor can simultaneously run different tests. When a Test Cartridge containing the sample mix is inserted, a barcode reader internal to the Verigene Processor modules reads the cartridge ID and sends it to the Verigene Reader. From this information the Verigene Reader establishes the hybridization parameters and automatically starts the Verigene Test.

The Test Cartridge consists of two parts: a Reagent Pack with reservoirs preloaded with test-specific reagents, and a Substrate Holder. The reagent pack creates an air-tight hybridization chamber surrounding the region of the substrate-containing target-specific capture array. As each step in the Verigene Test is completed, old reagents are moved out of the hybridization chamber and new reagents are added from the reagent pack via microfluidic channels and pumps." cal dctecion

To run the Verigene Test, the user mixes the sample with a test-specific Sample Buffer and loads this sample mix into the Test Cartridge. The Test Cartridge is subsequently inserted into the Processor. Both the identity of the test contained in the Test Cartridge and the associated patient specimen are linked with a common barcode that is unique to each Test Cartridge. Test information entered at the beginning of each test session is used to direct the associated module of the Processor unit on what steps to execute to process the Test Cartridge, and the patient specimen information is used for tracking and results reporting. Once the test is complete, the Test Cartridge is removed from the Processor unit and the reagent pack'is snapped off and discarded. The remaining Test Substrate is now ready for imaging and analysis in the Reader unit.

The Test Substrate is inserted into the Reader wherein it is iluminated along its side. The gold-silver aggregates at the test sites scatter the light, which is in turn captured by a photosensor. The relative intensity arising from each arrayed test site is tabulated. Net signals, defined as the absolute signal intensities with background signals subtracted, are compared with thresholds determined by negative and positive controls within the slide in order to arrive at a decision regarding the presence or absence of target. These results are linked to the test and patient information entered at the beginning of each test session to provide a comprehensive results file.

specimens obtained from patients symptomatic for viral upper respiratory infection. The test is intended to be used on the Verigene System as an aid in the differential diagnosis of Influenza A, Influenza B, and RSV infections. The test is not intended to detect Influenza C virus.

Negative results do not preclude influenza virus or RSV infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative test results be confirmed by culture.

Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

# Predicate device:

The Prodesse ProFlu+ TM. Assay [K073029 and K081030] is claimed as the predicate device for the Verigene® Respiratory Virus Nucleic Acid Test.

The ProFlu $^ +$ Assay is a multiplex Real Time RT-PCR in vitro diagnostic test for,the rapid and qualitative detection and discrimination of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens obtained from symptomatic patients. This test is intended for use to aid in the differential diagnosis of Influenza A, Influenza B, and RSV viral infections in humans and is not intended to detect Influenza C virus.

Negative results do not preclude influenza virus or RsV infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative RsV results be confimed by culture?aiem

The tables below compare the Verigene Respiratory Virus Nucleic Acid Test with the predicate device:

Similarities to predicate device

<table><tr><td rowspan=1 colspan=1>Feature   </td><td rowspan=1 colspan=1>Verigene Respiratory Virus vilIse Nucleic Acid TestOr local</td><td rowspan=1 colspan=1>La.r{}$    pProFlutstAssayi:..:mro song&quot;</td></tr><tr><td rowspan=1 colspan=1>Intended use</td><td rowspan=1 colspan=1>il  &#x27;The Verigene Respiratory VirusNucleic Acid Test is a qualitative invitro diagnostic for the identificationof Influenza A virus, Influenza Bvirus, and Respiratory SyncytialVirus (RSV) in nasopharyngealswab specimens as an aid in thedifferential diagnosis of patientssymptomatic or ial upperrespiratory infection. The test is not.intended to detect Influenza C virus.The test is intended to be used onthe Verigene System.Negative results do not precludeinfluenza virus or RsV infection andshould not be used as the sole basisfor treatment or other management</td><td rowspan=1 colspan=1>.!The ProFlu+ Assay is a multiplexReal Time RT-PCR in vitrodiagnostic test for the rapid andqualitative detection anddiscrimination of Influenza A Virus,Influenza 8 Virus, and RespiratorySyncytial Virus (RSV) nucleic acidsisolated and purified fromgnasopharyngeal (NP) swabspecimens obtained fromsymptomatic patients. This test isintended for use to aid in thedifferential diagnosis of Influenza A,Influenza B and RSV viral infectionsin humans and is not intended todetect Influenza C.Negative results do not preclude</td></tr></table>

daybun fudenc/ed fest wit the psiake da

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>decisions. It is recommended thatnegative RSV results be confirmedby culture.</td><td rowspan=1 colspan=1>influenza or RSV virus infection andshould not be used as the sole basisfor treatment or other managementdecisions. It is recommended thatnegative RSV results be confirmedby culture.: :.</td></tr><tr><td rowspan=1 colspan=1>Targets</td><td rowspan=1 colspan=1>Influenza AInfluenza BRSV</td><td rowspan=1 colspan=1>Influenza AInfluenza BRSV</td></tr><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Nasopharyngeal swabs in samplematrix</td><td rowspan=1 colspan=1>Nasopharyngeal swabs in samplematrix</td></tr><tr><td rowspan=1 colspan=1>Sample preparation</td><td rowspan=1 colspan=1>Automated extraction of nucleicacids</td><td rowspan=1 colspan=1>Automated extraction of nucleicacids</td></tr><tr><td rowspan=1 colspan=1>Sample size</td><td rowspan=1 colspan=1>200 μL</td><td rowspan=1 colspan=1>200 μL</td></tr><tr><td rowspan=1 colspan=1>Quality control</td><td rowspan=1 colspan=1>Internal procedural quality controlsincluding a process control and aninhibition control and external qualitycontrol solutions</td><td rowspan=1 colspan=1>Internal procedural quality control,external quality control solutions</td></tr><tr><td rowspan=2 colspan=1>Amplification method</td><td rowspan=1 colspan=1>Multiplex RT-PCR</td><td rowspan=1 colspan=1>Multiplex RT-PCR</td></tr><tr><td rowspan=1 colspan=1>M-MLV Reverse Transcriptase</td><td rowspan=1 colspan=1>M-MLV Reverse Transcriptase</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Positive or negative qualitativeresults</td><td rowspan=1 colspan=1>Positive or negative qualitativeresults</td></tr></table>

S ls .i" ·

# Differences from predicate device

<table><tr><td rowspan=1 colspan=1>Feature     &quot;&quot;I</td><td rowspan=1 colspan=1>Verigene Respiratory VirusNucleic Acid Test</td><td rowspan=1 colspan=1>by pProdesse ProFlu Assay</td></tr><tr><td rowspan=1 colspan=1>Amplification method</td><td rowspan=1 colspan=1>Tfi polymerase</td><td rowspan=1 colspan=1>Taq polymerase</td></tr><tr><td rowspan=1 colspan=1>Contamination control</td><td rowspan=1 colspan=1>Thermolabile Uracil DNAGlycosylase (UDG) to limit ampliconcontamination</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Test principle</td><td rowspan=1 colspan=1>Gold nanoparticle probe-basedchemistry</td><td rowspan=1 colspan=1>NisopygTaqman chemistry</td></tr><tr><td rowspan=3 colspan=1>Detection method</td><td rowspan=1 colspan=1>End-point detection of the amplifiedmaterial on the Verigene System</td><td rowspan=1 colspan=1>Real-time RT-PCR</td></tr><tr><td rowspan=1 colspan=1>Single-image sensor wherenanoparticles are illuminated using afixed-wavelength light source</td><td rowspan=1 colspan=1>Fluorescence-based detection</td></tr><tr><td rowspan=1 colspan=1>Microarray format: vastly greatermultiplex capability. Allows forgreater coverage of strain variantswithin the viruses..</td><td rowspan=1 colspan=1>Real-time detection format: limitedmultiplex capability.</td></tr><tr><td rowspan=1 colspan=1>Reagent storage</td><td rowspan=1 colspan=1>Test cartridges and Sample Bufferstored at 2 - 8 °C.Amplification kit stored at -20 .Controls stored at -20 C.</td><td rowspan=1 colspan=1>117.1All reagents (opened andunopened) stored at -70 C.</td></tr><tr><td rowspan=1 colspan=1>Quality control</td><td rowspan=1 colspan=1>Two internal procedural controls:Process Control - controls forsample isolation and the RT-PCRsteps.(ii) Inhibition Control - controls forthe amplification step.</td><td rowspan=1 colspan=1>Singie internal procedural control.</td></tr></table>

# Performance Characteristics

A. Clinical Performance

A total of 720 nasopharyngeal swab specimens were prospectively collected during the 2007-2008 respiratory seon or outine influenz or RV testing by DAculturemeths.he residual ecimens were roze an l   e uc patient population was distributed by age as presented in the table below:

<table><tr><td rowspan=1 colspan=1>Patient Population by Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>0-1</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>1-5</td><td rowspan=1 colspan=1>229</td></tr><tr><td rowspan=1 colspan=1>5-20</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=1>20 -65</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>        Total</td><td rowspan=1 colspan=1>row       720;, &quot;.ti</td></tr></table>

Samples with an initial 'No Cal' result were e-tested successfully by following the recommendations in he 'Results Interpretation' section (see above). A total of 21 samples $( 2 . 9 \% )$ generated a $\ " \mathsf { N o }$ Call" result; ten (10) of the samples $( 1 . 4 \% )$ gave a 'No Call — Indeterminate Zone' result indicating an equivocal call. Repeat testing resolved each of these samples. Another eleven (11) samples $( 1 . 5 \% )$ gave a 'No Call — Pos Ctrl' call indicating an inability to detect an internal control. All the samples were resolved upon retest.

The VRNAT performance was compared to a culture-based reference method followed by direct fluorescent y )nlcultu posipTeu rprine below. Discordant results between the VRNAT and the reference method were followed up by using bidirectional sequencing at an independent reference laboratory and described in footnotes.

# Influenza A Results

<table><tr><td rowspan=2 colspan=2>All SitesInfluenza A</td><td rowspan=1 colspan=2>DFAViral Culture</td><td rowspan=1 colspan=1>- . · &#x27;    1</td><td rowspan=1 colspan=1>&quot;</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>:Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=2 colspan=1>VerigeneSystem</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>Sensitivity 99.2%(95%C|=95.5% - 99.9%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>538</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>Specificity 90.1%(95%Cl=87.5% - 92.3%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>597</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1>:1</td></tr></table>

a 58 samples were positive by sequencing. 1 sample was negative by sequencing. b. 1 sample was negative by sequencing

# Influenza B Results

<table><tr><td rowspan=2 colspan=2>All SitesInfluenza B</td><td rowspan=1 colspan=2>DFAViral Culture</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=2 colspan=1>VerigeneSystem</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10 </td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Sensitivity 96.8%(95%C|=83.5% - 99.4%)</td></tr><tr><td rowspan=1 colspan=1>i101Negative7w</td><td rowspan=1 colspan=1>PA</td><td rowspan=1 colspan=1>· J</td><td rowspan=1 colspan=1>I.SEL</td><td rowspan=1 colspan=1>·...Specificity 98.5%and679relei+1e-680+the4-v(95%Cl=97:3%~99.2%): ·</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total  </td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>689</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1></td></tr></table>

a. 4 samples were positive by sequencing. 6 samples were negative by sequencing b. 1 sample was negative by sequencing

![](images/d9dd4af8bc301f09b63de7020a25dc407cd79964a85cd06ef3aebe6ea9c86bf0.jpg)

# RSV Results

<table><tr><td rowspan=2 colspan=2>All SitesRSV</td><td rowspan=1 colspan=2>DFA/Viral Culture</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive.</td><td rowspan=1 colspan=1>Negative-</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>VerigeneSystemI</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>44 </td><td rowspan=1 colspan=1>57b</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Sensitivity 89.8%(95%C|=78.2% - 95.6%)</td></tr><tr><td rowspan=1 colspan=1>Negative&quot;ii  $</td><td rowspan=1 colspan=1>5c</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>Specificity 91.5%(95%CI=89.2% - 93.4%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total  i</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1></td></tr></table>

a All 44 samples were positive by sequencing. b. 46 samples were positive by sequencing. 11 samples were negative by sequencing. C. 4 samples were positive by sequencing. 1 sample was negative by sequencing.

# Dual Infections

O e 720 samples included in the clinical study, 2 samples were positive for infections by the VRNAT, translating to $1 . 7 \%$ of the samples tested. Comparable RT-PCR—based real-time detection methods have reported a significantly lower percentage for dual infections $( \sim 0 . 1 \% )$ . Two inter-related factors may contribute to thehigher perntag ealinfet sr n heRACpetitihibtion is fns ualpeahenetivgen   ownt trati ultie amplicons below the detection limits of fluorescence-based methodsThe Verigene System uses ol nanoparticle probe-based technology that is at least 2-3 lg orders more sensitive than fluorescencebased methods

# B. Reproducibility & Precision

en p ea ylt crat c wirlrr Meor  l RSV wereuse  preparehe amplesInfluenzaAWisconsi/67/2005;Influenz / Florida/04/2006;RSV Strain Long; RSV B Strain B-1 Wild Type (B WV/14617/85). Each strain was represented at 3 distinct concentrations: high negative, low positive, and moderate positive.

<table><tr><td rowspan=1 colspan=1>Unique Samples</td><td rowspan=1 colspan=1>Viral Strains and Levels</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A - High Negative; Influenza B - High Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>RSV A - High Negative; RSV B - High Negative</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Influenza A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>RSV A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>RSV B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Influenza A - Moderate Positive; RSV A - Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Influenza B - Moderate Positive; RSV B - Moderate Positive</td></tr></table>

The precision study Site1)used the same set  8 samples and was performed ovr2on-consecutiv ys. yeao RAT,vioatnrpltn VT  p lat ple   y alIn e pe   es we alyzeneVerg st riplicat aiy y on each of five non-consecutive days after undergoing sample isolation and target amplification.

InRebly n recsnshea posiivpl r s postiv contl they were expected to provide positive decisions $100 \%$ of the time. The same samples also served as negative conrols; Not Detecteddecisions were expeced at Influenza  andRSV Bin he moderate positive samples containing Influenza A and RSV A and conversely Not Detected'decisions were expected at Influenza A and

RSV A in hemoderate positive samples containig Influenza B anRSV B.The positive andnegative cols provided expected decisions during the studies. Calls made by the Verigene System were compared to the expected outcome as the measure of performance. The table below shows the summary results of the reproducibility study.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Characteristic</td><td rowspan=1 colspan=4>Verigene Respiratory Virus Nucleic Acid Test</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>RSV B</td></tr><tr><td rowspan=6 colspan=1>Too Tj</td><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>75/78(96.2%)</td><td rowspan=1 colspan=1>78/78(100%)</td><td rowspan=1 colspan=1>76/78(97.4%)</td><td rowspan=1 colspan=1>74/78(94.9%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>89.3%-98.7%</td><td rowspan=1 colspan=1>95.3%-100%</td><td rowspan=1 colspan=1>91.1%-99.3%</td><td rowspan=1 colspan=1>87.5%-98.0%</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>77178(98.7%)</td><td rowspan=1 colspan=1>77/78(98.7%)</td><td rowspan=1 colspan=1>77/78(98.7%)</td><td rowspan=1 colspan=1>78/78(100%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>93.1%- 99.8%</td><td rowspan=1 colspan=1>93.1%-99.8%</td><td rowspan=1 colspan=1>93.1%-99.8%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>78/78(100%)</td><td rowspan=1 colspan=1>78/78(100%)</td><td rowspan=1 colspan=1>78/78(100%)</td><td rowspan=1 colspan=1>78/78(100%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.3%-100%</td><td rowspan=1 colspan=1>95.3%-100%</td><td rowspan=1 colspan=1>95.3%-100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr></table>

# C. Analytical Sensitivity

Talylty y e w y ilpi nilT Me li ic as the lowest concentration at which $\tt R9 5 \%$ of the replicates tested positive, was assessed based on the ep  al sr s elL s pal  ati a of the time.

<table><tr><td rowspan=1 colspan=1>Limits of Detection         zitis          18178F0A0</td><td rowspan=1 colspan=1>Concentration247 407</td></tr><tr><td rowspan=1 colspan=1>Influenza A strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza AWisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Caledonia/20/99 (H1N1)</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Port Chalmers/1/73 (H3N2)</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/04/2006</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Lee/40</td><td rowspan=1 colspan=1>0.01 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Hong Kong/5/72</td><td rowspan=1 colspan=1>0.05 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A Strain A2                            i!</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A Strain Long</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B Strain B-1 Wild Type (B WV/14617/85)iLy 2. TCID50/mL</td><td rowspan=1 colspan=1>.2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain Wash/18537/62.ch in Ordtr </td><td rowspan=1 colspan=1>0.5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain 9320</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td></tr></table>

# 7 of 14

1 (:32)-----50 (CiDC

The Verigene Respiratory Virus Nucleic Acid Test (VRNAT) uses a microarray-based platform in which the i ne hibt onrol 1)an pro contol )aere b onit lment Te Substraeitin elment  ots aeviussecilgonceotde quencs at in aplifil argh ui olarc ros ditl it so rc proecale acnepos i epeTet fh ot eVegeReaerangise  alinesinn e it ments, heTes Substrate as sot speci to positi contol C)an egative conrol NC). Three conditions were identified that together served as a single set of clinical cutoff criteria.

Condition 1: Noise Threshold

ContionNoralizeRat o NegativeContro(Rato-o-NC intensiy t thevirs-speciicrecoit element normalized against the intensity values at the negative control elements.

Conition3:Normalized Rato toPositiveControl Ratio-to-C) intensity at hevirus-specificecoitn element normalized against the intensity values at the positive control elements.

The Noise Threshold was determined empirically to be equal to Negative Control $\star 2$ SD. The Noise Threshold was determined empirically to be equal to Negative Control $^ { + 2 }$ SD. The cut-offs for the normalized ratios, NCn e RatoC'we e y sig ROCursor  positi eec following criteria apply:

Condition 1: Signal intensity is above the noise threshold Condition 2: Ratio-to-NC is above 0.85 Condition 3: Ratio-to-PC is above -0.4

o the three viruses falls between -0.4 and $- 0 . 5 .$ an equivocal (i.e., No Call - IND ZONE) decision is provided. Criteria set for each of the three conditions are required to be met for a 'Detected' call.

# VRNAT Decision Process

sulopncie piobes viaaonalrooc I.. !w.. . ..

For  rult, he ecsn eeees he eceot C1 n ita ee Shemac. Bo 1 PC signal intensits have to meet the detection criteria before the analysis isallowed to proceed toward establishing the presence or absence (i.e., "Detected" or "Not Detected") of the individual viruses.

![](images/672fb19a7ef27542873d11a538568d6af75b0ae29e52d3ee96c236d3b52d46cc.jpg)

# Analytical Reactivity

TaviciviReuc aval srains o Infuenza A, Influenza B, RSV A, andRSV B that ae representativef temporal an geographical iversiynon connratinshe ffrent ltra rown  cultue we ilu n sampl nirsal Transport Meda, Copan) to achieve the titers listed ee Table belowAtotal of 4 strais or Inz  wes  0/oza  toalstr we est1 TCID/mL. A total of 5RSV strains wee tested at 100 TCiD50/mL r RSV A (2 strains) and RSV B (3 strains). Eas saenTlculu e VRNAT analytical reactivity demonstrated $100 \%$ concordance with expected results.

<table><tr><td rowspan=1 colspan=1>Analytical Reactivity</td><td rowspan=1 colspan=1>Conc.</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>Influenza A strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Canada/578/04 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Caledonia/20/99 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Hong Kong/29/2006 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza AWisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/PR/8/34 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Aichi/68 (H3N2)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza AWS/33 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Port Chalmers/1/73 (H3N2)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/ Brisbane/59/ 2007 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Brisbane/10/2007 (H3N2)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A1/Denver/1/57 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A1/FM/1/47 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>1.1 +</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Jersey/8/76 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>I +</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/NWS/33 (H1N1)</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza 8/Yamanashi/166/98</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Panama/45/90</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/02/2006</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/04/2006</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/ Lee/40</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Egypt/2040/04</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>Infl</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/GL/1739/54</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Hong Kong/5/72</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Maryland/1/59</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Taiwan/2/62</td><td rowspan=1 colspan=1>100 EID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A Strain A2</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV A Strain Long</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B Strain B-1 Wild Type (B WV/14617/85)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain Wash/18537/62</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain 9320</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>+</td></tr></table>

# F. Analytical Specificity/Cross-Reactivity

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>..Inifluenza.A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>pfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus Type 1</td><td rowspan=1 colspan=1>Adenoid 71</td><td rowspan=1 colspan=1>3.1x10^r}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human Adenovirus Type 7</td><td rowspan=1 colspan=1>Gomen</td><td rowspan=1 colspan=1>3.1x10^r}$</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human caronavirus (OC43)</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>1.1x10r}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human coronavirus (229E)</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>$1.1x10^r}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>68-1</td><td rowspan=1 colspan=1>$1x10fr{}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Enterovirus, H. coxsackievirus B4</td><td rowspan=1 colspan=1>J.V.B.</td><td rowspan=1 colspan=1>$1x10fr{}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>B95-8</td><td rowspan=1 colspan=1>6.2x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Human Parainfiuenza Type 1</td><td rowspan=1 colspan=1>C35</td><td rowspan=1 colspan=1>6.2x10}$</td><td rowspan=1 colspan=1>…</td><td rowspan=1 colspan=1>·</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>$2x10fr}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 3</td><td rowspan=1 colspan=1>C243</td><td rowspan=1 colspan=1>1.1x107</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>Edmonston</td><td rowspan=1 colspan=1>$1.1x10r}$</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>Enders</td><td rowspan=1 colspan=1>1.1x10f}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 4a</td><td rowspan=1 colspan=1>M-25</td><td rowspan=1 colspan=1>1.1x10fr}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus Type 1A</td><td rowspan=1 colspan=1>2060</td><td rowspan=1 colspan=1>$1.1x10fr}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>B-1</td><td rowspan=1 colspan=1>5.6x10^4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$3x10r}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$3x10r}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>CM-1</td><td rowspan=1 colspan=1>1.6x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium pseudodiphtheriticum</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.2x10}$</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.5x10}$</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumaniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>15x10r}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.1x10^</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.1x10f}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Listeria innocua</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.2×10^</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$1x10^{}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5x10}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$1x10r{}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6x10^8}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6.9x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$6.1x1}</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.3x10}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$9x10}$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$2x10r}$</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.5x10}$</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.3x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$3x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis, attenuated*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.6x10}$</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

87x178-1-

Analytical specificity studies were performed to assess potential cross-reactivity of the VRNAT with common respiratory pathogens and other microorganisms commonly present in specimens collected from the al 8t w f s at s wi whic atnay eh clvs nac ra whic we oga th eeidoscrosscvi gni as iu  slai Unl or Mednheanishblelo knro VRNAT protocol.

No  henis es nee wih e teal cnoo l en sVA cros-eactivity'Not Detected calls were made by theVergeReader r eachf the three viruses n VRNAT, Influenza A, Influenza B, and RSV. No cross-reactivity was observed with he comon respry pathogens and organisms infecting the majority of the population.

# G. Competitive Inhibition

Competitive hibtion in eVRNAT was assessed by usig samples containing 2 viruses where e vis was ${ \sim } 3$ lie han er vTypcaly eise wen was l near nTus, ech viusn te panel was tes agaist the igtoviss hihercner a heeg esult was uss eu ossg heollowgrais wep thestudy: Influenza AWisconsin/67/05 (H3N2); InfluenzaB/Florida/04/2006; RSV A StrainA2; RSV B Strain Wash/18537/62.

In the binary combinations tested in these studies there was no evidence of competitive inhibition.

<table><tr><td rowspan=1 colspan=1>Low</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TCID50/ML</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>&#x27;Influenza B</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=2>Influenza B</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=2>RSV A</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=2>RSV B</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>RSVA</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>RSV B</td><td rowspan=1 colspan=1>50000  ,</td><td rowspan=1 colspan=1>..</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=2>Influenza B</td><td rowspan=1 colspan=1>50000</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>*</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=1>50000.1</td><td rowspan=1 colspan=1>  </td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=2>Influeniza B</td><td rowspan=1 colspan=1>1500000</td><td rowspan=1 colspan=1>porconconirn</td><td rowspan=1 colspan=1>&quot;71&#x27;95 a^&#x27;1na</td><td rowspan=1 colspan=1>I·*°+</td></tr></table>

H. Fresh vs. Frozen

A s  92 respiratory specimens were collected prospectively during the 2007-2008 respiratory season and e o eVReay Vi ucAesi anz spleAlqot  e ses wee s byusiVRAT whiln eeh,zen sa presence of Flu AB and RSV. Far each sample, the remaining sample'volume was stored frozen at $\leq - 7 0 ^ { \circ } C$ for a minimum of four 4) months.The frozen samples were thawed and re-tested by using the VRNAT.The Vergne s results r e e sampleandhe corepondin ozen samples we copared tasss te VRNAT performance.

zp cnaly was insufficent volue to perorm testsr both fresh and frozen samples. total o 9 pai samples w nal p g psDul viruss detected by VRNATAnother 9 paired fresh and frozen samples gave 'Not Detected cals for all e i l ewe hre irnt ul s thr  ozpls hat geat positive calls gave 'Not Detected' calls for the corresponding fresh samples.

<table><tr><td rowspan=3 colspan=2>Fresh vs. FrozenComparison</td><td rowspan=1 colspan=6>FRESH..... ... .  . -,</td></tr><tr><td rowspan=1 colspan=2>Influenza A</td><td rowspan=1 colspan=2>Influenza B</td><td rowspan=1 colspan=2>RSV</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>DetectedNot Detected</td><td rowspan=1 colspan=1>DetectedNot Detected</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td></tr><tr><td rowspan=4 colspan=1>FROZEN</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Positive %Agreement</td><td rowspan=1 colspan=2>100%(95%CI=56.6%-100%)</td><td rowspan=1 colspan=2>100%(95%C1=78.5%-100%)</td><td rowspan=1 colspan=2>100%(95%Cl=67.6%-100%)</td></tr><tr><td rowspan=1 colspan=1>Negative %Agreement</td><td rowspan=1 colspan=2>100%(95%CI=95.6%-100%)</td><td rowspan=1 colspan=2>100%(95%CI=91.5%-100%)</td><td rowspan=1 colspan=2>96.30%(95%CI=89.7%-98.7%)</td></tr></table>

The discordant results hree pairdsamples negative or al nalytes when tested fresh but positive or RSV h ts  ey   zw enalyde oa oe nz saples was subjected to i-dretnal sequencing.Sequncing results confirmed that wo the e saple pairs wereRV-positivein feh n frozen states; eatatpts mpliy eAfm e fresh and frozen aliquots of the third sample failed to yield an amplicon needed for sequencing.

1. Carry-Over / Cross-Contamination Study

Biwasv sample preparation on the NucliSens EasyMAG (bioMérieux), RT-PCR step, and the Verigene Test. Moreover, there was no evidence of any cross over within the Verigene Processor modules when high titer samples were alternated with low titer samples.

# J. Interferences

T a seciens on the Verigene Respiratory Nucleic Acid Test (VRNAT) was assessed.The viral strains and the n he sds e iste TTheineen d e pl ne described in Table 2.

Table 1   

<table><tr><td rowspan=1 colspan=1>Viral Strain                                      ;</td><td rowspan=1 colspan=1>Concentration (TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Influenza AWisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Maryland/1/59</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>RSVA Stran.Long5</td><td rowspan=1 colspan=1>5071</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain B-1 Wild Type (B WV/14617/85)</td><td rowspan=1 colspan=1>10</td></tr></table>

Non ofthe poenial inteerents,excpt the Fluvacci ffed he VRNAT perforanceakig thecor 'Detd' calls for he panel viuses preent in smples containng the interferents.Seril dilutis  e injableFluvaciecoisinnacivatflenzavisvaccie trais Infnz  Flu A/S Islans/3/2006; Flu AWisconsin/67/2005) and Influenza B (Flu B/Malaysia/2506/2004) was tested in the VRNAT c nz nRSVAB a.et viserigalv

distributed equally among the three strains, was $9 . 0 \times 1 0 ^ { - 5 } \ g / m L$ .Based on this value, it was determined that Influenza A was not detectable below $4 . 5 \times 1 0 ^ { - 1 1 }$ g/mL'and Influenza B was not detectable below 4.5x10"13 g/mL.

Table 2   

<table><tr><td rowspan=1 colspan=1>Active Interferent</td><td rowspan=1 colspan=1>Source     7. MTYF</td><td rowspan=1 colspan=1>Amount</td></tr><tr><td rowspan=1 colspan=1>No Interferent</td><td rowspan=1 colspan=1>Universal Transport Media</td><td rowspan=1 colspan=1>Not Applicable</td></tr><tr><td rowspan=1 colspan=1>Human Blood</td><td rowspan=1 colspan=1>Human Blood</td><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>3 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>Tamiflu</td><td rowspan=1 colspan=1>15 mg/mL</td></tr><tr><td rowspan=1 colspan=1>NaCl</td><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>10% v/v</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>MucinexD</td><td rowspan=1 colspan=1>4 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>Neo-Synephrine</td><td rowspan=1 colspan=1>10% v/v of sample</td></tr><tr><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>Sinex</td><td rowspan=1 colspan=1>10% v/v of sample</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>Pulmicort</td><td rowspan=1 colspan=1>40 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Fluticasone propionate</td><td rowspan=1 colspan=1>Flonase</td><td rowspan=1 colspan=1>2.5% v/v</td></tr><tr><td rowspan=1 colspan=1>Homeopathic Remedies</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Luffa opperculata</td><td rowspan=1 colspan=1>Similason Sinus Relief</td><td rowspan=1 colspan=1>1% v/v of sample</td></tr><tr><td rowspan=1 colspan=1>Sulfur</td><td rowspan=1 colspan=1>Boiron</td><td rowspan=1 colspan=1>4.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Galphimia Glauca</td><td rowspan=1 colspan=1>Boiron</td><td rowspan=1 colspan=1>4.5 mg/ml.</td></tr><tr><td rowspan=1 colspan=1>Histaminum Hydrochloricum</td><td rowspan=1 colspan=1>Boiran</td><td rowspan=1 colspan=1>4.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Beclomethasone dipropionate</td><td rowspan=1 colspan=1>Beclomethasone dipropionate</td><td rowspan=1 colspan=1>210 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>Flunisolide</td><td rowspan=1 colspan=1>125 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>Triamcinolone acetonide</td><td rowspan=1 colspan=1>27.5 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Guaifenesin</td><td rowspan=1 colspan=1>Robitussin</td><td rowspan=1 colspan=1>2 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Diphenhydramine hydrochlorine</td><td rowspan=1 colspan=1>Benadryl</td><td rowspan=1 colspan=1>0.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Dextromethorphan hydrobromide</td><td rowspan=1 colspan=1>Delsym</td><td rowspan=1 colspan=1>1 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Pseudoephedrine hydrochloride</td><td rowspan=1 colspan=1>Sudafed</td><td rowspan=1 colspan=1>20 μg/mL</td></tr><tr><td rowspan=1 colspan=1>Benzocoaine</td><td rowspan=1 colspan=1>Cepacol</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Menthol</td><td rowspan=1 colspan=1>Halls</td><td rowspan=1 colspan=1>5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>1.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>10 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>1 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Flu Vaccine*</td><td rowspan=1 colspan=1>Afluria</td><td rowspan=1 colspan=1>Dilutions: 4.5x10 g/mL - 4.5x105 g/mL</td></tr></table>

\* Flu Vaccine was tested in the absence of the Influenza A/B and RSV A/B strains.

icihasomdpropionat 1\*-

![](images/bd89285c1836b890298daf9bd6f8b05ccdefc7ec1366871e7c9cc0875e6a4679.jpg)

# MAY - 1 2009

Gregory W. Shipp, MD   
Chief Medical Officer   
Vice President, Medical and Regulatory Affairs . .. and Quality Assurance   
Nanosphere, Inc   
4088 Commercial Avenue   
Northbrook, IL 60062

Re: K083088 Trade/Device Name: Verigene® Respiratory Virus Nucleic Acid Test Regulation Number: 21 CFR 866.3980.. .. Regulation Name: Respiratory viral panel multiplex nucleic acid assay Regulatory Class: Class II Product Code: OCC, NSU Dated: December 26, 2008 Received: April 7, 2009

Dear Dr. Shipp:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or class III (PMA), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895: In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820).

Page 2 -

This letter willallow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at 240-276-0450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). For questions regarding postmarket surveillance, please contact CDRH's Office of Surveillance and Biometric's (OSB's) Division of Postmarket Surveillance at 240-276-3474. For questions regarding the reporting of device adverse events (Medical Device Reporting (MDR)), please contact the Division of Surveillance Systems at 240-276-3464. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (240) 276-3150 or at its Internet address http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices Office of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and   
Radiological Health

Enclosure

# Indication for Use

510(k) Number (if known): K083088

Device Name: Verigene® Respiratory Virus Nucleic Acid Test

Indication For Use:

The Verigene® Respiratory Virus Nucleic Acid Test is a qualitative multiplex in vitro diagnostic test for the detection and identification of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids purified from nasopharyngeal swab specimens obtained from patients symptomatic for viral upper respiratory infection. The test is intended to be used on the Verigene® System as an aid in the differential diagnosis of Influenza A, Influenza B, and RSV infections. The test is not intended to detect Influenza C virus.

Negative results do not preclude influenza virus or RSV infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative test results be confirmed by culture.

Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

And/Or

Prescription Use X (21 CFR Part 801 Subpart D)

Over the Counter Use (21 CFR Part 801 Subpart C)