# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K083088   
B. Purpose for Submission: New device

A, Influenza B, and RSV are detected.

# D. Type of Test:

Multiplex nucleic acid assay for qualitative detection and identification of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) target sequences in nasopharyngeal swab specimens including nucleic acid isolation, multiplex RT-PCR amplification, capture of the amplicons hybridized to gold-labeled probes on a microarray-based chip, and detection of the signal enhanced with elemental silver using the Verigene® System

E. Applicant: Nanosphere, Inc.

F. Proprietary and Established Names: Verigene® Respiratory Virus Nucleic Acid Test Common or Usual Name: VRNAT

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OCC and NSU</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3980Respiratory viral panelmultiplex nucleic acid assay</td><td rowspan=1 colspan=1>Microbiology (83)</td></tr></table>

# H. Intended Use:

1. Intended use(s):

The Verigene® Respiratory Virus Nucleic Acid Test is a qualitative multiplex in vitro diagnostic test for the detection and identification of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids purified from nasopharyngeal swab specimens obtained from patients symptomatic for viral upper respiratory infection. The test is intended to be used on the Verigene® System as an aid in the differential diagnosis of Influenza A, Influenza B, and RSV infections. The test is not intended to detect influenza C virus.

Negative results do not preclude influenza virus or RSV infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative test results be confirmed by culture.

Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infections with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Verigene® System consisting of the Verigene Processor (software version 0.7) and the Verigene Reader (software version 1.5)

# I. Device Description:

The Verigene® Respiratory Virus Nucleic Acid Test (VRNAT) is a set of reagents and supplies designed to be utilized in RT-PCR and hybridization assay using the Verigene® instrument system for detection and identification of human influenza A, influenza B, and RSV viral RNA. The VRNAT primers target matrix (M) gene of influenza A, non structural (NS) and M genes of influenza B, and L gene and F gene of RSV. The amplicons are hybridized to gold nanoparticle probes through a mediator oligonucleotide and target-specific capture oligonucleotides on a microarray-based chip in a disposable test cartridge. The signal enhanced with elemental silver at the test site is detected using the Verigene Processor.

The VRNAT also includes control materials:

Internal Positive Control   
The internal positive control (PC) refers to a set of ‘capture’ oligonucleotides on the test microarray that directly hybridize to a specific gold nanoparticle probe present in the Reagent Pack of each Test Cartridge. The PC spots are further amplified during the silver amplification step.

Internal Negative Control The internal negative control (NC) refers to a set of oligonucleotides on the test microarray that have no significant homology to Influenza A, Influenza B, and RSV sequences available in public

databases.

Inhibition Control

Inhibition Control (PC1) or positive amplification control is diagnostic of potential inhibition effects during the target amplification step. It is a synthetic DNA construct designed to act as an artificial target within the VRNAT. It is included in the Primer Mix along with PC1- specific forward and reverse primer. The PC1 target binds specifically to a capture spot on the test chip and the Verigene Reader analyzes the PC1 signal.

# Process Control

The process control (PC2) is an intact MS2 bacteriophage and therefore controls for the nucleic acid extraction step and the target amplification or the RT-PCR step. In the VRNAT, isolated MS2 bacteriophage RNA is amplified in the RT-PCR step by virus-specific primers. The amplicon is then mixed with the target buffer containing MS2-specific mediators, it binds to the MS2-specific capture spot and to the probe via the mediator oligonucleotide, and is detected in the Verigene® system.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Prodesse Multiplex RT-PCR ProFlu™ Plus Assay

2. Predicate K number(s): K073029

1) Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Verigene® Respiratory Virus NucleicAcid Test</td><td colspan="1" rowspan="1">Prodesse ProFlu+ Assay</td></tr><tr><td colspan="1" rowspan="1">Intended use</td><td colspan="1" rowspan="1">The Verigene® Respiratory VirusNucleic Acid Test is a qualitativemultiplex in vitro diagnostic test forthe detection and identification ofInfluenza A Virus, Influenza B Virus,and Respiratory Syncytial Virus (RSV)nucleic acids purified fromnasopharyngeal swab specimensobtained from patients symptomaticfor viral upper respiratory infection.</td><td colspan="1" rowspan="1">The ProFlu+ Assay is a multiplex RealTime RT-PCR in vitro diagnostic testfor the rapid and qualitative detectionand discrimination of Influenza AVirus, Influenza B Virus, andRespiratory Syncytial Virus (RSV)nucleic acids isolated and purifiedfrom nasopharyngeal (NP) swabspecimens obtained fromsymptomatic patients.</td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Influenza AInfluenza BRSV</td><td colspan="1" rowspan="1">Influenza AInfluenza BRSV</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">SamplePreparation</td><td colspan="1" rowspan="1">Automated extraction of nucleic acids</td><td colspan="1" rowspan="1">Automated extraction of nucleicacids</td></tr><tr><td colspan="1" rowspan="1">AmplificationMethod</td><td colspan="1" rowspan="1">Multiplex RT-PCR; M-MLV ReverseTranscriptase</td><td colspan="1" rowspan="1">Multiplex RT-PCR; M-MLVReverse Transcriptase</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">Positive or negative qualitative results</td><td colspan="1" rowspan="1">Positive or negative qualitativeresults</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Verigene® Respiratory Virus NucleicAcid Test</td><td rowspan=1 colspan=1>Prodesse ProFlu+ Assay</td></tr><tr><td rowspan=1 colspan=1>Test Principle</td><td rowspan=1 colspan=1>Gold nanoparticle probe-basedchemistry</td><td rowspan=1 colspan=1>TaqMan chemistry</td></tr><tr><td rowspan=1 colspan=1>AmplificationEnzyme</td><td rowspan=1 colspan=1>Tfi polymerase</td><td rowspan=1 colspan=1>Taq polymerase</td></tr><tr><td rowspan=1 colspan=1>Test Format</td><td rowspan=1 colspan=1>Microarray: multiplex capabilityallows for greater coverage of viralstrain variants</td><td rowspan=1 colspan=1>Real-time PCR format: limitedmultiplex capability</td></tr><tr><td rowspan=1 colspan=1>ContaminationControl</td><td rowspan=1 colspan=1>Heat-labile Uracil DNA Glycosylase(UDG) to limit ampliconcontamination</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>DetectionMethod</td><td rowspan=1 colspan=1>Single-image sensor; nanoparticlesilluminated using a fixed-wavelengthlight source</td><td rowspan=1 colspan=1>Fluorescence-based detection</td></tr><tr><td rowspan=1 colspan=1>Qualitycontrol</td><td rowspan=1 colspan=1>Two internal procedural controls:Process Control —and InhibitionControl</td><td rowspan=1 colspan=1>Single internal procedural control</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

• Guidance on In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path - http://www.fda.gov/cdrh/oivd/guidance/1594.pdf. Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses -http://www.fda.gov/cdrh/oivd/guidance/1638.pdf

# L. Test Principle:

The Verigene® Respiratory Virus Nucleic Acid Test (VRNAT) includes a set of target-specific oligonucleotides immobilized on the Test Substrate (microarray); target-specific mediator oligonucleotides, and Universal Gold Nanoparticle Probe for use in multiplex RT-PCR amplification and hybridization assay on the Verigene® System instruments.

Summary of Steps in the VRNAT Assay:

Nucleic acids are isolated from clinical specimens on an automated NucliSENS easyMAG System; the targeted regions of viral RNA are transcribed into complimentary DNA and amplified in a multiplex polymerase chain reaction (PCR). The amplicons are mixed with a sample buffer containing target-specific mediator oligonucleotides and applied to the Test Substrate on a microarray chip in a disposable test cartridge. The cartridge is inserted into the Verigene Processor. The mediator oligonucleotides are chimeras with one part complementary to the target and another part complementary to the Universal Gold Nanoparticle Probe. The Probe mix is added to form the capture-target-mediator-Gold Nanoparticle Probe complex. The signal is amplified by deposition of elemental silver at the test sites to generate nanoscopic goldsilver aggregates which are illuminated and the light scatter is measured. The test results are analyzed in the Verigene Reader.

# Interpretation of Results:

The Verigene System software automatically determines the specimen results for influenza A virus, influenza B virus, and RSV. The test is designed to provide a “Detected” or “Not Detected” decision for each of the three viruses evaluated in each test sample. A valid ‘call’ is made only when both inhibition control and process control are ‘Detected’ during analysis of each test signifying that the extraction and target amplification processes performed correctly. Failure of the system, cartridge or sample to produce a test result of “Detected” or “Not Detected” for one or more of the targeted viral agents can produce a “No call” result. The descriptions of possible “No call” results with the recommended recourse to be taken can be found in the table below:

<table><tr><td rowspan=1 colspan=1>ResultFailure</td><td rowspan=1 colspan=1>Cause</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>No Call -POS CTRL</td><td rowspan=1 colspan=1>Detection criteria failureof the internal controlswhich guide decisionsregarding the validity ofthe test process.</td><td rowspan=1 colspan=1>1)Repeat the VRNAT involving the steps (i) samplepreparation, (ii) RT-PCR, and (iii) Verigene Test.2) If the repeat test provides &quot;Detected&quot; or &quot;Not Detected&quot;results for the three targets, the results are accepted.3) If the repeat test provides the same &quot;No Call  Pos Ctrl&quot;decision, the sample is inadequate for testing on theVRNAT. A fresh specimen should be collected and tested.</td></tr><tr><td rowspan=1 colspan=1>No Call -IND ZONE</td><td rowspan=1 colspan=1>An inability of the test tomake a decisive callbecause signal intensitiesfor a specific agent fallinto an indeterminatezone close to the signalcut-off limits.</td><td rowspan=1 colspan=1>1  Repeat the VRNAT using leftover amplified DNA sample.2)  If the repeat test also provides a &quot;No call  IND ZONE&quot;result, repeat the VRNAT steps (i) sample preparation, (ii)RT-PCR, and (iii) Verigene Test.3)  If this repeat test provides the same &quot;No Call&quot; decisionusing the 2nd amplified DNA sample, the sample isinadequate for testing on the VRNAT. A fresh specimenshould be collected and tested.</td></tr><tr><td rowspan=1 colspan=1>No Call -VARIATION</td><td rowspan=1 colspan=1>An inability to obtain thetest result because of highvariability in the target-specific signals.</td><td rowspan=1 colspan=1>Repeat the Verigene test with amplified DNA. Results areaccepted if the repeat test provides &quot;Detected&quot; or &quot;NotDetected&quot; results for the three targets.</td></tr><tr><td rowspan=1 colspan=1>No Call - NoGRIDNo Call -BKGD</td><td rowspan=1 colspan=1>Verigene System or theTest Cartridge failure.</td><td rowspan=1 colspan=1>Repeat the Verigene test with amplified DNA. Results areaccepted if the repeat test provides &quot;Detected&quot; or &quot;NotDetected&quot; results for the three targets.</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The Verigene® Respiratory Virus Nucleic Acid Test reproducibility study was performed at each of three sites. Eight unique samples were created by diluting known concentrations of viral particles with Viral Transport Medium. The following strains representing Influenza A, Influenza B, RSV A, and RSV B were used to prepare the samples: Influenza A/Wisconsin/67/2005; Influenza B/ Florida/04/2006; RSV A Strain Long; RSV B Strain B-1 Wild Type (B WV/14617/85). Each strain was represented at 3 distinct concentrations: high negative, low positive, and moderate positive.

<table><tr><td rowspan=1 colspan=1>UniqueSamples</td><td rowspan=1 colspan=1>Viral Strains and Levels</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A - High Negative; Influenza B - High Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>RSV A - High Negative; RSV B - High Negative</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Influenza A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>RSV A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>RSV B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Influenza A - Moderate Positive; RSV A - Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Influenza B - Moderate Positive; RSV B - Moderate Positive</td></tr></table>

At Site 1, the reproducibility study was part of a larger precision study (see below) which used the same set of 8 samples and was performed over 12 non-consecutive days. On each test day, two operators performed in duplicate the VRNAT, involving sample isolation, target amplification, and the Verigene Test. In the reproducibility study performed at sites 2 and 3, the samples were analyzed on the Verigene System in triplicate daily by 2 operators on each of five non-consecutive days after undergoing sample isolation and target amplification.

In the ‘Reproducibility and Precision’ studies, the moderate positive samples served as positive controls since they were expected to provide positive decisions $100 \%$ of the time. The same samples also served as negative controls; ‘Not Detected’ decisions were expected at Influenza B and RSV B in the moderate positive samples containing Influenza A and RSV A and conversely ‘Not Detected’ decisions were expected at Influenza A and RSV A in the moderate positive samples containing Influenza B and RSV B. The positive and negative controls provided expected decisions during the studies.

Reproducibility Study Summary for the Verigene® Respiratory Virus Nucleic Acid Test.   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=3>Total</td></tr><tr><td rowspan=1 colspan=1>Agreewithexpectedresults</td><td rowspan=1 colspan=1>MeanRatio</td><td rowspan=1 colspan=1>RatioCV, %</td><td rowspan=1 colspan=1>Agree withexp.results</td><td rowspan=1 colspan=1>MeanRati</td><td rowspan=1 colspan=1>RatioCV, %</td><td rowspan=1 colspan=1>Agree withexp.results</td><td rowspan=1 colspan=1>MeanRatio</td><td rowspan=1 colspan=1>RatioCV, %</td><td rowspan=1 colspan=1>Agree withexp.results</td><td rowspan=1 colspan=1>%Agree withexp.result</td><td rowspan=1 colspan=1>95%ScoreCI</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNeg(0.001TCID50/mL)</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>75/78</td><td rowspan=1 colspan=1>96.20%</td><td rowspan=1 colspan=1>89.3%-98.7%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPos(10 TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.9994</td><td rowspan=1 colspan=1>0.11%</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>0.9972</td><td rowspan=1 colspan=1>0.09%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9981</td><td rowspan=1 colspan=1>0.07%</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.70%</td><td rowspan=1 colspan=1>93.1%-99.8%</td></tr><tr><td rowspan=1 colspan=1>Flu A ModPos(50TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.9986</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.997</td><td rowspan=1 colspan=1>0.17%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9977</td><td rowspan=1 colspan=1>0.13%</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNeg(0.03TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr><tr><td rowspan=1 colspan=1>FluB LowPos(60TCID50/mL)</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>0.9995</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9985</td><td rowspan=1 colspan=1>0.02%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9978</td><td rowspan=1 colspan=1>0.31%</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.70%</td><td rowspan=1 colspan=1>93.1%-99.8%</td></tr><tr><td rowspan=1 colspan=1>Flu B ModPosi(600TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.9992</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9986</td><td rowspan=1 colspan=1>0.02%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.999</td><td rowspan=1 colspan=1>0.07%</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-00%</td></tr><tr><td rowspan=1 colspan=1>RSV AHigh Neg(0.01TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>97.40%</td><td rowspan=1 colspan=1>91.1%-99.3%</td></tr><tr><td rowspan=1 colspan=1>RSV ALow Pos(50TCID50/mL)</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>0.9988</td><td rowspan=1 colspan=1>0.19%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9968</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9985</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.70%</td><td rowspan=1 colspan=1>93.1%-99.8%</td></tr><tr><td rowspan=1 colspan=1>RSV AMod Pos(250TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.9948</td><td rowspan=1 colspan=1>1.44%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9882</td><td rowspan=1 colspan=1>1.46%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9885</td><td rowspan=1 colspan=1>1.88%</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr><tr><td rowspan=1 colspan=1>RSV BHih Neg(0.005)TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>74/78</td><td rowspan=1 colspan=1>94.90%</td><td rowspan=1 colspan=1>87.5%-98.0%</td></tr><tr><td rowspan=1 colspan=1>RSV BLow Pos(10TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.999</td><td rowspan=1 colspan=1>0.18%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9985</td><td rowspan=1 colspan=1>0.02%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9987</td><td rowspan=1 colspan=1>0.03%</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr><tr><td rowspan=1 colspan=1>RSV BMod Pos(100TCID50/mL)</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>0.9989</td><td rowspan=1 colspan=1>0.32%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9984</td><td rowspan=1 colspan=1>0.05%</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>0.9992</td><td rowspan=1 colspan=1>0.05%</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3%-100%</td></tr></table>

# b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The Verigene System quality control regimen rests on the foundation of a system design that reduces the probability for errors and includes a unit-use disposable Test Cartridge containing all the reagents needed to run a single test:

• A series of automated on-line quality measurements that monitor instrument functionality, software performance, fluidics, test conditions, reagent integrity, and procedural steps in each assay each time a test is performed.

• A series of automated on-line procedural checks guide the user each time a test is performed.

• Several layers of controls built into the VRNAT ensure that failures at any step within the VRNAT are identified in the end-point image analysis of the Test Cartridge.

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Function</td></tr><tr><td rowspan=1 colspan=1>InhibitionControl</td><td rowspan=1 colspan=1>Double-stranded DNA targetpresent in the primer mix.Amplified with every RT-PCR reaction.</td><td rowspan=1 colspan=1>Controls for PCR inhibitiondue to sample- or process-related inhibitors or due toreagent failures.</td></tr><tr><td rowspan=1 colspan=1>ProcessControl</td><td rowspan=1 colspan=1>MS2 bacteriophage with anintact viral RNA genome.Added to each test sampleincluding external positiveand negative controls.</td><td rowspan=1 colspan=1>Controls for sample isolationstep (or nucleic acidextraction step) and theRT-PCR step.</td></tr><tr><td rowspan=1 colspan=1>ExternalPositiveControl</td><td rowspan=1 colspan=1>Any of the three viral particles(Influenza A, Influenza B, orRSV) to which Process Control(MS2 bacteriophage) is addedprior to sample isolation.</td><td rowspan=1 colspan=1>Serve as external controls forthe extraction, targetamplification, and detectionsteps. Used to verify reagentperformance</td></tr><tr><td rowspan=1 colspan=1>ExternalNegativeControl</td><td rowspan=1 colspan=1>Viral Transport Media towhich the Process Control(MS2 bacteriophage) is addedprior to sample isolation.</td><td rowspan=1 colspan=1>Controls for reagent and/orenvironmentalcontamination.</td></tr></table>

# External Controls

Regardless of the choice of quality control materials, all external quality control requirements and testing should be performed in conformance with local, state, and federal regulations or accreditation organizations as applicable and should follow the user’s laboratory’s standard quality control procedures. External Controls can be prepared by using influenza A, influenza B, and RSV viral particles diluted in negative clinical matrix or Universal Transport Media. It is recommended that the user refer to CLSI document C24-A2, Statistical Quality Control for Quantitative Measurements: Principles and Definitions: [Approved Guideline-Second Edition] or other published guidelines for general quality

control recommendations. For further guidance on appropriate quality control practices, refer to 42CFR 493.1202(c).

# d. Detection limit:

The analytical sensitivity was demonstrated by determining the Limit of Detection or (LoD) of the VRNAT using strains with established titers for the following viruses: Influenza A, Influenza B, RSV A, and RSV B. Each virus stock was serially diluted into a sample matrix (Universal Transport Media, Copan). The dilution series was taken through the VRNAT process and tested in triplicate at each dilution level for each virus. The limit of detection defined as the lowest concentration at which $2 9 5 \%$ of the replicates tested positive, was assessed based on the performance of the samples within each dilution series as judged by the Verigene calls. This LoD was confirmed by testing an additional 20 replicates for each strain in order to demonstrate that the virus was detected $2 9 5 \%$ of the time.

Limit of Detection Summary   

<table><tr><td rowspan=1 colspan=1>Limits of Detection</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Influenza A strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Wisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Caledonia/20/99 (H1N1)</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Port Chalmers/1/73 (H3N2)</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/04/2006</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Lee/40</td><td rowspan=1 colspan=1>0.01 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Hong Kong/5/72</td><td rowspan=1 colspan=1>0.05 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A Strain A2</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A Strain Long</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B strains</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B Strain B-1 Wild Type (B WV/14617/85)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain Wash/18537/62</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain 9320</td><td rowspan=1 colspan=1>0.05 TCID50/mL</td></tr></table>

# e. Analytical specificity:

Analytical specificity of Verigene® Respiratory Virus Nucleic Acid Test was evaluated with respect to 1) reactivity (inclusivity) with a number of geographically diverse influenza virus and RSV strains and 2) potential cross-reactivity with pathogens associated with respiratory tract infections.

# 1) Reactivity/Inclusivity

The analytical reactivity was evaluated against strains of Influenza A, Influenza B, RSV A, and RSV B that are representative of temporal and geographical diversity. Known concentrations of the viral strains grown in culture were diluted in sample matrix (Universal Transport Media, Copan) to achieve the titers listed in the table below. A total of 14 strains for Influenza A, 10 strains for influenza B, and five (5) RSV strains were tested in triplicate at $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All viral cultures were detected; the VRNAT analytical reactivity demonstrated $100 \%$ concordance with expected results for all primer and probe sets included in the device.

<table><tr><td rowspan=1 colspan=1>Analytical Reactivity</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>InfluenzaA</td><td rowspan=1 colspan=1>InfluenzaB</td><td rowspan=1 colspan=1>RSV</td></tr><tr><td rowspan=1 colspan=1>Influenza A strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Canada/578/04 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Caledonia/20/99 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Hong Kong/29/2006 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Wisconsin/67/05 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/PR/8/34 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Aichi/68 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/WS/33 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/Port Chalmers/1/73 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/ Brisbane/59/ 2007 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A/Brisbane/10/2007 (H3N2)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A1/Denver/1/57 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A1/FM/1/47 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/New Jersey/8/76 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A/NWS/33 (H1N1)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B/Yamanashi/166/98</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Panama/45/90</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/02/2006</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Florida/04/2006</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/ Lee/40</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Egypt/2040/04</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/GL/1739/54</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Hong Kong/5/72</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Maryland/1/59</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Taiwan/2/62</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>RSV A strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A Strain A2</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV A Strain Long</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B strains</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B Strain B-1 Wild Type (B WV/14617/85)</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain Wash/18537/62</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain 9320</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr></table>

# Cross-Reactivity Evaluation

Analytical specificity (cross-reactivity) studies were performed to assess potential crossreactivity of the VRNAT with common respiratory pathogens and other microorganisms commonly present in specimens collected from the nasopharynx. A total of 38 organisms of interest were identified as respiratory pathogens with which the majority of the population may be infected. These included 15 viruses and 23 bacterial strains which were propagated and the titers were determined. To assess cross-reactivity, each organism was diluted in a sample matrix (Universal Transport Medium, Copan) to the concentration listed in the table below and taken through all the steps of the VRNAT protocol.

Cross-reactivity Test Results with Common Human Respiratory Pathogens   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">InfluenzaA</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">pfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 1</td><td colspan="1" rowspan="1">Adenoid 71</td><td colspan="1" rowspan="1">3.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 7</td><td colspan="1" rowspan="1">Gomen</td><td colspan="1" rowspan="1">3.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus (OC43)</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus (229E)</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">68-1</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus, H. coxsackievirusB4</td><td colspan="1" rowspan="1">J.V.B.</td><td colspan="1" rowspan="1">1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.2x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">6.2x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">2x10}$</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">C243</td><td colspan="1" rowspan="1">1.1x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">Enders</td><td colspan="1" rowspan="1">1.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 4a</td><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus Type 1A</td><td colspan="1" rowspan="1">2060</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">B-1</td><td colspan="1" rowspan="1">5.6x104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">cYanmaGYellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">InfluenzaA</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Acinetobacter baumannii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">CM-1</td><td colspan="1" rowspan="1">1.6x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumpseudodiphtheriticum</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8.2x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.1x107</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.2x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1x10}$</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6x108</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.9x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.1x106</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.3x106</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8.3x106</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis,attenuated*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.6x106</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

None of the organisms tested interfered with the internal controls. For all the organisms tested in the VRNAT for cross-reactivity ‘Not Detected’ calls were made by the Verigene Reader for each of the three viruses in the VRNAT, Influenza A, Influenza B, and RSV. No cross-reactivity was observed with the common respiratory pathogens and organisms infecting the majority of the population.

# f. Assay cut-off:

The Verigene Respiratory Virus Nucleic Acid Test (VRNAT) uses a microarray-based platform in which the viruses and the inhibition control (PC1) and process control (PC2) are represented by recognition elements on the Test Substrate. The recognition elements are virus-specific oligonucleotides that bind to the amplified viral targets, which in turn bind to gold nanoparticle probes via additional recognition elements. A gold nanoparticle probespecific signal enhancement reaction deposits silver at the virus-specific spots. The scatter from the spots is detected by the Verigene Reader and registered as signal intensity. In addition to the above recognition elements, the Test Substrate has spots specific to positive control (PC) and negative control (NC). Three conditions were identified that together served as a single set of clinical cutoff criteria.

Condition 1: Noise Threshold

Condition 2: Normalized ‘Ratio to Negative Control’ (Ratio-to-NC) – intensity at the virusspecific recognition element normalized against the intensity values at the negative control elements.

Condition 3: Normalized ‘Ratio to Positive Control’ (Ratio-to-PC) – intensity at the virusspecific recognition element normalized against the intensity values at the positive control elements.

The Noise Threshold was determined empirically to be equal to Negative Control $^ { + 2 }$ SD. The cut-offs for the normalized ratios, ‘Ratio-to-NC’ and the ‘Ratio-to-PC’, were determined by using ROC curves. For a positive ‘Detected’ decision the following criteria apply:

Condition 1: Signal intensity is above the noise threshold   
Condition 2: Ratio-to-NC is above 0.85   
Condition 3: Ratio-to-PC is above -0.4   
If any one of these criteria is not met, a negative ‘Not Detected’ decision is provided. If the Ratio-to-PC for any one of the three viruses falls between -0.4 and -0.5, an equivocal (i.e., No Call – IND ZONE) decision is provided. Criteria set for each of the three conditions are

required to be met for a ‘Detected’ call.

# VRNAT Decision Process

For a result, the decision tree examines the presence of both PC1 and PC2 initially (see Schematic). Both PC1 and PC2 signal intensities have to meet the detection criteria before the analysis is allowed to proceed toward establishing the presence or absence (i.e., “Detected” or “Not Detected”) of the individual viruses.

![](images/5a59b7f83602d7f7658d015c99d91178bab6523e25dd277e16a4ca3e41073762.jpg)

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable, performance of the assay was evaluated in comparison to the gold standard/reference method, viral culture followed by DFA and/or viral culture followed by sequencing

b. Matrix comparison:

Not applicable

3. Clinical studies:

Performance characteristics of the Verigene® Respiratory Virus Nucleic Acid Test were established during a prospective study during the 2007-2008 respiratory virus season. A total of 720 nasopharyngeal swab specimens were prospectively collected for routine influenza or RSV testing by DFA/culture methods. The residual specimens were frozen and later tested at three clinical sites, 240 at each site, using the Verigene® Respiratory Virus Nucleic Acid Test. The patient population was distributed by age as presented in the table below:

<table><tr><td rowspan=1 colspan=1>Patient Population byAge</td><td rowspan=1 colspan=1>Number ofSubjects</td></tr><tr><td rowspan=1 colspan=1>0 - 1</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>1-5</td><td rowspan=1 colspan=1>229</td></tr><tr><td rowspan=1 colspan=1>5-20</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=1>20 - 65</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>720</td></tr></table>

The VRNAT performance was compared to a culture-based reference method followed by direct fluorescent antibody (DFA) identification of all culture positive samples. Samples which demonstrated discordant results between the VRNAT and the reference method were further analyzed by using bi-directional sequencing at an independent reference laboratory.

A total of 21 samples $( 2 . 9 \% )$ generated a “No Call” result; ten (10) of the samples $( 1 . 4 \% )$ gave a ‘No Call – Indeterminate Zone’ result indicating an equivocal call. Another eleven (11) samples $( 1 . 5 \% )$ gave a ‘No Call – Pos Ctrl’ call indicating an inability to detect an internal control. All the samples with an initial ‘No Call’ result were re-tested and resolved successfully by following the recommendations in the ‘Results Interpretation’ section (see above).

# a and b. Clinical Sensitivity and Specificity:

# Influenza A Results

<table><tr><td rowspan=2 colspan=2>All SitesInfluenza A</td><td rowspan=1 colspan=2>DFA/Viral Culture</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>VerigeneSystem</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>59 a</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>Sensitivity 99.2%(95%CI=95.5% - 99.9%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1 b</td><td rowspan=1 colspan=1>538</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>Specificity 90.1%(95%CI=87.5% - 92.3%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>597</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1></td></tr></table>

a. 58 samples were positive by sequencing. 1 sample was negative by sequencing. b. 1 sample was negative by sequencing

# Influenza B Results

<table><tr><td rowspan=2 colspan=2>All SitesInfluenza B</td><td rowspan=1 colspan=2>DFA/Viral Culture</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>VerigeneSystem</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10 a</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Sensitivity 96.8%(95%CI=83.5% - 99.4%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1 b</td><td rowspan=1 colspan=1>679</td><td rowspan=1 colspan=1>680</td><td rowspan=1 colspan=1>Specificity 98.5%(95%CI=97.3% - 99.2%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>689</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1></td></tr></table>

a. 4 samples were positive by sequencing. 6 samples were negative by sequencing b. 1 sample was negative by sequencing

# RSV Results

<table><tr><td rowspan=2 colspan=2>All SitesRSV</td><td rowspan=1 colspan=2>DFA/Viral Culture</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>VerigeneSystem</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>44 a</td><td rowspan=1 colspan=1>57 b</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>Sensitivity 89.8%(95%CI=78.2% - 95.6%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>Specificity 91.5%(95%CI=89.2% - 93.4%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>671</td><td rowspan=1 colspan=1>720</td><td rowspan=1 colspan=1></td></tr></table>

a. All 44 samples were positive by sequencing. b. 46 samples were positive by sequencing. 11 samples were negative by sequencing. c. 4 samples were positive by sequencing. 1 sample was negative by sequencing.

# Dual Infections

Of the 720 samples included in the clinical study, 12 samples were positive for 2 infections by the VRNAT, translating to $1 . 7 \%$ of the samples tested. Comparable RT-PCR—based real-time detection methods have reported a significantly lower percentage for dual infections $( \sim 0 . 1 \% )$ . Two inter-related factors may contribute to the higher percentage of the dual infections observed in the VRNAT. Competitive inhibition is often observed in dual infections, especially when one infective agent is at a much lower initial concentration resulting thereby in amplicons below the detection limits of fluorescence-based methods.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

During the 2005-2006 flu season, $12 \%$ of 148,636 samples tested for influenza were positive for either Influenza A or Influenza B [based on data from laboratories in the US which collaborated with World Health Organization (WHO) and National Respiratory and Enteric Virus Surveillance System (NREVSS)]. During the 2006-2007 and 2007-2008 flu seasons, the prevalence was $1 3 \%$ of 179,268 samples tested and $1 8 \%$ of 225,329 samples tested, respectively.

According to data reported to the National Respiratory and Enteric Virus Surveillance System (NREVSS), the prevalence of RSV was $1 6 \%$ in the 120,503 samples tested for RSV during the 2005-2006 season and $1 7 \%$ of the 126,617 samples tested during the 2006-2007 season.

In the Verigene Respiratory Virus Nucleic Acid Test multi-site study, which analyzed 720 samples collected during the $2 0 0 7 - 2 0 0 8$ flu season, the prevalence of Influenza A observed by culture and DFA was $1 7 . 1 \%$ , of Influenza B was $4 . 3 \%$ , and of RSV $6 . 8 \%$ . By VRNAT, of the set of 720 samples, $2 5 . 0 \%$ were positive for Influenza A, $4 . 0 \%$ were positive for Influenza B, and $1 4 . 0 \%$ were positive for RSV. While no dual infections were detected by culture and DFA, the VRNAT detected 12 dual infections or $1 . 7 \%$ (see “Dual Infections” in the Performance Characteristics section). Because the incidence of a triple infection of Influenza A, Influenza B, and RSV is low, it is recommended that the samples undergo repeat testing if nucleic acids from all three analytes are detected in a single sample.

<table><tr><td rowspan=1 colspan=1>Virus Infection</td><td rowspan=1 colspan=1>Identified byDFA/Culture(% samples tested)</td><td rowspan=1 colspan=1>Identified byVRNAT(% samples tested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>25.0</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>5.6</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>Dual infections</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>1.7</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.