# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K090073 B. Purpose for Submission:

This is a new 510(k) application for the Diagnostic Hybrids, Inc. device, $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit, which is intended for the qualitative detection and identification of human metapneumovirus (hMPV) in nasal and nasopharyngeal swabs and aspirates or cell culture by detecting hMPV antigens by immunofluorescence using a blend of three monoclonal antibodies (MAbs), from patients with signs and symptoms of acute respiratory infection.

# C. Measurand:

Viral antigens of the nucleoprotein of the human metapneumovirus species

# D. Type of Test:

Direct Fluorescence Antibody (DFA) test using direct specimens or cell culture specimens

E. Applicant: Diagnostic Hybrids Incorporated

F. Proprietary and Established Names: $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit

# G. Regulatory Information:

1. Regulation section: 866.3980   
2. Classification: Class II   
3. Product code: OMG   
4. Panel: Microbiology (83)   
Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Diagnostic Hybrids, Inc. device, $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit, is intended for the qualitative detection and identification of human metapneumovirus (hMPV) in nasal and nasopharyngeal swabs and aspirates/washes or cell culture. The assay detects hMPV antigens by immunofluorescence using a blend of three monoclonal antibodies (MAbs), from patients with signs and symptoms of acute respiratory infection. This assay detects but is not intended to differentiate the four recognized genetic sub-lineages of hMPV.

Negative results do not preclude hMPV infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. It is recommended that specimens found to be negative after examination of the direct specimen results be confirmed by an FDA cleared hMPV molecular assay.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

Fluorescence microscope with the appropriate filter combination for FITC (excitation peak $= 4 9 0 \ \mathrm { n m }$ , emission peak $= 5 2 0 \mathrm { n m }$ ); magnification 200-400X.

# I. Device Description:

The device, ${ \mathrm { D } } ^ { 3 }$ DFA Metapneumovirus Identification Kit, consists of the following components:

1. Metapneumovirus DFA Reagent (5-mL): One dropper bottle containing a blend of fluorescein-labeled murine monoclonal antibodies directed against hMPV. The buffered, stabilized, aqueous solution contains Evans Blue as a counter-stain and $0 . 1 \%$ sodium azide as a preservative.

2. hMPV Antigen Control Slides (5 slides): Five individually packaged control slides, each with a well containing cell culture-derived MPV positive cells and a well containing cell culture-derived negative cells. Each slide is intended to be stained only one time. Control material has been treated to be non-infectious; however normal laboratory precautions are required when the material is handled.

3. 40X PBS Concentrate $( 2 5 \mathrm { - m L } )$ ): One bottle containing a 40X concentrate consisting of $4 \%$ sodium azide ( $0 . 1 \%$ sodium azide after dilution to 1X using de-mineralized water) in PBS.

4. Mounting Fluid (7-mL): One dropper bottle containing an aqueous, buffer stabilized solution of glycerol and $0 . 1 \%$ sodium azide.

The D3 DFA Metapneumovirus Identification Kit (D3 MPV Kit) uses a blend of three hMPV antigen-specific murine MAbs that are directly labeled with fluorescein for detection of hMPV. The reagent detects but does not differentiate between the four recognized subtypes of hMPV (subtypes A1, A2, B1, and B2). The MAbs all recognize a hMPV protein, approximately 46 kilo Daltons in size, which corresponds with the size of hMPV nucleoprotein, but do not compete with one another for binding sites.

An overview of the procedure is as follows:

The cells to be tested, derived from a clinical specimen or cell culture, are placed onto a glass slide and allowed to air dry. The cells are fixed in acetone. The MPV DFA Reagent is added to the cells which are then incubated for 15 to 30 minutes at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ in a humidified chamber or humidified incubator. The stained cells are then washed with the diluted phosphate buffered saline (PBS), a drop of the supplied Mounting Fluid is added and a cover slip is placed on the prepared cells. The cells are examined using a fluorescence microscope. The hMPV infected cells will fluoresce apple-green. Uninfected cells will contain no fluorescence but will be stained red by the Evans Blue counter-stain.

Materials Provided

1. Metapneumovirus DFA Reagent (5-mL): One dropper bottle containing a blend of fluorescein-labeled murine monoclonal antibodies directed against hMPV antigens. The buffered, stabilized, aqueous solution contains Evans Blue as a counter-stain and $0 . 1 \%$ sodium azide as a preservative.

2. hMPV Antigen Control Slides (5 slides): Five individually packaged control slides, each with two wells, one well containing cell culture-derived hMPV positive cells and one well containing cell culture-derived negative cells. Each slide is intended for staining only one time. Control material has been treated to be non-infectious; however, biohazard safety precautions for handling infectious materials are required.

3. 40X PBS Concentrate $( 2 5 \mathrm { - } \mathrm { m L } )$ ): One bottle of 40X PBS concentrate containing $4 \%$ sodium azide $0 . 1 \%$ sodium azide after dilution to 1X using de-mineralized water).

4. Mounting Fluid (7-mL): One dropper bottle containing an aqueous buffer-stabilized solution of glycerol with $0 . 1 \%$ sodium azide.

Materials Required But Not Provided

1. Fluorescence microscope with the appropriate filter combination for FITC (excitation peak $= 4 9 0 \ \mathrm { n m }$ , emission peak $= 5 2 0 \mathrm { n m }$ ); magnification 200-400X. 2. Cell culture for hMPV isolation. Suggested cell lines include LLC-MK2, HEp-2, A549 cells; R-Mix™ and R-Mix Too™ MixedCells $\mathbf { T M _ { \lambda \lambda \mathbf { a } } }$ ; and primary Rhesus monkey kidney cells.

3. Live control viruses for positive culture controls: Known strains of hMPV for use in monitoring cell culture and staining procedures.   
4. Cover slips $( 2 2 \mathrm { ~ x ~ } 5 0 \mathrm { - m m } )$ ) for antigen control and for specimen slides.   
5. Universal Transport Medium.   
6. R-Mix™ Refeed Medium (for use with R-Mix™ and R-Mix Too™ MixedCells™) or other standard refeed medium.   
7. Reagent grade acetone $( > 9 9 \%$ pure) chilled at $2 ^ { \circ } \mathrm { C }$ to $8 \mathrm { { } ^ { \circ } C }$ for fixing direct specimen slides, shell-vials, and cultured cell preparations.   
8. Sterile graduated pipettes: $1 0 { \cdot } \mathrm { m L }$ , $5 { \mathrm { - m L } }$ , and $1 { \mathrm { - m L } }$ .   
9. Sterile Pasteur pipettes or other “transfer”-type pipettes.   
10. Fine-tipped forceps.   
11. $2 0 0 { \cdot } \mathrm { m L }$ wash bottle.   
12. Bent-tip teasing needle (for removal of cover slip from a shell-vial): Fashion the teasing needle by bending the tip of a syringe needle or similar object (e.g., mycology teasing needle) against a bench top or with a pair of forceps taking care to avoid injury.   
13. Sodium hypochlorite solution (1:10 dilution of household bleach).   
14. Humid chamber (e.g., covered Petri dish with a damp paper towel placed in the bottom).   
15. Glass microscope slides.   
16. Acetone-cleaned glass microscope slides.   
17. Sterile, nylon flocked swab   
18. Incubator, $3 5 ^ { \circ }$ to $3 7 ^ { \circ } \mathrm { C }$ ( $\mathrm { C O } _ { 2 }$ or non- $C 0 _ { 2 }$ , depending on the cell culture format used).   
19. Centrifuge with free swinging bucket rotor.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Pro hMPV $^ +$ Assay, Prodesse, Inc.

2. Predicate k number(s): K082688

3. Comparison with predicates:

The intended use of the $\mathrm { D } ^ { 3 } \mathrm { M P V }$ Kit is similar to the predicate device, Pro hMPV+ Assay. Both devices are intended to detect the presence of hMPV from individuals exhibiting signs and symptoms of acute respiratory infection. Characteristics of the D3 MPV Kit are compared to those of the predicate device, Pro hMPV $+$ Assay, in the table below:

<table><tr><td rowspan=1 colspan=2>Technological Characteristics Comparison of Devices</td></tr><tr><td rowspan=1 colspan=1>D³ DFA Metapneumovirus Identification Kit (Subject)</td><td rowspan=1 colspan=1>Pro hMPV+ Assay (Predicate)</td></tr><tr><td rowspan=1 colspan=1>Target:</td><td></td></tr><tr><td rowspan=1 colspan=1>Searches of the National Center forBiotechnology Information (NCBI) databases yieldedpresumptive evidence that the target for each of the 3 MAbclones is the MPV nucleoprotein. Nine proteins are known tobe encoded in the hMPV genome. Of the nine proteins, onlythe nucleoprotein is of a size equivalent to the 46 kDa sizenoted on the Western Blot of the 3 MAb clones.</td><td rowspan=1 colspan=1>The Pro hMPV+ Supermix contains oligonucleotide primersand target-specific oligonucleotide probes. The primers arecomplementary to highly conserved regions of geneticsequences of the Nucleocapsid of hMPV.</td></tr><tr><td rowspan=1 colspan=2>Specimen:</td></tr><tr><td rowspan=1 colspan=1>Nasal and nasopharyngeal swabs and aspirates or cellculture.</td><td rowspan=1 colspan=1>Nasopharyngeal swab specimens</td></tr><tr><td rowspan=1 colspan=2>Detection Methods:</td></tr><tr><td rowspan=2 colspan=1>The assay detects specific hMPV viral antigens byimmunofluorescence using monoclonal antibodies (MAbs).The cells to be tested, derived from a clinical specimen orcell culture, are placed onto a glass slide and allowed to airdry. The cells are fixed in acetone. The hMPV DFA reagentis added to the cells which are then incubated for 15 to30 minutes at 35°C to 37°C in a humidified chamber orhumidified incubator. The stained cells are then washed withthe diluted phosphate buffered saline (PBS), a drop of thesupplied Mounting Fluid is added and a cover slip is placedon the prepared cells. The cells are examined using afluorescence microscope. The hMPV infected cells willfluoresce apple-green. Uninfected cells will contain nofluorescence but will be stained red by the Evans Bluecounter-stain.</td><td rowspan=2 colspan=1>Reverse transcription of the hMPV RNA in the sample intocomplementary DNA (cDNA) and subsequent amplificationof DNA is performed in a Cepheid SmartCycler IIinstrument. In this process, the probe anneals specifically tothe template followed by primer extension and amplification.The Pro hMPV+ Assay is based on TaqMan chemistry,which utilizes the 5&#x27; - 3&#x27; exonuclease activity of the Taqpolymerase to cleave the probe thus separating the reporterdye from the quencher. This generates an increase influorescent signal upon excitation from a light source. Witheach cycle, additional reporter dye molecules are cleavedfrom their respective probes, further increasing fluorescentsignal. The amount of fluorescence at any given cycle isdependent on the amount of amplification products presentat that time. Fluorescent intensity is monitored during eachPCR cycle by the SmartCycler II instrument.</td></tr><tr><td rowspan=1 colspan=1>polym</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Special controls guidance documents will be promulgated   
Guidance on Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses (March 2006) – http://www.fda.gov/cdrh/oivd/g uidance/1596.html   
Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) –   
http://www.fda.gov/cdrh/oivd/guidance/1588.html   
Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Guidance for Industry and FDA Reviewers (March 2007) –   
http://www.fda.gov/cdrh/osb/guidance/1620.html   
Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff – http://www.fda.gov/cdrh/ode/guidance/1567.html   
Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (Feb 2008) – http://www.fda.gov/cdrh/oivd/guidance/1638.pdf

# L. Test Principle:

The $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit $( \mathrm { D } ^ { 3 } \mathrm { M P V }$ Kit) uses a blend of three hMPV antigen-specific murine MAbs that are directly labeled with fluorescein for detection of hMPV antigens. The reagent detects but does not differentiate between the four recognized subtypes of hMPV (subtypes A1, A2, B1, and B2).

The cells to be tested, derived from a clinical specimen or cell culture, are placed onto a glass slide, allowed to air dry and are fixed in acetone. The Metapneumovirus DFA Reagent is added to the cells which are then incubated for 15 to 30 minutes at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ in a humidified chamber or humidified incubator. The stained cells are then washed with the diluted phosphate buffered saline (PBS), a drop of the supplied Mounting Fluid is added and a cover slip is placed on the prepared cells. The cells are examined using a fluorescence microscope. The hMPV infected cells will fluoresce apple-green. Uninfected cells will contain no fluorescence but will be stained red by the Evans Blue counter-stain.

It is recommended that specimens found to contain no fluorescent cells after examination of the direct specimen be confirmed by an FDA cleared hMPV molecular assay.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Assay precision, intra-assay variability and inter assay variability were assessed with a panel of proficiency-level antigen control slides. The reproducibility panel consisted of 5 panel members. Each panel member was a 2-well slide spotted with the same cell preparation in each well. The cell preparations used to construct the slides are the following:

1. Non-infected $\mathrm { L L C } { \mathrm { - M K } _ { 2 } }$ cells   
2. Low level hMPV (A1 strain) grown in LLC- $\mathrm { M K } _ { 2 }$ cells (manufactured to contain between 4 to $10 \%$ infected cells).   
3. Mid level hMPV (A1 strain) grown in LLC- $\mathrm { . M K } _ { 2 }$ cells (manufactured to contain between 20 to $30 \%$ infected cells).   
4. High level hMPV (A1 strain) grown in LLC- $\mathrm { . M K } _ { 2 }$ cells (manufactured to contain between 50 to $7 5 \%$ infected cells).

Each panel was tested daily in two separate runs for 5-days by three different laboratories (30 total runs). The panel members were randomized with different slide identification numbers to act as a “blinded” panel. An hMPV Antigen Control Slide (two-well slide, one well contains cell culture-derived hMPV positive cells and one well contains cell culture-derived negative cells) was stained according to the D3 DFA Metapneumovirus Identification Kit instructions for use with each run. The following results were recorded for both the control slides and the panel slides:

1. Presence or absence of green fluorescence.   
2. Percent of cells exhibiting green fluorescence.

A single lot of D3 MPV Kit was used. A total of 210 data points were included in the reproducibility study data analysis (7 samples and controls/run X 2 runs/day X 5 days X 3 sites $= 2 1 0$ ). The combined data from the three sites demonstrated that the detection of hMPV occurs in a reproducible manner. The presence of hMPV infected cells was reported in $100 \%$ (120/120) of the wells in which infected cells were present. The combined data from the three sites also demonstrated that no hMPV was detected in non-infected cells. The absence of hMPV was reported in $100 \%$ (90/90) of the wells in which infected cells were not present. The total percent agreement for the D3 DFA Metapneumovirus Identification Kit was $100 \%$ (210/210).

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>PanelMember</td><td rowspan=1 colspan=1>hMPV A1LowLevel</td><td rowspan=1 colspan=1>hMPV A1Mid Level</td><td rowspan=1 colspan=1>hMPV A1Highevel</td><td rowspan=1 colspan=1>hMPV A1Negative</td><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>Negative Control</td><td rowspan=2 colspan=1>Total %Agreement</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>20 to 30%infectedcells</td><td rowspan=1 colspan=1>50 to 75%infectedcells</td><td rowspan=1 colspan=1>Non-infectedcells</td><td rowspan=1 colspan=1>50 to 75%infectedcells</td><td rowspan=1 colspan=1>Non-infectedcells</td></tr><tr><td rowspan=1 colspan=1>Site1</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>20/20(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site2</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>20/20(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site3</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>20/20(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwith Expectedresult</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>60/60(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>210/210(100%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>88.4%-100%</td><td rowspan=1 colspan=1>88.4%-100%</td><td rowspan=1 colspan=1>88.4%-100%</td><td rowspan=1 colspan=1>94.0%-100%</td><td rowspan=1 colspan=1>88.4%-100%</td><td rowspan=1 colspan=1>88.4%-100%</td><td rowspan=1 colspan=1>98.3%-100%</td></tr></table>

$b$ . Linearity/assay reportable range: Not applicable c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Development and Characterization of Monoclonal Antibodies (MAbs)

# Hybridoma production and purification

The hybridomas used in the D3 DFA Metapneumovirus Identification Kit were procured from the University of Pavia in Pavia, Italy. The hybridomas were developed as described in the following publication: Rapid Detection of Human Metapneumovirus Strains in Nasopharyngeal Aspirated and Shell Vial Cultures by Monoclonal Antibodies. Percivalle E, Sarasini A, Visai L, Revello MG, Gerna G. Journal of Clinical Microbiology 2005, 43 :3443-3446. Three hybridoma cell lines (designated as clones #4, #23, and $\# 2 8$ ) were received as frozen cell suspensions, then were accessioned for archival and production level stocks. Preparations of monoclonal antibodies from each cell line were initially evaluated by indirect immunofluorescence assay against cell cultures infected with known hMPV. Antibodies were purified by Protein G affinity using fast protein liquid chromatography (FPLC).

# MAbs characterization

A variety of methods (e.g., SDS-PAGE Electrophoresis, ELISA, Western Blot, and available databases, such as NCBI) were used to characterize the subject MAbs in terms of purity, sensitivity, and specificity. MAb purity was checked by SDS-PAGE electrophoresis. A minimum purity of $90 \%$ has been set as the QC acceptance criterion. Purity is typically ${ \sim } 9 5 \%$ . Candidate MAbs Binding Affinities were evaluated by measuring the dissociation constants (Kd) for the MAbs using ELISA. The binding specificities of the candidate MAbs were characterized using Western Blots. All three candidate MAbs all recognized a $4 6 \mathrm { k D a }$ protein present in LLC- $\mathrm { . M K } _ { 2 }$ cell lysates that had been infected by any of the four genetic sunlineages of hMPV. Searches of the National Center for Biotechnology Information (NCBI) databases yielded presumptive evidence that the target for each of the three MAb clones is the hMPV nucleoprotein. Nine proteins are known to be encoded in the hMPV genome. Of the nine hMPV proteins, only the nucleoprotein is of a size equivalent to the $4 6 \mathrm { k D a }$ size noted on the Western blot of the three MAb clones.

Each of the three anti-hMPV MAbs were labeled with fluorescein, using fluorescein isothiocyanate (FITC), with standard procedures. The labeled MAbs exhibited staining patterns on hMPV-infected cells similar to the patterns seen with the unlabeled MAbs. A final blended solution of the three labeled MAbs was formulated to yield optimal fluorescence intensity on hMPV-infected cells, and lowest background. Studies were conducted to determine that the three MAbs do not compete with one another for the same antigen binding sites. The selfcompetitive epitope blocking study demonstrated that reduced fluorescence could be observed when targets are first blocked by the same MAb, unlabeled. The “Cross clone” competitive blocking study was carried out by reacting hMPV infected (hMPV types A1, B1, and B2) R-Mix cells in plates with the three unlabeled MAbs at increasing concentrations, and then counter-staining each set with one of the other two clones (only fluorescein-labeled) at standard concentrations. This study illustrated that fluorescein staining intensity unchanged regardless of increased concentration of unlabeled MAb in all labeled and unlabeled variations.

# MAbs analytical sensitivity performance evaluations on cell culture systems

The final blended solution of the three labeled MAbs was also directly compared to commercially available anti-hMPV MAbs (labeled as ‘research use only’ or ‘analyte specific reagent’) for sensitivity performance on cell culture systems.

Analytical sensitivity of the final blended solution of the three labeled MAbs testing cell culture specimens was evaluated using cell culture system against a non-labeled murine monoclonal antibody (MAb-8) to hMPV strain MPV75-

1998/CAN98-75, which was developed at the CDC by standard methods. (Note: The MAb-8 is a research use only device that has not been cleared by the FDA. However, there is data presented in the literature addressing its specificity and sensitivity to hMPV: Landry, Marie L., Ferguson, David, Cohen, Sandra, Peret, Teresa C. T., Erdman, Dean D., “Detection of Human Metapneumovirus in Clinical Samples by Immunofluorescence Staining of Shell Vial Centrifugation Cultures Prepared from Three Different Cell Lines”, J. Clin. Microbiol. 2005 43: 1950-1952.) Each of the hMPV subtypes was diluted to a level of ${ \sim } 3 5 0$ TCID50, and then serial 2-fold dilutions were prepared from that suspension to a final level of 0.7 TCID50. Each dilution of virus was inoculated $\left( 0 . 5 \mathrm { - m L } \right)$ into 6 shell vials of R-Mix (containing monolayers on cover slips), centrifuged at $7 0 0 \mathrm { x g }$ for 60 minutes and incubated at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ for 48 hours. The final blended solution of the three labeled MAbs and the anti-hMPV MAb comparator (CDC MAb-8 used as IFA) were each used to stain 3 shell vials of each viral dilution. The following table provides a count of fluorescent-staining viral plaques on each cover slip at the indicated viral dilutions:

<table><tr><td rowspan=1 colspan=3>Analytical sensitivity of the final blended solution of the 3 MAbs comparing to CDC MAb-8 using viraldilutions amplified in a cell culture system (Shell vial culture)</td></tr><tr><td rowspan=1 colspan=3>a) hMPV subtype A1 in triplicate shell vials</td></tr><tr><td rowspan=2 colspan=1>∼ TCID50 per vial</td><td rowspan=1 colspan=2>Fluorescent stained plaques/vial</td></tr><tr><td rowspan=1 colspan=1>DHI blended MAbs as DFA</td><td rowspan=1 colspan=1>CDC MAb-8 (indirect)</td></tr><tr><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>2,3,1</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1,0,1</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0,1,0</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>b) hMPV subtype A2 in triplicate shell vials</td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>∼ TCID50 per vial</td><td rowspan=1 colspan=2>Fluorescent stained plaques/vial</td></tr><tr><td rowspan=1 colspan=1>DHI blended MAbs as DFA</td><td rowspan=1 colspan=1>CDC MAb-8 (indirect)</td></tr><tr><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>5,3,3</td><td rowspan=1 colspan=1>0,1,1</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1,3,2</td><td rowspan=1 colspan=1>1,2,0</td></tr><tr><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1,0,1</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>c) hMPV subtype B1 in triplicate shell vials</td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>∼ TCID50 per vial</td><td rowspan=1 colspan=2>Fluorescent stained plaques/vial</td></tr><tr><td rowspan=1 colspan=1>DHI blended MAbs as DFA</td><td rowspan=1 colspan=1>CDC MAb-8 (indirect)</td></tr><tr><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>7,5,5</td><td rowspan=1 colspan=1>2,2,4</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>3,2,4</td><td rowspan=1 colspan=1>1,3,0</td></tr><tr><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1,1,2</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>d) hMPV subtype B2 in triplicate shell vials</td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>~ TCID50 per vial</td><td rowspan=1 colspan=2>Fluorescent stained plaques/vial</td></tr><tr><td rowspan=1 colspan=1>DHI blended MAbs as DFA</td><td rowspan=1 colspan=1>CDC MAb-8 (indirect)</td></tr><tr><td rowspan=1 colspan=1>5.6</td><td rowspan=1 colspan=1>2,2,3</td><td rowspan=1 colspan=1>1,0,1</td></tr><tr><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>1,2,3</td><td rowspan=1 colspan=1>0,1,0</td></tr><tr><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0,0,0</td></tr><tr><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0,0,0</td></tr></table>

Analytical sensitivity of the final blended solution of the three labeled MAbs testing cell culture specimens was also evaluated using another cell culture system against a the same non-labeled murine monoclonal antibody (MAb-8) developed at the CDC. Multi-well plate cultures (96-well format) of R-Mix cells were inoculated with a virus suspension at a concentration of 1 $\mathrm { T C I D } _ { 5 0 } /$ well. The plates were incubated at $3 7 ^ { \circ } \mathrm { C }$ for 24 hours and then stained with the final blended solution of the three labeled MAbs using the draft procedure of the D3 DFA Metapneumovirus Identification Kit, and with the anti-hMPV MAb comparator (CDC MAb-8 used as IFA). Each well was examined for fluorescent cells, and scored as “positive” (fluorescent cells present) or “negative” (no fluorescent cells). The experiment was repeated 3 times. Results in the following table are noted as numbers of positive wells counted in each 96-well plate.

<table><tr><td rowspan=1 colspan=5>Analytical sensitivity of the final blended solution of the 3 MAbs comparing to CDC MAb-8 using viral dilutionsamplified in a cell culture system (Multi-well plate culture)</td></tr><tr><td rowspan=2 colspan=1>hMPV subtypes</td><td rowspan=1 colspan=2>DHI blended MAbs as DFA</td><td rowspan=1 colspan=2>CDC MAb-8 (indirect)</td></tr><tr><td rowspan=1 colspan=1>Number of positivewells/96 wellsexamined</td><td rowspan=1 colspan=1>Mean (95%CI)</td><td rowspan=1 colspan=1>Number of positivewells/96 wellsexamined</td><td rowspan=1 colspan=1>Mean (95%CI)</td></tr><tr><td rowspan=1 colspan=1>A1</td><td rowspan=1 colspan=1>21,19,26</td><td rowspan=1 colspan=1>22.0 (13.0- 31.0)</td><td rowspan=1 colspan=1>0,0,0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>17,29,18</td><td rowspan=1 colspan=1>21.3 (4.8- 37.9)</td><td rowspan=1 colspan=1>14,18,19</td><td rowspan=1 colspan=1>17.0 (10.4- 23.6)</td></tr><tr><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>21,22,29</td><td rowspan=1 colspan=1>24.0 (13.2- 34.8)</td><td rowspan=1 colspan=1>20,18,21</td><td rowspan=1 colspan=1>19.7 (15.9- 23.5)</td></tr><tr><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>26,24,19</td><td rowspan=1 colspan=1>23.0 (14.0- 32.0)</td><td rowspan=1 colspan=1>20,17,16</td><td rowspan=1 colspan=1>17.7 (12.5- 22.8)</td></tr></table>

The combined results from the two analytical studies mentioned above showed significantly better staining with the DHI final blended solution of the three labeled MAbs than with the RUO unlabeled single MAb.

Analytical performance of the final blended solution of the three labeled MAbs as DFA reagent was also compared with that of a new commercial hMPV DFA. The study tested both reagents on acetone-fixed cell spots and hMPV infected shell vial cultures. The cell spots were prepared from the four laboratory virus subtypes A1, A2, B1, and B2. The staining results of the DHI hMPV DFA Reagent and the ASR reagent were very similar.

# Stability Studies

# Shelf life for the complete kit

The shelf life of the device $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit is established at 18 months, with 24 months of real-time data collected. At time of the 510(k) submission, November 2008, existing data supported a 24-month shelf life exclusive of the antigen control slide that has an 18-month shelf life. Stability studies has been conducted in two phases: (1) using kits produced during the development phase according to draft written procedures, and (2) using kits produced according to established procedures by manufacturing staff (this phase is on-going). Kits are stored at $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ , and are tested at time intervals during storage according to the study plan. Characteristics monitored are performance, as well as pH, color and clarity. Acceptance criterion is “bright fluorescence” (as opposed to “dim fluorescence” or “no fluorescence) observed in fixed, stained, infected cells (infected to a level of 3 to $^ { 4 + }$ cytopathic effect) using DFA Reagent at 1/16 dilution. The table below summarizes performance results to date:

<table><tr><td rowspan=1 colspan=4>Real-time Stability Test Results for D³ DFA Metapneumovirus Identification Kit</td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Lot number</td><td rowspan=1 colspan=2>Manufacture date(mm/dd/yyyy)</td><td rowspan=1 colspan=1>Date tested(mm/dd/yyyy)</td><td rowspan=1 colspan=1>DFA Reagent max.dilution for acceptableresult</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Time elapsed</td></tr><tr><td rowspan=1 colspan=1>MFG lots</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>07/14/2006</td><td rowspan=1 colspan=1>1/64</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>0-month</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/32</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/64</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>08/10/2006</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>09/26/2006</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>3-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>10/20/2006</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>01/03/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>6-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>01/23/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>04/03/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>9-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>04/24/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>07/02/2007</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>12-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>07/24/2007</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>10/01/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>15-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>10/15/2007</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>01/03/2008</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>18-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>01/15/2008</td><td rowspan=1 colspan=1>1/256</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006A</td><td rowspan=3 colspan=2>06/30/2006</td><td rowspan=3 colspan=1>07/07/2008</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td><td rowspan=4 colspan=1>24-months</td></tr><tr><td rowspan=1 colspan=1>063006B</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>063006C</td><td rowspan=1 colspan=1>1/128</td><td rowspan=1 colspan=1>Pass</td></tr><tr><td rowspan=1 colspan=1>071306</td><td rowspan=1 colspan=2>07/13/2006</td><td rowspan=1 colspan=1>07/14/2008</td><td rowspan=1 colspan=1>1/64</td><td rowspan=1 colspan=1>Pass</td></tr></table>

# Shelf life for the hMPV antigen control slides

Real-time stability studies were conducted to establish shelf life of the hMPV Antigen Control Slide. An expiration dating of 18 months from date of manufacture, when stored refrigerated at $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ was established. Lots of slides are reserved for staining with MPV DFA Reagent according to the device’s instructions for use. Each lot is tested at fresh and at expiration. As part of the release testing, an additional randomly selected lot may be included in the testing. Acceptance criterion is “bright fluorescence” observed in fixed, stained, infected cells using undiluted DFA Reagent. The table below summarizes performance results to date:

<table><tr><td rowspan=1 colspan=5>Real-time Stability Test Results for D³ DFA Metapneumovirus Antigen Control Slides</td></tr><tr><td rowspan=1 colspan=1>Lot number</td><td rowspan=1 colspan=1>Manufacture date(mm/dd/yyyy)</td><td rowspan=1 colspan=1>Date tested(mm/dd/yyyy)</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Time elapsed</td></tr><tr><td rowspan=1 colspan=1>112205B</td><td rowspan=1 colspan=1>11/22/2005</td><td rowspan=1 colspan=1>09/21/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>22-months</td></tr><tr><td rowspan=2 colspan=1>011106</td><td rowspan=2 colspan=1>01/11/2006</td><td rowspan=1 colspan=1>03/26/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>14-months</td></tr><tr><td rowspan=1 colspan=1>09/19/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>20-months</td></tr><tr><td rowspan=2 colspan=1>012606</td><td rowspan=2 colspan=1>01/26/2006</td><td rowspan=1 colspan=1>03/26/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>14-months</td></tr><tr><td rowspan=1 colspan=1>09/19/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>19-months</td></tr><tr><td rowspan=1 colspan=1>041406</td><td rowspan=1 colspan=1>04/14/2006</td><td rowspan=1 colspan=1>09/19/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>17-months</td></tr><tr><td rowspan=1 colspan=1>070606B</td><td rowspan=1 colspan=1>07/06/2006</td><td rowspan=1 colspan=1>07/16/2007</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>12-months</td></tr></table>

# d. Analytical Sensitivity (Detection limit):

# LoD studies on cell culture amplified specimens

Detection limit on cell culture amplified specimens of the $\mathrm { D } ^ { 3 } \mathrm { D F A }$ Metapneumovirus Identification Kit were addressed using cell culture systems. Results of these studies also provide evidence of analytical reactivity with representatives of each of the four recognized genetic sub-lineages of hMPV, A1, A2, B1, and B2. Analytical detection limits on cell culture amplified specimens for the four virus subtypes detected by the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit were established with results reported in numbers of fluorescent cells per cell monolayer. Each master stock virus preparation was diluted in a tenfold manner. Eight wells of a 48-well R-Mix cell culture plate were inoculated with each dilution. The plates were centrifuged at $7 0 0 \mathrm { x g }$ for 60 minutes, and then incubated at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ for 48-hours. Each well was stained with the $\mathrm { \Delta D } ^ { 3 }$ DFA Metapneumovirus Identification Kit then examined at $2 0 0 \mathrm { x }$ magnification and the number of fluorescent cells counted. In this study, the detection limit for the test on cell culture amplified specimens is defined as the lowest inoculum level at which positive wells (i.e. containing fluorescent cells) are observed, in terms of $\mathrm { T C I D } _ { 5 0 }$ . The table below summaries the results:

<table><tr><td rowspan=1 colspan=3>Limit of Detection of the D3 DFA Metapneumovirus Identification Kit on Cell Culture AmplifiedSpecimens (values are numbers of fluorescent staining cells per cell monolayer)</td></tr><tr><td rowspan=2 colspan=1>Virus strain</td><td rowspan=2 colspan=1>Virus input per well</td><td rowspan=1 colspan=1>Fluorescent staining cells/well</td></tr><tr><td rowspan=1 colspan=1>D³ DFA hMPV Identification Kit</td></tr><tr><td rowspan=1 colspan=1>hMPV A1 subtype</td><td rowspan=1 colspan=1>50-TCID50</td><td rowspan=1 colspan=1>47,39,41,31,26,30,21,29</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5-TCID50</td><td rowspan=1 colspan=1>0,0,0,3,1,0,2,0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.5-TCID50</td><td rowspan=1 colspan=1>0,0,0,0,0,0,0,0</td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>hMPV A2 subtype</td><td rowspan=1 colspan=1>50-TCID50</td><td rowspan=1 colspan=1>10,13,23,13,23,15,17,12</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>5-TCID50</td><td rowspan=1 colspan=1>3,1,1,4,2,2,0,0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.5-TCID50</td><td rowspan=1 colspan=1>0,0,0,0,0,0,0,0</td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>hMPV B1 subtype</td><td rowspan=1 colspan=1>50-TCID50</td><td rowspan=1 colspan=1>36,56,23,41,28,29,34,28</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>5-TCID50</td><td rowspan=1 colspan=1>4,7,0,3,1,0,4,4</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>0.5-TCID50</td><td rowspan=1 colspan=1>0,0,0,0,0,0,0,0</td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>hMPV B2 subtype</td><td rowspan=1 colspan=1>50-TCID50</td><td rowspan=1 colspan=1>25,49,36,41,53,68,43,27</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>5-TCID50</td><td rowspan=1 colspan=1>0,3,1,1,5,6,3,5</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1>0.5-TCID50</td><td rowspan=1 colspan=1>0,0,0,0,0,0,0,0</td></tr></table>

# LoD studies on direct specimens

Analytical detection limits on direct specimens for the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit were addressed using quantified cultures of characterized isolates of each of the 4 recognized genetic sublineages of hMPV (A1, A2, B1, and B2). The infected culture cells from a 1000 infected cells/mL culture were serially diluted with a suspension of uninfected LLC- $\mathrm { . M K } _ { 2 }$ cells. $2 5 \mathrm { - } \mu \mathrm { L }$ aliquots from each dilution level were spotted onto 10 replicate microscope slides, then fixed and stained according to the instructions for use described in this product insert. Each cell spot was examined at $2 0 0 \mathrm { x }$ magnification. Results were reported as numbers of positive replicates for each set of 10. Analytical detection limits for each of the 4 hMPV genetic sublineages were defined as the lowest dilutions at which at least 9 out of 10 replicates were detected. Results are summarized in the table below:

<table><tr><td rowspan=1 colspan=4>Limit of Detection of the D³ DFA Metapneumovirus Identification Kit for Direct Specimens</td></tr><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Infected cells/mL</td><td rowspan=1 colspan=1>Number of replicates with positive cells</td><td rowspan=1 colspan=1>LOD determination</td></tr><tr><td rowspan=10 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>25 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>200 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>8/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>50 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/10</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>100 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr></table>

# e. Analytical Reactivity:

Refer to analytical sensitivity section

# f. Analytical specificity:

The $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit was tested for cross-reactivity against a variety of cells and microorganisms. Stringent conditions for crossreactivity testing were achieved by using a high concentration of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Reagent and relatively high titers of microorganisms. The $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Reagent was prepared at $1 . 5 \mathrm { X }$ the concentration that is provided in the kit. No cross-reactivity was observed for 59 virus strains or for 16 host culture cell types. Twenty-five (25) bacterial strains, one yeast, three Chlamydia sp. and one protozoan were evaluated for cross-reactivity, including Staphylococcus aureus, a protein-Aproducing bacterium. Except for Staphylococcus aureus, which was cross reactive with the $\bar { \mathrm { D } } ^ { 3 }$ DFA Metapneumovirus Identification Kit, all other microorganisms tested negative. Staining of S. aureus appeared as small points of fluorescence. The Protein A produced by the bacterium, Staphylococcus aureus, may bind the Fc portion of some fluorescein-labeled monoclonal antibodies. Such binding can be distinguished from viral antigen binding on the basis of morphology, i.e., S. aureus-bound fluorescence appears as small ${ \sim } 1$ -micron diameter), bright dots. Results from cell cultures with bacterial contamination must, therefore, be interpreted with caution. The following language was added to the “Limitations of Procedure” section of the product insert to address this issue: “Light background staining may occur with specimens contaminated with Staphylococcus aureus strains containing large amounts of protein A. Protein A will bind to the Fc portions of conjugated antibodies. Such binding can be distinguished from viral antigen binding on the basis of morphology, for example, S. aureusbound fluorescence appears as small ( $\mathbf { \tilde { \Pi } }$ micron diameter), bright dots. Therefore, results from cell cultures with bacterial contamination must be interpreted with caution.”

Fifty-nine (59) virus strains were tested for cross reactivity. Depending on the particular virus, 71 to $7 1 4 \mathrm { T C I D } _ { 5 0 }$ viruses were inoculated into shell vial or multi-well plate cultures and incubated for 24 to 48 hours, to yield a $1 +$ to $^ { 4 + }$ cytopathic effect, processed and stained with the $1 . 5 \mathrm { X } \mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Reagent according to the procedure as detailed in the product insert. Stained cells were examined at 200X magnification. No cross reactivity was observed for the viruses listed below:

<table><tr><td colspan="4" rowspan="1">Virus Strains Tested for Cross Reactivity with D' DFA Metapneumovirus Identification Kit Reagent</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain or Type</td><td colspan="1" rowspan="1">D³ DFA hMPV Results</td><td colspan="1" rowspan="1">Inoculum (TCID50)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="4" rowspan="1">Virus Strains Tested for Cross Reactivity with D DFA Metapneumovirus Identification Kit Reagent</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain or Type</td><td colspan="1" rowspan="1">D³ DFA hMPV Results</td><td colspan="1" rowspan="1">Inoculum (TCID50)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 10</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 13</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 14</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 18</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 31</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 40</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 41</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Aichi (H3N2)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mal (H1N1)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Hong Kong (H3N2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Denver (H1N1)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Port Chalmers (H3N2)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Victoria (H3N2)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">New Jersey (HSWN1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WS (H1N1)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PR (H1N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Hong Kong</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Maryland</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mass</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">GL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Taiwan</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">JH-001 isolate</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Russia</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Long</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Wash</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9320</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">C-35</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4b</td><td colspan="1" rowspan="1">CH-19503</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">1(f)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MacIntyre</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1">HSV-2</td><td colspan="1" rowspan="1">MS</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Strain G</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1">CMV</td><td colspan="1" rowspan="1">Towne</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Davis</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AD169</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">714</td></tr><tr><td colspan="1" rowspan="1">Varicella-zoster</td><td colspan="1" rowspan="1">Webster</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Ellen</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus</td><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Bion (CDC V5-004)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr><tr><td colspan="1" rowspan="1">Rubeola (Measles)</td><td colspan="1" rowspan="1">Bion</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Control Slide</td></tr></table>

 Seventeen (16) host culture cell types were tested for cross reactivity. Cells were tested as intact monolayers or scraped and dotted cell spots and all were fixed in acetone. Confluent monolayers or cell spots were stained with the 1.5X preparation of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Reagent according to the procedure as detailed in the product insert, and then examined for cross reactivity. No cross reactivity was observed for the following cell lines:

<table><tr><td rowspan=1 colspan=4>Cell lines Tested for Cross Reactivity with D3 DFA Metapneumovirus Identification Kit Reagent</td></tr><tr><td rowspan=1 colspan=1>Cell lines</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>D3DFA hMPVResults</td><td rowspan=1 colspan=1>Monolayer/cell spot</td></tr><tr><td rowspan=1 colspan=1>A-549</td><td rowspan=1 colspan=1>Human lung carcinoma</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>Vero</td><td rowspan=1 colspan=1>African green monkey kidney</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>HEp-2</td><td rowspan=1 colspan=1>Human epidermoid larynxcarcinoma</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>MRC-5</td><td rowspan=1 colspan=1>Human embryonic lung</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>Mv1Lu</td><td rowspan=1 colspan=1>Mink lung</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>MDCK</td><td rowspan=1 colspan=1>Canine kidney</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>pRK</td><td rowspan=1 colspan=1>Rabbit kidney, primary</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>cell spot</td></tr><tr><td rowspan=1 colspan=1>pCMK</td><td rowspan=1 colspan=1>Cynomolgus monkey kidney,primary</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>cell spot</td></tr><tr><td rowspan=1 colspan=1>pRhMK</td><td rowspan=1 colspan=1>Rhesus monkey kidney, primary</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>R-Mix</td><td rowspan=1 colspan=1>Mixed A-549 + Mv1Lu</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>LLC-MK2</td><td rowspan=1 colspan=1>Rhesus monkey kidney</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>BGMK</td><td rowspan=1 colspan=1>African green monkey kidney</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>MRHF</td><td rowspan=1 colspan=1>Human foreskin fibroblast</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>WI-38</td><td rowspan=1 colspan=1>Human embryonic lung</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>cell spot</td></tr><tr><td rowspan=1 colspan=1>NCI-H292</td><td rowspan=1 colspan=1>Huamn pulmonarymucoepidermoid carcinoma</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr><tr><td rowspan=1 colspan=1>RD</td><td rowspan=1 colspan=1>Human rhabdomyosarcoma</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>monolayer</td></tr></table>

Thirty (30) microorganisms, including 25 bacterial and one yeast cultures, three Chlamydia sp. and one protozoan commercially available control slides were tested for cross reactivity. Bacteria were cultured, processed as suspensions, then spotted on microscope slides at CFUs (colony forming units) ranging from $6 . 4 \mathrm { ~ x ~ } \mathrm { { 1 0 } ^ { 4 } }$ to $2 . 9 3 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ well in a 10-uL dot, depending on the bacterium, then stained with the 1.5X DFA Reagent according to the procedure as detailed in the product insert. Stained cells were examined at 400X for cross reactivity. Concentrations for each bacterial organism cultured by DHI for cross reactivity testing were determined from suspensions of the bacteria in PBS by spectrophotometer according to McFarland standards of levels 1.0 and 2.0 (equaling approximately $3 . 0 \bar { \mathrm { ~ x ~ } } 1 0 ^ { 6 }$ and $6 . 0 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU per mL). Slides were prepared with spots of 10-uL of the suspensions to give either $3 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ or $\overline { { 6 . 0 \mathrm { ~ x ~ } 1 0 ^ { 4 } } }$ per spot. At the same time, $1 { \mathrm { - m L } }$ of each suspension was plated on an appropriate agar dish for colony confirmation. According to the confirmation agar cultures, initial concentrations of the bacterial organisms in the study ranged from $6 . 4 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ to $2 . 9 3 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ CFU/well.

Except for Staphylococcus aureus, which was cross reactive with the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Reagent, all other microorganisms tested negative. Reactivity with Staphylococcus aureus is more than likely due to binding the protein A produced by Staphylococcus aureus. Microorganisms tested are listed in the table below:

<table><tr><td rowspan=1 colspan=3>Microorganisms Tested for Cross Reactivity with D3 DFA Metapneumovirus Identification Kit Reagent</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>D³ DFA hMPV Results</td><td rowspan=1 colspan=1>CFU tested</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Acholeplasma laidlawii</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>~1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9.7 x 10^{}$</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.8 x 10</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>4.7 x 106</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Control Slides</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila psittaci</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Control Slides</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Control Slides</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.5 x 106</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.6 x 105</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Haemophilis influenzae type A</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9.3 x 105</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6.4 x 106</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6.5 x 104</td></tr><tr><td rowspan=1 colspan=1>Moraxella cartarrhalis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6.4 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>~1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma orale</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>~1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>~1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma salivarium</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>~1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.3 x 106</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.1 x 106</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Salmonella enteriditis</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.5 x 106</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1.8 x 106</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1.0 x 107</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>9.6 x 106</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>8.0 x 105</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2.9 x 107</td></tr><tr><td rowspan=1 colspan=1>Ureaplasma uralyticum</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>~1.0 x 104</td></tr><tr><td rowspan=1 colspan=1>Protozoan</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Trichomonas vaginalis</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Control Slides</td></tr><tr><td rowspan=1 colspan=1>Yeast</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8.7 x 106</td></tr></table>

g. Assay cut-off: Not applicable h. Interfering Substances: Not applicable i. Comparator Assay Description and Analytical Validation Studies

At the time of the DHI $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit clinical studies, there were no standard or FDA cleared methods to detect hMPV in respiratory specimens from patients with signs and symptoms of acute respiratory infection. For purposes of these studies, Diagnostic Hybrids, Inc (DHI) established and validated a real-time Reverse Transcriptase (RT-PCR) procedure for use as a comparative test method to detect hMPV RNA in a

specimen. This real time RT-PCR assay was developed by DHI using a primer set and TaqMan probe as described in the publication: Maertzdorf, J., CK Wang, JB Brown, JD Quinto, M Chu, M de Graff, BG van den Hoogen, R Spaete, ADME Osterhaus, and RAM Fouchier. “Real-time Reverse Transcriptase PCR Assay for Detection of Human Metapneumoviruses from All Known Lineages” J. Clin. Microbiol. 2004:42:981-986. A nucleic acid (RNA) extraction procedure was also developed and validated, as was the RT-PCR reaction, including primer sets, and gel electrophoresis. Detection of an amplicon in the PCR end-product is presumptive evidence of hMPV RNA presence in a specimen. The amplicons were sequenced by a third party (Agencourt). DHI then compared these sequences against an established/published set of hMPV sequences.

Confirmation of the hMPV RNA sequences in the amplicon was interpreted as verification of hMPV RNA in the specimen.

# Analytical Sensitivity

The analytical sensitivity of the hMPV real-time RT-PCR/Sequencing Comparator Assay was determined for 4 hMPV strains (subtypes A1, A2, B1 and B2). Viral stocks of 4 hMPV strains (subtypes A1, A2, B1 and B2) were generated in LLC- $\mathrm { M K } _ { 2 }$ cultured cells. The viral stocks were frozen at $\mathrm { - 7 0 ^ { \circ } C }$ . The $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ of each stock was determined by cell culture with monoclonal antibody detection. A nasopharyngeal clinical matrix was created from nasopharyngeal swabs placed in transport medium obtained from patients with signs and symptoms of acute respiratory infection. The matrix was tested for the presence of hMPV RNA using the hMPV real-time RT-PCR Comparator Assay. Patient specimens found to be negative were pooled to create sufficient material for all of the dilutions. The quantified viral stocks were diluted with the pooled nasopharyngeal clinical matrix to produce a suspension equivalent to the following $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ levels: 1000, 500, 250, 100, 50, 10, 1, and 0.1. Each $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ level dilution was extracted 10 times (80 per hMPV strain) using the Qiagen Viral RNA Mini Kit. The hMPV real-time RT-PCR/Sequencing Comparator Assay was performed in singlet using $2 0 \mathrm { - } \mu \mathrm { L }$ volumes of each extraction. Analytical detection limits for each of the 4 hMPV genetic sublineages were defined as the lowest dilutions at which at least 9 out of 10 replicates were detected. Detailed analytical sensitivity data for the hMPV realtime RT-PCR/Sequencing Comparator Assay are presented in the following table:

<table><tr><td rowspan=1 colspan=7>hMPV real-time RT-PCR/Sequencing Comparator Assay Analytical Sensitivity Summary</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageCT</td><td rowspan=1 colspan=1>StandardDeviation CT</td><td rowspan=1 colspan=1>Min CT</td><td rowspan=1 colspan=1>Max CT</td><td rowspan=1 colspan=1>ReplicatesDetected</td></tr><tr><td rowspan=8 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>33.28</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>32.99</td><td rowspan=1 colspan=1>33.81</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>34.30</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>33.46</td><td rowspan=1 colspan=1>35.29</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>35.25</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>34.24</td><td rowspan=1 colspan=1>36.18</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>37.69</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>36.26</td><td rowspan=1 colspan=1>39.43</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>38.48</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>39.94</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=8 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>29.67</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>32.06</td><td rowspan=1 colspan=1>33.17</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>30.88</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>33.36</td><td rowspan=1 colspan=1>35.28</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>32.16</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>33.98</td><td rowspan=1 colspan=1>36.47</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>33.30</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>35.52</td><td rowspan=1 colspan=1>37.38</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>34.67</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>37.33</td><td rowspan=1 colspan=1>39.08</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>35.11</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>38.51</td><td rowspan=1 colspan=1>41.43</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40.96</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40.96</td><td rowspan=1 colspan=1>40.96</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=8 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>31.39</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>30.90</td><td rowspan=1 colspan=1>32.85</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>32.22</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>31.79</td><td rowspan=1 colspan=1>32.53</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>32.85</td><td rowspan=1 colspan=1>.047</td><td rowspan=1 colspan=1>32.07</td><td rowspan=1 colspan=1>33.59</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.37</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>33.67</td><td rowspan=1 colspan=1>34.77</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>35.10</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>34.42</td><td rowspan=1 colspan=1>35.85</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>37.82</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>35.51</td><td rowspan=1 colspan=1>40.07</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>37.25</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>36.46</td><td rowspan=1 colspan=1>38.03</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=8 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>32.96</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>32.49</td><td rowspan=1 colspan=1>33.34</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>35.37</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>33.37</td><td rowspan=1 colspan=1>37.18</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>34.13</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>34.66</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>40.63</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>34.85</td><td rowspan=1 colspan=1>38.36</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>40.32</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>39.46</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>39.46</td><td rowspan=1 colspan=1>39.46</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/10</td></tr></table>

# Analytical Specificity

The specificity of the hMPV real-time RT-PCR Comparator Assay was evaluated by analyzing nucleic acid extracts of Diagnostic Hybrids, Inc. Respiratory Viruses Panel (Lot 111406), human gDNA (Biochain, Lot A804328), human total RNA (Lot 11140405, Catalog # 540017-41) and the Negative Clinical Specimen. $1 4 0 \mu \mathrm { L }$ of each sample was extracted and tested by

the DHI hMPV RT-PCR assay.

hMPV real-time RT-PCR Comparator Assay demonstrated $100 \%$ specificity and data is presented in the follow table:

<table><tr><td rowspan=1 colspan=2>hMPV real-time RT-PCR Comparator Assay Analytical Specificity Summary</td></tr><tr><td rowspan=1 colspan=1>Panel Key</td><td rowspan=1 colspan=1>hMPV real-time RT-PCR ComparatorAssay Result</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B B/Taiwan/2/62</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>RSV RSV/B/Wash/18537/62</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2 Greer</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>RSV Long</td><td></td></tr><tr><td rowspan=1 colspan=1>Influenza A A/WS/33</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1, Adenoid 71</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>RSV 9320</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/Mal/302/54</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/Victoria/3/75</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1 C-35</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3 C234</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 5, Adenoid 75</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza B B/GL/1734/54</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/PortChalmers/1/73</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human gDNA</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Human Total RNA</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Negative Clinical Specimen</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>In-house Positive hMPV Strain</td><td rowspan=1 colspan=1>+</td></tr></table>

# Extraction Efficiency

The extraction efficiency was determined by extracting serial 10-fold dilutions of cRNA and comparing the actual copies of cRNA in the eluate versus the calculated copies if $100 \%$ recovery. The actual copies of cRNA were obtained from the qRT-PCR standard curve constructed by a serial dilution of unextracted cRNA standards. The extraction efficiency ranges from 25 to $40 \%$ . The range was obtained from triplicate extractions repeated over 3 days. Duplicate PCR reactions were performed on each dilution. Detailed results of this study are presented in the following table:

<table><tr><td rowspan=2 colspan=1>Pre-extraction viralload (copies/mL)</td><td rowspan=1 colspan=3>% of Recovery</td></tr><tr><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Day 3</td></tr><tr><td rowspan=1 colspan=1>8.58E+08</td><td rowspan=1 colspan=1>3.36E+01</td><td rowspan=1 colspan=1>2.50E+01</td><td rowspan=1 colspan=1>5.72E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+07</td><td rowspan=1 colspan=1>3.94E+01</td><td rowspan=1 colspan=1>2.35E+01</td><td rowspan=1 colspan=1>1.60E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+06</td><td rowspan=1 colspan=1>2.74E+01</td><td rowspan=1 colspan=1>1.98E+01</td><td rowspan=1 colspan=1>4.29E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+05</td><td rowspan=1 colspan=1>2.91E+01</td><td rowspan=1 colspan=1>2.55E+01</td><td rowspan=1 colspan=1>5.17E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+04</td><td rowspan=1 colspan=1>2.60E+01</td><td rowspan=1 colspan=1>3.36E+01</td><td rowspan=1 colspan=1>3.81E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+03</td><td rowspan=1 colspan=1>1.69E+01</td><td rowspan=1 colspan=1>3.76E+01</td><td rowspan=1 colspan=1>2.73E+01</td></tr><tr><td rowspan=1 colspan=1>4.19E+03</td><td rowspan=1 colspan=1>4.45E+01</td><td rowspan=1 colspan=1>1.19E+01</td><td rowspan=1 colspan=1>1.80E+01</td></tr></table>

Based on thorough analytical validations, the hMPV real-time RT-PCR Comparator Assay is an acceptable method to be used as a part of the composite reference methods in determining “clinical diagnostic truth” for the DHI $\mathbf { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit Clinical Trial.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable b. Matrix Description and Comparison: Not applicable

3. Clinical studies:

a. Prospective Clinical Studies Testing Direct Respiratory Specimens

Performance characteristics of the DHI $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit testing direct respiratory specimens were established during prospective studies at 3 geographically diverse U.S. clinical laboratories during the 2005 and 2006 respiratory virus seasons (December 2005 – April 2006 and December $2 0 0 6 -$ March 2007). All specimens used in the studies meeting the inclusion and exclusion criteria represented excess, remnants of respiratory specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study.

Performance of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit was assessed and compared to a predetermined algorithm that used composite comparator methods at clinical study site 1 and 2. The composite comparator methods consisted of viral culture and a validated hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparator assay. The hMPV real-time RT-PCR comparator assay targets the hMPV Nucleocapsid gene. “True” hMPV positive was defined as any sample that either tested positive by viral culture, or had bi-directional

sequencing data meeting pre-defined quality acceptance criteria that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E-values. “True” hMPV negative was defined as any sample that tested negative by both viral culture and the hMPV real-time RT-PCR comparator assay.

Performance of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit was evaluated and compared to the same validated hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparator assay as described earlier, alone, at clinical study site 3. Any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E-values, was considered as hMPV positive, and the real-time hMPV RT-PCR comparator assay negatives were considered as hMPV negatives at this site.

The E-values generated from the clinical trials range from a low of 2e-77 to a high of 2e-67. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone.   
(http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).   
Therefore an E-Value ranging from 2e-67 to 2e-77 has a very low probability of occurring purely by chance.

Study Site 1 evaluated a total of 1564 fresh respiratory specimens submitted, December 2006 through March 2007, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and fixed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 1:

<table><tr><td rowspan=1 colspan=3>Site 1  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>687</td><td rowspan=1 colspan=1>877</td></tr><tr><td rowspan=1 colspan=1>Age: &lt; 1m</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>≥ 1m to &lt; 2y</td><td rowspan=1 colspan=1>444</td><td rowspan=1 colspan=1>617</td></tr><tr><td rowspan=1 colspan=1>≥ 2y to &lt; 12y</td><td rowspan=1 colspan=1>164</td><td rowspan=1 colspan=1>185</td></tr><tr><td rowspan=1 colspan=1>≥ 12y to &lt; 18y</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>≥ 18y to &lt; 21y</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>≥21y</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Of the 1564 fresh respiratory specimens tested, 1509 were nasal wash/nasopharyngeal aspirate specimens. Due to insufficient sample numbers to establish performance of the $\mathrm { D } ^ { 3 ^ { \bullet } }$ DFA Metapneumovirus Identification Kit, 55 other types of respiratory specimens were removed from performance analysis. Of the 1509 fresh nasal wash/nasopharyngeal aspirate specimens tested, 27 were further excluded from the performance analysis due to insufficient volume for the comparator methods, resulting in a total of 1482 fresh nasal wash/nasopharyngeal aspirate specimens for analysis. The table below shows the study results of the claimed specimen type at study site 1:

<table><tr><td rowspan=1 colspan=1>Fresh NasalWash/NasopharyngealAspirate</td><td rowspan=1 colspan=3>Composite Comparator Methods</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>1249</td><td rowspan=1 colspan=1>1357</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>1252</td><td rowspan=1 colspan=1>1482</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>122/230</td><td rowspan=1 colspan=1>53.0%</td><td rowspan=1 colspan=1>46.6%-59.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>1249/1252</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3%-99.9%</td></tr></table>

Study Site 2 evaluated a total of 371 fresh respiratory specimens submitted, December 2005 through January 2006, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and fixed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 2:

<table><tr><td rowspan=1 colspan=3>Site 2  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>216</td></tr><tr><td rowspan=1 colspan=1>Age: &lt; 1m</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>≥ lm to &lt; 2y</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>83</td></tr><tr><td rowspan=1 colspan=1>≥ 2y to &lt; 12y</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>≥ 12y to &lt; 18y</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>≥ 18y to &lt; 21y</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥21y</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Of the 371 fresh respiratory specimens tested, all were nasal/nasopharyngeal swab specimens. 3 were excluded from the performance analysis due to insufficient volume for the comparator methods, resulting in a total of 368 fresh nasal/nasopharyngeal swab specimens for analysis. The table below shows the study results of the claimed specimen type at study site 2:

<table><tr><td rowspan=1 colspan=1>Fresh Nasal/NasopharyngealSwab</td><td rowspan=1 colspan=3>Composite Comparator Methods</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>326</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>368</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>41/58</td><td rowspan=1 colspan=1>70.7%</td><td rowspan=1 colspan=1>57.3%-81.9%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>309/310</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.2%-100%</td></tr></table>

Study Site 3 evaluated a total of 174 fresh respiratory specimens submitted, March 2006 through April 2006, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and fixed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 3:

<table><tr><td rowspan=1 colspan=4>Site 3  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Sex NotReported</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Age: &lt; 1m</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥ lm to &lt; 2y</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥ 2y to &lt; 12y</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥ 12y to &lt; 18y</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥ 18y to &lt; 21y</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>≥21y</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td></tr></table>

Of the 174 fresh respiratory specimens tested, 62 were nasal wash/nasopharyngeal aspirate specimens, and 110 were nasal/nasopharyngeal swab specimens. Of the 62 nasal wash/nasopharyngeal aspirate specimens, 30 were excluded from the performance analysis due to insufficient volume for the comparator method, resulting in a total of 32 fresh nasal wash/nasopharyngeal aspirate specimens for analysis. Of the 110 nasal/nasopharyngeal swab specimens, 44 were excluded from the performance analysis due to insufficient volume for the comparator method, resulting in a total of 66 fresh nasal/nasopharyngeal swab specimens for analysis. The tables below show the study results of the claimed specimen types at study site 3:

<table><tr><td rowspan=1 colspan=1>Fresh NasalWash/NasopharyngealAspirate</td><td rowspan=1 colspan=3>hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparatorassay</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>66.4%-100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement*</td><td rowspan=1 colspan=1>23/23</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.2%-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Fresh Nasal/NasopharyngealSwab</td><td rowspan=1 colspan=3>hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparatorassay</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>66</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement*</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>19.4%-99.4%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement*</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.2%-100%</td></tr></table>

\*Since the performance of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit at clinical study site 3 was not assessed against the predetermined composite comparator methods, positive and negative percent agreements, instead of sensitivity and specificity, are used in the performance presentation.

# b. Cultured Cells Testing

Performance characteristics of the DHI $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit testing cultured cell specimens were established during a prospective study at DHI during the 2007 respiratory virus seasons (January 2008 – April). All specimens used in the studies meeting the inclusion and exclusion criteria represented excess, remnants of respiratory specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each collection site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code.

Performance of the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit testing cultured cell specimens was evaluated and compared to the same validated hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparator assay as described earlier, at clinical study site 4. Any cultured cell specimens that had bidirectional sequencing data meeting pre-defined quality acceptance criteria that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable Evalues, was considered as hMPV positive. The real-time hMPV RT-PCR comparator assay negative cultured cell specimens were considered as hMPV

negatives.

The E-values generated from the clinical trials range from a low of 2e-77 to a high of 2e-67. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone.   
(http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).   
Therefore an E-Value ranging from 2e-67 to 2e-77 has a very low probability of occurring purely by chance.

A total of 74 freeze-thawed nasopharyngeal swab specimens were cultured and stained in accordance with the $\mathrm { D } ^ { \frac { 3 } { 2 } }$ DFA Metapneumovirus Identification Kit procedure. The table below shows the study results testing cultured cell specimens at study site 4:

<table><tr><td rowspan=1 colspan=1>Freeze-thawedNasopharyngeal SwabAmplified in Cell Culture</td><td rowspan=1 colspan=3>DHI hMPV RT-PCR Followed by Sequencing</td></tr><tr><td rowspan=1 colspan=1>DHI DFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>69</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>83.3%</td><td rowspan=1 colspan=1>35.9%-99.6%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.7%-100%</td></tr></table>

c. Retrospective Clinical studies

Not applicable.

d. Other clinical supportive data (when a. and b. are not applicable):

# Addition analytical study to support the cultured cells testing claim for the DHI $\mathbf { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit

An additional analytical study using well characterized hMPV isolates in cell cultures was carried out by operators in three laboratories to complement the clinical study data generated by testing cultured cells at study site 4, in supporting the cultured cells claim for the DHI $\mathrm { D } ^ { \overline { { 3 } } }$ DFA Metapneumovirus Identification Kit.

Cultured LLC- $\mathrm { . M K } _ { 2 }$ cells were inoculated with known (well characterized) isolates of hMPV obtained from the University of Iowa Emerging Pathogens Laboratory located in Coralville, IA. The cell culture was incubated for 48-hours to amplify the virus. Cell spots were prepared on glass slides as described in the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit, using cell suspensions from each of the

cultures at three concentration levels of infected cells (less than $10 \%$ , between 20 and $30 \%$ , and between 40 and $50 \%$ ). The slides were sent to three outside investigators where they were stained using the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit and examined. Each investigator provided interpretations of test results, i.e., presence or absence of fluorescent cells. Replicate prepared slides were evaluated at DHI by staining and examining with an alternative non-labeled murine monoclonal antibody (MAb-8) to hMPV strain MPV75-1998/CAN98-75, which was developed at the CDC by standard methods. (Note: The MAb-8 is a research use only device that has not been cleared by the FDA. However, there is data presented in the literature addressing its specificity and sensitivity to hMPV: Landry, Marie L., Ferguson, David, Cohen, Sandra, Peret, Teresa C. T., Erdman, Dean D., “Detection of Human Metapneumovirus in Clinical Samples by Immunofluorescence Staining of Shell Vial Centrifugation Cultures Prepared from Three Different Cell Lines”, J. Clin. Microbiol. 2005 43: 1950-1952.). DHI also performed DHI hMPV RT-PCR/Sequencing Assay on each of the virus isolates to verify their identities as hMPV.

The following table summarizes the data from the three external sites using the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit, compared to the MAb-8 results generated at the DHI facility:

<table><tr><td rowspan=1 colspan=1>Culture Slides</td><td rowspan=1 colspan=3>CDC MAb-8</td></tr><tr><td rowspan=1 colspan=1>DHI DFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>74/75</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>92.9%-99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.0%-100%</td></tr></table>

The following table summarizes the data from the three external sites using the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit, compared to the DHI hMPV RTPCR/Sequencing Assay results generated at the DHI facility:

<table><tr><td rowspan=1 colspan=1>Culture Slides</td><td rowspan=1 colspan=3>hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparatorassay</td></tr><tr><td rowspan=1 colspan=1>DHI DFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>74/75</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>92.9%-99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.0%-100%</td></tr></table>

In conclusion, the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit detected $9 8 . 7 \%$ (74/75) of the suspensions expected to contain infected cells from cell culture. The CDC MAb-8 reagent detected $100 \%$ (75/75) of the suspensions expected to contain infected cells from cell culture. One suspension, which was hMPV strain B2, was missed by the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit reagent. The estimated infected level, as seen on the CDC MAb-8 stained suspension, was less than $2 \%$ infected. It is probable that due to sampling issues, the $\mathrm { \Delta D } ^ { 3 }$ DFA Metapneumovirus Identification Kit reagent-stained slide did not contain any infected cells.

4. Clinical cut-off: Not applicable

# 5. Expected values/Reference range:

In the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit multicenter prospective clinical study testing direct respiratory specimens, a total of 2109 respiratory specimens were tested from three U.S. clinical laboratories across the United States during the 2005 and 2006 respiratory virus seasons (December 2005 – April 2006 and December 2006 – March 2007). hMPV prevalence as determined by the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit direct specimen testing varied from $8 . 1 \%$ to $1 1 . 3 \%$ by site and averaged $9 . 3 \%$ . The number and percentage of hMPV positive cases by the $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit direct specimen testing, calculated by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Total SpecimensEvaluated</td><td rowspan=1 colspan=1>hMPV Positive By theDHI D3 DFAMetapneumovirusIdentification Kit</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Number Positive</td><td rowspan=1 colspan=1>Observed Prevalence</td></tr><tr><td rowspan=1 colspan=1>&lt; 1 month</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>≥ 1 month to &lt; 2 years</td><td rowspan=1 colspan=1>1249</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>9.9%</td></tr><tr><td rowspan=1 colspan=1>≥ 2 years to &lt; 5 years</td><td rowspan=1 colspan=1>293</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>10.2%</td></tr><tr><td rowspan=1 colspan=1>≥ 5 years to &lt; 12 years</td><td rowspan=1 colspan=1>151</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>7.3%</td></tr><tr><td rowspan=1 colspan=1>≥ 12 years to &lt; 18 years</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>≥ 18 years to &lt; 21 years</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10.0%</td></tr><tr><td rowspan=1 colspan=1>≥ 21 years to &lt; 60 years</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>10.6%</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2109</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>8.7%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.