# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K090239   
B. Purpose for Submission: New device clearance   
C. Measurand: Clostridium difficile nucleic acids

# D. Type of Test:

A Real Time PCR in vitro diagnostic test for the qualitative detection of toxigenic Clostridium difficile nucleic acids isolated and purified from liquid or soft stool specimens obtained from symptomatic patients.

E. Applicant: Prodesse Incorporated

F. Proprietary and Established Names: ProGastro™ Cd Assay

# G. Regulatory Information:

1. Regulation section: 866.2660

2. Classification: I

3. Product code: LLH

4. Panel: Microbiology

# H. Intended Use:

The ProGastro™ Cd Assay is a Real Time PCR in vitro diagnostic test for the qualitative detection of toxigenic Clostridium difficile nucleic acids isolated and purified from liquid or soft stool specimens obtained from symptomatic patients. This test targets the Clostridium difficile toxin B gene (tcdB) and is intended for use to aid in the diagnosis of toxigenic Clostridium difficile infections.

2. Indication(s) for use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Cepheid SmartCycler II Real Time Instrument

# I. Device Description:

The ProGastro Cd Assay detects toxigenic Clostridium difficile and an Internal Control by a process of nucleic acid extraction from patient specimens followed by PCR amplification and detection. Following collection of a soft or liquid stool sample from a symptomatic patient, a portion of the sample is diluted in Stool Transport and Recovery (S.T.A.R.) Buffer and the solids separated via centrifugation (Stool Clarification). The Internal Control is added to the sample prior to extraction to monitor for PCR inhibitors that may be present. The nucleic acids from the sample are extracted and purified using the bioMérieux NucliSENS easyMAG automated extractor. Nucleic acids are added to the $C .$ diff Mix for subsequent PCR   
amplification and detection using the Cepheid   
SmartCycler II.

The $C .$ . diff Mix contains oligonucleotide primers and probes that target the tcdB gene of toxigenic strains of $C .$ diff. The probes are dual-labeled with a reporter dye attached to the $5 ^ { \circ }$ -end and a quencher dye attached to the $3 ^ { \circ }$ -end (see table below). During PCR amplification the primers and probes anneal to the template (if present) followed by primer extension and template amplification. The $5 ' - 3 '$ exonuclease activity of the Taq polymerase cleaves the probe thus separating the reporter dye from the quencher and generating an increase in fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification product present. The SmartCycler II instrument and software monitors the process, interprets the data, and presents a report upon completion.

# J. Substantial Equivalence Information:

1. Predicate device name(s): C. difficile Toxin/Antitoxin Kit BD GeneOhm™ Cdiff Assay

2. Predicate 510(k) number(s): K923463 K081920

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Features</td><td colspan="1" rowspan="1">Prodesse ProGastro CdAssay</td><td colspan="1" rowspan="1">TechLab C. difficileToxin/Antitoxin Kit</td><td colspan="1" rowspan="1">BD Geneohm CDiffAssay</td></tr><tr><td colspan="1" rowspan="1">510(k)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">K923463</td><td colspan="1" rowspan="1">K081920</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.2660</td><td colspan="1" rowspan="1">866.2660</td><td colspan="1" rowspan="1">866.2260</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">LLH</td><td colspan="1" rowspan="1">LLH</td><td colspan="1" rowspan="1">LLH</td></tr><tr><td colspan="1" rowspan="1">ClassificationName</td><td colspan="1" rowspan="1">Regents, Clostridiumdifficile toxin</td><td colspan="1" rowspan="1">Regents, Clostridiumdifficile toxin</td><td colspan="1" rowspan="1">Regents, Clostridiumdifficile toxin</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">Class I</td><td colspan="1" rowspan="1">Class I</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Real Time PCR in vitrodiagnostic test for thequalitative detection oftoxigenic Clostridium</td><td colspan="1" rowspan="1">Use in conjunction withthe tissue culturecytotoxicity assay for theconfirmation of</td><td colspan="1" rowspan="1">Rapid in vitro diagnostictest for the direct,quantitative detection ofC. difficile toxin B gene</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">difficile nucleic acidsisolated and purified fromstool specimens obtainedfrom symptomaticpatients. This test targetsthe Clostridium difficiletoxin B gene (tcdB) and isintended for use to aid inthe diagnosis of toxigenicClostridium difficileinfections.</td><td colspan="1" rowspan="1">Clostridium difficile toxinin patient specimens(stool).</td><td colspan="1" rowspan="1">(tcdB) in human liquid orsoft stool specimens frompatients suspected ofhaving Clostridiumdifficile-associated disease(CDAD).</td></tr><tr><td colspan="1" rowspan="1">Technology/Detection</td><td colspan="1" rowspan="1">Real Time PCR</td><td colspan="1" rowspan="1">Tissue culture cytotoxicity/ cell rounding</td><td colspan="1" rowspan="1">Real Time PCR</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Raw stool (not formed)</td><td colspan="1" rowspan="1">Raw stool</td><td colspan="1" rowspan="1">Liquid or soft raw stool</td></tr><tr><td colspan="1" rowspan="1">CollectionandTransportMedium</td><td colspan="1" rowspan="1">Standard stool collectionand handling procedures.Specimens should bestored refrigerated (2-8°C)for up to 48 hours beforeprocessing. If StoolClarification cannot beperformed within 48 hoursof collection, storespecimens at ≤ - 70°C.</td><td colspan="1" rowspan="1">Standard stool collectionand handling procedures.Specimens should bestored at 2-8 fortransport and storage andtested with 48 hours.</td><td colspan="1" rowspan="1">Transfer liquid or softstool (but not urine) into adry sterile container.Specimens should be keptat 2-25°C during transportand protected againstfreezing or exposure toexcessive heat.Specimens can be storedat 2-8C for up to 5 daysbefore testing. Specimenscan be kept at 15-25°C forup to 48 hours beforetesting. Specimens can betested after 1 freeze-thawcycle.</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidIsolation</td><td colspan="1" rowspan="1">bioMérieux NucliSENSeasyMAG</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Instrument/AssayPltform</td><td colspan="1" rowspan="1">Cepheid SmartCycler IISystem</td><td colspan="1" rowspan="1">Tissue culture/invertedmicroscope (visualize cellrounding)</td><td colspan="1" rowspan="1">Cepheid SmartCycler</td></tr><tr><td colspan="1" rowspan="1">Time toResult</td><td colspan="1" rowspan="1">2 hours</td><td colspan="1" rowspan="1">24 to 96 hours</td><td colspan="1" rowspan="1">2 hours</td></tr><tr><td colspan="1" rowspan="1">AssayControls</td><td colspan="1" rowspan="1">Positive Control andInternal Control</td><td colspan="1" rowspan="1">Positive Toxin Controlreagent</td><td colspan="1" rowspan="1">Control DNA (positivecontrol) and negativecontrol</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">NegativePositiveUnresolvedNot Determined</td><td colspan="1" rowspan="1">NegativePositiveUnresolvedCytotoxic activity presentbut is not characteristic ofC. difficile toxin</td><td colspan="1" rowspan="1">NegativePositiveUnresolvedNot Determined</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

The ProGastro Cd Assay detects toxigenic Clostridium difficile and an Internal Control by a process of nucleic acid extraction from patient specimens followed by PCR amplification and detection. Following collection of a soft or liquid stool sample from a symptomatic patient, a portion of the sample is diluted in Stool Transport and Recovery (S.T.A.R.) Buffer and the solids separated via centrifugation (Stool Clarification). The Internal Control is added to the sample prior to extraction to monitor for PCR inhibitors that may be present. The nucleic acids from the sample are extracted and purified using the bioMérieux NucliSENS easyMAG automated extractor. Nucleic acids are added to the $C .$ . diff Mix for subsequent PCR amplification and detection using the Cepheid SmartCycler II.

The C. diff Mix contains oligonucleotide primers and probes that target the tcdB gene of toxigenic strains of $C .$ . diff. The probes are dual-labeled with a reporter dye attached to the $5 '$ -end and a quencher dye attached to the $3 ^ { \circ }$ -end (see table below). During PCR amplification the primers and probes anneal to the template (if present) followed by primer extension and template amplification. The $5 ' - 3 '$ exonuclease activity of the Taq polymerase cleaves the probe thus separating the reporter dye from the quencher and generating an increase in fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification product present. The SmartCycler II instrument and software monitors the process, interprets the data, and presents a report upon completion.

# M. Performance Characteristics (if/when applicable):

# Expected Values

The prevalence of $C .$ . difficile Infection can vary between institutions. Variables that affect the frequency of the disease are patient population, type of institution and epidemiology. C. difficile is responsible for approximately $20 \%$ of all cases of antibiotic associated diarrhea and the overall incidence of disease is approximately $0 . 1 \cdot 2 \%$ among all hospitalized patients1 . In the ProGastro Cd multicenter prospective study, the prevalence among symptomatic patients varied from $1 0 . 4 \%$ to $1 5 . 8 \%$ by site and averaged $1 3 . 4 \%$ overall. The number and percentage of $C .$ . diff positive cases determined by the ProGastro Cd Assay during this study, stratified by patient age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Total (N)</td><td rowspan=1 colspan=1>Total # C. diff Positive by theProGastro Cd Assay</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>2-5 years</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>13.3%</td></tr><tr><td rowspan=1 colspan=1>6- 10 years</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11.6%</td></tr><tr><td rowspan=1 colspan=1>11- 15 years</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>16.4%</td></tr><tr><td rowspan=1 colspan=1>16-21years</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>17.0%</td></tr><tr><td rowspan=1 colspan=1>2259years</td><td rowspan=1 colspan=1>292</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>12.7%</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>256</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>12.9%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>771</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>13.4%</td></tr></table>

# Clinical Performance

Performance characteristics of the ProGastro Cd Assay were established during a prospective study at three U.S. clinical laboratories from July through October 2008. Samples used for this study were leftover raw stool specimens that were collected for routine Clostridium difficile testing from patients over two years of age by each site. The reference method was tissue culture cytotoxin assay (CTA). Demographic details for this patient population are summarized in the following table:

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects(Percentageof Total)</td></tr><tr><td rowspan=1 colspan=1>2-5 years</td><td rowspan=1 colspan=1>60 (7.8 %)</td></tr><tr><td rowspan=1 colspan=1>6-21years</td><td rowspan=1 colspan=1>163(21.1%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>292(37.9%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>256 (33.2%)</td></tr></table>

A total of 771 raw stool samples were tested with the ProGastro Cd Assay and by CTA.   
None of the 771 samples were inhibited when tested with the ProGastro Cd Assay.

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>CTA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Comments</td></tr><tr><td rowspan=2 colspan=1>asCss p</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>37a</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>Sensitivity 91.7%(83.0% - 96.1%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6b</td><td rowspan=1 colspan=1>662</td><td rowspan=1 colspan=1>668</td><td rowspan=1 colspan=1>Specificity 94.7%(92.8% -96.1%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>699</td><td rowspan=1 colspan=1>771</td><td rowspan=1 colspan=1>EY</td></tr></table>

Discrepant analysis for samples where ProGastro Cd Assay and CTA results were in disagreement was performed using a predetermined algorithm including a molecular (PCR) test (which targeted a different region of the tcdB gene than that of the ProGastro Cd Assay) followed by bidirectional genetic sequencing, enzyme immunoassay (EIA), and culture followed by PCR and bidirectional sequencing.

a 34 samples positive by discrepant analysis. Of these 33 were positive by sequencing, and one (1) was positive by culture followed by sequencing.   
b Four (4) samples positive by discrepant analysis. Of these, one (1) was positive by sequencing, one (1) was positive by EIA, and two (2) were positive by culture followed by sequencing.

# Reproducibility

The reproducibility of the ProGastro Cd Assay was evaluated at three laboratory sites. Reproducibility was assessed using a panel of six simulated samples that included medium positive, low positive (near the assay limit of detection,   
$\ge 9 5 \%$ positive) and “high negative” $\text{‰}$ positive) samples. Panels and controls were tested at each site by two operators for five days (6 samples and 4 controls and 2 operators for 5 days at 3 sites $= 3 0 0$ ). The overall percent agreement with the expected result for the ProGastro Cd Assay was $9 9 . 0 \%$ .

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwith expectedresult (%)</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td><td rowspan=2 colspan=1>|OverallAverage CrVValue</td><td rowspan=2 colspan=1>Overall%%C</td></tr><tr><td rowspan=1 colspan=1>PanelMember ID</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedrresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>High Negatives1</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>59/6(98.3%)</td><td rowspan=1 colspan=1>91.1% -99.7%</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>1.35</td></tr><tr><td rowspan=1 colspan=1>LowPositives</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>59/60(98.3%)</td><td rowspan=1 colspan=1>91.1%-99.7%</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>1.05</td></tr><tr><td rowspan=1 colspan=1>MediumPositives</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>59/60(98.3%)</td><td rowspan=1 colspan=1>91.1%-99.7%</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=1 colspan=1>Positive CControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>3.45</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7%-00%</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>3.88</td></tr><tr><td rowspan=1 colspan=1>PositiveMatrix Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7% -100%</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>1.05</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7%-100%</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.38</td></tr><tr><td rowspan=1 colspan=1>NegativeMatrix Control1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.06</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>2.62</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7% -100%</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>1.80</td></tr><tr><td rowspan=1 colspan=1>TotalAgreementAll</td><td rowspan=1 colspan=3>97/100 (97%)</td><td rowspan=1 colspan=3>100/100 (100%)</td><td rowspan=1 colspan=3>100/100 (100%)</td><td rowspan=1 colspan=1>297/300(99.0%)</td><td rowspan=1 colspan=1>97.1%99.7%</td><td rowspan=1 colspan=2></td></tr></table>

1 Average Ct value is calculated for the Internal Control (IC).

An additional reproducibility study was performed to assess samples that were at an intermediate concentration, below the assay’s LoD but above the “high negatives” tested during the original reproducibility study. The percent positive for the intermediate member across all sites was $4 2 . 2 \%$ . This result was expected as the intermediate concentration should be positive in the range of $5 - 9 5 \%$ and the samples were lower in concentration than the LoD concentration $( \geq 9 5 \%$ positive) and higher than the “high negative” concentration $( < 5 \%$ positive).

<table><tr><td rowspan=2 colspan=1>PanelMember ID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=1>Total</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td><td rowspan=2 colspan=1>OverallAverage CTValue</td><td rowspan=2 colspan=1>Overall%CV</td></tr><tr><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwithexpectedresult</td><td rowspan=1 colspan=1>AVE CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agreementwith expectedresult (%)</td></tr><tr><td rowspan=1 colspan=1>Intermediate</td><td rowspan=1 colspan=1>13/30*</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>12/30*</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>3.50</td><td rowspan=1 colspan=1>13/30*</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>38/90**(42.2%)</td><td rowspan=1 colspan=1>32.5%-52.5%</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>2.55</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>2.64</td><td rowspan=1 colspan=1>30/30(1100%)</td><td rowspan=1 colspan=1>88.7%-100%</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>1.88</td></tr><tr><td rowspan=1 colspan=1>PositiveMatrix Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7%-00%</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>1.00</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7%-100%</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>1.28</td></tr><tr><td rowspan=1 colspan=1>NegativeMatrix Control1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>1.43</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.7%-100%</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>1.55</td></tr></table>

\* Number positive 1 Average Ct value is calculated for the Internal Control (IC).

# Analytical Sensitivity

The analytical sensitivity (limit of detection or LoD) of the ProGastro Cd Assay was determined using 10-fold serial dilutions of two C. diff strains that were spiked into raw stool and processed according to the Instructions for Use. The LoD concentration of each bacterial strain was confirmed by extracting and testing 20 replicates. Portions of the same serial dilutions were cultured and quantified in units of CFU/mL and used to determine CFU/reaction. Analytical sensitivity (LoD) was defined as the lowest concentration at which $\ge 9 5 \%$ of all replicates tested positive.

<table><tr><td rowspan=1 colspan=1>Bacterial Strain</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>NAP1, Ribotype 027, Toxinotype III (Tox A+/B+), clinicalisolate</td><td rowspan=1 colspan=1>9 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>ATCC 43598, Strain 1470, Toxinotype VIII, (Tox A-/B+)</td><td rowspan=1 colspan=1>29 CFU/reaction</td></tr></table>

In addition, the analytical sensitivity was determined using quantified genomic DNA that was serially diluted and run in PCR according to the Instructions for Use. Analytical sensitivity of the ProGastro Cd Assay is 5 genomic copies/reaction.

# Reactivity

In addition to the strains used in the Analytical Sensitivity (LoD) Study, an additional 35 well characterized isolates representing at least 25 different strains, five different toxinotypes, nine different restriction enzyme analysis (REA) types, five different serogroups, and three different ribotypes were used to evaluate the reactivity of the ProGastro Cd Assay. The strains were tested in triplicate at a concentration of 3 times each strain's LoD and all strains were reactive with the ProGastro Cd Assay.

# Analytical Specificity

The analytical specificity of the ProGastro Cd Assay was evaluated by testing a panel of 50 cultures consisting of 41 bacteria, 8 viruses, and one yeast strain representing common gastrointestinal pathogens or microorganisms. Bacteria and yeast were tested at concentrations of $1 . 3 \bar { 5 } \mathrm { ~ x ~ } 1 0 ^ { 8 }$ organisms $\mathrm { \Delta } m \mathrm { L }$ . Viruses were tested at concentrations of $1 0 ^ { 3 . 5 }$ to $1 0 ^ { 6 . 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The cultures were spiked with Internal Control and were extracted using the bioMérieux NucliSENS easyMAG and tested in triplicate. The ProGastro Cd Assay did not cross react with any of the organisms tested except the toxigenic Loyola 02 NAP1 strain.

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">ProGastro CdResult</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic ATCC 43593)</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic ATCC 43601)</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic ATCC 43603)</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic ATCC 700057)</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (non-toxigenic Strain 3232)</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (toxigenic Loyola 02 NAP1strain)</td><td colspan="1" rowspan="1">3.3 x 10-4McFarland (1 x 105 orgs/mL)</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Clostridium haemolyticum</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">0.05 McFarland*</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli 0157:H7</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Porphyromonas asaccharolyctica</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella typhimurium</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermis</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus 18</td><td colspan="1" rowspan="1">1x103.5 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus 40</td><td colspan="1" rowspan="1">1x105.5 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="1" rowspan="1">1x104.5TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">1x104.25 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1x106.5 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">1x103.5 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">Clinical isolate**</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">1x103.5 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">0.45 McFarland</td><td colspan="1" rowspan="1">-</td></tr></table>

† Bacteria and fungus tested at 0.05 or 0.45 McFarland $\overline { { ( 1 . 5 \mathrm { ~ x ~ } 1 0 ^ { 7 } \mathrm { { o r g s } / \mathrm { { m L } } } } }$ or $\overline { { 1 . 3 5 \mathrm { ~ x ~ } 1 0 ^ { 8 } \mathrm { ~ o r g s / m L } } }$ . Viruses tested at the highest concentration available in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . \* Higher concentrations of Clostridium sordellii exhibited inhibition of the Internal Control. No cross-reaction in the FAM channel was observed at 0.45 McFarland. \*\* Cultured and titered Norovirus was unavailable; nucleic acids from a positive clinical sample (Milwaukee Public Health Lab Real Time PCR assay with a $\mathrm { C _ { T } }$ value $= 2 0 . 5$ ) were tested.

# Interference

Blood, mucin, fecal fats, over the counter and prescription medicines (Kaopectate, Immodium AD, Pepto Bismol, Vancomycin, and Metronidazole) and a radiological contrast agent (barium sulfate) were spiked into aliquots of a raw fecal pool along with one cultured and titered strain of $C .$ . difficile (C. diff). C. diff was spiked in at a concentration of 2 times the assay’s Limit of Detection (LoD). Clinically relevant amounts of the potential inhibiting substances were added to spiked samples. Samples were clarified and spiked with the Internal DNA Control (IC). Nucleic acid from the samples was extracted with the bioMérieux NucliSENS easyMAG. The nucleic acids were tested in triplicate reactions with ProGastro Cd on the Cepheid SmartCycler II. The following table shows the potential interfering substances used for this study.

<table><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration Tested</td></tr><tr><td rowspan=1 colspan=1>Whole Blood</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40% (volume/volume)</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>3.5% (weight/volume)</td></tr><tr><td rowspan=1 colspan=1>Fecal Fats(stearic and palmitic acid)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>40% (weight/volume)</td></tr><tr><td rowspan=1 colspan=1>Kaopectate</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>10% (volume/volume)</td></tr><tr><td rowspan=1 colspan=1>Pepto Bismol</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>10% (volume/volume)</td></tr><tr><td rowspan=1 colspan=1>Immodium AD</td><td rowspan=1 colspan=1>Loperamide HCl</td><td rowspan=1 colspan=1>10% (volume/volume)</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>12.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>12.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Barium Sulfate</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5% (weight/volume)</td></tr></table>

None of the potentially interfering exogenous or endogenous substances interfered with detection of $C .$ . diff at a concentration near the LoD of the ProGastro Cd Assay.

# Carry-over/Cross-contamination

To evaluate the level of carry-over/cross-contamination with the ProGastro Cd Assay, simulated Clostridium difficile (C. diff) high positive samples run in series alternating with $C .$ diff high negative samples were tested over five separate extraction and PCR runs. High Positive samples represented the upper range of Ct values obtained in the ProGastro Cd clinical trials (lowest $\mathrm { C t } = 2 5 . 4 )$ . High Negative samples consisted of negative stool spiked with a concentration of $C .$ . diff below but near the analytical LoD concentration such that $> 9 5 \%$ of samples should be negative. The samples were processed and extracted in a “High Positive/High Negative” alternating fashion on the bioMérieux NucliSENS easyMAG and likewise processed and run on the Cepheid SmartCycler II instrument in an alternating fashion. Three out of fifty-five high negative samples tested showed potential $C .$ . diff contamination. Due to the fact that the high negative samples did include a low amount of $C .$ . diff that is detectable no more than $5 \%$ of the time, it cannot be ruled out that these results may actually fall in this category. Potential contamination could have occurred during sample preparation, during extraction, during transferring of the purified nucleic acid from the sample vessel into the microfuge tube, or during preparation of the PCR reactions.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.