# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K090824   
B. Analyte: Chlamydia trachomatis DNA   
C. Type of Test: Strand displacement nucleic acid amplification assay   
D. Applicant: Becton, Dickinson and Company   
E. Proprietary and Established Names:   
BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay

# F. Regulatory Information:

1. Regulation section: 866.3120   
2. Classification: I   
3. Product Code: MKZ   
4. Panel: Microbiology 083

# G. Intended Use:

1. Intended use(s):

The BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { x }$ Amplified DNA Assay, when tested with the BD Viper™ System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Chlamydia trachomatis

DNA in clinician-collected female endocervical and male urethral swab specimens, patient–collected vaginal swab specimens (in a clinical setting), and male and female urine specimens (both UPT and Neat). The assay is also intended for use with gynecological specimens collected in PreservCyt® Solution using an aliquot that is removed prior to processing for additional gynecological testing. The assay is indicated for use with asymptomatic and symptomatic individuals to aid in the diagnosis of chlamydial urogenital disease.

2. Indication(s) for use: Same as intended use

3. Special condition for use statement(s): NA

4. Special instrument Requirements: BD Viper System

# H. Device Description:

The BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe (8, 9). The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, pre-warmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermallycontrolled fluorescent readers. The presence or absence of $C .$ . trachomatis DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified $C .$ . trachomatis target DNA, a second fluorescently-labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the $C .$ . trachomatis-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and $C .$ . trachomatis-specific signals to report specimen results as positive, negative, or EC failure.

# I. Substantial Equivalence Information:

1. Predicate device name(s): BD ProbeTecET CT/GC Amplified DNA Assay, APTIMA Combo 2 Assay   
2. Predicate K number(s): K984631, K003395   
3. Comparison with predicate:

Table 1 Comparison of Operating and Technological Characteristics: Assay   

<table><tr><td rowspan=1 colspan=1>BD ProbeTecCTQ Assay</td><td rowspan=1 colspan=2>BD ProbeTecET CT/GC Assay(K984361)</td></tr><tr><td rowspan=1 colspan=1>Amplification Technology</td><td rowspan=1 colspan=1>Same as BD ProbeTecETCT/GC (K984631)</td><td rowspan=1 colspan=1>Strand DisplacementAmplification</td></tr><tr><td rowspan=1 colspan=3>Priming Microwell</td></tr><tr><td rowspan=1 colspan=1>Primers</td><td rowspan=1 colspan=1>Alternate region of cryptic CTplasmid</td><td rowspan=1 colspan=1>Region of cryptic CTplasmid</td></tr><tr><td rowspan=1 colspan=1>Detector</td><td rowspan=1 colspan=1>Linear Detector (Figure 1B)Flourescein (fluorophore)Dabcyl (quencher)</td><td rowspan=1 colspan=1>Hairpin Detector (Figure 1A)Flourescein (fluorophore)Rhodamine (quencher)</td></tr><tr><td rowspan=1 colspan=1>Nucleotides</td><td rowspan=1 colspan=1>4 of 4 nucleotides required forSDA</td><td rowspan=1 colspan=1>1 of 4 nucleotides required forSDA</td></tr><tr><td rowspan=1 colspan=1>Non-specific reagents andcofactors</td><td rowspan=1 colspan=1>Same as BD ProbeTecETCT/GC (K984631)</td><td rowspan=1 colspan=1>Buffering components,magnesium ions, salt andstabilizing reagents</td></tr><tr><td rowspan=1 colspan=3>Amplification Microwell</td></tr><tr><td rowspan=1 colspan=1>Restriction Enzyme</td><td rowspan=1 colspan=1>Same as BD ProbeTecETCT/GC (K984631)</td><td rowspan=1 colspan=1>BsoBI restriction enzyme</td></tr><tr><td rowspan=1 colspan=1>Polymerase</td><td rowspan=1 colspan=1>Same as BD ProbeTecETCT/GC (K984631)</td><td rowspan=1 colspan=1>Bst DNA polymerase</td></tr><tr><td rowspan=1 colspan=1>Nucleotides</td><td rowspan=1 colspan=1>0 of 4 nucleotides required forSDA</td><td rowspan=1 colspan=1>3 of 4 nucleotides required forSDA</td></tr><tr><td rowspan=1 colspan=1>Non-specific reagents andcofactors</td><td rowspan=1 colspan=1>Same as BD ProbeTecETCT/GC (K984631)</td><td rowspan=1 colspan=1>Buffering components,magnesium ions, salt andstabilizing reagents</td></tr><tr><td rowspan=1 colspan=1>Assay Buffer</td><td rowspan=1 colspan=1>Bicine-potassium hydroxide-based</td><td rowspan=1 colspan=1>Potassium phosphate-based</td></tr></table>

# J. Standard/Guidance Document Referenced (if applicable): NA

K. Test Principle: See Device Description

L. Performance Characteristics (if/when applicable): 1. Analytical performance:

# CT ${ \bf Q } ^ { \bf X }$ Assay Analytical Sensitivity:

The Limits of Detection (LoDs) for the CT $\mathrm { Q } ^ { \mathrm { X } }$ Assay with $C .$ trachomatis serovar H in urine and swab specimens when extracted on the BD Viper System were determined to be $< 1 5$ CT elementary bodies (EB) per mL for neat and $\mathrm { Q } ^ { x }$ UPT urine and $< 3 0$ CT EB per mL for expressed vaginal and endocervical swab, and PreservCyt specimens. A correlation of EB to Inclusion-forming units (IFU) suggests that the CT $\mathrm { Q } ^ { \bf X }$ assay LoDs with serovar H in urine and swab specimens correspond to $< 1$ IFU per mL (15). The CT $\mathrm { Q } ^ { \bf X }$ Assay on the BD Viper System in extracted mode was able to detect 16 isolates representing $1 5 \mathrm { { C T } }$ serovars (A, B, Ba, C, D, E $( 2 ) ^ { * }$ , F, G, H, I, J, K, LGV1, LGV2, and LGV3) with $\ge 9 5 \%$ proportion positive at a concentration of 15 EB per mL in CT/GC $\mathrm { Q } ^ { \bf X }$ Swab Diluent.

\* Testing with CT serovar E included the nvCT strain, a new variant with a deletion in the cryptic plasmid. (21)

# CT ${ \bf Q } ^ { \bf X }$ Assay Analytical Specificity:

DNA from 141 organisms listed in Table 2 was extracted on the BD Viper System and tested with the BD ProbeTecCT $\mathrm { Q } ^ { x }$ Amplified DNA Assay. All potential cross-reactive species were tested a $\mathrm { \cdot \geq 1 x 1 0 ^ { 8 } c e l l s / m L }$ except where noted. The CT $\mathrm { Q } ^ { x }$ Assay did not cross-react with any of the organisms tested.

Table 2: Potential Cross-reacting Microorganisms.   

<table><tr><td colspan="1" rowspan="1">Acinetobactercalcoaceticus</td><td colspan="1" rowspan="1">Epstein Barr Virus***</td><td colspan="1" rowspan="1">Peptostreptococcusproductus</td><td colspan="1" rowspan="1">Neisseria elongatasubsp.nitroreduscens (2)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacterlwoffi</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Plesiomonasshigelloides</td><td colspan="1" rowspan="1">Neisseria elongata</td></tr><tr><td colspan="1" rowspan="1">Actinomycesisraelii</td><td colspan="1" rowspan="1">Flavobacteriummeningosepticum</td><td colspan="1" rowspan="1">Propionibacteriumacnes</td><td colspan="1" rowspan="1">Neisseria flava (4)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus***</td><td colspan="1" rowspan="1">Gardnerellavaginalis</td><td colspan="1" rowspan="1">Providenciastuartii</td><td colspan="1" rowspan="1">Neisseriaflavescens (4)</td></tr><tr><td colspan="1" rowspan="1">Aeromonashydrophilia</td><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Neisseriagonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Alcaligenesfaecalis*</td><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">Salmonellaminnesota</td><td colspan="1" rowspan="1">Neisseria lactamica(7)</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis*</td><td colspan="1" rowspan="1">Herpes SimplexVirus **</td><td colspan="1" rowspan="1">Salmonellatyphimurium</td><td colspan="1" rowspan="1">Neisseriameningitidis (12)</td></tr><tr><td colspan="1" rowspan="1">Bacteroidesfragilis</td><td colspan="1" rowspan="1">Humanpapillomavirus (16and 18)***</td><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">Neisseria mucosa(5)</td></tr><tr><td colspan="1" rowspan="1">Candida albicans*</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Neisseria perflava(8)</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata*</td><td colspan="1" rowspan="1">Klebsiellapneumoniae</td><td colspan="1" rowspan="1">Streptococcusagalactiae</td><td colspan="1" rowspan="1">Neisseriapolysaccharea (2)</td></tr><tr><td colspan="1" rowspan="1">Candidatropicalis*</td><td colspan="1" rowspan="1">Lactobacillusacidophilus*</td><td colspan="1" rowspan="1">Streptococcus mitis</td><td colspan="1" rowspan="1">Neisseria sicca (5)</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapneumoniae****</td><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Streptococcusmutans</td><td colspan="1" rowspan="1">Neisseria subflava(15)</td></tr><tr><td colspan="1" rowspan="1">Chlamydiapsittaci*</td><td colspan="1" rowspan="1">Lactobacillusjensenii*</td><td colspan="1" rowspan="1">Streptococcuspneumoniae*</td><td colspan="1" rowspan="1">Neisseria weaverii(3)</td></tr><tr><td colspan="1" rowspan="1">Citrobacterfreundii</td><td colspan="1" rowspan="1">Listeriamonocytogenes</td><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="9"></td></tr><tr><td colspan="1" rowspan="1">Clostridiumperfringens</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">Streptomycesgriseus *</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumrenale</td><td colspan="1" rowspan="1">Moraxella lacunata*</td><td colspan="1" rowspan="1">Trichomonasvaginalis **</td></tr><tr><td colspan="1" rowspan="1">Cryptococcusneoformans *</td><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus**</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Vibrioparahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Mycobacteriumgordonae</td><td colspan="1" rowspan="1">Yersiniaenterocolitica</td></tr><tr><td colspan="1" rowspan="1">Enterobactercloacae</td><td colspan="1" rowspan="1">Mycobacteriumsmegmatis</td><td colspan="1" rowspan="1">Branhamellacatarrhalis (5)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecalis</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">Neisseria cinerea(2)</td></tr><tr><td colspan="1" rowspan="1">Enterococcusfaecium</td><td colspan="1" rowspan="1">Peptostreptococcusasaccharolyticus</td><td colspan="1" rowspan="1">Neisseria elongatasubsp. glycolytica</td></tr></table>

(n) number of strains tested in the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay\* Tested at $\mathrm { \varepsilon } > 1 \mathrm { x } 1 0 ^ { 7 }$ cells or EB per mL; \*\*Tested at $> 1 \mathrm { x } 1 0 ^ { 6 }$ cells or viral particles per mL; $^ { \ast \ast \ast }$ Tested at $\geq 1 \mathrm { x } 1 0 ^ { 6 }$ genomic equivalents per mL;\*\*\*\* tested $\mathrm { 1 t \geq 1 x 1 0 ^ { 5 } T C I D _ { 5 0 } / m L }$

# CT $\mathbf { Q } ^ { \mathbf { x } }$ Interfering Substances

The performance of the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode was evaluated in the presence of potential interfering substances which may be encountered in swab, urine and/or PreservCyt specimens. Potential interfering substances were spiked into $\mathrm { Q } ^ { x } \mathrm { U P T }$ urine and vaginal swab specimen matrices as well as PreservCyt specimens in LBC Specimen Dilution Tubes, in both the presence and the absence of CT elementary bodies (30 CT $\mathrm { E B / m L }$ in urine matrix and $9 0 \mathrm { C T E B / m L }$ in swab/LBC Specimen Dilution Tube matrix). Results are summarized in Table 3.

Table 3: CT $\mathbf { Q } ^ { x }$ Interfering Substances.   

<table><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>PreservCyt</td></tr><tr><td rowspan=1 colspan=1>NoInterferenceObserved</td><td rowspan=1 colspan=1>Blood (≤ 60%)Seminal FluidMucusOver The Countervaginal products andcontraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 cells/mLNeisseriagonorrhoeae</td><td rowspan=1 colspan=1>Blood (≤1%)Seminal fluidMucusAntibioticsAnalgesicsPhenazopyridineOver The Counterdeodorant sprays andpowdersHormonesLeukocytesAlbumin &lt;1 mg/mLGlucoseAcidic urine (pH 4.0)Alkaline urine (pH9.0)Bilirubin1x106 cells/mLNeisseria gonorrhoeaeOrganisms associatedwith Urinary TractInfections</td><td rowspan=1 colspan=1>Blood (≤ 1%)Seminal FluidMucusOver The Countervaginal products andcontraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 cells/mLNeisseriagonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>May causeextractioncontrol (EC)failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Glacial Acetic Acid +Blood (≤5%/1% V/V)</td></tr><tr><td rowspan=1 colspan=1>May causeFalse Negativeresults</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Glacial Acetic Acid +Blood (≤5%/1% V/V)</td></tr></table>

# Neat and $\mathbf { Q } ^ { x }$ UPT Urine Stability

Pools of CT negative male and female urine specimens were used in analytical experiments to support the urine storage and transport stability claims.

For neat urine, pools were co-spiked with CT serovar H and GC strain ATCC 19424 at 45 EB per mL and 150 cells per mL, respectively. Neat urine specimens were stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 8, 24 or $3 0 \mathrm { h }$ ; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

For $\mathrm { Q } ^ { x }$ UPT urine, pooled specimens were co-spiked with CT serovar H and GC strain ATCC 19424 at 45 EB per mL and 150 cells per $\mathrm { m L }$ , respectively. The spiked urine specimen pools were then stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for $2 4 \mathrm { h }$ or $3 0 ^ { \circ } \mathrm { C }$ for $^ { 8 \mathrm { ~ h ~ } }$ prior to transfer into $\mathrm { Q } ^ { \mathrm { x } }$ UPT tubes. The $\mathrm { Q } ^ { x }$ UPT specimen pools were then stored either at $2 { - } 8 ^ { \mathrm { { \circ } } } \mathrm { C }$ for 14, 21 or 30 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 14, 21 or 30 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 60 days. At each time point $\mathrm { Q } ^ { x }$ UPT specimens were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Vaginal Dry and Expressed Swab Stability

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for dry vaginal swab specimens. Pools were cospiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively, when seeded onto swabs and expressed in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent. Seeded dry swabs were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3, 7, or 14 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 3, 7 or 14 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 30 or 60 days. At each time point, dry swabs were removed from storage and expressed into $2 \mathrm { m L }$ of CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent and evaluated with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for expressed vaginal swab specimens. Pools were spiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per $\mathrm { m L }$ , respectively. The spiked swab matrix was stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30 or 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Endocervical and Urethral Swab Specimen Stability

Pools of CT negative endocervical swab matrix were used in analytical experiments to support the storage and transport stability claims for endocervical and urethral swab specimens. Pools of swab matrix were spiked with CT serovar H and GC strain ATCC 19424 at 90 EB per mL and 300 cells per mL, respectively. The pools were dispensed in

$2 \mathrm { m L }$ volumes into BD sample tubes to simulate “wet” endocervical specimens and stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30 or 60 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample   
type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Post Pre-warm Specimen Stability

Pools of male and female CT negative neat urine were used in analytical experiments to support the storage stability claims for pre-warmed neat and $\mathrm { Q } ^ { x }$ UPT urine specimens. Pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at $4 5 \mathrm { E B }$ per mL and 150 cells per mL, respectively and either added to $\mathrm { Q } ^ { x }$ UPT tubes or left untreated as neat urine. Both specimen types were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ , and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, specimen tubes were stored at either 2- $8 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $- 2 0 ^ { \circ } \mathrm { C }$ for 30 days. At each time point samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal and endocervical swab specimen matrices in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent were used in analytical experiments to support the storage stability claims for pre-warmed expressed vaginal, endocervical, and male urethral swab specimens. For both types of matrix, pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at $9 0 \mathrm { E B }$ per mL and 300 cells per mL, respectively and aliquotted into 2 mL volumes in BD specimen tubes. The tubes were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, the specimen tubes were stored either at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3 or 7 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 3 or 7 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30 days. At each time point, samples were removed from storage and tested with the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# PreservCyt Specimen Stability

Pools of CT and GC negative PreservCyt clinical specimens were used in analytical experiments to support the storage and stability claims. Pools were co-spiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively. The pools were dispensed in $2 0 ~ \mathrm { m L }$ volumes in PreservCyt vials and stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ or $3 0 ^ { \circ } \mathrm { C }$ . After 30 days, $0 . 5 \mathrm { m L }$ from each vial was removed and added to an LBC Specimen Dilution Tube. The specimens in the LBC Specimen Dilution Tube were then stored at $2 { - } 8 ^ { \mathrm { { o } } } \mathrm { C }$ for 30 or 90 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 30 or 90 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 90 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Twenty-four assay replicates were generated for each condition (temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Reproducibility

Reproducibility of the BD Viper System using the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Simulated endocervical and urethral specimens contained a clean endocervical swab whereas the simulated urine and vaginal swab specimens did not. Uninoculated swab diluent for the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in Table 4.

Table 4: Summary of Reproducibility Data on the BD Viper System for the CT $\mathbf { Q } ^ { x }$ Assay.   

<table><tr><td rowspan=1 colspan=6></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRuns WithinSite</td><td rowspan=1 colspan=2>Between Site</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>CTEB&#x27;s/mL</td><td rowspan=1 colspan=1>GCCells/mL</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MaxRFUMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=5 colspan=1>Endocervical /Urethral</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5%(133/135)</td><td rowspan=1 colspan=1>(94.8-99.8%)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>233.0</td><td rowspan=1 colspan=1>778.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>113.4</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>2011.2</td><td rowspan=1 colspan=1>114.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>442.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>76.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1991.9</td><td rowspan=1 colspan=1>118.0</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1954.8</td><td rowspan=1 colspan=1>169.4</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=5 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>542.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1999.8</td><td rowspan=1 colspan=1>131.8</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>442.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1995.2</td><td rowspan=1 colspan=1>125.8</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>2014.4</td><td rowspan=1 colspan=1>109.5</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr></table>

A second study was conducted internally to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LoD) of the BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into $\mathrm { Q } ^ { \mathrm { x } }$ swab diluent at two different levels each of which was below the respective analytical LoD for the organisms (1:10, 1:100). These levels were selected to fall within the dynamic range of the analytical LOD curve of the assay. Fifteen replicates of each panel member were tested every day for five days across three BD Viper Systems. The data are summarized in Table 5.

Table 5: Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay.   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Dilutionof Analytical LOD</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>Max RFUMean(Positive)</td><td rowspan=1 colspan=1>%Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>Max RFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>70.2(158/225)</td><td rowspan=1 colspan=1>(63.8, 76.1)</td><td rowspan=1 colspan=1>1794.2</td><td rowspan=1 colspan=1>29.8(67/225)</td><td rowspan=1 colspan=1>(23.9, 36.2)</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>Endocervical/UrethralUrine/Vaginal</td><td rowspan=1 colspan=1>1:1001:10</td><td rowspan=1 colspan=1>10.2(23/225)64.4(145/225)</td><td rowspan=1 colspan=1>(6.6,14.9)(57.8, 70.7)</td><td rowspan=1 colspan=1>1643.81733.9</td><td rowspan=1 colspan=1>89.8(202/225)35.6(80/225)</td><td rowspan=1 colspan=1>(85.1, 93.4)(29.3, 42.2)</td><td rowspan=1 colspan=1>1.64.6</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>10.7(24/225)</td><td rowspan=1 colspan=1>(7.0, 15.5)</td><td rowspan=1 colspan=1>1666.6</td><td rowspan=1 colspan=1>89.3(201/225)</td><td rowspan=1 colspan=1>(84.5, 93.0)</td><td rowspan=1 colspan=1>2.4</td></tr></table>

A reproducibility study of the BD Viper System using the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was also conducted for Liquid Based Cytology (LBC) specimens at three clinical sites on one BD Viper System per site. A panel of simulated specimens comprising CT and GC organisms seeded into LBC Specimen Dilution Tubes containing PreservCyt Solution was tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Uninoculated LBC Specimen Dilution Tubes containing PreservCyt Solution were used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in Table 6. Two additional target levels were included in the panels to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LoD) of the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. These additional specimens comprised CT and GC organisms seeded into LBC Specimen Dilution Tubes containing PreservCyt Solution at dilutions of 1:10 and 1:100 of the respective analytical LoDs of each analyte. These levels were selected to fall within the dynamic range of the analytical LoD curves for the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ and GC $\mathrm { Q } ^ { \mathrm { x } }$ assays. Nine replicates of each panel member were tested every day for five days across the three BD Viper Systems. The data are summarized in Table 7.

Table 6: Summary of Reproducibility Data for LBC Specimens on the BD Viper System for the CT $\underline { { \mathbf { O } } } ^ { x }$ Assay.   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunsWithin Site</td><td rowspan=1 colspan=2>Between Site</td></tr><tr><td rowspan=1 colspan=1>CTEBs/mL</td><td rowspan=1 colspan=1>GCCells/mL</td><td rowspan=1 colspan=1>% Correct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanMaxRFU</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3% -100.0%)</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>4.66</td><td rowspan=1 colspan=1>357.64</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>65.29</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>15.12</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3% -00.0%)</td><td rowspan=1 colspan=1>2021.95</td><td rowspan=1 colspan=1>225.94</td><td rowspan=1 colspan=1>11.17</td><td rowspan=1 colspan=1>16.58</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>21.52</td><td rowspan=1 colspan=1>1.06</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3% -100.0%)</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>3.63</td><td rowspan=1 colspan=1>268.97</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>64.48</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3% -00.0%)</td><td rowspan=1 colspan=1>2028.41</td><td rowspan=1 colspan=1>155.45</td><td rowspan=1 colspan=1>7.66</td><td rowspan=1 colspan=1>9.93</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3% -00.0%)</td><td rowspan=1 colspan=1>1964.40</td><td rowspan=1 colspan=1>170.91</td><td rowspan=1 colspan=1>8.70</td><td rowspan=1 colspan=1>44.37</td><td rowspan=1 colspan=1>2.26</td><td rowspan=1 colspan=1>8.70</td><td rowspan=1 colspan=1>0.44</td></tr></table>

Table 7: Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay for LBC Specimens.   

<table><tr><td rowspan=1 colspan=1>Diln ofAnalyticalLOD</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>MaxRFUMean(Positive)</td><td rowspan=1 colspan=1>% Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>MaxRFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>50.4 (68/135)</td><td rowspan=1 colspan=1>(41.6 - 59.1)</td><td rowspan=1 colspan=1>1935.9</td><td rowspan=1 colspan=1>49.6 (67/135)</td><td rowspan=1 colspan=1>(40.9 - 58.4)</td><td rowspan=1 colspan=1>11.5</td></tr><tr><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>7.4 (10/135)</td><td rowspan=1 colspan=1>(3.6 - 13.2)</td><td rowspan=1 colspan=1>1835.7</td><td rowspan=1 colspan=1>92.6 (125/135)</td><td rowspan=1 colspan=1>(86.8 - 96.4)</td><td rowspan=1 colspan=1>9.4</td></tr></table>

# System Cross Contamination and Carryover

An internal study was conducted to evaluate the risk of producing a false positive result in either the same run on the BD Viper System in extracted mode (within run crosscontamination) or in a subsequent run (between run carryover). Testing was conducted using negative and positive samples on three BD Viper Systems. Negative samples consisted of CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent/LBC Specimen Dilution Tube with PreservCyt Solution. Positive samples consisted of a representative analyte (at $1 0 ^ { 5 } \mathrm { C T E B / m L }$ ) spiked into CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent/LBC Specimen Dilution Tube with PreservCyt Solution. The overall rate of cross-contamination (i.e., with alternating columns of positive and negative samples and a prevalence of $50 \%$ ) was $0 . 4 1 \%$ (9/2208). The overall rate of carryover contamination (i.e., carryover between successive runs when the prevalence was $50 \%$ in the previous run) was $0 . 3 6 \%$ (8/2208). Cross-contamination and carryover rates across the three BD Viper Systems are summarized in Tables 8 and 9.

Table 8: Cross Contamination and Carryover Contamination.   

<table><tr><td rowspan=2 colspan=1>AssayDispenseModeSelected</td><td rowspan=2 colspan=1>BD ViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>DualAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.36</td></tr><tr><td rowspan=4 colspan=1>SingleAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td></tr></table>

Table 9: Cross Contamination and Carryover Contamination (PreservCyt).   

<table><tr><td rowspan=2 colspan=1>MediaType</td><td rowspan=2 colspan=1>BD ViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>PreservCytSolution</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.54</td></tr></table>

# M. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.