# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K091724

# B. Purpose for Submission:

To obtain a Substantial equivalence determination for the addition of pre-aliquot samples from gynecological specimens stored in SurePathTM Preservative Fluid as an additional sample type to be tested on the BD Viper™ System in extracted mode previously cleared (K081824).

C. Measurand: Chlamydia trachomatis DNA

# D. Type of Test:

Qualitative Determination of Chlamydia trachomatis DNA using the Strand Displacement Nucleic Acid Amplification technology

# E. Applicant:

Becton, Dickinson and Company

F. Proprietary and Established Names:

BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>MKZ</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>866.3120</td><td rowspan=1 colspan=1>Microbiology</td></tr></table>

# H. Intended Use:

1. Intended use:

The BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { x }$ Amplified DNA Assay, when tested with the BD Viper™ System in Extracted Mode, uses Strand Displacement Amplification technology for the direct, qualitative detection of Chlamydia trachomatis DNA in clinician-collected female endocervical and male urethral swab specimens, patient–collected vaginal swab specimens (in a clinical setting), and male and female urine specimens (both UPT and Neat). The assay is also intended for use with gynecological specimens collected in BD SurePath™

Preservative Fluid using an aliquot that is removed prior to processing for the BD SurePath Pap test. The assay is indicated for use with asymptomatic and symptomatic individuals to aid in the diagnosis of chlamydial urogenital disease.

2. Indications for use: Same as indications for use   
3. Special condition for use statement(s): For Prescription Use Only   
4. Special instrument Requirements: BD Viper System with automated nucleic acid extraction mode

# I. Device Description:

The BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe. The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, prewarmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of $C .$ . trachomatis DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified C. trachomatis target DNA, a second fluorescently-labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the C. trachomatis-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and $C .$ . trachomatis-specific signals to report specimen results as positive, negative, or EC failure.

# J. Substantial Equivalence Information:

1. Predicate device names:

BD ProbeTec™ Chlamydia trachomatis (CT) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay Gen-Probe ACT Assay

2. Predicate K number(s):

3. Comparison with predicate:

# Device Comparison: CTQ Assay Specimen Collection for the BD Viper System in Extracted Mode

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD ProbeTec CTQAssay,PreservCytSolution Specimens(Device)</td><td rowspan=1 colspan=1>BD ProbeTec CTQ Assay,Swab and Urine Specimens(K081824)</td><td rowspan=1 colspan=1>Gen-Probe ACT(K053446)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Same as K081825Gynecological specimenin SurePath PreservativeFluid</td><td rowspan=1 colspan=1>Endocervical swab (females)• Vaginal self-collected swab(in a clinical setting) (females)•Urethral swab (males)Neat urine (female and male)UPT urine (female and male)</td><td rowspan=1 colspan=1>•Endocervical swab (females)Vaginal swab (females)Urethral swab (males)Neat urine (female and male) UTT urine (female and male)• Gynecological specimen inPreservCyt Solution</td></tr><tr><td rowspan=1 colspan=1>SpecimenCollectionandTransportAccessories</td><td rowspan=1 colspan=1>Same as K081824Liquid Based CytologySpecimen (LBC) DilutionTube</td><td rowspan=1 colspan=1>•Endocervical kitUrethral kitVaginal kitUPTNeat urine (Qx Sample Tube)</td><td rowspan=1 colspan=1>• Unisex swab kitVaginal swab kitUrine collection kitSpecimen transfer kit (forgynecological specimen inPreservCyt Solution)</td></tr></table>

Device Comparison: Specimen Collection   

<table><tr><td></td><td>BD ProbeTec CTQ Assay, SurePath Preservative Fluid (Device)</td><td>Gen-Probe ACT (K053446)</td></tr><tr><td>Specimen Collection</td><td>•Gynecological specimen collected and placed in SurePath Preservative Fluid (per TriPath&#x27;s instruction for use). •Sample for CTQ/GCQ testing is drawn from original cytology specimen vial before the specimen is processed for cytology testing.</td><td>• Gynecological specimen collected and placed in PreservCyt Solution (per Cytyc&#x27;s instruction for use). • LBC specimen is first processed for cytology and then an aliquot is drawn from the remaining specimen in the vial for CT/GC testing.</td></tr></table>

Device Comparison: Specimen Processing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD ProbeTec CTQ Assay,SurePath Preservative Fluid (Device)</td><td rowspan=1 colspan=1>BD ProbeTec CTQ Assay,Swab and Urine Specimens(K081824)</td></tr><tr><td rowspan=1 colspan=1>SpecimenProcessing</td><td rowspan=1 colspan=1>•Same as K081824 without the 15 minutepre-warm step for LBC Dilution Tubespecimens• Use of LBC Specimen Rack to prevent pre-warming of LBC specimens</td><td rowspan=1 colspan=1>Pre-warm specimens (swabs and urines) for15 minutes before running BD Viper System</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): Not Applicable

# L. Test Principle:

The BD ProbeTecCT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay is based on the simultaneous amplification and detection of target DNA using amplification primers and a fluorescently-labeled detector probe (8, 9). The reagents for SDA are dried in two separate disposable microwells: the Priming Microwell contains the amplification primers, fluorescently-labeled detector probe, nucleotides and other reagents necessary for amplification, while the Amplification Microwell contains the two enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA. The BD Viper System pipettes a portion of the purified DNA solution from each Extraction Tube into a Priming Microwell to rehydrate the contents. After a brief incubation, the reaction mixture is transferred to a corresponding, prewarmed Amplification Microwell which is sealed to prevent contamination and then incubated in one of the two thermally-controlled fluorescent readers. The presence or absence of $C .$ . trachomatis DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

In addition to the fluorescent probe used to detect amplified C. trachomatis target DNA, a second fluorescently-labeled oligonucleotide is incorporated in each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the C. trachomatis-specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is re-hydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and $C .$ . trachomatis-specific signals to report specimen results as positive, negative, or EC failure.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility

Reproducibility of the BD Viper System using the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Simulated endocervical and urethral specimens contained a clean endocervical swab whereas the simulated urine and vaginal swab specimens did not. Uninoculated swab diluent for the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized on page 20 Table 9.

# $b$ . Linearity/assay reportable range:

Not Applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Control Set for the BD ProbeTec CT/GC $\mathrm { Q } x$ Amplified DNA Assays: 24 CT/GC $\mathrm { Q } x$ Positive Control Tubes containing approximately 2400 copies each of pCTB4 and pGCint3 linearized plasmids in carrier nucleic acid, and 24 CT/GC $\mathrm { Q } x$ Negative Controls Tubes containing carrier nucleic acid alone. The concentrations of the pCTB4 and pGCint3 plasmids are determined by UV spectrophotometry.

# d. Detection limit:

CT $\mathrm { Q } ^ { \mathrm { X } }$ Assay Analytical Sensitivity:

The Limits of Detection (LODs) for the CT $\mathrm { Q } ^ { \mathrm { X } }$ Assay with $C .$ trachomatis serovar H in urine and swab specimens when extracted on the BD Viper System were determined to be $< 1 5$ CT elementary bodies (EB) per mL for neat and $\mathrm { Q } ^ { x }$ UPT urine and $< 3 0$ CT EB per mL for expressed vaginal, endocervical swab, and BD SurePath specimens. A correlation of EB to Inclusion-forming units (IFU) suggests that the CT $\mathrm { Q } ^ { \bf X }$ assay LODs with serovar H in urine and swab specimens correspond to $< 1$ IFU per mL (15). The CT $\mathrm { Q } ^ { \bf X }$ Assay on the BD Viper System in extracted mode was able to detect 16 isolates representing $1 5 \mathrm { { C T } }$ serovars (A, B, Ba, C, D, E $( 2 ) ^ { * }$ , F, G, H, I, J, K, LGV1, LGV2, and LGV3) with $\ge 9 5 \%$ proportion positive at a concentration of 15 EB per mL in CT/GC $\mathrm { Q } ^ { \bf X }$ Swab Diluent.

\* Testing with CT serovar E included both the type strain and the nvCT strain, a new variant with a deletion in the cryptic plasmid.

# e. Analytical specificity:

DNA from 141 organisms listed in Table 1 was extracted on the BD Viper System and tested with the BD ProbeTec CT $\mathrm { Q } ^ { x }$ Amplified DNA Assay. All potential cross-reactive species were tested at $\geq 1 \mathrm { x } 1 \hat { 0 } ^ { 8 }$ cells/mL except where noted. The CT $\mathrm { Q } ^ { x }$ Assay did not cross-react with any of the organisms tested.

Table 1: Potential Cross-reacting Microorganisms.   

<table><tr><td rowspan=1 colspan=1>Acinetobactercalcoaceticus</td><td rowspan=1 colspan=1>Epstein Barr Virus ***</td><td rowspan=1 colspan=1>Peptostreptococcusproductus</td><td rowspan=1 colspan=1>Neisseria elongatasubsp. nitroreduscens(2)</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter lwoffi</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Plesiomonasshigelloides</td><td rowspan=1 colspan=1>Neisseria elongata</td></tr><tr><td rowspan=1 colspan=1>Actinomyces israelii</td><td rowspan=1 colspan=1>Flavobacteriummeningosepticum</td><td rowspan=1 colspan=1>Propionibacteriumacnes</td><td rowspan=1 colspan=1>Neisseria flava (4)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus***</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>Neisseria flavescens (4)</td></tr><tr><td rowspan=1 colspan=1>Aeromonashydrophilia</td><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Pseudomonasaeruginosa</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Alcaligenesfaecalis*</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Salmonellaminnesota</td><td rowspan=1 colspan=1>Neisseria lactamica (7)</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis*</td><td rowspan=1 colspan=1>Herpes Simplex Virus **</td><td rowspan=1 colspan=1>Salmonellatyphimurium</td><td rowspan=1 colspan=1>Neisseria meningitidis(12)</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Human papillomavirus(16 and 18)***</td><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>Neisseria mucosa (5)</td></tr><tr><td rowspan=1 colspan=1>Candida albicans*</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>Neisseria perflava (8)</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata*</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Streptococcusagalactiae</td><td rowspan=1 colspan=1>Neisseriapolysaccharea (2)</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis*</td><td rowspan=1 colspan=1>Lactobacillusacidophilus*</td><td rowspan=1 colspan=1>Streptococcus mitis</td><td rowspan=1 colspan=1>Neisseria sicca (5)</td></tr><tr><td rowspan=1 colspan=1>Chlamydiapneumoniae****</td><td rowspan=1 colspan=1>Lactobacillus brevis</td><td rowspan=1 colspan=1>Streptococcus mutans</td><td rowspan=1 colspan=1>Neisseria subflava (15)</td></tr><tr><td rowspan=1 colspan=1>Chlamydia psittaci*</td><td rowspan=1 colspan=1>Lactobacillus jensenii*</td><td rowspan=1 colspan=1>Streptococcuspneumoniae*</td><td rowspan=1 colspan=1>Neisseria weaverii (3)</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Streptococcuspyogenes</td><td rowspan=9 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Clostridiumperfringens</td><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>Streptomycesgriseus**</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumrenale</td><td rowspan=1 colspan=1>Moraxella lacunata*</td><td rowspan=1 colspan=1>Trichomonasvaginalis **</td></tr><tr><td rowspan=1 colspan=1>Cryptococcusneoformans*</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>Veillonella parvula</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus**</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Vibrioparahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>Mycobacterium gordonae</td><td rowspan=1 colspan=1>Yersiniaenterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterobactercloacae</td><td rowspan=1 colspan=1>Mycobacteriumsmegmatis</td><td rowspan=1 colspan=1>Branhamellacatarrhalis (5)</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecalis</td><td rowspan=1 colspan=1>Peptostreptococcusanaerobius</td><td rowspan=1 colspan=1>Neisseria cinerea (2)</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>Peptostreptococcusasaccharolyticus</td><td rowspan=1 colspan=1>Neisseria elongatasubsp. Glycolytica</td></tr></table>

(n) number of strains tested in the BD ProbeTec CT $\mathbf { Q } ^ { \mathrm { x } }$ Assay

\* Tested at $> 1 \mathrm { x } 1 0 ^ { 7 }$ cells or EB per mL; \*\*Tested at $> \mathrm { 1 x 1 0 } ^ { 6 }$ cells or viral particles per mL; \*\*\*Tested at > $1 \mathrm { x } 1 0 ^ { 6 }$ genomic equivalents per mL; $^ { \ast \ast \ast \ast }$ tested a $\mathrm { t } \geq 1 \mathrm { x } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$

$f .$ Assay cut-off:

The presence or absence of $C .$ . trachomatis DNA is determined by calculating the peak fluorescence (MaxRFU) over the course of the amplification process and by comparing this measurement to a predetermined threshold value. The magnitude of the MaxRFU score is not indicative of the level of organism in the specimen. If the $C .$ trachomatis-specific signal is greater than or equal to a threshold of 125 MaxRFU, the EC fluorescence is ignored by the algorithm. If the $C .$ . trachomatis-specific signal is less than a threshold of 125 MaxRFU, the EC fluorescence is utilized by the algorithm in the interpretation of the result. If assay control results are not as expected, patient results are not reported.

# CT $\mathrm { Q } ^ { \mathrm { x } }$ Interfering Substances

The performance of the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode was evaluated in the presence of potential interfering substances which may be encountered in swab, urine and/or BD SurePath specimens. Potential interfering substances were spiked into $\mathrm { Q } ^ { x }$ UPT urine and vaginal swab specimen matrices as well as BD SurePath specimens in LBC Specimen Dilution Tubes, in both the presence and the absence of CT elementary bodies (30 CT $\mathrm { E B / m L }$ in urine matrix and 90 CT EB/mL in swab/LBC Specimen Dilution Tube matrix). Results are summarized in Table 2.

Table 2: CT $\mathbf { Q } ^ { x }$ Interfering Substances.   

<table><tr><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>SurePath</td></tr><tr><td rowspan=3 colspan=1>No InterferenceObserved</td><td rowspan=1 colspan=1>Blood (≤ 60%)Seminal Fluid</td><td rowspan=3 colspan=1>Blood (≤1%)Seminal fluidMucusAntibioticsAnalgesicsPhenazopyridineOver The Counterdeodorant sprays andpowdersHormonesLeukocytesAlbumin &lt;1 mg/mLGlucoseAcidic urine (pH 4.0)Alkaline urine (pH 9.0)Bilirubin1x106 cells/mL NeisseriagonorrhoeaeOrganisms associated withUrinary Tract Infections</td><td rowspan=3 colspan=1>Blood (≤ 1%)Seminal FluidMucusOver The Counter vaginalproducts and contraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 cells/mL Neisseriagonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>Mucus</td></tr><tr><td rowspan=1 colspan=1>Over The Counter vaginalproducts andcontraceptivesHemorrhoidal creamPrescription vaginaltreatmentsLeukocytes (1x106cells/mL)1x106 cells/mL Neisseriagonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>May causeextraction control(EC) failures</td><td rowspan=1 colspan=1>Blood (&gt; 60%)</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr><tr><td rowspan=1 colspan=1>May cause FalseNegative results</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td><td rowspan=1 colspan=1>Not applicable</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

# Swab and Urine Specimen Clinical Study

Clinician-collected endocervical and male urethral swab specimens, patient-collected vaginal swab specimens (in a clinical setting), and male and female $\mathrm { Q } x \mathrm { U P T }$ and neat urine specimens were collected from 1059 female subjects and 479 male subjects attending OB/GYN, sexually transmitted disease (STD) and family planning clinics at seven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, urethral discharge, coital pain/difficulty/bleeding, testicular or scrotum pain/swelling, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms. Sixty five female subjects and 7 male subjects were excluded from the data analysis due to age requirement violations, antibiotic treatment in the last 21 days, opting to withdraw from the study after initially consenting, failure to obtain paired swab and urine specimens, urine quantity less than $2 0 ~ \mathrm { m L }$ , or transport and storage errors related to specimen collection. Therefore, the final data analysis included 994 compliant female subjects and 472 compliant male subjects. Five specimens were collected from each of the 994 eligible female subjects. A urine specimen was collected and split into $\mathrm { Q } x$ UPT, neat urine and the two reference urine specimen collection devices followed by a vaginal swab specimen and three randomized endocervical swab specimens. Up to four specimens were collected from each of the 472 eligible male subjects. Up to three randomized urethral swab specimens were collected followed by a urine specimen that was split into $\mathrm { Q } x$ UPT, neat urine and the two reference urine specimen collection devices. BD ProbeTec CT $\mathrm { Q } x$ assay results were generated from the $\mathrm { Q } x$ UPT and neat urine specimens, the vaginal swab specimen, one endocervical swab specimen and one male urethral swab specimen. The remaining two endocervical swab specimens, up to two male urethral swab specimens, and the two reference urine specimens for each male and female subject were tested using two reference methods: the BD ProbeTec ET CT/AC assay and another commercially available NAAT (Nucleic Acid Amplification Test). Specimen testing was conducted either at the site of specimen collection or at a designated BD Viper testing site. All performance calculations were based on the total number of BD ProbeTec CT $\mathrm { Q } x$ assays results for endocervical, vaginal and male urethral swab specimens, and male and female $\mathrm { Q } x$ UPT and neat urine specimens compared to a patient infected status (PIS) algorithm for each gender. In the algorithm, the designation of a subject as being infected with CT or not was based on endocervical swab and urine specimen results from the commercially available BD ProbeTec ET CT/AC assay and the other commercially available NAAT. Subjects were considered infected with CT if two of the four endocervical swab and urine

specimens (or two of the three or four urethral swab and urine specimens) tested positive in the BD ProbeTec ET CT/AC assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 5388 BD ProbeTec CT $\mathrm { Q } x$ assay results was used to calculate sensitivity and specificity. Performance of the assay with endocervical swabs, patient collected vaginal swab specimens (in a clinical setting), female UPT and neat urine was assessed in the clinical study. Separate performance was calculated for specimens collected from pregnant females. For the latter, sensitivity compared to patient infected status for FS was $6 2 . 5 \%$ (5/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FV sensitivity was $7 5 \%$ (6/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FNU and FUPT sensitivity were $100 \%$ (8/8). Specificity was $9 4 . 7 \%$ (18/19) for FS, FV, FNU, and FUPT separately.

# a. Clinical Sensitivity:

Sensitivity compared to patient infected status for FS was $6 2 . 5 \% ( 5 / 8 )$ : the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FV sensitivity was $7 5 \%$ (6/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FNU and FUPT sensitivity were $100 \%$ (8/8).

# Neat and $\mathbf { Q } ^ { x }$ UPT Urine Stability

Pools of CT negative male and female urine specimens were used in analytical experiments to support the urine storage and transport stability claims. For neat urine, pools were co-spiked with CT serovar H and GC strain ATCC 19424 at 45 EB per mL and 150 cells per mL, respectively. Neat urine specimens were stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 8, 24 or $3 0 \mathrm { h }$ ; or at $- 2 0 ^ { \circ } \mathrm { C }$ for 180 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

For $\mathrm { Q } ^ { x } \mathrm { U P T }$ urine, pooled specimens were co-spiked with CT serovar H and GC strain ATCC 19424 at $4 5 \mathrm { E B }$ per mL and 150 cells per mL, respectively. The spiked urine specimen pools were then stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for $2 4 \mathrm { h }$ or $3 0 ^ { \circ } \mathrm { C }$ for $^ { 8 \mathrm { ~ h ~ } }$ prior to transfer into $\mathrm { Q } ^ { \mathrm { x } }$ UPT tubes. The $\mathrm { Q } ^ { x }$ UPT specimen pools were then stored either at 2- $8 ^ { \circ } \mathrm { C }$ for 14, 21 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 14, 21 or 30 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 180 days. At each time point $\mathrm { Q } ^ { x }$ UPT specimens were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Vaginal Dry and Expressed Swab Stability

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for dry vaginal swab specimens. Pools were co-spiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively, when seeded onto swabs and expressed in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent. Seeded dry swabs were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3, 7, or 14 days; or at $3 0 \mathrm { { ^ \circ C } }$ for 3, 7 or 14 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30, 60, or 180 days. At each time point, dry swabs were removed from storage and expressed into $2 \mathrm { m L }$ of CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent and evaluated with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal swab matrix were used in analytical experiments to support the storage and transport stability claims for expressed vaginal swab specimens. Pools were spiked with CT serovar H and GC strain ATCC 19424 to achieve $9 0 \mathrm { E B }$ per mL and 300 cells per mL, respectively. The spiked swab matrix was stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \circ } \mathrm { C }$ for 30, 60, or 180 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Endocervical and Urethral Swab Specimen Stability

Pools of CT negative endocervical swab matrix were used in analytical experiments to support the storage and transport stability claims for endocervical and urethral swab specimens. Pools of swab matrix were spiked with CT serovar H and GC strain ATCC 19424 at $9 0 \mathrm { E B }$ per mL and 300 cells per mL, respectively. The pools were dispensed in $2 \mathrm { m L }$ volumes into BD sample tubes to simulate “wet” endocervical specimens and stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 7, 14 or 30 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30, 60, or 180 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# Post Pre-warm Specimen Stability

Pools of male and female CT negative neat urine were used in analytical experiments to support the storage stability claims for pre-warmed neat and $\mathrm { Q } ^ { x }$ UPT urine specimens. Pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at 45 EB per mL and 150 cells per mL, respectively and either added to $\mathrm { Q } ^ { x } \mathrm { U P T }$ tubes or left untreated as neat urine. Both specimen types were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { m i n }$ , and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, specimen tubes were stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 1, 3 or 7 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 30 or 180 days. At each time point samples were removed from storage and tested

with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Pools of CT negative vaginal and endocervical swab specimen matrices in CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent were used in analytical experiments to support the storage stability claims for pre-warmed expressed vaginal, endocervical, and male urethral swab specimens. For both types of matrix, pooled specimens were spiked with CT serovar H and GC strain ATCC 19424 at 90 EB per mL and 300 cells per mL, respectively and aliquotted into $2 \mathrm { m L }$ volumes in BD specimen tubes. The tubes were pre-warmed at $1 1 4 ^ { \circ } \mathrm { C }$ for $1 5 \mathrm { { m i n } }$ and cooled for $1 5 \mathrm { { m i n } }$ . After the pre-warm process, the specimen tubes were stored either at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 3 or 7 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 3 or 7 days; or at $- 2 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ for 30 or 180 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Thirty-two assay replicates were generated for each condition (sample type/temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

# BD SurePath Specimen Stability

Pools of CT and GC negative BD SurePath clinical specimens were used in analytical experiments to support the storage and stability claims. Pools were co-spiked with CT serovar H and GC strain ATCC 19424 to achieve 90 EB per mL and 300 cells per mL, respectively. The pools were dispensed in $1 0 ~ \mathrm { m L }$ volumes in BD SurePath vials and stored at either $2 { - } 8 ^ { \circ } \mathrm { C }$ or $3 0 \mathrm { { ^ \circ C } }$ . After 30 days, $0 . 5 \mathrm { m L }$ from each vial was removed and added to an LBC Specimen Dilution Tube. The specimens in the LBC Specimen Dilution Tube were then stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ for 30 or 90 days; or at $3 0 ^ { \circ } \mathrm { C }$ for 30 or 90 days; or at $- 2 0 ^ { \mathrm { o } } \mathrm { C }$ for 90 days. At each time point, samples were removed from storage and tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay on the BD Viper System in extracted mode. Twenty-four assay replicates were generated for each condition (temperature/duration). The expected results were obtained with the CT $\mathrm { Q } ^ { \mathrm { x } }$ assay under all conditions tested.

Hypothetical positive and negative predictive values (PPV & NPV) for the CT $\mathrm { Q } x$ Assay with swab and urine specimens are shown in Table 3A. Hypothetical positive and negative predictive values (PPV & NPV) for the CT $\mathrm { Q } x$ Assay from the multicenter clinical trial for BD SurePath specimens are shown in Table 3B. Hypothetical positive and negative predictive values (PPV & NPV) for the CT $\mathrm { Q } x$ Assay from the multi-center clinical trial for PreservCyt specimens are shown in Table 3C. These calculations are based on hypothetical prevalence and overall sensitivity and specificity (compared to the patient infected status) of $9 4 . 5 \%$ and $9 8 . 9 \%$ for swab and urine specimens, of $9 5 . 0 \%$ and $9 9 . 7 \%$ for BD SurePath specimens, and of $9 4 . 1 \%$ and $9 9 . 8 \%$ for PreservCyt specimens.

Table 3A: CT Hypothetical Positive and Negative Predictive Values (Swabs/Urines) Compared to Patient Infected Status.   

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Sensitivity(%)</td><td rowspan=1 colspan=1>Specificity(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>64.1</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>82.1</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>90.7</td><td rowspan=1 colspan=1>99.4</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>95.6</td><td rowspan=1 colspan=1>98.6</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>97.4</td><td rowspan=1 colspan=1>97.7</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.3</td><td rowspan=1 colspan=1>96.4</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>94.7</td></tr></table>

Table 3B: CT Hypothetical Positive and Negative Predictive Values (BD SurePath) Compared to Patient Infected Status.   

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Sensitivity(%)</td><td rowspan=1 colspan=1>Specificity(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>86.6</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=2 colspan=1>94.397.2</td><td rowspan=2 colspan=1>99.799.4</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.8</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.3</td><td rowspan=3 colspan=1>97.996.895.2</td></tr><tr><td rowspan=2 colspan=1>4050</td><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=1>95.0</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>99.7</td></tr></table>

Table 3C: CT Hypothetical Positive and Negative Predictive Values (PreservCyt) Compared to Patient Infected Status.   

<table><tr><td rowspan=1 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=1>Sensitivity(%)</td><td rowspan=1 colspan=1>Specificity(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=2 colspan=1>25</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>90.6</td><td rowspan=2 colspan=1>99.999.7</td></tr><tr><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>96.1</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>99.3</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>99.2</td><td rowspan=2 colspan=1>98.597.5</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>96.2</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>94.4</td></tr></table>

A total of $5 3 8 8 \mathrm { C T Q } ^ { x }$ Assay results from swab and urine specimens was evaluated from seven geographically diverse clinical sites. A frequency distribution of the initial MaxRFU values for the CT $\mathrm { Q } ^ { x }$ assay is shown in Figure A and B. The distribution of MaxRFU values from CT $\mathrm { Q } ^ { x }$ true positive, true negative, false positive and false negative specimens (i.e., from those specimens that yielded results which were discordant with the patient infected status (PIS)) is shown in Table 4A.

A total of 1714 CT $\mathrm { Q } ^ { x }$ Assay results from BD SurePath specimens was evaluated from eleven geographically diverse clinical sites. A frequency distribution of the initial MaxRFU values for the CT $\mathrm { Q } ^ { x }$ assay is shown in Figure B. The distribution of MaxRFU values from CT $\mathrm { Q } ^ { x }$ true positive, true negative, false positive and false negative specimens (i.e., from those specimens that yielded results which were discordant with the patient infected status (PIS)) is shown in Table 4B.

![](images/9bf4554b02f8ca9237d7c8d0e3787c731fa80e2b9415c3783a3f67d5a2a9c4b1.jpg)  
Figure A: Frequency Distribution of MaxRFU for the CT $\mathbf { Q } ^ { x }$ Assay (Swabs and Urines).

![](images/50ad5228ab43bf5ddda842ae92ddb5666e32125c973bbb59ff005a1d85a6e566.jpg)  
Figure B: Frequency Distribution of MaxRFU for the CT $\mathbf { Q } ^ { x }$ Assay (BD SurePath Specimens).

Table 4A: CT $\mathbf { Q } ^ { x }$ MaxRFU Ranges for False Negative, False Positive, True Negative and True Positive Results (Swabs/Urines).   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MaxRFUrange</td><td rowspan=1 colspan=1>0-49</td><td rowspan=1 colspan=1>50-99</td><td rowspan=1 colspan=1>100-124</td><td rowspan=1 colspan=1>125-149</td><td rowspan=1 colspan=1>150-199</td><td rowspan=1 colspan=1>200-249</td><td rowspan=1 colspan=1>250-349</td><td rowspan=1 colspan=1>350-499</td><td rowspan=1 colspan=1>500-799</td><td rowspan=1 colspan=1>≥ 800</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4590</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>755</td></tr><tr><td rowspan=8 colspan=1>FN</td><td rowspan=1 colspan=1>FNU</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MNU</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>FP</td><td rowspan=1 colspan=1>FNU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>MNU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=5 colspan=1>TN</td><td rowspan=1 colspan=1>FNU</td><td rowspan=1 colspan=1>868</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>857</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1>866</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1>866</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MNU</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Page 15 of 25   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>MaxRFUrange</td><td rowspan=1 colspan=1>0-49</td><td rowspan=1 colspan=1>50-99</td><td rowspan=1 colspan=1>100-124</td><td rowspan=1 colspan=1>125-149</td><td rowspan=1 colspan=1>150-199</td><td rowspan=1 colspan=1>200-249</td><td rowspan=1 colspan=1>250-349</td><td rowspan=1 colspan=1>350-499</td><td rowspan=1 colspan=1>500-799</td><td rowspan=1 colspan=1>≥ 800</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4590</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>755</td></tr><tr><td rowspan=3 colspan=1>b</td><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>364</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1>359</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4548</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>TP</td><td rowspan=1 colspan=1>FNU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>MNU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>717</td></tr></table>

Table 4B: CT $\mathbf { Q } ^ { x }$ MaxRFU Ranges for False Negative, False Positive, True Negative and True Positive Results (BD SurePath Specimens).   

<table><tr><td rowspan=1 colspan=11>MaxRFU Range</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0-49</td><td rowspan=1 colspan=1>50-99</td><td rowspan=1 colspan=1>100-124</td><td rowspan=1 colspan=1>125-149</td><td rowspan=1 colspan=1>150-199</td><td rowspan=1 colspan=1>200-249</td><td rowspan=1 colspan=1>250-349</td><td rowspan=1 colspan=1>350-499</td><td rowspan=1 colspan=1>500-799</td><td rowspan=1 colspan=1>≥800</td></tr><tr><td rowspan=1 colspan=1>FN</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>1562</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>TP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>133</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1569</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135</td></tr></table>

# Controls

During the swab/urine clinical evaluation, there were no CT $\mathrm { Q } ^ { x }$ positive control failures from 253 CT $\mathrm { Q } ^ { x }$ plate runs. For the CT $\mathrm { Q } ^ { x }$ negative control, a failure was observed in 1 of 253 CT $\mathrm { Q } ^ { x }$ plate runs. During the BD SurePath specimen clinical evaluation, there was one CT $\mathrm { Q } ^ { x }$ positive control failure from 120 CT $\mathrm { Q } ^ { x }$ plates that were run. The CT/GC $\mathrm { Q } ^ { x }$ positive and negative control MaxRFU values observed in the clinical trials are shown in Table 5.

Table 5: Distribution of MaxRFU Results for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay Negative and Positive Controls   

<table><tr><td>Control CT Qx Negative Control MaxRFU</td><td>Statistic N</td><td>Swab and Urine Specimen Clinical Study 252</td><td>BD SurePath specimen Clinical Study 119</td></tr><tr><td></td><td>Maximum 95th Percentile</td><td>41 4</td><td>11 0</td></tr><tr><td rowspan="5">CT Qx Positive Control</td><td>Median Mean</td><td>0 1</td><td>0 0</td></tr><tr><td>5th Percentile</td><td>0</td><td>0</td></tr><tr><td>Minimum</td><td>0</td><td>0</td></tr><tr><td></td><td></td><td></td></tr><tr><td>N</td><td>253</td><td>119</td></tr><tr><td rowspan="5">MaxRFU</td><td>Maximum</td><td>2378</td><td>2173</td></tr><tr><td>95th Percentile</td><td>2184</td><td>2128</td></tr><tr><td>Median</td><td>1968</td><td>1942</td></tr><tr><td>Mean</td><td>1939</td><td>1864</td></tr><tr><td>5th Percentile</td><td>1597</td><td>1340</td></tr><tr><td></td><td>Minimum</td><td>629</td><td>168</td></tr></table>

# Swab and Urine Specimen Clinical Study

Clinician-collected endocervical and male urethral swab specimens, patientcollected vaginal swab specimens (in a clinical setting), and male and female $\mathrm { Q } ^ { x }$ UPT and neat urine specimens were collected from 1059 female subjects and 479 male subjects attending OB/GYN, sexually transmitted disease (STD) and family planning clinics at seven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, urethral discharge, coital pain/difficulty/bleeding, testicular or scrotum pain/swelling, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms. Sixty five female subjects and 7 male subjects were excluded from the data analysis due to age requirement violations, antibiotic treatment in the last 21 days, opting to withdraw from the study after initially consenting, failure to obtain paired swab and urine specimens, urine quantity less than $2 0 ~ \mathrm { m L }$ , or transport and storage errors related to specimen collection.

Therefore, the final data analysis included 994 compliant female subjects and 472 compliant male subjects.

Five specimens were collected from each of the 994 eligible female subjects. A urine specimen was collected and split into $\mathbf { Q } ^ { x } \mathbf { U P T }$ , neat urine and the two reference urine specimen collection devices followed by a vaginal swab specimen and three randomized endocervical swab specimens. Up to four specimens were collected from each of the 472 eligible male subjects. Up to three randomized urethral swab specimens were collected followed by a urine specimen that was split into $\mathrm { Q } ^ { x } \mathrm { U P T }$ , neat urine and the two reference urine specimen collection devices. BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ assay results were generated from the $\mathrm { Q } ^ { x }$ UPT and neat urine specimens, the vaginal swab specimen, one endocervical swab specimen and one male urethral swab specimen. The remaining two endocervical swab specimens, up to two male urethral swab specimens, and the two reference urine specimens for each male and female subject were tested using two reference methods: the BD ProbeTec ET CT/AC assay and another commercially available NAAT (Nucleic Acid Amplification Test). Specimen testing was conducted either at the site of specimen collection or at a designated BD Viper testing site.

All performance calculations were based on the total number of BD ProbeTec CT $\mathrm { Q } ^ { x }$ assays results for endocervical, vaginal and male urethral swab specimens, and male and female $\mathrm { Q } ^ { x } \mathrm { U P T }$ and neat urine specimens compared to a patient infected status (PIS) algorithm for each gender. In the algorithm, the designation of a subject as being infected with CT or not was based on endocervical swab and urine specimen results from the commercially available BD ProbeTec ET CT/GC/AC assay and the other commercially available NAAT. Subjects were considered infected with CT if two of the four endocervical swab and urine specimens (or two of the three or four urethral swab and urine specimens) tested positive in the BD ProbeTec ET CT/AC assay and the other reference NAAT (one specimen testing positive in each NAAT). Subjects were considered non-infected if less than two reference NAAT results were positive. A total of 5388 BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ assay results was used to calculate sensitivity and specificity.

Performance of the assay with endocervical swabs, patient collected vaginal swab specimens (in a clinical setting), female UPT and neat urine was assessed in the clinical study. Separate performance was calculated for specimens collected from pregnant females. Sensitivity compared to patient infected status for FS was $6 2 . 5 \%$ (5/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FV sensitivity was $7 5 \%$ (6/8): the test and reference NAAT swab specimens were negative; the test and reference NAAT urine specimens were positive yielding a PIS positive result. FNU and FUPT sensitivity were $100 \%$ (8/8). Specificity was $9 4 . 7 \%$ (18/19) for FS, FV, FNU, and FUPT separately.

# BD SurePath Specimen Clinical Study

Endocervical swab specimens and BD SurePath specimens were collected from 1728 compliant female subjects attending family planning, OB/GYN, and sexually transmitted disease clinics at eleven geographically diverse clinical sites in North America. Subjects were classified as symptomatic if they reported symptoms such as dysuria, coital pain/difficulty/bleeding, abnormal vaginal discharge, or pelvic/uterine/adnexal pain. Subjects were classified as asymptomatic if they did not report symptoms.

Three randomized endocervical swab specimens and a BD SurePath specimen were collected from each female subject. The three reference endocervical swabs were tested with the BD ProbeTec ET CT/GC/AC assay, the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ assay, and another commercially available NAAT (Nucleic Acid Amplification Test). Sensitivity and specificity for BD SurePath specimens were calculated by comparing results to a patient infected status (PIS) algorithm. The designation of positive or negative PIS was based on the endocervical swab specimen results from the three reference methods. At least two positive reference results were required to establish a subject as PISpositive. At least two negative reference results were required to establish a subject as PIS-negative.

The distribution of cervical sampling devices used in the clinical study according to clinical collection site is summarized in Table 6.

Table 7 summarizes the number of results from symptomatic and asymptomatic subjects designated as infected or non-infected with CT according to the PIS algorithm.

Table 8 summarizes the CT $\mathrm { Q } ^ { x }$ Assay Performance for BD SurePath Specimens Compared to Patient Infected Status (by symptomatic status).

Table 6: Summary of Cervical Sampling Devices Used in the BD SurePath specimen Clinical Study   

<table><tr><td rowspan="2">Cervical Sampling Device Used</td><td colspan="11">Clinical Collection Site Number</td><td rowspan="2">Total</td></tr><tr><td>1</td><td>2</td><td>3</td><td>4</td><td>5</td><td>6</td><td>7</td><td>8</td><td>9</td><td>10</td><td>11</td></tr><tr><td>Broom-Type Device</td><td>54</td><td>50</td><td>511</td><td>18</td><td>374</td><td>0</td><td>127</td><td>0</td><td>0</td><td>71</td><td>0</td><td>1205</td></tr><tr><td>Spatula/Cytobr uush</td><td>0</td><td>25</td><td>0</td><td>0</td><td>182</td><td>112</td><td>32</td><td>24</td><td>103</td><td>8</td><td>37</td><td>523</td></tr></table>

Table 7: CT $\mathbf { Q } ^ { x }$ Assay Performance for Swabs and Urines Compared to Patient Infected Status (by specimen type and symptomatic status).   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SymptomaticStatus</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>PPV%</td><td rowspan=1 colspan=1>NPV%</td><td rowspan=1 colspan=1>ErrorInitial/Final</td></tr><tr><td rowspan=1 colspan=1>FS</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>93.0%(53/57)</td><td rowspan=1 colspan=1>(83.0% -98.1%</td><td rowspan=1 colspan=1>98.0%(385/393)</td><td rowspan=1 colspan=1>(96.0% -99.1%</td><td rowspan=1 colspan=1>86.9</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>2/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>89.7%(52/58)</td><td rowspan=1 colspan=1>(78.8% -96.1%)</td><td rowspan=1 colspan=1>98.6%(478/485)</td><td rowspan=1 colspan=1>(97.0% -99.4%)</td><td rowspan=1 colspan=1>88.1</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>1/1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>91.3%(105/115)</td><td rowspan=1 colspan=1>(84.6% -95.8%</td><td rowspan=1 colspan=1>98.3%(863/878)</td><td rowspan=1 colspan=1>(97.2%-99.0%</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>3/1</td></tr><tr><td rowspan=1 colspan=1>FV</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>449</td><td rowspan=1 colspan=1>98.2%(56/57)</td><td rowspan=1 colspan=1>(90.6% -100.0%)</td><td rowspan=1 colspan=1>99.5%(390/392)</td><td rowspan=1 colspan=1>(98.2% -99.9%)</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>99.7</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>544</td><td rowspan=1 colspan=1>94.8%(55/58)</td><td rowspan=1 colspan=1>(85.6% -98.9%)</td><td rowspan=1 colspan=1>99.0%(481/486)</td><td rowspan=1 colspan=1>(97.6% -99.7%)</td><td rowspan=1 colspan=1>91.7</td><td rowspan=1 colspan=1>99.4</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>96.5%(111/115)</td><td rowspan=1 colspan=1>(91.3% -99.0%)</td><td rowspan=1 colspan=1>99.2%(871/878)</td><td rowspan=1 colspan=1>(98.4% -99.7%)</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FNU</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>93.0%(53/57)</td><td rowspan=1 colspan=1>(83.0%-98.1%)</td><td rowspan=1 colspan=1>100.0%(393/393)</td><td rowspan=1 colspan=1>(99.1% -100.0%)</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>93.1%(54/58)</td><td rowspan=1 colspan=1>(83.3% -98.1%)</td><td rowspan=1 colspan=1>99.0%(480/485)</td><td rowspan=1 colspan=1>(97.6% -99.7%)</td><td rowspan=1 colspan=1>91.5</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>93.0%(107/115)</td><td rowspan=1 colspan=1>(86.8% -96.9%)</td><td rowspan=1 colspan=1>99.4%(873/878)</td><td rowspan=1 colspan=1>(98.7% -99.8%)</td><td rowspan=1 colspan=1>95.5</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>FUPT</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>450</td><td rowspan=1 colspan=1>94.7%(54/57)</td><td rowspan=1 colspan=1>(85.4% -98.9%)</td><td rowspan=1 colspan=1>99.5%(391/393)</td><td rowspan=1 colspan=1>(98.2% -99.9%)</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>543</td><td rowspan=1 colspan=1>91.4%(53/58)</td><td rowspan=1 colspan=1>(81.0% -97.1%)</td><td rowspan=1 colspan=1>99.0%(480/485)</td><td rowspan=1 colspan=1>(97.6% -99.7%)</td><td rowspan=1 colspan=1>91.4</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>93.0%(107/115)</td><td rowspan=1 colspan=1>(86.8% -96.9%)</td><td rowspan=1 colspan=1>99.2%(871/878)</td><td rowspan=1 colspan=1>(98.4% -99.7%</td><td rowspan=1 colspan=1>93.9</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>MS</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>88.6%(31/35)</td><td rowspan=1 colspan=1>(73.3% -96.8%)</td><td rowspan=1 colspan=1>98.9%(178/180)</td><td rowspan=1 colspan=1>(96.0%-99.9%)</td><td rowspan=1 colspan=1>93.9</td><td rowspan=1 colspan=1>97.8</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>93.9%(62/66)</td><td rowspan=1 colspan=1>(85.2% -98.3%)</td><td rowspan=1 colspan=1>97.9%(187/191)</td><td rowspan=1 colspan=1>(94.7% -99.4%)</td><td rowspan=1 colspan=1>93.9</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>1/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>92.1%(93/101)</td><td rowspan=1 colspan=1>(85.0% -96.5%)</td><td rowspan=1 colspan=1>98.4%(365/371)</td><td rowspan=1 colspan=1>(96.5% -99.4%)</td><td rowspan=1 colspan=1>93.9</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>2/0</td></tr><tr><td rowspan=1 colspan=1>MNU</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0%(35/35)</td><td rowspan=1 colspan=1>(90.0% -100.0%</td><td rowspan=1 colspan=1>98.9%(178/180)</td><td rowspan=1 colspan=1>(96.0%-99.9%)</td><td rowspan=1 colspan=1>94.6</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>97.0%(64/66)</td><td rowspan=1 colspan=1>(89.5%-99.6%</td><td rowspan=1 colspan=1>99.5%(190/191)</td><td rowspan=1 colspan=1>(97.1% -100.0%)</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>99.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>98.0%(99/101)</td><td rowspan=1 colspan=1>(93.0% -99.8%</td><td rowspan=1 colspan=1>99.2%(368/371)</td><td rowspan=1 colspan=1>(97.7% -99.%</td><td rowspan=1 colspan=1>97.1</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>0/0</td></tr></table>

Page 20 of 25   

<table><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>SymptomaticStatus</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>PPV%</td><td rowspan=1 colspan=1>NPV%</td><td rowspan=1 colspan=1>ErrorInitial/Final</td></tr><tr><td rowspan=1 colspan=1>MUPT</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>215</td><td rowspan=1 colspan=1>100.0%(35/35)</td><td rowspan=1 colspan=1>(90.0% -100.0%)</td><td rowspan=1 colspan=1>98.9%(178/180)</td><td rowspan=1 colspan=1>(96.0% -9.9%</td><td rowspan=1 colspan=1>94.6</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>S</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>97.0%(64/66)</td><td rowspan=1 colspan=1>(89.5% -99.6%)</td><td rowspan=1 colspan=1>97.4%(186/191)</td><td rowspan=1 colspan=1>(94.0% -91%</td><td rowspan=1 colspan=1>92.8</td><td rowspan=1 colspan=1>98.9</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>98.0%(9/101)</td><td rowspan=1 colspan=1>(93.0% -99.8%</td><td rowspan=1 colspan=1>98.1%(364/371)</td><td rowspan=1 colspan=1>(96.2% -99.2%)</td><td rowspan=1 colspan=1>93.4</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>0/0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5388</td><td rowspan=1 colspan=1>94.5%(721/763)</td><td rowspan=1 colspan=1>(92.6% -96.0%</td><td rowspan=1 colspan=1>98.9%(4575/4625)</td><td rowspan=1 colspan=1>(98.6% -99.2%</td><td rowspan=1 colspan=1>93.5</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>5/1</td></tr></table>

Note: For female subjects, infections localized to the endocervix or urethra have been reported in the literature (16-20). Analyses were performed on the endocervical swab specimens and female UPT and neat urine specimens to further characterize the ten negative female endocervical swabs (105/115) and the eight negative female UPT and neat urine specimens (107/115).

Of the 115 female subjects defined as positive by the PIS algorithm, ten had infections localized to the urethra as indicated by the urine reference result (i.e, BD ProbeTec ET CT/AC assay and other NAAT endocervical swab specimens were negative, BD ProbeTec ET CT/AC assay and other NAAT urine specimens were positive.) The BD ProbeTec CT Qx assay was negative for nine of the ten endocervical swab specimens from these subjects.

Of the 115 female subjects defined as positive by the PIS algorithm, three had infections localized to the endocervix as indicated by the endocervical reference result (i.e, BD ProbeTec ET CT/AC assay and other NAAT urine specimens were negative, BD ProbeTec ET CT/AC assay and other NAAT endocervical swab specimens were positive.) The BD ProbeTec CT Qx assay was negative for UPT and neat urine for these three subjects.

Table 8: CT $\mathbf { Q } ^ { x }$ Assay Performance for BD SurePath Specimens Compared to Patient Infected Status (by symptomatic status).   

<table><tr><td rowspan="2">Symptoma tic Status</td><td rowspan="2">N</td><td colspan="4">Performance Compared to Patient Infected Status</td><td colspan="3"></td></tr><tr><td>Sensitivity</td><td>95% C.I.</td><td>Specificity</td><td>95% C.I.</td><td>PPV %</td><td>NPV %</td><td>Error Initial/Fi nal</td></tr><tr><td>A</td><td>115 6</td><td>92.7% (76/82)</td><td>(84.8% - 97.3%)</td><td>99.7% (1071/1074)</td><td>(99.2% - 99.9%)</td><td>95.9</td><td>99.4</td><td>2/1</td></tr><tr><td>S</td><td>558</td><td>98.3% (57/58)</td><td>(90.8% - 100.0%</td><td>99.6% (498/500)</td><td>(98.6% - 100.0%)</td><td>96.6</td><td>99.8</td><td>0/0</td></tr><tr><td>Total</td><td>171 41</td><td>95.0% (133/140)</td><td>(90.0% - 98.0%)</td><td>99.7% (1569/1574)</td><td>(99.3% - 99.9%)</td><td>96.6</td><td>99.6</td><td>2/1</td></tr></table>

Reproducibility Reproducibility of the BD Viper System using the BD ProbeTec CT $\mathrm { Q } x$ Assay was evaluated at three clinical sites on one BD Viper System per site. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent for the BD ProbeTec CT $\mathrm { Q } x$ Assay. Simulated endocervical and urethral specimens contained a clean endocervical swab whereas the

simulated urine and vaginal swab specimens did not. Uninoculated swab diluent for the BD ProbeTec CT $\mathrm { Q } x$ Assay was used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in Table 9.

Table 9: Summary of Reproducibility Data for Swabs and Urines on the BD Viper System for the CT $\mathbf { Q } ^ { x }$ Assay.   

<table><tr><td rowspan=1 colspan=6></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>BetweenRuns WithinSite</td><td rowspan=1 colspan=2>Between Site</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>CTEB&#x27;s/mL</td><td rowspan=1 colspan=1>GCCells/mL</td><td rowspan=1 colspan=1>%Correct</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MaxRFUMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=5 colspan=1>Endocervical /Urethra1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>98.5%(133/135)</td><td rowspan=1 colspan=1>(94.8-99.8%)</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>233.0</td><td rowspan=1 colspan=1>778.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>113.4</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>2011.2</td><td rowspan=1 colspan=1>114.1</td><td rowspan=1 colspan=1>5.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>14.8</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>442.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>76.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-00.0%)</td><td rowspan=1 colspan=1>1991.9</td><td rowspan=1 colspan=1>118.0</td><td rowspan=1 colspan=1>5.9</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>10.4</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1954.8</td><td rowspan=1 colspan=1>169.4</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=5 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>542.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1999.8</td><td rowspan=1 colspan=1>131.8</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>442.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>1995.2</td><td rowspan=1 colspan=1>125.8</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>52.9</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100.0%(135/135)</td><td rowspan=1 colspan=1>(97.3-100.0%)</td><td rowspan=1 colspan=1>2014.4</td><td rowspan=1 colspan=1>109.5</td><td rowspan=1 colspan=1>5.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr></table>

A second study was conducted internally to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LOD) of the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. A panel of simulated specimens was tested that comprised CT and GC organisms seeded into swab diluent at two different levels (1:10, 1:100) each of which was below the analytical LOD for the respective organism. These levels were selected to fall within the dynamic range of the analytical LOD curve for this assay. Fifteen replicates of each panel member were tested every day for five days across three BD Viper Systems. The data are summarized in Table 10.

Table 10: Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay for Swabs and Urines.   

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Dilutionof Analytical LOD</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>MaxRFUMean(Positive)</td><td rowspan=1 colspan=1>%Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>Max RFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>70.2(158/225)</td><td rowspan=1 colspan=1>(63.8, 76.1)</td><td rowspan=1 colspan=1>1794.2</td><td rowspan=1 colspan=1>29.8(67/225)</td><td rowspan=1 colspan=1>(23.9, 36.2)</td><td rowspan=1 colspan=1>2.6</td></tr><tr><td rowspan=1 colspan=1>Endocervical/Urethral</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>10.2(23/225)</td><td rowspan=1 colspan=1>(6.6,14.9)</td><td rowspan=1 colspan=1>1643.8</td><td rowspan=1 colspan=1>89.8(202/225)</td><td rowspan=1 colspan=1>(85.1, 93.4)</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>64.4(145/225)</td><td rowspan=1 colspan=1>(57.8, 70.7)</td><td rowspan=1 colspan=1>1733.9</td><td rowspan=1 colspan=1>35.6(80/225)</td><td rowspan=1 colspan=1>(29.3, 42.2)</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>Urine/Vaginal</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>10.7(24/225)</td><td rowspan=1 colspan=1>(7.0, 15.5)</td><td rowspan=1 colspan=1>1666.6</td><td rowspan=1 colspan=1>89.3(201/225)</td><td rowspan=1 colspan=1>(84.5, 93.0)</td><td rowspan=1 colspan=1>2.4</td></tr></table>

A reproducibility study of the BD Viper System using the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay was also conducted for Liquid Based Cytology (LBC) specimens at three clinical sites on one BD Viper System per site. A panel of simulated specimens comprising CT and GC organisms seeded into LBC Specimen Dilution Tubes containing LBC medium was tested with the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Uninoculated LBC Specimen Dilution Tubes containing LBC medium were used for the CT negative samples. Nine replicates of each panel member were tested every day for five days on each BD Viper System. The data are summarized in Table 13. Two additional target levels were included in the panels to characterize the reproducibility of test results (i.e., proportion positive or negative) at target levels below the analytical Limit of Detection (LOD) of the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ Assay. These additional specimens comprised CT and GC organisms seeded into LBC Specimen Dilution Tubes containing LBC medium at dilutions of 1:10 and 1:100 of the respective analytical LODs of each analyte. These levels were selected to fall within the dynamic range of the analytical LOD curves for the BD ProbeTec CT $\mathrm { Q } ^ { \mathrm { x } }$ and GC $\mathrm { Q } ^ { \mathrm { x } }$ assays. Nine replicates of each panel member were tested every day for five days across the three BD Viper Systems. The data are summarized in Tables 11 and 12.

Table 11: Summary of Reproducibility Data for LBC Specimens on the BD Viper System for the CT $\mathbf { Q } ^ { x }$ Assay.   

<table><tr><td rowspan="2">GC EBs/mL Cells/mL</td><td rowspan="2">% Correct</td><td rowspan="2"></td><td rowspan="2">Mean 95% CI MaxRFU</td><td rowspan="2">Within Run</td><td colspan="3">Between Runs Within Site</td><td colspan="2">Between Site</td></tr><tr><td>SD</td><td>%CV</td><td>SD</td><td>%CV</td><td>SD</td><td>%CV</td></tr><tr><td>0</td><td>0</td><td>100.0% (135/135)</td><td>(97.3% - 100.0%)</td><td>1.30</td><td>4.66</td><td>357.64</td><td>0.85</td><td>65.29</td><td>0.20</td><td>15.12</td></tr><tr><td>30</td><td>0</td><td>100.0% (135/135)</td><td>(97.3% - 100.0%)</td><td>2021.95</td><td>225.94</td><td>11.17</td><td>16.58</td><td>0.82</td><td>21.52</td><td>1.06</td></tr><tr><td>0</td><td>100</td><td>100.0% (135/135)</td><td>(97.3% - 100.0%)</td><td>1.35</td><td>3.63</td><td>268.97</td><td>0.00</td><td>0.00</td><td>0.87</td><td>64.48</td></tr><tr><td>30</td><td>250</td><td>100.0% (135/135)</td><td>(97.3% - 100.0%)</td><td>2028.41</td><td>155.45</td><td>7.66</td><td>9.93</td><td>0.49</td><td>0.00</td><td>0.00</td></tr><tr><td>75</td><td>100</td><td>100.0% (135/135)</td><td>(97.3% - 100.0%)</td><td>1964.40</td><td>170.91</td><td>8.70</td><td>44.37</td><td>2.26</td><td>8.70</td><td>0.44</td></tr></table>

Table 12: Characterization of System Reproducibility at Target Levels below the Analytical Limit of Detection for the CT $\mathbf { Q } ^ { \mathbf { x } }$ Assay for LBC Specimens.   

<table><tr><td rowspan=1 colspan=1>DilutionofAnalyticaILOD</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>95% CI(Positive)</td><td rowspan=1 colspan=1>MaxRFUMean(Positive)</td><td rowspan=1 colspan=1>% Negative</td><td rowspan=1 colspan=1>95% CI(Negative)</td><td rowspan=1 colspan=1>MaxRFUMean(Negative)</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>50.4(68/135)</td><td rowspan=1 colspan=1>(41.6 - 59.1)</td><td rowspan=1 colspan=1>1935.9</td><td rowspan=1 colspan=1>49.6 (67/135)</td><td rowspan=1 colspan=1>(40.9 - 58.4)</td><td rowspan=1 colspan=1>11.5</td></tr><tr><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>7.4(10/135)</td><td rowspan=1 colspan=1>(3.6 - 13.2)</td><td rowspan=1 colspan=1>1835.7</td><td rowspan=1 colspan=1>92.6 (125/135)</td><td rowspan=1 colspan=1>(86.8 - 96.4)</td><td rowspan=1 colspan=1>9.4</td></tr></table>

# System Cross Contamination and Carryover

An internal study was conducted to evaluate the risk of producing a false positive result in either the same run on the BD Viper System in extracted mode (within run cross-contamination) or in a subsequent run (between run carryover). Testing was conducted using negative and positive samples on three BD Viper Systems. Negative samples consisted of CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent/LBC Specimen Dilution Tube with PreservCyt Solution. Positive samples consisted of a representative analyte (at $1 0 ^ { 5 } \mathrm { C T }$ $\mathrm { E B / m L }$ ) spiked into CT/GC $\mathrm { Q } ^ { x }$ Swab Diluent/LBC Specimen Dilution Tube with PreservCyt Solution. The overall rate of cross-contamination (i.e., with alternating columns of positive and negative samples and a prevalence of $5 0 \%$ ) was $0 . 4 1 \%$ (9/2208) for the CT/GC Qx Swab diluent and $0 . 4 5 \%$ (5/1104) for the LBC Specimen Dilution Tube with PreservCyt Solution. The overall rate of carryover contamination (i.e., carryover between successive runs when the prevalence was $50 \%$ in the previous run) was $0 . 3 6 \%$ (8/2208) for the CT/GC $\mathrm { Q } ^ { \mathrm { x } }$ Swab diluent and $0 . 5 4 \%$ (6/1104) for the LBC Specimen Dilution Tube with PreservCyt Solution. Cross-contamination and carryover rates across the three BD Viper Systems are summarized in Tables 13 and 14.

Table 13: Cross Contamination and Carryover Contamination (Swab/Urine).   

<table><tr><td rowspan=2 colspan=1>AssayDispenseModeSelected</td><td rowspan=2 colspan=1>BD ViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>DualAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.14</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.41</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>736</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>2208</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.36</td></tr><tr><td rowspan=4 colspan=1>SingleAssay</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.54</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>188</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>566</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>568</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.18</td></tr></table>

Table 14: Cross Contamination and Carryover Contamination (LBC Medium).   

<table><tr><td rowspan=2 colspan=1>MediumType</td><td rowspan=2 colspan=1>BD ViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carryover Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=4 colspan=1>PreservCyt</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.45</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0.54</td></tr></table>

# INTERPRETATION OF TABLES Symbols and Abbreviations

# Symbols

$( + )$ positive (-) negative # number $\%$ percentage

# Abbreviations

A Asymptomatic   
CI Confidence Interval   
CT Chlamydia trachomatis   
CV Coefficient of Variation   
EC Extraction Control   
FN False Negative   
FNU Female Neat Urine   
FP False Positive   
FS Female endocervical swab   
FUPT Female urine in $\mathrm { Q } ^ { \mathrm { x } }$ UPT   
FV Female vaginal swab   
GC Neisseria gonorrhoeae   
HIV Human Immunodeficiency Virus   
I Indeterminate   
IFU Inclusion Forming Units   
LBC Liquid Based Cytology   
LE Liquid level error   
LOD Limit of Detection   
MaxRFU Maximum relative fluorescent units   
MNU Male Neat Urine   
MS Male urethral swab   
MUPT Male urine in $\mathrm { Q } ^ { \mathrm { x } }$ UPT   
n number   
NA non-applicable   
NAAT Nucleic Acid Amplification Test   
NPA Negative Percent Agreement   
NPV Negative Predictive Value   
OB/GYN Obstetrics/Gynecology   
PA Percent Agreement   
PBS Phosphate Buffered Saline   
PIS Patient Infected Status   
PPA Positive Percent Agreement   
PPV Positive Predictive Value   
QC Quality Control   
S Symptomatic   
SD Standard Deviation   
SDA Strand Displacement Amplification   
STD Sexually Transmitted Disease   
TN True Negative   
TP True Positive   
UPT Urine Preservative Transport   
N. Instrument: Same as described in K081824.   
O. System Descriptions: Same as described in K081824   
P. Other Supportive Device and Instrument Information: NA   
Q. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.   
R. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.