# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K091171 B. Purpose for Submission:

This is a new 510(k) application for the Diagnostic Hybrids, Inc. device, $\mathrm { D } ^ { 3 }$ FastPoint LDFA Respiratory Virus Identification Kit, which is intended for the qualitative identification of influenza A virus, influenza B virus, respiratory syncytial virus, metapneumovirus, adenovirus and to screen for the presence of parainfluenza virus types 1, 2, and 3 in nasal and nasopharyngeal swabs and aspirates/washes specimens from patients with signs and symptoms of respiratory infection by direct detection of immunofluorescence using monoclonal antibodies (MAbs).

# C. Measurand:

Respiratory viral antigens (Influenza A, Influenza B, RSV, Human Metapneumovirus, Adenovirus, Parainfluenza virus types 1, 2 and 3)

# D. Type of Test:

Direct Fluorescence Antibody (DFA) test using direct specimens

# E. Applicant:

Diagnostic Hybrids Inc.

F. Proprietary and Established Names: $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit

# G. Regulatory Information:

1. Regulation section: 866.3980

2. Classification: Class II

3. Product codes:

OMG (secondary product codes : LKT, GNX, GQS, GNY)

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Diagnostic Hybrids, Inc. device, $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit is intended for the qualitative identification of influenza A virus, influenza B virus, respiratory syncytial virus, human metapneumovirus, adenovirus and to screen for the presence of parainfluenza virus types 1, 2, and 3 in nasal and nasopharyngeal swabs and aspirates/washes specimens from patients with signs and symptoms of respiratory infection by direct detection of immunofluorescence using monoclonal antibodies (MAbs).

It is recommended that specimens found to be negative for influenza A virus, influenza B virus, respiratory syncytial virus, adenovirus or parainfluenza viruses after examination of the direct specimen result be confirmed by cell culture. Specimens found to be negative for human metapneumovirus after examination of the direct specimen results should be confirmed by an FDA cleared human metapneumovirus molecular assay. Negative results do not preclude respiratory virus infection and should not be used as the sole basis for diagnosis, treatment or other management decisions.

Performance characteristics for influenza A virus detection and identification were established when influenza A (H3N2) and influenza A (H1N1) were the predominant influenza A strains circulating in the United States. Since influenza strains display antigenic drift and shift from year to year, performance characteristics may vary. If infection with a novel influenza A virus is suspected, based on clinical and epidemiological screening criteria communicated by public health authorities, collect specimens following appropriate infection control precautions and submit to state or local health departments, for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility1 is available to receive and culture specimens.2

2. Indication(s) for use:

Same as intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

Fluorescence microscope with the appropriate filter combination for FITC (excitation peak $= 4 9 0 \ \mathrm { n m }$ , emission peak $= 5 2 0 \mathrm { n m }$ ) and for R-PE; magnification 200-400X.

# I. Device Description:

The $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit uses three blends (each called a “L-DFA Reagent”) of viral antigen-specific murine monoclonal antibodies that are directly labeled with either R-PE (influenza A virus, respiratory syncytial virus, and parainfluenza virus) or fluorescein (influenza B virus, human metapneumovirus, and adenovirus) for the rapid identification of respiratory viruses in nasal and nasopharyngeal swabs and aspirates/washes from patients with signs and symptoms of respiratory infection.

Kit Components:

1. Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, $4 . 0 { \cdot } \mathrm { m L }$ . One dropper bottle containing a mixture of PE-labeled murine monoclonal antibodies directed against influenza A virus antigens and FITC-labeled murine monoclonal antibodies directed against influenza B virus antigens. The buffered, stabilized, aqueous solution contains Evans Blue and propidium iodide as counter-stains and $0 . 1 \%$ sodium azide as preservative.

2. RSV/hMPV $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, $4 . 0 { \cdot } \mathrm { m L }$ . One dropper bottle containing a mixture of PE-labeled murine monoclonal antibodies directed against respiratory syncytial virus antigens and FITC-labeled murine monoclonal antibodies directed against human metapneumovirus antigens. The buffered, stabilized, aqueous solution contains Evans Blue and propidium iodide as counter-stains and $0 . 1 \%$ sodium azide as preservative.

3. HPIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, $4 . 0 { \cdot } \mathrm { m L }$ . One dropper bottle containing a mixture of PE-labeled murine monoclonal antibodies directed against parainfluenza virus types 1, 2, or 3 antigens and FITC-labeled murine monoclonal antibodies directed against adenovirus antigens. The buffered, stabilized, aqueous solution contains Evans Blue and propidium iodide as counter-stains and $0 . 1 \%$ sodium azide as preservative.

4. 40X PBS Concentrate, $2 5 { \mathrm { - m L } }$ . One bottle of 40X PBS concentrate containing $4 \%$ sodium azide ( $0 . 1 \%$ sodium azide after dilution to 1X using de-mineralized water).

5. Re-suspension Buffer, $6 . 0 { \cdot } \mathrm { m L }$ . One bottle of a buffered glycerol solution and $0 . 1 \%$ sodium azide.

6. $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Antigen Control Slides, 5-slides. Five individually packaged control slides containing 6 wells with cell culture derived positive and negative control cells. Each positive well is identified as to the virus infected cells present, i.e., influenza A virus, influenza B virus, respiratory syncytial virus, human metapneumovirus, parainfluenza virus, and adenovirus. The negative wells contain non-infected cells. Each slide is intended to be stained only one time.

An overview of the procedure is as follows:

The cells to be tested are derived from respiratory specimens from patients with signs and symptoms of respiratory infection. The cells are permeablized and stained concurrently in a liquid suspension format in 3 separate vials, each containing one of the 3 “DFA Reagents”. After incubating at $3 5 \mathrm { { ^ \circ C } }$ to $3 7 \mathrm { { ^ \circ C } }$ for 5 minutes, the stained cell suspensions are rinsed with PBS. The rinsed cells are pelleted by centrifugation and then re-suspended with the re-suspension buffer and loaded onto a specimen slide well. The cells are examined using a fluorescence microscope. Cells infected with influenza A virus, respiratory syncytial virus, or parainfluenza virus types 1, 2 and 3 will exhibit golden yellow fluorescence due to the PE. Cells infected with influenza B virus, human metapneumovirus or adenovirus will exhibit apple-green fluorescence due to the FITC. Non-infected cells will exhibit red fluorescence due to the Evans Blue counter-stain. Nuclei of intact cells will exhibit orange-red fluorescence due to the propidium iodide.

Materials Provided:

1. Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent   
2. RSV/hMPV $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent   
3. PIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent   
4. Re-suspension Buffer   
5. $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Antigen Control Slides   
6. 40X PBS Concentrate

Materials Required But Not Provided:

1. Fluorescence microscope with the correct filter combination for FITC (excitation $\mathrm { p e a k } = 4 9 0 \mathrm { n m }$ , emission peak $= 5 2 0 \mathrm { n m }$ ) and for R-PE; magnification 200 to 400X.   
2. Fifty pack of 3-well specimen slides.   
3. Cover slips $( 2 2 \mathrm { ~ x ~ } 5 0 \mathrm { m m } )$ ) for Antigen Control Slides and for specimen slides.   
4. Adjustable pipettes (20 to 200 and 200 to $1 0 0 0 { \cdot } \mu \mathrm { L }$ .   
5. Pipette tips (20 to 200 and 200 to $1 0 0 0 { \cdot } { \mu } \mathrm { L }$ )   
$6 . 2 0 0 – \mathrm { m L }$ wash bottle.   
$7 . 1 . 7 – \mathrm { m L }$ centrifuge vials.   
8. 15-mL conical centrifuge tube.   
9. Sodium hypochlorite solution (1:10 final dilution of household bleach).   
10. Humidified chamber (e.g., covered Petri dish with a damp paper towel placed in the bottom) or humidified incubator.   
11. Incubator, $3 5 ^ { \circ }$ to $3 7 ^ { \circ } \mathrm { C }$ (CO2 or non-CO2, depending on the cell culture format used).

12. Centrifuge with free-swinging bucket rotor.   
13. De-mineralized water for dilution of 40X PBS Concentrate.   
14. Stat-Spin Centrifuge (or bench top centrifuge capable of 2-minutes at $2 0 0 0 \mathrm { x g } )$ .

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Diagnostic Hybrids, Inc. $\mathrm { D } ^ { 3 }$ Ultra DFA Respiratory Virus Screening & ID Kit Diagnostic Hybrids, Inc. $\mathrm { D } ^ { 3 }$ Duet DFA Influenza A/Respiratory Virus Screening Kit Diagnostic Hybrids, Inc. $\mathrm { D } ^ { 3 }$ Duet DFA RSV/Respiratory Virus Screening Kit Diagnostic Hybrids, Inc. $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit

2. Predicate k number(s):

(k061101), (k081928), (k081746), (k090073)

3. Comparison with predicates:

The intended use of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit is similar to the predicate devices $\mathrm { \Delta } ^ { \mathrm { \scriptsize { D } } ^ { 3 } }$ Ultra DFA Respiratory Virus Screening & ID Kit, $\mathrm { D } ^ { 3 }$ Duet DFA RSV/Respiratory Virus Screening Kit, and $\mathrm { \dot { D } } ^ { 3 }$ DFA Metapneumovirus Identification Kit. Characteristics of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit are compared to those of the predicate devices, in the Table below:

<table><tr><td colspan="4" rowspan="1">Technological Characteristics Comparison of Devices</td></tr><tr><td colspan="1" rowspan="1">D³ FastPoint L-DFARespiratory VirusIdentification Kit (Subject)</td><td colspan="1" rowspan="1">D³ Ultra DFA RespiratoryVirus Screening &amp; ID Kit(Predicate)</td><td colspan="1" rowspan="1">D³ Duet DFARSV/Respiratory VirusScreening Kit (Predicate)</td><td colspan="1" rowspan="1">D³ DFA MetapneumovirusIdentification Kit (Predicate)</td></tr><tr><td colspan="2" rowspan="1">Target Viruses</td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Flu A, Flu B, RSV, hMPV,Adenovirus, HPIV-1,2,3</td><td colspan="1" rowspan="1">Flu A, Flu B, RSV,Adenovirus, HPIV-1,2,3</td><td colspan="1" rowspan="1">Flu A, Flu B, RSV,Adenovirus, HPIV-1,2,3</td><td colspan="1" rowspan="1">hMPV</td></tr><tr><td colspan="1" rowspan="1">Monoclonal antibodies (MAbs)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">18 MAbs to 8 differentrespiratory viruses(Flu A, Flu B, RSV, hMPV,Adenovirus, HPIV-1,2,3)</td><td colspan="1" rowspan="1">15 MAbs to7 differentrespiratory viruses(Flu A, Flu B, RSV,Adenovirus, HPIV-1,2,3)</td><td colspan="1" rowspan="1">15 MAbs to7 differentrespiratory viruses(Flu A, Flu B, RSV,Adenovirus, HPIV-1,2,3)</td><td colspan="1" rowspan="1">3 MAbs to hMPV</td></tr><tr><td colspan="1" rowspan="1">Labeling method</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Direct labelingusing R-Phycoerythrin(R-PE) to label theMAbs to FluA, RSV andHPIV-1,2,3using fluoresceinisothiocyanate (FITC) to labelFluB, hMPV and AdenovirusMAbs with fluorescein</td><td colspan="1" rowspan="1">Direct labelingusing fluoresceinisothiocyanate (FITC) tolabel Flu A, Flu B, RSV,Adenovirus, HPIV 1,2,3MAbs with fluorescein</td><td colspan="1" rowspan="1">Direct labelingusing R-Phycoerythrin(R-PE) to label theMAbs to RSVusing fluoresceinisothiocyanate (FITC) tolabel Flu A, Flu B,Adenovirus, and HPIV-1,2,3MAbs with fluorescein</td><td colspan="1" rowspan="1">Direct labelingusing fluoresceinisothiocyanate (FITC) tolabel hMPV MAbs withfluorescein</td></tr><tr><td colspan="4" rowspan="1">R-Phycoerythrin-labeledMAbs</td></tr><tr><td colspan="1" rowspan="1">FluA, RSV and HPIV-1,2,3</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">None</td></tr><tr><td>Fluorescein-labeled MAbs</td><td></td><td></td><td></td></tr><tr><td>FluB, hMPV and Adenovirus</td><td>Flu A, Flu B, RSV, Adenovirus and HPIV 1,2,3</td><td>Flu A, Flu B, Adenovirus, and HPIV-1,2,3</td><td>hMPV</td></tr><tr><td>Cell Fixative</td><td colspan="3"></td></tr><tr><td>Sapogenin</td><td>Acetone</td><td>Acetone</td><td>Acetone</td></tr><tr><td>Cell Counter-stain</td><td></td><td></td><td></td></tr><tr><td>Propidium Iodide and Evans Blue</td><td>Evans Blue</td><td>Evans Blue</td><td>Evans Blue</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Special controls guidance documents will be promulgated   
Guidance on Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses (March 2006) – http://www.fda.gov/cdrh/oivd/g uidance/1596.html   
Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) –   
http://www.fda.gov/cdrh/oivd/guidance/1588.html   
Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Guidance for Industry and FDA Reviewers (March 2007) –   
http://www.fda.gov/cdrh/osb/guidance/1620.html   
Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff – http://www.fda.gov/cdrh/ode/guidance/1567.html   
Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (Feb 2008) – http://www.fda.gov/cdrh/oivd/guidance/1638.pdf

# L. Test Principle:

The $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit uses three blends (each called a “L-DFA Reagent”) of viral antigen-specific murine monoclonal antibodies that are directly labeled with either R-PE (influenza A virus, respiratory syncytial virus, and parainfluenza virus) or fluorescein (influenza B virus, human metapneumovirus, and adenovirus) for the rapid identification of respiratory viruses in nasal and nasopharyngeal swabs and aspirates/washes from patients with signs and symptoms of respiratory infection.

The cells to be tested are derived from respiratory specimens from patients with signs and symptoms of respiratory infection. The cells are permeablized and stained concurrently in a liquid suspension format in 3 separate vials, each containing one of the 3 “DFA Reagents.” After incubating at $3 5 \mathrm { { ^ \circ C } }$ to $3 7 \mathrm { { ^ \circ C } }$ for 5 minutes, the stained cell suspensions are rinsed with PBS. The rinsed cells are pelleted by centrifugation and then re-suspended with the re-suspension buffer and loaded onto a specimen slide well. The cells are examined using a fluorescence microscope. Cells infected with influenza A virus, respiratory syncytial virus, or parainfluenza virus types 1, 2 and 3 will exhibit golden yellow fluorescence due to the PE. Cells infected with influenza B virus, human metapneumovirus or adenovirus will exhibit apple-green fluorescence due to the FITC. Non-infected cells will exhibit red fluorescence due to the Evans Blue counter-stain. Nuclei of intact cells will exhibit orange-red fluorescence due to the propidium iodide.

It is recommended that results for specimens found to contain no fluorescent cells after examination of the direct specimen result be confirmed by cell culture. In the case of hMPV, it is recommended that specimens found to contain no fluorescent cells after examination of the direct specimen be confirmed by an FDA cleared hMPV molecular assay.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Assay precision, intra-assay variability and inter assay variability were assessed with 3 panels of proficiency-level antigen control slides. Each of the 3 reproducibility panels consisted of 5 randomized panel members.

The Influenza A/B panel consisted of the following:

a. Low level influenza A (Victoria strain) infected cells.   
b. Low level influenza B (Taiwan strain) infected cells.   
c. Low level influenza A (Victoria strain) infected cells mixed with mid level influenza B (Taiwan strain) infected cells.   
d. Low level influenza B (Victoria strain) infected cells mixed with mid level influenza A (Victoria strain) infected cells.   
e. Mid level non-infected (negative) cells.

The RSV/hMPV panel consisted of the following:

a. Low level RSV (Washington strain) infected cells.   
b. Low level hMPV (A1 subtype) infected cells.   
c. Low level RSV (Washington strain) infected cells mixed with mid level hMPV (A1 subtype) infected cells.   
d. Low level hMPV (A1 subtype) infected cells mixed with mid level RSV (Washington strain) infected cells.   
e. Mid level non-infected (negative) cells.

The HPIV/Adenovirus panel consisted of the following:

a. Low level Para 1 (C-35 strain) infected cells.   
b. Low level Adenovirus (ATCC type 1) infected cells.   
c. Low level Para 1 (C-35 strain) infected cells mixed with mid level Adenovirus (ATCC type 1) infected cells.   
d. Low Adenovirus (ATCC type 1) infected cells mixed with mid level Para 1 (C-35 strain) infected cells.   
e. Mid level non-infected (negative) cells.

The low level is estimated to contain between 4 to $10 \%$ infected cells in the sample. The mid level is estimated to contain between 20 to $2 5 \%$ infected cells in the sample. Each sample contains $2 . 5 \mathrm { x } 1 0 ^ { 5 }$ to $3 . 5 \mathrm { x } 1 0 ^ { 5 }$ total cells.

Each panel was tested daily in two separate runs for 5-days by four different laboratories (40 total runs). The following results were recorded:

a. Presence or absence of golden-yellow fluorescence.   
b. Percent of cells exhibiting golden-yellow fluorescence.   
c. Presence or absence of apple-green fluorescence.   
d. Percent of cells exhibiting apple-green fluorescence.

Note: “Processing of specimen”, although a source of variability, was done according to each laboratory’s established practices. The product insert for this device instructs the laboratory to process a specimen according to Clinical Microbiology Handbook (H.D. Isenberg, 2004, publ. by ASM; sections 10.7.1- 10.7.10). As such, testing reproducibility of “processing of specimen” is beyond the scope of this reproducibility study. This study accessed reproducibility of the test alone, i.e., “chemistry of assay” (DFA staining) and “interpretation of result”. “Interpretation of result” is considered to be the largest source for variability for this test. Interpretation of test is subjective, according to potential variability in an individual technician’s competence, experience, and/or diligence in microscopic evaluations of stained cells.

A total of 840 data points were included in the reproducibility study data analysis (3 panels X 7 analytes/run X 2 runs/day X 5 days X 4 sites $= 8 4 0$ ).

For the Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, the combined data from the four Study Sites demonstrated reproducible detection of influenza A virus by the R-PE labeled MAbs and reproducible detection of influenza B virus by the FITC-labeled MAbs. The presence of influenza A virus infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The presence of influenza B virus infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The absence of infected cells was reported in $9 5 \%$ (38/40) of the wells in which infected cells were not present. The total percent agreement for the Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent was $9 9 . 3 \%$ (278/280):

Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent   

<table><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="2">PanelMember</td><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="2">Flu ALowLevel</td><td colspan="1" rowspan="2">Flu BLowLevel</td><td colspan="2" rowspan="1">Mixed Infection</td><td colspan="2" rowspan="1">Mixed Infection</td><td colspan="1" rowspan="3">Total %Agreement</td></tr><tr><td colspan="1" rowspan="1">Flu A MidLevel</td><td colspan="1" rowspan="1">Flu B LowLevel</td><td colspan="1" rowspan="1">Flu A LowLevel</td><td colspan="1" rowspan="1">Flu B MidLevel</td></tr><tr><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Noinfectedcells</td><td colspan="1" rowspan="1">4 to 10%infectedcells</td><td colspan="1" rowspan="1">4 to 10%infectedcells</td><td colspan="1" rowspan="1">20 to 30%infectedcells</td><td colspan="1" rowspan="1">4 to 10%infectedcells</td><td colspan="1" rowspan="1">4 to 10%infectedcells</td><td colspan="1" rowspan="1">20 to 30%infectedcells</td></tr><tr><td colspan="1" rowspan="1">Site1</td><td colspan="1" rowspan="1">AgreementwithExpectedresult</td><td colspan="1" rowspan="1">8/10(80%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">68/70(97.1%)</td></tr><tr><td colspan="1" rowspan="1">Site2</td><td colspan="1" rowspan="1">AgreementwithExpectedresult</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">70/70(100%)</td></tr><tr><td colspan="1" rowspan="1">Site3</td><td colspan="1" rowspan="1">AgreementwithExpectedresult</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">70/70(100%)</td></tr><tr><td colspan="1" rowspan="1">Site4</td><td colspan="1" rowspan="1">AgreementwithExpectedresult</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">10/10(100%)</td><td colspan="1" rowspan="1">70/70(100%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TotalAgreementwith Expectedresult</td><td colspan="1" rowspan="1">38/40(95%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">40/40(100%)</td><td colspan="1" rowspan="1">278/280(99.3%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">83.1-99.4%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">91.2-100%</td><td colspan="1" rowspan="1">97.499.9%</td></tr></table>

For the RSV/hMPV $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, the combined data from the four Study Sites demonstrated reproducible detection of RSV by the R-PE labeled MAbs and reproducible detection of hMPV by the FITC-labeled MAbs. The presence of RSV infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The presence of hMPV infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The absence of infected cells was reported in $100 \%$ (40/40) of the wells in which infected cells were not present. The total percent agreement for the RSV/hMPV

RSV/hMPV D3 FastPoint L-DFA Reagent   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Negative</td><td rowspan=2 colspan=1>RSVLowLevel</td><td rowspan=2 colspan=1>hMPVLowLevel</td><td rowspan=1 colspan=2>Mixed Infection</td><td rowspan=1 colspan=2>Mixed Infection</td><td rowspan=3 colspan=1>Total %Agreement</td></tr><tr><td rowspan=1 colspan=1>RSV MidLevel</td><td rowspan=1 colspan=1>hMPVLow Level</td><td rowspan=1 colspan=1>RSV LowLevel</td><td rowspan=1 colspan=1>hMPV MidLevel</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Noinfectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>20 to 30%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>20 to 30%infectedcells</td></tr><tr><td rowspan=1 colspan=1>Site1</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site2</td><td rowspan=1 colspan=1>AgreementwithExpectedesult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site3</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site4</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwith Expectedresult</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>280/280(100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>98.7-100%</td></tr></table>

For the HPIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent, the combined data from the four Study Sites demonstrated reproducible detection of HPIV-1 by the R-PE labeled MAbs and reproducible detection of Adenovirus by the FITC-labeled MAbs. The presence of HPIV-1 infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The presence of Adenovirus infected cells was reported in $100 \%$ (120/120) of the wells in which the infected cells were expected. The absence of infected cells was reported in $100 \%$ (40/40) of the wells in which infected cells were not present. The total percent agreement for the HPIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent was $100 \%$ (280/280):

HPIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>Negative</td><td rowspan=2 colspan=1>HPIV-1LowLevel</td><td rowspan=2 colspan=1>AdenovirusLow Level</td><td rowspan=1 colspan=2>Mixed Infection</td><td rowspan=1 colspan=2>Mixed Infection</td><td rowspan=3 colspan=1>Total %Agreement</td></tr><tr><td rowspan=1 colspan=1>HPIV-1MidLevel</td><td rowspan=1 colspan=1>AdenovirusLow Level</td><td rowspan=1 colspan=1>HPIV-1Low Level</td><td rowspan=1 colspan=1>AdenovirusMid Level</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Noinfectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>20 to30%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>4 to 10%infectedcells</td><td rowspan=1 colspan=1>20 to 30%infectedcells</td></tr><tr><td rowspan=1 colspan=1>Site1</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site2</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site3</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1>Site4</td><td rowspan=1 colspan=1>AgreementwithExpectedresult</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>10/10(100%)</td><td rowspan=1 colspan=1>70/70(100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwith Expectedresult</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>40/40(100%)</td><td rowspan=1 colspan=1>280/280(100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>91.2-100%</td><td rowspan=1 colspan=1>98.7100%</td></tr></table>

$b$ . Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Development and Characterization of Reagents Development and Characterization of MAbs

Development and characterization of each MAb includes immunogen preparation, immunization, hybridoma preparation, clone selection, MAb purification, determination of relative binding affinities, Western blot testing and isotype identification. All of the monoclonal antibodies (MAbs) included in the $\mathbf { \dot { D } } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit have also been used in one the following DHI devices:

1. $\mathrm { D } ^ { 3 }$ Ultra DFA Respiratory Virus Screening & ID Kit that was cleared for marketing via section 510(k) k061101 on November 20, 2006. 2. $\mathrm { D } ^ { 3 }$ Duet DFA Influenza A/Respiratory Virus Screening Kit that was cleared for marketing via section 510(k) k081746 on December 23, 2008. 3. $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit that was cleared for marketing via section 510(k) k090073 on March 6, 2009.

# Performance Evaluation of PE-labeled MAbs

Reactivity of PE-labeled MAbs with Acetone-Fixed Infected Model Cells (Model Cells are A549 cell cultures infected with known isolates of influenza A virus, influenza B virus, respiratory syncytial virus, metapneumovirus, parainfluenza virus types 1, 2 and 3, or adenovirus at a high (0.1) MOI. These cultures are incubated for 20 to 22 hours at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ , then processed by scraping the monolayers and resuspending the cells in a viral transport medium.)

An antibody may exhibit high affinity for its target antigen until labeled with a reporter moiety such as PE due to blocking or modification of the antigen-binding site. Each of the two influenza A virus MAbs, the two respiratory syncytial virus MAbs, and the six parainfluenza virus (2 for each parainfluenza virus types 1, 2 and 3) MAbs were labeled with PE. The individual PE-labeled MAbs were used to stain acetone-fixed influenza A virus, respiratory syncytial virus, and parainfluenza virus types 1, 2, and 3 model cells (respectively) in order to verify that each of the MAbs remains reactive with its target after labeling with PE. The same model cells were stained concurrently with FITC-labeled MAbs for comparison. This comparison of the reactivity is summarized in the following table:

<table><tr><td rowspan=1 colspan=4>PE-labeled MAb versus FITC-labeled MAb Reactivity Comparison in Acetone-fixed Cells</td></tr><tr><td rowspan=1 colspan=1>D3 FastPoint L-DFAReagent Kit MAb #</td><td rowspan=1 colspan=1>Target Virus</td><td rowspan=1 colspan=1>R-PE</td><td rowspan=1 colspan=1>FITC</td></tr><tr><td rowspan=1 colspan=1>A(6)B11</td><td rowspan=1 colspan=1>influenza A</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>2H3C5</td><td rowspan=1 colspan=1>influenza A</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>3A4D9</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>4F9G3</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>1D8E10</td><td rowspan=1 colspan=1>parainfluenza 1</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>9F61C9</td><td rowspan=1 colspan=1>parainfluenza 1</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>2E4D7</td><td rowspan=1 colspan=1>parainfluenza 2</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>5E4E11</td><td rowspan=1 colspan=1>parainfluenza 2</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>1F6C8</td><td rowspan=1 colspan=1>parainfluenza 3</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>4G5(1)E2H9</td><td rowspan=1 colspan=1>parainfluenza 3</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr></table>

# Reactivity of PE-labeled MAbs with Permeablized Infected Model Cells

(Model Cells are A549 cell cultures infected with known isolates of influenza A virus, influenza B virus, respiratory syncytial virus, metapneumovirus, parainfluenza virus types 1, 2 and 3, or adenovirus at a high (0.1) MOI. These cultures are incubated for 20 to 22 hours at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ , then processed by scraping the monolayers and resuspending the cells in a viral transport medium.)

Studies were conducted to demonstrate that the PE-labeled MAbs would stain infected cells in liquid suspension that have been permeablized. The individual PE-labeled MAbs were used to stain influenza A virus, respiratory syncytial virus, and parainfluenza virus types 1, 2, and 3 model cells (respectively) that had been permeablized in order to verify that each of the PE-labeled MAbs remains reactive with its target. The same cells were stained with FITC-labeled MAbs. This comparison of the reactivity is summarized in the following table:

<table><tr><td rowspan=1 colspan=4>PE-labeled MAb versus FITC-labeled MAb Reactivity Comparison in Permeablized Cells</td></tr><tr><td rowspan=1 colspan=1>D^3 FastPoint L-DFAReagent KitMAb #</td><td rowspan=1 colspan=1>Target Virus</td><td rowspan=1 colspan=1>R-PE</td><td rowspan=1 colspan=1>FITC</td></tr><tr><td rowspan=1 colspan=1>A(6)B11</td><td rowspan=1 colspan=1>influenza A</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>2H3C5</td><td rowspan=1 colspan=1>influenza A</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>3A4D9</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>4F9G3</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>1D8E10</td><td rowspan=1 colspan=1>parainfluenza 1</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>9F61C9</td><td rowspan=1 colspan=1>parainfluenza 1</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>2E4D7</td><td rowspan=1 colspan=1>parainfluenza 2</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>5E4E11</td><td rowspan=1 colspan=1>parainfluenza 2</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>1F6C8</td><td rowspan=1 colspan=1>parainfluenza 3</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr><tr><td rowspan=1 colspan=1>4G5(1)E2H9</td><td rowspan=1 colspan=1>parainfluenza 3</td><td rowspan=1 colspan=1>Reactive</td><td rowspan=1 colspan=1>Reactive</td></tr></table>

# Concentration of PE-labeled MAbs

Final blended solutions of the 2 influenza A virus MAbs, the 2 respiratory syncytial virus MAbs and the 6 parainfluenza virus MAbs were formulated to yield optimal fluorescence intensity and lowest background on their respectively infected cells, permeablized and stained in suspension.

# Performance Evaluation of FITC-labeled MAbs

Reactivity of FITC-labeled MAbs with Acetone-Fixed Infected Model Cells (Model Cells are A549 cell cultures infected with known isolates of influenza A virus, influenza B virus, respiratory syncytial virus, metapneumovirus, parainfluenza virus types 1, 2 and 3, or adenovirus at a high (0.1) MOI. These cultures are incubated for 20 to 22 hours at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ , then processed by scraping the monolayers and resuspending the cells in a viral transport medium.)

The 2 influenza B virus, 3 human metapneumovirus, and 3 adenovirus FITClabeled MAbs used in the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit have all previously been FDA cleared for use with acetone-fixed cells. No additional testing was performed.

Reactivity of FITC-labeled MAbs with Permeablized Infected Model Cells (Model Cells are A549 cell cultures infected with known isolates of influenza A virus, influenza B virus, respiratory syncytial virus, metapneumovirus, parainfluenza virus types 1, 2 and 3, or adenovirus at a high (0.1) MOI. These cultures are incubated for 20 to 22 hours at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ , then processed by scraping the monolayers and resuspending the cells in a viral transport medium.)

Studies were conducted to demonstrate that the FITC-labeled MAbs would stain infected cells in solution that have been permeablized. The individual FITClabeled MAbs were used to stain influenza B virus, human metapneumovirus, adenovirus model cells (respectively) that had been permeablized in order to verify that each of the FITC-labeled MAbs remains reactive with its target. All FITC-labeled MAbs reacted with the appropriate permeablized model cells as expected, and similar to acetone-fixed cells.

# Concentration of FITC-labeled MAbs

Final blended solutions of the 2 influenza B virus MAbs, the 3 human metapneumovirus MAbs and the 3 adenovirus MAbs were formulated to yield optimal fluorescence intensity and lowest background on their respectively infected cells, permeablized and stained in suspension.

# Cell Permeablization and Counterstaining

# Selection of Permeablization Reagent

The $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit uses Sapogenin to permeablize the cell membrane instead of acetone to allow the MAbs to react with their respective antigens. Permeablization involves treatment of cells with a mild surfactant. This treatment will dissolve portions of the cell membranes and allow larger dye molecules and antibodies access to the cell's interior. This allows the cells to maintain their three dimensional structure while being stained with labeled antibodies and counter-stain. By doing this, cells can remain in liquid suspension. Studies were conducted to compare performance using acetone (with that using another permeablizing reagent, Sapogenin. Sapogenins are the aglycones, or nonsaccharide portions of the family of natural products known as saponins. The amphipathic nature of saponins gives them activity as surfactants that can be used to enhance penetration of macromolecules such as proteins through cell membranes. Using influenza A virus, influenza B virus and respiratory syncytial virus model cells, acetone and Sapogenin were tested at various concentrations. Acetone was tested at concentrations from $20 \%$ to $100 \%$ . Sapogenin was used at $0 . 1 \%$ based on what has been published in the literature. Data generated from the study indicated that at all acetone concentrations Sapogenin had greater numbers of infected cells in the liquid format. Based on these studies, Sapogenin was chosen as the Permeablization reagent in the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents.

# Determination of Sapogenin Concentration

Experiments were conducted to optimize the concentration of Sapogenin in the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit. Non-infected model cells were treated with different concentrations of Sapogenin and counter-stain for 5- minutes. The cells were then counted, and the values compared. Data generated from the study indicated that there was no difference in numbers of cells when Sapogenin concentrations at $0 . 1 \%$ to $0 . 0 2 5 \%$ were used. When $0 . 2 \%$ Sapogenin was used, reduced numbers of cells were noted, which was an indication that Sapogenin at that concentration may cause cell disruption. The $0 . 1 \%$ level was chosen to be used in the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents to better ensure adequate Permeablization of clinical nasal pharyngeal cells.

# Propidium Iodide Counter-stain

To assist the end user in the use of the $\mathrm { D } ^ { 3 }$ Ultra Duet Kits, improvement to the counter-staining of cells was developed. Propidium Iodide was added to the three $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents. The stained cell’s nuclei fluoresce red. This improves the ability to assess specimen quality compared to standard acetone fixation, Evans Blue stained DSFA specimens. Subjective studies were conducted to determine the optimal concentration of Propidium Iodide. Higher concentrations of Propidium Iodide $1 6 \mathrm { - } \mu \mathrm { g } / \mathrm { m L }$ or higher) began interfering with the ability to see low level fluorescence generated by either the PE- or FITCstained cells. Lower levels of Propidium Iodide $\scriptstyle 4 - \mu \mathrm { g } / \mathrm { m L }$ or lower) made it difficult to see the stained nuclei. $8 \mathrm { - } \mu \mathrm { g } / \mathrm { m L }$ was the optimal concentration to allow easy identification of cells with no quenching of PE or FITC fluorescence. A low level of Evans Blue $2 5 \mathrm { - } \mu \mathrm { g } / \mathrm { m L }$ compared to $2 5 0 \mathrm { - } \mu \mathrm { g } / \mathrm { m L }$ in each of the predicate respiratory devices) is also included to help reduce background of non-specific antibody staining sometimes seen in clinical specimens.

# Reagent Interference Studies of MAbs

Studies were conducted to demonstrate that the final blend of PE- and FITClabeled MAbs in each $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent did not affect the ability to detect low level positive infected cells that are stained by one fluor when they are in the same sample, with a high level of positive infected cells which are stained by the other fluor in the same well. The following cell preparations were permeablized and stained with the appropriate reagent.

1. For the Influenza $\mathbf { A } /$ Influenza B Reagent:   
a. Low level ${ \sim } 2 5$ or lower infected cells) infected influenza A virus model cells were spiked into non-infected cells.   
b. Low level influenza A virus model cells were spiked into high level $( 4 + )$ influenza B virus model cells.   
c. Low level ${ \sim } 2 5$ or lower infected cells) infected influenza B virus model cells were spiked into non-infected cells.

d. Low level influenza B virus model cells were spiked into high level $( 4 + )$ of influenza A virus model cells.

2. For the RSV/hMPV Reagent:

a. Low level ${ \sim } 2 5$ or lower infected cells) infected respiratory syncytial virus model cells were spiked into non-infected cells.   
b. Low level respiratory syncytial virus model cells were spiked into high level $( 4 + )$ metapneumovirus model cells.   
c. Low level ${ \sim } 2 5$ or lower infected cells) metapneumovirus model cells were spiked into non-infected cells.   
d. Low level metapneumovirus model cells were spiked into high level $( 4 + )$ of respiratory syncytial virus model cells.

3. For the PIV/Adenovirus Reagent:

a. Low level ${ \sim } 2 5$ or lower infected cells) parainfluenza virus type model cells were spiked into non-infected cells.   
b. Low level parainfluenza viruses model cells were spiked into high level $( 4 + )$ adenovirus model cells.   
c. Low level ${ \sim } 2 5$ or lower infected cells) adenovirus model cells were spiked into non-infected cells.   
d. Low level adenovirus model cells were spiked into high level $( 4 + )$ parainfluenza viruses model cells.

The following table summarizes the study data for the three DFA Reagents:

<table><tr><td rowspan=1 colspan=3>Staining Interference of High Level Infected Model Cells</td></tr><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>Infected Cell Countsof Low Level ModelCells</td><td rowspan=1 colspan=1>Average Infected CellCounts of Low LevelModel Cells</td></tr><tr><td rowspan=1 colspan=1>Low Flu A virus model cells in non-infected cells</td><td rowspan=1 colspan=1>8, 6, 13</td><td rowspan=1 colspan=1>9.0</td></tr><tr><td rowspan=1 colspan=1>Low Flu A virus in High Flu B virus model cells</td><td rowspan=1 colspan=1>9,9, 10</td><td rowspan=1 colspan=1>9.3</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low Flu B virus model cells in non-infected cells</td><td rowspan=1 colspan=1>17, 9, 16</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>Low Flu B virus in High Flu A virus model cells</td><td rowspan=1 colspan=1>8,9, 12</td><td rowspan=1 colspan=1>9.6</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low RSV model cells in non-infected cells</td><td rowspan=1 colspan=1>28, 12, 12</td><td rowspan=1 colspan=1>17.3</td></tr><tr><td rowspan=1 colspan=1>Low RSV model cells in High hMPV model cells</td><td rowspan=1 colspan=1>14, 12, 8</td><td rowspan=1 colspan=1>11.3</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low hMPV model cells in non-infected cells</td><td rowspan=1 colspan=1>5,1,5</td><td rowspan=1 colspan=1>3.6</td></tr><tr><td rowspan=1 colspan=1>Low hMPV model cells in High RSV model cells</td><td rowspan=1 colspan=1>4,0,3</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low HPIV-1 model cells in non-infected cells</td><td rowspan=1 colspan=1>7,4,7</td><td rowspan=1 colspan=1>6.0</td></tr><tr><td rowspan=1 colspan=1>Low HPIV-1 model cells in High Adenovirus model cells</td><td rowspan=1 colspan=1>7,3,5</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low HPIV-2 model cells in non-infected cells</td><td rowspan=1 colspan=1>3,3,7</td><td rowspan=1 colspan=1>4.4</td></tr><tr><td rowspan=1 colspan=1>Low HPIV-2 model cells in High Adenovirus model cells</td><td rowspan=1 colspan=1>8,4,4</td><td rowspan=1 colspan=1>5.3</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low HPIV-3 model cells in non-infected cells</td><td rowspan=1 colspan=1>3,7,5</td><td rowspan=1 colspan=1>5.0</td></tr><tr><td rowspan=1 colspan=1>Low HPIV-3 model cells in High Adenovirus model cells</td><td rowspan=1 colspan=1>3,5,6</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Low Adenovirus model cells in non-infected cells</td><td rowspan=1 colspan=1>13, 15, 9</td><td rowspan=1 colspan=1>12.3</td></tr><tr><td rowspan=1 colspan=1>Low Adenovirus model cells in High HPIV-1 model cells</td><td rowspan=1 colspan=1>7,9,8</td><td rowspan=1 colspan=1>8.0</td></tr><tr><td rowspan=1 colspan=1>Low Adenovirus model cells in High HPIV-2 model cells</td><td rowspan=1 colspan=1>17, 13, 12</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>Low Adenovirus model cells in High HPIV-3 model cells</td><td rowspan=1 colspan=1>9, 18, 23</td><td rowspan=1 colspan=1>16.6</td></tr></table>

For each combination of low level infected cells spiked into high level of infected cells, there was not a significant difference in detection compared to the low level positive cells spiked into non-infected cells (control).

# Binding Competition Studies of MAbs

The three $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents contain blends of virus-specific MAbs. The Influenza A/Influenza B L-DFA Reagent contains 2 influenza A virus MAbs and 2 influenza B virus MAbs. The RSV/hMPV L-DFA Reagent contains 2 respiratory syncytial virus MAbs and three metapneumovirus MAbs. The PIV/Adenovirus L-DFA Reagent contains 2 parainfluenza virus type 1 MAbs, 2 parainfluenza virus type 2 MAbs, 2 parainfluenza virus type 3 MAbs and 3 adenovirus MAbs. The purpose of combining either two or three MAbs specific per virus is to ensure that all strains will be detected. Studies were conducted to determine if the individual MAbs compete with one another for the same binding sites since originally, the clones were selected for their individual and highest level of staining intensity of the respective virus antigens.

Model cells of influenza A virus, influenza B virus, respiratory syncytial virus, adenovirus, human metapneumovirus, and parainfluenza virus types 1, 2, and 3 were permeablized. Cells were stained with each unlabeled clone of the appropriate virus. The cells were then stained with the PE- or FITC-labeled MAbs for each pair (a combination of 3 for hMPV and adenovirus). Labeled MAbs were used individually, each at their standard concentrations used in the assay.

Results of the initial study indicated that there was no evidence of self or cross epitope blocking for all the pairs (a combination of 3 for hMPV and adenovirus) of MAbs, except for the two influenza A and the two parainfluenza type 3 MAbs. The two influenza A and two parainfluenza type 3 MAbs were labeled with FITC and tested using the same protocol. The MAbs again blocked each other. This suggested that the blocking is unrelated to the labeling chemistry. The testing was repeated using acetone fixation of the model cells in place of the permeablization. The MAbs again blocked each other. This suggested that the blocking is unrelated to the fixation chemistry.

It is unknown if this blocking of the parainfluenza virus type 3 MAbs is related to the model cell system being used for the studies, or if blocking also occurs on clinical nasopharyngeal epithelial cells. For the influenza A virus MAbs, blocking of the epitopes is strain dependant. Even though these clones appear to bind to the same epitope for parainfluenza virus type 3 and for influenza A virus using the H3N2 virus strain, they are still used because of an “additive” effect on the brightness of fluorescent cells, i.e. doubling the concentration of one clone and using it alone is not as bright as using the two clones with the same final concentration.

# Stability Studies

# Shelf life for the complete kit

Kits were tested at time intervals during storage according to the study plan. Characteristics monitored were performance, as well as $\mathrm { p H }$ , color and clarity. Among the acceptance criteria was fluorescence (as opposed to no fluorescence) observed in processed, infected model cells at a high level of infection ( $2 +$ to $4 +$ ) for each of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents at 1:16 dilution. This dilution was what was used for both the $\mathrm { D } ^ { 3 }$ Ultra and $\mathbf { \bar { D } } ^ { 3 } \mathbf { M P V }$ Reagents in their respective devices. Stability studies have been conducted in two phases: (1) using kits produced during the development phase according to draft written procedures, and (2) using kits produced according to established procedures by manufacturing staff (both phases are on-going). An initial indication of the shelf life of the device $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit is currently being determined using accelerated storage at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ . As of April 2009, stability as demonstrated by the accelerated studies is 8 months. To establish the final shelf life of the device, real-time testing (under labeled storage conditions of $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ is also being conducted. As of April 2009, stability has been demonstrated to 2 months.

# Shelf life for the $\mathbf { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit antigen control slides

The $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit Antigen Control Slides are prepared by combining infected cells (influenza A virus and influenza B virus in one well, RSV and hMPV in another well, and parainfluenza virus types 1, 2, and 3 and adenovirus in another well). Non-infected cells are spotted onto 3 wells for a negative control for each reagent. Stability studies are currently being conducted for the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit Antigen Control Slides; however, since they are prepared using the same procedure and same infected cell cultures as the control slides in the $\mathrm { D } ^ { 3 }$ Ultra Kit and $\mathrm { D } ^ { 3 } \mathrm { M P V }$ Kit, a shelf life of 18 months is anticipated. Stability studies have been conducted in two phases: (1) using slides produced during the development phase according to draft written procedures, and (2) using slides produced according to established procedures by manufacturing staff (both phases are ongoing). An initial indication of the shelf life of the $\mathrm { D } ^ { \bar { 3 } }$ Ultra Duet Antigen Control Slides is currently being determined using accelerated storage at $3 5 ^ { \circ } \mathrm { C }$ to $3 7 ^ { \circ } \mathrm { C }$ . As of April 2009, stability as demonstrated by the accelerated studies is 8 months. To establish shelf life of the device, real-time testing (under labeled storage conditions of $2 ^ { \circ } \mathrm { C }$ to $8 ^ { \circ } \mathrm { C }$ ) is also being conducted. As of April 2009, stability has been demonstrated to 2 months.

# d. Analytical Sensitivity (Detection limit):

Analytical Limit of Detections of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagents were addressed using dilution series of infected model cells. Model cells for 8 characterized respiratory virus isolates; influenza A virus (ATCC Victoria strain), influenza B virus (ATCC Taiwan strain), respiratory syncytial virus (ATCC Washington strain), adenovirus (ATCC type 1), human metapneumovirus subtype A1 (clinical strain), parainfluenza virus types 1, 2, and 3 (ATCC strains C-35, Greer, and C243 respectively) were diluted with non-infected cells to produce a suspension equivalent to 1,000 infected cells per milliliter. This level theoretically yields approximately 25 infected cells per $2 5 \mathrm { - } \mu \mathrm { L }$ of suspension. This suspension was then serially diluted to a theoretical level of less than 1 cell per milliliter. (NOTE: This level was the target to begin with a low positive level. Actual starting levels vary, however, and are within 1 dilution of the 25 infected cell target level). $2 5 \mathrm { - } \mu \mathrm { L }$ aliquots from each dilution level were spotted onto 10 replicate microscope slides, and then stained according to the instructions for use described in this product insert. Each cell spot was examined at $2 0 0 \mathrm { x }$ magnification. Results were reported as numbers of positive replicates for each set of 10. Analytical detection limits for each of the 8 analytes were defined as the lowest dilutions at which at least 9 out of 10 replicates were detected. Results are summarized in the table below:

<table><tr><td rowspan=1 colspan=5>Limit of Detections of the D³ FastPoint L-DFA Respiratory Virus Identification Kit</td></tr><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Infected cells/mL</td><td rowspan=1 colspan=1>Number of replicates with positive cells</td><td rowspan=1 colspan=2>LOD determination</td></tr><tr><td rowspan=10 colspan=1>Flu A(ATCC Victoria strain)</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=2>50 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>5/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>3/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>Flu B(ATCC Taiwan strain)</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=2>50 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>4/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>RSV(ATCC Washington strain)</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=2>100 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>7/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=5 colspan=1>hMPV A1(Clinical strain)</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1></td><td rowspan=5 colspan=1>100 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>10/10</td><td></td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td><td></td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td><td></td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>6/10</td><td></td></tr></table>

<table><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2/10</td><td rowspan=5 colspan=1></td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>Adenovirus(ATCC type 1)</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>100 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>5/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>HPIV-1(ATCC strain C-35)</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>100 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>HPIV-2(ATCC strain Greer)</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>25 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=10 colspan=1>HPIV-3(ATCC strain C243)</td><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>10/10</td><td rowspan=10 colspan=1>50 infected cells/mL</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10/10</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>9/10</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>6/10</td></tr><tr><td rowspan=1 colspan=1>12.5</td><td rowspan=1 colspan=1>2/10</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0/10</td></tr></table>

# e. Analytical Reactivity (Inclusivity):

Analytical reactivity (inclusivity) of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Influenza A/Influenza B DFA Reagent was evaluated using 13 influenza A virus and 7 influenza B virus strains. Low concentration infected cell suspensions (approximately $4 \%$ cells infected, 25-50 infected cells) were prepared for each viral strain. The suspensions were stained with the Influenza A/Influenza B $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent. The following table summarizes the data:

<table><tr><td rowspan=1 colspan=3>AayiRiviyiviFaR A virus and influenza B virus strains</td></tr><tr><td rowspan=1 colspan=1>Influenza Strains</td><td rowspan=1 colspan=1>Infected Cell Concentration (asmultiples of the respectiveestablished LoD concentration</td><td rowspan=1 colspan=1>Influenza A/ Influenza BD³ FastPoint L-DFA Reagent Results</td></tr><tr><td rowspan=1 colspan=1>Influenza A Mexico/4108/2009 (H1N1)from CDC*</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>19 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A California/07/2009 (H1N1)from CDC*</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>26 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Wisconsin/56/2005 (H3N2)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>39 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A WS, VR-1520 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>67 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Hong Kong, VR-544 (H3N2)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>13 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A New Jersey, VR-897 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>15 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A A/NWS/33 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>10 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Victoria, VR-822 (H3N2)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>10 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A PR, VR-95 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>20 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Port Chalmers, VR-810(H3N2)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>8 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Aichi, VR-547 (H3N2)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>28 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Denver, VR-546 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>30 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza A Mal, VR-98 (H1N1)</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>21 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>Influenza B GL/1739/54, VR-103</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>13 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B Taiwan/2/62, VR-295</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>44 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B Hong Kong/5/72, VR-823</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>21 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B Maryland/1/59, VR-296</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>22 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B Russia, VR-790</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>36 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B B/Lee/40</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>41 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Influenza B Massachusetts, VR-523</td><td rowspan=1 colspan=1>20x LoD</td><td rowspan=1 colspan=1>67 Apple-green fluorescent cells</td></tr></table>

\*Although the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Influenza A/Influenza B DFA Reagent has been shown to detect the $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ virus in two culture isolates provided by the CDC, and wellcharacterized by the CDC rRT-PCR Swine Flu Detection Panel (under an EUA), the performance characteristics of this device with clinical specimens that are positive for the 2009 H1N1 influenza virus have not been established. The $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Influenza A/Influenza B DFA Reagent can distinguish between influenza A and B viruses, but it can not differentiate influenza subtypes.

Analytical reactivity (inclusivity) of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA RSV/hMPV DFA Reagent was evaluated using 3 RSV virus and $4 \mathrm { h M P V }$ virus strains. Low concentration infected cell suspensions (approximately $4 \%$ cells infected, 25-50 infected cells) were prepared for each viral strain. The suspensions were stained with the RSV/hMPV $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent. The following table summarizes the data:

<table><tr><td colspan="3" rowspan="1">AayicalReaciviy ncuiviy theFastoint LDFARSV/MP DFARea vars RSV viu nd MPVvirus strains</td></tr><tr><td colspan="1" rowspan="1">Influenza Strains</td><td colspan="1" rowspan="1">Infected Cell Concentration (asmultiples of the respectiveestablished LoD concentration</td><td colspan="1" rowspan="1">Influenza A/ Influenza BD³ FastPoint L-DFA Reagent Results</td></tr><tr><td colspan="1" rowspan="1">RSV 9320</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">22 Golden-yellow fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">RSV Washington</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">22 Golden-yellow fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">RSV Long</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">32 Golden-yellow fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">hMPV A1</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">25 Apple-green fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">hMPV A2</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">25 Apple-green fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">hMPV B1</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">25 Apple-green fluorescent cells</td></tr><tr><td colspan="1" rowspan="1">hMPV B2</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">37 Apple-green fluorescent cells</td></tr></table>

Analytical reactivity (inclusivity) of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA HPIV/Adenovirus DFA Reagent was evaluated using 3 HPIV virus and 10 Adenovirus strains. Low concentration infected cell suspensions (approximately $4 \%$ cells infected, 25-50 infected cells) were prepared for each viral strain. The suspensions were stained with the HPIV/Adenovirus $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Reagent. The following table summarizes the data:

<table><tr><td rowspan=1 colspan=3>Analytical Reactiviy ncusivty o theDFastPoint L-DFARSV/hMP DFA Reagen nvars RSV virus and MPVvirus strains</td></tr><tr><td rowspan=1 colspan=1>Influenza Strains</td><td rowspan=1 colspan=1>Infected Cell Concentration (asmultiples of the respectiveestablished LoD concentration</td><td rowspan=1 colspan=1>Influenza A/ Influenza BD³ FastPoint L-DFA Reagent Results</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1 C-35</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>9 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2 Greer</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>11 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3 C-243</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>22 Golden-yellow fluorescent cells</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1 VR-1</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>26 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3 VR-3</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>17 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 5 VR-5</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>15 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 6 VR-6</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>22 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7 VR-7</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>16 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 8 VR-1366</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>29 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 10 VR-1087</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>34 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus VR-14</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>37 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Dewitt ATCC Strain</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>15 Apple-green fluorescent cells</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 31 VR-1109</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>42 Apple-green fluorescent cells</td></tr></table>

# $f .$ Analytical Specificity:

$\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit was tested for crossreactivity against a variety of microorganisms. Stringent conditions for crossreactivity testing were achieved by using both the $1 . 5 \mathrm { X }$ concentration of MAbs and relatively high titers of microorganisms. No cross-reactivity was observed for 59 virus strains. Twenty-two (22) bacterial strains, one yeast, and one Chlamydia sp. were also evaluated for cross-reactivity, including Staphylococcus aureus, a protein-A-producing bacterium. Except for Staphylococcus aureus, which was cross reactive with the $\mathrm { D } ^ { 3 }$ FastPoint LDFA Respiratory Virus Identification Kit, all other microorganisms tested negative.

Staining of S. aureus appeared as small points of fluorescence. The Protein A produced by the bacterium, Staphylococcus aureus, may bind the Fc portion of some fluorescein-labeled monoclonal antibodies. Such binding can be distinguished from viral antigen binding on the basis of morphology, i.e., S. aureus-bound fluorescence appears as small ${ \sim } 1$ -micron diameter), bright dots.

Results from testing direct respiratory specimens with bacterial contamination must be interpreted with caution. The following language was added to the “Limitations of Procedure” section of the product insert to address this issue: “Light background staining may occur with specimens contaminated with Staphylococcus aureus strains containing large amounts of protein A. Protein A will bind to the Fc portions of conjugated antibodies. Such binding can be distinguished from viral antigen binding on the basis of morphology, for example, S. aureus-bound fluorescence appears as small ( $\mathbf { \tilde { \Pi } } ^ { \mathbf { \tilde { \Pi } } }$ micron diameter), bright dots. Therefore, results from testing direct respiratory specimens with bacterial contamination must be interpreted with caution.”

Fifty-nine (59) virus strains were tested for cross reactivity. Depending on the particular virus, $1 . 4 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ to $1 . 4 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { \dot { T } C I D } _ { 5 0 }$ viruses were inoculated into multi-well plate cultures and incubated for 24 to 72 hours to yield a $1 +$ to $^ { 4 + }$ cytopathic effect. For each virus, a confirmation stain was done with the appropriate MAb to ensure the desired titer was achieved. These cells were then prepared as Model Cells (scraped and resuspended in UTM). Each cell suspension of infected Model Cells was processed according to the $\mathrm { D } ^ { \hat { 3 } }$ FastPoint LDFA Respiratory Virus Identification Kit protocol, using 2X MAb and was examined at 200X magnification. No cross reactivity was observed for the viruses listed below:

<table><tr><td colspan="7" rowspan="1">Virus Strains Tested for Cross Reactivity with the D FastPoint L-DFA Respiratory Virus Identification Kit</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain or Type</td><td colspan="1" rowspan="1">D3 FastPoint L-DFAInfluenza A/BReagentResults</td><td colspan="1" rowspan="1">D³ FastPoint L-DFARSV/MPVReagentResults</td><td colspan="1" rowspan="1">D³ FastPoint L-DFAHPIV/AdenovirusReagentResults</td><td colspan="2" rowspan="1">Inoculum (TCID50)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 5</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 7</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 16</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Type 17</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus (hMPV)</td><td colspan="1" rowspan="1">Subtype A1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subtype A2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subtype B1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Subtype B2</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Aichi (H3N2)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mal (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Hong Kong (H3N2)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Denver (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Port Chalmers (H3N2)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Victoria (H3N2)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">New Jersey (HSWN1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">WS (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PR (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Wisconsin (H3N2)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 104</td></tr><tr><td colspan="7" rowspan="1">Virus Strains Tested for Cross Reactivity with the D' FastPoint L-DFA Respiratory Virus Identification Kit</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain or Type</td><td colspan="1" rowspan="1">D3 FastPoint L-DFAInfluenza A/BReagentResults</td><td colspan="1" rowspan="1">D3 FastPoint L-DFARSV/MPVReagentResults</td><td colspan="1" rowspan="1">D3 FastPoint L-DFAHPIV/AdenovirusReagentResults</td><td colspan="2" rowspan="1">Inoculum (TCID50)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">A/NWS/33 (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">A Mexico/4108/2009 (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">A California/07/2009 (H1N1)</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Hong Kong</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Maryland</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Mass</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">GL</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Taiwan</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Russia</td><td colspan="1" rowspan="1">Apple-Green Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">Long</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Wash</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9320</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1</td><td colspan="1" rowspan="1">C-35</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Golden-Yellow Fluor.</td><td colspan="1" rowspan="1">1.4 x 104</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4</td><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 4b</td><td colspan="1" rowspan="1">CH-19503</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV-1</td><td colspan="1" rowspan="1">1(f)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">MacIntyre</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV-2</td><td colspan="1" rowspan="1">Clinical Isolate CWOH-0011</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Strain G</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CMV</td><td colspan="1" rowspan="1">Towne</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">AD169</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Varicella-zoster</td><td colspan="1" rowspan="1">AV92-3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus</td><td colspan="1" rowspan="1">A9</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1.4 x 105</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B3</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">B4</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">209 Picornavirus</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">VR-836</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">VR-1432</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="2" rowspan="1">1.4 x 105</td></tr></table>

Twenty four (24) microorganisms, including 22 bacterial, 1 yeast, and 1 Chlamydia sp. were tested for cross-reactivity. Bacteria were cultured, processed as suspensions, then spiked into non-infected Model Cells suspensions at levels (as CFUs, colony-forming units) ranging from $1 . { \dot { 6 } } \times 1 0 ^ { 9 }$ to $3 . 5 \mathrm { ~ x ~ } 1 0 ^ { 1 \dot { 0 } }$ CFUs depending on the bacterium. These suspensions of Model Cells with bacteria were then processed according to the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit protocol, using 2X MAb reagents. Except for Staphylococcus

aureus, which was cross reactive with the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit, all other microorganisms tested negative. Reactivity with Staphylococcus aureus is more than likely due to binding the protein A produced by Staphylococcus aureus. Microorganisms tested are listed in the table below:

<table><tr><td rowspan=1 colspan=5>Microorganisms Tested for Cross Reactivity with D&#x27; FastPoint L-DFA Respiratory Virus Identification Kit</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>D³ FastPoint L-DFAInfluenza A/BReagentResults</td><td rowspan=1 colspan=1>D3³ FastPoint L-DFARSV/MPVReagentResults</td><td rowspan=1 colspan=1>D³ FastPoint L-DFAHPIV/AdenovirusReagentResults</td><td rowspan=1 colspan=1>CFU tested</td></tr><tr><td rowspan=1 colspan=5>Bacteria</td></tr><tr><td rowspan=1 colspan=1>Acholeplasma laidlawii</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Control Slide</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter calcoaceticus</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3.6 x 109</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.1 x 1010</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4.3 x 109</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis (Apache-2)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>LGV-II/Control Slide</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5.7 x 107</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Control Slide</td></tr><tr><td rowspan=1 colspan=1>Haemophilis influenzae type A</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4.1 x 109</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.2 x 109</td></tr><tr><td rowspan=1 colspan=1>Moraxella cartarrhalis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.2 x 1010</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3.5 x 100</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma orale</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6.6 x 109</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7.9 x 109</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma salivarium</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7.7 x 108</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3.6 x 109</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.0 x 108</td></tr><tr><td rowspan=1 colspan=1>Salmonella enteriditis</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8.7 x 109</td></tr><tr><td rowspan=1 colspan=1>Salmonella typhimurium</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7.5 x 109</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus*</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6.3 x 109</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5.5 x 108</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6.7 x 109</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6.9 x 109</td></tr><tr><td rowspan=1 colspan=1>Yeast</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1.6 x 106</td></tr></table>

\* Reactivity with Staphylococcus aureus is more than likely due to binding the protein A produced by Staphylococcus aureus.

g. Assay cut-off: Not applicable.

h. Interfering Substances: Not applicable.   
i. Comparator Assay Description and Analytical Validation Studies

At the time of the DHI the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit clinical studies, there were no standard or FDA cleared molecular methods to detect hMPV in nasopharyngeal aspirates or washes from patients with signs and symptoms of acute respiratory infection. For purposes of the clinical studies, Diagnostic Hybrids, Inc. (DHI) established and validated a real-time Reverse Transcriptase (RT-PCR) procedure for use as a comparative test method to detect hMPV RNA in a clinical specimen. This real time RT-PCR assay was developed by DHI using a primer set and TaqMan probe as described in the publication: Maertzdorf, J., CK Wang, JB Brown, JD Quinto, M Chu, M de Graff, BG van den Hoogen, R Spaete, ADME Osterhaus, and RAM Fouchier. “Real-time Reverse Transcriptase PCR Assay for Detection of Human Metapneumoviruses from All Known Lineages” J. Clin. Microbiol. 2004:42:981-986. A nucleic acid (RNA) extraction procedure was also developed and validated, as was the RT-PCR reaction, including primer sets, and gel electrophoresis. Detection of an amplicon in the PCR end-product is presumptive evidence of hMPV RNA presence in a specimen. The amplicons were sequenced by a third party (Agencourt). DHI then compared these sequences against an established/published set of hMPV sequences. Confirmation of the hMPV RNA sequences in the amplicon was interpreted as verification of hMPV RNA in the specimen.

Extraction of RNA from specimens is performed with Qiagen QIAamp Viral RNA Mini Kit according to the manufacturer’s suggested protocol (section 12.4). Briefly, the sample is lysed under highly denaturing conditions to inactivate RNase. RNA is then bound to the QIAamp membrane, washed several times to remove contaminants, and eluted in $6 0 { \mathrm { u L } }$ RNase-free buffer. The Human Metapneumovirus Real-Time Reverse Transcription PCR Comparator Assay master mix contains oligonucleotide primers and target specific oligonucleotide probes. The primers are complementary to highly conserved sequences within the nucleocapsid gene of hMPV. Reverse transcription of the RNA in the sample into complementary DNA (cDNA) and subsequent amplification of DNA is performed in a Stratagene $\operatorname { M x } 3 0 0 0 \mathrm { p }$ instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The hMPV real-time RT-PCR Comparator Assay is based on TaqMan chemistry, which utilizes the $5 " 3 "$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the Stratagene Mx3000p instrument.

# Analytical Sensitivity

The analytical sensitivity of the hMPV real-time RT-PCR/Sequencing Comparator Assay was determined for 4 hMPV strains (subtypes A1, A2, B1 and B2). Viral stocks of 4 hMPV strains (subtypes A1, A2, B1 and B2) were generated in LLC- $\mathrm { M K } _ { 2 }$ cultured cells. The viral stocks were frozen at $\mathrm { - 7 0 ^ { \circ } C }$ . The $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ of each stock was determined by cell culture with monoclonal antibody detection. A nasopharyngeal clinical matrix was created from nasopharyngeal swabs placed in transport medium obtained from patients with signs and symptoms of acute respiratory infection. The matrix was tested for the presence of hMPV RNA using the hMPV real-time RT-PCR Comparator Assay. Patient specimens found to be negative were pooled to create sufficient material for all of the dilutions. The quantified viral stocks were diluted with the pooled nasopharyngeal clinical matrix to produce a suspension equivalent to the following $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ levels: 1000, 500, 250, 100, 50, 10, 1, and 0.1. Each $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ level dilution was extracted 10 times (80 per hMPV strain) using the Qiagen Viral RNA Mini Kit. The hMPV real-time RT-PCR/Sequencing Comparator Assay was performed in singlet using $2 0 \mathrm { - } \mu \mathrm { L }$ volumes of each extraction. Analytical detection limits for each of the 4 hMPV genetic sublineages were defined as the lowest dilutions at which at least 9 out of 10 replicates were detected. Detailed analytical sensitivity data for the hMPV realtime RT-PCR/Sequencing Comparator Assay are presented in the following table:

<table><tr><td colspan="7" rowspan="1">hMPV real-time RT-PCR/Sequencing Comparator Assay Analytical Sensitivity Summary</td></tr><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">ConcTCID50/mL</td><td colspan="1" rowspan="1">AverageCT</td><td colspan="1" rowspan="1">StandardDeviation CT</td><td colspan="1" rowspan="1">Min CT</td><td colspan="1" rowspan="1">Max CT</td><td colspan="1" rowspan="1">ReplicatesDetected</td></tr><tr><td colspan="1" rowspan="8">hMPV A1</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">33.28</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">32.99</td><td colspan="1" rowspan="1">33.81</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">34.30</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">33.46</td><td colspan="1" rowspan="1">35.29</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">35.25</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">34.24</td><td colspan="1" rowspan="1">36.18</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">37.69</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">36.26</td><td colspan="1" rowspan="1">39.43</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">38.48</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">39.94</td><td colspan="1" rowspan="1">9/10</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">1/10</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/10</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/10</td></tr><tr><td colspan="1" rowspan="8">hMPV A2</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">29.67</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">32.06</td><td colspan="1" rowspan="1">33.17</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">30.88</td><td colspan="1" rowspan="1">0.57</td><td colspan="1" rowspan="1">33.36</td><td colspan="1" rowspan="1">35.28</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">32.16</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">33.98</td><td colspan="1" rowspan="1">36.47</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">33.30</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">35.52</td><td colspan="1" rowspan="1">37.38</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">34.67</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">37.33</td><td colspan="1" rowspan="1">39.08</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">35.11</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">38.51</td><td colspan="1" rowspan="1">41.43</td><td colspan="1" rowspan="1">7/10</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">40.96</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">40.96</td><td colspan="1" rowspan="1">40.96</td><td colspan="1" rowspan="1">1/10</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/10</td></tr><tr><td colspan="1" rowspan="5">hMPV B1</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">31.39</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">30.90</td><td colspan="1" rowspan="1">32.85</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">32.22</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">31.79</td><td colspan="1" rowspan="1">32.53</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">32.85</td><td colspan="1" rowspan="1">.047</td><td colspan="1" rowspan="1">32.07</td><td colspan="1" rowspan="1">33.59</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.37</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">33.67</td><td colspan="1" rowspan="1">34.77</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">35.10</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">34.42</td><td colspan="1" rowspan="1">35.85</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">ConcTCID50/mL</td><td colspan="1" rowspan="1">AverageCT</td><td colspan="1" rowspan="1">StandardDeviation CT</td><td colspan="1" rowspan="1">Min CT</td><td colspan="1" rowspan="1">Max CT</td><td colspan="1" rowspan="1">Replicates Detected</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">37.82</td><td colspan="1" rowspan="1">1.37</td><td colspan="1" rowspan="1">35.51</td><td colspan="1" rowspan="1">40.07</td><td colspan="1" rowspan="1">9/10</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">37.25</td><td colspan="1" rowspan="1">1.11</td><td colspan="1" rowspan="1">36.46</td><td colspan="1" rowspan="1">38.03</td><td colspan="1" rowspan="1">2/10</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/10</td></tr><tr><td colspan="1" rowspan="8">hMPV B2</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">32.96</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">32.49</td><td colspan="1" rowspan="1">33.34</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">35.37</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">33.37</td><td colspan="1" rowspan="1">37.18</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">34.13</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">34.66</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">1.39</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">40.63</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">1.16</td><td colspan="1" rowspan="1">34.85</td><td colspan="1" rowspan="1">38.36</td><td colspan="1" rowspan="1">10/10</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">40.32</td><td colspan="1" rowspan="1">6/10</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39.46</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">39.46</td><td colspan="1" rowspan="1">39.46</td><td colspan="1" rowspan="1">1/10</td></tr><tr><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/10</td></tr></table>

# Analytical Specificity

The specificity of the hMPV real-time RT-PCR Comparator Assay was evaluated by analyzing nucleic acid extracts of Diagnostic Hybrids, Inc. Respiratory Viruses Panel (Lot 111406), human gDNA (Biochain, Lot A804328), human total RNA (Lot 11140405, Catalog # 540017-41) and the Negative Clinical Specimen. $1 4 0 ~ \mathrm { u L }$ of each sample was extracted with the Qiagen QIAamp Viral RNA Mini Kit, and tested by the DHI hMPV RT-PCR assay.

hMPV real-time RT-PCR Comparator Assay demonstrated $100 \%$ specificity and data is presented in the follow table:

<table><tr><td colspan="2" rowspan="1">hMPV real-time RT-PCR Comparator Assay Analytical Specificity Summary</td></tr><tr><td colspan="1" rowspan="1">Panel Key</td><td colspan="1" rowspan="1">hMPV real-time RT-PCR ComparatorAssay Result</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B B/Taiwan/2/62</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV RSV/B/Wash/18537/62</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 2 Greer</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV Long</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A A/WS/33</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1, Adenoid 71</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV 9320</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A A/Mal/302/54</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A A/Victoria/3/75</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 1 C-35</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3 C234</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 5, Adenoid 75</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B B/GL/1734/54</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A A/PortChalmers/1/73</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human gDNA</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Total RNA</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Negative Clinical Specimen</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">In-house Positive hMPV Strain</td><td colspan="1" rowspan="1">+</td></tr></table>

# Extraction Efficiency

The QIAamp Viral RNA Mini Kit extraction efficiency was determined by extracting serial 10-fold dilutions of cRNA and comparing the actual copies of cRNA in the eluate versus the calculated copies if $100 \%$ recovery. The actual copies of cRNA were obtained from the qRT-PCR standard curve constructed by a serial dilution of un-extracted cRNA standards. The extraction efficiency ranges from 25 to $40 \%$ . The range was obtained from triplicate extractions repeated over 3 days. Duplicate PCR reactions were performed on each dilution. Detailed results of this study are presented in the following table:

<table><tr><td rowspan=2 colspan=1>Pre-extraction viralload (copies/mL)</td><td rowspan=1 colspan=3>% of Recovery</td></tr><tr><td rowspan=1 colspan=1>Day 1</td><td rowspan=1 colspan=1>Day 2</td><td rowspan=1 colspan=1>Day 3</td></tr><tr><td rowspan=1 colspan=1>8.58E+08</td><td rowspan=1 colspan=1>3.36E+01</td><td rowspan=1 colspan=1>2.50E+01</td><td rowspan=1 colspan=1>5.72E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+07</td><td rowspan=1 colspan=1>3.94E+01</td><td rowspan=1 colspan=1>2.35E+01</td><td rowspan=1 colspan=1>1.60E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+06</td><td rowspan=1 colspan=1>2.74E+01</td><td rowspan=1 colspan=1>1.98E+01</td><td rowspan=1 colspan=1>4.29E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+05</td><td rowspan=1 colspan=1>2.91E+01</td><td rowspan=1 colspan=1>2.55E+01</td><td rowspan=1 colspan=1>5.17E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+04</td><td rowspan=1 colspan=1>2.60E+01</td><td rowspan=1 colspan=1>3.36E+01</td><td rowspan=1 colspan=1>3.81E+01</td></tr><tr><td rowspan=1 colspan=1>8.58E+03</td><td rowspan=1 colspan=1>1.69E+01</td><td rowspan=1 colspan=1>3.76E+01</td><td rowspan=1 colspan=1>2.73E+01</td></tr><tr><td rowspan=1 colspan=1>4.19E+03</td><td rowspan=1 colspan=1>4.45E+01</td><td rowspan=1 colspan=1>1.19E+01</td><td rowspan=1 colspan=1>1.80E+01</td></tr></table>

# Sample Freeze and Thaw Study

A study was performed to demonstrate that freezing the specimens would not alter the performance of the hMPV real-time RT-PCR Comparator Assay. Characterized hMPV isolates (subtypes A1, A2, B1, and B2) were spiked separately into fresh nasopharyngeal swab and nasopharyngeal aspirate specimens at known concentrations. The specimens were split into two aliquots: one was extracted and tested using the hMPV real-time RT-PCR Comparator Assay, the other was frozen at $- 7 0 \mathrm { { } ^ { \circ } C }$ overnight and then extracted and tested using the hMPV real-time RT-PCR Comparator Assay.

An amplicon from each specimen was analyzed by bi-directional sequencing and confirmed as hMPV (highlighted in the table below).   

<table><tr><td rowspan="4">TCID50</td><td colspan="11"></td></tr><tr><td colspan="6">A1</td><td colspan="6">NW</td></tr><tr><td colspan="5">NPS</td><td></td><td colspan="4"></td><td colspan="2"></td></tr><tr><td>Fresh Rep. 1</td><td>Rep. 2</td><td>Rep. 3 1</td><td>Frozen Rep. 2</td><td>Rep. 3</td><td>Rep.</td><td>Fresh Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td><td>Frozen Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td></tr><tr><td>Concentration 1.00E+02 1.00E+01 1.00E+00</td><td>35.46 39.16</td><td>36.23 40.29</td><td>35.25 38.66</td><td>34.85 38.51 36</td><td>35.07</td><td>35.02 38.79</td><td>37.76 40.12</td><td>41.04 40.38</td><td>38.19</td><td>36.27 39.1</td><td>37.07 42.98</td><td>37.15</td></tr><tr><td rowspan="4">1.00E-01 TCID50 Concentration</td><td colspan="8">A2</td><td colspan="4"></td></tr><tr><td colspan="6"></td><td>NW</td><td colspan="4"></td><td></td></tr><tr><td colspan="4">NPS Fresh</td><td></td><td></td><td colspan="4">Fresh</td><td></td><td></td></tr><tr><td>Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td><td>Frozen Rep. 1 2</td><td>Rep. 3</td><td>Rep.</td><td>Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td><td>Frozen Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td></tr><tr><td>1.00E+02 1.00E+01 1.00E+00</td><td>32.89 36.13 39.08</td><td>32.66 36.55 39.68</td><td>32.77 37.17 40.37</td><td>32.84 36.44 39.81 40.5 39.17</td><td>33.21 32.98 37.11 35.77 39.84</td><td></td><td>34.36 38.3 41.02 41.09</td><td>34.77 37.89 41.54</td><td>34.62 38.11</td><td>35.44 37.32 40.72</td><td>34.39 38 42.81</td><td>34.53 38.57</td></tr><tr><td rowspan="4">1.00E-01 TCID50</td><td colspan="8"></td><td colspan="4"></td></tr><tr><td colspan="4">B1</td><td colspan="2"></td><td></td><td colspan="4"></td><td></td></tr><tr><td colspan="3">NPS</td><td colspan="3"></td><td></td><td colspan="3">NW</td><td colspan="3"></td></tr><tr><td>Fresh Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td><td>Frozen Rep. 1</td><td>Rep. 2 3</td><td>Rep.</td><td>Fresh Rep. 1</td><td>Rep.</td><td>Rep. 3</td><td>Frozen Rep. 1</td><td>Rep. 2</td><td>Rep. 3</td></tr><tr><td>Concentration 1.00E+02 1.00E+01 1.00E+00</td><td>31.23 35.86</td><td>31.64 35.63 39.38</td><td>32.1 36.04</td><td>30.51 34.83 36.59</td><td>31.24 34.44 36.67</td><td>31.51 34.55 36.44</td><td>33.96 36.15 40.09</td><td>2 33.83 36.5 39.62</td><td>34.26 36.52 41.59</td><td>33.16 36.16 41.06</td><td>33.19 35.66 40.51</td><td>33.04 36.12 42.39</td></tr><tr><td rowspan="4">1.00E-01</td><td colspan="8">38.59 39.42 43.18 40.66 41.46</td><td colspan="7"></td></tr><tr><td colspan="8">39.59 43.62</td><td colspan="7"></td></tr><tr><td colspan="8">B2 NPS NW</td><td colspan="7"></td></tr><tr><td colspan="2">Fresh TCID50 Rep. 1</td><td colspan="2">Rep. Rep.</td><td colspan="3">Frozen Rep. Rep. Rep.</td><td></td><td colspan="3">Fresh Rep. Rep. Rep.</td><td>Frozen Rep.</td><td colspan="2">Rep. Rep. 2</td></tr><tr><td colspan="2">Concentration 1.00E+02 1.00E+01</td><td>2 32.17 32.82</td><td>3 32.07</td><td>1 34.12</td><td>2 34.5 39.73</td><td>3 34.22 39.26</td><td></td><td>1 35.14 36</td><td>2 3 35.02 36.67</td><td>1 34.87 36.81</td><td>35.59 37.65</td><td>3 35.6 35.97</td></tr><tr><td colspan="2"></td><td></td><td></td><td>37.52</td><td></td><td rowspan="3"></td><td rowspan="3"></td><td rowspan="3"></td><td rowspan="3"></td><td rowspan="3">41.72</td><td rowspan="3">37.77</td><td rowspan="3">37.77</td></tr><tr><td colspan="2" rowspan="3">1.00E+00 1.00E-01</td><td>36.51 37.15</td><td>36.8 37.57</td><td>37.15 38.47</td><td>41.3</td><td></td></tr><tr><td>43.6</td><td></td><td></td><td></td><td></td><td>40.38</td></tr></table>

The data indicates that freezing and thawing nasopharyngeal swab and nasopharyngeal aspirate specimens does not alter the performance of the hMPV real-time RT-PCR Comparator Assay significantly.

Based on thorough analytical validations, the hMPV real-time RT-PCR Comparator Assay is an acceptable method to be used to confirm all hMPV negative samples as determined by the FDA cleared comparator DSFA device (D3 Metapneumovirus DFA Reagent) in the DHI $\mathbf { D } ^ { 3 }$ FastPoint LDFA Respiratory Virus Identification Kit Clinical Trial.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.   
b. Matrix Description and Comparison: Not applicable.

3. Clinical studies:

a. Prospective Clinical Studies Testing Direct Respiratory Specimens

Performance of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit testing direct respiratory specimens were established during prospective studies at 4 geographically diverse U.S. clinical laboratories during the 2008/2009 respiratory virus seasons (January 2008 – March 2009). All specimens used in the studies meeting the inclusion and exclusion criteria represented excess, remnants of respiratory specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study.

Performance of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit detecting influenza A, influenza B, respiratory syncytial virus, parainfluenza virus, and adenovirus from direct specimens was assessed and compared to DSFA testing using FDA cleared comparator DSFA devices ( $\mathrm { D } ^ { 3 }$ Ultra DFA Respiratory Virus Screening & ID Kit or $\hat { \mathrm { D } } ^ { 3 }$ Duet DFA RSV/Respiratory Virus Screening Kit), followed by viral culture confirmation of all negative specimens (as determined by the FDA cleared DSFA comparator devices), using FDA cleared DFA reagents.

Performance of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit detecting human metapneumovirus from direct specimens was assessed and compared to DSFA testing using FDA cleared comparator DSFA device $\mathrm { [ D ^ { 3 } }$ Metapneumovirus DFA Reagent), followed by confirmation of all frozen negative specimens (as determined by the FDA cleared DSFA comparator device), using a validated hMPV real-time RT-PCR followed by bi-directional sequencing analysis comparator assay. The hMPV real-time RT-PCR comparator assay targets the hMPV Nucleocapsid gene. “True” hMPV positive was defined as any sample that either tested positive by the FDA cleared DSFA comparator device, or had bidirectional sequencing data meeting pre-defined quality acceptance criteria that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E values. “True” hMPV negative was defined as any sample that tested negative by both the FDA cleared DSFA comparator device and the hMPV real-time RT-PCR comparator assay.

The E-values generated from the clinical trials range from a low of 5e-78 to a high of 1e-20. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone.   
(http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).   
Therefore an E-Value ranging from 1e-20 to 5e-78 has a very low probability of occurring purely by chance.

Study Site 1 evaluated a total of 323 fresh respiratory specimens submitted, January 2009 through March 2009, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and processed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 1:

<table><tr><td rowspan=1 colspan=3>Site 1  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>173</td></tr></table>

Of the 323 fresh respiratory specimens tested, all were nasal wash/nasopharyngeal aspirate specimens. Of the 323 fresh nasal wash/nasopharyngeal aspirate specimens tested, 2 nasal wash/nasopharyngeal aspirate specimens were excluded from the performance analysis due to insufficient sample volume for the investigational device testing. 70 specimens for Flu A, 79 specimens for Flu B, 37 specimens for RSV, 75 specimens for adenovirus, 79 specimens for HPIV, and 3 specimens for hMPV, were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 251 fresh nasal wash/nasopharyngeal aspirate specimens for Flu A, 242 fresh nasal wash/nasopharyngeal aspirate specimens for Flu B, 284 fresh nasal wash/nasopharyngeal aspirate specimens for RSV, 246 fresh nasal wash/nasopharyngeal aspirate specimens for adenovirus, 242 fresh nasal wash/nasopharyngeal aspirate specimens for HPIV, and 318 fresh nasal wash/nasopharyngeal aspirate specimens for hMPV to be included in the respective performance analysis. The tables below summarized the study results of the claimed specimen type at study site 1:

<table><tr><td rowspan=1 colspan=4>Flu A</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>229</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>226</td><td rowspan=1 colspan=1>251</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>21/25</td><td rowspan=1 colspan=1>84.0%</td><td rowspan=1 colspan=1>63.9-95.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>225/226</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>97.6-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>240</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>242</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>15.8-100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.5-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">137</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">138</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">145</td><td colspan="1" rowspan="1">146</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">138</td><td colspan="1" rowspan="1">146</td><td colspan="1" rowspan="1">284</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">137/138</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">96.0-100%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">145/146</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">96.2-100%</td></tr><tr><td colspan="4" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">238</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">238</td><td colspan="1" rowspan="1">246</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">63.1-100%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">238/238</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.5-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>HPIV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>232</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>236</td><td rowspan=1 colspan=1>242</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.1-100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>232/236</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>95.7-99.5%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>296</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>318</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>22/30</td><td rowspan=1 colspan=1>73.3%</td><td rowspan=1 colspan=1>54.1-87.7%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>288/288</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7-100%</td></tr></table>

Study Site 2 evaluated a total of 105 fresh respiratory specimens submitted, February 2009 through March 2009, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and processed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 2:

<table><tr><td rowspan=1 colspan=3>Site 2  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Age</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>57</td></tr></table>

Of the 105 fresh respiratory specimens tested, 86 were nasal wash/nasopharyngeal aspirate specimens. Due to insufficient sample numbers to establish performance of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit, 19 other types of respiratory specimens were removed from performance analysis. None of the nasal wash/nasopharyngeal aspirate samples for Flu A, Flu B, RSV, Adenovirus and HPIV were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 86 fresh nasal wash/nasopharyngeal aspirate specimens for Flu A, Flu B, RSV, Adenovirus and HPIV to be included in the respective performance analysis. 10 specimens for hMPV were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 76 fresh nasal wash/nasopharyngeal aspirate specimens for hMPV to be included in the performance analysis. The tables below summarized the study results of the claimed specimen type at study site 2:

<table><tr><td rowspan=1 colspan=4>Flu A</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.1-100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>78/80</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>91.3-99.7%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">Flu B</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">82</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">86</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">28.4-99.5%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">81/81</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">95.5-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>RSV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>81.5-100%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>68/68</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.7-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>86/86</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>95.8-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">HPIV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Comparator DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">85</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">86</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">1/1</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">85/85</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">95.8-100%</td></tr><tr><td colspan="4" rowspan="1">hMPV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Comparator DSFA (negatives confirmed bya validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">71</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">76</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">5/7</td><td colspan="1" rowspan="1">71.4%</td><td colspan="1" rowspan="1">29.0-96.3%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">69/69</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.8-100%</td></tr></table>

Study Site 3 evaluated a total of 443 fresh respiratory specimens submitted, February 2009 through March 2009, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and processed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 3:

<table><tr><td rowspan=1 colspan=4>Site 3  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Sex Not Reported</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=4>Age</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>3</td></tr></table>

Of the 443 fresh respiratory specimens tested, 301 were nasal wash/nasopharyngeal aspirate specimens, and 140 were nasal/nasopharyngeal swab specimens. One (1) nasal wash/nasopharyngeal aspirate specimen was excluded from the performance analysis due to the fact that the sample was tested by the investigational device greater than 48 hours post sample collection. Due to insufficient sample numbers to establish performance of the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit, 2 other types of respiratory specimens were further removed from performance analysis.

None of the remaining nasal wash/nasopharyngeal aspirate samples for Flu A, Flu B,

RSV, Adenovirus, HPIV and hMPV were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 300 fresh nasal wash/nasopharyngeal aspirate specimens for Flu A, Flu B, RSV, Adenovirus, HPIV and hMPV to be included in the respective performance analysis. None of the nasal/nasopharyngeal swab specimens for Flu A, Flu B, RSV, Adenovirus and HPIV were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 140 nasal/nasopharyngeal swab specimens for Flu A, Flu B, RSV, Adenovirus and HPIV to be included in the respective performance analysis. 1 nasal/nasopharyngeal swab specimen for hMPV was excluded from the performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 139 nasal/nasopharyngeal swab specimens for hMPV to be included in the performance analysis. The tables below summarized the study results of the claimed specimen types at study site 3:

<table><tr><td rowspan=1 colspan=4>Flu A</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>271</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>29/35</td><td rowspan=1 colspan=1>82.9%</td><td rowspan=1 colspan=1>66.4-93.4%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>265/265</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.6-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>19.4-99.4%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>296/296</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.8-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">49</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">249</td><td colspan="1" rowspan="1">251</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">249</td><td colspan="1" rowspan="1">300</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">49/51</td><td colspan="1" rowspan="1">96.1%</td><td colspan="1" rowspan="1">86.5-99.5%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">249/249</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.5-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>296</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>28.4-99.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>295/295</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.8-100%</td></tr></table>

HPIV   

<table><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>282</td><td rowspan=1 colspan=1>284</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>282</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>16/18</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>65.3-98.6%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>282/282</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>272</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>257</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>28/43</td><td rowspan=1 colspan=1>65.1%</td><td rowspan=1 colspan=1>49.1-79.0%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>257/257</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.6-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealswab</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">130</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">129</td><td colspan="1" rowspan="1">140</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">10/11</td><td colspan="1" rowspan="1">90.9%</td><td colspan="1" rowspan="1">58.7-99.8%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">129/129</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">97.2-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>138</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>9.4-99.2%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>137/137</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.3-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>RSV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>10/11</td><td rowspan=1 colspan=1>90.9%</td><td rowspan=1 colspan=1>58.7-99.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>129/129</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>140/140</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.4-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">HPIV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealswab</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">133</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">133</td><td colspan="1" rowspan="1">140</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">7/7</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">59.0-100%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">133/133</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">97.3-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>130</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>9/17</td><td rowspan=1 colspan=1>52.9%</td><td rowspan=1 colspan=1>27.8-77.0%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>122/122</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.0-100%</td></tr></table>

Study Site 4 evaluated a total of 648 fresh respiratory specimens submitted, February 2009 through March 2009, to the laboratory for respiratory virus testing. Slides were prepared from Phosphate Buffered Saline (PBS)-washed cells from the fresh specimens and processed according to the prescribed protocol. The slides were stained in accordance with the procedure in the product insert.

The following table shows the age and gender distribution for individuals studied at site 4:

<table><tr><td rowspan=1 colspan=4>Site 4  Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>Sex Not Reported</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=4>Age</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>279</td><td rowspan=1 colspan=1>38</td></tr></table>

Of the 648 fresh respiratory specimens tested, all were nasal/nasopharyngeal swab specimens. 5 nasal/nasopharyngeal swab specimens were excluded from the performance analysis due to insufficient sample volume for the investigational device testing, 1 nasal/nasopharyngeal swab specimen was excluded from the performance analysis due to un-interpretable result generated by the investigational device because of high background. 93 samples for Flu A, 72 samples for Flu B, 96 samples for RSV, 102 samples for adenovirus, 101 samples for HPIV, and 106 samples for hMPV, were excluded from the respective performance analysis due to insufficient sample volume for the comparator methods, resulting in a total of 549 fresh nasal wash/nasopharyngeal aspirate specimens for Flu A, 570 fresh nasal wash/nasopharyngeal aspirate specimens for Flu B, 546 fresh nasal wash/nasopharyngeal aspirate specimens for RSV, 540 fresh nasal wash/nasopharyngeal aspirate specimens for adenovirus, 541 fresh nasal wash/nasopharyngeal aspirate specimens for HPIV, and 536 fresh nasal wash/nasopharyngeal aspirate specimens for hMPV to be included in the respective performance analysis. The tables below summarized the study results of the claimed specimen type at study site 4:

<table><tr><td rowspan=1 colspan=4>Flu A</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>494</td><td rowspan=1 colspan=1>501</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>495</td><td rowspan=1 colspan=1>549</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>47/54</td><td rowspan=1 colspan=1>87.0%</td><td rowspan=1 colspan=1>75.1-94.6%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>494/495</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.9-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>202</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>341</td><td rowspan=1 colspan=1>368</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>570</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>201/228</td><td rowspan=1 colspan=1>88.2%</td><td rowspan=1 colspan=1>84.0-92.4%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>341/342</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.4-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealswab</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">517</td><td colspan="1" rowspan="1">517</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">517</td><td colspan="1" rowspan="1">546</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">88.1-100%</td></tr><tr><td>Specificity</td><td>517/517</td><td>100.0%</td><td>99.3-100%</td></tr><tr><td></td><td></td><td></td><td></td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>539</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>539</td><td rowspan=1 colspan=1>540</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>539/539</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>HPIV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>534</td><td rowspan=1 colspan=1>535</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>534</td><td rowspan=1 colspan=1>541</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>6/7</td><td rowspan=1 colspan=1>85.7%</td><td rowspan=1 colspan=1>42.1-99.6%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>534/534</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>509</td><td rowspan=1 colspan=1>521</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>509</td><td rowspan=1 colspan=1>536</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>15/27</td><td rowspan=1 colspan=1>55.6%</td><td rowspan=1 colspan=1>35.3-74.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>509/509</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3-100%</td></tr></table>

The following tables summarized study results from all clinical sites combined, stratified by the claimed specimen types:

<table><tr><td colspan="4" rowspan="1">Flu A</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">59</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">568</td><td colspan="1" rowspan="1">578</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">571</td><td colspan="1" rowspan="1">637</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">56/66</td><td colspan="1" rowspan="1">84.8%</td><td colspan="1" rowspan="1">73.9-92.5%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">568/571</td><td colspan="1" rowspan="1">99.5%</td><td colspan="1" rowspan="1">98.5-99.9%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>619</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>617</td><td rowspan=1 colspan=1>628</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>9/11</td><td rowspan=1 colspan=1>81.8%</td><td rowspan=1 colspan=1>48.2-97.7%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>617/617</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>RSV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>205</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>465</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>463</td><td rowspan=1 colspan=1>670</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>204/207</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>95.8-99.7%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>462/463</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.8-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>620</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>619</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>12/13</td><td rowspan=1 colspan=1>92.3%</td><td rowspan=1 colspan=1>64.0-99.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>619/619</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

<table><tr><td colspan="4" rowspan="1">HPIV</td></tr><tr><td colspan="1" rowspan="1">Freshnasal/nasopharyngealwash/aspirate</td><td colspan="3" rowspan="1">Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td colspan="1" rowspan="1">DHI DSFA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">27</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">599</td><td colspan="1" rowspan="1">601</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">603</td><td colspan="1" rowspan="1">628</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Sensitivity</td><td colspan="1" rowspan="1">23/25</td><td colspan="1" rowspan="1">92.0%</td><td colspan="1" rowspan="1">74.0-99.0%</td></tr><tr><td colspan="1" rowspan="1">Specificity</td><td colspan="1" rowspan="1">599/603</td><td colspan="1" rowspan="1">99.3%</td><td colspan="1" rowspan="1">98.3-99.8%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealwash/aspirate</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>639</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>614</td><td rowspan=1 colspan=1>694</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>55/80</td><td rowspan=1 colspan=1>68.8%</td><td rowspan=1 colspan=1>57.4-78.7%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>614/614</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu A</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>58</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>623</td><td rowspan=1 colspan=1>631</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>624</td><td rowspan=1 colspan=1>689</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>57/65</td><td rowspan=1 colspan=1>87.7%</td><td rowspan=1 colspan=1>77.2-94.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>623/624</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.1-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Flu B</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>204</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>478</td><td rowspan=1 colspan=1>506</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>479</td><td rowspan=1 colspan=1>710</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>203/231</td><td rowspan=1 colspan=1>87.9%</td><td rowspan=1 colspan=1>83.7-92.1%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>478/479</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.8-100%</td></tr></table>

<table><tr><td colspan="2">RSV</td></tr><tr><td>Fresh nasal/nasopharyngeal swab</td><td>Predicate DSFA (negatives followed by culture with DFA)</td></tr></table>

<table><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>646</td><td rowspan=1 colspan=1>647</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>646</td><td rowspan=1 colspan=1>686</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>39/40</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>86.8-99.9%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>646/646</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>679</td><td rowspan=1 colspan=1>679</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>679</td><td rowspan=1 colspan=1>680</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>679/679</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.5-100%</td></tr></table>

<table><tr><td rowspan=1 colspan=4>HPIV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Predicate DSFA (negatives followed byculture with DFA)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>667</td><td rowspan=1 colspan=1>668</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>667</td><td rowspan=1 colspan=1>681</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>13/14</td><td rowspan=1 colspan=1>92.9%</td><td rowspan=1 colspan=1>66.1-99.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>667/667</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

c. Retrospective Clinical studies   

<table><tr><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>Freshnasal/nasopharyngealswab</td><td rowspan=1 colspan=3>Comparator DSFA (negatives confirmedby a validated hMPV real-time RT-PCRfollowed by bi-directional sequencinganalysis comparator assay)</td></tr><tr><td rowspan=1 colspan=1>DHI DSFA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>651</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>631</td><td rowspan=1 colspan=1>675</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>24/44</td><td rowspan=1 colspan=1>54.5%</td><td rowspan=1 colspan=1>38.8-69.9%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>631/631</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.4-100%</td></tr></table>

Not applicable. d. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

In the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit multicenter prospective clinical study testing direct respiratory specimens, a total of 1519 eligible respiratory specimens were tested using the $\mathrm { \dot { D } } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit from four U.S. clinical laboratories across the United States during the 2008/2009 respiratory virus seasons (January 2008 – March 2009). Prevalence for each analyte (i.e., Flu A, Flu B, RSV, hMPV, Adenovirus, HPIV) as determined by the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit direct specimen testing varied from $7 . 1 \%$ to $8 . 8 \%$ by site and averaged $7 . 6 \%$ for Flu A; varied from $0 . 6 \%$ to $3 1 . 2 \%$ by site and averaged $1 4 . 0 \%$ for Flu B; varied from $4 . 5 \%$ to $3 1 . 0 \%$ by site and averaged $1 3 . 4 \%$ for RSV; varied from $2 . 2 \%$ to $8 . 4 \%$ by site and averaged $5 . 1 \%$ for hMPV; varied from $0 \%$ to $2 . 5 \%$ by site and averaged $0 . 9 \%$ for Adenovirus; and varied from $0 . 9 \%$ to $5 . 2 \%$ by site and averaged $2 . 6 \%$ for HPIV. The number and percentage of positive cases determined by the $\mathrm { D } ^ { 3 }$ FastPoint L-DFA Respiratory Virus Identification Kit direct specimen testing, calculated by age group, are presented in the following tables:

Site 1   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>TotalSpecimensEvaluated</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HPIV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7(35%)</td><td rowspan=1 colspan=1>1 (5%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (5%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>231</td><td rowspan=1 colspan=1>13 (5.6%)</td><td rowspan=1 colspan=1>2 (0.9%)</td><td rowspan=1 colspan=1>84 (36.4%)</td><td rowspan=1 colspan=1>14 (6.1%)</td><td rowspan=1 colspan=1>7 (3.0%)</td><td rowspan=1 colspan=1>8 (3.5%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>10 (14.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8 (11.4%)</td><td rowspan=1 colspan=1>6 (8.6%)</td><td rowspan=1 colspan=1>1 (1.4%)</td><td rowspan=1 colspan=1>1 (1.4%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (50%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>323</td><td rowspan=1 colspan=1>23 (7.1%)</td><td rowspan=1 colspan=1>2 (0.6%)</td><td rowspan=1 colspan=1>100 (31.0%)</td><td rowspan=1 colspan=1>21 (6.5%)</td><td rowspan=1 colspan=1>8 (2.5%)</td><td rowspan=1 colspan=1>10 (3.1%)</td></tr></table>

Site 2   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>TotalSpecimensEvaluated</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HPIV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td></tr><tr><td rowspan=1 colspan=1>0  1 month</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (16.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>1 (3.1%)</td><td rowspan=1 colspan=1>1 (3.1%)</td><td rowspan=1 colspan=1>13 (40.6%)</td><td rowspan=1 colspan=1>1 (3.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (10%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1 (10.0%)</td><td rowspan=1 colspan=1>1 (10.0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1 (25.0%)</td><td rowspan=1 colspan=1>1 (25.0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3 (30%)</td><td rowspan=1 colspan=1>1 (10%)</td><td rowspan=1 colspan=1>1 (10%)</td><td rowspan=1 colspan=1>1 (10%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1 (9.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (9.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (11.1%)</td><td rowspan=1 colspan=1>1 (11.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>(50%)</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>8 (7.6%)</td><td rowspan=1 colspan=1>4 (3.8%)</td><td rowspan=1 colspan=1>17 (16.2%)</td><td rowspan=1 colspan=1>5 (4.8%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (1.0%)</td></tr></table>

Site 3   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>TotalSpecimensEvaluated</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HPIV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>7 (25%)</td><td rowspan=1 colspan=1>1 (3.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>11 (4.4%)</td><td rowspan=1 colspan=1>2 (0.8%)</td><td rowspan=1 colspan=1>38 (15.1%)</td><td rowspan=1 colspan=1>24 (9.6%)</td><td rowspan=1 colspan=1>4 (1.6%)</td><td rowspan=1 colspan=1>19 (7.6%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>17 (14.8%)</td><td rowspan=1 colspan=1>1 (1.1%)</td><td rowspan=1 colspan=1>10 (11.5%)</td><td rowspan=1 colspan=1>5 (5.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (2.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>6 (26.1%)</td><td rowspan=1 colspan=1>1 (4.3%)</td><td rowspan=1 colspan=1>1 (4.3%)</td><td rowspan=1 colspan=1>2 (8.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (4.3%)</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1 (14.3%)</td><td rowspan=1 colspan=1>1 (14.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (14.3%)</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3 (23.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (7.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1 (8.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (8.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (16.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (33.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>443</td><td rowspan=1 colspan=1>39 (8.8%)</td><td rowspan=1 colspan=1>5 (1.1%)</td><td rowspan=1 colspan=1>56 (12.6%)</td><td rowspan=1 colspan=1>37 (8.4%)</td><td rowspan=1 colspan=1>4 (0.9%)</td><td rowspan=1 colspan=1>23 (5.2%)</td></tr></table>

Site 4   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>TotalSpecimensEvaluated</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HPIV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td></tr><tr><td rowspan=1 colspan=1>0 − 1 month</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>2 (3.1%)</td><td rowspan=1 colspan=1>15 (23.4%)</td><td rowspan=1 colspan=1>19 (29.7%)</td><td rowspan=1 colspan=1>2 (3.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (3.1%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>223</td><td rowspan=1 colspan=1>16 (7.2%)</td><td rowspan=1 colspan=1>103 (46.8%)</td><td rowspan=1 colspan=1>6 (2.7%)</td><td rowspan=1 colspan=1>6 (2.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3 (1.3%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>12 (8.7%)</td><td rowspan=1 colspan=1>39 (28.3%)</td><td rowspan=1 colspan=1>2 (1.4%)</td><td rowspan=1 colspan=1>1 (0.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (0.7%)</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>1 (2.2%)</td><td rowspan=1 colspan=1>12 (26.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (2.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>6 (12.5%)</td><td rowspan=1 colspan=1>8 (16.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (2.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>4 (11.4%)</td><td rowspan=1 colspan=1>5 (14.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2 (6.9%)</td><td rowspan=1 colspan=1>3 (10.32%)</td><td rowspan=1 colspan=1>1 (3.4%)</td><td rowspan=1 colspan=1>1 (3.4%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2 (11.8%)</td><td rowspan=1 colspan=1>2 (11.8%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (20%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (33.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>2 (5.1%)</td><td rowspan=1 colspan=1>14 (35.9%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (2.6%)</td><td rowspan=1 colspan=1>1 (2.6%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>648</td><td rowspan=1 colspan=1>48 (7.4%)</td><td rowspan=1 colspan=1>202 (31.2%)</td><td rowspan=1 colspan=1>29 (4.5%)</td><td rowspan=1 colspan=1>14 (2.2%)</td><td rowspan=1 colspan=1>1 (0.2%)</td><td rowspan=1 colspan=1>6 (0.9%)</td></tr></table>

All Sites Combined   

<table><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>TotalSpecimensEvaluated</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HPIV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td><td rowspan=1 colspan=1># positive(prevalence)</td></tr><tr><td rowspan=1 colspan=1>0 - 1 month</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15 (27.3%)</td><td rowspan=1 colspan=1>2 (3.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 1 month to 2 years</td><td rowspan=1 colspan=1>577</td><td rowspan=1 colspan=1>27 (4.7%)</td><td rowspan=1 colspan=1>20 (3.5%)</td><td rowspan=1 colspan=1>154 (26.7%)</td><td rowspan=1 colspan=1>41 (7.1%)</td><td rowspan=1 colspan=1>11 (1.9%)</td><td rowspan=1 colspan=1>29 (5.0%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 2 years to 12 years</td><td rowspan=1 colspan=1>391</td><td rowspan=1 colspan=1>43 (11.0%)</td><td rowspan=1 colspan=1>104 (26.6%)</td><td rowspan=1 colspan=1>25 (6.4%)</td><td rowspan=1 colspan=1>17 (4.3%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>6 (1.5%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 12 years to 21 years</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>19 (11.0%)</td><td rowspan=1 colspan=1>41 (23.7%)</td><td rowspan=1 colspan=1>4 (2.3%)</td><td rowspan=1 colspan=1>3 (1.7%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (1.2%)</td></tr><tr><td rowspan=1 colspan=1>22 years to 30 years</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>3 (5.3%)</td><td rowspan=1 colspan=1>14 (24.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (1.8%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (1.8%)</td></tr><tr><td rowspan=1 colspan=1>31 years to 40 years</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>9 (12.7%)</td><td rowspan=1 colspan=1>9 (12.7%)</td><td rowspan=1 colspan=1>1 (1.4%)</td><td rowspan=1 colspan=1>3 (4.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>41 years to 50 years</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>5 (9.6%)</td><td rowspan=1 colspan=1>5 (9.6%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (1.9%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>51 years to 60 years</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>3 (6.5%)</td><td rowspan=1 colspan=1>3 (6.5%)</td><td rowspan=1 colspan=1>1 (2.2%)</td><td rowspan=1 colspan=1>3 (6.5%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>61 years to 70 years</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>2 (6.1%)</td><td rowspan=1 colspan=1>2 (6.1%)</td><td rowspan=1 colspan=1>1 (3.0%)</td><td rowspan=1 colspan=1>1 (3.0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>71 years to 80 years</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>2 (12.5%)</td><td rowspan=1 colspan=1>1 (6.3%)</td><td rowspan=1 colspan=1>1 (6.3%)</td><td rowspan=1 colspan=1>4 (25.0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>81 years and above</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (14.3%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (14.3%)</td></tr><tr><td rowspan=1 colspan=1>Age Not Reported</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>2 (4.9%)</td><td rowspan=1 colspan=1>14 (34.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (2.4%)</td><td rowspan=1 colspan=1>1 (2.4%)</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1519</td><td rowspan=1 colspan=1>115 (7.6%)</td><td rowspan=1 colspan=1>213 (14.0%)</td><td rowspan=1 colspan=1>203 (13.4%)</td><td rowspan=1 colspan=1>77(5.1%)</td><td rowspan=1 colspan=1>13 (0.9%)</td><td rowspan=1 colspan=1>40 (2.6%)</td></tr></table>

Note: There were seven (7) co-infections detected: $2 \cdot \mathrm { R S V } + \mathrm { h M P V } , 2$ 2- adenovirus $^ +$ RSV, 2- influenza $\mathbf { A } +$ hMPV, 1-RSV $^ +$ adenovirus and 1-RSV $^ +$ parainfluenza virus

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.