# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K091053 B. Purpose for Submission:

This is a new 510k application for a qualitative real-time reverse transcription-polymerase chain reaction (RT-PCR) assay used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a for the qualitative detection and discrimination of Parainfluenza 1 Virus, Parainfluenza 2 Virus and Parainfluenza 3 Virus (HPIV-1, HPIV-2 and HPIV-3) nucleic acids isolated and purified from nasopharyngeal swab (NP) specimens obtained from symptomatic patients. The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# C. Measurand:

Target RNA sequences for the respective conserved regions of the HemagglutininNeuraminidase (HN) gene for HPIV-1, HPIV-2 and HPIV-3, and for the transcript derived from $E .$ . coli Bacteriophage MS2 A-protein gene (Internal Control).

# D. Type of Test:

Real-time reverse transcription-polymerase chain reaction (RT-PCR), qualitative detection and discrimination of Parainfluenza 1 Virus, Parainfluenza 2 Virus and Parainfluenza 3 Virus (HPIV-1, HPIV-2 and HPIV-3) in nasopharyngeal swabs using nucleic acid isolation (The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux)), amplification and detection on the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a, which generates signals based on the acquisition of spectrofluorometric data.

E. Applicant: Prodesse Incorporated   
F. Proprietary and Established Names: Prodesse ProParaflu $^ +$ Assay   
G. Regulatory Information:

1. Regulation section: 866.3980

2. Classification: Class II

3. Product code: OOU

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The ProParaflu $^ +$ Assay is a multiplex Real Time RT-PCR in vitro diagnostic test for the qualitative detection and discrimination of Parainfluenza 1 Virus, Parainfluenza 2 Virus and Parainfluenza 3 Virus (HPIV-1, HPIV-2 and HPIV-3) nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens obtained from individuals exhibiting signs and symptoms of respiratory tract infections. This assay targets the conserved regions of the Hemagglutinin-Neuraminidase (HN) gene of HPIV-1, HPIV2 and HPIV-3, respectively. The detection and discrimination of HPIV-1, HPIV-2 and HPIV-3 nucleic acids from symptomatic patients aid in the diagnosis of human respiratory tract parainfluenza infections if used in conjunction with other clinical and laboratory findings. This test is not intended to detect Parainfluenza 4a or Parainfluenza 4b Viruses.

Negative test results are presumptive and should be confirmed by cell culture. Negative results do not preclude Parainfluenza 1, 2 or 3 virus infections and should not be used as the sole basis for treatment or other management decisions.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements:

To be used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a and either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# I. Device Description:

The ProParaflu $^ +$ Assay is a multiplex Taqman based Real Time RT-PCR Assay that enables detection and differentiation of Parainfluenza 1 Virus, Parainfluenza 2 Virus, Parainfluenza 3 Virus and Internal Control.

An overview of the procedure is as follows:

1. Collect nasopharyngeal swab specimens from symptomatic patients using a polyester, rayon or nylon tipped swab and place into viral transport medium (not provided with kit).

2. Add an Internal Control (IC) to every sample to monitor for inhibitors present in the specimens.

3. Perform isolation and purification of nucleic acids using a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

4. Add purified nucleic acids to ProParaflu $^ +$ Supermix along with enzymes included in the assay kit. The ProParaflu $^ +$ Supermix contains oligonucleotide primers and targetspecific oligonucleotide probes. The primers are complementary to conserved regions of the Hemagglutinin-Neuraminidase (HN) genetic sequences for HPIV-1, HPIV-2 and HPIV-3 viruses. The probes are dual-labeled with a reporter dye attached to the $5 '$ -end and a quencher dye attached to the $3 ^ { \circ }$ -end (see table below).

5. Perform reverse transcription of RNA into complementary DNA (cDNA) and subsequent amplification of DNA in a Cepheid SmartCycler® II instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProParaflu $^ +$ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification product present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

ProParaflu $+$ Assay Analyte Gene Targets and Probe Labels:   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Gene Targeted</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=1>EmissionPeak</td><td rowspan=1 colspan=1>InstrumentChannel</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1Virus</td><td rowspan=1 colspan=1>Hemagglutininneuraminidase</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3Virus</td><td rowspan=1 colspan=1>Hemagglutininneuraminidase</td><td rowspan=1 colspan=1>Cal Orange 560</td><td rowspan=1 colspan=1>540 nm</td><td rowspan=1 colspan=1>561 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2Virus</td><td rowspan=1 colspan=1>Hemagglutininneuraminidase</td><td rowspan=1 colspan=1>Cal Red 610</td><td rowspan=1 colspan=1>595 nm</td><td rowspan=1 colspan=1>615 nm</td><td rowspan=1 colspan=1>Texas Red</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>667 nm</td><td rowspan=1 colspan=1>Cy5</td></tr></table>

# Materials Provided

ProParaflu+ Assay Kit (Cat. # H81VK00)   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity/Tube</td><td rowspan=1 colspan=1>CapColor</td><td rowspan=1 colspan=1>Cat. #</td><td rowspan=1 colspan=1>Reactions/Tube</td></tr><tr><td rowspan=1 colspan=1>ProParaflu+Supermix</td><td rowspan=1 colspan=1>•  Taq DNA polymerase•  4 oligonucleotide primer pairs• 4 oligonucleotide probes•  Buffer containing dNTPs (dATP,dCTP, dGTP, dTTP),•  MgCl2 and stabilizers</td><td rowspan=1 colspan=1>1030 μL</td><td rowspan=1 colspan=1>Brown</td><td rowspan=1 colspan=1>HSM81</td><td rowspan=1 colspan=1>50(2 tubesprovided)</td></tr><tr><td rowspan=1 colspan=1>M-MLV ReverseTranscriptase II</td><td rowspan=1 colspan=1>•  11.4 U/μL</td><td rowspan=1 colspan=1>36 μL</td><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>GLS32</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RNase Inhibitor II</td><td rowspan=1 colspan=1>• 40 U/μL</td><td rowspan=1 colspan=1>120 μL</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>GLS33</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaRNA Control</td><td rowspan=1 colspan=1>•  Non-infectious in vitro transcribedRNA specific viral sequences</td><td rowspan=1 colspan=1>500 μL</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>HCT81</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Internal RNAControl III</td><td rowspan=1 colspan=1>•  Non-infectious in vitro transcribedRNA</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>GCT12</td><td rowspan=1 colspan=1>100</td></tr></table>

# Materials Required But Not Provided

# Plasticware and consumables

Polyester, rayon or nylon tipped nasopharyngeal swabs   
• RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes   
• Sterile RNase/DNase-free filter or positive displacement micropipettor tips MagNA Pure LC System Disposables (Reagent Tubs, Reaction Tips, Tip Trays, Cartridges) or easyMAG System Disposables (Sample Vessels and Tips) Biohit Pipette Tips for use with easyMAG System   
• Greiner Break Four uncoated plates for use with easyMAG System   
• Cepheid PCR reaction tubes, $2 5 ~ \mu \mathrm { L }$ Parafilm $^ { \textregistered } \mathbf { M }$ or MagNA Pure LC Cartridge Seals

# Reagents

Roche MagNA Pure LC Total Nucleic Acid Isolation Kit (Roche Cat.   
No.03038505001) for 192 isolations or bioMérieux NucliSENS easyMAG reagents (Buffer 1 Cat. No. 280130, Buffer 2 Cat. No. 280131, Buffer 3 Cat. No. 280132, Magnetic Silica Cat. No. 280133, and Lysis Buffer Cat. No. 280134)   
Micro TestTM ${ \bf M } 4 ^ { \mathbb { B } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12500), Micro TestTM $\mathbf { M } \boldsymbol { 5 } ^ {  { \textregistered } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12515), Micro TestTM $\mathbf { M } 6 ^ { \mathbf { \mathfrak { B } } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12530), Micro TestT M M4RT® Viral Transport Medium (Remel, Inc. Cat. No. R12505), BD Universal Viral Transport medium (UVT; Becton, Dickinson and Co. Cat. No. 220223) or Copan Universal Transport Medium (Copan Cat. No. 330C)   
Molecular Grade Water (RNase/DNase Free)

 Extraction Control (e.g. previously characterized positive sample or negative sample spiked with a well characterized HPIV-1, HPIV-2 or HPIV-3 strain)

# Equipment

• - $\mathrm { \cdot 7 0 ^ { \circ } C }$ Freezer Roche MagNA Pure LC System with software version 3.0.11 or bioMérieux NucliSENS easyMAG System with Software version 1.0.1 or 2.0   
• Biohit multi-channel pipettor for use with easyMAG System   
• Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a   
• Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { \cdot } 1 0 0 0 \ \mu \mathrm { L }$   
• Mini-centrifuge with adapter for Cepheid Reaction Tubes   
• Cepheid cooling block

# Interpretation of Sample Results

The SmartCycler Dx software automatically determines the specimen results. The interpretation of the assay specimen results is as follows:

<table><tr><td colspan="1" rowspan="1">SampleID1</td><td colspan="1" rowspan="1">AssayResult</td><td colspan="1" rowspan="1">ICResult</td><td colspan="1" rowspan="1">Warning/ErrorCode</td><td colspan="1" rowspan="1">HPIV-1Result</td><td colspan="1" rowspan="1">HPIV-2Result</td><td colspan="1" rowspan="1">HPIV-3Result</td><td colspan="1" rowspan="1">Interpretation of Results</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Pass</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">HPIV-1, -2 and -3 nucleicacid not detected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">HPIV-1 nucleic aciddetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">HPIV-2 nucleic aciddetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">HPIV-3 nucleic aciddetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Unresolved</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Unresolved - PCR inhibitionor reagent failure. Repeattesting from the purifiednucleic acid, re-test fromoriginal sample, or collectand test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">ND2</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">30792</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Not Determined  error code3079</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">HPIV-1 and HPIV-3 nucleicacid detected. Multipleinfections are rare. Repeattesting from the purifiednucleic acid or re-test fromoriginal sample, or collectand test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">HPIV-1 and HPIV-2 nucleicacid detected. Multiple</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">infections are rare. Repeattesting from the purifiednucleic acid, re-test fromoriginal sample, or collectand test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">HPIV-1, HPIV-2 and HPIV-3 nucleic acid detected.Multiple infections are rare.Repeat testing from thepurified nucleic acid, re-testfrom original sample, orcollect and test a newsample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">40983</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Not Determined - error code4098</td></tr></table>

1 Columns and data not used for interpretation are not included 2 Error Code 3079: Warning/Error Code 3079 is periodically observed. Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for the sample are reported by the Dx software as ND (Not Determined). If a $C t$ value $\geq I 3$ is reported in any analyte column, results can be recorded as POSITIVE for that analyte. 3 An Invalid Assay run will display Error code 4098 \* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High viral load can lead to reduced or absent Internal Control signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ID-Tag Respiratory Virus Panel, Luminex Molecular Diagnostics, Inc.

2. Predicate K number(s): K063765

3. Comparison with predicates:

Both assays detect HPIV-1, HPIV-2 and HPIV-3 using nucleic acid amplification techniques. Both assays use nasal pharyngeal swabs as the collection device and the MagNA Pure LC system for nucleic acid isolation. In addition, the ProParaflu $^ +$ Assay can also utilize the bioMérieux NucliSENS easyMAG System for nucleic acid extraction. The detection system with both assays involves spectrophotometric detection. The assays differ in that the predicate is a multiplex Nucleic Acid amplification and end-point detection assay also detects Influenza A, Influenza B, RSV, Influenza A subtypes H1 and H3, hMPV, Rhinovirus, and Adenovirus. The ProParaflu $^ +$ assay is a multiplex real-time Nucleic Acid amplification and detection assay detects HPIV-1, HPIV-2 and HPIV-3 only.

# K. Standard/Guidance Document Referenced (if applicable):

Guidance on Class II Special Controls Guidance Document: Nucleic Acid Amplification   
Assay for the Detection of Enterovirus RNA (January 2009) –   
http://www.fda.gov/cdrh/oivd/guidance/1665.html

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) – http://www.fda.gov/cdrh/oivd/guidance/1588.html Draft Guidance on Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens (Dec 2005) – http://www.fda.gov/cdrh/oivd/guidance/1560.html Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Guidance for Industry and FDA Reviewers (March 2007) – http://www.fda.gov/cdrh/osb/guidance/1620.html Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff – http://www.fda.gov/cdrh/ode/guidance/1567.html Guidance for Off-the-Shelf Software Use in Medical Devices; Final (Sept 1999) – http://www.fda.gov/cdrh/ode/guidance/585.html Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses (Feb 2008) – http://www.fda.gov/cdrh/oivd/guidance/1638.pdf CLSI EP17-A: Guidance for Protocols for Determination of Limits of Detection and Limits of Quantitation (Vol. 2, No. 34) (Oct 2004). CLSI MM13-A: Guidance for the Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (Vol. 25, No. 31) (Dec 2005). • CLSI EP7-A2: Guidance for Interference Testing in Clinical Chemistry (Vol. 25, No.27 Second Ed) (Nov 2005). CLSI EP12-A: Guidance for User Protocol for Evaluation of Qualitative Test Performance (Vol. 22, No. 14) (Sept 2002). CLSI MM3-A2: Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline (Vol. 26, No. 8) (Feb 2006).

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. The ProParaflu $^ +$ Assay enables the detection and differentiation of human Parainfluenza virus (HPIV) types 1, 2, and 3 and an Internal Control and is based on three processes: nucleic acid isolation, reverse transcription (RT) and Real Time polymerase chain reaction (PCR) amplification/detection. The Internal Control (IC) is added to each nasopharyngeal (NP) swab from symptomatic patients to monitor for inhibitors present. Nucleic acids are isolated and purified from the NP swab sample. Purified nucleic acid is added to the ProParaflu+ Supermix (includes Taq polymerase) along with the appropriate enzymes. The ProParaflu $^ +$ Supermix contains oligonucleotide primers complementary to highly conserved regions of the hemagglutinin-neuraminidase gene for HPIV-1, 2, and 3 and target-specific oligonucleotide probes dual-labeled with a reporter dye attached to the $5 ^ { \circ }$ end and a quencher dye attached to the $3 ^ { \circ }$ end. After initial reverse transcription of RNA into complementary DNA (cDNA), amplification proceeds during which the primers and probe anneal specifically to the template (if present) followed by primer extension and amplification. The ProParaflu $^ +$ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing the fluorescent signal. The

amount of fluorescence at any given cycle is dependent on the amount of amplification product present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility Study

An Inter-Laboratory Reproducibility study was conducted by extracting and testing a panel of 9 samples at three sites performed by two operators at each site for 5 days per operator (for a total of 10 runs per site). The Reproducibility Panel used in the study was prepared by spiking nasopharyngeal (NP) swab pools with cultured and titered stock solutions of human Parainfluenze virus (HPIV) types 1, 2 and 3 at low positive $( 2 \mathrm { ~ x ~ L o D } )$ , medium positive $1 0 \mathrm { { x } \mathrm { { L o D ) } } }$ or high negative $( 0 . 0 1 { - } 0 . 0 0 1 \mathrm { ~ x ~ L o D } )$ concentrations. Panel members were coded and randomly sorted to make 9- sample test panels and to ensure sample identification was unknown to the operators. The 9-sample test panels were stored at $\mathrm { \leq - 7 0 ^ { \mathrm { o } } C }$ and shipped to the sites frozen.

A set of ten panels (90 total samples) was provided to each laboratory; one set was used for each of the 5 testing days per operator. Each operator thawed one panel set per day, spiked the samples and a Negative Control with the Internal Control, and performed nucleic acid extraction on the panel samples using either the Roche MagNA Pure LC system (Site #3) or the bioMérieux NucliSENS easyMAG (Site #1 and Site #2). An Extraction Control, a well characterized human parainfluenza 3 virus, was also included with each nucleic acid isolation run. Extracted nucleic acid was tested with the ProParaflu $^ +$ Assay on the Cepheid SmartCycler II. The Positive Parainfluenza RNA Control was also included with each PCR panel run. Each of two operators performed these activities on five days for a total of 10 reproducibility runs per site. A single lot of ProParaflu+ reagents was used. A total of 358 data points were included in the reproducibility study data analysis (12 samples and controls/run X 1 run/day/operator $_ { \textrm { X 2 } }$ operators X 5 days X 3 sites $= 3 6 0$ ; 2 purified nucleic acid samples from site 1 (1 HPIV-1 medium positive and 1 HPIV-2 low positive) were mistakenly combined following extraction. These 2 samples were excluded from analysis). The total percent agreement for the ProParaflu $^ +$ Assay was $9 7 . 8 \%$ . The ProParaflu $^ +$ Assay is a qualitative assay based partially on numerical Cycle Threshold $\mathrm { ( C t ) }$ values. The overall Ct value $\% \mathrm { C V }$ across all sites for all samples and controls ranged from $1 . 5 7 \%$ to $8 . 1 3 \%$ depending upon analyte type, target type, and concentration tested.

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>PanelMember ID</td><td rowspan=2 colspan=1>  </td><td rowspan=2 colspan=1>Jod Mo I-AIdH</td><td rowspan=2 colspan=1>Hd n  </td><td rowspan=2 colspan=1> H-AIdH</td><td rowspan=2 colspan=1>ood aIH-IdY</td><td rowspan=2 colspan=1>H n -IdH</td><td rowspan=2 colspan=1>h E-AH</td><td rowspan=2 colspan=1>etMAIE-AI</td><td rowspan=2 colspan=1> n-IH</td><td rowspan=2 colspan=1>Car cr cd</td><td rowspan=1 colspan=3>Para RNAControl</td><td rowspan=2 colspan=1>Ne et</td><td rowspan=3 colspan=1>Loe % Jnh</td></tr><tr><td rowspan=1 colspan=2>I-AIdH  H-AIdT</td><td rowspan=1 colspan=1>E-AIdH</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>0.001xLoD</td><td rowspan=1 colspan=1>2 XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.001xLoD</td><td rowspan=1 colspan=1>2 XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.01xLoD</td><td rowspan=1 colspan=1>2 XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=3 colspan=1>Site 1</td><td rowspan=1 colspan=1>AgreementwithExpectedResult</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>8b/1080%</td><td rowspan=1 colspan=1>9c/9100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>9c/9100%</td><td rowspan=1 colspan=1>9d/1090%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>9/1090%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=3>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>114/11896.6%</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>27.73</td><td rowspan=1 colspan=1>28.31</td><td rowspan=1 colspan=1>26.33</td><td rowspan=1 colspan=1>27.90</td><td rowspan=1 colspan=1>28.62</td><td rowspan=1 colspan=1>26.27</td><td rowspan=1 colspan=1>27.76</td><td rowspan=1 colspan=1>31.21</td><td rowspan=1 colspan=1>29.47</td><td rowspan=1 colspan=1>27.33</td><td rowspan=1 colspan=1>27.37</td><td rowspan=1 colspan=1>29.37</td><td rowspan=1 colspan=1>28.61</td><td rowspan=1 colspan=1>27.69</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>1.60</td><td rowspan=1 colspan=1>2.84</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>2.43</td><td rowspan=1 colspan=1>3.21</td><td rowspan=1 colspan=1>1.91</td><td rowspan=1 colspan=1>1.60</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>1.67</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>AgreementwithExpectedResult</td><td rowspan=1 colspan=1>8/1080%</td><td rowspan=1 colspan=1>8b/1080%%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/1000%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=3>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>116/12096.7%</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>28.59</td><td rowspan=1 colspan=1>28.47</td><td rowspan=1 colspan=1>26.12</td><td rowspan=1 colspan=1>28.83</td><td rowspan=1 colspan=1>28.91</td><td rowspan=1 colspan=1>26.61</td><td rowspan=1 colspan=1>28.30</td><td rowspan=1 colspan=1>31.64</td><td rowspan=1 colspan=1>29.51</td><td rowspan=1 colspan=1>27.56</td><td rowspan=1 colspan=1>23.86</td><td rowspan=1 colspan=1>26.09</td><td rowspan=1 colspan=1>25.23</td><td rowspan=1 colspan=1>28.68</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>2.80</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>1.83</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>2.17</td><td rowspan=1 colspan=1>2.66</td><td rowspan=1 colspan=1>2.47</td><td rowspan=1 colspan=1>1.48</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=4 colspan=1>Site 3</td><td rowspan=2 colspan=1>AgreementwithExpectedResult</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=2 colspan=1>10/1010%</td><td rowspan=2 colspan=1>10/10100%</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=2 colspan=3>10/10100%</td><td rowspan=2 colspan=1>10/1010%</td><td rowspan=2 colspan=1>120/120100%</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Average CtValue</td><td rowspan=1 colspan=1>26.35</td><td rowspan=1 colspan=1>29.91</td><td rowspan=1 colspan=1>27.67</td><td rowspan=1 colspan=1>26.28</td><td rowspan=1 colspan=1>29.51</td><td rowspan=1 colspan=1>27.44</td><td rowspan=1 colspan=1>26.67</td><td rowspan=1 colspan=1>33.13</td><td rowspan=1 colspan=1>30.43</td><td rowspan=1 colspan=1>28.61</td><td rowspan=1 colspan=1>28.73</td><td rowspan=1 colspan=1>30.98</td><td rowspan=1 colspan=1>29.84</td><td rowspan=1 colspan=1>26.46</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>2.25</td><td rowspan=1 colspan=1>4.13</td><td rowspan=1 colspan=1>2.36</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>3.31</td><td rowspan=1 colspan=1>3.39</td><td rowspan=1 colspan=1>3.19</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwithExpectedResult</td><td rowspan=1 colspan=1>28/3093.3%</td><td rowspan=1 colspan=1>26/3086.7%</td><td rowspan=1 colspan=1>29/29100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/29100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>29/3096.7%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=3>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>350/35897.8%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>8.7%98.2%</td><td rowspan=1 colspan=1>70.3%-94.7%</td><td rowspan=1 colspan=1>88.3%-100%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>88.3%-100%</td><td rowspan=1 colspan=1>83.3%-99.4%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>83.3%-99.%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=3>88.6% - 100%</td><td rowspan=1 colspan=1>88.6%-100%</td><td rowspan=1 colspan=1>95.6% -98.9%</td></tr><tr><td rowspan=1 colspan=1>OverallAverage CtValue</td><td rowspan=1 colspan=1>27.56</td><td rowspan=1 colspan=1>28.90</td><td rowspan=1 colspan=1>26.72</td><td rowspan=1 colspan=1>27.67</td><td rowspan=1 colspan=1>29.03</td><td rowspan=1 colspan=1>26.79</td><td rowspan=1 colspan=1>27.58</td><td rowspan=1 colspan=1>32.02</td><td rowspan=1 colspan=1>29.80</td><td rowspan=1 colspan=1>27.83</td><td rowspan=1 colspan=1>26.65</td><td rowspan=1 colspan=1>28.81</td><td rowspan=1 colspan=1>27.89</td><td rowspan=1 colspan=1>27.62</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>3.88</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>2.95</td><td rowspan=1 colspan=1>4.55</td><td rowspan=1 colspan=1>1.57</td><td rowspan=1 colspan=1>2.54</td><td rowspan=1 colspan=1>3.68</td><td rowspan=1 colspan=1>3.61</td><td rowspan=1 colspan=1>2.42</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>8.13</td><td rowspan=1 colspan=1>7.49</td><td rowspan=1 colspan=1>7.39</td><td rowspan=1 colspan=1>3.54</td></tr></table>

a Average Ct value for the Internal Control (IC) b Two samples positive for both HPIV-1 (true) and HPIV-3 (false). c Two purified nucleic acid samples (one HPIV-1 medium positive and one HPIV-2 low positive) mistakenly combined following extraction. These samples were excluded from data analysis. d One sample positive for both HPIV-2 (true) and HPIV-3 (false). e One sample false negative for HPIV-3. f Two samples false positive for HPIV-1.

Four HPIV-1 positive samples (two at Site 1 and two at Site 2) were also positive for HPIV-3. All 4 of these samples registered a 3079 error (Fluorescence Signal Too High) and had a distinct shaped growth curve. The sponsor believed that the false positive HPIV-3 signal is an artifact of the SmartCycler Dx software due to signal bleed-over. This limitation is considered to be low risk, as it is most

important to detect HPIV and there are no differences in treatment of HPIV based on HPIV type. This limitation was addressed in the following two ways: 1) In the Interpretation of Specimen Results section of the ProParaflu+ Assay Instructions for Use it is stated that “dual or multiple parainfluenza infections are rare, and they may be artifacts of the SmartCycler Dx software due to signal bleed-over. It is required that repeat testing for these samples be performed starting from the purified nucleic acid, original sample, or a newly collected sample.” 2) In the Limitation section of the ProParaflu $^ +$ Assay Instructions for Use it is also stated that “dual or multiple parainfluenza infections are rare, and they may be artifacts of the SmartCycler Dx software due to signal bleed-over. It is required that repeat testing for these samples be performed starting from the purified nucleic acid, original sample, or a newly collected sample.”

Also, there were two false HPIV-1 positive samples detected in HPIV-1 High negative samples at Site 2. The Ct values for these two samples were 41.6 and 38.8. Low level carry-over/cross contamination could have occurred during creation of the sample, during sample preparation for extraction, during extraction or when transferring the purified nucleic acid samples from the sample cartridge to microfuge tubes. (Note: since the high-negative samples include a low amount of HPIV that is detectable no more than $5 \%$ of the time, the possibility of these two samples actually being samples that fall in the $5 \%$ category can not be ruled out). This limitation was addressed by including the following statements in the Limitation section of the ProParaflu+ Assay Instructions for Use: “The detection of viral nucleic acid is dependent upon proper specimen collection, handling, transportation, storage, and preparation (including extraction). Failure to observe proper procedures in any one of these steps can lead to incorrect results.”; “There is a risk of false positive values resulting from crosscontamination by target organism or its nucleic acids.”

# Supplemental Reproducibility Study

Due to the fact that the clinical trial for the ProParaflu $^ +$ Assay resulted in more than $10 \%$ of the clinical samples that were positive by the ProParaflu+ Assay with $\mathrm { C t }$ values beyond the mean Ct values of the LoDs (approximately $3 1 ~ \mathrm { C t }$ for HPIV1, $2 9 \ \mathrm { C t }$ for HPIV-2 and $3 3 \mathrm { \ C t }$ for HPIV-3), but before the assay cutoff at 45 cycles, a supplemental Inter-Laboratory Reproducibility Study was also conducted at three sites to evaluate inter-laboratory reproducibility of the ProParaflu $^ +$ Assay (including variables of operator, instrument, location, and time) with samples having a concentration below the assay’s Limit of Detection (LoDs) but above the High Negatives previously tested in the reproducibility study (i.e. “intermediate samples”). The Reproducibility Panel consisted of 3 nasopharyngeal (NP) swab samples, one each spiked with HPIV-1, HPIV-2 or HPIV-3 at 1 log below the LoD. Panel members were coded, stored at $\le - 7 0 ^ { \mathrm { o } } \mathrm { C }$ , and shipped to the sites frozen.

A set of ten panels (30 total samples) was provided to each laboratory; one set was used for each of the 5 testing days per operator. Each operator thawed one panel set per day, spiked the samples and Negative Control with the Internal Control, and performed nucleic acid extraction using either the bioMérieux NucliSENS easyMAG (Sites 1 and 2) or the Roche MagNA Pure LC (Site 3). A Negative Control, Extraction Control (a well characterized human parainfluenza 3 virus) and the Parainfluenza Positive RNA Control were included with each panel run. Extracted nucleic acids were tested with the ProParaflu+ Assay on the Cepheid SmartCycler II. Each of two operators performed these activities on five days for a total of 10 supplemental reproducibility runs per site. A single lot of ProParaflu+ reagents was used. A total of 180 data points were included in the supplemental reproducibility study data analysis (6 samples and controls/run X 1 run/day/operator $\textrm { X } 2$ operators $\mathrm { ~ X ~ } 5$ days $\textrm { X 3 }$ sites $= 1 8 0$ ). The supplemental reproducibility study data is presented in the following table and are the total number of test results that agree with the expected results and the percent $( \% )$ agreement. Although some fractions of the intermediate samples were anticipated to be negative since the concentration was below the LoD, the $\%$ agreement was calculated for an “expected” positive outcome:

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=2 colspan=1>Hrn r</td><td rowspan=2 colspan=1>HrnHr</td><td rowspan=2 colspan=1>Hrn d</td><td rowspan=2 colspan=1>Ca.a Co c</td><td rowspan=1 colspan=3>Parainfluenza RNAControl</td><td rowspan=2 colspan=1>No e</td></tr><tr><td rowspan=1 colspan=1>I-AIdH</td><td rowspan=1 colspan=1>7-AIdH</td><td rowspan=1 colspan=1>E-AIdH</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>0.1 X LoD</td><td rowspan=1 colspan=1>0.1 X LoD</td><td rowspan=1 colspan=1>0.1 X LoD</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=3 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement with PositiveResult</td><td rowspan=1 colspan=1>4/1040%</td><td rowspan=1 colspan=1>8/1080%</td><td rowspan=1 colspan=1>1/1010%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=3>10/10100%</td><td rowspan=1 colspan=1>10/10*100%</td></tr><tr><td rowspan=1 colspan=1>Average Ct Value</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>28.4</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>6.19</td><td rowspan=1 colspan=1>2.78</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.80</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>3.47</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>Agreement with PositiveResult</td><td rowspan=1 colspan=1>8/1080%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=1>7/1070%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=3>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td></tr><tr><td rowspan=1 colspan=1>Average Ct Value</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>27.7</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>2.22</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.80</td></tr><tr><td rowspan=3 colspan=1>Site 3</td><td rowspan=1 colspan=1>Agreement with PositiveResult</td><td rowspan=1 colspan=1>5/1050%</td><td rowspan=1 colspan=1>8/1080%</td><td rowspan=1 colspan=1>1/1010%</td><td rowspan=1 colspan=1>10/10100%</td><td rowspan=1 colspan=3>10/10100%</td><td rowspan=1 colspan=1>10/10100%</td></tr><tr><td rowspan=1 colspan=1>Average Ct Value</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>27.9</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>2.90</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>1.69</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>1.94</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Total Agreement withPositive Result</td><td rowspan=1 colspan=1>17/3056.7%</td><td rowspan=1 colspan=1>26/3086.7%</td><td rowspan=1 colspan=1>9/3030.0%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=3>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>39.2% -72.6%</td><td rowspan=1 colspan=1>70.3% -94.7%</td><td rowspan=1 colspan=1>16.7% -47.9%</td><td rowspan=1 colspan=1>88.7100%</td><td rowspan=1 colspan=3>88.7 - 100%</td><td rowspan=1 colspan=1>88.7-100%</td></tr><tr><td rowspan=1 colspan=1>Overall Average CtValue</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>28.0</td></tr><tr><td rowspan=1 colspan=1>Overall % CV</td><td rowspan=1 colspan=1>3.68</td><td rowspan=1 colspan=1>3.25</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>3.46</td><td rowspan=1 colspan=1>1.91</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>2.92</td></tr></table>

a Average Ct value for the Internal Control (IC)

The percent positive for the “intermediate samples” across all sites was $5 6 . 7 \%$ for HPIV-1 (mean $\mathrm { C t } = 3 5 . 1 $ ), $8 6 . 7 \%$ for HPIV-2 (mean $\mathrm { C t } = 3 3 . 0 $ ), and $3 0 . 0 \%$ for HPIV-3 (mean $\mathrm { C t } = 3 7 . 1 $ ). This result was expected as the “intermediate concentration” should be positive between $5 \% - 9 5 \%$ (the “intermediate samples” were of lower concentration than the LoD concentration (defined $a s \ge 9 5 \%$ positive) and higher than the High Negative concentration (defined as $< 5 \%$ positive)). This supplemental reproducibility data demonstrated that as expected with all real-time PCR assays, the ProParaflu $^ +$ Assay may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration. This limitation should be addressed by including the following statement in the Limitation section of the ProParaflu $^ +$ Assay Instructions for Use: “The ProParaflu $^ +$ Assay may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration.”

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

The following controls are provided in the ProParaflu $^ +$ Assay kit:

Positive Control (PC): The ProParaflu $^ +$ Assay kit contains a parainfluenza positive RNA control that consists of three RNA transcripts (specific for each HPIV type 1, 2, and 3). These transcripts are derived from plasmids containing the viral sequences of interest and are not intact virus particles. The Parainfluenza PC contains unequal amounts of HPIV-1, HPIV-2 and HPIV-3. The approximate concentration of each transcript was $5 \mathrm { x } 1 0 ^ { 4 }$ copies ${ \bf \Pi } ^ { \prime } \mu \lambda .$ , $2 . 5 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies $\mathbf { \mu } ^ { \prime } \mathbf { \mu } \mathbf { l }$ and $2 . 5 \mathrm { \bf ~ X }$ $1 0 ^ { 3 }$ copies $\mathbf { \mu } ^ { \prime } \mathbf { \mu } \mathbf { l }$ for HPIV-1, HPIV-2 and HPIV-3, respectively. The PC does not go through nucleic acid isolation and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance. The PC is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc.). Because the PC is assessing global failure and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable range was set to 20.0 to 40.0 in the Clinical Trial Protocol for all HPIV targets to allow for dilution variability (e.g., pipetting) of the PC by the end users. Of a total of 94 PC tested by the ProParaflu $^ +$ Assay during the clinical trial, $100 \%$ (94/94) of these controls gave correct results. The average Ct of the PCs tested was 27.1 (Min 23.7- Max 30.0), with 1.22 Standard Deviation (STDEV) and $4 . 5 0 \%$ CV; 28.3 (Min 25.1- Max 30.6), with 1.08 Standard Deviation (STDEV) and $3 . 8 3 \%$ ;

and 29.2 (Min 26.1- Max 31.7), with 1.05 Standard Deviation (STDEV) and 3.60 $\%$ CV for HPIV-1, HPIV-2, and HPIV-3 respectively. The pivotal clinical study data validated the pre-determined PC Ct acceptance range of 20.0 to 40.0.

Internal Control (IC): An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working. The IC is meant to assess global failure (reagent or process) and monitor for PCR inhibition. Because the IC is assessing global failure and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable range for the Internal Control (IC) was set to 15 - $5 0 ~ \mathrm { C t }$ in the Clinical Trial Protocol to accommodate for dilution variability (e.g. pipetting) by the user in spiking the IC, and also accommodate slight inhibition. The average $\mathrm { C t }$ of all eligible prospective ProParaflu $^ +$ Assay negative clinical specimens was 27.8 (Min 24.5- Max 36.8), with 1.41 Standard Deviation (STDEV) and $5 . 0 7 \%$ CV. The average Ct of all eligible ProParaflu $^ +$ Assay Negative Control was 27.4 (Min 24.8- Max 30.4), with 1.16 Standard Deviation (STDEV) and $4 . 2 4 \%$ CV. The pivotal clinical study data validated the predetermined IC Ct acceptance range of 15.0 to 50.0.

These RNA transcripts (PC and IC) were serially diluted in molecular grade water containing RNase Inhibitor at $1 \mathrm { U / \mu L }$ and made into the manufactured product included in the ProParaflu $^ +$ Assay kit. They were tested for final release by performing a prescribed dilution and using the ProParaflu $^ +$ Assay to ascertain the cycles at which these dilutions cross the threshold (cutoff). The specifications for final release are acceptable ranges of cycles at which the RNA control dilution can cross the threshold. Final release specifications were formally established using Test Method Characterization (TMC) then validated against the appropriate analytical performance characteristics. Final release specifications were developed to ensure First Pass Acceptance (FPA) rates $) f \geq 9 9 \%$ at lot release and customer rejection rate of $\leq 1 \%$ for a lot released at Acceptable Quality Level (AQL) (i.e. $9 5 \%$ probability of QC lot acceptance at lot release). For customer specifications, the maximum allowable customer lot rejection rate at the Rejectable Quality Level (RQL) Limit (i.e., $10 \%$ probability of QC acceptance at lot release) is $1 \%$ . A false valid risk of $\leq 5 \%$ was used to determine the validity specifications.

QC release specification ranges were set using Operator Characteristics (OC) curve analysis. Lot, technician, instrument, run, and replicate variance components were obtained during test method characterization. Observed Ct means were adjusted for stability effects and specifications determined to meet the validity and the above mentioned specification requirements.

The sponsor proposed to revise the customer Ct specification range for the PC and

IC based on the clinical trial control data. For both controls, a two sided prediction interval was determined using the observed clinical trial data, an alpha error of $1 \%$ and 10,000 customer runs. For the PC and IC, the customer specifications were modified per the prediction interval and adjusted to maintain internal process capabilities:

<table><tr><td rowspan=2 colspan=1>Control</td><td rowspan=1 colspan=2>Customer CtSpecification</td><td rowspan=2 colspan=1>Kit Control FinalReleaseSpecifications</td></tr><tr><td rowspan=1 colspan=1>Current</td><td rowspan=1 colspan=1>Proposed</td></tr><tr><td rowspan=1 colspan=1>PC (HPIV-1)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>20-39</td><td rowspan=1 colspan=1>23.55  32.45</td></tr><tr><td rowspan=1 colspan=1>PC (HPIV-2)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>21-38</td><td rowspan=1 colspan=1>25.37—32.83</td></tr><tr><td rowspan=1 colspan=1>PC (HPIV-3)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>21-40</td><td rowspan=1 colspan=1>25.41−34.43</td></tr><tr><td rowspan=1 colspan=1>Internal Control (IC)</td><td rowspan=1 colspan=1>15-50</td><td rowspan=1 colspan=1>22-34</td><td rowspan=1 colspan=1>24.74  31.06</td></tr></table>

The following controls are not provided in the ProParaflu $^ +$ Assay kit, but are required or recommended and described in the ProParaflu+ Assay Instructions for Use:

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the ProParaflu $^ +$ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination. Of a total of $9 5 \mathrm { N C }$ tested by the ProParaflu $^ +$ Assay during the clinical trial, $100 \%$ (95/95) of these controls gave correct results. The average Ct of the IC in tested NC was 27.4 (Min 24.8- Max 30.4), with 1.16 Standard Deviation (STDEV) and $4 . 2 4 \%$ CV.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a well characterized human parainfluenza 3 virus, was spiked with the IC and included with each nucleic acid isolation run as EC. The EC serves to monitor for lysis and contamination during nucleic acid isolation. Of a total of $1 3 3 \mathrm { E C }$ tested by the ProParaflu $^ +$ Assay during the clinical trial, $9 7 . 0 \%$ (129/133) of these controls gave correct results (3 HPIV-1 bleed-over, and 1 HPIV-2 contamination). The average Ct of the EC (HPIV-3) tested was 27.8 (Min 24.8 - Max 30.6), with 1.16 Standard Deviation (STDEV) and $4 . 1 8 \%$ CV. The sponsor is recommending an extraction control in each nucleic acid extraction run to the end users in the package insert.

# Freeze/Thaw and Stability

Reagents and Controls stability

An Accelerated Stability study concluded that the ProParaflu $^ +$ Mix and controls (closed and open tubes) can be stored at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ for up to 18 months.

A Freeze/Thaw Study demonstrated that the ProParaflu $^ +$ Mix can undergo up to 5 freeze-thaws. The Controls (Positive and Internal Control) can undergo up to 2 freeze-thaws. For the Controls, the ProParaflu $^ +$ Assay Package Insert specifies that controls should not undergo more than 1 freeze-thaw cycle. Although internal studies demonstrated that up to 2 freeze-thaws of controls would not adversely affect performance. To mitigate risk it is recommended that they not undergo more than 1 freeze-thaw.

Specimen stability

The ProParaflu $^ +$ Assay recommends that samples be stored refrigerated $( 2 ^ { \circ } \mathrm { C } -$ $8 ^ { \circ } \mathrm { C }$ ) for up to 72 hours prior to processing the samples. This recommendation is supported by a) The ProFlu $^ +$ Assay (detection of Influenza A, Influenza B and RSV nucleic acid) and the Pro $\mathrm { { h M P V + } }$ Assay (detection of human Metapneumovirus (hMPV) nucleic acid) recommended that samples be stored refrigerated $( 2 ^ { \mathrm { { o } } } \mathrm { { C } - 8 ^ { \mathrm { { o } } } \mathrm { { C } ) } }$ for up to 72 hours prior to processing the samples. Human parainfluenza viruses are members of the paramyxoviridae family along with RSV and hMPV. It is reasonable to assume that the same sample stability conditions that are recommended and used for culture as well as those used for similar devices (ProFlu $^ +$ and Pr ${ } _ { 0 } \mathrm { h M P V ^ { + } } ,$ ) are applicable to human Parainfluenza viruses and the ProParaflu $^ +$ Assay; b) A 72 hour timeframe is widely recommended and used (CLSI Guidance M41 Viral Culture) for viral culture; c) The viral transport medium that nasopharyngeal swab samples are collected in also serves to further “preserve” and “stabilize” viruses, specifically influenza, RSV and parainfluenza viruses to maintain their viability for culture and NAAT; d) The CLSI Guidance: Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (MM13) does not specify requirements for nasopharyngeal (NP) swabs, but does recommend that bronchioalveolar lavage samples be refrigerated for up to 72 hours or frozen (at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ ) for longer periods prior to processing for molecular methods.

Furthermore, the average time to extraction was 44 hours (Min. 17 hours to Max. 76 hours during the clinical trial. Site 1 has the longest average time to extraction (48.2 hours) while Site 3 has the shortest time (35.9 hours). There was no observed impact of times to extraction on HPIV-2 and HPIV-3 Ct values generated from testing prospective clinical specimens.

The ProParaflu $^ +$ Assay also recommends that extracted nucleic acids that are not run immediately after extraction may be stored at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ until RT-PCR testing, and they should be tested after no more than one freeze-thaw cycle. This recommendation is supported by a) Both CLSI guidance documents MM13 (Collection, Transport, Preparation and Storage of Specimens for Molecular Methods) and MM17 (Verification and Validation of Multiplex Nucleic Acid Assays) recommend storage of isolated RNA at $- 8 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ and limiting the number of freeze-thaw cycles before testing; b) Human parainfluenza viruses are members of the paramyxoviridae family along with RSV and hMPV. It is reasonable to assume that the same nucleic acids storage conditions that are recommended and used for similar devices (ProFlu+ and Pro hMPV+) are applicable to human

Parainfluenza viruses and the ProParaflu+ Assay. In the Pro hMPV+ assay $5 1 0 \mathrm { k }$ submission (k082688), the sponsor conducted the following two studies to support the nucleic acids storage condition recommendations: 1) A study that compared the extracted nucleic acid isolated from the Extraction Control (EC) during the clinical trial being tested “fresh” or after undergoing one freeze-thaw cycle was conducted. Sites 1 and 2 ran both “fresh” and “frozen” EC nucleic acids and t-test analysis demonstrated that there is no significant difference between the two conditions $( \mathtt { p } > 0 . 0 5$ for each site); 2) A subset of hMPV positive samples from the clinical trial were re-run from the purified nucleic acid at the same time the reference assays were being set up. All of the samples in the subset were initially run as “fresh” nucleic acids and the repeat testing was indicative of 1 freeze-thaw cycle. Although a paired $t \cdot$ -test analysis indicates that the mean Ct values of hMPV-positive sample nucleic acids that have undergone one freeze-thaw cycle are statistically different $( \mathrm { p } { < } 0 . 0 5 )$ than the mean Ct values of “fresh” hMPVpositive sample nucleic acids, the mean Ct difference is clinically insignificant (mean $= 0 . 2 2$ ; range $= 0 - 1 . 5 \mathrm { C t }$ ); c) Stability data for the ProParaflu+ Positive RNA Control also supports the storage of RNA at ultralow temperature. The table below compares the extracted nucleic acid isolated from the Extraction Control (EC) during the Clinical and Reproducibility Studies. For the Reproducibility Study the EC nucleic acid was run “fresh” for each run at all sites $\scriptstyle ( \mathrm { n } = 6 0 )$ . During the Clinical Study, Sites 1 and 2 ran all nucleic acids frozen $[ \mathtt { n } = 3 6$ and 39 respectively). There were $2 3 \mathrm { E C }$ nucleic acid samples run “fresh” at Site 3 (frozen; $\scriptstyle \mathrm { n = 9 }$ ) and 3 at Site 4 (frozen; $\mathrm { n } { = } 2 3$ ). Comparing the two data sets by student t-test indicates a statistically significant difference in mean Ct $( \mathtt { p } < 0 . 0 3 )$ . However, as shown in the table below the mean $\mathrm { C t }$ difference is only 0.53, representing a theoretical 1.4-fold concentration difference, and the range of the fresh Ct’s lies within the range of frozen Ct’s. So while the data are statistically different, the difference is thought to be clinically insignificant.

# Comparison of the Parainfluenza Extraction Control (HPIV-3) RNA run “fresh” or “frozen”

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>Frozen</td></tr><tr><td rowspan=1 colspan=1>AVE</td><td rowspan=1 colspan=1>28.12</td><td rowspan=1 colspan=1>27.59</td></tr><tr><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.97</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>3.52</td></tr><tr><td rowspan=1 colspan=1>COUNT</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>107</td></tr><tr><td rowspan=1 colspan=1>MIN</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>24.8</td></tr><tr><td rowspan=1 colspan=1>MAX</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>30.6</td></tr></table>

# d. Analytical Sensitivity (Detection limit):

# LoD estimation studies using the Roche MagNA Pure LC

The analytical sensitivity (limit of detection or LoD) of the ProParaflu $^ +$ Assay was estimated using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of viral strains of human Parainfluenza virus (HPIV) types 1, 2 and 3 (HPIV-1 Strain C35, ATCC VR-94; HPIV-2 Strain Greer, ATCC VR-92; and HPIV-3 Strain C243, ATCC VR-93) serially diluted in negative nasopharyngeal (NP) pools prepared from leftover NP swab clinical samples. Each viral strain was previously cultured and titered to determine its stock concentration in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Each viral stock was spiked into negative NP swab pools at concentrations of 1 log above, 1 log below, and at the estimated LoD for that virus (based on preliminary studies). Two dilution series were prepared for each viral stock by two operators for a total of 4 dilution series per virus. An Internal RNA Control (IC) was spiked into all dilution series samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Each dilution series was extracted on the Roche MagNA Pure LC and tested in quintuplicate reactions using two lots of ProParaflu $^ +$ reagents on multiple (4) Cepheid SmartCycler II instruments. This resulted in a total of 20 data points (20 RT-PCR replicates per concentration, not 20 individual nucleic acid extractions per concentration) per viral strain dilution. A negative control, which consisted of negative NP swab matrix spiked with IC was included with each set of dilution series. Nucleic acid isolation of the negative control was performed along with the viral strain dilution series. The negative control served to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The Parainfluenza Positive RNA Control, non-infectious in vitro transcribed RNA of specific HPIV-1, HPIV-2 and HPIV-3 viral sequences, was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

Analytical sensitivity was estimated for each virus strain. The LoD was estimated as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 19 out of 20 replicates were determined to be positive). The data are presented in the following table:

Analytical Sensitivity Estimation Results   

<table><tr><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageC</td><td rowspan=1 colspan=1>StandardDeviationCT</td><td rowspan=1 colspan=1>MinC</td><td rowspan=1 colspan=1>Max CT</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=3 colspan=1>HPIV-1</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>10²2 TCID50/mL</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>101 TCID50/mL</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=3 colspan=1>HPIV-2</td><td rowspan=1 colspan=1>102 TCID50/mL</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>101 TCID50/mL</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=3 colspan=1>HPIV-3</td><td rowspan=1 colspan=1>102 TCID50/mL</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>101 TCID50/mL</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>2.86</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0%</td></tr></table>

The sponsor estimated LoDs of $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and $1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for the HPIV-1, HPIV-2 and HPIV-3 strains tested, respectively.

# LoD confirmation studies using the Roche MagNA Pure LC

The estimated analytical sensitivity (limit of detection or LoD) of the ProParaflu+ Assay was confirmed/determined using quantified $\mathrm { \ T C I D _ { 5 0 } / m L }$ ) cultures of viral strains of human Parainfluenza virus (HPIV) types 1, 2 and 3 (HPIV-1 Strain C35, ATCC VR-94; HPIV-2 Strain Greer, ATCC VR-92; and HPIV-3 Strain C243, ATCC VR-93) diluted in negative nasopharyngeal (NP) pools prepared from leftover NP swab clinical samples. Each viral strain was previously cultured and titered to determine its stock concentration in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Each viral stock was spiked into negative NP swab pools at concentrations of the estimated LoD for that virus (based on LoD estimation studies). The spiked pools were then divided into 20 individual aliquots. An Internal RNA Control (IC) was spiked into all samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Each aliquot was extracted on the Roche MagNA Pure LC and tested in single RT-PCR reactions using a single lot of ProParaflu $^ +$ reagents on multiple Cepheid SmartCycler II instruments. This resulted in a total of 20 data points (20 individual nucleic acid extractions) per viral strain dilution. A negative control, which consisted of negative NP swab matrix spiked with IC was included with each set of dilution series. Nucleic acid isolation of the negative control was performed along with the viral strain dilution series. The negative control served to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The Parainfluenza Positive RNA Control, non-infectious in vitro transcribed RNA of specific HPIV-1, HPIV-2 and HPIV-3 viral sequences, was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

Analytical sensitivity was determined/confirmed for each virus strain. The LoD was confirmed as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 19 out of 20 replicates were determined to be positive). For HPIV-3 a $2 9 5 \%$ detection rate was not achieved at $1 \mathrm { ~ x ~ } 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , thus a $\mathrm { 5 \times 1 0 ^ { 1 } T C I D _ { 5 0 } / m L }$ dilution of HPIV-3 was generated and a total of 20 samples at that concentration were tested. The data are presented in the following table:

Analytical Sensitivity Confirmation Results   

<table><tr><td rowspan=1 colspan=1>ViralStrain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageC</td><td rowspan=1 colspan=1>StandardDeviationCT</td><td rowspan=1 colspan=1>Min CT</td><td rowspan=1 colspan=1>Max CT</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>% Detected</td></tr><tr><td rowspan=1 colspan=1>HPIV-1</td><td rowspan=1 colspan=1>102 TCID50/mL</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HPIV-2</td><td rowspan=1 colspan=1>102 TCID50/mL</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>HPIV-3</td><td rowspan=1 colspan=1>101 TCID50/mL</td><td rowspan=1 colspan=1>33.5*</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>55%</td></tr><tr><td rowspan=1 colspan=1>5 x101 TCID50/mL</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

\* $_ { \mathrm { N } } { = } 1 1$ . Average based on number of replicates detected.

The sponsor confirmed LoDs of $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and $5 \mathrm { ~ x ~ } 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for the HPIV-1, HPIV-2 and HPIV-3 strains tested, respectively.

# Extraction Equivalency Study

To evaluate the equivalency of nucleic acid extraction using the Roche MagNA Pure LC (used throughout ProParaflu $^ +$ Assay development and Analytical Studies, Clinical Trials and Reproducibility) and the bioMérieux NucliSens easyMAG automated extraction instruments (used in Clinical Trials and Reproducibility) for use with the ProParaflu+ Assay, an extraction equivalency study was carried out to determine and compare the Limit of Detection (LoD) for human Parainfluenza virus type 3 (HPIV-3) on each automated extractor.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles.

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads.

For the Extraction Equivalency LoD Study, HPIV-3 (Strain C 243,ATCC VR-93; $1 \mathrm { x } 1 0 ^ { 7 . 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l } )$ was spiked (along with an IC) into individual aliquots of negative nasopharyngeal swab (NP) matrix pools at concentrations of 1 log above, at, and 1 log below the preliminary limit of detection (LoD) (as determined by a preliminary Analytical Sensitivity Study). Each viral strain dilution was extracted in replicates of 10 on each automated extractor. An Extraction Control and a Negative Control (NP spiked with Internal Control - NPIC) were included in each run on each extractor.

The 10 extractions of each of the three concentrations from both extraction instruments (total of 30 samples) for a specific virus were tested using the ProParaflu+ Assay (a single ProParaflu+ mastermix). Included with each run was the Parainfluenza RNA Positive Control and the relevant NPICs from each automated extraction run for the specific samples being tested. A total of 10 data points were generated at each virus dilution per extraction instrument. Results were analyzed as specified in the ProParaflu+ Instructions for Use. This study was limited to one operator, one extraction instrument (one easyMAG and one MagNA Pure LC), and one lot of instrument specific extraction reagents per

instrument, one lot of ProParaflu $^ +$ reagents.

LoD for each extractor was determined by $2 9 5 \%$ positivity at a specific concentration and are presented in the following table:

Extraction Equivalence LoD Study   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=2 colspan=1>Extractorused</td><td rowspan=1 colspan=2>HPIV-3 CT</td><td rowspan=1 colspan=2>IC CT</td><td rowspan=2 colspan=1>Comments</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>Mean+</td><td rowspan=1 colspan=1>StdDev+</td></tr><tr><td rowspan=2 colspan=1>HPIV-3 10{2TCID50/mL</td><td rowspan=1 colspan=1>EasyMAG</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>10/10 positive samples</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>10/10 positive samples</td></tr><tr><td rowspan=2 colspan=1>HPIV-3 101TCID50/mL</td><td rowspan=1 colspan=1>EasyMAG</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>10/10 positive samples</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>7/11*positive samples</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 100</td><td rowspan=1 colspan=1>EasyMAG</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1/10 positive samples</td></tr><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0/10 positive samples</td></tr></table>

$^ { * } { } _ { 1 { 1 } }$ total samples due to one sample being retested in duplicate due to IC failure. Initial results were replaced with values from retesting. $^ + \mathrm { I C } \mathrm { C } _ { \mathrm { T } }$ and SD calculated from all replicates (including HPIV-3 negatives) for each concentration

The mean $\mathrm { C t }$ values for the easyMAG were slightly lower than the MagNA Pure at each HPIV-3 concentration tested. The average $\mathrm { C t }$ values between the extractors differed by 1.1 $( 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ and 0.3 $( 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) Ct’s which represent less than a half-log change in concentration. These differences are statistically significant at $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ $\scriptstyle ( { \mathrm { p } } { = } 0 . 0 0 4 )$ but not statistically significant at $1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ $\scriptstyle ( { \mathrm { p } } { = } 0 . 4 9 4 )$ . The values at both concentrations are not clinically significant given the theoretical change in concentration is less than a log.

This limited study demonstrated that the easyMAG may have some performance advantage over the MagNA Pure in regards to Analytical Sensitivity of HPIV-3. Moreover, it is reasonable to assume any performance claims made using the MagNA Pure with the ProParaflu+ Assay should be achievable using the easyMAG. Clinical trial and reproducibility studies also support comparable performance of both automated extraction systems.

# e. Analytical Reactivity:

Due to minimal genetic or antigenic variability over the years and the fact that only single strains of HPIV 1-3 are commercially available, a Ubiquity/Reactivity study was not performed for the ProParaflu $^ +$ Assay. It is acceptable.

# f. Analytical specificity:

The analytical specificity of the ProParaflu $^ +$ Assay was determined with a panel of 52 viral, bacterial and yeast (27 viruses, 24 bacteria, and 1 yeast) strains of common respiratory pathogens or flora commonly present in the nasopharynx. All the viral strains were obtained from ATCC or University of Iowa and cultured and titered by Tricore Reference Laboratories (Albuquerque, NM) except for human Metapneumovirus (hMPV) which was cultured and titered by ZeptoMetrics (Buffalo, NY). The majority of cultured and titered bacterial and yeast strains were obtained from MicroBioLogics, Inc. (St. Cloud, MN). A few others were obtained from ATCC and cultured and titered by the Clinical Microbiology Laboratory at Resurrection Medical Center (Chicago, IL) or Waukesha Memorial Hospital (Waukesha, WI). None of the facilities listed above were able to provide fresh cultured and titered stocks of $C .$ . pneumoniae or $C .$ . trachomatis. Additional laboratories (commercial and private) were contacted, but none were able to provide titered cultures at a reasonable cost and timeframe. Thus, ATCC frozen cultures were used and ATCC supplied titers were used for $C .$ . pneumoniae and $C .$ . trachomatis.

Aliquots of a negative nasopharygeal (NP) swab pool matrix were spiked with the analytical specificity organisms at concentrations of $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l } \dot { - } 1 0 ^ { 6 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ for viruses and $1 0 ^ { 6 } \mathrm { C F U / m l - 7 . 4 \times 1 0 ^ { 7 } C F U / m l }$ for the bacterial and yeast strains. The ProParaflu $^ +$ Internal Control (IC) was spiked into all samples prior to nucleic acid isolation. A Negative Control, which consisted of the M4 viral transport media spiked with the IC, was included and nucleic acid isolation of the negative control was performed along with Analytical Specificity Panel samples. The Parainfluenza Positive Control and was included with each detection run to test for global failure. The Analytical Specificity Panel samples and the Negative Control were extracted on the Roche MagNA Pure LC and tested in triplicate on a Cepheid SmartCycler II using one lot of ProParaflu+ reagents.

The ProParaflu $^ +$ Assay did not cross-react with any of the Analytical Specificity Panel samples tested. Detailed Analytical Specificity results are presented in the following table:

# Analytical Specificity Results

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentrationtested</td><td colspan="1" rowspan="1">HPIV-1Detection(Ct)</td><td colspan="1" rowspan="1">HPIV-3Detection(Ct)</td><td colspan="1" rowspan="1">HPIV-2Detection(Ct)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1 High</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">23.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2 High</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">22.3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3 High</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">20.3</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1 Low</td><td colspan="1" rowspan="1">5x102 TCID50/ml</td><td colspan="1" rowspan="1">27.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2 Low</td><td colspan="1" rowspan="1">5x102 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">26.4</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3 Low</td><td colspan="1" rowspan="1">5x101 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">29.9</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid 71</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B5/10/2006</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">106 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HMPV A2</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">105 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 39</td><td colspan="1" rowspan="1">103 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B. bronchiseptica</td><td colspan="1" rowspan="1">5x107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae*</td><td colspan="1" rowspan="1">104 TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis*</td><td colspan="1" rowspan="1">104TCID50/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="1" rowspan="1">107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">7.4x 107 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.3x107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">3x107 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">2x106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">3x 106 CFU/ml</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

\* A few laboratories (commercial and private) were contacted, but none were able to provide titered cultures at a reasonable cost and timeframe. Thus, ATCC frozen cultures were used and ATCC supplied titers were used for $C .$ . pneumoniae and $C .$ . trachomatis.

# g. Assay cut-off:

The “cutoff value” represents the fluorescent intensity signal level (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction.

If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

Cutoff Determination and Confirmation studies were conducted with two operators and two reagent lots.

Cutoff values (RFUs) were determined upon completion of the Cutoff Determination Study which included a training set of 58 negative and simulated positive samples prepared from individual NP swab clinical samples. Specifically, individual samples $( \mathrm { n } = 6 )$ ) were spiked with each type of HPIV (1, 2 and 3) at 3 logs above Limit of Detection (LoD) representing high positive NP swab samples. Individual samples $( \mathtt { n } = 1 8 )$ ) were spiked with each type of HPIV (1, 2 and 3) at 2 times LoD representing low positive NP swab samples. 34 samples were left unspiked. The high and low positive samples, as well as 30 negative samples were spiked with the Internal Control. Four (4) negative samples were not spiked with the Internal Control. Each series was extracted using the Roche MagNA Pure LC. A negative viral transport media control spiked with IC was included with each extraction run. The extracted nucleic acids and the Parainfluenza Positive RNA Control and the IC diluted in water were run on the Cepheid SmartCycler II.

The cutoff values were then verified in the Cutoff Confirmation Study against a set of clinical samples and controls (32 HPIV-1 positive, 32 HPIV-2 positive, 48 HPIV-3 positive, 36 HPIV negative NP swab samples and controls). The determined cutoff was confirmed if $2 9 0 \%$ sensitivity and specificity was attained for each channel with the above mentioned set of clinical samples and controls. In addition, 18 Parainfluenza RNA Controls, $1 6 \ : \mathrm { I C s }$ in TM and 4 Negative Controls were also tested in the Cutoff Confirmation Study.

In the FAM, TET, TxR and Cy5 channels, the final RFUs for all samples of the positives and negatives demonstrated minimal overlap. For a given detection, a range of possible cutoffs was generated by determining robust lower and upper boundaries of the threshold, based on final RFU that would minimize false positives and false negatives. A Receiver Operator Characteristics (ROC) analysis was performed for HPIV-1, HPIV-2, HPIV-3 and Internal Control threshold settings. In addition, population distribution statistics were analyzed to aid in selecting a cutoff threshold.

Acceptable ranges for the potential cutoff were generated in the following manners: 1) ROC curves for HPIV-1, HPIV-2, HPIV-3 and Internal Control cutoffs were generated. The threshold range that yielded the maximum accuracy was identified. 2) The distributions of the negatives and positives were first assessed for normality. a) If normal, the averages and standard deviations of the final RFU were calculated. For the lower limit of the acceptable range of cutoff, negative samples were analyzed. The lower threshold was defined as the average final RFU plus three standard deviations of the negatives, that is, the point at which greater than $9 9 \%$ of true negatives are expected to fall below assuming normality. Similarly, the upper limit of the acceptable range of cutoff threshold was determined using positives and was defined as the average final RFU minus three standard deviations. This is the point at which at least $9 9 \%$ of true positives are expected to be greater than, assuming normality. b) If the distribution was not normal, for the negatives the lower limit of the cutoff threshold range was defined minimally as the $9 7 . 5 \%$ percentile. For the positives the upper limit of the cutoff threshold range was defined maximally as the $2 . 5 \%$ percentile. 3) After settings were finalized the clinical samples were evaluated for sensitivity and specificity.

The cutoffs used in the Cutoff Determination and Confirmation Studies are presented in the following table:

Cutoffs used in Cutoff Determination and Confirmation Studies   

<table><tr><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>EndPt Threshold (RFU)</td></tr><tr><td rowspan=1 colspan=1>Preliminary Cutoff used inCutoff DeterminationStudy*</td><td rowspan=1 colspan=1>Cutoff used inCutoff ConfirmationStudy</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>HPIV-1</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>HPIV-3</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>TxR</td><td rowspan=1 colspan=1>HPIV-2</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>40</td></tr></table>

Identified during product development

Analysis settings and threshold settings were determined for FAM (60), TET (40), Texas Red (50), and Cy5 (40). These settings produced $2 9 0 \%$ sensitivity and specificity in the original set of samples used during Cutoff Determination and met all acceptance criteria for the subsequent Cutoff Confirmation study, achieving $9 6 . 9 \substack { - 1 0 0 \% }$ sensitivity and $100 \%$ specificity. Therefore, cutoff values were confirmed as acceptable for use during the clinical trials and all subsequent analytical studies

# h. General Assay Analysis Settings:

<table><tr><td colspan="5" rowspan="1">Cepheid SmartCycler II</td></tr><tr><td colspan="5" rowspan="1">1.7b, Dx Software</td></tr><tr><td colspan="1" rowspan="1">Channel:</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Dye Name:</td><td colspan="1" rowspan="1">FAM</td><td colspan="1" rowspan="1">TET</td><td colspan="1" rowspan="1">TxR</td><td colspan="1" rowspan="1">Cy5</td></tr><tr><td colspan="1" rowspan="1">Target:</td><td colspan="1" rowspan="1">HPIV-1</td><td colspan="1" rowspan="1">HPIV-3</td><td colspan="1" rowspan="1">HPIV-2</td><td colspan="1" rowspan="1">Internal Control</td></tr><tr><td colspan="1" rowspan="1">Usage:</td><td colspan="1" rowspan="1">Assay**</td><td colspan="1" rowspan="1">Assay**</td><td colspan="1" rowspan="1">Assay**</td><td colspan="1" rowspan="1">Internal Control</td></tr><tr><td colspan="1" rowspan="1">Curve Analysis:</td><td colspan="1" rowspan="1">PrimaryCurve</td><td colspan="1" rowspan="1">PrimaryCurve</td><td colspan="1" rowspan="1">PrimaryCurve</td><td colspan="1" rowspan="1">Primary Curve</td></tr><tr><td colspan="1" rowspan="1">Thresh Setting:</td><td colspan="1" rowspan="1">Manual</td><td colspan="1" rowspan="1">Manual</td><td colspan="1" rowspan="1">Manual</td><td colspan="1" rowspan="1">Manual</td></tr><tr><td colspan="1" rowspan="1">Manual Thresh:</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">Auto Thresh:</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Auto MinCycle:</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Auto MaxCycle:</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">Valid MinCycle:</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">Valid MaxCycle:</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">BackgrdSubtract:</td><td colspan="1" rowspan="1">On</td><td colspan="1" rowspan="1">On</td><td colspan="1" rowspan="1">On</td><td colspan="1" rowspan="1">On</td></tr><tr><td colspan="1" rowspan="1">Bkgnd MinCycle</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Bkgnd MaxCycle</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">Boxcar Avg</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">EndPt Thresh</td><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">40</td></tr><tr><td colspan="1" rowspan="1">NC IC %</td><td colspan="1" rowspan="1">10*</td><td colspan="1" rowspan="1">10*</td><td colspan="1" rowspan="1">10*</td><td colspan="1" rowspan="1">10*</td></tr><tr><td colspan="1" rowspan="1">IC Delta</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td></tr></table>

\*\* Dx 3.0a = Target $\mathbf { \mathrm { ^ * D x } } 3 . 0 \mathbf { a } = \mathbf { N A }$

# Interfering Substances:

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors affected the performance of the ProParaflu $^ +$ Assay. Blood, mucin, or medications (prescription and over-the-counter) for relief of congestion, sore throat, allergy and asthma symptoms were spiked into simulated human Parainfluenza (HPIV) positive NP samples. A single strain of HPIV-3 (Strain C 243 (ATCC VR-93); $1 \mathrm { x } 1 0 ^ { 7 . 2 } \mathrm { T C I D } _ { 5 0 } \mathrm { \dot { / } m l ) }$ was used and spiked into NP pools at 2X Limit of Detection (LoD) and 1 log above LoD. (Note: The LoD had only been estimated in an Analytical Sensitivity Estimation Study. The LoD for HPIV-3 was later established and confirmed in the Analytical Sensitivity Confirmation Study to be $5 \mathrm { x } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l } .$ .) High clinically relevant amounts of the potential inhibiting substances were added to spiked samples. An Internal Control (IC) was added to each sample. Nucleic acid from the samples was extracted with the Roche MagNA Pure LC. ProParaflu+ Assay was performed in triplicate reactions for each sample on the Cepheid SmartCycler II. Any substance in which detection of HPIV-3 at 10X LoD was not observed was considered an inhibitor of PCR. A Negative Control, which consisted of M4 viral transport media spiked with an IC, was included with Interfering Substances Panel samples. The Parainfluenza Positive Control and was included with each detection run to test for global errors.

The following table shows the interfering substances used for this study. As described in the justification column, the concentrations spiked directly into samples ranged from approximately $2 \%$ to $1 5 \%$ of recommended dose for some substances, while others were tested at concentrations reported in scientific literature or in references from other IVD package inserts. The substances consisted of nasal sprays (liquid and powder), ingestible pills and lozenges, injectables, and endogenous substances (blood and mucin).

Interfering Substances Test Concentrations   

<table><tr><td rowspan=1 colspan=1>Substance Name</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Justification</td></tr><tr><td rowspan=1 colspan=1>Mucin (BovineSubmaxillarygland, typeI-S)</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>60μg/ml</td><td rowspan=1 colspan=1>1000x maximum levelpresent in serum*</td></tr><tr><td rowspan=1 colspan=1>Blood (human), heparinanticoagulent</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2% (volume/volume)</td><td rowspan=1 colspan=1>Other Respiratory IVD&#x27;sPackage Insert*.</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine®</td><td rowspan=1 colspan=1>Phenylephrine HCl</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μl)*</td></tr><tr><td rowspan=1 colspan=1>Walgreens OriginalAnefrin Nasal Spray</td><td rowspan=1 colspan=1>OxymetazolineHydrochloride</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μl)*</td></tr><tr><td rowspan=1 colspan=1>Zicam Homepathic Non-Drowsy Allergy ReliefNo Drip Liquid Nasalgel</td><td rowspan=1 colspan=1>Luffa Operculata,Galphimia Glauca,HistaminumHydrochloricum,</td><td rowspan=1 colspan=1>5% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(15μl)*</td></tr><tr><td rowspan=1 colspan=1>Walgreens Saline NasalSpray</td><td rowspan=1 colspan=1>Sodium chloride withpreservatives</td><td rowspan=1 colspan=1>15% (volume/volume)of dose</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μl)*</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic® SoreThroat Lozenges, Cherry</td><td rowspan=1 colspan=1>Oralanesthetic/analgesic</td><td rowspan=1 colspan=1>0.63mg/ml; 1/20 drop,crushed; activeingredients: 1.0mg/mlbenzocaine, 1.7mg/mlmenthol</td><td rowspan=1 colspan=1>5% of total dose*</td></tr><tr><td rowspan=1 colspan=1>Relenza®</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>3.3mg/ml;</td><td rowspan=1 colspan=1>10% of total spraydose*</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4.0μg/ml</td><td rowspan=1 colspan=1>10% of totalrecommended dose*</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>6.6mg/ml</td><td rowspan=1 colspan=1>10% of totalrecommended dose inMupirocin ointment*</td></tr><tr><td rowspan=1 colspan=1>Rebitol</td><td rowspan=1 colspan=1>Ribavirin</td><td rowspan=1 colspan=1>20mg/ml</td><td rowspan=1 colspan=1>10% of totalrecommended dose*</td></tr><tr><td rowspan=1 colspan=1>TamiFlu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>25mg/ml</td><td rowspan=1 colspan=1>10% of totalrecommended dose*</td></tr><tr><td rowspan=1 colspan=1>Beconase AQ®</td><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>5% volume/volume =14μg/ml</td><td rowspan=1 colspan=1>10% of totalrecommended dose*</td></tr></table>

$^ { * } =$ Concentration tested is $10 \%$ of suggested dose into a $0 . 3 \mathrm { m l }$ sample, or otherwise stated in table above.

Some inhibition was observed at the 2X LoD HPIV-3 concentration $\left( 2 \mathrm { x } 1 0 ^ { 1 } \right.$

$\mathrm { T C I D } _ { 5 0 } / \mathrm { m l } )$ ), although the LoD for HPIV-3 was later established and confirmed in the Analytical Sensitivity Confirmation Study to be $5 \mathrm { x } 1 0 ^ { 1 }$ T $\mathrm { C I D } _ { 5 0 } / \mathrm { m l }$ . All three replicates were not detected for the sample containing the Chloraseptic® sore throat lozenge, two of three replicates were not detected for the sample containing Relenza, and 1 of three replicates was not detected for sample containing Rebitol. None of the potential interferants inhibited detection of HPIV-3 at 1 log above LoD in all replicates. As the low concentration of HPIV-3 used may have a lower than $9 5 \%$ detection rate, it cannot be ruled out that HPIV-3 was not detected due to testing at just below the Assay LoD rather than an effect of the substances present. (No inhibition was observed at the $1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ HPIV-3 concentration for samples containing Chloraseptic $^ \mathrm { \textregistered }$ sore throat lozenge, Relenza and Rebitol.)

The Interfering Substances Study results demonstrated that there are no interfering substances that significantly inhibit the detection of HPIV-3 nucleic acid using the ProParaflu $^ +$ Assay (per the pre-determined study acceptance criteria). Interfering Substances Study results are presented in the following table:

<table><tr><td rowspan=3 colspan=1>InterferingSubstance</td><td rowspan=1 colspan=4>HPIV-3 Ct value</td><td rowspan=1 colspan=4>HPIV-3 Ct value</td></tr><tr><td rowspan=1 colspan=4>10X LoD102 TCID50/ml</td><td rowspan=1 colspan=4>2X LoD2x101 TCID50/ml</td></tr><tr><td rowspan=1 colspan=1>Ave.HPIV-3Ct</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>Ave. ICCt</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>Ave.HPIV-3Ct</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>Ave. ICCt</td><td rowspan=1 colspan=1>Std Dev</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>0.12</td></tr><tr><td rowspan=1 colspan=1>Anefrin NasalSpray</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>0.23</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Beconase AQ</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic ®Sore ThroatLozenges</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>34.5*</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>0.20</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>Rebitol</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>34.4**</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1>TAMIFLU(oseltamivir)</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>0.15</td></tr></table>

$^ { * 1 }$ of 3 replicates detected $^ { * * 2 }$ of 3 replicates detected

$j .$ . Internal Control Interference: Competitive inhibition of the ProParaflu $^ +$ Assay due to the presence of the IC was assessed. Simulated samples were tested with and without IC to determine if the presence of the IC inhibited the reaction. Two sets of IC Interference Panel members were prepared by spiking serial dilutions of cultured and titered $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ strains of human Parainfluenza virus (HPIV) types 1, 2 and 3 into individual aliquots of negative nasopharyngeal (NP) swab pools at concentrations of 1 log below and at the preliminary Limit of Detection (LoD) (as estimated in the Analytical Sensitivity Estimation Study). The IC was added to one set of IC Interference Panel samples prior to nucleic acid isolation; the second set did not contain the IC. Two negative controls were generated which consisted of negative NP swab matrix with or without the Internal Control. Nucleic acid isolation of the negative controls was performed along with the IC Interference Panel samples. The negative control served to monitor for contamination during the testing procedure or in the case of negative control spiked with IC, also served to monitor for the presence of inhibitors in the sample matrix. The Parainfluenza Positive RNA Control was included with each RT-PCR run to test for global errors (absence of reagent, instrument failure, etc.). The Positive Control does not require nucleic acid isolation and was diluted just prior to set up of RT-PCR reactions.

Each virus dilution was split into two aliquots and extracted on the Roche MagNA Pure LC. The purified nucleic acids from each duplicate extraction were combined ( $1 0 0 \mu \mathrm { L }$ total volume) and tested with the ProParaflu $^ +$ Assay in 10 replicate RT-PCR reactions using 1 lot of ProParaflu $^ +$ reagents on the Cepheid Smartcycler II. This resulted in a total of 10 data points for each viral strain at each concentration spiked with IC and 10 data points for each viral strain at each concentration without the IC. Note: Since greater than $9 5 \%$ of the replicates at 1 log below LoD were positive for HPIV-2, an additional dilution series was generated using concentrations of 1 log below and 2 logs below the estimated LoD for HPIV-2. Nucleic acid extraction and RT-PCR testing were repeated on these new dilutions.

Competitive inhibition of the IC was assessed by comparing the percent of samples detected at concentrations of 1 log below and at the LoD of each viral strain when spiked with or without the IC. Competitive inhibition was to be considered if the presence of the IC decreased the sensitivity (LoD) of the ProParaflu $^ +$ Assay.

IC Interference Study results are presented in the following table:

<table><tr><td colspan="1" rowspan="2">ViralStrain</td><td colspan="1" rowspan="2">Concentration</td><td colspan="3" rowspan="1">Internal Control</td><td colspan="3" rowspan="1">No Internal Control</td></tr><tr><td colspan="1" rowspan="1">AverageCT Value</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%ReplicatesDetected</td><td colspan="1" rowspan="1">AverageCT Value</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%ReplicatesDetected</td></tr><tr><td colspan="1" rowspan="1">HPIV-1</td><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">30.7</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">30.3</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">HPIV-1</td><td colspan="1" rowspan="1">1 log belowLoD</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="1">HPIV-2</td><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">28.7</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">29.3</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">HPIV-2*</td><td colspan="1" rowspan="1">1 log belowLoD</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0.18</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">HPIV-2*</td><td colspan="1" rowspan="1">2 logs belowLoD</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">1.66</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">1.86</td><td colspan="1" rowspan="1">70</td></tr><tr><td colspan="1" rowspan="1">HPIV-3</td><td colspan="1" rowspan="1">At LoD</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">1.86</td><td colspan="1" rowspan="1">90</td></tr><tr><td colspan="1" rowspan="1">HPIV-3</td><td colspan="1" rowspan="1">1 log belowLoD</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10</td></tr></table>

$* =$ Repeat testing summary results

Note: The differences in detection at 1 or 2 logs below LoD analyte concentrations are not necessarily attributable to the presence or absence of the IC as variable detection rates of HPIV are expected below the assay’s LoD and confirmed by the Analytical Sensitivity Study.

No competitive inhibition at the assay LoD was observed for detection of HPIV-1, -2 or -3 in the presence IC using the ProParaflu $+$ Assay.

# k. Competitive Inhibition Study

Effects of competitive inhibition on the ProParaflu $^ +$ Assay when two of the Assay’s target organisms are present in a single sample were evaluated in the Competitive Inhibition Study. The competition between HPIV types was examined at varying concentrations of HPIV in combinations of high/high and high/low concentrations. Samples containing only one HPIV type were run at both concentrations to act as controls. Each HPIV type was spiked into negative NP pools at final concentrations of two (HPIV-1 and HPIV-2) or three (HPIV-3) logs above LoD and at 2X LoD (HPIV-1 and HPIV-2) and 10X LoD (HPIV-3) (preliminary LoD as determined by a preliminary Analytical Sensitivity Study). An Internal RNA Control (IC) was spiked into all panel samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Each sample in the Dual Positive Panel was extracted on the Roche MagNA Pure LC and tested in triplicate RT-PCR reactions with one lot of ProParaflu+ reagents. A negative control, which consists of negative NP swab matrix spiked with IC, was included in the Panel testing. Nucleic acid isolation of the negative control was performed along with panel samples. The negative control serves to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The Parainfluenza Positive RNA Control was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

Competitive Inhibition Study results are presented in the following table:

<table><tr><td rowspan=1 colspan=2>Panel Samples</td><td rowspan=1 colspan=1>AverageHPIV-1 Ct</td><td rowspan=1 colspan=1>AverageHPIV-2 Ct</td><td rowspan=1 colspan=1>AverageHPIV-3 Ct</td><td rowspan=1 colspan=1>AverageIC Ct</td></tr><tr><td rowspan=1 colspan=2>HPIV-1 2X LoD</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30.9</td></tr><tr><td rowspan=1 colspan=2>HPIV-1 100X LoD</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27.9</td></tr><tr><td rowspan=1 colspan=2>HPIV-2 2X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=2>HPIV-2 100X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27.8</td></tr><tr><td rowspan=1 colspan=2>HPIV-3 10X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>28.4</td></tr><tr><td rowspan=1 colspan=2>HPIV-3 1000X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>27.9</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 2X LoD</td><td rowspan=1 colspan=1>HPIV-2 2X LoD</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 2X LoD</td><td rowspan=1 colspan=1>HPIV-2 100X LoD</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>22.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.0</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 2X LoD</td><td rowspan=1 colspan=1>HPIV-3 10X LoD</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>28.0</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 2X LoD</td><td rowspan=1 colspan=1>HPIV-3 1000X LoD</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>27.4</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 2X LoD</td><td rowspan=1 colspan=1>HPIV-3 10X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>27.2</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 2X LoD</td><td rowspan=1 colspan=1>HPIV-3 1000X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 2X LoD</td><td rowspan=1 colspan=1>HPIV-1 100X LoD</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 10X LoD</td><td rowspan=1 colspan=1>HPIV-1 100X LoD</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>28.4</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 10X LoD</td><td rowspan=1 colspan=1>HPIV-2100X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>30.7*</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 100X LoD</td><td rowspan=1 colspan=1>HPIV-2 100X LoD</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 100X LoD</td><td rowspan=1 colspan=1>HPIV-3 1000X LoD</td><td rowspan=1 colspan=1>25.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25.2</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 100X LoD</td><td rowspan=1 colspan=1>HPIV-3 1000X LoD</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>28.4</td></tr></table>

$\ast _ { \mathrm { \scriptsize ~ n = 2 } }$ ; one replicate negative for HPIV-3. Upon retest 3/3 replicates were positive for HPIV-3

# No competitive inhibition was observed for detection of HPIV using the ProParaflu $^ +$ Assay.

# l. Carry-Over Contamination:

To evaluate the degree of carry-over/cross-contamination that occurs with the use of the ProParaflu $^ +$ Assay in association with nucleic acid extraction on the Roche MagNA Pure LC and the bioMérieux NucliSens easyMAG instruments and PCR on the Cepheid SmartCycler II thermocycler, an internal Carry-Over study was carried out by testing simulated human Parainfluenza type 2 (HPIV-2) high positive samples run in series with HPIV-2 high negative samples over 5 runs. The High Positive samples consisted of negative nasopharyngeal (NP) swab matrix spiked with a high concentration of HPIV-2, strain Greer at 2 logs above the Limit of Detection (LoD). High Positive samples represented the upper range of $\mathrm { C t }$ values $( \mathord { \sim } 2 3 - 2 5 \mathrm { C t } )$ obtained in the ProParaflu $^ +$ clinical trials (The Ct range of the true positives for HPIV-2 in the clinical trials was 23.9 – 34.4). High Negative samples consisted of negative NP swab matrix spiked with a concentration of HPIV-2 below but near the analytical LoD concentration (3 logs below LoD) such that $> 9 5 \%$ of samples should be negative. The samples were processed and extracted in a “High Positive/High Negative” alternating fashion (i.e. checkerboard pattern) on the bioMérieux NucliSENS easyMAG and likewise processed on the Roche MagNA Pure LC and run on the Cepheid SmartCycler II instrument in an alternating fashion. A negative control (viral transport medium) spiked with the IC was also included with each

extraction run. Eleven (11) High Positive (HP) and 11 High Negative (HN) samples and a Negative Control were extracted per run on the easyMAG and on the MagNA Pure LC for a total of 110 High Positive samples and 110 High Negative samples over 5 runs.

Carry-Over Contamination Study results are presented in the following tables:

Carry-Over/Cross-Contamination results for easyMAG Extractor   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Total/Expected</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>Std. Dev</td><td rowspan=1 colspan=1>Mean ICCt</td><td rowspan=1 colspan=1>Std.Dev.</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 Positive</td><td rowspan=1 colspan=1>1a/0</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 Negative</td><td rowspan=1 colspan=1>109/110</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.00</td><td rowspan=1 colspan=1>1.21</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Positive</td><td rowspan=1 colspan=1>57b/55</td><td rowspan=1 colspan=1>22.77</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Negative</td><td rowspan=1 colspan=1>53b/55</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>27.42</td><td rowspan=1 colspan=1>0.75</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 Positive</td><td rowspan=1 colspan=1>4/0</td><td rowspan=1 colspan=1>28.35</td><td rowspan=1 colspan=1>5.37</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 Negative</td><td rowspan=1 colspan=1>106/110</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>27.99</td><td rowspan=1 colspan=1>1.21</td></tr></table>

$\mathrm { ^ { a } H P I V { - } 1 }$ false positive (1), retested in duplicate starting from RT-PCR. Repeat test results negative for HPIV-1 for both replicates. b HPIV-2 false positives (2) retested in duplicate starting from RT-PCR. Repeat test results negative for HPIV-2 for all replicates. c HPIV-3 false positives (4) retested in duplicate starting from RT-PCR. Repeat test results negative for HPIV-3 for all replicates.

Carry-Over/Cross-Contamination results for MagNA Pure Extractor   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Total /Expected</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>Std.Dev</td><td rowspan=1 colspan=1>Mean ICCt</td><td rowspan=1 colspan=1>Std.Dev.</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 Positive</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 Negative</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.53</td><td rowspan=1 colspan=1>0.70</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Positive</td><td rowspan=1 colspan=1>58/55</td><td rowspan=1 colspan=1>23.27</td><td rowspan=1 colspan=1>3.03</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Negative</td><td rowspan=1 colspan=1>52²/55</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>2.76</td><td rowspan=1 colspan=1>28.27</td><td rowspan=1 colspan=1>0.53</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 Positive</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 Negative</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.53</td><td rowspan=1 colspan=1>0.70</td></tr></table>

a HPIV-2 false positives (3) retested in duplicate starting from RT-PCR. Repeat test results negative for HPIV-2 for all replicates with the exception of one sample that was still HPIV-2 positive for 1 of 2 replicates.

Combined Carry-Over/Cross-Contamination Data Summary   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Total/Expected</td><td rowspan=1 colspan=1>MeanHPIV-2 Ct</td><td rowspan=1 colspan=1>Std.Dev</td><td rowspan=1 colspan=1>MeanIC Ct</td><td rowspan=1 colspan=1>Std.Dev.</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Positive</td><td rowspan=1 colspan=1>110/110</td><td rowspan=1 colspan=1>22.41</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HPIV-2 Negative</td><td rowspan=1 colspan=1>105/110 (95.4%)</td><td rowspan=1 colspan=1>36.44</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>27.84</td><td rowspan=1 colspan=1>0.77</td></tr><tr><td rowspan=1 colspan=1>HPIV-1 Negative</td><td rowspan=1 colspan=1>219a,b/220 (99.5%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.27</td><td rowspan=1 colspan=1>1.02</td></tr><tr><td rowspan=1 colspan=1>HPIV-3 Negative</td><td rowspan=1 colspan=1>216a,b/220 (98.6%)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>28.27</td><td rowspan=1 colspan=1>1.02</td></tr></table>

a One sample (HPIV-2 High Negative) was positive for HPIV-1, -2 and 3. Repeat testing starting from RT-PCR in duplicate resulted in a negative result for all 3 analytes. An additional four samples (HPIV-2 High Negative) were positive for HPIV-2. Repeat testing starting from RT-PCR in duplicate resulted in 3 of the 4 negative for HPIV-2 in both replicates and 1 sample positive for HPIV-2 in one replicate and negative for HPIV-2 in the other.

b Three samples (HPIV-2 High Positive) were positive for HPIV-2 and -3. Repeat testing starting from RT-PCR in duplicate resulted in all 3 samples positive for HPIV-2 only for both replicates.

There was some evidence of negative samples having a positive HPIV-2 signal, however this was not consistent by extraction instrument or by extraction run. As repeat testing starting from the purified nucleic acid demonstrated negative results, potential contamination could have occurred when transferring the purified nucleic acid samples from the sample cartridge to microfuge tubes or during the RT-PCR set-up. However, the possibility that these samples fall into the $5 \%$ category of High Negatives could not be ruled out. HPIV-1 and $^ { - 3 }$ positives were also found in both positive (3) and negative (1) samples, which may indicate contamination from the positive control or from the environment due to the high volume of Parainfluenza testing that occurred prior to and during performance of the Carryover Study. Repeat ProParaflu $^ +$ results of these samples were negative for HPIV-1 and HPIV-3 when the purified nucleic acid was retested.

This ProParaflu+ Carryover Study demonstrated that low level carry-over/cross contamination with the ProParaflu $^ +$ Assay using either the Roche MagNA Pure LC or the bioMériuex NucliSens easyMAG automated nucleic acid extraction instruments and the Cepheid Smartcycler II could happen. Therefore, the following statements were included in the Limitations section to address the potential of cross-contamination when performing the assay: “The detection of viral nucleic acid is dependent upon proper specimen collection, handling, transportation, storage, and preparation (including extraction). Failure to observe proper procedures in any one of these steps can lead to incorrect results.”; “There is a risk of false positive values resulting from crosscontamination by target organisms, their nucleic acids or amplified product, or from non-specific signals in the assay.”

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix Description and Comparison:

Analytical performance of the ProParaflu $^ +$ Assay using six different viral transport media (VTM) was evaluated. Analytical sensitivity was determined using cultured and titered strains of human parainfluenza viruses (HPIV-1 C35, HPIV-2 Greer, and HPIV-3 C243) spiked into each of the different VTMs. The VTM were considered “equivalent” if similar Limits of Detection (LoD) were achieved for each.

The intended use of Remel M4, M5, and M6, Copan Universal Transport Medium (UTM), and Becton Dickinson Universal Viral Transport (UVT) is for the transport of clinical specimens containing viruses, chlamydiae, mycoplasma, and ureaplasma from the collection site to the laboratory for microbiological

procedures. Remel M4RT VTM is for the transport of clinical specimens containing viruses or chlamydiae, but is not recommended for the transport of mycoplasmas and ureaplasmas. The VTMs have essentially the same components as shown in the table below:

Composition of Viral Transport Media.   

<table><tr><td rowspan=1 colspan=1>Remel M4</td><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>Remel M6</td><td rowspan=1 colspan=2>Remel M4 RT</td><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>Copan UTM</td></tr><tr><td rowspan=10 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericin BCollistin</td><td rowspan=10 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminProteinStabilizersSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericin BCollistin</td><td rowspan=1 colspan=1>Hank&#x27;sBalanced Salts</td><td rowspan=1 colspan=2>Hank&#x27;sBalanced Salts</td><td rowspan=1 colspan=1>Hank&#x27;sBalanced Salts</td><td rowspan=10 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericin BCollistinL-Cysteine</td></tr><tr><td rowspan=1 colspan=1>Bovine Serum</td><td rowspan=1 colspan=2>Bovine Serum</td><td rowspan=1 colspan=1>Bovine Serum</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=2>Albumin</td><td rowspan=1 colspan=1>Albumin</td></tr><tr><td rowspan=4 colspan=1>GelatinSucroseL-GlutamicacidHEPES BufferPhenol Red</td><td rowspan=1 colspan=2>Gelatin</td><td rowspan=3 colspan=1>GelatinSucroseL-Glutamicacid</td></tr><tr><td rowspan=2 colspan=2>SucroseL-Glutamicacid</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>HEPES BufferPhenol Red</td><td rowspan=1 colspan=1>HEPES BufferPhenol Red</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=2>Amphotericin B</td><td rowspan=1 colspan=1>Vancomycin</td></tr><tr><td rowspan=1 colspan=1>Amphotericin B</td><td rowspan=1 colspan=2>Collistin</td><td rowspan=1 colspan=1>Amphotericin B</td></tr><tr><td rowspan=1 colspan=1>Collistin</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>CollistinL-Cysteine</td></tr></table>

Cultured and titered viruses were spiked at one log above, at, and one log below their limit of detection (as determined in the ProParaflu $^ +$ Analytical Sensitivity Study using M4) along with an Internal RNA Control into each of the media. Samples were extracted using the bioMerieux NucliSens easyMAG and tested in triplicate RT-PCR reactions on the Cepheid Smartcycler II using one lot of ProParaflu+ reagents. Negative Controls consisted of each of the six different VTM spiked with IC. Nucleic acid isolation of the Negative Controls was performed along with the VTM study samples. The Negative Controls serve to monitor for contamination or the presence of inhibitors. A ProParaflu+ Positive Control was included with each run to test for global errors (absence of reagents, instrument failure, etc.). The Positive Control did not require nucleic acid isolation and was diluted just prior to set up of RT-PCR reactions.

Media were considered “equivalent” to Remel M4 (the reference standard) if:

a) for each virus and strain, all three replicates for at least one concentration was positive and within one log of the lowest concentration in which all 3 replicates of M4 are positive, and   
b) the average $\mathrm { C _ { T } }$ values at all concentrations in which 3 of 3 replicates were positive for each virus did not differ by more than $3 \mathrm { C r s }$ from the average $\mathrm { C _ { T } }$ of that virus diluted in M4.

Each of the VTM performed “equivalently” to Remel M4 as they all met the required criteria described above. Detailed data are presented in the following table:

VTM Study Data   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">Media</td><td colspan="1" rowspan="1">AverageCT</td><td colspan="1" rowspan="1">StDev</td><td colspan="1" rowspan="1">Ct Differenceto M4</td></tr><tr><td colspan="1" rowspan="18">HPIV-1</td><td colspan="1" rowspan="6">1 x 103</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">27.17</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">26.80</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">27.07</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">27.17</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">27.00</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.17</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">27.27</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="6">1 x 102(LoD)</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">30.50</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">30.50</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">30.43</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.07</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">31.33</td><td colspan="1" rowspan="1">1.01</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">32.67</td><td colspan="1" rowspan="1">1.63</td><td colspan="1" rowspan="1">2.17</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">30.90</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.40</td></tr><tr><td colspan="1" rowspan="6">1 x 101</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">34.23</td><td colspan="1" rowspan="1">0.65</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">34.15*</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.08</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">35.03</td><td colspan="1" rowspan="1">1.76</td><td colspan="1" rowspan="1">0.80</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">34.13</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">34.63</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.40</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">34.83</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">0.60</td></tr><tr><td colspan="1" rowspan="18">HPIV-2</td><td colspan="1" rowspan="6">1 x 103</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">26.07</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">25.97</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">25.47</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.60</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">25.33</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">0.74</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">25.60</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.47</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">25.70</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">0.37</td></tr><tr><td colspan="1" rowspan="6">1 x 102(LoD)</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">29.33</td><td colspan="1" rowspan="1">0.06</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">28.90</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.43</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">28.97</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.36</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">29.10</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">29.17</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.16</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">29.10</td><td colspan="1" rowspan="1">0.95</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="6">1 x 101</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">32.60</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">33.30</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">0.70</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">32.57</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">32.05*</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">0.55</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">33.07</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">0.47</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">32.80</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.20</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">Media</td><td colspan="1" rowspan="1">AverageCT</td><td colspan="1" rowspan="1">StDev</td><td colspan="1" rowspan="1">Ct Differenceto M4</td></tr><tr><td colspan="1" rowspan="18">HPIV-3</td><td colspan="1" rowspan="6">5 x 102</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">27.07</td><td colspan="1" rowspan="1">0.60</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">27.53</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">0.46</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">27.37</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">26.97</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">27.47</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.40</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">27.90</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.83</td></tr><tr><td colspan="1" rowspan="6">5 x 101(LoD)</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">31.90</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">31.60</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">0.30</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">31.57</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">0.33</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">31.50*</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.40</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">31.30*</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.60</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">31.83</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">0.70</td></tr><tr><td colspan="1" rowspan="6">5 x 100</td><td colspan="1" rowspan="1">Remel M4</td><td colspan="1" rowspan="1">35.13</td><td colspan="1" rowspan="1">0.12</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">34.60</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.53</td></tr><tr><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">34.90*</td><td colspan="1" rowspan="1">0.14</td><td colspan="1" rowspan="1">0.23</td></tr><tr><td colspan="1" rowspan="1">Remel M4RT</td><td colspan="1" rowspan="1">34.80*</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">0.33</td></tr><tr><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">35.20</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">0.70</td></tr><tr><td colspan="1" rowspan="1">BD UVT</td><td colspan="1" rowspan="1">35.57</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">0.44</td></tr></table>

\* $_ { \mathrm { N } = 2 }$

# All six VTM evaluated are compatible with the ProParaflu $+$ Assay.

3. Clinical studies:

# a. Prospective Clinical studies

The clinical performance of the ProParaflu $^ +$ Assay was established during prospective studies at 4 U.S. clinical laboratories during May 2008 to September 2009. All specimens used in the study meeting the inclusion and exclusion criteria represented excess, remnants of nasopharyngeal (NP) swab specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study. Inclusion criteria included, but were not limited to: the specimen was from a symptomatic patient (for respiratory infection) that was submitted for routine respiratory pathogen testing; the specimen was a NP swab in appropriate viral transport media (i.e. Remel M4 or M5 viral transport medium); the specimen contained adequate volume for performing the culture and the ProParaflu+ Assay; age and gender information were available; specimen was stored properly (refrigerated at $2 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C } )$ ; and initiation of the culture and the ProParaflu $^ +$ Assay (nucleic acid extraction) took place within 72 hours of sample collection.

Performance of the ProParaflu $^ +$ Assay was assessed and compared to the reference method of culture followed by direct fluorescent antibody (DFA) for differentiation of HPIV types. For site 1 and 3, the reference method consisted of traditional cell culture using Rhesus Monkey Kidney (RMK) cells (Viromed, Minnetonka, MN) followed by hemadsorption (HAd) using guinea pig red blood cells. Samples determined to be positive by HAd were then stained for HPIV typing using DFA and the Parainfluenza Virus Screening and Identification Kit (Light Diagnostics, Temecula, CA). One HPIV positive and one negative (uninoculated tube) was run with each set of samples. Samples were determined to be negative if there was no HAd observed after 10 days or if the DFA was negative for HPIV. Samples were determined to be positive if the HAd test was positive and the HPIV DFA was positive for either HPIV-1, -2 or -3. For site 2 and 4, the reference method consisted of rapid cell culture using Mv1Lu/A549 cells (R-Mix, Diagnostic Hybrids, Inc, Athens, OH). Shell vials were set up in duplicate and one vial was stained after 48 hours with a mixture of monoclonal antibodies (Influenza A, Influenza B, RSV, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3, and Adenovirus) using the Respiratory Panel 1 Virus Screening and Identification Kit (Light Diagnostics, Temecula, CA). If the screen was positive, an 8 well slide was prepared using the second well and the cells were tested with individual monoclonal antibodies for each of the above 7 viruses. One positive (rotating among Influenza A, Influenza B, RSV, Parainfluenza 1, Parainfluenza 2, Parainfluenza 3 and Adenovirus) and one negative (uninoculated well) was run with each set of samples. A control slide containing wells with cell culture derived positive and negative control cells (provided with the Respiratory Panel 1 Screening Kit) was also run each time the staining procedure was performed. Samples were determined to be positive if bright green fluorescence was observed in cells, negative if no staining was observed in cells, and invalid or toxic if non-specific staining or dead cells were observed.

A total of 857 eligible prospective NP swab samples (407 from female patients and 450 from male patients) were tested with the ProParaflu $^ +$ Assay and with the reference method at the 4 clinical sites. The NP swab specimens underwent nucleic acid extraction using either the Roche MagNA Pure Total Nucleic Acid Isolation Kit on the MagNA Pure LC Instrument (Site 4) or using the bioMérieux NucliSENS easyMAG instrument with Automated Magnetic Extraction Reagents (Sites 1, 2, 3). An Internal RNA Control (IC) was added to every NP swab sample. A Negative Control which consisted of viral transport medium and an Extraction Control (a well characterized human parainfluenza 3 virus) were included with each extraction run. Purified nucleic acids were further processed immediately following extraction or were stored frozen $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ and thawed prior to ProParaflu+ Assay set-up. Nucleic acid detection was performed using the ProParaflu $^ +$ Assay on the Cepheid SmartCycler II.

The overall assay failure rate of the ProParaflu+ Assay during the clinical trials due to failure of positive, negative or extraction controls not related to procedural errors was $0 . 9 \%$ (total of 1 of 113 runs across all 4 sites). In this run the NC failed due to no detection of the IC. Upon repeat testing starting from the purified nucleic acid from the NC and all samples the NC was Valid and the repeat run results accepted and used in final analysis.

A total of 1015 prospective respiratory specimens were initially included in the clinical trial. One hundred thirty eight (138) samples (56 samples from Site 1, 60 samples from Site 2, 17 samples from Site 3, and 5 samples from Site 4) were excluded from the clinical study due to $> 7 2$ hours from time of collection to ProParaflu $^ +$ Assay testing and/or culture testing. Five (5) specimens (all from Site 1) were excluded because of improper sample type. Twelve (12) samples (all from Site 1) were excluded because they had data from only one method (ProParaflu+ or culture). One (1) sample was excluded because it had too few cells to culture (from Site 1). And 2 samples (1 from Site 2 and 1 from Site 4) were excluded because they remained “Unresolved” after repeat testing. Of the ProParaflu $^ +$ Assay run on all eligible specimens, $9 9 . 2 \%$ (852/857) of these specimens were successful on the first attempt (Site 1: $1 2 4 / 1 2 6 = 9 8 . 4 \%$ Site 2: $4 5 5 / 4 5 6 = 9 9 . 8 \%$ ; Site 3: $1 8 2 / 1 8 2 \ =$ $1 0 0 . 0 \%$ ; Site $4 \colon 9 1 / 9 3 = 9 7 . 8 \%$ ). The remaining 5 gave “Unresolved” results on the first attempt (2 from Site 1, 1 from Site 2, none from Site 3, and 2 from Site 4). Unresolved results occur when the sample is negative for all three HPIVs and the Internal Control, indicating potentially PCR-inhibiting samples. Of the 5 “Unresolved” specimens on the first attempt with sufficient sample for retest, only $6 0 . 0 \%$ (3/5) gave a valid result on the second attempt (2 from Site 1 and 1 from Site 4). The remaining 2 were “Unresolved” on the second attempt (both were tested negative by the culture reference method), therefore, were not eligible for inclusion in the data analysis.

Site poolability of performance data was determined based on the fact that similar ProParaflu $^ +$ performance were obtained from the 4 clinical sites:

HPIV 1   

<table><tr><td rowspan=2 colspan=1>ProParaflu+HPIV 1ResultsSite 1</td><td rowspan=1 colspan=3>Culture Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 0/0      0% (95% CI: NA)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 126/126 100.0% (95% CI: 97.1 % - 100.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>ProParaflu+ HPIV 1ResultsSite 2</td><td rowspan=1 colspan=5>Culture Results</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=2>16</td><td rowspan=1 colspan=2>1</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=2>439</td><td rowspan=1 colspan=1>439</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=2>16</td><td rowspan=1 colspan=2>440</td><td rowspan=1 colspan=1>456</td></tr><tr><td rowspan=1 colspan=6>Sensitivity: 16/16      100% (95% CI: 80.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=6>Specificity: 439/440 99.8% (95% CI: 98.7 % - 100.0%)</td></tr><tr><td rowspan=2 colspan=2>ProParaflu+HPIV 1ResultsSite 3</td><td rowspan=1 colspan=4>Culture Results</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=2>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=2>2</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>182</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>2</td><td></td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>182</td></tr><tr><td rowspan=1 colspan=6>Sensitivity: 0/2      0% (95% CI: NA)</td></tr><tr><td rowspan=1 colspan=6>Specificity: 180/180 100.0% (95% CI: 98.0 % - 100.0%)</td></tr><tr><td rowspan=2 colspan=2>ProParaflu+HPIV 1ResultsSite 4</td><td rowspan=1 colspan=4>Culture Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=2>Positive</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>0</td><td></td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=6>Sensitivity: 0/0    0% (95% CI: NA)</td></tr><tr><td rowspan=1 colspan=6>Specificity: 93/93 100.0% (95% CI: 96.1 % - 100.0%)</td></tr></table>

HPIV 2   

<table><tr><td colspan="1" rowspan="2">ProParaflu+HPIV 2ResultsSite 1</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">126</td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">126</td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 0/0      0%(95%CI:NA)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 126/126 100.0% (95% CI: 97.1 % - 100.0%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+ HPIV 2ResultsSite 2</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">449</td><td colspan="1" rowspan="1">449</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">449</td><td colspan="1" rowspan="1">456</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 7/7       100% (95% CI: 64.6% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Specificity:449/449 100.0% (95% CI: 99.2 % - 100.0%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+ HPIV 2ResultsSite 3</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">161</td><td colspan="1" rowspan="1">161</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">163</td><td colspan="1" rowspan="1">182</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 19/19   100.0% (95% CI: 82.4% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Specificity:: 161/163 98.8% (95% CI: 95.6 % - 99.8%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+ HPIV 2ResultsSite 4</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 0/1    0% (95% CI: NA)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 92/92 100.0% (95% CI: 96.1 % - 100.0%)</td></tr></table>

HPIV 3   

<table><tr><td colspan="1" rowspan="2">ProParaflu+HPIV 3ResultsSite 1</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">124</td><td colspan="1" rowspan="1">124</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">125</td><td colspan="1" rowspan="1">126</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 1/1      100.0% (95% CI: NA)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 124/125 99.2% (95% CI: 95.6% - 100.0%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+HPIV 3ResultsSite 2</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">432</td><td colspan="1" rowspan="1">433</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">436</td><td colspan="1" rowspan="1">456</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 19/20   95.0% (95%CI: 76.4% - 99.1%)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 432/436 99.1% (95% CI: 97.7 % - 99.6%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+HPIV 3ResultsSite 3</td><td colspan="3" rowspan="1">Culture Results</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">14</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">168</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">171</td><td colspan="1" rowspan="1">182</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 11/11   100.0% (95% CI: 71.5% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 168/171 98.2% (95% CI: 95.0 % - 99.6%)</td></tr><tr><td colspan="1" rowspan="2">ProParaflu+HPIV 3ResultsSite 4</td><td colspan="2" rowspan="1">Culture Results</td><td colspan="1" rowspan="1">cYanmlow</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">88</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">93</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 5/5   100.0% (95% CI: 47.8% - 100.0%)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 88/88 100.0% (95% CI: 95.9 % - 100.0%)</td></tr></table>

Samples that did not yield the same results between ProParaflu $^ +$ and viral culture were considered “discrepant”. The ProParaflu $^ +$ Clinical Trial Discrepant Analysis was designed to study these discrepancies by performing RT-PCR on the original purified nucleic acids followed by bi-directional sequencing of the resultant PCR product to determine the presence or absence of ProParaflu $^ +$ Assay target viruses (HPIV-1, -2, or -3). Oligonucleotide primer sets used were derived from literature and were specific for each HPIV type (1, 2 and 3). The gene target was the hemagglutinin gene, but primers targeted a different region than the ProParaflu+ Assay. Samples were sequenced using both the forward or reverse PCR primers used in the PCR reaction.

The performance data from all study sites combined with discrepant analysis results as footnotes are presented in the following table:

HPIV-1   

<table><tr><td rowspan=2 colspan=1>ProParaflu+HPIV-1 Results, allsites combined</td><td rowspan=1 colspan=3>Culture Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>838</td><td rowspan=1 colspan=1>840</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>839</td><td rowspan=1 colspan=1>857</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 16/18            88.9% (95%CI:67.2% - 96.9%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 838/839     99.9% (95% CI: 99.3 % - 100.0%)</td></tr></table>

a One sample positive for HPIV-1 by sequence analysis. b One sample positive for HPIV-3 using ProParaflu+ and confirmed HPIV-3 positive by sequencing. Result from repeat culture followed by DFA testing using freeze-thawed specimen was also HPIV-3 positive. Results from repeat culture followed by DFA testing using freeze-thawed specimens for both samples were negative for HPIV-1. Both samples were also negative for HPIV-1 by sequence analysis.

HPIV-2   

<table><tr><td rowspan=2 colspan=1>ProParaflu+ HPIV-2 Results, all sitescombined</td><td rowspan=1 colspan=3>Culture Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>829</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>830</td><td rowspan=1 colspan=1>857</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 26/27          96.3%(95%CI::81.7 % - 99.3%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 828/830      99.8% (95% CI: 99.1 % - 99.9%)</td></tr></table>

a Two (2) samples positive for HPIV-2 by sequence analysis. b One sample negative for HPIV-2 by sequence analysis.

HPIV-3   

<table><tr><td rowspan=2 colspan=1>ProParaflu+ HPIV-3 Results, all sitescombined</td><td rowspan=1 colspan=3>Culture Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>8a</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>813</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>820</td><td rowspan=1 colspan=1>857</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 36/37          97.3%(95%CI:86.2 % - 99.5%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 599/605      99.2%(95%CI: 98.1 % - 99.5%)</td></tr></table>

a Seven (7) samples positive for HPIV-3 by sequence analysis and 1 sample negative for HPIV-3 by sequence analysis. b One (1) sample negative for HPIV-3 by sequence analysis.

The general demographic data for all eligible prospective specimens $\scriptstyle ( \mathrm { N } = 8 5 7$ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>407 (47.5%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>450 (52.5%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>580 (67.7%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>168 (19.6%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>67 (7.8%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>42 (4.9%)</td></tr></table>

# $b$ . Retrospective Clinical studies

Throughout the course of the study, only 17 prospective HPIV-1 positive samples as determined by the ProParaflu $^ +$ Assay were found at any of the four clinical sites. A total of 91 retrospective samples that included 29 HPIV-1 positive and 62 HPIV-1 negative NP swab samples were tested at Sites 2, 3 and 4. These samples (obtained from Sites 2 and 3) had been previously determined to be HPIV-1 positive or negative by direct DFA testing on fresh specimens and were then subsequently frozen at $- 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ . These data were analyzed separately from the prospective samples and are summarized in the following table:

<table><tr><td rowspan=2 colspan=1>ProParaflu+ HPIV-1 Results, all sitescombined</td><td rowspan=1 colspan=3>Direct DFA Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5a</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 24/2982.8% (95% CI: 65.4% - 92.4%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 62/62 100% (95% CI: 94.2% - 100.0%)</td></tr></table>

a Five (5) samples negative for HPIV-1 by sequence analysis.

The general demographic data for all eligible retrospective specimens $( \mathrm { N } { = } 9 1 $ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>Number of Subjects (Percentage of Total)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>40 (44.4%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>50 (55.6%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>81 (90.0%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21 years</td><td rowspan=1 colspan=1>5 (5.6%)</td></tr><tr><td rowspan=1 colspan=1>22 - 59 years</td><td rowspan=1 colspan=1>2 (2.2%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>2 (2.2%)</td></tr></table>

c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range:

In the prospective ProParaflu $^ +$ Assay clinical study, a total of 857 eligible nasopharyngeal (NP) swab specimens were tested from four U.S. clinical laboratories across the United States from May 2008 to September 2009. The number and percentage of HPIV-1, HPIV-2 and HPIV-3 RNA positive cases as determined by the ProParaflu $^ +$ Assay, calculated by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1>AgeGroup</td><td rowspan=1 colspan=1>TotalN</td><td rowspan=1 colspan=1>HPIV-1Positive BytheProParaflu+Assay</td><td rowspan=1 colspan=1>HPIV-2Positive BytheProParaflu+Assay</td><td rowspan=1 colspan=1>HPIV-3Positive BytheProParaflu+Assay</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>ObservedPrevalenceHPIV-1</td><td rowspan=1 colspan=1>ObservedPrevalenceHPIV-2</td><td rowspan=1 colspan=1>ObservedPrevalenceHPIV-3</td></tr><tr><td rowspan=1 colspan=1>&lt;1year</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>8.8%</td></tr><tr><td rowspan=1 colspan=1>1-5years</td><td rowspan=1 colspan=1>249</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>4.4%</td></tr><tr><td rowspan=1 colspan=1>6-10years</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>11-15years</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>16-21years</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>&gt;21years</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>2.8%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>857</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>5.0%</td></tr></table>

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a

MagNA Pure LC Instrument (Roche)

NucliSENS® easyMAG™ System (bioMérieux)

# O. System Descriptions:

# 1. Modes of Operation:

The Cepheid Smart Cycler II Real Time instrument with Dx software version 1.7b or 3.0a is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Five other nucleic acid amplification tests that use the Smart Cycler II instrument have received 510(k) clearance: Prodesse’s ProhMPV $^ +$ Assay (K082688) and ProFlu $^ +$ Assay (K081030), IDI-MRSA test (K033415), the IDI-Strep B Assay (K022504), and the Influenza A/H5 (Asian lineage) Virus Real-Time RT-PCR Primer and Probe Set (K060159). The Cepheid SmartCycler instrument is an integrated nucleic acid amplification and detection instrument system based on Cepheid’s proprietary microprocessor-controlled I-CORE module. For purified RNA samples, the SmartCycler instrument enables reversetranscriptase (RT) to transcribe target viral RNA into cDNA, polymerase chain reaction (PCR) for the amplification of cDNA, and hybridization of fluorogenic target-specific probes for the detection of the amplified cDNA.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles. Four other tests that use the Roche MagNA Pure LC system have received 510(k) clearance: Prodesse’s ProhMPV $+$ Assay (K082688) and ProFlu+ Assay (K081030), Roche Factor V Leiden Kit (K033607) and the Roche Factor II (Prothrombin) G20210A Kit (K033612).

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads. Prodesse’s ProFlu+ Assay (K081030) that received Special $5 1 0 ( \mathrm { k } )$ clearance and Prodesse’s ProhMPV $^ +$ Assay (K082688) also use the easyMAG system for automated nucleic acid extraction.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The following controls are provided in the ProParaflu $^ +$ Assay kit:

Positive Control (PC): The ProParaflu+ Assay kit contains a parainfluenza positive RNA control that consists of three RNA transcripts (specific for each HPIV type 1, 2, and 3). These transcripts are derived from plasmids containing the viral sequences of interest and are not intact virus particles. The Parainfluenza PC contains unequal amounts of HPIV-1, HPIV-2 and HPIV-3. The approximate concentration of each transcript was $\mathrm { 5 x 1 0 ^ { 4 } c o p i e s / \mu l } ,$ $2 . 5 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ copies $\mu \mathrm { l }$ and $2 . 5 \mathrm { \bf ~ X }$ $1 0 ^ { 3 }$ copies $\mu \mathrm { l }$ for HPIV-1, HPIV-2 and HPIV-3, respectively. The PC does not go through nucleic acid isolation and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance.

An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working.

The following controls are not provided in the ProParaflu $^ +$ Assay kit, but are required or recommended and described in the ProParaflu $^ +$ Assay Instructions for Use:

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the ProParaflu $^ +$ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a well characterized human parainfluenza 3 virus, was spiked with the IC and included with each nucleic acid isolation run as EC. The EC serves to monitor for lysis and contamination during nucleic acid isolation. The sponsor is recommending an extraction control in each nucleic acid extraction run to the end users in the package insert.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision