# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k091409

E. Applicant: Roche Molecular Systems, Inc

F. Proprietary and Established Names: LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test

# G. Regulatory Information:

1. Regulation section: 21CFR866. 1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code: NQX - Nucleic Acid Amplification test, MRSA, Direct Specimen OOI - Nuclei Acid Amplification System, Real Time

4. Panel: 83 - Microbiology

# H. Intended Use:

1. Intended use: The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test is a qualitative in vitro diagnostic test for the direct detection of nasal colonization with methicillin-resistant (MRSA) to aid in the prevention and control of MRSA infections in healthcare settings. The test is performed on the LightCycler $^ \mathrm { \textregistered }$ 2.0 Instrument with nasal swab specimens from patients suspected of colonization, uses swab extraction and mechanical lysis for specimen preparation followed by polymerase chain reaction (PCR) for the amplification of MRSA DNA, and fluorogenic target specific hybridization probes for the detection of the amplified DNA.

The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test is not intended to diagnose, guide or monitor treatment for MRSA infections. Concomitant cultures are necessary to recover organisms for epidemiology typing or for further susceptibility testing.

2. Indication(s) for use:

The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test is a qualitative in vitro diagnostic test for the direct detection of nasal colonization with methicillin-resistant (MRSA) to aid in the prevention and control of MRSA infections in healthcare settings. The test is performed on the LightCycler $^ \mathrm { \textregistered }$ 2.0 Instrument with nasal swab specimens from patients suspected of colonization, uses swab extraction and mechanical lysis for specimen preparation followed by polymerase chain reaction (PCR) for the amplification of MRSA DNA, and fluorogenic target specific hybridization probes for the detection of the amplified DNA.

The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test is not intended to diagnose, guide or monitor treatment for MRSA infections. Concomitant cultures are necessary to recover organisms for epidemiology typing or for further susceptibility testing.

3. Special conditions for use statement(s): Prescription Use

4. Special instrument requirements: To be used with the LightCycler 2.0 Instrument

# I. Device Description:

Specimen collection swab heads are cut into lysis tubes and subjected to heating to inactivate specimens. Subsequently lysis tubes are transferred in the MagNA Lyser Instrument where mechanical lysis of bacterial cell walls occurs, resulting in crude lysate preparations. After a brief centrifugation step to spin down glass beads and swab fibers, the processed specimen are subjected to PCR analysis using the LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test.

The processed samples and the amplification mixture containing hot-start Taq polymerase are placed in LightCycler $\textsuperscript { \textregistered }$ Capillaries $( 2 0 \mu \mathrm { L } )$ in which PCR amplification will occur. Each LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test reaction contains an internal control, which is designed to control for specimen inhibition, and to monitor reagent integrity. Also present in each LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test is the AmpErase (uracil-N-glycosylase) enzyme. It recognizes and catalyzes the destruction of DNA strands containing deoxyuridine, but not DNA containing

deoxythymidine. Since amplicons produced with the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test contain deoxyuridine, potential amplicon contaminants are eliminated during a prolonged heating step performed prior to the start of PCR amplification.

A target sequence in a plasmid is simultaneously amplified in the Positive Control. The Positive Control is intended to monitor for reagent failure and is included into each run. Each run also includes a Negative Control used to detect reagent or environmental contamination by MRSA DNA.

MRSA and Internal Control amplicons are detected by fluorescence using a specific pair of

hybridization probes. The probes attach to a specific internal sequence in the amplified fragment and are positioned in a closed proximity to one another. Upon excitation, these bound probes emit a fluorescence signal of a specific wavelength using a process called Fluorescence Resonance Energy Transfer (FRET). The emitted light is measured by the LightCycler $\textsuperscript { \textregistered }$ 2.0 Instrument. MRSA or internal control specific amplicons are detected in parallel in two different detection channels and thus can be differentiated.

After completion of the real-time PCR process, a melting peak analysis is performed by the LightCycler $^ \mathrm { \textregistered }$ 2.0 Instrument. Single stranded DNA amplicons with bound hybridization probes are subjected to increasing temperatures. When the PCR products reach a specific temperature one of the two bound hybridization probes melts off, resulting in a loss of fluorescence signal. The decrease in the fluorescence signal occurs at a specific temperature and results in melting peaks which are used to identify and distinguish MRSA- and IC-specific amplicons.

After a visual identification of the melting peaks is performed using Tm Bars in the LightCycler $^ \mathrm { \textregistered }$ Software, test results are transferred to a dedicated interpretation tool (the Micro Analysis Software) and a report is generated.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD GeneOhm MRSA Assay

1. Predicate 510(k) number(s): K033415 K042357

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>LightCycler® MRSAAdvanced Test</td><td rowspan=1 colspan=1>BD GeneOhmTMMRSA Assay</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Direct and qualitative detection ofMRSA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Nasal swab</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Lysis</td><td rowspan=1 colspan=1>Mechanical lysis using glass beads</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Mode of detection</td><td rowspan=1 colspan=1>Nucleic acid amplification (DNA)utilizing real-time PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Internal Assay Controls</td><td rowspan=1 colspan=1>Internal assay control to detectinhibitory specimens and to confirmintegrity of reagent in negativesamples</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>External Controls</td><td rowspan=1 colspan=1>Provided: positive control tomonitor for reagent integrity</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DNA TargetSequence</td><td rowspan=1 colspan=1>Sequence incorporating theinsertion site of the SCCmec in theS. aureus orfX gene</td><td rowspan=1 colspan=1>Similar but not identical</td></tr><tr><td rowspan=1 colspan=1>Workflow</td><td rowspan=1 colspan=1>Multiple manual steps includingpipetting, fluid transfer, vortexing,centrifugation and sample heating</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Assay platform</td><td rowspan=1 colspan=1>LightCycler</td><td rowspan=1 colspan=1>SmartCycler</td></tr><tr><td rowspan=1 colspan=1>Detection Chemistry</td><td rowspan=1 colspan=1>Paired target-specifichybridization probes usingfluorescence resonance energytransfer (FRET)</td><td rowspan=1 colspan=1>Target-specific hybridizationprobes (molecular beacontechnology)</td></tr><tr><td rowspan=1 colspan=1>Result analysis</td><td rowspan=1 colspan=1>Melting peak analysis</td><td rowspan=1 colspan=1>Amplification curves analysis</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>2 hours</td><td rowspan=1 colspan=1>60- 75 minutes</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): Not applicable

# L. Test Principle:

The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test relies on three major processes: 1) Specimen preparation by mechanical lysis of the bacterial cell walls

2) PCR amplification of target DNA and detection by specific hybridization probes   
3) Automated result generation after melting peak analysis.

# Specimen Preparation

The mechanical lysis of the nasal swab specimens is performed by using the LightCycler $\textsuperscript { \textregistered }$ Advanced Lysis Kit and the MagNA Lyser Instrument. Swab heads are cut into lysis tubes and subjected to heating to inactivate all bacteria. Subsequently lysis tubes are transferred in the MagNA Lyser Instrument where mechanical lysis of bacterial cell walls occurs, resulting in crude lysate preparations. After a brief centrifugation step to spin down glass beads and swab fibers, the processed specimen are subjected to PCR analysis using the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test.

# PCR Amplification

Target Selection

The primers and probes in the LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test detect a proprietary sequence indicative of the integration of the SCCmec cassette into the $S .$ aureus chromosome, indicating the presence of MRSA DNA.

# Amplification

The processed samples and the amplification mixture containing hot-start Taq polymerase are placed in LightCycler $\textsuperscript { \textregistered }$ Capillaries $( 2 0 \mu \mathrm { L } )$ in which PCR amplification will occur. Each LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test reaction contains an internal control, which is designed to control for specimen inhibition, and to monitor reagent integrity. The AmpErase (uracil-N-glycosylase) enzyme included in the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test recognizes and catalyzes the destruction of DNA strands containing deoxyuridine, but not DNA containing deoxythymidine. Since amplicons produced with the LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test contain deoxyuridine, potential amplicon contaminants are eliminated during a prolonged heating step performed prior to the start of PCR amplification.

A target sequence in a plasmid is simultaneously amplified in the Positive Control. The Positive Control is intended to monitor for reagent failure and is included into each run. Each run also includes a Negative Control used to detect reagent or environmental contamination by MRSA DNA.

# Specific detection of PCR products by Hybridization Probes

MRSA and Internal Control amplicons are detected by fluorescence using a specific pair of hybridization probes. The probes attach to a specific internal sequence in the amplified fragment and are positioned in a closed proximity to one another. Upon excitation, these bound probes emit a fluorescence signal of a specific wavelength using a process called Fluorescence Resonance Energy Transfer (FRET). The emitted light is measured by the LightCycler $^ \mathrm { \textregistered }$ 2.0 Instrument. MRSA or internal control specific amplicons are detected in parallel in two different detection channels and thus can be differentiated.

After completion of the real-time PCR process, a melting peak analysis is performed automatically by the LightCycler $^ \mathrm { \textregistered }$ 2.0 Instrument. Single stranded DNA amplicons with bound hybridization probes are subjected to increasing temperatures. When the PCR products reach a specific temperature one of the two bound hybridization probes melts off, resulting in a loss of fluorescence signal. The decrease in the fluorescence signal occurs at a specific temperature and results in melting peaks which are used to identify and distinguish MRSA- and IC-specific amplicons.

# Automated result generation after melting peak analysis

After a visual identification of the melting peaks is performed using TM Bars in the LightCycler $\textsuperscript { \textregistered }$ Software, test results are transferred to a dedicated interpretation tool (the Micro Analysis Software) and a report is generated.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility study was performed using a 12 member panel with varying concentrations of MRSA and methicillin sensitive Staphylococcus epidermidis (MSSE) at a constant level. Two operators at each of three sites performed each per day for five days on three reagent lots; 4 specimens x 3 replicates x 5 days x 3 sites x 3 lots x 2 operators.

The panel included a negative sample, below the LOD (20 CFU/swab expected to yield a positivity rate of between 30 to $70 \%$ ), weak positive (300 CFU/swab), and positive (800 CFU/swab). The negative panel member yield negative results from a low of $9 9 \%$ to $100 \%$ . The below LOD panel member positivity rate ranged from a low of $42 \%$ to $47 \%$ , the weak positive panel member positivity rate ranged from a low of $9 9 \%$ to $100 \%$ , and the positive panel member positivity rate ranged from a low of $9 9 \%$ to $100 \%$ depending on the site.

# Summary of Reproducibility Results

<table><tr><td rowspan=2 colspan=1>MRSACFU/swab</td><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=3>Lot to Lot</td><td rowspan=1 colspan=3>Site to Site</td><td rowspan=1 colspan=3>Day to Day</td></tr><tr><td rowspan=1 colspan=1>Lot</td><td rowspan=1 colspan=1>No./No.Tested</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>No./No.Tested</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>Day</td><td rowspan=1 colspan=1>No./No.Tested</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=5 colspan=1>0</td><td rowspan=5 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>53/54**</td><td rowspan=1 colspan=1>98%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>89/90**</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>89/90**</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=5 colspan=1>20*</td><td rowspan=5 colspan=1>BelowLOD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>52/90</td><td rowspan=1 colspan=1>58%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>38/90</td><td rowspan=1 colspan=1>42%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>25/54</td><td rowspan=1 colspan=1>46%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30/90</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>41/90</td><td rowspan=1 colspan=1>46%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>26/54</td><td rowspan=1 colspan=1>48%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>39/90</td><td rowspan=1 colspan=1>43%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>42/90</td><td rowspan=1 colspan=1>47%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>22/54</td><td rowspan=1 colspan=1>41%</td></tr><tr><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>25/54</td><td rowspan=1 colspan=1>46%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>23/54</td><td rowspan=1 colspan=1>43%</td></tr><tr><td rowspan=5 colspan=1>300</td><td rowspan=5 colspan=1>WeakPositive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>53/54</td><td rowspan=1 colspan=1>98%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>89/90</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=5 colspan=1>800</td><td rowspan=5 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>53/54**</td><td rowspan=1 colspan=1>98%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>89/90**</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>89/90**</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=6></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td></tr></table>

\* Concentration that produces approximately $30 \% - 7 0 \%$ positive results.

$^ { * * }$ There was an apparent transposition of one positive and one negative sample positioned adjacent to each other in the same run. Excluding these samples from the analysis results for lot to lot, site to site, and day to day reproducibility would be 100%.

$b$ . Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or   
methods): The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test includes an Internal Control (IC); at least one Positive Control (PC) and one Negative Control (NC, required but not included in the kit) must be processed with each run. The valid run rate was $59 5 \%$ and was acceptable. A MRSA strain representative of RE2 types is used for this external positive control. MRSA strains representing RE3 and RE7 types may be used as additional external positive controls to monitor assay primers and probes not directly controlled in the assay. External controls may be used in accordance with local, state, and federal accrediting organization, as applicable.   
d. Detection limit:

# Limit of Detection Study

The Limit of Detection (LoD) study was determined using three MRSA types (RE2, RE3, and RE7) covered by the primers and probes in the assay and for the three swab types.

Cultures of these strains were quantified, diluted to values spanning the range of 100 to 400 colony forming units (CFU) per swab, and absorbed onto swabs previously soaked into various transport media. All dilutions around the LOD value were tested in replicates of at least 30. The limit of detection obtained for each strain type and swab type tested represents the lowest number of CFU/swab at which a positive result will be obtained with at least $9 5 \%$ confidence. Results indicate that the LightCycler® MRSA Advanced Test will produce a positive result with $9 5 \%$ confidence for a swab containing 240 CFU.

Summary of Results for three swab types tested

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>MRSA types</td></tr><tr><td rowspan=1 colspan=1>Swab Types</td><td rowspan=1 colspan=1>RE2</td><td rowspan=1 colspan=1>RE3</td><td rowspan=1 colspan=1>RE7</td></tr><tr><td rowspan=1 colspan=1>Liquid Stuart transportmedia</td><td rowspan=1 colspan=1>240 cfu/swab</td><td rowspan=1 colspan=1>140 cfu/swab</td><td rowspan=1 colspan=1>240 cfu/swab</td></tr><tr><td rowspan=1 colspan=1>Amies gel without charcoaltransport media</td><td rowspan=1 colspan=1>180 cfu/swab</td><td rowspan=1 colspan=1>224 cfu/swab</td><td rowspan=1 colspan=1>240 cfu/swab</td></tr><tr><td rowspan=1 colspan=1>Amies gel with charcoaltransport media</td><td rowspan=1 colspan=1>240 cfu/swab</td><td rowspan=1 colspan=1>224 cfu/swab</td><td rowspan=1 colspan=1>240 cfu/swab</td></tr></table>

# Analytical Inclusivity

A total of 137 well characterized MRSA strains were obtained through the Network on Antimicrobial Resistance in S. aureus at a concentration of $1 0 ^ { 5 }$ $\mathrm { C F U / m L }$ .

<table><tr><td rowspan=1 colspan=1>USA type</td><td rowspan=1 colspan=1>Number ofisolatestested</td><td rowspan=1 colspan=1>Isolates with positiveresults by theLightCycler®MRSAAdvanced Test</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>54*</td><td rowspan=1 colspan=1>53</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>700</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>800</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>1000</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>1100</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Not typable/unknown</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>136 (99.3%)</td></tr></table>

$\ast _ { \mathrm { O f } }$ the 54 USA100 strains, one strain (NRS642) was negative by the LightCyclerMRSA Advanced Test

An additional 1,643 isolates were collected in Europe $( 6 5 \% )$ and in the U.S.

$( 3 5 \% )$ . There were 1,561 $( 9 5 \% )$ correctly identified as MRSA; 82 were negative by the LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test.

e. Analytical specificity:

# Cross Reactivity

The study included pathogenic microorganisms and contaminants potentially present in nasal microflora. Microorganisms tested were 13 viral, 66 bacterial and 7 fungal species at concentrations of $\overline { { 1 0 } } ^ { 6 }$ to $1 0 ^ { 7 } \mathrm { C F U / m L }$ , or as genomic DNA at concentrations $1 0 \mathrm { p g / P C R }$ . Human DNA at concentration of 5ng/reaction was also tested. Rothia mucilaginosa revealed a weak positive melting peak in the target channel 610. The results were negative for MRSA $9 8 . 9 \%$ of the organisms tested.

In addition, 100 methicillin sensitive S. aureus (MSSA), 117 methicillin resistant Coagulase negative Staph (CoNS) and 104 methicillin sensitive CoNS at concentrations $1 0 ^ { 4 } - 1 0 ^ { 5 }$ CFU/reaction were tested. Results were all negative by the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test.

Ten Staphylococcus aureus isolates characterized as Borderline Oxacillin Resistant S. aureus (BORSA) were included in the evaluation of BORSA study. They have an oxacillin MIC of $\geq 4 \mu \mathrm { g / m L }$ and were negative by an assay for the detection of PBP $2 ^ { \prime }$ of S. aureus. The BORSA isolates were tested at a concentration range of 10,000 – 100,000 CFU/lysis tube. All the results were negative.

# Interference Study

# Exogenous Interferent

The following exogenous substances, which are components of decongestants and substances used to relieve nasal dryness and/or irritation, have been shown not to interfere with the detection of MRSA by the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test. Ointments and sprays were spiked to MRSA negative and MRSA positive Liquid Stewart swabs.

Gels from the Amies gel swab container with and without charcoal were also included. No interferences observed.

Exogenous Substances Tested For Interference With The LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test   

<table><tr><td rowspan=1 colspan=1>Trade name</td><td rowspan=1 colspan=1>Active ingredient</td></tr><tr><td rowspan=1 colspan=1>InfectoPyoderm</td><td rowspan=1 colspan=1>Mupirocin 20 mg/g (antibiotic)</td></tr><tr><td rowspan=1 colspan=1>Turixin</td><td rowspan=1 colspan=1>Mupirocin calcium 21.5 mg/g (antibiotic)</td></tr><tr><td rowspan=1 colspan=1>Bepanthen® nasal ointment</td><td rowspan=1 colspan=1>Dexpanthenol 50 mg/g (pro vitamine B5)</td></tr><tr><td rowspan=1 colspan=1>Accumed 12-hour Nasal Spray</td><td rowspan=1 colspan=1>Oxymetazoline hydrochloride 0.05% (decongestant)</td></tr><tr><td rowspan=1 colspan=1>Otriven</td><td rowspan=1 colspan=1>Xylometazoline 0.1 % (decongestant)</td></tr></table>

Endogenous Interferent

Blood was found to inhibit the LightCycler $^ \mathrm { \textregistered }$ MRSA Advanced Test. The IC amplification is inhibited at the lowest blood volume tested $( 5 ~ \mu \mathrm { L } )$ while the MRSA amplification is affected at higher levels $( > 1 0 ~ \mu \mathrm { L } )$ . However, the control concept in the MRSA Advanced Test insures that inhibitory samples are invalidated when tested, thus preventing false negative results from being reported. The data also indicated that excessive amounts of nasal mucus may be inhibitory and result in an invalid test result.

# Carry over/contamination

Samples with very high concentration of MRSA (i.e. $1 0 ^ { 5 }$ , and $1 0 ^ { 6 } \mathrm { C F U / s w a b } )$ were alternated with MRSA negative samples through sample preparation, PCR preparation and amplification in the LightCycler 2.0 Instrument. Data demonstrated cross-contamination at MRSA concentration higher than $\mathrm { ( > ) } 1 0 ^ { 5 }$ CFU/swab.

# f. Assay cut-off:

Multiple LightCycler $\textsuperscript { \textregistered }$ runs on multiple instruments were analyzed to determine the average Tm for the Positive Control/Target signals in channel $( 6 1 0 \mathrm { n m } )$ and valid Internal Control signals in channel $( 6 7 0 \mathrm { n m } )$ . The baseline (assay cut-off) was determined by the analysis of negative samples. The baseline values were then confirmed in an additional study.

2. Comparison studies:

# a. Method comparison with predicate device:

A total of 1,620 nasal swab specimens were collected from subjects at five sites across the United States and tested with the LightCycler® MRSA Advanced Test.

Of the 1,620 specimen tested, 1,402 specimens were eligible to be included in statistical analyses. Performance characteristics were determined by comparing the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test with a second FDA-cleared nucleic acid amplification test (NAAT), direct culture, and broth culture.

A double-headed nasal swab was collected from each subject. One swab head was directly streaked onto a chromogenic agar plate with cefoxitin, and then processed according to the package insert for testing with the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test. The second swab head was directly streaked onto a separate chromogenic plate with cefoxitin, and then processed for testing with the second FDA-cleared NAAT test (according to the package insert). Thereafter, the second swab head was transferred into Trypticase Soy Broth and incubated for 48 hours at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ and then sub-cultured onto a chromogenic plate with cefoxitin (broth culture). Chromogenic culture plates were incubated at $3 5 { - } 3 7 ^ { \circ } \mathrm { C }$ for 20–48 hours. Presumptive MRSA colonies from all culture plates were confirmed by coagulase testing and Gram staining if found after 44-48 hours of incubation. Each participating site performed all tests.

Samples that grew MRSA on direct chromogenic culture from either swab head A and/or swab head B were considered MRSA positive. The following are the performance summary tables of the LightCycler $\textsuperscript { \textregistered }$ MRSA Advanced Test with CHROMagar, another NAAT, and enrichment culture:

Comparison of the LightCycler MRSA Advanced Test with Direct Chromogenic Culture   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="2">Direct chromogenic culturePositive            Negative          Total</td></tr><tr><td colspan="1" rowspan="1">LightCyclerMRSAAdvanced Test</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">178</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">222</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Direct chromogenic culture</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LightCycler® MRSAAdvanced Test</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1,171</td><td colspan="1" rowspan="1">1,180</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">187</td><td colspan="1" rowspan="1">1,215</td><td colspan="1" rowspan="1">1,402</td></tr><tr><td colspan="1" rowspan="1">Positive PercentAgreement (95% exact CIa)</td><td colspan="1" rowspan="1">95.2%(91.1%, 97.8%)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative PercentAgreement (95% exact CI)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">96.4%(95.2%, 97.4%)</td><td colspan="1" rowspan="1"></td></tr></table>

Note: Included in this summary are 1,402 evaluable specimens that had valid LightCycler MRSA Advanced Test and direct culture results. ${ } ^ { \mathrm { a } } \mathrm { C I } = ($ Confidence Interval

Comparison of LightCycler MRSA Advanced Test with the second FDA cleared NAAT   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=2 colspan=3>The Second FDA-Cleared NAATPositive           Negative         Total</td></tr><tr><td rowspan=1 colspan=1>LightCycler® MRSAAdvanced Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>215</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>1,093</td><td rowspan=1 colspan=1>1,170</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>1,113</td><td rowspan=1 colspan=1>1,385</td></tr><tr><td rowspan=1 colspan=1>Positive PercentAgreement (95% exact CI)</td><td rowspan=1 colspan=1>71.7%(65.9%, 77.0%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative PercentAgreement (95% exact CI)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>98.2%(97.2%, 98.9%)</td><td rowspan=1 colspan=1></td></tr></table>

Note: Included in this summary are 1,385 specimen with valid results for the MRSA Advanced Test and second FDA-cleared NAAT test which had additional concordant direct culture results from swab heads A and B ${ } ^ { \mathrm { a } } { \mathrm { C I } } =$ Confidence Interval

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Broth Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>LightCycler® MRSAAdvanced Test</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>222</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1,152</td><td rowspan=1 colspan=1>1,173</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>1,190</td><td rowspan=1 colspan=1>1,395</td></tr><tr><td rowspan=1 colspan=1>Positive PercentAgreement (95% exact CI)</td><td rowspan=1 colspan=1>89.8%(84.8%- 93.5%)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative PercentAgreement (95% exact CI)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>96.8%(95.6%, 97.7%)</td><td rowspan=1 colspan=1></td></tr></table>

Note: A total of 1395/1402 evaluable specimens that had valid LightCycler MRSA Advanced Test and enrichment culture results are included in this summary table. Enrichment culture results were missing /invalid for 7/1402 evaluable specimens.   
${ } ^ { \mathrm { a } } { \mathrm { C I } } = \scriptstyle ($ Confidence Interval

$b$ . Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable   
b. Clinical specificity: Not applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The overall number of positive MRSA samples by direct culture was 187 $( 1 3 . 3 \% )$ . The study population in the clinical trial was grouped into subjects in hospital non-intensive care units (1,007; $7 1 . 8 \%$ ), hospital intensive care units (67; $4 . 8 \%$ ) nursing homes/extended care facilities (255; $1 8 . 2 \% )$ , and medical facility staff (73; $5 . 2 \% )$ .

N. Instrument Name: Roche Molecular Systems LightCycler 2.0

# O. System Descriptions:

1. Modes of Operation:

The instrument operates in batch mode with samples in glass capillaries covered by plastic stoppers.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X____ or No

The LightCycler 2.0 instrument is operated through a separate PC connected by cable. The LightCycler software operates the instrument through the PC. The complete LightCycler Version 4.1 software documentation was reviewed in this submission as we consider the 2.0 a new instrument.

The MRSA assay uses an assay specific software module (MAS) to interact with LC 4.1 software and to calculate results. The complete MAS version 1.2 software documentation was reviewed in this submission.

In addition, the LightCycler 2.0 instrument falls under 862.2570 Instrumentation for clinical multiplex test systems regulation and its special controls. The appropriate product code from this regulation is OOI Real Time Nucleic Acid Amplification System.

The LightCycler 2.0 instrument also has RUO functionalities that can be used by end users to develop their own assays. Therefore, we applied the recently devised Combination RUO/IVD Instrumentation Policy. The company supplied all required software documentation and the IVD only user manual for review.

3. Specimen Identification:

Specimens are identified by their position in the sample carousel.

4. Specimen Sampling and Handling: Specimen preparation is performed off-line, pipetted into glass capillaries, and manually loaded into a sample carousel for addition to the LightCycler 2.0.

5. Calibration: No calibration needed.

6. Quality Control:

The use of quality control material is described in each specific assay package insert that uses the LightCycler 2.0 instrument.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.