# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K091109   
B. Purpose for Submission: Substantial equivalence for a new device.   
C. Measurand: C. difficile toxin B gene sequences   
D. Type of Test: Nucleic Acid Amplification Test   
E. Applicant: Cepheid   
F. Proprietary and Established Names: Xpert $^ \mathrm { \textregistered }$ C. difficile

# G. Regulatory Information:

1. Regulation section: 866.2660

2. Classification: I

3. Product code: LLH

4. Panel: 83 Microbiology

# H. Intended Use:

1. Intended use(s):

The Cepheid Xpert $^ \mathrm { \textregistered }$ C. difficile Assay, performed on the Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System, is a qualitative in vitro diagnostic test for rapid detection of toxin B gene sequences from unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile infection (CDI). The test utilizes automated real-time polymerase chain reaction (PCR) to detect toxin gene sequences associated with toxin producing C. difficile. The Xpert C. difficile Assay is intended as an aid in the diagnosis of CDI. Concomitant testing is necessary only if further typing or organism recovery is required.

# 2. Indication(s) for use

The Cepheid Xper $^ \mathrm { \textregistered }$ C. difficile Assay, performed on the Cepheid GeneXpert $^ \mathrm { { \textregistered } }$ Dx System, is a qualitative in vitro diagnostic test for rapid detection of toxin B gene sequences from unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile infection (CDI). The test utilizes automated real-time polymerase chain reaction (PCR) to detect toxin gene sequences associated with toxin producing C. difficile. The Xpert C. difficile Assay is intended as an aid in the diagnosis of CDI. Concomitant testing is necessary only if further typing or organism recovery is required.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System

# I. Device Description:

The GeneXpert Dx System automates and integrates sample purification, nucleic acid amplification, and detection of the target sequence in simple or complex samples using real-time PCR and RT-PCR assays. The system consists of an instrument, personal computer, and preloaded software for running tests and viewing the results. The system requires the use of single-use disposable cartridges that hold the PCR reagents and host the PCR process. Because the cartridges are self-contained, cross-contamination between samples is eliminated. For a full description of the system, see the GeneXpert Dx System Operator Manual.

The Xpert C. difficile Assay includes reagents for the detection of toxigenic C. difficile, as well as a Sample Processing Control (SPC). The SPC is present to control for adequate processing of the target bacteria and to monitor the presence of inhibitors in the PCR

reaction. The Probe Check Control (PCC) verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD GeneOhm™ Cdiff Assay

2. Predicate 510(k) number(s): K081920

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">cYanmaGnYellob</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Xpert C. difficile Assay</td><td colspan="1" rowspan="1">BD GeneOhm Cdiff Assay(K081920)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">An automated test for thequalitative detection oftoxigenic C. difficile inunformed (liquid or soft) stoolspecimens.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Indication for Use</td><td colspan="1" rowspan="1">Identification of C. difficilefrom patients suspected ofhaving C. difficile Infection(CDI).</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Fully-automated nucleic acidamplification (DNA); real-timePCR</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Unformed (liquid or soft) Stool</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge</td><td colspan="1" rowspan="1">Disposable single-use, multi-chambered fluidic cartridge.</td><td colspan="1" rowspan="1">Disposable single-use PCRtube</td></tr><tr><td colspan="1" rowspan="1">DNA TargetSequence(s)</td><td colspan="1" rowspan="1">C. difficile toxin B and binarytoxin gene sequences and thesingle base pair deletion atnucleotide 117 in tcdC(tcdCΔ117).</td><td colspan="1" rowspan="1">C. difficile toxin B only</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Cepheid GeneXpert Dx System</td><td colspan="1" rowspan="1">Cepheid SmartCycler DxSystem</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction</td><td colspan="1" rowspan="1">Self-contained and automatedafter swab elution and twosingle-dose reagent additions.</td><td colspan="1" rowspan="1">Manual</td></tr><tr><td colspan="1" rowspan="1">Probes</td><td colspan="1" rowspan="1">TaqMan® Probes</td><td colspan="1" rowspan="1">Molecular Beacons</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Manual</td></tr><tr><td colspan="1" rowspan="1">Rapid test results</td><td colspan="1" rowspan="1">Less than 45 minutes to results.</td><td colspan="1" rowspan="1">Approximately 75-90 minutesto results.</td></tr><tr><td>Users</td><td>Operators with no clinical lab experience to experienced clinical laboratory technologists.</td><td>CLIA High Complexity Laboratory Users</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

# L. Test Principle:

The assay is a rapid, automated in vitro diagnostic test for the qualitative detection of $C$ . difficile directly from unformed (liquid or soft) stool specimens of patients suspected of having C. difficile infection (CDI). The assay detects the following three elements: toxin B gene (tcdB), binary toxin gene (CDT), and the single-base-pair deletion at nucleotide 117 within the gene encoding a negative regulator of toxin production (tcdCΔ117). The assay is performed on the Cepheid GeneXpert Dx System. This is the same instrument system that is used with the Cepheid Xpert MRSA [510(k) #K070462], Xpert MRSA/SA SSTI [510(k) #K080837], and Xpert MRSA Blood Culture [510(k) #K082140] as well as the Cepheid Xpert GBS and Xpert EV Assays.

The assay uses real-time PCR amplification and fluorogenic target-specific hybridization detection. An important aspect of this submission is that the assay intends to detect toxigenic C. difficile and presumptive identification of an epidemic strain of C. difficile known as 027/NAP1/BI strain (Note: this strain designation terminology refers to designations given to this strain based on PCR ribotyping, Pulsed Field Gel Electrophoresis (PFGE) and restriction endonuclease analysis (REA), respectively).

This strain has been described as having a single-base-pair deletion at nucleotide 117 within the gene encoding a negative regulator of toxin production (tcdCΔ117). Cepheid selected primers and probes to identify the single-base- pair deletion at position 117 of the tcdC gene to detect and differentiate the 027/NAP1/BI hypervirulent C. difficile strains from other isolates.

# M. Performance Characteristics (if/when applicable):

Performance characteristics of the Xpert C. difficile Assay were determined in a multi-site prospective investigation study at seven US and Canadian institutions by comparing the Xpert C. difficile Assay to reference culture followed by cell cytotoxicity testing on the isolates

Subjects included individuals whose routine care called for $C .$ . difficile testing. A portion of the leftover unformed stool specimens were obtained for testing by the Xpert $C .$ . difficile Assay. The remaining excess specimen was sent to a central laboratory for reference culture and cytotoxin B isolate testing. Each stool specimen was inoculated onto pre-reduced cycloserine-cefoxitin-fructose agar –direct plate (CCFA-D) and cycloserine cefoxitin mannitol broth with taurocholate lysozyme cysteine (CCMB-TAL). After 24 hours the

CCMBTAL was subcultured on to a second CCFA-E plate (CCFA- Enriched). This directenriched culture method is referred to hereafter as “reference culture”. If C. difficile was isolated from the CCFA-D plate and the isolate was positive by cell cyotoxicity assay, the specimen was classified as “toxigenic C. difficile positive” and CCFA-E plate was not further analyzed. If no C. difficile was isolated from the CCFA-D plate or if the isolate was negative by cell cytotoxicity assay, the CCFA-E plate was further analyzed. If CCFA-E was positive for $C .$ . difficile and the isolate was positive for cell cytotoxicity assay, the specimen was classified as “toxigenic C. difficile positive”. The specimen was reported as “negative” if CCFA-E is negative for C. difficile or the isolate was tested negative by cell cytotoxicity assay.

Performance of the Xpert C. difficile Assay was calculated relative to the results of direct culture and reference culture.

# Overall Results

A total of 2296 specimens were tested by Xpert C. difficile Assay and culture.

Performance vs. Direct Culture

Relative to direct culture with, the Xpert $C .$ . difficile Assay demonstrated a sensitivity and specificity for toxigenic C. difficile of $9 8 . 7 9 \%$ and $9 0 . 8 2 \%$ , respectively.

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=3>DirectCulture</td></tr><tr><td rowspan=1 colspan=1>C. diff</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Xpert C.difficile</td><td rowspan=1 colspan=1>Toxin B+</td><td rowspan=1 colspan=1>245 (240)</td><td rowspan=1 colspan=1>188 (183)</td><td rowspan=1 colspan=1>433 (423)</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3 (3)</td><td rowspan=1 colspan=1>1860 (1795)</td><td rowspan=1 colspan=1>1863(1798)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>248 (243)</td><td rowspan=1 colspan=1>2048 (1978)</td><td rowspan=1 colspan=1>2296(2221)</td></tr><tr><td rowspan=1 colspan=5>Sensitivity:            98.79%Specificity:            90.82%Accuracy:              91.68%PPVa:                   56.58%NPVb:                 99.83%Prevalence:            10.80%</td></tr></table>

# Antibiotic Usage

Among the 2296 cases included in the main dataset, antibiotic use within the 2 months prior to sample collection was reported for 1633 and no antibiotic use was confirmed for 570; for 93 cases, antibiotic status was unknown. Antibiotic use did not cause a statistically significant difference in assay performance

# Analytical Specificity

Fifty-five (55) strains were collected, quantitated and tested using the Xpert C. difficile Assay. The strains originated from the American Type Culture Collection (ATCC), Culture Collection University of Göteborg (CCUG), German Collection of Microorganisms and Cell Cultures (DSMZ), the Centers for Disease Control and Prevention (CDC), the Institute of Public Health, Maribor, Slovenia and Swedish Institute for Infectious Disease Control (SMI).

Of the tested species, ten (10) non-toxigenic C. difficile strains and eleven (11) non $C .$ . difficile Clostridium species were included. The organisms tested were identified as either Gram-positive (37) or Gram negative (18). The organisms were further classified as aerobic (24), anaerobic (29) or microaerophilic (2).

Each strain was tested in triplicate at concentrations ranging from $1 . 1 \times { 1 0 } ^ { 8 }$ to $2 . 2 \times { 1 0 } ^ { 1 0 }$ CFU/swab. Positive and negative controls were included in the study. Under the conditions of the study, all isolates were reported “Toxigenic C. diff NEGATIVE”. The analytical specificity was $100 \%$ .

# Analytical Sensitivity

Studies were consisted of human liquid feces (C. difficile negative by Xpert C. difficile Assay) diluted in PBS with $1 5 \%$ glycerol. The LoD is defined as the lowest number of colony forming units (CFU) per swab that can be reproducibly

Studies were performed to determine the $9 5 \%$ confidence intervals for the analytical limit of detection (LoD) of $C .$ . difficile diluted into a fecal matrix of human origin that can be detected by the Xpert $C .$ . difficile Assay. The fecal matrix consisted of human liquid feces (C. difficile negative by Xpert C. difficile Assay) diluted in PBS with $1 5 \%$ glycerol. The LoD is defined as the lowest number of colony forming units (CFU) per swab that can be reproducibly   
distinguished from negative samples with $9 5 \%$ confidence.   
Replicates of 20 were evaluated at each $C .$ . difficile concentration tested (CFU/swab) for 7 different C. difficile strains representing toxinotypes 0 (two strains), III (two strains), IV, V and VIII (one of each strain).

The estimate and confidence intervals were determined using logistic regression with data (number of positive results per number of replicates at each level) over the range of CFUs tested. The confidence intervals were determined using maximum likelihood estimates on the logistic model parameters using the large sample variance-covariance matrix. The LoD point estimates and $9 5 \%$ upper and lower confidence intervals for each $C .$ difficile toxinotype tested are summarized in Table 4.

$9 5 \%$ Confidence Intervals for Analytical LoD – C. difficile   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>LoD95%(CFU/swab)</td><td rowspan=1 colspan=1>Lower 95%CI</td><td rowspan=1 colspan=1>Upper95% CI</td></tr><tr><td rowspan=1 colspan=1>VPI 10463(CCUG19126)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>90556-M6S(ATCC9689)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>419</td><td rowspan=1 colspan=1>587</td></tr><tr><td rowspan=1 colspan=1>LUMC-1(027/NAP1/BI)a</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>LUMC-5(027/NAP1/BI)a</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=1>LUMC-7</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=1>LUMC-6</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>9101</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>49</td></tr></table>

The results of this study indicate that the Xpert $C .$ . difficile Assay will produce a positive $C .$ .   
difficile result $9 5 \%$ of the time with $9 5 \%$ confidence for a fecal sample containing 460 CFU.

In addition to the LoD determination, eighteen C. difficile strains representing 12 variant toxinotypes, including four 027/NAP1/BI toxinotype III isolates, were tested using the Xpert $C .$ . difficile Assay. C. difficile strains were selected to broadly represent the majority of $C .$ difficile toxinotypes encountered in practice. Stock cultures were prepared by suspending the bacterial growth from agar plates in PBS buffer containing $1 5 \%$ glycerol. The concentration of each stock was adjusted to 1.4-5.9 McFarland units. All strains were serially diluted to approximately 900 CFU/swab and tested in triplicate.

Under the conditions of this study, the Xpert C. difficile Assay correctly identified all 18 toxinotypes tested as “Toxigenic C. diff POSITIVE”. Included in the panel were 8 toxinotypes reported to be positive for binary toxin (CDT) production as well. All were CDT positive using the Xpert C. difficile Assay. All four 027/NAP1/BI isolates representing toxinotype III were correctly identified as “Toxigenic C. diff POSITIVE”.

# Interfering Substances

Twenty-one (21) biological and chemical substances occasionally used or found in stool specimens were tested for interference with the Xpert C. difficile Assay. Potentially interfering substances include, but are not limited to, Vagisil cream and zinc oxide paste. The 19 substances listed in Table 5 showed no detectable interference with the Xpert C. difficile Assay

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Substance</td></tr><tr><td rowspan=1 colspan=1>Whole Blood KarolinskaUniversity Hospital</td><td rowspan=1 colspan=1>K-Y Jelly/Gelée® McNeil-PPC</td></tr><tr><td rowspan=1 colspan=1>Mucin (porcine) Sigma</td><td rowspan=1 colspan=1>Vaseline Unilever</td></tr><tr><td rowspan=1 colspan=1>Kaopectate® Chattem</td><td rowspan=1 colspan=1>Dulcolax® Boehringer IngelheimPharmaceuticals</td></tr><tr><td rowspan=1 colspan=1>Imodium® McNeil-PPC</td><td rowspan=1 colspan=1>Preparation H Portable Wipes WyethConsumer Healthcare</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol® Procter &amp;Gamble</td><td rowspan=1 colspan=1>Vaginal Contraceptive Film (VCF)Apothecus Pharmaceutical</td></tr><tr><td rowspan=1 colspan=1>Preparation H® WyethConsumer Healthcare</td><td rowspan=1 colspan=1>Vancomycin Fluka</td></tr><tr><td rowspan=1 colspan=1>Fleet® CB Fleet Company</td><td rowspan=1 colspan=1>Metronidazole Actavis</td></tr><tr><td rowspan=1 colspan=1>Fecal fats KarolinskaUniversity Hospital</td><td rowspan=1 colspan=1>Anusol® Plus TM Warner-LambertCompany</td></tr><tr><td rowspan=1 colspan=1>Monistat® McNeil-PPC</td><td rowspan=1 colspan=1>E-Z-HDTM High Density BariumSulfate for suspension E-Z-EM Canada</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone Cream LongsDrugs</td><td rowspan=1 colspan=1></td></tr></table>

# Reproducibility

A panel of 7 specimens with varying concentrations of toxigenic $C .$ . difficile and C. difficile, 027/NAP1/BI were tested on 10 different days by two different operators at each of the three sites (7 specimens x 2 operators/ day $\textbf { \scriptsize x } 1 0$ days $_ { \textrm { X 3 } }$ sites). One lot of Xpert $C .$ . difficile Assay was used at each of the 3 testing sites. Xpert $C .$ difficile Assays were performed according to the Xpert C. difficile Assay procedure. Results are summarized below.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">%Agreement</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen ID</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">% Total Agreement bySample</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. difficileHigh Negative</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. difficileLow Positive</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">85% (17/20)</td><td colspan="1" rowspan="1">85% (17/20)</td><td colspan="1" rowspan="1">90.0% (54/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. difficileModerate Positive</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BI HighNegative</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BI LowPositive</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">96.7% (58/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BIModerate Positive</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">% Total Agreementby Site</td><td colspan="1" rowspan="1">100%(140/140)</td><td colspan="1" rowspan="1">97.1%(136/140)</td><td colspan="1" rowspan="1">97.1%(136/140)</td><td colspan="1" rowspan="1">98.1% (412/420)</td></tr></table>

<table><tr><td rowspan=1 colspan=4>SPC</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Ave.</td><td rowspan=1 colspan=1>Std. Dev.</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff highneg</td><td rowspan=1 colspan=1>32.17</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.83%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff lowpos</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.66%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff modpos</td><td rowspan=1 colspan=1>31.98</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.47%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI high neg</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.03%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI low pos</td><td rowspan=1 colspan=1>31.93</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.26%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI mod pos</td><td rowspan=1 colspan=1>31.96</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.90%</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>32.26</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.22%</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4>tcdB</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Ave.</td><td rowspan=1 colspan=1>Std. Dev.</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff highneg</td><td rowspan=1 colspan=1>39.59</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.77%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff lowpos</td><td rowspan=1 colspan=1>35.88</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.24%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. diff modpos</td><td rowspan=1 colspan=1>32.17</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.39%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI high neg</td><td rowspan=1 colspan=1>39.11</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>2.50%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI low pos</td><td rowspan=1 colspan=1>35.49</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.65%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI mod pos</td><td rowspan=1 colspan=1>32.10</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.97%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>%Agreement</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specimen ID</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>% Total Agreementby Sample</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>100% (90/90)</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficileHigh Negative</td><td rowspan=1 colspan=1>60.0%(18/30)</td><td rowspan=1 colspan=1>60.0%(18/30)</td><td rowspan=1 colspan=1>53.3%(16/30)</td><td rowspan=1 colspan=1>57.8%(52/90)</td></tr></table>

# N. Instrument Name:

Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System

# O. System Descriptions:

1. Modes of Operation: Automated

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No _

3. Specimen Identification:

Unformed (liquid or soft) stool specimens collected from patients suspected of having Clostridium difficile infection (CDI).

4. Specimen Sampling and Handling:

(1) Collect the unformed stool specimen in a clean container. Follow your institution’s   
guidelines for collecting samples for $C .$ difficile testing.   
(2) Label with Sample ID and send to the laboratory.   
(3) Store specimen at $2 - 8 \mathrm { { } ^ { \circ } C }$ . The specimen is stable for up to 5 days when stored at $2 - 8 \ ^ { \circ } \mathrm { C }$ . Alternatively, specimens can be kept at room temperature $( 2 0 - 3 0 \ ^ { \circ } \mathrm { C } )$ for up to 24 hours.

5. Calibration:

NA

6. Quality Control: Each sample contains a Sample Processing Control and a Probe Check Control

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

N/A

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.