# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K092566

B. Purpose for Submission: New System comprising of a reader unit and a Processor (SP) containing a hybridization module with modules for sample extraction and target amplification added.

C. Measurand: Influenza virus and Respiratory Syncytial Virus (RSV) nucleic acids target sequences. Influenza A, Influenza B, and RSV are detected.

D. Type of Test: Multiplex nucleic acid assay for qualitative detection and identification of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) target sequences in nasopharyngeal swab specimens including nucleic acid isolation, multiplex RTPCR amplification, capture of the amplicons hybridized to gold-labeled probes on a microarray-based chip, and detection of the signal enhanced with elemental silver using the Verigene® System

E. Applicant: Nanosphere, Inc.

F. Proprietary and Established Names:

For instrument:   
Verigene® SP System   
For the assay:   
Verigene® Respiratory Virus Nucleic Acid Test on the Verigene $^ \mathrm { \textregistered }$ SP System $( \mathrm { R V N A T } _ { S P } )$ )

G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>OCC and NSU</td><td>Class II</td><td>21CFR 866.3980 Respiratory viral panel multiplex nucleic acid assay; 21 CFR 862.2570 Instrumentation for clinical multiplex test systems</td><td>Microbiology (83)</td></tr></table>

H. Intended Use: Same as K083088 described below:

The Verigene® Respiratory Virus Nucleic Acid Test is a qualitative multiplex in vitro diagnostic test for the detection and identification of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids purified from nasopharyngeal swab specimens obtained from patients symptomatic for viral upper respiratory infection. The test is intended to be used on the Verigene® System as an aid in the differential diagnosis of Influenza A, Influenza B, and RSV infections. The test is not intended to detect influenza C virus.

Negative results do not preclude influenza virus or RSV infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative test results be confirmed by culture.

Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infections with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3) Special conditions for use statement(s): For Prescription use only

4) Special instrument requirements: Verigene® System consisting of the Verigene Processor (software version 2.0) and the Verigene Reader (software version 1.5.6).

# I. Device Description:

The Verigene $S P$ consists of two core instruments, the Verigene Reader and the Verigene $S P$ that offers ‘sample-to-result’ testing capability. The Verigene $S P$ System utilizes single-use disposable test units. Up to $3 2 \ : S P$ units may be connected to a single Verigene Reader. Each $S P$ unit can be independently accessed through the Reader to run different tests. The Verigene Reader in the Verigene $S P$ is the same as in the cleared system (K083088).

The $S P$ unit is a reconfiguration of the Processor within the cleared Verigene System to allow on-board Sample Extraction and Target Amplification – steps that were previously performed externally. Added in the $S P ' s$ package are an Extraction Module to extract nucleic acids from clinical samples and an Amplification Module to amplify the extracted nucleic acids. The amplified target is then detected in the Verigene Test as in the cleared Verigene System. All fluid transfers and mixing operations in the $S P$ are performed by an automated pipettor.

Assay technology is same as described below in K083088: Magnetic bead-based isolation technology for sample preparation, multiplex RT-PCR for target amplification, gold nanoparticle probe technology for the Verigene test. The VRNAT also includes control materials as described in K083088.

The Verigene® Respiratory Virus Nucleic Acid Test (VRNAT) is a set of reagents and supplies designed to be utilized in RT-PCR and hybridization assay using the Verigene® instrument SP system for detection and identification of human influenza A, influenza B, and RSV viral RNA. The VRNAT primers target matrix (M) gene of influenza A, non structural (NS) and M genes of influenza B, and L gene and F gene of RSV. The amplicons are hybridized to gold nanoparticle probes through a mediator oligonucleotide and target-specific capture oligonucleotides on a microarray-based chip in a disposable test cartridge. The signal enhanced with elemental silver at the test site is detected using the Verigene Processor.

J. Substantial Equivalence Information:

a) Predicate device name (s): Verigene® Respiratory Virus Nucleic Acid Test on the Verigene System (VRNAT)   
b) Predicate Numbers (s): K083088

Comparison with predicate:   

<table><tr><td colspan="1" rowspan="1">Verigene® (Cleared System) K083088</td><td colspan="1" rowspan="1">Verigene® SP (New System) K092566</td></tr><tr><td colspan="2" rowspan="1">DEVICE</td></tr><tr><td colspan="1" rowspan="1">Processor</td><td colspan="1" rowspan="1">Processor SP</td></tr><tr><td colspan="1" rowspan="1">1) Sample Extraction is performedexternally on the NucliSENS EasyMAG(bioMerieux)</td><td colspan="1" rowspan="1">1) Sample Extraction is performed within the addedExtraction Module using Roche reagents. NOTE:Roche licenses extraction technology frombioMerieux</td></tr><tr><td colspan="1" rowspan="1"> Target Amplification by RT-PCR isperformed on an external thermocylcer</td><td colspan="1" rowspan="1">2) Target Amplification by RT-PCR is performedwithin the added Amplification Module using samereagents and procedure</td></tr><tr><td colspan="1" rowspan="1">3) Verigene Test (hybridization) isperformed in the hybridization modulehoused in the Processor of the Verigene®System</td><td colspan="1" rowspan="1">3) No change in reagent or procedure</td></tr><tr><td colspan="1" rowspan="1">4) Pipetting between the three steps isperformed by the user.</td><td colspan="1" rowspan="1">4) Pipetting module added to automate fluid transfersteps</td></tr><tr><td colspan="1" rowspan="1">Reader  provides the user interface,controls the Processor, performs imageanalysis, and provides results.</td><td colspan="1" rowspan="1">Reader  No change</td></tr><tr><td colspan="1" rowspan="1">Verigene® System is built under QSR</td><td colspan="1" rowspan="1">Verigene® SP is built under QSR</td></tr><tr><td colspan="2" rowspan="1">SOFTWARE</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Software — no change in the architecture. Softwareprogramming to control the Extraction andAmplification Modules has been added.</td></tr><tr><td colspan="2" rowspan="1">ASSAY PARAMETERS</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction  parameters as on theNucliSENS EasyMAG (bioMerieux)</td><td colspan="1" rowspan="1">Sample Extraction  parameters are similar to theNucliSENS EasyMAG (bioMerieux)</td></tr><tr><td colspan="1" rowspan="1">Target Amplification - parameters formultiplex RT-PCR were developed towork on thermocyclers.</td><td colspan="1" rowspan="1">RT-PCR - no change</td></tr><tr><td>Verigene Test - for the detection of amplified DNA targets on the Verigene System</td><td>No change</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005. http://www.fda.gov/cdrh/ode/guidance/337.pdf

2. Draft Guidance for Industry and FDA Staff: Assay Migration Studies for In Vitro Diagnostic Devices - http://www.fda.gov/OHRMS/DOCKETS/98fr/FDA-2008-D-0642-gdl.pdf

3. Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses - http://www.fda.gov/cdrh/oivd/guidance/1638.pdf

L. Test Principle: Same as K083088

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# Limit of Detection (Analytical Sensitivity)

The analytical sensitivity of the RVNATSP was compared to the cleared VRNAT (K083088) by determining the Limit of Detection (LoD) of Influenza A, Influenza B, RSV A, and RSV B viruses. Strains with established titers were used for each virus. Each virus stock was serially diluted into a sample matrix (Universal Transport Media, Copan), and each concentration was tested in quadruplicate using the RVNATSP. The LoD was confirmed by performing an additional 20 replicates for each strain in order to demonstrate that the virus was detected $2 9 5 \%$ of the time. The LoD for each virus was identical to the LoD observed with the same strains on the cleared VRNAT (K083088)

Limit of Detection

<table><tr><td rowspan=1 colspan=1>Limits of Detection</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Influenza A (A/Wisconsin/67/05)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Florida/04/2006)</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV A (Strain Long)</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>RSV   B   (B-1   Wild   Type   (BWV/14617/85))</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr></table>

Carry-over contamination studies: High positive samples of Influenza A, Influenza B, RSV A, and RSV B were alternated with high negative samples for all four viruses. Based on the collective data, there was no evidence of cross-contamination from any of the test steps including sample preparation, multiplex RT-PCR step, and the Verigene Hybridization Test.

A. Precision/Reproducibility Studies Comparison between the RVNAT $\dot { s } _ { P }$ and the cleared VRNAT (K083088)

The Precision/Reproducibility Studies were performed at each of three sites. At Site 1, the reproducibility study was part of a larger precision study.

Precision/Reproducibility Studies for the $\mathrm { R V N A T } _ { S P }$ were conducted exactly as for the previously cleared VRNAT (K083088) to allow equivalency comparisons. As before, eight unique samples were created by diluting known concentrations of viral particles with Viral Transport Medium. Since the Analytical Sensitivity of the RVNATSP was identical to the cleared VRNAT, the same strains and levels were used in the studies. Each strain was represented at 3 distinct concentrations: high negative, low positive, and moderate positive.

Sample panel for the Precision/Reproducibility Studies.

<table><tr><td rowspan=1 colspan=1>UniqueSamples</td><td rowspan=1 colspan=1>Viral Strains and Levels</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Influenza A - High Negative; Influenza B - High Negative</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>RSV A - High Negative; RSV B - High Negative</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Influenza A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>RSV A - Low Positive</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>RSV B - Low Positive</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Influenza A - Moderate Positive; RSV A - Moderate Positive</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Influenza B - Moderate Positive; RSV B - Moderate Positive</td></tr></table>

The Precision Study (Site 1) tested the sample set over 12 non-consecutive days. On each test day, two operators performed the RVNAT $\dot { s } _ { P }$ in duplicate for each sample (i.e., 4 sample sets per day total). In the reproducibility study performed by sites 2 and 3, the sample set was tested in triplicate daily by 2 operators on each of five nonconsecutive days.

Performance characteristics of the $\mathrm { R V N A T } _ { S P }$ in the Precision/Reproducibility Studies were equivalent to those for the cleared VRNAT as shown in the following table.

Comparison of Precision/Reproducibility data from the new RVNATSP to the cleared VRNAT

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=12>Agreement of &#x27;Observed Results&#x27; to &#x27;Expected Results&#x27;</td></tr><tr><td rowspan=1 colspan=6>New RVNAT sp</td><td rowspan=1 colspan=6>Cleared VRNAT</td></tr><tr><td rowspan=1 colspan=2>Panel Member</td><td rowspan=1 colspan=1>Site 1NSPH</td><td rowspan=1 colspan=1>Site 2MPLN</td><td rowspan=1 colspan=1>Site 3MCW</td><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95% Score CI</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95% Score CI</td></tr><tr><td rowspan=3 colspan=1>h tzunaA</td><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>75/78</td><td rowspan=1 colspan=1>96%</td><td rowspan=1 colspan=1>89.3 - 98.7%</td></tr><tr><td rowspan=1 colspan=1>LowPo sitive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr><tr><td rowspan=3 colspan=1>gtzne</td><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr><tr><td rowspan=3 colspan=1>A FSA•</td><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>76/78</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>91.1 - 99.3%</td></tr><tr><td rowspan=1 colspan=1>LowPossitive</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1 - 99.8%</td></tr><tr><td rowspan=1 colspan=1>Moderatesitive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr><tr><td rowspan=3 colspan=1>SFS</td><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>14/15</td><td rowspan=1 colspan=1>74/78</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>87.5 - 98.0%</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>95.3 - 100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>78/78</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>95.3 - 100%</td></tr></table>

B. Method Comparison Studies between RVNAT $\dot { s } { } _ { P }$ and the cleared VRNAT (K083088)

A sample set representing Influenza A, Influenza B, and RSV was prepared by diluting culture positive nasopharyngeal swab samples with negative samples. Dilutions were aimed to yield viral load levels close to the low positive levels for each virus type. A total of 62 unique samples were diluted, aliquoted, and frozen. The sample set was tested at the internal site (Site 1) using the cleared VRNAT,and at all three sites using the RVNATSP. This sample testing yielded a total of $6 2 \times 4 { = } 2 4 8$ unique tests.

Each sample set yielded a total of 186 decisions from 62 unique samples as each test provides a decision of ‘Detected’ or ‘Not Detected’ for each of the 3 viruses, Influenza A, Influenza B, and RSV. Of the 62 samples, 3 samples had dual infections where 2 viruses were present.

Study Sample Set   

<table><tr><td rowspan=1 colspan=1>Sample Set</td><td rowspan=1 colspan=1>Positives</td><td rowspan=1 colspan=1>Negatives</td><td rowspan=1 colspan=1>Totals</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>RSV A/B</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65*</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>186</td></tr></table>

3 samples had 2 viruses raising the total number of positives from 62 to 65

Decisions on the $\mathrm { R V N A T } _ { S P }$ for each site were compared to the cleared VRNAT. The data is shown in the tables below:

$\mathrm { R V N A T } _ { S P }$ Method Comparison Data collected at Site 1   

<table><tr><td rowspan=2 colspan=2>All Viruses</td><td rowspan=1 colspan=2>VRNAT(Old System)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>RVNATSP(New System)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>PPA 100.0 %(95%CI=94.4% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>NPA 100.0%(95%CI=96.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1></td></tr></table>

$\mathrm { R V N A T } _ { S P }$ Method Comparison Data collected at Site 2   

<table><tr><td rowspan=2 colspan=2>All Viruses</td><td rowspan=1 colspan=2>VRNAT (Old System)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>cYanmaGellowb</td></tr><tr><td rowspan=2 colspan=1>RVNATSP(New System)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>PPA 98.5 % (95%CI=91.8%-99.7%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>122</td><td rowspan=1 colspan=1>NPA 100.0%(95%CI=96.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1></td></tr></table>

a Low positive discordant for Influenza A. Repeat tests were positive and gave the expected result.

RVNATSP Method Comparison Data collected at Site 3   

<table><tr><td rowspan=2 colspan=2>All Viruses</td><td rowspan=1 colspan=2>VRNAT (Old System)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>RVNATSP(New System)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>PPA 95.4 % (95%CI=87.3%-98.4%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>NPA 100.0%(95%CI=96.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>186</td><td rowspan=1 colspan=1></td></tr></table>

aAll discordant samples were Influenza A low positive. Repeat tests were positive and gave the expected result.

RVNATSP Method Comparison Data – Combined data from all 3 sites   

<table><tr><td rowspan=2 colspan=2>All Viruses/All Sites</td><td rowspan=1 colspan=2>VRNAT (Old System)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>RVNATSP(New System)</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>191</td><td rowspan=1 colspan=1>PPA 97.9 %(95%CI=94.8% - 99.0%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>367</td><td rowspan=1 colspan=1>NPA 100.0%(95%CI=99.0% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>195</td><td rowspan=1 colspan=1>363</td><td rowspan=1 colspan=1>558</td><td rowspan=1 colspan=1></td></tr></table>

aAll discordant samples were Influenza A low positive. Repeat tests were positive and gave the expected result.

N. Proposed Labeling: The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

O. Conclusion: The submitted information in this premarket notification is complete and supports a substantial equivalence decision.