# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

# C. Measurand:

# E. Applicant:

F. Proprietary and Established Names: Xpert $^ \mathrm { \textregistered }$ vanA Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NIJ OOI

4. Panel: 83 Microbiology

# H. Intended Use:

1. Intended use(s):

The Cepheid Xpert $^ \mathrm { \textregistered }$ vanA Assay performed in the GeneXpert $^ \mathrm { \textregistered }$ Dx System is a qualitative in vitro diagnostic test designed for rapid detection of the vanA gene sequence associated with vancomycin resistance in bacteria obtained from rectal swab specimens from patients at risk for intestinal colonization with vancomycin-resistant bacteria. The test utilizes automated real-time polymerase chain reaction (PCR) to detect the vanA gene that is frequently associated with vancomycin-resistant enterococci (VRE). The Xpert vanA Assay is intended to aid in the recognition, prevention, and control of vancomycin resistant organisms that colonize patients in healthcare settings. The Xpert vanA Assay is not intended to diagnose infections caused by vancomycin-resistant bacteria nor to guide or monitor treatment for vancomycin-resistant bacterial infections. Concomitant cultures are necessary to recover organisms for confirmatory identification of vancomycinresistant bacteria, antimicrobial susceptibility testing, and for epidemiological typing.

# 2. Indication(s) for use:

The Cepheid Xpert $^ \mathrm { \textregistered }$ vanA Assay performed in the GeneXpert $^ \mathrm { \textregistered }$ Dx System is a qualitative in vitro diagnostic test designed for rapid detection of the vanA gene sequence associated with vancomycin resistance in bacteria obtained from rectal swab specimens from patients at risk for intestinal colonization with vancomycin-resistant bacteria. The test utilizes automated real-time polymerase chain reaction (PCR) to detect the vanA gene that is frequently associated with vancomycin-resistant enterococci (VRE). The Xpert vanA Assay is intended to aid in the recognition, prevention, and control of vancomycin resistant organisms that colonize patients in healthcare settings. The Xpert vanA Assay is not intended to diagnose infections caused by vancomycin-resistant bacteria nor to guide or monitor treatment for vancomycin-resistant bacterial infections. Concomitant cultures are necessary to recover organisms for confirmatory identification of vancomycinresistant bacteria, antimicrobial susceptibility testing, and for epidemiological typing.

3. Special conditions for use statement(s): For Prescription Use only

4. Special instrument requirements:

GeneXpert $^ \mathrm { \textregistered }$ Dx System (GX-4 or GX-16 instruments, and the GeneXpert $^ \mathrm { \textregistered }$ Dx System Version 2.1 software)

# I. Device Description:

The Cepheid Xpert vanA Assay is a rapid, automated in vitro diagnostic test for qualitative detection of the vanA gene sequence associated with vancomycin resistance in bacteria obtained directly from rectal swab specimens. The Xpert vanA Assay system performs realtime multiplex polymerase chain reaction (PCR) for detection of DNA after an initial sample processing step. The assay is performed on the Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System.

The specimen is collected on a double swab, one of which is placed in a tube containing elution reagent. Following brief vortexing, the eluted material and two single-use reagents (Reagent 1 and Reagent 2) that are provided with the assay are transferred to different, uniquely-labeled chambers of the disposable fluidic cartridge (the Xpert vanA cartridge). The user initiates a test from the system user interface and places the cartridge into the GeneXper $^ \mathrm { \textregistered }$ Dx System instrument platform, which performs hands-off realtime, multiplex polymerase chain reaction (PCR) for detection of DNA. In this platform, additional sample preparation, amplification, and real-time detection are all fully automated and completely integrated.

The GeneXpert $^ \mathrm { \textregistered }$ System consists of a GeneXpert instrument, personal computer, and the multi-chambered fluidic cartridges that are designed to complete sample preparation and real-time PCR for detection of the vanA gene that is associated with vancomycin-resistant enterococci (VRE) in less than 45 minutes. Each instrument system has 1 to 16 randomly accessible modules that are each capable of performing separate sample preparation and real-time PCR tests. Each module contains a syringe drive for dispensing fluids, an ultrasonic horn for lysing cells or spores, and a proprietary I-CORE $^ \mathrm { \textregistered }$ thermocycler for performing realtime PCR and detection.

The Xpert vanA Assay includes reagents for the detection of the vanA resistant gene as well as an internal sample processing control (SPC) to control for adequate processing of the target bacteria and to monitor the presence of inhibitor(s) in the PCR assay. The SPC also ensures the PCR conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. The Probe Check Control (PCC) verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

IDI-VanR Assay Remel Bile Esculin Azide agar with $6 ~ { \mu \mathrm { g / m L } }$ vancomycin (BEAV)

2. Predicate 510(k) number(s):

K061686   
K972359

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Xpert vanA Assay</td><td colspan="1" rowspan="1">Remel Bile EsculinAzide agar with 6μg/mL vancomycin(BEAV)</td><td colspan="1" rowspan="1">IDI-VanR Assay</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Cepheid Xpert® vanAAssay performed in theGeneXpert® Dx System is aqualitative in vitro diagnostictest designed for rapid detectionof the vanA gene sequenceassociated with vancomycinresistance in bacteria obtainedfrom rectal swab specimensfrom patients at risk forintestinal colonization withvancomycin-resistant bacteria.The test utilizes automated real-time polymerase chain reaction(PCR) to detect the vanA genethat is frequently associated withvancomycin resistantenterococci (VRE). The XpertvanA Assay is intended to aid inthe recognition, prevention, andcontrol of vancomycin resistantorganisms that colonize patientsin healthcare settings. The XpertvanA Assay is not intended todiagnose infections caused byvancomycin-resistant bacterianor to guide or monitortreatment for vancomycinresistant bacterial infections.Concomitant cultures arenecessary to recover organismsfor confirmatory identification ofvancomycin-resistant bacteria,antimicrobial susceptibilitytesting, and for epidemiologicaltyping.</td><td colspan="1" rowspan="1">Remel Bile Esculin Azideagar with 6 µg/mLvancomycin is a platedmedium recommendedfor use in qualitativeprocedures as a selectiveanddifferential medium forthe primary isolation ofvancomycin-resistantenterococci fromsurveillance cultures. Thisproduct is not intendedfor use as a method ofantimicrobialsusceptibility testing.Confirmation ofvancomycin resistance byan approved method isrecommended as someorganisms on initialisolation may overcomethe inhibitory effects ofthe medium.</td><td colspan="1" rowspan="1">The IDI-VanR® Assay is aqualitative in vitro test forthe rapid detection ofvancomycin-resistanceenterococci (VRE). Theassay is performed on anautomated real-time PCRinstrument with rectalswabs from patients at riskfor VRE colonization. TheIDI-VanR® Assay can beused as an aid to identify,prevent and controlvancomycin-resistantcolonization in healthcaresettings. Concomitantcultures are necessary torecover organisms forepidemiological typing,susceptibility testing and forfurther confirmatoryidentification. The IDIVanR® Assay is notintended to diagnose VREinfections nor to guide ormonitor treatment for VREinfections.</td></tr><tr><td colspan="1" rowspan="1">Type of test</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Fully-automated nucleic acidamplification (DNA); real-timePCR</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Rectal swabs</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge</td><td colspan="1" rowspan="1">Disposable single-use,multichambered, fluidiccartridge.</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Disposable single-use PCRtube</td></tr><tr><td colspan="1" rowspan="1">Probes</td><td colspan="1" rowspan="1">TaqMan® Probes</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Molecular Beacons</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal sample processingcontrol (SPC) and probe checkcontrol (PCC).External controls available.</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">One internal reagent controland external positive andnegative controls requiredper run</td></tr><tr><td colspan="1" rowspan="1">DNA TargetSequence</td><td colspan="1" rowspan="1">Detects gene sequences for thevanA encoded resistance tovancomycin / teicoplanin</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Detects gene sequences forVanR (vanA and vanB)encoded resistance tovancomycin / teicoplanin.</td></tr><tr><td colspan="1" rowspan="1">Rapid test results</td><td colspan="1" rowspan="1">Less than 45 minutes to results.</td><td colspan="1" rowspan="1">48 hours</td><td colspan="1" rowspan="1">Approximately 120minutes.</td></tr><tr><td colspan="1" rowspan="1">Interpretation of testresults</td><td colspan="1" rowspan="1">Diagnostic software of theCepheid GeneXpert DX system</td><td colspan="1" rowspan="1">Visual interpretation</td><td colspan="1" rowspan="1">Diagnostic software of theCepheid SmartCycler DXsystem</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Xpert VanA Assay</td><td colspan="1" rowspan="1">Remel Bile EsculinAzide agar with 6μg/mL vancomycin(BEAV)</td><td colspan="1" rowspan="1">IDI-VanR Assay</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Cepheid GeneXpert Dx System</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">CepheidSmartCycler</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Fully-automated nucleic acidamplification (DNA); real-timePCR</td><td colspan="1" rowspan="1">Phenotypic detection ofvancomycin-resistantenterococci (VRE) basedon culture growth</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Mode of Detection</td><td colspan="1" rowspan="1">Presence of vanA gene</td><td colspan="1" rowspan="1">Growth or no growth on6ug/mL vancomycin agar</td><td colspan="1" rowspan="1">Presence of VanR (vanAand vanB) gene</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Rectal swabs</td><td colspan="1" rowspan="1">Culture grown direct fromrectal swab or stool</td><td colspan="1" rowspan="1">Rectal swabs</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Internal sample processingcontrol (SPC) and probe checkcontrol (PCC).External controls available.</td><td colspan="1" rowspan="1">Internal Controls - N/A</td><td colspan="1" rowspan="1">One internal reagent controland external positive andnegative controls requiredper run</td></tr><tr><td colspan="1" rowspan="1">DNA TargetSequence</td><td colspan="1" rowspan="1">Detects sequences for the vanAgene.</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Detects sequences forvancomycin resistance[vanR (vanA and vanB)]gene, but does notdifferentiate vanA fromvanB.</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction /Fluidics</td><td colspan="1" rowspan="1">Self-contained and automatedafter swab elution and twosingle-dose reagent additions.</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Manual</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

The primers and probes in the Xpert vanA Assay detect the presence of unique sequences for vanA resistance gene. Rectal swabs are collected and transported to the GeneXpert $^ \mathrm { \textregistered }$ System area. The swab is placed in a tube containing $1 . 7 \mathrm { m L }$ Sample Reagent. Following a brief vortex, the eluted material and two other liquid reagents are transferred to different chambers of the cartridge. The user initiates a test from the system user interface, the instrument signals the user where to place the cartridge by flashing a green light, and the cartridge is placed into the indicated module in the GeneXpert $^ \mathrm { \textregistered }$ Dx System instrument. The instrument moves the sample and reagents to and from different chambers within the Xpert vanA Assay cartridge. The GeneXpert $^ \mathrm { \textregistered }$ Dx System performs sample preparation by mixing the eluted sample with the Sample Preparation Control (Bacillus globigii in the form of a dry spore cake within the cartridge, is used as the SPC) and treatment reagents, capturing the bacterial cells on a filter, lysing the cells using glass beads and an ultrasonic horn, then eluting the released DNA. The DNA solution is then mixed with dry PCR reagents and transferred into the PCR tube for real-time PCR and detection. The Xpert vanA Assay completes sample preparation and realtime PCR in less than 45 minutes. Internal controls in Xpert vanA Assay check key automated steps.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

Reproducibility study was performed using a panel of 4 specimens with varying concentrations of vanA on 10 different days at each of the three sites (4 specimens $_ { \textrm { X 2 } }$ operators/ day $\textbf { \scriptsize x } 1 0$ days x 3 sites). One lot of Xpert vanA assay was used at each of the 3 testing sites.

Summary of Reproducibility Results (all)a   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">% Agreement</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen ID</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total Agreement</td></tr><tr><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">90%(18/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">96.7% (58/60)</td></tr><tr><td colspan="1" rowspan="1">vanA High Neg</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">98.3% (59/60)</td></tr><tr><td colspan="1" rowspan="1">vanA Low Pos</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">vanA Moderate Pos</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">95%(19/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">98.3% (59/60)</td></tr><tr><td colspan="1" rowspan="1">% Total Agreement bySite</td><td colspan="1" rowspan="1">100%(80/80)</td><td colspan="1" rowspan="1">96.3%(77/80)</td><td colspan="1" rowspan="1">98.8%(79/80)</td><td colspan="1" rowspan="1">98.3% (236/240)</td></tr></table>

a For negative and high negative samples, %Agreement $=$ (#negative results/total samples run); for low and moderate positive samples, $\%$ Agreement $=$ (# positive results/total samples run)

Summary of Ct Value Results by Sample Level and Target   

<table><tr><td colspan="4">Bg</td></tr><tr><td>Level</td><td>Mean</td><td>Std Dev</td><td>%CV</td></tr><tr><td>vanA High Neg</td><td>32.88</td><td>0.60</td><td>1.83</td></tr><tr><td>vanA Low Pos</td><td>32.88</td><td>0.77</td><td>2.34</td></tr><tr><td>vanA Moderate Pos</td><td>32.80</td><td>0.78</td><td>2.38</td></tr><tr><td>Neg</td><td>33.15</td><td>0.65</td><td>196</td></tr></table>

<table><tr><td colspan="4">vanA</td></tr><tr><td>Level</td><td>Mean</td><td>Std Dev</td><td>%CV</td></tr><tr><td>vanA Low Pos</td><td>33.76</td><td>1.00</td><td>2.95</td></tr><tr><td>vanA Moderate Pos</td><td>30.35</td><td>1.33</td><td>4.40</td></tr></table>

1 Ct cutoff for vanA $\scriptstyle \lfloor = 4 0$

# b. Linearity/assay reportable range:

Linearity was evaluated using Enterococcus faecium (vanA) cells serially diluted over 6 logs and processed using the Xpert vanA Assay. The diluted cells resulted in a cell concentration dose range of 50 CFU/test to $\bar { 5 } \mathrm { x } 1 0 ^ { 7 }$ CFU/test. Replicates of four (4) were tested at each concentration. For enterococci cells, under the conditions of this study, the Xpert vanA Assay responds linearly $( \mathrm { r } 2 = 0 . 9 9 4 )$ ) with respect to vanA detection as a function of Enterococcus faecium cell input over 6 logs $( 5 0 - 5 \mathrm { x } 1 0 ^ { 7 }$ CFU/test). The reportable Ct range is 12.1 to 35.3 (cutoff $\mathrm { C t } =$ 40.0). PCR efficiency for the vanA reaction is $8 7 . 7 \%$ .

The following external controls were evaluated for use with the GeneXpert Dx System and Xpert vanA assay.

External Controls   

<table><tr><td rowspan=1 colspan=1>External Control ID</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Control</td></tr><tr><td rowspan=1 colspan=1>MicroBioLogics 0366(Kwik-Stik)</td><td rowspan=1 colspan=1>Vancomycin-sensitive Enterococcus faecalis</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>CCUG36804(Culture Collection ofGöteborg)</td><td rowspan=1 colspan=1>Vancomycin-resistant Enterococcus faecium</td><td rowspan=1 colspan=1>Positive (vanA)</td></tr><tr><td rowspan=1 colspan=1>Gibson Laboratories, LLC#CeVRE-01</td><td rowspan=1 colspan=1>Vancomycin-resistant Enterococcus faecium</td><td rowspan=1 colspan=1>Positive (vanA)</td></tr><tr><td rowspan=1 colspan=1>Gibson Laboratories, LLC#CeVRE-02</td><td rowspan=1 colspan=1>Vancomycin-sensitive Enterococcus faecalis</td><td rowspan=1 colspan=1>Negative</td></tr></table>

The external quality control organisms were tested at the different testing sites with the following results.

KWIK-STIK™ Negative Control Data   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Test Result</td><td rowspan=1 colspan=1>SPC Ct</td><td rowspan=1 colspan=1>vanA Ct</td></tr><tr><td rowspan=1 colspan=1>Neg01</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg 02</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg 03</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg04</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg05</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg06</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg07</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg08</td><td rowspan=1 colspan=1>ERROR</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg09</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg 10</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0</td></tr></table>

One negative control result reported “ERROR” due to a probe check failure.

Control Data, CCUG36804 (vanA positive E. faecium)   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Test Result</td><td rowspan=1 colspan=1>SPC Ct</td><td rowspan=1 colspan=1>vanA Ct</td></tr><tr><td rowspan=1 colspan=1>vanA01</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>vanA02</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>29.7</td></tr><tr><td rowspan=1 colspan=1>vanA03</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>29.6</td></tr><tr><td rowspan=1 colspan=1>vanA04</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>30.2</td></tr><tr><td rowspan=1 colspan=1>vanA05</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>30.3</td></tr><tr><td rowspan=1 colspan=1>vanA06</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>29.5</td></tr><tr><td rowspan=1 colspan=1>vanA07</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>30.2</td></tr><tr><td rowspan=1 colspan=1>vanA08</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>29.9</td></tr><tr><td rowspan=1 colspan=1>vanA09</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>29.4</td></tr><tr><td rowspan=1 colspan=1>vanA10</td><td rowspan=1 colspan=1>vanA POSITIVE</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>30.5</td></tr></table>

Control Data, Gibson Laboratories (vanA positive E. faecium)   

<table><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Test Result</td><td colspan="1" rowspan="1">SPC Ct</td><td colspan="1" rowspan="1">vanA Ct</td></tr><tr><td colspan="1" rowspan="1">vanA 01</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">23.3</td></tr><tr><td colspan="1" rowspan="1">vanA02</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">23.6</td></tr><tr><td colspan="1" rowspan="1">vanA 03</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">23.8</td></tr><tr><td colspan="1" rowspan="1">vanA 04</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">23.3</td></tr><tr><td colspan="1" rowspan="1">vanA 05</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">31.5</td><td colspan="1" rowspan="1">23.2</td></tr><tr><td colspan="1" rowspan="1">vanA 06</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">23.5</td></tr><tr><td colspan="1" rowspan="1">vanA 07</td><td colspan="1" rowspan="1">ERROR</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">vanA 08</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">31.5</td><td colspan="1" rowspan="1">23.2</td></tr><tr><td colspan="1" rowspan="1">vanA 09</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">24.4</td></tr><tr><td colspan="1" rowspan="1">vanA 10</td><td colspan="1" rowspan="1">vanA POSITIVE</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">23.4</td></tr></table>

One sample result reported “ERROR” due to high syringe pressure.

Control Data, Gibson Laboratories, LLC (Negative Control, E. faecalis)   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Test Result</td><td rowspan=1 colspan=1>SPC Ct</td><td rowspan=1 colspan=1>vanA Ct</td></tr><tr><td rowspan=1 colspan=1>Neg01</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg02</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg03</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg04</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg05</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg06</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg07</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg08</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg 09</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Neg 10</td><td rowspan=1 colspan=1>vanA NEGATIVE</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>0</td></tr></table>

External Controls may be used in accordance with accrediting institutions and government regulations. External Controls are not provided in the test kit; however they are available for purchase from outside sources. The outside source and the catalog numbers are provided to the costumer in the ‘Materials Available but Not Provided’ section of the Xpert vanA Assay Package Insert.

# d. Analytical Sensitivity:

# Limit of Detection

Limit of Detection (LoD) studies were performed using Enterococcus faecium (vanA) diluted into a fecal matrix of human origin that can be detected by the Xpert vanA Assay. The fecal matrix consisted of autoclaved human liquid feces (vanA negative) diluted 1:10 in Tris buffer.

The analytical LoD was estimated using 4 to 10 replicates at each dilution. The LoD was confirmed by running a total of 20 replicates at the estimated LoD concentration. Under the conditions of this study, the limit of detection for the Xpert vanA Assay on a simulated rectal swab specimen is 37 CFU.

Summary of Mean Ct   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">CFU*</td><td colspan="1" rowspan="1">Positives/ReplicatesTested^</td><td colspan="1" rowspan="1">SPC</td><td colspan="1" rowspan="1">vanA</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Ct</td><td colspan="1" rowspan="1">Ct</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">35.1</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">35.6</td></tr><tr><td rowspan="4">vanA^</td><td>68</td><td>5/5</td><td>34.3</td><td>34.9</td></tr><tr><td>107</td><td>5/5</td><td>32.9</td><td>34.1</td></tr><tr><td>219</td><td>5/5</td><td>32.4</td><td>32.9</td></tr><tr><td></td><td></td><td></td><td></td></tr></table>

(\*) determined by the direct plate method (^) CCUG36804 Strain Source: Culture Collection University of Göteborg (CCUG), Gothenburg, Sweden

# Analytical Reactivity (Inclusivity) / Challenge Panel

Challenge panel included thirty vancomycin-resistant enterococci strains and 20 vancomycin sensitive enterococci strains, provided by the CDC, and tested using the Xpert vanA Assay. Of the 30 vancomycin-resistant enterococci strains, 10 were identified as vanA positive. Enterococci strains were selected to broadly represent the genetic diversity found in enterococci. Stock cultures were prepared by suspending the bacterial growth from agar plates in PBS buffer containing $1 5 \%$ glycerol. The concentration of each stock was adjusted to $\mathsf { 5 . 6 x l 0 } ^ { \mathsf { 9 } }$ to $2 . 1 \mathrm { x } 1 0 ^ { - 1 0 } \mathrm { C F U / m L }$ . All strains were serially diluted to approximately 360 CFU/swab and tested in triplicate.

All 20 vancomycin sensitive strains were reported as “vanA NEGATIVE”. Among the 10 vanA positive vancomycin-resistant enterococci strains tested, one strain was reported as “vanA NEGATIVE.” When this strain was sequenced, it was reported to be a vanB. The remaining 9 vanA positive vancomycin resistant enterococci strains were correctly reported as “vanA POSITIVE.”. Among the 20 non-vanA vancomycin resistant enterococci strains, all were reported as “vanA NEGATIVE.”

Summary Table of Analytical Reactivity (Inclusivity) Results of the Xpert vanA Assay on a CDC-Supplied Panel of Enterococci Specimens   

<table><tr><td colspan="1" rowspan="2">Sample ID</td><td colspan="1" rowspan="2">Organism</td><td colspan="1" rowspan="2">Genotype</td><td colspan="1" rowspan="1">Xpert vanA Result</td></tr><tr><td colspan="1" rowspan="2">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">NJ-5</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td></tr><tr><td colspan="1" rowspan="1">VA32</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS110</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS119</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS307</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS314</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS406</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS413</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">Y</td></tr><tr><td colspan="1" rowspan="1">VS414</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS418</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS517</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS604</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS615</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS719</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS804</td><td colspan="1" rowspan="1">E. casseliflavus</td><td colspan="1" rowspan="1">Sensitive</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">NJ-4</td><td colspan="1" rowspan="1">E. gallinarium</td><td colspan="1" rowspan="1">Sensitive (vanC)</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS106</td><td colspan="1" rowspan="1">E. gallinarium</td><td colspan="1" rowspan="1">Sensitive (vanC)</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS411</td><td colspan="1" rowspan="1">E. gallinarium</td><td colspan="1" rowspan="1">Sensitive (vanC)</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS608</td><td colspan="1" rowspan="1">E. gallinarium</td><td colspan="1" rowspan="1">Sensitive (vanC)</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS807</td><td colspan="1" rowspan="1">E. gallinarium</td><td colspan="1" rowspan="1">Sensitive (vanC)</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">E38-10</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">E6-1</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">NJ-2</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA16</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA36</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA38</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA63</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA8</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA89</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS102</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS103</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="2">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS111</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td></tr><tr><td colspan="1" rowspan="1">VS112</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS319</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS415</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS416</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS501</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS506</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS514</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VS605</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanB</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">A256</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">NJ-1</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VA100B</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA NEGATIVE</td></tr><tr><td colspan="1" rowspan="1">VA29</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VA6</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VS105</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VS318</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VS420</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VS511</td><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr><tr><td colspan="1" rowspan="1">VS611</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">vanA</td><td colspan="1" rowspan="1">vanA POSITIVE</td></tr></table>

A The genotype information contained in the grey column was provided by the CDC B.Sequencing confirmed that this specimen is a vanB subtype, not vanA as typed by CDC

# Cross Reactivity

Forty-two bacterial and fungal strains were collected, quantitated and tested using the Xpert vanA Assay. The strains originated from the American Type Culture Collection (ATCC), Culture Collection University of Göteborg (CCUG), German Collection of Microorganisms and Cell Cultures (DSMZ), and the Centers for Disease Control and Prevention (CDC).

The organisms tested were identified as Gram-positive (22), Gram-negative (18), including antibiotic-resistant strains of Pseudomonas spp. and Acinetobacter spp., and yeast (2). The organisms were further classified as aerobic (24), anaerobic (14) or microaerophillic (2). Of the species tested, 2 vancomycin-sensitive strains of $E .$ . faecalis and $E$ . faecium were included.

Each strain was tested in triplicate at concentrations ranging from $8 . 5 \mathrm { x } 1 0 ^ { 8 }$ to $2 . 3 \mathrm { x } 1 0 ^ { 1 0 }$ CFU/swab. Positive and negative controls were included in the study. Under the conditions of the study, all isolates were reported “vanA NEGATIVE”.

# Interference Study

Sixteen exogenous substances occasionally used or found in stool were tested for interference with the Xpert vanA Assay. These potentially interfering substances in rectal swab specimens include, but are not limited to, blood, mucus, fecal fats, laxatives, stool softeners, anti-diarrhea medications, anti-itch creams, lubricants, antibiotics and hemorrhoid ointments. The substances were tested at levels representing approximately $1 - 2 5 \%$ (v/v or w/v) final concentrations. Two (2) of the sixteen (16) exogenous substances tested in this study gave a slightly higher Ct relative to the buffer control (hydrocortisone cream and Pepto-Bismol $^ \mathrm { \textregistered }$ ).

Mean Cts for the SPC and vanA Target per Potentially Interfering Substance   

<table><tr><td colspan="1" rowspan="1">Substance</td><td colspan="1" rowspan="1">SPC Mean Ct</td><td colspan="1" rowspan="1">vanA Mean Ct</td></tr><tr><td colspan="1" rowspan="1">PBS 15% glycerol (Buffer Control)</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">32.0</td></tr><tr><td colspan="1" rowspan="1">Whole Blood</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">31.3</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">30.7</td></tr><tr><td colspan="1" rowspan="1">Kaopectate®</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">32.7</td></tr><tr><td colspan="1" rowspan="1">Imodium®</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol®</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">Fleet®</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">31.2</td></tr><tr><td colspan="1" rowspan="1">Fecal fats</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">30.2</td></tr><tr><td colspan="1" rowspan="1">Hydrocortisone Cream</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">34.4</td></tr><tr><td colspan="1" rowspan="1">K-Y Jelly/Gelée®</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Vaseline</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">32.5</td></tr><tr><td colspan="1" rowspan="1">Dulcolax®</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">32.2</td></tr><tr><td colspan="1" rowspan="1">Preparation H Portable Wipes</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">32.6</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">31.8</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">31.9</td></tr><tr><td colspan="1" rowspan="1">Anusol® Plus</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">31.5</td></tr></table>

Carry-Over Contamination study consisted of a negative sample (no enterococci present) processed in the same GeneXpert module immediately following a very high positive sample made up of ${ 1 0 } ^ { 6 }$ CFU Enterococcus faecium (vanA) spiked into the elution buffer. This was repeated 20 times between two GeneXpert modules for a total of 40 runs. All 20 negative samples were correctly reported “vanA NEGATIVE.” All 20 high positive samples were correctly reported “vanA POSITIVE.”

# f. Assay cut-off:

# Lot Specific Parameters and Assay Settings

Lot specific assay settings are generated for every lot manufactured to account for slight variations in reagent production. The lot specific assay settings (LSP file) are incorporated into the 2-D barcode on each cartridge label and are transferred to the GeneXpert $^ \mathrm { \textregistered }$ Dx system via a hand-held barcode scanner prior to initiating the Xpert vanA Assay.

# General Assay Settings

General assay settings are used for all reagent lots. They are fixed and not part of the LSP process. The following table lists general assay settings:

<table><tr><td rowspan=1 colspan=1>Attribute</td><td rowspan=1 colspan=1>Setting</td></tr><tr><td rowspan=1 colspan=1>Background Subtraction</td><td rowspan=1 colspan=1>Always ON</td></tr><tr><td rowspan=1 colspan=1>Background Minimum Cycle</td><td rowspan=1 colspan=1>Default setting = 5</td></tr><tr><td rowspan=1 colspan=1>Background Maximum Cycle</td><td rowspan=1 colspan=1>Manual setting = 30</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (vanA and SPC)</td><td rowspan=1 colspan=1>Manual setting = 30</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis</td><td rowspan=1 colspan=1>Primary</td></tr><tr><td rowspan=1 colspan=1>Boxcar Average Cycles</td><td rowspan=1 colspan=1>Zero (Off)</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (vanA)</td><td rowspan=1 colspan=1>Default setting = 5</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (SPC)</td><td rowspan=1 colspan=1>Manual setting = 25</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (SPC and vanA)</td><td rowspan=1 colspan=1>Manual setting = 40</td></tr></table>

The valid cycle range for the vanA target is 5 to 40 cycles based upon pre-clinical results to maximize percent sensitivity and percent specificity. The valid cycle range for the SPC is 25 to 40 cycles. The SPC ensures the PCR conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. SPC performance is designed to respond to the presence of potentially inhibitory conditions and provide an invalid test result in lieu of a believable but erroneous test result.

To obtain a valid positive test result, the vanA Ct value must be reported within the valid cycle range. To obtain a valid negative test result, the SPC Ct must be reported within its valid cycle range. If the SPC falls outside the valid cycle range, the test result is “Invalid” and must be repeated.

Valid minimum and maximum Ct settings (25 to 40 cycles) for the SPC were derived from analytical data (vanA lot specific parameter (LSP) testing of three production lots), inhibitory studies with potentially interfering substances (IS), and pre-clinical true negative vanA samples collected during the design and development phase.

A valid maximum SPC Ct of 40 cycles was used to differentiate between valid negative results and invalid results in the clinical study. The SPC was designed to be sensitive to the presence of inhibition, while at the same time limiting the number of invalid test results.

# 2. Comparison studies:

# a. Method comparison with predicate device:

Performance characteristics of the Xpert vanA Assay were determined in a multi-site prospective investigation study at three US sites by comparing the Xpert vanA Assay to reference culture followed by bi-directional sequencing for confirmation on vancomycin-resistant Enterococcus isolates.

Subjects included individuals whose routine care called for VRE testing. One swab from a double swab set was used for patient management; the other swab was used for the Xpert vanA Assay testing. The leftover swab designated for patient management was sent to a central laboratory for reference culture.

Leftover specimen swabs designated for culture testing were stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ and shipped on ice packs to the central culture laboratory within 48 hours of collection. Reference culture was initiated within 16 hours of receipt or within 5 days of swab collection.

Each swab was subsequently placed into an enrichment broth. The plates were incubated at $3 5 \mathrm { { ^ \circ C } }$ and examined at 48 and 72 hours. The broth was also incubated at $3 5 \mathrm { { ^ \circ C } }$ for 48 hours and subcultured to a bile esculin azide agar with $6 \mu \mathrm { g } / \mathrm { m l }$ of vancomycin.

Small, gray colonies with a black halo were considered suspicious for VRE. Presumptive identification was accomplished by performing a Gram stain, catalase and PYR disc (L-pyrrolidonyl-beta-naphthylamide) test. Presumptive VRE specimens were Gram-positive cocci or coccobacilli and PYR positive. Presumptive VRE was definitively identified using the API20S strip (BioMérieux, France). Finally, VRE isolates were tested for their susceptibility to glycopeptides using vancomycin E-test strips (AB Biodisk, Sweden). Susceptibility to teicoplanin for the isolates was determined by agar dilution.

Following reference culture testing, DNA was prepared from vancomycin-resistant Enterococcus isolates, and sent to a second reference laboratory for bi-directional sequencing using alternative vanA specific primers (i.e., different from those used in the Xpert vanA Assay).

Performance of the Xpert vanA Assay was calculated relative to the results of direct culture with bi-directional sequencing, and enriched culture with bi-directional sequencing.

# Overall Results

A total of 1231 specimens were tested by Xpert vanA Assay, culture and bi-directional sequencing.

Xpert vanA Assay Performance vs. Direct Culture with Bi-directional Sequencing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Direct Culture + Sequencing</td></tr><tr><td rowspan=4 colspan=1>Ses u kdX</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>210</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1019</td><td rowspan=1 colspan=1>1021</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>128</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>1231</td></tr><tr><td rowspan=1 colspan=1>′</td><td rowspan=1 colspan=4>% Positive Agreement: 98.4%% Negative Agreement:  92.4%Accuracy: 93.0%PPV:  60.0%NPV:  99.8%Prevalence:  10.4%</td></tr></table>

Of the Xpert vanA Assays run on eligible specimens, $9 4 . 0 \%$ (1180/1255) of these specimens were successful on the first attempt. The remaining 75 gave indeterminate results on the first attempt (26 “INVALID”, 49 “ERROR” and 0 “NO RESULT”). Sixty two (62) of the75 indeterminates on the first attempt had sufficient sample for retest, $8 2 . 3 \%$ (51/62) gave a result on the second the attempt. Overall assay success rate (combining the first and second attempts) was $9 8 . 1 \%$ (1231/1255).

Xpert vanA Assay Performance vs. Enriched Culture with Bi-directional Sequencing   

<table><tr><td></td><td colspan="4">Enriched Culture + Sequencing</td></tr><tr><td rowspan="4">Suss vu edy</td><td></td><td>Pos</td><td>Neg</td><td>Total</td></tr><tr><td>Pos</td><td>141</td><td>69</td><td>210</td></tr><tr><td>Neg</td><td>22</td><td>999</td><td>1021</td></tr><tr><td>Total</td><td>163</td><td>1068</td><td>1231</td></tr><tr><td rowspan="4"></td><td colspan="2">% Positive Agreement: % Negative Agreement: Accuracy:</td><td colspan="2">86.5% 93.5%</td></tr><tr><td colspan="2">PPV: 67.1%</td><td colspan="2">92.6%</td></tr><tr><td colspan="2">NPV:</td><td colspan="2">97.8%</td></tr><tr><td colspan="2">Prevalence:</td><td colspan="2">13.2%</td></tr></table>

Of the Xpert vanA Assays run on eligible specimens, $9 4 . 0 \%$ (1180/1255) of these specimens were successful on the first attempt. The remaining 75 gave indeterminate results on the first attempt (26 “INVALID”, 49 “ERROR” and 0 “NO RESULT”). Sixty two (62) of the75 indeterminates on the first attempt had sufficient sample for retest, $8 2 . 3 \%$ (51/62) gave a result on the second the attempt. Overall assay success rate (combining the first and second attempts) was $9 8 . 1 \%$ (1231/1255).

b. Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity: Not applicable

b. Clinical specificity: Not applicable   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Not applicable

# N. Instrument Name:

GeneXpert Dx System

• GeneXpert® instrument, computer with proprietary software,   
• hand-held barcode scanner, and Operator Manual

# O. System Descriptions:

1. Modes of Operation:

The GeneXpert® Dx System automates and integrates sample preparation, nucleic acid amplification, and detection of the target DNA sequence in patient specimens using realtime Polymerase Chain Reaction (PCR).

Each GeneXpert instrument is similar in that the GeneXpert I (GX I), GeneXpert IV (GX IV) and GeneXpert XVI (GX XVI) all contain the same modules. The difference between the GeneXpert models is that the GX I contains one module, the GX IV can hold up to four modules, and the GX XVI can hold up to sixteen modules, each of which processes one sample at a time. Once the cartridge is loaded in the instrument, all fluids are completely contained within the disposable, single-use plastic Xpert™ Assay cartridges throughout the sample handling, amplification and detection processes.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X____ or No

The software that controls the operation of the sample processing and the I-CORE module, and collects, analyzes and interprets the acquired optical data is the GeneXpert Dx Version 2.1 software.

3. Specimen Identification: Barcode

4. Specimen Sampling and Handling: Automated

# GeneXpert Dx System Hardware Components for Automated Sample Processing

<table><tr><td rowspan=1 colspan=1>Module HardwareComponents</td><td rowspan=1 colspan=1>Function</td></tr><tr><td rowspan=1 colspan=1>Valve Drive</td><td rowspan=1 colspan=1>Rotates the cartridge valve body to address the different cartridge chambers.</td></tr><tr><td rowspan=1 colspan=1>Syringe Pump drive</td><td rowspan=1 colspan=1>Dispenses fluids to and from the different cartridge chambers.</td></tr><tr><td rowspan=1 colspan=1>Ultrasonic horn</td><td rowspan=1 colspan=1>Lyses the bacterial cells and sample prep control.</td></tr><tr><td rowspan=1 colspan=1>I-CORE® module</td><td rowspan=1 colspan=1>Performs PCR amplification and detection. As the user inserts the cartridgeinto the system, the reaction tube component of the cartridge is inserted intothe I-CORE module. After sample preparation within the cartridge, the sampleand reagent mixture is transferred from the cartridge chamber into the reactiontube. During the amplification process, the I-CORE heater heats up and thefan cools down the reaction tube contents. Two optical blocks positionedwithin the ICORE excite the dye molecules that make up the probes and detectthe fluorescence emitted. The system uses calibration and data analysisalgorithms to determine a relative fluorescence value for each reporter dyeafter each thermal cycle.</td></tr><tr><td rowspan=1 colspan=1>Hand-held BarcodeScanner</td><td rowspan=1 colspan=1>Scans cartridge barcode and optional Patient or Sample ID barcode intothe GeneXpert Dx System.</td></tr></table>

5. Calibration:

Optical and thermal calibration of the GeneXpert Dx System is performed by Cepheid at the time of manufacture prior to installation and once yearly or after 1000 runs per module. The user does not calibrate or perform any serviceable functions on the instrument. The normalization function compensates for any optical degradation between calibrations.

The thermal reaction chamber thermistors are calibrated $\mathrm { t o } \pm 0 . 5 0 ^ { \circ } \mathrm { C }$ using National Institute of Standards and Technology (NIST)-traceable standards. During the manufacturing process, the temperature of the heating system is measured at two temperatures: $6 0 \mathrm { { ^ \circ C } }$ and $9 5 \mathrm { { ^ \circ C } }$ . Calibration coefficients that correct for small errors in the raw thermistor readings of the heaters are stored in the memory of each I-CORE module.

The optical system is calibrated using standard concentrations of individual unquenched fluorescent dye-oligos. For each optical channel, the signal produced by a tube alone (the blank signal) is subtracted from the raw signal produced by the dye-oligo standard to determine the spectral characteristics. Using the individual spectral characteristics of the pure dye-oligos, signals from an unknown mixture of dye-oligos can be resolved into corrected signals for the individual dye-oligos in the mixture.

6. Quality Control:

The Xpert vanA Assay includes a system control, referred to as the Probe Check Control (PCC), and an internal control, referred to as the Sample Processing Control (SPC). These internal controls are contained within the Xpert vanA cartridge. An additional control is the System Control Check for Temperature. This check ensures that the GeneXpert® Dx Instrument is operating within validated heating and cooling specifications.

The Probe Check Control (PCC) verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for vanA target or SPC, a probe check error is reported and the test will not continue.

The SPC verifies that conditions adequate for processing the target bacteria containing the vanA gene have occurred. It consists of Bacillus globigii spores formulated into a dry reagent bead included in each Xpert vanA cartridge. In addition, the SPC verifies the effectiveness of each sample preparation step, including reaction tube filling, that all reaction components are present and functioning, and monitoring for the presence of potential inhibitor(s) in the PCR Assay.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Failure Modes Testing was performed to determine the effect of failure modes that might occur with the vanA Assay. Assay failures may be attributed to operator error, manufacturing error, or instrument malfunction.

Operator error may include failing to add a liquid reagent to the cartridge or adding a liquid reagent to the wrong chamber. The following table includes results of the study describing possible operator errors

Test Results – Possible Operator Errors   

<table><tr><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=1>Liquid Reagent Addition</td><td rowspan=1 colspan=2>Xpert Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>A (Control)</td><td rowspan=1 colspan=1>All liquid reagents added correctly</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Omit addition of both Reagents 1 and 2</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>Omit addition of Reagent 1 only</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>Omit addition of Reagent 2 only</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>Swap Reagent 1 and 2 addition locations</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr></table>

A manufacturing error may result in beads being loaded incorrectly into the cartridge before packaging (missing beads or double beads), or a cartridge being assembled incorrectly (missing the filter required for cell capture). Results of the manufacturing error study is presented in the table below

Test Results – Possible Manufacturing Errors   

<table><tr><td rowspan=1 colspan=1>Condition</td><td rowspan=1 colspan=3>Bead</td><td rowspan=1 colspan=2>Xpert Results</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EZR</td><td rowspan=1 colspan=1>TSR</td><td rowspan=1 colspan=1>SPC</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>A (Control)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Error</td><td rowspan=1 colspan=1>Error</td></tr><tr><td rowspan=1 colspan=1>L*</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid</td></tr></table>

(\*) cartridges intentionally assembled without capture filter material

Some examples of instrument malfunctions include ultrasonic horn failure, motion of the syringe drive not detected, syringe pressure reading exceeds protocol limit, the system failed to find the plunger home position, a valve positioning error was detected, digital temperature reading of thermistor(s) not within acceptable range, and the optical signal from the detector(s) did not reach the expected value. Because the software performs self-check procedures prior to the start of each test, if any of the instrument malfunctions described above occur the test is aborted; no assay results are reported. Operation-terminated error messages are stored for each test and appear in the View Results screen. Instrument malfunctions are not part of this failure mode effect and criticality testing. However, failure mode effect and criticality analysis (FMECA) and hazard analysis has been completed on the GeneXpert Dx system.

Thermal Calibration Study was performed to determine if the thermal calibration is stable for 2,000 runs regardless of the time period of instrument use. The data from the study reported that a 2,000 run thermal stability claim can be supported with a predicted failure rate of $0 . 7 \%$ . The current labeling for the GeneXpert Dx System is 1 year or 1000 runs/module, whichever comes first.

Calibrator Variation Study was conducted to quantify possible effects of degradation of GeneXpert optical calibrations on assay performance. At $9 5 \%$ confidence, the data from the altered calibrations are the same as the data from the nominal calibration. These data provide evidence that the Xpert assays will continue to function effectively even when the instrument calibrations are off by as much as $\pm 5 0 \%$ .

Decay of GeneXpert Optical Calibration study was performed to determine the rate of degradation of the GeneXpert optical calibration. Based on the average measured rate of optical calibration decay following 2000 runs and the demonstrated effectiveness of probe check normalization to mitigate optical calibration decay, Xpert assays will meet specifications for optical performance with $9 9 \%$ confidence. No adverse impact on assay performance is expected. If assay-specific probe check acceptance criteria are not met, Xpert tests are aborted prior to initiation of PCR.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.