# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for a new $5 1 0 \mathrm { k }$ application for a qualitative real-time polymerase chain reaction (PCR) assay.

# C. Measurand:

Target DNA sequences for the SCCmec/orfX junction area of methicillin-resistant Staphylococcus aureus. More specifically, target DNA sequences for the junction area of the SCCmec adjacent to the integration site, called MREJ for Mec Right Extremity Junction. The BD GeneOhmTM MRSA ACP Assay is designed to detect MREJ genotypes i, ii, iii, iv, v and vii.

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay using 1 forward primer, 5 reverse primers, and 2 molecular beacon probes for the amplification and detection of methicillinresistant Staphylococcus aureus (MRSA) DNA.

# E. Applicant:

# G. Regulatory Information:

Regulation section: 21 CFR section 866.1640, Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NQX

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD GeneOhm™ MRSA ACP Assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization. The test utilizes polymerase chain reaction (PCR) for the amplification of MRSA DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD GeneOhm™ MRSA ACP Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose MRSA infections nor to guide or monitor treatment for MRSA infections. Concomitant cultures are necessary only to recover organisms for epidemiological typing or for further susceptibility testing.

# 2. Indication(s) for use:

The BD GeneOhm™ MRSA ACP Assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization. The test utilizes polymerase chain reaction (PCR) for the amplification of MRSA DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD GeneOhm™ MRSA ACP Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose MRSA infections nor to guide or monitor treatment for MRSA infections. Concomitant cultures are necessary only to recover organisms for epidemiological typing or for further susceptibility testing.

3. Special conditions for use statement(s): For prescription use.

4. Special instrument requirements: To be used with the Cepheid SmartCycler II System with Dx Software version 1.7b.

# I. Device Description:

The lysis of bacterial cells in nasal swab specimens is performed using the BD GeneOhm™ MRSA ACP Lysis kit that incorporates enzymatic Achromopeptidase (ACP) lysis technology for sample preparation of nasal swab specimens used with the BD GeneOhm™ MRSA ACP Assay. Lysis Reagent is reconstituted with Lysis Diluent and aliquoted into single use lysis tubes. A nasal swab specimen is collected and transported to the laboratory using the recommended swab collection/transport device. The nasal swab is introduced and broken into a Sample Buffer tube. An aliquot of the liquid sample suspension is then transferred into the tube containing the Lysis Reagent and incubated to break the bacterial cell wall and release the genetic material.

An aliquot of the lysed sample is then processed with the BD GeneOhm™ MRSA ACP Assay. An aliquot of the lysate is added to prepared PCR reagents which contain MRSA-specific primers that will amplify in the presence of specific genetic target. The assay also includes an Internal Control (IC) to monitor for the presence of inhibitors in the PCR reaction and to confirm the integrity of assay reagents. Controls and specimen lysates are added to disposable reaction tubes and placed in the SmartCycler II instrument. The amplification, detection and results interpretation are automatically performed by the SmartCycler II software.

The primers (1 forward primer and 5 reverse primers) and probes (2 molecular beacon probes) in the BD GeneOhm™ MRSA ACP Assay detect a proprietary sequence of the junction area of the staphylococcal chromosomal cassette (SCCmec) adjacent to the integration site, called MREJ for Mec Right Extremity Junction, indicating the presence of MRSA DNA. Amplification of IC and MRSA DNA are detected using specific hybridization probes that bind to a specific sequence of the amplified target. Differentiation of MRSA DNA and IC is done using molecular beacon probes which contain different fluorometric properties. The molecular beacon-target hybrid fluoresces at a different wavelength for MRSA and IC and the emitted light from this reaction is measured by the SmartCycler II instrument. MRSA and/or IC specific amplicons are detected simultaneously in two different fluorescence channels on the SmartCycler II and can therefore be differentiated. The operation of the SmartCycler II instrument is based on the proprietary microprocessor-controlled $\mathrm { I - C O R E } ^ { \mathbb { \mathrm { ‰} } }$ (Intelligent Cooling/Heating Optical Reaction) module.

# BD GeneOhm™ MRSA ACP Assay Reagents

# Master Mix

• $< 0 . 0 0 0 5 \%$ DNA polymerase complex $< 0 . 0 0 1 \%$ Internal Control: non-infectious DNA containing MRSA-primer binding sequences and a unique sequence for probe hybridization $< 0 . 0 6 \%$ primers $< 0 . 0 2 \%$ Molecular beacon probes $< 1 \%$ Nucleotide mix (dATP, dCTP, dGTP, dTTP)   
• Bovine serum albumin   
• Carbohydrate   
• $\mathrm { M g C l } _ { 2 }$   
• $< 0 . 0 0 1 \%$ non-infectious Staphylococcus epidermidis genomic DNA (ATCC 14990)

# Control DNA

• Tris-EDTA buffer   
• Carbohydrate $< 0 . 0 0 1 \%$ non-infectious genomic MRSA DNA (ATCC 43300)

# Diluent

• Tris-HCl buffer   
• $\mathrm { M g C l } _ { 2 }$   
• $( \mathrm { N H } 4 ) _ { 2 } \mathrm { S O } _ { 4 }$   
• KCl Tween-20

# Materials Provided

• Master Mix   
• Control DNA   
• Diluent   
• SmartCycler $^ \mathrm { \textregistered }$ reaction tubes, $2 5 ~ \mu \mathrm { L }$ capacity

# Materials Required But Not Provided

BD GeneOhm™ MRSA ACP Lysis Kit (BD catalog no. 441638) Vortex Genie 2 (VWR catalog no. 58815-234) with $1 . 5 \mathrm { m L }$ microtube holder or equivalent; for processing multiple samples, adaptor designed for multiple tubes can be used (VWR catalog no. 58816-146). Calibrated Micropipettors (accurate range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 \mathrm { - } 1 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { - } 1 0 0 0 ~ \mu \mathrm { L }$ ) Sterile DNase-free filter-blocked or positive displacement micropipettor tips $( 1 - 1 0 ~ \mu \mathrm { L }$ , $1 0 \mathrm { - } 1 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { \cdot } 1 0 0 0 \mu \mathrm { L } $ ) Ice or cooling block for $1 . 5 \mathrm { m L }$ and SmartCycler $^ \mathrm { \textregistered }$ tubes • Microcentrifuge for SmartCycler $^ \mathrm { \textregistered }$ tubes • SmartCycler $^ \mathrm { \textregistered }$ starter system with Dx Software (processing block, user manual, accessory kit, and computer) • Cap removal tool (e.g. MATRIX catalog no. 4469) (optional) Disposable gloves, powderless Stopwatch or timer • BD GeneOhm™ MRSA ACP Assay Definition CD

# Interpretation of Results

The decision algorithm for the BD GeneOhm™ MRSA ACP Assay is embedded in the SmartCycler $^ \mathrm { \textregistered }$ software. The interpretation of assay results is performed according to the following criteria:

<table><tr><td rowspan=1 colspan=1>Assay result reported</td><td rowspan=1 colspan=1>IC result reported</td><td rowspan=1 colspan=1>Interpretation of result</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>No MRSA DNA detected</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>MRSA DNA detected</td></tr><tr><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>Unresolved  inhibitory specimen or reagent failure</td></tr><tr><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not determined due to I-CORE® Module failure(with Warning or Error Codes2 )</td></tr></table>

IC - Internal Control; NA – not applicable; ND – not determined 1 BD GeneOhm™ MRSA ACP Assay results may be used to guide isolation and level of precautions in accordance

with institutional programs and practices.   
2 Refer to the SmartCycler $^ \mathrm { \textregistered }$ Dx Software Operator Manual for interpretation of warning and error codes.

# J. Substantial Equivalence Information:

1. Predicate device name(s): IDI MRSATM Assay Cepheid Xpert MRSA Assay

2. Predicate K number(s): (K042357) and (K070462)

3. Comparison with predicates:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BD GeneOhm™M MRSA ACP Assay(K093346)</td><td colspan="1" rowspan="1">BD GeneOhm™M MRSA Assay(K042357)</td><td colspan="1" rowspan="1">Cepheid Xpert MRSA Assay(K070462)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD GeneOhm™M MRSA ACP Assayis a qualitative in vitro diagnostic test forthe direct detection of methicillin-resistant Staphylococcus aureus (MRSA)DNA from nasal swabs in patients at riskfor nasal colonization. The test utilizespolymerase chain reaction (PCR) for theamplification of MRSA DNA andfluorogenic target-specific hybridizationprobes for the detection of the amplifiedDNA. The BD GeneOhmTM MRSA ACPAssay is intended to aid in the preventionand control of MRSA infections inhealthcare settings. It is not intended todiagnose MRSA infections nor to guideor monitor treatment for MRSAinfections. Concomitant cultures arenecessary only to recover organisms forepidemiological typing or for furthersusceptibility testing.</td><td colspan="1" rowspan="1">BD GeneOhmTM MRSA assay is aqualitative in vitro diagnostic test for thedirect detection of nasal colonization bymethicillin-resistant   Staphylococcusaureus (MRSA) to aid in the preventionand control of MRSA infections inhealthcare settings. The test performedon the Smart Cycler® instrument with anasal swab specimen from patients atrisk for colonization, utilizes polymerasechainreaction (PCR)for theamplification of MRSA DNA andfluorogenic target-specific hybridizationprobes for the detection of the amplifiedDNA. IDI-MRSATM asSay is notintended to diagnose MRSA infectionsnor to guide or monitor treatment forMRSA infections. Concomitant culturesare necessary only to recover organismsfor epidemiological typing or for furthersusceptibility testing.</td><td colspan="1" rowspan="1">The Cepheid Xpert MRSA Assayperformed on the GeneXpert® DxSystem (Xpert MRSA) is aqualitative in vitro diagnostic testdesigned for rapid detection ofmethicillin-resistant Staphylococcusaureus (MRSA) from nasal swabsin patients at risk for nasalcolonization. The test utilizesautomated real-time polymerasechain reaction (PCR) to detectMRSA DNA. The Xpert 2MRSA Assay is intended to aid inthe prevention and control ofMRSA infections in healthcaresettings. The Xpert MRSA Assay isnot intended to diagnose MRSA norto guide or monitor treatment forMRSA infections.Concomittant cultures are necessaryonly to recover organisms forepidemiological typing for furthersusceptibility testing.</td></tr><tr><td colspan="1" rowspan="1">Mode ofidentificationof S. aureus</td><td colspan="1" rowspan="2">Presence of SCCmec cassette (geneticelement that carries the mecA gene) atorfX junction (specific to S. aureus)</td><td colspan="1" rowspan="2">Presence of SCCmec cassette (geneticelement that carries the mecA gene) atorfX junction (specific to S. aureus)</td><td colspan="1" rowspan="2">Presence of the staphylococcalcassette chromosome (SCC)inserted into the SA chromosomalattB site</td></tr><tr><td colspan="1" rowspan="1">Mode ofdetection formethicillinresistance</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Nasal swabs</td><td colspan="1" rowspan="1">Nasal swabs</td><td colspan="1" rowspan="1">Nasal swabs</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCRDetection: Fluorogenic target-specifichybridization</td><td colspan="1" rowspan="1">Amplification: PCRDetection: Fluorogenic target-specifichybridization</td><td colspan="1" rowspan="1">Amplification: PCRDetection: Fluorogenic target-specific hybridization</td></tr><tr><td colspan="1" rowspan="1">AssayControls</td><td colspan="1" rowspan="1">Positive PCR control (DNA from S.aureus ATCC 43300).Negative PCR control (DNA from S.epidermidis ATCC 14990).Internal procedural control</td><td colspan="1" rowspan="1">Positive PCR control (DNA from S.aureus ATCC 43300).Negative PCR control (DNA from S.epidermidis ATCC 14990).Internal procedural control</td><td colspan="1" rowspan="1">Positive: KWICK-STIK™M fromMicroBiologis, Catalog # 0158MRSA (ATCC 700699)Negative: KWICK-STIK™M fromMicroBiologis, Catalog # 0371MSSE (methicillin sensitiveStaphylococcus epidermidis, ATCC12228)</td></tr><tr><td colspan="1" rowspan="1">Interpretationof test results</td><td colspan="1" rowspan="1">Automated (Diagnostic software ofSmartCycler® system)</td><td colspan="1" rowspan="1">Automated (Diagnostic software ofSmartCycler® system)</td><td colspan="1" rowspan="1">GeneXpert® Dx System instrument</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable.

# L. Test Principle:

The lysis of bacterial cells in nasal swab specimens is performed using the BD GeneOhm™ MRSA ACP Lysis kit that incorporates enzymatic Achromopeptidase (ACP) lysis technology for sample preparation of nasal swab specimens used with the BD GeneOhm™ MRSA ACP Assay. Lysis Reagent is reconstituted with Lysis Diluent and aliquoted into single use lysis tubes. A nasal swab specimen is collected and transported to the laboratory using the recommended swab collection/transport device. The nasal swab is introduced and broken into a Sample Buffer tube. An aliquot of the liquid sample suspension is then transferred into the tube containing the Lysis Reagent and incubated to break the bacterial cell wall and release the genetic material.

An aliquot of the lysed sample is then processed with the BD GeneOhm™ MRSA ACP Assay. An aliquot of the lysate is added to prepared PCR reagents which contain MRSA-specific primers that will amplify in the presence of specific genetic target. The assay also includes an Internal Control (IC) to monitor for the presence of inhibitors in the PCR reaction and to confirm the integrity of assay reagents. Controls and specimen lysates are added to disposable reaction tubes and placed in the SmartCycler II instrument. The amplification, detection and results interpretation are automatically performed by the SmartCycler II software.

The primers (1 forward primer and 5 reverse primers) and probes (2 molecular beacon probes) in the BD GeneOhm™ MRSA ACP Assay detect a proprietary sequence of the junction area of the staphylococcal chromosomal cassette (SCCmec) adjacent to the integration site, called MREJ for Mec Right Extremity Junction, indicating the presence of MRSA DNA. Amplification of IC and MRSA DNA are detected using specific hybridization probes that bind to a specific sequence of the amplified target. Differentiation of MRSA DNA and IC is done using molecular beacon probes which contain different fluorometric properties. The molecular beacon-target hybrid fluoresces at a different wavelength for MRSA and IC and the emitted light from this reaction is measured by the SmartCycler II instrument. MRSA and/or IC specific amplicons are detected simultaneously in two different fluorescence channels on the SmartCycler II and can therefore be differentiated. The operation of the SmartCycler II instrument is based on the proprietary microprocessor-controlled $\mathrm { I - C O R E } ^ { \mathbb { \mathrm { ‰} } }$ (Intelligent Cooling/Heating Optical Reaction) module.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility and Precision of the BD GeneOhm™ MRSA ACP Assay using simulated nasal specimens and the recommended processing techniques described in the BD GeneOhm™ MRSA ACP Lysis Kit and BD GeneOhm™ MRSA ACP Assay package inserts were evaluated in three studies: lot-to-lot (LTL) reproducibility, site-tosite (STS) reproducibility, and assay precision.

The MRSA strains used in the precision/reproducibility panels were Staphylococcus aureus ATCC 43300 (MREJ type ii) and IDI-19 (MREJ type vii). Assuming that a McFarland 5 corresponds to $1 . 5 \mathrm { E } { + 0 9 }$ CFU/mL, serial dilutions were prepared from a semi-logarithmic broth culture in TE 1X (Tris10mM, EDTA 1mM) supplemented with glycerol $( 1 0 \% )$ to obtain the required bacterial concentrations. Cell numbers in bacterial suspensions were verified by colony counts. A bacterial suspension of Staphylococcus epidermidis (ATCC 14990) was used to simulate the flora present in the nose (simulated negative nasal matrix). The S. epidermidis bacterial suspension was prepared in the same manner as described for the MRSA suspensions. Final concentrations of S. epidermidis for the negative nasal matrix were between $1 . 2 5 { + } \mathrm { E } 3$ and $2 . 0 7 \mathrm { + E 3 }$ CFU/PCR reaction. Description of the panel members and their respective concentrations were summarized in the following table:

<table><tr><td rowspan=1 colspan=2>Panel Member</td><td rowspan=1 colspan=1>MRSA Concentration (multiples of LoD)</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii High Negative 100</td><td rowspan=1 colspan=1>HN100 type ii</td><td rowspan=1 colspan=1>0.01 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii High Negative 100</td><td rowspan=1 colspan=1>HN100 type vii</td><td rowspan=1 colspan=1>0.01 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii High Negative 10</td><td rowspan=1 colspan=1>HN10 type ii</td><td rowspan=1 colspan=1>0.1 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii High Negative 10</td><td rowspan=1 colspan=1>HN10 type vii</td><td rowspan=1 colspan=1>0.1 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii Low Positive</td><td rowspan=1 colspan=1>LP type ii</td><td rowspan=1 colspan=1>1 to &lt;2 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii Low Positive</td><td rowspan=1 colspan=1>LP type vii</td><td rowspan=1 colspan=1>1 to &lt;2 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii Moderate Positive</td><td rowspan=1 colspan=1>MP type ii</td><td rowspan=1 colspan=1>2 to 5 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii Moderate Positive</td><td rowspan=1 colspan=1>MP type vii</td><td rowspan=1 colspan=1>2 to 5 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA Negative with S. epidermidis only</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0 X LoD</td></tr></table>

Panels of 27 tubes (labeled 01R to 27R) were prepared for the reproducibility studies. Panels of 18 tubes (labeled 01R to 08R, and 14R to 23R) were prepared for the precision study. The tubes for the precision study contain the same MRSA target concentration as the corresponding tubes for the reproducibility studies (Table 5.1-2). All positive and high negative samples were prepared with $5 0 ~ \mu \mathrm { L }$ of a MRSA suspension and $5 0 \mu \mathrm { L }$ of a $S .$ . epidermidis suspension. Negative samples were prepared with $1 0 0 \mu \mathrm { L }$ of a $S .$ . epidermidis suspension. All aliquots were frozen at ${ } _ { - 8 0 ^ { \circ } \mathrm { C } }$ prior to testing. Detailed tube descriptions for panel members are presented in the following table:

<table><tr><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>Panel Category</td><td rowspan=1 colspan=1>Tube ID of Panel Members</td></tr><tr><td rowspan=4 colspan=1>MRSA ATCC43300MREJ type ii</td><td rowspan=1 colspan=1>HN100 type ii</td><td rowspan=1 colspan=1>07R, 12R, 14R</td></tr><tr><td rowspan=1 colspan=1>HN10 type ii</td><td rowspan=1 colspan=1>19R,, 21R, 26R,</td></tr><tr><td rowspan=1 colspan=1>LP type ii</td><td rowspan=1 colspan=1>02R,08R, 10R</td></tr><tr><td rowspan=1 colspan=1>MP type ii</td><td rowspan=1 colspan=1>03R,06R, 09R</td></tr><tr><td rowspan=4 colspan=1>MRSAIDI-19MREJ type vii</td><td rowspan=1 colspan=1>HN100 type vii</td><td rowspan=1 colspan=1>05R,13R,15R</td></tr><tr><td rowspan=1 colspan=1>HN10 type vii</td><td rowspan=1 colspan=1>16R,23R, 25R</td></tr><tr><td rowspan=1 colspan=1>LP type vii</td><td rowspan=1 colspan=1>01R,04R,11R</td></tr><tr><td rowspan=1 colspan=1>MP type vii</td><td rowspan=1 colspan=1>18R,20R, 24R</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>17R,22R, 27R</td></tr></table>

# Lot-to-Lot (LTL) Reproducibility Study

Three BD GeneOhm™ MRSA ACP Lysis kit lots, 09T09003, 09T09004, and 09T09005, and three BD GeneOhm™ MRSA ACP Assay kit lots, 03T09120z, 03T09121z, and 03T09122z, were used for the LTL Reproducibility Study. The LTL Reproducibility Study was performed in- house at BD Diagnostics GeneOhm (GSCI). Thirty pre-labeled reproducibility panels were used in the study. Each panel consisted of 27 tubes labeled 01R to 27R as described previously. The study was performed on 15 distinct days (consecutive or not), wherein each day two panels were tested, one by each of the two technologists. Individual boxes containing the required quantity of frozen specimens for a single day’s testing were provided, i.e. 2 series of 27 tubes in each box. Specimens were thawed on a cooling block, vortexed, and spun at low speed. The contents $( 1 0 0 \mu \mathrm { L } )$ of each tube were completely absorbed onto a sterile rayontipped swab. The swab was then transferred directly into a labeled Sample Preparation Buffer tube and processed according to the BD GeneOhm™ MRSA ACP Lysis Kit and the BD GeneOhmTM MRSA ACP Assay package inserts. Appropriate run controls as well as SPCs were processed with each run.

The qualitative LTL reproducibility summary data are shown for each MREJ type for all Lots (pooled Days, Runs (Technologists) and Replicates) in the following table:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">PanelMember ID</td><td colspan="1" rowspan="1">MRSAtype iiHighNegative100</td><td colspan="1" rowspan="1">MRSAtype viiHighNegative100</td><td colspan="1" rowspan="1">MRSAtype iiHighNegative10</td><td colspan="1" rowspan="1">MRSAtype viiHighNegative10</td><td colspan="1" rowspan="1">MRSAtype iiLowPositive</td><td colspan="1" rowspan="1">MRSAtype viiLowPositive</td><td colspan="1" rowspan="1">MRSAtype iiModeratePositive</td><td colspan="1" rowspan="1">MRSAtype viiModeratePositive</td><td colspan="1" rowspan="1">Negative(S.epidermidis)</td><td colspan="1" rowspan="2">Total %Agreement</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">BacterialConcentration</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">1-&lt;2XLoD</td><td colspan="1" rowspan="1">1-&lt;2XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Lysis Kit Lot/Assay Kit Lot09T09003/03T09120z</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">21/30(70.0%)</td><td colspan="1" rowspan="1">6/30(20.0%)</td><td colspan="1" rowspan="1">30/30(100.0%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">235/270(87.0%)</td></tr><tr><td colspan="1" rowspan="1">Lysis Kit Lot/As Kit Lot09T09004/03T121z</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">27/30(90.0%)</td><td colspan="1" rowspan="1">27/30(90.0%)</td><td colspan="1" rowspan="1">22/30(73.3%)</td><td colspan="1" rowspan="1">6/30(20.0%)</td><td colspan="1" rowspan="1">28/30(93.3%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">230/270(85.2%)</td></tr><tr><td colspan="1" rowspan="1">Lysis Kit Lot/Assay Kit Lot0T09005/</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">27/30(90.0%)</td><td colspan="1" rowspan="1">26/30(86.7%)</td><td colspan="1" rowspan="1">16/30(53.3%)</td><td colspan="1" rowspan="1">4/30(13.3%)</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">222/270(82.2%)</td></tr><tr><td colspan="1" rowspan="1">03T09122z</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">TotalAgreementwith Expectedresult</td><td colspan="1" rowspan="1">83/90(92.2%)</td><td colspan="1" rowspan="1">82/90(91.1%)</td><td colspan="1" rowspan="1">59/90(65.6%)</td><td colspan="1" rowspan="1">16/90(17.8%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">687/810(84.8%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">84.6% -96.8%</td><td colspan="1" rowspan="1">83.2% -96.1%</td><td colspan="1" rowspan="1">54.8% -75.3%</td><td colspan="1" rowspan="1">10.5% -27.3%</td><td colspan="1" rowspan="1">90.6% -99.3%</td><td colspan="1" rowspan="1">96.0% -100%</td><td colspan="1" rowspan="1">96.0% -100%</td><td colspan="1" rowspan="1">96.0% -100%</td><td colspan="1" rowspan="1">96.0% -100%</td><td colspan="1" rowspan="1">82.3%-87.3%</td></tr></table>

Note: For the High Negative panel members, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negativ results. For the Low Positive and Moderate Positive panel members, the expected assay result was deemed to be positive. Therefore, percent agreement was calculated for positive results.

The quantitative LTL reproducibility summary data of the PCR parameters (CT, SDP, and EP) are shown for each MREJ type across Lots, Days, Runs (Technologists), and Replicates (all pooled) in the following table:   

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=2>PanelMember ID</td><td rowspan=2 colspan=1>MRSAtypee iiHighNegative100</td><td rowspan=2 colspan=1>MRSAtype viiHighNegative10</td><td rowspan=2 colspan=1>MRSAtype eiiHighNegative10</td><td rowspan=2 colspan=1>MRSAtype viHighN Ngative10</td><td rowspan=2 colspan=1>MRSAtype iiLlowPositive</td><td rowspan=2 colspan=1>MRSAtyppe viiLowPositive</td><td rowspan=2 colspan=1>MRSAtype ii Moderateositive</td><td rowspan=1 colspan=1>MRSAtyype vi Moderate</td><td rowspan=2 colspan=1>Negative(S.epiermidis)</td></tr><tr><td rowspan=1 colspan=1>ositive</td></tr><tr><td rowspan=1 colspan=2>Bacterial Concentration</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>2-5 XLoD</td><td rowspan=1 colspan=1>2-5XLoD</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=11 colspan=1>CT</td><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>34.8</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=11 colspan=1>SDP</td><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>67.2</td><td rowspan=1 colspan=1>44.7</td><td rowspan=1 colspan=1>80.3</td><td rowspan=1 colspan=1>89.8</td><td rowspan=1 colspan=1>117.5</td><td rowspan=1 colspan=1>121.6</td><td rowspan=1 colspan=1>142.0</td><td rowspan=1 colspan=1>158.4</td><td rowspan=1 colspan=1>158.4</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>17.9</td><td rowspan=1 colspan=1>16.1</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>11.5</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>11.3</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>7.3</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>18.7</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>10.1</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>45.9</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>6.4</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>4.7</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>11.5</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>7.3</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>6.9</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>17.2</td><td rowspan=1 colspan=1>13.3</td><td rowspan=1 colspan=1>19.2</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>45.9</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>42.1</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>26.2</td><td rowspan=1 colspan=1>15.2</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>12.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=11 colspan=1>EP</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>60.9</td><td rowspan=1 colspan=1>42.8</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>107.2</td><td rowspan=1 colspan=1>154.6</td><td rowspan=1 colspan=1>191.1</td><td rowspan=1 colspan=1>226.5</td><td rowspan=1 colspan=1>312.5</td><td rowspan=1 colspan=1>320.6</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>47.6</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>33.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>17.0</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>10.3</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>15.9</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>22.6</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>10.9</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Lot</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>9.8</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>3.1</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>40.1</td><td rowspan=1 colspan=1>48.1</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>49.1</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>18.5</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>17.0</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>15.3</td></tr></table>

Note: Values shown for each MREJ types are those obtained for the MRSA target in the samples that gave a positive result and the values shown for the Negative panel member are those obtained for the Internal Control (IC).

In conclusion, statistical analysis demonstrated no statistically significant difference between lots $\mathrm { ( p = 0 . 5 1 ) }$ )

# Site-to-Site (STS) Reproducibility Study

One BD GeneOhm™ MRSA ACP Lysis kit lot, 09T09003, and one BD GeneOhm™ MRSA ACP Assay kit lot, 03T09120z, were used for the STS Reproducibility Study. The lot tested during the STS Reproducibility study was the same as the first lot tested in the LTL Reproducibility Study. Three sites were chosen to conduct the study:

• NorthShore University Health System (EVAN), Site 1 • BD Diagnostics GeneOhm (GSCI), Site 2 Medical College of Wisconsin (WISC), Site 3

Each site was provided with a total of 10 pre-labeled test panels. Each panel consisted of 27 tubes labeled 01R to 27R as described previously. Each site was asked to perform the study on five distinct days (consecutive or not), wherein each day, two panels were tested, one by each of the two technologists. Of note, data obtained from GSCI as part of the LTL Reproducibility were used to represent the GSCI data for the STS Reproducibility study. For EVAN and WISC, individual boxes containing the required quantity of frozen specimens for a single day’s testing were provided, i.e. two series of 27 tubes. All boxes were shipped frozen on dry ice. At GSCI, all panel member tubes were frozen at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ until tested. For testing, specimens were thawed on a cooling block, vortexed, and spun at low speed. The contents $( 1 0 0 \mu \mathrm { L } )$ of each tube were completely absorbed onto a sterile rayon-tipped swab. The swab was then transferred directly into a labeled Sample Preparation Buffer tube and processed according to the BD GeneOhm™ MRSA ACP Lysis Kit and the BD GeneOhm™ MRSA ACP Assay package inserts. Appropriate run controls as well as SPCs were processed with each run.

The qualitative STS reproducibility summary data are shown for each MREJ type for all Sites (pooled Days, Runs (Technologists) and Replicates) in the following table:

<table><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PanelMember ID</td><td colspan="1" rowspan="1">MRSAtype iiHighNegative100</td><td colspan="1" rowspan="1">MRSAtype  viiHighNegative100</td><td colspan="1" rowspan="1">MRSAtype iiHighNegative10</td><td colspan="1" rowspan="1">MRSAtype viiHighNegative10</td><td colspan="1" rowspan="1">MRSAtype iiLowPositive</td><td colspan="1" rowspan="1">MRSAtype viiLlowPositive</td><td colspan="1" rowspan="1">MRSAtype iiModeratePositive</td><td colspan="1" rowspan="1">MRSAtype viiModeratePositive</td><td colspan="1" rowspan="1">Negative(S.epidermidis)</td><td colspan="1" rowspan="2">Total %Agreement</td></tr><tr><td colspan="1" rowspan="1">Bacterial Concentration</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">1-&lt;2 XLoD</td><td colspan="1" rowspan="1">1-&lt;2XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">26/30(86.7%)</td><td colspan="1" rowspan="1">23/30(76.7%)</td><td colspan="1" rowspan="1">17/30(56.7%)</td><td colspan="1" rowspan="1">9/30(30.0%)</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">224/270(83.0%)</td></tr><tr><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">21/30(70.0%)</td><td colspan="1" rowspan="1">6/30(20.0%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">23/270(87.0%)</td></tr><tr><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Agreement withExpected result</td><td colspan="1" rowspan="1">29/30(96.7%)</td><td colspan="1" rowspan="1">23/30(76.7%)</td><td colspan="1" rowspan="1">22/30(73.3%)</td><td colspan="1" rowspan="1">10/30(33.3%)</td><td colspan="1" rowspan="1">25/30(83.3%)</td><td colspan="1" rowspan="1">27/30(90.0%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">226/270(83.7%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total Agreement withExpectedresult</td><td colspan="1" rowspan="1">84/90(93.3%)</td><td colspan="1" rowspan="1">75/90(83.3%)</td><td colspan="1" rowspan="1">60/90(66.7%)</td><td colspan="1" rowspan="1">25/90(27.8%)</td><td colspan="1" rowspan="1">84/90(93.3%)</td><td colspan="1" rowspan="1">87/90(96.7%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">685/810(84.6%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">86.1% -</td><td colspan="1" rowspan="1">74.0%-</td><td colspan="1" rowspan="1">56.0% -</td><td colspan="1" rowspan="1">18.9%-</td><td colspan="1" rowspan="1">86.1% -</td><td colspan="1" rowspan="1">90.6% -</td><td colspan="1" rowspan="1">96.0%-</td><td colspan="1" rowspan="1">96.0% -</td><td colspan="1" rowspan="1">96.0% -100%</td><td colspan="1" rowspan="1">82.1% -</td></tr><tr><td></td><td>97.5%</td><td>90.4%</td><td>76.3%</td><td>38.2%</td><td>97.5%</td><td></td><td>99.3%</td><td>100%</td><td>100%</td><td></td><td>87.1%</td></tr></table>

Note: For the High Negative panel members, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results. For the Low Positive and Moderate Positive panel members, the expected assay result was deemed to be positive. Therefore, percent agreement was calculated for positive results. 1 Eight (8) LP specimens initially reported as negative were positive upon retesting from frozen lysates.

The quantitative STS reproducibility summary data of the PCR parameters (CT, SDP, and EP) are shown for each MREJ type across Sites, Days, Runs (Technologists), and Replicates (all pooled) in the following table:   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>PanelMember ID</td><td rowspan=1 colspan=1>MRSAtypee iiHighNegative100</td><td rowspan=1 colspan=1>MRSAtype viiHighhNegative10</td><td rowspan=1 colspan=1>MRSAtypee iiHighNegative10</td><td rowspan=1 colspan=1>MRSAtype ye viBHighN Ngative10</td><td rowspan=1 colspan=1>MRSAtype ii LowPositive</td><td rowspan=1 colspan=1>MRSAtype viLlowPositive</td><td rowspan=1 colspan=1>MRSAtype ii ModeratePositive</td><td rowspan=1 colspan=1>MRSAtyype viModerateositive</td><td rowspan=1 colspan=1>Negative(S.epidermidis)</td></tr><tr><td rowspan=1 colspan=2>Bacterial Concentration</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>2-5XLoD</td><td rowspan=1 colspan=1>2-5XLoD</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=11 colspan=1>CT</td><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>40.4</td><td rowspan=1 colspan=1>40.9</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>34.9</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=11 colspan=1>SDP</td><td rowspan=1 colspan=2>Mean</td><td rowspan=1 colspan=1>88.0</td><td rowspan=1 colspan=1>83.1</td><td rowspan=1 colspan=1>95.4</td><td rowspan=1 colspan=1>100.9</td><td rowspan=1 colspan=1>126.7</td><td rowspan=1 colspan=1>139.0</td><td rowspan=1 colspan=1>153.6</td><td rowspan=1 colspan=1>152.8</td><td rowspan=1 colspan=1>175.8</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>13.7</td><td rowspan=1 colspan=1>21.1</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>21.7</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>16.6</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>20.9</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>17.0</td><td rowspan=1 colspan=1>14.1</td><td rowspan=1 colspan=1>14.5</td><td rowspan=1 colspan=1>9.5</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>20.0</td><td rowspan=1 colspan=1>10.6</td><td rowspan=1 colspan=1>13.1</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>18.0</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>7.6</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>12.8</td><td rowspan=1 colspan=1>13.0</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>7.2</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>4.1</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>43.2</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>19.5</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>49.1</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>20.7</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>11.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=11 colspan=1>EP</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>83.3</td><td rowspan=1 colspan=1>70.5</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>117.1</td><td rowspan=1 colspan=1>160.7</td><td rowspan=1 colspan=1>215.3</td><td rowspan=1 colspan=1>242.9</td><td rowspan=1 colspan=1>293.2</td><td rowspan=1 colspan=1>361.7</td></tr><tr><td rowspan=2 colspan=1>Within Run</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>46.1</td><td rowspan=1 colspan=1>40.7</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>48.5</td><td rowspan=1 colspan=1>47.4</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>4.4</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>16.5</td><td rowspan=1 colspan=1>13.2</td></tr><tr><td rowspan=2 colspan=1>Between Run Within Day</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>8.9</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>22.6</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>30.3</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>22.3</td><td rowspan=1 colspan=1>19.3</td><td rowspan=1 colspan=1>22.6</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>11.6</td><td rowspan=1 colspan=1>8.4</td></tr><tr><td rowspan=2 colspan=1>Between Day Within Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>19.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>13.6</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>20.8</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.6</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=2 colspan=1>Between Site</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>23.7</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>20.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>13.9</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>6.6</td></tr><tr><td rowspan=2 colspan=1>Overall</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>58.9</td><td rowspan=1 colspan=1>56.9</td><td rowspan=1 colspan=1>60.1</td><td rowspan=1 colspan=1>67.4</td><td rowspan=1 colspan=1>61.3</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>41.6</td><td rowspan=1 colspan=1>45.2</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>26.4</td><td rowspan=1 colspan=1>24.7</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>16.9</td></tr></table>

Note: Values shown for each MREJ types are those obtained for the MRSA target in the samples that gave a positive result and the values shown for the Negative panel member are those obtained for the Internal Control (IC).

In conclusion, after study completion, statistical analysis was conducted as planned on the STS Reproducibility data and demonstrated that the results obtained at WISC (Site 3) were not different than those obtained at EVAN (Site 1) and GSCI (Site 2) $\scriptstyle ( \mathtt { p } = 0 . 4 1 )$ .

# Precision Study

One BD GeneOhm™ MRSA ACP Lysis kit lot, 09T09003, and one BD GeneOhm™ MRSA ACP Assay kit lot, 03T09120z, were used for the Precision Study. The lot tested during the Precision was the same as the first lot tested in the LTL Reproducibility Study. The Precision Study was performed in-house at BD Diagnostics GeneOhm (GSCI). Twenty-four pre-labeled test panels were used for the study. Each panel consisted of 18 tubes labeled 01R to 08R and 14R to 23R as described previously. The study was performed on 12 distinct days (consecutive or not), wherein each day two panels were tested, one by each of the two technologists. Of note, data from the first five days of the LTL Reproducibility Study were used to represent the first five days of the Precision Study. Individual boxes containing the required quantity of frozen specimens for a single day’s testing were provided, i.e. two series of 18 tubes in each box. Specimens were thawed on a cooling block, vortexed, and spun at low speed. The contents $( 1 0 0 \mu \mathrm { L } )$ of each tube were completely absorbed onto a sterile rayon-tipped swab. The swab was then transferred directly into a labeled Sample Preparation Buffer tube and processed according to the BD GeneOhm™ MRSA ACP Lysis Kit and the BD GeneOhmTM MRSA ACP Assay package inserts. Appropriate run controls as well as SPCs were processed with each run.

The qualitative Precision Study summary data are shown for each MREJ type (pooled Days, Runs (Technologists) and Replicates) in the following table:

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>PanelMember ID</td><td rowspan=1 colspan=1>MRSAtype iiHighNegative100</td><td rowspan=1 colspan=1>MRSAtype viiHighNegative100</td><td rowspan=1 colspan=1>MRSAtype iiHighNegative10</td><td rowspan=1 colspan=1>MRSAtype viiHighNegative10</td><td rowspan=1 colspan=1>MRSAtype iiLowPositive</td><td rowspan=1 colspan=1>MRSAtype viiLowPositive</td><td rowspan=1 colspan=1>MRSAtype iiModeratePositive</td><td rowspan=1 colspan=1>MRSAtype vii ModeratePositive</td><td rowspan=1 colspan=1>Negative(S.epidermidis)</td><td rowspan=2 colspan=1>Total %Agreement</td></tr><tr><td rowspan=1 colspan=1>Bacterial Concentration</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.01 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>0.1 XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>1-&lt;2XLoD</td><td rowspan=1 colspan=1>2-5 XLoD</td><td rowspan=1 colspan=1>2-5 XLoD</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement withExpected result</td><td rowspan=1 colspan=1>45/48(93.8%)</td><td rowspan=1 colspan=1>47/48(97.9%)</td><td rowspan=1 colspan=1>31/48(64.6%)</td><td rowspan=1 colspan=1>9/48(18.8%)</td><td rowspan=1 colspan=1>48/48(100%)</td><td rowspan=1 colspan=1>48/48(100%)</td><td rowspan=1 colspan=1>48/48(100%)</td><td rowspan=1 colspan=1>48/48(100%)</td><td rowspan=1 colspan=1>48/48(100%)</td><td rowspan=1 colspan=1>372/432(86.1%)</td></tr></table>

Note: For the High Negative panel members, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results. For the Low Positive and Moderate Positive panel members, the expected assay result was deemed to be positive. Therefore, percent agreement was calculated for positive results.

The quantitative Precision Study summary data of the PCR parameters (CT, SDP, and EP) are shown for each MREJ type across Days, Runs (Technologists), and Replicates (all pooled) in the following table:

<table><tr><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="1">PanelMember ID</td><td colspan="1" rowspan="1">MRSAtype iiHighNegative100</td><td colspan="1" rowspan="1">MRSAtype viHighNegative100</td><td colspan="1" rowspan="1">MRSAtype iHighNegative10</td><td colspan="1" rowspan="1">MRSAtype viiHighNegative10</td><td colspan="1" rowspan="1">MRSAtypee iiLowPositive</td><td colspan="1" rowspan="1">MRSAtype viLowPositive</td><td colspan="1" rowspan="1">MRSAtye iiModeratePositive</td><td colspan="1" rowspan="1">MRSAtyppe vi MooderatePositive</td><td colspan="1" rowspan="1">Negative(S.epidermidis)</td></tr><tr><td colspan="2" rowspan="1">Bacterial Concentration</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.01 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">0.1 XLoD</td><td colspan="1" rowspan="1">1-&lt;2XLoD</td><td colspan="1" rowspan="1">1-&lt;2 XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">2-5 XLoD</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="6">CT</td><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">35.1</td></tr><tr><td colspan="1" rowspan="2">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="2">Between Run Within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">Between Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.2</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="9">SDP</td><td colspan="2" rowspan="1">Mean</td><td colspan="1" rowspan="1">62.8</td><td colspan="1" rowspan="1">62.9</td><td colspan="1" rowspan="1">89.0</td><td colspan="1" rowspan="1">92.3</td><td colspan="1" rowspan="1">128.6</td><td colspan="1" rowspan="1">133.2</td><td colspan="1" rowspan="1">152.1</td><td colspan="1" rowspan="1">164.4</td><td colspan="1" rowspan="1">176.4</td></tr><tr><td colspan="1" rowspan="2">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">24.6</td><td colspan="1" rowspan="1">21.4</td><td colspan="1" rowspan="1">16.1</td><td colspan="1" rowspan="1">20.6</td><td colspan="1" rowspan="1">10.5</td><td colspan="1" rowspan="1">8.0</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">26.6</td><td colspan="1" rowspan="1">16.6</td><td colspan="1" rowspan="1">12.1</td><td colspan="1" rowspan="1">13.5</td><td colspan="1" rowspan="1">6.4</td><td colspan="1" rowspan="1">4.5</td></tr><tr><td colspan="1" rowspan="2">Between Run Within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">22.1</td><td colspan="1" rowspan="1">21.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">3.6</td><td colspan="1" rowspan="1">6.4</td><td colspan="1" rowspan="1">10.5</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">24.0</td><td colspan="1" rowspan="1">16.8</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">2.3</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">5.9</td></tr><tr><td colspan="1" rowspan="2">Between Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">23.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">61.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">26.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">30.4</td><td colspan="1" rowspan="1">16.1</td><td colspan="1" rowspan="1">20.9</td><td colspan="1" rowspan="1">12.3</td><td colspan="1" rowspan="1">13.2</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">61.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">46.4</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">23.6</td><td colspan="1" rowspan="1">12.1</td><td colspan="1" rowspan="1">13.7</td><td colspan="1" rowspan="1">7.5</td><td colspan="1" rowspan="1">7.5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="9">EP</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">62.0</td><td colspan="1" rowspan="1">58.0</td><td colspan="1" rowspan="1">88.8</td><td colspan="1" rowspan="1">104.1</td><td colspan="1" rowspan="1">170.5</td><td colspan="1" rowspan="1">207.3</td><td colspan="1" rowspan="1">242.8</td><td colspan="1" rowspan="1">327.4</td><td colspan="1" rowspan="1">355.0</td></tr><tr><td colspan="1" rowspan="2">Within Run</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">30.0</td><td colspan="1" rowspan="1">42.6</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">27.5</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">19.9</td><td colspan="1" rowspan="1">14.5</td><td colspan="1" rowspan="1">17.5</td><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">7.8</td></tr><tr><td colspan="1" rowspan="2">Between Run Within Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">25.7</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">7.1</td><td colspan="1" rowspan="1">23.6</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">41.1</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">24.7</td><td colspan="1" rowspan="1">20.2</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">9.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">11.6</td></tr><tr><td colspan="1" rowspan="2">Between Day</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">25.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">10.7</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">57.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">28.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">6.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0/0</td></tr><tr><td colspan="1" rowspan="2">Overall</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">40.9</td><td colspan="1" rowspan="1">46.5</td><td colspan="1" rowspan="1">49.5</td><td colspan="1" rowspan="1">30.8</td><td colspan="1" rowspan="1">48.7</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">49.5</td></tr><tr><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">57.3</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">46.0</td><td colspan="1" rowspan="1">44.6</td><td colspan="1" rowspan="1">29.0</td><td colspan="1" rowspan="1">14.9</td><td colspan="1" rowspan="1">20.0</td><td colspan="1" rowspan="1">10.0</td><td colspan="1" rowspan="1">13.9</td></tr></table>

Note: Values shown for each MREJ types are those obtained for the MRSA target in the samples that gave a positive result and the values shown for the Negative panel member are those obtained for the Internal Control (IC).

# Overall

The following table summarizes the qualitative results for the LTL and STS Reproducibility Studies and the Precision Study in percent agreement (with exact $9 5 \%$ confidence intervals) for each panel category (MREJ types pooled).

<table><tr><td colspan="3" rowspan="1">LTL Reproducibility</td></tr><tr><td colspan="1" rowspan="1">Category</td><td colspan="2" rowspan="1">Agreement with 95% CI</td></tr><tr><td colspan="1" rowspan="1">HN100</td><td colspan="1" rowspan="1">91.7% (165/180)</td><td colspan="1" rowspan="1">(86.6%, 95.3%)</td></tr><tr><td colspan="1" rowspan="1">HN10</td><td colspan="1" rowspan="1">41.7% (75/180)</td><td colspan="1" rowspan="1">(34.4%, 49.2%)</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">98.3% (177/180)</td><td colspan="1" rowspan="1">(95.2%, 99.7%)</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">100.0% (180/180)</td><td colspan="1" rowspan="1">(98.0%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">100.0% (90/90)</td><td colspan="1" rowspan="1">(96.0%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">STS Reproducibility</td></tr><tr><td colspan="1" rowspan="1">Category</td><td colspan="2" rowspan="1">Agreement with 95% CI</td></tr><tr><td colspan="1" rowspan="1">HN100</td><td colspan="1" rowspan="1">88.3% (159/180)</td><td colspan="1" rowspan="1">(82.7%, 92.6%)</td></tr><tr><td colspan="1" rowspan="1">HN10</td><td colspan="1" rowspan="1">47.2% (85/180)</td><td colspan="1" rowspan="1">(39.8%, 54.8%)</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">95.0% (171/180)</td><td colspan="1" rowspan="1">(90.7%, 97.7%)</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">100.0% (180/180)</td><td colspan="1" rowspan="1">(98.0%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">100.0% (90/90)</td><td colspan="1" rowspan="1">(96.0%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Precision</td></tr><tr><td colspan="1" rowspan="1">Category</td><td colspan="2" rowspan="1">Agreement with 95% CI</td></tr><tr><td colspan="1" rowspan="1">HN100</td><td colspan="1" rowspan="1">95.8% (92/96)</td><td colspan="1" rowspan="1">(89.7%, 98.9%)</td></tr><tr><td colspan="1" rowspan="1">HN10</td><td colspan="1" rowspan="1">41.7% (40/96)</td><td colspan="1" rowspan="1">(31.7%, 52.2%)</td></tr><tr><td colspan="1" rowspan="1">LP</td><td colspan="1" rowspan="1">100.0% (96/96)</td><td colspan="1" rowspan="1">(96.2%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1">MP</td><td colspan="1" rowspan="1">100.0% (96/96)</td><td colspan="1" rowspan="1">(96.2%, 100.0%)</td></tr><tr><td colspan="1" rowspan="1">TN</td><td colspan="1" rowspan="1">100.0% (48/48)</td><td colspan="1" rowspan="1">(92.6%, 100.0%)</td></tr></table>

For Site-to-Site Reproducibility, the overall percent agreement was $100 \%$ for MP and Neg categories, $9 5 . 0 \%$ for LP, $8 8 . 3 \%$ and $4 7 . 2 \%$ negative agreement for HN1:100 and HN1:10 categories, respectively. (Refer to Table 1)

Table 1: Site-To-Site Reproducibility Study Results using One Lot   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=4>Site</td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=3>Overall Percent</td><td rowspan=2 colspan=3>CT Values1</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=3>Agreement</td><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=2>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>HN1002</td><td rowspan=1 colspan=1>49/60</td><td rowspan=1 colspan=1>81.7%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>52/60</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=2>159/180</td><td rowspan=1 colspan=1>88.3%</td><td rowspan=1 colspan=1>40.8</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>HN102</td><td rowspan=1 colspan=1>26/60</td><td rowspan=1 colspan=1>43.3%</td><td rowspan=1 colspan=1>27/60</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>32/60</td><td rowspan=1 colspan=1>53.3%</td><td rowspan=1 colspan=2>85/180</td><td rowspan=1 colspan=1>47.2%</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3.8</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>52/603</td><td rowspan=1 colspan=1>86.7%</td><td rowspan=1 colspan=2>171/180</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.8</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=2>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.6</td></tr></table>

1For the Neg category, CT values reported are for the internal control. For other categories, CT values reported are for the MRSA target. 2 For the High Negative categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results. 3 Eight (8) LP specimens initially reported as negative were positive upon retesting from frozen lysates.

For Lot-to-Lot Reproducibility, the overall percent agreement was $100 \%$ for MP and Neg categories, $9 8 . 3 \%$ for LP, $9 1 . 7 \%$ and $4 1 . 7 \%$ negative agreement for HN1:100 and HN1:10 categories, respectively. (Refer to Table 2)

Table 2: Lot-To-Lot Reproducibility Study Results using three Lot   

<table><tr><td rowspan=3 colspan=1>Category</td><td rowspan=1 colspan=6>Lot</td><td rowspan=3 colspan=2>Overall PercentAgreement</td><td rowspan=2 colspan=3>CT Values1</td></tr><tr><td rowspan=1 colspan=2>Lot 1</td><td rowspan=1 colspan=2>Lot 2</td><td rowspan=1 colspan=2>Lot 3</td></tr><tr><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=2>PercentAgreement</td><td rowspan=1 colspan=1>OverallMean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1>HN1002</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>54/60</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>53/60</td><td rowspan=1 colspan=1>88.3%</td><td rowspan=1 colspan=1>165/180</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>2.7</td></tr><tr><td rowspan=1 colspan=1>HN102</td><td rowspan=1 colspan=1>27/60</td><td rowspan=1 colspan=1>45.0%</td><td rowspan=1 colspan=1>28/60</td><td rowspan=1 colspan=1>46.7%</td><td rowspan=1 colspan=1>20/60</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>75/180</td><td rowspan=1 colspan=1>41.7%</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>177/180</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.4</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.5</td></tr></table>

1For the Neg category, CT values reported are for the internal control. For other categories, CT values reported are for the MRSA target. 2 For the High Negative categories, the expected assay result was deemed to be negative. Therefore, percent agreement was calculated for negative results.

Cycle threshold $\mathrm { ( C t ) }$ , an internal criteria used to determine a final assay result, was selected as an additional means of assessing assay reproducibility. Overall mean Ct values with variance components (SD and $\% C V$ ) are shown in Tables 1 and 2.

In conclusion, the qualitative data generated during the LTL and STS Reproducibility Studies of the BD GeneOhm™ MRSA ACP Assay for the low and moderate positive, and true negative panel categories met the acceptance criteria. No qualitative acceptance criteria were defined for the high negative panel categories. No quantitative PCR parameter acceptance criteria were defined for any categories in either the LTL or STS reproducibility studies.

The qualitative data generated during the Precision Study of the BD GeneOhm™ MRSA ACP Assay for the low and moderate positive and true negative panel categories met the acceptance criteria. No acceptance criteria were defined for the high negative panel categories. The coefficient of variation of the quantitative PCR parameter data generated during the Precision Study of the BD GeneOhm™ MRSA ACP Assay for the low and moderate positive, and true negative panel categories met the acceptance criteria. No acceptance criteria were defined for the high negative panel categories.

This precision/reproducibility data also demonstrated that as expected with all real-time PCR assays, the BD GeneOhm™ MRSA ACP Assay may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration. This limitation should be addressed by including the following statement in the Limitation section of the BD GeneOhm™ MRSA ACP Assay Instructions for Use: “As with all PCR based in vitro diagnostic tests, extremely low levels of target below the LoD of the assay may be detected, but results may not be reproducible.”

$^ b$ . Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Internal Control (IC)

The BD GeneOhm™ MRSA ACP Assay includes an internal control (IC). The purpose of the IC in the BD GeneOhmTM MRSA ACP Assay is to monitor the presence of PCR inhibitors in clinical samples as part of a comprehensive control scheme. The strategy takes advantage of the NC IC $\%$ feature of the SmartCycler $\textsuperscript { \textregistered }$ software. This feature allows the identification of an unresolved result when the internal control signal does not reach a preset threshold. The NC $1 C \%$ value is defined as the EP value of the internal control (IC) in a given assay tube expressed as a percentage of the EP value of the IC of the negative control (NC) of the same run. It is used to determine the level of signal reduction of the internal control that is associated with the loss of a weak positive signal due to the presence of PCR inhibitors. To avoid false-negative results from possibly inhibited clinical specimens, the NC $1 C \%$ cut-off of the BD GeneOhm™MRSA ACP Assay algorithm was set to $36 \%$ . A verification study with specimens from clinical studies validated the NC IC $\%$ cut-off. (Refer to section M (j) of this review).

Positive and Negative Controls

The BD GeneOhm™ MRSA ACP Assay includes a positive control and negative control. The positive control is purified full length double stranded genomic DNA from Staphylococcus aureus ATCC 43300 (66 equivalent genome copies per PCR reaction). The negative control is purified full length double stranded genomic DNA from Staphylococcus epidermidis ATCC 14990 (2,720 equivalent genome copies per PCR reaction). S. aureus ATCC 43300 is a well characterized type strain. It is a MREJ genotype ii. It represents the most frequent MREJ type observed in clinical specimens. It yields the longest amplicon of the assay, thus presumably the most sensitive to any stress. S. epidermidis ATCC 14990 is a coagulase negative staphylococci that does not carry the mecA gene and is a species closely related to S. aureus. A verification study with specimens from clinical studies validated the PC and NC cut-offs. (Refer to section M (j) of this review).

Positive and negative controls are assay controls, as they go through the lysis step. Thus they control the DNA integrity from sample preparation to the PCR assay. They are used to monitor assay performance and validate PCR runs. The positive control is an assay run control. An invalid control invalidates the run. It is used in combination with the internal control to verify reagent and system functionality. The negative control is also an assay run control. An invalid control invalidates the run. The negative control is used to detect reagent or environmental contamination.

Specimen Processing Controls

During the clinical trial, in addition to the assay positive and negative controls, Specimen Processing Controls (SPCs) were tested in each run in which specimens were processed through the lysis step.

Since the BD GeneOhm™ MRSA ACP Assay is a multiplex assay with several primers and probes, not all of them are directly controlled by the IC. In the BD GeneOhm™ MRSA ACP assay, two beacon probes (XSau-B5-A0 and XSau-B8-A0) and four primers (XSau325, mecII579, mecIII519, and mecVII560) are not directly controlled by the IC. However, XSau-B5-A0, mecII579 and XSau325 are directly controlled by the assay positive control. Indeed, Control DNA consists of genomic DNA from MRSA MREJ type ii that is amplified with XSau325 and mecII579 and detected by the beacon probe XSauB5-A0. This leaves XSau-B8-A0, mecIII519 and mecVII560 not controlled directly in the BD GeneOhm™MRSA ACP assay. Primer mecIII519 is designed to amplify MRSA type iii, primer mecVII560 and probe X-SauB8-A0 are designed to ensure amplification and detection of MRSA type vii.

The following language should be included in the product package insert to address this issue: “A reference MRSA strain (e.g. ATCC 43300) or a well characterized MRSA clinical isolate may be used as a positive SPC; MRSA MREJ type iii and vii strains, if available, may be used as additional positive SPC, to monitor assay probes and primers not directly controlled in the assay. A strain of methicillin susceptible Staphylococcus aureus (e.g. ATCC 25923) or any other non-aureus Staphylococci (e.g. Staphylococcus epidermidis ATCC 14990) may be used as a negative SPC.”

# Recovery of Methicillin-resistant S. aureus Cells from Collection Device

An analytical study was carried out to demonstrate viability of MRSA when low bacterial loads are present in collection devices under varying storage conditions. Nasal swabs obtained from patients were characterized with the BD GeneOhm™ MRSA ACP Assay as described in the package insert. Negative samples were pooled and used as negative nasal matrix to prepare study samples. The following 3 MRSA reference strains were used in this study to take into account intra-species variability: ATCC43300 (MREJ type ii), CCRI-9773 (MREJ type vii), and NRS384 (MREJ type ii/PFGE USA300).

Cultures were prepared and used to obtain bacterial suspensions of various concentrations. Twenty-five microliters $( 2 5 ~ \mu \mathrm { L } )$ of these suspensions were used to spike $5 0 ~ \mu \mathrm { L }$ of negative nasal matrix. This produced study samples with volumes of 75 $\mu \mathrm { L }$ . The study samples that correspond to target concentrations consistent with $2 { \cdot } 5 \mathrm { X }$ LoD were used to determine the viability of MRSA when low bacterial loads are present in the collection device.

Three collection devices were tested, including Liquid Stuart (LS), Liquid Amies (LA) and Amies Gel w/o charcoal (AG). For each collection device, study samples were tested in triplicate for each storage condition. Collection devices were used to absorb the $7 5 ~ \mu \mathrm { L }$ study samples. After complete absorption, the swabs were inserted into their respective transport tube and stored under the 7 following conditions:

At $4 \ { } ^ { \circ } \mathrm { C }$ for 0, 24, 48 hours and 5 days At $2 5 ~ ^ { \circ } \mathrm { C }$ for 0, 24, 48 hours

After storage, the swab was removed from the transport tube, inserted and broken in a tube containing $6 0 0 \mu \mathrm { L }$ of ID Broth and discharged (vortex-mixing for 60 seconds). Then, $1 0 0 \mu \mathrm { L }$ of the eluted sample suspension were spread on BBL™ CHROMagar™ MRSA plates. The plates were incubated at $3 5 \mathrm { { ^ \circ C } }$ for 24 to 48 hours and mauve-colored colonies, corresponding to MRSA, were counted.

Results of cultures performed after storage of the study samples are shown in the following tables:

# Storage at $\bf { 2 5 ^ { \circ } C }$ -Growth on selective media per condition

<table><tr><td rowspan=2 colspan=2>Collection device type</td><td rowspan=1 colspan=3>ATCC 43300</td><td rowspan=1 colspan=3>CCRI-9773</td><td rowspan=1 colspan=3>NRS384</td></tr><tr><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td></tr><tr><td rowspan=2 colspan=1>Stuart Liquid</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Amie Liquid</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Amies Gel w/0charcoal</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr></table>

# Storage at ${ \bf 4 } ^ { \circ } { \bf C }$ -Growth on selective media per condition

<table><tr><td rowspan=2 colspan=2>Collection device type</td><td rowspan=1 colspan=4>ATCC 43300</td><td rowspan=1 colspan=2>CCRI</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>NRS384</td></tr><tr><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>Oh</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td><td rowspan=1 colspan=1>5 days</td><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>24h</td><td rowspan=1 colspan=1>48h</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=2 colspan=1>Stuart Liquid</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Amie Liquid</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=2 colspan=1>Amies Gel w/ocharcoal</td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Growth</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr></table>

In conclusion, all collection devices tested exhibited preservation of viable MRSA for all strains tested at 2-5 X LoD under all seven storage conditions.

# ACP Lysis Compatibility with PCR

An analytical study was carried out to demonstrate that lysis using the BD GeneOhm™ MRSA ACP lysis kit does not interfere with the PCR reaction of the BD GeneOhm™ MRSA ACP Assay. Positive Controls and Negative Controls were prepared following the instructions in the BD GeneOhm™ MRSA ACP Assay package insert.

Reconstitution of ACP was prepared following the instructions in the BD GeneOhm™ MRSA ACP Lysis Kit package insert. Master Mix was prepared following the instructions in the BD GeneOhm™ MRSA ACP Assay package insert. Positive and Negative controls were tested using three lots of ACP and three lots of Sample Buffer. ACP lysis compatibility with PCR was demonstrated when specimens submitted to the lysis with ACP displayed the expected assay result and the appropriate CT relative value $( 9 5 \% - 1 0 5 \% )$ when compared to specimens not submitted to the lysis. The formula used to calculate the relative CT value was the following:

200 - (average CT $\div$ average CT (reference) x 100).

The following table shows the CT values recorded in the FAM channel (target) for both conditions (PC with Sample Buffer (1 x TE) or PC submitted to lysis process (ACP)). No significant change in relative CT value for processed PC was observed in the FAM channel in comparison with non processed PC. In addition, all PC results were valid.

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=2>PC</td></tr><tr><td rowspan=1 colspan=1>1 X TE</td><td rowspan=1 colspan=1>ACP</td></tr><tr><td rowspan=3 colspan=1>FAM</td><td rowspan=2 colspan=1>CT (cycles)</td><td rowspan=2 colspan=1>MeanSD</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>34.9</td></tr><tr><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0,35</td></tr><tr><td rowspan=1 colspan=2>CT Relative value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Result (valid)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

The following table shows the CT values recorded in the TET channel (internal control) obtained for both conditions (NC with Sample Buffer $( 1 \mathrm { ~ x ~ T E ) }$ or NC submitted to lysis process (ACP)). No significant change in relative CT value for processed NC was

observed in the TET channel in comparison with non processed NC. In addition, no contamination of Lysis reagent was observed since all processed NC were negative and all NC results were valid.

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=2>NC</td></tr><tr><td rowspan=1 colspan=1>1 XTE</td><td rowspan=1 colspan=1>ACP</td></tr><tr><td rowspan=4 colspan=1>TET</td><td rowspan=2 colspan=1>CT (cycles)</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>35.1</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.26</td></tr><tr><td rowspan=1 colspan=2>CT Relative value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td></tr><tr><td rowspan=1 colspan=2>Result (valid)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr></table>

In conclusion, comparison of data of the BD GeneOhm™ MRSA ACP assay performed using processed and unprocessed PC and NC demonstrated that there is no inhibitory effect of the BD Gene $\mathrm { O h m ^ { T M } }$ MRSA ACP Lysis kit sample preparation components on positive and negative controls. In addition, no unexpected positive signal was obtained during testing which revealed the absence of contaminants in the BD GeneOhm™ MRSA ACP Lysis kit components.

# Lysis Efficiency Study

An analytical study was carried out to determine the lysis efficiency of ACP for MRSA strains following the BD GeneOhm™ MRSA ACP Lysis kit procedures.

Three (3) MRSA reference strains listed in the following table were used in this study to take into account intra-species variability

<table><tr><td rowspan=1 colspan=1># IDI</td><td rowspan=1 colspan=1>Strains</td><td rowspan=1 colspan=1>MREJ type</td></tr><tr><td rowspan=1 colspan=1>IDI-38</td><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>i</td></tr><tr><td rowspan=1 colspan=1>IDI-19</td><td rowspan=1 colspan=1>CCRI-9773</td><td rowspan=1 colspan=1>vii</td></tr><tr><td rowspan=1 colspan=1>IDI-2957</td><td rowspan=1 colspan=1>NRS384</td><td rowspan=1 colspan=1>ii ( PFGE USA300)</td></tr></table>

Cultures were prepared and used to obtain bacterial suspensions at a theoretical concentration of $6 . 5 0 \mathrm { E } { + } 0 7$ CFU/mL for each strain. Actual concentrations were then determined by colony count and were used to calculate the lysis efficiency of the ACP for each MRSA strain tested. The Collection Devices tested in this study were: Liquid Stuart, Liquid Amies and Amies $\mathrm { G e l } \mathrm { w } / \mathrm { o }$ charcoal. Eighteen replicates of each collection device were tested with each MRSA strain. An aliquot of $7 5 ~ \mu \mathrm { L }$ of the $6 . 5 0 \mathrm { E } { + } 0 7$ CFU/mL suspension of MRSA was absorbed by each swab and inserted into their respective transport tubes for a minimum of 30 minutes. Following the 30 minute incubation period, each swab was discharged by vortexing for 60 seconds in $6 0 0 \mu \mathrm { L }$ of Sample Buffer. Then, $9 0 ~ \mu \mathrm { L }$ of the cell suspension were transferred to a lysis tube containing $2 0 \mu \mathrm { L }$ of ACP. The tubes were incubated at $3 7 ^ { \circ } \mathrm { C }$ for 20 minutes. After the 20-minute incubation at $3 7 ^ { \circ } \mathrm { C }$ , $8 9 0 ~ \mu \mathrm { L }$ of trypticase soy broth (TSB) were added to each lysis tube. Tubes were centrifuged at $1 0 0 0 0 \times \mathbf { g }$ for 5 minutes. The supernatant was removed and the pellet was re-suspended in $5 0 ~ \mu \mathrm { L }$ of TSB. Then all $5 0 ~ \mu \mathrm { L }$ of the

suspension was spread on a Blood Agar Plate (BAP). The plates were incubated at $3 5 \mathrm { { ^ \circ C } }$ for 18-24 hours and CFU were counted.

The highest percentage and the lowest percentage of lysis were calculated. The highest percentage was calculated as:

[(the highest initial CFU count observed before lysis - the lowest count after lysis)/the highest initial CFU count observed before lysis] $\mathbf { x } 1 0 0$

The lowest percentage was calculated as:

[(the lowest initial CFU count observed before lysis - the highest count after lysis)/the lowest initial CFU count observed before lysis] x100

The collection swabs (LS, LA and AG transport media) were spiked with a MRSA load of $5 . 8 1 \mathrm { E } \substack { + 0 6 } \pm 9 . 6 8 \mathrm { E } \substack { + 0 5 }$ , $4 . 4 6 \mathrm { E } \substack { + 0 6 } \pm 8 . 1 8 \mathrm { E } \substack { + 0 5 }$ and $7 . 0 5 \mathrm { E } + 0 6 \pm 8 . 8 5 \mathrm { E } + 0 5$ CFU for MRSA strains ATCC 43300, CCRI-9773 and NRS384, respectively. Assuming a $100 \%$ swab discharge in the $6 0 0 \mu \mathrm { L }$ of Sample Buffer, the estimated CFU loads, in the $9 0 \mu \mathrm { L }$ volumes transferred in lysis tubes are of $8 . 7 1 \mathrm { E } \substack { + 0 5 \pm 1 . 4 5 \mathrm { E } \mathopen { + } 0 5 }$ , $6 . 6 9 \mathrm { E } + 0 5 \pm 1 . 2 3 \mathrm { E } + 0 5$ and $1 . 0 6 \mathrm { E } + 0 6 \pm 1 . 3 3 \mathrm { E } + 0 5$ , respectively.

The lysates were plated on BAP and incubated at $3 5 \mathrm { { ^ \circ C } }$ for $1 8 { - } 2 4 \mathrm { h }$ and colonies were counted. A range of counts from 0 to 1900 CFU/plates was obtained. Two plates could not be counted because growth was confluent. Counts of plates containing more than $4 0 0 \mathrm { C F U }$ were estimated. A summary of results obtained for all collection devices used with the three strains tested are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Collection device type</td><td rowspan=1 colspan=1>Lowest CFUcount</td><td rowspan=1 colspan=1>Highest CFUcount</td><td rowspan=1 colspan=1>LowestCFU countafterlysis</td><td rowspan=1 colspan=1>HighestCFU countafterlysis</td><td rowspan=1 colspan=1>Highest% of lysis</td><td rowspan=1 colspan=1>Lowest% of lysis</td></tr><tr><td rowspan=3 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>LA (n=18)</td><td rowspan=3 colspan=1>7.26E+05</td><td rowspan=3 colspan=1>1.02E+06</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>700</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.90</td></tr><tr><td rowspan=1 colspan=1>AG (n=18)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.93</td></tr><tr><td rowspan=1 colspan=1>LS (n=18)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>800</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.89</td></tr><tr><td rowspan=3 colspan=1>CCRI-9773</td><td rowspan=1 colspan=1>LA (n=18)</td><td rowspan=3 colspan=1>5.47E+05</td><td rowspan=3 colspan=1>7.92E+05</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1900</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.65</td></tr><tr><td rowspan=1 colspan=1>AG (n=18)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>660</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.88</td></tr><tr><td rowspan=1 colspan=1>LS (n=18)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.89</td></tr><tr><td rowspan=3 colspan=1>NRS384</td><td rowspan=1 colspan=1>LA (n=17)1</td><td rowspan=3 colspan=1>9.25E+05</td><td rowspan=3 colspan=1>1.19E+06</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.96</td></tr><tr><td rowspan=1 colspan=1>AG (n=18)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.96</td></tr><tr><td rowspan=1 colspan=1>LS (n=17)1</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>100.00</td><td rowspan=1 colspan=1>99.94</td></tr></table>

1 Confluent growth was observed for one of the plate.

In conclusion, the results presented in this study demonstrated that a minimum lysis efficiency of $9 9 . 7 \%$ was obtained across all 3 MRSA strains and all 3 collection devices.

# Preservation of DNA on Collection Device

The preservation of amplifiable DNA on collection devices was evaluated in an analytical study.

Twenty seven Methicillin-resistant S. aureus (MRSA) strains and 27 Methicillinsusceptible Staphylococcus sp. (MSSSp) strains were used in this study. The MRSA strains were of diverse geographical origin and included MREJ types i, ii, iii, iv, v, and vii. For MRSA strains, $7 5 \mu \mathrm { L }$ of a bacterial suspension corresponding to a target concentration of 2-3X LOD was used to determine the preservation of amplifiable DNA on collection devices. For MSSSp strains, $7 5 \mu \mathrm { L }$ of a bacterial suspension corresponding to a theoretical concentration of $1 . 0 0 \mathrm { E } { + 0 7 }$ CFU/mL was used to determine the preservation of amplifiable DNA on collection devices.

Each collection device was tested in triplicate with nine MRSA and nine MSSSp strains at baseline and under 4 different storage conditions. Collection Devices tested in this study were: Liquid Stuart, Liquid Amies and Amies Gel w/o charcoal.

The collection device swabs were used to completely absorb $7 5 \mu \mathrm { L }$ of the relevant bacterial suspension, and then the swabs were inserted into their respective transport tube and stored under the following conditions:

room temperature for 30 minutes $( \mathrm { T } { = } 0 )$ )   
• $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 24 hours   
• $2 5 \pm 2 ^ { \circ } \mathrm { C }$ for 48 hours   
• $2 { - } 8 ^ { \circ } \mathrm { C }$ for 48 hours $2 { - } 8 ^ { \circ } \mathrm { C }$ for 5 days

After incubation, study samples were prepared and analyzed according to the BD GeneOhm™ MRSA Lysis Kit package insert and the BD GeneOhm™ MRSA ACP Assay package insert, respectively. To be considered acceptable, the percentage of conforming results had to be equal to or greater than $9 5 \%$ . Results were calculated for each collection device separately.

Summary results obtained for Liquid Stuart collection device with MRSA strains are presented in the following table:

<table><tr><td rowspan=3 colspan=1>Liquid Stuart Collection Devicewith MRSA strains</td><td rowspan=1 colspan=5>Storage time and temperature</td></tr><tr><td rowspan=2 colspan=1>T=0 (Std)</td><td rowspan=1 colspan=2>25 ± 2°</td><td rowspan=1 colspan=2>2-8°</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=1 colspan=1>Sample results (pos)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Sample results (neg)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total sample tested</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

Summary results obtained for Liquid Amies collection device with MRSA strains are presented in the following table:

<table><tr><td rowspan="2">Liquid Amies Collection Device with MRSA strains</td><td colspan="5">Storage time and temperature</td></tr><tr><td rowspan="2">T=0 (Std)</td><td colspan="2">25 ± 2</td><td colspan="2">2-8 </td></tr><tr><td></td><td>24 hours</td><td>48 hours</td><td>48 hours</td><td>5 days</td></tr><tr><td>Sample results (pos)</td><td>27</td><td>27</td><td>27</td><td>27</td><td>27</td></tr><tr><td>Sample results (neg)</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td></tr><tr><td>Total sample tested</td><td>27</td><td>27</td><td>27</td><td>27</td><td>27</td></tr><tr><td>% of conforming results</td><td>100</td><td>100</td><td>100</td><td>100</td><td>100</td></tr></table>

Summary results obtained for Amies Gel without Charcoal collection device with MRSA strains are presented in the following table:

<table><tr><td rowspan=3 colspan=1>Amies Gel without Charcoal CollectionDevice with MRSA strains</td><td rowspan=1 colspan=5>Storage time and temperature</td></tr><tr><td rowspan=2 colspan=1>T=0 (Std)</td><td rowspan=1 colspan=2>25 ± 2</td><td rowspan=1 colspan=2>2-8°</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=1 colspan=1>Sample results (pos)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Sample results (neg)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total sample tested</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td></tr></table>

Summary results obtained for Liquid Stuart collection device with MSSSp strains are presented in the following table:

<table><tr><td rowspan=3 colspan=2>Liquid Stuart Collection Devicewith MSSSp strains</td><td rowspan=1 colspan=5>Storage time and temperature</td></tr><tr><td rowspan=2 colspan=1>T=0 (Std)</td><td rowspan=1 colspan=2>25 ± 2</td><td rowspan=1 colspan=2>2-8 </td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=1 colspan=2>Sample results (neg)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=2 colspan=1>Sample results</td><td rowspan=1 colspan=1>(unr)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>(pos)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total sample tested</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

Summary Results obtained for Liquid Amies collection device with MSSSp strains are presented in the following table:

Summary Results obtained for Amies Gel without Charcoal collection device with MSSSp strains with MSSSp strains are presented in the following table:   

<table><tr><td rowspan=3 colspan=2>Liquid Amies Collection Devicewith MSSSp strains</td><td rowspan=1 colspan=5>Storage time and temperature</td></tr><tr><td rowspan=2 colspan=1>T=0 (Std)</td><td rowspan=1 colspan=2>25 ± 2</td><td rowspan=1 colspan=2>2-8 </td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=1 colspan=2>Sample results (neg)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=2 colspan=1>Sample results</td><td rowspan=1 colspan=1>(unr)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>(pos)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total sample tested</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

<table><tr><td rowspan=3 colspan=2>Amies Gel without Charcoal Collection Devicewith MSSSp strains</td><td rowspan=1 colspan=5>Storage time and temperature</td></tr><tr><td rowspan=2 colspan=1>T=0 (Std)</td><td rowspan=1 colspan=2>25 ± 2</td><td rowspan=1 colspan=2>2-8°</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>48 hours</td><td rowspan=1 colspan=1>5 days</td></tr><tr><td rowspan=1 colspan=2>Sample results (neg)</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=2 colspan=1>Sample results</td><td rowspan=1 colspan=1>(unr)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>(pos)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Total sample tested</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=2>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td></tr></table>

In conclusion, DNA from MRSA strains absorbed on swabs is stable up to 48 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ $( 9 6 \% - 1 0 0 \% )$ and up to 5 days at $\bf { 2 . 8 ^ { \circ } C }$ $( \mathbf { 1 0 0 \% } )$ for the 3 collection devices tested. DNA from MSSSp strains absorbed on swabs gave the expected negative result up to 48 hours at $2 5 \pm 2 ^ { \circ } \mathrm { C }$ $( 9 6 \% - 1 0 0 \% )$ ) and up to 5 days at $\bf { 2 . 8 ^ { \circ } C }$ $( \mathbf { 1 0 0 \% } )$ for the 3 collection devices tested.

# Stability of Crude DNA

The stability of crude DNA lysates after processing according to the BD GeneOhm™ MRSA ACP Lysis Kit package insert procedures and storage under varying conditions was evaluated in an analytical study.

Twenty seven (27) Methicillin-resistant S. aureus (MRSA) strains and 27 Methicillinsusceptible Staphylococcus sp. (MSSSp) strains were used in this study. The MRSA strains were of diverse geographical origin and included MREJ types i, ii, iii, iv, v, and vii. For each MRSA strain, $2 5 ~ \mu \mathrm { L }$ of the bacterial suspension at 2-3X LOD and $6 5 ~ \mu \mathrm { L }$ of negative nasal matrix for a total volume of $9 0 \mu \mathrm { L }$ were transferred into a lysis tube containing $2 0 ~ \mu \mathrm { L }$ of reconstituted lysis reagent (ACP). For each MSSSp strain, $2 5 \mu \mathrm { L }$ of the bacterial suspension at a theoretical concentration of $1 . 0 0 \mathrm { E } { + 0 7 }$ CFU/mL and $6 5 ~ \mu \mathrm { L }$ of negative nasal matrix for a total volume of $9 0 \mu \mathrm { L }$ were transferred into a lysis tube containing $2 0 \mu \mathrm { L }$ of ACP. The tubes were incubated at $3 7 ^ { \circ } \mathrm { C }$ for 20 minutes followed by an incubation of 5 minutes at $9 9 \mathrm { { } ^ { \circ } C }$ per the specimen lysis procedure in the BD GeneOhm™ ACP Lysis Kit package insert. The tubes were then stored under the following conditions:

• Baseline, no storage $( \mathrm { T } { = } 0 )$ )   
• $2 { - } 8 ^ { \circ } \mathrm { C }$ for 4 hours with septum cap $2 { - } 8 ^ { \circ } \mathrm { C }$ for 72 hours with screw cap $2 { - } 8 ^ { \circ } \mathrm { C }$ for 8 days with screw cap - ${ \bf \cdot } 2 0 ^ { \circ } \mathrm { C }$ for 18 – 19 hours with screw cap (For the MSSSp arm, storage was for 18 – 21 hours). They were subsequent tested after 2 and 5 freeze-thaw cycles, respectively. - ${ \cdot 2 0 ^ { \circ } } \mathrm { C }$ for 72 hours with screw cap - ${ \cdot 2 0 ^ { \circ } } \mathrm { C }$ for 8 days with screw cap

For each of the storage conditions, strains were tested in triplicate. At baseline or after storage, study samples were prepared and analyzed according to the BD GeneOhm™ MRSA ACP Assay package insert. For a storage condition to be considered acceptable, the percentage of conforming results had to be equal to or greater than $9 5 \%$ .

Summary results obtained for MRSA lysates after various storage conditions are presented in the following table:

<table><tr><td rowspan=5 colspan=2>MRSA Lysates</td><td rowspan=1 colspan=9>Storag e time and temperature</td></tr><tr><td rowspan=1 colspan=2>Septum cap</td><td rowspan=1 colspan=7>Screw cap</td></tr><tr><td rowspan=3 colspan=1>T=0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2-8 </td><td rowspan=1 colspan=5>-20 </td></tr><tr><td rowspan=2 colspan=1>4 hours</td><td rowspan=2 colspan=1>72 hours</td><td rowspan=1 colspan=1>8 days</td><td rowspan=1 colspan=1>18-191 hours</td><td rowspan=1 colspan=1>72 hours</td><td rowspan=1 colspan=1>8 days</td><td rowspan=2 colspan=1>After 2freeze- thawcycles</td><td rowspan=2 colspan=1>After 5freeze- thawcycles</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Sample results (pos)</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>78</td></tr><tr><td rowspan=2 colspan=1>Sampleresults</td><td rowspan=1 colspan=1>(neg)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>(unr)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Sample lysates tested</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>802</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=2>o of conform ing results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>96</td></tr></table>

1 Corresponds with one freeze-thaw cycle. An atypical PCR curve was obtained for one replicate of strain IDI-4471 and the results were excluded from calculation

Summary results for MSSSp strains under various storage conditions are presented in the following table:

<table><tr><td rowspan=4 colspan=1>MSSSp Lysates</td><td rowspan=1 colspan=9>Storage time and temperature</td></tr><tr><td rowspan=1 colspan=2>Septum cap</td><td rowspan=1 colspan=7>Screw cap</td></tr><tr><td rowspan=2 colspan=1>T=0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>2-8°</td><td rowspan=1 colspan=5>-20 </td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>72 hours</td><td rowspan=1 colspan=1>8 days</td><td rowspan=1 colspan=1>18-21 hours</td><td rowspan=1 colspan=1>72 hours</td><td rowspan=1 colspan=1>8 days</td><td rowspan=1 colspan=1>After 2freeze- thawcycles</td><td rowspan=1 colspan=1>After 5freeze- thawcycles</td></tr><tr><td rowspan=1 colspan=1>Sample results (Neg)</td><td rowspan=1 colspan=1>811</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>79</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sampleresults</td><td rowspan=1 colspan=1>(Unr)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>(Pos)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>Sample lysates tested</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>782</td><td rowspan=1 colspan=1>803</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>81</td></tr><tr><td rowspan=1 colspan=2>% of conforming results</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>98</td></tr></table>

1 For 3 strains at $\mathrm { T } { = } 0$ and $^ { 4 \mathrm { h } }$ of storage at 2 - ${ } . 8 ^ { \circ } \mathrm { C }$ was retested due to IC contamination. Results from the first tests were excluded from the calculation. 2 Results from strain IDI-2254 for $^ { 4 \mathrm { h } }$ of storage at $2 - 8 ^ { \circ } \mathrm { C }$ were not obtained $^ { 3 } { \bf A } { \bf n }$ atypical PCR curve was obtained for one replicate and the result was excluded from calculations.

In conclusion, lysates of MRSA strains were demonstrated to be stable up to 4 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ with septum cap and up to 8 days at ${ - 2 0 ^ { \circ } C }$ (including 2 and 5 freeze-thaw cycles) with screw cap. Lysates of MRSA strains did not appear to be stable after storage at 2- $8 \mathrm { { } ^ { \circ } C }$ for 72 hours or 8 days with a screw cap. Lysates of MSSSp strains stored up to 4 hours at $2 { - } 8 ^ { \circ } \mathrm { C }$ with septum cap and up to 8 days at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ (including 2 and 5 freezethaw cycles) with screw cap demonstrated the expected negative results with the BD GeneOhm™ MRSA ACP Assay.

# LoD determination using genomic DNA

The analytical sensitivity (limits of detection or LoDs) of the BD GeneOhm™ MRSA ACP Assay testing genomic DNA were determined using decreasing amount of quantified (copies/PCR reaction) genomic DNA from cultures of 6 MRSA strains that represent 6 MREJ genotypes (i, ii, iii, iv, v, and vii) and 4 SCCmec types(I, II, III, IV). The list of MRSA strains used in the study is provided in the following table:

<table><tr><td rowspan=1 colspan=1>MRSA Strains</td><td rowspan=1 colspan=1>MREJ type</td><td rowspan=1 colspan=1>SCCmec type</td></tr><tr><td rowspan=1 colspan=1>CCRI-8894</td><td rowspan=1 colspan=1>type i</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>type ii</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>CCRI-1262</td><td rowspan=1 colspan=1>typeiii</td><td rowspan=1 colspan=1>III</td></tr><tr><td rowspan=1 colspan=1>ATCC 33592</td><td rowspan=1 colspan=1>typeiv</td><td rowspan=1 colspan=1>III</td></tr><tr><td rowspan=1 colspan=1>CCRI-2025</td><td rowspan=1 colspan=1>typeV</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>CCRI-9773</td><td rowspan=1 colspan=1>type vii</td><td rowspan=1 colspan=1>I</td></tr></table>

Dilutions of MRSA genomic DNA (20, 15, 10, 5, 2.5, 1 and 0 copies/PCR reaction) were tested with the BD GeneOhm™ MRSA ACP Assay. To increase variability, 3 different lots of lyophilized BD GeneOhm™ MRSA ACP Assay Master Mixes were tested for each strain. Each lot of lyophilized Master Mix was reconstituted with $2 2 5 \mu \mathrm { L }$ of Diluent, and 25 $\mu \mathrm { L }$ aliquots were distributed in SmartCycler $\textsuperscript { \textregistered }$ tubes as instructed in the BD GeneOhm™ MRSA ACP Assay package insert. Then, $3 \mu \mathrm { L }$ of each MRSA genomic DNA dilution were added as specimens in 4 replicates per Master Mix lot (a total of 12 replicates per dilution). PC and NC were included with each PCR run as described in the BD GeneOhm™ MRSA ACP Assay package insert. This protocol was repeated 3 more times generating a total of 48 results for each DNA dilution. Two (2) different operators conducted the study in order to further increase the variability.

Analytical sensitivity testing MRSA genomic DNA was determined for each MRSA strain.

The LoD was determined as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 46 out of 48 replicates, or at least 45 out of 47 replicates, if a “not determined” result was obtained due to $\mathrm { I - C O R E } ^ { \textregistered }$ Module failure, were determined to be positive). The data are presented in the following table:

<table><tr><td colspan="1" rowspan="1">MRSAStrainMREJTypes</td><td colspan="1" rowspan="1">Genomic DNAConcentrationooies/PCreaction)</td><td colspan="1" rowspan="1">Average FAM CT</td><td colspan="1" rowspan="1">StandardDeviationFAM CT</td><td colspan="1" rowspan="1">Min.FAMC</td><td colspan="1" rowspan="1">Max.FAM</td><td colspan="1" rowspan="1">ReplicatesDeetected</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="7">Type i</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">40.9</td><td colspan="1" rowspan="1">30/48</td><td colspan="1" rowspan="1">62.5%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">36.9</td><td colspan="1" rowspan="1">40.9</td><td colspan="1" rowspan="1">41/48</td><td colspan="1" rowspan="1">85.4%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">41.0</td><td colspan="1" rowspan="1">46/48</td><td colspan="1" rowspan="1">95.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="7">Type ii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">28/48</td><td colspan="1" rowspan="1">58.3%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">42.3</td><td colspan="1" rowspan="1">41/48</td><td colspan="1" rowspan="1">85.4%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">41.7</td><td colspan="1" rowspan="1">46/48</td><td colspan="1" rowspan="1">95.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">0.59</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">39.1</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="7">Type iii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">40.9</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">42.3</td><td colspan="1" rowspan="1">25/48</td><td colspan="1" rowspan="1">52.1%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">42.1</td><td colspan="1" rowspan="1">38/48</td><td colspan="1" rowspan="1">79.2%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">41.6</td><td colspan="1" rowspan="1">46/48</td><td colspan="1" rowspan="1">95.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">0.64</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">39.8</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="7">Type iv</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">42.1</td><td colspan="1" rowspan="1">23/48</td><td colspan="1" rowspan="1">47.9%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.83</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">40/47</td><td colspan="1" rowspan="1">85.1%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">40.9</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">39.7</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">36.9</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">47/47</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="7">Type v</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">1.03</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">35/48</td><td colspan="1" rowspan="1">72.9%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">1.00</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">47/48</td><td colspan="1" rowspan="1">97.9%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">0.82</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">39.7</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">0.91</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">0.69</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="5">Type vii</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/48</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">40.2</td><td colspan="1" rowspan="1">0.99</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">42.5</td><td colspan="1" rowspan="1">30/48</td><td colspan="1" rowspan="1">62.5%</td></tr><tr><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">39.8</td><td colspan="1" rowspan="1">1.06</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">41.8</td><td colspan="1" rowspan="1">36/48</td><td colspan="1" rowspan="1">75.0%</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">40.8</td><td colspan="1" rowspan="1">46/48</td><td colspan="1" rowspan="1">95.8%</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">MRSAStrainMREJTypes</td><td colspan="1" rowspan="1">Genomic DNAConcentration(Copies/PCRreaction)</td><td colspan="1" rowspan="1">AverageFAM CT</td><td colspan="1" rowspan="1">StandardDeviationFAM CT</td><td colspan="1" rowspan="1">Min.FAMCT</td><td colspan="1" rowspan="1">Max.FAMCT</td><td colspan="1" rowspan="1">ReplicatesDetected</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">48/48</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

The number of positive results is out of 47 (instead of 48) because a “not determined” (ND) result was obtained due to I-CORE® Module failure

The sponsor demonstrated that LoD testing MRSA MREJ Type i, ii, iii, iv, and vii genomic DNA is 5 genomic DNA copies/PCR reaction (approximately 1,250 CFU/swab1 ). LoD testing MRSA MREJ Type v genomic DNA is 2.5 genomic DNA copies/PCR reaction (approximately 625 CFU/swab).

LoD determination using viable bacteria with or without the clinical nasal matrix

The analytical sensitivity (limits of detection or LoDs) of the BD GeneOhm™ MRSA ACP Assay testing viable MRSA strains with or without clinical nasal matrix were determined using simulated positive swabs that were prepared by soaking swabs in a wide range of MRSA bacterial suspensions, and then discharged/eluted in either pooled negative clinical nasal matrix or Sample Buffer.

Cultures were prepared and concentrations were determined by colony count. The quantified $\mathrm { ( C F U / m L ) }$ cultures were used to obtain bacterial suspensions at a theoretical concentration of $1 . 2 5 \mathrm { E } { + 0 2 }$ to $1 . 0 0 \mathrm { E } { + } 0 8 $ CFU/mL. The list of MRSA strains used in the study is provided in the following table:

<table><tr><td rowspan=1 colspan=1>MRSA Strains</td><td rowspan=1 colspan=1>MREJ type</td><td rowspan=1 colspan=1>SCCmec type</td><td rowspan=1 colspan=1>Starting Conc.CFU/mL</td></tr><tr><td rowspan=1 colspan=1>CCRI-8894</td><td rowspan=1 colspan=1>type i</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>4.49E + 07</td></tr><tr><td rowspan=1 colspan=1>ATCC 43300</td><td rowspan=1 colspan=1>type ii</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>5.76E + 07</td></tr><tr><td rowspan=1 colspan=1>CCRI-1262</td><td rowspan=1 colspan=1>typeiii</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>5.55E + 07</td></tr><tr><td rowspan=1 colspan=1>ATCC 33592</td><td rowspan=1 colspan=1>typeiv</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>2.12E + 08</td></tr><tr><td rowspan=1 colspan=1>CCRI-2025</td><td rowspan=1 colspan=1>typeV</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>1.13E + 08</td></tr><tr><td rowspan=1 colspan=1>CCRI-9773</td><td rowspan=1 colspan=1>type vii</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>3.86E + 08</td></tr></table>

The bacterial suspensions prepared above were used to simulate positive samples with a wide range of MRSA loads. Swabs were soaked with bacterial suspensions at different concentrations. A total of 6 swabs were soaked with each bacterial suspension (3 were used to determine LoD in negative nasal matrix and 3 in Sample Buffer). After the bacterial suspensions were absorbed by the swabs, samples were further prepared in either pooled negative clinical nasal matrix or Sample Buffer as the following: Swabs soaked with bacterial suspension were placed in tubes containing either $6 0 0 \mu \mathrm { L }$ of Sample Buffer or 600 $\mu \mathrm { L }$ of negative nasal matrix, the stem was then broken and the swab discharged by vortexing for 1 minute.

Note: To prepare negative clinical nasal matrix, nasal swabs obtained from patients were characterized with the BD GeneOhm™ MRSA ACP Assay following instructions in the package insert. Negative samples were pooled to create the negative nasal matrix.

Next, $9 0 \mu \mathrm { L }$ of the swab discharge/cell suspension were transferred to lysis tubes containing $2 0 \mu \mathrm { L }$ of reconstituted ACP Lysis Reagent. The tubes were incubated at $3 7 ^ { \circ } \mathrm { C }$ for 20 minutes and then at $9 9 \mathrm { { } ^ { \circ } C }$ for 5 minutes. The samples were then stored between 2- $8 ^ { \circ } \mathrm { C }$ , until they were used in testing. Two (2) series of lysates with negative nasal matrix and 2 series without negative nasal matrix (in Sample Buffer) were prepared for each MRSA strain per concentration.

A total of 6 processed samples representing decreasing bacterial concentrations per MRSA strain per sample matrix (with or without nasal matrix) were tested with the BD GeneOhm™ MRSA ACP Assay. To increase variability, 3 different lyophilized BD GeneOhm™ MRSA ACP Master Mix lots were tested with each MRSA strain per concentration per sample matrix (with or without nasal matrix). Each lyophilized Master Mix was reconstituted with $2 2 5 \mu \mathrm { L }$ of Diluent, and $2 5 \mu \mathrm { L }$ aliquots were distributed in SmartCycler® tubes as instructed in the BD GeneOhm™ MRSA ACP Assay package insert. Then, $3 \mu \mathrm { L }$ of each processed samples were added as specimen in 4 replicates per Master Mix lot (a total of 12 replicates with negative nasal matrix and 12 replicates without nasal matrix per MRSA strain per concentration per lysate). PC and NC were added as controls for each PCR run as described in the package insert. This protocol was repeated once generating a total of 24 results for each DNA dilution. Two (2) different operators conducted the study in order to further increase the variability.

Analytical sensitivity testing viable MRSA was determined for each MRSA strain with or without clinical nasal matrix. The LoD was determined as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 23 out of 24 replicates were determined to be positive). The data are presented in the following tables:

<table><tr><td colspan="1" rowspan="1">MRSA StrainMREJ Types withor without clinicalnasal matrix</td><td colspan="1" rowspan="1">Viable MRSAConcentration(CFU/swab)</td><td colspan="1" rowspan="1">AverageFAM CT</td><td colspan="1" rowspan="1">StandardDeviationFAM CT</td><td colspan="1" rowspan="1">Min.FAMCT</td><td colspan="1" rowspan="1">Max.FAMCT</td><td colspan="1" rowspan="1">ReplicatesDetected</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="6">Type i (with clinicalnasal matrix)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">38.4</td><td colspan="1" rowspan="1">41.4</td><td colspan="1" rowspan="1">17/24</td><td colspan="1" rowspan="1">70.8%</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">39.7</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">41.1</td><td colspan="1" rowspan="1">15/24</td><td colspan="1" rowspan="1">62.5%</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">0.80</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">40.2</td><td colspan="1" rowspan="1">22/24</td><td colspan="1" rowspan="1">91.7%</td></tr><tr><td colspan="1" rowspan="1">207</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">414</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Type i (without</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="5">clinical nasal matrix</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">19/24</td><td colspan="1" rowspan="1">79.1%</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">38.4</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">17/24</td><td colspan="1" rowspan="1">70.8%</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.97</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">21/24</td><td colspan="1" rowspan="1">87.5%</td></tr><tr><td colspan="1" rowspan="1">207</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">38.4</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">414</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>MRSA StrainMREJ Types withor without clinicalnasal matrix</td><td rowspan=1 colspan=1>Viable MRSAConcentration(CFU/swab)</td><td rowspan=1 colspan=1>AverageFAM CT</td><td rowspan=1 colspan=1>StandardDeviationFAM CT</td><td rowspan=1 colspan=1>Min.FAMCT</td><td rowspan=1 colspan=1>Max.CT</td><td rowspan=1 colspan=1>ReplicatesDDetected</td><td rowspan=1 colspan=1>% Detected</td></tr><tr><td rowspan=6 colspan=1>Type ii (withclinial nasalmatrix)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>15/24</td><td rowspan=1 colspan=1>62.5%</td></tr><tr><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>15/24</td><td rowspan=1 colspan=1>62.5%</td></tr><tr><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>41.8</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>1152</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>2304</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=6 colspan=1>Type ii (withoutclinical nasal matrix</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>40.5</td><td rowspan=1 colspan=1>14/24</td><td rowspan=1 colspan=1>58.3%</td></tr><tr><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>19/24</td><td rowspan=1 colspan=1>79.1%</td></tr><tr><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>576</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>38.1</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>1152</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

<table><tr><td colspan="1" rowspan="1">MRSA StrainMREJ Types withor without clinicalnasal matrix</td><td colspan="1" rowspan="1">Viable MRSAConcentration(CFU/swab)</td><td colspan="1" rowspan="1">AverageFAM CT</td><td colspan="1" rowspan="1">StandardDeviationFAM CT</td><td colspan="1" rowspan="1">Min.FAMCT</td><td colspan="1" rowspan="1">Max.FAMCT</td><td colspan="1" rowspan="1">ReplicatesDetected</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="6">Type iii (withclinial nasalmatrix</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">128</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">0.74</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">17/24</td><td colspan="1" rowspan="1">70.8%</td></tr><tr><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">42.3</td><td colspan="1" rowspan="1">23/24</td><td colspan="1" rowspan="1">95.8%</td></tr><tr><td colspan="1" rowspan="1">512</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">39.7</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1025</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">2049</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.88</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="6">Type ii (withoutclinical nasal matrix</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">256</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">41.8</td><td colspan="1" rowspan="1">21/24</td><td colspan="1" rowspan="1">87.5%</td></tr><tr><td colspan="1" rowspan="1">512</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">0.68</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">22/24</td><td colspan="1" rowspan="1">91.7%</td></tr><tr><td colspan="1" rowspan="1">1025</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">2049</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">4098</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="6">Type iv (withcliil nasalmatrix)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">0.96</td><td colspan="1" rowspan="1">38.7</td><td colspan="1" rowspan="1">42.2</td><td colspan="1" rowspan="1">16/24</td><td colspan="1" rowspan="1">66.7%</td></tr><tr><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">39.8</td><td colspan="1" rowspan="1">0.94</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">41.4</td><td colspan="1" rowspan="1">20/24</td><td colspan="1" rowspan="1">83.3%</td></tr><tr><td colspan="1" rowspan="1">245</td><td colspan="1" rowspan="1">39.2</td><td colspan="1" rowspan="1">0.78</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">40.8</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">489</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">0.72</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">978</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="3" rowspan="1"></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="6">Type iv (withoutclinical nasal matrix</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">—</td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">39.9</td><td colspan="1" rowspan="1">0.90</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">41.5</td><td colspan="1" rowspan="1">18/24</td><td colspan="1" rowspan="1">75.0%</td></tr><tr><td colspan="1" rowspan="1">122</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">0.86</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">41.4</td><td colspan="1" rowspan="1">21/24</td><td colspan="1" rowspan="1">87.5%</td></tr><tr><td colspan="1" rowspan="1">245</td><td colspan="1" rowspan="1">39.2</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">489</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">0.76</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">978</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

<table><tr><td rowspan=1 colspan=1>MRSA StrainMREJ Types withor without clinicalnasal matrix</td><td rowspan=1 colspan=1>Viable MRSAConcentration(CFU/swab)</td><td rowspan=1 colspan=1>AverageFAM CT</td><td rowspan=1 colspan=1>StandardDeviationFAM CT</td><td rowspan=1 colspan=1>Min.FAMCT</td><td rowspan=1 colspan=1>Max.FAMCT</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>% Detected</td></tr><tr><td rowspan=6 colspan=1>Type v (withclinial nasalmatrix)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>15/24</td><td rowspan=1 colspan=1>62.5%</td></tr><tr><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>41.0</td><td rowspan=1 colspan=1>20/24</td><td rowspan=1 colspan=1>83.3%</td></tr><tr><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>40.2</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>261</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>522</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=3></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=6 colspan=1>Type v (withoutclinical nasal matrix</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/24</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>40.9</td><td rowspan=1 colspan=1>18/24</td><td rowspan=1 colspan=1>75.0%</td></tr><tr><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>21/24</td><td rowspan=1 colspan=1>87.5%</td></tr><tr><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>41.0</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>261</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td></tr><tr><td rowspan=1 colspan=1>522</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td></tr></table>

<table><tr><td colspan="1" rowspan="1">MRSA StrainMREJ Types withor without clinicalnasal matrix</td><td colspan="1" rowspan="1">Viable MRSAConcentration(CFU/swab)</td><td colspan="1" rowspan="1">AverageFAM CT</td><td colspan="1" rowspan="1">StandardDeviationFAM CT</td><td colspan="1" rowspan="1">Min.FAMCT</td><td colspan="1" rowspan="1">Max.FAMCT</td><td colspan="1" rowspan="1">ReplicatesDtected</td><td colspan="1" rowspan="1">% Detected</td></tr><tr><td colspan="1" rowspan="6">Type vii (withclinical nasalmatrix)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">1.08</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">17/24</td><td colspan="1" rowspan="1">70.8%</td></tr><tr><td colspan="1" rowspan="1">193</td><td colspan="1" rowspan="1">39.8</td><td colspan="1" rowspan="1">1.45</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">42.2</td><td colspan="1" rowspan="1">14/24</td><td colspan="1" rowspan="1">58.3%</td></tr><tr><td colspan="1" rowspan="1">386</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">40.1</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">772</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1544</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="6">Type vii (withoutclinical nasal matrix</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/24</td><td colspan="1" rowspan="1">0.0%</td></tr><tr><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">39.7</td><td colspan="1" rowspan="1">1.34</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">41.3</td><td colspan="1" rowspan="1">11/24</td><td colspan="1" rowspan="1">45.8%</td></tr><tr><td colspan="1" rowspan="1">193</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">1.30</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">42.2</td><td colspan="1" rowspan="1">16/24</td><td colspan="1" rowspan="1">66.7%</td></tr><tr><td colspan="1" rowspan="1">386</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">0.98</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">40.1</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">772</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">36.7</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">1544</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">24/24</td><td colspan="1" rowspan="1">100.0%</td></tr></table>

The sponsor demonstrated that LoDs testing viable MRSA MREJ Type i, ii, iii, iv, v, and vii with clinical nasal matrix are 207 CFU/swab, 576 CFU/swab, 256 CFU/swab, 245 CFU/swab, 130 CFU/swab, and 386 CFU/swab, respectively. LoDs testing viable MRSA MREJ Type i, ii, iii, iv, v, and vii without clinical nasal matrix are 207 CFU/swab, 288 CFU/swab, 1025 CFU/swab, 245 CFU/swab, 130 CFU/swab, and 386 CFU/swab, respectively.

In conclusion, with clinical nasal matrix, LoDs testing viable MRSA MREJ Type i, ii, iii, iv, v, and vii range between 130 and 576 CFU/swab with an average value of 300 CFU/swab. Without clinical nasal matrix, LoDs testing viable MRSA MREJ Type i, ii, iii, iv, v, and vii range between 130 and 1025 CFU/swab with an average value of 380 CFU/swab.

In addition, it appears that clinical nasal matrix has minimal impact on the BD GeneOhm™ MRSA ACP Assay’s LoD.

# e. Analytical inclusivity:

Analytical ubiquity of the BD GeneOhmTM MRSA ACP assay was evaluated in the analytical inclusivity study. A variety of Staphylococcus aureus strains were included in the study taking into account geographic origin, MREJ genotype, SCCmec type, pulse field gel electrophoresis (PFGE) type, temporal diversity and susceptibility pattern. Onehundred-forty (140) strains from 31 countries were tested in this analytical inclusivity study, including 52 from public collections and 88 from well-characterized clinical isolates, including Vancomycin-resistant Staphylococcus aureus (VRSA) and Vancomycinintermediate Staphylococcus aureus (VISA) strains.

Isolated colonies from BAP of all Staphylococcus aureus strains tested in the study, excluding those known to be VRSA and VISA, were subcultured on BBL™ CHROMagar™ MRSA plates for confirmation as MRSA.

Bacterial lysates tested were obtained from an internal BD Diagnostics bank of lysates and were tested with the BD GeneOhm™ MRSA ACP Assay as the following: Lysates for each strain were prepared to be tested at both a relevant clinical load (roughly 100 genome

copies $/ \mu \mathrm { L }$ ) and at roughly $2 { - } 3 \mathbf { x }$ LoD concentration.

BD GeneOhm™ MRSA ACP Assay lyophilized Master Mix were reconstituted with 225 $\mu \mathrm { L }$ of BD GeneOhm™ MRSA ACP Diluent and $2 5 \mu \mathrm { L }$ of reconstituted Master Mix were distributed into assay and control tubes as described in the BD GeneOhm™ MRSA ACP Assay package insert. Three (3) $\mu \mathrm { L }$ of sample were added to each assay tube. For PCR controls, three (3) $\mu \mathrm { L }$ of Control DNA (reconstituted with $2 2 5 \mu \mathrm { L }$ of Sample Buffer) were added into the PC tube, while $3 \mu \mathrm { L }$ of Sample Buffer were added into the NC tube. At the end, all assay tubes were centrifuged and amplification was performed using the SmartCycler II instrument. One (1) replicate was tested at the relevant clinical load dilution and 3 replicates were tested at the 2-3X LoD concentration. At the end of the amplification protocol, assay tubes amplified by using the relevant clinical load dilution were analyzed by agarose gel electrophoresis to further assess if the expected amplicon had been produced.

When the lysates of the analytical inclusivity panel were tested at the relevant clinical load (roughly 100 genome copies $ { ' }  { \mu \mathrm { L } }$ ), 139 out of 140 MRSA strains tested gave a positive result. One false negative result was obtained with the IDI-3368 strain, which is known to be a MREJ type ii, SCCmec type III. No MREJ amplicon was produced on the gel corresponding to IDI-3368; however, the IC amplicon was produced. Other MRSA strains of MREJ type ii and SCCmec type III tested in this study were all detected by the BD GeneOhm™ MRSA ACP assay. In addition, all MREJ genotypes tested in this study were detected. All MRSA SCCmec types (I to VI), PFGE MRSA USA 100 to 800, 1000 and 1100 as well as VRSA and VISA tested in this study were also detected by the BD GeneOhm™ MRSA ACP assay.

When the lysates of the analytical inclusivity panel were tested at the 2-3X LoD concentration with three (3) replicates, 2 assay results were discarded due to an electrical malfunction in the I-CORE®. A total of 116 strains $(8 3 \% )$ gave 3 out of 3 positive assay results (or 2 out of 2 for both strains that experienced an $\mathrm { I - C O R E } ^ { \mathrm { \textregistered } }$ malfunction with the third replicate). For the other 24 MRSA strains, 23 strains had 1 or 2 out of 3 positive assay results. The FAM CT range obtained among these 23 strains was between 38.0 and 41.6. Since the FAM CT range at the LoD concentration is 38.4 to 40.7, it is likely that the 23 strains noted above were actually tested at concentrations lower than the targeted 2-3X LoD concentration in this study. For this reason, detection rates of less than $100 \%$ are not unexpected. IDI-3368 was the only strain that exhibited a false negative assay result (all 3 replicates) at the 2-3X LoD concentration. Note: IDI-3368 also had a false negative assay result when tested at the relevant clinical load.

In conclusion, 140 MRSA strains from 31 countries were tested with the BD GeneOhm™ MRSA ACP Assay. When tested at the relevant clinical load (roughly 100 genome copies $/ \mu \mathrm { L }$ ), $9 9 \%$ of all the strains were detected. The BD GeneOhm™ MRSA ACP assay detected all of the MREJ wild types i, ii, iii, iv, v and vii tested, as well as MREJ mutant types ii mut16, ii mut25 and iii mut25. The BD GeneOhm™ MRSA ACP Assay detected MRSA SCCmec types I to VI and PFGE MRSA USA 100 to 800, 1000 and 1100. Staphylococcus aureus strains displaying additional resistance to vancomycin (VRSA and VISA) were also detected. When the same strains were tested in triplicate at 2-3X LoD concentration, $8 3 \%$ were detected in all three replicates and $9 9 \%$ were detected in at least one out of three replicates.

Staphylococcus aureus isolates identified as “empty cassette variants” were not tested using the BD GeneOhm™ MRSA ACP Assay. Since the BD GeneOhm™ MRSA ACP Assay targets the SCCmec/orfX junction area of methicillin-resistant Staphylococcus aureus, not the actual mecA gene, it will very likely to report a false positive MRSA result for an empty cassette variant. The following language should be included in the “Limitations” section of the product package insert to address this issue: “The BD GeneOhm™ MRSA ACP Assay does not detect the mecA gene directly nor the penicillin binding protein (PBP2a) encoded by this gene. A false positive result may occur if an empty cassette S. aureus variant is present.”

# g. Evaluation of a well characterized challenge strain panel:

An additional analytical study was carried out to evaluate the analytical performance of the BD GeneOhm™ MRSA ACP Assay by testing a well characterized challenge strain panel containing MRSA strains with high and low oxacillin MICs, including PFGE types USA 100, 300, and 400 (with emphasis on USA 300), borderline oxacillin-resistant S. aureus (BORSA) (BORSA strains are mecA negative, but usually exhibit an oxacillin minimum inhibitory concentration $( \mathrm { M I C } ) > = 2$ and $< = 8 \mathrm { \ u g / m l }$ by a mechanism not completely understood), MSSA and methicillin-resistant Staphylococcus epidermidis (MRSE) strains. The challenge strain panel used in this study was composed of 15 MRSA, 4 BORSA, 1 MRSE and 5 MSSA strains. All MRSA strains tested belong to MREJ type ii, with the exception of strain IDI-5064, which belongs to MREJ type iii. These strains have previously been shown to display a broad range of oxacillin and cefoxitin MICs. All these strains were tested with FDA-cleared broth dilution susceptibility tests (BD Phoenix Broth Microdilution System) using Oxacillin (k023301) and Cefoxitin (k053241) for determination of the MIC values.

For the MRSA strains, cultures were prepared and used to obtain a bacterial suspension adjusted to the turbidity of a 0.5 McFarland standard. This suspension was used to perform serial dilutions to obtain suspensions with theoretical concentrations ranging between $1 . 0 0 \mathrm { E } { + } 0 2$ and $6 . 4 0 \mathrm { E } { + } 0 4 $ CFU/mL. Fifty microliters $( 5 0 \mu \mathrm { L } )$ of the suspensions containing $1 . 0 0 \mathrm { E } { + } 0 2$ , $1 . 0 0 \mathrm { E } { + 0 3 }$ and $8 . 0 0 \mathrm { E } { + 0 3 }$ CFU/mL (theoretical concentrations) were spread on BAP and incubated at $3 5 \mathrm { { ^ \circ C } }$ (5 or 6 replicate plates per concentration). After an $1 8 \mathrm { h } { - } 2 4 \mathrm { h }$ incubation, colonies were counted for each replicate plate and an average was calculated for each dilution. The results were used to determine the dilution corresponding to 2-3X LoD. For each MRSA strain as well as for the positive SPC, 3 aliquots of $7 5 \mu \mathrm { L }$ from 6 dilutions ranging between theoretical concentrations of $2 . 0 0 \mathrm { E } { + } 0 3$ and $6 . 4 0 \mathrm { E } { + } 0 4 $ CFU/mL were each absorbed onto a swab and discharged in a Sample Buffer tube $( 6 0 0 \mu \mathrm { L } )$ by vortexing for 60 seconds. Then, $9 0 \mu \mathrm { L }$ of the discharge were transferred to a Lysis tube containing $2 0 \mu \mathrm { L }$ of ACP. The tube was incubated at $3 7 ^ { \circ } \mathrm { C }$ for 20 minutes and then for 5 minutes at $9 9 \mathrm { { } ^ { \circ } C }$ to inactivate the enzyme. The samples were stored between $2 { - } 8 ^ { \circ } \mathrm { C }$ for at least 10 minutes until they were added to the PCR tubes.

For the non-MRSA strains, BAPs were inoculated from frozen stock cultures. A second set of BAPs were inoculated from the first set and incubated for 18-24h at $3 5 \mathrm { { ^ \circ C } }$ to obtain fresh cultures. Isolated colonies from plates prepared as described above were used to prepare a bacterial suspension adjusted to the turbidity of a 0.5 McFarland standard. This suspension was used to perform serial dilutions to obtain suspensions with theoretical concentrations ranging between $1 . 0 0 \mathrm { E } { + } 0 2$ and $1 . 0 0 \mathrm { E } { + 0 6 }$ CFU/mL. Fifty microliters $( 5 0 \mu \mathrm { L } )$ of the suspensions containing $1 . 0 0 \mathrm { E } { + } 0 2$ , $1 . 0 0 \mathrm { E } { + 0 3 }$ and $1 . 0 0 \mathrm { E } { + } 0 4$ CFU/mL (theoretical concentrations) were spread on BAPs and incubated at $3 5 \mathrm { { ^ \circ C } }$ (6 replicate plates per concentration). After a 18h-24h incubation, colonies were counted for each replicate plate and an average was calculated for each dilution. The results were used to determine the concentration of the suspension used. For each MSSA, BORSA and MRSE strain as well as for negative SPCs, 3 swabs were dipped into a suspension with a theoretical concentration of $1 . 0 0 \mathrm { E } { + 0 6 }$ CFU/mL and were pressed against the tube wall to remove excess fluid. The bacteria were then discharged into Sample Buffer tubes $( 6 0 0 \mu \mathrm { L } )$ by vortexing for 60 seconds. Then, $9 0 \mu \mathrm { L }$ of the discharge were transferred to a Lysis tube containing $2 0 \mu \mathrm { L }$ of ACP. The tubes were incubated at $3 7 ^ { \circ } \mathrm { C }$ for 20 minutes and then for 5 minutes at $9 9 \mathrm { { } ^ { \circ } C }$ to inactivate the enzyme. The samples were stored between $2 \mathrm { { } ^ { \circ } C }$ and $8 \mathrm { { } ^ { \circ } C }$ for at least 10 minutes until they were added to the PCR tubes.

Lyophilized Master Mix was reconstituted with $2 2 5 \mu \mathrm { L }$ of Diluent per tube. Then, $3 \mu \mathrm { L }$ of each sample were transferred into SmartCycler® tubes containing $2 5 \mu \mathrm { L }$ of reconstituted Master Mix. Three (3) $\mu \mathrm { L }$ of Control DNA (PC), $3 \mu \mathrm { L }$ of Sample Buffer (NC) and $3 \mu \mathrm { L }$ of positive and negative SPCs were also added into SmartCycler® tubes containing reconstituted Master Mix. The analysis was performed with the SmartCycler II instrument.

In conclusion, all the MRSA strains tested as part of this analytical study testing challenge strains with the BD GeneOhm™ MRSA ACP Assay (including PFGE types USA 100, 300 and 400) exhibited positive results when tested at 2-3X LoD concentration. All BORSA, MSSA and MRSE strains tested exhibited negative results when tested at high concentrations.

# h. Analytical specificity:

The analytical specificity of the BD GeneOhm™ MRSA ACP Assay was evaluated by testing a panel consisting of well-characterized isolates of methicillin-sensitive Staphylococcus aureus (MSSA), coagulase negative Staphylococci (CNS), closely related genera and other pathogenic and commensal flora found in the nares. Forty-two (42) nonstaphylococcal strains from 39 species, 36 coagulase-negative staphylococci strains representing 25 species and 111 MSSA strains were tested in the cross-reactivity study. All these species are phylogenetically related to Staphylococcus aureus or commonly found in the human nares. For 13 out of the 42 non-staphylococci strains, purified DNA was tested. All DNA were provided by the CRI (Centre de recherché en infectiologie du CHUL, Centre Hospitalier Universitaire de Québec) at a concentration of approximately $1 0 0 \mathrm { n g / \mu L }$ . Purified DNA was tested at roughly $1 \mathrm { ~ x ~ i 0 } ^ { 5 }$ genome copies $ { { } ^ { \prime } \mu \mathrm { L } }$ $( \approx 1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ genome copies $/ \mathrm { m L }$ ) in a single PCR reaction, corresponding to roughly $\dot { 3 } \mathrm { ~ x ~ i 0 } ^ { 5 }$ genome copies/PCR reaction. For all the other isolates/strains, bacterial lysates were tested. Lysates were obtained from an internal BD Diagnostics bank of well characterized lysates and were tested also at approximately $1 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies $ { { } ^ { \prime } \mu \mathrm { L } }$ $( \approx 1 \mathrm { ~ x ~ } 1 0 ^ { 8 }$ genome copies/mL) in single PCR reaction per strain, corresponding to roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction. Analysis was performed using the SmartCycler $\mathrm { I I }$ instrument in accordance with the package insert. At the end of the amplification protocol, all assay tubes were analyzed by agarose gel electrophoresis to further assess whether amplicons had been produced. PCR reaction controls were also loaded on each gel.

Forty one (41) out of 42 non-staphylococci strains of various species exhibited negative results with the BD GeneOhm™ MRSA ACP Assay. The lack of non specific amplification of MRSA-specific amplicons was confirmed by gel electrophoresis analysis for all species tested. Although some faint non-specific bands were observed for the following species: K. pneumoniae, S. marcescens, C. genitalium, E. coli, C. flavescens, H. influenzae, $P$ . aerogenes, N. gonorrhoeae, E. cloacae, M. catarrhalis, and Homo sapiens DNA, because the sequence of the molecular beacon is in a highly conserved and specific region of S. aureus, there was no hybridization of the molecular beacon with any of nonspecific amplified product from the species noted above. One (1) strain out of 42, Salmonella typhimurium (ATCC 14028) gave an initial positive result with the BD GeneOhm™ MRSA ACP Assay. Repeat testing in triplet PCR reactions using a fresh lysate of Salmonella typhimurium (ATCC 14028) at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction and repeat testing in duplicate PCR reactions using the original lysate material at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { \dot { 5 } }$ genome copies/PCR reaction all produced negative results. It was therefore determined that the positive PCR result initially obtained for Salmonella typhimurium (ATCC 14028) was very likely due to carry-over contamination by MRSA DNA, and the BD GeneOhm™ MRSA ACP Assay did not cross react with Salmonella typhimurium (ATCC 14028).

Nineteen (19) Methicillin-sensitive coagulase negative Staphylococci (MSCNS) strains, 15 Methicillin-resistant coagulase negative Staphylococci (MRCNS) strains and 2 coagulase negative Staphylococci (CNS) strains with unknown susceptibility status were also tested in the analytical specificity study. Thirty-five of the 36 strains tested with the BD GeneOhm™ MRSA ACP Assay exhibited negative results. The absence of non specific amplification was confirmed by gel electrophoresis analysis for all strains tested except for Staphylococcus pulvereri, Staphylococcus epidermidis (IDI-564) and Staphylococcus auricularis where non specific bands were observed. However, because the sequence of the molecular beacon is in a highly specific region of S. aureus, the molecular beacon did not hybridize with these non specific amplification products and thus gave negative real-time PCR results. A fresh lysate of S. epidermidis (IDI-564) at roughly $\bar { 3 } \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction was prepared and tested in triplicate. Negative results were obtained by PCR with the fresh lysate, but 1 replicate out of 3 exhibited an amplification product of 280 bp, similar to what had previously been observed with the original lysate. However, there was no hybridization of the molecular beacon with this non specific amplified product. These results confirm that Staphylococcus pulvereri, Staphylococcus epidermidis (IDI-564) and Staphylococcus auricularis were not detected by the BD GeneOhm™ MRSA ACP Assay.

Staphylococcus chromogenes (ATCC 43764) initially exhibited an unresolved result. Repeat testing in triplet PCR reactions using a fresh lysate of Staphylococcus chromogenes (ATCC 43764) at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction and repeat testing in triplicate PCR reactions using the original lysate material at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction all produced negative results. Therefore, it was determined that the unresolved result obtained initially was very likely due to an inhibitor introduced into the

PCR tube during the preparation of this specific tube, and the BD GeneOhm™ MRSA ACP Assay did not cross react with Staphylococcus chromogenes (ATCC 43764).

In conclusion, 41 out of 42 strains of various non-staphylococcal species exhibited negative results with the BD GeneOhm™ MRSA ACP Assay and 1 strain, Salmonella typhimurium (ATCC 14028) was detected by the BD GeneOhm™ MRSA ACP Assay initially. However, an investigation demonstrated that the BD GeneOhm™ MRSA ACP Assay did not detect Salmonella typhimurium (ATCC 14028) and the initial positive result obtained was due to contamination.

Ninteen (19) MSCNS strains, 15 MRCNS strains and 2 CNS strains tested with the BD GeneOhm™ MRSA ACP Assay exhibited negative results, except for Staphylococcus chromogenes (ATCC 43764) for which had an unresolved result. Investigational tests were performed on Staphylococcus chromogenes (ATCC 43764) demonstrating that PCR inhibitory substances were initially present and repeat testing negative results were observed. Therefore, the BD GeneOhm™ MRSA ACP Assay did not cross react with Staphylococcus chromogenes (ATCC 43764).

Even though non specific amplification was visible on gel electrophoresis for some species, these amplicons were not detected by the BD GeneOhm™ MRSA ACP Assay. Therefore, the analytical specificity of the BD GeneOhm™ MRSA ACP Assay with closely related organisms, including CNS, normal nasal flora and pathogenic nasal flora, was $100 \%$ (78/78).

Out of a total of 111 MSSA strains isolated from 16 different countries tested in the analytical specificity study, 103 exhibited negative results with the BD GeneOhm™ MRSA ACP Assay. Of the 103 strains that exhibited negative results, the lack of amplification was confirmed by gel electrophoresis analysis. For some MSSA strains, faint bands were observed but were not detected by the assay.

For the remaining 8 strains that did not exhibit a negative result initially, 7 were positive and one was unresolved on initial testing. One (1) MSSA strain generated an unresolved result at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction (IDI-1603). Repeat testing of this strain result at roughly $3 \mathrm { ~ \bar { x } ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction after freezing and repeat testing at roughly $\bar { 3 } \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome copies/PCR reaction without freezing was performed in triplicate. In all cases, negative results were obtained for all replicates, and on gel a normal amplification level of the IC was visible. Therefore, IDI-1603 was considered as generating a negative result with the BD GeneOhm™ MRSA ACP Assay at a concentration of roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction.

Seven MSSA strains generated false positive results with the BD GeneOhm™ MRSA ACP Assay initially. For MSSA strain IDI-2236, contamination with an MRSA strain was suspected due to a high detection signal (CT of 34.2) and an amplicon consistent in size with MRSA MREJ type i of 178 bp on gel. A fresh lysate was prepared and tested in triplicate. Negative results were obtained and on gel no MRSA amplicon or any other non specific amplicon was visible, indicating that the positive result initially obtained was not due to the strain itself, but due to contamination. In order to confirm that the original lysate was indeed contaminated by an MRSA strain, another PCR test was performed in triplicate with the original lysate. PCR results were positive upon retesting and amplicons of the size of MRSA MREJ type i were again visible, confirming that the original lysate was indeed contaminated. The remaining six MSSA strains (IDI-183, IDI-184, IDI-188, IDI-214, IDI-187, IDI-193) generated false positive results at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction. Five (5) of the 6 MSSA strains with false positive results had evidence of amplicons on gel that could explain the assay result. IDI-188 did not have MRSA amplicon or non specific amplicons on gel which could explain the initial positive result obtained. Additional investigation was conducted with MSSA strain IDI-188 which included repeat testing from the original lysate at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction in triplicate as well as at roughly $\overline { { 3 \mathrm { ~ x ~ } 1 0 ^ { 4 } } }$ genome copies/PCR reaction. These tests all yielded negative assay results and on gel no amplicon except the IC was visible. Therefore, root cause for the false positive result could not be determined; however, the investigation confirmed that the BD GeneOhm™ MRSA ACP Assay did not cross-react with MSSA IDI-188.

The fve remaining MSSA strains (IDI-183, IDI-184, IDI-214, IDI-187 and IDI-193) exhibited negative results when tested at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome copies/PCR reaction. For 1 strain, IDI-193, 1 replicate out of 3 was unresolved. This replicate was retested from frozen lysate at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome copies/PCR reaction and exhibited a negative result. So, overall, 5 MSSA strains generated false-positive results at a concentration of roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction. This concentration is believed to exceed the concentrations seen in clinical specimens by the sponsor. When tested at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ genome copies/PCR reaction, which the sponsor believes is more consistent with a clinical relevant MSSA load, these 5 MSSA strains all generated negative results.

In conclusion, 106 out of 111 MSSA strains were not detected by the assay at roughly $3 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ genome copies/PCR reaction in the analytical specificity study. The remaining 5 MSSA strains were all positive at approximately $\textbf { 3 5 } \mathbf { 1 0 ^ { 5 } }$ genome copies/PCR reaction, but were all negative at approximately $3 \ \textbf { x } \textbf { 1 0 } ^ { 4 }$ genome copies/PCR reaction. Therefore, the analytical specificity of the BD GeneOhm™ MRSA ACP Assay with MSSA was $1 0 0 \%$ (111/111) at approximately $3 \mathrm { \bf ~ x } 1 0 ^ { 4 }$ genome copies/PCR reaction, and was $9 5 . 5 \%$ (106/111) at approximately $\bar { \textbf { 3 5 } } \mathbf { 1 0 ^ { 5 } }$ genome copies/PCR reaction.

The following limitation language should be included in the “limitation” section of the BD GeneOhm™ MRSA ACP Assay product insert: “The BD GeneOhm™ MRSA ACP Assay may cross-react with certain strains of methicillin-sensitive Staphylococcus aureus (MSSA) when they are present at extremely high concentrations. The BD GeneOhm™ MRSA ACP Assay generated 5 false positive results during an analytical specificity study testing a total of 111 well characterized MSSA strains at approximately ${ \bar { 3 } } x i \theta ^ { 5 }$ genome copies/PCR reaction $( \sim 7 x I 0 ^ { 7 } C F U / s w a b ) . ^ { \prime }$ ”

Overall, the analytical specificity of the BD GeneOhm™ MRSA ACP Assay determined in the analytical specificity study was $1 0 0 \%$ (189/189) at approximately 3 $\bf { x 1 0 ^ { 4 } }$ genome copies/PCR reaction, and was $9 7 . 4 \%$ (184/189) at approximately $3 \ : \mathrm { \bf ~ \dot { ~ } \mathbf { ~ 1 0 ~ } ^ { 5 } ~ }$ genome copies/PCR reaction.

i. Assay cut-offs:

There are 5 PCR parameter cut-offs (acceptance criteria) used by the SmartCycler II instrument and software to determine the results of clinical specimens and the validity of controls for the BD GeneOhm™ MRSA ACP Assay. They are:

Endpoint threshold (EP): the minimum endpoint that must be reached for the assay to be positive or the Internal Control (IC) to pass; $1 C \%$ endpoint threshold: the minimum $1 C \%$ endpoint of the assay IC calculated as a percentage of the IC endpoint of the NC;   
• $2 ^ { \mathrm { n d } }$ derivative threshold value (SDP): the minimum $2 ^ { \mathrm { n d } }$ derivative peak value that must be reached in order for a cycle threshold (CT) to be determined;   
• Valid minimum and maximum cycle thresholds (CTs): range within which the CT must be found for the assay to be positive or the IC to pass;

NOTES: Data acquired for the IC are only considered for specimens that are recorded negative. For data to be interpreted as a positive result, criteria for EP, SDP, and valid CTs for the assay must be met simultaneously. For data to be interpreted as a negative result, criteria for EP, SDP, $1 C \%$ endpoint and valid CTs for the IC must be met simultaneously.

The following table summarizes these PCR parameter cut-offs that had been determined/established in pre-clinical validation studies:

<table><tr><td rowspan=1 colspan=6>PCR parameter cut-offs for the BD GeneOhmTM MRSA ACP Assay</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Clinical specimens</td><td rowspan=1 colspan=1>Positive control (PC)</td><td rowspan=1 colspan=2>Negative control (NC)</td></tr><tr><td rowspan=1 colspan=1>PCR parameter</td><td rowspan=1 colspan=1>Assay (FAM)</td><td rowspan=1 colspan=1>IC (TET)</td><td rowspan=1 colspan=1>Assay (FAM)</td><td rowspan=1 colspan=1>Assay (FAM)</td><td rowspan=1 colspan=1>IC (TET)</td></tr><tr><td rowspan=1 colspan=1>Endpoint threshold (EP)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>2nd derivative threshold(SDP)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>301</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Minimum cycle threshold(CT)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Maximum cyclethreshold (CT)</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>IC% endpoint threshold</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>36%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

1Note: An assay cycle threshold value between 15 and 45, combined with assay endpoint and $2 ^ { \mathrm { n d } }$ derivative thresholds exceeding the specified values, invalidate a negative control.

Receiver operator characteristics (ROC) analysis was performed using the clinical data to verify/validate the cut-offs used in the clinical study. The data collected during the clinical performance study (samples and controls) validated the established PCR parameter cut-offs for the BD GeneOhm™ MRSA ACP Assay for nasal swab specimens and controls appropriately.

# $j$ . Interfering Substances:

The potential inhibitory effect of biological and chemical materials that may be present on nasal swab specimens on the BD GeneOhm™ MRSA ACP Assay was assessed. Substances used in bacterial culture, transport media, as well as biological and chemical substances occasionally used in the nares or found on nasal swab specimens, were tested. The amount of each compound to be tested was determined, and these values were considered as the highest amount likely to be found at the sampling site or on the nasal swabs specimen. All substances were tested in triplicate. The experiment was performed over 5 days. The inhibition level was measured on positive and negative samples. Reference samples without inhibitor were also tested. Negative tests were performed using $7 5 \mu \mathrm { L }$ of Sample Buffer. Positive tests were performed by using MRSA ATCC 43300 at 2 different

concentrations: 3X LoD and at typical clinical concentration (CT values in the FAM channel between 30.0 and 35.0).

Different sample preparation methods were used for samples containing liquid, solid or gel substances. For liquid substances, the required volume was dispensed directly in the Sample Buffer tube. Each sample $7 5 ~ \mu \mathrm { L }$ of bacterial suspension or Sample Buffer) was absorbed on a swab. The swab stem was then broken in the Sample Buffer tube containing the substance to test. The tubes were then vortexed for 60 seconds at maximum speed. For solid substances, a swab was rubbed on the substance before absorbing the $7 5 ~ \mu \mathrm { L }$ of sample. The swab stem was broken in the Sample Buffer tube. The tubes were then vortexed for 60 seconds at maximum speed. For gel substances, each sample ( $7 5 ~ \mu \mathrm { L }$ of bacteria suspension or Sample Buffer) was absorbed on a swab. Each swab stem was broken in a Sample Buffer tube. The tubes were then vortexed for 60 seconds at maximum speed. The eluted sample suspensions were transferred in the previously prepared $2 \ \mathrm { m L }$ tubes containing the substance, and the tubes were vortexed 5 seconds at maximum speed to solubilize the substance.

Control DNA was reconstituted with $2 2 5 ~ \mu \mathrm { L }$ of Sample Buffer. Ninety (90) $\mu \mathrm { L }$ of sample, Control DNA and Sample Buffer were subjected to the sample lysis process as indicated in the BD GeneOhm™ MRSA Lysis kit package insert. Three (3) ACP Lysis Reagent lots were tested,

Master Mix was reconstituted with $2 2 5 ~ \mu \mathrm { L }$ of Diluent. Twenty-five (25) $\mu \mathrm { L }$ of Master Mix were distributed in SmartCycler® tubes. Three (3) $\mu \mathrm { L }$ of sample were added to PCR tubes. For PCR controls, $3 ~ \mu \mathrm { L }$ of lysed Control DNA were added into one PC tube while $3 ~ \mu \mathrm { L }$ of lysed Sample Buffer were added into one NC tube for each P C R run. PCR analysis was performed on the SmartCycler II instrument.

The acceptance criteria is defined as specimens with potential inhibitor display the expected assay result and the appropriate CT relative value $( 9 5 \% - 1 0 5 \% )$ when compared to specimens without potential inhibitor. The formula used to calculate the relative CT value was the following:

200 - (average CT $\div$ average CT (reference) x 100)

All samples had acceptable expected results except for blood (in an excessive amount), which exhibited inhibition. It is noteworthy that the volume of blood tested was not representative of what is observed with clinical samples where only traces of blood are usually seen on swabs. Slight inhibition was noted for Rhinaris $^ \mathrm { \textregistered }$ and Secaris $^ \mathrm { \textregistered }$ based on the relative CT values observed, however, the expected positive and negative assay results were obtained.

Summary results of the Interfering Substances Study are presented in the following tables:

a) samples at 3X LoD   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>3 X LoD</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Dristan</td><td rowspan=1 colspan=1>Drixoral</td><td rowspan=1 colspan=1>Flonase</td><td rowspan=1 colspan=1>Nasonex</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>37.0</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>97</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Rhinaris</td><td rowspan=1 colspan=1>Secaris</td><td rowspan=1 colspan=1>TSB</td><td rowspan=1 colspan=1>MSA</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>35.8</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Stuart L</td><td rowspan=1 colspan=1>Amies G</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Nasal secr.</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>36.1</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.50</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>2/2</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Petr. jelly</td><td rowspan=1 colspan=1>Amies L</td><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>Otrivin</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>36.7</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>CHROMagar</td><td rowspan=1 colspan=1>Cromolyn</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>FAMCT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1: 3/3 UNR 2: One tube leaked during the vortex step so it was excluded from experiment

b) samples at typical clinical concentration (CT values in the FAM channel between 30.0 and 35.0)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Typical clinical concentration</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Dristan</td><td rowspan=1 colspan=1>Drixoral</td><td rowspan=1 colspan=1>Flonase</td><td rowspan=1 colspan=1>Nasonex</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>32.6</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.31</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Rhinaris</td><td rowspan=1 colspan=1>Secaris</td><td rowspan=1 colspan=1>TSB</td><td rowspan=1 colspan=1>MSA</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>31.5</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>98</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Stuart L</td><td rowspan=1 colspan=1>Amies G</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Nasal secr.</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>32.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.15</td></tr></table>

1: 1POS 2UNR   

<table><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Petr. jelly</td><td rowspan=1 colspan=1>Amies L</td><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>Otrivin</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>32.2</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>0.32</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>CHROMagar</td><td rowspan=1 colspan=1>Cromolyn</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

c) negative samples (Sample Buffer)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Sample Buffer TE</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Dristan</td><td rowspan=1 colspan=1>Drixoral</td><td rowspan=1 colspan=1>Flonase</td><td rowspan=1 colspan=1>Nasonex</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>35.9</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.06</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Rhinaris</td><td rowspan=1 colspan=1>Secaris</td><td rowspan=1 colspan=1>TSB</td><td rowspan=1 colspan=1>MSA</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Stuart L</td><td rowspan=1 colspan=1>Amies G</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Nasal secr.</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.23</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1> 20/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>Petr. jelly</td><td rowspan=1 colspan=1>Amies L</td><td rowspan=1 colspan=1>Saline</td><td rowspan=1 colspan=1>Otrivin</td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>35.3</td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.21</td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>CHROMagar</td><td rowspan=1 colspan=1>Cromolyn</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>FAM CT</td><td rowspan=1 colspan=1>Mean CT</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>CT Relative Value (%)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Assay Result (Pos)</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

1: one value excluded because fluorescence signal too high 2: 3/3 UNR

In conclusion, no inhibition was observed with 17 out of 18 substances tested with the BD GeneOhm™ MRSA ACP Assay. Inhibition was observed, for both positive (at the 2 tested loads) and for negative samples, when an excess of blood was present in specimens. Rhinaris $^ \mathrm { \textregistered }$ and Secaris $^ \mathrm { \textregistered }$ provoked slight inhibition, as measured by the relative CT value. Despite the slight inhibition noted with these substances, the expected positive and negative assay results were obtained.

The following limitation language should be included in the “limitation” section of the BD GeneOhm™ MRSA ACP Assay product insert: “An excess amount of blood in a specimen may inhibit the BD GeneOhm™ MRSA ACP Assay. Rhinaris $\textsuperscript { \textregistered }$ and Secaris $\textsuperscript { \textregistered }$ at high concentrations may cause slight inhibition in the BD GeneOhm™ MRSA ACP Assay.”

# k. Carry-Over Contamination:

To evaluate the risk of carry-over while processing specimens with high MRSA bacterial load in the BD GeneOhm™ MRSA ACP Assay, an analytical study was carried out to evaluate the entire process from sample preparation (including utilization of septum and screw caps on lysis tubes) to PCR result. Two (2) MRSA strains (MREJ type ii and vii) were used as the high positive MRSA panel members $\mathrm { 1 . 4 ~ X ~ i 0 } ^ { 4 }$ $5 . 1 \mathrm { ~ X ~ } 1 0 ^ { ^ { 4 } }$ .   
( genome copies/PCR reaction and genome copies/PCR reaction respectively). Negative members were prepared with Sample Buffer. Three (3) replicates of each high positive panel member (a total of 6 high positives) and 6 replicates of the negative panel member were tested by alternating negative and positive samples. Each run contained positive and negative controls prepared according to the package insert. All panel members and controls were processed following the BD GeneOhm™ MRSA ACP Assay package insert. The same experiment was performed by 4 operators with each of the two types of lysis tube cap (septum and screw caps). The experiment was performed a second time 72 hours later using the original run controls from the first experiment to assess for possible contamination during storage.

Pooled results of the carry-over study are presented in the following table:

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>FAM channel results</td><td rowspan=1 colspan=2>Control validity</td></tr><tr><td rowspan=1 colspan=1>MRSA type ii</td><td rowspan=1 colspan=1>MRSA type vii</td><td rowspan=1 colspan=1>Sample Buffer(TE)</td><td rowspan=1 colspan=1>Positivecontrol</td><td rowspan=1 colspan=1>NegativeT CControl</td></tr><tr><td rowspan=1 colspan=1>First experiment</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>0/48</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr><tr><td rowspan=1 colspan=1>Second experiment (SamePC/NC 72 hrs at 4°C)</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>1/481</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>8/8</td></tr></table>

The positive sample showed no target amplicon in the gel electrophoresis analysis. Therefore it was likely not due to carry-over contamination.

For 95 out of 96 negative panel members tested, negative results were observed as expected. One false positive result was obtained. However, gel electrophoresis analysis demonstrated no target amplicon.

In conclusion, no false positive result due to carry-over contamination was observed during a total of 16 runs (8 with septum caps tubes and 8 with screw caps tubes by 4 operators). Storage of the PC and NC tubes at $4 ^ { \mathbf { 0 } } \mathbf { C }$ for 72 hours and the subsequent use in retesting did not lead to invalid runs or carry-over contamination.

l. Molecular Beacon Probe Cross-hybridization Study

A study was conducted to demonstrate the specificity of each molecular beacon probe for its respective target of the BD GeneOhm™ MRSA ACP Assay. Specific molecular beacon probes were designed using DNA sequence databases and sequence alignments. Briefly, probe XSauB5A0 targets MRSA MREJ types i, ii, iii, iv, and v. Probe XSauB8A0 differs from XSauB5A0 by a single base and targets MRSA MREJ type vii. Probe SignB4B0 targets the internal control (IC). In order to assess the specificity of each beacon, a matrix of Master Mix formulations was created allowing selective inclusion of the molecular beacons. The experimental conditions created in order to assess each beacon individually are outlined in the f o l l o w i n g t a b l e :

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Molecular Beacons and IC presence in Master Mix</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>XSauB5A0</td><td rowspan=1 colspan=1>XSauB8A0</td><td rowspan=1 colspan=1>SignB4B0</td><td rowspan=1 colspan=1>IC (pSarmf)</td></tr><tr><td rowspan=1 colspan=1>Master Mix a</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Master Mix b</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+(2,000 cp/rxn)</td></tr><tr><td rowspan=1 colspan=1>Master Mix c</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(200,000 cp/rxn)</td></tr><tr><td rowspan=1 colspan=1>Master Mix d</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Master Mix e</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(200,000 cp/rxn)</td></tr><tr><td rowspan=1 colspan=1>Master Mix f</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr></table>

Each Master Mix was made of different stock solutions to increase possible variability. As a result, 3 different lots of Master Mix components were used allowing for testing of beacon specificity in triplicate. High concentrations of DNA (MREJ types or IC) were added in order to challenge cross-reactivity. At the end of the amplification protocol, all assay tubes were analyzed by agarose gel electrophoresis in order to verify that the expected amplicon had been produced.

This study demonstrated that each molecular beacon of the BD GeneOhm™ MRSA ACP Assay is specific to its respective target. No evidence of crosshybridization between the XSauB5A0 or XSauB8A0 beacon and the IC amplification product was observed even in the presence of high concentrations of IC. No evidence of cross-hybridization between the IC beacon (SignB4B0) and MRSA MREJ type i, ii, iii, iv, v and vii amplification products was observed,

# even in the presence of high concentrations of MRSA amplification product.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

$^ b$ . Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Clinical studies

Performance characteristics of the BD GeneOhm™ MRSA ACP Assay were determined in a multi-site prospective investigational study. Three geographically diverse U.S. clinical sites having Methicillin-resistant Staphylococcus aureus (MRSA) culture-based methods already in place participated in the study. Patients were included in the study only if they were eligible for MRSA testing according to institutional policies. All specimens in the study meeting the inclusion and exclusion criteria represented excess, de-identified left over material from normal standard of care laboratory testing. All clinical sites were granted waivers of informed consent by their IRB for this study. Eligibility requirements for targeted screening as per clinical sites policies included, but were not limited to: all patients admitted into the particular healthcare system; patients admitted to the Intensive Care Unit; patients transferred to the Intensive Care Unit; pre-elective surgery patients; and patients being admitted from long term care facilities. Patients previously enrolled in the study were denied repeat entry.

All clinical sites used the same culture media and procedure for S. aureus recovery, confirmation that conformed to CLSI M47-A was used as comparative reference method for assessing the performance of the BD GeneOhm™ MRSA ACP Assay. The culture method consisted of an initial analysis on BBL™ CHROMagar™ Staph aureus selective chromogenic media (Class I exempt) followed by subculture on Blood Agar (BA) of presumptive S. aureus colonies. S. aureus identification was confirmed with an agglutination test, the BBL™ Staphyloslide™ agglutination latex test (k940557), from the subcultured BA plate. Susceptibility testing was performed on all confirmed Staphylococcus aureus colonies at all study sites in accordance with the CLSI documents M2-A9 and M100-S17, using the Cefoxitin disc testing to detect methicillin/oxacillin resistance $< = 2 1 \ \mathrm { m m } =$ resistant; $> = 2 2 ~ \mathrm { m m } =$ susceptible). Enrichment in Trypticase Soy Broth (TSB) (BBLTM TrypticaseTM Soy Broth with $6 . 5 \%$ Sodium Chloride, Class I exempt) was completed in the event that methicillin-resistant $S .$ . aureus was not confirmed by the initial method. The TSB was used to inoculate additional BBL™ CHROMagar™ Staph aureus selective chromogenic media (Class I exempt) and BA plates, and MRSA confirmation was performed as described above.

The BD GeneOhm™ MRSA ACP Assay procedure was performed according to the BD GeneOhm™ MRSA ACP Lysis kit and BD GeneOhm™ MRSA ACP Assay Package Inserts. The BD GeneOhm™ MRSA ACP Assay positive (PC) and negative $\mathrm { ( N C ) }$ controls were tested with each batch of samples analyzed in a SmartCycler® run to monitor Assay performance. For each run in which samples were processed through the lysis step, appropriate positive and negative SPCs (A reference MRSA strain ATCC 43300 was used as a positive SPC and a reference strain of methicillin-susceptible S. aureus ATCC 25923 was used as a negative SPC) were tested in addition to the Assay positive and negative controls.

There were 1296 specimens enrolled for this clinical study. Of those, 80 specimens were found to be non-compliant and were excluded from performance analysis. See the table below for details:

<table><tr><td rowspan=2 colspan=1>Compliance</td><td rowspan=2 colspan=1>Description</td><td rowspan=1 colspan=3>Site Name</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Site 1EVAN</td><td rowspan=1 colspan=1>Site 2NORT</td><td rowspan=1 colspan=1>Site 3WISC</td></tr><tr><td rowspan=4 colspan=1>N</td><td rowspan=1 colspan=1>Broth placed at 2-8°</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Swab held after collection at 2-8°C for &gt;5 days before the brothwas inoculated</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Swab held after collection at 15-25°C for &gt;48h before the brothwas inoculated</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Control strategy not followed</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>Total Compliant</td><td rowspan=1 colspan=1>579</td><td rowspan=1 colspan=1>308</td><td rowspan=1 colspan=1>329</td><td rowspan=1 colspan=1>1216</td></tr><tr><td rowspan=1 colspan=2>Total of Specimens Enrolled</td><td rowspan=1 colspan=1>581</td><td rowspan=1 colspan=1>329</td><td rowspan=1 colspan=1>386</td><td rowspan=1 colspan=1>1296</td></tr></table>

A total of 1228 nasal swab specimens were tested with both the comparative reference method and the BD GeneOhm™ MRSA ACP Assay, producing 1216 reportable results. Of these, 12 nasal swab specimens were excluded from performance analysis due to not following the appropriate control strategy specified in the clinical study protocol (refer to the table above). Out of the 1216 nasal swab specimens in compliance with the clinical study protocol that were tested with the BD GeneOhm™ MRSA ACP Assay, 12 $( 1 . 0 \% )$ were initially reported as unresolved. Upon repeat testing from the frozen lysates per the product package insert, all 12 had reportable results. The following table summarized the initial “unresolved” rates and the “unresolved” rates after retest:

<table><tr><td rowspan=1 colspan=1>Clinical Sites</td><td rowspan=1 colspan=2>Initial unresolved rate with 95% CI</td><td rowspan=1 colspan=2>Unresolved rate after repeat with 95% CI</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>1.0% (6/579)</td><td rowspan=1 colspan=1>(0.4% - 2.2%)</td><td rowspan=1 colspan=1>0.0% (0/579)</td><td rowspan=1 colspan=1>(0.0% - 0.6%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>0.3% (1/308)</td><td rowspan=1 colspan=1>(0.0% - 1.8%)</td><td rowspan=1 colspan=1>0.0% (0/308)</td><td rowspan=1 colspan=1>(0.0% - 1.2%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>1.5% (5/329)</td><td rowspan=1 colspan=1>(0.5% - 3.5%)</td><td rowspan=1 colspan=1>0.0% (0/329)</td><td rowspan=1 colspan=1>(0.0% - 1.1%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1.0% (12/1216)</td><td rowspan=1 colspan=1>(0.5% - 1.7%)</td><td rowspan=1 colspan=1>0.0% (0/1216)</td><td rowspan=1 colspan=1>(0.0% - 0.3%)</td></tr></table>

Three (3) runs were reported “invalid” due to Run Control failure $( 4 . 4 \% )$ . The runs were reported valid upon repeat testing of the specimen lysates per the product package insert. The following table summarized the “invalid” run rates:

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>Invalid Run Rates with 95% CI</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>0.0% (0/25)</td><td rowspan=1 colspan=1>(0.0% - 13.7%)</td></tr><tr><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>4.5% (1/22)</td><td rowspan=1 colspan=1>(0.1% - 22.8%)</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>9.5% (2/21)</td><td rowspan=1 colspan=1>(1.2% - 30.4%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>4.4% (3/68)</td><td rowspan=1 colspan=1>(0.9% - 12.4%)</td></tr></table>

In comparison to the comparative reference method, the BD GeneOhm™ MRSA ACP Assay identified $9 2 . 0 \%$ of the MRSA positive specimens and $9 4 . 6 \%$ of the negative specimens. For the combined population tested at the three clinical sites, this resulted in a Negative Predictive Value (NPV) of $9 8 . 5 \%$ and a Positive Predictive Value (PPV) of $7 5 . 4 \%$ .

The performance data stratified by study site are presented in the following tables:

<table><tr><td rowspan=2 colspan=1>BDGeneOhmTMMRSA ACPAssay ResultsSite 1</td><td rowspan=1 colspan=3>Comparative Culture ReferenceMethod Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>495</td><td rowspan=1 colspan=1>499</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>512</td><td rowspan=1 colspan=1>579</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 63/67   94.0% (95% CI: 85.4% - 98.3%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 495/512 96.7% (95% CI: 94.7% - 98.1%)</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4>Prevalence as determined by the comparative culturereference method: 67/579 11.6%</td></tr><tr><td rowspan=1 colspan=4>PPV: 63/80   78.8% (95% CI: 68.2% -78.1%)</td></tr><tr><td rowspan=1 colspan=4>NPV: 495/499 99.2% (95% CI: 98.0% - 99.8%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>BDGeneOhmTMMRSA ACPAssay ResultsSite 2</td><td rowspan=1 colspan=3>Comparative Culture ReferenceMethod Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>228</td><td rowspan=1 colspan=1>234</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>254</td><td rowspan=1 colspan=1>308</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 48/54   88.9% (95% CI: 77.4% - 95.8%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 228/254 89.8% (95% CI: 85.4% - 93.2%)</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4>Prevalence as determined by the comparative culturereference method: 56/320* 17.5%</td></tr><tr><td rowspan=1 colspan=4>PPV: 48/74     64.9% (95% CI: 52.9% - 75.6%)</td></tr><tr><td rowspan=1 colspan=4>NPV: 228/234  97.4% (95% CI: 94.5% - 99.1%)</td></tr></table>

$^ { * 1 2 }$ samples were excluded from the performance analysis due to not following the appropriate control strategy specified in the clinical study protocol for the BD GeneOhm™ MRSA ACP Assay

<table><tr><td rowspan="3">BD GeneOhmTM MRSA ACP</td><td colspan="3">Comparative Culture Reference Method Results</td></tr><tr><td>Positive</td><td>Negative</td><td>Total</td></tr><tr><td>Assay Results Site 3</td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">74</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">250</td><td colspan="1" rowspan="1">255</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">263</td><td colspan="1" rowspan="1">329</td></tr><tr><td colspan="4" rowspan="1">Sensitivity: 61/66    92.4% (95% CI: 83.2% - 97.5%)</td></tr><tr><td colspan="4" rowspan="1">Specificity: 250/263  95.1% (95% CI: 91.7% - 97.3%)</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td colspan="4" rowspan="1">Prevalence as determined by the comparative culturereference method: 66/32920.1%</td></tr><tr><td colspan="4" rowspan="1">PPV: 61/74    82.4% (95% CI: 71.8% - 90.3%)</td></tr><tr><td colspan="4" rowspan="1">NPV: 250/255 98.0% (95% CI: 95.5% - 99.4%)</td></tr></table>

The performance data of all study sites combined are presented in the following table:

<table><tr><td rowspan=2 colspan=1>BDGeneOhmTMMRSA ACPAssay ResultsAll Sites</td><td rowspan=1 colspan=3>Comparative Culture ReferenceMethod Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>228</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>973</td><td rowspan=1 colspan=1>988</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>1029</td><td rowspan=1 colspan=1>1216</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 172/187   92.0% (95% CI: 87.1% - 95.4%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 973/1029  94.6% (95% CI: 93.0% - 95.9%)</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4>Prevalence as determined by the comparative culturereference method: 189/ 1228* 15.4%</td></tr><tr><td rowspan=1 colspan=4>PPV: 172/228 75.4% (95% CI: 69.9% - 81.0%)</td></tr><tr><td rowspan=1 colspan=4>NPV: 973/988 98.5% (95% CI: 97.5% - 99.2%)</td></tr></table>

$^ { * 1 2 }$ samples were excluded from the performance analysis due to not following the appropriate control strategy specified in the clinical study protocol for the BD GeneOhm™ MRSA ACP Assay

The Clinical Sites were instructed to record the swab type used for specimen collection. Analyses were also performed per swab transport medium, allowing data stratification as shown in the following table:

<table><tr><td rowspan=1 colspan=1>Swab Media</td><td rowspan=1 colspan=2>Sensitivity(95% CI)</td><td rowspan=1 colspan=2>Specificity 5% C)</td></tr><tr><td rowspan=1 colspan=1>Amies Gel w/o Charcoal</td><td rowspan=1 colspan=1>92.9% (52/56)</td><td rowspan=1 colspan=1>(82.7% - 98.0%)</td><td rowspan=1 colspan=1>93.9% (153/163)</td><td rowspan=1 colspan=1>(89.0% - 97.0%)</td></tr><tr><td rowspan=1 colspan=1>Liquid Amies</td><td rowspan=1 colspan=1>92.6% (63/68)</td><td rowspan=1 colspan=1>(83.7% - 97.6%)</td><td rowspan=1 colspan=1>96.7% (495/512)</td><td rowspan=1 colspan=1>(94.7% - 98.1%)</td></tr><tr><td rowspan=1 colspan=1>Liquid Stuart</td><td rowspan=1 colspan=1>90.5% (57/63)</td><td rowspan=1 colspan=1>(80.4% - 96.4%)</td><td rowspan=1 colspan=1>91.8% (325/354)</td><td rowspan=1 colspan=1>(88.4% - 94.4%)</td></tr><tr><td rowspan=1 colspan=1>Overall study</td><td rowspan=1 colspan=1>92.0% (172/187)</td><td rowspan=1 colspan=1>(87.1% - 95.4%)</td><td rowspan=1 colspan=1>94.6% (973/1029)</td><td rowspan=1 colspan=1>(93.0% - 95.9%)</td></tr></table>

b. Retrospective Clinical studies Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off:

Not applicable.

5. Expected values/Reference range:

In the BD GeneOhm™ MRSA ACP Assay clinical study, a total of 1228 specimens were tested from three geographically diverse U.S. clinical sites. The study population was grouped into subjects in Intensive Care, Long Term Care Facility, and Medical Services. The number and percentage of positive and negative cases as determined by the comparative culture reference method are calculated and presented in the table below:

<table><tr><td rowspan=2 colspan=2>Group                   Total N</td><td rowspan=2 colspan=2>MRSA By CultureNumber Positive      Number Negative</td><td rowspan=2 colspan=1>Observed Prevalence</td></tr><tr><td rowspan=1 colspan=1>Total N</td><td rowspan=1 colspan=1>Number Positive</td></tr><tr><td rowspan=1 colspan=1>Intensive Care</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>7.3%</td></tr><tr><td rowspan=1 colspan=1>Long Term Care Facility</td><td rowspan=1 colspan=1>263</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>22.1%</td></tr><tr><td rowspan=1 colspan=1>Medical Services1</td><td rowspan=1 colspan=1>828</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>707</td><td rowspan=1 colspan=1>14.6%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1228</td><td rowspan=1 colspan=1>189</td><td rowspan=1 colspan=1>1039</td><td rowspan=1 colspan=1>15.4%</td></tr></table>

1 The Medical Services category includes specimens identified as Medical, OB/GYN, Pediatric, Surgical and Other.

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b

O. System Descriptions:

1. Modes of Operation: The Cepheid SmartCycler II Real Time Instrument operates in random access mode.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

User enters Patient ID/Sample ID

4. Specimen Sampling and Handling:

Not applicable

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The information submitted in this premarket notification is complete and supports a substantial equivalence decision.