# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

k093815

Clearance of new device

C. Measurand:

Viral antigens of the nuclear protein and fusion protein of the human metapneumovirus species

D.Type of Test:

Direct Fluorescence Antibody (DFA) test using direct specimens

E. Applicant:

Millipore Corporation

# F. Proprietary and Established Names:

Light Diagnostics™ Human Metapneumovirus Direct Immunofluorescence Assay Light Diagnostics™ Human Metapneumovirus DFA Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR §866.3980

2. Classification: Class II

3. Product code: OMG (Antisera, fluorescent, human metapneumovirus)

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

Light Diagnostics Human Metapneumovirus Direct Immunofluorescence Assay is intended for qualitative detection and identification of human metapneumovirus (hMPV) in direct respiratory specimen cell preparations from nasopharyngeal swab samples collected from patients with febrile respiratory illness. This assay detects but is not intended to differentiate the four recognized genetic sublineages of hMPV.

Negative results do not preclude hMPV infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. It is recommended that specimens found to be negative after examination of the direct specimen results be confirmed by FDA cleared hMPV molecular assay.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

The instrument used is a general laboratory use instrument. The instrument required for this device is a fluorescence microscope with 100 watt mercury or halogen lamp, appropriate filter combination for FITC (excitation peak at $4 9 0 \mathrm { n m }$ , emission peak at $5 2 0 \mathrm { n m }$ ); and 100X, 200X, and 400X (optional) magnification (dry objective).

# I. Device Description:

Light Diagnostics™ hMPV Direct Immunofluorescence Assay is a qualitative test that uses a specific typing reagent and FITC (fluorescein isothiocyanate) filter fluorescence microscope to detect and identify human metapneumovirus (hMPV) in direct specimens from symptomatic patients with respiratory illness. The kit consists of one vial hMPV Reagent, two hMPV Control Slides, a PhosphateBuffered Saline Packet, a Tween 20/Sodium Azide Solution (100X) vial, and a Mounting Fluid vial. The kit utilizes a specific reagent for the detection and identification of hMPV. One specimen cell spot on a slide is necessary to perform the test. Unbound reagent is removed by rinsing with phosphate-buffered saline (PBS). Illumination allows visualization of the antigen-antibody complexes by fluorescence microscopy. hMPV-infected cells will exhibit apple-green fluorescence with the specific reagent while cells stain a dull red due to the presence of Evans blue in the reagents. The controls contained in this kit are acetone fixed slides, each with one well hMPV, subtype A infected cells, one well with hMPV, subtype B infected cells, and two wells of uninfected cells to verify functioning of reagents, methodology and functioning of the microscope.

Materials Provided

1. hMPV Reagent - (Catalog No. 5091) One $2 \mathrm { m L }$ dropper vial containing FITClabeled anti-hMPV monoclonal antibodies , protein stabilizer, Evans blue and $0 . 1 \%$ sodium azide as preservative.   
2. hMPV Control Slides - (Catalog No. 5092). Two control slides containing two hMPV (A and B) infected (positive) wells and two non-infected (negative) wells.   
3. Phosphate Buffered Saline - (Catalog No. 5087) phosphate buffered saline salts.   
4. Tween 20 / Sodium Azide Solution (100X) - (Catalog No. 5037). One $1 0 ~ \mathrm { m L }$ vial containing Tween 20/sodium azide concentrate $2 \%$ sodium azide).   
5. Mounting Fluid - (Catalog No. 5013) One $1 0 ~ \mathrm { m L }$ dropper vial containing Tris

buffered glycerin, a fluorescence enhancer, and $0 . 1 \%$ sodium azide as preservative.

Materials Required But Not Provided

1. Viral transport medium, which is non-inhibitory to human metapneumovirus and the tissue culture cells, used for viral isolation.   
2. Acetone, reagent grade; stored in glass container; stored separate from hMPV Reagent.   
3. Acetone-cleaned glass slides with wells $6 { - } 8 \mathrm { m m }$ in diameter, nonfluorescing   
4. Aspirator device with disposable sterile Pasteur pipettes.   
5. Humid chamber.   
6. Sodium hypochlorite solution $0 . 0 5 \%$ , (1:100 dilution of household bleach)   
7. No. 1 coverslips.   
8. Incubator, $3 7 \pm 1 ^ { \circ } \mathrm { C }$ .   
9. Vials for collection and transport of specimens.   
10. Fluorescence microscope with 100 watt mercury or halogen lamp, appropriate filter combination for FITC (excitation peak $= 4 9 0 \ \mathrm { n m }$ , emission peak $= 5 2 0 \mathrm { n m }$ ), $1 6 0 { - } 2 0 0 \mathrm { x }$ and $4 0 0 \mathrm { x }$ magnification (dry objective).   
11. Distilled or deionized water.   
12. Vortex mixer.

Overview of the procedure:

PBS resuspended cells, derived from a clinical specimen, are placed onto a glass slide and allowed to air dry completely. The cells are fixed in acetone for 10 minutes. After they have been allowed to air dry, one drop of the hMPV Reagent is added to cover the cells which are then incubated at $3 7 ^ { \circ } \mathrm { C }$ for 15 minutes in a humidified chamber. The stained cells are then washed with the diluted PBS/Tween 20 for 10-15 seconds and rinsed. A drop of the Mounting Fluid is added and a cover slip is placed on the slide. The cells are examined using a fluorescence microscope. The hMPV infected cells will exhibit an apple-green fluorescence. Uninfected cells will stain a dull red due to the presence of Evans blue in the reagent.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

During the study no FDA cleared predicate device was available. The performance characteristics of the Light Diagnostics™ Human Metapneumovirus DFA Assay were established by direct evaluation of clinical specimens compared to a composite algorithm based on cell culture results and a validated RT-PCR method followed by bidirectional sequencing for confirmation and identification of human metapneumovirus.

The methods differ in that: Light Diagnostics™ Human Metapneumovirus DFA Assay contains a single reagent which contains FITC-labeled monoclonal antibodies directed against hMPV. This reagent detects hMPV antigens present in the patient specimen. Slides are necessary to detect and identify the virus from sample and yield an apple-green fluorescence color if positive. The RTPCR/sequencing method detects viral RNA isolated from patient specimens.

A more recently FDA cleared similar device could now serve as main predicate: $\mathrm { D } ^ { 3 }$ DFA Metapneumovirus Identification Kit (DHI, k090073). Characteristics of the Light Diagnostics™ Human Metapneumovirus DFA Assay are compared to those of such predicate device in the Table below:

# 2. Comparison with main predicate:

<table><tr><td rowspan=1 colspan=2>Technological Characteristics Comparison of Devices</td></tr><tr><td rowspan=1 colspan=1>Light Diagnostics ™ Human MetapneumovirusDFA Assay (Device)</td><td rowspan=1 colspan=1>D³ DFA Metapneumovirus Identification Kit(Predicate)</td></tr><tr><td rowspan=1 colspan=1>Target:</td><td></td></tr><tr><td rowspan=1 colspan=1>Screening selection of unique licensed proprietarymonoclonal antibody clones characterized byimmunofluorescence and ELISA to be the bestreactive alternatives for conserved epitopes of allhMPV subtypes.</td><td rowspan=1 colspan=1>Searches of the National Center forBiotechnology Information (NCBl) databases yieldedpresumptive evidence that the target for each of the3 MAb clones is the MPV nucleoprotein. Nineproteins are known to be encoded in the hMPVgenome. Of the nine proteins, only the nucleoproteinis of a size equivalent to the 46 kDa size noted onthe Western Blot of the 3 MAb clones.</td></tr><tr><td rowspan=1 colspan=2>Specimen:</td></tr><tr><td rowspan=1 colspan=1>Nasopharyngeal swab specimens</td><td rowspan=1 colspan=1>Nasal and nasopharyngeal swabs and aspirates orcell culture.</td></tr><tr><td rowspan=1 colspan=2>Detection Methods:</td></tr><tr><td rowspan=1 colspan=1>PBS resuspended cells, derived from a clinicalspecimen, are placed onto a glass slide and allowedto air dry completely. The cells are fixed in acetonefor 10 minutes. After they have been allowed to airdry, one drop of the hMPV Reagent is added tocover the cells which are then incubated at 37°C for15 minutes in a humidified chamber. The stainedcells are then washed with the diluted PBS/Tween20 for 10-15 seconds and rinsed. A drop of theMounting Fluid is added and a cover slip is placedon the slide. The cells are examined using afluorescence microscope. The hMPV infected cellswill exhibit an apple-green fluorescence. Uninfectedcells will stain a dull red due to the presence ofEvans blue in the reagent.</td><td rowspan=1 colspan=1>The assay detects specific hMPV viral antigens byimmunofluorescence using monoclonal antibodies(MAbs). The cells to be tested, derived from aclinical specimen or cell culture, are placed onto aglass slide and allowed to air dry. The cells are fixedin acetone. The hMPV DFA reagent is added to thecells which are then incubated for 15 to30 minutes at 35°C to 37°C in a humidified chamberor humidified incubator. The stained cells are thenwashed with the diluted phosphate buffered saline(PBS), a drop of the supplied Mounting Fluid isadded and a cover slip is placed on the preparedcells. The cells are examined using a fluorescencemicroscope. The hMPV infected cells will fluoresceapple-green. Uninfected cells will contain nofluorescence but will be stained red by the EvansBlue counter-stain.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

CLSI EP15-A2, “User Verification of Performance for Precision and Trueness;   
Approved Guideline- Second Edition”.

CLSI M41-A, "Viral Culture; Approved Guideline”.

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006).

Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests;   
Guidance for Industry and FDA Reviewers (March 2007).

# L. Test Principle:

Light Diagnostics Human Metapneumovirus DFA Assay utilizes a single reagent for the detection and identification of human metapneumovirus (hMPV) in cell preparations made from respiratory specimens. A blend of two monoclonal antibodies, specific for hMPV is labeled with FITC (fluorescein isothiocyanate) which fluoresces apple-green. The labeled antibody will bind to viral antigen present in the specimen in hMPV infected cells. Unbound reagent is removed by washing with buffer. Cells in positive specimens will fluoresce apple-green while uninfected cells will stain dull red due to the presence of Evans blue in the reagent.

M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

# a. Precision/Reproducibility:

Assay precision and reproducibility studies were performed at three sites (two external and one internal site). The internal precision study was conducted using the protocol recommended in CLSI EP15-A2, with the exceptions that two replicate samples of each subtype were run each day, and two runs were analyzed each day. The specific protocol was to analyze two runs per day with two replicate samples for each of two subtypes daily for five days. All other procedures in CLSI EP15-A2, section 8.2, were followed. In addition, the protocol was run in duplicate by two operators (i.e., each operator analyzed two runs per day for five days).

Because the statistical methods in CLSI EP15-A2 refer to quantitative tests and the Light Diagnostics™ Human Metapneumovirus DFA Assay yields qualitative results, precision study results were analyzed for accordance with expected results. Results were $100 \%$ in accordance with expected results (80/80 tests). There were no invalid or equivocal results.

The reproducibility study was conducted at three clinical sites (the internal site plus the external Sites 1 and 2 from the clinical study). Each site evaluated the Light Diagnostics™ hMPV DFA Reagent using a five day testing protocol with two runs per day and three replicates of each panel member per test, and two operators performing the test each day. At Site 1, each operator performed one run per day; at Site 2, each operator performed two runs per day. The test panel consisted of one true negative sample and samples at three levels of viral load including a high negative (sample with an analyte concentration

below the LoD such that repeat testing of this sample should be negative at least $9 5 \%$ of the time), a low positive sample, and a moderate positive sample. Results were in accordance with expected results greater than $9 9 \%$ of the time (438/440 tests). Precision/reproducibility results are summarized in the following table:

<table><tr><td rowspan=1 colspan=1>Number ofOperators</td><td rowspan=1 colspan=1>Numberof Sites</td><td rowspan=1 colspan=1>Viral controllevel</td><td rowspan=1 colspan=1>ExpectedResults</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Accordance withexpected results</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive90/90</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive130/130</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>High Negative(viralconcentrationbelow Limit ofDetection</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative88/90Positive2/90</td><td rowspan=1 colspan=1>97%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Uninfected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative130/130</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=4>Overall</td><td rowspan=1 colspan=1>438/440</td><td rowspan=1 colspan=1>99%</td></tr></table>

# Quality Control:

Control slides should be stained each time the staining procedure is performed to ensure proper test performance. The control slides supplied in the kit are intended to demonstrate the proper function of kit components and immunofluorescence staining procedure. The controls included in the kit are acetone fixed slides with human metapneumovirus (hMPV) infected and uninfected cells. Each control slide has four wells. The positive control wells are fixed with hMPV reference strain in the two wells, one with hMPV subtype A, one with hMPV subtype B. The negative control is fixed with uninfected cells in two wells. Control slides should be tested with each sample batch. The control slides supplied in the kit are intended to demonstrate the proper function of reagents, culture methodology and functionality of the microscope. The control slides are for one time use only and are not subjected to open and close stability study. Control slides prepared from other reference hMPV infected and uninfected cells may also be used to ensure proper staining procedures during analysis. When viewed in a fluorescence microscope an hMPV positive reaction is indicated by a bright apple-green fluorescence in the nucleus and/or cytoplasm of the infected cells in the appropriate well. Uninfected cells in the negative control well will exhibit dull red fluorescence. Results for patient samples should not be reported until controls perform as expected.

b. Linearity/assay reportable range: N/A   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Stability:

Reagents: store at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ up to the stated expiration date. Shelf life recommendations based on the accelerated stability study performed during development of the kit; with a real time stability study follow-up.

Control slides: at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ ; one time use only and are not subjected to open and close stability study. Expiration date based on shortest dated kit component (antibody reagent in the kit).

Samples: Specimens should not be frozen before testing. They may be maintained at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to 72 hours without loss of viability, as per the general protocols for clinical specimen collection and handling, such as those outlined in CLSI M41-A: "Viral Culture; Approved Guideline."

# d. Analytical Sensitivity (Detection limits):

Analytical detection limits for the Light Diagnostics™ hMPV DFA Reagent was tested using slides made from titrated virus culture. Four strains of hMPV were used, representing all four known subtypes of hMPV (A1, A2, B1, and B2). Five ten-fold dilutions were made from stocks of known concentration for each strain. Samples of virus at each dilution were dropped on slides in duplicate, and four batches of slides were prepared from four cultures (eight replicates at each concentration). Positive results were obtained with at least $9 5 \%$ of replicates using the Light Diagnostics™ hMPV DFA Reagent on hMPV A1 infected cells down to $4 0 0 \ { \mathrm { P F U / m L } } ,$ 100 PFU/mL of hMPV A2 infected cells, $6 2 5 ~ \mathrm { P F U / m L }$ of hMPV B1 infected cells, and $2 7 5 ~ \mathrm { P F U / m L }$ of hMPV B2 infected cells. Results are summarized in the following table.

<table><tr><td rowspan=1 colspan=1>Virus strain</td><td rowspan=1 colspan=1>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>4.0 x 102 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>1.0 x 102 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>6.25 x 102 PFU/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV B2</td><td rowspan=1 colspan=1>2.75 x 102 pFU/mL</td></tr></table>

e. Analytical Reactivity:

In addition to the Analytical Sensitivity results, a study was performed to demonstrate that the Light Diagnostics™ hMPV DFA Reagent detects virus isolates representing each of the four known genetic sublineages of hMPV (A1, A2, B1, B2). Positive specimens from the clinical study were genotyped to further support this study. The Light Diagnostics™ hMPV DFA Reagent detected hMPV in 21 clinical isolates from sublineage A2, 10 from sublineage B1, and 32 from sublineage B2 (including positive specimens from Site 3). No specimens from the A1 sublineage were received in the clinical study.

# f. Analytical specificity:

Cross-reactivity:

A study was performed to determine reactivity when staining slides with the Light Diagnostics™ hMPV DFA Reagent were tested against a variety of viruses and bacteria found in the respiratory tract, and cell lines commonly used to isolate respiratory viruses. Bacteria were grown and colonies containing $1 0 ^ { 6 } \mathrm { C F U / m L }$ were dried onto slides. Viruses stocks with concentrations in the order of $1 0 ^ { 5 } \mathrm { P F U / m L }$ were dropped on slides and fixed with acetone. Cross reactivity was defined as any positive (apple-green fluorescence) reaction in cells other than human metapneumovirus infected cells.

Study procedure:

1. Slides were brought to room temperature   
2. Slides were stained with the Light Diagnostics™ hMPV DFA Reagent. In addition, each slide was either stained with positive control reagent, or crystal violet (when no positive control reagent was available).   
3. Slides were incubated for 30 minutes at $3 7 ^ { \circ } \mathrm { C }$ .   
4. Slides were rinsed with PBS, mounted and read at 200X magnification with a fluorescent microscope.

The results in the table below show that there is no cross-reactivity between the hMPV Reagent when tested with viruses, bacteria, or cell lines.

<table><tr><td colspan="1" rowspan="1">Cross-reactant</td><td colspan="1" rowspan="1">Titer</td><td colspan="1" rowspan="1">HumanmetapneumovirusDFA reactivity</td></tr><tr><td colspan="1" rowspan="1">Adenoviruses</td><td colspan="1" rowspan="1">$10} PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Herpes viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV type 1</td><td colspan="1" rowspan="1">$10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV type 2</td><td colspan="1" rowspan="1">10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CMV</td><td colspan="1" rowspan="1">$10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VZV</td><td colspan="1" rowspan="1">10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza virus A</td><td colspan="1" rowspan="1">$10^} PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">10}PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus (RSV)</td><td colspan="1" rowspan="1">10 PFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A549</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">BGMK</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Hep-2</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">LLC-MK2</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MDCK</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MRC-5</td><td colspan="1" rowspan="1">10 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td>RD</td><td>10 CFU/mL</td><td></td></tr><tr><td>Vero</td><td>10 CFU/mL</td><td></td></tr></table>

Protein A, produced by certain bacteria, will bind to the Fc portion of the monoclonal antibodies used in the hMPV Reagent. Staining, however, can be differentiated by size and morphology. The presence of Staphylococcus Aureus and protein A will result in small $( 0 . 8 \mu \mathrm { m } )$ fluorescent spheres.

# Interference:

A study was performed to determine potentially interfering substances with the Light Diagnostics™ hMPV DFA Reagent. Substances considered were those that could potentially be present in nasopharyngeal swab specimens. Substances were added to negative samples (uninfected cultured cells), hMPV A infected cells, and hMPV B infected cells. Interference was defined as any negative results on hMPV infected cells in the presence of an interfering substance. The results in the table below show that there is no interference between the Light Diagnostics™ hMPV DFA Reagent when tested substances listed.

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Conc.</td><td rowspan=1 colspan=3>Results with Light Diagnostics ™ hMPVDFA Reagent</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>hMPV Ainfected cells</td><td rowspan=1 colspan=1>hMPV Binfected cells</td><td rowspan=1 colspan=1>Uninfectedcells</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Purified mucin protein,bovine submaxillarygland</td><td rowspan=1 colspan=1>0.1 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Saliva (human)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>undiluted</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=3 colspan=1>Nasal sprays</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>0.125 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>0.05 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Sodium chloride withpreservatives</td><td rowspan=1 colspan=1>0.65 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=2 colspan=1>Nasalcorticosteroids</td><td rowspan=1 colspan=1>Beclomethasone</td><td rowspan=1 colspan=1>0.042 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Triamcinolone</td><td rowspan=1 colspan=1>0.01 mg/ml</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>HomeopathicNasal gel</td><td rowspan=1 colspan=1>Luffa opperculata,sulfur, Galphimiaglauca</td><td rowspan=1 colspan=1>UndilutedOTC product</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Homeopathiccold reliefmedicine</td><td rowspan=1 colspan=1>Anas barbariaehepatic et cordisextractum</td><td rowspan=1 colspan=1>200CK; OTCproductdissolved in2 ml dH2Or</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Homeopathicthroat lozenges</td><td rowspan=1 colspan=1>Zinc gluconatetrihydrate</td><td rowspan=1 colspan=1>20.8 mg/ml</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=4 colspan=1>Throat lozenges,oral anestheticand analgesic</td><td rowspan=1 colspan=1>Menthol</td><td rowspan=1 colspan=1>20.8 mg/ml</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Benzocaine,glycerine</td><td rowspan=1 colspan=1>5.0 %, 33 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Ascorbic acid, zinccitrate</td><td rowspan=1 colspan=1>10 mg/ml,1.5 mg/ml</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Phenol</td><td rowspan=1 colspan=1>1.4 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Children&#x27;sexpectorant</td><td rowspan=1 colspan=1>Guaifenesin</td><td rowspan=1 colspan=1>20 mg/ml</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasalointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>2 %</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>positive</td><td rowspan=1 colspan=1>negative</td></tr></table>

g. Assay cut-off:

The Light Diagnostics™ Human Metapneumovirus DFA Assay is a qualitative test. The results are either positive or negative. Positive staining for hMPV is indicated by the presence of two (2) or more intact cells exhibiting specific fluorescence (bright apple green). A presumptive negative result is indicated by the absence of fluorescence in a minimum sampling of 20 epithelial cells. All presumptive negative results should be reported as “No virus observed” and should be confirmed using an FDA cleared hMPV molecular assay. A sample containing fewer than 20 epithelial cells is considered inadequate, and the test is thus invalid. A new specimen should be obtained and tested. These cut-offs (minimum 2 positive cells for positive; minimum 20 epithelial cells for adequate sampling) represent consensus of general virological techniques common in the laboratories used for clinical studies.

Interpretation of Results:

<table><tr><td rowspan="2"></td><td colspan="3">hMPV Results</td></tr><tr><td>Positive Infected Cells</td><td>Presumptive Negative* No infected Cells Detected</td><td>Invalid</td></tr><tr><td>Observation under FITC filter of fluorescent microscope</td><td>Bright apple-green fluorescence of nucleus and/or cytoplasm of infected cells Presence of 2 or more cells exhibiting</td><td>Absence of cell-associated green fluorescence in a minimum sampling of 20 epithelial cells</td><td>Fewer than 20 epithelial cells present in the sample or</td></tr></table>

\* Presumptive negative results should be followed up using an FDA cleared hMPV molecular assay.

# 2. Comparison studies:

a. Method comparison with predicate device:

At the time of the clinical studies, no FDA cleared predicate device was available to serve as comparator method to establish test performance. In order to establish performance characteristics, Millipore Corporation determined a composite algorithm to compare the Light Diagnostics™ Human Metapneumovirus DFA Assay results with.

The composite algorithm consisted of viral culture results and a validated endpoint RT-PCR method followed by bidirectional sequencing for confirmation and identification of human metapneumovirus. Within the composite algorithm, any sample that tested positive by viral culture, or had bidirectional sequencing data meeting predefined quality acceptance criteria that matched hMPV sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with

acceptable E-values1 , was considered a “true” hMPV positive. “True” hMPV negatives were any sample that tested negative by both viral culture and the RT-PCR assay.

The RT-PCR assay was standardized at St. Joseph Healthcare, Hamilton, Canada following internal validation and characterization through reactivity and analytical sensitivity studies. RNA was isolated using the EasyMag (bioMerieux) instrument as per manufacturer’s instructions. Gel electrophoresis was used to purify positive RT-PCR amplified specimens, using the QIAquick purification kit (Qiagen), prior to performing bidirectional sequencing with the ABI 3700 at McMaster University Sequencing Core Facility.

Method Comparison Algorithm:

<table><tr><td rowspan=1 colspan=1>RT-PCR assay on alldirect specimens. Bi-directional sequencingassay on all RT-PCRpositive direct samples</td><td rowspan=1 colspan=1>Cell culture on alldirect specimens</td><td rowspan=1 colspan=1>ClinicalDiagnostic Truth</td><td rowspan=1 colspan=1>Number of Specimensmeeting this criteria</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>346</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>1 (not included insensitivity andspecificity analysis)</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>0</td></tr></table>

Validation of the RT-PCR/sequencing method:

1. Analytical Sensitivity $( L o D )$ : The Reference RT-PCR Assay for hMPV was tested with each of two strains of hMPV: A (A2 subtype), and B (B1 subtype). Control strains of virus with known starting concentration diluted 10-fold from $1 0 ^ { - 1 }$ to $1 0 ^ { - 8 }$ were tested using the same protocol and conditions used in the clinical study. Replicates of five were extracted and tested at each dilution for each virus strain. The limit of detection was defined at the concentration at which $59 5 \%$ of replicates were positive. For the A2 strain, all replicates were positive at the $1 0 ^ { - 3 }$ dilution. Four of five were positive at $1 0 ^ { \dot { - } 4 }$ dilution $( 8 0 \% )$ , and none were positive at $1 0 ^ { - 5 }$ dilution. The limit was then defined as the $1 0 ^ { - 3 }$ dilution of the stock of $1 . 5 8 \mathrm { ~ x ~ } 1 0 ^ { 4 }$ $\mathrm { T C I D 5 0 / m L }$ . For the B1 strain, all replicates were positive at the $1 0 ^ { - 2 }$ and $1 0 ^ { - 3 }$ dilution. None were positive at $1 0 ^ { - 4 }$ dilution. The limit was then defined as the $1 0 ^ { - 3 }$ dilution of the stock of $1 . 5 8 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ $\mathrm { T C I D 5 0 / m L }$ . Following the initial five replicate testing, 30 additional samples at the limit of detection were extracted and tested. For both strains, 30/30 replicates at the limit of detection were positive. Results are summarized below:

<table><tr><td rowspan=1 colspan=1>Virus strain</td><td rowspan=1 colspan=1>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=1>15.8TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>1.58 TCID50/mL</td></tr></table>

2. Reactivity: The Reference RT-PCR Assay for hMPV was tested with each of four known subtypes of hMPV: A1, A2, B1, B2. Control strains of virus with known subtypes were tested using the same protocol and conditions used in the clinical study. Positive results were obtained with each subtype, and the resultant PCR products were bidirectionally sequenced. The results of BLAST alignments for each of the sequences from the PCR products show correct GenBank match, with high coverage score, high maximum identity $\%$ and significant E values $\mathord { \sim } \mathrm { e } ^ { - 4 0 }$ to e-50).

Based on these analytical validations, the hMPV RT-PCR assay is an acceptable method to be used as a part of the composite reference algorithm in determining “clinical diagnostic truth” for the Light Diagnostics™ Human Metapneumovirus DFA Assay Clinical Trial.

b. Matrix Description and Comparison: Not applicable

# 3. Clinical studies:

a. Clinical Sensitivity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

Sensitivity and specificity relative to a consensus comparator algorithm was determined as specified under Comparison Studies (see above).

The clinical study was done using prospective specimens. Specimens used in the study were freshly collected from leftover specimens. Accurate detection of human metapneumovirus (hMPV) is dependent upon sample collection, transport, and storage. The Manual of Clinical Microbiology, Murray, P. et al., eds. 8th ed. (2003) was referenced in this clinical study as guideline for proper specimen collection. The following protocol was used for specimen collection: Specimen swabs should be placed in viral transport medium. All specimens should be transported on wet ice or cold pack to the testing laboratory immediately after collection. Specimens should not be frozen before testing. They may be maintained at $2 { - } 8 ^ { \circ } \mathrm { C }$ for up to 72 hours without loss of viability.

Studies were done at three clinical sites (two US, one Canada), all performed between February and July, 2007. A total of four hundred eleven (411) NPS specimens from patients from three sites were used for evaluation in this study. One specimen was obtained per patient and 2 measurements were taken from each qualified specimen. Measurements were taken using the new device on direct specimens and the second measurement was taken using the culture/RT-PCR/sequencing composite algorithm method.

# Site 1:

Two hundred fifteen (215) specimens (all nasopharyngeal swabs, NPS) from patients were collected from a hospital laboratory in the southeast of Canada (St. Joseph’s Healthcare Hamilton, Site 1). From the 215, 7 specimens were excluded due to insufficient number of cells available or lack of comparator data available, leaving a total of 208 NPS specimens tested from Site 1.

hMPV was identified by RT-PCR in 20 specimens of the 208 tested for an overall prevalence of $9 . 6 \%$ . Half of the patients from whom hMPV was identified were under the age of 5 years. In addition, 7 of 20 $( 3 5 \% )$ patients positive for hMPV were older than 60 years of age. The relative prevalence for Site 1 of hMPV is indicated in the following tables:

Summary of Relative Prevalence of hMPV in the Southeastern Canada   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Southeastern Canada</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>20/208</td><td rowspan=1 colspan=1>9.6%</td></tr></table>

Age range of specimens tested for hMPV (Site 1)   
Site 2:   

<table><tr><td rowspan=1 colspan=1>Patient AgeRange</td><td rowspan=1 colspan=2>&lt; 5 years</td><td rowspan=1 colspan=2>6-21 years</td><td rowspan=1 colspan=2>22-59 years</td><td rowspan=1 colspan=2>&gt;60 years</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=2>12.2%</td><td rowspan=1 colspan=2>0%</td><td rowspan=1 colspan=2>9.4%</td><td rowspan=1 colspan=2>8.2%</td></tr></table>

Two hundred and two (202), including $2 0 0 \mathrm { N P S }$ , and 2 broncheoalveolar lavage (BAL), specimens from patients were collected from a clinical diagnostic testing facility in the northeast United States (Yale University, School of Medicine, Site 2). From these 202, one NPS specimen was excluded for insufficient number of cells, and the 2 BALs for not being sufficient to establish claim for this sample type, leaving a total of 199 NPS samples tested at site 2.

hMPV was identified by RT-PCR in 43 specimens of 199 tested, for an overall prevalence of $2 1 . 6 \%$ . All specimens included were nasopharyngeal swabs. Forty-one (41) of the 43 $( 9 5 \% )$ patients positive for hMPV were under the age of 5 years. The relative prevalence for Site 1 of hMPV is indicated in the following tables:

Summary of Relative Prevalence of hMPV in the Northeastern United States   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Northeastern U.S.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>43/199</td><td rowspan=1 colspan=1>21.6%</td></tr></table>

Age range of specimens tested for hMPV (Site 2)   

<table><tr><td rowspan=1 colspan=1>Patient AgeRange</td><td rowspan=1 colspan=2>&lt; 5 years</td><td rowspan=1 colspan=2>6-21 years</td><td rowspan=1 colspan=2>22-59 years</td><td rowspan=1 colspan=2>&gt;60 years</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>TotalSpecimens</td><td rowspan=1 colspan=1>hMPVPositive</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=2>21.9%</td><td rowspan=1 colspan=2>14.3%</td><td rowspan=1 colspan=2>25.0%</td><td rowspan=1 colspan=2>0%</td></tr></table>

Site 3:

Two hundred (200) specimens, including 149 nasopharyngeal wash (NPW), 4 NPS, $3 5 \mathrm { B A L }$ , and 12 tracheal aspirates (TA) specimens were collected at a facility in the Midwest United States (University of Iowa, Site 3). From the 200, only the 4 NPS were retained for this study, since the other sample types did not have enough representation through the three clinical sites to establish appropriate performance.

The 4 samples from Site 3 were all hMPV negative. The small size limits any remarkable outcome on these specimens.

Independent performance of the Light Diagnostics™ Human Metapneumovirus DFA Assay was comparable on the clinical sites (Sites 1 and 2, Site 3 had uninformative sample size), performance is therefore described below for the three sites together:

<table><tr><td rowspan=2 colspan=2>Detecting hMPV</td><td rowspan=1 colspan=2>Composite Result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Light Diagnostics ™ hMPVDFA Assay on directspecimens</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>346</td><td rowspan=1 colspan=1>351</td></tr><tr><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>348</td><td rowspan=1 colspan=1>411</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=3>Sensitivity</td><td rowspan=1 colspan=1>92% (58/63)</td><td rowspan=1 colspan=1>83-97%</td></tr><tr><td rowspan=1 colspan=3>Specificity</td><td rowspan=1 colspan=1>99% (346/348)</td><td rowspan=1 colspan=1>98-100%</td></tr><tr><td rowspan=1 colspan=3>Positive Predictive Value</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>89-99%</td></tr><tr><td rowspan=1 colspan=3>Negative Predictive Value</td><td rowspan=1 colspan=1>99%</td><td rowspan=1 colspan=1>97-99%</td></tr></table>

# 4. Clinical cut-off:

Not international standard exist for hMPV, refer to cutoff section above for additional details on interpretation.

# 5. Expected values/Reference range:

Four hundred and seven (411) specimens were tested at 3 sites (Southeast portion of Canada, Northeastern United States, Midwest United States) between February, 2007 and July 2007. In Southeastern Canada, hMPV was identified by the Light Diagnostics™ Human Metapneumovirus DFA Assay in 18 specimens for a prevalence of $8 . 6 \%$ . Half of the patients from whom hMPV was identified were under the age of 5 years. In addition, 6 of 18 $( 3 3 \% )$ patients positive for hMPV were older than 60 years of age. In the Northeastern United States, hMPV was identified in 42 specimens (prevalence $2 1 . 1 \%$ ). At this site, 40 of the 42 $( 9 5 \% )$ patients positive for hMPV were under the age of 5 years. All samples from the Midwest US were hMPV negative.

Prevalence of hMPV:   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Southeastern Canada Northeastern U.S.</td><td rowspan=1 colspan=2>Northeastern U.S.</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>18/208</td><td rowspan=1 colspan=1>8.6%</td><td rowspan=1 colspan=1>42/199</td><td rowspan=1 colspan=1>21.1%</td></tr></table>

Age distribution-Number of specimens and hMPV positivity rate:   

<table><tr><td rowspan=1 colspan=1>Patient Age Range</td><td rowspan=1 colspan=2>&lt; 5 years</td><td rowspan=1 colspan=2>6-21 years</td><td rowspan=1 colspan=2>22-59 years</td><td rowspan=1 colspan=2>&gt;60 years</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>S-E Canada</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>N-E US</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>269</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Percent Positive</td><td rowspan=1 colspan=2>18.5%</td><td rowspan=1 colspan=2>6.3%</td><td rowspan=1 colspan=2>8.3%</td><td rowspan=1 colspan=2>6.9%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.