# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K100148

# B. Purpose for Submission:

New Device

# C. Measurand:

Specific influenza virus nucleic acid target sequences. Influenza types and subtypes detected: a well-conserved region of the matrix (M) gene from influenza A viruses and a unique region in the hemagglutinin gene from the 2009 H1N1 influenza virus.

# D. Type of Test:

Real-time reverse transcription-polymerase chain reaction (RT-PCR) assay for the qualitative detection and differentiation of influenza A and 2009 H1N1 influenza viral RNA in nasopharyngeal swabs (NPS), nasal swabs (NS), and nasopharyngeal aspirates (NPA) using nucleic acid isolation. The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a Qiagen QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit. Amplification and detection is performed on the 3M Integrated Cycler with Integrated Cycler Studio Software version 2.1 or higher.

# E. Applicant:

Focus Diagnostics, Inc.

# F. Proprietary and Established Names:

Proprietary: Simplexa™ Influenza A H1N1 (2009) Generic: Influenza A H1N1 2009 Real Time RT-PCR

# G. Regulatory Information:

1. Regulation section: 866.3332   
2. Classification: Class II   
3. Product code: OQW, NXD, OEP, OOI   
4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Focus Diagnostics Simplexa™ Influenza A H1N1 (2009) assay is intended for use on the 3M Integrated Cycler as part of the Microfluidic Molecular System for the in vitro qualitative detection and differentiation of influenza A and 2009 H1N1 influenza viral RNA in nasopharyngeal swabs (NPS), nasal swabs (NS), and nasopharyngeal aspirates (NPA) from human patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2009-2010 influenza season when 2009 H1N1 influenza was the predominant influenza A virus in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s):

For prescription use only

4. Special instrument requirements:

To be used with the 3M Integrated Cycler with Integrated Cycler Studio Software version 2.1 or higher and either a Roche MagNA Pure LC automated nucleic acid extraction system or the Qiagen QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit.

# I. Device Description:

The Simplexa™ Influenza A H1N1 (2009) assay is a nucleic acid amplification test that uses real-time reverse transcriptase polymerase chain reaction (RT-PCR) to detect and differentiate influenza A and 2009 H1N1 influenza from nasopharyngeal swab (NPS), nasal swab (NS) and nasopharyngeal aspirate (NPA) specimens.

Patient specimens are collected and placed in sterile viral transport media containing protein stabilizer, antibiotics to inhibit bacterial and fungal growth, and buffer solution. An extraction control (AR IC) is added to the specimen prior to nucleic acid extraction by either the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a Qiagen QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit.

Specific lots of the ancillary reagent of the MagNA Pure LC Total Nucleic Acid Isolation Kit and QIAamp® Viral RNA Mini Kit will be qualified on an ongoing basis as part of the kit release process. The list of acceptable ancillary reagents will be maintained on the Focus Diagnostics website (www.focusdx.com).

NOTE: The SimplexaTM Influenza A H1N1 (2009) assay product performance requires that only qualified manufacturer lots of the MagNA Pure LC Total Nucleic Acid Isolation Kit and QIAamp® Viral RNA Mini Kit be used with the device. Any lots not specifically qualified by Focus Diagnostics for use with the SimplexaTM Influenza A H1N1 (2009) assay are not validated for use with this assay, and may cause erroneous results.

A sample of the extracted nucleic acid is added to the Simplexa™ Influenza A H1N1 (2009) reagents that contain a primer and a bi-functional fluorescent probe-primer set specific to the matrix gene of influenza A viruses and the hemagglutinin gene of 2009 H1N1 influenza, and a second primer pair specific to the SPC sequence. PCR amplification is performed on the 3M integrated cycler. The instrument fluorescence output is analyzed and test results are determined using Integrated Cycler Studio Software.

Simplexa™ Influenza A H1N1 (2009) kit contains Primer Mix (PM), RNA Master Mix (RMM), RT Mix (RT), Armored RNA Internal Control (AR IC), No Template Control (NTC) , and H1N1 positive control (PC), Simplexa™ Influenza A H1N1 (2009) Barcode Card, and Package Insert.

# Interpretation of Sample Results:

Reporting results is a three step process.

1. Determine if the run is valid by examining the H1N1 Positive Control, No Template Control, and Armored RNA Internal Control.

Criteria for a Valid Control (Simplified)\*   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>H1N1 Ct</td><td rowspan=1 colspan=1>FLUA Ct</td><td rowspan=1 colspan=1>AR IC Ct</td></tr><tr><td rowspan=1 colspan=1>No TemplateControl</td><td rowspan=1 colspan=1>0(If ≤ 40 then patient resultscannot be reported)</td><td rowspan=1 colspan=1>0(If ≤ 40 then patientresults cannot bereported)</td><td rowspan=1 colspan=1>≤ 40, ≠ 0</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>≤ 40, ≠ 0</td><td rowspan=1 colspan=1>≤ 40, ≠ 0</td><td rowspan=1 colspan=1>Not Applicable(N/A)</td></tr></table>

\* See notes below for full description.

a. If the No Template Control is:

i. Positive $\mathrm { C t }$ value $\leq 4 0 , \neq 0$ for either the H1N1 or FLUA), then this indicates possible contamination of prepared samples. The control is invalid and all patient specimens must be re-extracted and re-assayed.

ii. Negative for H1N1 and FLUA detector $\mathrm { ( C t = 0 ) }$ ), then this control is valid and acceptable.

iii. If the AR IC is not detected in the No Template Control, the assay run is invalid and all patient specimens must be re-extracted and reassayed.

iv. If the AR IC is detected for the No Template Control, the assay run is considered valid and acceptable.

# b. Positive Control

i. If the Positive Control result is a $\mathrm { C t } = 0$ for H1N1 and/or FLUA, the assay run is considered invalid and unacceptable. All patient specimens must be re-assayed.

ii. If the Ct values for H1N1 and FLUA are, $\leq 4 0 , \neq 0$ the assay run is considered valid and acceptable.

# 2. Examination of Patient Specimen Results

Examination of clinical specimen results should be performed after the Positive and No Template Controls have been examined and determined to be valid and acceptable. H1N1, FLUA and AR IC results must be examined for each patient specimen.

Criteria for a Valid Patient Specimen (Simplified)\*   

<table><tr><td rowspan=1 colspan=1>Patient Specimen H1N1 Ct and FLUA Ct</td><td rowspan=1 colspan=1>Amplification Plot</td><td rowspan=1 colspan=1>AR IC Ct</td></tr><tr><td rowspan=1 colspan=1>Either detector or both detectors ≤ 40, ≠ 0</td><td rowspan=1 colspan=1>Shows exponential increase</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Both detectors at 0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>≤ 40, ≠ 0</td></tr></table>

\* See notes below for full description

a. Amplification plots should be examined for every result with a “Data Quality” message. From the Data tab select the curve to review and click Refresh. The software will draw the selected curves and adjust the scale of the graph. A valid amplification curve shows a smooth, exponential increase. An invalid amplification curve may be a non-exponential or linear curve or a curve with data “spikes” where the curve may cross the threshold. If the curve is valid after examination, the Ct value reported may be used to determine if FLU A or H1N1 targets are detected.

b. If the amplification curve is valid for FLU A or H1N1, the AR IC is not required to be detected to report a positive result for FLU A or H1N1.

3. Interpretation of Results

a. A specimen that does not contain any influenza A virus will be negative $\mathrm { ( C t = 0 ) }$ for the FLU A and H1N1 detectors. A specimen positive for influenza A virus other than 2009 H1N1 influenza will most likely have a positive result $( \mathrm { C t } \leq 4 0 , \neq 0 )$ for the FLU A detector and will be negative $\mathrm { C t } =$ 0) for H1N1 detector. A specimen that is positive for 2009 H1N1 influenza will be positive for both the FLU A and the H1N1 detector.

b. If only the H1N1 detector is positive and not the FLU A detector, the result is indeterminate

c. If the FLU A Ct value of a patient sample is not detected and the AR IC Ct value falls within or below the acceptable range, the “Influenza A RNA” result is reported as “Not Detected”.

d. If the FLU A Ct value of a patient specimen is ${ \le } 4 0$ , $\neq 0$ and an amplification curve is observed for the well, the “Influenza A RNA” result is reported as “Detected”. If the Ct value for the well is $\leq 4 0$ but no amplification curve is observed (nonspecific fluorescence is observed in the well), the “Influenza A RNA” result is reported as “Not Detected.”

e. If the H1N1 Ct value of a patient sample is listed as $^ { \circ }$ and the AR IC Ct value falls within or below the acceptable range, the “2009 H1N1 Influenza RNA” result is reported as “Not Detected”.

f. If the H1N1 Ct value of a patient specimen is ${ \le } 4 0$ , $\neq 0$ and an amplification curve is observed and FLU A is also detected, the “2009 H1N1 Influenza RNA” result is reported as “Detected”. If the H1N1 Ct value for the well is ${ \le } 4 0$ , $\neq 0$ but no amplification curve is observed in the well (nonspecific fluorescence is observed in the well), the “2009 H1N1 Influenza RNA” result is reported as “Not Detected”.

g. If the FLU A and H1N1 Ct value of a patient specimen is 0 and the AR IC $\mathrm { C t }$ value is 0, the specimen must be re-assayed. If upon repeat testing, the same situation occurs, the patient result is reported as “Indeterminate due to possible inhibition” with the additional comment: “After repeat analysis, nonamplification of the internal control suggests the presence of PCR inhibitors in the patient sample. An additional sample should be submitted for testing if clinically warranted.”

h. If upon repeat testing the result is still indeterminate for H1N1 then the “2009 H1N1 Influenza RNA” result is reported as “Indeterminate.”

Interpretation of Results   

<table><tr><td rowspan=1 colspan=1>Example</td><td rowspan=1 colspan=1>FLUACtValue</td><td rowspan=1 colspan=1>H1N1CtValue</td><td rowspan=1 colspan=1>AR ICCtValue</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>≤ 40</td><td rowspan=1 colspan=1>≤ 40</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>Influenza A RNA: Detected2009 H1N1 Influenza RNA: Detected</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>≤ 40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Influenza A RNA: Detected2009 H1N1 Influenza RNA: Not Detected</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>≤ 40</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Indeterminate, re-assay</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>≤ 40</td><td rowspan=1 colspan=1>Influenza A RNA: Not Detected2009 H1N1 Influenza RNA: Not Detected</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Invalid, re-assay. If AR IC is still 0 onrepeat, test with a new sample ifclinically warranted</td></tr></table>

$\mathrm { C t } =$ cycle threshold. Detected is a $\mathrm { C t } \leq 4 0$ . Not Detected is a $\mathrm { C t } = 0$ . \* Detection of the SimplexaTM Armored RNA Internal Control (AR IC) is not required for a valid result.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Luminex xTAG Respiratory Viral Panel and CDC Human Influenza Virus Real-Time RT- PCR Detection and Characterization Panel

2. Predicate 510(k) number(s): K063765 and K080570

3. Comparison with predicate:

Similarities   
Differences   

<table><tr><td rowspan=1 colspan=10>Slallies</td></tr><tr><td rowspan=1 colspan=1>Device Characteristics</td><td rowspan=1 colspan=5>SimplexaTM InfluenzaA H1N1 (2009) (NewDevice)</td><td rowspan=1 colspan=3>xTAG Respiratory ViralPanel  FLUA (PredicateDevice #1)</td><td rowspan=1 colspan=1>CDC Human Influenza Virus Real-Time RT- PCR Detection andCharacterization Panel (PredicateDevice #2)</td></tr><tr><td rowspan=10 colspan=1>Intended Use</td><td rowspan=1 colspan=5>The Focus DiagnosticsSimplexaT Influenza AH1N1 (2009) assay isintended for use on the3M Integrated Cycler aspart of the MicrofluidicMolecular System for</td><td rowspan=6 colspan=3>The xTAG® RespiratoryViral Panel (RVP) is aqualitative nucleic acidmultiplex test intended forthe simultaneous detectionand identification ofmultiple respiratory virusnucleic acids innasopharyngeal swabs fromindividuals suspected ofrespiratory tract infections.The following virus typesand subtypes are identifiedusing RVP: Influenza A,Influenza A subtype H1,Influenza A subtype H3,Influenza B, Respiratory</td><td rowspan=10 colspan=1>The Human Influenza Virus Real-timeRT-PCR Detection andCharacterization Panel (rRT-PCR FluPanel) is intended for use in Real-timeRT-PCR assays on an ABI 7500 FastDx Real-time PCR instrument inconjunction with clinical andepidemiological information: forqualitative detection of influenza virustype A or B in symptomatic patientsfrom viral RNA in nasopharyngealand/or nasal swab specimens, fordetermination of the subtype ofseasonal human influenza A virus, asseasonal A/HI or A/H3, if present,from viral RNA in nasopharyngealand/or nasal swab specimens, forpresumptive identification of virus inpatients who may be infected withinfluenza A subtype A/H5 (Asianlineage) from viral RNA in humanrespiratory specimens and viralculture in conjunction with clinicaland epidemiological risk factors toprovide epidemiologic information forsurveillance for influenza viruses.</td></tr><tr><td rowspan=2 colspan=5>the in vitro qualitativedetection anddifferentiation ofinfluenza A and 2009H1N1 influenza viral</td></tr><tr><td rowspan=2 colspan=5>H1N1 influenza viralRNA in nasopharyngeal</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5>RNA in nasopharyngeal</td><td rowspan=1 colspan=2>and s</td></tr><tr><td rowspan=1 colspan=5>swabs (NPS), nasalswabs (NS), andnasopharyngeal</td><td rowspan=1 colspan=4>swabs (NPS), nasal</td></tr><tr><td rowspan=1 colspan=3>aspirates (NPA) from</td><td></td><td></td></tr><tr><td rowspan=4 colspan=5>human patients withsigns and symptoms ofrespiratory infection inconjunction withclinical andepidemiological riskfactors.</td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=3>Syncytial Virus subtype A,Respiratory Syncytial Virussubtype B, Parainfluenza 1,Parainfluenza 2, andParainfluenza 3 virus,Human Metapneumovirus,Rhinovirus, andAdenovirus.</td></tr><tr><td rowspan=1 colspan=1>nz A subtyp</td></tr><tr><td rowspan=1 colspan=1>Identification of Inf A</td><td rowspan=1 colspan=5>Yes</td><td rowspan=1 colspan=3>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=5>Qualitative</td><td rowspan=1 colspan=3>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidExtraction</td><td rowspan=1 colspan=5>Yes</td><td rowspan=1 colspan=3>Yes</td><td rowspan=1 colspan=1>Yes</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Device Characteristics</td><td rowspan=1 colspan=1>Simplexa™M InfluenzaA H1N1 (2009) (NewDevice)</td><td rowspan=1 colspan=1>xTAG Respiratory ViralPanel  FLUA (PredicateDevice #1)</td><td rowspan=1 colspan=1>CDC Human Influenza Virus Real-Time RT- PCR Detection andCharacterization Panel (PredicateDevice #2)</td></tr><tr><td rowspan=1 colspan=1>Sample types</td><td rowspan=1 colspan=1>NPS, NS, NPA</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>NPS, NS</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Real-time PCR</td><td rowspan=1 colspan=1>PCR followed by bead-based hybridization</td><td rowspan=1 colspan=1>Real-Time PCR</td></tr><tr><td rowspan=1 colspan=1>Identification of 2009H1N1 Subtype</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>RequiredInstrumentation</td><td rowspan=1 colspan=1>Integrated cycler withIntegrated CyclerStudio software v. 2.0</td><td rowspan=1 colspan=1>Luminex Instrument(100/200)</td><td rowspan=1 colspan=1>Applied Biosystems 7500 Fast DxReal- Time PCR Instrument with SDSsoftware version 1.4</td></tr><tr><td rowspan=1 colspan=1>Multiplex Capability</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Reagents for Detection of Specific Novel Influenza A Viruses, March   
22, 2006   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm078583.htm   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Testing for Detection and Differentiation of Influenza A Virus   
Subtypes Using Multiplex Assays October 9, 2009   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm180307.htm   
Draft Guidance for Industry and FDA Staff - Establishing Performance   
Characteristics of In Vitro Diagnostic Devices for Detection or Detection and   
Differentiation of Influenza Viruses, February 15, 2008   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm079171.htm   
Guidance for Industry and FDA Staff, In Vitro Diagnostic Devices to Detect   
Influenza A Viruses: Labeling and Regulatory Path, May 1, 2007   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm078538.htm   
Guidance for Industry and FDA Staff, Format for Traditional and Abbreviated   
510(k), August 12, 2005   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm084365.htm   
Guidance for the Content of Premarket Submissions for Software Contained in   
Medical Devices, May 11, 2005   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm089543.htm

# L. Test Principle:

The Simplexa™ Influenza A H1N1 (2009) test is a nucleic acid amplification assay that uses real-time reverse transcriptase polymerase chain reaction (RT-PCR) amplification to enable simultaneous and distinct detection of viral nucleic acids from influenza A and 2009 H1N1 influenza from nasopharyngeal swabs (NPS), nasal swabs (NS) and nasopharyngeal aspirates (NPA) from human patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors.

The assay combines real-time PCR amplification with fluorescent signal detection technology. A bi-functional fluorescent probe-primer is used together with a reverse primer to amplify a specific target (for each analyte and internal control). A fluorescent signal is generated after the separation of the fluorophore from the quencher as a result of the binding of a probe element to the extended RNA fragment synthesized during amplification.

The 3M Integrated Cycler is a rapid real-time Polymerase Chain Reaction thermocycler used for the identification of nucleic acid from prepared biological samples. The instrument utilizes disk media to contain and to process samples. The instrument uses real time flourometric detection to identify targets within the sample wells. The instrument is controlled by an external computer running the Integrated Cycler Studio Software. Together, the instrument, software and test kit are referred to as the “Microfluidic Molecular System.”

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

# a. Precision/Reproducibility:

The following samples were used to assess inter-laboratory, inter-assay, and intra-assay reproducibility: high negative, low positive and medium positive samples for 2009 H1N1-influenza, influenza A (H1N1), and influenza A (H3N2). Samples were prepared using both negative swab and negative aspirate matrices.

Precision/Reproducibility Study Samples   

<table><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Details</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-1</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>2009 H1N1 high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-2</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>2009 H1N1 low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-3</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>2009 H1N1 medium positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-4</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H1N1) high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-5</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H1N1) low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-6</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H1N1) medium positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-7</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H3N2) high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-8</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H3N2) low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-9</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Influenza A (H3N2) medium positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-10</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>2009 H1N1 high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-11</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>2009 H1N1 low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-12</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>2009 H1N1 medium positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-13</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H1N1) high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-14</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H1N1) low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-15</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H1N1) medium positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-16</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H3N2) high negative</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-17</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H3N2) low positive</td></tr><tr><td rowspan=1 colspan=1>REPRO- FLU-18</td><td rowspan=1 colspan=1>Nasopharyngeal Aspirate</td><td rowspan=1 colspan=1>Influenza A (H3N2) medium positive</td></tr></table>

# Inter-lot Precision:

Inter-lot precision was assessed by testing nine (9) samples and a positive control across three different lots (REPRO-FLU-1 – REPRO-FLU-9). These samples were generated using negative swab matrix and were run in triplicate. One operator at one study site made one extraction of each sample and tested across lots.

The variability attributable to lot-to-lot imprecision was found by fitting the appropriate general linear mixed model, with “lot” as a random variable, and partitioning out the variance. Total variability was split into inter-lot variability, and intra-lot variability (the unexplained variability, or error term of the model). The inter-lot variability component is summarized in the tables below. Imprecision estimates for all samples in the panel the $\% \mathrm { C V }$ was $\leq 0 . 9$ .

<table><tr><td rowspan=1 colspan=5>Simplexa Reproducibility - FLUA1</td></tr><tr><td rowspan=1 colspan=5>Inter-lot</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>InterLot SD</td><td rowspan=1 colspan=1>Inter Lot% CV</td></tr><tr><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-7</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.3</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Simplexa Reproducibility - H1N11</td></tr><tr><td rowspan=1 colspan=5>Inter-lot</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>InterLot SD</td><td rowspan=1 colspan=1>Inter Lot% CV</td></tr><tr><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-4</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-5</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-6</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-7</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-8</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>REPRO-FLU-9</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

1) For the purposes of calculating averages and variance components, samples that offered a negative result $\scriptstyle ( \mathrm { C t = 0 } )$ were assigned to a value of 40.0, as a value of 40.0 is more representative of negative samples which have Ct values at the upper limit of the range.

# Inter-laboratory, Inter-assay and Intra-assay Reproducibility:

Each sample (REPRO-FLU-1 – REPRO-FLU-18) was tested in triplicate at three separate sites: one run per operator, per day, two operators per site, at three sites for 5 days. Eighteen samples, the No Template Control, and Positive Control were tested in this fashion. A total of 1800 samples were tested in the reproducibility studies. Estimates of variability were determined with a random effects model that includes run and site as predictors, with run nested within site. This allowed for partitioning of the total variance to provide best estimates of inter-assay, intra-assay, and inter-lab variability.

Three sites assessed the device's inter-laboratory reproducibility and inter/intra-assay reproducibility. Site 1 and 3 performed the extraction using the MagNA Pure LC Total Nucleic Acid Isolation Kit; Site 2 performed the extraction step using the Qiagen QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit.

Four sample results were excluded from the analysis due to pipetting errors reported by the laboratory. Thirty six wells, $2 \%$ , had an invalid result due to a failed internal control (not detected) and were not used in the estimation of Ct variability. There was no association between the failed internal control and specimen type or viral strain, however 23 of the 36 invalid results were associated with one user. Eleven sample results $( 0 . 6 \% )$ were indeterminate. The combined results for all sites: FLUA Inter-Assay $\%$ CV range (0.0 to 4.8), FLU A Intra-Assay $\%$ CV range (0.0 to 6.6), H1N1 Inter-Assay $\%$ CV range (0.0 to 1.8) and H1N1 Intra-Assay $\%$ CV range (0.0 to 4.7).

Reproducibility – FLU A   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=2>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>No TemplateControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/301</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=2>89/90(98.9%)</td><td rowspan=1 colspan=1>94% -999.8%</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>29/292</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=2>89/89100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Swabs</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3></td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Fluhigh negative</td><td rowspan=1 colspan=1>25/293</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>20/30</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>19/30</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>64/89(71.9%)</td><td rowspan=1 colspan=2>61.8% -80.2%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flulow positive</td><td rowspan=1 colspan=1>26/283</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>28/293</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>83/86(96.5%)</td><td rowspan=1 colspan=2>90.2% -98.8%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flumedium positive</td><td rowspan=1 colspan=1>27/273</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>87/87(100%)</td><td rowspan=1 colspan=2>95.8% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(1N1)high negative</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>80/9088.9%</td><td rowspan=1 colspan=2>80.7% -93.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(11)low positive</td><td rowspan=1 colspan=1>25/293</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>81/89(91.0%)</td><td rowspan=1 colspan=2>83.3% -95.4%</td></tr><tr><td rowspan=1 colspan=1>Influenza AH1N1)medium positive</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>89/9098.9%)</td><td rowspan=1 colspan=2>94% -99.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(3N2)high negative</td><td rowspan=1 colspan=1>25/28³3</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>82/88(93.%)</td><td rowspan=1 colspan=2>85.9% -96.8%</td></tr><tr><td rowspan=1 colspan=1>Inffluenza A(3N2)low positive</td><td rowspan=1 colspan=1>18/243</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>77/84(91.7%)</td><td rowspan=1 colspan=2>83.8% -95.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(3N2)medium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>27/273</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>87/8700%)</td><td rowspan=1 colspan=2>95.8% -100%</td></tr><tr><td rowspan=1 colspan=13>Aspirate</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Fluhigh negative</td><td rowspan=1 colspan=1>25/25³3</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/294</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>83/83(100%)</td><td rowspan=1 colspan=2>95.6% -100%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flulow positive</td><td rowspan=1 colspan=1>24/30</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>2.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>28/28³3</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>81/8892.0%)</td><td rowspan=1 colspan=2>84.5% -96.%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flumedium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=2>94% -99.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(11)high negative</td><td rowspan=1 colspan=1>25/28³3</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>81/88(92.0%)</td><td rowspan=1 colspan=2>84.5% -96.1%</td></tr><tr><td rowspan=1 colspan=1>Inffuenza A(11)low positive</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>84/89(94.4%)</td><td rowspan=1 colspan=2>87.5% -97.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H11)medium positive</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>27/30</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>28/28³</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>84/87(96.6%)</td><td rowspan=1 colspan=2>90.3% -98.8%</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedesults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Influenza A(3N2)high negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>26/274</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>28/293</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>84/86(97.7%)</td><td rowspan=1 colspan=1>91.9% -99.4%</td></tr><tr><td rowspan=1 colspan=1>Infuenza A3N2)low positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza AH3N)medium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr></table>

1) The 1st replicate of the NTC for Site 2, Day 4, Run 1, was detected in the Flu A channel with a Ct value of 39.5. This well appeared to have a valid amplification curve. The other two replicates were negative, and the data from this run are included in the analysis. 2) The 2nd replicate of the Positive Control for Site 1, Day 4, Run 1 was invalid. The other two replicates of this run were positive, and the data from this run are included in the analysis. 3) Well(s) had an invalid result, and is (are) not used in the estimation of Ct variability. 4) Sample results have been excluded from the analysis due to pipetting errors reported by the laboratory sites.

Reproducibility – H1N1   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=2>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AVT..</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AVT.</td><td rowspan=1 colspan=1>o</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>AVT.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>No TemplateControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/301</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=2>90/90(10%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>29/292</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=2>89/89(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=12>Swabs</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Fluhigh negative</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>25/30</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>82/892.1%)</td><td rowspan=1 colspan=2>84.6% -96.1%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flulow positive</td><td rowspan=1 colspan=1>25/28³3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>6.1</td><td rowspan=1 colspan=1>28/293</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>82/86(95.3%)</td><td rowspan=1 colspan=2>88.6% -98.%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flumedium positive</td><td rowspan=1 colspan=1>27/27³3</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>87/87100%)</td><td rowspan=1 colspan=2>95.8% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H1N1)high negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=2>94% -99.8%</td></tr><tr><td rowspan=1 colspan=1>n flunza A(11)low positive</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>89/89(100%)</td><td rowspan=1 colspan=2>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>In fluenza A(H11)medium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=2>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(N2)high negative</td><td rowspan=1 colspan=1>27/283</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>86/88(97.7%)</td><td rowspan=1 colspan=2>92.1% -99.4%</td></tr><tr><td rowspan=1 colspan=1>Influenza AH3N2)low possitive</td><td rowspan=1 colspan=1>23/243</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>81/84(96.4%)</td><td rowspan=1 colspan=2>90% -98.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza AH3N2)medium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>27/273</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>87/87(100%)</td><td rowspan=1 colspan=2>95.8% -100%</td></tr><tr><td rowspan=1 colspan=13>Aspirate</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Fluhigh negative</td><td rowspan=1 colspan=1>25/25³3</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>83/83(100%)</td><td rowspan=1 colspan=2>95.6% -100%</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flulow positive</td><td rowspan=1 colspan=1>24/30</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>28/283</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>81/88(92.0%)</td><td rowspan=1 colspan=1>84.5% -96.1%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Flumedium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>94% -99.8%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H1N1)high negative</td><td rowspan=1 colspan=1>28/283</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>88/88100%)</td><td rowspan=1 colspan=1>95.8% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(11)low positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>89/89100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H1N1)medium positive</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>28/283</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>87/87(100%)</td><td rowspan=1 colspan=1>95.8% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H3N2)high negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>27/274</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>29/293</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>86/86(100%)</td><td rowspan=1 colspan=1>95.7% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H3N2)low positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(H3N2)medium positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>90/90100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr></table>

1) The 1st replicate of the NTC for Site 2, Day 4, Run 1, was detected in the Flu A channel with a Ct value of 39.5. This well appeared to have a valid amplification curve. The other two replicates were negative, and the data from this run are included in the analysis. 2) The 2nd replicate of the Positive Control for Site 1, Day 4, Run 1 was invalid. The other two replicates of this run were positive, and the data from this run are included in the analysis. 3) Well(s) had an invalid result, and is (are) not used in the estimation of Ct variability. 4) Sample results have been excluded from the analysis due to pipetting errors reported by the laboratory sites.

b Linearity/reportable range:

Not applicable

c. Traceability, stability, expected values (controls calibrators or methods)

# Controls:

The following controls are provided in the Simplexa™ Influenza A H1N1 (2009) kit:

Positive Control (PC): Inactivated 2009 H1N1 influenza virus. The PC in conjunction with the AR IC is used to verify reagent and system performance. The positive control is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc.). A positive control should be included in each run.

Internal Control (IC): The internal control is an RNA sequence encapsidated in protein (Armored RNA Internal Control (AR IC)). The AR IC is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification. The AR IC is meant to monitor for PCR inhibition.

No Template Control (NTC): The NTC includes nuclease-free water in the PCR reactions instead of RNA. The NTC reaction should not exhibit fluorescence growth curves that cross the threshold line. If any of the NTC reactions exhibit a growth curve that crosses the cycle threshold, sample contamination may have occurred. A no template control should be included in each run.

Quality control ranges have been established as indicated in the table below. If the controls are not within these parameters, patient results should be considered invalid and the assay repeated. Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations and standard good laboratory practice.

Expected Control Ranges   

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Simplexa™ H1N1PositiveControlFLUA Ct Value</td><td rowspan=1 colspan=1>Simplexa™ H1N1PositiveControlH1N1 Ct Value</td><td rowspan=1 colspan=1>Simplexa TArmored RNAInternalControl(AR IC)</td></tr><tr><td rowspan=1 colspan=1>No Template Control</td><td rowspan=1 colspan=1>Ct = 0</td><td rowspan=1 colspan=1>Ct = 0</td><td rowspan=1 colspan=1>Ct &lt; 40</td></tr><tr><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Ct &lt; 40</td><td rowspan=1 colspan=1>Ct &lt; 40</td><td rowspan=1 colspan=1>Not applicable*</td></tr></table>

\* Detection of the Simplexa™ Armored RNA Internal Control (AR IC) is not required for a valid resul

# d. Detection limit:

Analytical sensitivity was estimated for six strains of influenza A: A/California/7/2009 NYMC x-179-A, A/Swine NY/02/2009 H1N1, A/Solomon Island/03/06 H1, A/Brisbane/59/07 H1, A/Brisbane/10/07 H3 and, A/Wisconsin/67/05 H3. The Limit of Detection (LoD) was determined by limiting dilution studies using six viral stocks of the Influenza A virus. The samples were grown, re-titered, and diluted with bulk negative matrix (swab and aspirate). Four dilutions around the theoretical LoD were extracted three times using Roche MagNA Pure LC System. A single extraction of Positive Control (PC) and No Template Control (NTC) were included in each extraction cartridge. Each extracted sample and control was assayed in a single well. The lowest concentration at which all three replicates are positive is treated as tentative LoD.

Confirmation of LoD was determined over multiple runs. Each strain was spiked into negative swab and negative aspirate matrices at the concentration of tentative LoD and extracted using both the Roche MagNA Pure LC System and the QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit. A single extraction of Positive Control (PC) and No Template Control (NTC) was included in each run.

The LoD was estimated as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 19 out of 20 replicates were determined to be positive). The data are presented below stratified by FLUA and H1N1 markers, swab and aspirate matrices, and either the QIAgen or Roche extraction methods.

Simplexa™ Influenza A H1N1 (2009) Limit of Detection Screening and Confirmation – FLUA   

<table><tr><td rowspan=2 colspan=1>Influenza A Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>InitialScreeningMagNAPureExtraction</td><td rowspan=1 colspan=2>Confirmation ofLoD MagNA PureExtraction</td><td rowspan=1 colspan=2>Confirmation ofLoD QIAgenExtraction</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009 NYMC x-179-A</td><td rowspan=1 colspan=1>5.3x10²2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.7x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x10</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x10-1</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Swine NY/02/2009 H1N1</td><td rowspan=1 colspan=1>1x10^3$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^{2}$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^2</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Solomon Island/03/06 H1</td><td rowspan=1 colspan=1>1x10^</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$1x10^{2}$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5x10$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07 H1</td><td rowspan=1 colspan=1>1x10^3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$1x10^{2}$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5x10°</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>5/5*</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07 H3</td><td rowspan=1 colspan=1>1x10^{</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>$1x10^{2}$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/19</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^{2</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/05 H3</td><td rowspan=1 colspan=1>1x10^3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^{2}$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10-2</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* One of the initial triplicates was invalid, screening dilution was repeated

Simplexa™ Influenza A H1N1 (2009) Limit of Detection Screening and Confirmation – H1N1   

<table><tr><td rowspan=2 colspan=1>Influenza A Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>InitialScreeningMagNAPureExtraction</td><td rowspan=1 colspan=2>Confirmation ofLoD MagNA PureExtraction</td><td rowspan=1 colspan=2>Confirmation ofLoD QIAgenExtraction</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009 NYMC x-179-A</td><td rowspan=1 colspan=1>5.3x10^²2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.7x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x10°</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5.3x10-1</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Swine NY/02/2009 H1N1</td><td rowspan=1 colspan=1>1x10^3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^{2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1x10^2</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The tables below outlines the sponsor confirmed LoD for each strain.

Simplexa™ Influenza A H1N1 (2009) Limit of Detection – FLUA   

<table><tr><td rowspan=2 colspan=1>Influenza A Strain</td><td rowspan=1 colspan=2>LoD MagNA PureExtraction(TCID5o/mL)</td><td rowspan=1 colspan=2>LoD QIAgenExtraction(TCID5o/mL)</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009 NYMC x-179-A</td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>2.7x101</td></tr><tr><td rowspan=1 colspan=1>A/Swine NY/02/2009 H1N1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Island/03/06 H1</td><td rowspan=1 colspan=1>5x10$</td><td rowspan=1 colspan=1>5x10°</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>1x10°$</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07 H1</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>1x10$</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07 H3</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>5x10-1</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/05 H3</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td></tr></table>

Simplexa™ Influenza A H1N1 (2009) Limit of Detection – H1N1   
e. Analytical specificity:   

<table><tr><td rowspan=2 colspan=1>2009 Influenza A Strain</td><td rowspan=1 colspan=2>LoD MagNA PureExtraction(TCID50/mL)</td><td rowspan=1 colspan=2>LoD QIAgenExtraction(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td><td rowspan=1 colspan=1>Swab</td><td rowspan=1 colspan=1>Aspirate</td></tr><tr><td rowspan=1 colspan=1>A/California/7/2009 NYMC x-179-A</td><td rowspan=1 colspan=1>1.3x101</td><td rowspan=1 colspan=1>2.7x101</td><td rowspan=1 colspan=1>2.7x101</td><td rowspan=1 colspan=1>2.7x101</td></tr><tr><td rowspan=1 colspan=1>A/Swine NY/02/2009 H1N1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1x10-1</td></tr></table>

# Cross reactivity:

A panel of thirty four (34) potential cross-reactants were individually spiked at high concentrations into a swab matrix. The concentration of the cross reactants were determined by growing and titering the organism listed. The unspiked matrix was also tested to serve as a baseline. Samples extracted with the Roche MagNA Pure LC System and tested in triplicate to screen for cross reactivity. If either the FLUA channel or 2009 H1N1 channel was detected in any of the three replicates, an additional 5 replicates were tested for confirmation. One extraction was used to make the original three replicates and the confirmatory five replicates.

One of the original triplicates for adenovirus was detected in the FLUA channel. The baseline matrix that this sample was spiked into also gave a positive signal in the same run, indicating possible contamination. Five additional replicates from the same extraction were tested, offering 2 detected, one indeterminate, and 2 not detected results. In order to evaluate the possibility of contamination of the sample during extraction versus true reactivity of adenovirus 1, the cross reactant was spiked into fresh matrix and a new extraction performed. Both adenovirus 1 and the baseline matrix were not detected in all replicates.

No cross reactivity was detected for either FLUA or 2009 H1N1 channel, after confirmatory testing. In the instances where the $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ channel was detected among the screening triplicate, the FLUA channel was never detected and hence the interpretation of the sample was indeterminate. In all these instances the confirmatory repeat testing gave “not detected” results in the $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ channel. Detailed analytical specificity results are presented in the following table:

Cross Reactivity   

<table><tr><td rowspan=1 colspan=1>Cross-Reactant</td><td rowspan=1 colspan=1>TestingConcentration</td><td rowspan=1 colspan=1>Units</td><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>H1N1</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1</td><td rowspan=1 colspan=1>1.02x106$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>-1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7A</td><td rowspan=1 colspan=1>4.57 x10fr}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>5.80 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>1.00 x10 }$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1.00 x10$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>2.87 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>1.04 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>1.00 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=1>6.04 x10}$</td><td rowspan=1 colspan=1>copies/mL²</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli, O157H7</td><td rowspan=1 colspan=1>2.34 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>1.04 x106$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>—</td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Florida/04/2006)</td><td rowspan=1 colspan=1>1.26 x10^fr}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B (B/Malaysia/2506/04)</td><td rowspan=1 colspan=1>1.26 x10}$</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>—</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum, 17-5</td><td rowspan=1 colspan=1>1.75 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella longbeachae</td><td rowspan=1 colspan=1>7.10 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>1.26 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=1>Metapneumovirus</td><td rowspan=1 colspan=1>1.04 x105$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis, Ne 11</td><td rowspan=1 colspan=1>6.83 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mumps</td><td rowspan=1 colspan=1>1.51 x105</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>2.20 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae, Strain M129</td><td rowspan=1 colspan=1>1.13 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>1.99 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>1.63 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>1.32 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>1.18 x106</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>1.32 x105}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.05 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV-B</td><td rowspan=1 colspan=1>1.51 x10}$</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 16</td><td rowspan=1 colspan=1>1.00 x105}$</td><td rowspan=1 colspan=1>TCID5o/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylcoccus aureus, COL</td><td rowspan=1 colspan=1>1.68 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>3.80 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>5.54 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>1.55 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>1.14 x106</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td></tr></table>

1) One of the original triplicates for adenovirus was detected in the Influenza A channel. The baseline matrix that this sample was spiked into also gave a positive signal in the same run, indicating possible contamination. Five additional replicates from the same extraction were tested, offering 2 detected, one indeterminate, and 2 not detected results. In order to evaluate the possibility of contamination of the sample during extraction versus true reactivity of adenovirus 1, the cross reactant was spiked into fresh matrix and a new extraction performed. Both adenovirus 1 and the baseline matrix were not detected in all replicates.   
2) The EBV virus is grown in a transformed cell line (marmoset leukocytes). Transformed cells are not an appropriate cell line for quantitation using $T C | \mathsf { D } _ { 5 0 } / \mathsf { m L }$ , instead, copies/mL is calculated using a quantitative PCR method.

In addition to laboratory testing, bioinformatics resources and computer simulations were used to predict cross reactivity of additional influenza A strains with the Simplexa™ Influenza A H1N1 (2009). Comparison of the sequence of the bi-functional fluorescent probe-primer and the reverse primer for H1N1 shows that bi-functional fluorescent probe-primer and reverse primers designed to detect the 2009 H1N1 influenza strain have a significant number of mismatches when compared to the sequences of all other listed subtypes. The $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza strain is unique, and primers directed to the $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza sequence would not detect the other subtypes listed in the table below.

Simulated Analytical Cross Reactivity (Sequence Matches) with Additional Influenza A Strains   

<table><tr><td>Influenza A Strain</td><td>GenBank Number</td><td>Simulated Reactivity</td></tr><tr><td>A/California/VRDL72/2009 (H1N1)</td><td>GenBank: CY055479.1</td><td>Positive for H1N1 (2009)</td></tr><tr><td>A/New York/417/2002(H1N2)</td><td>GenBank: CY003769.1</td><td>Negative for H1N1 (2009)</td></tr><tr><td>A/swine/Italy/30073/2006 (H1N2)</td><td>GenBank: FJ770267.1</td><td>Negative for H1N1 (2009)</td></tr><tr><td>A/chicken/New A/chicken/New York (H7N2)</td><td>GenBank: CY035946.1</td><td>Negative for H1N1 (2009)</td></tr><tr><td>A/mallard/Geumgang/1/2007 (H7N7)</td><td>GenBank: FJ767719.1</td><td>Negative for H1N1 (2009)</td></tr></table>

# Reactivity:

In addition to the six influenza A strains tested for LoD, various dilutions (in negative swab matrix) of eight additional influenza A strains were tested for reactivity with the Simplexa™ Influenza A H1N1 (2009) assay. Four dilutions around the theoretical LoD were extracted three times using the Roche MagNA Pure LC instrumentation. A single extraction of Positive Control (PC) and No Template Control (NTC) were included in each extraction cartridge. Each extracted sample and control was assayed in a single well. The lowest concentration at which all three replicates are positive is treated as tentative LoD. The sequences of the two tissue culture adapted swine strains were confirmed to match published sequences. Results of screening with the additional strains are presented in the table below.

Analytical Reactivity with Additional Influenza A Strains   

<table><tr><td rowspan=1 colspan=1>Influenza A Strain</td><td rowspan=1 colspan=1>Lowest ConcentrationDetected(TCID50/mL)</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34 H1N1</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99 H1N11</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06 H1N11</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>A/WS/33 H1N1</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68 H3N2</td><td rowspan=1 colspan=1>1x10$</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>Tissue Culture Adapted InfluenzaA/Swine/lowa/15/30</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>Tissue Culture Adapted InfluenzaA/1976/31</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H5N1 (AsianLineage) Positive Control</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>Positive for FLUA</td></tr></table>

1) One of three replicates of A/New Caledonia/20/99 H1N1 and A/Taiwan/42/06 H1N1 and had a positive H1N1 result (CT 39.1) at the lowest concentration tested. There were no positive $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ results at the higher concentration.

Additional evidence of reactivity with other influenza A strains was generated by comparing of the sequence of the bi-functional fluorescent probe-primer and the reverse primer for FLUA and sequences matrix gene of the influenza A strains identified below. Sequence alignments demonstrate that the bifunctional fluorescent probe-primer and reverse primers have a near perfect match to all subtypes, and the forward primer has a maximum of 3 mismatches, which would theoretically enable the assay to detect all subtypes listed.

Simulated Analytical Reactivity (Sequence Matches) with Additional Influenza A Strains   

<table><tr><td>Influenza A Strain</td><td>GenBank Number</td><td>Simulated Reactivity</td></tr><tr><td>A/California/VRDL72/2009 (H1N1)</td><td>GenBank: CY055480.1</td><td>Positive for FLUA</td></tr><tr><td>A/swine/ltaly/306907/2003 (H1N1)</td><td>GenBank: FJ975097.1</td><td>Positive for FLUA</td></tr><tr><td>A/mallard/Korea/GH171/2007 (H7N7)</td><td>GenBank: FJ959087.1</td><td>Positive for FLUA</td></tr><tr><td>A/swine/Sweden/1021/2009 (H1N2)</td><td>GenBank: GQ495135.1</td><td>Positive for FLUA</td></tr><tr><td>A/Baden-Wuerrtemberg/20/03 (H1N2)</td><td>GenBank: EU249175.1</td><td>Positive for FLUA</td></tr><tr><td>A/Thailand/CU-B1697/2009 (H3N2)</td><td>GenBank: GU271985.1</td><td>Positive for FLUA</td></tr><tr><td>A/ruddy turnstone/New Jersey/563/2006 (H7N2)</td><td>GenBank: GQ257382</td><td>Positive for FLUA</td></tr><tr><td>A/swine/Hong Kong/NS857/2001 (H1N2)</td><td>GenBank: GQ229350.1</td><td>Positive for FLUA</td></tr></table>

# Interference

Analysis of Clinical Dataset for Effects of Potential Interfering Medications -

Note: No interference study was performed by spiking known concentrations of potentially interfering substances (e.g. cold medications, FluMist vaccine, blood, etc) into the sample matrix containing the assay analytes.

Data analysis of a sub-population of patients from the prospective clinical study prescribed various medications (i.e. anti-bacterials, anti-virals, steroids, common cold medications) showed similar sensitivity per target compared to the population not receiving medication. Although the concentration of the interferents in the total extracted nucleic acid preparation is unknown, the results represent doses that are typically prescribed to the intended use population. A complete list of medications recorded in patient charts extracted from the clinical dataset is presented in the Table below.

<table><tr><td rowspan=1 colspan=4>Reported Medications</td></tr><tr><td rowspan=1 colspan=1>Afrin (nasal spray)</td><td rowspan=1 colspan=1>Buitroban</td><td rowspan=1 colspan=1>Neoral</td><td rowspan=1 colspan=1>Penicillin</td></tr><tr><td rowspan=1 colspan=1>Albuterol</td><td rowspan=1 colspan=1>Carba - XP</td><td rowspan=1 colspan=1>Nexium</td><td rowspan=1 colspan=1>Pepto Bismol</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>Codral</td><td rowspan=1 colspan=1>Nuprin</td><td rowspan=1 colspan=1>Prevacid</td></tr><tr><td rowspan=1 colspan=1>Amoxil</td><td rowspan=1 colspan=1>Codril</td><td rowspan=1 colspan=1>NyQuil</td><td rowspan=1 colspan=1>Robitussin</td></tr><tr><td rowspan=1 colspan=1>Antihistime</td><td rowspan=1 colspan=1>Cortef</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>Rondec</td></tr><tr><td rowspan=1 colspan=1>Atuss Hs</td><td rowspan=1 colspan=1>Demazine</td><td rowspan=1 colspan=1>Omnicef</td><td rowspan=1 colspan=1>Sigmacort</td></tr><tr><td rowspan=1 colspan=1>Augmentin</td><td rowspan=1 colspan=1>Diostat</td><td rowspan=1 colspan=1>Oroxine</td><td rowspan=1 colspan=1>Stemetil</td></tr><tr><td rowspan=1 colspan=1>Azithromycin</td><td rowspan=1 colspan=1>Ipratropium</td><td rowspan=1 colspan=1>Panadol</td><td rowspan=1 colspan=1>Triaminic</td></tr><tr><td rowspan=1 colspan=1>Bactrim</td><td rowspan=1 colspan=1>Ketoprofen</td><td rowspan=1 colspan=1>Paracetamol</td><td rowspan=1 colspan=1>Vicodin</td></tr><tr><td rowspan=1 colspan=1>Bisacodyl</td><td rowspan=1 colspan=1>Keflex</td><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>Voltaren</td></tr></table>

f. Assay cut-off:

Fifty cycles of amplification were performed during assay development to allow for the appropriate determination of assay cut off. Analysis of the Limit of Detection study shows that the average Ct value for the samples at the Limit of Detection was ${ < } 3 7$ . The Limit of Detection is defined as the lowest concentration of sample where $\ge 9 5 \%$ of twenty replicates are detected. In the table below, the Sub LoD data indicate the next dilution in the series below the Limit of Detection where the replicates were detected $- 9 5 \%$ of the time. A review of the Method Comparison data show that the $5 9 0 \%$ of the positive specimens have a $\mathrm { C t }$ value ${ \le } 3 5$ . Specimens which were positive for 2009 H1N1 influenza, had Ct values in the range of 16.1 to $3 9 . 0 \mathrm { C t }$ . Specimens that were positive for influenza A, had $\mathrm { C t }$ values in the range of 18.4 to 39.2 with one sample at $\mathrm { C t }$ value of 42.5. Based on the available data, the assay cut off was set at $\mathrm { C t } = 4 0$ for both detectors.

Assay Cut-Off   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>2009 H1N1</td><td rowspan=1 colspan=3>Influenza A (H1N1)</td><td rowspan=1 colspan=3>Influenza A (H3N2)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td></tr><tr><td rowspan=1 colspan=1>FLUA</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>38.1</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Sub</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>Max</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>Ct</td></tr><tr><td rowspan=1 colspan=1>FLUA</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>37.1</td><td rowspan=1 colspan=1>43.8</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>39.2</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td></tr></table>

# g. Amplification Carry-Over Contamination:

The study was designed by alternately placing a high negative $( 1 . 0 \mathrm { ~ x ~ } 1 0 ^ { - 3 }$ $\mathrm { T C I D 5 0 / m L }$ ) and high positive $( 7 . 5 \mathrm { \scriptsize ~ \dot { \times } ~ } \mathrm { \dot { 1 } 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { \bar { m } L } } )$ samples on both the MagNA Pure cartridge for extraction and the Universal disc for amplification. A total of 180 high negative samples were tested across 5 runs. The sample size of 180 was chosen to provide $80 \%$ power of detecting a carry over/contamination effect of $5 \%$ or greater, with alpha $= 0 . 0 5$ . The carryover effect was evaluated by comparing the observed negative rate for the high negative sample with the expected rate under normal reproducibility conditions (i.e. $9 5 \%$ , the $\%$ negative rate outside of contamination). A one sample, one sided test for a proportion (binomial) was used to test for significance for both the FLUA and H1N1 channels.

The baseline negative sample was negative approximately $9 5 \%$ of the time for both FLUA and H1N1 channels at $6 . 7 \%$ positive for FLUA, $4 . 4 \%$ positive for H1N1, across 90 replicates). When tested with high positive samples, no significant carry-over contamination effect was seen in either channel.

Carry Over Contaimination   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Channel</td></tr><tr><td rowspan=1 colspan=1>FLUA</td><td rowspan=1 colspan=1>H1N11</td></tr><tr><td rowspan=1 colspan=1>Positive Count</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Negative Count</td><td rowspan=1 colspan=1>168</td><td rowspan=1 colspan=1>167</td></tr><tr><td rowspan=1 colspan=1>Total observations²</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>180</td></tr><tr><td rowspan=1 colspan=1>Percent positive</td><td rowspan=1 colspan=1>6.7%</td><td rowspan=1 colspan=1>7.2%</td></tr><tr><td rowspan=1 colspan=1>One-sided binomial test (p-value)</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.12</td></tr></table>

1) Only two (2) of the H1N1 results were actual H1N1 positives per the PI, as the FLUA channel was negative for 11 of detected H1N1 samples. These results were still included in the statistical testing for the purpose of detecting contamination of that channel.

2) One (1) of the 180 replicates of high negative came up as invalid due to a “Not Detected” AR IC result. With this replicate excluded, the one-sided binomial test still offered a non-significant result for both FLUA and H1N1 (p-value of 0.19 and 0.12, respectively).

# 2. Comparison studies:

# a. Method comparison with gold standard/reference method:

Performance of the assay was evaluated in comparison to a composite reference method for the Flu A target including the Luminex xTAG RVP Flu A target, a validated PCR assay using primer and probe sequences published by the CDC and a well characterized PCR followed by sequencing. The sequencing data was used to determine the 2009 H1N1 subtype.

b. Matrix Comparison:

Not Applicable

c. Performance in Fresh vs. Frozen Clinical Specimens:

Performance of the Simplexa™ Influenza A H1N1 (2009) assay was determined by comparing fresh (never frozen, unextracted and tested within 72 hours of collection) versus frozen (unextracted) clinical specimens. One hundred (100) samples, consisting of both swab and nasopharyngeal aspirates, were gathered for testing. Three aliquots of each sample were prepared, representing fresh, short-term frozen (at least 2 hours frozen at $- 2 0 ^ { \circ } \mathrm { C }$ ), and long-term frozen (at least 7 days frozen at $- 7 0 \mathrm { { ^ \circ C } } )$ states. All testing was performed at Focus, using one Simplexa ™ Influenza A H1N1 (2009) lot.

The plots of short term frozen Ct values and long term frozen Ct values vs. Fresh Ct values visually show the existence of a linear relationship. Additionally, for all comparisons, the value of coefficient of determination (R2) is greater than 0.97, meaning that at least $9 7 \%$ of the variation seen in the frozen (long term and short term) Ct values are explained by the variation seen in fresh Ct values.

The slopes along with the $9 5 \%$ confidence interval and percent bias for the above comparisons are summarized in the following table. The $9 5 \%$ confidence interval includes 1.00, which shows slope estimates are not statistically significant at $5 \%$ level of significance. This indicates that long term and short term frozen samples are stable in comparison with fresh samples.

Fresh vs. Frozen Specimen Comparison   

<table><tr><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>FrozenTerm</td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td><td rowspan=1 colspan=1>Bias</td></tr><tr><td rowspan=1 colspan=1>FLUA1</td><td rowspan=1 colspan=1>LongTerm</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.94 - 1.00</td><td rowspan=1 colspan=1>-3.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ShortTerm</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.94 - 1.00</td><td rowspan=1 colspan=1>-3.0%</td></tr><tr><td rowspan=1 colspan=1>H1N11</td><td rowspan=1 colspan=1>LongTerm</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>0.98 - 1.05</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Short Term</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>0.94 - 1.00</td><td rowspan=1 colspan=1>-3.0%</td></tr></table>

1) For the purposes of regression analysis, samples that offered a negative result $\scriptstyle ( \mathrm { C t = 0 } )$ were assigned to a value of 40.0, as a value of 40.0 is more representative of negative samples which have Ct values at the upper limit of the range.

![](images/2d572ac4918a822e76c1dbb6ad5bac4bfd4aa5df512a96002150f8cb08bd15ca.jpg)

![](images/9e9a0991e20d7970bb46396f4bfed7a069c0a51a1b78b44507b9a66a3d561c4f.jpg)

# d. Extraction Efficiency:

Assessment of the equivalency of the extraction methods was determined using a multifaceted approach. (1) One clinical site performed Clinical Agreement and Reproducibility using the QIAamp $^ \mathrm { \textregistered }$ Viral RNA mini kit and two sites used the MagNA Pure LC Total Nucleic Acid Isolation kit. (2) Limit of detection determinations were performed using both extraction methods. (3) In a direct comparison of the performance of Simplexa™ assay using two extraction methods, a panel of clinical specimens was extracted with MagNA Pure LC Total Nucleic Acid Isolation kit and QIAamp $^ \mathrm { \textregistered }$ Viral RNA mini kit, and each extracted sample was tested with the Simplexa™ Influenza A H1N1 (2009) assay.

A panel of 139 clinical specimens, including nasal swabs (NS) and nasopharyngeal swabs (NPS), with previous results blinded to the operator, was selected for this study. This panel contained 63 specimens previously reported as positive for $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza, 33 specimens previously reported as positive for influenza A, and 43 specimens previously reported as negative for influenza A. Each specimen was extracted with MagNA Pure LC Total Nucleic Acid Isolation kit and QIAamp $^ \mathrm { \textregistered }$ Viral RNA mini kit, and each extracted sample was tested in singlet with the Simplexa™ Influenza A H1N1 (2009) assay. The results of the specimens with both extraction methods were compared for positive and negative agreements.

Based on the three studies described above, there does not appear to be any significant difference in the performance of the Simplexa™ using either the MagNA Pure LC Total Nucleic Acid Isolation kit or the QIAamp $^ \mathrm { \textregistered }$ Viral RNA mini kit.

Extraction Method Concordance for 2009 H1N1 Influenza Simplexa™ Influenza A H1N1 (2009)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Roche MagNA Pure LC Extraction Method</td></tr><tr><td rowspan=4 colspan=1>QiagenQlAamp®Viral RNAMini Kitextractionmethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2009 H1N1Positive</td><td rowspan=1 colspan=1>2009 H1N1Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=2 colspan=1>% PositiveAgreement 100.0%(62/62)95% CI: 94.2- 100.0</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1Positive</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>63</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>76</td><td rowspan=2 colspan=1>% NegativeAgreement 98.7%(76/77)95% CI: 93.0- 99.8</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>139</td></tr></table>

\*All three targets were detected for one specimen with QIAamp $^ \mathrm { \textregistered }$ extraction, whereas only FLUA and IC targets, but not H1N1 target, were detected with MagNA Pure extraction. Upon re-extraction of frozen clinical specimen, SimplexaTM Influenza A H1N1 (2009) assay detected the specimen as positive for H1N1 with both extraction methods.

Extraction Method Concordance for Influenza A Simplexa™ Influenza A H1N1 (2009)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Roche MagNA Pure LC Extraction Method</td></tr><tr><td rowspan=4 colspan=1>QiagenQIAamp®Viral RNAMini Kitextractionmethod</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2009 H1N1Positive</td><td rowspan=1 colspan=1>2009 H1N1Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=2 colspan=1>% PositiveAgreement 100.0%(96/96)95% CI: 96.2- 100.0</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1Positive</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>2**</td><td rowspan=1 colspan=1>98</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>41</td><td rowspan=2 colspan=1>% NegativeAgreement 95.3%(41/41)95% CI: 84.5- 98.7</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>139</td></tr></table>

\*\*FLUA and AR IC targets were detected for two specimens with QIAamp $^ \mathrm { \textregistered }$ extraction, whereas only AR IC was detected for both specimens with MagNA Pure extraction. Upon re-extraction of frozen clinical specimens, Simplexa™ assay did not detect the FLUA target with either extraction method.

# 3. Clinical studies:

Specimens were prospectively collected from patients with signs and symptoms of influenza like illness from sites in Austin, TX (September 2009) and the New South Wales region of Australia (July – September 2009). An additional 214 nasal/nasopharyngeal swabs and two nasal washes were retrospectively collected from the Focus Sample Bank and were remnants of samples submitted to the Focus Reference Laboratory for influenza testing. Samples from all procurement sites were randomly distributed to each of the testing sites.

Samples were distributed to three testing sites. Site 1, located at University of Rochester Medical Center, Rochester, NY, generated data using the Roche MagNA Pure extraction method. Site 2, located at Nationwide Children’s Hospital in Columbus, Ohio, generated data using the Qiagen extraction method. Site 3, Focus Diagnostics, Inc., generated data using the Roche MagNA Pure extraction method.

Specimens were determined to be positive for 2009 H1N1 influenza by a composite reference method for the Flu A target including the Luminex xTAG RVP Flu A target, a validated PCR assay using primer and probe sequences published by the CDC and a well characterized PCR followed by sequencing. The sequencing data was used to determine the 2009 H1N1 subtype. Two results were generated for each specimen, an influenza A result and a 2009 H1N1 influenza subtyping result. Both results must be positive to determine that a specimen is 2009 H1N1 influenza positive.

A total of 410 prospectively collected specimens (299 nasal/nasopharyngeal swabs and 112 nasopharyngeal aspirates) were analyzed using the Simplexa™ Influenza A H1N1 (2009) assay. The data presented below are stratified by both result and specimen type. Fifteen (15) specimens (10 swabs and 5 aspirates) were excluded from the analysis because there was no consensus among the reference assays for the influenza A result. Twelve specimens (9 swabs and 3 aspirates) were excluded from the 2009 H1N1 Influenza Clinical Agreement Summary tables as sequencing data used to determine subtype was not available. These 12 specimens are included in the Influenza A Clinical Agreement Summary tables. Site poolability of performance data was determined based on the fact that similar SimplexaTM performance were obtained from the 3 clinical sites. For the clinical studies, the initial rate of indeterminate results was $0 . 1 6 \%$

2009 H1N1 Influenza Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Prospectively Collected Swabs   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>H1N1 Result - Simplexa™M Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>coueeed eesodceFs</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>H1N1Not Detected</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% PositiveAgreement100%(101/101)95% CI:96.3-100%</td></tr><tr><td rowspan=1 colspan=1>H1N1Not Detected</td><td rowspan=1 colspan=1>179</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>% NegativeAgreement95.5%(171/179)95% CI:91.4-97.7%</td></tr></table>

2009 H1N1 Influenza Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Prospectively Collected Aspirates   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>H1N1 Result - Simplexa™M Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>coueeed aosoceFsl</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>H1N1Not Detected</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% PositiveAgreement100%(24/24)95% CI:86.2-100%</td></tr><tr><td rowspan=1 colspan=1>H1N1Not Detected</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>% NegativeAgreement92.5%(74/80)95% CI:84.6-96.5%</td></tr></table>

Influenza A Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Prospectively Collected Swabs   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Influenza A Result - Simplexa™ Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>coJeged aosdcePs</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% PositiveAgreement100%(116/116)95% CI:96.8-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>% NegativeAgreement92.5%(160/173)95% CI:87.6-95.6%</td></tr></table>

Due to the low prevalence of other strains of influenza A during the testing period, all FLU A responses from prospectively collected swabs were combined to demonstrate the performance of the FLU A bi-functional fluorescent primer-probe. Of the 116 specimens determined to be positive for FLU A: 101 were $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza positive, zero (0) were H1N1, four (4) were H3N2, two (2) were not detected by the alternate PCR and could not be sequenced, and nine (9) were not subtyped.

Influenza A Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Prospectively Collected Aspirates   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=5>Influenza A Result - Simplexa™ Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>coueed eesodcceFll</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% PositiveAgreement100%(31/31)95% CI:89-100%</td></tr><tr><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>% NegativeAgreement96.1%(73/76)95% CI:89-98.6.1%</td></tr></table>

Due to the low prevalence of other strains of influenza A during the testing period; all FLU A responses from prospectively collected aspirates were combined to demonstrate the performance of the FLU A bi-functional fluorescent primer-probe. Of the 31 specimens determined to be positive for Flu A, 24 were $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza positive, one (1) was sequenced but the sub-type could not be determined, three (3) were not detected by the alternate PCR and could not be sequenced, three (3) did not have sufficient volume to sequence to determine sub-type.

An additional 214 retrospectively collected nasal/nasopharyngeal swabs and two nasal washes from the Focus Sample Bank were also tested at 3 sites. Three swab specimens were excluded from the analysis because there was no consensus among the reference assay results. One swab specimen was excluded from the 2009 H1N1 Influenza Clinical Agreement Summary tables as sequencing data used to determine subtype was not available. This specimen is included in the Influenza A Clinical Agreement Summary tables.

2009 H1N1 Influenza Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Retrospectively Collected Swabs   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>H1N1 Result - Simplexa™M Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>Rrr rctcosodwe</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>H1N1Not Detected</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>H1N1Detected</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% Positive Agreement100%(57/57)95% Cl:93.7-100%</td></tr><tr><td rowspan=1 colspan=1>H1N1NotDetected</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>% Negative Agreement90.8%(139/153)95% CI:85.2-94.5%</td></tr></table>

Two retrospectively collected washes were found to be positive for 2009 H1N1 influenza by the composite reference method and by the Simplexa assay.

Influenza A Clinical Agreement Summary Simplexa™ Influenza A H1N1 (2009) vs. Composite Reference Result – Retrospectively Collected Swabs   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=6>Influenza A Result - Simplexa™M Influenza A H1N1 (2009)</td></tr><tr><td rowspan=3 colspan=1>coJ aoodceRssct</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>Influenza ANot Detected</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>% Agreement</td></tr><tr><td rowspan=1 colspan=1>Influenza ADetected</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>% Positive Agreement99.2%(131/132)95% CI:95.8-99.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza ANotDetected</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>% Negative Agreement83.5%(66/79)95% CI:73.9-90.1%</td></tr></table>

Due to the low prevalence of other strains of influenza A during the testing period; all FLU A responses from retrospectively collected samples were combined to demonstrate the performance of the FLU A bi-functional fluorescent primer-probe. Of the 132 specimens determined to be positive for FLU A, 57 were $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza positive, two (2) were H1N1, 59 were H3N2, one (1) was sequenced but the sub-type could not be determined, one (1) was indeterminate by Simplexa, 11 were not detected by the alternate PCR and could not be sequenced, and one (1) did not have sufficient volume to sequence to determine sub-type.

Two retrospectively collected washes were found to be positive for influenza A by the composite reference method and by the Simplexa assay.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Prospective specimens used in this clinical study were obtained from the Texas region of the United States and the New South Wales region of Australia. The prevalence of influenza A in Texas (Region VI) ranged from $2 5 . 6 - 2 9 . 4 \%$ during the September 2009 collection period; $9 9 \%$ of those cases were $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza. In New South Wales, the prevalence during the July to September collection period ranged from $20 \%$ , with $8 3 . 9 2 \%$ of these cases representing 2009 H1N1 influenza.

# N. Instrument Name:

Integrated Cycler with Integrated Cycler Studio Software version 2.1 or higher (3M)

MagNA Pure LC Instrument (Roche)

# O. System Descriptions:

# 1. Modes of Operation:

The Microfluidic Molecular System includes the computer, related peripherals, handheld barcode scanner, Integrated Cycler, Integrated Cycler Studio software and operator manual. The Integrated Cycler is a rapid realtime Polymerase Chain Reaction (PCR) thermocycler used for identification of nucleic acid from prepared biological samples. The instrument utilizes disc media to contain and to process samples and real-time fluorometric detection to identify targets within the sample wells. The instrument’s operation parameters are controlled by the use of an external personal computer and associated software. The disc can process up to 96 independent samples.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User manually enters Patient ID/Sample ID.

4. Specimen Sampling and Handling: Not applicable

5. Calibration:

Not applicable

6. Quality Control:

The following controls are provided:

Positive Control (PC): H1N1 Positive Control Inactivated 2009 H1N1 influenza virus. The PC in conjunction with the AR IC is used to verify reagent and system performance. The positive control is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc.). A positive control should be included in each run.

Internal Control (IC): The internal control is an RNA sequence encapsidated in protein (Armored RNA Internal Control (AR IC)) The AR IC is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification. The AR IC is meant to monitor for PCR inhibition.

No Template Control (NTC): The NTC includes nuclease-free water in the PCR reactions instead of RNA. The NTC reaction should not exhibit fluorescence growth curves that cross the threshold line. If any of the NTC reactions exhibit a growth curve that crosses the cycle threshold, sample contamination may have occurred. A no template control should be included in each run.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision