# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K100336   
B. Purpose for Submission: Clearance of New Device   
C. Measurand: HSV-1 and HSV-2 nucleic acids target sequences   
D. Type of Test: Real Time Polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test for the detection and typing of herpes simplex virus 1 & 2 (HSV 1&2) DNA using vaginal swab specimens   
E. Applicant: Eragen Biosciences, Inc.   
F. Proprietary and Established Names: MultiCode®-RTx Herpes Simplex Virus & 2 Kit   
G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305

2. Classification: Class II

3. Product code: OQO, HSV NAAT assays, HSV PCR assays

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The MultiCode®-RTx Herpes Simplex Virus 1 & 2 Kit is a polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test for the detection and typing of herpes simplex virus (HSV 1 & 2) DNA in vaginal lesions. It is indicated for use in the detection and typing of HSV-1 or HSV-2 in vaginal lesion swab specimens from symptomatic female patients as an aid in the diagnosis of genital herpes infection.

Warning: The device is not FDA cleared for use with cerebral spinal fluid CSF; for any lesions other than vaginal. The assay is not intended to be used for male penile specimens, for prenatal screening, or for females under the age of 18 years.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Extraction: Roche MagNA Pure LC Total NA Kit and Magna Pure Instrument PCR: Roche LightCycler 1.2 Software: Eragen MultiCode®-RTx HSV 1&2 Kit Analysis Software (V2.0 RC1)

# I. Device Description:

The EraGen MultiCode®-RTx Herpes Simplex Virus 1 & 2 kit is a polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test for the detection and typing of herpes simplex virus 1 & 2 (HSV 1 & 2) DNA using vaginal swab specimens.

Patient vaginal swab specimens are collected in Copan Universal Transport Medium, or identical Copan manufactured media formulations (Becton Dickinson Universal Viral Transport Media, Copan branded Universal Transport Medium for LabCorp, and the Quest Viral Culture Media) and transported to the laboratory. An extractable sample processing control (SPC) target is added to the specimen prior to lysis. The SPC controls for specimen lysis, for recovery of extracted nucleic acid, for inhibitory substances and for PCR reagent and instrument integrity. The specimen is lysed and nucleic acid is extracted using the Roche MagNA Pure LC instrument using the Roche MagNA Pure LC Total Nucleic Acid Isolation Kit.

A sample of the extracted nucleic acid is added to the MultiCode®-RTx Herpes Simplex Virus 1 & 2 Kit reagents that contain a primer pair specific to HSV-1 and HSV-2 and a second primer pair specific to the SPC sequence. The two specific primer pairs are labeled with distinct fluorophore labels. PCR amplification is performed and assay fluorescence is monitored using the Roche LightCycler 1.2 real-time PCR instrument. Incorporation of the quencherlabeled nucleotide causes a decrease in assay fluorescence. Following amplification, the reaction is slowly heated and fluorescence is monitored. The strands of the amplification products will separate at a specific melting temperature $\left( \mathrm { T _ { m } } \right)$ that is determined by an increase in fluorescence as the strands are separated. The instrument fluorescence output is analyzed and test results are determined using the MultiCode®-RTx Herpes Simplex Virus 1 & 2 Kit Analysis Software. A printed results report is generated.

MultiCode®-RTx HSV 1&2 Kit contains HSV-1&2 Primer Mix, HSV-1&2

Reaction Buffer, Nuclease Free Water, DNA Sample Processing Control, HSV1 Positive Control, and HSV-2 Positive Control, Analysis Software, Certificate of Analysis, and Package Insert.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.)

2. Predicate Numbers (s): K971662

3. Comparison with predicate: An FDA cleared ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) was used as the predicate device. The performance of the MultiCode®-RTx HSV 1&2 assay was compared with the predicate which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system.

Note: To establish the clinical performance, it is important to compare qualitative HSV 1 & 2 DNA detecting devices to a gold standard/reference method such as cell culture using an enzyme linked virus inducible system

Similarities   

<table><tr><td colspan="1" rowspan="1">DeviceCharacteristic</td><td colspan="1" rowspan="1">EraGen BiosciencesMultiCode®-RTx HerpesSimplex Virus 1 &amp; 2 Kit(New Device)</td><td colspan="1" rowspan="1">Diagnostics Hybrid, Inc.ELVIS® HSV ID/Typing TestSystem (Predicate Device)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The MultiCode®-RTx HSV 1&amp;2Kit is a polymerase chain reaction(PCR)-based qualitative in vitrodiagnostic test for the detectionand typing of herpes simplex virus(HSV 1 &amp; 2) DNA in vaginallesions. It is indicated for use in thedetection and typing of HSV-1 orHSV-2 in vaginal lesion swabspecimens from symptomaticfemale patients as an aid in thediagnosis of genital herpesinfection.Warning: The device is not FDAcleared for the use with cerebralspinal fluid (CSF) or any lesionsother than vaginal. The assay is notintended to be used for male penilespecimens, for prenatal screening,or females under the age of 18years.</td><td colspan="1" rowspan="1">The ELVISRHSV ID Typing TestSystem is a qualitative testindicated for use in isolation andidentification of HSV from lesionsand body fluids suspected ofcontaining viable HSV-1 and/orHSV-2. Both serotypes have beenisolated in various parts of thebody, particularly when HSV-associated disease is indicated.Performance of this assay has notbeen established for use withantiviral therapy or prenatalmonitoring.</td></tr><tr><td colspan="1" rowspan="1">Identification andTyping of HSV-1and HSV-2</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">DeviceCharacteristic</td><td colspan="1" rowspan="1">EraGen BiosciencesMultiCode®-RTx HerpesSimplex Virus 1 &amp; 2 Kit(New Device)</td><td colspan="1" rowspan="1">Diagnostics Hybrid, Inc.HSV ID/Typing TestSystem (Predicate Device)</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr></table>

Differences   

<table><tr><td rowspan=1 colspan=1>DeviceCharacteristic</td><td rowspan=1 colspan=1>EraGen BiosciencesMultiCode®-RTx HerpesSimplex Virus 1 &amp; 2 Kit(New Device)</td><td rowspan=1 colspan=1>Diagnostics Hybrid, Inc.ELVIS®HSV ID/Typing TestSystem (Predicate Device)</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Real-Time PCR</td><td rowspan=1 colspan=1>Cell Culture using an enzymelinked virus inducible system.</td></tr><tr><td rowspan=1 colspan=1>Analysis SoftwareProvided</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Printed ResultsReport Provided</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Kit ReagentStorage Conditions</td><td rowspan=1 colspan=1>-15°C to -30°</td><td rowspan=1 colspan=1>2°C to 8°C and 22°C to 28°</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, May 11, 2005. http://www.fda.gov/cdrh/ode/guidance/337.pdf   
2. CLSI EP17-A, “Protocols for Determination of Limits of Detection”.

# L. Test Principle:

Following nucleic acid extraction from a patient swab specimen, a fluorophorelabeled PCR primer pair amplifies a segment of the glycoprotein B gene of HSV-1 and HSV-2. The sample processing control (SPC) will also be amplified by a distinct fluorophore-labeled PCR primer pair unless there are sample processing errors, inhibitory substances in the PCR reaction, reagent failure, or instrument malfunction.

The MultiCode®-RTx system is based on an expanded genetic alphabet technology, consisting of $2 ^ { \circ }$ -deoxy-5-methyl-isocytidine (iC) and $2 ^ { \circ }$ -deoxyisoguanosine (iG) nucleotide bases also known as isobases. The isobases pair specifically with each other and not with natural nucleotides. In addition isobases are efficiently incorporated during PCR. The isobase pair allows sitespecific incorporation of a dabcyl quencher directly adjacent to a fluorophorelabeled primer. During PCR amplification, a quencher-modified iGTP is incorporated by the polymerase opposite an iC and a fluorophore reporter attached to a PCR primer. If target is present and is amplified, assay fluorescence decreases with every cycle as amplification product accumulates. The decrease in assay fluorescence is monitored in real time using the Roche LightCycler 1.2 instrument. Following PCR, the amplification products are thermally denatured and assay fluorescence is monitored. The strands of the amplification products are separated and assay fluorescence increases, thus determining the melting temperature $\mathrm { ( T _ { m } ) }$ profile of the amplicon. The sequences between the PCR primer binding sites of the HSV-1 and HSV-2 amplicons have different base compositions that are distinguished by their different melting temperatures using the MultiCode®-RTx HSV 1&2 Analysis Software.

# Interpretation of Control Results:

A. Validation of Assay Run: The MultiCode®-RTx HSV 1&2 Kit Analysis Software automatically determines results for the controls based on the amplification cycle threshold (Ct) value and the melting temperature $\left( \mathrm { T } _ { \mathrm { m } } \right)$ value.

B. Valid Assay Run:

For a valid run, the following conditions must be met:

<table><tr><td rowspan=2 colspan=1>Control ID</td><td rowspan=2 colspan=1>AssayResult</td><td rowspan=1 colspan=3>F1</td><td rowspan=1 colspan=2>F2</td></tr><tr><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>HSV-1Tm</td><td rowspan=1 colspan=1>HSV-2Tm</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>SPC Tm</td></tr><tr><td rowspan=1 colspan=1>HSV-1 PositiveControl</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>HSV-2 PositiveControl</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=2 colspan=1>Negative Control</td><td rowspan=2 colspan=1>Pass</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr></table>

A $^ { \mathfrak { c } \mathfrak { c } } + ^ { \mathfrak { s } \mathfrak { s } }$ symbol in the Ct column indicates that the amplification curve crosses the amplification threshold. A $^ { \mathfrak { c } \mathfrak { c } } + ^ { \mathfrak { s } \mathfrak { s } }$ symbol in the $\mathrm { { T } _ { m } }$ column indicates that a melt peak is observed in the appropriate temperature range. A “-“ symbol in the Ct column indicates that the amplification curve does not cross the amplification threshold. A “-“ symbol in the Tm column indicates that no melt peak was observed in the appropriate temperature range. N/A indicates that the result is not dependent on the value.

• The clinical report displays only the Assay Result column. If the HSV-1 Positive Control fails, no data will be available for the HSV-2 Positive Control or the Negative Control because the criteria for Pass/Fail are based on the results of the HSV-1 Positive Control.

C. Invalid Assay Run: If the conditions for a Valid Assay Run are not met, the run must be repeated. Start from the original specimen(s), repeat the target extraction using a new Negative Control, and repeat the reaction setup using new Positive Controls.

# Interpretation of Specimen Results:

The MultiCode®-RTx HSV 1&2 Kit Analysis Software automatically determines results for the specimens based on the amplification cycle threshold (Ct) and the melting temperature $\mathrm { ( T _ { m } ) }$ value.

# Clinical Report Call Logic

<table><tr><td rowspan=2 colspan=2>Clinical Report</td><td rowspan=1 colspan=5>Input</td></tr><tr><td rowspan=1 colspan=3>F1</td><td rowspan=1 colspan=2>F2</td></tr><tr><td rowspan=1 colspan=2></td><td></td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>HSV-1Tm</td><td rowspan=1 colspan=1>HSV-2Tm</td><td rowspan=1 colspan=1>Ct</td></tr><tr><td rowspan=1 colspan=2>HSV-2 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>HSV-2 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>HSV-2 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>HSV-2 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>HSV-1 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>HSV-1 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>HSV-1 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>HSV-1 Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>HSV-1&amp;2Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>HSV-1&amp;2Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>HSV-1&amp;2Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=2>HSV-1&amp;2Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=2>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr></table>

A $^ { \mathfrak { c } \mathfrak { c } } + ^ { \mathfrak { s } \mathfrak { s } }$ symbol in the Ct column indicates that the amplification curve crosses the amplification threshold. A $^ { \mathfrak { c } \mathfrak { c } } + ^ { \mathfrak { s } \mathfrak { s } }$ symbol in the $\mathrm { T } _ { \mathrm { m } }$ column indicates that a melt peak is observed in the appropriate temperature range. A “-“ symbol in the $\mathrm { C t }$ column indicates that the amplification curve does not cross the amplification threshold. A “-“ symbol in the $\mathrm { T } _ { \mathrm { m } }$ column indicates that no melt peak was observed in the appropriate temperature range. $\mathrm { N } / \mathrm { A }$ indicates that the result is not dependent on the value.

A sample is Positive if both the $\mathrm { C t }$ and $\mathrm { T } _ { \mathrm { m } }$ are positive in the F1 channel irrespective of the $\mathrm { C t }$ and $\mathrm { T } _ { \mathrm { m } }$ in the F2 channel. A sample is Negative if the $\mathrm { C t }$ and $\mathrm { T } _ { \mathrm { m } }$ are negative in the F1 channel and the $\mathrm { C t }$ and $\mathrm { T } _ { \mathrm { m } }$ are positive in the F2 channel.

# Invalid/Fail Report Call Logic

<table><tr><td rowspan=3 colspan=1>Invalid/FailReport</td><td rowspan=1 colspan=5>Input</td></tr><tr><td rowspan=1 colspan=3>F1</td><td rowspan=1 colspan=2>F2</td></tr><tr><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>HSV-1Tm</td><td rowspan=1 colspan=1>HSV-2Tm</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>SPCTm</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>b</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr></table>

A $\smile ^ { c c } { + \arrow } ^ { \prime }$ symbol in the Ct column indicates that the amplification curve crosses the amplification threshold. A $^ { \mathfrak { c } \mathfrak { c } } + ^ { \mathfrak { s } \mathfrak { s } }$ symbol in the $\mathrm { T } _ { \mathrm { m } }$ column indicates that a melt peak is observed in the appropriate temperature range. A “-“ symbol in the Ct column indicates that the amplification curve does not cross the amplification threshold. A “-“ symbol in the $\mathrm { { T } _ { m } }$ column indicates that no melt peak was observed in the appropriate temperature range. N/A indicates that the result is not dependent on the value.

A sample is INVALID if the $\mathrm { T } _ { \mathrm { m } }$ is positive and the $\mathrm { C t }$ is negative in the F1 channel. This condition warrants repeat testing using the extracted nucleic acid. If the result is INVALID again, then the specimen needs to be reextracted and tested. A sample is FAIL if the $\mathrm { T } _ { \mathrm { m } }$ is negative in the F1 channel, and the $\mathrm { C t }$ or $\mathrm { T } _ { \mathrm { m } }$ is negative in the F2 channel. This condition warrants repeat testing following extraction of another aliquot of the specimen. If the result is again a FAIL, then the specimen needs to be re-collected and tested.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The Precision/Reproducibility of the MultiCode®-RTx HSV 1&2 Kit was evaluated at 3 U.S. clinical laboratories. A panel was prepared containing 6 simulated HSV-1 and HSV-2 samples that included High Negative, Low Positive (near the assay limit of detection) and Moderate Positive samples. The panel along with the external HSV-1 and HSV-2 positive and negative controls (Cytomegalovirus) was assayed in triplicate. The external Controls were included to monitor the study site DNA extraction and amplification processes during each assay run. Kit positive and negative controls were included with each assay run. Panels and controls were tested at each site by 2 operators, 1 time each per day for 5 days $( \mathrm { N } = 9 0 0 )$ ). Results of the Precision/Reproducibility studies for the MultiCode®-RTx HSV 1&2 assay are presented from three sites in the tables below.

Precision/Reproducibility Study Summary for the MultiCode®-RTx HSV 1&2 PCR Assay   

<table><tr><td rowspan=2 colspan=1>Targets</td><td rowspan=1 colspan=4>Site #1</td><td rowspan=1 colspan=4>Site #2</td><td rowspan=1 colspan=3>Site #3</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Totalagreementwithexpectedresults (%)</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Agreementwithexpectedresults-# correct /#tested</td><td rowspan=1 colspan=1>Avg Tm</td><td rowspan=1 colspan=1>%CV-Tm</td><td rowspan=1 colspan=1>Avg %Deffection</td><td rowspan=1 colspan=1>Agreementwithexpectedresults-# correct /#tested</td><td rowspan=1 colspan=1>Avg Tm</td><td rowspan=1 colspan=1>%CV-Tm</td><td rowspan=1 colspan=1>Avg %Deflection2</td><td rowspan=1 colspan=1>Agreementwithexpectedresults-# correct /#tested</td><td rowspan=1 colspan=1>Avg Tm</td><td rowspan=1 colspan=1>%CV-Tm</td><td rowspan=1 colspan=1>Avg %Deflection2</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Positive Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>84.3</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>85.0</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>84.7</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>78.3</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>84.7</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>72.8</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.4%-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Positive Control</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>87.2</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>70.9</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>70.3</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>68.8</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.4%-100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1/HSV-2 NegativeControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>77.9</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>78.0</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>88.4%-100%</td></tr><tr><td rowspan=1 colspan=1>PN 1750 HS V-1 PositiveExternal Control</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>79.2</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>68.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.7</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>67.3</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>93.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>PN 1751 HSV-2 PositiveExternal Control</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.4</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>73.0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>68.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>69.3</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>93.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>PN 1754 HS V-1/HS V-2Negative ExternalControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.6</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.7</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>HS V-1 High Negative1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.6</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.7</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.8</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>HS V-1 -Low Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.4</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>75.7</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>64.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>65.1</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>93.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 High Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>94.4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.6</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>92.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>84.8</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>94.3</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>HS V-2 High Negative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.6</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.7</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>77.7</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Low Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.3</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>71.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>63.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.6</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>67.5</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr><tr><td rowspan=1 colspan=1>HS V-2 High Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.4</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.5</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>87.7</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>91.9</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9% -100%</td></tr></table>

$\mathrm { T } _ { \mathrm { m } }$ value of Samp le Proces sing Control (SP C) was used, since these targets did not generate any detectable HSV-1 or HSV-2 signal. $\%$ deflectio $\%$ deflection is the average deflection for that panel member across al the 10 runs for the site.

b. Linearity/assay reportable range: N/A   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Assay Controls: Quality control procedures are intended to monitor reagent and assay performance. The kit includes the following controls:

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Use</td></tr><tr><td rowspan=1 colspan=1>HSV-1 Positive Control</td><td rowspan=1 colspan=1>Verify proper reagent performance and Roche LightCyclerinstrument setup</td></tr><tr><td rowspan=1 colspan=1>HSV-2 Positive Control</td><td rowspan=1 colspan=1>Verify proper reagent performance and Roche LightCyclerinstrument setup</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>Detect contamination and verify proper reagent performanceand Roche LightCycler instrument setup</td></tr><tr><td rowspan=1 colspan=1>DNA SampleProcessing Control</td><td rowspan=1 colspan=1>Verify proper specimen lysis, verify nucleic acid extraction andto verify proper reagent performance and Roche LightCyclerinstrument setup</td></tr></table>

Stability Studies: were performed in-house to determine the shelf life for the multiCode®-RTx Herpes Simplex Virus 1 & 2 Kit.

Accelerated Stability: The product is stable for a minimum of 15 months when stored at an accelerated temperature of $4 \mathrm { { } ^ { \circ } C }$ .   
Real-Time Stability: The product is stable for a minimum of 6 months when stored the recommended storage temperature of - $. 1 5 \mathrm { ^ \circ C }$ to ${ - 3 0 ^ { \circ } C }$ .   
Real time stability studies will continue to be performed to support the shelf life of this product.

# d. Detection limits:

A Limit of Detection (LOD) and Limit of Blank (LOB) study was performed in-house in accordance with CLSI EP-17-A (Protocols for Determination of Limits of Detection and Limits of Quantitation: Approved Guideline) to determine the analytical LOD and LOB performance obtained with the MultiCode®-RTx Herpes Simplex Virus 1 & 2 Kit.

A Limit of Detection (LoD) and Limit of Blank (LoB) study was performed in house to determine the analytical LoD and LoB performance using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of HSV-1 and HSV-2 serially diluted in Copan Universal Transport Media. Each viral strain was extracted using the MagNA Pure LC and tested in replicates of 60 per concentration of virus. The LOD was determined to be the lowest concentration of target that was detected in at least $9 5 \%$ of replicates. The LOB was determined to be the highest concentration of target that was detected in $\leq 5 \%$ of replicates.

The LOD for HSV-1 was determined to be $2 . 0 0 \mathrm { \times 1 0 ^ { 3 } T C I D _ { 5 0 } / m L }$ . At this concentration $100 \%$ of samples were detected with a $9 5 \%$ Confidence Interval of $9 3 . 9 4 - 1 0 0 \%$ . The LOD for HSV-2 was determined to be $6 . 4 0 \mathrm { x }$ $1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . At this concentration $9 8 . 3 \%$ of samples were detected with a $9 5 \%$ Confidence Interval of $9 1 . 0 6 - 9 9 . 9 6 \%$ .

The LOB for HSV-1 was determined to be $2 . 5 0 \mathrm { ~ x ~ } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . At this concentration $3 . 3 3 \%$ of samples were detected with a $9 5 \%$ Confidence Interval of $0 . 4 1 - 1 1 . 5 3 \%$ . The LOD for HSV-2 was determined to be 4.00 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . At this concentration $0 . 0 0 \%$ of samples were detected with a $9 5 \%$ Confidence Interval of $0 . 0 0 - 5 . 9 6 \%$ .

# LoD for HSV 1

LOD

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Pos/Total Calls</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>95% C.I.</td></tr><tr><td rowspan=1 colspan=1>4.00 x 103</td><td rowspan=1 colspan=1>59/59*</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>91.06 ± 99.96%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>59/59*</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>91.06 ± 99.96%</td></tr><tr><td rowspan=1 colspan=1>1.00 x 103</td><td rowspan=1 colspan=1>17/60</td><td rowspan=1 colspan=1>28.33%</td><td rowspan=1 colspan=1>17.45 ± 41.44%</td></tr><tr><td rowspan=1 colspan=1>,00 x 102</td><td rowspan=1 colspan=1>6/59*</td><td rowspan=1 colspan=1>10.17%</td><td rowspan=1 colspan=1>4.82 ± 22.57%</td></tr><tr><td rowspan=1 colspan=1>2.50 x 102</td><td rowspan=1 colspan=1>2/60</td><td rowspan=1 colspan=1>3.33%</td><td rowspan=1 colspan=1>0.41 ± 11.53%</td></tr><tr><td rowspan=1 colspan=1>1.25 x 102</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00 ± 5.96%</td></tr></table>

2\*One “Invalid” call due to baselines of amp curves by newest $\mathrm { V } 2 . 0 \mathrm { R C } 1$ software

LOB

# LoD for HSV 2

LOD

<table><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>Pos/Total Calls</td><td rowspan=1 colspan=1>Positivity</td><td rowspan=1 colspan=1>95% C.I.</td></tr><tr><td rowspan=1 colspan=1>1.28 x 102</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.00%</td><td rowspan=1 colspan=1>94.04 ± 100.00%</td></tr><tr><td rowspan=1 colspan=1>6.40 x 101</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.33%</td><td rowspan=1 colspan=1>91.06 ± 99.96%</td></tr><tr><td rowspan=1 colspan=1>3.20 x 101</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>93.33%</td><td rowspan=1 colspan=1>83.80 ± 98.15%</td></tr><tr><td rowspan=1 colspan=1>1.60 x 101</td><td rowspan=1 colspan=1>17/57*</td><td rowspan=1 colspan=1>29.82%</td><td rowspan=1 colspan=1>18.43 ± 43.40%</td></tr><tr><td rowspan=1 colspan=1>8.00</td><td rowspan=1 colspan=1>4/60</td><td rowspan=1 colspan=1>6.67%</td><td rowspan=1 colspan=1>1.85 ± 16.20%</td></tr><tr><td rowspan=1 colspan=1>4.00</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0.00%</td><td rowspan=1 colspan=1>0.00 ± 5.96%</td></tr></table>

\*Three ”Invalid” calls due to baselines of amp curves by newest $\mathrm { V } 2 . 0 \mathrm { R C } 1$ software

LOB

Note: The LOD value for HSV-1 was shown higher than HSV-2. This could be due to mismatch in the forward primer because a single set of forward and reverse HSV primers was used to amplify the target region from HSV-1 and HSV-2. The HSV Forward primer is a perfect match with the HSV-2 glycoprotein B sequence but contains a one base mismatch with the HSV-1 glycoprotein B sequence. The mismatch is in the center of the primer with 10 bases $3 ^ { \circ }$ to the mismatch. The predicted $\mathrm { T } _ { \mathrm { m } }$ for the forward primer to HSV-1 is $5 7 . 4 ^ { \circ } \mathrm { C }$ ; the predicted $\mathrm { T } _ { \mathrm { m } }$ for the forward primer to HSV-2 is

$5 9 . 8 ^ { \circ } \mathrm { C }$ . It is likely that in the initial rounds of PCR, HSV-2 is amplified more efficiently than HSV-1 which could result in a small increase in analytical sensitivity for HSV-2.

# e. Analytical specificity:

Analytical Specificity/Cross Reactivity of the MultiCode®-RTx HSV 1&2 Kit was evaluated at EraGen Biosciences. A panel was prepared containing 22 different organisms representing near-neighbors to the HSV-1 and HSV2 virus and organisms reasonably expected to be present in vaginal swab specimens.

The cross-reactivity panel was tested in a background of Copan Universal Transport Media at the concentration indicated in the table below. No HSV positive results were observed for any of the organisms tested and the DNA SPC was detected in all cases. The organism panel was also spiked into HSV-1 and HSV-2 near the device’s Limit of Detection (LoD) and tested. No interference was observed from any of the tested organisms, and all the results were positive for HSV-1 or HSV-2 as expected.

# Cross Reactivity Panel

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>2.8 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>2.5 x108 EBs/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>2.0 x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>4.5 x105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>1.8 x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>2.4 x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td><td rowspan=1 colspan=1>1.2 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyrogenes</td><td rowspan=1 colspan=1>1.4 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Trichomonas vaginalis</td><td rowspan=1 colspan=1>1.1 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>3.3 x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>1.4 x106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>1.0 x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Toxoplasma gondii</td><td rowspan=1 colspan=1>6.6 x105 Tachyzoites/mL</td></tr><tr><td rowspan=1 colspan=1>Treponema pallidum</td><td rowspan=1 colspan=1>1.0 x107 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus (AD169 strain)</td><td rowspan=1 colspan=1>4.2 x103 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus (Type 71)</td><td rowspan=1 colspan=1>1.4 x104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus (B95-8 strain)</td><td rowspan=1 colspan=1>9.3 x107copies/mL</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster virus</td><td rowspan=1 colspan=1>2.4 x107copies/mL</td></tr><tr><td rowspan=1 colspan=1>Human Herpes 6 virus (Z29 strain)</td><td rowspan=1 colspan=1>1.9 x106 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Herpes 7 virus (SB strain)</td><td rowspan=1 colspan=1>3.4 x106TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Papilloma virus</td><td rowspan=1 colspan=1>5 - 8 x105 copies/mL</td></tr><tr><td rowspan=1 colspan=1>Rubella virus</td><td rowspan=1 colspan=1>1.7 x104 TCID50/mL</td></tr></table>

# f. Interfering Substances:

An Interfering Substance study was performed at EraGen Biosciences to evaluate the effects of potential interfering substances on the MultiCode®- RTx HSV 1&2 Kit. A panel was prepared containing 6 substances and was tested at two different concentrations that could reasonably be expected to be present in vaginal swab specimens. The substance panel was tested near the device’s Limit of Detection (LoD) for HSV-1 and HSV-2. No interference was observed in the presence of exogenous and endogenous substances in an extractable sample.

# Interfering Substances Panel

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>Whole Blood (with EDTA)</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Whole Blood (with EDTA)</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Female Urine</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>Female Urine</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Protein (Albumin)</td><td rowspan=1 colspan=1>10 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Protein (Albumin)</td><td rowspan=1 colspan=1>1 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Protein (Casein)</td><td rowspan=1 colspan=1>10 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Protein (Casein)</td><td rowspan=1 colspan=1>1 mg/ml</td></tr><tr><td rowspan=1 colspan=1>K-Y Brand Jelly</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=1 colspan=1>K-Y Brand Jelly</td><td rowspan=1 colspan=1>0.5%</td></tr><tr><td rowspan=1 colspan=1>Acyclovir (Acycloguanosine)</td><td rowspan=1 colspan=1>2.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Acyclovir (Acycloguanosine)</td><td rowspan=1 colspan=1>0.25 mg/ml</td></tr></table>

# g. Carry-over/Cross-Contamination

Carry-over/Contamination studies were done only with HSV-1 target, since both HSV-1 and HSV-2 share a single set of primers. The MultiCode®-RTx HSV 1&2 Kit was evaluated internally using simulated samples at the LoB and High Positive HSV-1 (1700X LoD) samples. Ten sets of samples in a sequence pattern of High Positive-LoB-LoB were aliquoted into a 32-well sample plate and extracted using the Roche MagNA Pure LC instrument. The high positive sample was positive for HSV-1 in all cases and all LoB samples were negative for HSV-1 indicating that there was no carry-over/cross-contamination of HSV-1 in the LoB samples.

2. Comparison studies:

# a. Method comparison with predicate device:

The clinical performance evaluation was done against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system.

For additional details please see section 3 Clinical studies subsection c.

b. Matrix Comparison

N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A b. Clinical specificity: N/A

c. Other clinical supportive data (when a. and b. are not applicable):

An FDA cleared ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) was used as the Predicate device. The performance of the MultiCode®- RTx HSV 1&2 assay was compared with the predicate which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system.

# Clinical Performance

Performance characteristics of the MultiCode®-RTx HSV 1&2 assay were established during a prospective study performed at 3 U.S. clinical laboratories from 2008-2009. A total of 1041 vaginal swab specimens were tested by the MultiCode®-RTx HSV 1&2 Kit and by the reference method. The swab specimens were collected from lesions in Copan Universal Transport Medium from the patient population ranged from 18 to 85 years. One swab specimen was collected from each patient in Copan Universal Transport Medium and was tested for HSV-1 and HSV-2 using both the Diagnostics Hybrids, In. ELVIS culture and typing test and the EraGen MultiCode-RTx test.

A total of 1041 swab specimens were tested using the MultiCode®-RTx HSV 1&2 assay at three sites. Results from Prospective Study are shown in the tables below.

<table><tr><td rowspan=1 colspan=5>Herpes Simplex Virus Type 1 Comparison Results</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV 1&amp;2 Kit</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>16a</td><td rowspan=1 colspan=1>113</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>gb</td><td rowspan=1 colspan=1>920</td><td rowspan=1 colspan=1>928</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>936</td><td rowspan=1 colspan=1>1041</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=2>95% Confidence Interval</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>92.4%</td><td rowspan=1 colspan=2>85.7 - 96.1%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=2>97.2  98.9%</td></tr></table>

a Sequence analysis detected HSV-1 in 12 of the 16 samples identified as HSV-1 by MultiCode®-RTx. Sequence analysis did not detect HSV-1 in 4 of the samples.

b Sequence analysis detected HSV-1 in 1 of the 8 samples identified as HSV-1 negative by MultiCode®-RTx. Sequence analysis did not detect HSV-1 in 7 of the samples. Of these 7 samples: 4 of the samples were identified as HSV-2 by both MultiCode®-RTx and sequencing 2 of the samples were negative by MultiCode®-RTx and not detected by sequencing 1 sample was negative by MultiCode®-RTx and HSV-2 positive by sequencing.

<table><tr><td rowspan=1 colspan=5>Herpes Simplex Virus Type 2 Comparison Results</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Method</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>HSV 1 and 2 Kit</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>198</td><td rowspan=1 colspan=1>53a</td><td rowspan=1 colspan=1>251</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10b</td><td rowspan=1 colspan=1>780</td><td rowspan=1 colspan=1>790</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>833</td><td rowspan=1 colspan=1>1041</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=2>95% Confidence Interval</td></tr><tr><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=2>91.4 - 97.4%</td></tr><tr><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>93.6%</td><td rowspan=1 colspan=2>91.8 - 95.1%</td></tr></table>

a Sequence analysis detected HSV-2 in 43 of the 53 samples identified as HSV-2 by MultiCode®-RTx. Sequence analysis did not detect HSV-2 in 10 of the discordant samples.

Sequence analysis detected HSV-2 in 2 of the 10 samples identified as HSV-2 negative by MultiCode®-RTx. Sequence analysis did not detect HSV-2 in 8 of the discordant samples. These 8 samples were identified as HSV-1 by both MultiCode®-RTx and sequencing.

A total of 69 discordant specimens were reference method negative and MultiCode®-RTx HSV 1&2 Kit positive for HSV-1 or HSV-2. These samples were further evaluated by DNA sequencing. In addition 22 HSV-1 and 24 HSV-2 samples that were tested positive by both the EraGen MultiCode-RTx and the Reference Method were bidirectionally sequenced by SeqWright.

# Competitive Inhibition

Competitive Inhibition of the MultiCode®-RTx HSV 1&2 Kit was evaluated internally using simulated samples with varying concentrations of HSV-1 virus (1X LoD to 1000X LoD) and HSV-2 virus (1X LoD to 1000X LoD). Competitive inhibition was observed. The highest concentration of co-infecting target that can be present while maintaining $9 5 \%$ detection of the 3X LoD target for HSV-1 was 1X LoD for HSV-2. The highest concentration of co-infecting target that can be present while maintaining $9 5 \%$ detection of the 3X LoD target for HSV-2 was 1X LoD for HSV-1.

Competitive Inhibition was also evaluated internally using simulated samples with equal concentrations of HSV-1 virus and HSV-2 virus (5X LoD to 500X LoD). Competitive inhibition was not observed at any of the concentrations tested when the concentrations of HSV-1 and HSV-2 were equal.

4. Clinical cut-off: N/A

# 5. Expected values/Reference range:

The prevalence of HSV-1 and HSV-2 during the 2008 - 2009 multi-site clinical study $\scriptstyle ( \ n = 1 0 4 1$ specimens collected December 2008 - June 2009) was estimated using the MultiCode®-RTx test. Geographical distribution of study population included in the study consisted of 74 Midwest (12 States), 813 South (16 States), 153 West (13 States), and 1 Northeast (9 States). The combined prevalence was used to calculate the positive predictive values (PPV) and negative predictive values (NPV) of the MultiCode®-RTx test. The calculations are based on the sensitivity and specificity values obtained from the clinical studies: sensitivity of $9 2 . 4 \%$ and specificity of $9 8 . 3 \%$ for HSV-1; sensitivity of $9 5 . 2 \%$ and specificity of $9 3 . 6 \%$ for HSV-2. The MultiCode®-RTx HSV 1 & 2 assay results are summarized in the following tables:

MultiCode®-RTx HSV 1& 2 Assay Distribution of Prospective Population by Age Group; All testing Sites   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=3 colspan=1>MultiCode®RTx HSV-1-Positive</td><td rowspan=3 colspan=1>MultiCode®RTx HSV-2-Positive</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Totalnumber ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>Age Range</td></tr><tr><td rowspan=1 colspan=1>18 to 25 years</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>316</td></tr><tr><td rowspan=1 colspan=1>26 to 30 years</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=1>31 to 35yyears</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>109</td></tr><tr><td rowspan=1 colspan=1>36 to 40 years</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>128</td></tr><tr><td rowspan=1 colspan=1>41 to 45 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>85</td></tr><tr><td rowspan=1 colspan=1>46 to 50 years</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>88</td></tr></table>

Prevalence vs Hypothetical Predictive Values   

<table><tr><td rowspan=1 colspan=1>51 to 55 years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>51</td></tr><tr><td rowspan=1 colspan=1>56 to 60 years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>61 to 65 years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>66 to 70 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>71 to 75 years</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>76 to 80 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>81 to 85 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>86 to 90 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>113 (10.85%)</td><td rowspan=1 colspan=1>251 (24.11%)</td><td rowspan=1 colspan=1>1041</td></tr></table>

<table><tr><td rowspan=2 colspan=1>Prevalence</td><td rowspan=1 colspan=2>HSV-1</td><td rowspan=1 colspan=2>HSV-2</td></tr><tr><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td><td rowspan=1 colspan=1>PPV</td><td rowspan=1 colspan=1>NPV</td></tr><tr><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>97.5%</td><td rowspan=1 colspan=1>95.1%</td><td rowspan=1 colspan=1>90.8%</td><td rowspan=1 colspan=1>96.7%</td></tr><tr><td rowspan=1 colspan=1>30%</td><td rowspan=1 colspan=1>96.0%</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>86.5%</td><td rowspan=1 colspan=1>97.8%</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>78.9%</td><td rowspan=1 colspan=1>98.7%</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>85.8%</td><td rowspan=1 colspan=1>99.1%</td><td rowspan=1 colspan=1>62.3%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>74.0%</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>43.9%</td><td rowspan=1 colspan=1>99.7%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.