# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K100287   
B. Purpose for Submission: New device   
C. Measurand: Influenza A subtype A/H5 (Asian lineage) virus nucleic acids target sequences.   
D. Type of Test: A real-time reverse-transcriptase polymerase chain reaction (Real-time RT-PCR) test intended for the qualitative in vitro detection of RNA from the H5 subtype (Asian lineage) of the Influenza A virus in nasopharyngeal and throat swabs using nucleic acid isolation, amplification, and detection the Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument.   
E. Applicant: U.S. Army Medical Materiel Development Activity   
F. Proprietary and Established Names: JBAIDS Influenza A/H5 (Asian lineage) Detection Kit Common Name: JBAIDS Influenza A/H5 (Asian lineage) rRT-PCR Kit

G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>NXD</td><td>Class II</td><td>21 CFR 866.3332 Reagents for detection of specific novel influenza A viruses</td><td>Microbiology (83)</td></tr><tr><td>001</td><td>Class II</td><td>21 CFR 862.2570 Instrumentation for clinical multiplex test systems</td><td>Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit is intended for use in real-time reverse-transcriptase polymerase chain reaction (rRT-PCR) assays on the Joint Biological Agent Identification and Diagnostic System (JBAIDS) instruments for the in vitro qualitative detection of Influenza A/H5 (Asian lineage) viral RNA in patient nasopharyngeal swab (NPS) or throat swab (TS) specimens for the presumptive laboratory identification of Influenza A/H5 (Asian lineage) virus.

Testing with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit should be in conjunction with other laboratory testing and clinical observations for the following indications:

1. Providing epidemiological information for the surveillance of human infection with Influenza A/H5 (Asian lineage) virus.   
2. Identifying patients who may be infected with Influenza A/H5 (Asian lineage) virus based on clinical and epidemiological risk factors.

Testing with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit should not be performed unless the patient meets the most current U.S. Department of Health and Human Services (DHHS) clinical and epidemiologic criteria for testing suspected A/H5 specimens.

The definitive identification of influenza A/H5 (Asian lineage) either directly from patient specimens or from virus cultures requires additional laboratory testing, along with clinical and epidemiological assessment in consultation with national influenza surveillance experts.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

Use is limited to laboratories with appropriate biosafety equipment and containment procedures. It is intended for use by experienced laboratory personnel who have training in standardized molecular testing procedures and expertise in viral diagnosis, and have received training on the JBAIDS Instrument.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a biosafety laboratory (BSL) $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument.

# I. Device Description:

The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit is a real time RT-PCR assay based on the influenza A/H5 (Asian lineage) assays that are part of the “CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, or the IT 1-2- $3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instruments.

Each JBAIDS Influenza A/H5 (Asian lineage) Detection Kit contains sufficient reagents for testing four specimens. There are four vacuum-sealed control pouches of freeze-dried PCR reagents (4 reagent vials each), four vacuum-sealed sample testing pouches of freeze-dried PCR reagents (3 reagent vials each), and four pouches containing one tube each of reconstitution buffer and reagent grade water. Once resuspended, each reagent vial provides enough material for two reactions, both of which are included in the test run.

Each control pouch contains the following vials:

One Target 1 Positive Control $( + )$ vial. Once reconstituted, the Target 1 Positive Control vial contains all reagents necessary for Target 1 RT-PCR, in addition to 395 copies/vial of RNA derived from a synthetic plasmid containing the target 1 sequence (and not the target 2 sequence) from a consensus influenza $\mathrm { A } / \mathrm { H } 5$ (Asian lineage) strain. The Target 1 Positive Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens. Amplification of the Target 1 Positive Control gives assurance that kit reagents are functioning properly and that the assay setup has been performed correctly. A Positive Target 1 Control must be included with each JBAIDS run.

Ingredients: Each Positive Target 1 Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ Reverse Transcriptase from Moloney Murine Leukemia Virus (MMLV- RT); $< 0 . 0 0 1 \%$ target-specific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; $< 0 . 0 0 1 \%$ Influenza A/H5 Target 1 RNA; bovine serum albumin; dithiothreitol; RNase OUT (recombinant ribonuclease inhibitor); and carbohydrate.

Reconstituted contents: $4 0 \mu \mathrm { L }$

• One Target 2 Positive Control $( + )$ vial. Once reconstituted, the Target 2 Positive Control vial contains all reagents necessary for Target 2 RT-PCR, in addition to 335 copies/vial of RNA derived from a synthetic plasmid containing the target 2 sequence (and not the target 1 sequence) from a consensus influenza $\mathrm { A } / \mathrm { H } 5$ (Asian lineage) strain. The Target 2 Positive Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens. Amplification in the Target 2 Positive Control gives assurance that kit reagents are functioning properly and that the assay setup has been performed correctly. A Positive Target 2 Control must be included with each JBAIDS run. Ingredients: Each Positive Target 2 Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ MMLV-RT; $< 0 . 0 0 1 \%$ targetspecific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific

hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; $< 0 . 0 0 1 \%$ Influenza A/H5 Target 2 RNA; bovine serum albumin; dithiothreitol; RNase OUT; and carbohydrate.   
Reconstituted contents: $4 0 \mu \mathrm { L }$

• One Target 1 Negative Control (-) vial. Once reconstituted, the Target 1 Negative Control vial contains all materials required to perform a RT-PCR reaction except the Target 1 template, and should therefore give negative results. The Target 1Negative Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens to provide assurance that the setup procedure has been performed without contamination. A Target 1 Negative Control must be included with each JBAIDS run.

Ingredients: Each Target 1 Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ MMLV-RT; $< 0 . 0 0 1 \%$ targetspecific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; bovine serum albumin; dithiothreitol; RNase OUT; and carbohydrate. Reconstituted contents: $4 0 \mu \mathrm { L }$

One Target 2 Negative Control (-) vial. Once reconstituted, the Target 2 Negative Control vial contains all materials required to perform a RT-PCR reaction except the Target 2 template, and should therefore give negative results. The Target 2 Negative Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens to provide assurance that the setup procedure has been performed without contamination. A Target 2 Negative Control must be included with each JBAIDS run.

Ingredients: Each Target 1 Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ MMLV-RT; $< 0 . 0 0 1 \%$ targetspecific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; bovine serum albumin; dithiothreitol; RNase OUT; and carbohydrate.

Reconstituted contents: $4 0 \mu \mathrm { L }$

Each sample testing pouch contains the following vials:

One Sample Control vial. Once reconstituted, the Sample Control (SC) vial contains reagents that amplify and detect the human genomic RNase P sequence. The target sequence consists of RNase P-specific primer binding sites and an RNase P-specific probe binding site. The Sample Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ purified patient sample to ensure patient nucleic acid is available in the sample, and that RT-PCR inhibiting substances in the test specimen are detected, thereby preventing false negative results.

Ingredients: Each Sample Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ MMLV-RT; $< 0 . 0 0 1 \%$ RNase P-specific forward and reverse primer; $< 0 . 0 0 1 \%$ RNAse P-specific hydrolysis probe; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; bovine serum albumin; dithiothreitol; RNase OUT; and carbohydrate.

Reconstituted contents: $4 0 \mu \mathrm { L }$

One Target 1 Unknown vial. Once reconstituted, the Target 1 Unknown vial contains all required materials to perform a RT-PCR reaction, except target template. The Target 1 Unknown vial is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ purified patient sample, and will therefore give negative results unless the Target 1 template is found in the patient sample. Ingredients: Identical formulation to Target 1 Negative Control Reconstituted contents: $4 0 \mu \mathrm { L }$

• One Target 2 Unknown vial. Once reconstituted, the Target 2 Unknown vial contains all required materials to perform a RT-PCR reaction, except target template. The Target 2 Unknown vial is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 ~ \mu \mathrm { L }$ purified patient sample, and will therefore give negative results unless the Target 2 template is found in the patient sample. Ingredients: Identical formulation to Target 2 Negative Control Reconstituted contents: $4 0 \mu \mathrm { L }$

Each reconstitution buffer/reagent grade water pouch contains:

2X Reconstitution Buffer (2X RB). The buffer solution (purple buffer) is matched to the Influenza A/H5 (Asian lineage) assays and is used undiluted. The buffer enhances RT-PCR kinetics and is used to resuspend the JBAIDS freeze-dried reagents. Once reconstituted, all freeze dried reagent vials contain all of the components required for RT-PCR.   
Contents: $6 0 0 \mu \mathrm { L }$ Reagent Grade Water. Reagent grade water is molecular biology grade water used to reconstitute the Positive and Negative Controls.   
Contents: $8 5 0 ~ \mu \mathrm { L }$

# Materials Provided

<table><tr><td rowspan=1 colspan=3>Kit ContentsAll vials in the assay pouches contain sufficient reagent for two capillary reactions.</td></tr><tr><td rowspan=1 colspan=1>Qty</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Contents</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A/H5 Controlpouches</td><td rowspan=1 colspan=1>One Target 1 Positive Control (+), One Target 1Negative Control (-), One Target 2 Positive Control(+), and One Target 2 Negative Control (-).</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A/H5 SampleTesting pouches</td><td rowspan=1 colspan=1>One Target 1Unknown (U), One Target 2 Unknown(U), and One Sample Control.</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Purple buffer pouches(purple foil pouch)</td><td rowspan=1 colspan=1>One 2X purple reconstitution buffer (2X RB) (600µL), and One Reagent grade water (850 µL).</td></tr><tr><td rowspan=1 colspan=3>Each kit contains sufficient reagents for testing four specimens.</td></tr></table>

# Materials Required But Not Provided

<table><tr><td rowspan=1 colspan=2>Required Equipment</td></tr><tr><td rowspan=1 colspan=1>Minicentrifuge capable of 2000 x g, 5215*(Labnet C-1200 or equivalent)</td><td rowspan=1 colspan=1>Capillary adaptor for minicentrifuge (RocheApplied Science 1750-1.5* or equivalent)</td></tr><tr><td rowspan=1 colspan=1>Micropipette: 20 µL200 μL Micropipette: 200μL-1000 μL</td><td rowspan=1 colspan=1>Vortex-Genie® 3823*(VWR 58810-163 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>JBAIDS Instrument</td><td rowspan=1 colspan=1>Extra sample carousel JRPD-SUB-0010*</td></tr><tr><td rowspan=1 colspan=2>Note: See appropriate IT 1-2-3TM Sample Purification Kit for additional equipment andmaterials required for sample purification.</td></tr></table>

\*Available from Idaho Technology

<table><tr><td rowspan=1 colspan=2>Materials and Reagents Not Provided</td></tr><tr><td rowspan=1 colspan=1>LightCycler glass capillaries and caps(Roche Applied Science 1 909 339)*</td><td rowspan=1 colspan=1>Aerosol-resistant (filter), nuclease-free pipettetips, appropriate for micropipettes</td></tr><tr><td rowspan=1 colspan=1>Microcentrifuge tube rack LABS-SUP-0001*</td><td rowspan=1 colspan=1>Powder-free latex or nitrile gloves or equivalent</td></tr><tr><td rowspan=1 colspan=1>IT 1-2-3TM RNA Module, ASAY-ASY-0501* Sodium hypochlorite solution (household bleach)</td><td rowspan=1 colspan=1>IT 1-2-3TM RNA Module, ASAY-ASY-0501* Sodium hypochlorite solution (household bleach)</td></tr><tr><td rowspan=1 colspan=1>DNAZapTM, (Ambion AM9890 or equivalentDNA degradation solution)</td><td rowspan=1 colspan=1>IT 1-2-3 VIBE Sample Purification Kit,(ASAY-ASY-0500)* orIT 1-2-3™ Platinum Path Purification Kit,(ASAY-ASY-0120)*</td></tr></table>

\*Available from Idaho Technology

Contents of IT 1-2-3 Sample Purification Kits   

<table><tr><td>IT 1-2-3 VIBE Sample Purification Kit</td><td>IT 1-2-3 Platinum Path Purification Kit</td></tr><tr><td>Bead tubes for bead-beating</td><td>MagBead strip tubes</td></tr><tr><td>Spin column filters</td><td>Covers for strip tubes</td></tr><tr><td>Receiver tubes</td><td>Bead tubes for bead-beating</td></tr><tr><td>Transfer pipettes</td><td>MB binding buffer</td></tr><tr><td>Buffer 1A (Binding Buffer)</td><td>Receiver tubes</td></tr><tr><td>Buffer 1B (Binding Buffer)</td><td>Transfer pipettes</td></tr><tr><td>Buffer 1C (Modified Wash Buffer)</td><td>Protease (VIBE and FLOW)</td></tr><tr><td>Buffer 2 (Wash Buffer)</td><td>Swabs</td></tr><tr><td>Buffer 3 (Elution Buffer) VIBE Protease</td><td>Buffers for use with alternate applications</td></tr><tr><td>Instruction Booklet</td><td>• SCOOP Lysis, Buffer 1 and Buffer 2 Instructions Booklet</td></tr><tr><td></td><td></td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Influenza A/H5 (Asian lineage) Virus Real-time RT-PCR Primer and Probe Set and the CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)

2. Predicate K number(s): K060159 K080570

# 3. Comparison with predicate(s):

<table><tr><td rowspan=1 colspan=4>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate 1</td><td rowspan=1 colspan=1>Predicate 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>JABIDS Influenza A/H5 (Asianlineage) Detection Kit</td><td rowspan=1 colspan=1>Influenza A/H5 (Asianlineage) Virus Real-time RT-PCR Primers andProbe Set (K060159)</td><td rowspan=1 colspan=1>CDC Human Influenza Virus Real-time RT-PCR Detection andCharacterization Panel (rRT-PCRFlu Panel) (K080570)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Nasopharyngeal or throatswab respiratory specimens</td><td rowspan=1 colspan=1>Human respiratoryspecimens or virus cultures</td><td rowspan=1 colspan=1>Nasopharyngeal or nasalswab respiratory specimens, orvirus culture</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Real-time RT-PCR</td><td rowspan=1 colspan=1>Real-time RT-PCR</td><td rowspan=1 colspan=1>Real-time RT-PCR</td></tr><tr><td rowspan=1 colspan=1>ExtractionMethod</td><td rowspan=1 colspan=1>• IT 1-2-3T VIBE SamplePurification Kit, IdahoTechnology Inc.IT 1-2-3TMPlatinum PathPurification Kit, IdahoTechnology Inc.</td><td rowspan=1 colspan=1>QIAamp® Viral RNAMini Kit, Qiagen Inc.• Qiagen RNeasy® MiniKit, Qiagen, Inc.MagNA Pure LC TotalNucleic Acid Isolation Kit,Roche Applied Science</td><td rowspan=1 colspan=1>• QIAamp® Viral RNA Mini Kit,Qiagen Inc.Qiagen RNeasy® MiniKit, Qiagen, Inc.•MagNA Pure LC RNA IsolationKit II, Roche Applied Science• MagNA Pure Total Nucleic AcidIsolation Kit, Roche AppliedScience</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate 1</td><td rowspan=1 colspan=1>Predicate 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>JABIDS Influenza A/H5(Asian lineage) Detection Kit[</td><td rowspan=1 colspan=1>Influenza A/H5 (Asiantlineage) Virus Real-time RT-PCR Primers andProbe Set (K060159)</td><td rowspan=1 colspan=1>CDC Human Influenza Virus Real-time RT-PCR Detection andCharacterization Panel (rRT-PCRFlu Panel) (K080570)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative in vitro detectionof influenza A/H5 (Asianlineage) virus</td><td rowspan=1 colspan=1>Qualitative in vitro detectionof influenza A/H5 (Asianlineage) virus</td><td rowspan=1 colspan=1>Qualitative in vitrodetection of influenza A/H1, A/H3,A/H5 (Asian lineage), andinfluenza B viruses</td></tr><tr><td rowspan=1 colspan=1>OrganismDetected</td><td rowspan=1 colspan=1>Influenza A/H5 (Asianlineage)</td><td rowspan=1 colspan=1>Influenza A virus,subtype H5 (Asian lineage)</td><td rowspan=1 colspan=1>Influenza A/H1, A/H3,A/H5 (Asian lineage), andinfluenza B viruses</td></tr><tr><td rowspan=1 colspan=1>EnzymeMasterMix</td><td rowspan=1 colspan=1>DNA polymerase complexand MMLV-RT suppliedwith the JABIDS InfluenzaA/H5 (Asian lineage)Detection Kit</td><td rowspan=1 colspan=1>Qiagen QuantiTectTM ProbeRT-PCR Kit, Qiagen, Inc.</td><td rowspan=1 colspan=1>InvitrogenSuperScript™ III Platinum® One-Step Quantitative RT-PCR Kits(with or without ROX)</td></tr><tr><td rowspan=1 colspan=1>RequiredInstrumentation</td><td rowspan=1 colspan=1>JBAIDS Instrument with itsIVD software.</td><td rowspan=1 colspan=1>Roche LightCycler®Cepheid SmartCycler®Applied Biosystems7000 SequenceDetection SystemApplied Biosystems Prism®7700 Sequence DetectionSystem</td><td rowspan=1 colspan=1>Applied Biosystems7500 Fast Dx Real- Time PCRInstrument with SDS softwareversion 1.4</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses • Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses

# L. Test Principle:

The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit is a real time RT-PCR assay based on the influenza A/H5 (Asian lineage) assays that are part of the “CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, or the IT 1-2- $3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instruments.

Real-time RT-PCR involves reverse transcription of specific RNA sequences into complementary DNA sequences, followed by logarithmic amplification and simultaneous detection of those DNA sequences. The JBAIDS Influenza A/H5 (Asian lineage) PCR primers specifically amplify conserved regions of the hemagglutinin (HA) gene of the Influenza A/H5 (Asian lineage) virus. The JBAIDS Influenza A/H5 (Asian lineage) assays use hydrolysis probes to detect amplification of the transcribed RNA sequence of interest. Each hydrolysis probe is labeled on $5 ^ { \circ }$ end with a fluorescent reporter moiety (6-FAM) and internally with a quencher (TAMRA), which prevents the probe from emitting fluorescent signal. During PCR, the probe binds to a target sequence in the PCR product. When the Taq polymerase replicates a template on which a hydrolysis probe is bound, the exonuclease activity of the polymerase cleaves the probe, separating the fluorophore from the quencher, and fluorescent signal is generated. This fluorescence is measured and displayed by the JBAIDS instrument during the PCR reaction. The fluorescent signal increases as additional template is amplified and more probe is hydrolyzed.

The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit amplifies two target RNA sequences within the hemagglutinin gene of Influenza A, a gene that encodes one of two antigenic surface glycoproteins on the viral envelope. The primers and probes for the two target sequences are specific to the H5 subtype (Asian lineage) of the Influenza A virus. The primers and probes detect all sequenced virulent isolates of H5 (Asian lineage) Influenza A, and no other hemagglutinin Influenza A subtypes (as of January 2010, NCBI Influenza Virus Database). Target one is a 149 base sequence $5 ^ { \circ }$ of the hemagglutinin precursor cleavage site. Target 2 is a 192 base sequence $3 ^ { \circ }$ of the hemagglutinin precursor cleavage site.

The Sample Control for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit amplifies the human gene encoding for ribonuclease P (RNase P).

The JBAIDS Influenza A/H5 (Asian lineage) Detection kit can be used to test human throat swabs and nasopharyngeal swabs. Samples must be purified prior to testing with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. The purpose of sample purification is to release nucleic acid (RNA and DNA) contained in the unknown sample and to remove extraneous materials (e.g., proteins and chemicals) that can damage the nucleic acid target or interfere with the PCR reaction. The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit can be used with either the IT 1-2-3 VIBE Sample Purification Kit or the IT 1-2-3 Platinum Path Purification Kit. The VIBE kit is based on spin column technology while the Platinum Path kit uses magnetic bead technology. Both methods have four basic steps; lysis to release the nucleic acids, binding of the nucleic acids to a substrate (silica spin column or silica magnetic beads), washing away extraneous materials and elution of the purified nucleic acid. A brief outline of the two purification kit methods is provided below:

Overview of VIBE and Platinum Path Sample Purification Protocols   

<table><tr><td rowspan=1 colspan=1>Step</td><td rowspan=1 colspan=1>IT -1-2-3 VIBE Sample Purification Kit</td><td rowspan=1 colspan=1>IT 1-2-3 Platinum Path SamplePurification Kit</td></tr><tr><td rowspan=1 colspan=1>Cell Lysis to releasenucleic acids</td><td rowspan=1 colspan=1>Heat is used to disturb the cell membranes. A Bead-beating (vortexing specimen in theprotease is included to degrade proteins thatmight interfere with the PCR reaction.Carrier RNA is added to protect nucleic acids degrade proteins that might interfere with thefor degradation.</td><td rowspan=1 colspan=1>Heat is used to disturb the cell membranes. A Bead-beating (vortexing specimen in thepresence of silica beads) is used to physicallydisrupt the cells. A protease is included toCarrier RNA is added to protect nucleic acids degrade proteins that might interfere with thePCR reaction.</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidBinding</td><td rowspan=1 colspan=1>The processed sample is added to silica spinfilter in the presence of binding buffer. Thesample is centrifuges and the nucleic acidsadhere to the spin filter.</td><td rowspan=1 colspan=1>The bead beaten sample is transfer to a wellof strip tube containing binding buffer andmagnetic beads. The nucleic acid binds to themagnetic beads.</td></tr><tr><td rowspan=1 colspan=1>Washing to removeextraneous materials</td><td rowspan=1 colspan=1>A wash buffer is added to the spin columnwhich is then centrifuged. The wash bufferremoves extraneous materials suchas high salts and proteins while the nucleicacid remains bond to the silica column.The wash process is repeated twice.</td><td rowspan=1 colspan=1>Using a pick-pen with a retractable magnet,the magnetic beads are collected from thefirst well of the strip tube and transferred to anew well containing a wash buffer. Thebuffer removes extraneous materials whilethe nucleic acid remains bond to the magneticbeads. The beads are washed twice (twowells of the strip tube).</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Elution of nucleic acids An elution buffer is added to the spin columnwith is then centrifuged to release the nucleicacid from the column. The collected materialis the purified patient sample.</td><td rowspan=1 colspan=1>[Using the pick-pen, the magnetic beads aretransferred to the last well of the strip tubecontaining elution buffer. The nucleic acidsare released from the magnetic beads. Thebeads are then removed from the sampleusing the pick pen.</td></tr></table>

Purified samples are tested for the presence of Influenza A/H5 (Asian lineage) RNA using the freeze-dried reagents in the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. Each JBAIDS assay run requires duplicate capillaries of a Positive

Control (PC) and a Negative Control (NC) for each of the two target assays. Each sample tested requires duplicate capillaries of a Sample Control and two target assays (Target 1 Unknown and Target 2 Unknown).

Reagents are set up in the following order:

1. Target 1 and 2 Unknowns – Reconstitute the reagent pellets in the Unknown vials by adding $2 0 \mu \mathrm { L }$ of the provided reconstitution buffer and $2 0 \mu \mathrm { L }$ of the purified sample.   
2. Sample Control – Repeat the above process with a Sample Control vial.   
3. Positive Control – After the Target 1 and Target 2 Unknowns and the Sample Control are set up for each sample, set up the Target 1 and 2 Positive Controls by reconstituting the pellet with $2 0 \mu \mathrm { L }$ of the provided reconstitution buffer and $2 0 \mu \mathrm { L }$ of the provided reagent-grade water.   
4. Negative Control – Set up the Target 1 and 2 Negative Control last, using reconstitution buffer and water, as described for the Positive Control.   
5. For each reagent vial, transfer $1 9 \mu \mathrm { L }$ of the reconstituted reagent to each of two capillaries following the same order used for reconstitution. Caps are placed on the capillary tubes before proceeding to the next capillary.   
6. Centrifuge the capillary tubes to ensure that the PCR reagent is in the tip of the capillary.

Each JBAIDS test is analyzed and assigned a final result by the Detector module of the JBAIDS Software. Possible final results are positive, negative, sample control failure, uncertain, or invalid. To assign a final test result, Detector first analyzes the data from each capillary independently before analyzing the sample duplicates together. Finally, the software assigns a final result, or a combined call, based on the results of the sample and all of its controls.

Each stage of the analysis is described below:

# 1. Independent Capillary Call

Detector does not rely on any single aspect of the amplification curve, but rather integrates a number of factors, such as curve shape and signal-to-noise ratio into a combined score. Curves scoring above a threshold are called positive, curves scoring below a threshold are called negative, and curves scoring in a small area in between are called uncertain. The scoring system has been carefully tuned to match human expert calls on real amplification curve data. Detector calls differ from expert calls about as frequently as two experts differ from each other.

A description of how Detector assigns test results is the following:

Negative – A curve is called negative if its shape can be closely approximated by a line (or smooth curve), if it has low signal to noise, and if it shows little or no increase in fluorescence during the PCR.

Positive – A curve is called positive if the fluorescence shows exponential growth out of the background, if it has high signal to noise, and if the slope of the exponential region of the amplification is consistent with a PCR efficiency of around two.

Uncertain – For a small number of curves (about $0 . 2 \%$ ) no clear call is possible and the reaction is called uncertain.

In addition to the above criteria, Positive Controls must have a Cp value earlier than an assay specific cutoff value. This cutoff value is established by evaluating the historic variability of the Cp for the Positive Control and is used to establish proper performance of the Positive Control.

# 2. Replicate Calls

Once each amplification curve is evaluated by Detector, the software evaluates the sample duplicates according to the following rules:

Positive Control Replicates – The Positive Control is called a success only if both capillaries are called positive.

Negative Control Replicates – The Negative Control is called a success only if both capillaries are called negative.

Sample Control Replicates – A Sample Control is called a success only if both capillaries are called positive.

Unknown Replicates – Unknowns are called as the following:

• Positive – An Unknown is called positive only if both capillaries are called positive. Negative – An Unknown is called negative only if both capillaries are called negative. Uncertain – An Unknown is called uncertain if the results of the two capillaries do not match (e.g., positive/negative) or when both capillaries are called uncertain.

# 3. Final Result for each Target Assay

The final test result is a combined call (or meta-call) based on the results of the test sample and its associated controls. Failure of the Positive or Negative Control results in invalid results for all associated samples. Sample Controls with a SCFail (Sample Control Failure) result are further analyzed before a final result is assigned. A SCFail result can indicate that the patient sample contains substances capable of inhibiting the PCR reaction, there was inefficient sample purification or that poor specimen collection occurred. In certain cases, sufficient viral target may be recovered and detection of the sample control is irrelevant. Therefore, failure of the Sample Control for a positive sample is of no consequence because amplification of the target assays indicates that the sample does not contain significant levels of PCR inhibitors.

Samples with SCFail (with a negative or uncertain Target assay result), invalid, or uncertain results require follow-up testing in order to achieve a valid (positive or negative) result.

Interpretation of Target 1 and Target 2 assays   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Explanation</td><td rowspan=1 colspan=1>Action</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>A red positive indicates that the PC andProceed to &quot;Interpretation of Patient Results&quot;.NC/NEC gave the expected results andthat both capillaries for the target assaywere positive.</td><td rowspan=1 colspan=1>A red positive indicates that the PC andProceed to &quot;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>A green negative indicates that the PC,NC/NEC, and Sample Control gave theexpected results and that bothcapillaries for the target assay werenegative.</td><td rowspan=1 colspan=1>Proceed to &quot;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Uncertain</td><td rowspan=1 colspan=1>A yellow uncertain indicates that thePC, NC/NEC, and Sample Controlgave the expected results and thatThe results for the twocapillaries for the target assaydo not agree ORThe results for the twocapillaries have anamplification curve that isuncertain (not definitive)</td><td rowspan=1 colspan=1>Retest using the same purified sample.If the retesting is positive or negative, interpretthe test using the retesting result.If the retesting result is uncertain, report theresult for the target assay as Uncertain.</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>A yellow invalid indicates a failure ofthe PC or NC/NEC for the associatedtarget assay (Target 1 or Target 2). Allsamples in the run will be assigned aninvalid result for the affected targetassay.</td><td rowspan=1 colspan=1>For PC or NC failure, retest using the samepurified sample. For a NEC failure, all samplesthat were purified in the same batch along withthe NEC should be re-purified starting fromanother aliquot of the original specimens or newsamples in VTM.If the retest result is positive or negative, interpretthe test using the retesting result.If the retesting result is uncertain, report theresult of the target assay as Uncertain.If the retesting result is invalid, refer to&#x27;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Sample ControlFailure (SCFail)</td><td rowspan=1 colspan=1>A yellow sample control failureindicates that the PCs and NCs/NECsfor the target assay were successful,ANDOne, or both, capillaries for a particularIf the retesting result is sample control failure, re-target assay are negative or uncertainANDAt least one Sample Control capillaryfrom a given sample is negative oruncertain.</td><td rowspan=1 colspan=1>Retest and interpret the results according to&#x27;Possible Retesting Results and Follow Up&quot;.If the retest result is positive or negative, interpretthe assay using the retesting result.One, or both, capillaries for a particularIf the retesting result is sample control failure, re-purify residual patient sample and test. If nosample is available, report the result of the targetassay as sample control failure.If the retesting result is uncertain, report theresult of the target assay as Uncertain.</td></tr></table>

# 4. JBAIDS Report

A JBAIDS report is generated automatically for each test run. When the Diagnostic Wizard is used, reports from the instrument are identified with the phrase “For In Vitro Diagnostic Use.”

# 5. Follow-up Testing

If the Influenza A/H5 (Asian lineage) assays both yield a negative or positive test result, then JBAIDS testing is complete. Any other result requires followup testing, as described below:

Invalid – If either the Positive Control or Negative Control (for either Target 1 or 2) fails to give the expected results, all Unknown samples for that specific target assay are called invalid and must be retested using the same purified samples.   
• Inconclusive – An inconclusive result on a single sample requires retesting with the same purified sample.   
• Sample Control Failure - A failed Sample Control can occur from PCR inhibition, inefficient sample purification, or poor specimen collection. Any sample having a failed Sample Control assay (i.e. negative or uncertain Sample Control assay associated with a negative or uncertain Target assay) will be retested using the original (undiluted) purified sample as well as a ten-fold dilution of the purified sample.

If the Sample Control Failure was due to an inhibitor present in the sample, the repeat testing will again yield a result of inhibited for the undiluted sample and a valid result (positive or negative) for the sample diluted 1:10. The results from the retesting can be used for diagnostic applications only if the diluted sample tests positive. A negative result cannot be reported due to reduced sensitivity of the test system at the 1:10 dilution.

If the Sample Control Failure result was caused by improper technique, or other factors, then the repeat testing for the undiluted sample may yield a valid result (positive or negative). In this case, the result from the retesting of the undiluted sample can be used for diagnostic applications.

If the Sample Control Failure was caused by ineffective sample purification or an improperly collected specimen, then the repeat testing will most likely be Sample Control Failure. In this case, residual patient specimen can be retested (provided it was appropriately stored) or a new specimen should be requested.

Possible Retesting Results and Follow-up   

<table><tr><td rowspan=1 colspan=1>Result of Retesting,Original Sample</td><td rowspan=1 colspan=1>Result of Testing1:10 Dilution</td><td rowspan=2 colspan=1>gTest Interpretation and Follow-up</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for the particular target. Proceedaccording to &quot;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive for the particular target. Proceedaccording to &quot;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Sample ControlFailure</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Sample control failure. (Note: DO NOT report asnegative due to reduced sensitivity of the testsystem). Obtain another aliquot of the originalsample or a new sample and re-purify.</td></tr><tr><td rowspan=1 colspan=1>Sample ControlFailure</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive for the tested target. Proceed according to&quot;Interpretation of Patient Results&quot;.</td></tr><tr><td rowspan=1 colspan=1>Sample ControlFailure</td><td rowspan=1 colspan=1>Sample ControlFailure</td><td rowspan=1 colspan=1>Sample failure. Obtain another aliquot of theoriginal sample or a new sample and re-purify.</td></tr></table>

# 6. Result Interpretation

The JBIADS software automatically interprets the results for each of the target assays. The results of the two target assay are used to make the final test interpretation and based upon the final test interpretation, specific actions are required.

Influenza A/H5 (Asian lineage) RNA not detected – The JBAIDS test results for Target 1 and Target 2 were Negative and all controls   
performed as expected.   
Presumptive positive for Influenza A/H5 (Asian lineage) virus - The JBAIDS test results for Target 1 and Target 2 assay were positive. Report to established DoD reporting structure. (All suspected, presumptive positive or confirmed cases of Influenza A/H5 (Asian lineage) shall be reported to the respective military installation/command Public Health Emergency Officer (PHEO). The PHEO will then notify, through   
established reporting channels, the appropriate chain-of- command, the Centers for Disease Control and Prevention (CDC), state/local   
government public health agencies, and host nations if outside of the United States (OCONUS)).   
Inconclusive for Influenza A/H5 (Asian lineage) virus – The JBAIDS test results for Target 1 and Target 2 are discordant (one target is positive or uncertain and the other target is negative, uncertain, invalid or sample control failure). Retesting of purified sample (if available) is required. If retest is still inconclusive and sufficient sample is available, re-purify and retest. If results remain inconclusive, it is considered a suspected case of Influenza A/H5 (Asian lineage). Report to established DoD reporting structure. (All suspected, presumptive positive or confirmed cases of Influenza A/H5 (Asian lineage) shall be reported to the respective   
military installation/command Public Health Emergency Officer (PHEO). The PHEO will then notify, through established reporting channels, the appropriate chain-of- command, the Centers for Disease Control and

Prevention (CDC), state/local government public health agencies, and host nations if outside of the United States (OCONUS)).

# Interpretation of Patient Results

<table><tr><td rowspan=1 colspan=1>Sample ResultFor Target 1</td><td rowspan=1 colspan=1>Sample ResultFor Target 2</td><td rowspan=1 colspan=1>Action</td><td rowspan=1 colspan=1>Result Interpretation /Report</td></tr><tr><td rowspan=1 colspan=1>Negative(both capillaries)</td><td rowspan=1 colspan=1>Negative(both capillaries)</td><td rowspan=1 colspan=1>No further testing</td><td rowspan=1 colspan=1>Influenza A/H5 (Asian lineage) RNAnot detectedThis result does not exclude influenzaviral infection.</td></tr><tr><td rowspan=1 colspan=1>Positive(both capillaries)</td><td rowspan=1 colspan=1>Positive(both capillaries)</td><td rowspan=1 colspan=1>Report to established DoDreporting structure</td><td rowspan=1 colspan=1>Presumptive positive for InfluenzaA/H5 (Asian lineage) RNAThis result does not rule out co-infections with other pathogens oridentify any other specific influenzaA virus subtype.</td></tr><tr><td rowspan=1 colspan=1>Positive(both capillaries),or Uncertain</td><td rowspan=1 colspan=1>Negative orUncertain</td><td rowspan=1 colspan=1>Report to established DoDreporting structure</td><td rowspan=1 colspan=1>Inconclusive of Influenza A/H5(Asian linage)</td></tr><tr><td rowspan=1 colspan=1>Negative, orUncertain</td><td rowspan=1 colspan=1>Positive(both capillaries)or Uncertain</td><td rowspan=1 colspan=1>Report to established DoDreporting structure</td><td rowspan=1 colspan=1>Inconclusive of Influenza A/H5(Asian linage)</td></tr><tr><td rowspan=1 colspan=1>Invalid or SampleControl Failure</td><td rowspan=1 colspan=1>Invalid or SampleControl Failure</td><td rowspan=1 colspan=1>Contact JBAIDS TechnicalAssistance/SME forguidance</td><td rowspan=1 colspan=1>Invalid</td></tr></table>

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Precision/Reproducibility:

A multicenter reproducibility study was performed to determine the overall system reproducibility of the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit on the JBAIDS instruments. This study was performed at Idaho Technology Inc. (ITI) and two external sites, Naval Health Research Center (NHRC) and Brooke Army Medical Center (BAMC). The study was conducted with two specimen types: throat swabs (TS) and nasopharyngeal swabs (NPS), both collected in viral transport medium (VTM). Both specimen types were purified using two different Sample Purification Kits, the IT 1-2-3 VIBE and the IT 1-2-3 Platinum Path Sample Purification Kit.

A panel of eight TS specimens, spiked with an inactivated Influenza A/H5 (Asian lineage) virus strain (Vietnam/ $1 2 0 3 / 0 4 \mathrm { x }$ PR8 strain) at three different concentrations (LoD/15, LoD, and 3X LoD), were tested twice each day for three consecutive days at the three testing sites. The panel was tested for an additional day at ITI (total of 4 testing days at ITI). On each testing day, two operators at each site purified and tested one aliquot of each specimen, 8 total specimens per operator per day. One of the operators used the IT 1-2-3 VIBE Sample Purification Kit and the other operator used the IT 1-2-3 Platinum Path Sample Purification Kit.

A second panel of eight NPS specimens, spiked with an inactivated Influenza A/H5 (Asian lineage) virus strain at three different concentrations (LoD/15, LoD, and 3X LoD), were tested twice each day for another three consecutive days at the three testing sites. The panel was tested for an additional day at ITI (total of 4 testing days at ITI). On each testing day, two operators at each site purified and tested one aliquot of each specimen, 8 total specimens per operator per day. One of the operators used the IT 1-2-3 VIBE Sample Purification Kit and the other operator used the IT 1-2-3 Platinum Path Sample Purification Kit.

Sample panels were prepared by spiking 8 pooled TS and 8 pooled NPS specimens collected in viral transport medium (VTM) from normal healthy volunteers with an inactivated Influenza A/H5 (Asian lineage) strain (Vietnam/ $1 2 0 3 / 0 4 \mathrm { ~ x ~ P R ~ }$ 8 strain) at designated levels. The TS and NPS specimen pools were spiked at three concentrations of the inactivated Influenza A/H5 (Asian lineage) virus, representing a high negative sample (approximately 15-fold lower than LoD), a low positive sample (at LoD), and a medium positive sample (3X LoD). The test system LoD using live virus was determined to be 50 $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for both TS and NPS samples in the LoD study. However, in a separate study, the “Template Comparison for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit Study”, it was determined that the stock of inactivated virus used in the reproducibility study contains nearly 15-fold less RNA as compared to the live reassortant virus used to determine the LoD. Therefore, the equivalent LoD for the inactivated virus is approximately $7 5 0 \ : \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ .

The NPS reproducibility panel composition is presented in the following table:   

<table><tr><td rowspan=1 colspan=1>Level of FluA/H5 ReassortantVirus in NPS Specimen</td><td rowspan=1 colspan=1>Live VirusConcentration(EID50/mL)</td><td rowspan=1 colspan=1>Equivalent InactivatedVirus Concentration(EID50/mL)</td><td rowspan=1 colspan=1>Panel MemberNumber</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/15)</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>#5,#6</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>#1, #4, #7</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>2,250</td><td rowspan=1 colspan=1>#2, #3, #8</td></tr></table>

The TS reproducibility panel composition is presented in the following table:   

<table><tr><td rowspan=1 colspan=1>Level of FluA/H5 ReassortantVirus in TS Specimen</td><td rowspan=1 colspan=1>Live VirusConcentration(EID50/mL)</td><td rowspan=1 colspan=1>Equivalent InactivatedVirus Concentration(EID50/mL)</td><td rowspan=1 colspan=1>Panel MemberNumber</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/15)</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>#2,#7</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>#1, #4, #8</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>2,250</td><td rowspan=1 colspan=1>#3, #5, #6</td></tr></table>

Following spiking, each sample pool was aliquoted into several individual use vials that were placed at $2 { - } 8 ^ { \circ } \mathrm { C }$ until testing. On the day the specimen pools were prepared, an appropriate number of single use aliquots were shipped to the two external sites (BAMC and NHRC) and one of the aliquots was tested at ITI.

Positive Control (PC) and Negative Control (NC) reactions for both Target 1 and Target 2 were included in all 102 JBAIDS runs performed in the reproducibility study. Overall, 99 out of 102 $( 9 7 . 1 \% )$ runs had successful PCs and NCs for both target assays. The PC results were of good run-to-run precision, with an average Cp of $3 2 . 5 2 \pm 0 . 3 5$ ( $1 . 0 8 \%$ CV) for the Target 1 PC and an average Cp of $3 2 . 9 5 \pm$ 0.44 ( $1 . 3 4 \%$ CV) for the Target 2 PC. Of the three invalid runs, two were the result of user error (in one run, the PCs were reconstituted incorrectly and in the other run the Target 2 capillary order was switched) and one was caused by a failure of the Target 2 PC. Upon retesting, all controls were successful.

The Sample Control assay (human RNase P assay) was tested with each purified sample to detect inhibition, poor sample extraction, or poor specimen collection. A total of 364 out of 367 $( 9 9 . 2 \% )$ Sample Control reactions produced the expected positive results in the reproducibility study. The Sample Control results were of good run-to-run precision, with an average $\mathrm { C p }$ of $3 0 . 1 1 \pm 1 . 4 4$ $4 . 7 8 \%$ CV) for the TS (Platinum Path) Sample Control, an average $\mathrm { C p }$ of $3 1 . 4 0 \pm 0 . 7 3$ $( 2 . 3 2 \%$ CV) for the TS (VIBE) Sample Control, an average $\mathrm { C p }$ of $2 9 . 0 4 \pm 0 . 8 5$ $( 2 . 9 3 \%$ CV) for the NPS (Platinum Path) Sample Control, and an average Cp of $3 1 . 0 4 \pm$ 1.08 $3 . 4 8 \%$ CV) for the NPS (VIBE) Sample Control. Sample Controls for two TS specimens purified using the IT 1-2-3 Platinum Path kit and one NPS specimen purified with IT 1-2-3 VIBE initially failed (gave negative results) initially, however all were successful upon retesting.

Reproducibility Study Summary (Agreement with Expected Positive Results) for the JBAIDS Influenza $\mathbf { A } / \mathbf { H } 5$ (Asian lineage) Detection Kit   

<table><tr><td rowspan=3 colspan=1>SampleType</td><td rowspan=3 colspan=1>Test Level</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path Sample Purification KitIT 1-2-3 VIBE Sample Purification Kit</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path Sample Purification KitIT 1-2-3 VIBE Sample Purification Kit</td><td rowspan=3 colspan=1>AllPurificationKits, All Sites</td><td rowspan=3 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=4>Test Location</td><td rowspan=1 colspan=4>Test Location</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=5 colspan=1></td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>94.0-100.0</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>29/3097%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098%</td><td rowspan=1 colspan=1>91.1-100.0</td></tr><tr><td rowspan=1 colspan=1>LoD/15</td><td rowspan=1 colspan=1>2/8</td><td rowspan=1 colspan=1>1/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>3/2015%</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>1/6</td><td rowspan=1 colspan=1>0/6</td><td rowspan=1 colspan=1>1/205%</td><td rowspan=1 colspan=1>4/4010%</td><td rowspan=1 colspan=1>2.8-23.7</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>24/24100%</td><td rowspan=1 colspan=1>18/1800%</td><td rowspan=1 colspan=1>17/1894%</td><td rowspan=1 colspan=1>59/6098%</td><td rowspan=1 colspan=1>24/24100%</td><td rowspan=1 colspan=1>18/1800%</td><td rowspan=1 colspan=1>18/1800%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>119/12099%</td><td rowspan=1 colspan=1>95.4-100.0</td></tr><tr><td rowspan=1 colspan=1>Detection allLevels</td><td rowspan=1 colspan=1>26/3287%</td><td rowspan=1 colspan=1>19/2479%</td><td rowspan=1 colspan=1>17/2471%</td><td rowspan=1 colspan=1>62/8077%</td><td rowspan=1 colspan=1>24/3275%</td><td rowspan=1 colspan=1>19/2479%</td><td rowspan=1 colspan=1>18/2475%</td><td rowspan=1 colspan=1>61/8076%</td><td rowspan=1 colspan=1>123/16077%</td><td rowspan=1 colspan=1>70.3-83.4</td></tr><tr><td rowspan=5 colspan=1>N</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>60/6000%</td><td rowspan=1 colspan=1>94.0-100.0</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>29/3097%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>59/6098%</td><td rowspan=1 colspan=1>91.1-100.0</td></tr><tr><td rowspan=1 colspan=1>LoD/15</td><td rowspan=1 colspan=1>5/8</td><td rowspan=1 colspan=1>5/6</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>12/2060%</td><td rowspan=1 colspan=1>2/8</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>2/6</td><td rowspan=1 colspan=1>6/2030%</td><td rowspan=1 colspan=1>18/4045%</td><td rowspan=1 colspan=1>29.3-61.5</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>24/24100%</td><td rowspan=1 colspan=1>18/18100%</td><td rowspan=1 colspan=1>17/1894%</td><td rowspan=1 colspan=1>59/6098%</td><td rowspan=1 colspan=1>24/24100%</td><td rowspan=1 colspan=1>18/18100%</td><td rowspan=1 colspan=1>18/18100%</td><td rowspan=1 colspan=1>60/60100%</td><td rowspan=1 colspan=1>119/12099%</td><td rowspan=1 colspan=1>95.4-100.0</td></tr><tr><td rowspan=1 colspan=1>Detection AllLevels</td><td rowspan=1 colspan=1>29/3290%</td><td rowspan=1 colspan=1>23/2496%</td><td rowspan=1 colspan=1>19/2479%</td><td rowspan=1 colspan=1>71/8088%</td><td rowspan=1 colspan=1>26/3287%</td><td rowspan=1 colspan=1>20/2483%</td><td rowspan=1 colspan=1>20/2483%</td><td rowspan=1 colspan=1>66/8083%</td><td rowspan=1 colspan=1>134/16084%</td><td rowspan=1 colspan=1>78.0-89.5</td></tr><tr><td rowspan=1 colspan=2>Detection ≥ LoD, Both SampleTypes</td><td rowspan=1 colspan=1>48/48100%</td><td rowspan=1 colspan=1>36/36100%</td><td rowspan=1 colspan=1>34/3694%</td><td rowspan=1 colspan=1>118/12098%</td><td rowspan=1 colspan=1>48/48100%</td><td rowspan=1 colspan=1>36/36100%</td><td rowspan=1 colspan=1>36/3600%</td><td rowspan=1 colspan=1>120/120100%</td><td rowspan=1 colspan=1>238/24099%</td><td rowspan=1 colspan=1>97.0-99.9</td></tr><tr><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>92.6-100.0</td><td rowspan=1 colspan=1>90.3-00.0</td><td rowspan=1 colspan=1>81.3-99.3</td><td rowspan=1 colspan=1>94.1-99.8</td><td rowspan=1 colspan=1>92.6-100.0</td><td rowspan=1 colspan=1>90.3-100.0</td><td rowspan=1 colspan=1>90.3-100.0</td><td rowspan=1 colspan=1>97.0-100.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Detection All Levels, BothSample Types</td><td rowspan=1 colspan=1>55/6486%</td><td rowspan=1 colspan=1>42/4888%</td><td rowspan=1 colspan=1>36/4875%</td><td rowspan=1 colspan=1>133/16083%</td><td rowspan=1 colspan=1>50/6478%</td><td rowspan=1 colspan=1>39/4881%</td><td rowspan=1 colspan=1>38/487%</td><td rowspan=1 colspan=1>127/16079%</td><td rowspan=1 colspan=1>260/32082%</td><td rowspan=1 colspan=1>77.0-85.5</td></tr><tr><td rowspan=1 colspan=2>95% CI</td><td rowspan=1 colspan=1>75.0-93.4</td><td rowspan=1 colspan=1>74.7-95.3</td><td rowspan=1 colspan=1>60.4-86.4</td><td rowspan=1 colspan=1>77.3-88.9</td><td rowspan=1 colspan=1>66.0-87.5</td><td rowspan=1 colspan=1>67.4-91.0</td><td rowspan=1 colspan=1>65.0-89.5</td><td rowspan=1 colspan=1>73.1-85.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

For the specimens spiked at 3X LoD (medium positive) all 30 gave positive test results for both sample types (TS and NPS) and both purification kits (VIBE and Platinum Path). Specimens spiked at the LoD (low positive) gave positive test results for all (30/30) specimens (TS and NPS) purified using the VIBE kit while 29/30 tested positive when purified using the Platinum Path purification kit. As expected, specimens spiked at $1 / 1 5 \mathrm { L o D }$ (high negative) gave inconclusive test results for most specimens due to negative results for the Target 2 assay. This finding is consistent with slight differences in sensitivity between the two target assays when testing Vietnam $/ 1 2 0 3 / 0 4 \mathrm { ~ x ~ I ~ }$ PR8 strain. For detection of the Anhui $_ { | 0 1 / 0 5 \mathrm { ~ x ~ } }$ PR8 strain, the Target 2 assay is slightly more sensitive than the Target 1 assay (refer to the LoD study of the JBAIDS Influenza A/H5(Asian lineage) Detection Kit). These results also confirm the estimated LoD of the system. There were no significant differences in test system sensitivity and reproducibility when specimens were processed using the Platinum Path purification kit vs. the VIBE purification kit.

This reproducibility data also demonstrated that as expected with all real-time PCR assays, the JBAIDS Influenza A/H5 (Asian lineage) Detection Test may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration. This limitation should be addressed by including the following statement in the Limitation section of the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit Instructions for Use: “The JBAIDS Influenza A/H5 (Asian lineage) Detection Kit may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration.”

The variability of the Cp value can be used as an additional measure of the variability of JBAIDS assays. Unlike other real time PCR systems, the JBAIDS does not employ the Cp value to determine if an amplification curve is called positive or negative. Instead, the Detector module of the JBAIDS software performs a series of evaluations (e.g., curve shape, signal to noise) to determine if each amplification curve is called positive, negative, or uncertain. Only curves with positive or uncertain results are assigned a Cp value. While the Cp result is not used to determine the test result, samples with the same template concentration tested with the same assay are expected to have a similar Cp. Therefore, the average and $\% \mathrm { C V }$ of $\mathrm { C p }$ values obtained in this study were used to further evaluate the precision/reproducibility of the test system.

Reproducibility Study Summary (Average Cp and $\% 0 \%$ ) for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit   

<table><tr><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>TestLevel</td><td rowspan=4 colspan=1>Target</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum Path Sample Purification Kit</td><td rowspan=1 colspan=8>IT 1-2-3 VIBE Sample Purification Kit</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$Ov</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$Oac</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$Ov</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$Ov</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$ov</td><td rowspan=1 colspan=1>Ar </td><td rowspan=1 colspan=1>$Ov</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$Ov</td><td rowspan=1 colspan=1>Ave.Cp</td><td rowspan=1 colspan=1>$ov</td></tr><tr><td rowspan=6 colspan=1></td><td rowspan=2 colspan=1>3 X LoD</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>29.15</td><td rowspan=1 colspan=1>3.77</td><td rowspan=1 colspan=1>29.23</td><td rowspan=1 colspan=1>4.99</td><td rowspan=1 colspan=1>29.32</td><td rowspan=1 colspan=1>2.76</td><td rowspan=1 colspan=1>29.23</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>29.82</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>29.33</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>29.75</td><td rowspan=1 colspan=1>1.68</td><td rowspan=1 colspan=1>29.65</td><td rowspan=1 colspan=1>1.82</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>32.19</td><td rowspan=1 colspan=1>2.17</td><td rowspan=1 colspan=1>31.82</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>32.57</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>32.19</td><td rowspan=1 colspan=1>2.08</td><td rowspan=1 colspan=1>32.16</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>31.89</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>32.69</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>32.24</td><td rowspan=1 colspan=1>1.58</td></tr><tr><td rowspan=2 colspan=1>LoD</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>31.22</td><td rowspan=1 colspan=1>4.30</td><td rowspan=1 colspan=1>31.65</td><td rowspan=1 colspan=1>4.11</td><td rowspan=1 colspan=1>31.24</td><td rowspan=1 colspan=1>3.04</td><td rowspan=1 colspan=1>31.30</td><td rowspan=1 colspan=1>3.90</td><td rowspan=1 colspan=1>31.74</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>31.13</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>31.79</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>31.57</td><td rowspan=1 colspan=1>1.81</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>33.92</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>33.38</td><td rowspan=1 colspan=1>1.68</td><td rowspan=1 colspan=1>34.12</td><td rowspan=1 colspan=1>2.61</td><td rowspan=1 colspan=1>33.82</td><td rowspan=1 colspan=1>2.25</td><td rowspan=1 colspan=1>33.65</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>33.35</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>33.61</td><td rowspan=1 colspan=1>2.23</td><td rowspan=1 colspan=1>33.54</td><td rowspan=1 colspan=1>2.03</td></tr><tr><td rowspan=2 colspan=1>LoD/15</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>35.55</td><td rowspan=1 colspan=1>3.57</td><td rowspan=1 colspan=1>35.09</td><td rowspan=1 colspan=1>3.11</td><td rowspan=1 colspan=1>35.30</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>35.41</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>34.71</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>34.79</td><td rowspan=1 colspan=1>1.38</td><td rowspan=1 colspan=1>35.36</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>34.87</td><td rowspan=1 colspan=1>2.18</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>35.23</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>33.02</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>34.74</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>31.64</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>32.93</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>32.07</td><td rowspan=1 colspan=1>n/a*</td></tr><tr><td rowspan=6 colspan=1>NA</td><td rowspan=2 colspan=1>3 X LoD</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>28.61</td><td rowspan=1 colspan=1>1.43</td><td rowspan=1 colspan=1>28.02</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>28.78</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>28.48</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>30.38</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>29.54</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>30.24</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>30.08</td><td rowspan=1 colspan=1>1.73</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>32.32</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>31.78</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>32.86</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>32.06</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>33.49</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>32.69</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>33.30</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>33.19</td><td rowspan=1 colspan=1>1.75</td></tr><tr><td rowspan=2 colspan=1>LoD</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>30.46</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>2.24</td><td rowspan=1 colspan=1>30.72</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>30.53</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>32.25</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>31.43</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>32.09</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>31.96</td><td rowspan=1 colspan=1>1.63</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>33.74</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>33.36</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>34.92</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>33.87</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>34.36</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>33.84</td><td rowspan=1 colspan=1>1.86</td><td rowspan=1 colspan=1>34.04</td><td rowspan=1 colspan=1>3.64</td><td rowspan=1 colspan=1>34.11</td><td rowspan=1 colspan=1>2.96</td></tr><tr><td rowspan=2 colspan=1>LoD/15</td><td rowspan=1 colspan=1>Target 1</td><td rowspan=1 colspan=1>34.87</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>33.88</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>34.84</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>34.55</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>35.67</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>34.86</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>35.34</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>35.32</td><td rowspan=1 colspan=1>1.95</td></tr><tr><td rowspan=1 colspan=1>Target 2</td><td rowspan=1 colspan=1>36.18</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>35.67</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>36.16</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>36.04</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>34.28</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>31.80</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>34.70</td><td rowspan=1 colspan=1>n/a*</td><td rowspan=1 colspan=1>33.82</td><td rowspan=1 colspan=1>n/a*</td></tr></table>

Note: Unless otherwise noted, the $\mathrm { C p }$ value for each test result represents an average of two positive capillaries. When only one capillary is positive, the result is uncertain and the provided $\mathrm { C p }$ value is for the single positive capillary. \*If less than $80 \%$ of results were positive, the $\% \mathrm { C V }$ is listed as $\mathrm { n / a }$ because there is insufficient data for statistical analysis (e.g. Influenza A/H5 Target 2 high negative results).

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The following controls are included in the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit:

# Negative Control (NC)

The NC is used to detect contamination from target-specific amplified product, synthetic RNA (as found in the PC vials), or organism. Each JBAIDS run requires one NC (resulting in two capillaries) for the Target 1 and one for the Target 2 assays (total of 4 capillaries). For each target assay, both of the NC capillaries must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run, and the run must be repeated. Frequent or repeated failures of NCs may indicate significant contamination of the work area.

# Positive Control (PC)

The PC serves as an amplification and detection control. Each JBAIDS run requires one PC (resulting in two capillaries) for the Target 1 assay and one for the Target 2 assay (total of 4 capillaries). For each target assay, both of the PC capillaries must be positive and have Cp results that are earlier than the assay‘s specific cutoff value. The assay specific cutoff was initially set at less than or equal to $4 0 \mathrm { { C p } }$ in the software. The assay specific cutoff in the final released software will be set to the top of the manufacturing release criteria (35.23 for Target 1 and 35.68 for Target 2) plus 3 standard deviations of the variability of the controls observed during the clinical trial (0.63 for Target 1 and 1.21 for Target 2). If either capillary fails (i.e. Target 1 less than or equal to $3 7 . 1 \mathrm { C p }$ , or Target 2 less than or equal to $3 9 . 3 \mathrm { C p }$ ), the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the run must be repeated. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument. If the SC capillaries in the same test run are positive, then the failure is most likely caused by an isolated error with the setup of the PC. If the Sample Control (SC) capillaries are also negative, possible causes for failure are 1) a systematic error in sample setup, 2) degradation of the reagents, or 3) a malfunction of the JBAIDS instrument. The product releasing criteria (in terms of Cp values) for the Target 1 PC is 30.73-35.23, and for Target 2 PC is 31.18-35.68.

1. Each JBAIDS run performed in the LoD study included Positive Control (PC) and Negative Control (NC) reactions. Of the 93 runs performed in the study, 91 $( 9 7 . 8 \% )$ had successful PC reactions. The mean PC Cp value for Target 1 was $3 3 . 3 8 \pm 0 . 3 6$ (1 $. 0 8 \%$ CV) and for Target 2 was $3 4 . 1 8 \pm 0 . 4 2$ ( $1 . 2 3 \%$ CV).   
2. Positive Control (PC) and Negative Control (NC) reactions for both Target 1 and Target 2 were also included in all 102 JBAIDS runs performed in the reproducibility study. Overall, 99 out of 102 $( 9 7 . 1 \% )$ runs had successful PCs and for both target assays. The mean PC Cp value for Target 1 was 32.52 $\pm 0 . 3 5$ ( $1 . 0 8 \%$ CV) and for Target 2 was $3 2 . 9 5 \pm 0 . 4 4$ ( $1 . 3 4 \%$ CV).   
3. Positive Control (PC) and Negative Control (NC) reactions for both Target 1 and Target 2 assays were included in every JBAIDS run during the surrogate sample testing at the Midwest Research Institute (MRI) to estimate clinical sensitivity. All PC and NC reactions were successful in all runs performed for this study. The mean PC Cp value for Target 1 was $3 4 . 3 8 \pm 0 . 7 4$ $2 . 1 5 \ \% \mathrm { C V } )$ and for Target 2 was $3 5 . 5 8 \pm 0 . 9 2$ $2 . 5 9 \%$ CV).   
4. Positive Control (PC) and Negative Control (NC) reactions for both Target 1 and Target 2 assays were also included in every JBAIDS run during the clinical study. Overall, 511 out of 519 $( 9 8 . 5 \% )$ runs had successful PCs for both target assays. The mean PC Cp value for Target 1 was $3 3 . 1 4 \pm 0 . 6 3$ (1.90 $\%$ CV) and for Target 2 was $3 3 . 8 5 \pm 1 . 2 1$ $3 . 5 7 \%$ CV).

# Sample Control (SC)

The Sample Control assay detects the human RNase P gene. This assay is designed to guard against false negative results caused by an improperly collected specimen, ineffective purification of nucleic acids and or inhibition of the PCR reaction. A properly collected throat or NPS specimen contains human cells from which the RNase P target is recovered during sample purification. Following purification, each sample is then tested with the two Influenza A/H5 target assays, as well as, the Sample Control assay. If extraction and amplification were successful, then the Sample Control Assay will give the expected positive test result for both capillaries. The Sample Control is considered to be successful only if both Sample Control capillaries are positive. The JBAIDS software automatically assigned a result of sample control failure when 1) the Sample Control is unsuccessful and 2) the target assay is negative (or uncertain). If the target assay is positive, then the sample is positive regardless of the result of the Sample Control assay. Because the results of the two Influenza A/H5 target assays are interpreted separately, it is possible to have a sample control failure for one of the target assays and a positive result of the other target.

a. A Sample Control assay (human RNase P) was tested with each purified sample for the refined LoD estimation and the LoD confirmation study. Of 328 Sample Controls, only 11 $( 3 . 4 \% )$ tests did not produce a positive result (i.e. were negative or uncertain):

Summary of Sample Control Results (LoD Study)   

<table><tr><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1>TotalSampleControls</td><td rowspan=1 colspan=1>PositiveSampleControls</td><td rowspan=1 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>87 (93.5%)</td><td rowspan=1 colspan=1>31.44*</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>6.42</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>73 (94.8%)</td><td rowspan=1 colspan=1>29.76</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>6.89</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>83 (98.8%)</td><td rowspan=1 colspan=1>31.71</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>5.27</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>74 (100%)</td><td rowspan=1 colspan=1>29.12</td><td rowspan=1 colspan=1>2.24</td><td rowspan=1 colspan=1>7.69</td></tr></table>

\*One or more capillaries received a ${ > } 4 0$ call and were assigned a Cp of 42.5

The Sample Control assay produced Cps having a standard deviation of approximately 2 cycles across the sample set. This level of variability was within expectations, as samples were prepared with individually collected throat swab and nasopharyngeal swab specimens in the LoD study.

b. The Sample Control assay (human RNase P assay) was also tested with each purified sample to detect inhibition, poor sample extraction, or poor specimen collection in the reproducibility study. A total of 364 out of 367 $( 9 9 . 2 \% )$ Sample Control reactions produced the expected positive results in the reproducibility study:

Summary of Sample Control Results (Reproducibility Study)   

<table><tr><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1>TotalSampleControls</td><td rowspan=1 colspan=1>PositiveSampleControls</td><td rowspan=1 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>92 (97.9%)</td><td rowspan=1 colspan=1>30.11</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>4.78</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>92 (100%)</td><td rowspan=1 colspan=1>29.04</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>2.93</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>89 (100%)</td><td rowspan=1 colspan=1>31.40</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.32</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>91 (98.9%)</td><td rowspan=1 colspan=1>31.04</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>3.48</td></tr></table>

The performance of the Sample Control for both sample types and purification kits was similar. Because the reproducibility study used specimen pools instead of individually prepared specimens, the observed variability of the Cp values for the Sample Controls in this study are slightly less (standard deviation ranging from 0.73-1.44) than that seen in other studies that used individually prepared specimens.

c. A Sample Control assay (human RNase P assay) was tested with each purified sample to detect inhibition or poor sample extraction during the surrogate sample testing at the Midwest Research Institute (MRI) to estimate clinical sensitivity. All samples produced positive Sample Control results:

# Summary of Sample Control Results (Surrogate Sample Testing at the MRI)

<table><tr><td rowspan=2 colspan=1>Kit</td><td rowspan=2 colspan=1>TotalSampleControls</td><td rowspan=2 colspan=1>PositiveSampleControls</td><td rowspan=2 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>32.79</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>4.09</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>30.81</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>6.39</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>53a</td><td rowspan=1 colspan=1>53 (100%)</td><td rowspan=1 colspan=1>33.46</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.66</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>31.80</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>6.79</td></tr></table>

a Total number including retests.

d. A Sample Control assay (human RNase $\mathrm { P }$ assay) was also tested with each purified sample to detect inhibition or poor sample extraction during the clinical study:

Summary of Sample Control Results (Clinical Study)   

<table><tr><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1>TotalSampleControls</td><td rowspan=1 colspan=1>PositiveSampleControls</td><td rowspan=1 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>326 (97.9%)</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>9.47</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>327</td><td rowspan=1 colspan=1>325(99.4%)</td><td rowspan=1 colspan=1>29.09</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>12.10</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>303 (100%)</td><td rowspan=1 colspan=1>30.77</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>7.44</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>357 (98.9%)</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>3.42</td><td rowspan=1 colspan=1>11.24</td></tr></table>

# Extraction Control (EC)

Positive Extraction Control (PEC) was not provided with the kit, and inclusion of a PEC with each batch of extracted samples is not a standard part of the testing protocol recommended in the product package insert. However, during the clinical study, a PEC and a NEC were purified with each batch of clinical samples. Inclusion of a NEC with each batch of purified samples is a standard part of the testing protocol recommended in the product package insert. The PEC was composed of pooled TS specimens spiked with inactive influenza A/H5 virus (Asian lineage) at approximately 2X LoD. The NEC consisted of nuclease-free water. When the expected results were not obtained, the failed PEC or NEC was retested. If the retest provided the expected result, then the control was considered successful and the associated samples were considered to have valid results. If the retest failed, then the results for the samples extracted with that PEC or NEC were considered invalid. Of a total of 172 PEC and NEC tested during the clinical trial, 164 of PEC $( 9 5 . 3 \% )$ and 167 of NEC controls $( 9 7 . 1 \% )$ gave correct results initially. The mean $\mathrm { C p }$ of the PEC tested for Target 1 was 31.08, with 1.85 Standard Deviation (SD) and $5 . 9 5 \%$ CV. The mean $\mathrm { C p }$ of the PEC tested for Target 2 was 34.46, with 1.62 Standard Deviation (SD) and $4 . 7 0 \%$ CV.

# Specimen Stability

The sponsor recommends that TS and NPS specimens be collected using Dacron or other synthetic swabs and placed into viral transport media (VTM). Once collected, specimens may be tested immediately after collection, transported to an off-site testing location, or temporarily stored until testing can be scheduled. Inappropriate handling or storage may compromise the integrity of the specimens and cause inaccurate test results. An analytical study determining the appropriate storage times and temperatures for both unprocessed specimens (e.g., the TS and NPS specimens) and processed samples (e.g., purified nucleic acid from TS and NPS specimens) was carried out. The specimen transport and storage conditions that were evaluated are based on common clinical laboratory workflows.

Results from a pilot study assessing the stability of stored specimens spiked with inactivated Flu A/H5 virus suggested that unprocessed specimens stored under the conditions evaluated in this study continued to give accurate test results.

For the unprocessed specimen transport and storage evaluation, both negative (i.e., matrix blank) and positive (i.e., spiked with reassortant Flu A/H5 virus at the established LoD concentration) specimens were assessed. Eight positive (two full JBAIDS runs) and four negative (one full JBAIDS run) specimens were processed at each time point and storage condition with each purification kit (VIBE and Platinum Path) prior to testing with the freeze-dried reagents. Both matrix types (TS and NPS) were evaluated independently.

For the purified sample transport and storage evaluation, eight positive samples and four negative samples that had been processed with each purification method (VIBE and Platinum Path) were tested immediately and also aliquoted for storage under various conditions prior to retesting with the freeze-dried reagents.

Testing of positive specimens was used to determine whether sample storage can cause false negative results due to degradation of the virus or the purified nucleic acid. Although an unlikely result, testing known negative specimens evaluated whether sample storage can cause false positive test results. Testing of negative specimens also allowed for an evaluation of the effect of storage on the result of the Sample Control assay. The storage conditions evaluated are summarized in the following table:

Storage Conditions Evaluated for Each Sample Matrix/Purification Kit Combination   

<table><tr><td>Unprocessed specimens</td><td>Purified samples</td></tr><tr><td>Day 0, no storage</td><td>Day 0, no storage</td></tr><tr><td>4 Hours, room temperature (18-30°C)</td><td>4 Hours, room temperature (18-30°C)</td></tr><tr><td>Day 3, refrigerated (2-8°C)</td><td>Day 1, refrigerated (2-8°C)</td></tr><tr><td>Day 30, frozen (&lt; -15°C)</td><td>Day 30 , frozen (&lt; -15°C)</td></tr><tr><td>Extended frozen (&lt; -15°C)</td><td></td></tr></table>

The average crossing point $\textstyle ( \mathrm { C p } )$ and relative maximum fluorescence (Fmax) values obtained when the stored samples were tested were compared to the Day 0 results. The Cp value depends on two factors: 1) the number of target copies in the reaction and 2) the efficiency of the PCR reaction. A significant increase ${ > } 3$ cycles) in the average Cp value for a specific time point was suggestive of either degradation of the template during storage or a reduction in the PCR efficiency (possibly due to the presence of inhibitors). Significantly reduced Fmax values are usually associated with flattening of the amplification curve and are an indication of lower PCR efficiency. A significant decrease $( > 5 0 \% )$ in the average Fmax for a sample set at a specific time point (e.g., Day 30) compared to the baseline (Day 0) average is suggestive of reduced test system performance.

The results from this evaluation study demonstrated that accurate test results were obtained when specimens and samples are handled according to the following conditions:

# Proper Storage of Specimens and Samples

<table><tr><td rowspan=1 colspan=1>Sample Matrix</td><td rowspan=1 colspan=1>Unprocessed specimens</td><td rowspan=1 colspan=1>Purified samples</td></tr><tr><td rowspan=1 colspan=1>Throat Swab</td><td rowspan=1 colspan=1>4 hours at room temperature (1828 °C)3 days at refrigerator temperatures (28 C)30 days in the freezer (&lt; -15°C)</td><td rowspan=1 colspan=1>4 hours at room temperature (1828 °C)1 day at refrigerator temperatures (28 °C)30 days in the freezer (&lt; -15°C)</td></tr><tr><td rowspan=1 colspan=1>NasopharyngealSwab</td><td rowspan=1 colspan=1>4 hours at room temperature (1828 °C)3 days at refrigerator temperatures (28 C)30 days in the freezer (&lt; -15°C)</td><td rowspan=1 colspan=1>4 hours at room temperature (1828 °C)1 day at refrigerator temperatures (28 °C)30 days in the freezer (&lt; -15°C)</td></tr></table>

In particular, these conditions are applicable to TS and NPS specimens collected in viral transport media and purified nucleic acid samples generated from the processing of these specimens using the IT 1-2-3 VIBE and IT 1-2-3 Platinum Path sample purification kits.

# d. Detection limit:

The limit of detection (LoD) for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit when testing individually collected throat swab (TS) and nasopharyngeal swab (NPS) specimens (not pooled) was determined using TS and NPS specimens spiked with serial dilutions of quantified live Influenza A/H5N1 reassortant viruses (Vietnam/1203/04 x PR8 and Anhui $/ 0 1 / 0 5 \mathrm { { x } }$ PR8). The LoD is 24 defined as the lowest concentration of virus per mL of specimen (expressed as $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ ) that was detected by both A/H5 (Asian lineage) assays (Target 1 and Target 2) $9 5 \%$ (19/20) of the time.

The LoD was initially estimated by spiking TS specimens with serial dilutions of quantified live Influenza A/H5N1 reassortant viruses (Vietnam/1203/04 x PR8 and Anhui $/ 0 1 / 0 5 \mathrm { \Delta x }$ PR8). The spiked specimens were purified using either the IT 1-2- $3 ^ { \mathrm { T M } }$ VIBE or IT $1 - 2 { - } 3 ^ { \mathrm { { T M } } }$ Platinum Path purification kit and then tested. The LoD was estimated to be the lowest concentration of organism for which all or most replicates tested positive with both of the Influenza A/H5 (Asian lineage) target assays (Target 1 and Target 2). Based upon the results of the initial estimate, refined dilution series were prepared and tested with both sample matrices (TS and NPS) and with all three detection kit assays (Target 1, Target 2, and the Sample Control assay). Results from the refined dilution series were used to estimate the LoD. Once determined, the estimated LoD was confirmed by testing 20 independent TS samples and 20 independent NPS samples spiked at the specified LoD level using both of the sample purification kits. Each LoD confirmation run included an unspiked matrix blank (MB) that served as an external negative control for each batch of purified specimens. The LoD was independently established using two different reassortant virus strains (Vietnam/1203/04 x PR8 and Anhui $/ 0 1 / 0 5 \mathrm { \Delta x }$ PR8).

The initial estimates of the Influenza A/H5N1 reassortant viruses were between 10 and $1 0 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Anhui/01/05 x PR8 strain and 10 and 1 $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Vietnam $/ 1 2 0 3 / 0 4 \mathrm { x }$ PR8 strain. Confirmation testing for the Anhui $/ 0 1 / 0 5 \mathrm { { x } }$ PR8 strain was initiated at $1 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . Due to a large number of indeterminate results (i.e., positive for only one of the target assays), the estimated LoD level was increased to $5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . At $5 0 ~ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ , all 20 individually spiked TS and NPS samples purified with both the IT 1-2-3 VIBE and IT 1-2-3 Platinum Path sample purification kits gave positive results for both assays and both viral strains (Vietnam/1203/04 x PR8 and Anhui $_ { 0 1 / 0 5 \mathrm { ~ x ~ } }$ PR8).

In conclusion, for TS and NPS specimens containing live Influenza A/H5N1, the confirmed JBAIDS Influenza $\mathrm { A } / \mathrm { H } 5$ (Asian lineage) Detection Kit system LoD is $5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ , which was detected in 80 out of 80 specimens ( $2 0 \mathrm { T S }$ samples purified with both kits and $2 0 \mathrm { N P S }$ samples purified with both kits) for each strain of Influenza A/H5N1:

Summary of LoD Confirmation Testing at $\mathbf { 5 0 ~ E I D _ { 5 0 } / m L }$   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">PurificationKit(Matrix)</td><td colspan="2" rowspan="1">Target 1</td><td colspan="2" rowspan="1">Target 2</td><td colspan="2" rowspan="1">Sample Control</td><td colspan="1" rowspan="2">FinalSampleResult</td></tr><tr><td colspan="1" rowspan="1">Positive Total</td><td colspan="1" rowspan="1">Avg. Cp(SD)</td><td colspan="1" rowspan="1">PositiveTotal</td><td colspan="1" rowspan="1">Avg. Cp(SD)</td><td colspan="1" rowspan="1">PositiveTotal</td><td colspan="1" rowspan="1">Avg.. Cp(SD)</td></tr><tr><td colspan="1" rowspan="2">Vietnam/</td><td colspan="1" rowspan="1">PlatinumPath (TS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.87(1.29)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.56(0.58)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.85(1.53)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">VIBE(TS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.76(0.87)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.74(0.81)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">32.33(1.64)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">PlatinumPath (NPS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">30.84(0.87)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.54(0.50)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">29.60(1.58)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">VIBE(NPS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.45(1.35)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.54(1.27)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">29.19(2.40)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="4">Anhui/01/05 xPR8</td><td colspan="1" rowspan="1">PlatinumPath (TS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">36.64(3.05)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.05(1.44)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">30.68(1.64)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">VIBE(TS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.41(1.41)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.23(1.19)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">31.06(1.35)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath (NPS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">36.20(0.86)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.08(0.71)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">28.70(1.33)</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">VIBE(NPS)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">34.46(0.52)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">33.01(0.72)</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">28.78(2.23)</td><td colspan="1" rowspan="1">20/20</td></tr></table>

# e. Analytical Reactivity:

An analytical reactivity study was carried out at the Midwest Research Institute (MRI) using surrogate samples. The avian influenza A viruses are divided between high pathogenicity (HPAI) strains and low pathogenicity (LPAI) strains based on their characteristic pathogenesis and genetic patterns. LPAI strains generally cause only mild disease in birds, while HPAI strains are associated with high morbidity and mortality. HPAI strains of the subtypes H5 and H7 (e.g. H5N1, H7N7, and H7N3) have also been associated with human infections, causing a wide range of disease severity from mild (H7N3 and H7N7) to sometimes fatal (H5N1 and H7N7). The HPAI H5N1 subtype, commonly referred to as the ‘Asian lineage’, has caused human infections in Africa, Asia, the Near East, the Pacific, and Europe. The majority of cases have been reported by four countries: Indonesia, Vietnam, Egypt, and Thailand. Within this subtype, the viruses are further subdivided into clades (e.g. Clade 1, Clade 2 and subclades). The subclades 2.1, 2.2, and 2.3 are among those that have been isolated from human infections.

In this analytical reactivity study, the following eight strains of HPAI/H5N1 viruses were tested:

Analytical Reactivity (Inclusivity) Panel of Live Influenza A/H5N1 (HPAI) Viruses   

<table><tr><td rowspan=1 colspan=1>H5N1 Virus</td><td rowspan=1 colspan=1>Clade</td></tr><tr><td rowspan=1 colspan=1>Avian precursor Yunan (A/Chicken/Yunnan/1251/03)</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>A/Vietnam/1203/2004(H5N1)-PR8/CDC-RG</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Avian precursor Hunan (A/Duck/Hunan/795/02)</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Korea/IS/06</td><td rowspan=1 colspan=1>2.2</td></tr><tr><td rowspan=1 colspan=1>Scaly Breasted Munia/Hong Kong/45/2006</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Japanese white eye/Hong Kong/1038/2006</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>Common Magpie/Hong Kong/645/2006</td><td rowspan=1 colspan=1>2.3</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG</td><td rowspan=1 colspan=1>2.3</td></tr></table>

Personnel at MRI grew virus cultures and quantification was performed by determining tissue culture infectious doses in accordance with a procedure approved by the CDC.

In accordance with the CDC approved protocol, the $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ quantification was confirmed by spiking a contrived sample type (A549 cells in F12K media) with designated levels of virus. The prepared samples were then purified using the Qiagen Viral RNA mini kit followed by testing with the appropriate FDA cleared CDC assays (FluA/H5a, FluA/H5b and the Sample Control). The results obtained were compared to previous lot quantification and determined to be acceptable.

Per protocol, the same prepared samples using a contrived sample matrix of A549 cells in F12K medium were also tested by the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. Poor detection of the viruses was observed at concentrations as high as $1 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for some of the virus strains. An investigation revealed that the F12K medium was causing inhibition of the system. Since this contrived sample matrix was not representative of the intended sample matrix, it was determined that TS in VTM would be used as the sample matrix for this study.

Using the same qualified viral stocks, MRI prepared spiked simulated specimens (TS in VTM) at concentrations of 10, 100, and $1 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for each of the eight inclusivity panel viruses. The spiked specimens were purified using the IT 1- 2-3 Platinum Path purification kit prior to testing with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit freeze-dried reagents on the JBAIDS instrument.

In addition, performance of the two reference strains   
(A/Vietnam/1203/2004(H5N1)-PR8/CDC-RG and A/Anhui/01/2005(H5N1)-PR8- IBCDC-RG), tested in this study in units of $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , were compared to the results obtained in the LoD study to determine a $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ level that gave similar performance to established system LoD $( 5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L } )$ .

# Analytical Reactivity (Inclusivity) Panel Testing using the FDA-cleared CDC rRT-PCR Flu Panel:

Eight live Influenza A/H5N1 (HPAI) viruses were spiked into a contrived matrix sample (A549 cells and virus in F12K medium) at concentrations of 10, 100, 1,000 and $1 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Sample purification and testing was performed according the CDC assay protocol. As per the CDC protocol, amplification curves are positive only when there is an amplification curve with a $\mathrm { C t }$ value ${ < } 3 7$ . The overall test for each sample is positive when both target assays are positive, negative when both target assays are negative and the sample control is positive and inconclusive when the results of the two target assays are discordant.

All viral strains gave positive test results at concentrations of 10,000 and 1,000 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Four of the strains (A/Chicken/Yunnan/1251/2003 (H5N1), A/JapaneseWhiteEye/HongKong/1038/2006, A/ScalyBreastedMunia/HongKong/45/2006, and A/Anhui/01/2005 (H5N1)-PR8- IBCDC-RG) gave positive results at $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and no strain was positive at $1 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All results are shown in the following table:

Influenza A/H5N1 (HPAI) Strains Tested with the CDC rRT-PCR Flu Panel Assays   

<table><tr><td rowspan=2 colspan=1>Virus(Lowest Concentration Detected)</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>H5a - Target 1</td><td rowspan=1 colspan=2>H5b - Target 2</td><td rowspan=1 colspan=2>RNaseP- SampleControl</td><td rowspan=2 colspan=1>OverallResult</td></tr><tr><td rowspan=1 colspan=1>Ct Value</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>CtValue</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>CtValue</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=4 colspan=1>A/Chicken/Yunnan/1251/2003 (H5N1)(100 TCID50/mL)</td><td rowspan=1 colspan=1>10.000</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,0000</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>no detect</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/JapaneseWhiteEye/HongKong/1038/2006HSN1(100 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/CommonMagpie/HongKong/645/2006HSN1)(1,000 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Inc</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/Duck/Hunan/795/2002 (H5N1)(1,000 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>nc</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>no detect</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/Chicken/Korea/IS/2006 (H5N1)(1,000 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Inc</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/ScalyBreastedMunia/HongKong/45/2006(H5N1)(100 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>A/Anhui/01/2005 (H5N1)-PR8-IBCDC-RG(100 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>no detect</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>VNH5N1-PR8/CDC-RG(1,000 TCID50/mL)</td><td rowspan=1 colspan=1>10,000</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Inc</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr></table>

Abbreviations are as follows: Pos, positive; Neg, negative; no detect, no detection; Inc, inconclusive; $\mathrm { T C I D 5 0 / m l }$ , $50 \%$ tissue culture infectious dose per mL; and Ct, cycle threshold.

# Analytical Reactivity (Inclusivity) Panel Testing Using the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit:

Eight Influenza A/ H5N1 (HPAI) viruses were spiked into TS specimens at 10, 100 and $1 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and extracted using the Platinum Path purification kit. All eight viruses produced positive results (detection with both Target assays) when spiked at 100 and 1,000 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . At 1 $0 \mathrm { \ T C I D } _ { 5 0 } / \mathrm { m L }$ two strains (A/Japanese white eye/Hong $\mathrm { K o n g } / 1 0 3 8 / 2 0 0 6$ and A/Anhui/01/2005 (H5N1)- PR8-IBCDC-RG ) were positive, three strains (A/Common Magpie/Hong Kong/645/2006, A/Chicken Korea/IS/2006, and A/Scaly Breasted Munia/Hong Kong/45/2006 ) were negative and three strains (VNH5N1-PR8/CDC-RG, A/Chicken/Yunnan/1251/03, and A/Duck/Hunan/795/2002) gave inconclusive test results.

For those samples that initially tested uncertain (typically one positive and one negative capillary) the extracted sample was retested. Results remained the same for both targets and the sample control, retest values are in italics. All results are shown in the table below. Unless otherwise noted, the Cp value for each test result is an average of the two capillaries associated with this test result. When only one capillary gives a positive result, the test result is uncertain. In those cases, the Cp value is for only one capillary. As opposed to the CDC test system, the Cp value is not used to determine the test result. Instead, the Detector module of the JBAIDS software evaluates each curve using several parameters (e.g., curve shape, signal to noise) to determine if it should be called positive or negative. Only curves with positive (or uncertain) results are assigned $\mathrm { C p }$ values.

Influenza A/H5N1 (HPAI) Strains Tested with the JBAIDS A/H5 (Asian lineage) Detection Kit   

<table><tr><td rowspan=2 colspan=1>Virus(Lowest Concentration Detected)</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>Target 1</td><td rowspan=1 colspan=2>Target 2</td><td rowspan=1 colspan=2>SampleControl</td><td rowspan=2 colspan=1>OverallResult</td></tr><tr><td rowspan=1 colspan=1>CpValue</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>CpValue</td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>CpValue</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=3 colspan=1>A/Chicken/Yunnan/1251/2003 (H5N1)(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>31.16</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.95</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.29</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.47</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>33.72</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>36.44 b42.50 ab</td><td rowspan=1 colspan=1>UncUnc</td><td rowspan=1 colspan=1>36.76b37.12b</td><td rowspan=1 colspan=1>UncUnc</td><td rowspan=1 colspan=1>30.4430.43</td><td rowspan=1 colspan=1>PosPos</td><td rowspan=1 colspan=1>Inc</td></tr><tr><td rowspan=3 colspan=1>A/Japanese white eye/Hong Kong/1038/2006HSN(10 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>30.30</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.37</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.80</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.13</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.63</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>37.42</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>37.86</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.91</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=3 colspan=1>A/Common Magpie/Hong Kong/645/20065N1(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>37.08</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.19</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.79</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>41.11a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>36.12</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.95</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>32.02</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>A/Duck/Hunan/795/2002 (H5N1)(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>33.37</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.32</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.56</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>35.45</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.62</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.90</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>42.50 a.b42.50 a.b</td><td rowspan=1 colspan=1>UncUnc</td><td rowspan=1 colspan=1>N/AN/A</td><td rowspan=1 colspan=1>NegNeg</td><td rowspan=1 colspan=1>32.6132.333</td><td rowspan=1 colspan=1>Posos</td><td rowspan=1 colspan=1>Inc</td></tr><tr><td rowspan=3 colspan=1>A/Chicken Korea/IS/2006 (H5N1)(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>33.28</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>34.50</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30.92</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>37.18</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>35.93</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>29.84</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>31.67</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>A/Scaly Breasted Munia/Hong Kong/45/2006H N(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>36.04</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>33.57</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.69</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>42.50 a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>39.77</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.78</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>31.81</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>A/Anhui/01/2005 (H5N1)-PR8-IBCDC-RG(10 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>35.47</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>34.42</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.80</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>40.45 a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>36.07</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.76</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>42.50 a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>36.52</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>31.58</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=3 colspan=1>VNH5N1-PR8/CDC-RG(100 TCID50)</td><td rowspan=1 colspan=1>1,000</td><td rowspan=1 colspan=1>31.23</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>33.88</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>34.78</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>39.11a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>32.34</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Pos</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>39.47 a</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>31.98</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Inc</td></tr></table>

Abbreviations are as follows: Pos, positive; Neg, negative; Unc, uncertain; Inc, inconclusive, N/A, not applicable; $\mathrm { T C I D 5 0 / m l }$ , $50 \%$ tissue culture infectious dose per mL; and $\mathrm { C p }$ , crossing point. a One or more of the replicate $\mathrm { C p }$ values was ${ > } 4 0$ and is reported as 42.5. b The $\mathrm { C p }$ represent results from only one capillary.

# Comparison and Correlation of $\bf { E I D _ { 5 0 } / m L }$ to $\bf { T C I D _ { 5 0 } / m L }$ based on Cp Values

The LoD study with Vietnam $Y 1 2 0 3 / 0 4 \mathrm { x P R }$ 8 and Anhui $1 0 1 / 0 5 \mathrm { x }$ PR8 H5N1 strains determined that the LoD of the JBAIDS system to be $5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . This was based on the lowest concentration of virus that was successfully detected $9 5 \%$ (19/20) of the time for both targets and both strains. The Cp values obtained at LoD for virus spiked into TS in VTM and extracted using Platinum Path were as follows: Target $1 ~ { \mathrm { C p } } { = } 3 6 . 6 4 { \pm } 3 . 0 5$ and Target $2 { \mathrm { C p } } { = } 3 4 . 0 5 { \pm } 1 . 4 4$ for Anhui $1 0 1 / 0 5 \mathrm { x }$ PR8 and Target 1 $\mathrm { C p } { = } 3 1 . 8 7 { \pm } 1 . 2 9$ and Target $\scriptstyle 2 \ C \ p = 3 3 . 5 6 \pm 0 . 5 8$ for Vietnam $/ 1 2 0 3 / 0 4 \mathrm { x }$ PR8. As shown in the table above, the Cps obtained for the VNH5N1-PR8/CDC-RG strain tested at $1 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , Target $1 = 3 1 . 2 3$ and Target $2 = 3 3 . 8 8$ , were very similar to those obtained at a LoD of $5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . When evaluating the data reported above for the A/Anhui/01/2005 (H5N1)-PR8-IBCDC-RG strain, the Cps obtained for both targets at $1 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } ,$ , Target $1 = 3 5 . 4 7$ and Target $2 =$ 34.42, were within one standard deviation of those obtained at the LoD of $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . This data demonstrates that the performance of the JBAIDS Influenza A/H5 Detection kit is roughly similar for sample spiked at roughly 500 to 1,000 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ of these live virus strains when compared to $5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ of the reassortant viral stocks used to establish the test system LoD.

In conclusion, the JBAIDS Influenza A/H5 (Asian lineage) Detection System detected all eight FluA/H5N1 (HPAI) virus strains included in the inclusivity panel. All strains gave positive results (detected by both target assays) at 1,000 and 100 $\mathbf { T C I D _ { 5 0 } / m L }$ . At 1 $\bf { 0 . T C I D _ { 5 0 } / m L }$ , two strains have positive result, 3 were negative and 3 gave inconclusive test results. These results are equivalent to those obtained with the FDAcleared CDC assays.

In addition, while testing with the JBAIDS system detected both target 1 and 2 for all strains at $\mathbf { 1 0 0 \ T C I D _ { 5 0 } / m L }$ , comparison to $\mathbf { C p }$ values at LoD (reported in $\bf { E I D _ { 5 0 } / m L }$ ) indicated that some viruses were at the edge of detection and therefore, the $\bf { T C I D _ { 5 0 } / m L }$ LoD was correlated (based on Cp) to the $\bf { E I D _ { 5 0 } / m L }$ LoD and determined to be roughly between 500 to 1,000 $\mathrm { { T C I D } _ { 5 0 } / m L }$ .

f. Analytical Specificity/Cross-reactivity Evaluation:

The analytical specificity study involved testing a defined panel of viruses/organisms that may be encountered in specimens collected for FluA/H5 (Asian lineage) diagnostic testing but that should not be detected by the assay by design. The organisms/viruses making up the exclusivity panel were grown, identity confirmed, and titered/quantified. Each organism/virus was tested once at a high concentration; bacteria/fungi were tested as close to $1 0 ^ { 6 } \mathrm { C F U / m L }$ as possible, while viruses were tested as close to $1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $1 0 ^ { 5 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ as possible depending on the stock concentrations. The organisms/viruses stocks were directly spiked into individually collected throat swab specimens in VTM (not pooled) and were not diluted or manipulated in any way prior to spiking. The spiked specimens were then processed using the IT 1-2-3 VIBE Sample Purification Kit prior to

testing with the JBAIDS Influenza A/H5 Detection Kit assays. (Note: The VIBE and Platinum Path kit were demonstrated to produce nearly identical results with the VIBE kit occasionally appearing to be slightly more efficient at nucleic acid extraction based on the LoD study).

A spiked positive control sample was included with each batch of organisms purified with the VIBE kit. This spiked positive control material consisted of pooled throat swab specimens spiked with inactivated influenza A/H5 virus (Vietnam/1203/04 x PR8) that was aliquoted and frozen. This is the same material that was used as a spiked positive control in the clinical (specificity) study. Stability of this material stored at $\leq - 1 5 ^ { \circ } \mathrm { C }$ for up to 90 days has been validated in a separate study.

# Exclusivity Testing of Seasonal Influenza Viruses:

Three FluA/H3N2 viruses, five FluA/H1N1 viruses, and five FluB viruses were each individually spiked into throat swab specimens at the highest concentration possible using $1 0 0 \mu \mathrm { L }$ of the Zeptometrix validated and titered stocks. The final test concentrations ranged from $3 . 8 \mathrm { ~ \dot { ~ } x ~ } 1 0 ^ { 3 }$ to $1 . 7 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { m L }$ . All 13 viruses produced negative results for both FluA/H5 (Asian lineage) target assays and positive results for the Sample Control assay:

Seasonal influenza Strains Tested with the JBAIDS A/H5 (Asian lineage) Detection Kit   

<table><tr><td>Organism/virus</td><td>ATCC# or Identifier</td><td>Relevance</td><td>Concentration Tested</td><td>JBAIDS Assay Result Target 1</td><td>JBAIDS Assay Result Target 2</td><td>Sample Control average CP</td></tr><tr><td>Influenza A/H3N2</td><td>A/Wisconsin/67/2005</td><td>Seasonal Flu A</td><td>8.17 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>34.63</td></tr><tr><td>Influenza A/H3N2</td><td>A/Victoria/3/75</td><td>Seasonal Flu A</td><td>3.80 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>30.47</td></tr><tr><td>Influenza A/H3N2</td><td>A/Port Chalmers/1/73</td><td>Seasonal Flu A</td><td>5.67 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>31.75</td></tr><tr><td>Influenza A/H1N1</td><td>A/Brisbane/59/07</td><td>Seasonal Flu A</td><td>8.17 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>37.46</td></tr><tr><td>Influenza A/H1N1</td><td>A/PR/8/34</td><td>Seasonal Flu A</td><td>1.71 x 106 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>30.70</td></tr><tr><td>Influenza A/H1N1</td><td>A/Denver/1/57</td><td>Seasonal Flu A</td><td>3.00 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>35.95</td></tr><tr><td>Influenza A/H1N1</td><td>A/FM/1/47</td><td>Seasonal Flu A</td><td>4.71 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>30.86</td></tr><tr><td>Influenza A/H1N1</td><td>A/NWS/33</td><td>Seasonal Flu A</td><td>4.70 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>29.66</td></tr><tr><td>Influenza B</td><td>B/FL/04/06</td><td>Seasonal Flu B</td><td>1.67 x 104 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>28.61</td></tr><tr><td>Influenza B</td><td>B/Lee/40</td><td>Seasonal Flu B</td><td>8.17 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>33.09</td></tr><tr><td>Influenza B</td><td>B/Taiwan/2/62</td><td>Seasonal Flu B</td><td>5.03 x 104 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>30.49</td></tr><tr><td>Influenza B</td><td>B/GL/1739/54</td><td>Seasonal Flu B</td><td>8.17 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>33.73</td></tr><tr><td>Influenza B</td><td>B/Maryland/1/59</td><td>Seasonal Flu B</td><td>8.18 x 103 (TCID50/mL)</td><td>Negative</td><td>Negative</td><td>30.41</td></tr></table>

# Exclusivity Testing of Non-H5 HA Type Influenza Strains That Are Known to Infect Human Either Sporadically or Epidemically, Low Pathogenicity Avian Influenza (LPAI) H5N1 Strains, and A/H5 Non-Asian Lineage Strains:

Four avian influenza A/H5 virus strains of North American lineage and five non-H5 influenza A strains that represent a broad range of avian strains with potential for human infection were tested to demonstrate the specificity of the JBAIDS A/H5 (Asian lineage) Detection Kit. These virus strains were each individually spiked into throat swab specimens at the highest concentration possible using $1 0 0 \mu \mathrm { L }$ of the validated and titered stocks. All nine viruses produced negative results for both FluA/H5 (Asian lineage) target assays and positive results for the Sample Control assay:

Non-H5 HA Type Influenza Strains that are Known to Infect Human either Sporadically or Epidemically, Low Pathogenicity Avian Influenza (LPAI) H5N1 Strains, and A/H5 NonAsian Lineage Strains Tested with the JBAIDS A/H5 (Asian lineage) Detection Kit   

<table><tr><td colspan="1" rowspan="1">Organism/virus</td><td colspan="1" rowspan="1">ATCC# or Identifier</td><td colspan="1" rowspan="1">Relevance</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">JBAIDS AssayResult Target 1</td><td colspan="1" rowspan="1">JBAIDS AssayResult Target 2</td><td colspan="1" rowspan="1">SampleControlaverageCP</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3N8</td><td colspan="1" rowspan="1">A/MAL/ALB/16/87</td><td colspan="1" rowspan="1">Non-H5 avianinfluenzastrains withpotential forhumaninfection</td><td colspan="1" rowspan="1">1.80 x 108(EID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">29.44</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H5N1</td><td colspan="1" rowspan="1">A/TY/MA/40550/87 - BEL/42</td><td colspan="1" rowspan="1">LPAI H5N1</td><td colspan="1" rowspan="1">1.04 x 104(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">31.94</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H5N1</td><td colspan="1" rowspan="1">A/DK/PA/4560069 - 9/06</td><td colspan="1" rowspan="1">LPAI H5N1</td><td colspan="1" rowspan="1">1.87 x 105(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30.51</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H5N1</td><td colspan="1" rowspan="1">A/MUTESWAN/MI/451072-2/06</td><td colspan="1" rowspan="1">LPAI H5N1</td><td colspan="1" rowspan="1">7.10 x 104(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30.92</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H5N2</td><td colspan="1" rowspan="1">A/DK/SC/318328 - 3/04</td><td colspan="1" rowspan="1">NorthAmericanlineage A/H5</td><td colspan="1" rowspan="1">3.30 x 103(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">33.14</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H6N2</td><td colspan="1" rowspan="1">A/Chicken/CA/32213 - 1/2000</td><td colspan="1" rowspan="1">Non-H5 avianinfluenzastrains withpotential forhumaninfection</td><td colspan="1" rowspan="1">9.60 x 103(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">31.38</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H7N3</td><td colspan="1" rowspan="1">A/TY/UT/24721 - 10/95</td><td colspan="1" rowspan="1">Non-H5 avianinfluenzastrains withpotential forhumaninfection</td><td colspan="1" rowspan="1">3.30 x 104(TCID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">29.90</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H7N7</td><td colspan="1" rowspan="1">A/Mallard/Netherlands/12/2000IB - CDC - 1</td><td colspan="1" rowspan="1">Non-H5 avianinfluenzastrains withpotential forhumaninfection</td><td colspan="1" rowspan="1">4.06 x 107(EID50/mL)</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">28.83</td></tr><tr><td>Influenza A/H9N2</td><td>A/Turkey/Wisconsin/1966</td><td>Non-H5 avian influenza strains with potential for human infection</td><td>1.02 x 107 (EID50/mL)</td><td>Negative</td><td>Negative</td><td>29.84</td></tr></table>

# Exclusivity Testing of Non-Influenza Respiratory Viruses:

Ten respiratory viruses that cause influenza-like illness symptoms were each individually spiked into throat swab specimens at the highest concentration possible using $1 0 0 \mu \mathrm { L }$ of the Zeptometrix validated and titered stocks. The final test concentrations ranged from $\mathbf { \dot { 3 } } . 3 3 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ to $3 . 4 0 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All ten viruses produced negative results for both FluA/H5 (Asian lineage) target assays and positive results for the Sample Control assay.

Non-influenza Respiratory Viruses Tested with the JBAIDS A/H5 (Asian lineage) Detection Kit   

<table><tr><td rowspan=1 colspan=1>Organism/virus</td><td rowspan=1 colspan=1>ATCC# orIdentifier</td><td rowspan=1 colspan=1>Relevance</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>JBAIDS AssayJBAIDS AssayResult Target Result Target</td><td rowspan=1 colspan=1>JBAIDS AssayJBAIDS AssayResult Target Result Target</td><td rowspan=1 colspan=1>SampleControlaverageCP</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>4.70 x 103(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>34.00</td></tr><tr><td rowspan=1 colspan=1>Adenovirus, type 1</td><td rowspan=1 colspan=1>Ad.71</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>3.40 x 106(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>31.84</td></tr><tr><td rowspan=1 colspan=1>Adenovirus, type 7a</td><td rowspan=1 colspan=1>S-1058</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>1.52 x 105(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30.09</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>7.30 x 104(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>28.93</td></tr><tr><td rowspan=1 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>5.67 x 103(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>29.98</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>1A</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>2.20 x 105(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>28.76</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus,type 2</td><td rowspan=1 colspan=1>PIV-2Patient isolate</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>1.67 x 104(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>33.56</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus,type 3</td><td rowspan=1 colspan=1>PIV-3Patient isolate</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>2.20 x 105(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>30.14</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>1.39 x 104(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>32.14</td></tr><tr><td rowspan=1 colspan=1>Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-8(Peru6-2003</td><td rowspan=1 colspan=1>Influenza-likeIllness</td><td rowspan=1 colspan=1>3.33 x 103(TCID50/mL)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>31.94</td></tr></table>

# Exclusivity Testing of Bacteria and Fungus:

Twelve bacterial strains and one fungus were each individually spiked into throat swab specimens at $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { C F U / m L }$ or higher with the exception of Mycoplasma pneumoniae which was spiked at the highest concentration possible ( $\dot { 1 } . 8 7 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ using $1 0 0 \mu \mathrm { L }$ of the Zeptometrix validated and titered stocks. Eight of the panel members represent bacteria that cause primary respiratory illness or cause secondary infection following viral infection, the remaining four panel members can be encountered as normal flora in the nasopharynx, throat, or other body sites. All twelve organisms produced negative results for both FluA/H5 (Asian lineage) target assays and positive results for the Sample Control assay. One organism, Bordetella pertussis, initially gave an indeterminate result (Target 1 positive, Target 2 uncertain) but was negative when a second sample was prepared and tested. The original purified sample was not retested. It is assumed that the first B. pertussis sample became contaminated with FluA/H5 material from the spiked positive control sample that was purified and tested directly alongside it, especially as the positive capillaries had very low maximum fluorescence values as well as a late Cp value for Target 1.

Bacteria and Fungi Tested with the JBAIDS A/H5 (Asian lineage) Detection Kit   

<table><tr><td colspan="1" rowspan="1">Dactella ald I' al</td><td colspan="1" rowspan="1">Ig: I estea wl</td><td></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cage) beeeeer</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Organism/virus</td><td colspan="1" rowspan="1">ATCC# orIdentifier</td><td colspan="1" rowspan="1">Relevance</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">JBAIDS AssayResult Target 1</td><td colspan="1" rowspan="1">JBAIDS AssayResult Target 2</td><td colspan="1" rowspan="1">SampleControlaverage Cp</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">1.17 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative*(PositiveCp = 36.66Fmax = 1.19)</td><td colspan="1" rowspan="1">Negative*(UncertainCp = 32.21Fmax = 0.96)</td><td colspan="1" rowspan="1">33.39*32.23</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">1.87 x 106TCID50/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30.98</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">Ne 11</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">1.14 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">28.75</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1">Zeptometrixpart#0801519</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">3.50 x 106CF/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">31.59</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">2.77 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">31.86</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">Type 59</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">9.23 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">31.84</td></tr><tr><td colspan="1" rowspan="1">Legionella mcdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">ILI, secondaryinfection</td><td colspan="1" rowspan="1">2.77 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30.60</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">Respiratory infection</td><td colspan="1" rowspan="1">n1.10 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">28.91</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">Related to Normalflora</td><td colspan="1" rowspan="1">7.80 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">36.31</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">Zeptometrixpart#0801510</td><td colspan="1" rowspan="1">Normal flora</td><td colspan="1" rowspan="1">1.33 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">32.34</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">Zeptometrixpart#0801651</td><td colspan="1" rowspan="1">Normal flora</td><td colspan="1" rowspan="1">1.03 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">30.73</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1">Zeptometrixpart#0801512</td><td colspan="1" rowspan="1">Normal flora/throatinfection</td><td colspan="1" rowspan="1">1.26 x 106CFU/mL</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">29.66</td></tr><tr><td>Candida albicans</td><td>Zeptometrix part #0801504</td><td>Normal flora</td><td>1.00 x 106 CFU/mL</td><td>Negative</td><td>Negative</td><td>31.04</td></tr></table>

\* The first Bordetella pertussis spiked sample gave a positive result for Target 1 and an uncertain result for Target 2. To ascertain if these results were due to possible contamination from the neighboring FluA/H5N1- spiked control sample, a new sample was spiked with $B _ { \cdot }$ . pertussis and re-purified/retested. Results of the retest were negative for both FluA/H5 (Asian lineage) target assays.

In conclusion, the JBAIDS Influenza $\mathbf { A } / \mathbf { H } 5$ (Asian lineage) Detection System correctly excluded all 45 organisms/viruses included in the exclusivity panels, including all types and subtypes of non-H5 influenza tested. No cross-reactivity to seasonal influenza strains, non-H5 HA type avian influenza strains that are known to infect human either sporadically or epidemically, low pathogenicity avian influenza (LPAI) H5N1 strains, and A/H5 North American lineage strains, other respiratory pathogens, or normal flora was detected.

# Exclusivity Testing of 2009 H1N1 Influenza:

2009 H1N1 Influenza strains were not included in the analytical crossreactivity study. However, one of the clinical sites in the clinical study (specificity) study, Tripler Army Medical Center (TAMC), Honolulu, Hawaii, tested residual frozen NPS and TS specimens collected when the 2009 H1N1 Influenza was prevalent (June 2009 to Dec. 2009). There was a total of 25 clinical specimens that were confirmed to be positive for 2009 H1N1 Influenza by the CDC’s Swine Flu Panel under an Emergency Use Authorization EUA from the FDA included in the clinical testing using the JBAIDS A/H5 (Asian lineage) Detection Kit at TAMC. All 25 specimens tested negative with the JBAIDS A/H5 (Asian lineage) Detection Kit demonstrating $100 \%$ specificity for the device testing 2009 H1N1 Influenza positive clinical specimens.

# g. Assay cut-off:

Each JBAIDS test is analyzed with the data analysis module. The module is called Detector. The module determines the outcome of the tests conducted on the samples. The module first analyzes the data from each capillary independently. Then the replicates of the samples are analyzed together. Finally the module interprets the outcome of these analyses for the sample and its controls.

The analysis module algorithm consists of two basic parts. The first identifies samples that are obviously positive or negative (Obvious Tests), and second that computes a refined estimate of the samples status when the sample is not called an obvious positive or obvious negative (Expert Test). The algorithms are based on an Expert System approach to determining positive and negative samples, but the approach is not learning after release, and is deterministic meaning that every sample will have the same outcome every time the algorithm evaluates the sample. The purpose of Obvious Tests is to call those samples that are clearly positive or negative. In this way, only non-obvious samples will need to be processed by the more computationally involved detector algorithms. Obvious tests produce internal scores and compare them against fixed thresholds to return a

positive/negative/uncertain call. Expert Tests algorithm is based on mathematically modeling the expected shapes of the amplification curves of positive and negative samples. The model itself is based on tests for nine distinct characteristics in the amplification curves and it assigns nine test scores to each curve. After the tests are scored, the amplification curve is scored as a weighted sum of the tests.

Thresholds for the Obvious Tests and weights in the Expert Tests for Detector were determined using training data sets and numerical optimization to pick parameters that ensure Detector minimizes the number of error observed in real-world data.

Validation data from JBAIDS Customer Validation Testing, and Dugway Operational Assessment were gathered and evaluated with Detector to validate the analysis module algorithm. These data contain samples from five separate data sets: typical data, atypical data, negative dominated, RNA data and Dugway field data. The data were gathered from varied sources, including a variety of assays, instruments and users, to obtain examples of all possible JBAIDS data. There are 28,467 samples in the data sets. $9 9 . 6 \%$ of the samples were correctly called by the Detector. $0 . 1 9 \%$ of the samples were called “Uncertain” and $0 . 2 1 \%$ of the samples were incorrectly called.

# h. Interfering Substances:

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors and technique-specific substances (substances that could be introduced into the PCR reaction as contaminants during sample purification or during reaction setup) affect the performance of the JBAIDS A/H5 (Asian lineage) Detection Kit. For each endogenous, exogenous and technique specific test substance, one TS and/or NPS specimen containing inactivated FluA/H5 virus $( \mathrm { V i e t n a m } / 1 2 0 3 / 0 4 \mathrm { ~ x ~ P R 8 } )$ ) at a concentration equivalent to 3X LoD was spiked with the appropriate amount of test substance. The LoD determined for the live reassortant viruses is $5 0 \ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ . This is approximately equivalent to $7 5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ of the inactivated reassortant virus as determined in the “Template Comparison for the JBAIDS Influenza A/H5 Detection Kit”. Specimens were therefore spiked at $2 , 2 5 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ with the inactivated reassortant virus. In each round of purifications, one specimen was prepared that did not contain a test substance. This sample served as a control to which the other specimens were compared. All specimens (except those containing Platinum Path technique specific substances) were purified with both the IT 1-2-3 VIBE Sample Purification Kit the IT 1-2-3 Platinum Path Sample Purification Kit. Each purified sample was tested with the FluA/H5 (Asian lineage) Target 1 and Target 2 assays and the Sample Control assay.

When possible, levels of tested substances were taken from CLSI document EP7-A, “Interference Testing in Clinical Chemistry: Approved Guideline”. For those substances not listed in document EP7-A, a level was chosen that would be above that expected in an actual collected swab sample. As the technique-specific substances associated with the IT 1-2-3 VIBE kit have been thoroughly assessed in previous FDA submissions (JBAIDS Plague Detection System k072631 and the 36

JBAIDS Tularemia Detection System k072547), this evaluation limited techniquespecific substances to components of the IT 1-2-3 Platinum Path sample purification kit.

# Endogenous Substances

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td><td rowspan=1 colspan=1>Specimen</td></tr><tr><td rowspan=1 colspan=1>EN1</td><td rowspan=1 colspan=1>Blood (with Na Citrate)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>VTM</td><td rowspan=1 colspan=1>TS, NPS</td></tr><tr><td rowspan=1 colspan=1>EN2</td><td rowspan=1 colspan=1>Mucin(bovine submaxillary gland, type I-S)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>VTM</td><td rowspan=1 colspan=1>TS, NPS</td></tr></table>

# Exogenous Substances

<table><tr><td colspan="1" rowspan="1">IS#</td><td colspan="1" rowspan="1">Test Substance</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Solvent</td><td colspan="1" rowspan="1">Specimen</td></tr><tr><td colspan="1" rowspan="1">EX1</td><td colspan="1" rowspan="1">Tobramycin (systemic antibiotic)</td><td colspan="1" rowspan="1">0.6 mg/mL</td><td colspan="1" rowspan="1">Water</td><td colspan="1" rowspan="1">TS, NPS</td></tr><tr><td colspan="1" rowspan="1">EX2</td><td colspan="1" rowspan="1">Antiseptic Mouth RinseActive Ingredients: Eucalyptol 0.092%Menthol 0.042%Methyl Salicylate 0.06% Thymol 0.064%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX3</td><td colspan="1" rowspan="1">Cough Syrup (Night-time)Active Ingredients: Dextromethorphan HBr 1mg/mL Doxylamine succinate 0.42 mg/mLAlcohol 10%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX4</td><td colspan="1" rowspan="1">Cough Syrup (Day-time)Active Ingredients: Acetaminophen 21.67mg/mL Dextromethorphan HBr 0.67 mg/mLPhenylephrine HCl 0.33 mg/mL</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX5</td><td colspan="1" rowspan="1">Sore Throat SprayActive Ingredient: Phenol 1.4%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX6</td><td colspan="1" rowspan="1">Sore Throat + Cough SprayActive Ingredients: Benzocaine 5.0%Dextromethorphan HBr 12.4 mg/mL Glycerin30%</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX7</td><td colspan="1" rowspan="1">Homeopathic Cold + Flu SyrupActive Ingredients in 5 mL dose:Pelargonium sidoides 1X Aconitumnapellus 4X Eucalyptus globules 2XBryonia alba 4X Eupatorium</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX8</td><td colspan="1" rowspan="1">Herbal Cough DropsActive Ingredients:Menthol 4.8 mg/lozenge</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX9</td><td colspan="1" rowspan="1">Cough Drops with Oral AnestheticActive Ingredients:Dyclonine HCl 3.0 mg/lozengeMenthol 6.0 mg/lozenge</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX10</td><td colspan="1" rowspan="1">Smokeless Tobacco</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">TS</td></tr><tr><td colspan="1" rowspan="1">EX11</td><td colspan="1" rowspan="1">Saline Nasal Spray with Preservatives(0.65% NaCl + Phenylcarbinol andBenzalkonium Chloride)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">NPS</td></tr><tr><td colspan="1" rowspan="1">EX12</td><td colspan="1" rowspan="1">Nasal Decongestant SprayActive Ingredient: OxymetazolineHCl 0.05%(also contains Benzalkonium chloride,menthol, eucalyptol, camphor, benzyl alcoholand phosphate buffers)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">VTM</td><td colspan="1" rowspan="1">NPS</td></tr></table>

# Technique Specific Substances

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Specimen</td></tr><tr><td rowspan=1 colspan=1>TS1</td><td rowspan=1 colspan=1>MB Binding Buffer</td><td rowspan=1 colspan=1>10% (v/v)</td><td rowspan=1 colspan=1>TS, NPS</td></tr><tr><td rowspan=1 colspan=1>TS2</td><td rowspan=1 colspan=1>Wash Buffer (from 4thwell)</td><td rowspan=1 colspan=1>10% (v/v)</td><td rowspan=1 colspan=1>TS, NPS</td></tr></table>

Additionally, the effect of different concentrations of human DNA on the assay was also evaluated. A dilution series of commercially-prepared human DNA was prepared and spiked into the reaction vials along with synthetic target RNA at a $3 \mathrm { ~ X ~ }$ nucleic acid LoD concentration. The highest DNA concentration ( $1 8 7 \mathrm { n g / \mu L } )$ tested in this study is considerably higher than what is normally achieved in a purified TS or NPS sample (as assessed by PCR amplification $\mathrm { C p }$ values).

Each of the test substances were assessed separately. Target 1, Target 2, and Sample Control results were used to evaluate the effect that each test substance had on amplification or detection of target sequences. Depending on the concentration, inhibitors can cause false negative test results, delays in Cp values or reduction in fluorescence. Substances were considered to be potential inhibitors if the result for any of the assays (Target 1, Target 2 or Sample Control) was negative, or if the Cp value was delayed by three or more cycles or for which the Fmax value was reduced by $50 \%$ or more when compared to the no-test substance control in the same JBAIDS run. These criteria allow for identification of potential PCR inhibitors while taking into account the intrinsic system variability. Any substance that showed evidence of interfering with PCR was retested to verify the original result and to evaluate the effect of lower concentrations of the test substance. Samples having uncertain results that retest uncertain were also re-evaluated.

# Endogenous Substances:

The endogenous substances did not show inhibition for the Influenza A/H5 (Asian lineage) detection system for either extraction kit. All substance-containing samples gave positive test results for both target assays as well as for the Sample Control assay. For the Target 1 assay, $\mathrm { C p }$ values for the test substances ranged from being 1.22 cycles earlier to1.70 cycles later when compared to the no-substance control. For Target 2, the $\Delta { \sf C p }$ values ranged from being 1.54 cycles earlier to 2.17 cycles later. The Sample Control $\mathrm { C p }$ value ranges were similar to those for Targets 1 and 2 (-1.87 to 1.97 cycles). For all assays, the Fmax values were similar to the nosubstance control values $( 8 0 \% - 1 3 3 \% )$ . All of these values are within the described acceptance criteria $( \Delta \mathrm { C p } < 3$ , Fmax $5 5 0 \%$ ). This data demonstrates that higher than expected levels of blood or mucin contained in NPS or TS samples will not interfere with obtaining accurate test results with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

# Exogenous Substances:

The exogenous substances did not show inhibition for the Influenza A/H5 (Asian lineage) detection system for either extraction kit. All substance-containing samples gave positive results for both target assays as well as for the Sample Control assay. For the Target 1 assay, the $\mathrm { C p }$ values ranged from being 1.61 cycles earlier to 0.91 cycles later when compared to the no-substance control in the same run. For the Target 2 assay, Cp values ranged from being 2.12 cycles earlier to 2.03 cycles later. The Sample Control Cp value ranges were similar to those for Targets 1 and 2 (- 0.93 to 0.98 cycles). For all assays, the Fmax values were similar to the nosubstance control values $( 6 2 \% - 2 6 5 \% )$ ). This data demonstrates that none of the exogenous substances tested in this study interfere with obtaining accurate test results with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

# Technique Specific Substances:

The technique-specific substances did not show inhibition for the Influenza A/H5 (Asian lineage) detection system for either extraction kit. All substance-containing samples gave positive results for both Influenza $\mathrm { A } / \mathrm { H } 5$ target assays as well as for the Sample Control assay. Target $1 \mathrm { C p }$ values ranged from being 0.26 cycles earlier to 1.16 cycles later when compared to the no- substance control in the same run. The $\Delta { \sf C p }$ for the Target 2 assay ranged from $+ 0 . 2 3$ cycles to -1.02 cycles. The Sample Control $\mathrm { C p }$ value ranges were similar to those for Targets 1 and 2 (-1.30 to 2.13 cycles). For all assays, the Fmax values were similar to the no-substance control values $( 9 3 \% - 1 2 1 \% )$ . This data shows that the binding buffer and wash buffer contained in the Platinum Path do not interfere with obtaining accurate test results with the JBAIDS Influenza $\mathrm { A } / \mathrm { H } 5$ (Asian lineage) Detection Kit, even if they are introduced to purified samples. This is contrary to the buffers used in the IT 1-2- 3 VIBE kit that are known to cause inhibition when they introduced into purified samples.

# Human Genomic DNA:

Even at the highest concentrations of genomic DNA, no inhibition was observed. A typical TS or NPS specimen from a healthy adult has a range of Cp values from 29- 31. Therefore, the highest level of human genomic DNA tested represents approximately 256 to 1,024-fold higher than what would be expected from a typical healthy adult.

For the Target 1 assay, the Cp values ranged from being 0.49 cycles earlier to 2.97 cycles later when compared to the no substance control in the same run. For the Target 2 assay, Cp values ranged from being 2.91 cycles earlier to 0.29 cycles earlier. The Sample Control $\mathrm { C p }$ value ranges were not calculated because the variability in the number of cells collected among different swab samples does not provide a sufficiently consistent value for comparison. For both target assays, the Fmax values were about half of the no-substance control values $( 5 0 \% - 6 7 \% )$ . While the reduction in Fmax is not below the $50 \%$ acceptance criteria, it is a very consistent finding and suggests that very high concentrations of human genomic DNA may cause a reduction in fluorescence for these assays. However, given that the test results are still unaffected when extraordinarily high concentrations of human DNA were tested in this study, this finding is unlikely to interfere with the accuracy of the test results for authentic specimens.

In conclusion, none of the potential interfering substances tested in this study, including high levels of human genomic DNA, or the buffers contained in the IT 1-2-3 Platinum Path Sample Purification kit cause inhibition of the assays contained in the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. A reduction in fluorescence was observed for both target assays at all concentrations of human genomic DNA tested. However, this reduction was not sufficient to generate false negative results. A potential delay in Cp can occur for the Target 1 assay at DNA concentrations 1,000-fold higher than would normally be seen in these sample types. The observed delay in Cp was insufficient to generate a false negative result.

During the execution of the “Evaluation of Inclusivity for the JBAIDS Influenza A/H5 Detection Kit” study, F12K tissue culture media was discovered to interfere with amplification of the Influenza A/H5 Target 1 and Target 2 assays. There is no known situation in which this media would be present in throat swab or nasopharyngeal swab specimens.

# i. Carry-Over Contamination:

In order to achieve accurate test results, it is important that the materials (i.e. reagent or template) and/or signal from one test do not cross-contaminate or carry over into adjacent tests. Carryover can result in a false positive result when material or signal from a positive test affects the outcome of an adjacent negative test. For the JBAIDS Influenza A/H5 (Asian lineage) Detection System, carryover could occur during sample purification, during reagent setup, or during signal acquisition in the JBAIDS instrument. To demonstrate that there is no significant carryover between samples in the JBAIDS Influenza A/H5 (Asian lineage) Detection System, an analytical carry-over contamination was carried out. Previous work with the

JBAIDS Plague (k072631) and Tularemia Detection kits (k072547) has indicated that carryover during signal acquisition does not occur and, therefore, this mode of carryover was not evaluated in this study. This study determined if there is significant carryover in the JBAIDS Influenza A/H5 (Asian lineage) Detection system when it is used as instructed by the product package insert.

To assess the potential for carryover, positive throat swab samples spiked with high $( \sim 3 . 2 \mathrm { X } 1 0 ^ { 4 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L } )$ levels of inactivated Flu A/H5N1 virus (Vietnam/1203/04 x PR8) and negative (unspiked) throat swab samples were purified in alternating order with the IT 1-2-3 VIBE and IT 1-2-3 Platinum Path kits and then tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit on a JBAIDS instrument. All 16 negative samples processed with the Platinum Path Kit gave the expected negative test results. Of the 16 negative samples processed with the IT 1-2-3 VIBE Sample Purification Kit, one sample purified with the VIBE Purification kit initially tested uncertain for the Target 2 assay, but was negative upon retesting. (Note: Retesting of uncertain samples is the standard procedure described in the product package insert instructions and users are instructed to accept the results of the retest as the final result.) Therefore, no false positive final results were seen in this study.

The results suggested that the risk for carryover may be higher when samples are purified with the IT 1-2-3 VIBE Purification Kit (1/16, or $6 . 2 5 \%$ , of the negative specimens were called uncertain) than when purified with the Platinum Path Purification kit $( 0 / 1 6 , 0 \% )$ . The differences in carryover frequency between kits may be attributed to differences in the purification procedures. The VIBE kit requires multiple transfers of the specimens during purification and several centrifugation steps. For the Platinum Path kit, the specimen is contained in a single strip tube and the purification process does not require any centrifugation steps. However, as the purified sample corresponding to the single false-positive capillary result (uncertain assay result) retested negative, it is possible that the contamination occurred during reaction setup or capillary loading and not during the purification process.

The sponsor included the recommendations for carryover prevention, such as the necessity for routine cleaning (benches, centrifuges, etc.), and thorough cleaning after processing a specimen with a strong positive test result in the package insert for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit to mitigate the risk of carry-over contamination.

# j. HPAI Influenza A/H5 Inactivation Study:

To determine if the lysis steps of the IT 1-2-3 VIBE and IT 1-2-3 Platinum Path sample purification kits can effectively inactivate highly pathogenic egg grown influenza A/H5 viruses, personnel at the Centers for Disease Control and Prevention (CDC) performed inactivation experiments using two highly pathogenic avian influenza viruses (HPAI; A/Japanese white eye/Hong Kong/1038/2006 and A/common magpie/Hong Kong/645/2006). Innocuity of treated viruses was determined by inoculation of embryonated chicken eggs followed by a hemagglutination assay. Eggs were candled at 3 days post-injection to determine if the embryo was alive or dead. Healthy, live embryos should have exhibited growth, have visible, intact blood vessels, and respond to light. Bad eggs with dead embryos were marked with an $ { { } ^ { \mathfrak { c } } } _ { \mathbf { X } } { } ^  , \}$ and not harvested. The embryos were killed by quick chilling the eggs at ${ - 2 0 ^ { \circ } C }$ for 30 minutes. The allantoic fluid was then harvested and tested for Influenza Virus by a hemagglutination assay. A negative result indicates that there is no Influenza Virus detectable and a positive result indicates that Influenza virus is present.

The CDC had determined that VIBE Buffer 1A and Platinum Path MB Binding Buffer were effective at inactivating these viruses when the buffer and virus sample were combined in a 1:1 ratio.

Based upon these data, HPAI-containing throat swab and nasopharyngeal swab specimens purified using the IT 1-2-3 VIBE and Platinum Path sample purification kit procedures can be expected to be non-infectious. The purified samples can therefore be handled safely outside of BSL-3 facilities; i.e. in BSL2 facilities.

2. Comparison studies:

a. Method comparison with predicate device:

Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# Clinical Study (specificity)

# Clinical Study Design:

To demonstrate that the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit has a high degree of clinical specificity, a clinical specificity study involving testing frozen banked NPS and TS specimens with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit (investigational device) and the influenza A/H5 (Asian lineage) assays (H5a, H5b and RNase P) from the FDA cleared Centers for Disease Control and Prevention (CDC) Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel) (comparator method) was conducted. These banked specimens had been previously tested for influenza using standard diagnostic methods, such as viral culture and direct fluorescence antibody staining (DFA). In addition, some of the specimens had also been tested for other respiratory pathogens, such as adenovirus, enterovirus, parainfluenza virus, respiratory syncytial virus, etc. The specimens represented prospectively collected archived (frozen)

specimens1 that were obtained from existing specimen banks at three test sites: Brooke Army Medical Center (BAMC), San Antonio, Texas; Naval Health Research Center (NHRC), San Diego, California; and Tripler Army Medical Center (TAMC), Honolulu, Hawaii. The study was written and conducted in accordance with the FDA guidance document “Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable”. Prior to testing, each site received approval from their local institutional review board (IRB) and other applicable IRBs.

Specimens were included in the study if they had sufficient volume to complete the required testing (at least $6 0 0 \mu \mathrm { L }$ ), but without bias to previous test results. Only specimens that did not have adequate sample volume, or were not collected for respiratory pathogen testing, were excluded from the study. Prior to testing, specimens that met the acceptance criteria were thawed, de-identified, aliquoted for testing, and assigned a specific study number. Specimens with sufficient volume for testing with the JBAIDS system were tested using both the VIBE and Platinum Path purification methods. If there was insufficient volume, then only one JBAIDS purification method was selected. All specimens were tested using the H5a, H5b and RNase P assays of the CDC rRT-PCR Flu Panel as a comparator to the JBAIDS Flu A/H5 test.

Specimens were purified using either or both of the IT 1-2-3 VIBE and IT 1-2-3 Platinum Path Sample Purification kits, depending on the available specimen volume. For this study, a Positive Extraction Control (PEC) and Negative Extraction Control (NEC) were included with each batch of purified specimens. The PEC was composed of pooled TS specimens spiked with inactive influenza A/H5 virus at approximately 2X LoD. The NEC consisted of nuclease-free water. Each purified specimen (including the PEC and NEC), was used to reconstitute freeze-dried reagent vials for the Target 1 assay, the Target 2 assay, and the Sample Control assay. For each JBAIDS run, a freeze-dried positive control (PC) vial and a negative control (NC) vial for the Target 1 and Target 2 assays (total of 4 vials) were reconstituted with water and reconstitution buffer. The contents of each rehydrated reagent vial was transferred into two glass capillaries and loaded in the JBAIDS carousel in a predefined loading pattern. The loaded carousel was place on the JBAIDS instrument which automatically applied the correct cycling and data analysis settings. Once the run was completed, the JBAIDS software automatically interpreted the result of each PCR reaction as well as the results of all associated controls and provided a final result for the Target 1 and Target 2 assays.

As with all JBAIDS assays, the software interprets each amplification curve and assigns a result of positive, negative, or in rare cases (about $0 . 2 \%$ ), uncertain. After each curve is interpreted, the result of the duplicate capillaries and the controls are combined to give a final result for each of the target assays. For the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit, possible results are positive, negative, uncertain, and invalid or sample control failure. Specimens with uncertain, invalid, or sample control failure test results were retested according to the product package insert and the final test result was used in the analysis of the data. While the JBAIDS software (Detector) provides a test result for each Target assay, the user is required to give a final interpretation. Possible final interpretations are positive (when both targets are positive), negative (when both targets are negative) and inconclusive (when the results of the two target assays do not match or are not valid). Samples with inconclusive test results were retested and the result of the retest was accepted as the final result.

In addition, a PEC and a NEC were purified with each batch of samples in this study. The PEC was expected to provide positive results for both target assays and the sample control assay. The NEC was expected to give a result of sample control failure for both target assays. When these results were not obtained, the failed PEC or NEC was retested. If the retest provided the expected result, then the control was considered successful and the associated samples were considered to have valid results. If the retest failed, then the results for the samples extracted with that PEC or NEC were considered invalid. Inclusion of a PEC and a NEC with each batch of extracted samples is not a standard recommendation in the product package insert. They were included as additional external controls only during this clinical evaluation per FDA recommendation.

The specimens collected at the three test sites reflect different patient populations, influenza seasons, specimen types and methods used to perform the previous testing. The specimen bank at BAMC was collected during the 2008/2009 respiratory season and before the 2009 H1N1 Influenza was identified. These specimens were tested using standard viral culture with direct fluorescence antigen (DFA) testing and had been frozen for validation studies of a new testing platform. The age range is large $\left[ < 1 \right]$ year to 80 years of age), but the average age (17) reflects a large proportion of pediatric specimens. This is consistent with the testing patterns for a community based hospital. Specimens from TAMC are similar to those collected at BAMC, except that these specimens were collected when $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ Influenza was prevalent (June 2009 to Dec. 2009). As with BAMC, the range of ages is large $\mathrm { ^ { \prime } } { < } 1$ to 84 years of age), and the average age (22) reflect a larger proportion of pediatric specimens. These specimens were tested with a variety of different methods, including rapid antigen tests, viral culture with DFA, PCR (FDA cleared or authorized, or LDT validated at NHRC), and heme adsorption followed by immunofluorescence assay. The specimens from NHRC appear to be significantly different than the specimens collected at BAMC and TAMC (BAMC and TAMC are community based hospitals that serve military personnel and their dependents). NHRC primarily received specimens from active military personnel, and particularly from recruit camps. This is evidenced by the fact that the patient population is composed primarily of young males (age range of 17-41 with an average of 21 years, $8 8 \%$ male). These specimens also differ from the BAMC and TAMC specimens in that they were collected during the 2006/2007 respiratory disease season.

Summary of Subject Demographics   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>BAMC</td><td rowspan=1 colspan=1>NHRC</td><td rowspan=1 colspan=1>TAMC</td></tr><tr><td rowspan=2 colspan=1>Number ofSubjects</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>250</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>218</td></tr><tr><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>378</td><td rowspan=1 colspan=1>140</td></tr><tr><td rowspan=2 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>38%</td><td rowspan=1 colspan=1>12%</td><td rowspan=1 colspan=1>43%</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>62%</td><td rowspan=1 colspan=1>88%</td><td rowspan=1 colspan=1>57%</td></tr><tr><td rowspan=3 colspan=1>Age</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=2>Specimen CollectionPeriod (m/yyyy)</td><td rowspan=1 colspan=1>2/2009 -4/2009a</td><td rowspan=1 colspan=1>11/2006-2/2007</td><td rowspan=1 colspan=1>6/2009-12/2009</td></tr><tr><td rowspan=1 colspan=2>PreviousDetections</td><td rowspan=1 colspan=1>Adenovirus (n=6)Influenza Ano subtype (n=7)Influenza A/H1 (n=12)Influenza B (n=7)Parainfluenza 2 (n=1)Parainfluenza 3 (n=3)</td><td rowspan=1 colspan=1>Adenovirus (n=198)Influenza A/H1 (n=10)Influenza A/H3 (n=17)Influenza Ano subtype (n=9)</td><td rowspan=1 colspan=1>Adenovirus (n=5)Enterovirus (n=3)2009 H1N1 Influenza (n=25)Influenza A no subtype (n=74)Influenza B (n=7)Parainfluenza 1 (n=2)Parainfluenza 2 (n=2)Parainfluenza 3 (n=2)RSV (n=1)</td></tr></table>

a Two specimens were collected 11/2009

# Clinical Study Results:

# NPS Specimens

A total of $4 6 8 \mathrm { N P S }$ specimens were tested in this clinical study. Three hundred and sixty one (361) were purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit and 327 were purified using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit. Of the 361 NPS specimens purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, 314 out of 361 $( 8 7 . 0 \% )$ provided valid test result with both the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit and CDC rRT-PCR Flu Panel Influenza A/H5 (Asian lineage) assays. All 314 specimens gave negative results with both the investigational device and the comparator device. Valid JBAIDS Influenza A/H5 (Asian lineage) Detection Kit and CDC rRT-PCR Flu Panel Influenza $\mathrm { A } / \mathrm { H } 5$ (Asian lineage) assays results were obtained for 299 out of a total of 327 $( 9 1 . 4 \% )$ of the NPS specimens purified with IT $1 - 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit. All 299 gave negative results with both devices.

Performance of the JBAIDS Influenza $\mathbf { A } / \mathbf { H } 5$ (Asian lineage) Detection Kit Testing NPS Specimens   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>CDC Flu A/H5(Asian lineage)result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>PurificationKit</td><td rowspan=1 colspan=1>JBAIDS Flu A/H5(Asian lineage) result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Performance</td></tr><tr><td rowspan=3 colspan=2>VIBE</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>Positive Percent Agreement N/A*Negative Percent Agreement 100% (314/314),(99-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>314</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>314</td></tr><tr><td rowspan=2 colspan=2>Platinum</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>Positive Percent Agreement N/A*Negative Percent Agreement 100% (299/299),(99-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>Negative Percent Agr</td></tr><tr><td rowspan=1 colspan=2>Path</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>299</td></tr></table>

\* Due to the lack of available specimens from individuals infected with influenza A/H5 (Asian lineage), it was not possible to establish the positive percent agreement of the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit to the CDC rRT-PCR Flu Panel Influenza A/H5 (Asian lineage) assays in this study.

Of the specimens processed with the VIBE purification kit, 47 were excluded from this statistical analysis due to lack of a valid result for the JBAIDS testing $( \mathrm { n } { = } 2 3 )$ ) and/or the CDC rRT-PCR Flu Panel testing $( \mathrm { n } { = } 3 2 )$ with 8 specimens having invalid results for both tests. JBAIDS failures occurred at BAMC $( \mathrm { n } { = } 9 )$ and TAMC $( \mathrm { n } { = } 1 4 )$ and included 19 specimens associated with missing or failed PEC and/or NECs and 4 specimens that had a sample control failure results. All 32 CDC rRT- PCR Flu Panel failures occurred with specimens from BAMC that were shipped to NHRC for testing. These included 4 that were not tested, 10 with sample control failures and 18 for which the H5a notemplate control was positive. All 10 sample control failures occurred with specimens that had been inadvertently shipped to NHRC at ambient temperature instead of at 2- $8 \mathrm { { } ^ { \circ } C }$ . Failure of the H5a no-template control was related to PCR amplicon contamination at NHRC.

Of the $3 2 7 \mathrm { N P S }$ specimens purified with the Platinum Path purification kit, 28 lacked a valid result for the investigational device testing $( \mathrm { n } { = } 8 )$ and/or the comparator method testing $( \mathrm { n } { = } 2 1 )$ ), with 1 specimen having invalid results for both tests. One (1) JBAIDS failure occurred at BAMC and 7 occurred at TAMC. Seven (7) of the failures were associated with a missing or invalid NEC or PEC and 1 was a sample control failure. As with the VIBE-purified NPS specimens, all CDC rRT-PCR Flu Panel failures occurred with specimens shipped from BAMC to NHRC for testing. This included 17 sample control failures and 7 for which the H5a no-template control was positive. These failures are associated with improper shipping of the specimens and contamination at NHRC.

# TS Specimens

A total of 518 TS specimens were tested in this clinical study. Three hundred and three (303) were purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit and 333 were purified using IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit. Of the 303 TS specimens purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, 298 $( 9 8 . 3 \% )$ provided valid test results with both the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit and CDC rRT-PCR Flu Panel Influenza A/H5 (Asian lineage) assays. All 298 $( 1 0 0 \% )$ specimens gave negative results with both the investigational device and the comparator device. Valid JBAIDS Influenza A/H5 (Asian lineage) Detection Kit and CDC rRT-PCR Flu Panel Influenza A/H5 (Asian lineage) assays were obtained for 283 out of 333 $( 8 5 . 0 \% )$ of the TS specimens purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Purification Kit. All 283 gave negative results with both devices.

Performance of the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit Testing TS Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>CDC Flu A/H5(Asian lineage) result</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Furication Ki</td><td rowspan=1 colspan=1>Purification Kit JBAIDS Flu A/H5 (Asianlineage) result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Performance</td></tr><tr><td rowspan=3 colspan=1>VIBE</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>Positive Percent Agreement (PPA) N/A*Negative Percent Agreement (NPA) 100% (298/298),(99-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>298</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>298</td></tr><tr><td rowspan=3 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>Positive Percent Agreement N/A*Negative Percent Agreement 100% (283/283),(99-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>283</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>283</td></tr></table>

\* Due to the lack of available specimens from individuals infected with influenza A/H5 (Asian lineage), it was not possible to establish the positive percent agreement of the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit to the CDC rRT-PCR Flu Panel Influenza A/H5 (Asian lineage) assays in this study.

Of the 303 TS specimens processed with the VIBE purification kit, only 5 were excluded from the statistical analysis due to lack of a valid result for the JBAIDS testing $( \mathrm { n } { = } 2 )$ and/or the CDC rRT- PCR Flu Panel $( \mathrm { n } { = } 3 )$ ). All invalid results occurred at NHRC $( \mathrm { n } { = } 9 )$ ). The JBAIDS failures included 1 specimen associated with failed PEC and NEC tests and 1 specimen had a sample control failure result. Three (3) specimens were never tested with the CDC rRT-PCR Flu Panel and therefore had to be excluded from the analysis.

Of the 333 TS specimens purified with the Platinum Path purification kit, 50 lacked a valid result for the JBAIDS testing $\left( \mathrm { n } { = } 3 7 \right)$ and/or the CDC rRT-PCR Flu Panel testing $( \mathrm { n } { = } 1 4 )$ , with 1 specimen having invalid results for both tests. Ten (10) JBAIDS failures occurred at NHRC and 27 occurred at TAMC. Thirty (30) of the failures were associated with an invalid NEC or PEC, 5 were sample control failures, 1 was invalid due to a failed PC for the Target 2 assay, and 1 gave an inconclusive result. For the CDC rRT-PCR Flu Panel testing, 9 specimens gave inconclusive results and 5 were not tested. The 9 specimens with inconclusive test results all occurred at NHRC and all had a positive result with the Target 1 (or H5a) assay and a negative result with the Target 2 (H5b assay). These test results were caused by amplicon contamination at NHRC.

The sponsor included the recommendations for carryover and amplicon contamination prevention, such as the necessity for routine cleaning (benches, centrifuges, etc.), thorough cleaning after processing a specimen with a strong positive test result, and decontamination immediately after capillary breakage in the package insert for the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit to mitigate the risk of carry-over and amplicon contamination.

# Results of Controls

Out of a total of 519 runs during the clinical specificity study, 511 $( 9 8 . 5 \% )$ runs had successful PCs for both target assays initially, and 512 $( 9 8 . 7 \% )$ runs had successful PCs for both target assays initially. The mean PC $\mathrm { C p }$ value for Target 1 was $3 3 . 1 4 \pm 0 . 6 3$ ( $1 . 9 0 \%$ CV) and for Target 2 was $3 3 . 8 5 \pm 1 . 2 1$ ( $3 . 5 7 \%$ CV).

Out of a total of 172 specimen purification runs during the clinical specificity study, 164 of PEC $( 9 5 . 3 \% )$ and 167 of NEC controls $( 9 7 . 1 \% )$ gave correct results initially. The mean Cp of the PEC tested for Target 1 was 31.08, with 1.85 Standard Deviation (SD) and $5 . 9 5 \%$ CV. The mean $\mathrm { C p }$ of the PEC tested for Target 2 was 34.46, with 1.62 Standard Deviation (SD) and $4 . 7 0 \%$ CV.

A Sample Control assay (human RNase $\mathrm { P }$ assay) was also tested with each purified sample to detect inhibition or poor sample extraction during the clinical specificity study:

Summary of Sample Control Results (Clinical Specificity Study)   

<table><tr><td rowspan=2 colspan=1>Kit</td><td rowspan=2 colspan=1>TotalSampleControls</td><td rowspan=2 colspan=1>PositiveSampleControls</td><td rowspan=2 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>326 (97.9%)</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>9.47</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>327</td><td rowspan=1 colspan=1>325 (99.4%)</td><td rowspan=1 colspan=1>29.09</td><td rowspan=1 colspan=1>3.52</td><td rowspan=1 colspan=1>12.10</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>303 (100%)</td><td rowspan=1 colspan=1>30.77</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>7.44</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>357 (98.9%)</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>3.42</td><td rowspan=1 colspan=1>11.24</td></tr></table>

# Conclusion

Three hundred fourteen (314) NPS specimens purified using the IT 1-2-3 VIBE Sample Purification Kit and $2 9 9 \mathrm { N P S }$ specimens purified using the IT 1-2-3 Platinum Path Purification kit had valid negative result for both the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit and the CDC rRT Flu Panel Influenza A/H5 (Asian lineage) assays. Similarly, 298 VIBE purified and 283 Platinum Path purified TS specimens had valid negative results with both tests. These results demonstrated that the negative percent agreement for NPS and TS specimens purified with the IT 1-2-3 VIBE or Platinum Path Purification Kits and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit is at least $9 9 \%$ with $9 5 \%$ confidence.

Summary of Clinical Trial Results   

<table><tr><td rowspan=2 colspan=1>SampleType</td><td rowspan=2 colspan=1>PurificationSiteKit</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Specimens</td><td rowspan=1 colspan=5>JBAIDS Results</td><td rowspan=2 colspan=1>Performance</td></tr><tr><td rowspan=1 colspan=1>TotalTested</td><td rowspan=1 colspan=1>Removed fromstudy a</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Inconclusive</td><td rowspan=1 colspan=1>SCFailure</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=6 colspan=1>Neoe a ryos</td><td rowspan=3 colspan=1>IT 1-2-3 TMVIBE</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>NPA (95% CI):99-100%PPA: NAd</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>201</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>187</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>12b</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>361</td><td rowspan=1 colspan=1>32 (8.9%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=3 colspan=1>T 1-2-3 TMPlatinumPath</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>153</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=3 colspan=1>NPA (95% CI):99-10%PPA: NAd</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>174</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>167</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6b</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>327</td><td rowspan=1 colspan=1>21 (6.4%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>299</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=6 colspan=1>Se Sas</td><td rowspan=3 colspan=1>T 1-2-3 TMVIBE</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1b,c</td><td rowspan=3 colspan=1>NPA (95% CI):99-100%PPA: NAd</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>132</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>303</td><td rowspan=1 colspan=1>3 (1.0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>298</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>IT 1-2-3 TMPlatinumPath</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9b</td><td rowspan=3 colspan=1>NPA (95% CI):99100%PPA: NAd</td></tr><tr><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22b</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>14 (4.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>31</td></tr></table>

a Specimens were removed from the study if they did not have a valid result for the CDC comparator assay.

# Study Testing Surrogate Clinical Samples

# Study Design:

True clinical specimens containing influenza A/H5N1 are not readily available for testing. Therefore, a study was carried out at the Midwest Research Institute (MRI) testing a panel of surrogate clinical samples consisting of 8 strains of Influenza A/H5N1 (the same strains tested in the Analytical Inclusivity Study, representing sub-clades 1.0, 2.1, 2.2, and 2.3) and 6 strains of seasonal influenza spiked into TS or NPS specimens collected in viral transport medium (VTM). The complete panel, for both TS and NPS specimens, consisted of the following: Influenza A/H5N1 viruses at predetermined LoD, 5X LoD, 10X LoD, and 100X LoD (LoD for this panel was set at $5 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ based on comparison and correlation of $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ to $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ data generated from the Analytical Inclusivity Study testing surrogate samples); seasonal influenza viruses at LoD and 100X LoD; and 6 specimens per sample type (NPS and TS) that were not spiked with any virus.

Overall, a total of 32 Influenza A/H5 (Asian lineage) positive NPS samples, 12 seasonal Influenza A or Influenza B positive NPS samples, and 6 Influenza virus negative NPS samples were included in the surrogate clinical NPS specimen panel, and a total of 32 Influenza A/H5 (Asian lineage) positive TS samples, 12 seasonal Influenza A or Influenza B positive TS samples, and 6 Influenza virus negative TS samples were included in the surrogate clinical TS specimen panel:

Surrogate Clinical Specimen Panel- NPS   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Purification Kit</td><td colspan="1" rowspan="1">ExpectedJBAIDSFluA/H5(Asian lineage)Detection KitResult</td></tr><tr><td colspan="1" rowspan="1">Avian precursor Yunan(A/Chicken/Yunnan/1251/03)(H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Vietnam/1203/2004(H5N1)-PR8/CDC-RG Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Avian precursor Hunan(A/Duck/Hunan/795/02) (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Psitive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,0000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Chicken/Korea/IS/06 (H5SN1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Scaly Breasted Munia/HongKong/45/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Japanese white eye/HongKong/1038/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,0000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Common Magpie/HongKong/645/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1"> Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/1999 (H1N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/2001 (H1N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/New York/55/2004 (H3N2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/2005 (H3N2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">B/Ohio/01/2005 (Victoria/2/87-like)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/2004(Yamagata/16/88-Like)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">NPS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr></table>

Abbreviations are as follows: PP, IT $1 - 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit; VIBE, IT 1- $2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit; VTM, Viral Transport Media; NPS, Nasopharyngeal Swab; $\mathrm { N } / \mathrm { A }$ , not applicable; and $\mathrm { T C I D 5 0 / m l }$ , $50 \%$ tissue culture infectious dose per mL.

Surrogate Clinical Specimen Panel- TS   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Purification Kit</td><td colspan="1" rowspan="1">ExpectedJBAIDSFluA/H5(Asian lineage)Detection KitResult</td></tr><tr><td colspan="1" rowspan="1">Avian precursor Yunan(A/Chicken/Yunnan/1251/03)(H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Vietnam/1203/2004(H5N1)-PR8/CDC-RG Avian</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Avian precursor Hunan(A/Duck/Hunan/795/02) (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">A/Chicken/Korea/IS/06(H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Scaly Breasted Munia/HongKong/45/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">Japanese white eye/HongKong/1038/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Common Magpie/HongKong/645/2006 (H5N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2,500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/1999 (H1N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/2001 (H1N1)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/New York/55/2004 (H3N2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/2005 (H3N2)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">B/Ohio/01/2005 (Victoria/2/87-like)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/2004(Yamagata/16/88-Like)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">50,000</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #5</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Unspiked #6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">PP</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td></td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">TS in VTM</td><td colspan="1" rowspan="1">VIBE</td><td colspan="1" rowspan="1">Negative</td></tr></table>

Abbreviations are as follows: PP, IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit; VIBE, IT 1- $2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit; VTM, Viral Transport Media; TS, Throat Swab; N/A, not applicable; and $\mathrm { T C I D 5 0 / m l }$ , $50 \%$ tissue culture infectious dose per mL.

Personnel at MRI grew virus cultures and quantification was performed by determining tissue culture infectious doses in accordance with a procedure approved by the CDC. In accordance with the CDC approved protocol, the $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ quantification was confirmed by spiking a contrived sample type (A549 cells in F12K media) with designated levels of virus. The prepared samples were then purified using the Qiagen

Viral RNA mini kit followed by testing with the appropriate FDA cleared CDC assays (FluA/H5a, FluA/H5b and the Sample Control). The results obtained were compared to previous lot quantification and determined to be acceptable.

An investigation revealed that the F12K medium was causing inhibition of the JBAIDS system. Since this contrived sample matrix was not representative of the intended sample matrix, it was determined that TS in VTM would be used as the sample matrix for this study. Using the same qualified viral stocks, MRI prepared the surrogate clinical sample panels described above. The specimens were randomized and blinded to the operators. Each specimen was purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE (VIBE) and IT 1- $2 { - } 3 ^ { \mathrm { T N } }$ Platinum Path (PP) Sample Purification Kits, and then tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit at MRI.

# Study Results:

# NPS Surrogate Specimens

All Sample Control assays gave the expected positive results.

$9 6 . 9 \%$ (31/32) of NPS samples spiked with influenza A/H5N1 virus were positive when purified using the IT $1 - 2 { - } 3 ^ { \mathrm { { \hat { T } M } } }$ Platinum Path Sample Purification Kit and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. (Note: Samples were considered positive only if both target assays were positive). One NPS sample spiked at 5X LoD gave a negative result with both the Target 1 and Target 2 assays. However this same sample gave a positive result when purified with the IT $1 - 2 { - } 3 ^ { \mathrm { \tilde { T } M } }$ VIBE Sample Purification Kit.

$100 \%$ (32/32) of NPS samples spiked with influenza A/H5N1 virus were positive when purified using the IT $1 { - } 2 { - } 3 ^ { \mathrm { \tiny { T M } } }$ VIBE Sample Purification Kit and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. (Note: Samples were considered positive only if both target assays were positive).

$100 \%$ (12/12) of NPS samples spiked with seasonal influenza A virus or influenza B virus were negative when purified using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit or the IT $\mathrm { i } { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

$100 \%$ (6/6) of unspiked NPS samples were negative when purified using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit or the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

Overall, across purification kits, for individual spike levels, positive results obtained were: $100 \%$ (16/16) at LoD, 10X LoD and 100X LoD; and $9 3 . 8 \%$ (15/16) at 5XLoD.

# TS Surrogate Specimens

All Sample Control assays gave the expected positive results.

$100 \%$ (32/32) of TS samples spiked with influenza A/H5N1 virus were positive when purified using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

$9 6 . 9 \%$ (31/32) of TS samples spiked with influenza A/H5N1 virus were positive when purified using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit. One TS sample spiked at LoD gave an inconclusive result. However this same sample gave a positive result when purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit.

$100 \%$ (12/12) of TS samples spiked with seasonal influenza A virus or influenza B virus were negative when purified using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit or the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

$100 \%$ (6/6) of unspiked NPS samples were negative when purified using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit or the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE Sample Purification Kit, and tested with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit.

Overall, across purification kits, for individual spike levels, positive results obtained were: $9 3 . 8 \%$ (15/16) at LoD; and $100 \%$ (16/16) at 5X LoD, 10X LoD and 100X LoD.

# Results of Controls

All PC and NC reactions were successful in all runs performed for this study. The mean PC Cp value for Target 1 was $3 4 . 3 8 \pm 0 . 7 4$ ( $2 . 1 5 \%$ CV) and for Target 2 was $3 5 . 5 8 \pm$ 0.92 $2 . 5 9 \%$ CV).

A Sample Control assay (human RNase P assay) was tested with each purified sample to detect inhibition or poor sample extraction during the surrogate sample testing at the Midwest Research Institute (MRI) to estimate clinical sensitivity. All samples produced positive Sample Control results:

Summary of Sample Control Results (Surrogate Sample Testing at the MRI)   

<table><tr><td rowspan=1 colspan=1>Kit</td><td rowspan=1 colspan=1>TotalSampleControls</td><td rowspan=1 colspan=1>PositiveSampleControls</td><td rowspan=1 colspan=1>MeanCp</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Platinum Path TS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>32.79</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>4.09</td></tr><tr><td rowspan=1 colspan=1>Platinum Path NPS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>30.81</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>6.39</td></tr><tr><td rowspan=1 colspan=1>VIBE TS</td><td rowspan=1 colspan=1>53a</td><td rowspan=1 colspan=1>53 (100%)</td><td rowspan=1 colspan=1>33.46</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.66</td></tr><tr><td rowspan=1 colspan=1>VIBE NPS</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>50 (100%)</td><td rowspan=1 colspan=1>31.80</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>6.79</td></tr></table>

a Total number including retests.

# Conclusion

These data indicated that the performance of the JBAIDS system is similar for TS and NPS specimens processed using either the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit or the IT $1 { - } 2 { - } 3 ^ { \mathrm { \overline { { T } } M } }$ VIBE Sample Purification Kit.

Overall, the JBIDS Influenza A/H5 (Asian lineage) Detection System successfully excluded $100 \%$ (72/72) of all Influenza A/H5 (Asian lineage) negative samples, including 48 seasonal influenza spiked samples (48/48) and 24 unspiked samples (24/24), and detected $9 8 . 4 \%$ (126/128) of all avian influenza A/H5N1 spiked samples. One sample at 1X LoD was inconclusive after IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ VIBE purification and a second sample at 5X LoD was negative after IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path purification:

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Performance</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="3">NPS samples spiked withinfluenza A/H5N1 (Asianlineage) virus</td><td colspan="1" rowspan="1">IT 1-2-3TM Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Positive Percent Agreement96.9% (31/32)</td><td colspan="1" rowspan="1">83.8-99.9</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3™M VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Positive Percent Agreement100% (32/32)</td><td colspan="1" rowspan="1">89.1-100.0</td></tr><tr><td colspan="1" rowspan="1">NPS Samples AcrossPurification Kits</td><td colspan="1" rowspan="1">Positive Percent Agreement98.4% (63/64)</td><td colspan="1" rowspan="1">91.6-100.0</td></tr><tr><td colspan="1" rowspan="3">TS samples spiked withinfluenza A/H5N1 (Asianlineage) virus</td><td colspan="1" rowspan="1">IT 1-2-3T Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Positive Percent Agreement100% (32/32)</td><td colspan="1" rowspan="1">89.1-100.0</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3TM VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Positive Percent Agreement96.9% (31/32)</td><td colspan="1" rowspan="1">83.8-99.9</td></tr><tr><td colspan="1" rowspan="1">TS Samples AcrossPurification Kits</td><td colspan="1" rowspan="1">Positive Percent Agreement98.4% (63/64)</td><td colspan="1" rowspan="1">91.6-100.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Overall (across sampletypes and purificationkits</td><td colspan="1" rowspan="1">Positive PercentAgreement98.4% (126/128)</td><td colspan="1" rowspan="1">94.5-99.8</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">NPS samples spiked withseasonal influenza virus</td><td colspan="1" rowspan="1">IT 1-2-3TM Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3TM VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td colspan="1" rowspan="1">Across Purification Kits</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (24/24)</td><td colspan="1" rowspan="1">85.8-100.0</td></tr><tr><td colspan="1" rowspan="3">Unspiked NPS samples</td><td colspan="1" rowspan="1">IT 1-2-3TM Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (6/6)</td><td colspan="1" rowspan="1">54.1-100.0</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3TM VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (6/6)</td><td colspan="1" rowspan="1">54.1-100.0</td></tr><tr><td colspan="1" rowspan="1">Across Purification Kits</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Non-A/H5N1 NPS Samples(across purification kits)</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (36/36)</td><td colspan="1" rowspan="1">90.3-100.0</td></tr><tr><td colspan="1" rowspan="3">TS samples spiked withseasonal influenza virus</td><td colspan="1" rowspan="1">IT 1-2-3 Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3TM VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td colspan="1" rowspan="1">Across Purification Kits</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (24/24)</td><td colspan="1" rowspan="1">85.8-100.0</td></tr><tr><td colspan="1" rowspan="3">Unspiked TS samples</td><td colspan="1" rowspan="1">IT 1-2-3TM Platinum PathSample Purification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (6/6)</td><td colspan="1" rowspan="1">54.1-100.0</td></tr><tr><td colspan="1" rowspan="1">IT 1-2-3T VIBE SamplePurification Kit</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (6/6)</td><td colspan="1" rowspan="1">54.1-100.0</td></tr><tr><td colspan="1" rowspan="1">Across Purification Kits</td><td colspan="1" rowspan="1">Negative Percent Agreement100% (12/12)</td><td colspan="1" rowspan="1">73.5-100.0</td></tr><tr><td></td><td>Non-A/H5N1 TS Samples (across purification kits) Overall (across sample</td><td>Negative Percent Agreement 100% (36/36) Negative Percent</td><td>90.3-100.0</td></tr><tr><td></td><td>types and purification kits</td><td>Agreement 100% (72/72)</td><td>95.0-100.0</td></tr></table>

Retesting of purified samples was minimal with only 3 out of 200 samples requiring retest. Of these, one was a seasonal influenza spiked sample which tested uncertain for target 2 and retesting of this sample was negative. Avian influenza A/H5N1 samples which required re-testing included one sample that remained uncertain and would have required re-purification, however, due to the nature of this study, sufficient sample was not available. A second avian influenza A/H5N1 sample initially tested uncertain for target 2 after IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum path purification but retested positive.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

The incidence of human infections with influenza A/H5 (Asian lineage) is rare. From 1996 through 2009, 467 cases have been reported in 5 countries according to WHO, Avian Influenza Update. http://www.who.int/csr/disease/avian_influenza/en/2.

Currently, influenza A/H5 (Asian lineage) virus has not been detected in poultry or humans in the United States. False-positive results are more likely to occur when disease prevalence in the community is low. Testing with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit should not be performed unless the patient meets the most current U.S. Department of Health and Human Services (DHHS) clinical and epidemiologic criteria for testing suspect A/H5 specimens.

# N. Instrument Name:

Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument.

# O. System Descriptions:

1. Modes of Operation:

The JBAIDS Instrument is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Three other nucleic acid amplification tests that use the JBAIDS Instrument have received 510(k) clearance: JBAIDS Anthrax Detection System (k051713), JBAIDS Plague Detection System (k072631) and JBAIDS Tularemia Detection System (k072547). The JBAIDS Instrument is a ruggedized, portable real-time PCR instrument designed to withstand the conditions of transport and use likely to be encountered in a military field laboratory. The instrument is composed of an air thermocycler that amplifies specific DNA sequences using PCR and a fluorimeter that measures fluorescence signals associated with production of PCR product (amplicon) during the course of the reaction. For thermocycling, samples contained in glass capillaries are placed in the sample chamber where they are heated and cooled by the JBAIDS instrument.

The JBAIDS Software is preloaded on a ruggedized laptop computer. The software controls the instrument’s thermal cycling functions, acquires the fluorescence data from the instrument, and displays the fluorescence data for the user during the run. When the run is finished, the software’s Detector module analyzes the data and displays test results. The instrument and software have the ability to perform either diagnostic (IVD) or surveillance (environmental) testing.

The IVD software includes the following diagnostic components:

A traceable database that limits the data that the user can change in a test run and that requires the user to enter change notes for any changes that are allowed. A Diagnostic Wizard that takes the user through the process of setting up the JBAIDS run in compliance with the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit package insert.

IVD testing is always performed using the Diagnostic Wizard and the traceable database. The Wizard requires each run to contain a Positive Control and a Negative Control for each target included in the run, and a Sample Control for each sample. The Wizard also requires that duplicate capillaries be included for each Unknown, Positive Control, Negative Control, and Sample Control. The Detector module that automatically analyzes the fluorescence amplification curves and that displays the final test results to the user.

The JBAIDS software was specifically modified to support the use of Influenza A/H5 (Asian lineage) assay.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification:

User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The following controls are included in the JBAIDS Influenza A/H5 (Asian lineage) Detection Kit:

# Negative Control (NC)

The NC is used to detect contamination from target-specific amplified product, synthetic RNA (as found in the PC vials), or organism. Each JBAIDS run requires one NC (resulting in two capillaries) for the Target 1 and one for the Target 2 assays (total of 4 capillaries). For each target assay, both of the NC capillaries must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run, and the run must be repeated. Frequent or repeated failures of NCs may indicate significant contamination of the work area.

# Positive Control (PC)

The PC serves as an amplification and detection control. Each JBAIDS run requires one PC (resulting in two capillaries) for the Target 1 assay and one for the Target 2 assay (total of 4 capillaries). For each target assay, both of the PC capillaries must be positive and have Cp results that are earlier than the assay‘s specific cutoff value. If either capillary fails, the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the run must be repeated. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument. If the SC capillaries in the same test run are positive, then the failure is most likely caused by an isolated error with the setup of the PC. If the Sample Control (SC) capillaries are also negative, possible causes for failure are 1) a systematic error in sample setup, 2) degradation of the reagents, or 3) a malfunction of the JBAIDS instrument.

# Sample Control (SC)

The Sample Control assay detects the human RNase P gene. This assay is designed to guard against false negative results caused by an improperly collected specimen, ineffective purification of nucleic acids and or inhibition of the PCR reaction. A properly collected throat or NPS specimen contains human cells from which the RNase P target is recovered during sample purification. Following purification, each sample is then tested with the two Influenza A/H5 (Asian lineage) target assays, as well as, the Sample Control assay. If extraction and amplification were successful, then the Sample Control Assay will give the expected positive test result for both capillaries. The Sample Control is considered to be successful only if both Sample Control capillaries are positive. The JBAIDS software automatically assigned a result of sample control failure when 1) the Sample Control is unsuccessful and 2) the target assay is negative (or uncertain). If the target assay is positive, then the sample is positive regardless of the result of the Sample Control assay. Because the results of the two Influenza A/H5 target assays are interpreted separately, it is possible to have a sample control failure for one of the target assays and a positive result of the other target.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above: Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision