# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

K101855

# B. Purpose for Submission:

This is a new $5 1 0 \mathrm { k }$ application for a qualitative real-time reverse transcription-polymerase chain reaction (RT-PCR) assay used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$ for the qualitative detection and discrimination of seasonal Influenza $\mathrm { A } / \mathrm { H } 1$ , seasonal Influenza A/H3 and 2009 H1N1 Influenza viral nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens from human patients with signs and symptoms of respiratory infection. The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# C. Measurand:

Target RNA sequences for the respective conserved regions of the Hemagglutinin (HA) gene for seasonal influenza A/H1, seasonal influenza A/H3 and 2009 H1N1 Influenza Virus (swine-origin), and for the transcript derived from $E$ . coli Bacteriophage MS2 Aprotein gene (Internal Control).

# D. Type of Test:

Real-time reverse transcription-polymerase chain reaction (RT-PCR), qualitative detection and discrimination of seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza in nasopharyngeal swabs using nucleic acid isolation (The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux)), amplification and detection on the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$ , which generates signals based on the acquisition of spectrofluorometric data.

# E. Applicant:

Gen-Probe Prodesse Incorporated

F. Proprietary and Established Names:

Prodesse ProFAST $^ +$ Assay

# G. Regulatory Information:

1. Regulation section: 866.3332

2. Classification: Class II

3. Product code: OQW

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The ProFAST $\mathrm { \cdot \mathrm { T M _ { + } } }$ Assay is a multiplex Real Time RT-PCR in vitro diagnostic test for the qualitative detection and discrimination of seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza viral nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens from human patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. This assay targets conserved regions of the Hemagglutinin (HA) gene for seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza Virus, respectively. This assay is not intended to detect Influenza B or Influenza C Viruses.

A negative ProFAST $^ +$ Assay result is a presumptive negative result for Influenza A. These results should be confirmed by an FDA cleared nucleic acid-based test (NAT) detecting Influenza A.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as the Intended Use

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements:

To be used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version $1 . 7 6$ or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$ and either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# I. Device Description:

The ProFAST $^ +$ Assay is a multiplex Taqman based Real Time RT-PCR test that enables detection and differentiation of human Influenza A Virus subtypes: seasonal H1, seasonal H3, and 2009 H1N1 Influenza.

An overview of the procedure is as follows:

1. Collect NP swab specimens from symptomatic patients using a polyester, rayon or nylon tipped swab and place into viral transport medium (not provided with the kit).   
2. Add an Internal Control (IC) to every sample to monitor for inhibitors present in the specimens.   
3. Perform isolation and purification of nucleic acids using a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche), or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux),   
4. Add purified nucleic acids to the supermix along with enzymes included in the ProFAST+ kit. The supermix contains target-specific oligonucleotide primers and probes. The probes are dual-labeled with a reporter dye and a quencher dye (see table below).   
5. Perform reverse transcription of RNA into complementary DNA (cDNA) and subsequent amplification of DNA in a Cepheid SmartCycler® II instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProFAST $^ +$ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

ProFAST $^ +$ Assay Analyte Gene Targets and Probe Labels:

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Gene Targeted</td><td rowspan=1 colspan=1>Probe Fluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=1>EmissionPeak</td></tr><tr><td rowspan=1 colspan=1>Seasonal Influenza A/H1</td><td rowspan=1 colspan=1>Hemagglutinin</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td></tr><tr><td rowspan=1 colspan=1>Seasonal Influenza A/H3</td><td rowspan=1 colspan=1>Hemagglutinin</td><td rowspan=1 colspan=1>Cal Orange 560</td><td rowspan=1 colspan=1>540 nm</td><td rowspan=1 colspan=1>561 nm</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Influenza Virus</td><td rowspan=1 colspan=1>Hemagglutinin</td><td rowspan=1 colspan=1>Cal Red 610</td><td rowspan=1 colspan=1>595 nm</td><td rowspan=1 colspan=1>615 nm</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>E.coli MS2 Phage</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>667 nm</td></tr></table>

# Materials Provided

ProFAST+ Assay Kit (Cat. # H34VK00)   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity/Tube</td><td rowspan=1 colspan=1>CapColor</td><td rowspan=1 colspan=1>Cat. #</td><td rowspan=1 colspan=1>Reactions/Tube</td></tr><tr><td rowspan=1 colspan=1>ProFAST+Supermix</td><td rowspan=1 colspan=1>•   Taq DNA polymerase0   Oligonucleotide primers + probes•   Buffer containing dNTPs•  MgCl2 and stabilizers</td><td rowspan=1 colspan=1>515 μL</td><td rowspan=1 colspan=1>Brown</td><td rowspan=1 colspan=1>HSM34</td><td rowspan=1 colspan=1>25(4 tubesprovided)</td></tr><tr><td rowspan=1 colspan=1>M-MLV ReverseTranscriptase II</td><td rowspan=1 colspan=1>•   10 U/μL</td><td rowspan=1 colspan=1>36 μL</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>GLS26</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>RNase Inhibitor II</td><td rowspan=1 colspan=1>•  40 U/μL</td><td rowspan=1 colspan=1>120 μL</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>GLS33</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Influenza ASubtyping RNAControl II(Positive Control)</td><td rowspan=1 colspan=1>•  Non-infectious in vitro transcribedRNA of specific viral sequences</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>Clear</td><td rowspan=1 colspan=1>HCT34</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Internal RNAControl III</td><td rowspan=1 colspan=1>•  Non-infectious in vitro transcribedRNA</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>GCT12</td><td rowspan=1 colspan=1>100</td></tr></table>

# Materials Required But Not Provided

# Plasticware and consumables

Polyester, rayon or nylon tipped nasopharyngeal swabs • RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes • Sterile RNase/DNase-free filter or positive displacement micropipettor tips MagNA Pure LC System Disposables (Reagent Tubs, Reaction Tips, Tip Trays, Cartridges) or easyMAG System Disposables (Sample Vessels and Tips) Biohit Pipette Tips for use with easyMAG System • Greiner Break Four uncoated plates for use with easyMAG System • Cepheid PCR reaction tubes, $2 5 ~ \mu \mathrm { L }$ • Parafilm $^ { \mathfrak { B } } \mathbf { M }$ or MagNA Pure LC Cartridge Seals

# Reagents

Roche MagNA Pure LC Total Nucleic Acid Isolation Kit (Roche Cat. No.03038505001) for 192 isolations or bioMérieux NucliSENS easyMAG reagents (Buffer 1 Cat. No. 280130, Buffer 2 Cat. No. 280131, Buffer 3 Cat. No. 280132, Magnetic Silica Cat. No. 280133, and Lysis Buffer Cat. No. 280134) Micro TestTM ${ \bf M } 4 ^ { \mathbb { B } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12500), Micro TestTM $\mathbf { M } \boldsymbol { 5 } ^ {  { \textregistered } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12515), Micro TestTM

$\mathbf { M } 6 ^ { \mathbf { \mathfrak { B } } }$ Viral Transport Medium (Remel, Inc. Cat. No. R12530), Micro TestTM M4RT® Viral Transport Medium (Remel, Inc. Cat. No. R12505), Copan Universal Transport Medium (Copan Cat. No. 330C), or BD Universal Viral Transport vial, 3mL (Becton, Dickinson and Co. Cat. # 220220)   
Molecular Grade Water (RNase/DNase Free)   
Extraction Control (recommended, e.g. previously characterized positive sample or negative sample spiked with a well characterized seasonal Influenza A/H1, A/H3 or 2009 H1N1 Influenza Virus strain)

# Equipment

• - $\cdot 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ Freezer Roche MagNA Pure LC System with software version 3.0.11 or bioMérieux NucliSENS easyMAG System with Software version 1.0.1 or 2.0   
• Biohit multi-channel pipettor for use with easyMAG System Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b, 3.0a or 3.0b   
• Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { \cdot } 1 0 0 0 \mu \mathrm { L } )$ Mini-centrifuge with adapter for Cepheid Reaction Tubes Cepheid cooling block

# Interpretation of Sample Results

The SmartCycler Dx software automatically determines the specimen results. The interpretation of the assay specimen results is as follows:

<table><tr><td colspan="1" rowspan="1">Sample ID1</td><td colspan="1" rowspan="1">AssayResult</td><td colspan="1" rowspan="1">ICResult</td><td colspan="1" rowspan="1">Warning /ErrorCode</td><td colspan="1" rowspan="1">SeasonalA/H1Result</td><td colspan="1" rowspan="1">SeasonalA/H3Result</td><td colspan="1" rowspan="1">2009H1N1Result</td><td colspan="1" rowspan="1">Interpretation of Results</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Pass</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Seasonal Influenza A/ H1,Seasonal Influenza A/H3 and2009 H1N1 Influenza RNA notdetected. A negative ProFAST+Assay result is a presumptivenegative result for Influenza A.These results should beconfirmed by an FDA clearedInfluenza A NAT assay.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Seasonal Influenza A/ H1 RNAdetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Seasonal Influenza A/ H3 RNAdetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">2009 H1N1 Influenza RNAdetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Unresolved</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Unresolved - PCR inhibition orreagent failure. Repeat testingfrom the purified nucleic acid,</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">re-test from original sample, orcollect and test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">Seasonal Influenza A/H1 and2009 H1N1 Influenza RNAsdetected. Multiple infectionsare rare. Repeat testing from thepurified nucleic acid or re-testfrom original sample, or collectand test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Seasonal Influenza A/H1 andA/H3 RNAs detected. Multipleinfections are rare. Repeattesting from the purified nucleicacid, re-test from originalsample, or collect and test a newsample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">Seasonal Influenza A/H3 and2009 H1N1 Influenza RNAsdetected. Multiple infectionsare rare. Repeat testing from thepurified nucleic acid, re-testfrom original sample, or collectand test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">Seasonal Influenza A/H1, A/H3and 2009 H1N1 InfluenzaRNAs detected. Multipleinfections are rare. Repeattesting from the purified nucleicacid, re-test from originalsample, or collect and test a newsample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">40982</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Not Determined  error code4098</td></tr></table>

1 Columns and data not used for interpretation are not included 2 An Invalid Assay run will display Error code 4098 \* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High viral load can lead to reduced or absent Internal Control signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

• xTag RVP (Respiratory Virus Panel), Luminex Molecular Diagnostics, Inc. FOCUS Simplexa Influenza A H1N1 (2009) CDC Human Influenza Virus Real Time RT-PCR Detection and Characterization Panel

2. Predicate K number(s): (K063765, K081483, K091667), (K100148), (K080570)

3. Comparison with predicates:

<table><tr><td rowspan=1 colspan=1>Features</td><td rowspan=1 colspan=1>ProFAST+ Assay</td><td rowspan=1 colspan=1>Focus SimplexaInfluenza A H1N1(2009)</td><td rowspan=1 colspan=1>CDC Human InfluenzaVirus Real Time RT-PCR Detection andCharacterization Panel</td><td rowspan=1 colspan=1>Luminex xTAG RVP</td></tr><tr><td rowspan=1 colspan=1>510(k)</td><td rowspan=1 colspan=1>K101855</td><td rowspan=1 colspan=1>K100148</td><td rowspan=1 colspan=1>K080570</td><td rowspan=1 colspan=1>K063765, K081483,K091667</td></tr><tr><td rowspan=1 colspan=1>Regulation</td><td rowspan=1 colspan=1>866.3332</td><td rowspan=1 colspan=1>866.3332</td><td rowspan=1 colspan=1>866.3980</td><td rowspan=1 colspan=1>866.3980</td></tr><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>OQW</td><td rowspan=1 colspan=1>OQW</td><td rowspan=1 colspan=1>NXD, OEP, OCC, NSU</td><td rowspan=1 colspan=1>OCC, OEM, OEP</td></tr><tr><td rowspan=1 colspan=1>Device Class</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>Class II</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Differential qualitativedetection of influenza Avirus types seasonal H1,seasonal H3, and 2009H1N1 Influenza viralnucleic acids.</td><td rowspan=1 colspan=1>For the in vitroqualitative detectionand differentiation ofinfluenza A and 2009H1N1 Influenza viralRNA.</td><td rowspan=1 colspan=1>Qualitative in-vitrodetection of influenzaA/H1, A/H3, A/H5(Asian lineage), andinfluenza B viral RNA.</td><td rowspan=1 colspan=1>Direct and differentialqualitative detection ofinfluenza types A and B,RSV types A and B,Parainfluenza types 1, 2and 3, Adenovirus andRhinovirus viral nucleicacids.</td></tr><tr><td rowspan=1 colspan=1>Technology/Detection</td><td rowspan=1 colspan=1>Real Time RT-PCRDetection</td><td rowspan=1 colspan=1>Real Time RT-PCRDetection</td><td rowspan=1 colspan=1>Real Time RT-PCRDetection</td><td rowspan=1 colspan=1>RT-PCRDetection:Amplified products arecoupled to microspheresand detected usingspectrofluorometricanalysis.</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>NP swabs</td><td rowspan=1 colspan=1>Nasopharyngeal swabs(NPS), nasal swabs(NS), andnasopharyngealaspirates (NPA)</td><td rowspan=1 colspan=1>nasopharyngeal and/ornasal swab specimens</td><td rowspan=1 colspan=1>NP swabs</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidIsolation</td><td rowspan=1 colspan=1>Roche MagNA Pure LCSystem and bioMerieuxNucliSENS easyMAG</td><td rowspan=1 colspan=1>Roche MagNA Pure LCSystem or QiagenQIAamp Viral RNAMini Kit</td><td rowspan=1 colspan=1>Qiagen (QIAamp®ViralRNA MiniKit or RNeasy Mini Kit)or Roche MagNA PureLC System</td><td rowspan=1 colspan=1>NucliSENS® miniMAGextraction Kit(bioMerieux)QIAamp® MiniElut®Virus Spin Kit (Qiagen)</td></tr><tr><td rowspan=1 colspan=1>Instrument/Assay Platform</td><td rowspan=1 colspan=1>Cepheid SmartCycler IISystem</td><td rowspan=1 colspan=1>3M Integrated Cycleras part of theMicrofluidic MolecularSystem.</td><td rowspan=1 colspan=1>ABI 7500 Fast Dx Real-Time PCR instrument</td><td rowspan=1 colspan=1>Luminex 100 or 200</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Influenza positive RNAtranscript control and anInternal RNA controlprovided</td><td rowspan=1 colspan=1>Armored RNA InternalControl (AR IC),No Template Control(NTC), andH1N1 Positive Control(PC) provided.</td><td rowspan=1 colspan=1>Human Specimen Control(HSC),Seasonal Influenza VirusControl (SIVC), andA/H5N1 Positive Control(H5VC) provided.</td><td rowspan=1 colspan=1>Bacteriophage lambdapositive control and E.coli MS2 phage InternalControl —ancillaryreagents not provided</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

• Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens - Draft Guidance for Industry and FDA Staff Guidance for Off-theShelf Software Use in Medical Devices; Final

Guidance for Off-the-Shelf Software Use in Medical Devices; Final Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Draft Guidance for Industry and FDA Reviewers Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable - Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff Draft Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection or Detection and Differentiation of Influenza Viruses In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path - Guidance for Industry and FDA Staff Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline (CLSI EP12-A 2002) Protocols for Determination of Limits of Detection and Limits of Quantitation (CLSI EP-17A 2004) Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline (CLSI MM3-A2 2006) Nucleic Acid Sequencing Methods in Diagnostic Laboratory Medicine; Approved Guideline (CLSI MM9-A 2004) • Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods (CLSI MM13-P 2005) Medical devices – Quality management systems – Requirements for regulatory purpose (ISO 13485:2003) Medical devices - Application of risk management to medical devices (ISO 14971:2007) Stability Testing of In Vitro Diagnostic Reagents (CEN 13640:2002)

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. The ProFAST $^ +$ Assay enables the detection and differentiation of seasonal Influenza A/H1, seasonal Influenza A/H3, 2009 H1N1 Influenza and an Internal Control and is based on three processes: nucleic acid isolation, reverse transcription (RT) and Real Time polymerase chain reaction (PCR) amplification/detection. The Internal Control (IC) is added to each nasopharyngeal (NP) swab from symptomatic patients to monitor for inhibitors present. Nucleic acids are isolated and purified from the NP swab sample. Purified nucleic acid is added to the ProFAST $^ +$ Supermix (includes Taq polymerase) along with the appropriate enzymes. The ProFAST $^ +$ Supermix contains oligonucleotide primers complementary to highly conserved regions of the hemagglutinin genes for seasonal Influenza A/H1, seasonal Influenza A/H3, 2009 H1N1 Influenza and target-specific oligonucleotide probes duallabeled with a reporter dye attached to the $5 ^ { \circ }$ end and a quencher dye attached to the $3 ^ { \circ }$ end. After initial reverse transcription of RNA into complementary DNA (cDNA),

amplification proceeds during which the primers and probe anneal specifically to the template (if present) followed by primer extension and amplification. The ProFAST+ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing the fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification product present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility Study

An Inter-Laboratory Reproducibility study was conducted by extracting and testing a panel of 18 samples at three sites (two external sites and at Gen-Probe Prodesse Inc.) performed by two operators at each site for 5 days per operator (for a total of 10 runs per site). The Reproducibility Panel used in the study was prepared by spiking confirmed negative nasopharyngeal (NP) swab pools with cultured and titered stock solutions of seasonal H1 Influenza A (A/H1), seasonal H3 Influenza A (A/H3) and $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ Influenza Virus $\left( { \mathrm { A } } / { 2 0 0 9 } { \mathrm { H } } 1 { \mathrm { N } } 1 \right)$ at low positive $( 2 \mathrm { ~ x ~ L o D } )$ , medium positive $1 0 \mathrm { ~ x ~ L o D ) }$ or negative (0.001 x LoD) concentrations. Each panel included two replicates of each Influenza A subtype at each concentration. This resulted in each Reproducibility Panel consisting of 2 Medium Positives, 2 Low Positives, and 2 Negatives for each of three Influenza A strains $\mathrm { \Delta } _ { \mathrm { n } } = 1 8$ total). Panel members were coded to make 18-sample test panels and to ensure sample identification was unknown to the operators. The 18-sample test panels were stored at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ and shipped to the sites frozen.

A set of ten panels (180 total samples) was provided to each laboratory; one set was used for each of the 5 testing days per operator. Each operator thawed one panel set per day and performed nucleic acid extraction on the panel samples using either the Roche MagNA Pure LC system (Site #1) or the bioMérieux NucliSENS easyMAG (Site $\# 2$ and Site #3), and the extracted nucleic acids were tested with the ProFAST $^ +$ Assay on the Cepheid SmartCycler II. ProFAST $^ +$ Assay controls (Positive, Extraction and Negative) were also included with each panel run. Two lots of ProFAST $^ +$ Supermix were used by each of the three sites. A total of 630 data points were included in the reproducibility study data analysis (21 samples and controls/run X 1 run/day/operator X 2 operators $\mathrm { ~ X ~ } 5$ days X 3 sites $= 6 3 0$ ). The total percent agreement for the ProFAST $^ +$ Assay was $9 9 . 7 \%$ . The ProFAST $^ +$ Assay is a qualitative assay based partially on numerical Cycle Threshold $\mathrm { ( C t ) }$ values. The overall $\mathrm { C t }$ value $\% \mathrm { C V }$ across all sites for all samples and controls ranged from $1 . 0 1 \%$ to $5 . 6 2 \%$ depending upon analyte type, target type, and concentration tested.

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>Panel MemberID</td><td rowspan=2 colspan=1>M ( wo sn)</td><td rowspan=2 colspan=1>M od tom I/E</td><td rowspan=2 colspan=1>e tntMA</td><td rowspan=2 colspan=1>Me c(o1 wo</td><td rowspan=2 colspan=1>Mid o I/</td><td rowspan=2 colspan=1>a  c</td><td rowspan=2 colspan=1>V I 600( co sn )</td><td rowspan=2 colspan=1>A TON IT 6001/</td><td rowspan=2 colspan=1>A n I /E</td><td rowspan=1 colspan=3>ExtractionControl</td><td rowspan=1 colspan=3>Influenza ASubtyping RNAControl</td><td rowspan=2 colspan=1>N t</td><td rowspan=3 colspan=1>Jo % n</td></tr><tr><td rowspan=1 colspan=1>MAI</td><td rowspan=1 colspan=1>MA</td><td rowspan=1 colspan=1>6002/0</td><td rowspan=1 colspan=1>MA</td><td rowspan=1 colspan=1>M</td><td rowspan=1 colspan=1>6007/0</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>0.001X LoD</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.01XLoD</td><td rowspan=1 colspan=1>2X LoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.001X LoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=3>N/A</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=3 colspan=1>Site 1MPa</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=3>10/10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>210/210(100%)</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>29.5</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>3.04</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>0.71</td></tr><tr><td rowspan=3 colspan=1>Site 2EMM</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=3>10/10</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>210/210(100%)</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>28.1</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>26.4</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>3.11</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>2.09</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>0.80</td><td rowspan=1 colspan=1>0.95</td></tr><tr><td rowspan=3 colspan=1>Site 3EMa</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>10/10</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>208/210(99.0%)</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>28.8</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>4.28</td><td rowspan=1 colspan=1>7.26</td><td rowspan=1 colspan=1>1.80</td><td rowspan=1 colspan=1>4.44</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>2.42</td><td rowspan=1 colspan=1>3.02</td><td rowspan=1 colspan=1>2.20</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>4.88</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=2>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>628/630(99.7%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>91.1-99.7%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>91.1-99.7%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=1>94.0 -100%</td><td rowspan=1 colspan=3>88.7% - 100%</td><td rowspan=1 colspan=3>88.7% - 100%</td><td rowspan=1 colspan=1>88.7-100%</td><td rowspan=1 colspan=1>98.8% -99.9%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CtValue</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>28.2</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>4.40</td><td rowspan=1 colspan=1>5.08</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>4.49</td><td rowspan=1 colspan=1>1.59</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>4.11</td><td rowspan=1 colspan=1>2.12</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>3.86</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>5.62</td></tr></table>

a MP $=$ Roche MagNA Pure LC extractor and $\mathrm { E M } =$ bioMérieux NucliSENS easyMAG extractor.

Due to the fact that the clinical trial for the ProFAST $^ +$ Assay resulted in less than $10 \%$ of the clinical samples that were positive by the ProFAST $^ +$ Assay with Ct values beyond the mean $\mathrm { C t }$ values of the LoDs (approximately $3 9 \mathrm { C t }$ for A/H1, 34 $\mathrm { C t }$ for $\mathrm { A } / \mathrm { H } 3$ and $3 2 \mathrm { C t }$ for $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ ), but before the assay cutoff at 45 cycles, a supplemental Inter-Laboratory Reproducibility Study at three sites to evaluate inter-laboratory reproducibility of the ProFAST $^ +$ Assay with samples having a concentration below the assay’s Limit of Detection (LoDs) but above the High Negatives previously tested in the reproducibility study is not necessary. However, the potential limitation of imprecision of testing extremely low levels of targets below the LoDs of the assay should still be addressed by including the following statement in the Limitation section of the ProFAST $^ +$ Assay Instructions for Use “As with all PCR based in vitro diagnostic tests, extremely low levels of target below the LoD of the assay may be detected, but results may not be reproducible.”

b. Linearity/assay reportable range: Not applicable c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

The following controls are provided in the ProFAST $^ +$ Assay kit:

Positive Control (PC): The ProFAST $^ +$ Assay kit contains an Influenza A Subtyping positive RNA control that consists of one pool of three RNA transcripts (specific for each Influenza A subtype: A/H1, A/H3 and A/2009 H1N1). These transcripts are derived from plasmids containing the viral sequences of interest and are not intact virus particles. The Influenza A Subtyping PC contains equal amounts of A/H1, A/H3 and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ . The approximate concentration of each transcript was $5 \mathrm { x } 1 0 ^ { 3 } \mathrm { c o p i e s } / \mu \mathrm { l }$ . The PC does not go through nucleic acid isolation and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance. The PC is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc). Because the PC is assessing global failure and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable range was set to 20.0 to 40.0 in the Clinical Trial Protocol for all Influenza A subtype targets to allow for dilution variability (e.g. pipetting) of the PC by the end users. Of a total of 75 PC tested by the ProFAST $^ +$ Assay during the clinical trial, $9 8 . 7 \%$ (74/75) of these controls gave correct results initially. The average Ct of the PCs tested was 32.35 (Min 31.50- Max 33.40), with 0.34 Standard Deviation (STDEV) and $1 . 0 5 \%$ CV; 30.54 (Min 29.80- Max 31.70), with 0.34 Standard Deviation (STDEV) and $1 . 1 2 \%$ ; and 31.09 (Min 30.10- Max 32.40), with 0.35 Standard Deviation (STDEV) and $1 . 1 4 \%$ CV for A/H1, A/H3, and A/2009H1N1 respectively. In addition, the average $\mathrm { C t }$ for the PC was 32.16, 31.03, and 31.25 (n $= 3 1$ ) during the Reproducibility Study. The clinical study data validated the predetermined PC Ct acceptance range of 20.0 to 40.0.

Internal Control (IC): An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working. The IC is meant to assess global failure (reagent or process) and monitor for PCR inhibition. Because the IC is assessing global failure and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable range for the Internal Control (IC) was set to $2 0 { - } 4 0 \mathrm { C t }$ in the Clinical Trial Protocol to accommodate for dilution variability (e.g., pipetting) by the user in spiking the IC, and also accommodate slight inhibition. The average Ct of all eligible prospective ProFAST $+$ Assay negative clinical specimens was 27.93 (Min 25.0- Max 35.20), with 1.32 Standard Deviation (STDEV) and $4 . 7 3 \%$ CV. The average Ct of all eligible ProFAST $^ +$

Assay Negative Control was 27.36 (Min 25.50- Max 32.70), with 1.12 Standard Deviation (STDEV) and $4 . 0 9 \%$ CV. In addition, the average $\mathrm { C t }$ for the IC was 28.24 (Min 25.80- Max 30.90) with 1.59 standard deviation and $5 . 6 2 \%$ CV during the Reproducibility Study. The clinical study data validated the pre-determined IC Ct acceptance range of 20.0 to 40.0.

Final release specifications were formally established during Test Method Characterization (TMC) then validated against the appropriate analytical performance characteristics of specificity and total imprecision. Final release specifications were developed to ensure First Pass Acceptance (FPA) rates of $\geq$ $9 9 \%$ at lot release and customer rejection rate of $\leq 1 \%$ for a lot released at Acceptable Quality Level (AQL) (i.e. $9 5 \%$ probability of QC lot acceptance at lot release). For customer specifications, the maximum allowable customer lot rejection rate at the Rejectable Quality Level (RQL) Limit (i.e., $10 \%$ probability of QC acceptance at lot release) is $1 \%$ . A false valid risk of $\leq 5 \%$ was used to determine the validity specifications.

QC release specification ranges were set using Operator Characteristics (OC) curve analysis. Lot, technician, instrument, run, and replicate variance components were obtained during test method characterization. Observed Ct means were adjusted for stability effects and specifications determined to meet the validity and specification requirements above.

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Customer CtSpecification</td><td rowspan=1 colspan=1>Kit Control Final ReleaseSpecifications</td></tr><tr><td rowspan=1 colspan=1>PC (A/H1)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>27.18-37.24</td></tr><tr><td rowspan=1 colspan=1>PC (A/H3)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>26.92-36.22</td></tr><tr><td rowspan=1 colspan=1>PC (A/2009 H1N1)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>27.20-36.56</td></tr><tr><td rowspan=1 colspan=1>Internal Control (IC)</td><td rowspan=1 colspan=1>20-40</td><td rowspan=1 colspan=1>24.20-31.16</td></tr></table>

The following controls are not provided in the $\mathrm { P r o F A S T ^ { + } }$ Assay kit, but are required or recommended and described in the ProFAST $^ +$ Assay Instructions for Use:

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the ProFAST $^ +$ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination. Of a total of $7 8 \mathrm { N C }$ tested by the ProParaflu $^ +$ Assay during the clinical trial, $9 7 . 4 \%$ (76/78) of these controls gave correct results initially. The average $\mathrm { C t }$ of the IC in tested NC was 27.36 (Min 25.50- Max 32.70), with 1.12 Standard Deviation (STDEV) and $4 . 0 9 \%$ CV.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a pool of seasonal A/H1N1, seasonal A/H3N2 and 2009 H1N1 influenza viruses was included with each nucleic acid isolation run. The EC served to monitor for lysis during nucleic acid isolation. Of a total of 77 EC tested by the ProFAST $^ +$ Assay during the clinical trial, $100 \%$ (77/77) of these controls gave correct results initially. The average $\mathrm { C t }$ of the PCs tested was 28.80 (Min 27.60- Max 30.10), with 0.66 Standard Deviation (STDEV) and $2 . 2 8 \%$ CV; 26.90 (Min 25.70- Max 28.10), with 0.59 Standard Deviation (STDEV) and $2 . 1 8 \%$ ; and 27.61 (Min 26.30- Max 29.20), with 0.72 Standard Deviation (STDEV) and $2 . 6 2 \%$ CV for A/H1, A/H3, and A/2009H1N1 respectively. The sponsor is recommending an extraction control in each nucleic acid extraction run to the end users in the package insert.

# Freeze/Thaw and Stability

Reagents and Controls stability

An Accelerated Stability study concluded that the ProFAST $^ +$ Supermix, enzymes and controls (closed and open tubes) can be stored at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ for up to 18 months. Real time stability studies are currently ongoing at Gen-Probe Prodesse.

A Freeze/Thaw Study demonstrated that the ProFAST $^ +$ Supermix can undergo up to 5 freeze-thaws. The Controls (Positive and Internal Control) can undergo up to 2 freeze-thaws. For the Controls, the ProFAST $^ +$ Assay Package Insert specifies that controls should not undergo more than 1 freeze-thaw cycle. Although internal studies demonstrated that up to 2 freeze-thaws of controls would not adversely affect performance. To mitigate risk it is recommended they not undergo more than 1 freeze-thaw.

Specimen stability

The ProFAST $^ +$ Assay recommends that samples be stored refrigerated $( 2 ^ { \circ } \mathrm { C } -$ $8 ^ { \circ } \mathrm { C } )$ ) for up to 72 hours prior to processing the samples. This recommendation is supported by 1) The ProFlu $^ +$ Assay (detection of Influenza A, Influenza B and RSV nucleic acid), the Pro $\mathrm { { h M P V + } }$ (detection of human Metapneumovirus (hMPV) nucleic acid), and the ProParaflu+ Assay (detection of human Parainfluenza virus nucleic acid) also recommend that samples be stored refrigerated $( 2 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C } )$ for up to 72 hours prior to processing the samples; 2) The CLSI Guidance: Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (MM13) does not specify requirements for nasopharyngeal (NP) swabs, but does recommend that bronchioalveolar lavage samples be refrigerated for up to 72 hours or frozen (at $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } } )$ ) for longer periods prior to processing for molecular methods; 3) A 72 hour timeframe is widely recommended and used (CLSI Guidance M41 Viral Culture) for viral culture. For viral culture sample stability is important as viable virus is required for growth, however, nucleic acid amplification tests (NAAT) do not require viable virus for detection.

The ProFAST $^ +$ Assay also recommends that extracted nucleic acids that are not run immediately after extraction may be stored at $\mathrm { { \le - 7 0 ^ { \circ } C } }$ before running ProFAST+. Typically, purified nucleic acids may be stored for extended periods of time at $\mathrm { { \le - 7 0 ^ { \circ } C } }$ without marked degradation. Both CLSI guidances MM13 (Collection, Transport, Preparation and Storage of Specimens for Molecular Methods) and MM17 (Verification and Validation of Multiplex Nucleic Acid Assays) recommend storage of isolated RNA at ultralow temperature and limiting the number of freeze-thaw cycles.

# Comparison between Fresh and Frozen Specimens

To determine the potential effects of freezing NP samples and purified nucleic acids may have on the ProFAST $^ +$ Assay, an analytical study was carried out. Contrived samples were generated using aliquots from a negative NP swab pool spiked with one of three Influenza A (IA) strains representing the viruses targeted by the ProFAST $^ +$ Assay which include seasonal H1 (A/H1), seasonal H3 (A/H3), and 2009 H1N1swine-origin (A/2009 H1N1). Fresh IA positive NP swab samples were not available for the study due to low prevalence of influenza during this past winter respiratory season (December 2009 – March 2010). Contrived samples were made at varying concentrations in negative NP swab matrix. For each of the three strains, twenty samples were generated $( \mathrm { n } { = } 6 0 )$ ) in the following manner: 2 samples each at 3 logs, 2 logs, and 1 log above LoD and 7 samples each of $3 \mathbf { x }$ LoD and at LoD (as determined during the Analytical Sensitivity Study). After sample generation, a portion of each sample was freshly extracted on the bioMérieux NucliSENS easyMAG. The remaining portion of the samples were frozen at $\mathrm { \leq } \mathrm { - } 7 0 \mathrm { { } ^ { \circ } C }$ and then extracted again as described above. The nucleic acids were run fresh (immediately subsequent to extraction) in a single replicate. The remaining nucleic acids were frozen at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ and later thawed on ice and subjected to RT-PCR as described above. The following tables summarize the results of the study:

<table><tr><td rowspan=1 colspan=6>Sample Stability Results — % Agreement to the Fresh NP / Fresh NA Data</td></tr><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Concentration</td><td rowspan=1 colspan=4>% Agreement</td></tr><tr><td rowspan=1 colspan=1>FR NP/FRNAa</td><td rowspan=1 colspan=1>FR NP/FZ NAb</td><td rowspan=1 colspan=1>FZ NP/FR NA</td><td rowspan=1 colspan=1>FZ NP/FZ NAd</td></tr><tr><td rowspan=6 colspan=1>AH/V</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2=100%</td><td rowspan=1 colspan=1>2/2= 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2=100%</td><td rowspan=1 colspan=1>2/2= 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td></tr><tr><td rowspan=6 colspan=1>AH/3</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7=100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7= 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td></tr><tr><td rowspan=6 colspan=1>ANTH 6001/N</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2/2=100%</td><td rowspan=1 colspan=1>2/2 = 100%</td><td rowspan=1 colspan=1>2/2 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7/7 = 100%</td><td rowspan=1 colspan=1>7*/7=100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td><td rowspan=1 colspan=1>20/20 = 100%</td></tr></table>

a FR NP/FR NA: Fresh contrived NP samples/Freshly extracted nucleic acids b FR NP/FZ NA: Fresh contrived NP samples/Frozen nucleic acids c FZ NP/FR NA: Frozen contrived NP samples/Fresh nucleic acids $\mathbf { ^ { d } F Z }$ NP/FZ NA: Frozen contrived NP samples/Frozen nucleic acids \* Frozen $( - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C } )$ nucleic acids for these 7 replicates were negative for A/2009 H1N1 initially. They were thawed a second time and retested using the same lot of ProFAST $^ +$ reagent and all 7 replicates were positive for $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ .

<table><tr><td rowspan=1 colspan=8>Mean Ct values and Mean Differences</td></tr><tr><td rowspan=2 colspan=1>Sample ID</td><td rowspan=2 colspan=1>FRNP/FRNA{ Ct</td><td rowspan=2 colspan=1>FRNP/FZNAb Ct</td><td rowspan=2 colspan=1>FZNP/FRNA Ct</td><td rowspan=2 colspan=1>FZNP/FZ$NA ^{ Ct</td><td rowspan=1 colspan=3>Mean Ct Difference to initial FRNP/FR NA Data</td></tr><tr><td rowspan=1 colspan=1>FR NP/FZNA</td><td rowspan=1 colspan=1>FZ NP/FRNA</td><td rowspan=1 colspan=1>FZ NP/FZ NA</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 at LoD</td><td rowspan=1 colspan=1>32.56</td><td rowspan=1 colspan=1>31.96</td><td rowspan=1 colspan=1>31.70</td><td rowspan=1 colspan=1>31.53*</td><td rowspan=1 colspan=1>-0.60</td><td rowspan=1 colspan=1>-0.86</td><td rowspan=1 colspan=1>-1.03*</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 3X LoD</td><td rowspan=1 colspan=1>29.39</td><td rowspan=1 colspan=1>29.53</td><td rowspan=1 colspan=1>29.21</td><td rowspan=1 colspan=1>29.61</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>-0.18</td><td rowspan=1 colspan=1>0.22</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 1 log above LoD</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>27.85</td><td rowspan=1 colspan=1>27.85</td><td rowspan=1 colspan=1>27.95</td><td rowspan=1 colspan=1>-0.35</td><td rowspan=1 colspan=1>-0.35</td><td rowspan=1 colspan=1>-0.25</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 2 logs above LoD</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>24.75</td><td rowspan=1 colspan=1>24.25</td><td rowspan=1 colspan=1>24.35</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>-0.15</td><td rowspan=1 colspan=1>-0.05</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 3 logs above LoD</td><td rowspan=1 colspan=1>21.25</td><td rowspan=1 colspan=1>21.45</td><td rowspan=1 colspan=1>21.2</td><td rowspan=1 colspan=1>21.35</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>A/H1 at LoD</td><td rowspan=1 colspan=1>35.09</td><td rowspan=1 colspan=1>34.43</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>36.56</td><td rowspan=1 colspan=1>-0.66</td><td rowspan=1 colspan=1>-0.79</td><td rowspan=1 colspan=1>1.47</td></tr><tr><td rowspan=1 colspan=1>A/H1 3X LoD</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>31.76</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>31.97</td><td rowspan=1 colspan=1>-0.35</td><td rowspan=1 colspan=1>-0.51</td><td rowspan=1 colspan=1>-0.14</td></tr><tr><td rowspan=1 colspan=1>A/H1 1 log above LoD</td><td rowspan=1 colspan=1>30.35</td><td rowspan=1 colspan=1>30.25</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>-0.10</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>-0.15</td></tr><tr><td rowspan=1 colspan=1>A/H1 2 logs above LoD</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>27.4</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>-0.10</td></tr><tr><td rowspan=1 colspan=1>A/H1 3 logs above LoD</td><td rowspan=1 colspan=1>24.15</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>24.15</td><td rowspan=1 colspan=1>24.05</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-0.10</td></tr><tr><td rowspan=1 colspan=1>A/H3 at LoD</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>32.04</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>-0.16</td><td rowspan=1 colspan=1>-0.09</td><td rowspan=1 colspan=1>-0.06</td></tr><tr><td rowspan=1 colspan=1>A/H3 3X LoD</td><td rowspan=1 colspan=1>30.54</td><td rowspan=1 colspan=1>30.49</td><td rowspan=1 colspan=1>30.61</td><td rowspan=1 colspan=1>30.24</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>-0.30</td></tr><tr><td rowspan=1 colspan=1>A/H3 1 log above LoD</td><td rowspan=1 colspan=1>29.15</td><td rowspan=1 colspan=1>28.95</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>-0.20</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>-0.35</td></tr><tr><td rowspan=1 colspan=1>A/H3 2 logs above LoD</td><td rowspan=1 colspan=1>26.25</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>26.25</td><td rowspan=1 colspan=1>25.95</td><td rowspan=1 colspan=1>-0.25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>-0.30</td></tr><tr><td rowspan=1 colspan=1>A/H3 3 logs above LoD</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>22.65</td><td rowspan=1 colspan=1>22.6</td><td rowspan=1 colspan=1>22.25</td><td rowspan=1 colspan=1>-0.25</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>-0.65</td></tr></table>

a FR NP/FR NA: Fresh contrived NP samples/Freshly extracted nucleic acids b FR NP/FZ NA: Fresh contrived NP samples/Frozen nucleic acids c FZ NP/FR NA: Frozen contrived NP samples/Fresh nucleic acids $\mathbf { ^ { d } F Z }$ NP/FZ NA: Frozen contrived NP samples/Frozen nucleic acids \* Frozen $( - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C } )$ nucleic acids for these 7 replicates were negative for A/2009 H1N1 initially. They were thawed a second time and retested using the same lot of ProFAST+ reagent and all 7 replicates were positive for $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ .

These results show that there is no negative affect due to freezing sample or nucleic acids when the sample and resultant nucleic acids have each been subjected to a freeze-thaw cycle.

d. Analytical Sensitivity (Detection limit):

# LoD estimation studies using the Roche MagNA Pure LC

The analytical sensitivity (limit of detection or LoD) of the ProFAST $^ +$ Assay was estimated using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of viral strains of A/H1, A/H3 and A/2009 H1N1 (H1N1 A/Virginia/1/06; H1N1 A/Hong Kong/2652/06; H3N2 A/Anhui/1239/05; H3N2 A/California/07/04; 2009 H1N1 Clinical Isolate #1; and 2009 H1N1 Clinical Isolate #2) serially diluted in negative nasopharyngeal (NP) pools prepared from leftover NP swab clinical samples. Each viral strain was previously cultured and titered to determine its stock concentration in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Each viral stock was spiked into negative NP swab pools at concentrations of 1 log above, $0 . 5 \log$ above, 1 log below, 0.5 log below, and at the estimated LoD for that virus (based on preliminary studies). Each virus concentration was tested in quintuplicate for a total of 5 data points per virus concentration.

An Internal RNA Control (IC) was spiked into all dilution series samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as reagent, procedural or instrumentation failures. A negative control, which consisted of negative NP swab matrix spiked with IC was included with each set of dilution series. Nucleic acid isolation of the negative control was performed along with the Analytical Sensitivity Panel samples. The negative control served to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The Influenza A Subtyping Positive RNA Control, non-infectious in vitro transcribed RNA of specific A/H1, A/H3, and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ viral sequences, was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

Analytical sensitivity was estimated for each virus strain. The LoD was estimated as the lowest concentration that was detected $2 9 5 \%$ of the time (i.e. concentration at which at least 5 out of 5 replicates were determined to be positive). The data are presented in the following table:

<table><tr><td rowspan=1 colspan=6>Analytical Sensitivity Estimation Results</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageCT</td><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>ReplicatesDetected</td></tr><tr><td rowspan=5 colspan=1>H1N1A/Virginia/1/06</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>30.60</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10</td><td rowspan=1 colspan=1>31.72</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>34.22</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>35.38</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>41.70</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1/5</td></tr><tr><td rowspan=5 colspan=1>H1N1 A/HongKong/2652/06</td><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>31.88</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.68</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>33.24</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>35.78</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-^2</td><td rowspan=1 colspan=1>39.76</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>3.15</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10-2</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=5 colspan=1>H3N2A/Anhui/1239/05</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>28.22</td><td rowspan=1 colspan=1>0.08</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x100</td><td rowspan=1 colspan=1>29.24</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>31.46</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.15</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>34.74</td><td rowspan=1 colspan=1>1.51</td><td rowspan=1 colspan=1>4.34</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=5 colspan=1>H3N2A/California/07/04</td><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>32.18</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>0.86</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>32.78</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>36.54</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>3.80</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10-2</td><td rowspan=1 colspan=1>37.18</td><td rowspan=1 colspan=1>1.84</td><td rowspan=1 colspan=1>4.95</td><td rowspan=1 colspan=1>4/5</td></tr><tr><td rowspan=1 colspan=1>1x10-2</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=5 colspan=1>2009 H1N1 ClinicalIsolate #1</td><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>28.18</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x102</td><td rowspan=1 colspan=1>30.10</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x102</td><td rowspan=1 colspan=1>32.44</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x10&#x27;</td><td rowspan=1 colspan=1>35.18</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td></tr><tr><td rowspan=5 colspan=1>2009 H1N1 ClinicalIsolate #5</td><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>28.50</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x102</td><td rowspan=1 colspan=1>29.30</td><td rowspan=1 colspan=1>0.10</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x102</td><td rowspan=1 colspan=1>32.18</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>5x101</td><td rowspan=1 colspan=1>33.38</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/5</td></tr></table>

Analytical sensitivity was estimated to be $5 \mathrm { x } 1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for H1N1 A/Virginia/1/06, $5 \mathrm { x } \mathrm { i } 0 ^ { - 2 }$ $\Gamma \mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ for H1N1 A/Hong $\mathrm { K o n g } / 2 6 5 2 / 0 6 , 1 \mathrm { x } 1 0 ^ { - 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for H3N2 A/Anhui/1239/05, $\mathrm { 1 x 1 0 ^ { - 1 } }$ $\mathrm { \Delta T I D _ { 5 0 } / m L }$ for H3N2 A/California/07/04, $5 \mathrm { x } 1 0 ^ { 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ A/2009 H1N1 Clinical Isolate #1 and $5 \mathrm { x } 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ A/2009 H1N1 Clinical Isolate #5.

# LoD confirmation studies using the Roche MagNA Pure LC

The “estimated” LoDs were then confirmed by preparing 20 additional replicates (simulated NP swab samples) at each of the proposed LoD concentrations.

An Internal RNA Control (IC) was spiked into all dilution series samples prior to nucleic acid isolation. The IC monitors for PCR inhibition as well as reagent, procedural or instrumentation failures. A negative control, which consisted of negative NP swab matrix spiked with IC was included with each set of dilution series. Nucleic acid isolation of the negative control was performed along with the Analytical Sensitivity Panel samples. The negative control served to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix. The Influenza A Subtyping Positive RNA Control, non-infectious in vitro transcribed RNA of specific A/H1, A/H3, and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ viral sequences, was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

For some of the strains, more than one concentration was included for the confirmation portion of the study, typically the two lowest concentrations that yielded $100 \%$ detection to ensure achievement of $2 9 5 \%$ detection for each strain. For strain H3N2 A/Anhui/1239/05, the lowest concentration tested for the estimation portion of the study resulted in $100 \%$ detection, thus that concentration and the next two lower half-log concentrations were tested in the confirmation portion of the study. Each of the 20 replicates was subjected to the entire test system from sample preparation and extraction to RT-PCR. All samples were extracted using the MagNA Pure LC Instrument. In the event that the initial LoD estimated concentration was not confirmed (i.e. ${ < } 1 9$ replicates were not positive), the LoD confirmation was repeated using the next half-log higher concentration. The data are presented in the following table:

<table><tr><td rowspan=1 colspan=9>Analytical Sensitivity Confirmation Results</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>MeanCT</td><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MinCT</td><td rowspan=1 colspan=1>MaxCt</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Virginia/1/06</td><td rowspan=1 colspan=1>5x10</td><td rowspan=1 colspan=1>35.76</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>H1N1 A/HongKong/2652/06</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>36.21</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5x10-^2</td><td rowspan=1 colspan=1>41.35</td><td rowspan=1 colspan=1>1.80</td><td rowspan=1 colspan=1>4.35</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>44.8</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>H3N2A/Anhui/1239/05</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>34.78</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.39</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5x10-2</td><td rowspan=1 colspan=1>36.51</td><td rowspan=1 colspan=1>0.89</td><td rowspan=1 colspan=1>2.45</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>1x10-3</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>5%</td></tr><tr><td rowspan=2 colspan=1>H3N2A/California/07/04</td><td rowspan=1 colspan=1>5x10-</td><td rowspan=1 colspan=1>33.17</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.23</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>37.16</td><td rowspan=1 colspan=1>1.60</td><td rowspan=1 colspan=1>4.30</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=2 colspan=1>2009 H1N1 ClinicalIsolate #1</td><td rowspan=1 colspan=1>1x10^2</td><td rowspan=1 colspan=1>32.86</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>5x101</td><td rowspan=1 colspan=1>31.80</td><td rowspan=1 colspan=1>4.24</td><td rowspan=1 colspan=1>13.34</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=2 colspan=1>2009 H1N1 ClinicalIsolate #5</td><td rowspan=1 colspan=1>1x10^{2}$</td><td rowspan=1 colspan=1>31.92</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5x101</td><td rowspan=1 colspan=1>33.98</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.75</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90%</td></tr></table>

The sponsor confirmed LoDs are presented in the following table:   

<table><tr><td rowspan=1 colspan=3>ProFAST+ Analytical Sensitivity - Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=2>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Virginia/1/06</td><td rowspan=1 colspan=2>5x10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Hong Kong/2652/06</td><td rowspan=1 colspan=2>5x10-2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/Anhui/1239/05</td><td rowspan=1 colspan=1>1x10-</td><td rowspan=1 colspan=1>1x10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/California/07/04</td><td rowspan=1 colspan=1>5x10-</td><td rowspan=1 colspan=1>5x10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #1</td><td rowspan=1 colspan=2>1x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #5</td><td rowspan=1 colspan=2>1x102 TCID50/mL</td></tr></table>

# Extraction Equivalency Study

To evaluate the equivalency of nucleic acid extraction of the bioMérieux NucliSENS easyMAG to the Roche MagNA Pure LC for use with the ProFAST+ Assay, an analytical study was carried out to determine “equivalency” by confirming the LoDs on the easyMAG extractor for each of the three (Seasonal A/H1, Seasonal A/H3 and A/2009 H1N1 Influenza) subtypes of Influenza A used in the Analytical Sensitivity Study (determined and confirmed using the MagNA Pure LC). Both instruments were also used throughout assay development, Clinical Trials and Reproducibility.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles.

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads.

One operator generated dilution series for each viral strain and spiked each diluted virus into the same negative nasopharyngeal (NP) swab pool used for the ProFAST $^ +$ Analytical Sensitivity Study at the LoD concentration confirmed in that study. Twenty independent data points from the specific (LoD) viral dilution were generated by performing 20 independent extractions on the easyMAG; each extracted sample was run in a single replicate on the Cepheid SmartCycler II. The two extraction instruments were considered equivalent if the detection rate using the easyMAG at the LoD was not statistically different (at significance level of $\mathrm { p } { < } 0 . 0 5 )$ than the rate obtained using the MagNA Pure instrument. Twenty of twenty replicates were positive using the MagNA Pure for 5 of the 6 strains tested and therefore, 18 replicates were required to test positive using the easyMAG $\scriptstyle ( \mathtt { p } ^ { = } 0 . 0 7 3 4 )$ to claim equivalency. The Prodesse 2009 H1N1 clinical isolate #1 strain only tested at $9 5 \%$ positive (19/20) for the LoD using the MagNA Pure and therefore, for that particular strain, 16 of 20 were required to test positive $\scriptstyle ( 1 9 ^ { = } 0 . 0 7 5 7$ ) to claim equivalency.

The Internal RNA Control II (IC) was spiked into all samples prior to nucleic acid isolation. The IC monitors for RT-PCR inhibition as well as any reagent, procedural or instrumentation failure. The Influenza A Subtyping RNA Control II (PC) was included with each RT-PCR run to test for global errors (absence of reagent, instrument failure, etc.). The Influenza PC does not require nucleic acid isolation and was diluted just prior to set up of RT-PCR reactions. A Negative Control which consisted of Viral Transport Media (VTM) spiked with the IC was included for each extraction run. The Negative Control serves to monitor for contamination during the testing procedure or the presence of inhibitors in the sample matrix.

NucliSENS easyMAG LoD data are presented in the following table. The data from the Analytical Sensitivity Study using the MagNA Pure LC as the extraction instrument is presented in the table alongside the easyMAG data for comparison. :

<table><tr><td rowspan=1 colspan=8>LoD Summary for both MagNA Pure LC (MP)* and easyMAG (EM)</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Extractor</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>AverageCT</td><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Replicates Detected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=2 colspan=1>H1N1A/Virginia/1/06</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>35.76</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>36.75</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>3.39</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>H1N1 A/HongKong/2652/06</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>41.35</td><td rowspan=1 colspan=1>1.80</td><td rowspan=1 colspan=1>4.35</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>40.68</td><td rowspan=1 colspan=1>2.36</td><td rowspan=1 colspan=1>5.80</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>H3N2A/Anhui/1239/05</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>34.78</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.39</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>34.49</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.24</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>H3N2A/California/07/04</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.17</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.23</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>32.47</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>32.86</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Clinical Isolate #1</td><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>32.16</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>3.95</td><td rowspan=1 colspan=1>16*/20</td><td rowspan=1 colspan=1>80%*</td></tr><tr><td rowspan=2 colspan=1>2009 H1N1Clinical Isolate #5</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>31.92</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>EM</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>31.42</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

† The MagNA Pure LC data was previously generated during the ProFAST+ Analytical Sensitivity Study. \*Although $< 9 5 \%$ was detected, the difference is not statistically significant using the easyMAG as compared to extraction with the MagNA Pure LC.

These data demonstrated that the easyMAG extractor is not statistically different than the MagNA Pure LC instrument when testing LoD concentrations and is therefore equivalent to the MagNA Pure LC extractor. Moreover, any performance claims made using the MagNA Pure with the ProFAST $^ +$ Assay should be achievable using the easyMAG. Clinical trial and reproducibility studies also supported comparable performance of both automated extraction systems.

# e. Analytical Reactivity:

The analytical reactivity of the ProFAST $^ +$ Assay was evaluated using multiple strains of Influenza A: 14 strains of seasonal H1, 14 strains of seasonal H3, 4 strains of 2009 H1N1. Viral strains were selected to include strains from 1933 to the present, as well as from various geographical locations, to incorporate the genetic variation that may be encountered by ProFAST+ users:

<table><tr><td rowspan=1 colspan=3>ProFAST+ Reactivity Panel List</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Concentration tested(2xLoD)</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Henan/8/05</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Fuijan/156/00</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Brazil/1137/99</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Kentucky/2/06</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/01</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/12/99</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/03/06</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Jiangxi/160/05</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>VR 1520 A/WS/33</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>5.62x104 TCID50/mL **</td></tr><tr><td rowspan=1 colspan=1>VR 98 A1/Mal/302/54</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>1.78x106 CEID50/mL **</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>2.0x106 TCID50/mL *</td></tr><tr><td rowspan=1 colspan=1>VR 546 A1/Denver/1/57</td><td rowspan=1 colspan=1>Seasonal H1</td><td rowspan=1 colspan=1>1.78x106 CEID50/mL **</td></tr><tr><td rowspan=1 colspan=1>A/Hiroshima/52/05</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/512/05</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>VR 822 A/Victoria/3/75</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brazil/02/99</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x103 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/New York/55/2004</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/2831/05</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Bahamas/2686/99</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Fuijan/411/02</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Kentucky/03/06</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Costa Rica/07/99</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/218/06</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x101 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>VR 544 A/Hong Kong/8/68</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>VR 547 A/Aichi/2/68</td><td rowspan=1 colspan=1>Seasonal H3</td><td rowspan=1 colspan=1>2x102 CEID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #2</td><td rowspan=1 colspan=1>Swine-Origin Influenza</td><td rowspan=1 colspan=1>2x103 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #3</td><td rowspan=1 colspan=1>Swine-Origin Influenza</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #4</td><td rowspan=1 colspan=1>Swine-Origin Influenza</td><td rowspan=1 colspan=1>2x10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 A/California/04/2009</td><td rowspan=1 colspan=1>Swine-Origin Influenza</td><td rowspan=1 colspan=1>2x102 TCID50/mL</td></tr></table>

\* Strains presumed to be non-reactive based on preliminary testing; all were tested at the highest concentration available for each strain. †Strains not re-cultured and titered. The original culture/titer from ATCC was used in this study.

A preliminary Limit of Detection (LoD) was established for most of the strains during pre-verification studies. Four of the $\mathrm { A } / \mathrm { H } 1$ strains appear non-reactive with the ProFAST $^ +$ Assay and do not have a preliminary LoD. The strains used in the study were tested at $2 \mathbf { x }$ LoD (as determined in pre-verification studies or at the highest concentration possible for strains that were non-reactive). One sample

was generated for each strain by spiking cultured and titered virus into aliquots of a negative nasopharyngeal (NP) swab pool. Samples were spiked with Internal Control before extraction on the MagNA Pure LC. Each sample was tested in triplicate on the Cepheid SmartCycler II.

A Negative Control, which consisted of IC spiked into VTM, was included for each set of extraction runs. The Negative Control serves to monitor for contamination during the testing procedure. The Influenza A Subtyping Positive RNA Control, non-infectious in vitro transcribed RNA of specific A/H1, A/H3, and A/2009 H1N1 viral sequences, was included with each RT-PCR run to test for procedural errors (absence of reagent, instrument failure, etc.).

The majority of strains analyzed in this study tested positive by the ProFAST+ Assay as expected. The strains that tested negative during pre-verification studies and were not expected to be reactive towards the assay remained non-reactive. Mean Cts and standard deviations for reactive strains were calculated:

<table><tr><td colspan="6" rowspan="1">ProFAST+ Analytical Reactivity Panel Results</td></tr><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Target</td><td colspan="1" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">A/H1DetectionMeanCt±StandardDeviation</td><td colspan="1" rowspan="1">A/H3DetectionMeanCt±StandardDeviation</td><td colspan="1" rowspan="1">A/2009 H1N1DetectionMeanCt±StandardDeviation</td></tr><tr><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">37.63 ± 0.91</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Henan/8/05</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">36.37 ± 0.61</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Fuijan/156/00</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x101 TCID50/mL</td><td colspan="1" rowspan="1">38.67 ± 0.21</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Brazil/1137/99</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x105 TCID50/mL</td><td colspan="1" rowspan="1">34.10 ± 0.44</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Kentucky/2/06</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">32.10 ± 0.10</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/01</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">35.70 ± 0.85</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/12/99</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">38.40 ± 1.42</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/2007</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">33.97 ± 0.45</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/03/06</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">33.47 ± 0.25</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Jiangxi/160/05</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">37.97 ± 1.42</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VR 1520 A/WS/33</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">5.62x104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VR 98 A1/Mal/302/54</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">1.78x106 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">2.0x106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VR 546 A1/Denver/1/57</td><td colspan="1" rowspan="1">A/H1</td><td colspan="1" rowspan="1">1.78x106 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Hiroshima/52/05</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">31.30 ± 0.00</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Victoria/512/05</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32.70 ± 0.10</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">VR 822 A/Victoria/3/75</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">33.57 ± 0.29</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Brazil/02/99</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">31.30 ± 0.10</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/New York/55/2004</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">32.27 ± 0.12</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/2831/05</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x101 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">34.47 ± 0.72</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x100 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">38.67 ± 1.11</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Bahamas/2686/99</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">34.23 ± 0.15</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Fuijan/411/02</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">31.10 ± 0.20</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Kentucky/03/06</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">30.43 ± 0.15</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Costa Rica/07/99</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">33.30 ± 0.10</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/218/06</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x10 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.80 ± 0.70</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">VR 544 A/Hong Kong/8/68</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">32.30 ± 0.35</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">VR 547 A/Aichi/2/68</td><td colspan="1" rowspan="1">A/H3</td><td colspan="1" rowspan="1">2x102 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">32.93 ± 0.06</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1 Clinical Isolate #2</td><td colspan="1" rowspan="1">S-OIV</td><td colspan="1" rowspan="1">2x103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">27.17 ± 0.12</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1 Clinical Isolate #3</td><td colspan="1" rowspan="1">S-OIV</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">31.90 ± 0.10</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1 Clinical Isolate #4</td><td colspan="1" rowspan="1">S-OIV</td><td colspan="1" rowspan="1">2x101 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">33.67 ± 0.55</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1A/California/04/2009</td><td colspan="1" rowspan="1">S-OIV</td><td colspan="1" rowspan="1">2x102 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">28.47 ± 0.25</td></tr></table>

There were four strains that were not reactive with the ProFAST $^ +$ Assay. These results were expected, as the strains were not reactive with the ProFAST $^ +$ Assay during pre-verification work. All four strains are reactive with the FDA cleared Prodesse ProFlu $^ +$ Assay (targeting the matrix gene) validating that the nonreactivity with the $\mathrm { P r o F A S T ^ { + } }$ Assay is not due to nucleic acid degradation. It was not surprising that the older $\mathrm { A } / \mathrm { H } 1$ strains did not react with ProFAST $^ +$ as the design emphasized more contemporary strains. It is difficult to find regions of the hemagglutinin gene that are conserved over time considering the mutability of the virus. The only sequence available for these older strains was for VR 546 A1/Denver/1/57 which showed several mismatches to the primers. However, the issue of the ProFAST $^ +$ assay may not react with non-contemporary influenza A strains should be addressed in the product package insert as the following: “The ProFAST+ Assay may not react with non-contemporary influenza A strains”

# f. Analytical specificity:

The analytical specificity of the ProFAST $^ +$ Assay was evaluated by testing a panel of 38 viruses (including the three targeted Influenza A viruses, four influenza viruses of swine origin, and two H5N1 strains), 25 bacteria, and 1 yeast strain representing common respiratory pathogens or flora commonly present in the nasopharynx. Bacteria and yeast were tested at concentrations of $1 0 ^ { 6 }$ to $1 0 ^ { 7 }$ CFU/mL. Viruses were tested at concentrations of $1 0 ^ { 3 }$ to $1 0 ^ { 6 }$ T $\mathrm { C I D } _ { 5 0 } / \mathrm { m L }$ . Samples were extracted and tested in triplicate.

Aliquots of a negative nasopharygeal (NP) swab pool matrix were spiked with the analytical specificity organisms at concentrations of at $1 . 0 { \times } 1 0 ^ { 3 } - 1 0 ^ { \dot { 6 } } \mathrm { T C I D } _ { 5 0 } / \mathrm { m l }$ (except Epstein Barr Virus, which was tested at1 $0 ^ { 8 }$ copies/mL) and the bacterial and yeast strains at $1 0 ^ { 6 } - 1 0 ^ { 7 } \mathrm { C F U / m l }$ , all of which are clinically relevant concentrations. Cultured and titered strains of Influenza A HIN1 (A/H1), Influenza A H3N2 $( \mathrm { A } / \mathrm { H } 3 )$ , and 2009 H1N1 Influenza A (A/2009 H1N1) were also included in the panel and tested at 2 logs above their respective Limit of Detection, and served as Extraction Controls for each target channel:

<table><tr><td rowspan=1 colspan=4>Analytical Specificity Panel Samples</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Cultured By</td><td rowspan=1 colspan=1>Original Source</td></tr><tr><td rowspan=1 colspan=4>Virus Strains</td></tr><tr><td rowspan=1 colspan=1>A/H1 A/Henan/8/05</td><td rowspan=1 colspan=1>102 TCID50/mL</td><td rowspan=1 colspan=1>TriCore</td><td rowspan=1 colspan=1>CDC, Atlanta, GA</td></tr><tr><td rowspan=1 colspan=1>A/H3 A/California/07/04</td><td rowspan=1 colspan=1>101 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>University ofWisconsin Madison,Madison, WI</td></tr><tr><td rowspan=1 colspan=1>A/2009 H1N1 Prodesse NovelH1N1 clinical isolate #2</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>WMH</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1/Adenoid 71</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 7</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B4</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Coxsackie B5/10/2006</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Echovirus 2</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Echovirus 3</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 68</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 70</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Ebstein Barr Virus</td><td rowspan=1 colspan=1>108 copies/mL</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>Zeptometrix</td></tr><tr><td rowspan=1 colspan=1>HSV Type 1 MacIntyre Strain</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>HSV Type 2 G strain</td><td rowspan=1 colspan=1>105 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A2</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Zeptometrix</td><td rowspan=1 colspan=1>University of Iowa,Iowa City, IA</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus 1a</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Influenza B/Wisconsin</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>CDC, Atlanta, GA</td></tr><tr><td rowspan=1 colspan=1>Measles/7/2000</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Mumps Virus</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 1</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 2</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 3</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Type 4</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Poliovirus 1</td><td rowspan=1 colspan=1>106 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>RSV A Strain Long</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>RSV B Strain Wash</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=2>Bacteria Strains</td><td rowspan=1 colspan=2>canmaGellow</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchoiseptica</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumonia</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium intracellulare</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>Gen-Probe, San</td><td rowspan=1 colspan=1>ATCC</td></tr></table>

<table><tr><td rowspan=1 colspan=5>Analytical Specificity Panel Samples</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>ConcentrationTested</td><td rowspan=1 colspan=1>Cultured By</td><td rowspan=1 colspan=1>Original Source</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Diego, CA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumonia</td><td rowspan=1 colspan=2>106 cfu/mL</td><td rowspan=1 colspan=1>Gen-Probe, SanDiego, CA</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Neisseria mucosa</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumonia</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium diptheriae</td><td rowspan=1 colspan=2>106 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=2>106 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>107 cfu/mL</td><td rowspan=1 colspan=1>RMC</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=2>106 cfu/mL</td><td rowspan=1 colspan=1>Microbiologics</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=5>Yeast Strain</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=2>107 cfu/mL</td><td rowspan=1 colspan=1>WMH</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=5>Influenza Virus of Swine Origin</td></tr><tr><td rowspan=1 colspan=1>VR 897 A/New Jersey/8/76</td><td rowspan=1 colspan=2>2x104 TCID50/mL</td><td rowspan=1 colspan=1>ND*</td><td rowspan=1 colspan=1>ATCC</td></tr><tr><td rowspan=1 colspan=1>A/South Dakota/03/2008</td><td rowspan=1 colspan=2>2x10³ TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>CDC</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/10/1998</td><td rowspan=1 colspan=2>2x103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>CDC</td></tr><tr><td rowspan=1 colspan=1>A/Iowa/2006</td><td rowspan=1 colspan=2>2x103 TCID50/mL</td><td rowspan=1 colspan=1>Tricore</td><td rowspan=1 colspan=1>CDC</td></tr><tr><td rowspan=1 colspan=5>Influenza A Virus of Avian Origin</td></tr><tr><td rowspan=1 colspan=1>H5N1/VN/1203</td><td rowspan=1 colspan=2>2.7ng/μL RNA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>University ofWisconsin, Madison,WI</td></tr><tr><td rowspan=1 colspan=1>H5N1/HK/486</td><td rowspan=1 colspan=2>1.4ng/μL RNA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>University ofWisconsin, Madison,WI</td></tr></table>

\* This strain was not recultured and titered. The ATCC provided titer was used and the strain was tested at as high a concentration as possible. Tricore $=$ TriCore Reference Laboratories, Albuquerque, NM Zeptometrix – Zeptometrix, Inc., Buffalo, NY Microbiologics $=$ MicroBioLogics, Inc., St. Cloud, MN RMC $=$ Resurrection Medical Center, Chicago, IL $\mathrm { W M H } =$ Waukesha Memorial Hospital, Waukesha, WI ATCC $=$ American Tissue Culture Collection, Manassas, VA $\mathrm { N D } = \mathrm { N o t }$ Done

A Negative Control, which consisted of the M4 viral transport media spiked with the IC, was included with Analytical Specificity Panel samples. The Influenza A Subtyping Positive Control and was included with each detection run to test for global failure (absence of reagents, instrument failure, etc.). The Analytical

Specificity Panel samples (except for the four influenza viruses of swine origin and the two H5N1 strains, which were extracted on the MagNA Pure LC) and the Negative Control were extracted on the bioMériuex NucliSENS easyMAG and tested in triplicate on a Cepheid Smartcycler II using one lot of ProFAST+ reagents.

The ProFAST $^ +$ Assay did not cross-react with any of the Analytical Specificity Panel samples tested, except for two of the four influenza viruses of swine origin. Detailed Analytical Specificity results are presented in the following table:

<table><tr><td colspan="6" rowspan="1">Analytical Specificity Results</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="2" rowspan="1">ConcentrationTested</td><td colspan="1" rowspan="1">A/H1DetectionMeanCt/StandardDeviation</td><td colspan="1" rowspan="1">A/H3DetectionMeanCt/StandardDeviation</td><td colspan="1" rowspan="1">A/2009 H1N1DetectionMeanCt/StandardDeviation</td></tr><tr><td colspan="1" rowspan="1">A/H1</td><td colspan="2" rowspan="1">102 TCID50/mL</td><td colspan="1" rowspan="1">28.60/0.17</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/H3</td><td colspan="2" rowspan="1">101 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">28.67/0.12</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/2009 H1N1</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">20.57/0.15</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid 71</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B5/10/2006</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10° copies/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyreStrain</td><td colspan="2" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human MetapneumovirusA2</td><td colspan="2" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 1a</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="2" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Poliovirus 1</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetellabronchoiseptica</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumonia</td><td colspan="2" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumintracellulare</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumonia</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumonia</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="2" rowspan="1">106 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="2" rowspan="1">10' cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="2" rowspan="1">107 cfu/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">VR 897 A/NewJersey/8/76</td><td colspan="2" rowspan="1">2x104 TCID50/mL*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">31.57/0.12</td></tr><tr><td colspan="1" rowspan="1">A/South Dakota/03/2008</td><td colspan="2" rowspan="1">2x10³ TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/10/1998</td><td colspan="2" rowspan="1">2x10³3 TCID5o/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">29.43/0.06</td></tr><tr><td colspan="1" rowspan="1">A/Iowa/2006</td><td colspan="2" rowspan="1">2x10³ TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">H5N1/VN/1203</td><td colspan="2" rowspan="1">2.7ng/μL RNA</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">H5N1/HK/486</td><td colspan="2" rowspan="1">1.4ng/μL RNA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr></table>

\* This strain was not recultured and titered. The ATCC provided titer was used and the strain was tested at as high a concentration as possible.

The cross-reactivity observed with two of the four influenza viruses of swine origin is expected, since the target for the $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ detection is the $H A$ gene. It is anticipated that targeting the $H A$ gene may also detect other viruses from the North American Swine lineage. However, swine-origin influenza viruses have

historically rarely surfaced in humans. Based on the literature, from 1958 until the current 2009 H1N1 Influenza outbreak, there have been 64 cases of S-OIV in humans. Fifty-seven of these cases were H1N1, six were characterized as belonging to the H3N2 lineage, and one was a triple-reassortant H1N2. Thirteen of these cases occurred in military personnel during a contained and short-lived (22 days) outbreak at Fort Dix in 1976. Excluding the Fort Dix outbreak, S-OIV cases in humans have averaged 1 to 2 per year and nearly all have been linked to exposure to swine. However, the issue of potential reactivity with other swine origin Influenza A strains should be addressed in the product package insert as the following: “The ProFAST $^ { \prime } +$ 2009H1N1 reagents may also react with other swine origin Influenza A strains. However, literature review indicated that Influenza Virus of Swine Origin have historically rarely surfaced in humans.”

Ideally, seasonal Influenza Viruses (A/H1 and $_ \mathrm { A } / \mathrm { H } 3$ ) and 2009 H1N1 Influenza strains, such as those tested in the analytical reactivity study, should also be tested at high virus titer (e.g., $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ in the analytical specificity study. However, the prospective and retrospective clinical studies included a total of 258 Influenza A positive NPS samples, and no cross-reactivity between seasonal Influenza Viruses ( $\mathrm { \bf A } / \mathrm { \bf H } 1$ and $_ \mathrm { A } / \mathrm { H } 3$ ) and 2009 H1N1 Influenza markers was observed. A substantial number of these Influenza A positive samples represented clinically high viral levels based on $\mathrm { C t }$ values generated from the FDA cleared ProFlu $^ +$ Assay. Therefore, an additional analytical specificity study testing high titers of seasonal Influenza Viruses (A/H1 and A/H3) and 2009 H1N1 Influenza strains is not necessary.

In addition to laboratory testing, bioinformatics resources were used to predict cross reactivity of additional influenza A strains with the seasonal A/H1, seasonal A/H3 and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ targets. A significant number of mismatches are observed for each primer and probe when compared to the sequences of other Influenza A subtypes listed below. Optimal primer/probe annealing temp is $2 ^ { \circ } C - 5 ^ { \circ } C$ below the melt temperature $\mathrm { ( T m ) }$ of the primer/probe. Due to the number of base pair mismatches for each Influenza A strain, the Tm for each analyte primer/probe set is estimated to be ${ > } 2 0 ^ { \mathrm { o } } \mathrm { C }$ below the actual primer/probe Tm. This Tm difference would prevent amplification and/or detection of the subtypes listed below with the ProFAST $^ +$ A/H1, A/H3, or A/2009 H1N1 primer/probe sets. The ProFAST+ Assay primer and probe sequences for each Influenza A subtype, A/H1, A/H3 and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ , are unique and are not expected to cross-react with the other Influenza A subtypes listed in the table below:

<table><tr><td colspan="3" rowspan="1">Simulated Analytical Specificity (sequence matches) with Additional Influenza A Strains</td></tr><tr><td colspan="1" rowspan="1">Influenza A Strain</td><td colspan="1" rowspan="1">GenBank</td><td colspan="1" rowspan="1">Simulated Cross Reactivity</td></tr><tr><td colspan="1" rowspan="1">A/Cambodia/R0405050/2007 (H5N1)</td><td colspan="1" rowspan="1">FJ225472</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/Japanese white-eye/HongKong/1038/2006 (H5N1)</td><td colspan="1" rowspan="1">DQ992842</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/chicken/India/NIV33487/06 (H5N1)</td><td colspan="1" rowspan="1">EF362418</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Vietnam/NCVD-016/2008(H5N1)</td><td colspan="1" rowspan="1">FJ842476</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Yunnan/1251/2003 (H5N1)</td><td colspan="1" rowspan="1">CY028979</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/common magpie/Hong Kong/645/2006(H5N1)</td><td colspan="1" rowspan="1">DQ992839</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/duck/Hunan/795/2002 (H5N1)</td><td colspan="1" rowspan="1">CY028963</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/chicken/Pennsylvania/Sg-00426/2004(H2N2)</td><td colspan="1" rowspan="1">CY036576</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/duck/PA/486/1969 (H6N1)</td><td colspan="1" rowspan="1">EU743286</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/mallard/Korea/GH171/2007 (H7N7)</td><td colspan="1" rowspan="1">FJ750872</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/turkey/Wisconsin/1/1966 (H9N2)</td><td colspan="1" rowspan="1">AB295601</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/swine/Hong Kong/NS857/2001 (H1N2)</td><td colspan="1" rowspan="1">GQ229348</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/swine/Sweden/1021/2009 (H1N2)</td><td colspan="1" rowspan="1">GQ495132</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/swine/Italy/306907/2003 (H1N1)</td><td colspan="1" rowspan="1">GQ175971</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Wisconsin/125/97 (H1N1)</td><td colspan="1" rowspan="1">AF222026</td><td colspan="1" rowspan="1">Negative for A/H1, A/H3 and 2009 H1N1</td></tr></table>

# g. Assay cut-off:

The “cutoff value” represents the fluorescent intensity signal level (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

Cutoff Determination and Confirmation studies were conducted with two operators using two different lots of ProFAST $^ +$ Supermix.

Cutoff values (RFUs) of the ProFAST $^ +$ Assay were determined upon completion of the Cutoff Determination Study which included a training set of 58 negative and simulated positive samples prepared from individual NP swab clinical samples. Specifically, individual samples $( \mathtt { n } = 6 )$ ) were spiked with each type of Influenza A virus (seasonal H1, seasonal H3 and 2009 H1N1) at 3 logs above the preliminary Limit of Detection (LoD) representing high positive NP swab samples. Individual samples $\mathrm { \Delta } _ { \mathrm { n } } = 1 8$ ) were spiked with each of the three Influenza A strains at 2 times the preliminary LoD representing low positive NP swab samples. 34 samples were left unspiked. The high and low positive samples, as well as 30 negative samples were spiked with the Internal Control. Four (4) negative samples were not spiked with the Internal Control. The samples were evenly split into two aliquots to be run by two operators.

Each operator evenly divided their Cutoff Determination Study Panel samples into two series. Samples were extracted using the Roche MagNA Pure LC or the bioMérieux NucliSENS easyMAG. A negative viral transport media control spiked with IC was included with each extraction run. The extracted nucleic acids and the Influenza A Subtyping Positive RNA Control and the IC diluted in water were run using two different lots of ProFAST $+$ Supermix on the Cepheid

SmartCycler II.

The cutoff values were then verified in the Cutoff Confirmation Study against a set of retrospective clinical samples and controls $3 2 \mathrm { A } / \mathrm { H } 1$ positive, 32 A/H3 positive, 32 A/2009 H1N1 positive, 36 Influenza A negative NP swab samples, 18 Influenza A Subtyping Positive RNA Controls, 16 ICs in TM and 4 Negative Controls). The determined cutoff was confirmed if $2 9 0 \%$ sensitivity and specificity was attained for each channel with the above mentioned set of retrospective clinical samples and controls. The Cutoff Confirmation Study Panel was split between two Operators.

Each Operator divided their Cutoff Confirmation Study Panel samples into two series. Internal Control (IC) was spiked into the appropriate samples prior to extraction. Samples were split evenly and extracted on the Roche MagNA Pure LC and the bioMérieux easyMAG. The negative controls (viral transport medium spiked with IC and unspiked) were included with each extraction run. The Influenza A Subtyping Positive RNA Control was included in each RT-PCR run.

In the FAM, TET, TxR and Cy5 channels, the final RFUs for all samples of the positives and negatives demonstrated minimal overlap. For a given detection, a range of possible cutoffs was generated by determining robust lower and upper boundaries of the threshold, based on final RFU that would minimize false positives and false negatives. Population distribution, Quantiles, and ROC curves were used in selecting cutoff thresholds for A/H1, A/H3, A/2009 H1N1 and Internal Control threshold settings.

Acceptable ranges for the potential cutoff were generated in the following manners: 1) ROC curves were generated and analyzed for all channels used with the ProFAST $^ +$ Assay. The threshold ranges that yielded the maximum accuracy were identified. 2) The distributions of the negatives and positives were first assessed for normality. a) If normal, the averages and standard deviations of the final RFU were calculated. For the lower limit of the acceptable range of cutoff, negative samples were analyzed. The lower threshold was defined as the average final RFU plus three standard deviations of the negatives, that is, the point at which greater than $9 9 . 7 \%$ of true negatives are expected to fall below assuming normality. Similarly, the upper limit of the acceptable range of cutoff threshold was determined using positives and was defined as the average final RFU minus three standard deviations. This is the point at which at least $9 9 . 7 \%$ of true positives are expected to be greater than, assuming normality. b) If the distribution was not normal, a Quantile approach was taken. The lower limit of the cutoff threshold range was defined minimally as the $9 9 . 5 \%$ quantile of the negative samples. The upper limit of the cutoff threshold range was defined maximally as the $0 . 5 \%$ quantile of the positive samples. 3) After settings were finalized the retrospective clinical samples were evaluated for sensitivity and specificity.

The cutoffs used in the Cutoff Determination and Confirmation Studies are

presented in the following table:

Cutoffs used in Cutoff Determination and Confirmation Studies   

<table><tr><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=2>EndPt Threshold (RFU)</td></tr><tr><td rowspan=1 colspan=1>Preliminary Cutoffused in CutoffDetermination Study*</td><td rowspan=1 colspan=1>Cutoff used in CutoffConfirmation Study</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>TxR</td><td rowspan=1 colspan=1>A/2009 H1N1</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td></tr></table>

Established during product development

Analysis settings and threshold settings were determined for FAM $\left( \mathrm { \mathbf { A } } / \mathrm { H 1 } \right) = 6 0 $ , TET $( \mathrm { A } / \mathrm { H } 3 ) = 4 0$ , TxRed $( \mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1 ) = 2 0 $ , Cy5 $\mathrm { ( I C ) } = 2 5$ . These settings produced $100 \%$ positive agreement and $100 \%$ negative agreement in the set of retrospective clinical samples used during Cutoff Confirmation and produced $100 \%$ positive and negative agreement for A/H1, $\mathrm { A } / \mathrm { H } 3$ , and IC; and $9 6 . 9 \%$ positive agreement and $100 \%$ negative agreement for $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ , and met all acceptance criteria for the Cutoff Confirmation study. Therefore, cutoff values were confirmed as acceptable for use during the clinical trials and all subsequent analytical studies.

# h. General Assay Analysis Settings:

<table><tr><td rowspan=1 colspan=5>Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=5>1.7b, Dx Software</td></tr><tr><td rowspan=1 colspan=1>Channel:</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Dye Name:</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>TxR</td><td rowspan=1 colspan=1>Cy5</td></tr><tr><td rowspan=1 colspan=1>Target:</td><td rowspan=1 colspan=1>Seasonal A/H1</td><td rowspan=1 colspan=1>Seasonal A/H3</td><td rowspan=1 colspan=1>A/2009 H1N1</td><td rowspan=1 colspan=1>Internal Control</td></tr><tr><td rowspan=1 colspan=1>Usage:</td><td rowspan=1 colspan=1>Assay**</td><td rowspan=1 colspan=1>Assay**</td><td rowspan=1 colspan=1>Assay**</td><td rowspan=1 colspan=1>Internal Control</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis:</td><td rowspan=1 colspan=1>Primary Curve</td><td rowspan=1 colspan=1>Primary Curve</td><td rowspan=1 colspan=1>PrimaryCurve</td><td rowspan=1 colspan=1>PrimaryCurve</td></tr><tr><td rowspan=1 colspan=1>Thresh Setting:</td><td rowspan=1 colspan=1>Manual</td><td rowspan=1 colspan=1>Manual</td><td rowspan=1 colspan=1>Manual</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>Manual Thresh:</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Auto Thresh:</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>Auto Min Cycle:</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Auto Max Cycle:</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Valid Min Cycle:</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>Valid Max Cycle:</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Backgrd Subtract:</td><td rowspan=1 colspan=1>On</td><td rowspan=1 colspan=1>On</td><td rowspan=1 colspan=1>On</td><td rowspan=1 colspan=1>On</td></tr><tr><td rowspan=1 colspan=1>Bkgnd Min Cycle</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Bkgnd Max Cycle</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Boxcar Avg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>EndPt Thresh</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>NC IC %</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>10*</td><td rowspan=1 colspan=1>10*</td></tr><tr><td rowspan=1 colspan=1>IC Delta</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

\*\* Dx 3.0a/3.0b = Target \* $\mathrm { D x } 3 . 0 \mathrm { a } / 3 . 0 \mathrm { b } = \mathrm { N A }$

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors affected the performance of the ProFAST $^ +$ Assay. Blood, mucin, or medications (prescription and overthe-counter) for relief of congestion, sore throat, allergy and asthma symptoms were spiked into simulated Influenza A positive NP swab matrix. A single strain of seasonal H1 influenza A, H1N1 A/Virginia/1/06, was used and spiked into negative NP pools at 2X Limit of Detection (LoD). High clinically relevant amounts of the potential inhibiting substances were added to spiked samples. An Internal Control (IC) was added to samples. Nucleic acid from the samples was extracted with the bioMérieux easyMAG. RT-PCR was performed in triplicate reactions for each sample on the Cepheid SmartCycler II. Any substance in which detection of A/H1 or the Internal RNA Control (IC) was not observed was considered an inhibitor of PCR. A Negative Control, which consisted of M4 viral transport media spiked with an IC, was included with the Interfering Substances Panel samples. The Influenza A Subtyping RNA Positive Control was included with each detection run to test for global errors.

The following table shows the interfering substances used for this study. As described in the justification column, the concentrations spiked directly into samples ranged from approximately $5 \%$ to $10 \%$ of recommended dose for some substances, while others were tested at concentrations reported in scientific literature or in references from other IVD package inserts. The substances consisted of nasal sprays (liquid and powder), ingestible pills and lozenges, injectables, and endogenous substances (blood and mucin).

<table><tr><td colspan="4" rowspan="1">Interfering Substances Test Concentrations</td></tr><tr><td colspan="1" rowspan="1">Substance Name</td><td colspan="1" rowspan="1">Active Ingredient</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Justification</td></tr><tr><td colspan="1" rowspan="1">Mucin (BovineSubmaxillarygland, type I-S)</td><td colspan="1" rowspan="1">Purified mucin protein</td><td colspan="1" rowspan="1">5%, 1%, and 0.1% w/v**</td><td colspan="1" rowspan="1">The amount possiblypresent on an NP swab andthen diluted in ~3.0mL ofVTM*</td></tr><tr><td colspan="1" rowspan="1">Blood (human), EDTAanticoagulated</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">2% (volume/volume)</td><td colspan="1" rowspan="1">Other Respiratory IVD'sPackage Insert*</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine®</td><td colspan="1" rowspan="1">Phenylephrine HCl</td><td colspan="1" rowspan="1">15% (volume/volume)</td><td colspan="1" rowspan="1">10% of total recommendeddose (45μl)*</td></tr><tr><td colspan="1" rowspan="1">Walgreens Original AnefrinNasal Spray</td><td colspan="1" rowspan="1">OxymetazolineHydrochloride</td><td colspan="1" rowspan="1">15% (volume/volume)</td><td colspan="1" rowspan="1">10% of total recommendeddose (45μl)*</td></tr><tr><td colspan="1" rowspan="1">Zicam Homeopathic Non-Drowsy Allergy Relief NoDrip Liquid Nasal gel</td><td colspan="1" rowspan="1">Luffa Operculata,Galphimia Glauca,HistaminumHydrochloricum,Sulphur</td><td colspan="1" rowspan="1">5% (volume/volume)</td><td colspan="1" rowspan="1">10% of total recommendeddose (15μl)*</td></tr><tr><td colspan="1" rowspan="1">Walgreens Saline NasalSpray</td><td colspan="1" rowspan="1">Sodium chloride withpreservatives</td><td colspan="1" rowspan="1">15% (volume/volume) ofdose</td><td colspan="1" rowspan="1">10% of total recommendeddose (45μl)*</td></tr><tr><td colspan="1" rowspan="1">Beconase AQ®</td><td colspan="1" rowspan="1">Beclomethasonedipropionate</td><td colspan="1" rowspan="1">5% volume/volume</td><td colspan="1" rowspan="1">10% of total recommendeddose*</td></tr><tr><td colspan="1" rowspan="1">Walgreens Sore ThroatLozenges, Cherry</td><td colspan="1" rowspan="1">Oralanesthetic/analgesic</td><td colspan="1" rowspan="1">0.62mg/ml; 1/20 drop,crushed; active ingredients:</td><td colspan="1" rowspan="1">5% of total dose*</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0mg/ml benzocaine,1.7mg/ml menthol</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Relenza®</td><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">5mg/ml</td><td colspan="1" rowspan="1">10% of total spray dose*</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4.0μg/ml</td><td colspan="1" rowspan="1">10% of total recommendeddose*</td></tr><tr><td colspan="1" rowspan="1">Bactroban Nasal ointment</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">10mg/ml</td><td colspan="1" rowspan="1">10% of total recommendeddose in Mupirocinointment*</td></tr><tr><td colspan="1" rowspan="1">TamiFlu</td><td colspan="1" rowspan="1">Oseltamivir</td><td colspan="1" rowspan="1">7.5mg/ml</td><td colspan="1" rowspan="1">10% of total recommendeddose*</td></tr></table>

\* Concentration tested is 10% of suggested dose in a 0.3ml sample, or otherwise stated in table above. \*\*Mucin was tested at 3 different concentrations

The Interfering Substances Study results demonstrated that with the exception of mucin at $5 \%$ and $1 \% \mathrm { w / v }$ , the interfering substances used in this study do not inhibit the detection of $\mathrm { A } / \mathrm { H } 1$ nucleic acid using the ProFAST $+$ Assay. Inhibition of $\mathrm { A } / \mathrm { H } 1$ at 2X LoD was observed for mucin at $5 \%$ and $1 \% \mathrm { \ : w / v }$ concentrations. A mucin concentration of $0 . 1 \% \mathrm { w / v }$ did not inhibit the assay. Interfering Substances Study results are presented in the following table:

<table><tr><td rowspan=1 colspan=5>Interfering Substances Test Results</td></tr><tr><td rowspan=1 colspan=1>Interfering Substance</td><td rowspan=1 colspan=1>Mean A/H1 Ct</td><td rowspan=1 colspan=1>Std Dev A/H1</td><td rowspan=1 colspan=1>Mean IC Ct</td><td rowspan=1 colspan=1>Std Dev IC</td></tr><tr><td rowspan=1 colspan=1>Mucin 5%*</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>Mucin 1%*</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>33.80</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Mucin 0.1%</td><td rowspan=1 colspan=1>33.57</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>27.23</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>37.60</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>29.47</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine</td><td rowspan=1 colspan=1>35.20</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>28.07</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Anefrin Nasal Spray</td><td rowspan=1 colspan=1>34.73</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>27.83</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>35.63</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>28.17</td><td rowspan=1 colspan=1>0.06</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Spray</td><td rowspan=1 colspan=1>35.07</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>27.90</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Beconase AQ</td><td rowspan=1 colspan=1>35.63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>28.23</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Chloraseptic  Sore Throat Lozenges</td><td rowspan=1 colspan=1>34.53</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>27.80</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>35.07</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>27.87</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>Tobromycin</td><td rowspan=1 colspan=1>35.57</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.20</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>34.87</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>27.57</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>TamiFlu</td><td rowspan=1 colspan=1>33.60</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.33</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>No Interfering Substance</td><td rowspan=1 colspan=1>34.87</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>28.10</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>Negative NP</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.33</td><td rowspan=1 colspan=1>0.2</td></tr></table>

\*These samples inhibited the detection of A/H1 (both concentrations of mucin) and IC (in the $5 \%$ samples).

The issue of inhibitory effect of mucin at concentrations of $5 \%$ and $1 \%$ w/v should be addressed in the product package insert as the following: “Highly viscous sample may have an inhibitory effect on detection of influenza $A$ subtypes and the Internal RNA Control (IC) near the LoDs using the ProFAST+ Assay”.

Competitive inhibition of the ProFAST $^ +$ Assay due to the presence of the IC was assessed. Simulated samples were tested with and without IC to determine if the presence of the IC inhibited the reaction. Three cultured and titered $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) strains of Influenza A (H1N1 A/Virginia/1/06, H3N2 A/California/07/04, and 2009 H1N1 Clinical Isolate) were used and were spiked into aliquots of a negative nasopharyngeal (NP) swab pool at one log above LoD, at LoD and at one log below the LoD for each strain as determined in the Analytical Sensitivity Study. Each concentration for each strain was prepared two different ways: spiked with the IC and without any Internal Control (IC) added. All dilution series were extracted in duplicate on the bioMérieux NucliSENS easyMAG. A Negative Control, Viral Transport Media (VTM) spiked with IC, was included for each extraction run. The Negative Control serves to monitor for contamination during the testing procedure. The Influenza A Subtyping RNA Control II (Positive Control) was included with each RTPCR run to test for global errors (absence of reagent, instrument failure, etc.). The Positive Control does not require nucleic acid isolation.

The extracted nucleic acids were tested by RT-PCR in quintuplicate using one lot of ProFAST $^ +$ Real Time Assay reagents on the Cepheid SmartCyler II. This resulted in a total of 10 data points for each viral strain at each concentration spiked with IC and 10 data points for each viral strain at each concentration without the IC. The data were analyzed and the effects of competitive inhibition due to the presence of the IC were assessed. Competitive inhibition was to be considered if the difference in the LoD of the samples with and without IC was greater than one log and the mean Ct for concentrations with and without IC were greater than $3 . 3 \mathrm { C T s }$ for all concentrations tested.

IC Interference Study results are presented in the following table:   

<table><tr><td rowspan=2 colspan=2>Strains</td><td rowspan=2 colspan=1>Conc.(TCID50/mL)</td><td rowspan=2 colspan=1>Results</td><td rowspan=1 colspan=3>RNA IC</td><td rowspan=1 colspan=3>No IC</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Avg.CT</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>ΔCT*</td></tr><tr><td rowspan=3 colspan=2>H1N1A/Virginia/1/061x104 TCID50/mL</td><td rowspan=1 colspan=1>5x100</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.25</td><td rowspan=1 colspan=1>0.03</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>30.96</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>0.29</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>39.02</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.90</td><td rowspan=1 colspan=1>37.97</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>1.05</td></tr><tr><td rowspan=1 colspan=1>5x10-2</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=2 colspan=2>H3N2A/California/07/04</td><td rowspan=1 colspan=1>5x100</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.23</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>29.34</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>-0.11</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.24</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>32.37</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>-0.13</td></tr><tr><td rowspan=1 colspan=2>1x1045 TCID50/mL</td><td rowspan=1 colspan=1>5x10-2</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=2>2009 H1N1 ClinicalIsolate #5 1x106.5TCID50/mL</td><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.49</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>27.69</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>-0.20</td></tr><tr><td rowspan=1 colspan=1>1x102</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.85</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.70</td><td rowspan=1 colspan=1>32.28</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.24</td><td rowspan=1 colspan=1>0.57</td></tr><tr><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr></table>

\* The ΔCT column is the average CT data column for the IC samples minus the average CT data column for the No IC samples.

There was no evidence of competitive inhibition at the assay LoDs for detection of A/H1, A/H3 or $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ in the presence IC using the ProFAST $^ +$ Assay. The assay demonstrated equivalent detection of LoD concentrations for three stains of influenza A with mean Ct value differences for each strain and

concentration ranging from -0.2 to 1.05.

# k. Competitive Inhibition Study

Effects of competitive inhibition on the ProFAST $^ +$ Assay when two of the Assay’s target organisms are present in a single sample were not evaluated in a Competitive Inhibition Study. However, multiple infections with different subtypes of influenza A are rare. The prospective and retrospective clinical studies conducted at four geographically diverse U.S. clinical laboratories for this $5 1 0 \mathrm { k }$ submission included a total of 1002 NPS samples, of which 258 were Influenza A positive samples. All of the 258 Influenza A positive samples represented samples positive for only one influenza A subtype, and no true coinfections of seasonal A/H1, A/H3 and A/2009 H1N1 was observed in the clinical studies. Therefore, an analytical Competitive Inhibition Study to assess the potential competition between Influenza A subtypes is not necessary.

# l. Carry-Over Contamination:

To evaluate the degree of carry-over/cross-contamination that occurs with the use of the ProFAST $^ +$ Assay in association with nucleic acid extraction on the Roche MagNA Pure LC and the bioMérieux NucliSens easyMAG instruments and PCR on the Cepheid SmartCycler II thermocycler, an internal Carry-Over study was carried out by testing simulated human Influenza A (2009 H1N1 Influenza A) high positive samples run in series with A/2009 H1N1 high negative samples over 5 runs. The High Positive samples consisted of negative nasopharyngeal (NP ) swab matrix spiked with a concentration 3 logs above the Limit of Detection (LoD) of a cultured and titered strain: Prodesse Influenza A 2009 H1N1 (Swine) Clinical Isolate #3. A maximum $\mathrm { C t }$ value of 23 was targeted for the High Positive samples as a Ct of 23 or lower represented the lower end of the “clinical range” of the ProFAST $^ +$ Assay as seen in development. High Negative samples consisted of negative NP swab matrix spiked with a concentration of A/2009 H1N1 below the analytical LoD concentration (3 logs below LoD) such that $> 9 5 \%$ of samples should be negative. The samples were processed and extracted in a High Positive/High Negative alternating fashion (i.e. checkerboard pattern) on the extraction instruments and likewise processed and run on the Cepheid SmartCycler II instrument in an alternating fashion. Eleven (11) High Positive (HP) and 11 High Negative (HN) samples and a Negative Control were extracted per run on the easyMAG and on the MagNA Pure LC for a total of 55 High Positive samples and 55 High Negative samples over 5 runs for each extraction instrument.

Carry-Over Contamination Study results are presented in the following table:   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=5>easyMAG</td><td rowspan=1 colspan=5>MagNA Pure LC</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>A/2009 H1N1</td><td rowspan=1 colspan=2>IC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>A/2009 H1N1</td><td rowspan=1 colspan=2>IC</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>TotalNumberCorrect</td><td rowspan=1 colspan=1>AverageCtValue</td><td rowspan=1 colspan=1>Std.Dev.</td><td rowspan=1 colspan=1>AverageCtValue</td><td rowspan=1 colspan=1>Std.Dev.</td><td rowspan=1 colspan=1>TotalNumberCorrect</td><td rowspan=1 colspan=1>AverageCtValue</td><td rowspan=1 colspan=1>Std.Dev.</td><td rowspan=1 colspan=1>AverageCtValue</td><td rowspan=1 colspan=1>Std.Dev.</td></tr><tr><td rowspan=1 colspan=1>HighPositive</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>22.09</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>22.97</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>28.89</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>28.83</td><td rowspan=1 colspan=1>0.5</td></tr></table>

The results of this study demonstrated no evidence of carryover/crosscontamination over a 5 day course of processing High Positive samples alongside High Negative samples using the ProFAST $^ +$ Assay. $1 0 0 \%$ of the High Positive and High Negative samples were accurately detected and the mean Ct value for High Positive samples was $\leq 2 3$ Ct (22.5).

m. Comparator/Reference Assays Analytical Validation Studies

The ProFAST $^ +$ Comparator/Reference Assays used as the comparison methods to the ProFAST $^ +$ Assay for the ProFAST $^ +$ Clinical Study include three independent 2-Step RT-PCR followed by b-directional sequencing assays which target conserved regions of the hemagglutinin (HA) gene of Seasonal $\mathrm { A } / \mathrm { H } 1$ , Seasonal A/H3 and A/2009 H1N1 Influenza A (Swine-Origin) subtypes. Although the gene target is the same as for the ProFAST $^ +$ Assay by necessity, the primers for the comparator/reference assays target different regions of the HA gene. The seasonal A/H1 target for the Reference Assay encompasses the ProFAST $^ +$ target in the HA gene. Targeting a different region within the HA gene was not possible due to the lack of conservation throughout the gene. The primers for the A/H1 Reference Assay are located outside the region detected by ProFAST $^ +$ resulting in detection of a total of 218 unique (additional) bases by the Reference Assay. A total of 19 Seasonal $\mathrm { A } / \mathrm { H } 1$ , 14 Seasonal A/H3 and 8 A/2009 H1N1 Swine-Origin primer sets were initially designed to target conserved regions of the Hemagglutinin $( H A )$ gene. A single primer set for each target was chosen based on conservation of primers to the Influenza A subtype specific alignments generated from available sequences at NCBI, BLAST analysis, amplicon length, melting temperature, hairpin/self dimerization of the selected primers, and analytical performance as compared to the ProFAST $^ +$ Assay. Each Influenza A subtype PCR Supermix underwent optimization of primer concentration, enzyme concentration and cycling parameters. The RT mix which uses random hexamers and MuLV Reverse Transcriptase had been previously optimized for reverse transcription for the Pro $\mathrm { { h M P V + } }$ Reference Assays, therefore, the same reagents and reactions conditions were used. PCR product generated using the ProFAST $^ +$ Reference Assays were analyzed on the Agilent 2100 Bioanalyzer for the presence of the correct size band for each Influenza A subtype.

The following analytical studies were independently performed for each of the three comparators: Analytical Sensitivity, Reactivity and Analytical Specificity.

# Analytical Sensitivity

The analytical sensitivity (limit of detection - LoD) of the ProFAST+ Comparator/Reference Assays was determined using two strains of each of the three Influenza A subtypes (Seasonal H1, Seasonal H3 and 2009 H1N1 Influenza A(Swine-Origin)). The viral strains used in this study were cultured and titered by an independent virology lab. Each virus was serially diluted and spiked at concentrations of 2 logs above, 1 log above, at, and 1 log below the expected LoD (as predetermined during assay development) into negative NP swab matrix. Each viral dilution was tested in quintuplicate using the appropriate Reference Assay (i.e. A/H1 virus dilution series was tested with the A/H1 Reference Assay only) starting from sample extraction through RT-PCR. Extraction was performed using the Roche MagNA Pure LC. Complementary DNA (cDNA) was reverse transcribed from the extracted nucleic acid followed by RT-PCR using the appropriate individual master mix for each subtype of Influenza A. PCR products generated with the ProFAST $^ +$ Reference Assays were analyzed using the Qiagen QIAxcel (capillary electrophoresis). Samples demonstrating the appropriate size PCR product were confirmed by bidirectional genetic sequencing and sequences verified using NCBI BLAST Analysis (www.ncbi.nlm.nih.gov/blast/Blast.cgi). A total of five data points per virus concentration were generated. Analytical Sensitivity was determined to be the lowest concentration of virus detected $2 9 5 \%$ of the time (5/5 replicates). This concentration was then confirmed by preparing 20 additional replicates at the proposed LoD concentration using the same negative NP swab matrix pool as the Determination portion of the study. In the event that the initial LoD concentration could not be confirmed (i.e. ${ < } 1 9$ replicates were not positive), the LoD confirmation was repeated using the next log higher concentration. At least $9 5 \%$ of the 20 replicates were required to test positive by Qiagen QIAxcel and a representative sample confirmed as positive by sequencing and BLAST Analysis to confirm the LoD for each viral strain.

Detailed analytical sensitivity data for the reference assays are presented in the following tables:

<table><tr><td colspan="6" rowspan="1">Analytical Sensitivity Determination Results</td></tr><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">ConcentrationTCID50/mL</td><td colspan="1" rowspan="1"># Replicatesdetected byQIAxcel</td><td colspan="1" rowspan="1">% detected byQIAxcel</td><td colspan="1" rowspan="1"># ReplicatesbySequencing</td><td colspan="1" rowspan="1">% detected bySequencing</td></tr><tr><td colspan="1" rowspan="4">H1N1A/Virginia/1/06</td><td colspan="1" rowspan="1">1x10</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="1">1x10-2</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="2">H1N1 A/HongKong/2652/06</td><td colspan="1" rowspan="1">1x10</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1x10-2</td><td colspan="1" rowspan="1">1/5</td><td colspan="1" rowspan="1">20%</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="4">H3N2A/Anhui/1239/05</td><td colspan="1" rowspan="1">5x101</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">5x100</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">5x10-1</td><td colspan="1" rowspan="1">3/5</td><td colspan="1" rowspan="1">60%</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40%</td></tr><tr><td colspan="1" rowspan="1">5x10-2</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="4">H3N2A/California/07/04</td><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="4">2009 H1N1 ClinicalIsolate #1</td><td colspan="1" rowspan="1">1x10{2</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1">2/5</td><td colspan="1" rowspan="1">40%</td><td colspan="1" rowspan="1">3/5</td><td colspan="1" rowspan="1">60%</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1">0/5</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="4">2009 H1N1 ClinicalIsolate #5</td><td colspan="1" rowspan="1">1x10^{2</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">5/5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td><td colspan="1" rowspan="1">4/5</td><td colspan="1" rowspan="1">80%</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1">1/5</td><td colspan="1" rowspan="1">20%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr><tr><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1">1/5</td><td colspan="1" rowspan="1">20%</td><td colspan="1" rowspan="1">N/T</td><td colspan="1" rowspan="1">N/T</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Analytical Sensitivity Confirmation Results</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=1>ReplicatesDetected byQIAxcel</td><td rowspan=1 colspan=1>% Detected byQIAxcel</td></tr><tr><td rowspan=2 colspan=1>H1N1 A/Virginia/1/06</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Hong Kong/2652/06</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/Anhui/1239/05</td><td rowspan=1 colspan=1>5x10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/California/07/04</td><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #1</td><td rowspan=1 colspan=1>1x10^{2</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #5</td><td rowspan=1 colspan=1>1x10^{2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

The following table shows the LoD comparison between the ProFAST $^ +$ Assay and the Comparator/Reference Assays. The LoD of the seasonal $\mathrm { A } / \mathrm { H } 3$ and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ strains tested was the same for both the ProFAST $^ +$ Assay and their respective Comparator/Reference Assay. The LoD for the seasonal A/H1 was $1 . 5 \log$ and $0 . 5 \log$ less sensitive with the A/H1 Comparator/Reference Assay as compared to the ProFAST $^ +$ Assay.

<table><tr><td rowspan=1 colspan=3>ProFAST+ Reference Analytical Sensitivity  Limit of Detection in comparison to theProFAST+ Assay</td></tr><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>ProFAST+ ReferenceAssay Limit of Detection</td><td rowspan=1 colspan=1>ProFAST+ Real TimeAssay Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Virginia/1/06</td><td rowspan=1 colspan=1>1x101 TCID50/mL</td><td rowspan=1 colspan=1>5x10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 A/Hong Kong/2652/06</td><td rowspan=1 colspan=1>1x10-1TCID50/mL</td><td rowspan=1 colspan=1>5x10-2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/Anhui/1239/05</td><td rowspan=1 colspan=1>5x100TCID50/mL</td><td rowspan=1 colspan=1>5x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 A/California/07/04</td><td rowspan=1 colspan=1>1x100 TCID50/mL</td><td rowspan=1 colspan=1>1x100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #1</td><td rowspan=1 colspan=1>1x102TCID50/mL</td><td rowspan=1 colspan=1>1x102 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #5</td><td rowspan=1 colspan=1>1x102 TCID50/mL</td><td rowspan=1 colspan=1>1x102 TCID50/mL</td></tr></table>

# Analytical Reactivity

The reactivity or inclusivity of the ProFAST $^ +$ Comparator/Reference Assays was evaluated using multiple strains of Influenza A: 14 strains of seasonal H1, 14 strains of seasonal H3 and 8 strains of swine-origin (four $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ isolates and 4 Swine-Origin). Viral strains were selected to include strains from 1933 to the present, as well as from various geographical locations, to incorporate the genetic variation that may be encountered during the ProFAST $^ +$ Clinical Study. The panel included the same strains tested in the ProFAST $^ +$ Reactivity Study to confirm detection of similar subtypes and strains. Remaining nucleic acid from the ProFAST $^ +$ Assay Reactivity study was used or new dilutions and extractions were made at a higher concentration and retested if necessary. The Reactivity Panel consisted of cultured and titered strains of Influenza A, spiked with Internal RNA Control and diluted in negative nasopharyngeal (NP) swab pool to a concentration of $2 \mathbf { x }$ LoD of the ProFAST $^ +$ Assay and extracted on the Roche MagNA Pure LC. Each sample was tested in duplicate reactions using the series of Reference Assays. Complementary DNA (cDNA) was reverse transcribed from the extracted nucleic acid using random hexamers and MuLV Reverse Transcriptase. PCR followed using an individual master mix for each subtype of Influenza A. The master mix contained the required components for PCR (buffer, $\mathrm { M g C l } _ { 2 }$ , dNTP’s, and Taq DNA Polymerase and subtype specific primer set). The PCR product generated with the Reference Assay Supermixes was analyzed for the appropriate sized product using the QIAGEN QIAxcel. A single replicate at the lowest concentration as determined by the QIAxcel was confirmed as positive for the appropriate subtype by bi-directional genetic sequencing and NCBI BLAST analysis.

The majority of strains analyzed in this study tested positive by the Comparator/Reference Assay as expected. The strains that tested negative during pre-verification studies and were not expected to be reactive towards the assay remained non-reactive. The following table summarizes the results:

<table><tr><td rowspan=1 colspan=7>ProFAST+ Reference Reactivity Panel Results</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=2>Concentration Detected</td><td rowspan=1 colspan=1>A/H1Detection</td><td rowspan=1 colspan=1>A/H3Detection</td><td rowspan=1 colspan=1>A/2009H1N1Detection</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/42/06</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Henan/8/05</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Fuijan/156/00</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x101 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Brazil/1137/99</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x105 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Kentucky/2/06</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/01</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/12/99</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/03/06</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Jiangxi/160/05</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 1520 A/WS/33</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>5.62x104 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 98 A1/Mal/302/54</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>1.78x106 CEID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>2.0x106 TCID50/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 546 A1/Denver/1/57</td><td rowspan=1 colspan=1>A/H1</td><td rowspan=1 colspan=2>1.78x106 CEID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hiroshima/52/05</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/512/05</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 822 A/Victoria/3/75</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 CEID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Brazil/02/99</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x10³ TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/New York/55/2004</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/2831/05</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x101 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x100 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Bahamas/2686/99</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Fuijan/411/02</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Kentucky/03/06</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Costa Rica/07/99</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/218/06</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x10&#x27; TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 544 A/Hong Kong/8/68</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 CEID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>VR 547 A/Aichi/2/68</td><td rowspan=1 colspan=1>A/H3</td><td rowspan=1 colspan=2>2x102 CEID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #2</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x103 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #3</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Clinical Isolate #4</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x101 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 A/California/04/2009</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x102 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>VR 897 A/New Jersey/8/76</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=1>2x10</td><td rowspan=1 colspan=1>2x104 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/South Dakota/03/2008</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=1>2x10</td><td rowspan=1 colspan=1>2x103 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/10/1998</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x103 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>A/Iowa/2006</td><td rowspan=1 colspan=1>S-OIV</td><td rowspan=1 colspan=2>2x103 TCID50/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr></table>

There were four strains that were not reactive with the ProFAST+ Comparator/Reference Assays. The first 3 listed are also not reactive with the ProFAST $^ +$ Assay, several base pair mismatches occur between the A/New Jersey/8/76 strain and the primers for A/2009 H1N1 which is why that strain is not detected.

• VR 1520 A/WS/33 (A/H1) • VR 98 A1/Mal/302/54 (A/H1) • VR 546 A1/Denver/1/57 (A/H1) • VR 897 A/New Jersey/8/76(A/S-OIV)

Based on the analytical validations (i.e., sensitivity and reactivity), these Comparator/Reference Assays performed equivalently to the ProFAST $^ +$ Assay for Analytical Sensitivity and Reactivity. They are therefore acceptable methods to be used as a part of composite reference method in determining “clinical diagnostic truth” for the ProFAST $+$ Clinical Trial.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix Description and Comparison:

Analytical performance of the ProFAST $^ +$ Assay using six different viral transport media (VTM) was evaluated. Analytical sensitivity was determined using cultured and titered strains of human Influenza A viruses (A/H1, A/H3, and A/2009 H1N1) spiked into each of the six different VTMs.

The intended use of Remel M4, M5, and M6, Copan Universal Transport Medium (UTM), and Becton Dickinson Universal Viral Transport (UVT) is for the transport of clinical specimens containing viruses, chlamydiae, mycoplasma, and ureaplasma from the collection site to the laboratory for microbiological procedures. Remel M4RT VTM is for the transport of clinical specimens containing viruses or chlamydiae, but is not recommended for the transport of mycoplasmas and ureaplasmas. The VTMs have essentially the same components as shown in the table below:

<table><tr><td colspan="6">Composition of Viral Transport Media</td></tr><tr><td></td><td>Remel M4</td><td>Remel M5</td><td>Remel M6</td><td>Remel M4 RT</td><td>Copan UTM</td><td>BD UTV</td></tr><tr><td>Components</td><td>Hank's Balanced Salts Bovine Serum Albumin Gelatin Sucrose L-Glutamic acid HEPES Buffer Phenol Red Phenol Red</td><td>Hank's Balanced Salts Bovine Serum Albumin Protein Stabilizers Sucrose L-Glutamic acid HEPES Buffer</td><td>Hank's Balanced Salts Bovine Serum Albumin Gelatin Sucrose L-Glutamic acid HEPES Buffer</td><td>Hank's Balanced Salts Bovine Serum Albumin Gelatin Sucrose L-Glutamic acid HEPES Buffer HEPES Buffer</td><td>Hank's Balanced salts Bovine Serum Albumin Gelatin Sucrose L-Glutamic Acid HEPES Buffer</td><td>Hank's Balanced Salts Bovine Serum Albumin Gelatin Sucrose L-Glutamic acid</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">L- Cysteine</td><td colspan="1" rowspan="1">L-Cysteine</td></tr><tr><td colspan="1" rowspan="1">pH</td><td colspan="1" rowspan="1">7.3 +/- 0.2 @25°</td><td colspan="1" rowspan="1">7.3 +/ 0.2 @25°</td><td colspan="1" rowspan="1">7.3 +/- 0.2 @25°</td><td colspan="1" rowspan="1">7.3 +/- 0.2 @25°</td><td colspan="1" rowspan="1">7.3 +/- 0.2 @25°</td><td colspan="1" rowspan="1">7.3 +/- 0.2 @25°</td></tr><tr><td colspan="1" rowspan="1">StorageTemperature</td><td colspan="1" rowspan="1">Frozen at -25° or 2-8until use</td><td colspan="1" rowspan="1">Frozen at -25°or 2-8°C untiluse</td><td colspan="1" rowspan="1">2-30°C until use</td><td colspan="1" rowspan="1">2-30°C until use</td><td colspan="1" rowspan="1">2-25°C untiluse</td><td colspan="1" rowspan="1">2-25°C untiluse</td></tr><tr><td colspan="1" rowspan="1">Volume perVial</td><td colspan="1" rowspan="1">3mL</td><td colspan="1" rowspan="1">3mL</td><td colspan="1" rowspan="1">1.5 ml</td><td colspan="1" rowspan="1">3.0 ml</td><td colspan="1" rowspan="1">3mL</td><td colspan="1" rowspan="1">3mL</td></tr></table>

Two cultured and titered Influenza A viral isolates $\mathrm { \Delta A / H 1 }$ and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1 $ ) were spiked at LoD, and one, and two logs below the LoD and the $\mathrm { A } / \mathrm { H } 3$ viral isolate was spiked at one log above, at, and one log below LoD (as determined in the ProFAST $^ +$ Analytical Sensitivity Study) into each VTM type. The testing was performed at lower concentrations for $\mathrm { A } / \mathrm { H } 1$ and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ because initial testing at 1 log below the LoD (as determined in the ProFAST+ Analytical Sensitivity Study) resulted in $100 \%$ detection. Differences in Analytical Sensitivity between NP swab matrix (as used in the Analytical Sensitivity Study) and VTM alone (as used in the present study) are not unusual and VTM alone typically results in a lower LoD.

All VTMs were compared to Remel M4 to determine equivalence because that is the VTM used in all other analytical verification studies, including the Analytical Sensitivity study, as well as by the majority of the Clinical Trial sites. For a VTM to be considered equivalent to M4:

1. All three replicates for at least one concentration were required to be positive and within one log of the lowest concentration in which all 3 replicates of M4 are positive.   
2. The average CT values at all concentrations in which 3 of 3 replicates are positive shall not differ by more than 3.3 CTs from the average CT of that concentration diluted in M4. If the sample differs by more than 3.3 CTs the media will not be considered equivalent.

Each of the VTM performed “equivalently” to Remel M4 as they all met the required criteria described above. Detailed data are presented in the following table:

<table><tr><td rowspan=1 colspan=15>VTM Study Results Summary Comparison</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=2>Remel M4</td><td rowspan=1 colspan=2>Remel M5</td><td rowspan=1 colspan=2>Remel M6</td><td rowspan=1 colspan=2>RemelM4RT</td><td rowspan=1 colspan=2>Copan UTM</td><td rowspan=1 colspan=2>BD UVT</td></tr><tr><td rowspan=1 colspan=1>A B</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AY</td><td rowspan=1 colspan=1>CtSD</td><td rowspan=1 colspan=1>A B</td><td rowspan=1 colspan=1>CtSD</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>CtSD</td><td rowspan=1 colspan=1>AVB</td><td rowspan=1 colspan=1>CtSD</td></tr><tr><td rowspan=11 colspan=1>Inn V zin</td><td rowspan=3 colspan=1>MA</td><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>35.37</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>35.17</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>35.40</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>35.93</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>34.77</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>35.20</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>5x10^{2</td><td rowspan=1 colspan=1>38.73</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>38.90</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>39.20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>38.33</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>38.17</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>38.87</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>5x10-3</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>40.80*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>40.40*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>39.90**</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40.10*</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=5 colspan=1>M</td><td rowspan=1 colspan=1>5x100</td><td rowspan=1 colspan=1>30.67</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>30.70</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>30.73</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>30.70</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>30.57</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>30.60</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>5x10-1</td><td rowspan=1 colspan=1>34.03</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>33.93</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>34.30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34.13</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>33.67</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>33.87</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>5x10^{2</td><td rowspan=1 colspan=1>36.47</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>37.17</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>37.40</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>36.73</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>37.53</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>36.40**</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>5x10-3$</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>38.65*</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>5x10-4</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=1>2007/0INIH</td><td rowspan=1 colspan=1>1x10^{2</td><td rowspan=1 colspan=1>32.73</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>32.80</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>33.50</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>33.07</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>33.40</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>32.33</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>1x10}</td><td rowspan=1 colspan=1>36.07</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>36.10</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>36.97</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>36.90</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>35.80</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>35.73</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>38.95*</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>37.30*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>38.60*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>38.55*</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>40.50*</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>38.30*</td><td rowspan=1 colspan=1>NA</td></tr></table>

\* One of three replicated detected. \*\*Two of three replicates detected.

# All six VTMs evaluated are compatible with the ProFAST $^ +$ Assay.

# 3. Clinical studies:

# a. Prospective Clinical studies

The clinical performance of the ProFAST $^ +$ Assay was established during prospective studies at 4 U.S. clinical laboratories. NP swab samples were collected and tested at three U.S. clinical laboratories (Sites 1, 2, and 4) during December 2009 thru May 2010. Due to the absence of seasonal (H1N1 or H3N2) and 2009 H1N1 Influenza A during the typical 2009-2010 winter season, prospectively collected archived samples were also included in the prospective studies. These samples were collected from January – March, 2008, February – March, 2009 and October – November, 2009, and tested at two U.S. clinical laboratories (Sites 1 and 3). All specimens used in the study meeting the inclusion and exclusion criteria represented excess, remnants of nasopharyngeal (NP) swab specimens that were prospectively collected from symptomatic individuals suspected of respiratory infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study. Inclusion criteria included, but were not limited to: the specimen was from a symptomatic patient (for respiratory infection) that was submitted for routine respiratory pathogen testing; the specimen was a NP swab in appropriate viral transport media (i.e., Remel M4 or M5 viral transport medium); the specimen contained adequate volume for comparator/reference methods and the ProFAST+ Assay; age and gender information were available; specimen was stored properly (refrigerated at $2 ^ { \circ } \mathrm { C } - 8 ^ { \circ } \mathrm { C }$ until nucleic acid extraction was performed within 72 hours of the sample collection time. Or, samples were stored frozen at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ within 72 hours of collection until processing through ProFAST $^ +$ and comparator/reference assays); and initiation of the ProFAST $^ +$ Assay (nucleic acid extraction) took place within 72 hours of sample collection, or samples were frozen at $\mathrm { \le - 7 0 ^ { \mathrm { o } } C }$ within 72 hours.

The specimens and controls ( underwent nucleic acid extraction using the bioMérieux NucliSENS easyMAG Instrument using bioMérieux NucliSENS easyMAG reagents (Sites 1 and 3) or using the Roche MagNA Pure LC Instrument using the Total Nucleic Acid Isolation kit (Sites 2 and 4). Purified nucleic acids were tested with the ProFAST $^ +$ Assay and the ProFlu $^ +$ Assay on the Cepheid SmartCycler II. (Note: Purified nucleic acids were further processed immediately following extraction or were stored frozen $( { \leq } { - } 7 0 ^ { \circ } \mathrm { C } )$ and thawed prior to ProFAST+ Assay and ProFlu $^ +$ Assay set-up). Remaining nucleic acids and original samples were sent back to Gen-Probe Prodesse for sequencing analysis.

Reverse transcription was performed on remaining nucleic acids that had been stored at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ . PCR was performed on the resulting cDNA using all three Comparator/Reference Assays and the PCR products were analyzed by capillary electrophoresis (QIAxcel, QIAGEN). PCR products with the appropriate base pair band size were sent for further analysis by genetic sequencing. Sequencing results that met acceptance criteria $( > 1 0 0$ contiguous bases, QV score (i.e., PHRED) $> 2 0$ , E-value scores ${ < } 1 \mathrm { e } { - } 2 0 $ ) for both the forward and reverse primer were recorded as positive for the specific Influenza A subtype.

Performance of the ProFAST $^ +$ Assay was compared to the composite comparator/reference method of the FDA cleared ProFlu+ Assay IVD (K081030) and individual well characterized Influenza A subtype specific RT-PCR assays followed by bi-directional sequencing. The sequencing assays targeted different regions of the hemagglutinin gene than the ProFAST $^ +$ Assay and were specific for each of the Influenza A subtypes (A/H1, A/H3, and A/2009 H1N1). “True” seasonal A/H1, A/H3 or A/2009 H1N1 RNA positives, were considered as any sample that was tested positive for Influenza A by the ProFlu $^ +$ Assay IVD, and had bidirectional sequencing data meeting pre-defined quality acceptance criteria, for both the forward and the reverse sequences that matched seasonal A/H1, A/H3, and A/2009 H1N1 sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” seasonal A/H1, A/H3 or A/2009 H1N1 RNA negatives were considered as any sample that was tested negative for Influenza A by the ProFlu+ Assay IVD, or any sample that was tested positive for Influenza A by the ProFlu $^ +$ Assay IVD, but was tested negative by the respective Influenza A subtype specific RT-PCR assay. The E-values generated from the clinical trial range from a low of <1e-180 to a high of 2e-35. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone. (http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).

Therefore an E-Value ranging from ${ < } 2 \mathrm { e } { - } 3 5$ to 1e-180 has a very low probability of occurring purely by chance.

The overall assay failure rate of the ProFAST $^ +$ Assay during the clinical trials (both the prospective and the retrospective trials) due to failure of Positive, Negative or Extraction Controls not related to procedural errors was $4 . 0 \%$ (total of 3 of 75 runs across all 4 sites). Two of the run failures were due to the Negative Control contamination (one run at Site 3 and one run at Site 4). The other invalid run was due to the PC failure (none of the analytes were detected) and it is hypothesized that the PC was mistakenly not added to the ProFAST $^ +$ master mix. All samples, the Negative Control and the Extraction Control were re-run using the ProFAST $^ +$ Assay starting from the purified nucleic acid and repeat test results were “Valid”.

A total of 874 prospective NP swab specimens were initially included in the prospective clinical trial (400 samples from Site 1, 129 samples from Site 2, 300 samples from Site 3, and 45 samples from Site 4). Thirty two (32) samples (18 samples from Site 1, 10 samples from Site 2, 2 samples from Site 3, and 2 samples from Site 4) were excluded from the prospective clinical study data analysis because they remained “Unresolved” after repeat testing for either the ProFlu $^ +$ Assay (comparator assay), or the ProFAST $^ +$ Assay, or both assays, resulted in a total 842 eligible prospective specimens to be included in the prospective clinical study data analysis.

Of the ProFAST $^ +$ Assay run on all prospective specimens, $9 8 . 9 \%$ (864/874) of these specimens were successful on the first attempt (Site $1 ; 3 9 2 / 4 0 0 = 9 8 . 0 \%$ ; Site 2: $1 2 8 / 1 2 9 = 9 9 . 2 \%$ ; Site 3: $2 9 9 / 3 0 0 = 9 9 . 7 \%$ ; Site 4: $4 5 / 4 5 = 1 0 \time 1 0 \%$ ). The remaining 10 ( $1 0 / 8 7 4 = 1 . 1 \%$ ) gave “Unresolved” results on the first attempt (8 from Site 1, 1 from Site 2, 1 from Site 3, and none from Site 4). Unresolved results occur when the sample is negative for all three Influenza A subtype markers and the Internal Control, indicating potentially PCR-inhibiting samples. Of the 10 “Unresolved” specimens on the first attempt with sufficient sample for retest, only $5 0 . 0 \%$ (5/10) gave a valid result on the second attempt (3 from Site 1, 1 from site 2, and 1 from Site 3). The remaining 5 were “Unresolved” on the second attempt.

The following tables represent the prospective clinical data by site and by influenza A subtypes:

Seasonal Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Site 1</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>367</td><td rowspan=1 colspan=1>367</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>382</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 14/14      100.0%         (95%CI: 76.8%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 367/368    99.7%           (95%CI: 98.5%-100.0%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the ProFlu+ Assay, but positive for seasonal $\mathrm { A } / \mathrm { H } 1$ by bidirectional sequence analysis.

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Site 2</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 0/0             N/A</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:119/119      100.0%       (95%CI: 96.9%-100.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Site 3</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>7*</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>252</td><td rowspan=1 colspan=1>252</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>298</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 39/39         100.0%        (95%CI:91.0%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:252/259      97.3%        (95%CI: 94.5%-98.9%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the ProFlu+ Assay, but positive for seasonal A/H1 by bi-directional sequence analysis. One (1) sample was negative for Influenza A by the $\mathrm { P r o F l u ^ { + } }$ Assay, and negative for seasonal $\mathrm { A } / \mathrm { H } 1$ by bi-directional sequence analysis, but positive for Influenza A, un-subtyptable, by the FDA cleared CDC rRT-PCR Influenza Panel. Five (5) samples were positive for Influenza A by the $\mathrm { P r o F l u ^ { + } }$ Assay, negative for $\mathrm { { A / H 1 } }$ , A/H3 and A/2009 H1N1 by bi-directional sequence analysis, but positive for A/H1 by the FDA cleared CDC rRT-PCR Influenza Panel.

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Site 4</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  0/0          N/A</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 43/43      100.0%           (95%CI: 91.8%-100.0%)</td></tr></table>

Seasonal Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3Results Site 1</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>366</td><td rowspan=1 colspan=1>366</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>366</td><td rowspan=1 colspan=1>382</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 16/16           100.0%       (95%CI:79.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:366/366        100.0%       (95%CI: 99.0%-100.0%)</td></tr></table>

<table><tr><td colspan="1" rowspan="2">ProFAST+ A/H3Results Site 2</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">119</td><td colspan="1" rowspan="1">119</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">119</td><td colspan="1" rowspan="1">119</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement: 0/0           N/A</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement:119/119     100.0%         (95%CI: 96.9%-100.0%)</td></tr><tr><td colspan="1" rowspan="2">ProFAST+ A/H3Results Site 3</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2*</td><td colspan="1" rowspan="1">11</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">287</td><td colspan="1" rowspan="1">287</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">289</td><td colspan="1" rowspan="1">298</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement: 9/9              100.0%       (95%CI:66.4%-100.0%)</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement:287/289          99.3%       (95%CI: 97.5%-99.9%)</td></tr></table>

\*One (1) sample was positive for Influenza A by the ProFlu+ Assay, negative for $\mathrm { { A / H 1 } }$ , A/H3 and A/2009 H1N1 by bi-directional sequence analysis, but positive for $_ \mathrm { A / H 3 }$ by the FDA cleared CDC rRT-PCR Influenza Panel. One (1) sample was positive for Influenza A by the ProFlu $^ +$ Assay, positive for $\mathrm { A } / \mathrm { H } 1$ and negative for A/H3 and A/2009 H1N1 by bi-directional sequence analysis.

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3ResultsSite 4</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>43</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:    0/0           N/A</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 42/43            97.7%         (95%CI: 87.7%-99.9%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the $\mathrm { P r o F l u ^ { + } }$ Assay, also negative for seasonal A/H1 and A/H3, and A/2009 H1N1 by bi-directional sequence analysis.

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/2009H1N1 Results Site 1</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>351</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>382</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   31/31          100.0%       (95%CI: 88.8%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 351/351        100.0%       (95%CI: 99.0%-100.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/2009H1N1 Results Site 2</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 0/0       N/A</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 119/119         100.0%     (95%CI: 96.9%-100.0%)</td></tr></table>

<table><tr><td colspan="1" rowspan="2">ProFAST+ A/2009H1N1 Results Site 3</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">31</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">264</td><td colspan="1" rowspan="1">267</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">264</td><td colspan="1" rowspan="1">298</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement:   31/34              91.2%      (95%CI: 76.3%-98.1%)</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement: 264/264           100.0%     (95%CI: 98.6%-100.0%)</td></tr><tr><td colspan="1" rowspan="2">Pr0FAST+ A/2009 H1N1ResultsSite 4</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement:       0/0        N/A</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement:     43/43          100.0%      (95%CI: 91.8%-100.0%)</td></tr></table>

The prospective performance data from all study sites combined are presented in the following tables by influenza A subtype:

Seasonal Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>8*</td><td rowspan=1 colspan=1>61</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>781</td><td rowspan=1 colspan=1>781</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>789</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:53/53          100.0%         (95%CI: 93.2%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 781/789        99.0%           (95%CI: 98.0%-99.5%)</td></tr></table>

\* Two (2) samples were negative for Influenza A by the ProFlu+ Assay, but positive for seasonal $\mathrm { A } / \mathrm { H } 1$ by bi-directional sequence analysis. One (1) sample was negative for Influenza A by the ProFlu $^ +$ Assay, and negative for seasonal $\mathrm { A } / \mathrm { H } 1$ by bi-directional sequence analysis, but positive for Influenza A, un-subtyptable, by the FDA cleared CDC rRT-PCR Influenza Panel. Five (5) samples were positive for Influenza A by the ProFlu $^ +$ Assay, negative for A/H1, A/H3 and $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ by bi-directional sequence analysis, but positive for A/H1 by the FDA cleared CDC rRT-PCR Influenza Panel.

Seasonal Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>814</td><td rowspan=1 colspan=1>814</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>817</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 25/25          100.0%             (95%CI:86.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:814/817        99.6%             (95%CI: 98.9%-99.9%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the ProFlu+ Assay, also negative for seasonal A/H1 and A/H3, and A/2009 H1N1 by bi-directional sequence analysis. $\ast _ { \mathrm { O n e } }$ (1) sample was positive for Influenza A by the ProFlu $^ +$ Assay, negative for A/H1, A/H3 and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ by bi-directional sequence analysis, but positive for A/H3 by the FDA cleared CDC rRT-PCR Influenza Panel. One (1) sample was positive for Influenza A by the ProFlu $^ +$ Assay, positive for A/H1 and negative for $_ \mathrm { A / H 3 }$ and $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ by bi-directional sequence analysis.

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=2 colspan=1>Pr0FAST+ A/2009 H1N1Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>777</td><td rowspan=1 colspan=1>780</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>777</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  62/65            95.4%           (95%CI:87.3%-98.4%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:777/777         100.0%           (95%CI:99.5%-100.0%)</td></tr></table>

The general demographic data for all eligible prospective specimens $\scriptstyle ( \mathrm { N } = 8 4 2$ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>439/842 (52.1%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>403/842 (47.9%)</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>439/842 (52.1%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>184/842 (21.9%)</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>168/842 (20.0%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>51/842 (6.0%)</td></tr></table>

# b. Retrospective Clinical studies

In addition to the prospective clinical studies, two clinical sites (Site 2 and site 4) also performed testing using retrospective samples that were collected from January – March, 2008, January – November 2009, and March 2010. The ProFAST $^ +$ Assay was compared to the same composite comparator/reference method that was employed for the prospective study to determine clinical Percent Positive   
Agreement and Percent Negative Agreement. A total of 160 retrospective   
nasopharyngeal (NP) swab samples were included in the retrospective studies (60 samples at Site 2 and 100 samples at site 4). The E-values generated from the retrospective clinical trial range from a low of ${ < } 1 \mathrm { e } { - } 1 8 0$ to a high of 7e-45. The EValue from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone.   
(http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).   
Therefore an E-Value ranging from 7e-45 to <1e-180 has a very low probability of occurring purely by chance.

These data were analyzed separately from the prospective samples and are summarized in the following tables by site and by influenza A subtype:

Seasonal Influenza A/H1   

<table><tr><td colspan="1" rowspan="2">ProFAST+ A/H1 ResultSite 2</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">43</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">60</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement:   17/18            94.4%        (95%CI: 72.7%-99.9%)</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement: 42/42             100.0%       (95%CI: 91.6%-100.0%)</td></tr><tr><td colspan="1" rowspan="2">ProFAST+ A/H1Results Site 4</td><td colspan="3" rowspan="1">Composite Reference</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1*</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">99</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="4" rowspan="1">Positive Percent Agreement:     0/0        N/A</td></tr><tr><td colspan="4" rowspan="1">Negative Percent Agreement:   99/100             99.0%        (95%CI: 94.5%-100.0%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the ProFlu $^ +$ Assay, but positive for seasonal A/H1 by bi-directional sequence analysis.

Seasonal Influenza $\mathrm { A } / \mathrm { H } 3$   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3Results Site 2</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  34/34             100.0%          (95%CI: 89.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 26/26             100.0%          (95%CI: 86.8%-100.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3Results Site 4</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  38/38              100.0%        (95%CI:90.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 62/62              100.0%        (95%CI:94.2%-100.0%)</td></tr></table>

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=1 colspan=1>ProFAST+ A/2009</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>H1N1 Results Site 2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   5/5             100.0%          (95%CI: 47.8%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:55/55           100.0%           (95%CI: 93.5%-100.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/2009H1N1 Results Site 4</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>80</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  20/20              100.0%        (95%CI:83.2%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 80/80              100.0%        (95%CI:95.5%-100.0%)</td></tr></table>

The retrospective performance data from all study sites combined are presented in the following tables by influenza A subtypes:

Seasonal Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H1Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>142</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 17/18           94.4%             (95%CI: 74.3%-99.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:141/142        99.3%             (95%CI: 96.1%-99.9%)</td></tr></table>

\* One (1) sample was negative for Influenza A by the ProFlu+ Assay, but positive for seasonal A/H1 by bidirectional sequence analysis.

Seasonal Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/H3Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>72</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  72/72           100.0%         (95%CI:94.9%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 88/88          100.0%         (95%CI: 95.8%-100.0%)</td></tr></table>

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=2 colspan=1>ProFAST+ A/2009H1N1 Results Combined</td><td rowspan=1 colspan=3>Composite Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>135</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>135</td><td rowspan=1 colspan=1>160</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   25/25             100.0%      (95%CI:86.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:  135/135           100.0%      (95%CI:97.2%-100.0%)</td></tr></table>

The general demographic data for all eligible retrospective specimens $\mathrm { ( N { = } } 1 6 0$ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Sex*</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>74/160 (46.3%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>84/160 (52.5%)</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1>EY</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>25/160 (15.6%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>24/160 (15.0%)</td></tr><tr><td rowspan=1 colspan=1>19- 64 years</td><td rowspan=1 colspan=1>91/160 (56.9%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>20/160 (12.5%)</td></tr></table>

\*For two of the subjects the gender was unknown

c. Other clinical supportive data (when a. and $b$ . are not applicable): Not applicable.

4. Clinical cut-off:

Not applicable.

# 5. Expected values/Reference range:

In the prospective ProFAST $^ +$ Assay clinical study, a total of 842 eligible prospective nasopharyngeal (NP) swab specimens were tested from four U.S. clinical laboratories across the United States. Of the total of 842 eligible NP swab samples, 257 were collected and tested at three U.S. clinical laboratories (Sites 1, 2, and 4) during December 2009 thru May 2010. Due to the absence of seasonal (H1N1 or H3N2) and 2009 H1N1 Influenza A during the typical 2009-2010 winter season, a total of 585 prospectively collected archived NP swab samples were also included in the prospective studies. Of the total of 585 prospectively collected archived NP swab samples, 195 samples were collected from January – March, 2008; 196 samples were collected from February – March, 2009; and 194 samples were collected from October – November, 2009. The prospectively collected archived NP swab samples were tested at two U.S. clinical laboratories (Sites 1 and 3).

The number and percentage of seasonal A/H1, seasonal A/H3 and A/2009 H1N1 RNA positive cases as determined by the ProFAST $^ +$ Assay, calculated by age group and prospective sample collection time period, are presented in the following table:

<table><tr><td colspan="1" rowspan="1">AgeGroup</td><td colspan="1" rowspan="1">TotalN</td><td colspan="1" rowspan="1">A/H1Positive BytheProFAST+Assay</td><td colspan="1" rowspan="1">A/H3Positive BytheProFAST+Assay</td><td colspan="1" rowspan="1">A/2009H1N1Positive BytheProFAST+Assay</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">−</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">NumberPositive</td><td colspan="1" rowspan="1">NumberPositive</td><td colspan="1" rowspan="1">NumberPositive</td><td colspan="1" rowspan="1">ObservedPrevalenceA/H1</td><td colspan="1" rowspan="1">ObservedPrevalenceA/H3</td><td colspan="1" rowspan="1">ObservedPrevalenceA/2009H1N1</td></tr><tr><td colspan="8" rowspan="1">December 2009 thru May 2010</td></tr><tr><td colspan="1" rowspan="1">&lt; 2 years</td><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">2-5 years</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">6-11years</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">4.3%</td></tr><tr><td colspan="1" rowspan="1">12-18years</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">19-64years</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">&gt; 65years</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">257</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="8" rowspan="1">January 2008 thru March 2008</td></tr><tr><td colspan="1" rowspan="1">&lt; 2 years</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8.5%</td><td colspan="1" rowspan="1">11.7%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">2-5 years</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">8.3%</td><td colspan="1" rowspan="1">12.5%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">6-11years</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">12.5%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">12-18years</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10.5%</td><td colspan="1" rowspan="1">15.8%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">19-64years</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">12.0%</td><td colspan="1" rowspan="1">16.0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">&gt; 65years</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">22.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">195</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10.8%</td><td colspan="1" rowspan="1">12.3%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="8" rowspan="1">February 2009 thru March 2009</td></tr><tr><td colspan="1" rowspan="1">&lt; 2 years</td><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">10.2%</td><td colspan="1" rowspan="1">1.1%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">2-5 years</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">36.0%</td><td colspan="1" rowspan="1">4.0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">6-11years</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">29.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">12-18years</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25.0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">19-64years</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">26.7%</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">&gt; 65years</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">196</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20.4%</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="8" rowspan="1">October 2009 thru November 2009</td></tr><tr><td colspan="1" rowspan="1">&lt; 2 years</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">22.5%</td></tr><tr><td colspan="1" rowspan="1">2-5 years</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">33.3%</td></tr><tr><td colspan="1" rowspan="1">6-11years</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">50.0%</td></tr><tr><td colspan="1" rowspan="1">12-18years</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">33.3%</td></tr><tr><td colspan="1" rowspan="1">19-64years</td><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">28.9%</td></tr><tr><td colspan="1" rowspan="1">&gt; 65years</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">7.1%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">194</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">31.4%</td></tr><tr><td colspan="8" rowspan="1">Combined</td></tr><tr><td colspan="1" rowspan="1">&lt; 2 years</td><td colspan="1" rowspan="1">324</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">5.2%</td><td colspan="1" rowspan="1">3.7%</td><td colspan="1" rowspan="1">2.8%</td></tr><tr><td colspan="1" rowspan="1">2-5 years</td><td colspan="1" rowspan="1">115</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">9.6%</td><td colspan="1" rowspan="1">3.5%</td><td colspan="1" rowspan="1">10.4%</td></tr><tr><td colspan="1" rowspan="1">6-11years</td><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">11.2%</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">17.2%</td></tr><tr><td colspan="1" rowspan="1">12-18years</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">8.8%</td><td colspan="1" rowspan="1">4.4%</td><td colspan="1" rowspan="1">10.3%</td></tr><tr><td colspan="1" rowspan="1">19-64years</td><td colspan="1" rowspan="1">168</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">6.5%</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">7.7%</td></tr><tr><td colspan="1" rowspan="1">&gt; 65years</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5.9%</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">2.0%</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">842</td><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">7.2%</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">7.4%</td></tr></table>

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b or 3.0a or 3.0b

MagNA Pure LC Instrument (Roche)

NucliSENS® easyMAG™ System (bioMérieux)

# O. System Descriptions:

# 1. Modes of Operation:

The Cepheid Smart Cycler II Real Time instrument with Dx software version $1 . 7 6$ or 3.0a or 3.0b is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Seven other nucleic acid amplification tests that use the Smart Cycler II instrument have received 510(k) clearance: Prodesse’s ProParaflu $^ +$ Assay (K091053), ProGastro Cd Assay (K090239), ProhMPV $+$ Assay (K082688) and ProFlu $^ +$ Assay (K081030), IDI-MRSA test (K033415), the IDI-Strep B Assay (K022504), and the Influenza A/H5 (Asian lineage) Virus Real-Time RT-PCR Primer and Probe Set (K060159). The Cepheid SmartCycler instrument is an integrated nucleic acid amplification and detection instrument system based on Cepheid’s proprietary microprocessor-controlled I-CORE module. For purified RNA samples, the SmartCycler instrument enables reverse-transcriptase (RT) to transcribe target viral RNA into cDNA, polymerase chain reaction (PCR) for the amplification of cDNA, and hybridization of fluorogenic target-specific probes for the detection of the amplified cDNA.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles. Six other tests that use the Roche MagNA

Pure LC system have received 510(k) clearance: Prodesse’s ProParaflu $^ +$ Assay (K091053), ProGastro Cd Assay (K090239), ProhMPV $+$ Assay (K082688) and ProFlu $^ +$ Assay (K081030), Roche Factor V Leiden Kit (K033607) and the Roche Factor II (Prothrombin) G20210A Kit (K033612).

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads. Prodesse’s ProFlu+ Assay (K081030) that received Special 510(k) clearance and Prodesse’s ProhMPV $+$ Assay (K082688), ProGastro Cd Assay (K090239) and ProParaflu $^ +$ (K091053) also use the easyMAG system for automated nucleic acid extraction.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

Positive Control (PC): The ProFAST $^ +$ Assay kit contains an Influenza A Subtyping positive RNA control that consists of one pool of three RNA transcripts (specific for each Influenza A subtype: A/H1, A/H3 and A/2009 H1N1). These transcripts are derived from plasmids containing the viral sequences of interest and are not intact virus particles. The Influenza A Subtyping PC contains equal amounts of A/H1, A/H3 and $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ . The approximate concentration of each transcript was $5 \mathrm { x } 1 0 ^ { 3 } \mathrm { c o p i e s } / \mu \mathrm { l } .$ . The PC does not go through nucleic acid isolation and purification, but is included during set-up of the RT-PCR reaction. The PC in conjunction with the IC is used to verify reagent and system performance. The PC is meant to be a control for global failure of the assay (missing reaction component, instrument failure, etc).

Internal Control (IC): An Internal RNA Control, a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The IC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working. The IC is meant to assess global failure (reagent or process) and monitor for PCR inhibition.

Negative Control (NC): A Negative Control (NC) is not provided with the kit, but is required and described in the ProFAST $^ +$ Assay Instructions for Use. Viral transport media spiked with the IC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, a pool of seasonal A/H1N1, seasonal A/H3N2 and 2009 H1N1 influenza viruses was included with each nucleic acid isolation run. The EC served to monitor for lysis during nucleic acid isolation. The sponsor is also recommending an extraction control in each nucleic acid extraction run to the end users in the package insert.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.