# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K101564   
B. Purpose for Submission: New Device   
C. Measurand:

Specific influenza virus nucleic acid target sequences. Influenza types and subtypes detected:

1) Highly conserved region of the matrix (M) gene from influenza A viruses; 2) Highly conserved region of the nucleoprotein (NP) gene specific for 2009 H1N1 influenza type A;   
3) Highly conserved region of hemagglutinin (HA) gene specific for 2009 H1N1 influenza A subtype H1.

# D. Type of Test:

Real-time reverse transcription-polymerase chain reaction (RT-PCR) assay for the qualitative detection and differentiation of influenza A and 2009 H1N1 influenza viral RNA in nasopharyngeal swabs (NPS), nasal swabs (NS), throat swabs (TS), nasal aspirates (NPA), nasal washes (NW), dual nasopharyngeal/throat swabs (NPS/TS), and broncheoalveolar lavage [BAL], bronchial wash [BW], and tracheal aspirate [TA] from human patients with signs and symptoms of respiratory infection and/or from viral culture. The isolation and purification of the nucleic acids is performed using one of the six recommended methods listed below in section “I. Device Description”. Amplification and detection is performed on the Applied Biosystems 7500 Fast Dx instrument (K082562) with Sequence Detection Software version 1.4.

# E. Applicant:

Centers for Disease Control and Prevention, Atlanta, GA

# F. Proprietary and Established Names:

Proprietary: The CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel Generic: CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel

# G. Regulatory Information:

1. Regulation section: 866.3332

2. Classification: Class II

3. Product code: OQW, NSU

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel (CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel) is intended for use in real-time RT-PCR assays on an Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR Instrument in conjunction with clinical and epidemiological information:

For the qualitative detection of influenza virus type A viral RNA from nasopharyngeal swabs (NPS), nasal swabs (NS), nasal aspirates (NA), nasal washes (NW), dual nasopharyngeal / throat swabs (NPS/TS), broncheoalveolar lavage (BAL), tracheal aspirate (TA), and bronchial wash (BW), collected from the respiratory tract of human patients with signs and symptoms of respiratory infection and/or from viral culture. For differentiation of 2009 H1N1 influenza virus RNA from nasopharyngeal swabs (NPS), nasal swabs (NS), nasal aspirates (NA), nasal washes (NW) ,dual nasopharyngeal / throat swabs (NPS/TS), broncheoalveolar lavage (BAL), tracheal aspirate (TA), and bronchial wash (BW) collected from the respiratory tract of human patients with signs and symptoms of respiratory infection and/or from viral culture. • To provide epidemiologic information for surveillance of the 2009 H1N1 influenza virus.

Performance characteristics for influenza A were established during the 2009-2010 influenza season when 2009 H1N1 influenza virus was the predominant influenza A virus in circulation. Performance characteristics may vary against other emerging influenza A viruses.

A negative test result for the broncheoalveolar lavage (BAL), tracheal aspirate (TA), and bronchial wash (BW) is presumptive and it is recommended these results be confirmed by viral culture.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

All users, analysts, and any person reporting results from use of this device should be trained to perform and interpret the results from this procedure by a competent instructor prior to use. CDC Influenza Division will limit the distribution of this   
device to only those users who have successfully completed a training course provided by CDC instructors or designees.

2. Indication(s) for use:

Same as intended use

3. Special conditions for use statement(s):

For prescription use only

4. Special instrument requirements:

To be used with the Applied Biosystems ABI 7500 Fast Dx Real-Time PCR Instrument with Sequence Detection Software version 1.4 and one of the six recommended RNA extraction methods.

# I. Device Description:

The CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel is a is a real-time RT-PCR test that uses oligonucleotide primers and dual-labeled hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes for the in vitro qualitative detection and differentiation of human influenza A and the 2009 H1N1 influenza viral RNA on the ABI 7500 Fast Dx Real-Time PCR instrument. Viral RNA can be detected from nasopharyngeal swabs (NPS), nasal swabs (NS), nasal aspirates (NA), nasal washes (NW), dual nasopharyngeal/throat swabs (NPS/TS, bronchoalveolar lavage (BAL), tracheal aspirate (TA), and bronchial wash (BW) collected from patients with signs and symptoms of respiratory infection and from viral culture isolates. CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel also includes control materials.

The use of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel requires ancillary reagents for which specific lots have been qualified by the CDC Influenza Division and incorporated in the CDC quality system. Any lots not specifically qualified by CDC for use with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel are not validated for use with this assay and may affect device performance. A supplemental cumulative list of qualified ancillary reagents lots for use with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel is provided with each shipment or can be requested by sending an email to FluSupport@cdc.gov.

The following is a list of ancillary reagents that are not supplied with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel and are included in CDC’s reagent qualification program.

<table><tr><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>Reagent</td><td rowspan=1 colspan=1>Quantity</td><td rowspan=1 colspan=1>Catalog No.</td></tr><tr><td rowspan=4 colspan=1>rRT-PCREnzymeMastermixOptions</td><td rowspan=2 colspan=1>Invitrogen SuperScript™M III Platinum®One-Step Quantitative RT-PCR System(without Rox)</td><td rowspan=1 colspan=1>100 reactions</td><td rowspan=1 colspan=1>11732-020</td></tr><tr><td rowspan=1 colspan=1>500 reactions</td><td rowspan=1 colspan=1>11732-088</td></tr><tr><td rowspan=2 colspan=1>Invitrogen SuperScriptTM III Platinum®One-Step Quantitative RT-PCR System(with Rox)</td><td rowspan=1 colspan=1>100 reactions</td><td rowspan=1 colspan=1>11745-100</td></tr><tr><td rowspan=1 colspan=1>500 reactions</td><td rowspan=1 colspan=1>11745-500</td></tr><tr><td rowspan=6 colspan=1>NucleicAcidPurificationKit Options</td><td rowspan=2 colspan=1>Qiagen QIAamp® Viral RNA Mini Kit</td><td rowspan=1 colspan=1>50 extractions</td><td rowspan=1 colspan=1>52904</td></tr><tr><td rowspan=1 colspan=1>250 extractions</td><td rowspan=1 colspan=1>52906</td></tr><tr><td rowspan=1 colspan=1>Roche MagNA Pure Compact NucleicAcid Isolation Kit I</td><td rowspan=1 colspan=1>32 extractions</td><td rowspan=1 colspan=1>03 730 964 001</td></tr><tr><td rowspan=1 colspan=1>Roche MagNA Pure LC 2.0 with TotalNucleic Acid (TNA) Kit</td><td rowspan=1 colspan=1>192 extractions</td><td rowspan=1 colspan=1>03 038 505 001</td></tr><tr><td rowspan=1 colspan=1>Roche MagNA Pure Compact RNAIsolation Kit</td><td rowspan=1 colspan=1>32 extractions</td><td rowspan=1 colspan=1>04 802 993 001</td></tr><tr><td rowspan=1 colspan=1>bioMérieux NucliSENS® easyMAG®(Automated magnetic extraction reagentssold separately)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): Focus Simplexa™ Influenza A H1N1 (2009) assay and CDC Human Influenza Virus Real-Time RT- PCR Detection and Characterization Panel

2. Predicate 510(k) number(s): K0100148 and K080570

3. Comparison with predicate:

Similarities   

<table><tr><td rowspan=1 colspan=1>DeviceCharacteristics</td><td rowspan=1 colspan=1>CDC rRT-PCR 2009A(H1N1)pdm Flu Panel(New Device)</td><td rowspan=1 colspan=1>SimplexaTM Influenza AH1N1 (2009) (K100148)</td><td rowspan=1 colspan=1>CDC Human Influenza Virus Real-Time RT- PCR Detection andCharacterization Panel (K080570)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative in vitro detectionof influenza virus type A and2009(H1N1 influenza viralRNA from nasopharyngealswabs (NPS), nasal swabs(NS), throat swabs (TS), nasalaspirates (NA), nasal washes(NW), dual nasopharyngeal /throat swabs (NPS/TS) andlower respiratory tractspecimens (LRTS) fromhuman patients with signs andsymptoms of respiratoryinfection and/or from viralculture, on the AppliedBiosystems (ABI) 7500 FastDx Real-Time PCR instrument</td><td rowspan=1 colspan=1>Qualitative in vitrodetection anddifferentiation of influenzaA and 2009 H1N1influenza viral RNA innasopharyngeal swabs(NPS), nasal swabs (NS),and nasopharyngealaspirates (NPA) fromhuman patients with signsand symptoms ofrespiratory infection on the3M Integrated Cycler</td><td rowspan=1 colspan=1>Qualitative in vitro detection ofinfluenza virus type A or B and fordetermination of the subtype ofseasonal human influenza A virus, asseasonal A/HI or A/H3, if present,from viral RNA in nasopharyngealand/or nasal swab specimens, forpresumptive identification of virus inpatients who may be infected withinfluenza A subtype A/H5 (Asianlineage) from viral RNA in humanrespiratory specimens and viralculture on an ABI 7500 Fast Dx Real-time PCR instrument</td></tr><tr><td rowspan=1 colspan=1>Identification ofInf A</td><td rowspan=1 colspan=1>Yes (Universal A)</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidExtraction</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Real-time RT-PCR</td><td rowspan=1 colspan=1>Real-time PCR</td><td rowspan=1 colspan=1>Real-time RT-PCR</td></tr></table>

Differences   

<table><tr><td colspan="1" rowspan="1">Device Characteristics</td><td colspan="1" rowspan="1">CDC rRT-PCR 2009A(H1N1)pdm FluPanel (New Device)</td><td colspan="1" rowspan="1">SimplexaTM Influenza AH1N1 (2009) (New Device)</td><td colspan="1" rowspan="1">CDC Human Influenza Virus Real-Time RT- PCR Detection andCharacterization Panel (PredicateDevice #2)</td></tr><tr><td colspan="1" rowspan="1">Sample types</td><td colspan="1" rowspan="1">NPS, NS, NA, NW,NPS/TS, BAL, TA, BW</td><td colspan="1" rowspan="1">NPS, NS, NPA</td><td colspan="1" rowspan="1">NPS, NS</td></tr><tr><td colspan="1" rowspan="1">Extraction Methods</td><td colspan="1" rowspan="1">QIAamp® Viral RNAMini Kit, Qiagen Inc.MagNA Pure Compact -Total Nucleic Acid Kit,Roche Applied Science•MagNA Pure Compact RNA Isolation Kit,Roche Applied Science• MagNA Pure LC - RNAIsolation Kit II, RocheApplied ScienceQiagen QIAcube withQIAamp® Viral RNAMini Kit, Qiagen Inc.NucliSENSeasyMAG®, bioMerieux</td><td colspan="1" rowspan="1">•QIAamp® Viral RNA MiniKit, Qiagen Inc.MagNA Pure TotalNucleic Acid Isolation Kit(Roche)MagNA Pure LCInstrument (Roche)</td><td colspan="1" rowspan="1">QIAamp® Viral RNA Mini Kit, QiagenInc.RNeasy® Mini Kit, Qiagen, Inc.• MagNA Pure LC RNA Isolation Kit II,Roche Applied Science•MagNA Pure Total Nucleic Acid Kit,Roche Applied Science</td></tr><tr><td colspan="1" rowspan="1">Identification of 2009H1N1 Subtype</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr><tr><td colspan="1" rowspan="1">RequiredInstrumentation</td><td colspan="1" rowspan="1">Applied Biosystems7500 Fast Dx Real-Time PCR Instrumentwith SDS softwareversion 1.4</td><td colspan="1" rowspan="1">Integrated cycler withIntegrated Cycler Studiosoftware v. 2.0</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Fast DxReal- Time PCR Instrument with SDSsoftware version 1.4</td></tr><tr><td colspan="1" rowspan="1">Multiplex Capability</td><td colspan="1" rowspan="1">No(Modular Use withCDC rRT-PCR FluPanel (K08570)</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">No</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Reagents for Detection of Specific Novel Influenza A Viruses, March   
22, 2006   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm078583.htm   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Testing for Detection and Differentiation of Influenza A Virus   
Subtypes Using Multiplex Assays October 9, 2009   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm180307.htm   
Draft Guidance for Industry and FDA Staff - Establishing Performance   
Characteristics of In Vitro Diagnostic Devices for Detection or Detection and   
Differentiation of Influenza Viruses, February 15, 2008   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm079171.htm   
Guidance for Industry and FDA Staff, In Vitro Diagnostic Devices to Detect   
Influenza A Viruses: Labeling and Regulatory Path, May 1, 2007   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDo   
cuments/ucm078538.htm

# L. Test Principle:

The CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel includes a set of oligonucleotide primers and dual-labeled hydrolysis (Taqman $\textsuperscript { \textregistered }$ ) probes for use in real-time RT-PCR assays on an ABI 7500 Fast Dx Real-Time PCR instrument for the in vitro qualitative detection of viral nucleic acids from human influenza A and 2009 H1N1 influenza in respiratory specimens. The assay consists of reagents to detect the matrix (M) gene RNA from influenza A virus, nucleoprotein (NP) gene and hemagglutinin (HA) gene RNA from 2009 H1N1 influenza virus, and the human RNase P gene RNA from the human specimen extraction control (HSC). An extraction control (HSC) is added to each batch of specimens to be extracted to monitor the quality of the extraction procedure, overall reagent integrity, and presence of PCR-inhibitory substances. A no template control (NTC) is included on each plate to serve as a negative control. Positive controls for each target are also provided and are to be included in each assay run. The targeted regions of viral RNA are transcribed into complimentary DNA (cDNA) and amplified by the polymerase chain reaction (PCR). The fluorescently labeled probes anneal to amplified DNA fragments and the fluorescent signal intensity is monitored by the ABI 7500 Fast Dx instrument during each PCR cycle. Amplification of target is recorded as increase of fluorescence over time in comparison to background signal.

# Interpretation of Results

Extraction and Control Results and Interpretation

# No Template Control (NTC)

The NTC consists of using nuclease-free water in the rRT-PCR reactions instead of RNA. The NTC reactions for all primer and probe sets should not exhibit fluorescence growth curves that cross the threshold line. If any of the NTC reactions exhibit a growth curve that crosses the cycle threshold, sample contamination may have occurred. Invalidate the run and repeat the assay with strict adherence to the guidelines.

# Influenza 2009 A(H1N1)pdm Positive Control

The positive control consists of an influenza 2009 H1N1 influenza virus and cultured human cells (A549). Purified RNA from the positive control will yield a positive result with the following primer and probe sets: InfA, pdm InfA, pdm H1, and RP.

# Human Specimen Control (HSC) (Extraction Control)

The HSC control consists of noninfectious cultured human cell (A549) material. The HSC is used as a RNA extraction procedural control to demonstrate successful recovery of RNA as well as extraction reagent integrity. Purified RNA from the HSC should yield a positive result with the RP primer and probe set and negative results with all influenza specific markers.

# Specimen Results and Interpretation

# RNase P (RP)

All clinical samples should exhibit fluorescence growth curves in the RP reaction that cross the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ , thus indicating the presence of the human RNase P gene. Failure to detect RP in any clinical specimens may indicate:

Improper extraction of nucleic acid from clinical materials resulting in loss of RNA and/or RNA degradation • Absence of sufficient human cellular material due to poor collection or loss of specimen integrity Improper assay set up and execution Reagent or equipment malfunction

If the RP assay does not produce a positive result for human clinical specimens, interpret as follows:

If the InfA, pdm InfA, and pdm H1 are positive, even in the presence of a negative RP, the influenza result should be considered valid. It is possible, that some samples may fail to exhibit RP growth curves due to low cell numbers in the original clinical sample. A negative RP signal does not preclude the presence of influenza virus RNA in a clinical specimen.

If all influenza markers AND RP are all negative for the specimen, the assay is “inconclusive” for the specimen. If residual specimen is available, repeat the extraction procedure and repeat the test. If all markers remain negative after retest, report the results as “inconclusive” and a new specimen should be collected if possible.

The RP assay may be negative when testing virus culture samples.

# Influenza Markers (InfA, pdm InfA, and pdm H1)

When all controls exhibit the expected performance, a specimen is considered negative if influenza marker growth curves DO NOT cross the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ and RP growth curve does cross the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ .

When all controls exhibit the expected performance, a specimen is considered positive for influenza A if the influenza marker (InfA) growth curve crosses the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ and the pdm InfA, and pdm H1markers DO NOT cross the threshold line within 38 cycles. RP may be positive or negative as described above.

When all controls exhibit the expected performance and none of the growth curves for the influenza markers or the RP marker cross the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ , the result is inconclusive. The extracted RNA from the specimen should be re-tested. If residual RNA is not available, re-extract RNA from residual specimen and re-test. If the re-tested sample is negative for all markers and all controls exhibit the expected performance, the result is “Inconclusive.”

When all controls exhibit expected performance and growth curves for InfA, pdm InfA, and pdm H1 markers cross the threshold line within 38 cycles $\left( < 3 8 \mathrm { C t } \right)$ , report the specimen to be “Positive for Influenza 2009 A/H1N1.”

CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel Users Guide for Interpretation of Results – Quick Reference and Reporting   

<table><tr><td rowspan=1 colspan=1>InfATarget</td><td rowspan=1 colspan=1>pdm InfATarget</td><td rowspan=1 colspan=1>pdm H1Target</td><td rowspan=1 colspan=1>RPTarget</td><td rowspan=1 colspan=1>Report</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>&quot;Influenza A Not Detected&quot;</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Positive for Influenza A,Influenza 2009 A/H1 Virus not detected&quot;</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Positive for Influenza A,Positive for Influenza 2009 A/H1 Virus&quot;</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Positive for Influenza A,Inconclusive for Influenza 2009 A/H1Virus&quot;</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Positive for Influenza A,Inconclusive for Influenza 2009 A/H1Virus&quot;*Reflex Testing with the CDC rRT-PCR FluPanel IVD Required</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Negative for Influenza A,*Inconclusive for 2009 A/H1 InfluenzaVirus&quot;</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>±</td><td rowspan=1 colspan=1>&quot;Negative for Influenza A,*Inconclusive for Influenza 2009 A/H1Virus&quot;</td></tr><tr><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>Inconclusive Test*Likely Poor Extraction or Specimen Quality</td></tr></table>

\* When an inconclusive result is obtained, re-extract the specimen and test the newly extracted RNA (recommended) or re-test the extracted RNA

# M. Performance Characteristics:

Analytical performance:

a) Precision/Reproducibility:

The precision and reproducibility of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel were evaluated at six laboratory sites. Each testing site assessed a panel of four simulated samples at relative moderate, low (near the assay lower limit of detection range), and “high negative” viral RNA concentration, and a negative sample. The

panels and assay controls were tested at each site by two operators on 5 different days within a 10-day period. Each site tested one of the six extraction methods associated with this device. Simulated samples in the qualification panel used in the reproducibility evaluation were:

Sample #1 Influenza 2009 H1N1influenza moderate viral RNA titer range Sample #2 Influenza 2009 H1N1 influenzalow viral RNA titer range Sample #3 Influenza 2009 H1N1 influenza “high negative” RNA titer range Sample #4 Influenza Negative (uninfected A549 cells)

Summary of Reproducibility Study for the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td rowspan=2 colspan=2>N=10 tests</td><td rowspan=1 colspan=3>bioMerieux NucliSENSeassyMAG</td><td rowspan=1 colspan=3>Roche MagNA Pure LC TNA</td><td rowspan=1 colspan=3>Roche MagNA PureCompact NA</td><td rowspan=1 colspan=3>Qiagen QlAcube ViralRNA</td><td rowspan=1 colspan=3>Qiagen Manual ViralRNA</td><td rowspan=1 colspan=3>Roche MagNA PureCompact RNA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Leeenpepecerest</td><td rowspan=1 colspan=1>A &#x27;</td><td rowspan=1 colspan=1>Ae</td><td rowspan=1 colspan=1>Aeeemm pepeceonrest</td><td rowspan=1 colspan=1>UAe</td><td rowspan=1 colspan=1>$A</td><td rowspan=1 colspan=1>Lueeem pepeceeW!hresst</td><td rowspan=1 colspan=1>0e</td><td rowspan=1 colspan=1>$A</td><td rowspan=1 colspan=1>Aeeenan pepeccenrest</td><td rowspan=1 colspan=1>0A</td><td rowspan=1 colspan=1>Ae</td><td rowspan=1 colspan=1>Aeeen pepeeeW!hresst</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>A%</td><td rowspan=1 colspan=1>Aeemn pepecceWhrest</td><td rowspan=1 colspan=1>A1</td><td rowspan=1 colspan=1>%A%</td><td rowspan=1 colspan=1>ReeemnTott*</td><td rowspan=1 colspan=1>I %96</td></tr><tr><td rowspan=1 colspan=2>No TemplateControl</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=2 colspan=1>Sample 1</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.52</td><td rowspan=1 colspan=1>2.59</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.53</td><td rowspan=1 colspan=1>2.55</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.99</td><td rowspan=1 colspan=1>4.23</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.92</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.85</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.54</td><td rowspan=1 colspan=1>2.12</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Pafm </td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.49</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.98</td><td rowspan=1 colspan=1>2.69</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.63</td><td rowspan=1 colspan=1>3.10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.68</td><td rowspan=1 colspan=1>2.25</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.72</td><td rowspan=1 colspan=1>1.59</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.81</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=2 colspan=1>moderate</td><td rowspan=1 colspan=1>pm</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.21</td><td rowspan=1 colspan=1>2.12</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.23</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.98</td><td rowspan=1 colspan=1>2.31</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.63</td><td rowspan=1 colspan=1>1.48</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.09</td><td rowspan=1 colspan=1>1.69</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.38</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.78</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.79</td><td rowspan=1 colspan=1>2.65</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.27</td><td rowspan=1 colspan=1>2.51</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.55</td><td rowspan=1 colspan=1>2.43</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.82</td><td rowspan=1 colspan=1>2.45</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.43</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=4 colspan=1>Sample 2low</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.10</td><td rowspan=1 colspan=1>3.26</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>31.75</td><td rowspan=1 colspan=1>35.61</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>36.12</td><td rowspan=1 colspan=1>4.11</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.95</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.49</td><td rowspan=1 colspan=1>3.14</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.48</td><td rowspan=1 colspan=1>2.88</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>88.5-99.6</td></tr><tr><td rowspan=1 colspan=1>onfa </td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.50</td><td rowspan=1 colspan=1>2.48</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>30.42</td><td rowspan=1 colspan=1>35.27</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>31.50</td><td rowspan=1 colspan=1>35.27</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.91</td><td rowspan=1 colspan=1>3.32</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>33.89</td><td rowspan=1 colspan=1>3.27</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.48</td><td rowspan=1 colspan=1>2.92</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>88.5-99.6</td></tr><tr><td rowspan=1 colspan=1>Ppm</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>35.19</td><td rowspan=1 colspan=1>3.10</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>28.85</td><td rowspan=1 colspan=1>52.94</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>29.34</td><td rowspan=1 colspan=1>52.77</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>34.66</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>36.12</td><td rowspan=1 colspan=1>2.18</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>36.57</td><td rowspan=1 colspan=1>3.70</td><td rowspan=1 colspan=1>54/60</td><td rowspan=1 colspan=1>79.5-96.2</td></tr><tr><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.44</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.33</td><td rowspan=1 colspan=1>2.78</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.44</td><td rowspan=1 colspan=1>3.27</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.41</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.34</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.57</td><td rowspan=1 colspan=1>5.11</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=4 colspan=1>Sample 3highnegative&quot;</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>7/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>54/60</td><td rowspan=1 colspan=1>79.5-96.2</td></tr><tr><td rowspan=1 colspan=1>Pnfm </td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>8/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>86.1-99.0</td></tr><tr><td rowspan=1 colspan=1>Pprm</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>58/60</td><td rowspan=1 colspan=1>88.5-99.6</td></tr><tr><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.04</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.08</td><td rowspan=1 colspan=1>4.92</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>31.12</td><td rowspan=1 colspan=1>2.60</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.89</td><td rowspan=1 colspan=1>2.09</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.33</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.45</td><td rowspan=1 colspan=1>3.31</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>Sample 4A549cells</td><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.68</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>30.20</td><td rowspan=1 colspan=1>5.68</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>32.10</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.06</td><td rowspan=1 colspan=1>2.65</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.60</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.33</td><td rowspan=1 colspan=1>4.00</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>HSC</td><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.51</td><td rowspan=1 colspan=1>2.30</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.80</td><td rowspan=1 colspan=1>4.48</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.55</td><td rowspan=1 colspan=1>3.11</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>25.89</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.30</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.35</td><td rowspan=1 colspan=1>4.98</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=2 colspan=1>Influenza2009A(H1N1)</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.71</td><td rowspan=1 colspan=1>2.10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>21.95</td><td rowspan=1 colspan=1>2.97</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.54</td><td rowspan=1 colspan=1>4.10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>21.62</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.66</td><td rowspan=1 colspan=1>2.63</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>24.95</td><td rowspan=1 colspan=1>2.21</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>onfa </td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>22.81</td><td rowspan=1 colspan=1>2.68</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>22.27</td><td rowspan=1 colspan=1>1.91</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.79</td><td rowspan=1 colspan=1>2.78</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>21.82</td><td rowspan=1 colspan=1>2.62</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>23.63</td><td rowspan=1 colspan=1>1.57</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>24.93</td><td rowspan=1 colspan=1>2.03</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=2 colspan=1>pdmositiveControl</td><td rowspan=1 colspan=1>pdmH1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>25.76</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>24.35</td><td rowspan=1 colspan=1>4.78</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>26.36</td><td rowspan=1 colspan=1>2.42</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>25.31</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>25.89</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.52</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr><tr><td rowspan=1 colspan=1>RP</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.84</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.44</td><td rowspan=1 colspan=1>3.23</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>29.03</td><td rowspan=1 colspan=1>4.03</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>27.35</td><td rowspan=1 colspan=1>1.84</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.59</td><td rowspan=1 colspan=1>2.35</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>28.82</td><td rowspan=1 colspan=1>3.26</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>94.0-100</td></tr></table>

b) Linearity/reportable range:

Not applicable

c) Traceability, stability, expected values (controls calibrators or methods)

Before analyzing the samples tested using the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel on the ABI 7500 Fast Dx RealTime PCR instrument, the ABI instrument must be prepared following the procedures described in the Package Insert for the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel and the ABI 7500 Fast Dx RealTime PCR instrument User Manual.

# Assay Controls

Quality Control procedures must be followed in conformance with local, state, and/or federal regulations or accreditation requirements and a laboratory’s standard quality control procedures. It is recommended that the user refer to CLSI document C24-A2.

The following control material should be used when testing with the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel:

Internal positive control, the human RNase P (RP) primer and probe set detects human RP and is used with each clinical specimen to indicate that adequate isolation of nucleic acid resulted from the extraction of the specimen. A positive RP assay result also ensures that common reagents and equipment are functioning properly, and demonstrates the absence of inhibitory substances. Influenza 2009 A(H1N1)pdm Positive Control (pdmPC) contains noninfectious (beta-propiolactone inactivated influenza 2009 H1N1 influenza virus and cultured human cells (A549). The positive control will yield a positive result with the following primer and probe sets: InfA, pdm InfA, pdm H1, and RP to ensure the detection of influenza 2009 H1N1 influenza virus. • A Human Specimen Control (HSC) is a noninfectious cultured human cell material for use as a RNA extraction procedural control. It is added to each batch of specimens to be extracted to monitor the successful recovery of RNA as well as extraction reagent integrity and presence of PCR-inhibitory substances. It should yield a positive result with the RP primer and probe set. A no template control (NTC) consists of nuclease-free water in the rRTPCR reactions instead of RNA. The NTC reactions for all primer and probe sets should not exhibit fluorescence growth curves that cross the threshold line. If any of the NTC reactions exhibit a growth curve that crosses the cycle threshold, sample contamination may have occurred. Invalidate the run and repeat the assay with stricter adherence to the

guidelines.

# d) Detection limit:

Analytical sensitivity was demonstrated for three influenza strains by determining the limit of detection (LoD) of each primer and probe set in the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel. The following strains were characterized and used for the LoD study:

A/California/07/09, 2009 H1N1 virus, $1 0 ^ { 8 . 4 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ stock concentration A/New York/18/09, 2009 H1N1 virus, $1 0 ^ { 7 . 8 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ stock concentration A/Brisbane/59/07, seasonal H1N1 virus, $1 0 ^ { 8 . 4 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ stock concentration

Five-fold serial dilutions of virus isolated from characterized embryonated chicken egg (ECE)-grown influenza viruses were tested five times $( \mathrm { N } { = } 5 )$ to identify an end-point for detection with each influenza primer and probe set. RNA was extracted from each virus using the Roche MagNA Pure Compact RNA kit. Once the end point or range was established, the end-point dilution and one dilution above and below the end point was tested and repeated twenty times for each virus and primer and probe set. The limit of detection for each primer and probe set was calculated to indicate the range of lowest detectable concentration of influenza virus $\mathrm { ( E I D _ { 5 0 } / m L ) }$ with a $9 5 . 0 \%$ or greater positivity. The limit of detection for each primer and probe set was determined using the results of the higher virus concentration of the two, type and subtype markers.

Limit of Detection of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Limit of Detection (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1Virus</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdmH1</td><td rowspan=1 colspan=1>FinalLoD</td></tr><tr><td rowspan=1 colspan=1>A/CA/07/2009</td><td rowspan=1 colspan=1>10 1.6</td><td rowspan=1 colspan=1>10 1.6</td><td rowspan=1 colspan=1>10 1.6</td><td rowspan=1 colspan=1>10 1.6</td></tr><tr><td rowspan=1 colspan=1>A/NY/18/2009</td><td rowspan=1 colspan=1>101.3</td><td rowspan=1 colspan=1>10 0.6</td><td rowspan=1 colspan=1>10 1.3</td><td rowspan=1 colspan=1>10 1.3</td></tr></table>

e) Analytical specificity:

# Inclusivity

To demonstrate the specificity of the primer and probe sets to detect a diverse population of 2009 H1N1 influenza viruses, ten 2009 H1N1 influenza viruses propagated in either Madin-Darby canine kidney (MDCK) cells in culture or in ECE as described previously in this section, were grown to high titer, and harvested. Each virus was subsequently serially diluted to approximately $1 0 ^ { 2 } \mathrm { I D } _ { 5 0 } / \mathrm { m L }$ , near the limit of detection of the assay. The diluted $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ influenza viruses were extracted using Roche MagNA Pure Compact RNA kit and were tested in triplicate using the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel.

2009 H1N1 Influenza Viruses Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td rowspan=2 colspan=1>2009 A/H1N1 Virus</td><td rowspan=1 colspan=2>Stock Virus Titer</td><td rowspan=2 colspan=1>ID50/mLTested</td><td rowspan=2 colspan=1>AverageInfA CtValue(N=3)</td><td rowspan=2 colspan=1>AveragepdmInfA CtValue(N=3)</td><td rowspan=2 colspan=1>Averagepdm H1CtValue(N=3)</td></tr><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>EID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/MEXICO/4108/2009</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 8.5</td><td rowspan=1 colspan=1>102.5</td><td rowspan=1 colspan=1>33.46</td><td rowspan=1 colspan=1>36.20</td><td rowspan=1 colspan=1>32.86</td></tr><tr><td rowspan=1 colspan=1>A/CALIFORNIA/8/2009</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 9.2</td><td rowspan=1 colspan=1>102.2</td><td rowspan=1 colspan=1>29.75</td><td rowspan=1 colspan=1>31.57</td><td rowspan=1 colspan=1>28.80</td></tr><tr><td rowspan=1 colspan=1>A/CALIFORNIA/7/2009</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 8.4</td><td rowspan=1 colspan=1>10 2.4</td><td rowspan=1 colspan=1>34.16</td><td rowspan=1 colspan=1>35.28</td><td rowspan=1 colspan=1>33.78</td></tr><tr><td rowspan=1 colspan=1>A/CALIFORNIA/04/2009</td><td rowspan=1 colspan=1>10 7.9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>101.9</td><td rowspan=1 colspan=1>32.26</td><td rowspan=1 colspan=1>33.54</td><td rowspan=1 colspan=1>32.13</td></tr><tr><td rowspan=1 colspan=1>A/TEXAS/48/2009</td><td rowspan=1 colspan=1>10 8.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 2.0</td><td rowspan=1 colspan=1>32.20</td><td rowspan=1 colspan=1>33.14</td><td rowspan=1 colspan=1>32.06</td></tr><tr><td rowspan=1 colspan=1>A/WASHINGTON/29/2009</td><td rowspan=1 colspan=1>107.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 2.5</td><td rowspan=1 colspan=1>30.58</td><td rowspan=1 colspan=1>32.35</td><td rowspan=1 colspan=1>30.10</td></tr><tr><td rowspan=1 colspan=1>A/SOUTHCAROLINA/18/2009</td><td rowspan=1 colspan=1>10 6.6</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 2.6</td><td rowspan=1 colspan=1>25.44</td><td rowspan=1 colspan=1>24.55</td><td rowspan=1 colspan=1>26.90</td></tr><tr><td rowspan=1 colspan=1>A/NEWYORK/18/2009</td><td rowspan=1 colspan=1>,</td><td rowspan=1 colspan=1>10 7.8</td><td rowspan=1 colspan=1>10 2.7</td><td rowspan=1 colspan=1>27.95</td><td rowspan=1 colspan=1>26.42</td><td rowspan=1 colspan=1>28.23</td></tr><tr><td rowspan=1 colspan=1>A/ENGLAND/195/2009</td><td rowspan=1 colspan=1>10 7.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10 2.0</td><td rowspan=1 colspan=1>26.92</td><td rowspan=1 colspan=1>27.03</td><td rowspan=1 colspan=1>29.36</td></tr><tr><td rowspan=1 colspan=1>A/NORTHCAROLINA/39/2009</td><td rowspan=1 colspan=1>10 7.7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>10 2.7</td><td rowspan=1 colspan=1>27.23</td><td rowspan=1 colspan=1>26.47</td><td rowspan=1 colspan=1>27.96</td></tr></table>

# Exclusivity with Seasonal Influenza Viruses

Characterized seasonal influenza specimens representing various geographical locations (12-A/H1, 12-A/H3, 12-Influenza B) were tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel to demonstrate the specificity of the primer and probe sets. Viruses were propagated in ECE or in MDCK cells in culture, grown to high titer, and harvested. RNA was extracted using the automated Roche MagNA Pure Compact RNA kit. The purified RNA was tested once with each primer and probe set in the panel following the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel protocol.

Seasonal Influenza Viruses Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Titer(logTCID50/mL)</td><td colspan="1" rowspan="1">InfA</td><td colspan="1" rowspan="1">pdmInfA</td><td colspan="1" rowspan="1">pdmH1</td></tr><tr><td colspan="1" rowspan="6">H1N1</td><td colspan="1" rowspan="1">A/JIANGXI/160/2005</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/SOLOMON ISLANDS/03/2006</td><td colspan="1" rowspan="1">6.2</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/TAIWAN/42/2006</td><td colspan="1" rowspan="1">4.7</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/FUKUSHIMA/141/2006</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1744/2007</td><td colspan="1" rowspan="1">5.3</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1729/2007</td><td colspan="1" rowspan="1">4.8</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="6">′</td><td colspan="1" rowspan="1">A/MEXICO/1677/2007</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/949/2007</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/BANGLADESH/7286/2007</td><td colspan="1" rowspan="1">6.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/2010/2007</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/BRISBANE/59/2007</td><td colspan="1" rowspan="1">8.4 (EID50/mL)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/PARAGUAY/61/2009</td><td colspan="1" rowspan="1">7.2</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="12">H3N2</td><td colspan="1" rowspan="1">A/HAWAII/08/2006</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/WISCONSIN/03/2007</td><td colspan="1" rowspan="1">8.2</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/HENAN/JINSHUI/147/2007</td><td colspan="1" rowspan="1">8.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/BRISBANE/10/2007</td><td colspan="1" rowspan="1">6.8</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/922/2007</td><td colspan="1" rowspan="1">7.8</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/AFGHANISTAN/2903/2008</td><td colspan="1" rowspan="1">5.0</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1938/2007</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1995/2007</td><td colspan="1" rowspan="1">4.3</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/ANHUI/1239/2005</td><td colspan="1" rowspan="1">8.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1842/2007</td><td colspan="1" rowspan="1">6.1</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/PERTH/16/2009</td><td colspan="1" rowspan="1">8.2 (EID50/mL)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/WISCONSIN/15/2009</td><td colspan="1" rowspan="1">8.1 (EID50/mL)</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="12">Inf B</td><td colspan="1" rowspan="1">B/FLORIDA/04/2006</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/CHONGQING/YONGCHUAN18/2007</td><td colspan="1" rowspan="1">7.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/FLORIDA/02/2006</td><td colspan="1" rowspan="1">6.0</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/PENNSYLVANIA/05/2007</td><td colspan="1" rowspan="1">6.6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/BANGLADESH/5972/2007</td><td colspan="1" rowspan="1">4.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/BANGLADESH/3461/2007</td><td colspan="1" rowspan="1">3.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/BANGLADESH/7110/2007</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/MEXICO/1819/2007</td><td colspan="1" rowspan="1">5.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/TEXAS/17/2007</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B/TEXAS/03/2007</td><td colspan="1" rowspan="1">5.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/BRISBANE/60/2008</td><td colspan="1" rowspan="1">8.5 (EID50/mL)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/TEXAS/26/2008</td><td colspan="1" rowspan="1">8.2 (EID50/mL)</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

# Exclusivity with Influenza Viruses with Pandemic Potential

To demonstrate the ability of the influenza A primer and probe sets to detect influenza A/H5 viruses with pandemic potential, the reactivity of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel was tested with ten avian A/H5 influenza viruses that have been shown to infect humans. The avian A/H5 influenza viruses were grown and harvested from allantoic/amniotic fluid from infected ECE and titered. Each virus was extracted using the Roche MagNA Pure Compact RNA Isolation kit, and the purified RNA was tested in triplicate with each marker following the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel protocol.

Avian A/H5 Influenza Viruses Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Clade</td><td rowspan=1 colspan=1>Titer(logEID50/mL)</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdmH1</td></tr><tr><td rowspan=10 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Japanese white eye/HongKong/1038/2006</td><td rowspan=1 colspan=1>2.3.4</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Duck/Hunan/795/2002</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Yunnan/1251/2003</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Common magpie/HongKong/645/2006</td><td rowspan=1 colspan=1>2.3.2</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Vietnam/1194/2004</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Egypt/321/2007</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Anhui/1/2005</td><td rowspan=1 colspan=1>2.3.4</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/India/NIV3487/2006</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>9.3</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Chicken/Vietnam/NCVD-016/2008</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Cambodia/R040505/2007</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

To demonstrate the ability of the influenza A primer and probe sets to detect potential pandemic influenza A viruses, the reactivity of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel was also tested with fifteen non-human influenza viruses that have been shown to infect humans. The non-human influenza viruses were propagated in ECE, grown to high titer, and harvested as described previously. Each virus was extracted using Roche MagNA Pure Compact RNA kit and the purified RNA was tested in triplicate with each marker following the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel protocol.

Animal Influenza Viruses Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Titer (log)EID50/mL)</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdmH1</td></tr><tr><td rowspan=1 colspan=1>EQUINE</td><td rowspan=1 colspan=1>A/equine/Ohio/01/2003</td><td rowspan=1 colspan=1>EQ-H3N8</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>SWINE</td><td rowspan=1 colspan=1>A/Maryland/12/1991</td><td rowspan=1 colspan=1>SW-H1N1</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+(3/3)</td></tr><tr><td rowspan=1 colspan=1>SWINE</td><td rowspan=1 colspan=1>A/Iowa/1/2006</td><td rowspan=1 colspan=1>SW-H1N1</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>+(3/3)</td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chicken/Pennsylvania/298101-4/2004</td><td rowspan=1 colspan=1>H2N2</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>CANINE</td><td rowspan=1 colspan=1>A/canine/Florida/43/2004</td><td rowspan=1 colspan=1>H3N8</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chicken/Alabama/1975</td><td rowspan=1 colspan=1>H4N8</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chucker/Minnesota/14591-7/1998</td><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/duck/Singapore-Q/F119-3/1997</td><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>8.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/duck/Pennsylvania/1969</td><td rowspan=1 colspan=1>H6N1</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chicken/California/32213-1/2000</td><td rowspan=1 colspan=1>H6N2</td><td rowspan=1 colspan=1>9.1</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chicken/New York/13237-6/1998</td><td rowspan=1 colspan=1>H6N8</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/mallard/NL/12/2000 IB-CDC-1</td><td rowspan=1 colspan=1>H7N7</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/turkey/Wisconsin/66</td><td rowspan=1 colspan=1>H9N2</td><td rowspan=1 colspan=1>9.7</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AVIAN</td><td rowspan=1 colspan=1>A/chicken/New Jersey/15906-9/1996</td><td rowspan=1 colspan=1>H11N1</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SWINE</td><td rowspan=1 colspan=1>A/swine/Wisconsin/125/1997</td><td rowspan=1 colspan=1>SW-H1N1</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>+(3/3)</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr></table>

Cross reactivity with Common Non-Influenza Human Respiratory Pathogens

Analytical specificity was evaluated by testing each primer/probe set within the device panel with nucleic acids extracted from 34 organisms (15 viruses, 18 bacteria, and 1 yeast) representing common respiratory pathogens or flora commonly present in specimens collected from the nasopharynx region. Each sample (bacteria and virus) was extracted using either the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I or the Roche MagNA Pure Compact RNA kit. Each nucleic acid sample was tested once following the testing procedure previously described for the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel.

# Common Respiratory Bacteria and Yeast Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Titer(logCFU/mL)</td><td rowspan=1 colspan=1>InfA</td><td rowspan=1 colspan=1>pdmInfA</td><td rowspan=1 colspan=1>pdm H1</td></tr><tr><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2001-21-196</td><td rowspan=1 colspan=1>Candida albicans (yeast)</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>40 IFU/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Corynebacterium diphtheriae</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>K12</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>9.6</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>7790-06</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M15709</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>6.4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M15757</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>H37Rv</td><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>95 ng/μl L</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>7.7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>M2578</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>10.7</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>10.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>249-06</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>6.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SS1672</td><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>8.4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

Common Respiratory Viruses Tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Titer(logTCID50/mL)</td><td colspan="1" rowspan="1">InfA</td><td colspan="1" rowspan="1">pdmInfA</td><td colspan="1" rowspan="1">pdm H1</td></tr><tr><td colspan="1" rowspan="1">Echo 6</td><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">6.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Ad. 71</td><td colspan="1" rowspan="1">Human Adenovirus, type 1</td><td colspan="1" rowspan="1">9.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">S-1058</td><td colspan="1" rowspan="1">Human Adenovirus, type 7a</td><td colspan="1" rowspan="1">7.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">50.4 ng/μl L</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">Human Coronavirus</td><td colspan="1" rowspan="1">31.6 ng/μl L</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">Human Rhinovirus A</td><td colspan="1" rowspan="1">5.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Human Parainfluenza1 virus</td><td colspan="1" rowspan="1">3.0 ng/µl L</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">Human Parainfluenza 2 virus</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">Human Parainfluenza 3 virus</td><td colspan="1" rowspan="1">7.9</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CH93-18b</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus(RSV)</td><td colspan="1" rowspan="1">6.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">KOS</td><td colspan="1" rowspan="1">Herpes Simplex Virus</td><td colspan="1" rowspan="1">8.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AV92-3</td><td colspan="1" rowspan="1">Varicella-zoster Virus</td><td colspan="1" rowspan="1">4.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">1.7 ng/μl L</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">5.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enders</td><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">7.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AD-169</td><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">6.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

No cross reactivity was detected for InfA, pdm InfA, or pdm H1.

# f) Assay cut-off:

To assess the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel assay cut-off, the data from the limit of detection study was analyzed. Data were plotted as virus concentration (lo $\mathrm { ; E I D _ { 5 0 } / m L }$ ) versus cycle threshold $\mathrm { ( C t ) }$ value at or below the limits of detection for each marker as shown in Figures 1 through 3. The average $\mathrm { C t }$ value at the lower end of the range of the limit of detection plus two standards of deviation afforded the Ct value cut-off for each of the markers in the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel as shown by the red line $( \mathrm { I n f A } = 3 9 $ ; pdm InfA $=$ 37; pdm $\mathrm { H } 1 = 4 0$ ).

![](images/a1c8a0a37160475464404cc926c642671c826f928c216d180b1fb82ee6c2a585.jpg)  
Figure 1: Assay Cut-Off Plot using Limit of Detection Data for the InfA Primer and Probe Set

![](images/e115b93739c578d262404229f868259c06aef643d6e63f8d9046f295ed3e960e.jpg)  
Figure 2: Assay Cut-Off Plot using Limit of Detection Data for the pdm InfA Primer and Probe Set

![](images/e348c80bf794f15db74380f9f8449ffa7030cbefa2ed3ab93194f31a050dd46b.jpg)  
Figure 3: Assay Cut-Off Plot using Limit of Detection Data for the pdm H1 Primer and Probe Set

Based on this analysis, the analytical cutoff value defined for the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel was less than 38. These data indicate that the InfA and pdm H1 primer and probe sets may be more sensitive. However, setting the cutoff Ct value too high could result in false positive results. The clinical study also supports this cutoff value.

2. Comparison studies:

a. Method comparison with gold standard/reference method: See clinical studies

b. Matrix Comparison: Not Applicable

c.Performance in Fresh vs. Frozen Clinical Specimens:

To determine the effect of a freeze/thaw cycle of a specimen on the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel result, a comparison study was performed utilizing a subset of specimens collected prospectively for the 2009-2010 clinical evaluation of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel. Based upon the initial CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel and virus culture results, fifty- one positive and twenty negative specimens were chosen among three different U.S. Public Health Laboratories. Frozen aliquots of these specimens were

thawed, extracted using the bioMérieux NucliSENS® easyMAG® (California), the manual QIAamp $^ \mathrm { \textregistered }$ Viral RNA Mini Kit (Massachuesettes), or the Roche MagNA Pure LC 2.0 with Total Nucleic Acid (TNA) Kit (Wisconsin), and retested using the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel. The study showed $100 \%$ concordance (qualitative) between the initial CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel result and the result after one freeze/thaw cycle.

The scatter plots of Ct values of four markers (Inf A, pdm Inf A, pdm H1, RP)

![](images/6e3a21ef727b6fe6fdc1a5a2bf4eb98ebbb43dc337fb4cf67f582a346dc8bfbb.jpg)

Deming regression was performed to account for the measurement errors for fresh and frozen specimens. Scatter plot demonstrates the overall relationship between fresh and frozen specimens.

The bias plots for ΔCt (Fresh Ct – Frozen Ct) of four markers (Inf A, pdm Inf A, pdm H1, RP)

![](images/214f650629d7ae465601a922e2f037dfb55ca9de906550afb49f41fd8d667e7d.jpg)

Bias plot displays the difference between fresh and frozen specimens versus their average.

# 3. Clinical studies:

Performance characteristics of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel were established during a prospective study at 8 U.S. public health laboratories and a Department of Defense (DoD) laboratory during the 2009-2010 respiratory virus season (February-April). Samples used for this study were collected for routine influenza testing at each site from individuals who were symptomatic with influenza-like illness (ILI) and included both upper and lower respiratory specimen types. Because of the low prevalence of influenza, retrospective specimens were used to supplement prospectively collected specimens.

The reference methods utilized in this study were virus culture with Immunofluorescent Antibody (IFA) or Direct Fluorescent Antibody (DFA) for identification of influenza type and bi-directional sequencing for confirmation of 2009 H1N1 influenza subtype. InfA analyte results from the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel were compared to virus culture results in the analysis. Sequencing was performed only with specimens that were first identified as positive for influenza A by virus culture. Pdm InfA and pdm H1 analyte results from the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel were compared to bi-directional sequencing results in the analysis.

A total of 1901 patient specimens were evaluated at the nine clinical testing sites: 1191 nasopharyngeal swabs (NPS) and nasal swabs (NS), 50 throat swabs, 519 nasal washes and nasal aspirates, 99 dual NPS/TS, and 42 lower respiratory specimens.

Specimens Received for the Performance Evaluation of the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel   

<table><tr><td rowspan=2 colspan=1>AgeGroup(years)</td><td rowspan=1 colspan=7>Total # Tested per Age Group and Specimen Type</td><td rowspan=2 colspan=1>Total #Testedper AgeGroup</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>NS</td><td rowspan=1 colspan=1>TS</td><td rowspan=1 colspan=1>NW</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>LowerRespiratory</td><td rowspan=1 colspan=1>NPS/TS</td></tr><tr><td rowspan=1 colspan=1>0-16</td><td rowspan=1 colspan=1>358</td><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>685</td></tr><tr><td rowspan=1 colspan=1>17-54</td><td rowspan=1 colspan=1>389</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>779</td></tr><tr><td rowspan=1 colspan=1>≥55</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>398</td></tr><tr><td rowspan=1 colspan=1>Age NotReported</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Total # ofEachSpecimenType</td><td rowspan=1 colspan=1>1021</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>462</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>99</td><td rowspan=1 colspan=1>1901</td></tr></table>

# Prospective Study

The U.S. World Health Organization (WHO) and National Respiratory and Enteric Virus Surveillance System (NREVSS) reported an average of positive influenza specimens at approximately $4 \%$ during the prospective clinical evaluation of the CDC Influenza 2009 A(H1N1)pdm Real-Time RT-PCR Panel. A total of 1323 specimens were collected and tested with the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel. Individual specimen types included in the analysis were 302 nasal aspirates and nasal washes, 877 nasopharyngeal swabs and nasal swabs, 65 dualcollected nasopharyngeal swabs/throat swabs, and 30 lower respiratory specimens.

The prospective study results are stratified by specimen type and are presented in the tables below.

Prospective Results from Nasal Aspirate (NA) and Nasal Wash (NW) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td colspan="2"></td><td colspan="4">Virus Culture</td></tr><tr><td></td><td>InfA Analyte</td><td>Influenza A Positive</td><td>Influenza A Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="4">RFP- 0 INIH</td><td>Influenza A Positive</td><td>19</td><td>2</td><td>21</td><td>100 % Sensitivity 83.2-100 (95% CI)</td></tr><tr><td>Influenza A Negative</td><td>0</td><td>281</td><td>281</td><td>99.3 % Specificity 97.5-99.8 (95% CI)</td></tr><tr><td>Total</td><td>19</td><td>283</td><td>302</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

<table><tr><td rowspan="2">2009 A/H1</td><td colspan="4">Comparator Method</td></tr><tr><td>pdm A and Influenza Positive</td><td>2009 A/H1 Influenza Negative</td><td>Total</td><td>Performance</td></tr><tr><td></td><td>pdm H1 Analyte 2009 A/H1 Influenza</td><td>16</td><td>4</td><td></td><td>100 % Sensitivity</td></tr><tr><td rowspan="4">CIH NN RFP-PRR</td><td>Positive</td><td></td><td></td><td>20</td><td>80.6-100 (95% CI)</td></tr><tr><td>2009 A/H1 Influenza</td><td>0</td><td>282</td><td>282</td><td>98.6 % Specificity 96.5-99.5 (95% CI)</td></tr><tr><td>Negative</td><td>16</td><td>286</td><td>302</td><td></td></tr><tr><td>Total</td><td></td><td></td><td></td><td></td></tr></table>

Prospective Results from Nasopharyngeal swab (NPS) and Nasal Swab (NS) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td colspan="2"></td><td colspan="4">Virus Culture</td></tr><tr><td></td><td>InfA Analyte</td><td>Influenza A Positive</td><td>Influenza A Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="4">RFP-R 0 INIH</td><td>Influenza A Positive</td><td>61</td><td>31</td><td>92</td><td>96.8 % Sensitivity 89.1-99.1 (95% Cl)</td></tr><tr><td>Influenza A Negative</td><td>2</td><td>783</td><td>785</td><td>96.2 % Specificity 94.6-97.3</td></tr><tr><td>Total</td><td>63</td><td>814</td><td>877</td><td>(95% CI)</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

<table><tr><td rowspan="2">pdm A and</td><td colspan="4">Comparator Method</td></tr><tr><td>2009 A/H1 Influenza Positive</td><td>2009 A/H1 Influenza</td><td></td><td>Performance</td></tr><tr><td rowspan="5">IH N RFP-PRR</td><td>pdm H1 Analyte 2009 A/H1 Influenza</td><td>57</td><td>Negative 27</td><td>Total 84</td><td>100 % Sensitivity</td></tr><tr><td>Positive</td><td></td><td></td><td></td><td>93.7-100 (95% CI) 96.7 % Specificity</td></tr><tr><td>2009 A/H1 Influenza Negative</td><td>0</td><td>793</td><td>793</td><td>95.3-97.7 (95% Cl)</td></tr><tr><td>Total</td><td>57</td><td>820</td><td>877</td><td></td></tr></table>

Prospective Results from Nasopharygeal/Throat Swab (NP/TS) Dual-collected Specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td colspan="2"></td><td colspan="4">Virus Culture</td></tr><tr><td></td><td>InfA Analyte</td><td>Influenza A Positive</td><td>Influenza A Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="4">RFP- 0 INIH</td><td>Influenza A Positive</td><td>6</td><td>0</td><td>6</td><td>100 % Sensitivity 61.0-100 (95% CI)</td></tr><tr><td>Influenza A Negative</td><td>0</td><td>59</td><td>59</td><td>100 % Specificity 93.9-100 (95% CI)</td></tr><tr><td>Total</td><td>6</td><td>59</td><td>65</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

<table><tr><td rowspan="2">pdm A and</td><td colspan="4">Comparator Method</td></tr><tr><td>2009 A/H1 Influenza Positive</td><td>2009 A/H1 Influenza Negative</td><td>Total</td><td>Performance</td></tr><tr><td></td><td>pdm H1 Analyte 2009 A/H1 Influenza</td><td>5</td><td>1</td><td>6</td><td>100 % Sensitivity</td></tr><tr><td rowspan="4">CIH NN RFPPPR</td><td>Positive</td><td></td><td></td><td></td><td>56.6-100 (95% CI)</td></tr><tr><td>2009 A/H1 Influenza Negative</td><td>0</td><td>59</td><td>59</td><td>98.3 % Specificity 91.1-99.7 (95% Cl)</td></tr><tr><td>Total</td><td>5</td><td>60</td><td>65</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

Prospective Results from Lower Respiratory Tract Specimens (BAL, BW, and TA) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td colspan="2"></td><td colspan="4">Virus Culture</td></tr><tr><td colspan="2">InfA Analyte</td><td>Influenza A Positive</td><td>Influenza A Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="4">RFP-CR 0 INIH</td><td>Influenza A Positive</td><td>5</td><td>4</td><td>9</td><td>83.3 % Sensitivity 43.6-97.0 (95% CI)</td></tr><tr><td>Influenza A Negative</td><td>1</td><td>20</td><td>21</td><td>83.3 % Specificity 64.1-93.3 (95% CI)</td></tr><tr><td>Total</td><td>6</td><td>24</td><td>30</td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr></table>

<table><tr><td colspan="2">2009 A/H1</td><td colspan="3">Comparator Method 2009 A/H1</td></tr><tr><td>pdm A and pdm H1 Analyte</td><td>Influenza Positive</td><td>Influenza Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="5">CIH NN RFP-PRR</td><td>2009 A/H1 Influenza</td><td>5</td><td>4</td><td>9</td><td>100 % Sensitivity</td></tr><tr><td>Positive 2009 A/H1 Influenza</td><td></td><td></td><td></td><td>56.6-100 (95% CI) 84.0 % Specificity</td></tr><tr><td>Negative</td><td>0</td><td>21</td><td>21</td><td>65.3-93.6 (95% CI)</td></tr><tr><td>Total</td><td>5</td><td>25</td><td>30</td><td></td></tr></table>

A total of 46 prospective and two retrospective throat swabs were tested. A total of eight (8) specimens were correctly identified as 2009 H1N1 influenza positive. Performance characteristics for 2009 H1N1 influenza virus using throat swab (TS) specimens were established testing a limited number of specimens. Users may wish to establish the sensitivity of this test for throat swab specimens by testing additional samples

# Retrospective Study

The prevalence of influenza decreased significantly during the prospective clinical evaluation of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel. This decrease prompted the supplementation of prospective specimens with archived specimens from earlier in the 2009-2010 influenza season. The archived specimens were selected for incorporation into the study if they met the same inclusion criteria as the prospective specimens and were also tested by virus culture at the time they were first received at each site. Individual specimen types included in the retrospective study were 214 nasal aspirates and nasal washes, 312 nasopharyngeal swabs and nasal swabs, 31 dual-collected nasopharyngeal swabs/throat swabs, and 10 lower respiratory specimens.

The retrospective study results are stratified by specimen type and are presented in the tables below

Retrospective Results from Nasal Aspirate (NA) and Nasal Wash (NW) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td rowspan=1 colspan=2>InfA Analyte</td><td rowspan=1 colspan=4>Virus CultureInfluenza A Influenza APositive     Negative      Total          Performance</td></tr><tr><td rowspan=3 colspan=1>CH GNNRFP-PC</td><td rowspan=3 colspan=1>Influenza A PositiveInfluenza A NegativeTotal</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>209</td><td rowspan=1 colspan=1>97.7 % PercentPositive Agreement94.6-99.0 (95% CI)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>214</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>214</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>pdm A andpdm H1 Analyte</td><td rowspan=1 colspan=4>Comparator Method2009 A/H1  2009 A/H1Influenza    InfluenzaPositive     Negative      Total           Performance</td></tr><tr><td rowspan=3 colspan=1>H GNNRFP-P</td><td rowspan=3 colspan=1>2009 A/H1 InfluenzaPositive2009 A/H1 InfluenzaNegativeTotal</td><td rowspan=1 colspan=1>203</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>208</td><td rowspan=1 colspan=1>99.0 % PercentPositive Agreement96.5-99.7 (95% CI)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>44.4% PercentNegative Agreement18.9-73.3 (95% CI)</td></tr><tr><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>214</td><td rowspan=1 colspan=1></td></tr></table>

Retrospective Results from Nasopharyngeal swab (NPS) and Nasal Swab (NS) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td rowspan=1 colspan=5>InfA Analyte</td><td rowspan=1 colspan=4>Virus CultureInfluenza A Influenza APositive     Negative      Total           Performance</td></tr><tr><td rowspan=2 colspan=1>CIH GNNRFP-PCR</td><td rowspan=3 colspan=4>Influenza A PositiveInfluenza A NegativeTotal</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>310</td><td rowspan=1 colspan=1>99.4 % PercentPositive Agreement97.7-99.8 (95% CI)</td></tr><tr><td rowspan=2 colspan=1>CIH GNNRFP-PCR</td><td rowspan=1 colspan=2>egati</td><td rowspan=1 colspan=2>ative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>312</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>312</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="2"></td><td colspan="4">Comparator Method 2009 A/H1 2009 A/H1</td></tr><tr><td></td><td>pdm A and pdm H1 Analyte</td><td>Influenza Positive</td><td>Influenza Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="5">C GNN RFP-C</td><td>2009 A/H1 Influenza Positive</td><td>301</td><td>6</td><td>307</td><td>99.7 % Percent Positive Agreement</td></tr><tr><td>2009 A/H1 Influenza</td><td>1</td><td>4</td><td>5</td><td>98.1-99.9 (95% CI) 40.0 % Percent Negative Agreement</td></tr><tr><td>Negative</td><td></td><td></td><td></td><td>16.8-68.7 (95% CI)</td></tr><tr><td>Total</td><td>302</td><td>10</td><td>312</td><td></td></tr></table>

Retrospective Results from Nasopharygeal/Throat Swab (NP/TS) Dual-collected Specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td rowspan=1 colspan=2>InfA Analyte</td><td rowspan=1 colspan=4>Virus CultureInfluenza A Influenza APositive      Negative      Total           Performance</td></tr><tr><td rowspan=3 colspan=1>CH GNNRFP-PR</td><td rowspan=3 colspan=1>Influenza A PositiveInfluenza A NegativeTotal</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>100 % Percent PositiveAgreement89.0-100 (95% CI)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA %</td></tr><tr><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="2">2009 A/H1</td><td colspan="3">Comparator Method 2009 A/H1</td></tr><tr><td>pdm A and pdm H1 Analyte</td><td>Influenza Positive</td><td>Influenza Negative</td><td>Total</td><td>Performance</td></tr><tr><td rowspan="6">CH GNN RFPR</td><td>2009 A/H1 Influenza</td><td>29</td><td>2</td><td>31</td><td>100 % Percent Positive Agreement</td></tr><tr><td>Positive</td><td></td><td></td><td></td><td>88.3-100 (95% CI)</td></tr><tr><td>2009 A/H1 Influenza Negative</td><td>0</td><td>0</td><td>0</td><td>NA</td></tr><tr><td>Total</td><td>29</td><td>2</td><td>31</td><td></td></tr></table>

Retrospective Results from Lower Respiratory Tract Specimens (BAL, BW, and TA) specimens for Universal Influenza A (InfA) and 2009 H1N1 (pdm InfA & pdm H1)   

<table><tr><td rowspan=1 colspan=2>InfA Analyte</td><td rowspan=1 colspan=4>Virus CultureInfluenza A Influenza APositive      Negative      Total           Performance</td></tr><tr><td rowspan=1 colspan=1>CIH GNNRFP-PR</td><td rowspan=3 colspan=1>Influenza A PositiveInfluenza A NegativeTotal</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100 % Percent PositiveAgreement72.3-100 (95% CI)</td></tr><tr><td rowspan=2 colspan=1>CIH GNNRFP-PR</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=2>pdm A andpdm H1 Analyte</td><td rowspan=1 colspan=4>Comparator Method2009 A/H1   2009 A/H1Influenza     InfluenzaPositive      Negative      Total           Performance</td></tr><tr><td rowspan=3 colspan=1>CH GNNRFP-PP</td><td rowspan=3 colspan=1>2009 A/H1 InfluenzaPositive2009 A/H1 InfluenzaNegativeTotal</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>100 % Percent PositiveAgreement72.3-100 (95% Cl)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td></tr></table>

The CDC 2009 A(H1N1)pdm rRT-PCR Panel Clinical Sensitivity, Specificity, and Percent Agreement Summary   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>InfA</td><td rowspan=1 colspan=3>2009 H1N1</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>Sensitivity% (95% Cl)</td><td rowspan=1 colspan=1>Specificity% (95% Cl)</td><td rowspan=1 colspan=1>PositivePercentAgreementRetrospective</td><td rowspan=1 colspan=1>Sensitivity% (95% Cl)</td><td rowspan=1 colspan=1>Specificity% (95% Cl)</td><td rowspan=1 colspan=1>PositivePercentAgreementRetrospective</td></tr><tr><td rowspan=1 colspan=1>NPS/NS</td><td rowspan=1 colspan=1>96.8(89.1-99.1)</td><td rowspan=1 colspan=1>96.2(94.6-97.3)</td><td rowspan=1 colspan=1>99.4(97.7-99.8)</td><td rowspan=1 colspan=1>100(93.7-100)</td><td rowspan=1 colspan=1>96.7(95.3-97.7)</td><td rowspan=1 colspan=1>99.7(98.1-99.9)</td></tr><tr><td rowspan=1 colspan=1>NA/NW</td><td rowspan=1 colspan=1>100(83.2-100)</td><td rowspan=1 colspan=1>99.3(97.5-99.8)</td><td rowspan=1 colspan=1>97.7(94.6-99.0)</td><td rowspan=1 colspan=1>100(80.6-100)</td><td rowspan=1 colspan=1>98.6(96.5-99.5)</td><td rowspan=1 colspan=1>99.0(96.5-99.7)</td></tr><tr><td rowspan=1 colspan=1>NPS/TS</td><td rowspan=1 colspan=1>100(61.0-100)</td><td rowspan=1 colspan=1>100(93.9-100)</td><td rowspan=1 colspan=1>100(89.0-100)</td><td rowspan=1 colspan=1>100(56.6-100)</td><td rowspan=1 colspan=1>98.3(91.1-99.7)</td><td rowspan=1 colspan=1>100(88.3-100)</td></tr><tr><td rowspan=1 colspan=1>BAL, TA,BWW</td><td rowspan=1 colspan=1>83.3(43.6-97.0)</td><td rowspan=1 colspan=1>83.3(64.1-93.3)</td><td rowspan=1 colspan=1>100(72.3-100)</td><td rowspan=1 colspan=1>100(56.6-100)</td><td rowspan=1 colspan=1>84.0(65.3-93.6)</td><td rowspan=1 colspan=1>100(72.3-100)</td></tr></table>

The overall call rate for the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel in the clinical evaluation at nine different sites was greater than $9 9 \%$ .

The clinical sensitivity of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel for all markers tested using upper respiratory specimens was greater than $9 6 \%$ and the clinical specificity for all markers was greater than $9 6 \%$ . The positive percent agreement was greater than $9 7 \%$ for all markers tested.

The clinical sensitivity and specificity of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel was also evaluated with lower respiratory specimens. The clinical sensitivity and specificity for all markers in the panel using these specimens were greater than $8 3 \%$ when compared to culture. The positive percent agreement of the CDC rRT-PCR 2009 A(H1N1)pdm Flu Panel was $100 \%$ .

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

# N. Instrument Name:

Applied Biosystems (ABI) 7500 Fast Dx Real-Time PCR (K082562)

# O. System Descriptions:

1. Modes of Operation:

The Applied Biosystems 7500 Fast Dx Real-Time PCR instrument integrates a thermal cycler, a fluorimeter and application specific software. The instrument houses the thermal cycler and the fluorimeter, while the application software is run on a PC that is attached to the instrument. Samples are placed in a tube strip or 96-well low-head space plate that is moved to a Peltier-based thermal block and positioned relative to the optics using a tray loading mechanism.

2. Software:

The Sequence Detection Software (SDS) version 1.4 for the 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-by-cycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamp-life monitoring and other instrument maintenance information

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification:

Sample ID is manually entered by user.

4. Specimen Sampling and Handling:

Not applicable

5. Calibration:

Not applicable

6. Quality Control:

Quality control is addressed for each specific assay to be run on the instrument (separately cleared).

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.