# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

# B. Purpose for Submission:

To obtain a substantial equivalence determination for a 510k for methicillin-resistant Staphylococcus aureus (MRSA) from nasal specimens

# C. Measurand:

MRSA DNA

# D. Type of Test:

The test utilizes NASBA™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA.

E. Applicant: bioMérieux, Inc

F. Proprietary and Established Names: NucliSENS EasyQ® MRSA Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866. 1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NQX- nucleic acid amplification test, DNA, methicillin resistant Staph aureus, direct specimen   
OOI- Nucleic acid amplification systems, real time

83 Microbiology

# H. Intended Use:

1. Intended use(s):

The NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) from nasal swabs in patients at risk for nasal colonization. The NucliSENS EasyQ® MRSA assay is performed on the NucliSENS EasyQ® platform.

The test utilizes NASBA™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA.

The NucliSENS EasyQ® MRSA assay is used as a screening tool to aid in the prevention and control of MRSA infections in health care institutions.

NucliSENS EasyQ® MRSA is not intended to diagnose, guide or monitor treatment for MRSA infections, or provide results of susceptibility to methicillin. A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

2. Indication(s) for use:

The NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) from nasal swabs in patients at risk for nasal colonization. The NucliSENS EasyQ® MRSA assay is performed on the NucliSENS EasyQ® platform.

The test utilizes NASBA™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA.

The NucliSENS EasyQ® MRSA assay is used as a screening tool to aid in the prevention and control of MRSA infections in health care institutions.

NucliSENS EasyQ® MRSA is not intended to diagnose, guide or monitor treatment for MRSA infections, or provide results of susceptibility to methicillin. A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

3. Special conditions for use statement(s): For Prescription Use only

4. Special instrument requirements:

NucliSENS EasyQ® MRSA assay protocol software (included in the NucliSENS   
EasyQ® MRSA assay kit)   
NucliSENS EasyQ® Analyzer   
NucliSENS EasyQ® Incubator II   
NucliSENS EasyQ® Director Software 2.6

# I. Device Description:

The NucliSENS EasyQ® MRSA assay kit contains reagents (nucleic acid primers, fluorescent probes, enzymes, buffers and other reagents), controls, software, accessories, and labeling (Instructions for Use) designed for use with the NucliSENS EasyQ System. The NucliSENS EasyQ System consists of NucliSENS EasyQ System instruments and software (NucliSENS EasyQ® Analyzer, NucliSENS EasyQ® Director software, and NucliSENS EasyQ® Incubator).

For the EasyQ MRSA assay kit, the indicated specimen is a nasal swab, collected using Copan nylon flocked swab. After transport to the laboratory in a dry collection tube, the swab is manually processed for lysis of cells and extraction of nucleic acids using lysis tubes provided in the EasyQ MRSA assay kit, in conjunction with a vortex instrument fitted with a specified vortex plate adaptor to accommodate the lysis tubes. The specimen lysates and controls serve as template material for the subsequent amplification reaction in the EasyQ System.

The NucliSENS EasyQ system is indicated for use solely for nucleic acid sequencebased amplification (NASBA) assays. Following specimen processing and nucleic acid extraction, an operator manually prepares amplification reactions, including a key semi-automated incubation step in the EasyQ Incubator.

Automated amplification and detection are performed simultaneously in a dedicated fluorescent reader, the NucliSENS EasyQ Analyzer. The Analyzer is attached to a computer, monitor, and printer. Amplification is detected by means of sequencespecific probes which each fluoresce upon binding of the probe to its target. Result calculation and reporting is performed automatically via dedicated operator software (NucliSENS EasyQ Director Software, in combination with assay-specific “assay protocols” provided on a CD-ROM in the reagent kit.) Results are based on the analysis of fluorescence signal curves measured by the NucliSENS EasyQ $^ \mathrm { \textregistered }$ Analyzer.

For the EasyQ MRSA assay, results of the Director Software analysis for each test are provided to users as 1 of 3 possible qualitative results for each test specimen: MRSA positive, MRSA negative, or invalid. Control tests yield qualitative output of valid or invalid.

The EasyQ MRSA kit includes 2 external controls: a blank control and a positive control to perform quality control for the amplification and detection portions of the test. The Instructions for Use also recommend use of specimen positive control (MRSA) and specimen negative control (MSSA) strains, which have been validated for use with the EasyQ MRSA test, to provide quality control for the specimen sample preparation processing steps of the test. Operators are instructed to invalidate all tests in a run if any control in the run yields an invalid result.

The NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA is developed for amplification of two targets simultaneously: the SCCmec cassette junction and the mecA gene. The NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA will report a MRSA positive result only if both targets, the SCCmec cassette junction and the mecA gene, are detected, thus reducing the risk of false positive detection for “mecA dropout”- methicillin-sensitive Staphylococcus aureus strains that lost the mecA gene but still harbor the cassette junction. Primers and beacons in the kit are complementary to the SCCmec cassette junction sequences of MREJ types i-v, vii, and xii.

Components of NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Composition</td><td rowspan=1 colspan=1>Number of tubes and tubedescription</td><td rowspan=1 colspan=1>Caps Color code</td></tr><tr><td rowspan=1 colspan=1>LYS Tub(lysis tube)</td><td rowspan=1 colspan=1>Proclin solution with glassbeads*</td><td rowspan=1 colspan=1>48 tubes1150 μl per tube</td><td rowspan=1 colspan=1>Uncolored captubes</td></tr><tr><td rowspan=1 colspan=1>REAGENT spheressachet(amplification/detectionreagents</td><td rowspan=1 colspan=1>Lyophilized white sphere:•  primers, molecular probesinhibition control:non-infectious DNA•   NASBA™ reagents(nucleotides, DTT, KCI,MgCl•   carbohydrateLyophilized orange sphere:Restriction enzymeBovine Serum Albumin**carbohydrate</td><td rowspan=1 colspan=1>6 tubes in foil pack with silica geldesiccant2 spheres per tube (white andorange)</td><td rowspan=1 colspan=1>Blue cap tubes</td></tr><tr><td rowspan=1 colspan=1>ENZI sphere sachet(enzyme reagents)</td><td rowspan=1 colspan=1>Lyophilized blue sphere :AMV-RT, RNase H,T7 RNA polymerase•   Bovine Serum Albumin**</td><td rowspan=1 colspan=1>6 tubes in foil pack with silica geldesiccant1 sphere per tube (blue)</td><td rowspan=1 colspan=1>Red cap tubes</td></tr><tr><td rowspan=1 colspan=1>REAGENTdil</td><td rowspan=1 colspan=1>Reagent spheres diluent(Tris/HCI, KCI, DMSO,PVP )</td><td rowspan=1 colspan=1>2 tubes.600 µl per tube</td><td rowspan=1 colspan=1>Blue cap tubes</td></tr><tr><td rowspan=1 colspan=1>ENZdil</td><td rowspan=1 colspan=1>Enzyme sphere diluent (Sorbitol,Bovine Serum Albumin**,Proclin</td><td rowspan=1 colspan=1>2 tubes.350 μl per tube</td><td rowspan=1 colspan=1>Red cap tubes</td></tr><tr><td rowspan=1 colspan=1>NW</td><td rowspan=1 colspan=1>NASBA ™ water with proclin fordissolution of the CONTROL+sphere and for the NASBA ™blank</td><td rowspan=1 colspan=1>2 tubes.1500 μl per tube</td><td rowspan=1 colspan=1>White cap tubes</td></tr></table>

Instruments/Software   

<table><tr><td rowspan=1 colspan=1>Component</td><td rowspan=1 colspan=1>Composition</td><td rowspan=1 colspan=1>Number of tubes and tubedescription</td><td rowspan=1 colspan=1>Caps Color code</td></tr><tr><td rowspan=1 colspan=1>CONTROL+spheresachet</td><td rowspan=1 colspan=1>NASBA ™M positive control DNA:lyophilized sphere containingnon infectious DNA plasmids</td><td rowspan=1 colspan=1>4 tubes in foil pack with silica geldesiccant1 sphere per tube (white)</td><td rowspan=1 colspan=1>White cap tubes</td></tr><tr><td rowspan=1 colspan=1>TBSTRCPSTR</td><td rowspan=1 colspan=1>8 tubes strip: 6 pieces8 cap strip: 12 pieces</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Not applicable</td></tr></table>

<table><tr><td>Product</td><td>Catalogue number</td></tr><tr><td>NucliSENS EasyQ® System It includes the following: NucliSENS EasyQ® Analyzer NucliSENS EasyQ® Incubator V2 NucliSENS EasyQ® Computer Monitor</td><td>4700016</td></tr></table>

Additional materials   

<table><tr><td rowspan=1 colspan=1>Product</td><td rowspan=1 colspan=1>Catalogue number</td></tr><tr><td rowspan=1 colspan=1>Micro tubes; 1.5 ml (500 x)</td><td rowspan=1 colspan=1>200294</td></tr><tr><td rowspan=1 colspan=1>0.2 ml 8-tube strips (125 pcs)</td><td rowspan=1 colspan=1>285048</td></tr><tr><td rowspan=1 colspan=1>0.2 ml 8-tube caps (125 pcs)</td><td rowspan=1 colspan=1>285051</td></tr><tr><td rowspan=1 colspan=1>EasyQ® Tube tray holder</td><td rowspan=1 colspan=1>45685049</td></tr><tr><td rowspan=1 colspan=1>EasyQ® Cap Installing Tool (Capping aid)</td><td rowspan=1 colspan=1>45685050</td></tr><tr><td rowspan=1 colspan=1>Pipetting aid</td><td rowspan=1 colspan=1>45685207</td></tr></table>

Interpretation of the assay results   

<table><tr><td rowspan=1 colspan=1>SCCmec cassette junction result (FAMsignal</td><td rowspan=1 colspan=1>pos</td><td rowspan=1 colspan=1>pos</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>neg</td></tr><tr><td rowspan=1 colspan=1>mecA result (Cyanine Dye signal)</td><td rowspan=1 colspan=1>pos</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>pos or neg</td><td rowspan=1 colspan=1>pos or neg</td></tr><tr><td rowspan=1 colspan=1>IC result (ROX signal)</td><td rowspan=1 colspan=1>pos or neg</td><td rowspan=1 colspan=1>pos or neg</td><td rowspan=1 colspan=1>pos</td><td rowspan=1 colspan=1>neg</td></tr><tr><td rowspan=1 colspan=1>Test validity</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=1 colspan=1>Test result</td><td rowspan=1 colspan=1>MRSA POS</td><td rowspan=1 colspan=1>MRSA NEG</td><td rowspan=1 colspan=1>MRSA NEG</td><td rowspan=1 colspan=1>Invalid</td></tr></table>

IC : Inhibition control, pos :positive, neg $:$ negative

Note: IC is co-amplified with the cassette junction. Therefore, when the cassette junction is positive but IC is negative, the test result by design is valid (2 co-amplified targets compete for the same primer(s)).

# J. Substantial Equivalence Information:

1. Predicate device name(s): Assay- BD GeneOhm MRSA (formerly IDI-MRSA)

2. Predicate 510(k) number(s):

k033415

3. Comparison with predicate:

Similarities   

<table><tr><td></td><td>Device</td><td>Predicate</td></tr><tr><td>Item</td><td>NucliSENS EasyQ® MRSA</td><td>BD GeneOhm MRSA (k033415)</td></tr><tr><td>Intended Use</td><td>Qualitative diagnostic screening for MRSA in nasal swab specimens</td><td>Same</td></tr><tr><td>Technological Principles</td><td>Manual specimen processing with automated amplification, detection,</td><td>Same</td></tr><tr><td>Probes</td><td>analysis and reporting of results Molecular Beacons</td><td>Same</td></tr><tr><td>Detection</td><td>Fluorogenic target-specific hybridization</td><td>Same</td></tr><tr><td>Analysis and reporting of results</td><td>Automated using diagnostic software</td><td>Same</td></tr><tr><td>DNA Target Sequences</td><td>Sequence incorporating the insertion site of Staphylococcal Cassette Chromosome mec (SCCmec) at orfX junction, with 6 variant sets primers and probes to target 7 MREJ types i, ii, ii, iv, v, vii, xii (target sequences for MREJ i and ii are identical)</td><td>Same</td></tr></table>

# Differences

<table><tr><td>Amplification</td><td>NASBA (Nucleic Acid Sequence-Based Amplification)</td><td>PCR (Polymerase Chain Reaction)</td></tr><tr><td>Instrument Additional DNA Target</td><td>NucleiSENS EasyQ® system</td><td>Cepheid SmartCycler®</td></tr><tr><td rowspan="2">Controls</td><td>mecA gene Internal control in each test, plus 2 external controls (positive and blank)</td><td>None One internal reagent control and external positive and negative controls required per</td></tr><tr><td>supplied in kit, plus 2 specimen controls (MRSA and MSSA) recommended for each run</td><td>run</td></tr></table>

EasyQ system components comparison Similarities   

<table><tr><td>System Component</td><td>NucliSENS EasyQ® MRSA</td><td>NucliSENS EasyQ Instrumentation and software (k093383)</td></tr><tr><td>Software</td><td>NucliSENS Director 2.6 software</td><td>Same</td></tr></table>

Similarities   

<table><tr><td>System Component</td><td>NucliSENS EasyQ® MRSA</td><td>NucliSENS EasyQ Instrumentation and software (k093383)</td></tr><tr><td>Incubator</td><td>Bath, incubator/water NucliSENS EasyQ incubator II</td><td>Same</td></tr><tr><td>Computer</td><td>NucliSENS EasyQ computer 110V</td><td>Same</td></tr><tr><td>Monitor</td><td>Monitor 110V</td><td>Same</td></tr><tr><td>Printer</td><td>Printer 110V</td><td>Same</td></tr></table>

Differences   

<table><tr><td>Specimen processing Assay Reagents</td><td>Manual</td><td>NucliSENS miniMAG Specific for Enterovirus</td></tr><tr><td>Analyzer</td><td>Specific for MRSA Use of three filter pairs</td><td>Use of two filter pairs</td></tr><tr><td>Software: Assay</td><td>Specific for MRSA</td><td>Specific for Enterovirus</td></tr><tr><td>Protocol Vortex Genie 2, Vortex adapter plate</td><td>Required</td><td>Not required</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): Not applicable

# L. Test Principle:

The NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay includes extraction, amplification and detection of MRSA DNA. Essential processes of the assay are the release of the MRSA-specific nucleic acids, NASBA™ and generation of fluorescence signals during the course of the reaction.

For fluorescence-based detection, specific molecular beacons are added to the NASBA™ reaction. These can instantaneously bind to the complementary singlestranded RNA amplicons that are generated in the amplification process. In DNA NASBA™, the bacterial DNA is first digested with a restriction enzyme and then copies of RNA derived from a target DNA sequence are manifold multiplied by isothermal amplification by the combined activities of 3 enzymes: Avian Myeloblastosis Virus Reverse Transcriptase (AMV-RT), Bacteriophage T7 RNA Polymerase (T7 RNA Pol), and RNase H, along with both ribo- and deoxyribonucleotides (rNTPs and dNTPs) and the cofactors dithiothreitol (DTT), $\mathbf { M g C l } 2$ , KCl and DMSO. To start the DNA NASBA™ process, the restriction enzyme Hae III cuts the target DNA near the binding site of primer 1 (P1). Afterwards, the reaction mixture is heated to $9 5 \mathrm { { ^ \circ C } }$ to denature the linearised DNA and to inactivate the restriction enzyme Subsequently, the reaction mixture is cooled down to $4 1 ^ { \circ } \mathrm { C }$ and the primer P1 anneals to the positive strand of the target DNA. This starts the “linear phase” of the $\mathrm { N A S B A ^ { T M } }$ reaction where double-stranded DNA amplicons with a T7 promoter tail are generated. In the cyclic phase of the $\mathrm { N A S B A ^ { T M } }$ reaction, the T7 promoter tail on the cDNA target amplicon serves as a binding site for T7 RNA Pol, which then transcribes from the cDNA amplicon using rNTP’s to generate a negative strand RNA copy. A single original positive strand target sequence (with T7 tail applied) will serve as a template for multiple RNA copies as the reaction proceeds.

Molecular beacons are hairpin structure-forming single-stranded oligonucleotides labeled at one end with a fluorescent dye named the fluorophore and at the other end with a quencher. In the absence of complementary target amplicons, the hairpin structure closes, bringing the fluorophore and quencher into close proximity, and as a result no fluorescent signal is emitted.

In the presence of target amplicons, the beacon hybridizes to its complementary target instead of forming an internal hairpin structure. Once the beacon opens the quencher and fluorophore separate, quenching ceases and fluorescence is emitted.

Internal Control DNA is used to monitor nucleic acid amplification. Primer binding sites on the Internal Control DNA and MRSA DNA are identical, whereas differently labeled molecular beacons are used for the detection of the different kinds of amplicons.

Reactions are performed in closed tubes in a NucliSENS EasyQ $^ \mathrm { \textregistered }$ Analyzer in which fluorescence is continuously measured. NucliSENS EasyQ $^ \mathrm { \textregistered }$ Director Software, in combination with NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay protocols software, provides automated analysis of the resulting fluorescence curves and reporting of assay results.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The precision and reproducibility studies were performed on the complete assay procedure from the swab specimen to the final result. Two lots of NASBA reagents were used for both the within- and between-laboratory precision studies. A frozen pool of nasal clinical swabs was used to avoid any variability due to the use of individual clinical matrix.

A panel of 3 bacterial inputs was tested on each of 12 days, using the first lot of reagents for the first 6 days and the second lot of reagents for the last 6 days. The study involved 2 operators who each performed one run per day. Two different instruments were used. The table below demonstrated the summary of the precision studies. The data was acceptable.

# Within-Laboratory Variation

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambda cassette junction</td></tr><tr><td rowspan=1 colspan=1>Sample Input&#x27;</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative (HN)</td><td rowspan=1 colspan=1>90.3% (65/72)</td><td rowspan=1 colspan=1>81.0-96.0%</td><td rowspan=1 colspan=1>2.669</td><td rowspan=1 colspan=1>0.0592</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.034</td><td rowspan=1 colspan=1>0.0723</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LP)</td><td rowspan=1 colspan=1>98.6% (71/72)</td><td rowspan=1 colspan=1>92.5-99.9%</td><td rowspan=1 colspan=1>2.641</td><td rowspan=1 colspan=1>0.0782</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.578</td><td rowspan=1 colspan=1>0.2178</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (MP)</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0-100%</td><td rowspan=1 colspan=1>2.680</td><td rowspan=1 colspan=1>0.0564</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.996</td><td rowspan=1 colspan=1>0.1498</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>Total Agreement</td><td rowspan=1 colspan=1>96.3% (208/216)</td><td rowspan=1 colspan=1>92.8-98.4%</td><td></td><td></td><td></td><td></td><td></td><td></td></tr></table>

${ } ^ { 1 } \mathsf { H N } =$ approx. 0.03 x LoD $\mathsf { L P } \mathsf { = }$ approx. 1.00 x LoD MP= approx. 4.82 x LoD

The between-laboratory reproducibility study was performed at 3 sites, over six days with two operators, one instrument at each site and two lots of reagent. A panel of 3 bacterial inputs was tested at each laboratory on each of 6 days, 3 days with one reagent lot and 3 days with a second reagent lot. The study involved 2 operators who each performed one run per day. Three different instruments were used (one instrument per site), with the following results:

Between –Laboratory Variation   

<table><tr><td rowspan=1 colspan=5></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=1>MaxLa</td></tr><tr><td rowspan=1 colspan=1>Sample Input&#x27;</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>% Agreementand 95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative(HN)</td><td rowspan=1 colspan=1>94.4%(34/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>86.1%(31/36)</td><td rowspan=1 colspan=1>93.5%(101/108)87.1 - 97.4%</td><td rowspan=1 colspan=1>2.650</td><td rowspan=1 colspan=1>0.0662</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.024</td></tr><tr><td rowspan=1 colspan=1>LowPositive(LP)</td><td rowspan=1 colspan=1>85.7%(31/36)</td><td rowspan=1 colspan=1>75.0%(27/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>87.0%(94/108)79.2 - 92.7%</td><td rowspan=1 colspan=1>2.631</td><td rowspan=1 colspan=1>0.0837</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.347</td></tr><tr><td rowspan=1 colspan=1>Low Positive 2(LP2)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>99.1%(107/108)94.9 - 99.9%</td><td rowspan=1 colspan=1>2.557</td><td rowspan=1 colspan=1>0.0995</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>1.735</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive (MP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(108/108)96.6 - 100%</td><td rowspan=1 colspan=1>2.670</td><td rowspan=1 colspan=1>0.1119</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.816</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>95.1%(137/144)</td><td rowspan=1 colspan=1>93.1%(134/144)</td><td rowspan=1 colspan=1>96.5%(139/144)</td><td rowspan=1 colspan=1>94.9%(410/432)92.4 - 96.8%</td><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=9>HN = approx. 0.03 x LoDLP=    approx. 1.00 x LoDLP2=  approx. 2.00 x LoDMP=   approx. 4.82 x LoD</td></tr></table>

Between-Lot Variation (between –Laboratories)   

<table><tr><td></td><td></td><td></td><td></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambda cassette junction</td></tr><tr><td rowspan=1 colspan=1>Sample Input&#x27;</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>% Agreementand 95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HighNegative(HN)</td><td rowspan=1 colspan=1>94.4%(51/54)</td><td rowspan=1 colspan=1>92.6%(50/54)</td><td rowspan=1 colspan=1>93.5%(101/108)87.1 -97.4%</td><td rowspan=1 colspan=1>2.650</td><td rowspan=1 colspan=1>0.0662</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.024</td><td rowspan=1 colspan=1>0.0558</td><td rowspan=1 colspan=1>5.4</td></tr><tr><td rowspan=1 colspan=1>Low  Positive(LP)</td><td rowspan=1 colspan=1>90.7%(49/54)</td><td rowspan=1 colspan=1>83.3%(45/54)</td><td rowspan=1 colspan=1>87.0%(94/108)79.2 - 92.7%</td><td rowspan=1 colspan=1>2.631</td><td rowspan=1 colspan=1>0.0837</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.347</td><td rowspan=1 colspan=1>0.2707</td><td rowspan=1 colspan=1>20.1</td></tr><tr><td rowspan=1 colspan=1>Low Positive2(LP2)</td><td rowspan=1 colspan=1>98.1%(53/54)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>99.1%(107/108)94.9 - 99.9%</td><td rowspan=1 colspan=1>2.557</td><td rowspan=1 colspan=1>0.0995</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>1.735</td><td rowspan=1 colspan=1>0.2405</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive (MP)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>100%(108/108)96.6 -100%</td><td rowspan=1 colspan=1>2.670</td><td rowspan=1 colspan=1>0.1119</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.816</td><td rowspan=1 colspan=1>0.2550</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>95.8%(207/216)</td><td rowspan=1 colspan=1>94.0%(203/216)</td><td rowspan=1 colspan=1>94.9%(410/432)92.4-96.8%</td><td rowspan=1 colspan=6></td></tr></table>

1HN = approx. 0.03 x LoD $\mathsf { L P } \mathsf { = }$ approx. 1.00 x LoD LP2= approx. 2.00 x LoD MP= approx. 4.82 x LoD

# Between lot Variation (Within-Laboratory)

<table><tr><td></td><td></td><td></td><td></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambda cassette junction</td></tr><tr><td rowspan=1 colspan=1>Sample Input</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>% Agreementand 95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HighNegative(HN)</td><td rowspan=1 colspan=1>91.7%(33/36)</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>90.3%(65/72)81.0-96.0%</td><td rowspan=1 colspan=1>2.669</td><td rowspan=1 colspan=1>0.0592</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.034</td><td rowspan=1 colspan=1>0.0723</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>Low Positive(LP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>97.2%(35/36)</td><td rowspan=1 colspan=1>98.6%(71/72)92.5 - 99.9%</td><td rowspan=1 colspan=1>2.641</td><td rowspan=1 colspan=1>0.0782</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.578</td><td rowspan=1 colspan=1>0.2178</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive (MP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(72/72)95.0 -100%</td><td rowspan=1 colspan=1>2.680</td><td rowspan=1 colspan=1>0.0564</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.996</td><td rowspan=1 colspan=1>0.1498</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>97.2%(105/108)</td><td rowspan=1 colspan=1>95.4%(103/108)</td><td rowspan=1 colspan=1>96.3%(208/216)92.8 -98.4%</td><td rowspan=1 colspan=6></td></tr></table>

${ } ^ { 1 } \mathsf { H N } =$ approx. 0.03 x LoD $\mathsf { L P } =$ approx. 1.00 x LoD $\mathrm { M P = }$ approx. 4.82 x LoD

The data was acceptable.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods): The controls of the EasyQ MRSA assay included:   
An internal Inhibition Control (IC) - An internal control plasmid is   
incorporated in each reaction tube to detect inhibition to ensure   
proper reporting of an individual specimen   
Two external controls   
One NASBA positive control to detect NASBA™ reagents and   
instrument failures   
One NASBA blank control to detect contamination with strains of   
MRSA or DNA and contamination due to carryover of amplicons   
specimen controls (MRSA and MSSA)   
Two specimen processing controls:   
One specimen positive control by rotating one of the six specified   
MRSA strains of different MREJ types (ii, iii, iv, v, vii and xii)   
into each run to check for integrity of lysis and NASBA™ reagents   
and absence of instrument failure   
One specimen negative control (MSSA ATCC 29213) to detect   
contamination with strains of MRSA or DNA and contamination   
due to carry-over of amplicons in the lysis and NASBA™ reagents   
or in the environment

The total valid run rate was $90 \%$ (114/127); there were 13 invalid runs; 11 $( 8 4 . 6 \% )$ were due to specimen negative control (SNC) failure.

# Specimen Controls (SPC and SNC) Stability Study

The stability study was performed using one type of SPC (MRSA MREJ type ii) and the SNC (MSSA strain). The use of one SPC only to verify the stability of the different MREJ types was accepted since the MREJ type represents the sequence of the right part of the SCCmec cassette, which is not linked to any phenotypic characteristics that could have an impact on the stability of MRSA. Moreover, MRSA MREJ type ii is the strain most frequently isolated in clinical practice. This was confirmed after sequencing the MRSA strains obtained during the European clinical trials of the NucliSENS EasyQ MRSA test, $87 \%$ of the 171 MRSA strains isolated were MREJ type ii.

Based on qualitative results, SPC (MRSA MREJ type ii) and SNC (MSSA), prepared in mucin, were shown stable up to 6 months, when frozen at $- 1 9 ^ { \circ } \mathrm { C } / .$ - $3 1 ^ { \circ } \mathrm { C }$ .

# Reagent Stability Study

# NASBA Reagent

Real-time stability data were provided from a study in real time on 3 NASBA reagent lots (PTB7, PVB1 and PVB2) produced in manufacturing conditions for a period of 19 months when stored at $2 { - } 8 ^ { \circ } \mathrm { C }$ , testing MREJ wild type (WT)

plasmids corresponding to MREJ types ii, iii, iv, v, vii and xii, each plasmid is also associated with mecA WT plasmid. NASBA reagents included the NASBA controls: positive control (PC), external blank control (BLK), are stable for at least 19 months when stored at $2 / 8 \mathrm { { ^ { \circ } C } }$ .

In-use and multiple use combined stabilities:

All reconstituted spheres are still functional within 2 weeks after first use and one (reagent and restriction enzyme spheres, enzyme sphere) or two freeze-thawing cycles (positive control sphere) upon storage at $- 1 9 /$ - $3 1 ^ { \circ } \mathrm { C }$ • Reconstituted sphere (reagent and restriction enzyme spheres, enzyme sphere and positive control sphere) are still functional within 30 minutes at room temperature. Six openings for the diluents (reagent and enzyme) and 16 openings for the NASBA water were performed on 6 non consecutive days separated by 12 days $+ / - 2$ days to cover a total period of 2 months when stored at 2- $8 \mathrm { { } ^ { \circ } C }$

# MRSA Lysis Tubes

The study was conducted on three batches, stored at $2 / 8 ^ { \circ } \mathrm { C }$ after thermal shocks.

Lot PTB4   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Time point</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>T&#x27;0</td><td rowspan=1 colspan=1>M3</td><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>M7</td><td rowspan=1 colspan=1>M9</td><td rowspan=1 colspan=1>M12</td><td rowspan=1 colspan=1>M13</td><td rowspan=1 colspan=1>M18</td><td rowspan=1 colspan=1>M19</td></tr><tr><td rowspan=1 colspan=4>ExperimentsBT L number</td><td rowspan=1 colspan=1>BTL061242</td><td rowspan=1 colspan=1>BTL061988</td><td rowspan=1 colspan=1>BTL062959</td><td rowspan=1 colspan=1>BTL064496</td><td rowspan=1 colspan=1>BTL065020</td><td rowspan=1 colspan=1>BTL066135</td><td rowspan=1 colspan=1>BTL067156</td><td rowspan=1 colspan=1>BTL067268</td><td rowspan=1 colspan=1>BTL068418</td><td rowspan=1 colspan=1>BTL068672</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Characteristics</td><td rowspan=1 colspan=4>Criteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=8 colspan=1>Aspectof reagents</td><td rowspan=1 colspan=1>Color</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>Texture</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4></td><td rowspan=2 colspan=1>Passed</td><td rowspan=2 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td><td rowspan=6 colspan=1>Passed</td></tr><tr><td rowspan=1 colspan=1>Cleanliness</td><td rowspan=1 colspan=1>No m</td><td rowspan=1 colspan=3>No modification</td><td rowspan=1 colspan=1>on</td></tr><tr><td rowspan=1 colspan=1>Odor</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td></td></tr><tr><td></td><td rowspan=3 colspan=4></td><td rowspan=3 colspan=1></td><td></td></tr><tr><td></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Precipitation</td></tr><tr><td rowspan=2 colspan=1>Runcontrols</td><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=4>3/3 valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=4>3/3 valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Acceptancecriteria</td><td rowspan=1 colspan=1>SNC low input</td><td rowspan=1 colspan=4>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>SPC low input</td><td rowspan=1 colspan=4>21/24 valid</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td></tr><tr><td rowspan=1 colspan=1>SPC high input</td><td rowspan=1 colspan=4>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=4>Passed/failed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td></tr></table>

Lot PVB2   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Time point</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>T&#x27;O</td><td rowspan=1 colspan=1>M3</td><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>M7</td><td rowspan=1 colspan=1>M9</td><td rowspan=1 colspan=1>M12</td><td rowspan=1 colspan=1>M13</td><td rowspan=1 colspan=1>M18</td><td rowspan=1 colspan=1>M19</td></tr><tr><td rowspan=1 colspan=1>ExperimentsBTL Number</td><td rowspan=1 colspan=1>BTL061896</td><td rowspan=1 colspan=1>BTL065260</td><td rowspan=1 colspan=1>BTL063585</td><td rowspan=1 colspan=1>BTL065166</td><td rowspan=1 colspan=1>BTL065832</td><td rowspan=1 colspan=1>BTL066556</td><td rowspan=1 colspan=1>BTL067393</td><td rowspan=1 colspan=1>BTL067650</td><td rowspan=1 colspan=1>BTL068674</td><td rowspan=1 colspan=1>BTL080</td></tr><tr><td rowspan=1 colspan=2>Characteristics</td><td rowspan=1 colspan=1>Criteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=5 colspan=1>Aspectof reagents</td><td rowspan=1 colspan=1>Color</td><td rowspan=4 colspan=1>Nomodification</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td></tr><tr><td rowspan=1 colspan=1>Texture</td></tr><tr><td rowspan=1 colspan=1>Cleanliness</td></tr><tr><td rowspan=1 colspan=1>Odor</td></tr><tr><td rowspan=1 colspan=1>Precipitation</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Run controls</td><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=1>3/3 valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=1>3/valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Acceptancecriteria</td><td rowspan=1 colspan=1>SNC low input</td><td rowspan=1 colspan=1>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>SPC low input</td><td rowspan=1 colspan=1>21/24 valid</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>24/24</td></tr><tr><td rowspan=1 colspan=1>SPC high input</td><td rowspan=1 colspan=1>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>8/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Passed/failed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td></tr></table>

Lot PVB1.1   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Time point</td><td rowspan=1 colspan=1>TO</td><td rowspan=1 colspan=1>T&#x27;0</td><td rowspan=1 colspan=1>M3</td><td rowspan=1 colspan=1>M6</td><td rowspan=1 colspan=1>M7</td><td rowspan=1 colspan=1>M9</td><td rowspan=1 colspan=1>M12</td><td rowspan=1 colspan=1>M13</td><td rowspan=1 colspan=1>M18</td><td rowspan=1 colspan=1>M19</td></tr><tr><td rowspan=1 colspan=1>ExperimentsBTL Number</td><td rowspan=1 colspan=1>BTL061867</td><td rowspan=1 colspan=1>BTL062686</td><td rowspan=1 colspan=1>BTL03586</td><td rowspan=1 colspan=1>BTL065154</td><td rowspan=1 colspan=1>BTL065831</td><td rowspan=1 colspan=1>BTL06360</td><td rowspan=1 colspan=1>BTL0667392</td><td rowspan=1 colspan=1>BTL06760</td><td rowspan=1 colspan=1>BTL068673</td><td rowspan=1 colspan=1>BTL06883</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Characteristics</td><td rowspan=1 colspan=1>Criteria</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=5 colspan=1>Aspectof reagents</td><td rowspan=1 colspan=1>Color</td><td rowspan=5 colspan=1>Nomodification</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td><td rowspan=5 colspan=1>Passed</td></tr><tr><td rowspan=1 colspan=1>Texture</td></tr><tr><td rowspan=1 colspan=1>Cleanliness</td></tr><tr><td rowspan=1 colspan=1>Odor</td></tr><tr><td rowspan=1 colspan=1>Precipitation</td></tr><tr><td rowspan=2 colspan=1>Runcontrols</td><td rowspan=1 colspan=1>Negative control</td><td rowspan=1 colspan=1>3/3 valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>Positive control</td><td rowspan=1 colspan=1>3/3 valid</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=3 colspan=1>Acceptanceciteria</td><td rowspan=1 colspan=1>SNC low input</td><td rowspan=1 colspan=1>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1>SPC low input</td><td rowspan=1 colspan=1>21/24 valid</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>22/24</td></tr><tr><td rowspan=1 colspan=1>SPC high input</td><td rowspan=1 colspan=1>9/9 valid</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>8/9*</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>9/9</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>Passed/failed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td><td rowspan=1 colspan=1>Passed</td></tr></table>

candidate and IC".

The additional real time stability data of the MRSA lysis tubes up to 19 months demonstrated the stability of 19 months for NucliSENS EasyQ® MRSA assay.

An additional study was conducted to compare the new pre-marked lysis tube to the former unmarked lysis tube. The study demonstrated that the unmarked tube were $8 5 \%$ positive results and the new pre-marked tubes were $90 \%$ positive when testing at slightly lower than the LoD (191 CFU/swab).

# Specimen, lysates stability

Specimen can be stored:

Up to 4 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ before testing (with or without a maximum of 4h transport at room temperature).   
Up to 4 days at room temperature before testing.   
Up to 1 day at room temperature followed by 3 days at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ .

Result summary for clinical swabs stored under various conditions and duration \* One result was discarded for the analysis because the second swab corresponding to the same patient was characterized MRSA positive

<table><tr><td colspan="3">Storage Conditions</td><td colspan="2">Result</td></tr><tr><td>Temp RT</td><td>Hours</td><td>MRSA(+)</td><td>MRSA (-)</td><td></td></tr><tr><td rowspan="4"></td><td>24</td><td>10</td><td>0</td><td></td></tr><tr><td>48</td><td>9*</td><td>0</td><td></td></tr><tr><td>72</td><td>10</td><td></td><td>0</td></tr><tr><td>96</td><td>10</td><td>0</td><td></td></tr><tr><td rowspan="4">2-8°C</td><td>24</td><td>10</td><td></td><td>0</td></tr><tr><td>48</td><td>10</td><td></td><td>0</td></tr><tr><td>72</td><td>10</td><td></td><td>0</td></tr><tr><td>96</td><td>10</td><td>0</td><td></td></tr><tr><td rowspan="4">4 hrs RT AND 96 hrs at 2-8°C</td><td></td><td></td><td></td><td></td></tr><tr><td>24 48</td><td>10 10</td><td></td><td>0 0</td></tr><tr><td>72</td><td>10</td><td></td><td></td></tr><tr><td>96</td><td>10</td><td></td><td>0 0</td></tr><tr><td rowspan="4">24 hrs RT AND additional 72 hrs at 2- 8</td><td></td><td></td><td></td><td></td></tr><tr><td>24 (RT)</td><td>10</td><td>0</td><td></td></tr><tr><td>48 (2-8°C)</td><td>10</td><td>0</td><td></td></tr><tr><td>72 (2-8°C)</td><td>10</td><td></td><td>0</td></tr><tr><td rowspan="2"></td><td>96 (2-8°C)</td><td>10</td><td></td><td>0</td></tr><tr><td></td><td></td><td></td><td></td></tr></table>

Lysates can be stored:

Up to 24 hours at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ .   
Up to 1 month at $- 1 9 \ { } ^ { \circ } { \bf C } / - 3 1 \ { } ^ { \circ } { \bf C }$ . A maximum of 5 freeze/thaw cycles are allowed within the month.

<table><tr><td colspan="4">Result summary for lysates stored under different conditions and duration</td></tr><tr><td colspan="3">Storage Conditions</td><td colspan="2">Result</td></tr><tr><td>Time</td><td>Temperature</td><td>freeze/thaw cycles#</td><td>MRSA (+)</td><td>MRSA (-)</td></tr><tr><td>Time zero*</td><td>on ice</td><td>NA</td><td>10</td><td>0</td></tr><tr><td>4 hrs</td><td>2-8°C</td><td>NA</td><td>9</td><td>1</td></tr><tr><td>24 hrs</td><td>2-8°C</td><td>NA</td><td>10</td><td>0</td></tr><tr><td>4 days</td><td>-19°C/-31°C</td><td>2</td><td>10</td><td>0</td></tr><tr><td>1 month</td><td>-19°C/-31°</td><td>2</td><td>10</td><td>0</td></tr><tr><td>1 month</td><td>-19°C/-31°</td><td>4</td><td>10</td><td>0</td></tr><tr><td>1 month</td><td>-19°C/-31°</td><td>5</td><td>10</td><td>0</td></tr></table>

\*Time zero: Lysate kept on ice or cooling block while preparing the NASBA #Freeze/Thaw cycle: freeze at indicated temperature for $\geq 4$ days, followed by thaw at RT.

# d. Detection limit:

# Limit of Detection Study

The analytical sensitivity of the NucliSENS EasyQ® MRSA assay was determined using various strains of MRSA representing the seven types of MREJ detected by the test (MREJ types i, ii, iii, iv, v, vii and xii) using frozen matrices of clinical swabs. Bacterial suspensions with a bacterial input ranging roughly from 10 to 1000 CFU per swab were tested in 24 replicates of each input using a single lot of reagent. The limit of detection was estimated as the input corresponding to the $9 5 \%$ Hit Rate predicted by statistical analysis using a statistical regression model (Probit). The estimated type-specific LoDs were then confirmed on two reagent lots with 15 replicates each.

The claimed type-specific LoD summarized in the following table corresponds to the higher concentrations claimed from these two studies, i.e. either real input determined by plating and counting bacteria colony forming units in the confirmation study, or predicted concentrations by the probit regression model.

LoD for the seven MRSA types   

<table><tr><td rowspan=1 colspan=1>MREJ type</td><td rowspan=1 colspan=1>SCCmec type</td><td rowspan=1 colspan=1>LoD(CFU/swab)</td></tr><tr><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>410</td></tr><tr><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>227</td></tr><tr><td rowspan=1 colspan=1>iii</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>493</td></tr><tr><td rowspan=1 colspan=1>iv</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>234</td></tr><tr><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>vii</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>482</td></tr><tr><td rowspan=1 colspan=1>xii</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>182</td></tr></table>

# Analytical Inclusivity

A total of 193 well characterized MRSA isolates were tested at an input level of approximately 3 times the LoD, in the presence of frozen specimen matrix from nasal swab specimens previously shown to be MRSA negative, using the NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay, in the Inclusivity Study. The strains included pulse field gel electrophoresis (PFGE) types: USA100, 200, 300, 400, 500, 600, 700, 800, 1000, and 1100. Nine were not detected, resulted a detection rate of $9 5 . 3 \%$ (184/193). Three of the undetected strains were MREJ type iii.

Ten heterogeneous oxacillin resistant strains were tested with acceptable results.

Six "empty cassette” variants (i.e. mecA dropouts) were tested with mecA gene negative clinical matrices at both input levels of $7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/swab and $\overline { { 1 } } . 0 \mathrm { ~ x ~ } 1 0 ^ { \overline { { 4 } } }$ CFU/swab. They all gave a positive signal for the cassette junction target and a negative signal for the mecA gene target, leading to a MRSA negative result. However, the risk of false positive results exists with swabs that possess the mecA gene (e.g. carried by methicillin/oxacillin resistant coagulase negative Staphylococci) and MSSA mecA drop-out strains.

# e. Analytical specificity:

# Cross Reactivity

A total of 100 phylogenetically related S. aureus and members of the nasal commensal flora were tested at an input of $7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/swab. There were 26 methicillin-resistant coagulase negative staphylococci (MRCN), 29 methicillin-sensitive coagulase-negative staphylococci (MSCN), 25 methicillin-sensitive coagulase-positive staphylococci (MSCP) of which 20 were MSSA, and 5 were S. intermedius; 20 organisms other than Staphylococci belonging to the nasal commensal flora ( 1 Gram-positive rods,

10 Gram-negative rods, 7 non-staphylococci Gram-positive cocci, 1 Gramnegative cocci, 1 yeast).

An additional study was conducted on eight well characterized $K .$ . pneumoniae subsp. pneumoniae, $K .$ . pneumoniae subsp. ozaenae with mecA gene negative clinical matrices at both input levels of $\bar { 7 . 5 \mathrm { ~ x ~ } } 1 0 ^ { 6 }$ CFU/swab and $1 . 0 \mathrm { ~ x ~ } \mathrm { { i 0 } ^ { 4 } }$ CFU/swab. Results demonstrated that at an input level of $7 . 5 \mathrm { ~ x ~ } 1 0 ^ { 6 }$ CFU/swab, the 8 strains showed a positive signal for the cassette junction target and a negative signal for the mecA target. When tested at an input level of $\mathbf { \bar { 1 } . 0 \times 1 0 ^ { 4 } C F U / s w a b }$ , the eight strains showed a negative signal for both targets. Therefore, the risk of false positive results exists with swabs that possess the mecA gene (e.g. carried by methicillin resistant coagulase negative Staphylococci in the clinical sample) with $K .$ . pneumoniae subsp. pneumoniae, $K .$ . pneumoniae subsp. ozaenae due to a positive signal for the cassette junction target at a high input of $7 . 5 \mathrm { ~ x ~ } 1 \bar { 0 } ^ { 6 } \mathrm { C F U / s w a b }$ .

# Biological Interference Study

To assess whether the NucliSENS EasyQ® MRSA assay could be affected by the presence of other microorganisms that are normal nasal flora (biological interference), an analytical study was carried out to test potential biological interference on both MRSA positive swabs, as well as on MRSA negative swabs using frozen matrices of clinical swabs.

To determine the potential interference of common nasal microorganisms on MRSA positive swabs, dry flocked swabs were spiked with a strain of MRSA at 3 times the LoD and with an excess of a potential interfering microorganism (approximately $1 0 ^ { 6 }$ CFU/swab). A control test was performed without interfering microorganisms using dry flocked swabs spiked with MRSA only and had to be MRSA positive validate the experiment.

To determine the potential interference of common nasal microorganisms on MRSA negative swabs, dry flocked swabs were spiked with an excess of the potentially interfering microorganism. A control test was performed without interfering microorganisms using dry flocked swabs spiked with sterile water only and had to be MRSA negative to validate the experiment.

The spiked flocked swabs were then introduced into the lysis tubes containing a thawed matrix of MRSA negative swabs and then submitted to the sample preparation and NASBA amplification   
Each type of spiked swab was tested in four replicates.

Two different MRSA strains (MREJ types ii and iv) were used in the study:

• MRSA MREJ type ii was used because it is the most frequently isolated strain in clinical practice (as confirmed by sequencing of the MRSA strains obtained during the European clinical trials for the NucliSENS

EasyQ MRSA test, $87 \%$ of the 171 MRSA strains isolated were MREJ type ii).

• MRSA MREJ type iv was used because the Internal control (IC) is also a type iv and competes with this target for the primers. Therefore, MREJ iv strain is expected to be more susceptible to the potential effect of biological interference than any other MREJ type.

Three strains were used as potentially interfering microorganisms in this study:

Methicillin Sensitive Staphylococcus aureus (MSSA) • Methicillin Resistant Staphylococcus epidermidis (MRSE) Methicillin Sensitive Staphylococcus epidermidis (MSSE)

The targeted input for the MRSA strains was 3 times the LoD for both type ii and type iv. The real input on the swab was determined by plate counting of the suspension used to spike the swab.

For both the MREJ type ii and the MREJ type iv spiked swabs, the 4 replicates all gave “MRSA positive” results in the presence of the interfering microorganisms (MSSA, MRSE or MSSE). For the negative swabs, the 4 replicates all gave “MRSA negative” results in the presence of the interfering microorganisms (MSSA, MRSE or MSSE).

# Chemical Interference Study

To assess whether the NucliSENS EasyQ® MRSA assay could be affected by the presence of well identified chemical substances, which can be naturally present in the nasal cavity or can be artificially introduced into the nose, an analytical study was carried out to test potential chemical interference on both MRSA positive swabs, as well as on MRSA negative swabs using frozen matrices of clinical swabs.

To determine the potential chemical interference on MRSA positive swabs, dry flocked swabs were spiked with a strain of MRSA at 3 times the LoD and with an excess of a potential interfering chemical substance (50 ul). A control test was performed without interfering chemical substance using dry flocked swabs spiked with MRSA only and had to be MRSA positive validate the experiment.

The spiked flocked swabs were then introduced into the lysis tubes containing a thawed matrix of MRSA negative swabs and then submitted to the sample preparation and NASBA amplification. Each type of spiked swab was tested in 4 replicates. If a chemical substance yielded 1) invalid results or false negative results for the swabs spiked with MRSA or 2) invalid results for the swabs without MRSA, then 20 additional replicates were performed to confirm the initial results.

Two different MRSA strains (MREJ types ii and iv) were used in the study:

MRSA MREJ type ii was used because it is the most frequently isolated strain in clinical practice (as confirmed by sequencing of the MRSA strains obtained during the European clinical trials for the NucliSENS EasyQ MRSA test, 87 $\%$ of the 171 MRSA strains isolated were MREJ type ii). • MRSA MREJ type iv was used because the Internal control (IC) is also a type iv and competes with this target for the primers. Therefore, MREJ iv strain is expected to be more susceptible to the potential effect of biological interference than any other MREJ type.

The following naturally or artificially introduced chemical substances were tested in the study:

• Whole blood at different levels   
• Mucus (using mucin) Astelin (azelastine HCl) antihistamine nasal spray Nasonex (mometasone furoate), steroidal nasal spray Nasacort (acetonide triamcinolone), glucocorticoid nasal spray Vicks Sinex (oxymetazoline HCl), topical decongestants nasal spray Neo-Synephrine (phenylephrine HCl $1 . 0 \%$ ), decongestant nasal spray Sino Fresh (eucalyptus globulus, kalium bichromicum), antiseptic homeopathic nasal spray NasalCrom (cromolyn sodium), antiallergic nasal spray Zicam cold remedy (zincum gluconicum), homeopathic nasal spray

The targeted input for the MRSA strains was 3 times the LoD for both MREJ type ii and MREJ type iv. The real input on the swab was determined by plate counting of the suspension used to spike the swab.

Five chemical substances were found to be interferents: Astelin, Sino Fresh, Nasal chrom, Zicam and Blood (15 and $5 0 \mu \mathrm { l } \dot { } ,$ ).

Carry-Over Contamination

Carryover study was performed to access the carry-over risk of the NucliSENS EasyQ MRSA assay during sample preparation and amplification/detection. Five NucliSENS EasyQ MRSA NASBA runs were performed on MRSA high positive (positive for both the cassette junction and mecA) and MRSA negative specimen (negative for both the cassette junction and mecA) in an alternating pattern, resulting in a total of 105 MRSA high positive samples and 105 MRSA negative samples.

This study was performed using frozen matrices of clinical swabs. In order to obtain MRSA high positive specimen, lysis tubes containing a matrix of MRSA negative clinical swabs were spiked with a high concentration of MRSA MREJ type ii $\left( 5 \mathrm { x } 1 0 ^ { 6 } \right.$ CFU/lysis tube). In order to obtain MRSA negative specimen, lysis tubes containing a matrix of MRSA negative clinical swabs, that were proven to be negative for the cassette and mecA, were spiked with sterile water.

All 105 replicates of the MRSA high positive sample were tested MRSA positive. All 103 out of 103 valid replicates of the MRSA negative sample were tested MRSA negative (negative for both the cassette and mecA). Two MRSA negative replicates were tested “invalid” due to failure of detecting IC. No cross-over contamination was observed amongst the 103 negative samples surrounded by high positive samples.

# f. Assay cut-off:

Studies were conducted to determine the following thresholds (cut-offs) for the interpretation of the run controls and the specimen controls. The final thresholds for the controls were further validated in the prospective U.S. clinical trial.

<table><tr><td>Threshold value</td><td>SPC</td><td>SNC</td><td>PC</td><td>BLK</td></tr><tr><td>MaxLambda cassette junction</td><td>1.2</td><td>1.08</td><td>1.5</td><td>1.08</td></tr><tr><td>MaxLambda mecA</td><td>2.0</td><td>1.1</td><td>2.0</td><td>1.1</td></tr><tr><td>MaxLambda IC</td><td>2.0</td><td>3.5</td><td>2.0</td><td>3.5</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

Prospective specimens were collected at seven geographically diverse U.S. sites, of which two were for specimen collections only, and were sent to the other five sites for testing. The testing sites also included two pediatric sites. The NucliSENS EasyQ MRSA Assay was compared to a reference enriched culture. The reference culture was trypticase soy broth with $6 . 5 \%$ NaCl, incubated for 48 hrs if negative; any suspected $S .$ aureus was confirmed by Gram stain, latex agglutination or DNase. The mecA mediated oxacillin resistance was tested by disk diffusion using a $3 0 \mu \mathrm { g }$ cefoxitin disk and report of ${ \le } 2 1 ~ \mathrm { m m }$ (R), ${ \ge } 2 2 ~ \mathrm { m m }$ (S).

There were a total of 1355 nasal specimens, with 1238 reported results. There were 42 initial invalid results, resulting in an invalid rate of $3 . 4 \%$ (42/1238) from 114 valid runs. Fourteen remained invalid after repeat testing from frozen lysate.

The overall invalid run rate was $1 0 . 2 \%$ (13/127).

# Clinical Sensitivity/Specificity:

The performance of the NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA assay (Adult $^ +$ pediatric) all sites combined is summarized in the following tables. Invalids were not included in the performance calculation of adults, and pediatric $^ +$ adults; there were no invalids in pediatric samples:

NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA Performance (Adult $^ +$ Pediatric) comparing to Reference Culture   

<table><tr><td rowspan=2 colspan=1>NucliSENS</td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>MRSA-</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>172</td></tr><tr><td rowspan=1 colspan=1>MRSA-</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1046</td><td rowspan=1 colspan=1>1052</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1081</td><td rowspan=1 colspan=1>1224</td></tr><tr><td rowspan=1 colspan=4>MRSA:                                          95%CI:Sensitivity: (137/143)         95.8%         (91.1 - 98.4%)Specificity: (1046/1081)      96.8%         (95.5 - 97.7%)Positive Predictive value      79.7%         (72.9 - 85.4%)Negative Predictive value     99.4%         (98.8 - 99.8%)</td></tr></table>

\* mecA mediated oxacillin resistance using $3 0 \mu \mathrm { g }$ cefoxitin disk

NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA Performance (Adults) comparing to Reference Culture   

<table><tr><td rowspan=2 colspan=1>NucliSENS</td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>MRSA-</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>133</td></tr><tr><td rowspan=1 colspan=1>MRSA-</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>730</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>863</td></tr><tr><td rowspan=1 colspan=4>MRSA:                                          95%CI:Sensitivity: (137/143)         94.7%         (88.8 - 98.0%)Specificity: (1046/1081)      96.5%         (95.0 - 97.7%)Positive Predictive value      80.5%         (72.7 - 86.8%)Negative Predictive value     99.2%         (98.2 - 99.7%)</td></tr></table>

\* mecA mediated oxacillin resistance using $3 0 \mu \mathrm { g }$ cefoxitin disk

NucliSENS EasyQ $^ \mathrm { \textregistered }$ MRSA Performance (Pediatric) comparing to Reference Culture

<table><tr><td rowspan=2 colspan=1>NucliSENS</td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=2 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>MRSA-</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>MRSA-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>322</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>361</td></tr><tr><td rowspan=1 colspan=4>MRSA:                                          95%CI:Sensitivity: (137/143)        100%          (88.4 - 100%)Specificity: (1046/1081)      97.3%         (94.9 -98.8%)Positive Predictive value      76.9%         (60.7 - 88.9%)Negative Predictive value    100%          (98.9 - 100%)</td></tr></table>

\* mecA mediated oxacillin resistance using $3 0 \mu \mathrm { g }$ cefoxitin disk

c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

A total of 1355 nasal specimens were collected from 1355 patients at 7 enrolling sites across the United States. There were a total of 1238 final evaluable specimens by reference culture. The study population was grouped into subjects hospitalized for greater than 3 days, hospitalized for 3 days or less, outpatients, Emergency Room (ER) patients, patients from long term care facilities and healthcare workers. Hospitalized patients included, but are not limited to patients previously known to be colonized with MRSA or with MRSA infection, transferred from a long term care institution/nursing home, history of prolonged hospitalization, previous hospital stay, contact with a known MRSA carrier, admitted to intensive care unit, cardiology unit, orthopedic surgery unit or hemodialysis unit, history of intravenous drug use, on contact precautions, and HIV infected. The number and percentage of positive and negative specimens with the reference culture method are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Group</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>% Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Hospitalized &gt;3 days</td><td rowspan=1 colspan=1>16.7% (65/390)</td><td rowspan=1 colspan=1>83.3% (325/390)</td><td rowspan=1 colspan=1>31.5% (390/1238)</td></tr><tr><td rowspan=1 colspan=1>Hospitalized ≤ 3 days</td><td rowspan=1 colspan=1>10.4% (49/471)</td><td rowspan=1 colspan=1>89.6% (422/471)</td><td rowspan=1 colspan=1>38.0% (471/1238)</td></tr><tr><td rowspan=1 colspan=1>Long Term Care/ NursingHome</td><td rowspan=1 colspan=1>16.7% (3/18)</td><td rowspan=1 colspan=1>83.3% (15/18)</td><td rowspan=1 colspan=1>1.5% (18/1238)</td></tr><tr><td rowspan=1 colspan=1>Outpatients</td><td rowspan=1 colspan=1>9.3% (21/227)</td><td rowspan=1 colspan=1>90.7% (206/227)</td><td rowspan=1 colspan=1>18.3% (227/1238)</td></tr><tr><td rowspan=1 colspan=1>ER</td><td rowspan=1 colspan=1>4.5% (4/88)</td><td rowspan=1 colspan=1>95.5% (84/88)</td><td rowspan=1 colspan=1>7.1% (88/1238)</td></tr><tr><td rowspan=1 colspan=1>Healthcare workers</td><td rowspan=1 colspan=1>4.6% (2/44)</td><td rowspan=1 colspan=1>95.4% (42/44)</td><td rowspan=1 colspan=1>3.6% (44/1238)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11.6% (144/1238)</td><td rowspan=1 colspan=1>88.4% (1094/1238)</td><td rowspan=1 colspan=1>100% (1238/1238)</td></tr></table>

# N. Instrument Name:

NucliSENS EasyQ® MRSA assay protocol software included in the NucliSENS   
EasyQ® MRSA assay kit   
NucliSENS EasyQ® Director Software 2.6 (Cleared in k093383)   
NucliSENS EasyQ® Analyzer (Cleared in k093383)   
NucliSENS EasyQ® Incubator II (Cleared in k093383)

# O. System Descriptions:

1. Modes of Operation: Batch or Random access through tube strip (Analyzer)

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X__ or No

3. Specimen Identification: Entered by user (Analyzer)

4. Specimen Sampling and Handling: Specimens are processed according to assay instructions

5. Calibration:

The analyzer contains an auto-calibration function that reads the fluorescence from the reference materials on the plate carrier. The reading is compared to the value in the system memory. The measured value can differ from the memory value.

6. Quality Control:

The EasyQ MRSA assay includes:

• Inhibition Control (IC) - An internal control plasmid is incorporated in each reaction tube to detect inhibition to ensure proper reporting of an individual specimen • NASBA positive control to detect NASBA™ reagents and instrument failures

NASBA blank control to detect contamination with strains of MRSA or DNA and contamination due to carryover of amplicons specimen controls (MRSA and MSSA) • Specimen positive control to check for integrity of lysis and NASBA™ reagents and absence of instrument failure Specimen negative control to detect contamination with strains of MRSA or DNA and contamination due to carry-over of amplicons in the lysis and NASBA™ reagents or in the environment

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision