# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION ONLY TEMPLATE

# B. Purpose for Submission:

The BD GeneOhm™ VanR Assay was submitted to obtain a substantial equivalence determination for vancomycin-resistance (vanA and vanB) from perianal or rectal swab specimens

# C. Measurand:

Vancomycin resistant (VanA and vanB) genes

# D. Type of Test:

Qualitative Nuclei Acid Amplification Test using Real-Time PCR technology, to detect vanA and vanB genes directly from perianal or rectal swabs

E. Applicant: BD Diagnostics (GeneOhm Sciences, Inc)

F. Proprietary and Established Names: BD GeneOhm™ VanR Assay

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.1640

2. Classification: Class II

3. Product code: NIJ- System, test, genotypic detection, resistant markers, Enterococcus species OOI- Nucleic acid amplification systems, real time

# 83 Microbiology

# H. Intended Use:

1. Intended use(s):

The BD GeneOhm™ VanR Assay is a qualitative in vitro test for the rapid detection of vancomycin-resistance (vanA and vanB) genes directly from perianal or rectal swabs. The BD GeneOhm™ VanR Assay detects the presence of the vanA and vanB genes that can be associated with vancomycin-resistant enterococci (VRE). The assay is performed on an automated real-time PCR instrument with perianal or rectal swabs from individuals at risk for VRE colonization. The BD GeneOhm™ VanR Assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. The BD GeneOhm™ VanR Assay is not intended to diagnose VRE infections nor to guide or monitor treatment for VRE infections. Concomitant cultures are necessary to recover organisms for epidemiological typing, susceptibility testing and for further confirmatory identification.

# 2. Indication(s) for use:

The BD GeneOhm™ VanR Assay is a qualitative in vitro test for the rapid detection of vancomycin-resistance (vanA and vanB) genes directly from perianal or rectal swabs. The BD GeneOhm™ VanR Assay detects the presence of the vanA and vanB genes that can be associated with vancomycin-resistant enterococci (VRE). The assay is performed on an automated real-time PCR instrument with perianal or rectal swabs from individuals at risk for VRE colonization. The BD GeneOhm™ VanR Assay can be used as an aid to identify, prevent and control vancomycin-resistant colonization in healthcare settings. The BD GeneOhm™ VanR Assay is not intended to diagnose VRE infections nor to guide or monitor treatment for VRE infections. Concomitant cultures are necessary to recover organisms for epidemiological typing, susceptibility testing and for further confirmatory identification.

3. Special conditions for use statement(s): Prescription use only

4. Special instrument requirements: Automated SmartCycler $^ \mathrm { \textregistered }$ System (instrument, Dx Software version 1.7b, 3.0a or 3.0b)

# I. Device Description:

The test uses a perianal and/or rectal swab specimen, which is resuspended in a sample preparation buffer tube. The suspension is added to a lysis tube. The lysate is then added to a PCR master mix including vanA and vanB specific primers and an internal control (IC) template. The reaction takes place in a disposable reaction tube placed in the SmartCycler $\textsuperscript { \textregistered }$ analyzer. Discrimination of amplicons indicative of drug resistance is done by using a molecular beacon with different fluorometric properties than those of the beacon for the detection of the internal control.

The SmartCycler $^ \mathrm { \textregistered }$ software simultaneously monitors the fluorescence emitted by each molecular beacon probe, interprets all data, and provides a final result at the end of the cycling program.

The operation of the SmartCycler $^ \mathrm { \textregistered }$ is based on the proprietary microprocessorcontrolled I-CORETM (Intelligent Cooling/Heating Optical Reaction) module (SmartCycler $\textsuperscript { \textregistered }$ Diagnostic, Operator Manual).

# BD GeneOhm™ VanR Assay Reagents

Master Mix Control DNA • Diluent

# Materials Provided

• Sample buffer   
• lysis tube   
• Master Mix   
• Control DNA   
• Diluent SmartCycler reaction tubes, $2 5 \mu \mathrm { L }$ Specimen identification labels

# Materials Required but not Provided

• Vortex Genie 2 (Fisher) with $1 . 5 \mathrm { m L }$ microtube holder or equivalent; for processing multiple samples, an adapter capable of holding multiple tubes can be used   
Micropipettors (accurate range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 \mathrm { - } 1 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { \cdot } 1 0 0 0 \ \mu \mathrm { L } )$   
• Sterile DNAse·free filter-blocked or positive displacement micropipettor tips   
• DNAse-free microcentrifuge tubes   
• Scissors   
Gauze   
• Disposable gloves, powder less   
• Microcentrifuge for low speed centrifugation Dry heating block specific for $1 . 5 \mathrm { m L }$ tubes or water bath Ice or cooling block specific for $1 . 5 \mathrm { m L }$ tubes

# Interpretation of Results

The decision algorithm for the BD GeneOhm VanR Assay is embedded in the SmartCycler software. The interpretation of assay results is provided by clinical specimen type, instrument reported assay result, and instrument reported IC result.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Remel Bile Esculin Azide agar with $6 ~ { \mu \mathrm { g / m L } }$ vancomycin

2. Predicate 510(k) number(s): k972359

3. Comparison with predicate:

Similarities   

<table><tr><td>Item</td><td>BD GeneOhmTM VanR Assay</td><td>Remel Bile Esculin Azide agar with 6μg/mL vancomycin, k972359)</td></tr><tr><td>Intended Use</td><td>Screening for vancomycin resistance associated with Enterococcus</td><td>Same</td></tr><tr><td>Sample type</td><td>Perianal or rectal swab</td><td>Same</td></tr><tr><td>Type of Test</td><td>Qualitative</td><td>Same</td></tr><tr><td>Mode of</td><td>Differences</td><td>Growth/no growth in the presence of</td></tr><tr><td>detection Assay format</td><td>vanA, vanB gene</td><td>6µg/mL vancomycin Phenotypic detection based on growth</td></tr><tr><td></td><td>Amplification: PCR Detection: Fluorogenic target- specific hybridization</td><td></td></tr><tr><td>Interpretation of test results</td><td>Diagnostic software of SmartCycler® version 1.7b, 3.0a or 3.0b</td><td>Visual interpretation</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

N/A

L. Test Principle:

The rectal or perianal swab is eluted in sample buffer and the specimen is lysed. Following specimen lysis, amplification of the vanA and vanB targets occurs. Amplification of the internal control (IC), a DNA fragment of 294-bp including a 254-bp sequence not found in VRE, will also take place unless there are PCR

inhibitory substances. The amplified DNA targets are detected with molecular beacons, hairpin-forming single stranded oligonucleotides labeled at one end with a quencher and at the other end with a fluorescent reporter dye (fluorophore). In the absence of target, the fluorescence is quenched. In the presence of target, the hairpin structure opens upon beacon/target hybridization, resulting in emission of fluorescence. For the detection of vanA amplicons, the molecular beacon contains the fluorophore FAM at the $5 ^ { \circ }$ end and the non-fluorescent quencher moiety DABCYL at the opposite end of the oligonucleotide. For the detection of the vanB amplicons, the molecular beacon contains the fluorophore Texas Red at the $5 '$ end and the quencher DABCYL at the $3 ^ { \circ }$ end. For the detection of the Internal Control (IC) amplicons, the molecular beacon contains the fluorophore TET at the $5 ^ { \circ }$ end and the quencher DABCYL at the $3 ^ { \circ }$ end. Each beacon-target hybrid fluoresces at a wavelength characteristic of the fluorophore used in the particular molecular beacon. The amount of fluorescence at any given cycle, or following cycling, depends on the amount of specific amplicons present at that time. The SmartCycler $\textsuperscript { \textregistered }$ software simultaneously monitors the fluorescence emitted by each beacon, interprets all data, and provides a final result at the end of the cycling program.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility panel consisted of low positives near the LoD, moderate and blank (negative) specimens. Panel members were prepared from samples containing negative matrix inoculated with organisms harboring the targeted genes. Specimen Processing Controls (SPC) and Run Controls (Positive and Negative) were included. Each panel member was tested in triplicate per panel run, over five days, with two runs per day and two technologists per site. Two external and one internal, with one lot of reagents, were used to assess site-to-site reproducibility. One clinical site participated in an extended study where two additional lots of reagents were tested to assess lot-to-lot reproducibility with low and moderate positive and negative members.

Site-to-site and lot-to-lot demonstrated $59 5 \%$ reproducibility across negative, low positive and moderate positive categories for both vanA and vanB.

Site-to-site Reproducibility study Results (one lot)   

<table><tr><td></td><td colspan="2">Site 1</td><td colspan="2">Site Site 2</td><td colspan="2">Site 3</td><td colspan="2">Overall %Agreement</td><td colspan="3">Ct Values</td></tr><tr><td></td><td colspan="2">%Agreement</td><td colspan="2">%Agreement</td><td colspan="2">% Agreement</td><td colspan="2"></td><td colspan="3">Overall SD</td></tr><tr><td>NEG</td><td>30/30</td><td>100%</td><td>29/30</td><td>96.7%</td><td>30/30</td><td>100%</td><td>89/90</td><td>98.9%</td><td>Mean</td><td>0.3†</td><td>0.9%†</td></tr><tr><td>Low pos</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>35.7 35.1</td><td>1.0</td><td>2.9%</td></tr><tr><td>vanA Low pos</td><td>30/30</td><td>100%</td><td>26/30</td><td>86.7%</td><td>30/30</td><td>100%</td><td>86/90</td><td>95.6%</td><td>38.4</td><td>1.4</td><td>3.6%</td></tr><tr><td>vanB Mod pos*</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>34.0</td><td>1.1</td><td>3.3%</td></tr><tr><td>vanA Low pos*</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>37.2</td><td>1.4</td><td>3.7%</td></tr></table>

nB † represent values from the internal control \* A single panel member contained both vanA and vanB containing VRE

Lot-to-lot Reproducibility study Results (three lots)   

<table><tr><td rowspan="2"></td><td colspan="2">Lot 1</td><td colspan="2">Lot Lot 2 %Agreement</td><td colspan="2">Lot 3</td><td colspan="2">Overall %Agreement</td><td colspan="3">Ct Values</td></tr><tr><td></td><td>%Agreement</td><td></td><td></td><td>% Agreement</td><td></td><td></td><td></td><td>Overall Mean</td><td>SD</td><td>%CV</td></tr><tr><td>NEG</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>35.6†</td><td>0.3†</td><td>1.0%</td></tr><tr><td>Low pos vanA</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>35.1</td><td>1.0</td><td>1.9%</td></tr><tr><td>Low pos vanB</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>38.4</td><td>1.4</td><td>2.0%</td></tr><tr><td>Mod pos* vanA</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>34.0</td><td>1.1</td><td>2.0%</td></tr><tr><td>Low pos* vanB</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>30/30</td><td>100%</td><td>90/90</td><td>100%</td><td>37.2</td><td>1.4</td><td>2.0%</td></tr></table>

† represent values from the internal control \* A single panel member contained both vanA and vanB containing VRE

An additional reproducibility study was conducted to assess high negative specimens below LoD. 100-fold and 10-fold dilutions were prepared to obtain the two high negative panel members. The more dilute panel member (i.e. 100-fold below the LoD) contained lower levels of target, demonstrated a higher $\%$ agreement for negative test results.

High negatives, site-to-site Reproducibility study Results   

<table><tr><td rowspan="2">High Negatives Panel</td><td colspan="2"></td><td colspan="2">Site Site 2</td><td colspan="2">Site 3</td><td colspan="2">Overall %Agreement</td><td colspan="3">Ct Values</td></tr><tr><td colspan="2">Site 1 %Agreement*</td><td colspan="2">%Agreement*</td><td colspan="2">% Agreement*</td><td colspan="2"></td><td>Overall</td><td>SD</td><td>%CV</td></tr><tr><td>1:100</td><td>22/30</td><td>73.3%</td><td>21/30</td><td>70%</td><td>23/30</td><td>76.7%</td><td>66/90</td><td>73.3%</td><td>Mean 39.2</td><td>0.9</td><td>2.2%</td></tr><tr><td>vanA 1:100</td><td>21/30</td><td>70.0%</td><td>27/30</td><td>90%</td><td>25/30</td><td>83.3%</td><td>73/90</td><td>81.1%</td><td>41.4</td><td>0.6</td><td>1.4%</td></tr><tr><td>vanB 1:10</td><td>0/30</td><td>0.0%</td><td>2/30</td><td>6.7%</td><td>1/30</td><td>3.3%</td><td>3/90</td><td>3.3%</td><td>38.0</td><td>1.1</td><td>2.9%</td></tr><tr><td>vanA 1:10 vanB</td><td>1/30</td><td>3.3%</td><td>3/30</td><td>10.0%</td><td>4/30</td><td>13.3%</td><td>8/90</td><td>8.9%</td><td>39.9</td><td>1.0</td><td>2.4%</td></tr></table>

\* Percent agreement for a negative result

# Precision

Within-laboratory precision was evaluated for the BD GeneOhm VanR Assay at one site. Samples included simulated specimens representing low and moderate positive vanA and vanB containing VRE, high negative specimens (100-fold and 10-fold dilutions of a sample prepared at the LoD), and samples negative for VRE. The study was performed over 12 days, with two (2) runs per day and two (2) sample replicates (negative specimens) or three (3) sample replicates (low/moderate positives and high negatives) per run. All samples produced reportable results. Precision study results for negative samples, low and moderate positive samples demonstrated agreement for $100 \%$ of the replicates. The more dilute (1:100) high negative panel member, containing lower levels of target, demonstrated a higher percent agreement $( 6 8 . 1 \%$ and $6 9 . 4 \%$ for vanA and vanB respectively) for negative test results than the less dilute (1:10) panel member ( $1 . 4 \%$ and $4 . 2 \%$ for vanA and vanB respectively), which contains higher levels of target.

The following tables show the mean CT values and $\% \mathrm { C V }$ obtained for the different types of samples.

Variance component analysis POSITIVE results   

<table><tr><td>Category</td><td>Mean CT</td><td>Within Run CV</td><td>Between Run CV</td><td>Between Day CV</td><td>Overall CV</td></tr><tr><td>Low Pos vanA</td><td>35.6</td><td>1.3%</td><td>0.0%</td><td>0.0%</td><td>1.3%</td></tr><tr><td>Low Pos vanB</td><td>37.2</td><td>1.2%</td><td>0.0%</td><td>0.1%</td><td>1.2%</td></tr><tr><td>Mod Pos vanA</td><td>33.5</td><td>0.9%</td><td>0.3%</td><td>0.0%</td><td>1.0%</td></tr><tr><td>Mod Pos vanB</td><td>36.3</td><td>1.3%</td><td>0.0%</td><td>0.0%</td><td>1.4%</td></tr></table>

Variance components analysis NEGATIVE results   

<table><tr><td>Category</td><td>Mean CT</td><td>Within Run CV</td><td>Between Run CV</td><td>Between Day CV</td><td>Overall CV</td></tr><tr><td>1:100 dil. vanA</td><td>36.0</td><td>0.9%</td><td>0.0%</td><td>0.0%</td><td>0.9%</td></tr><tr><td>1:100 dil. vanB</td><td>35.9</td><td>0.8%</td><td>0.5%</td><td>0.0%</td><td>1.0%</td></tr><tr><td>1:10 dil. vanA</td><td>35.8</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td>1:10 dil. vanB</td><td>35.8</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td></tr><tr><td>NEG</td><td>35.6</td><td>0.9%</td><td>0.4%</td><td>0.0%</td><td>1.0%</td></tr></table>

NA: Not enough data for calculation

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Positive and negative controls are assay controls (run controls). Assay controls were performed at each site demonstrating acceptable QC results most of the time. The overall invalid result (i.e. failure of the positive and/or negative run controls) rates were $3 . 4 \%$ (6/177). The overall initial unresolved results (i.e. failure of the internal control) rate was $1 . 1 \%$ .

External specimen processing controls (E. faecalis ATCC 51299 for VRE, and E. faecalis ATCC 29212 as VSE) were also performed. There were a total of 156 SPC runs for the clinical trial, with an acceptable valid SPC rate of $9 1 . 7 \%$ (143/156).

# Stability Studies

# Reagents Stability

The reagents in their protective sealed pouches for the stability claim of 2- $2 5 \mathrm { { ^ \circ C } }$ were tested at $5 { \pm } 3 ~ ^ { \circ } \mathrm { C }$ for 21 months and $2 5 { \pm } 2 \mathrm { ~ } ^ { \circ } \mathrm { C }$ for 12 months on three lots. Data supported that the reconstituted Master Mix and Control DNA were stable:

a) In the original container, in an ice bucket or cooling block at ambient temperature, for up to 3 hours.   
b) In the SmartCycler tubes, in an ice bucket or cooling block at ambient temperature, for up to 1 hour.   
c) Outside their protective pouch before reconstitution, for up to 2 hours at ambient temperature.

# Specimen Stability

The DNA amplifiability results showed that the four transport media allowed the sample eluate storage at ambient temperature for 48 hrs or $4 \mathrm { { ^ \circ C } }$ for 5 days.

# d. Detection limit:

The limit of detection (LOD) was determined in rectal and perianal matrix with one vancomycin-resistant Enterococcus faecium containing the vanA gene (ATCC 700221) and with three vancomycin-resistant Enterococcus faecalis, each containing one of the vanB1 (ATCC 51299), vanB2 (TUH1-75) or vanB3 (TUH7-68) gene variants. Quantitative culture diluted into negative or perianal matrix was tested in 24 replicates. The LOD was determined using statistical modeling of the lowest concentration of viable organisms achieving $9 5 \%$ detection with the assay. The LoD was determined as follows:

<table><tr><td colspan="3">Rectal</td><td colspan="2">Perianal</td></tr><tr><td></td><td>CFU/rxn</td><td>CFU/swab</td><td>CFU/rxn</td><td>CFU/swab</td></tr><tr><td>E. faecium vanA</td><td>3</td><td>2500</td><td>1</td><td>833</td></tr><tr><td>E. faecalis vanB1</td><td>2</td><td>1449</td><td>5</td><td>3623</td></tr><tr><td>E. faecalis vanB2</td><td>3</td><td>2174</td><td>2</td><td>1449</td></tr><tr><td>E. faecalis vanB3</td><td>5</td><td>3623</td><td>14</td><td>10145</td></tr></table>

# Analytical Reactivity

A total of 156 vancomycin resistant strains (representing 19 countries) from well characterized c1inical isolates or public collections were evaluated using the BD GeneOhm™ VanR Assay. The detection rate was $9 9 . 4 \%$ (155/156).

# Challenge Study

A challenge set comprising of 50 well characterized Enterococci spp. was evaluated. The study included 30 vancomycin resistance due to the vanA or vanB, 10 intrinsically vancomycin resistant (i.e. vanC), and10 vancomycin susceptible Enterococci. All results were as expected.

# e. Analytical specificity:

Genomic DNA or culture lysate were tested for the cross-reactivity study. It included one human DNA specimen, 98 closely related organisms, normal and pathogenic perianal and/or rectal flora, 27 strains of vancomycin-susceptible enterococci and vancomycin-resistant enterococci other than vanA/vanB (i.e. vanC, van $C _ { 2 }$ , vanD, vanE, and vanG). All results were negative.

Vancomycin resistant Staphylococcus aureus (VRSA) strains

Genomic DNA of five VRSA strains from Network on Antimicrobial Resistance in Staphylococcus aureus (NARSA) were tested with the BD GeneOhm™ VanR Assay to determine if amplification/detection of vanA and vanB genes occurred. The GeneOhm™ VanR Assay detected the vanA gene in all the VRSA strains; vanB was not detected in any of these strains. They were also positive on agarose gel for vanA.

# Potentially interfering substances

Potentially interfering substances were tested using materials that may be found in the rectum, such as petroleum jellies, creams, blood, suppositories. Undiluted hydrocortisone cream USP gave unresolved results; blood demonstrated an inhibitory effect and could cause a negative result for a low positive signal.

# f. Assay cut-off:

Acceptance Criteria for the BD GeneOhm™ VanR Assay were established during assay development and pre-validated at one external site with 216 specimens prior to the clinical trial. The acceptance criteria included: Endpoint threshold, $2 ^ { \mathsf { n d } }$ derivative threshold, minimum cycle threshold, maximum cycle threshold, and $1 C \%$ endpoint threshold.

The following table demonstrates the acceptance criteria for the assay:

# Acceptance Criteria for BD GeneOhm™ VanR Assay

<table><tr><td rowspan=1 colspan=3>Clinical specimens</td><td rowspan=1 colspan=2>PositiveControl</td><td rowspan=1 colspan=1>NegativeControl</td></tr><tr><td rowspan=2 colspan=1>Acceptance criteria</td><td rowspan=1 colspan=2>Assay</td><td rowspan=2 colspan=1>InternalControl</td><td rowspan=1 colspan=2>Assay</td><td rowspan=2 colspan=1>InternalControl</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>vanB</td></tr><tr><td rowspan=1 colspan=1>Endpoint threshold</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1> derivative threshold</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Minimum cycle threshold</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Maximum cycle threshold</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>41</td></tr><tr><td rowspan=1 colspan=1>% IC NC endpoint</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr></table>

2. Comparison studies:

The reference method consisted of direct culture complemented by enriched culture. Enriched culture analysis was completed for all specimens that were negative for VRE by direct culture. Direct culture was performed by inoculating specimens onto a primary isolation media containing Bile Esculin Azide agar supplemented with $6 \mu \mathrm { g / m L }$ vancomycin (BEAV). Enriched culture was performed by inoculating $3 0 0 \mu \mathrm { L }$ of sample buffer containing the specimen into BEA broth or BEAV broth (BEA broth supplemented with $6 \mu \mathrm { g / m L }$ vancomycin). Vancomycin resistance was determined for confirmed enterococcal colonies by using a MIC method. Confirmed vancomycin-resistant enterococcal isolates were further tested for genotypic characterization (i.e. vanA and vanB genes) by use of an alternative PCR method.

a. Method comparison with predicate device: See clinical studies below

b. Matrix comparison: Not Applicable

3. Clinical studies:

The clinical study was conducted at five medical centers. A total of 2156 specimens were tested using Direct/Enriched culture with alternative PCR and the BD GeneOhm VanR™ Assay, producing 2150 (1316 perianal and 834 rectal specimens) reportable results. Two hundred and twenty-two specimens (123 perianal and 99 rectal) were culture negative but positive for vanB only by the BD GeneOhm VanR™ PCR and may represent detection of the vanB gene in nonenterococcal organisms. Further investigation for non-enterococcal organisms

that might contain the vanB gene was not performed.

In comparison to Direct/Enriched culture with alternative PCR, the BD GeneOhm™ VanR Assay identified $9 2 . 9 \%$ and $9 3 . 1 \%$ of the perianal and rectal positive specimens, respectively, and identified $8 6 . 0 \%$ and $8 2 . 2 \%$ of the perianal and rectal negative specimens, respectively.

Clinical Performance by Perianal Specimens in comparison to Direct/Enriched culture with Alternative PCR   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sensitivity(95% CI&#x27;)</td><td rowspan=1 colspan=1>Specificity5% CI&#x27;)</td><td rowspan=1 colspan=1>VRE Prevalence</td><td rowspan=1 colspan=1>PPV(95% CI&#x27;)</td><td rowspan=1 colspan=1>NPV(95% CI&#x27;)</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>88.2%(134/152)(81.9% - 92.8%)</td><td rowspan=1 colspan=1>96.6%(1124/1164)(95.3% - 97.5%)</td><td rowspan=1 colspan=1>11.1%(152/1372)</td><td rowspan=1 colspan=1>77.0%(134/174)(70.0% - 83.0%)</td><td rowspan=1 colspan=1>98.4%(1124/1142)(97.5% - 99.1%)</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>100.0%(3/3)(29.2% - 100.0%)</td><td rowspan=1 colspan=1>87.6%(1150/1313)(85.7% - 89.3%)</td><td rowspan=1 colspan=1>0.2%(3/1372)</td><td rowspan=1 colspan=1>1.8%(3/166)(0.4% - 5.2%)</td><td rowspan=1 colspan=1>100.0%(1150/1150)(99.7% - 100.0%))</td></tr><tr><td rowspan=1 colspan=1>VanR</td><td rowspan=1 colspan=1>92.9%(144/155)(87.7% - 96.4%)</td><td rowspan=1 colspan=1>86.0%(998/1161)(83.8% - 87.9%)</td><td rowspan=1 colspan=1>11.3%(155/1372)</td><td rowspan=1 colspan=1>46.9%(144/307)(41.2% - 52.7%)</td><td rowspan=1 colspan=1>98.9%(998/1009)(98.1% - 99.5%)</td></tr></table>

Binomial $9 5 \%$ exact confidence intervals.

Clinical Performance by Rectal Specimens in comparison to Direct/Enriched culture with Alternative PCR   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Sensitivity(5% CI)</td><td rowspan=1 colspan=1>Specificity 5% CI&#x27;)</td><td rowspan=1 colspan=1>VRE Prevalence</td><td rowspan=1 colspan=1>PPV(95% CI&#x27;)</td><td rowspan=1 colspan=1>NPV(95% CI&#x27;)</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>86.4%(108/125)(79.1% - 91.9%)</td><td rowspan=1 colspan=1>96.1%(681/709)(94.3% - 97.4%)</td><td rowspan=1 colspan=1>15.4%(133/863)</td><td rowspan=1 colspan=1>79.4%(108/136)(71.6% - 85.9%)</td><td rowspan=1 colspan=1>97.6%(681/698)(96.1% - 98.6%)</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>100.0%(7/7)(59.0% - 100.0%)</td><td rowspan=1 colspan=1>82.5%(682/827)(79.7% - 85.0%)</td><td rowspan=1 colspan=1>1.0%(9/863)</td><td rowspan=1 colspan=1>4.6%(7/152)(1.9% - 9.3%)</td><td rowspan=1 colspan=1>100.0%(682/682)(99.5% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>VanR</td><td rowspan=1 colspan=1>93.1%(122/131)(87.4% - 96.8%)</td><td rowspan=1 colspan=1>82.2%(578/703)(79.2% - 85.0%)</td><td rowspan=1 colspan=1>16.3%(141/863)</td><td rowspan=1 colspan=1>49.4%(122/247)(43.0% - 55.8%)</td><td rowspan=1 colspan=1>98.5%(578/587)(97.1% - 99.3%)</td></tr></table>

Binomial $9 5 \%$ exact confidence intervals.

A total of 2152 specimens were tested using Direct Culture with alternative PCR and the BD GeneOhm VanR™ Assay, producing 2146 (1314 perianal, 832 rectal) reportable results. Two hundred and thirty-one specimens (129 perianal and 102 rectal) were culture negative but positive for vanB only by the BD GeneOhm VanR™ PCR and may represent detection of the vanB gene in non-enterococcal organisms. Further investigation for non-enterococcal organisms that might contain the vanB gene was not performed.

In comparison to direct culture with alternative PCR, the BD GeneOhm™ VanR Assay identified $9 5 . 0 \%$ and $9 5 . 5 \%$ of the perianal and rectal positive specimens, respectively, and identified $8 3 . 9 \%$ and $8 1 . 0 \%$ of the perianal and rectal negative specimens, respectively.

Clinical Performance by Perianal Specimens in comparison to Direct culture with Alternative PCR

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive Percent Agreement(95% CI&#x27;)</td><td rowspan=1 colspan=1>Negative Percent Agreement(95% CI&#x27;)</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>92.4%(109/118)(86.0%-96.5%)</td><td rowspan=1 colspan=1>94.7%(1133/1196)(93.3%-95.9%)</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>100.0%(1/1)(2.5%-100.0%)</td><td rowspan=1 colspan=1>87.4%(1148/1313)(85.5%-89.2%)</td></tr><tr><td rowspan=1 colspan=1>VanR</td><td rowspan=1 colspan=1>95.0%(113/119)(89.3%-98.1%)</td><td rowspan=1 colspan=1>83.9%(1003/1195)(81.7%-86.0%)</td></tr></table>

1 Binomial $9 5 \%$ exact confidence intervals.

Clinical Performance by Rectal Specimens in comparison to Direct/Enriched culture with Alternative PCR

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive Percent Agreement(95% CI&#x27;)</td><td rowspan=1 colspan=1>Negative Percent Agreement(95% CI&#x27;1)</td></tr><tr><td rowspan=1 colspan=1>vanA</td><td rowspan=1 colspan=1>91.5%(97/106)(84.5%-96.0%)</td><td rowspan=1 colspan=1>94.9%(689/726)(93.0%-96.4%)</td></tr><tr><td rowspan=1 colspan=1>vanB</td><td rowspan=1 colspan=1>100.0%(7/7)(59.0%-100.0%)</td><td rowspan=1 colspan=1>82.5%(681/825)(79.8%-85.1%)</td></tr><tr><td rowspan=1 colspan=1>VanR</td><td rowspan=1 colspan=1>95.5%(107/112)(89.9%-98.5%)</td><td rowspan=1 colspan=1>81.0%(583/720)(77.9%-83.8%)</td></tr></table>

1 Binomial $9 5 \%$ exact confidence intervals.

a. Clinical Sensitivity:

See perianal and rectal clinical performance studies above “…in comparison to Direct/Enriched culture with Alternative PCR”

b. Clinical specificity: See perianal and rectal clinical performance studies above “…in comparison to Direct/Enriched culture with Alternative PCR”   
c. Other clinical supportive data (when a. and b. are not applicable): Not Applicable

4. Clinical cut-off: Not applicable

# Perianal Specimens

<table><tr><td rowspan=1 colspan=1>HealthSystem</td><td rowspan=1 colspan=1>Total N</td><td rowspan=1 colspan=1>Number ofVRE withvanA</td><td rowspan=1 colspan=1>Number ofVRE withvanB</td><td rowspan=1 colspan=1>Number ofVRE WwithvanA and/orvanB</td><td rowspan=1 colspan=1>Prevalence ofVRE with vanA</td><td rowspan=1 colspan=1>Prevalence of VREwith vanB</td><td rowspan=1 colspan=1>Prevalence ofVRE with vanAand/or vanB</td></tr><tr><td rowspan=1 colspan=1>Multi-Institutional</td><td rowspan=1 colspan=1>789</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>9.3% (73/789)</td><td rowspan=1 colspan=1>0.3% (2/789)</td><td rowspan=1 colspan=1>9.5% (75/789)</td></tr><tr><td rowspan=1 colspan=1>Academic</td><td rowspan=1 colspan=1>583</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>13.6% (79/583)</td><td rowspan=1 colspan=1>0.2% (1/583)</td><td rowspan=1 colspan=1>13.7% (80/583)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1372</td><td rowspan=1 colspan=1>152</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>155</td><td rowspan=1 colspan=1>11.1% (152/1372)</td><td rowspan=1 colspan=1>0.2% (3/1372)</td><td rowspan=1 colspan=1>11.3% (155/1372)</td></tr></table>

# Rectal Specimens

<table><tr><td rowspan=1 colspan=1>Health System</td><td rowspan=1 colspan=1>Total N</td><td rowspan=1 colspan=1>Number ofVRE withvanA</td><td rowspan=1 colspan=1>Number ofVRE withvanB</td><td rowspan=1 colspan=1>Number ofVE withvanA and/orvanB</td><td rowspan=1 colspan=1>Prevalence ofVRE with vanA</td><td rowspan=1 colspan=1>Prevalence of VREwith vanB</td><td rowspan=1 colspan=1>Prevalence ofVRE with vanAand/or vanB</td></tr><tr><td rowspan=1 colspan=1>Multi-Institutional</td><td rowspan=1 colspan=1>417</td><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>18.9% (79/417)</td><td rowspan=1 colspan=1>2.2% (9/417)</td><td rowspan=1 colspan=1>20.9% (87/417)</td></tr><tr><td rowspan=1 colspan=1>Academic</td><td rowspan=1 colspan=1>446</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>12.1% (54/446)</td><td rowspan=1 colspan=1>0.0% (0/446)</td><td rowspan=1 colspan=1>12.1% (54/446)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>863</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>141</td><td rowspan=1 colspan=1>15.4% (133/863)</td><td rowspan=1 colspan=1>1.0% (9/863)</td><td rowspan=1 colspan=1>16.3% (141/863)</td></tr></table>

# N. Instrument Name:

Automated SmartCycler $^ \mathrm { \textregistered }$ System (instrument, Dx Software version 1.7b, 3.0a or 3.0b)

# O. System Descriptions:

# 1. Modes of Operation:

The operation of the Smart Cycler instrument is based on the proprietary microprocessor –controlled I-CORE (Intelligent Cooling/Heating Optical Reaction) module. Each Smart Cycler processing block contains 16 independently controlled, programmable I-CORE modules, each with one reaction site. Thermally optimized proprietary reaction tubes combined with the design of the I-CORE modules allow very rapid temperature cycling and rapid amplification. Up to 6 Smart Cycler processing blocks can be daisy-chained together, allowing simultaneous analysis of 96 discrete samples. Information regarding data acquisition, data analysis and diagnostic algorithm was submitted and cleared under k022504 and k042357

# 2. Software:

FDA has reviewed the applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X_ _ or No

3. Specimen Identification:

A liquid or soft stool specimen is collected and transported to the laboratory. A sterile dry swab is dipped into the liquid or soft stool material and processed.

4. Specimen Sampling and Handling:

For testing, the swab is eluted in sample buffer and the specimen is lysed. An aliquot of the lysate is added to PCR reagents which contain the vanA and vanB specific primers used to amplify the genetic target, if present. Amplified targets are detected with hybridization probes labeled with quenched fluorophores (molecular beacons). The amplification, detection and interpretation of the signals are done automatically by the Cepheid SmartCycler software. The entire procedure takes about 60 to 75 minutes

5. Calibration:

N/A

6. Quality Control:

Positive and Negative Controls

Quality control procedures are designed to monitor assay performance. The positive control is intended to monitor substantial reagent failure. The negative control is used to detect reagent or environmental contamination (or carry-over) by either DNA containing vanA or vanB genes or amplicons. Positive and negative controls are assay controls (run controls). An invalid control invalidates the run. Finally, an internal control incorporated into each reaction mixture is intended to monitor the reagent integrity and PCR inhibition in each specimen.

One positive control and one negative control must be included in each assay run on the SmartCycler®. The software automatically assigns the position of the controls on the instrument.

Specimen Processing Controls

A reference VRE strain (e.g. American Type Culture Collection, ATCC 51299) or a well characterized VRE clinical isolate may be used as a positive specimen processing control. A vancomycin sensitive enterococci (e.g. ATCC 29212) or non-vanA/vanB VRE (e.g. Enterococcus gallinarum ATCC 700425) may be used as a negative specimen processing control.

Re-suspend isolated colonies from an 18 to $2 4 \mathrm { h } 5 \%$ sheep blood agar plate in saline to a turbidity of 0.5 McFarland (approximately $1 . 5 \dot { \mathrm { ~ x ~ } } 1 0 ^ { 5 } \mathrm { C F U } / \mathrm { m } \dot { \mathrm { L } }$ ). Dilute with saline to obtain a suspension of approximately $1 0 ^ { 6 } \mathrm { C F U / m L }$ . Dip a recommended swab into the bacterial suspension and press out the excess fluid. Process and test as a clinical specimen. The specimen processing controls should yield valid results.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.