# 510(k) SUMMARY

This summary of $5 1 0 ( 1 \times )$ safety and effectiveness information is being submitted in accordance with the requirements of SMDA 1990 and 21 CFR 807.92.

The assigned 510(k) number is: k102740

Name of device NucliSENS EasyQ® MRSA Assay, including the following components:

Reagents NucliSENS EasyQ® MRSA Instrumentation and Software

NucliSENS EasyQ® MRSA assay protocol software included in the NucliSENS EasyQ® MRSA assay kit

Classification Device Class: Class 2

# Assay Reagent System:

Common/Classification Name: SYSTEM, NUCLEIC ACID AMPLIFICATION TEST, DNA, METHICILLIN RESISTANT STAPHYLOCOCCUS AUREUS, DIRECT SPECIMEN

. Product code: NQX

# Assay Instrumentation and Software System:

Common/Classification Name: INSTRUMENTATION FOR CLINICAL MULTIPLEX TEST SYSTEMS

Product code: 00I

Premarket Notification submitter:

Company Name: bioMérieux, Inc. Company Address: 100 Rodolphe Street Durham, NC 27712

Contact:   
Telephone #:   
FAX #:   
Jocelyn Jennings, Senior Manager, Regulatory Affairs   
919-620-2894   
919-620-2548

Preparation Date: September 20, 2010

# Device to which equivalence is claimed

The assay reagent system which is the subject of this submission is claimed equivalent to the BD GeneOhm test, which received 510(k) clearance from FDA as the IDI-MRSA ™m, #K033415, cleared on 18-Mar-2004.

The instrumentation and software in this submission are claimed equivalent to the instrumentation and software reviewed by FDA as, the NucliSENS EasyQ Enterovirus v1.1, K093383, cleared on 6-July-2010.

# Intended Use of the Device

The NucliSENS EasyQ® MRSA assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) from nasal swabs in patients at risk for nasal colonization. The NucliSENS Easy $a ^ { \infty }$ MRSA assay is performed on the NucliSENS EasyQ® platform.

The test utilizes NASBA ™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA.

The NucliSENS EasyQ® MRSA assay is used as a screening tool to aid in the prevention and control of MRSA infections in health care institutions.

The NucliSENS EasyQ® MRSA is not intended to diagnose, guide or monitor treatment for MRSA infections, or provide results of susceptibility to methicillin.A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

# Description of the Device

1. A dry flocked swab, that is not pre-moistened, is used to collect a specimen from the anterior nares. After collection the swab is transported to the laboratory in its container for specimen processing.   
2. The swab is introduced into the lysis tube and bacteria are recovered from the swab by rotating the swab manually.   
3. Bacteria in suspension in the lysis tube are concentrated by centrifugation.   
4. After carefully discarding the supernatant, the remaining bacterial pellet is subjected to mechanical lysis on a Genie 2 vortex machine. The lysate is subsequently heated at 95 $^ \circ \mathsf { C }$ for 2 minutes.   
5. Five microliters of the raw lysate are added to the NASBA ™m amplification mixture that contains the restriction enzyme, DNA sequence-specific primers and beacons.   
6. Next, the MRSA DNA is digested by the restriction enzyme and denatured in the EasyQ® Incubator.   
7. The NASBA™ enzymes are then added for amplification of the DNA targets by NASBA™ in the EasyQ® Analyzer.   
8. Amplification and detection are performed simultaneously in a dedicated fluorescence reader, the NucliSENS EasyQ® Analyzer.   
9. Amplification is detected by means of sequences-specific probes, which fluoresce upon binding of the probe to its target.   
10. The assay is run in a closed tube format minimizing any risk for amplicons contamination.   
11. The kit contains enough reagents for 48 reactions. Up to 44 specimens (plus 2 required NASBA™ controls and 2 recommended specimen processing controls) can be tested in a single run within 3 hours.   
12. Result calculation is performed automatically via dedicated operator software and is basd on he analysis of fuorecence sigal curves measured by the Nuclises Easy Analyzer. Qualitative output (MRSA positive, MRSA negative or invalid) is reported to users.   
13. An inhibition control (IC) is present in the reaction mix to detect inhibitory specimen.   
14. The NucliSENS EasyQ® MRSA assay includes reagents for the run controls: two NASBA ™ controls (blank and positive control) that must be included in each run.   
15. Positive and negative controls for specimen preparation are recommended for each Easy $\ a ^ { \otimes }$ amplification run; these specimen controls are not provided in the kit.Strains suitable for use as specimen negative control (SNC) and the six specimen positive controls (SPC) are described. These strains allow the operator to perform quality control for all components of the test, including all primers and probes in the kit. The specimen positive controls may be rotated in series leading to one SPC and one SNC per run.

# Reagents and Test Components

Assay Kit Components

Table 1: Components of NucliSENS EasyQ® MRSA   

<table><tr><td colspan="1" rowspan="1">Component</td><td colspan="1" rowspan="1">Composition</td><td colspan="1" rowspan="1">Number of tubes and tubedescription</td><td colspan="1" rowspan="1">Caps Color code</td></tr><tr><td colspan="1" rowspan="1">LYS Tub(lysis tube)</td><td colspan="1" rowspan="1">Proclin solution with glassbeads*</td><td colspan="1" rowspan="1">48 tubes1150 μl per tube</td><td colspan="1" rowspan="1">Uncolored captubes</td></tr><tr><td colspan="1" rowspan="1">REAGENTspheressachet(amplification/detectionreagents)</td><td colspan="1" rowspan="1">Lyophilized white sphere:•  primers, molecularprobesinhibition control:non-infectious DNANASBA™ reagents(nucleotides, DTT,KCI, MgCl2)carbohydrateLyophilized orange sphere:•  Restriction enzymeBovine SerumAlbumin**•  carbohydrate</td><td colspan="1" rowspan="1">6 tubes in foil pack with silica geldesiccant2 spheres per tube (white andorange)</td><td colspan="1" rowspan="1">Blue cap tubes</td></tr><tr><td colspan="1" rowspan="1">ENZ I sphere sachet(enzyme reagents)</td><td colspan="1" rowspan="1">Lyophilized blue sphere :•   AMV-RT, RNase H,T7 RNA polymeraseBovine SerumAlbumin**</td><td colspan="1" rowspan="1">6 tubes in foil pack with silica geldesiccant1 sphere per tube (blue)</td><td colspan="1" rowspan="1">Red cap tubes</td></tr><tr><td colspan="1" rowspan="1">REAGENTOI</td><td colspan="1" rowspan="1">Reagent spheres diluent(Tris/HCI, KCI, DMSO,PVP )</td><td colspan="1" rowspan="1">2 tubes.600 μl per tube</td><td colspan="1" rowspan="1">Blue cap tubes</td></tr><tr><td colspan="1" rowspan="1">ENZdi</td><td colspan="1" rowspan="1">Enzyme sphere diluent (Sorbitol,Bovine Serum Albumin**Proclin     I</td><td colspan="1" rowspan="1">2 tubes.350 μl per tube</td><td colspan="1" rowspan="1">Red cap tubes</td></tr><tr><td colspan="1" rowspan="1">NW</td><td colspan="1" rowspan="1">NASBA ™ water with proclin fordissolution of the CONTROL+sphere and for the NASBA ™blank</td><td colspan="1" rowspan="1">2 tubes.1500 µl per tube</td><td colspan="1" rowspan="1">White cap tubes</td></tr><tr><td colspan="1" rowspan="1">CONTROL+ spheresachet</td><td colspan="1" rowspan="1">NASBA ™ positive control DNA:lyophilized sphere containingnon infectious DNA plasmids</td><td colspan="1" rowspan="1">4 tubes in foil pack with silica geldesiccant1 sphere per tube (white)</td><td colspan="1" rowspan="1">White cap tubes</td></tr><tr><td colspan="1" rowspan="1">TBSTRCPSTR</td><td colspan="1" rowspan="1">8 tubes strip: 6 pieces8 cap strip: 12 pieces</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Not applicable</td></tr><tr><td colspan="1" rowspan="1">CD-ROM  containingInstructions for use forNucliSENS  EasMRSA assay, Assayprotocols, sample loginfile and worksheets</td><td colspan="1" rowspan="1">MRSA SPC US 1.0 for thespecimen positive control,MRSA SNC US 1.0 for thespecimen negative control,MRSA PC US 1.0 for the positivecontrol,MRSA BLK US 1.0 for the blankcontrol,MRSA US 1.0 for clinicalspecimen(s).Controls_MRSA_US_1.0.sI forthe sample login file.MRSA Worksheet SpecimenPreparation.MRSA Worksheet Amplificationand Detection.MRSA Worksheet Preparation ofSpecimen Controls.</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">′</td></tr></table>

$\{ { } ^ { \star } \}$ The volume of beads can vary which does not affect the test performance.   
$( ^ { \star \star } )$ Warnings and precautions: Bovine Serum Albumin is included in Enzyme diluents, Reagent sphere and Enzyme sphere. Certified knowledge of the origin and/or sanitary state of the animals does not totally guarantee the bsenc transmissible pathogenicagents. It is therefore ecommended that these proucts be treated as potentially infectious and handled observing the usual safety precautions (do not ingest or inhale).

Caution: Handle with care; treat as a potentially infectious material.

Assay Components Sold Separately by bioMérieux   

<table><tr><td>Product</td><td>Catalogue number</td></tr><tr><td>NucliSENS EasyQ® System It includes the following: NucliSENS EasyQ® Analyzer NucliSENS EasyQ® Incubator V2 NucliSENS EasyQ® Computer Monitor Printer NucliSENS EasyQ® Director Software 2.6</td><td>4700016</td></tr></table>

# Additional Materials

<table><tr><td rowspan=1 colspan=1>Product</td><td rowspan=1 colspan=1>Catalogue number</td></tr><tr><td rowspan=1 colspan=1>Micro tubes; 1.5 ml (500 x)</td><td rowspan=1 colspan=1>200294</td></tr><tr><td rowspan=1 colspan=1>0.2 ml 8-tube strips (125 pcs)</td><td rowspan=1 colspan=1>285048</td></tr><tr><td rowspan=1 colspan=1>0.2 ml 8-tube caps (125 pcs)</td><td rowspan=1 colspan=1>285051</td></tr><tr><td rowspan=1 colspan=1>EasyQ® Tube tray holder</td><td rowspan=1 colspan=1>45685049</td></tr><tr><td rowspan=1 colspan=1>EasyQ® Cap Installing Tool (Capping aid)</td><td rowspan=1 colspan=1>45685050</td></tr><tr><td rowspan=1 colspan=1>Pipetting aid</td><td rowspan=1 colspan=1>45685207</td></tr></table>

Other

Sterile dry flocked swab with transport tube (bioMerieux reference 280101)   
Sterile water in ampoule compatible with densitometer   
E $2 m |$ and $1 0 m |$ sterile tubes (for preparation of the specimen controls)   
Densitometer   
Standards for calibration of densitometer   
Vortex Genie 2 with MOBlO horizontal 24-positions adaptor only (bioMerieux reference   
for the MOBiO adapter 270677)   
Vortex Genie 2 for mixing individual tubes   
Vortex with flathead dimpled adaptor   
Ice or cooling block for $1 . 5 \mathsf { m }$ tubes   
High speed centrifuge (capable of reaching speeds of up to 16 000 g) for $1 . 5 m |$ tubes   
Calibrated monochannel micropipettes with variable volume: 5 to $1 , 0 0 0 \mu \up$ delivery   
volume   
Sterile-packaged, disposable, aerosol resistant tips (with filter)   
Disposable gloves, powderless   
Tabletop centrifuge for $1 . 5 m 1$ test tube   
Waste container with cap (e.g. 50 ml tubes or zip-bags)   
EY ${ \mathfrak { g } } { \mathfrak { s } } ^ { \circ } { \mathfrak { C } }$ dry bath (calibrated) for incubation of 1.5 ml tubes (range of calibration e.g.t $1 . 7 ^ { \circ } \mathsf { C }$   
Timer   
Sterile $1 . 5 \mathsf { m } ^ { \mathsf { t } }$ tubes   
Mucin from submaxillary gland (Mucin type I-S from Sigma reference M3895-1G)   
MRSA strains: MREJ type ii (ATCC 43300\*), MREJ type ii (ATCC BAA- $3 9 ^ { \star }$ MREJ type   
iv (ATCC BAA- $4 0 ^ { \star } ]$ , MREJ type v (ATCC BAA-2096\*), MREJ type vii (ATCC BAA-2095\*)   
and MREJ type xii (ATCC BAA-2094\*)   
MSSA strain (ATCC 29213\*)   
TSA agar plate   
Blood agar plate   
Applicator

\* Strains can be purchased from ATCC or other vendors.

# Assay Instruments and Software

The instrumentation and software for the NucliSENS EasyQ System, including the EasyQ Analyzer, EasyQ Incubator, and Director 2.5 software, received FDA clearance in June 2008 as part of the 510(k) submission for the NucliSENS EasyQ Enterovirus v1.0 test (submission K063261).

The instrumentation and software for the NucliSENS EasyQ System was re-submitted as K093383 in November 2009, to comply with an FDA request. In K093383, the EasyQ Incubator wa pa ye syna Dar wa software 2.6.

Instrumentation and software for the NucliSENS EasyQ MRSA test is largely unchanged from the system as reviewed in K093383.

Instrumentation: the use of the vortex adaptor plate is new for the EasyQ MRSA test and is documented in this submission (see BTL065234). No changes have been made to the EasyQ Incubator, to the PC (personal computer) or the PC monitor sold for use with the EasyQ Analyzer.

One change has been made to the EasyQ Analyzer. The Analyzer design includes the capacity for 8 filter pairs; however only 2 of the 8 positions were utilized in conjunction for the NucliSENS EasyQ Enterovirus v1.0 test (for detection of FAM and ROX beacons) '. For the NucliSENS EasyQ MRSA test, a third filter pair is used to enable detection of beacons labeled with Cy5.

Software: The Director 2.6 software is unmodified versus the version submitted in K093383. Note that Director software was already configured to enable use of up to 5 filter pairs, dependent on parameters incorporated in the assay protocol developed for each assay application2.

The assay protocol distributed with the EasyQ MRSA kit was developed using the same basic paradigm as for the Enterovirus v1.1 assay protocol in K093383. However whereas the Enterovirus v1.1 assay protocol was customized from the QL1 algorithm, the MRSA assay protocol was customized from a diferent base algorithm, T4. Full software design documentation to support the MRSA assay protocol is therefore included in this submission.

Per BTL036512 Product Design Requirements Labsystems Fluorescence Reader and related Software (Revision of   
Doc. 8730.99.5126 00), as submitted in K093383, the EasyQ Analyzer had 2 filtersets:   
PR23 Filters (Product Requirement 23, page7Current available filtersets:   
Fluorescein $\mathsf E \times 4 8 5 ( 1 4 \pm 2 )$ (half bandwidth) Em $5 3 8 \pm ( 2 5 \pm 3 )$ EPY   
ROX Ex 584±(16±2) Em 620±(15±5)   
Blocking factor outside HBW 10-3   
Filter specifications should enable to meet crosstalk & performance specification   
Per BTL013130 PRD NucliSens EasyQ Director Software, NucliSens EasyQ Director IV $\{ \bullet$ Product Requirements   
Docment, Director 2.5, as submittd in K093383), per requirement PRD1.1.4 on p. 10: "The software willoffr   
support for assays that use more than two fluorescent labels up to 8 labels."

# SUBSTANTIAL EQUIVALENCE INFORMATION

Predicate device name(s): BD GeneOhm MRSA Assay

Predicate device 510(k) number(s) K033415

Comparison with predicate

The EasyQ MRSA test is claimed substantially equivalent to the BD GeneOhm ™ MRSA Assay (K033415).

imilarities and differences between the tests are outlined below:

<table><tr><td rowspan=1 colspan=3>Similarities and Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>EasyQ MRSA</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative diagnostic screening forMRSA in nasal swab specimens</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Nasal swab</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technological Principles</td><td rowspan=1 colspan=1>Manual specimen processing withautomated amplification, detection,analysis and reporting of results</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Amplification technology</td><td rowspan=1 colspan=1>NASBA (Nucleic Acid Sequence-Based Amplification)</td><td rowspan=1 colspan=1>PCR (Polymerase Chain Reaction)</td></tr><tr><td rowspan=1 colspan=1>Detection technology</td><td rowspan=1 colspan=1>Fluorogenic target-specifichybridization</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>EasyQ® System</td><td rowspan=1 colspan=1>Cepheid SmartCycler</td></tr><tr><td rowspan=1 colspan=1>Specimen Processing</td><td rowspan=1 colspan=1>Manual processing in single 1.5-mLtube with glass beads using vortex andcentrifuge</td><td rowspan=1 colspan=1>Manual processing in two microtubesusing glass beads, vortex and centrifuge</td></tr><tr><td rowspan=1 colspan=1>Analysis and reporting ofresults</td><td rowspan=1 colspan=1>Automated using diagnostic software(EasyQ Director and assay protocol)on the EasyQ® System</td><td rowspan=1 colspan=1>Automated using diagnostic software onthe Smart Cycler</td></tr><tr><td rowspan=1 colspan=1>DNA Target Sequence(s)</td><td rowspan=1 colspan=1>1 - Sequence incorporating theinsertion site of StaphylococcalCassette Chromosome mec (SCCmec)at orfX junction, with 6 variant setsprimers and probes to target 7 MREJtypesi,   , ,i iatsequences for MREJ i and ii areidentical)2 - mecA gene</td><td rowspan=1 colspan=1>1- Same2- Not applicable</td></tr><tr><td rowspan=1 colspan=1>Probes</td><td rowspan=1 colspan=1>Molecular Beacons</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Internal control in each test, plus 2external controls (positive and blank)supplied in kit, plus 2 specimencontrols (MRSA and MSSA)recommended for each run</td><td rowspan=1 colspan=1>One internal reagent control andexternal positive and negative controlsrequired per run</td></tr></table>

# Standard/Guidance Document Referenced (if applicable):

Not applicable

# Test Principle:

Specimen Preparation:

# Collection

Trained personnel should perform the collection of swabs and care should be taken to obtain adequate specimen as outlined below:

Open the tube and remove the flocked swab from its empty tube.

Insert the flocked swab into the patient's nostril; rotate the swab 5 times to obtain a sample. Use swab dry without pre-moistening.

Insert the same swab into the second nostril and repeat the sampling.

Insert the swab back into its dry transport tube.

Label the plastic transport tube with patient identification.

# Storage

Specimens once collected can be stored for a period:

up to 4 days, both at room temperature or at $2 . 8 \cdot ^ { \circ } \mathsf { C }$ before testing up to 1 day at room temperature followed by 3 days at $2 . 8 ~ \textdegree C$ before testing.

Nucleic acid amplification and detection

# DNA NASBA TM

In DNA NASBA ™, the bacterial DNA is first digested with a restriction enzyme and then copies of RNA derived from a target DNA sequence are manifold multiplied by isothermal amplification by the combined activities of 3 enzymes: Avian Myeloblastosis Virus Reverse Transcriptase (AMV-RT), Bacteriophage T7 RNA Polymerase (T7 RNA Pol), and RNase H, along with both ribo- and deoxyribo-nucleotides (rNTPs and dNTPs) and the cofactors dithiothreitol (DTT), $M g C l _ { 2 }$ , KCI and DMSO. This process is described below.The figure also shows the differences between NASBA ™ targeted to DNA templates versus RNA templates.

![](images/0ec4f431126dbace7e58e9009d7225e5e9a84d79d77e7331c568204fcd5bc471.jpg)  
Figure 1. DNA vs RNA NASBA™

To start the DNA NASBA™ process, the restriction enzyme Hae II cuts the target DNA near the binding site of primer 1 (P1). Afterwards, the reaction mixture is heated to $\mathfrak { s } \mathfrak { s } ^ { \circ } \mathfrak { C }$ dnatu eaz Ancivat etin nzymSubu the reaction mixture is cooled down to $4 1 ^ { \circ } \mathsf { C }$ and the primer P anneals to the positive strand of the target DNA. This starts he "inear phase" f the NASBA™reaction whee double-stranded DNA amplicons with a T7 promoter tail are generated.

In the cyclic phase of the NASBA ™ reaction, the $\boldsymbol { \mathsf { T 7 } }$ promoter tail on the cDNA target amplicon serves as a binding site for T7 RNA Pol, which then transcribes from the cDNA amplicon using rNTP's to generate a negative strand RNA copy. A single original positive strand target sequence (with T7 tail applied) will serve as a template for multiple RNA copies as the reaction proceeds.

# Detection with Molecular Beacons

Molecular beacons are hairpin structure-forming single-stranded oligonucleotides labelled at one end with a fluorescent dye named the fluorophore and at the other end with a quencher. In the absence of complementary target amplicons, the hairpin structure closes, bringing the fluorophore and quencher into close proximity, and as a result no forescent sinal is mitted. In the presence targe amplicons, the beacon hybriizes complementary target instead  forming an nternal hairpin structure.Onche beacon opens, the quencher and fluorophore separate, quenching ceases and fluorescence is emitted.

![](images/f2cfdf427c78acef62a5962ade6f92e36449fcec18c89dfcf188133facc47413.jpg)  
Figure 2: Principle of Molecular beacon detection

# MRSA assay

The NucliSENS EasyQ® MRSA test utilizes NASBA™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA in nasal swabs from patients at risk for nasal colonization with MRSA. Amplification and detection occur simultaneously in the EasyQ Analyzer.

The NucliSENS EasyQ® MRSA is developed for amplification of two targets simultaneously: the SCCmec cassette junction and the mecA gene. False positive results have been reported in the literature with tests that detect the cassette junction only due to a phenomenon known as "mecA dropout" leading to methicillin-sensitive Staphylococcus aureus strains that lost the mecA gene but still harbour the cassette junction. Such strains are detected as MRSA positive by such tests. (See reference 1 in the Bibliography section). The NucliSENS EasyQ® MRSA will report a MRSA positive result only if both targets, the SCCmec cassette junction and the mecA gene, are detected, thus reducing the risk of false positive detection. Primers and beacons in the kit are complementary to the SCCmec cassette junction sequences of MREJ types i-v, vi, and xii.

# Explanation of the assay

A dry flocked swab, that is not pre-moistened, is used to collect a specimen from the anterior nares. After collection the swab is transported to the laboratory in its container for specimen processing.

The swab is introduced into the lysis tube and bacteria are recovered from the swab by rotating the swab manually.

Bacteria in suspension in the lysis tube are concentrated by centrifugation.

After carefully discarding the supernatant, the remaining bacterial pellet is subjected to mechanical lysis on a Genie 2 vortex machine. The lysate is subsequently heated at $9 5 ~ ^ { \circ } \mathsf { C }$ for 2 minutes.

Five microliters of the raw lysate are added to the NASBA ™ amplification mixture that contains the restriction enzyme, DNA sequence-specific primers and beacons.

Next, the MRSA DNA is digested by the restriction enzyme and denatured in the EasyQ® Incubator.

The NASBA ™ enzymes are then added for amplification of the DNA targets by NASBA™ in the EasyC $\bigotimes$ analyzer.

Amplification and detection are performed simultaneously in a dedicated fluorescence reader, the NucliSENS EasyQ® Analyzer.

• Amplification is detected by means of sequence-specific probes which fluoresce upon binding of the probe to its target.

The assay is run in a closed tube format minimizing any risk for amplicons contamination.

The kit contains enough reagents for 48 reactions. Up to 44 specimens (plus 2 required NASBA ™ controls and 2 recommended specimen processing controls) can be tested in a single run within 3 hours.

Result calculation is performed automatically via dedicated operator software and is based on the analysis of fluorescence signal curves measured by the NucliSENS EasyQ® Analyzer. Qualitative output (MRSA positive, MRSÁ negative or invalid) is reported to users.

An inhibition control (IC) is present in the reaction mix to detect inhibitory specimen.

The NucliSENS EasyQ® MRSA assay includes reagents for the run controls: two NASBA ™ controls (blank and positive control) that must be included in each run

Positive and negative controls for specimen preparation are recommended for each EasyQ® amplification run; these specimen controls are not provided in the kit. Strains suitable for use as specimen negative control (SNC) and the six specimen positive controls (SPC) are described in the package insert. These strains allow the operator to perform quality control for all components of the test, including all primers and probes in the kit. The specimen positive controls may be rotated in series with one SPC per run.

# Performance Characteristics

# Interfering substances

A study was conducted with potentially interfering substances encountered in nasal swabs. Substances tested were chemical substances that can either be naturally present in the nasal cavity or that can be artificially introduced into the nasal cavity.

Accordingly, the following substances were tested and evaluated: blood, mucin, azelastine HCI (Astelin), Eucalyptus globulus/ kalium bichromicus (Sino Fresh), phenylephrine HCI (Neo-synephrine), cromolyn sodium (NasalChrom), Zicum Gluconium (Zicam gel), oxymetazoline HCl (Vicks Sinex), Triamcinolone acetonide (Nasacort), mometasone furoate (Nasonex).

The following drugs / biological substances have been shown to interfere with the performance of the assay:

Astelin (50 μl per swab)

Sino Fresh (50 μl per swab) Nasal chrom (50 μl per swab) Zicam (50 µl per swab) Blood ( $1 5 \mu \mu$ and 50 µl per swab).

In the context of the clinical trials, blood was reported on nasal swabs for 20/1238 $( 1 . 6 \% )$ specimens, one of which yielded an invalid result.

# Biological interference

An excess of methicillin-sensitive Staphylococcus aureus or methicillin-resistant Staphylococcus epidermidis or methicillin-sensitive Staphylococcus epidermidis did not interfere with the detection of MRSA or with the reporting of MRSA negative swabs.

Analytical sensitivity: Limits of Detection (LOD) for strains of MRSA with different types of MREJ

The analytical sensitivity of the NucliSENS EasyQ® MRSA assay was determined using clinical nasal matrices and various strains of MRSA representing the seven types of MREJ detected by the test. Bacterial suspensions with a bacterial input ranging from 10 to 1000 CFU per swab were tested in 24 replicates of each input.

The limit of detection was defined as the input corresponding to the $9 5 \%$ Hit Rate predicted by statistical analysis using a probit regression model.

The estimated type-specific LoDs were further confirmed with two batches of reagents. The claimed type-specific LoD summarized below correspond to the higher concentrations claimed from these two studies either real input determined by plating and counting bacteria colony forming units in the confirmation study, or predicted concentrations by the probit regression model.

# LoD for seven MRSA types

<table><tr><td rowspan=1 colspan=1>MREJtype</td><td rowspan=1 colspan=1>SCCmectype</td><td rowspan=1 colspan=1>LOD(CFU/swab)</td></tr><tr><td rowspan=1 colspan=1>i</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>410</td></tr><tr><td rowspan=1 colspan=1>ii</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>227</td></tr><tr><td rowspan=1 colspan=1>iii</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>493</td></tr><tr><td rowspan=1 colspan=1>iv</td><td rowspan=1 colspan=1>IlI</td><td rowspan=1 colspan=1>•           234</td></tr><tr><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>200</td></tr><tr><td rowspan=1 colspan=1>vii</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>482</td></tr><tr><td rowspan=1 colspan=1>xii</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>182</td></tr></table>

# Analytical specificity

# Sequence alignments

Primers and probes were designed for optimal specificity by comparative nucleic acid sequence analysis of genetically related strains of Staphylococci and of unrelated species known to be present in nasal specimens.

Using this sequence analysis, the combination of MRSA primers and beacons was shown to be highly specific to MRSA strains.

# Experimental data

Fresh cultures from 100 phylogenetically related S.aureus and members of the nasal commensal flora were tested at an input of 7.5X10E6 CFU/swab (26 strains of methicillin-resistant coagulase negative staphylococci, 29 strains of methicillin

sensitive coagulase-negative staphylococci, 25 strains of methicillin-sensitive coagulase-positive staphylococci including 20 strains of MSSA, 20 strains other than staphylococci belonging to the nasal commensal flora including 1 strain of Grampositive rods, 10 strains of Gram-negative rods, 7 strains of non staphylococci Grampositive cocci, 1 strain of Gram-negative cocci, 1 strain of yeast).

One strain of MSSA was initially detected positive for the cassette junction due to contamination. This MSSA strain was re-tested and found negative for the cassette junction.

All the strains were detected MRSA negative except two strains when tested with mecA positive clinical samples.

# Analytical Reactivity

# Sequence alignments

Genomic nucleic acid sequences of 319 strains of MRSA were found in public databases and were analyzed for inclusivity with the NucliSENS EasyQ® MRSA assay. Sequence analysis indicated that 314 strains were theoretically covered by the design of primers and beacons $( 9 8 . 4 \% )$ .

# Experimental data

Fresh cultures from 193 MRSA strains were tested at an input of 3 times the LOD. The 193 strains comprised:

Strains of MRSA belonging to the following MREJ types: 21 MREJ type i, 119 MREJ type ii, 16 MREJ type ii, 2 MREJ type iv, 1 MREJ type V, and 2 MREJ types vii. In addition, strains belonging to the new MREJ types ix, xii, ii, and xiv were also included.   
Strains MRSA belonging to each of the SCCmec types I, II, II, IV, V and VI. Strains belonging to the worldwide most prevalent MRSA clones.   
Strains of MRSA belonging to the PFGE types USA 100, 200, 300, 400, 500, 600, 700, 800, 1000 and 1100.   
Strains of MRSA characterized for MIC to oxacillin including both homogeneous highly resistant strains with high oxacillin MIC and heterogeneous resistant strains with lower oxacillin MIC $( 2 - 8 | | g / | m | )$ values.   
Strains of MRSA recently isolated in USA hospitals.

The NucliSENS EasyQ® MRSA assay accurately detected 184 of the 193 isolates of MRSA tested, $9 5 . 3 \%$ .

# Precision within-laboratory and between-laboratory

The precision of the NucliSENS EasyQ MRSA assay was evaluated using artificial samples consisting of human nasal matrix spiked with MRSA strains at 3 different inputs. The precision study was determined with respect to the complete assay procedure, comprising all steps from the extraction of the specimen from the swab to the final result.

The panel of tested samples was comprised of three different bacterial inputs, tested in 3 replicates per run: "High Negative" samples corresponding to approximately 0.03 x LOD, "Low Positive" samples corresponding to approximately $1 . 0 0 \times \mathsf { L O D }$ and "Moderate Positive" samples corresponding to approximately $4 . 8 2 \times \mathsf { L O D }$ .

The precision was analyzed by determining for each bacterial input the percentage of correct NucliSENS EasyQ MRSA test results based on the expected outcome.

A "within-laboratory" precision study was performed at one internal site over 12 days, with 2 operators, using 2 instruments and 2 lots of reagents.

The overall percent agreement with the expected outcome was $9 0 . 3 \%$ for the high negative input, $9 8 . 6 \%$ for the low positive input and $100 \%$ for the moderate positive input in the table below. The table below also shows for the maxLambda mecA and maxLambda cassette junction values (internal measurements to determine the final assay result), the overall means, standard deviations, and coefficients of variation obtained for testing with the different bacterial inputs.

Within-Laboratory Variation   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambda cassettejunction</td></tr><tr><td rowspan=1 colspan=1>Sample Input&#x27;</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative (HN)</td><td rowspan=1 colspan=1>90.3% (65/72)</td><td rowspan=1 colspan=1>81.0-96.0%</td><td rowspan=1 colspan=1>2.669</td><td rowspan=1 colspan=1>0.0592</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.034</td><td rowspan=1 colspan=1>0.0723</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LP)</td><td rowspan=1 colspan=1>98.6% (71/72)</td><td rowspan=1 colspan=1>92.5-99.9%</td><td rowspan=1 colspan=1>2.641</td><td rowspan=1 colspan=1>0.0782</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.578</td><td rowspan=1 colspan=1>0.2178</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive(MP)</td><td rowspan=1 colspan=1>100% (72/72)</td><td rowspan=1 colspan=1>95.0-100%</td><td rowspan=1 colspan=1>2.680</td><td rowspan=1 colspan=1>0.0564</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.996</td><td rowspan=1 colspan=1>0.1498</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>Total Agreement</td><td rowspan=1 colspan=1>96.3%(208/216)</td><td rowspan=1 colspan=1>92.8-98.4%</td><td rowspan=1 colspan=6></td></tr></table>

HN = approx. 0.03 x LOD LP= approx. 1.00 x LOD MP= approx. 4.82 x LOD

A "between-laboratory" precision study was performed at 3 sites (1 internal site and 2 external sites in the US), over 6 days, with 2 operators at each site, 1 instrument at each site and 2 lots of reagents.

The overall percent agreement with the expected outcome was $9 3 . 5 \%$ for the high negative input, $8 7 . 0 \%$ for the low positive input, and $100 \%$ for the moderate positive input in the table below. The table below also shows for the maxLambda mecA and maxLambda cassette junction values the overall means, standard deviations, and coefficients of variation obtained for testing with the different bacterial inputs.

# Between-Laboratory Variation

<table><tr><td rowspan=1 colspan=5>EY</td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambdacassette junction</td></tr><tr><td rowspan=1 colspan=1>SampleI nput</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%vV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>cv</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agremmentand 95% CI</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>“</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hig NegativeX(HN)</td><td rowspan=1 colspan=1>94.4%(34/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>86.1%(31/36)</td><td rowspan=1 colspan=1>93.5%(101/108)87.1-97.4%</td><td rowspan=1 colspan=1>2.650</td><td rowspan=1 colspan=1>0.0662</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.024</td><td rowspan=1 colspan=1>0.0558</td><td rowspan=1 colspan=1>5.4</td></tr><tr><td rowspan=1 colspan=1>Low PositiveLP)</td><td rowspan=1 colspan=1>85.7%(31/36)</td><td rowspan=1 colspan=1>75.0% (227/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>87.0%(94/108)79.2 -92.7%</td><td rowspan=1 colspan=1>2.631</td><td rowspan=1 colspan=1>0.0837</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.347</td><td rowspan=1 colspan=1>0.270720.1</td><td rowspan=1 colspan=1>20.11</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LP2)</td><td rowspan=1 colspan=1>100%36/36)</td><td rowspan=1 colspan=1>97.2%35/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>99.1%(107/108)94.9 -99.9%</td><td rowspan=1 colspan=1>2.557</td><td rowspan=1 colspan=1>0.0995</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>1.735</td><td rowspan=1 colspan=1>0.2405</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>ModeratePositiveMP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(108/108)96  100%</td><td rowspan=1 colspan=1>2.670</td><td rowspan=1 colspan=1>0.1119</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.816</td><td rowspan=1 colspan=1>0.2550</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>95.1%(137/144)</td><td rowspan=1 colspan=1>93.1%(134/144)</td><td rowspan=1 colspan=1>96.5%(139/144)</td><td rowspan=1 colspan=1>94.9%(410/432)92.4-96.8%</td><td rowspan=1 colspan=6></td></tr></table>

THN = approx. 0.03 x LoD LP= approx. 1.00 x LoD $\mathsf { L P 2 } \mathsf { = }$ EPY approx. 2.00 x LoD MP= approx. 4.82 x LoD

Between-Lot Variation ( Between-Laboratory)   

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambdacassette junction</td></tr><tr><td rowspan=1 colspan=1>Sample Input&#x27;</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>% Agreementand 95% CI</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=1 colspan=1>High Negative (HN)</td><td rowspan=1 colspan=1>94.4%551/54)</td><td rowspan=1 colspan=1>92.6%50/54</td><td rowspan=1 colspan=1>93.5%(101/108)87.1 -97.4%</td><td rowspan=1 colspan=1>2.650</td><td rowspan=1 colspan=1>0.0662</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.024</td><td rowspan=1 colspan=1>0.0558</td><td rowspan=1 colspan=1>5.4</td></tr><tr><td rowspan=1 colspan=1>LowPositiveLP)</td><td rowspan=1 colspan=1>90.7%49/54)</td><td rowspan=1 colspan=1>83.3%5/54)</td><td rowspan=1 colspan=1>87.0%(94/108)79.2 - 92.7%</td><td rowspan=1 colspan=1>2.631</td><td rowspan=1 colspan=1>0.0837</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>1.3470.2707</td><td rowspan=1 colspan=1>1.3470.2707</td><td rowspan=1 colspan=1>20.1</td></tr><tr><td rowspan=1 colspan=1>LowPositive (LP2)</td><td rowspan=1 colspan=1>98.1%53/54)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>99.1%(107/108)94.9 - 99.9%</td><td rowspan=1 colspan=1>2.557</td><td rowspan=1 colspan=1>0.0995</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>1.735</td><td rowspan=1 colspan=1>0.2405</td><td rowspan=1 colspan=1>13.9</td></tr><tr><td rowspan=1 colspan=1>Moderateooie (MP)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>100%(54/54)</td><td rowspan=1 colspan=1>100%(108/108)96.6 -100%</td><td rowspan=1 colspan=1>2.670</td><td rowspan=1 colspan=1>0.1119</td><td rowspan=1 colspan=1>4.2</td><td rowspan=1 colspan=1>1.8160.2550</td><td rowspan=1 colspan=1>1.8160.2550</td><td rowspan=1 colspan=1>14.0</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>95.8%(207/216)</td><td rowspan=1 colspan=1>94.0%(203/216)</td><td rowspan=1 colspan=1>94.9%(410/432)92.4 -96.8%</td><td rowspan=1 colspan=6>*</td></tr></table>

HN = approx. 0.03 x LoD LP2= approx. 2.00 x LoD LP= approx. 1.00 x LoD MP= approx. 4.82 x LoD

# Between lot variation (Within —Laboratory)

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=3>MaxLambda mecA</td><td rowspan=1 colspan=3>MaxLambdacassette junction</td></tr><tr><td rowspan=1 colspan=1>Sample Input</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>% Agreementand 95% CI</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative(HN)</td><td rowspan=1 colspan=1>91.7%(33/36)</td><td rowspan=1 colspan=1>88.9%(32/36)</td><td rowspan=1 colspan=1>90.3% (651/72)81.0-96.0%</td><td rowspan=1 colspan=1>2.669</td><td rowspan=1 colspan=1>0.0592</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.034</td><td rowspan=1 colspan=1>0.00723</td><td rowspan=1 colspan=1>7.0</td></tr><tr><td rowspan=1 colspan=1>LowPositive (LP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>97.2%(3/36)</td><td rowspan=1 colspan=1>98.6% (172)92.5 - 99.9%</td><td rowspan=1 colspan=1>2.641</td><td rowspan=1 colspan=1>0.0782</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.578</td><td rowspan=1 colspan=1>0.2178</td><td rowspan=1 colspan=1>13.8</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive (MP)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(36/36)</td><td rowspan=1 colspan=1>100%(72172)95.0 -100%</td><td rowspan=1 colspan=1>2.680</td><td rowspan=1 colspan=1>0.0564</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.9960.1498</td><td rowspan=1 colspan=1>1.9960.1498</td><td rowspan=1 colspan=1>7.5</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>97.2%(105/108)</td><td rowspan=1 colspan=1>95.4%(103/108)</td><td rowspan=1 colspan=1>96.3%(208/216)92.8- 98.4%</td><td rowspan=1 colspan=6></td></tr></table>

EPY ${ } ^ { 1 } \mathsf { H N } =$ EPY approx. 0.03 x LoD $\mathsf { L P } \mathsf { \Lambda } =$ approx. 1.00 x LoD ${ \mathsf { M P } } =$ EY approx. 4.82 x LoD

# Carry-Over Contamination

To evaluate the risk of carry-over contamination for the NucliSENS EasyQ MRSA assay during the processes of sample preparation, amplification and detection, five MRSA NASBA runs were performed with alternating high MRSA positive $( 5 \times 1 0 ^ { 6 }$ CFU/lysis tube) and MRSA negative samples (negative for both the mecA and the SCCmec cassette). Artificial samples were used consisting of lysis tubes with frozen matrix from clinical swabs, the positive artificial samples being spiked with a strain of MRSA at the concentration indicated above.

In total, 105 high positive and 105 MRSA negative samples were tested divided over 5 EasyQ runs.

The results of this carry-over contamination study showed that no cross-over contamination was observed amongst 103 valid negative specimens results surrounded by high positive specimens (105 MRSA negative samples were tested, but 2 were classified as invalid and thus discarded).

Thus, the workflow and design of the assay were shown to be robust against carry-over contamination.

# Clinical Performance

# Clinical Study Design

Performance characteristics of the NucliSENS EasyQ® MRSA assay were determined in a multicenter prospective investigational study employing 7 geographically diverse institutions in the US, including 2 pediatric sites. Five (5) sites collected and tested their own specimens and $2$ collection-only sites sent their specimens to one of the testing sites. The NucliSENs EasyQ MRSA assay was compared to the reference enriched culture, the most sensitive culture method. Subjects included patients from healthcare institutions at risk for colonization with MRSA and re creenig culturs.Each subject was enroll n he sud only on.Subjets akin antibiotics for eradication of MRSA colonization or for treatment of MRSA infection in the 7 days prior to specimen collection, patients that had contraindications to nasal swab collection or children under 2 years old were excluded from the trial.

Two (2) nasal swabs were prospectively collected from each patient; One (1) swab was tested with the NucliSENS Easy $\textstyle \mathbf { Q } ^ { \mathfrak { P } }$ MRSA assay and 1 swab with the reference culture method. For the enriched culture method, a swab was directly inoculated into an enrichment broth consisting of trypticase soy broth (TSB) with $6 . 5 \%$ NaCl and incubated for 18-24 hours at 35 $. 3 7 ^ { \circ } C$ The enrichment broth was then streaked onto a blood agar plate (BAP) and incubated for 24-48 hours at $3 5 - 3 7 ^ { \circ } C$ The plate was then examined and suspect S. aureus colonies were confirmed with a Gram stain and latex agglutination test. Confirmed S. aureus colonies were tested for methicillin resistance using the cefoxitin disk test as described in CLSI M2A9 and CLSI M100 S17.

Performance (sensitivity and specificity) of the NucliSENS EasyQ® MRSA test was calculated relative to the reference culture results.

# Overall Clinical Study Results: Agreement of NucliSENS EasyQ® Test with Reference Culture

Results from the NucliSENS EasyQ® MRSA assay compared with results from the reference culture method are presented in the tables below.

There were 144 final evaluable specimens that were positive for MRsA by the reference culture method. There were $1 4 ^ { 1 }$ specimens that were invalid for NucliSENS EasyQ ® MRSA; 1 was positive by reference culture and 13 were negative by reference culture.

NucliSENS EasyQ @ MRSA had a clinical sensitivity $9 5 . 8 \%$ clinical specificity o $9 6 . 8 \%$ ,a Positive Predictive Value (PPV) o $7 9 . 7 \%$ and a Negative Predictive Value (NPV) of $9 9 . 4 \%$ .

A total of 55 nasal specimens were colected of which 138 had reportable results and are included in the analysis. There were 117 specimens excluded due to non-compliance with the clinical protocol.

# Results Comparing NucliSENS EasyQ® MRSA with Reference Culture.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>NucliSENSEasyQ® MRSA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>172</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1046</td><td rowspan=1 colspan=1>1052</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>1081</td><td rowspan=1 colspan=1>1224</td></tr></table>

\*mecA mediated oxacillin resistance using $3 0 \mu \ g$ cefoxitin disk 1invalids not included in the calculation

Performance Parameters - NucliSENS EasyQ® MRSA versus Reference Culture   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ParameterEstimate</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Sensitivity: EasyQ vs. Reference Culture (137/143)</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>91.1-98.4%</td></tr><tr><td rowspan=1 colspan=1>Specificity: EasyQ vs. Reference Culture (1046/1081)</td><td rowspan=1 colspan=1>96.8 %</td><td rowspan=1 colspan=1>95.5-97.7%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Positive Predictive Value with Reference Culture (137/172)</td><td rowspan=1 colspan=1>79.7%</td><td rowspan=1 colspan=1>72.9-85.4%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Negative Predictive Value with Reference Culture (1046/1052)</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>98.8-99.8 %</td></tr></table>

Positive Predictive Value and Negative Predictive Value of the NucliSENS EasyQ MRSA Assay Compared to the Reference Culture Method and Prevalence of MRSA at Individual Sites.   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MRSAPrevalence</td><td rowspan=1 colspan=1>PPVNucliSENSMRSA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPVNuclisENS MRSA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>10.0% (20/200)</td><td rowspan=1 colspan=1>67.9% (19/28)</td><td rowspan=1 colspan=1>47.7-84.1%</td><td rowspan=1 colspan=1>99.4% (168/169)</td><td rowspan=1 colspan=1>96.8-99.9%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9.1% (39/427)</td><td rowspan=1 colspan=1>83.3%(35/42)</td><td rowspan=1 colspan=1>68.6-93.0%</td><td rowspan=1 colspan=1>99.0% (378/382)</td><td rowspan=1 colspan=1>97.3-99.7%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>33.6% (50/149)</td><td rowspan=1 colspan=1>85.7%(48/56)</td><td rowspan=1 colspan=1>73.8-93.6%</td><td rowspan=1 colspan=1>98.9%(90/91)</td><td rowspan=1 colspan=1>94.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10.3% (6/58)</td><td rowspan=1 colspan=1>85.7% (6/7)</td><td rowspan=1 colspan=1>42.1-99.6%</td><td rowspan=1 colspan=1>100% (45/45)</td><td rowspan=1 colspan=1>92.1-100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>8.1% (24/296)</td><td rowspan=1 colspan=1>77.4% (24/31)</td><td rowspan=1 colspan=1>58.9-90.4%</td><td rowspan=1 colspan=1>100% (265/265)</td><td rowspan=1 colspan=1>98.6-100%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6.5% (3/46)</td><td rowspan=1 colspan=1>100% (3/3)</td><td rowspan=1 colspan=1>29.2-100%</td><td rowspan=1 colspan=1>100% (43/43)</td><td rowspan=1 colspan=1>91.8-100%</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3.2% (2/62)</td><td rowspan=1 colspan=1>40.0% (2/5)</td><td rowspan=1 colspan=1>5.3-85.3%</td><td rowspan=1 colspan=1>100% (57/57)</td><td rowspan=1 colspan=1>93.7-100%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11.6% (144/1238)</td><td rowspan=1 colspan=1>79.7% (137/172)</td><td rowspan=1 colspan=1>72.9-85.4%</td><td rowspan=1 colspan=1>99.4% (1046/1052)</td><td rowspan=1 colspan=1>98.8-99.8%</td></tr></table>

# Clinical Study Results: Agreement of NucliSENS EasyQ® MRSA Test with Reference Culture for Adults

Of the 1238 nasal specimens that had reportable results, 877 were collected from adult patients. Results from the NucliSENS Easy $\boldsymbol { \Omega } ^ { \circledast }$ MRSA assay with adult specimens compared with results from the reference culture method are presented in tables 13 and 14 below.

There were 114 final evaluable specimens that were positive for MRSA by the reference culture method. There were $1 4 ^ { 1 }$ specimens that were invalid for NucliSENS EasyQ MRSA; 1 was positive by reference culture and 13 were negative by reference culture.

NucliSeNS Easy $Q ^ { \otimes }$ MRSA had a clinical sensitivityf $9 4 . 7 \%$ and a clinical specificity of $9 6 . 5 \%$ .

Results Comparing NucliSENS EasyQ MRSA with Reference Culture for Adult Specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>NucliSENS EasyQ® MRSA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>133</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>724</td><td rowspan=1 colspan=1>730</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>750</td><td rowspan=1 colspan=1>863</td></tr></table>

\* mecA mediated oxacillin resistance using $3 0 \mu \ g$ cefoxitin disk 1 invalids not included in the calculation

Performance Parameters - NucliSENS EasyQ® MRSA versus Reference Culture for Adult Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ParameterEstimate</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Sensitivity: EasyQ vs.Reference Culture (107/113)</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>88.8-98.0%</td></tr><tr><td rowspan=1 colspan=1>Specificity: EasyQ vs. Reference Culture(724/750)</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>95.0-97.7%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Positive Predictive Value with Reference Culture(107/133)</td><td rowspan=1 colspan=1>80.5%</td><td rowspan=1 colspan=1>72.7-86.8%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Negative Predictive Value with Reference Culture(724/730)</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.2-99.7%</td></tr></table>

# Clinical Study Results: Agreement of NucliSENS EasyQ® MRSA Test with Reference Culture for Pediatrics

Pediatric samples included specimens from Child, Adolescent, and Transitional Adolescent patients. Of the 1238 nasal specimens that had reportable results, 361 were collected from pediatric patients. Results from the NucliSENS EasyQ® MRSA assay with pediatric specimens compared against results from the reference culture method are presented in the tables below. Age distribution is presented in the table below.

There were 30 final evaluable specimens that were positive for MRSA by the reference culture method. NucliSENS EasyQ® MRSA had a clinical sensitivity of $100 \%$ and a clinical specificity of $9 7 . 3 \%$ .

Results Comparing NucliSENS EasyQ $\textcircled{8}$ MRSA Assay with Reference Culture for Pediatric Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Reference Culture*</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>NucliSENSEasyQ® MRSA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>322</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>331</td><td rowspan=1 colspan=1>361</td></tr></table>

mecA mediated oxacillin resistance using $3 0 \mu \ g$ cefoxitin disk

Performance Parameters - NucliSENS EasyQ® MRSA versus Reference Culture for Pediatric Specimens   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ParameterEstimate</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Sensitivity: EasyQ vs. Reference Culture (30/30)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.4-100%</td></tr><tr><td rowspan=1 colspan=1>Specificity: EasyQ vs. Reference Culture (322/331)</td><td rowspan=1 colspan=1>97.3%</td><td rowspan=1 colspan=1>94.9-98.8%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Positive Predictive Value with Reference Culture (30/39)</td><td rowspan=1 colspan=1>76.9%</td><td rowspan=1 colspan=1>60.7-88.9%</td></tr><tr><td rowspan=1 colspan=1>EasyQ Negative Predictive Value with Reference Culture (322/322)</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9-100%</td></tr></table>

# Age Distribution of Patients

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Child (2years - &lt;12years)</td><td rowspan=1 colspan=1>Adolescent (12years - &lt;18 years)</td><td rowspan=1 colspan=1>TransitionalAdolescents (18years - &lt;21 years)</td><td rowspan=1 colspan=1>Adults (≥ 21 years)</td><td rowspan=1 colspan=1>Total All Ages</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>13% (158/1238)</td><td rowspan=1 colspan=1>14% (178/1238)</td><td rowspan=1 colspan=1>2% (25/1238)</td><td rowspan=1 colspan=1>71% (877/1238)</td><td rowspan=1 colspan=1>100% (1238/1238)</td></tr><tr><td rowspan=1 colspan=1>MRSA Prevalence</td><td rowspan=1 colspan=1>5.1% (8/158)</td><td rowspan=1 colspan=1>10.7% (19/178)</td><td rowspan=1 colspan=1>12.0% (3/25)</td><td rowspan=1 colspan=1>13.0% (114/877)</td><td rowspan=1 colspan=1>11.6% (144/1238)</td></tr></table>

# Clinical Study Results: Invalid Results

Testing of clinical specimens showed an initial invalid result rate of $3 . 4 \%$ (42/1238). There were 42 initial invalid results for NucliSENS Easy ${ \sf Q } ^ { \otimes }$ MRSA from 114 valid runs of which $1 . 1 \%$ E (14/1238) remained invalid after repeat testing from frozen lysate. The overall invalid run rate was $10 . 2 \%$ (13/127).

# Proposed labeling

The proposed labeling is complete.

# Conclusion

Toatoh parke oatiplert  usanl equv decision.

# Bibliography

J.Rupp et al. Be aware of the possibility of false positive result in single locus PCR assays for methicillin-resistant Staphylococcus aureus - J. Clin. Microbiol. - 2006, vol. 44, p. 2317.

bioMérieux, Inc. c/o-Jocelyn Jennings, R.A.C. Sr. Manager, Regulatory Affairs 100 Rodolphe Street Durham, NC 27712

Re: K102740 Trade/Device Name: NucliSENS EasyQ® MRSA Assay Regulation Number: 21 CFR 866.1640 Regulation Name: Antimicrobial susceptibility test powder Regulatory Class: Class II Product Code: NQX, OOI Dated: May 10, 2011 Received: May 12, 2011

Dear Ms. Jennings:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce-prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class II (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section 510(k) premarket

notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html. Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

![](images/f71993ecccfb3f2b7d7b3b00f012b8b9a3c25643706d971a7e6277ed72e37d7a.jpg)  
Page 2 - Jocelyn Jennings

# INTENDED USE STATEMENT

510(k) Number (if known): k102740

Device Name: NucliSENS EasyQ® MRSA

Intended Use: The NucliSENS Easy $Q ^ { \otimes }$ MRSA assay is a qualitative in vitro diagnostic test for the direct detection of methicillin-resistant Staphylococcus aureus (MRSA) from nasal swabs in patients at risk for nasal colonization. The NucliSENS EasyQ® MRSA assay is performed on the NucliSENS EasyQ® platform.

The test utilizes NASBA ™ (nucleic acid sequence-based amplification) coupled with molecular beacons (sequence-specific fluorescent probes) to detect the presence of MRSA DNA.

The NucliSENS EasyQ® MRSA assay is used as a screening tool to aid in the prevention and control of MRSA infections in health care institutions.

NucliSENS EasyQ® MRSA is not intended to diagnose, guide or monitor treatment for MRSA infections, or provide results of susceptibility to methicillin. A negative result does not preclude MRSA nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

Prescription Use X Over-The-Counter Use (Part 21 CFR 801 Subpart D) AND/OR (21 CFR 801 Subpart C)