# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

K102952

# B. Purpose for Submission:

This is a new 510k application for a real time polymerase chain reaction (PCR) assay used with the Cepheid SmartCycler II Real Time instrument with Dx Software version 1.7b, 3.0a, or 3.0b for the qualitative detection of human Adenovirus (HAdV) DNA isolated and purified from nasopharyngeal (NP) swab specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infection.

# C. Measurand:

Target DNA sequences for a conserved region of the hexon gene of human Adenovirus and for the transcript derived from a Universal Internal Control.

# D. Type of Test:

Real Time PCR-based, qualitative, in vitro diagnostic test for the detection of human Adenovirus DNA using NPS specimens. The test detects serotypes 1-51 of human Adenovirus. The isolation and purification of the nucleic acids is performed using either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG® System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux). Amplification and detection is carried out on the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b, 3.0a, or $3 . 0 \mathrm { b }$ , which generates signals based on the acquisition of spectrofluorometric data.

# E. Applicant:

Gen-Probe Prodesse Incorporated

F. Proprietary and Established Names: Prodesse® ProAdeno $^ \mathrm { T M _ { + } }$ Assay

# G. Regulatory Information:

1. Regulation section:   
21 CFR 866.3980

2. Classification: Class II

3. Product code: OCC

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The ProAdeno $\mathbf { T M } +$ Assay is a multiplex Real Time PCR in vitro diagnostic test for the qualitative detection of human Adenovirus (HAdV) DNA isolated and purified from nasopharyngeal (NP) swab specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infection. This test is intended for use to aid in the diagnosis of HAdV infections in humans in conjunction with other clinical and laboratory findings. The test detects, but does not differentiate, serotypes 1-51.

Negative results do not preclude HAdV infection and should not be used as the sole basis for treatment or other patient management decisions.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

To be used with the Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b, 3.0a, or 3.0b and either a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche) or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).

# I. Device Description:

The ProAdeno $\mathbf { T M } +$ Assay is a multiplex TaqMan based Real Time PCR test that

enables detection of human Adenovirus.

An overview of the procedure is as follows:

1. NP swab specimens are collected from symptomatic patients using a polyester, rayon or nylon tipped swab and placed into viral transport medium (not provided with the kit).   
2. Universal Internal Control (UIC) is added to every sample to monitor for inhibitors present in the specimens.   
3. Isolation and purification of nucleic acids are performed using a MagNA Pure LC Instrument (Roche) and the MagNA Pure Total Nucleic Acid Isolation Kit (Roche), or a NucliSENS® easyMAG™ System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux),   
4. Purified nucleic acids are added to the ProAdeno $^ +$ Supermix that contains target-specific oligonucleotide primers and probes and Taq DNA polymerase. The probes are dual-labeled with a reporter dye and a quencher dye (see table below).   
5. Amplification of DNA is performed in a Cepheid SmartCycler® II instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProAdeno $^ +$ Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This process generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing the fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real time instrument.

ProAdeno $^ +$ Assay Analyte Gene Targets and Probe Labels:   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>GeneTargeted</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=1>EmissionPeak</td><td rowspan=1 colspan=1>InstrumentChannel</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hexon</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>UniversalInternal Control</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>667 nm</td><td rowspan=1 colspan=1>Cy5</td></tr></table>

# Materials Provided:

ProAdeno+ Assay Kit (Cat. # H72VK00)   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity/Tube</td><td rowspan=1 colspan=1>CapColor</td><td rowspan=1 colspan=1>Cat. #</td><td rowspan=1 colspan=1>Reactions/ Tube</td></tr><tr><td rowspan=1 colspan=1>ProAdeno+ Supermix</td><td rowspan=1 colspan=1>Taq DNA polymeraseOligonucleotide primer pairsOligonucleotide probesBuffer containing dNTPs,MgCl2 and stabilizers</td><td rowspan=1 colspan=1>1030 μL</td><td rowspan=1 colspan=1>Brown</td><td rowspan=1 colspan=1>HSM72</td><td rowspan=1 colspan=1>50(2 tubesprovided)</td></tr><tr><td rowspan=1 colspan=1>RNase Inhibitor III</td><td rowspan=1 colspan=1>40U/μL</td><td rowspan=1 colspan=1>70 μL</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>GLS37</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Positive Control (PC) -Adenovirus DNAControl</td><td rowspan=1 colspan=1>•Non-infectious DNAplasmid containing a portionof the targeted gene (hexon)</td><td rowspan=1 colspan=1>500 μL</td><td rowspan=1 colspan=1>White</td><td rowspan=1 colspan=1>HCT72</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Universal InternalControl (UIC)</td><td rowspan=1 colspan=1>•Non-infectious double-stranded DNA (PCRproduct)Non-infectious in vitrotranscribed RNA</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Purple</td><td rowspan=1 colspan=1>GCT13</td><td rowspan=1 colspan=1>100</td></tr></table>

# Materials Required But Not Provided:

# Plasticware and consumables

• Polyester, rayon or nylon tipped nasopharyngeal swabs   
RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes   
• Sterile RNase/DNase-free filter or positive displacement micropipettor tips MagNA Pure LC System Disposables (Reagent Tubs, Reaction Tips, Tip Trays, Cartridges) or easyMAG System Disposables (Sample Strips and Tips) Biohit Pipette Tips for use with easyMAG System   
• Greiner Break Four uncoated plates for use with easyMAG System   
• Cepheid PCR reaction tubes, $2 5 ~ \mu \mathrm { L }$   
• Parafilm $^ { \textregistered } \mathbf { M }$ or MagNA Pure LC Cartridge Seals

# Reagents

• † Roche MagNA Pure LC Total Nucleic Acid Isolation Kit for 192 isolations or bioMérieux NucliSENS easyMAG reagents

• Micro Test M4 Viral Transport Medium, Micro Test M5 Viral Transport Medium, Micro Test M6 Viral Transport Medium, Micro Test M4RT

Viral Transport Medium, Copan Universal Transport Medium, or BD Universal Viral Transport vial, Molecular Grade Water (RNase/DNase Free) • Extraction Control (e.g. previously characterized positive sample)

† NOTE: Only qualified lots of the MagNA Pure LC Total Nucleic Acid Isolation Kit can be used with the ProAdeno $^ +$ Assay. Any lots not specifically qualified by Gen-Probe Prodesse, Inc. for use with the ProAdeno $^ +$ Assay are not verified for use with this assay, and may cause erroneous results.

# Equipment

• $- 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ Freezer   
• Roche MagNA Pure LC System with software version 3.0.11 or bioMérieux NucliSENS easyMAG System with Software version 1.0.1 or 2.0   
• Biohit multi-channel pipettor for use with easyMAG System   
• Cepheid SmartCycler® II Real Time Instrument with Dx Software version 1.7b, 3.0a, or 3.0b   
• Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ and $1 0 0 { \cdot } 1 0 0 0 \mu \mathrm { L } )$   
• Mini-centrifuge with adapter for Cepheid Reaction Tubes   
Cepheid cooling block   
• Wet ice or microcentrifuge tube cooling block   
• Biological safety cabinet

# Interpretation of Sample Results:

The SmartCycler® Dx software automatically determines the specimen results. The interpretation of the assay results for specimens is summarized in the table below:

<table><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>UICResult</td><td rowspan=1 colspan=1>Warning/ErrorCode</td><td rowspan=1 colspan=1>AdenoResult</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Adenovirus nucleic acid not detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Adenovirus nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Unresolved  PCR inhibition or reagent failure. Repeattesting from the purified nucleic acid, re-test fromoriginal sample, or collect and test a new sample.</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>ND2</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>30792</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined  error code 30792</td></tr><tr><td rowspan=1 colspan=1>SampleID</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>40983</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined - error code 40983</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): ID-Tag Respiratory Virus Panel (Luminex Molecular Diagnostics Inc.)

2. Predicate Numbers (s): K063765, K081483, K091667

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Features</td><td colspan="1" rowspan="1">ProAdeno+ Assay</td><td colspan="1" rowspan="1">Luminex RVP</td></tr><tr><td colspan="1" rowspan="1">510(k)</td><td colspan="1" rowspan="1">K102952</td><td colspan="1" rowspan="1">K063765, K081483, K091667</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.3980</td><td colspan="1" rowspan="1">866.3980</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">OCC, OEM, OEP</td></tr><tr><td colspan="1" rowspan="1">Classification Name</td><td colspan="1" rowspan="1">Respiratory viral panelmultiplex nucleic acid assay</td><td colspan="1" rowspan="1">Respiratory viral panel multiplex nucleicacid assay</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative detection of humanAdenovirus viral nucleic acids.</td><td colspan="1" rowspan="1">Direct and differential qualitative detectionof influenza types A and B, RSV types Aand B, Parainfluenza types 1, 2 and 3,Adenovirus and Rhinovirus viral nucleicacids.</td></tr><tr><td colspan="1" rowspan="1">Technology/Detection</td><td colspan="1" rowspan="1">Real Time PCRDetection: Instrument basedFluorogenic target-specifichybridization</td><td colspan="1" rowspan="1">RT-PCRDetection:Amplified products are coupled tomicrospheres and detected usingspectrofluorometric analysis.</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">NP swabs</td><td colspan="1" rowspan="1">NP swabs</td></tr><tr><td colspan="1" rowspan="1">Collection andTransport Medium</td><td colspan="1" rowspan="1">M4 and M5 Viral TransportMedium (Remel), UVT (BectonDickinson), UTM (Copan)</td><td colspan="1" rowspan="1">Universal Transport Medium (Copan)</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidIsolation</td><td colspan="1" rowspan="1">MagNA Pure LC System(Roche),NucliSENS easyMAG(bioMérieux)</td><td colspan="1" rowspan="1">NucliSENS® miniMAG extraction KitQIAamp® MiniElute® Virus Spin Kit</td></tr><tr><td colspan="1" rowspan="1">Instrument /AssayPlatform</td><td colspan="1" rowspan="1">Cepheid SmartCycler® IISystem</td><td colspan="1" rowspan="1">Luminex 100 or 200</td></tr><tr><td colspan="1" rowspan="1">Time to Result</td><td colspan="1" rowspan="1">3 to 5 hours</td><td colspan="1" rowspan="1">8 hours</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Adenovirus positive DNAtranscript control and an Internal</td><td colspan="1" rowspan="1">Bacteriophage lambda positive control andE. coli MS2 phage Internal Control </td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">DNA/RNA control provided</td><td colspan="1" rowspan="1">ancillary reagents not provided</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">NegativePositiveUnresolved</td><td colspan="1" rowspan="1">NegativePositiveNo Call</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

• Guidance for Industry and FDA Staff: Format for Traditional and Abbreviated 510(k)s, August 12, 2005 Draft Guidance for Industry and FDA Staff: Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens Guidance for Industry, FDA Reviewers and Compliance: Off-the-Shelf Software Use in Medical Devices, September 9, 1999 Guidance for Industry and FDA Staff: Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests, March 13, 2007 Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff: Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable, April 25, 2006 Guidance for Industry and FDA Staff: Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay, October 9, 2009 User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline Second Edition (CLSI EP12-A2 2008) Protocols for Determination of Limits of Detection and Limits of Quantitation (CLSI EP-17A 2004)   
Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline (CLSI MM3-A2 2006) Nucleic Acid Sequencing Methods in Diagnostic Laboratory Medicine; Approved Guideline (CLSI MM9-A 2004) Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods (CLSI MM13-P 2005)   
• Medical devices – Quality management systems – Requirements for regulatory purpose (ISO 13485:2003)   
• Medical devices - Application of risk management to medical devices (ISO 14971:2007)   
• Stability Testing of In Vitro Diagnostic Reagents (CEN 13640:2002)

# L. Test Principle:

The Real Time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. The ProAdeno $^ +$ Assay enables detection of Adenovirus DNA and the Universal Internal Control and is based on two processes: nucleic acid isolation and Real Time PCR amplification/detection. Human

respiratory specimens (nasopharyngeal swabs) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. Each sample of purified nucleic acid is added to the ProAdeno $^ +$ Supermix. The ProAdeno $^ +$ Supermix contains target-specific oligonucleotide primers and probes, and a Taq DNA polymerase. The primers are complementary to highly conserved regions of the hexon gene for HAdV. The probes are dual-labeled with a reporter dye and a quencher dye. Amplification of DNA is carried out in a Cepheid SmartCycler® II instrument. In this process, a probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This cleavage reaction generates an increase in fluorescent signal upon excitation from an LED light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time. Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

An Inter-Laboratory Reproducibility study was conducted at 3 laboratory sites. The Reproducibility Panel used in the study was prepared by spiking negative nasopharyngeal (NP) swab pools with cultured and titered stock suspensions of HAdV (serotypes HAdV-3 and HAdV-31). The Reproducibility Panel consisted of 12 simulated clinical samples that included the two Adenovirus serotypes at medium and low positive levels (near the assay limit of detection, $\ge 9 5 \%$ positive) and two high negative samples (high negative-1 at 0.1xLoD; high negative-2 at 0.001xLoD). Panels and controls were tested at each site by 2 operators for 5 days (12 samples and 3 controls $_ { \textrm { X 2 } }$ operators x 5 days x 3 sites $= 4 5 0$ ).

Each Panel included three replicates of each level of HAdV with at least one replicate of each serotype at each concentration. The Panels were stored at $\le - 7 0 ^ { \mathrm { { o } } } \mathrm { { C } }$ and delivered to the sites frozen. One Panel was used for each of the 5 testing days per operator. The samples were extracted each day using the laboratory’s standard extraction method (Roche MagNA Pure LC or bioMérieux NucliSENS easyMAG) and the extracted nucleic acids were tested with the ProAdeno $^ +$ Assay on the Cepheid SmartCycler II. ProAdeno $^ { + }$ Assay controls (Positive, Extraction and Negative) were also included with each panel run. Two lots of ProAdeno $^ +$ Supermix were used by each of the three sites. The overall percent agreement for the ProAdeno $^ +$ Assay was

$9 9 . 2 \%$ with the overall $\mathrm { C t }$ value $\% \mathrm { C t }$ across all sites for all samples and controls ranging from $1 . 1 \%$ to $6 . 1 \%$ depending upon analyte type, target type, and concentration tested . The summary data from the InterLaborato ry Reproducibility dy are su stu mmarized in the table below.

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Panel Member ID</td><td rowspan=1 colspan=1> nE-APVH</td><td rowspan=1 colspan=1> E-APVH</td><td rowspan=1 colspan=1>ood oE-APVH</td><td rowspan=1 colspan=1>e nE-APVH</td><td rowspan=1 colspan=1> hHE-APVH</td><td rowspan=1 colspan=1>etHE-APVH</td><td rowspan=1 colspan=1>ood oHE-APVH</td><td rowspan=1 colspan=1>eo pHE-APVH</td><td rowspan=1 colspan=1>RrtxConut</td><td rowspan=1 colspan=1>Aa SCo</td><td rowspan=1 colspan=1>Ne</td><td rowspan=2 colspan=1>Lo h % </td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>0.001XLoD</td><td rowspan=1 colspan=1>0.1XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.001XLoD</td><td rowspan=1 colspan=1>0.1XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>5/15b</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>10/15b</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>120/120d (100%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mean CT Value</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>40.4c</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>38.6c</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>36.7</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>2.8c</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.2c</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>3/16b</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>15/16</td><td rowspan=1 colspan=1>6/14 b</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>122/123d (99.2%)</td></tr><tr><td rowspan=2 colspan=1>S</td><td rowspan=1 colspan=1>Mean Cr Value</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>39.8c</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>39.6c</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>36.5</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3.5c</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>6.6c</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>2/15b</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>3/15b</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>118/120d (98.3%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Mean CT Value</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>40.1c</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>39.2c</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>36.5</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>5.3c</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>6.1c</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>1.1</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>10/46 b</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>43/46</td><td rowspan=1 colspan=1>19/44b</td><td rowspan=1 colspan=1>44/44</td><td rowspan=1 colspan=1>46/46</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>360/363d (99.2%)</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>92.0-100%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>92.3-100</td><td rowspan=1 colspan=1>92.0-100%</td><td rowspan=1 colspan=1>82.1-98.6%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>92.0-100%</td><td rowspan=1 colspan=1>92.3-100</td><td rowspan=1 colspan=1>88.8-1 00%</td><td rowspan=1 colspan=1>88.8-1 00%</td><td rowspan=1 colspan=1>88.8-100%</td><td rowspan=1 colspan=1>98.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CTValue</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>40.2c</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>39.0c</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>36.6</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Overall%V</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>3.1c</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>6.1c</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>3.3</td><td rowspan=1 colspan=1>4.5</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.1</td></tr></table>

a Mean $\mathrm { C _ { T } }$ calculated from Universal Internal Control b Number of positive samples c Average and $\% \mathrm { C V }$ based on number of positive samples d Does not include high negative-1 samples as those are at a concentration that is not reproducible e Performed study using the bioMerieux NucliSENS easyMAG f Performed study using the Roche MagNA Pure

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

The following controls are provided in the ProAdeno $^ +$ Assay kit:

Positive Control (PC): The ProAdeno $^ +$ Assay kit contains an Adenovirus positive DNA control that consists of a non-infectious plasmid containing a portion of the targeted gene (hexon). The PC does not go through nucleic acid isolation and purification, but is included during set-up of the PCR reaction. The PC in conjunction with the UIC is used to verify reagent and system performance.

Universal Internal Control (UIC): A Universal Internal Control, containing a non-infectious double-stranded DNA (PCR product) and a non-infectious RNA transcript, is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimens or reactions tube. The Universal Internal Control contains a DNA and an RNA portion to allow for one nucleic acid extract to be used with multiple RT-PCR/PCR assays whether the assay detects a DNA or an RNA target. The UIC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working.

Negative Control: A Negative Control is not provided with the kit, but is required and described in the ProAdeno $^ +$ Assay Instructions for Use. Viral transport media spiked with the UIC is to be used as the negative control and is to be processed starting from nucleic acid isolation. The Negative Control serves to monitor for contamination.

Extraction Control (EC): An Extraction Control (EC) is not provided with the kit, however, during the clinical trial, an inactivated Adenovirus was included with each extraction run as EC. The EC served to monitor for lysis during nucleic acid isolation. The sponsor is recommending an extraction control (with Ct between 13-40) in each nucleic acid extraction run to the end users in the Instructions for Use.

The user must interpret the Positive Control (PC) and the Extraction Control (if included) results to determine if the run is VALID and that results may be reported. The SmartCycler Dx software will automatically interpret the Negative Control result.

For a VALID Extraction run, the following conditions must be met:

<table><tr><td rowspan=1 colspan=1>Sample ID1</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>UICResult</td><td rowspan=1 colspan=1>WarningErrorCode</td><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>UIC Ct</td><td rowspan=1 colspan=1>AdenoResult</td><td rowspan=1 colspan=1>Adeno Ct</td></tr><tr><td rowspan=1 colspan=1>ExtractionControl</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>**</td><td rowspan=1 colspan=1>SPEC</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>13-40</td></tr><tr><td rowspan=1 colspan=1>Neg Cntrl</td><td rowspan=1 colspan=1>Valid²2</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NC1</td><td rowspan=1 colspan=1>20-41</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>0</td></tr></table>

For a VALID PCR run, the conditions in the table below must be met:

<table><tr><td rowspan=1 colspan=1>Sample ID1</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>UICResult</td><td rowspan=1 colspan=1>Warning /ErrorCode</td><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>UIC Ct</td><td rowspan=1 colspan=1>AdenoResult</td><td rowspan=1 colspan=1>Adeno Ct</td></tr><tr><td rowspan=1 colspan=1>Adeno PC</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>**</td><td rowspan=1 colspan=1>SPEC</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>20-41</td></tr><tr><td rowspan=1 colspan=1>Neg Control</td><td rowspan=1 colspan=1>Valid2</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NC1</td><td rowspan=1 colspan=1>20-41</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>0</td></tr></table>

1 Columns and data not used for interpretation are not included. 2 (Typical) An Invalid assay will display Error Code 4098. \*\* Error Code 3079: Warning/Error Code 3079 is periodically observed with Adenovirus positives (Adenovirus Positive Control, Adenovirus positive NP swab samples). Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a $C t$ value between 20-41 is reported in the “Adenovirus Ct” column for the Adenovirus Positive DNA Control, results can be recorded as positive and the run considered VALID.

# d. Detection limits:

The analytical sensitivity (Limit of Detection - LoD) of the ProAdeno $^ +$ Assay was determined for six serotypes of HAdV, one from each species (A through F). The Analytical Sensitivity Panel was prepared by spiking the viruses into negative nasopharyngeal (NP) pools prepared from leftover NP swab clinical samples. Each viral strain was previously cultured and titered to determine its stock concentration in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . A preliminary estimate of the LoD for each virus was carried out by testing five dilutions of each virus around the expected LoD with each dilution extracted five times using the Roche MagNA Pure LC System and the bioMérieux NucliSENS easyMAG for a total of 10 data points per virus level. The preliminary estimate of LoD was taken to be the lowest level of virus detected $\ge 9 5 \%$ of the time on both extraction instruments (10/10 replicates); in the case of HAdV-3, the preliminary estimate of LOD selected for confirmation had 9/10 positive replicates.

The preliminary LoD was then confirmed by preparing 40 additional replicates (20 per extraction instrument) at the proposed LoD. The LoD was confirmed if $\ge 9 5 \%$ of the samples at that level were positive. Each replicate was subject to the entire test system from sample preparation and extraction to PCR. If the preliminary LoD was not confirmed, then the LoD confirmation was repeated using the next higher (10-fold) level of the particular virus.

A Universal Internal Control (UIC) was spiked into all samples prior to nucleic acid isolation. The UIC monitored for PCR inhibition as well as reagent, procedural or instrumentation failures.

A Negative Control, which consisted of viral transport media spiked with UIC was included with each set of dilution series. Nucleic acid isolation of the Negative Control was performed along with the Analytical Sensitivity Panel samples. The Negative Control served to monitor for contamination during the testing procedure.

The Adenovirus DNA Positive Control was included with each PCR run to test for procedural errors (absence of reagent, instrument failure).

The tables below summarize the LoD values for the six viral serotypes. The LoD values reported are the lowest levels that yield $\ge 9 5 \%$ positivity with both methods of extraction.

<table><tr><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 1</td><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>100TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 4</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>10-1TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 19</td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>100TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 31</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>10-2TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 41</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>10-1.5TCID50/mL</td></tr></table>

# e. Analytical specificity:

# Cross reactivity

A panel of fifty-six (56) potentially cross reacting microorganisms consisting of 29 viral, 26 bacterial, and 1 yeast representing common respiratory pathogens or flora commonly present in the nasopharynx was evaluated for cross reactivity. Each potential cross reactant was individually spiked at a high level into a negative swab matrix. The level of each microorganism was determined by growing and titering the microorganism listed except for Chlamydia trachomatis for which the original titer provided by the supplier was used. Bacteria and yeast were tested at concentrations of $\mathrm { { \dot { 1 } 0 } ^ { 6 } }$ to $\mathrm { i } \mathrm { \dot { 0 } ^ { 7 } }$ CFU/mL; the exceptions were Chlamydia pneumoniae $( 1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) and Chlamydia trachomatis $( 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { \dot { m L } ) }$ . Viruses were tested at concentrations of $1 0 ^ { 3 }$ to $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ; the exception was Epstein Barr Virus ( $1 0 ^ { 8 }$ copies/mL).

The Analytical Specificity Panel samples and the Positive and Negative Controls were extracted on the bioMérieux NucliSENS easyMAG instrument and tested in triplicate on a Cepheid SmartCycler II. The ProAdeno $^ +$ Assay did not cross-react with any of the Analytical Specificity Panel samples tested. Detailed Analytical Specificity results are presented in the following table:

Analytical Specificity Results   

<table><tr><td colspan="1" rowspan="1">Strains</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Adenovirus(FAM)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 3</td><td colspan="1" rowspan="1">101 TCID50/mL</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B4</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Coxsackie B5/10/2006</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 2</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 3</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">108 copies/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacIntyre Strain</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus A2</td><td colspan="1" rowspan="1">104TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus 1A</td><td colspan="1" rowspan="1">103TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1</td><td colspan="1" rowspan="1">105 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Wisconsin</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles/7/2000</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Poliovirus 1</td><td colspan="1" rowspan="1">106 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">104 TCID50/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">103 TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">104TCID50/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Strains</td><td colspan="2" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Adenovirus(FAM)</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="2" rowspan="1">106 CFU/mL</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">CFU/mL</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">107 CFU/mL</td><td colspan="1" rowspan="1"></td></tr></table>

# Reactivity

Reactivity of the ProAdeno $^ +$ Assay was established for six serotypes of Adenovirus (HAdV-1, HAdV-3, HAdV-4, HAdV-19, HAdV-31, and HAdV41) during the LoD study. An additional forty-five serotypes of Adenovirus were also tested for reactivity with the ProAdeno $^ +$ Assay along with nine other strains with genomic changes that do not result in serological changes. One prototype strain (HAdV-52) was not included in this reactivity study due to a lack of availability.

A preliminary Limit of Detection (LoD) was established for each strain during pre-verification studies. In the reactivity study, the strains were tested at $3 \mathbf { x }$ the preliminary LoD. One sample was generated for each strain by spiking cultured and titered virus into aliquots of a negative nasopharyngeal (NP) swab pool. Samples were spiked with Universal Internal Control before extraction on the bioMérieux NucliSENS easyMAG. Each sample was tested in triplicate on the Cepheid SmartCycler II using one lot of ProAdeno+ reagents. A Negative Control, which consisted of UIC spiked into VTM, was included for each set of extraction runs. The Adenovirus DNA Control was included with each PCR run to test for procedural errors.

The data are presented in the following table:

ProAdeno+ Reactivity Panel Results   

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="2" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">HAdVDetectionMean Ct±StandardDeviation</td></tr><tr><td colspan="1" rowspan="1">HAdV-2</td><td colspan="2" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">32.9±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-3a2</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">35.5±0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-4a</td><td colspan="2" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">36.0±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-4p3</td><td colspan="2" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">36.4±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-5</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">35.6±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-6</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">38.1±1.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-7</td><td colspan="2" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">28.4±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-7</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">40.8*</td></tr><tr><td colspan="1" rowspan="1">HAdV-7a</td><td colspan="2" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">35.3±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-7d4</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">35.3±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-7h1</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">35.7±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-8</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">34.0±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-9</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">36.5±0.7</td></tr><tr><td colspan="1" rowspan="1">HAdV-10</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">34.5±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-11</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">33.0±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-12</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">34.3±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-13</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">34.5±0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-14</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">37.2±2.9</td></tr><tr><td colspan="1" rowspan="1">HAdV-14-4</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">33.9±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-15</td><td colspan="2" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">36.7±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-16</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">37.3±1.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-17</td><td colspan="2" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">33.8±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-18</td><td colspan="2" rowspan="1">3 x 102 TCID50/mL</td><td colspan="1" rowspan="1">34.8±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-20</td><td colspan="2" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">33.6±0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-21</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">35.8±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-21a</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">36.1±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-21c</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.9±1.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-22</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">35.6±0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-23</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">35.0±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-24</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10 TCID50/mL</td><td colspan="1" rowspan="1">35.3±0.8</td></tr><tr><td colspan="1" rowspan="1">HAdV-25</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 101 TCID50/mL</td><td colspan="1" rowspan="1">35.8±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-26</td><td colspan="1" rowspan="1">3 x 10-</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">35.6±0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-27</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">36.3±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-28</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">34.9±0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-29</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">34.2±0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-30</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">34.1±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-32</td><td colspan="1" rowspan="1">3 x 10</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">34.5±0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-33</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">35.4±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-34</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">33.5±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-35</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">34.6±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-36</td><td colspan="2" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">37.3±0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-37</td><td colspan="2" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">37.7±1.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-38</td><td colspan="2" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">33.2±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-39</td><td colspan="2" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">34.2±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-40</td><td colspan="2" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">33.0±0.2</td></tr><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">HAdVDetectionMean Ct±StandardDeviation</td></tr><tr><td colspan="1" rowspan="1">HAdV-42</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">34.5±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-43</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">34.1±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-44</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">33.9±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-45</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">35.3±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-46</td><td colspan="1" rowspan="1">3 x 10-2 TCID50/mL</td><td colspan="1" rowspan="1">38.0±0.7</td></tr><tr><td colspan="1" rowspan="1">HAdV-47</td><td colspan="1" rowspan="1">3 x 10-3 TCID50/mL</td><td colspan="1" rowspan="1">35.6±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-48</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">34.2±0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-49</td><td colspan="1" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">32.6±0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-50</td><td colspan="1" rowspan="1">3 x 10-1 TCID50/mL</td><td colspan="1" rowspan="1">35.6±0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-51</td><td colspan="1" rowspan="1">3 x 100 TCID50/mL</td><td colspan="1" rowspan="1">33.1±0.4</td></tr></table>

$^ { * } 2 / 3$ replicates were not detected at $\overline { { 3 \mathrm { ~ x ~ } 1 0 ^ { 0 } \mathrm { ~ T C I D } _ { 5 0 } / \mathrm { { m L } } } }$ . Repeat testing was performed at $3 \mathrm { ~ x ~ } 1 0 ^ { 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ with $3 / 3$ replicates detected.

# $f .$ Interference Studies:

# Interfering Substances

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential PCR inhibitors affected the performance of the ProAdeno $^ +$ assay. Blood, mucin, or medications (prescription and over-the-counter) for relief of congestion, sore throat, allergy and asthma symptoms were spiked into simulated human adenovirus (HAdV) positive NP swab matrix. A single cultured and titered strain of HAdV-3 was used and spiked into negative NP pools at 2 $\mathrm { \Delta x 1 0 ^ { 0 } T C I D _ { 5 0 } / m l }$ which was 2X Limit of Detection (LoD). High, clinically relevant, amounts of the potential inhibiting substances were added to spiked samples. One sample that was not spiked with a potential interfering substance was included to serve as a baseline for comparison. A Universal Internal Control (UIC) was added to samples. The inclusion of the UIC in each sample served to monitor for inhibition and aid in preventing reporting of false negative results. Nucleic acid from the samples was extracted with the bioMérieux easyMAG. PCR was performed in triplicate reactions for each sample on the Cepheid SmartCycler II using one lot of ProAdeno $^ +$ reagents. A Negative Control, which consisted of viral transport media spiked with UIC, was included with the Interfering Substances Panel samples. The Adenovirus DNA Control was included with each PCR run.

The following table shows the interfering substances used for this study. As described in the justification column, the concentrations spiked directly into samples ranged were generally $10 \%$ of recommended dose for the most of the substances, while others were tested at concentrations reported in scientific literature or in references from other IVD package inserts. The substances consisted of nasal sprays (liquid and powder), ingestible pills and lozenges, injectables, and endogenous substances (blood and mucin).

Interfering Substances Test Concentrations   

<table><tr><td rowspan=1 colspan=1>Substance Name</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Justification</td></tr><tr><td rowspan=1 colspan=1>Mucin (Bovine SubmaxillaryGland, type I-S)</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>EnodgenousSubstance</td><td rowspan=1 colspan=1>60μg/mL</td><td rowspan=1 colspan=1>1000x maximum levelpresent in serum</td></tr><tr><td rowspan=1 colspan=1>Blood (human), EDTAanticoagulated</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>EndogenousSubstance</td><td rowspan=1 colspan=1>2% (volume/volume)</td><td rowspan=1 colspan=1>Other Respiratory IVD&#x27;sPackag Insert</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine®</td><td rowspan=1 colspan=1>Phenylephrine HCl</td><td rowspan=1 colspan=1>Nasal Spray/Inhalant</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μL)</td></tr><tr><td rowspan=1 colspan=1>Walgreens Original AnefrinNasal Spray</td><td rowspan=1 colspan=1>OxymetazolineHydrochloride</td><td rowspan=1 colspan=1>Nasal Spray/Inhalant</td><td rowspan=1 colspan=1>15% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μL)8$</td></tr><tr><td rowspan=1 colspan=1>Zicam Homeopathic Non-Drowsy Allergy Relief NoDrip Liquid Nasal gel</td><td rowspan=1 colspan=1>Luffa Operculata,Galphimia Glauca,HistaminumHydrochloricum,Sulphur</td><td rowspan=1 colspan=1>Nasal Spray/Inhalant</td><td rowspan=1 colspan=1>5% (volume/volume)</td><td rowspan=1 colspan=1>10% of totalrecommended dose(15μL)</td></tr><tr><td rowspan=1 colspan=1>Walgreens Saline NasalSpray</td><td rowspan=1 colspan=1>Sodium chloride withpreservatives</td><td rowspan=1 colspan=1>Nasal Spray/Inhalant</td><td rowspan=1 colspan=1>15% (volume/volume) ofdose</td><td rowspan=1 colspan=1>10% of totalrecommended dose(45μL)</td></tr><tr><td rowspan=1 colspan=1>Beconase AQ®</td><td rowspan=1 colspan=1>Beclomethasonedipropionate</td><td rowspan=1 colspan=1>IngestiblePills/Lozenges</td><td rowspan=1 colspan=1>5% volume/volume</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Walgreens Sore ThroatLozenges, Cherry</td><td rowspan=1 colspan=1>Oralanesthetic/analgesic</td><td rowspan=1 colspan=1>IngestiblePills/Lozenges</td><td rowspan=1 colspan=1>0.62mg/ml; 1/20 drop,crushed; activeingredients: 1.0mg/mLbenzocaine, 1.7mg/mLmenthol</td><td rowspan=1 colspan=1>5% of total dose</td></tr><tr><td rowspan=1 colspan=1>Relenza®</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>Nasal Spray/Inhalant</td><td rowspan=1 colspan=1>5mg/mL</td><td rowspan=1 colspan=1>10% of total spray dose</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>4.0μg/mL</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Bactroban Nasal ointment</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>Other</td><td rowspan=1 colspan=1>10mg/mL</td><td rowspan=1 colspan=1>10% of totalrecommended dose inMupirocin ointment</td></tr><tr><td rowspan=1 colspan=1>TamiFlu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>IngestiblePills/Lozenges</td><td rowspan=1 colspan=1>7.5mg/mL</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr></table>

HAdV results were positive for all triplicate reactions for all potentially interfering substances. Interfering Substances Study results are presented in the following table.

Interfering Substances Test Results   

<table><tr><td colspan="1" rowspan="1">Interfering Substance</td><td colspan="1" rowspan="1">Mean HAdV Ct</td><td colspan="1" rowspan="1">Std Dev HAdV</td><td colspan="1" rowspan="1">Mean UIC Ct</td><td colspan="1" rowspan="1">Std Dev UIC</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Blood</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">Anefrin Nasal Spray</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Zicam</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Beconase AQ</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">Walgreens Sore Throat Lozenges</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">Relenza</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">Tobromycin</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">Bactroban Nasal Ointment</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">TamiFlu</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">No Interfering Substance</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.2</td></tr></table>

# Internal Control Interference

Competitive inhibition of the ProAdeno $^ +$ Assay due to the presence of the UIC was assessed. Simulated samples were tested with and without UIC to determine if the presence of the UIC inhibited the reaction. One cultured and titered strain of human adenovirus (HAdV-1) was used and was spiked into aliquots of a negative nasopharyngeal (NP) swab pool at LoD $\mathrm { ~ I ~ i ~ n ~ } ^ { 0 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) and at one log below the LoD $\mathrm { 1 \times 1 \hat { 0 } ^ { - 1 } T C I D _ { 5 0 } / m L } \mathrm { \Omega }$ ). Each concentration was prepared with the UIC and without any UIC added. Ten replicates of each concentration with and without UIC were extracted on the bioMérieux NucliSENS easyMAG instrument. The extracted nucleic acids were tested by PCR in a single replicate using one lot of ProAdeno $^ +$ Assay reagents on the Cepheid SmartCyler II. This procedure resulted in a total of 10 data points for each concentration spiked with UIC and 10 data points for each concentration without the UIC.

Competitive inhibition of the UIC was assessed by comparing the percent of samples detected with and without the UIC for HAdV at $\mathrm { 1 \times \bar { 1 0 } ^ { 0 } T C I D _ { 5 0 } / m L }$ and at $1 \mathrm { \Delta x 1 0 ^ { - 1 } T C I D _ { 5 0 } / m L }$ . Competitive inhibition was to be considered if the presence of the IC decreased the sensitivity of the ProAdeno $^ +$ Assay.

No competitive inhibition at the assay LoD was observed for detection of HAdV in the presence of UIC using the ProAdeno $^ +$ Assay.

# g. Assay cut-off:

The “cutoff value” represents the fluorescent intensity signal (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

Cutoff values (RFUs) were determined upon completion of the Cutoff Determination Study which used a training set of simulated samples. The training set of 56 samples contained:

High and low level HAdV-1 spiked NP swabs with UIC High and low level HAdV-3 spiked NP swabs with UIC • Negative NP swab samples with UIC

• Negative NP swab samples without UIC Two operators each ran the set of 56 samples.

In addition to the simulated samples described above, a panel consisting of Negative Controls (viral transport medium control spiked with UIC) and the HAdV Positive Control were run by the two operators.

The cutoff values resulting from the Cutoff Determination Study were then verified in the Cutoff Confirmation Study against a set of 42 HAdV positive NP swab samples, 36 negative NP swab samples and Controls.

The cutoff values used in the Cutoff Determination and Confirmation Studies are presented in the following table:

Suggested Cutoff Threshold Values for ProAdeno+   

<table><tr><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>EndPt Threshold (RFU)</td></tr><tr><td rowspan=1 colspan=1>PreliminaryCutoff Used*</td><td rowspan=1 colspan=1>Cutoff RangeIndicated byAnalysis</td><td rowspan=1 colspan=1>DeterminationCutoff ValueChosen</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>HAdV</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>15-113</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>UniversalInternal Control</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>18-37</td><td rowspan=1 colspan=1>35</td></tr></table>

\*Established during assay development

Threshold settings were determined and confirmed as FAM $( { \mathrm { H A d V } } ) = 4 0 $ and Cy5 $\mathrm { ( U I C ) } = 3 5$ . These settings produced $100 \%$ sensitivity and specificity for the verification set of samples used for Cutoff Confirmation and met acceptance criteria for this study. Therefore, the cutoff values were confirmed as acceptable for use in the clinical study and analytical studies.

# h. Carry-over/Cross-Contamination

This study assessed the level of carry-over/cross-contamination with the ProAdeno $^ +$ Assay by testing simulated human adenovirus (HAdV) High Positive samples run in series with HAdV Negative samples over 5 runs. The High Positive samples consisted of negative nasopharyngeal (NP) swab matrix spiked with a level 5 logs above the Limit of Detection (LoD) of a cultured and titered strain of HAdV-1. The samples were processed and extracted in a High Positive/Negative alternating fashion (i.e. checkerboard pattern) on the extraction instruments and likewise processed and run on the Cepheid SmartCycler II instrument in an alternating fashion.

Eleven High Positive and eleven Negative samples and a Negative Control were extracted per run on the easyMAG and on the MagNA Pure LC for a total of 110 High Positive samples and 110 Negative samples over 5 runs for each extraction instrument. When tested with high positive samples, no significant carry-over contamination effect was seen for the ProAdeno+ Assay. The data from the study are summarized in the table below.

Carryover Results   

<table><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>easyMAG</td><td rowspan=1 colspan=1>MagNA Pure LC</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>110/110(100%)</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>110/110(100%)</td></tr></table>

2. Comparison studies:

a. Method comparison with reference methods:

The performance of the ProAdeno $^ +$ Assay was compared with rapid culture (shell vial) followed by direct fluorescent antibody (DFA) screening and identification.

# b. Matrix Comparison:

Analytical performance of the ProAdeno $^ +$ Assay using six different viral transport media (VTM) was evaluated. Analytical sensitivity was determined using six serotypes of cultured and titered HAdV (one from each species) spiked into each of the six different VTM. After the initial testing, the samples were stored at $2 { - } 8 \mathrm { { ^ \circ C } }$ and tested again after 72 hours to demonstrate stability.

The six VTM tested were Remel M4, M4RT, M5, and M6, Copan Universal Transport Medium (UTM), and Becton Dickinson Universal Viral Transport (UVT). The intended use of these VTM is for the transport of clinical specimens containing viruses, chlamydiae, mycoplasma, and ureaplasma from the collection site to the laboratory for microbiological procedures. The exception is M4RT which is not recommended for the transport of mycoplasmas and ureaplasmas. The VTM have essentially the same components as shown in the table below.

Composition of Viral Transport Media   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Remel M4</td><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>Remel M6</td><td rowspan=1 colspan=1>Remel M4 RT</td><td rowspan=1 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>BD UTV</td></tr><tr><td rowspan=2 colspan=1>Components</td><td rowspan=2 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericinBCollistin</td><td rowspan=2 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminProteinStabilizersSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericin BCollistin</td><td rowspan=1 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericin B</td><td rowspan=1 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedGentamicinAmphotericin B</td><td rowspan=2 colspan=1>Hank&#x27;sBalanced saltsBovine SerumAlbuminGelatinSucroseL-GlutamicAcidHEPES BufferPhenol RedVancomycinAmphotericinBColistinL- Cysteine</td><td rowspan=2 colspan=1>Hank&#x27;sBalanced SaltsBovine SerumAlbuminGelatinSucroseL-GlutamicacidHEPES BufferPhenol RedVancomycinAmphotericinBCollistinL-Cysteine</td></tr><tr><td rowspan=1 colspan=1>Collistin</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>pH</td><td rowspan=1 colspan=1>7.3 +/- 0.2 @25°</td><td rowspan=1 colspan=1>7.3 +/- 0.2 @25°</td><td rowspan=1 colspan=1>7.3 +/ 0.2 @25°</td><td rowspan=1 colspan=1>7.3 +/ 0.2 @25°</td><td rowspan=1 colspan=1>7.3 +/ 0.2 @25°</td><td rowspan=1 colspan=1>7.3 +/- 0.2 @25°</td></tr><tr><td rowspan=1 colspan=1>StorageTemperature</td><td rowspan=1 colspan=1>Frozen at -25° or 2-8until use</td><td rowspan=1 colspan=1>Frozen at -25°or 2-8°C untiluse</td><td rowspan=1 colspan=1>2-30°C until use</td><td rowspan=1 colspan=1>2-30°C until use</td><td rowspan=1 colspan=1>2-25°C untiluse</td><td rowspan=1 colspan=1>2-25°C untiluse</td></tr><tr><td rowspan=1 colspan=1>Volume perVial</td><td rowspan=1 colspan=1>3mL</td><td rowspan=1 colspan=1>3mL</td><td rowspan=1 colspan=1>1.5 ml</td><td rowspan=1 colspan=1>3mL</td><td rowspan=1 colspan=1>3mL</td><td rowspan=1 colspan=1>3mL</td></tr></table>

Cultured and titered strains of HAdV-1, HAdV-3, HAdV-31, and HAdV-41 were spiked at one log above, at and one log below LoD; HAdV-4 was spiked at LoD, and one and two logs below LoD; HAdV-19 was spiked at one log above, at, and one and two logs below LoD into each VTM type. Testing at concentrations two logs below LoD was performed because higher concentrations resulted in $100 \%$ detection. Differences in Analytical Sensitivity between NP swab matrix (as used in the Analytical Sensitivity Study) and VTM alone (as used in the present study) are are not unusual and VTM alone typically results in a lower LoD. Following the initial testing time point, a second testing time point was performed after storage at $2 { - } 8 \mathrm { { ^ \circ C } }$ for 72 hours.

All VTM were compared to Remel M4 to determine equivalence because that was the VTM used in all other analytical verification studies, including the Analytical Sensitivity study, as well as by the majority of the Clinical Trial sites. For a VTM to be considered equivalent to M4:

i. All three replicates for at least one concentration were required to be positive and within one log of the lowest concentration in which all 3 replicates of M4 were positive.

ii. The average Ct values at all concentrations in which 3 of 3 replicates were positive should not differ by more than $3 . 3 \mathrm { C t }$ from the average $\mathrm { C t }$ of that concentration diluted in M4. If the sample differed by more than $3 . 3 \mathrm { C t }$ , the media are not considered equivalent.

Each of the VTM gave equivalent performance to Remel M4 as they all met the required criteria described above. Detailed data are presented in the following table:

Detection of HAdV in Six Different VTM   

<table><tr><td rowspan=1 colspan=20>Average HAdV CT and Standard Deviation (SD) at Initial Time point *</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>ConcTCID50/mL</td><td rowspan=1 colspan=3>M4</td><td rowspan=1 colspan=2>M4RT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>M5</td><td rowspan=1 colspan=3>M6</td><td rowspan=1 colspan=2>BD UVT</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Copan UTM</td></tr><tr><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>ACY</td><td rowspan=1 colspan=1>C SD</td><td rowspan=1 colspan=1>#Po</td><td rowspan=1 colspan=1>ACY</td><td rowspan=1 colspan=1>C SD</td><td rowspan=1 colspan=1>#P</td><td rowspan=1 colspan=1>A F</td><td rowspan=1 colspan=1>C SD</td><td rowspan=1 colspan=1>#P</td><td rowspan=1 colspan=1>AY</td><td rowspan=1 colspan=1>C SD</td><td rowspan=1 colspan=1>P$</td><td rowspan=1 colspan=1>ACYF</td><td rowspan=1 colspan=1>C SD</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>AY</td><td rowspan=1 colspan=1>CT SD</td></tr><tr><td rowspan=3 colspan=1>-APVH</td><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.7</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=1>APVH3</td><td rowspan=1 colspan=1>1x10</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>44.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=3 colspan=1>A4</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>1x10-2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>1x10-3</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>&quot;APVH=</td><td rowspan=1 colspan=1>1x101</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>1x100</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.9</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>1x10-^2</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>39.8</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.0</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=3 colspan=1>APVH3</td><td rowspan=1 colspan=1>1x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>1x10-2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>1x10-3</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=3 colspan=1>-APVHF</td><td rowspan=1 colspan=1>1x10-0.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.8</td></tr><tr><td rowspan=1 colspan=1>1x10-1.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>38.2</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>38.5</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>0.6</td></tr><tr><td rowspan=1 colspan=1>1x10-2.5</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1/3</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>NA</td></tr></table>

\*Average $\mathrm { C _ { T } }$ and SD based on the number of samples detected as indicated in the # Pos Column.

All six VTM evaluated are compatible with the ProAdeno $^ +$ Assay.

c. Freeze/Thaw and Stability:

# Reagents and Controls Stability

An Accelerated Stability study concluded that the ProAdeno $^ +$ Supermix, enzyme and controls (closed and open tubes) can be stored at $\leq - 7 0 ^ { \mathrm { { \circ } } } \mathrm { C }$ for up to 18 months. Real time stability studies are currently ongoing at Gen-Probe Prodesse.

A Freeze/Thaw Study demonstrated that the ProAdeno $^ +$ Supermix and enzyme can undergo up to 10 freeze-thaws. The Controls (Positive and Internal Control) can undergo up to 2 freeze-thaws. For the Controls, the

ProAdeno $^ +$ Assay Package Insert specifies that controls should not undergo more than 1 freeze-thaw cycle. Although internal studies demonstrated that up to 2 freeze-thaws of controls would not adversely affect performance. To mitigate risk it is recommended they not undergo more than 1 freeze-thaw.

# Comparison between Fresh and Frozen Specimens

An analytical study was carried out to determine the potential effect on the ProAdeno $^ +$ Assay of freezing NP samples and purified DNA. Contrived samples were generated using aliquots from a negative NP swab pool spiked individually with six human adenovirus (HAdV) serotypes (HAdV-1, HAdV3, HAdV-4, HAdV-19, HAdV-31, HAdV-41). A sufficient number of fresh HAdV positive NP swab samples were not available for this study due to low prevalence of the virus during the sample collection period. Contrived samples were made at varying concentrations in negative NP swab matrix such that approximately $60 \% 8 0 \%$ of the samples had analyte levels close to the cutoff. Of these samples, approximately half were close to the $\mathrm { C _ { 9 5 } }$ (LoD) and the other half were Moderate Positive $\left( 3 { - } 4 \mathrm { X L o D } \right)$ samples. The remaining positive samples covered the full detection range of the assay. Therefore, for each of the six serotypes, fifteen samples were generated $\mathbf { \bar { n } = }$ 90) in the following manner: 1 sample each at 3 logs, 2 logs, and 1 log above LoD, 6 samples at 3x LoD, and 6 samples at LoD. After sample generation, a portion of each sample was freshly extracted on the bioMérieux NucliSENS easyMAG. The remaining portions of the samples were frozen at $\mathrm { { \le - 7 0 ^ { \circ } C } }$ and later extracted as described above. The nucleic acids were run fresh (immediately subsequent to extraction) in a single replicate. The remaining nucleic acids were frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ and later thawed on ice and subjected to PCR as described above. The data was analyzed for the effect of freezing NP samples and purified nucleic acids on the ProAdeno $^ +$ Assay. The following tables summarize the results of the study:

Sample Stability Results — $\%$ Agreement to the Fresh NP / Fresh NA Data   

<table><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Concentration</td><td colspan="4" rowspan="1">% Agreement</td></tr><tr><td colspan="1" rowspan="1">FR NP/FR NA</td><td colspan="1" rowspan="1">FR NP/FZ NAb</td><td colspan="1" rowspan="1">FZ NP/FRNA</td><td colspan="1" rowspan="1">FZ NP/FZ NAd</td></tr><tr><td colspan="1" rowspan="6">I-APVH</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6= 100%</td></tr><tr><td colspan="1" rowspan="1">@ LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td></tr><tr><td colspan="1" rowspan="6">E-APVH</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td></tr><tr><td colspan="1" rowspan="1">@ LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15= 100%</td></tr><tr><td colspan="1" rowspan="2">&lt;</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="4">−</td><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6=100%</td></tr><tr><td colspan="1" rowspan="1">@LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 =100%</td><td colspan="1" rowspan="1">6/6=100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td></tr><tr><td colspan="1" rowspan="6">6H-APVH</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 =100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6=100%</td></tr><tr><td colspan="1" rowspan="1">@LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6= 100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td></tr><tr><td colspan="1" rowspan="6">HE-APVT</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6 = 100%</td></tr><tr><td colspan="1" rowspan="1">@ LoD</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6= 100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td></tr><tr><td colspan="1" rowspan="6">HA-APVH</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">2 logs above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">1 log above LoD</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td><td colspan="1" rowspan="1">1/1 = 100%</td></tr><tr><td colspan="1" rowspan="1">3x LoD</td><td colspan="1" rowspan="1">6/6 =100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6=100%</td><td colspan="1" rowspan="1">6/6 =100%</td></tr><tr><td colspan="1" rowspan="1">@ LoD</td><td colspan="1" rowspan="1">6/6= 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6 = 100%</td><td colspan="1" rowspan="1">6/6= 100%</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15 = 100%</td><td colspan="1" rowspan="1">15/15= 100%</td><td colspan="1" rowspan="1">15/15= 100%</td></tr></table>

a FR NP/FR NA: Fresh contrived NP samples/Freshly extracted nucleic acids b FR NP/FZ NA: Fresh contrived NP samples/Frozen nucleic acids c FZ NP/FR NA: Frozen contrived NP samples/Fresh nucleic acids d FZ NP/FZ NA: Frozen contrived NP samples/Frozen nucleic acids

Mean CT values and Mean Differences   

<table><tr><td colspan="1" rowspan="2">Sample ID</td><td colspan="1" rowspan="2">FR NP/FR NAaCt</td><td colspan="1" rowspan="2">FR NP/FZNAbCt</td><td colspan="1" rowspan="2">FZ NP/FRNACt</td><td colspan="1" rowspan="2">FZ NP/FZNACt</td><td colspan="3" rowspan="1">Mean Cr Difference to initialFR NP/FR NA Data</td></tr><tr><td colspan="1" rowspan="1">FR NP/FZ NA</td><td colspan="1" rowspan="1">FZ NP/FRNA</td><td colspan="1" rowspan="1">FZ NP/FZ NA</td></tr><tr><td colspan="1" rowspan="1">HAdV-1 1x LoD</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-1 3x LoD</td><td colspan="1" rowspan="1">34.2</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">-0.3</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">-0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-1 10x LoD</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">32.0</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">-0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-1 100x LoD</td><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">29.1</td><td colspan="1" rowspan="1">28.9</td><td colspan="1" rowspan="1">29.0</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-1 1000x LoD</td><td colspan="1" rowspan="1">25.7</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">25.5</td><td colspan="1" rowspan="1">-0.3</td><td colspan="1" rowspan="1">-0.3</td><td colspan="1" rowspan="1">-0.2</td></tr><tr><td colspan="1" rowspan="1">HAdV-3 1x LoD</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-3 3x LoD</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-3 10x LoD</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-3 100x LoD</td><td colspan="1" rowspan="1">29.7</td><td colspan="1" rowspan="1">29.5</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">-0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-3 1000x LoD</td><td colspan="1" rowspan="1">25.7</td><td colspan="1" rowspan="1">25.3</td><td colspan="1" rowspan="1">26.3</td><td colspan="1" rowspan="1">26.0</td><td colspan="1" rowspan="1">-0.4</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-4 1x LoD</td><td colspan="1" rowspan="1">35.9</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">-0.5</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">-0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-4 3x LoD</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">-0.5</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">-0.7</td></tr><tr><td colspan="1" rowspan="1">HAdV-4 10x LoD</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">32.0</td><td colspan="1" rowspan="1">-0.5</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">-0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-4100x LoD</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">28.6</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">28.8</td><td colspan="1" rowspan="1">-1.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">-0.8</td></tr><tr><td colspan="1" rowspan="1">HAdV-41000x LoD</td><td colspan="1" rowspan="1">25.6</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">24.6</td><td colspan="1" rowspan="1">24.7</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">-1.0</td><td colspan="1" rowspan="1">-0.9</td></tr><tr><td colspan="1" rowspan="1">HAdV-191x LoD</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-193x LoD</td><td colspan="1" rowspan="1">31.5</td><td colspan="1" rowspan="1">31.6</td><td colspan="1" rowspan="1">31.4</td><td colspan="1" rowspan="1">31.6</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-1910x LoD</td><td colspan="1" rowspan="1">29.8</td><td colspan="1" rowspan="1">30.0</td><td colspan="1" rowspan="1">29.3</td><td colspan="1" rowspan="1">29.3</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">-0.5</td><td colspan="1" rowspan="1">-0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-19100x LoD</td><td colspan="1" rowspan="1">26.3</td><td colspan="1" rowspan="1">26.2</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">25.8</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">-0.9</td><td colspan="1" rowspan="1">-0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-191000x LoD</td><td colspan="1" rowspan="1">21.0</td><td colspan="1" rowspan="1">21.1</td><td colspan="1" rowspan="1">21.6</td><td colspan="1" rowspan="1">21.5</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-311x LoD</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">34.4</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">-0.3</td><td colspan="1" rowspan="1">-0.9</td><td colspan="1" rowspan="1">-1.0</td></tr><tr><td colspan="1" rowspan="1">HAdV-31 3x LoD</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">-0.1</td></tr><tr><td colspan="1" rowspan="1">HAdV-3110x LoD</td><td colspan="1" rowspan="1">30.8</td><td colspan="1" rowspan="1">30.6</td><td colspan="1" rowspan="1">31.4</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">-0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1">HAdV-31100x LoD</td><td colspan="1" rowspan="1">22.9</td><td colspan="1" rowspan="1">22.8</td><td colspan="1" rowspan="1">26.4</td><td colspan="1" rowspan="1">26.5</td><td colspan="1" rowspan="1">-0.1</td><td colspan="1" rowspan="1">3.5</td><td colspan="1" rowspan="1">3.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-311000x LoD</td><td colspan="1" rowspan="1">23.1</td><td colspan="1" rowspan="1">23.2</td><td colspan="1" rowspan="1">23.5</td><td colspan="1" rowspan="1">23.5</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-411x LoD</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.4</td></tr><tr><td colspan="1" rowspan="1">HAdV-41 3x LoD</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">-0.3</td></tr><tr><td colspan="1" rowspan="1">HAdV-4110x LoD</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-41100x LoD</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">31.6</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">31.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">HAdV-411000x LoD</td><td colspan="1" rowspan="1">25.7</td><td colspan="1" rowspan="1">25.4</td><td colspan="1" rowspan="1">28.3</td><td colspan="1" rowspan="1">28.1</td><td colspan="1" rowspan="1">-0.3</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">2.4</td></tr></table>

Samples at 10x LoD, 100X LoD, and $1 0 0 0 \mathrm { x }$ LoD only had one replicate a FR NP/FR NA: Fresh contrived NP samples/Freshly extracted nucleic acids b FR NP/FZ NA: Fresh contrived NP samples/Frozen nucleic acids c FZ NP/FR NA: Frozen contrived NP samples/Fresh nucleic acids d FZ NP/FZ NA: Frozen contrived NP samples/Frozen nucleic acids

These results show that there is no negative affect due to freezing sample or nucleic acids when the sample and resultant nucleic acids have each been subjected to a freeze-thaw cycle.

# d. Extraction Efficiency:

Assessment of the equivalence of the two extraction methods was determined by several approaches: (1) Two of the clinical testing sites tested clinical samples using the MagNA Pure LC system while the other two clinical sites used the bioMérieux NucliSENS easyMAG instrument. (2) The Reproducibility study included both extraction systems. (3) Limits of Detection were determined on six serotypes and were within 1 log across both extraction instruments for each serotype.

Based on the results from these studies, there does not appear to be any significant difference in the performance of the ProAdeno $^ +$ Assay using either the MagNA Pure LC Total Nucleic Acid Isolation kit or the NucliSENS easyMAg system.

# 3. Clinical studies:

Clinical performance of the ProAdeno $^ +$ Assay was established at four external sites using prospectively collected fresh samples and prospectively collected archived samples. The prospective study with fresh samples was conducted at four U.S. clinical laboratories during October 2009 thru August 2010. Due to the overall low prevalence of human adenovirus (HAdV), fresh prospective samples were supplemented with prospectively collected archived samples to evaluate clinical performance. The prospectively collected archived samples were collected at one site from December 2009-February 2010.

The ProAdeno $^ +$ Assay was compared to culture followed by direct fluorescent antibody (DFA) staining to determine clinical sensitivity and clinical specificity. Two of the clinical sites used the MagNA Pure LC system while the other two clinical sites used the bioMérieux NucliSENS easyMAG instrument. A total of 1167 nasopharyngeal (NP) swab samples were tested with the ProAdeno $^ +$ Assay and by culture/DFA. One sample that initially gave unresolved results remained unresolved upon retesting with the ProAdeno $^ +$ Assay and was not included in the analysis.

All samples used for this study were remnants of NP swab specimens that were collected from symptomatic individuals for routine analysis and that otherwise would have been discarded. Individual specimens were delinked from all patient identifiers and given a study sample code. All clinical sites were granted waivers of informed consent by their IRBs for this study. Inclusion criteria included, but were not limited to: the specimen was from a symptomatic patient (for respiratory infection); the specimen was an NP swab in appropriate transport media; the specimen contained adequate volume for performing the ProAdeno $^ +$ Assay; age and gender information were available; specimen was stored properly; sample collection date and time were known; and initiation of the ProAdeno $^ +$ assay (nucleic acid extraction) took place within 72 hours of sample collection.

The clinical performance of the ProAdeno $^ +$ Assay is summarized in the following table.

<table><tr><td rowspan=2 colspan=1>ProAdeno+Results</td><td rowspan=1 colspan=3>Culture/DFA Results</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>49a</td><td rowspan=1 colspan=1>88</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1077</td><td rowspan=1 colspan=1>1078</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>1126</td><td rowspan=1 colspan=1>1166</td></tr><tr><td rowspan=1 colspan=4>Clinical Sensitivity: 97.5% (87.1%-94.3%)95% CI</td></tr><tr><td rowspan=1 colspan=4>Clinical Specificity: 95.6% (94.3%-96.7%)95% CI</td></tr></table>

$^ { \mathrm { a } } 3 5$ samples positive for HAdV by bi-directional sequence analysis, 14 samples negative for HAdV by bi-directional sequence analysis. $^ { \mathsf { b } } 1$ sample negative for HAdV by bi-directional sequence analysis

The general demographic data for all eligible prospective specimens $\mathrm { N } { = } 1 1 6 6 $ ) are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>613/1166 (52.6%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>553/1166 (47.4%)</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1>canmaGllow</td></tr><tr><td rowspan=1 colspan=1>&lt; 2 years</td><td rowspan=1 colspan=1>485/1166 (41.6%)</td></tr><tr><td rowspan=1 colspan=1>2-5 years</td><td rowspan=1 colspan=1>184/1166 (15.8%)</td></tr><tr><td rowspan=1 colspan=1>6-11 years</td><td rowspan=1 colspan=1>101/1166 (8.7%)</td></tr><tr><td rowspan=1 colspan=1>12-18 years</td><td rowspan=1 colspan=1>67/1166(5.7%)</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>240/1166(20.6%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>89/1166 (7.6%)</td></tr></table>

4. Clinical cut-off: N/A

# 5. Expected values/Reference range:

In the ProAdeno $^ +$ Assay clinical study, a total of 1166 eligible nasopharyngeal (NP) swab specimens were tested from four U.S. clinical laboratories across the United States during the study (October 2009-August 2010). The number and percentage of adenovirus positive cases determined by the ProAdeno $^ +$ Assay during this study, stratified by patient age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Prospective</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Total (N)</td><td rowspan=1 colspan=1>Total # Positive byProAdeno+ Assay</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>&lt; 2 years</td><td rowspan=1 colspan=1>485</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>12.0%</td></tr><tr><td rowspan=1 colspan=1>2-5 years</td><td rowspan=1 colspan=1>184</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>9.8%</td></tr><tr><td rowspan=1 colspan=1>6-11 years</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>6.9%</td></tr><tr><td rowspan=1 colspan=1>12-18 years</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.5%</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1166</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>7.5%</td></tr></table>

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b, 3.0a, or 3.0b

MagNA Pure LC Instrument (Roche)

# O. System Descriptions:

# 1. Modes of Operation:

The Cepheid Smart Cycler II Real Time instrument with Dx software version 1.7b, 3.0a, or 3.0b is used to perform polymerase chain reaction (PCR) amplification and detection of nucleic acid. Other nucleic acid amplification tests that use the Smart Cycler II instrument have previously received 510(k) clearance: these tests include Prodesse’s ProFAST $^ +$ Assay (K101855), ProParaflu $^ +$ Assay (K091053), ProGastro Cd Assay (K090239), ProhMPV $^ +$ Assay (K082688) and others. The Cepheid SmartCycler instrument is an integrated nucleic acid amplification and detection instrument system based on Cepheid’s proprietary microprocessor-controlled I-CORE module. For purified DNA samples, the SmartCycler instrument enables PCR for the amplification of DNA and hybridization of fluorogenic target-specific probes for the detection of the amplified cDNA.

The Roche MagNA Pure LC is an automated nucleic acid isolation and purification system based upon binding of nucleic acids to glass particles and has the capability to process a total of 32 reactions within one run. Nucleic acid is purified in multiple plastic reaction tips and cartridges by several steps that include cell lysis and binding of nucleic acid to magnetic glass particles, wash steps, and a heated elution to unbind the nucleic acid from the glass particles.

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the same silica extraction technology as the MagNA Pure. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User manually enters Patient ID/Sample ID.

4. Specimen Sampling and Handling:

Not applicable

5. Calibration:

Not applicable

6. Quality Control:

Positive Control (PC): The ProAdeno $^ +$ Assay kit contains an Adenovirus positive DNA control that consists of a non-infectious plasmid containing a portion of the targeted gene (hexon). The PC does not go through nucleic acid isolation and purification, but is included during set-up of the PCR reaction. The PC in conjunction with the UIC is used to verify reagent and system performance.

Universal Internal Control (UIC): The UIC contains an Internal DNA Control (generated by PCR) and is recognized by the UIC primers and probe present in the ProAdeno $^ +$ Assay. The UIC is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The UIC serves also as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working.

Negative Control (NC): A Negative Control is not provided with the kit, but is required and described in the ProAdeno $^ +$ Assay Instructions for Use. Viral transport medium spiked with the UIC is to be used as the Negative Control and is processed starting from nucleic acid isolation. The Negative Control serves to monitor for contamination.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.