# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K102342   
B. Purpose for Submission: To obtain a 510(k) SE determination for the KeyPathTM MRSA/MSSA Blood Culture Test – BT.   
C. Measurand: Bacteriophage amplification specific to S. aureus and assesses the phenotypic response of the target organism to cefoxitin as an analog to methicillin.   
D. Type of Test: Qualitative lateral flow identification and AST test using bacteriophage amplification growth based detection   
E. Applicant: MicroPhage, Inc.   
F. Proprietary and Established Names: KeyPathTM MRSA/MSSA Blood Culture Test – BT

G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>RegulationSection</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OUS</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>866.2050 -Staphylococcaltypingbacteriophage</td><td rowspan=1 colspan=1>Microbiology (83)</td></tr></table>

# H. Intended Use:

1. Intended use(s):

The KeyPathTM MRSA/MSSA Blood Culture Test – BT is a qualitative in vitro diagnostic test for the timely identification of Staphylococcus aureus (S. aureus) and determination of methicillin susceptibility (MSSA) or methicillin resistance (MRSA) directly from positive blood cultures.

The Test uses bacteriophage amplification to identify the presence of S. aureus and assess the phenotypic response of the target organism to cefoxitin, an indicator of oxacillin (a methicillin analog) resistance.

The assay is performed directly on positive blood culture specimens that are determined as Gram Positive Cocci in singles (GPC) or as Gram Positive Cocci in Clusters (GPCC) by Gram stain.

The KeyPathTM MRSA/MSSA Blood Culture Test – BT is performed directly on positive blood culture specimens from BD BACTECTM blood culture bottles (Plus Aerobic/F and Plus Anaerobic/F).

The Test is indicated for use in conjunction with other laboratory and clinical data available to the physician as an aid in the detection of MRSA/MSSA from positive blood cultures.

Subculturing of positive blood cultures is necessary for additional susceptibility test determinations, differentiation of mixed growth and for epidemiological typing.

# 2. Indication(s) for use:

The KeyPathTM MRSA/MSSA Blood Culture Test – BT is a qualitative in vitro diagnostic test for the timely identification of Staphylococcus aureus (S. aureus) and determination of methicillin susceptibility (MSSA) or methicillin resistance (MRSA) directly from positive blood cultures.

The Test uses bacteriophage amplification to identify the presence of $S .$ . aureus and assess the phenotypic response of the target organism to cefoxitin, an indicator of oxacillin (a methicillin analog) resistance.

The assay is performed directly on positive blood culture specimens that are determined as Gram Positive Cocci in singles (GPC) or as Gram Positive Cocci in Clusters (GPCC) by Gram stain.

The KeyPathTM MRSA/MSSA Blood Culture Test – BT is performed directly on positive blood culture specimens from BD BACTECTM blood culture bottles (Plus Aerobic/F and Plus Anaerobic/F).

The Test is indicated for use in conjunction with other laboratory and clinical data available to the physician as an aid in the detection of MRSA/MSSA from positive blood cultures.

Subculturing of positive blood cultures is necessary for additional susceptibility test determinations, differentiation of mixed growth and for epidemiological typing.

3. Special conditions for use statement(s): For prescription use

4. Special instrument requirements: Manual readings only

# I. Device Description:

The KeyPathTM MRSA/MSSA Blood Culture Test – BT test uses lytic bacteriophage, specific for Staphylococcus aureus (S. aureus, SA), as an amplification technology for detection of S. aureus and determination of methicillin resistance or susceptibility in positive blood cultures. To detect S. aureus (ID Reaction Tube), the bacteriophage infect the S. aureus (if present), replicate within the host (culminating in bacterial lysis) and over the incubation period, produce several cycles of bacteriophage amplification. In a separate Reaction Tube (RS), the Test uses cefoxitin (an oxacillin and methicillin analog) which inhibits bacteriophage amplification for susceptible organisms (MSSA) and fails to inhibit bacteriophage amplification when the organism is resistant to methicillin (MRSA). The Test then uses a selfperforming immunoassay (Detector) to detect the increase in concentration of bacteriophage using antibodies specific to the Test bacteriophage, and calibrated such that at above a threshold concentration, it produces a visible signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s): K071026 BD GeneOhm™ StaphSR Assay K851949 Wellcome (Remel) Staphaurex $^ \mathrm { \textregistered }$ ZL30 K011710 Oxoid PBP2’ Latex Agglutination (Preamendment) BBL (BD) cefoxitin $3 0 \mu \mathrm { g }$ Sensi-Disc (Preamendment) BBL (BD) oxacillin $1 ~ { \mu \mathrm { g } }$ Sensi-Disc (Preamendment) Coagulase Test (multiple manufacturers) (Preamendment) Catalase Test (multiple manufacturers)

2. Predicate 510(k) number(s): K071026 BD GeneOhm™ StaphSR Assay K851949 Wellcome (Remel) Staphaurex $^ \mathrm { \textregistered }$ ZL30 K011710 Oxoid PBP2’ Latex Agglutination

3. Comparison with predicate:

<table><tr><td colspan="3">SIMILARITIES</td></tr><tr><td>Items</td><td>KeyPathTM MRSA/MSSA Blood Culture Test - BT</td><td>BD GeneOhmTM StaphSR Assay</td></tr><tr><td rowspan="3">Intended Use -</td><td>The KeyPathTM</td><td>The BD GeneOhmT</td></tr><tr><td>MRSA/MSSA Blood Culture Test - BT is a</td><td>StaphSR Assay is a qualitative in vitro</td></tr><tr><td>qualitative in vitro diagnostic test for the timely</td><td>diagnostic test for the rapid detection of</td></tr><tr><td>identification of cultures. resistance. by Gram stain. The KeyPathTM blood cultures.</td><td>Staphylococcus aureus Staphylococcus aureus (S. (SA) and methicillin- aureus) and determination resistant Staphylococcus of methicillin susceptibility aureus (MRSA) directly (MSSA) or methicillin from positive blood culture. resistance (MRSA) directly The assay utilizes from positive blood polymerase chain reaction (PCR) for the amplification The Test uses bacteriophage of specific targets and fluorogenic target-specific amplification to identify the hybridization probes for the presence of S. aureus and real-time detection of the assess the phenotypic amplified DNA. The assay response of the target is performed on gram organism to cefoxitin, an indicator of oxacillin (a positive cocci, identified by Gram stain, from positive methicillin analog) blood cultures. The BD GeneOhmTM StaphSR The assay is performed Assay is not intended to directly on positive blood monitor treatment for culture specimens that are MRSA/SA infections. determined as Gram Subculturing of positive Positive Cocci in singles blood cultures is necessary (GPC) or as Gram Positive for further susceptibility Cocci in Clusters (GPCC) testing. MRSA/MSSA Blood Culture Test - BT is performed directly on positive blood culture specimens from BD BACTECTM blood culture bottles (Plus Aerobic/F and Plus Anaerobic/F). The Test is indicated for use in conjunction with other laboratory and clinical data available to the physician as an aid in the detection of MRSA/MSSA from positive Subculturing of positive blood cultures is necessary for additional susceptibility</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>test determinations,differentiation of mixedgrowth and forepidemiological typing.</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Single Use</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Indications for Use</td><td rowspan=1 colspan=1>Professional Use</td><td rowspan=1 colspan=1>Professional Use</td></tr><tr><td rowspan=1 colspan=1>Interpretation ofresults</td><td rowspan=1 colspan=1>Visual</td><td rowspan=1 colspan=1>Visual</td></tr><tr><td rowspan=1 colspan=1>Patient population</td><td rowspan=1 colspan=1>Clinical patients</td><td rowspan=1 colspan=1>Clinical patients</td></tr><tr><td rowspan=1 colspan=1>Specimen type</td><td rowspan=1 colspan=1>Positive blood culture</td><td rowspan=1 colspan=1>Positive blood culture</td></tr><tr><td rowspan=1 colspan=1>Assay controls</td><td rowspan=1 colspan=1>Pos Control 1: MRSAPos Control 2: MSSANeg Control: NSA</td><td rowspan=1 colspan=1>Pos Control: MRSAPos Control: SANeg Control: NSA</td></tr></table>

<table><tr><td rowspan=1 colspan=3>DISSIMILARITIES</td></tr><tr><td rowspan=1 colspan=1>Items</td><td rowspan=1 colspan=1>KeyPath™ MRSA/MSSABlood Culture Test - BT</td><td rowspan=1 colspan=1>BD GeneOhmTMStaphSR Assay</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>5 hours</td><td rowspan=1 colspan=1>60-75 minutes</td></tr><tr><td rowspan=1 colspan=1>Mode of action</td><td rowspan=1 colspan=1>The test uses bacteriophageamplification with cefoxitinto rapidly determine thepresence of MRSA in SApopulations.</td><td rowspan=1 colspan=1>The assay utilizespolymerase chain reaction(PCR) for the amplificationof specific targets andfluorogenic target specifichybridization probes for thereal-time detection of theamplified DNA.</td></tr><tr><td rowspan=1 colspan=1>Assay format</td><td rowspan=1 colspan=1>Amplification:bacteriophageamplificationDetection: Lateral flowimmunoassay with colloidalgold particles withmonoclonal antibodiesspecific to assaybacteriophage.</td><td rowspan=1 colspan=1>Amplification: polymerasechain reaction (PCR)Detection: Fluorogenictarget-specific hybridizationprobes of the amplifiedDNA.</td></tr></table>

<table><tr><td colspan="3" rowspan="1">SIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPathTM MRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">Coagulase Tube, CatalaseSlide Tests(pre-amendment)</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">See above</td><td colspan="1" rowspan="1">The Coagulase Tube andCatalase Slide Tests(multiple manufacturers)are qualitative in vitrodiagnostic tests for the</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">from isolated colonies froma positive blood culture."Positive" results for bothcatalase and coagulase areindicative of SA. Theassays contain rabbit serumwithEDTA for Coagulase Testand peroxide (H2O2) forCatalase Test substrateswhich will react withcoagulase and catalaseenzymes expressed by SA.A clumping of rabbit serumconfirms the presence ofcoagulase and productionof O2 bubbles confirms thepresence of catalase (i.e.both results indicatingpresence of SA). The assayis performed on grampositive cocci which havebeen isolated by streakingon culture plates, identifiedby Gram stain, frompositive blood cultures.Further subculturing ofpositive blood cultures arenecessary for susceptibilitytesting.</td></tr><tr><td colspan="1" rowspan="1">Single Use</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">Professional Use</td><td colspan="1" rowspan="1">Professional Use</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofresults</td><td colspan="1" rowspan="1">Visual</td><td colspan="1" rowspan="1">Visual</td></tr><tr><td colspan="1" rowspan="1">Patient population</td><td colspan="1" rowspan="1">Clinical patients</td><td colspan="1" rowspan="1">Clinical patients</td></tr><tr><td colspan="1" rowspan="1">Specimen type</td><td colspan="1" rowspan="1">Positive blood culture</td><td colspan="1" rowspan="1">Positive blood culture</td></tr><tr><td colspan="1" rowspan="1">Assay controls</td><td colspan="1" rowspan="1">Pos Control 1: MRSAPos Control 2: MSSANeg Control: NSA</td><td colspan="1" rowspan="1"></td></tr></table>

<table><tr><td colspan="3" rowspan="1">DISSIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPathTMMRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">Coagulase Tube, CatalaseSlide Tests(pre-amendment)</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Positive blood culture</td><td colspan="1" rowspan="1">Overnight purified plate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">culture (i.e. isolatedcolonies) originating from apositive blood culture.</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">5 hours</td><td colspan="1" rowspan="1">Catalase: 5 minutesCoagulase: 4-24 hours</td></tr><tr><td colspan="1" rowspan="1">Mode of action</td><td colspan="1" rowspan="1">The test uses bacteriophageamplification to rapidlyidentify the presence of SA.</td><td colspan="1" rowspan="1">The assays contain rabbitserum with EDTA forCoagulase Test andperoxide (H2O2) forCatalase Test substrateswhich will react withcoagulase and catalaseenzymes expressed by SA.A clumping of rabbit serumconfirms the presence ofcoagulase and productionof O2 bubbles confirms thepresence of catalase (i.e.both results indicatingpresence of SA.)</td></tr><tr><td colspan="1" rowspan="1">Assay format</td><td colspan="1" rowspan="1">Amplification:bacteriophageamplificationDetection: Lateral flowimmunoassay with colloidalgold particles withmonoclonal antibodiesspecific to assaybacteriophage.</td><td colspan="1" rowspan="1">Amplification: noneDetection: Clumping ofcoagulase substrate and O2gas production (i.e.bubbles) of catalasesubstrate when metabolizedby the respective enzymes.</td></tr></table>

<table><tr><td colspan="3" rowspan="1">SIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPathT MRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">Oxoid PBP2' LatexAgglutination Test</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">See above</td><td colspan="1" rowspan="1">This test is a rapid latexagglutination assay,detecting PBP2' (alsocalled PBP2a), in isolatesof Staphylococcus, as anaid in identifying MRSAand methicillin-resistantcoagulase-negativestaphylococci.</td></tr><tr><td colspan="1" rowspan="1">Single Use</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">Professional Use</td><td colspan="1" rowspan="1">Professional Use</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofresults</td><td colspan="1" rowspan="1">Visual</td><td colspan="1" rowspan="1">Visual</td></tr><tr><td>Patient population</td><td>Clinical patients</td><td>Clinical patients</td></tr><tr><td rowspan="2">Assay controls</td><td>Pos Control 1: MRSA</td><td>Pos Control: MRSA</td></tr><tr><td>Pos Control 2: MSSA Neg Control: NSA</td><td>Neg Control: NSA</td></tr></table>

<table><tr><td colspan="3" rowspan="1">DISSIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPath TM MRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">Oxoid PBP2' LatexAgglutination Test</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Positive blood culture</td><td colspan="1" rowspan="1">Overnight Purified Culture(16 24 hrs)</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">5 hours</td><td colspan="1" rowspan="1">45 minutes</td></tr><tr><td colspan="1" rowspan="1">Mode of action</td><td colspan="1" rowspan="1">The test uses bacteriophageamplification to rapidlyidentify the presence ofMRSA in S. aureuspopulations.</td><td colspan="1" rowspan="1">Latex particles sensitizedwith a monoclonal antibodyagainst PBP2' willspecifically react withmethicillin-resistantstaphylococci to causeagglutination visible to theunaided eye.</td></tr><tr><td colspan="1" rowspan="1">Assay format</td><td colspan="1" rowspan="1">Amplification:bacteriophageamplificationDetection: Lateral flowimmunoassay with colloidalgold particles withmonoclonal antibodiesspecific to assaybacteriophage.</td><td colspan="1" rowspan="1">Amplification: noneDetection: Agglutination oflatex particles.</td></tr><tr><td colspan="1" rowspan="1">cYanmaGelwb</td><td colspan="2" rowspan="1">SIMILARITIES</td><td colspan="1" rowspan="1">cYanmaGaYellowb</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPathT MRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">BD BBLcefoxitin 30ugSensi-disc</td><td colspan="1" rowspan="1">BD BBLcefoxitin1 ug Sensi-disc</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">See above</td><td colspan="1" rowspan="1">These discs areused for semi-quantitativein vitrosusceptibilitytesting by theagar discdiffusion testprocedure ofcommon,rapidly growingand certainfastidiousbacterialpathogens.</td><td colspan="1" rowspan="1">These discs areusedfor semi-quantitativein vitrosusceptibilitytesting by theagar discdiffusion testprocedure ofcommon,rapidly growingand certainfastidiousbacterialpathogens.</td></tr><tr><td colspan="1" rowspan="1">Single Use</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Indications forUse</td><td colspan="1" rowspan="1">Professional Use</td><td colspan="1" rowspan="1">ProfessionalUse</td><td colspan="1" rowspan="1">ProfessionalUse</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofresults</td><td colspan="1" rowspan="1">Visual</td><td colspan="1" rowspan="1">Visual</td><td colspan="1" rowspan="1">Visual</td></tr><tr><td colspan="1" rowspan="1">Patientpopulation</td><td colspan="1" rowspan="1">Clinical patients</td><td colspan="1" rowspan="1">Clinical patients</td><td colspan="1" rowspan="1">Clinical patients</td></tr><tr><td colspan="1" rowspan="1">Assay controls</td><td colspan="1" rowspan="1">Pos Control 1: MRSAPos Control 2: MSSANeg Control: NSA</td><td colspan="1" rowspan="1">Pos Control 1:MRSANeg Control:MSSA</td><td colspan="1" rowspan="1">Pos Control:MRSANeg Control:MSSA</td></tr></table>

<table><tr><td colspan="4" rowspan="1">DISSIMILARITIES</td></tr><tr><td colspan="1" rowspan="1">Items</td><td colspan="1" rowspan="1">KeyPathTMRSA/MSSABlood Culture Test - BT</td><td colspan="1" rowspan="1">BD BBLcefoxitin 30ugSensi-disc</td><td colspan="1" rowspan="1">BD BBLcefoxitin1 ug Sensi-disc</td></tr><tr><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">Positive blood culture</td><td colspan="1" rowspan="1">Overnightculture</td><td colspan="1" rowspan="1">Overnightculture</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">5 hours</td><td colspan="1" rowspan="1">18-24 hours</td><td colspan="1" rowspan="1">18-24 hours</td></tr><tr><td colspan="1" rowspan="1">Mode of action</td><td colspan="1" rowspan="1">The test uses bacteriophageamplification to rapidlyidentify the presence ofMRSA in S. aureuspopulations.</td><td colspan="1" rowspan="1">Diffusion ofantibiotic intolawn of SA.</td><td colspan="1" rowspan="1">Diffusion ofantibiotic intolawn of SA.</td></tr><tr><td>Assay format</td><td>Amplification: bacteriophage amplification Detection: Lateral flow immunoassay with colloidal gold particles with monoclonal antibodies specific to assay bacteriophage.</td><td>Amplification: none Detection: Visual interpretation of zone of inhibition.</td><td>Amplification: none Detection: Visual interpretation of zone of inhibition.</td></tr></table>

# Summary of Gold Standard and Predicate Use:

After Gram stain, samples were tested with: a) the KeyPathTM MRSA/MSSA Blood Culture Test – BT per MP2009-B and b) a cohort of Gold Standard and predicate tests for identification including: coagulase tube test, catalase slide test, and Staphaurex $^ \mathrm { \textregistered }$ Test. The identification Gold Standard is defined as concordant results between the catalase, coagulase and Staphaurex $^ \mathrm { \textregistered }$ tests.

For SA positive samples, PBP2’ test, Cefoxitin Sensi-disc $( 3 0 { \mathrm { u g } } )$ test and Oxacillin Sensi-disc (1 ug) test were performed to determine antibiotic susceptibility. The Cefoxitin Sensi-disc test was defined as the Gold Standard for MRSA/MSSA determination on S. aureus-positive samples.

Except for MRSA/MSSA Blood Culture Test – BT, all other Gold Standard/predicates used purified colonies from overnight culture of positive blood culture samples on $5 \%$ sheep blood-tryptic soy agar (TSA) plates. Sensi-disc testing (i.e. Kirby-Bauer disc diffusion) included secondary plating on Mueller-Hinton plates.

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: Antimicrobial Susceptibility Test (AST) Systems

CLSI M100-S19, Performance Standards for Antimicrobial Susceptibility Testing; Twentieth Informational Supplement CLSI EP12-A2, User Protocol for Evaluation of Qualitative Test Performance, 9/9/2008

CLSI M100-S20 Performance Standards for Antimicrobial Susceptibility Testing; Twentieth Informational Supplement, M100-S20, 2010

CLSI EP25-A, Evaluation of Stability of In Vitro Diagnostic Reagents, Approved Guideline, Sept. 2009

CLSI M23-A2, Development of In Vitro Susceptibility Testing Criteria and Quality Control Parameters, 09/08/2009

# CLSI EP7-A2, Interference Testing in Clinical Chemistry; Approved Guideline-Second Edition, November 2005

# L. Test Principle:

The KeyPathTM MRSA/MSSA Blood Culture Test – BT uses lytic bacteriophage, specific for Staphylococcus aureus, as an amplification technology for detection of S. aureus and determination of methicillin resistance or susceptibility in positive blood cultures. To detect S. aureus (ID Reaction Tube), the bacteriophage infect the S. aureus (if present), replicate within the host (culminating in bacterial lysis) and over the incubation period, produce several cycles of bacteriophage amplification. In a separate Reaction Tube (RS), the Test uses cefoxitin (methicillin analog) which inhibits bacteriophage amplification for susceptible organisms (MSSA) and fails to inhibit bacteriophage amplification when the organism is resistant to methicillin (MRSA). The Test then uses a self-performing immunoassay (Detector) to detect the increase in concentration of bacteriophage using antibodies specific to the Test bacteriophage, and calibrated such that at a known concentration, it will produce a visible signal. The test formulation consists of 4 components: Identification (ID) liquid reagents (Reaction Media), ID dried reagents (Reaction Tubes), Resistance/Susceptibility (RS) liquid reagents (Reaction Media) and RS dried reagents (Reaction Tubes). The dried reagents are formulated specifically for use in the BactecTM (BectonDickinson) blood culture systems with PLUS Aerobic and PLUS Anaerobic bottles. The Test is also composed of a dual lane self-performing lateral flow immunoassay (LFI Detector). The Test includes two $1 5 0 \mu \mathrm { L }$ transfer pipettes that facilitate transfer of the incubated sample from each of the Reaction Tubes to each of the corresponding Detector wells. The pipettes are color coded (blue and red) to ensure proper transfer.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

In the analytical in-house performance testing, all protocols used a single technician to set up the Test but this technician does not interpret the results. Multiple, independent, blinded, readers interpret the results.

# a. Precision/Reproducibility:

Of 240 test points, $9 6 . 3 \%$ (231/240) of samples the KeyPathTM MRSA/MSSA Blood Culture Test – BT (Test) were correctly called as S. aureus. For Resistance and Susceptibility (RS) determinations, none (0/120) of the MSSA samples were incorrectly determined to be MRSA. There was a $100 \%$ agreement. (111/111) for MRSA samples which had been correctly classified as S. aureus by the Identification component of the MRSA/MSSA Blood Culture Test – BT. The breakout of performance by site and strain for the initial reproducibility showed inconsistencies so additional reproducibility studies were conducted. Inconsistencies were due to sample preparation for testing.

In the additional study, reproducibility of the KeyPathTM MRSA/MSSA Blood Culture Test – BT was assessed over 6 days at 3 study sites (2 external, 1 internal), using 6 operators (2 at each site), with the following strains – 2 MRSA (JMI-520, JMI-912), 2 MSSA, (JMI-2226, JMI-7812) 1 NSA (ATCC-12228, S. epidermidis). Additionally, 3 GMP kit lots and 2 replicates per sample per session were tested. The analyte levels included 2 for MRSA and MSSA, 1 for NSA. The MRSA and MSSA samples were tested neat, and at 10-fold dilution, a level shown to be at or near the limit of detection. Accuracy was $6 4 4 / 6 4 8 = 9 9 . 4 \%$ . All 4 errors were false-resistance calls of MSSA runs. The 4 errors were distributed between the three sites and the 3 GMP lots. Accuracy among MRSAs was 288/288, $100 \%$ . Accuracy among MSSAs was $2 8 4 / 2 8 8 = 9 8 . 6 \%$ . Accuracy among NSAs was $100 \%$ .

b. Linearity/assay reportable range: Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Quality control was performed each day of testing on both the reference method and the test method at three testing sites. Reproducibility of the controls exceeded $9 4 . 5 \%$ at all sites during the course of the clinical trials. MicroPhage supplied frozen bacterial stocks of three control bacteria (Methicillin-resistant Staphylococcus aureus - MRSA, Methicillin-susceptible Staphylococcus aureus - MSSA, and nonStaphylococcus aureus - NSA) which required an additional 2 hours of incubation before testing for clinical design validation trial.

External Quality Control Procedure Verification:

A new protocol was external quality control was developed, using American Type Culture Collection (ATCC) stocks of MRSA, MSSA, and NSA as listed in the table below. Bacterial stocks of the three control strains were streaked to isolate single colonies which were grown overnight on trypticase soy agar with $5 \%$ sheep blood (TSA II). A 0.5 McFarland dilution was prepared with each isolate and 30 uL samples were directly inoculated into each Reaction Tube of the

KeyPathTM MRSA/MSSA Blood Culture Test – BT. Tests were run as per the Test procedure. There were no invalid runs and no tests were repeated

Indicated control strains.   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>ATCC Strain Number</td></tr><tr><td rowspan=1 colspan=1>Methicillin-resistant Staphylococcus aureus</td><td rowspan=1 colspan=1>43300</td></tr><tr><td rowspan=1 colspan=1>Methicillin-susceptible Staphylococcus aureus</td><td rowspan=1 colspan=1>14775</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis (Coagulase Negative)</td><td rowspan=1 colspan=1>12228</td></tr></table>

The External QC Control Protocol was executed by 3 operators over 3 days. No differences were seen between days and operators. The total number of valid runs and results for each control strain are shown below.

QC validation results   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>N valid results</td><td rowspan=1 colspan=1>N correct results</td><td rowspan=1 colspan=1>% (95% CI) correct results</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>100%(96.7% - 100%)</td></tr><tr><td rowspan=1 colspan=1>MSSA</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>100%(96.7% - 100%)</td></tr><tr><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>100% (96.7% - 100%)</td></tr></table>

The proposed External QC Control procedure demonstrated acceptable reliability between operators and days of testing. This change was accepted and used for the package insert procedure.

# Stability

Three KeyPathTM MRSA/MSSA Blood Culture Test – BT kit lots were assembled/configured using a combination of unique lots of each of five kit components. Shelf-life stability of the MRSA/MSSA Test was assessed, per applicable CLSI Standards (CLSI EP25-A: Evaluation of Stability of In Vitro Diagnostic Reagents (2009)), by real-time storage at labeled storage conditions $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ . The claimed shelf life is 24 weeks.

Detector devices, contained in their sealed storage pouches with desiccant pack, were placed at -20, 42 or $6 0 \mathrm { { } ^ { \circ } C }$ for 4 hours. After exposure, pouches were opened and Detectors were used under ambient laboratory conditions. A total of three lots of Detector were evaluated during this study.

A study was also conducted to determine the interaction of temperature and humidity while the detector was in use. A twofactorial designed experiment, covering the interactions of effects of temperature ranges of $4 { - } 3 5 ^ { \circ } \mathrm { C }$ $( 4 0 – 9 5 ^ { \circ } \mathrm { F } )$ and relative humidity (RH) ranges of $2 \mathrm { - } 9 0 \%$ , was employed. Detectors were placed in sealed chambers in the presence of saturated salt solutions (used to maintain a specific relative humidity), then chambers were placed in incubators (used to maintain a specific temperature) and left to equilibrate for 60 minutes. Temperature and RH achieved under these conditions was confirmed with a calibrated temperature/RH probe. A total of three lots of Detector were evaluated during this study.

Under normal ambient temperature conditions ( $1 8 { - } 2 5 ^ { \circ } \mathrm { C }$ or $6 5 { - } 7 8 ^ { \circ } \mathrm { F }$ ), relative humidity does not affect the ability of the Detector to accurately call negative and positive bacteriophage-containing samples. In addition, short-term exposure of the Detector to extremes in temperature (i.e. $- 2 0$ to $4 2 \mathrm { { } ^ { \circ } C }$ or -4 to $1 0 5 ^ { \circ } \mathrm { F }$ ) which may arise during shipment of the product does not adversely affect the accuracy of call rates for negative and positive bacteriophage-containing samples. However, the trend in call rates was observed at the extremes of the temperature range:

For high negative samples, a significant increase in positive call rate was seen as humidity increased when the Detector was incubated at $3 5 \mathrm { { ^ \circ C } }$ $( 9 5 ^ { \circ } \mathrm { F } )$ . For high negative and low positive samples, a significant decrease in positive call rates were seen when the incubation temperature was lowered to $4 \mathrm { { } ^ { \circ } C }$ $( 4 0 ^ { \circ } \mathrm { F } )$ .

# Specimen stability:

MicroPhage tested whether the age of the blood would affect bacteriophage amplification. It was determined that the KeyPathTM MRSA/MSSA Blood Culture Test must be performed $\leq 2 4$ hours after the Bactec instrument alarms for positive blood culture bottles confirmed to have GPCC or GPC by Gram stain.

MicroPhage tested the robustness of the operating parameters and Twenty-seven combinations of parameters were tested, in addition to controls. The 27 runs were performed on each of five strains of S. aureus and S. epidermidis. The operating range was set by the highest minimum and lowest maximum for each of the five strain types and summarized in the following table.

Operating Parameters   

<table><tr><td colspan="1" rowspan="1">Test parameter</td><td colspan="1" rowspan="1">Minimal</td><td colspan="1" rowspan="1">Maximal</td></tr><tr><td colspan="1" rowspan="1">mL blood added toculture bottle</td><td colspan="1" rowspan="1">5 ml</td><td colspan="1" rowspan="1">12 ml</td></tr><tr><td colspan="1" rowspan="1">Time after alarm, hrs</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">24</td></tr><tr><td colspan="1" rowspan="1">μl sample added</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">Min lag to incubation</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">Incubation temp</td><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1">Hours incubation</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">5.5</td></tr><tr><td>Min lag before detector</td><td>0</td><td>60</td></tr><tr><td>start</td><td></td><td></td></tr></table>

Research and development studies determined that the bacteriophage lose titer if they undergo more than one freeze-thaw cycle.

# d. Detection limit:

The analytical sensitivity of the KeyPathTM MRSA/MSSA Blood Culture Test – BT, for correct call rate for $S .$ . aureus ID, correct call rate for MSSA, and correct call rate for MRSA, was determined to be $6 . 0 { \mathrm { x } } 1 0 ^ { 5 } \mathrm { C F U / m L }$ from Bactec blood culture bottle samples.

The Test Detectors return a negative result with $5 9 5 \%$ probability at bacteriophage concentrations $\mathrm { 1 . 8 \times 1 0 ^ { 7 } P F U / m L }$ , and returned a positive result with $59 5 \%$ probability at bacteriophage concentrations $\mathrm { \dot { 1 } } . 4 \mathrm { \times 1 0 ^ { 8 } P F U / m L }$ . The cutoff level ( $50 \%$ call probability) is $4 . 9 \mathrm { ~ x ~ } 1 0 ^ { 7 }$ PFU/mL.

The prozone study used a high challenge concentration of bacteriophage $( \mathrm { 1 x 1 0 ^ { 1 1 } P F U / m L } )$ ), approximately 1.5 logs greater than the median phage output post-amplification of clinical samples, which failed to produce false negative results on the Detector indicating the Detector is visually insensitive to hook effect between $\mathrm { 1 x 1 0 ^ { 9 } }$ and $1 \mathrm { x } 1 0 ^ { 1 1 } \mathrm { P F U / m L }$ .

# e. Analytical specificity:

# Co-Infection (or Cross-Contamination) Study

The following permutations of Target and Contaminant strains were tested in the Co-Infection Study: MRSA x MS-CoNS, MRSA x MSSA, hMRSA (Heteroresistant MRSA) x MSSA, and MSSA x MRCoNS. Individual strains were spiked into charged Bactec™ blood culture bottles, per the Spiking Model, and grown to alarm on a Bactec™ 9050 blood culture incubator. Target strains were mixed with contaminant strains at volumes of 10:1, 1:1, and 1:10, and then the MRSA/MSSA Test was run. Visual calls of MRSA, MSSA or NSA (not S. aureus) were made by a panel of three readers who were blinded to strain identity. Tests were run in triplicate from $3 \mathrm { k i t }$ lots on three successive days. The MicroPhage Test showed a high ability to detect MRSA, even in a sample containing $5 9 0 \%$ MSSA (2 MSSA calls were made out of 81 total calls). Heteroresistant MRSA showed more sensitivity to mixture with an MSSA strain The MicroPhage Test can reliably $( \mathrm { P } \geq 9 5 \% )$ detect hMRSA from samples containing up to $3 7 \%$ MSSA. MRSA could be reliably detected in a culture

comprised of $> 9 0 \%$ methicillin-sensitive Coagulase Negative Staphylococci (MS-CoNS). Heteroresistant MRSA (hMRSA) could be reliably detected from samples containing up to $3 7 \%$ MSSA. At mixtures of $\ge 7 9 \%$ MSSA the test becomes more likely $( \mathrm { P } \geq 5 0 \% )$ to return a call of MSSA than of MRSA. Mixture of methicillin-resistant CoNS with MSSA yielded two types of results: 1) if the MR-CoNS strains were of a type that is intrinsically unreactive with the Test bacteriophage, no effect was seen, or 2) if the strain was intrinsically highly reactive (a rare phenotype), the Test returned a call of MRSA at contamination levels $> 2 5 \%$ .

Individual strains were spiked into BACTECTM blood culture bottles (Aerobic Plus) containing $8 { \cdot } 1 0 \mathrm { m L }$ whole blood, per the Spiking Model, and grown to alarm on a BACTECTM 9050 blood culture incubator. MRSA target strains were mixed with contaminant strains at volumes of 9:1, 1:1, and 1:9, and then the MRSA/MSSA Blood Culture Test was run. Visual calls of MRSA, MSSA or NSA (not S. aureus) were made by a panel of three technicians who were blinded to strain/sample identity. Bacterial input concentrations from blood bottle samples were determined by dilutional plate count techniques on tryptic soy agar, and colony forming unit (CFU) data was used to analyze percentage of MRSA call failures by cell percentage of nonMRSA co-infecting strain. Tests were run as one test on a single MRSA/MSSA kit lot on three successive days. Pure cultures of MRSA or NSA strains were run as controls each day. Correct calls of $100 \%$ were observed for pure cultures of MRSA and NSA (54/54 and 81/81 for MRSA and NSA, respectively). Mixed cultures of MRSA and NSA generated 486 total calls that were classified as either True (T) or False (F). Using CFU data, the cell input fraction comprising MRSA (as $\%$ of NSA) for each run were calculated. Assay outcome (T or F) was plotted against log (fractional MRSA) and a logistic regression fit was obtained. Based on the logistic fit, failure probabilities $10 \%$ , $50 \%$ and $90 \%$ ) of NSA co-infection were calculated. In the case of Gram negative rods (e.g. $E .$ . coli) and Gram positive cocci in clusters and pairs (e.g. S. anginosis) the Test is sensitive enough to return an accurate call $90 \%$ of the time with as little as $< 1 \%$ of bacterial input being MRSA. In presence of Gram positive rods (e.g. $B .$ . cereus), the Test was able to detect MRSA $90 \%$ of the time with ${ \sim } 1 0 \%$ of the bacteria input being MRSA.

When yeast was used as the co-infectant, $90 \%$ MRSA call accuracy required $\sim 5 4 \%$ of the input CFU to be MRSA. This observation resulted from disparities in the cell densities of each organism at blood culture alarm. Yeast cultures alarm at ${ \sim } 1 . 6 \mathrm { E } { + } 0 6$ CFU/mL compared to average concentration of $5 . 4 \mathrm { E } { + 0 7 }$ CFU/mL for MRSA cultures. As a consequence, the cell input of MRSA was reduced significantly relative to the other NSA strains evaluated in this study. As yeast

reach cell density at a log lower than MRSA at alarm and grow slower than S. aureus, under true mixed infection of yeast and S. aureus, yeast CFU would reach only as high as $10 \%$ of total CFU at alarm.

Additionally, the assay correctly returned a NSA call in all cases of yeast pure cultures run as control (18/18 calls) and indicates no risk of false response in the Test for this organism.

Study strains used and their characteristics.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Cefoxitin ZONE OFINHIBITION mm</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Strain*</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Description</td></tr><tr><td rowspan=1 colspan=1>JMI 520</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>7.9</td><td rowspan=1 colspan=1>mecA* Staphylococcal aureus, used asMicroPhage QC and control strain forMRSA</td></tr><tr><td rowspan=1 colspan=1>NARSA192</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>mecA* Staphylococcal aureus, wellcharacterized MRSA strain</td></tr><tr><td rowspan=1 colspan=1>ATCC43300</td><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>mecA* Staphylococcal aureus, used asMicroPhage QC and control strain forMRSA</td></tr><tr><td rowspan=1 colspan=1>ATCC3555</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td></tr><tr><td rowspan=1 colspan=1>JMI 017</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>JMI 27-10560</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Streptococcus anginosis</td></tr><tr><td rowspan=1 colspan=1>ATCC51188</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Enterococcus faecalis</td></tr><tr><td rowspan=1 colspan=1>ATCC13061</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Bacillus cereus</td></tr><tr><td rowspan=1 colspan=1>JMI 4220</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Escherichia coli</td></tr><tr><td rowspan=1 colspan=1>ATCC14053</td><td rowspan=1 colspan=1>NSA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Candida albicans</td></tr></table>

# Cross Reactivity Studies:

The Test was challenged against 33 Gram-negatives, 7 yeasts and 123 Gram-positives organisms (including 65 Staphylococcus species not SA, 14 Enterococcus spp. (table below), 16 Streptococcus spp., 12 Gram positive rod species, and 16 additional miscellaneous gram positive organisms). No evidence of cross-reaction was seen with the seven yeasts (C. albicans, C. glabrata, C. parapsilosis, C. krusei, C.

tropicalis, Cryptococcus neoformans, and Rhodotorula mucilaginosa), while there were 2 false-positive calls among the Gram-negatives and 12 false-positive calls among the Gram-positives. The Gram-negative false-positive was observed in a Proteus vulgaris strain. The strain was retested on two subsequent occasions, but no phage amplification and no LFI signal was observed on any replicate. The initial false-positive result may have been an operator error. The Gram-positive false positives are accounted for by two strains, JMI-7302 (9 MRSA/0 NSA calls) and JMI-3716 (3 MSSA/6 NSA calls), both S. epidermidis. These strains are known to be capable of cross-reacting with the bacteriophage cocktail, and thus likely represent true Test crossreactions. The Gram negative false positives could not be repeated on secondary testing. Based on the strains tested, the overall analytical specificity of the Test is $9 8 . 8 \%$ . This value is consistent with the results of clinical validation, which showed a value of $9 8 . 3 \%$ specificity in S. aureus identification.

The following Enterococcus species were evaluated in the analytical specificity studies with no misidentifications.

<table><tr><td rowspan=1 colspan=1>Enterococcus species</td><td rowspan=1 colspan=1>Strain</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>ATCC51188</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>JMI 13133</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>JMI 5515</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>JMI 5991</td></tr><tr><td rowspan=1 colspan=1>Enterococcusfaecium</td><td rowspan=1 colspan=1>JMI 3122</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>JMI 5430</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>VRE JMI 14275</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>VRE JMI 1698</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>ATCC 14025</td></tr><tr><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>ATCC 700327</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>JMI 12-581</td></tr><tr><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1>ATCC49573</td></tr><tr><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>ATCC10541</td></tr><tr><td rowspan=1 colspan=1>Enterococcus mundtii</td><td rowspan=1 colspan=1>ATCC43178</td></tr><tr><td rowspan=1 colspan=1>Enterococcus raffiniosus</td><td rowspan=1 colspan=1>ATCC49464</td></tr></table>

Thirty two different viruses, including Influenza Virus A H1, Influenza Virus A H2, Influenza Virus B, Respiratory Syncytial Virus (RSV) A, Respiratory Syncytial Virus (RSV) B , Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Adenovirus 7A, Enterovirus – Coxsackie, Rhinovirus, Metapneumovirus, Coronavirus 229E, Coronavirus SARS, Hepatitis B Virus (HBV), Hepatitis C Virus (HCV), Hepatitis D Virus (HDV), Human Immunodeficiency Virus (HIV) 1, Rubella Virus, Cytomegalovirus (CMV) , Mumps Virus ,

Varicella Zoster Virus (VZV) , Coxsackie virus , Epstein-Barr Virus , Herpes Virus 1 , Herpes Virus 2, Human T-Lymphotrophic Virus (HTLV) I, Human T-Lymphotrophic Virus (HTLV) II, Human Immunodeficiency Virus (HIV), 2 strains West Nile Virus (WNV), and Echovirus were tested. None (0/33) of the Reaction Media samples containing viral antigens or PBS-control resulted in falsereactive results when tested with the KeyPathTM MRSA/MSSA Blood Culture – BT Detector. In addition, $100 \%$ (33/33) of the bacteriophage-spiked Reaction Media samples containing viral antigens or PBS-control were detected (i.e. no false-negative results) when tested with the KeyPathTM MRSA/MSSA Blood Culture – BT Detector.

Analytical Reactivity (Inclusivity) Studies:

Multi-Locus Strain Type (MLST) Reactivity Inclusivity studies for the KeyPathTM MRSA/MSSA Blood Culture Test – BT was evaluated on a total of 114 MRSA and MSSA strains from North America and Western Europe and represented geographic and phylogenetic diversity including 46 MLS (multilocus sequence) types representing 17 clonal complexes. The MRSA or MSSA category of each strain was confirmed by cefoxitin disk diffusion testing following the guidance of CLSI M100-S20. All disk diffusion results were in agreement with source categorization of the strain.

The 114 strains were tested in replicate for a total of 228 test runs. Of the 22 incorrect calls, 20 were ID false-negatives (NSA) and 2 were false resistant (a single strain, NRS 274, OXA $\mathrm { M I C } = 2 ~ \mu \mathrm { g / m L }$ ). The overall sensitivity of detection for $S .$ . aureus was $2 0 8 / 2 2 8 = 9 1 . 8 \%$ . Among S. aureus true positives, accuracy of methicillin resistance and susceptibility determination is $2 0 4 / 2 0 6 = 9 9 . 0 \%$ .

The only MLS Type with more than one false call was ST239, a member of CC8, although it has also been described as a chimera of CC8 and CC30. These strains grew extremely slowly on initial testing, requiring up to 36 hours before alarm in the BACTEC™ 9050. (Normal time to alarm in the spike model is 12-16 hours for $S .$ . aureus.) Upon repeat testing, the strains alarmed in the normal time window, and returned true positive results. The initial result was thought to be an artifact.

Pulsed-Field Gel Electrophoresis (PFGE) USA Type Strain Reactivity Ten PFGE USA Type strains (representing 20 NARSA strains) were tested for reactivity with the MP Test. These strains included USA100 (382,741), USA200 (383, 71, 722), USA 300 (643, 739),USA 300- 0114 (384), USA 400 (123, 192, 193), USA500 (385, 708), USA 600 (648, 715), USA700( 386), USA800 (387), USA 1000 (NRS 483 ) and USA1100 (NRS 484).

All of the major clinical types (100, 300, 500) were represented by multiple isolates, and all yield correct results (MRSA) upon initial testing with the exception of one of three USA 300 strains which tested as false negative initially, but was correctly identified at repeat testing.

Types 700 and 1000 were represented by single isolates and gave false results (NSA) on initial testing. NRS 386 (USA 700) gave correct results on retesting, but NRS 483 (USA 1000) repeated the NSA result on retesting.

# Panton-Valentine Leukocidin (PVL) Strain Reactivity

Fourteen NARSA strains (8 positive for PVL (123, 192, 193, 384, 483, 484, 643, 739) and 6 negative for PVL (22, 648, 708, 715, 722, 741) strains were evaluated for this study. The KeyPathTM MRSA/MSSA Blood Culture Test – BT detected 6/6 PVL-negative strains and $6 / 7$ (plus one mixed result (NRS384)) PVL-positive strains. The false negative PVL positive strain was NRS 483.

# SCCmec Strain Reactivity

Strains with known SCCmec types from NARSA were obtained, and all isolates were MRSA by cefoxitin disk diffusion $( \leq 2 1 \mathrm { m m } )$ and had a vancomycin MIC of $\dot { \leq } 2 ~ \mu \mathrm { g / m L }$ . Seventeen strains representing two SCC mec types were tested. These included SCC mec II type (71, 192, 382, 383, 648, 715, 722, and 741) and SCC mec IV / Iva type (123, 193, 384, 385, and 386). Initial testing of the 17 strains (68 runs) demonstrated that the MP Test identified 56 of the runs as MRSA for an initial sensitivity of $8 5 \%$ . The three discordant strains were repeated and 67/68 runs $9 8 . 5 \%$ strain sensitivity) were correctly identified as MRSA by the KeyPathTM Test.

Borderline Oxacillin Resistant (BORSA) S. aureus Reactivity Fourteen Borderline Oxacillin Resistant (BORSA) S. aureus strains were obtained the Centers for Disease Control, Atlanta GA (CDC, $\mathrm { n } { = } 6$ isolates) and a private collection $\mathrm { \ n = 1 0 }$ isolates). Both institutions published on these and similar strains with the same definition for BORSA (Swenson et al. Diag. Microbiol- Infect. Dis. 58 (2007) 33-39, Louie et al. - J. Clin. Microbiol. 38 (6) (2000) 2170-2173).

Fourteen isolates were received by MicroPhage with oxacillin sensitivity classifications, which were determined by the source laboratory using Broth Micro-Dilution (BMD) techniques and were known by the source laboratory as mecA negative. ß-lactam resistance in these strains was also characterized using $1 \mu \mathrm { g }$ oxacillin or $3 0 ~ \mu \ g$ cefoxitin disk diffusion testing (DD) at MicroPhage. Criteria for antibiotic categorization followed CLSI M100-S19 as: ${ \ge } 2 2 ~ \mathrm { m m }$

cefoxitin-sensitive; $\geq 1 3 \mathrm { m m }$ oxacillin- sensitive and $1 1 { - } 1 2 \mathrm { m m }$ oxacillin-intermediate. As controls, ATCC 43300 (known MRSA) and ATCC 25923 (known MSSA) strains were tested. All strains were mecA-negative but had MIC values of $4 { - } 8 \mu \mathrm { g } / \mathrm { m l }$ by oxacillin broth microdilution testing by OXA-BMD methods at the source laboratories.

Oxacillin disk diffusion testing performed at MicroPhage, 3 strains tested R, 8 strains tested I, and 5 strains tested S. In cefoxitin disk diffusion testing, all 16 strains tested S. In the KeyPathTM Test, 1 strain tested R, 2 strains tested $\mathrm { R } / \mathrm { S }$ (meaning the replicates were equally divided between R and S) and 13 strains tested S. There were no differences in performance between kit lots or blood culture formulations.

# Modified intrinsic PBP-bearing Staphylococcus aureus strains (MODSA)

The performance of the KeyPathTM MRSA/MSSA Blood Culture Test – BT on rare modified intrinsic PBP-bearing Staphylococcus aureus strains (MOD-SA) was not evaluated.

# SCC mec drop outs

A panel of 28 empty cassette variant S. aureus strains was evaluated that harbor elements of the SCCmec cassette, but were phenotypically MSSA. These strains were obtained from two laboratories with private collections. These strains were extensively characterized with regard to susceptibility testing and molecular structure, the details of which have been published in peer-reviewed journals (Donnio et al. JCM (2005) vol. 43 (8) pp. 4191-3). The strains were multiple independent isolates, with various mutations in the SCCmec cassette, and included a variety of MLSTs. The phenotype of these strains was confirmed by cefoxitin disk diffusion. The KeyPathTM MRSA/MSSA Blood Culture Test – BT called all 28 strains as MSSA.

# Interferent Studies

Three different abnormal sera types (hemolytic, icteric and lipemic) for a total of 180 tests were performed (4 serum types $_ { \textrm { X 3 } }$ strain types $\textbf { X } 1 5$ serum samples) and high concentrations of four different serum components (hemoglobin, bilirubin, triglycerides and Intralipids) were evaluated at for a total of 130 tests using ‘high levels’ of hemoglobin, bilirubin, triglycerides and Intralipids at $5 ~ \mathrm { m g / m L }$ , $2 0 0 ~ \mu \mathrm { g / m L }$ , 30 $\mathrm { m g / m L }$ and $3 \mathrm { m m o l / L }$ $\mathrm { 8 3 4 u g / m L }$ of Intralipid) respectively. It was determined that detector visual calls were not affected by any of four probable interfering agents or the 3 abnormal sera types tested with the three test strains (MRSA, MSSA and NSA). Test accuracy remained $100 \%$ for all circumstances.

A 20-member Heterophilic Assessment Panel, comprised of 18 samples of human plasma samples containing heterophile antibodies, one sample of human anti-mouse antibodies (HAMA) and one purchased sample of Rheumatoid factor (Rf) and spiked into high negative (HN) and low positive (LP) bacteriophage samples. None of the HAMA, Rf, or heterophilic plasma samples used in this study interfered with interpretation of Detectors in the KeyPathTM MRSA/MSSA Blood Culture Test – BT when challenged at higher than normal sample volumes. Under normal clinical concentrations, these antibodies are not expected to interfere with interpretation of the Test. There was no effect of the component antibiotics on the functionality of bacteriophage in the KeyPathTM test.

Potential Drug Interferents and Culture Media Interferents Five antibiotics: 1) cephalexin, 2) ciprofloxacin, 3) gentamicin, 4) sulfisoxazole and 5) tetracycline, and three analgesics/antiinflammatories class 1) acetaminophen, 2) acetylsalicylic acid and 3) ibuprofen were tested. Acyclovir (antiviral), SPS (anticoagulant), and antibiotic-absorbing resin from blood culture bottles were also tested. Each test compound was spiked into the assay at the level corresponding to the CLSI recommended “test high dose” in blood. Where CLSI guidance was not available, the compound was tested above the highest likely or possible level. Two lots of the KeyPathTM Test were used in the study. The control assay did not contain any additive. Three control strains were tested in presence of each additive and under the control condition.

Control strains included were methicillin resistant Staphylococcus aureus (MRSA), a methicillin sensitive S. aureus (MSSA) and two coagulase negative Staphylococcus epidermidis (NSA - not-S. aureus). A total of 330 tests were performed. Results show detection/call accuracy for MRSA, MSSA and NSA bacteria was $100 \%$ and indicated the MRSA/MSSA Blood Culture Test - BT was not adversely affected by any of the potential interfering antibiotics, analgesics/anti-inflammatories, antiviral, SPS or blood culture bottle resin compounds.

f. Assay cut-off: Not Applicable

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

# a. Prospective Clinical studies

Subjects included individuals 18 years of age or older with positive blood cultures on a BactecTM System (9000 series or F/X), and the sample was tested on the KeyPathTM MRSA/MSSA Blood Culture Test – BT within 24 hours of positive determination on Bactec™ System (i.e. alarm). Aliquots of the blood culture were used to perform standard culture identification for S. aureus. The KeyPathTM MRSA/MSSA Blood Culture Test – BT was compared to: 1) the traditional enzymatic methods of coagulase tube and catalase slide tests and the latex agglutination method of Staphaurex $^ \mathrm { \textregistered }$ (Remel) for detection of Staphylococcus aureus and 2) the latex agglutination PBP2’ Test (Oxoid) and antibiotic susceptibility methods Cefoxitin and Oxacillin Sensi-disc (Becton-Dickinson) for determination of methicillin-resistant (MRSA) or methicillin–susceptible (MSSA) S. aureus in accordance with CLSI M100-S19. All clinical validation was performed and interpreted by single users, consistent with the intended use.

A total of 1116 ( $3 6 6 S .$ aureus, of which 191 were MRSA, 173 were MSSA, and 2 not evaluated) paired samples were tested for MRSA/MSSA by the KeyPathTM MRSA/MSSA Blood Culture Test – BT and the culture gold standard across all study sites. Sensitivity and specificity of the MRSA/MSSA Blood Culture Test vs. Gold Standard/predicate methods was $9 1 . 8 \%$ (88.5, 94.4; $\mathrm { n } { = } 1 5 6$ ) and $9 8 . 3 \%$ (97.1, 99.1; $\mathrm { n } = 1 8 0 $ ), respectively.

The sponsor compared the results of the KeyPathTM MRSA/MSSA Blood Culture Test – BT to $3 0 ~ \mu \ g$ cefoxitin disks and $1 ~ { \mu \mathrm { g } }$ oxacillin disks in accordance with CLSI M100-S19 in 336 paired tests. Additional MRSA characterization was performed using the PBP2’ latex agglutination test.

The Very Major Errors (vmj) and Major Errors (maj) were calculated as described in the AST guidance. The MRSA detection was acceptable using the cefoxitin disk testing, but not with the oxacillin disk testing. For cefoxitin, the vmj rate $1 . 1 1 \%$ with a $9 5 \%$ confidence interval (CI) of 0.1348, 3.956 and the maj rate was $0 . 6 \%$ with a $9 5 \%$ CI of 0.01644, 3.565. The oxacillin vmj rate was $8 . 7 6 \%$ with a $9 5 \%$ CI of 5.19, 13.66 and maj errors were $0 . 8 2 \%$ with a $9 5 \%$ CI 0.02058, 4.446.

The category NSA (not Staphylococcus aureus) indicates isolates which were not identified as Staphylococcus aureus and do not progress further to susceptibility testing. In these studies the percent of reference MRSA positive results which were KeyPathTM Test NSA was $5 . 7 6 \%$ (11/191) and the percent of reference MSSA positive results which were NSA was $1 0 . 9 8 \%$ (19/173) as compared to cefoxitin.

Within S. aureus positives determined to be either MRSA or MSSA by KeyPathTM Test, the MRSA call was $9 8 . 9 \%$ (178/180) accurate and the MSSA determination was $9 9 . 4 \%$ (153/154) accurate compared to the cefoxitin disk diffusion result. These results are summarized in the following tables:

KeyPathTM Test VS. TUBE COAGULASE/STAPHAUREX $+ 3 0$ UG CEFOXITIN DISK DIFFUSION STANDARD:

<table><tr><td rowspan="2">MRSA</td><td colspan="3">SA/TCT + FOX DD std MRSA MSSA</td></tr><tr><td>178</td><td>1</td><td>NSA 4</td></tr><tr><td rowspan="2">MWioG MSSA</td><td>2</td><td>153</td><td>9</td></tr><tr><td>NSA 11</td><td>19</td><td>737</td></tr></table>

$$
{ \mathsf N } = \mathsf { \Omega } 1 1 1 4
$$

• Very Major Errors (False-susceptible) $2 / 1 8 0 = 1 . 1 1 \%$ CI: $0 . 1 3 \%$ – $4 . 0 \%$

• Major Errors (False-resistant) $1 / 1 5 4 = 0 . 6 \%$ CI: $0 . 0 2 \% - 3 . 6 \%$

Not Staphylococcus aureus (NSA) Categories with Cefoxitin Disk Testing

<table><tr><td rowspan=1 colspan=1>NSA categories</td><td rowspan=1 colspan=1>Calculation</td><td rowspan=1 colspan=1>95% Confidence interval</td></tr><tr><td rowspan=1 colspan=1>Percent of referenceMRSA+ results that areMicrophage NSA</td><td rowspan=1 colspan=1>11/191 = 5.759%</td><td rowspan=1 colspan=1>CI:  2.91, 10.07</td></tr><tr><td rowspan=1 colspan=1>Percent of referenceMSSA+ results that areMicrophage NSA</td><td rowspan=1 colspan=1>19/173 = 10.98%</td><td rowspan=1 colspan=1>CI:    6.743, 16.62</td></tr><tr><td rowspan=1 colspan=1>Percent of reference SA+(MRSA+ or MSSA+)results that areMicrophage NSA</td><td rowspan=1 colspan=1>30/364 = 8.242%</td><td rowspan=1 colspan=1>CI: 5.63, 11.56</td></tr><tr><td rowspan=1 colspan=1>Percent of reference NSAresults that areMicrophage SA+(MRSA+ or MSSA+)</td><td rowspan=1 colspan=1>13/750 = 1.733 %</td><td rowspan=1 colspan=1>CI: 0.9261, 2.946</td></tr></table>

There were two isolates which were identified as MSSA by MicroPhage BT but identified as MRSA by the cefoxitin gold standard. The two strains, 52217 and 54011, were found to be mecApositive based on PBP2a results, and neither appeared to be borderline resistant based on zone of inhibition results. Sample 54011 was the only sample tested from this patient, while sample 52217 was one of 9 positive blood cultures tested from its respective patient. The additional 8 samples were all called MRSA by both the MP BT Test and the reference methods. Strain 52217 was not further characterized as it gave correct results on retesting (4/4 trials) and that clinical testing error of 52217 was attributed to either manufacturer defect or operator error. Strain 54011 was determined to be USA100/800, mecA-Type 2, PVL and TSST negative. It was noted that the MP BT Test discrepancy appeared to be due to inadequate bacteriophage amplification due to strain variance as molecular characterization. In this evaluation it was noted that of the 60 strains typed by the reference lab, 12 had the 100/800 PFGE type, and 11 were called correctly. Of 10 strains with mecA type 2, 9 were called correctly.

KeyPathTM Test VS. TUBE COAGULASE/STAPHAUREX $^ +$ 1 UG OXACILLIN DISK DIFFUSION STANDARD\*:

N = 1091   

<table><tr><td colspan="2"></td><td>SA/TCT MRSA</td><td>+ OXA MSSA</td><td>DD std NSA</td></tr><tr><td rowspan="4">Miieoae</td><td>MRSA</td><td>177</td><td>1</td><td>4</td></tr><tr><td>MSSA</td><td>17</td><td>121</td><td>9</td></tr><tr><td>NSA</td><td>11</td><td>14</td><td>737</td></tr></table>

• Very Major Errors (False-susceptible) $1 7 / 1 9 4 = 8 . 8 \%$ CI: $5 . 2 \%$ - $1 3 . 7 \%$

• Major Errors (False-resistant) $1 / 1 2 2 { = } 0 . 8 2 \%$ CI: $0 . 0 2 \% - 4 . 4 \%$ $^ { * } \mathrm { I n }$ the studies performed, the KeyPathTM Test significantly under called MRSA by the oxacillin disk diffusion gold standard as compared to the cefoxitin disk diffusion gold standard. Of the 17 Very Major Errors, 15 disagreed with both the cefoxitin disk diffusion results and the MicroPhage Test results. Of 16 samples that were called MRSA by oxacillin and MSSA by cefoxitin, 15 were from one site. The difference in reading method at this site was thought to account for most or all of the discrepancies between oxacillin and cefoxitin diffusion test results. Further investigation revealed a systematic downshift of $3 \mathrm { m m }$ in zone of inhibition diameter.

Not Staphylococcus aureus (NSA) Categories with Oxacillin Disk Testing   

<table><tr><td rowspan=1 colspan=1>NSA categories</td><td rowspan=1 colspan=1>Calculation</td><td rowspan=1 colspan=1>95% Confidence interval</td></tr><tr><td rowspan=1 colspan=1>Percent of referenceMRSA+ results that areMicrophage NSA</td><td rowspan=1 colspan=1>11/205 = 5.366 %</td><td rowspan=1 colspan=1>CI: 2.709, 9.398</td></tr><tr><td rowspan=1 colspan=1>Percent of referenceMSSA+ results that areMicrophage NSA</td><td rowspan=1 colspan=1>14/136 = 10.29 %</td><td rowspan=1 colspan=1>CI: 5.743, 16.67</td></tr><tr><td rowspan=1 colspan=1>Percent of reference SA+(MRSA+ or MSSA+)results that areMicrophage NSA</td><td rowspan=1 colspan=1>25/341 = 7.331 %</td><td rowspan=1 colspan=1>CI: 4.80, 10.63</td></tr><tr><td rowspan=1 colspan=1>Percent of reference NSAresults that areMicrophage SA+(MRSA+ or MSSA+)</td><td rowspan=1 colspan=1>13/750= 1.733 %</td><td rowspan=1 colspan=1>CI: 0.9261, 2.946</td></tr></table>

One strain each of Enterococcus faecalis, Staphylococcus warneri, Staphylococcus gallinarum, Staphylococcus lugdunensis and 2 strains of Staphylococcus epidermidis cross reacted with the Test. At all sites 72 Enterococcus samples and 19 "Gram-positive, non-Staph" samples (comprised of Enterococcus and Streptococcus) were identified. Of these ${ \sim } 8 0$ Enterococcus samples, only one gave a false-positive result. Fatty acid GC analysis and repeat sequencing data of this strain show that the best (not perfect) match to E. faecalis. This isolate was thought to be an atypical strain of $E$ . faecalis.

b. Retrospective Clinical studies Not applicable.

c. Other clinical supportive data (when a. and b. are not applicable):

Not applicable.

a. Clinical Sensitivity and b. Clinical Specificity Four clinical sites were included in the study. Per protocol, each site was instructed to collect samples from positive blood cultures, 0 – 24 hours after detection (i.e. alarm), from the BactecTM Culture System. Acceptable blood culture bottle types included BactecTM plus Aerobic and Plus Anaerobic bottles. Only patients $> = 1 8$ years of age were included in the study. A Gram stain was performed, bacteria were examined and only samples tested as Gram positive cocci in clusters (GPCC) were carried forward for additional testing. After Gram stain, samples were tested with a) the KeyPathTM MRSA/MSSA Blood

Culture Test – BT per MP2009-B and b) a cohort of Gold Standard and predicate tests including: coagulase tube test, catalase slide test, and Staphaurex $^ \mathrm { \textregistered }$ Test. The ID Gold Standard is defined as concordant results between the catalase, coagulase and Staphaurex $^ \mathrm { \textregistered }$ tests. In order to preserve blinding of sample information/results different technologists performed the Gold Standard tests and another group of technologists performed the predicate and MRSA/MSSA Blood Culture Tests. For S. aureus positive samples, PBP2’ Test, Cefoxitin Sensi-disc Test and Oxacillin Sensi-disc Test were performed to determine antibiotic susceptibility. The Cefoxitin Sensi-disc test was defined as the Gold Standard for MRSA/MSSA determination on $S .$ . aureus-positive samples.

The following chart summarizes all the clinical studies performed.   

<table><tr><td rowspan=1 colspan=1>Summary of Clinical Studies(Lower Confidence Interval %Sensitivity )</td><td rowspan=1 colspan=1>% Sensitivity(95% ConfidenceInterval</td><td rowspan=1 colspan=1>% Specificity(95% ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>MRSA determination, Cefoxitin,Overall Population</td><td rowspan=1 colspan=1>93.2 (88.6, 96.3)</td><td rowspan=1 colspan=1>99.5 (98.7, 99.8)</td></tr><tr><td rowspan=1 colspan=1>MRSA determination,Oxacillin,Overall Population</td><td rowspan=1 colspan=1>86 (80.5, 90.4)</td><td rowspan=1 colspan=1>99.4 (98.7, 99.8)</td></tr><tr><td rowspan=1 colspan=1>MSSA determination, Cefoxitin,Overall Population</td><td rowspan=1 colspan=1>88.4 (82.7, 92.8)</td><td rowspan=1 colspan=1>98.7 (97.8, 99.3)</td></tr><tr><td rowspan=1 colspan=1>MSSA determination,oxacillin,Overall Population</td><td rowspan=1 colspan=1>80 (82.5, 93.7)</td><td rowspan=1 colspan=1>97.2 (95.9, 98.1)</td></tr><tr><td rowspan=1 colspan=1>MRSA determination, Cefoxitin,by lot</td><td rowspan=1 colspan=1>78.6, 81.8, 87.8</td><td rowspan=1 colspan=1>99.7, 99.7, 99.0</td></tr><tr><td rowspan=1 colspan=1>MSSA determination, Cefoxitin,by lot</td><td rowspan=1 colspan=1>88.7, 73.8, 74.6</td><td rowspan=1 colspan=1>98.2, 99.3, 98.9</td></tr></table>

<table><tr><td colspan="1" rowspan="1">Staph. aureus Detection, OverallPopulation</td><td colspan="1" rowspan="1">88.51-94.40 (91.80)</td><td colspan="1" rowspan="1">97.05-99.07(98.27)</td></tr><tr><td colspan="1" rowspan="1">MRSA/MSSA Blood Culture TestPerformance by Kit Lot: Detectionof Staph. aureus</td><td colspan="1" rowspan="1">91-99.7 (97.4)81.9  92. 7 (88.1)87.2-96.8 (93)</td><td colspan="1" rowspan="1">98.099.697.4</td></tr><tr><td colspan="1" rowspan="1">Staph. aureus Detection, Noantibiotic exposure</td><td colspan="1" rowspan="1">87.9-94.5(91.6)</td><td colspan="1" rowspan="1">98.1 (96.6 - 99.1)</td></tr><tr><td colspan="1" rowspan="1">Staph. aureus Detection,Antibiotic exposure</td><td colspan="1" rowspan="1">83.4 - 97.5 (92.5)</td><td colspan="1" rowspan="1">98.6 (96-99.7)</td></tr><tr><td colspan="1" rowspan="1">Staph. aureus Detection, Noantiviralexposure</td><td colspan="1" rowspan="1">88.4 -94.3 (91.7)</td><td colspan="1" rowspan="1">100 (89.7-100)</td></tr><tr><td colspan="1" rowspan="1">Staph. aureus Detection, Antiviralexposure</td><td colspan="1" rowspan="1">39.8- 100 (100)too few data points</td><td colspan="1" rowspan="1">89.7-100 (100)</td></tr><tr><td colspan="1" rowspan="1">Predicate % agreement cat/coagS. aureus population</td><td colspan="1" rowspan="1">Pos agreement 88.4-94.4(91.8)</td><td colspan="1" rowspan="1">Neg agreement94.4 -98.2 (96.7)</td></tr><tr><td colspan="1" rowspan="1">Predicate % agreement StaphurexS. aureus population</td><td colspan="1" rowspan="1">Pos agreement 87.7 -93.8 (91.1)</td><td colspan="1" rowspan="1">Neg Agreement95.1-98.5 (97.1)</td></tr><tr><td colspan="1" rowspan="1">Predicate % agreement PBP2' S.aureus population</td><td colspan="1" rowspan="1">Pos agreement96.0-100 (98.9)</td><td colspan="1" rowspan="1">Neg Agreement94.5-100 (98.1)</td></tr><tr><td colspan="1" rowspan="1">Staph aureus by site (MicroPhagevs ID Gold Standard) - except 1site &gt;= 90%</td><td colspan="1" rowspan="1">85.41  98.35 (94)86.62  95.85 (92)80.06  95.28 (89.33)82.74  96.88 (94)</td><td colspan="1" rowspan="1">94.83 - 99.11 (97.59)97.27-99.99 (99.50)92.44-98.92 (96.69)96.29-99.98 (99.32)</td></tr><tr><td colspan="1" rowspan="1">Staph aureus by bottle typePlus AerobicPlus Anaerobic</td><td colspan="1" rowspan="1">86.77 - 94.20 (93)86.98  97.33 (95)</td><td colspan="1" rowspan="1">96.49-99.01 (98.03)98.07-100 (100)</td></tr></table>

c. Other clinical supportive data (when a. and b. are not applicable):

There were no subjects less than 18 years of age, seven subjects were between 18 and 21 years of age. All other subjects, 1109 total, were greater than 21 years of age at the time their blood was drawn.

The table below lists the organism identifications reported by site microbiology. 2 sites employed a Microscan, 1 used a Vitek 2, and 1 used a variety of manual culture methods.

<table><tr><td rowspan=1 colspan=1>Organism 1</td><td rowspan=1 colspan=1>Organism 2</td><td rowspan=1 colspan=1>N</td></tr><tr><td rowspan=1 colspan=1>Not reported</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter sp.</td><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Anaerobe</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>B. cepacia</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Bacillus sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Bacillus sp.</td><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Bacteroides sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Candida sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Candida sp.</td><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Candida sp.</td><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Capnocytophaga sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Clostridium sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>222</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>E. coli</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>GPNS</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>CoNS</td><td rowspan=1 colspan=1>1</td></tr></table>

Page 29 of 31   

<table><tr><td colspan="1" rowspan="1">E. aerogenes</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">CoNs</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">57</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">E. coli</td><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">GN</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">S. haemolyticus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">E. faecium</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">30</td></tr><tr><td colspan="1" rowspan="1">Enterococcus sp</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Gemella sp.</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">GN</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">GPNS</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">18</td></tr><tr><td colspan="1" rowspan="1">GPNS</td><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">K. pneumoniae</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Micrococcus sp.</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">P. aeruginosa</td><td colspan="1" rowspan="1">Viridans Strep</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">P. mirabilis</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">15</td></tr><tr><td colspan="1" rowspan="1">S. agalactiae</td><td colspan="1" rowspan="1">CoNS</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">365</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">Acinetobacter sp.</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">CoNS</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. aureus</td><td colspan="1" rowspan="1">E. cloacae</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. capitis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1">S. epidermidis</td><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">S. haemolyticus</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">S. hominis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">S. lugdunensis</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">S. maltophilia</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">Citrobacter sp.</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">S. marcescens</td><td colspan="1" rowspan="1">E. faecalis</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">S. mutans</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">S. pneumoniae</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">S. warneri</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Viridans Strep</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">Viridans Strep</td><td colspan="1" rowspan="1">GN</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Y</td><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1116</td></tr></table>

Abbreviations: CoNS, coagulase-negative Staphylococci; GPNS, Gram-positive, not Staphylococcus; GN, Gram-negative; Y, yeast.

4. Clinical cut-off:

Not Applicable

5. Expected values/Reference range:

In the KeyPathTM MRSA/MSSA Blood Culture Test – BT clinical study, a total 1114 clinical specimens were tested and determined to be MRSA, MSSA or not-S. aureus by reference culture methods. The number and percentage of MRSA and MSSA positive cases by age group are shown in the table below. For comparison, $1 6 . 5 \%$ of Gram-positive blood cultures in the US in 2002 were determined to be $S .$ . aureus (Ann Clin Microbiology Antimicrobial (2004) vol. 3 pp. 7). Of these, $8 . 1 \%$ were methicillin-resistant (MRSA) and $8 . 4 \%$ were methicillin-susceptible (MSSA). The frequency of MRSA and MSSA isolated at a given location should be confirmed by the user.

<table><tr><td rowspan=1 colspan=1>AgeGroup</td><td rowspan=1 colspan=1>Totalspecimens</td><td rowspan=1 colspan=1>nMRSA</td><td rowspan=1 colspan=1>%MRSA</td><td rowspan=1 colspan=1>nMSSA</td><td rowspan=1 colspan=1>%MSSA</td></tr><tr><td rowspan=1 colspan=1>0-17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>18-20</td><td rowspan=1 colspan=1>8*</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>62.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>21-30</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>39.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>31-40</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>9.9%</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>25.2%</td></tr><tr><td rowspan=1 colspan=1>41-50</td><td rowspan=1 colspan=1>148*</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>20.9%</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>12.8%</td></tr><tr><td rowspan=1 colspan=1>51-60</td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>13.8%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>18.8%</td></tr><tr><td rowspan=1 colspan=1>61-70</td><td rowspan=1 colspan=1>251</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>17.9%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>13.9%</td></tr><tr><td rowspan=1 colspan=1>&gt;70</td><td rowspan=1 colspan=1>322</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>14.9%</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>15.5%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1116</td><td rowspan=1 colspan=1>193</td><td rowspan=1 colspan=1>17.3%</td><td rowspan=1 colspan=1>173</td><td rowspan=1 colspan=1>15.5%</td></tr></table>

\* One sample in this group was identified as S. aureus by the gold standard ID method, but did not get a gold standard disk diffusion test. Sample was PBP2' positive and thus is classified as MRSA.

# Reference Range

Positive for Staphylococcus aureus Negative for Staphylococcus aureus Methicillin-resistant Staphylococcus aureus (MRSA) Methicillin-susceptible Staphylococcus aureus (MSSA)

# N. Proposed Labeling:

The labeling is sufficient and satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.