# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K103798 B. Purpose for Submission:

This is a new 510k application for Strand Displacement Amplification (SDA) assays used with the BD ViperTM System in Extracted Mode for the qualitative detection and differentiation of Herpes Simplex virus type 1 (HSV1) and Herpes Simplex virus type 2 (HSV2) DNA extracted from external anogenital lesion specimens obtained from symptomatic individuals.

# C. Measurand:

Target DNA sequences from HSV1 and HSV2

# D. Type of Test:

Qualitative, in vitro diagnostic tests using SDA technology for the detection and differentiation of HSV1 and HSV2 DNA extracted from external anogenital lesion specimens. Automated extraction of nucleic acids is performed using the BD ViperTM System in Extracted Mode. Amplification and detection of target DNA are carried out simultaneously using separate microwells for HSV1 and HSV2 specific reactions. The presence or absence of HSV DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

# E. Applicant:

BD Diagnostic Systems

F. Proprietary and Established Names:

BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305   
2. Classification: Class II   
3. Product code: OQO

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays (HSV $\mathrm { Q } ^ { x }$ Assays), when tested with the BD Viper™ System in Extracted Mode, use Strand Displacement Amplification technology for the direct, qualitative detection and differentiation of Herpes Simplex virus type 1 (HSV1) and Herpes Simplex virus type 2 (HSV2) DNA in clinician-collected external anogenital lesion specimens. The assays are indicated for use with symptomatic individuals to aid in the diagnosis of anogenital HSV1 and HSV2 infections.

Warning: The BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) Qx Amplified DNA Assays (HSV $\mathrm { Q } ^ { x }$ Assays) are not FDA cleared for use with cerebrospinal fluid (CSF). The assays are not intended to be used for prenatal screening or for individuals under the age of 17 years.

The Control Set for the BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays contains Positive and Negative Controls that are intended for Quality Control use with the BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays (HSV $\mathrm { Q } ^ { x }$ Assays) when tested with the BD Viper™ System in Extracted Mode.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: To be used with the BD Viper™ System in Extracted Mode

# I. Device Description:

The BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays are based on the simultaneous amplification and detection of target DNA using amplification primers and Fluorescently labeled detector probes. The reagents for SDA are dried in four separate disposable microwells:

1. Two Priming Microwells (one each for HSV1 and HSV2) contain the amplification primers, fluorescently labeled detector probe, nucleotides and other reagents necessary for amplification, and

2. Two Amplification Microwells (one each for HSV1 and HSV2) contain the enzymes (a DNA polymerase and a restriction endonuclease) that are required for SDA.

In addition to the fluorescent probe used to detect amplified HSV target DNA, a second labeled oligonucleotide is incorporated into each reaction. This Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the HSV specific target and is used to confirm the validity of the extraction process. The EC is dried in the extraction tubes and is rehydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and HSV specific signals to report specimen results as positive, negative, or EC failure.

An overview of the procedure is as follows:

1. The patient swab is collected from external anogenital lesions using any of the following specimen and collection transport devices: BD ProbeTec $\mathrm { Q } ^ { x }$ Collection Kit for Endocervical or Lesion Specimens, BD Universal Viral Transport (UVT), or Copan Universal Transport Medium (UTM-RT), which is formulated identically to BD UVT.   
2. The specimen is pre-warmed at $1 1 4 ^ { 0 } \mathrm { C }$ in $\mathrm { Q } ^ { x }$ Swab Diluent Tube placed into a BD Viper Lysing Heater.   
3. After cooling, the specimen is loaded onto the BD Viper System.   
4. The specimen is transferred by the BD Viper System to an Extraction Tube that contains ferric oxide particles in a dissolvable film and dried Extraction Control. Viruses are lysed at high $\mathsf { p H }$ to liberate the DNA into solution. Acid is then added to lower the $\mathsf { p H }$ and induce a positive charge on the ferric oxide, which in turn binds the negatively charged DNA. The particles and bound DNA are then pulled to the sides of the extraction tube by magnets and the remainder of the treated specimen is aspirated to waste. The particles with bound DNA are washed and a high pH Elution Buffer is added to elute the purified DNA. Finally, a Neutralization Buffer is used to adjust the extracted solution to the optimal pH for amplification of the target.   
5. The BD Viper System pipettes portions of the purified DNA solution from each extraction tube into two separate Priming Microwells to rehydrate their contents, one well corresponding to HSV1 and the other to HSV2. After a brief incubation, the reaction mixtures are transferred to corresponding, prewarmed Amplification Microwells which are then sealed to prevent contamination and incubated in one of the two thermally controlled fluorescent readers. The presence or absence of HSV DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value.

Each BD ProbeTec Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assays Reagent Pack contains:

HSV1 $\mathbf { Q } ^ { x }$ Amplified DNA Assay Priming Microwells, 3 pouches of 32 microwells each (96 microwells per carton): each Priming Microwell contains approximately 108 pmol oligonucleotides, 27 pmol fluorescently labeled detector probe, 150 nmol dNTPs, with stabilizers and buffer components. HSV1 $\mathbf { Q } ^ { x }$ Amplified DNA Assay Amplification Microwells, 3 pouches of 32 microwells each (96 microwells per carton): each Amplification Microwell contains approximately 100 units DNA polymerase and 500 units restriction enzyme, with stabilizers and buffer components. HSV2 $\mathbf { Q } ^ { x }$ Amplified DNA Assay Priming Microwells, 3 pouches of 32 microwells each (96 microwells per carton): each Priming Microwell contains approximately 105 pmol oligonucleotides, 36 pmol fluorescently labeled detector probe, 120 nmol dNTPs, with stabilizers and buffer components. • HSV2 $\mathbf { Q } ^ { x }$ Amplified DNA Assay Amplification Microwells, 3 pouches of 32 microwells each (96 microwells per carton): each Amplification Microwell contains approximately 35 units DNA polymerase and 500 units restriction enzyme, with stabilizers and buffer components.

External Control Reagents are adjunct reagents to the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays. Each Control Set for the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays contains 24 HSV $\mathrm { Q } ^ { \mathrm { x } }$ Positive Control Tubes and 24 HSV $\mathrm { Q } ^ { x }$ Negative Control Tubes. The Control Set is intended for Quality Control use with The BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays when tested with the BD Viper™ System in Extracted Mode. The HSV $\mathrm { Q } ^ { \mathrm { x } }$ Positive Control will monitor for substantial reagent failure only. The HSV $\mathrm { Q } ^ { \mathrm { x } }$ Negative Control monitors for reagent and/or environmental contamination.

The devices that can be used to collect patient specimens are:

BD Universal Viral Transport (UVT) with $3 ~ \mathrm { m L }$ fill volume and regular sized polyester-fiber-tip swab with plastic shaft. The Copan Universal Transport Medium (UTM-RT) which is identical to the BD UVT may also be used. The Package Insert for the BD ProbeTec™ Herpes Simplex Viruses (HSV 1 & 2) $\mathrm { Q } ^ { x }$ Amplified DNA Assays lists the catalog numbers for both BD and Copan specimen collection kits.

• BD ProbeTec $\mathrm { Q } ^ { x }$ Collection Kit for Endocervical or Lesion Specimens.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ELVIS $^ \mathrm { \textregistered }$ HSV ID/Typing Test System (Diagnostic Hybrid, Inc.)

2. Predicate Numbers (s): K971662, K091753

3. Comparison with predicate:

# Device Comparison: Predicate Device for the BD ProbeTec HSV $\mathbf { Q } ^ { x }$ Assays on the BD Viper System in Extracted Mode

<table><tr><td rowspan=1 colspan=1>Features</td><td rowspan=1 colspan=1>BD ProbeTec HSV Qx Amplified DNAAssays on the BD Viper System inExtracted Mode(New Device)</td><td rowspan=1 colspan=1>Diagnostics Hybrid ,Inc.ELVIS® HSV ID/Typing Test System(Predicate Device)</td></tr><tr><td rowspan=1 colspan=1>510(k)</td><td rowspan=1 colspan=1>K103798</td><td rowspan=1 colspan=1>K971662, K091753</td></tr><tr><td rowspan=1 colspan=3>SIMILARITIES</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The BD ProbeTecTM Herpes Simplex Viruses(HSV 1 &amp; 2) Q&quot; Amplified DNA Assays(HSV Q&quot; Assays), when tested with the BDViperTM System in Extracted Mode, useStrand Displacement Amplificationtechnology for the direct, qualitative detectionand differentiation of Herpes Simplex virustype 1 (HSV1) and Herpes Simplex virus type2 (HSV2) DNA in clinician-collected externalanogenital lesion specimens. The assays areindicated for use with symptomaticindividuals to aid in the diagnosis ofanogenital HSV1 and HSV2 infections.Warning: The BD ProbeTecTM HerpesSimplex Viruses (HSV 1 &amp; 2) Qx AmplifiedDNA Assays (HSV Qx Assays) are not FDAcleared for use with cerebrospinal fluid (CSF).The assays are not intended to be used forprenatal screening or for individuals under theage of 17 years.</td><td rowspan=1 colspan=1>The ELVIS®HSV ID and D³ Typing TestSystem provides Cells, ReplacementMedium and Test Reagents for the culture,qualitative identification and typing ofherpes simplex virus (HSV) from cutaneousor mucocutaneous specimens as an aid in thediagnosis of HSV type 1 (HSV-1) and HSVtype 2 (HSV-2) infections. The performancecharacteristics of this assay have not beenestablished for antiviral therapy, prenatalmonitoring or use with cerebral spinal fluidspecimens.</td></tr><tr><td rowspan=1 colspan=1>Identificationand Typing ofHSV-1 andHSV-2</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=3>DIFFERENCES</td></tr><tr><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Strand Displacement Amplification of DNAtarget</td><td rowspan=1 colspan=1>Cell culture using an enzyme linkedinducible system</td></tr><tr><td rowspan=1 colspan=1>AnalysisSoftwareProvided</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Printed ResultsReportProvided</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td></tr><tr><td rowspan=1 colspan=1>Kit reagentStorageConditions</td><td rowspan=1 colspan=1>2°C to 33°</td><td rowspan=1 colspan=1>2°C to 8°C and 22°C to 28°C</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Guidance for Industry and FDA Staff: Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, issued on May 11, 2005.   
Protocols for Determination of Limits of Detection and Limits of Quantitation (CLSI EP-17A 2004)

# L. Test Principle:

The BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays target analyte-specific sequences within a common region of the herpes simplex viral genome. Differences in the oligonucleotide sequences between the two assays allow specific detection of either HSV1 or HSV2. DNA is extracted and purified from a patient swab specimen by the BD Viper System using ferric oxide particles. The purified DNA is then transferred to two Priming Microwells (one each for HSV1 and HSV2) that contain specific primers and fluorescently labeled probes. After an incubation period, the contents of the Priming Microwells are transferred to two Amplification Microwells that contain the enzymes (DNA polymerase and restriction endonuclease) that are necessary for Strand Displacement Amplification (SDA) to occur. SDA is carried out under isothermal conditions. If HSV1 DNA is present in the patient sample, the target sequence is amplified by SDA in the Amplification Microwell containing HSV1 specific reagents but not in the Amplification Microwell containing HSV2 specific reagents. As HSV1 target is amplified, the HSV1 fluorescently labeled probe is degraded, resulting in a concomitant increase in fluorescent signal in the HSV1 Amplification Microwell (but not in the HSV2 Amplification Microwell). If HSV2 DNA is present in the patient sample, amplification of the HSV2 target sequence occurs in the HSV2 Amplification Microwell, and there is an increase in fluorescent signal in the HSV2 Amplification Microwell (but not in the HSV1 Amplification Microwell). In the rare event that a patient sample contains both HSV1 and HSV2 DNA, there will be an increase in fluorescence signal in both HSV1 and HSV2 Amplification Microwells. If there is no HSV DNA in the patient sample, amplification of target sequence does not occur and there is no increase in fluorescence signal in either the HSV1 or HSV2 Amplification Microwells.

The change in fluorescence in the Amplification Microwells is monitored during the course of the amplification reaction. The presence or absence of HSV DNA is determined by calculating the peak fluorescence (Maximum Relative Fluorescent Units (MaxRFU)) over the course of the amplification process and by comparing this measurement to a predetermined threshold value of 125 MaxRFU..

In addition to the fluorescent probe used to detect amplified HSV target DNA, a second labeled oligonucleotide is incorporated into each reaction. The Extraction Control (EC) oligonucleotide is labeled with a different dye than that used for detection of the HSV specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is rehydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an

automated algorithm is applied to both the EC and HSV specific signals to report specimen results as positive, negative, or EC failure.

# Interpretation of Control Results:

The HSV1 and HSV2 Positive Control and the HSV1 and HSV2 Negative Control must test as positive and negative, respectively, in order to obtain patient results. If controls do not perform as expected, the run is considered invalid and patient results will not be reported by the instrument. If either of the controls does not provide the expected results, the entire run should be repeated using a new set of controls, new extraction tubes, new extraction reagent trough, new lysis trough and new microwells. The following table summarizes the interpretation of quality control results.

Interpretation of Quality Control Results   

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Tube Result ReportSymbol*</td><td rowspan=1 colspan=1>MaxRFU</td><td rowspan=1 colspan=1>QC Disposition</td></tr><tr><td rowspan=1 colspan=1>HSV Positive Control</td><td rowspan=1 colspan=1>OK</td><td rowspan=1 colspan=1>≥125</td><td rowspan=1 colspan=1>QC Pass</td></tr><tr><td rowspan=1 colspan=1>HSV Positive Control</td><td rowspan=1 colspan=1>P*</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1>HSV Positive Control</td><td rowspan=1 colspan=1>or or </td><td rowspan=1 colspan=1>Any value</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HSV Negative Control</td><td rowspan=1 colspan=1>OK</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>QC Pass</td></tr><tr><td rowspan=1 colspan=1>HSV Negative Control</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>≥125</td><td rowspan=1 colspan=1>QC Failure</td></tr><tr><td rowspan=1 colspan=1>HSV Negative Control</td><td rowspan=1 colspan=1>or or or</td><td rowspan=1 colspan=1>Any value</td><td rowspan=1 colspan=1>QC Failure</td></tr></table>

\* = Fail, $=$ Extraction Transfer failure, $=$ Liquid Level failure, $\bigstar \bigstar _ { \mathbf { \xi } }$ Extraction Control failure, $\pm \pmb { \theta } = \mathsf { E r r o r }$

# Interpretation of Specimen Results:

The BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays use fluorescent energy transfer as the detection method to test for the presence of Herpes Simplex Types 1 and 2 in clinical specimens. All calculations are performed automatically by the BD Viper software. The presence or absence of HSV1 or HSV2 DNA is determined by calculating the peak fluorescence (MaxRFU) over the course of the amplification process and by comparing this measurement to a predetermined threshold value. The magnitude of the MaxRFU score is not indicative of the level of organism in the specimen. If the HSV1 or HSV2 specific signal is greater than or equal to a threshold of 125 MaxRFU, the EC fluorescence is ignored by the algorithm. If the HSV1 or HSV2 specific signal is less than a threshold of 125 MaxRFU, the EC fluorescence is utilized by the algorithm in the interpretation of the result. If assay control results are not as expected, patient results are not reported. Reported results are determined as summarized in the following table.

# Interpretation of Specimen Results for the HSV1 and HSV2 $\mathbf { Q } ^ { x }$ Assays

<table><tr><td rowspan=1 colspan=1>TubeReportResult</td><td rowspan=1 colspan=1>HSV QxMaxRFU</td><td rowspan=1 colspan=1>Report</td><td rowspan=1 colspan=1>Interpretation</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>≥125</td><td rowspan=1 colspan=1>HSV DNA detected by SDA</td><td rowspan=1 colspan=1>Positive for HSV DNA</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>HSV DNA not detected by SDA</td><td rowspan=1 colspan=1>Negative for HSV DNA</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>&lt;125</td><td rowspan=1 colspan=1>Extraction control failure. Repeat testfrom initial specimen tube or obtainanother specimen for testing.</td><td rowspan=1 colspan=1>Non-reportable result</td><td rowspan=1 colspan=1>ExtractionControlFailure</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>Anyvalue</td><td rowspan=1 colspan=1>Extraction Transfer Failure. Repeattest from initial specimen tube orobtain another specimen for testing.</td><td rowspan=1 colspan=1>Non-reportable result</td><td rowspan=1 colspan=1>ExtractionTransferFailure</td></tr><tr><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>Anyvalue</td><td rowspan=1 colspan=1>Liquid Level Failure. Repeat testfrom initial specimen tube or obtainanother specimen for testing.</td><td rowspan=1 colspan=1>Non-reportable result</td><td rowspan=1 colspan=1>LiquidLevelFailure</td></tr><tr><td rowspan=1 colspan=1>→0</td><td rowspan=1 colspan=1>Anyvalue</td><td rowspan=1 colspan=1>Error. Repeat test from initialspecimen tube or obtain anotherspecimen for testing.</td><td rowspan=1 colspan=1>Non-reportable result</td><td rowspan=1 colspan=1>Error</td></tr></table>

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

An Inter-Laboratory Reproducibility study was conducted at 3 laboratory sites (two external and one internal) with one BD Viper System per site. Two panels of simulated specimens were tested. One panel consisted of HSV1 and/or HSV2 viral particles spiked into $\mathrm { Q } ^ { x }$ Swab Diluent containing a swab (simulated $\mathrm { Q } ^ { x }$ Swab Diluent). The other panel consisted of HSV1 and/or HSV2 viral particles spiked into UVT mixed with $\mathrm { Q } ^ { x }$ Swab Diluent (simulated UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimen type). Uninoculated $\mathrm { Q } ^ { x }$ Swab Diluent or UVT in $\mathrm { Q } ^ { x }$ Swab Diluent was used for the negative samples. Six replicates of each panel member were tested every day for five days on each BD Viper System for a total of ninety results per panel member. The data are summarized in the following two tables for HSV1 and HSV2.

Summary of Reproducibility Data when tested on the BD Viper System with the HSV1 $\mathbf { Q } ^ { x }$ Amplified DNA Assay   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Site #1</td><td rowspan=1 colspan=4>Site #2</td><td rowspan=1 colspan=3>Site #3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>SimulatedSpecimenType</td><td rowspan=2 colspan=1>TargetsHSV1/HSV2(xLOD)</td><td rowspan=2 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>MaxRFU</td><td rowspan=1 colspan=1>MaxRFU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>AgreementwithExpected</td><td rowspan=2 colspan=2>Max  MaxRFU  RFUMeanStdDev</td><td rowspan=2 colspan=1>%CV</td><td rowspan=1 colspan=1>AgreementwithExpected</td><td rowspan=1 colspan=1>Max  MaxRFU RFU</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>TotalAgreementwithExpecte</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>StdDev</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>MeanStdDev</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Relts (%)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>6.7</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>12.9</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.7    2.2</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=4 colspan=1>UVT inQ SwabDiluent</td><td rowspan=1 colspan=1>2x/0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>1432.6</td><td rowspan=1 colspan=1>447.6</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1375.4</td><td rowspan=1 colspan=1>326.2</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1474.5  307.9</td><td rowspan=1 colspan=1>20.9</td><td rowspan=1 colspan=1>89/90 (98.9%)</td><td rowspan=1 colspan=1>94% - 100%</td></tr><tr><td rowspan=1 colspan=1>0/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>10.9</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>7.8</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>7.3   10.7</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>2/5x</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>1123.1</td><td rowspan=1 colspan=1>622.9</td><td rowspan=1 colspan=1>55.5</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1144.9</td><td rowspan=1 colspan=1>397.7</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1441.4 205.8</td><td rowspan=1 colspan=1>14.3</td><td rowspan=1 colspan=1>89/90 (98.9%)</td><td rowspan=1 colspan=1>94% - 100%</td></tr><tr><td rowspan=1 colspan=1>5x/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1586.2</td><td rowspan=1 colspan=1>346.8</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1531.5</td><td rowspan=1 colspan=1>165.4</td><td rowspan=1 colspan=1>10.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1619.7  175.7</td><td rowspan=1 colspan=1>10.8</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=5 colspan=1>Q&quot; SwabDiluent</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>15.5</td><td rowspan=1 colspan=1>84.7</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.9    3.8</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>88/90 (97.8%)</td><td rowspan=1 colspan=1>92.2% - 99.7%</td></tr><tr><td rowspan=1 colspan=1>2x/0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1477.2</td><td rowspan=1 colspan=1>392.3</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>999.2</td><td rowspan=1 colspan=1>482.9</td><td rowspan=1 colspan=1>48.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1276.7 396.0</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>88/90 (97.8%)</td><td rowspan=1 colspan=1>92.2% - 99.7%</td></tr><tr><td rowspan=1 colspan=1>0/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>12.3</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.3    0.8</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>2x/5x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1551.0</td><td rowspan=1 colspan=1>369.1</td><td rowspan=1 colspan=1>23.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1395.1</td><td rowspan=1 colspan=1>144.3</td><td rowspan=1 colspan=1>10.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1320.5 349.9</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>5x/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1751.5</td><td rowspan=1 colspan=1>174.0</td><td rowspan=1 colspan=1>9.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1441.8</td><td rowspan=1 colspan=1>214.8</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1535.1 244.1</td><td rowspan=1 colspan=1>15.9</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr></table>

$^ { * * }$ HSV1 and HSV2 were spiked at either 2-fold $( 2 \mathbf { x } )$ or 5-fold (5x) the LOD.

Summary of Reproducibility Data when tested on the BD Viper System with the HSV2 $\mathbf { Q } ^ { x }$ Amplified DNA Assay   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=4>Site #1</td><td rowspan=1 colspan=4>Site #2</td><td rowspan=1 colspan=4>Site #3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=2>SimulatedSpecimenType</td><td rowspan=2 colspan=1>TargetsHSV1/HSV2(xL)*</td><td rowspan=2 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Max</td><td rowspan=2 colspan=2>MaxRFUStdDev%CV</td><td rowspan=2 colspan=1>AgreementwithExpectedResults</td><td rowspan=2 colspan=1>MaxRFUMean</td><td rowspan=2 colspan=1>MaxRFUStdDev</td><td rowspan=2 colspan=1>%CV</td><td rowspan=2 colspan=1>AgreementwithExpectedResults</td><td rowspan=2 colspan=1>MaxRFUMean</td><td rowspan=2 colspan=1>MaxRFUStdDev</td><td rowspan=2 colspan=1>%CV</td><td rowspan=2 colspan=1>TotalAgreementwithExpectedRests (%)</td><td rowspan=2 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>RFUMean</td></tr><tr><td rowspan=5 colspan=2>N/A*UVTin QnSwabDiluent</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>29/29***</td><td rowspan=1 colspan=1>6.5</td><td rowspan=1 colspan=2>9.2N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>5.2</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>7.4</td><td rowspan=1 colspan=1>8.1</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>89/89 (100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>2x/0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=2>9.8N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>12.2</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>14.4</td><td rowspan=1 colspan=1>13.5</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>0/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1768.6</td><td rowspan=1 colspan=2>390.0 22.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1806.1</td><td rowspan=1 colspan=1>263.6</td><td rowspan=1 colspan=1>14.6</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1910.6</td><td rowspan=1 colspan=1>236.5</td><td rowspan=1 colspan=1>12.4</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>2x/5x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1946.5</td><td rowspan=1 colspan=1>246.8</td><td rowspan=1 colspan=1>12.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1800.1</td><td rowspan=1 colspan=1>267.6</td><td rowspan=1 colspan=1>14.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1965.1</td><td rowspan=1 colspan=1>171.7</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>5x/2x</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1866.1</td><td rowspan=1 colspan=1>319.9</td><td rowspan=1 colspan=1>17.1</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1823.8</td><td rowspan=1 colspan=1>250.8</td><td rowspan=1 colspan=1>13.8</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1861.2</td><td rowspan=1 colspan=1>224.8</td><td rowspan=1 colspan=1>12.1</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=3 colspan=2>Q&quot; SwabDiluent</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>29/29*</td><td rowspan=1 colspan=1>8.2</td><td rowspan=1 colspan=1>19.1</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>2.9</td><td rowspan=1 colspan=1>11.9</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>89/89 (100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>2x/0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>11.2</td><td rowspan=1 colspan=1>23.6</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>N/A*</td><td rowspan=1 colspan=1>90/90 (100%)</td><td rowspan=1 colspan=1>96% - 100%</td></tr><tr><td rowspan=1 colspan=1>vab</td><td rowspan=1 colspan=1>0/2x</td><td rowspan=1 colspan=1>29/29*</td><td rowspan=1 colspan=1>2007.1</td><td rowspan=1 colspan=2>219.4 10.9</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1679.5</td><td rowspan=1 colspan=1>331.1</td><td rowspan=1 colspan=1>19.7</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1757.3</td><td rowspan=1 colspan=1>289.2</td><td rowspan=1 colspan=1>16.5</td><td rowspan=1 colspan=1>89/89 (100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2x/5x</td><td rowspan=1 colspan=1>29/29*</td><td rowspan=1 colspan=1>2028.3</td><td rowspan=1 colspan=2>230.6 11.4</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1787.5</td><td rowspan=1 colspan=1>327.9</td><td rowspan=1 colspan=1>18.3</td><td rowspan=1 colspan=2>30/30</td><td rowspan=1 colspan=1>1882.7</td><td rowspan=1 colspan=1>342.1</td><td rowspan=1 colspan=1>18.2</td></tr><tr><td rowspan=1 colspan=1>5x/2x</td><td rowspan=1 colspan=1>29/29*</td><td rowspan=1 colspan=1>1960.7</td><td rowspan=1 colspan=2>254.7 13.0</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1756.2</td><td rowspan=1 colspan=1>286.7</td><td rowspan=1 colspan=1>16.3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>1762.7</td><td rowspan=1 colspan=1>350.3</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>89/89 (100%)</td><td rowspan=1 colspan=1>95.9% - 100%</td></tr></table>

$^ { * * } \mathrm { H S V 1 }$ and HSV2 w sike t old $( 2 \mathrm { x } )$ or 5-fold $( 5 \mathrm { x } )$ the LOD. $^ { \ast \ast \ast }$

Tlu negative) at target levels below the analytical LOD. These additional specimens were comprised of HSV1 $+ 2$ viral particles seeded into either $\mathrm { Q } ^ { x }$ Swab Diluent or UVT in $\mathrm { Q } ^ { x }$ Swab Diluent at 1:10 and 1:100 dilutions of the respective for the BD ProbeTec HSV1 $\mathrm { Q } ^ { x }$ and HSV2 $\mathrm { Q } ^ { x }$ Amplified DNA Assays. Six replicates of each panel member were tested y HSV1 and HSV2.

SysReucily Tar Leve Belohealyical Lim e  h S $\mathbf { Q } ^ { x }$ Amplified DNA Assay   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Site #1</td><td rowspan=1 colspan=4>Site #2</td><td rowspan=1 colspan=3>Site #3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>SimulatedSpecimenType</td><td rowspan=1 colspan=1>DilutionofAnalytical LOD</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Total-ve</td><td rowspan=1 colspan=1>Max RFUMean-ve</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Total-ve</td><td rowspan=1 colspan=1>Max RFUMean-ve</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Max RFUTotal   Mean-ve-ve</td><td rowspan=1 colspan=1>TotalPositives(%)</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=2 colspan=1>UVT inQ&quot; Swab Diluent</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>501.0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>439.0</td><td rowspan=1 colspan=1>29       1.3</td><td rowspan=1 colspan=1>2/90(2.2%)</td><td rowspan=1 colspan=1>0.3% - 7.8%</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>667.4</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>674.0</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>635.2</td><td rowspan=1 colspan=1>25      10.6</td><td rowspan=1 colspan=1>19/90(21.1%)</td><td rowspan=1 colspan=1>13.2% - 31%</td></tr><tr><td rowspan=2 colspan=1>Q&quot; Swab Diluent</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>389.0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>228.0</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>30      2.0</td><td rowspan=1 colspan=1>3/90(3.3%)</td><td rowspan=1 colspan=1>0.7% - 9.4%</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>424.5</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>423.6</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>13.4</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>434.4</td><td rowspan=1 colspan=1>22      12.4</td><td rowspan=1 colspan=1>27/90(30%)</td><td rowspan=1 colspan=1>20.8% - 40.6%</td></tr></table>

System Reproducibility at Target Levels Below the Analytical Limit of Detection for the $\mathbf { H } \mathbf { S } \mathbf { V } \pmb { 2 } \mathbf { \Lambda } \mathbf { Q } ^ { x }$ Amplified DNA Assay   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Site #1</td><td rowspan=1 colspan=4>Site #2</td><td rowspan=1 colspan=4>Site #3</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>SimulatedSpecimenType</td><td rowspan=1 colspan=1>DilutionofAnalyticalLOD</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Total-ve</td><td rowspan=1 colspan=1>Max RFUMean-ve</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Total-ve</td><td rowspan=1 colspan=1>Max RFUMean-ve</td><td rowspan=1 colspan=1>Total+ve</td><td rowspan=1 colspan=1>Max RFUMean+ve</td><td rowspan=1 colspan=1>Total-ve</td><td rowspan=1 colspan=1>Max RFUMean-ve</td><td rowspan=1 colspan=1>TotalPositives(%)</td><td rowspan=1 colspan=1>95%ConfidenceInterval</td></tr><tr><td rowspan=1 colspan=1>UVT in Q&quot; Swab</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>700.3</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1476.0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>6/90(6.7%)</td><td rowspan=1 colspan=1>2.5% - 13.9%</td></tr><tr><td rowspan=1 colspan=1>Diluent</td><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>807.0</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>890.3</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>15.8</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>572.8</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>9.5</td><td rowspan=1 colspan=1>44/90(48.9%)</td><td rowspan=1 colspan=1>38.2% - 59.7%</td></tr><tr><td rowspan=2 colspan=1>Q&quot; Swab Diluent</td><td rowspan=1 colspan=1>1:100</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>29*</td><td rowspan=1 colspan=1>4.3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>458.5</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>781.8</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>5.3</td><td rowspan=1 colspan=1>6/89(6.7%)</td><td rowspan=1 colspan=1>2.5% - 14.1%</td></tr><tr><td rowspan=1 colspan=1>1:10</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>872.4</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>852.5</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>862.5</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>8.8</td><td rowspan=1 colspan=1>37/90(41.1%)</td><td rowspan=1 colspan=1>30.8% - 52%</td></tr></table>

No    h.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Extraction Control (EC): Each Extraction Tube contains a dried fluorescently labeled oligonucleotide that is incorporated into each reaction. This Extraction Control (EC) is labeled with a different dye than that used for detection of the HSV specific target and is used to confirm the validity of the extraction process. The EC is dried in the Extraction Tubes and is rehydrated upon addition of the specimen and extraction reagents. At the end of the extraction process, the EC fluorescence is monitored by the BD Viper instrument and an automated algorithm is applied to both the EC and HSV specific signals to report specimen results as positive, negative, or EC failure. The Extraction Tubes containing the EC are sold separately from the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays.

External Controls: External Control Reagents are adjunct reagents to the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays. Each Control Set for the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays contains 24 HSV $\mathrm { Q } ^ { \mathrm { x } }$ Positive Control Tubes and 24 HSV $\mathrm { Q } ^ { x }$ Negative Control Tubes. The Control Set is intended for Quality Control use with The BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays when tested with the BD Viper™ System in Extracted Mode. The HSV $\mathrm { Q } ^ { \mathrm { x } }$ Positive Control will monitor for substantial reagent failure only. The HSV $\mathrm { Q } ^ { \mathrm { x } }$ Negative Control monitors for reagent and/or environmental contamination.

# d. Detection limits:

A Limit of Detection (LOD) study was performed in-house in accordance with CLSI EP-17-A (Protocols for Determination of Limits of Detection and Limits of Quantitation: Approved Guideline). The LOD of each of the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Assays was determined by diluting HSV1 strain VR-539 and HSV2 strain VR-734 in representative matrix at varying concentrations of viral particles to create an LOD panel consisting of six target levels. Labial swab matrix was selected as representative as it was the most challenging matrix for the HSV $\mathrm { Q } ^ { x }$ Assays. The labial swab matrix was prepared by collecting labial swab specimens from HSV negative donors and expressing the swabs in $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent or UVT. The $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent containing the labial swab specimens was spiked directly with HSV in order to prepare the LOD panel for the $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent specimen type. The UVT containing the labial swab specimens was mixed 1:4 $\mathbf { \Pi } ( \mathbf { v } / \mathbf { v } )$ with $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent prior to spiking with HSV. Data were generated for each target level (66 to 72 replicates per level)

and the resulting MaxRFU values were analyzed to determine positive rates at each level. LOD was determined as the lowest level of virus detected $\ge 9 5 \%$ of the time. The LOD for the UVT specimen type corresponds to the level of HSV in UVT prior to diluting 1:4 with $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent. Consequently, the LOD for the UVT specimen will be higher than the LOD for the $\mathrm { Q } ^ { x }$ Swab Diluent specimen.

The LOD values for the HSV1 $\mathrm { Q } ^ { x }$ Assay with Human Herpes Virus type 1 strain VR-539 in labial swab matrix were determined to be 23 viral particles/mL $\left( \mathrm { v p / m L } \right)$ for $\mathrm { Q } ^ { x }$ Swab Diluent and $8 5 ~ \mathrm { v p / m L }$ for UVT. These values were also determined in units of $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ as $7 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ( $\mathrm { \Delta Q ^ { \mathrm { { x } } } }$ Swab Diluent) and $2 5 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (UVT).

The LOD values for the HSV2 $\mathrm { Q } ^ { x }$ Assay with Human Herpes Virus type 2 strain VR-734 in labial swab matrix were determined to be ${ 8 4 } ~ \mathrm { v p / m L }$ (35 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) for $\mathrm { Q } ^ { x }$ Swab Diluent and $6 3 5 ~ \mathrm { v p / m L }$ $2 6 2 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ) for UVT.

An LOD study was also carried out with strains of HSV1 (VR-260, VR-733, VR-735) and HSV2 (VR-540) diluted in clean UVT and $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent.

The results from the LOD studies are summarized in the following tables.

HSV1 in Labial Swab Matrix   

<table><tr><td rowspan=1 colspan=1>HSV1 Strain</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>LOD (vp/mL)</td><td rowspan=1 colspan=1>LOD(TCID50/mL)</td></tr><tr><td rowspan=2 colspan=1>VR-539</td><td rowspan=1 colspan=1>Qx Swab Diluent</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>25</td></tr></table>

# HSV1 in Clean Media

<table><tr><td rowspan=1 colspan=1>HSV1 Strain</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>LOD (vp/mL)</td></tr><tr><td rowspan=2 colspan=1>VR-260</td><td rowspan=1 colspan=1>Q* Swab Diluent</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=2 colspan=1>VR-733</td><td rowspan=1 colspan=1>Q* Swab Diluent</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>185</td></tr><tr><td rowspan=2 colspan=1>VR-735</td><td rowspan=1 colspan=1>Qx Swab Diluent</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>130</td></tr></table>

HSV2 in Labial Swab Matrix   
HSV2 in Clean Media   

<table><tr><td rowspan=1 colspan=1>HSV1 Strain</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>LOD (vp/mL)</td><td rowspan=1 colspan=1>LOD(TCID50/mL)</td></tr><tr><td rowspan=2 colspan=1>VR-734</td><td rowspan=1 colspan=1>Q* Swab Diluent</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>635</td><td rowspan=1 colspan=1>262</td></tr></table>

<table><tr><td rowspan=1 colspan=1>HSV1 Strain</td><td rowspan=1 colspan=1>Medium</td><td rowspan=1 colspan=1>LOD (vp/mL)</td></tr><tr><td rowspan=2 colspan=1>VR-540</td><td rowspan=1 colspan=1>Qx Swab Diluent</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>BD UVT</td><td rowspan=1 colspan=1>380</td></tr></table>

# e. Analytical specificity:

DNA from 57 organisms listed in the table below was extracted on the BD Viper System and tested with the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays. All potential cross-reactive species were tested at approximately $\mathrm { 1 x 1 0 } ^ { \mathbf { \bar { 8 } } }$ Colony Forming Units/mL for bacteria and yeast or ${ \dot { 1 } } \mathbf { \dot { x } } \mathbf { \dot { l } } 0 ^ { 6 }$ Plaque Forming Units/mL or greater for viruses, except where noted. The HSV $\mathrm { Q } ^ { x }$ Assays did not cross-react with any of the organisms tested.

# Cross Reactivity Panel

<table><tr><td colspan="1" rowspan="1">Actinomyces israelii</td><td colspan="1" rowspan="1">Human papillomavirus 16a</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Human papillomavirus 18a</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Kingella kingae</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis serovar H</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis serovar LGV-2</td><td colspan="1" rowspan="1">Listeria monocytogenes</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Moraxella lacunata</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Mycobacterium tuberculosisa</td></tr><tr><td colspan="1" rowspan="1">HHV-5 Cytomegalovirus (CMV)a</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Neisseria meningitidis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Propionibacterium acnes</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 11)a</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">HHV-4 Epstein Barr virusb</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (strain K1)</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Hepatitis B Virusa</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">HHV-6 (Roseolovirus)a</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">HHV-6B (Roseolovirus) a</td><td colspan="1" rowspan="1">Treponema pallidum</td></tr><tr><td colspan="1" rowspan="1">HHV-7 (Roseolovirus) a</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">HHV-8 (Rhadinovirus) a</td><td colspan="1" rowspan="1">Varicella Zoster Virus (HHV-3)a</td></tr><tr><td colspan="1" rowspan="1">Human Immunodeficiency Virus 1 (HIV-1)b</td><td colspan="1" rowspan="1">Veillonella parvula</td></tr><tr><td colspan="1" rowspan="1">Human Immunodeficiency Virus 2 (HIV-2) b</td><td colspan="1" rowspan="1">Herpes Virus Type 1 (HSV1)b, c</td></tr><tr><td colspan="1" rowspan="1">Human papillomavirus 6a</td><td colspan="1" rowspan="1">Herpes Virus Type 2 (HSV2)b, d</td></tr><tr><td colspan="1" rowspan="1">Human papillomavirus 11a</td><td colspan="1" rowspan="1"></td></tr></table>

a Genomic DNA tested at $1 \mathrm { x } 1 0 ^ { 6 }$ DNA copies/mL b Viruses tested at $\mathrm { 1 x 1 0 ^ { 8 } }$ viral particles/mL c Tested as a cross-reactant in the HSV2 $\mathrm { Q } ^ { \mathrm { x } }$ Assay only d Tested as a cross-reactant in the HSV1 $\mathrm { Q } ^ { \mathrm { x } }$ Assay only

# $f .$ Interference Studies:

The performance of the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays was evaluated in the presence of potentially interfering substances which may be encountered in clinical specimens. Potentially interfering substances were spiked into $\mathrm { Q } ^ { x }$ Swab Diluent and UVT specimen matrices, in both the presence and the absence of HSV1 and HSV2 (69 $\mathrm { H S V 1 \ v p / m L }$ and $2 5 2 ~ \mathrm { v p / m L }$ HSV2 in $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent matrix, and 255 $\mathrm { H S V 1 \ v p / m L }$ and $1 9 0 5 ~ \mathrm { v p / m L }$ HSV2 in UVT swab matrix). These levels are three times the Limits of Detection. The strains of HSV1 and HSV2 used in this study were VR-539 and VR-734, respectively. Testing was carried out in replicates of thirty-two. No interference was observed for any of the substances at the concentrations listed in the following table.

Potentially Interfering Substances Panel   

<table><tr><td>Q* Swab Diluent</td><td>UVT</td></tr><tr><td>Blood (2.5% v/v) Seminal fluid (2.5% v/v) Mucus (2.5% v/v)</td><td>Blood (3.3% v/v) Seminal fluid (3.3% v/v) Mucus (3.3% v/v)</td></tr><tr><td>Feces (2.5%)</td><td>Feces (3.3%)</td></tr><tr><td>Urine (2.5% v/v) Cornstarch (2.5%)</td><td>Urine (3.3% v/v) Cornstarch (3.3%)</td></tr><tr><td>Over the counter vaginal products and contraceptives (2.5% v/v)</td><td>Over the counter vaginal products and contraceptives (3.3% v/v)</td></tr><tr><td>Over the counter cold sore products (2.5%</td><td>Over the counter cold sore products (3.3%</td></tr><tr><td>v/v)</td><td>v/v)</td></tr><tr><td>Hemorrhoidal cream (2.5% v/v)</td><td>Hemorrhoidal cream (3.3% v/v)</td></tr><tr><td>Prescription vaginal and anti-viral treatments (2.5% v/v)</td><td>Prescription vaginal and anti-viral treatments (3.3% v/v)</td></tr><tr><td>Leukocytes (1x106 cells/mL)</td><td>Leukocytes (1x106 cells/mL)</td></tr><tr><td>1x106 viral particles/mL HSV1 (when tested</td><td>1x106 viral particles/mL HSV1 (when tested</td></tr><tr><td>in the HSV2 Q&quot; assay) 1x106 viral particles/mL HSV2 (when tested in the HSVi Q&quot; assay)</td><td>in the HSV2 Q&quot; assay) 1x106 viral particles/mL HSV2 (when tested in the HSVi Q&quot; assay)</td></tr></table>

# g. Assay cut-off:

The “cutoff value” represents the MaxRFU score at which a “positive” reaction reaches a value above the background or baseline of a “negative” reaction. A preliminary estimate of the cutoff values for the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays was determined in-house by ROC analysis of 827 total assay results from 223 unique patients. The preliminary cutoff value determined for the HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays was 125 MaxRFU. The preliminary EC cutoff value that was selected was the same as previously established for the CTQ and GCQ Assays on the BD Viper System. The performance of the HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays with the preliminary assay cutoff (125 MaxRFU) and EC cutoff was evaluated in three phases at external clinical sites: a cutoff analysis (to verify the preliminary assay and EC cutoffs prior to the main clinical trial), a clinical trial (to validate the performance of the assays in different clinical settings), and a reproducibility study. These three phases confirmed that the preliminary cutoff values for assay and EC were acceptable.

# h. Carry-over/Cross-Contamination

An internal study was conducted to evaluate the risk of producing a false positive result in either the same run on the BD Viper System in Extracted Mode (within run cross-contamination) or in a subsequent run (between run carry-over). Testing was carried out using negative and positive samples on three BD Viper Systems. Negative samples consisted of UVT in $\mathrm { Q } ^ { \mathrm { x } }$ Swab Diluent. HSV positive samples consisted of UVT medium diluted in $\mathrm { Q } x$ Swab Diluent and spiked with $1 0 ^ { 5 } \mathrm { v p / m L }$ of HSV2 strain VR-734. HSV2 was selected as the representative analyte due to its higher prevalence in anogenital lesion specimens. Two runs with alternating columns of positive and negative samples were carried out on each instrument to determine the rate of cross-contamination. The overall rate of cross-contamination (i.e., with alternating columns of positive and negative samples) was $0 . 3 6 \%$ (4/1104). A third run was then carried out on each instrument with negative samples to determine the rate of carry-over contamination. The overall rate of carry-over contamination (i.e., carry-over between successive runs when the prevalence was $50 \%$ in the previous run) was $0 . 0 9 \%$ (1/1104). Crosscontamination and carry-over rates across the three BD Viper Systems are summarized in the following table.

Cross Contamination and Carryover Contamination (UVT)   

<table><tr><td rowspan=2 colspan=1>BD ViperSystem</td><td rowspan=1 colspan=3>Cross-Contamination</td><td rowspan=1 colspan=3>Carry-over Contamination</td></tr><tr><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>PositiveResults</td><td rowspan=1 colspan=1>PercentPositive</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>368</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1104</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.09</td></tr></table>

2. Comparison studies:

a. Method comparison with reference methods:

The clinical performance evaluation was done against a gold standard/reference method i.e., cell culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

b. Matrix Comparison:

N/A

3. Clinical studies:

Clinical performance of the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays was established by testing external anogenital lesion swab specimens collected from 564 compliant male and female subjects attending family planning, OB/GYN, and sexually transmitted disease clinics at nine geographically diverse clinical sites in North America. Fifty-six subjects were excluded from the data analysis due to age requirement violations, antiviral use in the last 21 days, opting to withdraw from the study after initially consenting, transport errors, informed consent issues, failing to meet the inclusion/exclusion criteria, collection errors, shipping errors, or labeling errors. Therefore, the final data analysis included 508 compliant subjects.

Two swab specimens were collected from external anogenital lesions from each of the 508 compliant subjects. The first specimen collected was always the Universal Viral Transport specimen followed by the BD ProbeTec $\mathrm { Q } ^ { x }$ Collection Kit for Endocervical or Lesion Specimens $\mathrm { \Delta } ^ { \mathrm { \prime } }$ Swab Diluent specimen). The UVT specimen was aliquoted into a $\mathrm { Q } ^ { x }$ Swab Diluent Tube (UVT in $\mathrm { Q } ^ { x }$ Swab Diluent) and a cryovial (vial suitable for $- 7 0 ~ ^ { \circ } \mathrm { C }$ storage). The remaining UVT specimen was sent to one of two laboratories for viral culture and typing by fluorescent antibody staining. The $\mathrm { Q } ^ { x }$ Swab Diluent specimen and the UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimen were transported to one of the three BD Viper testing laboratories where they were tested using the BD ProbeTec HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays on the BD Viper System. The cryovial containing the UVT aliquot was sent to a laboratory for PCR testing.

Calculations of sensitivity and specificity were based on the BD ProbeTec HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays results for $\mathrm { Q } ^ { x }$ Swab Diluent specimens and UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens as compared to a commercially available viral culture and fluorescent antibody typing procedure as the reference method. A PCR method was used for analysis of discrepant results from the ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays and the reference culture/typing method.

The lesion was considered to be positive for HSV1 if the reference viral culture/typing method was positive for HSV and typed as HSV1. The lesion was considered negative for HSV1 if the reference viral culture/typing method was negative for HSV. There were a total of seven subjects that failed to provide a reference viral culture/typing result leaving 501 evaluable subjects. Of the 501 subjects with a reference viral culture/typing result, 189 were positive for HSV2. The viral culture/typing method cannot detect HSV1 in specimens that are positive for HSV2 and thus the 189 HSV2 positive specimens were not included in the analysis of HSV1 performance. A total of four $\mathrm { Q } ^ { x }$ Swab Diluent specimens were not included in the HSV1 analysis due to insufficient volume, aliquot error, collection error, or extraction error. The remaining specimens, 312 UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens and $3 0 8 \mathrm { Q } ^ { x }$ Swab Diluent specimens, were available to calculate sensitivity and specificity

The lesion was considered to be positive for HSV2 if the reference viral culture/typing method was positive for HSV and typed as HSV2. Subjects were considered negative for HSV2 if the reference viral culture/typing method was negative for HSV or the reference viral culture/typing method was positive for HSV and typed as HSV1. There were a total of 7 subjects that failed to provide a reference viral culture typing result leaving 501 evaluable subjects. There were three noncompliant $\mathrm { Q } ^ { x }$ Swab Diluent specimens due to insufficient volume, aliquot error or collection error. The remaining specimens, 501 UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens and $4 9 8 \mathrm { Q } ^ { x }$ Swab Diluent specimens, were available to calculate sensitivity and specificity.

Sensitivity and specificity by specimen type for the HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays are summarized in the following two tables.

HSV1 $\mathbf { Q } ^ { x }$ Assay Performance Compared to Viral Culture (by specimen type)   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Performance Compared to Viral Culture</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td></tr><tr><td rowspan=1 colspan=1>UVT inQr SwabDiluent</td><td rowspan=1 colspan=1>312</td><td rowspan=1 colspan=1>96.8% (60/62)3</td><td rowspan=1 colspan=1>(88.8% -99.6%)</td><td rowspan=1 colspan=1>97.6%(244/250) 4</td><td rowspan=1 colspan=1>(94.8% - 99.1%)</td></tr><tr><td rowspan=1 colspan=1>Q&quot; SwabDiluent</td><td rowspan=1 colspan=1>3082</td><td rowspan=1 colspan=1>96.7% (59/61)3</td><td rowspan=1 colspan=1>(88.7% -99.6%</td><td rowspan=1 colspan=1>95.1%(235/247)5</td><td rowspan=1 colspan=1>(91.7% - 97.5%)</td></tr></table>

1 The reference viral culture used in this study was unable to detect co-infected specimens. Only if the specimen is negative for HSV2 is it typed for HSV1. The number of samples used for HSV1 analysis equals the number of samples with a reference viral culture typing result (501) minus the number of samples positive for HSV2 by the reference method (189). $^ 2 \mathrm { \ A }$ total of four $\mathrm { Q } ^ { x }$ Swab Diluent specimens were not included in the HSV1 analysis due to insufficient volume, aliquot error, collection error, or extraction error. 3 Both subjects that were identified as positive for HSV1 by viral culture/typing were negative for HSV1 with the HSV1 $\mathrm { Q } ^ { \mathrm { x } }$ Assay and PCR. Both the HSV $\mathrm { Q } ^ { \mathrm { x } }$ Assays and the PCR assay identified the subjects as positive for HSV2. 4 There were six UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens that were identified as negative for HSV1 by viral culture/typing but were positive for both the HSV1 $\mathrm { Q } ^ { \mathrm { x } }$ Assay and the PCR assay.

5 There were twelve $\mathrm { Q } ^ { x }$ Swab Diluent specimens that were identified as negative for HSV1 by viral culture/typing but were positive with the HSV1 $\mathrm { Q } ^ { \mathrm { x } }$ Assay. Seven of these specimens were also positive with the PCR assay.

HSV2 $\mathbf { Q } ^ { x }$ Assay Performance Compared to Viral Culture (by specimen type)   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Performance Compared to Viral Culture</td></tr><tr><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>95% C.I.</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>95% C.I.</td></tr><tr><td rowspan=1 colspan=1>UVT inQrSwabDiluent</td><td rowspan=1 colspan=1>501</td><td rowspan=1 colspan=1>98.4% (186/189)6</td><td rowspan=1 colspan=1>(95.4% -99.7%</td><td rowspan=1 colspan=1>83.7%(261/312)7,9</td><td rowspan=1 colspan=1>(79.1% - 87.6%)</td></tr><tr><td rowspan=1 colspan=1>Qr SwabDiluent</td><td rowspan=1 colspan=1>498</td><td rowspan=1 colspan=1>98.4% (186/189)6</td><td rowspan=1 colspan=1>(95.4% -99.7%)</td><td rowspan=1 colspan=1>80.6%(249/309)8,9</td><td rowspan=1 colspan=1>(75.7% - 84.8%)</td></tr></table>

6 All three subjects that were identified as positive for HSV2 by viral culture/typing were negative for HSV2 with both the HSV2 $\mathrm { Q } ^ { x }$ Assay and PCR assay. Both the HSV $\mathrm { Q } ^ { x }$ Assays and the PCR assays identified all three subjects as positive for HSV1.

7 There were 51 UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens that were identified as negative for HSV2 by viral culture/typing but were positive by the HSV2 $\mathrm { Q } ^ { \mathrm { x } }$ Assay. PCR was positive for HSV2 for 46 of these 51 specimens.

8 There were $6 0 \mathrm { Q } ^ { x }$ Swab Diluent specimens that were identified as negative for HSV2 by viral culture/typing but were positive by the HSV2 $\mathrm { Q } ^ { \mathrm { x } }$ Assay. PCR was positive for HSV2 for 49 of these 60 specimens.

9 Detection of nucleic acid by PCR and the HSV2 $\mathrm { Q } x$ Assay in samples that were culture negative could indicate detection of nonviable viral particles.

The results obtained with the HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays during the clinical performance testing were analyzed to generate frequency distributions of MaxRFU values. The data summarized in the following four tables indicate that positive and negative results obtained with the HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays were well separated from the cutoff of 125 MaxRFU.

![](images/850ae82b9eb182ebd7d38480a30619ec928507c2f4b786d7e5889dfd92c06033.jpg)

![](images/0cf0818c88d6f065027d3d8f5e036ac91d47757c14281e1853c090e78d2bc1c3.jpg)  
Frequency Distribution of MaxRFU for the HSV2 $\mathbf { Q } ^ { x }$ Assay (UVT in $\mathbf { Q } ^ { \mathbf { x } }$ Swab Diluent)

![](images/a1fd680c56a8a0a4e1bbf1c678ea56ba61b4f0b50ab413fb7e0d6f78fdeb8c29.jpg)  
Frequency Distribution of MaxRFU for the HSV2 $\mathbf { Q } ^ { x }$ Assay ( $\mathbf { \nabla } [ \mathbf { Q } ^ { \mathbf { x } }$ Swab Diluent)

4. Clinical cut-off:

A preliminary value for the clinical cutoff was estimated by ROC analysis of 827 assay results from 223 unique patients. The preliminary cutoff value determined for the HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays was 125 MaxRFU. The performance of the HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays with the preliminary cutoff (125 MaxRFU) was evaluated in three phases at external clinical sites: a cutoff analysis (to verify the preliminary cutoff prior to the main clinical trial), a clinical trial (to validate the performance of the assays with the cuoff value), and a reproducibility study. These studies confirmed that the cutoff value of 125 MaxRFU was appropriate for incorporation into the algorithm software used by the BD Viper System in Extracted Mode.

# 5. Expected values/Reference range:

The prevalence observed during the multi-center clinical trial (Feburary/2010 – August/2010) for anogenital lesion specimens was estimated using the BD ProbeTec HSV $\mathrm { Q } ^ { x }$ Amplified DNA Assays. The prevalence for HSV1 for the UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens was $1 3 . 2 \%$ (66/501) and for the $\mathrm { Q } ^ { x }$ Swab Diluent specimens was $1 4 . 3 \%$ (71/497). The prevalence for HSV2 for the UVT in $\mathrm { Q } ^ { x }$ Swab Diluent specimens was $4 7 . 3 \%$ (237/501) and for the $\mathrm { Q } ^ { x }$ Swab Diluent specimens was $4 9 . 4 \%$ (246/498).

The following table summarizes the number of positive results and total number of specimens for each assay and specimen type by the age of the subject.

HSV $\mathbf { Q } ^ { x }$ Assays Distribution by Age Group: All Clinical Sites   

<table><tr><td rowspan=3 colspan=1>Age Range</td><td rowspan=1 colspan=6>Specimen Type</td></tr><tr><td rowspan=1 colspan=3>UVT in Q&quot; Swab Diluent</td><td rowspan=1 colspan=3>Q&quot; Swab Diluent</td></tr><tr><td rowspan=1 colspan=1>HSV1 QrPositive</td><td rowspan=1 colspan=1>HSV2 QrPositive</td><td rowspan=1 colspan=1>TotalNumber ofSpecimens</td><td rowspan=1 colspan=1>HSV1 QrPositive</td><td rowspan=1 colspan=1>HSV2 QrPositive</td><td rowspan=1 colspan=1>TotalNumber ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>18 to 25 years</td><td rowspan=1 colspan=1>48 (18.5 %)*</td><td rowspan=1 colspan=1>115 (44.4 %)</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>49 (19.0%)</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>258</td></tr><tr><td rowspan=1 colspan=1>26 to 30 years</td><td rowspan=1 colspan=1>11 (10.1%)</td><td rowspan=1 colspan=1>55 (50.5%)</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>12 (11.1%)</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>108</td></tr><tr><td rowspan=1 colspan=1>31 to 35 years</td><td rowspan=1 colspan=1>3 (5.3%)</td><td rowspan=1 colspan=1>31 (54.4%)</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>5 (8.8%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>57</td></tr><tr><td rowspan=1 colspan=1>36 to 40 years</td><td rowspan=1 colspan=1>4 (16.0%)</td><td rowspan=1 colspan=1>13 (52.0%)</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>4 (16.0%)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>41 to 45 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8 (40.0%)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>46 to 50 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6 (37.5%)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>1 (6.7%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>51 to 55 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6 (66.7%)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>56 to 60 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1 (33.3%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>61 to 65 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>66 to 70 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2 (100.0%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>237</td><td rowspan=1 colspan=1>501</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>246</td><td rowspan=1 colspan=1>498***</td></tr><tr><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>13.2%</td><td rowspan=1 colspan=1>47.3%</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>14.3%</td><td rowspan=1 colspan=1>49.4%</td><td rowspan=1 colspan=1>NA</td></tr></table>

\* Percentages for a specific age range were calculated from the number of positives (HSV1 or HSV2) for

that age range and the total number of specimens for that age range.

\*\*Of the 501 evaluable subjects, three noncompliant $\mathrm { Q } ^ { x }$ swab diluent specimens were not available for the HSV1 and HSV2 data analysis. They were noncompliant due to insufficient volume, aliquot error, or collection error.

\*\*\*One $\mathrm { Q } ^ { x }$ Swab Diluent specimen resulted in a HSV1 extraction error on the BD Viper System leaving a total of 497 HSV1 results for this specimen type.

Hypothetical positive and negative predictive values (PPV & NPV) for the HSV1 and HSV2 $\mathrm { Q } ^ { x }$ Assays were calculated based on hypothetical prevalence and overall sensitivity and specificity per specimen type as determined in the clinical trial. For the HSV1 $\mathrm { Q } ^ { x }$ Assay, these calculations are based upon an overall sensitivity and specificity of $9 6 . 8 \%$ and $9 7 . 6 \%$ , respectively, for the UVT in $\mathrm { Q } ^ { x }$ Swab Diluent and $9 6 . 7 \%$ and $9 5 . 1 \%$ , respectively, for the $\mathrm { Q } ^ { x }$ Swab Diluent. For the HSV2 $\mathrm { Q } ^ { x }$ Assay, the hypothetical PPV and NPV are based upon an overall sensitivity and specificity of $9 8 . 4 \%$ and $8 3 . 7 \%$ , respectively, for the UVT in $\mathrm { Q } ^ { x }$ Swab Diluent and $9 8 . 4 \%$ and $8 0 . 6 \%$ , respectively, for the $\mathrm { Q } ^ { x }$ Swab Diluent. These calculations are summarized in the following table.

HSV $\mathbf { Q } ^ { x }$ Assays Distribution by Age Group: All Clinical Sites   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>UVT in Qx Swab Diluent</td><td rowspan=1 colspan=4>Q&quot; Swab Diluent</td></tr><tr><td rowspan=2 colspan=1>Prevalence(%)</td><td rowspan=1 colspan=2>HSV1</td><td rowspan=1 colspan=2>HSV2</td><td rowspan=1 colspan=2>HSV1</td><td rowspan=1 colspan=2>HSV2</td></tr><tr><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>45.1</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>9.4</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>68.0</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>99.9</td><td rowspan=1 colspan=1>50.9</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>21.1</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>81.8</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>40.1</td><td rowspan=1 colspan=1>99.8</td><td rowspan=1 colspan=1>68.7</td><td rowspan=1 colspan=1>99.6</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>99.8</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>91.0</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>60.1</td><td rowspan=1 colspan=1>99.5</td><td rowspan=1 colspan=1>83.1</td><td rowspan=1 colspan=1>99.1</td><td rowspan=1 colspan=1>55.9</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>94.5</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>72.1</td><td rowspan=1 colspan=1>99.2</td><td rowspan=1 colspan=1>89.4</td><td rowspan=1 colspan=1>98.5</td><td rowspan=1 colspan=1>68.5</td><td rowspan=1 colspan=1>99.2</td></tr><tr><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>96.4</td><td rowspan=1 colspan=1>97.9</td><td rowspan=1 colspan=1>80.1</td><td rowspan=1 colspan=1>98.7</td><td rowspan=1 colspan=1>92.9</td><td rowspan=1 colspan=1>97.7</td><td rowspan=1 colspan=1>77.2</td><td rowspan=1 colspan=1>98.7</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>96.8</td><td rowspan=1 colspan=1>85.8</td><td rowspan=1 colspan=1>98.1</td><td rowspan=1 colspan=1>95.2</td><td rowspan=1 colspan=1>96.6</td><td rowspan=1 colspan=1>83.5</td><td rowspan=1 colspan=1>98.1</td></tr></table>

# N. Instrument Name:

BD ViperTM System in Extracted Mode

# O. System Descriptions:

1. Modes of Operation:

The instrument modules and instrumentation components remain unchanged from the previously cleared instrumentation (K081824, K081825)

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.