# 510(k) Summary Idaho Technology Inc. FilmArray RP Test System

Introduction: According to the requirements of 21 CFR 807.92, the following information provides ufficient detail tounderstand the basis or a determination substantial equivalenc

# Submitted by:

Idaho Technology Inc 390 Wakara Way Salt Lake City, UT 84108

Telephone: 801-736-6354   
Facsimile: 801-588-0507

Contact: Beth Lingenfelter, ext. 407

Date Submitted: February 2, 2011

# Device Name and Classification:

Trade Name: FilmArray RP System   
Regulation Number: 21 CFR 866.3980   
Classification Name: Respiratory Viral Panel Multiplex Nucleic Acid Assay

# Predicate Device:

K063765, K081483, K091677 - Luminex $^ \mathrm { \textregistered }$ xTAGTM Respiratory Viral Panel (RVP).

# Intended Use:

The FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following virus types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus HKU1, Coronavirus NL63, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype $2 0 0 9 \mathrm { H } 1$ , Influenza B, Parainfluenza virus 3, Rhinovirus/Enterovirus, and Respiratory Syncytial Virus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and

Idaho Technology Inc. ${ \mathfrak { s l } } \ 0 ( { \mathfrak { k } } )$ EMPY FilmArray Respiratory Panel System

symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. Negative results do not preclude respiratory infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. Positive results do not rule out bacterial infection or co-infection with other organisms. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) and clinical presentation must be taken into considertation in order to obtain the final diagnosis of respiratory infection.

Due to seasonal prevalence, performance characteristics for Influenza A/H1, Influenza A/H3, Influenza $\mathrm { A } / 2 0 0 9 \mathrm { H } 1$ , and Influenza B were established primarily with retrospective clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g. cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g. FDA-cleared molecular test or cell culture).

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

# Device Description:

The FilmArray RP System is multiplex nucleic acid test system composed of the FilmArray instrument, the FilmArray software (preinstalled on a laptop computer) and the FilmArray RP pouch. The FilmArray RP pouch contains freeze-dried reagents to perform nucleic acid purification, reverse transcription, and nested, multiplex PCR with DNA melt analysis. The Respiratory Panel (RP) pouch identifies 12 common and emerging viral respiratory pathogens (see Table 1).

Table 1. Viruses Detected by the FilmArray Respiratory Panel   

<table><tr><td>Viral Respiratory Pathogens</td><td></td></tr><tr><td>Influenza A</td><td rowspan="3"></td></tr><tr><td>H1 subtype H3 subtype 2009 H1 subtype</td></tr><tr><td>Influenza B Adenovirus Coronavirus HKU1 Coronavirus NL63</td></tr></table>

A test is initiated by loading Hydration Solution and an unprocessed patient nasopharyngeal swab (NPS) specimen (i.e. specimen mixed with Sample Buffer) into the FilmArray RP pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification and initiating the run.

The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the reverse transcription reactions, the PCR reactions, and the melting curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green®Plus, Idaho Technology). This second master mix solution, is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The second stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the $2 ^ { \mathfrak { n } \mathfrak { d } }$ stage PCR, the array is interrogated by melting curve analysis for the detection of signature amplicons denoting the presence of specific viral or bacterial targets A digital camera placed in front of the second stage PCR captures fluorescent images of the PCR reactions in real time.

The FilmArray software automatically interprets the results of each DNA melting curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Substantial Equivalence:

The Luminex $\textsuperscript { \textregistered }$ xTAGTM RVP is a PCR-based system for detecting the presence of viral nucleic acid in nasopharyngeal swabs collected from individuals with signs and symptoms of respiratory illness. The similarities and differences between the xTAG RVP and the FilmArray RP are outlined below.

Table 2. Similarities between the xTAG RVP and the FilmArray RP   

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel TestSystem</td><td rowspan=1 colspan=1>Luminex® xTAGT RVP</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1,Influenza A subtype H3, Influenza B,Respiratory Syncytial Virus, humanMetapneumovirus, Adenovirus,Parainfluenza 3, andRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>RNA/DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>multiplex nucleic acid</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td rowspan=1 colspan=1>Same</td></tr></table>

Table 3. Differences between the xTAG RVP and the FilmArray RP   

<table><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel TestSystem</td><td colspan="1" rowspan="1">Luminex xTAGTM RVP</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Can distinguish Influenza A subtype 2009HI from Influenza A subtype H1.Also detects Coronavirus NL63 andCoronavirus HKU1.</td><td colspan="1" rowspan="1">Can distinguish Respiratory Syncytial VirusType A from Respiratory Syncytial VirusType B. Detects Parainfluenza virus 1 andParainfluenza virus 2.</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followed by highresolution melting analysis to confirmidentity of amplified product.</td><td colspan="1" rowspan="1">Multiplex RT-PCR and multiplex TSPEfollowed by Fluorescence-activated sortingof labeled beads coupled to streptavidin-conjugated biotinylated products</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray Instrument</td><td colspan="1" rowspan="1">PCR ThermocyclerLuminex 100 IS or 200 system</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">Less than 1 hour</td><td colspan="1" rowspan="1">Approximately 8 hours</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation and reportgeneration. User cannot access raw data.</td><td colspan="1" rowspan="1">Semi-automated test interpretation. Usermust review all "no call" results todetermine cause and retesting strategy.</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample Processing is automated in theFilmArray instrument.</td><td colspan="1" rowspan="1">Up front sample processing is required toextract nucleic acid.</td></tr><tr><td colspan="1" rowspan="1">Element</td><td colspan="1" rowspan="1">FilmArray Respiratory Panel TestSystem</td><td colspan="1" rowspan="1">Luminex® xTAGT RVP</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Reagents are stored at room temperature.</td><td colspan="1" rowspan="1">Reagents stored at 4°C and -20°C.</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td colspan="1" rowspan="1">Internal control added to each sample.External control processed with each batchof samples.</td></tr><tr><td colspan="1" rowspan="1">UserComplexity</td><td colspan="1" rowspan="1">Moderate/Low</td><td colspan="1" rowspan="1">High</td></tr></table>

# Summary of Performance Data

# Clinical Performance

The clinical performance of the FilmArray RP system was established during a prospective study at 3 U.S. clinical sites over a 6 month time period (December 2009 through May 2010). Subjects with signs and/or symptoms of respiratory infection were invited to participate. Upon obtaining informed consent, NPS samples were collected for FilmArray and comparator testing; a second respiratory sample was collected from each subject for viral culture reference testing. A total of 857 subjects were initially enrolled in the study and four were withdrawn. Table 4, provides a summary of demographic information for the 853 subjects that participated in the prospective study.

Table 4. Demographic Summary for FilmArray RP Prospective Study   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>449 (53%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>404 (47%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>484 (57%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>95 (11%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>190 (22%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>84 (10%)</td></tr></table>

Each NPS specimen was tested with the FilmArray RP. The performance of the FilmArray RP was evaluated by comparing the FilmArray RP test result for each member of the panel with the appropriate comparator/reference methods shown in Table 5.

Table 5. Reference/Comparator Methods Used to Assess FilmArray RP Performance   

<table><tr><td rowspan=1 colspan=1>Organism/Virus</td><td rowspan=1 colspan=1>Reference/Comparator Method(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=5 colspan=1>Viral culture followed by DFAidentification</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>FluA/H1 subtyping</td><td rowspan=3 colspan=1>1 PCR test of influenza A positive viralculture with bi-directional sequenceconfirmation2</td></tr><tr><td rowspan=1 colspan=1>FluA/H3 subtyping</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1subtyping</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=5 colspan=1>2 PCR tests of patient specimen with bi-directional sequence confirmation3</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr></table>

1Peormanef the FilmArray RP system detect Adenovirs, Flu A, Flu B, PIV, or RSV, respeively was cr vculu ollwy nientnrvius, I I B, Parainfluenza virus 3, or RSV positives, respecively, were consiered as any sample that tested positive or V rAeiu Ina Inza  z viu   RVnei, epvely c hat iu Iz u R respectively, by viral culture followed by DFA testing.

Perorc  the FilmAay RP syst dete InluezaAH1, AH3, o A/2009 H1, respecivel, was c t vialculture ollowe byne analyticy valiate R assay withbidirecnal sequence confimatn. The casss we e   ifnt ucohat l he iArr ass(N of the comparator PCR assays overlapped any FilmArray amplicon sequence even  the same gene was targeted. r InzaAHAHr A/200 H posiive speively wee cone as ny sal that tes piv that matched Influenza A/H1, A/H3, or A/2009 H sequences deposited in the National Center or Biotehnolgy Inoation (NCB) GenBank database (ww.ncbi.nlmnih.gov), espectively, wit acceptable E-values.True Inunza A/H1, A/H3, o A/2009 H negatives, repectively were considered as any smple that ested negativ by the respective Influenza A subtype specific PCR assay.

3Peormance  the FilmArrayR syste detcting Human Rhinovirus, Enterovirus, Coronavirus NL63, Coroavius HKU1, or Human Metapneumovirus, respectively, was compared to a predetermined algorithm that used composite comparatormethods.The methods consist of two analytically validate Rassays followed by bi-diretional sequencing. The comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s). None f the comparator CR assays overlapped any FilmArray amplicon sequence even  he same gene was targeted. True" Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus positives, respectively were considered as any sample that had bi-directional sequencig data meeting pre-defined quality acceptanc criteria that matched Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus sequences deposited in the National Center for Biotechnology Inoation (NCBl) GenBank database (ww.ncbinlm,nih.gov) respectively, with acceptableE-values. True" Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus negatives, respectively, were considered as any sample that tested negative by both of the comparator PCR assays.

A total of 853 specimens were evaluated in this study. Clinical sensitivity or positive percent agreement (PPA) was calculated as $1 0 0 \% \mathbf { x }$ $\left( \mathrm { T P } / \mathrm { T P } + \mathrm { F N } \right)$ .True positive (TP) indicates that both the FilmArray RP and comparator method had a positive result for this specific analyte and false negative (FN) indicates that the FilmArray result was negative while the comparator result was positive. Specificity was calculated as $1 0 0 \% \mathbf { x }$ (TN /TN $+ \mathrm { F P } )$ Tru negative (TN) indicates that both the FilmArray RP and the comparator method had negative results and a false positive (FP) indicates that the FilmArray RP result was positive but the comparator results was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. The results are summarized in Table 6.

Table 6. Clinical Sensitivity and Speciicity r the FilArray R Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>24/27 </td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>70.8 - 97.7%</td><td rowspan=1 colspan=1>812/826 b</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>97.2 - 99.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A                        .</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>55.5 - 99.8%</td><td rowspan=1 colspan=1>841/843 d</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2 -100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6 - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6 - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>51.8 - 99.7%</td><td rowspan=1 colspan=1>841/844 </td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0 - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6 - 100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9 - 99.9%</td><td rowspan=1 colspan=1>819/829</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>97.8 - 99.4%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.2 - 100%</td><td rowspan=1 colspan=1>714/8018</td><td rowspan=1 colspan=1>89.1%</td><td rowspan=1 colspan=1>86.8 - 91.2%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9 - 99.9%</td><td rowspan=1 colspan=1>827/829</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.1 - 100%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9 - 99.9%</td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.6 - 100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>88/93</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>87.9 - 98.2%</td><td rowspan=1 colspan=1>754/760</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.3 - 99.7%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>190/205</td><td rowspan=1 colspan=1>92.7%</td><td rowspan=1 colspan=1>88.2 - 95.8%</td><td rowspan=1 colspan=1>613/648</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>92.6 - 96.2%</td></tr></table>

The FilmArray RP system detected Adenovirus in $1 / 3$ false negative specimens when retested. The Adenovirus in the retested specimens were identified as species C and species B. species C, two as species B, and one as species E. V     l  nal i assay. Influenza A viruses ${ 2 0 0 9 } \mathrm { H l }$ $2 / 2$ false pstiv A culture alone) using sequence analysis with an alternate assay. ThFiAy se V  egaiv he . 8RSV viues we enf in /87 alse posiv pemes us idirecinal sequce nalysis.

The FilmArray RP system detected a total of 81 mixed infections in the prospective clinical evaluation. This represents $1 6 . 1 \%$ of the total positive specimens (81/502). Seventy-six (76/81; $9 3 . 8 \% )$ were double infections, and 5 (5/81; $6 . 2 \% )$ were triple infections. The total number of test results comiprising these co-infections was 167. The single most common co-infection (21/81; $2 5 . 9 \% )$ was Human Rhinovirus/Enterovirus with Respiratory Syncytial Virus. These viruses were the most prevalent in the tested population. Out of the 81 co-infections, 47 contained one or more analytes that had not been detected with the reference/comparator methods, i.e. discrepant co-infection.

Table 7. Mixed Infections Detected by FilmArray RP and Prevalence of Individual Analytes in Mixed Infections   

<table><tr><td rowspan=1 colspan=1>Organism Combinations</td><td rowspan=1 colspan=1>NumberofPositiveSamples</td><td rowspan=1 colspan=1>Percentageof TotalSamplesTested</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Number ofMixedInfections</td><td rowspan=1 colspan=1>Prevalencein MixedInfections</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + RSV</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2.46%</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>19.75%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.06%</td><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>12.35%</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.94%</td><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>16.05%</td></tr><tr><td rowspan=1 colspan=1>hMPV + HRV/Entero</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.82%</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>27.16%</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.47%</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>69.14%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + HRV/entero</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.47%</td><td rowspan=1 colspan=1>FluA/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>FluA/H1-2009</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>FluA/H3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU! + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>17.28%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>36.</td><td rowspan=1 colspan=1>44.44%</td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=10 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero +PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + RSV</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoVNL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + HRV/Entero + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero +RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td></tr><tr><td rowspan=1 colspan=1>Total Mixed Infections</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>9.50%</td></tr></table>

Table 8 provides the prevalence and age distribution of FilmArray RP detected analytes in the clinical study. The most prevalent analytes were Human Rhinovirus/Enterovirus, Respiratory Syncytial Virus, and Human Metapneumovirus; these three analytes comprised $78 \%$ of all positive results. Positive results were obtained across all age groups tested; however, the majority was detected in children 5 years or younger.

Table 8. Prevalence and Age Distribution of Analytes in the Clinical Study   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Total(Prevalence)</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>6-21years</td><td rowspan=1 colspan=1>22-49years</td><td rowspan=1 colspan=1>≥ 50years</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38 (4.5%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKUI</td><td rowspan=1 colspan=1>25 (2.9%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23 (2.7%)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>94 (11%)</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>225 (26.4%)</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Influenza A (all subtypes)</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>33 (3.9%)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>139 (16.3%)</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td></tr></table>

Several analytes, such as influenza viruses were either not encountered in the clinical study or had a low prevalence. To supplement the results of the prospective clinical study, an evaluation of preselected archived samples was performed.

# Testing of Preselected Archived Specimens

In addition to the prospective clinical study, archived clinical NPS specimens were also tested using the FilmArray RP. The specimens were selected because they had previously tested positive for one of the following organisms: Adenovirus, Enterovirus, Influenzas A/H1, 2009 H1N1, and H3, Influenza B, and Parainfluenza Virus 3, or had been negative by previous testing methods. Prior to testing with the FilmArray RP, the presence or absence of the analyte of interest was confirmed in each specimen using analyte specific PCR and bi-directional sequencing. Of 400 specimens, 349 were confirmed to contain the analyte of interest (or lack thereof for negative specimens). The specimens were organized into "test panels" and randomized such that the users testing the samples with the FilmArray RP were blinded as to the expected test result. Each panel contained specimens known to be positive and negative for the specific analyte being evaluated allowing the calculation of a positive percent agreement (PPA) and a negative percent agreement (NPA). A summary of the available demographic information of the tested samples is provided in Table 9 and the results of the FilmArray testing are presented in Table 10.

Table 9. Demographic Summary of FilmArray RP Archived Specimen Study   

<table><tr><td rowspan=1 colspan=2>Total Specimens</td><td rowspan=1 colspan=1>349</td></tr><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>82 (23.5%)</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>79 (22.6%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>188( 53.9%)</td></tr><tr><td rowspan=4 colspan=1>Age</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>83.0</td></tr><tr><td rowspan=5 colspan=1>Age Range</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>89 (25.5%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>35 (10.0%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>23 (6.6%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>14 (4.0%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>188(53.9%)</td></tr></table>

a o i     oy a psp s haen $< 1$ yrs to 21 yrs.

Table 10. FilmArray Archived Specimen Performance Data Summary   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)</td></tr><tr><td rowspan=1 colspan=1>TP/TP +FN</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.2 - 100%</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.7 - 100%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>78.0 - 99.9%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.0 - 100%</td></tr><tr><td rowspan=1 colspan=1>FluA/H1</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.1 - 100%</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1 - 100%</td></tr><tr><td rowspan=1 colspan=1>FluA/H1-2009</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.7 - 100%</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1 - 100%</td></tr><tr><td rowspan=1 colspan=1>FluA/H3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.4 - 100%</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.5 - 100%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.4 - 100%</td><td rowspan=1 colspan=1>129/129</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2 - 100%</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3 - 100%</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.1 - 100%</td></tr></table>

# Selected Analytic Studies

# Limit of Detection

The analytical sensitivity or Limit of Detection (LoD) for each FilmArray RP analyte (except for Coronavirus HKU1) was determined by testing limiting dilutions of live, quantified viruses. The LoD for Coronavirus HKU1 was determined by testing limiting dilutions of a clinical specimen containing a high viral load of Coronavirus HKU1. LoD is defined as the lowest concentration at which the analyte is consistently detected (detection in $2 9 5 \%$ of samples tested). Simulated NPS sample matrix (cultured human cells in VTM) was spiked with one or more analytes and at least 20 replicates were tested at the LoD concentration. The LoD for each FilmArray RP analyte is listed in Table 11.

Table 11. LoD for Analytes Detected by FilmArray RP   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain Identification</td><td rowspan=1 colspan=1>Limit of Detection</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Serotype 1 (Species C)</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical Specimen (Type B)</td><td rowspan=1 colspan=1>1.9 x 106 RNA copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NR-470</td><td rowspan=1 colspan=1>5 TCID5/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Type A1 (hMPV-16, IA10-2003)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>30,000 TCID5g/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>AI</td><td rowspan=1 colspan=1>1 TCID5/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>200 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>2,000 TCIDs/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1</td><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>100 TCID5/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>5 TCID5/mL</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>60 TCIDs/mL</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>2 TCIDs/mL</td></tr></table>

Corovius HKU1 was quantied by anon-FiArray real-ime CR assy aainst  tanar curve c Coronavirus HKU1 RNA transcript to obtain quantification of the viral nucleic acid in the clinical specimen (RNA copies/mL).

NOTE: Most analytes were re-grown and quantified in $\mathrm { T C I D } _ { 5 0 }$ $5 0 \%$ Tissue Culture Infectious Dose). The unit $\mathrm { T C I D } _ { 5 0 }$ is a measure fectiviyctotoxiciyrathe than umbe  aniss orcpnuclciVr $\mathrm { \Delta T C I D _ { 5 \circ } / m L }$ EPY o of the same organism.

# Analytical Reactivity (Inclusivity)

The analytical reactivity of the FilmArray RP system assays was evaluated with an inclusivity panel consisting of 99 strains or isolates that represent the genetic, temporal, and geographic diversity of the FilmArray RP analytes. The tested organisms include: 17 Adenovirus, 4 Coronavirus (3 HKU1 and 1 NL63), 10 human Metapneumovirus, 12 Enterovirus, 14 Rhinovirus, 22 Influenza A (including 10 Influenza A/H1, 3 Influenza A/2009 H1 and 9 Influenza A/H3), 11 Influenza B, 3 Parainfluenza virus 3 and 6 Respiratory Syncytial Virus. Each organism was initially tested in a simulated NPS sample matrix at or near the system LoD. Higher concentrations were tested if the analyte was not detected at LoD.

Each of the 99 strains tested in this study were reactive with the FilmArray RP system. Reactivity of the FilmArray RP system with Adenovirus C serotype 6 was 10,000-fold less than the system LoD due to sequence variation in the region targeted by the FilmArray RP Adenovirus assay.

Additional clinical data analyses, and in silico analyses were also carried out to supplement the testing of the inclusivity panel.

Results from inclusivity testing are presented below. The concentration and multiple of LoD at which each strain was detected by the FilmArray RP system is indicated.

Table 12. Results of Inclusivity Testing for Adenovirus   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Serotype (Isolate)</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Multiple of LoD</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="8">B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">300 TCID5/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">7a</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">7d2 (Iowa/2001)</td><td colspan="1" rowspan="1">300 TCID5/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">7h (lowa/1999)</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">11 (Wisconsin/2005)</td><td colspan="1" rowspan="1">3,000 TCID5/mL</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">14 (Missouri/2005)</td><td colspan="1" rowspan="1">300 TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">21 (Missouri/2005)</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">34 (Texas/2005)</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="4">C</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">300 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">2 (New York/2004)</td><td colspan="1" rowspan="1">30,000 TCID50/mL</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">3,000 TCIDs/mL</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">6 (Colorado/2005)*</td><td colspan="1" rowspan="1">3,000,000 TCIDs/mL</td><td colspan="1" rowspan="1">10,000x</td></tr><tr><td colspan="1" rowspan="1">D</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">3,000 TCID5g/mL</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="2">E</td><td colspan="1" rowspan="1">4a (S Carolina/2004)</td><td colspan="1" rowspan="1">300 TCID5g/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">4p3 (New Jersey/2005)</td><td colspan="1" rowspan="1">300 TCID5/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">41 (Indiana/2004)</td><td colspan="1" rowspan="1">300 TCIDs0/mL</td><td colspan="1" rowspan="1">1x</td></tr></table>

efficiently than with other adenovirus serotypes.

# Supplemental Adenovirus Reactivity Information (Clinical Data and in silico analyses):

A subset of prospective and retrospective archived clinical specimens that were FilmArray positive for Adenovirus was subjected to PCR and bi-directional sequence analysis. BLAST analysis of the sequence data obtained for a region of the Adenovirus polymerase gene identified adenovirus species B (serotypes 3, $3 + 1 1 { \mathsf p }$ , 7, 16 and 21), species C (serotypes 1, 2, and 5), and species E (serotype 4) in these clinical specimens. Species B, C and E are the most common Adenovirus species associated with respiratory illness. Species A, D, and F (often associated with conjunctivitis and gastroenteritis) were not identified in clinical specimens.

In addition to laboratory testing, bioinformatics resources were also used to predict reactivity of additional Adenovirus species and serotypes with the FilmArray RP System. Simulated reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). Table 13 lists the adenovirus types that were not tested by the FilmArray system either in analytical or clinical testing. Based on the bioinformatics analysis, the FilmArray RP system is predicted to react with all of the indicated Adenovirus species and serotypes.

Table 13. Simulated FilmArray RP Reactivity with Untested Adenovirus Serotypes   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td rowspan=1 colspan=1>Virus   1</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=15 colspan=1>HumanAdenovirus</td><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>AB330093</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>DQ149610</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>AB330097</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>AB052912</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>DQ149643</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=10 colspan=1>D</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>AB330090</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>DQ149615</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>DQ149616</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>DQ149617</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>AB330098</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>DQ149618</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>DQ149619</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>DQ149620</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>DQ149621</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>DQ149622</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Table 13. Simulated FilmArray RP Reactivity with Untested Adenovirus Serotypes   

<table><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=24 colspan=1></td><td rowspan=22 colspan=1></td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>DQ149623</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>DQ149624</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>DQ149625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>DQ149626</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>DQ149627</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>DQ149628</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>DQ149629</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>DQ149630</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>DQ149631</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>DQ149632</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>DQ149633</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>DQ149634</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>DQ149635</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>DQ149636</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>DQ149637</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>DQ149638</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>DQ149639</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>DQ149640</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>AB330129</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>DQ149641</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>DQ149642</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>FJ169625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>AB330121</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>DQ923122</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Table 14. Results of Inclusivity Testing for Coronaviruses   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain / Isolate</td><td rowspan=1 colspan=1>Concentration ab</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=3 colspan=1>HKU1</td><td rowspan=1 colspan=1>Clinical Sample #1120           •</td><td rowspan=1 colspan=1>2.08 x 10RNA copies/mL</td><td rowspan=1 colspan=1>1.1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample # 6123</td><td rowspan=1 colspan=1>1.41 x 104 TCID50/mL~1.9 x 106RNA copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample #6213 (Type B)c</td><td rowspan=1 colspan=1>1.9 x 10}$RNA copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>BEI Resources NR-470</td><td rowspan=1 colspan=1>50 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

Virus contained in Clinical Sample #6123 was grown in culture and quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ by infectivity assay. b Quantiicationf the viral RNA contained in clinical specimens containing Coronavirus HKU1 was performed using real-time RT-PCR against a standard curve generated from a synthetic RNA template. c Phylogenetic information for Coronavirus HKU1 Clinical Sample $\# 6 2 1 3$ was obtained from bi-directional sequence analysis of the nucleocapsid (N) gene.

'able 15. Results of Inclusivity Testing for Human Metapneumovirus   

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=2>Strain ID</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=2 colspan=1>AI</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>IA3-2002</td><td rowspan=1 colspan=1>2 TCIDsq/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>IA10-2003</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>IA14-2003</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>IA27-2004</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=4 colspan=1>B1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Peru2-2002</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Peru3-2003</td><td rowspan=1 colspan=1>2 TCIDs0/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>IA7-2003</td><td rowspan=1 colspan=1>2 TCIDs0/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>IA18-2003</td><td rowspan=1 colspan=1>2 TCIDs0/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>B2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Peru6-2003</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>IA16-2003</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

Table 16. Results of Inclusivity Testing for Enterovirus and Rhinovirus   

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=3 colspan=1>Enterovirus A</td><td rowspan=1 colspan=1>Coxsackievirus A10 ATCC VR-168</td><td rowspan=1 colspan=1>30,000 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71 ATCC VR-1432</td><td rowspan=1 colspan=1>1:30,000dilution of stock</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>9400 TCID5/mL b</td><td rowspan=1 colspan=1>&lt;lx</td></tr><tr><td rowspan=6 colspan=1>Enterovirus B</td><td rowspan=1 colspan=1>Coxsackievirus A9</td><td rowspan=1 colspan=1>9400 TCIDsg/mL b</td><td rowspan=1 colspan=1>&lt;1x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B3</td><td rowspan=1 colspan=1>30,000 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus B4</td><td rowspan=1 colspan=1>30,000 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>30,000 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>9400 TCIDs/mL b</td><td rowspan=1 colspan=1>&lt;Ix</td></tr><tr><td rowspan=1 colspan=1>Echovirus 11</td><td rowspan=1 colspan=1>300,000 TCID5g/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=2 colspan=1>Enterovirus C</td><td rowspan=1 colspan=1>Coxsackievirus A21 /KuykendallATCC VR-850</td><td rowspan=1 colspan=1>30,000 TCIDsg/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Coxsackievirus A24 DN-19ATCC VR-583</td><td rowspan=1 colspan=1>30,000 TCIDsq/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Enterovirus D</td><td rowspan=1 colspan=1>Enterovirus 68 (F02-3607 corn)ATCC VR-1197</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>Ix</td></tr><tr><td rowspan=6 colspan=1>Rhinovirus Av</td><td rowspan=1 colspan=1>A1</td><td rowspan=1 colspan=1>1 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A2 (HGP) ATCC VR-482</td><td rowspan=1 colspan=1>10 TCID5/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A7 (68-CV11) ATCC VR-1601</td><td rowspan=1 colspan=1>1 TCID5g/mL</td><td rowspan=1 colspan=1>Ix</td></tr><tr><td rowspan=1 colspan=1>A16 (11757) ATCC VR-283</td><td rowspan=1 colspan=1>10 TCID5/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A34 (137-3) ATCC VR-507</td><td rowspan=1 colspan=1>1 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A57 (Ch47) ATCC VR-1600</td><td rowspan=1 colspan=1>100 TCID5/mL</td><td rowspan=1 colspan=1>100x</td></tr></table>

Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration </td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>A77 (130-63) ATCC VR-1187</td><td rowspan=1 colspan=1>1 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A85 (50-525-CV54) ATCC VR-1195</td><td rowspan=1 colspan=1>10 TCID5/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=6 colspan=1>Rhinovirus B</td><td rowspan=1 colspan=1>B3 (FEB) ATCC VR-483</td><td rowspan=1 colspan=1>1 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B14 (1059) ATCC VR-284</td><td rowspan=1 colspan=1>1 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B17 (33342) ATCC-282</td><td rowspan=1 colspan=1>100 TCIDsg/mL</td><td rowspan=1 colspan=1>100x</td></tr><tr><td rowspan=1 colspan=1>B27 (5870) ATCC VR-1137</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B42 (56822) ATCC VR-338</td><td rowspan=1 colspan=1>10 TCIDs/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B83 (Baylor 7) ATCC VR-1 193</td><td rowspan=1 colspan=1>1 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

The LoD for Enterovirus is $\overline { { 3 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } } }$ The LoD for Rhinovirus is $1 ~ \mathrm { T C I D } _ { 5 \mathrm { o } } / \mathrm { m L }$ . Sains wee testd below he oD concentration due o a lesser concenration  virus in the cultue ui.

# Supplemental Human Rhinovirus/Enterovirus Reactivity Information (clinical data and in silico analyses):

In addition to the analytical inclusivity testing, BLAST analysis was performed on sequence data $\zeta ^ { \prime }$ UTR) obtained from prospective and retrospective archived clinical specimens that were FilmArray positive for Human Rhinovirus/Enterovirus. The following species and subtypes were identified in clinical specimens:

Enterovirus Species A: Enterovirus serotype 71 Coxsackievirus A2 and A6   
Enterovirus Species B: Echovirus serotypes 3, 6, 7, 11, 15, 21 and 30 Coxsackievirus B1, B3 and A9 Enterovirus serotypes 81 and 88

Rhinovirus Species A: Human Rhinovirus serotypes 1B, 8, 9,10, 13, 19, 21, 22, 23, 28, 30, 32, 34, 38, 39, 40, 46, 47, 49, 51, 54, 56, 58, 59, 61, 62, 66, 68, 75, 77, 78, 80, 82, 98 and 100

Rhinovirus Species B: Human Rhinovirus serotypes 27, 69, 83 and 91

Rhinovirus Species C: at least 3 individual strains and 12 distinct isolatesa

Enterovirus species C includes Poliovirus types 1-3 (PV1, PV2, and PV3). Simulated reactivity of the FilmArray RP Human Rhinovirus/Enterovirus assays with Enterovirus Species C Poliovirus sequences was generated using a bioinformatics approach. Alignment of assay primer sequences with the GenBank sequences indicates the FilmArray RP assay will react with Poliovirus types 1, 2 and 3, giving a Human Rhinovirus/Enterovirus result.

Table 17. Simulated Reactivity of the FilmArray RP HRV and Entero Assays with Poliovirus Sequences   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArray RP Result</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 1 strainCHN8264c/GZ/CHN/2004</td><td rowspan=1 colspan=1>FJ769385</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 2, complete genome</td><td rowspan=1 colspan=1>AY177685</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 3 strain IRA10853, completegenome</td><td rowspan=1 colspan=1>EU684056</td><td rowspan=1 colspan=1>Human Rhinovirus/Entcrovirus</td></tr></table>

Table 18. Results of Inclusivity Testing for Influenza A   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Multiple of LoD</td></tr><tr><td colspan="1" rowspan="10">Influenza A(H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">200 TCID5/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/01CDC#2001701117</td><td colspan="1" rowspan="1">1:300 a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">ANew Caledonia/20/99</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">2000 TCIDs/mL</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">200 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="3">Influenza A(H1N1-2009)</td><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">100 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Swine NY/02/2009</td><td colspan="1" rowspan="1">100 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="4">Influenza A(H3N2)</td><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">5 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">5 TCIDs0/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/NewYork/55/2005CDC#2005705561</td><td colspan="1" rowspan="1">1:300,000 </td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">5 TCIDs/mL:</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5 TCIDs/mL</td><td colspan="1" rowspan="1">Ix</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">50 TCIDs/mL</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">5 TCID5s/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Alice (vaccine) A/England/42/72ATCC VR-776</td><td colspan="1" rowspan="1">5 TCIDs/mL</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">MRC-2 Recombinant strainATCC VR-777</td><td colspan="1" rowspan="1">5 TCID5/mL</td><td colspan="1" rowspan="1">1x</td></tr></table>

Strain was not re-cultured and titered. Dilutions of the original culture/titer from the CDC were used. The HA titer for A/Hawai/15/01 is unknown and the HA titer for A/NewYork/55/2005 was 256.

# Supplemental Reactivity Information for Influenza Strains of Human, Swine and Avian Origin (analytical testing and in silico analyses):

Results of testing viral isolates or nucleic acid from viral culture indicate that the FilmArray RP pan-influenza and subtyping assays react with virus of swine and avian origin as expected (Table 19).

Table 19. Results of Inclusivity Testing with Swine and Avian Isolates of Influenza A   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate / Strain</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>FilmArrayResult</td></tr><tr><td rowspan=2 colspan=1>Swine</td><td rowspan=1 colspan=1>HINI</td><td rowspan=1 colspan=1>Influenza A/Swine/1976/31</td><td rowspan=1 colspan=1>~1 x 10 6EIDsq/mL</td><td rowspan=2 colspan=1>Influenza A H1 a</td></tr><tr><td rowspan=1 colspan=1>HINI</td><td rowspan=1 colspan=1>Influenza A/Swine/Iowa/15/30</td><td rowspan=1 colspan=1>~1 x 107 EID5/mL</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>Kilbourne F63A/NWS/34 (HA) x A/RockefellerInstitute/5/57 (NA)</td><td rowspan=1 colspan=1>14.8 ng RNA b</td><td rowspan=1 colspan=1>Influenza A HI</td></tr></table>

\* No reactivity was observed with the $2 0 0 9 \mathrm { H } 1$ subtyping assay or other FilmArray RP assays. b Purified and quantified RNA from avian influenza culture was obtained from BEI Resources.

Laboratory testing of influenza A strains was supplemented with in silico predictions of reactivity using bioinformatics and sequence alignments between FilmArray RP assay primers and sequences for influenza A strains of human, swine, and avian origin. For each strain, multiple (3) GenBank IDs were evaluated, corresponding to the gene segments targeted by the FilmArray RP assays (matrix (MA), non-structural (NS) and hemagglutinin(HA)). Simulated reactivity was determined based on the number and location of mismatches in the targeted region. The strains listed in Table 20 are predicted to react with the FilmArray RP pan-influenza A and H1, H1-2009 or H3 subtyping assays as indicated.

Table 20. Simulated Reactivity of FilmArray Influenza A Assays with Human, Swine, and Avian Influenza Strains   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArray RPReactivity</td></tr><tr><td rowspan=33 colspan=1>Human</td><td rowspan=3 colspan=1>H1N1</td><td rowspan=3 colspan=1>A/California/UR06-0393/2007(HIN1)</td><td rowspan=1 colspan=1>CY026540</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY026543</td></tr><tr><td rowspan=1 colspan=1>CY026539</td></tr><tr><td rowspan=3 colspan=1>H1N1-2009</td><td rowspan=3 colspan=1>A/Aalborg/INS133/2009(H1N1)</td><td rowspan=1 colspan=1>CY063606</td><td rowspan=3 colspan=1>Influenza A H1-2009</td></tr><tr><td rowspan=1 colspan=1>CY063610</td></tr><tr><td rowspan=1 colspan=1>CY063607</td></tr><tr><td rowspan=3 colspan=1>HIN2</td><td rowspan=3 colspan=1>A/New York/297/2003(H1N2)</td><td rowspan=1 colspan=1>CY002668</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY002665</td></tr><tr><td rowspan=1 colspan=1>CY002664</td></tr><tr><td rowspan=3 colspan=1>H2N2</td><td rowspan=3 colspan=1>A/Albany/20/1957(H2N2)</td><td rowspan=1 colspan=1>CY022013</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>CY022014</td></tr><tr><td rowspan=1 colspan=1>CY022017</td></tr><tr><td rowspan=3 colspan=1>H3N2</td><td rowspan=3 colspan=1>A/Boston/38/2008(H3N2)</td><td rowspan=1 colspan=1>CY044581</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY044584</td></tr><tr><td rowspan=1 colspan=1>CY044580</td></tr><tr><td rowspan=6 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/Cambodia/R0405050/2007(H5N1)</td><td rowspan=1 colspan=1>HQ200572</td><td rowspan=6 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>HQ200573</td></tr><tr><td rowspan=1 colspan=1>FJ225472</td></tr><tr><td rowspan=3 colspan=1>A/Hong Kong/486/97(H5N1)</td><td rowspan=1 colspan=1>AF084281</td></tr><tr><td rowspan=1 colspan=1>AF255368</td></tr><tr><td rowspan=1 colspan=1>AF115289</td></tr><tr><td rowspan=3 colspan=1>H7N2</td><td rowspan=3 colspan=1>A/New York/107/2003(H7N2)</td><td rowspan=1 colspan=1>EU587368</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>EU587373</td></tr><tr><td rowspan=1 colspan=1>EU587374</td></tr><tr><td rowspan=3 colspan=1>H7N3</td><td rowspan=3 colspan=1>A/Canada/rv504/2004(H7N3)</td><td rowspan=1 colspan=1>CY015006</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY015007</td></tr><tr><td rowspan=1 colspan=1>CY015010</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=3 colspan=1>A/Netherlands/219/03(H7N7)</td><td rowspan=1 colspan=1>AY340089</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>AY342422</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AY338459</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=3 colspan=1>A/Hong Kong/1073/99(H9N2)</td><td rowspan=1 colspan=1>AJ404626</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>AJ278647</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AJ278649</td></tr><tr><td rowspan=12 colspan=1>Swine</td><td rowspan=3 colspan=1>HIN1</td><td rowspan=3 colspan=1>A/swine/Wisconsin/1/1971(H1N1)</td><td rowspan=1 colspan=1>CY022414</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY022417</td></tr><tr><td rowspan=1 colspan=1>CY022413</td></tr><tr><td rowspan=6 colspan=1>H1N2</td><td rowspan=3 colspan=1>A/swine/Hong Kong/NS857/2001(H1N2)</td><td rowspan=1 colspan=1>GQ229348</td><td rowspan=3 colspan=1>Influenza A HI</td></tr><tr><td rowspan=1 colspan=1>GQ229350</td></tr><tr><td rowspan=1 colspan=1>GQ229347</td></tr><tr><td rowspan=3 colspan=1>A/swine/Sweden/1021/2009(H1N2)</td><td rowspan=1 colspan=1>GQ45135</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ495136</td></tr><tr><td rowspan=1 colspan=1>GQ495132</td></tr><tr><td rowspan=3 colspan=1>HSN1</td><td rowspan=3 colspan=1>A/swine/East Java/UT6010/2007(H5N1)</td><td rowspan=1 colspan=1>HM440124</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>HM440111</td></tr><tr><td rowspan=1 colspan=1>HM440123</td></tr><tr><td rowspan=6 colspan=1>Avian</td><td rowspan=6 colspan=1>H2N2</td><td rowspan=3 colspan=1>A/chicken/New York/13828-3/1995(H2N2)</td><td rowspan=1 colspan=1>CY014822</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY014825</td></tr><tr><td rowspan=1 colspan=1>CY014821</td></tr><tr><td rowspan=3 colspan=1>A/Japan/305/1957(H2N2)</td><td rowspan=1 colspan=1>CY045804</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY014977</td></tr><tr><td rowspan=1 colspan=1>CY014980</td></tr></table>

Table 20. Simulated Reactivity of FilmArray Influenza A Assays with Human, Swine, and Avian Influenza Strains   

<table><tr><td rowspan=1 colspan=5>nuenza Strains</td></tr><tr><td rowspan=1 colspan=1>lost</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArray RPReactivity</td></tr><tr><td></td><td rowspan=3 colspan=1></td><td rowspan=3 colspan=1>A/Korea/426/1968(H2N2)</td><td rowspan=1 colspan=1>CY031595</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td></td><td rowspan=1 colspan=1>CY031596</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>CY031599</td></tr><tr><td></td><td rowspan=3 colspan=1>H3N1</td><td rowspan=3 colspan=1>A/blue-winged teal/ALB/452/1983(H3N1)</td><td rowspan=1 colspan=1>CY004635</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td></td><td rowspan=1 colspan=1>CY004638</td></tr><tr><td></td><td rowspan=1 colspan=1>CY005940</td></tr><tr><td></td><td rowspan=3 colspan=1>H3N2</td><td rowspan=3 colspan=1>A/American black duck/North Carolina/675-075/2004(H3N2)</td><td rowspan=1 colspan=1>GU051135</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td></td><td rowspan=1 colspan=1>GU051136</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>GU051137</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=3 colspan=1>H3N5</td><td rowspan=3 colspan=1>A/mallard/Netherlands/2/1999(H3N5)</td><td rowspan=1 colspan=1>CY060261</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>CY060264</td></tr><tr><td></td><td rowspan=1 colspan=1>CY060265</td></tr><tr><td></td><td rowspan=3 colspan=1>H3N6</td><td rowspan=3 colspan=1>A/Amcrican black duck/NewBrunswick/25182/2007(H3N6)</td><td rowspan=1 colspan=1>CY047696</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td></td><td rowspan=1 colspan=1>CY047697</td></tr><tr><td></td><td rowspan=1 colspan=1>CY047700</td></tr><tr><td></td><td rowspan=3 colspan=1>H3N7</td><td rowspan=3 colspan=1>A/northemshoveler/California/HKWF1367/2007(H3N7)</td><td rowspan=1 colspan=1>CY033372</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td></td><td rowspan=1 colspan=1>CY033375</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>CY033376</td></tr><tr><td></td><td rowspan=3 colspan=1>H3N8</td><td rowspan=3 colspan=1>A/American blackduck/Washington/699/1978(H3N8)</td><td rowspan=1 colspan=1>GU052299</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td></td><td rowspan=1 colspan=1>GU052302</td></tr><tr><td></td><td rowspan=1 colspan=1>GU052300</td></tr><tr><td></td><td rowspan=3 colspan=1>H4N6</td><td rowspan=3 colspan=1>A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td><td rowspan=1 colspan=1>CY063977</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td></td><td rowspan=1 colspan=1>CY063978</td></tr><tr><td></td><td rowspan=1 colspan=1>CY063981</td></tr><tr><td></td><td rowspan=11 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/rook/Rostov-on-Don/26/2007(H5N1)</td><td rowspan=1 colspan=1>EU814503</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td></td><td rowspan=1 colspan=1>EU814504</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>EU814507</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=3 colspan=1>A/turkey/VA/505477-18/2007(H5N1)</td><td rowspan=1 colspan=1>GU186509</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>GU186510</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>GU186513</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=5 colspan=1>A/chicken/Bangladesh/1151-10/2010(H5N1)</td><td rowspan=3 colspan=1>HQ156765</td><td></td></tr><tr><td></td><td></td></tr><tr><td></td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td></td><td rowspan=1 colspan=1>HQ156766</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>HQ156764</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=3 colspan=1>HSN2</td><td rowspan=3 colspan=1>A/duck/Pennsylvania/10218/1984(H5N2)</td><td rowspan=1 colspan=1>AB295603</td><td rowspan=3 colspan=1>Influenza A(no subtypedtected</td></tr><tr><td></td><td rowspan=1 colspan=1>AB286120</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>AB286652</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=5 colspan=1>H5N3</td><td rowspan=5 colspan=1>A/duck/Singapore/F119/3/1997(H5N3)</td><td rowspan=3 colspan=1>GU052802</td><td></td></tr><tr><td></td><td></td></tr><tr><td></td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td></td><td rowspan=1 colspan=1>GU052803</td><td rowspan=1 colspan=1>−</td></tr><tr><td></td><td rowspan=1 colspan=1>GU052805</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=3 colspan=1>H6N1</td><td rowspan=3 colspan=1>A/duck/PA/486/1969(H6N1)</td><td rowspan=1 colspan=1>EU743286</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>EU743287</td></tr><tr><td></td><td rowspan=1 colspan=1>EU743289</td></tr><tr><td></td><td rowspan=3 colspan=1>H6N2</td><td rowspan=3 colspan=1>A/mallard/Czech Republic/15902-17K/2009(H6N2)</td><td rowspan=1 colspan=1>HQ244430</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td></td><td rowspan=1 colspan=1>HQ244433</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>HQ244434</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=3 colspan=1>H7N7</td><td rowspan=3 colspan=1>A/mallard/Korea/GH171/2007(H7N7)</td><td rowspan=1 colspan=1>FJ750872</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td></td><td rowspan=1 colspan=1>FJ959087</td><td rowspan=1 colspan=1></td></tr><tr><td></td><td rowspan=1 colspan=1>FJ959090</td></tr><tr><td></td><td rowspan=1 colspan=1>H9N2</td><td rowspan=1 colspan=1>A/turkey/Wisconsin/1/1966(H9N2)</td><td rowspan=1 colspan=1>CY014663</td><td rowspan=1 colspan=1>Influenza A</td></tr></table>

Table 20. Simulated Reactivity of FilmArray Influenza A Assays with Human, Swine, and Avian Influenza Strains   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArray RPReactivity</td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>CY014664</td><td rowspan=2 colspan=1>(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY014667</td></tr><tr><td rowspan=3 colspan=1>H10N7</td><td rowspan=3 colspan=1>A/chicken/Germany/N/1949(H10N7)</td><td rowspan=1 colspan=1>GQ176136</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ176135</td></tr><tr><td rowspan=1 colspan=1>GQ176132</td><td rowspan=1 colspan=1>−</td></tr><tr><td rowspan=3 colspan=1>H11N9</td><td rowspan=3 colspan=1>A/duck/Memphis/546/1974(H11N9)</td><td rowspan=1 colspan=1>CY014691</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ257441</td></tr><tr><td rowspan=1 colspan=1>CY014687</td></tr></table>

Table 21. Results of Inclusivity Testing for Influenza B   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>60 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005CDC#2005743348</td><td rowspan=1 colspan=1>1:3,000,00 a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>60 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>600 TCIDs/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72 ATCC VR-823</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62 ATCC VR-295</td><td rowspan=1 colspan=1>60 TCID5g/mL</td><td rowspan=1 colspan=1>Ix</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59 ATCC VR-296</td><td rowspan=1 colspan=1>600 TCID5/mL</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54 ATCC VR-103</td><td rowspan=1 colspan=1>60 TCID5/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45 ATCC VR-102</td><td rowspan=1 colspan=1>6,000 EID50/mL</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40ATCC VR-101</td><td rowspan=1 colspan=1>60 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit RecombinantATCC VR-786</td><td rowspan=1 colspan=1>60 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

Sin was not e-culud nd titdDilutins  terigial cultue/t om e C wee d The HA titer for B/Ohio/01/2005 was 128.

Table 22. Results of Inclusivity Testing for Parainfluenza Virus 3   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain or Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=3 colspan=1>3</td><td rowspan=1 colspan=1>Zeptometrix#0810016CF</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>C-243ATCC VR-93</td><td rowspan=1 colspan=1>500 TCID5/mL</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=1 colspan=1>NIH 47885BEI NR-3233</td><td rowspan=1 colspan=1>100 TCIDs/mL</td><td rowspan=1 colspan=1>10x</td></tr></table>

Table 23. Results of Inclusivity Testing for Respiratory Syncytial Virus   

<table><tr><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain or Source</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix#0810040ACF</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/A2ATCC VR-1540</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/LongATCC VR-26</td><td rowspan=1 colspan=1>2 TCIDs/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>B/9320ATCC VR-955</td><td rowspan=1 colspan=1>2 TCIDsg/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Wash18537/62ATCC VR-1580</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/WV/14617/85ATCC VR-1400</td><td rowspan=1 colspan=1>2 TCIDsg/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

# Analytical Specificity (Cross-reactivity and Exclusivity)

The potential for cross-reactivity between assays contained in the FilmArray RP system was evaluated by testing simulated NPS samples containing high concentrations of respiratory panel viruses (tens to thousands-fold higher than LoD). No cross-reactivity was observed at the concentrations listed in Error! Reference source not found.24.

Table 24. Results of Testing for Cross-Reactivity with FilmArray RP Analytes   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Virus or Bacterium</td><td colspan="1" rowspan="1">Type / Strain</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Multiple of LoD</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Serotype 1(Species C)</td><td colspan="1" rowspan="1">1.00x10³ TCIDsq/mL</td><td colspan="1" rowspan="1">333 x</td></tr><tr><td colspan="1" rowspan="2">Coronavirus</td><td colspan="1" rowspan="1">HKU1 - Type BClinical specimen</td><td colspan="1" rowspan="1">2.78x10 copies/mL</td><td colspan="1" rowspan="1">1,463 x</td></tr><tr><td colspan="1" rowspan="1">NL63NR-470</td><td colspan="1" rowspan="1">5.67x10³ TCID50/mL</td><td colspan="1" rowspan="1">1,134 x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Type A1 - hMPV-16IA10-2003 A1</td><td colspan="1" rowspan="1">8.17x10³ TCID5g/mL</td><td colspan="1" rowspan="1">4,085 x</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">3.40x106 TCIDs0/mL</td><td colspan="1" rowspan="1">113 x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus A1</td><td colspan="1" rowspan="1">5.67x10³ TCID50/mL</td><td colspan="1" rowspan="1">5,670 x</td></tr><tr><td colspan="1" rowspan="7">Influenza AHINI</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00x10³ TCID50/mL</td><td colspan="1" rowspan="1">500 x</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">1.00x10³ TCIDs/mL</td><td colspan="1" rowspan="1">500 x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.00x10° TCIDs/mL</td><td colspan="1" rowspan="1">5,000 x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">4.70x103³ TCIDs/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">4.70x10³3 TCID5/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57</td><td colspan="1" rowspan="1">4.70x10³ TCID50/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">1.39x104 TCID5/mL</td><td colspan="1" rowspan="1">70 x</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">1.39x104 TCIDs/mL</td><td colspan="1" rowspan="1">70 x</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1N1-2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">4.00x10³ TCIDsg/mL</td><td colspan="1" rowspan="1">4,000 x</td></tr><tr><td colspan="1" rowspan="8">Influenza AH3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">8.17x10³ TCIDsg/mL</td><td colspan="1" rowspan="1">1634 x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">940 x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5.67x103³ TCIDs/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.00x10° TCIDs0/mL</td><td colspan="1" rowspan="1">20,000 x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.00x10° TCID5s/mL</td><td colspan="1" rowspan="1">20,000 x</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">4.70x10³ TCID5/mL</td><td colspan="1" rowspan="1">940 x</td></tr><tr><td colspan="1" rowspan="1">A/MRC 2</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">1,634 x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">8.17x10³ TCIDsg/mL</td><td colspan="1" rowspan="1">1,634 x</td></tr><tr><td colspan="1" rowspan="10">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">1.67x104 rCIDs/mL</td><td colspan="1" rowspan="1">278 x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">5.03x104 TCIDs/mL</td><td colspan="1" rowspan="1">838 x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">8.17x10³ TCID5/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">1.00x10° TCIDs0/mL</td><td colspan="1" rowspan="1">1,667 x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">5.67x10³ TCID5/mL</td><td colspan="1" rowspan="1">95x</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1.00x10³ TCIDs0/mL</td><td colspan="1" rowspan="1">1,667 x</td></tr><tr><td colspan="1" rowspan="1">B/HongKong/5/72</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Brigit</td><td colspan="1" rowspan="1">3.50x104 TCIDs0/mL</td><td colspan="1" rowspan="1">583 x</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">1.00x10° TCID5/mL</td><td colspan="1" rowspan="1">10,000 x</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus .</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">1.39x104 TCID50/mL</td><td colspan="1" rowspan="1">6,950 x</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">2.14x104 TCID5/mL</td><td colspan="1" rowspan="1">10,700 x</td></tr></table>

The potential for the FilmArray RP system to cross-react with non-FilmArray RP organisms was evaluated by testing an exclusivity panel consisting of 45 bacteria, 13 viruses, and 1 fungus. These organisms were selected based on their relatedness to FilmArray RP analytes, clinical relevance (cause respiratory symptoms or represent nasopharyngeal flora), or high prevalence within the population (e.g. Herpes Simplex Virus). Negative sample matrix was spiked with bacteria or fungi at a concentration of $\mathbf { \dot { 1 0 ^ { 6 } C F U / m L } }$ and viruses at a concentration between $1 0 ^ { 4 } - 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , or the highest concentration possible. The FilmArray RP system did not cross-react with the exclusivity panel organisms listed in Table 25. One measles virus strain was found to contain Adenovirus, which was detected by the FilmArray RP.

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Strain / Isolate</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Strain / Isolate</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>clinical isolate</td><td rowspan=1 colspan=1>Bocavirus</td><td rowspan=1 colspan=1>Type 1 - Clinical specimen</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>F061</td><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>A747</td><td rowspan=1 colspan=1>Coronavirus SARS</td><td rowspan=1 colspan=1>Zeptometrix -Nucleic Acid</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>E431</td><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>ATCC VR-759</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>Cytomegalovirus (CMV)</td><td rowspan=1 colspan=1>AD-169 (VR-538)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 8467</td><td rowspan=1 colspan=1>Epstein-Barr Virus(EBV)</td><td rowspan=1 colspan=1>B95-8</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>Herpes Simplex Virus</td><td rowspan=1 colspan=1>Type I</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 51445</td><td rowspan=1 colspan=1>Parainfluenza Virus I</td><td rowspan=1 colspan=1>Zeptometrix # 0810014CF</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC BAA-589</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Zeptometrix # 0810015CF</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9340</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Zeptometrix #0810060CF</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 10380</td><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Edmonston</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC BAA-1335</td><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Zeptometrix # 0810025CFa</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>D-UW3</td><td rowspan=1 colspan=1>Mumps</td><td rowspan=1 colspan=1>Zeptometrix # 0810079CF</td></tr><tr><td rowspan=1 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>Fungi</td><td rowspan=1 colspan=1>Strain / Isolate</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiptheriae</td><td rowspan=1 colspan=1>ATCC14779</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Zeptometrix #0801504</td></tr></table>

Table 25. Non-FilmArray RP Exclusivity Panel   
Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Strain / Isolate</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">clinical isolate</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">E431</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 8467</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9797</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 51445</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC BAA-589</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9340</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 10380</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC BAA-1335</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">ATCC14779</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeacheae</td><td colspan="1" rowspan="1">Long Beach 4</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">Philadelphia</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11 (type strain)</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15531</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15293</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15377</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15492</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 29085</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 29342</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 39505</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 49894</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 700825</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M1027 (typestrain)</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Zeptometrix#0801519</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">COL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">RP62A</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix#0801512</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618</td></tr></table>

stc poele posidei esu.The e posiive was ce   y A virus.

# Supplemental Analytical Exclusivity Testing for Influenza Strains of Avian Origin:

Additional analytical exclusivity testing was carried out with either live isolates or purified genomic RNA of avian host influenza A strains with the following results:

Table 26. Results of Exclusivity Testing of Virus or Nucleic Acid from Culture of Avian Influenza A   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate / Strain</td><td rowspan=1 colspan=1>Test Concentration </td><td rowspan=1 colspan=1>FilmArrayResult</td></tr><tr><td rowspan=8 colspan=1>Avian</td><td rowspan=2 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Japan/305/57</td><td rowspan=1 colspan=1>3.3 ng RNA</td><td rowspan=8 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>Kilbourne F38A/Korea/426/68 (HA, NA) x A/PuertoRico/8/34</td><td rowspan=1 colspan=1>6.3 ng RNA</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Vietnam/1203/2004 R-H5</td><td rowspan=1 colspan=1>N/Ab</td></tr><tr><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>A/duck/Pennsylvania/10218/84</td><td rowspan=1 colspan=1>2.5 ng RNA</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>Kilbourne F181A/duck/Singapore/645/97</td><td rowspan=1 colspan=1>247 ng RNA</td></tr><tr><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/NewYork/107/2003</td><td rowspan=1 colspan=1>N/A b</td></tr><tr><td rowspan=1 colspan=1>H7N3</td><td rowspan=1 colspan=1>A/Mallard/Netherlands/12/2000</td><td rowspan=1 colspan=1>N/Ab</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>A/chicken/Germany/N/49</td><td rowspan=1 colspan=1>68 ng RNA</td></tr></table>

'Purified and quantified RNA from avian influenza cultures was obtained from BEI Resources ' Stock virus HA titre from CDC $= 1 2 8$ .Twenty microliters of virus stock tested.

# Precision (Reproducibility)

A multicenter reproducibility study was performed to determine between-site and overall reproducibility of the FilmArray RP system.

Reproducibility testing occurred at three test sites utilizing a panel of twelve simulated NPS specimens spiked with various combinations of live respiratory pathogens (analytes) at three different test levels (high negative (LoD/10), low positive (1X LoD), and medium positive (3X LoD)). On each testing day, two operators at each site tested two aliquots of specimens on two different FilmArray instruments (six specimens per operator per instrument per day). Every specimen was tested four times a day on five days at the three testing sites, for a total of 60 tests per analyte per concentration. A total of 26 lots of reagents and 20 FilmArray instruments were utilized in the reproducibility study. Summary results for each analyte are summarized below.

Results for each analyte are summarized below:

Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing of Single Assay Analytes   

<table><tr><td rowspan=1 colspan=2>AdenovirusSpecies C Serotype 1</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResult</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)900 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>83.74</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>83.33 - 84.26</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>84.23</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>83.95 - 84.60</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>83.03 - 84.13</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>83.93</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.03 - 84.60</td></tr><tr><td rowspan=4 colspan=1>Low Positive(IX LoD)300 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>83.42</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>83.01 - 83.98</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>0.26.</td><td rowspan=1 colspan=1>83.54 - 84.30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td><td rowspan=1 colspan=1>83.62</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.05 - 84.64</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>83.64</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>83.01 - 84.64</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(LoD/10)30 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>68.3%-98.8%</td><td rowspan=1 colspan=1>83.33</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>83.02 - 83.76</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>56.3%-94.3%</td><td rowspan=1 colspan=1>83.71</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>82.93 - 84.29</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% - 72.8%</td><td rowspan=1 colspan=1>83.26</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>82.61 - 83.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>44/60</td><td rowspan=1 colspan=1>16/60</td><td rowspan=1 colspan=1>73.3%</td><td rowspan=1 colspan=1>60.3% - 83.9%</td><td rowspan=1 colspan=1>83.43</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>82.61 - 84.29</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0%- 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100%</td></tr></table>

H result for the "Negative" panel member is negative. High negative samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Coronavirus HKU1Type BClinical Specimen 6123</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResult</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)5.7 x 10  RNAcopies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>75.61</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>75.37 - 76.02</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>76.02</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>75.58 -76.33</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>75.49</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>74.96 - 75.90</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>75.69</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>74.96 - 76.33</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)1.9 x 106 RNAcopies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>75.47</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>74.96 - 75.79</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>75.89</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>75.59 - 76.12</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>75.35</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>74.83 - 75.81</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>75.55</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>74.83 - 76.12</td></tr><tr><td rowspan=4 colspan=1>High NegativebLoD/10)1.9 x 103 RNAcopies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>50.9%-91.3%</td><td rowspan=1 colspan=1>75.51</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>75.06 - 75.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1%-96.8%</td><td rowspan=1 colspan=1>75.85</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>75.26 - 76.23</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1%-99.9%</td><td rowspan=1 colspan=1>75.33</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>74.98 - 75.59</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>51/60</td><td rowspan=1 colspan=1>9/60</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>73.4% - 92.9%</td><td rowspan=1 colspan=1>75.55</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>74.98 - 76.23</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

H result for the "Negative" panel member is negative. b High negative samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Coronavirus NL63BEI Resources NR-470</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResuult</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedT&#x27;m Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)15 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>80.21</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>79.64 -80.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>80.55</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>79.99-81.03</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>80.04</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.67 - 80.62</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.29</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>79.64-81.03</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)5 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>80.08</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.57 -80.42</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>79.98 - 80.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>79.88</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>79.36 -80.40</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>79.14 -80.89</td></tr><tr><td rowspan=4 colspan=1>High NegativebLoD/10)0.5 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>40.8% - 84.6%</td><td rowspan=1 colspan=1>80.08</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>79.21 -80.82</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>45.7%-88.1%</td><td rowspan=1 colspan=1>80.36</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>79.98 - 80.91</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% - 72.8%</td><td rowspan=1 colspan=1>79.91</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>79.24 -80.30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>37/60</td><td rowspan=1 colspan=1>23/60</td><td rowspan=1 colspan=1>61.7%</td><td rowspan=1 colspan=1>48.2% - 73.9%</td><td rowspan=1 colspan=1>80.11</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>79.21 - 80.91</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

H result for the "Negative" panel member is negative. b High negative samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Human MetapneumovirushMPV-16(A1)</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedRResut</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)6 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>77.61</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>77.06-77.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>78.07</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>77.67 - 78.61</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1%-99.9%</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>77.45 -78.11</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% - 100%</td><td rowspan=1 colspan=1>77.82</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>77.06 - 78.61</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)2 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>77.39</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>77.04 - 77.79</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>77.88</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>77.37 -78.11</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=1>77.60</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>77.16-77.99</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>77.62</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>77.04 - 78.11</td></tr><tr><td rowspan=4 colspan=1>High NegativeLD/10)0.2 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% - 96.8%</td><td rowspan=1 colspan=1>77.34</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>77.05 -77.59</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>77.76</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>77.06 -78.11</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1%-80.9%</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>76.74 -77.79</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>48/60</td><td rowspan=1 colspan=1>12/60</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>67.7% -89.2%</td><td rowspan=1 colspan=1>77.50</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>76.74 - 78.11</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% -100%</td></tr></table>

H result for the "Negative" panel member is negative. Hiv  v $2 0 { - } 8 0 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Influenza BB/FL/04/06</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedRResut</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=2>MeanTm</td><td rowspan=1 colspan=2>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)180 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>80.47</td><td rowspan=1 colspan=2>0.26</td><td rowspan=1 colspan=1>79.88 -80.93</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>80.88</td><td rowspan=1 colspan=2>0.31</td><td rowspan=1 colspan=1>80.30-81.30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% -100%</td><td rowspan=1 colspan=2>80.36</td><td rowspan=1 colspan=2>0.32</td><td rowspan=1 colspan=1>79.78 - 80.80</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=2>80.56</td><td rowspan=1 colspan=2>0.40</td><td rowspan=1 colspan=1>79.78 -81.30</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)60 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>80.44</td><td rowspan=1 colspan=2>0.27</td><td rowspan=1 colspan=1>80.00 - 80.92</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>80.79</td><td rowspan=1 colspan=2>0.29</td><td rowspan=1 colspan=1>80.40 -81.33</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% -100%</td><td rowspan=1 colspan=2>80.34</td><td rowspan=1 colspan=2>0.22</td><td rowspan=1 colspan=1>79.77 -80.81</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=2>80.51</td><td rowspan=1 colspan=2>0.37</td><td rowspan=1 colspan=1>79.77 -81.33</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(LD/10)6 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=2>80.42</td><td rowspan=1 colspan=2>0.32</td><td rowspan=1 colspan=1>79.84 -80.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% -72.8%</td><td rowspan=1 colspan=2>80.78</td><td rowspan=1 colspan=2>0.25</td><td rowspan=1 colspan=1>80.40 -81.17</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1>19.1% -64.0%</td><td rowspan=1 colspan=2>80.30</td><td rowspan=1 colspan=2>0.21</td><td rowspan=1 colspan=1>79.79-80.69</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>30/60</td><td rowspan=1 colspan=1>30/60</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>36.8% - 63.2%</td><td rowspan=1 colspan=2>80.50</td><td rowspan=1 colspan=2>0.36</td><td rowspan=1 colspan=1>79.79 -81.17</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=5></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0%- 100%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

H result for the "Negative" panel member is negative. b High negative samples are targeted to be positive $2 0 { \cdot } 8 0 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Parainfuenza Virus 3Zeptometrix #0810016CF</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResut</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=2>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)30 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td><td rowspan=1 colspan=2>81.17</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>80.71-81.86</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>81.48</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>81.03-81.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>80.94</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>80.63 -81.37</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0%-100%</td><td rowspan=1 colspan=2>81.22</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>80.63 - 81.89</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)10 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.97</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>80.36 -81.52</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td><td rowspan=1 colspan=2>81.35</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>80.93-81.79</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% -96.8%</td><td rowspan=1 colspan=2>80.86</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>80.10-81.21</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>3/60</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>86.1% - 99.0%</td><td rowspan=1 colspan=2>81.08</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>80.10 -81.79</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(LoD/10)1 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% - 72.8%</td><td rowspan=1 colspan=2>80.99</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>80.30-81.34</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1>15.4% - 59.2%</td><td rowspan=1 colspan=2>81.29</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>80.61-81.77</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>8.7%- 49.1%</td><td rowspan=1 colspan=2>80.84</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>80.41-81.24</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>22/60</td><td rowspan=1 colspan=1>38/60</td><td rowspan=1 colspan=1>36.7%</td><td rowspan=1 colspan=1>24.6% - 50.1%</td><td rowspan=1 colspan=2>81.05</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>80.30 -81.77</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=2 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0%- 100%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% -100%</td></tr></table>

result for the "Negative" panel member is negative. b High negative samples are targeted to be positive $2 0 { - } 8 0 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Respiratory Syncytial VirusType A</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResult</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)6 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.44</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>79.46 - 80.83</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.86</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>80.41-81.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.39</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>80.08 - 80.91</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.8% - 100%</td><td rowspan=1 colspan=1>80.58</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>79.46 - 81.56</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)2 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>79.82</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>78.93 - 80.62</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>79.47-81.03</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>80.13</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>79.13 - 80.79</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.0% - 100%</td><td rowspan=1 colspan=1>80.10</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>78.93 - 81.03</td></tr><tr><td rowspan=4 colspan=1>High Negative(LoD/10)0.2 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>68.3% - 98.8%</td><td rowspan=1 colspan=1>79.63</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>78.72-80.72</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% -96.8%</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>79.26 - 80.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5%-76.9%</td><td rowspan=1 colspan=1>79.97</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>78.83 -80.84</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>46/60</td><td rowspan=1 colspan=1>14/60</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>64.0% -86.6%</td><td rowspan=1 colspan=1>79.90</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>78.72 - 80.89</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.0% -100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.0%- 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.0% - 100%</td></tr></table>

H result for the "Negative" panel member is negative. Hi   e $2 0 { \cdot } 8 0 \%$ of the time.

# Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing of MultiAssay Analytes

Reproducibility Agreement Summary for Enterovirus (Human Rhinovirus/Enterovirus)   

<table><tr><td rowspan=1 colspan=2>EnterovirusEchovirus 6 (Species B)</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)90,000 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(IX LoD)30,000 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(LoD/10)3,000 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.8% - 100%</td></tr></table>

result for the "Negative" panel member is negative. Hiv $20 \%$ of the time.

Reproducibility Tm Summary (by Assay) for Enterovirus   
Idaho Technology Inc. $5 \mathbb { i } \mathbb { 0 } ( \mathbb { k } )$ FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">EnterovirusEchovirus 6 (Species B)</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="12">Entero 1</td><td colspan="1" rowspan="4">Medium Positive3x LoD90,000 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.12</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">86.79-87.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.51</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.99 - 88.05</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.98</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">86.37 - 87.64</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.18</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">86.37 - 88.05</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD30,000 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.00</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">86.17 - 87.64</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.36</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.80 - 87.85</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.81</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.15 - 87.41</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.05</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">86.15 - 87.85</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD3,000 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.89</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.06 - 87.40</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.34</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">86.67 - 87.86</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.67</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">85.97 - 87.30</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.96</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">85.97 - 87.86</td></tr><tr><td colspan="1" rowspan="8">Entero 2</td><td colspan="1" rowspan="4">Medium Positive3x LoD90,000 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.09</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">86.68 - 87.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.47</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.82 - 88.00</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.93</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">86.16 - 87.64</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.14</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">86.16 - 88.00</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD30,000 TCIDso/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.98</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.28 - 87.53</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.34</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">86.89 - 87.82</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.77</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.05 - 87.52</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.02</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">86.05 - 87.82</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">High Negative0.1x LoD3,000 TCIDsg/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.86</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.17-87.54</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.26</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.59 - 87.94</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.65</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">86.27 - 87.20</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.92</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">86.17 - 87.94</td></tr><tr><td colspan="1" rowspan="12">HRV4</td><td colspan="1" rowspan="4">Medium Positive3x LoD90,000 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.70</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">85.21 - 86.18</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">86.19</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">85.35 - 86.77</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.59</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">84.91 - 86.19</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.81</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">84.91 - 86.77</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD30,000 TCIDsg/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.45</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.81 - 86.06</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.87</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">85.44 - 86.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.32</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">84.69 - 86.26</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.54</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">84.69 - 86.36</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD3,000 TCIDsq/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.37</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.80 - 86.04</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.82</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">85.43 - 86.23</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.22</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">84.82 - 85.58</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.46</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">84.80 - 86.23</td></tr></table>

:producibility Agreement Summary for Rhinovirus (Human Rhinovirus/Enteroviru   

<table><tr><td rowspan=1 colspan=2>Human RhinovirusAl</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)3 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.8% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)1 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negative(LoD/10)0.1 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>32/40</td><td rowspan=1 colspan=1>8/40</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>64.4%-91.0%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>112/120</td><td rowspan=1 colspan=1>8/120</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>87.3% - 97.1%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.8% - 100%</td></tr></table>

result for the "Negative" panel member is negative. b High negative samples are targeted to be positive $2 0 { - } 8 0 \%$ of the time.

Idaho Technology Inc. 510(k) FilmArray Respiratory Panel System   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>Human RhinovirusAI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedT&#x27;m Range</td></tr><tr><td rowspan=12 colspan=1>HRV1</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>83.25 - 85.09</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.06</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.43-84.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.68</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>83.22 - 84.09</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.84</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.22 - 85.09</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.71</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>83.07- 84.35</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.07</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.44 - 84.66</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.93</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.24 -84.71</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.07 - 84.71</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.56</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>83.02 - 84.26</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.91</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.22 - 84.52</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.04-84.48</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.74</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.02 - 84.52</td></tr><tr><td rowspan=12 colspan=1>HRV2</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.33</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>82.55-84.67</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.65</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>83.11 - 84.17</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.30</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>82.70 - 83.77</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.42</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>82.55 - 84.67</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.28</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>82.57-83.86</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.66</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.02 - 84.37</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.52</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>82.82 -84.29</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.48</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>82.57 - 84.37</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.17</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>82.42 -83.88</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.58</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>82.80 - 84.31</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.37</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>82.51 - 83.92</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.37</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>82.42 - 84.31</td></tr><tr><td rowspan=12 colspan=1>HRV3</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.73</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>81.99-83.94</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.25</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>82.49 -83.88</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.89</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>82.10 -83.88</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.96</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>81.99- 83.94</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.67</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>81.76 -83.65</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.24</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>82.40 -84.08</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.07</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>82.13-83.76</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.98</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>81.76 -84.08</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.68</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>81.78 -83.67</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.21</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>82.18-84.37</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.97</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>81.85-83.64</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.96</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>81.78 -84.37</td></tr><tr><td rowspan=12 colspan=1>HRV4</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.81</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.28 - 84.98</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.14</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.43 - 84.83</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.80</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>83.44 - 84.20</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.28 - 84.98</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.18 -84.45</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.08</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.55-84.61</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.04</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.35 - 84.93</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.94</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.18 - 84.93</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.58</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>83.11-84.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.34 - 84.49</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.86</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>83.24 -84.58</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>83.11 - 84.58</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Influenza A H1N1A/Brisbane/59/07</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)600 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)200 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1%- 99.9%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1%-96.8%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>3/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>93%</td><td rowspan=1 colspan=1>83.8% - 98.2%</td></tr><tr><td rowspan=4 colspan=1>High Negative(LoD/10)20 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1%-96.8%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>45.7% - 88.1%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>51/60</td><td rowspan=1 colspan=1>7/60</td><td rowspan=1 colspan=1>2/60</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>73.4% - 92.9%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

result for the "Negative" panel member is negative. $20 \%$ of the time.

Reproducibility Tm Summary (by Assay) for Influenza A/H1   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>Influenza A H1N1A/Brisbane/59/07</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=12 colspan=1>FluApanl</td><td rowspan=4 colspan=1>Moderate Positive3x LoD600 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.67</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>84.27 - 85.12</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.10</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>84.78 - 85.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.64</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.76 - 85.23</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>84.80</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.76 - 85.56</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD200 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.57</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>84.17 - 85.31</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.01</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>84.59 - 85.75</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.68</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>84.16 - 85.18</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>84.75</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>84.16 - 85.75</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD20 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.27</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>83.85 - 84.81</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.75</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>84.29- 85.32</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.46</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>83.89- 85.48</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>84.48</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.85 - 85.48</td></tr><tr><td rowspan=12 colspan=1>FluApan2</td><td rowspan=4 colspan=1>Moderate Positive3x LoD600 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.42</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.78 - 80.63</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.85</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>80.39 - 81.26</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.31</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>79.89 - 80.72</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.52</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>79.78 - 81.26</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD200 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.36</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>79.99 - 80.73</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.80</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>80.42 - 81.15</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.52</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>80.19-80.89</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.57</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>79.99 - 81.15</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD20 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.91</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>79.15 - 80.41</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.49</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>79.67 - 80.83</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.10</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>79.56 - 80.73</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.17</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>79.15 - 80.83</td></tr><tr><td rowspan=4 colspan=1>FluAH1-pan</td><td rowspan=4 colspan=1>Moderate Positive3x LoD600 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.79</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>78.31 - 79.25</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.20</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>78.30 - 79.57</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.76</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>77.79 - 79.25</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.91</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>77.67 - 79.57</td></tr></table>

Idaho Technology Inc. $5 ! 0 ( \mathbf { k } )$ EY FilmArray Respiratory Panel System

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>Influenza A H1N1A/Brisbane/59/07</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=8 colspan=1></td><td rowspan=4 colspan=1>Low Positive1x LoD200 TCIDs/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>78.77</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>78.42 - 79.34</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.20</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>78.72 - 79.67</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>78.80</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>78.30 - 79.26</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>78.93</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>78.30 - 79.67</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD20 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>77.65</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>77.15 - 78.21</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.18</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>77.47 - 79.26</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>77.93</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>77.43 - 79.04</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>77.92</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>77.15 - 79.26</td></tr></table>

Reproducibility Agreement Summary for Influenza A/2009 H1   

<table><tr><td rowspan=1 colspan=2>Influenza A 2009 H1N1A/Swine NY/03/2009</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreement withExpected Result</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)300 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)100 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%- 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negative(LoD/10)10 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1%-100%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3%-100%</td></tr></table>

H result for the "Negative" panel member is negative. $2 0 { - } 8 0 \%$ of the time.

Reproducibility Tm Summary (by Assay) for Influenza A/2009 H1   
Idaho Technology Inc. $5 1 0 ( \mathbf { k } )$ FilmArray Respiratory Panel System   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">Influenza A 2009 H1N1A/Swine NY/03/2009</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVT'm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="12">FluApan1</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCIDsg/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.27 - 85.12</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.78 - 85.56</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.64</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">83.76 - 85.23</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.76 - 85.56</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCIDsg/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">84.17 - 85.31</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">84.59 - 85.75</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.16-85.18</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">84.16 - 85.75</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCID5/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.27</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">83.85 - 84.81</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">84.29 - 85.32</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.46</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">83.89-85.48</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.48</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.85 - 85.48</td></tr><tr><td colspan="1" rowspan="4">FluApan2</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">80.31-80.83</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.01</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">80.70 -81.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.69</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">80.19 - 81.02</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.81</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">80.19-81.36</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">Influenza A 2009 H1N1A/Swine NY/03/2009</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID5/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.39</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">79.87 - 80.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.91</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.20 - 81.24</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">80.05 - 80.61</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">79.87 - 81.24</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.43</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.08 - 81.03</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.72</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.11- 81.14</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">79.86 - 80.82</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.54</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">79.86 - 81.14</td></tr><tr><td colspan="1" rowspan="12">FluAH1-pan</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCIDs/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.87</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">78.20 - 79.56</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.44</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">78.94 - 79.99</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.62</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">77.92 - 79.44</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.97</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">77.92 - 79.99</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCIDsg/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.37</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">77.89 - 79.24</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.90</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">78.21 - 79.76</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.16</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">77.83 - 78.69</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.47</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">77.78 - 79.76</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCID5/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.34</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">77.90 - 79.37</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">77.99 - 79.68</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.08</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">77.68 - 78.51</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.40</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">77.68 - 79.68</td></tr><tr><td colspan="1" rowspan="12">FluAH1-2009</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCID5/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.73</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.31 - 79.14</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.84 - 79.67</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.54</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">77.89 - 78.92</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.81</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">77.89 - 79.67</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID5/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.64</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">78.10 - 79.14</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.03</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">78.53 - 79.48</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.49</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.12 - 78.82</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.71</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">78.10 - 79.48</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCIDso/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.60</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">77.90 - 79.04</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.00</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">78.52 - 79.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.52</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">78.10 - 78.93</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.70</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">77.90 - 79.36</td></tr></table>

Reproducibility Agreement Summary for Influenza A/H3   

<table><tr><td rowspan=1 colspan=2>Influenza A H3N2A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreement withExpected Result </td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)15 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0%- 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)5 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negative(LoD/10)0.5 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>3.2% - 37.9%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5.7% - 43.7%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>3.2% - 37.9%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>10/60</td><td rowspan=1 colspan=1>31/60</td><td rowspan=1 colspan=1>19/60</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>8.3% - 28.5%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% -100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3%-100%</td></tr></table>

H result for the "Negative" panel member is negative. bHigh negative samples are targeted to be positive $2 0 { - } 8 0 \%$ of the time.

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>Influenza A H3N2A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=12 colspan=1>FluApan1</td><td rowspan=4 colspan=1>Moderate Positive3x LoD15 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>85.33</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>84.79 - 85.73</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.48</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>84.91 - 86.27</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.06</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>84.50 - 85.40</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>85.36</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>84.50 - 86.27</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD5 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.95</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>84.19 - 86.03</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.31</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>84.79 - 86.02</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.83</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>84.37 - 85.25</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>85.03</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>84.19 - 86.03</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.5 TCID5/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>84.91</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>84.22 - 85.60</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.24</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>84.78 - 85.77</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.86</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>84.48 - 85.33</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>85.01</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>84.22 - 85.77</td></tr><tr><td rowspan=12 colspan=1>FluApan2</td><td rowspan=4 colspan=1>Moderate Positive3x LoD15 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.75</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.51 - 79.99</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.94</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>79.40 - 80.37</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.60</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>79.14 - 79.86</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>79.81</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>79.14 - 80.37</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD5 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.42</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>79.00 - 80.30</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.69</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>79.14 - 80.29</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.25</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>78.82 - 79.59</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>79.45</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>78.82 - 80.30</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.5 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.22</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>78.64 - 79.75</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>79.59</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>79.05 - 79.99</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.24</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>78.84 - 79.54</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>79.35</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>78.64 - 79.99</td></tr><tr><td rowspan=12 colspan=1>FluAH3</td><td rowspan=4 colspan=1>Moderate Positive3x LoD15 TCID5g/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.58</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>81.87 - 83.01</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.89</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>82.60 - 83.11</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.44</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>81.98 - 82.81</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.62</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>81.87 - 83.11</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD5 TCIDs/ml</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.32</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>81.88 - 82.69</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.69</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>82.16 - 83.32</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.21</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>81.76 - 82.73</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.39</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>81.67 - 83.32</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.5 TCIDsg/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.28</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>81.71 - 82.91</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>82.61</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>82.17- 83.05</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.20</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>81.87- 82.57</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.37</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>81.71 - 83.05</td></tr></table>

# Precision (Repeatability)

The repeatability of the FilmArray RP System results was evaluated by repeated testing of the same 12 specimens tested in the reproducibility study while minimizing as many sources of varibility as possible. The in-house repeatabilty testing was performed at Site C over the course of 12 testing days for a total of 48 test results per specimen. On each day, all 12 specimens were tested 4 times by two operators on two FilmArray instruments.

Table 27. Summary of Positive Agreement Results for Repeatability Testing   

<table><tr><td rowspan=2 colspan=1>Spiked Organism</td><td rowspan=1 colspan=2>Moderate Positive(3x LoD)</td><td rowspan=1 colspan=2>Low Positive(1x LoD)</td><td rowspan=1 colspan=2>High Negative(0.1x LoD)</td></tr><tr><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34/48</td><td rowspan=1 colspan=1>70.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>44/48</td><td rowspan=1 colspan=1>91.7%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>83/96</td><td rowspan=1 colspan=1>86.5%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43/48</td><td rowspan=1 colspan=1>89.6%</td><td rowspan=1 colspan=1>39/48b</td><td rowspan=1 colspan=1>81.3%b</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1N1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>47/48c</td><td rowspan=1 colspan=1>97.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>7/48d</td><td rowspan=1 colspan=1>14.6%d</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>25/48</td><td rowspan=1 colspan=1>52.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>41/48</td><td rowspan=1 colspan=1>85.4%</td><td rowspan=1 colspan=1>14/48</td><td rowspan=1 colspan=1>29.2%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>% positive (all analytes)per test level</td><td rowspan=1 colspan=2>767/76899.9%</td><td rowspan=1 colspan=2>611/62497.9%</td><td rowspan=1 colspan=2>432/62469.2%</td></tr></table>

Te iv 5non-pos esut or InflzHat LocludeNegative)InuaAH e and (3) Influenza A equivocal results. $\mathsf { A } / \mathsf { H } 1$ at $0 . 1 \textbf { x }$ Lncudative subtype detected), (1) Influenza A/H1 equivocal, and (4) Influenza A equivocal results. On (1) equivocal Influenza A/H1-2009 result at the $0 . 1 \textbf { x }$ LoD test level. i u $0 . 1 \textbf { x }$ Leive detected) (15) Influenza A H3 equivocal, and (7) Influenza A equivocal results.

# Interference

Substances that could be present in NPS samples or introduced during sample handling were evaluated for their potential to interfere with assay performance. Four different organism mixes containing FilmArray RP analytes were spiked into a simulated NPS (sNPS) sample matrix (human epithelial cells in VTM) at ${ \boldsymbol { 5 } } \ \mathbf { x }$ their respective LoDs. The 5x LoD organism concentration was chosen to be near the analyte LoD but also to provide consistent results for sample-to-sample comparison. Each FilmArray RP analyte was tested in the presence of each potentially interfering substance listed in Table . None of the substances tested were found to compete or interfere with the control or analyte assays in the FilmArray RP.

Table 28. List of Potentially Interfering Substances Evaluated   

<table><tr><td colspan="2">Endogenous Substances</td><td colspan="2">Competitive / Interfering Microorganisms</td></tr><tr><td colspan="2">Human Blood (with Na Citrate)(1% v/v)</td><td>Respiratory Syncytial Virus A</td><td>2.8 x 104 TCIDs/mL</td></tr><tr><td colspan="2">Mucin (bovine submaxillary gland) (1% v/v)</td><td>Human Rhinovirus</td><td>1.1 x 104 TCID5/mL</td></tr><tr><td colspan="2">Human Genomic DNA: 0.2 ng/μL</td><td>Influenza A 2009 HIN1</td><td>1.0 x 105 TCIDs/mL</td></tr><tr><td colspan="2">2 ng/μL 20 ng/μL</td><td>Staphylococcus aureus Neisseria meningitides</td><td>1.0 x 10 CFU/mL 1.0 x 10 CFU/mL</td></tr><tr><td colspan="2">Exogenous Substances</td><td>Corynebacterium diptheriae</td><td>1.0 x 10 CFU/mL</td></tr><tr><td colspan="2">Saline Nasal Spray with Preservatives (1% v/v) Nasal Decongestant Spray (Oxymetazoline HCl) (1%v/v)</td><td colspan="2">Analgesic ointment (1% w/v) Petroleum Jelly (1% w/v)</td></tr><tr><td colspan="2">Tobramycin (0.6 mg/mL) Mupirocin (2% w/v)</td><td colspan="2">Smokeless Tobacco (1% w/v)</td></tr><tr><td colspan="4">Technique Specific Substances</td></tr><tr><td>Laboratory Reagents:</td><td>Viral Transport Media:</td><td>Swabs:</td><td></td></tr><tr><td>Bleach (1%, 2%, 5% v/v)</td><td>Remel M4</td><td>Copan 168C (rayon / twisted aluminum shaft)</td><td></td></tr><tr><td>Disinfecting wipes</td><td>Remel M4-RT</td><td>Copan FloQ (flocked nylon / plastic shaft)</td><td></td></tr><tr><td>Ethanol (7% v/v)</td><td>Rcmel M5</td><td>Copan 175KS01 (polyester / aluminum shaft)</td><td></td></tr><tr><td>DNAzap (1% v/v)</td><td>Remel M6</td><td>Millipore 519CS01M (flocked nylon / plastic shaft)</td><td></td></tr><tr><td>RNaseOut (1% v/v)</td><td>Copan UTM</td><td></td><td></td></tr></table>

Evaluation of the FilmArray RP system was not performed using clinical NPS specimens obtained from individuals who had recently received the FluMist® nasal influenza vaccine (MedImmune). However, analytical testing was performed with simulated samples containing various concentrations of the 2009-2010 formulation of the vaccine material. The FilmArray RP assays react with the Influenza A H1, Influenza A H3 and Influenza B viral material contained in the vaccine (Table 29). No cross-reactivity was observed with other, non-influenza FilmArray RP assays.

Table 29. Evaluation of FluMist® Nasal Vaccine as a Potentially Interfering Substance   

<table><tr><td rowspan=2 colspan=1>FluMist2009-2010(% v/v)</td><td rowspan=2 colspan=1>Areo</td><td rowspan=2 colspan=1>CHNLI</td><td rowspan=2 colspan=1>CE9TN</td><td rowspan=2 colspan=1>MeonHuunn</td><td rowspan=2 colspan=1>H RnSh</td><td rowspan=1 colspan=3>Influenza A</td><td rowspan=2 colspan=1>I tann</td><td rowspan=2 colspan=1>ParnnnnedE snIA</td><td rowspan=2 colspan=1>Fe SSnl</td></tr><tr><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>H1-2009</td><td rowspan=1 colspan=1>H3</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>\ra}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>1%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>\frac}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.01%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.001%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.0001%</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Equiva</td><td rowspan=1 colspan=1>$\fra { { }$</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.00001%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>0.000001%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

' Equivocal Influenza A/H3 result

Beth Lingenfelter, M.S. Director, Regulated Products Idaho Technology Inc. 390 Wakara Way Salt Lake City, UT 84108

# FEB 17 251

Re: k103175 Trade/Device Name: FilmArray Respiratory Panel (RP) System Regulation Number: 21 CFR $\$ 866,3980$ Regulation Name: Respiratory Viral Panel Multiplex Nucleic Acid Assay Regulatory Class: Class II Product Codes: OCC, OEM, OOU, OEP, OTG, NXD, O0I Dated: February 2, 2011 Received: February 3, 2011

Dear Ms. Lingenfelter:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into either class II (Special Controls) or class II (PMA), it may be subject to additional controls. Existing major regulations affecting your device can be found in the Code of Federal Regulations, Title 21, Parts 800 to 898. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Part 801); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820); and if applicable, the electronic product radiation control provisions (Sections 531-542 of the Act); 21 CFR·1000-1050.

If you desire specific advice for your device on our labeling regulation (21 CFR Part 801), please go to htup://www.fda.gov/AboutFDA/CentersOffices/CDRH/CDRHOffices/ucm115809.htm for the Center for Devices and Radiological Health's (CDRH's) Office of Compliance. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblcm/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance.

You may obtain other general information on your responsibilities under the Acl from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address http://www.fda.gov/MedicalDevices/ResourcesforYou/Industry/default.htm.

Sincerely yours, the Set fo

Sally A. Hojvat, M.Sc., Ph.D. Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device Evaluation and Safety   
Center for Devices and Radiological Health

# Indications for Use

510(k) Number: k103175 Device Name: FilmArray Respiratory Panel (RP) System

Indications for Use:

The FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following virus types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus HKU1, Coronavirus NL63, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype 2009 H1, Influenza B, Parainfluenza virus 3, Rhinovirus/Enterovirus, and Respiratory Syncytial Virus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. Negative results do not preclude respiratory infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. Positive results do not rule out bacterial infection or co-infection with other organisms. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) and clinical presentation must be taken into consideration in order to obtain the final diagnosis of respiratory infection.

Due to seasonal prevalence, performance characteristics for Influenza A/H1, Influenza A/H3, Influenza A/2009 H1, and Influenza B were established primarily with retrospective clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g. cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g. FDA cleared molecular test or cell culture).

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza

A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

Concurrence of CDRH, Office of In Vitro Diagnostic Device Evaluation and Safety (OIVD)

Division Sign-Off Office of In Vitro Diagnbstic Device Evaluation and Safety_ 510(k) k 10.3175