# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K103209   
B. Purpose for Submission: New device

# C. Measurand:

The $\mathrm { R V + }$ identifies virus-specific nucleic acids for Influenza A virus, Influenza B virus, and Respiratory Syncytial Virus (RSV). The $\mathrm { R V } +$ targets the following genes within the viruses: matrix gene (Influenza A); hemagglutinin gene (Influenza A subtypes H1 and H3); nucleoprotein gene (Influenza A subtype 2009 H1N1); non structural gene (Influenza B); polymerase gene (RSV A and RSV B).

# D. Type of Test:

E. Applicant: Nanosphere, Inc.

F. Proprietary and Established Names: Verigene Respiratory Virus Plus Nucleic Acid Test $( \mathrm { R V } + )$

# G. Regulatory Information:

1. Regulation section: 866.3980

2. Classification: Class II

3. Product code: OCC; NSU

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Verigene® Respiratory Virus Plus Nucleic Acid Test $( \mathrm { R V } + )$ on the Verigene® System is a qualitative nucleic acid multiplex test intended to simultaneously detect and identify multiple respiratory virus nucleic acids in nasopharyngeal (NP) swab specimens from individuals with signs and symptoms of respiratory tract infection. The following virus types and subtypes are identified using the $\mathrm { R V + }$ : Influenza A, Influenza A subtype H1, Influenza A subtype H3, 2009 H1N1, Influenza B, Respiratory Syncytial Virus (RSV) subtype A, and RSV subtype B. The test is not intended to detect Influenza C virus. Detecting and identifying specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection, if used in conjunction with other clinical and laboratory findings.

Negative results for Influenza A, Influenza B, or RSV do not preclude influenza virus or RSV infection and should not be used as the sole basis for diagnosis, treatment, or patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease. The use of additional laboratory testing and clinical presentation must be considered in order to obtain the final diagnosis of respiratory viral infection.

Performance characteristics for Influenza A Virus were established when Influenza A/H3, A/H1, and 2009 H1N1 were the predominant Influenza A viruses circulating. These characteristics may vary when other Influenza A viruses are emerging.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions used specifically for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as the Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

Verigene® System consisting of two instruments, the Verigene Processor $S P$ and the Verigene Reader

# I. Device Description:

The $\mathrm { R V + }$ is designed to identify virus-specific nucleic acids for Influenza A virus, Influenza A virus subtype H1, Influenza A virus subtype H3, Influenza A virus subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ , Influenza B virus, and respiratory Syncytial virus (RSV A. RSV B). The $\mathrm { R V + }$ involves:

1) Sample Preparation – magnetic bead-based viral RNA extraction from nasopharyngeal swab specimens obtained from symptomatic patients;

2) Target Amplification – multiplex RT-PCR-based amplification of the eluted viral RNA targets to generate virus-specific amplicons;

3) Verigene Hybridization Test and Analysis – detection and identification of virus-specific amplicons by using gold nanoparticle probe-based technology.

The entire $\mathrm { R V + }$ test is performed on the Verigene® System, which is a bench-top molecular diagnostics workstation that consists of two instruments, the Verigene Processor $S P$ and the Verigene Reader. The Verigene Processor $S P$ performs the assay steps on each sample by using a robotic pipettor to transfer and mix reagents within and between separate testing modules designed for nucleic acid extraction, target amplification, and the Verigene Hybridization Test. The Verigene Hybridization Test module is the same as in the original Verigene System with added modules for nucleic acid extraction and RT-PCR target amplification.

Key functions of the Verigene Processor $S P$ include:

1) Reading of the barcode identification label on inserted Test Consumables to maintain positive identification of patient samples throughout processing.

2) Facilitation of nucleic acid extraction, multiplex RT-PCR target amplification, and the Verigene Hybridization Test.

3) Real-time communication of test processing status to the Reader.

The Verigene Reader is the same instrument as in the FDA-cleared RVNATSP. It is a free-standing instrument with a touch screen control panel and a wand-based barcode scanner. It utilizes a graphical user interface to guide the user through the process of ordering tests and reporting results. There are no serviceable parts and no user calibration is required. Interaction with the touch screen is minimized through barcode use. This instrument also serves as the reader of the Test Cartridges using advanced optics. The key functions of the Verigene Reader include:

1) Entry and tracking of specimen identification numbers via manual keyboard input or via barcode-reader wand.

2) Test selection for each specimen.

3) Automated transfer of specimen processing instructions on Test Cartridgespecific basis to linked Processor $S P$ unit(s). A single Reader unit can control up to 32 Processor units.

4) Automated imaging and analysis of Test Cartridges.

5) Results display

6) Results report generation.

$\mathrm { R V + }$ consumables within each single-use disposable test kit include: (i) Tip Holder Assembly; (ii) Extraction Tray; (iii) Amplification Tray; and (iv) $\mathrm { R V + }$ Test Cartridge. The kit components are inserted into the corresponding module of the Verigene Processor $S P$ prior to each test, and the sample is added to the Extraction Tray. Patient information is entered into the Reader to initiate the test procedure.

1) Tip Holder Assembly – The robotic pipettor picks up pipettes from the Tip Holder Assembly. The pipettes are used for mixing and transferring reagents within the test procedure.

2) Extraction Tray – Nucleic acids are extracted from the sample by using magnetic bead-based methods within the Extraction Tray. Each Tray contains reagents for a single extraction procedure. A robotic pipette transfers reagents to designated wells within the Extraction Tray to affect the steps of lysis, capture of nucleic acids onto the magnetic beads, washing, and eluting the isolated nucleic acids from the magnetic beads.

3) Amplification Tray – The isolated nucleic acids are amplified by using multiplex RT-PCR within the Amplification Tray. Each Tray contains reagents for a single multiplex RT-PCR procedure. A robotic pipette transfers the reagents to a specific well within the Amplification Tray. A set thermal profile is then initiated to perform all of the amplification related steps including UDG-based decontamination, reverse transcription, and multiplex PCR in a single tube. Upon completion, an aliquot of the amplified sample is mixed with hybridization buffer containing the virus specific mediator probes. The sample is then transferred to the Test Cartridge.

4) $\mathrm { R V + }$ Test Cartridge for Verigene Hybridization Test – The virus-specific and subtype-specific amplicons are detected and identified within a Test Cartridge by using specific nucleic acid probes in conjunction with gold nanoparticle probe-based detection technology. Each Test Cartridge is a self-contained, laboratory consumable that consists of two parts. The upper housing of each cartridge is called the “reagent pack” and contains reservoirs filled with the detection reagents. When in place with the ‘substrate holder’, the reagent pack creates an air-tight hybridization chamber surrounding the region of the substrate containing a target-specific capture array. As each step of the test is completed, old reagents are moved out of the hybridization chamber and new reagents are added from the reagent pack via microfluidic channels and pumps. Once the test is complete, the Test Cartridge is removed from the Verigene Processor $S P$ unit and the reagent pack is snapped off and discarded. The remaining slide is now ready for imaging and analysis in the Verigene Reader.

5) End-point detection on the Verigene Reader: The test slide is inserted into the Verigene Reader wherein it is illuminated along its side. The gold-silver aggregates at the test sites scatter the light, which is in turn captured by a photosensor. The relative intensity arising from each arrayed test site is tabulated. Net signals, defined as the absolute signal intensities with background signals subtracted, are compared with thresholds determined by negative controls within the slide in order to arrive at a decision regarding the presence or absence of target. These results are linked to the test and patient information entered at the beginning of each test session to provide a comprehensive results file.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Verigene® Respiratory Virus Nucleic Acid Test Prodesse ProFAST $^ +$ assay

# 2. Predicate 510(k) number(s): K092566, K101855

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Feature</td><td colspan="1" rowspan="1">Verigene RV+Subject Device</td><td colspan="1" rowspan="1">Verigene RVNAT SPPredicate 1</td><td colspan="1" rowspan="1">Prodesse ProFAST+Predicate 2</td></tr><tr><td colspan="1" rowspan="1">510(k) #</td><td colspan="1" rowspan="1">K103209</td><td colspan="1" rowspan="1">K092566</td><td colspan="1" rowspan="1">K101855</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.3980</td><td colspan="1" rowspan="1">866.3980</td><td colspan="1" rowspan="1">866.3332</td></tr><tr><td colspan="1" rowspan="1">Product Codes</td><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">OCC; NSU</td><td colspan="1" rowspan="1">OQW</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">Class II</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The Verigene® Respiratory Virus Plus NucleicAcid Test (RV+) on the Verigene® System is aqualitative nucleic acid multiplex test intendedto simultaneously detect and identify multiplerespiratory virus nucleic acids in nasopharyngeal(NP) swab specimens from individuals withsigns and symptoms of respiratory tractinfection. The following virus types andsubtypes are identified using the RV+: InfluenzaA, Influenza A subtype H1, Influenza A subtypeH3, 2009 H1N1, Influenza B, RespiratorySyncytial Virus (RSV) subtype A, and RSVsubtype B. The test is not intended to detectInfluenza C virus. Detecting and identifyingspecific viral nucleic acids from individualsexhibiting signs and symptoms of respiratoryinfection aids in the diagnosis of respiratoryviral infection, if used in conjunction with otherclinical and laboratory findings.</td><td colspan="1" rowspan="1">The Verigene® RespiratoryVirus Nucleic Acid Test(RVNATp) is a qualitativemultiplex in vitro diagnostictest for the detection andidentification of Influenza AVirus, Influenza B Virus, andRespiratory Syncytial Virus(RSV) nucleic acids purifiedfrom nasopharyngeal swabspecimens obtained frompatients symptomatic forviral upper respiratoryinfection.</td><td colspan="1" rowspan="1">The Prodesse ProFAST+ Assayis a multiplex Real Time RT-PCR in vitro diagnostic test forthe qualitative detection anddiscrimination of seasonalInfluenza A/H1, seasonalInfluenza A/H3 and InfluenzaA/2009 H1N1 Influenza viralnucleic acids isolated andpurified from nasopharyngeal(NP) swab specimens fromhuman patients with signs andsymptoms of respiratoryinfection in conjunction withclinical and epidemiological riskfactors.</td></tr><tr><td colspan="1" rowspan="1">Targets</td><td colspan="1" rowspan="1">Influenza AInfluenza A/H1Influenza A/H3Influenza A/2009 H1N1Influenza BRSV ARSV B</td><td colspan="1" rowspan="1">Influenza AInfluenza BRSV</td><td colspan="1" rowspan="1">Influenza A/H1Influenza A/H3Influenza A/2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">Specimen</td><td colspan="1" rowspan="1">Nasopharyngeal swabs in sample matrix</td><td colspan="1" rowspan="1">Nasopharyngeal swabs insample matrix</td><td colspan="1" rowspan="1">Nasopharyngeal swabs in samplematrix</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidIsolation</td><td colspan="1" rowspan="1">Automated internal extraction of nucleic acidsperformed on the Processor SP using silicacoated magnetic beads and chaotropic salts.</td><td colspan="1" rowspan="1">Automated internalextraction of nucleic acidsperformed on the ProcessorSP using silica coatedmagnetic beads andchaotropic salts.</td><td colspan="1" rowspan="1">External isolation of nucleicacids on the Roche MagNA PureLC System and bioMerieuxNucliSENS easyMAG.</td></tr><tr><td colspan="1" rowspan="1">Qualitycontrol</td><td colspan="1" rowspan="1">Internal procedural quality controls: IC1 -inhibition control; IC2  process control;internal positive and negative controls; externalpositive controls</td><td colspan="1" rowspan="1">Internal procedural qualitycontrols: PC1 (IC1) -inhibition control; PC2 (IC2) process control; internalpositive and negativecontrols; external positivecontrols</td><td colspan="1" rowspan="1">Influenza positive RNAtranscript control and an internalRNA control provided</td></tr><tr><td colspan="1" rowspan="2">Amplificationmethod</td><td colspan="1" rowspan="1">Internal multiplexed RT-PCR performed withinthe Amplification Module on the Processor SP</td><td colspan="1" rowspan="1">Internal multiplexed RT-PCR performed within theAmplification Module on theProcessor SP</td><td colspan="1" rowspan="1">Real Time RT-PCR detection</td></tr><tr><td colspan="1" rowspan="1">M-MLV Reverse Transcriptase</td><td colspan="1" rowspan="1">M-MLV ReverseTranscriptase</td><td colspan="1" rowspan="1">M-MLV Reverse Transcriptase</td></tr><tr><td colspan="1" rowspan="1">Pipetting</td><td colspan="1" rowspan="1">Pipetting module to automate fluid transfer steps</td><td colspan="1" rowspan="1">Pipetting module to automatefluid transfer steps</td><td colspan="1" rowspan="1">Mostly manual</td></tr><tr><td colspan="1" rowspan="1">DetectionMethod</td><td colspan="1" rowspan="1">Verigene Test (hybridization) is performed inthe hybridization module housed in theProcessor SP of the Verigene® System by usingsingle-use Test Cartridges</td><td colspan="1" rowspan="1">Verigene Test(hybridization) is performedin the hybridization modulehoused in the Processor SPof the Verigene® System byusing single-use TestCartridges</td><td colspan="1" rowspan="1">In the Cepheid Smartcycler IIinstrument, the amount offluorescence at a given cycle isdependent on the amount ofamplification products present atthe time</td></tr><tr><td colspan="1" rowspan="1">Decisionalgorithm</td><td colspan="1" rowspan="1">Target-specific signal intensities are comparedto a signal threshold and ratioed against positiveand negative controls for a decision.</td><td colspan="1" rowspan="1">Target-specific signalintensities are compared to asignal threshold and ratioedagainst positive and negativecontrols for a decision.</td><td colspan="1" rowspan="1">Fluorescent intensity ismonitored during each PCRcycle by the real-time instrument.A specific number of cycles aredefined as a threshold forpresence or absence of a target.</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">Positive or negative qualitative results</td><td colspan="1" rowspan="1">Positive or negativequalitative results</td><td colspan="1" rowspan="1">Positive or negative qualitativeresults</td></tr><tr><td colspan="1" rowspan="1">Reader</td><td colspan="1" rowspan="1">Provides the user interface, controls theProcessor SP, performs image analysis, andprovides results.</td><td colspan="1" rowspan="1">Provides the user interface,controls the Processor SP,performs image analysis, andprovides results.</td><td colspan="1" rowspan="1">Not applicable</td></tr><tr><td colspan="1" rowspan="1">Software</td><td colspan="1" rowspan="1">A custom embedded software applicationrunning under the Micro-C/OS real-timeoperating system. Additional softwareprogramming to control the Extraction andAmplification Modules.</td><td colspan="1" rowspan="1">A custom embeddedsoftware application runningunder the Micro-C/OS real-time operating system.Additional softwareprogramming to control theExtraction and AmplificationModules.</td><td colspan="1" rowspan="1">Not applicable</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

1. CLSI EP5-A2; Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline - Second Edition.   
2. CLSI EP12-A; User Protocol for Evaluation of Qualitative Test Performance; Approved Guideline - Second Edition.   
3. CLSI MM3-A2; Molecular Diagnostic Methods for Infectious Diseases; Approved Guideline – Second Edition.   
4. FDA document, Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable - Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff   
5. FDA document #337, Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices - Guidance for Industry and FDA Staff.   
6. FDA document #1638, Establishing the Performance Characteristics of IVDs for the Detection and Differentiation of Influenza Viruses, February 15, 2008.   
7. FDA document #1594, In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path, May 2, 2007.   
8. FDA document #1596, Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses, March 22, 2006.   
9. Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Nucleic Acid Assays, October 9, 2009.   
10. Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems, March 10, 2005.   
11. Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay, October 9, 2009.   
12. Class II Special Controls Guidance Document: Respiratory Viral Panel Nucleic Acid   
13. Assay: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays, October 9, 2009.

# L. Test Principle:

$\mathrm { R V + }$ is a molecular test and relies on the specific detection of target nucleic acids. For each of the viruses/analytes detected in $\mathrm { R V + }$ , at least 4 sets of oligonucleotides are required: (i) and (ii) Specific forward and reverse primers for the generation of virus-/analyte-specific amplicons; (iii) Capture oligonucleotides; and (iv) Mediator oligonucleotides. The Capture oligonucleotides (or captures) and Mediator oligonucleotides (or mediators) are needed for the endpoint gold nanoparticle probe-based detection process. The Capture oligonucleotides (or captures) are printed on the Test Substrate (or the microarray) and are designed to specifically bind to one part of the target amplicon. The Mediator oligonucleotides (or mediators) bind to another part of the same target amplicon and allow the binding of a gold nanoparticle probe to a portion complementary to a gold nanoparticle probe. Notably, in a multiplexed detection system, numerous unique target-specific mediators can co-exist and form unique ‘sandwich’ hybridizations at the different captures on the microarray. Since all the mediators have a target specific region and a gold nanoparticle probe region, a single ‘universal’ gold nanoparticle probe is sufficient. Specific silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency. Light scatter from the capture spots is imaged by the Verigene Reader and intensities from the microarray spots are used to make decisions regarding the presence (Detected) or absence (Not Detected) of a virus/analyte.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The reproducibility panel was developed by using 6 unique virus strains to represent all of the analytes in the $\mathrm { R V + }$ . The 6 virus strains were combined to generate 12 unique combinations. The combinations were designed such that each virus strain was represented at 3 levels: High Negative, Low Positive, and Moderate. For two of the unique samples, virus strains were combined as in the case of INFA/H3 and RSVA and Influenza B and RSV B. For the Reproducibility/Precision study, the Test Panel comprised the 12 unique samples in duplicate for a total of 24 samples. The Test Panel samples were then divided equally into Panel A (12 samples) and Panel B (12 samples). In order to prepare the samples, the strains were diluted at the desired concentrations in the above-mentioned combinations into Universal Transport Medium, aliquoted into individual test aliquots, and stored frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ until the time of use. Panel A and Panel B composition is outlined in the table below.

<table><tr><td rowspan=1 colspan=1>ViralPanel</td><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>UniqueSamples</td></tr><tr><td rowspan=12 colspan=1>A</td><td rowspan=6 colspan=1>INFB +RSVB</td><td rowspan=2 colspan=1>High Negative</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=6 colspan=1>INFA/2009H1N1</td><td rowspan=2 colspan=1>High Negative</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=12 colspan=1>B</td><td rowspan=6 colspan=1>INFA/H1</td><td rowspan=2 colspan=1>High Negative</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=6 colspan=1>INFA/H3 +RSVA</td><td rowspan=2 colspan=1>High Negative</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=2 colspan=1>Low Positive</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=2 colspan=1>Moderate Positive</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>12</td></tr></table>

Reproducibility testing was carried out at three sites. One site was Nanosphere, Inc, while the other two were external testing sites. At the two external study sites, Test Panel A and Test Panel B were tested on separate days. Testing each day involved 2 operators testing the same Test Panel in duplicate resulting in 24 tests per day.

The internal site running the 12-day precision study utilized the same four unique samples at the three levels. The complete Test Panel (Test Panel A and B) was tested separately by two operators each day for. The precision study was run for a total testing period of 12 non-consecutive days. The following table summarizes the cumulative data for $\mathrm { R V + }$ across the three test sites. The Confidence Interval (CI) was calculated using the exact binomial method in SAS 9.1.3.

# Clinical Reproducibility Summary

<table><tr><td rowspan=1 colspan=2>Site</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=5>CV, %</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=4>CV, %</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=4>CV, %</td><td rowspan=1 colspan=2>All 3 Sites</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Specific PanelMember</td><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=4>Analyte</td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=3>Analyte</td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=3>Analyte</td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>95%CI</td></tr><tr><td rowspan=6 colspan=1>INFA/H1-MT</td><td rowspan=2 colspan=1>MP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.074%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.073%</td><td rowspan=2 colspan=1>46/481,2</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.132%</td><td rowspan=2 colspan=1>70/72</td><td rowspan=2 colspan=1>97.2%</td><td rowspan=2 colspan=1>90.4%99.2%</td></tr><tr><td rowspan=1 colspan=4>H1</td><td rowspan=1 colspan=1>0.530%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>0.627%</td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>0.381%</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.097%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.120%</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.030%</td><td rowspan=2 colspan=1>72/72</td><td rowspan=2 colspan=1>100%</td><td rowspan=2 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=4>H1</td><td rowspan=1 colspan=1>0.542%</td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>1.292%</td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>0.271%</td></tr><tr><td rowspan=2 colspan=1>HN</td><td rowspan=2 colspan=1>11/123</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>71/72</td><td rowspan=2 colspan=1>98.6%</td><td rowspan=2 colspan=1>92.5%99.8%</td></tr><tr><td rowspan=1 colspan=4>H1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>H1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=6 colspan=1>INFA/H3</td><td rowspan=2 colspan=1>MP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.151%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.033%</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.067%</td><td rowspan=2 colspan=1>72/72</td><td rowspan=2 colspan=1>100%</td><td rowspan=2 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=4>H3</td><td rowspan=1 colspan=1>0.343%</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>0.346%</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>0.067%</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.337%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.965%</td><td rowspan=2 colspan=1>47/484</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.005%</td><td rowspan=2 colspan=1>71/72</td><td rowspan=2 colspan=1>98.6%</td><td rowspan=2 colspan=1>92.5%-99.8%</td></tr><tr><td rowspan=1 colspan=4>H3</td><td rowspan=1 colspan=1>0.572%</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>0.621%</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>0.172%</td></tr><tr><td rowspan=2 colspan=1>HN</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>72/72</td><td rowspan=2 colspan=1>100%</td><td rowspan=2 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=4>H3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>H3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=6 colspan=1>INFA/2009H1N1-MT</td><td rowspan=2 colspan=1>MP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.081%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.035%</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.001%</td><td rowspan=2 colspan=1>72/72</td><td rowspan=2 colspan=1>100%</td><td rowspan=2 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=4>2009H1N1</td><td rowspan=1 colspan=1>0.547%</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>0.050%</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>0.001%</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>0.136%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.067%</td><td rowspan=2 colspan=1>48/48</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>0.013%</td><td rowspan=2 colspan=1>72/72</td><td rowspan=2 colspan=1>100%</td><td rowspan=2 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=4>2009H1N1</td><td rowspan=1 colspan=1>0.274%</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>0.273%</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>0.181%</td></tr><tr><td rowspan=2 colspan=1>HN</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=4>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>12/12</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>47/485</td><td rowspan=1 colspan=3>INFA</td><td rowspan=1 colspan=1>N/A</td><td rowspan=2 colspan=1>71/72</td><td rowspan=2 colspan=1>98.6%</td><td rowspan=2 colspan=1>92.5%99.8%</td></tr><tr><td rowspan=1 colspan=4>2009H1N1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=3>2009H1N1</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=7 colspan=1>INFB</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>0.149%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0.086%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0.002%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=5 colspan=1>LP</td><td rowspan=5 colspan=1>12/12</td><td rowspan=5 colspan=3>INFB</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=5 colspan=4>INFB</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=1>INFB</td><td rowspan=3 colspan=1>0.704%</td><td rowspan=3 colspan=1>12/12</td><td rowspan=3 colspan=2>INFB</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=3>INFB</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>0.066%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3>INFB</td><td rowspan=1 colspan=1>0.025%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>11/12a</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>71/72</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>92.5%-99.8%</td></tr><tr><td rowspan=4 colspan=1>RSVA</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>12/12</td><td rowspan=2 colspan=4>RSVA</td><td rowspan=1 colspan=1>0.422%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=3 colspan=3>RSVA</td><td rowspan=1 colspan=1>0.245%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0.192%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=2 colspan=1>LP</td><td rowspan=2 colspan=1>12/12</td><td rowspan=2 colspan=3>RSVA</td><td rowspan=2 colspan=1>RSVA</td><td rowspan=2 colspan=1>0.682%</td><td rowspan=2 colspan=1>12/12</td><td rowspan=2 colspan=1>0.500%</td><td rowspan=2 colspan=1>46/48b,4</td><td rowspan=2 colspan=3>RSVA</td><td rowspan=2 colspan=1>0.176%</td><td rowspan=2 colspan=1>70/72</td><td rowspan=2 colspan=1>97.2%</td><td rowspan=2 colspan=1>90.4%-99.2%</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=3 colspan=1>RSVB</td><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>0.127%</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>0.087%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=2 colspan=3>RSVB</td><td rowspan=1 colspan=1>0.002%</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.9%- 100%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>11/12c</td><td rowspan=1 colspan=3>RSVB</td><td rowspan=1 colspan=2>RSVB</td><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>0.163%</td><td rowspan=1 colspan=3>12/12</td><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=2>0.302%</td><td rowspan=1 colspan=1>47/48d</td><td rowspan=1 colspan=1>RSVB</td><td rowspan=1 colspan=1>0.060%</td><td rowspan=1 colspan=1>70/72</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>11/12e</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>71/72</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>92.5%99.8%</td></tr></table>

Expected Calls: a One (1) $\mathrm { I N F B + R S V B }$ HN detected Influenza B. b One (1) $\mathrm { I N F A } / \mathrm { H } 3 +$ RSVA LP sample detected Influenza A and H3 but did not detect RSV A. c One (1) INFB + RSVB LP sample detected Influenza B but did not detect RSV B. $\mathrm { ^ d O n e \ I N F B + R S V B }$ LP sample detected Influenza B but did not detect RSV B. e One INFB $^ +$ RSVB HN sample detected RSV B but did not detect Influenza B. Additional Positive Calls: 1 One (1)

INFA/H1-MT MP sample detected both Influenza A and H1 as expected, but also detected H3 which was unexpected. 2 One (1) INFA/H1-MT MP sample detected both Influenza A and H1 as expected, but also detected H3, Flu B and RSV A which was unexpected. 3 One (1) INFA/H1-MT HN sample detected Influenza A, but did not detect H1 which was expected as this is a high negative sample. However, RSV A was also detected which was unexpected. $^ { 4 } \mathrm { O n e } \mathrm { I N F A } / \mathrm { H } 3 + \mathrm { R S V A } \bar { \mathrm { I } }$ P sample detected both Influenza A, H3 and RSV A as expected for a low positive sample, but also detected 2009H1N1 which was not expected. 5 One INFA/2009H1N1-MT HN sample did not detect Influenza A and 2009H1N1 which was expected as this is a high negative sample. However, Influenza B and RSV B were also detected which was not expected. 6 One INFA/H3/RSVA LP sample detected Influenza A, H3 RSV A which was expected as this is a low positive. However, 2009H1N1 was also detected which was not expected. NOTE: All the samples with additional positive calls gave the expected results for the intended virus; there were no mis-calls. The source of the additional positives is likely cross-contamination artifacts during the Test Panel preparation.

b. Linearity/assay reportable range: Not Applicable/ Qualitative Assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Blanks or no template control

Not required as the $\mathrm { R V + }$ is performed in a single-test use disposables. Also, the $\mathrm { R V + }$ employs a thermolabile UDG enzyme that eliminates the potential of amplicon cross-over and potential contamination between tests. Moreover, the $\mathrm { R V + }$ is an end-point detection test and high background within the Test Cartridge owing to the amplification step (or related steps) results in a ‘No Call – BKGD’.

# Negative Sample Control

External Negative Control – In the $\mathrm { R V + }$ is Universal Transport Media. The External Negative Control is the equivalent of the proposed Negative Sample Control in the Guidance document. Since the $\mathrm { R V + }$ is an endpoint detection test, non-specific priming during the target amplification step, if any, does not interfere with the test since detection requires binding of specific captures and mediators. The expected calls after test analysis are ‘Not Detected’ for all the viruses/analytes. The frequency of running this test is dictated by the test site. It is recommended that the $\mathrm { R V + }$ kit be tested upon receipt by running the external negative control to ensure proper performance of the kit.

Note that in the processing of the External Negative Control, RNA from IC2 gets extracted and is amplified along with IC1, which is present in the amplification mix. The inability to detect either IC1 or IC2 indicates issues with isolation or the target amplification within the particular test alone. The recourse is to isolate and amplify a fresh external negative control sample, just the same as if it were a sample. NC or Negative Control is a set of non-target oligonucleotide spots that serve as negative controls and are used to normalize signal intensities at target spots (Ratio-to-NC)

# Positive control for complete assay

External positive controls – In the $\mathrm { R V + }$ , ‘positive control for the complete assay’ are suggested to be inactivated intact viral particles with altered surface proteins making them unable to infect a host cell. Obtained from Zeptometrix Corporation, these viral particles, referred to as NATtrol, possess viral integrity and an intact viral genome. Thus, in the $\mathrm { R V + }$ , the NATtrol serves as a control for the extraction, the target amplification, and detection steps.

# Positive control for amplification/detection

IC1 serves as the internal inhibition control or ‘positive control for amplification/detection’. It is a 99-mer duplex DNA synthetic construct present in the primer mix and is amplified along with every test sample (along with potential targets and IC2). It independently controls for inhibitors to the PCR step in each test. The presence of IC1 in the analysis indicates that the reaction components performed as expected. PC or Imaging control is a set of oligonucleotide spots on the substrate that act as a quality check for proper fluid control and movement between the Test Cartridge and the instrument. If the imaging controls are absent, then a “No Call – No Grid” is an output. The imaging control signal is due to signal probe hybridization to an oligonucleotide on the substrate. Signal will only be present if the probe reagent and the signal enhancement reagents are pumped into and held in the hybridization chamber for the specified time period. The PC also serves to normalize signal intensities at target spots (Ratio-to-PC).

# Internal Control

IC2 (Escherichia coli MS2 bacteriophage) serves as an internal process control in the $\mathrm { R V + }$ . The IC2 has an intact viral genome in a viral capsid and therefore controls for the sample isolation or the nucleic acid extraction step and the target amplification or the RTPCR step. In the context of the $\mathrm { R V + }$ , IC2 is added to every test sample and is coextracted with the sample. The extracted RNA undergoes RT-PCR and controls for inhibitors within the test sample.

NOTE: A valid ‘Call’ is made only after both inhibition control (IC1) and process control (IC2) are verified during analysis of each test, signifying that the extraction and target amplification processes performed correctly. If IC1 or IC2 are ‘Not Detected’ a ‘No Call – INT CTL $1 ^ { \circ }$ or a ‘No Call – INT CTL $2 ^ { \circ }$ is provided respectively. If both IC1 and IC2 are ‘Not Detected’, a ‘NO CALL – INT CTL’ result is provided. The following exception exists: IC2 detection alone is sufficient for a valid call if any of the viral targets are also detected; there is no requirement for IC1 to also be ‘Detected’. The recourse for the ‘No Call – INT CTL decision is to repeat the $\mathrm { R V + }$ test. In addition, separate internal positive and negative controls are present within each Test Cartridge that inform regarding the proper functioning of the Test Cartridge.

The following table summarizes the controls found in the $\mathrm { R V + }$ with a description of the composition of the control as well as the function.

<table><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">Description</td><td colspan="1" rowspan="1">Function</td></tr><tr><td colspan="1" rowspan="1">InhibitionControl(IC1)</td><td colspan="1" rowspan="1">Double-stranded DNA target presentin the primer mix. Amplified withevery RT-PCR reaction.</td><td colspan="1" rowspan="1">Controls for PCR inhibition due to sample- orprocess-related inhibitors or due to reagentfailures.</td></tr><tr><td colspan="1" rowspan="1">ProcessControl(IC2)</td><td colspan="1" rowspan="1">MS2 bacteriophage with an intactviral RNA genome. Addedautomatically to each test sampleincluding external positive andnegative controls.</td><td colspan="1" rowspan="1">Controls for sample isolation step (or nucleic acidextraction step) and the RT-PCR step.</td></tr><tr><td colspan="1" rowspan="1">ExternalPositiveControl</td><td colspan="1" rowspan="1">Any of the three viral particles(Influenza A, Influenza B, or RSV).</td><td colspan="1" rowspan="1">Serve as external controls for the extraction, targetamplification, and detection steps. Used to verifyreagent performance: (a) during installation,system validation, and when troubleshootingdictates (see Verigene System User's Manual); (b)to verify the performance of a new lot/batch ofreagents; (c) when the integrity of storageconditions is in question.</td></tr><tr><td colspan="1" rowspan="1">ExternalNegativeControl</td><td colspan="1" rowspan="1">Viral Transport Media</td><td colspan="1" rowspan="1">Controls for reagent and/or environmentalcontamination.</td></tr></table>

# d. Detection limit:

The analytical sensitivity of $\mathrm { R V + }$ was assessed and confirmed by using virus strains with established titers to represent all analytes in the test. The strains were previously cultured and titered (see the table below for stock titers). In order to determine the Limits of Detections (LODs), the strains were serially diluted into negative pools of nasopharyngeal (NP) swab samples. Each concentration level in the dilution series was tested in the $\mathrm { R V + }$ in triplicate to establish an estimated LOD.

For confirmation, 20 replicate samples were prepared at the estimated LOD level chosen from the dilution series experiment and tested in the $\mathrm { R V + }$ . Acceptance criteria for LOD confirmation required that at least 19 of the 20 samples tested gave a ‘Detected’ call for the analyte. The confirmed LOD levels results for the different analytes and the associated virus strains are presented in the table below.

<table><tr><td colspan="1" rowspan="1">VIRUS</td><td colspan="1" rowspan="1">ANALYTE</td><td colspan="1" rowspan="1">VIRAL STRAINS  ANALYTICALSTUDIES</td><td colspan="1" rowspan="1">LOD (TCID5o/mL)</td><td colspan="1" rowspan="1">20 REPS</td></tr><tr><td colspan="1" rowspan="8">INFA</td><td colspan="1" rowspan="5">INFA</td><td colspan="1" rowspan="1">Influenza A/Virginia/01/2006 (H1N1)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">19/20</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009H1N1 Clinical Isolate (H1N1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/2007 Clinical Isolate(H1N1)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/04/2009 (H1N1)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05 (H3N2)</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="2">H1</td><td colspan="1" rowspan="1">Influenza A/Virginia/01/2006 (H1N1)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/2007 Clinical Isolate(H1N1)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">19/20</td></tr><tr><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">Influenza A/2009H1N1 Clinical Isolate (H1N1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">19/20</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza A/California/04/2009 (H1N1)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05 (H3N2)</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">Influenza B/Wisconsin/2/2006</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Influenza B/Florida/02/06</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSV A Strain A2</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">20/20</td></tr><tr><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">RSV B Strain Wash/18537/62</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">20/20</td></tr></table>

# e. Analytical specificity:

Analytical specificity studies were performed to assess potential cross-reactivity of the $\mathrm { R V + }$ with common respiratory pathogens and other microorganisms commonly present in the respiratory tract. In the cross-reactivity studies, 23 bacterial strains were tested at concentrations between $1 0 ^ { 5 }$ to $1 0 ^ { 7 }$ cfu/mL while 29 virus strains were tested at concentrations between $1 0 ^ { 4 }$ to $1 0 ^ { 6 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The bacterial strains were obtained from ATCC, re-grown and their titers confirmed. The virus strains were obtained from several sources including ATCC, Zeptometrix, Intelligent MDx, and Tricore. All virus strains were re-grown and their titers were confirmed. The microorganisms were diluted into a sample matrix, Universal Transport Medium (Copan), at the concentrations indicated and tested subsequently in the $\mathrm { R V + }$ .

In the $\mathrm { R V + }$ , IC1 and IC2 are verified before providing a valid call. IC1 is an inhibition control for the Target Amplification step and IC2 is a process control (bacteriophage, MS2) that is added to the sample automatically prior to the extraction step. The IC2 goes through all steps of $\mathrm { R V + }$ : Sample Extraction, Target Amplification, and Verigene Hybridization Test. In all the samples tested, a valid test result was provided indicating that the microorganisms did not negatively effect the $\mathrm { R V + }$ processing. Significantly, all the microorganisms tested gave a ‘Not Detected’ call for each analyte on the $\mathrm { R V ^ { + } } .$ ; the microorganisms tested did not demonstrate cross reactivity with any of the analytes in the $\mathrm { R V + }$ .

$\mathrm { R V + }$ did not cross-react with any of the microorganisms chosen to represent pathogens residing in the respiratory tract. The table below details the test results from the study.

<table><tr><td colspan="1" rowspan="1">Viruses</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="2">pfu/mL</td><td colspan="1" rowspan="2">INFA</td><td colspan="1" rowspan="2">H1</td><td colspan="1" rowspan="2">H3</td><td colspan="1" rowspan="2">2009H1N1</td><td colspan="1" rowspan="2">INFB</td><td colspan="1" rowspan="2">RSVA</td><td colspan="1" rowspan="2">RSVB</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 1</td><td colspan="1" rowspan="1">Adenoid 71</td><td colspan="1" rowspan="1">6.2x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 2</td><td colspan="1" rowspan="1">Adenoid 6</td><td colspan="1" rowspan="1">1.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 3</td><td colspan="1" rowspan="1">G.B.</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 4</td><td colspan="1" rowspan="1">RI-67</td><td colspan="1" rowspan="1">1.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 5</td><td colspan="1" rowspan="1">Adenoid 75</td><td colspan="1" rowspan="1">5.5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 7</td><td colspan="1" rowspan="1">Gomen</td><td colspan="1" rowspan="1">6.2x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 11</td><td colspan="1" rowspan="1">Slobitski</td><td colspan="1" rowspan="1">1.9x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 14</td><td colspan="1" rowspan="1">de Wit</td><td colspan="1" rowspan="1">5.5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 31</td><td colspan="1" rowspan="1">1315</td><td colspan="1" rowspan="1">3.5x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Adenovirus Type 35</td><td colspan="1" rowspan="1">holden</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus (OC43)</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">1.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus (229E)</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus (NL63)</td><td colspan="1" rowspan="1">NL63</td><td colspan="1" rowspan="1">9.9x104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">68-1</td><td colspan="1" rowspan="1">2.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus, H.coxsackievirus B4</td><td colspan="1" rowspan="1">J.V.B.</td><td colspan="1" rowspan="1">1.9x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Enterovirus, echovirus 11</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.4x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">2.2x109copies/m</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex virus Type 1</td><td colspan="1" rowspan="1">MacIntyre</td><td colspan="1" rowspan="1">2.4x10}$</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">Edmonst-on</td><td colspan="1" rowspan="1">1.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus 21</td><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">1.1x104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus</td><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">1.1x104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Metapneumovirus</td><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">3.5x10</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">Enders</td><td colspan="1" rowspan="1">1.1x104</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">C35</td><td colspan="1" rowspan="1">1.2x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">Greer</td><td colspan="1" rowspan="1">2.2x10</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">C243</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 4</td><td colspan="1" rowspan="1">M-25</td><td colspan="1" rowspan="1">7.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus Type 1A</td><td colspan="1" rowspan="1">2060</td><td colspan="1" rowspan="1">1.1x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Varicella-Zoster virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacteria</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">cfu/mL</td><td colspan="1" rowspan="1">INFA</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter Baumannii</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.7x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">CM-1</td><td colspan="1" rowspan="1">1.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumpseudodiphtheriticum</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8.2x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.5x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5.0x10}$</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.1x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.2x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.5x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.0x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis,attenuated</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.6x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.0x105</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Boston41501</td><td colspan="1" rowspan="1">3.5x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">6.1x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3.7x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">9.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">O90R</td><td colspan="1" rowspan="1">2.0x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.3x106</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">5.5x107</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

# f. Analytical Reactivity:

The analytical reactivity of the $\mathrm { R V + }$ was evaluated using multiple strains of Influenza A (8 seasonal H1 strains, 7 seasonal H3 strains, $4 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ strains, 2 swine origin H1N1 strains, 1 H5N1 strain, and 1 H7N7 strain collectively representing temporal and geographical diversity), 9 Influenza B strains, 2 RSV A strains and 3 RSV B strains.

In order to demonstrate analytical reactivity, cultured and titered strains of Influenza $\mathrm { A } / \mathrm { H } 3$ , Influenza $\mathrm { A } / \mathrm { H } 1$ , Influenza A/2009H1N1, Influenza B, RSV A, and RSV B were diluted in sample matrix, Universal Transport Media (Copan), to a nominal concentration of $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (or $\mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ ). Each strain dilution was tested in triplicate in the $\mathrm { R V + }$ . Most strains were reactive at $1 0 0 \mathrm { T C I D } 5 0 / \mathrm { m L }$ ; however, concentration levels for a few strains were increased in order for all three replicate tests to yield ‘Detected’. All the Influenza $\mathrm { A } / \mathrm { H } 3$ , Influenza $\mathrm { A } / \mathrm { H } 1$ , and Influenza A/2009 H1N1 strains yielded expected subtyping decisions.

The two swine H1N1 strains of non-human origin gave a ‘Detected’ result for Influenza A and for H1 at $1 0 0 \mathrm { C E I D } 5 0 / \mathrm { m L }$ and gave a ‘Not Detected’ result for $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ . Higher concentrations of these strains $\mathrm { ( 1 x 1 0 ^ { 3 } C E I D _ { 5 0 } / m L }$ and $\mathrm { 1 x 1 0 ^ { 4 } C E I D _ { 5 0 } / m L } \mathrm { \Omega }$ ) were also tested to assess cross-reactivity at 2009 H1N1 but none was found as evidenced by a ‘Not Detected’ call for 2009 H1N1.

The H5 and H7 Influenza strains in this study were grown in culture, titered, and inactivated before they were tested in the $\mathrm { R V + }$ . The H5 and H7 strains gave a ‘Detected’ result for Influenza A and a ‘Not Detected’ result for H1, H3, and 2009 H1N1. As expected, these strains were untypeable in the $\mathrm { R V + }$ .

In summary, the $\mathrm { R V + }$ demonstrated good reactivity against a range of panel viruses. Against, Influenza viruses H5 and H7, $\mathrm { R V + }$ gave a ‘Detected’ call for Influenza A but these viruses did not cross-react with any of the subtyping systems even at very high concentrations demonstrating high assay specificity. Similarly, the two swine-origin H1N1 strains gave ‘Detected’ calls for Influenza A and HI only; no cross-reactivity was observed even at very high titers. Finally, no cross-reactivity was observed within the remaining panel viruses.

<table><tr><td colspan="1" rowspan="1">Virus Strains</td><td colspan="1" rowspan="2">Titer(TCID50/mL)</td><td colspan="1" rowspan="2">INFA</td><td colspan="1" rowspan="2">H1</td><td colspan="1" rowspan="2">H3</td><td colspan="1" rowspan="2">2009H1N1</td><td colspan="1" rowspan="2">INFB</td><td colspan="1" rowspan="2">RSVA</td><td colspan="1" rowspan="2">RSVB</td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/NewCaledonia/20/99 (H1N1)</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/PR/8/34 (H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A1/FM/1/47 (H1N1)</td><td colspan="2" rowspan="1">5X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/NWS/33 (H1N1)</td><td colspan="2" rowspan="1">1X103</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">,</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A1/Denver/1/57H1N1)</td><td colspan="2" rowspan="1">1X103</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">,</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hawaii/15/01(H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/2007Clinical Isolate (H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/ Virginia/01/2006(H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers/1/73(H3N2)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/8/68(H3N2)</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Aichi2/68 (H3N2)</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/3/75(H3N2)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05(H3N2)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hiroshima/52/05(H3N2)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/NY/55/04 (H3N2)</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/04/2009(H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009H1N1/PSF1Clinical Isolate 09 (H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/629-D01606/ 2009 (H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009H1N1 ClinicalIsolate (H1N1)</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Swine/Iowa/15/30*</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Swine/1976/31*</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/Duck/Hunan/795/02 (H5N1)**</td><td colspan="2" rowspan="1">2.3X102</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/Netherlands/219/2003(H7N7) **</td><td colspan="2" rowspan="1">1.1X104</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="2" rowspan="1">Titer(TCID50/mL)</td><td colspan="1" rowspan="1">INFA</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td></tr><tr><td colspan="1" rowspan="1">Influenza B/ Wisconsin/2/2006</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/2006</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">InfluenzaB/Malaysia/2506/2004</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Ohio/1/2005</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/ Lee/40</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/GL/1739/54</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Taiwan/2/62</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hong Kong/5/72</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Maryland/1/59</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/2006</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">,</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV A</td><td colspan="1" rowspan="1">Titer(TCID50/mL)</td><td colspan="1" rowspan="1">L)</td><td colspan="1" rowspan="1">INFA</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td></tr><tr><td colspan="1" rowspan="1">RSV A Strain A2</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV A Strain Long</td><td colspan="1" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B</td><td colspan="2" rowspan="1">Titer(TCID50/mL)</td><td colspan="1" rowspan="1">INFA</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">2009H1N1</td><td colspan="1" rowspan="1">INFB</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSVB</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain Wash/18537/62</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain B-1 Wild Type</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV B Strain 9320</td><td colspan="2" rowspan="1">1X102</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr></table>

\* In addition to the above testing, both swine-origin H1N1 strains were tested at 1- and 2-logs higher concentrations $\mathrm { 1 x 1 0 ^ { 3 } }$ and $\mathrm { 1 x 1 0 ^ { 4 } T C I D _ { 5 0 } / m L }$ ). No cross-reactivity to $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ was observed. \*\* In addition to the above testing, the H5 and H7 strains were tested at $1 \mathrm { - }$ and 2-logs higher concentration. No crossreactivity to any sub-types was observed.

$g .$ . Assay cut-off:

$\mathrm { R V + }$ provides a qualitative result for the presence (Detected) or absence (Not Detected) of the following viruses/analytes: Influenza A, Influenza B, RSV A, RSV B. For Influenza A positive samples, $\mathrm { R V + }$ provides subtyping information as H1, H3, or 2009 H1N1.

Image analysis of the Test Substrate provides image intensities from target-specific capture spots. Two Internal Controls guide decisions regarding the validity of the test. The Internal Controls, IC1 – inhibition control and IC2 – the process control are also represented by unique captures and are treated as targets and their presence is verified before a valid result is provided.

In order to provide a ‘Detected’ and ‘Not Detected’ result in $\mathrm { R V } +$ , three conditions or filters are examined. These conditions serve as a single-set of clinical cut-off criteria.

− Filter 1: Signal $>$ Noise Threshold Filter 2: Normalized ‘Ratio to Negative Control’ $> 0 . 8 5$ Filter 3: Normalized ‘Ratio to Positive Control $\geq 0 . 4$ A True Positive (or a False Positive) requires that all three filter conditions are met. A True Negative (or a False Negative) is provided if any or all the filter conditions are not met.

For a result, the decision tree verifies the presence of IC1 and IC2 in conjunction with each of the viruses. Both IC1 and IC2 signal intensities have to meet the detection criteria for a valid call. A valid ‘Call’ is made only after both inhibition control (IC1) and process control (IC2) are verified during analysis of each test, signifying that the extraction and target amplification processes performed correctly. If IC1 or IC2 are ‘Not Detected’ a ‘No Call – INT CTL $1 ^ { \circ }$ or a ‘No Call – INT CTL $2 ^ { \circ }$ is provided respectively. If both IC1 and IC2 are ‘Not Detected’, a ‘NO CALL – INT CTL’ result is provided. The following exception exists: IC2 detection alone is sufficient for a valid call if any of the viral targets are also detected; there is no requirement for IC1 to also be ‘Detected’. The recourse for the ‘No Call – INT CTL decision is to repeat the $\mathrm { R V + }$ test. In addition, separate internal positive and negative controls are present within each Test Cartridge that inform regarding the proper functioning of the Test Cartridge.

# Cut-off verification

A set of retrospective clinical samples (71) positive by DFA/Culture for either Influenza A, Influenza B, or RSV was tested in the $\mathrm { R V + }$ to verify the cut offs.

The plots of the logistic fit of Expected Results by Ratio-to-PC and Ratio-to-NC for these retrospective samples was provided for this data set along with the predictive value of the algorithm. Decisions were made based on the three-filter method by using the cut-off limits for Ratio-to-PC and Ratio-to-NC determined in the above study. The results from the $\mathrm { R V + }$ showed perfect agreement with the decisions obtained by culture/DFA for Influenza A, Influenza B, and RSV. Moreover, the Influenza A subtype decisions (H1, H3, and $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ ) and RSV A and RSV B decisions made by $\mathrm { R V + }$ were all justified by sequencing. Overall, the three-filter method provided excellent predictive power. The cut-offs determined in these studies were used throughout the analytical studies, the reproducibility and precision studies, and in the methods comparison studies.

h. Fresh vs. Frozen equivalency:

Freezing virus-containing samples can potentially compromise viral integrity, which may translate to a lower effective concentration in frozen samples. In order to determine the impact of freezing samples on RVNAT $^ +$ performance, unfrozen negative nasopharyngeal swab samples were obtained and spiked with Influenza A, Influenza B, and RSV viruses to represent all the analytes in the RVNAT $^ +$ . Each virus was tested at multiple concentrations. Paired comparisons were conducted for each sample by testing it before and after freezing. The impact of 2 additional freeze-thaw cycles on the RVNAT $^ +$ (for a total of 3 freeze-thaw cycles) was also assessed.

The ‘Fresh-vs.-Frozen’ comparison study involved testing a total of 60 negative nasopharyngeal (NP) swab samples across 6 unique virus types representing all the targets and analytes in the RVNAT $^ +$ (see Table 13 & 14)). The negative NP swab samples were not pooled; each virus was represented by 10 unique NP swab samples.

Each unique sample had sufficient volume to allow ${ \sim } 4$ tests. Thus, each sample was tested four (4) times: once while fresh or unfrozen and once after each of the three (3) freeze-thaw cycles. Logistically, after a sample was spiked with the appropriate virus, an aliquot of this sample was tested in the $\mathrm { R V + }$ while the sample was ‘fresh’. The residual sample-containing tube was frozen by placing it at or below $- 7 0 ~ ^ { \circ } \mathrm { C }$ . For the first freeze thaw, the sample-containing tube was thawed and an aliquot was removed and tested in the $\mathrm { R V + }$ . This activity constituted the first ‘freeze-thaw’ cycle, and the sample-containing tube was placed again at or below $- 7 0 ~ ^ { \circ } \mathrm { C }$ . The ‘freeze-thaw’ process was repeated for the second and third ‘freeze-thaw’ cycles.

The performance of each virus/analyte after each freeze-thaw cycle was compared to the corresponding results obtained when the sample was ‘fresh’ or unfrozen. Of the 240 tests, 236 tests gave valid results $( 9 8 . 3 \% )$ . The results for all the viruses/analytes within this set of valid results were in perfect agreement between the paired ‘fresh’ and ‘frozen’ samples at each of the three ‘freeze-thaw’ cycles. Overall, the results showed that the $\mathrm { R V + }$ detected the viruses/analytes correctly even after the sample was subjected to three sets of ‘freeze-thaw’ cycles. There was no trend of decreased performance with the number of ‘freeze-thaw’ cycles.

# i. Competitive Inhibition Studies:

Competitive inhibition or interference in the $\mathrm { R V + }$ was assessed in clinically relevant coinfections. By definition, competitive inhibition or interference in a co-infection setting refers to the potential ability of the virus present at a high concentration to inhibit efficient target amplification.

Considering the $\mathrm { R V + }$ is a multiplexed test designed to detect multiple viruses simultaneously, combinations of Influenza A viruses and RSV A/B viruses were identified as clinically relevant co-infections. A set of 12 unique ‘high titer-low titer’ combination samples, each containing a virus at a high titer and another virus at a low titer, were generated. The samples represented combinations of Influenza A subtypes (H1, H3, and 2009 H1N1) and RSV A/B viruses. The levels for individual strains were chosen based on the limits of detection determined for the individual strains. The high titers for the strains were at least 3 log orders higher than their limits of detection and the low titers were held close to the limits of detection.

<table><tr><td colspan="1" rowspan="2">Sample</td><td colspan="2" rowspan="1">High Titer</td><td colspan="2" rowspan="1">Low Titer</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">2X104</td><td colspan="1" rowspan="1">INFA/H3</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">2X104</td><td colspan="1" rowspan="1">INFA/H1</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">2X104</td><td colspan="1" rowspan="1">INFA/2009H1N1</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5X103</td><td colspan="1" rowspan="1">INFA/H3</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5X103</td><td colspan="1" rowspan="1">INFA/H1</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5X103</td><td colspan="1" rowspan="1">INFA/2009H1N1</td><td colspan="1" rowspan="1">10</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">INFA/H3</td><td colspan="1" rowspan="1">2.5X103</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">INFA/H3</td><td colspan="1" rowspan="1">2.5X103</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">INFA/H1</td><td colspan="1" rowspan="1">2X104</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">INFA/H1</td><td colspan="1" rowspan="1">2X104</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">INFA/2009H1N1</td><td colspan="1" rowspan="1">1X104</td><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">100</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">INFA/2009H1N1</td><td colspan="1" rowspan="1">1X104</td><td colspan="1" rowspan="1">RSVB</td><td colspan="1" rowspan="1">5</td></tr></table>

The samples were created by diluting stocks for each virus strain with verified titers into a sample matrix (Universal Transport Media, Copan). Each binary combination was tested in triplicate in the $\mathrm { R V + }$ . The expected calls were compared to the Verigene Calls to assess inhibition. All the calls for the low titer sample and the high titer sample would have to be ‘Detected’ in a sample for there to be no competitive inhibition.

In the $\mathrm { R V + }$ , IC1 and IC2 are verified before providing a valid call. IC1 is an inhibition control for the Target Amplification step and IC2 is a Process Control (bacteriophage, MS2) that is added to the sample. The IC2 goes through all steps of $\mathrm { R V + }$ : Sample Extraction, Target Amplification, and the Verigene Hybridization Test. Both IC1 and IC2 undergo Target Amplification along with the targets in the sample and may be prone to competitive inhibition effects similar to the low titer sample. In all the samples tested, a valid test result was provided indicating that the presence of the high titer target did not inhibit the internal controls during the different steps of the $\mathrm { R V + }$ . Moreover, all the replicates within the binary combinations gave expected calls for both viruses in each sample. The presence of the higher titer viruses did not inhibit detection of the lower titer virus at the levels tested. Thus, in the binary combinations tested in these studies there was no evidence of competitive inhibition.

$j .$ . Carry-Over / Cross-Contamination Study   
A study was performed to assess the carry over/cross-contamination of the $\mathrm { R V + }$ by alternately running ‘High Positive’ samples followed by ‘High Negative’ samples. Multiple Processor $S P$ units were used in the study and each Processor $S P$ was subjected to a comprehensive set of tests.

Viruses and virus subtypes, as well as specific virus strains used in the study are presented in the table below. The concentrations associated with the High Positive (HP) and High Negative (HN) for the virus strains are also listed. A total of six (6) unique strains, representing all the analytes in the $\mathrm { R V + }$ , were tested at High Positive and High Negative concentrations.

Viruses used for Carry-over/Cross-Contamination Study   

<table><tr><td colspan="1" rowspan="1">Virus Type</td><td colspan="1" rowspan="1">Virus Strain</td><td colspan="1" rowspan="1">Level</td><td colspan="1" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="2">INFA/H3</td><td colspan="1" rowspan="2">Influenza A/Wisconsin/67/05</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">200</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.001</td></tr><tr><td colspan="1" rowspan="2">INFA/H1</td><td colspan="1" rowspan="2">Influenza A /Virginia/01/2006</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">2000</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.01</td></tr><tr><td colspan="1" rowspan="2">INFA/2009H1N1</td><td colspan="1" rowspan="2">Influenza A/California/04/2009</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">2000</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.01</td></tr><tr><td colspan="1" rowspan="2">INFB</td><td colspan="1" rowspan="2">Influenza B/Wisconsin/02/2006</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">500</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.001</td></tr><tr><td colspan="1" rowspan="1">RSVA</td><td colspan="1" rowspan="1">RSV A/Strain A2</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">5000</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.01</td></tr><tr><td colspan="1" rowspan="2">RSVB</td><td colspan="1" rowspan="2">RSV B/Washington/18537/62</td><td colspan="1" rowspan="1">High Positive</td><td colspan="1" rowspan="1">1000</td></tr><tr><td colspan="1" rowspan="1">High Negative</td><td colspan="1" rowspan="1">0.001</td></tr></table>

The samples were divided into two sets as described. Each set comprised a total of six (6) unique alternating High Positive and High Negative samples. The samples within each set were tested sequentially in the order described. Each set was tested in triplicate on separate Processor $S P \mathrm { s }$ (2 Sets x 3 Processor $S P s = 6$ Processor $S P s$ . Thus, a total of 36 samples [6 strains x 2 $\left( \mathrm { H P + H N } \right)$ ) x 3 replicates] were tested across 6 Processor SPs over 6 sequential test runs. The 6 test runs were conducted over a 3 day period.

Considering each High Positive test was followed by a High Negative test of a different viral type, ‘Observed Results’ that differed from ‘Expected Results’ for the High Negative test would be considered as carry over/cross-contamination. The only exception to this would be if a High Negative sample gave a positive result for that particular High Negative viral strain as by definition, a High Negative sample should be detected $< 5 \%$ . Regardless, this would be distinguished easily from the previous viral type because of the order of testing.

Decisions regarding the presence or absence of a virus were based on the ‘Calls’ (‘Detected’ or ‘Not Detected’) displayed by the Verigene® Reader. Carry over/crosscontamination would be a probable cause if the Verigene® System detected a virus other than or in addition to the one that was intended in the sample.

All High Positive samples yielded the expected ‘Detected’ results for the intended virus and ‘Not Detected’ results for the other viruses or analytes. All but one of the High Negative samples yielded a ‘Not Detected’ for the intended virus as well as all other viruses. One Influenza B high negative sample gave a ‘Detected’ call for Influenza B. High Negative samples are expected to be positive $< 5 \%$ of the time; this result was expected. Moreover, the High Positive preceding this sample was Influenza A 2009H1N1 confirming that the Influenza B result was not due to sample carry over. Overall, the studies confirmed that there was no evidence of carry over/cross-contamination from the High Positive samples, or any other internal or external sources, in any of the three steps of the $\mathrm { R V + }$ : Sample Extraction, Target Amplification, and Verigene Test.

# k. Interfering Substances Studies

The potential inhibitory effects of interfering substances or interferents that may be encountered in nasopharyngeal specimens on the $\mathrm { R V + }$ was assessed. A set of 22 interferents and two types of influenza vaccines were identified for a total of 24 potential interferents.

All the interferents (except the vaccines) were dissolved into a sample matrix, Universal Transport Medium (Copan), to make intermediate stocks. The concentration of the interferent was based on the maximum dosage and the interferent’s solubility in the sample matrix. Next, each interferent was spiked into samples containing different combinations of viral strains. A set of four unique virus combinations were chosen to represent all the analytes in the $\mathrm { R V + }$ . The different target combinations and interferents amounted to 88 unique samples (22 interferents x 4 unique samples). The titers chosen for each virus strain were near their limits of detection. Samples containing viruses and the interferents were then processed through the $\mathrm { R V + }$ . The vaccines were tested as a dilution series.

In the $\mathrm { R V + }$ , Internal Control (IC1) and Internal Control 2 (IC2) are verified before providing a valid call. IC1 is an inhibition control for the Target Amplification step and IC2 is a process control (bacteriophage, MS2) that is added to the sample automatically prior to the extraction step. The IC2 goes through all steps of $\mathrm { R V + }$ : Sample Extraction, Target Amplification, and the Verigene Hybridization Test. In all the samples tested, a valid test result was provided indicating that the interferents did not pose a problem during the different steps of the $\mathrm { R V + }$ . Moreover, in each of these samples the expected calls were obtained for each of the analytes tested; none of the interferents at the tested concentration prevented the $\mathrm { R V + }$ from making the correct ‘Detected’ calls for the virus strains and analytes present in samples containing the interferents.

<table><tr><td colspan="1" rowspan="1">ActiveInterferent</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Amount</td></tr><tr><td colspan="1" rowspan="1">No Interferent</td><td colspan="1" rowspan="1">Universal Transport Media</td><td colspan="1" rowspan="1">Not Applicable</td></tr><tr><td colspan="1" rowspan="1">Human Blood</td><td colspan="1" rowspan="1">Human Blood</td><td colspan="1" rowspan="1">5% v/v</td></tr><tr><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">WalFour Nasal Spray</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Oxymetazoline</td><td colspan="1" rowspan="1">Anefrin Nasal Spray</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">NaCl</td><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Luffa opperculata</td><td colspan="1" rowspan="1">Similasan Sinus Relief</td><td colspan="1" rowspan="1">1.0% v/v</td></tr><tr><td colspan="1" rowspan="1">Benzocoaine</td><td colspan="1" rowspan="1">Anbesol</td><td colspan="1" rowspan="1">0.5% v/v of sample</td></tr><tr><td colspan="1" rowspan="1">Beclomethasonedipropionate</td><td colspan="1" rowspan="1">Beclomethasone dipropionate</td><td colspan="1" rowspan="1">16 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">50 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">Flunisolide</td><td colspan="1" rowspan="1">58 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Triamcinolone acetonide</td><td colspan="1" rowspan="1">Triamcinolone acetonide</td><td colspan="1" rowspan="1">5.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">Budesonide</td><td colspan="1" rowspan="1">25 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Mometasone Furoate</td><td colspan="1" rowspan="1">Mometasone Furoate</td><td colspan="1" rowspan="1">2.5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Fluticasone propionate</td><td colspan="1" rowspan="1">Fluticasone propionate</td><td colspan="1" rowspan="1">5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Sulphur</td><td colspan="1" rowspan="1">Boiron</td><td colspan="1" rowspan="1">4.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">Menthol</td><td colspan="1" rowspan="1">0.5 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">5 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">0.150 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">0.1 mg/mL</td></tr><tr><td colspan="1" rowspan="1">OseltamamivirPhosphate</td><td colspan="1" rowspan="1">Tamiflu</td><td colspan="1" rowspan="1">33 μg/mL</td></tr><tr><td colspan="1" rowspan="1">Fluticasone furoate</td><td colspan="1" rowspan="1">Veramyst</td><td colspan="1" rowspan="1">10% v/v</td></tr><tr><td colspan="1" rowspan="1">Galphimia GlaucaHomeopathic Remedy</td><td colspan="1" rowspan="1">Boiron</td><td colspan="1" rowspan="1">115 μg/mL</td></tr><tr><td colspan="1" rowspan="1">HistaminumHydrochloricum</td><td colspan="1" rowspan="1">Boiron</td><td colspan="1" rowspan="1">115 μg/mL</td></tr><tr><td colspan="1" rowspan="1">FluMist InfluenzaVaccine Live, Intranasal</td><td colspan="1" rowspan="1">Medimmune</td><td colspan="1" rowspan="1">•Influenza A and subtypes H1 and H3: Nodetection at or below ~5x105 dilution.*Influenza B: No detection at or below~5x106 dilution.*</td></tr><tr><td>FluLaval</td><td>GlaxoSmithKline/ID Biomedical</td><td>•Influenza A and subtypes H1 and H3: No detection at or below ~1x106 dilution.* •Influenza B: No detection at or below ~1x107 dilution.*</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable, performance of the assay was evaluated in comparison to the gold standard/reference method, viral culture followed by DFA and/or viral culture followed by sequencing

b. Matrix comparison: Not Applicable.

3. Clinical studies:

a. Clinical Sensitivity and Specificity:

The clinical performance of the $\mathrm { R V + }$ assays was established through a study conducted at three external clinical sites: TriCore Reference Laboratory, Albuquerque, NM (TC); Midwest Research Institute, Palm Bay, FL (MRI); and Medical College of Wisconsin, Milwaukee, WI (MCW). Across the three sites, there were one Verigene Reader and 12 Verigene Processor SPs utilized per site for testing during the study. The testing for this study was performed using multiple lots (at least 4 lots at each site) of Test Cartridges, Extraction Trays and Amplification Trays at each site. Verigene Processor SPs and Verigene Readers used in these studies were standard production instruments. The software revisions were validated for intended use according to documented test protocols and developed according to Standard Operating Procedures.

Clinical nasopharyngeal specimens for the clinical study were collected prospectively at three collection laboratory sites during the 2008-2009 and 2009-2010 respiratory seasons. The three collection sites were TriCore Reference Laboratories (TC), Esoterix/LabCorp (ES) and University of North Carolina (UNC). Two of the sites, Esoterix and UNC collected specimens in universal transport medium (UTM) and one site, TriCore, collected specimens in M5 medium.

Culture-based/DFA confirmation was performed on these prospectively-collected specimens while still fresh and in an unfrozen state as part of normal laboratory practice. Frozen specimens were only used when confirmation was performed by using nucleic acid amplification tests (NAAT). Each site then saved the residual from these specimens, which were de-identified on-site, frozen at $- 7 0 ^ { \circ } \mathrm { C }$ , and shipped on dry ice to Nanosphere. At Nanosphere, the specimens were stored at or below $- 7 0 ~ ^ { \circ } \mathrm { C }$ .

Prior to distribution to the test sites, the specimens were randomized by using the random generator function in Microsoft Excel (2007). Each specimen was blinded and labeled with a Nanosphere-generated unique identifier while removing the collection site identifier. All specimen culture/DFA were kept blinded to all study personnel until $\mathrm { R V + }$ testing was completed. The specimens were shipped to the testing sites overnight on dry ice. The sites stored the specimens at $- 7 0 \mathrm { { } ^ { \circ } C }$ before and after testing.

A total of 1022 specimens were tested at the three study sites (TC—314; MRI—385; MCW— 323). All tests were performed by laboratory personnel at the three sites using the $\mathrm { R V + }$ on the Verigene System. Multiple lots of Amplification Trays, Extraction Trays and Test Cartridges were used at each site. The Influenza A, Influenza B and RSV results in the methods comparison study were compared to results obtained for the specimens using culture-based methods confirmed with a FDA-cleared DFA test Influenza A subtyping and RSV subtyping results were compared to results obtained for the specimens utilizing bi-directional sequencing. Discordant results were resolved utilizing bi-directional sequencing or an acceptable NAAT–based comparator. Bi-directional sequencing was performed at an independent laboratory (ACGT; Wheeling, IL) and footnoted in the performance comparison tables.

Per protocol, if a test resulted in a “No Call” or a “pre-analysis error” (“pre-ae”), the specimen could be repeat tested. There were 34 “No Calls” and 25 “pre-aes” in the study. Of these, 57 of the 59 specimens were repeat tested successfully. The other two “No Call” specimens repeated as “No Calls”. Thus, in this study the total “No-Call” rate was $3 . 3 \%$ (34/1022) and the “pre-ae” failure rate was $2 . 4 \%$ (25/1022). The “pre-ae”, by definition, is any test that did not go to completion regardless of the cause. A “pre-ae” may occur due to a number of causes including a user- initiated procedure termination (for example, upon realizing that an incorrect specimen was loaded) or an instrumentinitiated procedure termination (for example, because internal checks detect an unexpected event), or due to an instrument failure, or non-availability of the Test Substrate for analysis (due to test consumable breakage, etc).

Combined results for the methods comparison study across the three sites are presented and reflect the results after all “no call” results were repeated per the protocol and the proposed package insert guidance. Results from the $\mathrm { R V + }$ were initially compared to results from culture-based DFA methods. An FDA cleared NAAT was utilized to resolve seven discrepant results between the $\mathrm { R V + }$ and culture/DFA which had failed sequencing. Bi-directional sequencing was used to subtype Influenza A, differentiate RSV A and RSV B and for discordant testing. The tables below show results for Influenza A, Influenza B, and RSV compared to culture/DFA with discordant resolution footnoted. For subtyping results for Influenza A and RSV specimens, $\mathrm { R V + }$ results are compared to a composite comparator result obtained from culture/DFA followed by bi-directional sequencing. Percent positive and negative agreement for each table are provided as well as the lower and upper two-sided $9 5 \%$ confidence limits that were calculated using the exact binomial methods in SAS 9.1.3 from: http://172.16.20.9/stat.html

The Influenza A combined results demonstrate strong results for the $\mathrm { R V + }$ assay with a Sensitivity of $9 8 . 7 \%$ $( 9 6 . 8 – 9 9 . 5 \% )$ and Specificity of $9 3 . 2 \%$ $9 1 . 1 { - } 9 4 . 8 \% )$ . The lower value for Percent Negative Agreement is due to a higher number of “false positives (FPs)” that are apparent when the $\mathrm { R V } +$ result is compared to culture/DFA/NAAT. Almost all of these (FPs) are considered TPs (true positives) when compared to bidirectional sequencing.

All 1022 patient nasopharyngeal swab specimens are accounted for in the combined tables for Influenza A, Influenza B, and RSV. However, a small number of Influenza A specimens that failed sequencing or did not produce a subtype result could not be placed in the subtyping tables. For the Influenza A subtypes, 5 out of 1022 results $( 0 . 4 9 \% )$ were either not subtyped by the $\mathrm { R V + }$ or failed sequencing (i.e., results for only 1017 specimens in the Influenza A subtype tables). Influenza B did not require sub-typing and, therefore, the table contains all 1022 specimen results. Three specimens failed RSV subtype sequencing and could not be included in the RSV subtyping tables (i.e., results for only 1019 specimens in the RSV subtype tables).

# Influenza A Combined Results

A: Influenza A Results: $\mathbf { R V + }$ vs. Culture/DFA

<table><tr><td rowspan=2 colspan=2>INFA</td><td rowspan=1 colspan=3>Culture/DFA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>311a,c</td><td rowspan=1 colspan=1>48 b</td><td rowspan=1 colspan=1>359</td><td rowspan=1 colspan=1>Sensitivity = 98.7% (96.8%-99.5%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4 de, f</td><td rowspan=1 colspan=1>659</td><td rowspan=1 colspan=1>663</td><td rowspan=1 colspan=1>Specificity = 93.2% (91.1%-94.8%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>315</td><td rowspan=1 colspan=1>707</td><td rowspan=1 colspan=1>1022 g</td><td rowspan=1 colspan=1></td></tr></table>

a 1 specimen was positive for Influenza A by both culture and $\mathrm { R V } +$ . No sub-typing was observed on the $\mathrm { R V } +$ result. Also, the specimen failed subtype sequencing for H1, H3 and $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ . $^ { \mathsf { b } } 4$ specimens were Influenza A positive by $\mathrm { R V } +$ . No subtyping was observed on the $\mathrm { R V + }$ . All 4 specimens were culture negative and failed sequencing. $^ \mathrm { c } 1$ specimen was Influenza $\mathrm { A } / \mathrm { H } 1$ by $\mathrm { R V } +$ and positive for Influenza A by culture. Sequencing resulted in a positive result for Influenza A/2009H1N1. ${ \bf { \check { d } } } _ { 1 }$ specimen was negative by $\mathrm { R V } +$ for Flu A and positive for Flu B. By culture/DFA the specimen was Flu A positive. By sequencing the specimen was positive for Flu B and negative for Flu A. By NAAT the specimen was positive for Flu B and negative for Flu A and RSV. $^ \mathrm { ~ e ~ } _ { 1 }$ specimen was negative for Flu A and positive for RSV B by $\mathrm { R V + }$ . By culture/DFA, the specimen was Flu A positive. By sequencing the specimen was positive for RSV B and negative for Flu A. By NAAT the specimen was positive for RSV and negative for Flu A and Flu B. $^ \mathrm { f } 2$ specimens were negative by $\mathrm { R V } +$ but Flu A positive by culture. By sequencing both specimens were negative for Flu A. Both specimens were negative for Flu A (and negative for Flu B and RSV B) by NAAT. g 5 specimens failed sequencing and are not included in the Subtyping Tables.

B: Influenza A Subtype H3 Results: $\mathbf { R V + }$ vs. Culture/DFA/Sequencing   

<table><tr><td rowspan=2 colspan=2>INFA/H3</td><td rowspan=1 colspan=3>Culture/DFA/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>Sensitivity = 100% (96.6%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>909</td><td rowspan=1 colspan=1>909</td><td rowspan=1 colspan=1>Specificity = 100% (99.6%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>108</td><td rowspan=1 colspan=1>909</td><td rowspan=1 colspan=1>1017</td><td rowspan=1 colspan=1></td></tr></table>

C: Influenza A Subtype H1 Results: $\mathbf { R V + }$ vs. Culture/DFA/Sequencing   

<table><tr><td rowspan=2 colspan=2>INFA/H1</td><td rowspan=1 colspan=3>Culture/DFA/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>Sensitivity = 100% (91.0%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>977</td><td rowspan=1 colspan=1>977</td><td rowspan=1 colspan=1>Specificity = 99.9% (99.4%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>978</td><td rowspan=1 colspan=1>1017</td><td rowspan=1 colspan=1></td></tr></table>

D: Influenza A Subtype 2009 H1N1 Results: $\mathbf { R V + }$ vs. Culture/DFA/Sequencing   

<table><tr><td rowspan=2 colspan=2>INFA/2009 H1N1</td><td rowspan=1 colspan=3>Culture/DFA/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>206</td><td rowspan=1 colspan=1>Sensitivity = 99.5% (97.3%-99.9%) 95% CI</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>811</td><td rowspan=1 colspan=1>Specificity = 100% (99.5%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>810</td><td rowspan=1 colspan=1>1017</td><td rowspan=1 colspan=1></td></tr></table>

# Influenza B

A: Influenza B Results: $\mathbf { R V + }$ vs. Culture/DFA results   

<table><tr><td rowspan=2 colspan=2>INFB</td><td rowspan=1 colspan=3>Culture/DFA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>Sensitivity = 100% (91.8%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>976</td><td rowspan=1 colspan=1>Specificity = 99.7% (99.1%-99.9%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>1022</td><td rowspan=1 colspan=1></td></tr></table>

a 1 specimen was positive for Influenza B by $\mathbf { R V + }$ and negative for Influenza B by culture/DFA. This specimen was tested by NAAT assay and found to be positive for Influenza B. Specimen was positive for Influenza B by sequencing and upon repeat culture/DFA. $^ { \flat } 2$ specimens were positive for Influenza B by $\mathbf { R V + }$ and negative for Influenza B by culture. Both specimens were positive for Influenza B by sequencing.

RSV

A: RSV Results: $\mathbf { R V + }$ vs. Culture/DFA results   

<table><tr><td rowspan=2 colspan=2>RSV</td><td rowspan=1 colspan=3>Culture/DFA</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>5,d</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>Sensitivity = 97.2% (92.1%-99.0%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3 b, c</td><td rowspan=1 colspan=1>910</td><td rowspan=1 colspan=1>913</td><td rowspan=1 colspan=1>Specificity = 99.5% (98.7%-99.8%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>915</td><td rowspan=1 colspan=1>1022</td><td rowspan=1 colspan=1></td></tr></table>

a 1 specimen was positive for RSV B by $\mathbf { R V } +$ and by sequencing but negative for RSV by culture/DFA. Specimen was positive for RSV by NAAT. $^ { \mathrm { ~ b ~ } } 1$ specimen was negative for RSV by $\mathbf { R V + }$ and by sequencing but positive for RSV by culture and NAAT assay. $^ \mathrm { c } 2$ specimens were negative for RSV by $\mathbf { R V + }$ and positive for RSV by culture. The specimens were negative for RSV by NAAT assay and failed sequencing. ${ ^ \mathrm { d } } _ { 4 }$ specimens were positive for RSV A or RSV B by $\mathbf { R V } +$ but negative by culture: 1 was positive for RSV A and 3 were positive for RSV B by $\mathbf { R V + }$ and by sequencing. $^ \mathrm { e } 3$ specimens failed subtype sequencing and are not included in the Subtyping Tables.

<table><tr><td rowspan=2 colspan=2>RSV A</td><td rowspan=1 colspan=3>Culture/DFA/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>Sensitivity = 100% (93.7%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>962</td><td rowspan=1 colspan=1>962</td><td rowspan=1 colspan=1>Specificity = 100% (99.6%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>962</td><td rowspan=1 colspan=1>1019</td><td rowspan=1 colspan=1></td></tr></table>

f 1 specimen was positive for both RSV A and RSV B (dual infection) by $\mathbf { R V + }$ and was culture positive for RSV. By sequencing specimen 0300 was positive for both RSV A and RSV B.

C: RSV B Subtype Results: $\mathbf { R V + }$ vs. Culture/DFA/Sequencing   

<table><tr><td rowspan=2 colspan=2>RSV B</td><td rowspan=1 colspan=3>Culture/DFA/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>RV+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>53f</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>Sensitivity = 100% (93.2%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>Specificity = 99.9% (99.6%-100%) 95% CI</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>966</td><td rowspan=1 colspan=1>1019</td><td rowspan=1 colspan=1></td></tr></table>

f 1 specimen was positive for both RSV A and RSV B (dual infection) by $\mathbf { R V + }$ and was culture positive for RSV. By sequencing specimen 0300 was positive for both RSV A and RSV B.

4. Clinical cut-off:

Not Applicable

5. Expected values/Reference range:

While the timing and duration of flu and RSV seasons can vary, the fall and winter months are the peak times in the US. RSV infections in the US typically occur during annual community outbreaks in the late fall, winter and early spring, and there may be variation in the timing of outbreaks between regions and between communities in the same region.

During the 2008-2009 and 2009-2010 flu seasons, $21 \%$ of 519,543 samples and $20 \%$ of 456,302 samples tested for influenza were positive for either Influenza A or Influenza B [based on data from laboratories in the US which collaborated with World Health Organization (WHO) and National Respiratory and Enteric Virus Surveillance System (NREVSS)]. During the 2006 2007 and 2007- 2008 flu seasons, the prevalence was $1 3 \%$ of 179,268 samples tested and $1 8 \%$ of 225,329 samples tested, respectively.

According to data reported to the NREVSS, the prevalence of RSV was $1 5 \%$ in the 404,798 samples tested for RSV during the 2008- 2009 season and $1 6 \%$ in the 369,944 samples tested during the 2007-2008 season.

In the Verigene Respiratory Virus Plus Nucleic Acid Test multi-site methods comparison study, which analyzed 1022 samples collected during the 2008-2009 and 2009-2010 flu season, the prevalence of Influenza A was $3 0 . 0 \%$ , for Influenza B was $4 . 3 \%$ , and of RSV was $1 0 . 2 \%$ . No dual infections were detected by culture and DFA, however, one specimen was detected by the Verigene Respiratory Virus Plus Nucleic Acid Test to have both RSV A and RSV B. It is recommended that the samples undergo repeat testing if nucleic acids from all three analytes are detected in a single sample.

N. Instrument Name: Verigene System

# O. System Descriptions:

1. Modes of Operation:

The $\mathrm { R V + }$ on the Verigene platform functions as a sample-to-answer system in which all sample processing steps as well as detections is carried out on a single use, disposable test cartridge.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X__ or No

3. Specimen Identification:

Each test cartridge handles one sample and has a unique barcode identifier that is scanned into the Verigene system. Upon scanning the user is prompted to enter sample information via barcode-reader or touch-screen interface thus linking patient information to test cartridge. The cartridge and consumables are loaded onto the Verigene system followed by the patient sample into the designated sample well and the test is initiated.

4. Specimen Sampling and Handling:

The steps automated within the Processor SP use test reagents that are self contained in the test cartridge and include:

1. Sample Preparation/Extraction – Magnetic bead-based viral RNA isolation from nasopharyngeal swab specimens   
2. Target Amplification – Multiplex RT-PCR-based amplification of the eluted viral RNA to generate virus-specific amplicons   
3. Verigene Hybridization Test – Gold nanoparticle probe-based detection of the virus-specific amplicons on a microarray

Upon completion of Specimen processing the cartridge is removed form the Verigene processor and placed into the Verigene Reader for analysis

5. Calibration:

Not required. Test cartridges are single use and part of a closed system.

6. Quality Control:

A series of internal controls used in conjunction with procedural checks monitors instrument functionality, performance, fluidics, reagent integrity, and result determination based on a pre-defined decision algorithm.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above: None

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.