# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number:

K103766

# B. Purpose for Submission:

New Device

# C. Measurand:

The Xpert Flu Assay is a rapid, automated in vitro diagnostic test for qualitative detection and differentiation of Influenza A, Influenza B and Influenza A, subtype 2009 H1N1 from nasal aspirates/washes (NA/W) and nasopharyngeal (NP) swab specimens from patients with signs and symptoms of respiratory infection.

# D. Type of Test:

Multiplex nucleic acid assay for qualitative detection and differentiation of influenza A, influenza B and influenza A, subtype 2009 H1N1 from nasal aspirates/washes (NA/W) and nasopharyngeal (NP) swab specimens including nucleic acid isolation and multiplex real-time RT-PCR amplification using the Gene Xpert Dx and Infinity Systems.

# E. Applicant:

Cepheid

# F. Proprietary and Established Names:

Xpert $^ \mathrm { \textregistered }$ Flu, Xpert Flu Assay

# G. Regulatory Information:

1. Regulation section: 866.3980 - Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code: OQW, OCC, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use:

The Cepheid $^ \mathrm { \textregistered }$ Xpert Flu Assay is an automated, multiplex real-time RT-PCR assay intended for the in vitro qualitative detection and differentiation of influenza A, influenza B and 2009 H1N1 influenza viral RNA. The Xpert Flu Assay uses nasal aspirates/washes and nasopharyngeal swab specimens collected from patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. The Xpert Flu Assay is intended as an aid in the diagnosis of influenza.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2009-2010 influenza season when 2009 H1N1 influenza was the predominant influenza A virus in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Requires the GeneXpert Dx (software version 2.1 or 4.0) or the GeneXpert Infinity system (software version 4.0) from Cepheid.

# I. Device Description:

The Xpert Flu Assay is an automated in vitro diagnostic test for qualitative detection and differentiation of influenza A, influenza B and influenza A subtype 2009 H1N1 from nasal aspirates/washes (NA/W) and nasopharyngeal (NP) swab specimens from patients with signs and symptoms of respiratory infection. The assay is performed on the Cepheid GeneXpert $^ \mathrm { \textregistered }$ instrument systems.

The GeneXpert instrument systems automate and integrate sample purification, nucleic acid amplification, and detection of the target sequence in simple or complex samples using real-time PCR and rRT-PCR assays. The GeneXpert Instrument System family comprises a GeneXpert (GX) instrument, GX-I, GX-IV, GX-IV or GeneXpert Infinity48, with 1, 4, 16 and 48 modules respectively, a computer, and preloaded software for running tests and viewing the results. The GeneXpert Infinity also contains robotic features for cartridge handling. Each module contains a syringe drive for dispensing fluids, an ultrasonic horn for lysing cells or spores, a valve drive for sample movement, and I-CORE $^ \mathrm { \textregistered }$ thermocycler for performing real-time PCR and detection.

All systems require the use of the assay-specific single-use disposable cartridges that hold the PCR reagents and host the PCR process. The patented single-use cartridges contain: (1) eleven chambers for holding sample, reagents, or other materials, (2) a valve body composed of a plunger and syringe barrel, (3) a rotary valve system for controlling the movement of fluids between chambers, (4) an area for capturing, concentrating, washing, and lysing cells, (5) dry real-time PCR reagents, (6) an integrated PCR reaction tube that can be automatically filled by the instrument, and (7) liquid reagents. To eliminate testto-test contamination, all fluids including amplicons, are contained within the disposable cartridge. The instrument never comes into contact with any fluids within the cartridge. Each disposable cartridge is intended to test one sample. Cartridges are not re-usable.

The GeneXpert cartridge is loaded onto the GeneXpert $^ \mathrm { \textregistered }$ Instrument System platform, which performs hands-off automated sample processing and real-time PCR for detection of RNA or DNA. Summary and detailed test results are obtained in 75 minutes and are displayed in tabular and graphic formats.

A sample processing control (SPC) and a system control (probe check control) are controls utilized by the GeneXpert Instrument System platform. The SPC is pre-loaded into the GeneXpert cartridge provided with the assay. The SPC is an encapsidated RNA made up of recombinant fragments developed so that there is no homology to the influenza genome. The SPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the PCR reaction to reduce the possibility of false negative results. The SPC also ensures the PCR reaction conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. The Probe Check Control verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity and dye stability.

Commercially-available external controls may also be run in accordance with local, state, and federal accrediting organizations, as applicable.

The Xpert Flu Assay includes reagents for the simultaneous detection of the target viruses. The primers and probes in the Xpert Flu Assay detect the presence of nucleic acid sequences for Influenza A (Flu A), Influenza B (Flu B) and Influenza A sub-type 2009 H1N1 (2009 H1N1). The assay for detection of Influenza A uses primers and probes designed against a region in segment 8 of the influenza genome that encodes the matrix gene. The assay for differentiation of 2009 H1N1 targets a region in segment 4 that encodes the hemagglutinin gene. The assay for detection of RNA from Influenza B uses primers and probes designed against segment 4 in a region that encodes the hemagglutinin gene.

Nasal aspirates/washes (NA/W) and nasopharyngeal (NP) swab specimens from patients suspected of having influenza are collected in Universal Transport Medium (UTM) and transported to the GeneXpert area.

The specimen is briefly mixed by inverting the collection tube 5 times. For NP swab specimens, using the supplied $3 0 0 \mu \mathrm { L }$ transfer pipette, $3 0 0 \mu \mathrm { L }$ of the sample are transferred to the sample chamber (large opening) of the Xpert Flu Assay Cartridge. For NA/W specimens, the specimen is first diluted by using the supplied $3 0 0 \mu \mathrm { L }$ transfer pipette and transferring $3 0 0 \mu \mathrm { L }$ of the specimen 2 times $6 0 0 \mu \mathrm { L }$ total) into a fresh $3 ~ \mathrm { m L }$ $( 3 0 0 0 ~ \mu \mathrm { L } )$ UTM tube. The diluted specimen is mixed by inverting the tube 5 times. Using a clean supplied $3 0 0 \mu \mathrm { L }$ transfer pipette, $3 0 0 \mu \mathrm { L }$ of the diluted specimen are transferred to the chamber with the large opening of the Xpert Flu Assay Cartridge. Reagent 1 (Binding Reagent) is dispensed by squeezing the entire contents into the chamber with the small opening of the Xpert Flu Assay Cartridge. The GeneXpert $^ \mathrm { \textregistered }$ Instrument system performs sample preparation by mixing the sample with the sample processing control and treatment reagents, capturing the nucleic acids on a cellulose column. The column is washed to remove contaminants and the purified nucleic acids are eluted and then mixed with dry real-time RT-PCR (rRT-PCR) reagents and transferred into the PCR tube for rRT-PCR and detection of Flu RNA. The complete process takes about 75 minutes.

The results are interpolated by the GeneXpert Instrument Systems software from measured fluorescent signals and embedded calculation algorithms and will be shown in the “View Results” window. All possible final test results are described below. The Xpert Flu Assay provides test results for influenza A, influenza B and influenza A, subtype 2009 H1N1, according to the following algorithms:

Flu A:   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=2 colspan=1>Test Result</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Flu A NEGATIVE*</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Flu A POSITIVE</td></tr></table>

\*If SPC is INVALID, overall test result is INVALID; if 2009 H1N1 is positive and FLU A is negative, overall test result is INVALID

Flu B:   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=2 colspan=1>Test Result</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Flu B NEGATIVE*</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>Flu B POSITIVE</td></tr></table>

\*If SPC is INVALID, overall test result is INVALID

2009 H1N1:   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=2 colspan=1>Test Result</td></tr><tr><td rowspan=1 colspan=1>Influenza A, 2009 H1N1</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>2009 H1N1 NOTDETECTED*</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>2009 H1N1DETECTED**</td></tr></table>

\*If SPC is INVALID, overall test result is INVALID \*\*If Flu A is NEGATIVE, overall test result is INVALID

The possible results of the GeneXpert Flu Assay are:

# Flu A POSITIVE; 2009 H1N1 NOT DETECTED; Flu B NEGATIVE

Flu A target RNA detected; 2009 H1N1 target RNA not detected; Flu B target RNA not detected.

• The Flu A target has a Ct within the valid range and endpoint above the minimum setting.   
• SPC – NA (not applicable); SPC is ignored because the Flu A target amplification may compete with this control.   
• Probe Check – PASS; all probe check results pass.

# Flu A POSITIVE; 2009 H1N1 DETECTED; Flu B NEGATIVE

Flu A target RNA detected; 2009 H1N1 target RNA detected; Flu B target RNA not detected.

• The Flu A target has a Ct within the valid range and endpoint above the minimum setting. • The 2009 H1N1 target has a Ct within the valid range and endpoint above the

minimum setting.

SPC – NA (not applicable); SPC is ignored because the Flu A and 2009 H1N1 target amplification may compete with this control.   
Probe Check – PASS; all probe check results pass.

# Flu A NEGATIVE; 2009 H1N1 NOT DETECTED; Flu B POSITIVE

Flu B target RNA detected; Flu A target RNA not detected; 2009 H1N1 target RNA not detected.

• The Flu B target has a Ct within the valid range and endpoint above the minimum setting.   
• SPC – NA (not applicable); SPC is ignored because the Flu B target amplification may compete with this control.   
• Probe Check – PASS; all probe check results pass.

# Flu A NEGATIVE; 2009 H1N1 NOT DETECTED; Flu B NEGATIVE

Flu A target RNA not detected; 2009 H1N1 target RNA not detected; Flu B target RNA not detected. SPC meets acceptance criteria.

• Flu A, 2009 H1N1 and Flu B target RNAs are not detected.   
• SPC – PASS; SPC has a Ct within the valid range and endpoint above the minimum setting. Probe Check – PASS; all probe check results pass.

# Flu A POSITIVE; 2009 H1N1 NOT DETECTED; Flu B POSITIVE

Flu A target RNA detected; 2009 H1N1 target RNA not detected; Flu B target RNA detected.

The Flu A target has a Ct within the valid range and endpoint above the minimum setting. The Flu B target has a Ct within the valid range and endpoint above the minimum setting. SPC – NA (not applicable); SPC is ignored because the Flu A and 2009 H1N1 target amplification may compete with this control. • Probe Check – PASS; all probe check results pass.

# Flu A POSITIVE; 2009 H1N1 DETECTED; Flu B POSITIVE

Flu A target RNA detected; 2009 H1N1 target RNA detected; Flu B target RNA detected.

The Flu A target has a Ct within the valid range and endpoint above the minimum setting. • The 2009 H1N1 target has a Ct within the valid range and endpoint above the minimum setting. • The Flu B target has a Ct within the valid range and endpoint above the minimum setting. SPC – – NA (not applicable); SPC is ignored because the Flu A and 2009 H1N1 target amplification may compete with this control. • Probe Check – PASS; all probe check results pass

# INVALID

SPC does not meet acceptance criteria. Presence or absence of the target RNAs cannot be determined. Repeat test according to the instructions in the Retest Procedure section below.

SPC – FAIL; SPC result is negative and the SPC Ct is not within valid range and the endpoint is below the minimum setting. • Probe Check – PASS; all probe check results pass.

# INVALID

Presence or absence of 2009 H1N1 target RNA cannot be determined. Repeat test according to the instructions in the Retest Procedure section below.

• Flu A target RNA not detected and 2009 H1N1 target RNA detected.   
• SPC – NA (not applicable); SPC is ignored because a target amplified.   
• Probe Check – PASS; all probe check results pass.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Gen-Probe Prodesse, ProFlu+ Assay Focus Diagnostics Simplexa Assay

2. Predicate 510(k) number(s): K073029 K100148

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Device</td><td colspan="2" rowspan="1">Predicates</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Cepheid Xpert Flu</td><td colspan="1" rowspan="1">Gen-Probe Prodesse,Inc. ProFLU+</td><td colspan="1" rowspan="1">Focus SimplexaInfluenza A H1N1</td></tr><tr><td colspan="1" rowspan="1">510(k) No.</td><td colspan="1" rowspan="1">To be assigned</td><td colspan="1" rowspan="1">K073029</td><td colspan="1" rowspan="1">K100148</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">866.3332 and 866.3980</td><td colspan="1" rowspan="1">866.3980</td><td colspan="1" rowspan="1">866.3332</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OQW, OCC, 001</td><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">OQW</td></tr><tr><td colspan="1" rowspan="1">Device Class</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">Ⅱ</td></tr><tr><td colspan="1" rowspan="1">Technology/Detection</td><td colspan="1" rowspan="1">Multiplex real time RT/PCR</td><td colspan="1" rowspan="1">Multiplex real timeRT/PCR</td><td colspan="1" rowspan="1">Multiplex real time RT/PCR</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">An automated, multiplex real-time RT-PCR assay intendedfor the in vitro qualitativedetection and differentiation ofinfluenza A, influenza B and2009 H1N1 influenza viralRNA. The Xpert Flu Assayuses nasal aspirates/washes</td><td colspan="1" rowspan="1">A multiplex Real TimeRT-PCR in vitrodiagnostic test for therapid and qualitativedetection anddiscrimination ofInfluenza A Virus,Influenza B Virus, and</td><td colspan="1" rowspan="1">For use on the 3M IntegratedCycler as part of theMicrofluidic MolecularSystem for the in vitroqualitative detection anddifferentiation of influenza Aand 2009 H1N1 influenzaviral RNA in nasopharyngeal</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">and nasopharyngeal swabspecimens collected frompatients with signs andsymptoms of respiratoryinfection in conjunction withclinical and epidemiologicalrisk factors. The Xpert FluAssay is intended as an aid inthe diagnosis of influenza.Negative results do notpreclude influenza virusinfection and should not beused as the sole basis fortreatment or other patientmanagement decisions.Performance characteristics forinfluenza A were establishedduring the 2009-2010influenza season when 2009H1N1 influenza was thepredominant influenza A virusin circulation. When otherinfluenza A viruses areemerging, performancecharacteristics may vary.If infection with a novelinfluenza A virus is suspectedbased on current clinical andepidemiological screeningcriteria recommended bypublic health authorities,specimens should be collectedwith appropriate infectioncontrol precautions for novelvirulent influenza viruses andsent to state or local healthdepartment for testing. Viralculture should not beattempted in these cases unlessa BSL 3+ facility is availableto receive and culturespecimens.</td><td colspan="1" rowspan="1">Respiratory SyncytialVirus (RSV) nucleic acidsisolated and purified fromnasopharyngeal (NP)swab specimens obtainedfrom symptomaticpatients. This test isintended for use to aid inthe differential diagnosisof Influenza A, InfluenzaB and RSV2 viralinfections in humans andis not intended to detectInfluenza C.A negative test ispresumptive and it isrecommended theseresults be confirmed bycell culture. Negativeresults do not precludeinfluenza or RSV virusinfection and should notbe used as the sole basisfor treatment or othermanagement decisions.</td><td colspan="1" rowspan="1">swabs (NPS), nasal swabs(NS), and nasopharyngealaspirates (NPA) from humanpatients with signs andsymptoms of respiratoryinfection in conjunction withclinical and epidemiologicalrisk factors.Negative results do notpreclude influenza virusinfection and should not beused as the sole basis fortreatment or other patientmanagement decisions.Performance characteristicsfor influenza A wereestablished during the 2009-2010 influenza season when2009 H1N1 influenza was thepredominant influenza A virusin circulation. When otherInfluenza A viruses areemerging, performancecharacteristics may vary.If infection with a novelInfluenza A virus is suspectedbased on current clinical andepidemiological screeningcriteria recommended bypublic health authorities,specimens should be collectedwith appropriate infectioncontrol precautions for novelvirulent Influenza viruses andsent to state or local healthdepartment for testing. Viralculture should not beattempted in these cases unlessa BSL 3+ facility is availableto receive and culturespecimens</td></tr><tr><td colspan="1" rowspan="1">Indication for Use</td><td colspan="1" rowspan="1">Patients with signs andsymptoms of respiratoryinfection in conjunction withclinical and epidemiologicalrisk factors.</td><td colspan="1" rowspan="1">Symptomatic patients</td><td colspan="1" rowspan="1">Patients with signs andsymptoms of respiratoryinfection in conjunction withclinical and epidemiologicalrisk factors</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">Influenza A, Influenza B, andInfluenza A subtype 2009H1N1</td><td colspan="1" rowspan="1">Influenza A, influenza B,Respiratory SyncytialVirus Type A and Type B</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1 influenza</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasal aspirates/washes(NA/W) and Nasopharyngeal(NP) swabs</td><td colspan="1" rowspan="1">NP swab</td><td colspan="1" rowspan="1">NP swab, Nasal Swab (NS)and nasopharyngeal aspirates(NA/W)</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">RT/PCR</td><td colspan="1" rowspan="1">RT/PCR</td><td colspan="1" rowspan="1">RT/PCR</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidExtraction</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Extraction Methods</td><td colspan="1" rowspan="1">Sample preparation integratedin GeneXpert Cartridge andGeneXpert InstrumentationSystem</td><td colspan="1" rowspan="1">Roche MagNA Pure LCTotal NA Isolation Kit</td><td colspan="1" rowspan="1">Roche MagNA Pure LC TotalNA Isolation Kit, QIAGENQIAamp Viral RNA mini Kit</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Instrument System</td><td colspan="1" rowspan="1">Cepheid GeneXpert InstrumentSystems</td><td colspan="1" rowspan="1">Cepheid Smartcycler® II</td><td colspan="1" rowspan="1">3M Integrated cycler</td></tr><tr><td colspan="1" rowspan="1">Assay Controls</td><td colspan="1" rowspan="1">Encapsulated (armored) RNApseudovirus as a sampleprocessing control.Available but not provided areinactivated virus controls forFlu A/B and Flu A H1N1 asexternal positive controls andCoxsackie virus as an externalnegative control.</td><td colspan="1" rowspan="1">Inf A RNA Control, Inf BRNA Control, RSV ARNA Control, RSV BRNA Control and aninternal</td><td colspan="1" rowspan="1">Armored RNA InternalControl, No Template Control,and H1N1 Positive Controlprovided</td></tr><tr><td colspan="1" rowspan="1">Test results</td><td colspan="1" rowspan="1">Total 75 minutes for samplepreparation andrRT-PCR</td><td colspan="1" rowspan="1">Total 205 minutes(~45 minutes for samplepreparation~2.0 hours forrRT-PCR)</td><td colspan="1" rowspan="1">Total 115 minutes(~45 minutes for samplepreparation~70 minutes forrRT-PCR)</td></tr><tr><td colspan="1" rowspan="1">Laboratory Users</td><td colspan="1" rowspan="1">CLIA Moderate Complexity</td><td colspan="1" rowspan="1">CLIA High Complexity</td><td colspan="1" rowspan="1">CLIA High Complexity</td></tr><tr><td colspan="1" rowspan="1"></td><td></td><td></td><td></td></tr></table>

# K. Standard/Guidance Document referenced (if applicable):

Not Applicable

# L. Test Principle:

The primers and probes in the Xpert Flu Assay detect the presence of proprietary sequences for influenza A (Flu A), influenza B (Flu B) and influenza A sub-type 2009 H1N1 (2009 H1N1). Nasopharyngeal (NP) swabs and nasal aspirate/wash (NA/W) samples are collected following the user institution’s standard procedures and placed into Universal Transport Medium ( $3 ~ \mathrm { m L }$ UTM tubes) prior to being transported to the GeneXper $^ \mathrm { \textregistered }$ System area. For nasopharyngeal (NP) swab specimens, the sample is mixed briefly by inverting the UTM tube 5 times. Using a supplied $3 0 0 \mu \mathrm { L }$ transfer

pipet, $3 0 0 \mu \mathrm { L }$ of the mixed sample is transferred to the chamber with the large opening of the Xpert Flu Assay Cartridge. For nasal aspirate/wash (NA/W) samples, the sample is diluted by placing $6 0 0 ~ \mu \mathrm { L }$ of the NAW sample (using a $3 0 0 \mu \mathrm { L }$ transfer pipetter 2 times) into a $3 ~ \mathrm { m L }$ UTM tube. The contents of the tube are mixed by inverting the tube 5 times. Following the mixing step, $3 0 0 \mu \mathrm { L }$ of the diluted, mixed sample is transferred to the chamber with the large opening of the Xpert Flu Assay Cartridge.

After the sample is added, Reagent 1, a Binding Reagent, is transferred by squeezing the entire contents of the Reagent 1 tube into the chamber with the small opening of the Xpert Flu Assay Cartridge.

The user initiates a test from the system user interface, the instrument signals the user where to place the cartridge, and the cartridge is placed into the indicated module in the GeneXpert Dx System instrument, or onto a conveyor belt on the GeneXpert Infinity System, which then transports the cartridge to the appropriate GeneXpert module or to the holding area for later transport to the appropriate GeneXpert module. Fluidic movements under control of the instrument move the sample and reagents to and from different chambers within the Xpert Flu Assay cartridge. The GeneXpert $^ \mathrm { \textregistered }$ instrument systems perform sample preparation by first mixing the sample with the Sample Processing Control (encapsidated RNA) in the form of a lyophilized bead within the cartridge. Lysis of the viral particles with Lysis Reagent is followed by mixing with Binding Reagent which allows capture of the nucleic acids on the cellulose column. The column is then washed to remove contaminants and finally, the purified nucleic acids are eluted with an elution reagent. The nucleic acid solution is mixed with dry rRT-PCR reagents and transferred into the PCR tube for real-time RT-PCR and detection of Flu RNA. The Xpert Flu Assay completes sample preparation and real-time RT-PCR in 75 minutes. Internal controls in the Xpert Flu Assay check key automated steps.

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

a. Precision/Reproducibility:

A panel of 10 specimens with varying concentrations of influenza A, influenza B, and influenza A subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ were tested in duplicate on 10 different days at each of three sites (10 specimens $_ { \textrm { X 2 } }$ times/ day x 10 days x 3 sites). One lot of Xpert Flu Assay was used at each of the 3 testing sites. Xpert Flu Assays were performed according to the Xpert Flu Assay procedure. Sites 1 and 2 used the GeneXpert Dx system while Site 3 used the GeneXpert Infinity system for analysis.

<table><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">% Total Agreement</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td></tr><tr><td colspan="1" rowspan="1">Flu Amoderate positive</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(19/19)a</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(59/59)</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">low positive</td><td colspan="1" rowspan="1">(20/20)</td><td colspan="1" rowspan="1">(20/20)</td><td colspan="1" rowspan="1">(20/20)</td><td colspan="1" rowspan="1">(60/60)</td></tr><tr><td colspan="1" rowspan="1">Flu Ahigh negative</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">95.0%(19/20)</td><td colspan="1" rowspan="1">90.0%(18/20)</td><td colspan="1" rowspan="1">95.0%(57/60)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1moderate positive</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1low positive</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(19/19)b</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(59/59)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1high negative</td><td colspan="1" rowspan="1">94.7%(18/19)a</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">89.5%(17/19)a</td><td colspan="1" rowspan="1">94.8%(55/58)c</td></tr><tr><td colspan="1" rowspan="1">Flu Bmoderate positive</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td></tr><tr><td colspan="1" rowspan="1">Flu Blow positive</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(20/20)</td><td colspan="1" rowspan="1">100%(60/60)</td></tr><tr><td colspan="1" rowspan="1">Flu Bhigh negative</td><td colspan="1" rowspan="1">90.0%(18/20)</td><td colspan="1" rowspan="1">95.0%(19/20)</td><td colspan="1" rowspan="1">55.0%(11/20)</td><td colspan="1" rowspan="1">80.0%(48/60)</td></tr><tr><td colspan="1" rowspan="1">% TotalAgreement</td><td colspan="1" rowspan="1">98.5%(196/199)</td><td colspan="1" rowspan="1">99.0%(196/198)</td><td colspan="1" rowspan="1">93.5%(186/199)</td><td colspan="1" rowspan="1">97.0%(578/596)</td></tr></table>

$\mathrm { \Delta } ^ { \mathrm { a } } { \mathrm { n } } { = } 1 9$ because repeat yielded indeterminate result. $\mathfrak { b } _ { \mathrm { n } = 1 9 }$ because one sample was indeterminate and not retested. $^ \mathrm { c } 9 / 5 5$ samples negative for $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ resulted in a valid Flu A positive call, as the Flu A signal was detected. A valid $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ positive call requires detection of both the Flu A and 2009 H1N1 signals.

An in-house reproducibility study was conducted to compare the performance of the GeneXpert Dx and the Infinity instrument systems. A panel of 10 specimens with varying concentrations of influenza A, influenza B, and influenza A subtype 2009 H1N1 were tested in duplicate on 12 different days by two operators. Each operator conducted four runs of each panel specimen per day on each of the two instrument systems (10 specimens $_ { \textrm { X 2 } }$ times/ day $\textbf { x } 1 2$ days x 2 operators $_ { \textrm { X 2 } }$ instrument systems). One lot of Xpert Flu Assay was used for the study. Xpert Flu Assays were performed according to the Xpert Flu Assay procedure.

<table><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">GeneXpert Dx</td><td colspan="1" rowspan="1">Infinity</td><td colspan="1" rowspan="1">% Total Agreement</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">Flu Amoderate positive</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">Flu Alow positive</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">Flu Ahigh negative</td><td colspan="1" rowspan="1">89.6%(86/96)</td><td colspan="1" rowspan="1">86.5%(83/96)</td><td colspan="1" rowspan="1">88.0%(169/192)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1moderate positive</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1low positive</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1high negative</td><td colspan="1" rowspan="1">84.4%(81/96)</td><td colspan="1" rowspan="1">92.6%(88/95)a</td><td colspan="1" rowspan="1">88.5%(169/191)</td></tr><tr><td colspan="1" rowspan="1">Flu Bmoderate positive</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(96/96)</td><td colspan="1" rowspan="1">100.0%(192/192)</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">100.0%</td></tr><tr><td colspan="1" rowspan="1">low positive</td><td colspan="1" rowspan="1">(96/96)</td><td colspan="1" rowspan="1">(96/96)</td><td colspan="1" rowspan="1">(192/192)</td></tr><tr><td colspan="1" rowspan="1">Flu Bhigh negative</td><td colspan="1" rowspan="1">87.5%(84/96)</td><td colspan="1" rowspan="1">82.3%(79/96)</td><td colspan="1" rowspan="1">84.9%(163/192)</td></tr><tr><td colspan="1" rowspan="1">% Total Agreement</td><td colspan="1" rowspan="1">96.1% (923/960)</td><td colspan="1" rowspan="1">96.1% (922/959)</td><td colspan="1" rowspan="1">96.1% (1845/1919)</td></tr></table>

$\mathrm { \Delta } ^ { \mathrm { a } } { \mathrm { n } } { = } 9 5$ because repeat yielded indeterminate result.

# b. Linearity:

Linearity was evaluated using two seasonal influenza A H1N1 (Influenza A H1N1 (A/Taiwan/42/06), Influenza A H1N1 (A/Denver/1/57)), two seasonal influenza A H3N2 (Influenza A H3N2 (A/Brisbane/10/07), Influenza A H3N2 (A/Hong Kong/6/68)), two influenza A 2009 H1N1 (Influenza A 2009 H1N1 (A/SwineNY/03/2009), Influenza A 2009 H1N1 (A/SwineCanada/6294/2009)) and two influenza B strains (Influenza B (B/Panama/45/90) Influenza B (B/Florida/02/2006)) serially diluted over 4-5 logs and processed using the Xpert Flu Assay. The range of analyte concentrations was tested with four replicates at each concentration. Three external controls (two positive and 1 negative) was run with this study. Under the conditions of the study, the Xpert Flu Assay responds linearly over 5 logs for seasonal influenza A H1N1, seasonal influenza A H3N2 and influenza A 2009 H1N1; and over 4 logs for influenza B strains tested. Of the 235 runs, 5 provided uninformative GeneXpert results (2 “ERROR” and 3 “INVALID”). The 2 “ERROR” tests were not repeated. The 3 “INVALID” tests were obtained with samples spiked with very low concentrations of 2009 H1N1 virus. These samples were reported “INVALID” because the test results were negative for influenza A and positive for 2009 H1N1. All three external controls were correctly identified by the Xpert Flu Assay.

Linearity Data for Seasonal Influenza A H1N1 (A/Taiwan/42/06)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu A Ct</td></tr><tr><td rowspan=1 colspan=1>6.31x106</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>17.2</td></tr><tr><td rowspan=1 colspan=1>6.31x105</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>21.2</td></tr><tr><td rowspan=1 colspan=1>6.31x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>24.7</td></tr><tr><td rowspan=1 colspan=1>6.31x103</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>27.7</td></tr><tr><td rowspan=1 colspan=1>6.31x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>30.5</td></tr><tr><td rowspan=1 colspan=1>6.31x101</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>34.0</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.292</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>100.1</td></tr></table>

$^ { * } 1 / 4$ replicates resulted in an ”ERROR”

Linearity Data for Seasonal Influenza A H1N1 (A/Denver/1/57)   

<table><tr><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">n</td><td colspan="1" rowspan="1">Log Concentration</td><td colspan="1" rowspan="1">Flu A Ct</td></tr><tr><td colspan="1" rowspan="1">1.12x105</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">5.05</td><td colspan="1" rowspan="1">18.5</td></tr><tr><td colspan="1" rowspan="1">1.12x104</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4.05</td><td colspan="1" rowspan="1">22.3</td></tr><tr><td colspan="1" rowspan="1">1.12x103</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">3.05</td><td colspan="1" rowspan="1">25.9</td></tr><tr><td colspan="1" rowspan="1">1.12x102</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">2.05</td><td colspan="1" rowspan="1">28.7</td></tr><tr><td colspan="1" rowspan="1">1.12x101</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.05</td><td colspan="1" rowspan="1">31.9</td></tr><tr><td colspan="1" rowspan="1">1.12</td><td colspan="1" rowspan="1">3*</td><td colspan="1" rowspan="1">0.05</td><td colspan="1" rowspan="1">36.4</td></tr><tr><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="1">Slope</td><td colspan="1" rowspan="1">-3.430</td></tr><tr><td colspan="1" rowspan="1">% PCR Efficiency</td><td colspan="1" rowspan="1">95.7</td></tr></table>

$^ { * } 1 / 4$ replicates resulted in a Negative test result

Linearity Data for Seasonal Influenza A H3N2 (A/Brisbane/10/07)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu A Ct</td></tr><tr><td rowspan=1 colspan=1>1.12x105</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.05</td><td rowspan=1 colspan=1>18.1</td></tr><tr><td rowspan=1 colspan=1>1.12x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.05</td><td rowspan=1 colspan=1>22.2</td></tr><tr><td rowspan=1 colspan=1>1.12x103</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>25.3</td></tr><tr><td rowspan=1 colspan=1>1.12x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>1.12x10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>31.4</td></tr><tr><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>35.2</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.311</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>100.4</td></tr></table>

$^ { * } 1 / 4$ replicates resulted in a Negative test result

# Linearity Data for Seasonal Influenza A H3N2 (A/Hong Kong/6/68)

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu A Ct</td></tr><tr><td rowspan=1 colspan=1>3.55x10</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.55</td><td rowspan=1 colspan=1>17.9</td></tr><tr><td rowspan=1 colspan=1>3.55x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.55</td><td rowspan=1 colspan=1>21.9</td></tr><tr><td rowspan=1 colspan=1>3.55x103</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.55</td><td rowspan=1 colspan=1>25.1</td></tr><tr><td rowspan=1 colspan=1>3.55x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.55</td><td rowspan=1 colspan=1>28.2</td></tr><tr><td rowspan=1 colspan=1>3.5510</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>31.0</td></tr><tr><td rowspan=1 colspan=1>3.55</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>34.7</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.255</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>102.9</td></tr></table>

Linearity Data for Influenza A 2009 H1N1 (A/SwineNY/03/2009)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>2009H1N1 Ct</td></tr><tr><td rowspan=1 colspan=1>1.26x105</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.1</td><td rowspan=1 colspan=1>15.0</td><td rowspan=1 colspan=1>16.5</td></tr><tr><td rowspan=1 colspan=1>1.26x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.1</td><td rowspan=1 colspan=1>19.4</td><td rowspan=1 colspan=1>20.5</td></tr><tr><td rowspan=1 colspan=1>1.26x103</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>23.9</td></tr><tr><td rowspan=1 colspan=1>1.26x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>25.9</td><td rowspan=1 colspan=1>27.0</td></tr><tr><td rowspan=1 colspan=1>1.26x101</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>30.3</td></tr><tr><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>33.9</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.545</td><td rowspan=1 colspan=1>-3.415</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>91.5</td><td rowspan=1 colspan=1>96.3</td></tr></table>

Linearity Data for Influenza A 2009 H1N1 (A/SwineCanada/6294/2009)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Logg Concentration</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>2009H1N1 Ct</td></tr><tr><td rowspan=1 colspan=1>6.31e6</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.8</td><td rowspan=1 colspan=1>16.1</td><td rowspan=1 colspan=1>16.9</td></tr><tr><td rowspan=1 colspan=1>6.31e5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.8</td><td rowspan=1 colspan=1>20.3</td><td rowspan=1 colspan=1>20.9</td></tr><tr><td rowspan=1 colspan=1>6.31e4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.8</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>24.3</td></tr><tr><td rowspan=1 colspan=1>6.31e3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.8</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>27.6</td></tr><tr><td rowspan=1 colspan=1>6.31e2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>30.2</td></tr><tr><td rowspan=1 colspan=1>6.31e1</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>33.5</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.4925</td><td rowspan=1 colspan=1>-3.262</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>93.3</td><td rowspan=1 colspan=1>102.6</td></tr></table>

$^ { * } 1 / 4$ replicates resulted in an “INVALID” test result

Linearity Data for Influenza B (B/Panama/45/90)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu B Ct</td></tr><tr><td rowspan=1 colspan=1>1.12x106</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6.05</td><td rowspan=1 colspan=1>19.2</td></tr><tr><td rowspan=1 colspan=1>1.12x105</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5.05</td><td rowspan=1 colspan=1>22.9</td></tr><tr><td rowspan=1 colspan=1>1.12x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.05</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>1.12x103</td><td rowspan=1 colspan=1>3*</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>29.1</td></tr><tr><td rowspan=1 colspan=1>1.12x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>32.6</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.2953</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>100.1</td></tr></table>

$^ { * } 1 / 4$ replicates resulted in an “ERROR”

Linearity Data for Influenza B (B/Florida/02/2006)   

<table><tr><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>Log Concentration</td><td rowspan=1 colspan=1>Flu B Ct</td></tr><tr><td rowspan=1 colspan=1>1.12x104</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.05</td><td rowspan=1 colspan=1>19.7</td></tr><tr><td rowspan=1 colspan=1>1.12x103</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>23.6</td></tr><tr><td rowspan=1 colspan=1>1.12x102</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.05</td><td rowspan=1 colspan=1>26.9</td></tr><tr><td rowspan=1 colspan=1>1.12x101</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>30.1</td></tr><tr><td rowspan=1 colspan=1>1.12</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>34.3</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=1>Slope</td><td rowspan=1 colspan=1>-3.5775</td></tr><tr><td rowspan=1 colspan=1>% PCR Efficiency</td><td rowspan=1 colspan=1>90.3</td></tr></table>

c. Traceability, Stability, Expected values (controls, calibrators, or methods): Because each Xpert Flu cartridge is a self-contained unit test device, the use of external controls has limited value in assuring proper device performance for any given assay. Therefore, it was necessary to design the device to include internal controls that would enable the system to detect specific failure modes that could potentially result in an erroneous, but believable result. A Failure Mode Effects and Criticality Analysis was performed and periodically updated during the design and development phase, and key failure modes were identified that might cause a false result. Control elements (probe check control and sample processing control) were created to address major key failure modes that could result in an unresolved or false negative determination, and for which the probability of occurrence was remote or higher.

As a result, the Xpert Flu Assay includes a system control, referred to as the Probe Check Control (PC), and an internal control, referred to as the Sample Processing Control (SPC). The SPC is pre-loaded into the GeneXpert cartridge provided with the assay. An additional control is the System Control Check for temperature. This check ensures that the GeneXpert Dx Instrument is operating within validated heating and cooling specifications.

The SPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the PCR reaction to reduce the possibility of false negative results. The SPC also ensures the PCR reaction conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. The Probe Check Control verifies bead reconstitution, PCR tube filling in the cartridge, probe integrity and dye stability.

External Controls may be used in accordance with local, state, or federal accrediting organizations, as applicable. External Controls are not provided in the test kit; however they are available for purchase from an outside source (ZeptoMetrix). The outside source and the catalog numbers are provided to the customer in the ‘Materials Available but Not Provided’ section of the Xpert Flu Assay Package Insert.

# Probe Check Control (PC)

After sample preparation, bead reconstitution, and reaction tube filling, but prior to thermal cycling, the GeneXpert Dx System is programmed to perform a probe check on the amplification mixture. Fluorescence readings are obtained at two temperatures. These readings are then compared to default settings established by Cepheid. The PC controls for:

Missing Enzyme Reagent (EZR) bead (contains polymerase, dNTPs, and 1 probe),   
• Missing Target Specific Reagent (TSR) bead (contain all primers and 3 probes),   
• Incomplete bead rehydration, Incomplete PCR reaction tube fill, and Probe degradation.

The PC is considered to PASS if the fluorescence generated meets the validated acceptance criteria. If the PCC fails for any Influenza A, Influenza B or Influenza A 2009 H1N1 target or SPC, a probe check error is reported and the test will not continue. For these conditions, the test will need to be repeated using a new specimen, a new cartridge and new reagents.

# Sample Processing Control (SPC)

The SPC is an encapsidated RNA pseudovirus that is provided in the form of a dry bead and is included in each cartridge. The encapsidated RNA consists of a recombinant RNA surrounded by the protein coat from MS2 virus and is designed to mimic the influenza virus. The SPC is mixed with the sample to control for adequate sample processing and to monitor the integrity of the RT-PCR assay. RT-PCR is performed to detect the encapsidated RNA sequence. SPC is considered to “pass” if it meets the validated acceptance criteria.

The SPC verifies (1) the effectiveness of each sample preparation step, (2) reaction tube filling, (3) that reaction components are present and functioning, and monitors the presence of potential inhibitor(s) in the PCR assay. This element controls for:

Adequate specimen processing – sample lysis, capture and elution from the cellulose column, Missing Target Specific Reagent (TSR) and/or Enzyme Specific Reagent (EZR) beads, • Incomplete bead rehydration, Incomplete PCR reaction tube fill, Appropriate PCR reaction conditions (time and temperature), Enzyme degradation, and Specimen inhibition of the PCR.

The controls in the Xpert Flu Assay both complement, but also provide a level of redundancy which is important for a self-validating system. For example, by definition the SPC will react to degradation of enzyme because SPC detection requires enzyme activity. The SPC also reacts to sample effects. Both SPC and probe check are able to respond to probe degradation.

# d. Detection limit:

The LoD study was conducted to determine the lowest concentration of eleven (11) influenza strains detected by the device. Each strain was diluted into a simulated background matrix that consisted of $2 . 5 \%$ (w/v) porcine mucin, $1 \% ( \mathrm { v / v } )$ human whole blood in $0 . 8 5 \%$ sodium chloride (NaCl) formulated in 1X PBS solution with $1 5 \%$ glycerol diluted with UTM at a 1:6 ratio. Negative specimens consisted of simulated background matrix only. Dilutions were prepared daily and kept on ice prior to testing. Twenty (20) replicates for each dilution level were analyzed and the LoD demonstrated to be the lowest concentration to yield 19/20 positive replicates.

# Seasonal Influenza A subtype H1N1 Mean Ct Values in Valid Replicates

<table><tr><td rowspan=1 colspan=1>Seasonal Influenza Asubtype H1N1 Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positives/ 20Replicates</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>SPC Ct</td></tr><tr><td rowspan=4 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=4 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>39.4</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr></table>

Seasonal Influenza A subtype H3N2 Mean Ct Values in Valid Replicates   

<table><tr><td rowspan=1 colspan=1>Seasonal Influenza Asubtype H3N2 Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positives/ 20Replicates</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>SPC Ct</td></tr><tr><td rowspan=5 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=4 colspan=1>A/Wisconsin/67/05</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr></table>

Seasonal Influenza A subtype 2009 H1N1 Mean Ct Values in Valid Replicates   

<table><tr><td rowspan=1 colspan=3>Influenza A subtype 2009H1N1 Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>2009 H1N1 Ct</td><td rowspan=1 colspan=1>SPC Ct</td></tr><tr><td rowspan=4 colspan=3>A/SwineNY/01/2009</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=5 colspan=3>A/SwineNY/02/2009</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>26.8</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>29.3</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>6/19</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>29.1</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=6 colspan=3>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>29.1</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>29.1</td></tr><tr><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10/19</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>29.2</td></tr><tr><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>3/19</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>38.4</td><td rowspan=1 colspan=1>29.1</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=3 colspan=3>A/SwineCanada/6294/2009</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>33.3</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>28.8</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Influenza A subtype 2009H1N1 Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=1>Flu A Ct</td><td rowspan=1 colspan=1>2009 H1N1 Ct</td><td rowspan=1 colspan=1>SPC Ct</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=6 colspan=1>A/WI/629-S1/2009</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>34.3</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>39.7</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr></table>

Influenza B Mean Ct Values in Valid Replicates   

<table><tr><td rowspan=1 colspan=1>Influenza B Strain</td><td rowspan=1 colspan=1>Concentration(TCID50/mL)</td><td rowspan=1 colspan=1>Positives/20Replicates</td><td rowspan=1 colspan=1>Flu B Ct</td><td rowspan=1 colspan=1>SPC Ct</td></tr><tr><td rowspan=4 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>28.6</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=5 colspan=1>B/Florida/02/06</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>28.7</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>28.9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>15/19</td><td rowspan=1 colspan=1>38.3</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>28.8</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28.7</td></tr></table>

# e. Analytical specificity: Inclusivity

This study was performed to evaluate a temporally and geographically diverse number of clinically relevant virus strains for each claimed analyte at viral levels at or near the LoD. All virus identities and titers were confirmed. Selections were made to represent the analytes detected by the Xpert Flu Assay and reflect the strains recommended in the most recent FDA guidance documents.

The analytical reactivity of the Xpert Flu Assay was evaluated against multiple strains of influenza A (H1N1, H3N2, H5N2, H5N1 and H7N3 subtypes), influenza A 2009 H1N1 and influenza B. Thirty-nine (39) influenza strains were tested using the Xpert Flu Assay (see Table 18.2). Of these, influenza A subtype H1N1 (12), influenza A subtype H3N2 (7), influenza A subtype 2009 H1N1 (6), influenza A subtype H5N1 (1), influenza A subtype H5N2 (1), influenza A subtype H7N3 (1) and influenza B (11) were included. Virus strains were obtained from ATCC (Manassas, VA) and ZeptoMetrix Corporation (Buffalo, NY) and sent to Medical College of Wisconsin, Children’s Corporate Center (Milwaukee, WI) for virus propagation and quantification.

All strains were tested in triplicate using cell stocks at 5 -500 TCID50/mL or $\mathrm { P F U / m L }$ . Due to shipping and handling restrictions on viable viral particles, purified nucleic acids were tested for three strains (influenza A subtypes H5N1, H5N2 and H7N3). Levels tested represent concentrations at or near the Limit of Detection of the assay. For positive samples the influenza virus strains were diluted into a simulated background matrix. Negative specimens consisted of simulated background matrix only. Dilutions were prepared daily and kept on ice prior to testing.

The results of the analytical inclusivity study demonstrated that the Xpert Flu Assay appropriately identifies all the influenza strains selected to represent the range of respiratory viral strains that the Xpert Flu Assay has been designed to detect. The Xpert Flu Assay correctly identified all 39 influenza strains; 22 as influenza A, 11 as influenza B, and 6 as influenza A subtype 2009 H1N1. Of the 128 runs, 4 provided uninformative GeneXpert results (“ERRORS”). All “ERRORs” were repeated for a total of 3 replicates tested per strain. All three external controls were correctly identified by the Xpert Flu Assay.

Analytical Reactivity Results of the Xpert Flu Assay on 39 Influenza Specimens   

<table><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/mL)</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Influenza A2009 H1N1</td><td colspan="1" rowspan="1">Influenza B</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Denver/1/57(H1N1)^</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/NewYork/55/2004(H1N1)^</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mal/302/54(H1N1)^</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/New Jersey/8/76(H1N1)^</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/NWS/33 (H1N1)^</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/PR/8/34 (H1N1)^</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Taiwan/42/06(H1N1)^</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/WS/33 (H1N1)^</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Swine/1976/31(Swine H1N1)^</td><td colspan="1" rowspan="1">500*</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Swine/Iowa/15/30(Swine H1N1)^</td><td colspan="1" rowspan="1">500*</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/07(H1N1)^</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/NewCalendonia/20/1999(H1N1)^</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/3/75(H3N2)</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Aichi2/68 (H3N2)</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/8/68(H3N2)</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hawaii/15/2001(H3N2)</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers/1/73(H3N2)</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/10/07(H3N2)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Wisconsin/67/05(H3N2)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/SwineNY/01/2009(09 H1N1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/SwineNY/02/2009(09 H1N1)</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/SwineNY/03/2009(09 H1N1)</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/California/4/2009(09 H1N1)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Canada/6294(09 H1N1)</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/WI/929-S1(09 H1N1)</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mallard/WI/34/75(H5N2)</td><td colspan="1" rowspan="1">3 pg/μL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Anhui/02/2005/PR8-IBCDC-RG5 (H5N1)</td><td colspan="1" rowspan="1">0. 122 pg/μL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/chicken/NJ/15086-3/94 (H7N3)</td><td colspan="1" rowspan="1">5 pg/μL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza B/Allen/45</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1">.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/06</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/04/06</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/07/04</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/GL/1739/54</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hong Kong/5/72</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Lee/40</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Malaysia/2506/04</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Maryland/1/59</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Panama/45/90</td><td colspan="1" rowspan="1">500</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Taiwan/2/62</td><td colspan="1" rowspan="1">50#</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr></table>

\*Concentration expressed as $\mathrm { P F U / m L }$ ^Seasonal influeneza A H1N1 (not 2009 H1N1) # Concentration expressed as $\mathrm { C E I D _ { 5 0 } / m L }$ ^^ Concentration expressed in $\mathfrak { p g / \mu L }$

# Exclusivity

The purpose of this study was to determine the analytical specificity (exclusivity) of the Xpert Flu Assay. Forty (40) viral, bacterial or yeast strains phylogenetically related to influenza virus or potentially present in nasopharyngeal flora were tested using the Xpert Flu Assay following the recommendations in the recent FDA guidance documents. The strains were diluted into a simulated background matrix. Negative specimens consisted of simulated background matrix only. Dilutions were prepared daily and kept on ice prior to testing.

Of the 40 strains tested, 18 were viruses, 21 were bacterial strains and 1 was yeast. Twenty (20) bacterial strains and one (1) yeast strain were obtained from the American Type Culture Collection (ATCC) and cultured at Cepheid. One (1) bacterial strain was obtained from Dr. David Alland’s laboratory, University of Medicine and Dentistry of New Jersey. Virus strains were obtained from ATCC (Manassas, VA) and ZeptoMetrix Corporation (Buffalo, NY) and sent to Medical College of Wisconsin, Children’s Corporate Center (Milwaukee, WI) for virus propagation and quantification.

Bacterial stock cultures were prepared by suspending the bacterial growth from an agar plate in PBS buffer containing $1 5 \%$ glycerol. Viral stock cultures were propagated using recommended tissue culture cell lines and hemagglutinin assays were used to establish titers. All bacterial strains were tested in triplicate at concentrations $\geq 1 0 ^ { 6 } \mathrm { C F U / m L }$ . All viral strains were tested in triplicate at concentrations ${ \geq } 1 0 ^ { 4 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ . Purified nucleic acids were tested at a concentration of $1 \mathrm { x } 1 0 ^ { 6 }$ genomic copies/mL for two bacterial strains (Bordetella pertussis and Haemophilus influenzae) and one virus strain (Cytomegalovirus).

Under the conditions of this study, the analytical specificity of the Xpert Flu Assay is $100 \%$ . None of the non-influenza isolates tested was detected by the Xpert Flu Assay as expected. All isolates were reported as “Flu A NEGATIVE; 2009 H1N1 NOT DETECTED; Flu B NEGATIVE”. These results demonstrate that a sample containing non-influenza isolates $\left( > \mathrm { { l x } 1 0 ^ { 6 } \ C F U / m L } \right)$ or ${ \geq } 1 0 ^ { 4 }$ $\Gamma \mathrm { C I D } 5 0 / \mathrm { m L }$ or $1 0 ^ { 6 }$ genomic copies $/ \mathrm { m L }$ ) will not falsely trigger a positive influenza test result using the Xpert Flu Assay. Of the 120 runs, 1 provided an uninformative GeneXpert result (“ERROR”). This sample was repeated for a total of 3 replicates tested per strain. All three external controls were correctly identified by the Xpert Flu Assay.

# Analytical Specificity Determination for Xpert Flu Assay

<table><tr><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(per Cartridge)</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">Influenza A 2009H1N1</td><td colspan="1" rowspan="1">Influenza B</td></tr><tr><td colspan="1" rowspan="1">Positive Control 1 - InfluenzaA/Influenza B</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Positive Control 2  Influenza A2009 H1N1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Negative Control</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7A</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">1x104 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus*</td><td colspan="1" rowspan="1">1x105Copies /mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterovirus Type 71</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus Type 1</td><td colspan="1" rowspan="1">1x105 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus Type 2</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus Type 3</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">1x106TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mumps Virus</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus A</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus B</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human HSV Type 1</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus Type 44</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1x106 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis*</td><td colspan="1" rowspan="1">1x106 Copies/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae*</td><td colspan="1" rowspan="1">1x106 Copies/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis(BCG strain)</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neisseria cinerea</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1x106CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr></table>

# $f .$ Assay cut-off:

Lot specific assay settings are generated for every lot manufactured to account for variations in reagent production. The lot specific assay settings (LSP file) are incorporated into the barcode on each cartridge label and are transferred to the GeneXpert instrument systems via a barcode scanner prior to initiating the Xpert Flu Assay. General Assay Settings, shown below, are parameters that are used for all Xpert Flu Assay reagent lots. These settings are fixed and not part of the LSP process.

General Assay Settings   

<table><tr><td rowspan=1 colspan=1>Attribute</td><td rowspan=1 colspan=1>Setting</td></tr><tr><td rowspan=1 colspan=1>Background Subtraction</td><td rowspan=1 colspan=1>Always ON</td></tr><tr><td rowspan=1 colspan=1>Background Minimum Cycle</td><td rowspan=1 colspan=1>Default setting = 5</td></tr><tr><td rowspan=1 colspan=1>Background Maximum Cycle</td><td rowspan=1 colspan=1>Default setting = 30</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (all targets and SPC)</td><td rowspan=1 colspan=1>Flu A = 202009 H1N1 = 20Flu B = 80SPC = 20</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis</td><td rowspan=1 colspan=1>Primary</td></tr><tr><td rowspan=1 colspan=1>Boxcar Average Cycles</td><td rowspan=1 colspan=1>Zero (OFF)</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (SPC)</td><td rowspan=1 colspan=1>Manual setting = 26</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (Flu A, 2009 H1N1 and Flu B) Manual setting = 12</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (SPC)</td><td rowspan=1 colspan=1>Manual setting = 40</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (Flu A, 2009 H1N1 and Flu B) Manual setting = 40</td><td rowspan=1 colspan=1></td></tr></table>

The valid minimum cycle threshold setting for influenza A, 2009 H1N1 and influenza B is 12 because in practice, a Ct cannot be calculated before the end of the minimum background subtraction range, which is 10 cycles. The earliest Flu A Ct, 2009 H1N1 and Flu B reported in true influenza A positive, 2009 H1N1 positive and influenza B positive results during pre-clinical testing was greater than 12. The valid maximum cycle threshold setting for influenza A, 2009 H1N1, and influenza B is 40.0.

To obtain a valid influenza A Positive test result, the Flu A Ct must be reported within the valid cycle range. To obtain a valid 2009 H1N1 Positive test result, both Flu A and 2009 H1N1 Cts must be reported within the valid cycle range and the fluorescent signal must cross the manual threshold fluorescence units. To obtain a valid influenza B Positive test result, the Flu B Ct must be reported within the valid cycle range and the Flu B fluorescent signal must cross the manual threshold fluorescence units.

To obtain a valid influenza A, influenza A 2009 H1N1 or influenza B Negative test result, the Flu A, 2009 H1N1, and Flu B Cts must not be reported within the valid cycle range and the SPC Ct must be reported within its valid cycle range. If the SPC falls outside the valid cycle range, the test result is “INVALID” and must be repeated. The valid maximum cycle threshold setting for the SPC in Xpert Flu Assay was set at 40 based on the pre-clinical data. Subsequently, true negative samples from interfering substances study $( \mathrm { n } { = } 1 2 1$ ) and LSP testing $\mathrm { ( n = 2 1 8 }$ ) were analyzed to support the valid maximum cycle threshold setting of 40.

# g. Potentially Interfering Substances

Potentially Interfering Substances Tested   

<table><tr><td rowspan=1 colspan=1>Substance</td><td rowspan=1 colspan=1>Supplier</td><td rowspan=1 colspan=1>Active Ingredient</td></tr><tr><td rowspan=1 colspan=1>Universal Transport Medium(UTM)</td><td rowspan=1 colspan=1>Copan</td><td rowspan=1 colspan=1>Control</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>Stanford</td><td rowspan=1 colspan=1>Blood (2% v/v)</td></tr><tr><td rowspan=1 colspan=1>Mucous (Mucin)</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>Porcine mucin representing denselyglycosylated proteins (mucous)</td></tr><tr><td rowspan=1 colspan=1>Neo-Synephrine (NasalDecongestant)</td><td rowspan=1 colspan=1>Bayer</td><td rowspan=1 colspan=1>0.5% Phenylephrine Hydrochloride</td></tr><tr><td rowspan=1 colspan=1>Anefrin Spray (Decongestant)</td><td rowspan=1 colspan=1>Walgreens</td><td rowspan=1 colspan=1>0.05% Oxymetazoline Hydrochloride</td></tr><tr><td rowspan=1 colspan=1>Zicam Nasal Gel (UpperRespiratory Allergy SymptomRelief)</td><td rowspan=1 colspan=1>Zicam/Matrixx Initiatives,Inc</td><td rowspan=1 colspan=1>Luffa opperculata, Galphimia glauca,Histaminum hydrochloricum Sulfur</td></tr><tr><td rowspan=1 colspan=1>Saline Nasal Moisturizing Spray</td><td rowspan=1 colspan=1>Walgreens</td><td rowspan=1 colspan=1>0.65% Sodium Chloride</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment</td><td rowspan=1 colspan=1>COSTCO Pharmacy</td><td rowspan=1 colspan=1>Mupirocin (10 mg/mL)</td></tr><tr><td rowspan=1 colspan=1>Antiviral</td><td rowspan=1 colspan=1>Roche</td><td rowspan=1 colspan=1>Oseltamivir (TamiFlu)  7.5 mg/mL</td></tr><tr><td rowspan=1 colspan=1>Antibacterial, systemic</td><td rowspan=1 colspan=1>Sigma</td><td rowspan=1 colspan=1>Tobramycin (4 µg/mL)</td></tr><tr><td rowspan=1 colspan=1>Lozenges</td><td rowspan=1 colspan=1>Walgreens</td><td rowspan=1 colspan=1>Menthol, 1.7mg/ml menthol (w/v)</td></tr></table>

Potentially interfering substances in nasal specimens were identified and testing was conducted in the presence of 10 substances that include human blood, nasal secretions or mucus (porcine mucin), nasal and throat medications used to relieve congestion, nasal dryness, irritation, or asthma and allergy symptoms, antivirals and antibiotics. Substances were tested at final concentrations ranging from $100 \%$ to $0 . 5 \%$ depending on the substance. Influenza-negative samples $( \mathtt { n } = 8 )$ were tested for each substance to

determine the effect on the performance of the sample processing control (SPC). Influenza-positive samples $( \mathrm { n } { = } 8 )$ were tested for each substance spiked with two seasonal influenza A H1N1 strains, two seasonal influenza A H3N2 strains, two influenza A 2009 H1N1 strains and two influenza B strains spiked near the analytical LoD determined for each isolate. Statistical significance for both positive and negative samples was determined by comparing cycle threshold (Ct) values between samples run in the presence of the interfering substance and controls run in UTM.

Highly viscous samples, resulting from the addition of $1 . 5 \%$ (w/v) and $2 . 5 \%$ (w/v) porcine mucin yielded false-negative test results from the Xpert Flu Assay. Inhibition of the Xpert Flu Assay was also observed from the addition of $1 \%$ (w/v) porcine mucin, resulting in delayed detection of influenza A, influenza A subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ , and influenza B. The effects of mucin are addressed in the Limitations Section of the Package Insert.

h. Fresh vs. Frozen Equivalency Study

This study was performed with contrived positive and negative specimens. Contrived positive specimens consisted of cultured viral targets diluted into a simulated background matrix. Contrived negative specimens consisted of simulated background matrix only. The positive specimens in the study consisted of one cultured viral strain for each of the three targets in the assay (seasonal influenza A strain, influenza A 2009 H1N1 strain and influenza B strain). Each viral target was tested at three different concentrations representing a low positive (2X LoD), a medium positive (5X LoD) and a high positive (10X LoD) specimen. Replicates of 20 were tested per target per concentration. All replicate contrived specimens were tested fresh, after 1 freeze-thaw cycle and after 2 freeze-thaw cycles. Replicates of 20 negative samples were also tested per condition (fresh, one (1) freeze-thaw and two (2) freeze-thaws).

All positive replicates were identified correctly with a mean Ct of 5 or more standard deviations earlier than the assay cut-off of 40. All negative replicates were identified correctly with a mean SPC Ct of 5 or more standard deviations earlier than the assay cutoff of 40. There was no statistically significant difference between fresh specimens and specimens tested after two sequential freeze thaw cycles for influenza $\mathrm { \Delta } \mathrm { A } 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ and influenza B. Although there were statistically significant differences for the seasonal influenza A strain, these differences were small (less than $0 . 5 \mathrm { C t }$ ) and were not of practical significance since the virus was accurately detected in all 20 replicates at the lowest concentration (2X LOD). Influenza B strain demonstrated a statistically significant difference with the first freeze-thaw cycle. The increase in Flu B Ct was small (less than $0 . 2 \mathrm { C t }$ ) and not of practical significance.

Of the 692 runs, 19 samples provided uninformative GeneXpert result (“ERROR”). All “ERROR” tests were repeated such that there were a total of 20 replicates per condition. All three external controls were correctly identified by the Xpert Flu Assay.

Influenza A H1 – Average Ct and Mean Ct Difference Relative to the Fresh Control   

<table><tr><td rowspan=2 colspan=1>Virus LevelX LOD</td><td rowspan=1 colspan=3>Average Ct (n=20 replicates)</td><td rowspan=1 colspan=2>Mean Ct Difference from FreshControl</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td></tr><tr><td rowspan=1 colspan=1>10 X</td><td rowspan=1 colspan=1>27.5</td><td rowspan=1 colspan=1>27.8</td><td rowspan=1 colspan=1>27.6</td><td rowspan=1 colspan=1>0.3p=0.003</td><td rowspan=1 colspan=1>0.1p=0.304</td></tr><tr><td rowspan=1 colspan=1>5 X</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>0.1p=0.463</td><td rowspan=1 colspan=1>0.2p=0.010</td></tr><tr><td rowspan=1 colspan=1>2 X</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>29.8</td><td rowspan=1 colspan=1>0.3p=0.055</td><td rowspan=1 colspan=1>0.5p=0.000</td></tr></table>

Influenza A 2009H1N1 - Average Ct and Mean Ct Difference Relative to the Fresh Control for Flu A Target   

<table><tr><td rowspan=2 colspan=1>Virus LevelX LOD</td><td rowspan=1 colspan=3>Average Ct (n=20 replicates)</td><td rowspan=1 colspan=2>Mean Ct Difference from FreshControl</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td></tr><tr><td rowspan=1 colspan=1>10 X</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>24.89</td><td rowspan=1 colspan=1>24.95</td><td rowspan=1 colspan=1>-0.01p=0.996</td><td rowspan=1 colspan=1>0.05p=0.900</td></tr><tr><td rowspan=1 colspan=1>5 X</td><td rowspan=1 colspan=1>25.88</td><td rowspan=1 colspan=1>25.72</td><td rowspan=1 colspan=1>25.92</td><td rowspan=1 colspan=1>-0.17p=0.205</td><td rowspan=1 colspan=1>0.04p=0.107</td></tr><tr><td rowspan=1 colspan=1>2 X</td><td rowspan=1 colspan=1>26.98</td><td rowspan=1 colspan=1>27.12</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>0.14p=0.256</td><td rowspan=1 colspan=1>0.10p=0.490</td></tr></table>

Influenza A 2009H1N1 - Average Ct and Mean Ct Difference Relative to the Fresh Control for 2009 H1N1 Target   

<table><tr><td rowspan=2 colspan=1>VirusLevelX LOD</td><td rowspan=1 colspan=3>Average Ct (n=20 replicates)</td><td rowspan=1 colspan=2>Mean Ct Difference from FreshControl</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td></tr><tr><td rowspan=1 colspan=1>10 X</td><td rowspan=1 colspan=1>25.83</td><td rowspan=1 colspan=1>25.7</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>-0.08p=0.714</td><td rowspan=1 colspan=1>-0.04p=0.914</td></tr><tr><td rowspan=1 colspan=1>5 X</td><td rowspan=1 colspan=1>26.78</td><td rowspan=1 colspan=1>26.48</td><td rowspan=1 colspan=1>26.69</td><td rowspan=1 colspan=1>-0.29p=0.010</td><td rowspan=1 colspan=1>-0.008p=0.651</td></tr><tr><td rowspan=1 colspan=1>2 X</td><td rowspan=1 colspan=1>27.89</td><td rowspan=1 colspan=1>27.88</td><td rowspan=1 colspan=1>27.74</td><td rowspan=1 colspan=1>0.01p=0.998</td><td rowspan=1 colspan=1>-0.13p=0.237</td></tr></table>

Influenza B - Average Ct and Mean Ct Difference Relative to the Fresh Control   

<table><tr><td rowspan=2 colspan=1>VirusLevelX LOD</td><td rowspan=1 colspan=3>Average Ct (n=20 replicates)</td><td rowspan=1 colspan=2>Mean Ct Difference from FreshControl</td></tr><tr><td rowspan=1 colspan=1>Fresh</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td><td rowspan=1 colspan=1>1 Freeze-Thaw</td><td rowspan=1 colspan=1>2 Freeze-Thaws</td></tr><tr><td rowspan=1 colspan=1>10 X</td><td rowspan=1 colspan=1>26.415</td><td rowspan=1 colspan=1>26.516</td><td rowspan=1 colspan=1>26.49</td><td rowspan=1 colspan=1>0.10p=0.278</td><td rowspan=1 colspan=1>0.08p=0.467</td></tr><tr><td rowspan=1 colspan=1>5 X</td><td rowspan=1 colspan=1>27.37</td><td rowspan=1 colspan=1>27.41</td><td rowspan=1 colspan=1>27.427</td><td rowspan=1 colspan=1>0.04p=0.908</td><td rowspan=1 colspan=1>0.06p=0.787</td></tr><tr><td rowspan=1 colspan=1>2 X</td><td rowspan=1 colspan=1>28.61</td><td rowspan=1 colspan=1>28.84</td><td rowspan=1 colspan=1>28.665</td><td rowspan=1 colspan=1>0.23p=0.006</td><td rowspan=1 colspan=1>0.06p=0.634</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable, performance of the assay was evaluated in comparison to the gold standard/reference method, viral culture followed by DFA and/or viral culture followed by sequencing

b. Matrix comparison: Not Applicable

3. Clinical studies:

# a. Clinical Sensitivity and Specificity:

Performance characteristics of the Xpert Flu Assay on prospective and archived specimens were evaluated at six institutions in the U.S. and Australia. Due to the low prevalence of influenza viruses and the difficulty in obtaining fresh influenza-positive specimens, the prospective specimen population for this study was supplemented with frozen archived specimens.

Subjects included individuals whose routine care called for collection of NA/W or NP swab specimens for influenza testing. For eligible subjects, aliquots of leftover sample were obtained for testing with the Xpert Flu Assay and reference testing, and patient management continued at the site per the standard practice.

The Xpert Flu Assay performance was compared to viral culture followed by direct fluorescent assay (DFA). Sequencing was performed for all influenza A positive specimens. For archived specimens, where viral culture was not performed prior to freezing a FDA cleared molecular assay was performed as the comparator assay. Samples included nasal NA/W and NP swab specimens collected for routine testing from patients suspected of influenza infection.

# Overall Results

# Prospective Specimens

A total of 342 prospective NA/W specimens were tested for influenza A, influenza A subtype 2009 H1N1 and influenza B by the Xpert Flu Assay and viral culture plus DFA. A total of 297 prospectively collected NP swab specimens were tested for influenza A, influenza A subtype 2009 H1N1 and influenza B by the Xpert Flu Assay and viral culture plus DFA. All influenza A positive specimens, indentified by viral culture/DFA, were sequenced to differentiate influenza A subtype 2009 H1N1 from other influenza A subtypes

On fresh, prospective NA/W specimens, the Xpert Flu Assay demonstrated a sensitivity and specificity for detection of influenza A of $8 5 . 7 \%$ and $9 9 . 1 \%$ , respectively, relative to viral culture plus DFA, with sequence confirmation of all influenza A positive viral isolates, or specimens in transport medium if isolates were not available. The Xpert Flu Assay sensitivity and specificity for influenza A subtype 2009 H1N1 with NA/W specimens were $100 \%$ and $9 8 . 8 \%$ , respectively. The Xpert Flu Assay sensitivity and

specificity for influenza B with NA/W specimens were $100 \%$ and $9 9 . 4 \%$ , respectively.

Xpert Flu Assay Performance on Prospective NA/W Specimens: Influenza A   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=4 colspan=1>Sees R sE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>332</td><td rowspan=1 colspan=1>333</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>335</td><td rowspan=1 colspan=1>342</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>85.7% (95% CI: 42.1-99.6)99.1% (95% CI: 97.4-99.8)</td></tr></table>

a Testing results by sequencing: 3 of 3 were H1N1. b Testing results by sequencing: Flu A

Xpert Flu Assay Performance on Prospective NA/W Specimens: Influenza A, 2009 H1N1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA &amp; Sequencing</td></tr><tr><td rowspan=4 colspan=1>Se R esX</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4a</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>334</td><td rowspan=1 colspan=1>334</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>338</td><td rowspan=1 colspan=1>342</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>100% (95% CI: 39.8-100)98.8% (95% CI: 97.0-99.7)</td></tr></table>

a Testing results by sequencing: 3 of 4 were H1N1; 1 of 4 was Flu A.

Xpert Flu Assay Performance on Prospective NA/W Specimens: Influenza B   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=4 colspan=1>Ses R E</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>333</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>335</td><td rowspan=1 colspan=1>342</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>100% (95% CI: 65.2-100)99.4% (95% CI: 98.1-99.9)</td></tr></table>

a Testing results by sequencing: 2 of 2 were Flu B.

On prospectively collected NP swabs, the Xpert Flu Assay demonstrated a sensitivity and specificity for detection of influenza A of $100 \%$ and $9 8 . 3 \%$ , respectively, relative to viral culture plus DFA, with sequence confirmation of all influenza A positive viral isolates, or specimens in transport medium if isolates were not available. The Xpert Flu Assay sensitivity and specificity for influenza A subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ with NP swabs were $100 \%$ and $9 9 . 0 \%$ , respectively. The Xpert Flu Assay sensitivity and specificity for influenza B with NP swabs were $8 7 . 5 \%$ and $9 9 . 7 \%$ , respectively.

Xpert Flu Assay Performance on Prospective NP Swab Specimens: Influenza A   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=4 colspan=1>Ses R esX</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>5a</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>285</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>290</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>100% (95% CI: 59.0-100)98.3% (95% CI: 96.0-99.4)</td></tr></table>

a Testing results by sequencing: 3 of 5 were H1N1; 2 of 5 were Flu A.

Xpert Flu Assay Performance on Prospective NP Swab Specimens: Influenza A, 2009 H1N1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA &amp; Sequencing</td></tr><tr><td rowspan=4 colspan=1>Seesh  esE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>289</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>292</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>100% (95% CI: 47.8-100)99.0% (95% CI: 97.0-99.8)</td></tr></table>

a Testing results by sequencing: 2 of 3 were H1N1; 1 of 3 was Flu A.

Xpert Flu Assay Performance on Prospective NP Swab Specimens: Influenza B   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=4 colspan=1>Ses R RsE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>288</td><td rowspan=1 colspan=1>289</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>289</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Sensitivity:Specificity:</td><td rowspan=1 colspan=2>87.5% (95% CI: 47.3-99.7)99.7% (95% CI: 98.1-100)</td></tr></table>

a Testing results by sequencing: Flu B. b Testing results by sequencing: Flu A.

# Archived Specimens

A total of 425 archived NA/W specimens were tested for influenza A, influenza A subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ and influenza B by the Xpert Flu Assay and a FDA cleared molecular assay. A total of 150 archived NP swab specimens were tested for influenza A, influenza A subtype 2009 H1N1 and influenza B by the Xpert Flu Assay and viral culture plus DFA; 177 archived NP swab specimens did not have viral culture results available and were tested by the FDA cleared molecular comparator assay. All influenza A positive specimens indentified by viral culture/DFA or the FDA cleared molecular comparator assay were sequenced to differentiate influenza A subtype 2009 H1N1 from other influenza A subtypes.

On archived NA/W specimens, the Xpert Flu Assay demonstrated a positive and negative agreement for detection of influenza A of $9 9 . 4 \%$ and $100 \%$ , respectively, relative to viral culture plus DFA, with sequence confirmation of all influenza A positive viral isolates, or specimens in transport medium if isolates were not available. The Xpert Flu Assay positive and negative agreement for influenza A subtype $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ with NA/W specimens were $9 8 . 4 \%$ and $9 9 . 7 \%$ . The Xpert Flu Assay positive and negative agreement for influenza B with NA/W specimens were $100 \%$ and $100 \%$ , respectively.

Xpert Flu Assay Performance on Archived NA/W Specimens: Influenza A   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator</td></tr><tr><td rowspan=4 colspan=1>Ses R ksE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>266</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>160</td><td rowspan=1 colspan=1>265</td><td rowspan=1 colspan=1>425</td></tr><tr><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>99.4% (95% CI: 96.6-100)100% (95% CI: 98.6-100)</td></tr></table>

a Testing by sequencing: no sequence match for Flu A, H1N1 or Flu B

Xpert Flu Assay Performance on Archived NA/W Specimens: Influenza A, 2009 H1N1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator &amp; Sequencing</td></tr><tr><td rowspan=4 colspan=1>Sss R esE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>295</td><td rowspan=1 colspan=1>297</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>126</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>422c</td></tr><tr><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>98.4% (95% CI: 94.4-99.8)99.7% (95% CI: 98.1-100)</td></tr></table>

a Testing results by sequencing: Flu A, not H1N1. b Testing results by sequencing: 2 of 2 H1N1. $^ \mathrm { c } _ { 3 }$ samples excluded due to PHRED score ${ < } 2 0$ .

Xpert Flu Assay Performance on Archived NA/W Specimens: Influenza B   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator</td></tr><tr><td rowspan=4 colspan=1>Seas R esE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>385</td><td rowspan=1 colspan=1>385</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>385</td><td rowspan=1 colspan=1>425</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>100% (95% CI: 91.2-100)100% (95% CI: 99.0-100)</td></tr></table>

On archived NP swabs, the Xpert Flu Assay demonstrated a positive and negative agreement for detection of influenza A of $9 7 . 5 \%$ and $100 \%$ , respectively, relative to viral culture plus DFA, with sequence confirmation of all influenza A positive viral isolates, or specimens in transport medium if isolates were not available. The Xpert Flu Assay positive and negative agreement for influenza A subtype 2009 H1N1 with NP swabs were $100 \%$ and $100 \%$ , respectively. The Xpert Flu Assay positive and negative for influenza B with NP swabs were $9 3 . 8 \%$ and $9 9 . 2 \%$ , respectively.

Xpert Flu Assay Performance vs. Comparator Method with NP Swab Specimens: Influenza A   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=1 colspan=1>FE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>115</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>97.5% (95% CI: 92.7-99.5)100% (95% CI: 89.1-100)</td></tr></table>

a Discrepant testing results by sequencing: 2 of 3 were no sequence match for Flu A, H1N1, or Flu B; 1 of 3 was Flu B.

Xpert Flu Assay Performance vs. Comparator Method with NP Swab Specimens: 2009 H1N1   
a One sample excluded due to PHRED score ${ < } 2 0$ .   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA &amp; Sequencing</td></tr><tr><td rowspan=4 colspan=1>Sees r esY</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>84</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>65</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>149a</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>100% (95% CI: 95.7-100)100% (95% CI: 94.5-100)</td></tr></table>

Xpert Flu Assay Performance vs. Comparator Method with NP Swab- Specimens: Influenza B   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Culture/DFA</td></tr><tr><td rowspan=4 colspan=1>Sees R sE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>119</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>93.8% (95% CI: 79.2-99.2)99.2% (95% CI: 95.4-100)</td></tr></table>

a Testing results by sequencing: Flu B. b Testing results by sequencing: no sequence match for Flu A, H1N1 or Flu B.

On archived NP swabs, the Xpert Flu Assay demonstrated a positive agreement and negative agreement for detection of influenza A of $9 8 . 1 \%$ and $9 9 . 2 \%$ , respectively, relative to the ProFlu $^ +$ Assay, with sequence confirmation of all influenza A positive viral isolates, or specimens in transport medium if isolates were not available. The Xpert Flu Assay positive and negative agreement for influenza A subtype 2009 H1N1 with NP swabs were $100 \%$ and $9 9 . 3 \%$ , respectively. The Xpert Flu Assay positive and negative agreements for influenza B with NP swabs were $9 3 . 8 \%$ and $100 \%$ , respectively.

Xpert Flu Assay Performance on Archived NP Swab Specimens: Influenza A   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator</td></tr><tr><td rowspan=4 colspan=1>Sees R sE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>lab</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>121</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>121</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=2>Positive Agreement:Negative Agreement:</td><td rowspan=1 colspan=2>98.1% (95% CI: 89.7-100)99.2% (95% CI: 95.5-100)</td></tr></table>

$^ \mathrm { a } \mathrm { N o }$ test results by sequencing available. b Testing results by sequencing: Flu A

Xpert Flu Assay Performance vs. Comparator Method with NP Swab Specimens: 2009 H1N1   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator &amp; Sequencing</td></tr><tr><td rowspan=4 colspan=1>Sees R esE</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>142</td><td rowspan=1 colspan=1>142</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>172b</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Positive Agreement: 100% (95% CI: 88.1-100)Negative Agreement: 99.3% (95% CI: 96.2-100)</td></tr></table>

${ } ^ { \mathrm { a } } \mathrm { N o }$ sequencing test results available. b Sequence confirmation not available for one sample.

Xpert Flu Assay Performance vs. Comparator Method with NP Swab Specimens: Influenza B   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>FDA Cleared Molecular Comparator</td></tr><tr><td rowspan=4 colspan=1>See R eX</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>158</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>173</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Positive Agreement: 93.8% (95% CI: 69.8-99.8)Negative Agreement: 100% (95% CI: 97.7-100)</td></tr></table>

a No test results by sequencing available.

Of the Xpert Flu Assays runs performed with eligible specimens, $9 7 . 1 \%$ (1351/1391) of these specimens were successful on the first attempt. The remaining 40 gave indeterminate results on the first attempt (26 ERROR, 10 INVALID and 4 NO

RESULT). Thirty-six of the 40 specimens yielded valid results after a single retest; four of the specimens were indeterminate on the second attempt. The assay success rate was equivalent for archived $[ 9 6 . 8 \%$ (727/751)] and fresh $[ 9 7 . 5 \%$ (624/640)] specimens.

4. Clinical cut-off: Not Applicable   
5. Expected values/Reference range:

In the prospective Cepheid Xpert Flu clinical study there was a total of 649 samples collected. A total of 342 prospective NA/W specimens were tested for influenza A, influenza A subtype 2009 H1N1 and influenza B by the Xpert Flu Assay and viral culture plus DFA. A total of 297 prospectively collected NP swab specimens were tested for influenza A, influenza A subtype 2009 H1N1 and influenza B by the Xpert Flu Assay and viral culture plus DFA. These samples were collected during the 2010 influenza season at three clinical laboratories in Australia from August through October of 2010 and from a U.S. clinical site from May to mid-August. The age demographics for the prospectively collected samples are as follows:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>NA/W</td><td rowspan=1 colspan=4>NP Swabs</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>FluA +</td><td rowspan=1 colspan=1>H1N1+</td><td rowspan=1 colspan=1>FluB+</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>FluA +</td><td rowspan=1 colspan=1>H1N1+</td><td rowspan=1 colspan=1>FluB+</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>259</td><td rowspan=1 colspan=1>2.3%(6/259)</td><td rowspan=1 colspan=1>1.9%(5/259)</td><td rowspan=1 colspan=1>2.3%(6/259)</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0.8%(1/120)</td><td rowspan=1 colspan=1>0.8%(1/120)</td><td rowspan=1 colspan=1>0%(0/120)</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>1.9%(1/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td><td rowspan=1 colspan=1>1.9%(1/52)</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>3.9%(2/51)</td><td rowspan=1 colspan=1>3.9%(2/51)</td><td rowspan=1 colspan=1>5.9%(3/59)</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>7.7%(2/26)</td><td rowspan=1 colspan=1>7.7%(2/26)</td><td rowspan=1 colspan=1>7.7%(2/26)</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>8.6%(7/82)</td><td rowspan=1 colspan=1>6.2%(5/82)</td><td rowspan=1 colspan=1>4.9%(4/82)</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>4.5%(2/44)</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>2.3%(1/44)</td></tr><tr><td rowspan=1 colspan=1>TOTALS</td><td rowspan=1 colspan=1>342</td><td rowspan=1 colspan=1>2.6%(9/342)</td><td rowspan=1 colspan=1>2.3%(8/342)</td><td rowspan=1 colspan=1>2.6%(9/342)</td><td rowspan=1 colspan=1>297</td><td rowspan=1 colspan=1>4.0%(12/297)</td><td rowspan=1 colspan=1>2.7%(8/297)</td><td rowspan=1 colspan=1>2.7%(8/297)</td></tr></table>

The number and percentage of Influenza A, $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ , and Influenza B positive cases prospectively collected as determined positive by the Xpert Flu Assay are: For NA/W prospective samples Influenza A is $2 . 6 \%$ (9/342), A/2009 H1N1 is $2 . 3 \%$ (8/342), and Influenza B is $2 . 6 \%$ (9/342). For NS prospective samples Influenza A is $4 . 0 \%$ (12/297), A/2009 H1N1 is $2 . 7 \%$ (8/297), and influenza B is $2 . 7 \%$ (8/297).

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.