# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k103207   
B. Purpose for Submission: To obtain 510(k) clearance of the JBAIDS Q Fever Detection kit assay   
C. Measurand: Coxiella burnetii (Q Fever) target DNA   
D. Type of Test: A multiplexed real-time Polymerase Chain Reaction (PCR) nucleic acid test with a qualitative result.   
E. Applicant: Idaho Technology, Inc.   
F. Proprietary and Established Names: Joint Biological Agent Identification and Diagnostic System (JBAIDS) Q Fever Detection Kit   
G. Regulatory Information:

#

1. Regulation section: 866.3500 Rickettsia serological reagents

2. Classification: Class I

3. Product code: OVF-Assay, direct, nucleic acid amplification, Q fever

83-Microbiology

# H. Intended Use:

1. Intended use(s):

The Joint Biological Agent Identification and Diagnostic System (JBAIDS) Q Fever Detection Kit is a qualitative real-time polymerase chain reaction (PCR) test kit intended to identify and detect target DNA sequence from Coxiella burnetii in serum collected from individuals suspected of having acute Q fever, typically 7-10 days after onset of symptoms or before antibody formation. This in vitro diagnostic (IVD) test is intended to aid in the diagnosis of Q fever in individuals presenting with signs and symptoms of acute Q fever when used in conjunction with other clinical and laboratory findings. This kit is only intended to aid in the diagnosis of Q fever of patients presenting in the acute stage of the disease. Negative results do not preclude C. burnetii infection and should not be used as the sole basis for diagnosis, treatment or other management decisions.

The JBAIDS Q Fever assay is run on the JBAIDS instrument using the Diagnostic Wizard. Results are for the presumptive identification of $C .$ . burnetii in conjunction with serology and/or other laboratory tests. The following considerations also apply:

The diagnosis of acute Q fever must be made based on history, signs, symptoms, exposure likelihood, and other laboratory evidence, in addition to the identification of $C .$ . burnetii from serum specimens.   
Sensitivity is decreased by ${ \sim } 2 5 { \ - } 4 0 \%$ , with no change in specificity, if the sample is collected after the patient has formed specific antibodies to $C .$ . burnetii, typically 7-10 days after onset of symptoms.   
The definitive identification of $C$ . burnetii from serum specimens requires additional testing and confirmation procedures in consultation with public health or other authorities for whom reports are required.

The test performance characteristics for this system were established with banked, frozen serum specimens that were sequentially received during a specified time period. The safety and effectiveness of other types of tests or sample types have not been established.

All users, analysts, and any person reporting diagnostic results from use of this device should be trained to perform and interpret the results from this procedure by JBAIDS instructors or designees prior to use. Use of this device is limited to designated Department of Defense (DoD) laboratories equipped with the JBAIDS instruments.

2. Indication(s) for use:

Same as Intended use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: JBAIDS instrument and JBAIDS software

# I. Device Description:

The JBAIDS Q Fever Detection Kit is specifically designed for performing real-time PCR in glass capillaries using the JBAIDS instrument and JBAIDS software. The reagent kit is a multiplexed assay that that can detect the target organism, C. burnetii, while monitoring for inhibition through the use of an internal inhibition control. Real-time PCR is accomplished with the use of a hydrolysis probe, which detects a specific sequence within the region amplified by the assay primers. The probe is labeled on one end with a fluorescent reporter molecule and on the other end with a quencher. When the probe is intact, the quencher absorbs the light emitted by the reporter molecule. During PCR, the intact probe hybridizes to the amplicon, and the exonuclease activity of Taq polymerase separates the fluorophore from the quencher, generating a fluorescent signal that is detected by the instrument. The fluorescent signal increases as additional template is amplified and more probe is hydrolyzed.

The JBAIDS Q Fever Detection Kit assay amplifies a unique insertion sequence (IS1111) that is found in multiple copies in the C. burnetii genome. Sequencing has shown the Nine Mile strain contains 20 copies of the IS1111 element per organism; however, the number of elements in other strains varies from 7 to 110. Isolates of C. burnetii missing the IS1111 element have not been identified.

The target assay has been multiplexed with an inhibition control. The internal inhibition control (IC) shares primers with the target assay but utilizes a probe that emits a fluorescent signal read in the $7 0 5 \mathrm { n m }$ channel while the target assay is detected in the $5 3 0 \mathrm { n m }$ channel. The template for the IC is a linearized plasmid containing an artificial intervening sequence flanked by the target assay primer binding sites. IC template is added to each reaction at a level that is expected to amplify reliably.

The primers and probes were selected based upon two basic criteria: 1) they detect all sequenced virulent isolates of $C .$ . burnetii, and 2) they do not detect sequences in other organisms. The selected primer and probe sequences were compared, using the Basic Local Alignment Search Tool (BLAST), to other sequences contained in the NCBI Nucleotide Collection and were confirmed to meet these criteria. In addition, primers and probes were only selected if they showed limited self- and cross-complementarities (e.g., hairpins, primer dimers) that would not form at temperatures encountered during manufacturing, reaction setup, or PCR. The target probe is labeled on the $5 '$ end with the fluorescent reporter moiety 6-carboxyfluorescein (6-FAM)) and on the $3 ^ { \circ }$ end with the quencher moiety

carboxytetramethylrhodamine (TAMRA). The IC probe is labeled on the $5 ^ { \circ }$ end with 6-FAM conjugated with cy5.5 and the $3 ^ { \circ }$ end is conjugated with Iowa Black quencher.

# Kit Configuration and Components

The JBAIDS Q Fever Detection Kit contains eight vacuum-sealed pouches of freeze-dried PCR reagents, containing three vials each, and eight pouches containing one tube each of reconstitution buffer and reagent grade water. Once resuspended, each reagent vial provides enough material for two reactions, both of which are included in the test run.

![](images/e3e7c251cffec910e3e1714e601dac2530c084929ac6e141f00123c5127fea3a.jpg)

Each reagent pouch contains the following vials:

One Positive Control $( + )$ vial Once reconstituted, the Positive Control vial contains all reagents necessary for PCR, in addition to ${ \sim } 1 3 5$ copies of a linearized plasmid containing the target assay template and ${ \sim } 5 0$ copies of a linearized plasmid containing the inhibition control template. The Positive Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens. Amplification in the Positive Control gives assurance that kit reagents are functioning properly and that the assay setup has been performed correctly. A Positive Control must be included with each JBAIDS run.

One Negative Control (-) vial. Once reconstituted, the Negative Control vial contains all materials required to perform a PCR reaction in addition to ${ \sim } 5 0$ copies of a linearized plasmid containing the inhibition control template, but not the target template, and should therefore give negative results. The Negative Control is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ water before testing in parallel with patient specimens to provide assurance that the setup procedure has been performed without contamination. A Negative Control must be included with each JBAIDS run.

One Unknown vial. The contents of the Unknown vial are identical to the Negative Control. The Unknown vial is resuspended with $2 0 \mu \mathrm { L }$ reconstitution buffer and $2 0 \mu \mathrm { L }$ purified patient sample, and will therefore give negative results unless the target template is found in the patient sample.

Each reconstitution buffer/reagent grade water pouch contains:

2X Reconstitution Buffer (2X RB). The buffer solution (purple buffer) is matched to the Q Fever assay and is used undiluted. The buffer enhances PCR kinetics and is used to resuspend JBAIDS freeze-dried reagents. Once reconstituted, all freeze dried reagent vials contain all of the components required for PCR.

Reagent Grade Water. Reagent grade water is molecular biology grade water used to reconstitute the Positive and Negative Controls.

# JBAIDS Instrument and Software

The JBAIDS Q Fever Detection Kit is designed for use with the JBAIDS instrument. Because the instrument and software were previously cleared for use as an IVD device (K051713), only a brief description of both is provided here.

The JBAIDS instrument is a ruggedized, portable real-time PCR instrument designed to withstand the conditions of transport and use likely to be encountered in a military field laboratory. The instrument is composed of an air thermocycler that amplifies specific DNA sequences using PCR and a fluorimeter that measures fluorescence signals associated with production of a PCR product (amplicon).

Samples are contained in glass capillaries (up to 32 individual PCR reactions per instrument run) that are loaded into a JBAIDS carousel and placed into the instrument for thermocycling. Glass capillaries have heat-transfer and optical properties that are favorable for thermocycling and fluorescence analysis. By using real-time PCR, the instrument can rapidly identify a wide array of specific DNA sequences, including those associated with infectious organisms.

![](images/d9f5ab09cc1c99e66cd766a9c936c04c48af25d9703d0396fb5121b104c01d29.jpg)

The JBAIDS instrument must be able to detect fluorescence signals generated during the PCR reaction. This is performed using a 3-channel fluorimeter, which is specifically

designed to excite a fluorescent molecule attached to the probe and capture the light emitted. For the JBAIDS Q Fever Detection Kit, the fluorescent signal from the target assay is detected in channel 1, while the signal from the IC assay is detected in channel 3.

The JBAIDS Software is preloaded on a ruggedized laptop computer. The software controls the instrument’s thermal cycling functions, acquires the fluorescence data from the instrument, and displays the fluorescence data for the user during the run. When the run is finished, the software’s Detector module analyzes the data and displays test results. The instrument and software have the ability to perform either diagnostic (IVD) or surveillance (environmental) testing.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Focus Diagnostics Q Fever IgM IFA Focus Diagnostics Q Fever IgG IFA

2. Predicate 510(k) number(s): K922374 K913906   
3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceJBAIDS Q Fever DetectionKit</td><td colspan="1" rowspan="1">PredicateFocus Diagnostics Q FeverIgG (K922374) and IgM(K913906) tests</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">Qualitative detection ofCoxiella burnetii.</td><td colspan="1" rowspan="1">Detection and semi-quantitation of the humanIgG and IgM antibodyresponse to phase I andphase II Coxiella burnetiiantigens and as an aid inthe diagnosis of Q fever.</td></tr><tr><td colspan="1" rowspan="1">Indications for Use</td><td colspan="1" rowspan="1">Identification of C. burnetiiin individuals suspected ofhaving acute Q Fever.</td><td colspan="1" rowspan="1">Serodiagnosis of both acuteand chronic C. burnetiiinfections.</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Serum</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time Required forAnalysis of Specimen</td><td colspan="1" rowspan="1">Less than 3 hours</td><td colspan="1" rowspan="1">Same</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">DeviceJBAIDS Q Fever DetectionKit</td><td colspan="1" rowspan="1">PredicateFocus Diagnostics Q FeverIgG (K922374) and IgM(K913906) tests</td></tr><tr><td colspan="1" rowspan="1">Technological Principles</td><td colspan="1" rowspan="1">Real-time PCR usinghydrolysis probes.</td><td colspan="1" rowspan="1">Microscopic visualizationof human antibodies for C.burnetii bacteria via a twostage "sandwich"procedure using afluorescently labeledantibody that binds to thehuman antibodies.</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">IS1111 DNA sequencesunique to C. burnetii.</td><td colspan="1" rowspan="1">Human antibody responseto C. burnetii.</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">JBAIDS instrument(K051713)</td><td colspan="1" rowspan="1">Fluorescent microscope</td></tr><tr><td colspan="1" rowspan="1">Sample PreparationMethod</td><td colspan="1" rowspan="1">Up front sample processingis required to extract nucleicacid.</td><td colspan="1" rowspan="1">Diluted samples are testeddirectly.</td></tr><tr><td colspan="1" rowspan="1">Sample Controls</td><td colspan="1" rowspan="1">Internal control ismultiplexed with the targettest.</td><td colspan="1" rowspan="1">No sample specific control.</td></tr><tr><td colspan="1" rowspan="1">Testing Strategy</td><td colspan="1" rowspan="1">One time point testingduring the symptomaticphase of the disease.</td><td colspan="1" rowspan="1">Two serology time pointsare suggested. One duringthe symptomatic phase ofthe disease with follow-uptesting 2-3 weeks later.</td></tr><tr><td colspan="1" rowspan="1">Optimal Window ofDetection</td><td colspan="1" rowspan="1">Early in the disease courseprior to antibody formation.</td><td colspan="1" rowspan="1">Later in the disease coursedue to the time required forantibody formation.</td></tr><tr><td colspan="1" rowspan="1">Test Interpretation</td><td colspan="1" rowspan="1">Automated testinterpretation and reportgeneration.</td><td colspan="1" rowspan="1">Subjective interpretation byuser.</td></tr><tr><td colspan="1" rowspan="1">Physical Properties</td><td colspan="1" rowspan="1">Freeze dried reagents withreconstitution buffer andwater provided in kit.</td><td colspan="1" rowspan="1">Liquid reagents. IgM andIgG are separate test kits.</td></tr><tr><td colspan="1" rowspan="1">Storage</td><td colspan="1" rowspan="1">Room temperature (1828C).</td><td colspan="1" rowspan="1">Refrigerator temperature(2-8 °C)</td></tr></table>

# K. Standard/Guidance Documents Referenced:

Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial   
Pathogens, FDA Guidance Document (DRAFT: December 8, 2005)   
Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses, FDA Guidance Document (DRAFT: February 15, 2008)   
Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline, MM3-P2 (February 2006)   
Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005)   
User Verification of Performance for Precision and Trueness, CLSI Approved Guideline EP15-A2 (April 2006, second printing)   
Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline, CLSI Approved Guideline MM13-A (January 2006) Procedures for the Collection of Diagnostic Blood Specimens by Venipuncture; Approved Standard-Sixth Edition, CLSI Approved Guideline H3-A6 (October 2007)

# L. Test Principle:

The JBAIDS Q Fever Detection Kit is based on four major processes:

Sample collection and purification   
Reconstitution of freeze-dried reagents   
Amplification and detection of target DNA on the JBAIDS instrument   
Automated interpretation of the amplification curves using the JBAIDS software with report generation.

# Sample Collection and Purification

The JBAIDS Q Fever Detection kit can be used to test human serum specimens. Samples must be purified prior to testing with the JBAIDS Q Fever Detection Kit. The purpose of sample purification is to release nucleic acid contained in the unknown sample and to remove extraneous materials (e.g., proteins and chemicals) that can interfere with the PCR reaction. The JBAIDS Q Fever Detection kit has been validated for use with either the IT 1-2-3 QFLOWdna Sample Purification Kit or the IT 1-2-3 Platinum Path Sample Purification Kit.

PCR Reagents and Controls

Purified samples are tested for the presence of $C .$ . burnetii DNA using freeze-dried PCR reagents specific to $C .$ . burnetii. Reagent vials that are intended for testing patient samples are labeled as Unknowns. To ensure accurate test results, each assay requires that a Positive Control and a Negative Control be included in each JBAIDS run.

# Instrument Operation and Thermocycling

Each JBAIDS run can accommodate up to 12 samples with accompanying controls. The Diagnostic Wizard guides the operator through the process of setting up the run in compliance with the JBAIDS Q Fever Detection Kit package insert. After selecting the appropriate assays and entering sample information, the operator transfers prepared capillaries to the instrument carousel, guided by the loading pattern displayed in the Diagnostic Wizard.

With the capillaries loaded into the instrument, the operator signals the instrument to start the PCR run. The instrument automatically controls the temperature heating cycle in the carousel. At intervals during temperature cycling, fluorescence emission is monitored as the carousel rotates to position each sample above the fluorimeter.

Temperature and fluorescence data are displayed in real-time. After starting the run, no input is required and the operator need not supervise the run. At the conclusion of the run, the operator may proceed directly to data analysis and reporting.

# Data Analysis and Reporting

Each JBAIDS test is analyzed and assigned a final result by the Detector module of the JBAIDS Software. Possible final results are positive, negative, inhibited, uncertain, or invalid. To assign a final test result, Detector first analyzes the data from each capillary independently before analyzing the sample duplicates together. Finally, the software assigns a final result, or a combined call, based on the results of the sample and all of its controls.

A JBAIDS report is generated automatically for each test run. When the Diagnostic Wizard is used, reports from the instrument are identified with the phrase “For In Vitro Diagnostic Use.”

# Follow-up Testing

If the Q Fever assay yields a negative or positive result, then the JBAIDS testing is complete.   
Any other result requires follow-up testing, as described below.   
• Invalid – If either the Positive Control or Negative Control fails to give the expected results, all Unknown samples associated with the control are called invalid and the samples must be retested in a new JBAIDS run.   
• Uncertain – An uncertain result on a single sample requires retesting with the same purified sample.

Inhibited – If the Inhibition Control indicates that an inhibitor is present in a specific sample, follow-up testing can determine if the sample truly contains an inhibitor or if the result was caused by other factors, such as improper technique. The effect of PCR inhibitors is often removed by performing a 1:10 dilution of the sample that yielded the inhibited test result. The diluted sample is retested along with the original sample that yielded the inhibited result.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Simulated test specimens were prepared by spiking serum with inactivated C. burnetii at four test concentrations. The samples were prepared at ITI, aliquoted and frozen prior to distribution to the test sites. Aliquots of each specimen were tested at three test sites. The 12-sample panel was tested twice each day for five days at the two external testing sites and 7 testing days at ITI. On each testing day, two users at each site purified and tested one aliquot of each specimen, 12 total specimens per operator per day. One of the operators used the IT 1-2- 3™ QFLOWdna Sample Purification Kit and the other operator used the IT ${ \cal I } { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit.

Between-site testing is a good indication of overall system performance because it represents the performance of the system in its intended use environment and includes many variables. Sources of variability included different runs, instruments, sites, operators, days, reagent lots, and sample purification kits.

Between-Site Reproducibility Results for the JBAIDS Q Fever Detection Kit (fractional positive test results)   

<table><tr><td rowspan=2 colspan=1>Test Level</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path SamplePurification Kit</td><td rowspan=1 colspan=4>IT 1-2-3 QFLOWdna SamplePurification Kit</td><td rowspan=3 colspan=1>Both Kits,All Sites</td></tr><tr><td rowspan=1 colspan=4>Test Location</td><td rowspan=1 colspan=4>Test Location</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site 1Overall</td><td rowspan=1 colspan=1>Site 2Overall</td><td rowspan=1 colspan=1>Site 3Overall</td><td rowspan=1 colspan=1>Overall for AllSites</td><td rowspan=1 colspan=1>Site 1Overall</td><td rowspan=1 colspan=1>Site 2IOverall</td><td rowspan=1 colspan=1>Site 3|Overall</td><td rowspan=1 colspan=1>Overall for AllSites</td></tr><tr><td rowspan=1 colspan=1>5X LoD</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>51/51100%</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>51/51100%</td><td rowspan=1 colspan=1>102/102100%</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>51/51100%</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>15/15</td><td rowspan=1 colspan=1>51/51100%</td><td rowspan=1 colspan=1>102/102100%</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>42/42100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>102/102100%</td><td rowspan=1 colspan=1>42/42100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>102/102100%</td><td rowspan=1 colspan=1>204/204100%</td></tr><tr><td rowspan=1 colspan=1>LoD/15</td><td rowspan=1 colspan=1>11/21</td><td rowspan=1 colspan=1>8/15</td><td rowspan=1 colspan=1>13/15</td><td rowspan=1 colspan=1>32/5163%</td><td rowspan=1 colspan=1>7/21</td><td rowspan=1 colspan=1>8/15</td><td rowspan=1 colspan=1>6/15</td><td rowspan=1 colspan=1>21/5141%</td><td rowspan=1 colspan=1>53/10252%</td></tr><tr><td rowspan=1 colspan=1>Detection allLevels</td><td rowspan=1 colspan=1>53/6384%</td><td rowspan=1 colspan=1>38/4584%</td><td rowspan=1 colspan=1>43/4596%</td><td rowspan=1 colspan=1>134/15388%</td><td rowspan=1 colspan=1>49/6378%</td><td rowspan=1 colspan=1>38/4584%</td><td rowspan=1 colspan=1>36/4580%</td><td rowspan=1 colspan=1>123/15380%</td><td rowspan=1 colspan=1>123/30684%</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0/210%</td><td rowspan=1 colspan=1>0/150%</td><td rowspan=1 colspan=1>0/150%</td><td rowspan=1 colspan=1>0/510%</td><td rowspan=1 colspan=1>0/210%</td><td rowspan=1 colspan=1>0/150%</td><td rowspan=1 colspan=1>0/150%</td><td rowspan=1 colspan=1>0/510%</td><td rowspan=1 colspan=1>0/210%</td></tr></table>

As shown in the table above, the JBAIDS Q Fever Detection Kit correctly made a positive call $100 \%$ of the time on samples containing $\geq \mathrm { L o D }$ of inactivated $C .$ . burnetii and a negative call $100 \%$ of the time on samples that did not contain inactivated $C .$ . burnetii. As expected, specimens spiked below LoD have variable results.

The JBAIDS Q Fever Detection system is reliable with respect to variability between site, between day, between run, between lot, between instrument, and between operators. The JBAIDS Q Fever Detection Kit provides reproducible test results for serum samples processed with either the Platinum Path or QFLOWdna Sample Purifications Kits.

While the JBAIDS Q Fever Detection Kit is designed to provide qualitative results, an evaluation of precision was performed to estimate the overall variability of the test system. Testing precision was determined by analyzing the overall variation of the Cp values, represented by the standard deviation (SD). It was expected that the SD of the Cp values would be less than one cycle for each test level.

Overall, system variability was low for samples tested at or above LoD as demonstrated by a SD for each test level that was below one cycle. Due to the high variability and a high occurrence of negatives and uncertains when testing samples below LoD, the High Negative samples were not included in this evaluation.

The observed variability seen per site, per specimen and per specimen at each site was overlapping and was similar to the overall variability. This suggests that the overall system is not significantly impacted by variability between specimens or variability between sites, which includes multiple operators and multiple instruments.

The variability of sample purified using QFLOWdna was slightly lower than for sample purified using Platinum Path, (Low Positive $\%$ CV for QFLOWdna 1.4 vs 1.9 for Platinum Path and Medium Positive $\%$ CV of 1.3 for QFLOWdna vs. 2.3 for Platinum Path). However, samples purified with either kit show reliable results.

The Reproducibility and Precision studies are acceptable for this type of device.

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Positive, Negative and Inhibition controls were included in each run and gave expected results. If not, the runs failed or were called invalid and the run was repeated.

# d. Detection limit:

Limit of Detection (LoD) was determined for the Q Fever target assay using serum specimens spiked with live Coxiella burnetii, Nine Mile. Specimens spiked with 10- fold dilutions of organism $( 1 0 0 0 { - } 0 . 3 \ \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ , were purified with the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ QFLOWdna Kit and the IT $\mathrm { i } - 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Kit, and tested with the target assays to determine an estimated limit of detection for the test system. The LoD was estimated by determining the lowest concentration of organism (in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) at which at least $9 5 \%$ of samples would test positive. Twenty individual specimens (from independent donors) spiked at the estimated LoD ( $1 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) were successfully detected by the target assay for both purification kits.

# e. Analytical specificity:

Analytical Inclusivity-A panel of $1 0 \mathrm { Q }$ Fever isolates, representative of the five distinct genomic groups (I-V) known to cause disease in humans, was spiked into serum specimens at concentrations determined to be equivalent to 1, 10 and 100 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The prepared specimens were purified using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit and then tested using the JBAIDS Q Fever Detection Kit.

Q Fever Isolates Used to Evaluate Inclusivity.

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Group</td></tr><tr><td rowspan=1 colspan=1>Nine Mile Phase I</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>QiYi</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>RSA334</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Henzerling Phase ISalk</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>M44 - Grita</td><td rowspan=1 colspan=1>II</td></tr><tr><td rowspan=1 colspan=1>Idaho Q</td><td rowspan=1 colspan=1>III</td></tr><tr><td rowspan=1 colspan=1>Q173-P</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>Q154-kAV</td><td rowspan=1 colspan=1>IV</td></tr><tr><td rowspan=1 colspan=1>WAV</td><td rowspan=1 colspan=1>V</td></tr><tr><td rowspan=1 colspan=1>Q229</td><td rowspan=1 colspan=1>V</td></tr></table>

Organisms in the inclusivity panel were expected to test positive and have Cp values near 32 at LoD. For the inclusivity panel testing, the test result and Cp value were compared to reference material ( $C .$ burnetii Nine Mile inactivated organism) and were considered equivalent to the reference strain, if the test result was positive and the average Cp value for the isolate at LoD was not more than 1.5 cycles later than the average Cp value obtained for the reference strain. Earlier Cp values were expected for some stains due to the assay target being a multi-copy target.

All strains gave positive results at all concentrations tested. Group VI was excluded from analysis since it is avirulent.

Analytic specificity-A panel of 22 phylogenetically related or clinically relevant organisms were used to evaluate the test system analytic specificity.

Exclusivity Panel

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Relevance</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Nearest Neighbor</td></tr><tr><td rowspan=1 colspan=1>Legionella longbeacheae</td><td rowspan=1 colspan=1>Nearest Neighbor</td></tr><tr><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>Nearest Neighbor/Cross reactive withserology</td></tr><tr><td rowspan=1 colspan=1>Bartonella henselae</td><td rowspan=1 colspan=1>Cross reactive with serology</td></tr><tr><td rowspan=1 colspan=1>Rickettsia prowazekii</td><td rowspan=1 colspan=1>Other small obligate intracellularbacteria, agent of zoonoses</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium tuberculosis</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Brucella meliteusis</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Orientia tsutsugamushi</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Francisella tularensis</td><td rowspan=1 colspan=1>Similar symptoms</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Aggregatibacteractinomycetemcomitans</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Haemophilus parainfluenzae</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Vibrio cincinnatiensis</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Commonly encountered</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>Commonly encountered</td></tr></table>

All 22 $( 1 0 0 \% )$ organisms, tested at high concentrations, were negative with the target assay.

An inability to obtain live Ehrlichia for exclusivity testing lead to in silico screening of primers and probe to determine cross reactivity of the JBAIDS Q Fever assay with Ehrlichia spp. and was extended to include Neorickettsia spp. In silico screening was performed using BLAST to search the genomes of these genera for sequence identity matches based upon a 15 word match between the primers or probes with the organism genome. This evaluation indicated that the primers and probes in the JBAIDS Q Fever assay would not cross-react with organisms from these genera.

# Interfering Substances

The JBAIDS Q Fever assay was tested against a panel of potentially interfering substances. The concentration of each substance tested represented a relevant concentration in accordance with NCCLS EP7-A2 Interference Testing in Clinical Chemistry Approved Guideline. All endogenous and exogenous substances typically found in serum were spiked into the sample matrix and subjected to sample processing with both IT 1-2-3 QFLOWdna and Platinum Path prior to testing with the JBAIDS Q Fever Detection Kit. The sample purification portion of the system was designed to isolate the DNA and remove other impurities; therefore, sample processing should eliminate most of the potentially interfering substances from the purified sample. Because technique-specific substances can be encountered during reaction setup, all such substances were added to the purified samples immediately prior to reaction setup.

Once prepared, each sample was tested with the target assay which is multiplexed with an internal Inhibition Control. The substance was considered to be an interfering substance if the target assay result was negative or inhibited. A substance was considered potentially inhibitory if the $\mathrm { C p }$ of the target assay was delayed by more than three cycles or if the Fmax of the target assay was decreased by more than $50 \%$ . Because high concentrations of $C .$ burnetii genomic DNA will competitively inhibit the IC assay (due to the shared primers), any sample with a positive result for the Q Fever assay is considered to be positive regardless of the IC result.

The following table lists the potentially interfering substances that were tested.

List of Evaluated Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=2>Exogenous Substances</td><td rowspan=1 colspan=1>Technique Specific Substances</td></tr><tr><td rowspan=1 colspan=1>AlbuminBilirubinCholesterol (total)GlucoseHemoglobinImmunoglobulinsTriglycerides</td><td rowspan=1 colspan=1>AcetaminophenAmoxicillinAscorbic AcidAspirinCefotaximeChloroquineCiprofloxacinDoxycyclineErythromycinGentamicin sulfateIbuprofenNaproxen sodium</td><td rowspan=1 colspan=1>RifampinStreptomycinSulfamethoxazoleTetracyclineTobramycinTrimethoprimAcid Citrate Dextrose (ACD)Ethylenediaminetetraacetic acid (EDTA)HeparinSodium CitrateSodium Polyanetholesulfonate (SPS)Specimen collection Device</td><td rowspan=1 colspan=1>Platinum Path MB Binding BufferPlatinum Path Wash Buffer (from 4th well)</td></tr></table>

The following substances were shown to interfere with the PCR reactions used in this test system.

• Platinum Path MB Binding Buffer Plasma obtained from the following types of blood tubes o Ethylenediaminetetraacetic acid (EDTA) o Heparin

o Sodium Polyanetholesulfonate (SPS) o Sodium Citrate

From previous studies, the following substances have been identified as interfering with PCR.

• Ethanol   
• Sodium hypochlorite (bleach)   
• DNAZap TM   
• Qiagen buffers AL and AW1 (with or without added ethanol) from the IT 1- 2-3 QFLOWdna kit

f. Assay cut-off:

A data analysis module within assay-specific software applies mathematical modeling of expected amplification curve shapes to each individual capillary. When fixed thresholds with a quadratic formula or crossing point determination are matched, samples are called negative or positive. For undecided samples, an expert system approach is applied that uses filters to assess the fluorescence change around the crossing point.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.   
b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Clinical Sensitivity:

The clinical performance of the JBAIDS Q Fever Detection Kit was evaluated by testing acute serum samples obtained from individuals suspected of having acute Q fever. The study was performed at 3 geographically distinct locations, Australia, France and the Netherlands, using banked, frozen serum specimens that were sequentially received during a specified time period for which standard paired serological testing for Q fever had been previously performed. The results of the serology-based indirect immunofluorescence assays (IFA) were used as the gold standard to which the JBAIDS results were compared. Samples were considered to be positive for acute Q fever if there was a four-fold rise in antibody titer or if there was seroconversion between the paired acute and convalescent specimens. To better simulate a prospective study, all samples of sufficient volume that were submitted for Q Fever serology testing at each site within a given time period were included in the study.

Serum specimens from a total of 749 subjects were evaluated in the study. However 284 were excluded (86 for inconclusive or incomplete serology, 21 due to environmental contamination, 7 for inconclusive JBAIDS results and all 170 samples tested at site 3 due to deviations from the study protocol).

Of the remaining 465 subjects, 254 $( 5 5 \% )$ were serology positive and 211 $( 4 5 \% )$ were serology negative. Sixty-two $( 2 4 \% )$ of the serology positive specimens were considered to be positive due to a seroconversion while 192 $( 7 6 \% )$ were deemed positive due to a 4-fold rise in antibody titer. Of the 211 serology negative specimens, 149 $( 7 1 \% )$ were sero-negative and remained negative and 62 $( 2 9 \% )$ did not exhibit a four-fold rise in titer between paired specimens.

Prior to testing with the JBAIDS Q Fever Detection kit, serum samples were purified using either the IT 1-2-3 Platinum Path or the IT 1-2-3 QFLOWdna sample purification kit. Out of the 465 specimens, 183 specimens had adequate volume to be purified using both purification methods. This resulted in a total of 324 final test results for Platinum Path (170 from serology positive specimens and 154 from serology negative specimens) and 324 final test results for QFLOW (171 from serology positive specimens and 153 from serology negative specimen).

Of the 252 tests on samples that were serology positive based upon a four-fold rise in antibodies, the JBAIDS Q Fever Detection system correctly identified the presence of C. burnetii in $1 7 . 8 \%$ (45/252) samples. However, the positive JBAIDS test result rate for those samples which exhibited seroconversion was significantly higher at $6 2 . 9 \%$ (56/89). These data are consistent with several published reports indicating that PCR detection of $C .$ . burnetii declines significantly once specific antibodies are detected by serology.

As shown in the table below, the clinical sensitivity for the JBAIDS Q Fever Detection Kit was different at the two study sites. Site 1 had a lower detection rate ( $2 9 \%$ for specimens purified using the QFLOWdna extraction kit and $47 \%$ for specimens purified using the Platinum Path purification kit) when compared to the detection rate at site 2 $81 \%$ for specimens purified using the QFLOWdna extraction kit and $7 7 \%$ for specimens purified using the Platinum Path purification kit). The differences observed at the two study sites may be the result of differences in organism strains, patient population or patient management.

Sensitivity and Specificity for the JBAIDS Q Fever Detection Kit in Specimens Tested for Acute Q Fever that were Seronegative

<table><tr><td rowspan=1 colspan=6>SERONEGATIVE SAMPLES</td></tr><tr><td rowspan=2 colspan=1>Site</td><td rowspan=2 colspan=1>Purification Kit</td><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=2>Specificity</td></tr><tr><td rowspan=1 colspan=1>TP/(TP + FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>TN/(TN + FP)</td><td rowspan=1 colspan=1>Percent</td></tr><tr><td rowspan=2 colspan=1>1</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>9/19</td><td rowspan=1 colspan=1>47%(95% Cl=24-71%)</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>100%(95% Cl=91-100%)</td></tr><tr><td rowspan=1 colspan=1>QFLOwdna</td><td rowspan=1 colspan=1>5/17</td><td rowspan=1 colspan=1>29%(95% Cl=10-56%)</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>100%(95% Cl=91-100%)</td></tr><tr><td rowspan=2 colspan=1>2</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>20/26</td><td rowspan=1 colspan=1>77%(95% Cl=56-91%)</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>100%(95% Cl=95-100%)</td></tr><tr><td rowspan=1 colspan=1>QFLOwdna</td><td rowspan=1 colspan=1>22/27</td><td rowspan=1 colspan=1>81%(95% CI=62-94%)</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>100%(95% Cl=95-100%)</td></tr></table>

# $b$ . Clinical specificity:

Of the 307 tests on serology negative samples in the overall analysis, 222 had no antibody to $C .$ . burnetii and 85 failed to show a 4-fold rise in antibody titer between the paired acute and convalescence samples, all but one $( 3 0 6 / 3 0 7 , 9 9 . 7 \% )$ gave the expected negative result when tested with the JBAIDS Q Fever Detection Kit. The one false positive was obtained from one of the samples that failed to show a fourfold rise in antibody titer. When only seronegative specimens samples were analyzed (those samples that didn’t have antibodies in the acute sample but become positive in the convalescent sample) clinical specificity was $100 \%$ . c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off: Not applicable.

5. Expected values/Reference range: Not applicable, qualitative test.

N. Instrument Name: JBAIDS Instrument

# O. System Descriptions:

1. Modes of Operation: Batch

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X_____ or No

3. Specimen Identification: Manual

4. Specimen Sampling and Handling: Manually but guided by the Diagnostic Wizard in the software for the Q Fever assay.

5. Calibration:

The fluorimeter is factory-calibrated. Internal self-check procedures are run with each startup. Cycling temperatures are monitored continuously during a run.

6. Quality Control:

Determined by Positive, Negative, Inhibitory, and DNA extraction controls specific to assay kit.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Carryover and Cross-contamination

To evaluate carryover, negative serum samples were processed and tested alongside highly positive serum samples (spiked with inactivated C. burnetii Nine Mile at $1 \mathrm { X } 1 \overline { { 0 } } ^ { 5 }$ organisms/mL, equivalent to approximately 1000X LoD). Potential carryover during sample purification and sample setup was assessed by processing samples in alternating order (i.e., a negative sample followed by a positive sample) and then testing the processed samples with the Q Fever target assay.

Using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ Platinum Path Sample Purification Kit, all 23 negative serum samples gave negative results for the target assay. Using the IT $1 { - } 2 { - } 3 ^ { \mathrm { T M } }$ QFLOWdna Sample Purification Kit, all 23 negative serum samples gave negative results for the target assay.

# Sample Transport and Storage

To evaluate the serum matrix, 12 individual specimens were used for both the IT 1-2-3 Platinum Path and QFLOWdna purification kits. Eight of the specimens were spiked with live C. burnetii at the LoD level of $1 0 \mathrm { T C I D } 5 0 / \mathrm { m L }$ . The remaining 4 specimens were not spiked.

All 8 positive serum specimens, stored unpurified, tested positive at Day 0, Day 3, Day 7, Day 30, and after a freeze/thaw cycle. The average Cp values for the stored specimens were all within 1 cycle of the day 0 time point suggesting no decrease in assay performance with storage. In addition, the average Fmax values for the time points were never less than half of the Fmax of the day 0 baseline indicating no reduction of test system performance. All 4 negative samples tested negative at all three time points. The average $\mathrm { C p }$ and Fmax values of the associated Inhibition Control assays were similar at all three time points.

These results indicate that serum specimens can be stored at $1 8 { - } 2 8 \ ^ { \circ } \mathrm { C }$ for up to three days, at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to seven days before testing, and at ${ - 2 0 ^ { \circ } C }$ with or without a freeze/thaw cycle for thirty days without compromising the results.

The stability of purified samples was only assessed after purification with IT 1-2-3 Platinum purified samples. All positive samples tested positive with the target assay with similar $\mathrm { C p }$ and Fmax values at Day 0, Day 7, and Day 30. All negative samples tested negative at all three time points with the target assay and the average $\mathrm { C p }$ and Fmax values of the Inhibition Control assays were similar at each time point. These results indicate that purified specimens can be stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to seven days before testing, and at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for thirty days without compromising the results.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.