# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K103175   
B. Purpose for Submission: New device

# C. Measurand:

Adenovirus, Coronavirus HKU1, Coronavirus NL63, Human Metapneumovirus, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype $2 0 0 9 \mathrm { H } 1$ Influenza B, Parainfluenza virus 3, Human Rhinovirus, Enterovirus, Respiratory Syncytial Virus nucleic acids target sequences.

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the FilmArray instrument for the qualitative in vitro detection and identification of multiple respiratory pathogen nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections.

E. Applicant: Idaho Technology Inc.

F. Proprietary and Established Names: FilmArray Respiratory Panel (RP) System Common Name: FilmArray Respiratory Panel (RP) System

G. Regulatory Information:

<table><tr><td colspan="1" rowspan="1">Product</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Regulation Section</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">OCC</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEM</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OTG</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td>NXD</td><td>Class II</td><td>21 CFR 866.3332 Reagents for Detection of Specific Novel Influenza A Viruses</td><td>Microbiology (83)</td></tr><tr><td>001</td><td>Class II</td><td>21 CFR 862.2570 Instrumentation for Clinical Multiplex Test Systems</td><td>Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

1. Intended use:

FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following virus types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus HKU1, Coronavirus NL63, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype $2 0 0 9 \mathrm { H } 1$ , Influenza B, Parainfluenza virus 3, Rhinovirus/Enterovirus, and Respiratory Syncytial Virus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and epidemiological information. Negative results do not preclude respiratory viral infection and should not be used as the sole basis for diagnosis, treatment or other management decisions. Positive results do not rule out bacterial infection or coinfection with other organisms. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (e.g., bacterial and viral culture, immunofluorescence, and radiography) and clinical presentation must be taken into consideration in order to obtain the final diagnosis of respiratory infection.

Due to seasonal low prevalence, performance characteristics for Influenza A/H1, Influenza A/H3, Influenza $\mathbf { A } / 2 \mathbf { 0 0 9 } \mathbf { H 1 }$ , and Influenza B were established primarily with retrospective clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture).

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary. If infection with a novel influenza A virus is suspected based on current clinical and

epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: FilmArray instrument

# I. Device Description:

The FilmArray RP pouch contains primers for the following respiratory pathogens: Adenovirus, Coronavirus (HKU1 and NL63), Influenza A (with subtyping for hemagglutinin genes H1, 2009 H1 and H3), Influenza B, human Metapneumovirus, Parainfluenza virus (serotypes 3), Respiratory Syncytial virus, and Rhinovirus/Enterovirus. The FilmArray RP pouch is a closed system disposable that houses all the chemistry required to isolate, amplify and detect nucleic acid from multiple respiratory pathogens within a single NPS specimen. The rigid plastic component (fitment) of the FilmArray RP pouch contains reagents in freeze-dried form. The flexible plastic portion of the pouch is divided into discrete segments (blisters) which, through interactions with actuators and sensors in the FilmArray instrument, are where the required chemical processes are carried out. The user of the FilmArray RP system loads the sample into the FilmArray RP pouch, places the pouch into the FilmArray instrument, and starts the run. All other operations are automated. A summary of the FilmArray assays and targets is presented in the following table:

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">AssayType</td><td colspan="1" rowspan="1">Number ofPCR2 Assays</td><td colspan="1" rowspan="1">Gene Target</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Hexon</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Nucleoprotein</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Nucleoprotein</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Matrix (Pan 1 assay)Non structural Protein(Pan 2 assay)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Hemagglutinin</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1 2009</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Hemagglutinin</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Hemagglutinin</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Hemagglutinin</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Nucleocapsid</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">5'UTR</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza 3</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Fusion</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial virus(RSV)</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Matrix</td></tr><tr><td colspan="1" rowspan="1">RNA Process Control</td><td colspan="1" rowspan="1">RNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Schizosaccharomycespombe</td></tr><tr><td colspan="1" rowspan="1">PCR2 Control</td><td colspan="1" rowspan="1">DNA</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Arabidposis/ModifiedSynthetic</td></tr></table>

# The FilmArray RP Reagent Kit

The FilmArray RP Regent Kit contains all materials required to complete 30 tests. The contents of the test kit, with a brief description of each component, are described below:

30 FilmArray RP pouches The RP pouches are used to test the patient samples. Each reagent pouch is packaged in a metal canister under vacuum. To maintain the vacuum, the metal canisters are packed in an outer foil wrapper.

• 30 transfer pipettes

The transfer pipettes are used to transfer approximately $2 5 0 \mu \mathrm { L }$ of patient NPS in viral transport media (VTM) to the single use Sample Buffer vials.

• 30 vials Sample Buffer (Red lid)

Each single use vial contains $5 0 0 \mu \mathrm { L }$ of Sample Buffer. Patient sample is added to the sample buffer before it is injected into the FilmArray RP pouch. The Sample Buffer is a guanidine hydrochloride solution that serves to inactivate RNases in the sample and promote binding of nucleic acids to the magnetic beads for extraction.

30 vials Hydration Solution (Blue lid) Each single use vial contains $1 . 5 \mathrm { m L }$ of Hydration Solution (molecular grade water). The Hydration Solution is used to rehydrate the freeze-dried reagents contained in the FilmArray RP pouch.   
30 individually packaged Sample Loading Syringes with attached cannula (denoted with a red cap) Used for adding the patient sample/ buffer mixture to the pouch.   
30 individually packaged Pouch Hydration Syringes with attached cannula (denoted with a blue cap) Used for adding Hydration Solution to the pouch prior to testing.

![](images/ee369b7a49a25115521ed4ba04c002800d5c0eab794dee334986fc4f7208cad8.jpg)

The FilmArray RP pouch is a closed system disposable that houses all the chemistry required to isolate, amplify and detect nucleic acid from an NPS specimen. The pouch is divided into discrete segments (blisters) where specific steps are carried out. Reagents are arranged in the pouch such that sample is moved through the disposable sequentially as each biochemical event is performed.

# The FilmArray Instrument

The FilmArray instrument interacts with the FilmArray pouch mechanically, thermally, and optically to drive a multi-step chemical process designed to detect specific nucleic acid targets using multiplex nested PCR followed by DNA melting analysis. The instrument follows a protocol that is defined by a set of codes that are downloaded from the host computer at runtime. The instrument protocol defines the specific timing and sequence parameters as the instrument performs the following key functions:

• Perform cell disruption using the bead beater Extract nucleic acid from the disrupted sample Perform stage 1 PCR thermocycling of multiplexed PCR reaction Perform stage 2 PCR thermocycling of the array   
• Execute a DNA melt and detect fluorescent signals generated   
• Monitor system performance in real time and communicates out of specification

conditions to the user via the software

# Materials Required But Not Provided

• FilmArray Instrument FilmArray Pouch Loading Station (provided with FilmArray Instrument)

# J. Substantial Equivalence Information:

1. Predicate device name(s): Luminex® xTAG™ Respiratory Viral Panel (RVP).

2. Predicate K number(s):

K063765, K081483 K091677

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel System</td><td rowspan=1 colspan=1>Luminex® xTAGTM RVP</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1,Influenza A subtype H3, Influenza B,Respiratory Syncytial Virus, HumanMetapneumovirus, Adenovirus,Parainfluenza virus 3, and Rhinovirus.</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>RNA/DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>FilmArray Respiratory Panel Test System</td><td rowspan=1 colspan=1>Luminex® xTAGTM RVP</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Can distinguish Influenza A subtype 2009H1 from Influenza A subtype H1. Alsodetects Coronavirus NL63, CoronavirusHKU1, and Enterovirus.</td><td rowspan=1 colspan=1>Can distinguish Respiratory Syncytial VirusType A from Respiratory Syncytial VirusType B. Detects Parainfluenza virus 1 andParainfluenza virus 2.</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Nested multiplex RT-PCR followed by highresolution melting analysis to confirmidentity of amplified product.</td><td rowspan=1 colspan=1>Multiplex RT-PCR and multiplex TSPEfollowed by Fluorescence-activated sorting oflabeled beads coupled to streptavidin-conjugated biotinylated products</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>FilmArray Instrument</td><td rowspan=1 colspan=1>PCR Thermocycler Luminex® 100 IS or 200system</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>Less than 1 hour</td><td rowspan=1 colspan=1>Approximately 8 hours</td></tr><tr><td rowspan=1 colspan=1>TestInterpretation</td><td rowspan=1 colspan=1>Automated test interpretation and reportgeneration. User cannot access raw data.</td><td rowspan=1 colspan=1>Semi-automated test interpretation. Usermust review all &quot;no call&quot; results to determinecause and retesting strategy.</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>Sample Processing is automated in theFilmArray instrument.</td><td rowspan=1 colspan=1>Up front sample processing is required toextract nucleic acid.</td></tr><tr><td rowspan=1 colspan=1>ReagentStorage</td><td rowspan=1 colspan=1>Reagents are stored at room temperature.</td><td rowspan=1 colspan=1>Reagents stored at 4°C and -20°C.</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>Internal control added to each sample.External control processed with each batch ofsamples.</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute Approved Guideline, EP12-A (August 2002) Molecular Diagnostic Methods for Infectious Diseases, Clinical and Laboratory Standards Institute Approved Guideline, MM3-A (December 1995)

Interference Testing in Clinical Chemistry, Clinical and Laboratory Standards   
Institute Approved Guideline EP7-A (December 2002)   
Guidance for the Content of Premarket Submissions for Software Contained in   
Medical Devices, FDA Guidance Document (May 11, 2005)   
Establishing Performance Characteristics of In Vitro Diagnostic Devices for   
Detection or Detection and Differentiation of Influenza Viruses (February 15, 2008)   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Testing for Detection and Differentiation of Influenza A Virus   
Subtypes Using Multiplex Assays (October 9, 2009)   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9,   
2009)   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid   
Assays (October 9, 2009)   
Guidance for Industry and FDA Staff - Class II Special Controls Guidance   
Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005)   
Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf   
Software Use in Medical Devices (September 9, 1999)

# L. Test Principle:

The FilmArray RP System is multiplex nucleic acid test system composed of the FilmArray instrument, the FilmArray software (preinstalled on a laptop computer) and the FilmArray RP pouch. The FilmArray RP pouch contains freeze-dried reagents to perform nucleic acid purification, reverse transcription, and nested, multiplex PCR with DNA melt analysis. The Respiratory Panel (RP) pouch identifies the following 11 respiratory pathogens: Adenovirus, Coronavirus HKU1, Coronavirus NL63, Human Metapneumovirus, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype $2 0 0 9 \mathrm { H } 1$ , Influenza B, Parainfluenza virus 3, Human Rhinovirus/Enterovirus and Respiratory Syncytial Virus.

A test is initiated by loading Hydration Solution and an unprocessed patient nasopharyngeal swab (NPS) specimen (i.e. specimen mixed with Sample Buffer) into the FilmArray RP pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format. The addition of Hydration Solution and Sample/Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification and initiating the run. The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the reverse transcription reactions, the PCR reactions, and the melting curve analysis. Nucleic acid extraction occurs within the 7

FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green®Plus, Idaho Technology Inc.). This second master mix solution, is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The second stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the second stage PCR, the array is interrogated by melting curve analysis for the detection of signature amplicons denoting the presence of specific viral or bacterial targets A digital camera placed in front of the second stage PCR captures fluorescent images of the PCR reactions in real time. The FilmArray software automatically interprets the results of each DNA melting curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Sample purification

Nucleic acid purification occurs in the first three blisters of the pouch using magnetic bead technology. Prior to loading the sample into the pouch, nucleases are inactivated by mixing the sample with a denaturing buffer. The sample/buffer mixture is then loaded into the pouch where it enters the large trapezoidal blister. The sample is lysed by agitation with zirconia beads (bead beating) and the liberated nucleic acid is captured by adsorption to suitably coated magnetic beads. The beads are then moved to a new blister in the pouch were they are collected using a retractable magnet positioned adjacent to the blister. The beads are first washed to remove proteins and cell debris. After the wash steps are completed, an elution buffer is added to the blister and the nucleic acid is released from the beads. The beads are then moved back to the trapezoidal blister and the nucleic acid is moved to the $1 ^ { \mathrm { s t } }$ stage PCR blister.

# Reverse transcription and 1st stage multiplex PCR

In the $1 ^ { \mathrm { s t } }$ stage PCR blister, the purified nucleic acid rehydrates a freeze-dried reagent pellet containing all outer primer sets. Since many pathogens are RNA viruses, a reverse transcription (RT) step is needed to convert the viral RNA into cDNA that can be amplified by PCR. Both the RT and the first stage of the nested PCR reaction are performed using the same outer primers and master mix. A PCR master mix, containing all components needed for PCR and RT, is moved from the fitment to the appropriate PCR1 blister in the pouch. A ‘hotstart’ PCR and RT is performed by preheating the blisters containing the purified sample and the blister containing the PCR master mix. Once the appropriate temperature is reached, the contents of the two blisters are mixed and the RT step and thermocycling is initiated. Thermocycling consists of an RT step $5 4 ^ { \circ } \mathrm { C }$ hold for 3 minutes) followed by 26 PCR cycles (denaturation at $9 6 \mathrm { { } ^ { \circ } C }$ for 4 seconds, annealing/extension $6 0 \mathrm { { } ^ { \circ } C }$ for 19 seconds).

# $2 ^ { \mathrm { n d } }$ Stage PCR and DNA melt analysis

Following completion of the RT and PCR1 steps, a second single product nested PCR (PCR2) is carried out. PCR2 occurs in a 102-well array that has specific primers in each well that nest within a particular 1st stage amplicon. To accomplish this, fresh PCR 8

master mix (without specific primers) is used to dilute the $1 ^ { \mathrm { s t } }$ stage reaction product, $\mathord { \sim } 1 0 0$ fold final dilution) and the material is then flooded over the array where it rehydrates the dried primers and provides a complete PCR reaction. Thermocyling is initiated (30 cycles with a $9 4 \mathrm { { } ^ { \circ } C }$ denaturation for 0 seconds and $6 3 ^ { \circ } \mathrm { C }$ annealing/extension for 19 seconds). The product of the $2 ^ { \mathrm { n d } }$ stage PCR is visualized with fluorescent LC Green® Plus (Idaho Technology Inc.) dye. At the conclusion of PCR2 the array is interrogated by melting curve analysis for the detection of signature amplicons denoting the presence of specific targets.

# FilmArray Software Interpretation

The melt detector module of the FilmArray Analysis software assigns positive and negative results to melt curves generated in each well of the PCR2 array based on properties of the curve such as signal-to-noise, area under the curve, and melting temperature (Tm). Each assay is present in triplicate wells distributed across the array. If at least 2 out of 3 wells have positive melt curves with similar profiles, a positive result is assigned to that assay. If less than 2 of the replicate wells have positive melt curves in the Tm window, or if the curves are dissimilar, a negative result is assigned to the assay. A meta-analysis determines the final result (Detected (positive), Not Detected (negative), or Equivocal) for each RP organism. For organisms represented by a single PCR2 assay, the assay result and organism result are the same. Results for Influenza A, and Human Rhinovirus/Enterovirus are determined by meta-analysis of multiple assay results. Meta-analysis for Human Rhinovirus/Enterovirus incorporates the results of 6 different assays (Entero 1, Entero 2, HRV1, HRV2, HRV3, and HRV4). The HRV assays recognize both Rhinoviruses and Enteroviruses, while the Entero assays target an insertion sequence in the $5 ^ { \circ }$ UTR that is not found in Rhinoviruses. Any positive assay result will produce a combined Human Rhinovirus/Enterovirus organism result. Organism results for Influenza A are determined by meta-analysis of multiple paninfluenza A and subtype targeted assay results. The FilmArray RP System is designed to both detect Influenza A and to differentiate commonly occurring hemagglutinin subtypes. To accomplish this, the FilmArray RP uses two Influenza A assays, (FluApan1and FluA-pan2) and three subtyping assays directed at the hemagglutinin gene (H1- pan, H1-2009 and H3). The H1-pan assay is designed to detect Influenza A/H1, including the Influenza A/2009 H1 variant. The H1-2009 assay specifically detects the pandemic 2009 H1N1 variant. Meta-analysis for Influenza A is outlined in the following table:

<table><tr><td rowspan=1 colspan=1>Assay Final Result</td><td rowspan=1 colspan=1>FluA Assays(n=2)</td><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>2009 H1</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>RequiredFollow-up</td></tr><tr><td rowspan=1 colspan=1>Influenza A Not Detected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 and Influenza AH3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=2 colspan=1>MultipleInfectionsare possiblebut rare,retest toconfirmresult</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009 andInfluenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A (no subtypedetected)</td><td rowspan=1 colspan=1>2 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>See below*</td></tr><tr><td rowspan=1 colspan=1>Influenza A equivocal</td><td rowspan=1 colspan=1>1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>Retest</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3 equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009 equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

\*If both of the influenza A assays are positive, but none of the hemagglutinin subtyping assays are positive, then the interpretation is ‘Influenza A (no subtype detected)’. This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the presence of a novel Influenza A strain. In both cases, the sample in question should be retested. If the retest provides a different result, test the sample a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the RP pouches should be verified by testing with appropriate external control materials (known positive samples for Influenza A H1, Influenza A H3 and Influenza A $2 0 0 9 \mathrm { H } 1$ ) and a negative control reaction should be run to test for PCR-product contamination. If the FilmArray RP accurately identifies the external controls, contact local or state public health authorities for confirmatory testing.

In general, Influenza A is determined to be detected if at least one of the two FluA pan assays is positive and the corresponding subtyping assay is also positive. If neither of the pan assays is positive, but a subtyping assay is positive, then the result is considered equivocal for that specific subtype and the sample should be retested. If one of the pan assays is positive and no subtyping assay is positive, the result is Influenza A Equivocal and the specimen should be retested. If both FluA pan assays are positive and no subtyping assay is positive the result is reported as ‘Influenza A (no subtype detected). The product literature instructs the operator to retest the specimen. If the ‘no subtype detected’ result is repeated, the product literature directs the operator concerning appropriate follow-up testing and, if needed, notification of public health authorities.

If either the RNA Process Control or the PCR2 Control fails (negative assay result), all organisms are assigned an ‘Invalid’ test result. Invalid results are also assigned to all assays if the FilmArray instrument protocol or analysis does not complete, an Instrument Error is reported, or a Software Error is reported. All test specimens that receive ‘Invalid’ results should be retested.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A precision/reproducibility study panel of twelve specimens (refer to the table below) was created by spiking simulated NPS sample matrix with known quantities of live RP organisms. The specimens in the panel were designed so that all $1 1 \ \mathrm { R P }$ organisms would be tested at three concentrations (3 x LoD, 1x LoD, and $0 . 1 \mathrm { { x } }$ LoD). Several analytes were spiked into each specimen to simulate the possibility of co-infections with multiple pathogens at various levels. Human Rhinovirus (HRV) and Respiratory Syncytial Virus (RSV) were included in more specimens than the other analytes because they were the most common panel members detected in co-infected clinical NPS specimens during the 2009-2010 prospective clinical evaluation performed by ITI.

Panel of Specimens Evaluated for Precision/Reproducibility Testing with the FilmArray RP System   

<table><tr><td rowspan=1 colspan=1>Sample #</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Humanmetapneumovirus</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample #</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample #</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus</td><td rowspan=1 colspan=1>0.1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample #</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>1 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td></tr><tr><td rowspan=1 colspan=1>Human rhinovirus</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>3 x LoD</td><td rowspan=1 colspan=1>0.1 x LoD</td></tr></table>

Following spiking, the composition of each specimen pool was verified by testing with the FilmArray RP system. Pools that did not perform as expected were remade. Pools that performed as expected were aliquoted into several individual use vials, and frozen until the scheduled day of testing. Specimen aliquots were hand-delivered (Site A) or shipped (Site B) to external testing sites on dry ice and stored frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ at each testing site until the day of testing. All specimen aliquots were tested on the same day they were removed from the freezer. The test operators at test sites were blinded to the organisms and test levels in each specimen.

Reproducibility testing was performed at three (3) test sites (Site A, Site B, and Site C) by multiple operators (4-5 per site) using multiple (26) lots of reagents and multiple (20) FilmArray instruments. Each specimen was tested 4 times per day (2 operators

and 2 instruments per specimen) for 5 days at each site, giving a total of 20 test results from each test site (a minimum of 60 total test results for each analyte at each concentration).

Negative analyte results were derived from any specimen that was not spiked with the analyte of interest. Every analyte was absent from at least six specimens, providing a minimum of 360 expected negative results for each analyte all sites combined.

Between the three test sites, 20 different instruments were utilized. A total of 1,168 runs were initiated over the course of the study and 64 $( 5 . 5 \% )$ did not go to completion. The incomplete runs consisted of Instrument Errors (46), Software Errors (10), and 8 runs that were either aborted by the operator or failed to complete. The majority of the instrument and software errors observed in this study (41/64, $6 4 \%$ ) were caused by a communication timing error between the FilmArray software and the driver of the instrument camera. This error mode was fully investigated and software engineers have resolved the underlying cause of the errors with an update to the FilmArray software. All samples for which the run did not complete were retested to obtain valid results.

At each test site between 4 and 5 operators participated in the study, with at least 2 different operators testing samples each day. The number of errors experienced by each test site was similar, though the percentage was lower for Site C due to a higher number of runs performed. The distribution of errors between operators at test sites A and B was somewhat variable, but there did not appear to be a strong correlation between system errors and particular operators.

Between the 3 test sites, twenty-six (26) different lots of FilmArray RP pouches were used. A total of 1104 runs were completed and 1056 of those 1104 runs $( 9 5 . 6 \% )$ passed all controls (positive RNA Process Control and PCR2 Control assay results). Of the 48 failed runs, 8 were negative for both control assays. The remaining 40 failed runs were negative for the RNA Process Control only. (Note: Due to the RNA Process Control failure rate being higher than expected in the later months of the clinical evaluation, the issue was investigated and it was determined that the existing quality control process for determining the level of organism used as the control material (yeast) was not adequately controlling the level of material in manufactured pouches. The pouches affected were those manufactured after 3/8/2010. The manufacturing and quality control process for determining the level of yeast has been improved to ensure a consistent level of yeast in each pouch lot.) All samples for which the pouch controls failed were retested to obtain valid results.

A summary of reproducibility study results for each analyte and a summary of Tm reproducibility results for each assay are provided below:

Reproducibility Summary Results for Adenovirus   

<table><tr><td rowspan=1 colspan=3>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=3>Adenovirus Species C(Serotype 1)Moderate Positive(3 x LoD)(900 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=3>Adenovirus Species C(Serotype 1)Low Positive(1 x LoD)(300 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>b</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=3>Adenovirus Species C(Serotype 1)High Negative(0.1 x LoD)(30 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>44/60</td><td rowspan=1 colspan=1>16/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>73.3%</td><td rowspan=1 colspan=1>60.3% -83.9%</td></tr><tr><td rowspan=4 colspan=3>Adenovirus Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Adenovirus Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism(Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>Adeno</td><td rowspan=4 colspan=1>Adenovirus SpeciesC(Serotype 1)Moderate Positive(3 x LoD)(900 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.74</td><td rowspan=1 colspan=1>83.71</td><td rowspan=1 colspan=1>0.2047</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(83.33,84.26)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.23</td><td rowspan=1 colspan=1>84.19</td><td rowspan=1 colspan=1>0.1727</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>(83.95,84.60)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>83.81</td><td rowspan=1 colspan=1>0.2362</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(83.03,84.13)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.93</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>0.3030</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(83.03,84.60)</td></tr><tr><td rowspan=4 colspan=1>Adenovirus SpeciesC(Serotype 1)Low Positive(1 x LoD)(300 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.42</td><td rowspan=1 colspan=1>83.43</td><td rowspan=1 colspan=1>0.2363</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(83.01,83.98)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>83.87</td><td rowspan=1 colspan=1>0.2180</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(83.54,84.30)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.62</td><td rowspan=1 colspan=1>83.56</td><td rowspan=1 colspan=1>0.3190</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(83.05,84.64)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.64</td><td rowspan=1 colspan=1>83.56</td><td rowspan=1 colspan=1>0.3303</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>(83.01,84.64)</td></tr><tr><td rowspan=4 colspan=1>Adenovirus SpeciesC(Serotype 1)High Negative(0.1 x LoD)(30 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.33</td><td rowspan=1 colspan=1>83.25</td><td rowspan=1 colspan=1>0.2150</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(83.02,83.76)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.71</td><td rowspan=1 colspan=1>83.66</td><td rowspan=1 colspan=1>0.2541</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(82.93,84.29)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.26</td><td rowspan=1 colspan=1>83.24</td><td rowspan=1 colspan=1>0.2605</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(82.61,83.86)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.43</td><td rowspan=1 colspan=1>83.43</td><td rowspan=1 colspan=1>0.3185</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(82.61,84.29)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>83.63</td><td rowspan=1 colspan=1>83.65</td><td rowspan=1 colspan=1>0.3742</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>(82.61,84.64)</td></tr></table>

Reproducibility Summary Results for Coronavirus HKU1   

<table><tr><td rowspan=1 colspan=5>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=5>Coronavirus HKU1 (Type B -Clinical Specimen 6123)Moderate Positive(3 x LoD)(5.7E+06 RNA copies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=5 colspan=5>Coronavirus HKU1 (Type B Clinical Specimen 6123)Low Positive(1 x LoD)(1.9E+06 RNA copies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=4></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2>Low Positive</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=5>Coronavirus HKU1 (Type B -Clinical Specimen 6123)High Negative(0.1 x LoD)(1.9E+05 RNA copies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>51/60</td><td rowspan=1 colspan=1>9/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>73.4% -92.9%</td></tr><tr><td rowspan=4 colspan=5>Coronavirus HKU1 Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Coronavirus HKU1 Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>CoV-HKU1</td><td rowspan=4 colspan=1>Coronavirus HKU1(Type B - ClinicalSpecimen 6123)Moderate Positive(3 x LoD)(5.7E+06 RNAcopies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>75.61</td><td rowspan=1 colspan=1>75.58</td><td rowspan=1 colspan=1>0.1734</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>(75.37,76.02)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>76.02</td><td rowspan=1 colspan=1>76.01</td><td rowspan=1 colspan=1>0.1945</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(75.58,76.33)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.49</td><td rowspan=1 colspan=1>75.51</td><td rowspan=1 colspan=1>0.2409</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>(74.96,75.90)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>75.69</td><td rowspan=1 colspan=1>75.68</td><td rowspan=1 colspan=1>0.3135</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>(74.96,76.33)</td></tr><tr><td rowspan=4 colspan=1>Coronavirus HKU1(Type B - ClinicalSpecimen 6123)Low Positive(1 x LoD)(1.9E+06 RNAcopies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>75.47</td><td rowspan=1 colspan=1>75.48</td><td rowspan=1 colspan=1>0.1664</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(74.96,75.79)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>75.89</td><td rowspan=1 colspan=1>75.90</td><td rowspan=1 colspan=1>0.1496</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>(75.59,76.12)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.35</td><td rowspan=1 colspan=1>75.36</td><td rowspan=1 colspan=1>0.2223</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(74.83,75.81)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>75.55</td><td rowspan=1 colspan=1>75.57</td><td rowspan=1 colspan=1>0.3038</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(74.83,76.12)</td></tr><tr><td rowspan=4 colspan=1>Coronavirus HKU1(Type B - ClinicalSpecimen 6123)High Negative(0.1 x LoD)(1.9E+05 RNAcopies/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>75.51</td><td rowspan=1 colspan=1>75.49</td><td rowspan=1 colspan=1>0.2083</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(75.06,75.90)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>75.85</td><td rowspan=1 colspan=1>75.89</td><td rowspan=1 colspan=1>0.1631</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(75.26,76.23)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>75.33</td><td rowspan=1 colspan=1>75.33</td><td rowspan=1 colspan=1>0.1652</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(74.98,75.59)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>75.55</td><td rowspan=1 colspan=1>75.57</td><td rowspan=1 colspan=1>0.2894</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(74.98,76.23)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>75.59</td><td rowspan=1 colspan=1>75.58</td><td rowspan=1 colspan=1>0.3075</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>(74.83,76.33)</td></tr></table>

Reproducibility Summary Results for Coronavirus NL63   

<table><tr><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63 (BEI Resources</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>NR-470)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63 (BEI Resources</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>NR-470)Low Positive(1 x LoD)(5 TCID50/mL)</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63 (BEI Resources</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>NR-470)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>37/60</td><td rowspan=1 colspan=1>23/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>61.7%</td><td rowspan=1 colspan=1>48.2% -73.9%</td></tr><tr><td rowspan=4 colspan=1>Coronavirus NL63 Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Coronavirus NL63 Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>CoV-NL63</td><td rowspan=4 colspan=1>Coronavirus NL63 (BEIResources NR-470)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.21</td><td rowspan=1 colspan=1>80.21</td><td rowspan=1 colspan=1>0.2454</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(79.64,80.90)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.55</td><td rowspan=1 colspan=1>80.54</td><td rowspan=1 colspan=1>0.2681</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>(79.99,81.03)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.04</td><td rowspan=1 colspan=1>80.07</td><td rowspan=1 colspan=1>0.2005</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>(79.67,80.62</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.29</td><td rowspan=1 colspan=1>80.27</td><td rowspan=1 colspan=1>0.3220</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(79.64,81.03)</td></tr><tr><td rowspan=4 colspan=1>Coronavirus NL63 (BEIResources NR-470)Low Positive(1 x LoD)(5 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.08</td><td rowspan=1 colspan=1>80.09</td><td rowspan=1 colspan=1>0.2009</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>(79.57,80.42)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>0.1961</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(79.98,80.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.88</td><td rowspan=1 colspan=1>79.88</td><td rowspan=1 colspan=1>0.2169</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>(79.36,80.40)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>80.11</td><td rowspan=1 colspan=1>0.3047</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(79.14,80.89)</td></tr><tr><td rowspan=4 colspan=1>Coronavirus NL63 (BEIResources NR-470)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.08</td><td rowspan=1 colspan=1>80.07</td><td rowspan=1 colspan=1>0.2724</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>(79.21,80.82</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.36</td><td rowspan=1 colspan=1>80.32</td><td rowspan=1 colspan=1>0.2408</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(79.98,80.91)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.91</td><td rowspan=1 colspan=1>79.90</td><td rowspan=1 colspan=1>0.2095</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(79.24,80.30)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>80.11</td><td rowspan=1 colspan=1>80.09</td><td rowspan=1 colspan=1>0.3141</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>(79.21,80.91</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>80.16</td><td rowspan=1 colspan=1>80.15</td><td rowspan=1 colspan=1>0.3211</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(79.14,81.03)</td></tr></table>

Reproducibility Summary Results for Human metapneumovirus   

<table><tr><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Human metapneumovirus(Type A1 HMPV-16)Moderate Positive(3 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Human metapneumovirus(Type A1 HMPV-16)Low Positive(1 x LoD)(2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Human metapneumovirus(Type A1 HMPV-16)High Negative(0.1 x LoD)(0.2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>48/60</td><td rowspan=1 colspan=1>12/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>67.7% -89.2%</td></tr><tr><td rowspan=4 colspan=1>Human metapneumovirusNegative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Human metapneumovirus Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism(Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>hMPV</td><td rowspan=4 colspan=1>Humanmetapneumovirus(Type A1 HMPV-16)Moderate Positive(3 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>77.61</td><td rowspan=1 colspan=1>77.67</td><td rowspan=1 colspan=1>0.1818</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>(77.06,77.90)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>78.07</td><td rowspan=1 colspan=1>78.00</td><td rowspan=1 colspan=1>0.2169</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(77.67,78.61)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>0.1669</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>(77.45,78.11)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>77.82</td><td rowspan=1 colspan=1>77.79</td><td rowspan=1 colspan=1>0.2809</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(77.06,78.61)</td></tr><tr><td rowspan=4 colspan=1>Humanmetapneumovirus(Type A1 HMPV-16)Low Positive(1 x LoD)(2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>77.39</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>0.1613</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>(77.04,77.79)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>77.88</td><td rowspan=1 colspan=1>77.89</td><td rowspan=1 colspan=1>0.1685</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(77.37,78.11)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>77.60</td><td rowspan=1 colspan=1>77.58</td><td rowspan=1 colspan=1>0.1841</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(77.16,77.99)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>77.62</td><td rowspan=1 colspan=1>77.58</td><td rowspan=1 colspan=1>0.2731</td><td rowspan=1 colspan=1>035</td><td rowspan=1 colspan=1>(77.04,78.11)</td></tr><tr><td rowspan=4 colspan=1>Humanmetapneumovirus(Type A1 HMPV-16)High Negative(0.1 x LoD)(0.2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>77.34</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>0.1510</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>(77.05,77.59)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>77.76</td><td rowspan=1 colspan=1>77.78</td><td rowspan=1 colspan=1>0.1732</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(77.06,78.11)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>0.2275</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(76.74,77.79)</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>77.50</td><td rowspan=1 colspan=1>77.48</td><td rowspan=1 colspan=1>0.2714</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(76.74,78.11)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>77.63</td><td rowspan=1 colspan=1>77.61</td><td rowspan=1 colspan=1>0.3028</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>(76.74,78.61)</td></tr></table>

Reproducibility Summary Results for Enterovirus   

<table><tr><td rowspan=1 colspan=2>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=2>Enterovirus (Echovirus 6)(Species B)Moderate Positive(3 x LoD)(90,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=2>Enterovirus (Echovirus 6)(Species B)Low Positive(1 x LoD)(30,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=6 colspan=2>Enterovirus (Echovirus 6)(Species B)High Negative(0.1 x LoD)(3,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>b</td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=2>EnterovirusNegative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/180*</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% -100.0%</td></tr></table>

\*The number of expected negative results is lower than for other analytes because only 3 specimens (Specimens #1, 2 and 3) were not spiked with either Enterovirus or Human rhinovirus, which both will produce positive Human Rhinovirus / Enterovirus results.

Reproducibility Summary Tm Results for Enterovirus Assays   

<table><tr><td rowspan=1 colspan=1>Assay*</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>Entero 1</td><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Moderate Positive(3 x LoD)(90,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>87.12</td><td rowspan=1 colspan=1>87.10</td><td rowspan=1 colspan=1>0.2126</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(86.79,87.73)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.51</td><td rowspan=1 colspan=1>87.47</td><td rowspan=1 colspan=1>0.2652</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(86.99,88.05)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.98</td><td rowspan=1 colspan=1>87.00</td><td rowspan=1 colspan=1>0.2827</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>(86.37,87.64)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>87.18</td><td rowspan=1 colspan=1>87.13</td><td rowspan=1 colspan=1>0.3429</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>(86.37,88.05)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Low Positive(1 x LoD)(30,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>87.00</td><td rowspan=1 colspan=1>87.01</td><td rowspan=1 colspan=1>0.2893</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>(86.17,87.64)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.36</td><td rowspan=1 colspan=1>87.34</td><td rowspan=1 colspan=1>0.2558</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(86.80,87.85)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.81</td><td rowspan=1 colspan=1>86.86</td><td rowspan=1 colspan=1>0.3054</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(86.15,87.41)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>87.05</td><td rowspan=1 colspan=1>87.05</td><td rowspan=1 colspan=1>0.3640</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>(86.15,87.85)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)High Negative(0.1 x LoD)(3,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.89</td><td rowspan=1 colspan=1>86.90</td><td rowspan=1 colspan=1>0.2540</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(86.06,87.40)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.34</td><td rowspan=1 colspan=1>87.33</td><td rowspan=1 colspan=1>0.2717</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(86.67,87.86)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.67</td><td rowspan=1 colspan=1>86.66</td><td rowspan=1 colspan=1>0.2528</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(85.97,87.30)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>86.96</td><td rowspan=1 colspan=1>86.92</td><td rowspan=1 colspan=1>0.3822</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>(85.97,87.86)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>87.06</td><td rowspan=1 colspan=1>87.04</td><td rowspan=1 colspan=1>0.3743</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>(85.97,88.05)</td></tr></table>

<table><tr><td rowspan=13 colspan=1>Entero 2</td><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Moderate Positive(3 x LoD)(90,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>87.09</td><td rowspan=1 colspan=1>87.03</td><td rowspan=1 colspan=1>0.2406</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(86.68,87.73</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.47</td><td rowspan=1 colspan=1>87.43</td><td rowspan=1 colspan=1>0.2583</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(86.82,88.00)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.93</td><td rowspan=1 colspan=1>86.96</td><td rowspan=1 colspan=1>0.3143</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(86.16,87.64</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>87.14</td><td rowspan=1 colspan=1>87.13</td><td rowspan=1 colspan=1>0.3589</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>(86.16,88.00)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Low Positive(1 x LoD)(30,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.98</td><td rowspan=1 colspan=1>87.00</td><td rowspan=1 colspan=1>0.2618</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(86.28,87.53</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.34</td><td rowspan=1 colspan=1>87.34</td><td rowspan=1 colspan=1>0.2446</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(86.89,87.82)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.77</td><td rowspan=1 colspan=1>86.82</td><td rowspan=1 colspan=1>0.3075</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(86.05,87.52)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>87.02</td><td rowspan=1 colspan=1>87.01</td><td rowspan=1 colspan=1>0.3565</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>(86.05,87.82)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus) (Species B)High Negative(0.1 x LoD)(3,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>86.86</td><td rowspan=1 colspan=1>86.89</td><td rowspan=1 colspan=1>0.2541</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(86.17,87.54)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>87.26</td><td rowspan=1 colspan=1>87.29</td><td rowspan=1 colspan=1>0.3041</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(86.59,87.94)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>86.65</td><td rowspan=1 colspan=1>86.67</td><td rowspan=1 colspan=1>0.2341</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>(86.27,87.20)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>86.92</td><td rowspan=1 colspan=1>86.90</td><td rowspan=1 colspan=1>0.3671</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>(86.17,87.94)</td></tr><tr><td rowspan=1 colspan=1>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>, all sites)</td><td rowspan=1 colspan=1>87.02</td><td rowspan=1 colspan=1>87.00</td><td rowspan=1 colspan=1>0.3708</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>(86.05, 88)</td></tr><tr><td rowspan=13 colspan=1>HRV 4</td><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Moderate Positive(3 x LoD)(90,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>85.70</td><td rowspan=1 colspan=1>85.74</td><td rowspan=1 colspan=1>0.2660</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(85.21,86.18</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>86.19</td><td rowspan=1 colspan=1>86.19</td><td rowspan=1 colspan=1>0.2576</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>(85.35,86.77</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.59</td><td rowspan=1 colspan=1>85.57</td><td rowspan=1 colspan=1>0.2891</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>(84.91,86.19</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>85.81</td><td rowspan=1 colspan=1>85.84</td><td rowspan=1 colspan=1>0.3788</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>(84.91,86.77</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)Low Positive(1 x LoD)(30,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>85.45</td><td rowspan=1 colspan=1>85.44</td><td rowspan=1 colspan=1>0.2249</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(84.81,86.06</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.87</td><td rowspan=1 colspan=1>85.85</td><td rowspan=1 colspan=1>0.2053</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(85.44,86.36)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.32</td><td rowspan=1 colspan=1>85.24</td><td rowspan=1 colspan=1>0.3399</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(84.69,86.26)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>85.54</td><td rowspan=1 colspan=1>85.54</td><td rowspan=1 colspan=1>0.3403</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(84.69,86.36)</td></tr><tr><td rowspan=4 colspan=1>Enterovirus (Echovirus6) (Species B)High Negative(0.1 x LoD)(3,000 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>85.37</td><td rowspan=1 colspan=1>85.35</td><td rowspan=1 colspan=1>0.2224</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(84.80,86.04)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>85.82</td><td rowspan=1 colspan=1>85.85</td><td rowspan=1 colspan=1>0.1934</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>(85.43,86.23</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>85.22</td><td rowspan=1 colspan=1>85.24</td><td rowspan=1 colspan=1>0.1843</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(84.82,85.58</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>85.46</td><td rowspan=1 colspan=1>85.43</td><td rowspan=1 colspan=1>0.3311</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>(84.8,86.23</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>85.60</td><td rowspan=1 colspan=1>85.57</td><td rowspan=1 colspan=1>0.3809</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>(84.69,86.77</td></tr></table>

$^ { * } \mathrm { H R V 1 }$ , HRV2 and HRV3 assays associated with samples containing Enterovirus were not considered in the overall evaluation of the data due to a very small number of observations for the data set. HRV4 is the only HRV assay that is consistently positive when testing at low concentrations of this Enterovirus (Echovirus 6).

Reproducibility Summary Results for Human rhinovirus   

<table><tr><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirus A1 ModeratePositive(3 x LoD)(3 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0%-100.0%</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirus A1 LowPositive(1 x LoD)(1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirus A1 HighNegative(0.1 x LoD)(0.1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>32/40</td><td rowspan=1 colspan=1>8/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>112/120</td><td rowspan=1 colspan=1>8/120</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>87.3% -93.1%</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirusNegative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/180*</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0%-100.0%</td></tr></table>

The number of expected negative results is lower than for other analytes because only 3 specimens (Specimens #1, 2 and 3) were not piked with either Enterovirus or Human rhinovirus, which both will produce positive Human Rhinovirus / Enterovirus results.

Reproducibility Summary Tm Results for Human rhinovirus Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="13">HRV 1</td><td colspan="1" rowspan="4">Human rhinovirus A1Moderate Positive(3 x LoD)(3 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.79</td><td colspan="1" rowspan="1">83.73</td><td colspan="1" rowspan="1">0.3700</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">(83.25,85.09)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.06</td><td colspan="1" rowspan="1">84.03</td><td colspan="1" rowspan="1">0.2584</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(83.43,84.56)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.68</td><td colspan="1" rowspan="1">83.69</td><td colspan="1" rowspan="1">0.1947</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(83.22,84.09)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.84</td><td colspan="1" rowspan="1">83.83</td><td colspan="1" rowspan="1">0.3224</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">(83.22,85.09)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1Low Positive(1 x LoD)(1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.71</td><td colspan="1" rowspan="1">83.74</td><td colspan="1" rowspan="1">0.2429</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(83.07,84.35)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.07</td><td colspan="1" rowspan="1">84.06</td><td colspan="1" rowspan="1">0.2865</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.44,84.66)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.93</td><td colspan="1" rowspan="1">83.87</td><td colspan="1" rowspan="1">0.2874</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.24,84.71)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.90</td><td colspan="1" rowspan="1">83.87</td><td colspan="1" rowspan="1">0.3108</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.07,84.71)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1High Negative(0.1 x LoD)(0.1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.56</td><td colspan="1" rowspan="1">83.55</td><td colspan="1" rowspan="1">0.2323</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(83.02,84.26)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.91</td><td colspan="1" rowspan="1">83.91</td><td colspan="1" rowspan="1">0.2891</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.22,84.52)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.76</td><td colspan="1" rowspan="1">83.75</td><td colspan="1" rowspan="1">0.2600</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(83.04,84.48)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.74</td><td colspan="1" rowspan="1">83.74</td><td colspan="1" rowspan="1">0.2975</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(83.02,84.52)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">83.81</td><td colspan="1" rowspan="1">83.77</td><td colspan="1" rowspan="1">0.3140</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.02,85.09)</td></tr><tr><td colspan="1" rowspan="13">HRV 2</td><td colspan="1" rowspan="4">Human rhinovirus A1Moderate Positive(3 x LoD)(3 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.33</td><td colspan="1" rowspan="1">83.28</td><td colspan="1" rowspan="1">0.4022</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">(82.55,84.67)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.65</td><td colspan="1" rowspan="1">83.62</td><td colspan="1" rowspan="1">0.2468</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(83.11,84.17</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.30</td><td colspan="1" rowspan="1">83.33</td><td colspan="1" rowspan="1">0.2341</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(82.70,83.77)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.42</td><td colspan="1" rowspan="1">83.42</td><td colspan="1" rowspan="1">0.3405</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(82.55,84.67)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1Low Positive(1 x LoD)(1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.28</td><td colspan="1" rowspan="1">83.30</td><td colspan="1" rowspan="1">0.2555</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(82.57,83.86)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.66</td><td colspan="1" rowspan="1">83.65</td><td colspan="1" rowspan="1">0.3014</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(83.02,84.37</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.52</td><td colspan="1" rowspan="1">83.53</td><td colspan="1" rowspan="1">0.3232</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(82.82,84.29)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.48</td><td colspan="1" rowspan="1">83.45</td><td colspan="1" rowspan="1">0.3317</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(82.57,84.37)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1High Negative(0.1 x LOD)(0.1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.17</td><td colspan="1" rowspan="1">83.14</td><td colspan="1" rowspan="1">0.2829</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(82.42,83.88)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.58</td><td colspan="1" rowspan="1">83.58</td><td colspan="1" rowspan="1">0.3029</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(82.80,84.31)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.37</td><td colspan="1" rowspan="1">83.39</td><td colspan="1" rowspan="1">0.2758</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(82.51,83.92)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">83.37</td><td colspan="1" rowspan="1">83.38</td><td colspan="1" rowspan="1">0.3333</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(82.42,84.31)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">83.42</td><td colspan="1" rowspan="1">83.43</td><td colspan="1" rowspan="1">0.3373</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(82.42,84.67</td></tr><tr><td colspan="1" rowspan="13">HRV 3</td><td colspan="1" rowspan="4">Human rhinovirus A1Moderate Positive(3 x LoD)(3 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.73</td><td colspan="1" rowspan="1">82.69</td><td colspan="1" rowspan="1">0.4420</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">(81.99,83.94)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.25</td><td colspan="1" rowspan="1">83.24</td><td colspan="1" rowspan="1">0.2972</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(82.49,83.88)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.89</td><td colspan="1" rowspan="1">82.96</td><td colspan="1" rowspan="1">0.3586</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">(82.10,83.88)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.96</td><td colspan="1" rowspan="1">83.02</td><td colspan="1" rowspan="1">0.4218</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">(81.99,83.94)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1Low Positive(1 x LoD)(1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.67</td><td colspan="1" rowspan="1">82.59</td><td colspan="1" rowspan="1">0.4060</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">(81.76,83.65)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.24</td><td colspan="1" rowspan="1">83.25</td><td colspan="1" rowspan="1">0.3635</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">(82.40,84.08)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">83.07</td><td colspan="1" rowspan="1">83.07</td><td colspan="1" rowspan="1">0.3443</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(82.13,83.76)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.98</td><td colspan="1" rowspan="1">83.02</td><td colspan="1" rowspan="1">0.4460</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">(81.76,84.08)</td></tr><tr><td colspan="1" rowspan="4">Human rhinovirus A1High Negative(0.1 x LoD)(0.1 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.68</td><td colspan="1" rowspan="1">82.66</td><td colspan="1" rowspan="1">0.4113</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">(81.78,83.67)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">83.21</td><td colspan="1" rowspan="1">83.24</td><td colspan="1" rowspan="1">0.3812</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">(82.18,84.37)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.97</td><td colspan="1" rowspan="1">83.01</td><td colspan="1" rowspan="1">0.3474</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">(81.85,83.64)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.96</td><td colspan="1" rowspan="1">83.00</td><td colspan="1" rowspan="1">0.4354</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">(81.78,84.37)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">82.97</td><td colspan="1" rowspan="1">83.01</td><td colspan="1" rowspan="1">0.4363</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">(81.76,84.37)</td></tr><tr><td colspan="1" rowspan="3">HRV 4</td><td colspan="1" rowspan="3">Human rhinovirus A1Moderate Positive(3 x LoD)(3 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">83.81</td><td colspan="1" rowspan="1">83.80</td><td colspan="1" rowspan="1">0.3180</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">(83.28,84.98)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.14</td><td colspan="1" rowspan="1">84.16</td><td colspan="1" rowspan="1">0.2902</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.43,84.83)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.80</td><td colspan="1" rowspan="1">82.77</td><td colspan="1" rowspan="1">0.2133</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(83.44</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>84.20)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>83.86</td><td rowspan=1 colspan=1>0.3120</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>(83.28,84.98)</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirus A1Low Positive(1 x LoD)(1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>83.78</td><td rowspan=1 colspan=1>0.2622</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(83.18,84.45)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.08</td><td rowspan=1 colspan=1>84.07</td><td rowspan=1 colspan=1>0.2616</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(83.55,84.61)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.04</td><td rowspan=1 colspan=1>84.02</td><td rowspan=1 colspan=1>0.3198</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(83.35,84.93)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>83.94</td><td rowspan=1 colspan=1>83.96</td><td rowspan=1 colspan=1>0.3083</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>(83.18,84.93)</td></tr><tr><td rowspan=4 colspan=1>Human rhinovirus A1High Negative(0.1 x LoD)(0.1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.58</td><td rowspan=1 colspan=1>83.56</td><td rowspan=1 colspan=1>0.2164</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(83.11,84.05)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>0.2602</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(83.34,84.49)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.86</td><td rowspan=1 colspan=1>83.86</td><td rowspan=1 colspan=1>0.2285</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>(83.24,84.58)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.2742</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>(83.11,84.58)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>83.84</td><td rowspan=1 colspan=1>83.84</td><td rowspan=1 colspan=1>0.3042</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(83.11,84.98)</td></tr></table>

Reproducibility Summary Results for Influenza A/H1N1   

<table><tr><td rowspan=1 colspan=3>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=3>Influenza A H1N1(A/Brisbane/59/07)Moderate Positive(3 x LoD)(600 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=3>Influenza A H1N1(A/Brisbane/59/07)Low Positive(1 x LoD)(200 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>3/60</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>83.8% -98.2%</td></tr><tr><td rowspan=4 colspan=3>Influenza A H1N1(A/Brisbane/59/07)High Negative(0.1 x LoD)(20 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>51/60</td><td rowspan=1 colspan=1>2/60</td><td rowspan=1 colspan=1>7*/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>73.4% -92.9%</td></tr><tr><td rowspan=4 colspan=3>Influenza A H1N1Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

\*Includes three (3) ‘Influenza A (no subtype detected)’ results.

Reproducibility Summary Tm Results for Influenza A/H1 Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="3">FluA-panl</td><td colspan="1" rowspan="3">Influenza A H1N1(A/Brisbane/59/07)Moderate Positive(3 x LoD)(600 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">84.69</td><td colspan="1" rowspan="1">0.2044</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(84.27,85.12)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">85.11</td><td colspan="1" rowspan="1">0.2014</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(84.78,85.56)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.64</td><td colspan="1" rowspan="1">84.60</td><td colspan="1" rowspan="1">0.2875</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.76,</td></tr><tr><td colspan="1" rowspan="11"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">85.23)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.3153</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.76,85.56)</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)Low Positive(1 x LoD)(200 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">84.50</td><td colspan="1" rowspan="1">0.2278</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(844.17),85.31</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.2351</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(84.59,85.75</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">0.2233</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(84.16,85.18)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">84.70</td><td colspan="1" rowspan="1">0.2916</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(84.16,85.75</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)High Negative(0.1  x LoD)(20 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.27</td><td colspan="1" rowspan="1">84.29</td><td colspan="1" rowspan="1">0.2226</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(83.85,84.81)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">84.72</td><td colspan="1" rowspan="1">0.1988</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(84.29,85.32)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.46</td><td colspan="1" rowspan="1">84.40</td><td colspan="1" rowspan="1">0.2928</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(83.89,85.48)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.48</td><td colspan="1" rowspan="1">84.47</td><td colspan="1" rowspan="1">0.3106</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.85,85.48)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.3373</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(83.76,85.75)</td></tr><tr><td colspan="1" rowspan="13">FluA-pan2</td><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)Moderate Positive(3 x LoD)(600 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.42</td><td colspan="1" rowspan="1">80.51</td><td colspan="1" rowspan="1">0.2015</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(79.78,80.63</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.85</td><td colspan="1" rowspan="1">80.83</td><td colspan="1" rowspan="1">0.2143</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(80.39,81.26)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.31</td><td colspan="1" rowspan="1">80.31</td><td colspan="1" rowspan="1">0.2267</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(79.89,80.72)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">80.52</td><td colspan="1" rowspan="1">80.51</td><td colspan="1" rowspan="1">0.3160</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(79.78,81.26)</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)Low Positive(1 x LoD)(200 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.36</td><td colspan="1" rowspan="1">80.31</td><td colspan="1" rowspan="1">0.1821</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(79.99,80.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.80</td><td colspan="1" rowspan="1">80.82</td><td colspan="1" rowspan="1">0.1674</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">(80.42,81.15</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.52</td><td colspan="1" rowspan="1">80.57</td><td colspan="1" rowspan="1">0.1811</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">(80.19,80.89)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">80.57</td><td colspan="1" rowspan="1">80.60</td><td colspan="1" rowspan="1">0.2544</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(79.99,81.15</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)High Negative(0.1 x LoD)(20 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.91</td><td colspan="1" rowspan="1">79.89</td><td colspan="1" rowspan="1">0.2978</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(79.15,80.41)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.49</td><td colspan="1" rowspan="1">80.51</td><td colspan="1" rowspan="1">0.2404</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.67,80.83</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.10</td><td colspan="1" rowspan="1">80.09</td><td colspan="1" rowspan="1">0.2820</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(79.56,80.73</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">80.17</td><td colspan="1" rowspan="1">80.18</td><td colspan="1" rowspan="1">0.3623</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">(79.15,80.83)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">80.37</td><td colspan="1" rowspan="1">80.40</td><td colspan="1" rowspan="1">0.3746</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">(79.15,81.26)</td></tr><tr><td colspan="1" rowspan="5">FluA-H1-pan</td><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)Moderate Positive(3 x LoD)(600 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.79</td><td colspan="1" rowspan="1">78.79</td><td colspan="1" rowspan="1">0.1998</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(788.31,79.25)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.2474</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(78.30,79.57</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.76</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.3040</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(77.79,79.25)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.91</td><td colspan="1" rowspan="1">78.93</td><td colspan="1" rowspan="1">0.3307</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">(77.67,79.57</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1(A/Brisbane/59/07)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.77</td><td colspan="1" rowspan="1">78.73</td><td colspan="1" rowspan="1">0.1994</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(78.42,79.34)</td></tr><tr><td colspan="1" rowspan="3">Low Positive(1 x LoD)(200 TCID50/mL)</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">79.23</td><td colspan="1" rowspan="1">0.2137</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(78.72,79.67</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.80</td><td colspan="1" rowspan="1">78.82</td><td colspan="1" rowspan="1">0.2009</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(78.30,79.26)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.93</td><td colspan="1" rowspan="1">78.93</td><td colspan="1" rowspan="1">0.2827</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(78.30,79.67</td></tr><tr><td colspan="1" rowspan="4">Influenza A H1N1(A/Brisbane/59/07)High Negative(0.1 x LoD)(20 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">77.65</td><td colspan="1" rowspan="1">77.59</td><td colspan="1" rowspan="1">0.2555</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(77.15,78.21)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.18</td><td colspan="1" rowspan="1">78.10</td><td colspan="1" rowspan="1">0.3554</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">(77.47,79.26)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">77.93</td><td colspan="1" rowspan="1">77.80</td><td colspan="1" rowspan="1">0.3834</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">(77.43,79.04)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">77.92</td><td colspan="1" rowspan="1">77.89</td><td colspan="1" rowspan="1">0.3989</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">(77.15,79.26)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">78.56</td><td colspan="1" rowspan="1">78.72</td><td colspan="1" rowspan="1">0.5913</td><td colspan="1" rowspan="1">0.75</td><td colspan="1" rowspan="1">(77.15,79.67</td></tr></table>

Reproducibility Summary Results for Influenza A/2009 H1N1   

<table><tr><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Influenza A 2009 H1N1(A/Swine/NY/03/2009)Moderate Positive(3 x LoD)(300 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza A 2009 H1N1(A/Swine/NY/03/2009)Low Positive(1 x LoD)(100 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0%-100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza A 2009 H1N1(A/Swine/NY/03/2009)High Negative(0.1 x LoD)(10 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza A 2009 H1N1Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Influenza A/2009 H1 Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="6">FluA-pan1</td><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Moderate Positive(3 x LoD)(300 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">84.69</td><td colspan="1" rowspan="1">0.2044</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(84.27,85.12)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">85.11</td><td colspan="1" rowspan="1">0.2014</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(84.78,85.56)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.64</td><td colspan="1" rowspan="1">84.60</td><td colspan="1" rowspan="1">0.2875</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.76,85.23)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.3153</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.76,85.56)</td></tr><tr><td colspan="1" rowspan="2">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Low Positive</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">84.50</td><td colspan="1" rowspan="1">0.2278</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(84.17,85.31)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.2351</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(84.59.</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="3">(1 x LoD)(100 TCID50/mL)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">85.75)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">0.2233</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(84.16,85.18)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">84.70</td><td colspan="1" rowspan="1">0.2916</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(84.16,85.75)</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)High Negative(0.1 x LoD)(10 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.27</td><td colspan="1" rowspan="1">84.29</td><td colspan="1" rowspan="1">0.2226</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(83.85,84.81)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">84.72</td><td colspan="1" rowspan="1">0.1988</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(84.29,85.32)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.46</td><td colspan="1" rowspan="1">84.40</td><td colspan="1" rowspan="1">0.2928</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(83.89,85.48)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">84.48</td><td colspan="1" rowspan="1">84.47</td><td colspan="1" rowspan="1">0.3106</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(83.85,85.48)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.3373</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(83.76,85.75</td></tr><tr><td colspan="1" rowspan="13">FluA-pan2</td><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Moderate Positive(3 x LD)(300 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">80.71</td><td colspan="1" rowspan="1">0.1583</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">(80..31,80.83</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.01</td><td colspan="1" rowspan="1">81.03</td><td colspan="1" rowspan="1">0.1411</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">(80.70,81.36)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.69</td><td colspan="1" rowspan="1">80.70</td><td colspan="1" rowspan="1">0.1870</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(80.19,81.02)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">80.81</td><td colspan="1" rowspan="1">80.82</td><td colspan="1" rowspan="1">0.2550</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(80.19,81.36)</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Low Positive(1 x LoD)(100 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.39</td><td colspan="1" rowspan="1">80.45</td><td colspan="1" rowspan="1">0.2316</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(79.87,80.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.91</td><td colspan="1" rowspan="1">80.93</td><td colspan="1" rowspan="1">0.2151</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(80.20,81.24)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">80.29</td><td colspan="1" rowspan="1">0.1342</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">(80.05,80.61)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">80.61</td><td colspan="1" rowspan="1">0.3526</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">(79.87,81.24</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)High Negative(0.1 xx LoD)(10 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.43</td><td colspan="1" rowspan="1">80.40</td><td colspan="1" rowspan="1">0.2134</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(80.08,81.03</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.72</td><td colspan="1" rowspan="1">80.79</td><td colspan="1" rowspan="1">0.2172</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(80. 11,81.14</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">0.2714</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(79.86,80.82</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.54</td><td colspan="1" rowspan="1">80.52</td><td colspan="1" rowspan="1">0.2763</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(79.86,81.14</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">80.63</td><td colspan="1" rowspan="1">80.65</td><td colspan="1" rowspan="1">0.3199</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(79.86,81.36)</td></tr><tr><td colspan="1" rowspan="8">FluA-H1-pan</td><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Moderate Positive(3 x LD)(300 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.87</td><td colspan="1" rowspan="1">78.93</td><td colspan="1" rowspan="1">0.3189</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(78.20,79.56)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.44</td><td colspan="1" rowspan="1">79.46</td><td colspan="1" rowspan="1">0.2756</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(78.94,79.99)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.62</td><td colspan="1" rowspan="1">78.68</td><td colspan="1" rowspan="1">0.3412</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">(77.92, ,79.44)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.97</td><td colspan="1" rowspan="1">79.02</td><td colspan="1" rowspan="1">0.4595</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">(77.92,79.99)</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Low Positive(1 x LoD)(100 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.37</td><td colspan="1" rowspan="1">78.36</td><td colspan="1" rowspan="1">0.2330</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(77.89,79.24)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.90</td><td colspan="1" rowspan="1">78.88</td><td colspan="1" rowspan="1">0.2900</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(78.21,79.76)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.16</td><td colspan="1" rowspan="1">78.11</td><td colspan="1" rowspan="1">0.1955</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(77.83,,78.69)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.47</td><td colspan="1" rowspan="1">78.41</td><td colspan="1" rowspan="1">0.4014</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">(77.7,79.76)</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)High Negative(0.1 x LoD)(10 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.34</td><td colspan="1" rowspan="1">78.31</td><td colspan="1" rowspan="1">0.2925</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(77.90,79.37</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.2949</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(77.99,79.68)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.08</td><td colspan="1" rowspan="1">78.07</td><td colspan="1" rowspan="1">0.2101</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(77.68,78.51)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.40</td><td colspan="1" rowspan="1">78.32</td><td colspan="1" rowspan="1">0.4176</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">(77.68,79.68)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">78.61</td><td colspan="1" rowspan="1">78.61</td><td colspan="1" rowspan="1">0.4950</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">(77.68,79.99)</td></tr><tr><td colspan="1" rowspan="13">FluA-H1-2009</td><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Moderate Positive(3 x LoD)(300 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.73</td><td colspan="1" rowspan="1">78.78</td><td colspan="1" rowspan="1">0.1928</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(78.31,79.14)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">79.24</td><td colspan="1" rowspan="1">0.1909</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(78.84,79.67)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.54</td><td colspan="1" rowspan="1">78.54</td><td colspan="1" rowspan="1">0.2373</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(77.89,78.92)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.81</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.3455</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">(77.89,79.67)</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)Low Positive(1 x LoD)(100 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.64</td><td colspan="1" rowspan="1">78.63</td><td colspan="1" rowspan="1">0.2041</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(78.10,79.14)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.03</td><td colspan="1" rowspan="1">79.05</td><td colspan="1" rowspan="1">0.2000</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(78.53,79.48)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.49</td><td colspan="1" rowspan="1">78.48</td><td colspan="1" rowspan="1">0.1909</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(78.12,78.82)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.71</td><td colspan="1" rowspan="1">78.70</td><td colspan="1" rowspan="1">0.3060</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(78.10,79.48)</td></tr><tr><td colspan="1" rowspan="4">Influenza A 2009 H1N1(A/Swine/NY/03/2009)High Negative(0.1 x LoD)(10 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.60</td><td colspan="1" rowspan="1">78.62</td><td colspan="1" rowspan="1">0.2187</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(77.90,79.04)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.00</td><td colspan="1" rowspan="1">79.04</td><td colspan="1" rowspan="1">0.1840</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(78.52,79.36)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.52</td><td colspan="1" rowspan="1">78.52</td><td colspan="1" rowspan="1">0.2182</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(78.10,78.93)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">78.70</td><td colspan="1" rowspan="1">78.72</td><td colspan="1" rowspan="1">0.3011</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">(77.90,79.36)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">78.74</td><td colspan="1" rowspan="1">78.73</td><td colspan="1" rowspan="1">0.3210</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(77.89,79.67)</td></tr></table>

Reproducibility Summary Results for Influenza A/H3N2   

<table><tr><td colspan="3" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">#Equivocal</td><td colspan="1" rowspan="1">% Agreementwith ExpectedResult</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="3" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.0% -100.0%</td></tr><tr><td colspan="3" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Low Positive(1 x LoD)(5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.0% -100.0%</td></tr><tr><td colspan="3" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">4/20</td><td colspan="1" rowspan="1">4/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">20.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">10/60</td><td colspan="1" rowspan="1">19/60</td><td colspan="1" rowspan="1">31/20</td><td colspan="1" rowspan="1">16.7%</td><td colspan="1" rowspan="1">8.3%-28.5%</td></tr><tr><td colspan="3" rowspan="1">Influenza A H3N2</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Negative(Un-spiked)</td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Influenza A/H3 Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="13">FluA-pan1</td><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.33</td><td colspan="1" rowspan="1">85.38</td><td colspan="1" rowspan="1">0.2243</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(84.79,85.73)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.48</td><td colspan="1" rowspan="1">85.45</td><td colspan="1" rowspan="1">0.3585</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">(84.91,86.27</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.06</td><td colspan="1" rowspan="1">85.11</td><td colspan="1" rowspan="1">0.3238</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">(84.5,85.4)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">85.36</td><td colspan="1" rowspan="1">85.34</td><td colspan="1" rowspan="1">0.3278</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">(84.5,86.27</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Low Positive(1 x LoD)(5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.95</td><td colspan="1" rowspan="1">84.88</td><td colspan="1" rowspan="1">0.3446</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(84.19,86.03</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.31</td><td colspan="1" rowspan="1">85.32</td><td colspan="1" rowspan="1">0.2611</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(84.79,86.02</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.83</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.2163</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(84.37,85.25</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">85.03</td><td colspan="1" rowspan="1">84.97</td><td colspan="1" rowspan="1">0.3466</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(84.19,86.03</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.91</td><td colspan="1" rowspan="1">84.96</td><td colspan="1" rowspan="1">0.3286</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(84.22,85.6)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.24</td><td colspan="1" rowspan="1">85.24</td><td colspan="1" rowspan="1">0.2599</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(84.78,85.77)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.86</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.2325</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(84.48,85.33)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.3176</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(84.22,85.77)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">85.05</td><td colspan="1" rowspan="1">85.02</td><td colspan="1" rowspan="1">0.3475</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(84.19,86.27</td></tr><tr><td colspan="1" rowspan="12">FluA-pan2</td><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.75</td><td colspan="1" rowspan="1">79.87</td><td colspan="1" rowspan="1">0.1995</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(79.51,79.99)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.94</td><td colspan="1" rowspan="1">79.97</td><td colspan="1" rowspan="1">0.2409</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.4,80.37</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.60</td><td colspan="1" rowspan="1">79.61</td><td colspan="1" rowspan="1">0.1911</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(79.14,79.86)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">79.81</td><td colspan="1" rowspan="1">79.78</td><td colspan="1" rowspan="1">0.2603</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(79.14,80.37</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Low Positive(1 x LoD)(5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.42</td><td colspan="1" rowspan="1">79.41</td><td colspan="1" rowspan="1">0.2462</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(79,80.3)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.69</td><td colspan="1" rowspan="1">79.67</td><td colspan="1" rowspan="1">0.2758</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(79.14,80.29)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.1608</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">(78.82,79.59)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">79.45</td><td colspan="1" rowspan="1">79.41</td><td colspan="1" rowspan="1">0.2926</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(78.82,80.3</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.22</td><td colspan="1" rowspan="1">79.21</td><td colspan="1" rowspan="1">0.2782</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(78.64,79.75)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.59</td><td colspan="1" rowspan="1">79.57</td><td colspan="1" rowspan="1">0.2298</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(79.05,79.99)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.24</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.1574</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">(78.84,79.54)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">79.35</td><td colspan="1" rowspan="1">79.35</td><td colspan="1" rowspan="1">0.2830</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(78.64,79.99)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">79.44</td><td colspan="1" rowspan="1">79.41</td><td colspan="1" rowspan="1">0.3106</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(78.64,80.37</td></tr><tr><td colspan="1" rowspan="13">FluA-H3</td><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Moderate Positive(3 x LoD)(15 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.58</td><td colspan="1" rowspan="1">82.60</td><td colspan="1" rowspan="1">0.2885</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(81.87,83.01)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.89</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">0.1544</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">(82.60,83.11)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.44</td><td colspan="1" rowspan="1">82.48</td><td colspan="1" rowspan="1">0.2740</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(81.98,82.81)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.62</td><td colspan="1" rowspan="1">82.68</td><td colspan="1" rowspan="1">0.3216</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(81.87,83.11)</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)Low Positive(1 x LoD)(5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.32</td><td colspan="1" rowspan="1">82.35</td><td colspan="1" rowspan="1">0.2409</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(81.88,82.69)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.69</td><td colspan="1" rowspan="1">82.69</td><td colspan="1" rowspan="1">0.2772</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(82.16,83.32)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.21</td><td colspan="1" rowspan="1">82.18</td><td colspan="1" rowspan="1">0.2072</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(81.76,82.73)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.39</td><td colspan="1" rowspan="1">82.39</td><td colspan="1" rowspan="1">0.3358</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(81.67,83.32)</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/Wiscosin/67/2005)High Negative(0.1 x LoD)(0.5 TCID50/mL)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.28</td><td colspan="1" rowspan="1">82.29</td><td colspan="1" rowspan="1">0.2985</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(81.71,82.91)</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.61</td><td colspan="1" rowspan="1">82.59</td><td colspan="1" rowspan="1">0.2435</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(82.17,83.05)</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.20</td><td colspan="1" rowspan="1">82.19</td><td colspan="1" rowspan="1">0.1886</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(81.87,82.57)</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">82.37</td><td colspan="1" rowspan="1">82.39</td><td colspan="1" rowspan="1">0.2985</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(81.71,83.05)</td></tr><tr><td colspan="2" rowspan="1">Overall (all concentrations, all sites)</td><td colspan="1" rowspan="1">82.42</td><td colspan="1" rowspan="1">82.40</td><td colspan="1" rowspan="1">0.3300</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(81.67,83.32)</td></tr></table>

Reproducibility Summary Results for Influenza B   

<table><tr><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)Moderate Positive(3 x LoD)(180 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)Low Positive(1 x LoD)(60 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)High Negative(0.1 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>40.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>30/60</td><td rowspan=1 colspan=1>30/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>36.8% -63.2%</td></tr><tr><td rowspan=4 colspan=1>Influenza BNegative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Influenza B Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>FluB</td><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)Moderate Positive(3 x LoD)(180 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.47</td><td rowspan=1 colspan=1>80.48</td><td rowspan=1 colspan=1>0.2081</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(79.88,80.93)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.88</td><td rowspan=1 colspan=1>80.91</td><td rowspan=1 colspan=1>0.2538</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>(80.30,81.30)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.36</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>0.2593</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>(79.78,80.80)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.56</td><td rowspan=1 colspan=1>80.53</td><td rowspan=1 colspan=1>0.3259</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(79.78,81.30)</td></tr><tr><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)Low Positive(1 x LoD)(60 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.44</td><td rowspan=1 colspan=1>80.42</td><td rowspan=1 colspan=1>0.2155</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>(80.00,80.92)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.79</td><td rowspan=1 colspan=1>80.81</td><td rowspan=1 colspan=1>0.2336</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>(80.40,81.33)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.34</td><td rowspan=1 colspan=1>80.32</td><td rowspan=1 colspan=1>0.1731</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>(79.77,80.81)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.51</td><td rowspan=1 colspan=1>80.46</td><td rowspan=1 colspan=1>0.2941</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>(79.77,81.33)</td></tr><tr><td rowspan=4 colspan=1>Influenza B (B/FL/04/06)High Negative(0.1 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.42</td><td rowspan=1 colspan=1>80.42</td><td rowspan=1 colspan=1>0.2554</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>(79.84,80.90)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.78</td><td rowspan=1 colspan=1>80.77</td><td rowspan=1 colspan=1>0.2049</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>(80.40,81.17)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.30</td><td rowspan=1 colspan=1>80.30</td><td rowspan=1 colspan=1>0.1692</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>(79.79,80.69)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.50</td><td rowspan=1 colspan=1>80.50</td><td rowspan=1 colspan=1>0.2906</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(79.79,81.17)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>80.52</td><td rowspan=1 colspan=1>80.50</td><td rowspan=1 colspan=1>0.2997</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>(79.77,81.33)</td></tr></table>

Reproducibility Summary Results for Parainfluenza virus 3   

<table><tr><td rowspan=1 colspan=3>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreement withExpected Result</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=3>Parainfluenza virus 3(Zeptometrix #0810016CF)Moderate Positive(3 x LoD)(30 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% -100.0%</td></tr><tr><td rowspan=4 colspan=3>Parainfluenza virus 3(Zeptometrix #0810016CF)Low Positive(1 x LoD)(10 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>3/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>86.1% -99.0%</td></tr><tr><td rowspan=4 colspan=3>Parainfluenza virus 3(Zeptometrix #0810016CF)High Negative(0.1 x LoD)(1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>22/60</td><td rowspan=1 colspan=1>38/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.7%</td><td rowspan=1 colspan=1>24.6% -50.1%</td></tr><tr><td rowspan=4 colspan=3>Parainfluenza virus 3Negative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% -100.0%</td></tr></table>

Reproducibility Summary Tm Results for Parainfluenza virus 3 Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>PIV3</td><td rowspan=4 colspan=1>Parainfluenza virus 3(Zeptometrix #0810016CF)Moderate Positive(3 x LoD)(30 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>81.17</td><td rowspan=1 colspan=1>81.09</td><td rowspan=1 colspan=1>0.2901</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(80.71,81.86)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.48</td><td rowspan=1 colspan=1>81.45</td><td rowspan=1 colspan=1>0.2853</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(81.03,81.89)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.94</td><td rowspan=1 colspan=1>80.84</td><td rowspan=1 colspan=1>0.2249</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(80.63,81.37)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.22</td><td rowspan=1 colspan=1>81.13</td><td rowspan=1 colspan=1>0.3358</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>(80.63,81.89)</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 3(Zeptometrix #0810016CF)Low Positive(1 x LoD)(10 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.97</td><td rowspan=1 colspan=1>81.00</td><td rowspan=1 colspan=1>0.2907</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>(80.36,81.52)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.35</td><td rowspan=1 colspan=1>81.37</td><td rowspan=1 colspan=1>0.2124</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(80.93,81.79)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.86</td><td rowspan=1 colspan=1>80.84</td><td rowspan=1 colspan=1>0.2296</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(80.10,81.21)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.08</td><td rowspan=1 colspan=1>81.06</td><td rowspan=1 colspan=1>0.3216</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(80.10,81.79)</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza virus 3(Zeptometrix #0810016CF)High Negative(0.1 x LoD)(1 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.99</td><td rowspan=1 colspan=1>80.96</td><td rowspan=1 colspan=1>0.2092</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>(80.30,81.34)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>81.29</td><td rowspan=1 colspan=1>81.27</td><td rowspan=1 colspan=1>0.2298</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>(80.61,81.77)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.84</td><td rowspan=1 colspan=1>80.84</td><td rowspan=1 colspan=1>0.1931</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>(80.41,81.24)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>81.05</td><td rowspan=1 colspan=1>81.03</td><td rowspan=1 colspan=1>0.2765</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>(80.30,81.77)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>81.09</td><td rowspan=1 colspan=1>81.04</td><td rowspan=1 colspan=1>0.3102</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>(80.1,81.89)</td></tr></table>

Reproducibility Summary Results for RSV   

<table><tr><td rowspan=1 colspan=2>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>% Agreement withExpected Result</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=2>RSV (A)(Zeptometrix#0810040ACF)Moderate Positive(3 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% -100.0%</td></tr><tr><td rowspan=4 colspan=2>RSV (A)(Zeptometrix#0810040ACF)Low Positive(1 x LoD)(2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.0% -100.0%</td></tr><tr><td rowspan=4 colspan=2>RSV (A)(Zeptometrix#0810040ACF)High Negative(0.1 x LoD)(0.2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>46/60</td><td rowspan=1 colspan=1>14/60</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>64.0% -86.6%</td></tr><tr><td rowspan=4 colspan=2>RSVNegative(Un-spiked)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/360*</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.0% -100.0%</td></tr></table>

\*The number of expected negative results is lower than for other analytes because only 6 specimens (Specimens #1, 2, 3, 7, 8, and 9) were not spiked with RSV.

Reproducibility Summary Tm Results for RSV Assay   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism (Strain)/TestConcentration</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>MedianTm</td><td rowspan=1 colspan=1>StandardDeviationTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=13 colspan=1>RSV</td><td rowspan=4 colspan=1>RSV (A)(Zeptometrix#0810040ACF)Moderate Positive(3 x LoD)(6 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>80.44</td><td rowspan=1 colspan=1>80.51</td><td rowspan=1 colspan=1>0.2843</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>(79.46,80.83)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.86</td><td rowspan=1 colspan=1>80.83</td><td rowspan=1 colspan=1>0.1991</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>(80.41,81.56)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.39</td><td rowspan=1 colspan=1>80.32</td><td rowspan=1 colspan=1>0.2625</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>(80.08,80.91)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.58</td><td rowspan=1 colspan=1>80.61</td><td rowspan=1 colspan=1>0.3232</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>(79.46,81.56)</td></tr><tr><td rowspan=4 colspan=1>RSV (A)(Zeptometrix#0810040ACF)Low Positive(1 x LoD)(2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.82</td><td rowspan=1 colspan=1>79.77</td><td rowspan=1 colspan=1>0.4000</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>(78.93,80.62)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>80.45</td><td rowspan=1 colspan=1>0.3727</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>(79.47,81.03)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>80.13</td><td rowspan=1 colspan=1>80.17</td><td rowspan=1 colspan=1>0.3792</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>(79.13,80.79</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>80.10</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>0.4564</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>(78.93,81.03)</td></tr><tr><td rowspan=4 colspan=1>RSV (A)(Zeptometrix#0810040ACF)High Negative(0.1 x LoD)(0.2 TCID50/mL)</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>79.63</td><td rowspan=1 colspan=1>79.55</td><td rowspan=1 colspan=1>0.4003</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>(78.72,80.72)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>80.07</td><td rowspan=1 colspan=1>0.4014</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>(79.26,80.89)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>79.97</td><td rowspan=1 colspan=1>79.94</td><td rowspan=1 colspan=1>0.4563</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>(78.83,80.84)</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>79.90</td><td rowspan=1 colspan=1>79.88</td><td rowspan=1 colspan=1>0.4661</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>(78.72,80.89)</td></tr><tr><td rowspan=1 colspan=2>Overall (all concentrations, all sites)</td><td rowspan=1 colspan=1>80.06</td><td rowspan=1 colspan=1>80.09</td><td rowspan=1 colspan=1>0.4989</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>(78.72,81.56)</td></tr></table>

The repeatability of the FilmArray RP System results was evaluated by repeated testing of the same 12 specimens while minimizing as many sources of variability as possible. The in-house precision testing was performed at ITI (Site C) over the course of 12 testing days for a total of 48 test results per specimen. On each day, all 12 specimens were tested 4 times by two operators on two FilmArray instruments. As much as possible, instrument and operator were held constant for each specimen. Results from the first 5 days of testing are included as the Reproducibility results for Site C. All operators were instructed to retest any samples that produced Invalid test results (failed controls or incomplete runs). Valid retest results were recorded as the final result.

A summary of precision study results for each analyte at each test level is provided below:

Summary of Repeatability Results   

<table><tr><td rowspan=2 colspan=1>Result / Spiked Organism</td><td rowspan=1 colspan=2>Moderate Positive(3 x LoD)</td><td rowspan=1 colspan=2>Low Positive(1 x LoD)</td><td rowspan=1 colspan=2>High Negative(0.1 x LoD)</td></tr><tr><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>34/48</td><td rowspan=1 colspan=1>70.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>44/48</td><td rowspan=1 colspan=1>91.7%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>83/96</td><td rowspan=1 colspan=1>86.5%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>43/48a</td><td rowspan=1 colspan=1>89.6%b</td><td rowspan=1 colspan=1>39/48b</td><td rowspan=1 colspan=1>81.3%b</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1N1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>47/48c</td><td rowspan=1 colspan=1>97.9%c</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>7/48d</td><td rowspan=1 colspan=1>14.6%d</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>25/48</td><td rowspan=1 colspan=1>52.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>41/48</td><td rowspan=1 colspan=1>85.4%</td><td rowspan=1 colspan=1>14/48</td><td rowspan=1 colspan=1>29.2%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr></table>

a The five (5) non-positive results for Influenza A/H1 at LoD include: (1) Negative, (1) Influenza A/H1 equivocal and (3) Influenza A equivocal results. b The nine (9) non-positive results for Influenza A/H1 at $0 . 1 \mathrm { { x } }$ LoD include: (1) Negative, (3) Influenza A (no subtype detected), (1) Influenza A/H1 equivocal, and (4) Influenza A equivocal results. c One (1) equivocal Influenza $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ result at the $0 . 1 \mathrm { ~ x ~ L o D }$ test level. d The 42 non-positive results for Influenza $\mathrm { A } / \mathrm { H } 3$ at $_ { 0 . 1 \mathrm { ~ x ~ } }$ LoD include: (18) Negative, (2) Influenza A (no subtype detected) (15) Influenza A H3 equivocal, and (7) Influenza A equivocal results.

Repeatability Summary Tm Results for FilmArray RP Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/TestConcentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">Adeno</td><td colspan="1" rowspan="1">Adenovirus (Species C - Serotype 1)Moderate Positive</td><td colspan="1" rowspan="1">83.78</td><td colspan="1" rowspan="1">83.84</td><td colspan="1" rowspan="1">0.2407</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(83.03, 84.18)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus (Species C - Serotype 1)Low Positive</td><td colspan="1" rowspan="1">83.52</td><td colspan="1" rowspan="1">83.45</td><td colspan="1" rowspan="1">0.2978</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">(83.03, 84.64)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus (Species C - Serotype 1)High Negative</td><td colspan="1" rowspan="1">83.28</td><td colspan="1" rowspan="1">83.33</td><td colspan="1" rowspan="1">0.2374</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(82.61, 83.87)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">83.5</td><td colspan="1" rowspan="1">83.45</td><td colspan="1" rowspan="1">0.3267</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">(82.61, 84.64)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">Standard DeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">CoV-HKU1</td><td colspan="1" rowspan="1">Coronavirus HKU1 (Type B)Moderate Positive</td><td colspan="1" rowspan="1">75.41</td><td colspan="1" rowspan="1">75.48</td><td colspan="1" rowspan="1">0.2375</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(74.87, 75.90)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1 (Type B)Low Positive</td><td colspan="1" rowspan="1">75.22</td><td colspan="1" rowspan="1">75.27</td><td colspan="1" rowspan="1">0.4053</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">(71.82, 75.81)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1 (Type B)High Negative</td><td colspan="1" rowspan="1">75.27</td><td colspan="1" rowspan="1">75.28</td><td colspan="1" rowspan="1">0.1951</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(74.76, 75.70)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">75.30</td><td colspan="1" rowspan="1">75.28</td><td colspan="1" rowspan="1">0.3045</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(71.82, 75.90)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">CoV-NL63</td><td colspan="1" rowspan="1">Coronavirus NL63 (BEI ResourcesNR-470)Moderate Positive</td><td colspan="1" rowspan="1">80.01</td><td colspan="1" rowspan="1">79.99</td><td colspan="1" rowspan="1">0.1944</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(79.57, 80.62)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63 (BEI ResourcesNR-70Low Positive</td><td colspan="1" rowspan="1">79.87</td><td colspan="1" rowspan="1">79.87</td><td colspan="1" rowspan="1">0.2456</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(79.14, 80.51)</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63 (BEI ResourcesNR-470)High Negative</td><td colspan="1" rowspan="1">79.85</td><td colspan="1" rowspan="1">79.88</td><td colspan="1" rowspan="1">0.2396</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.05, 80.52)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">79.89</td><td colspan="1" rowspan="1">79.89</td><td colspan="1" rowspan="1">0.2409</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.05, 80.62)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">hMPV</td><td colspan="1" rowspan="1">hMPV (hMPV-16 /IA10-2003Moderate Positive</td><td colspan="1" rowspan="1">77.69</td><td colspan="1" rowspan="1">77.68</td><td colspan="1" rowspan="1">0.1830</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">(77.27, 78.11)</td></tr><tr><td colspan="1" rowspan="1">hMPV (hMPV-16 /IA10-003Low Positive</td><td colspan="1" rowspan="1">77.59</td><td colspan="1" rowspan="1">77.57</td><td colspan="1" rowspan="1">0.2643</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(77.05, 78.36)</td></tr><tr><td colspan="1" rowspan="1">hMPV (hMPV-16 /IA10-200High Negative</td><td colspan="1" rowspan="1">77.41</td><td colspan="1" rowspan="1">77.37</td><td colspan="1" rowspan="1">0.2355</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(76.74, 78.10)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">77.55</td><td colspan="1" rowspan="1">77.57</td><td colspan="1" rowspan="1">0.2610</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(76.74, 78.36)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">Entero 1</td><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)Moderate Positive</td><td colspan="1" rowspan="1">86.91</td><td colspan="1" rowspan="1">86.89</td><td colspan="1" rowspan="1">0.2689</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(86.37, 87.64)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus )Low Positive</td><td colspan="1" rowspan="1">86.74</td><td colspan="1" rowspan="1">86.71</td><td colspan="1" rowspan="1">0.2692</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(86.15, 87.41)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)High Negative</td><td colspan="1" rowspan="1">86.64</td><td colspan="1" rowspan="1">86.6</td><td colspan="1" rowspan="1">0.2349</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(85.97, 87.30)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">86.75</td><td colspan="1" rowspan="1">86.77</td><td colspan="1" rowspan="1">0.2788</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(85.97,87.64)</td></tr><tr><td colspan="1" rowspan="4">Entero 2</td><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)Moderate Positive</td><td colspan="1" rowspan="1">86.88</td><td colspan="1" rowspan="1">86.88</td><td colspan="1" rowspan="1">0.2861</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(86.16, 87.64)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)Low Positive</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">0.2779</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(86.05, 87.52)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)High Negative</td><td colspan="1" rowspan="1">86.62</td><td colspan="1" rowspan="1">86.6</td><td colspan="1" rowspan="1">0.2211</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(86.07, 87.20)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">86.72</td><td colspan="1" rowspan="1">86.7</td><td colspan="1" rowspan="1">0.2808</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(86.05,87.64)</td></tr><tr><td colspan="1" rowspan="4">HRV 4</td><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)Moderate Positive</td><td colspan="1" rowspan="1">85.51</td><td colspan="1" rowspan="1">85.45</td><td colspan="1" rowspan="1">0.268</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(84.91, 86.19)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 6Low Positive</td><td colspan="1" rowspan="1">85.29</td><td colspan="1" rowspan="1">85.31</td><td colspan="1" rowspan="1">0.2734</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(84.62, 86.26)</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Echovirus 6)High Negative</td><td colspan="1" rowspan="1">85.16</td><td colspan="1" rowspan="1">85.14</td><td colspan="1" rowspan="1">0.1866</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">(84.80, 85.74)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">85.32</td><td colspan="1" rowspan="1">85.31</td><td colspan="1" rowspan="1">0.2845</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(84.62, 86.26)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">HRV 1</td><td colspan="1" rowspan="1">Human Rhinovirus (1A)Moderate Positive</td><td colspan="1" rowspan="1">83.85</td><td colspan="1" rowspan="1">83.85</td><td colspan="1" rowspan="1">0.2715</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(83.23, 84.71)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)Low Positive</td><td colspan="1" rowspan="1">83.67</td><td colspan="1" rowspan="1">83.66</td><td colspan="1" rowspan="1">0.2320</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(83.04, 84.40)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)High Negative</td><td colspan="1" rowspan="1">83.72</td><td colspan="1" rowspan="1">83.74</td><td colspan="1" rowspan="1">0.2563</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(82.91, 84.48)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">83.75</td><td colspan="1" rowspan="1">83.75</td><td colspan="1" rowspan="1">0.2653</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(82.91, 84.71)</td></tr><tr><td colspan="1" rowspan="4">HRV 2</td><td colspan="1" rowspan="1">Human Rhinovirus (1A)Moderate Positive</td><td colspan="1" rowspan="1">83.45</td><td colspan="1" rowspan="1">83.45</td><td colspan="1" rowspan="1">0.3099</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">(82.61, 84.29)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)Low Positive</td><td colspan="1" rowspan="1">83.26</td><td colspan="1" rowspan="1">83.32</td><td colspan="1" rowspan="1">0.2456</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(82.51, 83.85)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)High Negative</td><td colspan="1" rowspan="1">83.33</td><td colspan="1" rowspan="1">83.34</td><td colspan="1" rowspan="1">0.2758</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(82.29, 84.17)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">83.35</td><td colspan="1" rowspan="1">83.35</td><td colspan="1" rowspan="1">0.2895</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(82.29, 84.29)</td></tr><tr><td colspan="1" rowspan="4">HRV 3</td><td colspan="1" rowspan="1">Human Rhinovirus (1A)Moderate Positive</td><td colspan="1" rowspan="1">82.97</td><td colspan="1" rowspan="1">83.00</td><td colspan="1" rowspan="1">0.3325</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(81.99, 83.76)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)Low Positive</td><td colspan="1" rowspan="1">82.85</td><td colspan="1" rowspan="1">82.91</td><td colspan="1" rowspan="1">0.3296</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(81.64, 83.88)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)High Negative</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">0.3455</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">(81.85, 83.86)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">82.92</td><td colspan="1" rowspan="1">82.93</td><td colspan="1" rowspan="1">0.3412</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(81.64, 83.88)</td></tr><tr><td colspan="1" rowspan="4">HRV 4</td><td colspan="1" rowspan="1">Human Rhinovirus (1A)Moderate Positive</td><td colspan="1" rowspan="1">84.02</td><td colspan="1" rowspan="1">84.03</td><td colspan="1" rowspan="1">0.2851</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(83.35, 84.93)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)Low Positive</td><td colspan="1" rowspan="1">83.81</td><td colspan="1" rowspan="1">83.77</td><td colspan="1" rowspan="1">0.2135</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(83.35, 84.28)</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus (1A)High Negative</td><td colspan="1" rowspan="1">83.84</td><td colspan="1" rowspan="1">83.85</td><td colspan="1" rowspan="1">0.2411</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(83.12, 84.58)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">83.89</td><td colspan="1" rowspan="1">83.86</td><td colspan="1" rowspan="1">0.2641</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(83.12, 84.93)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">FluA-pan1</td><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Moderate Positive</td><td colspan="1" rowspan="1">84.65</td><td colspan="1" rowspan="1">84.60</td><td colspan="1" rowspan="1">0.2571</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(83.76, 85.23)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Low Positive</td><td colspan="1" rowspan="1">84.70</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.2304</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(84.16, 85.21)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)High Negative</td><td colspan="1" rowspan="1">84.38</td><td colspan="1" rowspan="1">84.38</td><td colspan="1" rowspan="1">0.2315</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(83.85, 85.48)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">0.2793</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(83.76,85.48)</td></tr><tr><td colspan="1" rowspan="1">FluA-pan2</td><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Moderate Positive</td><td colspan="1" rowspan="1">80.42</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">0.2438</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.89, 80.84)</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Low Positive</td><td colspan="1" rowspan="1">80.44</td><td colspan="1" rowspan="1">80.50</td><td colspan="1" rowspan="1">0.4988</td><td colspan="1" rowspan="1">0.62</td><td colspan="1" rowspan="1">(76.74, 80.92)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)High Negative</td><td colspan="1" rowspan="1">80.09</td><td colspan="1" rowspan="1">80.09</td><td colspan="1" rowspan="1">0.2633</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(79.27, 80.73)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">80.27</td><td colspan="1" rowspan="1">80.3</td><td colspan="1" rowspan="1">0.3747</td><td colspan="1" rowspan="1">0.47</td><td colspan="1" rowspan="1">(76.74,80.92)</td></tr><tr><td colspan="1" rowspan="4">FluA-H1-pan</td><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Moderate Positive</td><td colspan="1" rowspan="1">78.81</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.2789</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">(77.67, 79.36)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)Low Positive</td><td colspan="1" rowspan="1">78.79</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.3146</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">(76.21, 79.26)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1 (A/Brisbane/59/07)High Negative</td><td colspan="1" rowspan="1">77.89</td><td colspan="1" rowspan="1">77.78</td><td colspan="1" rowspan="1">0.3806</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">(77.36, 79.04)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">78.46</td><td colspan="1" rowspan="1">78.70</td><td colspan="1" rowspan="1">0.5513</td><td colspan="1" rowspan="1">0.70</td><td colspan="1" rowspan="1">(76.21, 79.36)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">FluA-pan1</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Moderate Positive</td><td colspan="1" rowspan="1">84.87</td><td colspan="1" rowspan="1">84.83</td><td colspan="1" rowspan="1">0.2637</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(84.30, 85.61)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Low Positive</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.2304</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(83.57, 85.19)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)High Negative</td><td colspan="1" rowspan="1">84.62</td><td colspan="1" rowspan="1">84.70</td><td colspan="1" rowspan="1">0.6492</td><td colspan="1" rowspan="1">0.77</td><td colspan="1" rowspan="1">(79.54, 85.23)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">84.72</td><td colspan="1" rowspan="1">84.71</td><td colspan="1" rowspan="1">0.4392</td><td colspan="1" rowspan="1">0.52</td><td colspan="1" rowspan="1">(79.54, 85.61)</td></tr><tr><td colspan="1" rowspan="4">FluA-pan2</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Moderate Positive</td><td colspan="1" rowspan="1">80.47</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">0.2617</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">(79.74, 81.02)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Low Positive</td><td colspan="1" rowspan="1">80.28</td><td colspan="1" rowspan="1">80.29</td><td colspan="1" rowspan="1">0.1395</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">(79.88, 80.63)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)High Negative</td><td colspan="1" rowspan="1">80.21</td><td colspan="1" rowspan="1">80.19</td><td colspan="1" rowspan="1">0.2428</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.20, 80.82)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">80.29</td><td colspan="1" rowspan="1">0.2409</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(79.20, 81.02)</td></tr><tr><td colspan="1" rowspan="4">FluA-H1-pan</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Moderate Positive</td><td colspan="1" rowspan="1">78.59</td><td colspan="1" rowspan="1">78.63</td><td colspan="1" rowspan="1">0.322</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">(77.92, 79.44)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Low Positive</td><td colspan="1" rowspan="1">78.09</td><td colspan="1" rowspan="1">78.09</td><td colspan="1" rowspan="1">0.1991</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(77.68, 78.69)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)High Negative</td><td colspan="1" rowspan="1">77.99</td><td colspan="1" rowspan="1">78.00</td><td colspan="1" rowspan="1">0.2631</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">(77.16, 78.51)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">78.22</td><td colspan="1" rowspan="1">78.13</td><td colspan="1" rowspan="1">0.3732</td><td colspan="1" rowspan="1">0.48</td><td colspan="1" rowspan="1">(77.16, 79.44)</td></tr><tr><td colspan="1" rowspan="4">FluA-H1-2009</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Moderate Positive</td><td colspan="1" rowspan="1">78.51</td><td colspan="1" rowspan="1">78.52</td><td colspan="1" rowspan="1">0.2278</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(77.79, 78.93)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)Low Positive</td><td colspan="1" rowspan="1">78.43</td><td colspan="1" rowspan="1">78.42</td><td colspan="1" rowspan="1">0.2010</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(78.00, 78.95)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/2009 H1N1(A/SwineNY/03/2009)High Negative</td><td colspan="1" rowspan="1">78.44</td><td colspan="1" rowspan="1">78.43</td><td colspan="1" rowspan="1">0.2406</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">(77.80, 78.93)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">78.46</td><td colspan="1" rowspan="1">78.46</td><td colspan="1" rowspan="1">0.2259</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(77.79, 78.95)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">FluA-pan1</td><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Moderate Positive</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">85.07</td><td colspan="1" rowspan="1">0.2315</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(84.50, 85.42)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Low Positive</td><td colspan="1" rowspan="1">84.79</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">0.2213</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(84.30, 85.25)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)High Negative</td><td colspan="1" rowspan="1">84.83</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.2370</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(84.07, 85.33)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">84.83</td><td colspan="1" rowspan="1">84.81</td><td colspan="1" rowspan="1">0.2404</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(84.07, 85.42)</td></tr><tr><td colspan="1" rowspan="4">FluA-pan2</td><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Moderate Positive</td><td colspan="1" rowspan="1">79.50</td><td colspan="1" rowspan="1">79.56</td><td colspan="1" rowspan="1">0.2261</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(78.84, 79.86)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Low Positive</td><td colspan="1" rowspan="1">79.24</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.1706</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">(78.82, 79.68)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)High Negative</td><td colspan="1" rowspan="1">79.23</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.1486</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">(78.84, 79.54)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">79.26</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.1855</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">(78.82, 79.86)</td></tr><tr><td colspan="1" rowspan="4">FluA-H3</td><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Moderate Positive</td><td colspan="1" rowspan="1">82.44</td><td colspan="1" rowspan="1">82.4</td><td colspan="1" rowspan="1">0.2428</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(81.98, 82.90)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)Low Positive</td><td colspan="1" rowspan="1">82.15</td><td colspan="1" rowspan="1">82.17</td><td colspan="1" rowspan="1">0.2403</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">(81.36, 82.73)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3(A/Wisconsin/67/2005)High Negative</td><td colspan="1" rowspan="1">82.17</td><td colspan="1" rowspan="1">82.18</td><td colspan="1" rowspan="1">0.1838</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">(81.77, 82.57)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">82.21</td><td colspan="1" rowspan="1">82.19</td><td colspan="1" rowspan="1">0.2433</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">(81.36, 82.90)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">FluB</td><td colspan="1" rowspan="1">Influenza B (B/FL/04/06)Moderate Positive</td><td colspan="1" rowspan="1">80.32</td><td colspan="1" rowspan="1">80.31</td><td colspan="1" rowspan="1">0.2584</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">(79.68, 80.91)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/FL/04/06)Low Positive</td><td colspan="1" rowspan="1">80.26</td><td colspan="1" rowspan="1">80.29</td><td colspan="1" rowspan="1">0.2116</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">(79.68, 80.81)</td></tr><tr><td colspan="1" rowspan="1">Influenza B (B/FL/04/06)High Negative</td><td colspan="1" rowspan="1">80.26</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">0.1993</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(79.37, 80.72)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">80.27</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">0.2179</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(79.37, 80.91)</td></tr><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism (Strain)/Test Concentration</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">MedianTm</td><td colspan="1" rowspan="1">StandardDeviationTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">PIV3</td><td colspan="1" rowspan="1">Parainfluenza virus 3Moderate Positive</td><td colspan="1" rowspan="1">81.01</td><td colspan="1" rowspan="1">81.04</td><td colspan="1" rowspan="1">0.2016</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(80.63, 81.47)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3Low Positive</td><td colspan="1" rowspan="1">80.86</td><td colspan="1" rowspan="1">80.92</td><td colspan="1" rowspan="1">0.2044</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">(80.10, 81.21)</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3High Negative</td><td colspan="1" rowspan="1">80.86</td><td colspan="1" rowspan="1">80.83</td><td colspan="1" rowspan="1">0.2206</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(79.99, 81.34)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">80.88</td><td colspan="1" rowspan="1">80.91</td><td colspan="1" rowspan="1">0.2177</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">(79.99, 81.47)</td></tr><tr><td colspan="1" rowspan="4">RSV</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus (Type A)Moderate Positive</td><td colspan="1" rowspan="1">80.38</td><td colspan="1" rowspan="1">80.33</td><td colspan="1" rowspan="1">0.2262</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">(79.89, 80.91)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus (Type A)Low Positive</td><td colspan="1" rowspan="1">80.03</td><td colspan="1" rowspan="1">80.09</td><td colspan="1" rowspan="1">0.3982</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">(78.95, 81.03)</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus (Type A)High Negative</td><td colspan="1" rowspan="1">79.89</td><td colspan="1" rowspan="1">79.88</td><td colspan="1" rowspan="1">0.4394</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">(78.83, 80.84)</td></tr><tr><td colspan="1" rowspan="1">Overall (all concentrations)</td><td colspan="1" rowspan="1">80.00</td><td colspan="1" rowspan="1">80.08</td><td colspan="1" rowspan="1">0.4334</td><td colspan="1" rowspan="1">0.54</td><td colspan="1" rowspan="1">(78.83, 81.03)</td></tr></table>

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The FilmArray Respiratory Panel (RP) pouch contains two internal control assays:

1. The RNA Process Control targets an mRNA of the yeast, Schizosaccharomyces pombe. During FilmArray RP pouch manufacture, whole yeast are freeze-dried into the sample injection port of each pouch. When the test specimen is loaded into the pouch, S. pombe is rehydrated and enters the pouch with the specimen. The yeast nucleic acid is extracted, purified and tested simultaneously with nucleic acids from the patient specimen. A positive result for the processing control indicates that all steps in the process (nucleic acid extraction, reverse transcription, PCR, melt, detection, and analysis) are functioning properly.   
2. The second stage PCR (PCR2) control assay detects a synthetic DNA template that is dried into triplicate wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

The RNA Process Control and the PCR2 Control assays are used to Pass or Fail each FilmArray RP pouch run. This combination of control assays monitors each of the critical mechanical and chemical processes that occur in a pouch run, while limiting the possibility of random control assay failures that could contribute to unnecessary pouch failures.

External controls are not provided with the FilmArray RP System. However, four (4) frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ external control mixes were provided to the clinical study sites for daily testing during the prospective clinical trial. Three control mixes contained pooled NPS specimens spiked with whole organisms with some plasmid DNA for hard to acquire organisms. Combined, the 3 mixes covered all panel analytes. A fourth mix was negative for all panel members and only contained pooled NPS:

<table><tr><td colspan="1" rowspan="1">Control Mix 1</td><td colspan="1" rowspan="1">Control Mix 2</td><td colspan="1" rowspan="1">Control Mix 3</td><td colspan="1" rowspan="1">Control Mix 4</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Influenza A/2009 H1N1</td><td colspan="1" rowspan="1">Negative for</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="5">all organisms</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="4"></td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Human Metapneumovirus</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td></tr></table>

The operators were required to complete a valid control mix run (correct results obtained) prior to beginning patient sample testing on each testing day. A total of 300 control mix runs were attempted. Seventeen (17) runs did not complete and 6 runs had failed pouch control(s). Of the remaining 277 runs (site 1, 99 runs; site 2, 70 runs; and site 3, 108 runs), 7 (site 1, 1 run; site 2, 2 runs; and site 3, 4 runs) (7/277; $2 . 5 \% )$ did not return the correct organism results either due to the detection of an extra analyte $( 4 / 7 )$ and/or the failure to detect one or more spiked analytes (4/7). These failures may be due to low level virus from the NPS donors, introduction of contamination during the preparation or testing of the samples, or improper handling of the frozen aliquots.

The sponsor is also recommending the following in the product package insert: “Good laboratory practice recommends running external positive and negative controls regularly. Use viral transport medium as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.”

# Specimen Stability

An analytical study was performed to establish the recommended transport and storage conditions for nasopharyngeal swab (NPS) specimens that will be tested using the Film Array Respiratory Panel System.

For the specimen transport and storage evaluation, both positive and negative specimens were assessed. Positive specimens consisted of simulated NPS (cultured human epithelial cells in VTM) spiked with mixes of targeted organisms at $5 \mathrm { { x } }$ the limit of detection (LoD) Negative data were collected from specimens that had been spiked with a different organism mix, such that results for the organism(s) of interest were expected to be negative. Ten independent specimens were prepared for each organism mix and tested “fresh” or without storage (0 Hour). Additional aliquots were made and two aliquots of 10 specimens per mix were stored at each of the conditions described below. Duplicate aliquots were stored for each specimen to allow for retesting if needed. For each RP organism, 10 positive and 30 negative specimens were evaluated at each storage condition. At least 9/10 positive results were required for each organism at each storage condition. In addition to positive and negative results, the average crossing point $\displaystyle ( \mathrm { C p } )$ of amplification curves and melting temperature (Tm) of melt curves were reviewed for each assay at each storage condition. A shift in Tm or $\mathrm { C p }$ from one storage condition to another could indicate that the integrity of the specimen was affected by storage. However, these data were used only as a secondary analysis tool and acceptance or rejection of a storage condition was based on the FilmArray result.

Composition of Organism Mixes spiked into simulated NPS Specimens for Specimen Transport and Storage Study   

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>1,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=1 colspan=1>1,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>150,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial</td><td rowspan=1 colspan=1>4,000 DNA copies/mL</td><td rowspan=1 colspan=1>20,000 DNA copies/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 4</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr></table>

Storage Conditions evaluated for simulated NPS Specimens   

<table><tr><td rowspan=1 colspan=2>Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>0 h</td><td rowspan=1 colspan=1>0 Hour, no storage</td></tr><tr><td rowspan=1 colspan=1>4 h</td><td rowspan=1 colspan=1>4 Hours, room temperature (18-30%C)</td></tr><tr><td rowspan=1 colspan=1>3 d</td><td rowspan=1 colspan=1>3 Days, refrigerated (2-8°C)</td></tr><tr><td rowspan=1 colspan=1>30 d</td><td rowspan=1 colspan=1>30 Days, frozen (&lt; -15%C)</td></tr></table>

False negative results by organism and storage condition are presented in the table below:

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=4>False Negative Results</td></tr><tr><td rowspan=1 colspan=1>0 Hour</td><td rowspan=1 colspan=1>4 Hours (18-30°C)</td><td rowspan=1 colspan=1>3 Days (2-8°C)</td><td rowspan=1 colspan=1>30 Days (&lt; -15°C)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009H1N1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td></tr></table>

In addition, the average crossing point $\displaystyle ( \mathrm { C p } )$ of amplification curves and melting temperature $\mathrm { ( T m ) }$ of melt curves were also reviewed for each assay at each storage condition. The difference in average $\mathrm { C p } \left( \Delta \mathrm { C p } \right)$ and average $\mathrm { T m } \left( \Delta \mathrm { T m } \right)$ between the test storage condition and the initial (0 Hour) test result was calculated for each assay. When $\Delta$ Cp for all assays was reviewed, no general trend toward later Cps (decreased sensitivity or fewer templates) was observed for any one particular storage condition.

The results of this study support the claim that NPS specimens in Viral Transport Medium (VTM) can be stored for up to 4 hours at room temperature $( 1 8 - 3 0 ^ { \circ } \mathrm { C } )$ , 3 days in the refrigerator $( 2 { - } 8 ^ { \circ } \mathrm { C } )$ or 30 days in the freezer $( < - 1 5 ^ { \circ } \mathrm { C } )$ without affecting the accuracy of FilmArray RP test results.

# Preservation of Vacuum for Pouch Hydration and Sample Loading and Stability of Reagents and Control Material after Hydration and Loading of a Pouch

To maintain pouch integrity and reagent stability, pouches are packaged in an aluminum canister, which is then sealed under vacuum in opaque mylar outer packaging. This packaging design maintains the pouch under vacuum and limits exposure of the pouch to light until the packaging has been opened and the pouch removed. Ideally, a pouch is hydrated and sample loaded immediately after the pouch packaging is opened, and the loaded pouch is immediately placed in the FilmArray instrument for testing. However, laboratory workflow and instrument availability may result in a pouch being opened before a sample can be loaded, or a pouch to be loaded before an instrument is available for testing.

The FilmArray product literature recommends the loading and testing of pouches immediately or within a limited timeframe (30 minutes for pouch loading and 60 minutes for testing a loaded pouch).

Analytical testing was performed to discover whether the integrity and performance of an opened pouch is affected by longer times outside of the vacuum-sealed packaging prior to loading and testing.

Pouches were removed from outer packaging and exposed to air at ambient temperature for various times up to 24 hours. The preservation of vacuum for pouch hydration and sample loading was evaluated by loading blank simulated NPS (sNPS) samples (no target analytes present) according to standard procedure. The volume of each solution (hydration solution and sample/sample buffer mix) drawn into the pouch was recorded and the loaded pouches were then tested to observe the performance of the internal controls assays. The control results from these same pouches were used to assess reagent stability. Overall, the measures of performance for these tests included the efficiency of pouch hydration and sample loading, visual inspection of pouches, and the results (pass or fail) of internal control assays (RNA Process Control and PCR2 Control).

The stability of reagents and control material after hydration and loading of a pouch was also examined. Pouches were hydrated and loaded with blank sNPS sample (no analytes present) according to standard protocol directly after removal from outer packaging. The loaded pouches were then stored under ambient conditions for various times up to 24 hours. The results of internal control reactions (pass or fail) were assessed to determine whether system performance was affected by storage of a loaded pouch prior to testing. In addition, four (4) replicate pouches were loaded with contrived sample containing live respiratory pathogens (Influenza A H1N1, human Metapneumovirus, Parainfluenza virus 3 and Respiratory Syncytial Virus) at concentrations both above and below the estimated system LoD. The loaded pouches were stored at ambient temperature for 8 hours prior to testing. The results of both 39

internal control and analyte assays from these 4 pouches were compared to 8 control pouches that were loaded with the same contrived sample but tested without storage (immediately after loading).

Each tested handling condition was considered successful if the variables (including mean Cp for the relevant assay) evaluated were equivalent (within 3 cycles for $\mathrm { C p }$ ) to that of the control sample (test at time 0 after pouch opening or loading). Any handling condition that resulted in improper pouch hydration, sample loading, failed control results, or unexpected negative analyte results was repeated to confirm the result. If the result was repeatable, then the integrity and/or stability of the pouch were considered diminished under the handling condition that caused the failure.

The data presented in the table below demonstrated that pouch vacuum was preserved for up to 24 hours after the removal of a pouch from its outer packaging, allowing for proper pouch hydration and sample uptake.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Pouchhydratedproperly?</td><td rowspan=1 colspan=2>Sampleloadedproperly?</td><td rowspan=1 colspan=2>Internalcontrolspass?</td><td rowspan=1 colspan=2>Internal ControlCp</td></tr><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>no</td><td rowspan=1 colspan=1>RNA PC</td><td rowspan=1 colspan=1>PCR2</td></tr><tr><td rowspan=6 colspan=1>Remove pouchfrom packaging atvarious times priorto loading</td><td rowspan=1 colspan=1>0 (control)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20.77</td><td rowspan=1 colspan=1>21.43</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.73</td><td rowspan=1 colspan=1>19.93</td></tr><tr><td rowspan=1 colspan=1>60 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.53</td><td rowspan=1 colspan=1>21.53</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.50</td><td rowspan=1 colspan=1>20.33</td></tr><tr><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.97</td><td rowspan=1 colspan=1>20.90</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.40</td><td rowspan=1 colspan=1>20.65</td></tr></table>

The data presented in the table below indicated that the stability of the control material and assay reagents is not significantly affected up to 24 hours after pouch hydration and sample loading.

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Internal controls pass?</td><td rowspan=1 colspan=2>Internal control Cp</td></tr><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>RNA PC</td><td rowspan=1 colspan=1>PCR2</td></tr><tr><td rowspan=6 colspan=1>Store a loaded pouch forvarious times prior totesting</td><td rowspan=1 colspan=1>0 (control)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20.77</td><td rowspan=1 colspan=1>21.43</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.90</td><td rowspan=1 colspan=1>21.57</td></tr><tr><td rowspan=1 colspan=1>60 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.67</td><td rowspan=1 colspan=1>20.20</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>23.47</td><td rowspan=1 colspan=1>20.63</td></tr><tr><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.80</td><td rowspan=1 colspan=1>21.23</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>23.40</td><td rowspan=1 colspan=1>21.07</td></tr></table>

The following table lists the number of positive and negative results, an overall $\%$ positive, and the mean $\mathrm { C p }$ for each analyte or assay at each of the two test conditions.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="2">Multiple ofLoDTested</td><td colspan="4" rowspan="1">Control(tested immediately)(n = 8 pouches)</td><td colspan="4" rowspan="1">Test(tested after 8 hours @ room temperature)(n = 4 pouches)</td></tr><tr><td colspan="1" rowspan="1">Organism / Assay</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">%Positive</td><td colspan="1" rowspan="1">MeanCp</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">%Positive</td><td colspan="1" rowspan="1">MeanCp</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">~18x</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.4</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.3</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="2">~0.2x</td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">2/8ab</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Flu A - pan1</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">26.9</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">27.1</td></tr><tr><td colspan="1" rowspan="1">Flu A - pan2</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">2/8</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">27.1</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">2/4</td><td colspan="1" rowspan="1">50%</td><td colspan="1" rowspan="1">27.1</td></tr><tr><td colspan="1" rowspan="1">FluA- H1</td><td colspan="1" rowspan="1">6/8</td><td colspan="1" rowspan="1">2/8</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">27.1</td><td colspan="1" rowspan="1">3/4</td><td colspan="1" rowspan="1">1/4</td><td colspan="1" rowspan="1">75%</td><td colspan="1" rowspan="1">27.1</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">~8x</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">22.1</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">23.1</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">~2x</td><td colspan="1" rowspan="1">7/8</td><td colspan="1" rowspan="1">1/8</td><td colspan="1" rowspan="1">88%</td><td colspan="1" rowspan="1">23.1</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">23.6</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">~7x</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">18.0</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">18.0</td></tr><tr><td colspan="1" rowspan="1">RNA Process Control</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">21.2</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">20.6</td></tr><tr><td colspan="1" rowspan="1">PCR2 Control</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">8/8</td><td colspan="1" rowspan="1">0/8</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">19.8</td><td colspan="1" rowspan="1">4/4</td><td colspan="1" rowspan="1">0/4</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">19.6</td></tr></table>

a One Influenza A (no subtype detected) b One (1) Equivocal Influenza A

In conclusion, the data supports the conservative recommendations in the FilmArray product package insert for the loading and testing of pouches within a limited timeframe (30 minutes for pouch loading and 60 minutes for testing a loaded pouch).

# Fresh vs. Frozen Study

In order to utilize frozen banked clinical samples in the evaluation of FilmArray Respiratory Panel System to supplement the prospective clinical study data, an analytical study was conducted to demonstrate that preservation of samples (by freezing at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ does not affect the accuracy of test results compared to freshly collected or freshly prepared samples.

The “fresh vs. frozen” analytical study was carried out as part of a larger reproducibility study using samples that had been stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ for as long as 30 days. The test samples were prepared in bulk and tested immediately after preparation (fresh) prior to storage. The fresh sample tests provided a baseline of performance to which the results from subsequent testing of the frozen samples was compared.

A panel of twelve specimens was created by spiking simulated NPS sample matrix with known quantities of live RP analytes. The specimens in the panel were designed so that all RP analytes would be tested at different concentrations. Multiple analytes were spiked into each specimen to simulate the possibility of co-infections with multiple pathogens. Human Rhinovirus (HRV) and Respiratory Syncytial Virus (RSV) were included in more specimens than the other analytes because they were the most common panel members detected in co-infected clinical samples during the 2009-2010 clinical evaluation performed by ITI. Immediately after sample preparation (spiking), each fresh specimen pool was screened with the FilmArray RP system to establish ‘baseline’ or expected results. The remaining specimen volume was then aliquoted into several individual use vials, and frozen until the scheduled day of testing (up 30 days frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ ). Specimen aliquots were hand-delivered or shipped to external testing sites on dry ice and stored frozen until the day of testing. All specimen aliquots were tested on the same day they were removed from the freezer.

In total, at least 120 samples were tested for each analyte at (1X) or near (3X) the LoD determined in the LoD studies. All contrived samples consisted of live organism spiked into negative simulated nasopharyngeal swab sample matrix. (Note: The simulated sample matrix consisted of human cells in viral transport media (VTM) and was validated in “Validation of a Simulated NPS Sample Matrix for Use with the

FilmArray Respiratory Panel System”. The validation study demonstrated that the sensitivity of the FilmArray RP system was comparable between samples prepared in nasopharyngeal swab (NPS) and simulated NPS sample matrices.) Since the nature NPS samples are required to be diluted in VTM before being tested by the FilmArray RP System, the simulated NPS matrix consisting of VTM and epithelial cells is considered to be similar enough to the nature matrix (i.e., NPS in VTM) for the purpose of this study.

The summary results of the study are presented in the table below:

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=2>Medium Positive (3 X LoD)</td><td rowspan=1 colspan=2>Low Positive (1 X LoD)</td><td rowspan=1 colspan=1>All Test</td><td rowspan=1 colspan=1>All Test Levels Combined</td></tr><tr><td rowspan=1 colspan=1># detected/total</td><td rowspan=1 colspan=1>% Positive[95% CI]</td><td rowspan=1 colspan=1># detected/total</td><td rowspan=1 colspan=1>% Positive[95% CI]</td><td rowspan=1 colspan=1># detected/total</td><td rowspan=1 colspan=1>% Positive[95% CI]</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04%  100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>CoronavirusHKU1</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04%  100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>CoronavirusNL63</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04%  100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>98.3%[91.06% - 99.96%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>119/120</td><td rowspan=1 colspan=1>99.2%[95.44% - 99.98%]</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%[97.97% - 100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>240/240</td><td rowspan=1 colspan=1>100%[98.47% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>56/60a</td><td rowspan=1 colspan=1>93.3% a[83.80% - 98.15%]</td><td rowspan=1 colspan=1>116/120</td><td rowspan=1 colspan=1>96.7%[91.69% - 99.08%]</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009H1N1</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97% - 100%]</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%[94.04%  100%]</td><td rowspan=1 colspan=1>57/60</td><td rowspan=1 colspan=1>95.0%[86.08% - 98.96%]</td><td rowspan=1 colspan=1>117/120</td><td rowspan=1 colspan=1>97.5%[92.87% - 99.48%]</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%[97.97% - 100%]</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%[96.97%  100%]</td><td rowspan=1 colspan=1>300/300</td><td rowspan=1 colspan=1>100%[98.78% - 100%]</td></tr></table>

a Three (3) Equivocal - not included as a positive result. Samples with Equivocal results were not retested in this study.

The results of this study demonstrated that freezing at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ for up to 30 days does not alter the performance of the FilmArray RP System in comparison to testing of fresh specimens.

# d. Detection limit:

Analytical studies were carried out to determine the LoD for each FilmArray RP targeted organism.

A preliminary estimate of sensitivity for many RP analytes was determined as part of the “FilmArray RP Melt Detector Tuning” study using limiting dilutions of single organisms spiked into negative NPS specimens. Based on these preliminary results, initial estimates of LoD were set by spiking negative NPS specimens with serial 10- fold dilutions of quantified live virus. Where possible, each stock organism was re42

# grown, verified, and quantified $\bf { T C I D _ { 5 0 } / m L }$ , $\mathbf { C F U / m L }$ , copies/mL, etc.).

Coronavirus HKU1 could not be re-grown for verification despite multiple attempts. Instead, clinical specimen containing a high viral load of the virus of interest were quantified by non-FilmArray real-time PCR against a standard curve of synthetic RNA transcript (Coronavirus HKU1) to obtain quantification of the stock material in RNA copies/mL, respectively. The identity of the virus contained in the clinical specimen was verified by bi-directional sequencing. The single-spiked specimens were then tested with the FilmArray RP System. The lowest concentration of organism for which all or most replicates were assigned a positive (detected) result by the FilmArray software was selected for additional testing in multi-spiked specimens.

Multiple organisms were combined and tested as a mix (multi-spike; see the table below) in replicate NPS specimens at a single concentration or as a dilution series centered on the estimated LoD. Results from single-spiked and multi-spiked specimens were compared to determine whether the presence of multiple organisms in a specimen would affect the system LoD.

Composition of Organism Mixes   

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>Mix 4</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Human Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Human Metapneumovirus Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=4>Influenza A/H3</td></tr></table>

Both percent detection $\%$ positive) and $\mathrm { C p }$ values were used to compare results for single-spiked and multi- spiked specimen data sets. It was concluded that multispiking and single-spiking results were substantially equivalent, since the $\%$ detection was within $2 5 \%$ for the two data sets and/or the mean Cp values were within 3 cycles. (Note: Based on an observed $\mathrm { C p }$ standard deviation for replicate samples of about 1-2 cycles, a 3 cycle difference is a conservative estimate of the expected variation when testing near LoD and does not indicate a significant difference in sensitivity.)

Additional LoD testing and confirmation of LoD for primary strains was then performed with multi-spiked specimens. Coronavirus HKU1 was not included in a multi- spiked mix for this study. Coronavirus HKU1 LoD was confirmed as a single-spiked sample but was shown in subsequent studies to be detected at singlespiked LoD levels when tested in multi-spiked specimens.

As needed, adjustments to the concentration of individual organisms within a multispiked mix were made prior to confirmation of final LoD concentrations. Confirmation of the system LoD for each organism included testing of 20 independent multi-spiked specimens at LoD and an additional 20 specimens spiked 10- fold below LoD ( $0 . 1 \mathbf { x }$ LoD). For confirmation, at least 19/20 positive results were required for each organism at the LoD concentration and fewer than $9 5 \%$ positive results were required at concentrations below LoD.

Additional sensitivity testing was performed on one or more alternate strains of several RP organisms. The majority of testing and LoD determination using

alternate strains of organism was performed by military laboratory personnel at USAMRIID (United States Army Medical Research Institute of Infectious Diseases). These alternate strains represented different major subgroups of common respiratory viruses (Adenovirus, Human Metapneumovirus, Human Rhinovirus and Respiratory Syncytial Virus) and isolates of Influenza A and B from different decades and geographic locations. Each alternate strain was tested as 10-fold dilutions around the previously determined LoD.

Note: Although this testing was performed in part to demonstrate that the FilmArray RP system can detect various species and strains of an organism with similar sensitivity, it is difficult to make comparative sensitivity evaluations between different organisms and different strains that have been quantified by $\mathrm { T C I D } _ { 5 0 }$ . This quantification method measures the infectivity and cytotoxicity or lethality of an organism in tissue culture and is therefore subject to many influences (strain-to-strain differences in infectivity, viability of the original material, culturing conditions, etc.). Quantification by infectivity assay will not be equivalent between strains or organisms. Also, since the measurand for the FilmArray system is nucleic acid, LoD concentrations established in $\mathrm { T C I D } _ { 5 0 }$ can vary dramatically between targets, but may not actually reflect significant differences in sensitivity of detection as measured by target nucleic acid concentration.

During the early stages of LoD testing, it was discovered that some NPS specimens collected from apparently healthy individuals did in fact contain respiratory pathogens (mostly Rhinovirus). As a result, a simulated NPS (sNPS) sample matrix, consisting of cultured human epithelial cells (HeLa) in Viral Transport Medium, was evaluated for use in LoD confirmation testing and subsequent analytical studies. The sensitivity of the FilmArray RP system with this simulated sample matrix was found to be comparable to a sample matrix comprised of pooled NPS collected from healthy individuals. The simulated NPS sample matrix was used for final LoD confirmations and all subsequent analytical studies. (Note: Refer to “Validation of Simulated NPS Sample Matrix for the FilmArray Respiratory Panel System” study for the results of a side-by-side comparison of multi-spiked specimens prepared in a collected NPS sample matrix versus a simulated NPS sample matrix.)

Throughout the LoD studies, crossing point values (Cps) were utilized as a supplement to $\%$ detection data. However, the significance of these numbers should not be over-interpreted, as the FilmArray RP is a qualitative rather than quantitative system. Cp assignments in this nested multiplex PCR system may not be as quantitative or reproducible as in traditional real-time PCR. In addition, positive results may still be obtained for targets with negative amplification curves (no Cp) due to detection by the more sensitive melt curve analysis.

Summary results for the LoD confirmation studies using primary strains and additional sensitivity testing of alternate strains are presented below by organism:

# Adenovirus

LoD Confirmation Results for Adenovirus C Serotype 1 (Primary strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=7>iuellovl us C Serotype 1</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>23.81 ± 1.59</td></tr><tr><td rowspan=1 colspan=1>30(0.1 x LoD)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.88 ± 0.78</td></tr></table>

# LoD testing results for Adenovirus B Serotype 3 (Alternate Strain for the LoD study)

<table><tr><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=2>Single SpikeDilution Series</td><td rowspan=1 colspan=1>3000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>16.2</td><td rowspan=1 colspan=1>16.24 ± 0.38</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>20.12 ± 1.53</td></tr><tr><td rowspan=1 colspan=1>ies</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>22.70 ± 0.64</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr></table>

# LoD testing results for Adenovirus E Serotype 4a (Alternate Strain for the LoD study)

<table><tr><td rowspan=1 colspan=8>Adenovirus E  Serotype 4a</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=2>Single SpikeDilution Series</td><td rowspan=1 colspan=1>3000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.5</td><td rowspan=1 colspan=1>21.78 ± 2.45</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>25.72 ± 1.72</td></tr><tr><td rowspan=1 colspan=1>Series</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# Coronavirus HKU1

LoD Confirmation Results for Coronavirus HKU1 Clinical Specimen #6213(Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=7>Coronavirus HKU1Clinical Specimen #6213</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(RNAconies/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Single Spike )</td><td rowspan=1 colspan=1>1.90 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.4</td><td rowspan=1 colspan=1>21.43 ± 1.01</td></tr><tr><td rowspan=1 colspan=1>1.90 x 105(0.1 x LoD)</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>70%</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>25.16 ± 2.19</td></tr></table>

# Coronavirus NL63

LoD Confirmation Results for Coronavirus NL63 NR-470 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=7>Coronavirus NL63 NR-470</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>21.32 ± 1.13</td></tr><tr><td rowspan=1 colspan=1>0.5(0.1 x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>80%</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>24.55 ± 2.39</td></tr></table>

# Human Metapneumovirus

LoD Confirmation Results for Human metapneumovirus (Type A1) hMPV-16, IA10-2003 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=7>Human metapneumovirus (Type A1) hMPV-16, IA10-2003</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix1)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>25.15 ± 1.64</td></tr><tr><td rowspan=1 colspan=1>0.2(0.1 x LoD)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>45%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.11 ± 0.02</td></tr></table>

<table><tr><td rowspan=1 colspan=7>LoD Testing Results for Human metapneumovirus (Type A2) (Alternate Strain for the LoD study)</td></tr><tr><td rowspan=1 colspan=7>Human metapneumovirus (Type A2) hMPV-20, IA14-2003</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>23.08 ± 1.44</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr></table>

LoD Testing Results for Human metapneumovirus (Type B1) (Alternate Strain for the LoD study) Human metapneumovirus (Type B1) hMPV-3, Peru2-2002   
LoD Testing Results for Human metapneumovirus (Type B2) (Alternate Strain for the LoD study)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.5</td><td rowspan=1 colspan=1>24.18 ± 1.69</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>24.46 ± 1.69</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>23.86 ± 1.53</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# Enterovirus

LoD Confirmation Results for Enterovirus (Echovirus 6) (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=13>Enterovirus Echovirus 6</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>SpikedConc.(TCID50/mL</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Entero 1</td><td rowspan=1 colspan=2>Entero 2</td><td rowspan=1 colspan=2>HRV3</td><td rowspan=1 colspan=2>HRV 4</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(MultipleSpike Mix 2)</td><td rowspan=1 colspan=1>30000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.1(19/20)</td><td rowspan=1 colspan=1>26.70 ±1.26</td><td rowspan=1 colspan=1>27.1(20/20)</td><td rowspan=1 colspan=1>26.51 ±1.28</td><td rowspan=1 colspan=1>27.1(14/20)</td><td rowspan=1 colspan=1>27.14 ±0.10</td><td rowspan=1 colspan=1>23.9(20/20)</td><td rowspan=1 colspan=1>24.66 ±1.63</td></tr><tr><td rowspan=1 colspan=1>3000(0.1 x LoD)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90%</td><td rowspan=1 colspan=1>27.1(17/20)</td><td rowspan=1 colspan=1>27.10 ±0.02</td><td rowspan=1 colspan=1>27.1(17/20)</td><td rowspan=1 colspan=1>27.05 ±0.41</td><td rowspan=1 colspan=1>n/a(0/20)</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>27.1(18/20)</td><td rowspan=1 colspan=1>26.88 ±0.76</td></tr></table>

# Human Rhinovirus

LoD Confirmation Results for Human rhinovirus Type 1A (Primary strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=13>Human Rhinovirus - Type 1A</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>HRV1</td><td rowspan=1 colspan=2>HRV2</td><td rowspan=1 colspan=2>HRV3</td><td rowspan=1 colspan=2>HRV 4</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±SStd.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp f$Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(MultipleSpike Mix 2)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.6(20/20)</td><td rowspan=1 colspan=1>23.46 ±1.30</td><td rowspan=1 colspan=1>21.1(20/20)</td><td rowspan=1 colspan=1>21.04 ±1.38</td><td rowspan=1 colspan=1>20.8(20/20)</td><td rowspan=1 colspan=1>20.81 ±1.42</td><td rowspan=1 colspan=1>23.9(19/20)</td><td rowspan=1 colspan=1>24.13 ±1.48</td></tr><tr><td rowspan=1 colspan=1>0.1(0.1 x LoD)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>15%</td><td rowspan=1 colspan=1>27.1(2/20)</td><td rowspan=1 colspan=1>27.10 ±0.00</td><td rowspan=1 colspan=1>27.1(3/20)</td><td rowspan=1 colspan=1>27.08 ±0.04</td><td rowspan=1 colspan=1>27.1(3/20)</td><td rowspan=1 colspan=1>27.06 ±0.10</td><td rowspan=1 colspan=1>27.1(2/20)</td><td rowspan=1 colspan=1>27.10 ±0.00</td></tr></table>

# LoD Confirmation Results for Human rhinovirus Type B14 (Alternate strain for the LoD Study) Human Rhinovirus B B14#1059 VR-284

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>HRV1</td><td rowspan=1 colspan=2>HRV2</td><td rowspan=1 colspan=2>HRV3</td><td rowspan=1 colspan=2>HRV 4</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp±$Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Single SpikeDilution</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19.0</td><td rowspan=1 colspan=1>18.88 ±1.72</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1>19.75 ±1.10</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>24.58 ±2.06</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ±0.00</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>23.68 ±1.69</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>24.38 ±2.07</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ±0.00</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>27.15 ±0.06</td></tr><tr><td rowspan=2 colspan=1>Series</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.18 ±1.69</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.78 ±0.97</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ±0.00</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>25.70 ±1.21</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>26.33 ±1.24</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ±0.00</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# Influenza A/H1N1

LoD Confirmation Results for Human Influenza A/H1N1 (A/Brisbane/59/07) (Primary strain for the LoD Study) Human Influenza A/H1N1 (A/Brisbane/59/07)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID0/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=1 colspan=2>FluA-pan1</td><td rowspan=1 colspan=2>FluA-pan2</td><td rowspan=1 colspan=2>FluA-H1-pan</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple SpikeMix 1)</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>20.3(20/20)</td><td rowspan=1 colspan=1>20.24 ±0.84</td><td rowspan=1 colspan=1>23.6(20/20)</td><td rowspan=1 colspan=1>23.88 ±1.46</td><td rowspan=1 colspan=1>22.3(19/20)</td><td rowspan=1 colspan=1>22.52 ±1.00</td></tr><tr><td rowspan=1 colspan=1>20(0.1 x LoD)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>27.1(12/20)</td><td rowspan=1 colspan=1>25.84 ±1.73</td><td rowspan=1 colspan=1>27.1(14/20)</td><td rowspan=1 colspan=1>27.10 ±0.02</td><td rowspan=1 colspan=1>27.1(12/20)</td><td rowspan=1 colspan=1>27.10 ±0.00</td></tr></table>

\* Influenza A (no subtype detected) result

LoD Confirmation Results for Human Influenza A/H1N1 (A/New Caledonia/20/99) (Alternate Strain for the LoD Study) Human Influenza A/H1N1 (A/New Caledonia/20/99)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=2>FluA-pan1</td><td rowspan=1 colspan=2>FluA-pan2</td><td rowspan=1 colspan=2>FluA-H1-pan</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>MedianC</td><td rowspan=1 colspan=1>Mean Cp ±Sd. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±td. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilutionSeries</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19.9</td><td rowspan=1 colspan=1>19.61 ± 0.57</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>24.11 ± 1.30</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>23.69 ± 1.35</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>25.94 ± 1.38</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>27.20 ± 0.00</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>LoDConfirmation</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>18.94 ± 1.17</td><td rowspan=1 colspan=1>22.65</td><td rowspan=1 colspan=1>23.15 ± 1.99</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>22.30 ± 1.94</td></tr></table>

# Influenza A/2009 H1N1

LoD Confirmation Results for Influenza A/2009 H1N1 (A/Swine/NY/2009/03) (Primary strain for the LoD Study) Influenza A/2009 H1N1 (A/Swine/NY/2009/03)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FluA-pan1</td><td rowspan=1 colspan=2>FluA-pan2</td><td rowspan=1 colspan=2>FluA-H1-pan</td><td rowspan=1 colspan=2>FluA-H1-2009</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MeanCp ±Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(MultipleSpike Mix 2)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.8(20/20)</td><td rowspan=1 colspan=1>22.42 ±1.88</td><td rowspan=1 colspan=1>27.1(20/20)</td><td rowspan=1 colspan=1>26.77 ±0.80</td><td rowspan=1 colspan=1>23.4(20/20)</td><td rowspan=1 colspan=1>24.22 ±1.83</td><td rowspan=1 colspan=1>22.2(20/20)</td><td rowspan=1 colspan=1>22.90 ±1.77</td></tr><tr><td rowspan=1 colspan=1>10(0.1 x LoD)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>85%</td><td rowspan=1 colspan=1>27.1(19/20)</td><td rowspan=1 colspan=1>25.74 ±1.84</td><td rowspan=1 colspan=1>27.1(10/20)</td><td rowspan=1 colspan=1>27.11 ±0.02</td><td rowspan=1 colspan=1>27.1(17/20)</td><td rowspan=1 colspan=1>27.04 ±0.42</td><td rowspan=1 colspan=1>27.1(17/20)</td><td rowspan=1 colspan=1>26.44 ±1.40</td></tr></table>

\* Equivocal Influenza A result

# Influenza A/H3N2

LoD Confirmation Results for Human Influenza A/H3N2 (A/Wisconsin/67/2005) (Primary Strain for the LoD Study) Human Influenza A/H3N2 (A/Wisconsin/67/2005)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=1 colspan=2>FluA-pan1</td><td rowspan=1 colspan=2>FluA-pan2</td><td rowspan=1 colspan=2>FluA-H3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Sd. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp± Std.Dev.</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple SpikeMix 3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.2(19/20)</td><td rowspan=1 colspan=1>23.95 ± 1.96</td><td rowspan=1 colspan=1>27.0(20/20)</td><td rowspan=1 colspan=1>25.56 ±1.67</td><td rowspan=1 colspan=1>23.0(20/20)</td><td rowspan=1 colspan=1>22.72 ±1.42</td></tr><tr><td rowspan=1 colspan=1>0.5(LoD/10)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8*</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>27.1(10/20)</td><td rowspan=1 colspan=1>26.88 ± 0.76</td><td rowspan=1 colspan=1>27.1(15/20)</td><td rowspan=1 colspan=1>27.11 ±0.02</td><td rowspan=1 colspan=1>24.0(10/20)</td><td rowspan=1 colspan=1>24.92 ±1.65</td></tr></table>

\* 2 Equivocal Influenza A/H3 results, 6 Influenza A Equivocal results

LoD Confirmation Results for Human Influenza A/H3N2 (A/Port Chalmers/1/73) (Alternate Strain for the LoD Study)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/m</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FluA-pan1</td><td rowspan=1 colspan=2>FluA-pan2</td><td rowspan=1 colspan=2>FluA-H3</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>Mean Cp ±Std. Dev.</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>24.0</td><td rowspan=1 colspan=1>24.04 ± 2.12</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.61 ± 1.15</td><td rowspan=1 colspan=1>22.6</td><td rowspan=1 colspan=1>22.46 ± 0.89</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.87 ± 0.70</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2**</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.10 ± 0.00</td></tr><tr><td rowspan=1 colspan=1>LoDConfirmation</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>19.55 ± 1.30</td><td rowspan=1 colspan=1>22.5</td><td rowspan=1 colspan=1>22.98 ± 1.31</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>18.48 ± 1.29</td></tr></table>

\* Influenza A Equivocal \*\* 1 Influenza A Equivocal and 1 Influenza A/H3 Equivocal

# Influenza B

# LoD Confirmation Results for Influenza B (B/FL/04/06) (Primary Strain for the LoD Study) Influenza B (B/FL/04/06)

<table><tr><td rowspan=1 colspan=7>(D/TL/04/00)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 3)</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>22.17 ± 1.62</td></tr><tr><td rowspan=1 colspan=1>6(0.1 x LoD)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>65%</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>24.58 ± 1.97</td></tr></table>

# LoD Testing Results for Influenza B (B/Taiwan/2/62) (Alternate Strain for the LoD study)

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18.0</td><td rowspan=1 colspan=1>18.24 ± 0.93</td></tr><tr><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>21.61 ± 0.58</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>23.5</td><td rowspan=1 colspan=1>23.59 ± 1.67</td></tr><tr><td rowspan=1 colspan=1>LoD Confirmation</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>22.35 ± 1.06</td></tr></table>

# Parainfluenza Virus 3

LoD Confirmation Results for Parainfluenza Virus 3 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=7>Parainfluenza Virus 3</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 3)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.4</td><td rowspan=1 colspan=1>23.17 ± 2.12</td></tr><tr><td rowspan=1 colspan=1>1(0.1 x LoD)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>24.57 ± 2.20</td></tr></table>

# Respiratory Syncytial Virus A

<table><tr><td rowspan=1 colspan=7>LoD Confirmation Results for Respiratory Syncytial Virus A (Primary Strain for the LoD Study)</td></tr><tr><td rowspan=1 colspan=7>Respiratory Syncytial Virus A</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 2)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>23.11 ± 1.43</td></tr><tr><td rowspan=1 colspan=1>0.2(0.1 x LoD)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>35%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>26.45 ± 1.38</td></tr></table>

# LoD Testing Results for Respiratory Syncytial Virus B (Alternate strain for the LoD study)

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp ± Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>Single Spike DilutionSeries</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.3</td><td rowspan=1 colspan=1>23.59 ± 2.67</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>25.51 ± 2.31</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>27.07 ± 0.06</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

In conclusion, confirmed LoD concentrations for each of the verified and quantified organisms listed are presented below. Two strains are listed for Influenza A/H1N1, Influenza A/H3N2, and Influenza B; the strain in bold is the primary strain:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Concentration</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Serotype 1 (Species C)</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical Specimen #6213</td><td rowspan=1 colspan=1>1.9 x 106 RNA copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NR-470</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-16, IA10-2003 (Type A1)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>1 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>200 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>2,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr></table>

# e. Analytical Reactivity:

The analytical inclusivity study was performed to determine whether the FilmArray RP System is able to detect a variety of strains (inclusivity panel) that represent the temporal and geographic diversity of each of RP target organism. This study expanded upon the limit of detection studies by determining whether different strains of the same organism can be detected at similar concentrations. (Note: Some differences in sensitivity may be expected as a result of sequence diversity within the gene targeted by the RP assay primers as well as differences in quantification of organisms based on infectivity $( \mathrm { T C I D } _ { 5 0 } ,$ ) rather than nucleic acid concentration.)

This study involves testing panels of various inclusivity strains organisms spiked into simulated NPS specimens using the FilmArray Respiratory Panel (RP) System. Inclusivity organisms were initially tested near the limit of detection (LoD) for each organism as determined in the LoD study. If a specimen containing a particular strain was positive (detected) at the LoD level, no further testing of that strain was required. If a strain was not detected at the LoD concentration, the strain was retested at the LoD level and 10-fold higher. If necessary, the same approach was followed until a positive result was obtained or the maximum concentration possible for that strain stock had been tested.

The inclusivity panel represents the evolutionary, temporal, and geographical diversity of the RP analytes. For example, the inclusivity panel includes 17 different Adenovirus serotypes representing species A, B, C, D, E and F, over 20 Influenza A strains (subtypes H3, H1 and $2 0 0 9 \mathrm { H } 1$ ) isolated from around the world as many as 77 years

ago (1933) and as recently as 2009, and 25 strains of the group that comprised of Human Rhinovirus A and B, Enterovirus, Echovirus, and Coxsackievirus. However, for some analytes, the inclusivity panel is restricted to only 1 or 2 strains because of limited strain diversity and availability.

For inclusivity (reactivity) testing, a total of 99 individual RP target organism strains were tested and accurately identified with the FilmArray RP system. Each of the 99 strains tested in this study were reactive with the FilmArray RP system. The sensitivity of detection for Adenovirus C serotype 6 was 10,000-fold less than the system LoD due to sequence variation (mismatches) within the primers of the second stage PCR reaction. The limited detection capability of the FilmArray RP system for this Adenovirus serotype was described in the FilmArray RP intended use as “The FilmArray RP detects Adenovirus species $c$ serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., cell culture).”

Summary tables of Analytical Reactivity (Inclusivity) Testing with the FilmArray RP System are presented below:

(Note: With a few exceptions as indicated in the tables, all organism stocks were obtained from Zeptometrix, ATCC, BEI Resources and the University of Iowa Center for Emerging Infectious Disease (CEID). These organism stocks were grown, quantified and verified by Zeptometrix Corporation (Buffalo, NY) prior to testing with the FilmArray RP system.)

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=19 colspan=1>Adenovirus</td><td rowspan=1 colspan=2>A</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=8 colspan=2>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7d2 (Iowa/2001)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7h (Iowa/1999)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>11 (Wisconsin/2005)</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>14 (Missouri/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>x</td></tr><tr><td rowspan=1 colspan=1>21 (Missouri/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>34 (Texas/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=6 colspan=2>C</td><td></td><td></td><td></td></tr><tr><td rowspan=2 colspan=1>′</td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>2 (New York/2004)</td><td rowspan=1 colspan=1>30,000</td><td rowspan=1 colspan=1>100x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>6 (Colorado/2005)a</td><td rowspan=1 colspan=1>3,000,00</td><td rowspan=1 colspan=1>10,000x</td></tr><tr><td rowspan=1 colspan=2>D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=2 colspan=2>E</td><td rowspan=1 colspan=1>4a (S Carolina/2004)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>4p3 (New Jersey/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=2>F</td><td rowspan=1 colspan=1>41 (Indiana/2004)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr></table>

a Alignment of the FilmArray RP primers with Adenovirus serotype 6 sequences demonstrated that there are a number of mismatches between the serotype 6 sequence of the Hexon gene and both outer and inner primers contained in the FilmArray RP pouch. It was found that the mismatches do not affect the efficiency of first stage PCR (PCR1) but that there is at least a 3000- fold reduction in amplification efficiency (relative to serotype 1) for the second stage (PCR2) reaction with the serotype 6 template.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Strain / Isolate</td><td rowspan=2 colspan=1>Concentration Detected</td><td rowspan=2 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Type</td></tr><tr><td rowspan=4 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>Clinical Sample #1120</td><td rowspan=1 colspan=1>2.08 x 106RNA copies/mL</td><td rowspan=1 colspan=1>1.1x</td></tr><tr><td rowspan=3 colspan=1>NL63</td><td rowspan=1 colspan=1>Clinical Sample# 6123 (Lot #304842)</td><td rowspan=1 colspan=1>1.41 x 104 TCID50 /mL(~1.9 x 106 copies/mL)</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample #6213Type B</td><td rowspan=1 colspan=1>1.9 x 106RNA copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>NR-470 (BEI)</td><td rowspan=1 colspan=1>50TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=10 colspan=1>HumanMetapneumovirus</td><td rowspan=2 colspan=1>Al</td><td rowspan=1 colspan=1>9(IA3-2002)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>16(IA10-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20(IA14-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>27(IA27-2004)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=4 colspan=1>B1</td><td rowspan=1 colspan=1>3(Peru2-2002)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5(Peru3-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>13(IA7-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>18(IA18-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>B2</td><td rowspan=1 colspan=1>8(Peru6-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>22(IA16-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr></table>

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration Detected(TCID50/ml)</td><td colspan="1" rowspan="1">Multiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="11">Enterovirus</td><td colspan="1" rowspan="3">A</td><td colspan="1" rowspan="1">Coxsackievirus A10ATCC VR-168</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71ATCC VR-1432</td><td colspan="1" rowspan="1">1:30,000dilution of stock</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="6">B</td><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">300,000</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="2">C</td><td colspan="1" rowspan="1">Coxsackievirus A21 /KuykendallATCC VR-850</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24 DN-19ATCC VR-583</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Db</td><td colspan="1" rowspan="1">Enterovirus 68 (F02-3607 corn)ATCC VR-1197</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="14">Human Rhinovirus</td><td colspan="1" rowspan="8">A</td><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A2 (HGP)ATCC VR-482</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A7 (68-CV11)ATCC VR-1601</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A16 (11757)ATCC VR-283</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A34 (137-3)ATCC VR-507</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A57 (Ch47)ATCC VR-1600a</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="1">A77 (130-63)ATCC VR-1187</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A85 (50-525-CV54)ATCC VR-1195</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="6">B</td><td colspan="1" rowspan="1">B3 (FEB)ATCC VR-483</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B14 (1059)ATCC VR-284</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B17 (33342)ATCC-282a</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="1">B27 (5870)ATCC VR-1137</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B42 (56822)ATCC VR-338</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">B83 (Baylor 7)ATCC VR-1193</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr></table>

a The 5’UTR sequences of these strains were aligned with the outer and inner PCR primers for the HRV assays to determine whether sequence mis-matches may be contributing to the somewhat reduced sensitivity of the FilmArray RP system for these strains. Primers for at least one of the 4 HRV assays (inner primers) aligned very well with published sequence for each strain ( $0 - 1$ mismatches in the primer region), suggesting that the observed differences in sensitivity are likely attributable to inconsistencies in quantification of the viral stock rather than the sensitivity of the assay(s) themselves. b Enterovirus species $\mathrm { D }$ is also referred to as Human Rhinovirus species C.

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration Detected(TCID50/mL)</td><td colspan="1" rowspan="1">Multiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="10">Influenza A (H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Hawaii/15/01CDC#2001701117</td><td colspan="1" rowspan="1">1:300 dilution of stock</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57ATCC VR-546</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54ATCC VR-98</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47ATCC VR-97</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43ATCC VR-96</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34ATCC VR-95</td><td colspan="1" rowspan="1">2000</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33ATCC VR-219</td><td colspan="1" rowspan="1">200</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Influenza A (2009 H1N1)</td><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Swine NY/02/2009</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="9">Influenza A (H3N2)</td><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/NewYork/55/2005CDC#2005705561</td><td colspan="1" rowspan="1">1:300,000 dilution of stock</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75ATCC VR-822</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73ATCC VR-810</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68ATCC VR-547</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68ATCC VR-544</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Alice (vaccine) A/England/42/72ATCC VR-776</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">MRC-2 Recombinant strainATCC VR-777</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1x</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/ml)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=11 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06Z-0810037CF</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005CDC#2005743348</td><td rowspan=1 colspan=1>1:3,000,000dilution of stock</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72ATCC VR-823</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62ATCC VR-295</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59ATC VR-296</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54ATCC VR-103</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45ATCC VR-102</td><td rowspan=1 colspan=1>6,000EID50/mL</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40ATCC VR-101</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit RecombinantATCC VR-786</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoD</td></tr><tr><td rowspan=3 colspan=1>Parainfluenza</td><td rowspan=3 colspan=1>PIV3</td><td rowspan=1 colspan=1>Zeptometrix#0810016CF</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>C-243ATCC VR-93</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=1 colspan=1>NIH 47885BEI NR-3233</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10x</td></tr></table>

<table><tr><td></td><td>Type</td><td>Strain</td><td>Concentration Detected (TCID50/mL)</td><td>Multiple of LoD Detected</td></tr><tr><td colspan="1" rowspan="6">Respiratory SyncytialVirus</td><td colspan="1" rowspan="3">A</td><td colspan="1" rowspan="1">Zeptometrix#0810040ACF</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/A2ATCC VR-1540</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A/LongATCC VR-26</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="3">B</td><td colspan="1" rowspan="1">B/9320ATCC VR-955</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B/Wash18537/62ATCC VR-1580</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B/WV/14617/85ATCC VR-1400</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">1x</td></tr></table>

# Supplemental Adenovirus Reactivity Information (clinical study data and in silico analyses):

A subset of prospective and retrospective archived clinical specimens that were FilmArray positive for Adenovirus was subjected to PCR and bi-directional sequence analysis. BLAST analysis of the sequence data obtained for a region of the Adenovirus polymerase gene identified adenovirus species B (serotypes 3, $3 { + } 1 1 _ { \mathrm { l } }$ p, 7, 16 and 21), species C (serotypes 1, 2, and 5), and species E (serotype 4) in these clinical specimens. Species B, C and E are the most common Adenovirus species associated with respiratory illness. Species A, D, and F (often associated with conjunctivitis and gastroenteritis) were not identified in clinical specimens,

In addition to laboratory testing, bioinformatics resources were also used to predict reactivity of additional Adenovirus species and serotypes with the FilmArray RP System. Simulated reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). Table below lists the adenovirus types that were not tested by the FilmArray system either in analytical or clinical testing. Bioinformatics analysis of these adenovirus sequences indicate that the FilmArray RP system is likely to react with all Adenovirus species and serotypes.

Simulated FilmArray RP reactivity with untested adenovirus serotypes   

<table><tr><td rowspan=1 colspan=3>Organism</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=11 colspan=3>Humanadenovirus</td><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>AB330093</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>DQ149610</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>AB330097</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>AB052912</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>DQ149643</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>AB330090</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>DQ149615</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=2 colspan=1></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>DQ149616</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>15</td><td rowspan=2 colspan=1>DQ149617</td><td rowspan=2 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=8 colspan=3>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>AB330098</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>DQ149618</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>DQ149619</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>DQ149620</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>DQ149621</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>DQ149622</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>DQ149623</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>DQ149624</td><td rowspan=1 colspan=1>Positive</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=22 colspan=1></td><td rowspan=20 colspan=1></td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>DQ149625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>DQ149626</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>DQ149627</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>DQ149628</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>DQ149629</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>DQ149630</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>DQ149631</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>DQ149632</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>DQ149633</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>DQ149634</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>DQ149635</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>DQ149636</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>DQ149637</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>DQ149638</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>DQ149639</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>DQ149640</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>AB330129</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>DQ149641</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>DQ149642</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>FJ169625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>AB330121</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>DQ923122</td><td rowspan=1 colspan=1>Positive</td></tr></table>

# Supplemental Human Rhinovirus/Enterovirus Reactivity Information (clinical study data and in silico analyses):

In addition to the analytical inclusivity testing, BLAST analysis was performed on sequence data ( $\zeta ^ { \gamma }$ UTR) obtained from prospective and retrospective archived clinical specimens that were FilmArray positive for Human Rhinovirus/Enterovirus. The following species and subtypes were identified in clinical specimens:

Enterovirus Species A:

Enterovirus Species B:

Rhinovirus Species A:

Enterovirus serotype 71   
Coxsackievirus A2 and A6   
Echovirus serotypes 3, 6, 7, 11, 15, 21 and 30   
Coxsackievirus B1, B3 and A9   
Enterovirus serotypes 81 and 88   
Human Rhinovirus serotypes 1B, 8 ,9 ,10, 13, 19, 21,   
22, 23, 28, 30, 32, 34, 38, 39, 40, 46, 47, 49, 51, 54,   
56, 58, 59, 61, 62, 66, 68, 75, 77, 78, 80, 82, 98 and   
100   
Human Rhinovirus serotypes 27, 69, 83 and 91   
At least 3 individual strains and 12 distinct isolates\*

Rhinovirus Species B: Rhinovirus Species C:

\* Human Rhinovirus species C (also known as Enterovirus species D) has not been classified into serotypes.

Simulated reactivity of the FilmArray RP Human Rhinovirus/Enterovirus assays with Enterovirus Species C Poliovirus strains was generated using a bioinformatics approach. Alignment of assay primer sequences with the GenBank sequences indicates the FilmArray RP assay is likely to react with Poliovirus types 1, 2 and 3, giving a Human Rhinovirus/Enterovirus result.

Simulated reactivity of the FilmArray RP HRV and Entero assays with Poliovirus sequences (Enterovirus Species C)   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArray Result</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 1 strainCHN8264c/GZ/CHN/2004</td><td rowspan=1 colspan=1>FJ769385</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 2, complete genome</td><td rowspan=1 colspan=1>AY177685</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 3 strain IRA10853,complete genome</td><td rowspan=1 colspan=1>EU684056</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr></table>

# Supplemental Reactivity Information for Influenza Strains of Human, Swine and Avian Origin (analytical testing data and in silico analyses):

Additional analytical reactivity testing was carried out with three swine and avian isolates of Influenza A. The results are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate / Strain</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>FilmArrayResult</td></tr><tr><td rowspan=2 colspan=1>Swine</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>Influenza A/Swine/1976/31</td><td rowspan=1 colspan=1>~1 x 106EID50/mL</td><td rowspan=1 colspan=1>Influenza A H1 a</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>Influenza A/Swine/Iowa/15/30</td><td rowspan=1 colspan=1>~1 x 107EID50/mL</td><td rowspan=1 colspan=1>Influenza A H1 a</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>Kilbourne F63A/NWS/34 (HA) xA/Rockefeller Institute/5/57(NA)</td><td rowspan=1 colspan=1>14.8 ng RNA b</td><td rowspan=1 colspan=1>Influenza A H1</td></tr></table>

a No reactivity was observed with the $2 0 0 9 \mathrm { H } 1$ subtyping assay. b Purified and quantified RNA from avian influenza culture was obtained from BEI Resources.

Simulated reactivity of the FilmArray RP Influenza assays with additional influenza strains of human, swine and avian origin was generated using a bioinformatics approach. Assay primer sequences were aligned with GenBank sequences corresponding to the appropriate gene targets and reactivity were predicated based on the number and location of mismatches in the targeted region. For each strain, multiple (3) GenBank IDs were evaluated, corresponding to the gene segments targeted by the FilmArray RP assays (matrix (MA), non-structural (NS) and hemagglutinin(HA)). The strains listed in the following table are predicted to react with the pan-influenza A and H1, H3 or $2 0 0 9 \mathrm { H } 1$ subtyping assays as indicated:

Simulated reactivity of FilmArray Influenza A assays with human, swine, and avian influenza strains   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArrayReactivity</td></tr><tr><td rowspan=9 colspan=1>Human</td><td rowspan=3 colspan=1>H1N1</td><td rowspan=3 colspan=1>A/California/UR06-0393/2007(H1N1)</td><td rowspan=1 colspan=1>CY026540</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY026543</td></tr><tr><td rowspan=1 colspan=1>CY026539</td></tr><tr><td rowspan=3 colspan=1>H1N1-2009</td><td rowspan=3 colspan=1>A/Aalborg/INS133/2009(H1N1)</td><td rowspan=1 colspan=1>CY063606</td><td rowspan=3 colspan=1>Influenza A2009 H1</td></tr><tr><td rowspan=1 colspan=1>CY063610</td></tr><tr><td rowspan=1 colspan=1>CY063607</td></tr><tr><td rowspan=3 colspan=1>H1N2</td><td rowspan=3 colspan=1>A/New York/297/2003(H1N2)</td><td rowspan=1 colspan=1>CY002668</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY002665</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>CY002664</td></tr></table>

<table><tr><td rowspan=25 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>Simulated FilmMArrayReactivity</td></tr><tr><td rowspan=3 colspan=1>H2N2</td><td rowspan=3 colspan=1>A/Albany/20/1957(H2N2)</td><td rowspan=1 colspan=1>CY022013</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY022014</td></tr><tr><td rowspan=1 colspan=1>CY022017</td></tr><tr><td rowspan=3 colspan=1>H3N2</td><td rowspan=3 colspan=1>A/Boston/38/2008(H3N2)</td><td rowspan=1 colspan=1>CY044581</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY044584</td></tr><tr><td rowspan=1 colspan=1>CY044580</td></tr><tr><td rowspan=6 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/Cambodia/R0405050/2007(H5N1)</td><td rowspan=1 colspan=1>HQ200572</td><td rowspan=6 colspan=1>Influenza A(no subtypeddetected</td></tr><tr><td rowspan=1 colspan=1>HQ200573</td></tr><tr><td rowspan=1 colspan=1>FJ225472</td></tr><tr><td rowspan=3 colspan=1>A/Hong Kong/486/97(H5N1)</td><td rowspan=1 colspan=1>AF084281</td></tr><tr><td rowspan=1 colspan=1>AF255368</td></tr><tr><td rowspan=1 colspan=1>AF115289</td></tr><tr><td rowspan=3 colspan=1>H7N2</td><td rowspan=3 colspan=1>A/New York/107/2003(H7N2)</td><td rowspan=1 colspan=1>EU587368</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>EU587373</td></tr><tr><td rowspan=1 colspan=1>EU587374</td></tr><tr><td rowspan=3 colspan=1>H7N3</td><td rowspan=3 colspan=1>A/Canada/rv504/2004(H7N3)</td><td rowspan=1 colspan=1>CY015006</td><td rowspan=3 colspan=1>Influenza A(no subtypedtected</td></tr><tr><td rowspan=1 colspan=1>CY015007</td></tr><tr><td rowspan=1 colspan=1>CY015010</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=3 colspan=1>A/Netherlands/219/03(H7N7)</td><td rowspan=1 colspan=1>AY340089</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>AY342422</td></tr><tr><td rowspan=1 colspan=1>AY338459</td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=3 colspan=1>A/Hong Kong/1073/99(H9N2)</td><td rowspan=1 colspan=1>AJ404626</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>AJ278647</td></tr><tr><td rowspan=1 colspan=1>AJ278649</td></tr><tr><td rowspan=12 colspan=1>Swine</td><td rowspan=3 colspan=1>H1N1</td><td rowspan=3 colspan=1>A/swine/Wisconsin/1/1971(H1N1)</td><td rowspan=1 colspan=1>CY022414</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>CY022417</td></tr><tr><td rowspan=1 colspan=1>CY022413</td></tr><tr><td rowspan=6 colspan=1>H1N2</td><td rowspan=3 colspan=1>A/swine/Hong Kong/NS857/2001(H1N2)</td><td rowspan=1 colspan=1>GQ229348</td><td rowspan=3 colspan=1>Influenza A H1</td></tr><tr><td rowspan=1 colspan=1>GQ229350</td></tr><tr><td rowspan=1 colspan=1>GQ229347</td></tr><tr><td rowspan=3 colspan=1>A/swine/Sweden/1021/2009(H1N2)</td><td rowspan=1 colspan=1>GQ495135</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ495136</td></tr><tr><td rowspan=1 colspan=1>GQ495132</td></tr><tr><td rowspan=3 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/swine/East Java/UT6010/2007(H5N1)</td><td rowspan=1 colspan=1>HM440124</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>HM440111</td></tr><tr><td rowspan=1 colspan=1>HM440123</td></tr><tr><td rowspan=15 colspan=1>Avian</td><td rowspan=9 colspan=1>H2N2</td><td rowspan=3 colspan=1>A/chicken/New York/13828-3/1995(H2N2)</td><td rowspan=1 colspan=1>CY014822</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY014825</td></tr><tr><td rowspan=1 colspan=1>CY014821</td></tr><tr><td rowspan=3 colspan=1>A/Japan/305/1957(H2N2)</td><td rowspan=1 colspan=1>CY045804</td><td rowspan=3 colspan=1>Influenza A(no subtypeddttected)</td></tr><tr><td rowspan=1 colspan=1>CY014977</td></tr><tr><td rowspan=1 colspan=1>CY014980</td></tr><tr><td rowspan=3 colspan=1>A/Korea/426/1968(H2N2)</td><td rowspan=1 colspan=1>CY031595</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY031596</td></tr><tr><td rowspan=1 colspan=1>CY031599</td></tr><tr><td rowspan=3 colspan=1>H3N1</td><td rowspan=3 colspan=1>A/blue-winged teal/ALB/452/1983(H3N1)</td><td rowspan=1 colspan=1>CY004635</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY004638</td></tr><tr><td rowspan=1 colspan=1>CY005940</td></tr><tr><td rowspan=3 colspan=1>H3N2</td><td rowspan=3 colspan=1>A/American black duck/North Carolina/675-075/2004(H3N2)</td><td rowspan=1 colspan=1>GU051135</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GU051136</td></tr><tr><td rowspan=1 colspan=1>GU051137</td></tr></table>

f. Analytical Specificity/Cross-reactivity Evaluation:   

<table><tr><td rowspan=49 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmMarrayReactivity</td></tr><tr><td rowspan=3 colspan=1>H3N5</td><td rowspan=3 colspan=1>A/mallard/Netherlands/2/1999(H3N5)</td><td rowspan=1 colspan=1>CY060261</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY060264</td></tr><tr><td rowspan=1 colspan=1>CY060265</td></tr><tr><td rowspan=3 colspan=1>H3N6</td><td rowspan=3 colspan=1>A/American black duck/NewBrunswick/25182/2007(H3N6)</td><td rowspan=1 colspan=1>CY047696</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY047697</td><td rowspan=1 colspan=1>(</td></tr><tr><td rowspan=1 colspan=1>CY047700</td></tr><tr><td rowspan=3 colspan=1>H3N7</td><td rowspan=3 colspan=1>A/northernshoveler/California/HKWF1367/2007(H3N7)</td><td rowspan=1 colspan=1>CY033372</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY033375</td></tr><tr><td rowspan=1 colspan=1>CY033376</td></tr><tr><td rowspan=3 colspan=1>H3N8</td><td rowspan=3 colspan=1>A/American blackduck/Washington/699/1978(H3N8)</td><td rowspan=1 colspan=1>GU052299</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>GU052302</td></tr><tr><td rowspan=1 colspan=1>GU052300</td></tr><tr><td rowspan=3 colspan=1>H4N6</td><td rowspan=3 colspan=1>A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td><td rowspan=1 colspan=1>CY063977</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY063978</td></tr><tr><td rowspan=1 colspan=1>CY063981</td></tr><tr><td rowspan=9 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/rook/Rostov-on-Don/26/2007(H5N1)</td><td rowspan=1 colspan=1>EU814503</td><td rowspan=3 colspan=1>Influenza A(no subtypedtected</td></tr><tr><td rowspan=1 colspan=1>EU814504</td></tr><tr><td rowspan=1 colspan=1>EU814507</td></tr><tr><td rowspan=3 colspan=1>A/turkey/VA/505477-18/2007(H5N1)</td><td rowspan=1 colspan=1>GU186509</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GU186510</td></tr><tr><td rowspan=1 colspan=1>GU186513</td></tr><tr><td rowspan=3 colspan=1>A/chicken/Bangladesh/1151-10/2010(H5N1)</td><td rowspan=1 colspan=1>HQ156765</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>HQ156766</td></tr><tr><td rowspan=1 colspan=1>HQ156764</td></tr><tr><td rowspan=3 colspan=1>H5N2</td><td rowspan=3 colspan=1>A/duck/Pennsylvania/10218/1984(H5N2)</td><td rowspan=1 colspan=1>AB295603</td><td rowspan=3 colspan=1>Influenza A(no subtypedetectedd</td></tr><tr><td rowspan=1 colspan=1>AB286120</td></tr><tr><td rowspan=1 colspan=1>AB286652</td></tr><tr><td rowspan=3 colspan=1>H5N3</td><td rowspan=3 colspan=1>A/duck/Singapore/F119/3/1997(H5N3)</td><td rowspan=1 colspan=1>GU052802</td><td rowspan=3 colspan=1>Influenza A(no subtypedtected</td></tr><tr><td rowspan=1 colspan=1>GU052803</td></tr><tr><td rowspan=1 colspan=1>GU052805</td></tr><tr><td rowspan=3 colspan=1>H6N1</td><td rowspan=3 colspan=1>A/duck/PA/486/1969(H6N1)</td><td rowspan=1 colspan=1>EU743286</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>EU743287</td></tr><tr><td rowspan=1 colspan=1>EU743289</td></tr><tr><td rowspan=3 colspan=1>H6N2</td><td rowspan=3 colspan=1>A/mallard/Czech Republic/15902-17K/2009(H6N2)</td><td rowspan=1 colspan=1>HQ244430</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>HQ244433</td></tr><tr><td rowspan=1 colspan=1>HQ244434</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=3 colspan=1>A/mallard/Korea/GH171/2007(H7N7)</td><td rowspan=1 colspan=1>FJ750872</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>FJ959087</td></tr><tr><td rowspan=1 colspan=1>FJ959090</td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=3 colspan=1>A/turkey/Wisconsin/1/1966(H9N2)</td><td rowspan=1 colspan=1>CY014663</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>CY014664</td></tr><tr><td rowspan=1 colspan=1>CY014667</td></tr><tr><td rowspan=3 colspan=1>H10N7</td><td rowspan=3 colspan=1>A/chicken/Germany/N/1949(H10N7)</td><td rowspan=1 colspan=1>GQ176136</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>GQ176135</td></tr><tr><td rowspan=1 colspan=1>GQ176132</td></tr><tr><td rowspan=3 colspan=1>H11N9</td><td rowspan=3 colspan=1>A/duck/Memphis/546/1974(H11N9)</td><td rowspan=1 colspan=1>CY014691</td><td rowspan=3 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ257441</td></tr><tr><td rowspan=1 colspan=1>CY014687</td></tr></table>

An analytical exclusivity study was carried out to assess the potential for false positive results due to cross-reactivity between RP assays and other RP or non-RP organisms. RP organisms and non-RP organisms were tested at high concentrations and the non-RP organism exclusivity panel consisted of organisms that are related to, but distinct from, RP target organisms, or that could be present in specimens collected from the intended test population.

Exclusivity (cross-reactivity) testing was performed at Idaho Technology, Inc. and an external testing site (USAMRIID – United States Army Medical Research Institute for Infectious Diseases). Samples were prepared by spiking RP and non-RP target organisms into NPS (or simulated NPS) sample matrix at a high concentration ( $1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viral targets and $1 0 ^ { 6 } \mathrm { C F U / m i }$ for bacterial targets, or the highest concentration possible based on the organism stock concentration).

The RP target exclusivity panel included all of the primary strains used to establish LoD levels for the system plus additional strains of some common organisms such as Influenza A and B. RP target organisms tested at high concentrations were positive only for the appropriate RP assay(s). No crossreactivity between RP assays was observed.

The following tables list all of the RP organisms tested at high concentration. The test concentration is listed followed by the multiple of the LoD concentration the test concentration represents.

Summary of Cross-Reactivity Testing with FilmArray RP Target Organisms   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Type / Strain / ID</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Multiple ofLoD Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Serotype 1(Species C)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">333 x</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">HKU1Clinical specimen</td><td colspan="1" rowspan="1">2.78E+09 copies/mL</td><td colspan="1" rowspan="1">1,463 x</td></tr><tr><td colspan="1" rowspan="1">NL63NR-470</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,134 x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">hMPV-16IA10-2003 A1</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">4,085 x</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">3.40E+06 TCID50/mL</td><td colspan="1" rowspan="1">113 x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">5,670 x</td></tr><tr><td colspan="1" rowspan="7">Influenza A H1N1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">500 x</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">500 x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td><td colspan="1" rowspan="1">5,000 x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">70 x</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24 x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">70 x</td></tr><tr><td colspan="1" rowspan="1">Influenza A2009 H1N1</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">4.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">4,000 x</td></tr><tr><td colspan="1" rowspan="8">Influenza A H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1634 x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">940 x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,134 x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">20,000 x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">20,000 x</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">940 x</td></tr><tr><td colspan="1" rowspan="1">A/MRC 2</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,634 x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,634 x</td></tr><tr><td colspan="1" rowspan="9">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td><td colspan="1" rowspan="1">278 x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">5.03E+04 TCID50/mL</td><td colspan="1" rowspan="1">838 x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">1,667 x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">95 x</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">1,667 x</td></tr><tr><td colspan="1" rowspan="1">B/HongKong/5/72</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">10,000 x</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">6,950 x</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">2.14E+04 TCID50/mL</td><td colspan="1" rowspan="1">10,700 x</td></tr></table>

The non-RP target exclusivity panel consisted of 45 bacteria, 12 viruses, and 1 fungus (Candida albicans). The organisms included in the non-RP exclusivity panel were selected either because they are related to RP target organisms, are clinically relevant (colonize the upper respiratory tract or cause respiratory symptoms), are common skin flora or laboratory contaminants, or are microorganisms for which much of the population may have been infected (Herpes Simplex Virus, etc.). None of the organisms in the non-RP target exclusivity panel were tested positive by the FilmArray RP System assays except for one measles virus strain. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

The following table lists all of the non-RP organisms tested at high concentration. The test concentration is listed:

<table><tr><td colspan="1" rowspan="1">Bacterium</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">clinical isolate</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">E431</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 8467</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 51445</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC BAA-589</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 9340</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC 10380</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC BAA-1335</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix#0801504</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">2.42E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">1.00E+06 IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC 14779CDC [NCTC 10681]</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">O157:H7</td><td colspan="1" rowspan="1">1.00+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">8.67E+04 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeacheae</td><td colspan="1" rowspan="1">Long Beach 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">Philadelphia Strain</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11- type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114PG21</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">1.88E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15531</td><td colspan="1" rowspan="1">4.27E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15377</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 15492</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 29342</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 39505</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 700825</td><td colspan="1" rowspan="1">1.05E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M1027-type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix#0801512</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">8.43E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618T-strain 960</td><td colspan="1" rowspan="1">5.23E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="2">Bocavirus</td><td colspan="1" rowspan="2">Type 1Clinical Specimen</td><td colspan="1" rowspan="1">3.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">4.67E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="2">Coronavirus</td><td colspan="1" rowspan="1">229EATCC VR-740</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">OC43ATCC VR-759</td><td colspan="1" rowspan="1">7.30E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="3">ParainfluenzaVirus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169ATCC VR-538</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes Virus</td><td colspan="1" rowspan="1">Simplex Type 1</td><td colspan="1" rowspan="1">1:40 dilution of stock</td></tr><tr><td colspan="1" rowspan="2">Measles Virus(Rubeola)</td><td colspan="1" rowspan="1">Zeptometrix# 0810025CFa</td><td colspan="1" rowspan="1">4.20E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">1.00E+05 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix# 0810079CF</td><td colspan="1" rowspan="1">5.03E+04 TCID50/mL</td></tr></table>

a This stock produced one false positive Adenovirus result. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

Additional analytical exclusivity testing was carried out with either live isolates or purified genomic RNA of avian host influenza A strains with the following results:

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate / Strain</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>FilmArrayResult</td></tr><tr><td rowspan=8 colspan=1>Avian</td><td rowspan=2 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Japan/305/57</td><td rowspan=1 colspan=1>3.3 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>Kilbourne F38A/Korea/426/68 (HA, NA) x A/Puerto Rico/8/34</td><td rowspan=1 colspan=1>6.3 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Vietnam/1203/2004 R-H5</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>A/duck/Pennsylvania/10218/84</td><td rowspan=1 colspan=1>2.5 ng RNA a</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>Kilbourne F181A/duck/Singapore/645/97</td><td rowspan=1 colspan=1>247 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H7N3</td><td rowspan=1 colspan=1>A/Mallard/Netherlands/12/2000</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/NewYork/107/2003</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>A/chicken/Germany/N/49</td><td rowspan=1 colspan=1>68 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr></table>

a Purified and quantified RNA from avian influenza cultures was obtained from BEI Resources b Stock virus HA titer from $\mathrm { C D C } = 1 2 8$ . Twenty microliters of virus stock tested.

Additional analytical testing was also performed using a SARS coronavirus nucleic acid sample obtained from Zeptometrix, Corp. (Buffalo, NY) with the following result:

<table><tr><td rowspan=1 colspan=1>Strain / Test Sample</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>FilmArray RP Result</td></tr><tr><td rowspan=1 colspan=1>SARS Coronavirus(VIBE prep from Zeptometrix)</td><td rowspan=1 colspan=1>unknown</td><td rowspan=1 colspan=1>Negative for all assays</td></tr></table>

# Supplemental Human, Swine, and Avian Host Influenza A Virus Strains Exclusivity Information (in silico analyses):

Laboratory testing of influenza strains was supplemented with in silico predictions of exclusivity using bioinformatics and sequence alignments between FilmArray RP assay primers and GenBank sequences for the strains shown in the table below. Exclusivity was predicated based on the number of and location of mismatches between assay primers and available strain sequences. For each strain, multiple GenBank IDs were evaluated, corresponding to the genes targeted by the FilmArray RP pan-influenza A and HA subtyping assays. All listed strains are predicted to produce an Influenza A (no subtype detected) FilmArray RP result.

Simulated exclusivity of FilmArray Influenza A assays with human, swine, and avian influenza strains   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>PredicatedFilmArray RPResults</td></tr><tr><td rowspan=21 colspan=1>Human</td><td rowspan=3 colspan=1>H2N2</td><td rowspan=1 colspan=1>CY022013</td><td rowspan=3 colspan=1>A/Albany/20/1957(H2N2)</td><td rowspan=21 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY022014</td></tr><tr><td rowspan=1 colspan=1>CY022017</td></tr><tr><td rowspan=6 colspan=1>H5N1</td><td rowspan=1 colspan=1>HQ200572</td><td rowspan=3 colspan=1>A/Cambodia/R0405050/2007(H5N1)</td></tr><tr><td rowspan=1 colspan=1>HQ200573</td></tr><tr><td rowspan=1 colspan=1>FJ225472</td></tr><tr><td rowspan=1 colspan=1>AF084281</td><td rowspan=3 colspan=1>A/Hong Kong/486/97(H5N1)</td></tr><tr><td rowspan=1 colspan=1>AF255368</td></tr><tr><td rowspan=1 colspan=1>AF115289</td></tr><tr><td rowspan=3 colspan=1>H7N2</td><td rowspan=1 colspan=1>EU587368</td><td rowspan=3 colspan=1>A/New York/107/2003(H7N2)</td></tr><tr><td rowspan=1 colspan=1>EU587373</td></tr><tr><td rowspan=1 colspan=1>EU587374</td></tr><tr><td rowspan=3 colspan=1>H7N3</td><td rowspan=1 colspan=1>CY015006</td><td rowspan=3 colspan=1>A/Canada/rv504/2004(H7N3)</td></tr><tr><td rowspan=1 colspan=1>CY015007</td></tr><tr><td rowspan=1 colspan=1>CY015010</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=1 colspan=1>AY340089</td><td rowspan=3 colspan=1>A/Netherlands/219/03(H7N7)</td></tr><tr><td rowspan=1 colspan=1>AY342422</td></tr><tr><td rowspan=1 colspan=1>AY338459</td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=1 colspan=1>AJ404626</td><td rowspan=3 colspan=1>A/Hong Kong/1073/99(H9N2)</td></tr><tr><td rowspan=1 colspan=1>AJ278647</td></tr><tr><td rowspan=1 colspan=1>AJ278649</td></tr><tr><td rowspan=6 colspan=1>Swine</td><td rowspan=3 colspan=1>H1N2</td><td rowspan=1 colspan=1>GQ495135</td><td rowspan=3 colspan=1>A/swine/Sweden/1021/2009(H1N2)</td><td rowspan=6 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>GQ495136</td></tr><tr><td rowspan=1 colspan=1>GQ495132</td></tr><tr><td rowspan=3 colspan=1>H5N1</td><td rowspan=1 colspan=1>HM440124</td><td rowspan=3 colspan=1>A/swine/East Java/UT6010/2007(H5N1)</td></tr><tr><td rowspan=1 colspan=1>HM440111</td></tr><tr><td rowspan=1 colspan=1>HM440123</td></tr><tr><td rowspan=25 colspan=1>Avian</td><td rowspan=9 colspan=1>H2N2</td><td rowspan=1 colspan=1>CY014822</td><td rowspan=3 colspan=1>A/chicken/New York/13828-3/1995(H2N2)</td><td rowspan=25 colspan=1>Influenza A(no subtypedetected</td></tr><tr><td rowspan=1 colspan=1>CY014825</td></tr><tr><td rowspan=1 colspan=1>CY014821</td></tr><tr><td rowspan=1 colspan=1>CY045804</td><td rowspan=3 colspan=1>A/Japan/305/1957(H2N2)</td></tr><tr><td rowspan=1 colspan=1>CY014977</td></tr><tr><td rowspan=1 colspan=1>CY014980</td></tr><tr><td rowspan=1 colspan=1>CY031595</td><td rowspan=3 colspan=1>A/Korea/426/1968(H2N2)</td></tr><tr><td rowspan=1 colspan=1>CY031596</td></tr><tr><td rowspan=1 colspan=1>CY031599</td></tr><tr><td rowspan=3 colspan=1>H3N2</td><td rowspan=1 colspan=1>GU051135</td><td rowspan=3 colspan=1>A/American black duck/North Carolina/675-075/2004(H3N2)</td></tr><tr><td rowspan=1 colspan=1>GU051136</td></tr><tr><td rowspan=1 colspan=1>GU051137</td></tr><tr><td rowspan=3 colspan=1>H3N6</td><td rowspan=1 colspan=1>CY047696</td><td rowspan=3 colspan=1>A/American black duck/NewBrunswick/25182/2007(H3N6)</td></tr><tr><td rowspan=1 colspan=1>CY047697</td></tr><tr><td rowspan=1 colspan=1>CY047700</td></tr><tr><td rowspan=3 colspan=1>H4N6</td><td rowspan=1 colspan=1>CY063977</td><td rowspan=3 colspan=1>A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td></tr><tr><td rowspan=1 colspan=1>CY063978</td></tr><tr><td rowspan=1 colspan=1>CY063981</td></tr><tr><td rowspan=7 colspan=1>H5N1</td><td rowspan=1 colspan=1>EU814503</td><td rowspan=3 colspan=1>A/rook/Rostov-on-Don/26/2007(H5N1)</td></tr><tr><td rowspan=1 colspan=1>EU814504</td></tr><tr><td rowspan=1 colspan=1>EU814507</td></tr><tr><td rowspan=1 colspan=1>GU186509</td><td rowspan=3 colspan=1>A/turkey/VA/505477-18/2007(H5N1)</td></tr><tr><td rowspan=1 colspan=1>GU186510</td></tr><tr><td rowspan=1 colspan=1>GU186513</td></tr><tr><td rowspan=1 colspan=1>HO156765</td><td rowspan=1 colspan=1>A/chicken/Bangladesh/1151-10/2010(H5N1)</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>HQ156766</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HQ156764</td></tr><tr><td rowspan=3 colspan=1>H5N2</td><td rowspan=1 colspan=1>AB295603</td><td rowspan=3 colspan=1>A/duck/Pennsylvania/10218/1984(H5N2)</td></tr><tr><td rowspan=1 colspan=1>AB286120</td></tr><tr><td rowspan=1 colspan=1>AB286652</td></tr><tr><td rowspan=3 colspan=1>H5N3</td><td rowspan=1 colspan=1>GU052802</td><td rowspan=3 colspan=1>A/duck/Singapore/F119/3/1997(H5N3)</td></tr><tr><td rowspan=1 colspan=1>GU052803</td></tr><tr><td rowspan=1 colspan=1>GU052805</td></tr><tr><td rowspan=3 colspan=1>H6N1</td><td rowspan=1 colspan=1>EU743286</td><td rowspan=3 colspan=1>A/duck/PA/486/1969(H6N1)</td></tr><tr><td rowspan=1 colspan=1>EU743287</td></tr><tr><td rowspan=1 colspan=1>EU743289</td></tr><tr><td rowspan=3 colspan=1>H6N2</td><td rowspan=1 colspan=1>HQ244430</td><td rowspan=3 colspan=1>A/mallard/Czech Republic/15902-17K/2009(H6N2)</td></tr><tr><td rowspan=1 colspan=1>HQ244433</td></tr><tr><td rowspan=1 colspan=1>HQ244434</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=1 colspan=1>FJ750872</td><td rowspan=3 colspan=1>A/mallard/Korea/GH171/2007(H7N7)</td></tr><tr><td rowspan=1 colspan=1>FJ959087</td></tr><tr><td rowspan=1 colspan=1>FJ959090</td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=1 colspan=1>CY014663</td><td rowspan=3 colspan=1>A/turkey/Wisconsin/1/1966(H9N2)</td></tr><tr><td rowspan=1 colspan=1>CY014664</td></tr><tr><td rowspan=1 colspan=1>CY014667</td></tr><tr><td rowspan=2 colspan=1>H10N7</td><td rowspan=1 colspan=1>GQ176136GQ176135</td><td rowspan=2 colspan=1>A/chicken/Germany/N/1949(H10N7)</td></tr><tr><td rowspan=1 colspan=1>GQ176132</td></tr><tr><td rowspan=3 colspan=1>H11N9</td><td rowspan=1 colspan=1>CY014691</td><td rowspan=3 colspan=1>A/duck/Memphis/546/1974(H11N9)</td></tr><tr><td rowspan=1 colspan=1>GQ257441</td></tr><tr><td rowspan=1 colspan=1>CY014687</td></tr></table>

# Supplemental SARS Coronavirus Exclusivity Information (in silico analyses):

In silico analysis of SARS virus sequence (GenBank ID NC_004718.3) against primers for each of the two non-SARS coronavirus assays (CoV HKU1 and $\operatorname { C o V }$ NL63) was conducted. When non-SARS coronavirus assay primers were aligned with the corresponding target sequence from SARS virus, each primer contained at least 6 mismatches. Based on these alignments, no reactivity was predicted between SARS virus and the 2 FilmArray RP non-SARS coronavirus assays.

# g. Assay cut-off:

# Melt Windows Determination:

Sequence diversity results in a range of melting temperatures (Tms) of the PCR2 melting curves for each of the target organism amplicons generated by the FilmArray Respiratory Panel (RP). For each RP organism there is a mean melting temperature at which PCR2 products from most isolates will melt. The range surrounding that mean is where amplicon from less common isolates are expected to melt. The width of each range is assay specific, and is used in the FilmArray meta-analysis software to ensure specificity in organism detection. Thousands of clinical samples would need to be tested from isolates spanning geography and time in order to estimate the temperature ranges for each organism experimentally. An alternative approach is to utilize a theoretical melt prediction model to determine the assay specific melt ranges. Within NCBI databases, tens to thousands of sequences encompassing the PCR2 amplicon region for each RP organism are available. These sequences have been submitted by laboratories worldwide. Sequence submissions include isolates from as far back as a century. Using these sequences to predict the melt temperature ranges for each organism provides an up to date and diverse sample selection.

A mathematical model was used to predict the melting temperatures of all known organism variants (within NCBI databases) for each assay. The distribution of predicted Tms was used to establish an initial melt range for each assay. In addition to sequence variability, the melt ranges must also account for instrument, chemistry, pouch lot and other sources of variability. This overall system variability, or the characteristic system standard deviation, was estimated using a synthetic template data set, in which 17 templates with known amplicon sequence were melted on various FilmArray instruments using multiple pouch lots. Historical RP Panel melting data were also considered to establish appropriate melt ranges. In particular, the Analytical Inclusivity data and all available patient specimen data from the clinical evaluation sites were considered. These data sets included overall system variability and potentially included additional organism sequence variants not accounted for in the NCBI databases. Furthermore, data collected at the beta sites were used as an overall check and validation for the proposed melting ranges.

For each of the pathogen assays, the lower limit of the melting window was calculated as the minimum of the model prediction based on published sequence variation, the clinical evaluation data, and the inclusivity data minus three times the characteristic standard deviation of the synthetic template data. The upper limit of the melting window was calculated as the maximum of the model prediction based on published sequence variation, the clinical evaluation data, and the inclusivity data plus three times the characteristic standard deviation of the synthetic template data:

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=1 colspan=3>Minimum Observed Tm</td><td rowspan=1 colspan=3>Maximum Observed Tm</td><td rowspan=2 colspan=1>LowerLimit</td><td rowspan=2 colspan=1>UpperLimit</td></tr><tr><td rowspan=1 colspan=1>Model</td><td rowspan=1 colspan=1>Inclusivity</td><td rowspan=1 colspan=1>Clinical</td><td rowspan=1 colspan=1>Model</td><td rowspan=1 colspan=1>Inclusivity</td><td rowspan=1 colspan=1>Clinical</td></tr><tr><td rowspan=1 colspan=1>Adeno</td><td rowspan=1 colspan=1>79.23</td><td rowspan=1 colspan=1>80.70</td><td rowspan=1 colspan=1>80.20</td><td rowspan=1 colspan=1>87.03</td><td rowspan=1 colspan=1>86.89</td><td rowspan=1 colspan=1>85.17</td><td rowspan=1 colspan=1>77.7</td><td rowspan=1 colspan=1>88.5</td></tr><tr><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>75.49</td><td rowspan=1 colspan=1>75.05</td><td rowspan=1 colspan=1>74.64</td><td rowspan=1 colspan=1>76.02</td><td rowspan=1 colspan=1>76.75</td><td rowspan=1 colspan=1>76.41</td><td rowspan=1 colspan=1>73.1</td><td rowspan=1 colspan=1>78.3</td></tr><tr><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>80.67</td><td rowspan=1 colspan=1>80.73</td><td rowspan=1 colspan=1>79.88</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>80.92</td><td rowspan=1 colspan=1>80.72</td><td rowspan=1 colspan=1>78.4</td><td rowspan=1 colspan=1>82.8</td></tr><tr><td rowspan=1 colspan=1>FluA - Pan1</td><td rowspan=1 colspan=1>82.66</td><td rowspan=1 colspan=1>83.02</td><td rowspan=1 colspan=1>84.90</td><td rowspan=1 colspan=1>87.53</td><td rowspan=1 colspan=1>85.86</td><td rowspan=1 colspan=1>86.25</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>89.0</td></tr><tr><td rowspan=1 colspan=1>FluA - Pan2</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>77.34</td><td rowspan=1 colspan=1>77.79</td><td rowspan=1 colspan=1>81.92</td><td rowspan=1 colspan=1>80.91</td><td rowspan=1 colspan=1>81.11</td><td rowspan=1 colspan=1>75.8</td><td rowspan=1 colspan=1>83.4</td></tr><tr><td rowspan=1 colspan=1>FluA - H1 Pan</td><td rowspan=1 colspan=1>75.46</td><td rowspan=1 colspan=1>75.97</td><td rowspan=1 colspan=1>77.66</td><td rowspan=1 colspan=1>79.95</td><td rowspan=1 colspan=1>78.93</td><td rowspan=1 colspan=1>79.32</td><td rowspan=1 colspan=1>74.0</td><td rowspan=1 colspan=1>81.4</td></tr><tr><td rowspan=1 colspan=1>FluA - 2009 H1</td><td rowspan=1 colspan=1>78.72</td><td rowspan=1 colspan=1>78.59</td><td rowspan=1 colspan=1>78.50</td><td rowspan=1 colspan=1>79.59</td><td rowspan=1 colspan=1>78.81</td><td rowspan=1 colspan=1>80.06</td><td rowspan=1 colspan=1>77.0</td><td rowspan=1 colspan=1>81.6</td></tr><tr><td rowspan=1 colspan=1>FluA - H3</td><td rowspan=1 colspan=1>80.20</td><td rowspan=1 colspan=1>82.38</td><td rowspan=1 colspan=1>82.30</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>84.17</td><td rowspan=1 colspan=1>82.40</td><td rowspan=1 colspan=1>78.7</td><td rowspan=1 colspan=1>85.7</td></tr><tr><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>80.82</td><td rowspan=1 colspan=1>80.05</td><td rowspan=1 colspan=1>80.50</td><td rowspan=1 colspan=1>81.82</td><td rowspan=1 colspan=1>82.20</td><td rowspan=1 colspan=1>80.71</td><td rowspan=1 colspan=1>78.6</td><td rowspan=1 colspan=1>83.7</td></tr><tr><td rowspan=1 colspan=1>HRV - 1</td><td rowspan=1 colspan=1>82.08</td><td rowspan=1 colspan=1>82.97</td><td rowspan=1 colspan=1>81.86</td><td rowspan=1 colspan=1>88.34</td><td rowspan=1 colspan=1>86.80</td><td rowspan=1 colspan=1>87.90</td><td rowspan=1 colspan=1>80.4</td><td rowspan=1 colspan=1>89.8</td></tr><tr><td rowspan=1 colspan=1>HRV-2</td><td rowspan=1 colspan=1>81.38</td><td rowspan=1 colspan=1>82.77</td><td rowspan=1 colspan=1>81.55</td><td rowspan=1 colspan=1>88.09</td><td rowspan=1 colspan=1>86.38</td><td rowspan=1 colspan=1>87.59</td><td rowspan=1 colspan=1>79.9</td><td rowspan=1 colspan=1>89.6</td></tr><tr><td rowspan=1 colspan=1>HRV-3</td><td rowspan=1 colspan=1>81.46</td><td rowspan=1 colspan=1>82.45</td><td rowspan=1 colspan=1>80.92</td><td rowspan=1 colspan=1>87.65</td><td rowspan=1 colspan=1>85.96</td><td rowspan=1 colspan=1>87.38</td><td rowspan=1 colspan=1>79.4</td><td rowspan=1 colspan=1>89.1</td></tr><tr><td rowspan=1 colspan=1>HRV-4</td><td rowspan=1 colspan=1>81.36</td><td rowspan=1 colspan=1>83.31</td><td rowspan=1 colspan=1>82.07</td><td rowspan=1 colspan=1>89.10</td><td rowspan=1 colspan=1>87.73</td><td rowspan=1 colspan=1>88.22</td><td rowspan=1 colspan=1>79.9</td><td rowspan=1 colspan=1>90.6</td></tr><tr><td rowspan=1 colspan=1>Entero - 1</td><td rowspan=1 colspan=1>87.31</td><td rowspan=1 colspan=1>86.04</td><td rowspan=1 colspan=1>88.94</td><td rowspan=1 colspan=1>89.15</td><td rowspan=1 colspan=1>89.71</td><td rowspan=1 colspan=1>89.35</td><td rowspan=1 colspan=1>84.5</td><td rowspan=1 colspan=1>91.2</td></tr><tr><td rowspan=1 colspan=1>Entero - 2</td><td rowspan=1 colspan=1>87.27</td><td rowspan=1 colspan=1>85.93</td><td rowspan=1 colspan=1>88.83</td><td rowspan=1 colspan=1>89.12</td><td rowspan=1 colspan=1>89.61</td><td rowspan=1 colspan=1>89.14</td><td rowspan=1 colspan=1>84.4</td><td rowspan=1 colspan=1>91.1</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>76.10</td><td rowspan=1 colspan=1>76.22</td><td rowspan=1 colspan=1>74.85</td><td rowspan=1 colspan=1>77.92</td><td rowspan=1 colspan=1>78.20</td><td rowspan=1 colspan=1>78.01</td><td rowspan=1 colspan=1>73.4</td><td rowspan=1 colspan=1>79.7</td></tr><tr><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>80.25</td><td rowspan=1 colspan=1>80.48</td><td rowspan=1 colspan=1>80.81</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>81.87</td><td rowspan=1 colspan=1>81.34</td><td rowspan=1 colspan=1>78.7</td><td rowspan=1 colspan=1>83.4</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>80.44</td><td rowspan=1 colspan=1>79.69</td><td rowspan=1 colspan=1>78.71</td><td rowspan=1 colspan=1>81.62</td><td rowspan=1 colspan=1>81.26</td><td rowspan=1 colspan=1>81.77</td><td rowspan=1 colspan=1>77.2</td><td rowspan=1 colspan=1>83.3</td></tr></table>

For each of the control assays, the Tms for all data in the IVD RP Inclusivity test, clinical evaluation set, and the synthetic template testing set were collected. For each assay, the mean and standard deviation of the melting temperatures were computed. The required four degree melt window around the mean gives the melting windows specified in the following table:

<table><tr><td rowspan=1 colspan=1>Control Assay</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Lower Limit</td><td rowspan=1 colspan=1>Upper Limit</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>76.46</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>2.79</td><td rowspan=1 colspan=1>74.5</td><td rowspan=1 colspan=1>78.5</td></tr><tr><td rowspan=1 colspan=1>yeastRNA</td><td rowspan=1 colspan=1>82.55</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>2.76</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>84.6</td></tr></table>

# Melt Windows Validation

The validation of the melting windows was done in two steps. First, the Tms for the data collected at the beta sites were compared to the melting windows proposed. This comparison did not identify any Tm outside of the proposed ranges. Second, as part of the FilmArray RP Melt Detector Tuning, multiple data sets collected by the IVD and R&D groups and data collected at the clinical evaluation sites were re-analyzed using the proposed melting ranges. This validation did not identify any missed calls as a direct result of the window location.

# h. Interfering Substances:

Interference study was carried out to evaluate the influence of potential interfering substances on the accuracy of test results obtained with the FilmArray RP System. Four different organism mixes (refer to the tables below) containing FilmArray RP analytes were spiked into a simulated NPS (sNPS) sample matrix (human epithelial cells in VTM) at $_ { 5 \mathrm { ~ X ~ } }$ their respective LoDs. The $_ { 5 \mathrm { ~ X ~ } }$ LoD organism concentration was chosen to be near the analyte LoD but also to provide consistent results for sample-tosample comparison.

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>1,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=1 colspan=1>1,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>150,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial</td><td rowspan=1 colspan=1>4,000 DNA copies/mL</td><td rowspan=1 colspan=1>20,000 DNA copies/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Mix4</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5 x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr></table>

The potentially interfering substances tested in this study included substances that are normally found in the nasopharynx or may be introduced into NPS specimens during specimen collection. Endogenous substances like blood, mucin, human genomic DNA and various infectious microorganisms were evaluated. Exogenous substances that may be introduced into NPS specimens before or during sample collection, such as medication, treatments, or topical applications for treating symptoms associated with respiratory infections, were also included in this study. (Note: An additional evaluation of potential interference from medications (prescription and over-the-counter) was included in the prospective clinical evaluation of the FilmArray RP System.) In addition, materials used to collect (swabs) or store (viral transport medium) NPS specimens or substances that could be introduced from the clinical laboratory environment (i.e., bleach, etc.) were tested as potential technique-specific interfering substances.

Potentially interfering test substances were spiked at a level predicted to be above the concentration of the substance likely to be found in an authentic NPS specimen. The following tables list the test substances and final test concentrations used in this study:

Endogenous Substances   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EN1</td><td rowspan=1 colspan=1>Blood (with Na Citrate)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>none</td></tr><tr><td rowspan=1 colspan=1>EN2</td><td rowspan=1 colspan=1>Mucin (bovine submaxillary gland,type I-S)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr></table>

Endogenous Substances – Human Genomic DNA   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>DNA1</td><td rowspan=3 colspan=1>Human genomic DNA 200ng/μl</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>none</td></tr><tr><td rowspan=1 colspan=1>DNA2</td><td rowspan=1 colspan=1>20 ng/μl</td><td rowspan=1 colspan=1>2 ng/μl</td><td rowspan=1 colspan=1>Reagent GradeWater</td></tr><tr><td rowspan=1 colspan=1>DNA3</td><td rowspan=1 colspan=1>2 ng/μl</td><td rowspan=1 colspan=1>0.2 ng/μl</td><td rowspan=1 colspan=1>Reagent GradeWater</td></tr></table>

Endogenous Substances – Potentially Interfering Microorganisms   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Lot Number</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1>MO1</td><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>Zeptometrix#0810040ACF</td><td rowspan=1 colspan=1>304756</td><td rowspan=1 colspan=1>2.8 x 104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO2</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1A</td><td rowspan=1 colspan=1>305067</td><td rowspan=1 colspan=1>1.1 x 104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO3</td><td rowspan=1 colspan=1>Influenza A /2009H1N1</td><td rowspan=1 colspan=1>A/Swine/NY/03/2009</td><td rowspan=1 colspan=1>305849</td><td rowspan=1 colspan=1>1.0 x 105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO4</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>COL</td><td rowspan=1 colspan=1>304535</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MO5</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>M1027</td><td rowspan=1 colspan=1>304320</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MO6</td><td rowspan=1 colspan=1>Corynebacteriumdiptheriae</td><td rowspan=1 colspan=1>ATCC14779</td><td rowspan=1 colspan=1>305297</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr></table>

Exogenous Substances  

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EX1</td><td rowspan=1 colspan=1>Tobramycin (systemic antibiotic)</td><td rowspan=1 colspan=1>0.6 mg/mL</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr><tr><td rowspan=1 colspan=1>EX2</td><td rowspan=1 colspan=1>Mupirocin (active ingredient in anti-bacterial ointment)</td><td rowspan=1 colspan=1>2% w/v</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr><tr><td rowspan=1 colspan=1>EX4</td><td rowspan=1 colspan=1>Saline Nasal Spray with Perservatives(0.65% NaCl Phenylcarbinol BenzalkoniumChloride)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX5</td><td rowspan=1 colspan=1>Nasal Decongestant SprayOxymetazoline HCl 0.05%(also contains Benzalkonium chloride,menthol,eucalyptol, camphor, benzyl alcohol andphosphate buffers)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX6</td><td rowspan=1 colspan=1>Analgesic ointment(Vicks®VapoRub®)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX7</td><td rowspan=1 colspan=1>Petroleum Jelly(Vaseline)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX8</td><td rowspan=1 colspan=1>Snuff(Nasal Tobacco)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Technique Specific Substances   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=2>Test Substance</td><td rowspan=1 colspan=1>TestConcentration</td></tr><tr><td rowspan=3 colspan=1>TS1</td><td rowspan=3 colspan=2>Bleach</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>TS2</td><td rowspan=1 colspan=2>Disinfecting wipes</td><td rowspan=1 colspan=1>½/4 - ½2 in{2</td></tr><tr><td rowspan=1 colspan=1>TS3</td><td rowspan=1 colspan=2>Ethanol</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>TS4</td><td rowspan=1 colspan=2>DNAZapTM</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>TS5</td><td rowspan=1 colspan=2>RNaseOUTTM</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=4 colspan=1>TS6</td><td rowspan=4 colspan=1>Swab</td><td rowspan=1 colspan=1>Copan 168C (rayon/twisted aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Copan FloQ (Flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Copan 175KS01 (polyester/aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Millipore 519CS01M (flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>TS7</td><td rowspan=5 colspan=1>ViralTransportMedia</td><td rowspan=1 colspan=1>Remel M4</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Remel M4-RT</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Remel M6</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>100%</td></tr></table>

On each day of testing, one specimen per organism mix was not spiked with any test substance, to serve as a positive control. For biological test substances that may contain genetic material (such as blood, mucin, DNA, microorganisms), one negative specimen (blank sNPS sample matrix with no organism mix) was spiked with only the test substance (negative control) to evaluate the potential for false positive results due to the test substance itself. For each endogenous, exogenous, and technique-specific test substance, one specimen per organism mix was spiked with the appropriate amount of test substance. All specimens were tested with the FilmArray RP System on the day they were prepared.

None of the potentially interfering endogenous, exogenous, or technique-specific substances tested in this study were found to affect the accuracy of respiratory pathogen detection in simulated NPS specimens using the FilmArray RP System.

# FluMist® nasal influenza vaccine

Simulated nasopharyngeal swab (sNPS) specimens were spiked with dilutions of FluMist nasal influenza vaccine (MedImmune, 2009-2010 version), resulting in final test concentrations of $10 \%$ , $1 \%$ and $0 . 1 \%$ (v/v) FluMist. Each specimen was tested with the FilmArray RP system to determine if the vaccine material would react with any of the FilmArray RP assays. If reactivity was observed at any or all dilutions, additional dilutions were tested until no reactivity was observed.

FluMist® Influenza Vaccine Live, Intranasal; Intranasal Spray, 2009-2010 Formula was provided by the manufacturer (MedImmune). The material was provided as individual sprayers containing a single $0 . 2 \mathrm { m L }$ dose. Each dose contains approximately $1 0 ^ { 6 . 5 - 7 . 5 }$ FFU (fluorescence focus units) each of three live, attenuated influenza virus reassortants including: A/SouthDakota/6/2007 (H1N1) (A/Brisbane/59/2007-like), A/Uraguay/716/2007 (H3N2) (A/Brisbane/10/2007-like), and B/Brisbane/60/2008. Non-viral components of the vaccine listed in the provided materials include: monosodium glutamate, gelatin, arginine, sucrose, gentamicin sulfate, monobasic potassium phosphate and dibasic potassium phosphate. (Note: For the 2009-2010 respiratory season, a separate $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ variant vaccine was prepared and administered. The 2009 H1N1 variant vaccine material was not provided by the manufacturer for testing with the FilmArray RP system.)

Positive results were obtained for Influenza $\mathrm { A } / \mathrm { H } 1$ , Influenza A/H3, and Influenza B in each of the samples tested at $10 \%$ , $1 \%$ and $0 . 1 \%$ (v/v) FluMist. Additional dilutions were made and tested until each of the Influenza viruses could no longer be detected by the FilmArray RP system ( $0 . 0 0 0 1 \%$ for Influenza A H1, $0 . 0 0 0 0 1 \%$ for Influenza A H3 and $0 . 0 0 0 0 0 1 \%$ for Influenza B). There was no reactivity with the Influenza $\mathrm { A } / 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ subtyping assay, which specifically detects the pandemic $2 0 0 9 \mathrm { A } / \mathrm { H } 1$ variant and was not included in the 2009-2010 FluMist formulation.

This reactivity is documented in the FilmArray RP System product package insert as a “limitation”.

In addition to the analytical interference substances study, the prospective clinical dataset was analyzed for evidence of the effects of potentially interfering medications (prescription and over-the-counter; OTC) administered to or taken by the study population. If clinically significant interference from medications had occurred, it would be expected that the analyte detection rate in the medicated population would be lower than that in the non-medicated population. This effect was not observed in the clinical dataset. One or more analytes was detected by the reference/comparator methods in 110 of 256 $(4 3 \% )$ non-medicated individuals. The FilmArray RP system correctly identified all analytes in 104 $( 9 4 . 5 \% )$ of these individuals. One or more analytes was detected by the reference/comparator methods in 313 of 597 $( 5 2 \% )$ medicated individuals. The FilmArray RP system correctly identified all analytes in 302 $( 9 6 . 5 \% )$ of these individuals. The table below lists the medications administered to 72

or taken by the enrolled study population (self-reported and/or recorded from patient charts).

<table><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>Clindamycin</td><td rowspan=1 colspan=1>Lansoprazole</td><td rowspan=1 colspan=1>Phenobarbital</td></tr><tr><td rowspan=1 colspan=1>Albuterol</td><td rowspan=1 colspan=1>Co-trimoxazole</td><td rowspan=1 colspan=1>Levalbuterol</td><td rowspan=1 colspan=1>Phenylephrine</td></tr><tr><td rowspan=1 colspan=1>Alendronate</td><td rowspan=1 colspan=1>Cough drops/syrups(various OTC)</td><td rowspan=1 colspan=1>Levothyroxine</td><td rowspan=1 colspan=1>Pilocarpine</td></tr><tr><td rowspan=1 colspan=1>Alprazolam</td><td rowspan=1 colspan=1>Cyproheptadine</td><td rowspan=1 colspan=1>Lisinopril</td><td rowspan=1 colspan=1>Pravachol (Pravastatin)</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>Deferasirox</td><td rowspan=1 colspan=1>Loperamide</td><td rowspan=1 colspan=1>Prednisolone</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin Clavulanate</td><td rowspan=1 colspan=1>Dextromethorphan</td><td rowspan=1 colspan=1>Loratadine</td><td rowspan=1 colspan=1>Prednisone</td></tr><tr><td rowspan=1 colspan=1>Azithromycin</td><td rowspan=1 colspan=1>Dextromethorphanhydrobromide</td><td rowspan=1 colspan=1>Lorazepam</td><td rowspan=1 colspan=1>Propranolol</td></tr><tr><td rowspan=1 colspan=1>Baclofen</td><td rowspan=1 colspan=1>Diphenhydramine</td><td rowspan=1 colspan=1>Metoclopramide</td><td rowspan=1 colspan=1>Pseudoephedrine</td></tr><tr><td rowspan=1 colspan=1>Benzonatate</td><td rowspan=1 colspan=1>Doxylaminesuccinate</td><td rowspan=1 colspan=1>Metolazone</td><td rowspan=1 colspan=1>Ranitidine</td></tr><tr><td rowspan=1 colspan=1>Betamethasone</td><td rowspan=1 colspan=1>Electrolyte solutions(rehydration salts)</td><td rowspan=1 colspan=1>Metoprolol</td><td rowspan=1 colspan=1>Saline Nasal Drops/Sprays(various OTC)</td></tr><tr><td rowspan=1 colspan=1>Brimonidine</td><td rowspan=1 colspan=1>Enoxaparin</td><td rowspan=1 colspan=1>Midazolam</td><td rowspan=1 colspan=1>Salmeterol</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>Eszopiclone</td><td rowspan=1 colspan=1>Montelukast</td><td rowspan=1 colspan=1>Simethicone</td></tr><tr><td rowspan=1 colspan=1>Budesonide/FormoterolFumarate Dihydrate</td><td rowspan=1 colspan=1>Fentanyl</td><td rowspan=1 colspan=1>Naproxen</td><td rowspan=1 colspan=1>Sirolimus</td></tr><tr><td rowspan=1 colspan=1>Bupropion hydrochloride</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>Nizatidine</td><td rowspan=1 colspan=1>Spironolactone</td></tr><tr><td rowspan=1 colspan=1>Carvedilol</td><td rowspan=1 colspan=1>Furosemide</td><td rowspan=1 colspan=1>Omeprazole</td><td rowspan=1 colspan=1>Temazepam</td></tr><tr><td rowspan=1 colspan=1>Cefdinir</td><td rowspan=1 colspan=1>Gabapentin</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>Tiotropium</td></tr><tr><td rowspan=1 colspan=1>Cefepime</td><td rowspan=1 colspan=1>Gemfibrozil</td><td rowspan=1 colspan=1>Oxybutynin</td><td rowspan=1 colspan=1>Ursodiol</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>Guaifenesin</td><td rowspan=1 colspan=1>Pancrelipase</td><td rowspan=1 colspan=1>Valacyclovir</td></tr><tr><td rowspan=1 colspan=1>Cephrazole</td><td rowspan=1 colspan=1>Hydrocodone</td><td rowspan=1 colspan=1>Pantoprazole</td><td rowspan=1 colspan=1>Vancomycin</td></tr><tr><td rowspan=1 colspan=1>Cetirizine</td><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>Penicillin</td><td rowspan=1 colspan=1>Vitamins/Multivitamins/Minerals (various OTC)</td></tr><tr><td rowspan=1 colspan=1>Chlorpheniraminemaleate</td><td rowspan=1 colspan=1>Inderal (Propranolol)</td><td rowspan=1 colspan=1>Pentamidine</td><td rowspan=1 colspan=1>Voriconazole</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>Isosorbide</td><td rowspan=1 colspan=2></td></tr></table>

# i. Carry-Over Contamination:

An analytical carry-over study was carried out to demonstrate that when recommended laboratory practices are followed, there is little risk of false positive results caused by carryover or cross-contamination in the FilmArray Respiratory Panel (RP) System. The data for this study were generated as part the exclusivity testing examining the potential for cross-reactivity between Respiratory Panel targets and assays. The majority of testing was performed by new users at a contracted laboratory as part of an early performance evaluation of the FilmArray RP system. Multiple spiked NPS (or simulated NPS) samples containing high levels of organism (approximately $1 0 ^ { 5 }$ for viruses) were tested over several days using 2 FilmArray instruments. Each sample was prepared and loaded into a RP pouch one at a time in a work area (biosafety cabinet or hood) that was separated from the location of the FilmArray instrument(s), and regular cleaning of the work area and pouch loading station were carried out 73

according to the recommended testing procedures. Consecutive testing of the same organism was discouraged in order to maximize the potential for detecting false positive results due to carryover from a previously tested sample. Over $6 0 \mathrm { R P }$ target organism spiked samples were tested in a total of 63 valid runs, and at least half of the samples were tested subsequently to a sample containing a different target organism at high concentration. No false positive results were reported ( $100 \%$ negative agreement). The consecutive set-up and testing of samples containing high concentrations of different target organisms demonstrated that these practices are effective in preventing false positive results due to carryover between samples.

Because the PCR2 array in each FilmArray pouch contains over 100 individual PCR reaction wells in close proximity, the possibility of well-to-well carryover was also examined. Individual false positive melt curves could be generated if (1) reaction material (template and rehydrated assay specific primers) carried over into an adjacent well early in second stage PCR cycling, or (2) amplicon from one well entered a neighboring well prior to melt analysis. However, the FilmArray device and pouch have been designed to interact in such a way that the transfer or mixing of liquid between reaction wells is unlikely. In addition, a positive assay result requires that at least 2 of the 3 wells are positive for melt curves within the expected Tm range of the assay. The scattering of assays with similar Tm windows across the array restricts the possibility that well-to-well carryover could produce 2 or more false positive wells for the same assay.

The 63 valid test results described above (over 6000 reaction wells with no false positive organism or assay results) were examined well-by-well and 4 false positive melt curves were observed as a result of carryover between neighboring wells. Each carryover event produced a single positive reaction well per assay and had no effect on the overall accuracy of the FilmArray RP results. This well-to-well carryover analysis resulted in an overall $\%$ negative agreement of $9 9 . 9 \%$ with the $9 5 \%$ CI of $9 9 . 9 \%$ to $1 0 0 . 0 \%$ (5342/5346 expected negative wells).

Combined, these results indicated that when proper laboratory practices are followed, the risk of false positive FilmArray RP test results due to sample cross-contamination or well-to-well carryover is insignificant.

# j. Simulated NPS Sample Matrix Validation Study:

NPS is the intended sample type to be used with the FilmArray RP system. A NPS is collected from an individual by inserting a flexible, fine-shafted synthetic swab into the nostril and back to the nasopharynx. After a few seconds, the swab is slowly withdrawn with a rotating motion and the swab is placed into a vial containing approximately $3 ~ \mathrm { m L }$ of viral transport medium (VTM). Once in VTM, vortexing helps to release the material collected by the swab (human cells, viral particles, bacteria, secretions, etc.) into the medium, which is then used as the test sample.

A series of studies was required to demonstrate the analytical performance characteristics of the FilmArray RP system. For these types of studies, the ideal sample matrix is NPS collected from healthy individuals that does not contain nucleic acid from any of the pathogens detected by the panel. Initially, sample matrix for contrived 74

samples was obtained by collecting NPS from donors at Idaho Technology, Inc (ITI). Each donor claimed to be healthy and free of symptoms associated with respiratory infection on the day the NPS was collected. Individual NPS were screened with the FilmArray RP system prior to use in the studies, to confirm that the sample matrix was negative for RP organisms.

This process began in the early fall of 2009, at the onset of the typical respiratory season, and it was found during screening that respiratory pathogens could be detected in some percentage of healthy donor NPS. For example, over 2 months of collection (November – December 2009), about $9 \%$ of the screened NPS were positive for one or more FilmArray RP analytes. Some weeks, the rate was as high as $2 6 \%$ . This pattern continued through the respiratory season with a positivity rate of nearly $12 \%$ from January to March 2010. Furthermore, even after the initial screening for negativity, false positive results were occasionally detected in contrived samples and the false positive organism could often be traced to the sample matrix.

Respiratory Panel Organisms Detected in Healthy Donor NPS Screening   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>#Total</td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>Rhino</td><td rowspan=1 colspan=1>FluA/2009H1N1</td><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adeno</td></tr><tr><td rowspan=1 colspan=1>Nov-Dec2009</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>11a</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Jan-Mar2010</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>10.7%</td><td rowspan=1 colspan=1>10b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total(% oftotal)</td><td rowspan=1 colspan=1>393</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>8.4%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr></table>

a 1 sample represents co-infection of Rhino / CoV-OC43 $^ { \mathsf { b } } 1$ sample represents co-infection of Rhino / CoV-HKU1

The NPS being screened were collected from donors at ITI, following an IRBapproved protocol. Under the terms of the protocol, the NPS specimens were deidentified. No personal information was collected from the donors and no test results could be traced back to the donor. As a result, if a NPS was positive for one or more RP organisms during screening, it was not possible to follow-up with the donor to inquire about signs of infection prior to or in the days following the date of donation.

Given the very large number of contrived samples needed for analytical studies, it became apparent that use of a sample matrix of NPS collected from donors would be problematic. Most importantly, the true negativity of the matrix could not be adequately confirmed, which placed the accuracy and integrity of analytical studies at risk. Consequently, the use of a simulated NPS matrix was investigated. The intent was for the simulated NPS to resemble the content of an actual NPS as closely as possible, while ensuring that the simulated NPS was free of respiratory pathogens and did not alter the performance of the system.

Ten-fold dilutions of HeLa cells in VTM were prepared (from a starting concentration of $2 \mathrm { ~ x ~ } 1 0 ^ { 5 }$ cells/mL) and tested with the FilmArray system both fresh and after being stored frozen $( - 2 0 ^ { \circ } \mathrm { C } )$ for one week. Results were compared to NPS collected from healthy donors and an appropriate cell concentration for the simulated NPS was

chosen by comparison of Cp values for the hRNA assay (targets mRNA of the human beta-catenin gene) between the true and artificial sample matrices. Once a human cell concentration was selected, contrived samples containing multiple live respiratory pathogens at or near the estimated system LoD were prepared in both NPS and simulated NPS sample matrices. The organism mixes included the following:

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>200 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>100 TCID50/mLa</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 4</td><td rowspan=1 colspan=1>LoD</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td></tr></table>

Crossing point $\textstyle ( \mathrm { C p } )$ values of amplification curves generated for all positive assays were also used to compare results from samples prepared in NPS and simulated NPS sample matrices. (Note: Though the use of $\mathrm { C p }$ values in this study was helpful in making comparisons, since the FilmArray RP is a qualitative rather than quantitative system, and the FilmArray RP uses two-stage nested PCR, therefore, Cps are inherently more variable than Cps in traditional real-time PCR systems, the significance of these numbers can not be over-interpreted.) When comparing mean Cp values between NPS and simulated NPS for the hRNA and organism assays, a difference of 3 cycles or less was considered equivalent.

All samples tested (NPS and simulated NPS) were negative for all panel organisms and positive amplification and melt curves were recorded for the hRNA assay. Average Cp values for the hRNA assay were calculated for each sample type and plotted for comparison. The $\mathrm { C p }$ values observed for the NPS (20.5 – 27.1) were consistent with historical data from NPS screening. The average Cp of the hRNA assay was similar for fresh and frozen simulated NPS dilutions. The average Cp for NPS was most similar to the $\mathrm { C p }$ values obtained with simulated NPS prepared at a concentration of $2 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ cells/mL. Based on this analysis, a concentration of $2 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ HeLa cells/mL in VTM was selected as the composition of simulated NPS.

Comparison of results for FilmArray RP assays between NPS and simulated NPS sample matrices is presented below:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Adeno</td><td rowspan=1 colspan=1>CoV-NL63</td><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Rhino</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>FluA/H3</td><td rowspan=1 colspan=1>FluA/H1</td></tr><tr><td rowspan=1 colspan=1>NPS Control</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>SimulatedNPS Test</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>Δ Mean Cp(SimulatedNPS- NPS)</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>-0.51</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>0.09-0.53a</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>1.50b</td><td rowspan=1 colspan=1>2.27b</td></tr></table>

a Range of delta mean Cp for the HRV1, HRV2, HRV3, and HRV4 assays.

b The delta mean Cp shown is for the FluA/H1 and FluA/H3 subtyping assays only. Delta mean Cp was 1.48 for the FluA-pan1 assay and 1.07 for the FluA-pan2 assay.

The study generated no false positive results, and the results demonstrated that the sensitivity of the FilmArray RP system was comparable between samples prepared in nasopharyngeal swab (NPS) and simulated NPS sample matrices. This sample matrix was then used in analytical performance studies of the FilmArray RP system in place of NPS collected from donors. Prior to use, all freshly prepared simulated NPS was screened with the FilmArray RP system to confirm that the hRNA assay provides the expected Cp values and that the sample matrix is negative for all RP organisms.

# k. Comparator Assays Analytical Validation Studies

In the prospective clinical trial for the FilmArray RP System, the FilmArray results were compared to the results from standard viral culture and fluorescent antibody testing performed by the clinical sites for several of the common viruses (Adenovirus, Influenza A, Influenza B, Parainfluenza virus 3, and Respiratory Syncytial Virus). The majority of the remaining pathogens require 2 PCR followed by bi-directional sequencing comparator assays (PCR/sequencing) which were tested directly from the NPS specimens. The exception to this was Influenza A subtyping, which was tested from viral culture material with 1 assay per virus:

Comparator PCR/sequencing Testing Scheme   

<table><tr><td rowspan=1 colspan=1>Organism/Virus</td><td rowspan=1 colspan=1>Comparator PCR Tests</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>2 PCR tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>2 PCR tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 PCR tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>2 PCR tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>2 PCR tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>FluA - H1 subtyping</td><td rowspan=1 colspan=1>1 PCR test on culture</td></tr><tr><td rowspan=1 colspan=1>FluA - H3 subtyping</td><td rowspan=1 colspan=1>1 PCR test on culture</td></tr><tr><td rowspan=1 colspan=1>FluA - 2009 H1N1 subtyping</td><td rowspan=1 colspan=1>1 PCR test on culture</td></tr></table>

Analytically validated PCR followed by bi-directional sequencing assays that target different sequences from those used in the FilmArray pouch were used by ITI as part of a pre-defined composite comparator method for assays in the FilmArray RP system.

Due to the need for a large number of assays and the requirement for quality bidirectional sequencing, ITI was unable to procure adequate services from any outside laboratory, or combination of laboratories. Therefore, ITI relied on its own Research & Development and Regulated Products groups to develop and validate all of the comparator PCR tests used in this study. All assays were designed to provide adequate amplicon length for sequencing while still being able to specifically detect the organisms of interest. Some viruses possess few regions to which a specific PCR assay can be targeted. This limited amplicon length and gene target choice in some instances. However, all PCR/sequencing comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s).

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>FilmArrayTarget</td><td rowspan=1 colspan=1>ComparatorAssayTarget 1</td><td rowspan=1 colspan=1>Target 1Chemistry</td><td rowspan=1 colspan=1>ComparatorAssayTarget 2</td><td rowspan=1 colspan=1>Target 2Chemistry</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>nucleocapsidgene</td><td rowspan=1 colspan=1>matrix gene(HMPV M)</td><td rowspan=1 colspan=1>Taqman</td><td rowspan=1 colspan=1>nucleocapsidgenne**(HMPV N)</td><td rowspan=1 colspan=1>LC green(nested)</td></tr><tr><td rowspan=1 colspan=1>Rhinoviruses</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5&#x27;UTR</td><td rowspan=1 colspan=1>5&#x27;UTR*(RhinovirusTQ)</td><td rowspan=1 colspan=1>Taqman(nested)</td><td rowspan=1 colspan=1>5&#x27;UTR*(Rhinovirus LC)</td><td rowspan=1 colspan=1>LC green(nested)</td></tr><tr><td rowspan=1 colspan=1>Enteroviruses</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>5&#x27;UTR</td><td rowspan=1 colspan=1>5&#x27;UTR*(Entero TQ)</td><td rowspan=1 colspan=1>Taqman</td><td rowspan=1 colspan=1>5&#x27;UTR*(Entero LC)</td><td rowspan=1 colspan=1>LC green(nested)</td></tr><tr><td rowspan=1 colspan=1>Coronaviruses</td><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>nucleocapsidgene</td><td rowspan=1 colspan=1>Spike gene(CoV HKU1Sp)</td><td rowspan=1 colspan=1>Taqman(nested)</td><td rowspan=1 colspan=1>polymerase geneCoV HKU1P2)</td><td rowspan=1 colspan=1>Taqman(nested)</td></tr><tr><td rowspan=1 colspan=1>Coronaviruses</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>nucleoproteingene</td><td rowspan=1 colspan=1>matrix gene(CoV NL63 M)</td><td rowspan=1 colspan=1>Taqman</td><td rowspan=1 colspan=1>polymerase gene(CoV NL63 P)</td><td rowspan=1 colspan=1>LC green(nested)</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>hemagglutiningene</td><td rowspan=1 colspan=1>hemagglutiningene*(FluA - H1)</td><td rowspan=1 colspan=1>Taqman</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=2 colspan=1>H1 2009</td><td rowspan=2 colspan=1>hemagglutiningene</td><td rowspan=2 colspan=1>hemagglutiningene*(FluA -H1_2009)</td><td rowspan=2 colspan=1>Taqman</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>hemagglutiningene</td><td rowspan=1 colspan=1>hemagglutiningene*(FluA - H3)</td><td rowspan=1 colspan=1>Taqman</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\*These PCR/sequencing comparator assays target a different region of the same gene targeted by the FilmArray Respiratory Panel. The assays amplify different amplicon sequences and have no overlap with the FilmArray targets.

Validation studies included analytical sensitivity (LoD) study, analytical reactivity study, and analytical exclusivity study.

The confirmed LoD for each comparator assay is presented in the table below:

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>LoD Concentration</td><td rowspan=1 colspan=1># Pos / Total</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 Pol2 Assay</td><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.9 x 106 RNA copies/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 Spike Assay</td><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.9 x 106 RNA copies/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 M Assay</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 Pol Assay</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HMPV M Assay</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HMPV N Assay</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus TQ Assay</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus LC Assay</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus TQ Assay</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus LC Assay</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Influenza H1 Assay</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Influenza H1N1 2009Assay</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Influenza H3N2 Assay</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

Analytical reactivity study results are summarized in the table below. All Coronavirus types did not have inclusivity strains available to test

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">LoD(TCID50/ml)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Assay 1</td><td colspan="1" rowspan="1">Assay 2</td></tr><tr><td colspan="1" rowspan="6">Enterovirus</td><td colspan="1" rowspan="6">30,000</td><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A21/Kuykendall</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 68 (F02-3607 corn)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="5">Influenza A(H1N1)</td><td colspan="1" rowspan="5">200</td><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Influenza A (2009H1N1</td><td colspan="1" rowspan="2">100</td><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="7">Influenza A(H3N2)</td><td colspan="1" rowspan="7">5</td><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="3">−</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">MRC-2</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">HumanMetapneumovirus</td><td colspan="1" rowspan="9">2</td><td colspan="1" rowspan="1">3 (Peru2-2002 B1)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">5 (Peru3-2003 B1)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">8 (Peru6-2003 B2)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">9 (IA3-2002 A1)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">18 (IA18-2003 B2)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">20 (IA14-2003 A2)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">Human Rhinovirus</td><td colspan="1" rowspan="9">1</td><td colspan="1" rowspan="1">A16 (11757)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A2 (HGP)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A34 (137-3)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A57 (Ch47)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A7 (68-CV11)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A77 (130-63)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A85 (50-525-CV54)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">B14 (1059)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B27 (5870)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr></table>

Analytical exclusivity study testing organisms and concentrations tested are presented in the table below. No cross-reactivity was observed for all tested organisms at the tested concentrations.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(TCID50/ml)</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">clinical isolate</td><td colspan="1" rowspan="1">1.30E+09</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">7.40E+09</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">9.80E+09</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.00E+08</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">O157:H7</td><td colspan="1" rowspan="1">2.34E+10</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">3,50E+09</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC14779</td><td colspan="1" rowspan="1">4.20E+09</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">2.12E+09</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">2.60E+06</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">8.30E+09</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11 (typestrain</td><td colspan="1" rowspan="1">6.83E+08</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">2.20E+08</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">3.16E+08</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.99E+09</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">M1027 (type strain)</td><td colspan="1" rowspan="1">1.63E+09</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.05E+10</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA)</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">8.40E+09</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">6.20E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">5.54E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.57E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">2.53E+07</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.93E+08</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.04E+09</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.26E+06</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.51E+06</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Typer 1</td><td colspan="1" rowspan="1">KB</td><td colspan="1" rowspan="1">1.02E+08</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">K340</td><td colspan="1" rowspan="1">1.40E+09</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">ATCC VR-740</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">ATCC VR-759</td><td colspan="1" rowspan="1">2.19E+06</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">6123</td><td colspan="1" rowspan="1">1.41E+05</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">NR-470</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.02E+08</td></tr><tr><td colspan="1" rowspan="1">hMPV-16</td><td colspan="1" rowspan="1">IA10-2003, A1</td><td colspan="1" rowspan="1">2.45E+05</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">3.80E+06</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1) var 2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">1.26E+06</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">2.45E+05</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">4.17E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">6.61E+06</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">4.17E+05</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">3.50E+09</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">7.26E+06</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">5.63E+06</td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">Promega (G3041)</td><td colspan="1" rowspan="1">182ng/ul</td></tr></table>

A retrospective clinical evaluation of the FilmArray RP system was performed at ITI to supplement the prospective evaluation data. In this study, retrospective pre-selected archived specimens were further validated/confirmed by PCR assays followed by sequencing analysis, and then tested with the FilmArray RP system.

Because it is possible that the provided samples had been misidentified, had degraded during storage or contained additional pathogens (such as Rhinovirus) not identified by the source laboratory, the presence of the expected analyte or the absence of the analytes was confirmed using analytically validated “validation” PCR assays for the analytes in the specific panel. These “validation” PCR assays are summarized in the table below.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">FilmArray Target(s)(assay names)</td><td colspan="1" rowspan="1">ValidationAssay Gene Target1 (assay name)</td><td colspan="1" rowspan="1">Target 1Chemistrya</td><td colspan="1" rowspan="1">Validation Assay GeneTarget 2 (assay name)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hexon gene(Adeno)</td><td colspan="1" rowspan="1">polymerase gene(Adpl)</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enteroviruses</td><td colspan="1" rowspan="1">5'UTR (Entero1, Entero2,HRV1, HRV2, HRV3,HRV4)</td><td colspan="1" rowspan="1">5'UTREnteroviruses(EnteroLC)b</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1">5'UTR Rhinoviruses(RhinoTq)b</td></tr><tr><td colspan="1" rowspan="3">Influenza A/H1</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="3">matrix gene(FluA Matrix)b</td><td colspan="1" rowspan="3">LC Green</td><td colspan="1" rowspan="3">hemagglutinin gene(F uA H1)</td></tr><tr><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td></tr><tr><td colspan="1" rowspan="1">hemagglutinin gene(FluA-H1-pan)</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H12009</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="1">matrix geneb(FluA Matrix)</td><td colspan="1" rowspan="2">LC Green</td><td colspan="1" rowspan="2">hemagglutinin geneb(FluA H1 09)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">hemagglutinin gene(FluA-H1-2009)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">Influenza A/H3</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="3">matrix gene(FluA Matrix)b</td><td colspan="1" rowspan="3">LC Green</td><td colspan="1" rowspan="3">hemagglutinin gene(FluA H3)b</td></tr><tr><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td></tr><tr><td colspan="1" rowspan="1">hemagglutinin gene(FluA-H3)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">hemagglutinin gene(FluB)</td><td colspan="1" rowspan="1">matrix gene(F(luB)</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Parainfluenzavirus 3</td><td colspan="1" rowspan="1">fusion gene(PIV3)</td><td colspan="1" rowspan="1">fusion gene(PIVIV3)b</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td></tr></table>

a All LC Green assays are nested assays bThese validation assays target a different region of the same gene targeted by the FilmArray RP. The assays amplify different amplicon sequences.

Validation studies for these additional “validation” PCR assays also included analytical sensitivity (LoD) study, analytical reactivity study, and analytical exclusivity study.

The confirmed LoD for each “validation” PCR assay is presented in the table below:

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>LoD Concentration # Pos / Total</td><td rowspan=2 colspan=1>LoD Concentration # Pos / Total</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adeno PolymeraseAssay</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Influenza A Matrix</td><td rowspan=1 colspan=1>Influenza A H1N1-Seasonal</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=3 colspan=1>19/20</td><td rowspan=3 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B Pan Assay</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3 AltAssay</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

Analytical reactivity study results are summarized in the table below:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>LOD (TCID50/ml)</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>AssayResults</td></tr><tr><td rowspan=16 colspan=1>Adenovirus</td><td rowspan=16 colspan=1>300</td><td rowspan=1 colspan=1>Adeno KB 3</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 2-1</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 4a</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 4p3</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno KB 5</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 6</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno KB 7A</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 7d/d2</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 7h</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno KB 8</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 11</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 14</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 21a</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno KB 31</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno A-549 34</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno HEK293A 41-1</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=17 colspan=1>Influenza A</td><td rowspan=17 colspan=1>Flu A H1N1 = 200Flu A 2009 H1N1 =100Flu A H3N2 = 5</td><td rowspan=1 colspan=1>A/New Caledonia/20/9 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A1/FM/1/47 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A1/Denver/1/57 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/3/2006 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54 (H1)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Swine NY/01/2009</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Swine NY/03/2009</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75 (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73 (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68 (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68 (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Alice (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>MRC-2 (H3 recombinant)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07 (H3)</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=4 colspan=1>Influenza B</td><td rowspan=4 colspan=1>60</td><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/FL/07/04</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>Positive</td></tr></table>

<table><tr><td rowspan=5 colspan=1></td><td rowspan=5 colspan=2></td><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=2 colspan=1>Parainfluenza 3</td><td rowspan=2 colspan=2>10</td><td rowspan=1 colspan=1>C-243</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>47885 NIH</td><td rowspan=1 colspan=1>Positive</td></tr></table>

Analytical exclusivity study results are summarized in the table below:

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concen.(TCID50/ml)</td><td colspan="1" rowspan="1">Adeno</td><td colspan="1" rowspan="1">Flu AMatrix</td><td colspan="1" rowspan="1">Flu BPan</td><td colspan="1" rowspan="1">PIV 3</td></tr><tr><td colspan="1" rowspan="1">Bordetellabronchiseptica</td><td colspan="1" rowspan="1">clinical isolate</td><td colspan="1" rowspan="1">1.30E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">7.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetellaparapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">9.80E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">2.34E+10</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Chlamydiatrachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">3,50E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">ATCC14779</td><td colspan="1" rowspan="1">4.20E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Lactobacillusacidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">2.12E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Haemophilusinfluenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">2.60E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">8.30E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Moraxellacatarrhalis</td><td colspan="1" rowspan="1">Ne 11 (type strain)</td><td colspan="1" rowspan="1">6.83E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">2.20E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmahomnis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">3.16E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.99E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Neisseriameningitides</td><td colspan="1" rowspan="1">M1027 (type strain)</td><td colspan="1" rowspan="1">1.63E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Pseudomonasaeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.05E+10</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusaureus (MRSA)</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">8.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidrmidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">6.20E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">5.54E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcuspyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.57E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcussalivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">2.53E+07</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmagenitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.93E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus(CMV)</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus(EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.04E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.26E+06</td><td colspan="1" rowspan="1">Pos*</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.51E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Typer1</td><td colspan="1" rowspan="1">KB</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Boca Virus</td><td colspan="1" rowspan="1">K340</td><td colspan="1" rowspan="1">1.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">229E (ATCC VR-740?</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">OC43 (ATCC VR-759)</td><td colspan="1" rowspan="1">2.19E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">HKU1 (6123)</td><td colspan="1" rowspan="1">1.41E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">NL63 (NR-470)</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">hMPV-16</td><td colspan="1" rowspan="1">IA10-2003, A1</td><td colspan="1" rowspan="1">2.45E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A(H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">3.80E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A(H1N1) var 2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">1.26E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A(H3N2)</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">2.45E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">6.61E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza</td><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">3.50E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">7.26E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">5.63E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">Promega (G3041)</td><td colspan="1" rowspan="1">182ng/ul</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr></table>

a This stock produced false positive Adenovirus result. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

# Prospective Clinical Study

The clinical performance of the FilmArray RP system was evaluated during prospective studies at 3 U.S. clinical laboratories. NP swab samples were collected and tested at three U.S. clinical laboratories (Sites 1, 2, and 3) during December 2009 thru May 2010. Clinical study sites were chosen based upon the expertise of the principal investigators, the quality of their personnel and laboratory resources, and the desired geographic/demographic variation of the potential subject populations.

Three geographically separated clinical study sites participated in the clinical evaluation of the FilmArray RP system. The study sites were located in the Southeast (Charleston, SC), the Southwest (Dallas, TX), and the Midwest (Detroit, MI). The size of the population in these locations ranges from approximately 107,000 in Charleston, SC to approximately 1.2 million in Dallas TX (source: U.S. Census Bureau). Each study location was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

The study sites enrolled subjects from diverse demographic groups. Site 1 (Medical University of South Carolina, MUSC) enrolled subjects from an adult emergency department. Site 2 (Detroit Medical Center, DMC) enrolled primarily children from the emergency department, but also enrolled immunocompromised adults from an outpatient bone marrow transplant clinic (approximately $10 \%$ of the enrolled subjects at this site). Site 3 (Children’s Medical Center of Dallas, CMC) enrolled subjects from a pediatric emergency department and associated urgent care clinic.

The following table provides a summary of demographic information for the 853 subjects that participated in the prospective study:

Demographic Summary for FilmArray RP Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>449 (53%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>404 (62%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>484 (57%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>95 (11%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>190 (22%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>84 (10%)</td></tr></table>

All specimens used in the study were prospectively collected from subjects had signs/symptoms of respiratory infection including but not limited to fever, cough, sore throat, runny/stuffy nose, myalgia, earache, headache, chills, or fatigue. Written informed consent was acquired from each subject and/or their parent/guardian (if under 18) at the time of enrollment into the prospective study. After informed consent was acquired, the subject was assigned a Volunteer Identification Number (VIN). The VIN was used to de-identify the specimen used for FilmArray testing and to provide data to the sponsor. A key that connected the subject’s identity to their assigned VIN was maintained in a secure location at each site in order to facilitate the tracking of standard of care testing and for monitoring purposes. Access to this key was limited to the study coordinator or laboratory supervisor at each site. At the time of enrollment the following information was recorded on the Case Report Form (CRF): 1) Age and sex; 2) Information about their suspected respiratory infection, i.e. signs and symptoms, date of onset; and 3) Current medications (self-reported and/or collected from medical records).

Two respiratory specimens were collected from each enrolled subject. One specimen (NPS or other standard specimen type, such as nasal aspirate/wash, used by the study site) was used for standard of care testing and for viral culture to detect seven common respiratory viruses (Adenovirus, Flu A, Flu B, PIV1, 2, and 3, and RSV). A frozen aliquot of viral culture material was prepared for comparator PCR/sequencing testing. The second specimen, a NPS in $3 ~ \mathrm { m L }$ of viral transport Moderate (VTM) was used for FilmArray testing according to the Instructions For Use and was aliquoted and frozen for further comparator PCR/sequencing testing. Individual specimens were delinked from all patient identifiers and given a study sample code.

Reference and comparator methods used to assess the performance of the FilmArray RP system include viral culture followed by fluorescent antibody identification for common viruses (Adenovirus, Flu A, Flu B, PIV3, and RSV), viral culture followed by 1 analytically validated PCR assay with bi-directional sequence confirmation (Flu A subtyping), and a composite comparator method of 2 analytically validated PCR assays with bi-directional sequence confirmation (all other panel analytes):

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Reference/Comparator Method(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>FluA/H1 subtyping</td><td rowspan=1 colspan=1>Culture + 1 PCR/sequencing test on culture</td></tr><tr><td rowspan=1 colspan=1>FluA/H3 subtyping</td><td rowspan=1 colspan=1>Culture + 1 PCR/sequencing test on culture</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1N1 subtyping</td><td rowspan=1 colspan=1>Culture + 1 PCR/sequencing test on culture</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>2 PCR/sequencing tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>2 PCR/sequencing tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>2 PCR/sequencing tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>2 PCR/sequencing tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 PCR/sequencing tests on direct sample</td></tr></table>

Viral culture was performed with fresh specimens using standard methods. Sites 1 and 3 used R-Mix shell vial culture format (mixed monolayer of human adenocarcenoma A549 cells and mink lung Mv1Lu cells) incubated for up to 72 hours. Site 2 used traditional cell culture format, employing three separately inoculated cell lines (primary rhesus monkey kidney pRhMK cells, human epidermoid carcinoma Hep-2 cells, and human lung fibroblast MRC5 cells) incubated for up to 14 days. Each site confirmed the presence or absence of the viruses (Adenovirus, Flu A, Flu B, PIV3, and RSV) using a FDA-cleared direct fluorescent antibody method (D3 Ultra Viral Respiratory Screening and ID Kit; Diagnostic Hybrids, Inc.).

Frozen aliquots of viral culture material and NPS specimens were shipped to ITI on a weekly basis for comparator PCR/sequencing testing. PCR testing was performed by ITI research associates in a blinded manner; specimens were labeled only with the subject’s VIN and the FilmArray results were not available to them. A bi-directional sequencing result for each comparator PCR positive result was required to demonstrate sequence identity with the target organism. All comparator PCR assays were designed at ITI to provide adequate amplicon length for sequencing while still being able to detect the organisms of interest. The assays were also designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. PCR plates containing potentially positive amplicons were not opened at ITI; they were sent directly to clean up and bi-directional sequencing. Sequencing results that met acceptance criteria (i.e., a minimum of $9 9 \%$ accuracy for at least $90 \%$ of the sequence, Phred quality score of 20 or higher) were recorded as positive for the specific analyte.

Performance of the FilmArray RP system detecting Adenovirus, Flu A, Flu B, PIV3, or RSV, respectively, was compared to viral culture followed by fluorescent antibody identification. “True” Adenovirus, Influenza A, Influenza B, Parainfluenza virus 3, or RSV positives, respectively, were considered as any sample that was tested positive for Adenovirus, Influenza A, Influenza B, Parainfluenza virus 3, or RSV, respectively, by viral culture followed by DFA testing. “True” Adenovirus, Influenza A, Influenza B, Parainfluenza virus 3, or RSV negatives, respectively, were considered as any sample that was tested negative for Adenovirus, Influenza A, Influenza B, Parainfluenza virus 3, or RSV, respectively, by viral culture followed by DFA testing.

Performance of the FilmArray RP system detecting Influenza A/H1, A/H3, or A/2009 H1N1, respectively, was compared to viral culture followed by 1 analytically validated PCR assay with bi-directional sequence confirmation. The comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. “True” Influenza A/H1, A/H3, or A/2009 H1N1 positives, respectively, were considered as any sample that was tested positive for Influenza A by viral culture, and had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Influenza A/H1, A/H3, or A/2009 H1N1 sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” Influenza A/H1, A/H3, or A/2009 H1N1 negatives, respectively, were considered as any sample that was tested negative for Influenza A by viral culture, or any sample that was tested positive for Influenza A virus by viral culture, but was tested negative by the respective Influenza A subtype specific PCR assay. The E-value ranges generated from the clinical trial per organism were presented in the following table:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-value High</td></tr><tr><td rowspan=1 colspan=1>FluA/H1 subtyping</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FluA/H3 subtyping</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1N1 subtyping</td><td rowspan=1 colspan=1>8.0E-122</td><td rowspan=1 colspan=1>1.4E-78</td></tr></table>

Performance of the FilmArray RP system detecting Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus, respectively, was compared to a predetermined algorithm that used composite comparator methods. The composite reference methods consist of 2 analytically validated PCR followed by bidirectional genetic sequencing assays. The comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. “True” Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus positives, respectively, were considered as any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, or Human Metapneumovirus negatives, respectively, were considered as any sample that was tested negative by both of the comparator PCR assays. The E-value ranges generated from the clinical trial per organism were presented in the following table:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-value High</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>4.51E-72</td><td rowspan=1 colspan=1>1.15E-08</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>9.23E-42</td><td rowspan=1 colspan=1>2.72E-12</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1.9E-83</td><td rowspan=1 colspan=1>8.0E-16</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>8.8E-148</td><td rowspan=1 colspan=1>6.5E-70</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>5.8E-77</td><td rowspan=1 colspan=1>2.3E-15</td></tr></table>

The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone. (http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).

A total of 1257 runs were attempted in the prospective clinical evaluation. These included pouches for patient samples, daily external control material testing, and weekly contamination surveillance testing. Greater than $9 6 \%$ of all attempted runs were completed. Forty-three (43) (43/1257; $3 . 4 \% )$ total runs did not complete. Three (3) $( 3 / 1 2 5 7 ; 0 . 2 \% )$ runs were aborted by the user. Four (4) $( 4 / 1 2 5 7 ; 0 . 3 \% )$ runs were designated ‘Incomplete’ due to software errors. Thirty-six (36) $( 3 6 / 1 2 5 7 ; 2 . 9 \% )$ of total runs were designated ‘Instrument Error’. Of these 36, two (2) were caused by an incorrect installation of a firmware/software update. Fifteen (15) of the instrument errors were caused by valve controller errors which often result from pressure leaks/failure in the instrument bladder system (a known failure mode). The remaining 19 instrument errors were caused by a camera communication error that was traced back to a problem with the camera driver software (a known failure mode). The FilmArray RP pouch is manufactured to contain a vacuum that draws in the required amount of Hydration Solution and Sample/Buffer Mix when the seals are broken by the

cannula-tipped syringes in each of the pouch ports. If either the Hydration Solution or the Sample/Buffer Mix is not drawn into the pouch, the operator is instructed to discard the faulty pouch and obtain a new pouch to test the specimen. Out of 1281 pouches used in the prospective clinical evaluation, thirty-seven (37) (37/1281; $2 . 9 \%$ ) failed to draw Hydration Solution or Sample/Buffer Mix. However, the majority (31/37) of these failures occurred early in the evaluation (December through early February). It has been observed that an operator can partially break a port seal, leading to loss of vacuum and failure to draw the solution. It may also be possible for a pouch to be manufactured with a faulty port seal or reduced vacuum. A decrease in the rate of this failure type over time suggested that either pouch manufacturing improved or the operators became more adept at reliably puncturing the port seals. During the prospective clinical evaluation, one operator observed a leak in a pouch when it was removed from the instrument after testing (1/1244 loaded pouches $=$ $0 . 0 8 \%$ failure rate). The pouches are manufactured from durable materials able to withstand the manipulations performed by the instrument; however, an occasional small rupture or tear may occur. Following discovery of the pouch leak, the operator followed recommended decontamination procedures and performed contamination surveillance swab testing of the area surrounding the instrument. No contamination was detected.

Two pouch control assays, the RNA Process Control and PCR2 Control, were used to pass or fail runs. A total of 1214 completed pouch runs were performed during the Prospective clinical evaluation. Of these, 894 were specimen tests and the remaining runs were for daily control mixes and contamination surveillance. Thirty-six (36) $( 3 6 / 1 2 1 4 ; 3 \% )$ of these failed due to negative results for one or both pouch control assays (RNA Process Control and/or PCR2 Control). The majority of failed pouches (27/36; $7 5 \%$ of failed pouches) were negative for the RNA Process Control assay. Nine (9) additional failures were the result of negative PCR2 Control results alone (with positive RNA Process Control result). All nine of these failures demonstrate the ability of the PCR2 Control to appropriately fail pouches that have had a ‘failure to fill’. This failure type results when liquid has not been completely distributed across the PCR2 array.

Four (4) frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ control mixes were provided to the study sites for daily testing. Three control mixes contained pooled NPS specimens spiked with whole virus/bacteria with some plasmid DNA for hard to acquire organisms. Combined, the 3 mixes covered all panel analytes. A fourth mix was negative for all panel members and only contained pooled NPS:

<table><tr><td rowspan=1 colspan=1>Control Mix 1</td><td rowspan=1 colspan=1>Control Mix 2</td><td rowspan=1 colspan=1>Control Mix 3</td><td rowspan=1 colspan=1>Control Mix 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Influenza A/2009 H1N1</td><td rowspan=6 colspan=1>Negative for allorganisms</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1></td></tr></table>

The operators were required to complete a valid control mix run (correct results obtained) prior to beginning patient sample testing on each testing day. A total of 300 control mix runs were attempted. Seventeen (17) runs did not complete and 6 runs had failed pouch control(s). Of the remaining 277 runs (site 1, 99 runs; site 2, 70 runs; and site 3,108 runs) 7 (site 1, 1 run; site 2, 2 runs; and site 3, 4 runs) (7/277; $2 . 5 \% )$ ) did not return the correct organism results either due to the detection of an extra analyte $( 4 / 7 )$ and/or the failure to detect one or more spiked analytes (4/7). These failures may be due to low level virus from 91

the NPS donors, introduction of contamination during the preparation or testing of the samples, or improper handling of the frozen aliquots.

The FilmArray RP system is a closed system; all steps from sample lysis through nucleic acid extraction, PCR, and melt analysis are performed inside the pouch and require no user manipulation. This dramatically reduces the possibility of contamination from environmental material or amplicon during the testing process. However, it is still possible for contaminants to be introduced during pouch loading. To determine if environmental contamination of workstations might interfere with the accuracy of test results, FilmArray operators were instructed to perform contamination surveillance swab testing at least weekly and following any pouch leaks. Forty (40) total swab test runs were attempted; 35 of the runs completed and the controls passed. No contamination was detected in any of the swab tests indicating that the routine cleaning procedures recommended in the clinical trial protocol are sufficient to prevent false positive test results caused by environmental contamination of workstations used for preparing pouches. These routine cleaning procedures were recommended in the package insert.

The prospective performance data (all sites combined) are presented in the following tables by analyte:

Adenovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>14a</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3b</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>815</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:   24/27       88.9% (95%CI: 70.8%-97.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 812/826     98.3% (95%CI: 97.2%-99.1%)</td></tr></table>

a Adenoviruses were identified in 13/14 of these false positive specimens by bidirectional sequence analysis. Ten were identified as Adenovirus group C, two as Adenovirus group B, and one as Adenovirus group E. b One of three specimens re-tested positive with the FilmArray RP System. Bidirectional sequence analysis identified two false negative specimens as Adenovirus group C and one as Adenovirus group B.

Coronavirus HKU1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>827</td><td rowspan=1 colspan=1>828</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:    23/24 95.8%(95%CI:78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:827/82999.8%(95%CI:99.1%-100.0%)</td></tr></table>

a CoV-HKU1 was detected in both false positive specimens using an alternate bidirectional sequence analysis assay.

Coronavirus NL63   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>830</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   23/24    95.8%(95%CI:78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>NegativePercent Agreement:829/829100.0% (95%CI:99.6%-100.0%)</td></tr></table>

Human metapneumovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>94</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>759</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:    88/9394.6%(95%CI:87.9%-98.2%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:754/76099.2% (95%CI:98.3%-99.7%)</td></tr></table>

Human Rhinovirus/Enterovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>225</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>613</td><td rowspan=1 colspan=1>628</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>648</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:190/205 92.7% (95%CI:88.2%-95.8%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:613/648 94.6% (95%CI:92.6%-96.2%)</td></tr></table>

Influenza A   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>843</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:    9/10   90.0%(95%CI: 55.5%-99.8%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 841/843  99.8% (95%CI: 99.2%-100.0%)</td></tr></table>

a. Influenza A virus (2009 H1N1 subtype) was detected in 2/2 False positive samples using bi-directional sequence analysis.

# Influenza A/H1

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:    0/0             n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 853/853 100.0% (95%CI: 99.6%-100.0%)</td></tr></table>

Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:−0/0             n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 853/853 100.0% (95%CI: 99.6%-100.0%)</td></tr></table>

# Influenza A/2009 H1N1

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3a,b</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>844</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:     8/9     88.9%(95%CI: 51.8%-99.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity:841/844   99.6%(95%CI: 99.0%-99.9%)</td></tr></table>

a For one of the Influenza A positive culture samples, Influenza A virus (2009 H1N1 subtype) was not detected by the PCR comparator assay followed by bi-directional sequence analysis. However, Influenza A virus $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ subtype) was detected from the direct sample and confirmed by bi-directional sequence analysis using an alternate assay.

b Influenza A virus $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ subtype) was detected in 2/2 False positive samples using bi-directional sequence analysis.

Influenza B   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:    0/0             n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity:853/853100.0% (95%CI: 99.6%-100.0%)</td></tr></table>

Parainfluenza 3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>10a</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>819</td><td rowspan=1 colspan=1>820</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:   23/24      95.8% (95%CI: 78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 819/829    98.8% (95%CI: 97.8%-99.4%)</td></tr></table>

a Parainfluenza 3 viruses were identified in 10/10 False Positive specimens using bidirectional sequence analysis. b Parainfluenza 3 was detected in this False Negative specimen when re-tested using the FilmArray RP system.

RSV

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>87a</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>714</td><td rowspan=1 colspan=1>714</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>801</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity:    52/52     100.0% (95%CI: 93.2%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 714/801      89.1% (95%CI: 86.8%-91.2%)</td></tr></table>

a RSV was detected in 83/87 False Positive specimens using bi-directional sequence analysis.

The prospective performance data (all sites combined) are presented in the following table by organism:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>70.8% -97.7%</td><td rowspan=1 colspan=1>812/826</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>97.2% -99.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(vs. culture alone)</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>55.5% -99.8%</td><td rowspan=1 colspan=1>841/843</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>51.8% -99.7%</td><td rowspan=1 colspan=1>841/844</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0% -99.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% -99.9%</td><td rowspan=1 colspan=1>819/829</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>97.8% -99.4%</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.2% -100.0%</td><td rowspan=1 colspan=1>714/801</td><td rowspan=1 colspan=1>89.1%</td><td rowspan=1 colspan=1>86.8% -91.2%</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% -99.9%</td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% -100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% -99.9%</td><td rowspan=1 colspan=1>827/829</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.1% -100.0%</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>88/93</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>87.9% -98.2%</td><td rowspan=1 colspan=1>754/760</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.3% -99.7%</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>190/205</td><td rowspan=1 colspan=1>92.7%</td><td rowspan=1 colspan=1>88.2% -95.8%</td><td rowspan=1 colspan=1>613/648</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>92.6% -96.2%</td></tr></table>

# Prospective Clinical Study Mixed Infection Analysis

The FilmArray RP system detected a total of 81 mixed infections in the prospective clinical evaluation. This represents $1 6 . 1 \%$ of the total positive specimens (81/502). Seventy-six (76/81; $9 3 . 8 \%$ ) were double infections, and 5 (5/81; $6 . 2 \% )$ were triple infections. The total number of test results comprising these co-infections was 167. The single most common co-infection (21/81; $2 5 . 9 \% )$ was Human Rhinovirus/Enterovirus with Respiratory Syncytial Virus. These viruses were the most prevalent in the tested population. Out of the 81 co-infections, 47 contained one or more analytes that had not been detected with the reference/comparator methods, i.e., discrepant co-infection.

Distinct Co-infection Combinations Detected by the FilmArray RP System in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=3>Distinct Co-infection CombinationsDetected by FilmArray RP</td><td rowspan=2 colspan=1>counsPe</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infectionsa</td><td rowspan=2 colspan=1>Discrepant Analyte(s)a</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Adenovirus (2); HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (1); RSV (2)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adenovirus (1); hMPV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>hMPV (1); RSV (3)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>hMPV (2); HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>PIV 3 (1)</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV (1)</td></tr><tr><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>RSV (2)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>HRV/Entero (3); PIV 3 (3)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>HRV/Entero (8); RSV (13)</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV (1)</td></tr><tr><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV (1)</td></tr><tr><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>RSV (1)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Adenovirus (2)</td></tr><tr><td rowspan=1 colspan=3>Total Co-infections</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>54/167</td></tr><tr><td rowspan=1 colspan=3>Total Double Infections</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>49/152</td></tr><tr><td rowspan=1 colspan=3>Total Triple Infections</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/15</td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by FilmArray RP but not detected by the reference/comparator methods.

Additional Distinct Co-infection Combinations Detected by Reference/comparator Methods, but not Detected by the FilmArray RP System in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=2>Distinct Co-infection Combinationsa</td><td rowspan=3 colspan=1>consCotL</td><td rowspan=3 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=3 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=2 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>hMPV (2); HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/Entero (1)</td></tr></table>

a This table includes only distinct co-infections that were detected by the reference/comparator method but not by FilmArray RP; the remaining co-infections detected by the reference/comparator methods are already represented in the table above.

Mixed Infections Detected by FilmArray RP and Prevalence of Individual Analytes in Mixed Infection   

<table><tr><td rowspan=1 colspan=1>Organism Combination</td><td rowspan=1 colspan=1># ofPositiveSamples</td><td rowspan=1 colspan=1>Percentage ofTotal SamplesTested</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Involvedin # ofMixedInfections</td><td rowspan=1 colspan=1>Prevalencein MixedInfections</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + RSV</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>2.46%</td><td rowspan=1 colspan=1>HRV/Enterovirus</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>69.14%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1.06%</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>44.44%</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV 3</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0.94%</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>27.16%</td></tr><tr><td rowspan=1 colspan=1>hMPV + HRV/Entero</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.82%</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>19.75%</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.47%</td><td rowspan=1 colspan=1>PIV 3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>17.28%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + HRV/Entero</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.47%</td><td rowspan=1 colspan=1>CoVNL63</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>16.05%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>12.35%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>Influenza A/2009H1N1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.35%</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.00%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + RSV</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV 3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero+ PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.23%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoVNL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + PIV 3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero+ RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + HRV/Entero+ RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.12%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>9.50%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Retrospective Clinical Study

A retrospective clinical evaluation of the FilmArray RP system was performed at ITI to supplement the prospective evaluation data. In this study, retrospective pre-selected archived specimens were further validated/confirmed by PCR/sequencing analysis, and then tested with the FilmArray RP system. Samples for this study included 400 retrospective preselected archived specimens. Most specimens had previous positive test results for selected organisms of interest from the source sites. Several negative specimens (as determined at the source sites) were also included in the retrospective sample set.

These retrospective archived samples were characterized previously at the source sites using a variety of methods including DSFA, viral culture followed by DFA, Luminex X-TAG RVP, and LDT PCR assays. Upon arrival at ITI, a 4-digit study number (VIN) was assigned to each sample and a key was created in order to group samples together into panels by organism. The organism panel was randomized such that the users testing the samples with the FilmArray RP were blinded as to the expected test result. Specimens were organized into “test panels” such that specimens positive for a particular analyte could serve as a negative for another analyte, allowing for calculations of negative percent agreement (NPA). Additional negative specimens were also included in each panel. There were 3 different panels in total, labeled 4 through 6, as indicated in the following table:

<table><tr><td rowspan=1 colspan=1>Panel #</td><td rowspan=1 colspan=1>Organisms</td><td rowspan=1 colspan=1>Number ofSamples</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A/H1, A/H3, A/2009 H1N1, Influenza B,and Negatives</td><td rowspan=1 colspan=1>178</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Parainfluenza virus 3, Enterovirus, and Negatives</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Adenovirus and Negatives</td><td rowspan=1 colspan=1>74</td></tr></table>

Because it is possible that the provided samples had been misidentified, had degraded during storage or contained additional pathogens (such as Rhinovirus) not identified by the source laboratory, the presence of the expected analyte or the absence of the analytes was confirmed using analytically validated “validation” PCR assays for the analytes in the specific panel. (Note that while some validation assays share the same gene target as the FilmArray RP panel, they targeted different regions of the gene). Positive PCR reactions were then subjected to bi-directional sequencing for definitive confirmation. “True” organism positives were considered as any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched the particular organism sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E-values. The E-value ranges generated from the retrospective clinical trial per organism were presented in the following table:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-Value High</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>6.4E-57</td><td rowspan=1 colspan=1>1.4E-28</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>9.3E-50</td><td rowspan=1 colspan=1>3.4E-30</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>4.1E-99</td><td rowspan=1 colspan=1>5.1E-35</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>4.9E-118</td><td rowspan=1 colspan=1>9.0E-61</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>9.0E-88</td><td rowspan=1 colspan=1>3.4E-73</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1.0E-180</td><td rowspan=1 colspan=1>4.9E-50</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>2.7E-69</td><td rowspan=1 colspan=1>1.6E-33</td></tr></table>

If this “validation” testing did not confirm the presence of the expected organism in the sample, it was not analyzed further as this result may have reflected that the sample had been improperly labeled or handled. If additional pathogens were identified in the sample (those not identified by the source laboratory), the samples was also omitted from the analysis for that specific analyte. The expected result (positive for reported organism, or negative for presumed negative specimens) was confirmed for 349 of these specimens using “validation” PCR assays and sequencing (the identity of 51 specimens could not be confirmed). Data collected from “validation” PCR assays showed that specimens contained a broad range of analyte concentrations for all organisms tested (based on Cp value distribution).

The following table provides a summary of demographic information for the 349 subjects with confirmed samples in the retrospective study:

Demographic summary for FilmArray RP retrospective clinical study   

<table><tr><td rowspan=1 colspan=2>Total Specimens</td><td rowspan=1 colspan=1>349</td></tr><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>82 (23.5%)</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>79 (22.6%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>188 (53.9%)</td></tr><tr><td rowspan=4 colspan=1>Age</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>14.1</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>4.0</td></tr><tr><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>83.0</td></tr><tr><td rowspan=5 colspan=1>Age Range</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>89 (25.5%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>35 (10.0%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>23 (6.6%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>14 (4.0%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>188 (53.9%)</td></tr></table>

a Demographic information was not provided for specimens from one source. Because the specimens were provided by a pediatric hospital, it is understood that the age range of specimens was from $< 1$ yrs to 21 yrs.

Validated/confirmed specimens were then analyzed on the FilmArray RP. The identity of the specimens was unknown to the FilmArray operators. If a FilmArray error was encountered or if a pouch failed, residual NPS specimen (when available) was used to retest the sample. Out of 349 analyzed specimens, the FilmArray RP provided the valid results 345 times. Four specimens failed, and were excluded from data analysis due to insufficient volume to retest the specimens.

The retrospective performance data from testing panel $\# 4$ samples (a total of 161 samples: 32 Flu A/H1, 54 A/H3, 35 A/2009 H1, 31 Flu B and 9 Negative) are presented in the following tables by analyte:

Seasonal Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>127</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 32/32     100.0% (95%CI:89.1%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 127/127   100.0% (95%CI:97.1%-100.0%)</td></tr></table>

Seasonal Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:  54/54  100.0%  (95%CI:93.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:105/105100.0%  (95%CI:96.5%-100.0%)</td></tr></table>

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   34/34 100.0%(95%CI:89.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement::125/125100.0%(95%CI:97.1%-100.0%)</td></tr></table>

\* The initial analysis of sample VIN 4-037 resulted in a pouch sample control failure. There was insufficient volume to retest the specimen; therefore it was excluded from further analysis.

Influenza B Virus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30*</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   30/30   100.0%(95%CI:88.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 129/129100.0%(95%CI:97.2%-100.0%)</td></tr></table>

\*The initial analysis of sample VIN 4-058 resulted in a pouch failure. There was insufficient volume to retest the specimen; therefore it was excluded from further analysis.

The retrospective performance data from testing panel #5 samples (a total of 129 samples: 23 Enterovirus, 36 PIV3 and 70 Negative) are presented in the following tables by analyte:

Enterovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>113*</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 22/23  95.7%(95%CI:78.0%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 90/90 100.0%(95%CI:96.0%-100.0%)</td></tr></table>

\*It was observed that 16 negative specimens were unexpectedly positive when tested with the Rhinovirus “validation” PCR. Because there was no previous test for Rhinovirus or Enterovirus for these particular specimens, the Rhinovirus PCR result was not investigated by sequencing and the Rhinovirus or Enterovirus status of these specimens were not included in the FilmArray RP percent agreement analysis for Enterovirus.

PIV-3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   36/36 100.0%(95%CI:90.3%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:  93/93 100.0%(95%CI:96.1%-100.0%)</td></tr></table>

The retrospective performance data from testing panel $\# 6$ samples (a total of 59 samples: 27 Adenovirus and 32 Negative) are presented in the following tables by analyte:

Adenovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>55*</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 27/27     100.0% (95%CI:87.2%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 28/28      100.0% (95%CI:87.7%-100.0%)</td></tr></table>

\*It was found that two negative specimens were positive for Adenovirus (VINs 6-025 & 6-032). Because there was no previous test for Adenovirus on these particular specimens, the Adenovirus PCR result was not investigated by sequencing and the Adenovirus status of these specimens were not included in the FilmArray RP percent agreement analysis for Adenovirus. Two specimens were lost due to incomplete FilmArray runs resulting from instrument and software errors (VINs 6-052 and 6-068), and there was insufficient volume of either sample to repeat the test, therefore it was omitted from further analysis.

The retrospective performance data combined are presented in the following table by analyte:

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.2% -100.0%</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.7% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>78.0% -99.9%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.1% -100.0%</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.4% -100.0%</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009H1N1</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.7% -100.0%</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.4% -100.0%</td><td rowspan=1 colspan=1>129/129</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3% -100.0%</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.1% - 100.0%</td></tr></table>

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Expected Value (as determined by the FilmArray RP) Summary by Site for the FilmArray RP Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Overall (n=853)</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 1 (n=275)</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 2 (n=333)</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>9.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenzavirus 3</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>12.7%</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>16.3%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>25.8%</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>20.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4.1%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>11.0%</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>12.3%</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>16.7%</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Entero</td><td rowspan=1 colspan=1>225</td><td rowspan=1 colspan=1>26.4%</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>13.1%</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>27.6%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>39.6%</td></tr></table>

Expected Value (as determined by FilmArray RP) Summary by Age Group for FilmArray RP Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Total(ExpectedValue)</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>≥ 50years</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38 (4.5%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>CoV-HKU1</td><td rowspan=1 colspan=1>25 (2.9%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>CoV-NL63</td><td rowspan=1 colspan=1>23 (2.7%)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>94 (11.0%)</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>225 (26.4%)</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>FluA (all subtypes)</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>FluA/H1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1N1</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>FluA/H3</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>FluB</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV3</td><td rowspan=1 colspan=1>33 (3.9%)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>139 (16.3%)</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td></tr></table>

Expected Value (co-infections as determined by FilmArray RP) Summary by Age Group for FilmArray RP Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td rowspan=1 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Total(ExpectedValue)</td><td rowspan=1 colspan=1>≤ 5 Years</td><td rowspan=1 colspan=1>6-21 Years</td><td rowspan=1 colspan=1>22-49 Years</td><td rowspan=1 colspan=1>≥ 50 Years</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + RSV</td><td rowspan=1 colspan=1>21 (2.46%)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + Adenovirus</td><td rowspan=1 colspan=1>9 (1.06%)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV3</td><td rowspan=1 colspan=1>8 (0.94%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + hMPV</td><td rowspan=1 colspan=1>7 (0.82%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>4 (0.47%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + CoV NL63</td><td rowspan=1 colspan=1>4 (0.47%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + hMPV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + HRV/Entero</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + RSV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + hMPV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + RSV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + RSV</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV3</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero + PIV3</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoV NL63</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + hMPV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + PIV3</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + HRV/Entero + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + HRV/Entero + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + hMPV + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

# N. Instrument Name:

FilmArray Instrument

# O. System Descriptions:

1. Modes of Operation:

The FilmArray instrument interacts with the FilmArray pouch mechanically, thermally, and optically to drive a multi-step chemical process designed to detect specific nucleic acid targets using multiplex nested PCR followed by DNA melting analysis. The instrument follows a protocol that is defined by a set of codes that are downloaded from the host computer at runtime. The instrument protocol defines the specific timing and sequence parameters as the instrument performs the following key functions:

• Perform cell disruption using the bead beater Extract nucleic acid from the disrupted sample Perform stage 1 PCR thermocycling of multiplexed PCR reaction Perform stage 2 PCR thermocycling of the array Execute a DNA melt and detect fluorescent signals generated Monitor system performance in real time and communicates out of specification

conditions to the user via the software

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The FilmArray Respiratory Panel (RP) pouch contains two internal control assays:

1. The RNA Process Control targets an mRNA of the yeast, Schizosaccharomyces pombe. During FilmArray RP pouch manufacture, whole yeast are freeze-dried into the sample injection port of each pouch. When the test specimen is loaded into the pouch, S. pombe is rehydrated and enters the pouch with the specimen. The yeast nucleic acid is extracted, purified and tested simultaneously with nucleic acids from the patient specimen. A positive result for the processing control indicates that all steps in the process (nucleic acid extraction, reverse transcription, PCR, melt, detection, and analysis) are functioning properly.

2. The second stage PCR (PCR2) control assay detects a synthetic DNA template that is dried into triplicate wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

The RNA Process Control and the PCR2 Control assays are used to Pass or Fail each FilmArray RP pouch run. This combination of control assays monitors each of the critical mechanical and chemical processes that occur in a pouch run, while limiting the possibility of random control assay failures that could contribute to unnecessary pouch failures.

Good laboratory practice recommends running external positive and negative controls regularly. Use viral transport medium as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision