# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

# B. Purpose for Submission:

To obtain substantial equivalence determination for a new 510(k).

# C. Measurand:

Target DNA sequences from the toxin B gene and the binary toxin gene of Clostridium difficile and the single base pair deletion at nucleotide 117 in the tcdC gene (which encodes for a negative regulator in toxin production).

# D. Type of Test:

Qualitative, in vitro diagnostic test using real-time polymerase chain reaction (PCR) for detection of toxin B gene sequences

# E. Applicant:

Cepheid

F. Proprietary and Established Names: Xpert $^ \mathrm { \textregistered }$ C. difficile/Epi

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.2660   
2. Classification: Class I   
3. Product code: OMN – C. difficile nucleic acids   
4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Cepheid Xpert® C. difficile/Epi Assay is a qualitative in vitro diagnostic test for rapid detection of toxin B gene sequences and for presumptive identification of 027/NAP1/BI strains of toxigenic Clostridium difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having C. difficile infection (CDI). Presumptive identification of 027/NAP1/BI strains of $C .$ . difficile is by detection of binary toxin (CDT) gene sequences and the single base pair deletion at nucleotide 117 in the tcdC gene. The tcdC gene encodes for a negative regulator in C. difficile toxin production. The test is performed on the Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System and utilizes automated real-time polymerase chain reaction (PCR) to detect toxin gene sequences associated with toxin producing $C .$ . difficile. The Xpert C. difficile/Epi Assay is intended as an aid in the diagnosis of CDI. Detection of 027/NAP1/BI strains of $C .$ . difficile by the Xpert $C .$ . difficile/Epi Assay is presumptive and is solely for epidemiological purposes and is not intended to guide or monitor treatment for $C .$ . difficile infections. Concomitant culture is necessary only if further typing or organism recovery is required.

# 2. Indication(s) for use:

The Cepheid Xpert® C. difficile/Epi Assay is a qualitative in vitro diagnostic test for rapid detection of toxin B gene sequences and for presumptive identification of 027/NAP1/BI strains of toxigenic Clostridium difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having C. difficile infection (CDI). Presumptive identification of 027/NAP1/BI strains of $C .$ . difficile is by detection of binary toxin (CDT) gene sequences and the single base pair deletion at nucleotide 117 in the tcdC gene. The tcdC gene encodes for a negative regulator in C. difficile toxin production. The test is performed on the Cepheid GeneXpert $^ \mathrm { \textregistered }$ Dx System and utilizes automated real-time polymerase chain reaction (PCR) to detect toxin gene sequences associated with toxin producing $C .$ . difficile. The Xpert C. difficile/Epi Assay is intended as an aid in the diagnosis of CDI. Detection of 027/NAP1/BI strains of C. difficile by the Xpert $C$ . difficile/Epi Assay is presumptive and is solely for epidemiological purposes and is not intended to guide or monitor treatment for $C .$ . difficile infections. Concomitant culture is necessary only if further typing or organism recovery is required.

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: To be used with the Cepheid GeneXpert® Dx System

# I. Device Description:

The Cepheid Xpert $^ \mathrm { \textregistered }$ C. difficile/Epi Assay is a rapid, automated in vitro diagnostic test for qualitative detection of toxin B gene sequences and for presumptive identification of 027/NAP1/BI strains of toxigenic Clostridium difficile from unformed (liquid or soft) stool specimens collected from patients suspected of having C. difficile infection (CDI). The Xpert C. difficile/Epi Assay system includes reagents for the detection of the toxin B gene of toxigenic C. difficile and the presumptive detection of sequences found in 027/NAP1/BI strains. Detection of DNA sequences is carried out by real- time multiplex polymerase chain reaction (PCR) after an initial sample processing step. The assay is performed on the Cepheid GeneXpert® Dx System.

A swab is placed into the unformed stool specimen, and then the swab is inserted into a vial containing the Sample Reagent. Following brief vortexing of the vial with swab, the eluted material and two single-use reagents (Reagent 1 and Reagent 2), that are provided with the assay, are transferred to different, uniquely-labeled chambers of the disposable fluidic cartridge (the Xpert C. difficile/Epi Assay cartridge). The user initiates a test from the system user interface and places the cartridge into the GeneXpert Dx System instrument platform, which performs real-time, multiplex PCR for detection of DNA sequences. In this platform, additional sample preparation, amplification, and real-time detection are all fully automated and completely integrated.

The GeneXpert $^ \mathrm { \textregistered }$ System consists of a GeneXpert instrument, personal computer, and the multi-chambered fluidic cartridges that are designed to complete sample preparation and real-time PCR for detection of the toxin B gene of C. difficile and sequences associated with 027/NAP1/BI strains.

The Xpert C. difficile/Epi Assay also includes an internal sample processing control (SPC) to control for adequate processing of the target bacteria and to monitor the presence of inhibitor(s) in the PCR assay. The SPC also ensures that the PCR conditions (temperature and time) are appropriate for the amplification reaction and that the PCR reagents are functional. The Probe Check Control (PCC) verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability.

Each Xpert C. difficile/Epi Assay kit contains sufficient reagents to process 10 specimens or quality control samples. The kit contains the following:

Xpert C. difficile/Epi Assay Cartridges with integrated reaction tubes, 10 per kit. Each Xpert C. difficile/Epi Assay Cartridge contains three freezedried beads (Bead1, Bead 2, and Bead 3). Bead 1 contains Polymerase, dNTPs, bovine serum albumin (BSA), and a probe. Bead 2 contains primers, probes, and BSA. Bead 3 contains Sample Processing Control (SPC) consisting of non-infectious sample preparation control spores.

Xpert C. difficile/Epi Assay Reagent pouches, 10 pouches per kit. Each pouch contains:

o Sample Reagent (Guanidinium thiocyanate and surfactants) $1 \mathrm { ~ x ~ } 2 . 0 \mathrm { m l }$ per bottle

o Reagent 1 (Tris Buffer, EDTA and surfactants) $1 \mathrm { ~ x ~ } 2 . 8 \mathrm { ~ m l }$ per bottle   
o Reagent 2 (Sodium Hydroxide) $1 \mathrm { ~ x ~ } 3 . 2 \mathrm { ~ m l }$ per bottle

# J. Substantial Equivalence Information:

1. Predicate device name(s): Xpert C. difficile Assay BD GeneOhm™ Cdiff Assay

2. Predicate Numbers (s): K091109, K081920

3. Comparison with predicate:

# Similarities

<table><tr><td colspan="1" rowspan="1">Item</td><td colspan="3" rowspan="1">New DeviceXpert C. difficile/Epi AssayK110203</td><td colspan="2" rowspan="1">Predicate 1Xpert C. difficile AssayK091109</td><td colspan="1" rowspan="1">Predicate 2BD GeneOhm Cdiff AssayK081920</td></tr><tr><td colspan="1" rowspan="4">Intended Use</td><td colspan="3" rowspan="4">The Cepheid Xpert® C. difficile/Epi Assay isa qualitative in vitro diagnostic test for rapiddetection of toxin B gene sequences and forpresumptive identification of 027/NAP1/BIstrains of toxigenic Clostridium difficilefrom unformed (liquid or soft) stoolspecimens collected from patients suspectedof having C. difficile infection (CDI).Presumptive identification of 027/NAP1/BIstrains of C. difficile is by detection of binarytoxin (CDT) gene sequences and the singlebase pair deletion at nucleotide 117 in thetcdC gene. The tcdC gene encodes for anegative regulator in C. difficile toxinproduction. The test is performed on theCepheid GeneXpert® Dx System andutilizes automated real-time polymerasechain reaction (PCR) to detect toxin genesequences associated with toxin producingC. difficile. The Xpert C. difficile/Epi Assayis intended as an aid in the diagnosis of CDI.Detection of 027/NAP1/BI strains of C.difficile by the Xpert C. difficile/Epi Assay ispresumptive and is solely forepidemiological purposes and is not intendedto guide or monitor treatment for C. difficileinfections. Concomitant culture is necessaryonly if further typing or organism recovery isrequired.</td><td colspan="2" rowspan="2">The Cepheid Xpert C.difficile Assay, performedon the Cepheid GeneXpert®Dx System, is a qualitativein vitro diagnostic test forrapid detection of toxin Bgenesequences from unformed(liquid or soft) stoolspecimens collected frompatients suspected of havingClostridiumdifficile infection (CDI).The test utilizes automatedreal-time polymerase chainreaction (PCR) to detecttoxin gene sequencesassociated with toxin</td><td colspan="1" rowspan="4">The BD GeneOhmTM CdiffAssay is a rapid in vitrodiagnostic test for the direct,qualitative detection of C.difficile toxin B gene (tcdB)in human liquid orsoft stool specimens frompatients suspected of havingClostridiumdifficile-associated disease(CDAD). The test, based onreal-time PCR, is intendedfor use as an aid indiagnosis of CDAD. Thetest is performed directly onthe specimen, utilizingpolymerase chain reaction(PCR) for the amplificationof specific targets andfluorogenic target-specificHybridization probes for thedetection of the amplifiedDNA.</td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">g</td><td colspan="2" rowspan="1">producing C. difficile. The</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">Xpert C. difficile Assay isintended as an aid in thediagnosis of CDI.Concomitant cultureis necessary only if furthertyping or organism recoveryis required.</td></tr><tr><td colspan="1" rowspan="1">Technological</td><td colspan="3" rowspan="1">Fully-automated nucleic acid amplification</td><td colspan="2" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">New DeviceXpert C. difficile/Epi AssayK110203</td><td colspan="1" rowspan="1">Predicate 1Xpert C. difficile AssayK091109</td><td colspan="1" rowspan="1">Predicate 2BD GeneOhm Cdiff AssayK081920</td></tr><tr><td colspan="1" rowspan="1">Principles</td><td colspan="1" rowspan="1">(DNA); real-time PCR</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Unformed (liquid or soft) Stool</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Test Cartridge</td><td colspan="1" rowspan="1">Disposable single-use, multichamberedfluidic cartridge.</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Disposable single use PCRtube</td></tr><tr><td colspan="1" rowspan="1">DNA TargetSequences</td><td colspan="1" rowspan="1">C. difficile toxin B, binary toxin and the tcdCdeletion nt 117 (tcdCΔ117)</td><td colspan="1" rowspan="1">C. difficile toxin B only</td><td colspan="1" rowspan="1">C. difficile toxin B only</td></tr><tr><td colspan="1" rowspan="1">InstrumentSystem</td><td colspan="1" rowspan="1">Cepheid GeneXpert Dx System</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Cepheid SmartCycler DxSystem</td></tr><tr><td colspan="1" rowspan="1">Sample Extraction</td><td colspan="1" rowspan="1">Self-contained and automated after swabelution and two single-dose reagentadditions.</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Manual</td></tr><tr><td colspan="1" rowspan="1">Probes</td><td colspan="1" rowspan="1">TaqMan® Probes</td><td colspan="1" rowspan="1">Same</td><td colspan="1" rowspan="1">Molecular Beacons</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

• CLSI EP5-A2, Evaluation of Precision Performance of Quantitative Measurement Methods; Approved Guideline –Second Edition   
• ASTM D4169-05 Standard Practice for Performance Testing of Shipping Containers and Systems Guidance for Industry and FDA Staff: Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable, issued on April 25, 2006 Guidance for Industry and FDA Staff: Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile, issued on November 29, 2010 Draft Guidance for Industry and FDA Staff: Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens, December 8, 2005. Guidance for Industry and FDA Staff: Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, issued on May 11, 2005

# L. Test Principle:

The assay is a rapid, automated in vitro diagnostic test for the qualitative detection of C. difficile DNA directly from unformed (liquid or soft) stool specimens of patients suspected of having C. difficile infection (CDI). The assay detects the following three elements: toxin B gene (tcdB), binary toxin gene (CDT), and the single-base-pair deletion at nucleotide 117 within the gene encoding a negative regulator of toxin production (tcdCΔ117). The combined presence of the genes encoding toxin B, binary toxin, and the tcdCΔ117deletion have been associated with an epidemic toxigenic C. difficile strain known as 027/NAP1/BI. This strain designation terminology refers to designations given to this strain based on PCR ribotyping, Pulsed Field Gel Electrophoresis (PFGE) and restriction endonuclease analysis (REA), respectively.

The assay is performed on the Cepheid GeneXpert Dx System, which consists of the GeneXpert instrument, personal computer, hand-held barcode scanner, and disposable fluidic cartridges that are designed to complete sample preparation and real-time PCR. Each instrument contains 1 to 16 randomly accessible modules that are each capable of performing separate sample preparation and real-time PCR tests. Each module contains a syringe drive for dispensing fluids, an ultrasonic horn for lysing cells or spores, and an $\mathrm { I \bar { C } O R E ^ { \mathbb { B } } }$ thermocycler for performing real-time PCR and detection with fluorescently labeled TaqMan® probes.

# M. Performance Characteristics:

1. Analytical performance:

a. Precision/Reproducibility:

A panel of 7 specimens with varying concentrations of toxigenic $C .$ . difficile (non-027/NAP1/BI strain) and C. difficile (027/NAP1/BI strain) were tested on 10 different days by two different operators at each of three laboratory sites (7 specimens x 2 operators/ day x 10 days x 3 sites) for a total of 60 test results per specimen. One lot of Xpert C. difficile/Epi Assay was used at each of the 3 testing sites. Xpert C. difficile/Epi Assays were performed according to the Xpert C. difficile/Epi Assay procedure. The results are summarized in the following two tables.

Summary of Reproducibility Results (all)   

<table><tr><td colspan="1" rowspan="2">Specimen ID and Level</td><td colspan="3" rowspan="1">% Agreementa</td><td colspan="1" rowspan="2">% Total Agreementby Sample</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. ifficile High Negative (0.1xLoDb</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. difficile Low Positive (1xLoD)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">85% (17/20)</td><td colspan="1" rowspan="1">85% (17/20)</td><td colspan="1" rowspan="1">90.0% (54/60)</td></tr><tr><td colspan="1" rowspan="1">Toxigenic C. difficile Moderate Positive(10xLoD)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BI High Negative (0.1xLoD)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BI Low Positive (1xLoD)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">95% (19/20)</td><td colspan="1" rowspan="1">96.7% (58/60)</td></tr><tr><td colspan="1" rowspan="1">027/NAP1/BI Moderate Positive (10xLoD)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (20/20)</td><td colspan="1" rowspan="1">100% (60/60)</td></tr><tr><td colspan="1" rowspan="1">% Total Agreement by Site</td><td colspan="1" rowspan="1">100% (140/140)</td><td colspan="1" rowspan="1">97.1% (136/140)</td><td colspan="1" rowspan="1">97.1% (136/140)</td><td colspan="1" rowspan="1">98.1% (412/420)</td></tr></table>

a For Negative and High Negative samples, $\%$ Agreement $=$ (# negative results /total samples run); for Low and Moderate Positive samples, $\%$ Agreement $=$ (# positive results / total samples run). $\mathsf { \Delta E _ { I , O D } = }$ Limit of Detection

Summary of Ct Value Results by Sample Level and Target   

<table><tr><td rowspan=1 colspan=4>SPC</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Ave</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>32.26</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.22%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile High Negative (0.1xLoD)</td><td rowspan=1 colspan=1>32.17</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.83%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Low Positive (1xLoD)</td><td rowspan=1 colspan=1>32.14</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.66%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Moderate Positive (10xLoD)</td><td rowspan=1 colspan=1>31.98</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.47%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI High Negative (0.1xLoD)</td><td rowspan=1 colspan=1>32.11</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>2.03%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI Low Positive (1xLoD)</td><td rowspan=1 colspan=1>31.93</td><td rowspan=1 colspan=1>0.72</td><td rowspan=1 colspan=1>2.26%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI Moderate Positive (10xLoD)</td><td rowspan=1 colspan=1>31.96</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>1.90%</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=4>tcdB (Toxin B)</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>Ave</td><td rowspan=1 colspan=1>Std Dev</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile High Negative (0.1xLoD)</td><td rowspan=1 colspan=1>39.59</td><td rowspan=1 colspan=1>0.70</td><td rowspan=1 colspan=1>1.77%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Low Positive (1xLoD)</td><td rowspan=1 colspan=1>35.88</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.24%</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C. difficile Moderate Positive (10xLoD)</td><td rowspan=1 colspan=1>32.17</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>1.39%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI High Negative (0.1xLoD)</td><td rowspan=1 colspan=1>39.11</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>2.50%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI Low Positive (1xLoD)</td><td rowspan=1 colspan=1>35.49</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>1.65%</td></tr><tr><td rowspan=1 colspan=1>027/NAP1/BI Moderate Positive (10xLoD)</td><td rowspan=1 colspan=1>32.10</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>1.97%</td></tr></table>

An additional panel of 6 specimens, three Negative and three Toxigenic $C .$ . difficile High Negative, were tested on 5 different days by two different operators at each of the three sites (6 specimens x 2 operators/day x 5 days x 3 sites). The High Negative specimens were prepared at a concentration below LoD such that they were expected to give a negative result 20 to $80 \%$ of the time. One lot of Xpert C. difficile/Epi Assay was used at each of the 3 testing sites. Xpert C. difficile/Epi Assays were performed according to the Xpert $C$ .

difficile/Epi Assay procedure. The results are summarized in the following table.

Summary of Additional Reproducibility Specimen Results   

<table><tr><td rowspan=2 colspan=1>Specimen ID and Level</td><td rowspan=1 colspan=3>% Agreementa</td><td rowspan=2 colspan=1>% Total Agreementby Sample</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>100% (30/30)</td><td rowspan=1 colspan=1>100% (90/90)</td></tr><tr><td rowspan=1 colspan=1>Toxigenic C.dfficile High Negative</td><td rowspan=1 colspan=1>60.0% (18/30)b</td><td rowspan=1 colspan=1>60.0% (18/30) b</td><td rowspan=1 colspan=1>53.3% (16/30)b</td><td rowspan=1 colspan=1>57.8% (52/90)b</td></tr></table>

a For Negative and High Negative samples, $\%$ Agreement $=$ (# negative results / total samples run) $2 0 \mathrm { - } 8 0 \%$ agreement expected for High Negative sample

b. Linearity/assay reportable range: Not applicable.   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls

Internal Controls): The internal controls for the Xpert C. difficile/Epi Assay are the Sample Processing Control (SPC) and the Probe Check Control (PCC). Both the SPC and PCC are contained within each Xpert C. difficile/Epi cartridge.

The SPC, which consists of non-infectious Bacillus globigii spores, controls for adequate processing of the target bacteria and to monitor for the presence of inhibitor(s) in the PCR assay to avoid false-negative results. The pivotal clinical study data validated the pre-determined SPC minimum and maximum acceptable Ct settings of 5 to 40 cycles, respectively.

The PCC verifies reagent rehydration, PCR tube filling in the cartridge, probe integrity, and dye stability. The PCC is carried out before the start of the PCR reaction when the GeneXpert Dx System measures the fluorescence signal from the probes to monitor bead rehydration, reaction-tube filling, probe integrity and dye stability. The PCC is considered to “PASS” if the fluorescence generated meets the validated acceptance criteria using the Lot Specific Parameters (LSP) determination process.

An additional control is the System Control Check for Temperature. This check ensures that the GeneXpert Dx Instrument is operating within validated heating and cooling specifications.

External Controls: External positive and negative controls were validated for use with the Xpert C. difficile/Epi Assay but are not provided with the Xpert

C. difficile/Epi Assay. Strains for external controls and validation studies are commercially available and catalog numbers of available strains from several suppliers are listed in the Xpert C. difficile/Epi Assay Package Insert. .

# d. Detection Limits:

LoD studies were carried out with seven different strains of C. difficile representing toxinotypes 0 (two strains), III (two strains), IV, V and VIII (one strain each). The C. difficile strains were diluted into a fecal matrix consisting of human liquid feces (verified $C .$ . difficile negative by the Xpert $C .$ . difficile/Epi Assay) diluted in PBS with $1 5 \%$ glycerol. Each C. difficile level (CFU/swab) that was evaluated in the study was tested in replicates of twenty. The LoD is defined as the lowest number of colony forming units (CFU) per swab that can be reproducibly distinguished from negative samples with $9 5 \%$ confidence.

The estimate and confidence intervals were determined using logistic regression with data (number of positive results per number of replicates at each level) over the range of CFUs tested. The confidence intervals were determined using maximum likelihood estimates on the logistic model parameters using the large sample variance-covariance matrix. The LoD point estimates and $9 5 \%$ upper and lower confidence intervals for each C. difficile toxinotype tested are summarized in the following table.

$9 5 \%$ Confidence Intervals for Analytical LoD – C. difficile   

<table><tr><td rowspan=1 colspan=1>Strain ID</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>LoD95%(CFU/swab)</td><td rowspan=1 colspan=1>Lower 95% CI</td><td rowspan=1 colspan=1>Upper 95% CI</td></tr><tr><td rowspan=1 colspan=1>VPI 10463 (CCUG19126)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>255</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>632</td></tr><tr><td rowspan=1 colspan=1>90556-M6S (ATCC9689)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>460</td><td rowspan=1 colspan=1>419</td><td rowspan=1 colspan=1>587</td></tr><tr><td rowspan=1 colspan=1>LUMC-1 (027/NAP1/BI)a</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>LUMC-5 (027/NAP1/BI)a</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>176</td></tr><tr><td rowspan=1 colspan=1>LUMC-7</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=1>LUMC-6</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>9101</td><td rowspan=1 colspan=1>XII</td><td rowspan=1 colspan=1>41</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>49</td></tr></table>

a By PCR-ribotyping/pulse-field gel electrophoresis/restriction endonuclease analysis

# e. Analytical Reactivity:

In addition to the LoD determination, eighteen $C .$ . difficile strains representing toxinotype 0 plus twelve variant toxinotypes, including four 027/NAP1/BI toxinotype III isolates, were tested using the Xpert C. difficile/Epi Assay. C. difficile strains were selected to broadly represent the majority of C. difficile toxinotypes encountered in practice. Stock cultures were prepared by suspending the bacterial growth from agar plates in PBS buffer containing

$1 5 \%$ glycerol. The concentration of each stock was adjusted to 1.4-5.9 McFarland units. All strains were serially diluted to approximately 900 CFU/swab and tested in triplicate.

Under the conditions of this study, the Xpert C. difficile/Epi Assay correctly identified all eighteen strains tested as “Toxigenic C. diff POSITIVE”.

Included in the panel were 8 toxinotypes (III, IV, V, VI, IX, X, XIV, and XXII) reported to be positive for binary toxin (CDT) production as well. All replicates of toxinotypes V, VI, IX, and XXII were reported as “Toxigenic C. diff POSITIVE; 027-NAP1-BI PRESUMPTIVE NEGATIVE”. Two of three replicates of the toxinotype IV strain and one of three replicates of the toxinotype X strain were reported as “Toxigenic C. diff POSITIVE; 027- NAP1-BI PRESUMPTIVE POSITIVE”, although these two strains do not contain the tcdCΔ117deletion. Three of three replicates of the toxinotype XIV strain were reported as “Toxigenic C. diff POSITIVE; 027-NAP1-BI PRESUMPTIVE POSITIVE”. Toxinotype XIV has been reported to contain the tcdCΔ117deletion. All replicates of the four 027/NAP1/BI isolates (representing toxinotype III) were reported as “Toxigenic C. diff POSITIVE; 027-NAP1-BI PRESUMPTIVE POSITIVE”. Statements were added to the Limitations Section of the Xpert C. difficile/Epi Assay Package Insert to reflect the reactivity of the Xpert C. difficile/Epi Assay with toxinotype XIV and occasional reactivity of the assay with toxinotypes IV and X. The results of the Analytical Reactivity Study are summarized below.

Reactivity of the Xpert C. difficile/Epi Assay with Strains of C. difficile   

<table><tr><td colspan="1" rowspan="1">Toxinotype</td><td colspan="1" rowspan="1">No. of Strains</td><td colspan="1" rowspan="1">Xpert C. difficile/Epi Assay Result</td></tr><tr><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEGa</td></tr><tr><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">I</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE POSb</td></tr><tr><td colspan="1" rowspan="1">IV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE POSc</td></tr><tr><td colspan="1" rowspan="1">V</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">VI</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">VIII</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">IX</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">X</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">027-NAP1-BI PRESUMPTIVE NEGd</td></tr><tr><td colspan="1" rowspan="1">XII</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">XIV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE POSe</td></tr><tr><td colspan="1" rowspan="1">XXI</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr><tr><td colspan="1" rowspan="1">XXII</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Toxigenic C. diff POSITIVE;027-NAP1-BI PRESUMPTIVE NEG</td></tr></table>

a All three strains gave the same results for their three replicates. $^ { \mathrm { b } } \mathrm { A l l }$ four strains gave the same results for their three replicates. c Two of three replicates were reported 027-NAP1-BI PRESUMPTIVE POS d One of three replicates was reported 027-NAP1-BI PRESUMPTIVE POS e Three of three replicates were reported 027-NAP1-BI PRESUMPTIVE POS

# f. Analytical Specificity:

Fifty-five (55) strains were collected, quantitated and tested using the Xpert $C .$ . difficile/Epi Assay. The strains originated from the American Type Culture Collection (ATCC), Culture Collection University of Göteborg (CCUG), German Collection of Microorganisms and Cell Cultures (DSMZ), the Centers for Disease Control and Prevention (CDC), the Institute of Public Health, Maribor, Slovenia and the Swedish Institute for Infectious Disease Control (SMI).

Of the tested species, ten (10) non-toxigenic C. difficile strains and eleven (11) non $C .$ . difficile Clostridium species were included. The organisms tested were identified as either Gram-positive (37) or Gram negative (18). The organisms were further classified as aerobic (24), anaerobic (29) or microaerophilic (2).

Each strain was tested in triplicate at concentrations ranging from $1 . 1 \mathrm { x } 1 0 ^ { 8 }$ to $2 . 2 \mathrm { x } 1 0 ^ { 1 0 }$ CFU/swab. Positive and negative controls were included in the study. Under the conditions of the study, all isolates were reported “Toxigenic C. diff NEGATIVE; 027-NAP1-BI PRESUMPTIVE NEG”. The Cross Reactivity Panel used in this study is summarized below.

# Cross Reactivity Panel

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Level(CFU/swab)</td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Level(CFU/swab)</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter</td><td colspan="1" rowspan="1">1.2x1010</td><td colspan="1" rowspan="1">Corynebacterium diphtheriae</td><td colspan="1" rowspan="1">1.4x1010</td></tr><tr><td colspan="1" rowspan="1">Anaerococcus prevotii</td><td colspan="1" rowspan="1">2.7x109</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.4x1010</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">6.8x109</td><td colspan="1" rowspan="1">Enterococcus casseliflavus</td><td colspan="1" rowspan="1">6.8x109</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">8.1x109</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.1x1010</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">3.1x10</td><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">4.7x109</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">3.1x109</td><td colspan="1" rowspan="1">Enterococcus gallinarum</td><td colspan="1" rowspan="1">8.1x109</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">5.4x109</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.3x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium botulinum</td><td colspan="1" rowspan="1">1.1x1010</td><td colspan="1" rowspan="1">Fusobacterium necrophorum</td><td colspan="1" rowspan="1">7.4x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">7.3x10</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">8.5x108</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">6.8x109</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">7.4x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">4.1x109</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">2.0x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">1.1x108</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">6.8x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">2.5x10</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">1.4x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">2.5x10</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">1.1x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">2.6x109</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">5.4x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">2.6x109</td><td colspan="1" rowspan="1">Prevotella loescheii</td><td colspan="1" rowspan="1">8.8x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">1.9x109</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.5x100</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">4.4x109</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">1.6x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium difficile (nontoxigenic)</td><td colspan="1" rowspan="1">2.8x109</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">4.7x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.1x1010</td><td colspan="1" rowspan="1">Salmonella enterica</td><td colspan="1" rowspan="1">1.3x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">3.8x109</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">8.8x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">1.6x109</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1.2x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">4.9x108</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">2.0x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium septicum</td><td colspan="1" rowspan="1">2.1x109</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">2.3x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">6.1x109</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">8.5x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium spiroforme</td><td colspan="1" rowspan="1">7.0x108</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">4.1x109</td></tr><tr><td colspan="1" rowspan="1">Clostridium tertium</td><td colspan="1" rowspan="1">4.9x108</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">1.7x1010</td></tr><tr><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">4.1x109</td><td colspan="1" rowspan="1">Negative control</td><td colspan="1" rowspan="1">Buffer only</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Positive control</td><td colspan="1" rowspan="1">1.5x103</td></tr></table>

# g. Interference Studies:

Twenty-one (21) biological and chemical substances occasionally used or found in stool specimens were tested for interference with the Xpert $C .$ . difficile/Epi Assay. All substances were tested at levels representing approximately $1 - 1 0 \%$ (v/v or w/v) final concentration. Four strains of $C .$ . difficile, representing toxinotypes 0, III (two strains), and VIII, were used in the study at 600 CFU/swab. Replicates of eight were tested using each strain and each substance for a total of 706 runs including controls. Interference was determined by comparing Ct values for each Xpert $C .$ . difficile/Epi target in the presence of each substance relative to the Ct values obtained for controls. The following table lists the substances that were tested in this study.

# Substances Tested

<table><tr><td rowspan=1 colspan=1>Substance (Supplier)</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>AmountTested</td></tr><tr><td rowspan=1 colspan=1>PBS 15% glycerol control (Cepheid)</td><td rowspan=1 colspan=1>Cepheid</td><td rowspan=1 colspan=1>Control</td></tr><tr><td rowspan=1 colspan=1>Whole Blood(Karolinska University Hospital)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Mucin (porcine) (Sigma)</td><td rowspan=1 colspan=1>Porcine mucin representing denselyglycosylated proteins</td><td rowspan=1 colspan=1>5%(w/v)</td></tr><tr><td rowspan=1 colspan=1>Kaopectate® (Chattem)</td><td rowspan=1 colspan=1>Docusate calcium (240 mg/tablet)</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Imodium® (McNeil-PPC)</td><td rowspan=1 colspan=1>Loperamide HCl (1 mg/7.5 mL)</td><td rowspan=1 colspan=1>10% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol® (Procter &amp; Gamble)</td><td rowspan=1 colspan=1>Bismuth subsalicylate (15%)</td><td rowspan=1 colspan=1>10% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Preparation H¥®(Wyeth Consumer Healthcare)</td><td rowspan=1 colspan=1>Light Mineral Oil (14%) Petrolatum(71.9%) Phenylephrine HC1 (0.25%)Shark liver oil (3%)</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Fleet® (C.B. Fleet Company)</td><td rowspan=1 colspan=1>Glycerin (2 g/suppository)</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Fecal fats* (Karolinska University Hospital</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>130 mmol</td></tr><tr><td rowspan=1 colspan=1>Monistat® (McNeil-PPC)</td><td rowspan=1 colspan=1>Miconazole Nitrate (2%)</td><td rowspan=1 colspan=1>1%(w/v)</td></tr><tr><td rowspan=1 colspan=1>Hydrocortisone Cream (Longs Drugs)</td><td rowspan=1 colspan=1>Hydrocortisone (1%)</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>K-Y Jell/Gelée® (McNeil-PPC)</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Vagisil® Original Strength Cream (Combe)a</td><td rowspan=1 colspan=1>Benzocaine (5%) Resorcinol (2%)</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Boudreaux&#x27;s Butt Paste® (Blairex Laboratories)b</td><td rowspan=1 colspan=1>Zinc Oxide (16%)</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Vaseline (Unilever)</td><td rowspan=1 colspan=1>Benzocaine (5%) Resorcinol (2%)</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Dulcolax® (Boehringer IngelheimPharmaceuticals)</td><td rowspan=1 colspan=1>Bisocadyl (5 mg)</td><td rowspan=1 colspan=1>0.02% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Preparation H Portable Wipes(Wyeth Consumer Healthcare)</td><td rowspan=1 colspan=1>Witch hazel (50%)</td><td rowspan=1 colspan=1>1 Wipe</td></tr><tr><td rowspan=1 colspan=1>Vaginal Contraceptive Film (VCF)(Apothecus Pharmaceutical)</td><td rowspan=1 colspan=1>Nonoxynol-9 (28%)</td><td rowspan=1 colspan=1>1 Film</td></tr><tr><td rowspan=1 colspan=1>Vancomycin (Fluka)</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>1% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Metronidazole (Actavis)</td><td rowspan=1 colspan=1>Metronidazole (500 mg/tablet)</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>Anusol® Plus (TM Warner-Lambert Company)</td><td rowspan=1 colspan=1>Pramoxine Hydrochloride (20mg/suppository) Zinc sulfatemonohydrate (10 mg/suppository)</td><td rowspan=1 colspan=1>10% (w/v)</td></tr><tr><td rowspan=1 colspan=1>E-Z-HD™ High Density Barium Sulfate forSuspension (E-Z-EM Canada)</td><td rowspan=1 colspan=1>Barium sulfate (210% w/v)</td><td rowspan=1 colspan=1>20% (stockdiluted 1:10)</td></tr></table>

a,b Potentially interfering

Potentially interfering substances include, but are not limited to, Vagisil cream and zinc oxide paste (noted in the Limitations Section of the Xpert C. difficile/Epi Assay Package Insert). The other nineteen substances listed in the table immediately above showed no detectable interference with the Xpert C. difficile/Epi Assay.

# h. Assay cut-off:

# Lot Specific Parameters and Assay Settings

Lot specific assay settings are generated for every lot manufactured to account for slight variations in reagent production. The lot specific assay settings (LSP file) are incorporated into the 2-D barcode on each cartridge label and are transferred to the GeneXpert Dx system via a hand-held barcode scanner prior to initiating the Xpert C. difficile/Epi Assay.

# General Assay Settings

General assay settings are used for all reagent lots. They are fixed and not part of the LSP process. The following table lists general assay settings:

<table><tr><td rowspan=1 colspan=1>Attribute</td><td rowspan=1 colspan=1>Setting</td></tr><tr><td rowspan=1 colspan=1>Background Subtraction</td><td rowspan=1 colspan=1>Always ON</td></tr><tr><td rowspan=1 colspan=1>Background Minimum Cycle</td><td rowspan=1 colspan=1>Default setting = 5</td></tr><tr><td rowspan=1 colspan=1>Background Maximum Cycle</td><td rowspan=1 colspan=1>Manual setting = 30</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (SPC)</td><td rowspan=1 colspan=1>Manual setting = 15</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (tcdB and tcdC)</td><td rowspan=1 colspan=1>Manual setting = 20</td></tr><tr><td rowspan=1 colspan=1>Manual Threshold (CDT)</td><td rowspan=1 colspan=1>Manual setting = 30</td></tr><tr><td rowspan=1 colspan=1>Curve Analysis</td><td rowspan=1 colspan=1>Primary</td></tr><tr><td rowspan=1 colspan=1>Boxcar Average Cycles</td><td rowspan=1 colspan=1>Zero (OFF)</td></tr><tr><td rowspan=1 colspan=1>Valid Minimum Ct (SPC, tcdB, CDT and tcdC)</td><td rowspan=1 colspan=1>Manual setting = 5</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (tcdB and CDT)</td><td rowspan=1 colspan=1>Manual setting = 37</td></tr><tr><td rowspan=1 colspan=1>Valid Maximum Ct (SPC and tcdC)</td><td rowspan=1 colspan=1>Manual setting = 40</td></tr><tr><td rowspan=1 colspan=1>Delta Ct Max</td><td rowspan=1 colspan=1>Manual setting = 4.0</td></tr></table>

The valid cycle range for binary toxin (CDT) target is 5 to 37 cycles and the valid range for the tcdCΔ117 target is 5 to 40 cycles as determined from preclinical results. Based on pre-clinical results relative to toxigenic culture and strain typing methods, a valid maximum $\mathrm { C t }$ of 37.0 for toxin B (tcdB) and binary toxin (CDT) and 40.0 for tcdCΔ117 were used as the cutoff $\mathrm { C t }$ values to differentiate between positive and negative results in the clinical study. These cutoffs were subsequently validated in the pivotal clinical study.

The valid cycle range for the SPC is 5 to 40 cycles and was derived from simulated inhibitory studies with potentially interfering substances and preclinical negative data $\scriptstyle ( \mathrm { n = 1 0 6 6 } )$ . The SPC Ct settings were subsequently validated in the pivotal clinical study. A valid maximum SPC Ct of 40 cycles was set to differentiate between valid negative results and invalid results in the pivotal clinical study. The SPC was designed to be sensitive to the presence of inhibition, while at the same time limiting the number of invalid test results. In both interference testing and pre-clinical testing, the observed SPC Ct distribution was wide enough that a maximum valid cycle of 40 is justified to ensure the detection of all sample processing control results, maintain a low invalid rate, and keep false negative results to a minimum.

# i. Carry-over/Cross-Contamination

A study was carried out to verify that single–use; self–contained GeneXpert cartridges prevent carry–over contamination in negative samples that are run immediately after very high positive samples have been run in the same GeneXpert module. The study consisted of a negative sample (no C. difficile cells present) processed in the same GeneXpert module immediately

following a very high positive sample made up of 1.5x10 Clostridium difficile (027/NAP1/BI strain) cells spiked into the elution buffer. This was repeated 20 times between two GeneXpert modules for a total of 40 runs. All 20 negative samples were correctly reported “Toxigenic C. diff NEGATIVE; 027- NAP1-BI PRESUMPTIVE NEGATIVE”. All 20 high positive samples were correctly reported as “Toxigenic C. diff POSITIVE; 027-NAP1-BI PRESUMPTIVE POSITIVE”.

2. Comparison studies:

# a. Method comparison with reference methods:

The clinical performance evaluation was done against reference culture, followed by cell cytotoxicity testing on the isolates and strain typing on the toxigenic strains by PCR-ribotyping, pulsed-field gel electrophoresis (PFGE) and restriction endonuclease analysis (REA) methods.

b. Matrix Comparison:

Not applicable

3. Clinical studies:

Clinical performance characteristics of the Xpert C. difficile/Epi Assay were determined by testing a total of 2293 leftover unformed stool specimens from subjects whose routine care called for $C .$ . difficile testing. The prospective clinical study was carried out at seven US and Canadian institutions by comparing the Xpert C. difficile/Epi Assay to reference culture followed by cell cytotoxicity

testing on the isolates and strain typing on the toxigenic strains by PCRribotyping, pulsed-field gel electrophoresis (PFGE) and restriction endonuclease analysis (REA) methods.

A portion of each leftover unformed stool specimen was obtained for testing by the Xpert C. difficile/Epi Assay. The remaining excess specimen was sent to a central laboratory for reference culture and cytotoxin B isolate testing. Each stool specimen was inoculated onto pre-reduced cycloserine-cefoxitin-fructose agar – direct plate (CCFA-D) and cycloserine cefoxitin mannitol broth with taurocholate lysozyme cysteine (CCMB-TAL). After 24 hours, the CCMB-TAL was subcultured on to a second CCFA-E plate (CCFA- Enriched). This directenriched culture method is referred to as “reference culture”.

If C. difficile was isolated from the CCFA-D plate and the isolate was positive by cell cytotoxicity assay, the specimen was classified as “toxigenic C. difficile positive” and the corresponding CCFA-E plate was not further analyzed. If no $C .$ . difficile was isolated from the CCFA-D plate or if the isolate was negative by cell cytotoxicity assay, the corresponding CCFA-E plate was further analyzed.

If CCFA-E was positive for $C .$ difficile and the isolate was positive for cell cytotoxicity assay, the specimen was classified as “toxigenic $C .$ difficile positive”. The specimen was reported as “negative” if CCFA-E was negative for $C .$ . difficile or the isolate was tested negative by cell cytotoxicity assay.

Following reference culture testing, the toxigenic $C .$ . difficile positive isolates were sent to a second set of reference laboratories for strain identification by REA, PFGE, and PCR-ribotyping.

Performance of the Xpert C. difficile/Epi Assay was calculated relative to the results of direct culture with strain typing, for each of the three strain typing methods, and relative to the results of reference culture with strain typing, for each of the three strain typing methods.

# Performance vs. Direct Culture and REA

Relative to direct culture with REA strain typing, the Xpert $C .$ . difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic C. difficile of $9 8 . 7 2 \%$ and $9 0 . 8 6 \%$ , respectively. The Xpert $C .$ . difficile/Epi Assay also demonstrated a $9 8 . 5 5 \%$ positive agreement and $9 7 . 6 5 \%$ negative agreement for BI. The results are summarized in the following table.

Xpert C. difficile/Epi Assay Performance vs. Direct Culture & REA   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Direct Culture &amp; REA</td></tr><tr><td rowspan=1 colspan=1>Toxin B +BI +</td><td rowspan=1 colspan=1>Toxin B +BI-</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totalb</td></tr><tr><td rowspan=4 colspan=1>  pl</td><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI +</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI </td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>299</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1860</td><td rowspan=1 colspan=1>1863</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>2047</td><td rowspan=1 colspan=1>2282</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity:Specificity:Accuracy:PPVcNPVd.</td><td rowspan=1 colspan=1>98.72% (232/235)90.86% (1860/2047)91.67% (2092/2282)55.37% (232/419)99.84% (1860/1863)</td><td rowspan=1 colspan=1>Pos Agreement:Neg Agreement:Accuracy:PPV:NPV:</td><td rowspan=1 colspan=1>98.55% (68/69)97.65% (2161/2213)97.68% (2229/2282)56.67% (68/120)99.95% (2161/2162)</td></tr></table>

$^ \mathrm { ~ b ~ } _ { 1 1 }$ specimens were culture positive but were not strain typed for the following reasons: incomplete restriction endonuclease digestion; or the isolate was not sent. These 11 specimens are not included in the performance characteristics above. c Positive Predictive Value d Negative Predictive Value

a Xpert results were obtained for 2208 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 2 \%$ of the specimens were indeterminate on the first attempt.

# Performance vs. Direct Culture and PFGE

Relative to direct culture with PFGE strain typing, the Xpert C. difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 8 . 7 6 \%$ and $9 0 . 8 6 \%$ , respectively. The Xpert $C .$ . difficile/Epi Assay also demonstrated a $100 \%$ positive agreement and $9 7 . 6 1 \%$ negative agreement for NAP1. The results are summarized in the following table.

Xpert C. difficile/Epi Assay Performance vs. Direct Culture & PFGE   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Direct Culture &amp; PFGE</td></tr><tr><td rowspan=1 colspan=1>Toxin B +NAP1 +</td><td rowspan=1 colspan=1>Toxin B +NAP1 -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totalb</td></tr><tr><td rowspan=4 colspan=1>  pdl</td><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI +</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>124</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI </td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>301</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1860</td><td rowspan=1 colspan=1>1863</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>169</td><td rowspan=1 colspan=1>2047</td><td rowspan=1 colspan=1>2288</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity:Specificity:Accuracy:PPVcNP</td><td rowspan=1 colspan=1>98.76% (238/241)90.86% (1860/2047)91.70% (2098/2288)56.00% (238/425)99.84% (1860/1863)</td><td rowspan=1 colspan=1>Pos Agreement:Neg Agreement:Accuracy:PPV:NPV:</td><td rowspan=1 colspan=1>100% (71/71)97.61% (2163/2216)97.68% (2234/2288)57.26% (71/124)100% (2164/2164)</td></tr></table>

a Xpert results were obtained for 2214 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 2 \%$ of the specimens were indeterminate on the first attempt. $^ { \mathrm { ~ b ~ } } 5$ specimens were culture positive but were not strain typed for the following reasons: incomplete restriction endonuclease digestion; no growth; or contamination. These 5 specimens are not included in the performance characteristics above. c Positive predictive value d Negative predictive value

# Performance vs. Direct Culture and PCR-Ribotyping

Relative to direct culture with PCR-ribotyping, the Xpert C. difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 8 . 7 8 \%$ and $9 0 . 8 6 \%$ , respectively. The Xpert $C .$ . difficile/Epi Assay also demonstrated a $100 \%$ positive agreement and $9 7 . 7 0 \%$ negative agreement for 027 by PCR-ribotyping. The results are summarized in the following table.

Xpert C. difficile/Epi Assay Performance vs. Direct Culture & PCR-Ribotyping   

<table><tr><td colspan="2" rowspan="2"></td><td colspan="4" rowspan="1">Direct Culture &amp; PCR-Ribotyping</td></tr><tr><td colspan="1" rowspan="1">Toxin B +027 +</td><td colspan="1" rowspan="1">Toxin B +027</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Totalb</td></tr><tr><td colspan="1" rowspan="3">  pdl</td><td colspan="1" rowspan="1">Toxin B +027/NAP1/BI +</td><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">125</td></tr><tr><td colspan="1" rowspan="1">Toxin B +027/NAP1/BI -</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">164</td><td colspan="1" rowspan="1">140</td><td colspan="1" rowspan="1">304</td></tr><tr><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1860</td><td colspan="1" rowspan="1">1863</td></tr><tr><td></td><td>Total</td><td>74</td><td>171</td><td>2047</td><td>2292</td></tr><tr><td colspan="2"></td><td colspan="2">Toxigenic C. difficile</td><td colspan="2">Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td colspan="2" rowspan="6"></td><td>Sensitivity:</td><td>98.78% (242/245)</td><td>Pos Agreement:</td><td>100% (74/74)</td></tr><tr><td>Specificity:</td><td>90.86% (1860/2047)</td><td>Neg Agreement:</td><td>97.70% (2167/2218)</td></tr><tr><td>Acccuracy.</td><td>91.71% (2102/2292)</td><td>Accuracy:</td><td>97.77% (2241/2292)</td></tr><tr><td>PPVc</td><td>56.41% (242/429)</td><td>PPV:</td><td></td></tr><tr><td>NPVd.</td><td>99.84% (1860/1863)</td><td>NPV:</td><td>59.20% (74/125) 100% (2218/2218)</td></tr></table>

a Xpert results were obtained for 2218 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 2 \%$ of the specimens were indeterminate on the first attempt. b One isolate could not be typed due to contamination; this specimen is not included in the performance statistics. c Positive predictive value d Negative predictive value

# Performance vs. Reference Culture and REA

Reference (enriched) culture is a more sensitive method for detection of $C .$ . difficile in symptomatic patients. For example, it enhances detection of low number of organisms in samples due to prior antibiotic treatment and potential loss of viability due to specimen transport.

Relative to reference culture with REA strain typing, the Xpert C. difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic C. difficile of $9 3 . 3 5 \%$ and $9 4 . 0 2 \%$ , respectively. The Xpert C. difficile/Epi Assay also demonstrated a $9 6 . 5 1 \%$ positive agreement and $9 8 . 3 1 \%$ negative agreement for BI.

Xpert C. difficile/Epi Assay Performance vs. Reference Culture & REA   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Reference Culture &amp; REA</td></tr><tr><td rowspan=1 colspan=1>Toxin B +BI +</td><td rowspan=1 colspan=1>Toxin B +BI -</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totalb</td></tr><tr><td rowspan=4 colspan=1>B  pl</td><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI +</td><td rowspan=1 colspan=1>83</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>120</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI </td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>204</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>292</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1841</td><td rowspan=1 colspan=1>1862</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>230</td><td rowspan=1 colspan=1>1958</td><td rowspan=1 colspan=1>2274</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity:Specificity:Accuracy:PPVcNPV</td><td rowspan=1 colspan=1>93.35% (295/316)94.02% (1841/1958)93.93% (2136/2274)71.60% (295/412)98.87% (1841/1862)</td><td rowspan=1 colspan=1>Pos Agreement:Neg Agreement:Accuracy:PPV:NPV:</td><td rowspan=1 colspan=1>96.51% (83/86)98.31% (2151/2188)98.24% (2234/2274)69.17% (83/120)99.86% (2151/2154)</td></tr></table>

a Xpert results were obtained for 2200 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 3 \%$ of the specimens were indeterminate on the first attempt.

$^ { \mathsf { b } } 1 9$ specimens were culture positive but were not strain typed for the following reasons: incomplete restriction endonuclease digestion; or the isolate was not sent. These 19 specimens are not included in the performance characteristics above.   
c Positive predictive value   
d Negative predictive value

# Performance vs. Reference Culture and PFGE

Relative to reference culture with PFGE strain typing, the Xpert C. difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic $C .$ . difficile of $9 3 . 6 0 \%$ and $9 4 . 0 2 \%$ , respectively. The Xpert C. difficile/Epi Assay also demonstrated a $9 7 . 7 3 \%$ positive agreement and $9 8 . 2 7 \%$ negative agreement for NAP1.

Xpert C. difficile/Epi Assay Performance vs. Reference Culture & PFGE   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Reference Culture &amp; PFGE</td></tr><tr><td rowspan=1 colspan=1>Toxin B +NAP1 +</td><td rowspan=1 colspan=1>Toxin B +NAP1-</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totalb</td></tr><tr><td rowspan=4 colspan=1>  dx</td><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI +</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>124</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI </td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>213</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1841</td><td rowspan=1 colspan=1>1862</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>240</td><td rowspan=1 colspan=1>1958</td><td rowspan=1 colspan=1>2286</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity:Specificity:Accuracy:PPVc:NPVd.</td><td rowspan=1 colspan=1>93.60% (307/328)94.02% (1841/1958)93.96% (2148/2286)72.41% (307/424)98.87% (1841/1862)</td><td rowspan=1 colspan=1>Pos Agreement:Neg Agreement:Accuracy:PPV:NPV:</td><td rowspan=1 colspan=1>97.73% (86/88)98.27% (2160/2198)98.25% (2246/2286)69.35% (86/124)99.91% (2160/2162)</td></tr></table>

a Xpert results were obtained for 2212 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 2 \%$ of the specimens were indeterminate on the first attempt. $^ \mathrm { b } 7$ specimens were culture positive but were not strain typed for the following reasons: incomplete restriction endonuclease digestion; no growth; or contamination. These 11 specimens are not included in the performance characteristics above. c Positive predictive value d Negative predictive value

# Performance vs. Reference Culture and PCR-Ribotyping

Relative to reference culture with PCR-ribotyping, the Xpert C. difficile/Epi Assay demonstrated a sensitivity and specificity for toxigenic C. difficile of $9 3 . 3 9 \%$ and $9 4 . 0 2 \%$ , respectively. The Xpert C. difficile/Epi Assay also demonstrated a $9 8 . 8 9 \%$ positive agreement and $9 8 . 3 6 \%$ negative agreement for 027 by PCR-ribotyping.

Xpert C. difficile/Epi Assay Performance vs. Reference Culture & PCR-Ribotyping   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=4>Reference Culture &amp; PCR-Ribotyping</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027+</td><td rowspan=1 colspan=1>Toxin B +027</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Totalb</td></tr><tr><td rowspan=4 colspan=1>B pl</td><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI +</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Toxin B +027/NAP1/BI </td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>217</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>303</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1841</td><td rowspan=1 colspan=1>1863</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>243</td><td rowspan=1 colspan=1>1958</td><td rowspan=1 colspan=1>2291</td></tr><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Toxigenic C. difficile</td><td rowspan=1 colspan=2>Toxigenic C. difficile / 027/NAP1/BI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity:Specificity:Accuracy:PPVc:NPVd.</td><td rowspan=1 colspan=1>93.39% (311/333)94.02% (1841/1958)93.93% (2152/2291)72.66% (311/428)98.82% (1841/1863)</td><td rowspan=1 colspan=1>Pos Agreement:Neg Agreement:Accuracy:PPV:NPV:</td><td rowspan=1 colspan=1>98.89% (89/90)98.36% (2165/2201)98.38% (2254/2291)71.20% (89/125)99.95% (2165/2166)</td></tr></table>

a Xpert results were obtained for 2217 samples on the first attempt. Results for the other 74 samples were obtained on the second attempt. Approximately $3 . 2 \%$ of the specimens were indeterminate on the first attempt. $^ { \flat } 2$ specimens were culture positive but were not strain typeable due to contamination and are not included in the performance characteristics above. b Positive predictive value c Negative predictive value

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

In the Xpert C. difficile/Epi Assay clinical study, a total of 2293 unformed stool specimens were included from seven centers across the United States and Canada. The number and percentage of toxigenic $C .$ . difficile positive cases by culture, calculated by age and gender, are presented in Table 2a and Table 2b, respectively.

Observed Prevalence of Toxigenic C. difficile by Age Groupa   

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Toxigenic C. difficile Prevalence</td><td rowspan=1 colspan=1>027/NAP1/BI Prevalence</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>37.5% (6/16)</td><td rowspan=1 colspan=1>12.5% (2/16)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>12.4% (13/105)</td><td rowspan=1 colspan=1>0.9% (1/105)</td></tr><tr><td rowspan=1 colspan=1>22-59</td><td rowspan=1 colspan=1>898</td><td rowspan=1 colspan=1>16.4% (147/898)</td><td rowspan=1 colspan=1>3.3% (30/898)</td></tr><tr><td rowspan=1 colspan=1>&gt;60</td><td rowspan=1 colspan=1>1274</td><td rowspan=1 colspan=1>20.7% (264/1274)</td><td rowspan=1 colspan=1>7.2% (92/1274)</td></tr></table>

a Prevalence based on Xpert results. b Toxigenic C. difficile prevalence includes both 027/NAP1/BI strains and non 027/NAP1/BI strains

Observed Prevalence of Toxigenic $C .$ difficile by Gendera   

<table><tr><td rowspan=1 colspan=1>Gender</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Toxigenic C. difficile Prevalenceb</td><td rowspan=1 colspan=1>027/NAP1/BI Prevalence</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>1072</td><td rowspan=1 colspan=1>18.2% (195/1072)</td><td rowspan=1 colspan=1>5.0% (54/1072)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=2 colspan=1>12212293</td><td rowspan=2 colspan=1>19.2% (235/1221)18.8% (430/2293)</td><td rowspan=2 colspan=1>5.8% (71/1221)5.5% (125/2293)</td></tr><tr><td rowspan=1 colspan=1>Total</td></tr></table>

a Prevalence based on Xpert results. b Toxigenic C. difficile prevalence includes both 027/NAP1/BI strains and non 027/NAP1/BI strains

# N. Instrument Name:

GeneXpert Dx System

GeneXpert $^ \mathrm { \textregistered }$ instrument, computer with proprietary software,   
hand-held barcode scanner, and   
Operator Manual

# O. System Descriptions:

1. Modes of Operation:

The GeneXpert $^ \mathrm { \textregistered }$ Dx System automates and integrates sample preparation, nucleic acid amplification, and detection of the target DNA sequence in patient specimens using realtime Polymerase Chain Reaction (PCR).

Each GeneXpert instrument is similar in that the GeneXpert I (GX I), GeneXpert IV (GX IV) and GeneXpert XVI (GX XVI) all contain the same modules. The difference between the GeneXpert models is that the GX I contains one module, the GX IV can hold up to four modules, and the GX XVI can hold up to sixteen modules, each of which processes one sample at a time. Once the cartridge is

loaded in the instrument, all fluids are completely contained within the disposable, single-use plastic Xpert™ Assay cartridges throughout the sample handling, amplification and detection processes.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.