# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number: K110923 B. Purpose for Submission:

To determine substantial equivalence for the cobas $^ \mathrm { \textregistered }$ CT/NG Test for detection of Chlamydia trachomatis and Neisseria gonorrhoeae DNA from self-collected vaginal swabs, and male urine.

# C. Measurand:

Chlamydia trachomatis DNA

Neisseria gonorrhoeae DNA

# D. Type of Test:

Qualitative in vitro diagnostic assay that utilizes amplification of target DNA by real-time Polymerase Chain Reaction.

# E. Applicant:

Roche Molecular Systems, Inc.

F. Proprietary and Established Names:

Roche cobas $^ \mathrm { \textregistered }$ CT/NG Test and cobas $^ \mathrm { \textregistered }$ 4800 System

# G. Regulatory Information:

1. Regulation section:

21 CFR 866.3120 Chlamydia serological reagents   
21 CFR 866.3390 Neisseria spp. direct serological test reagents   
21 CFR 862.2570 Instrumentation for Clinical Multiplex Systems

2. Classification:

Class II

3. Product code:

MKZ: DNA Probe, Nucleic Acid Amplification, Chlamydia LSL: DNA-Reagents, Neisseria OOI: Real Time Nucleic Acid Amplification System

Panel: Microbiology 083

# H. Intended Use:

1. Intended use(s):

# Assay:

The cobasÒ CT/NG Test is an in vitro nucleic acid amplification test that utilizes the Polymerase Chain Reaction (PCR) and nucleic acid hybridization for the qualitative detection of Chlamydia trachomatis (CT) and/or Neisseria gonorrhoeae (NG) DNA to aid in the diagnosis of chlamydial and gonococcal disease. The test may be used with vaginal swab specimens self-collected in a clinical setting and male urine from both symptomatic and asymptomatic individuals. Specimens to be tested should be collected in cobas® PCR Media.

# Ancillary Collection Kits:

The cobas $^ \mathrm { \textregistered }$ PCR Female Swab Sample Kit is used to collect and transport self-collected vaginal swab specimens in a clinical setting. The cobas $^ \mathrm { \textregistered }$ PCR Media serves as a nucleic acid stabilizing transport and storage medium for gynecological specimens. Use this collection kit only with the cobas $^ \mathrm { \textregistered }$ CT/NG Test. NOTE: This collection kit should not be used for collection of alternative gynecological specimens.

The cobas $^ \mathrm { \textregistered }$ PCR Urine Sample Kit is used to collect and transport male urine specimens. The cobas $^ \mathrm { \textregistered }$ PCR Media serves as a nucleic acid stabilizing transport and storage medium for urine specimens. Use this collection kit only with the cobas $^ \mathrm { \textregistered }$ CT/NG Test. NOTE: This collection kit should not be used for collection of female urine specimens.

2. Indication(s) for use:

# Assay:

The cobasÒ CT/NG Test is an in vitro nucleic acid amplification test that utilizes the Polymerase Chain Reaction (PCR) and nucleic acid hybridization for the qualitative detection of Chlamydia trachomatis (CT) and/or Neisseria gonorrhoeae (NG) DNA to aid in the diagnosis of chlamydial and gonococcal disease. The test may be used with vaginal swab specimens self-collected in a clinical setting and male urine from both symptomatic and asymptomatic individuals. Specimens to be tested should be collected in cobas® PCR Media.

# Ancillary Collection Kits:

The cobas $^ \mathrm { \textregistered }$ PCR Female Swab Sample Kit is used to collect and transport self-collected vaginal swab specimens in a clinical setting. The cobas $^ \mathrm { \textregistered }$ PCR Media serves as a nucleic acid stabilizing transport and storage medium for gynecological specimens. Use this collection kit only with the cobas $^ \mathrm { \textregistered }$ CT/NG Test. NOTE: This collection kit should not be used for collection of alternative gynecological specimens.

The cobas $^ \mathrm { \textregistered }$ PCR Urine Sample Kit is used to collect and transport male urine specimens. The cobas $^ \mathrm { \textregistered }$ PCR Media serves as a nucleic acid stabilizing transport and storage medium for urine specimens. Use this collection kit only with the cobas $^ \mathrm { \textregistered }$ CT/NG Test. NOTE: This collection kit should not be used for collection of female urine specimens.

3. Special conditions for use statement(s): For Prescription Use Only

4. Special instrument requirements: cobas $^ \mathrm { \textregistered }$ 4800 System

# I. Device Description

The cobas $^ \mathrm { \textregistered }$ CT/NG 4800 System is a multi-instrument platform that will perform qualitative in vitro nucleic acid amplification tests from self-collected vaginal specimens and male urine specimens. The system integrates automated total nucleic acid isolation, PCR setup, and realtime PCR. The cobas $^ \mathrm { \textregistered }$ CT/NG 4800 System consists of the cobas $\textbf { x } 4 8 0$ Instrument for specimen preparation, and the cobas $\mathbf { z } 4 8 0$ Analyzer for amplification and detection. The cobas $^ \mathrm { \textregistered }$ 4800 system software integrates the sample preparation with nucleic acid amplification and detection to generate test results.

The Roche Molecular Systems (RMS) cobas $^ \mathrm { \textregistered }$ CT/NG Test consists of six reagent kits:

• cobas $^ \mathrm { \textregistered }$ 4800 System Sample Preparation Kit • cobas $^ \mathrm { \textregistered }$ 4800 CT/NG Amplification/Detection Kit • cobas $^ \mathrm { \textregistered }$ 4800 CT/NG Controls Kit • cobas $^ \mathrm { \textregistered }$ 4800 System Wash Buffer Kit • cobas $^ \mathrm { \textregistered }$ 4800 System Control Diluent Kit • cobas $^ \mathrm { \textregistered }$ 4800 System Liquid Cytology Preparation Kit

Sample Collection Kits to be used with the cobas $^ \mathrm { \textregistered }$ CT/NG Test are: • cobas $^ \mathrm { \textregistered }$ PCR Female Swab Sample Kit

# • cobas $^ \mathrm { \textregistered }$ PCR Urine Sample Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Gen-Probe APTIMA Combo 2 Assay BD ProbeTecTM CT $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay BD ProbeTecTM GC $\mathrm { Q } ^ { \mathrm { x } }$ Amplified DNA Assay

2. Predicate 510(k) number(s):

K060652   
K091724   
K091730

3. Comparison with predicate:

<table><tr><td colspan="4" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Roche        cobasCT/NG Test</td><td colspan="1" rowspan="1">Gen-ProbeAptimaCombo 2 Assay</td><td colspan="1" rowspan="1">BD ProbeTec CTQ" and GC QxAssays</td></tr><tr><td colspan="1" rowspan="1">GeneralIntended Use</td><td colspan="1" rowspan="1">Qualitative in vitrodiagnostic test forthe directqualitativedetection ofChlamydiatrachomatis and/orNeisseriagonorrohoeae inpatient specimens</td><td colspan="1" rowspan="1">same</td><td colspan="1" rowspan="1">same</td></tr><tr><td colspan="1" rowspan="1">Subject Status</td><td colspan="1" rowspan="1">Asymptomatic andsymptomatic</td><td colspan="1" rowspan="1">Asymptomatic andsymptomatic</td><td colspan="1" rowspan="1">Asymptomatic andsymptomatic</td></tr><tr><td colspan="1" rowspan="1">SampleCollectionDevices</td><td colspan="1" rowspan="1">Urine collection kitSwab collection kit</td><td colspan="1" rowspan="1">Urine collection kitSwab collection kit</td><td colspan="1" rowspan="1">Urine collection kitSwab collection kit</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationProcedure</td><td colspan="1" rowspan="1">Semi-automated</td><td colspan="1" rowspan="1">Semi-automated/automated</td><td colspan="1" rowspan="1">Manual/semi-automated</td></tr><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Roche        cobasCT/NG Test</td><td colspan="1" rowspan="1">Gen-Probe AptimaCombo 2 Assay</td><td colspan="1" rowspan="1">BD ProbeTec CTQx and GC QxAssays</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Male urinePatient-collectedvaginal swabs</td><td colspan="1" rowspan="1">Male urineMale urethral swabsFemale urineEndocervical swabsClinician-collectedvaginal swabsPatient-collectedvaginal swabsCervical specimensin PreservCytmedia</td><td colspan="1" rowspan="1">Male urineMale urethral swabsFemale urineEndocervical swabsPatient-collectedvaginal swabsCervical specimensin PreservCyt® andSurePath® media</td></tr><tr><td colspan="1" rowspan="1">CT AnalyteTargets</td><td colspan="1" rowspan="1">CT cryptic plasmidDNACT ompA gene</td><td colspan="1" rowspan="1">CT ribosomal RNA</td><td colspan="1" rowspan="1">CT cryptic plasmidDNA</td></tr><tr><td colspan="1" rowspan="1">NG AnalyteTargets</td><td colspan="1" rowspan="1">NG genomic DNA</td><td colspan="1" rowspan="1">NG ribosomal RNA</td><td colspan="1" rowspan="1">NG genomic DNA</td></tr><tr><td colspan="1" rowspan="1">AmplificationTechnology</td><td colspan="1" rowspan="1">Real-time PCR</td><td colspan="1" rowspan="1">Ribosomal      RNAtranscriptionmediatedamplification (TMA)</td><td colspan="1" rowspan="1">StranddisplacementDNAamplification(SDA)</td></tr><tr><td colspan="1" rowspan="1">DetectionChemistry</td><td colspan="1" rowspan="1">Paired reporter andquencherfluorescencelabeled probes(TaqManTechnology) usingfluorescenceresonance energytransfer (FRET)</td><td colspan="1" rowspan="1">Photon measurementfrom selectivelyhybridizedchemiluminescentprobes reported asRelative Light Units(RLU)</td><td colspan="1" rowspan="1">Fluorescent dyelabeled probes usingfluorescenceresonance energytransfer (FRET)</td></tr><tr><td colspan="1" rowspan="1">Result Analysis</td><td colspan="1" rowspan="1">Based on PCRcycle threshold(Ct) analysis</td><td colspan="1" rowspan="1">Determined by a cut-off based on the totalFLU and kineticcurve type</td><td colspan="1" rowspan="1">Determined byrelating MOTAscore (signal strengthto pre-determinedcutoff values)</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not Applicable

# L. Test Principle:

The cobas® CT/NG Test for Chlamydia trachomatis and Neisseria gonorrhoeae is based on 2 major processes: (1) automated sample preparation to obtain nucleic acids, including CT and NG DNA; (2) simultaneous PCR amplification of target DNA sequences using both CT and NG specific primer pairs and real-time detection of cleaved fluorescent-labeled CT and NG specific oligonucleotide detection probes. An Internal Control, containing CT and NG DNA, is added to all samples prior to automated sample preparation and is amplified and detected simultaneously with each sample to monitor the entire process.

Sample preparation for the cobas® CT/NG Test is automated with the use of the cobas $\mathbf { x } 4 8 0$ instrument. Specimens are lysed in the collection device by the chaotropic agent in the cobas® PCR Media. Released nucleic acids, along with added CT/NG Internal Control DNA, are purified through binding to magnetic glass particles, washed, and finally separated from these particles making them ready for PCR amplification and detection.

# Target Selection

In addition to chromosomal DNA, C. trachomatis contains an approximately 7,500 base pair cryptic plasmid that is common to all serovars of C. trachomatis. The cobas® CT/NG Test uses the CT primers CP102 and CP103 to define a sequence of approximately 206 nucleotides within the cryptic plasmid DNA of $C .$ . trachomatis. In addition, the cobas® CT/NG Test uses the CT primers CTMP101 and CTMP102 to define a sequence of approximately 182 nucleotides within the chromosomal DNA of C. trachomatis.

The N. gonorrhoeae target site is a highly conserved direct repeat region called DR-9. The cobas® CT/NG Test uses the NG primers NG514 and NG519 to define a sequence of approximately 190 nucleotides (DR-9A) from this region. In addition, the cobas® CT/NG Test uses another set of NG primers, NG552 and NG579, to define a second sequence of approximately 215 nucleotides (DR-9B) from this region.

# Target Amplification

Processed samples are added to the amplification mixture in a microwell plate, in which PCR amplification occurs. The reaction mixture is heated to separate the isolated double-stranded DNA and expose the primer target sequences. As the mixture cools, the primers anneal to the target DNA. Z05 DNA polymerase, in the presence of $\mathrm { M n } ^ { 2 + }$ and excess dNTPs, extends the annealed primers along the target templates to produce double-stranded DNA. This completes the first cycle of PCR, yielding a double-stranded DNA copy of the target regions of the CT and/or NG DNA and the CT/NG Internal Control DNA. Repetition of this process results in the amplification of DNA between the primer target sequences, producing a double-stranded DNA molecule termed an amplicon. The cobas $\mathbf { z } 4 8 0$ analyzer automatically repeats this process for a designated number of cycles, with each cycle intended to double the amount of amplicon DNA. The required number of cycles is preprogrammed into the cobas® 4800 Software. Amplification occurs only in the specific CT and/or NG targets between their respective primers; the entire CT cryptic plasmid or CT and/or NG genomes are not amplified.

The CT/NG Internal Control is a combination of two non-infectious recombinant plasmid DNAs, each with primer binding regions identical to those of either the $C .$ . trachomatis or the N. gonorrhoeae genomic target sequences. Both recombinant plasmid DNAs have an identical randomized internal target sequence, and a unique probe binding region that differentiates the CT/NG Internal Control from target amplicon. These features were selected to ensure independent detection of both the CT/NG Internal Control and the $C .$ trachomatis and $N .$ gonorrhoeae target DNAs. The CT/NG Internal Control Reagent is included in the cobas® CT/NG Test and is introduced into each sample on the cobas $\mathbf { x } 4 8 0$ instrument during sample processing.

# Selective Amplification

Selective amplification of target nucleic acid from the specimen is achieved in the cobas® CT/NG Test by the use of AmpErase (uracil-N-glycosylase) enzyme and deoxyuridine triphosphate (dUTP). The AmpErase enzyme recognizes and catalyzes the destruction of DNA strands containing deoxyuridine, but not DNA containing deoxythymidine. Deoxyuridine is not present in naturally occurring DNA, but is always present in amplicon due to the use of deoxyuridine triphosphate in place of thymidine triphosphate as one of the dNTPs in the Master Mix reagent; therefore, only amplicon contain deoxyuridine. Deoxyuridine renders contaminating amplicon susceptible to destruction by AmpErase enzyme prior to amplification of the target DNA. AmpErase enzyme, which is included in the Master Mix reagent, catalyzes the cleavage of deoxyuridine-containing DNA at the deoxyuridine residues by opening the deoxyribose chain at the C1-position. When heated in the first thermal cycling step at the alkaline $\mathsf { p H }$ of Master Mix, the amplicon DNA chain breaks at the position of the deoxyuridine, thereby rendering the DNA non-amplifiable. AmpErase enzyme is inactive at temperatures above $5 5 \mathrm { ^ \circ C }$ , i.e., throughout the thermal cycling steps, and therefore does not destroy target amplicon. The cobas® CT/NG Test has been demonstrated to inactivate at least $1 0 ^ { 3 }$ copies of deoxyuridine-containing CT/NG amplicon per PCR.

# Detection of PCR Products

The cobas® CT/NG Test utilizes real-time PCR technology. The use of fluorescent probes provides for real-time detection of PCR product accumulation by monitoring the emission intensity of fluorescent dyes released during the amplification process. The probes include CT cryptic plasmid, CT ompA, NG DR-9A, NG DR-9B and CT/NG Internal Control-specific oligonucleotides, all labeled with a reporter dye and a quencher. When the fluorescent dyelabeled probes are intact, the reporter fluorescence is suppressed by the proximity of the quencher due to Förster-type energy transfer effects. During PCR, the probes hybridize to their respective target sequence and are cleaved by the $5 ^ { \circ }$ to $3 ^ { \circ }$ nuclease activity of the thermostable Z05 DNA polymerase. Once the reporter and quencher are separated, quenching no longer occurs, and the fluorescent emission of the reporter dyes increases. The amplification of CT targets, NG targets and the CT/NG Internal Control are measured independently and at different wavelengths. This process is repeated for a designated number of cycles, each cycle increasing the emission intensity of the individual reporter dyes.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision: In-house Precision was examined using a panel composed of CT and NG cultures diluted into cobas® PCR Media and cobas® PCR Media mixed with negative urine. The precision panel was designed to include members with either CT or NG at approximately the LOD for the panel matrix, members with both CT and NG at approximately the LOD and $2 . 5 \mathrm { \bf x }$ LOD for the panel matrix and a negative level. Testing was done with three unique lots of cobas® CT/NG Test reagents and three instruments for a total of 24 runs. A description of the precision panels and the study performance hit rate is shown in the table below. All positive panel levels yielded the anticipated hit rates. All negative panel levels tested negative throughout the study. An analysis of the variance of the Ct values from valid tests performed on positive panel members yielded overall CV $( \% )$ ranges from $1 . 1 \%$ to $1 . 8 \%$ for CT and from $1 . 2 \%$ to $1 . 9 \%$ for NG.

In-House Precision Study Hit Rate Analysis   

<table><tr><td rowspan=3 colspan=1>PanelNumber</td><td rowspan=3 colspan=1>Panel Matrix</td><td rowspan=2 colspan=2>Target Conc.</td><td rowspan=3 colspan=1>N Tested</td><td rowspan=3 colspan=1>N PosCT</td><td rowspan=3 colspan=1>N PosNG</td><td rowspan=3 colspan=1>HitRate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=2 colspan=1>Lower</td><td rowspan=2 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>NG</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>cobas® PCR Media</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>cobasPCR Media</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>cobasPCR Media</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>cobasPCR Media</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>2.5 XLOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>cobas® PCR Media</td><td rowspan=1 colspan=1>2.5 XLOD</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>cobasPCR Media +Urine</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>cobasPCR Media +Urine</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>cobasPCR Media +Urine</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>cobasPCR Media +Urine</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>2.5 XLOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>cobasPCR Media +Urine</td><td rowspan=1 colspan=1>2.5 XLOD</td><td rowspan=1 colspan=1>1 X LOD</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.5</td><td rowspan=1 colspan=1>100.0</td></tr></table>

b. Reproducibility: A Reproducibility Study was performed across lots, testing sites, operators, runs, and days for the cobasÒ 4800 CT/NG Test using 2 panels prepared from swabs and urine collected in cobas PCR Media. Testing was performed at two external sites as well as in-house at Roche Molecular Systems. A run for cobas® PCR Media (urine or swab) included 3 replicates of each of 5 panel members and 1 positive and 1 negative control (17 total tests). If cobas® PCR Media panels were combined in a run, only 1 positive and 1 negative control were included (32 total tests). The 2 operators at each site performed 2 runs per day, for a total of 3 days of testing per operator per panel type (6 days of testing total for each panel type and reagent lot). Testing was performed with 2 reagent lots (6 days of testing per lot). Overall, 74 runs were performed, and 72 valid runs were obtained for urine and swab panel types. The 2 invalid runs were due to instrument errors. A total of 1,080 tests were performed on the 5 panel members for each panel type. There was 1 invalid test result in the urine panel type, and 2 invalid test results in the swab panel type. These invalid tests were due to instrument errors. All valid test results were included in the analyses of the percent agreement for CT and NG for each panel type separately. There were no false positive results for either analyte (CT and NG) for both panel types for negative panel members, thus giving negative percent agreement (NPA) of $100 \%$ for each analyte.

C. trachomatis Reproducibility results: For both matrix types, percent agreement for CT positive and negative samples was $100 \%$ . Analysis of variance components of the Ct values performed on positive panel members yielded overall CV $( \% )$ ranges from $1 . 1 \%$ to $1 . 5 \%$ for the urine panel type; $1 . 6 \%$ to $1 . 8 \%$ for the swab panel type.

C. trachomatis: Percent Agreement by Panel Member for Lot, Site/Instrument, and Day - PCR Media/Urine   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>CtSD</td><td rowspan=2 colspan=1>CtCV%</td><td rowspan=1 colspan=9>Percent Agreement *</td></tr><tr><td rowspan=1 colspan=3>Lot</td><td rowspan=1 colspan=3>Site/Instrument</td><td rowspan=1 colspan=3>Day</td></tr><tr><td rowspan=3 colspan=1>Negative CT,Negative NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LOD CT,Negative NG</td><td rowspan=3 colspan=1>0.54</td><td rowspan=3 colspan=1>1.5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>Negative CT,1 X LOD NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LOD CT,2.5 X LOD NG</td><td rowspan=3 colspan=1>0.48</td><td rowspan=3 colspan=1>1.3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>2.5 X LOD CT,1 X LOD NG</td><td rowspan=3 colspan=1>0.40</td><td rowspan=3 colspan=1>1.1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr></table>

\* For Negative samples, Percent Agreement $=$ (number of negative results/total valid results); For Positive samples, Percent Agreement $=$ (number of positive results/total valid results)

C. trachomatis: Percent Agreement by Panel Member for Lot, Site/Instrument, and Day - PCR Media/Swab   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>CtSD</td><td rowspan=2 colspan=1>CtCV%</td><td rowspan=1 colspan=9>Percent Agreement *</td></tr><tr><td rowspan=1 colspan=3>Lot</td><td rowspan=1 colspan=3>Site/Instrument</td><td rowspan=1 colspan=3>Day</td></tr><tr><td rowspan=3 colspan=1>Negative CT,Negative NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LODCT,Negative NG</td><td rowspan=3 colspan=1>0.61</td><td rowspan=3 colspan=1>1.6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=3 colspan=1>Negative CT,1 X LOD NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LODCT,2.5 X LODNG</td><td rowspan=3 colspan=1>0.66</td><td rowspan=3 colspan=1>1.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>2.5 X LODCT,1 X LOD NG</td><td rowspan=3 colspan=1>0.59</td><td rowspan=3 colspan=1>1.6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr></table>

\* For Negative samples, Percent Agreement $=$ (number of negative results/total valid results); For Positive samples, Percent Agreement $=$ (number of positive results/total valid results)

C. trachomatis : Overall Mean, Standard Deviations, and Coefficients of Variation $( \% )$ for Cycle Threshold, Estimated   

<table><tr><td colspan="3" rowspan="1"></td><td colspan="14" rowspan="1">Standard Deviation [SD] and Percent Coefficient of Variation [CV(%)]</td></tr><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="2" rowspan="1"></td><td colspan="2" rowspan="1">Within-Run</td><td colspan="2" rowspan="1">Between-Run</td><td colspan="2" rowspan="1">Between-Day</td><td colspan="2" rowspan="1">Between-Operator</td><td colspan="2" rowspan="1">Between-Lot</td><td colspan="2" rowspan="1">Between-Site/Instrument</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">$\frac}$</td><td colspan="1" rowspan="1">MeanCt</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">CV%</td></tr><tr><td colspan="17" rowspan="1">PCR Media/Urine</td></tr><tr><td colspan="1" rowspan="1">1x LODCT,NegativeNG</td><td colspan="1" rowspan="1">216216</td><td colspan="1" rowspan="1">36.14</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">1.4%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">0.m</td><td colspan="1" rowspan="1">0.0%%</td><td colspan="1" rowspan="1">0.15</td><td colspan="1" rowspan="1">0.4%6</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.%%</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.55%</td></tr><tr><td colspan="1" rowspan="1">1x LODCT, 2.5xLODNG</td><td colspan="1" rowspan="1">216216</td><td colspan="1" rowspan="1">36.060.43</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">1.22%</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.0%.</td><td colspan="1" rowspan="1">0. 00</td><td colspan="1" rowspan="1">0.5%</td><td colspan="1" rowspan="1">0.0</td><td colspan="1" rowspan="1">0.00%%</td><td colspan="1" rowspan="1">0.11</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0.46</td><td colspan="1" rowspan="1">1.%</td></tr><tr><td colspan="1" rowspan="1">2.5xLODCT, 1xLODNG</td><td colspan="1" rowspan="1">216216</td><td colspan="1" rowspan="1">35.01</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">1.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.x</td><td colspan="1" rowspan="1">.00</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">.000</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">.00</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0.100</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="17" rowspan="1">PCR Media/Swab</td></tr><tr><td colspan="1" rowspan="1">1x LODCT,NegativeNG</td><td colspan="1" rowspan="1">215215</td><td colspan="1" rowspan="1">37.10</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.09</td><td colspan="1" rowspan="1">0.1%%</td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.3%%</td><td colspan="1" rowspan="1">0.000</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.61</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">1x LoDCT, 2.5xOD NG</td><td colspan="1" rowspan="1">216216</td><td colspan="1" rowspan="1">37.26</td><td colspan="1" rowspan="1">0.55</td><td colspan="1" rowspan="1">1.5%%</td><td colspan="1" rowspan="1">002</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">0.6%%</td><td colspan="1" rowspan="1">0.099</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0.00</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">0.66</td><td colspan="1" rowspan="1">1.8%</td></tr><tr><td colspan="1" rowspan="1">2.5xLODCT, 1xLODNG</td><td colspan="1" rowspan="1">216216</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.10</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0.2%0.59</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="17" rowspan="1">n is the number of positive tests, which contribute Ct values to the analysis. N is the total number ofvalid tests for the panel member.</td></tr></table>

$\underline { { N } } .$ gonorrhoeae Reproducibility results: The lowest overall PPA for positive panel members was $9 9 . 5 2 \%$ for the “Negative CT, $_ { 1 \mathrm { ~ x ~ } }$ LOD NG” panel member for PCR Media/Urine panel type. Analysis of variance components of the Ct values from valid tests performed on positive panel members yielded overall CV $( \% )$ ranges from $1 . 2 \%$ to $1 . 5 \%$ for the urine panel type and $1 . 4 \%$ to $1 . 9 \%$ for the swab panel type.

N. gonorrhoeae: Percent Agreement by Panel Member for Lot, Site/Instrument, and Day - PCR Media/Urine   

<table><tr><td colspan="1" rowspan="2">PanelMember</td><td colspan="1" rowspan="2">CtSD</td><td colspan="1" rowspan="2">CtCV%</td><td colspan="9" rowspan="1">Percent Agreement 1</td></tr><tr><td colspan="3" rowspan="1">Lot</td><td colspan="3" rowspan="1">Site/Instrument</td><td colspan="3" rowspan="1">Day</td></tr><tr><td colspan="1" rowspan="3">Negative CT,Negative NG</td><td colspan="1" rowspan="3">n/a</td><td colspan="1" rowspan="3">n/a</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">71/71</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">107/107</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">71/71</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="3">1 X LOD CT,Negative NG</td><td colspan="1" rowspan="3">n/a</td><td colspan="1" rowspan="3">n/a</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="3">Negative CT,1 X LOD NG</td><td colspan="1" rowspan="3">0.53</td><td colspan="1" rowspan="3">1.5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">99.1</td><td colspan="1" rowspan="1">107/108</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">71/72</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">98.6</td><td colspan="1" rowspan="1">71/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="3">1 X LOD CT,2.5 X LOD NG</td><td colspan="1" rowspan="3">0.41</td><td colspan="1" rowspan="3">1.2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="3">2.5 X LOD CT,1 X LOD NG</td><td colspan="1" rowspan="3">0.54</td><td colspan="1" rowspan="3">1.5</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">108/108</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">100.0</td><td colspan="1" rowspan="1">72/72</td></tr></table>

¹ For Negative samples, Percent Agreement $=$ (number of negative results/total valid results); For Positive samples, Percent Agreement $=$ (number of positive results/total valid results)

N. gonorrhoeae: Percent Agreement by Panel Member for Lot, Site/Instrument, and Day - PCR Media/Swab   

<table><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=2 colspan=1>CtSD</td><td rowspan=2 colspan=1>CtCV%</td><td rowspan=1 colspan=9>Percent Agreement 1</td></tr><tr><td rowspan=1 colspan=3>Lot</td><td rowspan=1 colspan=3>Site/Instrument</td><td rowspan=1 colspan=3>Day</td></tr><tr><td rowspan=3 colspan=1>Negative CT,Negative NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LOD CT,Negative NG</td><td rowspan=3 colspan=1>n/a</td><td rowspan=3 colspan=1>n/a</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=3 colspan=1>Negative CT,1 X LOD NG</td><td rowspan=3 colspan=1>0.68</td><td rowspan=3 colspan=1>1.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>107/107</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>71/71</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>1 X LOD CT,2.5 X LOD NG</td><td rowspan=3 colspan=1>0.49</td><td rowspan=3 colspan=1>1.4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=3 colspan=1>2.5 X LOD CT,1 X LOD NG</td><td rowspan=3 colspan=1>0.71</td><td rowspan=3 colspan=1>1.9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>72/72</td></tr></table>

¹ For Negative samples, Percent Agreement $=$ (number of negative results/total valid results); For Positive samples, Percent Agreement $=$ (number of positive results/total valid results)

N. gonorrhoeae: Overall Mean, Standard Deviations, and Coefficients of Variation $( \% )$ for   

<table><tr><td rowspan=1 colspan=17>Cycle Threshold, Estimated</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=14>Standard Deviation [SD] and Percent Coefficient of Variation [CV(%)]</td></tr><tr><td rowspan=2 colspan=1>PanelMember</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Site/Instrument</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>$\rac }$</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV%</td></tr><tr><td rowspan=1 colspan=15>PCR Media/Urine</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>NegativeT, 1xLODNG</td><td rowspan=1 colspan=1>215216</td><td rowspan=1 colspan=1>36.53</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.07</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1>0.%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0..1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.33%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>1x LODCT, 2.5xLODNG</td><td rowspan=1 colspan=1>216216</td><td rowspan=1 colspan=1>35.42</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0. 1%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>000</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>2.5xLODCT, 1xLODNG</td><td rowspan=1 colspan=1>216216</td><td rowspan=1 colspan=1>36.58</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1>p.0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>004</td><td rowspan=1 colspan=1>05</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=17>PCR Media/Swab</td></tr><tr><td rowspan=1 colspan=1>NegativeCT, 1xLODNG</td><td rowspan=1 colspan=1>215215</td><td rowspan=1 colspan=1>37.26</td><td rowspan=1 colspan=1>0.64</td><td rowspan=1 colspan=1>1.7%0.21</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.m</td><td rowspan=1 colspan=1>0.%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.%.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.</td></tr><tr><td rowspan=1 colspan=1>1x LODCT, 2.5xLODNG</td><td rowspan=1 colspan=1>216216</td><td rowspan=1 colspan=1>35.94</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.09</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.79%</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.29.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>a</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0%0.49</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>2.5xLODCT, 1xLODNG</td><td rowspan=1 colspan=1>216216</td><td rowspan=1 colspan=1>37.09</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=17>n is the number of positive tests, which contribute Ct values to the analysis.N is the total number ofvalid tests for the        panel member.</td></tr></table>

Precision of High-Negative CT and NG Samples:

A second study was conducted internally at Roche Molecular Systems to characterize the precision of test results at target levels below the analytical Limit of Detection of the assay. High-negative panels were prepared by spiking CT and NG cultures into urine and vaginal specimen stabilized in cobas $^ \mathrm { \textregistered }$ PCR Media to levels producing from 20 to $80 \%$ negative results. Negative panel members were also prepared for each matrix. For each sample matrix, panels were tested over the course of 12 days by two operators using two lots of reagents and three cobas $^ \mathrm { \textregistered }$ 4800 Systems. Five replicates of each panel member were tested in each run, generating up to 120 test results for each panel level. The high-negative panel testing yielded the anticipated hit rates.

In-House Precision Study Hit Rate Analysis for High –Negative Levels   

<table><tr><td rowspan=1 colspan=9>Urine Stabilized in cobas® PCR Media</td></tr><tr><td rowspan=2 colspan=1>PanelLevel</td><td rowspan=1 colspan=4>CT</td><td rowspan=1 colspan=4>NG</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>HitRate</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>HitRate</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-2.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-2.5%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>36.7-55.2%</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>45.8</td><td rowspan=1 colspan=1>36.7-55.2%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=8>Vaginal Swab Collected in cobas®]PCR Media</td></tr><tr><td rowspan=2 colspan=1>PanelLevel</td><td rowspan=1 colspan=4>CT</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>NG</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>HitRate</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>HitRate</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-2.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0-2.5%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>47.5</td><td rowspan=1 colspan=1>38.3-56.8%</td><td rowspan=1 colspan=1>81</td><td rowspan=1 colspan=1>120</td><td rowspan=1 colspan=1>67.5</td><td rowspan=1 colspan=1>58.3-75.8%</td></tr></table>

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

One set of cobas® CT/NG Test Positive and Negative Controls are included in each run. For any run, valid results must be obtained for both the Positive and Negative Control for the cobas® 4800 Software to display the reportable cobas® CT/NG Test results from that run.

The CT/NG $( + )$ Control contains non-infectious DNA plasmids of both $C .$ . trachomatis and N. gonorrhoeae sequences and is used as a run control to monitor the target capture, amplification, and detection steps of the test. The CT/NG (-) Control contains buffer with no nucleic acid.

The CT/NG Internal Control is a combination of two non-infectious recombinant plasmid DNAs, each with primer binding regions identical to those of either the $C .$ . trachomatis or the N. gonorrhoeae genomic target sequences. The Internal Control is added to all specimens and the Positive and Negative Controls during sample preparation on the cobas $\mathbf { x } 4 8 0$ instrument. The Internal Control confirms the reliability of test specimens by monitoring for the presence of PCR inhibitors. The Internal Control is also required for validation of the run controls.

# d. Detection limit:

The analytical sensitivity (Limit of Detection or LOD) for the cobas® CT/NG Test was determined by analyzing dilutions of quantified Chlamydia trachomatis and Neisseria gonorrhoeae cultures. CT and NG cultures were diluted into a matrix of negative vaginal swab specimen in cobas® PCR Media and into a matrix of negative urine specimen plus cobas® PCR Media to determine the LOD for vaginal swab and urine specimens. All levels were analyzed using the full cobas® CT/NG Test workflow across 3 unique lots of cobas® CT/NG Test reagents. LOD for this test is defined as the target concentration which can be detected as positive i $12 \%$ of the replicates tested. Since LOD evaluation is done with samples stabilized in cobas® PCR Media, the LOD for neat urine will be twice the level reported in the table below. The LOD for the CT serovar D culture and NG strain 19424 in vaginal swab specimens stabilized in cobas® PCR Media and urine specimens diluted into cobas® PCR Media are shown in the following table.

cobas® CT/NG Test Limit of Detection   

<table><tr><td rowspan=2 colspan=1>Specimen Types</td><td rowspan=1 colspan=3>C. trachomatis</td><td rowspan=1 colspan=3>N. gonorrhoeae</td></tr><tr><td rowspan=1 colspan=1>LevelsTested</td><td rowspan=1 colspan=1>Replicates/Level</td><td rowspan=1 colspan=1>LOD(IFU/mL)</td><td rowspan=1 colspan=1>LevelsTested</td><td rowspan=1 colspan=1>Replicates/Level</td><td rowspan=1 colspan=1>LOD(CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>Vaginal Swabs</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>192**</td><td rowspan=1 colspan=1>10.00</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>192**</td><td rowspan=1 colspan=1>100.00</td></tr><tr><td rowspan=1 colspan=1>Urine</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>192*</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>192*</td><td rowspan=1 colspan=1>2.25</td></tr></table>

\*Testing included one negative level with 167-168 replicates \*\*Testing included one negative level with 82-84 replicates

# e. Inclusivity

Inclusivity testing with the cobas® CT/NG Test was performed for 14 additional CT serovars, the Swedish new variant strain (nvCT) and an additional 44 independently isolated strains of NG. Testing was done to demonstrate that these targets can be detected around the LOD levels determined during analytical sensitivity testing for the CT serovar D culture and NG strain 19424. At least 49 replicates were tested for each panel level using one lot of cobas® CT/NG Test reagents. Results are shown in the tables below. In cobas® PCR Media plus urine, positive hit rates for the $1 4 \mathrm { C T }$ serovars plus the nvCT variant (Table 18) were $100 \%$ for concentrations ranging from 0.13 to $0 . 7 5 \mathrm { I F U / m l }$ . All CT serovars and the nvCT variant were tested at $1 0 \ : \mathrm { I F U / m L }$ in stabilized negative vaginal specimen, yielding $100 \%$ positive hit rates. All $4 4 \mathrm { N G }$ strains were tested at 3.75 Colony Forming Units $\mathrm { ( C F U ) / m L }$ in cobas® PCR Media plus urine and at $1 0 0 \mathrm { C F U / m L }$ in stabilized negative vaginal specimen. Positive hit rates ranged from 96 to $100 \%$ . In the table below, all NG strains with identical results are presented as a group, shown in the columns labeled “Numbers of NG Strains”.

Summary of CT Serovars/Variant Inclusivity Verification Results   

<table><tr><td rowspan=3 colspan=1>Serovar Type orVariant</td><td rowspan=1 colspan=4>Reactivity Results for C. trachomatis</td></tr><tr><td rowspan=1 colspan=2>Vaginal Swabs*</td><td rowspan=1 colspan=2>Urine</td></tr><tr><td rowspan=1 colspan=1>IFU/mL</td><td rowspan=1 colspan=1>% Pos</td><td rowspan=1 colspan=1>IFU/mL</td><td rowspan=1 colspan=1>% Pos</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Ba</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>C</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>H</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>J</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>K</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>LVType 1</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>LVType 2</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>LVType 3</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>nvCT</td><td rowspan=1 colspan=1>10.0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>100%</td></tr></table>

Summary of NG Strains Inclusivity Verification Results   

<table><tr><td rowspan=2 colspan=1>Numbers of NGStrains</td><td rowspan=1 colspan=2>Inclusivity Results forUrine</td></tr><tr><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>%Hit Rate</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.75</td><td rowspan=1 colspan=1>96%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.75</td><td rowspan=1 colspan=1>98%</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>3.75</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Total = 44</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=3></td></tr><tr><td rowspan=2 colspan=1>Numbers of NGStrains</td><td rowspan=1 colspan=2>Inclusivity Results forVaginal Swabs</td></tr><tr><td rowspan=1 colspan=1>CFU/mL</td><td rowspan=1 colspan=1>% Hit Rate</td></tr><tr><td rowspan=1 colspan=1>Total = 44</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>100%</td></tr></table>

# f. Analytical specificity:

A panel of 184 bacteria, fungi and viruses, including those commonly found in the female urogenital tract, as well as representatives of $N .$ . cineria, N. flava N. lactamica, N. perflava and N. subflava and other phylogenetically unrelated organisms, were tested with the cobas® CT/NG Test to assess analytical specificity. The organisms listed in the first table below were spiked at concentrations of $\boldsymbol { \mathrm { ~ i ~ x ~ } } \boldsymbol { 1 } 0 ^ { 6 }$ Units\*/mL or higher .into cobas® PCR Media, pooled negative urine in cobas® PCR Media and pooled negative vaginal matrix in cobas® PCR Media. Organisms listed in the second table below were tested at varying concentrations below $1 \mathrm { ~ x ~ } 1 0 ^ { 6 } \mathrm { U n i t s ^ { * } / m l }$ . Testing was performed with each potential interfering organism

alone as well as with each organism mixed with CT and NG cultures at 3 times the limit of detection. Results indicated that none of these organisms interfered with detection of CT and NG or produced false positive results in the CT/NG negative matrices.

\*All bacteria were quantified as Colony Forming Units (CFU) except Chlamydophila pneumoniae as Inclusion Forming Units (IFU). Treponema pallidum and HBV were quantified as DNA copies. Adenovirus was quantified as Plaque Forming Units (PFU). CMV, EBV, HSV1 and HSV-2 were quantified as Viral Particles (VP). HCV and HIV-1 were quantified in International Units (IU). Trichomonas vaginalis, HPV16 and HPV18 were quantified as cells/mL.

Microorganisms Tested for Analytical Specificity   

<table><tr><td colspan="1" rowspan="1">Achromobacter xerosis</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">Neisseria sicca</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus</td><td colspan="1" rowspan="1">Hepatitis B virus (HBV)</td><td colspan="1" rowspan="1">Neisseria subflava</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi</td><td colspan="1" rowspan="1">Hepatitis C virus (HCV)</td><td colspan="1" rowspan="1">Neisseria subflava 6458</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter sp. genospecies 3</td><td colspan="1" rowspan="1">Human immunodeficiency virus</td><td colspan="1" rowspan="1">Neisseria subflava 6617</td></tr><tr><td colspan="1" rowspan="1">Actinomyces israelii</td><td colspan="1" rowspan="1">Human papillomavirus type 16(CaSki cells)</td><td colspan="1" rowspan="1">Neisseria subflava 6618</td></tr><tr><td colspan="1" rowspan="1">Actinomyces pyogenes</td><td colspan="1" rowspan="1">Human papillomavirus type 18(HeLa cells)</td><td colspan="1" rowspan="1">Neisseria subflava 7441</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Herpes Simplex Virus (HSV-1)</td><td colspan="1" rowspan="1">Neisseria subflava 7452</td></tr><tr><td colspan="1" rowspan="1">Aerococcus viridans</td><td colspan="1" rowspan="1">Herpes Simplex Virus (HSV-2)</td><td colspan="1" rowspan="1">Neisseria weaverii</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">Kingella dentrificans</td><td colspan="1" rowspan="1">Pantoea agglomerans</td></tr><tr><td colspan="1" rowspan="1">Alcaligenes faecalis</td><td colspan="1" rowspan="1">Kingella kingae</td><td colspan="1" rowspan="1">Paracoccus denitrificans</td></tr><tr><td colspan="1" rowspan="1">Bacillus subtilis</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">Pasteurella maltocida</td></tr><tr><td colspan="1" rowspan="1">Bacillus thuringiensis</td><td colspan="1" rowspan="1">Klebsiella pneumoniae ss ozaenae</td><td colspan="1" rowspan="1">Pediococcus acidilactica</td></tr><tr><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">Lactobacillus acidophillus</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">Lactobacillus brevis</td><td colspan="1" rowspan="1">Peptostreptococcus asacharolyticus</td></tr><tr><td colspan="1" rowspan="1">Bacteroides ureolyticus</td><td colspan="1" rowspan="1">Lactobacillus crispatus</td><td colspan="1" rowspan="1">Peptostreptococcus magnus</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">Lactobacillus delbrueckii subsp.lactis</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium breve</td><td colspan="1" rowspan="1">Lactobacillus jensenii</td><td colspan="1" rowspan="1">Prevotella bivia</td></tr><tr><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">Lactobacillus lactis</td><td colspan="1" rowspan="1">Prevotella corporis</td></tr><tr><td colspan="1" rowspan="1">Branhamella catarrhalis</td><td colspan="1" rowspan="1">Lactobacillus oris</td><td colspan="1" rowspan="1">Prevotella intermedia</td></tr><tr><td colspan="1" rowspan="1">Brevibacterium linens</td><td colspan="1" rowspan="1">Lactobacillus parabuchnerri</td><td colspan="1" rowspan="1">Propionibacterium acnes</td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">Lactobacillus vaginalis</td><td colspan="1" rowspan="1">Proteus mirabilis</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">Lactococcus lactis cremoris</td><td colspan="1" rowspan="1">Proteus vulgaris</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Legionella bozemnii</td><td colspan="1" rowspan="1">Providencia stuartii</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondi</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">Micrococcus luteus</td><td colspan="1" rowspan="1">Pseudomonas putida</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">Mobiluncus curtisii subsp. curtisii</td><td colspan="1" rowspan="1">Rahnella aquatilis</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">Mobiluncus curtisii subsp.holmesii</td><td colspan="1" rowspan="1">Rhizobium radiobacter</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">Mobiluncus mulieris</td><td colspan="1" rowspan="1">Rhodospirillum rubrum</td></tr><tr><td colspan="1" rowspan="1">Chromobacter violaceum</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ruminococcus productus</td></tr><tr><td colspan="1" rowspan="1">Chryseobacteriummeningosepticum</td><td colspan="1" rowspan="1">Moraxella lacunata</td><td colspan="1" rowspan="1">Saccharomyces cerevisiae</td></tr><tr><td colspan="1" rowspan="1">Citrobacter braakii</td><td colspan="1" rowspan="1">Moraxella osloensis</td><td colspan="1" rowspan="1">Salmonella Choleraesuis</td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">Morganella morganii</td><td colspan="1" rowspan="1">Salmonella Minnesota</td></tr><tr><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">Salmonella typhimurium</td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">Mycobacterium gordonae</td><td colspan="1" rowspan="1">Serratia denitrificans</td></tr><tr><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">Mycobacterium smegmatis</td><td colspan="1" rowspan="1">Serratia marcescens</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">Staphylococcus aureus</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium renale</td><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium xerosis</td><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">Staphylococcus saprophyticus</td></tr><tr><td colspan="1" rowspan="1">Cryptococcus neoformans</td><td colspan="1" rowspan="1">Neisseria cinerea 832</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">Neisseria cinerea 3306</td><td colspan="1" rowspan="1">Streptococcus anginosus</td></tr><tr><td colspan="1" rowspan="1">Deinococcus radiodurans</td><td colspan="1" rowspan="1">Neisseria cinerea 3307</td><td colspan="1" rowspan="1">Streptococcus bovis</td></tr><tr><td colspan="1" rowspan="1">Deinococcus radiopugnans</td><td colspan="1" rowspan="1">Neisseria cinerea 3308</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td></tr><tr><td colspan="1" rowspan="1">Derxia gummosa</td><td colspan="1" rowspan="1">Neisseria cinerea 6317</td><td colspan="1" rowspan="1">Streptococcus equinis</td></tr><tr><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">Neisseria dentrificans</td><td colspan="1" rowspan="1">Streptococcus mitis</td></tr><tr><td colspan="1" rowspan="1">Eikenella corrodens</td><td colspan="1" rowspan="1">Neisseria elongata subsp.niroreducans</td><td colspan="1" rowspan="1">Streptococcus mutans</td></tr><tr><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">Neisseria flava</td><td colspan="1" rowspan="1">Streptococcus pneumoniae</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">Neisseria flavescens</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td></tr><tr><td colspan="1" rowspan="1">Enterococcus avium</td><td colspan="1" rowspan="1">Neisseria kochi</td><td colspan="1" rowspan="1">Streptococcus salivarius</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">Neisseria lactamica</td><td colspan="1" rowspan="1">Streptococcus sanguis</td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecium</td><td colspan="1" rowspan="1">Neisseria meningitidis 135</td><td colspan="1" rowspan="1">Streptomyces griseinus</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">Neisseria meningitidis SerogroupA</td><td colspan="1" rowspan="1">Treponema pallidum</td></tr><tr><td colspan="1" rowspan="1">Erwinia herbicola</td><td colspan="1" rowspan="1">Neisseria meningitidis SerogroupB</td><td colspan="1" rowspan="1">Trichomonas vaginalis</td></tr><tr><td colspan="1" rowspan="1">Erysipelothrix rhusiopathiae</td><td colspan="1" rowspan="1">Neisseria meningitidis SerogroupC</td><td colspan="1" rowspan="1">Ureaplasma urealyticum</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">Neisseria meningitidis SerogroupD</td><td colspan="1" rowspan="1">Veillonela parvula</td></tr><tr><td colspan="1" rowspan="1">Ewingella americana</td><td colspan="1" rowspan="1">Neisseria meningitidesSerogroupY</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td></tr><tr><td colspan="1" rowspan="1">Flavobacteriummeningosepticum</td><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">Weissella paramesenteroides</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum</td><td colspan="1" rowspan="1">Neisseria perflava 837</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">Neisseria perflava 911</td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">Gemella haemolysans</td><td colspan="1" rowspan="1">Neisseria perflava 6339</td></tr><tr><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">Neisseria perflava 6340</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">Neisseria perflava 6341</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">Neisseria polysaccharea</td></tr></table>

List of Microorganisms Tested at less than $\bf { 1  { X } 1 0 ^ { 6 } }$ copies/mL for Analytical Specificity   

<table><tr><td rowspan=2 colspan=1>Microorganism Tested</td><td rowspan=1 colspan=2>Concentration Tested in ListedMatrix*</td></tr><tr><td rowspan=1 colspan=1>Negative VaginalSpecimen</td><td rowspan=1 colspan=1>Negative UrineSpecimen</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>8x10 PFU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1.1x104 IFU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>4.5 x 104 CFU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hepatitis C virus (HCV)</td><td rowspan=1 colspan=1>5.6 x 104 IU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human papillomavirus (HPV) type 16(SiHa cells)</td><td rowspan=1 colspan=1>1x104 cells /mL</td><td rowspan=1 colspan=1>5x104 cells /mL</td></tr><tr><td rowspan=1 colspan=1>Human papillomavirus (HPV) type 18(HeLa cells)</td><td rowspan=1 colspan=1>1x104 cells /mL</td><td rowspan=1 colspan=1>1x104 cells /mL</td></tr><tr><td rowspan=1 colspan=1>Neisseria cinerea 3307</td><td rowspan=1 colspan=1>4x105 CFU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Prevotella bivia</td><td rowspan=1 colspan=1>9x104 CFU /mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Prevotella corporis</td><td rowspan=1 colspan=1>1.4x10 CFU /mL</td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr><tr><td rowspan=1 colspan=1>Treponema pallidum</td><td rowspan=1 colspan=1>1x105 copies/mL</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Trichomonas vaginalis</td><td rowspan=1 colspan=1>6.5x103 cells/mL</td><td rowspan=1 colspan=1>cYanmaGYellowb</td></tr></table>

\*Gray cells indicate concentration tested was $\overline { { \geq 1 \mathrm { ~ x ~ } 1 0 ^ { 6 } } }$ copies/mL in that matrix

# g. Interference Study

Interference testing was performed using cobas® PCR Media plus negative urine and negative vaginal swab specimen (stabilized in cobas® PCR Media) spiked with CT and NG cultures at $\sim$ $3 \mathrm { ~ x ~ }$ LOD for each target. Eighteen over-the-counter (OTC) products, including contraceptive jelly, lubricants, feminine sprays, anti-fungal cream and anti-itch cream, as well as whole blood, cervical mucus and PBMC cells were tested for interference. In addition, several prescription drugs were tested in cobas® PCR Media plus negative urine, including clindamycin phosphate, estradiol, metronidazole and estrogen. Metronidazole vaginal gel was found to produce invalid and/or false negative results in cobas® PCR Media plus negative urine spiked with CT and NG cultures at $\sim 3 \mathrm { ~ x ~ }$ LOD for each target. The levels of whole blood, mucus and PBMC cells shown in the following table represent maximum allowable concentrations which will not interfere with cobas® CT/NG Test performance. Concentrations in urine samples were determined using total sample volume, including stabilizing media.

Results from Endogenous Interference Testing   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=2>Blood (v/v)</td><td rowspan=1 colspan=2>PBMC (cells/mL)</td><td rowspan=1 colspan=2>Mucus</td></tr><tr><td rowspan=1 colspan=1>Conc.Tested</td><td rowspan=1 colspan=1>InterferenceObserved</td><td rowspan=1 colspan=1>Conc.Tested</td><td rowspan=1 colspan=1>InterferenceObserved</td><td rowspan=1 colspan=1>Conc.Tested</td><td rowspan=1 colspan=1>InterferenceObserved</td></tr><tr><td rowspan=1 colspan=1>cobas® PCR Media + Urine</td><td rowspan=1 colspan=1>0, 0.25%,0.35%,0.5%,1%, 3%</td><td rowspan=1 colspan=1>&gt; 0.35%</td><td rowspan=1 colspan=1>0, 1.0E+05,1.0E+06,1.0E+07</td><td rowspan=1 colspan=1>&gt; 1x 105$</td><td rowspan=1 colspan=1>NT</td><td rowspan=1 colspan=1>NT</td></tr><tr><td rowspan=1 colspan=1>Vaginal Specimen stabilizedincobas® PCR Media</td><td rowspan=1 colspan=1>0, 1%,3%, 5%,10%</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>0, 1.0E+05,1.0E+06,1.0E+07</td><td rowspan=1 colspan=1>&gt; 1 x 105</td><td rowspan=1 colspan=1>Routinelevel*</td><td rowspan=1 colspan=1>None</td></tr></table>

Varying levels of albumin, glucose, bilirubin, low $\mathsf { p H }$ and high pH were also tested in cobas® PCR Media plus negative urine spiked with CT and NG cultures at $\sim 3 \mathrm { ~ x ~ }$ LOD for each target. Results, shown in the table below, indicate no interference except for bilirubin at $0 . 5 \%$ or higher.

Results from Additional Endogenous Interference Testing in Urine Stabilized in cobas® PCR Media   

<table><tr><td rowspan=1 colspan=1>SubstanceTested</td><td rowspan=1 colspan=1>Levels Tested</td><td rowspan=1 colspan=1>InterferenceObserved</td></tr><tr><td rowspan=1 colspan=1>Albumin</td><td rowspan=1 colspan=1>0%. 0.5%, 1%, 2%, and 5% (w/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Glucose</td><td rowspan=1 colspan=1>0%, 0.1% and 1% (w/v)</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Bilirubin</td><td rowspan=1 colspan=1>0%, 0.1%, 0.2%, 0.5% and 1%(w/v)</td><td rowspan=1 colspan=1>≥ 0.5%</td></tr><tr><td rowspan=1 colspan=1>AcidicCondition</td><td rowspan=1 colspan=1>pH 4</td><td rowspan=1 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Basic Condition</td><td rowspan=1 colspan=1>pH 9</td><td rowspan=1 colspan=1>None</td></tr></table>

# h. Carry over/contamination

Sample-to-sample and run-to-run cross-contamination carryover studies were performed on the cobas $^ \mathrm { \textregistered }$ 4800 System using the CT/NG Workflow. Testing was performed on three instruments and included testing of five runs of high-positive and negative samples in a checkerboard configuration, followed by a sixth run of all negative samples. The sample-tosample cross contamination rate for these studies was $1 . 2 4 \%$ and the run-to-run carry over rate was $0 . 0 \%$ .

# i. Assay cut-off:

A preliminary Ct value cut-off for both CT and NG was initially verified using data from the Limit of Detection (LOD) studies. CT and NG spiked samples at, above and below the LOD of the test were used to identify the Ct value range for known positive samples at diminishingly low levels. The lowest level tested produced positive hit rates near $50 \%$ , which would correspond to approximately 1 copy of target per PCR based on a Poisson distribution. Samples without the addition of CT or NG were used to confirm that negative results did not generate Ct values. Next, data from the cobas $^ \mathrm { \textregistered }$ CT/NG clinical trial were compiled and analyzed to identify the distribution of CT and NG Ct values for all specimen types and to validate the appropriate separation between the highest Ct value observed in positive specimens and the assay cut-off.

# j. Competitive Inhibition Studies

Panels were prepared by spiking CT and NG cultures into urine and vaginal specimens stabilized in cobas® PCR Media to various concentration levels to examine the potential for competitive inhibition. Panels were prepared with two strains each of CT and NG. Panels were tested in one run per day over the course of 5 days. Two replicates of each panel member were tested in every run, generating a maximum of 10 test results for each level and CT and NG strain respectively. Average Ct values for each of the panel levels are summarized in the tables below. All CT and NG hit rates were $100 \%$ for all panel levels in both matrices. Competitive inhibition was not seen in any combination of CT and NG levels in either matrix.

Competitive Inhibition Study for CT and NG Cultures in Urine Stabilized in cobas® PCR Media (Ct Values)   

<table><tr><td rowspan=1 colspan=2>Panel Level</td><td rowspan=1 colspan=2>Strain 1</td><td rowspan=1 colspan=2>Strain 2</td></tr><tr><td rowspan=1 colspan=1>CT Level/ IFU/mL</td><td rowspan=1 colspan=1>NG Level/(CFU/mL)</td><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>NG</td><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>NG</td></tr><tr><td rowspan=1 colspan=1>Low/2</td><td rowspan=1 colspan=1>Low/6</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>34.8</td></tr><tr><td rowspan=1 colspan=1>Low/2</td><td rowspan=1 colspan=1>Medium/24</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>33.0</td></tr><tr><td rowspan=1 colspan=1>Medium/8</td><td rowspan=1 colspan=1>Low/6</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>High/1.00E+05</td><td rowspan=1 colspan=1>Low/6</td><td rowspan=1 colspan=1>19.4</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>18.6</td><td rowspan=1 colspan=1>35.0</td></tr><tr><td rowspan=1 colspan=1>Low/2</td><td rowspan=1 colspan=1>High/1.00E+06</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>17.6</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>17.4</td></tr><tr><td rowspan=1 colspan=1>High/1.00E+05</td><td rowspan=1 colspan=1>High/1.00E+06</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>18.1</td><td rowspan=1 colspan=1>18.9</td><td rowspan=1 colspan=1>17.7</td></tr></table>

Competitive Inhibition Study for CT and NG Cultures in Vaginal Swabs Collected in cobas® PCR Media (Ct Values)   

<table><tr><td rowspan=1 colspan=2>Panel Level</td><td rowspan=1 colspan=2>Strain 1</td><td rowspan=1 colspan=2>Strain 2</td></tr><tr><td rowspan=1 colspan=1>CTLevel/IFU/mL</td><td rowspan=1 colspan=1>NG LevelI(CFU/mL)</td><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>NG</td><td rowspan=1 colspan=1>CT</td><td rowspan=1 colspan=1>NG</td></tr><tr><td rowspan=1 colspan=1>Low/25</td><td rowspan=1 colspan=1>Low/250</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>34.7</td></tr><tr><td rowspan=1 colspan=1>Low/25</td><td rowspan=1 colspan=1>Medium/1000</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>33.7</td><td rowspan=1 colspan=1>31.8</td></tr><tr><td rowspan=1 colspan=1>Medium/100</td><td rowspan=1 colspan=1>Low/250</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>34.4</td></tr><tr><td rowspan=1 colspan=1>High/1.00E+05</td><td rowspan=1 colspan=1>Low/250</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>33.8</td></tr><tr><td rowspan=1 colspan=1>Low/25</td><td rowspan=1 colspan=1>High/1.00E+06</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>22.8</td><td rowspan=1 colspan=1>33.8</td><td rowspan=1 colspan=1>22.0</td></tr><tr><td rowspan=1 colspan=1>High/1.00E+05</td><td rowspan=1 colspan=1>High/1.00E+06</td><td rowspan=1 colspan=1>23.7</td><td rowspan=1 colspan=1>23.2</td><td rowspan=1 colspan=1>21.9</td><td rowspan=1 colspan=1>22.1</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate devices: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

Specimen collection took place at 12 collection sites in the US, which included family planning and Obstetrics/Gynecology (OB/GYN) clinics, and sexually transmitted disease clinics. A total of 4 laboratory sites performed specimen testing. In the clinical study, a total of 12/286 $( 4 . 2 \% )$ runs were classified as invalid due to instrument or operator error. There were no invalid runs due to test controls failure. At collection sites, female subjects provided a self-collected vaginal specimen and male subjects provided a urine specimen in the respective cobas® media collection kits. To determine the patient infected status, female patients provided a urine specimen and clinician-collected endocervical swab specimen in collection media from two reference nucleic acid amplification tests (NAAT), and a cervical specimen in PreservCyt® Solution (Hologic Corporation, Bedford, MA) obtained with a spatula/cytobrush or a broom. To determine the patient infected status, male subjects provided urethral swab specimens and urine specimens in collection media from two reference nucleic acid amplification tests (NAAT). Subjects were classified as symptomatic if they reported symptoms indicative of CT or NG infection, as listed below.

Dysuria/pain during urination, coital pain/difficulty/bleeding, discharge, or pelvic pain   
· Abnormal vaginal discharge   
· Pelvic/uterine/ovarian pain   
· Urethral discharge, testicular pain/scrotal pain/swelling

Subjects were classified as asymptomatic if they did not report these symptoms. Samples were tested for CT and NG using the cobas® CT/NG Test and two reference nucleic acid amplification tests (NAAT). Testing with all devices followed the manufacturers’ instructions. The clinical performance of the cobas® CT/NG Test was evaluated by comparing the results from collected sample types to a pre-specified PIS (Patient Infected Status) algorithm as determined by combined results from 2 commercially available nucleic acid amplification tests (reference devices).

Determination of Patient Infected Status   

<table><tr><td rowspan=1 colspan=1>NAAT1Urine/Endocervical</td><td rowspan=1 colspan=1>NAAT2Urine/Endocervical</td><td rowspan=1 colspan=1>NAAT2Cervical Swab inPreservCyt Solution</td><td rowspan=1 colspan=1>Patient InfectedStatus (PIS)</td></tr><tr><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>+/- or -/+</td><td rowspan=1 colspan=1>+or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>+/- or -/+</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>-/+</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>-/+</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>-/+</td><td rowspan=1 colspan=1>-/+</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Infected</td></tr><tr><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1>+/-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Infected (Urine)Non-Infected(Swabs)</td></tr><tr><td rowspan=1 colspan=1>+/- or -/+</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Non-Infected</td></tr><tr><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Non-Infected</td></tr><tr><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/+</td><td rowspan=1 colspan=1>+or -</td><td rowspan=1 colspan=1>Non-Infected</td></tr><tr><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+/- or -/+</td><td rowspan=1 colspan=1>+or -</td><td rowspan=1 colspan=1>Non-Infected</td></tr><tr><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>-/-</td><td rowspan=1 colspan=1>+ or -</td><td rowspan=1 colspan=1>Non-Infected</td></tr></table>

For the primary objective, sensitivity (SENS), specificity (SPEC), positive predictive value (PPV), and negative predictive value (NPV) of the cobas® CT/NG Test were calculated separately for detection of CT or NG by using PIS as the reference standard. For each gender and within each sample type, the analyses were performed overall and then subset by symptom status, collection site, and age category. In addition, the predictive values were calculated based on sensitivity and specificity with all data combined for a range of hypothetical prevalence values.

Of the 2,985 subjects (2,195 females and 790 males) tested with the cobas® CT/NG Test, 3 were excluded from the analyses because they did not meet study entry criteria or because they withdrew consent; 131 were considered non-evaluable and excluded from all statistical analyses because of errors in specimen collection, transport, and storage; unknown PIS for both CT and NG; or invalid cobas® CT/NG test results after initial testing and/or retesting. Therefore, of 2,982 subjects enrolled, 2,851 $( 9 5 . 6 \% )$ were evaluable for CT and NG primary analyses (2,083 females and 768 males). All performance calculations were based on valid cobas® CT/NG Test results from male urine specimens and self-collected vaginal swab specimens.

# Chlamydia trachomatis (CT)

The following tables summarize the results from symptomatic and asymptomatic subjects designated as infected or non-infected with CT (females and males, respectively) according to the PIS algorithm. A total of 131 females and 126 males were infected with CT. Symptoms were reported in $6 1 . 1 \%$ (80/131) of infected and $5 0 . 9 \%$ (993/1,952) of non-infected women. Similarly, symptoms were reported in $5 8 . 7 \%$ (74/126) of infected and $3 4 . 6 \%$ (222/642) of non-infected men. Overall, the CT prevalence was $6 . 3 \%$ (131/2,083), $1 6 . 4 \%$ (126/768), and $9 . 0 \%$ (257/2,851) in women, men, and the entire study population, respectively.

CT: Positive/Negative Analysis for Female Patient Infected Status   

<table><tr><td></td><td></td><td></td><td rowspan=1 colspan=7>C1: 1 osltive/Negative Allalysis lor Femlale 1 atient linected Status</td></tr><tr><td rowspan=2 colspan=1>Patient InfectedStatus</td><td rowspan=1 colspan=2>NAAT1</td><td rowspan=1 colspan=3>NAAT2</td><td rowspan=1 colspan=1>cobas CT/NGTest</td><td rowspan=1 colspan=2>Symptom Statusa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>PC Pre</td><td rowspan=1 colspan=1>VG</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total Infected</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>80</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>131</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>957</td><td rowspan=1 colspan=1>911</td><td rowspan=1 colspan=1>1868</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=7>Total Non-Infected</td><td rowspan=1 colspan=1>993</td><td rowspan=1 colspan=1>959</td><td rowspan=1 colspan=1>1952</td></tr></table>

a Symp $=$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with CT if at least 2 NAATs with different target regions gave positive results for the endocervical swab and/or urine specimen. However, females were categorized as non–infected for any swab specimen if the swab specimens and the PreservCyt specimen (NAAT2) were negative and the urine specimens were positive. Note: Subjects with designated infection status and valid cobas® CT/NG Test results were considered evaluable and are included in this summary table. Note: $^ +$ denotes Positive; – denotes Negative; NA indicates specimen was not obtained for testing or test result was missing/invalid. Note: ${ \mathsf { S W } } =$ endocervical swab; UR $=$ urine; VG $=$ vaginal swab; PC Pre $=$ PreservCyt (pre–aliquot).

CT: Positive/Negative Analysis for Male Patient Infected Status   

<table><tr><td rowspan=3 colspan=2>PatientInfectedStatus</td><td rowspan=2 colspan=2>NAAT1</td><td rowspan=2 colspan=2>NAAT2</td><td rowspan=2 colspan=1>cobasCT/NG Test</td><td rowspan=2 colspan=2>Symptom Statusa</td><td rowspan=3 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>Asymp</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Total Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>126</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>218</td><td rowspan=1 colspan=1>412</td><td rowspan=1 colspan=1>630</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=2>Non-Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=7>Total Non-Infected</td><td rowspan=1 colspan=1>222</td><td rowspan=1 colspan=1>420</td><td rowspan=1 colspan=1>642</td></tr></table>

a Symp $=$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with CT if at least 2 NAATs with different target regions gave positive results for the urethral swab and/or the urine specimen. Note: Subjects with designated infection status and valid cobas® CT/NG Test results were considered evaluable and are included in this summary table. Note: $^ +$ denotes Positive; – denotes Negative. Note: ${ \mathsf { S W } } =$ urethral swab; $\mathsf { U R } =$ urine.

Sensitivity, specificity, and predictive values of the cobas® CT/NG Test for CT defined by PIS are presented by gender, sample type, and symptom status in the following table. Sensitivity was $9 3 . 9 \%$ and $9 7 . 6 \%$ , respectively for self-collected vaginal swabs and male urine specimens. Sensitivity was similar by symptom status. Regardless of symptom status, specificity for CT ranged from $9 9 . 5 \% 9 9 . 7 \%$ in both females and males. With all data combined, PPV and NPV were $9 6 . 5 \%$ and $9 9 . 6 \%$ , respectively.

CT: Clinical Performance Compared With Patient Infected Status by Gender, Sample Type, and Symptom Status   

<table><tr><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>Symptom Statusb</td><td rowspan=1 colspan=1>Total(n)</td><td rowspan=1 colspan=1>SENS</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>SPEC</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>PREV(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=10>Female</td></tr><tr><td rowspan=3 colspan=1>VG</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>1073</td><td rowspan=1 colspan=1>93.8%(75/80)</td><td rowspan=1 colspan=1>(86.2%,97.3%)</td><td rowspan=1 colspan=1>99.7%(990/993)</td><td rowspan=1 colspan=1>(99.1%,99.9%%</td><td rowspan=1 colspan=1>7.5</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>1010</td><td rowspan=1 colspan=1>94.1%(48/51)</td><td rowspan=1 colspan=1>(84.1%,98.0%)</td><td rowspan=1 colspan=1>99.7%(956/959)</td><td rowspan=1 colspan=1>(99.1%,99.9%)</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>94.1</td><td rowspan=1 colspan=1>99.7</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2083</td><td rowspan=1 colspan=1>93.9%(123/131)</td><td rowspan=1 colspan=1>(88.4%,96.9%)</td><td rowspan=1 colspan=1>99.7%(1946/1952)</td><td rowspan=1 colspan=1>(99.3%,99.9%)</td><td rowspan=1 colspan=1>6.3</td><td rowspan=1 colspan=1>95.3</td><td rowspan=1 colspan=1>99.6</td></tr><tr><td rowspan=1 colspan=1>Male</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=3 colspan=1>UR</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>97.3%(72/74)</td><td rowspan=1 colspan=1>(90.7%,99.3%)</td><td rowspan=1 colspan=1>99.5%(221/222)</td><td rowspan=1 colspan=1>(97.5%,99.9%)</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>98.6</td><td rowspan=1 colspan=1>99.1</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>98.1%(51/52)</td><td rowspan=1 colspan=1>(89.9%,99.7%)</td><td rowspan=1 colspan=1>99.5%(418/420)</td><td rowspan=1 colspan=1>(98.3%,99.9%)</td><td rowspan=1 colspan=1>11.0</td><td rowspan=1 colspan=1>96.2</td><td rowspan=1 colspan=1>99.8</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>768</td><td rowspan=1 colspan=1>97.6%(123/126)</td><td rowspan=1 colspan=1>(93.2%,99.2%)</td><td rowspan=1 colspan=1>99.5%(639/642)</td><td rowspan=1 colspan=1>(98.6%,99.8%)</td><td rowspan=1 colspan=1>16.4</td><td rowspan=1 colspan=1>97.6</td><td rowspan=1 colspan=1>99.5</td></tr><tr><td rowspan=1 colspan=2>All Combined</td><td rowspan=1 colspan=1>2851</td><td rowspan=1 colspan=1>95.7%(246/257)</td><td rowspan=1 colspan=1>(92.5%,97.6%)</td><td rowspan=1 colspan=1>99.7%(2585/2594)</td><td rowspan=1 colspan=1>(99.3%,99.8%</td><td rowspan=1 colspan=1>9.0</td><td rowspan=1 colspan=1>96.5</td><td rowspan=1 colspan=1>99.6</td></tr></table>

a VG- $\updownarrow =$ self-collected vaginal swab; $\mathrm { U R } = \mathrm { u r i n e }$ . b Symp $=$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with CT if at least 2 NAATs with different target regions gave positive results for the endocervical swab (urethral swab for males) and/or the urine specimen. However, females were categorized as non-infected for any swab specimen if the swab specimens and the PreservCyt specimen (NAAT2) were negative and the urine specimens were positive. Note: Subjects with designated infection status and valid cobas® CT/NG Test results were considered evaluable and included in this summary table. Note: $\mathrm { C I } =$ (score) confidence interval; $\mathrm { P R E V = }$ prevalence; SENS $=$ sensitivity; SPEC $=$ specificity; $\mathrm { P P V = }$ positive predictive value; $\mathrm { N P V = }$ negative predictive value.

# Neisseria gonorrhoeae (NG)

The following tables summarize the results from symptomatic and asymptomatic subjects designated as infected or non-infected with NG (females and males, respectively) according to the PIS algorithm. A total of 33 females and 71 males were infected with NG. Symptoms were reported in $6 9 . 7 \%$ (23/33) of infected and $5 1 . 2 \%$ (1,050/2,050) of non-infected women. Of the 768 male subjects enrolled, 296 $( 3 8 . 5 \% )$ were symptomatic and 472 $( 6 1 . 5 \% )$ were asymptomatic. Overall, the NG prevalence was $1 . 6 \%$ (33/2,083), $9 . 2 \%$ (71/768), and $3 . 6 \%$ (104/2,851), respectively, in women, men, and the entire study population.

NG Positive/Negative Analysis for Female Patient Infected Status   

<table><tr><td colspan="1" rowspan="2">Patient InfectedStatus</td><td colspan="2" rowspan="1">NAAT1</td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">NAAT2</td><td colspan="1" rowspan="1">cobas CT/NGTest</td><td colspan="2" rowspan="1">Symptom Statusa</td><td colspan="1" rowspan="2">Total</td></tr><tr><td colspan="1" rowspan="1">SW</td><td colspan="1" rowspan="1">UR</td><td colspan="1" rowspan="1">SW</td><td colspan="1" rowspan="1">UR</td><td colspan="1" rowspan="1">PCPre</td><td colspan="1" rowspan="1">VG</td><td colspan="1" rowspan="1">Symp</td><td colspan="1" rowspan="1">Asymp</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">25</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Total Infected</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">33</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1017</td><td colspan="1" rowspan="1">958</td><td colspan="1" rowspan="1">1975</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">44</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">9</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+$</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Non-Infected</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="7" rowspan="1">Total Non-Infected</td><td colspan="1" rowspan="1">1050</td><td colspan="1" rowspan="1">1000</td><td colspan="1" rowspan="1">2050</td></tr></table>

a Symp $=$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with NG if at least 2 NAATs with different target regions gave positive results for the endocervical swab and/or the urine specimen. Note: Subjects with designated infection status and valid cobas® CT/NG Test results were considered evaluable and are included in this summary table. Note: $^ +$ denotes Positive; – denotes Negative; NA indicates specimen was not obtained for testing or test result was missing/invalid. Note: ${ \mathsf { S W } } =$ endocervical swab; UR $=$ urine; VG $=$ vaginal swab; PC Pre $=$ PreservCyt (pre-aliquot).

NG: Positive/Negative Analysis for Male Patient Infected Status   

<table><tr><td rowspan=2 colspan=1>PatientInfectedStatus</td><td rowspan=1 colspan=2>NAAT1</td><td rowspan=1 colspan=2>NAAT2</td><td rowspan=1 colspan=1>cobasCT/NG Test</td><td rowspan=1 colspan=2>Symptom Statusa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>SW</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>UR</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=6>Total Infected</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>71</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>227</td><td rowspan=1 colspan=1>464</td><td rowspan=1 colspan=1>691</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Non-Infected</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=6>Total Non-Infected</td><td rowspan=1 colspan=1>232</td><td rowspan=1 colspan=1>465</td><td rowspan=1 colspan=1>697</td></tr></table>

a Symp $=$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with NG if at least 2 NAATs with different target regions gave positive results for the urethral swab and/or the urine specimen. Note: Subjects with designated infection status and valid cobas® CT/NG Test results are considered evaluable and are included in this summary table. Note: $^ +$ denotes Positive; – denotes Negative. Note: SW $=$ urethral swab; $\mathsf { U R } =$ urine.

Sensitivity, specificity, and predictive values of the cobas® CT/NG Test for NG as defined by PIS are shown by gender, sample type, and symptom status in the table below. Sensitivity ranged from $9 5 . 7 \% - 1 0 0 . 0 \%$ . Overall specificity ranged from $9 9 . 1 \% - 1 0 0 . 0 \%$ for both females and males. Performance estimates for NG detection were similar between symptomatic and asymptomatic subjects for vaginal swab specimens and urine specimens. With all data combined, PPV and NPV were $9 7 . 2 \%$ and $1 0 0 . 0 \%$ , respectively.

NG: Clinical Performance Compared With Patient Infected Status by Sex, Sample Type, and Symptom Status   

<table><tr><td rowspan=1 colspan=1>SampleType</td><td rowspan=1 colspan=1>Symptom Status</td><td rowspan=1 colspan=1>Total(n)</td><td rowspan=1 colspan=1>SENS</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>SPEC</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>PREV(%)</td><td rowspan=1 colspan=1>PPV(%)</td><td rowspan=1 colspan=1>NPV(%)</td></tr><tr><td rowspan=1 colspan=10>Female</td></tr><tr><td rowspan=3 colspan=1>VG</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>1073</td><td rowspan=1 colspan=1>95.7%(22/23)</td><td rowspan=1 colspan=1>(79.0%,99.2%</td><td rowspan=1 colspan=1>99.9%(1049/1050)</td><td rowspan=1 colspan=1>(99.5%,10.0%)</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>99.9</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>1010</td><td rowspan=1 colspan=1>100.0%(10/10)</td><td rowspan=1 colspan=1>(72.2%,100.0%)</td><td rowspan=1 colspan=1>100.0%(1000/1000)</td><td rowspan=1 colspan=1>(99.6%,100.0%)</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>2083</td><td rowspan=1 colspan=1>97.0%(32/33)</td><td rowspan=1 colspan=1>(84.7%,99.5%</td><td rowspan=1 colspan=1>100.0%(2049/2050)</td><td rowspan=1 colspan=1>(99.7%,100.0%)</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>UR</td><td rowspan=1 colspan=1>Symp</td><td rowspan=1 colspan=1>296</td><td rowspan=1 colspan=1>100.0%(64/64)</td><td rowspan=1 colspan=1>(94.3%,100.0%)</td><td rowspan=1 colspan=1>99.1%(230/232)</td><td rowspan=1 colspan=1>(96.9%,99.8%)</td><td rowspan=1 colspan=1>21.6</td><td rowspan=1 colspan=1>97.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Asymp</td><td rowspan=1 colspan=1>472</td><td rowspan=1 colspan=1>100.0%(7/7)</td><td rowspan=1 colspan=1>(64.6%,1000%)</td><td rowspan=1 colspan=1>100.0%(465/465)</td><td rowspan=1 colspan=1>(99.2%,100.0%)</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>100.0</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>768</td><td rowspan=1 colspan=1>100.0%(71/71)</td><td rowspan=1 colspan=1>(94.9%,100.0%)</td><td rowspan=1 colspan=1>99.7%(695/697)</td><td rowspan=1 colspan=1>(99.0%,99.9%)</td><td rowspan=1 colspan=1>9.2</td><td rowspan=1 colspan=1>97.3</td><td rowspan=1 colspan=1>100.0</td></tr><tr><td rowspan=1 colspan=2>All Combined</td><td rowspan=1 colspan=1>2851</td><td rowspan=1 colspan=1>99.0%(103/104)</td><td rowspan=1 colspan=1>(94.8%,99.8%</td><td rowspan=1 colspan=1>99.9%(2744/2747)</td><td rowspan=1 colspan=1>(99.7%,100.0%)</td><td rowspan=1 colspan=1>3.6</td><td rowspan=1 colspan=1>97.2</td><td rowspan=1 colspan=1>100.0</td></tr></table>

${ } ^ { \mathrm { a } } \mathrm { V G } { - } \mathrm { S } =$ self-collected vaginal swab; $\mathrm { U R } =$ urine. ${ } ^ { \mathrm { b } } \operatorname { S y m p } =$ symptomatic; Asymp $=$ asymptomatic. Note: Subjects were designated as being infected with NG if at least 2 NAATs with different target regions gave positive results for the endocervical swab (urethral swab for males) and/or the urine specimen. Note: Subjects with designated infection status and valid cobas® CT/NG Test results were considered evaluable and included in this summary table. Note: $\mathrm { C I } =$ (score) confidence interval; $\mathrm { P R E V = }$ prevalence; SENS $=$ sensitivity; SPEC $=$ specificity; $\mathrm { P P V = }$ positive predictive value; $\mathrm { N P V = }$ negative predictive value.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

The prevalence of CT observed with the cobas $^ \mathrm { \textregistered }$ CT/NG Test during the clinical study ranged from $5 . 0 \%$ to $7 . 5 \%$ in females, and from $1 1 . 0 \%$ to $2 5 . 0 \%$ in males; the prevalence of NG ranged from $1 . 0 \%$ to $2 . 1 \%$ in females, and from $1 . 5 \%$ to $2 1 . 6 \%$ in males.

# N. Instrument Name:

Roche cobas $^ \mathrm { \textregistered }$ 4800 System which consists of cobas $4 8 0 \mathrm { x }$ and cobas $4 8 0 z$ instruments.

# O. System Descriptions:

1. Modes of Operation:

The Roche cobas $^ \mathrm { \textregistered }$ 4800 System operates in a batch mode with open sample tubes. The system may operate in full extraction and amplification mode or in PCR only mode with previously extracted samples.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for   
this line of product types:   
Yes __X_ or No

3. Specimen Identification: Specimens are identified using barcodes.

4. Specimen Sampling and Handling: Samples are placed on the instrument as open tubes.

5. Calibration:

No calibration is required by the user. Roche technicians perform calibration periodically as required.

6. Quality Control:

Positive and Negative Controls are included in every run. An Internal control is introduced for each Control and Specimen during sample preparation on the cobas x 480 instrument reaction.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.