# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K111860   
B. Purpose for Submission: To obtain a substantial equivalence determination for the BD MAXTM GBS Assay on the next generation BD MAXTM system.   
C. Measurand: Group B Streptococcus (S. agalactiae) (GBS) DNA (124 bp region of cfb gene sequence)   
D. Type of Test: Nucleic acid amplification assay system, automated   
E. Applicant: BD Diagnostics   
F. Proprietary and Established Names: BD MAX™ GBS Assay BD MAXTM System   
G. Regulatory Information:

#

1. Regulation section: 866.3740 – Streptococcal spp. serological reagents 862.2570 – Instrumentation for clinical multiplex test systems

2. Classification: Class I, II

NJR – Nucleic Acid Amplification Assay System, Group B Streptococcus, Direct Specimen   
OOI – Real-time nucleic acid amplification

4. Panel:

Microbiology (83)

# H. Intended Use:

1. Intended uses:

The BD MAX™ GBS Assay as implemented on the BD MAX™ System is a qualitative in vitro diagnostic test designed to detect Group B Streptococcus (GBS) DNA in Lim Broth cultures after incubation for greater than or equal to $( > ) 1 8$ hours, obtained from vaginal-rectal swab specimens from antepartum pregnant women. The test incorporates automated DNA extraction to isolate the target nucleic acid from the specimen and realtime polymerase chain reaction (PCR) to detect a 124 bp region of the $c f b$ gene sequence of the Streptococcus agalactiae chromosome. Results from the BD MAX™ GBS Assay can be used as an aid in determining colonization status in antepartum women.

The BD MAX™ GBS Assay does not provide susceptibility results. Cultured isolates are needed for performing susceptibility testing as recommended for penicillin-allergic women. Subculture to solid media for additional testing when indicated.

The BD MAX™ System is intended for in vitro diagnostic (IVD) use in performing FDA cleared or approved nucleic acid testing in clinical laboratories. The BD MAX System is capable of automated extraction and purification of nucleic acids from multiple specimen types as well as the automated amplification and detection of target nucleic acid sequences by fluorescence-based PCR.

# 2. Indications for use:

The BD MAX™ GBS Assay as implemented on the BD MAX™ System is a qualitative in vitro diagnostic test designed to detect Group B Streptococcus (GBS) DNA in Lim Broth cultures after incubation for greater than or equal to $( > ) 1 8$ hours, obtained from vaginal-rectal swab specimens from antepartum pregnant women. The test incorporates automated DNA extraction to isolate the target nucleic acid from the specimen and realtime polymerase chain reaction (PCR) to detect a 124 bp region of the $c f b$ gene sequence of the Streptococcus agalactiae chromosome. Results from the BD $\mathbf { M A X T M }$ GBS Assay can be used as an aid in determining colonization status in antepartum women.

The BD MAX™ GBS Assay does not provide susceptibility results. Cultured isolates are needed for performing susceptibility testing as recommended for penicillin-allergic women. Subculture to solid media for additional testing when indicated.

The BD MAX™ System is intended for in vitro diagnostic (IVD) use in performing FDA cleared or approved nucleic acid testing in clinical laboratories. The BD MAX System is capable of automated extraction and purification of nucleic acids from multiple specimen types as well as the automated amplification and detection of target nucleic acid sequences by fluorescence-based PCR.

3. Special conditions for use statement(s): Prescription Use Only

4. Special instrument requirements: BD MAX™ System

# I. Device Description:

The BD MAX™ System and BD MAX™ GBS Assay are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, BD MAX GBS Master Mix, BD MAX DNA Unitized Reagent Strips, BD MAX GBS Extraction Reagent, and BD MAX GBS Sample preparation Reagent. These components are used to extract, amplify, and detect GBS nucleic acid from vaginal/rectal swabs. This process is fully automated and requires user intervention only for loading and unloading samples. Each run has a minimum of one to a maximum of 24 samples, which requires one to 24 disposable unitized reagent strips and one or two microfluidic cartridges. On completion of a run, the user removes the used cartridges and unitized reagent strips and disposes of them in normal biological waste. For the GBS Assay, the results are displayed and stored as Positive, Negative or Indeterminate.

# J. Substantial Equivalence Information:

1. Predicate device name: BD MAX™ GBS Assay performed on the BD MAX™ System

2. Predicate 510(k) number: K090191

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD MAX GBS Assay on the BD MAX2nd Generation System</td><td rowspan=1 colspan=1>BD MAX GBSAssay on the BDMAX 1rstGenerationSystem</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>For detection of GBS</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>GBS DNA - Cfb gene</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimentype</td><td rowspan=1 colspan=1>Vaginal-Rectal Swab (Enriched LimBroth)</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>Automated DNA extraction</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Assay Format</td><td rowspan=1 colspan=1>Amplification: Real Time PCRDetection: Fluorogenic</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Probe Design</td><td rowspan=1 colspan=1>Scorpion</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>DNA Target</td><td rowspan=1 colspan=1>124 bp region of cfb gene</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>AutomaticAssay</td><td rowspan=1 colspan=1>Yes-result interpretation</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>InternalProcessControl</td><td rowspan=1 colspan=1>Extraction and PCR internal control is aprocess monitor</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>ExternalControl</td><td rowspan=1 colspan=1>Materials available commercially butnot required to run the test</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>DNA extraction is automated on BDMAX System</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>BD MAX GBS Assay on the BD MAX2nd Generation System</td><td rowspan=1 colspan=1>BD MAX GBSAssay on the BDMAX 1rstGeneration System</td></tr><tr><td rowspan=1 colspan=1>GBS AssayCartridge</td><td rowspan=1 colspan=1>Can be used twice  Contains 24 testchannels</td><td rowspan=1 colspan=1>Can be used onlyonce - Contains 12test channels</td></tr><tr><td rowspan=1 colspan=1>Single Use</td><td rowspan=1 colspan=1>Cartridge can be used twice</td><td rowspan=1 colspan=1>Cartridge can be usedonly once</td></tr><tr><td rowspan=1 colspan=1>BD MAXInstrument</td><td rowspan=1 colspan=1>Contains 6 channels</td><td rowspan=1 colspan=1>Contains 2 channels</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

‘Draft Guidance for Industry and FDA Staff - Assay Migration Studies for In Vitro   
Diagnostic Devices’ issued January 5, 2009   
‘Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document:   
Instrumentation for Clinical Multiplex Test Systems’, issued March 10, 2005   
‘Guidance for the Content of Premarket Submissions for Software Contained in Medical   
Devices’, issued May 11, 2005

# L. Test Principle:

Following $\mathtt { a } > 1 8$ hour enrichment process in Lim broth, a $1 5 ~ \mu \mathrm { L }$ aliquot of the broth is used for detection of the presence of GBS. The aliquot of the broth is mixed with BD MAX Sample Preparation Reagent and processed using the BD MAX System. The BD MAX System automates and integrates DNA extraction and concentration, reagent preparation, and nucleic acid amplification and detection of the target sequence using real-time Polymerase Chain Reaction (PCR). An internal process control is also incorporated into the lysis, extraction, concentration and amplification steps to monitor for the presence of potential inhibitory substances as well as system or reagent failures. No operator intervention is necessary once the clinical sample is loaded onto the BD MAX System.

The BD MAX System uses a combination of lytic and extraction reagents to perform cell lysis, DNA extraction and removal of inhibitors. Following cell lysis, with a combination of heat and lytic enzymes, the released nucleic acids are captured by magnetic affinity beads. The beads, with the bound nucleic acids, are washed and the nucleic acids are eluted using release solution and prepared for PCR by addition of neutralization reagent. The BD MAX System then uses the PCR-ready DNA solution to rehydrate a freeze-dried PCR pellet containing all the reagents necessary for amplification of the GBS-specific target. The freezedried PCR pellet also contains reagents to amplify a section of the process control sequence to enable simultaneous amplification and detection of both target and control DNA sequences. After reconstitution of the freeze-dried amplification reagents, the BD MAX System dispenses the prepared PCR-ready solution into one lane (per specimen) of the BD MAX PCR Cartridge. Microvalves in the BD MAX PCR Cartridge are sealed by the system prior to initiation of PCR to prevent evaporation as well as amplicon contamination.

The amplified targets are detected in real time using Scorpions® chemistry- based fluorogenic oligonucleotide probe molecules specific to the amplicons for the respective target. Scorpion chemistry features a bi-functional molecule which includes a PCR primer covalently attached to a probe. The Scorpion primers used in the BD MAX GBS Assay have a fluorophore and quencher held together by an internal stem loop. First, the Scorpion primer is extended on the target DNA. The extended primer is then heat-denatured, along with the stem loop of the

probe, thereby causing the quencher and the fluorophore to disassociate. Next, the extended Scorpion primer is rearranged and binds to the newly extended DNA strand as it cools and begins to fluoresce in a target-specific manner, while the un-extended primer is quenched. The difference between Scorpion chemistry and other detection systems is that the probe and primer are on the same molecule so that signal generation is through a uni-molecular rearrangement, as opposed to a bimolecular collision. This results in extremely rapid signal generation kinetics for Scorpion reactions.

A Scorpion probe labeled with a fluorophore (Ex $\because 4 9 0 { \mathrm { n m } }$ & Em: 521nm) at the $5 ^ { \circ }$ end, and a dark quencher at the $3 ^ { \circ }$ end, is used to detect GBS DNA. For detection of the internal process control, the Scorpion probe is labeled with an alternate fluorescent dye (Ex: $5 9 0 \ \mathrm { n m } \ \& \ \mathrm { E m } \colon 6 1 0 \ \mathrm { n m } )$ at the $5 ^ { \circ }$ end, and a dark quencher at the $3 ^ { \circ }$ end. The BD MAX System monitors the fluorescent signal emitted by the Scorpion probes at the end of each amplification cycle. When amplification is complete, the BD MAX System analyzes the data and provides a final result.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Reproducibility Study

A Reproducibility study was performed using a panel of samples spiked at the GBS concentrations described below. Each panel consisted of 20 samples which included five replicates of four GBS concentrations. Samples for the study were prepared internally at BD using Streptococcus agalactiae (ATCC strain 27956) and shipped frozen to the testing sites. Samples were thawed and vortexed prior to testing. The run panels were masked to the operators according to a designated randomization scheme. Testing was performed at two external and one internal site using the 2nd generation BD MAX system at each site. Six runs were performed over three days at each of the three testing sites for a total of 360 samples tested. The results of the studies are summarized in the tables below.

Inter-Laboratory Reproducibility Panel Composition   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Target concentration - CFU/assay</td><td rowspan=1 colspan=1>Replicates</td></tr><tr><td rowspan=1 colspan=1>High Negative (HN)</td><td rowspan=1 colspan=1>~6( ~ 1:50 dilution of LoD )</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LP)</td><td rowspan=1 colspan=1>~315(~ 1x LoD )</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (MP)</td><td rowspan=1 colspan=1>~ 1050(~ 3x LoD )</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>True Negative (TN)</td><td rowspan=1 colspan=1>0( No Target)</td><td rowspan=1 colspan=1>5</td></tr></table>

Qualitative Reproducibility Study Results Percent Agreement and Confidence Intervals (CI)   

<table><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Category</td></tr><tr><td rowspan=1 colspan=1>100.0% (30/30)(88.6%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (30/30(88.6%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (35/35)(90.1%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (95/95)(96.1%, 100.0%)</td><td rowspan=1 colspan=1>MP</td></tr><tr><td rowspan=1 colspan=1>100.0% (30/30)(88.6%, 100.0%)</td><td rowspan=1 colspan=1>96.7%(29/30)(83.3%, 99.4%)</td><td rowspan=1 colspan=1>100.0% (35/35)(90.1%, 100.0%)</td><td rowspan=1 colspan=1>99.0% (94/95)(94.3%, 99.8%)</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>83.3% (25/30)(66.4%, 92.7%)</td><td rowspan=1 colspan=1>70.0%(21/30)(52.1%, 83.3%)</td><td rowspan=1 colspan=1>85.7%(30/35)(70.6%, 93.7%)</td><td rowspan=1 colspan=1>80.0% (76/95)(70.9%, 86.8%)</td><td rowspan=1 colspan=1>HN</td></tr><tr><td rowspan=1 colspan=1>100.0% (30/30)(88.6%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (30/30)(88.6%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (35/35)(90.1%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (95/95)(96.1%, 100.0%)</td><td rowspan=1 colspan=1>TN</td></tr></table>

Variance Component Analysis of Reproducibility Results on 2nd Generation BD MAX System   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between RunWitin Day</td><td rowspan=1 colspan=2>Between DayWithinSite</td><td rowspan=1 colspan=2>BetweenSites</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=13>GBS: Variance Component Analysis Positive Results</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.11</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>3.5%</td></tr><tr><td rowspan=1 colspan=13>IPC: Variance Component Analysis Negative Results</td></tr><tr><td rowspan=1 colspan=1>HN(1:50)</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>2.0%</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>2.8%</td></tr></table>

Reproducibility Summary for 1st and 2nd Generation BD MAX Systems   

<table><tr><td rowspan=2 colspan=1>Category</td><td rowspan=1 colspan=4>1st Generation</td><td rowspan=1 colspan=4>2nd Generation</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>84</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>85</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>2.61</td><td rowspan=1 colspan=1>8.7</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>3.5</td></tr><tr><td rowspan=1 colspan=1>HN</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>4.24</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>2.39</td><td rowspan=1 colspan=1>7.1</td></tr></table>

# In-house Precision Study

A precision study was performed on three 2nd generation (6-channel) BD MAX systems at an internal BD site. Two runs were performed per day on each system over 12 days for a total of 72 runs and 1440 samples tested. The precision panel was prepared using Streptococcus agalactiae (ATCC strain 27956) spiked into pooled negative Lim broth matrix. Panel member composition was masked to the operator according to a designated randomization scheme. Each run included testing of one panel which consisted of 20 samples; four replicates each of the five concentrations listed below. One external positive and one external negative control as well as two blank samples were included in each run. The Precision panel Composition and precision study results are provided in the following tables.

# Within Laboratory Precision Panel Composition

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Target concentration - CFU/assay</td><td rowspan=1 colspan=1>Replicates</td></tr><tr><td rowspan=1 colspan=1>High Negative-1 (HN-1)</td><td rowspan=1 colspan=1>~ 2.2(~ 1:100 dilution of LoD )</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>High Negative-2 (HN-2)</td><td rowspan=1 colspan=1>~22( ~ 1:10 dilution of LoD )</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LP)</td><td rowspan=1 colspan=1>~ 500(~ 1.5 x LoD)</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (MP)</td><td rowspan=1 colspan=1>~ 1000(~3 xLoD)</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>True Negative (TN)</td><td rowspan=1 colspan=1>0(No Target)</td><td rowspan=1 colspan=1>4</td></tr></table>

Qualitative GBS Precision Study Results Percent Agreement and Confidence Intervals (CI)   

<table><tr><td rowspan=1 colspan=1>PanelLevel</td><td rowspan=1 colspan=1>PP0020</td><td rowspan=1 colspan=1>PP0021</td><td rowspan=1 colspan=1>PP0032</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>MPLP</td><td rowspan=1 colspan=1>100.0% (96/96)(96.2%, 100.0%)94.8% (91/96)(88.4%, 97.8%)</td><td rowspan=1 colspan=1>100.0% (93/93)(96.0%, 100.0%)100.0% (95/95)(96.1%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (94/94) (96.1%,100.0%)99.0% (95/96) (94.3%,99.8%)</td><td rowspan=1 colspan=1>100.0% (283/283)(98.7%, 100.0%)97.9% (281/287)(95.5%, 99.0%)</td></tr><tr><td rowspan=1 colspan=1>HN-2(1:10)</td><td rowspan=1 colspan=1>70.5% (67/95)(60.7%, 78.8%)</td><td rowspan=1 colspan=1>79.2% (76/96) (70.0%,86.1%)</td><td rowspan=1 colspan=1>81.1% (77/95) (72.0%,87.7%)</td><td rowspan=1 colspan=1>76.9% (220/286)(71.7%, 81.4%)</td></tr><tr><td rowspan=1 colspan=1>HN-1(1:100)</td><td rowspan=1 colspan=1>94.8% (91/96)(88.4%, 97.8%)</td><td rowspan=1 colspan=1>98.9% (93/94) (94.2%,99.8%)</td><td rowspan=1 colspan=1>96.8% (90/93) (90.9%,98.9%)</td><td rowspan=1 colspan=1>96.8% (274/283)(94.1%, 98.3%)</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100.0% (96/96)(96.2%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (96/96)(96.2%, 100.0%)</td><td rowspan=1 colspan=1>100.0% (93/93) (96.0%,100%</td><td rowspan=1 colspan=1>100.0% (285/285)(98.7%, 100.0%)</td></tr></table>

GBS Precision SD Ratio of Old System and New System for Target (Ct) of GBS Assay Positive Samples   

<table><tr><td rowspan=2 colspan=1>PanelLevel</td><td rowspan=1 colspan=4>Old System</td><td rowspan=1 colspan=4>New System</td><td rowspan=2 colspan=1>SD Ratio</td><td rowspan=2 colspan=1>P-Value</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>277</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.82(0.73,0.92)</td><td rowspan=1 colspan=1>0.0004</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>281</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>0.54(0.48,0.61)</td><td rowspan=1 colspan=1>0.0000</td></tr><tr><td rowspan=1 colspan=1>HN-2(1:10)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>1.30(0.88,1.80)</td><td rowspan=1 colspan=1>0.0884</td></tr><tr><td rowspan=1 colspan=1>HN-1(1:100)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>2.19</td><td rowspan=1 colspan=1>7.2</td><td rowspan=1 colspan=1>7.48(0.24,20.58)</td><td rowspan=1 colspan=1>0.1031</td></tr></table>

Variance Component Analysis of Precision Results on 2nd Generation BD MAX System   

<table><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Within RunWithin DayWithinInstrument</td><td rowspan=1 colspan=2>Between RunWithin Day</td><td rowspan=1 colspan=2>Between DayWithinInstrument</td><td rowspan=1 colspan=2>BetweenInstruments</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Level</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=6>GBS: Variance Component Analysis Positive Results</td><td rowspan=1 colspan=4></td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>1.8 %</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.8 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0 %</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>2.1 %</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>281</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.8 %</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.02</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.9 %</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>2.1 %</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=11>IPC: Variance Component Analysis Negative Results</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HN-2(1:10)</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.3 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0 %</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.2 %</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.9 %</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>HN-1(1:100)</td><td rowspan=1 colspan=1>274</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>2.0 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0 %</td><td rowspan=1 colspan=1>0.04</td><td rowspan=1 colspan=1>0.2 %</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.6 %</td><td rowspan=1 colspan=1>2.19</td><td rowspan=1 colspan=1>2.1 %</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>285</td><td rowspan=1 colspan=1>27.3</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>1.6 %</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>0.8 %</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0 %</td><td rowspan=1 colspan=1>0.14</td><td rowspan=1 colspan=1>0.5 %</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.8 %</td></tr></table>

Precision Summary for 1st and 2nd Generation BD MAX Systems   

<table><tr><td rowspan=2 colspan=1>PanelLevel</td><td rowspan=1 colspan=4>1st Generation</td><td rowspan=1 colspan=4>2nd Generation</td></tr><tr><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>277</td><td rowspan=1 colspan=1>28.7</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>283</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>4.0</td><td rowspan=1 colspan=1>281</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>2.1</td></tr><tr><td rowspan=1 colspan=1>HN-2(1:10)</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>3.0</td></tr><tr><td rowspan=1 colspan=1>HN-1 (1:100)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>2.19</td><td rowspan=1 colspan=1>7.2</td></tr></table>

b. Linearity/assay reportable range:

Not applicable. The BD MAX GBS Assay is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

External controls are not provided by the manufacturer. Commercially available control material may be used or $\mathtt { a } \ge 1 8$ hour GBS culture in Lim Broth may be utilized as positive control material. GBS ATCC BAA-22 is a recommended strain to use for a positive control. An uninoculated GBS Sample Preparation Reagent tube or a 15 ul aliquot of pure Lim Broth is recommended for use as an external negative control. External positive and negative controls were included in all analytical and clinical studies performed in support of this submission. The External positive control was prepared by diluting a GBS pellet to a concentration of 1500 CFU/assay. The External negative control consisted of negative clinical matrix in Sample Preparation Buffer without GBS.

An Internal Process Control (IPC) provided in each BD MAX GBS test. The IPC is extracted, amplified and detected along with each specimen tested and monitors the efficacy of the DNA extraction and PCR amplification processes.

# d. Detection limit:

A study was performed to confirm that the BD MAX GBS assay as performed on the 2nd generation (6-channel) BD MAX instrument was able to achieve the LoD demonstrated by the 1rst generation (2-channel) BD MAX instrument $( 2 0 0 \mathrm { C F U / m L }$ Sample Preparation Reagent or 300 CFU/assay). GBS suspensions at $2 0 0 \ : \mathrm { C F U / m L }$ and 165 CFU/mL of Sample Preparation Reagent were prepared in negative clinical matrix with 64 replicates, resulting in respective detection rates of $100 \%$ and $9 8 \%$ . An LoD study was performed with a second GBS strain, resulting in an LoD of 160 CFU/mL Sample Preparation. All LoD testing confirmed that the LoD for the BD MAX GBS Assay on the 2nd generation BD MAX is comparable to the LoD for the assay on the 1rst generation system.

# e. Analytical Inclusivity:

A total of 12 strains including 11 different serotypes and one non-hemolytic strain of GBS were tested to demonstrate the ability of the BD MAX GBS assay to detect clinically relevant variants. Eight samples per strain were prepared in Sample

Preparation Reagent and negative clinical matrix. The BD MAX GBS Assay run on the 2nd Generation BD MAX System was able to detect all major serotypes of GBS at $3 0 0 \mathrm { C F U / m l }$ and $9 0 0 \mathrm { C F U / m l }$ Sample Preparation Reagent.

GBS Variants Tested   

<table><tr><td rowspan=1 colspan=1>GBS serotype</td><td rowspan=1 colspan=1>Source</td></tr><tr><td rowspan=1 colspan=1>Ia</td><td rowspan=1 colspan=1>ATCC 12400</td></tr><tr><td rowspan=1 colspan=1>Ib</td><td rowspan=1 colspan=1>NCS1&#x27;, blood</td></tr><tr><td rowspan=1 colspan=1>Ic</td><td rowspan=1 colspan=1>ATCC 27591</td></tr><tr><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>ATCC 12973</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>ATCC 12403</td></tr><tr><td rowspan=1 colspan=1>IV</td><td rowspan=1 colspan=1>ATCC 49446</td></tr><tr><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>ATCC BAA-611</td></tr><tr><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>NCS&#x27;, Placenta</td></tr><tr><td rowspan=1 colspan=1>VII</td><td rowspan=1 colspan=1>NCS1, blood</td></tr><tr><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>Clinical Isolate</td></tr><tr><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>ATCC BAA-22</td></tr><tr><td rowspan=1 colspan=1>Non-hemolytic strain</td><td rowspan=1 colspan=1>ATCC 13813</td></tr></table>

1National Centre for Streptococcus, Alberta, Canada

# f. Analytical specificity:

Cross Reactivity: A total of 128 non-target specimens that include commensal organisms and pathogens of the urogenital and digestive tract, species phylogenetically related to $S .$ agalactiae, and human DNA were tested in this study and are listed in the table below. The following concentrations of non-target organisms were tested: bacterial and fungal organisms at ${ \sim } 1 0 ^ { 6 } \mathrm { C F U / m L }$ of Sample Preparation Reagent, viral organisms at ${ > } 2 \mathrm { x } 1 0 ^ {  { \overleftarrow } . 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ of Sample Preparation Reagent, and DNA stocks at $\scriptstyle \sim 3 \mathrm { ~ n g / m L }$ of Sample Preparation Reagent. Three replicates of each potential cross-reactant were tested with the BD MAX GBS assay on the 2nd generation BD Max System.

<table><tr><td rowspan=1 colspan=3>Organisms</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Kingella denitrificans</td><td rowspan=1 colspan=1>Salmonella enterica</td></tr><tr><td rowspan=1 colspan=1>Aerococcus viridans</td><td rowspan=1 colspan=1>Kingella kingae</td><td rowspan=1 colspan=1>Salmonella enterica Minn</td></tr><tr><td rowspan=1 colspan=1>Aeromonas hydrophila</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Salmonella enterica typhi</td></tr><tr><td rowspan=1 colspan=1>Alcaligenes faecalis</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Salmonella newport</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Salmonella typhimurium</td></tr><tr><td rowspan=1 colspan=1>Bacillus subtilis</td><td rowspan=1 colspan=1>Lactobacillus brevis</td><td rowspan=1 colspan=1>Serratia marcescans</td></tr><tr><td rowspan=1 colspan=1>Bacteroides fragilis</td><td rowspan=1 colspan=1>Lactobacillus casei</td><td rowspan=1 colspan=1>Shigella flexneri</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium adolescentis</td><td rowspan=1 colspan=1>Lactobacillus delbreuckii</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Bifidobacterium breve</td><td rowspan=1 colspan=1>Lactobacillus jensenii</td><td rowspan=1 colspan=1>Staphylococcus aureus</td></tr><tr><td rowspan=1 colspan=1>Brevibacterium linens</td><td rowspan=1 colspan=1>Lactobacillus lactis</td><td rowspan=1 colspan=1>Staphylococcus epidermidis</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Staphylococcus spp</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>Streptococcus anginosus</td></tr><tr><td rowspan=1 colspan=1>Candida krusei</td><td rowspan=1 colspan=1>Mobiluncus mulieris</td><td rowspan=1 colspan=1>Streptococcus bovis</td></tr><tr><td rowspan=1 colspan=1>Candida parapsilosis</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Moraxella lacunata</td><td rowspan=1 colspan=1>Streptococcus intermedius</td></tr><tr><td rowspan=1 colspan=1>Chromobacterium violaceum</td><td rowspan=1 colspan=1>Moraxella osloensis</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Clostridium perfringens</td><td rowspan=1 colspan=1>Myobacterium smegmatis</td><td rowspan=1 colspan=1>Streptococcus oralis</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium xerosis</td><td rowspan=1 colspan=1>Neisseria flava</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Neisseria flavescens</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium spp</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium urealyticum</td><td rowspan=1 colspan=1>Neisseria lactamica</td><td rowspan=1 colspan=1>Streptococcus sanguinis</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Neisseria meningitidis 158</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td></tr><tr><td rowspan=1 colspan=1>Eikenella corrodens</td><td rowspan=1 colspan=1>Neisseria meningitidis A</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Neisseria meningitidis B</td><td rowspan=1 colspan=1>Achromobacter xerosis</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Neisseria meningitides M1883</td><td rowspan=1 colspan=1>Deinococcus radiodurans</td></tr><tr><td rowspan=1 colspan=1>Enterococcus avium</td><td rowspan=1 colspan=1>Neisseria perflava</td><td rowspan=1 colspan=1>Derxia gummosa</td></tr><tr><td rowspan=1 colspan=1>Enterococcus dispar</td><td rowspan=1 colspan=1>Peptostreptococcus anaerobius</td><td rowspan=1 colspan=1>Mycoplasma genitalium</td></tr><tr><td rowspan=1 colspan=1>Enterococcus durans</td><td rowspan=1 colspan=1>Plesiomonas shigelloides</td><td rowspan=1 colspan=1>Mycoplasma hominis</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Propionibacterium acnes</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>Streptomyces griseus</td></tr><tr><td rowspan=1 colspan=1>Enterococcus spp</td><td rowspan=1 colspan=1>Proteus vulgaris</td><td rowspan=1 colspan=1>Ureaplasma urealyticum</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Providencia stuartii</td><td rowspan=1 colspan=1>Chlamydia pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Chlamydia trachomatis</td></tr><tr><td rowspan=1 colspan=1>Gemella haemolysans</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td><td rowspan=1 colspan=1>Rhodospirillum rubrum</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenza</td><td rowspan=1 colspan=1>Pseudomonas putida</td><td rowspan=1 colspan=1>Trichomonas vaginalis</td></tr><tr><td rowspan=1 colspan=1>Hemophilus ducreyi</td><td rowspan=1 colspan=1>Rahnella aquatilis</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Hemophilus influenzae type B</td><td rowspan=1 colspan=1>Saccharomyces cerevisiae</td><td rowspan=1 colspan=1></td></tr></table>

Nine of the organisms tested, Aerococcus viridans (2/3), Candida albicans (1/3), Deinococcus radiodurans (1/3), Enterococcus durans (1/3), Lactobacillus jensenii (3/3), Proteus vulgaris (1/3), Providencia stuartii (1/3), Pseudomonas aeruginosa (1/3), and Streptococcus pyogenes (1/3) initially gave positive results. One replicate of human DNA gave a positive result.

An expanded study was conducted in which twenty replicates of each of the potential cross-reactants were tested on the 2nd Generation BD MAX System. No reactivity was observed with the C. albicans, $D .$ . radiodurans, $L .$ . jensenii, S. pyogenes or human DNA samples. Five organisms continued to demonstrate cross-reactivity.

<table><tr><td rowspan=1 colspan=1>Non-Target Organism</td><td rowspan=1 colspan=1>No. Positive (n=20)</td></tr><tr><td rowspan=1 colspan=1>A. viridans</td><td rowspan=1 colspan=1>1/20</td></tr><tr><td rowspan=1 colspan=1>E. durans</td><td rowspan=1 colspan=1>1/20</td></tr><tr><td rowspan=1 colspan=1>*P. aeruginosa</td><td rowspan=1 colspan=1>1/20</td></tr><tr><td rowspan=1 colspan=1>*P. stuartii</td><td rowspan=1 colspan=1>2/20</td></tr><tr><td rowspan=1 colspan=1>*P. vulgaris</td><td rowspan=1 colspan=1>4/20</td></tr></table>

\* The organisms indicated with an asterisk are gram negative. Lim broth enrichment is designed to suppress growth of gram negative organisms.

# g. Interference with Non-target Organisms:

A study was conducted to determine the effect of non-target organisms present in a clinical specimen on the detection of GBS at low concentrations. Testing included the 127 non-target organisms tested in the cross-reactivity study as well as human DNA. Samples were prepared in negative clinical Lim Broth matrix at high concentration levels of the potential interferent mixed with suspensions of GBS at concentrations near the LoD. Interference was initially observed with Achromobacter xerosis, Haemophilus influenza and Enterobacter cloacae. For these three organisms, an expanded study was performed which included additional testing of 20 replicates. No interference was observed with A. xerosis and $H .$ influenzae in this additional testing. Interference (2/20 replicates) was observed in the presence of $E$ . cloacae when tested with a GBS target concentration of $3 0 0 \mathrm { C F U / m L }$ of Sample Preparation Reagent.

h. Interference with Exogenous and Endogenous Substances

Potential exogenous and endogenous interfering substances that may be present in clinical vaginal/rectal specimens were tested in an interference study and are listed in the tables below. Samples were prepared in negative clinical Lim Broth matrix with low concentrations of GBS and each potential interfering substance. To prepare the spermicidal insert and the rectal suppository for evaluation, one insert or suppository was added to $5 \mathrm { m L }$ of non-clinical LIM broth. The remaining samples were prepared by placing a swab of the exogenous interfering agent into $1 \mathrm { m L }$ of negative clinical matrix. Endogenous agents were tested in a negative clinical Lim Broth matrix at the following concentrations: blood $( 1 0 \% )$ , amniotic fluid $( 1 0 \% )$ , urine $(4 3 \% )$ , human DNA (1550 ng/assay), feces (small amount of feces on a swab in $1 ~ \mathrm { m l }$ of negative

matrix, mucus $1 5 \%$ .) In all cases, the BD MAX GBS Assay detected GBS at concentrations of $3 0 0 \mathrm { C F U / m L }$ and 3000 CFU/mL of Sample Preparation Reagent in the presence of the endogenous and exogenous substances tested.

Agents tested for potential interference with the BD MAX™ GBS assay   

<table><tr><td colspan="2" rowspan="1">Description</td><td colspan="1" rowspan="1">Active Ingredients(or inactive if no active listed)</td></tr><tr><td colspan="1" rowspan="13">ExogenousAgents</td><td colspan="1" rowspan="1">Miconazole</td><td colspan="1" rowspan="1">4% Miconazole nitrate</td></tr><tr><td colspan="1" rowspan="1">Hemorrhoidal Cooling gel</td><td colspan="1" rowspan="1">Phenyephrine HCl (0.25%)Witch Hazel, 50%</td></tr><tr><td colspan="1" rowspan="1">Spermicidal</td><td colspan="1" rowspan="1">Nonoxynol-9 (100 mg)</td></tr><tr><td colspan="1" rowspan="1">Contraceptive jelly</td><td colspan="1" rowspan="1">2% Nonoxynol-9 (100 mg)</td></tr><tr><td colspan="1" rowspan="1">Contraceptive gel</td><td colspan="1" rowspan="1">Nonoxynol-9 (100 mg)</td></tr><tr><td colspan="1" rowspan="1">Feminine Deodorant spray</td><td colspan="1" rowspan="1">Isobutane; Isopropyl Myristate; Zea Mays(Corn) Starch; Magnesium Stearate;Chamomilla Recutita (Matricaria) Extract;Tocopherol; Mineral Oil (ParaffinumLiquidum); Sodium Bicarbonate; ZincRicinoleate; Lanolin Alcohol; Oleyl Alcohol;Aloe Barbadensis Leaf Extract; BenzylAlcohol; Laureth-3; TetrahydroxypropylEthylenediamine; Propylene Glycol; Fragrance(Parfum)</td></tr><tr><td colspan="1" rowspan="1">Feminine Deodorant spray</td><td colspan="1" rowspan="1">Isobutane; Isopropyl Myristate; Zea Mays(Corn) Starch; Neutresse® (Ordenone®),Magnesium Stearate, Hydrated Silica,Fragrance, Mineral Oil, Lanolin, Alcohol,Benzyl Alcohol, Sodium Bicarbonate,Tocopherol</td></tr><tr><td colspan="1" rowspan="1">Lubricating gel</td><td colspan="1" rowspan="1">Chlorhexidine Gluconate and MethylHydroxybenzoate; Hydroxyethylcellulose</td></tr><tr><td colspan="1" rowspan="1">Moisturizing lotion</td><td colspan="1" rowspan="1">Mineral oil; Isopropyl Myristate; PEG-40;sorbitan; peroleate; others</td></tr><tr><td colspan="1" rowspan="1">Body oil</td><td colspan="1" rowspan="1">Isopropyl Myristate; sesame oil; PEG-40;sorbitan; peroleate; others</td></tr><tr><td colspan="1" rowspan="1">Body powder</td><td colspan="1" rowspan="1">Corn starch, Sodium bicarbonate, MagnesiumStearate, Mineral oil, Silica, Benzethoniumchloride, Fragrance, Aloe barbadensis gel</td></tr><tr><td colspan="1" rowspan="1">Body powder</td><td colspan="1" rowspan="1">Talc, Parfum</td></tr><tr><td colspan="1" rowspan="1">Laxative suppository</td><td colspan="1" rowspan="1">Bisacodyl USP (10 mg)</td></tr><tr><td colspan="1" rowspan="6">EndogenousAgents</td><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="6">N/A</td></tr><tr><td colspan="1" rowspan="1">Whole blood (ClinicalSpecimen)</td></tr><tr><td colspan="1" rowspan="1">Urine (Clinical Specimen)</td></tr><tr><td colspan="1" rowspan="1">Mucus (Bovine Powder)</td></tr><tr><td colspan="1" rowspan="1">Feces (Clinical Specimen)</td></tr><tr><td colspan="1" rowspan="1">Amniotic fluid (ClinicalSpecimen)</td></tr></table>

# i. Carry-Over and Cross Contamination Studies

Within run carry-over and cross-contamination using the BD MAX GBS Assay on the 2nd Generation BD MAX System was evaluated by testing five consecutive runs on the same instrument in which high positive and true negative samples were loaded in alternating positions. No false positives or false negative results were observed in this study.

Between run carry-over and cross-contamination using the BD MAX GBS Assay on the 2nd Generation BD MAX System was evaluated by testing five consecutive runs on the same instrument. Alternating runs consisted of all high positive samples and all true negative samples. No false positives or negative results were observed in this study.

Four runs of top/bottom PCR row testing were performed which included testing of the TOP row of a cartridge with a following run of testing of the BOTTOM row of the same cartridge. This study demonstrated no cross-contamination between cartridge rows used in successive runs.

2. Comparison studies:

# a. Method comparison with predicate device:

The performance of the BD MAX GBS Assay on the 2nd Generation BD MAX System was evaluated in a comparison study performed at three testing sites, two external and one internal. Testing consisted of 214 residual clinical Lim Broth specimens with approximately equal numbers of positive and negatives. The Lim Broth specimens were obtained from five clinical laboratories that had inoculated each vaginal-rectal swab specimen in Lim Broth and then incubated overnight for ${ > } 1 8$ hours. An aliquot of each residual specimen was tested at two external sites and internally at BD on 2nd Generation BD MAX systems. An aliquot of each specimen was also tested internally at BD on three 1rst Generation BD MAX systems. Results from three (3) New Systems (A, B, and C below) were evaluated and compared to results generated from the same clinical panel tested on three (3) Old Systems (D, E, and F below.) The GBS status of each sample was determined by the result generated by the 1rst Generation BD MAX System. In the event of discordant or IND results, the result generated by two of the three 1rst Generation instruments determined the GBS status.

# Decision Algorithm for 1rst Generation System Testing

The following results matrix was used to determine the final result status for each of the specimens in the Comparison Panel:

<table><tr><td rowspan=1 colspan=3>Individual Old System Result</td><td rowspan=2 colspan=1>Final Old SystemResult</td></tr><tr><td rowspan=1 colspan=1>Old System (D)</td><td rowspan=1 colspan=1>Old System (E)</td><td rowspan=1 colspan=1>Old System (F)</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td></tr><tr><td rowspan=1 colspan=3>Any combination of 2 POS and 1 NEG or IND</td><td rowspan=1 colspan=1>POS</td></tr><tr><td rowspan=1 colspan=3>Any combination of 2 NEG and 1 POS or IND</td><td rowspan=1 colspan=1>NEG</td></tr></table>

Internal and external controls were included in the study to monitor assay performance. The performance of the BD MAX GBS assay performed on the 2nd Generation BD MAX system as compared to the 1rst Generation BD MAX system is summarized in the tables below:

Descriptive Statistics for 1rst Generation System Target Ct of GBS Positive Results   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Range (Min-Max)</td></tr><tr><td rowspan=1 colspan=1>Site D</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>17.31</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>(14.28 - 24.27)</td></tr><tr><td rowspan=1 colspan=1>Site E</td><td rowspan=1 colspan=1>109</td><td rowspan=1 colspan=1>16.93</td><td rowspan=1 colspan=1>2.03</td><td rowspan=1 colspan=1>(13.72  23.94)</td></tr><tr><td rowspan=1 colspan=1>Site F</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>17.11</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>(12.3 - 28.59)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>329</td><td rowspan=1 colspan=1>17.12</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>(12.3 - 28.59)</td></tr></table>

Descriptive Statistics for 2nd Generation System Target Ct of GBS Positive Results   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Range (Min-Max)</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>18.23</td><td rowspan=1 colspan=1>2.84</td><td rowspan=1 colspan=1>(14.32-32.17)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>17.50</td><td rowspan=1 colspan=1>2.61</td><td rowspan=1 colspan=1>(13.47 - 35.94)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>17.13</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>(14.31-24.01)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>333</td><td rowspan=1 colspan=1>17.63</td><td rowspan=1 colspan=1>2.52</td><td rowspan=1 colspan=1>(13.47 - 35.94)</td></tr></table>

Positive Percent Agreement (PPA) and Negative Percent Agreement (NPA) by Site   

<table><tr><td rowspan=2 colspan=2>Site A</td><td rowspan=1 colspan=2>Old System</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>New System</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>112</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (96.6%,100%)NPA: 98.1% (93.3%, 99.5%)</td></tr><tr><td rowspan=2 colspan=2>Site B</td><td rowspan=1 colspan=1>Old S</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>New System</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>111</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>103</td><td rowspan=1 colspan=1>103</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (96.6%, 100%)NPA: 99% (94.8%, 99.8%)</td></tr><tr><td rowspan=2 colspan=2>Site C</td><td rowspan=1 colspan=1>Old System</td><td rowspan=1 colspan=1>Old System</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>New System</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>110</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>104</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>104</td><td rowspan=1 colspan=1>214</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (96.6%, 100%)NPA: 100% (96.4%, 100%)</td></tr></table>

1 Discrepant repeat testing was performed on samples GA0107 and GA0226 in duplicate. Both replicates for each sample produced negative results. 2 Discrepant repeat testing was performed on sample GB0159 in duplicate. Both replicates produced negative result

Positive Percent Agreement and Negative Percent Agreement for all Sites Combined   

<table><tr><td rowspan=2 colspan=2>Overall</td><td rowspan=1 colspan=2>Old System</td><td rowspan=2 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>New System</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>333</td></tr><tr><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>309</td><td rowspan=1 colspan=1>309</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>312</td><td rowspan=1 colspan=1>642</td></tr><tr><td rowspan=1 colspan=5>PPA: 100% (100.0%, 100.0%)NPA: 99% (97.8%, 100.0%)</td></tr></table>

Positive Percent Agreement and Negative Percent Agreement with $9 5 \%$ - Confidence Interval   

<table><tr><td>Site</td><td colspan="2">Positive Percent Agreement</td><td colspan="2">Negative Percent Agreement</td></tr><tr><td>Site A</td><td>100.0% (110/110)</td><td>(96.6% , 100.0%)</td><td>98.1% (102/104)</td><td>(93.3% , 99.5%)</td></tr><tr><td>Site B</td><td>100.0% (110/110)</td><td>(96.6% , 100.0%)</td><td>99.0% (103/104)</td><td>(94.8% , 99.8%)</td></tr><tr><td>Site C</td><td>100.0% (110/110)</td><td>(96.6% , 100.0%)</td><td>100.0% (104/104)</td><td>(96.4% , 100.0%)</td></tr><tr><td>Combined</td><td>100.0% (330/330)</td><td>(100.0%, 100.0%)</td><td>99.0% (309/312)</td><td>(97.8%, 100.0%)</td></tr></table>

Numerators are results from New System (Test) and denominators are results from Old System (Reference) Each site $9 5 \%$ CI by score method and combined sites $9 5 \%$ CI by bootstrap approach.

System Initial and Final Indeterminate (IND) Rates on the 2nd Generation BD MAX System   

<table><tr><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>Initial Unresolved Rates with95% Confidence Intervals</td><td rowspan=1 colspan=2>Final Unresolved Rates with95% Confidence Intervals</td></tr><tr><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>3.7% (8/214)</td><td rowspan=1 colspan=1>(1.9%, 7.2%)</td><td rowspan=1 colspan=1>0.0% (0/214)</td><td rowspan=1 colspan=1>(0.0%, 1.8%)</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>2.8% (6/214)</td><td rowspan=1 colspan=1>(1.3%, 6.0%)</td><td rowspan=1 colspan=1>0.0% (0/214)</td><td rowspan=1 colspan=1>(0.0%, 1.8%)</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>4.2% (9/214)</td><td rowspan=1 colspan=1>(2.2%, 7.8%)</td><td rowspan=1 colspan=1>0.0% (0/214)</td><td rowspan=1 colspan=1>(0.0%, 1.8%)</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>3.6% (23/642)</td><td rowspan=1 colspan=1>(2.2%, 5.3%)</td><td rowspan=1 colspan=1>0.0% (0/642)</td><td rowspan=1 colspan=1>(0.0%, 0.6%)</td></tr><tr><td rowspan=1 colspan=5>Each site 95% Cl by score method and combined sites 95% Cl by bootstrap approach.</td></tr></table>

b. Matrix comparison: Not applicable

3. Clinical studies: Not Applicable

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

Approximately $2 5 { - } 4 0 \%$ of pregnant women are colonized with GBS. Culture screening of both the vagina and rectum for GBS in late gestation, during prenatal care, can detect women who are likely to be colonized with GBS at the time of delivery. In the investigational study for the BD MAX GBS Assay on the 1st Generation BD MAX System (K090191), the overall GBS prevalence rate as determined by culture was $2 3 . 0 \%$ (143/623) with a $9 5 \ \%$ CI of $1 9 . 7 - 2 6 . 5 \%$ .

N. Instrument Name: BD MAX System

# O. System Descriptions:

1. Modes of Operation:

The 2nd generation BD MAX system fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

3. Specimen Identification:

Barcodes are used to identify patient specimens. The specimen rack is also has a barcode to identify both specimen and assay.

4. Specimen Sampling and Handling:

An aliquot of an overnight Lim Broth culture is manually added to a BD MAX GBS Sample Preparation Reagent tube. The tube is then inserted into the DNA Unitized Reagent Strip and placed on the BD MAX instrument. All further specimen handling is automated.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control:

An Internal Process Control (IPC) provided in each BD MAX GBS test. The IPC is extracted, amplified and detected along with each specimen tested and monitors the efficacy of the DNA extraction and PCR amplification processes.

External controls are not provided by the manufacturer. Commercially available control material may be used or $\mathtt { a } \ge 1 8$ hour GBS culture in Lim Broth may be utilized as positive control material. GBS ATCC BAA-22 is a recommended strain to use for a positive control. An uninoculated GBS Sample Preparation Reagent tube or a $1 5 \mathrm { u l }$ aliquot of pure Lim Broth is recommended for use as an external negative control.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.