# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: K111387 B. Purpose for Submission:

The purpose of this submission is evaluation of a new assay (Liat™ Influenza A/B Assay) for the detection of Influenza A and Influenza B performed on a new point-of-care instrument (Liat™ Analyzer).

# C. Measurand:

The Liat Influenza A/B Assay is a rapid, automated in vitro diagnostic test for qualitative detection of Influenza A and Influenza B from nasopharyngeal (NP) swab specimens from patients with signs and symptoms of respiratory infection.

# D. Type of Test:

Multiplex nucleic acid assay for qualitative detection of Influenza A and Influenza B from NP specimens including nucleic acid isolation and multiplex real-time RT-PCR amplification using the Liat Analyzer.

# E. Applicant:

IQuum, Inc.

F. Proprietary and Established Names:

Liat™ Influenza A/B Assay, Liat™ Analyzer

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code: OCC, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IQuum Liat™ Influenza A/B Assay performed on the Liat™ Analyzer is an automated multiplex real-time RT-PCR assay for the rapid in vitro qualitative detection and discrimination of influenza A virus and influenza B virus RNA in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. The test is intended for use as an aid in the differential diagnosis of influenza A and influenza B in humans and is not intended to detect influenza C.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease.

Performance characteristics for influenza A were established when influenza A/H1 and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Requires the Liat Analyzer (Software version 1.0)

# I. Device Description:

The Liat™ Influenza A/B Assay targets a well-conserved region of the matrix gene of Influenza A viral RNA (Inf A target) and non-structural protein gene of Influenza B (Inf B target). An Internal Process Control (IPC) is also included. The IPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the RTPCR reactions.

The Liat Influenza A/B Assay is performed on the lab-in-a-tube technology platform. The system consists of a disposable Liat Influenza A/B Assay Tube and the Liat™ Analyzer. The Liat™ Tube uses a flexible tube as a sample vessel. It contains all required unit dose reagents pre-packed in tube segments, separated by peelable seals, in the order of reagent use. The Liat Analyzer performs all assay steps from clinical sample and reports assay result automatically. During the testing process, multiple sample processing actuators of the analyzer compress the Liat Tube to selectively release reagents from tube segments, move the sample from one segment to another, and control reaction volume, temperature, and time to conduct sample preparation, nucleic acid extraction, target enrichment, inhibitor removal, nucleic acid elution and real-time RT-PCR. An embedded microprocessor controls and coordinates the actions of these sample processors to perform all required assay processes within the closed Liat Tube. Turnaround time for analysis of a sample is ${ \sim } 2 0$ minutes.

Positive control is provided in the Liat Influenza A/B Assay Quality Control Kit. The positive control comprises a pooled sample of inactivated Influenza A and B virus. Negative control is provided in the Liat Influenza A/B Assay Quality Control Kit. The negative control comprises universal transport medium (UTM). The solution is provided in unit dose quantity and labeled as Dilution UTM. To use the negative control, an operator transfers the entire contents of the Dilution UTM tube into the Liat tube using a transfer pipette and runs the assay following the Instructions for Use.

The results are interpolated by the Liat Analyzer software from measured fluorescent signals and real time curve recognition algorithm. All possible final test results are described and depicted below.

1. Influenza A Not Detected, Influenza B Not Detected

Inf A target and Inf B target are not detected. Internal process control (IPC) has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of the IPC fits the pattern recognition setting.

# IQM-RESULT

[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1231   
Time/Date: 15:39:48, 2011-03-24   
Report Results: Influenza A Not Detected Influenza B Not Detected   
Details:   
Run status: OK   
Device S/N: M1-D-00082   
SW Ver: H   
Run No. : 1129   
Tube S/N: 13JA997   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

2. Influenza A Detected, Influenza B Not Detected

The Inf A target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf A target fits the pattern recognition setting. • Inf B target is not detected. High concentration of Inf A target may inhibit the PCR of the IPC. The IPC will not be taken into consideration in these cases.

# IQM-RESULT

[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1232   
Time/Date: 15:59:38, 2011-03-24   
Report Results: Influenza A Detected Influenza B Not Detected   
Details:   
Run status: OK   
Device S/N: M1-D-00065   
SW Ver: H   
Run No. : 2129   
Tube S/N: 13JA995   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

3. Influenza A Not Detected, Influenza B Detected

# IQM-RESULT

Inf A target is not detected.   
The Inf B target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf B target fits the pattern recognition setting.   
High concentration of Inf B target may inhibit the PCR of IPC. The IPC will not be taken into consideration in these cases.   
[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1233   
Time/Date: 16:19:42, 2011-03-24   
Report Results: Influenza A Not Detected Influenza B Detected   
Details:   
Run status: OK   
Device S/N: M1-D-00054   
SW Ver: H   
Run No. : 896   
Tube S/N: 13JA053   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

4. Influenza A Detected, Influenza B Detected

# IQM-RESULT

The Inf A target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf A target fits the pattern recognition setting.   
The Inf B target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf B target fits the pattern recognition setting.   
High concentration of Inf A or Inf B targets may inhibit the PCR of the IPC. The IPC will not be taken into   
consideration in these cases.   
[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1234   
Time/Date: 16:39:38, 2011-03-24   
Report Results: Influenza A Detected Influenza B Detected   
Details:   
Run status: OK   
Device S/N: M1-D-00034   
SW Ver: H   
Run No. : 2658   
Tube S/N: 13JA996   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

5. Influenza A Detected, Influenza B Indeterminate

The Inf A target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf A target fits the pattern recognition setting. • Presence or absence of Influenza B cannot be determined. The Inf B target has a Ct within the valid range and fluorescence endpoint above the minimum setting. However, the PCR curve of Inf B target does NOT fit the pattern recognition setting. High concentration of Inf A or Inf B targets may inhibit the PCR of the IPC. The IPC will not be taken into consideration in these cases. Requires retest to determine if Inf B is detected

# IQM-RESULT

[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1236   
Time/Date: 16:49:18, 2011-03-24   
Report Results: Influenza A Detected Influenza B Indeterminate   
Details:   
Pattern Code: B110   
Run status: OK   
Device S/N: M1-D-00030   
SW Ver: H   
Run No. : 2158   
Tube S/N: 13JA956   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

6. Influenza A Indeterminate, Influenza B Detected

Presence or absence of Influenza A cannot be determined. The Inf A target has a Ct within the valid range and fluorescence endpoint above the minimum setting. However, the PCR curve of Inf A target does NOT fit the pattern recognition setting. The Inf B target has a Ct within the valid range and fluorescence endpoint above the minimum setting. The PCR curve of Inf B target does NOT fits the pattern recognition setting. High concentration of Inf A or Inf B targets may inhibit the PCR of the IPC. The IPC will not be taken into consideration in these cases. Requires retest to determine if Inf A is detected

# IQM-RESULT

[Report]   
Assay: Liat Influenza Assay (FABA)   
Use: For in vitro Diagnostic Use   
Sample ID: 1236   
Time/Date: 15:29:23, 2011-03-24   
Report Results: Influenza A Indeterminate Influenza B Detected   
Details:   
Pattern Code: A62   
Run status: OK   
Device S/N: M1-D-00036   
SW Ver: H   
Run No. : 2489   
Tube S/N: 13JA731   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

7. Assay Invalid. Repeat Test.

# IQM-RESULT

Presence or absence of Influenza A and B cannot be determined.   
Inf A and Inf B targets are not detected or the PCR curves do not fit the pattern recognition setting.   
IPC Ct is not within the valid range, fluorescence endpoint is not above the minimum setting, or the PCR curve of IPC does not fit the pattern recognition setting. [Report]   
Assay: Liat Influenza Assay (FABA) Use: For in vitro Diagnostic Use Sample ID: 1237   
Time/Date: 15:10:08, 2011-03-24   
Report Results:   
Assay Invalid. Repeat test.   
Failure code: r0   
Details:   
Run status: OK   
Device S/N: M1-D-00042   
SW Ver: H   
Run No. : 2186   
Tube S/N: 13JA998   
Tube Lot: 13JA   
Tube Exp: 2011-09-30   
Ctrl Exp: N/A   
Operator: USER01   
Approved by: N/A

[Status] Completed OK

# Reasons to Repeat the Assay

If any of the test results mentioned below occur, repeat the test according to instructions in Starting Liat Assay section using a new Liat Tube (do not reuse Liat Tubes).

An “Assay Invalid. Repeat test.” result indicates that (1) the IPC was not detected when Inf A and Inf B were both not detected, or (2) all detected PCR curve(s) for Inf A and/or Inf B have abnormal PCR curve pattern. This may result from the sample not being properly processed or the RT-PCR being inhibited.   
• An “Error” is reported and the assay is aborted. Possible causes include temperature control, motion control failure and power failure.   
• An “Indeterminate” result is reported for one Influenza target and “Detected” for the other Influenza target. This results from an abnormal PCR curve pattern for the “Indeterminate” Influenza target. Retest is required to determine whether the “Indeterminate” Influenza target is detected.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Cepheid Xpert Flu Assay

# 2. Predicate 510(k) number(s): K103766

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">Device:Liat Influenza A/B</td><td colspan="1" rowspan="1">Predicate:Cepheid Xpert Flu</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The IQuum LiatTM Influenza A/BAssay performed on the LiatTMAnalyzer is an automated multiplexreal-time RT-PCR assay for the rapidin vitro qualitative detection anddiscrimination of influenza A virusand influenza B virus RNA innasopharyngeal swab specimens frompatients with signs and symptoms ofrespiratory infection in conjunctionwith clinical and epidemiological riskfactors. The test is intended for useas an aid in the differential diagnosisof influenza A and influenza B inhumans and is not intended to detectinfluenza C.Negative results do not precludeinfluenza virus infection and shouldnot be used as the sole basis fortreatment or other patientmanagement decisions. Conversely,positive results do not rule-outbacterial infection or co-infectionwith other viruses. The agent detectedmay not be the definite cause ofdisease.Performance characteristics forinfluenza A were established wheninfluenza A/H3 and A/H1 were thepredominant influenza A viruses incirculation. When other influenza Aviruses are emerging, performancecharacteristics may vary.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiologicalscreening criteria recommended by</td><td colspan="1" rowspan="1">The Cepheid® Xpert Flu Assay is anautomated, multiplex real-time RT-PCRassay intended for the in vitroqualitative detection and differentiationof influenza A, influenza B and 2009H1N1 influenza viral RNA. The XpertFlu Assay uses nasal aspirates/washesand nasopharyngeal swab specimenscollected from patients with signs andsymptoms of respiratory infection inconjunction with clinical andepidemiological risk factors. The XpertFlu Assay is intended as an aid in thediagnosis of influenza.Negative results do not precludeinfluenza virus infection and should notbe used as the sole basis for treatment orother patient management decisions.Performance characteristics forinfluenza A were established during the2009-2010 influenza season when 2009H1N1 influenza was the predominantinfluenza A virus in circulation. Whenother influenza A viruses are emerging,performance characteristics may vary.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentinfluenza viruses and sent to state orlocal health department for testing. Viralculture should not be attempted in thesecases unless a BSL 3+ facility isavailable to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">public health authorities, specimensshould be collected with appropriateinfection control precautions fornovel virulent Influenza viruses andsent to state or local healthdepartment for testing. Viral cultureshould not be attempted in these casesunless a BSL 3+ facility is availableto receive and culture specimens.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3980</td><td colspan="1" rowspan="1">21 CFR 866.3332</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OCC, 001</td><td colspan="1" rowspan="1">OQW, OCC, 001</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">Influenza AInfluenza B</td><td colspan="1" rowspan="1">Influenza AInfluenza BInfluenza A subtype 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasopharyngeal Swab</td><td colspan="1" rowspan="1">Nasopharyngeal SwabNasal aspirates/washes</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Influenza A ViralTarget</td><td colspan="1" rowspan="1">Matrix gene</td><td colspan="1" rowspan="1">Matrix gene</td></tr><tr><td colspan="1" rowspan="1">Influenza B ViralTarget</td><td colspan="1" rowspan="1">Non-structural protein (NSP) gene</td><td colspan="1" rowspan="1">Hemagglutinin gene</td></tr><tr><td colspan="1" rowspan="1">Assay Method</td><td colspan="1" rowspan="1">RT-PCR for detecting the presence /absence of viral RNA in clinicalspecimens</td><td colspan="1" rowspan="1">RT-PCR for detecting the presence /absence of viral RNA in clinicalspecimens</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnique</td><td colspan="1" rowspan="1">Multiplex assay using differentreporter dyes for each target</td><td colspan="1" rowspan="1">Multiplex assay using differentreporter dyes for each target</td></tr><tr><td colspan="1" rowspan="1">Assay Result</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Assay Instrument</td><td colspan="1" rowspan="1">Liat Analyzer</td><td colspan="1" rowspan="1">GeneXpert instruments</td></tr><tr><td colspan="1" rowspan="1">Self-containedSystem</td><td colspan="1" rowspan="1">Integrated PC, software, and touch-screen display</td><td colspan="1" rowspan="1">External PC computer and software forrunning tests and viewing the results</td></tr><tr><td colspan="1" rowspan="1">All AssayReagentsContained inDisposable</td><td colspan="1" rowspan="1">No manual reagent addition required</td><td colspan="1" rowspan="1">Requires manual dispensing ofBinding Reagent into the chamberwith the small opening of the XpertFlu Assay Cartridge</td></tr><tr><td colspan="1" rowspan="1">Automated Assay</td><td colspan="1" rowspan="1">Yes, sample preparation,amplification, detection and resultinterpretation</td><td colspan="1" rowspan="1">Yes, sample preparation,amplification, detection and resultinterpretation</td></tr><tr><td colspan="1" rowspan="1">ResultInterpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Laboratory Users</td><td colspan="1" rowspan="1">CLIA moderate complexity and highcomplexity laboratory</td><td colspan="1" rowspan="1">CLIA moderate complexity and highcomplexity laboratory</td></tr><tr><td colspan="1" rowspan="1">Time-to-result</td><td colspan="1" rowspan="1">~20 minutes</td><td colspan="1" rowspan="1">~75 minutes</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses. DRAFT GUIDANCE. Document issued on February 15, 2008. Docket number FDA-2008-D-0095.

# L. Test Principle:

Liat Influenza A/B Assay uses an established nucleic acid test chemistry and assay protocol for detection of viral RNA. The sample preparation methodology is based on lysis by a chaotropic agent followed by magnetic particle based nucleic acid purification. First, the NPS sample in UTM is diluted and mixed with an internal process control (IPC) comprising an encapsulated RNA. Chaotropic and proteolytic reagents then disrupt the three dimensional structure of the macromolecules such as proteins and nucleic acids in the sample. Second, nucleic acids are isolated from lysates through binding to silica magnetic beads in the presence of the chaotropic salt. Third, the beads with the bound nucleic acid are separated from the lysate using a magnetic field, and the lysate removed. Fourth, the beads with captured nucleic acids are washed to remove possible inhibitors in the sample. Finally, the captured nucleic acids are then eluted under low-salt conditions into a small volume of elution buffer.

Target amplification and detection uses TaqMan-probe based real-time polymerase chain reaction (RT-PCR). The Inf A primer and probe set is designed for the detection of matrix RNA from type A Influenza virus; while the Inf B primer and probe sets are designed to specifically detect the non-structural protein (NSP) RNA from Influenza B viruses. An IPC primer and probe set is also included to amplify the target region of the encapsulated RNA internal control.

Eluted viral RNA is first transcribed into cDNA using reverse transcriptase. This cDNA then undergoes a polymerase chain reaction (PCR) where the reaction mixture is repeatedly heated to denature the nucleic acid and cooled to allow annealing of primers and extension of annealed primers by DNA polymerase to logarithmically amplify a specific region of the cDNA. Duallabeled fluorogenic hydrolysis (TaqMan) probes anneal to specific target sequences located between the binding regions of forward and reverse primers. During the extension phase of the PCR cycle, the $5 ^ { \prime }$ nuclease activity of polymerase degrades the probes, causing the reporter dyes (e.g., FAM) to separate from the quenchers (e.g., BHQ, Black Hole Quencher), thus generating fluorescent signals. Fluorescence intensities are monitored at each PCR cycle. When the fluorescence intensities exceed pre-determined thresholds, cycle threshold (Ct) values are returned for the specific analyte corresponding to the fluorescence channel.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The multi-site reproducibility study assesses the total variability of the Liat Influenza A/B Assay across operators, study sites, testing days, Liat Analyzers, and Liat assay tube lots. The

Liat assay was evaluated at 3 sites. Two operators at each of the 3 sites tested an 8 member reproducibility panel in triplicate on 5 different days, for a total of 720 runs (8 panel members $\times 3$ replicates $\times 2$ operators $\times 5$ days $\times 3$ sites). Fifteen (15) Liat Analyzers and 3 Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay tube lots were used. The reproducibility panel comprises a high negative, a low positive and a medium positive of each of Influenza A and B, and the assay positive and negative controls. Percent agreement with expected result, mean Ct, and Ct $\% \mathrm { C V }$ for each site are shown below. The $\% \mathrm { C V }$ for Influenza A ranged between $1 . 3 \%$ and $3 . 8 \%$ and that for Influenza B ranged between $1 . 1 \%$ and $3 . 4 \%$ . Total percent agreement was $2 9 9 . 9 \%$ .

The reproducibility panel samples were constructed by spiking viruses of known titer into negative NPS matrix. A/NY/01/09 and B/Florida/04/06 were used in this study. The medium positive, low positive, and high negative concentrations used for each of the strains corresponded to C100 (4xLOD), C95 (1xLOD), and C5 (as determined by Probit analysis using LOD data), respectively.

Positive and negative control panel members were the Liat assay positive and negative controls. Positive control, comprising a mixture of inactivated flu A and B virus at concentration close to LOD, was provided in a unit dose $5 \mathbf { x }$ solution. An additional step of diluting the positive control 5x solution with UTM using a transfer pipette was required prior to loading into the Liat tube. Negative control comprised UTM.

Influenza A and B Reproducibility Panel   

<table><tr><td rowspan=1 colspan=1>#</td><td rowspan=1 colspan=1>Panel Members</td><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Positive Control</td><td rowspan=1 colspan=1>Pooled inactivatedInfluenza A &amp; B virus</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu A Medium Positive (C100)</td><td rowspan=3 colspan=1>A/NY/01/09</td><td rowspan=1 colspan=1>1.20</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Flu A Low Positive (C95)</td><td rowspan=1 colspan=1>3.0x10-1</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu A High Negative (C5)</td><td rowspan=1 colspan=1>1.7x10-2</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Flu B Medium Positive (C100)</td><td rowspan=3 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>9.6x10-1</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Flu B Low Positive (C95)</td><td rowspan=1 colspan=1>2.4x10-1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Flu B High Negative (C5)</td><td rowspan=1 colspan=1>5.4x10-3</td></tr></table>

The six operators at 3 sites tested the members of the reproducibility panel in triplicate on 5 different days. Five (5) Liat Analyzers were used at each site for a total of 15 Liat Analyzers. The sites also used different lots of Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay tubes for a total of 3 lots. A total of 720 runs (8 panel members $\times 3$ replicates $\times 2$ operators $\times 5$ days $\times 3$ sites) were performed.

The $\%$ total agreement with expected results, $\mathrm { C t }$ average and $\mathrm { C t \% C V }$ for each panel member from each site is shown below. For Ct data analysis, a $\mathrm { C t }$ of 39.0 is used for all negative results given the $\mathrm { C t }$ cutoff of $3 8 . 5 . \ \% \mathrm { C V }$ for Influenza A ranged between $1 . 3 \%$ and $3 . 8 \%$ and that for Influenza B ranged between $1 . 1 \%$ and $3 . 4 \%$ .

Of the 90 total replicates, one Flu B high negative (C5) was detected as Inf B positive $1 . 1 \%$ detection rate, $9 5 \%$ CI $0 . 0 \% { - } 6 . 0 \%$ ). This result can be expected given that the C5 sample is expected to be detected $5 \%$ of the time. All other results agree with the expected result.

Across 3 sites, 6 operators, 15 Liat Analyzers, 3 Liat Influenza A/B Assay tube lots, and 5 days of testing, $100 \%$ and $9 9 . 9 \%$ agreement with expected results was achieved for Influenza A and B reproducibility, respectively.

# Influenza A Reproducibility

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NegativeL Control</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A Med.Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B Med.Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr></table>

# Influenza B Reproducibility

<table><tr><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="2" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Agree-ment w/expectedresult</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">Agree-ment w/expectedresult</td><td colspan="1" rowspan="1">A Ye</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">Agree-ment w/expectedresult</td><td colspan="1" rowspan="1">AvgCt</td><td colspan="1" rowspan="1">Total%CV</td><td colspan="1" rowspan="1">Agree-ment w/expectedresult</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">PositiveL Control</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.0</td><td colspan="1" rowspan="1">2.2%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.6</td><td colspan="1" rowspan="1">1.1%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">NegativeCControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">FuA HighN gative</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Flu A LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Flu A Med.Positive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Flu B HighN Negative</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">29/30</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">1.7%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.0%</td><td colspan="1" rowspan="1">89/90(98.9%)</td><td colspan="1" rowspan="1">94.0%-99.8%</td></tr><tr><td colspan="1" rowspan="1">Flu B LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">2.3%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.0</td><td colspan="1" rowspan="1">3.4%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">1.7%</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Flu B Med.</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.4</td><td colspan="1" rowspan="1">2.6%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.5</td><td colspan="1" rowspan="1">3.0%</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">28.6</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">90/90</td><td colspan="1" rowspan="1">95.9%-100.0%</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">(100%)</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">TotalAgreement</td><td colspan="3" rowspan="1">240/240 (100%)</td><td colspan="3" rowspan="1">239/240 (99.6%)</td><td colspan="3" rowspan="1">240/240 (100%)</td><td colspan="1" rowspan="1">719/720(99.9%)</td><td colspan="1" rowspan="1">99.2%-100.0%</td></tr></table>

# $b$ . Inter-lot Precision

Inter-lot precision assesses the Liat tube manufacturing process as a potential source of assay variability. Three lots of Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay tubes were tested with one Influenza A strain at C95 (LOD, $\mathrm { \tt n } { = } 1 8 0$ ), C100 (4xLOD, $\scriptstyle \mathrm { n = 6 0 }$ ) and C0 (negative NPS matrix, $\mathrm { \tt n } { = } 6 0$ ). Results from all runs agreed with expected results. Imprecision in $\mathrm { C t }$ from inter-lot variability was $< 1 . 8 \%$ .

Inter-lot Precision   

<table><tr><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>Avg Ct</td><td rowspan=1 colspan=1>StdevInter-lot</td><td rowspan=1 colspan=1>%CVInter-lot</td></tr><tr><td rowspan=1 colspan=1>C100</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>C95</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>CO</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

Three Liat Influenza A/B Assay tube lots were manufactured at IQuum according to released procedures. One hundred (100) Liat tubes were selected at random from each lot.

Influenza A/Brisbane/10/2007 was used as the sample in this study. The virus was spiked into negative NPS matrix at C95 (LOD, $1 . 0 \mathrm { x } 1 0 ^ { - 1 } \mathrm { T } \mathrm { \dot { C } I D } _ { 5 0 } / \mathrm { m L } )$ or C100 (4xLOD, $4 . 0 \dot { \mathrm { x } } 1 0 ^ { - 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Negative samples (C0) comprised UTM without spiked virus. Samples were aliquoted into unit dose quantities and stored at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ until use.

From each of the 3 Liat tube lots, 60 tubes were tested at C95 $\mathrm { \ m = 1 8 0 }$ total), 20 at C100 $\scriptstyle \overbrace { \mathbf { n } \mathbf { - } 6 0 }$ total) and 20 at C0 $\scriptstyle \mathrm { n = 6 0 }$ total). One operator performed all runs on one Liat Analyzer. Testing occurred over 18 days, with an equal number of tubes from each lot being tested on each day. A total of 300 runs were performed in this study.

Results for all tubes from all lots agreed with the expected results as shown below. One-way Analysis of Variance (ANOVA) showed no statistically significant difference in $\mathrm { C t }$ between lots (p-value $= 0 . 6 3$ for C95, and 0.52 for C100). Ct $\% \mathrm { C V }$ due to inter-lot variation (calculated as square root of the inter-lot variance (MS) divided by average Ct) was $1 . 6 \%$ for C100 and $1 . 8 \%$ for C95 and is summarized below.

# Inter-lot Precision Data

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Liat Tube Lot 1</td><td rowspan=1 colspan=3>Liat Tube Lot 2</td><td rowspan=1 colspan=3>Liat Tube Lot 3</td><td rowspan=1 colspan=3>Total</td></tr><tr><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>Agree-ment w/ExpectedResult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agree-ment w/ExpectedResult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agree-ment w/Expected RResult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Agree-ment w/ExpectedResult</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>C100</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>60 / 60</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=1 colspan=1>C95</td><td rowspan=1 colspan=1>60 / 60</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>60 / 60</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>60 / 60</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>180 / 180</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>CO</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>60 / 60</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=3>100 / 100 (100%)</td><td rowspan=1 colspan=3>100 / 100 (100%)</td><td rowspan=1 colspan=3>100 / 100 (100%)</td><td rowspan=1 colspan=3>300 / 300 (100%)</td></tr></table>

c. Linearity/assay reportable range:

Not Applicable

d. Traceability, Stability, Expected values (controls, calibrators, or methods): The Liat Influenza A/B Assay has 3 controls: (1) internal process control, (2) positive control and (3) negative control.

# Internal process control

The internal process control (IPC) comprises a bacteriophage MS2 that is pre-packed in each Liat tube. The IPC is designed to be detected at a Ct of ${ \sim } 3 2$ on the Liat Influenza A/B Assay, similar to the Ct obtained from the 4x LOD concentration of Influenza viruses. When conducting an assay, it is first mixed with sample and then processed through all the test steps to monitor both the sample preparation and the RT-PCR reaction performance. The IPC RNA is detected in a separate channel by MS2 specific primers and probe.

# Positive Control

Positive control is provided in the Liat Influenza A/B Assay Quality Control Kit. The positive control comprises a pooled sample of inactivated Influenza A and B virus. The inactivated virus is obtained from ZeptoMetrix as individual NATtrol Influenza A and B Viruses (NATFLUA-ST, A/Singapore/63/04 H1N1 and NATFLUB-ST,

B/Yamanishi/166/98). NATrol products are prepared from highly purified infectious viruses, and are isolated from cell culture. Once purified, the virus is chemically and enzymatically treated to alter its surface proteins. This treatment results in a completely intact viral particle that is unable to bind, penetrate or infect a host cell. This viral particle, while modified on its surface, still retains a complete viral genome in its core, making it the ideal control for all steps of amplification assays.

The individual NATrol viruses are diluted and titered at IQuum to be detected at near LOD Ct for Influenza A and B using Liat Influenza A/B Assay. The viruses are then pooled to make the positive control solution. Positive control is provided in unit dose quantities as a 5x solution. To use the positive control, an operator first dilutes the positive control using UTM (also provided in unit dose) by transferring the entire contents of the Dilution UTM tube into the positive control tube using a transfer pipette, mixing the sample and then transferring the entire mixture into the Liat tube. The Liat tube is then run on a Liat Analyzer according to the Instructions for Use.

The stability of positive control has been assessed using 3 lots. The studies show that the positive control is stable for ${ > } 3$ months at $4 \%$ . The 3 month time point at $4 \%$ was the last point tested and there was no significant change in the detected Inf A and $\mathrm { ~ B ~ C t ~ }$ and amplitude. Based on this data, the labeled shelf life of the Quality Control Kit is 3 months.

The positive control is required to be run during the “Add Lot” process, where the Liat tube lot is validated at the end user site. Additional positive control runs may be performed by the end-user to confirm the performance of a Liat Analyzer and a Liat tube lot through positive target detection, to check if there is a false negative result, or as required by the end user’s quality control standards.

# Negative Control

Negative control is provided in the Liat Influenza A/B Assay Quality Control Kit. The negative control comprises UTM. The solution is provided in unit dose quantity and labeled as Dilution UTM. To use the negative control, an operator transfers the entire contents of the Dilution UTM tube into the Liat tube using a transfer pipette and runs the assay following the Instructions for Use.

The negative control is required to be run during the “Add Lot” process. Additional negative control runs may be performed by the end-user to check if contamination is present resulting in a false positive result, or as required by the end user’s quality control standards.

# e. Detection limit:

LOD studies determine the lowest detectable concentration of influenza virus at which greater than or equal to $9 5 \%$ of all replicates test positive. The LOD of the Liat assay for Influenza A and Influenza B was determined by limiting dilution studies using titered virus. Three strains of Influenza A (A/Brisbane/10/2007, A/Brisbane/59/2007, and A/NY/01/2009) and two strains of Influenza B (B/Malaysia/2506/04 and B/Florida/04/06) were used.

Virus strains obtained from ZeptoMetrix (Buffalo, NY) were cultured and re-titered using $\mathrm { T C I D } _ { 5 0 }$ procedure before their use in LOD determination. Each characterized influenza strain was first adjusted to the highest $\log 1 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in UTM as diluent. The working viral stock was further serially diluted in 10 fold increments in UTM to obtain the test concentrations.

Influenza Strains & Viral Stock Titer Used in LOD Study   

<table><tr><td rowspan=1 colspan=1>Viral Strain</td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>TCID50/mLSupplied</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>A/H3N2</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>5.6x104</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>A/H1N1</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>1.4x105</td></tr><tr><td rowspan=1 colspan=1>A/NY/01/2009</td><td rowspan=1 colspan=1>A/2009 H1N1</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>1.7x105</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>1.4x105</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>ZeptoMetrix</td><td rowspan=1 colspan=1>1.4x105</td></tr></table>

To prepare the testing sample, the virus at different concentrations was spiked into a pooled nasopharyngeal swab (NPS) sample matrix. NPS were collected throughout the study period from healthy donors that did not show symptoms of flu. The swabs were eluted in UTM, pooled, and then tested to confirm that this NPS matrix was negative for Influenza. Virus was spiked into the NPS matrix at no more than $5 \%$ the final volume to prepare the test sample.

For each run, $1 0 0 \mu \mathrm { L }$ of the spiked sample was added into the Liat Influenza A/B Assay Tube according to the operation procedures indicated in the Liat Influenza A/B Assay Instructions for Use (IFU). The Liat Tube was then inserted into the Liat Analyzer. Serial dilutions of the characterized influenza viruses were tested in triplicates. The lowest concentration at which all three replicates tested positive was treated as the tentative LOD for each strain. The LOD of each strain was then confirmed by testing at least 20 replicates with concentrations at the tentative LOD.

A total of 188 runs were performed in this study. Runs were performed on 27 Liat Analyzers and primarily using 3 lots of Liat tubes.

The test concentrations and corresponding call rates (positive result/total replicates) for each strain are listed below. The LOD of each strain was determined as the lowest concentration resulting in positive detection of at least 19 out of 20 replicates.

# LOD Determination

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Stock VirusConcentration(TCID50/ml)</td><td rowspan=1 colspan=1>TestConcentration(TCID50/ml)</td><td rowspan=1 colspan=1>CallRate</td><td rowspan=1 colspan=1>CtAvg</td><td rowspan=1 colspan=1>Ct%CV</td></tr><tr><td rowspan=2 colspan=1>A/Brisbane/10/2007</td><td rowspan=2 colspan=1>Inf A</td><td rowspan=2 colspan=1>5.6×104</td><td rowspan=1 colspan=1>1.0x10-2</td><td rowspan=1 colspan=1>0/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>1.0x10-1</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=3 colspan=1>A/Brisbane/59/2007</td><td rowspan=3 colspan=1>Inf A</td><td rowspan=3 colspan=1>1.4x105</td><td rowspan=1 colspan=1>4.0x10-3</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>8.0x10-3</td><td rowspan=1 colspan=1>21/22</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>6.2%</td></tr><tr><td rowspan=1 colspan=1>1.0x10-2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=5 colspan=1>A/NY/01/2009</td><td rowspan=5 colspan=1>Inf A</td><td rowspan=5 colspan=1>1.7x105</td><td rowspan=1 colspan=1>1.0x10-3</td><td rowspan=1 colspan=1>0 /3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>1.0x10-2</td><td rowspan=1 colspan=1>0 /3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2.0x10-2</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>5.0x10^2</td><td rowspan=1 colspan=1>8 /10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>1.0x10-1</td><td rowspan=1 colspan=1>21 /21</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>4.9%</td></tr><tr><td rowspan=6 colspan=1>B/Florida/04/06</td><td rowspan=6 colspan=1>Inf B</td><td rowspan=6 colspan=1>1.4x105$</td><td rowspan=1 colspan=1>2.5x10-3</td><td rowspan=1 colspan=1>0 /10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>1.0x10-2</td><td rowspan=1 colspan=1>5/14</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2.0x10-2</td><td rowspan=1 colspan=1>3 /10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>4.0x10-2</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>8.0x10^2</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>32.5</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>1.0x10-1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=3 colspan=1>B/Malaysia/2506/04</td><td rowspan=3 colspan=1>Inf B</td><td rowspan=3 colspan=1>1.4x105</td><td rowspan=1 colspan=1>1.0x10-3</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>2.0x10-3</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>1.0x10-2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr></table>

The Limit of Detection (LOD) of the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay was tested using 3 strains of Influenza A (A/Brisbane/10/2007, A/Brisbane/59/2007, and $\mathrm { A } / \mathrm { N Y } / 0 1 / 2 0 0 9 )$ and 2 strains of Influenza B (B/Malaysia/2506/04 and B/Florida/04/06). The LOD was determined by

limiting dilution studies using these titered viruses. The viruses were spiked into nasopharyngeal swab (NPS) sample matrix, and then tested using the Liat Influenza A/B Assay. The LOD was determined as the lowest log virus concentration that was detected $2 9 5 \%$ of the time (i.e. log concentration at which at least 19 out of 20 replicates tested positive). The LOD for 3 strains of Influenza A were $1 0 ^ { - 2 }$ to $1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } \mathrm { \dot { / } m L }$ , while those for the 2 strains of Influenza B were $1 0 ^ { - 3 }$ to $1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

# f. Analytical specificity (reactivity):

Reactivity study evaluates the ability of the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay to detect influenza strains representing temporal and geographical diversity. The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay was evaluated for reactivity with 22 Influenza A strains and 10 Influenza B strains. Influenza A strains included 8 seasonal Influenza A/H1 strains, 8 seasonal Influenza A/H3 strains, 3 Influenza A $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ strains, 3 swine origin Influenza A strains. The Liat Influenza A/B Assay detected all strains tested.

Cultured viral strains were obtained from ZeptoMetrix Corporation (Buffalo, NY), American Type Culture Collection (Manassas, VA) and Advanced Biotechnology (Columbia, MD). All strains were frozen from the vendor and were accompanied by Certificate of Analysis including $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $\mathrm { C E I D _ { 5 0 } / m L }$ titer information.

Each viral strain under test was serially diluted in 10 fold increments in UTM using the same procedure as that for the LOD study. The virus was then spiked in negative NPS matrix, and $1 0 0 \mu \mathrm { L }$ of this spiked sample was tested using the Liat assay.

Each viral strain was tested in triplicate at the LOD for influenza strains used in LOD studies, and at “near LOD” level for other strains. To estimate “near LOD” level, each strain was first tested at $1 . 0 { \times } 1 0 ^ { 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ or $1 . 0 { \times } 1 0 ^ { 2 } \mathrm { C E I D } _ { 5 0 } / \mathrm { m L }$ and then higher or lower concentrations until a concentration that gave a Ct between 32 and 34 was determined. Given that the Ct at LOD was 32-34 based on LOD study data, a $\mathrm { C t }$ value in this range was used to estimate the “near LOD” level. This concentration was then tested in triplicate to demonstrate reactivity. If less than $3 / 3$ runs are detected as positive, viral concentration was increased in log increments until triplicate positive results are observed.

A total of 115 runs were performed in this study. Runs were preformed on primarily 13 Liat Analyzers and using 2 lots of Liat tubes.

Influenza Strains Used for Reactivity Study   

<table><tr><td colspan="1" rowspan="1">Influenza Strain</td><td colspan="2" rowspan="1">Supplied Concentration</td><td colspan="1" rowspan="1">Source</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/2007</td><td colspan="1" rowspan="1">1.4×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">3.4×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Island/3/2006</td><td colspan="1" rowspan="1">4.2×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">8.9×109</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">8.9×108</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/FM/1/47</td><td colspan="1" rowspan="1">1.6×109</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC2</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">3.0×109</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">AB</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">5.6×1010</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/2007</td><td colspan="1" rowspan="1">5.6×104</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">5.0×108</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/MRC2</td><td colspan="1" rowspan="1">8.9×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">8.9×106</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">AB</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">8.9×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">3.2×106</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">1.5×108</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.6×108</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/NY/01/2009</td><td colspan="1" rowspan="1">1.7×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">2.5×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/NY/03/2009</td><td colspan="1" rowspan="1">1.0×106</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">8.9×106</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Swine/1976/31</td><td colspan="1" rowspan="1">8.9×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Iowa/15/30</td><td colspan="1" rowspan="1">8.9×108</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06</td><td colspan="1" rowspan="1">1.4×10</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">1.4×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">B/Florida/7/04</td><td colspan="1" rowspan="1">4.2×10}$</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ZeptoMetrix</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1.6×106</td><td colspan="1" rowspan="1">CEID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">1.6×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">2.8×105</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">8.9×104</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Mass/3/66</td><td colspan="1" rowspan="1">8.9×107</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">8.9×106</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">ATCC</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">1.4×109</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">AB</td></tr></table>

Reactivity   

<table><tr><td colspan="1" rowspan="2">Influenza A Strain</td><td colspan="1" rowspan="2">Type / Subtype</td><td colspan="1" rowspan="2">Test Concentration</td><td colspan="2" rowspan="1">Inf A</td></tr><tr><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Avg Ct</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×102TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.4</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Island/3/2006</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.3</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×103 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">5.0×102TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">A/FM/1/47</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×102TCID50/mL</td><td colspan="1" rowspan="1">3 / 3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">2.5×101TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.3</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">2.5×103 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">A/MRC2</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">1.0×102 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.5×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td></tr><tr><td colspan="1" rowspan="1">A/Wisconsin/67/05</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.4</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×102TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.9</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.0×102 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td></tr><tr><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">Influenza A, 2009 H1N1</td><td colspan="1" rowspan="1">2.5×10-2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.4</td></tr><tr><td colspan="1" rowspan="1">A/NY/03/2009</td><td colspan="1" rowspan="1">Influenza A, 2009 H1N1</td><td colspan="1" rowspan="1">4.0×10-1 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.7</td></tr><tr><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">1.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.5</td></tr><tr><td colspan="1" rowspan="1">A/Swine/1976/31</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">2.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">A/Swine/Iowa/15/30</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">2.0×102 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td></tr><tr><td colspan="1" rowspan="2">Influenza B Strain</td><td colspan="1" rowspan="2">Type</td><td colspan="1" rowspan="2">Test Concentration</td><td colspan="1" rowspan="1">Inf B</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Avg Ct</td></tr><tr><td colspan="1" rowspan="1">B/Florida/7/04</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-1 CEID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.6</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-2 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10-3 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.1</td></tr><tr><td colspan="1" rowspan="1">B/Mass/3/66</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×101 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.5×10-1 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×100TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td></tr></table>

The Influenza strains tested for reactivity and the concentration of each strain that provided $3 / 3$ positive results. At the determined “near LOD” level, all triplicate runs for each strain were detected as positive, with average Ct in the 32-34 range. Data demonstrates that the Liat Influenza A/B Assay is reactive against all strains tested.

# g. Analytical specificity (Cross-reactivity):

Cross reactivity study evaluates the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay’s potential cross-reactivity with non-influenza respiratory pathogens and other microorganisms with which the majority of the population may have been infected. The Liat assay was evaluated against a panel of 31 human pathogens. Bacteria were tested at $1 0 ^ { 5 } – 1 0 ^ { 6 } \mathrm { C F U / m L }$ . Viruses were tested at $1 0 ^ { 3 } – 1 0 ^ { 5 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay showed no cross reactivity for the tested organisms. The test panel comprised 15 viral and 16 bacterial strains. Cultured virus and bacteria were obtained from ZeptoMetrix Corporation (Buffalo, NY), American Type Culture Collection (Manassas, VA) and Advanced Biotechnology (Columbia, MD). All cultured viruses were frozen from the vendor and were accompanied by Certificate of Analysis including $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ titers. All bacteria were cultured and titered at IQuum following receipt from vendor. Bacterial culture and titering were performed based on methods described in Bacteriological Analytical Manual.

Each virus or bacteria was spiked individually into negative NPS matrix with the spiked volume not exceeding $5 \%$ of the sample volume. The tested concentrations were $\geq { 1 0 } ^ { 6 }$

CFU/mL for bacteria and $\geq 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for virus, or the highest titers obtained. One hundred microliters $( 1 0 0 \mu \mathrm { L } )$ of the spiked sample was tested using the Liat Influenza A/B Assay. Each pathogen was tested in triplicate. A total of 93 runs were performed in this study. Runs were preformed primarily on 9 Liat Analyzers and using 2 lots of Liat assay tubes. Negative results in all runs confirmed no cross-reactivity for the pathogens tested.

Pathogens and Test Levels for Potential Cross-reactivity   

<table><tr><td colspan="1" rowspan="1">Viral Pathogens</td><td colspan="1" rowspan="1">Source</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8.9×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.4×103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">2.8×103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.4×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">ZeptoMetrix</td><td colspan="1" rowspan="1">7×103 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus type B</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.4×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus Type 1A</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">AB</td><td colspan="1" rowspan="1">1.9×105 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.8×105 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">8×104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">5.0×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">6.6×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">3×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">7×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">5.8×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">3.2×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.8×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">4.5×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">6×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">ATCC</td><td colspan="1" rowspan="1">3.7×106 CFU/mL</td></tr><tr><td>Streptococcus salivarius</td><td>ATCC</td><td>4.3×106 CFU/mL</td></tr></table>

# h. Interfering Microorganisms:

Interfering microorganism study evaluates whether non-influenza respiratory pathogens and other microorganisms with which the majority of the population may have been infected can interfere in the detection of Influenza A or B by the Liat assay. The panel of 31 human pathogens tested in the cross-reactivity study was tested for potential interference. Bacteria were tested at $1 0 ^ { 5 } – 1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $1 \dot { 0 } ^ { 3 } { - } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in the presence of either A/Brisbane/59/2007 or B/Malaysia/2506/04 at 3x LOD concentration in negative NPS matrix. One hundred microliters $( 1 0 0 \mu \mathrm { L } )$ of the spiked sample was tested using the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay. Each microorganism was tested in triplicate with each of A/Brisbane/59/2007 or B/Malaysia/2506/04. Results demonstrate that the presence of the tested microorganisms do not interfere with the detection of Inf A or Inf B.

Interfering Microorganisms – Influenza A Detection   

<table><tr><td colspan="1" rowspan="2">Pathogen</td><td colspan="1" rowspan="2">PathogenConcentration</td><td colspan="4" rowspan="1">A/Brisbane/59/2007 (2.4×10-2 TCID50/mL)</td></tr><tr><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Inf AAvg Ct</td><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Inf B AvgCt</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.0</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">8.9×105 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">1.4×103 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">2.8×103 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">1.4×105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">7×10³ TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus type B</td><td colspan="1" rowspan="1">1.4×104 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus Type 1A</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.9×105 copies/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.8×105 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">8×104 TCID50/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">5.0×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6.6×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3×106 CFU/mL</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">7×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.8×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">3.2×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">1.8×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.5</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">4.5×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">6×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3.7×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.8</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">4.3×106 CFU/mL</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.6</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td></tr></table>

# Interfering Microorganisms – Influenza B Detection

<table><tr><td colspan="1" rowspan="2">Pathogen</td><td colspan="1" rowspan="2">PathogenConcentration</td><td colspan="4" rowspan="1">B/Malaysia/2506/04 (6.0×10-3 TCID50/mL)</td></tr><tr><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Inf AAvg Ct</td><td colspan="1" rowspan="1">CallRate</td><td colspan="1" rowspan="1">Inf B AvgCt</td></tr><tr><td colspan="1" rowspan="1">Control</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.1</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">8.9×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.1</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 7</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus 229E</td><td colspan="1" rowspan="1">1.4×103 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.3</td></tr><tr><td colspan="1" rowspan="1">Human Coronavirus OC43</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.1</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.5</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 1</td><td colspan="1" rowspan="1">2.8×103 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.9</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 2</td><td colspan="1" rowspan="1">1.4×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.4</td></tr><tr><td colspan="1" rowspan="1">Human Parainfluenza Type 3</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.5</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.6</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">7×10³ TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">7.9×104 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.2</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus type B</td><td colspan="1" rowspan="1">1.4×104 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.7</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus Type 1A</td><td colspan="1" rowspan="1">1.6×105 TCID50/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.9</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">4.5×104 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.2</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">1.9×105 copies/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">30.9</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.8×105 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.3</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">8×104 TCID50/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">5.0×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6.6×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.3</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">34.6</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.0</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">7×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.8×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">35.1</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">3.2×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3 /3</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">1.8×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.6×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">4.5×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.6</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">6×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.9</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">1.9×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3.7×106 CFU/mL</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">4.3×106 CFU/mL</td><td colspan="1" rowspan="1">0 /3</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.9</td></tr></table>

A total of 192 runs were performed in this study. Runs were preformed on 18 Liat Analyzers and using 3 lots of Liat tubes.

# i. Interfering substances

The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay was evaluated with potentially interfering substances that may be encountered in respiratory specimens. Medically and/or physiologically relevant concentrations of potential interferents were tested with 2 influenza A strains and 2 influenza B strains at $3 \mathrm { { x } \ L O D }$ $( 1 0 ^ { - 1 } { - } 1 0 ^ { - 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ . Results showed that substances tested did not interfere in the detection of influenza A and B strains.

# Interfering Substance Panel

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>MucinBovine submaxillary gland, type I-S</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>0.1 mg/ml and 25mg/ml</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal spray  Afrin</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal corticosteroids  Veramyst</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal gel  Zicam</td><td rowspan=1 colspan=1>Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulphur</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Throat lozenges, oral anesthetic andanalgesic  Cepacol</td><td rowspan=1 colspan=1>Benzocaine, Menthol</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment  Bactroban</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>7.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antimicrobial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/ml</td></tr></table>

Concentration of Influenza Stains used in Interfering Substance Study   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration (3xLOD)</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>3.0×10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>2.4×10-2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>2.4×10-1 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>6.0×10-3 TCID50/mL</td></tr></table>

Each potential interferent was individually spiked into negative NPS matrix at the indicated concentration along with one of the 4 influenza viruses. One hundred microliters $( 1 0 0 \mu \mathrm { L } )$ of these spiked samples were tested using the Liat assay. Each interferent and influenza virus combination was tested in triplicate. Results were compared against a control containing no spiked interferent. Nasal administered Influenza vaccine was not evaluated as part of this study, and therefore the performance of this device in the presence of this potential interferent can not be evaluated. A statement in the warnings and precautions has been included.

A total of 132 runs were performed in this study. Runs were preformed primarily on 19 Liat Analyzers and using 2 lots of Liat tubes.

All triplicate runs agreed with the expected results for all interfering substances tested. The average Ct for testing each interfering substance is also shown below. Results showed that substances tested did not interfere in the detection of influenza A and B strains.

Interfering Substance Results   

<table><tr><td rowspan=2 colspan=1>PotentialInterferent</td><td rowspan=2 colspan=1>Conc.</td><td rowspan=1 colspan=2>A/Brisbane/10/07</td><td rowspan=1 colspan=2>A/Brisbane/59/07</td><td rowspan=1 colspan=2>B/Florida/04/06</td><td rowspan=1 colspan=2>B/Malaysia/2506/04</td></tr><tr><td rowspan=1 colspan=1>Inf ADetected</td><td rowspan=1 colspan=1>Inf ACt Avg</td><td rowspan=1 colspan=1>Inf ADetected</td><td rowspan=1 colspan=1>Inf ACt Avg</td><td rowspan=1 colspan=1>Inf BDetected</td><td rowspan=1 colspan=1>Inf BCt Avg</td><td rowspan=1 colspan=1>Inf BDetected</td><td rowspan=1 colspan=1>Inf BCt Avg</td></tr><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td></tr><tr><td rowspan=1 colspan=1>Afrin</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.3</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td></tr><tr><td rowspan=1 colspan=1>Cepacol</td><td rowspan=1 colspan=1>5 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.6</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>0.1 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td></tr><tr><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>5 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>5 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.1</td></tr><tr><td rowspan=1 colspan=1>Tamiflu</td><td rowspan=1 colspan=1>7.5 mg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.9</td></tr><tr><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/ml</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.1</td><td rowspan=1 colspan=1>4/3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.6</td></tr><tr><td rowspan=1 colspan=1>Veramyst</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>31.7</td></tr><tr><td rowspan=1 colspan=1>Zicam</td><td rowspan=1 colspan=1>5% (v/v)</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.7</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>30.9</td></tr></table>

# $j .$ . Fresh vs. Frozen Equivalency

The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay was tested by comparing its performance using fresh and frozen specimens. One Influenza A strain (A/Brisbane/10/07) and one Influenza B strain (B/Malaysia/2506/04) were individually spiked into negative NPS matrix at different levels of viral load, including near LOD, and levels reflecting the clinical range. For each strain, 60 samples were tested immediately while another 60 samples were frozen at ${ } _ { - 8 0 } { } ^ { \circ } \mathrm { C }$ for 7 days, thawed and then tested. The strains, test concentration, and number of replicates (n) are shown below.

Comparing fresh and frozen sample results, Ct showed a delay of $0 . 5 \mathrm { C t }$ on average for flu A and $1 . 2 \mathrm { C t }$ on average for flu B. Such Ct delay reflects the degradation of virus during freezethaw cycle and is well established in the literature. However, fresh and frozen samples demonstrated $100 \%$ detection across all levels of viral load, including those near LOD. This demonstrates that sample freeze thaw had no effect on the detection of Influenza A and B by the Liat Influenza A/B Assay. A total of 240 runs were performed in this study. Runs were preformed on primarily 16 Liat Analyzers and using 1 lot of Liat tubes.

# Freeze Thaw Study Results

<table><tr><td rowspan=1 colspan=4></td><td rowspan=1 colspan=4>Fresh</td><td rowspan=1 colspan=4>Frozen</td></tr><tr><td rowspan=1 colspan=1>Inf</td><td rowspan=1 colspan=1>x LOD</td><td rowspan=1 colspan=2>Conc.(TCID50/mL)   n</td><td rowspan=1 colspan=1>CallRate</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Stdev</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>CallRate</td><td rowspan=1 colspan=1>AvgCt</td><td rowspan=1 colspan=1>Stdev</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=5 colspan=1>LA/03/s/</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3.00E-01</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>30.8</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.5%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.00E+00</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>1.0%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>2.00E+01</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>24.7</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>2,000</td><td rowspan=1 colspan=1>2.00E+02</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>21.3</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>21.7</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>1.2%</td></tr><tr><td rowspan=1 colspan=1>20,000</td><td rowspan=1 colspan=1>2.00E+03</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>17.4</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>17.9</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.9%</td></tr><tr><td rowspan=5 colspan=1>B0/037/s/</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.00E-03</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>3.5%</td><td rowspan=1 colspan=1>30 / 30</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.00E-02</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>28.8</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>3.2%</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.6%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4.00E-01</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>25.8</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>10 / 10</td><td rowspan=1 colspan=1>26.6</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>2.9%</td></tr><tr><td rowspan=1 colspan=1>2,000</td><td rowspan=1 colspan=1>4.00E+00</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>23.8</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>1.7%</td></tr><tr><td rowspan=1 colspan=1>50,000</td><td rowspan=1 colspan=1>1.00E+02</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>17.4</td><td rowspan=1 colspan=1>0.19</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>18.7</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>2.0%</td></tr></table>

# $k .$ Assay cut-off:

The PCR crossing threshold $\mathrm { ( C t ) }$ for different Influenza stains at LOD was determined to be $3 3 . 5 \pm 0 . 8$ (average $\pm$ standard deviation), with the maximum $\mathrm { C t }$ average of 34.7. Using Probit analysis for two Influenza strains (B/Florida/04/06 and A/NY/01/2009), the lowest Ct was determined by linear regression. The Ct at the lowest detectable virus concentration (as determined by Probit analysis) was estimated to be 37.3 for B/Florida/04/06 and 38 for A/NY/01/2009. A Ct cutoff of 38.5 was selected for both Influenza A and Influenza B. In order to reliably detect the Ct of 38.5 an additional 3.5 cycles are run after the indicated cutoff Ct resulting in a total of 42 PCR cycles.

# l. Carry-over/Cross-contamination:

The Liat Influenza A/B Assay utilizes a single-use consumable. All reagents are pre-packed and stored in unit-dose quantities in tube segments contained within the disposable Liat tube. Once the sample is loaded into the Liat tube, the tube remains closed during the entire assay

processing. Additionally, each Liat Analyzer performs one run on one sample at a time. As such, given the sealed single-use nature of the consumable, the risk of carry-over or crosscontamination is negligible.

# 2. Comparison studies:

a. Method comparison with predicate device:

Not applicable, performance of the assay was evaluated in comparison to the gold standard/reference method, viral culture followed by DFA and/or viral culture followed by sequencing.

b. Matrix comparison:

Not applicable

3. Clinical studies:

a. Clinical Sensitivity and Specificity:

# Prospective Samples

Prospective samples were collected at 6 clinical sites between 12-Feburary-2009 to 26- March-2009. Patients were enrolled based on the inclusion/exclusion criteria summarized in the table below.

<table><tr><td rowspan=1 colspan=2>Inclusion criteria</td></tr><tr><td rowspan=1 colspan=1>|</td><td rowspan=1 colspan=1>Symptomatic: Patients exhibiting 2 or more flu-like clinical signs and symptoms withinthe past 24-48 hours. Signs and symptoms of the flu include fever (usually high);headache; extreme tiredness; dry cough; sore throat; runny or stuffy nose; muscle aches;and, gastro-intestinal symptoms, such as nausea, vomiting, and diarrhea.*</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Patients who can understand and read English or Spanish;</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Patients who complete the Informed Consent Form;</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>When under the age of 18, Patients who are accompanied by a parent to provide guardianconsent; and,</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>When aged 6-17, Patients who complete the Minor Assent form.</td></tr><tr><td rowspan=1 colspan=2>Exclusion Criteria</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Subjects not experiencing Flu-like symptoms; and, subjects currently taking anti-viralmedication.</td></tr></table>

\* Symptoms and time from on-set of symptoms were self-reported by the patient in order to identify patients who had contemporaneous signs and symptoms of respiratory infection.

Clinical samples (nasopharyngeal swabs (NPS)) were collected for the validation of the Liat Influenza Assay, which detects Influenza A and B. Data from prospective samples were compared against viral culture as the reference method. The two-by-two tables comparing the

Liat Influenza A/B Assay against viral culture for Influenza A and B detection are shown below.

No false negative results were detected in $4 3 5 \mathrm { N P S }$ samples tested, while 13 false positives according to culture results were reported for Influenza A and 24 false positive results according to culture were reported for Influenza B. PCR and bi-directional sequencing were used to investigate these discordant results. This method confirmed the Liat assay result in the majority of samples tested (see footnotes in performance tables).

Overall, the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay demonstrated $100 \%$ sensitivity ( $9 5 \%$ CI: $8 9 . 8 \%$ - $1 0 0 . 0 \% )$ and $9 6 . 8 \%$ specificity $9 5 \%$ CI: $9 4 . 5 \% - 9 8 . 1 \% )$ for Influenza A, and $100 \%$ sensitivity ( $9 5 \%$ CI: $8 8 . 6 \% - 1 0 0 . 0 \% )$ and $9 4 . 1 \%$ specificity $9 5 \%$ CI: $9 1 . 3 \% - 9 6 . 0 \%$ for Influenza B.

# Liat Assay Clinical Performance – Prospective Samples

<table><tr><td rowspan=1 colspan=3>EY</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(89.8% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>(94.5% - 98.1%)</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Influenza A</td><td rowspan=1 colspan=3>Viral Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>13a</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>388</td><td rowspan=1 colspan=1>388</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>435</td></tr></table>

Of 13 false positive samples, 8 were Influenza A positive by PCR/sequencing, 4 were negative by PCR/sequencing, and 1 was indeterminate by PCR/sequencing due to poor sequence quality score.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(88.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>(91.3% - 96.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Influenza B</td><td rowspan=1 colspan=3>Viral Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24b</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>381</td><td rowspan=1 colspan=1>381</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>435</td></tr></table>

b Of 24 false positive samples, 13 were Influenza B positive by PCR/sequencing, 3 were negative by PCR/sequencing, and 8 samples were indeterminate by PCR/sequencing due to poor sequence quality scores.

# Retrospective Samples

Data from retrospective samples were compared against PCR/sequencing as the reference method. The two-by-two tables comparing Liat Influenza A/B Assay against PCR/sequencing for Influenza A and B detection are shown below.

Of the 180 samples tested, 1 sample was indeterminate by PCR/sequencing, and was thus excluded from the analysis.

Of 74 Influenza A positive retrospective samples, 44 were confirmed to be 2009 H1N1 and 20 were confirmed and A/H3 by PCR/sequencing; the remaining 10 samples were confirmed to be Influenza A positive by PCR/sequencing but subtype sequencing was not successful. The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay detected all these samples as Influenza A positive.

In retrospective sample tests, the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay demonstrated $100 \%$ positive agreement ( $9 5 \%$ CI: $9 5 . 1 \% - 1 0 0 . 0 \% )$ and $9 7 . 1 \%$ negative agreement $9 5 \%$ CI: $9 1 . 9 \%$ -

$9 9 . 0 \%$ ) for Influenza A, and $100 \%$ positive agreement ( $9 5 \%$ CI: $6 4 . 6 \% - 1 0 0 . 0 \% )$ and $9 9 . 4 \%$ negative agreement ( $9 5 \%$ CI: $9 6 . 8 \% 9 9 . 9 \%$ ) for Influenza B.

# Liat Assay Clinical Performance – Retrospective Samples

<table><tr><td rowspan=2 colspan=2>Influenza A</td><td rowspan=1 colspan=2>PCR/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74c</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>179</td></tr></table>

<table><tr><td rowspan=1 colspan=3>EY</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveAgreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>NegativeAgreement</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>(91.9% - 99.0%)</td></tr></table>

c Of 74 Influenza A positive retrospective samples, 44 were 2009 H1N1 positive, and 20 were seasonal H3 positive by PCR/sequencing. The remaining 10 samples were verified to be Flu A positive, no further subtyping was done.

<table><tr><td rowspan=2 colspan=2>Influenza B</td><td rowspan=1 colspan=2>PCR/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>171</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>179</td></tr></table>

<table><tr><td rowspan=1 colspan=3>EY</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveAgreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(64.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>NegativeAgreement</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>(96.8% - 99.9%)</td></tr></table>

4. Clinical cut-off:

Not applicable

# 5. Expected values/Reference range:

In the prospective Liat Influenza A/B clinical study there was a total of $4 3 5 \mathrm { N P S }$ samples analyzed. All $4 3 5 \mathrm { N P S }$ specimens were tested for influenza A and influenza B by the Liat Influenza A/B Assay and compared to viral culture followed by fluorescence analysis. These samples were collected during the 2009 influenza season at six clinical laboratories in the U.S. from February and March. The age demographics for the prospectively collected samples are as follows:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Prospective</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>%</td></tr><tr><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>11%</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>130</td><td rowspan=1 colspan=1>30%</td></tr><tr><td rowspan=1 colspan=1>22-59 years</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>48%</td></tr><tr><td rowspan=1 colspan=1>≥60 years</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>11%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>197</td><td rowspan=1 colspan=1>45%</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>238</td><td rowspan=1 colspan=1>55%</td></tr></table>

# N. Instrument Name:

Liat Analyzer

# O. System Descriptions:

1. Modes of Operation:

The Liat Analyzer functions as a point-of-care platform with sample-to-answer capabilities in which all sample processing steps as well as detection are carried out using a single-use, disposable Liat Tube.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification:

Each test cartridge handles one sample and has a unique barcode identifier that is scanned into the Liat Analyzer to code the sample ID. A nasopharyngeal swab sample is loaded directly into a Liat assay tube using a transfer pipette. After the tube is capped, the Liat Analyzer scans the tube barcode, and the tube is inserted into the analyzer.

4. Specimen Sampling and Handling:

Specimen sampling and handling is controlled using 11 sample processing modules contained within the Liat Analyzer. The sample processing modules are composed of two assemblies, a moving side assembly comprised of 11 sample processing plungers and 10 clamps and a fixed side assembly. When performing an assay, a Liat tube is inserted into the tube slot of a Liat Analyzer. The plungers and clamps selectively compress the Liat tube segments against the fixed side assembly to release reagents from the segments, move the sample from one segment to another, and control reaction conditions.

5. Calibration:

Not required. Test cartridges are single use and part of a closed system.

6. Quality Control:

An internal control used in conjunction with procedural checks monitors instrument functionality, performance, fluidics, and result determination based on a pre-defined decision algorithm.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.