# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K111951   
B. Purpose for Submission: Clearance of New Device   
C. Measurand: Herpes Simplex Virus (HSV) gB DNA target sequence   
D. Type of Test: An in vitro molecular diagnostic test for the direct, qualitative detection of the Herpes Simplex Viruses in male and female genital and oral lesions   
E. Applicant: BioHelix Corporation   
F. Proprietary and Established Names: IsoAmp® HSV Assay   
G. Regulatory Information:

1. Regulation section: 21 CFR 866.3305. Herpes Simplex Virus

2. Classification: Class II

3. Product code: OQO, HSV NAAT assays

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The IsoAmp ${ } ^ { \mathfrak { B } } \mathrm { H S V }$ Assay is an in vitro diagnostic test for the direct, qualitative detection of the Herpes Simplex Virus (HSV-1 & HSV-2) DNA in male and female genital and oral lesions. The test is intended for use as an aid in diagnosis of HSV infection in symptomatic patients.

Warning: The IsoAmp® HSV Assay is not FDA cleared for use with cerebrospinal fluid (CSF). The assay does not provide specific typing information to differentiate HSV-1 and HSV-2. The assay is not intended to be used for prenatal screening.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s):

For prescription use only

4. Special instrument requirements:

N/A

# I. Device Description:

The IsoAmp ${ } ^ { \mathfrak { B } } \mathrm { H S V }$ Assay is an in-vitro diagnostic test for the direct, qualitative detection of Herpes Simplex Virus (HSV-1 & HSV-2) DNA in male and female genital and oral lesions from patients suspected of HSV infections. The assay utilizes Helicase-Dependent Amplification (HDA) for the amplification of the HSV glycoprotein B target and a target-specific hybridization probe for colorimetric detection of the amplicon on a lateral-flow strip embedded in a self-contained disposable plastic cassette. The assay doesn’t provide specific typing information to differentiate HSV-1 and HSV-2.

The IsoAmp® HSV Assay Kit contains reagents for 50 tests and is provided in two separate boxes:

(1) One box containing the Amplification-related Kit Components (ARKC) including the Amplification Reagent, Enzyme Reagent, HSV-1 Assay Positive Control, HSV-2 Assay Positive Control and Assay Negative control, and   
(2) One box containing the Non-amplification related Kit Components (NKC) including mineral oil, transfer pipettes, reaction and dilution tubes, cassette disposal bags and TypeII BESt Cassettes.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

MultiCode® RTx Herpes Simplex Virus 1&2 Kit (Eragen Bioscience, Inc.)

Reference Method for clinical evaluation: ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.)

2. Predicate Numbers (s):

K100336   
K971662

2. Comparison with predicate: The IsoAmp® HSV Assay was compared to the MultiCode® RTx Herpes Simplex Virus 1&2 Kit (Eragen Bioscience, Inc.) for the substantial equivalence.

Note: To establish the clinical performance, the FDA cleared ELVIS® HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) was used as the reference

method. The performance of the IsoAmp® HSV Assay was compared with the reference method which is a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system.

<table><tr><td colspan="3">Similarities</td></tr><tr><td>Item</td><td>IsoAmp® HSV Assay</td><td>Eragen Biosciences MultiCode- RTx Herpes Simplex Virus 1 &amp; 2 Kit</td></tr><tr><td>Intended Use</td><td>The IsoAmp® HSV Assay is a Rapid in vitro diagnostic test for the direct, qualitative detection of the Herpes Simplex Viruse (HSV-1 &amp; HSV-2) DNA in male and female genital and oral lesions. The test is intended for use as an aid in diagnosis of HSV infection in symptomatic patients. Warning: The IsoAmp® HSV Assay is not FDA cleared for the use with cerebrospinal fluid (CSF). The assay does not provide specific typing information to differentiate HSV-1 and HSV-2.</td><td>The MultiCode®-RTx HSV 1&amp;2 Kit is a polymerase chain reaction (PCR)-based qualitative in vitro diagnostic test for the detection and typing of herpes simplex virus (HSV 1 &amp; 2) DNA in vaginal lesions. It is indicated for use in the detection and typing of HSV-1 or HSV-2 in vaginal lesion swab specimens from symptomatic female patients as an aid in the diagnosis of genital herpes infection. Warning: The device is not FDA cleared for the use with cerebral spinal fluid (CSF) or any lesions other than vaginal. The assay is not intended to be used for male penile specimens, for prenatal screening, or females under the age of 18</td></tr><tr><td>Detection of HSV-1 and HSV-2 Assay Results</td><td>Yes</td><td>Yes</td></tr><tr><td colspan="3">Qualitative Qualitative Differences</td></tr><tr><td>Item</td><td>IsoAmp®HSV Assay</td><td>Eragen Biosciences MultiCode- RTx Herpes Simplex Virus 1 &amp; 2 Kit</td></tr><tr><td>Methodology</td><td>Helicase-Dependent Amplification (HDA)</td><td>Real-Time PCR</td></tr><tr><td>Typing of HSV-1 and HSV-2</td><td>No</td><td>Yes</td></tr><tr><td colspan="1" rowspan="1">AnalysisSoftwareProvided</td><td colspan="1" rowspan="1">No</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Packaging</td><td colspan="1" rowspan="1">The product is supplied as twoseparate labeled boxes.1. Amplification-related KitComponents (ARKC)2. Non-amplification relatedKit Components (NKC)</td><td colspan="1" rowspan="1">The product is supplied as twoseparate labeled boxes.1. MultiCode®-RTx HSV 1&amp;2 Kitcontents2. MultiCode®-RTx HSV 1&amp;2 KitAnalysis Software and PackageInsert</td></tr><tr><td colspan="1" rowspan="1">Kit ReagentStorageConditions</td><td colspan="1" rowspan="1">ARKC: &lt;-15°C; NKC: 15-30°C</td><td colspan="1" rowspan="1">-15°C to -30°C</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Male, Female Genital Lesions,Oral Lesions</td><td colspan="1" rowspan="1">Female Genital Lesions</td></tr><tr><td colspan="1" rowspan="1">Printed ResultsReportProvided</td><td colspan="1" rowspan="1">No (Visual colored band)</td><td colspan="1" rowspan="1">Yes</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Protocols for Determination of Limits of Detection (CLSI EP17-A 2004)

# L. Test Principle:

The IsoAmp® HSV Assay consists of three major steps: 1) specimen preparation: 2) isothermal HDA of HSV glycoprotein B (gB) gene using biotinylated primers; and 3) detection of the amplified DNA by target-specific hybridization probe via a colorimetric reaction on a lateral-flow strip which is embedded in a self-contained disposable cassette to prevent amplicon contamination.

Specimen preparation includes a simple dilution step in which specimens in viral transport medium are diluted 40-fold in dilution buffer. The diluted samples are mixed with Helicase-Dependent Amplification (HDA) reagents. Incubation at $6 4 ^ { \circ } \mathrm { C }$ results in the release of the HSV DNA and subsequent isothermal amplification of the target sequence. A competitive internal control (IC) is present in the reaction to monitor inhibitory substances in negative samples, reagent failure or device failure.

After incubation for one hour, the amplified DNA targets are detected by two detection probes, one labeled with fluorescein isothiocyanate (FITC) for hybridizing to the HSV target and the other labeled with digoxigenin (DIG) for binding to the IC target. The hybrid of FITC-labeled probe and HSV amplicon is captured at the Test Line (T-Line) on the strip by anti-FITC antibodies, while the DIG-labeled IC amplicon is captured at the Control Line (C-Line) on the strip by anti-DIG antibodies. The biotin label in each amplicon captures the streptavidin-conjugated color particles for visualization and the test result is shown as colored lines that are visually read.

The self-contained Type II BEStTM cassettes contain lateral-flow DNA detection strips coated with anti-FITC antibodies and anti-DIG antibodies that serve as T line and C line respectively in the assay. A positive result (detection of HSV DNA) is reported when the T line is visible through the detection window of the cassette. A negative result (no detection of HSV DNA is reported when only the C line is displayed. The assay result is regarded as invalid when both the T line and C line are not present and should be repeated.

# Recording and interpretation of the assay results:

• Positive: Always read the Test (T) line first. When T line is visible $( \mathrm { T } + )$ , report the assay result as “HSV DNA detected”.

• Negative: When no visible T line is present (T-), a visible C line indicates that the Internal Control DNA has been amplified and detected, eliminating the possibility of a false negative due to failure of amplification or device, and thus the assay result should be reported as negative - “no HSV DNA detected”.

• Invalid: If both T and C lines are not present (T-/C-), then the assay is invalid and the test needs to be repeated.

• Any visible $\mathrm { T }$ line and C line, regardless of intensity of that line, are recorded as a reactive test $( ^ { 6 6 } + 7 3 )$ , while the complete absence of any visible lines are recorded as a nonreactive test (“-”).

# Interpretation of the assay results

![](images/b88514dc3b0474860d8fe135ac28c8d018c6b97836df79d8f75f8462b8dcca2b.jpg)

The interpretation of the assay results is done according to the following criteria:

<table><tr><td>T line and C line Result</td><td>Interpretation of Result</td></tr><tr><td colspan="1" rowspan="1">T+/C+ or T+/C-</td><td colspan="1" rowspan="1">Positive (T-line Reactive): HSV DNAdetected</td></tr><tr><td colspan="1" rowspan="1">T-/C+</td><td colspan="1" rowspan="1">Negative (T-line Non-Reactive; C-lineReactive): no HSV DNA detected</td></tr><tr><td colspan="1" rowspan="1">T-/C-</td><td colspan="1" rowspan="1">Invalid: process or test failure  repeatassay</td></tr></table>

Note: Read the test and control lines after 15 minutes (but no longer than 60 minutes).

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The Precision/Reproducibility of the IsoAmp® HSV Assay was evaluated at three (3) test sites. A panel of seven (7) members was prepared containing one negative control sample (HSV negative pooled swab specimens) and six stimulated HSV-1 and HSV-2 samples that included High Negative, Low Positive ( $\mathrm { ~ 1 ~ x ~ L O D }$ , near the assay limit of detection) and Moderate positive samples $3 \mathrm { \Delta x }$ LOD). The panel along with the external HSV-1 and HSV-2 positive and negative controls (Remel M4 transport media) was tested at each site for five (5) days by two operators with each operator running the panel two times a day using a single lot of the IsoAmp® HSV Assay. One (1) site tested the panel using three (3) lots. Results of the Precision/Reproducibility study for the IsoAmp ${ } ^ { \textregistered } { \mathrm { H S V } }$ Assay at three sites are presented in the table below.

Precision/Reproducibility Study Summary for the IsoAmp® HSV Assay   

<table><tr><td colspan="1" rowspan="3">Category</td><td colspan="6" rowspan="1">LOT</td><td colspan="2" rowspan="3">OverallPercentAgreement</td><td colspan="1" rowspan="3">95%ConfidenceInterval</td></tr><tr><td colspan="2" rowspan="1">Site #1*</td><td colspan="2" rowspan="1">Site #2</td><td colspan="2" rowspan="1">Site #3</td></tr><tr><td colspan="2" rowspan="1">PercentAgreement</td><td colspan="2" rowspan="1">PercentAgreement</td><td colspan="2" rowspan="1">PercentAgreement</td></tr><tr><td colspan="1" rowspan="1">HSV-1HighNegative</td><td colspan="1" rowspan="1">13/60</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">32/100</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">24- 42</td></tr><tr><td colspan="1" rowspan="1">HSV-1LowPositive</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">99/100</td><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">94 - 100</td></tr><tr><td colspan="1" rowspan="1">HSV-1ModeratePositive</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100/100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1">HSV-2HighNegative</td><td colspan="1" rowspan="1">19/60</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">32/100</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">24 - 42</td></tr><tr><td colspan="1" rowspan="1">HSV-2LowPositive</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">96/100</td><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">90 - 98</td></tr><tr><td colspan="1" rowspan="1">HSV-2ModeratePositive</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100/100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1">Negative1</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">99/100</td><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1">HSV-1PositiveControl</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100/100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1">HSV-2PositiveControl</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100/100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1">AssayNegative Control2</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100/100</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">96 - 100</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="9" rowspan="1">*Site#1 tested two additional lots</td></tr></table>

1 Negative pooled serum control 2 Remel M4 transport media

b. Linearity/assay reportable range: N/A c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Internal control

The competitive internal control (IC) consists of plasmid DNA. The IC target sequence is amplified using the same primer set that amplifies the HSV target sequence. The internal sequence of the IC target is different from the HSV target sequence and is detected by an IC-specific probe. After amplification, the IC amplicon-probe complexes are detected as a visible Control line on the Type II BESt Cassette. The IC DNA and probe are pre-mixed in the Amplification Reagent

External Assay (positive and negative) controls

The Assay Positive Controls are HSV-1 or HSV-2 DNA and are intended to monitor reagent and cassette failure. The Assay Negative Control consists of blank viral transport medium and is used to detect reagent or environmental contamination (or carry-over) by either HSV DNA or amplicons. HSV-1 and HSV-2 Positive Control consist of plasmid DNA with the target sequence of HSV-1 and HSV-2 respectively. Plasmid DNA is diluted to 246 copies/ $\mu \mathrm { L }$ for HSV-1 and 492 copies/ $\mu \mathrm { L }$ for HSV-2 in Remel M4 transport medium to make the assay positive controls. The Assay Negative Control consists of blank viral transport medium and is used to detect reagent or environmental contamination (or carry-over) by either HSV DNA or amplicons

# Specimen processing controls

Additional external controls may be tested in accordance with the guidelines or requirements of local, state and/or federal regulations or accreditation organizations. HSV-1 (Catalog Number: 10-110-000) and HSV-2 (Catalog Number: 10-111-000) viruses can be purchased from Advanced Biotechnologies Inc. (Columbia, MD) and they can be used as specimen processing controls with appropriate amount of viral titer.

# d. Detection limit:

The Limit of Detection (LoD) for the IsoAmp ${ } ^ { \mathfrak { P } } \mathrm { H S V }$ Assay was determined using two (2) representative strains of HSV-1 (McIntyre & HF) and HSV-2 (G & MS). The virus strains were serially diluted to five concentrations and tested in replicates of ten (10) using three (3) reagent lots. To confirm the observed LoD, two additional studies were performed. In one, the four (4) representative strains (two (2) HSV-1 and two (2) HSV-2) were diluted to the observed LoD and run in 20 replicates using three (3) reagent lots. In the second study, 20 HSV-1 and 20 HSV-2 clinical isolates with known $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ concentrations were diluted to the LoD and tested in triplicate using a single lot of reagents.

To dilute all representative HSV-1 and HSV-2 strains and clinical isolates (used for LoD Confirmation testing) to the desired concentration, two pools of HSV Negative Matrix were prepared from 112 ELVIS culture HSV negative/ IsoAmp® HSV Assay negative clinical samples obtained from one of the clinical study sites.

Two representative strains of HSV-1 and HSV-2 were cultured and quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ). Each strain was serially diluted using the HSV Negative Matrix pools to make four (4) LoD panels (one for each strain) consisting of five (5) concentration levels of approximately $9 \mathrm { x } , 3 \mathrm { x } , 1 \mathrm { x } , 1 / 3 \mathrm { x }$ and $1 / 9 { \mathrm { ~ x ~ } }$ of the expected LoD level (the expected LoD was generated from original development data) for each virus type.

Each panel member was tested in replicates of 10 on three (3) reagent lots for a combined total of 30 measurements per concentration level per panel. In consideration of the total number of tests, the LoD testing for each panel was performed in several test runs by three different operators in which two replicates at each of the five concentration levels were tested in the same test run. Assay positive and negative controls were included in each test run.

The observed LoD of a HSV strain was determined as the lowest concentration level demonstrating a positive result at $2 9 5 \%$ . Since two (2) strains of HSV-1 and HSV-2 were used, the higher of the concentrations observed were used to define the final LoD.

LoD of HSV-1 McIntyre strain   

<table><tr><td rowspan=1 colspan=1>McIntyre(TCID50/mL)</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>3.3 x 10^$</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.1 x 105</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>3.7 x 104</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>97%</td><td rowspan=1 colspan=1>83.33%</td><td rowspan=1 colspan=1>99.41%</td></tr><tr><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>18/30</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>42.32%</td><td rowspan=1 colspan=1>75.41%</td></tr><tr><td rowspan=1 colspan=1>4.1 x 103</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33%</td><td rowspan=1 colspan=1>19.23%</td><td rowspan=1 colspan=1>51.22%</td></tr></table>

LoD of HSV-1 HF strain   

<table><tr><td rowspan=1 colspan=1>HF (TCID50/mL)</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>3.3 x 105</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.1 x 10</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>3.7 x 104</td><td rowspan=1 colspan=1>28/30</td><td rowspan=1 colspan=1>93%</td><td rowspan=1 colspan=1>78.68%</td><td rowspan=1 colspan=1>98.15%</td></tr><tr><td rowspan=1 colspan=1>1.2 x 104</td><td rowspan=1 colspan=1>19/30</td><td rowspan=1 colspan=1>63%</td><td rowspan=1 colspan=1>45.51%</td><td rowspan=1 colspan=1>78.13%</td></tr><tr><td rowspan=1 colspan=1>4.1 x 103</td><td rowspan=1 colspan=1>9/30</td><td rowspan=1 colspan=1>30%</td><td rowspan=1 colspan=1>16.66%</td><td rowspan=1 colspan=1>47.88%</td></tr></table>

# LoD of HSV-2 G strain

<table><tr><td rowspan=1 colspan=3>G (TCID50/mL)   Positive/Total</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>3.3 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.1 x 104</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=2>3.7 x 103</td><td rowspan=1 colspan=1>26/30</td><td rowspan=1 colspan=1>87%</td><td rowspan=1 colspan=1>70.32%</td><td rowspan=1 colspan=1>94.69%</td></tr><tr><td rowspan=1 colspan=2>1.2 x 10^{3</td><td rowspan=1 colspan=1>14/30</td><td rowspan=1 colspan=1>47%</td><td rowspan=1 colspan=1>30.23%</td><td rowspan=1 colspan=1>63.86%</td></tr><tr><td rowspan=1 colspan=2>4.1 x 10^2</td><td rowspan=1 colspan=1>8/30</td><td rowspan=1 colspan=1>27%</td><td rowspan=1 colspan=1>14.18%</td><td rowspan=1 colspan=1>44.45%</td></tr></table>

LoD of HSV-2 MS strain   

<table><tr><td rowspan=1 colspan=1>MS(TCID50/mL)</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>PositivityRate</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>3.3 x 104</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>1.1 x 104</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.65%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>3.7 x 103</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.30%</td><td rowspan=1 colspan=1>100.00%</td></tr><tr><td rowspan=1 colspan=1>3.3 x 104</td><td rowspan=1 colspan=1>25/30</td><td rowspan=1 colspan=1>83%</td><td rowspan=1 colspan=1>66.44%</td><td rowspan=1 colspan=1>92.66%</td></tr><tr><td rowspan=1 colspan=1>1.1 x 104</td><td rowspan=1 colspan=1>8/30</td><td rowspan=1 colspan=1>27%</td><td rowspan=1 colspan=1>14.18%</td><td rowspan=1 colspan=1>44.45%</td></tr></table>

LoD confirmation:

i. Additional HSV-1 and HSV-2 strain testing: All four (4) representative strains, diluted to the observed LoD for each viral type (HSV-1: McIntyre and HF, HSV$2 \colon \mathbf { G }$ and MS) and tested in replicates of 20 with three (3) validation lots showed a positivity rate of $100 \%$ .

LoD of Additional HSV-1 and HSV-2 strains Testing   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LOD(TCID50/mL)</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>Positivity Rate</td></tr><tr><td rowspan=1 colspan=1>HSV-1 McIntyre</td><td rowspan=1 colspan=1>1.1 x 10^}$</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HSV-1 HF</td><td rowspan=1 colspan=1>1.1 x 105</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 G</td><td rowspan=1 colspan=1>1.1 x 104</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>HSV-2 MS</td><td rowspan=1 colspan=1>1.1 x 104</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td></tr></table>

ii. Clinical Isolate Testing: All twenty (20) HSV-1 and (20) HSV-2 clinical isolates were tested in triplicate. The ${ \mathrm { I s o A m p } } ^ { \mathrm { \textregistered } } { \mathrm { H S V } }$ Assay was able to detect all 20 HSV-1 and 20 HSV-2 clinical isolates. One of the HSV-2 clinical isolates was HSV negative in one of the three (3) replicates; however, when the isolate was tested at $3 \mathbf { x }$ and $9 \mathbf { x }$ LoD, all three replicates were HSV-Positive by the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay.

The observed LoD for HSV-1 was $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The observed LoD for HSV-2 was $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 4 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Since the $\mathrm { I s o A m p } ^ { \mathbb { B } } \mathrm { H S V }$ Assay does not differentiate viral types, the final assay LoD is defined as the higher of the HSV-1 and HSV-2 concentrations where $9 5 \%$ positivity was observed. The final assay LoD claim is $1 . 1 \mathrm { ~ x ~ } 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

# e. Analytical specificity:

Cross Reactivity: A cross-reactivity study was performed to determine if any organisms which may present with the same clinical symptoms as HSV, which are associated with bacterial vaginosis or which are commonly found in the

genital track and oral area could give positive results with the IsoAmp® HSV Assay reporting accurate results. Forty-eight (48) specificity panel members including purified DNA and cultured organisms were tested with the IsoAmp® HSV Assay in triplicate.

Three (3) methods were used to prepare the test organisms: (1) Genomic DNA [GD] purified and quantified, (2) Quantified Cultures (QC) [bacterial/fungal (CFU/mL), viral $\mathrm { ( T C I D _ { 5 0 } / m L ) } ]$ , and (3) In-House Cultures (IHC) cultured and diluted to a bacterial concentration of $\mathrm { 1 0 ^ { 8 } C F U / m L }$ . Each organism or purified genomic DNA was tested in triplicate at $1 . 0 \mathrm { { x } 1 0 ^ { 7 } \mathrm { { C F U / m L } } }$ (copies/mL) for bacterial and fungal organisms and at $1 . 0 \mathrm { { x } 1 0 ^ { 6 } { p f u } / { m L } }$ (copies/mL) for viruses (in Remel M4 viral transport medium).

No cross-reactivity was observed with any panel member tested at clinically significant concentrations.

Cross Reactivity Panel   

<table><tr><td colspan="1" rowspan="1">Organisms</td><td colspan="1" rowspan="1">Member Type(GD, QC, IHC)</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter calcoaceticus var.anitratus (ATCC 51432)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Acinetobacter lwoffi (ATCC 17925)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 2</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida albicans (ATCC 14053)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida glabrata</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida guilliermondii</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida krusei</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida lusitaniae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida parapsilosis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Candida tropicalis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis LGV-II434</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.0 x 107 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae (ATCC 13047)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus (Type 71)</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.0 x 106 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (ATCC 25922)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Fusobacterium nucleatum (ATCC25586)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">Gardnerella vaginalis (ATCC 14018)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus ducreyi</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">8.5 x 105 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 6 virus (Z29 strain)</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Herpes 7 virus (SB strain)</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human papilloma virus 16 (HPV16)</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.0 x 106 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Human papilloma virus 18 (HPV18)</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.0 x 105 cp/mL</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus Z0481</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus curtisii V125 [DSM 2711]</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mobiluncus mulieris BV 64-5</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis (ATCC 23114)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae (ATCC 21823)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Rubella virus</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">4.17 x 105TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Simian Virus type 40 (SV40)</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MRSA (ATCC33591)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus MSSA (ATCC25923</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis MRSE(ATCC700566)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus saprophyticus MRSE(ATCC 15305)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mitis clinical isolate2</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus mutans Z0723</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes: (ATCC19615)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius (ATCC BAA-1024)</td><td colspan="1" rowspan="1">IHC</td><td colspan="1" rowspan="1">1.0 x 107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Toxoplasma gondii</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">6.6 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Treponema pallidum</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 107 TP/mL</td></tr><tr><td colspan="1" rowspan="1">Trichomonas vaginalis</td><td colspan="1" rowspan="1">QC</td><td colspan="1" rowspan="1">1.0 x 106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Varicella-Zoster Virus (VZV)</td><td colspan="1" rowspan="1">GD</td><td colspan="1" rowspan="1">1.0 x 106 cp/mL</td></tr></table>

# f. Interference studies:

Potential interfering substances i. e. viral transport media, substances that might be present in clinical samples and organisms/cross reactive panel members listed under analytical specificity that might be carried over from patient samples and co-infection of HSV-1 and HSV-2 were tested to confirm that they did not interfere with the performance of the IsoAmp® HSV Assay.

All interference testing was carried out in the presence of HSV-1 and HSV-2 at three times the observed LoD (3x LoD). HSV-1 HF and HSV-2 MS strains were used. All test runs were conducted in triplicate. Controls were tested with each run.

# i. Interfering Substances

Performance of the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay was characterized in the presence of twenty-four (24) potential interfering substances which could reasonably be expected to be present in genital and oral swab specimens. Interfering substances were tested at the highest (“worst case”) concentration expected in clinical samples. Each interfering substance was introduced into the assay by directly wetting a clean, dry Remel M4 kit swab with the substance and placing the swab directly in transport media. Calculated concentrations are based on an estimated volume of $2 0 0 \mu \mathrm { L }$ of substance introduced by the swab. Each panel member was tested in triplicate spiked with HSV-1 HF and HSV-2 MS strains separately at $3 \mathrm { ~ x ~ }$ LoD. The panel was also tested in triplicate in the absence of HSV transport media to see if the potentially interfering substances interfere with the detection of the internal control. No interference was observed in the presence of the potential interfering substances tested.

Interfering Substances Panel   

<table><tr><td colspan="1" rowspan="1">Substances (active ingredients)</td><td colspan="1" rowspan="1">Calculated Concentration</td></tr><tr><td colspan="1" rowspan="1">Whole blood with EDTA</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Female Urine</td><td colspan="1" rowspan="1">7%(v/v)</td></tr><tr><td colspan="1" rowspan="1">Male Urine</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Acyclovir (Acycloguanosine) 10%</td><td colspan="1" rowspan="1">7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Albumin</td><td colspan="1" rowspan="1">3.3 mg/mL</td></tr><tr><td colspan="1" rowspan="1">Casein</td><td colspan="1" rowspan="1">7 mg/mL</td></tr><tr><td colspan="1" rowspan="1">K-Y Brand Jelly</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Douche (Decyl Glucoside; Octoxynol-9)</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Contraceptive Jelly</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">YeastGard (Phosphoricum Acidum 4X)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Monistat 1 (Miconazole Nitrate cream (2%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Vagisil Crème (Benzocaine (20%), Resorcinol(3%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Monistat 3 (Miconazole Nitrate Cream (4%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Triconazole 1 (Tioconazole (300 mg) (6.5%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Balneol Hygienic Cleansing Lotion</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Clotrimazole 3 Vaginal Cream (Clotrimazole 100mg (2%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">CVS Anti-Itch Cream (Benzocaine 5%;Benzalkonium Chloride 0.13%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Listerine Antiseptic Mouth Wash</td><td colspan="1" rowspan="1">7% (v/v)</td></tr><tr><td colspan="1" rowspan="1">Abreva (Docosanol 10%)</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Carmex Cold Sore Lip Balm (Menthol (0.7%),Camphor (1.7%), Phenol (0.4%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Releev cold sore treatment (BenzalkoniumChloride (0.13%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Lip clear Lysine+ (Zinc Oxide (1.2%))</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Toothpaste</td><td colspan="1" rowspan="1">7% (w/v)</td></tr><tr><td colspan="1" rowspan="1">Buffy coat</td><td colspan="1" rowspan="1">7% (v/v)</td></tr></table>

# ii. Viral Transport Media

The performance of the IsoAmp ${ } ^ { \mathfrak { P } } \mathrm { H S V }$ Assay was assessed with Remel M4, Remel M5, Remel M4RT, Bartels VTM, and BD Universal Viral Transport (UVT). Each medium was tested after spiking with HSV-1 HF and HSV-2 MS strain to a final concentration of approximately $3 \mathrm { ~ x ~ L o D }$ to determine if the viral transport media interfere with the detection of HSV targets in positive samples. The media were tested in the absence of HSV-1 and HSV-2 (medium only) to see if the viral transport media interfere with the detection of the internal control in negative samples. There was no interference observed with the Remel M4, Remel M4RT, Remel M5, Bartels VTM, and BD UVT media for the detection of HSV-1 and HSV- 2 target or the internal control.

# iii. Specificity/Cross Reactivity Panel Members

The performance of the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay was characterized by testing the organisms that were evaluated for analytical specificity and cross reactivity in the presence of HSV-1 HF and HSV-2 MS at 3xLoD separately to see if the presence

of these organisms interferes with the detection of HSV target. Each panel member was tested in triplicate. None of the cross reactivity panel members interfered with the detection of HSV-1 and HSV-2 target.

# g. Carry-Over/Cross Contamination:

Carry-over/Contamination studies Study was done only with HSV-1 target since both HSV-1 and HSV-2 share a single set of primers and probes for target amplification and detection. The HSV-1 McIntyre $\mathrm { 6 . 6 5 \times 1 0 ^ { 8 } T C I D _ { 5 0 } / \bar { m } L } \mathrm { ) }$ was used directly without dilution. Remel M4 viral transport media was used as the negative sample. Ten (10) replicates of negative sample together with assay controls were run by two (2) operators to confirm that negative samples (Remel M4 viral transport media) generate a negative result $100 \%$ of the time. Five (5) replicates of high-concentration positive and negative samples were tested in a series, alternating sample types. All results were as expected. Negative samples tested were negative (10/10) and positive samples were positive (10/10).

# h. Sample Stability:

Sample stability testing was done to confirm the stability of HSV-1 and HSV-2 in four different viral transport media, Remel M4, Remel M5, Remel M4RT and BD UVT. The media were spiked with HSV-1 or HSV-2 at $3 \mathrm { ~ x ~ }$ LoD, and stored at 2 - $8 \mathrm { ^ \circ C }$ for one week. The $\mathrm { I s o A m p } ^ { \textregistered } \mathrm { H S V }$ Assay was performed with these spiked samples every day. Each run consisted of 3 replicates with each medium, as well as assay negative and positive controls. All the samples were shown stable in all viral transport media for 7 days, supporting the claim for 5 day sample stability when stored at $2 - 8 \mathrm { { ^ \circ C } }$ .

# 2. Comparison studies:

a. Method comparison with reference method:

The clinical performance evaluation was done against a gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

For additional details please see section 3 Clinical studies subsection c.

b. Matrix Comparison : N/A

3. Clinical studies:

a. Clinical Sensitivity: N/A b. Clinical specificity: N/A

The FDA cleared MultiCode $\textsuperscript { \textregistered }$ -RTx HSV 1&2 assay was used as the Predicate device. The performance of the IsoAmp® HSV Assay was compared with the ELVIS $\textsuperscript { \textregistered }$ HSV ID/Typing Test System (Diagnostic Hybrid, Inc.) which is the gold standard/reference method i.e., Cell Culture using an enzyme linked virus inducible system with HSV typing by fluorescently labeled antibodies.

# Clinical Performance

The performance of the IsoAmp® HSV Assay was evaluated at five geographically diverse locations within the United States from 2010 - 2011. A total of nine hundred ninety-four (994) swab samples were evaluated from male and female genital and oral lesions collected in Viral Transport Media (Remel M4, Remel M4RT, BD Universal Viral Transport and Bartels) from the patient population ranging from $< 1$ year to 92 years. Of the 994 specimens, there were 962 prospective and 32 retrospective samples. Of the 962 prospective samples, 803 genital and 159 oral samples were tested. Of the 32 retrospective samples, 15 genital and 17 oral samples were tested at a single study site. Female and male genital swab specimens were collected from vaginal, labial, and penile lesions. Oral swab specimens were collected from lips, gums, and mouth.

The performance of the IsoAmp® HSV Assay was compared with the reference method which is the gold standard/reference method i.e., cell culture using an enzyme linked virus inducible system. Quality Controls (HSV-1 positive, HSV-2 positive and HSV negative) were run on the IsoAmp ${ } ^ { \textregistered } { \mathrm { H S V } }$ Assay.

# Overall Prospective - Genital Samples

<table><tr><td rowspan=2 colspan=2>GENITAL SAMPLES</td><td rowspan=1 colspan=3>Reference Method</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IsoAmp®HSV Assay</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>264</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>299</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>496</td><td rowspan=1 colspan=1>504</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>272</td><td rowspan=1 colspan=1>531</td><td rowspan=1 colspan=1>803</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=3>95%Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.1%(264/272)</td><td rowspan=1 colspan=3>94.3 - 98.5%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>93.4%(496/531)</td><td rowspan=1 colspan=3>91.0 - 95.2%</td></tr></table>

1 Thirty five (35) samples were tested using bidirectional sequencing analysis. Sequence analysis detected HSV target in 29 of the 35 discordant samples (6 HSV-1, 23 HSV-2) identified as HSV Positive by the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay. Sequence analysis did not detect HSV in six (6) of the discordant samples.

2 Eight (8) samples were tested using bidirectional sequencing analysis. Sequence analysis did not detect HSV target in four (4) of the 8 samples identified as HSV Negative by the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay. Sequence analysis did detect HSV in four (4) samples (2 HSV-1, 2 HSV-2).

# Overall Prospective - Oral Samples

<table><tr><td rowspan=2 colspan=2>ORAL SAMPLES</td><td rowspan=1 colspan=3>ReferenceMethod</td></tr><tr><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=3 colspan=1>IsoAmp®HSV Assay</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>143</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>159</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Value</td><td rowspan=1 colspan=3>95% Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>93.8%(45/48)</td><td rowspan=1 colspan=3>83.2 - 97.9%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>87.4%(97/111)</td><td rowspan=1 colspan=3>79.9 - 92.3%</td></tr></table>

3 Fourteen (14) samples were tested using bidirectional sequencing analysis. Sequence analysis detected HSV target in 13 of the 14 discordant samples (12 HSV-1) identified as HSV Positive by the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay. Sequence analysis did not detect HSV in one (1) of the discordant samples.

4 Three (3) samples were tested using bidirectional sequencing analysis. Sequence analysis did not detect HSV target in two (2) of the 3 samples identified as HSV Negative by the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay. Sequence analysis did detect HSV in one (1) of the discordant samples.

All of the 32 retrospective samples, 15 genital and 17 oral samples were shown positive by both the IsoAmp® HSV Assay and the reference assay.

4. Clinical cut-off: N/A

5. Expected values/Reference range:

The prevalence of HSV-1 and HSV-2 in genital and oral swab specimens during the multi-site clinical study $\scriptstyle ( \mathrm { n } = 9 6 2 ^ { * }$ ) was estimated using the IsoAmp $\textsuperscript { \textregistered }$ HSV Assay. \* Note: Retrospective samples were not included in this tabulation.

IsoAmp $^ \mathrm { \textregistered }$ Distribution of Prospective Population by Age Group:   
Genital Lesion Swab Specimens   

<table><tr><td colspan="1" rowspan="1">Age Range</td><td colspan="1" rowspan="1">IsoAmpHSV AssayPositive</td><td colspan="1" rowspan="1">Total Number ofSpecimens</td></tr><tr><td colspan="1" rowspan="1">&lt;1 to 17years</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">58</td></tr><tr><td colspan="1" rowspan="1">18 to 25years</td><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">268</td></tr><tr><td colspan="1" rowspan="1">26 to30years</td><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">110</td></tr><tr><td colspan="1" rowspan="1">31 to 35years</td><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">83</td></tr><tr><td colspan="1" rowspan="1">36 to40years</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">68</td></tr><tr><td colspan="1" rowspan="1">41 to45years</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">56</td></tr><tr><td colspan="1" rowspan="1">46 to50years</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">51 to 55 years</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="1">56 to 60 years</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">21</td></tr><tr><td colspan="1" rowspan="1">61 to 65 years</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">23</td></tr><tr><td colspan="1" rowspan="1">66 to 70 years</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">71 to 75 years</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">76 to80 years</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">81 to 85 years</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">6</td></tr><tr><td colspan="1" rowspan="1">86 to 90 years</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">90 to 95 years</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">299</td><td colspan="1" rowspan="1">803</td></tr><tr><td colspan="1" rowspan="1">Prevalence</td><td colspan="1" rowspan="1">37.2%</td><td colspan="1" rowspan="1">N/A</td></tr></table>

IsoAmp $^ \mathrm { \textregistered }$ Distribution of Prospective Population by Age Group: Oral Lesion Swab Specimens   

<table><tr><td rowspan=1 colspan=1>Age Range</td><td rowspan=1 colspan=1>IsoAmp® HSV AssayPositive</td><td rowspan=1 colspan=1>Total Number ofSpecimens</td></tr><tr><td rowspan=1 colspan=1>&lt;1 to 17 years</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>18 to 25years</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>31</td></tr><tr><td rowspan=1 colspan=1>26 to 30years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>31 to35years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>36 to40years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>41 to 45years</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>46 to50years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>51 to55years</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>56 to60years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>61 to65years</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>66 to70years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>71 to75years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>76 to80years</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>81 to85years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>86 to90years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>90 to 95 years</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>159</td></tr><tr><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>37.1%</td><td rowspan=1 colspan=1>N/A</td></tr></table>

The combined prevalence was used to calculate the hypothetical positive predictive values (PPV) and hypothetical negative predictive values (NPV) of the $\mathrm { \Delta I s o A m p ^ { \textregistered } H S V }$ Assay. The calculations are based on the sensitivity and specificity obtained from the clinical studies: sensitivity of $9 7 . 1 \%$ and specificity of $9 3 . 4 \%$ for genital lesion samples; sensitivity of $9 3 . 8 \%$ and specificity of $8 7 . 4 \%$ for oral lesion samples. The prevalence observed by a laboratory may vary and the distribution in the table below may be used to establish the frequency distributions based on a specific laboratory’s patient population.

Prevalence vs. Hypothetical Predictive Values Genital Lesion Swab Specimens   

<table><tr><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>PositivePredictiveValue (PPV)</td><td rowspan=1 colspan=1>NegativePredictive Value(NPV)</td></tr><tr><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>93.6%</td><td rowspan=1 colspan=1>97.0%</td></tr><tr><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>90.7%</td><td rowspan=1 colspan=1>98.0%</td></tr><tr><td rowspan=1 colspan=1>30%</td><td rowspan=1 colspan=1>86.3%</td><td rowspan=1 colspan=1>98.7%</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>78.6%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>62.0%</td><td rowspan=1 colspan=1>99.7%</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>43.6%</td><td rowspan=1 colspan=1>99.8%</td></tr></table>

Prevalence vs. Hypothetical Predictive Values Oral Lesion Swab Specimens   

<table><tr><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>PositivePredictiveValue(PPV)</td><td rowspan=1 colspan=1>NegativePredictive Value(NPV)</td></tr><tr><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>88.2%</td><td rowspan=1 colspan=1>93.4%</td></tr><tr><td rowspan=1 colspan=1>40%</td><td rowspan=1 colspan=1>83.2%</td><td rowspan=1 colspan=1>95.5%</td></tr><tr><td rowspan=1 colspan=1>30%</td><td rowspan=1 colspan=1>76.1%</td><td rowspan=1 colspan=1>97.0%</td></tr><tr><td rowspan=1 colspan=1>20%</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>98.3%</td></tr><tr><td rowspan=1 colspan=1>10%</td><td rowspan=1 colspan=1>45.3%</td><td rowspan=1 colspan=1>99.2%</td></tr><tr><td rowspan=1 colspan=1>5%</td><td rowspan=1 colspan=1>28.2%</td><td rowspan=1 colspan=1>99.6%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.