# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k111775   
B. Purpose for Submission: New device   
C. Measurand: Target RNA sequence within the Matrix protein gene (segment 7) of Influenza A viruses, target RNA sequence within the non-structural protein gene (segment 8) of Influenza B viruses, and target sequence within the human gene encoding ribonuclease P (RNase P).   
D. Type of Test: A real-time reverse-transcriptase polymerase chain reaction (Real-time RT-PCR) test intended for the qualitative in vitro detection and differentiation of Influenza A and Influenza B in nasopharyngeal swab (NPS) and nasopharyngeal wash (NPW) specimens using nucleic acid isolation, amplification, and detection.   
E. Applicant: US Army Office of the Surgeon General, U.S. Army Medical Materiel Development Activity   
F. Proprietary and Established Names: JBAIDS Influenza A & B Detection Kit Common Name: JBAIDS Influenza A & B rRT-PCR Kit Real-time PCR assay for detection of Influenza A and Influenza B

# G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>OCC</td><td>Class II</td><td>21 CFR 866.3980 Respiratory Viral Panel Multiplex Nucleic Acid Assay</td><td>Microbiology (83)</td></tr><tr><td>0O1</td><td>Class II</td><td>21 CFR 862.2570 Instrumentation for clinical multiplex test systems</td><td>Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The Joint Biological Agent Identification and Diagnostic System (JBAIDS) Influenza A & B Detection Kit is intended for use on the JBAIDS instruments, for the in vitro qualitative detection of Influenza A and Influenza B viral nucleic acids isolated and purified from nasopharyngeal swab (NPS) and nasopharyngeal wash (NPW) specimens from human patients with signs and symptoms of respiratory infection. The JBAIDS Influenza A & B Detection Kit contains reverse transcriptase real-time polymerase chain reaction (rRT-PCR) assays that target the Matrix protein gene of Influenza A viruses, and the Non-structural protein gene of Influenza B viruses. This kit is not intended to detect Influenza C viruses.

Test results are to be used in conjunction with other clinical and epidemiological information. Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

Performance characteristics for detection of influenza A were established when 2009 H1N1 Influenza, Influenza A H1N1, and Influenza A H3N2 were the predominant influenza A viruses in circulation. Due to low seasonal prevalence, performance characteristics for detection of seasonal Influenza A/H1 were established primarily with retrospective and surrogate clinical specimens. When other influenza A viruses are present, performance characteristics may vary.

All users, analysts, and any person reporting diagnostic results from use of this device should be trained to perform and interpret the results from this procedure by JBAIDS instructors or designees prior to use. Use of this device is limited to designated Department of Defense (DoD) laboratories equipped with the JBAIDS instruments.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a biosafety laboratory (BSL) $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument

# I. Device Description:

The JBAIDS Influenza A & B Detection Kit is a real time RT-PCR assay based on the influenza A and Influenza B assays that are part of the “CDC Human Influenza Virus Realtime RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT 1-2-3 Platinum Path Purification Kit or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in conjunction with the Roche MagNA Pure Compact system) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instrument.

Each JBAIDS Influenza A & B Detection Kit contains sufficient reagents for testing 20 specimens. There are eight vacuum-sealed control pouches of freeze-dried PCR reagents (4 reagent vials each), 20 vacuum-sealed sample testing pouches of freeze-dried PCR reagents (3 reagent vials each), and eight pouches containing one tube each of reconstitution buffer and reagent grade water. Once resuspended, each reagent vial provides enough material for one reaction.

Each control pouch contains the following four vials:

Positive Control $( + )$ vials (Flu A and Flu B). Once reconstituted, each Positive Control vial contains all reagents necessary for RT-PCR of the appropriate target, including an assay-specific synthetic RNA target. The Positive Controls are resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens. Amplification in the Positive Control gives assurance that kit reagents are functioning properly and that the assay setup has been performed correctly. A Positive Control must be included for each assay in the JBAIDS run.

Ingredients: Each Positive Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex;   
$< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ target-specific forward and reverse primer;   
$< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP;   
$< 0 . 0 0 1 \%$ assay-specific synthetic target RNA; bovine serum albumin; dithiothreitol;   
RNase inhibitor; and carbohydrate.

Negative Control $( - )$ vials (Flu A and Flu B). Once reconstituted, each Negative Control vial contains all reagents required to perform a RT-PCR reaction except the target RNA, and should therefore give negative results. The Negative Controls are resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 \mu \mathrm { L }$ nuclease-free water that has been purified with a nucleic acid purification kit (NEC). The reconstituted negative control is then tested in parallel with patient specimens to provide assurance that the purification and setup procedures have been performed without contamination. A Negative Control must be included for each assay in the JBAIDS run.

Ingredients: Each Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex;   
$< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ target-specific forward and reverse primer;   
$< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP;   
bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Each sample testing pouch contains the following three vials:

Unknown vials (Flu A and Flu B). Once reconstituted, each Unknown vial contains all reagents necessary for RT-PCR of the appropriate target, except target template. The Unknown vials are resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ purified patient sample, and will therefore give negative results unless the appropriate template is found in the patient sample.

Ingredients: Each Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex;   
$< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ target-specific forward and reverse primer;   
$< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP;   
bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Flu SC vial. Once reconstituted, the Flu SC vial contains reagents that amplify and detect the human genomic RNase $\mathrm { P }$ sequence. The target sequence consists of RNase $\mathrm { P }$ -specific primer binding sites and an RNase P-specific probe binding site. The Flu SC is resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ purified patient sample to ensure patient nucleic acid is available in the sample, and that RT-PCR-inhibiting substances in the test specimen are detected, thereby preventing false negative results.

Ingredients: Each Flu SC vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$   
Reverse Transcriptase; $< 0 . 0 0 1 \%$ RNase P-specific forward and reverse primer;   
$< 0 . 0 0 1 \%$ RNase P-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP;   
bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Each reconstitution buffer/reagent grade water pouch contains the following two vials:

2X Reconstitution Buffer (2X RB). The buffer solution (purple buffer) is matched to the Influenza assays and is used undiluted. The buffer enhances PCR kinetics and is used to re-suspend JBAIDS freeze-dried reagents. Once reconstituted, all freeze dried reagent vials contain all of the components required for PCR.

Contents: 600 μL

Reagent Grade Water. Reagent grade water is molecular biology grade water used to reconstitute the Positive Controls.

Contents: 850 μL

# Materials Provided

<table><tr><td colspan="3" rowspan="1">Kit ContentsAll vials in the assay pouches contain sufficient reagent for one capillary reaction.</td></tr><tr><td colspan="1" rowspan="1">Qty</td><td colspan="1" rowspan="1">Description</td><td colspan="1" rowspan="1">Contents</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Influenza A &amp; B DetectionKit Control Pouch</td><td colspan="1" rowspan="1">One Flu A Positive Control (+)One Flu A Negative Control (-)One Flu B Positive Control (+)One Flu B Negative Control (-)</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Influenza A &amp; B DetectionOne Flu A Unknown (U)Kit Sample Pouch</td><td colspan="1" rowspan="1">Influenza A &amp; B DetectionOne Flu A Unknown (U)One Flu B Unknown (U)One Flu Sample Control (SC)</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Purple ReconstitutionBuffer Pouch(purple foil pouch)</td><td colspan="1" rowspan="1">One purple reconstitution buffer (600 µL)One reagent grade water (850 μL)</td></tr><tr><td colspan="3" rowspan="1">Each kit contains sufficient reagents for testing 20 specimens.</td></tr></table>

# Materials Required But Not Provided

<table><tr><td rowspan=1 colspan=2>Required Equipment</td></tr><tr><td rowspan=1 colspan=1>Minicentrifuge capable of 2000 x g, 52151(Labnet C-1200 or equivalent)</td><td rowspan=1 colspan=1>Capillary adaptor for minicentrifuge (RocheApplied Science 1750-1.51 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>Micropipette: 2 μL20 μLMicropipette: 20 μL200 μLMicropipette: 200 μL−1000 μL</td><td rowspan=1 colspan=1>Vortex-Genie® 38231(VWR 58810-163 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>JBAIDS Instrument</td><td rowspan=1 colspan=1>Extra sample carousel JRPD-SUB-00101</td></tr><tr><td rowspan=1 colspan=1>Roche MagNA Pure Compact Instrument2</td><td rowspan=1 colspan=1>PickPen 1-M Magnetic Tool3 (Bio-Nobile)</td></tr><tr><td rowspan=1 colspan=2>Note: See appropriate Sample Purification Kit for additional equipment and materials requiredfor sample purification.</td></tr></table>

1 Available from Idaho Technology 2 Only required when performing automated sample purification with the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I 3 Only required when performing manual purification using the IT 1-2-3 Platinum Path Sample Purification Kit

<table><tr><td rowspan=1 colspan=2>Materials and Reagents Not Provided</td></tr><tr><td rowspan=1 colspan=1>LightCycler glass capillaries and caps(Roche Applied Science 1 909 339) 1</td><td rowspan=1 colspan=1>Aerosol-resistant (filter), nuclease-free pipettetips, appropriate for micropipettes</td></tr><tr><td rowspan=1 colspan=1>Microcentrifuge tube rack LABS-SUP-0001&#x27; Powder-free latex or nitrile gloves or equivalent</td><td rowspan=1 colspan=1>Microcentrifuge tube rack LABS-SUP-0001&#x27; Powder-free latex or nitrile gloves or equivalent</td></tr><tr><td rowspan=1 colspan=1>PickPen Tips3 (Bio-Nobile)</td><td rowspan=2 colspan=1>Sodium hypochlorite solution (household bleach)</td></tr><tr><td rowspan=1 colspan=1>Molecular biology grade water (SigmaW4502 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>DNAZapTM, (Ambion AM9890 or equivalent IT 1-2-3 Platinum Path Sample Purification KitDNA degradation solution)</td><td rowspan=1 colspan=1>DNAZapTM, (Ambion AM9890 or equivalent IT 1-2-3 Platinum Path Sample Purification Kit(ASAY-ASY-0120)1,.3orRoche MagNA Pure Compact Nucleic AcidIsolation Kit I2</td></tr></table>

1 Available from Idaho Technology 2 Only required when performing automated sample purification with the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I. Only qualified lots of the MagNA Pure Compact Nucleic Acid Isolation Kit I can be used with the JBAIDS Influenza A & B Detection Kit. Qualified Roche MagNA Pure Compact Nucleic Acid Isolation Kit I lot numbers are posted and viewable on the secure JBAIDS website. 3 Only required when performing manual purification using the IT 1-2-3 Platinum Path Sample Purification Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s): CDC rRT-PCR Flu Panel

2. Predicate k number(s): K080570

3. Comparison with predicate(s):

<table><tr><td rowspan=1 colspan=3>Similarity</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>JABIDS Influenza A &amp; BDetection Kit</td><td rowspan=1 colspan=1>CDC rRT-PCR Flu Panel(K080570)</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Real-time PCR using hydrolysis Real-time PCR using hydrolysisprobes</td><td rowspan=1 colspan=1>Real-time PCR using hydrolysis Real-time PCR using hydrolysisprobes</td></tr><tr><td rowspan=1 colspan=1>Viruses Detected</td><td rowspan=1 colspan=1>Qualitative in vitro detection ofInfluenza A and Influenza BRNA</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidExtraction</td><td rowspan=1 colspan=1>Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td><td rowspan=1 colspan=1>SameSee below for differences</td></tr></table>

<table><tr><td colspan="3" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">JABIDS Influenza A &amp; B DetectionKit</td><td colspan="1" rowspan="1">CDC rRT-PCR Flu Panel(K080570)</td></tr><tr><td colspan="1" rowspan="1">Viruses DetectedI</td><td colspan="1" rowspan="1">Does not subtype Influenza A</td><td colspan="1" rowspan="1">Differentiation of Influenza A H1,A/H3 and A/H5 (Asian lineage)</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabs,nasopharyngeal washes</td><td colspan="1" rowspan="1">Upper respiratory tract specimens(including NPS, NS, TS, NA, NW,[NPS/TS) and lower respiratory tractspecimens (including BAL, BW,TA, sputum, and lung tissue) andvirus culture</td></tr><tr><td colspan="1" rowspan="1">RequiredInstrumentation</td><td colspan="1" rowspan="1">JBAIDS instrument</td><td colspan="1" rowspan="1">Applied Biosystems 7500 Fast DxReal-time PCR instrument with SDSsoftware v 1.4</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofTest Results</td><td colspan="1" rowspan="1">Automated analysis of testresults and controls</td><td colspan="1" rowspan="1">User required to interpret test andcontrol results</td></tr><tr><td colspan="1" rowspan="1">Enzyme MasterMix</td><td colspan="1" rowspan="1">Assays come in freeze-driedsingle use vials that include allcomponents of master mix</td><td colspan="1" rowspan="1">Invitrogen SuperScriptTM IIIPlatinum® One- Step QuantitativeRT-PCR Kits</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Reagents are stored at roomtemperature</td><td colspan="1" rowspan="1">Reagents are stored at &lt; -15°C</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethods</td><td colspan="1" rowspan="1">•IT 1-2-3TM Platinum PathSample Purification Kit•Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td><td colspan="1" rowspan="1">QIAamp ®Viral RNA Mini Kit,QIAGEN RNeasy® Mini Kit, orRoche MagNA Pure TNA Kit</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens, FDA Guidance Document (DRAFT: December 8, 2005) Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses, FDA Guidance Document (DRAFT: February 15, 2008) Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline, MM3-P2 (February 2006) Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005) User Verification of Performance for Precision and Trueness, CLSI Approved Guideline EP15-A2 (April 2006, second printing) Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline, CLSI Approved Guideline MM13-A (January 2006) • User Protocol for Evaluation of Qualitative Test Performance; Approved GuidelineSecond Edition, CLSI Approved Guideline EP12-A2 (January 2008) Protocols for Determination of Limits of Detection and Limits of Quantization, CLSI Approved Guidance EP17-A (2004) Evaluation of Precision Performance of Quantitative Measurements Methods; Approved Guidance-Second Edition, CLSI Approved Guidance EP5-A2 (August 2004) In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Pa Guidance for Industry and FDA Staff (May 1, 2007). Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses, Guidance for Industry and FDA Staff (March 22, 2006).

# L. Test Principle:

The JBAIDS Influenza A & B Detection Kit is a real time RT-PCR assay based on the influenza A and Influenza B assays that are part of the “CDC Human Influenza Virus Realtime RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT 1-2-3 Platinum Path Purification Kit or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in

conjunction with the Roche MagNA Pure Compact system) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instrument.

Real-time RT-PCR involves reverse transcription of specific RNA sequences into complementary DNA sequences, followed by logarithmic amplification and simultaneous detection of those DNA sequences. The JBAIDS Influenza A & B Detection Kit assays use hydrolysis probes to detect amplification of the reverse transcribed RNA sequence of interest. Each hydrolysis probe is labeled on $5 ^ { \circ }$ end with a fluorescent reporter moiety (6- FAM) and elsewhere with a quencher (TAMRA), which prevents the probe from emitting fluorescent signal. During PCR, the probe binds to a target sequence in the PCR product. When the Taq polymerase replicates a template to which a hydrolysis probe is bound, the exonuclease activity of the polymerase cleaves the probe, separating the fluorophore from the quencher, and fluorescent signal is generated. This fluorescence is measured and displayed by the JBAIDS instrument during the PCR reaction. The fluorescent signal increases as additional template is amplified and more probes are hydrolyzed.

The JBAIDS Influenza A & B Detection Kit contains three assays: Influenza A (Flu A); Influenza B (Flu B); and the RNase P Sample Control Assay (Flu SC). The Flu A assay amplifies a sequence within the matrix protein gene (segment 7) of influenza A viruses. The Flu B assay amplifies a sequence within the non-structural protein gene (segment 8) of influenza B viruses. The Flu SC assay amplifies a sequence within the human gene encoding ribonuclease P (RNase P).

The JBAIDS Influenza A & B Detection Kit is based on four major processes:

• Sample collection and purification Reconstitution of freeze-dried reagents Amplification and detection of target nucleic acid on the JBAIDS instrument   
• Automated interpretation of the amplification curves and report generation using the   
JBAIDS software.

The JBAIDS Influenza A & B Detection Kit can be used to test NPS and NPW specimens. Samples must be purified prior to testing with the JBAIDS Influenza A & B Detection Kit. The purpose of sample purification is to release nucleic acid contained in the patient sample and to remove extraneous materials (e.g., proteins and chemicals) that can interfere with the PCR reaction. The JBAIDS Influenza A & B Detection Kit has been validated for use with either the IT 1-2-3 Platinum Path Sample Purification Kit or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact system. The Platinum Path kit is a manual purification, while the Roche MagNA Pure system is automated. Both methods have four basic steps: lysis to release the nucleic acids, binding of the nucleic acids to a magnetic substrate, washing away extraneous materials, and elution of the purified nucleic acid. A brief outline of the two purification kit methods is provided below:

Method Summary for Purification Kits   

<table><tr><td rowspan=1 colspan=1>Step</td><td rowspan=1 colspan=1>IT 1-2-3 Platinum Path SamplePurification Kit</td><td rowspan=1 colspan=1>Roche MagNA Pure Compact NucleicAcid Isolation Kit I*</td></tr><tr><td rowspan=1 colspan=1>Cell Lysis to releasenucleic acids</td><td rowspan=1 colspan=1>Bead-beating (vortexing specimen in thepresence of silica beads and binding buffer)is used to physically disrupt the cells.</td><td rowspan=1 colspan=1>The samples are lysed by incubation withProteinase K and a special lysis buffercontaining a chaotropic salt.</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidBinding</td><td rowspan=1 colspan=1>The bead-beaten sample is transferred to awell of a strip tube containing magneticbeads. The nucleic acid binds to the magneticbeads.</td><td rowspan=1 colspan=1>Nucleic acids are immobilized onmagnetic glass particles.</td></tr><tr><td rowspan=1 colspan=1>Washing to removeextraneous materials</td><td rowspan=1 colspan=1>Using a pick-pen with a retractable magnet,the magnetic beads are collected from thefirst well of the strip tube and transferred to anew well containing a wash buffer. Thebuffer removes extraneous materials whilethe nucleic acid remains bound to themagnetic beads. The beads are washed threetimes (three wells of the strip tube).</td><td rowspan=1 colspan=1>Unbound substances are removed byseveral washing steps.</td></tr><tr><td rowspan=1 colspan=1>Elution of nucleicacids</td><td rowspan=1 colspan=1>Using the pick-pen, the magnetic beads aretransferred to the last well of the strip tubecontaining elution buffer. The nucleic acidsare released from the magnetic beads. Thebeads are then removed from the sampleusing the pick pen.</td><td rowspan=1 colspan=1>Purified nucleic acids are eluted from themagnetic glass particles.</td></tr></table>

\*All steps are automated within the Roche MagNA Pure Compact instrument.

Negative Extraction Controls (NECs) are processed along with each batch of purified specimens. When processing NECs, the user utilizes the Sample Purification Kit to purify a sample of nuclease-free water. The purified NEC sample is then used to reconstitute the freeze-dried Negative Control vials. The use of an NEC allows for detection of contamination during the entire testing process (purification to reagent setup).

Purified samples are tested using the freeze-dried reagents in the JBAIDS Influenza A & B Detection Kit. Reagent vials that are intended for testing patient samples are labeled as Unknowns. Each JBAIDS run requires a Positive Control and a Negative Control for each target assay. The JBAIDS runs should be started within about 30 minutes after reconstitution of the freeze-dried reagents. If necessary, the reconstituted reagents can be stored for up to four hours on ice or in the refrigerator $( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } )$ .

Reagents are set up in the following order:

1. Unknowns – Reconstitute the reagent pellet in the Unknown vials (Flu A and Flu B) by adding $1 0 ~ \mu \mathrm { L }$ of the provided reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ of the purified sample. 2. Sample Control – Repeat the above process with a Sample Control vial. 3. Positive Controls – After the Unknowns and the Sample Control are set up for each sample, set up the Positive Controls by reconstituting the pellet with $1 0 \mu \mathrm { L }$ of the provided reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ of the provided reagent-grade water. 4. Negative Controls – Set up the Negative Controls last by reconstituting the pellet with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ of the purified NEC.

5. For each reagent vial, transfer $1 9 \mu \mathrm { L }$ of the reconstituted reagent to a capillary following the same order used for reconstitution. The cap is placed on the capillary tube before proceeding to the next capillary.   
6. Centrifuge the capillary tubes to ensure that the PCR reagent is in the tip of the capillary.

Each JBAIDS run can accommodate up to 9 samples with accompanying controls. The Diagnostic Wizard guides the operator through the process of setting up the run in compliance with the JBAIDS Influenza A & B Detection Kit package insert. After selecting the appropriate assays and entering sample information, the operator transfers prepared capillaries to the instrument carousel, guided by the loading pattern displayed in the Diagnostic Wizard. With the capillaries loaded into the instrument, the operator signals the instrument to start the PCR run. The instrument automatically controls the temperature heating cycle in the carousel. At intervals during temperature cycling, fluorescence emission is monitored as the carousel rotates to position each sample above the fluorimeter. Temperature and fluorescence data are displayed in real-time. After starting the run, no input is required and the operator need not supervise the run. At the conclusion of the run, the operator may proceed directly to data analysis and reporting.

Each JBAIDS test is analyzed and assigned a final result by the Detector module of the JBAIDS Software. Possible final results are positive, negative, uncertain, sample control failure (SC Failure) or invalid. To assign a final test result, Detector first analyzes the data from each capillary independently. Then, the software assigns a final result, or a combined call, based on the results of the sample and all of its controls.

Each stage of the analysis is described below:

# 1. Independent Capillary Call

Detector does not rely on any single aspect of the amplification curve, but rather integrates a number of factors, such as curve shape and signal-to-noise ratio into a combined score. Curves scoring above a threshold are called positive, curves scoring below a threshold are called negative, and curves scoring in a small area in between are called uncertain. The scoring system has been carefully tuned to match human expert calls on real amplification curve data.

Detector calls are characterized as follows:

Negative – A curve is called negative if its shape can be closely approximated by a line (or smooth curve), if it has low signal to noise, and if it shows little or no increase in fluorescence during the PCR.

Positive – A curve is called positive if the fluorescence shows exponential growth out of the background, if it has high signal to noise, and if the slope of the exponential region of the amplification is consistent with PCR amplification.

Uncertain – For a small number of curves (about $0 . 2 \%$ ) no clear call is possible and the reaction is called uncertain.

# 2. Test Result or Combined Call

The final test result is a combined call (or meta-call) based on the results of the test sample and its associated controls.

![](images/9e40646a778e42eb14832298fa18d6128c0b6c6ff96a528bf17a79b567fbc923.jpg)  
Flow chart outlining how final results are determined

# 3. Follow-up Testing for Invalid, Uncertain or Sample Control (SC) Failure Results

If the Flu A and Flu B assays yield a negative or positive result, then the JBAIDS testing is complete. Any other result requires follow-up testing, as described below:

Invalid – If either the Positive Control or Negative Control fails to give the expected results, all Unknown samples for that specific assay are called invalid and must be retested using the same purified sample. This testing can be done using the Custom Testing Method. • Uncertain – An uncertain result on a single sample requires retesting with the same purified sample. This testing can be done using the Custom Testing Method. Sample Control Failure – A failed Sample Control can occur from PCR inhibition, inefficient sample purification, or poor specimen collection. Any sample having a failed Sample Control assay (i.e. negative or uncertain Sample

Control assay associated with a negative or uncertain Target assay) will be retested using the original (undiluted) purified sample as well as a ten-fold dilution of the purified sample.

If the Sample Control Failure was due to an inhibitor present in the sample, the repeat testing should yield a result of Sample Control Failure for the undiluted sample and a valid result (positive or negative) for the sample diluted 1:10. The results from the retesting can be used for diagnostic applications only if the diluted sample tests target positive. A target negative result for the sample cannot be reported for the sample diluted to 1:10, because it is unknown if the target negative result is a true negative, is due to inhibitors, or is due to target template being diluted below the limit of detection.   
If the Sample Control Failure result was caused by improper technique, or other factors, then the repeat testing for the undiluted sample may yield a valid target assay result (positive or negative) with a positive SC result. In this case, the target assay result from the retesting of the undiluted sample can be used to support diagnostic decisions.   
If the Sample Control Failure was caused by ineffective sample purification or an improperly collected specimen, then the repeat testing result will most likely be Sample Control Failure. In this case, residual patient specimen can be re-extracted and tested (provided it was appropriately stored) or a new specimen should be requested, re-extracted, and tested.

# 4. JBAIDS Report

A JBAIDS report is generated automatically for each test run. With the JBAIDS Report, the user integrates the instrument-provided results for each influenza assay to make a final result interpretation.

# 5. Integrated Result Interpretation

The JBAIDS software automatically interprets the results for each of the target assays. However, the user is required to make the final test interpretation based upon the final test results for each of the two target assays. All possible results and interpretations are presented in the following table:

Result Interpretation for the JBAIDS Influenza A & B Detection Kit   

<table><tr><td colspan="2" rowspan="1">Test Result</td><td colspan="1" rowspan="1">Interpretation1,2</td><td colspan="1" rowspan="1">Action</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Influenza A and Influenza Bviral RNA not detected</td><td colspan="1" rowspan="1">No further testing required</td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Influenza A RNA detected</td><td colspan="1" rowspan="1">No further testing required.Optional: Follow up testingwith the JBAIDS Influenza ASubtyping kit may providefurther information.</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Influenza B RNA detected</td><td colspan="1" rowspan="1">No further testing required</td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Influenza A and Influenza Bviral RNA detected</td><td colspan="1" rowspan="1">Multiple infections arepossible, but rare.</td></tr><tr><td></td><td></td><td></td><td>Repeat testing from the residual purified nucleic acid, re-test from the residual original sample, or collect and test a new specimen.1 Optional: Follow up testing with the JBAIDS Influenza A Subtyping kit may provide further information.</td></tr></table>

1 Recent patient exposure to the FluMist $^ \mathrm { \textregistered }$ intranasal vaccine may result in false positive results and should be considered as a possible interferent when multiple influenza infections are detected.

# 6. Custom Test Method

The Custom Test Method can be used to test individual assays from the JBAIDS Influenza A & B Detection Kit. Reasons to use the Custom Test Method include:

PC or NC failure invalidates the results for only one of the three assays requiring retesting of all samples with only that assay. • A sample is uncertain for one of the assays, requiring retesting with only that assay.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multicenter reproducibility study was performed to determine the overall system reproducibility of the JBAIDS Influenza A & B Detection Kit on the JBAIDS instruments. This study was performed at Idaho Technology Inc. (ITI) and two external sites. The study was conducted with two specimen types: nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS), both collected in viral transport medium (VTM). Both specimen types were purified using two different Sample Purification Kits, the Roche MagNA Pure Nucleic Acid Isolation Kit I and the IT 1-2-3 Platinum Path Sample Purification Kit.

A total of four different panels of 9 samples each were prepared for the reproducibility study.

Panel 1 contained 9 NPS specimens that were screened with the Flu A and Flu B assays to ensure that no influenza nucleic acid was present, each spiked with a representative Influenza A virus strain (A/New Caledonia/20/1999) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). Therefore, Panel 1 consisted of 3 Influenza A virus spiked medium positive NPS specimens (3X LoD), 3 Influenza A virus spiked low positive NPS specimens (LoD), and 3 Influenza A virus spiked high negative NPS specimens. Panel 2 contained 9 NPW specimens that were screened with the Flu A and Flu B assays to ensure that no influenza nucleic acid was present, each spiked with a representative Influenza A virus strain (A/New Caledonia/20/1999) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). Therefore, Panel 2 consisted of 3 Influenza A virus spiked medium positive NPW specimens (3X LoD), 3 Influenza A virus spiked low positive NPW specimens (LoD), and 3 Influenza A virus spiked high negative NPW specimens.

Panel 3 contained 9 simulated NPS (sNPS) specimens that were screened with the Flu A and Flu B assays to ensure that no influenza nucleic acid was present, each spiked with a representative Influenza B strain (B/Ohio/1/2005) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPS were generated from diluting HeLa cells into VTM. (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 3 consisted of 3 Influenza B virus spiked medium positive sNPS specimens (3X LoD), 3 Influenza B virus spiked low positive sNPS specimens (LoD), and 3 Influenza B virus spiked high negative sNPS specimens.

Panel 4 contained 9 simulated NPW (sNPW) specimens that were screened with the Flu A and Flu B assays to ensure that no influenza nucleic acid was present, each spiked with a representative Influenza B strain $\left( \mathrm { B } / \mathrm { O h i o } / 1 / 2 0 0 5 \right)$ at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPW were generated from diluting HeLa cells into $50 \%$ VTM and $50 \%$ saline $( 0 . 9 \% \mathrm { N a C l } )$ . (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 4 consisted of 3 Influenza B virus spiked medium positive sNPW specimens (3X LoD), 3 Influenza B virus spiked low positive sNPW specimens (LoD), and 3 Influenza B virus spiked high negative sNPW specimens.

Note: Panels 3 and 4 specimens were also co-spiked with a representative seasonal influenza A H3N2 virus strain (A/New York/55/2004) for more efficient reproducibility testing of the JBAIDS Influenza A Subtyping Kit. Refer to the “Co-spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus.

The NPS and sNPS reproducibility panel compositions are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Level of Flu A (A/New Caledonia/20/1999)Virus in NPS Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1>Level of Flu B (B/Ohio/1/2005) Virus in sNPSSpecimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>15</td></tr></table>

The NPW and sNPW reproducibility panel compositions are presented in the following table:   

<table><tr><td rowspan=1 colspan=1>Level of Flu A (A/New Caledonia/20/1999)Virus in NPW Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD) - 3 specimens</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1>Level of Flu B (B/Ohio/1/2005) Virus in sNPW Virus Concentration (EID50/mL)Specimen</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>15</td></tr></table>

Each panel of 9 samples was tested twice daily for five days at each testing site. Every testing day, two users each tested each of the nine samples. Prior to testing with the JBAIDS assays, one user used the Roche MagNA Pure Nucleic Acid Isolation Kit I and the other user used the IT 1-2-3 Platinum Path Sample Purification Kit to extract viral nucleic acid from each sample. For each panel, each sample was tested a total of 30 times (2 runs/day/site X 5 days X 3 Sites; 15 times after purification with the Platinum Path kit and 15 times after purification with the MagNA Pure kit), and each concentration level per sample type was tested a total of 90 times (3 samples spiked at each concentration level in each of the 4 panels).

Each JBAIDS run performed in this study included Positive Control (PC) and negative extraction control (NEC) reactions. (Note: to ensure that no contamination of virus or amplification product was contracted during sample preparation and reaction setup, a NEC, which was an aliquot of reagent grade water, was prepared and tested with each batch of purified samples.)

Of the 63 Flu A PC and NEC reactions tested in this study, all were valid. The Flu A PC reactions had a mean $\mathrm { C p }$ of $2 8 . 8 7 \pm 0 . 6 9$ $( 2 . 3 9 \%$ CV). Of the 65 Flu B PC reactions tested in this study, three Flu B PC reactions failed $( 4 . 6 \% )$ . The other 62 Flu B PC reactions $( 9 6 . 4 \% )$ yielded positive results. Two of the failed Flu B PC reactions were due to run failures successfully detected by the controls. These run failures appear to be due to the carousel being improperly seated by the user. This shifted the capillaries during the test run, resulting in a loss of fluorescence detection in all samples. The JBAIDS software correctly failed the PC reactions and the test runs were called invalid. No data were collected from these failed runs. The purified samples from these failed runs were valid upon retesting. The remaining failed Flu B PC was from an unknown assay vial, which contained no PC template, being used for the PC reaction. Upon retesting the purified samples with the correct controls, the run was successful. All Flu B NEC samples tested in this study yielded negative results. The Flu B PC reactions had a mean $\mathrm { C p }$ of $2 6 . 5 9 \pm$ 0.23 ( $0 . 8 6 \%$ CV).

The Flu Sample Control assay (human RNase P assay) was tested with each purified sample to detect inhibition, poor sample extraction, or poor specimen collection. A total of 1085 out of 1093 $( 9 9 . 3 \% )$ Sample Control reactions produced the expected positive results in the reproducibility study. All 8 Flu SC negative results were from NPW or sNPW samples, which may contain fewer human cells per volume than swab samples. Because negative Flu SC assay results only invalidate negative influenza assays, 6 of these 8 negative Flu SC results had no impact on the study; the corresponding influenza assays had the expected positive metacall results. The other 2 negative Flu SC tests resulted in SC Failures, and both occurred with the same sample. On the initial run the sample gave an SC Failure call. According to the protocol, the purified sample was retested; both undiluted and diluted 1:10. The retest results were again a SC Failure for the undiluted sample. Therefore a new aliquot of the same sample was re-extracted and tested. Results of the second retest were positive for both the Flu B and Flu SC assays. NPS and NPW samples exhibited mean $\mathrm { C p }$ values of $3 0 . 1 5 \pm 2 . 3 6$ $7 . 8 3 \% \mathrm { C V }$ and 33.18 $\pm 1 . 8 8$ ( $5 . 6 7 \%$ CV), respectively, with the Flu SC assay. The HeLa cell concentrations determined for sNPS and sNPW samples were chosen to mimic Flu SC Cp values of fresh specimens. sNPS and sNPW samples had mean $\mathrm { C p }$ values of $3 2 . 0 9 \pm 1 . 7 7$ $( 5 . 5 2 \%$ CV) and $3 4 . 6 1 \pm 2 . 1 2$ $6 . 1 3 \%$ CV), respectively, with the Flu SC assay. The roughly $5 \text{‰}$ CV measured for each sample type is from $\mathrm { C p }$ variation for the Flu SC assay between the two sample purification kits. Typically MagNA Pure extracted samples exhibit $\mathrm { C p }$ values 4 cycles earlier than respective Platinum Path extracted samples.

Reproducibility Study Summary (Agreement with Expected Positive Results) for the JBAIDS Influenza A & B Detection Kit – Influenza A Assay   

<table><tr><td rowspan=3 colspan=1>SampleType</td><td rowspan=3 colspan=1>Test Level</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path Sample PurificationKit# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=1 colspan=4>Roche MagNA Pure Compact Nucleic AcidIsolation Kit I# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=3 colspan=1>AllPurificationKits, All Sites</td><td rowspan=3 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=4>Test Location</td><td rowspan=1 colspan=4>Test Location</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=5 colspan=1>N</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(1000%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>94.0-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>9/15(60%)</td><td rowspan=1 colspan=1>11/15(73%)</td><td rowspan=1 colspan=1>11/15(73%)</td><td rowspan=1 colspan=1>31/45(69%)</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>42/45(93%)</td><td rowspan=1 colspan=1>73/90(81%)</td><td rowspan=1 colspan=1>71.5-88.6</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>179/180(99%)</td><td rowspan=1 colspan=1>96.9-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection allLevels</td><td rowspan=1 colspan=1>39/45(87%)</td><td rowspan=1 colspan=1>41/45(91%)</td><td rowspan=1 colspan=1>41/45(91%)</td><td rowspan=1 colspan=1>121/135(90%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>131/135(97%)</td><td rowspan=1 colspan=1>252/270(93%)</td><td rowspan=1 colspan=1>89.7-96.0</td></tr><tr><td rowspan=5 colspan=1>MdN</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>94.0-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>11/15(73%)</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>38/45(84%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>81/90(90%)</td><td rowspan=1 colspan=1>81.9-95.3</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>179/180(99%)</td><td rowspan=1 colspan=1>96.9-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection AllLevels</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>41/45(91%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>128/135(95%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>42/45(93%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>132/135(98%)</td><td rowspan=1 colspan=1>260/270(96%)</td><td rowspan=1 colspan=1>93.3-98.2</td></tr></table>

Reproducibility Study Summary (Agreement with Expected Positive Results) for the JBAIDS Influenza A & B Detection Kit – Influenza B Assay   

<table><tr><td rowspan=3 colspan=1>SampleType</td><td rowspan=3 colspan=1>Test Level</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path Sample PurificationKit# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=1 colspan=4>Roche MagNA Pure Compact Nucleic AcidIsolation Kit I# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=3 colspan=1>AllPurificationKits, All Sites</td><td rowspan=3 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=4>Test Location</td><td rowspan=1 colspan=4>Test Location</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=5 colspan=1>NAN</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>88/90(98%)</td><td rowspan=1 colspan=1>92.2-99.7</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>98.3-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection allLevels</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>134/135(99%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>134/135(99%)</td><td rowspan=1 colspan=1>268/270(99%)</td><td rowspan=1 colspan=1>97.4-99.9</td></tr><tr><td rowspan=5 colspan=1>MdNs</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15a(100%)</td><td rowspan=1 colspan=1>12/15(80%)</td><td rowspan=1 colspan=1>41/45(91%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>86/90(96%)</td><td rowspan=1 colspan=1>89.0-98.8</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>180/180(100%)</td><td rowspan=1 colspan=1>98.3-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection AllLevels</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>42/45(93%)</td><td rowspan=1 colspan=1>131/135(97%%</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>135/135(100%)</td><td rowspan=1 colspan=1>266/270(99%)</td><td rowspan=1 colspan=1>96.3-99.6</td></tr></table>

a Retest result included in total; first test was invalid due to SC Failure.

This reproducibility study demonstrated that as expected with all real-time PCR assays, the JBAIDS Influenza A & B Detection Kit may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration. This limitation should be addressed by including the following statement in the Limitation section of the JBAIDS Influenza A & B Detection Kit Instructions for Use: “The JBAIDS Influenza A & $\pmb { B }$ Detection Kit may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration.”

The variability of the Cp value can be used as an additional measure of the variability of JBAIDS assays. Therefore, the average and $\% \mathrm { C V }$ of $\mathrm { C p }$ values obtained in this study were used to further evaluate the precision/reproducibility of the test system.

Reproducibility Study Summary (Mean Cp, SD and $\% \mathbf { C V }$ ) for the JBAIDS Influenza A & B Detection Kit – Influenza A Assay   

<table><tr><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>Virus SpikeLevel</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum PathSample Purification Kit</td><td rowspan=1 colspan=8>Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>MeanCp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>33.77(0.28)</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>33.68(0.36)</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>33.22(0.62)</td><td rowspan=1 colspan=1>1.86</td><td rowspan=1 colspan=1>33.55(0.50)</td><td rowspan=1 colspan=1>1.48</td><td rowspan=1 colspan=1>30.64(0.46)</td><td rowspan=1 colspan=1>1.51</td><td rowspan=1 colspan=1>31.08(0.51)</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>30.66(0.58)</td><td rowspan=1 colspan=1>1.88</td><td rowspan=1 colspan=1>30.79(0.55)</td><td rowspan=1 colspan=1>1.78</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>35.15(0.33)</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>34.56(0.87)</td><td rowspan=1 colspan=1>2.53</td><td rowspan=1 colspan=1>34.14(0.66)</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>34.61(0.74)</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>32.01(0.47)</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>31.95(0.43)</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>31.76(0.50)</td><td rowspan=1 colspan=1>1.57</td><td rowspan=1 colspan=1>31.91(0.47)</td><td rowspan=1 colspan=1>1.46</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>38.02 a(1.95)</td><td rowspan=1 colspan=1>5.12a</td><td rowspan=1 colspan=1>36.35(0.66)</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>36.82(0.53)</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>37.01 a(1.28)</td><td rowspan=1 colspan=1>3.47</td><td rowspan=1 colspan=1>34.17(1.20)</td><td rowspan=1 colspan=1>3.53</td><td rowspan=1 colspan=1>33.96(0.76)</td><td rowspan=1 colspan=1>2.22</td><td rowspan=1 colspan=1>34.03(0.89)</td><td rowspan=1 colspan=1>2.63</td><td rowspan=1 colspan=1>34.05(0.94)</td><td rowspan=1 colspan=1>2.76</td></tr><tr><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>33.17(0.73)</td><td rowspan=1 colspan=1>2.20</td><td rowspan=1 colspan=1>32.81b(0.44)</td><td rowspan=1 colspan=1>1.35b</td><td rowspan=1 colspan=1>33.37(0.36)</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>33.11(0.57)</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>30.47(0.44)</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>31.21(0.42)</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>30.00(0.60)</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>30.56(0.70)</td><td rowspan=1 colspan=1>2.28</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>34.48(0.79)</td><td rowspan=1 colspan=1>2.29</td><td rowspan=1 colspan=1>34.25(0.58)</td><td rowspan=1 colspan=1>1.70</td><td rowspan=1 colspan=1>34.69(0.50)</td><td rowspan=1 colspan=1>1.43</td><td rowspan=1 colspan=1>34.47(0.65)</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>31.84(0.52)</td><td rowspan=1 colspan=1>1.62</td><td rowspan=1 colspan=1>32.40(0.43)</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>31.35(0.39)</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>31.85(0.61)</td><td rowspan=1 colspan=1>1.93</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>38.22 a(2.06)</td><td rowspan=1 colspan=1>5.39a</td><td rowspan=1 colspan=1>37.63 a,b(1.69)</td><td rowspan=1 colspan=1>4.48 a,b</td><td rowspan=1 colspan=1>38.17a(1.84)</td><td rowspan=1 colspan=1>4.82 </td><td rowspan=1 colspan=1>38.03 a(1.85)</td><td rowspan=1 colspan=1>4.86</td><td rowspan=1 colspan=1>34.69(0.40)</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>35.42(0.62)</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>34.73(0.49)</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>34.92(0.59)</td><td rowspan=1 colspan=1>1.69</td></tr></table>

a Capillary had a Cp of ${ > } 4 0 . 0 0$ and was assigned a Cp of 42.50 cycles. b One medium positive sample and a high negative sample were switched during testing. Based on Cp values they have been assigned to the correct virus spike level during data analysis.

Reproducibility Study Summary (Mean Cp, SD and $\% \mathbf { C V }$ ) for the JBAIDS Influenza A & B Detection Kit – Influenza B Assay   

<table><tr><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>VirusSpikeLevel</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum PathSample Purification Kit</td><td rowspan=1 colspan=8>Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>0V</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>sNPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>30.59(0.24)</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>30.34(0.42)</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>30.75(0.47)</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>30.56(0.42)</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>29.08(0.30)</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>28.88(0.38)</td><td rowspan=1 colspan=1>1.31</td><td rowspan=1 colspan=1>27.97(0.44)</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>28.65(0.61)</td><td rowspan=1 colspan=1>2.14</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>32.00(0.23)</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>31.73(0.43)</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>32.09(0.31)</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>31.94(0.36)</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>30.47(0.47)</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>30.29(0.51)</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>29.28(0.21)</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>30.01(0.67)</td><td rowspan=1 colspan=1>2.23</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>35.26(0.54)</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>34.27(0.54)</td><td rowspan=1 colspan=1>1.56</td><td rowspan=1 colspan=1>34.92(0.56)</td><td rowspan=1 colspan=1>1.62</td><td rowspan=1 colspan=1>34.83(0.68)</td><td rowspan=1 colspan=1>1.94</td><td rowspan=1 colspan=1>34.32(0.43)</td><td rowspan=1 colspan=1>1.24</td><td rowspan=1 colspan=1>33.47(0.90)</td><td rowspan=1 colspan=1>2.68</td><td rowspan=1 colspan=1>33.00(0.21)</td><td rowspan=1 colspan=1>0.65</td><td rowspan=1 colspan=1>33.60(0.80)</td><td rowspan=1 colspan=1>2.37</td></tr><tr><td rowspan=3 colspan=1>sNPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>30.75(0.29)</td><td rowspan=1 colspan=1>0.94</td><td rowspan=1 colspan=1>30.57(0.46)</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>31.00(1.06)</td><td rowspan=1 colspan=1>3.41</td><td rowspan=1 colspan=1>30.77(0.69)</td><td rowspan=1 colspan=1>2.26</td><td rowspan=1 colspan=1>27.98(0.62)</td><td rowspan=1 colspan=1>2.21</td><td rowspan=1 colspan=1>28.51(0.96)</td><td rowspan=1 colspan=1>3.37</td><td rowspan=1 colspan=1>27.64(0.14)</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>28.04(0.74)</td><td rowspan=1 colspan=1>2.65</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>32.54(1.26)</td><td rowspan=1 colspan=1>3.87</td><td rowspan=1 colspan=1>31.84(0.38)</td><td rowspan=1 colspan=1>1.20</td><td rowspan=1 colspan=1>32.05(0.40)</td><td rowspan=1 colspan=1>1.26</td><td rowspan=1 colspan=1>32.14(0.83)</td><td rowspan=1 colspan=1>2.59</td><td rowspan=1 colspan=1>29.56(0.67)</td><td rowspan=1 colspan=1>2.27</td><td rowspan=1 colspan=1>30.32(1.07)</td><td rowspan=1 colspan=1>3.54</td><td rowspan=1 colspan=1>29.10(0.18)</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>29.66(0.88)</td><td rowspan=1 colspan=1>2.98</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>34.94(1.09)</td><td rowspan=1 colspan=1>3.11</td><td rowspan=1 colspan=1>34.57(0.65)</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>35.38(0.48)</td><td rowspan=1 colspan=1>1.37</td><td rowspan=1 colspan=1>34.93(0.84)</td><td rowspan=1 colspan=1>2.41</td><td rowspan=1 colspan=1>33.37(0.41)</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>33.59(0.91)</td><td rowspan=1 colspan=1>2.72</td><td rowspan=1 colspan=1>33.15(0.44)</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>33.37(0.64)</td><td rowspan=1 colspan=1>1.92</td></tr></table>

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The following controls are included in the JBAIDS Influenza A & B Detection Kit:

# Negative Control

The NC is used to detect contamination from target-specific amplified product (amplicon), synthetic RNA (as found in the PC vials), or virus. The NC vial is reconstituted using the purified negative extraction control (NEC) (molecular biology grade water). The NEC should be processed along with each batch of specimens purified. The NC detects possible contamination during sample purification and during reconstitution of freeze- dried reagents. Each JBAIDS run requires one NC for the Flu A assay and one NC for the Flu B assay. For each target assay, the NC must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run. If the NC fails, the JBAIDS run must be repeated using the same purified samples. If the NC fails again, all associated samples are considered to have invalid results and should be re-purified starting from another aliquot of the original specimens.

Of the 63 Flu A NC reactions tested in the reproducibility study, all were valid. All Flu B NC samples tested in the reproducibility study also yielded negative results.

Of 229 runs, there were a total of one $( 0 . 4 \% )$ run in which the NC failed. The failure appears to be the result of user errors rather than failure of the test system. This failure occurred at Site 1, and was a result of switching the positions of the NCs and PCs for both the Flu A and Flu B assays when loading the capillaries into the JBAIDS carousel.

# Positive Control

The PC serves as an amplification and detection control. Each JBAIDS run requires one PC for the Flu A assay and one PC for the Flu B assay. For each target assay, the PC must be positive. If the PC fails, the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the JBAIDS run must be repeated starting from the same purified specimens. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument.

All of the 63 Flu A PC reactions tested in the reproducibility study were valid. The Flu A PC reactions had a mean $\mathrm { C p }$ of $2 8 . 8 7 \pm 0 . 6 9$ $2 . 3 9 \%$ CV). Of the 65 Flu B PC reactions tested in this study, three Flu B PC reactions failed $( 4 . 6 \% )$ . The Flu B PC reactions had a mean $\mathrm { C p }$ of $2 6 . 5 9 \pm 0 . 2 3$ ( $0 . 8 6 \%$ CV).

Of 229 runs in the prospective clinical study, there were a total of three $( 1 . 3 \% )$ runs in which the PCs failed. All of the failures appear to be the result of user errors rather than failure of the test system. Two failures occurred at Site 1. The first failure was a result of switching the positions of the NCs and PCs for both the Flu A and Flu B assays when loading the capillaries into the JBAIDS carousel. The second failure occurred because the PCs for both the Flu A and Flu B assays failed to amplify. The third failure occurred at Site 2 due to an improperly seated carousel causing the amplification curves to be interpreted as negative. There was minimal site to site variation with the PCs. The mean PC Cp value for Flu A was $2 8 . 3 \pm 0 . 7$ ( $2 . 3 \%$ CV) and for Flu B was $2 6 . 5 \pm 0 . 2$ $( 0 . 9 \%$ CV).

The final $\mathrm { C p }$ cutoff value for the Flu A and Flu B assay positive controls are 35.5 and 32.0 respectively, and are based on the upper bound of the manufacturing QA Review range for reagent lot release plus $4 \mathrm { \Delta x }$ SD. The estimated standard deviation (SD) was set to $0 . 9 \mathrm { C p }$ to reflect the amount of variation that is consistent with data obtained in the JBAIDS Influenza A Subtyping Kit and the JBAIDS Influenza A & B Detection Kit analytical and clinical studies. If an amplification curve for a PC fails to amplify or has a $\mathrm { C p }$ value later than the cutoff, the PC will be interpreted as a failure and the associated patient test results will be assigned an invalid test result.

# Sample Control (SC)

The sample control assay (Flu SC) detects the human RNase P gene. This assay is designed to guard against false negative results caused by an improperly collected specimen, ineffective purification of nucleic acids and or inhibition of the PCR reaction. A properly collected NPS or NPW specimen contains human cells from which the RNase P target is recovered during sample purification. Following purification, each sample is then tested with the Flu A and Flu B target assays, as well as the Flu SC. If purification and amplification were successful, then the Flu SC will give the expected positive test result. The JBAIDS software automatically assigns a result of sample control failure when (1) the Flu SC is unsuccessful and (2) the target assay is negative (or uncertain).

# Extraction Control (EC)

Positive Extraction Control (PEC) was not provided with the kit, and inclusion of a PEC with each batch of extracted samples is not a standard part of the testing protocol recommended in the product package insert. However, during the clinical study, a PEC and a NEC were purified with each batch of clinical samples. Inclusion of a NEC with each batch of purified samples is a standard part of the testing protocol recommended in the product package insert. PECs were simulated NPS (sNPS; HeLa cells in viral transport media) spiked with four representative influenza strains (Influenza B, seasonal Influenza A H1N1, seasonal Influenza A H3N2, and Influenza A 2009 H1N1) at concentrations near the LoD for the influenza assays. The PEC was processed and tested in the same manner as a patient sample. The PEC was considered successful when all tested target assays were positive. A failed PEC was retested using the same purified sample and the results of the retest were accepted as the final result. If the retest was unsuccessful, all associated samples were considered to have invalid results and required re-purification in order to obtain a valid test result. During the clinical evaluation, four out of a total of 190 PECs $( 2 . 1 1 \% )$ required retesting (three at site 1 and one at site 2). Al four PECs were successful upon retesting. The mean $\mathrm { C p }$ of the PEC tested for the Flu A assay was 28.7, with 2.2 Standard Deviation (SD) and $7 . 7 \%$ CV. The mean $\mathrm { C p }$ of the

PEC tested for the Flu B assay was 29.0, with 1.7 Standard Deviation (SD) and $5 . 9 \%$ CV.

The NEC was a nuclease-free water sample that was processed with each batch of specimen purifications. The purified NEC sample was then used to reconstitute the freeze-dried NC.

# Specimen Stability

The sponsor recommends that MPW and NPS specimens be collected and placed into viral transport media (VTM). Once collected, specimens may be tested immediately after collection, transported to an off-site testing location, or temporarily stored until testing can be scheduled. Inappropriate handling or storage may compromise the integrity of the specimens and cause inaccurate test results. An analytical study determining the appropriate storage times and temperatures for both unprocessed specimens (e.g., the NPW and NPS specimens) and processed samples (e.g., purified nucleic acid from NPW and NPS specimens) was carried out. The specimen transport and storage conditions that were evaluated are based on common clinical laboratory workflows.

Due to the potential combined use of the JBAIDS Influenza A & B Detection Kit with the JBAIDS Influenza A Subtyping Kit, three Influenza A subtype strains (i.e., Influenza A/H1, Influenza A/H3, and Influenza A 2009 H1N1) were evaluated with the Flu A assay for both unprocessed and purified samples. One influenza B strain was evaluated with the Flu B assay.

Note: Specimens spiked with the representative Influenza B virus (B/Ohio/1/2005) were also co-spiked with a representative seasonal influenza A H3N2 virus strain (A/New York/55/2004) for more efficient testing of the JBAIDS Influenza A Subtyping Kit. Refer to the $^ { \circ \mathfrak { c } } \mathrm { C o } { } .$ -spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus

For the unprocessed specimen transport and storage evaluation, both negative and positive specimens were assessed. Positive specimens were spiked with Influenza A and/or B viruses at the established LoD for the respective assay. Eight positive (per influenza strain) and four negative specimens were processed at each time point and storage condition per purification kit prior to testing with the freeze-dried reagents. Both specimen types (NPW and NPS) were evaluated independently.

For the purified sample transport and storage evaluation, sets of eight positive samples (per influenza strain) and four negative samples that had been processed with either purification method were tested immediately and also aliquoted for storage under various conditions prior to retesting with the freeze-dried reagents. Purified samples were also tested to determine the storage of reconstituted freeze-dried reagents.

The following table summarizes the final conditions that were evaluated in this study:

Final Storage Conditions Evaluated for each Sample Matrix/Purification Kit Combination   

<table><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=1>Conditions</td></tr><tr><td rowspan=1 colspan=1>Unprocessed Specimens</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, ambient (18-30°C)Day 3, refrigerated (2-8°C)Day 30, frozen (≤ -15°C)</td></tr><tr><td rowspan=1 colspan=1>Purified Samples</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, ambient (18-30°C)Day 1, refrigerated (2-8°C)Day 30 , frozen (≤ -15°C) Platinum Path onlyDay 30, frozen (≤ -70°C) MagNA Pure only</td></tr><tr><td rowspan=1 colspan=1>Reconstituted Freeze Dried-Reagents</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, refrigerated (2-8°C)</td></tr></table>

The performance at each time point was evaluated by comparing the JBAIDS assay result (positive, negative, uncertain, or SC fail) to the expected result and the result at the initial time point. To be acceptable, all negative samples must be negative for the influenza assays and positive for the Flu SC assay, and seven of eight positive samples must test positive for the relevant assays (Flu SC, Flu A and/or Flu B). In addition, the mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values of the stored sample set were compared to those obtained at the initial time point. For an acceptable storage condition, the mean $\mathrm { C p }$ of the influenza assays for the positive sample set should not increase by more than 3 cycles, and the mean $\mathrm { F } _ { \mathrm { m a x } }$ must not decrease by more than $50 \%$ . For the Flu SC assay, the amount of target material in individual NPW and NPS specimens is variable; mean $\mathrm { C p }$ and mean $\mathrm { F } _ { \mathrm { m a x } }$ values were calculated and monitored for major shifts in performance for both positive and negative samples.

Unprocessed Specimen Storage Conditions Test Results

At least seven of the eight positive human derived NPS and NPW specimens in each sample set tested positive at all four time points with the respective assay. With the exception of Influenza A/H1 NPS samples processed by MagNA Pure after 30 days of storage, the mean Cp values for the stored specimens were all within 3 cycles of the day 0 time point, suggesting no decrease in assay performance with storage. The one set of samples (i.e., Influenza A/H1 NPS samples processed by MagNA Pure after 30 days of storage) that the Influenza A assay mean Cp values for the stored specimens were greater than 3 cycles of the day 0 time point exceeded the acceptance criteria and could indicate degradation of the samples. However, all of the samples gave the expected positive results and this greater than 3 cycle shift in Cp was not consistent for other samples processed with MagNA Pure at this time point, indicating that accurate test results are not significantly compromised under this storage condition. In addition, the average $\mathrm { F } _ { \mathrm { m a x } }$ values for the time points were never less than $50 \%$ of the $\mathrm { F } _ { \mathrm { m a x } }$ of the day 0 baseline indicating no reduction in test system performance. All 4 negative samples for both specimen types tested negative at all three time points. The mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values of the associated Sample Control assays were similar at all four time points.

These results indicate that NPS and NPW specimens can be stored at $1 8 { - } 3 0 ~ ^ { \circ } \mathrm { C }$ for up to four hours, at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to three days, and at $\leq - 1 5 ~ ^ { \circ } \mathrm { C }$ for up to thirty days prior to testing without compromising the results.

# Purified Sample Storage Conditions Test Results

At least seven of the eight purified human derived NPS and NPW positive samples in each sample set tested positive at each time point with the respective assay. The average Cp values for the stored samples were all within 3 cycles of the day 0 time point suggesting no decrease in assay performance with storage. In addition, the average $\mathrm { F _ { m a x } }$ values for the time points were never less than $50 \%$ of the $\mathrm { F } _ { \mathrm { m a x } }$ of the day 0 baseline indicating no reduction of test system performance. All negative specimens tested negative at each time point. The mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values of the associated Flu SC assays were similar at all four time points.

These results indicate that purified samples can be stored at $1 8 { - } 3 0 ~ ^ { \circ } \mathrm { C }$ for up to four hours, at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to one day, and frozen $( \leq - 1 5 ^ { \circ } \mathrm { C }$ for Platinum Path purified samples, ≤ - $7 0 \mathrm { { } ^ { \circ } C }$ for MagNA Pure purified samples) for up to thirty days prior to testing without compromising the results.

# Reconstituted Reagent Storage Conditions Test Results

All but one of the positive samples tested positive at both the initial time point and after four hours of refrigeration. The mean Cp values for the stored reactions were all within 3 cycles of the initial time point suggesting no decrease in assay performance with storage. In addition, the mean $\mathrm { F } _ { \mathrm { m a x } }$ values for the stored reagents were never less than $50 \%$ of the $\mathrm { F } _ { \mathrm { m a x } }$ of the initial baseline indicating no reduction of test system performance. All negative reactions tested negative at both time points. The mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values of the associated Flu SC assays were similar at all both time points.

These results indicate that reconstituted reagents can be stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to four hours without compromising the results.

# d. Detection limit:

Initial estimates for setting the LoD were performed by spiking simulated NPW (sNPW) and NPS samples with serial dilutions of quantified representative influenza A or B viruses (Influenza A H1N1 strain A/New Caledonia/20/1999, Influenza A H3N2 strain A/New York/55/2004, Influenza A 2009 H1N1 strain A/New York/18/2009, Influenza B strain B/Ohio/1/2005, and Influenza B strain B/Florida/7/2004). The spiked samples were then processed using the IT 1-2-3 Platinum Path Sample Purification Kit and the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I, and the purified samples were tested with the associated target influenza assays (Flu A or Flu B). An initial estimate of the LoD was made by evaluating the reaction quality at each dilution. Reactions were evaluated using multiple criteria, including the JBAIDS test results, the Cp values, and a visual examination of the amplification curves. LoD estimates were then further evaluated by testing human-derived or simulated samples spiked with refined serial dilutions of the representative influenza viruses.

Once the LoD was estimated, a set of 20 specimens was collected from 20 unique donors for each combination of specimen type (NPS or NPW) and purification kit (Platinum Path or MagNA Pure). The samples were spiked at the estimated LoD level, processed, and tested with the JBAIDS Flu A or Flu B assay to determine the frequency of positive results. The LoD was confirmed if at least 19 of the 20 samples gave a positive test result. The LoD levels for the Flu A assay were independently established using three influenza A subtype viruses (Influenza A H1N1, A H3N2, and A 2009 H1N1). The LoD levels for the Flu B assay were independently established using two representative influenza B virus strains.

Based on the titration results for each strain, the LoD levels were set at the concentrations for which $2 9 5 \%$ detection was observed for both sample types and both nucleic acid purification kits. All sets of 20 samples spiked with influenza A and B viruses gave at least 19/20 positive results with the JBAIDS Flu A or Flu B assay. The LoD for the Flu A assay is $5 0 ~ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the seasonal Influenza A H1N1 strain A/New Caledonia/20/1999, $5 \ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Influenza A H3N2 strain A/New York/55/2004, and $5 0 \ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Influenza A 2009 H1N1 strain A/New York/18/2009. The LoD for the Flu B assay is 5 $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Influenza B strain B/Ohio/1/2005, and $1 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for the Influenza B strain B/Florida/7/2004

Confirmation of Estimated LoD Levels for the JBAIDS Influenza A & B Detection Kit   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Influenza Type(Strain)</td><td colspan="1" rowspan="1">LoD(EID50/mL)</td><td colspan="1" rowspan="1">SpecimenType</td><td colspan="1" rowspan="1">PurificationKit</td><td colspan="1" rowspan="1">SamplesTested</td><td colspan="1" rowspan="1">Samples Detected(%)</td><td colspan="1" rowspan="1">Mean Cp(SD)</td><td colspan="1" rowspan="1">Mean Fmax(SD)</td></tr><tr><td colspan="1" rowspan="12">Flu A</td><td colspan="1" rowspan="4">Influenza A H1N1(A/NewCaledonia/20/1999)</td><td colspan="1" rowspan="4">50</td><td colspan="1" rowspan="2">NPS</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">31.69 (0.56)</td><td colspan="1" rowspan="1">13.13 (2.38)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">33.84 (0.55)</td><td colspan="1" rowspan="1">12.42 (2.10)</td></tr><tr><td colspan="1" rowspan="2">NPW</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">31.92 (0.47)</td><td colspan="1" rowspan="1">17.52 (1.91)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">34.36 (0.41)</td><td colspan="1" rowspan="1">14.15 (2.11)</td></tr><tr><td colspan="1" rowspan="4">Influenza A H3N2(A/NewYork/55/2004)</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="2">NPS</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">33.34 (0.64)</td><td colspan="1" rowspan="1">8.59 (3.41)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19 (95%)</td><td colspan="1" rowspan="1">35.13 (0.68)</td><td colspan="1" rowspan="1">6.97 (2.71)</td></tr><tr><td colspan="1" rowspan="2">NPW</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">34.25 (0.52)</td><td colspan="1" rowspan="1">12.98 (1.47)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">35.92 (0.61)</td><td colspan="1" rowspan="1">7.44 (2.45)</td></tr><tr><td colspan="1" rowspan="4">2009 Influenza A H1N1(A/NewYork/18/2009)</td><td colspan="1" rowspan="4">50</td><td colspan="1" rowspan="2">NPS</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">33.02 (0.61)</td><td colspan="1" rowspan="1">11.43 (2.81)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">34.41 (0.82)</td><td colspan="1" rowspan="1">8.76 (2.67)</td></tr><tr><td colspan="1" rowspan="2">NPW</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">33.23 (0.61)</td><td colspan="1" rowspan="1">15.00 (2.42)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">35.23 (0.57)</td><td colspan="1" rowspan="1">11.43 (2.85)</td></tr><tr><td colspan="1" rowspan="8">Flu B</td><td colspan="1" rowspan="4">Influenza B(B/Ohio/1/2005)</td><td colspan="1" rowspan="4">5</td><td colspan="1" rowspan="2">NPS</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">29.79 (0.28)</td><td colspan="1" rowspan="1">38.49 (6.72)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20(100%)</td><td colspan="1" rowspan="1">31.27(0.39)</td><td colspan="1" rowspan="1">30.33(4.23)</td></tr><tr><td colspan="1" rowspan="2">NPW</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19 (95%)</td><td colspan="1" rowspan="1">29.56 (0.47)</td><td colspan="1" rowspan="1">39.27 (10.95)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19(95%)</td><td colspan="1" rowspan="1">31.14(1.41)</td><td colspan="1" rowspan="1">20.66(7.56)</td></tr><tr><td colspan="1" rowspan="4">Influenza B(B/Florida/7/2004)</td><td colspan="1" rowspan="4">10</td><td colspan="1" rowspan="2">NPS</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20(100%)</td><td colspan="1" rowspan="1">31.07(0.40)</td><td colspan="1" rowspan="1">40.05(5.27)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20 (100%)</td><td colspan="1" rowspan="1">32.39 (1.86)</td><td colspan="1" rowspan="1">24.53 (9.48)</td></tr><tr><td colspan="1" rowspan="2">NPW</td><td colspan="1" rowspan="1">MagNA Pure</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">19(95%)</td><td colspan="1" rowspan="1">30.13(0.46)</td><td colspan="1" rowspan="1">40.22(4.54)</td></tr><tr><td colspan="1" rowspan="1">PlatinumPath</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">20(100%)</td><td colspan="1" rowspan="1">31.45(0.36)</td><td colspan="1" rowspan="1">27.39(4.77)</td></tr></table>

# e. Analytical Reactivity:

For inclusivity testing, sNPS samples were spiked with various strains of influenza target viruses within the A and B genera prior to purification with the IT 1-2-3 Platinum Path Sample Purification Kit and testing with the JBAIDS Influenza A & B Detection Kit. Viruses were spiked at concentrations estimated to be near the LoD of each analyte. Due to differences in quantification methods between primary strains (quantified in $\mathrm { E I D } _ { 5 0 } / \mathrm { m L } )$ and analytical reactivity panel strains (mostly quantified in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , concentrations spanning the primary strain LoD $( 0 . 1 \times , 1 \times ,$ $1 0 { \times } \mathrm { L o D }$ , assuming $1 \mathrm { E I D } _ { 5 0 }$ was equivalent to $1 \mathrm { T C I D } _ { 5 0 }$ ) were tested. If a sample containing a particular strain was positive at any of the concentrations, no further testing of that strain was required. If a strain was not detected at any of these concentrations, the strain was retested in 10-fold higher increments until a positive result was obtained or until the maximum concentration possible for that strain stock had been tested.

The following two tables provide a complete listing of the Influenza A and B strains that were used to assess the analytical reactivity or inclusivity of the test system. The 25 Influenza A strains tested were representative of strains isolated from different hosts and locations around the world as early as 77 years ago (1933) and as recently as 2009. The 9 influenza B strains were isolated from locations around the world as early as 71 years ago and as recently as 2006.

Influenza A Strains Detected by the JBAIDS Flu A Assay   

<table><tr><td colspan="1" rowspan="1">Type/Subtype</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Lowest Concentration</td><td colspan="1" rowspan="1">Cp</td></tr><tr><td colspan="1" rowspan="1">H2N2 (Avian)</td><td colspan="1" rowspan="1">A/chicken/Pennsylvania/298101-</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">36.92</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Avian)</td><td colspan="1" rowspan="1">A/MAL/ALB/16/87</td><td colspan="1" rowspan="1">50 TCID50/mL</td><td colspan="1" rowspan="1">36.60</td></tr><tr><td colspan="1" rowspan="1">H4N8 (Avian)</td><td colspan="1" rowspan="1">A/chicken/Alabama/1975</td><td colspan="1" rowspan="1">5 EID50/mL</td><td colspan="1" rowspan="1">34.31</td></tr><tr><td colspan="1" rowspan="1">H5N1 (Avian)</td><td colspan="1" rowspan="1">A/DK/PA/4560069-9/06</td><td colspan="1" rowspan="1">50 TCID50/mL</td><td colspan="1" rowspan="1">36.99</td></tr><tr><td colspan="1" rowspan="1">H5N1 (Avian-Human</td><td colspan="1" rowspan="1">A/Vietnam/1203/2004(H5N1)-PR8</td><td colspan="1" rowspan="1">0.5 EID50/mL</td><td colspan="1" rowspan="1">39.19</td></tr><tr><td colspan="1" rowspan="1">H6N2 (Avian)</td><td colspan="1" rowspan="1">A/Chicken/CA/32213-1/2000</td><td colspan="1" rowspan="1">0.5 EID50/mL</td><td colspan="1" rowspan="1">38.28</td></tr><tr><td colspan="1" rowspan="1">H7N3 (Avian)</td><td colspan="1" rowspan="1">A/TY/UT/24721-10/95</td><td colspan="1" rowspan="1">500 TCID50/mL</td><td colspan="1" rowspan="1">36.89</td></tr><tr><td colspan="1" rowspan="1">H9N2 (Avian)</td><td colspan="1" rowspan="1">A/Turkey/Wisconsin/1966</td><td colspan="1" rowspan="1">0.5 EID50/mL</td><td colspan="1" rowspan="1">37.08</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Canine)</td><td colspan="1" rowspan="1">A/canine/Florida/43/2004</td><td colspan="1" rowspan="1">5,000 TCID50/mL</td><td colspan="1" rowspan="1">34.80</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Equine)</td><td colspan="1" rowspan="1">A/Equine/Ohio/01/2009</td><td colspan="1" rowspan="1">50 TCID50/mL</td><td colspan="1" rowspan="1">38.17</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Swine)</td><td colspan="1" rowspan="1">A/SW/GB/19582/92</td><td colspan="1" rowspan="1">50 TCID50/mL</td><td colspan="1" rowspan="1">42.50a</td></tr><tr><td colspan="1" rowspan="1">H1N1(Swine)</td><td colspan="1" rowspan="1">A/swine/Wisconsin/125/1997</td><td colspan="1" rowspan="1">500 TCID50/mL</td><td colspan="1" rowspan="1">39.35</td></tr><tr><td colspan="1" rowspan="1">H3N2 (Swine)</td><td colspan="1" rowspan="1">A/SW/IA/1/99</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">42.50a</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Human of swine lineage)</td><td colspan="1" rowspan="1">A/Iowa/1/2006</td><td colspan="1" rowspan="1">500 TCID50/mL</td><td colspan="1" rowspan="1">39.22</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Human of swine lineage)</td><td colspan="1" rowspan="1">A/Maryland/12/1991</td><td colspan="1" rowspan="1">5,000 TCID50/mL</td><td colspan="1" rowspan="1">37.67</td></tr><tr><td colspan="1" rowspan="1">H7N2 (Human)</td><td colspan="1" rowspan="1">A/New York/107/2003</td><td colspan="1" rowspan="1">1×10-10Dilution of Stock</td><td colspan="1" rowspan="1">35.41</td></tr><tr><td colspan="1" rowspan="3">Seasonal H1N1 (Human)</td><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">36.00</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">37.74</td></tr><tr><td colspan="1" rowspan="1">A/1/Denver/1/57</td><td colspan="1" rowspan="1">0.5 TCID50/mL</td><td colspan="1" rowspan="1">37.82</td></tr><tr><td colspan="1" rowspan="3">Seasonal H3N2 (Human)</td><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">50 TCID50/mL</td><td colspan="1" rowspan="1">35.57</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">42.50a</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">39.54</td></tr><tr><td colspan="1" rowspan="3">2009 swine lineage H1N1 (Human)</td><td colspan="1" rowspan="1">A/England/195/2009</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">37.47</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">500 EID50/mL</td><td colspan="1" rowspan="1">36.96</td></tr><tr><td colspan="1" rowspan="1">A/New York/18/2009</td><td colspan="1" rowspan="1">50 EID50/mL</td><td colspan="1" rowspan="1">36.43</td></tr></table>

a Cp calls of ${ \geq } 4 0 . 0 0 $ are assigned a final $\mathrm { C p }$ value of 42.50 cycles.

Influenza B Strains Detected by the JBAIDS Flu B Assay   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Lowest ConcentrationDetected</td><td rowspan=1 colspan=1>Cp</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>34.48</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>35.50</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>36.70</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>35.65</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>35.21</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>35.13</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>34.30</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>33.17</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>34.86</td></tr></table>

The JBAIDS Influenza A & B Detection Kit correctly identified all 25 Influenza A strains and 9 Influenza B strains included in the inclusivity panels. The lowest concentration detected ranged from 0.5 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ to $5 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for the Influenza A assay and $0 . 5 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ to $5 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for Influenza B. Variation in the lowest concentration detected may be due to sequence variation. However an in silico investigation of available sequences did not identify any significant mismatches with the assays primers or probes. Another possible cause for the variation is differences in the quantification method of the virus. Quantification using $\mathrm { E I D } _ { 5 0 }$ or $\mathrm { T I C D } _ { 5 0 }$ measures the infectivity and cytotoxicity or lethality of an organism in tissue culture and is therefore subject to many influences (strain-to-strain differences in infectivity, viability of the original material, culturing conditions, etc.). Quantification by infectivity assay will not be equivalent between strains or organisms. Also, since the measurand for the JBAIDS assays is starting amount of target nucleic acid, nucleic acid concentrations established in $\mathrm { E I D } _ { 5 0 }$ or $\mathrm { T C I D } _ { 5 0 }$ can vary dramatically between organism stocks, but may not actually reflect significant differences in sensitivity of detection by target nucleic acid concentration.

Analytical Specificity (Exclusivity) testing involved the following organisms: 1) Nontarget influenza types (those not specific to the type assay); 2) Non-influenza viruses; and 3) Other closely related organisms that colonize the upper respiratory tract, cause respiratory symptoms, are common skin flora or laboratory contaminants, or are microorganisms for which much of the population may have been infected.

Each organism was spiked at a high level (when possible, viruses were spiked at $1 0 ^ { 5 } \times$ provided stock units, and other organisms were spiked at $1 0 ^ { 6 } \times$ provided stock units; otherwise organisms were spiked at the highest allowable concentration) into a simulated NPS (sNPS) sample prior to extraction using the Roche MagNA Pure Nucleic Acid Isolation Kit I and testing with the JBAIDS Influenza A & B Detection Kit.

The following 3 tables provide the exclusivity panels that were used to evaluate the test system analytical specificity or exclusivity:

Exclusivity Panel: Influenza B Strain Evaluated with the JBAIDS Flu A Assay   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>4.54E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>5.09E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>1.50E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=1>3.14E+04 TCID50/mL</td></tr></table>

Exclusivity Panel: Influenza A Strains Evaluated with the JBAIDS Flu B Assay   

<table><tr><td colspan="1" rowspan="1">Subtype (Host)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">ConcentrationTested</td></tr><tr><td colspan="1" rowspan="1">H2N2 (Avian)</td><td colspan="1" rowspan="1">A/chicken/Pennsylvania/298101-4/2004</td><td colspan="1" rowspan="1">3.16E+07 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Avian)</td><td colspan="1" rowspan="1">A/MAL/ALB/16/87</td><td colspan="1" rowspan="1">1.72E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H4N8 (Avian)</td><td colspan="1" rowspan="1">A/chicken/Alabama/1975</td><td colspan="1" rowspan="1">1.00E+08 EID50/mL</td></tr><tr><td colspan="1" rowspan="1">H5N1 (Avian-HumanRecombinant)</td><td colspan="1" rowspan="1">A/Vietnam/1203/2004(H5N1)-PR8</td><td colspan="1" rowspan="1">3.16E+07 EID50/mL</td></tr><tr><td colspan="1" rowspan="1">H5N1 (Avian)</td><td colspan="1" rowspan="1">A/DK/PA/4560069-9/06</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H7N3 (Avian)</td><td colspan="1" rowspan="1">A/TY/UT/24721-10/95</td><td colspan="1" rowspan="1">3.06E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H6N2 (Avian)</td><td colspan="1" rowspan="1">A/Chicken/CA/32213-1/2000</td><td colspan="1" rowspan="1">1.26E+07 EID50/mL</td></tr><tr><td colspan="1" rowspan="1">H9N2 (Avian)</td><td colspan="1" rowspan="1">A/Turkey/Wisconsin/1966</td><td colspan="1" rowspan="1">5.60E+07EID50/mL</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Canine)</td><td colspan="1" rowspan="1">A/canine/Florida/43/2004</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H3N8 (Equine)</td><td colspan="1" rowspan="1">A/Equine/Ohio/01/2009</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Swine)</td><td colspan="1" rowspan="1">A/swine/Wisconsin/125/1997</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Swine)</td><td colspan="1" rowspan="1">A/SW/GB/19582/92</td><td colspan="1" rowspan="1">5.64E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H3N2 (Swine)</td><td colspan="1" rowspan="1">A/SW/IA/1/99</td><td colspan="1" rowspan="1">1.41E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Human of swinelineage)</td><td colspan="1" rowspan="1">A/Maryland/12/1991</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H1N1 (Human of swinelineage)</td><td colspan="1" rowspan="1">A/Iowa/1/2006</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">H7N2 (Human)</td><td colspan="1" rowspan="1">A/New York/107/2003</td><td colspan="1" rowspan="1">30 μl of an unknownconcentration into 1mL</td></tr><tr><td colspan="1" rowspan="8">Seasonal H1N1 (Human)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">4.24E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.00E+05TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">4.24E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/1/Denver/1/57</td><td colspan="1" rowspan="1">4.24E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">4.24E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="7">Seasonal H3N2 (Human)</td><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5.10E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4.24E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Alice (VR-776)</td><td colspan="1" rowspan="1">4.24E+033 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/MRC-2 recomb. (VR-777)</td><td colspan="1" rowspan="1">7.36E+033 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">7.36E+03TCID50/mL</td></tr><tr><td colspan="1" rowspan="11">2009 swine lineage H1N1(Human)</td><td colspan="1" rowspan="1">Swine NY/02/2009</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">7.36E+03 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">3.78E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">1.00E+05 EID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/California/8/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/California/04/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Texas/48/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/Washington/29/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/South Carolina/18/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/England/195/2009</td><td colspan="1" rowspan="1">4.74E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">A/North Carolina/39/2009</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td></tr></table>

Exclusivity Panel: Non-influenza Viruses and Organisms   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Bacteria/Fungi</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">4.20E+07 copies/mL</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">7.35E+03 TCID50/mL</td><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">6.57E+04 TCID50/mL</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">5.10E+03 TCID50/mL</td><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">7.80E+04 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">1.50E+04 TCID50/mL</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">7.35E+03 TCID50/mL</td><td colspan="1" rowspan="1">Mycoplasma pneumonia</td><td colspan="1" rowspan="1">1.69E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">5.10E+03 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles Virus (Rubeola)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">4.53E+04 TCID50/mL</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">1.50E+04 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">7.59E+05 CFU/mL</td></tr></table>

All 9 Influenza B viruses tested at high concentrations gave the expected negative results for the Flu A assay. All 42 Influenza A viruses similarly gave the expected negative results for the Flu B assay. All 36 non-influenza organisms, tested at high concentrations, (bacteria or fungi spiked at $1 0 ^ { 6 } \mathrm { C F U / m L }$ or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and viruses spiked at $1 0 ^ { 3 } - 1 0 ^ { 5 }$ copies $/ \mathrm { m L }$ or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) gave the expected negative results for both the Flu A and Flu B assays.

# g. Assay cut-off:

Each JBAIDS test is analyzed with the data analysis software module called Detector. The module determines the outcome of the tests conducted on the samples. The module first analyzes the data from each capillary independently. Then the replicates of the samples are analyzed together (if applicable). Finally the module interprets the outcome of these analyses for the sample and its controls.

The analysis module algorithm consists of two parts. The first identifies samples that are obviously positive or negative (Obvious Tests), and second that computes a refined estimate of the samples status when the sample is not called an obvious positive or obvious negative (Expert Test).The algorithms are based on an Expert System approach to determining positive and negative samples, and is deterministic meaning that every sample will have the same outcome every time the algorithm evaluates the sample. The purpose of Obvious Tests is to call those samples that are clearly positive or negative. In this way, only non-obvious samples will need to be processed by the more computationally involved detector algorithms. Obvious tests produce internal scores and compare them against fixed thresholds to return a positive/negative/uncertain call. Expert Tests algorithm is based on mathematically modeling the expected shapes of the amplification curves of positive and negative samples. The model itself is based on tests for nine distinct characteristics in the amplification curves and it assigns nine test scores to each curve. After the tests are scored, the amplification curve is scored as a weighted sum of the tests.

Thresholds for the Obvious Tests and weights in the Expert Tests for Detector were determined using training data sets and numerical optimization to pick parameters that ensure Detector minimizes the number of error observed in real-world data.

Validation data from JBAIDS Customer Validation Testing, and Dugway Operational Assessment were gathered and evaluated with Detector to validate the analysis module algorithm. These data contain samples from five separate data sets: typical data, atypical data, negative dominated, RNA data and Dugway field data. The data were gathered from varied sources, including a variety of assays, instruments and users, to obtain

examples of all possible JBAIDS data. There are 28,467 samples in the data sets. $9 9 . 6 \%$ of the samples were correctly called by the Detector. $0 . 1 9 \%$ of the samples were called “Uncertain” and $0 . 2 1 \%$ of the samples were incorrectly called.

# h. Interfering Substances:

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors and technique-specific substances (substances that could be introduced into the PCR reaction as contaminants during sample purification or during reaction setup) affect the performance of the JBAIDS Influenza A & B Detection Kit. The concentration of substances tested represented a relevant concentration in accordance with CLSI EP7-A2 Interference Testing in Clinical Chemistry Approved Guideline, or for test substances not listed in the CLSI document, concentrations were determined by evaluating the scientific literature or references from other IVD package inserts. For each endogenous, exogenous and technique specific test substance, one sNPW specimen containing a representative Influenza A or Influenza B virus (A/New Caledonia/20/1999 or B/Ohio/1/2005) at a concentration equivalent to $5 { \times } \mathrm { L o D }$ was spiked with the appropriate amount of test substance.

Note: sNPW specimens spiked with the representative Influenza B virus (B/Ohio/1/2005) were also co-spiked with a representative seasonal influenza A H3N2 virus strain (A/New York/55/2004) for more efficient interfering substances testing of the JBAIDS Influenza A Subtyping Kit. Refer to the $^ { \circ \mathfrak { c } } \mathrm { C o } { } .$ -spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus

In each round of purifications, an influenza-containing sNPW sample that had not been spiked with the interfering substance was also included. This sample served as a control to which the other specimens were compared. All specimens, except those containing Roche MagNA Pure Compact Nucleic Acid Isolation Kit I technique specific substances (i.e., MagNA Pure buffers), were purified using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact System. Each purified sample was tested with the influenza assay (Flu A or Flu B) corresponding to the virus with which it was spiked, and also tested with the Flu SC assay. The Roche MagNA Pure Compact Nucleic Acid Isolation Kit I technique-specific substances that could be introduced during sample purification (i.e., MagNA Pure buffers) were added to the purified samples immediately prior to influenza assay reaction setup. Substances were considered to be potential inhibitors if the results for the tested assays were negative.

Note: Only sNPW samples purified using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact System were evaluated. NPS samples purified with the IT 1-2-3 Platinum Path Sample Purification Kit were previously evaluated with the FDA cleared JBAIDS Influenza A/H5 (Asian lineage) Detection System (k100287). The Platinum Path kit was shown to remove both endogenous and exogenous substances from NPS and throat swab samples. In addition, the wash and elution buffers contained in the IT 1-2-3 Platinum Path Sample Purification Kit (technique-specific substances) did not interfere with detection of the

JBAIDS Influenza A/H5 assays.

List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=5>Exogenous</td></tr><tr><td rowspan=1 colspan=1>Blood (with Na Citrate) Mucin(bovine submaxillary gland,type I-S)Human Genomic DNA</td><td rowspan=1 colspan=4>TobramycinMupirocinFluMist® intranasal vaccineDexamethasoneFluticasonepropionate Snuff(nasal tobacco)Zicam Nasal Gel</td><td rowspan=1 colspan=1>Nasal Decongestant Spray (Phenylephrine HCl 1.0%)Nasal Decongestant spray (Oxymetazoline HCl 0.05%)Saline nasal spray with preservatives (0.65% NaClPhenylcarbinol Benzalkonium Chloride)Analgesic OintmentPetroleum and Glycerin topical gel (Chlorseptic®)</td></tr><tr><td rowspan=1 colspan=1>Solvents Used1</td><td rowspan=1 colspan=5>Technique-specific Substances</td></tr><tr><td rowspan=2 colspan=1>WaterAcetoneDMSO</td><td rowspan=2 colspan=3>Disinfecting WipesEthanolRNAseOutMagNA Pure Buffers fromcartridge wells #1-6</td><td rowspan=2 colspan=2>Swabs•  Copan 168C (rayon/twisted aluminum shaft)Copan 503CS01 (flocked nylon/plastic shaft)Copan 175KS01 (polyester/aluminum shaft)Copan 502CS01 (flocked nylon/plastic shaft)Millipore 519CS01M (flocked nylon/plasticshaft) Viral Transport Media•   Remel M5Remel M6</td></tr><tr><td rowspan=1 colspan=1>Ethanol</td></tr></table>

1 These are solvents used to dissolve potentially interfering substances in preparation for testing.

Test Concentrations for Endogenous Substances   

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EN1</td><td rowspan=1 colspan=1>Blood(with Na Citrate)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>EN2</td><td rowspan=1 colspan=1>Mucin(bovine submaxillary gland, type I-S)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>Water</td></tr></table>

Test Concentrations for Exogenous Substances   

<table><tr><td colspan="1" rowspan="1">IS#</td><td colspan="1" rowspan="1">Test Substance</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Solvent</td></tr><tr><td colspan="1" rowspan="1">EX1</td><td colspan="1" rowspan="1">Tobramycin(systemic antibiotic)</td><td colspan="1" rowspan="1">0.6 mg/mL</td><td colspan="1" rowspan="1">Water</td></tr><tr><td colspan="1" rowspan="1">EX2</td><td colspan="1" rowspan="1">Mupirocin(active ingredient in anti-bacterial ointment)</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX3Blank</td><td colspan="1" rowspan="1">FluMist intranasal vaccine(2009-2010 Formula)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPW Blank</td></tr><tr><td colspan="1" rowspan="1">EX4</td><td colspan="1" rowspan="1">Saline Nasal Spray with Preservatives0.65% NaClPhenylcarbinol Benzalkonium Chloride</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX5</td><td colspan="1" rowspan="1">Nasal Decongestant SprayOxymetazoline HCl 0.05%(also contains Benzalkonium chloride, menthol, eucalyptol,camphor, benzyl alcohol and phosphate buffers)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX6</td><td colspan="1" rowspan="1">Analgesic ointment (Vicks®VapoRub®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX7</td><td colspan="1" rowspan="1">Petroleum Jelly (Vaseline®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX8</td><td colspan="1" rowspan="1">Zicam® Nasal Gel</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX9</td><td colspan="1" rowspan="1">Snuff (nasal tobacco)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX10</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.084 μg/mL</td><td colspan="1" rowspan="1">Acetone</td></tr><tr><td colspan="1" rowspan="1">EX11</td><td colspan="1" rowspan="1">Fluticasone propionate</td><td colspan="1" rowspan="1">0.22 μg/mL</td><td colspan="1" rowspan="1">DMSO</td></tr><tr><td colspan="1" rowspan="1">EX12</td><td colspan="1" rowspan="1">Nasal Decongestant SprayPhenylephrine HCl 1.0%(also contains benzalkonium chloride, anhydrous citric acid,purified water, sodium chloride, and sodium citrate)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX13</td><td colspan="1" rowspan="1">Petroleum and Glycerin topical gel(Chlorseptic®)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr></table>

Test Concentrations for Technique-Specific Substances   

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=3>Test Substance</td><td rowspan=1 colspan=1>TestConcentration</td></tr><tr><td rowspan=1 colspan=1>TS1</td><td rowspan=1 colspan=3>Disinfecting Wipes</td><td rowspan=1 colspan=1>1/4 - 1/2 in2</td></tr><tr><td rowspan=1 colspan=1>TS2</td><td rowspan=1 colspan=3>Ethanol</td><td rowspan=1 colspan=1>7%</td></tr><tr><td rowspan=1 colspan=1>TS3</td><td rowspan=1 colspan=3>RNaseOut</td><td rowspan=1 colspan=1>1 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS4</td><td rowspan=4 colspan=2>MagNA PureReagentCartridge</td><td rowspan=1 colspan=1>Well 1: Proteinase K solution</td><td rowspan=1 colspan=1>10 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS5</td><td rowspan=1 colspan=1>Well 2: Lysis Buffer solution</td><td rowspan=1 colspan=1>10 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS6</td><td rowspan=1 colspan=1>Well 3: Magnetic Glass Particles</td><td rowspan=1 colspan=1>10 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS7</td><td rowspan=1 colspan=1>e</td><td rowspan=1 colspan=1>Wells 4,5: Wash Buffer I</td><td rowspan=1 colspan=1>10 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS8</td><td rowspan=2 colspan=2>Contents</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Well 6: Wash Buffer II</td></tr><tr><td rowspan=1 colspan=1>TS9</td><td rowspan=1 colspan=1>Well 7: Wash Buffer III</td><td rowspan=1 colspan=1>10 % v/v</td></tr><tr><td rowspan=1 colspan=1>TS10</td><td rowspan=5 colspan=2>Swabs</td><td rowspan=1 colspan=1>Copan 168C(rayon/twisted aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>TS11</td><td rowspan=1 colspan=1>Copan 503CS01(flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>TS12</td><td rowspan=1 colspan=1>Copan 175KS01(polyester/aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>TS13</td><td rowspan=1 colspan=1>Millipore 519CS01M(flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>TS14</td><td rowspan=1 colspan=1>Copan 502CS01(flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr></table>

<table><tr><td rowspan=1 colspan=1>TS15</td><td rowspan=3 colspan=1>ViralTransportMedium</td><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TS16</td><td rowspan=1 colspan=1>Remel M6</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>TS17</td><td rowspan=1 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Test Concentrations for Human Genomic DNA

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=2>Test Substance</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>DNA1</td><td rowspan=4 colspan=2>Human genomic DNA263 ng/μL</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>20 ng/μL (500 ng)</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>DNA2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>200 ng/μL</td><td rowspan=1 colspan=1>2 ng/μL (50 ng)</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>DNA3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20 ng/μL</td><td rowspan=1 colspan=1>0.2 ng/μL (5 ng)</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>DNA4</td><td rowspan=1 colspan=1>2 ng/μL</td><td rowspan=1 colspan=1>0.02 ng/μL (0.5 ng)</td><td rowspan=1 colspan=1>sNPW sample</td></tr></table>

Of all of the endogenous and exogenous substances tested, only snuff (nasal tobacco) has the potential to cause inaccurate negative test results due to inhibition. Of the technique- specific substances tested, improper introduction of contents from four of the MagNA Pure reagent cartridge wells to purified samples have the potential to cause inaccurate negative test results due to inhibition:

MagNA Pure Proteinase K Solution (well 1) MagNA Pure Lysis Solution (well 2) MagNA Pure Magnetic Glass Particles (well 3) MagNA Pure Wash Buffer I (well 4)

These substances, along with ethanol, bleach, and DNAZap™, which are known PCR inhibitors, are included in the package insert as potentially interfering substances.

In addition, the study also demonstrated that the JBAIDS influenza A and B assays react with the viral material contained in the 2009-2010 version of the FluMist intranasal influenza vaccine (MedImmune). When NPS and NPW specimens are collected from individuals who have recently received a dose of the nasal vaccine, the Flu A and Flu B assays may react with the vaccine material to give inaccurate positive results.

Note: The material being tested (FluMist® Influenza Vaccine Live, Intranasal; Intranasal Spray, 2009-2010 Formula) was provided by the manufacturer (MedImmune) in individual sprayers, each containing a single $0 . 2 \mathrm { m L }$ dose. Each dose contained approximately $1 0 ^ { \overline { { 6 } } . 5 - 7 . \bar { 5 } }$ FFU (fluorescence focus units) each of three live, attenuated influenza virus reassortants including:

• A/South Dakota/6/2007 (H1N1) (A/Brisbane/59/2007-like) • A/Uruguay/716/2007 (H3N2) (A/Brisbane/10/2007-like) • B/Brisbane/60/2008.

Non-viral components of the vaccine listed in the provided materials include: monosodium glutamate, gelatin, arginine, sucrose, gentamicin sulfate, monobasic potassium phosphate and dibasic potassium phosphate.

# i. Carry-Over Contamination:

An analytical study was carried out to evaluate the frequency of carryover in the JBAIDS Influenza A & B Detection Kit when it is used as instructed by the product insert. Significant levels of carryover will result in false positive test results for negative samples that are contaminated by adjacent positive samples. For the JBAIDS Influenza A & B Detection Kit, carryover could occur during sample purification or reagent setup.

To assess the potential for carryover, sample sets consisting of negative and high positive samples were processed in an alternating pattern using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact system and then tested with the JBAIDS Influenza A & B Detection Kit.

Note: The potential for carryover in samples purified with the IT 1-2-3 Platinum Path Purification Kit was previously evaluated with JBAIDS Influenza A/H5 (Asian lineage) Detection Kit (k100287) and demonstrated that significant levels of carryover do not occur when samples are processed and tested according to the instructions in the package insert.

Positive sNPS samples were spiked with Influenza A H3N2 strain A/New York/55/2004 at approximately $2 0 0 0 \times$ LoD $( \sim 1 . 0 \times 1 0 ^ { 4 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L } )$ ). Five independent sample sets were tested for a total of 20 positive and 20 negative samples. Positive and negative samples were purified in an alternating pattern, and the PCR reactions for the positive and negative samples were set up in the order in which the samples were processed. The frequency of false positive and uncertain results for the negative samples was evaluated using the Flu A assay as the representative assay from the JBAIDS Influenza A & B Detection Kit.

All positive samples resulted in early Flu A assay Cp values $( < 2 4 . 0 0 $ cycles) and robust $\mathrm { F } _ { \mathrm { m a x } }$ values $( > 1 9 . 0 0 )$ . All negative samples produced negative results, with no detectable $\mathrm { C p }$ values. No carryover was observed in this evaluation. The test results are shown in the following table:

Flu A Assay Carryover Results for sNPS Samples Purified with the Magna Pure Compact Nucleic Acid Isolation Kit I Purification Kit   

<table><tr><td colspan="1" rowspan="2">TrialNumber</td><td colspan="1" rowspan="2">Number ofPositiveSamples</td><td colspan="2" rowspan="1">Flu A Sample Results forPositive Samples</td><td colspan="1" rowspan="2">Number ofNegativeSamples</td><td colspan="2" rowspan="1">Flu A Sample Results forNegative Samples</td></tr><tr><td colspan="1" rowspan="1">Mean Cp(SD)</td><td colspan="1" rowspan="1">Mean Fmax(SD)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23.47 (0.16)</td><td colspan="1" rowspan="1">19.51 (0.65)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23.69 (0.26)</td><td colspan="1" rowspan="1">19.34 (0.83)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23.28 (0.23)</td><td colspan="1" rowspan="1">21.95 (0.60)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23.39 (0.43)</td><td colspan="1" rowspan="1">21.99 (1.65)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">23.41 (0.32)</td><td colspan="1" rowspan="1">19.48 (1.80)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">23.45 (0.29)</td><td colspan="1" rowspan="1">20.45 (1.67)</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">20</td></tr></table>

Overall, sample purification and reaction setup in this carryover study resulted in no false positive results when samples were processed in accordance with the purification kit and reaction setup directions. PCR is a very sensitive technique and false positives caused by contamination are a recognized risk. Users are reminded in the product package insert that failure to follow protocol and improper technique may result in false positive results.

# j. Validation of Simulated Sample Matrices

Nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS) specimens are the two sample matrices evaluated with the JBAIDS Influenza A & B Detection Kit and used for most of the non-clinical evaluations. However, given the need for large numbers of individual samples without influenza virus, simulated NPS (sNPS) and simulated NPW (sNPW) samples were developed and validated to prevent inaccurate test results arising from the presence of influenza viruses in human derived nasopharyngeal specimens. Simulated samples were comprised of human HeLa cells diluted to concentrations in viral transport medium (VTM) that produced similar Cp values as the Flu SC assay produced when testing human derived NPS and NPW samples.

HeLa cell concentrations of $1 0 ^ { 3 . 2 }$ cells/mL and $1 0 ^ { 4 . 3 }$ cells/mL for sNPW and sNPS sample matrix types, respectively, resulted in similar $\mathrm { C p }$ values for the Flu SC assay as human derived NPW and NPS specimens. To confirm that the simulated sample matrix types did not have any advantageous or detrimental effects on the performance of the system, multiple aliquots of simulated and human derived samples were spiked at the LoD with a representative seasonal influenza A virus (H3N2 strain A/New York/55/2004) and tested with the applicable influenza and control assays (Flu A, Flu A H3, and Flu SC). Simulated and human derived samples exhibited equivalent rates of detection. Mean Cp and $\mathrm { F } _ { \mathrm { m a x } }$ values were also comparable between the human derived and simulated samples.

# k. Co-spiked Specimen Validation

For efficient use of resources during non-clinical testing of the JBAIDS Influenza A & B Detection and Influenza A Subtyping Kits, Influenza A H3N2 and Influenza B viruses were co-spiked in some studies. Specifically, co-spiked samples were used in the following analytic evaluations: Reproducibility, Sample Transport and Storage, and Interfering Substances.

An analytical study was carried out to validate the use of specimens co-spiked with Influenza A H3N2 and Influenza B viruses. This study was performed by evaluating single-spiked and co-spiked samples containing Influenza A H3N2 A/New York/55/2004 and/or Influenza B B/Ohio/01/2005. NPW specimens purified with the Platinum Path kit were evaluated in this study because the Cp values for the Flu A assay in NPW samples, purified with Platinum Path kit, were greater than Cp values for the other sample type/purification kit combinations. The first part of this study evaluated freshly collected NPW samples spiked with an influenza A H3N2 virus at LoD only (single spike) or additionally spiked with influenza B virus at $5 0 0 \times \mathrm { L o D }$ (cospiked). The second part of this study evaluated the opposite, NPW samples spiked with influenza B virus at $1 { \times } \mathrm { L o D }$ (single spike) or further spiked with influenza A H3N2 at $5 0 0 \times \mathrm { L o D }$ (co-spiked).

For each part of the study, 60 single and 60 co-spiked samples were prepared and evaluated. In particular, pooled NPW samples were spiked with virus at $1 { \times } \mathrm { L o D }$ . Half of this pool was co-spiked with the alternate virus at ${ 5 0 0 \times } \mathrm { L o D }$ , resulting in one single spiked pool and one co-spiked pool. Following spiking, the samples were purified using the Platinum Path kit and then tested with appropriate freeze-dried reagent test assays (Flu A, Flu B, and Flu A H3). A total of 240 samples were evaluated.

For the samples spiked with influenza B at $1 { \times } \mathrm { L o D }$ , all single (60) and co-spiked (60) samples were detected with the Flu B assay. In addition, the influenza A H3N2 virus at $5 0 0 \times \mathrm { L o D }$ was detected in $6 0 / 6 0$ samples by the Flu A and Flu A H3 assays. These results demonstrate that $100 \%$ detection was achieved for single (influenza B at $1 { \times } \mathrm { L o D } )$ and co- spiked (influenza B at $1 \times \mathrm { L o D } /$ influenza A H3N2 at ${ 5 0 0 } { \times } \mathrm { L o D }$ ) samples. For the Flu A assay, samples spiked with influenza A H3N2 at $1 { \times } \mathrm { L o D }$ were detected in 59/60 single- spiked and 58/60 co-spiked samples, resulting in 98.3 and $9 6 . 7 \%$ detection rates, respectively. For the Flu A H3 assay the same samples were positive for 60/60 single- spiked and 59/60 co-spiked samples, resulting in 100 and $9 8 . 3 \%$ detection rates, respectively. In addition, the influenza B virus at ${ 5 0 0 \times } \mathrm { L o D }$ was detected in 60/60 samples by the Flu B assay.

Differences in mean $\mathrm { C p }$ values between single- and co-spiked samples did not vary by more than 0.19 cycles for the three assays, and mean $\mathrm { F } _ { \mathrm { m a x } }$ values for co-spiked samples were reduced by less than $5 \%$ from mean $\mathrm { F } _ { \mathrm { m a x } }$ values for single-spiked samples. The quantitative differences in mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values between single- and co-spiked samples were lower than the standard deviations for each assay and spike set.

The study results demonstrate that there is no significant quantitative or qualitative difference between specimens single spiked or co-spiked with influenza B and influenza A H3N2 viruses.

# l. Fresh vs. Frozen Specimen Study

The JBAIDS Influenza A & B Detection Kit includes assays for the detection of Influenza A and B viruses in NPS and NPW specimens collected from individuals exhibiting influenza-like-illness. A prospective clinical evaluation of this test kit was performed in order to establish the clinical sensitivity and specificity of the assays contained in the JBAIDS Influenza A & B Detection Kit. The prospective clinical evaluation was performed from December 2010 through April 2011 at five geographically diverse locations. During this study, sufficient positive and negative samples were obtained to establish the clinical sensitivity and specificity for detection of Influenza A 2009 H1N1 and Influenza A/H3 viruses. However, seasonal Influenza A/H1 did not circulate during the trial period. In addition, it was unclear if sufficient influenza B positive samples would be obtained to establish the performance of the Influenza B assay, especially when testing NPS samples.

To account for the possibility that frozen archived samples might be required to supplement the prospective clinical trial data, an analytical study was performed to establish that the results obtained with frozen samples are equivalent to those obtained when testing freshly collected NPS or NPW samples.

Pools of freshly collected NPS and NPW specimens were spiked at $1 0 0 \times \mathrm { L o D }$ , $1 0 { \times } \mathrm { L o D }$ , $3 { \times } \mathrm { L o D }$ , and $1 { \times } \mathrm { L o D }$ with a representative Influenza A/ H1 virus strain (A/New Caledonia/20/1999) or Influenza B virus strain (B/Ohio/1/2005). The spiked pools were then aliquoted. Samples that were to be tested after freezing were aliquoted from each pool and stored at $- 8 0 ^ { \circ } \mathrm { C }$ for at least 24 hours. The remainder of each pool was aliquoted for immediate analysis. Testing of sample sets consisted of purifying individual aliquots with either the Platinum Path or the MagNA Pure kits and assaying with the Flu A or Flu B assays. The following table lists the number of samples tested at each virus concentration:

Virus Concentrations and Numbers of Fresh and Frozen Samples Evaluated with both Purification Methods   

<table><tr><td rowspan=2 colspan=1>Virus Spike Level</td><td rowspan=1 colspan=2>Influenza Virus Concentration(Total Fresh and Frozen Replicates)</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td><td rowspan=1 colspan=1>B/Ohio/1/2005</td></tr><tr><td rowspan=1 colspan=1>Unspiked</td><td rowspan=1 colspan=1>N/A (6)</td><td rowspan=1 colspan=1>N/A (6)</td></tr><tr><td rowspan=1 colspan=1>100×LoD</td><td rowspan=1 colspan=1>5000 EID50/mL (6)</td><td rowspan=1 colspan=1>500 EID50/mL(6)</td></tr><tr><td rowspan=1 colspan=1>10×LoD</td><td rowspan=1 colspan=1>500 EID50/mL (6)</td><td rowspan=1 colspan=1>50 EID50/mL (6)</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>150 EID50/mL (24)</td><td rowspan=1 colspan=1>15 EID50/mL (24)</td></tr><tr><td rowspan=1 colspan=1>1×LoD</td><td rowspan=1 colspan=1>50 EID50/mL (24)</td><td rowspan=1 colspan=1>5 EID50/mL (24)</td></tr></table>

Successful validation of frozen samples required that the JBAIDS test results were equivalent to the results obtained when testing freshly collected samples. Frozen samples were considered acceptable to use in the clinical evaluation if the JBAIDS results are at least $90 \%$ in agreement between fresh and frozen samples with a lower $9 5 \%$ CI bound of $8 6 \%$ . If this statistical confidence was not reached with 60 samples, additional samples were evaluated. For informational purposes only, the crossing point $\textstyle ( \mathrm { C p } )$ and relative maximum fluorescence $\mathrm { ( F _ { m a x } ) }$ values were evaluated for the fresh and frozen samples. Ideally the $\mathrm { C p }$ values should not vary by more than 3 cycles and the $\mathrm { F } _ { \mathrm { m a x } }$ values should not be more than $50 \%$ different between the two data sets. The 3 cycle $\mathrm { C p }$ and $50 \%$ $\mathrm { F } _ { \mathrm { m a x } }$ differences allow for intrinsic system variability in JBAIDS Influenza A & B Detection Kit. If the observed $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ differences were greater than these, the data were reviewed to determine if using frozen samples interferes with

detection.

For the Flu A assay, JBAIDS test results for fresh and frozen spiked NPS samples were in $100 \%$ agreement $( 9 5 \% \mathrm { C I } 9 6  – 1 0 0 \% )$ . Fresh and frozen NPW samples also exhibited $100 \%$ agreement $( 9 5 \% \mathrm { C I } 9 6  – 1 0 0 \% )$ for all virus spike levels. The acceptance criteria were met for the equivalence of fresh and frozen sample sets. The $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values were also compared between fresh and frozen influenza A spiked NPS and NPW samples, and they were within the $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ ranges outlined in the acceptance criteria. Differences in $\mathrm { C p }$ values for fresh and frozen samples ranged from $0 . 5 6 \mathrm { C p s }$ earlier to $0 . 6 9 \mathrm { C p s }$ later for the Flu A assay with Platinum Path-purified samples and $0 . 4 5 \mathrm { C p s }$ earlier to $0 . 7 3 \mathrm { C p s }$ later with MagNA Pure-purified samples. $\mathrm { F } _ { \mathrm { m a x } }$ values for frozen, Platinum Path purified samples were 4 to $44 \%$ greater than fresh samples. $\mathrm { F } _ { \mathrm { m a x } }$ values for frozen MagNA Pure- purified samples were $1 7 \%$ less to $22 \%$ greater than fresh samples.

The Flu B assay results for fresh and frozen spiked NPS samples were in $100 \%$ agreement ( $9 5 \%$ CI $9 6 \mathrm { - } 1 0 0 \%$ for all virus spike levels. The acceptance criteria were met for the equivalent fresh and frozen sample sets. The $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values were also compared between fresh and frozen NPS samples, and they were within the $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ ranges outlined in the acceptance criteria. Differences in $\mathrm { C p }$ values for fresh and frozen samples ranged from 0.31 Cps earlier to 0.01 Cps later for the Flu B assay with Platinum Path purified samples and $0 . 1 4 \mathrm { C p s }$ earlier to $0 . 1 4 \mathrm { C p s }$ later for MagNA Pure purified samples. $\mathrm { F } _ { \mathrm { m a x } }$ values for frozen, Platinum Path purified samples were $1 7 \%$ less to $1 3 \%$ greater than fresh samples. $\mathrm { F } _ { \mathrm { m a x } }$ values for frozen, MagNA Pure purified samples were $8 \%$ less to $6 \%$ greater than fresh samples.

Note: When this study was initiated there were preliminary data from the clinical trial sites to suggest that Influenza B positive NPW samples would be sufficiently represented. Therefore, only Influenza B NPS spiked samples were examined to prepare for the possibility that this sample type would be underrepresented.

The results of the fresh vs. frozen study demonstrated that there are no significant qualitative or quantitative differences between JBAIDS test results for fresh or frozen samples spiked with Influenza A and B viruses. All acceptance criteria were met for this study. Accurate test results were obtained from fresh samples as well as samples that had been subjected to a freeze-thaw cycle.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

# a. Prospective Clinical study

The clinical performance of the JBAIDS Influenza A & B Detection Kit Assay was established during prospective studies at 5 geographically separated clinical study sites in the U.S. The study sites were located in the West (Fairfield, CA, and San Diego, CA), the Southwest (San Antonio, TX), the Midwest (Dayton, OH), and the Southeast (Fayetteville, NC). NP swab or wash samples were collected and tested during the 2010/2011 influenza season (December 2010 to April 2011). Subjects with signs and symptoms of influenza-like illness were invited to participate. Upon obtaining informed consent, an NPW or NPS specimen was collected for JBAIDS and comparator assay testing.

Written informed consent (or assent, if appropriate) was acquired from each subject and/or their parent/guardian (if under 18) at the time of enrollment into the study. After informed consent was acquired, the subject was assigned a Volunteer Identification Number (VIN). The VIN was used to de-identify the specimen used for JBAIDS testing and to provide data to the sponsor.

Patient specimens were divided into single use aliquots for testing immediately after collection. Typically, one sample aliquot was placed at $4 \mathrm { { } ^ { \circ } C }$ to be purified within one day and tested using the JBAIDS Influenza A & B Detection Kit. Specimens collected on Fridays were placed at $< - 1 5 ^ { \circ } \mathrm { C }$ over the weekend and thawed for testing on Monday. Aliquots for comparator testing were placed at $< - 1 5 ^ { \circ } \mathrm { C }$ and sent to Idaho Technology, Inc., on a weekly basis. Once received at Idaho Technology, the samples were placed at $\mathrm { < - 7 0 \mathrm { { } ^ { \circ } C } }$ until they were tested using the comparator assays.

Patient samples were purified using either the IT 1-2-3 Platinum Path Purification Kit (3 testing sites), or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in conjunction with the Roche MagNA Pure Compact system (4 testing sites), and tested with the JBAIDS Influenza A & B Detection Kit.

Comparator PCR testing using the CDC rRT- PCR Flu Panel assays was performed by ITI research associates in a blinded manner (i.e., specimens were labeled only with the subject’s VIN and the JBAIDS test results were not available to them. Specimen testing with the CDC rRT- PCR Flu Panel assays was performed in accordance with the package insert.

A total of 231 runs were performed during the JBAIDS Influenza A & B Detection Kit prospective clinical trial and 226 $( 9 7 . 8 \% )$ JBAIDS runs were completed successfully. Two runs $( 0 . 9 \% )$ that failed to complete were due to a software defect. Three $( 1 . 3 \% )$ additional runs that failed to complete were due to run controls (NCs and or PCs) failure. The first failure was a result of switching the positions of the NC and PC reactions for both the Flu A and Flu B assays when loading the capillaries into the JBAIDS carousel. The second failure occurred because the PC reactions for both the Flu A and Flu B assays failed to amplify. The third failure occurred due to an improperly seated carousel causing the amplification curves to be interpreted as negative.

During the prospective clinical study, Positive and Negative nucleic acid extraction controls (PEC and NEC, respectively) were processed with each batch of purification specimens to ensure that the purification process was effectively performed. The PEC was considered successful when all tested target assays were positive. A failed PEC was retested using the same purified sample and the results of the retest were accepted as the final result. If the retest was unsuccessful, all associated samples were considered to have invalid results and required re-purification in order to obtain a valid test result. During the clinical evaluation, 4 out of a total of 190 PECs $( 2 . 1 1 \% )$ required retesting (3 at site 1 and 1 at site 2). All 4 PEC samples were successful upon retesting.

A total of 833 subjects were enrolled in the prospective study initially. Eighteen (18) $( 2 . 2 \% )$ subjects withdrew or were withdrawn from the study for various reasons, including discrepancy in consent documents $( \mathrm { n } { = } 7 )$ , patient refusing sample collection or withdrawal from study $( \mathrm { n } { = } 7 )$ , or improper specimen collection $( \mathrm { n } { = } 4 )$ . Five (5) $( 0 . 6 \% )$ specimens were withdrawn from the study due to invalid testing results from the comparator assay. Four (4) specimens were withdrawn due to a contamination event at one of the testing sites. An additional 2 specimens were excluded from demographic and clinical performance analyses due to repeated JBAIDS sample control failures. The remaining 804 subjects (490 NPW specimens and 314 NPS specimens) were included in the demographic and clinical performance analyses.

Of the 804 prospective specimens that were included in the clinical performance analyses, $9 7 \%$ (778/804) of these specimens were successful on the first attempt (Site 1: $3 5 6 / 3 7 6 = 9 5 \%$ ; Site 2: $2 0 3 / 2 0 7 = 9 8 \%$ ; Site 3: $5 6 / 5 6 = 1 0 0 \%$ ; Site $4 ; 1 1 8 / 1 1 9 = 9 9 \%$ ; Site 5: $: 4 5 / 4 6 = 9 8 \%$ ). The remaining $3 \%$ (26/804) required retesting: “Invalid”(16/26), “Uncertain”(0/26), “SC Failure $\ ' ( 5 / 2 6 )$ , positive for both the Flu A and Flu B assays (2/26), or were re-extracted and retested due to user labeling error (3/26) (20 samples from Site 1; 4 samples from Site 2; 0 samples from Site 3; 1 sample from Site 4; and 1 sample from Site 5). All 26 samples were resolved upon a subsequent retest.

Summary of Included Subject Demographics   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td></tr><tr><td rowspan=1 colspan=2>NPS</td><td rowspan=1 colspan=1>314</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=2>NPW</td><td rowspan=1 colspan=1>490</td><td rowspan=1 colspan=1>325</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=1 colspan=2>Total</td><td rowspan=1 colspan=1>804</td><td rowspan=1 colspan=1>376</td><td rowspan=1 colspan=1>207</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>46</td></tr><tr><td rowspan=2 colspan=1>Sex</td><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>410 (51%)</td><td rowspan=1 colspan=1>192 (50.9%)</td><td rowspan=1 colspan=1>122 (58.9%)</td><td rowspan=1 colspan=1>23 (41.8%)</td><td rowspan=1 colspan=1>57 (47.9%)</td><td rowspan=1 colspan=1>16 (34.8%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>394 (49%)</td><td rowspan=1 colspan=1>185 (49.1%)</td><td rowspan=1 colspan=1>85 (41.1%)</td><td rowspan=1 colspan=1>32 (58.2%)</td><td rowspan=1 colspan=1>62 (52.1%)</td><td rowspan=1 colspan=1>30 (65.2%)</td></tr><tr><td rowspan=4 colspan=1>Agea</td><td rowspan=1 colspan=1>Mean</td><td rowspan=1 colspan=1>26.5</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>30.3</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>31.0</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>18.0</td></tr><tr><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>92.0</td><td rowspan=1 colspan=1>92.0</td><td rowspan=1 colspan=1>69.0</td><td rowspan=1 colspan=1>81.0</td><td rowspan=1 colspan=1>68.0</td><td rowspan=1 colspan=1>62.0</td></tr><tr><td rowspan=4 colspan=1>AgeRangeb</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>149 (18.5%)</td><td rowspan=1 colspan=1>88 (23.4%)</td><td rowspan=1 colspan=1>40 (19.3%)</td><td rowspan=1 colspan=1>4 (7.1%)</td><td rowspan=1 colspan=1>17 (14.3%)</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>6-21c</td><td rowspan=1 colspan=1>232 (28.9%)</td><td rowspan=1 colspan=1>80 (21.3%)</td><td rowspan=1 colspan=1>75 (36.2%)</td><td rowspan=1 colspan=1>10 (17.9%)</td><td rowspan=1 colspan=1>55 (46.2%)</td><td rowspan=1 colspan=1>12c (26.1%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>337 (41.9%)</td><td rowspan=1 colspan=1>183 (48.7%)</td><td rowspan=1 colspan=1>54 (26.1%)</td><td rowspan=1 colspan=1>35 (62.5%)</td><td rowspan=1 colspan=1>34 (28.6%)</td><td rowspan=1 colspan=1>31 (67.4%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>86 (10.7%)</td><td rowspan=1 colspan=1>25 (6.6%)</td><td rowspan=1 colspan=1>38 (18.4%)</td><td rowspan=1 colspan=1>7 (12.5%)</td><td rowspan=1 colspan=1>13 (10.9%)</td><td rowspan=1 colspan=1>3 (6.5%)</td></tr></table>

The following table shows the summary of positive percent agreement (PPA) and negative percent agreement (NPA) for the JBAIDS Influenza A & B Detection Kit comparing to the comparator CDC rRT- PCR Flu Panel assays, stratified by specimen type and extraction method:

JBAIDS Influenza A & B Detection Kit Clinical Performance Summary   

<table><tr><td rowspan=1 colspan=1>InfluenzaAssay</td><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>PurificationKit</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>Flu A</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.5-100%</td><td rowspan=1 colspan=1>213/214</td><td rowspan=1 colspan=1>99.5%</td><td rowspan=1 colspan=1>97.4-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>39/39</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.8-100%</td><td rowspan=1 colspan=1>170/172</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>95.9-99.9%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>104/104</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.5-100%</td><td rowspan=1 colspan=1>383/386</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>97.8-99.8%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>91.6-100%</td><td rowspan=1 colspan=1>91/91</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.0-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>83.2-100%</td><td rowspan=1 colspan=1>160/161</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>95.6-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.2-100%</td><td rowspan=1 colspan=1>251/252</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>97.8-100%</td></tr><tr><td rowspan=6 colspan=1>Flu B</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>39/41</td><td rowspan=1 colspan=1>95.1%</td><td rowspan=1 colspan=1>83.5-99.4</td><td rowspan=1 colspan=1>237/238</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>97.7-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>29/31</td><td rowspan=1 colspan=1>93.5%</td><td rowspan=1 colspan=1>78.6-99.2%</td><td rowspan=1 colspan=1>177/180</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>95.2-99.7%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>68/72</td><td rowspan=1 colspan=1>94.4%</td><td rowspan=1 colspan=1>86.4-98.5%</td><td rowspan=1 colspan=1>414/418</td><td rowspan=1 colspan=1>99.0%</td><td rowspan=1 colspan=1>97.6-99.7%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>18/19</td><td rowspan=1 colspan=1>94.7%</td><td rowspan=1 colspan=1>74.0-99.9%</td><td rowspan=1 colspan=1>114/114</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.8-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.0-100%</td><td rowspan=1 colspan=1>175/175</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.9-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0%</td><td rowspan=1 colspan=1>79.7-99.9%</td><td rowspan=1 colspan=1>289/289</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7-100%</td></tr></table>

The following two tables show JBAIDS Influenza A & B Detection Kit clinical performance by site for NPW and NPS specimens, respectively. Due to similar performance when samples were purified with the Platinum Path and MagNA Pure nucleic acid extraction kits, data obtained for samples purified by both methods were combined.

JBAIDS Influenza A & B Detection Kit Clinical Performance by Site for NPW Specimens   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>49/49</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.8-100%</td><td rowspan=1 colspan=1>275/276</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>41/41</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.4-100%</td><td rowspan=1 colspan=1>77/78</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>93.1-100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76.8-100%</td><td rowspan=1 colspan=1>31/32</td><td rowspan=1 colspan=1>96.9%</td><td rowspan=1 colspan=1>83.8-99.9%</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>44/47</td><td rowspan=1 colspan=1>93.6%</td><td rowspan=1 colspan=1>82.5-98.7%</td><td rowspan=1 colspan=1>275/279</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>96.4-99.6%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22/22</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>84.6-100%</td><td rowspan=1 colspan=1>97/97</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.3-100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2/3</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>9.4-99.2%</td><td rowspan=1 colspan=1>42/43</td><td rowspan=1 colspan=1>97.7%</td><td rowspan=1 colspan=1>87.7-99.9%</td></tr></table>

JBAIDS Influenza A & B Detection Kit Clinical Performance by Site for NPS Specimens   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Flu A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>2.5-100%</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9-100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.2-100%</td><td rowspan=1 colspan=1>154/155</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.5-99.98%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>66.4-100%</td><td rowspan=1 colspan=1>47/47</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.5-100%</td></tr><tr><td rowspan=3 colspan=1>Flu B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.0-100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>76.2-99.9%</td><td rowspan=1 colspan=1>186/186</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>39.8-100%</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.2-100%</td></tr></table>

# b. Retrospective Clinical study

The seasonal Influenza A/H1 virus was not circulating during the 2010-2011 influenza season (http://www.cdc.gov/flu/) and was not detected during the prospective clinical evaluation of the JBAIDS Influenza A & B Detection Kit. Therefore, performance of the JBAIDS Influenza A & B Detection Kit testing seasonal Influenza A/H1 could not be established during the prospective clinical study. A retrospective clinical study was carried out at two different clinical sites to supplement the prospective evaluation data by analyzing pre-selected, archived specimens that were known to be positive for seasonal Influenza A/H1. Specimens were evaluated using both the Platinum Path and MagNA Pure nucleic acid purification kits.

A total of 56 NPS specimens were received for testing in this study. Thirty- five (35) of the specimens had previously tested positive for seasonal Influenza A/H1. These specimens were obtained from the CDC and were originally collected from medical facilities and reference laboratories where they were previously tested by CDC and FDA-cleared laboratory methods and found to contain seasonal Influenza A/H1. No demographic information was available for these specimens. Specimens may have experienced as many as three freeze/thaw cycles prior to this study. The specimens were selected only for their previous test result; there was no effort to choose specimens with specific analyte levels. An additional 21 Influenza A negative NPS specimens from patients with respiratory illness, but known to be negative for influenza, were obtained from ARUP Laboratories. Upon arrival at ITI, a 4-digit study number (VIN) was assigned to each sample and a key was created in order to randomize the specimens such that the users testing the samples were blinded as to the expected test result. Specimens were divided approximately evenly for testing between both purification kits.

Because it is possible that the provided samples had been misidentified or had degraded during storage or previous handling, the presence or absence of the expected analyte was confirmed using “validation” PCR assays. The validation PCR assays were identical to the comparator assays that were used for the prospective clinical evaluation study. Briefly, specimens were first evaluated with the CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel) for Influenza A and B. Specimens positive for Influenza A were then subject to additional subtype-specific PCR, in this case using the comparator validation assay used for the prospective clinical evaluation of the JBAIDS Influenza A Subtyping Kit, an assay

specifically designed to detect seasonal Influenza A/H1. Positive PCR reactions for the seasonal Influenza A/H1 comparator validation assay were analyzed using bidirectional sequencing for definitive confirmation. If this validation testing did not confirm the presence or absence of the expected organism in the sample, the sample was not analyzed further as this result may have reflected that the sample had been improperly labeled or handled. Validation assay operators were blinded as to the expected result for each sample. Validated specimens with confirmed identities were then tested. Specimens were purified using either the Platinum Path or MagNA Pure sample purification kits then analyzed with the JBAIDS. The Influenza A (Flu A) assay from the JBAIDS Influenza A & B Detection Kit was run in a Custom Test Method with the influenza sample control assay (Flu SC) assay.

Validation testing confirmed all 21 influenza-negative specimens and 30 of the 35 seasonal Influenza A/H1 samples. Validation PCR testing revealed that Ct values ranged from 19.2 to 32.0 (median 25.1) out of 45 cycles, demonstrating that the specimen population contained a diverse level of analyte. Half of the samples were extracted using the Platinum Path purification kit and half using MagNA pure.

The following table presents the PPA and NPA for the archived clinical specimens.   
Data from both extraction kits are combined due to identical performance.

# Performance Summary of Seasonal Influenza A/H1 Archived Clinical Specimens

<table><tr><td rowspan=1 colspan=1>InfluenzaAssay</td><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu A</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.4-100%</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.4-100%</td></tr></table>

# c. Surrogate Clinical Specimen Testing for Seasonal Influenza A/H1

The seasonal Influenza A/H1 virus was not circulating during the 2010-2011 influenza season (http://www.cdc.gov/flu/) and was not detected during the prospective clinical evaluation of the JBAIDS Influenza A & B Detection Kit. In addition, archived clinical NPW samples that had previously tested positive for the seasonal Influenza A/H1 virus were not available. Therefore, a clinical study to evaluate the performance of the JBAIDS Influenza A & B Detection Kit was supplemented using residual clinical NPS and NPW specimens spiked with a seasonal Influenza A H1N1 strain.

One hundred thirty-six (136) surrogate seasonal Influenza A H1N1 NPS and NPW clinical specimens (68 of each specimen type) were prepared using residual influenzanegative specimens that were collected during the prospective clinical trial of the JBAIDS influenza kits. These specimens had been previously evaluated with both the JBAIDS as well as comparator assays and shown to be influenza- negative. Residual specimens were spiked at ITI with a representative seasonal influenza A H1N1 (A/New Caledonia/20/1999) strain as follows:

NPS and NPW Sample Spike Levels and Replicate Numbers   

<table><tr><td rowspan=2 colspan=1>Spike Level</td><td rowspan=1 colspan=2>NPS and NPW Replicates</td></tr><tr><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>MagNA Pure</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>1×LoD</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>10×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>100×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>1,000×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=3>Total              68                 68</td></tr></table>

A study number (VIN) was assigned to each sample and a key was created in order to randomize the samples such that the users testing the samples were blinded as to the expected test result. Frozen samples were then sent to two clinical trial sites for testing. Samples were purified using either the Platinum Path or MagNA Pure sample purification kits, and then analyzed with the JBAIDS Flu A, Flu B, and Flu SC assays.

Valid JBAIDS Flu A results were obtained for 62 NPW samples. Valid results were not obtained for 6 samples: five (5) of the invalid results were due to a Flu A NC failure, and one (1) was due to a Flu SC failure and could not be retested due to limited sample volume. These 6 NPW samples were excluded from the performance calculation. Valid JBAIDS Flu B results were obtained for 67 NPW samples. One (1) sample had a result of SC Failure and could not be retested due to limited sample volume, and was excluded from the performance calculation.

Valid JBAIDS Flu A results were obtained for 66 NPS samples. Valid results were not obtained for two (2) NPS samples due to insufficient volume for retesting (one spiked and one un-spiked sample. These two (2) NPS samples were excluded from the performance calculation. Valid JBAIDS Flu B results were obtained for 65 NPS samples. Three (3) NPS samples had a result of SC Failure and could not be retested due to limited sample volume, and were excluded from the performance calculation.

The following table presents the PPA and NPA for the surrogate clinical specimens. Half of the samples were extracted using the Platinum Path purification kit and half using the MagNA pure kit.

Surrogate Seasonal Influenza A H1N1 Performance Summary   

<table><tr><td rowspan=2 colspan=3></td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>BR</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>24/25</td><td rowspan=1 colspan=1>96.0%</td><td rowspan=1 colspan=1>79.7 - 99.9%</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39.8 - 100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.1 -100%</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39.8 - 100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>53/54</td><td rowspan=1 colspan=1>98.1%</td><td rowspan=1 colspan=1>90.1 - 100%</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>63.1 - 100%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.4 - 100%</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>29.2 - 100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.1 - 100%</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>39.8 - 100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.9 - 100%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>59.0 - 100%</td></tr><tr><td rowspan=1 colspan=1>EC</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>33/341</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>84.7 - 99.9%</td></tr></table>

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.4 - 100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>66/671</td><td rowspan=1 colspan=1>98.5%</td><td rowspan=1 colspan=1>92.0 - 100%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.1 - 100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.4 - 100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>65/65</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.5 - 100%</td></tr></table>

1 Out of 67 valid sample results, one (1) false positive Flu B result was obtained, almost certainly due to switching of the Flu A and Flu B capillaries in the run setup.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

The prospective clinical trial of the JBAIDS Influenza A & B Detection Kit tested nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS) specimens obtained from individuals with influenza-like illness (ILI). Specimens were collected from volunteers at five sites across the United States from December 2010 through April 2011. A total of 804 patient specimens were analyzed. The number and percentage of positive cases as determined with the JBAIDS Influenza A & B Detection Kit by testing site and sample type or by age group and sample type are presented in the following tables. Seasonal Influenza A/H1 was not circulating while this study was conducted.

Expected Value by Site and Sample Type for the JBAIDS Influenza A & B Detection Kit   

<table><tr><td rowspan=2 colspan=1>PositiveReeults byInfluenzaAssay</td><td rowspan=2 colspan=1>SampleType</td><td rowspan=1 colspan=2>Overall (NPS:n=314; NPW:n=490)</td><td rowspan=1 colspan=2>Site 1 (NPS:n=51, NPW:n=325)</td><td rowspan=1 colspan=2>Site 2 (n=207)</td><td rowspan=1 colspan=2>Site 3 (n=56)</td><td rowspan=1 colspan=2>Site 4 (n=119)</td><td rowspan=1 colspan=2>Site 5 (n=46)</td></tr><tr><td rowspan=1 colspan=1>Nqunr</td><td rowspan=1 colspan=1>Recante</td><td rowspan=1 colspan=1>Nqunr</td><td rowspan=1 colspan=1>RepecccAetu</td><td rowspan=1 colspan=1>Nqunb</td><td rowspan=1 colspan=1>Rpeccetu</td><td rowspan=1 colspan=1>Nqunb</td><td rowspan=1 colspan=1>Rececaatu</td><td rowspan=1 colspan=1>Nqunr</td><td rowspan=1 colspan=1>Recccantu</td><td rowspan=1 colspan=1>Neueb</td><td rowspan=1 colspan=1>Recaatee</td></tr><tr><td rowspan=2 colspan=1>Flu A</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>20.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>25.6%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>16.1%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>NPW</td><td rowspan=1 colspan=1>107</td><td rowspan=1 colspan=1>21.8%</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>15.4%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>35.3%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>32.6%</td></tr><tr><td rowspan=1 colspan=1>TOTAL</td><td rowspan=1 colspan=1>AL</td><td rowspan=1 colspan=1>170</td><td rowspan=1 colspan=1>21.1%</td><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>17.4%</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>25.6%</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>16.1%</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>35.3%</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>32.6%</td></tr><tr><td rowspan=2 colspan=1>Flu B</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>NPW</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>14.7%</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>14.5%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>18.5%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.5%</td></tr><tr><td rowspan=1 colspan=2>TOTAL</td><td rowspan=1 colspan=1>96</td><td rowspan=1 colspan=1>11.9%</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>14.5%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>7.1%</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>18.5%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>6.5%</td></tr></table>

Expected Values by Age Group and Sample Type for the JBAIDS Influenza A & B Detection Kit   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Total (Expected Value)</td><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>≥50 years</td></tr><tr><td rowspan=2 colspan=1>Influenza A</td><td rowspan=1 colspan=1>NPS (n=314)</td><td rowspan=1 colspan=1>63 (20.1%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>13</td></tr><tr><td rowspan=1 colspan=1>NPW (n=490)</td><td rowspan=1 colspan=1>107 (21.8%)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>NPS (n=314)</td><td rowspan=1 colspan=1>24 (7.6%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>NPW (n=490)</td><td rowspan=1 colspan=1>72 (14.7%)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>3</td></tr></table>

# N. Instrument Name:

Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument.

# O. System Descriptions:

1. Modes of Operation:

The JBAIDS Instrument is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Three other nucleic acid amplification tests that use the JBAIDS Instrument have received 510(k) clearance: JBAIDS Anthrax Detection System (k051713), JBAIDS Plague Detection System (k072631), JBAIDS Tularemia Detection System (k072547), JBAIDS Influenza A/H5 (Asian Lineage) Detection Kit (k100287), and JBAIDS Q Fever Detection System (k103207). The JBAIDS Instrument is a ruggedized, portable real-time PCR instrument designed to withstand the conditions of transport and use likely to be encountered in a military field laboratory. The instrument is comprised of an air thermocycler that amplifies specific DNA sequences using PCR and a fluorimeter that measures fluorescence signals associated with production of PCR product (amplicon) during the course of the reaction. For thermocycling, samples contained in glass capillaries are placed in the sample chamber where they are heated and cooled by the JBAIDS instrument.

The JBAIDS Software is preloaded on a ruggedized laptop computer. The software controls the instrument’s thermal cycling functions, acquires the fluorescence data from the instrument, and displays the fluorescence data for the user during the run. When the run is finished, the software’s Detector module analyzes the data and displays test results. The instrument and software have the ability to perform either diagnostic (IVD) or surveillance (environmental) testing.

The IVD software includes the following diagnostic components:

A traceable database that limits the data that the user can change in a test run and that requires the user to enter change notes for any changes that are allowed. • A Diagnostic Wizard that guides the operator through the process of setting up the JBAIDS run in compliance with the package insert. IVD testing is always performed using the Diagnostic Wizard and the traceable database. The wizard initially presents to the operator a predefined selection of FDA cleared JBAIDS IVD assays. JBAIDS assays under evaluation for FDA clearance are also available in limited release software. Once the user has selected the IVD assays to test, the Wizard guides the user through the process of setting up and loading each PCR reaction into the instrument. The wizard requires each run to contain a Positive Control and a Negative Control for each target included in the run as well as the inclusion of additional controls (e.g., inhibition controls, sample controls), when they are required. • A Detector module that automatically analyzes the fluorescence amplification curves and displays the final test results to the user. The Detector module ensures that the correct rules are applied both to the interpretation of each amplification

curve and to the final interpretation (e.g., metacall), based on the results of the samples and the relevant controls.

A report certification that reflects the current regulatory status of the test being performed. When an assay is being evaluated for FDA clearance, the JBAIDS test report includes the statement ‘For Investigational Use Only. The performance characteristics of this product have not been established.’ Once the assay has received FDA clearance, the software is modified and the report certification statement becomes ‘For In Vitro Diagnostic Use Only’.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The following controls are included in the JBAIDS Influenza A & B Detection Kit:

# Negative Control

The NC is used to detect contamination from target-specific amplified product (amplicon), synthetic RNA (as found in the PC vials), or virus. The NC vial is reconstituted using the purified negative extraction control (NEC) (molecular biology grade water). The NEC should be processed along with each batch of specimens purified. The NC detects possible contamination during sample purification and during reconstitution of freeze- dried reagents. Each JBAIDS run requires one NC for the Flu A assay and one NC for the Flu B assay. For each target assay, the NC must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run. If the NC fails, the JBAIDS run must be repeated using the same purified samples. If the NC fails again, all associated samples are considered to have invalid results and should be re-purified starting from another aliquot of the original specimens.

# Positive Control

The PC serves as an amplification and detection control. Each JBAIDS run requires one PC for the Flu A assay and one PC for the Flu B assay. For each target assay, the PC must be positive. If the PC fails, the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the JBAIDS run must be repeated starting from the same purified specimens. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument.

# Sample Control (SC)

The sample control assay (Flu SC) detects the human RNase P gene. This assay is designed to guard against false negative results caused by an improperly collected specimen, ineffective purification of nucleic acids and or inhibition of the PCR reaction. A properly collected NPS or NPW specimen contains human cells from which the RNase P target is recovered during sample purification. Following purification, each sample is then tested with the Flu A and Flu B target assays, as well as the Flu SC. If purification and amplification were successful, then the Flu SC will give the expected positive test result. The JBAIDS software automatically assigns a result of sample control failure when (1) the Flu SC is unsuccessful and (2) the target assay is negative (or uncertain).

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision