# 510(K) SUMMARY

# Applicant

IQuum, Inc. 700 Nickerson Road Marlborough, MA 01752 Tel: 508-970-0099 Fax: 508-970-0119

# Contact

Lingjun Chen   
Title: Vice President, Operations & Business Development   
Tel: 508-970-0099 ext. 116   
Email: lingjun@iquum.com

# Summary Date

May 16, 2011

Trade Name LiatTM Influenza A/B Assay, LiatTM Analyzer   
Common Name Influenza A, B Panel   
Regulation Number 21 CFR 866.3980   
Classification Name Respiratory Viral Panel Multiplex Nucleic Acid Assay

Predicate Device: Cepheid Xpert $\textcircled{8}$ Flu Assay (K103766)

# Intended Use

The IQuum Liat™M Influenza A/B Assay performed on the Liat™M Analyzer is an automated multiplex real-time RT-PCR assay for the rapid in vitro qualitative detection and discrimination of influenza A virus and influenza B virus RNA in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. The test is intended for use as an aid in the differential diagnosis of influenza A and influenza B in humans and is not intended to detect influenza C.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease.

Performance characteristics for influenza A were established when influenza A/H1 and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

# Device Description

The Liat™M Influenza A/B Assay is a rapid, automated in vitro diagnostic test for the detection and differentiation of Influenza type A and type B viral RNA in nasopharyngeal swab (NPS) specimens in universal transport media (UTM) from patients signs and symptoms of respiratory infection. The assay targets a well-conserved region of the matrix gene of Influenza A viral RNA (Inf A target) and non-structural protein (NS) gene of Influenza B (Inf B target). An Internal Process Control (IPC) is also included. The IPC is present to control for adequate processing of the target viruses and to monitor the presence of inhibitors in the RT-PCR reactions.

The Liat Influenza A/B Assay is performed on the lab-in-a-tube technology platform. The system consists of a disposable Liat Influenza A/B Assay Tube and the LiatTM Analyzer. The Liat™M Tube uses a flexible tube as a sample vessel. It contains all required unit dose reagents prepacked in tube segments, separated by peelable seals, in the order of reagent use. Manipulating a Liat Tube, the Liat Analyzer performs all assay steps from raw sample and report assay result automatically. During the testing process, multiple sample processing actuators of the analyzer compress the Liat Tube to selectively release reagents from tube segments, move the sample from one segment to another, and control reaction volume, temperature and time to conduct sample preparation, nucleic acid extraction, target enrichment, inhibitor removal, nucleic acid elution and real-time RT-PCR. An embedded microprocessor controls and coordinates the actions of these sample processors to perform all required assay processes within the closed Liat Tube. Turnaround time from sample input to result is ${ \sim } 2 0$ minutes.

# Test Operation

A nasopharyngeal swab can be collected following the user institution's standard procedures. For nasopharyngeal swab samples suspended in UTM, the user transfers $1 0 0 \mu \mathrm { l }$ of the UTM sample into Liat Influenza A/B Assay tube. The user then scans the tube barcode to identify the test and scans sample barcode to code the sample ID using the Liat system. The Liat Tube is then inserted into the Liat Analyzer. The analyzer performs all test steps and outputs interpreted results in 20 minutes. A report of the interpreted results can be viewed in the View Results window, and printed directly through a USB connected printer.

No reagent preparation or addition steps are required, other than adding the sample to the Liat tube. Because all the reagents are contained within the Liat assay tube and no sample or reagent needs to be removed from the tube, cross-contamination between samples is minimized.

# Predicate Device Comparison

Predicate Device: Cepheid Xpert $\textsuperscript { \textregistered }$ Flu Assay (K103766)   

<table><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">Device:Liat Influenza A/B</td><td colspan="1" rowspan="1">Predicate:Cepheid Xpert Flu</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The IQuum LiatTM Influenza A/B Assayperformed on the LiatTM Analyzer is anautomated multiplex real-time RT-PCRassay for the rapid in vitro qualitativedetection and discrimination ofinfluenza A virus and influenza B virusRNA in nasopharyngeal swabspecimens from patients with signs andsymptoms of respiratory infection inconjunction with clinical andepidemiological risk factors. The test isintended for use as an aid in thedifferential diagnosis of influenza A andinfluenza B in humans and is notintended to detect influenza C.Negative results do not precludeinfluenza virus infection and should notbe used as the sole basis for treatment orother patient management decisions.Conversely, positive results do not rule-out bacterial infection or co-infectionwith other viruses. The agent detectedmay not be the definite cause of disease.Performance characteristics forinfluenza A were established wheninfluenza A/H1 and A/H3 were thepredominant influenza A viruses incirculation. When other influenza Aviruses are emerging, performancecharacteristics may vary.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentInfluenza viruses and sent to state orlocal health department for testing. Viralculture should not be attempted in these</td><td colspan="1" rowspan="1">The Cepheid® Xpert Flu Assay is anautomated, multiplex real-time RT-PCR assay intended for the in vitroqualitative detection anddifferentiation of influenza A,influenza B and 2009 H1N1 influenzaviral RNA. The Xpert Flu Assay usesnasal aspirates/washes andnasopharyngeal swab specimenscollected from patients with signs andsymptoms of respiratory infection inconjunction with clinical andepidemiological risk factors. TheXpert Flu Assay is intended as an aidin the diagnosis of influenza.Negative results do not precludeinfluenza virus infection and shouldnot be used as the sole basis fortreatment or other patient managementdecisions.Performance characteristics forinfluenza A were established duringthe 2009-2010 influenza season when2009 H1N1 influenza was thepredominant influenza A virus incirculation. When other influenza Aviruses are emerging, performancecharacteristics may vary.If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiological screeningcriteria recommended by public healthauthorities, specimens should becollected with appropriate infectioncontrol precautions for novel virulentinfluenza viruses and sent to state orlocal health department for testing.Viral culture should not be attemptedin these cases unless a BSL 3+ facilityis available to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">cases unless a BSL 3+ facility isavailable to receive and culturespecimens.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Item Name</td><td colspan="1" rowspan="1">Device:Liat Influenza A/B</td><td colspan="1" rowspan="1">Predicate:Cepheid Xpert Flu</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="1" rowspan="1">21 CFR 866.3980</td><td colspan="1" rowspan="1">21 CFR 866.3332</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">OCC, 001</td><td colspan="1" rowspan="1">OQW, OCC, 001</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">Influenza AInfluenza B</td><td colspan="1" rowspan="1">Influenza AInfluenza BInfluenza A subtype 2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">Sample Type</td><td colspan="1" rowspan="1">Nasopharyngeal Swab</td><td colspan="1" rowspan="1">Nasopharyngeal SwabNasal aspirates/washes</td></tr><tr><td colspan="1" rowspan="1">Internal Control</td><td colspan="1" rowspan="1">Yes</td><td colspan="1" rowspan="1">Yes</td></tr><tr><td colspan="1" rowspan="1">Influenza A ViralTarget</td><td colspan="1" rowspan="1">Matrix gene</td><td colspan="1" rowspan="1">Matrix gene</td></tr><tr><td colspan="1" rowspan="1">Influenza B ViralTarget</td><td colspan="1" rowspan="1">Non-structural protein (NSP) gene</td><td colspan="1" rowspan="1">Hemagglutinin gene</td></tr><tr><td colspan="1" rowspan="1">Assay Instrument</td><td colspan="1" rowspan="1">LiatT Analyzer</td><td colspan="1" rowspan="1">GeneXpert instruments</td></tr><tr><td colspan="1" rowspan="1">Self-containedSystem</td><td colspan="1" rowspan="1">Integrated PC, software, and touch-screen display</td><td colspan="1" rowspan="1">External PC computer and softwarefor running tests and viewing theresults</td></tr><tr><td colspan="1" rowspan="1">All AssayReagentsContained inDisposable</td><td colspan="1" rowspan="1">No manual reagent addition required</td><td colspan="1" rowspan="1">Requires manual dispensing ofBinding Reagent into the chamberwith the small opening of the XpertFlu Assay Cartridge</td></tr><tr><td colspan="1" rowspan="1">Automated Assay</td><td colspan="1" rowspan="1">Yes, sample preparation, amplification,detection and result interpretation</td><td colspan="1" rowspan="1">Yes, sample preparation,amplification, detection and resultinterpretation</td></tr><tr><td colspan="1" rowspan="1">Extraction Method</td><td colspan="1" rowspan="1">Sample preparation integrated in Liat Tubeand Liat Analyzer</td><td colspan="1" rowspan="1">Sample preparation integrated inGeneXpert Cartridge and GeneXpertInstrument</td></tr><tr><td colspan="1" rowspan="1">Assay Method</td><td colspan="1" rowspan="1">RT-PCR for detecting the presence /absence of viral RNA in clinicalspecimens</td><td colspan="1" rowspan="1">RT-PCR for detecting the presence /absence of viral RNA in clinicalspecimens</td></tr><tr><td colspan="1" rowspan="1">DetectionTechnique</td><td colspan="1" rowspan="1">Multiplex assay using different reporterdyes for each target</td><td colspan="1" rowspan="1">Multiplex assay using differentreporter dyes for each target</td></tr><tr><td colspan="1" rowspan="1">ResultInterpretation</td><td colspan="1" rowspan="1">Automated</td><td colspan="1" rowspan="1">Automated</td></tr><tr><td colspan="1" rowspan="1">Assay Result</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Laboratory Users</td><td colspan="1" rowspan="1">CLIA moderate complexity and highcomplexity laboratory</td><td colspan="1" rowspan="1">CLIA moderate complexity and highcomplexity laboratory</td></tr><tr><td colspan="1" rowspan="1">Time-to-result</td><td colspan="1" rowspan="1">~20 minutes</td><td colspan="1" rowspan="1">~75 minutes</td></tr></table>

# Substantial Equivalence

# Limit of Detection

The Limit of Detection (LOD) of the Liat Influenza A/B Assay was tested using 3 strains of Influenza A (A/Brisbane/10/2007, A/Brisbane/59/2007, and A/NY/01/2009) and 2 strains of Influenza B (B/Malaysia/2506/04 and B/Florida/04/06). The LOD was determined by limiting dilution studies using these titered viruses. The viruses were spiked into nasopharyngeal swab (NPS) sample matrix, and then tested using the Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay. The LOD was determined as the lowest log virus concentration that was detected $2 9 5 \%$ of the time (i.e. log concentration at which at least 19 out of 20 replicates tested positive). The LOD for 3 strains of Influenza A were $1 0 ^ { - 2 }$ to $1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , while those for the 2 strains of Influenza B were $1 0 ^ { - 3 }$ to $1 0 ^ { - 1 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

<table><tr><td rowspan=1 colspan=1>Virus Strain</td><td rowspan=1 colspan=1>LOD (TCID5o/mL)</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/2007</td><td rowspan=1 colspan=1>10-1</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/2007</td><td rowspan=1 colspan=1>10^2</td></tr><tr><td rowspan=1 colspan=1>A/NY/01/2009</td><td rowspan=1 colspan=1>10-1</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/06</td><td rowspan=1 colspan=1>10-1</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>10-3$</td></tr></table>

# Reactivity

Reactivity study evaluates the ability of the Liat Influenza A/B Assay to detect influenza strains representing temporal and geographical diversity. The Liat Influenza A/B Assay was evaluated for reactivity with 22 Influenza A strains and 10 Influenza B strains. Influenza A strains included 8 seasonal Influenza A/H1 strains, 8 seasonal Influenza A/H3 strains, 3 Influenza A 2009 H1N1 strains, 3 swine origin Influenza A strains. The Liat Influenza A/B Assay detected all strains tested.

<table><tr><td colspan="1" rowspan="1">Influenza Strain</td><td colspan="1" rowspan="1">Type / Subtype</td><td colspan="1" rowspan="1">TestConcentration</td><td colspan="1" rowspan="1">Inf AResult</td><td colspan="1" rowspan="1">Inf BResult</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/59/2007</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">8.0×10-3 TCID5/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×10²TCID5o/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Solomon Island/3/2006</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">5.0×10^2TCID5/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×10³3TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">ADenver/1/57</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">5.0×10²TCID5o/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AFM/1/47</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">1.0×10²2TCIDso/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">2.5×101TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AWeiss/43</td><td colspan="1" rowspan="1">Influenza A, seasonal H1</td><td colspan="1" rowspan="1">2.5×10³ TCIDso/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/2007</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">1.0×101TCID5o/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AAlice</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×101TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/MRC2</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">1.0×10²TCID5so/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">AHong Kong/8/68</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.0×101TCID5/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AVictoria/3/75</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.5×101TCIDso/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">AWisconsin/67/05</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×101TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">5.0×10^²2 TCIDso/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">Influenza A, seasonal H3</td><td colspan="1" rowspan="1">2.0×10²2 CEID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ANY/01/2009</td><td colspan="1" rowspan="1">Influenza A, 2009 H1N1</td><td colspan="1" rowspan="1">1.0×101TCIDs/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/NY/02/2009</td><td colspan="1" rowspan="1">Influenza A, 2009 H1N1</td><td colspan="1" rowspan="1">2.5×10^²2TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ANY/03/2009</td><td colspan="1" rowspan="1">Influenza A, 2009 H1N1</td><td colspan="1" rowspan="1">4.0×101TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">1.0×101TCIDs/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Swine/1976/31</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">2.0×101TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">A/Swine/lowa/15/30</td><td colspan="1" rowspan="1">Influenza A, H1N1 non 2009</td><td colspan="1" rowspan="1">2.0×10²2TCID5o/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/06</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">8.0×10²2TCIDs0/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.0×10³3TCIDso/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Florida/7/04</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×102TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×101CEID5o/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.0×10'TCID5d/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10²2TCIDso/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">5.0×10³TCIDso/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Mass/3/66</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×101TCIDso/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/HongKong/5/72</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">2.5×101TCID5/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1.0×10°TCIDso/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr></table>

# Cross Reactivity

Cross reactivity study evaluates the Liat Influenza A/B Assay's potential cross-reactivity with non-influenza respiratory pathogens and other microorganisms with which the majority of the population may have been infected. The Liat assay was evaluated against a panel of 31 human pathogens. Bacteria were tested at $1 0 ^ { 5 } – 1 0 ^ { 6 } \mathrm { C F U / \bar { m L } }$ . Viruses were tested at $1 0 ^ { 3 } – 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ The Liat Influenza $\mathbf { A } / \mathbf { B }$ Assay showed no cross reactivity for the tested organisms.

<table><tr><td rowspan=1 colspan=1>Viral/Bacterial pathogen</td><td rowspan=1 colspan=1>Test concentration</td><td rowspan=1 colspan=1>Inf A Result</td><td rowspan=1 colspan=1>Inf B Result</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>8.9×10^ TCIDso/mL</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7</td><td rowspan=1 colspan=1>4.5×104 TCIDso/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus 229E</td><td rowspan=1 colspan=1>1.4×10^³3 TCID5g/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus OC43</td><td rowspan=1 colspan=1>7.9×104 TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>1×10^ TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 1</td><td rowspan=1 colspan=1>2.8×10³ TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 2</td><td rowspan=1 colspan=1>1.4×10^ TCIDso/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 3</td><td rowspan=1 colspan=1>1.6×10 TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>7.9×104 TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>7×103 TCIDso/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>7.9×104 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus type B</td><td rowspan=1 colspan=1>1.4×104 TCID5o/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>1.6×10 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>4.5×104 TCID5/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Epstein Barr virus</td><td rowspan=1 colspan=1>1.9x10 copies/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1.8×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>8×104 TCID50/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>5.0×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>6.6×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>3×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lactobacillus sp.</td><td rowspan=1 colspan=1>1.6×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>7×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>5.8×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>3.2×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neisseria sp.</td><td rowspan=1 colspan=1>1.8×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.6×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>4.5×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>6×10^6 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1.9×10^ $CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>3.7×10 CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>4.3×10^ CFU/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Inhibition by other Microorganisms

Interfering microorganism study evaluates whether non-influenza respiratory pathogens and other microorganisms with which the majority of the population may have been infected can interfere in the detection of Influenza A or B by the Liat assay. The panel of 31 human pathogens tested in the cross-reactivity study was tested for potential interference. Bacteria were tested at $1 0 ^ { 5 } – 1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses were tested at $1 0 ^ { 3 } { - } \mathrm { i } 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in the presence of either A/Brisbane/59/2007 or B/Malaysia/2506/04 at $3 \mathbf { x }$ LOD concentration in negative NPS matrix. Results show that the presence of the tested microorganisms did not interfere with the detection of Inf A or Inf B.

<table><tr><td rowspan=2 colspan=1>Pathogen</td><td rowspan=2 colspan=1>PathogenConcentration</td><td rowspan=1 colspan=2>A/Brisbane/59/07</td><td rowspan=1 colspan=2>B/Malaysia/2506/04</td></tr><tr><td rowspan=1 colspan=1>Inf AResult</td><td rowspan=1 colspan=1>Inf BResult</td><td rowspan=1 colspan=1>Inf AResult</td><td rowspan=1 colspan=1>Inf BResult</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 1</td><td rowspan=1 colspan=1>8.9×10^³ TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Adenovirus Type 7</td><td rowspan=1 colspan=1>4.5×104 TCIDs/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus 229E</td><td rowspan=1 colspan=1>1.4×10³ TCID5o/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Coronavirus OC43</td><td rowspan=1 colspan=1>7.9×10 TCID5o/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>1×10^ TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 1</td><td rowspan=1 colspan=1>2.8×10³ TCID5o/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 2</td><td rowspan=1 colspan=1>1.4×10 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Parainfluenza Type 3</td><td rowspan=1 colspan=1>1.6×10 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>7.9×104 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>7×10^³3 TCID5o/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>7.9×104 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Respiratory syncytial virus type B</td><td rowspan=1 colspan=1>1.4×104 TCIDsg/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Type 1A</td><td rowspan=1 colspan=1>1.6×10 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>4.5×104 TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr virus</td><td rowspan=1 colspan=1>1.9×10^ copies/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>1.8×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Chlamydia pneumoniae</td><td rowspan=1 colspan=1>8×104 TCID5o/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium sp.</td><td rowspan=1 colspan=1>5.0×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>6.6×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>3×10^$CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus sp.</td><td rowspan=1 colspan=1>1.6×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>7×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>5.8×10 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>3.2×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Neisseria sp.</td><td rowspan=1 colspan=1>1.8×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>1.6×10 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>4.5×10 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>6×10^ $CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>1.9×10 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>3.7×10 CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>4.3×10^ CFU/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr></table>

# Interfering Substances

The Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay was evaluated with potentially interfering substances that may be encountered in respiratory specimens. Medically and/or physiologically relevant concentrations of potential interferents were tested with 2 influenza A strains and 2 influenza B strains at 3x LOD $( 1 0 ^ { - 1 } { - } 1 0 ^ { - 2 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ . Results showed that substances tested did not interfere in the detection of influenza A and B strains.

<table><tr><td rowspan=1 colspan=1>Potential Interferent</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>Concentration.</td></tr><tr><td rowspan=1 colspan=1>MucinBovine submaxillary gland, type I-S</td><td rowspan=1 colspan=1>Purified mucin protein</td><td rowspan=1 colspan=1>0.1 mg/ml and25 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal spray - Afrin</td><td rowspan=1 colspan=1>Oxymetazoline</td><td rowspan=1 colspan=1>5%(v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal corticosteroids - Veramyst</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>5%(v/v)</td></tr><tr><td rowspan=1 colspan=1>Nasal gel - Zicam</td><td rowspan=1 colspan=1>Galphimia glauca, Histaminumhydrochloricum, Luffa operculata, Sulphur</td><td rowspan=1 colspan=1>5% (v/v)</td></tr><tr><td rowspan=1 colspan=1>Throat lozenges, oral anesthetic andanalgesic - Cepacol</td><td rowspan=1 colspan=1>Benzocaine, Menthol</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antibiotic, nasal ointment - Bactroban</td><td rowspan=1 colspan=1>Mupirocin</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antiviral drug - Tamiflu</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>7.5 mg/ml</td></tr><tr><td rowspan=1 colspan=1>Antimicrobial, systemic</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>4 μg/ml</td></tr></table>

# Inter-lot Precision

Inter-lot precision assesses the Liat tube manufacturing process as a potential source of assay variability. Three lots of Liat Influenza Assay tubes were tested with 1 Influenza A strain at C95 (LOD, $\mathrm { \tt m } { = } 1 8 0 $ ), C100 (4xLOD, $\mathtt { n } { = } 6 0 $ ) and C0 (negative NPS matrix, $\mathrm { n } { = } 6 0 _ { . }$ . Results from all runs agreed with expected results. Imprecision in $\mathrm { C t }$ from inter-lot variability was $< 1 . 8 \%$ .

<table><tr><td rowspan=1 colspan=1>Inf A</td><td rowspan=1 colspan=1>n</td><td rowspan=1 colspan=1>%Agreement</td><td rowspan=1 colspan=1>Avg Ct</td><td rowspan=1 colspan=1>StdevInter-lot</td><td rowspan=1 colspan=1>%CVInter-lot</td></tr><tr><td rowspan=1 colspan=1>C100</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.6%</td></tr><tr><td rowspan=1 colspan=1>C95</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.8%</td></tr><tr><td rowspan=1 colspan=1>CO</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>42.0</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td></tr></table>

# Reproducibility

Reproducibility study assesses the total variability of the Liat Influenza A/B Assay across operators, study sites, testing days, Liat Analyzers, and Liat assay tube lots. The Liat assay was evaluated at 3 sites, including 2 CLIA waived sites. Two operators at each of the 3 sites tested an 8 member reproducibility panel in triplicate on 5 different days, for a total of 720 runs (8 panel members $\times 3$ replicates $\times 2$ operators $\times 5$ days $\times 3$ sites). Fifteen (15) Liat Analyzers and 3 Liat Influenza $\mathrm { A } / \mathrm { B }$ Assay tube lots were used. The reproducibility panel comprises a high negative, a low positive and a medium positive of each of Influenza A and B, and the assay positive and negative controls. Percent agreement with expected result, mean $\mathrm { C t } ,$ and $C t \% C V$ for each site are show in tables below. $\% \mathrm { C V }$ for Influenza A ranged between $1 . 3 \%$ and $3 . 8 \%$ and that for Influenza B ranged between $1 . 1 \%$ and $3 . 4 \%$ Total percent agreement was $2 9 9 . 9 \%$ .

# Influenza A Reproducibility

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agree-ment w/expctedresult</td><td rowspan=1 colspan=1>A 9</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment wlexpectedresult</td><td rowspan=1 colspan=1>A iq</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>A q 9</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment wlexpectedresult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.5</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NegativeControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.1</td><td rowspan=1 colspan=1>3.8%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.8</td><td rowspan=1 colspan=1>1.8%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A Med.Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.3</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B Med.ositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=1>720/720(100%)</td><td rowspan=1 colspan=1>99.5%-100.0%</td></tr></table>

# Influenza B Reproducibility

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>Aq</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expectedresult</td><td rowspan=1 colspan=1>A9</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ment w/expecteddresult</td><td rowspan=1 colspan=1>Aq</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>Agree-ent wlexpectedresult</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.6</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>NegativeCControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A Med.Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B HighNegative</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>89/90(98.9%)</td><td rowspan=1 colspan=1>94.0%-99.8%</td></tr><tr><td rowspan=1 colspan=1>Fiu B LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.3</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>31.0</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>30.9</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B Med.Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.4</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>29.5</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=3>239/240 (99.6%)</td><td rowspan=1 colspan=3>240/240 (100%)</td><td rowspan=1 colspan=1>719/720(99.9%)</td><td rowspan=1 colspan=1>99.2%-100.0%</td></tr></table>

# Performance using Fresh vs. Frozen Samples

The Liat Influenza A/B Assay was tested by comparing its performance using fresh and frozen specimens. One influenza A strain and one influenza B strain were individually spiked into NPS matrix at different viral loads, including levels near LOD and levels reflecting the clinical range. For each strain, 60 samples were tested immediately while another 60 samples were frozen at -80°C for 7 days, thawed and then tested. Fresh and frozen samples demonstrated 100% detection across all levels of viral load, demonstrating that the Liat Influenza A/B Assay had equivalent performance for fresh and frozen samples.

# Clinical Agreement

Three sites participated in the clinical study for the Liat Influenza A/B Assay. A total of 615 clinical specimens were tested including 435 prospectively collected samples and 180 retrospective samples. Prospective samples were collected from patients with signs and symptoms of influenza in the Eastern and Southwestern US from 12 February 2009 to 26 March 2009. Viral culture and IFA staining was used as the reference method for these prospectively collected samples. Discordance results were investigated using PCR and bi-directional sequencing.

Due to the low prevalence of influenza during the collection period as well as the subsequent emergence of the 2009 H1N1 virus, retrospective samples collected between 2008-2010, including 2009 H1N1 samples from the 2009-2010 flu season, were tested. Reference testing for these samples was performed by PCR and bi-directional sequencing based on published methods (Ghedin et al. 2005; Ghedin et al. 2009; World Health Organization Sequencing Primers and Protocols, 2009). Of the 180 samples, 1 sample was indeterminate by PCR/sequencing, and remained indeterminate upon retest; this sample was excluded from the analysis:

Clinical sample testing was performed at 3 sites in April-May 2011 using 30 Liat Analyzers in total. Tables below summarize the clinical performance of the Liat Influenza A/B Assay.

Liat Influenza A/B Assay Clinical Performance - Prospective Samples   

<table><tr><td rowspan=2 colspan=2>Influenza A</td><td rowspan=1 colspan=2>Viral Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>13a</td><td rowspan=1 colspan=1>47</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>388</td><td rowspan=1 colspan=1>388</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>401</td><td rowspan=1 colspan=1>435</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(89.8% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>(94.5% - 98.1%)</td></tr></table>

a Of 13 false positive samples, 8 were Influenza A positive by PCR/sequencing, 4 were negative by PCR/sequencing, and 1 was indeterminate by PCR/sequencing due to low sequence quality score.

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(88.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>94.1%</td><td rowspan=1 colspan=1>(91.3% - 96.0%)</td></tr></table>

<table><tr><td rowspan=2 colspan=2>Influenza B</td><td rowspan=1 colspan=2>Viral Culture</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>24b</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>381</td><td rowspan=1 colspan=1>381</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>435</td></tr></table>

al posiv p  w Infz  posiv y R/cgatiy PCR/sequencing, and 8 samples were indeterminate by PCR/sequencing due to low sequence

quality score.

# Liat Influenza A/B Assay Clinical Performance - Retrospective Samples

<table><tr><td rowspan=2 colspan=2>Influenza A</td><td rowspan=1 colspan=2>PCR/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>1Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>74c</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>77</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>179</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>PositiveAgreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(95.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>NegativeAgreement</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>(91.9% - 99.0%)</td></tr></table>

O 74 Influenza A positive retrospective samples, 44 were 2009 H11 positive, and 20 were AH3 positive by PCR/sequencing. The 10 remaining samples were verified to be Influenza A however, no further subtyping analysis was done.

<table><tr><td rowspan=2 colspan=2>Influenza B</td><td rowspan=1 colspan=2>PCR/Sequencing</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=2 colspan=1>Liat</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>171</td><td rowspan=1 colspan=1>171</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>172</td><td rowspan=1 colspan=1>179</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>%</td><td rowspan=1 colspan=1>95% Cl</td></tr><tr><td rowspan=1 colspan=1>PositiveAgreement</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>(64.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=1>NegativeAgreement</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>(96.8% - 99.9%)</td></tr></table>

# Conclusion

The results of the analytical and clinical performance studies submitted in this premarket notification are complete and demonstrate that the Liat Influenza $\mathrm { A } / \mathrm { B }$ assay on the Liat Analyzer are substantially equivalent to the predicate device.

IQuum Inc.   
c/o Lingjun Chen   
Vice President, Operations and Business Development 700 Nickerson Road   
Marlborough, MA 01752

Re: K111387 Trade/Device Name: Liat Influenza A/B Assay, Liat Analyzer Regulation Number: 21 CFR§ 866.3980 Regulation Name: Respiratory Viral Panel Multiplex Nucleic Acid Assay Regulatory Class: Class II Product Code: OCC, 001 Dated: May 16, 2011 Received: May 18, 2011

Dear Dr. Chen:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class I (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section 510(k) premarket

notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html. Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

![](images/0e818c6883ec049d30f8ea435f8f95a01856356a62dfac61206d266bf89c5a1c.jpg)

# Indications for Use

510(k) Number (if known): Ki11387

Device Name: Liat™M Influenza A/B Assay, Liat™M Analyzer

Indications for Use:

The IQuum Liat™M Influenza A/B Assay performed on the LiatTM Analyzer is an automated multiplex real-time RT-PCR assay for the rapid in vitro qualitative detection and discrimination of influenza A virus and influenza B virus RNA in nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. The test is intended for use as an aid in the differential diagnosis of influenza A and influenza B in humans and is not intended to detect influenza C.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions. Conversely, positive results do not rule-out bacterial infection or co-infection with other viruses. The agent detected may not be the definite cause of disease.

Performance characteristics for influenza A were established when influenza A/H1 and A/H3 were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

Prescription Use _X (Part 21 CFR 801 Subpart D)

Over-The-Counter Use (21 CFR 801 Subpart C)

# (PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE IF NEEDED)

Concurrence of CDRH, Office of In Vitro Diagnostics (OIVD)

Office of In Vitro Diagnostic Device Evaluation and Safety

510(k) K111387