# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k111778   
B. Purpose for Submission: New device

# C. Measurand:

Target RNA sequences for the respective regions of the Hemagglutinin (HA) gene for seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza Virus (swine-origin), and for the region of the Nucleocapsid Protein (NP) gene of the 2009 H1N1 Influenza virus, as well as some other Influenza A viruses of swine lineage.

# D. Type of Test:

A real-time reverse-transcriptase polymerase chain reaction (Real-time RT-PCR) test intended for the qualitative in vitro detection and differentiation of seasonal Influenza A/H1, seasonal Influenza $\mathrm { A } / \mathrm { H } 3$ , and 2009 H1N1 Influenza in nasopharyngeal swab (NPS) and nasopharyngeal wash (NPW) specimens using nucleic acid isolation, amplification, and detection.

E. Applicant: US Army Office of the Surgeon General, U.S. Army Medical Materiel Development Activity

JBAIDS Influenza A Subtyping Kit Common Name: JBAIDS Influenza A Subtyping rRT-PCR Kit Real-time PCR assay for differentiation of influenza A subtypes

# G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OQW</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3332Reagents for detectionof specific novelInfluenza A viruses</td><td rowspan=1 colspan=1>Microbiology (83)</td></tr><tr><td rowspan=1 colspan=1>OEP</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic AcidAssay</td><td rowspan=1 colspan=1>Microbiology (83)</td></tr><tr><td rowspan=1 colspan=1>00I</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 862.2570Instrumentation forclinical multiplex testsystems</td><td rowspan=1 colspan=1>Clinical Chemistry(75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The Joint Biological Agent Identification and Diagnostic System (JBAIDS) Influenza A Subtyping Kit is intended for the in vitro qualitative detection and differentiation of seasonal Influenza A/H1, seasonal Influenza A/H3, and 2009 H1N1 Influenza viral nucleic acids isolated and purified from nasopharyngeal swab (NPS) and nasopharyngeal wash (NPW) specimens from human patients with signs and symptoms of respiratory infection, in conjunction with clinical and epidemiological risk factors. The JBAIDS Influenza A Subtyping Kit contains reverse transcriptase real-time polymerase chain reaction (rRT-PCR) assays for use on the JBAIDS instruments. The Flu A H1, Flu A H3, and Flu A H1 2009 assays of the JBAIDS Influenza A Subtyping Kit target a region of the hemagglutinin (HA) gene of the respective Influenza A virus. The Flu A Sw assay of the JBAIDS Influenza A Subtyping Kit targets a region of the nucleocapsid protein (NP) gene of the 2009 H1N1 Influenza virus, as well as some other Influenza A viruses of swine lineage. This kit is not intended to detect Influenza B or Influenza C Viruses.

A negative result for all assays in the JBAIDS Influenza A subtyping kit is a presumptive negative result for Influenza A. These results should be confirmed using the JBAIDS Influenza A & B Detection Kit.

Test results are to be used in conjunction with other clinical and epidemiological information. Negative results do not preclude influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions.

Due to low seasonal prevalence, performance characteristics for detection of seasonal Influenza A/H1 were established primarily with retrospective and contrived clinical specimens.

All users, analysts, and any person reporting diagnostic results from use of this kit should be trained to perform and interpret the results from this procedure by JBAIDS instructors or designees prior to use. Use of this device is limited to designated Department of Defense (DoD) laboratories equipped with the JBAIDS instruments.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a biosafety laboratory (BSL) $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument

# I. Device Description:

The JBAIDS Influenza A Subtyping Kit is a real time RT-PCR assay based on the influenza A subtyping assays that are part of the “CDC Human Influenza Virus Realtime RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)” and the “CDC Influenza 2009 A (H1N1)pdm Real-Time RT-PCR Panel”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT 1-2-3 Platinum Path Purification Kit, or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in conjunction with the Roche MagNA Pure Compact system) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instrument.

Each JBAIDS Influenza A Subtyping Kit contains sufficient reagents for testing six specimens. There are six vacuum-sealed control pouches of freeze-dried PCR reagents (8 reagent vials each), six vacuum-sealed sample testing pouches of freeze-dried PCR reagents (4 reagent vials each), and six pouches containing one tube each of reconstitution buffer and reagent grade water. Once resuspended, each reagent vial provides enough material for one reaction.

Each control pouch contains the following eight vials:

Positive Control $( + )$ vials (Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw). Once reconstituted, each Positive Control vial contains all reagents necessary for RT-PCR of the appropriate target, including an assay-specific synthetic RNA target. The Positive Controls are resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ reagent grade water before testing in parallel with patient specimens. Amplification in the Positive Control gives assurance that kit reagents are functioning properly and that the assay setup has been performed correctly. A Positive Control must be included for each assay in the JBAIDS run.

Ingredients: Each Positive Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ target-specific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; $< 0 . 0 0 1 \%$ assay-specific synthetic target RNA; bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Negative Control (-) vials (Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw). Once reconstituted, each Negative Control vial contains all reagents required to perform a RT-PCR reaction except the target RNA, and should therefore give negative results. The Negative Controls are re-suspended with $1 0 \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ nuclease-free water that has been purified with a nucleic acid purification kit (NEC). The reconstituted negative control is then tested in parallel with patient specimens to provide assurance that the purification and setup procedures have been performed without contamination. A Negative Control must be included for each assay in the JBAIDS run.

Ingredients: Each Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ targetspecific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Each sample testing pouch contains the following four vials:

• Unknown vials (Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw). Once reconstituted, each Unknown vial contains all reagents necessary for RT-PCR of the appropriate target, except target template. The Unknown vials are resuspended with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ purified patient sample, and will therefore give negative results unless the appropriate template is found in the patient sample.

Ingredients: Each Negative Control vial contains $< 0 . 0 0 1 \%$ DNA polymerase complex; $< 0 . 0 0 1 \%$ Reverse Transcriptase; $< 0 . 0 0 1 \%$ targetspecific forward and reverse primer; $< 0 . 0 0 1 \%$ target-specific hydrolysis probes; $< 0 . 0 5 \%$ dATP, dCTP, dGTP, dTTP; bovine serum albumin; dithiothreitol; RNase inhibitor; and carbohydrate.

Each reconstitution buffer/reagent grade water pouch contains the following two vials:

2X Reconstitution Buffer (2X RB). The buffer solution (purple buffer) is matched to the Influenza assays and is used undiluted. The buffer enhances PCR kinetics and is used to re-suspend JBAIDS freeze-dried reagents. Once reconstituted, all freeze dried reagent vials contain all of the components required for PCR.

Contents: $6 0 0 \mu \mathrm { L }$

Reagent Grade Water. Reagent grade water is molecular biology grade water used to reconstitute the Positive Controls.

Contents: 850 μL

# Materials Provided

<table><tr><td colspan="3" rowspan="1">Kit ContentsAll vials in the assay pouches contain sufficient reagent for one capillary reaction.</td></tr><tr><td colspan="1" rowspan="1">Qty</td><td colspan="1" rowspan="1">Description</td><td colspan="1" rowspan="1">Contents</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Influenza A Subtyping KitControl Pouch</td><td colspan="1" rowspan="1">One Flu A H1 Positive Control (+)One Flu A H1 Negative Control (-)One Flu A H3 Positive Control (+)One Flu A H3 Negative Control (-)One Flu A H1 2009 Positive Control (+)One Flu A H1 2009 Negative Control (-)One Flu A Sw Positive Control (+)One Flu A Sw Negative Control (-)</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Influenza A Subtyping KitSample Pouch</td><td colspan="1" rowspan="1">One Flu A H1Unknown (U)One Flu A H3 Unknown (U)One Flu A H1 2009 Unknown (U)One Flu A Sw Unknown (U)</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Purple ReconstitutionBuffer Pouch(purple foil pouch)</td><td colspan="1" rowspan="1">One purple reconstitution buffer (600 µL)One reagent grade water (850 µL)</td></tr><tr><td colspan="3" rowspan="1">Each kit contains sufficient reagents for testing six specimens.</td></tr></table>

# Materials Required But Not Provided

<table><tr><td rowspan=1 colspan=2>Required Equipment</td></tr><tr><td rowspan=1 colspan=1>Minicentrifuge capable of 2000 x g, 52151(Labnet C-1200 or equivalent)</td><td rowspan=1 colspan=1>Capillary adaptor for minicentrifuge (RocheApplied Science 1750-1.51 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>Micropipette: 2 μL20 μLMicropipette: 20 μL200 μLMicropipette: 200 μL−1000 μL</td><td rowspan=1 colspan=1>Vortex-Genie® 3823(VWR 58810-163 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>JBAIDS Instrument</td><td rowspan=1 colspan=1>Extra sample carousel JRPD-SUB-00101</td></tr><tr><td rowspan=1 colspan=1>Roche MagNA Pure Compact Instrument2</td><td rowspan=1 colspan=1>PickPen 1-M Magnetic Tool3 (Bio-Nobile)</td></tr><tr><td rowspan=1 colspan=2>Note: See appropriate Sample Purification Kit for additional equipment and materials requiredfor sample purification.</td></tr><tr><td rowspan=1 colspan=2>Available from Idaho Technology2 Only required when performing automated sample purification with the Roche MagNA PureCompact Nucleic Acid Isolation Kit I3 Only required when performing manual purification using the IT 1-2-3 Platinum Path SamplePurification Kit</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Materials and Reagents Not Provided</td></tr><tr><td rowspan=1 colspan=1>LightCycler glass capillaries and caps(Roche Applied Science 1 909 339)1</td><td rowspan=1 colspan=1>Aerosol-resistant (filter), nuclease-free pipettetips, appropriate for micropipettes</td></tr><tr><td rowspan=1 colspan=1>Microcentrifuge tube rack LABS-SUP-00011</td><td rowspan=1 colspan=1>Powder-free latex or nitrile gloves or equivalent</td></tr><tr><td rowspan=1 colspan=1>PickPen Tips3 (Bio-Nobile)</td><td rowspan=2 colspan=1>Sodium hypochlorite solution (household bleach)</td></tr><tr><td rowspan=1 colspan=1>Molecular biology grade water (SigmaW4502 or equivalent)</td></tr><tr><td rowspan=1 colspan=1>DNAZapTM, (Ambion AM9890 orequivalent DNA degradation solution)</td><td rowspan=1 colspan=1>IT 1-2-3 Platinum Path Sample Purification Kit(ASAY-ASY-0120)1,3orRoche MagNA Pure Compact Nucleic AcidIsolation Kit I2</td></tr></table>

1 Available from Idaho Technology 2 Only required when performing automated sample purification with the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I. Only qualified lots of the MagNA Pure Compact Nucleic Acid Isolation Kit I can be used with the JBAIDS Influenza A & B Detection Kit. Qualified Roche MagNA Pure Compact Nucleic Acid Isolation Kit I lot numbers are posted and viewable on the secure JBAIDS website. 3 Only required when performing manual purification using the IT 1-2-3 Platinum Path Sample Purification Kit

# J. Substantial Equivalence Information:

1. Predicate device name(s):

CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)

CDC Influenza 2009 A (H1N1)pdm Real-Time RT-PCR Panel

2. Predicate k number(s): K080570 K101564

1. Comparison with predicate(s):

<table><tr><td rowspan=2 colspan=4>Similarities</td></tr><tr></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate 1</td><td rowspan=1 colspan=1>Predicate 2</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>JABIDS Influenza A SubtypingKit</td><td rowspan=1 colspan=1>CDC rRT-PCR Flu Panel(K080570)</td><td rowspan=1 colspan=1>CDC rRT-PCR 2009A (H1N1)pdm Flu Panel (K101564)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Qualitative in vitro detectionand differentiation of seasonalInfluenza A/H1, seasonalInfluenza A/H3 and2009 H1N1 Influenza viralnucleic acids fromnasopharyngealswab (NPS) andnasopharyngeal wash (NPW)specimens on the JBAIDSinstrument.</td><td rowspan=1 colspan=1>Qualitative in vitro detection ofinfluenza virus type A or B and fordetermination of the subtypeof seasonal human influenza A virus,as seasonal A/H1 or A/H3, ifpresent, from viral RNA innasopharyngeal and/or nasal swabspecimens, for presumptiveidentification of virus in patientswho may be infected with influenzaA subtype A/H5 (Asian lineage)from viral RNA in human respiratorysymptoms of respiratoryspecimens and viral culture on anABI 7500 Fast Dx Real-time PCRinstrument.</td><td rowspan=1 colspan=1>Qualitative in vitro detection ofinfluenza virus type A and2009(H1N1) influenza viral RNAfrom nasopharyngeal swabs(NPS), nasal swabs (NS), throatswabs (TS), nasal aspirates(NA), nasal washes (NW), dualnasopharyngeal / throat swabs(NPS/TS) and lower respiratorytract specimens (LRTS) fromhuman patients with signs andinfection and/or from viralculture, on the AppliedBiosystems (ABI) 7500 Fast DxReal-Time PCR instrument.</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Real-time PCR using hydrolysisprobes</td><td rowspan=1 colspan=1>Real-time PCR using hydrolysisprobes</td><td rowspan=1 colspan=1>Real-time PCR using hydrolysisprobes</td></tr><tr><td rowspan=1 colspan=1>Assay Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidExtraction</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>Yes</td></tr></table>

<table><tr><td colspan="4" rowspan="1">Differences</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate 1</td><td colspan="1" rowspan="1">Predicate 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">JABIDS Influenza A Subtyping Kit</td><td colspan="1" rowspan="1">CDC rRT-PCR Flu Panel(K080570)</td><td colspan="1" rowspan="1">CDC rRT-PCR 2009A (H1N1)pdm Flu Panel (K101564)</td></tr><tr><td colspan="1" rowspan="1">Viruses Detected</td><td colspan="1" rowspan="1">Influenza A/H1, Influenza A/H3and Influenza A/2009 H1N1</td><td colspan="1" rowspan="1">Influenza A, Influenza B,Influenza A/H1, Influenza A/H3and Influenza A/H5</td><td colspan="1" rowspan="1">Influenza A and Influenza A/2009 H1N1</td></tr><tr><td colspan="1" rowspan="1">Specimen Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabs,nasopharyngeal washes</td><td colspan="1" rowspan="1">Upper respiratory tract specimens(including NPS, NS, TS, NA, NW,NPS/TS) and lower respiratory tractspecimens (including BAL, BW, TA,sputum, and lung tissue) and virusculture</td><td colspan="1" rowspan="1">Nasopharyngeal swabs, nasalswabs, nasal aspirates, nasalwashes, dual nasopharyngeal /throat swabs, broncheoalveolarlavage, tracheal aspirate,bronchial wash and viral culture</td></tr><tr><td colspan="1" rowspan="1">RequiredInstrumentation</td><td colspan="1" rowspan="1">JBAIDS instrument</td><td colspan="1" rowspan="1">Applied Biosystems 7500 FastDx Real-time PCR instrumentwith SDS software v 1.4</td><td colspan="1" rowspan="1">Applied Biosystems 7500 FastDx Real-time PCR instrument</td></tr><tr><td colspan="1" rowspan="1">Interpretation ofTest Results</td><td colspan="1" rowspan="1">Automated analysis of testresults and controls</td><td colspan="1" rowspan="1">User required to interpret testand control results</td><td colspan="1" rowspan="1">User required to interpret testand control results</td></tr><tr><td colspan="1" rowspan="1">Enzyme MasterMix</td><td colspan="1" rowspan="1">Assays come in freeze-driedsingle use vials that include allcomponents of master mix</td><td colspan="1" rowspan="1">Invitrogen SuperScriptTM IIIPlatinum One-StepQuantitative RT-PCR Kits</td><td colspan="1" rowspan="1">Invitrogen SuperScriptTM IIIPlatinum® One-StepQuantitative RT-PCR Kits</td></tr><tr><td colspan="1" rowspan="1">Reagent Storage</td><td colspan="1" rowspan="1">Reagents are stored at roomtemperature</td><td colspan="1" rowspan="1">Reagents are stored at ≤ -20°C</td><td colspan="1" rowspan="1">Reagents are stored at ≤ -20°C</td></tr><tr><td colspan="1" rowspan="1">ExtractionMethods</td><td colspan="1" rowspan="1">IT 1-2-3TM Platinum PathSample Purification KitRoche MagNA Pure CompactNucleic Acid Isolation Kit I</td><td colspan="1" rowspan="1">Qiagen QIAamp® Viral RNAMini KitQiagen RNeasy® Mini KitRoche MagNA Pure TNA KitRoche MagNA Pure LC RNAIsolation Kit II</td><td colspan="1" rowspan="1">Qiagen QIAamp® Viral RNAMini KitRoche MagNA Pure CompactNucleic Acid Isolation Kit I•Roche MagNA Pure TNA Kit•Roche MagNA Pure LC RNAisolation Kit IIQiagen QIAcube with QIAamp viralRNA mini kitbioMerieux NucliSENSeasyMAG</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens, FDA Guidance Document (DRAFT: December 8, 2005) Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses, FDA Guidance Document (DRAFT: February 15, 2008) • Molecular Diagnostic Methods for Infectious Diseases, CLSI Approved Guideline, MM3-P2 (February 2006) Interference Testing in Clinical Chemistry, CLSI Approved Guideline EP7-A2 (November 2005) User Verification of Performance for Precision and Trueness, CLSI Approved Guideline EP15-A2 (April 2006, second printing) Collection, Transport, Preparation, and Storage of Specimens for Molecular Methods; Approved Guideline, CLSI Approved Guideline MM13-A (January 2006) User Protocol for Evaluation of Qualitative Test Performance; Approved GuidelineSecond Edition, CLSI Approved Guideline EP12-A2 (January 2008) Protocols for Determination of Limits of Detection and Limits of Quantization, CLSI Approved Guidance EP17-A (2004) Evaluation of Precision Performance of Quantitative Measurements Methods; Approved Guidance-Second Edition, CLSI Approved Guidance EP5-A2 (August 2004)

In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path, Guidance for Industry and FDA Staff (May 1, 2007). Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses, Guidance for Industry and FDA Staff (March 22, 2006).

# L. Test Principle:

The JBAIDS Influenza A Subtyping Kit is a real time RT-PCR assay based on the influenza A subtyping assays that are part of the “CDC Human Influenza Virus Realtime RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel)” and the “CDC Influenza 2009 A (H1N1)pdm Real-Time RT-PCR Panel”. These assays have been re-optimized to work with Idaho Technology’s proprietary freeze-dried PCR reagent formulation. Purified patient samples (using either the IT 1-2-3 Platinum Path Purification Kit or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in conjunction with the Roche MagNA Pure Compact system) are used to reconstitute the freeze-dried reagents which are then tested using the JBAIDS instrument.

Real-time RT-PCR involves reverse transcription of specific RNA sequences into complementary DNA sequences, followed by logarithmic amplification and simultaneous detection of those DNA sequences. The JBAIDS Influenza A Subtyping assays use hydrolysis probes to detect amplification of the reverse transcribed RNA sequence of interest. Each hydrolysis probe is labeled on $5 ^ { \circ }$ end with a fluorescent reporter moiety (6-FAM) and elsewhere with a quencher (TAMRA), which prevents the probe from emitting fluorescent signal. During PCR, the probe binds to a target sequence in the PCR product. When the Taq polymerase replicates a template to which a hydrolysis probe is bound, the exonuclease activity of the polymerase cleaves the probe, separating the fluorophore from the quencher, and fluorescent signal is generated. This fluorescence is measured and displayed by the JBAIDS instrument during the PCR reaction. The fluorescent signal increases as additional template is amplified and more probes are hydrolyzed.

The JBAIDS Influenza A Subtyping Kit contains four assays: seasonal Influenza A/H1 (Flu A H1); seasonal Influenza A/H3 (Flu A H3); Influenza A, swine lineage (Flu A Sw); and Influenza A/2009 H1 (Flu A H1 2009). The Flu A H1, Flu A H3, and Flu A H1 2009 assays amplify a region of the hemagglutinin gene of the respective influenza A virus. The Flu A Sw assay amplifies a region of the nucleocapsid protein gene of 2009 H1N1 Influenza viruses, as well as some other Influenza A viruses of swine lineage.

The JBAIDS Influenza A Subtyping Kit is based on four major processes:

• Sample collection and purification   
• Reconstitution of freeze-dried reagents   
• Amplification and detection of target nucleic acid on the JBAIDS instrument Automated interpretation of the amplification curves using the JBAIDS software.

The JBAIDS Influenza A Subtyping kit can be used to test NPS and NPW specimens. Samples must be purified prior to testing with the JBAIDS Influenza A Subtyping Kit. The purpose of sample purification is to release nucleic acid contained in the patient sample and to remove extraneous materials (e.g., proteins and chemicals) that can interfere with the PCR reaction. The JBAIDS Influenza A Subtyping kit has been validated for use with either the IT 1-2-3 Platinum Path Sample Purification Kit or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact system. The Platinum Path kit is a manual purification, while the Roche MagNA Pure system is automated. Both methods have four basic steps: lysis to release the nucleic acids, binding of the nucleic acids to a magnetic substrate, washing away extraneous materials and elution of the purified nucleic acid. A brief outline of the two purification kit methods is provided below:

Method Summary for Purification Kits   

<table><tr><td rowspan=1 colspan=1>Step</td><td rowspan=1 colspan=1>IT 1-2-3 Platinum Path SamplePurification Kit</td><td rowspan=1 colspan=1>Roche MagNA Pure Compact NucleicAcid Isolation Kit I*</td></tr><tr><td rowspan=1 colspan=1>Cell Lysis to releasenucleic acids</td><td rowspan=1 colspan=1>Bead-beating (vortexing specimen in thepresence of silica beads and binding buffer)is used to physically disrupt the cells.</td><td rowspan=1 colspan=1>The samples are lysed by incubation withProteinase K and a special lysis buffercontaining a chaotropic salt.</td></tr><tr><td rowspan=1 colspan=1>Nucleic AcidBinding</td><td rowspan=1 colspan=1>The bead-beaten sample is transferred to awell of a strip tube containing magneticbeads. The nucleic acid binds to the magneticbeads.</td><td rowspan=1 colspan=1>Nucleic acids are immobilized on magneticglass particles.</td></tr><tr><td rowspan=1 colspan=1>Washing to removeextraneous materials</td><td rowspan=1 colspan=1>Using a pick-pen with a retractable magnet,the magnetic beads are collected from thefirst well of the strip tube and transferred to anew well containing a wash buffer. Thebuffer removes extraneous materials whilethe nucleic acid remains bound to themagnetic beads. The beads are washed threetimes (three wells of the strip tube).</td><td rowspan=1 colspan=1>Unbound substances are removed by severalwashing steps.</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Elution of nucleic acidsUsing the pick-pen, the magnetic beads aretransferred to the last well of the strip tubecontaining elution buffer. The nucleic acidsare released from the magnetic beads. Thebeads are then removed from the sampleusing the pick pen.</td><td rowspan=1 colspan=1>Purified nucleic acids are eluted from themagnetic glass particles.</td></tr></table>

\*All steps are automated within the Roche MagNA Pure Compact instrument.

Negative Extraction Controls (NECs) are processed along with each batch of purified specimens. When processing NECs, the user utilizes the Sample Purification Kit to purify a sample of nuclease-free water. The purified NEC sample is then used to reconstitute the freeze-dried Negative Control vials. The use of an NEC allows for detection of contamination during the entire testing process (purification to reagent setup).

Purified samples are tested using the freeze-dried reagents in the JBAIDS Influenza A Subtyping kit. Reagent vials that are intended for testing patient samples are labeled as Unknowns. Each JBAIDS run requires a Positive Control and a Negative Control for each target assay. The JBAIDS runs should be started within about 30 minutes after

reconstitution of the freeze-dried reagents. If necessary, the reconstituted reagents can be stored for up to four hours on ice or in the refrigerator $( 2 \mathrm { - } 8 \ ^ { \circ } \mathrm { C } )$ .

Reagents are set up in the following order:

1. Unknowns – Reconstitute the reagent pellet in the Unknown vials by adding $1 0 ~ \mu \mathrm { L }$ of the provided reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ of the purified sample.   
2. Positive Controls – After the Unknowns are set up for each sample, set up the Positive Controls by reconstituting the pellet with $1 0 ~ \mu \mathrm { L }$ of the provided reconstitution buffer and $1 0 \mu \mathrm { L }$ of the provided reagent-grade water.   
3. Negative Controls – Set up the Negative Controls last by reconstituting the pellet with $1 0 ~ \mu \mathrm { L }$ reconstitution buffer and $1 0 ~ \mu \mathrm { L }$ of the purified NEC.   
4. For each reagent vial, transfer $1 9 \mu \mathrm { L }$ of the reconstituted reagent to a capillary following the same order used for reconstitution. The cap is placed on the capillary tube before proceeding to the next capillary.   
5. Centrifuge the capillary tubes to ensure that the PCR reagent is in the tip of the capillary.

Each JBAIDS run can accommodate up to 6 samples with accompanying controls when all subtyping assays are used to test each sample. The Diagnostic Wizard guides the operator through the process of setting up the run in compliance with the JBAIDS Influenza A Subtyping Kit package insert. After selecting the appropriate assays and entering sample information, the operator transfers prepared capillaries to the instrument carousel, guided by the loading pattern displayed in the Diagnostic Wizard. With the capillaries loaded into the instrument, the operator signals the instrument to start the PCR run. The instrument automatically controls the temperature heating cycle in the carousel. At intervals during temperature cycling, fluorescence emission is monitored as the carousel rotates to position each sample above the fluorimeter. Temperature and fluorescence data are displayed in real-time. After starting the run, no input is required and the operator need not supervise the run. At the conclusion of the run, the operator may proceed directly to data analysis and reporting.

Each JBAIDS test is analyzed and assigned a final result by the Detector module of the JBAIDS Software. Possible final results are positive, negative, uncertain, or invalid. To assign a final test result, Detector first analyzes the data from each capillary independently. Then, the software assigns a final result, or a combined call, based on the results of the sample and all of its controls.

Each stage of the analysis is described below:

# 1. Independent Capillary Call

Detector does not rely on any single aspect of the amplification curve, but rather integrates a number of factors, such as curve shape and signal-to-noise ratio into a combined score. Curves scoring above a threshold are called positive, curves scoring below a threshold are called negative, and curves scoring in a small area in between are called uncertain. The scoring system has been carefully tuned to match human expert calls on real amplification curve data.

Detector calls are characterized as follows:

Negative – A curve is called negative if its shape can be closely approximated by a line (or smooth curve), if it has low signal to noise, and if it shows little or no increase in fluorescence during the PCR.

Positive – A curve is called positive if the fluorescence shows exponential growth out of the background, if it has high signal to noise, and if the slope of the exponential region of the amplification is consistent with PCR amplification.

Uncertain – For a small number of curves (about $0 . 2 \%$ ) no clear call is possible and the reaction is called uncertain.

# 2. Test Result or Combined Call

The final test result is a combined call (or meta-call) based on the results of the test sample and its associated controls.

![](images/dd08a26c46300e99e6b2cc44394a5f48999b952d1bb7cb9b4b77e0b869ee4666.jpg)

Flow chart outlining how final results are determined

# 3. Follow-up Testing for Invalid or Uncertain Results

Failure of Positive or Negative Control results in invalid results for all associated samples. Samples with invalid or uncertain results require follow-up testing in order to achieve a valid (positive or negative) result as described below:

Invalid – If either the Positive Control or Negative Control fails to give the expected results, all Unknown samples for that specific assay are called invalid and must be retested using the same purified sample. This testing can be done using the Custom Testing Method.

Uncertain – An uncertain result on a single sample requires retesting with the same purified sample. This testing can be done using the Custom Testing Method.

# 4. JBAIDS Report

A JBAIDS report is generated automatically for each test run. With the JBAIDS Report, the user integrates the instrument-provided results for the prior performed influenza A test and each of the influenza A subtyping tests to make a final result interpretation.

# 5. Integrated Result Interpretation

The JBAIDS software automatically interprets the results for each of the target assays. However, the user is required to make the final test interpretation based upon the software test results for each of the four target assays.

Result Interpretation for Use of the JBAIDS Influenza A Subtyping Kit   

<table><tr><td colspan="3" rowspan="1">Subtyping Assay Results</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="2">Interpretation</td><td colspan="1" rowspan="2">Action</td></tr><tr><td colspan="1" rowspan="1">Flu A H1</td><td colspan="1" rowspan="1">Flu A H3</td><td colspan="1" rowspan="1">Flu A Sw</td><td colspan="2" rowspan="1">Flu A H1 2009</td></tr><tr><td colspan="1" rowspan="3">Neg.</td><td colspan="1" rowspan="3">Neg.</td><td colspan="1" rowspan="2">b</td><td colspan="2" rowspan="2"></td><td colspan="1" rowspan="3">Influenza A/H1,Influenza A/H3, and2009 H1N1 Influenzaviral RNA not detectedIf an Influenza Atest has not yet beenperformed, this is apresumptivenegative result forInfluenza A.If the sample waspositive forInfluenza A usingthe JBAIDSInfluenza A &amp; BDetection Kit, theseresults may indicatethe presence of anovel strain forInfluenza A</td><td colspan="1" rowspan="3">If the sample has notpreviously been tested forInfluenza A, these resultsshould be confirmed by theJBAIDS Influenza A &amp; BDetection Kit.If the sample was positivefor Influenza A using theJBAIDS Influenza A &amp; BDetection Kit, see"Required Actions for NoSubtype Detected Results"section.</td></tr><tr><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="2" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H1 viral RNAdetected</td><td colspan="1" rowspan="1">No further testing</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="2" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H3 viral RNAdetected</td><td colspan="1" rowspan="1">No further testing</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="2" rowspan="1">Pos.</td><td colspan="1" rowspan="1">2009 H1N1 Influenzaviral RNA detected</td><td colspan="1" rowspan="1">No further testing</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="2" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Inconclusive fordetection of 2009 H1N1Influenza viral RNA</td><td colspan="1" rowspan="1">Retest or re-purify andretest with the Flu A Swand Flu A H1 2009 assays</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="2" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Inconclusive fordetection of 2009 H1N1Influenza viral RNA</td><td colspan="1" rowspan="1">using the Custom TestMethod. If the retest is alsoinconclusive, obtain a new</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">sample and retest.</td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H1 and SeasonalInfluenza A/H3 viralRNAs detected</td><td colspan="1" rowspan="4">Multiple infections arepossible, but rare.Repeat testing using theresidual purified nucleicAcid. If the retest confirmsthis result, collect and testa new specimen'. ContactIdaho Technology ifmultiple samples providethis result.</td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H1 and 2009 H1N1Influenza viral RNAsdetected</td></tr><tr><td colspan="1" rowspan="1">Neg.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H3 and 2009 H1N1Influenza viral RNAsdetected</td></tr><tr><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Pos.</td><td colspan="1" rowspan="1">Seasonal InfluenzaA/H1, SeasonalInfluenza A/H3, and2009 H1N1Influenzaviral RNAs detected</td></tr></table>

1 Recent patient exposure to the FluMis $^ \mathrm { \textregistered }$ intranasal vaccine may result in false positive results for multiple influenza A subtypes, and should be considered as a possible interferent when multiple influenza infections are detected.

# 6. Required Actions for No Subtype Detected Results

If the specimen tested positive for influenza A using the JBAIDS Influenza A & B Detection Kit, but none of the subtyping assays are positive, then the interpretation is ‘No Subtype Detected’. This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the presence of a novel and/or newly emerging influenza A strain. In both cases, the residual purified nucleic acid from the specimen in question, or the reextracted nucleic acid from the residual specimen in question if residual purified nucleic acid is not available, should be retested with the JBAIDS Influenza A & B Detection Kit (or other FDA cleared nucleic acid-based tests detecting Influenza A) and the JBAIDS Influenza A Subtyping Kit to confirm that the influenza A result is positive and that no influenza A subtype is detected. (Note: the Custom Test Method may be used to include the Influenza A test from the JBAIDS Influenza A & B Detection Kit along with the Influenza A subtyping tests in the same JBAIDS run). If the retest provides a different result, test the sample a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the JBAIDS Influenza A Subtyping reagents should be verified by testing with appropriate external control materials (known positive samples for seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza). A negative control (known negative sample or viral transport media) should also be run to test for PCR-product contamination. If the JBAIDS Influenza A Subtyping reagents accurately identify the external positive and negative controls, notify through the chain-of command of the appropriate DoD agencies, or contact the state public health authorities or CDC Influenza Division for confirmatory testing.

# 7. Custom Test Method

The Custom Test Method can be used to test individual assays from the JBAIDS Influenza A Subtyping Kit. Reasons to use the Custom Test Method include:

PC or NC failure invalidates the results for only one of the four assays requiring retesting of all samples with only that assay.   
• A sample is uncertain for one of the assays, requiring retesting with only that assay.   
• A sample is inconclusive for 2009 H1N1 Influenza, requiring retesting with only the Flu A Sw and Flu A H1 2009 assays.   
• A “No Subtype Detected” sample is retested using all four influenza A subtype assays and the Influenza A assay from the JBAIDS Influenza A&B Detection Kit. During an influenza outbreak, the Department of Defense may choose to recommend use of the custom test method for more efficient and rapid identification of a specific virus. In this case, the relevant subtyping assay(s) from the JBAIDS Influenza A Subtyping Kit may be used in conjunction with the influenza A assay from the JBAIDS Influenza A & B Detection Kit.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A multicenter reproducibility study was performed to determine the overall system reproducibility of the JBAIDS Influenza A Subtyping Kit on the JBAIDS instruments. This study was performed at Idaho Technology Inc. (ITI) and two external sites. The study was conducted with two specimen types: nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS), both collected in viral transport medium (VTM). Both specimen types were purified using two different Sample Purification Kits, the Roche MagNA Pure Nucleic Acid Isolation Kit I and the IT 1-2-3 Platinum Path Sample Purification Kit.

A total of six different panels of nine samples each were prepared for the reproducibility study.

Panel 1 contained nine NPS specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative seasonal Influenza A H1N1 virus strain (A/New Caledonia/20/1999) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). Therefore, Panel 1 consisted of three seasonal Influenza A H1N1 virus spiked medium positive NPS specimens (3X LoD), three seasonal Influenza A H1N1 virus spiked low positive NPS specimens (LoD), and three seasonal Influenza A H1N1 virus spiked high negative NPS specimens. Panel 2 contained nine NPW specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative seasonal Influenza A H1N1 virus strain (A/New Caledonia/20/1999) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). Therefore, Panel 2 consisted of three seasonal Influenza A H1N1 virus spiked medium positive

NPW specimens (3X LoD), three seasonal Influenza A H1N1 virus spiked low positive NPW specimens (LoD), and three seasonal Influenza A H1N1 virus spiked high negative NPW specimens.

Panel 3 contained nine simulated NPS (sNPS) specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative seasonal Influenza A H3N2 strain (A/New York/55/2004) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPS were generated from diluting HeLa cells into VTM. (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 3 consisted of three seasonal Influenza A H3N2 virus spiked medium positive sNPS specimens (3X LoD), three seasonal Influenza A H3N2 virus spiked low positive sNPS specimens (LoD), and three seasonal Influenza A H3N2 virus spiked high negative sNPS specimens.

Panel 4 contained nine simulated NPW (sNPW) specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative seasonal Influenza A H3N2 strain (A/New York/55/2004) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPW were generated from diluting HeLa cells into $50 \%$ VTM and $50 \%$ saline $( 0 . 9 \%$ NaCl). (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 4 consisted of three seasonal Influenza A H3N2 virus spiked medium positive sNPW specimens (3X LoD), three seasonal Influenza A H3N2 virus spiked low positive sNPW specimens (LoD), and three seasonal Influenza A H3N2 virus spiked high negative sNPW specimens.

Panel 5 contained nine simulated NPS (sNPS) specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative 2009 H1N1 Influenza strain (A/New York/18/2009) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPS were generated from diluting HeLa cells into VTM. (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 5 consisted of three 2009 H1N1 Influenza virus spiked medium positive sNPS specimens (3X LoD), three 2009 H1N1 Influenza virus spiked low positive sNPS specimens (LoD), and three 2009 H1N1 Influenza virus spiked high negative sNPS specimens.

Panel 6 contained nine simulated NPW (sNPW) specimens that were screened with the Flu A and Flu B assays of the JBAIDS Influenza A & B Detection Kit to ensure that no influenza nucleic acid was present, each spiked with a representative 2009 H1N1 Influenza strain (A/New York/18/2009) at one of the three different concentrations (LoD/20, LoD, and 3X LoD). sNPW were generated from diluting HeLa cells into $50 \%$ VTM and $50 \%$ saline $( 0 . 9 \%$ NaCl). (Note: refer to the “Validation of Simulated Sample Matrices” section for details). Therefore, Panel 6 consisted of three 2009 H1N1 Influenza virus spiked medium positive sNPW specimens (3X LoD), three 2009 H1N1 Influenza virus spiked low positive sNPW specimens (LoD), and three 2009 H1N1 Influenza virus spiked high negative sNPW specimens.

Note: Panels 3 and 4 specimens were also co-spiked with a representative Influenza B virus strain (B/Ohio/1/2005) for more efficient reproducibility testing of the JBAIDS

Influenza A Subtyping Kit. Refer to the “Co-spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus.

The NPS and sNPS reproducibility panel compositions are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Level of Seasonal Flu A H1N1 (A/NewCaledonia/20/1999) Virus in NPS Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1>Level of Seasonal Flu A H3N2 (A/NewYork/55/2004) Virus in sNPS Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD) - 3 specimens</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Level of 2009 H1N1 Influenza (A/NewYork/18/2009) Virus in sNPS Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>1500</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>4500</td></tr></table>

The NPW and sNPW reproducibility panel compositions are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Level of Seasonal Flu A H1N1 (A/NewCaledonia/20/1999) Virus in NPW Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>2.5</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD) - 3 specimens</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>150</td></tr><tr><td rowspan=1 colspan=1>Level of Seasonal Flu A H3N2 (A/NewYork/55/2004) Virus in sNPW Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20)  3 specimens</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Level of 2009 H1N1 Influenza (A/NewYork/18/2009) Virus in sNPW Specimen</td><td rowspan=1 colspan=1>Virus Concentration (EID50/mL)</td></tr><tr><td rowspan=1 colspan=1>High Negative (LoD/20) - 3 specimens</td><td rowspan=1 colspan=1>75</td></tr><tr><td rowspan=1 colspan=1>Low Positive (LoD)  3 specimens</td><td rowspan=1 colspan=1>1500</td></tr><tr><td rowspan=1 colspan=1>Medium Positive (3X LoD)  3 specimens</td><td rowspan=1 colspan=1>4500</td></tr></table>

Each panel of 9 samples was tested twice daily for five days at each testing site. Every testing day, two users each tested each of the nine samples. Prior to testing with the JBAIDS assays, one user used the Roche MagNA Pure Nucleic Acid Isolation Kit I and the other user used the IT 1-2-3 Platinum Path Sample Purification Kit to extract viral nucleic acid from each sample. For each panel, each sample was tested a total of 30 times (2 runs/day/site X 5 days X 3 Sites; 15 times after purification with the Platinum Path kit and 15 times after purification with the MagNA Pure kit), and each concentration level per sample type was tested a total of 90 times (3 samples spiked at each concentration level in each of the 6 panels).

Each JBAIDS run performed in this study included Positive Control (PC) and negative extraction control (NEC) reactions. (Note: to ensure that no contamination of virus or amplification product was contracted during sample preparation and reaction setup, a NEC, which was an aliquot of reagent grade water, was prepared and tested with each batch of purified samples.)

Of the 63 Flu A H1 PC reactions tested in this study, all were valid (63/63; $100 \%$ . The Flu A H1 PC reactions had a mean $\mathrm { C p }$ of $3 1 . 4 1 \pm 0 . 5 6$ ( $1 . 7 8 \%$ CV). Sixtyone of the 63 Flu A H1 NEC reactions (61/63; $9 6 . 8 \%$ ) tested in this study yielded the expected negative target results and were valid. Two failed NEC reactions were observed and the associated runs were invalid. One failed NEC was likely due to the control capillaries being incorrectly loaded in the JBAIDS carousel, as reported by the user. The purified samples were retested with new controls. The other failed NEC had a positive result for the Flu A H1 assay, possibly due to contamination. The work area was cleaned and fresh samples were re-extracted and tested. All controls were valid on the retest run.

Sixty-two out of 65 Flu A H3 PC and NEC reactions tested in this study were valid (62/65; $9 5 . 4 \% )$ . The mean $\mathrm { C p }$ of the Flu A H3 PC reactions was $3 0 . 8 6 \pm$ 0.20 $0 . 6 5 \%$ CV). Three invalid runs were successfully detected by the controls. These run failures appear to be due to the carousel being improperly seated by the user. This shifted the capillaries during the test, resulting in a loss of fluorescence detection in all samples. The JBAIDS software correctly assigned failure results to the controls in these runs. The data from these invalid runs were not included in this study. All controls were valid on the retest run.

All 63 Flu A Sw PC and NEC reactions tested in this study were valid. The mean Cp of the Flu A Sw PC reactions was $2 8 . 2 7 \pm 0 . 3 0$ ( $1 . 0 6 \%$ CV). All 63 Flu A H1 2009 PC and NEC reactions tested were valid. The mean $\mathrm { C p }$ of the Flu A H1 2009 PC reactions was $3 2 . 0 1 \pm 0 . 3 9$ ( $1 . 2 2 \%$ CV).

Although the JBAIDS Influenza A Subtyping Kit does not contain a sample control assay, the Flu SC assay from the JBAIDS Influenza A & B Detection Kit was tested with each sample. This assay targets the human RNaseP sequence and serves as an external control to detect inhibition, inadequate sample extraction, or poor specimen collection. It is recommended that any sample tested with the Influenza A Subtyping Kit first be tested with the Influenza A & B Detection kit; therefore the Flu SC assay would have been previously tested with each sample. Of the 1,628 Flu SC reactions tested in this study, 1,620 Flu SC reactions gave positive results, while 8 negative results were observed (1620/1628; $9 9 . 5 \%$ ). All Flu SC negative results were from NPW or sNPW samples, which may contain fewer human cells per unit volume than swab samples. Because negative Flu SC results only cause a control failure for negative influenza assays, six of the eight negative Flu SC results did not affect the positive metacall; the corresponding influenza assays had the expected positive metacall results. The other two negative Flu SC assays results were called SC Failures, and both occurred with the same sample. On the initial run the sample gave an SC Failure call. According to the protocol the purified sample was retested, both undiluted and diluted 1:10. The retest result for the undiluted sample was again an SC Failure. Therefore a new aliquot of the same sample was re-extracted and tested.

Results for the second retest were positive for the Flu A H3 and Flu SC assays.

For the Flu SC assay, NPS samples exhibited a mean $\mathrm { C p }$ of $3 0 . 1 5 \pm 2 . 3 6$ ( $7 . 8 3 \%$ CV) and NPW samples had a mean Flu SC Cp of $3 3 . 1 8 \pm 1 . 8 8$ $5 . 6 7 \%$ CV). The cell concentrations used to prepare the sNPS and sNPW samples were chosen to mimic Flu SC Cp values of fresh specimens. The sNPS samples had a mean $\mathrm { C p }$ of $3 2 . 0 6 \pm 2 . 0 0$ $( 6 . 2 4 \%$ CV) and sNPW samples exhibited a mean $\mathrm { C p }$ of $3 4 . 2 9 \pm 2 . 2 9$ ( $6 . 6 8 \%$ CV) for the Flu SC assay. These Cp values are consistent with previous testing of frozen simulated samples prepared at $1 0 ^ { 4 . 2 }$ and $1 0 ^ { 3 . 7 } \mathrm { H e L } \colon$ a cells/mL for sNPS and sNPW samples, respectively. The $6 \text{‰}$ CV in the Flu SC mean $\mathrm { C p }$ values measured for each sample type arises primarily from differences in Flu SC Cp values observed for the two sample purification kits. Typically MagNA Pure extracted samples exhibit $\mathrm { C p }$ values 4 cycles earlier than respective Platinum Path extracted samples.

Reproducibility Study Summary (Agreement with Expected Positive Results) for the JBAIDS Influenza A Subtyping Kit – Influenza A/H1 Assay   

<table><tr><td rowspan=3 colspan=1>SampleType</td><td rowspan=3 colspan=1>Test Level</td><td rowspan=1 colspan=4>IT 1-2-3 Platinum Path Sample PurificationKit# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=1 colspan=4>Roche MagNA Pure Compact Nucleic AcidIsolation Kit I# of Positive Samples/# of Total Samples(% Positive Detection)</td><td rowspan=3 colspan=1>AllPurificationKits, All Sites</td><td rowspan=3 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=4>Test Location</td><td rowspan=1 colspan=4>Test Location</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=5 colspan=1>N</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>88/90(98%)</td><td rowspan=1 colspan=1>92.2-99.7</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>8/15(53%)</td><td rowspan=1 colspan=1>36/45(80%)</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>11/15(73%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>38/45(84%)</td><td rowspan=1 colspan=1>74/90(82%)</td><td rowspan=1 colspan=1>72.7-89.4</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>178/180(99%)</td><td rowspan=1 colspan=1>96.0-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection allLevels</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>42/45(93%)</td><td rowspan=1 colspan=1>38/45(84%)</td><td rowspan=1 colspan=1>125/135(93%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>40/45(89%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>127/135(94%)</td><td rowspan=1 colspan=1>252/270(93%)</td><td rowspan=1 colspan=1>90.0-96.0</td></tr><tr><td rowspan=5 colspan=1>MdN</td><td rowspan=1 colspan=1>3X LoD</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>90/90(100%)</td><td rowspan=1 colspan=1>96.7-99.9</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>88/90(98%)</td><td rowspan=1 colspan=1>92.2-99.7</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>13/15(87%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>9/15(60%)</td><td rowspan=1 colspan=1>37/45(82%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>14/15(93%)</td><td rowspan=1 colspan=1>15/15(100%)</td><td rowspan=1 colspan=1>43/45(96%)</td><td rowspan=1 colspan=1>80/90(89%)</td><td rowspan=1 colspan=1>80.5-94.5</td></tr><tr><td rowspan=1 colspan=1>Detection ≥ LoD</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>29/30(97%)</td><td rowspan=1 colspan=1>30/30(100%)</td><td rowspan=1 colspan=1>89/90(99%)</td><td rowspan=1 colspan=1>178/180(99%)</td><td rowspan=1 colspan=1>96.0-99.9</td></tr><tr><td rowspan=1 colspan=1>Detection AllLevels</td><td rowspan=1 colspan=1>42/45(93%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>39/45(87%)</td><td rowspan=1 colspan=1>126/135(93%)</td><td rowspan=1 colspan=1>44/45(98%)</td><td rowspan=1 colspan=1>43/4596%)</td><td rowspan=1 colspan=1>45/45(100%)</td><td rowspan=1 colspan=1>132/135(98%)</td><td rowspan=1 colspan=1>258/270(96%)</td><td rowspan=1 colspan=1>92.4-97.7</td></tr></table>

Reproducibility Study Summary (Agreement with Expected Positive Results) for the JBAIDS Influenza A Subtyping Kit – Influenza A/H3 Assay   

<table><tr><td colspan="1" rowspan="1">SampleType</td><td colspan="1" rowspan="1">Test Level</td><td colspan="1" rowspan="1">IT 1-2-3 Platinum Path Sample PurificationKit# of Positive Samples/# of Total Samples(% Positive Detection)</td><td colspan="1" rowspan="1">Roche MagNA Pure Compact Nucleic AcidIsolation Kit I# of Positive Samples/# of Total Samples(% Positive Detection)</td><td colspan="1" rowspan="1">AllPurificationKits, All Sites</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="4" rowspan="1">Test Location</td><td colspan="4" rowspan="1">Test Location</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="5">N</td><td colspan="1" rowspan="1">3X LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">13/15(87%)</td><td colspan="1" rowspan="1">43/45(96%)</td><td colspan="1" rowspan="1">88/90(98%)</td><td colspan="1" rowspan="1">92.2-99.7</td></tr><tr><td colspan="1" rowspan="1">LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD/20</td><td colspan="1" rowspan="1">12/15(80%)</td><td colspan="1" rowspan="1">10/15(67%)</td><td colspan="1" rowspan="1">9/15(60%)</td><td colspan="1" rowspan="1">31/45(69%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">13/15(87%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">41/45(91%)</td><td colspan="1" rowspan="1">72/90(80%)</td><td colspan="1" rowspan="1">70.2-87.7</td></tr><tr><td colspan="1" rowspan="1">Detection ≥ LoD</td><td colspan="1" rowspan="1">30/3000%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">28/30(93%)</td><td colspan="1" rowspan="1">88/9098%)</td><td colspan="1" rowspan="1">178/180(99%)</td><td colspan="1" rowspan="1">96.0-99.9</td></tr><tr><td colspan="1" rowspan="1">Detection allLevels</td><td colspan="1" rowspan="1">42/45(93%)</td><td colspan="1" rowspan="1">40/45(89%)</td><td colspan="1" rowspan="1">39/45(87%)</td><td colspan="1" rowspan="1">121/135(90%)</td><td colspan="1" rowspan="1">44/45(98%)</td><td colspan="1" rowspan="1">43/45(96%)</td><td colspan="1" rowspan="1">42/45(93%)</td><td colspan="1" rowspan="1">129/135(96%)</td><td colspan="1" rowspan="1">250/270(93%)</td><td colspan="1" rowspan="1">88.8-95.4</td></tr><tr><td colspan="1" rowspan="5">MATE</td><td colspan="1" rowspan="1">3X LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">44/45(98%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">89/90(99%)</td><td colspan="1" rowspan="1">94.0-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">44/45(98%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">89/90(99%)</td><td colspan="1" rowspan="1">94.0-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD/20</td><td colspan="1" rowspan="1">11/15(73%)</td><td colspan="1" rowspan="1">10/15(67%)</td><td colspan="1" rowspan="1">9/15(60%)</td><td colspan="1" rowspan="1">30/45(67%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">14/15(93%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">44/45(98%)</td><td colspan="1" rowspan="1">74/90(82%)</td><td colspan="1" rowspan="1">72.7-89.5</td></tr><tr><td colspan="1" rowspan="1">Detection ≥ LoD</td><td colspan="1" rowspan="1">29/30(97%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">29/30(97%)</td><td colspan="1" rowspan="1">88/90(98%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(10%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">178/180(99%)</td><td colspan="1" rowspan="1">96.0-99.9</td></tr><tr><td colspan="1" rowspan="1">Detection AllLevels</td><td colspan="1" rowspan="1">40/45(89%)</td><td colspan="1" rowspan="1">40/45(89%)</td><td colspan="1" rowspan="1">38/45(84%)</td><td colspan="1" rowspan="1">118/135(87%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">44/45(98%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">134/135(99%)</td><td colspan="1" rowspan="1">252/270(93%)</td><td colspan="1" rowspan="1">89.7-96.0</td></tr></table>

Reproducibility Study Summary (Agreement with Expected Positive Results) for the Combined Results of the Influenza A/2009 H1 and the Influenza A Sw Assays in the JBAIDS Influenza A Subtyping Kit   

<table><tr><td colspan="1" rowspan="3">SampleType</td><td colspan="1" rowspan="3">Test Level</td><td colspan="4" rowspan="1">IT 1-2-3 Platinum Path Sample PurificationKit# of Positive Samples/# of Total Samples(% Positive Detection)</td><td colspan="4" rowspan="1">Roche MagNA Pure Compact Nucleic AcidIsolation Kit I# of Positive Samples/# of Total Samples(% Positive Detection)</td><td colspan="1" rowspan="3">AllPurificationKits, All Sites</td><td colspan="1" rowspan="3">95% CI</td></tr><tr><td colspan="4" rowspan="1">Test Location</td><td colspan="4" rowspan="1">Test Location</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">All Sites</td></tr><tr><td colspan="1" rowspan="5">N</td><td colspan="1" rowspan="1">3X LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD/20</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15a(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15a(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/9000%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">Detection ≥ LoD</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">98.3-99.9</td></tr><tr><td colspan="1" rowspan="1">Detection allLevels</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">135/135(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">135/135(100%)</td><td colspan="1" rowspan="1">270/270(100%)</td><td colspan="1" rowspan="1">98.9-99.9</td></tr><tr><td colspan="1" rowspan="3">MdN</td><td colspan="1" rowspan="1">3X LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15a(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90 (100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="1">LoD/20</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">15/15(1100%)</td><td colspan="1" rowspan="1">15/15a(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">15/1500%)</td><td colspan="1" rowspan="1">15/15a(100%)</td><td colspan="1" rowspan="1">15/15(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">90/90 (100%)</td><td colspan="1" rowspan="1">96.7-99.9</td></tr><tr><td colspan="1" rowspan="2">−</td><td colspan="1" rowspan="1">Detection ≥ LoD</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">30/30(100%)</td><td colspan="1" rowspan="1">90/90(100%)</td><td colspan="1" rowspan="1">180/180(100%)</td><td colspan="1" rowspan="1">98.3-99.9</td></tr><tr><td colspan="1" rowspan="1">Detection AllLevels</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">135/135(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">45/45(100%)</td><td colspan="1" rowspan="1">135/135(100%)</td><td colspan="1" rowspan="1">270/270(100%)</td><td colspan="1" rowspan="1">98.9-99.9</td></tr></table>

a Results of retest shown; first tests were inconclusive.

Note: The rate of positive test results at the high negative level was higher than expected and is most likely the result of variation in the preparation of the samples used for this study, potentially resulting in somewhat higher than expected levels of virus in these samples.

This reproducibility study demonstrated that as expected with all real-time PCR assays, the JBAIDS Influenza A Subtyping Kit may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration. This limitation should be addressed by including the following statement in the Limitation section of the JBAIDS Influenza A & B Detection Kit Instructions for Use: “The JBAIDS Influenza A Subtyping Kit may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LoD concentration, but higher than the assay cutoff concentration.”

The variability of the Cp value can be used as an additional measure of the variability of JBAIDS assays. Therefore, the average and $\% \mathrm { C V }$ of $\mathrm { C p }$ values obtained in this study were used to further evaluate the precision/reproducibility of the test system.

Reproducibility Study Summary (Mean Cp, SD and $\% \mathbf { C V } ,$ ) for the Influenza A/H1 Assay in the JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>Virus pikeLevel</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum PathSample Purification Kit</td><td rowspan=1 colspan=8>Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>MeanCp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>32.87(0.29)</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>33.24(0.47)</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>33.54(0.68)</td><td rowspan=1 colspan=1>2.03</td><td rowspan=1 colspan=1>33.22(0.57)</td><td rowspan=1 colspan=1>1.71</td><td rowspan=1 colspan=1>31.64(0.65)</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>31.91(0.54)</td><td rowspan=1 colspan=1>1.70</td><td rowspan=1 colspan=1>31.62(0.65)</td><td rowspan=1 colspan=1>2.06</td><td rowspan=1 colspan=1>31.72(0.62)</td><td rowspan=1 colspan=1>1.95</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>34.63(0.59)</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>34.31(0.71)</td><td rowspan=1 colspan=1>2.07</td><td rowspan=1 colspan=1>34.84(0.42)</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>34.56(0.61)</td><td rowspan=1 colspan=1>1.77</td><td rowspan=1 colspan=1>32.85(0.81)</td><td rowspan=1 colspan=1>2.48</td><td rowspan=1 colspan=1>32.82(0.71)</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>32.75(0.65)</td><td rowspan=1 colspan=1>1.98</td><td rowspan=1 colspan=1>32.81(0.71)</td><td rowspan=1 colspan=1>2.16</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>38.19a(2.03)</td><td rowspan=1 colspan=1>5.31a</td><td rowspan=1 colspan=1>37.07(1.03)</td><td rowspan=1 colspan=1>2.77</td><td rowspan=1 colspan=1>38.73a(1.71)</td><td rowspan=1 colspan=1>4.41</td><td rowspan=1 colspan=1>37.90a(1.74)</td><td rowspan=1 colspan=1>4.60 a</td><td rowspan=1 colspan=1>34.69(0.80)</td><td rowspan=1 colspan=1>2.31</td><td rowspan=1 colspan=1>35.32(0.92)</td><td rowspan=1 colspan=1>2.61</td><td rowspan=1 colspan=1>35.01(0.85)</td><td rowspan=1 colspan=1>2.43</td><td rowspan=1 colspan=1>34.99(0.87)</td><td rowspan=1 colspan=1>2.49</td></tr><tr><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>33.12(0.7)</td><td rowspan=1 colspan=1>2.32</td><td rowspan=1 colspan=1>33.04b(0.97)</td><td rowspan=1 colspan=1>2.94b</td><td rowspan=1 colspan=1>34.03(0.49)</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>33.39(0.88)</td><td rowspan=1 colspan=1>2.63</td><td rowspan=1 colspan=1>31.95(0.53)</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>32.38(0.51)</td><td rowspan=1 colspan=1>1.58</td><td rowspan=1 colspan=1>31.32(0.51)</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>31.88(0.67)</td><td rowspan=1 colspan=1>2.09</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>34.76(0.65)</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>34.19(0.35)</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>35.57(0.82)</td><td rowspan=1 colspan=1>2.31</td><td rowspan=1 colspan=1>34.84(0.85)</td><td rowspan=1 colspan=1>2.44</td><td rowspan=1 colspan=1>33.19(0.48)</td><td rowspan=1 colspan=1>1.45</td><td rowspan=1 colspan=1>33.54(0.44)</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>32.85(0.36)</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>33.18(0.50)</td><td rowspan=1 colspan=1>1.52</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>40.02a(2.46)</td><td rowspan=1 colspan=1>6.16</td><td rowspan=1 colspan=1>38.60a,b(2.23)</td><td rowspan=1 colspan=1>5.79a,b</td><td rowspan=1 colspan=1>40.60a(2.39)</td><td rowspan=1 colspan=1>5.89 </td><td rowspan=1 colspan=1>39.59a(2.44)</td><td rowspan=1 colspan=1>6.17a</td><td rowspan=1 colspan=1>36.05(0.69)</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>36.95(1.05)</td><td rowspan=1 colspan=1>2.85</td><td rowspan=1 colspan=1>36.07(0.69)</td><td rowspan=1 colspan=1>1.92</td><td rowspan=1 colspan=1>36.35(0.91)</td><td rowspan=1 colspan=1>2.50</td></tr></table>

a One or more capillaries had a Cp of ${ \tt > } 4 0 . 0 0 $ and was assigned a Cp of 42.50 cycles. b One medium positive sample and a high negative sample were switched during testing. Based on $\mathrm { C p }$ values they have been assigned to the correct virus spike level during data analysis.

Reproducibility Study Summary (Mean Cp, SD and $\% \mathbf { C V }$ ) for the Influenza A/H3 Assay in the JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>ViruspikeLevel</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum PathSample Purification Kit</td><td rowspan=1 colspan=8>Roche MagNA Pure CompactNucleic Acid Isolation Kit I</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>MeanCp(SD)</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>ov</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Cv</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>ov</td></tr><tr><td rowspan=3 colspan=1>sNPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>34.14(0.33)</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>33.63(0.84)</td><td rowspan=1 colspan=1>2.50</td><td rowspan=1 colspan=1>34.09(0.91)</td><td rowspan=1 colspan=1>2.66</td><td rowspan=1 colspan=1>33.96(0.76)</td><td rowspan=1 colspan=1>2.23</td><td rowspan=1 colspan=1>32.65(0.36)</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>32.57(0.48)</td><td rowspan=1 colspan=1>1.49</td><td rowspan=1 colspan=1>31.71(0.17)</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>32.34(0.55)</td><td rowspan=1 colspan=1>1.70</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>35.32(0.54)</td><td rowspan=1 colspan=1>1.52</td><td rowspan=1 colspan=1>34.99(1.07)</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>35.64(0.40)</td><td rowspan=1 colspan=1>1.13</td><td rowspan=1 colspan=1>35.32(0.76)</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>33.85(0.33)</td><td rowspan=1 colspan=1>0.97</td><td rowspan=1 colspan=1>33.89(0.32)</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>32.81(0.33)</td><td rowspan=1 colspan=1>1.02</td><td rowspan=1 colspan=1>33.52(0.60)</td><td rowspan=1 colspan=1>1.78</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>39.99a(1.96)</td><td rowspan=1 colspan=1>4.90 </td><td rowspan=1 colspan=1>39.11a(2.60)</td><td rowspan=1 colspan=1>6.64 a</td><td rowspan=1 colspan=1>39.70 a(3.01)</td><td rowspan=1 colspan=1>7.57</td><td rowspan=1 colspan=1>39.62 a(2.45)</td><td rowspan=1 colspan=1>6.17 a</td><td rowspan=1 colspan=1>36.81(0.64)</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>36.85(0.60)</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>36.01(0.37)</td><td rowspan=1 colspan=1>1.03</td><td rowspan=1 colspan=1>36.55(0.66)</td><td rowspan=1 colspan=1>1.81</td></tr><tr><td rowspan=3 colspan=1>sNPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>34.29(0.43)</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>34.32(0.56)</td><td rowspan=1 colspan=1>1.64</td><td rowspan=1 colspan=1>34.31(0.49)</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>34.31(0.49)</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>31.97(0.69)</td><td rowspan=1 colspan=1>2.15</td><td rowspan=1 colspan=1>32.11(0.61)</td><td rowspan=1 colspan=1>1.89</td><td rowspan=1 colspan=1>31.46(0.28)</td><td rowspan=1 colspan=1>0.88</td><td rowspan=1 colspan=1>31.85(0.61)</td><td rowspan=1 colspan=1>1.91</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>35.99(0.61)</td><td rowspan=1 colspan=1>1.70</td><td rowspan=1 colspan=1>36.28a(1.85)</td><td rowspan=1 colspan=1>5.10 a</td><td rowspan=1 colspan=1>35.90(0.80)</td><td rowspan=1 colspan=1>2.22</td><td rowspan=1 colspan=1>36.06 a(1.21)</td><td rowspan=1 colspan=1>3.35a</td><td rowspan=1 colspan=1>33.36(0.50)</td><td rowspan=1 colspan=1>1.50</td><td rowspan=1 colspan=1>34.01(0.81)</td><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>32.94(0.23)</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>33.44(0.71)</td><td rowspan=1 colspan=1>2.12</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>42.14a(1.21)</td><td rowspan=1 colspan=1>2.87</td><td rowspan=1 colspan=1>40.80a(2.25)</td><td rowspan=1 colspan=1>5.52 a</td><td rowspan=1 colspan=1>41.74a(1.55)</td><td rowspan=1 colspan=1>3.72a</td><td rowspan=1 colspan=1>41.57a(1.75)</td><td rowspan=1 colspan=1>4.22 </td><td rowspan=1 colspan=1>37.20(1.02)</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>37.80 a(1.88)</td><td rowspan=1 colspan=1>4.97</td><td rowspan=1 colspan=1>36.52(0.60)</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>37.16 a(1.34)</td><td rowspan=1 colspan=1>3.61a</td></tr></table>

a One or more capillaries had a Cp of ${ \tt > } 4 0 . 0 0 $ and was assigned a Cp of 42.50 cycles.

Reproducibility Study Summary (Mean Cp, SD and $\% \mathbf { C V }$ ) for the 2009 H1N1 Influenza Assays in the JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=4 colspan=1>InfluenzaAssay</td><td rowspan=4 colspan=1>SampleType</td><td rowspan=4 colspan=1>VirusSpike</td><td rowspan=1 colspan=8>IT 1-2-3 Platinum Path SamplePurification Kit</td><td rowspan=1 colspan=8>Roche MagNA Pure Compact NucleicAcid Isolation Kit I</td></tr><tr><td rowspan=1 colspan=8>Test Location</td><td rowspan=1 colspan=8>Test Location</td></tr><tr><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All Sites</td><td rowspan=1 colspan=2>Site 1</td><td rowspan=1 colspan=2>Site 2</td><td rowspan=1 colspan=2>Site 3</td><td rowspan=1 colspan=2>All Sites</td></tr><tr><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>MeanCp (SD)% CVC</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>)% CV</td><td rowspan=1 colspan=1>MeanCp (SD) % CV</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>OV</td><td rowspan=1 colspan=1>MeanCp (SD)</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=6 colspan=1>Flu A Sw</td><td rowspan=3 colspan=1>sNPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>28.43(0.44)</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>28.78(0.38)</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>28.85(0.28)</td><td rowspan=1 colspan=1>0.99</td><td rowspan=1 colspan=1>28.69(0.41)</td><td rowspan=1 colspan=1>1.43</td><td rowspan=1 colspan=1>25.60a(0.40)</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>26.15(0.34)</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>25.81(0.29)</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>28.85(0.41)</td><td rowspan=1 colspan=1>1.57</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>30.07(0.60)</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>30.26(0.75)</td><td rowspan=1 colspan=1>2.46</td><td rowspan=1 colspan=1>30.03(0.35)</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>30.12(0.59)</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>26.98(0.33)</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>27.49(0.24)</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>27.12(0.28)</td><td rowspan=1 colspan=1>1.05</td><td rowspan=1 colspan=1>27.20(0.35)</td><td rowspan=1 colspan=1>1.29</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>32.81(0.47)</td><td rowspan=1 colspan=1>1.44</td><td rowspan=1 colspan=1>32.98(0.55)</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>32.97(0.62)</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>32.92(0.54)</td><td rowspan=1 colspan=1>1.65</td><td rowspan=1 colspan=1>29.94(0.55)</td><td rowspan=1 colspan=1>1.83</td><td rowspan=1 colspan=1>30.47(0.39)</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>29.99(0.47)</td><td rowspan=1 colspan=1>1.57</td><td rowspan=1 colspan=1>30.31(0.52)</td><td rowspan=1 colspan=1>1.73</td></tr><tr><td rowspan=3 colspan=1>sNPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>30.01(0.72)</td><td rowspan=1 colspan=1>2.38</td><td rowspan=1 colspan=1>29.85(0.57)</td><td rowspan=1 colspan=1>1.90</td><td rowspan=1 colspan=1>29.25(0.62)</td><td rowspan=1 colspan=1>2.13</td><td rowspan=1 colspan=1>29.74(0.70)</td><td rowspan=1 colspan=1>2.34</td><td rowspan=1 colspan=1>26.32(0.32)</td><td rowspan=1 colspan=1>1.22</td><td rowspan=1 colspan=1>26.62(0.52)</td><td rowspan=1 colspan=1>1.96</td><td rowspan=1 colspan=1>26.25(0.34)</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>26.40(0.43)</td><td rowspan=1 colspan=1>1.62</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>31.20(0.56)</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>31.28(0.57)</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>30.85(0.68)</td><td rowspan=1 colspan=1>2.20</td><td rowspan=1 colspan=1>31.11(0.62)</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>27.70(0.34)</td><td rowspan=1 colspan=1>1.24</td><td rowspan=1 colspan=1>28.13(0.33)</td><td rowspan=1 colspan=1>1.18</td><td rowspan=1 colspan=1>27.63(0.31)</td><td rowspan=1 colspan=1>1.14</td><td rowspan=1 colspan=1>27.82(0.39)</td><td rowspan=1 colspan=1>1.41</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>33.80(0.39)</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>33.86(0.73)</td><td rowspan=1 colspan=1>2.16</td><td rowspan=1 colspan=1>33.32(0.54)</td><td rowspan=1 colspan=1>1.61</td><td rowspan=1 colspan=1>33.66(0.61)</td><td rowspan=1 colspan=1>1.81</td><td rowspan=1 colspan=1>30.89(0.53)</td><td rowspan=1 colspan=1>1.72</td><td rowspan=1 colspan=1>31.07(0.42)</td><td rowspan=1 colspan=1>1.34</td><td rowspan=1 colspan=1>30.48(0.43)</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>30.81(0.52)</td><td rowspan=1 colspan=1>1.68</td></tr><tr><td rowspan=1 colspan=1>→</td><td rowspan=1 colspan=1>sNPS</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>31.19(0.48)</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>31.13(0.41)</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>31.41(0.41)</td><td rowspan=1 colspan=1>1.32</td><td rowspan=1 colspan=1>31.24(0.44)</td><td rowspan=1 colspan=1>1.42</td><td rowspan=1 colspan=1>28.84a(0.50)</td><td rowspan=1 colspan=1>1.73</td><td rowspan=1 colspan=1>29.06(0.33)</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>28.91(0.23)</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>28.93(0.38)</td><td rowspan=1 colspan=1>1.30</td></tr></table>

<table><tr><td rowspan=3 colspan=1>b</td><td rowspan=2 colspan=1>LoD</td><td rowspan=2 colspan=1>32.92(0.52)</td><td rowspan=2 colspan=1>1.59</td><td rowspan=2 colspan=1>32.86(0.83)</td><td rowspan=2 colspan=1>2.54</td><td rowspan=1 colspan=1>32.99</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>32.92</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30.34</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>30.56(0.31)</td><td rowspan=2 colspan=1>1.01</td><td rowspan=2 colspan=1>30.36(0.30)</td><td rowspan=2 colspan=1>0.99</td><td rowspan=2 colspan=1>30.42(0.34)</td><td rowspan=2 colspan=1>1.12</td></tr><tr><td rowspan=1 colspan=1>(0.40)</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>(0.60)</td><td rowspan=1 colspan=1>1.83</td><td rowspan=1 colspan=1>(0.38)</td><td rowspan=1 colspan=1>1.26</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>37.11(0.76)</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>36.94(0.54)</td><td rowspan=1 colspan=1>1.46</td><td rowspan=1 colspan=1>37.78a(1.99)</td><td rowspan=1 colspan=1>5.26 a</td><td rowspan=1 colspan=1>37.28a(1.29)</td><td rowspan=1 colspan=1>3.46 a</td><td rowspan=1 colspan=1>34.24(0.57)</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>34.45(0.37)</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>34.24(0.31)</td><td rowspan=1 colspan=1>0.92</td><td rowspan=1 colspan=1>34.31(0.43)</td><td rowspan=1 colspan=1>1.27</td></tr><tr><td rowspan=3 colspan=1>sNPW</td><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>31.90(0.24)</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>32.03(0.52)</td><td rowspan=1 colspan=1>1.63</td><td rowspan=1 colspan=1>31.67(0.56)</td><td rowspan=1 colspan=1>1.78</td><td rowspan=1 colspan=1>31.89(0.48)</td><td rowspan=1 colspan=1>1.51</td><td rowspan=1 colspan=1>29.23(0.36)</td><td rowspan=1 colspan=1>1.24</td><td rowspan=1 colspan=1>29.44(0.32)</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>29.34(0.30)</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>29.34(0.33)</td><td rowspan=1 colspan=1>1.13</td></tr><tr><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>33.49(0.38)</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>33.48(0.60)</td><td rowspan=1 colspan=1>1.80</td><td rowspan=1 colspan=1>33.33(0.66)</td><td rowspan=1 colspan=1>1.99</td><td rowspan=1 colspan=1>33.44(0.55)</td><td rowspan=1 colspan=1>1.66</td><td rowspan=1 colspan=1>30.72(0.39)</td><td rowspan=1 colspan=1>1.28</td><td rowspan=1 colspan=1>30.92(0.23)</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>30.76(0.12)</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>30.80(0.28)</td><td rowspan=1 colspan=1>0.91</td></tr><tr><td rowspan=1 colspan=1>LoD/20</td><td rowspan=1 colspan=1>37.49 a(1.73)</td><td rowspan=1 colspan=1>4.62 a</td><td rowspan=1 colspan=1>37.59 a(1.89)</td><td rowspan=1 colspan=1>5.04 a</td><td rowspan=1 colspan=1>36.88 a(1.69)</td><td rowspan=1 colspan=1>4.57a</td><td rowspan=1 colspan=1>37.32a(1.76)</td><td rowspan=1 colspan=1>4.72 a</td><td rowspan=1 colspan=1>34.28(0.39)</td><td rowspan=1 colspan=1>1.15</td><td rowspan=1 colspan=1>34.59(0.49)</td><td rowspan=1 colspan=1>1.41</td><td rowspan=1 colspan=1>34.32(0.36)</td><td rowspan=1 colspan=1>1.04</td><td rowspan=1 colspan=1>34.40(0.43)</td><td rowspan=1 colspan=1>1.25</td></tr></table>

a One or more capillaries had a Cp of ${ > } 4 0 . 0 0$ and was assigned a Cp of 42.50 cycles.

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The following controls are included in the JBAIDS Influenza A Subtyping Kit:

# Negative Control

The NC is used to detect contamination from target-specific amplified product (amplicon), synthetic RNA (as found in the PC vials), or virus. The NC vial is reconstituted using the purified negative extraction control (NEC) (molecular biology grade water). The NEC should be processed along with each batch of specimens purified. The NC detects possible contamination during sample purification and during reconstitution of freeze- dried reagents. Each JBAIDS Influenza A Subtyping Kit run requires one NC for each of the 4 assays (i.e., the Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw assays). For each target assay, the NC must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run. If the NC fails, the JBAIDS run must be repeated using the same purified samples. If the NC fails again, all associated samples are considered to have invalid results and should be re-purified starting from another aliquot of the original specimens.

Sixty-one of the 63 Flu A H1 NEC reactions (61/63; $9 6 . 8 \%$ tested in the multisite reproducibility study yielded the expected negative target results and were valid. All controls were valid on the retest run. Sixty-two out of 65 Flu A H3 NEC reactions tested in this study were valid $( 6 2 / 6 5 ; 9 5 . 4 \% )$ . All controls were valid on the retest run. All 63 Flu A Sw and Flu A H1 2009 NEC reactions tested in this study were valid.

During the prospective clinical study, of 261 runs, there were a total of 8 $( 3 . 1 \% )$ runs in which the NCs failed. Site 1 and Site 2 had 3 NC failures each, all of which were associated with the Flu A H1 2009 assay. Site 4 and Site 6 each had 1 run failure with the Flu A H1 2009 NC. All associated specimens were retested and no specimens were lost due to these failures.

# Positive Control

The PC serves as an amplification and detection control. Each JBAIDS Influenza A Subtyping Kit run requires one PC for each of the 4 assays (i.e., the Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw assays). For each target assay, the PC must be positive. If the PC fails, the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the JBAIDS run must be repeated starting from the same purified specimens. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument.

Of the 63 Flu A H1 PC reactions tested in the reproducibility study, all were valid $( 6 3 / 6 3 ; 1 0 0 \% )$ . The Flu A H1 PC reactions had a mean $\mathrm { C p }$ of $3 1 . 4 1 \pm 0 . 5 6$ (1.78 $\% C V )$ . Sixty-two out of 65 Flu A H3 PC reactions tested in this study were valid (62/65; $9 5 . 4 \%$ . The mean $\mathrm { C p }$ of the Flu A H3 PC reactions was $3 0 . 8 6 \pm 0 . 2 0$ $( 0 . 6 5 \%$ CV). All controls were valid on the retest run. All 63 Flu A Sw PC reactions tested in this study were valid. The mean $\mathrm { C p }$ of the Flu A Sw PC reactions was $2 8 . 2 7 \pm 0 . 3 0$ ( $1 . 0 6 \%$ CV). All 63 Flu A H1 2009 PC reactions tested were valid. The mean $\mathrm { C p }$ of the Flu A H1 2009 PC reactions was $3 2 . 0 1 \pm 0 . 3 9$ ( $1 . 2 2 \%$ CV).

During the prospective clinical study, of 261 runs, there were a total of 2 $( 0 . 8 \% )$ runs in which the PCs failed. Site 2 had 1 Flu A H1 2009 PC failure. Site 6 had 1 Flu A Sw PC failure. All associated specimens were retested and no specimens were lost due to these failures. The Flu A H1 PC reactions had a mean $\mathrm { C p }$ of 32.1 $\pm 0 . 6$ ( $1 . 8 \%$ CV). The mean $\mathrm { C p }$ of the Flu A H3 PC reactions was $3 0 . 3 \pm 0 . 3$ $( 1 . 0 \%$ CV). The mean $\mathrm { C p }$ of the Flu A Sw PC reactions was $2 8 . 9 \pm 0 . 4$ ( $1 . 5 \%$ CV). The mean Cp of the Flu A H1 2009 PC reactions was $3 1 . 4 \pm 0 . 3$ ( $1 . 1 \%$ CV).

The final $\mathrm { C p }$ cutoff values for the Flu A H1, Flu A H3, Flu A Sw and Flu A H1 2009 positive controls are 37.1, 36.3, 33.4, and 36.7 respectively, and are based on the upper bound of the manufacturing QA Review range for reagent lot release plus $4 \mathrm { x } \mathrm { S D }$ . The estimated standard deviation (SD) was set to $0 . 9 \mathrm { C p }$ to reflect the amount of variation that is consistent with data obtained in the JBAIDS Influenza A Subtyping Kit and the JBAIDS Influenza A & B Detection Kit analytical and clinical studies. If an amplification curve for a PC fails to amplify or has a Cp value later than the cutoff, the PC will be interpreted as a failure and the associated patient test results will be assigned an invalid test result.

# Extraction Control (EC)

Positive Extraction Control (PEC) was not provided with the kit, and inclusion of a PEC with each batch of extracted samples is not a standard part of the testing protocol recommended in the product package insert. However, during the clinical study, a PEC and a NEC were purified with each batch of clinical samples. Inclusion of a NEC with each batch of purified samples is a standard part of the testing protocol recommended in the product package insert. PECs were simulated NPS (sNPS; HeLa cells in viral transport media) spiked with four representative influenza strains (Influenza B,

seasonal Influenza A H1N1, seasonal Influenza A H3N2, and 2009 H1N1 Influenza) at concentrations near the LoD for the influenza assays. The PEC was processed and tested in the same manner as a patient sample. The PEC was considered successful when all tested target assays were positive. A failed PEC was retested using the same purified sample and the results of the retest were accepted as the final result. If the retest was unsuccessful, all associated samples were considered to have invalid results and required re-purification in order to obtain a valid test result.

During the prospective clinical evaluation, 11 out of a total of 194 PECs required retesting. Nine retests (3 at Site 1, 3 at Site 2, 1 at Site 4, and 2 at Site 6) were required due to an invalid assay result (PC or NC). Two PEC failures occurred as a result of one of the four target assays giving a negative result (Flu A H1 one time and Flu A H1 2009 one time). Retesting was performed for one of the failures (Flu A H1 2009) which resolved the failure. There was no retest performed for the other failure. This lack of retesting was a protocol deviation. Five (5) samples associated with this run were excluded from data analysis. The mean $\mathrm { C p }$ of the PEC tested for the Flu A H1 assay was 31.1, with 2.7 Standard Deviation (SD) and $8 . 6 \%$ CV. The mean $\mathrm { C p }$ of the PEC tested for the Flu A H3 assay was 30.7, with 1.5 Standard Deviation (SD) and $4 . 7 \%$ CV. The mean $\mathrm { C p }$ of the PEC tested for the Flu A H1 2009 assay was 31.2, with 1.9 Standard Deviation (SD) and $6 . 1 \%$ CV. The mean $\mathrm { C p }$ of the PEC tested for the Flu A Sw assay was 29.6, with 2.0 Standard Deviation (SD) and $6 . 8 \%$ CV.

The NEC was a nuclease-free water sample that was processed with each batch of purified specimens. The purified NEC sample was then used to reconstitute the freeze- dried NC.

# Specimen Stability

The sponsor recommends that MPW and NPS specimens be collected and placed into viral transport media (VTM). Once collected, specimens may be tested immediately after collection, transported to an off-site testing location, or temporarily stored until testing can be scheduled. Inappropriate handling or storage may compromise the integrity of the specimens and cause inaccurate test results. An analytical study determining the appropriate storage times and temperatures for both unprocessed specimens (e.g., the NPW and NPS specimens) and processed samples (e.g., purified nucleic acid from NPW and NPS specimens) was carried out. The specimen transport and storage conditions that were evaluated are based on common clinical laboratory workflows.

Three Influenza A subtype strains (i.e., Influenza A/H1, Influenza A/H3, and 2009 H1N1 Influenza) were evaluated with the JBAIDS Influenza A Subtyping Kit for both unprocessed and purified samples.

Note: Specimens spiked with a representative Influenza A/H3 virus strain (A/New York/55/2004) were also co-spiked with a representative Influenza B virus strain (B/Ohio/1/2005) for more efficient testing of the JBAIDS Influenza A Subtyping Kit. Refer to the “Co-spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus.

For the unprocessed specimen transport and storage evaluation, both negative and positive specimens were assessed. Positive specimens were spiked with appropriate Influenza A viruses at the established LoD for the respective assay. Eight positive (per influenza strain) and four negative specimens were processed at each time point and storage condition per purification kit prior to testing with the freeze-dried reagents. Both specimen types (NPW and NPS) were evaluated independently.

For the purified sample transport and storage evaluation, sets of eight positive samples (per influenza strain) and four negative samples that had been processed with either purification method were tested immediately and also aliquoted for storage under various conditions prior to retesting with the freeze-dried reagents. Purified samples were also tested to determine the storage of reconstituted freeze-dried reagents.

The following table summarizes the final conditions that were evaluated in this study:

Final Storage Conditions Evaluated for each Sample Matrix/Purification Kit Combination   

<table><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=1>Conditions</td></tr><tr><td rowspan=1 colspan=1>Unprocessed Specimens</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, ambient (18-30°C)Day 3, refrigerated (2-8°C)Day 30, frozen (≤ -15°C)</td></tr><tr><td rowspan=1 colspan=1>Purified Samples</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, ambient (18-30°C)Day 1, refrigerated (2-8°C)Day 30 , frozen (≤ -15°C) Platinum Path onlyDay 30, frozen (≤ -70°C) MagNA Pure only</td></tr><tr><td rowspan=1 colspan=1>Reconstituted Freeze Dried-Reagents</td><td rowspan=1 colspan=1>Day 0, no storage4 Hours, refrigerated (2-8°C)</td></tr></table>

The performance at each time point was evaluated by comparing the JBAIDS assay result (positive, negative, uncertain, or SC fail) to the expected result and the result at the initial time point. To be acceptable, all negative samples must be negative for the influenza subtyping assays and positive for the Flu SC assay, and seven of eight positive samples must test positive for the relevant assays (Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw). In addition, the mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values of the stored sample set were compared to those obtained at the initial time point. For an acceptable storage condition, the mean $\mathrm { C p }$ of the influenza subtyping assays for the positive sample set should not increase by more than 3 cycles, and the mean $\mathrm { F } _ { \mathrm { m a x } }$ must not decrease by more than $50 \%$ .

Unprocessed Specimen Storage Conditions Test Results

At least seven of the eight positive human derived NPS and NPW specimens in each sample set tested positive at all four time points with the respective Influenza A subtyping assay. The mean Cp values for the stored specimens were all within three cycles of the day 0 time point suggesting no decrease in assay performance with storage. In addition, the mean $\mathrm { F } _ { \mathrm { m a x } }$ values for the time points were never less than $50 \%$ of the

$\mathrm { F } _ { \mathrm { m a x } }$ of the day 0 baseline indicating no reduction of test system performance. All negative specimens tested negative at each time point.

These results indicate that NPS and NPW specimens can be stored at $1 8 { - } 3 0 ~ ^ { \circ } \mathrm { C }$ for up to four hours, at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to three days, and at $\leq - 1 5 ~ ^ { \circ } \mathrm { C }$ for up to thirty days prior to testing without compromising the results.

Purified Sample Storage Conditions Test Results

At least seven of the eight purified human derived NPS and NPW positive samples in each sample set tested positive at each time point with the respective Influenza A subtyping assay. The average Cp values for the stored samples were all within 3 cycles of the day 0 time point suggesting no decrease in assay performance with storage. In addition, the average $\mathrm { F } _ { \mathrm { m a x } }$ values for the time points were never less than $50 \%$ of the $\mathrm { F } _ { \mathrm { m a x } }$ of the day 0 baseline indicating no reduction of test system performance. All negative specimens tested negative at each time point.

These results indicate that purified samples can be stored at $1 8 { - } 3 0 ~ ^ { \circ } \mathrm { C }$ for up to four hours, at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to one day, and frozen $\mathrm { \Omega } \leq - 1 5 \mathrm { \Omega } ^ { \circ } \mathrm { C }$ for Platinum Path purified samples, $\le - 7 0 ^ { \circ } \mathrm { C }$ for MagNA Pure purified samples) for up to thirty days prior to testing without compromising the results.

Reconstituted Reagent Storage Conditions Test Results

All but one of the positive samples tested positive at both the initial time point and after four hours of refrigeration. The mean $\mathrm { C p }$ values for the stored reactions were all within 3 cycles of the initial time point suggesting no decrease in assay performance with storage. In addition, the mean $\mathrm { F } _ { \mathrm { m a x } }$ values for the stored reagents were never less than $50 \%$ of the $\mathrm { F } _ { \mathrm { m a x } }$ of the initial baseline indicating no reduction of test system performance. All negative reactions tested negative at both time points.

These results indicate that reconstituted reagents can be stored at $2 { - } 8 ~ ^ { \circ } \mathrm { C }$ for up to four hours without compromising the results.

# d. Detection limit:

Initial estimates for setting the LoD were performed by spiking simulated (s)NPW and sNPS samples with serial dilutions of quantified representative Influenza A viruses (Influenza A H1N1 strain A/New Caledonia/20/1999, Influenza A H1N1 strain A/Hawaii/15/2001, Influenza A H3N2 strain A/New York/55/2004, Influenza A H3N2 strain A/Wisconsin/67/2005, 2009 H1N1 Influenza strain A/New York/18/2009, 2009 H1N1 Influenza strain A/California/7/2009). The spiked samples were then processed using the IT 1-2-3 Platinum Path Sample Purification Kit and the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I, and the purified samples were tested with the associated Influenza A subtyping assays (Flu A H1, Flu A H3, Flu A H12009, and Flu A Sw). An initial estimate of the LoD was made by evaluating the reaction quality at each dilution. Reactions were evaluated using multiple criteria, including the JBAIDS test results, the Cp values, and a visual examination of the amplification curves. LoD estimates were then further evaluated by testing human-derived or simulated samples spiked with refined serial dilutions of the representative influenza viruses.

Once the LoD was estimated, a set of 20 specimens was collected from 20 unique donors for each combination of specimen type (NPS or NPW) and purification kit (Platinum Path or MagNA Pure). The samples were spiked at the estimated LoD level, processed, and tested with the JBAIDS Influenza A subtyping assays to determine the frequency of positive results. The LoD was confirmed if at least 19 of the 20 samples gave a positive test result.

The LoD levels for detection of each virus subtype were independently established using two representative Influenza A strains of each subtype (seasonal Influenza A H1N1, seasonal Influenza A H3N2, and 2009 H1N1 Influenza). The LoD for detection of the 2009 H1N1 Influenza strain required positive results for both the Flu A Sw and Flu A H1 2009 assays. Specifically, the lowest concentration of 2009 H1N1 Influenza virus where both assays exhibited $\ge 9 5 \%$ positive detection was considered the final LoD.

Based on the titration results for each strain, the LoD levels were set at the concentrations for which $2 9 5 \%$ detection was observed for both sample types and both nucleic acid purification kits. All sets of 20 samples spiked with Influenza A viruses gave at least 19/20 positive results with the appropriate JBAIDS Flu A subtyping assay. The LoD for the Flu A/ H1 assay is $5 0 ~ \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/New Caledonia/20/1999 and $5 , 0 0 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/Hawaii/15/2001. The LoD for the Flu A H3 assay is 5 $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/New York/55/2004 and $1 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/Wisconsin/67/2005. The LoD for detection of the Influenza $\mathbf { A } / 2 0 0 9 \mathrm { H } 1$ virus is $1 { , } 5 0 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/New York/18/2009 and $5 , 0 0 0 \mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ for A/California/7/2009.

Confirmation of Estimated LoD Levels for the Flu A /H1 Assay in the JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>LoD(EID50/mL)</td><td rowspan=2 colspan=1>Kit(Sample Type)</td><td rowspan=1 colspan=3>Flu A H1</td></tr><tr><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Mean Fmax(SD)</td></tr><tr><td rowspan=4 colspan=1>ANuo cn6661/07</td><td rowspan=4 colspan=1>50</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.01(0.72)</td><td rowspan=1 colspan=1>19.12(4.59)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.49(0.54)</td><td rowspan=1 colspan=1>22.34(4.78)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.02(0.99)</td><td rowspan=1 colspan=1>21.74(6.06)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.17(0.52)</td><td rowspan=1 colspan=1>23.69(3.78)</td></tr><tr><td rowspan=4 colspan=1>AWMMy/l1303/90</td><td rowspan=4 colspan=1>5,000</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.66(0.61)</td><td rowspan=1 colspan=1>24.18(4.03)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.96(0.69)</td><td rowspan=1 colspan=1>23.30(5.83)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.81(0.55)</td><td rowspan=1 colspan=1>26.74(2.65)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.86(0.43)</td><td rowspan=1 colspan=1>30.72(3.62)</td></tr></table>

# Confirmation of Estimated LoD Levels for the Flu A /H3 Assay in the

JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>LoD(EID50/mL)</td><td rowspan=2 colspan=1>Kit(Sample Type)</td><td rowspan=1 colspan=3>Flu A H3</td></tr><tr><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>MeanCp(SD)</td><td rowspan=1 colspan=1>Mean Fmax(SD)</td></tr><tr><td rowspan=4 colspan=1>AM M5000/09</td><td rowspan=4 colspan=1>5</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>32.63(0.48)</td><td rowspan=1 colspan=1>21.52(4.54)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.45(0.52)</td><td rowspan=1 colspan=1>26.59(5.87)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.10(0.55)</td><td rowspan=1 colspan=1>29.08(7.32)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>35.50(0.65)</td><td rowspan=1 colspan=1>23.46(5.31)</td></tr><tr><td rowspan=4 colspan=1>/aams!i∀</td><td rowspan=4 colspan=1>10</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.67(0.70)</td><td rowspan=1 colspan=1>27.43(5.50)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.61(0.88)</td><td rowspan=1 colspan=1>25.84(6.30)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.16(0.47)</td><td rowspan=1 colspan=1>35.17(5.17)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.75(0.42)</td><td rowspan=1 colspan=1>36.86(5.01)</td></tr></table>

Confirmation of Estimated LoD Levels detection of 2009 H1N1 Influenza virus with the Flu A H1 2009 and Flu A Sw assays of JBAIDS Influenza A Subtyping Kit   

<table><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>LoD(EID50/mL)</td><td rowspan=2 colspan=1>Kit(Sample Type)</td><td rowspan=1 colspan=3>Flu A Sw</td><td rowspan=1 colspan=3>Flu A H1 2009</td></tr><tr><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Mean Fmax(SD)</td><td rowspan=1 colspan=1>Positive/Total</td><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Mean Fmax(SD)</td></tr><tr><td rowspan=4 colspan=1>AM MN6007/81</td><td rowspan=4 colspan=1>1,500</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>27.59(0.45)</td><td rowspan=1 colspan=1>11.40(1.50)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.52(0.42)</td><td rowspan=1 colspan=1>15.95(1.09)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>29.85(0.80)</td><td rowspan=1 colspan=1>9.25(2.76)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>34.01(0.71)</td><td rowspan=1 colspan=1>17.61(3.15)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>28.57(0.89)</td><td rowspan=1 colspan=1>11.68(2.54)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.09(0.43)</td><td rowspan=1 colspan=1>16.05(1.06)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.15(0.53)</td><td rowspan=1 colspan=1>11.70(2.63)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.68(0.39)</td><td rowspan=1 colspan=1>13.98(1.34)</td></tr><tr><td rowspan=4 colspan=1>Arc6007/20</td><td rowspan=4 colspan=1>5,000</td><td rowspan=1 colspan=1>MagNA Pure(NPS)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>27.88(0.53)</td><td rowspan=1 colspan=1>9.36(1.58)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.98(0.40)</td><td rowspan=1 colspan=1>16.10(1.02)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPS)</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>29.78(0.58)</td><td rowspan=1 colspan=1>10.15(2.83)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>32.99(0.37)</td><td rowspan=1 colspan=1>13.40(1.58)</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>28.83(0.47)</td><td rowspan=1 colspan=1>13.62(1.11)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>31.55(0.52)</td><td rowspan=1 colspan=1>17.87(1.12)</td></tr><tr><td rowspan=1 colspan=1>Platinum Path(NPW)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>30.82(0.63)</td><td rowspan=1 colspan=1>13.74(1.84)</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>33.15(0.64)</td><td rowspan=1 colspan=1>15.13(0.90)</td></tr></table>

Note: One possible cause for the variation in LoD concentrations between strains is differences in the method of quantification and detection. Quantification using $\mathrm { E I D } _ { 5 0 }$ measures the infectivity and cytotoxicity or lethality of an organism in egg culture, and is therefore subject to many influences (strain-to-strain differences in infectivity, viability of the original material, culturing conditions, etc.). In contrast, the JBAIDS assays detect the presence of target nucleic acid using PCR; differences in detection reflect the quantity of starting target nucleic acid present as well as the complementarities of the assay primers and probes. Consequently, even with closely related/identical target sequences, strains/isolates tested at similar concentrations as quantified by $\mathrm { E I D } _ { 5 0 }$ , may contain quite different concentrations of target nucleic acid as measured by PCR. Therefore, LoD concentrations established for the JBAIDS PCR assays may not reflect significant differences in sensitivity of detection, rather they reflect differences in starting concentration of target nucleic acid. This was the case for the two representative seasonal Influenza A/H1 strains: the A/New Caledonia/20/1999 isolate appears to contain 17 to 45 times more target nucleic acid (as evidenced by a 4.1 to $5 . 5 \mathrm { C p }$ shift, assuming amplification efficiency is 2 ) per $\mathrm { E I D } _ { 5 0 }$ than the A/Hawaii/15/2001 isolate.

# e. Analytical Reactivity:

For inclusivity testing, sNPS samples were spiked with various strains of influenza target viruses within the specific Influenza A subtyping genera prior to testing with the JBAIDS Influenza A Subtyping Kit. Viruses were spiked at concentrations estimated to be near the LoD of each analyte. Due to differences in quantification methods between primary strains (quantified in $\mathrm { E I D } _ { 5 0 } / \mathrm { m L }$ ) and analytical reactivity panel strains (mostly quantified in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ), concentrations spanning the primary strain LoD $( 0 . 1 \times , 1 \times , 1 0 { \times } \mathrm { L o D }$ , assuming $1 \mathrm { E I D } _ { 5 0 }$ was equivalent to $1 \mathrm { T C I D } _ { 5 0 }$ ) were tested. If a sample containing a particular strain was positive at any of the concentrations, no further testing of that strain was required. If a strain was not detected at any of these concentrations, the strain was retested in 10-fold higher increments until a positive result was obtained or until the maximum concentration possible for that strain stock had been tested.

The following 3 tables provide a complete listing of the Influenza A strains that were used to assess the analytical reactivity or inclusivity of the test system. The 8 Influenza A H1N1 strains tested represents human isolates collected in United States, Australia, Europe, Asia, and the Solomon Islands between 1933 and 2007. The 10 Influenza A H3N2 strains tested represents human isolates collected in South America, North America, Europe and Asia between 1968 and 2009. The 11 Influenza A (swine lineage) $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ strains are representative of human isolates collected throughout the United States in 2009.

Influenza A H1N1 Strains Tested by the Flu A H1 Assay   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Lowest ConcentrationDetected</td><td rowspan=1 colspan=1>Cp</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>500000 TCID50/mL</td><td rowspan=1 colspan=1>37.22</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>37.56</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43</td><td rowspan=1 colspan=1>500 TCID50/mL</td><td rowspan=1 colspan=1>42.50a</td></tr><tr><td rowspan=1 colspan=1>A1/FM/1/47</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>36.59</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>5000 TCID50/mL</td><td rowspan=1 colspan=1>36.37</td></tr><tr><td rowspan=1 colspan=1>A1/Denver/1/57</td><td rowspan=1 colspan=1>NDb</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/3/2006</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>36.32</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>42.50a</td></tr></table>

a Cp calls of ${ \dot { \bf \gamma } } { > } 4 0 . 0 0$ cycles are assigned a final $\mathrm { C p }$ value of 42.50 cycles. b ND stands for “not detected”.

Influenza A H3N2 Strains Tested by the Flu A H3 Assay   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Lowest ConcentrationDetected</td><td rowspan=1 colspan=1>Cp</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>NDa</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>NDa</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>38.15</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>36.70</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>38.56</td></tr><tr><td rowspan=1 colspan=1>A/Taiwan/760/2007</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>35.99</td></tr><tr><td rowspan=1 colspan=1>A/Uruguay/716/2007</td><td rowspan=1 colspan=1>0.5 EID50/mL</td><td rowspan=1 colspan=1>37.88</td></tr><tr><td rowspan=1 colspan=1>A/Perth/16/09</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>35.84</td></tr><tr><td rowspan=1 colspan=1>A/Alice</td><td rowspan=1 colspan=1>0.5 TCID50/mL</td><td rowspan=1 colspan=1>39.65</td></tr><tr><td rowspan=1 colspan=1>A/MRC-2 recomb</td><td rowspan=1 colspan=1>NDa</td><td rowspan=1 colspan=1>-</td></tr></table>

a ND stands for “not detected”.

2009 H1N1 Influenza (swine lineage) Strains Tested by the Flu A H1 2009 and Flu A Sw Assays   

<table><tr><td rowspan=2 colspan=1>Isolate</td><td rowspan=2 colspan=1>Lowest ConcentrationDetected</td><td rowspan=1 colspan=2>Cp</td></tr><tr><td rowspan=1 colspan=1>Flu A H12009</td><td rowspan=1 colspan=1>Flu A Sw</td></tr><tr><td rowspan=1 colspan=1>A/California/4/2009</td><td rowspan=1 colspan=1>1500 TCID50/mL</td><td rowspan=1 colspan=1>35.71</td><td rowspan=1 colspan=1>32.72</td></tr><tr><td rowspan=1 colspan=1>A/California/8/2009</td><td rowspan=1 colspan=1>150 EID50/mL</td><td rowspan=1 colspan=1>35.77</td><td rowspan=1 colspan=1>32.06</td></tr><tr><td rowspan=1 colspan=1>A/England/195/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>34.82</td><td rowspan=1 colspan=1>31.28</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>150 EID50/mL</td><td rowspan=1 colspan=1>38.03</td><td rowspan=1 colspan=1>32.76</td></tr><tr><td rowspan=1 colspan=1>A/North Carolina/18/2009</td><td rowspan=1 colspan=1>1500 TCID50/mL</td><td rowspan=1 colspan=1>33.90</td><td rowspan=1 colspan=1>31.97</td></tr><tr><td rowspan=1 colspan=1>A/South Carolina/18/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>33.86</td><td rowspan=1 colspan=1>30.85</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/01/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>30.29</td><td rowspan=1 colspan=1>28.01</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/02/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>27.86</td><td rowspan=1 colspan=1>25.79</td></tr><tr><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>31.20</td><td rowspan=1 colspan=1>28.59</td></tr><tr><td rowspan=1 colspan=1>A/Texas/48/2009</td><td rowspan=1 colspan=1>1500 TCID50/mL</td><td rowspan=1 colspan=1>36.84</td><td rowspan=1 colspan=1>33.04</td></tr><tr><td rowspan=1 colspan=1>A/Washington/29/2009</td><td rowspan=1 colspan=1>150 TCID50/mL</td><td rowspan=1 colspan=1>37.22</td><td rowspan=1 colspan=1>32.78</td></tr></table>

# JBAIDS Influenza A/H1 assay results

The JBAIDS Influenza A/H1 assay accurately identified 7/8 individual Influenza A H1N1 isolates. However, there was high variability in concentrations of virus that was detected (ranging from $5 { - } 5 0 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ). The Flu A H1 assay was unable to detect the A1/Denver/1/57 strain at a final concentration of $5 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (higher concentrations could not be tested due to low stock concentration).

Sequence alignments of isolates evaluated (or similar isolates) indicated that there were 0-4 mismatches per primer or probe. Older strains for the virus (prior to 1957) tended to have more mismatches than the more contemporary isolates evaluated. A/Solomon Islands/3/2006 and A/Brisbane/59/07 isolates had zero to one mismatch combined in primers and probe sequences; the Brisbane isolate had 1 mismatch in the forward primer region. Sequence alignments for all available strains isolated between 2006-2011 demonstrated 0-1 mismatches per primer or probe sequence reflecting that this assay was designed to detect contemporary virus strains. Sequence alignment of A/PR/8/34, which was detected at $5 0 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , has a mismatch within the first few bases of the $3 ^ { \circ }$ end of both the forward and reverse primers. Exact sequences for A/Mal/302/54 and A1/Denver/1/57 were not available. However, sequences from isolates collected in the same geographical region and in the same year were aligned. For both isolates, the sequences had 3 mismatches in both the forward and reverse primers and two consecutive mismatches in the probe. These mismatches may contribute to the decrease in sensitivity of the Flu A H1 assay. Another possible cause for the variation is differences in the quantification methods. LoD concentration differences established in $\mathrm { E I D } _ { 5 0 }$ or $\mathrm { T C I D } _ { 5 0 }$ can vary dramatically between organism stocks, but may not actually reflect significant differences in target nucleic acid concentration.

# JBAIDS Influenza A/H3 assay results

The JBAIDS Influenza A/H3 assay accurately identified 7/10 individual Influenza A H3N2 isolates with a range of virus concentrations from 0.5 to $5 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . The following isolates were not detected even at the maximum concentration allowed based upon stock concentration: A/Aichi/2/68 at $1 1 4 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , A/Hong Kong/8/68 at $1 3 7 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , and A/MRC-2 recomb at $7 , 3 5 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

Sequence alignments of isolates evaluated (or similar isolates) indicated that there were 0-1 mismatches per primer and 1-5 mismatches with the probe. More contemporary isolates evaluated (strains isolated from 2007-2009) had significantly fewer mismatches in the probe region. In addition, viruses isolated prior to 1975 also had a mismatch within the first 3 bases of the reverse primer binding region. In general, contemporary strains (strains isolated between 2006-2011) have 0-1 mismatches per primer or probe sequence.

Of the isolates with observed decreased sensitivity or missed detection, sequences that were available were aligned with the PCR primers and probe for the Flu A H3 assay to determine whether sequence mismatches may be contributing to the potentially reduced sensitivity of the JBAIDS influenza A/H3 assay. In addition to the mismatch in the $3 ^ { \circ }$ end of the reverse primer, both the A/Aichi/2/68 and A/Hong Kong/8/68 isolates had five mismatches in the probe sequence, suggesting that the observed decreased sensitivity for these two isolates may be attributed to the high number of mismatches under the probe. The sequence for A/MRC-2 recomb was not available.

JBAIDS 2009 H1N1 Influenza (Flu A Sw and Flu A H1 2009) assays results

The JBAIDS 2009 H1N1 Influenza (Flu A Sw and Flu A H1 2009) assays accurately identified 11/11 individual 2009 H1N1 Influenza (swine lineage) isolates tested with a range of virus concentration from 150 to $1 , 5 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . Variation in the lowest concentration detected may be due to differences in relative quantification methods as well as virus strain sequence variation.

The analytical inclusivity data for the 4 Influenza A subtyping assays combined with in silico analysis indicated that the assays detect contemporary strains of human Influenza A viruses with acceptable analytical sensitivity.

f. Analytical Specificity/Cross-reactivity Evaluation:

Analytical Specificity (Exclusivity) testing involved the following organisms: 1) Nontarget influenza types (those not specific to the type assay); 2) Non-influenza viruses; and 3) Other closely related organisms that colonize the upper respiratory tract, cause respiratory symptoms, are common skin flora or laboratory contaminants, or are microorganisms for which much of the population may have been infected.

Each organism was spiked at a high level (when possible, viruses were spiked at $1 0 ^ { 5 } \times$ provided stock units, and other organisms were spiked at $1 0 ^ { 6 } \times$ provided stock units; otherwise organisms were spiked at the highest allowable concentration) into a sNPS sample prior to extraction using the Roche MagNA Pure Nucleic Acid Isolation Kit I and testing with the JBAIDS Influenza A & B Detection Kit.

The following 2 tables provide the exclusivity panels that were used to evaluate the test system analytical specificity or exclusivity:

Non-Target Influenza Viruses Evaluated for Cross reactivity with the Influenza A Subtyping Assays   

<table><tr><td rowspan=1 colspan=2>Type/Subtype</td><td rowspan=1 colspan=1>Isolate</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>SubtypingAssays</td></tr><tr><td rowspan=16 colspan=1>V nzunaA</td><td rowspan=1 colspan=1>H2N2 (Avian)</td><td rowspan=1 colspan=1>A/chicken/Pennsylvania/298101-4/2004</td><td rowspan=1 colspan=1>3.16E+07 TCID50/mL</td><td rowspan=16 colspan=1>Flu A H1Flu A H3Flu A SwFlu A H1 2009</td></tr><tr><td rowspan=1 colspan=1>H3N8 (Avian)</td><td rowspan=1 colspan=1>A/MAL/ALB/16/87</td><td rowspan=1 colspan=1>1.72E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H4N8 (Avian)</td><td rowspan=1 colspan=1>A/chicken/Alabama/1975</td><td rowspan=1 colspan=1>1.00E+08 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>H5N1 (Avian-HumanRecombinant)</td><td rowspan=1 colspan=1>A/Vietnam/1203/2004(H5N1)-PR8</td><td rowspan=1 colspan=1>3.16E+07 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>H5N1 (Avian)</td><td rowspan=1 colspan=1>A/DK/PA/4560069-9/06</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H7N3 (Avian)</td><td rowspan=1 colspan=1>A/TY/UT/24721-10/95</td><td rowspan=1 colspan=1>3.06E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H6N2 (Avian)</td><td rowspan=1 colspan=1>A/Chicken/CA/32213-1/2000</td><td rowspan=1 colspan=1>1.26E+07 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>H9N2 (Avian)</td><td rowspan=1 colspan=1>A/Turkey/Wisconsin/1966</td><td rowspan=1 colspan=1>5.60E+07 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N8 (Canine)</td><td rowspan=1 colspan=1>A/canine/Florida/43/2004</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N8 (Equine)</td><td rowspan=1 colspan=1>A/Equine/Ohio/01/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 (Swine)</td><td rowspan=1 colspan=1>A/swine/Wisconsin/125/1997</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 (Swine)</td><td rowspan=1 colspan=1>A/SW/GB/19582/92</td><td rowspan=1 colspan=1>5.64E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H3N2 (Swine)</td><td rowspan=1 colspan=1>A/SW/IA/1/99</td><td rowspan=1 colspan=1>1.41E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 (Human of swinelineage)</td><td rowspan=1 colspan=1>A/Maryland/12/1991</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H1N1 (Human of swinelineage)</td><td rowspan=1 colspan=1>A/Iowa/1/2006</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>H7N2 (Human)</td><td rowspan=1 colspan=1>A/New York/107/2003</td><td rowspan=1 colspan=1>30 μl of an unknownconcentration into 1mL</td></tr><tr><td rowspan=3 colspan=2>Influenza B</td><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td><td rowspan=3 colspan=1>Flu A H1Flu A H3Elu C</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr></table>

<table><tr><td rowspan=6 colspan=1></td><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td><td rowspan=6 colspan=1></td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>4.54E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>5.09E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>1.50E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=1>3.14E+04 TCID50/mL</td></tr><tr><td rowspan=8 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td><td rowspan=8 colspan=1>Flu A H3Flu A SwFlu A H1 2009</td></tr><tr><td rowspan=1 colspan=1>A1/FM/1/47</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A1/Denver/1/57</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/3/2006</td><td rowspan=1 colspan=1>1.25E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54</td><td rowspan=1 colspan=1>1.25E+04 TCID50/mL</td></tr><tr><td rowspan=7 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>5.10E+03 TCID50/mL</td><td rowspan=7 colspan=1>Flu A H1Flu A SwFlu A H1 2009</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Alice (VR-776)</td><td rowspan=1 colspan=1>4.24E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/MRC-2 recomb (VR-777)</td><td rowspan=1 colspan=1>7.36E+03TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>7.36E+03TCID50/mL</td></tr><tr><td rowspan=11 colspan=1>2009 H1N1 Influenza (swinelineage)</td><td rowspan=1 colspan=1>Swine NY/02/2009</td><td rowspan=1 colspan=1>1.25E+04 TCID50/mL</td><td rowspan=11 colspan=1>Flu A H1Flu A H3</td></tr><tr><td rowspan=1 colspan=1>Swine NY/03/2009</td><td rowspan=1 colspan=1>7.36E+03 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Swine NY/01/2009</td><td rowspan=1 colspan=1>3.78E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>1.00E+05 EID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/California/8/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/California/04/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Texas/48/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Washington/29/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/South Carolina/18/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/England/195/2009</td><td rowspan=1 colspan=1>4.74E+04 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/North Carolina/39/2009</td><td rowspan=1 colspan=1>1.00E+05 TCID50/mL</td></tr></table>

Exclusivity Panel: Non-influenza Viruses and Organisms   

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">Bacteria/Fungi</td><td colspan="1" rowspan="1">Concentration Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">4.20E+07 copies/mL</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">7.35E+03 TCID50/mL</td><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">6.57E+04 TCID50/mL</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">5.10E+03 TCID50/mL</td><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">7.80E+04 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">1.50E+04 TCID50/mL</td><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">7.35E+03 TCID50/mL</td><td colspan="1" rowspan="1">Mycoplasma pneumonia</td><td colspan="1" rowspan="1">1.69E+05 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">5.10E+03 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Measles Virus (Rubeola)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">4.53E+04 TCID50/mL</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">1.50E+04 TCID50/mL</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">1.25E+04 TCID50/mL</td><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">7.59E+05 CFU/mL</td></tr></table>

For the non-target influenza virus exclusivity panel, the expected negative results were obtained for all of the assays with the following exceptions. Three Influenza A viruses were detected by the Flu A Sw assay: A/Maryland/12/1991, A/Iowa/1/2006, A/swine/Wisconsin/125/1997. These results were not unexpected since these isolates were all of subtype H1N1 with the first two isolated from humans but containing swine lineage sequences and the third isolated from swine. BLAST evaluation of the Flu A Sw assay primers and probes against recent influenza sequences (from 2009-2011) indicated that of the top 500 sequence alignments (from 2009-2011), 8 percent of the isolates that aligned within an E value range of 7e-06 to 3e-05 were of isolates from swine.

In addition, Influenza A H3N2 virus isolate A/SW/IA/1/99 (isolated from swine) was detected by both the Flu A H3 and Flu A Sw assays. Detection of swine Influenza A/H3 viruses by the Flu A H3 assay is not unexpected as the hemagglutinin sequences for swine and human isolates are very similar.

All 36 non-influenza organisms, tested at high concentrations, (bacteria or fungi spiked at $1 0 ^ { 6 } \mathrm { C F U / m L }$ or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ and viruses spiked at $1 0 ^ { 3 } - 1 0 ^ { 5 }$ copies/mL or $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) gave the expected negative results for all Influenza A subtyping assays. This evaluation indicated that the primers and probes in the JBAIDS Influenza Subtyping assays do not cross-react with the non-influenza exclusivity organisms tested.

# g. Assay cut-off:

Each JBAIDS test is analyzed with the data analysis software module. The module is called Detector. The module determines the outcome of the tests conducted on the samples. The module first analyzes the data from each capillary independently. Then the replicates of the samples are analyzed together (if applicable). Finally the module interprets the outcome of these analyses for the sample and its controls.

The analysis module algorithm consists of two basic parts. The first identifies samples that are obviously positive or negative (Obvious Tests), and second that computes a refined estimate of the samples status when the sample is not called an obvious positive or obvious negative (Expert Test).The algorithms are based on an Expert System approach to determining positive and negative samples, but the approach is not learning after release, and is deterministic, meaning that every sample will have the same outcome every time the algorithm evaluates the sample. The purpose of Obvious Tests is to call those samples that are clearly positive or negative. In this way, only non-obvious samples will need to be processed by the more computationally involved detector algorithms. Obvious tests produce internal scores and compare them against fixed thresholds to return a positive/negative/uncertain call. Expert Tests algorithm is based on mathematically modeling the expected shapes of the amplification curves of positive and negative samples. The model itself is based on tests for nine distinct characteristics in the amplification curves and it assigns nine test scores to each curve. After the tests are scored, the amplification curve is scored as a weighted sum of the tests.

Thresholds for the Obvious Tests and weights in the Expert Tests for Detector were determined using training data sets and numerical optimization to pick parameters that ensure Detector minimizes the number of error observed in real-world data.

Validation data from JBAIDS Customer Validation Testing, and Dugway Operational Assessment were gathered and evaluated with Detector to validate the analysis module algorithm. These data contain samples from five separate data sets: typical data, atypical data, negative dominated, RNA data and Dugway field data. The data were gathered from varied sources, including a variety of assays, instruments and users, to obtain examples of all possible JBAIDS data. There are 28,467 samples in the data sets. $9 9 . 6 \%$ of the samples were correctly called by the Detector. $0 . 1 9 \%$ of the samples were called “Uncertain” and $0 . 2 1 \%$ of the samples were incorrectly called.

# h. Interfering Substances:

An interfering substances study was carried out to examine whether a panel of endogenous and exogenous potential RT-PCR inhibitors and technique-specific substances (substances that could be introduced into the PCR reaction as contaminants during sample purification or during reaction setup) affect the performance of the JBAIDS Influenza Subtyping Kit. The concentration of substances tested represented a relevant concentration in accordance with CLSI EP7-A2 Interference Testing in Clinical Chemistry Approved Guideline, or for test substances not listed in the CLSI document, concentrations were determined by evaluating the scientific literature or references from other IVD package inserts. For each endogenous, exogenous and technique specific test substance, sNPW specimens containing a representative Influenza A/H1, Influenza A/H3, or 2009 H1N1 Influenza virus (A/New Caledonia/20/1999, A/New York/55/2004, or A/New York/18/2009) at a concentration equivalent to $5 { \times } \mathrm { L o D }$ were spiked with the appropriate amount of test substance.

Note: sNPW specimens spiked with a representative seasonal Influenza A H3N2 virus strain (A/New York/55/2004) were also co-spiked with Influenza B virus (B/Ohio/1/2005) for more efficient interfering substances testing of the JBAIDS Influenza A & B Detection Kit. Refer to the $^ { \circ \mathfrak { c } } \mathrm { C o }$ -spiking Specimen Validation” section for details demonstrating that combining these two viruses in a sample, even at high concentrations, had no detrimental effect on the detection of either virus.

In each round of purifications, an influenza-containing sNPW sample that had not been spiked with the interfering substance was also included. This sample served as a control to which the other specimens were compared. All specimens, except those containing Roche MagNA Pure Compact Nucleic Acid Isolation Kit I technique specific substances (i.e., MagNA Pure buffers), were purified using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact System. Each purified sample was tested with the Influenza A Subtyping assays (Flu A H1, Flu A H3, or Flu A H1 and Flu A Sw) corresponding to the virus with which it

was spiked, and also tested with the Flu SC assay. The Roche MagNA Pure Compact Nucleic Acid Isolation Kit I technique-specific substances that could be introduced during sample purification (i.e., MagNA Pure buffers) were added to the purified samples immediately prior to influenza assay reaction setup. Substances were considered to be potential inhibitors if the results for the tested assays were negative.

Note: Only sNPW samples purified using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact System were evaluated. NPS samples purified with the IT 1-2-3 Platinum Path Sample Purification Kit were previously evaluated with the FDA cleared JBAIDS Influenza A/H5 (Asian lineage) Detection System (k100287). The Platinum Path kit was shown to remove both endogenous and exogenous substances from NPS and throat swab samples. In addition, the wash and elution buffers contained in the IT 1-2-3 Platinum Path Sample Purification Kit (technique-specific substances) did not interfere with detection of the JBAIDS Influenza A/H5 assays.

List of Potentially Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Endogenous Substances</td><td rowspan=1 colspan=3>Exogenous Substances</td></tr><tr><td rowspan=1 colspan=1>Blood (with Na Citrate)Mucin (bovine submaxillarygland, type I-S)Human Genomic DNA</td><td rowspan=1 colspan=2>TobramycinMupirocinFluMist® intranasal vaccineDexamethasoneFluticasone propionateSnuff (nasal tobacco)Zicam Nasal Gel</td><td rowspan=1 colspan=1>Nasal Decongestant Spray (Phenylephrine HCl 1.0%)Nasal Decongestant spray (Oxymetazoline HCl 0.05%)Saline nasal spray with preservatives (0.65% NaClPhenylcarbinol Benzalkonium Chloride)Analgesic Ointment(VicksVaoRub) Petroleum JellyPetroleum and Glycerin topical gel (Chlorseptic®)</td></tr><tr><td rowspan=1 colspan=1>Solvents Used</td><td rowspan=1 colspan=3>Technique-specific Substances</td></tr><tr><td rowspan=1 colspan=1>WaterAcetoneDMSO</td><td rowspan=1 colspan=1>Disinfecting WipesEthanolRNAseOutMagNA Pure Buffers fromcartridge wells #1-6</td><td rowspan=1 colspan=2>Swabs•   Copan 168C (rayon/twisted aluminum shaft)Copan 503CS01 (flocked nylon/plastic shaft)Copan 175KS01 (polyester/aluminum shaft)Copan 502CS01 (flocked nylon/plastic shaft)Millipore 519CS01M (flocked nylon/plastic shaft)Viral Transport MediaRemel M5Remel M6</td></tr></table>

1 These are solvents used to dissolve potentially interfering substances in preparation for testing.

Test Concentrations for Endogenous Substances   

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EN1</td><td rowspan=1 colspan=1>Blood(with Na Citrate)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>EN2</td><td rowspan=1 colspan=1>Mucin(bovine submaxillary gland, type I-S)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>Water</td></tr></table>

Test Concentrations for Exogenous Substances   

<table><tr><td colspan="1" rowspan="1">IS#</td><td colspan="1" rowspan="1">Test Substance</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Solvent</td></tr><tr><td colspan="1" rowspan="1">EX1</td><td colspan="1" rowspan="1">Tobramycin(systemic antibiotic)</td><td colspan="1" rowspan="1">0.6 mg/mL</td><td colspan="1" rowspan="1">Water</td></tr><tr><td colspan="1" rowspan="1">EX2</td><td colspan="1" rowspan="1">Mupirocin(active ingredient in anti-bacterial ointment)</td><td colspan="1" rowspan="1">2% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX3Blank</td><td colspan="1" rowspan="1">FluMist intranasal vaccine(2009-2010 Formula)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPW Blank</td></tr><tr><td colspan="1" rowspan="1">EX4</td><td colspan="1" rowspan="1">Saline Nasal Spray with Preservatives0.65% NaClPhenylcarbinol Benzalkonium Chloride</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX5</td><td colspan="1" rowspan="1">Nasal Decongestant SprayOxymetazoline HCl 0.05%(also contains Benzalkonium chloride, menthol, eucalyptol,camphor, benzyl alcohol and phosphate buffers)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX6</td><td colspan="1" rowspan="1">Analgesic ointment (Vicks®VapoRub®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX7</td><td colspan="1" rowspan="1">Petroleum Jelly (Vaseline®)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX8</td><td colspan="1" rowspan="1">Zicam® Nasal Gel</td><td colspan="1" rowspan="1">5% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX9</td><td colspan="1" rowspan="1">Snuff (nasal tobacco)</td><td colspan="1" rowspan="1">1% w/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX10</td><td colspan="1" rowspan="1">Dexamethasone</td><td colspan="1" rowspan="1">0.084 μg/mL</td><td colspan="1" rowspan="1">Acetone</td></tr><tr><td colspan="1" rowspan="1">EX11</td><td colspan="1" rowspan="1">Fluticasone propionate</td><td colspan="1" rowspan="1">0.22 μg/mL</td><td colspan="1" rowspan="1">DMSO</td></tr><tr><td colspan="1" rowspan="1">EX12</td><td colspan="1" rowspan="1">Nasal Decongestant SprayPhenylephrine HCl 1.0%(also contains benzalkonium chloride, anhydrous citric acid,purified water, sodium chloride, and sodium citrate)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr><tr><td colspan="1" rowspan="1">EX13</td><td colspan="1" rowspan="1">Petroleum and Glycerin topical gel(Chlorseptic)</td><td colspan="1" rowspan="1">1% v/v</td><td colspan="1" rowspan="1">sNPWsample</td></tr></table>

Test Concentrations for Technique-Specific Substances   

<table><tr><td colspan="1" rowspan="1">IS#</td><td colspan="2" rowspan="1">Test Substance</td><td colspan="1" rowspan="1">TestConcentration</td></tr><tr><td colspan="1" rowspan="1">TS1</td><td colspan="2" rowspan="1">Disinfecting Wipes</td><td colspan="1" rowspan="1">1/4 - 1/2 in2</td></tr><tr><td colspan="1" rowspan="1">TS2</td><td colspan="2" rowspan="1">Ethanol</td><td colspan="1" rowspan="1">7%</td></tr><tr><td colspan="1" rowspan="1">TS3</td><td colspan="2" rowspan="1">RNaseOut</td><td colspan="1" rowspan="1">1 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS4</td><td colspan="1" rowspan="4">MagNA PureReagentCartridgeContents</td><td colspan="1" rowspan="1">Well 1: Proteinase K solution</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS5</td><td colspan="1" rowspan="1">Well 2: Lysis Buffer solution</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS6</td><td colspan="1" rowspan="1">Well 3: Magnetic Glass Particles</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS7</td><td colspan="1" rowspan="1">Wells 4,5: Wash Buffer I</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS8</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Well 6: Wash Buffer II</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS9</td><td colspan="1" rowspan="1">Well 7: Wash Buffer III</td><td colspan="1" rowspan="1">10 % v/v</td></tr><tr><td colspan="1" rowspan="1">TS10</td><td colspan="1" rowspan="5">Swabs</td><td colspan="1" rowspan="1">Copan 168C(rayon/twisted aluminum shaft)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">TS11</td><td colspan="1" rowspan="1">Copan 503CS01(flocked nylon/plastic shaft)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">TS12</td><td colspan="1" rowspan="1">Copan 175KS01(polyester/aluminum shaft)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">TS13</td><td colspan="1" rowspan="1">Millipore 519CS01M(flocked nylon/plastic shaft)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">TS14</td><td colspan="1" rowspan="1">Copan 502CS01(flocked nylon/plastic shaft)</td><td colspan="1" rowspan="1">1 swab</td></tr><tr><td colspan="1" rowspan="1">TS15</td><td colspan="1" rowspan="3">ViralTransportMedium</td><td colspan="1" rowspan="1">Remel M5</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">TS16</td><td colspan="1" rowspan="1">Remel M6</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">TS17</td><td colspan="1" rowspan="1">Copan UTM</td><td colspan="1" rowspan="1">100%</td></tr></table>

# Test Concentrations for Human Genomic DNA

<table><tr><td rowspan=1 colspan=1>IS#</td><td rowspan=1 colspan=3>Test Substance</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>DNA1</td><td rowspan=2 colspan=3>Human genomic DNA</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>20 ng/μL (500 ng)</td><td rowspan=1 colspan=1>sNPW sample</td></tr><tr><td rowspan=1 colspan=1>DNA2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>200 ng/μL</td><td rowspan=1 colspan=1>2 ng/μL (50 ng)</td><td rowspan=1 colspan=1>water</td></tr><tr><td rowspan=1 colspan=1>DNA3</td><td rowspan=2 colspan=3>263 ng/μL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20 ng/μL</td><td rowspan=1 colspan=1>0.2 ng/μL (5 ng)</td></tr><tr><td rowspan=1 colspan=1>DNA4</td><td rowspan=1 colspan=1>2 ng/μL</td><td rowspan=1 colspan=1>0.02 ng/μL (0.5 ng)</td><td rowspan=1 colspan=1>water</td></tr></table>

Of all of the endogenous and exogenous substances tested, only snuff (nasal tobacco) has the potential to cause inaccurate negative test results due to inhibition. Of the technique- specific substances tested, improper introduction of contents from four of the MagNA Pure reagent cartridge wells to purified samples have the potential to cause inaccurate negative test results due to inhibition:

MagNA Pure Proteinase K Solution (well 1) MagNA Pure Lysis Solution (well 2) MagNA Pure Magnetic Glass Particles (well 3) MagNA Pure Wash Buffer I (well 4)

These substances, along with ethanol, bleach, and DNAZap™, which are known PCR inhibitors, are included in the package insert as potentially interfering substances.

In addition, the study also demonstrated that the JBAIDS Influenza A Subtyping assays react with the viral material contained in the 2009-2010 version of the FluMist intranasal influenza vaccine (MedImmune). When NPS and NPW specimens are collected from individuals who have recently received a dose of the nasal vaccine, the JBAIDS Influenza A Subtyping assays may react with the vaccine material to give inaccurate positive results.

Note: The material being tested (FluMist® Influenza Vaccine Live, Intranasal; Intranasal Spray, 2009-2010 Formula) was provided by the manufacturer (MedImmune) in individual sprayers, each containing a single $0 . 2 \mathrm { m L }$ dose. Each dose contained approximately 106.5-7.5 FFU (fluorescence focus units) each of three live, attenuated influenza virus reassortants including:

• A/South Dakota/6/2007 (H1N1) (A/Brisbane/59/2007-like) • A/Uruguay/716/2007 (H3N2) (A/Brisbane/10/2007-like) B/Brisbane/60/2008.

Non-viral components of the vaccine listed in the provided materials include: monosodium glutamate, gelatin, arginine, sucrose, gentamicin sulfate, monobasic potassium phosphate and dibasic potassium phosphate.

# i. Carry-Over Contamination:

An analytical study was carried out to evaluate the frequency of carryover in the JBAIDS Influenza A Subtyping Kit when it is used as instructed by the product insert. Significant levels of carryover will result in false positive test results for negative samples that are contaminated by adjacent positive samples. For the JBAIDS Influenza A Subtyping Kit, carryover could occur during sample purification or reagent setup.

To assess the potential for carryover, sample sets consisting of negative and high positive samples were processed in an alternating pattern using the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I on the Roche MagNA Pure Compact system and then tested with the JBAIDS Influenza A Subtyping Kit.

Note: The potential for carryover in samples purified with the IT 1-2-3 Platinum Path Purification Kit was previously evaluated with JBAIDS Influenza A/H5 (Asian lineage) Detection Kit (k100287) and demonstrated that significant levels of carryover do not occur when samples are processed and tested according to the instructions in the package insert.

High positive sNPS samples were spiked with Influenza A H3N2 strain A/New York/55/2004 at approximately $2 0 0 0 \times$ LoD $( \sim 1 . 0 \times 1 0 ^ { 4 } \mathrm { E I D } _ { 5 0 } / \mathrm { m L } )$ . Five independent sample sets were tested for a total of 20 positive and 20 negative samples. Positive and negative samples were purified in an alternating pattern, and the PCR reactions for the positive and negative samples were set up in the order in which the samples were processed. The frequency of false positive and uncertain results for the negative samples was evaluated using the Flu A H3 assay as the representative assay from the JBAIDS Influenza A Subtyping Kit.

All positive samples resulted in early Flu A H3 assay $\mathrm { C p }$ values $( < 2 3 . 0 0 $ cycles) and robust $\mathrm { F } _ { \mathrm { m a x } }$ values $( > 4 9 . 0 0 )$ . All negative samples produced negative results, with no detectable $\mathrm { C p }$ values. No carryover was observed in this evaluation. The test results are shown in the following table:

Flu A H3 Assay Carryover Results for sNPS Samples Purified with the Magna Pure Compact Nucleic Acid Isolation Kit I Purification Kit   

<table><tr><td rowspan=2 colspan=1>TrialNumber</td><td rowspan=2 colspan=1>Number ofPositiveSamples</td><td rowspan=1 colspan=2>Flu A H3 Sample Results forPositive Samples</td><td rowspan=1 colspan=1>Number of</td><td rowspan=1 colspan=2>Flu A H3 Sample Resultsfor Negative Samples</td></tr><tr><td rowspan=1 colspan=1>Mean Cp(SD)</td><td rowspan=1 colspan=1>Mean Fmax(SD)</td><td rowspan=1 colspan=1>NegativeSamples</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.60(0.10)</td><td rowspan=1 colspan=1>50.78(3.61)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.74(0.18)</td><td rowspan=1 colspan=1>50.20(2.42)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.17(0.35)</td><td rowspan=1 colspan=1>51.79(2.24)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.22(0.40)</td><td rowspan=1 colspan=1>56.45(3.79)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>22.41(0.22)</td><td rowspan=1 colspan=1>49.67(2.62)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>22.43(0.33)</td><td rowspan=1 colspan=1>51.78(3.66)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>20</td></tr></table>

Overall, sample purification and reaction setup in this carryover study resulted in no false positive results when samples were processed in accordance with the purification kit and reaction setup directions. PCR is a very sensitive technique and false positives caused by contamination are a recognized risk. Users will be reminded that failure to follow protocol and improper technique may result in false positive results.

# j. Validation of Simulated Sample Matrices

Nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS) specimens are the two sample matrices evaluated with the JBAIDS Influenza A Subtyping Kit and used for most of the non-clinical evaluations. However, given the need for large numbers of individual samples without influenza virus, simulated NPS (sNPS) and simulated NPW (sNPW) samples were developed and validated to prevent inaccurate test results arising from the presence of influenza viruses in human derived nasopharyngeal specimens. Simulated samples were comprised of human HeLa cells diluted to concentrations in viral transport medium (VTM) that produced similar Cp values as the Flu SC assay produced when testing human derived NPS and NPW samples.

HeLa cell concentrations of $1 0 ^ { 3 . 2 } \mathrm { c e l l s / m L }$ and $1 0 ^ { 4 . 3 }$ cells/mL for sNPW and sNPS sample matrix types, respectively, resulted in similar $\mathrm { C p }$ values for the Flu SC assay as human derived NPW and NPS specimens. To confirm that the simulated sample matrix types did not have any advantageous or detrimental effects on the performance of the system, multiple aliquots of simulated and human derived samples were spiked at the LoD with a representative seasonal influenza A virus (H3N2 strain A/New York/55/2004) and tested with the applicable influenza and control assays (Flu A, Flu A H3, and Flu SC). Simulated and human derived samples exhibited equivalent rates of detection. Mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values were also comparable between the human derived and simulated samples.

For efficient use of resources during non-clinical testing of the JBAIDS Influenza A & B Detection and Influenza A Subtyping Kits, Influenza A H3N2 and Influenza B viruses were co-spiked in some studies. Specifically, co-spiked samples were used in the following analytic evaluations: Reproducibility, Sample Transport and Storage, and Interfering Substances.

An analytical study was carried out to validate the use of specimens co-spiked with Influenza A H3N2 and Influenza B viruses. This study was performed by evaluating single-spiked and co-spiked samples containing Influenza A H3N2 A/New York/55/2004 and/or Influenza B B/Ohio/01/2005. NPW specimens purified with the Platinum Path kit were evaluated in this study because the Cp values for the Flu A assay in NPW samples, purified with Platinum Path kit, were greater than Cp values for the other sample type/purification kit combinations. The first part of this study evaluated freshly collected NPW samples spiked with an influenza A H3N2 virus at LoD only (single spike) or additionally spiked with influenza B virus at $5 0 0 \times \mathrm { L o D }$ (cospiked). The second part of this study evaluated the opposite, NPW samples spiked with influenza B virus at $1 { \times } \mathrm { L o D }$ (single spike) or further spiked with influenza A H3N2 at ${ 5 0 0 } \mathrm { { \times } L o D }$ (co-spiked).

For each part of the study, 60 single and 60 co-spiked samples were prepared and evaluated. In particular, pooled fresh NPW samples were spiked with virus at $1 { \times } \mathrm { L o D }$ . Half of this pool was co-spiked with the alternate virus at $5 0 0 \times \mathrm { L o D }$ , resulting in one single spiked pool and one co-spiked pool. Following spiking, the samples were purified using the Platinum Path kit and then tested with appropriate freeze-dried reagent test assays (Flu A, Flu B, and Flu A H3). A total of 240 samples were evaluated.

For the samples spiked with influenza B at $1 { \times } \mathrm { L o D }$ , all single (60) and co-spiked (60) samples were detected with the Flu B assay. In addition, the influenza A H3N2 virus at $5 0 0 \times \mathrm { L o D }$ was detected in $6 0 / 6 0$ samples by the Flu A and Flu A H3 assays. These results demonstrate that $100 \%$ detection was achieved for single (influenza B at $1 { \times } \mathrm { L o D } )$ and co- spiked (influenza B at $1 \times \mathrm { L o D } /$ influenza A H3N2 at ${ 5 0 0 } { \times } \mathrm { L o D } )$ ) samples. For the Flu A assay, samples spiked with influenza A H3N2 at $1 { \times } \mathrm { L o D }$ were detected in 59/60 single- spiked and 58/60 co-spiked samples, resulting in 98.3 and $9 6 . 7 \%$ detection rates, respectively. For the Flu A H3 assay the same samples were positive for $6 0 / 6 0$ single- spiked and 59/60 co-spiked samples, resulting in 100 and $9 8 . 3 \%$ detection rates, respectively. In addition, the influenza B virus at $5 0 0 \times$ LoD was detected in 60/60 samples by the Flu B assay.

Differences in mean $\mathrm { C p }$ values between single- and co-spiked samples did not vary by more than 0.19 cycles for the three assays, and mean $\mathrm { F } _ { \mathrm { m a x } }$ values for co-spiked samples were reduced by less than $5 \%$ from mean $\mathrm { F } _ { \mathrm { m a x } }$ values for single-spiked samples. The quantitative differences in mean $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values between single- and co-spiked samples were lower than the standard deviations for each assay and spike set.

The study results demonstrate that there is no significant quantitative or qualitative difference between specimens single spiked or co-spiked with influenza B and influenza A H3N2 viruses.

l. PCR followed by Bi-directional Sequencing Comparator Assays

The comparator assay for the JBAIDS Influenza A Subtyping Kit is PCR followed bi-directional sequencing of amplicons. Comparator assays for the JBAIDS Influenza A Subtyping Kit were designed by Idaho Technology and validated with Limit of Detection (LoD) and reactivity studies.

The research and development group designed the Influenza A/H1, Influenza A/H3, and the 2009 H1N1 Influenza assays such that they recognized different locations on the hemagglutinin gene(s) than targeted by the JBAIDS Influenza A Subtyping assays.

JBAIDS Influenza A Subtyping and Comparator Assay Gene Targets   

<table><tr><td rowspan=1 colspan=1>Test Panel</td><td rowspan=1 colspan=1>PurificationMethod</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>JBAIDS Target</td><td rowspan=1 colspan=1>ComparatorAssay Target</td><td rowspan=1 colspan=1>Chemistry</td></tr><tr><td rowspan=3 colspan=1>Subtyping</td><td rowspan=3 colspan=1>Qiagen QIAampViral RNA</td><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>SeasonalH1</td><td rowspan=1 colspan=1>Hemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hydrolysis probe</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>SeasonalH3</td><td rowspan=1 colspan=1>Hemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hydrolysis probe 1</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>2009H1N1</td><td rowspan=1 colspan=1>Nucleocapsidprotein (NP) andHemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hemagglutinin(HA) gene</td><td rowspan=1 colspan=1>Hydrolysis probe1</td></tr></table>

1Nested assays.

The LoD confirmation results for the comparator assays are listed in the following table:

Comparator Assays LoD Confirmation Results   

<table><tr><td colspan="1" rowspan="1">Comparator Assay</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">SampleType</td><td colspan="1" rowspan="1">LoDConcentration</td><td colspan="1" rowspan="1"># Pos/Total</td><td colspan="1" rowspan="1">% Positive</td></tr><tr><td colspan="1" rowspan="1">Big Flu H1 Assay</td><td colspan="1" rowspan="1">SeasonalInfluenza A/H1</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">50 EID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Big Flu H1 Assay</td><td colspan="1" rowspan="1">SeasonalInfluenza A/H1</td><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">50 EID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Big Flu H1-2009Assay</td><td colspan="1" rowspan="1">2009 H1N1 Influenza</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">1500 EID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Big Flu H1-2009Assay</td><td colspan="1" rowspan="1">2009 H1N1 Influenza</td><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">1500 EID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Big Flu H3 Assay</td><td colspan="1" rowspan="1">SeasonalInfluenza A/H3</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">5 EID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Big Flu H3 Assay</td><td colspan="1" rowspan="1">Seasonal influenza AH3N1</td><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">5EID50</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr></table>

The inclusivity and exclusivity validation results are listed in the following tables:

# Inclusivity Results

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Strain</td><td rowspan=1 colspan=7>Concentration(EI50/ml)</td><td rowspan=1 colspan=1>Big FluA H Assay</td><td rowspan=1 colspan=1>Big FluA H1-09 Assay</td><td rowspan=1 colspan=1>Big FluA 3 Assay</td></tr><tr><td rowspan=2 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=2>A/Brisbane/59/07</td><td rowspan=2 colspan=7>5</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/Solomon Islands/3/2006</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=14 colspan=1>2009 H1N1Innfluenza</td><td rowspan=1 colspan=2>Swine NY/02/2009</td><td rowspan=6 colspan=7>1500</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>Swine NY/03/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>Swine NY/01/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/California/8/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/California/04/2009</td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=3></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=2 colspan=2>A/Texas/48/2009</td><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=4></td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2></td><td rowspan=2 colspan=1></td><td></td></tr><tr><td rowspan=3 colspan=2>A/Washington/29/2009</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=4 colspan=7></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/South Carolina/18/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/England/195/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=2>A/North Carolina/39/2009</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=7 colspan=1>Influenza A H3</td><td rowspan=1 colspan=2>A/Port Chalmers/1/73</td><td rowspan=7 colspan=7>50</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/Victoria/3/75</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/Aichi/2/68</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/Hong Kong/8/68</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/Alice (VR-776)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/MRC-2 recomb (VR-777)</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=2>A/Brisbane/10/07</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr></table>

# Exclusivity Results

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration(EID50/ml)</td><td colspan="1" rowspan="1">Big FluA H1Assay</td><td colspan="1" rowspan="1">Big FluA H12009</td><td colspan="1" rowspan="1">Big FluA H3Assay</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Type 1 (Zeptometrix</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Clinical Sample (ProvLab)</td><td colspan="1" rowspan="1">1.40E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">ATCC VR-740</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">ATCC VR-759</td><td colspan="1" rowspan="1">2.19E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">NR-470 (BEI resources)</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">PCMC 6213</td><td colspan="1" rowspan="1">4.17E+10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">hMPV-16/IA10-2003 (Type A1)</td><td colspan="1" rowspan="1">2.45E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 1</td><td colspan="1" rowspan="1">Zeptometrix #0810014CF</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 2</td><td colspan="1" rowspan="1">Zeptometrix #0810015CF</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 3</td><td colspan="1" rowspan="1">Zeptometrix #0810016CF</td><td colspan="1" rowspan="1">6.61E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza virus 4</td><td colspan="1" rowspan="1">Zeptometrix #0810060CF</td><td colspan="1" rowspan="1">2.82E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">A (Zeptometrix #0810040ACF)</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169 ATCC VR-538</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.04E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Measles Virus (Rubeola)</td><td colspan="1" rowspan="1">Zeptometrix #0810025CF</td><td colspan="1" rowspan="1">1.26E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix #0810079CF</td><td colspan="1" rowspan="1">1.51E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">3.50E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix # 0801504</td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC 14779 CDC [NCTC</td><td colspan="1" rowspan="1">4.20E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">2.34E+10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenza</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">2.60E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td><td colspan="1" rowspan="1">1.75E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">Philadelphia Strain</td><td colspan="1" rowspan="1">2.63E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11- type strain</td><td colspan="1" rowspan="1">6.83E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">2.20E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumonia</td><td colspan="1" rowspan="1">M129 (Type 1)</td><td colspan="1" rowspan="1">5.63E+06</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongate</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.99E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">M1027-type strain</td><td colspan="1" rowspan="1">1.63E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">1.05E+10</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">8.40E+09</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">6.20E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumonia</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">5.54E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix # 0801512</td><td colspan="1" rowspan="1">7.57E+08</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">2.53E+07</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr></table>

# m. Fresh vs. Frozen Specimen Study

The JBAIDS Influenza A Subtyping Kit includes assays for the detection of Influenza A subtypes in NPS and NPW specimens collected from individuals exhibiting influenza-like-illness. A prospective clinical evaluation of this test kit was performed in order to establish the clinical sensitivity and specificity of the assays contained in the JBAIDS Influenza A Subtyping Kit. The prospective clinical evaluation was performed from December 2010 through April 2011 at five geographically diverse locations. During this study, sufficient positive and negative samples were obtained to establish the clinical sensitivity and specificity for detection of 2009 H1N1 Influenza and Influenza A/H3 viruses. However, seasonal Influenza A/H1 did not circulate during the trial period.

To account for the possibility that frozen archived samples might be required to supplement the prospective clinical trial data, an analytical study was performed to establish that the results obtained with frozen samples are equivalent to those obtained when testing freshly collected NPS or NPW samples.

Pools of freshly collected NPS and NPW specimens were spiked at $1 0 0 \times \mathrm { L o D }$ , $1 0 { \times } \mathrm { L o D }$ , $3 { \times } \mathrm { L o D }$ , and $1 { \times } \mathrm { L o D }$ with a representative Influenza $\mathrm { A } / \mathrm { H } 1$ virus strain (A/New Caledonia/20/1999). The spiked pools were then aliquoted. Samples that were to be tested after freezing were aliquoted from each pool and stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ for at least 24 hours. The remainder of each pool was aliquoted for immediate analysis. Testing of sample sets consisted of purifying individual aliquots with either the Platinum Path or the MagNA Pure kits and assaying with the Influenza A subtyping assays. The following table lists the number of samples tested at each virus concentration:

Virus Concentrations and Numbers of Fresh and Frozen Samples Evaluated with both Purification Methods   

<table><tr><td rowspan=1 colspan=1>Virus Spike Level</td><td rowspan=1 colspan=1>Influenza Virus Concentration(Total Fresh and Frozen Replicates)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>A/New Caledonia/20/1999</td></tr><tr><td rowspan=1 colspan=1>Unspiked</td><td rowspan=1 colspan=1>N/A (6)</td></tr><tr><td rowspan=1 colspan=1>100×LoD</td><td rowspan=1 colspan=1>5000 EID50/mL (6)</td></tr><tr><td rowspan=1 colspan=1>10×LoD</td><td rowspan=1 colspan=1>500 EID50/mL (6)</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>150 EID50/mL (24)</td></tr><tr><td rowspan=1 colspan=1>1×LoD</td><td rowspan=1 colspan=1>50 EID50/mL (24)</td></tr></table>

Successful validation of frozen samples required that the JBAIDS test results were equivalent to the results obtained when testing freshly collected samples. Frozen samples were considered acceptable to use in the clinical evaluation if the JBAIDS results are at least $90 \%$ in agreement between fresh and frozen samples with a lower $9 5 \%$ CI bound of $8 6 \%$ . If this statistical confidence was not reached with 60 samples, additional samples were evaluated. For informational purposes only, the crossing point $\textstyle ( \mathrm { C p } )$ and relative maximum fluorescence $\mathrm { ( F _ { m a x } ) }$ values were evaluated for the fresh and frozen samples. Ideally the $\mathrm { C p }$ values should not vary by more than 3 cycles and the $\mathrm { F } _ { \mathrm { m a x } }$ values should not be more than $50 \%$ different between the two data sets. The 3 cycle $\mathrm { C p }$ and $5 0 \% \mathrm { F _ { m a x } }$ differences allow for intrinsic system variability in JBAIDS Influenza A Subtyping Kit. If the observed $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ differences were greater than these, the data were reviewed to determine if using frozen samples interferes with detection.

For the Flu A H1 assay, JBAIDS test results for fresh and frozen spiked NPS samples were in $100 \%$ agreement $( 9 5 \% \mathrm { C I } 9 6  – 1 0 0 \% )$ . Test results for fresh and frozen NPW samples were also in $100 \%$ agreement $( 9 5 \% \mathrm { C I } 9 6  – 1 0 0 \% )$ for all virus spike levels. More NPW samples were tested than originally planned due to a user error in which samples were lost. While loading frozen NPW samples into the MagNA Pure kit, the user mishandled four samples (two at $3 { \times } \mathrm { L o D }$ and two at $1 \times \mathrm { L o D } $ ). Additional samples were prepared and tested, both fresh and frozen, to replace the lost samples.

The $\mathrm { C p }$ and $\mathrm { F } _ { \mathrm { m a x } }$ values were also compared between fresh and frozen NPS and NPW samples. Differences between fresh and frozen $\mathrm { C p }$ values ranged between 0.76 cycles earlier and 0.80 cycles later for the Flu A H1 assay. Differences in $\mathrm { F } _ { \mathrm { m a x } }$ values in frozen samples ranged from $23 \%$ less to $3 8 \%$ greater than fresh samples tested with the Flu A H1 assay.

The results of this study demonstrate that there are no significant qualitative differences between JBAIDS test results for fresh or frozen NPS and NPW samples spiked with seasonal Influenza A H1N1 virus and evaluated with the Flu A H1 assay. Accurate test results were obtained from fresh specimens as well as specimens that had been subjected to freezing. The results of this study support the use of frozen archived seasonal Influenza A H1N1 NPS and NPW samples to supplement the performance claims of the JBAIDS Influenza A Subtyping Kit.

2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# a. Prospective Clinical study

The clinical performance of the JBAIDS Influenza A Subtyping Kit was established during prospective studies at 5 geographically separated clinical study sites in the U.S. The study sites were located in the West (Fairfield, CA, and San Diego, CA), the Southwest (San Antonio, TX), the Midwest (Dayton, OH), and the Southeast (Fayetteville, NC). NP swab or wash samples were collected and tested during the 2010/2011 influenza season (December 2010 to April 2011). Subjects with signs and symptoms of influenza-like illness were invited to participate. Upon obtaining informed consent, an NPW or NPS specimen was collected for JBAIDS and comparator assay testing.

Written informed consent (or assent, if appropriate) was acquired from each subject and/or their parent/guardian (if under 18) at the time of enrollment into the study. After informed consent was acquired, the subject was assigned a Volunteer Identification Number (VIN). The VIN was used to de-identify the specimen used for JBAIDS testing and to provide data to the sponsor.

Patient specimens were divided into single use aliquots for testing immediately after collection. Typically, one sample aliquot was placed at $4 \mathrm { { } ^ { \circ } C }$ to be purified within one day and tested using the JBAIDS Influenza A Subtyping Kit. Specimens collected on Fridays were placed at $< - 1 5 ^ { \circ } \mathrm { C }$ over the weekend and thawed for testing on Monday. Aliquots for comparator testing were placed at $< - 1 5 ^ { \circ } \mathrm { C }$ and sent to Idaho Technology, Inc., on a weekly basis. Once received at Idaho Technology, the samples were placed at $\mathrm { < - 7 0 \mathrm { \Omega } ^ { \mathrm { o } } C }$ until they were tested using the comparator assays.

Patient samples were purified using either the IT 1-2-3 Platinum Path Purification Kit (3 testing sites), or the Roche MagNA Pure Compact Nucleic Acid Isolation Kit I in conjunction with the Roche MagNA Pure Compact system (4 testing sites), and tested with the JBAIDS Influenza A Subtyping Kit.

All samples were tested for Influenza A subtypes according to the instructions in the clinical trial protocol. All collected samples, regardless of the Influenza A test result generated by the JBAIDS Influenza A & B Detection Kit, were tested with all assays contained in the JBAIDS Influenza A Subtyping kit. Frozen aliquots of NPS or NPW specimens were shipped to ITI on a weekly basis for comparator PCR testing using the CDC rRT-PCR Flu Panel Assays for Influenza A & B viruses, as well as additional subtype-specific PCR assays followed by bi-directional sequencing of amplicons. Refer to “PCR followed by Bi-directional Sequencing Comparator Assays” section for details regarding assays targets and analytical validation results. Comparator PCR testing was performed by ITI research associates in a blinded manner; specimens were labeled only with the subject’s VIN and the JBAIDS test results were not available to them. Specimen testing with the CDC rRT- PCR Flu Panel assays was performed in accordance with the package insert. PCR followed by bi-directional sequencing comparator assays were performed on all specimens that were tested positive for Influenza A by the CDC rRT- PCR Flu Panel. Bi-directional sequence data was used to verify the identity of each amplicon. “True” seasonal Influenza A/H1, seasonal Influenza A/H3, and 2009 H1N1 Influenza positives, respectively, were considered as any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched seasonal Influenza A/H1, seasonal Influenza $\mathrm { A } / \mathrm { H } 3$ , or 2009 H1N1 Influenza sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values.

The E-value ranges generated from the prospective clinical trial per Influenza A subtype were presented in the following table:

<table><tr><td rowspan=1 colspan=1>Influenza A Subtype</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-value High</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2009 H1N1 Influenza</td><td rowspan=1 colspan=1>2.28E-55</td><td rowspan=1 colspan=1>1.62E-35</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>1.2E-133</td><td rowspan=1 colspan=1>5.25E-105</td></tr></table>

1 The seasonal Influenza A/H1 virus was not circulating during the 2010-2011 influenza season (http://www.cdc.gov/flu/) and was not detected during the prospective clinical evaluation of the JBAIDS Influenza A Subtyping Kit, except for 1 false positive A/H1 result.

A total of 269 runs were performed during the JBAIDS Influenza A Subtyping Kit clinical trial and 261 $( 9 7 . 0 \% )$ JBAIDS runs were completed successfully. Of the 8 $( 3 . 0 \% )$ runs that failed to complete, 6 occurred due to a software defect. The remaining 2 instrument errors were due to a “case blower error” causing the run to abort. There were a total of 9 $( 3 . 4 \% )$ additional runs in which the run controls (NCs and or PCs) failed. Eight failures were associated with the Flu A H1 2009 NC and (or) PC, and one additional failure occurred with the Flu A SW PC. All associated specimens were retested and no specimens were lost due to these failures.

During the prospective clinical study, positive and negative nucleic acid extraction controls (PEC and NEC, respectively) were processed with each batch of purification specimens to ensure that the purification process was effectively performed. The PEC was considered successful when all tested target assays were positive. A failed PEC was retested using the same purified sample and the results of the retest were accepted as the final result. If the retest was unsuccessful, all associated samples were considered to have invalid results and required re-purification in order to obtain a valid test result. During the clinical evaluation, 11 out of a total of 194 PECs required retesting. Nine retests were required due to an invalid assay result (PC or NC). Two PEC failures occurred as a result of one of the four target assays giving a negative result (Flu A H1 one time and Flu A H1 2009 one time). Retesting was performed for one of the failures (Flu A H1 2009) which resolved the failure. There was no retest performed for the other failure. This lack of retesting was a protocol deviation. Five (5) samples associated with this run were excluded from analysis.

A total of 833 subjects were enrolled in the prospective study initially. Eighteen (18) $( 2 . 2 \% )$ withdrew or were withdrawn from the study for various reasons, including discrepancy in consent documents $( \mathrm { n } { = } 7 )$ ), patient refusing sample collection or withdrawal from study $( \mathrm { n } { = } 7 )$ , or improper specimen collection $( \mathrm { n } { = } 4 )$ . Seven (7) specimens were withdrawn from the study due to invalid testing results from the comparator assay. Four (4) specimens were withdrawn due to a contamination event at Site 5. An additional 4 samples were excluded due to inconclusive JBAIDS Influenza A Subtyping Kit results. All data collected from these individuals were removed from the study analyses. Five (5) additional specimens were removed from analysis due to a batch control failure that was not retested. The remaining 795 subjects (483 NPW specimens and $3 1 2 \mathrm { N P S }$ specimens) were included in the demographic and clinical performance analyses.

Of the 795 prospective specimens that were included in the clinical performance analyses, $94 \%$ (751/795) of these specimens were successful on the first attempt (Site $1 \colon 3 5 9 / 3 7 0 = 9 7 \%$ ; Site 2: $1 9 1 / 2 0 6 = 9 3 \%$ ; Site 3: $5 4 / 5 6 = 9 6 \%$ ; Site $4 ; 1 1 0 / 1 1 8 = 9 3 \%$ ; Site 5: $: 3 7 / 4 5 = 9 8 \%$ ). The remaining $6 \%$ (44/795) required retesting: “Invalid” (32/44), “Inconclusive” (11/44), and “Unsubtypeable” (1/44) (11 samples from Site 1; 15 samples from Site 2; 2 samples from Site 3; 8 sample from Site 4; and 8 sample from Site 5). Forty (40) out of 44 samples resolved upon a $1 ^ { \mathrm { s t } }$ retest and the remaining 4 samples required a re-extraction and retest and resolved.

Summary of Included Subject Demographics   

<table><tr><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1">Overall</td><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Site 4</td><td colspan="1" rowspan="1">Site 5</td></tr><tr><td colspan="2" rowspan="1">NPS</td><td colspan="1" rowspan="1">312</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="2" rowspan="1">NPW</td><td colspan="1" rowspan="1">483</td><td colspan="1" rowspan="1">320</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="2" rowspan="1">Total</td><td colspan="1" rowspan="1">795</td><td colspan="1" rowspan="1">370</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">45</td></tr><tr><td colspan="1" rowspan="1">Sav</td><td colspan="1" rowspan="1">Female</td><td colspan="1" rowspan="1">405 (50.9%)</td><td colspan="1" rowspan="1">188 (50.8%)</td><td colspan="1" rowspan="1">122 (59.2%)</td><td colspan="1" rowspan="1">23 (41.1%)</td><td colspan="1" rowspan="1">56 (47.5%)</td><td colspan="1" rowspan="1">16 (35.6%)</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Male</td><td colspan="1" rowspan="1">390 (49.1%)</td><td colspan="1" rowspan="1">182 (49.2%)</td><td colspan="1" rowspan="1">85 (40.8%)</td><td colspan="1" rowspan="1">33 (58.9%)</td><td colspan="1" rowspan="1">62 (52.5%)</td><td colspan="1" rowspan="1">29 (64.4%)</td></tr><tr><td colspan="1" rowspan="4">Agea</td><td colspan="1" rowspan="1">Mean</td><td colspan="1" rowspan="1">26.4</td><td colspan="1" rowspan="1">23.4</td><td colspan="1" rowspan="1">24.5</td><td colspan="1" rowspan="1">30.3</td><td colspan="1" rowspan="1">23.1</td><td colspan="1" rowspan="1">30.8</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">24.0</td><td colspan="1" rowspan="1">24.0</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">27.5</td><td colspan="1" rowspan="1">17.0</td><td colspan="1" rowspan="1">26.0</td></tr><tr><td colspan="1" rowspan="1">Min</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">18.0</td></tr><tr><td colspan="1" rowspan="1">Max</td><td colspan="1" rowspan="1">92.0</td><td colspan="1" rowspan="1">92.0</td><td colspan="1" rowspan="1">69.0</td><td colspan="1" rowspan="1">81.0</td><td colspan="1" rowspan="1">68.0</td><td colspan="1" rowspan="1">62.0</td></tr><tr><td colspan="1" rowspan="4">AgeRangeb</td><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">149 (18.7%)</td><td colspan="1" rowspan="1">88 (23.8%)</td><td colspan="1" rowspan="1">40 (19.4%)</td><td colspan="1" rowspan="1">4 (7.1%)</td><td colspan="1" rowspan="1">17 (14.4%)</td><td colspan="1" rowspan="1">0 (0%)</td></tr><tr><td colspan="1" rowspan="1">6-21c</td><td colspan="1" rowspan="1">229 (28.8%)</td><td colspan="1" rowspan="1">79 (21.4%)</td><td colspan="1" rowspan="1">74 (35.9%)</td><td colspan="1" rowspan="1">10 (17.9%)</td><td colspan="1" rowspan="1">54 (45.8%)</td><td colspan="1" rowspan="1">12c (26.7%)</td></tr><tr><td colspan="1" rowspan="1">22-49</td><td colspan="1" rowspan="1">330 (41.6%)</td><td colspan="1" rowspan="1">178 (48.1%)</td><td colspan="1" rowspan="1">54 (26.2%)</td><td colspan="1" rowspan="1">35 (62.5%)</td><td colspan="1" rowspan="1">34 (28.8%)</td><td colspan="1" rowspan="1">30 (66.7%)</td></tr><tr><td colspan="1" rowspan="1">≥50</td><td colspan="1" rowspan="1">86 (10.8%)</td><td colspan="1" rowspan="1">25 (6.8%)</td><td colspan="1" rowspan="1">38 (18.4%)</td><td colspan="1" rowspan="1">7 (12.5%)</td><td colspan="1" rowspan="1">13 (11.0%)</td><td colspan="1" rowspan="1">3 (6.7%)</td></tr></table>

a 0.5 was used for all ages under 1 year for these calculations. b The age groups $\leq 5$ years and $\ge 5 0$ years correspond to high risk groups for which the CDC strongly recommends seasonal Influenza vaccination (http://www.cdc.gov/flu/protect/keyfacts.htm). c Site 5 enrolled adults only; this category reflects participants 18 to 21 years of age.

The following table shows the summary of positive percent agreement (PPA) and negative percent agreement (NPA) for the JBAIDS Influenza A Subtyping Kit comparing to the comparator RT- PCR followed by bi-directional sequencing assays, stratified by specimen type and extraction method:

JBAIDS Influenza A Subtyping Kit Clinical Performance Summary   

<table><tr><td rowspan=2 colspan=1>Influenza ASubtype</td><td rowspan=2 colspan=1>SampleMatrix</td><td rowspan=2 colspan=1>PurificationKit</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=6 colspan=1>SaE SnT</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>277/278</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>205/205</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.2-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>482/483</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>132/132</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>312/312</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=6 colspan=1>INIH 6007</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>92.9-100%</td><td rowspan=1 colspan=1>227/228</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>97.6-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>16/16</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>79.4-100%</td><td rowspan=1 colspan=1>187/189</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>96.2-99.9%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.6-100%</td><td rowspan=1 colspan=1>414/417</td><td rowspan=1 colspan=1>99.3%</td><td rowspan=1 colspan=1>97.9-99.9%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.8-100%</td><td rowspan=1 colspan=1>108/108</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.6-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>69.2-100%</td><td rowspan=1 colspan=1>169/170</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.8-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.7-100%</td><td rowspan=1 colspan=1>277/278</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=6 colspan=1>Sa/ S</td><td rowspan=3 colspan=1>NPW</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>76.8-100%</td><td rowspan=1 colspan=1>264/264</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.6-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>19/19</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>82.4-100%</td><td rowspan=1 colspan=1>186/186</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>33/33</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.4-100%</td><td rowspan=1 colspan=1>450/450</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.2-100%</td></tr><tr><td rowspan=3 colspan=1>NPS</td><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>18/18</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>81.5-100%</td><td rowspan=1 colspan=1>115/115</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.8-100%</td></tr><tr><td rowspan=1 colspan=1>MagNA Pure</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>63.1-100%</td><td rowspan=1 colspan=1>171/171</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.9-100%</td></tr><tr><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>86.8-100%</td><td rowspan=1 colspan=1>286/286</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.7-100%</td></tr></table>

The following two tables show JBAIDS Influenza A Subtyping Kit clinical performance by site for NPW and NPS specimens, respectively. Due to similar performance when samples were purified with the Platinum Path and MagNA Pure nucleic acid extraction kits, data obtained for samples purified by both methods were combined.

JBAIDS Influenza A Subtyping Kit Clinical Performance by Site for NPW Specimens   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>320/320</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.9-100%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>117/118</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>95.4-100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>45/45</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.1-100%</td></tr><tr><td rowspan=3 colspan=1>Flu A H1 2009 andFlu A Sw</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>26/26</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86.8-100%</td><td rowspan=1 colspan=1>293/294</td><td rowspan=1 colspan=1>99.7%</td><td rowspan=1 colspan=1>98.1-100%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>37/37</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.5-100%</td><td rowspan=1 colspan=1>80/81</td><td rowspan=1 colspan=1>98.8</td><td rowspan=1 colspan=1>93.3-99.97%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.2-100%</td><td rowspan=1 colspan=1>41/42</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>87.4-99.9%</td></tr><tr><td rowspan=3 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2-100%</td><td rowspan=1 colspan=1>300/300</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.8-100%</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>29.2-100%</td><td rowspan=1 colspan=1>115/115</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96.8-100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>69.2-100%</td><td rowspan=1 colspan=1>35/35</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>90.0-100%</td></tr></table>

JBAIDS Influenza A Subtyping Kit Clinical Performance by Site for NPS Specimens   

<table><tr><td rowspan=1 colspan=2>Assay</td><td rowspan=1 colspan=1>Site</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=3 colspan=2>Flu A H1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9-100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>206/206</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.2-100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>56/56</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.6-100%</td></tr><tr><td rowspan=3 colspan=2>Flu A H1 2009 andFlu A Sw</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9-100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.4-100%</td><td rowspan=1 colspan=1>175/176</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.9-99.99%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>39.8-100%</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.2-100%</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9-100%</td></tr><tr><td rowspan=2 colspan=2>Flu A H3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td><td rowspan=1 colspan=1>185/185</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47.8-100%</td><td rowspan=1 colspan=1>51/51</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.0-100%</td></tr></table>

# b. Retrospective Clinical study

The seasonal Influenza A/H1 virus was not circulating during the 2010-2011 influenza season (http://www.cdc.gov/flu/) and was not detected during the prospective clinical evaluation of the JBAIDS Influenza A Subtyping Kit, except for 1 false positive A/H1 result. Therefore, performance of the JBAIDS Influenza A Subtyping Kit testing seasonal Influenza A/H1 could not be established during the prospective clinical study. A retrospective clinical study was carried out at two different clinical sites to supplement the prospective evaluation data by analyzing pre-selected, archived specimens that were known to be positive for seasonal Influenza A/H1. Specimens were evaluated using both the Platinum Path and MagNA Pure nucleic acid purification kits.

A total of 56 NPS specimens were received for testing in this study. Thirty- five (35)

of the specimens had previously tested positive for seasonal Influenza A/H1. These specimens were obtained from the CDC and were originally collected from medical facilities and reference laboratories where they were previously tested by CDC and FDA-cleared laboratory methods and found to contain seasonal Influenza A/H1. No demographic information was available for these specimens. Specimens may have experienced as many as three freeze/thaw cycles prior to this study. The specimens were selected only for their previous test result; there was no effort to choose specimens with specific analyte levels. An additional 21 Influenza A negative NPS specimens from patients with respiratory illness, but known to be negative for influenza, were obtained from ARUP Laboratories. Upon arrival at ITI, a 4-digit study number (VIN) was assigned to each sample and a key was created in order to randomize the specimens such that the users testing the samples were blinded as to the expected test result. Specimens were divided approximately evenly for testing between both purification kits.

Because it is possible that the provided samples had been misidentified or had degraded during storage or previous handling, the presence or absence of the expected analyte was confirmed using “validation” PCR assays. The validation PCR assays were identical to the comparator assays that were used for the prospective clinical evaluation study. Briefly, specimens were first evaluated with the CDC Human Influenza Virus Real-time RT-PCR Detection and Characterization Panel (rRT-PCR Flu Panel) for Influenza A and B. Specimens positive for Influenza A were then subject to additional subtype-specific PCR, in this case using the comparator validation assay used for the prospective clinical evaluation of the JBAIDS Influenza A Subtyping Kit, an assay specifically designed to detect seasonal Influenza A/H1. Positive PCR reactions for the seasonal Influenza A/H1 comparator validation assay were analyzed using bi-directional sequencing for definitive confirmation. If this validation testing did not confirm the presence or absence of the expected organism in the sample, the sample was not analyzed further as this result may have reflected that the sample had been improperly labeled or handled. Validation assay operators were blinded as to the expected result for each sample. Validated specimens with confirmed identities were then tested. Specimens were purified using either the Platinum Path or MagNA Pure sample purification kits then analyzed with the JBAIDS. The Influenza A/H1 assay from the JBAIDS Influenza A Subtyping Kit was run in a Custom Test Method with the Influenza A (Flu A) and influenza sample control (Flu SC) assays from the JBAIDS Influenza A & B Detection Kit.

Validation testing confirmed all 21 influenza-negative specimens and 30 of the 35 seasonal Influenza A/H1 samples. Validation PCR testing revealed that Ct values ranged from 19.2 to 32.0 (median 25.1) out of 45 cycles, demonstrating that the specimen population contained a diverse level of analyte. Half of the samples were extracted using the Platinum Path purification kit and half using MagNA pure.

One (1) seasonal Influenza A/H1 sample did not have a valid test result due to SC failure, and was excluded from data analysis. The remaining 29 seasonal Influenza A/H1 samples and the 21 influenza-negative specimens were included in the performance analysis.

The following table presents the PPA and NPA for the archived clinical specimens.   
Data from both extraction kits are combined due to identical performance.

# Performance Summary of Seasonal Influenza A/H1 Archived Clinical NPS Specimens

c. Surrogate Clinical Specimen Testing for Seasonal Influenza A/H1   

<table><tr><td rowspan=1 colspan=1>InfluenzaAssay</td><td rowspan=1 colspan=1>SpecimenType</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>88.1-100%</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.4-100%</td></tr></table>

The seasonal Influenza A/H1 virus was not circulating during the 2010-2011 influenza season (http://www.cdc.gov/flu/) and was not detected during the prospective clinical evaluation of the JBAIDS Influenza A Subtyping Kit, except for one false positive A/H1 result. In addition, archived clinical NPW samples that had previously tested positive for the seasonal Influenza A/H1 virus were not available. Therefore, a clinical study was performed to evaluate the performance of the JBAIDS Influenza A Subtyping Kit using residual clinical NPS and NPW specimens spiked with a seasonal Influenza A H1N1 strain.

One hundred thirty-six (136) surrogate seasonal Influenza A H1N1 NPS and NPW clinical specimens (68 of each specimen type) were prepared using residual influenzanegative specimens that were collected during the prospective clinical trial of the JBAIDS influenza kits. These specimens had been previously evaluated with both the JBAIDS as well as comparator assays and shown to be influenza- negative. Residual specimens were spiked at ITI with a representative seasonal influenza A H1N1 (A/New Caledonia/20/1999) strain as follows:

NPS and NPW Sample Spike Levels and Replicate Numbers   

<table><tr><td rowspan=2 colspan=1>Spike Level</td><td rowspan=1 colspan=2>NPS and NPW Replicates</td></tr><tr><td rowspan=1 colspan=1>Platinum Path</td><td rowspan=1 colspan=1>MagNA Pure</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>1×LoD</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>3×LoD</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>10×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>100×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=1>1,000×LoD</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=1 colspan=3>Total               68                 68</td></tr></table>

A study number (VIN) was assigned to each sample and a key was created in order to randomize the samples such that the users testing the samples were blinded as to the expected test result. Frozen samples were then sent to 2 different clinical trial sites for testing. Samples were purified using either the Platinum Path or MagNA Pure sample purification kits, and then analyzed with the JBAIDS Flu A subtyping assays.

Of the 136 surrogate samples included in this study, a valid Influenza A test result was obtained for 128 samples (62 NPW and 66 NPS). The remaining 8 samples with invalid Influenza A test results were not included in the analysis.

The following table presents the PPA and NPA for the surrogate clinical specimens. Half of the samples were extracted using the Platinum Path purification kit and half using the MagNA pure kit. Data from both extraction kits are combined due to identical performance.

Performance Summary of Seasonal Influenza A/H1 Surrogate Clinical Specimens   

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=2 colspan=1>Sample Type</td><td rowspan=1 colspan=3>PPA</td><td rowspan=1 colspan=3>NPA</td></tr><tr><td rowspan=1 colspan=1>TP/(TP+FN)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/(TN+FP)</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=2 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>NPW</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.4-100%</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>63.1-100%</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>59/59</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>93.9-100%</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>59.0-100%</td></tr><tr><td rowspan=2 colspan=1>Flu A H1 2009andFlu A Sw</td><td rowspan=1 colspan=1>NPW</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.2-100%</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.6-100%</td></tr><tr><td rowspan=2 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>NPW</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>62/62</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.2-100%</td></tr><tr><td rowspan=1 colspan=1>NPS</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>66/66</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.6-100%</td></tr></table>

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

The prospective trial of the JBAIDS Influenza A Subtyping Kit tested nasopharyngeal wash (NPW) and nasopharyngeal swab (NPS) specimens obtained from individuals with influenza-like illness (ILI). Specimens were collected from volunteers at five sites across the United States from December 2010 through April 2011. A total of 795 patient specimens were analyzed. The number and percentage of positive cases as determined with the JBAIDS Influenza A Subtyping Kit by testing site and sample type or by age group and sample type are presented in the following tables. Seasonal Influenza A/H1 was not circulating while this study was conducted.

Expected Values (as Determined by the JBAIDS) Summary by Site and Sample Type   

<table><tr><td colspan="1" rowspan="2">PositiveResults byInfluenzaAssay</td><td colspan="1" rowspan="2">SampleType</td><td colspan="2" rowspan="1">Overall (NPS:n=312; NPW:n=483)</td><td colspan="2" rowspan="1">Site 1 (NPS:n=50; NPW:n=320)</td><td colspan="2" rowspan="1">Site 2 (n=206)</td><td colspan="2" rowspan="1">Site 3 (n=56)</td><td colspan="2" rowspan="1">Site 4 (n=118)</td><td colspan="2" rowspan="1">Site 5 (n=45)</td></tr><tr><td colspan="1" rowspan="1">Neueb</td><td colspan="1" rowspan="1">Rpececete</td><td colspan="1" rowspan="1">Neunb</td><td colspan="1" rowspan="1">Rpeccarp</td><td colspan="1" rowspan="1">Neuer</td><td colspan="1" rowspan="1">ppeccrteu</td><td colspan="1" rowspan="1">Nqunb</td><td colspan="1" rowspan="1">Eepececaicu</td><td colspan="1" rowspan="1">Neunr</td><td colspan="1" rowspan="1">ppeccnt</td><td colspan="1" rowspan="1">Neueb</td><td colspan="1" rowspan="1">peccaecnu</td></tr><tr><td colspan="1" rowspan="2">SeasonalInfluenzaA/H1</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="2" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="2">2009H1N1Influenza</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">11.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">7.1%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">14.1%</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">8.2%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">32.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8.9%</td></tr><tr><td colspan="2" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">13.0%</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">7.0%</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">15.0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">7.1%</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">32.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">8.9%</td></tr><tr><td colspan="1" rowspan="1">SeasonalInfluenza</td><td colspan="1" rowspan="1">NPS</td><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">8.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">10.2%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">8.9%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NPW</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">6.8%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">6.3%</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.2%</td></tr><tr><td colspan="2" rowspan="1">TOTAL</td><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">7.4%</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">5.4%</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">10.2%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">8.9%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2.5%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">22.2%</td></tr></table>

Expected Value (as Determined by the JBAIDS) Summary by Age Group and Sample Type   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Sample Type</td><td rowspan=1 colspan=1>Total(Expected Value)</td><td rowspan=1 colspan=1>≤5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>≥50 years</td></tr><tr><td rowspan=2 colspan=1>Seasonal Influenza A/H1</td><td rowspan=1 colspan=1>NPS (n=312)</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>NPW (n=483)</td><td rowspan=1 colspan=1>1 (0.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=2 colspan=1>2009 H1N1 Influenza</td><td rowspan=1 colspan=1>NPS (n=312)</td><td rowspan=1 colspan=1>35 (11.2%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>NPW (n=483)</td><td rowspan=1 colspan=1>68 (14.1%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>8</td></tr><tr><td rowspan=2 colspan=1>Seasonal Influenza A/H3</td><td rowspan=1 colspan=1>NPS (n=312)</td><td rowspan=1 colspan=1>26 (8.3%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>NPW (n=483)</td><td rowspan=1 colspan=1>33 (6.8%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>3</td></tr></table>

# N. Instrument Name:

Joint Biological Agent Identification and Diagnostic System (JBAIDS) Instrument.

# O. System Descriptions:

# 1. Modes of Operation:

The JBAIDS Instrument is used to perform real time reverse transcription, PCR amplification and detection of nucleic acid. Three other nucleic acid amplification tests that use the JBAIDS Instrument have received 510(k) clearance: JBAIDS Anthrax Detection System (k051713), JBAIDS Plague Detection System (k072631), JBAIDS Tularemia Detection System (k072547), JBAIDS Influenza A/H5 (Asian Lineage) Detection Kit (k100287), and JBAIDS Q Fever Detection System (k103207). The JBAIDS Instrument is a ruggedized, portable real-time PCR instrument designed to withstand the conditions of transport and use likely to be encountered in a military field laboratory. The instrument is comprised of an air thermocycler that amplifies specific DNA sequences using PCR and a fluorimeter that measures fluorescence signals associated with production of PCR product (amplicon) during the course of the reaction. For thermocycling, samples contained in glass capillaries are placed in the sample chamber where they are heated and cooled by the JBAIDS instrument.

The JBAIDS Software is preloaded on a ruggedized laptop computer. The software controls the instrument’s thermal cycling functions, acquires the fluorescence data from the instrument, and displays the fluorescence data for the user during the run. When the run is finished, the software’s Detector module analyzes the data and displays test results. The instrument and software have the ability to perform either diagnostic (IVD) or surveillance (environmental) testing.

The IVD software includes the following diagnostic components:

A traceable database that limits the data that the user can change in a test run and that requires the user to enter change notes for any changes that are allowed. A Diagnostic Wizard that guides the operator through the process of setting up the JBAIDS run in compliance with the package insert. IVD testing is always performed using the Diagnostic Wizard and the traceable database. The wizard initially presents to the operator a predefined selection of FDA cleared JBAIDS IVD assays. JBAIDS assays under evaluation for FDA clearance are also available in limited release software. Once the user has selected the IVD assays to test, the Wizard guides the user through the process of setting up and loading each PCR reaction into the instrument. The wizard requires each run to contain a Positive Control and a Negative Control for each target included in the run as well as the inclusion of additional controls (e.g., inhibition controls, sample controls), when they are required.   
• A Detector module that automatically analyzes the fluorescence amplification curves and displays the final test results to the user. The Detector module ensures that the correct rules are applied both to the interpretation of each amplification curve and to the final interpretation (e.g., metacall), based on the results of the samples and the relevant controls.   
• A report certification that reflects the current regulatory status of the test being performed. When an assay is being evaluated for FDA clearance, the JBAIDS test report includes the statement ‘For Investigational Use Only. The performance characteristics of this product have not been established.’ Once the assay has received FDA clearance, the software is modified and the report certification statement becomes ‘For In Vitro Diagnostic Use Only’.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No _

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

The following controls are included in the JBAIDS Influenza A Subtyping Kit

# Negative Control

The NC is used to detect contamination from target-specific amplified product (amplicon), synthetic RNA (as found in the PC vials), or virus. The NC vial is reconstituted using the purified negative extraction control (NEC) (molecular biology grade water). The NEC should be processed along with each batch of specimens purified. The NC detects possible contamination during sample purification and during reconstitution of freeze- dried reagents. Each JBAIDS Influenza A Subtyping Kit run requires one NC for each of the 4 assays (i.e., the Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw assays). For each target assay, the NC must be negative, or the JBAIDS software will assign invalid results for that target assay to all of the samples in that run. If the NC fails, the JBAIDS run must be repeated using the same purified samples. If the NC fails again, all associated samples are considered to have invalid results and should be re-purified starting from another aliquot of the original specimens.

# Positive Control

The PC serves as an amplification and detection control. Each JBAIDS Influenza A Subtyping Kit run requires one PC for each of the 4 assays (i.e., the Flu A H1, Flu A H3, Flu A H1 2009, and Flu A Sw assays). For each target assay, the PC must be positive. If the PC fails, the JBAIDS software will assign invalid results for that target assay to all of the associated samples, and the JBAIDS run must be repeated starting from the same purified specimens. Failure of the PCs may indicate errors in sample setup, degradation of the reagents, or a malfunction of the JBAIDS instrument.

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision