# 510(k) Summary

This summary of 510(k) safety and effectiveness information is being submitted in accordance with the requirements of 21 CFR 807.92.

$\pmb { 5 1 0 ( k ) }$ Number: k112781

Purpose for Submission: Modification to PCR primer mix of the previously cleared $\mathsf { \pmb { x } } \mathsf { T } \mathsf { A G } ^ { \otimes }$ RVP (k112199) originally cleared under k063765 to improve reactivity to influenza A/H3 strains.

Measurand: Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Human Metapneumovirus, Rhinovirus, and Adenovirus

Type of Test: Qualitative nucleic acid multiplex test

Applicant: Luminex Molecular Diagnostics Inc.

Proprietary and Established Names: xTAG Respiratory Viral Panel (RVP)

Regulatory Information:   

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Review Panel</td></tr><tr><td rowspan=1 colspan=1>OCC, OEM,OEP, NSU, JJH</td><td rowspan=1 colspan=1>Class II</td><td rowspan=1 colspan=1>21 CFR 866.3980 Respiratory viral panelmultiplex nucleic acid assay</td><td rowspan=1 colspan=1>Microbiology(83)</td></tr></table>

# Intended Use:

The $\mathsf { x T A G } ^ { \otimes }$ Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of multiple respiratory virus nucleic acids in nasopharyngeal swabs from individuals suspected of respiratory tract infections. The following vius types and subtypes are identified using RVP: Influenza A, Influenza B, Respiratory Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Human Metapneumovirus, Rhinovirus, and Adenovirus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and laboratory findings.

xTAG RVP can also differentiate the hemagglutinin (HA) gene of some Influenza A subtypes H1 and H3 strains. Differentiation of Influenza A HA subtypes is based on both a positive result for the Influenza A matrix gene and an accompanying positive result for the Influenza A HA subtype H1 (circulating prior to the emergence of 2009 H1N1pdm) or Influenza A HA subtype H3. This device cannot differentiate the Influenza A HA subtype 2009 H1N1pdm by design, and may not be able to differentiate potential newly emerging Influenza A HA subtypes.

# Luminex.

Luminex Molecular Diagnostics

u acl n  v deteced may not be he definite cause f disease.The use f aditional laboratory testing .. bacterial culture, immunofluorescence, radiography) and clinical presentation must be taken into consideration in order toobtain the final diagnosis of respiratory viral infection.

Negative results do not preclude respiratory virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

The RVP assay cannot adequately detect Adenovirus species C, or serotypes 7a and 41. It is recommended that specimens found to be negative for Adenovirus after examination using RVP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture). The RVP primers for detection of rhinovirus cross-react with enterovirus. A rhinovirus reactive result should be confirmed by an alternate method (e.g. cell culture).

Performance characteristics for Influenza A virus were established when Influenza A HA subtype H3, subtype H1 (prior to the emergence of 2009 H1N1pdm), and when subtype 2009 H1N1pdm were the predominant Influenza A in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to a state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

Indication(s) for use: Same as intended use.

Special conditions for use statement(s): N/A

Special instrument requirements: Luminex 100 or 200 instrument with IS or xPONENT software

# Device Description:

The modified RVP is a PCR-based test system for detecting the presence / absence of viral DNA / RNA in clinical specimens. The modified device is the same as the predicate device, except for a reformulation of the PCR primer mix.

# Substantial Equivalence Information:

a. Predicate device name(s): xTAG Respiratory Viral Panel

b. Predicate 510(k) number(s): k063765, k081843, k091667 and k112199

c. Comparison with predicate:

The following table compares the modified xTAG Respiratory Viral Panel with the xTAG Respiratory Viral Panel (k063765, k081843, k091667, k112199).

Luminex Molecular Diagnostics

Table 1: Comparison between Modified (New) Device and Predicate   

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Modified Device(k112781)XTAG RVP</td><td rowspan=1 colspan=1>Predicates(k063765, k081483, k091667, k112199)XTAG RVP</td></tr><tr><td rowspan=1 colspan=1>Manufacturer</td><td rowspan=1 colspan=1>Luminex Molecular Diagnostics</td><td rowspan=1 colspan=1>Luminex Molecular Diagnostics</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td></tr><tr><td rowspan=1 colspan=1>Amplification Method</td><td rowspan=1 colspan=1>Multiplex end point RT-PCR</td><td rowspan=1 colspan=1>Multiplex end point RT-PCR</td></tr><tr><td rowspan=1 colspan=1>Test Format</td><td rowspan=1 colspan=1>Multiplex bead-based universal arraysorting on Luminex 100/200 instrument</td><td rowspan=1 colspan=1>Multiplex bead-based universal array sortingon Luminex 100/200 instrument</td></tr><tr><td rowspan=1 colspan=1>Detection Method</td><td rowspan=1 colspan=1>Fluorescence based</td><td rowspan=1 colspan=1>Fluorescence based</td></tr><tr><td rowspan=1 colspan=1>Quality Control</td><td rowspan=1 colspan=1>Internal Control (E. coli phage MS2),Run Control (bacteriophage LambdaDNA), rotating analyte control andnegative controls</td><td rowspan=1 colspan=1>Internal Control (E. coli phage MS2) and RunControl (bacteriophage Lambda DNA),rotating analyte control and negative controls</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>Qualitative</td><td rowspan=1 colspan=1>Qualitative</td></tr><tr><td rowspan=1 colspan=1>Instrument</td><td rowspan=1 colspan=1>LX100 or LX200 with xMAP system (ISor xPONENT)</td><td rowspan=1 colspan=1>LX100 or LX200 with xMAP system (IS orxPONENT)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Same as predicate</td><td rowspan=1 colspan=1>See above</td></tr><tr><td rowspan=1 colspan=1>Targets Reported</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1,Influenza A subtype H3, Influenza B,Respiratory Syncytial Virus A,Respiratory Syncytial Virus B,Parainfluenza 1, Parainfluenza 2,Parainfluenza 3, HumanMetapneumovirus, Rhinovirus, andAdenovirus</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1, InfluenzaA subtype H3, Influenza B, RespiratorySyncytial Virus A, Respiratory Syncytial VirusB, Parainfluenza 1, Parainfluenza 2,Parainfluenza 3, Human Metapneumovirus,Rhinovirus, and Adenovirus</td></tr><tr><td rowspan=1 colspan=1>Sample Preparation</td><td rowspan=1 colspan=1>QIAGEN QlAamp MinElute, BiomérieuxNucliSENS® EasyMag®, and BiomérieuxMiniMag™</td><td rowspan=1 colspan=1>QIAGEN QIAamp MinElute, BiomérieuxNucliSENS® EasyMag, and BiomérieuxMiniMag™</td></tr><tr><td rowspan=1 colspan=1>Amplification Enzyme</td><td rowspan=1 colspan=1>xTAG® OneStep Enzyme Mix andancillary reagent TaKaRa Taq™ Hot Start</td><td rowspan=1 colspan=1>xTAG® OneStep Enzyme Mix and ancillaryreagent TaKaRa Taq™ Hot Start</td></tr><tr><td rowspan=1 colspan=1>Primer Mixes</td><td rowspan=1 colspan=1>Two primer mixes (1 for PCR and 1 forTSPE). Modified PCR primer mix</td><td rowspan=1 colspan=1>Two primer mixes (1 for PCR and 1 for TSPE)</td></tr><tr><td rowspan=1 colspan=1>Software</td><td rowspan=1 colspan=1>xTAG Data Analysis Software RVP (US)</td><td rowspan=1 colspan=1>xTAG Data Analysis Software RVP (US)</td></tr></table>

# Standards/Guidance Documents referenced (if applicable):

Table 2: Guidance Documents

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Title</td><td rowspan=1 colspan=1>Date</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Class Il Special Controls Guidance: Respiratory Viral Panel MultiplexNucleic Acid Assay</td><td rowspan=1 colspan=1>Oct. 9, 2009</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Class Il Special Control Guidance Document: Testing for Detection andDifferentiation of Influenza A Virus Subtypes Using Multiplex Assays</td><td rowspan=1 colspan=1>Oct. 9, 2009</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Guidance (Draft) for Establishing the Performance Characteristics of InVitro Diagnostic Devices for the Detection or Detection andDifferentiation of Influenza Viruses</td><td rowspan=1 colspan=1>Feb. 15, 2008</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Guidance for In Vitro Diagnostic Devices to Detect Influenza A Viruses:Labeling and Regulatory Path</td><td rowspan=1 colspan=1>May 1, 2007</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Class II Special Controls Guidance: Reagents for Detection of SpecificNovel Influenza A Viruses</td><td rowspan=1 colspan=1>Mar. 22, 2006</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Class II Special Control Guidance Document: &quot;Testing for HumanMetapneumovirus (hMPV) Using Nucleic Acid Assays&quot;</td><td rowspan=1 colspan=1>Oct. 9, 2009</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Guidance for the Content of Premarket Submissions for SoftwareContained in Medical Devices</td><td rowspan=1 colspan=1>May 11, 2005</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Guidance document for Format for Traditional and Abbreviated 510(k)s</td><td rowspan=1 colspan=1>Aug. 12, 2005</td></tr></table>

Table 3: Standards   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>StandardsNo.</td><td rowspan=1 colspan=1>RecognitionNumber(FDA)</td><td rowspan=1 colspan=1>Standards Title</td><td rowspan=1 colspan=1>Date</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>MM13-A</td><td rowspan=1 colspan=1>7-191</td><td rowspan=1 colspan=1>Collection, Transport, Preparation andStorage of Specimens</td><td rowspan=1 colspan=1>03/18/2009</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>MM03-A2</td><td rowspan=1 colspan=1>7-132</td><td rowspan=1 colspan=1>Molecular Diagnostic Methods for InfectiousDiseases (2nd edition)</td><td rowspan=1 colspan=1>09/09/2008</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>EP12-A2</td><td rowspan=1 colspan=1>7-152</td><td rowspan=1 colspan=1>User Protocol for Evaluation of QualitativeTest Performance (2d edition)</td><td rowspan=1 colspan=1>09/09/2008</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ISO14971</td><td rowspan=1 colspan=1>5-40</td><td rowspan=1 colspan=1>Medical devices - Application of riskmanagement to medical devices</td><td rowspan=1 colspan=1>09/12/2007</td></tr></table>

Test Principle: Same as predicate

# Performance Characteristics:

Analytical Performance: Precision/Reproducibility: Same as predicate.

Luminex Molecular Diagnostics

# Limit of Detection (LoD):

The LoD for Influenza A subtype H3 was determined using two strains of influenza A comparing results of the predicate for these analytes to those of the modified device (see Table 4).

Table 4: Comparison of (LoD) for Influenza A H3 between Modified and Original RVP   

<table><tr><td rowspan=2 colspan=1>Strain ID</td><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Modified xTAG RVP</td><td rowspan=1 colspan=2>Original xTAG RVP</td></tr><tr><td rowspan=1 colspan=1>TCIDso/mL(atestimatedLoD)</td><td rowspan=1 colspan=1>Average MFIfrom 22replicates atLoD</td><td rowspan=1 colspan=1>TCIDso/mL (atestimatedLoD)</td><td rowspan=1 colspan=1>Average MFIfrom 22replicates atLoD</td></tr><tr><td rowspan=2 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>Flu AMatrix</td><td rowspan=1 colspan=1>0.4768</td><td rowspan=1 colspan=1>1806.84</td><td rowspan=1 colspan=1>0.4768</td><td rowspan=1 colspan=1>1776.05</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>0.4768</td><td rowspan=1 colspan=1>974.36</td><td rowspan=1 colspan=1>7.629</td><td rowspan=1 colspan=1>1219.64</td></tr><tr><td rowspan=2 colspan=1>A/Perth/16/2009</td><td rowspan=1 colspan=1>Flu AMatrix</td><td rowspan=1 colspan=1>0.1347</td><td rowspan=1 colspan=1>1225.16</td><td rowspan=1 colspan=1>0.5388*</td><td rowspan=1 colspan=1>2796.07</td></tr><tr><td rowspan=1 colspan=1>Flu A H3</td><td rowspan=1 colspan=1>0.1347</td><td rowspan=1 colspan=1>706.39</td><td rowspan=1 colspan=1>8.621</td><td rowspan=1 colspan=1>1441.16</td></tr></table>

v wa  wi   ak Ma  . $T C l D _ { 5 0 } / m L$ (one dilution below 0.5388 $T C 1 0 _ { 5 0 } / m L$ level), 18 out of 22 replicates made the correct Flu A Matrix POS al l a  al below the cut-off, thus generating $^ { \prime \prime } \mathsf { N o } \mathsf { C a } \mathsf { N } ^ { \prime \prime }$ results for Flu A Matrix.

In addition, the limit of detection study compared the LoD of the modified xTAG RVP assay with the original xTAGRVP assay for all targets in the RVP panel using one strain for each target (Table 5). For each strain, 20 replicates of the dilutions at the estimated LoD level and at least the two bracketing levels were tested.

Luminex Molecular Diagnostics

Table 5: Summary of Limit of Detection (LoD) for the non-H3 Targets   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=2 colspan=1>Strain ID</td><td rowspan=2 colspan=1>Dilution Levels (L1 andL3 are at 4 fold belowand above theestimated LOD,respectively}</td><td rowspan=1 colspan=3>Original xTAG RVP</td><td rowspan=1 colspan=3>Modifled xTAG&quot; RVP</td></tr><tr><td rowspan=1 colspan=1>TCID5o/mL (atestimatedLoD)</td><td rowspan=1 colspan=1>Average MFIfrom 20replicates atLoD</td><td rowspan=1 colspan=1>No. of POSCalls from 20replicates</td><td rowspan=1 colspan=1>TCDso/mL (atestimatedLoD</td><td rowspan=1 colspan=1>Average MFIfrom 20replicates atLoD</td><td rowspan=1 colspan=1>No. of POSCalls from 20replicates</td></tr><tr><td rowspan=3 colspan=1>Flu A Matrix</td><td rowspan=3 colspan=1>Solomon Island/3/2006</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>1.91E+00</td><td rowspan=1 colspan=1>3464.9</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.91E+00</td><td rowspan=1 colspan=1>2914.9</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level}</td><td rowspan=1 colspan=1>4.77E-01</td><td rowspan=1 colspan=1>3059.4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.77E-01</td><td rowspan=1 colspan=1>2099.6</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>1.19E-01</td><td rowspan=1 colspan=1>229.8</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1.19E-01</td><td rowspan=1 colspan=1>209.9</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>Flu A H1</td><td rowspan=3 colspan=1>Solomon Island/3/2006</td><td rowspan=1 colspan=1>.1</td><td rowspan=1 colspan=1>1.91E+00</td><td rowspan=1 colspan=1>944.3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.91E+00</td><td rowspan=1 colspan=1>721.9</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>L.2 (LOD Level</td><td rowspan=1 colspan=1>4.77E-01</td><td rowspan=1 colspan=1>857.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.77E-01</td><td rowspan=1 colspan=1>492.9</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>1.19E-01</td><td rowspan=1 colspan=1>72.7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1.19E-01</td><td rowspan=1 colspan=1>74.2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Influenza B</td><td rowspan=3 colspan=1>Brisbane/33/08</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>7.82E-01</td><td rowspan=1 colspan=1>1872.8</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.82E-01</td><td rowspan=1 colspan=1>2101.9</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level}</td><td rowspan=1 colspan=1>1.96E-01</td><td rowspan=1 colspan=1>923.9</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.96E-01</td><td rowspan=1 colspan=1>753.1</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>4.89E-02</td><td rowspan=1 colspan=1>187.5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4.89E-02</td><td rowspan=1 colspan=1>206.4</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=3 colspan=1>RSV A</td><td rowspan=3 colspan=1>Long</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>7.63E-02</td><td rowspan=1 colspan=1>2473.4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.63E-02</td><td rowspan=1 colspan=1>2694.7</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level</td><td rowspan=1 colspan=1>1.91E-02</td><td rowspan=1 colspan=1>595.4</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.91E-02</td><td rowspan=1 colspan=1>595.9</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>4.77E-03</td><td rowspan=1 colspan=1>159.4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4.77E-03</td><td rowspan=1 colspan=1>212.3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=3 colspan=1>RSV B</td><td rowspan=3 colspan=1>Wash/18537/62</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>4.88E+00</td><td rowspan=1 colspan=1>2820.2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.88E+00</td><td rowspan=1 colspan=1>3604.8</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level</td><td rowspan=1 colspan=1>1.22E+00</td><td rowspan=1 colspan=1>923.5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.22E+00</td><td rowspan=1 colspan=1>921.2</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>3.05E-01</td><td rowspan=1 colspan=1>202.0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.05E-01</td><td rowspan=1 colspan=1>337.8</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=3 colspan=1>hMPV</td><td rowspan=3 colspan=1>CDC Isolate</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>1.60E+00</td><td rowspan=1 colspan=1>5844.5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.60E+00</td><td rowspan=1 colspan=1>6284.475</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level}</td><td rowspan=1 colspan=1>4.00E-01</td><td rowspan=1 colspan=1>1395.625</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.00E-01</td><td rowspan=1 colspan=1>1494.4</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>1.00E-01</td><td rowspan=1 colspan=1>345.5</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.00E-01</td><td rowspan=1 colspan=1>466.125</td><td rowspan=1 colspan=1>16</td></tr><tr><td rowspan=3 colspan=1>Para-1</td><td rowspan=3 colspan=1>C-35</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>3.91E-01</td><td rowspan=1 colspan=1>2188.2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3.91E-01</td><td rowspan=1 colspan=1>2062.5</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L.2 (LOD Level}</td><td rowspan=1 colspan=1>9.77E-02</td><td rowspan=1 colspan=1>865.7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>9.77E-02</td><td rowspan=1 colspan=1>749.5</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>2.44E-02</td><td rowspan=1 colspan=1>164.8</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.44E-02</td><td rowspan=1 colspan=1>240.9</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=3 colspan=1>Para-2</td><td rowspan=3 colspan=1>Greer</td><td rowspan=1 colspan=1>LI</td><td rowspan=1 colspan=1>7.63E-01</td><td rowspan=1 colspan=1>5113.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.63E-01</td><td rowspan=1 colspan=1>5889.8</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level</td><td rowspan=1 colspan=1>1.91E-01</td><td rowspan=1 colspan=1>4238.5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.91E-01</td><td rowspan=1 colspan=1>5340.9</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>[L3</td><td rowspan=1 colspan=1>4.77E-02</td><td rowspan=1 colspan=1>467.0</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>4.77E-02</td><td rowspan=1 colspan=1>675.3</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=3 colspan=1>Para-3</td><td rowspan=3 colspan=1>Zeptometrix 0810016CF</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>3206.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.00E+01</td><td rowspan=1 colspan=1>2524.6</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level]</td><td rowspan=1 colspan=1>2.51E+00</td><td rowspan=1 colspan=1>729.5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>2.51E+00</td><td rowspan=1 colspan=1>1148.5</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>6.27E-01</td><td rowspan=1 colspan=1>38.8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6.27E-01</td><td rowspan=1 colspan=1>8.3</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Adenovirus</td><td rowspan=3 colspan=1>Type 1</td><td rowspan=1 colspan=1>L1</td><td rowspan=1 colspan=1>4.07E+01</td><td rowspan=1 colspan=1>1272.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4.07E+01</td><td rowspan=1 colspan=1>737.8</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level</td><td rowspan=1 colspan=1>1.02E+01</td><td rowspan=1 colspan=1>494.1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1.02E+01</td><td rowspan=1 colspan=1>468.3</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>2.54E+00</td><td rowspan=1 colspan=1>193.6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2.54E+00</td><td rowspan=1 colspan=1>158.6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=3 colspan=1>Rhinovirus</td><td rowspan=3 colspan=1>Type 54</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>3.00E-02</td><td rowspan=1 colspan=1>3052.7</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3.00E-02</td><td rowspan=1 colspan=1>3773.0</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L2 (LOD Level}</td><td rowspan=1 colspan=1>7.50E-03</td><td rowspan=1 colspan=1>1006.6</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>7.50E-03</td><td rowspan=1 colspan=1>1387.2</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>L3</td><td rowspan=1 colspan=1>1.88E-03</td><td rowspan=1 colspan=1>399.6</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>1.88E-03</td><td rowspan=1 colspan=1>366.3</td><td rowspan=1 colspan=1>13</td></tr></table>

The results from this LoD study indicate that the modified xTAG® RVP is equivalent to that of the original ${ \tt x T A G } ^ { \otimes }$ RVP for all target calls.

Carryover Contamination Limit of Blank (LoB): Same as predicate

Analytical Specificity (Reactivity, Cross-Reactivity and Competitive Inhibition):

A total of 48 potentially cross-reactive pathogens (bacterial and viral) were assessed in replicates with RVP. Each replicate underwent a single EasyMag (bioMerieux NucliSENs\*) extraction prior to testing.

Luminex Molecular Diagnostics

a   

<table><tr><td rowspan=2 colspan=1>Organlsm</td><td rowspan=2 colspan=1>Straln</td><td rowspan=2 colspan=1>TiterTested</td><td rowspan=2 colspan=1>TiterUnits</td><td rowspan=1 colspan=12>Analyte, Results Positive )r Negative Reactivity</td></tr><tr><td rowspan=1 colspan=1>Bk  ry</td><td rowspan=1 colspan=1>TH A DJ</td><td rowspan=1 colspan=1>EH  DJ</td><td rowspan=1 colspan=1>g ml3</td><td rowspan=1 colspan=1>P ered</td><td rowspan=1 colspan=1>Zead</td><td rowspan=1 colspan=1>Eaad</td><td rowspan=1 colspan=1>HASA</td><td rowspan=1 colspan=1>HASE</td><td rowspan=1 colspan=1>Rr</td><td rowspan=1 colspan=1>Mnnouneutn</td><td rowspan=1 colspan=1>Aae</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>NEOAS 1505</td><td rowspan=1 colspan=1>13.66</td><td rowspan=1 colspan=1>ct*</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium glutamicum</td><td rowspan=1 colspan=1>Type straln 534 [NCIB10025}</td><td rowspan=1 colspan=1>6.00 x 10}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Escherichla coll</td><td rowspan=1 colspan=1>ATCC 8739</td><td rowspan=1 colspan=1>5.60 x 10}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Type b (Zeptometrix0801680</td><td rowspan=1 colspan=1>2.63 x 10}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Lactobacillus cosei</td><td rowspan=1 colspan=1>03 [7, IAM 12473, Orland L-323, R.P. Tittsler 303]</td><td rowspan=1 colspan=1>6.00 x 108$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>ATCC 33152</td><td rowspan=1 colspan=1>15.42</td><td rowspan=1 colspan=1>ct*</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Moraxella (Branhamella)catarrhalis</td><td rowspan=1 colspan=1>Ne 11</td><td rowspan=1 colspan=1>5.00 x 104</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>~</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Mycobacterium avium subsp.avium</td><td rowspan=1 colspan=1>ATCC 15769</td><td rowspan=1 colspan=1>2.50 x 104</td><td rowspan=1 colspan=1>cfu/ml</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Mycobacterium intracellulare</td><td rowspan=1 colspan=1>ATCC 13209</td><td rowspan=1 colspan=1>2.50 × 10^{}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>×</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumonice</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>5.63 x 10}$</td><td rowspan=1 colspan=1>TCIDs_/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata subsp.elangata</td><td rowspan=1 colspan=1>NCTC 10660</td><td rowspan=1 colspan=1>2.50 x 10^</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitides</td><td rowspan=1 colspan=1>Zeptometrix 0801511</td><td rowspan=1 colspan=1>3.37× 10^}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>&quot;</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>ATCC15442</td><td rowspan=1 colspan=1>4.00 x 10^8}$</td><td rowspan=1 colspan=1>cfu/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>Zepto 0801638</td><td rowspan=1 colspan=1>4.00 × 108}}$</td><td rowspan=1 colspan=1>cfu/ml</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>L</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>ATCC 12228</td><td rowspan=1 colspan=1>$4.00 x 10^}$</td><td rowspan=1 colspan=1>cfu/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>Type 59</td><td rowspan=1 colspan=1>15.95</td><td rowspan=1 colspan=1>ct</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>ATCC 51500</td><td rowspan=1 colspan=1>2.0.x10^}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>75 [NCTC 8618]</td><td rowspan=1 colspan=1>6.00 x 10^}$</td><td rowspan=1 colspan=1>cfu/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=16> .</td></tr></table>

These bacterial pathogens did not cross-react or interfere with any viral target probed by RVP in either the original or modified device.

T   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TiterTested</td><td rowspan=2 colspan=1>TiterUnits</td><td rowspan=1 colspan=12>Andlyte, Resuits Posltive (+) or Negative (-)forReactivity</td></tr><tr><td rowspan=1 colspan=1>Ba  rky</td><td rowspan=1 colspan=1>BH A A</td><td rowspan=1 colspan=1>EH Y NJ</td><td rowspan=1 colspan=1>g nIJ</td><td rowspan=1 colspan=1>I esed</td><td rowspan=1 colspan=1>Paaa2</td><td rowspan=1 colspan=1>aaaA3</td><td rowspan=1 colspan=1>Y ASA</td><td rowspan=1 colspan=1>BAS</td><td rowspan=1 colspan=1>Rroi</td><td rowspan=1 colspan=1>Mmenounudnw</td><td rowspan=1 colspan=1>Aeeons</td></tr><tr><td rowspan=1 colspan=1>Flu A H1 (Seasonai)</td><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>5.00 x 10^{$</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>3.16× 10^}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/Mass/3/66</td><td rowspan=1 colspan=1>3.16 x 10</td><td rowspan=1 colspan=1>TCIDso/mL.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>C-35</td><td rowspan=1 colspan=1>1.58 x 10^}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>Greer</td><td rowspan=1 colspan=1>5.00 x 10^}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>C-243</td><td rowspan=1 colspan=1>5.00 × 10^{}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 4</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>$4.17× 10^}$</td><td rowspan=1 colspan=1>TCIDso/m</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 48</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>2.45x 10^}$</td><td rowspan=1 colspan=1>TCIDs/\m</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>RSVA</td><td rowspan=1 colspan=1>Long</td><td rowspan=1 colspan=1>5.00 x 102</td><td rowspan=1 colspan=1>TCID/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>Wash/18537/62</td><td rowspan=1 colspan=1>1.00 × 10</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Enterovirus (Echo 13)</td><td rowspan=1 colspan=1>Del Carmen</td><td rowspan=1 colspan=1>5.00 x 10</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>v</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>*</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>¤</td></tr><tr><td rowspan=1 colspan=1>Enterovirus (Coxsackie B)</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>5.00 x 10$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Type 68</td><td rowspan=1 colspan=1>Fermon</td><td rowspan=1 colspan=1>1.00 x 10$</td><td rowspan=1 colspan=1>TCIDsg/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Type 69</td><td rowspan=1 colspan=1>Toluca-1</td><td rowspan=1 colspan=1>2.00 x 10^$</td><td rowspan=1 colspan=1>TCID_/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Strain 1A</td><td rowspan=1 colspan=1>$1.26x 10}$</td><td rowspan=1 colspan=1>TCIDx/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type 60</td><td rowspan=1 colspan=1>5.00 x 107</td><td rowspan=1 colspan=1>TCIDyo/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>HMPV</td><td rowspan=1 colspan=1>CAN97-83 (CDC I5olate26583)</td><td rowspan=1 colspan=1>5.00 x 103}$</td><td rowspan=1 colspan=1>TCIDsc/mL</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 1, Adenoid 71</td><td rowspan=1 colspan=1>5.00 x 10^}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>4.17×10}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 7A</td><td rowspan=1 colspan=1>5.37×10^8}$</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>5.00 x 10}$</td><td rowspan=1 colspan=1>TCID_/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>~</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NL63</td><td rowspan=1 colspan=1>5.00 × 10}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>5.00 × 10^}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster virus</td><td rowspan=1 colspan=1>Isolate A</td><td rowspan=1 colspan=1>1.86 × 10^$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Measles virus</td><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>1.26× 10}$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>AD-169</td><td rowspan=1 colspan=1>9.55 x 10}$</td><td rowspan=1 colspan=1>TCIDso/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Epstein-Barr virus</td><td rowspan=1 colspan=1>B95-8</td><td rowspan=1 colspan=1>3.00 x 10}$</td><td rowspan=1 colspan=1>cp/ml</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>N/A (Zeptometrix)</td><td rowspan=1 colspan=1>7.57×10^4$</td><td rowspan=1 colspan=1>TCIDs/mL</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>Mumps virus</td><td rowspan=1 colspan=1>N/A (Cultured fromparotid swab)</td><td rowspan=1 colspan=1>16.36</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Herpes simplex virus</td><td rowspan=1 colspan=1>Mcintyre</td><td rowspan=1 colspan=1>1.45 X 1020</td><td rowspan=1 colspan=1>TCID3/mL</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

The original and modified xTAG RVP assays did not generate non-specific positive call for these viral strains with the following exceptions (where a contaminated sample is suspected in each instance since the result was observed in both the original and modified devices): Flu A H1 (Seasonal) demonstrated some signal for the run control near the cutoff (lambdoid DNA);

# Luminex.

Luminex Molecular Diagnostics

Parainfluenza 3 demonstrated a low-level influenza A signal; Enterovirus (Echo 13) and Enterovirus (Coxsackie B) showed an Adenovirus signal; and Adenovirus (Type 1, Adenoid 71) showed an H3 (but not influenza A matrix) signal.

For the reactivity study, the initial stocks were diluted to approximately 2x to 3x the LoD established for the two reference strains in the LoD study. At least three replicates per strain were evaluated starting from the extraction step with both original and modified xTAG RVP assays. Both were able to successfully detect all five H3 strains tested (see Table 8).

Table 8: Influenza A Subtype H3 Strains Tested in the Reactivity Study   

<table><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68</td></tr><tr><td rowspan=1 colspan=1>A2/Aichi2/68</td></tr><tr><td rowspan=1 colspan=1>A/Alice</td></tr><tr><td rowspan=1 colspan=1>MRC2</td></tr></table>

The following four additional strains were identified from the clinical sample data set in the accuracy study (Table 9).

Table 9: Influenza A Subtype H3 Strains Tested in the Accuracy Study   

<table><tr><td>A/District of Columbia/WRAIRO301/2010(H3N2)</td></tr><tr><td>A/Texas/NHRC0001/2011(H3N2)</td></tr><tr><td>A/South Carolina/AF2724/2011(H3N2)</td></tr><tr><td>A/lasi/47326/2010(H3N2)</td></tr></table>

Eight additional strains representing other cleared analytes were tested in the analytical reactivity study (Table 10). The initial stocks were diluted to approximately 3 times the LoD established for reference strains. Three replicates per strain were evaluated with both original and modified xTAG RVP assays. Both devices were able to successfully detect all eight strains tested.

Table 10: Additional Strains Tested in the Reactivity Study   

<table><tr><td rowspan=1 colspan=1>Flu A H1</td><td rowspan=1 colspan=1>A/New Caledonia/20/99(H1N1)</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>B/Malaysia/2506/04</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>AUS/A2/61</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>B WV/14617/&#x27;85</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>C-243</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>Type 39</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Type 8, strain Peru6-2003 B2</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Type 3</td></tr></table>

Luminex Molecular Diagnostics

# Competitive Inhibition Study

The combinations of analytes tested in the competitive inhibition study are listed in Table 11. Each analyte was tested at two different concentrations, High Positive (HP, approximately at 1.3 to 4-fold dilution of the original stock) and Low Positive (LP, approximately at 2 to 4 times LoD for that analyte). The results show that the modifications made to the device did not inhibit the detection of the competing analytes. The performance of the Original and Modified devices was equivalent. All expected positive calls were present.

Table 11. Analyte Combinations Tested in the Competitive Inhibition Study   

<table><tr><td rowspan=1 colspan=1>Count</td><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Concentration</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>RSV A, strain Long</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>RSV A, strain Long</td><td rowspan=1 colspan=1>HP</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>RSV B, strainWash/18537/62</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>RSV B, strainWash/18537/62</td><td rowspan=1 colspan=1>HP</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>Rhinovirus, Type54</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>Rhinovirus, Type54</td><td rowspan=1 colspan=1>HP</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>hMPV 5, Peru3-2003 B1</td><td rowspan=1 colspan=1>LP                1</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>hMPV 5, Peru3-2003 B1</td><td rowspan=1 colspan=1>HP</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>HP</td><td rowspan=1 colspan=1>Adenovirus, Type1</td><td rowspan=1 colspan=1>LP</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Flu A H3, strainA/Victoria/3/75</td><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>Adenovirus, Type1</td><td rowspan=1 colspan=1>HP</td></tr></table>

No differences between the modified and the original xTAGRVP were observed in reactivity, cross-reactivity or competitive inhibition studies.

# Clinical Comparison Studies (Accuracy)

The accuracy study evaluated the positive agreement and negative agreement between the original and modified xTAG RVP devices. Table 12 shows the list of analytes tested by both the original and modified xTAG RVP Assays.

Luminex Molecular Diagnostics

Table 12: Analytes Tested   

<table><tr><td rowspan=1 colspan=1>Human Influenza A</td></tr><tr><td rowspan=1 colspan=1>Human H1 seasonal subtype of Influenza A</td></tr><tr><td rowspan=1 colspan=1>Human H3 subtype of Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>RSV A</td></tr><tr><td rowspan=1 colspan=1>RSV B</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus / Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td></tr></table>

A total of 369 retrospectively collected left-over clinical samples (nasopharyngeal swabs) obtained primarily from the 2010-2011 influenza season were collected from 14 clinical sites in the United States and Canada. To preserve the confidentiality of the subjects, clinical specimens were individually numbered so the identity of the subject could not be readily ascertained by the investigator or any other individual associated with the study. Nucleic acid extraction was performed either by the clinical site or at Luminex Molecular Diagnostics (LMD), using one of the following methods: BioMerieux EasyMAG, BioMerieux MiniMAG or QIAGEN MinElute Viral Spin Kit, as directed in the instructions for use of the original device. Extracted samples were stored frozen at a temperature of $- 7 0 ^ { \circ } C$ until used in the study.

All Flu A matrix positive samples (158) from either the original or modified xTAG RVP device were bi-directionally sequenced for Flu A subtype H3. 132 of the 158 samples were found to be Flu A H3 sequence positive (see Table 13), leaving 26 samples that were Flu A H3 sequence negative. Four out of these 26 Flu A H3 sequencing negative samples were H1 positive by both the original RVP and the modified RVP assays. Three samples out of the 26 did not have adequate sample left over and therefore could not be sequenced. The remaining 19 samples $( 4 + 3 + 1 9 = 2 6 )$ were sequenced with an in-house designed set of 2009 H1N1pdm primers and the majority of these samples (13) were 2009 H1N1pdm positive.

<table><tr><td rowspan=1 colspan=2>EY</td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1>Sequencing POS for H3</td><td rowspan=1 colspan=1>132 Samples</td><td rowspan=2 colspan=1>Lower</td><td rowspan=2 colspan=1>Upper</td></tr><tr><td rowspan=1 colspan=1>Modified xTAG® RVP POS</td><td rowspan=1 colspan=1>121 Samples</td></tr><tr><td rowspan=1 colspan=1>Percent Positive Agreement(TP/TP+FN)</td><td rowspan=1 colspan=1>121/132=91.7%</td><td rowspan=1 colspan=1>87.82%</td><td rowspan=1 colspan=1>96.91%</td></tr></table>

Positive agreement and negative agreement for each analyte were evaluated between the original and modified xTAG RVP devices (see Table 14).

Luminex Molecular Diagnostics

Table 14: Clinical Comparison of Modified $x T A G ^ { \otimes }$ RVP and Original xTAG® RVP   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>PositivePercentAgreement(PPA)</td><td rowspan=1 colspan=1>Confidence Interval</td><td rowspan=1 colspan=1>NegativePercentAgreement(NPA)</td><td rowspan=1 colspan=1>Confidence Interval</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>98.09%(154/157)</td><td rowspan=1 colspan=1>94.52% - 99.60%</td><td rowspan=1 colspan=1>99.06%(210/212)</td><td rowspan=1 colspan=1>96.63% - 99.89%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>100%(4/4)</td><td rowspan=1 colspan=1>39.76% - 100.00%</td><td rowspan=1 colspan=1>100%(365/365)</td><td rowspan=1 colspan=1>98.99% - 100.00%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>100%(80/80)</td><td rowspan=1 colspan=1>95.49%  100.00%</td><td rowspan=1 colspan=1>85.47%(247/289)</td><td rowspan=1 colspan=1>80.87% - 89.32%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.43% - 100.00%</td><td rowspan=1 colspan=1>100%(339/339)</td><td rowspan=1 colspan=1>98.92% - 100.00%</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>100%(23/23)</td><td rowspan=1 colspan=1>85.18% - 100.00%</td><td rowspan=1 colspan=1>99.71%(345/346)</td><td rowspan=1 colspan=1>98.40% - 99.99%</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>96.30%(26/27)</td><td rowspan=1 colspan=1>81.03% - 99.91%</td><td rowspan=1 colspan=1>100%(342/342)</td><td rowspan=1 colspan=1>98.93% - 100.00%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1</td><td rowspan=1 colspan=1>100%(6/6)</td><td rowspan=1 colspan=1>54.07% - 100.00%</td><td rowspan=1 colspan=1>99.72%(362/363)</td><td rowspan=1 colspan=1>98.47% - 99.99%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2</td><td rowspan=1 colspan=1>100%(8/8)</td><td rowspan=1 colspan=1>63.06% - 100.00%</td><td rowspan=1 colspan=1>99.72%(360/361)</td><td rowspan=1 colspan=1>98.47% - 99.99%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>100%(24/24)</td><td rowspan=1 colspan=1>85.75% - 100.00%</td><td rowspan=1 colspan=1>100%(345/345)</td><td rowspan=1 colspan=1>98.94% - 100.00%</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>96.43%(27/28)</td><td rowspan=1 colspan=1>81.65% - 99.91%</td><td rowspan=1 colspan=1>100%(341/341)</td><td rowspan=1 colspan=1>98.92% - 100.00%</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus</td><td rowspan=1 colspan=1>92.16%(47/51)</td><td rowspan=1 colspan=1>81.12% - 97.82%</td><td rowspan=1 colspan=1>99.69%(317/318)</td><td rowspan=1 colspan=1>98.26% - 99.99%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>100%(5/5)</td><td rowspan=1 colspan=1>47.82% - 100.00%</td><td rowspan=1 colspan=1>100%(364/364)</td><td rowspan=1 colspan=1>98.99% - 100.00%</td></tr></table>

Clinical Cut-off: Not applicable.

# Expected values/ reference range: Not applicable.

Luminex Molecular Diagnostics, Inc. c/o Ms. Lubna Syed Director, Regulatory Affairs 439 University Avenue, Suite 900 Toronto, Ontario, M5G 1Y8, CANADA

Re: k112781 Trade Name: xTAG®Respiratory Viral Panel (RVP) Regulation Number: 21 CFR 866.3980 Regulation Name: Respiratory Viral Panel Multiplex Nucleic Acid Assay Regulatory Class: Class II Product Code: OCC, OEM, OEP, NSU, JJH Dated: December 19, 2011 Received: December 22, 2011

Dear Ms. Syed:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class II (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter

Page 2 - Lubna Syed

will allow you to begin marketing your device as described in your Section $5 1 0 ( \mathbf { k } )$ premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours, lve Sat for

Sally_ A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

# Indications for Use

510(k) Number (if known): k112781

Device Name: xTAG® Respiratory Viral Panel (RVP)

The $\mathbf { x T A G } ^ { \otimes }$ Respiratory Viral Panel (RVP) is a qualitative nucleic acid multiplex test intended for the simultaneous detection and identification of multiple respiratory virus nucleic acids in nasopharyngeal swabs from individuals suspected of respiratory tract infections. The following v types n subtype denusRVP:Infuza Infuza  Respy Syncytial Virus subtype A, Respiratory Syncytial Virus subtype B, Parainfluenza 1, Parainfluenza 2, and Parainfluenza 3 virus, Human Metapneumovirus, Rhinovirus, and Adenovirus. The detection and identification of specific viral nucleic acids from individuals exhibiting signs and symptoms of respiratory infection aids in the diagnosis of respiratory viral infection if used in conjunction with other clinical and laboratory findings.

xTAG RVP can also differentiate the hemagglutinin (HA) gene of some Influenza A subtypes H1 and H3 strains. Differentiation of Influenza A HA subtypes is based on both a positive result for the Influenza A matrix gene and an accompanying positive result for the Influenza A HA subtype H1 (circulating prior to the emergence of 2009 H1N1pdm) or Influenza A HA subtype H3. This device cannot differentiate the Influenza A HA subtype 2009 H1N1pdm by design, and may not be able to differentiate potential newly emerging Influenza A HA subtypes.

Positive results do not rule out bacterial infection, or co-infection with other viruses. The agent detected may not be the definite cause of disease. The use of additional laboratory testing (.. bacterial culture, immunofluorescence, radiography) and clinical presentation must be taken into consideration in order to obtain the final diagnosis of respiratory viral infection.

Negative results do not preclude respiratory virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

The RVP assay cannot adequately detect Adenovirus species C, or serotypes 7a and 41. It is recommended that specimens found to be negative for Adenovirus after examination using RVP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture). The RVP primers for detection of rhinovirus cross-react with enterovirus. A rhinovirus reactive result should be confirmed by an alternate method (e.g. cell culture).

Performance characteristics for Influenza A virus were established when Influenza A HA subtype H3, subtype H1 (prior to the emergence of 2009 H1N1pdm), and when subtype 2009 H1N1pdm were the predominant Influenza A in circulation. When other Influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to a state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

# (PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE IF NEEDED)

Concurrence of CDRH, Office of In Vitro Diagnostic Device Evaluation and Safety (OIVD)

Tana Felab lerT

Office of In Vitro Diagnostic Device Evaluation and Safety

510(k) K //2781