# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

Specific target RNA sequences for the RNA dependant RNA polymerase gene of hMPV.

# D. Type of Test:

Qualitative real-time reverse transcription-polymerase chain reaction (RT-PCR) for determination of human metapneumovirus (hMPV) RNA in nasal swabs and nasopharyngeal swabs using the nucleic acid isolation NucliSENS $^ \mathrm { \textregistered }$ easyMAG™ System (bioMérieux) and the amplification and detection on the ABI 7500 Fast Dx Instrument with Software version 1.4.

E. Applicant: Quidel Corporation   
F. Proprietary and Established Names: Quidel Molecular hMPV Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code: OEM

4. Panel:

Microbiology

# H. Intended Use:

1. Intended use(s):

The Quidel Molecular hMPV assay is a multiplex Real Time RT-PCR assay for the in vitro qualitative detection of human metapneumovirus RNA in nasal and nasopharyngeal swab specimens from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of human metapneumovirus infections in humans in conjunction with clinical and epidemiological risk factors. This test is not intended to differentiate the four genetic sub-lineages of hMPV.

Negative results do not preclude hMPV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

2. Indication(s) for use: Same as intended use

3. Special conditions for use statement(s): Not Applicable

NucliSENS easyMAG automated extraction platform

Applied Biosystems 7500 Fast Dx platform (software version 1.4)

# I. Device Description:

The Quidel Molecular hMPV Assay detects hMPV viral nucleic acids that have been extracted from a patient sample using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ automated extraction platform. A multiplex RT-PCR reaction is carried out under optimized conditions in a single tube generating amplicons for each of the target viruses present in the sample. This reaction is performed utilizing the Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx platform. Identification of hMPV occurs by the use of target specific primers and a fluorescent-labeled probe that hybridizes to a highly conserved region in the RNA dependent RNA polymerase gene of hMPV.

The following is a summary of the procedure:

1. Sample Collection: Obtain Nasal swab and nasopharyngeal swab specimens using standard techniques from symptomatic patients. These specimens are transported, stored, and processed according to established laboratory procedures.

2. Nucleic Acid Extraction: Extract Nucleic Acids from the specimens with the NucliSENS easyMAG System following the manufacturer’s instructions and using the appropriate reagents. Prior to the extraction procedure add $2 0 ~ \mu \mathrm { L }$ of the Process Control (PRC) to each $1 8 0 ~ \mu \mathrm { L }$ aliquot of specimen. The PRC serves to monitor inhibitors in the extracted specimen, assures that adequate amplification has taken place and confirms that the nucleic acid extraction was sufficient.

3. Rehydration of Master Mix: Rehydrate the lyophilized Master Mix using the Rehydration Solution. The Master Mix contains oligonucleotide primers, fluorophore and quencher-labeled probes targeting highly conserved regions of hMPV as well as the PRC sequence. The primers are complementary to highly specific and conserved regions in the genome of these viruses. The probes are dual labeled with a reporter dye attached to the $5 '$ end and a quencher attached to the $3 ^ { \circ }$ end.

4. Nucleic Acid Amplification and Detection: Add $1 5 ~ \mu \mathrm { L }$ of the rehydrated Master Mix to each reaction tube or plate well. $5 \mu \mathrm { L }$ of extracted nucleic acids (specimen with PRC) is then added to the plate well. Place the plate into the ABI 7500 FastDx. Once the plate is added to the instrument, the assay protocol is initiated. This protocol initiates reverse transcription of the RNA targets generating complementary DNA, and the subsequent amplification of the target sequences occurs. The Quidel Molecular hMPV assay is based on TaqMan $^ \mathrm { \textregistered }$ chemistry, and uses an enzyme with reverse transcriptase, DNA polymerase, and $5 ' - 3 '$ exonuclease activities. During DNA amplification, this enzyme cleaves the probe bound to the complementary DNA sequence, separating the quencher dye from the reporter dye. This step generates an increase in fluorescent signal upon excitation by a light source of the appropriate wavelength. With each cycle, additional dye molecules are separated from their quenchers resulting in additional signal. If sufficient fluorescence is achieved by 35 cycles during the data collection stage of amplification, the sample is reported as positive for the detected target sequences.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Gen-Probe Prodesse hMPV+

2. Predicate 510(k) number(s): k082688

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=4>Subject Device and Comparator Device Comparison</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=2>Subject DeviceQuidel Molecular hMPV Assay</td><td rowspan=1 colspan=1>Predicate DeviceProdesse ProFlu+</td></tr><tr><td rowspan=3 colspan=1>Intended Use</td><td rowspan=3 colspan=2>The Quidel Molecular hMPVassay is a multiplex Real TimeRT-PCR assay for the in vitroqualitative detection of humanmetapneumovirus RNA innasal and nasopharyngeal swabspecimens from patients withsigns and symptoms ofrespiratory infection. This testis intended for use as an aid inthe differential diagnosis ofhuman metapneumovirusinfections in humans inconjunction with clinical andepidemiological risk factors.This test is not intended todifferentiate the four geneticsub-lineages of hMPV.Negative results do notpreclude hMPV infection andshould not be used as the solebasis for diagnosis, treatmentor other patient managementdecisions.</td><td rowspan=3 colspan=1>The Pro hMPV+ Assay is aReal Time RT-PCR in vitrodiagnostic test for thequalitative detection of humanMetapneumovirus (hMPV)nucleic acid isolated andpurified from nasopharyngealswab (NP) specimens obtainedfrom individuals exhibitingsigns and symptoms of acuterespiratory infection. This assaytargets a highly conservedregion of the Nucleocapsid geneof hMPV. The detection ofhMPV nucleic acid fromsymptomatic patients aids in thediagnosis of human respiratoryhMPV infection if used inconjunction with other clinicaland laboratory findings. Thistest is not intended todifferentiate the four geneticsub-lineages of hMPV.Negative results do not precludehMPV infection and should notbe used as the sole basis fordiagnosis, treatment or othermanagement decisions.</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>differe</td></tr><tr><td rowspan=1 colspan=1>Assay Target</td><td rowspan=1 colspan=2>hMPV</td><td rowspan=1 colspan=1>hMPV</td></tr><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=2>nasal swab, nasopharyngeal swab</td><td rowspan=1 colspan=1>nasopharyngeal swab</td></tr><tr><td rowspan=1 colspan=1>Extraction Methods</td><td rowspan=1 colspan=2>bioMérieux easyMAGAutomated Magnetic ExtractionReagents</td><td rowspan=1 colspan=1>Roche MagNA Pure LC TotalNucleic Acid Isolation Kit or thebioMérieux easyMAG AutomatedMagnetic Extraction Reagents</td></tr><tr><td rowspan=1 colspan=1>Assay Methodology</td><td rowspan=1 colspan=2>PCR-based system for detectingthe presence or absence of viralRNA in clinical specimens</td><td rowspan=1 colspan=1>PCR-based system for detectingthe presence or absence of viralRNA in clinical specimens</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Subject Device and Comparator Device Comparison</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Subject DeviceQuidel Molecular hMPV Assay</td><td rowspan=1 colspan=1>Predicate DeviceProdesse ProFlu+</td></tr><tr><td rowspan=1 colspan=1>DetectionTechniques/Platform</td><td rowspan=1 colspan=1>Multiplex assay using differentreporter dyes for each target onthe ABI 7500 Fast Dx</td><td rowspan=1 colspan=1>Multiplex assay using differentreporter dyes for each target on theCepheid SmartCycler</td></tr><tr><td rowspan=1 colspan=1>Viral Targets</td><td rowspan=1 colspan=1>RNA polymerase gene</td><td rowspan=1 colspan=1>Nucleocapsid</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 2009).

Guidance for Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff; Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable (April 25, 2006).

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. Detection of hMPV and the Process Control (PRC) is based on three processes: nucleic acid isolation, reverse transcription, and real time PCR amplification/detection. Human respiratory specimens (nasal swabs and nasopharyngeal swabs) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. After initial reverse transcription of RNA into complementary DNA (cDNA), amplification proceeds during which the probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This generates an increase in fluorescent signal upon excitation from a light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time. Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance: a. Precision/Reproducibility:

The reproducibility of the Quidel Molecular hMPV assay was evaluated at three laboratory sites. The reproducibility panel and controls were tested at each site by two operators for five days in triplicate (2 operators X 5 days X triplicate testing X 3 sites $= 9 0$ results per sample). The panels and controls were extracted using the bioMérieux easyMAG system and tested on the ABI 7500Fast Dx.

The reproducibility panel was composed of four simulated samples made by diluting hMPV-A1 into negative nasal matrix. The panel included a medium positive (5x LoD) hMPV sample ( $2 . 6 5 \mathrm { E } { + } 0 2$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ), a low positive $2 \mathbf { x }$ LoD) hMPV sample $\mathrm { 1 . 0 6 E ^ { + 0 2 } }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ), a high negative (0.3X LoD) hMPV sample ( $1 . 5 9 \mathrm { E } { + 0 1 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ) and a negative. The results of the analysis of reproducibility are summarized below.

<table><tr><td rowspan=1 colspan=11>Quidel Molecular hMPV Reproducibility</td></tr><tr><td rowspan=2 colspan=1>PanelMember ID</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalResults</td></tr><tr><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>Results</td><td rowspan=1 colspan=1>AVECt</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>hMPV HighNegative(1.59E+01TCID50/mL)</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>33.36*</td><td rowspan=1 colspan=1>5.08</td><td rowspan=1 colspan=1>10/30</td><td rowspan=1 colspan=1>32.76*</td><td rowspan=1 colspan=1>3.38</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>20/90</td></tr><tr><td rowspan=1 colspan=1>hMPV LowPositive(1.06E+02TCID50/mL)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.73</td><td rowspan=1 colspan=1>5.28</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>28.18</td><td rowspan=1 colspan=1>8.42</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>30.18</td><td rowspan=1 colspan=1>4.89</td><td rowspan=1 colspan=1>89/90</td></tr><tr><td rowspan=1 colspan=1>hMPV MediumPositive(2.65E+02TCID50/mL)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>25.09</td><td rowspan=1 colspan=1>6.65</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>25.18</td><td rowspan=1 colspan=1>7.63</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>26.58</td><td rowspan=1 colspan=1>5.68</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>Negative Sample</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td></tr><tr><td rowspan=1 colspan=1>hMPV PositiveControl</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.37</td><td rowspan=1 colspan=1>3.88</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.48</td><td rowspan=1 colspan=1>4.35</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>18.60</td><td rowspan=1 colspan=1>3.08</td><td rowspan=1 colspan=1>90/90</td></tr><tr><td rowspan=1 colspan=1>Negative Control</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0/30</td></tr></table>

\* CV of positive results

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Freeze-Thaw Equivalency

The impact of using fresh clinical samples versus frozen clinical specimens on the Quidel Molecular hMPV assay ability to detect hMPV was evaluated. Briefly, 220 clinical specimens were aliquoted and processed fresh $( < 7 2 -$ hours post collection) and frozen (a minimum of 7-days storage at $- 7 0 ^ { \circ } \mathrm { C }$ ). Twelve (12) samples were discarded from analysis due to invalid results. The invalid rate of the fresh samples was 7/220 $( 3 \% )$ and the invalid rate of the frozen samples was $1 1 / 2 2 0 ( 5 \% )$ . Six of the invalids overlapped between fresh and frozen. There were two hMPV positive samples that did not overlap between fresh and frozen in the comparison study. The positive samples evaluated in the study spanned virus concentrations detected from 11 Ct to 30 Ct. The results are summarized in the $2 \mathbf { x } 2$ table below and indicate that the use of fresh specimens and frozen specimens, as defined above, are equivalent.

<table><tr><td rowspan=1 colspan=4>Quidel Molecular hMPV Assay  Fresh vs. Frozen Analysis</td></tr><tr><td rowspan=1 colspan=1>Nasopharyngeal Swab</td><td rowspan=1 colspan=3>Fresh Specimen</td></tr><tr><td rowspan=1 colspan=1>Frozen Specimen</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>180</td><td rowspan=1 colspan=1>181</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>208</td></tr><tr><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=3>96.3%</td></tr><tr><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=3>99.4%</td></tr></table>

# Kit Stability

An evaluation of real-time stability using 3 lots of the Quidel Molecular hMPV kits at $2 ^ { \circ } C \cdot 8 ^ { \circ } C$ and room temperature was done over the course of 73 days, with additional real time results to be collected for 460 days (15 months). The following information summarizes the acceptance criteria; furthermore these criteria will also be used throughout the stability study:

1. Quidel Molecular hMPV kit stability demonstrates 15 months real time stability at $2 ^ { \circ } \mathrm { C }$ to $8 \mathrm { { } ^ { \circ } C }$ .   
2. Quidel Molecular hMPV kit stability demonstrates 15 months equivalent accelerated stability at $2 ^ { \circ } \mathrm { C }$ to $8 \mathrm { { } ^ { \circ } C }$ .   
3. Quidel Molecular hMPV kit stability demonstrates 15 months real time stability at room temperature. (RT)   
4. hMPV tests positive at each time point tested, 3x, 100x and 1000x LoD $( \geq 5$ cycles and ${ \leq } 3 5$ cycles for $7 5 0 0 \mathrm { D x }$ ).   
5. The NTCs test negative at each time point tested (undetermined).

No significant failure of the assay was detected at any of the experimental conditions to date (real time stability data collected to day 73) when compared to the $\scriptstyle { \mathrm { t = 0 } }$ time point. This was calculated by assuming a maximum $C V \%$ of $5 \%$ and calculating the $9 5 \%$ upper confidence limit (UCL) for each $\scriptstyle { \mathrm { t = 0 } }$ sample. Also, all NTCs tested negative at each time point tested.

# Master Mix Stability after Rehydration

An evaluation of master mix stability after rehydration was carried out over a 5 day period using various storage conditions, including ${ } _ { - 2 0 } \mathrm { { \circ C } }$ , $4 \mathrm { { } ^ { \circ } C }$ , and $2 5 \mathrm { { ^ \circ C } }$ (room temperature). Mock specimens at 100X, 10X, and 3X LoD were used in the analysis. Results show that the rehydrated master mix can be stored for up to 24-hours at room temperature without any significant affect on the $\mathrm { C t }$ values for the hMPV or the PRC. The data demonstrates that for times longer than 24-hours the rehydrated master mix can be stored at $4 \mathrm { { } ^ { \circ } C }$ or ${ } _ { - 2 0 } \mathrm { { \circ C } }$ for up to 5 days.

# Extracted Specimen Stability

An evaluation of the stability of extracted specimens was carried out using the easyMAG extraction system followed by real-time RT-PCR analysis. Parallel aliquots of samples that were hMPV A (3X LoD) and hMPV B (3X LoD) positive were used for this study. Samples were analyzed at time 0 (immediately after extraction), after 2h at room temperature, after 8h at $2 { - } 8 ^ { \circ } \mathrm { C }$ , after 1 month at $- 2 0 ^ { \circ } \mathrm { C }$ , and after 1 month at ${ } _ { - 2 0 } \mathrm { { \circ C } }$ with up to three freeze thaw cycles. There was no indication of degradation of the extracted specimen in the conditions tested.

# d. Detection limit:

The limit of detection (LoD) of the Quidel Molecular hMPV assay on the ABI 7500 Fast Dx instrument was determined using limiting dilutions of recultured and re-titered viral stocks. The following isolates were used for the analysis and represent both lineages and subtypes: A1 Italy #6621 MK/2 LLCMK2/25, A2 Vanderbilt TN 94-49, B1 Italy #4702 LLCMK2/18, and B2 Italy #3817 LLCMK2/30.

Each of the viral stocks were spiked in negative nasal matrix and serially diluted. The replicate dilutions were extracted using the NucliSENS easyMAG. In total there were 21 extraction runs per dilution level tested to establish the LoD. Each extraction was analyzed on the ABI 7500 Fast Dx and the LoD was established as the lowest concentration that yielded positive result rate of $2 9 5 \%$ . The LoD established for hMPV A1, A2, B1, and B2 are summarized below.

<table><tr><td rowspan=1 colspan=1>hMPVType</td><td rowspan=1 colspan=1>Sub-Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>A1</td><td rowspan=1 colspan=1>Italy #6621 MK/2LLCMK2/25</td><td rowspan=1 colspan=1>5.29E+01</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>A2</td><td rowspan=1 colspan=1>Vanderbilt TN94-49</td><td rowspan=1 colspan=1>1.39E+01</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>B1</td><td rowspan=1 colspan=1>Italy #4702LLCMK2/18</td><td rowspan=1 colspan=1>3.15E+00</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>Italy #3817LLCMK2/30</td><td rowspan=1 colspan=1>1.07E+01</td></tr></table>

<table><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=3>TCID50/mL</td><td rowspan=1 colspan=1>hMPV A2</td><td rowspan=1 colspan=3>TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>9.76E+01</td><td rowspan=1 colspan=1>5.29E+01</td><td rowspan=1 colspan=1>2.65E+01</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>4.14E+01</td><td rowspan=1 colspan=1>1.39E+01</td><td rowspan=1 colspan=1>4.14E+00</td></tr><tr><td rowspan=1 colspan=1>Replicate-1</td><td rowspan=1 colspan=1>27.5898</td><td rowspan=1 colspan=1>29.5677</td><td rowspan=1 colspan=1>34.0142</td><td rowspan=1 colspan=1>Replicate-1</td><td rowspan=1 colspan=1>19.2623</td><td rowspan=1 colspan=1>22.7783</td><td rowspan=1 colspan=1>28.8796</td></tr><tr><td rowspan=1 colspan=1>Replicate-2</td><td rowspan=1 colspan=1>26.751</td><td rowspan=1 colspan=1>29.7919</td><td rowspan=1 colspan=1>33.7657</td><td rowspan=1 colspan=1>Replicate-2</td><td rowspan=1 colspan=1>18.5254</td><td rowspan=1 colspan=1>20.0743</td><td rowspan=1 colspan=1>Und.</td></tr><tr><td rowspan=1 colspan=1>Replicate-3</td><td rowspan=1 colspan=1>27.3281</td><td rowspan=1 colspan=1>28.0972</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-3</td><td rowspan=1 colspan=1>17.7657</td><td rowspan=1 colspan=1>22.3149</td><td rowspan=1 colspan=1>28.9211</td></tr><tr><td rowspan=1 colspan=1>Replicate-4</td><td rowspan=1 colspan=1>27.9529</td><td rowspan=1 colspan=1>28.0159</td><td rowspan=1 colspan=1>33.6861</td><td rowspan=1 colspan=1>Replicate-4</td><td rowspan=1 colspan=1>17.8513</td><td rowspan=1 colspan=1>23.882</td><td rowspan=1 colspan=1>22.644</td></tr><tr><td rowspan=1 colspan=1>Replicate-5</td><td rowspan=1 colspan=1>27.952</td><td rowspan=1 colspan=1>29.6624</td><td rowspan=1 colspan=1>34.2712</td><td rowspan=1 colspan=1>Replicate-5</td><td rowspan=1 colspan=1>18.2482</td><td rowspan=1 colspan=1>24.1073</td><td rowspan=1 colspan=1>33.8105</td></tr><tr><td rowspan=1 colspan=1>Replicate-6</td><td rowspan=1 colspan=1>27.3933</td><td rowspan=1 colspan=1>29.3138</td><td rowspan=1 colspan=1>32.1021</td><td rowspan=1 colspan=1>Replicate-6</td><td rowspan=1 colspan=1>18.3644</td><td rowspan=1 colspan=1>23.7096</td><td rowspan=1 colspan=1>33.5286</td></tr><tr><td rowspan=1 colspan=1>Replicate-7</td><td rowspan=1 colspan=1>26.1488</td><td rowspan=1 colspan=1>28.1323</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-7</td><td rowspan=1 colspan=1>18.3857</td><td rowspan=1 colspan=1>20.2891</td><td rowspan=1 colspan=1>34.5839</td></tr><tr><td rowspan=1 colspan=1>Replicate-8</td><td rowspan=1 colspan=1>27.2575</td><td rowspan=1 colspan=1>29.8472</td><td rowspan=1 colspan=1>34.2665</td><td rowspan=1 colspan=1>Replicate-8</td><td rowspan=1 colspan=1>18.1594</td><td rowspan=1 colspan=1>23.2851</td><td rowspan=1 colspan=1>29.7116</td></tr><tr><td rowspan=1 colspan=1>Replicate-9</td><td rowspan=1 colspan=1>27.05</td><td rowspan=1 colspan=1>27.7465</td><td rowspan=1 colspan=1>34.7051</td><td rowspan=1 colspan=1>Replicate-9</td><td rowspan=1 colspan=1>19.0474</td><td rowspan=1 colspan=1>20.1477</td><td rowspan=1 colspan=1>28.5702</td></tr><tr><td rowspan=1 colspan=1>Replicate-10</td><td rowspan=1 colspan=1>27.8662</td><td rowspan=1 colspan=1>29.1535</td><td rowspan=1 colspan=1>33.9066</td><td rowspan=1 colspan=1>Replicate-10</td><td rowspan=1 colspan=1>18.2692</td><td rowspan=1 colspan=1>22.5548</td><td rowspan=1 colspan=1>25.644</td></tr><tr><td rowspan=1 colspan=1>Replicate-11</td><td rowspan=1 colspan=1>26.6194</td><td rowspan=1 colspan=1>30.3058</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-11</td><td rowspan=1 colspan=1>17.624</td><td rowspan=1 colspan=1>20.8038</td><td rowspan=1 colspan=1>31.4981</td></tr><tr><td rowspan=1 colspan=1>Replicate-12</td><td rowspan=1 colspan=1>27.2965</td><td rowspan=1 colspan=1>29.4415</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-12</td><td rowspan=1 colspan=1>17.7512</td><td rowspan=1 colspan=1>19.8295</td><td rowspan=1 colspan=1>33.0868</td></tr><tr><td rowspan=1 colspan=1>Replicate-13</td><td rowspan=1 colspan=1>25.976</td><td rowspan=1 colspan=1>33.4114</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-13</td><td rowspan=1 colspan=1>18.7787</td><td rowspan=1 colspan=1>26.462</td><td rowspan=1 colspan=1>23.3888</td></tr><tr><td rowspan=1 colspan=1>Replicate-14</td><td rowspan=1 colspan=1>24.6651</td><td rowspan=1 colspan=1>28.5803</td><td rowspan=1 colspan=1>33.6673</td><td rowspan=1 colspan=1>Replicate-14</td><td rowspan=1 colspan=1>17.9668</td><td rowspan=1 colspan=1>22.0258</td><td rowspan=1 colspan=1>25.1626</td></tr><tr><td rowspan=1 colspan=1>Replicate-15</td><td rowspan=1 colspan=1>26.5171</td><td rowspan=1 colspan=1>26.5084</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-15</td><td rowspan=1 colspan=1>17.6512</td><td rowspan=1 colspan=1>21.6152</td><td rowspan=1 colspan=1>24.8982</td></tr><tr><td rowspan=1 colspan=1>Replicate-16</td><td rowspan=1 colspan=1>26.9674</td><td rowspan=1 colspan=1>26.9903</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-16</td><td rowspan=1 colspan=1>18.3541</td><td rowspan=1 colspan=1>20.1346</td><td rowspan=1 colspan=1>Und.</td></tr><tr><td rowspan=1 colspan=1>Replicate-17</td><td rowspan=1 colspan=1>27.9973</td><td rowspan=1 colspan=1>29.0467</td><td rowspan=1 colspan=1>31.7243</td><td rowspan=1 colspan=1>Replicate-17</td><td rowspan=1 colspan=1>18.0115</td><td rowspan=1 colspan=1>24.3418</td><td rowspan=1 colspan=1>Und.</td></tr><tr><td rowspan=1 colspan=1>Replicate-18</td><td rowspan=1 colspan=1>27.0961</td><td rowspan=1 colspan=1>27.772</td><td rowspan=1 colspan=1>32.9554</td><td rowspan=1 colspan=1>Replicate-18</td><td rowspan=1 colspan=1>17.8084</td><td rowspan=1 colspan=1>22.4488</td><td rowspan=1 colspan=1>26.1243</td></tr><tr><td rowspan=1 colspan=1>Replicate-19</td><td rowspan=1 colspan=1>26.7276</td><td rowspan=1 colspan=1>29.9229</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Replicate-19</td><td rowspan=1 colspan=1>18.331</td><td rowspan=1 colspan=1>20.4704</td><td rowspan=1 colspan=1>30.8669</td></tr><tr><td rowspan=1 colspan=1>Replicate-20</td><td rowspan=1 colspan=1>27.7816</td><td rowspan=1 colspan=1>28.555</td><td rowspan=1 colspan=1>33.8992</td><td rowspan=1 colspan=1>Replicate-20</td><td rowspan=1 colspan=1>18.1839</td><td rowspan=1 colspan=1>19.8139</td><td rowspan=1 colspan=1>26.2264</td></tr><tr><td rowspan=1 colspan=1>Replicate-21</td><td rowspan=1 colspan=1>25.9937</td><td rowspan=1 colspan=1>28.1505</td><td rowspan=1 colspan=1>34.0623</td><td rowspan=1 colspan=1>Replicate-21</td><td rowspan=1 colspan=1>18.0977</td><td rowspan=1 colspan=1>19.3523</td><td rowspan=1 colspan=1>25.6508</td></tr><tr><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>NTC</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Und.</td><td rowspan=1 colspan=1>Und.</td></tr><tr><td rowspan=1 colspan=1># (35&gt;Ct&gt;5)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>13/21</td><td rowspan=1 colspan=1># (35&gt;Ct&gt;5)</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>percentage</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>61.9%</td><td rowspan=1 colspan=1>percentage</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>86%</td></tr><tr><td rowspan=1 colspan=1>average Ct</td><td rowspan=1 colspan=1>26.996543</td><td rowspan=1 colspan=1>28.95301</td><td rowspan=1 colspan=1>33.839688</td><td rowspan=1 colspan=1>average Ct</td><td rowspan=1 colspan=1>18.21131</td><td rowspan=1 colspan=1>21.925771</td><td rowspan=1 colspan=1>28.510911</td></tr><tr><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>0.8276129</td><td rowspan=1 colspan=1>1.4379573</td><td rowspan=1 colspan=1>0.7746443</td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>0.4355696</td><td rowspan=1 colspan=1>1.9106898</td><td rowspan=1 colspan=1>3.7537431</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td colspan="1" rowspan="1">hMPV B1</td><td colspan="3" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">hMPV B2</td><td colspan="3" rowspan="1">TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">1.05E+01</td><td colspan="1" rowspan="1">3.15E+00</td><td colspan="1" rowspan="1">2.36E+00</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">3.21E+01</td><td colspan="1" rowspan="1">1.07E+01</td><td colspan="1" rowspan="1">3.21E+00</td></tr><tr><td colspan="1" rowspan="1">Replicate-1</td><td colspan="1" rowspan="1">22.7485</td><td colspan="1" rowspan="1">29.1931</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-1</td><td colspan="1" rowspan="1">19.849</td><td colspan="1" rowspan="1">25.1686</td><td colspan="1" rowspan="1">31.8105</td></tr><tr><td colspan="1" rowspan="1">Replicate-2</td><td colspan="1" rowspan="1">23.9339</td><td colspan="1" rowspan="1">30.8059</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-2</td><td colspan="1" rowspan="1">21.5502</td><td colspan="1" rowspan="1">25.777</td><td colspan="1" rowspan="1">32.6886</td></tr><tr><td colspan="1" rowspan="1">Replicate-3</td><td colspan="1" rowspan="1">24.0032</td><td colspan="1" rowspan="1">30.3606</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-3</td><td colspan="1" rowspan="1">21.3323</td><td colspan="1" rowspan="1">25.6818</td><td colspan="1" rowspan="1">31.2016</td></tr><tr><td colspan="1" rowspan="1">Replicate-4</td><td colspan="1" rowspan="1">24.5789</td><td colspan="1" rowspan="1">28.9146</td><td colspan="1" rowspan="1">32.9987</td><td colspan="1" rowspan="1">Replicate-4</td><td colspan="1" rowspan="1">20.6455</td><td colspan="1" rowspan="1">25.4246</td><td colspan="1" rowspan="1">34.2754</td></tr><tr><td colspan="1" rowspan="1">Replicate-5</td><td colspan="1" rowspan="1">25.8666</td><td colspan="1" rowspan="1">28.8763</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-5</td><td colspan="1" rowspan="1">19.425</td><td colspan="1" rowspan="1">27.2705</td><td colspan="1" rowspan="1">31.7572</td></tr><tr><td colspan="1" rowspan="1">Replicate-6</td><td colspan="1" rowspan="1">24.5199</td><td colspan="1" rowspan="1">27.4292</td><td colspan="1" rowspan="1">31.2083</td><td colspan="1" rowspan="1">Replicate-6</td><td colspan="1" rowspan="1">20.9775</td><td colspan="1" rowspan="1">25.3866</td><td colspan="1" rowspan="1">30.7778</td></tr><tr><td colspan="1" rowspan="1">Replicate-7</td><td colspan="1" rowspan="1">23.11</td><td colspan="1" rowspan="1">29.7065</td><td colspan="1" rowspan="1">27.3465</td><td colspan="1" rowspan="1">Replicate-7</td><td colspan="1" rowspan="1">20.8207</td><td colspan="1" rowspan="1">24.3929</td><td colspan="1" rowspan="1">31.0495</td></tr><tr><td colspan="1" rowspan="1">Replicate-8</td><td colspan="1" rowspan="1">24.3543</td><td colspan="1" rowspan="1">26.5085</td><td colspan="1" rowspan="1">32.5957</td><td colspan="1" rowspan="1">Replicate-8</td><td colspan="1" rowspan="1">20.9963</td><td colspan="1" rowspan="1">24.7287</td><td colspan="1" rowspan="1">33.045</td></tr><tr><td colspan="1" rowspan="1">Replicate-9</td><td colspan="1" rowspan="1">21.7705</td><td colspan="1" rowspan="1">30.205</td><td colspan="1" rowspan="1">34.5152</td><td colspan="1" rowspan="1">Replicate-9</td><td colspan="1" rowspan="1">19.6393</td><td colspan="1" rowspan="1">24.6584</td><td colspan="1" rowspan="1">31.4825</td></tr><tr><td colspan="1" rowspan="1">Replicate-10</td><td colspan="1" rowspan="1">24.3884</td><td colspan="1" rowspan="1">28.6596</td><td colspan="1" rowspan="1">30.3947</td><td colspan="1" rowspan="1">Replicate-10</td><td colspan="1" rowspan="1">20.4821</td><td colspan="1" rowspan="1">25.6951</td><td colspan="1" rowspan="1">34.2537</td></tr><tr><td colspan="1" rowspan="1">Replicate-11</td><td colspan="1" rowspan="1">24.6276</td><td colspan="1" rowspan="1">27.079</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-11</td><td colspan="1" rowspan="1">19.9232</td><td colspan="1" rowspan="1">18.9505</td><td colspan="1" rowspan="1">30.3872</td></tr><tr><td colspan="1" rowspan="1">Replicate-12</td><td colspan="1" rowspan="1">24.2194</td><td colspan="1" rowspan="1">29.6216</td><td colspan="1" rowspan="1">33.2577</td><td colspan="1" rowspan="1">Replicate-12</td><td colspan="1" rowspan="1">20.4557</td><td colspan="1" rowspan="1">22.0397</td><td colspan="1" rowspan="1">28.6635</td></tr><tr><td colspan="1" rowspan="1">Replicate-13</td><td colspan="1" rowspan="1">24.4698</td><td colspan="1" rowspan="1">28.6556</td><td colspan="1" rowspan="1">30.0068</td><td colspan="1" rowspan="1">Replicate-13</td><td colspan="1" rowspan="1">17.9482</td><td colspan="1" rowspan="1">25.1784</td><td colspan="1" rowspan="1">Und.</td></tr><tr><td colspan="1" rowspan="1">Replicate-14</td><td colspan="1" rowspan="1">24.6023</td><td colspan="1" rowspan="1">29.9063</td><td colspan="1" rowspan="1">31.1036</td><td colspan="1" rowspan="1">Replicate-14</td><td colspan="1" rowspan="1">20.6474</td><td colspan="1" rowspan="1">23.4714</td><td colspan="1" rowspan="1">34.253</td></tr><tr><td colspan="1" rowspan="1">Replicate-15</td><td colspan="1" rowspan="1">24.4274</td><td colspan="1" rowspan="1">29.3225</td><td colspan="1" rowspan="1">34.2523</td><td colspan="1" rowspan="1">Replicate-15</td><td colspan="1" rowspan="1">19.6833</td><td colspan="1" rowspan="1">22.6455</td><td colspan="1" rowspan="1">Und.</td></tr><tr><td colspan="1" rowspan="1">Replicate-16</td><td colspan="1" rowspan="1">23.083</td><td colspan="1" rowspan="1">27.9436</td><td colspan="1" rowspan="1">34.3973</td><td colspan="1" rowspan="1">Replicate-16</td><td colspan="1" rowspan="1">20.2427</td><td colspan="1" rowspan="1">24.4066</td><td colspan="1" rowspan="1">Und.</td></tr><tr><td colspan="1" rowspan="1">Replicate-17</td><td colspan="1" rowspan="1">25.26</td><td colspan="1" rowspan="1">28.0269</td><td colspan="1" rowspan="1">29.7855</td><td colspan="1" rowspan="1">Replicate-17</td><td colspan="1" rowspan="1">19.4036</td><td colspan="1" rowspan="1">25.0521</td><td colspan="1" rowspan="1">Und.</td></tr><tr><td colspan="1" rowspan="1">Replicate-18</td><td colspan="1" rowspan="1">24.1283</td><td colspan="1" rowspan="1">28.7603</td><td colspan="1" rowspan="1">30.4092</td><td colspan="1" rowspan="1">Replicate-18</td><td colspan="1" rowspan="1">20.5268</td><td colspan="1" rowspan="1">23.1831</td><td colspan="1" rowspan="1">32.2944</td></tr><tr><td colspan="1" rowspan="1">Replicate-19</td><td colspan="1" rowspan="1">24.5536</td><td colspan="1" rowspan="1">30.0526</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-19</td><td colspan="1" rowspan="1">20.7651</td><td colspan="1" rowspan="1">24.6162</td><td colspan="1" rowspan="1">29.8554</td></tr><tr><td colspan="1" rowspan="1">Replicate-20</td><td colspan="1" rowspan="1">24.2826</td><td colspan="1" rowspan="1">28.4218</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-20</td><td colspan="1" rowspan="1">21.292</td><td colspan="1" rowspan="1">25.6423</td><td colspan="1" rowspan="1">31.0606</td></tr><tr><td colspan="1" rowspan="1">Replicate-21</td><td colspan="1" rowspan="1">24.4037</td><td colspan="1" rowspan="1">30.1913</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Replicate-21</td><td colspan="1" rowspan="1">20.2716</td><td colspan="1" rowspan="1">26.1634</td><td colspan="1" rowspan="1">30.0698</td></tr><tr><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">NTC</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Und.</td><td colspan="1" rowspan="1">Und.</td></tr><tr><td colspan="1" rowspan="1"># (35&gt;Ct&gt;5)</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1"># (35&gt;Ct&gt;5)</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">17</td></tr><tr><td colspan="1" rowspan="1">percentage</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">61.9%</td><td colspan="1" rowspan="1">percentage</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">81.0%</td></tr><tr><td colspan="1" rowspan="1">average Ct</td><td colspan="1" rowspan="1">24.158662</td><td colspan="1" rowspan="1">28.982895</td><td colspan="1" rowspan="1">31.50985</td><td colspan="1" rowspan="1">average Ct</td><td colspan="1" rowspan="1">20.3275</td><td colspan="1" rowspan="1">24.54921</td><td colspan="1" rowspan="1">31.701512</td></tr><tr><td colspan="1" rowspan="1">STDEV</td><td colspan="1" rowspan="1">0.876732</td><td colspan="1" rowspan="1">1.1404107</td><td colspan="1" rowspan="1">2.4938263</td><td colspan="1" rowspan="1">STDEV</td><td colspan="1" rowspan="1">0.8259372</td><td colspan="1" rowspan="1">1.7634874</td><td colspan="1" rowspan="1">1.6164972</td></tr></table>

# e. Analytical specificity (Cross-reactivity):

The analytical specificity of the Quidel Molecular hMPV assay was evaluated by testing a panel consisting of 26 viruses, 24 bacteria, and one yeast strain that represent common respiratory pathogens or flora commonly present in nasopharynx. The bacteria and yeast were tested at concentrations of $1 0 ^ { 5 }$ to $1 0 ^ { 1 0 } \mathrm { \dot { C } F U / m L }$ with the actual test concentration indicated for each organism in the following table. Viruses were tested at concentrations of $1 \overline { { 0 } } ^ { 3 }$ to ${ 1 0 } ^ { 6 }$ $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ . All samples were extracted using the NucliSENS easyMAG instrument and tested in triplicate. The results are presented below. Briefly, the analytical specificity of the Quidel Molecular hMPV assay was $100 \%$ with no cross reactivity observed with the tested organisms.

<table><tr><td colspan="3" rowspan="1">Cross-reactivity</td></tr><tr><td colspan="1" rowspan="1">Organism ID</td><td colspan="1" rowspan="1">TCID50/mLor CFU/mL.</td><td colspan="1" rowspan="1">hMPV Result</td></tr><tr><td colspan="1" rowspan="1">RSV Long</td><td colspan="1" rowspan="1">4.40E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">RSV Washington</td><td colspan="1" rowspan="1">1.75E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mexico/4108/2009</td><td colspan="1" rowspan="1">1.40E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/04/2006</td><td colspan="1" rowspan="1">5.25E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 1/Adenoid 71</td><td colspan="1" rowspan="1">5.67E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.70E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1.67E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.43E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">2.28E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">8.76E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">5.38E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">1.50E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.05E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.50E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.68E+03</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.66E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">5,000cp/mL</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 Maclnytre strain</td><td colspan="1" rowspan="1">1.95E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">3.67E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Rubeola</td><td colspan="1" rowspan="1">3.78E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">8.43E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">2.50E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">2.20E+04</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">9.10E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">9.57E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">7.50E+02</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">1.04E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">2.55E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2.10E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">2.05E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium intracellulare</td><td colspan="1" rowspan="1">6.90E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">6.60E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">1.36E+10</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">5.90E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">5.15E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">2.65E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">2.75E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">2.15E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria menigitidis</td><td colspan="1" rowspan="1">1.85E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">1.85E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">3.30E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">6.80E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">5.85E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">6.0E+05</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.03E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">4.5E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.05E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">2.50E+06</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">2.6E+07</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">5.15E+08</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.07E+06</td><td colspan="1" rowspan="1">Negative</td></tr></table>

# f. Analytical specificity (Interfering Substances):

The performance of the Quidel Molecular hMPV in the presence of potential inhibitors that may be found in nasopharyngeal specimens was evaluated. The potentially interfering substances were evaluated using hMPV A1, hMPV B1 at a concentration of 3x LoD. The results of the analysis are presented in the following table. No evidence of interference caused by the substances tested was observed.

<table><tr><td colspan="1" rowspan="1">Substance Name</td><td colspan="1" rowspan="1">Concentration Tested</td><td colspan="1" rowspan="1">hMPV A1Result (3x LoD)</td><td colspan="1" rowspan="1">hMPV B1Result (3x LoD)</td></tr><tr><td colspan="1" rowspan="1">Mucin (Bovine SubmaxillaryGland, type I-S)</td><td colspan="1" rowspan="1">60 μg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Blood (human), EDTAanticoagulated</td><td colspan="1" rowspan="1">2% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Neo-Synephrine</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Afrin Nasal Spray</td><td colspan="1" rowspan="1">15% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Zicam Homeopathic Non-Drowsy Allergy Relief No DripLiquid Nasal Gel</td><td colspan="1" rowspan="1">5% (vol/vol)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Saline Nasal Spray</td><td colspan="1" rowspan="1">15% (vol/vol) of dose</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Throat Lozenges</td><td colspan="1" rowspan="1">0.68g/mL; 1/18 drop,crushed; activeingredients: 1.7 mg/mLmenthol</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Zanamivir</td><td colspan="1" rowspan="1">3.3-5 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Tobramycin</td><td colspan="1" rowspan="1">4.0 μg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Mupirocin</td><td colspan="1" rowspan="1">6.6-10 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Oseltamivir phosphate</td><td colspan="1" rowspan="1">7.5-25 mg/mL</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr></table>

g. Assay cut-off:

The “cutoff value” represents the fluorescent intensity signal (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

The cut-off for the Quidel Molecular hMPV Assay was determined and confirmed through a phased approach. The preliminary threshold was established using data obtained from the LoD studies and from the analysis of a set of clinical specimens. Data from the analysis of multiple replicates near the LoD of the assay were used to establish the threshold such that sensitivity was maximized. Similarly, the latest Ct value form the LoD data and those observed in the analysis of the set of clinical specimens were used the set the cut-off. Using the parameters summarized in the table below, a verification study was done to confirm the threshold and cut-off values.

<table><tr><td rowspan=1 colspan=3>Quidel hMPV Molecular Assay on the ABI 7500 Fast DxInstrument</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Confirmed Threshold(RFU)</td><td rowspan=1 colspan=1>Confirmed Ct Cut-Off*</td></tr><tr><td rowspan=1 colspan=1>hMPV A1</td><td rowspan=1 colspan=1>5.4e4</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>hMPV B1</td><td rowspan=1 colspan=1>5.4e4</td><td rowspan=1 colspan=1>35</td></tr></table>

\*These values reflect subtraction of the first 10 cycles. Fluorescence is monitored after the first 10 cycles during the data collection stage of amplification.

Results from the comparison to an FDA-cleared molecular device to validate the proposed cut-off and threshold are:

The Quidel Molecular hMPV assay showed $100 \%$ (7/7) PPA with an FDA cleared molecular comparator device.   
The Quidel Molecular hMPV assay showed $100 \%$ (58/58) NPA with an FDA cleared molecular comparator device.

These results demonstrated that the preliminary threshold and Ct cut-off settings determined have been confirmed and effective for maximizing sensitivity of the Quidel hMPV Molecular Assay.

# g. Comparison of transport media:

Analytical performance of the Quidel Molecular hMPV assay was evaluated with five different transport media. A stock of hMPV-B1 was used to evaluate performance across the different transport media at a level close to the LoD (3X). Briefly, dilutions of hMPV-B1 were made and each of the dilutions was extracted in triplicate. Each extraction was then tested in duplicate on the ABI 7500 Fast Dx using the Quidel Molecular hMPV standard protocol. No differences between the tested transport media were observed and the results are summarized below.

<table><tr><td rowspan=1 colspan=6>Evaluation of Transport Media in the Quidel Molecular hMPV Assay</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Test Media</td><td rowspan=1 colspan=1>Template</td><td rowspan=1 colspan=1>MPV</td><td rowspan=1 colspan=1>PRC</td><td rowspan=1 colspan=1>MPV</td><td rowspan=1 colspan=1>PRC</td></tr><tr><td rowspan=2 colspan=1>MS4</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>21.86</td><td rowspan=1 colspan=1>16.29</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.9</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.51</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=2 colspan=1>MS4-RT</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>21.40</td><td rowspan=1 colspan=1>16.37</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.51</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.18</td></tr><tr><td rowspan=2 colspan=1>MS5</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>21.37</td><td rowspan=1 colspan=1>16.46</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.56</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.11</td></tr><tr><td rowspan=2 colspan=1>MS6</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>21.21</td><td rowspan=1 colspan=1>16.32</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>0.08</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.75</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.49</td></tr><tr><td rowspan=2 colspan=1>UTMLot 49L606</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>22.56</td><td rowspan=1 colspan=1>16.57</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>0.37</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.48</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=2 colspan=1>UTMLot 51N618</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>20.96</td><td rowspan=1 colspan=1>16.47</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>0.25</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.70</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.19</td></tr><tr><td rowspan=2 colspan=1>UTMLot 49C600</td><td rowspan=1 colspan=1>3x LoD hMPV</td><td rowspan=1 colspan=1>21.98</td><td rowspan=1 colspan=1>16.07</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>0.18</td></tr><tr><td rowspan=1 colspan=1>Media + PrC</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>16.54</td><td rowspan=1 colspan=1>Und</td><td rowspan=1 colspan=1>0.15</td></tr><tr><td rowspan=1 colspan=1>Overall Av</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.62</td><td rowspan=1 colspan=1>16.36</td><td rowspan=1 colspan=1>0.74</td><td rowspan=1 colspan=1>0.28</td></tr></table>

# h. Carry-over Contamination Analysis:

An evaluation of potential carry–over contamination during extraction and/or amplification was carried out. High negative and positive samples were run in alternating series to evaluate performance of the device and to monitor for potential carry-over contamination. The evaluation included alternating extractions high and low concentration of virus on the indicated platform. During the amplification stage the plate was setup in a checkerboard approach locating negative and positive samples in adjoining wells. No cross contamination was observed in the evaluation of the Quidel Molecular hMPV kit.

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Prospective Clinical Studies:

Performance characteristics of the Quidel Molecular hMPV assay was established in a prospective study during the 2010-2011 respiratory virus season (January to March 2011). A total of 1116 specimens were evaluated in this study (742-fresh, 374-frozen). Of the fresh specimens, 399 specimens were nasal swabs, and 343 were nasopharyngeal swabs. The fresh specimens were collected for routine respiratory virus testing at thirteen sites across the United States. The frozen specimens were collected and tested at two geographically distinct locations and were comprised of 374 nasopharyngeal swabs.

The 1116 specimens were tested by both the subject and a FDA cleared molecular comparator device for human metapneumovirus RNA. Sixteen of these specimens were invalid on initial testing with the subject device $( 1 . 4 \% )$ . Retesting of the specimens according to the Interpretation algorithm described above also yielded invalid results. Thirty-six specimens were invalid on initial and repeat testing (as per the device's package insert) on the comparator device $( 3 . 2 \% )$ . Eight specimens were invalid in both devices, therefore, a total of 44 invalid specimens were removed from additional analysis. The remaining 1072 specimens were used in the $2 \mathbf { x } 2$ analysis of clinical performance.

The evaluation of the Quidel Molecular hMPV Assay was conducted at two external laboratory sites and one internal site. Each site used the ABI 7500Fast Dx platform for amplification/detection and the bioMerieux easyMAG for nucleic acid extraction. Each site included in the study was capable of performing highly complex laboratory tests.

Each specimen included in the study had the following testing performed. Prospective testing performed at the trial sites was done within 72-hours postcollection using the Quidel Molecular hMPV Assay with the bioMerieux easyMAG for nucleic acid extraction. Archival reference testing performed at a central testing location using samples that have been frozen within 72-hours postcollection and stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ . The frozen specimens were analyzed with a FDA cleared molecular device for hMPV according to the instructions for use provided at a central location and were not preselected based on previous results and are representative of an all-comers based study.

For inclusion in the study the specimens meet the following criteria:

Approximately $1 { \mathrm { - m L } }$ of leftover or waste specimens in viral transport media that were submitted for respiratory virus culture.   
· Specimens were tested in a prospective manner. Acceptable specimen types used for the evaluation: nasal swab, nasopharyngeal swab   
· Specimens transported, received and stored at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ for up to 72 hours.   
· No gender restrictions.   
· No age restrictions.   
· No results from the investigational device were made available to the patient’s physician. All patient identifiers were removed and replaced with study identifiers prior to inclusion in the study.

Exclusion of specimens from the study was based on the following criteria:

There were no patient exclusion criteria since the specimens were from patients that the treating physician has deemed qualified for diagnostic testing.   
· Inadequate specimen volume to perform the three devices as required.   
· Specimen inadequately stored or shipped.   
· Minimal patient information consisting of age or and gender was inadequate   
− Samples not received within 72 hours of collection, or stored at ${ > } 8 { ^ \circ } \mathrm { C }$ .

The following table summarizes the gender and age distribution of all patients enrolled in the evaluation.

<table><tr><td colspan="3" rowspan="1">Age and Gender Distribution</td></tr><tr><td colspan="1" rowspan="1">Sex</td><td colspan="1" rowspan="1">F</td><td colspan="1" rowspan="1">M</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">539</td><td colspan="1" rowspan="1">577</td></tr><tr><td colspan="1" rowspan="1">cYanmaGellowb</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">≤ 5 years</td><td colspan="1" rowspan="1">233</td><td colspan="1" rowspan="1">268</td></tr><tr><td colspan="1" rowspan="1">6-21 years</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">157</td></tr><tr><td colspan="1" rowspan="1">22 - 59 years</td><td colspan="1" rowspan="1">141</td><td colspan="1" rowspan="1">121</td></tr><tr><td colspan="1" rowspan="1">≥ 60 years</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">31</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">539</td><td colspan="1" rowspan="1">577</td></tr></table>

The overall clinical performance data are summarized in the following $2 \mathbf { x } 2$ table:

<table><tr><td rowspan=1 colspan=1>Nasal/nasopharyngeal swab(N=1072)</td><td rowspan=1 colspan=3>FDA Cleared RT-PCR</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecular</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>2*</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1007</td><td rowspan=1 colspan=1>1010</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>1009</td><td rowspan=1 colspan=1>1072</td></tr><tr><td></td><td></td><td></td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>60/63</td><td rowspan=1 colspan=1>95.2%</td><td rowspan=1 colspan=1>86.7 to 99.0%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>1007/1009</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.3% to 100%</td></tr></table>

\* Specimens negative for hMPV by sequence analysis

4. Clinical cut-off:

Not Applicable

5. Expected values/Reference range:

In the Quidel Molecular hMPV Assay clinical studies, a total of 1100 $( 1 . 4 \% )$ eligible nasopharyngeal (NP) swab and nasal swabs (NS) specimens were tested from 15 clinical laboratories across the United States during the in the winter of 2011 (January 2011 – March 2011). Originally 1116 specimens were available for the study, sixteen of these specimens were invalid on initial testing with the subject device, resulting in 1100 specimens. The number and percentage of the hMPV RNA positive cases by the Quidel Molecular hMPV Assay, calculated by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=4>hMPV RNA Positive by the Quidel Molecular hMPV Assay (N=1100)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Number ofPatients</td><td rowspan=1 colspan=1>Number of Positives</td><td rowspan=1 colspan=1>Prevalence</td></tr><tr><td rowspan=1 colspan=1>≤ 1 year</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>7.6%</td></tr><tr><td rowspan=1 colspan=1>1 to 5 years</td><td rowspan=1 colspan=1>358</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>7.0%</td></tr><tr><td rowspan=1 colspan=1>6 to 10 years</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>4.8%</td></tr><tr><td rowspan=1 colspan=1>11 to 15 years</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.4%</td></tr><tr><td rowspan=1 colspan=1>16 to 21 years</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>&gt; 21 years</td><td rowspan=1 colspan=1>339</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>5.6%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1100</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>5.6%</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.