# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT COMBINATION TEMPLATE

A. 510(k) Number: k112424 B. Purpose for Submission:

To obtain a substantial equivalence determination for this 510(k) for Staphylococcus aureus (SA), Staphylococcus epidermidis (SE) and mecA-mediated methicillin/oxacillin resistance of SA-positive and SE-positive from positive blood cultures.

C. Measurand: Staphylococcus aureus (gyrB), Staphylococcus epidermidis (hsp60), and mecA

# D. Type of Test:

Nucleic Acid Test, DNA, Staphylococcus aureus, Staphylococcus epidermidis, and/or mecA, qualitative

# E. Applicant:

Nanosphere, Inc.

F. Proprietary and Established Names:

Verigene® Staphylococcus Blood Culture (BC-S) Nucleic Acid Test Verigene® System (Verigene Reader, Verigene Processor SP)

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code: NQX- Nucleic acid amplification test, DNA, methicillin resistant Staph aureus, direct specimen

NSU- Instrumentation for clinical multiplex test systems

4. Panel:

83- Microbiology

# H. Intended Use:

1. Intended use(s):

The Verigene® Staphylococcus Blood Culture (BC-S) Nucleic Acid Test performed using the sample-to-result Verigene System is a qualitative, multiplexed in vitro diagnostic test for the simultaneous detection and identification of potentially pathogenic gram-positive bacterial species Staphylococcus aureus ("SA") and Staphylococcus epidermidis ("SE") which may cause bloodstream infection (BSI). In addition, the BC-S test detects the mecA resistance marker inferring mecA-mediated methicillin/oxacillin resistance. In mixed growth, the BC-S Test does not specifically attribute mecA-mediated methicillin/oxacillin resistance to either SA or SE.

The BC-S test is performed directly on positive blood culture using BACTECTM Plus Aerobic/F and BacT/ALERT FA $\mathrm { \dot { F } A N } ^ { \mathfrak { \otimes } }$ blood culture bottles, which contain gram-positive cocci in clusters (GPCCL) observed on Gram stain. Sub-culturing of positive blood cultures is necessary to recover organisms for susceptibility testing, differentiation of mixed growth, association of the mecA gene to an organism, or for epidemiological typing.

The BC-S test is indicated for use in conjunction with other clinical and laboratory findings, such as culture, to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor bloodstream infections.

2. Indication(s) for use:

The Verigene® Staphylococcus Blood Culture (BC-S) Nucleic Acid Test performed using the sample-to-result Verigene System is a qualitative, multiplexed in vitro diagnostic test for the simultaneous detection and identification of potentially pathogenic gram-positive bacterial species Staphylococcus aureus ("SA") and Staphylococcus epidermidis ("SE") which may cause bloodstream infection (BSI). In addition, the BC-S test detects the mecA resistance marker inferring mecA-mediated methicillin/oxacillin resistance. In mixed growth, the BC-S Test does not specifically attribute mecA-mediated methicillin/oxacillin resistance to either SA or SE.

The BC-S test is performed directly on positive blood culture using BACTECTM Plus Aerobic/F and BacT/ALERT FA FAN® blood culture bottles, which contain gram-positive cocci in clusters (GPCCL) observed on Gram stain. Sub-culturing of positive blood cultures is necessary to recover organisms for susceptibility testing, differentiation of mixed growth, association of the mecA gene to an organism, or for epidemiological typing.

The BC-S test is indicated for use in conjunction with other clinical and laboratory findings, such as culture, to aid in the diagnosis of bacterial bloodstream infections; however, it is not used to monitor bloodstream infections.

3. Special conditions for use statement(s): Prescription Use

4. Special instrument requirements: Verigene® System

# I. Device Description:

The Verigene Staphylococcus Blood Culture (BC-S) Nucleic Acid Test is a molecular assay which relies on detection of specific nucleic acid targets in a microarray format. For each of the bacterial nucleic acid sequences detected by the BC-S test, Capture and Mediator oligonucleotides are utilized for gold nanoparticle probe-based endpoint detection. The Capture oligonucleotides bind to a specific portion of the nucleic acid target and are themselves bound onto a substrate in microarray. The Mediator oligonucleotides bind to a different portion of the same nucleic acid target and allow binding of a gold nanoparticle probe to a portion complementary to a gold nanoparticle probe. Specific silver enhancement of the bound gold nanoparticle probes at the capture sites results in gold-silver aggregates that scatter light with high efficiency.

The BC-S test is performed on the Verigene System, a fully automated, bench-top molecular diagnostics workstation. The Verigene System consists of two components: the Verigene Reader and the Verigene Processor SP. The BC-S test utilizes single-use disposable test consumables and a self-contained Verigene Test Cartridge for each sample tested. For the BC-S test, the Verigene System allows automated nucleic acid extraction from Gram positive bacteria-containing blood culture specimens and target detection of bacteria-specific DNA.

The Reader is the Verigene System’s user interface, which serves as the central control unit for all aspects of test processing and results generation. The Reader’s graphical user interface guides the user through test processing and test results using a barcode scanner.

The user inserts the Test Cartridge into the Verigene Processor SP, which executes the test procedure, automating the steps:

(1) Sample Preparation – Cell lysis and magnetic bead based bacterial DNA isolation from blood culture samples and (2) Verigene Hybridization Test – Detection and identification of bacterial-specific DNA in a microarray format by using gold nanoparticle probe-based technology.

After test processing is complete, to obtain the test results the user removes the Test Cartridge from the Processor SP, removes the reagent pack from the substrate holder, and inserts the substrate holder into the Reader for analysis. Light scatter from the capture spots is imaged by the Reader and intensities from the microarray spots are used to make decisions regarding the presence (Detected) or absence (Not Detected) of a bacterial nucleic acid sequence/analyte.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

Xpert™ MRSA/SA Blood Culture Assay (Cepheid) Verigene® Respiratory Virus Nucleic Acid Test

2. Predicate 510(k) number(s):

k082140   
k092566

<table><tr><td rowspan="2">Item</td><td>Device</td><td>Predicate</td></tr><tr><td>Verigene® Staphylococcus Blood Culture (BC-S) Nucleic Acid Test</td><td>Xpert™M MRSA/SA Blood Culture Assay (k082140)</td></tr><tr><td>Intended Use</td><td>Similarities Qualitative in vitro diagnostic test for the detection and identification of Staphylococcus aureus (SA) and</td><td>Same</td></tr><tr><td>Test Cartridge</td><td>methicillin resistant Staphylococcus aureus Disposable single-use, multi-</td><td>Same</td></tr><tr><td>Nuclei Acid</td><td>chambered fluidic cartridge MRSA-mecA</td><td>Same</td></tr><tr><td>Target Specimen</td><td>Positive blood culture, determined</td><td>Same</td></tr><tr><td>Sample Prep Quality Control</td><td>as Gram positive cocci in clusters (GPCCL) by Gram stain Automated DNA extraction Internal procedural/instrument</td><td>Same Same</td></tr><tr><td>Interpretation of</td><td>quality controls; Internal negative Control, Sample processing control, external positive and negative assay controls</td><td></td></tr><tr><td>Results</td><td>Diagnostic Software/Decision Algorithm</td><td>Same</td></tr><tr><td>Intended Use</td><td>Differences Detection of S. epidermidis</td><td>Detection of S. aureus</td></tr><tr><td>Sample size Time to Result</td><td>350 μL</td><td>50 μL ~ 50 min</td></tr><tr><td>Detection Method</td><td>~ 2.5 hrs Gold/Ag nanoparticle probe</td><td>Fluorogenic RT target-specific</td></tr><tr><td>Software</td><td>detection of bacterial-specific DNA on complementary oligo-microarray Custom-embedded software</td><td>hybridization probe amplicon detection GeneXpert Dx System</td></tr><tr><td>Reader</td><td>running under the Micro-C/OS realtime operating system Optical intensities analyzed adter</td><td>Software I-CORE thermocycler and</td></tr><tr><td>Nuclei Acid</td><td>target-specific hybridization of probes</td><td>detection by fluorogenic target- specific hybridization.</td></tr><tr><td>Target</td><td>SA- tuf SE- hsp60 mecA</td><td>SA- spa MRSA- SCCmec</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not applicable

# L. Test Principle:

The BC-S is performed on the Verigene System, a molecular diagnostics consisting of two instruments: the Verigene Processor SP and the Verigene Reader. The Verigene Processor SP automates the following steps of the BC-S:

i. Sample Preparation Magnetic bead-based bacterial DNA isolation from positive blood culture specimens   
ii. Verigene Hybridization Hybridization of the bacterial DNA to the test substrate in the Processor SP

Detection and identification of the hybridized bacterial DNA are by the Verigene Reader, using gold nanoparticle probe-based detection technology. The Verigene Reader also serves as the user interface. It stores and tracks sample information throughout the assay process, and it also analyzes test results once the assay is complete.

The Verigene Processor SP utilizes the single-use consumables to perform the BC-S, including Verigene BC-S Test Kits (BC-S Test Cartridge and BC-S Extraction Trays), and Utility Kits. A separate Tip Holder Assembly contains two pipette tips that are used to transfer and mix reagents during the assay. The user tests a sample by loading the single-use disposables into the Verigene Processor SP and pipetting the sample into the Extraction Tray. The user initiates the test protocol on the Verigene Reader by scanning or entering the barcode ID located on the Test Cartridge along with sample information. Following assay completion, the user collects data on the Verigene Reader by scanning the barcode ID on the Test Cartridge and inserting it into the Verigene Reader for analysis.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The Reproducibility Study was conducted at three external sites and the Precision Study at one site. The panel comprised of twelve (12) specimens consisting of pure isolates grown in blood culture bottles until bottle positivity and bottle positivity plus an additional 8 hours at three external sites in

duplicates twice daily by two operators for five days. The specimens included 2 levels of 2 different MRSA strains, 2 levels of a MRSE strain, 2 negative controls (1 blood culture media only and 1 bacterial), 2 levels of an SA strain, and 2 levels of an MSSE strain. Each specimen was tested in duplicate twice daily by two operators for twelve days (precision) and five days (reproducibility). The study demonstrated reproducibility across multiple reagents lots, days, operators, runs, instruments and replicates.

Precision and Reproducibility Sample Panel Composition   

<table><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Incubation Time</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Negative Control  blood culture media only</td><td rowspan=1 colspan=1>48 hours</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Negative Control  Corynebacterium urealyticum</td><td rowspan=1 colspan=1>BP*</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Methicillin Resistant S. aureus Strain 1</td><td rowspan=1 colspan=1>BP</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Methicillin Resistant S. aureus Strain 1</td><td rowspan=1 colspan=1>BP plus 8 hrs</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Methicillin Resistant S. aureus Strain 2</td><td rowspan=1 colspan=1>BP</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Methicillin Resistant S. aureus Strain 2</td><td rowspan=1 colspan=1>BP plus 8 hrs</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>BP</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>S. aureus</td><td rowspan=1 colspan=1>BP plus 8 hrs</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Methicillin Resistant S. epidermidis</td><td rowspan=1 colspan=1>BP</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Methicillin Resistant S. epidermidis</td><td rowspan=1 colspan=1>BP plus 8 hrs</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>BP</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>S. epidermidis</td><td rowspan=1 colspan=1>BP plus 8 hrs</td></tr></table>

“BP  bottle positivity or evidence of bacterial growth in the automated blood culture monitoring system.

Of the 1296 samples tested in the precision and reproducibility study, the percent agreement for all panel members for the combined sites was $100 \%$ $9 6 . 6 \%$ - $100 \%$ ) $9 5 \%$ CI. There were 15 “No Calls” and 8 “pre-analytical” errors in the study. All of these samples were tested successfully on repeat testing.

Precision and Reproducibility Study Results- Agreement by Panel Member   

<table><tr><td rowspan=3 colspan=1>Sample</td><td rowspan=1 colspan=1>Precision</td><td rowspan=1 colspan=3>Reproducibility</td><td rowspan=2 colspan=1>Percent Total Agreement95% Cl</td></tr><tr><td rowspan=1 colspan=1>Agreement</td><td rowspan=1 colspan=3>Agreement</td></tr><tr><td rowspan=1 colspan=1>NS</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td><td rowspan=1 colspan=1>All Sites</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108/ 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 /108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 / 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>48 /48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 /108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 /108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108/ 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>48 /48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 / 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 /108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>48 /48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108/ 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>20 /20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 /108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>48 /48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108/ 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>48 /48</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>20 / 20</td><td rowspan=1 colspan=1>100 %(108 / 108)96.6 - 100 %</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>100 %(576/576)99.4 - 100 %</td><td rowspan=1 colspan=1>100 %(240/240)98.5 - 100 %</td><td rowspan=1 colspan=1>100 %(240/240)98.5 - 100 %</td><td rowspan=1 colspan=1>100 %(240/240)98.5 - 100 %</td><td rowspan=1 colspan=1>100 %(1296 /1296)99.7 - 100 %</td></tr></table>

b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The BC-S is performed using single-use disposable reagent trays and cartridges, in which all reagents are prepackaged to prevent reagent dispensing errors. Several levels of controls are built into the BC-S to ensure that failures at any procedural step of the BC-S are identified during the procedure.

# Internal Controls

An internal processing control, designated “IC1”, comprises a non-specific target organism Bacillus subtilis, a Gram-positive bacterium with an intact genome. It is automatically added to each sample in the processor immediately prior to Sample Extraction. The IC1 functions as a complete assay control, the primary purpose of which is to monitor failures likely to be attributable to the sample preparation step (i.e., lysis and nucleic acid extraction); it also functions as non-specific target hybridization/detection control.

A second internal processing control, designated “IC2”, comprises an assay-specific single-stranded DNA target present in the Sample Hybridization Mix reagent and is added by the System to each sample as a means to monitor hybridization inhibition (due to sample- or processrelated inhibitors or reagent failures).

For each test performed, both controls (IC1 and IC2) must yield correct results to enable reporting of a valid test result.

# External Controls

Culture confirmed Staphylococcus aureus, Staphylococcus epidermidis, and mecA-gene positive blood culture should be tested at least once per week and/or under the following circumstances:

Instrument installation, test validation, and when troubleshooting is   
necessary   
Performance verification for receipt of a new set/lot of   
consumables   
When the integrity of consumable storage conditions is in question

All external quality control requirements and testing should be performed in conformance with local, state, and federal regulations or accreditation organizations as applicable and should follow the user’s laboratory’s standard quality control procedures.

There were 16 “No calls” and three “pre-analysis error” (pre-ae) in the study, resulting initial call rate of $9 5 . 2 \%$ (316/332), final call rate $100 \%$ , pre-ae failure rate of $0 . 9 \%$ (3/332).

# d. Detection limit:

# Limit of Detection Study

Analytical sensitivity was determined with a set of S. aureus and $S .$ . epidermidis strains both with and without methicillin resistance (mecA). The strains for methicillin-resistant $S .$ . aureus (MRSA) included representative SCCmec types. The limit of detection (LOD) was assessed and confirmed by using bacterial strains with established titers. For each strain, the dilution series began with the sample obtained at ‘bottle positivity’ by the BACTEC resin blood culture bottles. The dilution series were prepared by using a diluent matrix that comprised blood culture broth containing charcoal, human blood, and a common commensal skin bacterium at a minimum concentration of ${ \sim } 1 0 ^ { 7 } \mathrm { C F U / m L }$ . Each dilution was tested in replicates of 4. The putative LOD was the lowest concentration level where all the replicates for the analyte were

‘Detected’. Once the putative LOD was established, 20 replicate samples were tested to confirm the LOD.

<table><tr><td rowspan=2 colspan=1>Bacteria Type</td><td rowspan=2 colspan=1>Strain Designation(Source ID)</td><td rowspan=2 colspan=1>ATCC</td><td rowspan=2 colspan=1>SCCmecType</td><td rowspan=2 colspan=1>PFGE</td><td rowspan=1 colspan=2>LOD</td></tr><tr><td rowspan=1 colspan=1>Conc,(CFU/mL)</td><td rowspan=1 colspan=1>ConfirmationResults</td></tr><tr><td rowspan=7 colspan=1>S. aureus (MRSA)</td><td rowspan=1 colspan=1>MRSA252</td><td rowspan=1 colspan=1>BAA-1720</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>USA200</td><td rowspan=1 colspan=1>4.6x10}$</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>GA217</td><td rowspan=1 colspan=1>BAA-1762</td><td rowspan=1 colspan=1>IV b</td><td rowspan=1 colspan=1>USA300</td><td rowspan=1 colspan=1>3.7x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>HDE288</td><td rowspan=1 colspan=1>BAA-42</td><td rowspan=1 colspan=1>VI</td><td rowspan=1 colspan=1>USA800</td><td rowspan=1 colspan=1>4.7x10 5$</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>E2125</td><td rowspan=1 colspan=1>BAA-38</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>USA500</td><td rowspan=1 colspan=1>3.7x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>HUSA304</td><td rowspan=1 colspan=1>BAA-39</td><td rowspan=1 colspan=1>III</td><td rowspan=1 colspan=1>Unnamed</td><td rowspan=1 colspan=1>2.1x10fr}$</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>HFH-30032</td><td rowspan=1 colspan=1>BAA-1688</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>Unnamed*</td><td rowspan=1 colspan=1>5.5x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>F182</td><td rowspan=1 colspan=1>43300</td><td rowspan=1 colspan=1>II</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>3.0 x10 5f}$</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>S. aureus (SA)</td><td rowspan=1 colspan=1>96308</td><td rowspan=1 colspan=1>BAA-1749</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>USA900</td><td rowspan=1 colspan=1>1.9x10f}$</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>MSSA476</td><td rowspan=1 colspan=1>BAA-1721</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>USA400</td><td rowspan=1 colspan=1>5.7x105</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=2 colspan=1>S. epidermidis (MRSE)</td><td rowspan=1 colspan=1>RP62A</td><td rowspan=1 colspan=1>35984</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>7.5x106</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>4477-2</td><td rowspan=1 colspan=1>700565</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.0x106</td><td rowspan=1 colspan=1>19/20</td></tr><tr><td rowspan=2 colspan=1>S. epidermidis (SE)</td><td rowspan=1 colspan=1>PCI1200</td><td rowspan=1 colspan=1>12228</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.7x106</td><td rowspan=1 colspan=1>20/20</td></tr><tr><td rowspan=1 colspan=1>15290</td><td rowspan=1 colspan=1>700583</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>6.9x106</td><td rowspan=1 colspan=1>20/20</td></tr></table>

\* Not USA100-1100

# Analytical Inclusivity

The inclusivity study included 99 MRSA strains (65 representative NARSA strains), 18 methicillin-sensitive Staphylococcus aureus (SA), eight borderline oxacillin-resistant Staphylococcus aureus (BORSA) strains, six methicillin resistant Staphylococcus epidermidis (MRSE), and seven methicillin-sensitive Staphylococcus epidermidis (SE). Separately, in silico analysis was performed by aligning the assay probes for each of the above strains against available GenBank sequence entries to ensure that the BC-S is able to detect these strains. All the strains were grown in blood culture bottles in automated blood culture instruments to bottle positivity. Samples were tested for purity and counted to estimate concentration $\mathrm { ( C F U / m L ) }$ ) and finally tested in duplicates with the BC-S. All results were as expected.

# Heterogeneous Resistance

The heterogeneous resistant S. aureus ATCC 43300 was included in the LoD studies.

e.

# Analytical specificity:

# Cross-Reactivity

The analytical specificity (exclusivity) study assessed potential crossreactivity of organisms phylogenetically related to panel organisms detected by the BC-S as well as those present as contaminants in blood culture

specimens. There were 157 bacteria (115 gram-positive, 40 gram-negative, one attenuated Mycobacterium tuberculosis, one Mycoplasma pneumoniae), and six yeast strains. No cross-reactivity was observed.

# Interference Study

The analytical specificity (interference) studies included strains of MRSA, SA, MRSE, and SE. They were tested with hemoglobin, triglycerides, bilirubin, gamma-globulin, and SPS at concentrations approximately one log higher than reference levels. The four strains were grown in several different BACTEC/F (Plus/Aerobic; Plus/Anaerobic; Standard/10 Aerobic; and PEDS Plus) and BacT/Alert (SA Aerobic, FA FAN Aerobic and PF Pediatric FAN) blood culture bottles. Negative blood culture bottles were also tested. For each condition, the interferents or negative blood culture bottles were tested in triplicates. No interference observed.

The BC-S is intended for use only with BACTECTM Plus Aerobic/F and BacT/ALERT FA FAN® Aerobic blood culture bottles.

# Fresh versus Frozen Samples Studies

The fresh versus frozen study was performed to assess the impact of freezing specimens on the performance of the BC-S. Two MRSA, one strain of each SA, MRSE, and SE were tested. Cultured samples were tested at baseline (fresh/unfrozen) and two additional time points: bottle positivity and bottle positivity plus 8 hours. After baseline testing, all the samples were subjected to 2 freeze-thaw cycles, which involved freezing at or below $\mathrm { - 7 0 ^ { \circ } C }$ and thawing at room temperature. Each time point for each condition (fresh/unfrozen and 2 freeze-thaw cycles) was tested in replicates of four. The BC-S results were not compromised when samples were exposed to up to two freeze/thaw cycles.

# Competitive Inhibition Studies

This study assessed competitive inhibition amongst combinations of organisms detected by the BC-S when combined in pairs (binary) with a set of related organisms commonly encountered in mixed blood culture infections. Specifically, the impact of an organism at high titer (defined as “bottle positivity $^ + 8$ hours”) on the detection of a second organism was assessed. Representative strains of MRSA, SA, MRSE, and SE, each at the LOD for the detection of that organism, were combined with the organism at the concentration listed in the table below. Each binary combination was tested in replicates of three. No evidence of competitive inhibition was observed for either the samples at the LOD or for the high titer samples.

Competitive Inhibition Results of the Testing Binary Combinations

<table><tr><td rowspan=1 colspan=6>Binary Combinations</td></tr><tr><td rowspan=2 colspan=1>High Titer Strains(bottie-positivity + 8 h)</td><td rowspan=2 colspan=1>Conc. Tested(CFU/mL)</td><td rowspan=1 colspan=4>Low Titer Strains (@ LOD in CFU/mL)</td></tr><tr><td rowspan=1 colspan=1>MRSA.(4.6x10^{)</td><td rowspan=1 colspan=1>SA(5.7x10^)</td><td rowspan=1 colspan=1>MRSE(7.5x10)</td><td rowspan=1 colspan=1>SE(6.9x10°)</td></tr><tr><td rowspan=1 colspan=1>S. aureus (MRSA)</td><td rowspan=1 colspan=1>8.9x107</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>S. aureus (SA)</td><td rowspan=1 colspan=1>4.0x10^8}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis (MRSE)</td><td rowspan=1 colspan=1>1.3x10}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>S. epidermidis (SE)</td><td rowspan=1 colspan=1>2.6x108$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>S. pneumoniae</td><td rowspan=1 colspan=1>2.8x108</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>S. pyogenes</td><td rowspan=1 colspan=1>5.9x108</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>S. agalactiae</td><td rowspan=1 colspan=1>1.3x10}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+*</td></tr><tr><td rowspan=1 colspan=1>S. anginosus</td><td rowspan=1 colspan=1>7.9x108$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>E. faecium (vanA)</td><td rowspan=1 colspan=1>1.1x10}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>E. faecalis (vanB)</td><td rowspan=1 colspan=1>8.3x108}$</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr><tr><td rowspan=1 colspan=1>M. luteus</td><td rowspan=1 colspan=1>4.1x107</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>+</td></tr></table>

\* This binary combination initially yielded a ‘Not Detected’ for SE in one replicate (1/3) but gave the expected ‘Detected’ result in the remaining replicates (2/3). Repeat-testing of this binary combination in replicates of 20 yielded a ‘Detected’ call for SE in all 20 tests, suggesting that the initial ‘Not Detected’ result was likely due to expected variability at the SE LOD and not due to competitive inhibition (see text for additional details).

# Carryover/Cross-Contamination Study

A study was performed to assess the potential for carryover/crosscontamination with the BC-S by alternately running ‘high positive’ samples followed by ‘true negative’ samples. All of the high positive samples yielded the expected ‘Detected’ results for the intended bacteria and ‘Not Detected’ results for the other analytes. The true negative samples gave a ‘Not Detected’ call for all analytes.

# f. Assay cut-off:

Sixty-five retrospective clinical samples were tested with the BC-S to verify the cut-off values of a three-tiered filter algorithm determined with known reference strains and by using logistic fit and ROC statistics. The cut-off values were verified by 2340 data points.

# 2. Comparison studies:

The BC-S results were compared to the results of standard biochemical detection techniques (i.e. coagulase, and VITEK 2 GP ID) for identification; Cefoxitin Disk Diffusion for mecA mediated methicillin resistance:

S. aureus ≤21 mm mecA (+) ≥22 mm mecA (-)

S. epidermidis ≤24 mm mecA (+) ≥25 mm mecA (-)

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

3. Clinical studies:

The clinical study was conducted at five external study sites. There were 330 culture-positive, gram-positive cocci in clusters (GPCCL) blood culture sample BC-S results compared with results from standard biochemical detection techniques. The results for $S .$ . aureus demonstrated a positive and negative agreement of $100 \%$ . The results for $S .$ . epidermidis demonstrated a positive and negative agreement of $9 7 . 0 \%$ and $9 9 . 6 \%$ respectively. The results for mecA demonstrated a positive agreement of $9 8 . 6 \%$ and a negative agreement of $9 8 . 8 \%$ respectively. The initial call rate in the study was $9 5 . 2 \%$ (16 initial no-call results), and the final call rate was $100 \%$ .

# a. Clinical Sensitivity/:

BC-S vs. Conventional Culture results   

<table><tr><td colspan="2">Staphylococcus aureus</td><td rowspan="2">SA + SA-</td><td rowspan="2">Culture</td><td rowspan="2">Total</td><td rowspan="2">Percent Agreement</td></tr><tr><td>(SÅ) Verigene</td><td>SA +</td></tr><tr><td rowspan="3">BC-S</td><td></td><td>124</td><td>0</td><td>124</td><td>Positive Agreement=100% (95%CI: 97.1 -100)</td></tr><tr><td>SA-</td><td>0</td><td>206</td><td>206</td><td>Negative Agreement=100%</td></tr><tr><td>Total</td><td>124</td><td>206</td><td>330</td><td>(95%CI: 98.2 -100)</td></tr></table>

BC-S vs. Conventional Culture results   

<table><tr><td colspan="2">Staphylococcus</td><td colspan="2">Culture</td><td rowspan="2">Total</td><td rowspan="2">Percent Agreement</td></tr><tr><td>epidermidis (SE) Verigene</td><td>SE +</td><td>SE + 94</td><td>SE- 1b</td><td>Positive Agreement=97.0%</td></tr><tr><td rowspan="3">BC-S</td><td></td><td></td><td></td><td>95</td><td></td></tr><tr><td rowspan="2">SE-</td><td rowspan="2">3a</td><td rowspan="2">232</td><td rowspan="2">235</td><td>(95%CI: 91.2 -99.4) Negative Agreement=99.6%</td></tr><tr><td></td></tr><tr><td></td><td>Total</td><td>97</td><td>233</td><td>330</td><td>(95%CI: 97.6- 99.9)</td></tr></table>

a Three specimens identified as Staphylococcus spp. with BC-S and coagulase-negative Staphylococcus by culture, while two of the three were S. epidermidis by Vitek 2 GP ID and one

was S. capitis/S. epidermidis by Vitek 2 GP ID b One specimen identified as Staphylococcus epidermidis, mecA $( + )$ with the BC-S and coagulasenegative Staphylococcus by culture and S. hominis by Vitek 2 GP ID, cefoxitin disc (-) by culture   

<table><tr><td colspan="6">BC-S vs. Cefoxitin Disk Diffusion</td></tr><tr><td colspan="6">mecA</td></tr><tr><td rowspan="3">Verigene BC-S</td><td>mecA +</td><td>mecA +</td><td>Cefoxitin Disk Diffusion mecA - 1b</td><td>Total</td><td>Percent Agreement Positive Agreement=98.6%</td></tr><tr><td></td><td>137</td><td></td><td>138</td><td></td></tr><tr><td>mecA -</td><td>2a</td><td>81</td><td>83</td><td>(95%CI: 94.9 -99.8) Negative Agreement=98.8%</td></tr><tr><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td></td><td></td><td></td><td></td><td></td><td>(95%CI: 93.4- 99.9)</td></tr><tr><td></td><td>Total</td><td>139</td><td>82</td><td>221</td><td></td></tr></table>

a Two S. epidermidis specimens were identified as mecA (-) with the BC-S and cefoxitin disc $( + )$ by culture.

b One specimen identified as Staphylococcus epidermidis, mecA $( + )$ with the BC-S and coagulasenegative Staphylococcus by culture and S. hominis by Vitek 2 GP ID, cefoxitin disc (-) by culture. c mecA results are only reported if Staphylococcus aureus or Staphylococcus epidermidis are detected in the specimen.

b. Clinical Specificity:   
See part 3 a above for Specificity data.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

<table><tr><td rowspan=2 colspan=1>US GeographicegionDivision</td><td rowspan=2 colspan=1>State</td><td rowspan=2 colspan=1>Totaln</td><td rowspan=1 colspan=2>SA</td><td rowspan=1 colspan=2>SE</td><td rowspan=1 colspan=2>mecA</td></tr><tr><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>ObservedPrevalence</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>ObservedPrevalence</td><td rowspan=1 colspan=1>NumberPositive</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>Northeast / Mid-Atlantic</td><td rowspan=1 colspan=1>NY</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>35.4%</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>27.1%</td></tr><tr><td rowspan=3 colspan=1>Midwest / East North Central</td><td rowspan=1 colspan=1>OH</td><td rowspan=1 colspan=1>92</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>35.9%</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>33.7%</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>45.7%</td></tr><tr><td rowspan=1 colspan=1>IL</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>63.2%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>21.1%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>52.6%</td></tr><tr><td rowspan=1 colspan=1>WI</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>284</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>16^-4</td><td rowspan=1 colspan=1>28.6%</td><td rowspan=1 colspan=1>2514</td><td rowspan=1 colspan=1>44.6%</td></tr><tr><td rowspan=1 colspan=1>South / West South Central</td><td rowspan=1 colspan=1>TX</td><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>29.6%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>33.0%</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>42.6%</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>330</td><td rowspan=1 colspan=1>124</td><td rowspan=1 colspan=1>37.6%</td><td rowspan=1 colspan=1>97</td><td rowspan=1 colspan=1>29.4%</td><td rowspan=1 colspan=1>139</td><td rowspan=1 colspan=1>42.1%</td></tr></table>

(1) Census Bureau Regions and Divisions with State FIPS Codes; US Census Bureau. http://www.census.gov/geo/www/us_regdiv.pdf. (20 June 2010). (2) One specimen (A024) tested positive for SA, $S E$ , and mecA

# N. Instrument Name:

Verigene® System (Verigene Processor SP and Verigene Reader)

# O. System Descriptions:

1. Modes of Operation:

The Verigene System allows random access, fully automated ‘sample to result’ operation. The Verigene System is comprised of two components: a Reader (user interface, central control unit, optics for reading the absence or presence of bacterial DNA on the microarray) and the Processor SP. The BC-S test, performed on the Verigene System, involves two steps:

Sample preparation-Cell lysis and magnetic bead-based bacterial DNA isolation from blood culture samples · Verigene Hybridization Test-Hybridization of bacterial-specific DNA to capture oligonucleotides on a microarray, using gold nanoparticles probebased technology to aid detection.

Operationally, single use disposables (Extraction Tray, Utility Tray, and Test Cartridge) are loaded into separate modules within the Processor SP. The only ‘userperformed’ pipetting step required to perform the assay is the addition of the grampositive blood culture sample into the extraction tray. The sample preparation occurs within the Processor SP’s Extraction Tray and is also aided by reagents present in the Utility Tray, while the Verigene Hybridization Test occurs within the Test Cartridge. An automated pipette using disposable pipette tips delivers and transfers reagents during the assay.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X____ or No

The results of the V&V testing of the result mask and other general updates were provided in version 1.8.0b6 of the reader software.

# 3. Specimen Identification:

Specimens are labeled with a Barcode. The Processor SP and Reader detect the specimen ID, the results are printed with this specimen identifier.

4. Specimen Sampling and Handling: Automated Verigene System

5. Calibration: Not required

6. Quality Control:

A series of internal controls used in conjunction with procedural checks monitors instrument functionality, performance, fluidics, reagent integrity, and result determination, based on a pre-defined decision algorithm.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.