# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number:

k112937

# B. Purpose for Submission:

To obtain a substantial equivalence determination for this 510k for MRSA/SA from nasal specimens.

# C. Measurand:

Target DNA sequences for the conserved regions in a Staphylococcus aureus-specific gene $( l d h I )$ , which is responsible for SA identification and the mecA gene, which is responsible for resistance to methicillin.

# D. Type of Test:

Nucleic Acid Amplification Test, DNA, Methicillin-resistant Staphylococcus aureus (MRSA) / Staphylococcus aureus (SA), qualitative

# E. Applicant:

ELITechGroup Epoch Biosciences

F. Proprietary and Established Names: MRSA/SA ELITe MGB®

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.1640 Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NQX- Nucleic acid amplification test, DNA, methicillin resistant Staph aureus, direct specimen   
NSU- Instrumentation for multiplex test systems   
JJH- Clinical sample concentrator

4. Panel: 83 Microbiology

# H. Intended Use:

1. Intended use(s):

MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbf { ( B ) } }$ is a qualitative in vitro diagnostic test for the direct detection of Staphylococcus aureus (SA) and methicillin-resistant Staphylococcus aureus (MRSA) using DNA purified from nasal swabs. MRSA/SA ELITe $\operatorname { M G B } ^ { \circledast }$ is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose, guide or monitor MRSA infections, or provide results of susceptibility to oxacillin/methicillin. A negative result does not preclude MRSA/SA (Staphylococcus aureus) nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

2. Indication(s) for use:

MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbf { ( B ) } }$ is a qualitative in vitro diagnostic test for the direct detection of Staphylococcus aureus (SA) and methicillin-resistant Staphylococcus aureus (MRSA) using DNA purified from nasal swabs. MRSA/SA ELITe $\operatorname { M G B } ^ { \circledast }$ is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose, guide or monitor MRSA infections, or provide results of susceptibility to oxacillin/methicillin. A negative result does not preclude MRSA/SA (Staphylococcus aureus) nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

3. Special conditions for use statement(s): Prescription Use

4. Special instrument requirements: bioMérieux NucliSENS® easyMAG® extraction system and the Applied Biosystems® 7500 Fast Dx PCR Instrument.

# I. Device Description:

MRSA/SA ELITe MGB is a real-time, multiplex polymerase chain reaction (PCR)

assay for the in vitro qualitative detection of MRSA and SA DNA extracted from human nasal swab samples. In this system, sample preparation and amplification/realtime detection are completed on separate instruments. Sample processing is completed on the bioMérieux NucliSENS® easyMAG® instrument with bioMérieux NucliSENS Nucleic Acid Extraction Reagents according to the manufacturer’s instructions. Following processing, the extracted sample is placed in the well of a 96 well plate to which “monoreagent” is added. The monoreagent contains the primers and probes for the genes of interest and the internal control combined with master mix. The assay is performed on an Applied Biosystems 7500 FAST Dx System that consists of the 7500 FAST Dx instrument, a personal computer, 96-well plates and seals. A Negative Specimen Processing Control and a Positive Specimen Processing Control are recommended to be run in each extraction run. The design of the assay includes systems to identify both the gene responsible for methicillin resistance and for a conserved portion of a gene unique to S. aureus. Thus, for a true “MRSA,” both targets will be identified in roughly equal proportions. Results are determined by using an algorithm that compares Ct values output from the cycler.

Instrument:

The Applied Biosystems (AB) 7500 Fast Dx Real-Time PCR instrument integrates a thermal cycler, a fluorimeter and application specific software. The instrument houses the thermal cycler and the fluorimeter, while the application software is run on a PC that is attached to the instrument. Samples are placed in a tube strip or 96-well lowhead space plate that is moved to a Peltier-based thermal block and positioned relative to the optics using a tray loading mechanism. Excitation for all samples is provided by a halogen tungsten white source that passes through 5 switchable excitation filters prior to reaching the sample.

Fluorescence emission is then detected through a 5 color emissions filter wheel to a charge coupled device (CCD) camera. The instrument is designed to complete quantitative RT-PCR runs in about 40 minutes. The Sequence Detection Software (SDS) version 1.4 for the 7500 Fast Dx Instrument is used for instrument control, data collection and data analysis. The software can measure cycle-by-cycle real-time signals from the sample. The software provides a variety of tools to help the user analyze the data extracted from the samples. The software also provides lamp-life monitoring and other instrument maintenance information. The software runs as an application on Windows® XP platform. Changes to the Dx software are subject to change control in accordance with 21 CFR Part 820.40.

# J. Substantial Equivalence Information:

1. Predicate device name(s):

BD GENEOHM MRSA ACP ASSAY

2. Predicate 510(k) number(s):

# K093346

3. Comparison with predicate:

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>MRSA/SA ELITe MGB</td><td rowspan=1 colspan=1>BD GENEOHM MRSAACP ASSAY (K093346)</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Direct detection of MRSA and SA</td><td rowspan=1 colspan=1>Direct detection of MRSA</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Fully-automated nucleic acidamplification (DNA); real-time PCR</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Direct from nasal swab</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>EY</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Positive PCR control (Plasmid DNA(microbial) containing MRSAsequences)Internal Control (Plasmid DNA(recombinant) containing InternalControl sequences)</td><td rowspan=1 colspan=1>Positive PCR control (DNAfrom S. aureus ATCC 43300).Negative PCR control (DNAfrom S. epidermidis ATCC14990).Internal procedural control</td></tr><tr><td rowspan=1 colspan=1>DNA TargetSequence</td><td rowspan=1 colspan=1>Sequence specific toStaphylococcus aureus species andmecA gene</td><td rowspan=1 colspan=1>SCCmec cassette (geneticelement that carries the mecAgene) at orfX junction(specific to S. aureus)</td></tr><tr><td rowspan=1 colspan=1>Instrument System</td><td rowspan=1 colspan=1>ABI 7500 Fast Dx</td><td rowspan=1 colspan=1>BD SmartCycler® II</td></tr><tr><td rowspan=1 colspan=1>Storage &amp; Expiry</td><td rowspan=1 colspan=1>Stored in -20 °C freezer. The deviceis stable until the expiry date statedon the label.</td><td rowspan=1 colspan=1>Stored at 2- 25 °C.Reagents are stable until theexpiry date stated on the label.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

FDA Draft Guidance for Industry and Food and Drug Administration Staff Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA), Issued January 5, 2011.

FDA Guidance for Industry, FDA Reviewers and Compliance on Off-the-Shelf Software Use in Medical Devices, Issued September 9, 1999.

# L. Test Principle:

The MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ test is a triplex real-time amplification-based assay that targets the conserved regions in a Staphylococcus aureus-specific gene, which is responsible for SA identification, mecA gene, which is responsible for resistance to methicillin and an exogenous internal control to monitor reaction inhibition and reagent integrity. The Staphylococcus aureus-specific gene will identify SA, and the mecA gene will identify the methicillin

resistance gene. Presence of both markers at the same relative quantity measured by a difference in cycle threshold (Ct) value is indicative of MRSA; different relative quantities or presence of only Staphylococcus aureus-specific gene marker is indicative of SA.

Detection of MRSA/SA with the MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ relies on two major processes: Specimen preparation by automated DNA extraction from nasal swabs and Real-time PCR amplification and detection of target DNA by specific hybridization probes.

Each nasal swab is inserted into a tube with trypticase soy broth and thoroughly mixed to create a cell suspension which is then subjected to automated DNA extraction using the NucliSENS® easyMAG® instrument. Internal control plasmid template is added to the silica solution to act as a control for the extraction process and monitor for PCR inhibitors.

The processed specimens and the MRSA/SA ELITe $\mathbf { M G B } ^ {  { ( B ) } }$ PCR Mix containing hot-start non-Taq thermostable DNA polymerase are placed in the Applied Biosystems® 7500 Fast Dx Real-Time PCR Instrument PCR plate for amplification and detection. The ELITe $\mathbf { M G B } ^ { \mathbf { \otimes } }$ Probes detect amplified target through a hybridization-triggered fluorescent mechanism. When the probe is unbound in solution, the three-dimensional conformation brings the quencher and MGB in close proximity to the fluorescent label, quenching the fluorescence. When the probe anneals to a target sequence, the probe unfolds, spatially separating the quencher from the fluorescent label to allow a strong fluorescent signal. ELITe MGB® Probes are not degraded during the amplification. The probe specific to the SA-specific gene is labeled with AP554 fluorophore (similar to TAMRA).The probe specific to the mecA gene is labeled with FAM fluorophore. The probe specific to the Internal Control is labeled with AP642 fluorophore (similar to Cy5). The Applied Biosystems $^ \mathrm { \textregistered }$ 7500 Fast Dx Real-Time PCR Instrument monitors simultaneously the fluorescence emitted by each probe. Following PCR, the results are interpreted to provide a final call.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

A reproducibility study was performed using a panel of 10 specimens with varying concentrations of MRSA and MSSA in a simulated nasal matrix was tested. Two MRSA strains (ATCC BAA-1556 and BAA-1720) and one MSSA strain (BAA-12600) were used. Simulated matrix contained human genomic DNA and mucin to imitate a normal human nasal matrix. For each MRSA/MSSA strain the panel included negative member, specimen below the limit of detection, LoD (expected to yield a positivity rate of between 20 to $80 \%$ ), low positive (at LoD, expected to yield a $9 5 \%$ positivity rate), and moderate positive (three times LoD, expected to have $100 \%$ positivity rate). Each of the two operators performed one run per day for 12 days on three reagent lots at one site. In two other sites two runs per day on one reagent lot were performed for 5 days (10 specimens x 3 replicates x 5 days x 2 runs). The negative panel member yielded negative results $100 \%$ , the below LoD specimens positivity rate was $7 7 \%$ , the low positive specimen positivity rate was $9 8 \%$ , and the moderate positive panel members positivity rate was $100 \%$ .

# A summary of the reproducibility study is shown below:

<table><tr><td rowspan=1 colspan=1>Specimen Type</td><td rowspan=1 colspan=1>Lot 1</td><td rowspan=1 colspan=1>Lot 2</td><td rowspan=1 colspan=1>Lot 3</td><td rowspan=1 colspan=1>Total Agreement (%)</td></tr><tr><td rowspan=1 colspan=1>Negative (R1)</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>58/58 (100%)</td></tr><tr><td rowspan=1 colspan=1>Below LoD (R2,R5,R8)</td><td rowspan=1 colspan=1>33/42</td><td rowspan=1 colspan=1>31/42</td><td rowspan=1 colspan=1>70/90</td><td rowspan=1 colspan=1>134/174 (77%)</td></tr><tr><td rowspan=1 colspan=1>Low Positive (R3,R6,R9)</td><td rowspan=1 colspan=1>40/42</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>88/90</td><td rowspan=1 colspan=1>170/174 (98%)</td></tr><tr><td rowspan=1 colspan=1>Moderate Positive (R4,R7,R10)</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>42/42</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>174/174 (100%)</td></tr></table>

The numerical results for the Site-to-Site Reproducibility Study are shown below as Ct value results by sample level and probe:

<table><tr><td rowspan=1 colspan=4>Internal Control</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Overall Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>R1 (Negative)</td><td rowspan=1 colspan=1>30.22</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=1 colspan=1>R2 (Below LoD MRSA strain1)</td><td rowspan=1 colspan=1>30.22</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>1.51</td></tr><tr><td rowspan=1 colspan=1>R3 (Low Positive MRSA strain 1)</td><td rowspan=1 colspan=1>30.27</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>1.00</td></tr><tr><td rowspan=1 colspan=1>R4 (Moderate Positive MRSA 1)</td><td rowspan=1 colspan=1>30.19</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.88</td></tr><tr><td rowspan=1 colspan=1>R5 (Below LoD MRSA strain 2)</td><td rowspan=1 colspan=1>30.18</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>1.65</td></tr><tr><td rowspan=1 colspan=1>R6 (Low Positive MRSA strain 2)</td><td rowspan=1 colspan=1>30.41</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>2.01</td></tr><tr><td rowspan=1 colspan=1>R7 (Moderate Positive MRSA strain 2)</td><td rowspan=1 colspan=1>30.29</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.22</td></tr><tr><td rowspan=1 colspan=1>R8 (Below LoD MSSA)</td><td rowspan=1 colspan=1>30.27</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.15</td></tr><tr><td rowspan=1 colspan=1>R9 (Low Positive MSSA)</td><td rowspan=1 colspan=1>30.27</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>1.21</td></tr><tr><td rowspan=1 colspan=1>R10 (Moderate Positive MSSA)</td><td rowspan=1 colspan=1>30.17</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>1.17</td></tr></table>

<table><tr><td rowspan=1 colspan=4>ldh1 target</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Overall Mean</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>R1 (Negative)</td><td rowspan=1 colspan=1>37.92</td><td rowspan=1 colspan=1>0.81</td><td rowspan=1 colspan=1>2.13</td></tr><tr><td rowspan=1 colspan=1>R2 (Below LoD MRSA strain1)</td><td rowspan=1 colspan=1>36.32</td><td rowspan=1 colspan=1>1.27</td><td rowspan=1 colspan=1>3.49</td></tr><tr><td rowspan=1 colspan=1>R3 (Low Positive MRSA strain 1)</td><td rowspan=1 colspan=1>35.65</td><td rowspan=1 colspan=1>1.33</td><td rowspan=1 colspan=1>3.74</td></tr><tr><td rowspan=1 colspan=1>R4 (Moderate Positive MRSA 1)</td><td rowspan=1 colspan=1>34.40</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>1.81</td></tr><tr><td rowspan=1 colspan=1>R5 (Below LoD MRSA strain 2)</td><td rowspan=1 colspan=1>36.93</td><td rowspan=1 colspan=1>0.98</td><td rowspan=1 colspan=1>2.65</td></tr><tr><td rowspan=1 colspan=1>R6 (Low Positive MRSA strain 2)</td><td rowspan=1 colspan=1>33.96</td><td rowspan=1 colspan=1>1.25</td><td rowspan=1 colspan=1>3.67</td></tr><tr><td rowspan=1 colspan=1>R7 (Moderate Positive MRSA strain 2)</td><td rowspan=1 colspan=1>31.99</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>2.56</td></tr><tr><td rowspan=1 colspan=1>R8 (Below LoD MSSA)</td><td rowspan=1 colspan=1>37.60</td><td rowspan=1 colspan=1>1.01</td><td rowspan=1 colspan=1>2.68</td></tr><tr><td rowspan=1 colspan=1>R9 (Low Positive MSSA)</td><td rowspan=1 colspan=1>34.59</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>2.64</td></tr><tr><td rowspan=1 colspan=1>R10 (Moderate Positive MSSA)</td><td rowspan=1 colspan=1>32.90</td><td rowspan=1 colspan=1>0.59</td><td rowspan=1 colspan=1>1.79</td></tr></table>

<table><tr><td colspan="4" rowspan="1">mecA target</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Overall Mean</td><td colspan="1" rowspan="1">SD</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">R1 (Negative)</td><td colspan="1" rowspan="1">38.76</td><td colspan="1" rowspan="1">0.63</td><td colspan="1" rowspan="1">1.62</td></tr><tr><td colspan="1" rowspan="1">R2 (Below LoD MRSA strain1)</td><td colspan="1" rowspan="1">37.03</td><td colspan="1" rowspan="1">1.13</td><td colspan="1" rowspan="1">3.06</td></tr><tr><td colspan="1" rowspan="1">R3 (Low Positive MRSA strain 1)</td><td colspan="1" rowspan="1">36.65</td><td colspan="1" rowspan="1">1.34</td><td colspan="1" rowspan="1">3.66</td></tr><tr><td colspan="1" rowspan="1">R4 (Moderate Positive MRSA 1)</td><td colspan="1" rowspan="1">35.28</td><td colspan="1" rowspan="1">0.67</td><td colspan="1" rowspan="1">1.89</td></tr><tr><td colspan="1" rowspan="1">R5 (Below LoD MRSA strain 2)</td><td colspan="1" rowspan="1">37.51</td><td colspan="1" rowspan="1">0.79</td><td colspan="1" rowspan="1">2.11</td></tr><tr><td colspan="1" rowspan="1">R6 (Low Positive MRSA strain 2)</td><td colspan="1" rowspan="1">34.89</td><td colspan="1" rowspan="1">1.34</td><td colspan="1" rowspan="1">3.83</td></tr><tr><td colspan="1" rowspan="1">R77 (Moderate Positive MRSA strain 2)</td><td colspan="1" rowspan="1">32.75</td><td colspan="1" rowspan="1">0.92</td><td colspan="1" rowspan="1">2.82</td></tr><tr><td colspan="1" rowspan="1">R8 (Below LoD MSSA)</td><td colspan="1" rowspan="1">39.34</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">2.90</td></tr><tr><td colspan="1" rowspan="1">R9 (Low Positive MSSA)</td><td colspan="1" rowspan="1">38.95</td><td colspan="1" rowspan="1">1.04</td><td colspan="1" rowspan="1">2.67</td></tr><tr><td colspan="1" rowspan="1">R10 (Moderate Positive MSSA)</td><td colspan="1" rowspan="1">38.71</td><td colspan="1" rowspan="1">1.29</td><td colspan="1" rowspan="1">3.33</td></tr></table>

# b. Linearity/assay reportable range:

Not applicable

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

The MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbb { B } }$ includes a non-infectious plasmid DNA which serves as an internal control.

Positive specimen processing control (PSPC; MRSA strain) and negative specimen processing controls (NSPC; S. epidermidis) were tested daily. The PSPC Methicillinresistant S. aureus ATCC BAA-1556 was tested at 3xLOD per swab. The NSPC S. epidermidis ATCC 12228 was tested at 1 x 107 CFU/mL in simulated nasal matrix.

If a PCR run included samples processed during several days, the corresponding processing controls were included in the run. A separate MSSA strain was not tested because the MRSA strain includes SA marker (ldh1).

External Controls may be used in accordance with accrediting institutions and government regulations. External Controls are not provided in the test kit; however, the outside source and the catalog numbers are provided in the “Materials Recommended but Not Provided” section of the Package Insert.

d. Detection limit:

# Limit of Detection Study

The analytical sensitivity of the MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbb { B } }$ was determined using one strain of $S .$ . aureus and five strains of methicillin-resistant S. aureus (MRSA).

Cultures of these strains were quantified, diluted in simulated nasal matrix to values spanning the range of approximately 5 to 1500 colonies forming units (CFU) and absorbed onto swabs. All dilutions were tested, and the limit of detection (LoD) was determined by Probit analysis. LoD for each strain represents the lowest number of

CFU/swab at which a positive result will be obtained with at least $9 5 \%$ confidence. The LoD was then verified by testing 20 replicates at estimated LoD levels for each strain. Results were analyzed by StatisPro software for each (mecA and “SA”) detector separately. The results from this study indicate that the MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ will produce a positive MRSA/SA result for over $9 5 \%$ of the time at LoD range of 90-220 CFU/swab with an average LoD of 165 CFU/swab. Details are shown in the table below:

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Strain Designation(Strain No.)</td><td rowspan=1 colspan=1>LoD(CFU/swab)</td><td rowspan=1 colspan=1>n/N(% positive)</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>MRSA252(ATCC BAA-1720)</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>FRP3757(ATCC BAA-1556)</td><td rowspan=1 colspan=1>220</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>GA-92(NRS694)</td><td rowspan=1 colspan=1>210</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>MN-095(NRS703)</td><td rowspan=1 colspan=1>110</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MRSA</td><td rowspan=1 colspan=1>OR-131(NRS722)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>S. aureus(MSSA)</td><td rowspan=1 colspan=1>Wichita (ATCC29213)</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

# Effect of Increasing Concentrations of MSSA or MRCoNS on MRSA detection at the LoD level

The potential competitive inhibitory effect of methicillin-susceptible S. aureus (MSSA) or methicillin-resistant coagulase negative Staphylococci (MRCoNS) on MRSA at the established LoD level, was evaluated by spiking specimens with increasing amounts of MSSA or MRCoNS (21 samples for both series for each MRSA to MSSA/MRCoNS ratio: 1:1, 1:10, $1 { : } 1 0 ^ { 2 }$ , $\mathrm { i } { : } 1 0 ^ { 3 }$ , $1 { : } 1 0 ^ { 4 }$ , $1 { : } 1 0 ^ { 5 } \mathrm { a n d } 1 { : } 1 0 ^ { 6 } $ ). Seven control samples with no interfering strains were also used for the study. Results show that $\mathrm { C q } \left( \mathrm { C t } \right)$ values for each challenged detector $\mathbf { \^ { \mathrm { e } \mathbf { \check { S } } A } } ^ { \mathrm { \prime } \mathrm { 3 } }$ and mecA) were not significantly different $( \% C V { < } 5 )$ for every ratio examined indicating absence of inhibition of one target at LoD level by excess of another. The study demonstrated that detection of each target (mecA or “SA”) is not impaired by increasing amounts of another target up to $1 { : } 1 0 ^ { \tilde { 6 } }$ ratio.

# Analytical Inclusivity

The MRSA/SA ELITe MGB assay correctly identified all 76 well-characterized MRSA and SA strains, as well as all MRSA/SA strains with high and low MIC values from the challenge panel. MRSE strains were not detected because of the absence of the “SA” gene. BORSA strains were not detected because the genome of these organisms does not have the SCCmec element, which carries the mecA gene.

A total of 76 well characterized MRSA and MSSA isolates were tested using the MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ assay. The isolates were representative of the global genetic diversity, including clonal complexes and sequence types as well as various PulseField Gel Electrophoresis (PFGE) types (e.g. USA100, 200, 300 and 400) and MIC values, with emphasis on the USA 300 epidemic clones. All strains were absorbed onto swabs at near detection limit and tested with MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ . In addition to that all MSSA strains were tested at $1 0 ^ { 6 } \mathrm { C F U } /$ swab.

All MSSA strains tested positive for SA and negative for MRSA. All MRSA strains tested positive for MRSA.

Two BORSA (Borderline Oxacillin Resistant S. aureus) isolates that lack mecA tested SA positive and MRSA negative.

# e. Analytical specificity:

# Cross Reactivity

The specificity of the MRSA/SA ELITe MGB $^ \mathrm { \textregistered }$ was evaluated by testing for crossreactivity to species phylogenetically related to $S .$ aureus, pathogenic microorganisms and to microorganisms commonly present in normal nasal microflora. The test panel consisted of three fungal species, one mycoplasma species, 17 viruses, and 41 bacterial species (16 coagulase negative staphylococcus species, three coagulase negative Staphylococcus species and 23 other Gram negative and Gram positive species). The microorganisms were tested as cultures in concentrations of $1 \mathrm { x } 1 0 ^ { 6 }$ CFU/swab for bacterial isolates and $1 \mathrm { x } 1 0 ^ { 5 }$ PFU/swab for viruses. In addition human cells in a concentration of $1 0 ^ { 6 }$ cells $/ \mathrm { m L }$ were tested. Human cells and all tested species were found negative for MRSA and SA with the MRSA/SA ELITe MGB $^ \mathrm { \textregistered }$ . The analytical specificity was $100 \%$ .

Two of the potentially interfering organisms tested, Human metapneumovirus (hMPV) and MRCoNS Staphylococcus epidermidis, strain NRS 34, caused interference at initial testing resulting in false positive SA but negative MRSA result.

Re-testing of MRCoNS Staphylococcus epidermidis, strain NRS 34 confirmed the interference results. MRCoNS Staphylococcus epidermidis, strain NRS 34, is considered to interfere with MRSA/SA ELITe MGB®. This was included as a limitation in the label.

MRCoNS Staphylococcus epidermidis when tested in a mix with near-detectionlimit -MRSA resulted in “SA positive, MRSA negative” call.

Re-testing of Human metapneumovirus (hMPV) did not reveal interference of hMPV with MRSA/SA ELITe $\mathbf { M } \mathbf { \hat { G } } \mathbf { B } ^ { \mathbb { \otimes } }$ .

# Co-infection

Methicillin Susceptible S. aureus (MSSA) was also tested for microbial interference. The microorganism was spiked at $1 { \times } 1 0 ^ { 6 } \mathrm { C F U / m L }$ $( 1 { \times } 1 0 ^ { 5 } \mathrm { P F U / m L } )$ , or higher, into a sample with MRSA strains at near-detection-limit and tested.

MRCoNS S. epidermidis and MSSA, when tested in a mix with near-detectionlimit of MRSA, resulted in “SA positive, MRSA negative” call.

A limitation statement was added to indicate this finding. None of the other tested species interfered with MRSA/SA detection.

# Evaluation of BORSA Strains

Two BORSA (Borderline Oxacillin Resistant S. aureus) isolates that lack mecA tested SA positive and MRSA negative in the MRSA/SA ELITe MGB $^ \mathrm { \textregistered }$ assay.

# Interference Study

Potentially interfering substances that may be found in nasal swabs were tested at their highest clinically relevant concentrations to determine the effect on the performance of the assay. Substances tested were chemical substances that can either be naturally present or that can be artificially introduced into the nasal cavity.

Positive samples (two MRSA strains spiked into simulated nasal matrix near the device cut off $\left( 3 { \times } \mathrm { L O D } \right)$ for each strain) were tested with each of potentially interfering substance (mucin, human blood, nasal sprays or drops, nasal corticosteroids, nasal gels, homeopathic relief medicine, live intranasal Influenza virus vaccine, throat lozenges, oral anesthetic and analgesic, anti-viral drugs, nasal antibiotic ointment, systemic antibacterial substances) at their highest clinically relevant concentrations to determine the effect on the performance of the assay. A swab was immersed in an aliquot of interfering substance $( \approx 2 0 0 ~ \mu \mathrm { L } )$ , soaked and then thoroughly rinsed into a $1 \mathrm { m L }$ sample of TSB. Simulated Nasal Matrix Sample containing $1 0 0 \mu \mathrm { L }$ aliquot of $3 { \times } \mathrm { L O D }$ of MRSA ATCC BAA-1556 or MRSA ATCC BAA-1720) was absorbed onto a swab and was thoroughly rinsed into the test sample of TSB buffer (that already contained the interferent).

The following substances were tested and evaluated: blood, mucin, phenylephrine (Neo-synephrine®), oxymetazoline (Dristan®, Zicam®), sodium chloride with preservatives, benzalkonium chloride, sodium phosphate, phenylcarbinol (Saline), propylene glycol $( \operatorname { A Y R } ^ { \mathfrak { B } }$ saline nasal gel), sorbitol, benzyl alcohol, disodium EDTA, hypromellose, phosphoric acid, dexamethasone, triamcinolone (Nasacort®), beclomethasone (Beconase $\mathsf { A Q } ^ { \mathbb { P } }$ ), flunisolide, budesonide (Rhinocort Aqua®), mometasone (Nasonex®), fluticasone (Flonase®), luffa opperculata, sulfur, Galphimia glauca, Histaminum hydrochloricum, live intranasal influenza virus vaccine (FluMist®), benzocaine, methol (Cepacol® sore throat lozenges), Zanamivir (Relenza®), Oseltamivir phosphate (Tamiflu $©$ ), Mupirocin, and Tobramycin.

Only the following substances have been shown to interfere with the performance of the assay causing an inhibition resulting in false negative MRSA/SA results:

· $\operatorname { A Y R } ^ { \mathbb { \otimes } }$ saline nasal gel and excessive amounts of nasal secretions/mucus. · Blood

A limitation statement was added to indicate that these substances have been shown to interfere with the performance of the assay: AYR $^ \mathrm { \textregistered }$ saline nasal gel and excessive amounts of nasal secretions/mucus and blood.

# Carry-Over Contamination

An analytical study was performed to evaluate the potential for cross-contamination between high MRSA $1 \times \mathrm { { 1 0 } ^ { 7 } }$ CFU per mL) specimens and negative specimens throughout the MRSA/SA ELITe $\operatorname { M G B } ^ { \circledast }$ workflow. Two operators performed five 24-sample (11 high MRSA samples. 11 negative samples, one positive specimen processing sample and one negative specimen processing sample per run) extraction runs in a checkerboard (high MRSA samples interrupted by completely negative samples) pattern. The eluted samples were then PCR amplified in five separate runs using two different checkerboard patterns. The MRSA/SA ELITe $\mathbf { M G B } ^ { \circledast }$ failed to pass the cross-contamination design requirements, resulting in one false positive sample from fifty-five TSB negative samples. Labeling will report these results as follows: The cross-contamination testing resulted in zero false negatives from fifty-five high MRSA positive samples and one false positive sample from fifty-five negative samples.

# f. Assay cut-off:

To assess the analytical cut off evaluation study, $1 0 ^ { 6 } \mathrm { C F U }$ of MRSA, SA and CNS cultures were extracted and eluted in $1 0 0 \mu \mathrm { L }$ . Three $1 0 ~ \mu \mathrm { L }$ aliquots of each extracted sample were analyzed by PCR. Average $\mathrm { C t }$ for MRSA and SA (mecA, “SA”) were in the range 23-24.5. Next, samples were diluted to provide a threshold cycle at 35-36 $\left( 2 { - } 3 \thinspace \mathrm { C t } \right.$ above the expected cut-off level). Three 3-fold dilutions in 10 replicates of each culture were analyzed in one PCR run. Analytical assay cut off for each criterion (mecA, “SA”) was determined by means of appropriate statistical methods.

Ct threshold optimization $\sim 3 9$ for mecA was attained by minimizing the amount of falsepositive results for MRSA. Ct threshold optimization $\sim 3 8$ for “SA” was attained by minimizing the amount of false-positive results for MRSA/SA. Delta $\mathrm { C T } = 1 . 8 8 3 5 ( { \sim } 2 )$ for the Ct range at or near LOD level (33.54 – 39.07).

Results interpretation algorithm is shown in the table below:\*

<table><tr><td rowspan=1 colspan=1>SA = CT1</td><td rowspan=1 colspan=1>mecA = CT2</td><td rowspan=1 colspan=1>ΔCT|Cπ −CT2</td><td rowspan=1 colspan=1>IC</td><td rowspan=1 colspan=1>MRSAResult</td><td rowspan=1 colspan=1>SA Result</td></tr><tr><td rowspan=2 colspan=1>Undeterminedor CT &gt;35.0</td><td rowspan=2 colspan=1>Undeterminedor CT &gt; 35.0</td><td rowspan=2 colspan=1>NA</td><td rowspan=1 colspan=1>CT &lt; 34.0</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Undeterminedor CT ≥ 34.0</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>Invalid</td></tr><tr><td rowspan=3 colspan=1>Determined,CT ≤ 35.0</td><td rowspan=1 colspan=1>Undeterminedor CT &gt; 35.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=3 colspan=1>NA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=2 colspan=1>Determined,CT ≤ 35.0</td><td rowspan=1 colspan=1>∆CT ≥ 2</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>∆CT &lt;2</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Undeterminedor CT &gt; 35.0</td><td rowspan=1 colspan=1>Determined,CT ≤ 35.0</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr></table>

\*Each well will have three (3) $\mathrm { C _ { T } }$ values: one for SA that is called $\bf { C } _ { T 1 }$ , one for mecA, $\mathbf { C } _ { \mathbf { T } 2 }$ , and one for Internal Control (IC). For a valid test the presence of both SA and mecA markers at the same relative quantity (that is the absolute value of $\mathrm { \bf C } _ { \mathrm { T 1 } } - \mathrm { \bf C } _ { \mathrm { T 2 } }$ , a “difference in $\mathrm { C _ { T } }$ called $\bf { \tilde { \mu } } \Delta \bf { \vec { \mu } } \Delta \bf { \tilde { \mu } } \Delta \bf { \tilde { \mu } } \Delta \bf { \tilde { \mu } } \Delta \bf { \tilde { \mu } } \Delta \bf { \tilde { \mu } } \Delta \bf { \mu }$ less than $2 ^ { \circ }$ ) is indicative of MRSA; different relative quantities (a difference in $\mathrm { C _ { T } }$ equal or greater than 2) or presence of only the Staphylococcus aureus-specific gene marker is indicative of SA

The results interpretation algorithm is as follows for each situation:

Situation 1:

IF $\mathrm { C } _ { \mathrm { T 1 } } { > } 3 5 . 0$ AND $\mathrm { C } _ { \mathrm { T } 2 } { > } 3 5 . 0$ AND IC $\mathrm { C _ { T } } < 3 4 . 0$ , then the result is "MRSA-negative/SAnegative."

Situation 2:

IF $\mathrm { C } _ { \mathrm { T 1 } } { > } 3 5 . 0$ AND $\mathrm { C } _ { \mathrm { T } 2 } { > } 3 5 . 0$ AND IC $\mathrm { C _ { T } } \ge 3 4 . 0$ , then the result is "Invalid."

Situation 3:

IF $\mathrm { C } _ { \mathrm { T 1 } } { \le } 3 5 . 0 $ AND $\mathrm { C } _ { \mathrm { T } 2 } { \le } 3 5 . 0 $ AND $\vert { \bf C } _ { \mathrm { T 1 } } - { \bf C } _ { \mathrm { T 2 } } \vert < 2$ , then the result is "MRSA-positive."

Situation 4:

IF $\mathrm { C } _ { \mathrm { T 1 } } { \le } 3 5 . 0 $ , AND $\left| \mathbf { C } _ { \mathrm { T 1 } } - \mathbf { C } _ { \mathrm { T 2 } } \right| \geq 2$ , then the result is "MRSA-negative/SA-positive."

Situation 5:

IF $\mathrm { C } _ { \mathrm { T 1 } } { > } 3 5 . 0$ AND $\mathrm { C } _ { \mathrm { T } 2 } { \le } 3 5 . 0 $ , then the result is "MRSA-negative/SA-negative."

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

$b$ . Matrix comparison: Not applicable

3. Clinical studies:

# Clinical Sensitivity/Specificity:

Performance characteristics of the MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbb { B } }$ were determined in a prospective investigational study at three sites by comparing the MRSA/SA ELITe $\operatorname { \dot { M } G B } ^ { \circledast }$ with reference culture which included latex agglutination and susceptibility test. Nasal swab specimens were collected from three unique geographically diverse institutions having MRSA culture-based screening programs in place. To be enrolled in the study, patients had to be eligible for MRSA screening according to the policies of the respective facilities.

All specimens were inoculated into trypticase soy enrichment broth with $6 . 5 \%$ NaCl and the results from this testing was used as the reference. The nasal swab was directly inoculated onto either Remel Spectra MRSA or a BD BBL Chromagar MRSA II. Following this step, the same swab was subjected to an extraction step in $1 \mathrm { m L }$ of Tryptic Soy Broth (TSB), vortexed for 10-15 seconds. The entire volume of the cell suspension was tested using MRSA/SA ELITe MGB. The swab was then placed in TSB with $6 . 5 \%$ Sodium Chloride for enrichment/subculture to service as a reference testing (subculture culture on Trypticase Soy Blood agar plates). All swabs were subjected to enrichment in trypticase soy broth with $6 . 5 \%$ NaCl. Confirmation of presumptive colonies was done by latex agglutination and mecA-mediated oxacillin resistance was tested by disk diffusion test using a $3 0 \mu \mathrm { g }$ cefoxitin disk and cutoff of ${ \le } 2 1 ~ \mathrm { m m }$ (R), $\geq 2 2 ~ \mathrm { m m }$ (S).

Performance of the MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbb { B } }$ was calculated relative to the reference culture (enrichment broth culture followed by latex agglutination and cefoxitin susceptibility test results).

# Overall Results

Performance of the MRSA/SA ELITe $\mathbf { M G B } ^ { \mathbb { B } }$ was calculated relative to the broth culture followed by latex agglutination and cefoxitin susceptibility test results. A total of 3271 nasal swab specimens were collected from hospitalized patients. Of the 3271 specimen tested, 3174 specimens were eligible to be included in statistical analyses (72 specimens were considered to be ineligible due to a duplicating error during samples collection and preparation and 25 specimens failed the extraction due to a technical error).

The overall performance of MRSA/SA ELITe MGB®is shown in the following table:

<table><tr><td rowspan=1 colspan=1>Combined Data</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>Reference Culture</td></tr><tr><td rowspan=6 colspan=1>MRSA/SA ELITeMGB®</td><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>SA+/MRSA-</td><td rowspan=1 colspan=1>Neg/NoGrowth</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>MRSA+</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>348</td></tr><tr><td rowspan=1 colspan=1>SA+/MRSA-</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>405</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>508</td></tr><tr><td rowspan=1 colspan=1>SA-</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>2288</td><td rowspan=1 colspan=1>2318</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>222</td><td rowspan=1 colspan=1>546</td><td rowspan=1 colspan=1>2406</td><td rowspan=1 colspan=1>3174</td></tr><tr><td rowspan=1 colspan=5>MRSA:Sensitivity: 92.3% (88.08%-95.16%)Specificity: 95.2% (94.32%-95.87%)PPV: 58.9% (53.67%-63.95%)NPV: 99.4% (99.04%-99.62%)SA:Sensitivity: 96.1% (94.48%-97.25%)Specificity: 95.1% (94.16%-95.89%)PPV: 86.2% (83.74%-88.36%)NPV: 98.7% (98.16%-99.09%)Note: The statistics shown are the calculated values with the 95%confidence interval in the parentheses.</td></tr></table>

Discrepant Analysis: Further investigation (testing for MRSA by sequencing of SCCmec right extremity junction) was performed on all specimens that gave discordant MRSA results between the reference culture method and MRSA/SA ELITe MGB™.

16 of the 17 specimens that were MRSA-positive by culture but MRSA-negative by MRSA/SA ELITe MGB™ were found to be MRSA positive by SCCmec right extremity junction sequencing.   
22 of the 143 specimens that were MRSA-negative by culture but MRSA-positive by MRSA/SA ELITe MGB™ were found to be MRSA positive by SCCmec right extremity junction sequencing.

Thus, after discrepant analysis, the positive and negative percent agreements for MRSA compared to reference culture were $93 \%$ and $9 6 \%$ , respectively. The positive and negative predictive values were $6 5 . 2 \%$ and $9 9 . 4 \%$ , respectively.

c. Other clinical supportive data (when a. and $b$ . are not applicable):

Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

A total of 3174 nasal specimens collected from hospitalized patients were included in this study from three institutions across the United States. The overall S. aureus nasal carriage rate determined by culture followed by Staphaurex latex agglutination test was $24 \%$ . Of the S. aureus isolates, $2 9 \%$ were methicillin resistant by agglutination/cefoxitin susceptibility test for an overall MRSA nasal carriage rate of $7 \%$ .

# N. Instrument Name:

The Applied Biosystems (AB) 7500 Fast Dx Real-Time PCR Instrument

# O. System Descriptions:

1. Modes of Operation: Batch via 96 well plate or tube strip

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes __X____ or No

3. Specimen Identification: The information is entered by the user.

4. Specimen Sampling and Handling: Specimens are processed according to assay instructions.

5. Calibration:

Calibration is performed at regular six-month intervals by AB service personnel. The user performs a background calibration. A background calibration measures the level of background fluorescence in the instrument. During a background calibration run, the

instrument:

· Performs continuous reads of a background plate containing PCR buffer for 10 minutes at $6 0 ~ ^ { \circ } \mathrm { C }$ .   
· Averages the spectra recorded during the run and extracts the resulting spectral component to a calibration file.

The software then uses the calibration file during subsequent runs to remove the background fluorescence from the run data. The user is directed to perform this calibration monthly or as often as necessary depending on instrument use, well as after replacing the lamp.

6. Quality Control:

Quality control is addressed for each separately cleared specific assay to be run on the instrument.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

None.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.