# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number:

k113777

# B. Purpose for Submission:

This is a new 510(k) application for a qualitative Real-Time Reverse TranscriptionPolymerase Chain Reaction (RT-PCR) assay used with the Cepheid SmartCycler® II for the in vitro qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal swabs (NS) and nasopharyngeal swabs (NPS) from symptomatic human patients.

# C. Measurand:

Target RNA sequences for the highly conserved regions of the matrix protein gene of influenza A virus and the neuraminidase gene of influenza B virus.

# D. Type of Test:

Multiplex Real-Time RT-PCR assay for the qualitative detection and differentiation of influenza A and influenza B viral RNA from nasal and nasopharyngeal swab specimens using nucleic acid isolation and amplification. The isolation and purification of the viral RNA is performed using the NucliSENS® easyMAGTM System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux). The amplification and detection can be performed on either the Cepheid SmartCycler® II software version 3.0b or the Applied Biosystems 7500 Fast Dx Real-Time PCR Instrument with the SDS Software version 1.4 (previously cleared - see k112172).

# E. Applicant:

Quidel® Corporation

# F. Proprietary and Established Names:

Quidel® Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product codes: OZE, OOI

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Quidel® Molecular Influenza $\mathbf { A } + \mathbf { B }$ Assay is a multiplex Real Time RT-PCR assay for the in vitro qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. The test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

Performance characteristics for influenza A were established during the 2010 to 2011 influenza season when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

bioMérieux NucliSENS easyMAG System (software version 2.0)

Cepheid SmartCycler® II (software version 3.0b)

Applied Biosystems (ABI) 7500 FAST Dx (software version 1.4) – see k112172

# I. Device Description:

The assay detects influenza A and influenza B viral RNA that has been extracted from a patient sample using the NucliSENS® easyMAG® automated extraction platform. A multiplex RT-PCR reaction is carried out under optimized conditions in a single tube generating amplicons for each of the target viruses present in the sample. This reaction is performed utilizing the Cepheid SmartCycler® II or the Applied Biosystems® 7500 Fast Dx platform. Identification of influenza A occurs by the use of target specific primers and a fluorescent-labeled probe that hybridizes to a conserved region within the matrix protein gene. Identification of influenza B occurs by the use of target specific primers and a fluorescent-labeled probe that hybridizes to a conserved influenza B sequence within the neuraminidase gene.

<table><tr><td rowspan=1 colspan=2>Quidel Molecular Probe Labels</td></tr><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Dye</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>CAL Fluor Orange 560</td></tr><tr><td rowspan=1 colspan=1>Process Control</td><td rowspan=1 colspan=1>Quasar 670</td></tr></table>

The following is a summary of the procedure:

1. Sample Collection: Obtain nasal swabs or nasopharyngeal swabs using standard techniques from symptomatic patients. These specimens are transported, stored, and processed according to established laboratory procedures.

2. Nucleic Acid Extraction: Extract viral RNA from the specimens with the NucliSENS easyMAG System following the manufacturer’s instructions and using the appropriate reagents. Prior to the extraction procedure add $2 0 \mu \mathrm { L }$ of the Process Control (PRC) to each $1 8 0 \mu \mathrm { L }$ aliquot of specimen. The PRC serves to monitor inhibitors in the extracted specimen, assures that adequate amplification has taken place and confirms that the nucleic acid extraction was sufficient.

3. Rehydration of Master Mix: Rehydrate the lyophilized Master Mix using the Rehydration Solution. The Master Mix contains oligonucleotide primers, fluorophore and quencher-labeled probes targeting highly conserved regions of the influenza A and influenza B viruses as well as the PRC sequence. The primers are complementary to highly specific and conserved regions in the genome of these viruses. The probes are dual labeled with a reporter dye attached to the $5 ^ { \circ }$ end and a quencher attached to the $3 ^ { \circ }$ end.

4. Nucleic Acid Amplification and Detection: Add $1 5 ~ \mu \mathrm { L }$ of the rehydrated Master Mix to each plate well then add $5 \mu \mathrm { L }$ of extracted nucleic acids (specimen with PRC) to the plate well. Place the reaction tube into the Cepheid SmartCycler® II or the plate into the ABI 7500 Fast Dx instrument. Once the plate is added to the instrument, the assay protocol is initiated. This protocol initiates reverse transcription of the RNA targets generating complementary DNA, and the subsequent amplification of the target sequences occurs. The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay is based on TaqMan® chemistry, and uses an enzyme with reverse transcriptase, DNA polymerase, and $5 ' - 3 '$ exonuclease activities. During DNA amplification, this enzyme cleaves the probe bound to the complementary DNA sequence, separating the quencher dye from the reporter dye. This step generates an increase in fluorescent signal upon excitation by a light source of the appropriate wavelength. With each cycle, additional dye molecules are separated from their quenchers resulting in additional signal. If sufficient fluorescence is achieved by 45 cycles for the Cepheid SmartCycler® II or 35 cycles for the Applied Biosystems® 7500 Fast Dx, during the data collection stage of amplification, the sample is reported as positive for the detected target sequence.

# Materials Provided

SKU # M100 Detection Kit (96 Reactions) – Store at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$

<table><tr><td>#</td><td>Component</td><td>Quantity</td></tr><tr><td>0</td><td>Rehydration Solution Part M5003</td><td>1 vial/kit 1.9 mL</td></tr><tr><td rowspan="4">2</td><td>Quidel Molecular Influenza A+B Master Mix Part M5004 Lyophilized Contents:</td><td rowspan="4">12 vials/kit, 8 reactions/vial</td></tr><tr><td>DNA polymerase enzyme with reverse transcriptase activity</td></tr><tr><td>Oligonucleotide primer pairs; Oligonucleotide probes</td></tr><tr><td>dNTPs (dATP, dCTP, dGTP, dUTP, dTTP)</td></tr><tr><td>CONTROL</td><td>Stabilizers Process Control Part M5005</td><td>1 vial/kit 2.0 mL</td></tr></table>

# Optional Materials

Positive controls for influenza A and influenza B (Quidel Molecular Influenza A/B Control Set #M106) which serve as an external processing and extraction control.

# Materials Required But Not Provided

· Micropipettors (range between 1 to $1 0 ~ \mu \mathrm { L }$ and 100 to $1 0 0 0 \mu \mathrm { L }$

· Non-aerosol pipette tips   
· Cepheid SmartCyclerII\*   
· Cepheid SmartCyclerII tubes\*   
· Cepheid SmartCyclerII centrifuge\*   
· Applied Biosystems 7500Fast Dx software version $1 . 4 ^ { * * }$   
· Applied Biosystems 7500Fast Dx 96 well PCR plate\*\* Applied Biosystems optical plate films\*\*   
· Plate centrifuge for ABI 96 well plate\*\*   
· bioMerieux NucliSENS easyMAG software version 2.0   
· bioMerieux NucliSENS easyMAG Buffers 1, 2, 3   
· bioMerieux NucliSENS easyMAG Lysis Buffer bioMerieux NucliSENS easyMAG Silica Magnetic Beads \*- required if using SmartCycler® II \*\*- required if using Applied Biosystems®7500Fast Dx

Interpretation of Results using the Cepheid SmartCycler II Thermocycler   

<table><tr><td rowspan=1 colspan=5>Interpretation of the Quidel Molecular Influenza A+B Assay Results on the CepheidSmartCycler II</td></tr><tr><td rowspan=1 colspan=1>Assay Result</td><td rowspan=1 colspan=1>Detector:Influenza A</td><td rowspan=1 colspan=1>Detector:Influenza B</td><td rowspan=1 colspan=1>Detector:ProcessControl</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>PASS</td><td rowspan=1 colspan=1>No influenza A or influenzaB viral RNA detected; PRCDetected</td></tr><tr><td rowspan=1 colspan=1>Influenza APositive</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza A viral RNAdetected</td></tr><tr><td rowspan=1 colspan=1>Influenza BPositive</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza B viral RNAdetected</td></tr><tr><td rowspan=1 colspan=1>Influenza A andB Positive</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>Influenza A and Influenza Bviral RNA detected**</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>FAIL</td><td rowspan=1 colspan=1>No Influenza A or InfluenzaB and no PRC viral RNAdetected; invalid test. Retestthe same purified sample. Ifthe test is also invalid, re-extract and re-test anotheraliquot of the same sampleor obtain a new sample andretest.</td></tr></table>

\*No Ct value is required for the Process Control to make a positive call. \*\* Dual infections are rare. Repeat testing using the purified sample. If the retest confirms this result, collect and test a new specimen. Contact Quidel if multiple samples provide this result.

Error Code 3079: Warning/Error Code 3079 may be observed with influenza A and influenza B positive samples. Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the $\mathrm { D x }$ software as ND (Not Determined). Repeat testing using the same purified sample. If the retest confirms this result, collect and test a new specimen. Contact Quidel if multiple samples provide this result.

# Interpretation of Results using the Applied Biosystems®7500Fast Dx – see k112172

# J. Substantial Equivalence Information:

1. Predicate device name(s): Prodesse ProFlu+

2. Predicate 510(k) number (s): k073029, k092500 and k081030

3. Comparison with predicate:

<table><tr><td>Item Intended Use</td><td>Subject Device: Quidel Molecular Influenza A+B Assay</td><td>Predicate Device: Prodesse ProFlu+</td></tr><tr><td rowspan="2"></td><td>The Quidel Molecular Influenza A+B assay is a multiplex Real Time RT- PCR assay for the in vitro qualitative detection and differentiation of influenza A and influenza B viral RNA in nasal and nasopharyngeal swabs from patients with signs and symptoms of respiratory infection. This test is intended for use as an aid in the differential diagnosis of influenza A and influenza B viral infections in humans in conjunction with clinical and epidemiological risk factors. The assay does not detect the presence of influenza C virus.</td><td>The ProFluTM+ Assay is a multiplex Real-Time PCR (RT-PCR) in vitro diagnostic test for the rapid and qualitative detection and discrimination of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens obtained from symptomatic patients. This test is intended for use to aid in the differential diagnosis of Influenza A, Influenza B and RSV viral infections in humans and is not intended to detect Influenza C.</td></tr><tr><td>Negative results do not preclude Influenza virus infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions. Performance characteristics for influenza A were established during the 2010 to 2011 influenza season when influenza A/H3 and 2009 H1N1 influenza were the predominant influenza A viruses in circulation. When other influenza A viruses are emerging, performance characteristics may vary.</td><td>Negative results do not preclude influenza or RSV virus infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative RSV results be confirmed by culture. Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject Device:Quidel Molecular Influenza A+BAssay</td><td colspan="1" rowspan="1">Predicate Device:Prodesse ProFlu+</td></tr><tr><td colspan="1" rowspan="1">b</td><td colspan="1" rowspan="1">If infection with a novel influenza Avirus is suspected based on currentclinical and epidemiologicalscreening criteria recommended bypublic health authorities, specimensshould be collected with appropriateinfection control precautions for novelvirulent Influenza viruses and sent tostate or local health department fortesting. Viral culture should not beattempted in these cases unless a BSL3+ facility is available to receive andculture specimens.</td><td colspan="1" rowspan="1">vary.If infection with a novel Influenza Avirus is suspected based on currentclinical and epidemiologicalscreening criteria recommended bypublic health authorities, specimensshould be collected with appropriateinfection control precautions fornovel virulent Influenza viruses andsent to state or local healthdepartment for testing. Viral cultureshould not be attempted in thesecases unless a BSL 3+ facility isavailable to receive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">Influenza A virus, influenza B virus</td><td colspan="1" rowspan="1">Influenza A virus, influenza B virus,respiratory syncytial virus</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">nasal swab and nasopharyngeal swab</td><td colspan="1" rowspan="1">nasopharyngeal swab</td></tr><tr><td colspan="1" rowspan="1">Extraction Methods</td><td colspan="1" rowspan="1">bioMérieux easyMAG AutomatedMagnetic Extraction Reagents</td><td colspan="1" rowspan="1">Roche MagNA Pure LC TotalNucleic Acid Isolation Kit or thebioMérieux easyMAG AutomatedMagnetic Extraction Reagents</td></tr><tr><td colspan="1" rowspan="1">Assay Methodology</td><td colspan="1" rowspan="1">PCR-based system for detecting thepresence or absence of viral RNA inclinical specimens</td><td colspan="1" rowspan="1">PCR-based system for detecting thepresence or absence of viral RNA inclinical specimens</td></tr><tr><td colspan="1" rowspan="1">Detection Techniques</td><td colspan="1" rowspan="1">Multiplex assay using differentreporter dyes for each target</td><td colspan="1" rowspan="1">Multiplex assay using differentreporter dyes for each target</td></tr><tr><td colspan="1" rowspan="1">Viral Targets</td><td colspan="1" rowspan="1">Influenza A: Matrix Gene;Influenza B: conserved influenza Bsequence within the neuraminidasegene</td><td colspan="1" rowspan="1">Influenza A: Matrix Gene;Influenza B: Non-structural NS1 andNS2</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Class II Special Controls Guidance Document: Reagents for Detection of Specific Novel Influenza A Viruses (March 2006)   
http://www.fda.gov/cdrh/oivd/guidance/1596.pdf. Guidance on In Vitro Diagnostic Devices to Detect Influenza A Viruses: Labeling and Regulatory Path (April 2006)   
http://www.fda.gov/cdrh/oivd/guidance/1594.pdf.

Guidance on Informed Consent for In Vitro Diagnostic Device Studies Leftover Human Specimens that are Not Individually Identifiable (April 2006) http://www.fda.gov/cdrh/oivd/guidance/1588.pdf.

Draft Guidance on Nucleic Acid Based In Vitro Diagnostic Devices for Detection of Microbial Pathogens (Dec 2005)   
http://www.fda.gov/cdrh/oivd/guidance/1560.html.

CLSI EP17-A: Guidance for Protocols for Determination of Limits of Detection and Limits of Quantitations (Vol. 2, No. 34) (Oct 2004).

CLSI MM13-A: Guidance for the Collection, Transport, Preparation and Storage of Specimens for Molecular Methods (Vol. 25, No. 31) (Dec 2005).

CLSI EP7-A2: Guidance for Interference Testing in Clinical Chemistry (Vol. 25, No.27 Second Ed) (Nov 2005).

CLSI EP12-A: Guidance for User Protocol for Evaluation of Qualitative Test Performance (Vol. 22, No. 14) (Sept 2002).

CLSI MM6-A: Guidance for the Quantitative Molecular Methods for Infectious Diseases (Vol. 23, No.28) (Oct 2003).

CLSI EP5-A2: Guidance for Evaluation of Precision Performance of Quantitative Measurement Methods (Vol. 24, No. 25 Second Ed.) (Aug 2004).

Establishing Performance Characteristics of In Vitro Diagnostic Devices for   
Detection or Detection and Differentiation of Influenza Viruses. Document issued on July 15, 2011,   
http://www.fda.gov/MedicalDevices/DeviceRegulationandGuidance/GuidanceDocum ents/ucm079171.htm.

Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices. Document issued on September 9, 1999. Docket number FDA-1999-D-585

# L. Test Principle:

The real-time RT-PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. Detection of influenza A, B and the Process Control (PRC) is based on three processes: nucleic acid isolation, reverse transcription and real time PCR amplification/detection. Human respiratory specimens (nasal swabs and nasopharyngeal swabs) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. After initial reverse transcription of RNA into complementary DNA (cDNA), amplification proceeds during which the probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This generates an increase in fluorescent signal upon excitation from a light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time. Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

The reproducibility of the Quidel Molecular Influenza A and B assay was evaluated at three laboratory sites. The reproducibility panel and controls were tested at each site by two operators for five days in triplicate (2 operators x 5 days x triplicate testing $_ { \textrm { X 3 } }$ sites $= 9 0$ results per sample). The panels and controls were extracted using the bioMérieux easyMAG system and tested on the Cepheid SmartCycler II.

The reproducibility panel was composed of four simulated samples each for influenza A and influenza B, made by diluting Influenza A H1N1 A/Mexico/4108/2009 or Influenza B Florida into negative nasal matrix. The panel included a medium positive ( $5 \mathbf { x }$ LoD) influenza A sample, a low positive (2x LoD) influenza A sample, a high negative $( 0 . 3 \mathrm { x } \mathrm { L o D } )$ influenza A sample, and an influenza A negative sample. The panel also included a medium positive (5x LoD) influenza B sample, a low positive (2x LoD) influenza B sample, a high negative (0.3x LoD) influenza B sample, and an influenza B negative sample.

<table><tr><td colspan="11" rowspan="1">Reproducibility Results</td></tr><tr><td colspan="1" rowspan="2">PanelMember ID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">TotalResults</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">AVECt</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">Influenza AHighNegative0.3x LoD(7.2E+00)</td><td colspan="1" rowspan="1">3/30(3 positiveesults</td><td colspan="1" rowspan="1">44.27*</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">4/30(4 positiveresults</td><td colspan="1" rowspan="1">43.45*</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">2/30(2 positiveresults</td><td colspan="1" rowspan="1">42.65*</td><td colspan="1" rowspan="1">0.50</td><td colspan="1" rowspan="1">9/90</td></tr><tr><td colspan="1" rowspan="1">Influenza ALowPositive2x LoD(4.8E+01)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">37.72</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">38.05</td><td colspan="1" rowspan="1">3.8</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">37.40</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">MedPositive5x LoD(1.2E+02)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">35.36</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">36.05</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">34.95</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Influenza ANegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">Influenza BHighNegative0.3x LoD(1.8E+00)</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">1/30(1 positiveesult)</td><td colspan="1" rowspan="1">39.9*</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">4/30(4 positiveresults)</td><td colspan="1" rowspan="1">42.43*</td><td colspan="1" rowspan="1">3.2</td><td colspan="1" rowspan="1">5/90</td></tr><tr><td colspan="1" rowspan="1">Influenza BLowPositive2x LoD(1.2E+01)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">35.42</td><td colspan="1" rowspan="1">1.14</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">36.06</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">34.86</td><td colspan="1" rowspan="1">3.6</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Influenza BMedPositive5x LoD(3.0E+01)</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">33.46</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">33.86</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">33.01</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Influenza BNegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">Influenza APositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.19</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.47</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.25</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Influenza BPositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27.76</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">28.05</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">27.57</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">NegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr></table>

\* CV of positive results

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Freeze-Thaw Equivalency, Kit Stability, Master Mix Stability after Rehydration, and Extracted Specimen Stability Studies

(see k112172)

d. Detection limit:

The analytical sensitivity (limit of detection or LoD) of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay was determined using quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ ) cultures of three influenza A strains (one H1N1, one 2009H1N1 and one H3N2) and three influenza B strains, serially diluted in negative nasopharyngeal matrix. Each dilution was extracted using the NucliSENS easyMAG System in replicates of 21 per concentration of virus and tested on the Cepheid Smarcycler® II platform. Analytical sensitivity (LoD) is defined as the lowest concentration at which at least $9 5 \%$ of all replicates tested positive.

LoD Determination   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Stock VirusConcentration(TCID50/ml)</td><td rowspan=1 colspan=1>TestConcentration(TCID50/ml)</td><td rowspan=1 colspan=1>CallRate</td><td rowspan=1 colspan=1>CtAvg</td><td rowspan=1 colspan=1>STDEV</td></tr><tr><td rowspan=1 colspan=1>A/Mexico/4108/2009</td><td rowspan=1 colspan=1>2.89E+08</td><td rowspan=1 colspan=1>2.40E+01</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>40.47</td><td rowspan=1 colspan=1>1.56</td></tr><tr><td rowspan=1 colspan=1>A1/Mal/302/54</td><td rowspan=1 colspan=1>4.19E+08</td><td rowspan=1 colspan=1>7.00E+00</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>41.12</td><td rowspan=1 colspan=1>1.45</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75</td><td rowspan=1 colspan=1>1.10E+08</td><td rowspan=1 colspan=1>3.10E+01</td><td rowspan=1 colspan=1>21/21</td><td rowspan=1 colspan=1>40.11</td><td rowspan=1 colspan=1>2.07</td></tr><tr><td rowspan=1 colspan=1>B/RCHIN 8/05</td><td rowspan=1 colspan=1>3.20E+06</td><td rowspan=1 colspan=1>1.80E+00</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>36.52</td><td rowspan=1 colspan=1>1.04</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>2.56E+06</td><td rowspan=1 colspan=1>6.00E+00</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>38.51</td><td rowspan=1 colspan=1>1.84</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/25/06/04</td><td rowspan=1 colspan=1>3.41E+06</td><td rowspan=1 colspan=1>1.30E+00</td><td rowspan=1 colspan=1>20/21</td><td rowspan=1 colspan=1>40.01</td><td rowspan=1 colspan=1>1.40</td></tr></table>

# e. Analytical specificity (inclusivity):

The inclusivity of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay was evaluated against multiple strains of influenza A and influenza B viruses. The clinical panel consisted of 10 influenza A subtype H1N1, two influenza A subtype 2009H1N1, eight influenza A subtype H3N2, two influenza A subtype H5N1, 13 Influenza B strains. An additional panel of non-clinical influenza isolates was also tested. Each panel member was extracted using the NucliSENS easyMAG instrument and tested in triplicate on the Cepheid SmartCycler II.

The Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay detected $100 \%$ of the influenza A (38/38) and influenza B strains (15/15) at 2 to $3 \mathbf { x }$ LoD levels including pandemic and avian influenza A strains, and recent circulating influenza B strains.

<table><tr><td colspan="5" rowspan="1">Clinical Panel Influenza A viruses</td></tr><tr><td colspan="1" rowspan="2">Subtype</td><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">TCID50/mL</td><td colspan="2" rowspan="1">SmartCycler II</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">B</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1</td><td colspan="1" rowspan="1">H1N1 A/California/07/2009</td><td colspan="1" rowspan="1">1.45E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/New Caledonia/20/1999</td><td colspan="1" rowspan="1">1.12E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/New Jersey/8/76</td><td colspan="1" rowspan="1">3.80E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">5.89E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Denver/1/57</td><td colspan="1" rowspan="1">1.26E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/FM/1/47</td><td colspan="1" rowspan="1">3.80E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">2009 H1N1</td><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">1.40E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A1/Mal/302/54</td><td colspan="1" rowspan="1">4.19E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Taiwan/42/06</td><td colspan="1" rowspan="1">3.39E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">7.24E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/Solomon Islands/3/06</td><td colspan="1" rowspan="1">1.41E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.15E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">7.24E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">4.17E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">4.57E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Perth/16/2009</td><td colspan="1" rowspan="1">9.83E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Uruguay/7/16/2007</td><td colspan="1" rowspan="1">1.03E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">2.19E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">4.17E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Clinical Panel Influenza B viruses</td></tr><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>TCID50/mL</td><td rowspan=1 colspan=2>SmartCycler II</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>B</td></tr><tr><td rowspan=1 colspan=1>B/HongKong/5/72</td><td rowspan=1 colspan=1>6.67E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Panama/45/90</td><td rowspan=1 colspan=1>1.02E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/02/2006</td><td rowspan=1 colspan=1>3.16E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/04/2006</td><td rowspan=1 colspan=1>3.80E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/2004</td><td rowspan=1 colspan=1>1.26E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/25/06/04</td><td rowspan=1 colspan=1>3.41E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59</td><td rowspan=1 colspan=1>1.15E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45</td><td rowspan=1 colspan=1>4.17E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>1.51E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Russia/69</td><td rowspan=1 colspan=1>2.19E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Mass/3/66</td><td rowspan=1 colspan=1>1.38E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40</td><td rowspan=1 colspan=1>1.95E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54</td><td rowspan=1 colspan=1>6.30E+02</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr></table>

<table><tr><td colspan="5" rowspan="1">Non-clinical Influenza Viruses</td></tr><tr><td colspan="1" rowspan="2">Subtype</td><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">TCID50/mL</td><td colspan="2" rowspan="1">SmartCycler II</td></tr><tr><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">B</td></tr><tr><td colspan="1" rowspan="1">H3N2</td><td colspan="1" rowspan="1">A/WI/629-2/2008 (H3N2)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/WI/629-S7(D02473)/2009 (H1N1pdm)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H1N1</td><td colspan="1" rowspan="1">A/WI/629-S5 (D02312)/2009 (H1N1pdm)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H2N2</td><td colspan="1" rowspan="1">A/Mallard/NY/6750/78 (H2N2)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H7N3</td><td colspan="1" rowspan="1">A/Chicken/NJ/15086-3/94 (H7N3)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H9N2</td><td colspan="1" rowspan="1">A/Chicken/NJ/12220/97 (H9N2)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H4N8</td><td colspan="1" rowspan="1">A/Mallard/OH/338/86 (H4N8)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H6N2</td><td colspan="1" rowspan="1">A/Chicken/CA/431/00 (H6N2)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H8N4</td><td colspan="1" rowspan="1">A/Blue Winged Teal/LA/B174/86 (H8N4)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H5N1</td><td colspan="1" rowspan="1">A/Anhui/01/2005(H5N1)-PR8-IBCDC-RG5</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H10N7</td><td colspan="1" rowspan="1">A/GWT/LA/169GW/88 (H10N7)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H11N9</td><td colspan="1" rowspan="1">A/Chicken/NJ/15906-9/96 (H11N9)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H12N5</td><td colspan="1" rowspan="1">A/Duck/LA/188D/87 (H12N5)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H13N6</td><td colspan="1" rowspan="1">A/Gull/MD/704/77 (H13N6)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H14N5</td><td colspan="1" rowspan="1">A/Mallard/GurjevRussia/262/82 (H14N5)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H15N9</td><td colspan="1" rowspan="1">A/Shearwater/Australia/2576/79 (H15N9)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">H16N3</td><td colspan="1" rowspan="1">A/Shorebird/DE/172/2006(H16N3)</td><td colspan="1" rowspan="1">2.00E+02</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td></tr></table>

f. Analytical specificity (cross-reactivity) (see k112172)

g. Interfering Microorganisms (see k112172)

h. Interfering Substances (see k112172)

i. Assay cut-off

The “cutoff value” represents the fluorescent intensity signal (reported in Relative Fluorescent Units) at which a “positive” reaction reaches a relative fluorescent intensity above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

The cut-off for the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ Assay was determined and confirmed through a phased approach. The preliminary threshold was established using data obtained from the LoD studies and from the analysis of a set of clinical specimens. Data from the analysis of multiple replicates near the LoD of the assay were used to establish the threshold such that sensitivity was maximized. Similarly, the latest Ct value from the LoD data was used to set the cut-off. Using the parameters summarized in the table below, a verification study was performed that confirmed the threshold and cut-off values.

<table><tr><td rowspan=1 colspan=3>Quidel Molecular Influenza A+B Assay on the CepheidSmartCycler II Instrument</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>ConfirmedThreshold (RFU)</td><td rowspan=1 colspan=1>Confirmed Ct Cut-Off*</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>PRC</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>45</td></tr></table>

An internal study was completed with the Cepheid SmartCycler® II where a number of PCR reactions were performed in five separate extraction and PCR runs. Each extraction run had alternating four high positive and four high negative samples within the same disposable reaction container. Each PCR run had alternating high positive and negative samples. All the high positive samples were positive for influenza A and influenza B $( 1 0 0 \% )$ . All of the high negative samples were negative for influenza A and influenza B. The data demonstrates that no carry-over or cross contamination was observed with the bioMériuex NucliSENS easyMAG automated nucleic acid extraction instrument and the Cepheid SmartCycler ${ } ^ { \mathfrak { B } } \operatorname { I I }$ instrument.

k. Comparison of transport media (see 112172)

2. Comparison studies:

a. Method comparison with predicate device: Not Applicable

b. Matrix comparison: Not Applicable

3. Clinical studies:

a. Prospective Clinical Studies:

Performance characteristics of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay using the Cepheid SmartCycler® II instrument were established in a prospective study during the 2010-2011 influenza virus season (January to March 2011). Samples used for this study were fresh nasal (427) and nasopharyngeal (352) swab specimens that were collected for routine influenza testing at thirteen (13) sites across the United States. A single specimen was collected per patient and tested within 72-hours of collection at one central location.

A comparator method (a high performance FDA Cleared Influenza A and B molecular test) was used in the evaluation of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay.

The gender and age demographics are presented in the table below

<table><tr><td rowspan=1 colspan=3>Age and Gender Distribution</td></tr><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>M</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>369</td><td rowspan=1 colspan=1>410</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>188 (50.9%)</td><td rowspan=1 colspan=1>199 (48.5%)</td></tr><tr><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>97 (26.2%)</td><td rowspan=1 colspan=1>132 (32.2%)</td></tr><tr><td rowspan=1 colspan=1>22  59 years</td><td rowspan=1 colspan=1>66 (17.9%)</td><td rowspan=1 colspan=1>68 (16.6%)</td></tr><tr><td rowspan=1 colspan=1>≥ 60 years</td><td rowspan=1 colspan=1>18 (4.9%)</td><td rowspan=1 colspan=1>11 (2.7%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>369</td><td rowspan=1 colspan=1>410</td></tr></table>

Seven hundred and seventy-nine (779) fresh specimens (427 nasal swabs and 352 nasopharyngeal swabs) were tested by the subject and comparator device for influenza A and influenza B viral RNA. Twelve (12) of these specimens were invalid on initial testing with the subject device $( 1 . 5 \% )$ . Re-testing of the specimens according to the Interpretation algorithm described above also yielded invalid results. Twenty-three (23) specimens were invalid on initial and repeat testing (as per the device's PI) on the comparator device $( 3 . 0 \% )$ . Nine specimens were invalid in both devices; therefore, a total of 26 invalid specimens have been removed from additional analysis. The table below details the results for the remaining 753 specimens.

<table><tr><td rowspan=1 colspan=4>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Fresh nasal andnasopharyngeal swabs(N=753)</td><td rowspan=1 colspan=3>Comparator: FDA Cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecular</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>8*</td><td rowspan=1 colspan=1>165</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>588</td><td rowspan=1 colspan=1>588</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>596</td><td rowspan=1 colspan=1>753</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>157/157</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.7% to 100%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>588/596</td><td rowspan=1 colspan=1>98.7%</td><td rowspan=1 colspan=1>97.4% to 99.4%</td></tr></table>

\*Eight specimens were negative by FDA Cleared RT-PCR device but positive for influenza A by sequence analysis

<table><tr><td rowspan=1 colspan=4>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Fresh nasal andnasopharyngeal swabs(N=753)</td><td rowspan=1 colspan=3>Comparator: FDA Cleared RT-PCR device</td></tr><tr><td rowspan=1 colspan=1>Quidel Molecular</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>28*</td><td rowspan=1 colspan=1>151</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>602</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>628</td><td rowspan=1 colspan=1>753</td></tr><tr><td rowspan=1 colspan=4>95% CI</td></tr><tr><td rowspan=1 colspan=1>Positive Percent Agreement</td><td rowspan=1 colspan=1>123/125</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>94.3% to 99.8%</td></tr><tr><td rowspan=1 colspan=1>Negative Percent Agreement</td><td rowspan=1 colspan=1>600/628</td><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>93.6% to 97.0%</td></tr></table>

\*Twenty-six specimens were negative by FDA Cleared RT-PCR device but positive for influenza B by sequence analysis. Two specimens were negative by FDA Cleared RT-PCR device but

negative by sequence analysis for influenza B.

The prospective clinical study had a dual infection rate for Influenza A and Influenza B of $2 . 4 \%$ (18/753) using the Quidel Molecular Influenza $\mathbf { A } + \mathbf { B }$ Assay. Three of these dual infections were concordant with the FDA Cleared RT-PCR comparator assay. Three of these dual infections were discordant with the Influenza A results from the FDA Cleared RT-PCR comparator assay. Twelve (12) of these dual infections were discordant with the Influenza B results from the FDA Cleared RT-PCR device comparator assay.

# b. Retrospective Clinical Studies:

Performance characteristics of the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay using the Cepheid SmartCycler ${ } ^ { \mathfrak { R } } \operatorname { I I }$ instrument were also evaluated in a retrospective study of frozen nasopharyngeal swab specimens collected during the 2010 to 2011 influenza virus season (January to March of 2011) for routine influenza testing. For this study the comparator method was a high performance FDA Cleared influenza A and B molecular device.

Three hundred fifty six (356) frozen nasopharyngeal swabs were tested by both the subject and comparator devices for influenza A and influenza B virus viral RNA. Two of these specimens were invalid on initial testing with the subject device $( 0 . 6 \% )$ . Re-testing of the specimens according to the Interpretation Algorithm described above also yielded invalid results. Two specimens were invalid on initial and repeat testing (as per the device's PI) on the comparator device $( 0 . 6 \% )$ . The invalid specimens were removed from performance analyses. The table below details the results for the remaining 352 specimens.

<table><tr><td colspan="4" rowspan="1">Influenza A</td></tr><tr><td colspan="1" rowspan="1">Frozen nasopharyngealswab (N=372)</td><td colspan="3" rowspan="1">Comparator: FDA Cleared RT-PCR device</td></tr><tr><td colspan="1" rowspan="1">Quidel Molecular</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">37</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">315</td><td colspan="1" rowspan="1">315</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">315</td><td colspan="1" rowspan="1">352</td></tr><tr><td colspan="4" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Positive Percent Agreement</td><td colspan="1" rowspan="1">37/37</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">90.5% to 100%</td></tr><tr><td colspan="1" rowspan="1">Negative Percent Agreement</td><td colspan="1" rowspan="1">315/315</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.8% to 100%</td></tr><tr><td colspan="4" rowspan="1">Influenza B</td></tr><tr><td colspan="1" rowspan="1">Frozen nasopharyngealswab (N=372)</td><td colspan="3" rowspan="1">Comparator: FDA Cleared RT-PCR device</td></tr><tr><td colspan="1" rowspan="1">Quidel Molecular</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">5*</td><td colspan="1" rowspan="1">42</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">309</td><td colspan="1" rowspan="1">310</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">314</td><td colspan="1" rowspan="1">352</td></tr><tr><td colspan="4" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Positive Percent Agreement</td><td colspan="1" rowspan="1">37/38</td><td colspan="1" rowspan="1">97.4%</td><td colspan="1" rowspan="1">86.2% to 99.9%</td></tr><tr><td colspan="1" rowspan="1">Negative Percent Agreement</td><td colspan="1" rowspan="1">309/314</td><td colspan="1" rowspan="1">98.4%</td><td colspan="1" rowspan="1">96.3% to 99.5%</td></tr></table>

\*Five specimens were negative by FDA Cleared RT-PCR device but positive for influenza B by sequence analysis.

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

Clinical studies were performed testing prospective specimens collected throughout the United States in the winter of 2011 (January 2011 – March 2011). The number and percentage of positive influenza A cases within this population as determined by the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay using the Cepheid SmartCycler II was $2 1 . 5 \%$ (165/767). The number and percentage of positive influenza B cases within this population as determined by the Quidel Molecular Influenza $\mathbf { A } { + } \mathbf { B }$ assay was $1 9 . 7 \%$ (151/767).

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.