# Applicant

Focus Diagnostics, Inc. 11331 Valley View Street Cypress, California 90630 USA

Establishment Registration No. Contact Person

2023365

Sharon Young Tel 562.240.6680 Fax 562.240.6529

Summary Date Proprietary Name Generic Name Classification Predicate Devices

April 5, 2012   
Simplexa ™ C. difficile Universal Direct   
C.difficile nucleic acid   
Class I   
illumigene ™ C. difficile (K110012)   
BD GeneOhm ™ Cdiff (K081920)

# Intended Use

T Focs DiagosticSplexa.iffieniversal Dct  real-time polyerase chareacion asy anisntended or useon he M Integrated Cyce nstrment orhedetection toxigenicClostr This test aids in the diagnosis of Clostridium difficile associated disease (CDAD).

# Device Description

The test is a real-time polymerase chain reaction (PCR) amplification and detection system that utilizes bil  p fu Cdifficile niveral Dirc t contais primer ezyes,buff ncontolsThe ssyipo aleHeat  stolspls platin if ntal DNA using bi-ncional uoresent probe-primers together wh rever primers.The DA internal contol is u tomonitor potenial prence  R inhibtorsThe assay rget a sequence whic is  a we conev region of C. difficile toxin B gene (tcdB).

Predicate Device Information   

<table><tr><td rowspan=1 colspan=1>Trade Name / Method</td><td rowspan=1 colspan=1>510(k) submitter</td><td rowspan=1 colspan=1>510(k)number</td><td rowspan=1 colspan=1>DecisionDate</td><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>ProductCode(s)</td></tr><tr><td rowspan=1 colspan=1>illumigene ™ C. difficile</td><td rowspan=1 colspan=1>MERIDIAN BIOSCIENCE,INC</td><td rowspan=1 colspan=1>K110012</td><td rowspan=1 colspan=1>02/24/2011</td><td rowspan=1 colspan=1>Microbiology</td><td rowspan=1 colspan=1>OMN</td></tr><tr><td rowspan=1 colspan=1>BD GeneOhm ™ CdiffAssay</td><td rowspan=1 colspan=1>BD DIAGNOSTICS(GENEOHM SCIENCES,INC.)</td><td rowspan=1 colspan=1>K081920</td><td rowspan=1 colspan=1>12/19/2008</td><td rowspan=1 colspan=1>Microbiology</td><td rowspan=1 colspan=1>LLH</td></tr></table>

# Comparison to Predicate

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate 1</td><td rowspan=1 colspan=1>Predicate 2</td></tr><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>Simplexa™ C. difficileUniversal Direct</td><td rowspan=1 colspan=1>illumigene ™ C. difficile</td><td rowspan=1 colspan=1>BD GeneOhm™ Cdiff</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>The Focus DiagnosticsSimplexa ™ C. difficileUniversal Direct is a real-time polymerase chainreaction (PCR) assay and isintended for use on the 3MIntegrated Cycler instrumentfor the detection of toxigenicClostridium difficile toxin Bgene (tcdB) in liquid orunformed stool samplesfrom individuals suspectedof C. difficile infection. Thistest aids in the diagnosis ofClostridium difficileassociated disease (CDAD).</td><td rowspan=1 colspan=1>The Illumigene ™ C. difficileDNA amplification assay,performed on the illumipro-10, is a qualitative in vitrodiagnostic test for the directdetection of toxigenic C.difficile in human stoolspecimens from pediatricand adult patients suspectedof having Clostridiumdifficile-associated disease(CDAD).The Illumigene™ C. difficileassay utilizes loop-mediatedisothermal DNAamplification (LAMP)technology to detect thepathogenicity locus (PaLoc)of toxigenic Clostridiumdifficile. The Clostridiumdifficile PaLoc is a genesegment present in allknown toxigenic C. difficilestrains. The C. difficilePaLoc codes for both theToxin A gene (tcdA) and theToxin B gene (tcdB), hasconserved border regions,and is found at the same siteon the C. difficile genome forall toxigenic strains. TheIllumigene ™ C. difficileassay detects the PaLoc bytargeting a partial DNAfragment on the Toxin Agene. The tcdA target regionwas selected as an intactregion remaining in allknown A+B+ and A-B+toxinotypes.Illumigene ™ C. difficile isintended for use in hospital,reference or state laboratorysettings. The device is notintended for point-of-careuse.</td><td rowspan=1 colspan=1>The BD GeneOhm™ C diffAssay is a rapid in vitrodiagnostic test for the direct,qualitative detection of C.difficile toxin B gene (tcdB)in human liquid or soft stoolspecimens from patientssuspected of havingClostridium difficile-associated disease (CDAD).The test, based on real-timePCR, is intended for use asan aid in diagnosis of CDAD.The test is performeddirectly on the specimen,utilizing polymerase chainreaction (PCR) for theamplification of specifictargets and fluorogenictarget-specific hybridizationprobes for the detection ofthe amplified DNA.</td></tr><tr><td rowspan=1 colspan=1>Assay Targets</td><td rowspan=1 colspan=1>C. difficile toxin B gene(tcdB)</td><td rowspan=1 colspan=1>PaLoc region (encodingtcdA and tcdB)</td><td rowspan=1 colspan=1>C. difficile toxin B gene(tcdB)</td></tr></table>

# K113433

<table><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate 1</td><td rowspan=1 colspan=1>Predicate 2</td></tr><tr><td rowspan=1 colspan=1>Name</td><td rowspan=1 colspan=1>SimplexaT C. difficileUniversal Direct</td><td rowspan=1 colspan=1>illumigene ™ C. difficile</td><td rowspan=1 colspan=1>BD GeneOhm™ Cdiff</td></tr><tr><td rowspan=1 colspan=1>Sample Types</td><td rowspan=1 colspan=1>Liquid or unformed stool</td><td rowspan=1 colspan=1>Unformed human stool</td><td rowspan=1 colspan=1>Liquid or soft stool specimen</td></tr><tr><td rowspan=1 colspan=1>Extraction Methods</td><td rowspan=1 colspan=1>Off-board 10 minutepreheating step.</td><td rowspan=1 colspan=1>Off-board 10 minutepreheating step, followed byvortexing.</td><td rowspan=1 colspan=1>5 minutes of vortexing,sample centrifugation,followed by a 5 minuteheating step.</td></tr><tr><td rowspan=1 colspan=1>Assay Methodology</td><td rowspan=1 colspan=1>The Simplexa C. difficileUniversal Direct assayincorporates direct,qualitative detection oftoxigenic C. difficile DNAfrom clinical specimens inhuman stool specimensusing the 3M IntegratedCycler. The assay utilizesreal-time PCR technologywith fluorescently labeled,bi-functional probe-primerthat amplify and detect aconserved region of thetoxin B (tcdB) gene.</td><td rowspan=1 colspan=1>The assay is performed onthe illumipro-10, and is aqualitative assay for directdetection of toxigenic C.difficile in human stoolspecimens. It utilizes loop-mediated isothermal DNAamplification technology todetect the pathogenicitylocus (PaLoc) of toxigenic C.difficile . The PaLoc is agene segment present in allknown toxigenic C. difficilestrains, and it codes for boththe Toxin A gene (tcdA) andthe Toxin B gene (tcdB).</td><td rowspan=1 colspan=1>The BD GeneOhm Cdiffassay is used for qualitativedetection of C. difficile toxinB gene (tcdB) in humanliquid or soft stool, using theCepheid SmartCycler. Thetest uses real-time PCR forthe amplification of specifictargets which are detectedby fluorogenic target-specifichybridization probes(molecular beacons).The amplification, detectionand interpretation of thesignals are doneautomatically by theCepheid SmartCyclersoftware.</td></tr><tr><td rowspan=1 colspan=1>Detection Techniques</td><td rowspan=1 colspan=1>Real time PCR with bi-functional fluorescent probe-primers using the 3MIntegrated Cycler.</td><td rowspan=1 colspan=1>Isothermal loop-mediatedamplification technology,with detection of lighttransmission change basedon magnesiumpyrophosphate precipitation.</td><td rowspan=1 colspan=1>Real time PCR withmolecular beacons using theCepheid SmartCycler.</td></tr><tr><td rowspan=1 colspan=1>Reference Method</td><td rowspan=1 colspan=1>Toxigenic Culture</td><td rowspan=1 colspan=1>Cytotoxigenic BacterialCulture</td><td rowspan=1 colspan=1>Cytotoxicity Assay</td></tr><tr><td rowspan=1 colspan=1>Limit of Detection</td><td rowspan=1 colspan=1>ATCC 43255560.7 CFU/mL or 1.12CFU/PCRNAP 1A 76.3 CFU/mL or0.15 CFU/PCR</td><td rowspan=1 colspan=1>VPI 10463, 4CFU/test2007431, 32 CFU/testCFI, 64 CFU/test2006240, 32 CFU/testBI8, 64 CFU/test2007858, 32 CFU/test8864, 64 CFU/test</td><td rowspan=1 colspan=1>ATCC 4325510 DNA copies/reaction,4 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Reproducibility</td><td rowspan=1 colspan=1>Low Positive 100% (90/90)Medium Positive 100%(89/89)High Negative 98.9%(89/90)No Template Control (NTC)98.9% (89/90)</td><td rowspan=1 colspan=1>Low Positive 100% (90/90)Positive 100% (60/60)High Negative 91% (82/90)Negative 100% (59/59)</td><td rowspan=1 colspan=1>Low positive 96.7% (87/90)Moderate Positive 100%(90/90)Negative 100% (90/90)Additional reproducibilityusing dilutions of highnegative at 1:100 dilution80% (72/90) and 1:10dilution 23.3% (21/90)</td></tr></table>

# REPRODUCIBILITY

alsy Eac of he ree isused the same pane whic consistd  ontrivd samples n stool-T buffer spid wit Cdif bacterialstock.The panel inclued high negative, low positive, and med posive l s  l oss v ays.On each y two rns we peforme  y ac opera. sumary  te resuls shown in Table 3.

Table 3. Reproducibility Results   

<table><tr><td rowspan=2 colspan=1>•Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>. TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV.</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.20</td><td rowspan=1 colspan=1>1.64</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.26</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.30</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>MediumPositive</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>32.71</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.60</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.65</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>100% (89/89)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PositiveControl(PC)^22}$</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.55</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>32.13</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>32.33</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>95.9% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>. ..v</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>  ^</td><td rowspan=1 colspan=1>.*</td><td rowspan=1 colspan=1>98.9% (89/90)</td><td rowspan=1 colspan=1>94.0% - 99.8%</td></tr><tr><td rowspan=1 colspan=1>NoTemplateControl(NTC)3}$</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>   :</td><td rowspan=1 colspan=1>:3</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>::. </td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1>:&quot;:</td><td rowspan=1 colspan=1>::</td><td rowspan=1 colspan=1>98.9% (89/90)</td><td rowspan=1 colspan=1>94.0% - 99.8%</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=3>148/149 (99.3%)</td><td rowspan=1 colspan=3>149/149 (100.0%)</td><td rowspan=1 colspan=3>150/151 (99.3%)</td><td rowspan=1 colspan=1>447/449 (99.6%)</td><td rowspan=1 colspan=1>98.4% - 99.9%</td></tr></table>

ne replicate was declared "invalid" based on he site operator discretion. It was "Not Detced" pla   Dn  i  tn Nowo samples T andHigh Negative" were excluerom reporting Quantitativ eproducibility Results.

# LIMIT OF DETECTION

T Lim et oase epleiffenival  ssyy lce NAP )wee culud and quanthe LoDwas determe usig e t  the Splexa™C.l UnivrsalDirct iTentativ Lo was dtiusi hree eplicates nse ollow  con using twenty replicates. LoD was determined to be 560.7 CFU/mL or 1.12 CFU/PCR for strain ATCC 43255 and 76.3 CFU/mL or 0.15 CFU/PCR for strain NAP 1A.

# ANALYTICAL REACTIVITY

AnalycaliviyalstrafewevaluatnaivsolTuf. i different strains were tested in triplicate.All of the tested strains were detected Table

Table 4. Analytical Reactivity Results for C. difficile strains   

<table><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Strainv :</td><td rowspan=1 colspan=1>Concentrationtcful/mL)</td><td rowspan=1 colspan=1>Toxinotype¥r</td><td rowspan=1 colspan=1>C.difficile Result#Detected/#Total</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 17857 (870) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 43594 (W1194) A+B+</td><td rowspan=1 colspan=1>1.12x 103</td><td rowspan=1 colspan=1>0      .</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 43596 (545) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^</td><td rowspan=1 colspan=1>0    .</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC 43597 A+B+</td><td rowspan=1 colspan=1>1.12 x 10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC 43598 (1470) A-B+</td><td rowspan=1 colspan=1>1.12 x 10^{3</td><td rowspan=1 colspan=1>VIII</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC 43599 (2022) A+B+</td><td rowspan=1 colspan=1>1.12 x 10</td><td rowspan=1 colspan=1>1      0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC 43600 (2149) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ATCC 51695 (BDMS 18 AN) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>ATCC 700792 (14797-2) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>ATCC 9689 (90556-M6S) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC BAA-1382 (630) A+B+</td><td rowspan=1 colspan=1>1.12 x 10^$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC BAA-1805 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{3}$</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>BAA-1814 A+B+</td><td rowspan=1 colspan=1>1.12x10^}$</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>BAA-1870 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{}$</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>BAA-1871 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>BAA-1872 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>BAA-1873 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>BAA-1874 A+B+</td><td rowspan=1 colspan=1>1.12 x 10^{}$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>BAA-1875 A+B+</td><td rowspan=1 colspan=1>1.12 x 10</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>CCUG 8864 A-B+</td><td rowspan=1 colspan=1>1.12 x 10^</td><td rowspan=1 colspan=1>×</td><td rowspan=1 colspan=1>3/3</td></tr></table>

# CROSS REACTIVITY

al seficar osslcrcts eoal oal cosswere tested. No cross reactivity was observed (Table 5).

Table 5. Cross Reactivity Results   

<table><tr><td colspan="4" rowspan="1">x1                              Tested Cross-Reactants</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Cross Reactant</td><td colspan="1" rowspan="1">Concentration     :</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Acinetobacter Iwofii</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Adenovirus 40</td><td colspan="1" rowspan="1">1.00 × 10³ TCIDso/ml.</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="4" rowspan="1">Tested Cross-Reactants</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Cross Reactant</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.00 × 10^° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. Faecalis</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">1.00  10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed..</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">1.00 × 10 cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">1.00 × 10 cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Candida catenulate</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">1.00 × 10^° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1.00 × 10 cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">1.00 × 10 cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">6.80 x 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Clostridium chauvoei</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic ATCC3593</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Clostridium difficile non-toxigenicATCC43601</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Clostridium fallax</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">6.90 × 10³ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">8.90 x 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Clostridium paraputrificum</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">6.70 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Clostridium sepiticum</td><td colspan="1" rowspan="1">1.00 × 10^³ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr></table>

# K113433

<table><tr><td rowspan=1 colspan=4>Tested Cross-Reactants</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Cross Reactant</td><td rowspan=1 colspan=1>Concentration*</td><td rowspan=1 colspan=1>Result       </td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>Clostridium sordelli</td><td rowspan=1 colspan=1>1.00 × 10^ cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>Clostridium sphenoides</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>Clostridium sporogenes</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>Clostridium symbiosum</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>Clostridium terdium</td><td rowspan=1 colspan=1>1.00 × 10^° cfu/ml</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>Clostridium tetani</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>Collinsella aerofaciens</td><td rowspan=1 colspan=1>8.60 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>Coxsackie virus A16</td><td rowspan=1 colspan=1>1.00 × 10 TCIDso/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>Cytomegalovirus AD-169</td><td rowspan=1 colspan=1>1.00 × 10^³ TCIDs/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>Desulfovibrio piger</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=1>1.00 × 10 TCIDso/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>Edwardsiella tarda</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>Eggerthellalenta</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>1.00 × 10^° cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>Enterococcu raffinosus</td><td rowspan=1 colspan=1>1.00 × 10^° cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>Enterococcus casseliflavus</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>Enterococcus cecorum</td><td rowspan=1 colspan=1>1.00 × 10^ cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>61</td><td rowspan=1 colspan=1>Enterococcus dispar</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross.Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>63</td><td rowspan=1 colspan=1>Enterococcusfaecalis vanB</td><td rowspan=1 colspan=1>1.00  10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>Enterococcusfaecium vanA</td><td rowspan=1 colspan=1>1.00 × 10^° cfu/ml</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>Enterococcusgallinarum vanC</td><td rowspan=1 colspan=1>1.00 × 10^ cfu/ml</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>66</td><td rowspan=1 colspan=1>Enterovirus 71</td><td rowspan=1 colspan=1>5.01 x 104 TCIDso/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>1.00 × 10° cfu/mL    ,</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>Escherichia fergusonii</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>69</td><td rowspan=1 colspan=1>Escherichia hermannii</td><td rowspan=1 colspan=1>1.00 × 10^° cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>Fusobacterium varium</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>Gardnerella vaginalis</td><td rowspan=1 colspan=1>1.00 × 10^ cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>Gemella morbillorum</td><td rowspan=1 colspan=1>1.00 × 10^ cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>Hafnia alvei</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>74</td><td rowspan=1 colspan=1>Helicobacter pylori</td><td rowspan=1 colspan=1>1.00 × 10 cfu/ml</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>75</td><td rowspan=1 colspan=1>Homo sapiens</td><td rowspan=1 colspan=1>3.07 pg/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>Klebsiella pneumoniae subsp.Pneumoniae</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>78</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>1.00 × 10° cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>79</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>1.00 × 10 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr></table>

# K113433

<table><tr><td colspan="4" rowspan="1">Tested Cross-Reactants</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Cross Reactant</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.00 x 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">Leminorela grimontii</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">Norovirus Group I (recombinant)</td><td colspan="1" rowspan="1">8.13 × 104 TCIDsg/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">Porphyromaonas asaccharolytica</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">Providencia stuartli</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">Rotavirus, Strain Wa</td><td colspan="1" rowspan="1">1.00 x 105 TCID5o/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Arizonae(formerly Choleraesuis arizonae)</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Choleraesuis</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Entericaserovar Typhimurium</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.00 x 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">1:00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">109</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">111</td><td colspan="1" rowspan="1">Streptococcus dysgalactiae</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">112</td><td colspan="1" rowspan="1">Streptococcus intermedius</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">113</td><td colspan="1" rowspan="1">Streptococcus uberis</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">114</td><td colspan="1" rowspan="1">Trabulsiella guamensis</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">115</td><td colspan="1" rowspan="1">Veillonella parvula</td><td colspan="1" rowspan="1">1.00 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">116</td><td colspan="1" rowspan="1">Vibrio cholerae</td><td colspan="1" rowspan="1">4.10 x 103 pg/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">117</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/ml</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">Yersinia bercovieri</td><td colspan="1" rowspan="1">1.00 × 10^ cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">119</td><td colspan="1" rowspan="1">Yersinia rohdei</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="4" rowspan="1">Note:1. T  baseline (un-spiked) sample.2. Each replicate of all 119 cross-reactants and baseline samples were "Not Detected</td></tr></table>

# INTERFERENCE

T  a  o tis ly in us spiked into a low positive C. difficile stool-TE buffer matrix and tested. No interference was observed

Table 6. Summary of Interfering Substances and Testing Results for two C. difficile strains   

<table><tr><td colspan="1" rowspan="2">Interferents;</td><td colspan="1" rowspan="2">, :Active Ingredient$3</td><td colspan="1" rowspan="2">InterferentRConcentration</td><td colspan="2" rowspan="1">Detected/Total</td></tr><tr><td colspan="1" rowspan="1">C. difficile Strain-x:ATCC43255 fac{2 }$</td><td colspan="1" rowspan="1">C. difficile Strain - NAPa     1A     5</td></tr><tr><td colspan="1" rowspan="1">1% Hydrocortisone Cream</td><td colspan="1" rowspan="1">Hydrocortisone</td><td colspan="1" rowspan="1">2% (w/v)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Aleve</td><td colspan="1" rowspan="1">Naproxen</td><td colspan="1" rowspan="1">14 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Antacid and Anti-gas generic</td><td colspan="1" rowspan="1">AluminumHydroxide,MagnesiumHydroxide</td><td colspan="1" rowspan="1">0.1 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Antacid Generic</td><td colspan="1" rowspan="1">CalciumCarbonate</td><td colspan="1" rowspan="1">0.1 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">Barium sulfate</td><td colspan="1" rowspan="1">5 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">·3/3</td></tr><tr><td colspan="1" rowspan="1">Fleet</td><td colspan="1" rowspan="1">Mineral Oil</td><td colspan="1" rowspan="1">2% (v/v)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Imodium AD</td><td colspan="1" rowspan="1">Loperamide</td><td colspan="1" rowspan="1">0.005 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">KY Jelly</td><td colspan="1" rowspan="1">Glycerin</td><td colspan="1" rowspan="1">2%(w/v)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Laxative generic</td><td colspan="1" rowspan="1">Sennosides</td><td colspan="1" rowspan="1">0.1 mg/ml</td><td colspan="1" rowspan="1">5/5*</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">Metronidazole</td><td colspan="1" rowspan="1">14 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Milk of Magnesia</td><td colspan="1" rowspan="1">MagnesiumHydroxide</td><td colspan="1" rowspan="1">0.2 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Moist towelettes generic</td><td colspan="1" rowspan="1">BenzalkoniumChloride</td><td colspan="1" rowspan="1">10%(v/v)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">Mucin</td><td colspan="1" rowspan="1">3 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">Nystatin</td><td colspan="1" rowspan="1">10000 USPunits/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">Palmitic acid</td><td colspan="1" rowspan="1">2 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Pepto-Bismol</td><td colspan="1" rowspan="1">BismuthSubsalicylate</td><td colspan="1" rowspan="1">0.175 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Preparation H</td><td colspan="1" rowspan="1">Phenylephrine</td><td colspan="1" rowspan="1">2% (w/v)</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="2">Interferents                 A   </td><td colspan="1" rowspan="2">Active Ingredient;</td><td colspan="1" rowspan="2">↑InterferentConcentrationv.</td><td colspan="2" rowspan="1">, "         Detected/Total</td></tr><tr><td colspan="1" rowspan="1">C. difficile StrainATCC43255</td><td colspan="1" rowspan="1">C.difficile Strain - NAP..        1Ah</td></tr><tr><td colspan="1" rowspan="1">Stearic acid</td><td colspan="1" rowspan="1">Stearic Acid</td><td colspan="1" rowspan="1">4 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Trojan with nonoxynol-9</td><td colspan="1" rowspan="1">Nonoxynol-9</td><td colspan="1" rowspan="1">1.4 mg/ml</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">Vancomycin</td><td colspan="1" rowspan="1">1.4 mg/ml</td><td colspan="1" rowspan="1">5/5*</td><td colspan="1" rowspan="1">3/3</td></tr><tr><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">Whole blood</td><td colspan="1" rowspan="1">3%</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">718**</td></tr></table>

repeat run. repeat run.

# CLINICAL STUDIES

Threxteal tt  cas Cas parpatCnAre . S prspetively colec e speens n teste them wi heSmplexaiffe nival i teste he it e lexaificie niveal ect fozen luo was a ul peo plexafficil s al e proively co fhe West Coas   U n Upper MidWes U.oze alquot ac te ee w s toor txieic cule.ecuced  ct n eed toecutu stg specimens.

For clinical specimens tested at Site , results were also generated using an FDA cleared molecular assay. Smilarly, or clinical specimens tested at Sie2, results were generated using an alternative FA cleared molecular assay.

A toal of 970 prospectively collected stool specimens were obtained from patients with signs and symptoms o iffcrapcholi a obtained.

Clinical Agreement summary results are presented in Table 7 and Table 8 below.

Table 7. Simplexa™ C. difficile Universal Direct Kit versus Direct Toxigenic Culture Method   

<table><tr><td rowspan=1 colspan=1>&quot;</td><td rowspan=1 colspan=3>Reference Method: (Direct Culture + Toxin Assay)</td></tr><tr><td rowspan=1 colspan=1>SimplexaC. difficile Universal Direct Kit</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>i Not Detected</td><td rowspan=1 colspan=1>Total&quot;Th }$</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>118.</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>779</td><td rowspan=1 colspan=1>792</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>838</td><td rowspan=1 colspan=1>969</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>90.1%(118/131)95% C1:83.8-94.1%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>93.0%(779/838)95% Cl:91.0-94.5%</td></tr></table>

Table 8. Simplexa™ C. diffcile Universal Direct Kit versus Enriched Toxigenic Culture Method   

<table><tr><td rowspan=1 colspan=1>$vfrac{}$                      $3.frac}$</td><td rowspan=1 colspan=3>Reference Method: (Enriched Culture + Toxin Assay)</td></tr><tr><td rowspan=1 colspan=1>Simplexa™ C. difficile Universal Direct Kit</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>755</td><td rowspan=1 colspan=1>792</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>969</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>79.6%(144/181)95% C:73.1-84.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>95.8%(755/788)95% C1:94.2-97.0%</td></tr></table>

Note: One sample was inadvertently missed from being cultured.

Intn he pleifnial Dc ssyhe pes e essg two iff FDA-cleared assays; 402 samples were assayed using one FDA cleared molecular assay, and 305 samples were assayed using another FA cleard molecular assay.he testing was performed at two diferent cal sites.These two FDA cleared molecular assays were compared to direct and enriched toxigenic cultures.

In n  oulu  tiviy eyplefn Direct Assay were $9 0 . 1 \%$ $( 9 5 \% \complement 1 ; 8 3 . 8 – 9 4 . 1 \% )$ and $9 3 \%$ $( 9 5 \% complement 1 : 9 1 . 9 4 . 5 \% )$ , respectively, as shown above. The sensitivities and specificities of the two FDA cleared molecular tests were $8 6 . 1 \%$ $( 9 5 \%$ EPY $\mathsf { C l } ; 7 6 . 3 \substack { - 9 2 . 3 \% } \}$ and $9 4 . 8 \%$ $( 9 5 \% \ C ! { : 9 1 . 9 - 9 6 . 8 \% } )$ for the first molecular assay and $8 1 . 8 \%$ $9 5 \%$ CI:65.6-91.4%)and $93 \%$ $( 9 5 \%$ $C 1 ; 8 9 . 3 \substack { - 9 5 . 5 \% }$ , for the second assay.

I  toulu evy Direct Assay were $7 9 . 6 \%$ $( 9 5 \%$ Cl:73.1-84.8%) and $9 5 . 8 \%$ $( 9 5 \%  C ! ; 9 4 . 2 { \cdot } 9 7 \% )$ , respectively, as shown above. The sensitivities and specificities of the two FDA cleared molecular tests were $7 8 . 7 \%$ $( 9 5 \%$ Cl:69-85.9%) and $9 7 . 1 \%$ $9 5 \% 0 . 5 4 . 6 { \cdot } 9 8 . 5 \% )$ for the first molecular assay and $6 9 . 6 \%$ $9 5 \%$ CI:56.7-80.1%) and $9 7 . 2 \%$ $( 9 5 \%$ EPY $\mathsf { C l } { : 9 4 . 3 { - 9 8 . 6 \% } ) }$ , for the second assay.

FOCUS Diagnostics, Inc. $\mathtt { c / o }$ Ms. Sharon Young Sr. Regulatory Affairs Specialist 11331 Valley View Street Cypress, California 90630

Re: K113433 Trade/Device Name: Simplexa™ C. difficile Universal Direct Regulation Number: 21 CFR 866.2660 Regulation Name : Microorganism differentiation and identification device Regulatory Class: Class I Product Code: OMN Dated: April 3, 2012 Received: April 4, 2012

Dear Ms. Young:

We have reviewed your Section 510(k) premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class I (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice

requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section 510(k) premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html.

Sincerely yours,

![](images/34a683091a9cc7316e2c113463be64594d2bac87136244cb1eef23a33c17a25b.jpg)

Sally A. Hojvat, M.Sc., Ph.D.   
Director   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

Enclosure

# Indications for Use

510(k) Number (if known): K113433

Device Name: SimplexaTM C. difficile Universal Direct

Indications for Use:

The Focus Diagnostics Simplexa™M C. difficile Universal Direct is a real-time polymerase chain reaction (PCR) assay and is intended for use on the 3M Integrated Cycler instrument for the detection of toxigenic Clostridium difficile toxin B gene (tcdB) in liquid or unformed stool samples from individuals suspected of C. difficile infection. This test aids in the diagnosis of Clostridium difficile associated disease (CDAD).

(Par  o  up ) AND/OR (ve-The-Co bpluse) (PLEASE DO NOT WRITE BELOW THIS LINE-CONTINUE ON ANOTHER PAGE OF NEEDED)

Concurrence of CDRH, Office of InVitro Diagnostics (OIVD)

Raque Leat for F. ole Division Sign-Off   
Office of In Vitro Diagnostic   
Device Evaluation and Safety 5T0(K) 113433