# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY ONLY TEMPLATE

A. 510(k) Number: K113433   
B. Purpose for Submission: To obtain substantial equivalence for Simplexa™ C. difficile Universal Direct assay   
C. Measurand: Clostridium difficile toxin B gene (tcdB)   
D. Type of Test: Real time PCR   
E. Applicant Focus Diagnostics Inc.   
F. Proprietary and Established Name: Simplexa™ C. difficile Universal Direct Assay   
G. Regulatory Information:

<table><tr><td rowspan=1 colspan=1>Product Code</td><td rowspan=1 colspan=1>Classification</td><td rowspan=1 colspan=1>Regulation Section</td><td rowspan=1 colspan=1>Panel</td></tr><tr><td rowspan=1 colspan=1>OMN</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>21 CFR 866.2660</td><td rowspan=1 colspan=1>83 - Microbiology</td></tr></table>

# H. Intended Use:

1. Intended use:

The Focus Diagnostics Simplexa™ C. difficile Universal Direct is a real-time polymerase chain reaction (PCR) assay and is intended for use on the 3M Integrated Cycler instrument for the detection of toxigenic Clostridium difficile toxin B gene (tcdB) in liquid or unformed stool samples from individuals suspected of C. difficile infection. This test aids in the diagnosis of Clostridium difficile associated disease (CDAD).

# 2. Indications for use:

The Focus Diagnostics Simplexa™ C. difficile Universal Direct is a real-time polymerase chain reaction (PCR) assay and is intended for use on the 3M Integrated Cycler instrument for the detection of toxigenic Clostridium difficile toxin B gene (tcdB) in liquid or unformed stool samples from individuals suspected of C. difficile infection. This test aids in the diagnosis of Clostridium difficile associated disease (CDAD).

3. Special conditions for use statement: For Prescription Use

4. Special instrument requirements:   
3M Integrated Cycler Instrument

# I. Device Description:

The test is a real-time polymerase chain reaction (PCR) amplification and detection system that utilizes bi-functional fluorescent probe-primers for the detection of $C .$ . difficile in liquid or unformed stool. The Focus Diagnostics Simplexa™ C. difficile Universal Direct assay contains sufficient reagents for 100 reactions. Kit components are listed below.

<table><tr><td rowspan=1 colspan=1>Kit Component</td><td rowspan=1 colspan=6>Component Description</td></tr><tr><td rowspan=4 colspan=1>SimplexaTM C.difficile PrimerMix</td><td rowspan=1 colspan=6>Bi-functional fluorescent probe-primers specific for detectionof C. difficile and for the DNA Internal Control includingDNA Internal Control template</td></tr><tr><td></td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>Excitation(nm)</td><td rowspan=1 colspan=1>Emission(nm)</td><td rowspan=1 colspan=1>TargetedGene</td></tr><tr><td></td><td rowspan=1 colspan=1>C.difficile</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495</td><td rowspan=1 colspan=1>520</td><td rowspan=1 colspan=1>Toxin B</td></tr><tr><td></td><td rowspan=1 colspan=1>IC</td><td rowspan=1 colspan=1>Q670</td><td rowspan=1 colspan=1>644</td><td rowspan=1 colspan=1>670</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>SimplexaT MasterMix</td><td rowspan=1 colspan=6>DNA polymerase, buffer and dNTPs</td></tr><tr><td rowspan=1 colspan=1>Simplexa C.difficile PositiveControl (PC)</td><td rowspan=1 colspan=6>Clostridium difficile Genomic DNA</td></tr><tr><td rowspan=1 colspan=1>SimplexaT C.difficile BarcodeCard</td><td rowspan=1 colspan=6>Assay specific parameters</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device names:

a. BD GeneOhm C.diff. Assay b. illumigene C. difficile Assay c. Bartel’s cytotoxicity assay for Clostridium

2. Predicate K numbers:

a. K081920

b. K110012   
c. K833447

3. Comparison with predicate:

Comparison is to BD GeneOhm C. diff Assay (K081920)

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Intended Use</td><td rowspan=1 colspan=1>Detection of C. difficiletoxin B gene</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Matrix</td><td rowspan=1 colspan=1>Liquid or uniformed stoolsamples</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Technology</td><td rowspan=1 colspan=1>Assay based on real timePCR</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>Test is performed on the 3MIntegrated Cycler</td><td rowspan=1 colspan=1>Test uses the CepheidSmart Cycler</td></tr><tr><td rowspan=1 colspan=1>Detection techniques</td><td rowspan=1 colspan=1>PCR with bifunctionalfluorescent primer-probes</td><td rowspan=1 colspan=1>PCR with molecularbeacons</td></tr><tr><td rowspan=1 colspan=1>Reference Method</td><td rowspan=1 colspan=1>Toxigenic Culture</td><td rowspan=1 colspan=1>Cytotoxicity Assay</td></tr></table>

# K. Standard/Guidance Document Referenced:

Draft Guidance for Industry and Food and Drug Administration Staff - Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile. November 2010.

# L. Test Principle:

The test is a real-time polymerase chain reaction (PCR) amplification and detection system that utilizes bi-functional fluorescent probe-primers for the detection of C. difficile in liquid or unformed stool. The assay is composed of two principal steps: (1) Heat treatment of stool samples, (2) Amplification of the C. difficile DNA and internal control DNA using bi-functional fluorescent probe-primers together with reverse primers. The DNA internal control is used to monitor potential presence of PCR inhibitors. The assay targets a sequence which is in a well conserved region of $C .$ . difficile toxin B gene (tcdB).

The 3M Integrated Cycler is a rapid real-time Polymerase Chain Reaction thermocycler used for the identification of nucleic acid from prepared biological samples. The instrument utilizes disc media to contain and process samples. The instrument uses real time flourometric detection to identify targets within the sample wells. The instrument is controlled by an external computer running the Integrated Cycler Studio Software.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Inter-laboratory reproducibility and inter/intra-assay reproducibility were assessed at three sites. Each of the three sites used the same panel, which consisted of contrived samples in stool-TE buffer matrix spiked with C. difficile bacterial stock. The panel included a high negative, low positive and medium positive sample. Each site utilized at least two testing operators and one lot of Simplexa™ C.difficile Universal Direct kit across five days. On each day two runs were performed, one by each operator. These results met the acceptance criteria and are acceptable. A summary of the reproducibility results are shown in the table below.

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>Site 1</td><td rowspan=1 colspan=3>Site 2</td><td rowspan=1 colspan=3>Site 3</td><td rowspan=2 colspan=1>TotalAgreementwithExpectedResults</td><td rowspan=2 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%V</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td><td rowspan=1 colspan=1>AgreementwithExpectedResults</td><td rowspan=1 colspan=1>Avg.Ct</td><td rowspan=1 colspan=1>Total%CV</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.20</td><td rowspan=1 colspan=1>1.64</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.26</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>35.30</td><td rowspan=1 colspan=1>2.04</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>1Medium Positive</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>32.71</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.60</td><td rowspan=1 colspan=1>1.07</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.65</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>100% (89/89)</td><td rowspan=1 colspan=1>95.9% -100.0%</td></tr><tr><td rowspan=1 colspan=1>2Positive Control(PC)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>32.55</td><td rowspan=1 colspan=1>1.11</td><td rowspan=1 colspan=1>29/29</td><td rowspan=1 colspan=1>32.13</td><td rowspan=1 colspan=1>0.87</td><td rowspan=1 colspan=1>31/31</td><td rowspan=1 colspan=1>32.33</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>High Negative</td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>98.9% (89/90)</td><td rowspan=1 colspan=1>94.0% -99.8%</td></tr><tr><td rowspan=1 colspan=1>3No TemplateControl (NTC)</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>29/30</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>98.9% (89/90)</td><td rowspan=1 colspan=1>94.0% -99.8%</td></tr><tr><td rowspan=1 colspan=1>Total Agreement</td><td rowspan=1 colspan=3>148/149 (99.3%)</td><td rowspan=1 colspan=3>149/149 (100.0%)</td><td rowspan=1 colspan=3>150/151 (99.3%)</td><td rowspan=1 colspan=1>447/449(99.6%)</td><td rowspan=1 colspan=1>98.4% -99.9%</td></tr></table>

1One replicate was declared “invalid” based on the site operator discretion. It was “Not Detected”. 2One replicate was “Invalid” at Site 2 and additional replicate was tested in Run-1, Day-1 at Site 3 because the site had thought that one of the three replicates had a ‘bubble’ and therefore as a precaution loaded an additional replicate at the end of one of the run.

3One replicate of the NTC is “Detected” and may be attributed to possible contamination due to handling.

Note: Two samples – “NTC” and “High Negative” were excluded from reporting Quantitative Reproducibility Results.

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Quality control ranges have been established as indicated in the table below. If the controls are not within these parameters, patient results should be considered invalid and the assay repeated. Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations and standard good laboratory practice.

Expected Control Ranges   

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Simplexa™M C. difficile Ct value</td></tr><tr><td rowspan=1 colspan=1>No Template Control (NTC)</td><td rowspan=1 colspan=1>Ct=0</td></tr><tr><td rowspan=1 colspan=1>Positive Control (PC)</td><td rowspan=1 colspan=1>Ct≤40, ≠0</td></tr></table>

# d. Detection limit:

The Limit of Detection (LoD) was determined for the Simplexa™ C. difficile Universal Direct assay by performing limiting dilution studies using bacterial stocks of two strains of each C. difficile bacteria strain. The strains (ATCC 43255 and NAP 1A) were cultured and quantified. The LoD was determined using one lot of the Simplexa™ C. difficile Universal Direct Kit. Tentative LoD was determined using three replicates in screening followed by confirmation using twenty replicates. LoD was determined to be 560.7 CFU/mL or 1.12 CFU/PCR for strain ATCC 43255 and $7 6 . 3 \mathrm { C F U / m L }$ or 0.15 CFU/PCR for strain NAP 1A. These results met the acceptance criteria and are acceptable.

e. Analytical specificity:

# Cross-Reactivity

Analytical specificity for various possible cross-reactants was performed. A total of 119 potential cross-reactants were tested. No cross-reactivity was observed as shown in the Table below.

<table><tr><td colspan="4" rowspan="1">Tested Cross-Reactants</td></tr><tr><td colspan="1" rowspan="1">No.</td><td colspan="1" rowspan="1">Cross Reactant</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Abiotrophia defective</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Acinetobacter baumanii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Acinetobacter lwofii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">Adenovirus 40</td><td colspan="1" rowspan="1">1.00 × 105 TCID50/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">Alcaligenes faecalis subsp. Faecalis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">Anaerococcus tetradius</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">Bacteroides caccae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">Bacteroides merdae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Bacteroides stercoris</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">Bifidobacterium adolescentis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Bifidobacterium longum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">Candida catenulate</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">Cedecea davisae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Citrobacter amalonaticus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Citrobacter koseri</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">Citrobacter sedlakii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">24</td><td colspan="1" rowspan="1">Clostridium beijerinckii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">Clostridium bifermentans</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">26</td><td colspan="1" rowspan="1">Clostridium bolteae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Clostridium butyricum</td><td colspan="1" rowspan="1">6.80 × 10 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">Clostridium chauvoei</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">29</td><td colspan="1" rowspan="1">Clostridium difficile non-toxigenic ATCC43593</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">Clostridium difficile on-toxienicATCC3601</td><td colspan="1" rowspan="1">1.00 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Clostridium fallax</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">32</td><td colspan="1" rowspan="1">Clostridium histolyticum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">Clostridium innocuum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">Clostridium methylpentosum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">Clostridium nexile</td><td colspan="1" rowspan="1">6.90 × 105 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">Clostridium novyi</td><td colspan="1" rowspan="1">8.90 × 105 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Clostridium paraputrificum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">6.70 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">39</td><td colspan="1" rowspan="1">Clostridium ramosum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Clostridium scindens</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Clostridium sepiticum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">42</td><td colspan="1" rowspan="1">Clostridium sordellii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">43</td><td colspan="1" rowspan="1">Clostridium sphenoides</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">44</td><td colspan="1" rowspan="1">Clostridium sporogenes</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">45</td><td colspan="1" rowspan="1">Clostridium symbiosum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">46</td><td colspan="1" rowspan="1">Clostridium terdium</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">Clostridium tetani</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">48</td><td colspan="1" rowspan="1">Collinsella aerofaciens</td><td colspan="1" rowspan="1">8.60 × 105 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">Corynebacterium genitalium</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">Coxsackie virus A16</td><td colspan="1" rowspan="1">1.00 × 105 TCID50/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">Cytomegalovirus AD-169</td><td colspan="1" rowspan="1">1.00 × 105 TCID50/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">52</td><td colspan="1" rowspan="1">Desulfovibrio piger</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">53</td><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">1.00 × 10° TCIDs0/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">Edwardsiella tarda</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">Eggerthellalenta</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">56</td><td colspan="1" rowspan="1">Enterobacter aerogenes</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">58</td><td colspan="1" rowspan="1">Enterococcus raffinosus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">59</td><td colspan="1" rowspan="1">Enterococcus cassliflavus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">60</td><td colspan="1" rowspan="1">Enterococcus cecorum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">61</td><td colspan="1" rowspan="1">Enterococcus dispar</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">62</td><td colspan="1" rowspan="1">Enterococcus hirae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">63</td><td colspan="1" rowspan="1">Enterococcus faecalis vanB</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">64</td><td colspan="1" rowspan="1">Enterococcus faecium vanA</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">65</td><td colspan="1" rowspan="1">Enterococcus gallinarum vanC</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">66</td><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">5.01 × 104 TCID50/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">67</td><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">68</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">69</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">70</td><td colspan="1" rowspan="1">Fusobacterium varium</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">71</td><td colspan="1" rowspan="1">Gardnerella vaginalis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">72</td><td colspan="1" rowspan="1">Gemella morbillorum</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">73</td><td colspan="1" rowspan="1">Hafnia alvei</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">74</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">75</td><td colspan="1" rowspan="1">Homo sapiens</td><td colspan="1" rowspan="1">3.07 pg/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">76</td><td colspan="1" rowspan="1">Klebsiella oxytoca</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">77</td><td colspan="1" rowspan="1">Klebsiella pneumoniae subsp. Pneumoniae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">78</td><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">79</td><td colspan="1" rowspan="1">Lactobacillus reuteri</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">80</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">81</td><td colspan="1" rowspan="1">Leminorela grimontii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">82</td><td colspan="1" rowspan="1">Listeria grayi</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">83</td><td colspan="1" rowspan="1">Listeria innocua</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">84</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">85</td><td colspan="1" rowspan="1">Norovirus Group I (recombinant)</td><td colspan="1" rowspan="1">8.13 × 104 TCID5/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">Peptoniphilus asaccharolyticus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">88</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">89</td><td colspan="1" rowspan="1">Porphyromaonas asaccharolytica</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">90</td><td colspan="1" rowspan="1">Prevotella melaninogenica</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">91</td><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">Proteus penneri</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">93</td><td colspan="1" rowspan="1">Providencia alcalifaciens</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">Providencia rettgeri</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">Providencia stuartli</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">96</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">Pseudomonas putida</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">98</td><td colspan="1" rowspan="1">Rotavirus, Strain Wa</td><td colspan="1" rowspan="1">1.00 × 105 TCID50/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">99</td><td colspan="1" rowspan="1">Salmonella enterica subsp.Arizonaeorerly Choleraesuis arizne</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Choleraesuis</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">101</td><td colspan="1" rowspan="1">Salmonella enterica subsp. Enterica serovarTyphimurium</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">102</td><td colspan="1" rowspan="1">Serratia liquefaciens</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">103</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">104</td><td colspan="1" rowspan="1">Shigella boydii</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">105</td><td colspan="1" rowspan="1">Shigella dysenteriae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">106</td><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">107</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">108</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.00 × 10° cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">109</td><td colspan="1" rowspan="1">Stenotrophomonas maltophilia</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr><tr><td colspan="1" rowspan="1">110</td><td colspan="1" rowspan="1">Streptococcus agalactiae</td><td colspan="1" rowspan="1">1.00 × 106 cfu/mL</td><td colspan="1" rowspan="1">No Cross Reactivity Observed</td></tr></table>

Note:   

<table><tr><td rowspan=1 colspan=4>Tested Cross-Reactants</td></tr><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Cross Reactant</td><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>111</td><td rowspan=1 colspan=1>Streptococcus dysgalactiae</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>Streptococcus intermedius</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>113</td><td rowspan=1 colspan=1>Streptococcus uberis</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>Trabulsiella guamensis</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>115</td><td rowspan=1 colspan=1>Veillonella parvula</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>116</td><td rowspan=1 colspan=1>Vibrio cholerae</td><td rowspan=1 colspan=1>4.10 × 10-3 pg/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>117</td><td rowspan=1 colspan=1>Vibrio parahaemolyticus</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>Yersinia bercovieri</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr><tr><td rowspan=1 colspan=1>119</td><td rowspan=1 colspan=1>Yersinia rohdei</td><td rowspan=1 colspan=1>1.00 × 106 cfu/mL</td><td rowspan=1 colspan=1>No Cross Reactivity Observed</td></tr></table>

1. Total 10 runs were performed to test 119 cross reactants in triplicate. Additionally, each run included five replicates of baseline (unspiked) sample. 2. Each replicate of all 119 cross-reactants and baseline samples were "Not Detected".

# Interference:

The performance of this assay was evaluated with potentially interfering substances that may be present in stool specimens at the concentrations indicated in the Table below. A total of 21 potentially interfering substances were tested. No Interference was observed.

<table><tr><td rowspan=1 colspan=1>Interferents</td><td rowspan=1 colspan=1>Active Ingredient</td><td rowspan=1 colspan=1>InterferentConcentration</td><td rowspan=1 colspan=2>DeteC. difficile         C. difficile Strain -Strain-              NAPATCC4325              1</td></tr><tr><td rowspan=1 colspan=1>1% HydrocortisoneCream</td><td rowspan=1 colspan=1>Hydrocortisone</td><td rowspan=1 colspan=1>2% (w/v)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Alleve</td><td rowspan=1 colspan=1>Naproxen</td><td rowspan=1 colspan=1>14 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Antacid and Anti-gasgeneric</td><td rowspan=1 colspan=1>Aluminum Hydroxide,Magnesium Hydroxide</td><td rowspan=1 colspan=1>0.1 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Antacid Generic</td><td rowspan=1 colspan=1>Calcium Carbonate</td><td rowspan=1 colspan=1>0.1 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>Barium sulfate</td><td rowspan=1 colspan=1>5 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Fleet</td><td rowspan=1 colspan=1>Mineral Oil</td><td rowspan=1 colspan=1>2% (v/v)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Imodium AD</td><td rowspan=1 colspan=1>Loperamide</td><td rowspan=1 colspan=1>0.005 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>KY Jelly</td><td rowspan=1 colspan=1>Glycerin</td><td rowspan=1 colspan=1>2%(w/v)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Laxative generic</td><td rowspan=1 colspan=1>Sennosides</td><td rowspan=1 colspan=1>0.1 mg/ml</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>Metronidazole</td><td rowspan=1 colspan=1>14 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Milk of Magnesia</td><td rowspan=1 colspan=1>Magnesium Hydroxide</td><td rowspan=1 colspan=1>0.2 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Moist towelettes</td><td rowspan=1 colspan=1>Benzalkonium Chloride</td><td rowspan=1 colspan=1>10%(v/v)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>3 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>Nystatin</td><td rowspan=1 colspan=1>10000 USP units/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Palmitic acid</td><td rowspan=1 colspan=1>Palmitic acid</td><td rowspan=1 colspan=1>2 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Pepto-Bismol</td><td rowspan=1 colspan=1>Bismuth Subsalicylate</td><td rowspan=1 colspan=1>0.175 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>Phenylephrine</td><td rowspan=1 colspan=1>2% (w/v)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Stearic acid</td><td rowspan=1 colspan=1>Stearic Acid</td><td rowspan=1 colspan=1>4 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Trojan with nonoxynol-</td><td rowspan=1 colspan=1>Nonoxynol-9</td><td rowspan=1 colspan=1>1.4 mg/ml</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>Vancomycin</td><td rowspan=1 colspan=1>1.4 mg/ml</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>Whole blood</td><td rowspan=1 colspan=1>3%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>7</td></tr></table>

\*One replicate reported as “Invalid” due to IC failure in initial run of three replicates. All three replicates reported as "Detected" in3 8 repeat run. \*\*One replicate reported as "Not Detected" in initial run of three replicates. However all five replicates reported as "Detected" in repeat run.

Analytical reactivity of additional strains of $C .$ . difficile was evaluated in negative stool-TE buffer matrix. Quantified bacterial material was spiked into the negative stool-TE buffer matrix at a single dilution. A total of 20 different strains were tested in triplicate. All of the tested strains were detected as shown in the table below

Analytical Reactivity Results for C. difficile strains   

<table><tr><td rowspan=1 colspan=1>No.</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration(cfu/mL)</td><td rowspan=1 colspan=1>Toxinotype</td><td rowspan=1 colspan=1>C.difficile Result#Detected / #Total</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>ATCC 17857 (870) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>ATCC 43594 (W1194) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>ATCC 43596 (545) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>ATCC 43597 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>ATCC 43598 (1470) A-B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>VII</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>ATCC 43599 (2022) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>ATCC 43600 (2149) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>ATCC 51695 (BDMS 18 AN) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>ATCC 700792 (14797-2) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>ATCC 9689 (90556-M6S) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>ATCC BAA-1382 (630) A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>ATCC BAA-1805 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>Ⅲ</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>BAA-1814 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>XXII</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>BAA-1870 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>BAA-1871 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>BAA-1872 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>BAA-1873 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>BAA-1874 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>BAA-1875 A+B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>V</td><td rowspan=1 colspan=1>3/3</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>CCUG 8864 A-B+</td><td rowspan=1 colspan=1>1.12 x 103</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>3/3</td></tr></table>

f. Assay cut-off:

Initial reportable range (assay cut-off) was determined by an analysis of Limit of Detection (LoD) Studies and Method Comparison studies during the feasibility phase where 45 cycles of amplification were run. The cycling time was later reduced to 40 cycles as testing demonstrated that samples at LoD were approximately at 38 cycles. Reportable range (assay cut-off) was determined by analysis of a Limit of Detection study and the Method Comparison data generated during assay verification testing. Forty cycles of amplification were performed during assay development to allow for the confirmation of the assay cut off. The Limit of Detection for C difficile was defined as the lowest concentration with $2 9 5 \%$ detection for at least 24 replicates as determined by probit analysis. Analysis of the Limit of Detection study shows that the range of Ct values for the C. difficile positive samples at or just above the Limit of Detection was ${ < } 3 9$ . The same LoD sample returned an average Ct value of 36.8. Review of the Method Comparison data showed that $9 5 \%$ of the positive samples have a Ct value $< 3 8 . 7$ . Samples which were positive for $C .$ . difficile had Ct values in the range of 23.3 to 39.7. Based on the available data, the assay cut-off was set at $\mathrm { C t } = 4 0$ .

2. Comparison studies:

a. Method comparison with predicate device:

Clinical specimens were tested using the Simplexa™ C. difficile Universal Direct assay and two different FDA cleared assays. A total of 402 samples were assayed using one FDA cleared molecular assay, and 305 samples were assayed using another FDA cleared molecular assay. The testing was performed at two different clinical sites. These two FDA cleared molecular assays and the Simplexa™ C. difficile were compared to direct and enriched toxigenic cultures.

In comparison to direct toxigenic culture, the sensitivity and specificity of the Simplexa™ C. difficile Universal Direct Assay were $9 0 . 1 \%$ $9 5 \%$ CI:83.8- $9 4 . 1 \% )$ and $93 \%$ $( 9 5 \% \mathrm { C I } { : } 9 1 { - } 9 4 . 5 \% )$ , respectively. The sensitivities and specificities of the two FDA cleared molecular tests were $8 6 . 1 \%$ $9 5 \%$ CI:76.3- $9 2 . 3 \% )$ and $9 4 . 8 \%$ $( 9 5 \% \mathrm { C I } { : 9 1 . 9 { - 9 6 . 8 \% } } )$ for the first molecular assay and $8 1 . 8 \%$ $9 5 \% \mathrm { C I } { : 6 5 . 6 { - 9 1 . 4 \% } }$ and $93 \%$ $9 5 \% \mathrm { C I } { : 8 9 . 3 { - 9 5 } . 5 \% } )$ , for the second assay.

In comparison to enriched toxigenic culture, the sensitivity and specificity of the Simplexa™ C. difficile Universal Direct Assay were $7 9 . 6 \%$ $9 5 \%$ CI:73.1- $8 4 . 8 \% $ and $9 5 . 8 \%$ $( 9 5 \% \mathrm { C I } { : 9 4 . 2 - 9 7 \% } )$ , respectively. The sensitivities and specificities of the two FDA cleared molecular tests were $78 . 7 \%$ ( $9 5 \%$ CI:69.0- $8 5 . 9 \% $ and $9 7 . 1 \%$ $( 9 5 \% \mathrm { C I } { : 9 4 . 6 { - 9 8 . 5 \% } } )$ for the first molecular assay and $6 9 . 6 \%$ $( 9 5 \% \mathrm { C I } { : } 5 6 . 7 { - } 8 0 . 1 \% )$ and $9 7 . 2 \%$ $( 9 5 \% \mathrm { C I } ; 9 4 . 3 { - } 9 8 . 6 \% )$ , for the second assay. These results met the acceptance criteria and are acceptable.

$b$ . Matrix comparison: Not applicable

3. Clinical studies:

a. Clinical Sensitivity:

A total of 970 prospectively collected stool specimens were obtained from patients with signs and symptoms of $C .$ . difficile infection from varied geographic locations. Demographic information, including age, gender was noted. Testing was performed at three external testing sites located on the East Coast. Tables below show results comparing the device to direct toxigenic culture method and to enriched toxigenic culture method.

<table><tr><td rowspan=1 colspan=1>canmaGYellowb</td><td rowspan=1 colspan=3>Reference Method: (Direct Culture + Toxin Assay)</td></tr><tr><td rowspan=1 colspan=1>Simplexa™M C. difficile Universal DirectKit</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>779</td><td rowspan=1 colspan=1>792</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>131</td><td rowspan=1 colspan=1>838</td><td rowspan=1 colspan=1>969</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>90.1%(118/131)95% CI:83.8-94.1%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>93.0%(779/838)95% CI:91.0-94.5%</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Reference Method: (Enriched Culture + Toxin Assay)</td></tr><tr><td rowspan=1 colspan=1>Simplexa™M C. difficile Universal DirectKit</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>144</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>177</td></tr><tr><td rowspan=1 colspan=1>Not Detected</td><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>755</td><td rowspan=1 colspan=1>792</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>181</td><td rowspan=1 colspan=1>788</td><td rowspan=1 colspan=1>969</td></tr><tr><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=3>79.6%(144/181)95% CI:73.1-84.8%</td></tr><tr><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=3>95.8%(755/788)95% CI:94.2-97.0%</td></tr></table>

Note: One sample was inadvertently missed from being cultured.

The Simplexa $C .$ . difficile Universal direct Assay showed a sensitivity of $9 0 . 1 \%$ $( 8 3 . 8 - 9 4 . 1 \% \mathrm { C I } )$ and a specificity of $93 \%$ $( 9 1 - 9 4 . 5 \% )$ I) when compared to direct toxigenic culture method.

b. Clinical specificity: See 3a. above   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

The prevalence of $C .$ . difficile varies between institutions. The frequency is affected by patient population, type of institution, and epidemiology. In the

SimplexaTM C. difficile Universal Direct prospective study, 177 of the 969 or $1 8 . 3 \%$ of the samples tested were positive. The percent positivity between the sites varied from 14.9 to 21.6.

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.