# 510(k) Summary Idaho Technology Inc.

Modification of FilmArray® Respiratory Panel (RP) to add assays for Coronavirus OC43, Coronavirus 229E, Bordetella pertussis, Mycoplasma pneumoniae and Chlamydophila pneumoniae ..

Introduction: According to the requirements of 21 CFR 807.92, the following information provides sufficient detail to understand the basis for a determination of substantial equivalence.

# Submitted by:

Idaho Technology Inc 390 Wakara Way Salt Lake City, UT 84108

Telephone: 801-736-6354   
Facsimile: 801-588-0507

Contact: Beth Lingenfelter, ext. 407

Date Submitted: January 27, 2012

# Device Name and Classification:

Trade Name: FilmArray® Respiratory Panel (RP)

Regulation Number: 21 CFR 866.3980, 21 CFR 866.3065, 21 CFR 866.3375,and 21   
CFR.3120

Classification Name: Respiratory Viral Panel Multiplex Nucleic Acid Assay, Bordetella spp. serologic reagents, Mycoplasma spp. serologic reagents, and Chlamydia serological reagents

# Predicate Device:

K 103175 and K110764 - FilmArray Respiratory Panel (RP) System

# Intended Use:

FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype ${ 2 0 0 9 } \mathrm { H } 1$ , Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Rhinovirus/Enterovirus, Respiratory Syncytial Virus, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test or lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the Film Array RP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis, Coronavirus 229E, Coronavirus OC43, Influenza A/H1, Influenza A/H3, Influenza A/2009 H1, Influenza B, Mycoplasma pneumoniae Parainfluenza Virus 1, Parainfluenza Virus 2, and Parainfluenza Virus 4 were established primarily with retrospective clinical specimens. Performance characteristics for Chlamydophila pneumoniae were established primarily using contrived clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 2 and serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture).

The FilmArray RP assay for Coronavirus OC43 may cross-react with some isolates of Coronavirus HKU1. A dual positive result may be due to cross-reactivity or may indicate a co-infection.

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circulation. Performance of detecting influenza A may vary if other influenza A strains are circulating or a novel influenza A virus emerges. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

# Device Description:

The FilmArray RP System is a multiplex nucleic acid test system composed of the FilmArray instrument, the FilmArray software (preinstalled on a laptop computer) and the FilmArray RP pouch. The FilmArray RP pouch contains freeze-dried reagents to perform nucleic acid purification, reverse transcription, and nested, multiplex PCR with DNA melt analysis. The Respiratory Panel (RP) pouch identifies 20 common and emerging viral respiratory pathogens (see Table 1).

Table 1. Organisms Detected by the FilmArray Respiratory Panel   

<table><tr><td>Viral Respiratory Pathogens</td><td>$a  }$</td></tr><tr><td>Influenza A H1 subtype H3 subtype 2009 H1 subtype Influenza B</td><td></td></tr><tr><td>Adenovirus Coronavirus 229E Coronavirus HKU1 Coronavirus NL63 Coronavirus OC43 Human Metapneumovirus</td><td></td></tr><tr><td>Parainfluenza Virus 2 Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus Rhinovirus and Enterovirus</td><td></td></tr><tr><td>Bacterial Respiratory Pathogens Bordetella pertussis</td><td></td></tr></table>

A test is initiated by loading Hydration Solution and an unprocessed patient nasopharyngeal swab (NPS) specimen (i.e., specimen mixed with Sample Buffer) into the FilmArray RP pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format; the addition of Hydration Solution and specimen/Sample Buffer Mix rehydrates the reagents. After the pouch is prepared, the

FilmArray software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run.

The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. When a bladder is inflated over a reagent blister, it forces liquid from the blister into connecting channels. Alternatively, when a seal is placed over a connecting channel it acts as a valve to open or close a channel. In addition, electronically controlled pneumatic pistons are positioned over multiple plungers in order to deliver the rehydrated reagents into the blisters at the appropriate times. Two Peltier devices control heating and cooling of the pouch to drive the reverse transcription reactions, the PCR reactions, and the melting curve analysis.

Nucleic acid extraction occurs within the FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single, large volume, highly multiplexed reverse transcription PCR (rt-PCR) reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green®Plus, Idaho Technology). This second master mix solution, is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The second stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the $2 ^ { \mathsf { n d } }$ ual signature amplicons denoting the presence of specific viral or bacterial targets. A digital camera placed in front of the second stage PCR captures fluorescent images of the PCR reactions in real time.

The FilmArray software automatically interprets the results of each DNA melting curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

# Substantial Equivalence:

The FilmArray Respiratory Panel System is substantially equivalent to the FilmArray Respiratory Panel System K103175 (cleared on February 17, 2011) and K110764 (cleared on April 27, 201 1). Both were determined to be class II devices.

The following tables compare the FilmArray RP to the previously cleared FilmArray RP (K103175 and K110764). The first table outlines the similarities between the two systems and the following table outlines the differences.

# Similarities between the New Device and the Predicate.

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:FilmArray Respiratory Panel System</td><td rowspan=1 colspan=1>Predicate:FilmArray Respiratory Panel System(K103175 and K110764)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>RNA/DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Nested multiplex RT-PCR followed by highresolution melting analysis to confirmidentity of amplified product.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrümentation</td><td rowspan=1 colspan=1>FilmArray Instrument</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>Less than I hour</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TestInterpretation</td><td rowspan=1 colspan=1>Automated test interpretation and reportgeneration. User cannot access raw data.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>Sample Processing is automated in theFilmArray instrument.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reagent Storage</td><td rowspan=1 colspan=1>Reagents are stored at room temperature.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>UserComplexity</td><td rowspan=1 colspan=1>Moderate/Low</td><td rowspan=1 colspan=1>Same</td></tr></table>

# Differences between the New Device and the Predicate.

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:FilmArray Respiratory Panel System</td><td rowspan=1 colspan=1>Predicate:FilmArray Respiratory Panel System(K103175 and K110764)</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Same as predicate with additionalorganisms: Mycoplasma pneumoniae,Chlamydophila pneumoniae, Bordetellapertussis, Coronavirus 229E, andCoronavirus OC43.</td><td rowspan=1 colspan=1>Influenza A, Influenza A subtype H1,Influenza A subtype H3, Influenza Asubtype 2009 H1, Influenza B, RespiratorySyncytial Virus, Human Metapneumovirus,Adenovirus, Parainfluenza 1, Parainfluenza2, Parainfluenza virus 3,Parainfluenza 4,Rhinovirus/Enterovirus, Coronavirus HKU1and Coronavirus NL63.</td></tr></table>

# Summary of Performance Data for Coronavirus 229E, Coronavirus OC43, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae

# Clinical Performance

The clinical performance of the FilmArray RP system for Coronavirus 229E, Coronavirus OC43, B. pertussis, C. pneumoniae, and M. pneumoniae was established during a prospective study at 3 U.S. clinical sites where enrollment spanned an 1 1 month time period encompassing two respiratory seasons (December 2009 - May 2010 and September 2010  January 2011). Subjects with signs and/or symptoms of respiratory infection were invited to participate. Upon obtaining informed consent, NPS samples were collected for FilmArray and comparator testing; a second respiratory sample was collected from each subject for viral culture reference testing. A total of 1 144 subjects were initially enrolled in the study (857 between December 2009 and May 2010; 287 between September 2010 and January 201 1) and four were withdrawn. Specimens from 20 subjects were omitted from analysis due to improper storage prior to testing, and three specimens were omitted due to external control failures on the day of testing. Table 2 provides a summary of demographic information for the remaining 1117 subjects that participated in the prospective study.

Table 2. Demographic Summary for FilmArray RP Prospective Study   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>600 (54%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>517 (46%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>EM</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>719 (64%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>124 (11%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>190 (17%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>84 (8%)</td></tr></table>

Each NPS specimen was tested with the FilmArray RP. The performance of the FilmArray RP for Coronavirus 229E, Coronavirus OC43, $B .$ pertussis, C. pneumoniae, and M. pneumoniae was evaluated by comparing the FilmArray RP test result for each virus or bacteria with the appropriate comparator/reference methods shown in Table 3.

Table 3. Reference/Comparator Methods Used to Assess FilmArray RP performance   

<table><tr><td>Organism/Virus 5.</td><td>Reference/Comparator Method(s)</td></tr><tr><td>Coronavirus 229E</td><td rowspan="3">. 2 PCR tests of patient specimen with bi-</td></tr><tr><td>Coronavirus OC43</td></tr><tr><td>Bordetella pertussis</td></tr><tr><td>Chlamydophila pneumoniae</td><td>directional sequence confirmation&#x27;</td></tr><tr><td>Mycoplasma pneumoniae</td><td></td></tr></table>

Prormance of the FilmArray RP system detecting Coronavirus 229E, Coronavius OC43, Bordetela pertussis, Chlamydophila peumoniae, and Mycoplasma pneumoniae, respectively, was compared to a predetermined algorithm that ucpos opratormethosThemetho conss  twoanalyticallyvalat assay ollw y es    hpl the FlmArray assay Noe te cparator R assay verappeay FlArray plicon equece eve s gene as taredTrue" postivs wee conire s y sapl that had i-real equnc meeting pre-defined quality acceptance criteria that matched sequences deposited in the National Center for BiotehnologyInformation (NCBl) GenBankdatabase (www.ncbi.nlm.nih.gov) with acceptable E-values. "Tru negatives were considered as any sample that tested negative by both of the comparator PCR assays.

A total of 1117 specimens were evaluated for Coronavirus 229E, Coronavirus OC43, $B$ pertussis, C. pneumoniae, and $M .$ pneumoniae in this study. Clinical sensitivity or positive percent agreement (PPA) was calculated as $1 0 0 \% \times \left( \mathrm { T P / \Delta T P + F N } \right)$ . True positive (TP) indicates that both the FilmArray RP and comparator method had a positive result for this specific analyte and false negative (FN) indicates that the FilmArray result was negative while the comparator result was positive. Specificity or negative percent agreement (NPA) was calculated as $100 \% \times$ $( \mathrm { T N } / \mathrm { T N } + \mathrm { F P } )$ . True negative (TN) indicates that both the FilmArray RP and the comparator method had negative results and a false positive (FP) indicates that the FilmArray RP result was positive but the comparator results was negative. The exact binomial two-sided $9 5 \%$ confidence interval was calculated. The results are summarized in Table 4.

Table 4. Clinical Sensitivity and Specificity for the FilmArray RP Prospective Clinical Study   

<table><tr><td rowspan=2 colspan=1>1              26:• Analyte</td><td rowspan=1 colspan=3> y                   ?Positive Percent Agreement &quot;</td><td rowspan=1 colspan=3>Negative Percent Agreement</td></tr><tr><td rowspan=1 colspan=1>TP/TP+ FN</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>TN/TN + Fpa</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E         &lt;</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>73.5-100%</td><td rowspan=1 colspan=1>1103/1105a</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4-100%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43      :t</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76.8-100%</td><td rowspan=1 colspan=1>1098/1103b.c</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0-99.9%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>54.1-100%</td><td rowspan=1 colspan=1>1110/1111</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5-100%</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1116/1116</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7-100%</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>39.8-100%</td><td rowspan=1 colspan=1>1113/1113</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7-100%</td></tr></table>

a. CoV-229E was ientifed by bi-drectional sequence analysisn 1/2 false positive specimes using an leae assay. b. CoV-OC43 was detected in 2/5 false positive specimens using an alternate assay with bi-directional sequence analysis. C. 2/5 false positives were determined to be cross-reactiv products derived from amplification of CoV-HKU vius nucleic acid with the CoV-OC43 assay primers.

A total of 121 co-infections $1 0 . 8 \%$ of all analyzed specimens; 121/1117) were detected by FilmArray during this study. The FilmArray detected 21 co-infections involving Coronavirus 229E, Coronavirus OC43, $B .$ pertussis, C. pneumoniae, and M. pneumoniae (Table 5), representing approximately $1 7 . 4 \%$ of all co-infections detected (21/121).

Table 5. Co-infections Involving CoVs OC43, CoV 229E, B. pertussis, C. pneumoniae, and M. ' pneumoniae as Detected by FilmArray RP   

<table><tr><td rowspan=1 colspan=4>Distinct Co-infection CombinationsDetected by FilmArray RP</td><td rowspan=2 colspan=1>TotalCo-infections$a$</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections*</td><td rowspan=2 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td><td rowspan=1 colspan=1>Analyte4</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>HRV/Enterob</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>.</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>CoV OC43 (2), RSV$(2)}$</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>CoV HKUI</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>CoV OC435d</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>Adenovirus  •</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Y</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adeno, RSVc</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>CoV NL63</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>CoV 229E, RSVc</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>I</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>Y</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>HRV/Enterob</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>•</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/Entero</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Adeno^, B. pertussisc</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>:</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>PIV 2</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=5>aTotal Co-infections21</td><td rowspan=1 colspan=2>Total Analytes in 21 Co-infections = 46Discrepant Analytes = 12/46</td></tr></table>

a. A discrepant co-infection or discrepant analyte was defined as one that was detected by FilmArray RP but not detected by the reference/comparator methods.   
b. HRV/Entero was not analyzed by a comparator method for $1 / 2$ of the $B$ pertussis $^ +$ HRV/Entero, or any of the HRV/Entero CoV OC43 co-infections due to the HRV/Entero analyte being cleared (k103175) prior to testing of these specimens.   
C. These 11 discrepant analytes were investigated using bi-directional sequence analysis: $6 / 1 1$ analytes were deted. Those ot detece included Adenovirus and 1 B. pertussis (Adeno/B. pertussis/HR-Entero infection), RSV (1 COV-OC43/RSV infection) and 2 CoV-OC43 (2 CoV-HKU1/CoV-OC43 infections).   
d. Both discrepant CoVOC43 results were false-positive due to cross-reactivity of the OC43 assay with HKU1 v heesUs wieoe psbly -civiy hboe reported as detected in the same specimen.

Table 6 provides a summary of the FilmArray RP test results for Coronavirus 229E, Coronavirus OC43, B. pertussis, C. pneumoniae, and $M .$ pneumoniae obtained during this study, including the prevalence of each organism detected by the FilmArray RP System and distribution among the age groups. The majority of Coronavirus 229E, Coronavirus OC43, B. pertussis, $C .$ pneumoniae, and $M$ pneumoniae organisms were detected in children five years and younger $( 6 0 \%$ ; 27/45). Of the remaining organisms $1 1 \%$ (5/45) were detected in subjects 6-21 years of age, $20 \%$ (9/45) in subjects 22-49 years of age, and $9 \%$ (4/45) in subjects $\ge 5 0$ years of age.

Table 6. Prevalence and Age Distribution of Analytes in the Clinical Study   

<table><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>Total(ExpectedValue</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>6-21years</td><td rowspan=1 colspan=1>22-49. years</td><td rowspan=1 colspan=1>≥50years:</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>14(1.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>I</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>19 (1.7%)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>7(0.6%)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>1 (0.09%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>4 (0.4%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr></table>

All five organisms were of low prevalence during the clinical study $( 0 . 0 9 - 1 . 7 \% )$ . To supplement the results of the prospective clinical study, an evaluation of preselected archived samples was performed.

# Testing of Preselected Archived Specimens

In addition to the prospective clinical study, archived clinical NPS specimens were also tested using the FilmArray RP. The specimens were selected because they had previously tested positive for B. pertussis, Coronavirus 229E, Coronavirus OC43, or M. pneumoniae. The analyte content of each specimen was confirmed using analyte-specific PCR and bi-directional sequencing; the results of confirmation testing (positive or negative for a particular analyte) were used for the final analysis regardless of the previous laboratory test result.

The specimens were organized into "test panels" and randomized such that the users testing the samples with the FilmArray RP were blinded as to the expected test result. Each panel contained specimens known to be positive and negative for the specific analyte being evaluated allowing the calculation of a positive percent agreement (PPA) and a negative percent agreement (NPA). A summary of the available demographic information of the tested samples is provided in Table 7, and Table 8 shows the performance data summary for B. pertussis, Coronavirus 229E, Coronavirus OC43, and M. pneumoniae, and the results of confirmation testing for each analyte are detailed in the footnotes of Table 8.

Table 7. Demographic Summary of FilmArray RP Archived Specimen Study   

<table><tr><td colspan="2" rowspan="1">'Total Specimens</td><td colspan="1" rowspan="1">305   :</td></tr><tr><td colspan="1" rowspan="3">Sex</td><td colspan="1" rowspan="1">Female (%)</td><td colspan="1" rowspan="1">126 (41.3%)</td></tr><tr><td colspan="1" rowspan="1">Male (%)</td><td colspan="1" rowspan="1">131 (43%)</td></tr><tr><td colspan="1" rowspan="1">Unknown</td><td colspan="1" rowspan="1">48 (15.7%)</td></tr><tr><td colspan="1" rowspan="3">Ageσ</td><td colspan="1" rowspan="1">Avg</td><td colspan="1" rowspan="1">13</td></tr><tr><td colspan="1" rowspan="1">Median</td><td colspan="1" rowspan="1">7</td></tr><tr><td colspan="1" rowspan="1">Min</td><td colspan="1" rowspan="1">0.5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Max</td><td colspan="1" rowspan="1">91</td></tr><tr><td colspan="1" rowspan="6">geRange</td><td colspan="1" rowspan="1">≤5</td><td colspan="1" rowspan="1">141 (53%)</td></tr><tr><td colspan="1" rowspan="1">6-21</td><td colspan="1" rowspan="1">76 (28.6%)</td></tr><tr><td colspan="1" rowspan="1">22-49</td><td colspan="1" rowspan="1">19 (7.1%)</td></tr><tr><td colspan="1" rowspan="1">≥50</td><td colspan="1" rowspan="1">27 (10.2%)</td></tr><tr><td colspan="1" rowspan="1">≥5b</td><td colspan="1" rowspan="1">3 (1.1%)</td></tr><tr><td colspan="1" rowspan="1">Unknowna</td><td colspan="1" rowspan="1">39 (12.8%)</td></tr></table>

urB a pediatric hospital, it is understood that the age range of specimens was from $< 1$ yrs to 21 yrs.

Table 8. FilmArray Archived Specimen Performance Data Summary for B. pertussis, Coronavirus 229E, Coronavirus OC43, and M. pneumoniae.   

<table><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Positive Percent Agreenient (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA)</td></tr><tr><td rowspan=1 colspan=1>TP/TP +FN</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI:</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CL</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>53/56</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>85.1-98.9%</td><td rowspan=1 colspan=1>56/58b</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>88.1-99.6%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>75.3-100%</td><td rowspan=1 colspan=1>45/47c</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>85.5-99.5%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>85.8-100%</td><td rowspan=1 colspan=1>33/36d</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>77.5-98.2%</td></tr><tr><td rowspan=1 colspan=1>M. pneuimoniae</td><td rowspan=1 colspan=1>54/64</td><td rowspan=1 colspan=1>84.4%</td><td rowspan=1 colspan=1>73.1-92.2%</td><td rowspan=1 colspan=1>58/56</td><td rowspan=1 colspan=1>89.2%</td><td rowspan=1 colspan=1>79.1-95.6%</td></tr></table>

Two (2) B. pertussis-positive specimens were originally identified by the source lab as negative for $B$ pertussis but were unexpectedly found to be positive by analyte specific PCR followed by bidirectional sequencing. Both of these specimens were negative when tested with the FilmArray, and both were found to be negative for $B .$ pertussis during discrepancy investigation of confirmation testing.

Ten (10) B. pertussi-negative specimens were originally identified by the source ab as positive for $B$ pertussis but this could not be confirmed by analyte specific PCR followed by bidirectional sequencing. Two (2) of these specimens were positive when tested with the FilmArray and both of these specimens were found to be positive for B. pertussis during discrepancy investigation of confirmation testing. Two (2) source laboratory positive samples were found to contain B. holmesii, both of these samples gave the expected negative result when tested with the FilmArray RP.

One (1) CoV 229E-negative specimen was originally identified by the source lab as positive for CoV 229E but this could not be confirmed by analyte specific PCR followed by bidirectional sequencing. This specimen was positive when tested with the FilmArray RP.

For   C-eatve een wee al e by e u   piv OC3 but this could not be confirmed by analyte specifc PCR followed by bidirectional sequencing. Two (2) of these specimens were positive when tested with the FilmArray RP.

The Ct results obtained during confirmation PCR testing for the 10 samples that were not detected by FilmArray indicated low analyte levels in the sample (Ct range 34.3-38.7) possible resulting from sample degradation during storage of these archived specimens.

Twenty-two (22) M. pneumoniae-negative specimens were originally identified by the source lab as positive for M. pneumoniae but this could not be confirmed by analyte specific PCR followed by bidirectional sequencing. Seven (7) of these specimens were positive when tested with the FilmArray RP.

# Testing of Contrived C. pneumoniae Specimens

Archived nasopharyngeal swab specimens that have previously tested positive for $C .$ pneumoniae were unavailable for testing. Therefore, contrived $C .$ pneumoniae specimens were used as surrogate clinical specimens to test the sensitivity and specificity of the

FilmArray RP C. pneumoniae assay. Residual specimens that had been collected during the prospective clinical evaluation were spiked with $C .$ pneumoniae at clinically relevant levels (or unspiked; 50 of each). The analyte status of each contrived specimen was blinded to the users analyzing the specimens. Results of FilmArray testing are presented in Table 9.

Table 9. FilmArray Performance Data Summary for contrived C. pneumoniae specimens.   

<table><tr><td rowspan=2 colspan=1>6^.</td><td rowspan=1 colspan=3>Positive Percent Agreement (PPA)</td><td rowspan=1 colspan=3>Negative Percent Agreement (NPA).</td></tr><tr><td rowspan=1 colspan=1>TP/TP +FN. Percent</td><td rowspan=1 colspan=1>TP/TP +FN. Percent</td><td rowspan=1 colspan=1>95% Cl</td><td rowspan=1 colspan=1>TN/TN+FP</td><td rowspan=1 colspan=1>Percent</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9 - 100%</td><td rowspan=1 colspan=1>50/50</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>92.9 - 100%</td></tr></table>

# Selected Analytic Studies

# Limit of Detection

The analytical sensitivity or Limit of Detection (LoD) for each FilmArray RP analyte was determined by testing limiting dilutions of quantified cultures of virus or bacterium. LoD is defined as the lowest concentration at which the analyte is consistently detected (detection in $2 9 5 \%$ of samples tested). Simulated NPS sample matrix (cultured human cells in VTM) was spiked with one or more analytes and at least 20 replicates were tested at the LoD concentration. The LoD concentrations for the FilmArray RP analytes CoV OC43, CoV 229E, $B .$ pertussis, $C .$ pneumoniae, and M. pneumoniae are listed in Table 10.

Table 10. LoD for Analytes Detected by FilmArray RP   

<table><tr><td rowspan=1 colspan=1>Organism, &#x27;g</td><td rowspan=1 colspan=1>Strain;</td><td rowspan=1 colspan=1>LoD Concentration</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>600 TCIDso/mL</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>4 TCIDsg/mL</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>4000 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>3000 DNA copies/mL</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M129 (Type l)</td><td rowspan=1 colspan=1>30 TCIDsg/mL</td></tr></table>

NOTE: CoV OC43, CoV 229E and $M .$ pneumoniae were grown and quantified in $\overline { { \mathrm { T C l D } _ { 5 0 } ( 5 0 \% } }$ Tissue Culture Infectious Dose). The unit $\mathrm { \Delta T C I D _ { 5 0 } }$ is a measure of infectivity or cytotoxicity rather than number of organisms or copies of nucleic acid. Variability in $\mathrm { \Delta T C I D _ { 5 0 } / m L }$ may not accurately reflect differences in the relative sensitivity of detection between different organisms or different strains of the same organism.

# Analytical Reactivity (Inclusivity)

The analytical reactivity of the FilmArray RP system assays was evaluated with an inclusivity panel consisting of strains or isolates that represent the genetic, temporal, and geographic diversity of the FilmArray RP analytes. The tested organisms include: 2 Coronavirus (1 229E and 1 OC43), 9 B. pertussis, 4 C. pneumoniae, and 9 M. pneumoniae. Each strain was initially tested in a simulated NPS sample matrix at or near the system LoD. Higher concentrations were tested if the analyte was not detected at LoD.

Results from inclusivity testing of CoV OC43, CoV 229E, B. pertussis, C. pneumoniae, and $M .$ pneumoniae are presented in Table 11. The concentration and multiple of LoD at which each strain was detected by the FilmArray RP system is indicated.

Table 11. Results of Inclusivity Testing   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1> ,51 Strain / Isolate</td><td rowspan=1 colspan=1>Concentrationk.Detected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>600 TCIDsg/mL</td><td rowspan=1 colspan=1>Ix</td></tr><tr><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>4 TCIDs/mL</td><td rowspan=1 colspan=1>Ix</td></tr><tr><td rowspan=9 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>E341</td><td rowspan=1 colspan=1>4000 CFU/mL.</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>F (ATCC 8467)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5 [17921] (ATCC 9340)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>18323 [NCTC 10739] (ATCC 9797)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>10-536 (ATCC 10380)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp 12/63 [623] (ATCC 51445)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Tohama I (ATCC BAA-589)(vaccine strain)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>MN2531 (ATCC BAA-1335)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=4 colspan=1>C. pneumoniae</td><td rowspan=1 colspan=1>AR-39 (ATCC 53592)</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CDC/CWL-029 (VR-1310)</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CM-1 (ATCC VR-1360)</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>TW183 (ATCC VR-2282)</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=9 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>M129 (Type I)</td><td rowspan=1 colspan=1>30 TCIDso/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>M129-B7 (ATCC 29342) (Type 1)</td><td rowspan=1 colspan=1>300 CCU/mL a</td><td rowspan=1 colspan=1>10x b</td></tr><tr><td rowspan=1 colspan=1>PI 1428 (ATCC 29085) (Type 1)</td><td rowspan=1 colspan=1>3,000 CCU/mL</td><td rowspan=1 colspan=1>$100x brx }$</td></tr><tr><td rowspan=1 colspan=1>FH strain [NCTC 10119] (ATCC 15531)(Type 2)</td><td rowspan=1 colspan=1>300 CFU/mL</td><td rowspan=1 colspan=1>n/ac</td></tr><tr><td rowspan=1 colspan=1>[Mac] (ATCC 15492) (Type 2</td><td rowspan=1 colspan=1>300 CCU/mL a</td><td rowspan=1 colspan=1>10x b</td></tr><tr><td rowspan=1 colspan=1>[M52] (ATCC 15293)</td><td rowspan=1 colspan=1>300 CCU/mL a</td><td rowspan=1 colspan=1>10x b</td></tr><tr><td rowspan=1 colspan=1>[Bru] (ATCC 15377)</td><td rowspan=1 colspan=1>30,000 CCU/mL a</td><td rowspan=1 colspan=1>1,000x b</td></tr><tr><td rowspan=1 colspan=1>Mutant 22 (ATCC 39505)</td><td rowspan=1 colspan=1>30 CCU/mL *</td><td rowspan=1 colspan=1>$1x br b}$</td></tr><tr><td rowspan=1 colspan=1>UMTB-10G (ATCC 49894)</td><td rowspan=1 colspan=1>300 CCU/mL *</td><td rowspan=1 colspan=1>$10x^bb}$</td></tr></table>

\$ CCU $=$ Color Changing Unit. Both $\mathrm { T C I D } _ { 5 0 }$ and CCU were determined according to the Reed-Muench method. Quantification in $\mathrm { C C U / m L }$ was considered equivalent to quantification in $\mathrm { \Delta T C I D _ { 5 0 } / m L }$ . time PCR against a standard curve of the LoD strain (M129, $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ ). Based on relative molecular quantification, all isolates were detected at levels $< 1 \mathrm { x } - \mathrm { 1 0 \mathrm { x } }$ LoD, rather than $_ { 1 - 1 , 0 0 0 \mathrm { x } }$ LoD determined by CCU/mL. ATCC 49894, ATCC 15293, ATCC 29342, and ATCC 39505 were detected at or below 1x LoD, while ATCC 15492, ATCC 29085, and ATCC 15377 were detected betwcen 1-10x LoD. was en  olahe p bl  co n pla n wa qt U/L (Colony Forming Units).

# Analytical Specificity (Cross-reactivity and Exclusivity)

The potential for cross-reactivity between assays contained in the FilmArray RP system was evaluated by testing simulated NPS samples containing high concentrations of respiratory panel viruses and bacteria (tens to thousands-fold higher than LoD). No cross-reactivity was observed at the concentrations listed in Table 12.

Table 12. Results of Testing for Cross-Reactivity with FilmArray RP Analytes   

<table><tr><td colspan="1" rowspan="1">Virus or Bacterium</td><td colspan="1" rowspan="1">Type / Strain</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Multiple of LoD</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Serotype 1(Species C)</td><td colspan="1" rowspan="1">1.00x10° TCIDsg/mL</td><td colspan="1" rowspan="1">333x</td></tr><tr><td colspan="1" rowspan="4">Coronavirus</td><td colspan="1" rowspan="1">229EATCC VR-740</td><td colspan="1" rowspan="1">5.67x10³ TCID5/mL</td><td colspan="1" rowspan="1">1,418x</td></tr><tr><td colspan="1" rowspan="1">HKU1 - Type BClinical specimen</td><td colspan="1" rowspan="1">2.78x10° copies/mL</td><td colspan="1" rowspan="1">1,463x</td></tr><tr><td colspan="1" rowspan="1">NL63NR-470</td><td colspan="1" rowspan="1">5.67x10³ TCIDs/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">OC43ATCC VR-759</td><td colspan="1" rowspan="1">7.30x104 TCIDs/mL</td><td colspan="1" rowspan="1">122x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Type A1 - hMPV-16IA10-2003 AI</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">4,085x</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">3.40x106 TCIDsg/mL</td><td colspan="1" rowspan="1">113x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus Al</td><td colspan="1" rowspan="1">5.67x10³ TCIDs/mL</td><td colspan="1" rowspan="1">5.670x</td></tr><tr><td colspan="1" rowspan="9">Influenza AH1NI</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00x10³TCIDs/mL</td><td colspan="1" rowspan="1">500x</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">1.00x10° TCIDs/mL</td><td colspan="1" rowspan="1">500x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.00x10° TCIDsg/mL</td><td colspan="1" rowspan="1">5,000x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">1.39x104 TCIDsg/mL</td><td colspan="1" rowspan="1">70x</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">1.39x104 TCIDs/mL</td><td colspan="1" rowspan="1">70x</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1N1-2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">4.00x10° TCIDs/mL</td><td colspan="1" rowspan="1">4.000x</td></tr><tr><td colspan="1" rowspan="8">Influenza AH3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">1634x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">940x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5.67x10³ TCIDs/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.00x10³TCIDs/mL</td><td colspan="1" rowspan="1">20,000x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.00x10³ TCIDsg/mL</td><td colspan="1" rowspan="1">20,000x</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">4.70x10³ TCIDs/mL</td><td colspan="1" rowspan="1">940x</td></tr><tr><td colspan="1" rowspan="1">A/MRC 2</td><td colspan="1" rowspan="1">8.17x10³3 TCID5/mL</td><td colspan="1" rowspan="1">1,634x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">8.17x103 TCIDs/mL</td><td colspan="1" rowspan="1">1,634x</td></tr><tr><td colspan="1" rowspan="7">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">1.67x10 TCIDs/mL</td><td colspan="1" rowspan="1">278x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">8.17x103 TCIDsq/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">5.03x104 TCIDs/mL</td><td colspan="1" rowspan="1">838x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">8.17x10³ TCIDs/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">1.00x105TCID5o/mL</td><td colspan="1" rowspan="1">1,667x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">5.67x10³ TCIDs/mL</td><td colspan="1" rowspan="1">95x</td></tr><tr><td colspan="1" rowspan="1">Virus or Bacterium</td><td colspan="1" rowspan="1">Type / Strain</td><td colspan="1" rowspan="1">,iTest Concentration</td><td colspan="1" rowspan="1">Multiple of LoD</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1.00x10³TCID5g/mL</td><td colspan="1" rowspan="1">1.667x</td></tr><tr><td colspan="1" rowspan="1">B/HongKong/5/72</td><td colspan="1" rowspan="1">8.17x103 TCIDsq/ml</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Brigit</td><td colspan="1" rowspan="1">3.50x104 TCIDs/mL</td><td colspan="1" rowspan="1">583x</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 1Zeptometrix # 0810014CF</td><td colspan="1" rowspan="1">1.39x104 TCIDso/mL</td><td colspan="1" rowspan="1">28x</td></tr><tr><td colspan="1" rowspan="1">Type 2Zeptometrix # 0810015CF</td><td colspan="1" rowspan="1">1.67x104 rCIDs/mL</td><td colspan="1" rowspan="1">1,670x</td></tr><tr><td colspan="1" rowspan="1">Type 3Zeptometrix # 0810016CF</td><td colspan="1" rowspan="1">1.00x105TCIDs/ml</td><td colspan="1" rowspan="1">10,000x</td></tr><tr><td colspan="1" rowspan="1">Type 4Zeptometrix #0810060CF</td><td colspan="1" rowspan="1">5.67x10³ TCIDs/ml.</td><td colspan="1" rowspan="1">1.13x^{a</td></tr><tr><td colspan="1" rowspan="2">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">1.39x104 TCID5g/mL</td><td colspan="1" rowspan="1">6,950x</td></tr><tr><td colspan="1" rowspan="1">B</td><td colspan="1" rowspan="1">2.14x104 TCIDs0/mL</td><td colspan="1" rowspan="1">10,700x</td></tr><tr><td colspan="1" rowspan="9">Bordetella pertussis</td><td colspan="1" rowspan="1">E431</td><td colspan="1" rowspan="9">1.00x106CFU/mL</td><td colspan="1" rowspan="9">250x</td></tr><tr><td colspan="1" rowspan="1">A639</td></tr><tr><td colspan="1" rowspan="1">ATCC 8467</td></tr><tr><td colspan="1" rowspan="1">ATCC 9797</td></tr><tr><td colspan="1" rowspan="1">ATCC 51445</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-589</td></tr><tr><td colspan="1" rowspan="1">ATCC 9340</td></tr><tr><td colspan="1" rowspan="1">ATCC 10380</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1335</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">2.42x10° copies/mL</td><td colspan="1" rowspan="1">81x</td></tr><tr><td colspan="1" rowspan="9">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">1.88x10 TCIDs/mL</td><td colspan="1" rowspan="1">6,267x</td></tr><tr><td colspan="1" rowspan="1">ATCC 15531</td><td colspan="1" rowspan="1">4.27x10° CFU/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="7">1.00x106CCCU/MlLb</td><td colspan="1" rowspan="7">33,333x</td></tr><tr><td colspan="1" rowspan="1">ATCC 15377</td></tr><tr><td colspan="1" rowspan="1">ATCC 15492</td></tr><tr><td colspan="1" rowspan="1">ATCC 29085</td></tr><tr><td colspan="1" rowspan="1">ATCC 29342</td></tr><tr><td colspan="1" rowspan="1">ATCC 39505</td></tr><tr><td colspan="1" rowspan="1">ATCC 49894</td></tr></table>

Highest test concentration possible based on the concentration of virus in the stock culture fluid. ${ } ^ { \mathsf { b } } { \mathsf { C C U } } =$ Color Changing Unit. Both $\mathrm { T C I D } _ { 5 0 }$ and CCU were determined according to the Reed-Muench method. Quantification in CCU/mL was considered equivalent to quantification in $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ .

The potential for the FilmArray RP system to cross-react with non-FilmArray RP organisms was evaluated by testing an exclusivity panel consisting of 25 bacteria, 8 viruses, and 1 fungus. These organisms were selected based on their relatedness to FilmArray RP analytes, clinical relevance (cause respiratory symptoms or represent nasopharyngeal flora), or high prevalence within the population (e.g. Herpes Simplex Virus). Negative sample matrix was spiked with bacteria or fungi at a concentration of $\mathbf { \dot { 1 0 ^ { 6 } } C F U / m L }$ and viruses at a concentration between $1 0 ^ { 4 } - 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , or the highest concentration possible. The

FilmArray RP system did not cross-react with the exclusivity panel organisms listed in Table 13.

Table 13. Non-FilmArray RP Exclusivity Panel   

<table><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Strain / Isolate</td><td rowspan=1 colspan=1>Viruses</td><td rowspan=1 colspan=1>Strain / Isolate</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>clinical isolate</td><td rowspan=1 colspan=1>Bocavirus</td><td rowspan=1 colspan=1>Type 1 - Clinical specimen</td></tr><tr><td rowspan=1 colspan=1>Bordetella holmesii</td><td rowspan=1 colspan=1>F061</td><td rowspan=1 colspan=1>Coronavirus SARS</td><td rowspan=1 colspan=1>Zeptometrix -Nucleic Acid</td></tr><tr><td rowspan=1 colspan=1>Chlamydia trachomatis</td><td rowspan=1 colspan=1>D-UW3</td><td rowspan=1 colspan=1>Cytomegalovirus (CMV)</td><td rowspan=1 colspan=1>AD-169 (VR-538)</td></tr><tr><td rowspan=1 colspan=1>Corynebacteriumdiptheriae</td><td rowspan=1 colspan=1>ATCC14779</td><td rowspan=1 colspan=1>Epstein-Barr Virus(EBV)</td><td rowspan=1 colspan=1>B95-8</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>0157:H7</td><td rowspan=1 colspan=1>Herpes Simplex Virus</td><td rowspan=1 colspan=1>Type I</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>MinnA          .</td><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Edmonston</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Type strain</td><td rowspan=1 colspan=1>Measles Virus</td><td rowspan=1 colspan=1>Zeptometrix # 0810025CFa</td></tr><tr><td rowspan=1 colspan=1>Lactobacillus plantarum</td><td rowspan=1 colspan=1>17-5</td><td rowspan=1 colspan=1>Mumps</td><td rowspan=1 colspan=1>Zeptometrix # 0810079CF</td></tr><tr><td rowspan=1 colspan=1>Legionella longbeacheae</td><td rowspan=1 colspan=1>Long Beach 4</td><td rowspan=1 colspan=1>Fungi</td><td rowspan=1 colspan=1>Strain / Isolate</td></tr><tr><td rowspan=1 colspan=1>Legionella micdadei</td><td rowspan=1 colspan=1>Tatlock</td><td rowspan=1 colspan=1>Candida albicans</td><td rowspan=1 colspan=1>Zeptometrix #0801504</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophilia</td><td rowspan=1 colspan=1>Philadelphia</td><td rowspan=15 colspan=2>−</td></tr><tr><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Ne I I (type strain)</td></tr><tr><td rowspan=1 colspan=1>Mycobacteriumtuberculosis</td><td rowspan=1 colspan=1>H37Ra-1</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma hominis</td><td rowspan=1 colspan=1>ATCC 23114</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma genitalium</td><td rowspan=1 colspan=1>ATCC 33530</td></tr><tr><td rowspan=1 colspan=1>Neisseria elongata</td><td rowspan=1 colspan=1>type strain</td></tr><tr><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>ATCC 700825</td></tr><tr><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>M1027 (typestrain</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>Zeptometrix#0801519</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>COL</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusepidermidis</td><td rowspan=1 colspan=1>RP62A</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pneumoniae</td><td rowspan=1 colspan=1>type 59</td></tr><tr><td rowspan=1 colspan=1>Streptococcus pyogenes</td><td rowspan=1 colspan=1>Zeptometrix#0801512</td></tr><tr><td rowspan=1 colspan=1>Streptococcus salivarius</td><td rowspan=1 colspan=1>ATCC 13419</td></tr><tr><td rowspan=1 colspan=1>Ureaplasma urealyticum</td><td rowspan=1 colspan=1>ATCC 27618</td></tr></table>

pos ue psiv i virus.

# Precision (Reproducibility and Repeatability)

A multicenter reproducibility study was performed to determine between-site and overall reproducibility of the FilmArray RP system. Reproducibility testing occurred at three test sites utilizing a panel of twelve simulated NPS specimens spiked with various combinations of respiratory pathogens (analytes) at three different test levels (high negative (LoD/10), low positive (1X LoD), and medium positive $\left( 3 \mathrm { X } \mathrm { L o D } \right)$ ). On each testing day, two operators at each site tested two aliquots of specimens on two different FilmArray instruments (six specimens per operator per instrument per day). Every specimen was tested four times a day on five days at the three testing sites, for a total of 60 tests per analyte per concentration. A total of 26 lots of reagents and 20 FilmArray instruments were utilized in the reproducibility study.

Repeatability testing included an additional 7 days of testing per analyte and concentration at one site, for a total of 12 days of testing.

Reproducibility and Repeatability study results for CoV OC43, CoV 229E, B. pertussis, C.   
pneumoniae, and M. pneumoniae are summarized in Tables 14-19.

Table 14. Summary f Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing o Coronavirus OC43   

<table><tr><td rowspan=1 colspan=2>Coronavirus OC43 ATCC VR-759</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpected Reesut</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)1,800 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>80.98</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>80.53 - 81.36</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.39</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>81.04 - 81.82</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.20</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>80.60 - 81.55</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>81.17</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>80.53 - 81.82</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LOD)600 TCIDs0/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>80.82</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>80.29 - 81.25</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>80.95 - 81.85</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.07</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>80.40 - 81.58</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>81.07</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>80.29 - 81.85</td></tr><tr><td rowspan=4 colspan=1>High NegativeLD/10)60 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>68.3% - 98.8%</td><td rowspan=1 colspan=1>80.86</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>80.49 - 81.25</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>45.7% - 88.1%</td><td rowspan=1 colspan=1>81.29</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>80.92 - 81.81</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>56.3% - 94.3%</td><td rowspan=1 colspan=1>81.04</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>80.11 - 81.45</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>48/60</td><td rowspan=1 colspan=1>12/60</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>67.7% - 89.2%</td><td rowspan=1 colspan=1>81.07</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>80.11 - 81.81</td></tr><tr><td rowspan=4 colspan=1>.Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3>i}$</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>1/180</td><td rowspan=1 colspan=1>179/180</td><td rowspan=1 colspan=1>99.4%</td><td rowspan=1 colspan=1>96.9% - 100%</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>1/540</td><td rowspan=1 colspan=1>539/540</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.0% - 100%</td></tr></table>

Expected result for the "Negative" panel member is negative. $2 0 . 8 0 \%$ of the time.

Table 15. Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing o Coronavirus 229E   

<table><tr><td rowspan=1 colspan=2>Coronavirus 229EATCC VR-7402</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedRe sut a</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)12 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.20</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>80.81 - 81.47</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.80</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>81.35 - 82.18</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.35</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>81.03 - 82.13</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%xt</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>81.37</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>80.81 - 82.18</td></tr><tr><td rowspan=4 colspan=1>•Low Positive(1X LoD)4 TCID5o/mL .</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.19</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>80.83 - 81.66</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.62</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>81.12 - 82.08</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>81.09</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>80.30 - 81.56</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% 100%</td><td rowspan=1 colspan=1>81.31</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>80.30 -82.08</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(LoD/10)0.4 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>70%</td><td rowspan=1 colspan=1>45.7% - 88.1%</td><td rowspan=1 colspan=1>81.06</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>80.72 - 81.43</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>35%</td><td rowspan=1 colspan=1>15.4% - 59.2%</td><td rowspan=1 colspan=1>81.54</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>81.15 - 82.08</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=1>81.17</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>80.70 - 81.68</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>32/60</td><td rowspan=1 colspan=1>28/60</td><td rowspan=1 colspan=1>53.3%</td><td rowspan=1 colspan=1>40.0% 66.3%</td><td rowspan=1 colspan=1>81.25</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>80.70 - 82.08:.</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

u   i   H ei Expected result for the "Negative" panel member is negative. $20 \text{‰}$ of the time.

Table 16. Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing o Bordetella pertussis   

<table><tr><td rowspan=1 colspan=2>Bordetella pertussisA639</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>·%AgreementwithExxpected Resut</td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>i t eMeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>bservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)12,000 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.36</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>87.46 - 89.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.81</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>88.26 - 89.20</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.34</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>87.97 - 88.58</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>88.50</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>87.46 - 89.20</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)4,000 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.39</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.73 - 88.96</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.64</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>88.07 - 89.20</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.22</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.74 - 88.67</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>88.42</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>87.73 - 89.20</td></tr><tr><td rowspan=4 colspan=1>High NegativebLLOD/10)400 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>56.3% - 94.3%</td><td rowspan=1 colspan=1>88.41</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>87.86 - 89.36</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=1>88.70</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>88.33 - 89.29</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=1>88.26</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.52 - 88.71</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>20/60</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>53.3% - 78.3%</td><td rowspan=1 colspan=1>88.46</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>87.52 - 89.36</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% 100%</td></tr></table>

Expected result for the "Negative" panel member is negative. $20 \text{‰}$ of the time

Table 1. Summary of Positive Agreement, Negative greement, and Tm Results from Reproducibility Testing Chlamydophila pneumoniae   

<table><tr><td colspan="2" rowspan="1">ChlamydophilapneumoniaeTW83</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedα Result </td><td colspan="1" rowspan="1">95%CI</td><td colspan="1" rowspan="1">*MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="4">Medium Positive(3X LoD).9,000 copies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">79.65</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">78.94 - 79.99</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.27</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">79.88 - 80.82</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">79.73</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">79.42 - 80.09</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">79.92</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">78.94 - 80.82</td></tr><tr><td colspan="1" rowspan="2">Low Positive(1X LoD)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">79.63</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">79.03 - 80.09</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">75.1% - 99.9%</td><td colspan="1" rowspan="1">80.15</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">79.68 - 80.62</td></tr><tr><td colspan="1" rowspan="2">3,000 copies/mL</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">79.61</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">78.84 - 80.09</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">1/60</td><td colspan="1" rowspan="1">98.3%</td><td colspan="1" rowspan="1">91.1% - 100%</td><td colspan="1" rowspan="1">79.79</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">78.84 - 80.62</td></tr><tr><td colspan="1" rowspan="4">High Negativeb(LD/10)300 copies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">55.0%</td><td colspan="1" rowspan="1">31.5% - 76.9%</td><td colspan="1" rowspan="1">79.55</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">79.25 - 80.08</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">45.7% - 88.1%</td><td colspan="1" rowspan="1">80.02</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">79.66 - 80.72</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">27.2% - 72.8%</td><td colspan="1" rowspan="1">79.61</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">79.16 - 80.21</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">35/60</td><td colspan="1" rowspan="1">25/60</td><td colspan="1" rowspan="1">58.3%</td><td colspan="1" rowspan="1">44.9% - 70.9%</td><td colspan="1" rowspan="1">79.72</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">79.16 - 80.72</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">AllSites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr></table>

p u  i  iHa p Expected result for the "Negative" panel member is negative. atpl t $20 \text{‰}$ of the time.

Table 18. Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing of Mycoplasma pneumoniae   

<table><tr><td rowspan=1 colspan=2>.Mycoplasma pneumoniaeType 1 - M129</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResut </td><td rowspan=1 colspan=1>95%CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3X LoD)90 TCIDso/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.42</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>77.04 - 77.71</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.94</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>77.59 - 78.30</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.83</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>77.44 - 78.12</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>77.75</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>77.04 - 78.30</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1X LoD)30 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>77.55</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>77.05 - 78.00</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% - 96.8%</td><td rowspan=1 colspan=1>77.96</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>77.36 - 78.37</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.68</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>76.65 - 78.10</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>56/60</td><td rowspan=1 colspan=1>4/60</td><td rowspan=1 colspan=1>93.3%</td><td rowspan=1 colspan=1>83.8% - 98.2%</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>76.65 - 78.37</td></tr><tr><td rowspan=4 colspan=1>High Negative(LD/10)3 TCID5o/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>8.7% - 49.1%</td><td rowspan=1 colspan=1>77.58</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>76.72 - 78.01</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5.7% - 43.7%</td><td rowspan=1 colspan=1>78.01</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>77.67 - 78.45</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=1>77.76</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>77.04 - 78.09</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>20/60</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>21.7% - 46.7%</td><td rowspan=1 colspan=1>77.78</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>76.72 - 78.45</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>AllSites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

u  e e iv"he L osin he Hih eaiv p   i Expected result for the "Negative" panel member is negative.

High negative" samples are targeted to be positive $20 \%$ of the time.

Table 19. Summary of Positive Agreement Results for Repeatability Testing of Coronavirus OC43, Coronavirus 229E, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae   

<table><tr><td rowspan=2 colspan=1>Spiked Organism</td><td rowspan=1 colspan=2>Moderate Positive(3x LoD)    -</td><td rowspan=1 colspan=2>Low Positive(Ix LoD)</td><td rowspan=1 colspan=2>High Negative(0.1x LoD)</td></tr><tr><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1>·# Positive /Total</td><td rowspan=1 colspan=1>% Positive. Results</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>33/48</td><td rowspan=1 colspan=1>68.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>.100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19/48</td><td rowspan=1 colspan=1>39.6%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>18/48</td><td rowspan=1 colspan=1>37.5%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>19/48</td><td rowspan=1 colspan=1>39.6%</td></tr></table>

# Interference

Substances that could be present in NPS samples or introduced during sample handling were evaluated for their potential to interfere with assay performance. Four different organism mixes containing FilmArray RP analytes were spiked into a simulated NPS (sNPS) sample matrix (human epithelial cells in VTM) at $5 \mathrm { \times }$ their respective LoDs. The 5x LoD organism concentration was chosen to be near the analyte LoD but also to provide consistent results for sample-to-sample comparison. Each FilmArray RP analyte was tested in the presence of each potentially interfering substance listed in Table 20. None of the substances tested were found to compete or interfere with the control or analyte assays in the FilmArray RP. The appropriate FilmArray RP assays do react with the Influenza A H1, Influenza A H3, and Influenza B viral material contained in the FluMist® vaccine.

Table 20. List of Potentially Interfering Substances Evaluated   

<table><tr><td>Endogenous Substances</td><td colspan="2">Competitive /·Interfering Microorganisms</td></tr><tr><td>Human Blood (with Na Citrate) (1% v/v)</td><td>Respiratory Syncytial Virus A</td><td>2.8 x 104 TCIDz/mL</td></tr><tr><td>Mucin (bovine submaxillary gland) (1% v/v)</td><td>Human Rhinovirus</td><td>1.1 x 104 TCIDsg/mL</td></tr><tr><td>Human Genomic DNA: 0.2 ng/μL</td><td>Influenza A 2009 HIN1</td><td>1.0 x 10 TCID5/mL</td></tr><tr><td>2 ng/μL</td><td>Staphylococcus aureus</td><td>1.0 x 10 CFU/mL</td></tr><tr><td>20 ng/μL</td><td>Neisseria meningitidis</td><td>1.0 x 106 CFU/mL</td></tr><tr><td></td><td>Corynebacterium diptheriae</td><td>1.0 x 106 CFU/mL</td></tr><tr><td colspan="3">Exogenous Substances</td></tr><tr><td>Saline Nasal Spray with Preservatives (1% v/v)</td><td colspan="2">Analgesic ointment (1% w/v)</td></tr><tr><td>Nasal Decongestant Spray (Oxymetazoline HCl) (1%v/v)</td><td colspan="2">Petroleum Jelly ((% w/v)</td></tr><tr><td>Tobramycin (0.6 mg/mL)</td><td colspan="2">Smokeless Tobacco (1% w/v)</td></tr><tr><td>Mupirocin (2% w/v)</td><td colspan="2">FluMist® Nasal Influenza Vaccine (2009-2010)</td></tr><tr><td colspan="3">Technique Specific Substances</td></tr><tr><td>Laboratory Reagents:</td><td>Viral Transport Media:</td><td>Swabs:</td></tr><tr><td>Bleach (1%, 2%, 5% v/v)</td><td>Remel M4</td><td>Copan 168C (rayon / twisted aluminum shaft)</td></tr><tr><td>Disinfecting wipes</td><td>Remel M4-RT</td><td>Copan FloQ (flocked nylon / plastic shaft)</td></tr><tr><td>Ethanol (7% v/v)</td><td>Remel M5</td><td>Copan 1 75KS01 (polyester / aluminum shaft)</td></tr><tr><td>DNAzap (1% v/v)</td><td>Remel M6</td><td>Millipore 519CS01M (flocked nylon / plastic shaft)</td></tr><tr><td>RNaseOut (1% v/v)</td><td>Copan UTM</td><td></td></tr></table>

Idaho Technology Inc. c/o Beth Lingenfelter 390 Wakara Way Salt Lake City, UT 84108

Re: K120267 Trade/Device Name: Regulation Number: Regulation Name: Regulatory Class: Product Code: Dated: Received:

FilmArray Respiratory Panel (RP)   
21 CFR 866.3980   
Respiratory Viral Panel Multiplex Nucleic Acid Assay   
Class II   
OCC, OEM, OOU, OEP, OTG, NXD, OOI, OZZ, OZY, OZX   
April 3, 2012   
April 5, 2012

Dear Ms. Lingenfelter:

We have reviewed your Section $5 1 0 ( \mathrm { k } )$ premarket notification of intent to market the device referenced above and have determined the device is substantially equivalent (for the indications for use stated in the enclosure) to legally marketed predicate devices marketed in interstate commerce prior to May 28, 1976, the enactment date of the Medical Device Amendments, or to . devices that have been reclassified in accordance with the provisions of the Federal Food, Drug, and Cosmetic Act (Act) that do not require approval of a premarket approval application (PMA). You may, therefore, market the device, subject to the general controls provisions of the Act. The general controls provisions of the Act include requirements for annual registration, listing of devices, good manufacturing practice, labeling, and prohibitions against misbranding and adulteration.

If your device is classified (see above) into class I (Special Controls), it may be subject to such additional controls. Existing major regulations affecting your device can be found in Title 21, Code of Federal Regulations (CFR), Parts 800 to 895. In addition, FDA may publish further announcements concerning your device in the Federal Register.

Please be advised that FDA's issuance of a substantial equivalence determination does not mean that FDA has made a determination that your device complies with other requirements of the Act or any Federal statutes and regulations administered by other Federal agencies. You must comply with all the Act's requirements, including, but not limited to: registration and listing (21 CFR Part 807); labeling (21 CFR Parts 801 and 809); medical device reporting (reporting of medical device-related adverse events) (21 CFR 803); and good manufacturing practice requirements as set forth in the quality systems (QS) regulation (21 CFR Part 820). This letter will allow you to begin marketing your device as described in your Section $5 1 0 ( \mathbf { k } )$ premarket notification. The FDA finding of substantial equivalence of your device to a legally marketed predicate device results in a classification for your device and thus, permits your device to proceed to the market.

If you desire specific advice for your device on our labeling regulation (21 CFR Parts 801 and 809), please contact the Office of In Vitro Diagnostic Device Evaluation and Safety at (301) 796- 5450. Also, please note the regulation entitled, "Misbranding by reference to premarket notification" (21 CFR Part 807.97). For questions regarding the reporting of adverse events under the MDR regulation (21 CFR Part 803), please go to   
http://www.fda.gov/MedicalDevices/Safety/ReportaProblem/default.htm for the CDRH's Office of Surveillance and Biometrics/Division of Postmarket Surveillance. You may obtain other general information on your responsibilities under the Act from the Division of Small Manufacturers, International and Consumer Assistance at its toll-free number (800) 638-2041 or (301) 796-7100 or at its Internet address   
http://www.fda.gov/cdrh/industry/support/index.html. Sally A. Hojvat, M.Sc., Ph.D.   
Director,   
Division of Microbiology Devices   
Office of In Vitro Diagnostic Device   
Evaluation and Safety   
Center for Devices and Radiological Health

![](images/63a6b4bdd3bbedc5a24129db77c68d765b466c6789356d63fa1dbb23aad4b525.jpg)  
Page 2 - Beth Lingenfelter

# Indications for Use

510(k) Number: K120267 Device Name: FilmArray Respiratory Panel (RP)

FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype 2009 H1, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Rhinovirus/Enterovirus, Respiratory Syncytial Virus, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test or lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the Film Array RP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis, Coronavirus 229E, Coronavirus OC43, Influenza A/H1, Influenza A/H3, Influenza A/2009 H1, Influenza B, Mycoplasma pneumoniae Parainfluenza Virus 1, Parainfluenza Virus 2, and Parainfluenza Virus 4 were established primarily with retrospective clinical specimens. Performance characteristics for Chlamydophila pneumoniae were established primarily using contrived clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 2 and serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture).

The FilmArray RP assay for Coronavirus OC43 may cross-react with some isolates of Coronavirus HKU1. A dual positive result may be due to cross-reactivity or may indicate a co-infection.

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circuilation. Performance of detecting influenza A may vary if other influenza A strains are circulating or a novel influenza A virus emerges. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

Prescription Use_ X AND/OR Over-the-Counter Use (Part 21 CFR 801 Subpart D) (21 CFR 801 Subpart C)