# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY ASSAY AND INSTRUMENT CO M BINATIO N TEMPLATE

A. 510(k) Number:

K120138

# B. Purpose for Submission:

To obtain a Substantial Equivalence Determination for the BD MAX™ MRSA Assay on the BD MAX™ System.

# C. Measurand:

Target DNA sequences in the SCCmec cassette (carrying the mecA gene) and S. aureus specific sequence located within the orfX gene. The BD MAX™ MRSA Assay is designed to detect MREJ genotypes i, ii, iii, iv, v and vii.

# D. Type of Test:

Qualitative real-time polymerase chain reaction (PCR) assay for the amplification and detection of methicillin resistant Staphylococcus aureus (MRSA) DNA.

# E. Applicant:

BD Diagnostics (GeneOhm Sciences Canada Inc.)

F. Proprietary and Established Names: BD MAX™ MRSA BD MAX™ System

# G. Regulatory Information:

1. Regulation section: 21 CFR section 866.1640, Antimicrobial susceptibility test powder

2. Classification: Class II

3. Product code:

NQX - nucleic acid amplification test, DNA, methicillin resistant Staph aureus, direct specimen

OOI - Real-time nucleic acid amplification

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The BD MAX™ MRSA Assay performed on the BD MAX™ System is an automated qualitative in vitro diagnostic test for the direct detection of Methicillin resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization. The test utilizes real-time polymerase chain reaction (PCR) for the amplification of MRSA DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD MAX™ MRSA Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose, guide or monitor MRSA infections. A negative result does not preclude nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

# 2. Indication(s) for use:

The BD MAX™ MRSA Assay performed on the BD MAX™ System is an automated qualitative in vitro diagnostic test for the direct detection of Methicillin resistant Staphylococcus aureus (MRSA) DNA from nasal swabs in patients at risk for nasal colonization. The test utilizes real-time polymerase chain reaction (PCR) for the amplification of MRSA DNA and fluorogenic target-specific hybridization probes for the detection of the amplified DNA. The BD MAX™ MRSA Assay is intended to aid in the prevention and control of MRSA infections in healthcare settings. It is not intended to diagnose, guide or monitor MRSA infections. A negative result does not preclude nasal colonization. Concomitant cultures are necessary to recover organisms for epidemiological typing or for further susceptibility testing.

3. Special conditions for use statement(s): For prescription use.

4. Special instrument requirements: The BD MAX™ System

# I. Device Description:

The BD MAX™ System and the BD MAX™ MRSA Assay are comprised of an instrument with associated hardware and accessories, disposable microfluidic cartridges, master mixes, unitized reagent strips, extraction reagents, and sample buffer tubes.

The BD MAX™ System automates sample lysis, DNA extraction and concentration, reagent rehydration, nucleic acid amplification and detection of the target nucleic acid sequence using real-time polymerase chain reaction (PCR). Amplified targets are detected with hydrolysis probes labeled with quenched fluorophores. The amplification, detection and interpretation of the signals are done automatically by the BD MAX™ System.

A nasal swab is placed in a BD MAX™ MRSA Sample Buffer Tube, which is vortexed and placed onto the BD MAX™ System. The following automated procedures then takes place: the bacterial cells are lysed, DNA is extracted on magnetic beads and concentrated, and then an aliquot of the eluted DNA is added to PCR reagents which contain the MRSA-specific primers used to amplify the genetic target, if present. The assay also includes a Sample Processing Control (SPC). The Sample Processing Control is present in the Extraction Tube and undergoes the extraction, concentration and amplification steps to monitor for inhibitory substances as well as process inefficiency due to instrument or reagent failure. No operator intervention is necessary once the clinical sample and reagent strip are loaded into the BD MAX™ System.

The amplified DNA targets are detected using hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect MRSA and SPC amplicons in two different optical channels of the BD MAX™ System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the two optical channels used for the BD MAX™ MRSA Assay is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX™ System measures these signals at the end of each amplification cycle, and interprets the data to provide a result.

The BD MAX™ System software automatically interprets test results. A test result may be called as NEG (negative), POS (positive) or UNR (unresolved) based on the amplification status of the target and of the Sample Processing Control. IND (indeterminate) or INC (incomplete) results are due to BD MAX™ System failure.

# J. Substantial Equivalence Information:

1. Predicate device name(s): BD GeneOhm™ MRSA ACP Assay

2. Predicate 510(k) number(s): K093346

3. Comparison with predicate:

<table><tr><td colspan="3" rowspan="1">Similarities</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Device</td><td colspan="1" rowspan="1">Predicate</td></tr><tr><td colspan="1" rowspan="1">Intended Use</td><td colspan="1" rowspan="1">The BD MAXTM MRSAAssay performed on the BDMAXTM System is anautomated qualitative invitro diagnostic test for thedirect detection ofMethicillin-resistantStaphylococcus aureus(MRSA) DNA from nasalswabs in patients at risk fornasal colonization. The testutilizes real-timepolymerase chain reaction(PCR) for the amplificationof MRSA DNA andfluorogenic target-specifichybridization probes for thedetection of the amplifiedDNA. The BD MAXTMMRSA Assay is intended toaid in the prevention andcontrol of MRSA infectionsin healthcare settings. It isnot intended to diagnose,guide or monitor MRSAinfections. A negative resultdoes not preclude nasalcolonization. Concomitantcultures are necessary torecover organisms forepidemiological typing or</td><td colspan="1" rowspan="1">The BD GeneOhmTMMRSA ACP Assay is aqualitative in vitrodiagnostic test for the directdetection of nasalcolonization by methicillin-resistant Staphylococcusaureus (MRSA) to aid inthe prevention and controlof MRSA infections inhealthcare settings. Thetest, performed from anasal swab specimen fromindividuals at risk forcolonization, utilizespolymerase chain reaction(PCR) for the amplificationof MRSA DNA andfluorogenic target-specifichybridization probes for thedetection of the amplifiedDNA. The BDGeneOhmTM MRSA ACPAssay is not intended todiagnose MRSA infectionsnor to guide or monitortreatment for MRSAinfections. Concomitantcultures are necessary onlyto recover organisms forepidemiological typing or</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">for further susceptibilitytesting.</td><td colspan="1" rowspan="1">for further susceptibilitytesting.</td></tr><tr><td colspan="1" rowspan="1">Specimen Type</td><td colspan="1" rowspan="1">Nasal Swabs</td><td colspan="1" rowspan="1">Nasal Swabs</td></tr><tr><td colspan="1" rowspan="1">Assay Format</td><td colspan="1" rowspan="1">Amplification: PCRDetection: Fluorogenictarget-specific hybridization</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Mode of Identification ofS. aureus</td><td colspan="1" rowspan="2">Presence of SCCmeccassette (genetic elementthat carries the mecA gene)at orfX junction (specific toS. aureus)</td><td colspan="1" rowspan="2">Same</td></tr><tr><td colspan="1" rowspan="1">Mode of Detection forMethicillin Resistance</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Device</td><td rowspan=1 colspan=1>Predicate</td></tr><tr><td rowspan=1 colspan=1>Interpretation of TestResults</td><td rowspan=1 colspan=1>Automated (Diagnosticsoftware of BD MAXTMSystem)</td><td rowspan=1 colspan=1>Automated (DiagnosticSoftware of SmartCycler®System)</td></tr><tr><td rowspan=1 colspan=1>Analysis Platform</td><td rowspan=1 colspan=1>BD MAXTM System</td><td rowspan=1 colspan=1>SmartCycler® System</td></tr><tr><td rowspan=1 colspan=1>PCR Sample Preparation</td><td rowspan=1 colspan=1>Automated by the BDMAXTM System</td><td rowspan=1 colspan=1>Manual</td></tr><tr><td rowspan=1 colspan=1>Detection Probes</td><td rowspan=1 colspan=1>TaqMan® Probe</td><td rowspan=1 colspan=1>Molecular Beacon Probe</td></tr><tr><td rowspan=1 colspan=1>Assay Controls</td><td rowspan=1 colspan=1>Specimen Process Control(SPC)</td><td rowspan=1 colspan=1>Positive PCR control(DNA from S. aureusATCC 43300).Negative PCR control(DNA from S. epidermidisATCC 14990).Internal procedural control</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Establishing the Performance Characteristics of Nucleic Acid-Based In vitro Diagnostic Devices for the Detection and Differentiation of Methicillin-Resistant Staphylococcus aureus (MRSA) and Staphylococcus aureus (SA). Draft Guidance for Industry and Food and Drug Administration Staff. Document issued on: January 5, 2011

# L. Test Principle:

The BD MAX™ System uses a combination of lytic and extraction reagents to perform cell lysis and DNA extraction.

Achromopeptidase (ACP), a lysyl endopeptidase is used to rupture bacterial cells in the BD MAXTM MRSA Assay. The nasal swabs are placed in sample buffer. After elution, sample buffer is transferred to the Extraction Tube containing ACP. Following enzymatic cell lysis at elevated temperature, the released nucleic acids are captured by magnetic affinity beads. The beads with the bound nucleic acids are washed and the nucleic acids are eluted by heat in Elution Buffer. Eluted DNA is neutralized with Neutralization Buffer and transferred to the Master Mix Tube to rehydrate PCR reagents. The reconstituted amplification reagent is dispensed into the BD MAX™ PCR Cartridge.

The amplified DNA targets are detected using hydrolysis (TaqMan $\textsuperscript { \textregistered }$ ) probes labeled at one end with a fluorescent reporter dye (fluorophore) and at the other end with a quencher moiety. Probes labeled with different fluorophores are used to detect MRSA and SPC amplicons in two different optical channels of the BD MAX™ System. When the probes are in their native state, the fluorescence of the fluorophore is quenched due to its proximity to the quencher. However, in the presence of target DNA, the probes hybridize to their complementary sequences and are hydrolyzed by the $5 ' - 3 '$ exonuclease activity of the DNA polymerase as it synthesizes the nascent strand along the DNA template. As a result, the fluorophores are separated from the quencher molecules and fluorescence is emitted. The amount of fluorescence detected in the two optical channels used for the BD MAX™ MRSA Assay is directly proportional to the quantity of the corresponding probe that is hydrolyzed. The BD MAX™ System measures these signals at the end of each amplification cycle, and interprets the data to provide a result.

Specific detection of MRSA organisms and discrimination from methicillin-resistant coagulase-negative Staphylococci (MRCNS) or MSSA is achieved by simultaneously targeting the Staphylococcus aureus species specific sequence that does not exhibit homology with nucleotide sequences surrounding the SCCmec integration site in other staphylococcal species (such as S. epidermidis and S. haemolyticus) and since SCCmec is absent from methicillin-susceptible S. aureus (MSSA), only DNA from MRSA should be amplified.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Reproducibility and Precision of the BD MAX™ MRSA Assay using simulated nasal specimens and the recommended processing techniques described in BD MAX™ MRSA Assay package insert were evaluated in three studies: lot-to-lot (LTL) reproducibility, site-to-site (STS) reproducibility, and assay precision.

The MRSA strains used in the panels were Staphylococcus aureus ATCC 43300 (MREJ type ii) and IDI-19 (MREJ type vii). Serial dilutions were prepared from a semi-logarithmic broth culture supplemented with glycerol $( 1 0 \% )$ to obtain the required bacterial concentrations. A bacterial suspension of Staphylococcus

epidermidis (ATCC 14990) was used to simulate the flora present in the nose (simulated negative nasal matrix). Final concentrations of S. epidermidis for the negative nasal matrix were between $1 . 2 5 { + } \mathrm { E } 3$ and $2 . 0 7 \mathrm { + E 3 }$ CFU/PCR reaction. Description of the panel members and their respective concentrations were summarized in the following table:

<table><tr><td rowspan=1 colspan=2>Panel Member</td><td rowspan=1 colspan=1>MRSA Concentration(multiples of LoD)</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii High Negative 100</td><td rowspan=1 colspan=1>HN100 type ii</td><td rowspan=1 colspan=1>0.01 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii High Negative 100</td><td rowspan=1 colspan=1>HN100 type vii</td><td rowspan=1 colspan=1>0.01 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii High Negative 10</td><td rowspan=1 colspan=1>HN10 type ii</td><td rowspan=1 colspan=1>0.1 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii High Negative 10</td><td rowspan=1 colspan=1>HN10 type vii</td><td rowspan=1 colspan=1>0.1 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii Low Positive</td><td rowspan=1 colspan=1>LP type ii</td><td rowspan=1 colspan=1>1 to &lt;2 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii Low Positive</td><td rowspan=1 colspan=1>LP type vii</td><td rowspan=1 colspan=1>1 to &lt;2 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type ii Moderate Positive</td><td rowspan=1 colspan=1>MP type ii</td><td rowspan=1 colspan=1>2 to 5 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ type vii Moderate Positive</td><td rowspan=1 colspan=1>MP type vii</td><td rowspan=1 colspan=1>2 to 5 X LoD</td></tr><tr><td rowspan=1 colspan=1>MRSA Negative with S. epidermidis only</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>0 X LoD</td></tr></table>

Panels of 18 tubes (labeled 01R to 18R) were prepared for the reproducibility and precision studies according to the Table below). For negative panels, only Staphylococcus epidermidis (ATCC 14990) was used to simulate the flora present in the nose. For low positive, moderate positive and high negative panel members, the MRSA bacterial suspensions were mixed with the S. epidermidis suspension. All simulated specimens contained $1 . 0 \mathrm { E } { + 0 5 }$ $\mathrm { C F U / 7 5 \mu L }$ of $S .$ . epidermidis. Cell numbers in bacterial suspensions were verified by colony counts.

<table><tr><td rowspan=1 colspan=1>MRSA Strain</td><td rowspan=1 colspan=1>Panel Category</td><td rowspan=1 colspan=1>Tube ID of PanelMembers</td><td rowspan=1 colspan=1>MRSA Concentration(multiples of LOD)</td></tr><tr><td rowspan=4 colspan=1>MRSA ATCC43300MREJ type ii</td><td rowspan=1 colspan=1>HN100 type ii</td><td rowspan=1 colspan=1>04R, 09R, 17R</td><td rowspan=1 colspan=1>0.01 X</td></tr><tr><td rowspan=1 colspan=1>HN10 type ii</td><td rowspan=1 colspan=1>02R, 11R, 13R,</td><td rowspan=1 colspan=1>0.1 X</td></tr><tr><td rowspan=1 colspan=1>LP type ii</td><td rowspan=1 colspan=1>01R, 03R, 15R</td><td rowspan=1 colspan=1>1 to &lt;2 X</td></tr><tr><td rowspan=1 colspan=1>MP type ii</td><td rowspan=1 colspan=1>10R, 12R, 14R</td><td rowspan=1 colspan=1>2 to 5X</td></tr><tr><td rowspan=1 colspan=1>MRSA IDI-19MREJ type vii</td><td rowspan=1 colspan=1>LP type vii</td><td rowspan=1 colspan=1>05R, 08R, 18R</td><td rowspan=1 colspan=1>1 to &lt; 2 X</td></tr><tr><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>06R, 07R, 16R</td><td rowspan=1 colspan=1>0X</td></tr></table>

Data analysis was conducted as follows:

The reproducibility and precision data were tabulated as percent correct and incorrect as compared to the expected results. Reproducibility was evaluated in two separate analyses: LTL and STS. · Percentage of correct results was calculated using the following formula: number of correct counts (assay results) divided by total count (assay results) for a particular MRSA targeted level.

Quantitative Analysis of PCR parameters was performed as part of this study. PCR parameters targeted for quantitative analyses were SDPA, SDPH and EP. For each HN100, HN10, LP, and MP specimen reported as positive upon testing in the MRSA PCR channel (FAM), the mean, standard deviation and coefficient of variation were calculated for each parameter. For each TN, HN100 and HN10 specimen reported as negative upon testing, the same values were calculated for the Specimen Processing Control channel. The acceptance criteria for the qualitative results for LTL, STS and Precision studies were as follows:

· MP overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI · LP overall correct percentage of approximately $9 5 \%$ with $9 5 \%$ CI · TN overall correct percentage of approximately $100 \%$ with $9 5 \%$ CI No specific acceptance criteria were defined for either HN100 or HN10.

# Lot-to-Lot (LTL) Reproducibility

Three BD MAX™ MRSA Assay kit lots (1161004, 1194001 and 1222001) as well as three different cartridge lots (1166001, 1168001 and 1168001), were used for the LTL Reproducibility. The LTL Reproducibility Study was performed in-house at BD Diagnostics. A total of 30 pre-labeled reproducibility panels were used in the study. Each panel consisted of 18 tubes labeled 01R to 18R as shown in the above table. Each category was represented by three specimens on the panel. Testing for each lot was performed on five distinct days wherein two panels were tested each day (one for each of two technologists).. This resulted in 30 assay results per lot for each MRSA strain tested at each level. Only the LP panel was represented by two different MRSA strains.

The overall correct percentage for each panel category was as follows:

· MP MRSA specimen category $100 \%$ · LP MRSA specimen category 100% · TN specimen category 100% · HN100 specimen category at 1/100 dilution $8 3 . 3 \%$ · HN10 specimen category at 1/10 dilution 34.4%

The rates obtained for the low and moderate positive MRSA categories as well as for the true negative category met the targeted specifications for the Reproducibility study. No specific acceptance criteria were defined for either of the high negative categories.

The following tables provide qualitative and quantitative reproducibility information:

The LTL reproducibility data tabulated as percent correct and incorrect are shown for all MREJ types pooled for all lots (pooled days, runs and Tech IDs) is shown in the table below.

Qualitative Lot-to-Lot Reproducibility Study Results Percent Agreement and $9 5 \%$ Confidence Intervals (CI) \*   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Lot1161004</td><td rowspan=1 colspan=1>194001</td><td rowspan=1 colspan=1>1222001</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=2 colspan=1>MPLPHN100</td><td rowspan=2 colspan=1>100.0%(30/30)(88.6%,100%)100.0%(60/60)(94.%,100%)86.7%(26/30)(70.3%,94.7%)</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%,100%)100%(60/60)(94.%,100%)</td><td rowspan=2 colspan=1>100.0%(30/30)(88.6%,100%)100.0%(60/60)(94.%,100%)83.3%(25/30)(66.4%,92.7%)</td><td rowspan=2 colspan=1>100.0%(90/90)(95.9%,100%)100%(180/180)(97.9%,100%)83.3%(75/90)(74.3%,89.6%)</td></tr><tr><td rowspan=1 colspan=1>80%(24/30)(62.7%,90.5%)</td></tr><tr><td rowspan=2 colspan=1>HN10TN</td><td rowspan=2 colspan=1>20%(6/30)(9.5%,37.3%)100.0%(30/30)(88.6%,100%)</td><td rowspan=1 colspan=1>40%</td><td rowspan=2 colspan=1>43.3%(13/30)(27.4%,60.8%)100.0%(30/30)(88.6%,100%)</td><td rowspan=2 colspan=1>34.4%(31/90)(25.4%,44.7%)100.0%(90/90)(95%, 100%)</td></tr><tr><td rowspan=1 colspan=1>(12/30)(24.6%,57.7%)100.0%(30/30)(88.6%,100%)</td></tr></table>

\* $\%$ Agreement $( \mathrm { n } / \mathrm { N }$ , number with correct results/number of tests) (CI)

Quantitative Lot-to-Lot Reproducibility Study Results Across Lots, Days, Runs and Replicates (all pooled), MREJ types pooled-ALL   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithn day</td><td rowspan=1 colspan=2>Between Daywithin lot</td><td rowspan=1 colspan=2>Between lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Parameter</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Mean*</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=4 colspan=1>EP</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>453.5</td><td rowspan=1 colspan=1>128.41</td><td rowspan=1 colspan=1>28.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>116.79</td><td rowspan=1 colspan=1>25.8%</td><td rowspan=1 colspan=1>131.17</td><td rowspan=1 colspan=1>28.9%</td><td rowspan=1 colspan=1>217.57</td><td rowspan=1 colspan=1>48.0%</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>469.8</td><td rowspan=1 colspan=1>199.71</td><td rowspan=1 colspan=1>42.5%</td><td rowspan=1 colspan=1>10.83</td><td rowspan=1 colspan=1>2.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>200.01</td><td rowspan=1 colspan=1>42.6%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>1214.8</td><td rowspan=1 colspan=1>153.27</td><td rowspan=1 colspan=1>12.6%</td><td rowspan=1 colspan=1>49.45</td><td rowspan=1 colspan=1>4.1%</td><td rowspan=1 colspan=1>35.10</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>126.14</td><td rowspan=1 colspan=1>10.4%</td><td rowspan=1 colspan=1>207.56</td><td rowspan=1 colspan=1>17.1%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>1263.7</td><td rowspan=1 colspan=1>168.39</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>111.92</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>64.55</td><td rowspan=1 colspan=1>5.1%</td><td rowspan=1 colspan=1>105.39</td><td rowspan=1 colspan=1>8.3%</td><td rowspan=1 colspan=1>236.97</td><td rowspan=1 colspan=1>18.8%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>SDPA</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>0.93</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.69</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>1.16</td><td rowspan=1 colspan=1>3.3%</td></tr><tr><td rowspan=1 colspan=1>N100</td><td rowspan=1 colspan=1>34.9</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>2.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>1.08</td><td rowspan=1 colspan=1>3.1%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.60</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.73</td><td rowspan=1 colspan=1>2.3%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>SDPH</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>6.0</td><td rowspan=1 colspan=1>1.54</td><td rowspan=1 colspan=1>25.6%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>29.2%</td><td rowspan=1 colspan=1>1.79</td><td rowspan=1 colspan=1>29.7%</td><td rowspan=1 colspan=1>2.94</td><td rowspan=1 colspan=1>48.9%</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>6.2</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>44.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>2.75</td><td rowspan=1 colspan=1>44.7%</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>2.39</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>0.83</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.71</td><td rowspan=1 colspan=1>9.8%</td><td rowspan=1 colspan=1>3.05</td><td rowspan=1 colspan=1>17.4%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>18.8</td><td rowspan=1 colspan=1>2.64</td><td rowspan=1 colspan=1>14.1%</td><td rowspan=1 colspan=1>1.82</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>0.91</td><td rowspan=1 colspan=1>4.9%</td><td rowspan=1 colspan=1>1.10</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>3.51</td><td rowspan=1 colspan=1>18.7%</td></tr></table>

\* mean EP, SDPA and SDPH values are shown for the MRSA target in the samples that gave positive results

Quantitative Lot-to-Lot Reproducibility Study Results Across Lots, Days, Runs and Replicates (all pooled), MREJ types pooledMRSA data for Negatives that gave correct values   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Within Run</td><td rowspan=1 colspan=2>Between Runwithin day</td><td rowspan=1 colspan=2>Between Day within lot</td><td rowspan=1 colspan=2>Between lot</td><td rowspan=1 colspan=2>Overall</td></tr><tr><td rowspan=1 colspan=1>Parameter</td><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Mean*</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=3 colspan=1>EP</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>4999.2</td><td rowspan=1 colspan=1>550.94</td><td rowspan=1 colspan=1>11.0%</td><td rowspan=1 colspan=1>68.43</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>99.32</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>435.78</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>712.74</td><td rowspan=1 colspan=1>14.3%</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>4860.5</td><td rowspan=1 colspan=1>689.21</td><td rowspan=1 colspan=1>14.2%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>227.81</td><td rowspan=1 colspan=1>4.7%</td><td rowspan=1 colspan=1>725.88</td><td rowspan=1 colspan=1>14.9%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>5183.2</td><td rowspan=1 colspan=1>413.79</td><td rowspan=1 colspan=1>8.0%</td><td rowspan=1 colspan=1>283.38</td><td rowspan=1 colspan=1>5.5%</td><td rowspan=1 colspan=1>68.55</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>461.94</td><td rowspan=1 colspan=1>8.9%</td><td rowspan=1 colspan=1>685.28</td><td rowspan=1 colspan=1>13.2%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>SDPA</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>31.4</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.05</td><td rowspan=1 colspan=1>0.2%</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>2.0%</td><td rowspan=1 colspan=1>0.79</td><td rowspan=1 colspan=1>2.5%</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>31.6</td><td rowspan=1 colspan=1>0.55</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>1.4%</td><td rowspan=1 colspan=1>0.71</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>31.2</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.66</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0.75</td><td rowspan=1 colspan=1>2.4%</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>SDPH</td><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>69.3</td><td rowspan=1 colspan=1>7.51</td><td rowspan=1 colspan=1>10.8%</td><td rowspan=1 colspan=1>0.67</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>1.76</td><td rowspan=1 colspan=1>2.5%</td><td rowspan=1 colspan=1>5.27</td><td rowspan=1 colspan=1>7.6%</td><td rowspan=1 colspan=1>9.37</td><td rowspan=1 colspan=1>13.5%</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>68.1</td><td rowspan=1 colspan=1>9.08</td><td rowspan=1 colspan=1>13.3%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>3.24</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>9.64</td><td rowspan=1 colspan=1>14.2%</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>71.7</td><td rowspan=1 colspan=1>5.85</td><td rowspan=1 colspan=1>8.2%</td><td rowspan=1 colspan=1>3.17</td><td rowspan=1 colspan=1>4.4%</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>5.66</td><td rowspan=1 colspan=1>7.9%</td><td rowspan=1 colspan=1>8.74</td><td rowspan=1 colspan=1>12.2%</td></tr></table>

\* mean EP, SDPA and SDPH values are shown for the Specimen Processing Control negative samples

One BD MAX™ MRSA Assay kit lot (# 1222001) and one cartridge lot (# 1168001) were used for the STS Reproducibility study. Each site was provided with a total of 10 pre-labeled reproducibility panels. Each panel consisted of the 18 tubes described as describe above. Each category was represented by three specimens on the panel. Each site performed testing on five distinct days, wherein two panels were tested each day (one for each of two technologists). This resulted in 30 assay results for each MRSA strain tested at each level. Only the LP panel was represented by two different MRSA strain.

Qualitative Site-to-Site Reproducibility Study Results Percent Agreement and $9 5 \%$ Confidence Intervals $( \mathbf { C l } ) ^ { * }$   

<table><tr><td rowspan=1 colspan=1>Category</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>MP</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%,100%)</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%,100%)</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%, 100%)</td><td rowspan=1 colspan=1>100.0%(90/90)(95.9%,100%)</td></tr><tr><td rowspan=1 colspan=1>LP</td><td rowspan=1 colspan=1>100.0%(60/60)(94.0%,100%)</td><td rowspan=1 colspan=1>100%(60/60)(94.0%,100%)</td><td rowspan=1 colspan=1>100.0%(60/60)(94.0%,100%)</td><td rowspan=1 colspan=1>100%(180/180)(97.9%,100%)</td></tr><tr><td rowspan=1 colspan=1>HN100</td><td rowspan=1 colspan=1>73.3%(22/30)(56.6%,85.8%)</td><td rowspan=1 colspan=1>83.3%(25/30)(66.4%,92.7%)</td><td rowspan=1 colspan=1>90%(27/30)(74.4%,96.5% )</td><td rowspan=1 colspan=1>82.2%(74/90)(73.1%,88.8%)</td></tr><tr><td rowspan=1 colspan=1>HN10</td><td rowspan=1 colspan=1>40%(12/30)(24.6%,57.7%)</td><td rowspan=1 colspan=1>43.3%(13/30)(27.4%,60.8%)</td><td rowspan=1 colspan=1>10%(3/30)(3.5%, 25.6%)</td><td rowspan=1 colspan=1>31.1%(28/90)(22.5%,41.3%)</td></tr><tr><td rowspan=1 colspan=1>TN</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%,100%)</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%, 100%)</td><td rowspan=1 colspan=1>100.0%(30/30)(88.6%, 100%)</td><td rowspan=1 colspan=1>100.0%(90/90)(95%, 100%)</td></tr></table>

$\%$ Agreement $( \mathrm { n } / \mathrm { N }$ , number with correct results/number of tests) (CI)

The overall correct percentage for each panel category was as follows:

· MP MRSA specimen category $100 \%$ · LP MRSA specimen category $100 \%$ · TN specimen category $100 \%$ · HN100 specimen category at 1/100 dilution $8 2 . 2 \%$ · HN10 specimen category at 1/10 dilution $3 1 . 1 \%$

In-house Precision (Repeatability):

The precision study was conducted using the five specimen categories indicated above. Testing was performed in duplicate, over 12 days with two runs per day by two technologists. This precision data demonstrated that as expected with all realtime PCR assays, the BD MAX™ MRSA Assay may not generate reproducibly positive results when testing samples that have analyte concentrations lower than the LOD concentration, but higher than the assay cut-off concentration (HN100 and HN10).

This point is addressed by the following statement in the Limitations section of package insert: “As with all PCR-based in vitro diagnostic tests, extremely low levels of target below the LoD of the assay may be detected, but results may not be reproducible.”

b. Linearity/assay reportable range:

Not applicable. The BD MAX™ MRSA is a qualitative assay.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

External controls are not provided by the manufacturer. Various types of external controls are recommended to allow the user to select the most appropriate control for their laboratory quality control program:

Commercially available control materials [e.g. a reference MRSA strain (ATCC 43300) and Methicillin-sensitive Staphylococcus aureus strain (e.g. ATCC 25923) can be used as positive and negative controls, respectively]. Previously characterized specimens known to be positive or negative for MRSA.

External positive and negative controls were included in all analytical and clinical studies performed in support of this submission.

The assay also includes a Sample Processing Control (SPC).

# d. Detection limit:

The analytical sensitivity (limit of detection or LoD) of the BD MAX™ MRSA was determined by testing viable MRSA strains. Two distinct LOD determination studies were conducted using six Methicillin-Resistant Staphylococcus aureus (MRSA) genotypes (mec Right Extremity Junction, MREJ) type i, ii, iii, iv, v and type vii) for each study. One study was performed in absence of nasal matrix (i.e. in the Sample Buffer Tube) and the other was performed in the presence of nasal matrix.

For each MREJ type/condition tested, 108 swabs were soaked with bacterial suspensions at eight different concentrations. Estimated loads per swab, based on bacterial count were between 10 and 1600 CFU/swab. Twenty-four swabs were used for each bacterial concentration (12 swabs x 2 dilution series). After the bacterial suspensions were absorbed by the swabs, sample preparation was performed as

described below.

To prepare negative nasal matrix, nasal swabs obtained from patients were characterized with the BD MAX™ MRSA Assay. Negative samples were pooled to create the negative nasal matrix. Swabs soaked with bacterial suspension, as described above, were placed in Sample Buffer Tubes containing negative nasal matrix or Sample Buffer; the stem of each swab was then broken and the swabs were discharged via vortex for 1 minute.

To evaluate the LoD of the MRSA target with viable bacteria, eight bacterial dilutions of each MREJ type were tested with the BD MAX™ MRSA Assay. To increase variability, three different lots of BD MAX™ MRSA Assay (including Sample Buffer Tubes, Master Mix, Extraction Tubes and Strips) were tested with each bacterial concentration on nine different BD MAX™ instruments.

The results were analyzed using a statistical linear logistic model that describes the relationship between the probability of the response and the bacterial concentration. Briefly, the method models the positive response (expressed in percentage) as a function of Log (CFU/swab). The logistic model equation for the fitted curve allows the computation of the LoD by inverse prediction using the parameter estimates and their $9 5 \%$ confidence interval.

The data and calculations are shown in the following table for MRSA MREJ Type i. This is shown as an example. Detailed data tables were also presented for all the MREJ types tested.

<table><tr><td rowspan=1 colspan=3>MRSA MREJ Type i</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>With nasal matrix</td><td rowspan=1 colspan=1>Without nasal matrix</td></tr><tr><td rowspan=1 colspan=1>CFU/Swab</td><td rowspan=1 colspan=1>#Positive/#Tested(%Positive)</td><td rowspan=1 colspan=1>#Positive/#Tested(%Positive)</td></tr><tr><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0/22 (0)</td><td rowspan=1 colspan=1>0/24 (0)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4/23 (17)</td><td rowspan=1 colspan=1>7/24 (29)</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3/24 (12.5)</td><td rowspan=1 colspan=1>13/24 (54)</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>8/24 (33)</td><td rowspan=1 colspan=1>16/24 (66)</td></tr><tr><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>13/24 (54)</td><td rowspan=1 colspan=1>20/24 (83)</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>17/23 (74)</td><td rowspan=1 colspan=1>20/22 (91)</td></tr><tr><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>20/21 (95)</td><td rowspan=1 colspan=1>22/22 (92)</td></tr><tr><td rowspan=1 colspan=1>800</td><td rowspan=1 colspan=1>24/24 (100)</td><td rowspan=1 colspan=1>24/24 (100)</td></tr><tr><td rowspan=1 colspan=1>1600</td><td rowspan=1 colspan=1>23/23 (100)</td><td rowspan=1 colspan=1>24/24 (100)</td></tr><tr><td rowspan=1 colspan=1>LOD (CFU/swab) [95%Confidence interval]</td><td rowspan=1 colspan=1>645 [314-1326]</td><td rowspan=1 colspan=1>235 [114-488]</td></tr></table>

LoD results (in CFU/swab) for different MRSA MREJ types with and without nasal matrix from the statistical analysis are summarized in the following table:

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>LOD (CFU/swab) [95% Confidence Interval]</td></tr><tr><td rowspan=1 colspan=1>MRSA MREJ Type(SCCmec type)</td><td rowspan=1 colspan=1>With nasal matrix</td><td rowspan=1 colspan=1>Without nasal matrix</td></tr><tr><td rowspan=1 colspan=1>i (I)</td><td rowspan=1 colspan=1>645 [314-1326]</td><td rowspan=1 colspan=1>235 [114-488]</td></tr><tr><td rowspan=1 colspan=1>ii (II)</td><td rowspan=1 colspan=1>400[237-678]</td><td rowspan=1 colspan=1>158[88-283]</td></tr><tr><td rowspan=1 colspan=1>iii (III)</td><td rowspan=1 colspan=1>346[197-608]</td><td rowspan=1 colspan=1>130[66-259]</td></tr><tr><td rowspan=1 colspan=1>iv (III)</td><td rowspan=1 colspan=1>490[264-908]</td><td rowspan=1 colspan=1>162[96-275]</td></tr><tr><td rowspan=1 colspan=1>v (IV)</td><td rowspan=1 colspan=1>273[148-504]</td><td rowspan=1 colspan=1>117[64-215]</td></tr><tr><td rowspan=1 colspan=1>vii (II)</td><td rowspan=1 colspan=1>357 [215-594]</td><td rowspan=1 colspan=1>140 [75-264]</td></tr></table>

LOD ranges for MRSA strains were as follows:

Without nasal matrix, the LOD ranged between 117 and 235 CFU/swab with an average LOD of 157 CFU/swab. · With nasal matrix, the LOD ranged between 273 and 645 CFU/swab with an average LOD of 418 CFU/swab.

e. Analytical specificity:

Inclusivity

An analytical inclusivity study was performed using a variety of Methicillin resistant Staphylococcus aureus strains, taking into account geographic origin, MREJ genotype, SCCmec type, pulse field gel electrophoresis (PFGE) type, temporal diversity and susceptibility pattern. Seventy-five (75) strains from 30 countries were tested in this study, including strains from public collections and from wellcharacterized clinical isolates, including vancomycin-resistant Staphylococcus aureus (VRSA) and vancomycin- intermediate Staphylococcus aureus (VISA) strains.

The BD MAX™ MRSA Assay detected all of the MREJ types i, ii, iii, iv, v and vii (wild and mutant) when tested at low bacterial load $( 2 \small { - } 3 \mathrm { ~ x ~ L o D } )$ ). The BD MAX™ MRSA Assay detected MRSA SCCmec types I, II, III, IV, V and VI, VII and VIII, as well as MRSA PFGE types USA 100 to 800, 1000 and 1100 at $2 { \cdot } 3 \ \mathbf { x }$ LoD. All methicillin-resistant Staphylococcus aureus strains displaying additional resistance to vancomycin (VRSA and VISA) were also detected.

A negative result was obtained for one strain. However, an investigation revealed that there was a dilution error. A new dilution was prepared and the PCR test demonstrated positive result.

# Empty Cassette Variants

The package insert includes the following limitation: “The BD MAX™ MRSA Assay does not detect the mecA gene directly nor the penicillin-binding protein (PBP 2a) encoded by this gene. A false positive MRSA result may occur if an “empty cassette” S. aureus variant is present”.

# Well Characterized Challenge

A study was conducted to determine the assay results obtained with a challenge strain panel containing MRSA strains with high and low oxacillin MIC values, including PFGE types USA 100, 300, and 400 (with emphasis on USA 300), BORSA, MSSA and methicillin-resistant Staphylococcus epidermidis (MRSE) strains using the BD MAX™ MRSA Assay.

The challenge strain panel used in this study was composed of 17 MRSA, four BORSA, one MRSE and five MSSA strains. All these strains were tested with FDAcleared methods for determination of the MIC. All the MRSA strains tested as part of the Challenge study exhibited positive results when tested at 2-3X LoD. All BORSA, MSSA and MRSE strains tested exhibited negative results when tested at high concentrations.

Six different mecA variant mecALGA251 strains were tested with the BD MAX™ MRSA Assay and the strains were negative in this assay. These strains do not belong to MREJ genotypes i, ii, iii, iv, v and vii.

This finding is addressed by the following statement in the Limitations section of package insert: “Methicillin-resistant S. aureus strains that carry the mecALGA251 gene mutation, a novel mecA variant, may not be detected by the BD MAX™ MRSA Assay, resulting in false negative results”.

# Cross-Reactivity (bacterial strains)

A study was conducted to assess the analytical specificity of the assay BD MAX™ MRSA with well-characterized isolates of Methicillin-susceptible Staphylococcus aureus (MSSA), Coagulase Negative and Positive Staphylococci (CoNS and CoPS), closely related genera and other pathogenic and commensal organisms of the flora found in the human nares. Fifty-seven non-staphylococcal strains (representing 54 species), $3 8 \mathrm { C o N S }$ and seven CoPS strains (representing 29 species) and 111 MSSA strains were tested using bacterial suspensions ranging between $1 . 1 \mathrm { E } { + } 0 7$ and $1 . 0 \mathrm { E } { + } 0 8 $ CFU/mL. All these species are either phylogenetically related to $S .$ aureus or are commonly found in the human nares.

Upon first testing of 213 non-target species with the BD MAX™ MRSA assay, five strains showed positive results. An investigation demonstrated that the false-positive results obtained were due to contamination, as evidenced by an electrophoresis gel analysis and by the profile of the PCR amplification curves. New suspensions of the strains that produced initial false-positive results were tested with a new suspension and generated the expected negative results. Therefore, all non-target strains yielded negative results when tested with the BD MAX™ MRSA Assay.

<table><tr><td rowspan=1 colspan=3>Non-Staphylococcal strains</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter baumannii</td><td rowspan=1 colspan=1>Enterococcus faecium</td><td rowspan=1 colspan=1>Streptococcus mitis</td></tr><tr><td rowspan=1 colspan=1>Acinetobacter haemolyticus</td><td rowspan=1 colspan=1>Enterococcus flavescens</td><td rowspan=1 colspan=1>Streptococcus mutans</td></tr><tr><td rowspan=1 colspan=1>Bacillus cereus</td><td rowspan=1 colspan=1>Enterococcus hirae</td><td rowspan=1 colspan=1>Streptococcus pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Enterococcus gallinarum</td><td rowspan=1 colspan=1>Streptococcus pyogenes</td></tr><tr><td rowspan=1 colspan=1>Candida albicans (2)</td><td rowspan=1 colspan=1>Escherichia coli (3)</td><td rowspan=1 colspan=1>Streptococcus salivarius</td></tr><tr><td rowspan=1 colspan=1>Candida guilliermondii</td><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>Streptococcus sanguinis</td></tr><tr><td rowspan=1 colspan=1>Candida tropicalis</td><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>Streptococcus suis</td></tr><tr><td rowspan=1 colspan=1>Candida glabrata</td><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>Streptococcus sp.</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>Lactobacillus crispatus</td><td rowspan=1 colspan=1>Pasteurella aerogenes</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>Lactobacillus reuteri</td><td rowspan=1 colspan=1>Proteus mirabilis</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium aquaticum</td><td rowspan=1 colspan=1>Lactobacillus acidophilus</td><td rowspan=1 colspan=1>Proteus vulgaris</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium bovis</td><td rowspan=1 colspan=1>Listeria monocytogenes</td><td rowspan=1 colspan=1>Providencia stuartii</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium flavescens</td><td rowspan=1 colspan=1>Micrococcus luteus</td><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td></tr><tr><td rowspan=1 colspan=1>Corynebacterium genitalium</td><td rowspan=1 colspan=1>Moraxella catarrhalis</td><td rowspan=1 colspan=1>Pseudomonas fluorescens</td></tr><tr><td rowspan=1 colspan=1>Cryptococcus neoformans</td><td rowspan=1 colspan=1>Neisseria gonorrhoeae</td><td rowspan=1 colspan=1>Salmonella enterica subsp. enterica</td></tr><tr><td rowspan=1 colspan=1>Enterobacter aerogenes</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>Serratia marcescens</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>Streptococcus anginosus</td><td rowspan=1 colspan=1>Shigella sonnei</td></tr><tr><td rowspan=1 colspan=1>Enterococcus faecalis</td><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>Yersinia enterocolitica</td></tr></table>

<table><tr><td rowspan=1 colspan=1>CoPS</td><td rowspan=1 colspan=2>CoNS</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus intermedius</td><td rowspan=1 colspan=1>Staphylococcus arlettae</td><td rowspan=1 colspan=1>Staphylococcus hominis (3)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus delphini</td><td rowspan=1 colspan=1>Staphylococcus auricularis</td><td rowspan=1 colspan=1>Staphylococcus hominis subsp.hominis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus lutrae (2)</td><td rowspan=1 colspan=1>Staphylococcus capitis</td><td rowspan=1 colspan=1>Staphylococcus kloosii</td></tr><tr><td rowspan=1 colspan=1>Staphylococcuspseudointermedius</td><td rowspan=1 colspan=1>Staphylococcus caprae</td><td rowspan=1 colspan=1>Staphylococcus lentus</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi</td><td rowspan=1 colspan=1>Staphylococcus carnosus</td><td rowspan=1 colspan=1>Staphylococcus lugdunensis</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus schleiferi subscoagulans</td><td rowspan=1 colspan=1>Staphylococcus chromogenes</td><td rowspan=1 colspan=1>Staphylococcus pasteuri</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus cohnii subsp.urealyticum</td><td rowspan=1 colspan=1>Staphylococcus pulvereri</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus epidermidis (9)</td><td rowspan=1 colspan=1>Staphylococcus saprophyticus</td></tr><tr><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>Staphylococcus equorum</td><td rowspan=1 colspan=1>Staphylococcus sciuri</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus felis</td><td rowspan=1 colspan=1>Staphylococcus simulans</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus gallinarum</td><td rowspan=1 colspan=1>Staphylococcus warneri (2)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Staphylococcus haemolyticus (3)</td><td rowspan=1 colspan=1>Staphylococcus xylosus (2)</td></tr></table>

A study was conducted to assess the analytical specificity of the BD MAX™ MRSA with different viruses that may be found in the human nares. Seventeen different titrated viral strains were used. Viral strains were tested at $> 1 . 0 \mathrm { E } + 0 5 \mathrm { P F U / m L }$ or equivalent unit (e.g. TCID50 units $ { \mathrm { / m L } }$ or $\mathsf { c p / m L }$ ). The Epstein-Barr virus was tested at $1 . 0 \mathrm { E } + 0 9 \mathrm { c p / m L }$ . All 17 viral strains yielded negative results when tested with the BD MAX™ MRSA Assay.

# Interfering substances:

Potentially interfering biological and chemical substances that may be found in nasal swabs were tested at their highest clinically relevant concentrations to determine the effect on assay performance. Substances tested were either endogenous (naturally present) or exogenous (artificially introduced into the nasal cavity).

The following twenty substances were tested using specimens spiked with MRSA at 2-3X LOD and the highest amount of each compound. The expected positive results were obtained with all substances. Tobramycin, showed slight inhibition (delay of Second Derivative Peak Abscissa), however, the expected assay results were still obtained.

<table><tr><td rowspan=1 colspan=1>Mucin, from bovine submaxillary glands</td><td rowspan=1 colspan=1>Rhinocort aquaTM</td></tr><tr><td rowspan=1 colspan=1>Dexamethasone Sodium PhosphateOphthalmic Solution USP, 0.1%Dexamethasone Phosphate Equivalent</td><td rowspan=1 colspan=1>NasonexTM</td></tr><tr><td rowspan=1 colspan=1>ChlorasepticT</td><td rowspan=1 colspan=1>Fluticasone Propionate</td></tr><tr><td rowspan=1 colspan=1>Taro-Mupirocin, Mupirocin OintmentUSP, 2%</td><td rowspan=1 colspan=1>LuffeelTM</td></tr><tr><td rowspan=1 colspan=1>Long Lasting DristanTM Nasal Mist</td><td rowspan=1 colspan=1>Zicam® No-Drip LiquidTM Nasal GeljTMExtreme Congestion Relief</td></tr><tr><td rowspan=1 colspan=1>Neo-SynephrineT</td><td rowspan=1 colspan=1>RelenzaTM</td></tr><tr><td rowspan=1 colspan=1>Otrivin™ Complete Nasal CareTM</td><td rowspan=1 colspan=1>Tobramycin</td></tr><tr><td rowspan=1 colspan=1>Beconase AQTM</td><td rowspan=1 colspan=1>Blood</td></tr><tr><td rowspan=1 colspan=1>Flunisolide Nasal Solution USP, 0.025%</td><td rowspan=1 colspan=1>MSSA (ATCC 29213)</td></tr><tr><td rowspan=1 colspan=1>Nasacort™M AQ</td><td rowspan=1 colspan=1>CNS (ATCC 35983)</td></tr></table>

# Carry over contamination

A panel made of one high positive member and one negative member was used to prepare numerous samples for this study. An MREJ type v MRSA strain was used for the high positive panel member $[ 8 \mathrm { E } + 0 7$ CFU/swab) as it is the MREJ type with the lowest LoD. ID Broth was used as the negative panel member. Twelve replicates of the high positive panel member and 12 replicates of the negative panel member were tested in each run; high positive and negative panel members were

alternated sequentially in each run. Three operators performed three consecutive runs for a total of nine runs of 24 samples. No false positive result due to cross contamination was observed in this study

2. Comparison studies:

a. Method comparison with predicate device: Not applicable.

b. Matrix comparison: Not applicable.

3. Clinical studies:

a. Prospective Clinical Studies:

Clinical performance characteristics of the BD MAX™ MRSA Assay were determined in a multi-site prospective investigational study. Four geographically diverse U.S. investigational centers participated in the study. To be enrolled in the study, patients had to be eligible for MRSA testing according to institutional policies. Eligibility requirements for targeted screening as per clinical site policies included, but were not limited to: patients admitted into the particular healthcare system; patients admitted to the Intensive Care Unit; patients transferred to the Intensive Care Unit; pre-elective surgery patients; and patients being admitted from long term care facilities. Specimens from patients previously enrolled in the study were excluded. All specimens in the study meeting the inclusion and exclusion criteria represented excess, de-identified left over material from normal standard of care laboratory testing.

All clinical sites used the same comparative reference method which consisted of direct culture complemented by enriched culture. Enriched culture analysis was completed for all specimens that were negative for MRSA by direct culture.

The culture method consisted of an initial analysis on BBL™ CHROMagar™ Staphylococcus aureus selective chromogenic media followed by subculture on Blood Agar (BA) of presumptive S. aureus colonies. Identification of S. aureus was confirmed by performing a Staphaurex PlusTM agglutination test from the subcultured BA plate. Methicillin/Oxacillin resistance was confirmed by cefoxitin disk diffusion susceptibility testing in accordance with the CLSI standards $\scriptstyle < = 2 1 { \mathrm { ~ m m } } =$ resistant; $> = 2 2 ~ \mathrm { m m } =$ susceptible). Enrichment in Trypticase Soy Broth (TSB) (BBLTM TrypticaseTM Soy Broth with $6 . 5 \%$ Sodium Chloride) was completed in the event that methicillin-resistant $S .$ . aureus was not confirmed by the initial method. The TSB was used to inoculate additional BBL™ CHROMagar™ Staphylococcus aureus selective chromogenic media and BA plates, and MRSA confirmation was performed as

described above.

A total of 1987 nasal swab specimens were tested with both direct culture and the BD MAX™ MRSA Assay. Of those, 106 specimens were regarded as non-compliant per protocol criteria for sensitivity and specificity calculations. Therefore, there were 1881 reportable results upon exclusion on noncompliant specimens from performance analysis. Non-compliant reason description and total non-compliant specimens were tabulated by site.

The following tables summarize the performance obtained with clinical specimens using the BD MAX™ MRSA Assay in comparison to the combined reference culture methods (Direct and Enriched). The overall BD MAX™ MRSA Assay sensitivity and specificity of these 1881 specimens were $9 3 . 0 \%$ (146/157) and $9 5 . 9 \%$ (1653/1724), respectively. For the population tested, the Negative and Positive Predictive Value were $9 9 . 3 \%$ , and $67 . 3 \%$ , respectively.

Overall (all study sites) Performance Results for the BD MAX™ MRSA Assay in Comparison with Direct and Enriched Culture Reference Methods:   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>Reference Culture Method</td></tr><tr><td rowspan=1 colspan=1>BD MAXTMMRSA</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>146</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>217</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1653</td><td rowspan=1 colspan=1>1664</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>1724</td><td rowspan=1 colspan=1>1881</td></tr><tr><td rowspan=1 colspan=1>Sensitiv</td><td rowspan=1 colspan=3>Sensitivity: 93.0% (87.9%, 96.0%)       Specificity: 95.9% (94.8%, 96.7%)*</td></tr></table>

\*Numbers in Parentheses represent the $9 5 \%$ confidence interval boundaries

Site-Specific Performance Results for the BD MAX™ MRSA Assay in Comparison with Direct and Enriched Culture Reference Methods:   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="3" rowspan="1">Reference Culture Method</td></tr><tr><td colspan="1" rowspan="1">Site</td><td colspan="1" rowspan="1">BD MAXTMMRSA</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">MUSC</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">19</td><td colspan="1" rowspan="1">47</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">435</td><td colspan="1" rowspan="1">435</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">28</td><td colspan="1" rowspan="1">454</td><td colspan="1" rowspan="1">482</td></tr><tr><td colspan="5" rowspan="1">*Sensitivity: 100% (87.9%, 100%)   Specificity: 95.8% (93.6%, 97.3%)</td></tr><tr><td colspan="1" rowspan="1">PORT</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">17</td><td colspan="1" rowspan="1">38</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">465</td><td colspan="1" rowspan="1">467</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">482</td><td colspan="1" rowspan="1">505</td></tr><tr><td colspan="1" rowspan="1">Sensitivity: 9</td><td colspan="4" rowspan="1">*Sensitivity: 91.3% (73.2%, 97.6%)  Specificity: 96.5% (94.4%, 97.8%)</td></tr><tr><td colspan="1" rowspan="1">WISC</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">65</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">346</td><td colspan="1" rowspan="1">351</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">55</td><td colspan="1" rowspan="1">361</td><td colspan="1" rowspan="1">416</td></tr><tr><td colspan="5" rowspan="1">*Sensitivity: 90.9% (80.4%, 96.1%)  Specificity: 95.8% (93.3%, 97.5%)</td></tr><tr><td colspan="1" rowspan="1">WISH</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">67</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">407</td><td colspan="1" rowspan="1">411</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">51</td><td colspan="1" rowspan="1">427</td><td colspan="1" rowspan="1">478</td></tr><tr><td colspan="5" rowspan="1">Specificity: 95.3% (92.9%, 96.9%)*Sensitivity: 92.2% (81.5%, 96.9%)</td></tr></table>

\*Numbers in Parentheses represent the $9 5 \%$ confidence interval boundaries

BD MAX™ MRSA was also compared to the direct culture alone (i.e. without enrichment). The BD MAX™ MRSA Assay identified $9 5 . 0 \%$ of the MRSA positive specimens and $9 5 . 2 \%$ of the MRSA negative specimens compared to direct culture.

Ten specimens out of 1884 total specimens $( 0 . 5 \% )$ were reported Unresolved after initial testing. The 1884 specimens include the 1881 compliant specimens used for the sensitivity and specificity calculations plus three additional specimens that produced compliant PCR results. These three specimens were not compliant for the reference method but these data are included here so as not to bias the non reportable rates.

In order to accurately reflect the Final Unresolved Rate, the following description and table will be included in the BD MAX MRSA assay package insert:

“Out of 1884 nasal swab specimens tested with the BD MAX™ MRSA Assay, 10 $( 0 . 5 \% )$ were reported as Unresolved after initial testing. The Unresolved Rate after repeat is based upon a total of 1882 specimen results (two specimens with initial Unresolved results were not retested). All specimens had reportable results after repeat testing.”

<table><tr><td rowspan=1 colspan=1>ClinicalSites</td><td rowspan=1 colspan=2>Initial unresolved rate (%, n/N)(95% CI)</td><td rowspan=1 colspan=2>−Unresolved rate (%, n/N) afterrepeat (95% CI)</td></tr><tr><td rowspan=1 colspan=1>MUSC</td><td rowspan=1 colspan=1>0.8% (4/484)</td><td rowspan=1 colspan=1>(0.3%, 2.1%)</td><td rowspan=1 colspan=1>0.0% (1/483)</td><td rowspan=1 colspan=1>(0.0%,0.8%)</td></tr><tr><td rowspan=1 colspan=1>PORT</td><td rowspan=1 colspan=1>0.0% (0/505)</td><td rowspan=1 colspan=1>(0.0%, 0.8%)</td><td rowspan=1 colspan=1>0.0% (0/505)</td><td rowspan=1 colspan=1>(0.0%, 0.8%)</td></tr><tr><td rowspan=1 colspan=1>WISC</td><td rowspan=1 colspan=1>0.2% (1/416)</td><td rowspan=1 colspan=1>(0.0%, 1.3%)</td><td rowspan=1 colspan=1>0.0% (0/416)</td><td rowspan=1 colspan=1>(0.0%, 0.9%)</td></tr><tr><td rowspan=1 colspan=1>WISH</td><td rowspan=1 colspan=1>1.0% (5/479)</td><td rowspan=1 colspan=1>(0.4%, 2.4%)</td><td rowspan=1 colspan=1>0.0% (0/478)</td><td rowspan=1 colspan=1>(0.0%, 0.8%)</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>0.5% (10/1884)</td><td rowspan=1 colspan=1>(0.3%, 1.0%)</td><td rowspan=1 colspan=1>0.0% (0/1882)</td><td rowspan=1 colspan=1>(0.0%, 0.2%)</td></tr></table>

b. Retrospective Clinical Studies: Not applicable.   
c. Other clinical supportive data (when a. and b. are not applicable): Not applicable.

4. Clinical cut-off:

Not applicable.

5. Expected values/Reference range:

In the BD MAX™ MRSA Assay clinical study, a total of 1903 specimens were tested from four geographically diverse U.S. clinical sites using combined Direct/Enriched culture. The study population was grouped into in-patient and out-patient categories. The number and percentage of positive cases as determined by the reference culture method are presented in the table below:

<table><tr><td rowspan=2 colspan=1>Group</td><td rowspan=2 colspan=1>Total N</td><td rowspan=1 colspan=2>MRSA by Reference (Directplus Enriched) Culture</td><td rowspan=2 colspan=1>Observed Prevalence*</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>In-Patient</td><td rowspan=1 colspan=1>1473</td><td rowspan=1 colspan=1>133</td><td rowspan=1 colspan=1>1340</td><td rowspan=1 colspan=1>9.0% (133/1473)</td></tr><tr><td rowspan=1 colspan=1>Out-Patient</td><td rowspan=1 colspan=1>430</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>404</td><td rowspan=1 colspan=1>6.0% (26/430)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1903</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>1744</td><td rowspan=1 colspan=1>8.4% (159/1903)</td></tr></table>

\*Prevalence based on reference method.

# N. Instrument Name:

BD MAX™ System

# O. System Descriptions:

1. Modes of Operation:

The 2nd generation BD MAX™ System fully automates cell lysis, nucleic acid extraction, PCR set-up, target amplification and detection. The system can process and analyze up to 24 specimens in one cartridge with two cartridges running simultaneously on the instrument. The system includes external and internal barcode reading, ensuring traceability throughout extraction and PCR process. The system includes a heater module, temperature sensors, and a fluorescence detection system with six optical channels.

2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X___ or No _

3. Specimen Identification: Specimens are labeled with a Barcode.

4. Specimen Sampling and Handling: The nasal swab is inserted in the sample buffer tube, vortexed and placed into the system.

5. Calibration:

The system is calibrated by the manufacturer on-site as part of the installation procedure as well as during biannual preventive maintenance.

6. Quality Control:

External controls are not provided by the manufacturer. Various types of external controls are recommended to allow the user to select the most appropriate control for their laboratory quality control program:

· Commercially available control materials [e.g. a reference MRSA strain (ATCC 43300) and Methicillin-sensitive Staphylococcus aureus strain (e.g. ATCC 25923) can be used as positive and negative controls, respectively]. Previously characterized specimens known to be positive or negative for MRSA.

The assay also includes a Sample Processing Control (SPC), which is a plasmid containing a synthetic target DNA sequence. The SPC is extracted, eluted and amplified along with other DNA present in the processed specimen. The SPC monitors for sample processing and efficiency of DNA amplification and detection.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable.

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.