# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number K120267

# B. Purpose for Submission

To provide additional clinical evaluation data to supplement previous FilmArray Respiratory® Panel (RP) System 510k submissions (k103175/k110764), and to support the unmasking of analytes Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus OC43, Coronavirus 229E, and Mycoplasma pneumoniae, which were masked in the FDA cleared version of the FilmArray RP System under k103175/k103175 due to lack of sufficient performance data.

# C. Measurands

Adenovirus, Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Human Metapneumovirus, Human Rhinovirus/ Enterovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype 2009 H1, Influenza B, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, and Respiratory Syncytial Virus nucleic acid target sequences.

# D. Type of Test

A multiplexed nucleic acid test intended for use with the FilmArray instrument for the qualitative in vitro detection and identification of multiple respiratory pathogen nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infection.

# E. Applicant

Idaho Technology Inc.

# F. Proprietary and Established Names

Established name: FilmArray® Respiratory Panel (RP)   
Common Name(s): FilmArray ® Respiratory Panel (RP) FilmArray® Respiratory Panel (RP) System FilmArray® Respiratory Panel (FilmArray RP) FilmArray RP Panel

# G. Regulatory Information

<table><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="1" rowspan="1">Classification</td><td colspan="1" rowspan="1">Regulation Section</td><td colspan="1" rowspan="1">Panel</td></tr><tr><td colspan="1" rowspan="1">OCC(respiratory viralpanel)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEM(HPMV)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOU(parainfluenzavirus)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OEP(influenza A)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OTG(non-SARShumancoronavirus)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">NXD(influenza Asubtypes)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3332Reagents for Detection ofSpecific Novel Influenza A</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OOI(Multiplexsystem)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 862.2570Instrumentation forClinical Multiplex Test</td><td colspan="1" rowspan="1">Clinical Chemistry(75)</td></tr><tr><td colspan="1" rowspan="1">OZZ(Bordetellapertussis)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZY(Chlamydophilapneumoniae)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr><tr><td colspan="1" rowspan="1">OZX(Mycoplasmapneumoniae)</td><td colspan="1" rowspan="1">Class II</td><td colspan="1" rowspan="1">21 CFR 866.3980Respiratory Viral PanelMultiplex Nucleic Acid Assay</td><td colspan="1" rowspan="1">Microbiology (83)</td></tr></table>

# H. Intended Use

# 1. Intended use:

FilmArray Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray instrument for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype $2 0 0 9 \mathrm { H } 1$ , Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Rhinovirus/Enterovirus, Respiratory Syncytial Virus, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test or lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out co-infection with other organisms: the agent(s) detected by the Film Array RP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis, Coronavirus 229E, Coronavirus OC43, Influenza A/H1, Influenza A/H3, Influenza $\mathbf { A } / 2 \mathbf { 0 0 9 } \mathbf { H 1 }$ , Influenza B, Mycoplasma pneumoniae Parainfluenza Virus 1, Parainfluenza Virus 2, and Parainfluenza Virus 4 were established primarily with retrospective clinical specimens. Performance characteristics for Chlamydophila pneumoniae were established primarily using contrived clinical specimens.

Due to the genetic similarity between human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis).

The FilmArray RP detects Adenovirus species C serotype 2 and serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture).

The FilmArray RP assay for Coronavirus OC43 may cross-react with some isolates of Coronavirus HKU1. A dual positive result may be due to cross-reactivity or may indicate a co-infection.

Performance characteristics for influenza A were established when influenza A/2009 H1N1, A/H1, and A/H3 were the predominant influenza A viruses in circulation. Performance of detecting influenza A may vary if other influenza A strains are circulating or a novel influenza A virus emerges. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication for Use: Same as Intended Use.

3. Special conditions for use statement(s): For prescription use only.

4. Special instrument requirements: FilmArray instrument.

# I. Device Description:

The FilmArray RP pouch contains primers for the following respiratory pathogens: Adenovirus, Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus (HKU1, NL63, OC43, 229E), Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A ((with subtyping for hemagglutinin genes H1, 2009 H1 and H3), Influenza B, Mycoplasma pneumoniae, Parainfluenza Virus 1 (serotypes 1, 2, 3, 4) and Respiratory Syncytial Virus. The FilmArray RP pouch is a closed system disposable that houses all the chemistry required to isolate, amplify and detect nucleic acid from multiple respiratory pathogens within a single NPS specimen. The rigid plastic component (fitment) of the FilmArray RP pouch contains reagents in freeze-dried form. The flexible plastic portion of the pouch is divided into discrete segments (blisters) which, through interactions with actuators and sensors in the FilmArray instrument, are where the required chemical processes are carried out. The user of the FilmArray RP system loads the sample into the FilmArray RP pouch, places the pouch into the FilmArray instrument, and starts the run. All other operations are automated. A summary of the FilmArray assays and targets is presented in the following table:

Film Array Assays and Targets   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Assay Type</td><td rowspan=1 colspan=1>Number ofPCR2Assays</td><td rowspan=1 colspan=1>Gene Target</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hexon</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Toxin (promoter region)</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Outer membrane protein</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Nucleoprotein</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Nucleoprotein</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Nucleoprotein</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Polymerase</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Matrix (Pan 1 assay)Non structural Protein (Pan 2 assay)</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hemagglutinin</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 2009</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hemagglutinin</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hemagglutinin</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hemagglutinin</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Nucleocapsid</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Toxin</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>5&#x27;UTR</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus1</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Hemagglutinin —Neuraminidase</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fusion</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fusion</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Fusion</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial virus(RSV)</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Matrix</td></tr><tr><td rowspan=1 colspan=1>RNA Process Control</td><td rowspan=1 colspan=1>RNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Schizosaccharomyces pombe</td></tr><tr><td rowspan=1 colspan=1>PCR2 Control</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>Arabidposs/Modified Synthetic</td></tr></table>

# 1. The FilmArray RP Reagent Kit:

The FilmArray RP Regent Kit contains all materials required to complete 30 tests. The contents of the test kit, with a brief description of each component, are described below:

• 30 FilmArray RP pouches: The RP pouches are used to test the patient samples. Each reagent pouch is packaged in a metal canister under vacuum. To maintain the vacuum, the metal canisters are packed in an outer foil wrapper.

30 transfer pipettes: The transfer pipettes are used to transfer approximately $2 5 0 \mu \mathrm { L }$ of patient NPS in viral transport media (VTM) to the single use Sample Buffer vials. 30 vials Sample Buffer (Red lid): Each single use vial contains $5 0 0 \mu \mathrm { L }$ of Sample Buffer. Patient sample is added to the sample buffer before it is injected into the FilmArray RP pouch. The Sample Buffer is a guanidine hydrochloride solution that serves to inactivate RNases in the sample and promote binding of nucleic acids to the magnetic beads for extraction. 30 vials Hydration Solution (Blue lid): Each single use vial contains $1 . 5 \mathrm { m L }$ of Hydration Solution (molecular grade water). The Hydration Solution is used to rehydrate the freeze-dried reagents contained in the FilmArray RP pouch. • 30 individually packaged Sample Loading Syringes with attached cannula (denoted with a red cap): Used for adding the patient sample/ buffer mixture to the pouch. 30 individually packaged Pouch Hydration Syringes with attached cannula (denoted with a blue cap): Used for adding Hydration Solution to the pouch prior to testing.

# 2. The FilmArray RP Pouch:

The FilmArray RP pouch is a closed system disposable that houses all the chemistry required to isolate, amplify and detect nucleic acid from an NPS specimen. The pouch is divided into discrete segments (blisters) where specific steps are carried out. Reagents are arranged in the pouch such that sample is moved through the FilmArray RP pouch sequentially as each biochemical event is performed.

# 3. The FilmArray Instrument:

The FilmArray instrument interacts with the FilmArray pouch mechanically, thermally, and optically to drive a multi-step chemical process designed to detect specific nucleic acid targets using multiplex nested PCR followed by DNA melting analysis. The instrument follows a protocol that is defined by a set of codes that are downloaded from the host computer at runtime. The instrument protocol defines the specific timing and sequence parameters as the instrument performs the following key functions:

• Perform cell disruption using the bead beater   
• Extract nucleic acid from the disrupted sample   
• Perform stage 1 PCR thermocycling of multiplexed PCR reaction   
• Perform stage 2 PCR thermocycling of the array   
• Execute a DNA melt and detect fluorescent signals generated   
• Monitor system performance in real time and communicates out of specification conditions to the user via the software

# 4. Materials Required But Not Provided:

• FilmArray Instrument • FilmArray Pouch Loading Station (provided with FilmArray Instrument)

# J. Substantial Equivalence Information

1. Predicate device name(s):

FilmArray Respiratory Panel (RP) System.

# 2. Predicate K number(s):

K 103175, K110764.

3. Comparison with predicate(s):

Similarities with Predicate Device   

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:FilmArray Respiratory Panel System</td><td rowspan=1 colspan=1>Predicate:FilmArray Respiratory Panel System(K103175 and K110764)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>RNA/DNA</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Specimen Types</td><td rowspan=1 colspan=1>Nasopharyngeal swabs</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>FilmArray Instrument</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>Less than 1 hour</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Test Interpretation</td><td rowspan=1 colspan=1>Automated test interpretation and reportgeneration. User cannot access raw data.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Sample PreparationMethod</td><td rowspan=1 colspan=1>Sample processing is automated in theFilmArray instrument.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Reagent Storage</td><td rowspan=1 colspan=1>Reagents are stored at room temperature.</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Two controls are included in each reagentpouch to control for sample processing andboth stages of PCR and melt analysis.</td><td rowspan=1 colspan=1>Same</td></tr></table>

# Differences with Predicate Device

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>New Device:FilmArray Respiratory Panel System</td><td rowspan=1 colspan=1>Predicate:FilmArray Respiratory Panel System(K103175 and K110764)</td></tr><tr><td rowspan=1 colspan=1>Organisms Detected</td><td rowspan=1 colspan=1>Same as predicate with additionalorganisms: Bordetella pertussis,Chlamydophila pneumoniae, CoronavirusOC43, Coronavirus OC229E, andMycoplasma pneumoniae.</td><td rowspan=1 colspan=1>Adenovirus, Coronavirus HKU1,Coronavirus NL63, HumanMetapneumovirus, Influenza A, InfluenzaA subtype H1, Influenza A subtype H3,Influenza A subtype 2009 H1, InfluenzaB, Parainfluenza 1, Parainfluenza 2,Parainfluenza virus 3,Parainfluenza 4,Respiratory Syncytial Virus, andRhinovirus/Enterovirus,</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable)

1. User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute Approved Guideline, EP12-A (August 2002)   
2. Molecular Diagnostic Methods for Infectious Diseases, Clinical and Laboratory Standards Institute Approved Guideline, MM3-A (December 1995)   
3. Interference Testing in Clinical Chemistry, Clinical and Laboratory Standards Institute Approved Guideline EP7-A (December 2002)   
4. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005)   
5. Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses (February 15, 2008)   
6. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays (October 9, 2009)   
7. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9, 2009)   
8. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 9, 2009)   
9. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005)   
10. Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999)   
11. Nucleic Acid Based in Vitro Diagnostic Devices for Detection of Microbial Pathogens, FDA Guidance Document (DRAFT: December 8, 2005)   
12. User Protocol for Evaluation of Qualitative Test Performance, National Committee on Clinical Laboratory Standards (NCCLS) Approved Guideline, EP12-A (August 2002)   
13. Guidance for Industry and FDA Staff – Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use (November 30, 2004)

# L. Test Principle

The FilmArray RP System is a multiplex nucleic acid test system composed of the FilmArray instrument, the FilmArray software (preinstalled on a laptop computer) and the FilmArray RP pouch. The FilmArray RP pouch contains freeze-dried reagents to perform nucleic acid purification, reverse transcription, and nested, multiplex PCR with DNA melt analysis. The Respiratory Panel (RP) pouch identifies the following 20 respiratory pathogens: Adenovirus, Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Human Metapneumovirus, Human Rhinovirus/Enterovirus, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype 2009 H1, Influenza B, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, and Respiratory Syncytial Virus.

A test is initiated by loading Hydration Solution and an unprocessed patient nasopharyngeal swab (NPS) specimen (i.e., specimen mixed with Sample Buffer) into the FilmArray RP pouch. The pouch contains all of the reagents required for specimen testing and analysis in a freeze-dried format. The addition of Hydration Solution and Sample Buffer Mix rehydrates the reagents. After the pouch is prepared, the FilmArray software guides the user though the steps of placing the pouch into the instrument, scanning the pouch barcode, entering the sample identification and initiating the run. The FilmArray instrument contains a coordinated system of inflatable bladders and seal points, which act on the pouch to control the movement of liquid between the pouch blisters. Two Peltier devices control heating and cooling of the pouch to drive the reverse transcription reactions, the PCR reactions, and the melting curve analysis. Nucleic acid extraction occurs within the FilmArray pouch using mechanical lysis and standard magnetic bead technology. After extracting and purifying nucleic acids from the unprocessed sample, the FilmArray performs a nested multiplex PCR that is executed in two stages. During the first stage, the FilmArray performs a single highly multiplexed reverse transcription PCR reaction. The products from first stage PCR are then diluted and combined with a fresh, primer-free master mix and a fluorescent double stranded DNA binding dye (LC Green®Plus, Idaho Technology Inc.). This second master mix solution is then distributed to each well of the array. Array wells contain sets of primers designed specifically to amplify sequences internal to the PCR products generated during the first stage PCR reaction. The second stage PCR, or nested PCR, is performed in singleplex fashion in each well of the array. At the conclusion of the second stage PCR, the array is interrogated by melting curve analysis for the detection of signature amplicons denoting the presence of specific viral or bacterial targets. A digital camera placed in front of the second stage PCR captures fluorescent images of the PCR reactions in real time. The FilmArray software automatically interprets the results of each DNA melting curve analysis and combines the data with the results of the internal pouch controls to provide a test result for each organism on the panel.

The melt detector module of the FilmArray Analysis software assigns positive and negative results to melt curves generated in each well of the PCR2 array. Each assay is present in triplicate wells distributed across the array. If at least 2 out of 3 wells have positive melt curves with similar profiles, a positive result is assigned to that assay. If less than 2 of the replicate wells have positive melt curves in the Tm window, or if the curves are dissimilar, a negative result is assigned to the assay. A meta-analysis determines the final result [Detected (positive), Not Detected (negative), or Equivocal] for each RP organism. For organisms represented by a single PCR2 assay, the assay result and organism result are the same. Results for Influenza A, and Human Rhinovirus/Enterovirus are determined by meta-analysis of multiple assay results. Meta-analysis for Human Rhinovirus/Enterovirus incorporates the results of 6 different assays (Entero1, Entero2, HRV1, HRV2, HRV3, and HRV4). The HRV assays recognize both Rhinoviruses and Enteroviruses, while the Entero assays target a unique sequence that is not found in Rhinoviruses. Any positive assay result will produce a combined Human Rhinovirus/Enterovirus organism result. Organism results for Influenza A are determined by meta-analysis of multiple pan-influenza A and subtype targeted assay results. The FilmArray RP System is designed to both detect Influenza A and to differentiate commonly occurring hemagglutinin subtypes. To accomplish this, the FilmArray RP uses two Influenza A assays, (FluA-pan1and FluA-pan2) and three subtyping assays directed at the hemagglutinin gene (H1-pan, H1-2009 and H3).

The H1-pan assay is designed to detect Influenza A/H1, including the Influenza $\mathbf { A } / 2 0 0 9 \mathrm { H } 1$ variant. The H1-2009 assay specifically detects the pandemic 2009 H1N1 variant. Metaanalysis for Influenza A is outlined in the following table:

Interpretation of Influenza A Results   

<table><tr><td rowspan=1 colspan=1>Assay Final Result</td><td rowspan=1 colspan=1>FluA Assays(n=2)</td><td rowspan=1 colspan=1>H1</td><td rowspan=1 colspan=1>2009 H1</td><td rowspan=1 colspan=1>H3</td><td rowspan=1 colspan=1>RequiredFollow-up</td></tr><tr><td rowspan=1 colspan=1>Influenza A NotDetected</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>None</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1 andInfluenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Multiple infectionsare possible but rare,</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009and Influenza A H3</td><td rowspan=1 colspan=1>≥1 positive</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>retest to confirmresult</td></tr><tr><td rowspan=1 colspan=1>Influenza A (no subtypedetected)</td><td rowspan=1 colspan=1>2 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>See belowa</td></tr><tr><td rowspan=1 colspan=1>Influenza A equivocal</td><td rowspan=1 colspan=1>1 positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=4 colspan=1>Retest</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1/2009equivocal</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Any result</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td></tr></table>

a If both of the influenza A assays are positive, but none of the hemagglutinin subtyping assays are positive, then the interpretation is ‘Influenza A (no subtype detected).’ This result could occur when the titer of the virus in the specimen is low and not detected by the subtyping assays. This result could also indicate the

presence of a novel Influenza A strain. In both cases, the sample in question should be retested. If the retest provides a different result, then the sample is tested a third time to ensure the accuracy of the result. If the retest provides the same result, then the function of the RP pouches should be verified by testing with appropriate external control materials (known positive samples for Influenza A H1, Influenza A H3 and Influenza A $2 0 0 9 \mathrm { H } 1$ ) and a negative control reaction should be run to test for PCR-product contamination. If the FilmArray RP accurately identifies the external controls, contact local or state public health authorities for confirmatory testing.

In general, Influenza A is determined to be detected if at least one of the two FluA pan assays is positive and the corresponding subtyping assay is also positive. If neither of the pan assays is positive, but a subtyping assay is positive, then the result is considered equivocal for that specific subtype and the sample should be retested. If one of the pan assays is positive and no subtyping assay is positive, the result is Influenza A Equivocal and the specimen should be retested. If both FluA pan assays are positive and no subtyping assay is positive the result is reported as ‘Influenza A (no subtype detected). The product literature instructs the operator to retest the specimen. If the ‘no subtype detected’ result is repeated, the product literature directs the operator concerning appropriate follow-up testing and, if needed, notification of public health authorities.

If either the RNA Process Control or the PCR2 Control fails (negative assay result), all organisms are assigned an ‘Invalid’ test result. Invalid results are also assigned to all assays if the FilmArray instrument protocol or analysis does not complete, an Instrument Error is reported, or a Software Error is reported. All test specimens that receive Invalid results should be retested.

# M. Performance Characteristics (if/when applicable):

# 1. Analytical performance:

# a. Precision/Reproducibility:

A precision/reproducibility study panel of twelve specimens (see table below) was created by spiking simulated NPS sample matrix with known quantities of live RP organisms. The specimens in the panel were designed so that all $2 0 \mathrm { R P }$ organisms would be tested at three concentrations (3x LoD, 1x LoD, and 0.1x LoD). Several analytes were spiked into each specimen to simulate the possibility of co-infections with multiple pathogens at various levels. Human Rhinovirus (HRV) and Respiratory Syncytial Virus (RSV) were included in more specimens than the other analytes because they were the most common panel members detected in co-infected clinical NPS specimens during the prospective clinical evaluation performed by ITI.

# Panel of Specimens Evaluated for Precision/Reproducibility Testing with the FilmArray RP System

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>01 x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Human metapneumovirus</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A (H1N1) 2009 var</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>0.1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>1x LoD</td><td rowspan=1 colspan=1>3x LoD</td></tr></table>

Following spiking, the composition of each specimen pool was verified by testing with the FilmArray RP system. Pools that did not perform as expected were remade. Pools that performed as expected were aliquoted into several individual use vials, and frozen until the scheduled day of testing. Specimen aliquots were hand-delivered (Site A) or shipped (Site B) to external testing sites on dry ice and stored frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ at each testing site until the day of testing. All specimen aliquots were tested on the same day they were removed from the freezer. The test operators at test sites were blinded to the organisms and test levels in each specimen.

Reproducibility testing was performed at three test sites (Site A, Site B, and Site C) by multiple operators (4-5 per site) using multiple (26) lots of reagents, and multiple (20) FilmArray instruments between the three sites. Each specimen was tested 4 times per day (2 operators and 2 instruments per specimen) for 5 days at each site, giving a total of 20 test results from each test site (a minimum of 60 total test results for each analyte at each concentration). Negative analyte results were derived from any specimen that was not spiked with the analyte of interest. Every analyte was absent from at least six specimens, providing a minimum of 360 expected negative results for each analyte across all sites combined.

A total of 1,056 specimens were tested over the course of the study, and $9 2 . 7 \%$ (979/1056) of these specimens yielded valid results on the first attempt (i.e., first loaded pouch). Invalid results or no results were obtained for the remaining 77 $( 7 . 3 \% )$ specimens (no results for 31 specimens due to incomplete runs; 46 specimens were “invalid” due to Pouch Controls failures). The 31 incomplete runs consisted of Instrument Errors (14), Software Errors (9), and 8 runs that were either aborted by the operator or failed to complete. All samples for which the first pouch runs did not complete were retested using a new pouch/sample to obtain valid results. All samples for which the pouch controls failed were also retested using a new pouch/sample to obtain valid results.

Software and Instrument Performance Reproducibility Study   

<table><tr><td rowspan=2 colspan=1>TotalSpecimensTested</td><td rowspan=2 colspan=1>TotalCompletedTests on FirstPouch</td><td rowspan=2 colspan=1>Total TestsNotCompletedon FirstPouch</td><td rowspan=2 colspan=1>AbortedRun byUser</td><td rowspan=2 colspan=1>IncompleteRuns</td><td rowspan=2 colspan=1>SoftwareErrors</td><td rowspan=1 colspan=3>Instrument Error</td></tr><tr><td rowspan=1 colspan=1>ValveControllerrora</td><td rowspan=1 colspan=1>CameraCommunicationErorb</td><td rowspan=1 colspan=1>InstrumentError Total</td></tr><tr><td rowspan=2 colspan=1>1056</td><td rowspan=1 colspan=1>1025</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>2.9%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.9%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>1.3%</td></tr></table>

a Known failure mode in the instrument bladder system. A new bladder material has been implemented to reduce this error mode. b An additional 8 errors occurred; however, valid results were obtained by restarting the sample using the same pouch. c Known failure mode caused by a communication timing error between the FilmArray software and the driver of the instrument camera. The underlying cause of the errors was resolved with an update to the FilmArray software. This error mode was eliminated in software version 1.1. (Validation study of a total of 872 runs using software version 1.1demonstrated $0 \%$ error rate caused by this particular error mode.)

Analysis of Pouch Controls - Reproducibility Study   

<table><tr><td rowspan=2 colspan=1>TotalCompletedTests on FirstPouch</td><td rowspan=2 colspan=1>Total Runswith PouchControlsPassed</td><td rowspan=2 colspan=1>TotalRuns withPouchControlsFaileda</td><td rowspan=1 colspan=3>Runs with RNA Processing ControlsFailed</td><td rowspan=1 colspan=3>Runs with PCR2 Controls Failed</td></tr><tr><td rowspan=1 colspan=1>Total Runswith RNAProcessControlsFailed</td><td rowspan=1 colspan=1>Both RNAProcess andPCR2ControlsFailed</td><td rowspan=1 colspan=1>Only RNAPProcessControlsFailedb</td><td rowspan=1 colspan=1>Total Runswith PCR2ControlsFailed</td><td rowspan=1 colspan=1>Both RNAProcess andPCR2ControlsFailed</td><td rowspan=1 colspan=1>OnlyPCR2ControlsFailed</td></tr><tr><td rowspan=2 colspan=1>1025</td><td rowspan=1 colspan=1>979</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>95.5%</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>3.9%</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0%</td></tr></table>

a Includes 1 run that failed because the wrong protocol was selected. b Investigation into the higher than expected rate of RNA process controls revealed that the QC process used to control the amount of control template was inadequate. The QC process has been revised to better control the concentration of control template.

At each test site between 4 and 5 operators participated in the study, with at least 2 different operators testing samples each day. The number of errors experienced by each test site was similar, though the percentage was lower for Site C due to a higher number of runs performed. The distribution of errors between operators at test sites A and B was somewhat variable, but there did not appear to be a strong correlation between system errors and particular operators.

A summary of reproducibility study results by each analyte are provided below:

Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing of Single Assay Analytes   

<table><tr><td colspan="2" rowspan="1">Adenovirus Species C Serotype 1</td><td colspan="1" rowspan="1">#.Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpected Resut</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="4">Medium Positive(3x LoD)900 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">83.74</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">83.33 - 84.26</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">84.23</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">83.95 - 84.60</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">83.76</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">83.03 - 84.13</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">83.93</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">83.03 - 84.60</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)300 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">83.42</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">83.01 - 83.98</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">83.90</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">83.54 - 84.30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">83.62</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">83.05 - 84.64</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">83.64</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">83.01 - 84.64</td></tr><tr><td colspan="1" rowspan="4">High Negativeb(0.1x LoD)30 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">2/20</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">68.3% - 98.8%</td><td colspan="1" rowspan="1">83.33</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">83.02 - 83.76</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">4/20</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">56.3% - 94.3%</td><td colspan="1" rowspan="1">83.71</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">82.93 - 84.29</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">27.2% - 72.8%</td><td colspan="1" rowspan="1">83.26</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">82.61 - 83.86</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">44/60</td><td colspan="1" rowspan="1">16/60</td><td colspan="1" rowspan="1">73.3%</td><td colspan="1" rowspan="1">60.3% - 83.9%</td><td colspan="1" rowspan="1">83.43</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">82.61 - 84.29</td></tr><tr><td colspan="1" rowspan="2">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="2" rowspan="1">Adenovirus Species CSertype 1</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">Observed TmRange</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Bordetella pertussisA639</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%Agreementwithxpected Result</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanMTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)12,000 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.36</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>87.46 - 89.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.81</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>88.26 - 89.20</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.34</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>87.97 - 88.58</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>88.50</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>87.46 - 89.20</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)4,000 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.39</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.73 - 88.96</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.64</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>88.07 - 89.20</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>88.22</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.74 - 88.67</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>88.42</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>87.73 - 89.20</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1x LoD)400 CFU/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>56.3% - 94.3%</td><td rowspan=1 colspan=1>88.41</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>87.86 - 89.36</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=1>88.70</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>88.33 - 89.29</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=1>88.26</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>87.52 - 88.71</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>20/60</td><td rowspan=1 colspan=1>66.7%</td><td rowspan=1 colspan=1>53.3% - 78.3%</td><td rowspan=1 colspan=1>88.46</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>87.52 - 89.36</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Chlamydophila pneumoniaeP TTW183</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithxpectedDResulta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)9,000 copies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>79.65</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>78.94 - 79.99</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>80.27</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>79.88 - 80.82</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>79.73</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>79.42 - 80.09</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>79.92</td><td rowspan=1 colspan=1>0.45</td><td rowspan=1 colspan=1>78.94 - 80.82</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)3,000 copies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>79.63</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>79.03 - 80.09</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>80.15</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>79.68 - 80.62</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>79.61</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>78.84 - 80.09</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% - 100%</td><td rowspan=1 colspan=1>79.79</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>78.84 - 80.62</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1x LoD)300 copies/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=1>79.55</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.25 - 80.08</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>45.7% - 88.1%</td><td rowspan=1 colspan=1>80.02</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>79.66 - 80.72</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% - 72.8%</td><td rowspan=1 colspan=1>79.61</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>79.16 - 80.21</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>35/60</td><td rowspan=1 colspan=1>25/60</td><td rowspan=1 colspan=1>58.3%</td><td rowspan=1 colspan=1>44.9% - 70.9%</td><td rowspan=1 colspan=1>79.72</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>79.16 - 80.72</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td colspan="3" rowspan="1">Coronavirus HKU1Tye BClinical Specimen 6123</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedD Resulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTm  Range</td></tr><tr><td colspan="2" rowspan="1">Medium Positive</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">75.61</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">75.37 - 76.02</td></tr><tr><td colspan="2" rowspan="3">(3x LoD)5.7 x 106 RNAcopies/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="2" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">76.02</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">75.49</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">74.96 - 75.90</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">75.69</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">74.96 - 76.33</td></tr><tr><td colspan="2" rowspan="4">Low Positive(1x LoD)1.9 x 106 RNAcopies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">75.47</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">74.96 - 75.79</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">75.89</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">75.59 - 76.12</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">75.35</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">74.83 - 75.81</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">75.55</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">74.83 - 76.12</td></tr><tr><td colspan="2" rowspan="2">High Negativeb0.1x LoD)1.9 x 105RNAcopies/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">5/20</td><td colspan="1" rowspan="1">75.0%</td><td colspan="1" rowspan="1">50.9% - 91.3%</td><td colspan="1" rowspan="1">75.51</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">75.06 - 75.90</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">62.1% - 96.8%</td><td colspan="1" rowspan="1">75.85</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">75.26 - 76.23</td></tr><tr><td colspan="2" rowspan="1">Coronavirus HKU1Type BClinical Specimen 6123</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResult</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">Observed Tm Range</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">75.1% - 99.9%</td><td colspan="1" rowspan="1">75.33</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">74.98 - 75.59</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">51/60</td><td colspan="1" rowspan="1">9/60</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">73.4% - 92.9%</td><td colspan="1" rowspan="1">75.55</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">74.98 - 76.23</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Coronavirus NL63BEI Resources NR-470</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResuta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)15 TCID50/ml</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.21</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>79.64 - 80.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.55</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>79.99 - 81.03</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.04</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.67 - 80.62</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=2>80.29</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>79.64 - 81.03</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)5 TCID50/ml</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.08</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>79.57 - 80.42</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>80.40</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>79.98 - 80.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=2>79.88</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>79.36 - 80.40</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=2>80.12</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>79.14 - 80.89</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1x LoD)0.5 TCID50/ml</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>40.8% - 84.6%</td><td rowspan=1 colspan=2>80.08</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>79.21 - 80.82</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>14/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>70.0%</td><td rowspan=1 colspan=1>45.7% - 88.1%</td><td rowspan=1 colspan=2>80.36</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>79.98 - 80.91</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>10/20</td><td rowspan=1 colspan=1>50.0%</td><td rowspan=1 colspan=1>27.2% - 72.8%</td><td rowspan=1 colspan=2>79.91</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>79.24 - 80.30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>37/60</td><td rowspan=1 colspan=1>23/60</td><td rowspan=1 colspan=1>61.7%</td><td rowspan=1 colspan=1>48.2% - 73.9%</td><td rowspan=1 colspan=2>80.11</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>79.21 - 80.91</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=4></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td>Coronavirus OC43 ATCC VR-759</td><td># Positive</td><td># Negative</td><td>% Agreement with Expected Resulta</td><td>95% CI</td><td>Mean Tm</td><td>%CV Tm</td><td>Observed Tm Range</td></tr><tr><td colspan="2" rowspan="1">Coronavirus OC43ATCC VR-759</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpected Result</td><td colspan="1" rowspan="1">95% CI</td><td colspan="2" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="4">Medium Positive(3x LoD)1,800 TCID50/ml</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">80.98</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">80.53 - 81.36</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">81.39</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">81.04 - 81.82</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">81.20</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">80.60 - 81.55</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="2" rowspan="1">81.17</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">80.53 - 81.82</td></tr><tr><td colspan="1" rowspan="4">Low Positive1xxLoD)600 TCID50/ml</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">80.82</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">80.29 - 81.25</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">81.31</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">80.95 - 81.85</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="2" rowspan="1">81.07</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">80.40 - 81.58</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="2" rowspan="1">81.07</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">80.29 - 81.85</td></tr><tr><td colspan="1" rowspan="4">High Negative b0.1x LoD)60 TCID50/ml</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">18/20</td><td colspan="1" rowspan="1">2/20</td><td colspan="1" rowspan="1">90.0%</td><td colspan="1" rowspan="1">68.3% - 98.8%</td><td colspan="2" rowspan="1">80.86</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">80.49 - 81.25</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">45.7% - 88.1%</td><td colspan="2" rowspan="1">81.29</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">80.92 - 81.81</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">4/20</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">56.3% - 94.3%</td><td colspan="2" rowspan="1">81.04</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">80.11 - 81.45</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">48/60</td><td colspan="1" rowspan="1">12/60</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">67.7% - 89.2%</td><td colspan="2" rowspan="1">81.07</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">80.11 - 81.81</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="4" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">1/180</td><td colspan="1" rowspan="1">179/180</td><td colspan="1" rowspan="1">99.4%</td><td colspan="1" rowspan="1">96.9% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">1/540</td><td colspan="1" rowspan="1">539/540</td><td colspan="1" rowspan="1">99.8%</td><td colspan="1" rowspan="1">99.0% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td colspan="2" rowspan="1">Coronavirus 229EATCC VR-740</td><td colspan="1" rowspan="1">#PositiveResults</td><td colspan="1" rowspan="1">#NegativeResults</td><td colspan="1" rowspan="1">%AgreementwithExpected Results</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTmRange</td></tr><tr><td colspan="1" rowspan="4">Medium Positive3x LoD)12 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.20</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">80.81 - 81.47</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.80</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">81.35 - 82.18</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.35</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">81.03 - 82.13</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">81.37</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">80.81 - 82.18</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)4 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.19</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">80.83 - 81.66</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.62</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">81.12 - 82.08</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.09</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">80.30 - 81.56</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">81.31</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">80.30 - 82.08</td></tr><tr><td colspan="1" rowspan="4">High Negativeb(x LoD)0.4 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">6/20</td><td colspan="1" rowspan="1">70%</td><td colspan="1" rowspan="1">45.7% - 88.1%</td><td colspan="1" rowspan="1">81.06</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">80.72 - 81.43</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">35%</td><td colspan="1" rowspan="1">15.4% - 59.2%</td><td colspan="1" rowspan="1">81.54</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">81.15 - 82.08</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">55%</td><td colspan="1" rowspan="1">31.5% - 76.9%</td><td colspan="1" rowspan="1">81.17</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">80.70 - 81.68</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">32/60</td><td colspan="1" rowspan="1">28/60</td><td colspan="1" rowspan="1">53.3%</td><td colspan="1" rowspan="1">40.0% - 66.3%</td><td colspan="1" rowspan="1">81.25</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">80.70 - 82.08</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">Coronavirus 229EATCC VR-740</td><td colspan="1" rowspan="1">#PositiveResults</td><td colspan="1" rowspan="1">#NegativeResults</td><td colspan="1" rowspan="1">%AgreementwithExpectedResults</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTmRange</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="3"></td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Human MetapneumovirushMPV-16 (A1)</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1># Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResultsa</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean Tm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>Observed TmRange</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)6 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.61</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>77.06 - 77.90</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.07</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>77.67 - 78.61</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>77.45 - 78.11</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% - 100%</td><td rowspan=1 colspan=1>77.82</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>77.06 - 78.61</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)2 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.39</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>77.04 - 77.79</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.88</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>77.37 - 78.11</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.60</td><td rowspan=1 colspan=1>0.24</td><td rowspan=1 colspan=1>77.16 - 77.99</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>77.62</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>77.04 - 78.11</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1 x LoD)0.2 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% - 96.8%</td><td rowspan=1 colspan=1>77.34</td><td rowspan=1 colspan=1>0.20</td><td rowspan=1 colspan=1>77.05 - 77.59</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>77.76</td><td rowspan=1 colspan=1>0.22</td><td rowspan=1 colspan=1>77.06 - 78.11</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=1>77.37</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>76.74 - 77.79</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>48/60</td><td rowspan=1 colspan=1>12/60</td><td rowspan=1 colspan=1>80.0%</td><td rowspan=1 colspan=1>67.7% - 89.2%</td><td rowspan=1 colspan=1>77.50</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>76.74 - 78.11</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td colspan="2" rowspan="1">Influenza BB/FL/04/06</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="2">Medium Positive(3x LoD)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.47</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">79.88 - 80.93</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.88</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">80.30 - 81.30</td></tr><tr><td colspan="2" rowspan="1">Influenza BB/04/06</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedT Tm Range</td></tr><tr><td colspan="1" rowspan="2">180 TCID50/mL</td><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.36</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">79.78 - 80.80</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">80.56</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">79.78 - 81.30</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)60 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.44</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.00 - 80.92</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.79</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">80.40 - 81.33</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.34</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">79.77 - 80.81</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">80.51</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">79.77 - 81.33</td></tr><tr><td colspan="1" rowspan="4">High Negativeb.1x LoD)6 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">60.0%</td><td colspan="1" rowspan="1">36.1% - 80.9%</td><td colspan="1" rowspan="1">80.42</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">79.84 - 80.90</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">27.2% - 72.8%</td><td colspan="1" rowspan="1">80.78</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">80.40 - 81.17</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">8/20</td><td colspan="1" rowspan="1">12/20</td><td colspan="1" rowspan="1">40.0%</td><td colspan="1" rowspan="1">19.1% - 64.0%</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">79.79 - 80.69</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">30/60</td><td colspan="1" rowspan="1">30/60</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">36.8% - 63.2%</td><td colspan="1" rowspan="1">80.50</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">79.79 - 81.17</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td colspan="3" rowspan="1">Mycoplasma pneumoniaeType 1 - M129</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="2" rowspan="4">Medium Positivex LoD)90 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">77.42</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">77.04 - 77.71</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">77.94</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">77.59 - 78.30</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">77.83</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">77.44 - 78.12</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">77.75</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">77.04 - 78.30</td></tr><tr><td colspan="2" rowspan="4">Low Positive(1x LoD)30 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">75.1% - 99.9%</td><td colspan="1" rowspan="1">77.55</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">77.05 - 78.00</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">62.1% - 96.8%</td><td colspan="1" rowspan="1">77.96</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">77.36 - 78.37</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">77.68</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">76.65 - 78.10</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">56/60</td><td colspan="1" rowspan="1">4/60</td><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">83.8% - 98.2%</td><td colspan="1" rowspan="1">77.73</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">76.65 - 78.37</td></tr><tr><td colspan="2" rowspan="3">High Negativeb(01x LoD)3 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">5/20</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">25.0%</td><td colspan="1" rowspan="1">8.7% - 49.1%</td><td colspan="1" rowspan="1">77.58</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">76.72 - 78.01</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">4/20</td><td colspan="1" rowspan="1">16/20</td><td colspan="1" rowspan="1">20.0%</td><td colspan="1" rowspan="1">5.7% - 43.7%</td><td colspan="1" rowspan="1">78.01</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">77.67 - 78.45</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">11/20</td><td colspan="1" rowspan="1">9/20</td><td colspan="1" rowspan="1">55.0%</td><td colspan="1" rowspan="1">31.5% - 76.9%</td><td colspan="1" rowspan="1">77.76</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">77.04 - 78.09</td></tr><tr><td colspan="2" rowspan="1">Mycoplasma pneumoniaeType 1 - M129</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm gea</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">20/60</td><td colspan="1" rowspan="1">40/60</td><td colspan="1" rowspan="1">33.3%</td><td colspan="1" rowspan="1">21.7% - 46.7%</td><td colspan="1" rowspan="1">77.78</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">76.72 - 78.45</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Parainfluenza Virus 1Zeptometrix #08 1 001CN</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResulta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean Tm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)1,500 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.86</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>78.42 - 79.25</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>79.32</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>78.83 - 79.78</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td><td rowspan=1 colspan=1>78.50</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>78.02 - 78.87</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% - 100%</td><td rowspan=1 colspan=1>78.91</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>78.02 - 79.78</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)500 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.60</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>77.99 - 79.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.93</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>78.31 - 79.36</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.50</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>77.90 - 79.16</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>78.67</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>77.90 - 79.36</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1x LoOD)50 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>50.1% - 91.3%</td><td rowspan=1 colspan=1>78.54</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>78.10 - 78.94</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>75.0%</td><td rowspan=1 colspan=1>50.1% - 91.3%</td><td rowspan=1 colspan=1>78.94</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>78.52 - 79.36</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>13/20</td><td rowspan=1 colspan=1>7/20</td><td rowspan=1 colspan=1>65.0%</td><td rowspan=1 colspan=1>41.0% - 84.6%</td><td rowspan=1 colspan=1>78.41</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>77.87 - 79.02</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>43/60</td><td rowspan=1 colspan=1>17/60</td><td rowspan=1 colspan=1>71.7%</td><td rowspan=1 colspan=1>58.6% - 82.6%</td><td rowspan=1 colspan=1>78.61</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>77.79 - 79.36</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Parainfluenza Virus 2Zeptometrix #0810015CF</td><td rowspan=1 colspan=1># Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResulta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=3>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)30 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>83.63</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.01 - 84.39</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>84.06</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>83.44 - 84.79</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>83.88</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>83.13 - 84.28</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=3>83.85</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>83.01 - 84.79</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)10 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>83.56</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>82.94 - 84.08</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>84.00</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.52 - 84.63</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=3>83.79</td><td rowspan=1 colspan=1>0.32</td><td rowspan=1 colspan=1>82.92 - 84.25</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=3>83.78</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>82.92 - 84.63</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1 x LoD)1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=3>83.43</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>82.71 - 83.96</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>60.0%</td><td rowspan=1 colspan=1>36.1% - 80.9%</td><td rowspan=1 colspan=3>83.91</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.43 - 84.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=3>83.71</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>82.91 - 84.30</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>35/60</td><td rowspan=1 colspan=1>25/60</td><td rowspan=1 colspan=1>58.3%</td><td rowspan=1 colspan=1>44.9% - 70.9%</td><td rowspan=1 colspan=3>83.69</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>82.71 - 84.56</td></tr><tr><td rowspan=5 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=5 colspan=5></td></tr><tr><td rowspan=2 colspan=1>Site B</td><td rowspan=2 colspan=1>0/180</td><td rowspan=2 colspan=1>180/180</td><td rowspan=2 colspan=1>100.0%</td><td rowspan=2 colspan=1>98.0% - 100%</td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td colspan="2" rowspan="1">Parainfluenza Virus 3Zeptometrix #0810016CF</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm ngeRan</td></tr><tr><td colspan="1" rowspan="4">Medium Positive(3x LoD)30 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.17</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">80.71 - 81.86</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.48</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">81.03 - 81.89</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.94</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">80.63 - 81.37</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td><td colspan="1" rowspan="1">81.22</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">80.63 - 81.89</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)10 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">80.97</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">80.36 - 81.52</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td><td colspan="1" rowspan="1">81.35</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">80.93 - 81.79</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">3/20</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">62.1% - 96.8%</td><td colspan="1" rowspan="1">80.86</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">80.10 - 81.21</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">57/60</td><td colspan="1" rowspan="1">3/60</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">86.1% - 99.0%</td><td colspan="1" rowspan="1">81.08</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">80.10 - 81.79</td></tr><tr><td colspan="2" rowspan="1">Parainfluenza Virus 3Zeptometrix #0810016CF</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%AgreementwithExpectedResulta</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">%CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="4">High Negativeb(0.1x LoD)1 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">10/20</td><td colspan="1" rowspan="1">50.0%</td><td colspan="1" rowspan="1">27.2% - 72.8%</td><td colspan="1" rowspan="1">80.99</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">80.30 - 81.34</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">7/20</td><td colspan="1" rowspan="1">13/20</td><td colspan="1" rowspan="1">35.0%</td><td colspan="1" rowspan="1">15.4% - 59.2%</td><td colspan="1" rowspan="1">81.29</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">80.61 - 81.77</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">5/20</td><td colspan="1" rowspan="1">15/20</td><td colspan="1" rowspan="1">25.0%</td><td colspan="1" rowspan="1">8.7% - 49.1%</td><td colspan="1" rowspan="1">80.84</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">80.41 - 81.24</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">22/60</td><td colspan="1" rowspan="1">38/60</td><td colspan="1" rowspan="1">36.7%</td><td colspan="1" rowspan="1">24.6% - 50.1%</td><td colspan="1" rowspan="1">81.05</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">80.30 - 81.77</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td><td colspan="3" rowspan="4"></td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Parainfluenza Virus 4aZeptometrix #0810060CF</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResulta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>Observedm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)15,000TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.70</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>77.36 - 78.10</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>78.09</td><td rowspan=1 colspan=1>0.56</td><td rowspan=1 colspan=1>77.48 - 78.74</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.73</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>77.05 - 78.21</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>77.82</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>77.05 - 78.74</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)5,000TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.11</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>76.64 - 77.68</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.65</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>76.73 - 78.71</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td><td rowspan=1 colspan=1>77.23</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>76.81 - 78.20</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>77.33</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>76.64 - 78.71</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.x LoOD)500 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>16/20</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5.7% - 43.7%</td><td rowspan=1 colspan=1>77.07</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>76.63 - 77.58</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>5/20</td><td rowspan=1 colspan=1>15/20</td><td rowspan=1 colspan=1>25.0%</td><td rowspan=1 colspan=1>8.7% - 49.1%</td><td rowspan=1 colspan=1>77.59</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>77.05 - 78.00</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=1>77.24</td><td rowspan=1 colspan=1>0.30</td><td rowspan=1 colspan=1>76.62 - 77.84</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>20/60</td><td rowspan=1 colspan=1>40/60</td><td rowspan=1 colspan=1>33.3%</td><td rowspan=1 colspan=1>21.7% - 46.7%</td><td rowspan=1 colspan=1>77.29</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>76.62 - 78.00</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative.   
b “High Negative” samples are targeted to be positive $20 \%$ of the time.

<table><tr><td rowspan=1 colspan=2>Respiratory Syncytial VirusType A</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>%AgreementwithExpectedResulta</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>%CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)6 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.44</td><td rowspan=1 colspan=1>0.35</td><td rowspan=1 colspan=1>79.46 - 80.83</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.86</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>80.41 - 81.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td><td rowspan=1 colspan=1>80.39</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>80.08 - 80.91</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.8% - 100%</td><td rowspan=1 colspan=1>80.58</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>79.46 - 81.56</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)2 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>79.82</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>78.93 - 80.62</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>80.40</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>79.47 - 81.03</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>40/40</td><td rowspan=1 colspan=1>0/40</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>91.2% - 100%</td><td rowspan=1 colspan=1>80.13</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>79.13 - 80.79</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97.0% - 100%</td><td rowspan=1 colspan=1>80.10</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>78.93 - 81.03</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1 x LoD)0.2 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>2/20</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>68.3% - 98.8%</td><td rowspan=1 colspan=1>79.63</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>78.72 - 80.72</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>85.0%</td><td rowspan=1 colspan=1>62.1% - 96.8%</td><td rowspan=1 colspan=1>80.12</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>79.26 - 80.89</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>55.0%</td><td rowspan=1 colspan=1>31.5% - 76.9%</td><td rowspan=1 colspan=1>79.97</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>78.83 - 80.84</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>46/60</td><td rowspan=1 colspan=1>14/60</td><td rowspan=1 colspan=1>76.7%</td><td rowspan=1 colspan=1>64.0% - 86.6%</td><td rowspan=1 colspan=1>79.90</td><td rowspan=1 colspan=1>0.58</td><td rowspan=1 colspan=1>78.72 - 80.89</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97% - 100%</td><td rowspan=4 colspan=3></td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/120</td><td rowspan=1 colspan=1>120/120</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>97% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/360</td><td rowspan=1 colspan=1>360/360</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.0% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

# Summary of Positive Agreement, Negative Agreement, and Tm Results from Reproducibility Testing of Multi- Assay Analytes

Reproducibility Agreement Summary for Enterovirus (Human Rhinovirus/Enterovirus)   

<table><tr><td rowspan=1 colspan=2>EnterovirusEchovirus 6 (Species B)</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>% Agreement withExpected Resulta</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)90,000 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)30,000 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1 x LoD)3,000 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.8% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

Reproducibility Tm Summary (by assay) for Enterovirus   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">EnterovirusEchovirus 6 (Species B)</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="11">Entero 1</td><td colspan="1" rowspan="4">Medium Positive(3x LoD)90,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.12</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">86.79 - 87.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.51</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.99 - 88.05</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.98</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">86.37 - 87.64</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.18</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">86.37 - 88.05</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)30,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.00</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">86.17 - 87.64</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.36</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.80 - 87.85</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.81</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.15 - 87.41</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.05</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">86.15 - 87.85</td></tr><tr><td colspan="1" rowspan="3">High Negative(0.1x LoD)3,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.89</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.06 - 87.40</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.34</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">86.67 - 87.86</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.67</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">85.97 - 87.30</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.96</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">85.97 - 87.86</td></tr><tr><td colspan="1" rowspan="12">Entero 2</td><td colspan="1" rowspan="4">Medium Positive(3x LoD)90,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">87.09</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">86.68 - 87.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.47</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.82 - 88.00</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.93</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">86.16 - 87.64</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.14</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">86.16 - 88.00</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD30,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.98</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">86.28 - 87.53</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.34</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">86.89 - 87.82</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.77</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.05 - 87.52</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">87.02</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">86.05 - 87.82</td></tr><tr><td colspan="1" rowspan="4">High Negative(0.1x LoD)3,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">86.86</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">86.17 - 87.54</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">87.26</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">86.59 - 87.94</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">86.65</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">86.27 - 87.20</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">86.92</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">86.17 - 87.94</td></tr><tr><td colspan="1" rowspan="12">HRV4</td><td colspan="1" rowspan="4">Medium Positive(3x LoD)90,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.70</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">85.21 - 86.18</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">86.19</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">85.35 - 86.77</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.59</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">84.91 - 86.19</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.81</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">84.91 - 86.77</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)30,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.45</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.81 - 86.06</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.87</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">85.44 - 86.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.32</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">84.69 - 86.26</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.54</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">84.69 - 86.36</td></tr><tr><td colspan="1" rowspan="4">High Negative(0.1x LoD)3,000 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.37</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.80 - 86.04</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.82</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">85.43 - 86.23</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.22</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">84.82 - 85.58</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.46</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">84.80 - 86.23</td></tr></table>

Reproducibility Agreement Summary for Rhinovirus (Human Rhinovirus/Enterovirus)   

<table><tr><td colspan="2" rowspan="1">Human RhinovirusA1</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">%Agreement withExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="4">Medium Positive(3x LoD)3 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)1 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="4">High Negativeb(0.1 x LoD)0.1 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">0/40</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">40/40</td><td colspan="1" rowspan="1">0/40</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">91.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">32/40</td><td colspan="1" rowspan="1">8/40</td><td colspan="1" rowspan="1">80.0%</td><td colspan="1" rowspan="1">64.4% - 91.0%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">112/120</td><td colspan="1" rowspan="1">8/120</td><td colspan="1" rowspan="1">93.3%</td><td colspan="1" rowspan="1">87.3% - 97.1%</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">97.8% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

Reproducibility Tm Summary (by assay) for Rhinovirus   

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=2>Human RhinovirusA1</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=12 colspan=1>HRV1</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>83.25 - 85.09</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.06</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.43 - 84.56</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.68</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>83.22 - 84.09</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.84</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.22 - 85.09</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.71</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>83.07 - 84.35</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.07</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.44 - 84.66</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.93</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.24 - 84.71</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.07 - 84.71</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.56</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>83.02 - 84.26</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.91</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.22 - 84.52</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.04 - 84.48</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.74</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.02 - 84.52</td></tr><tr><td rowspan=12 colspan=1>HRV2</td><td rowspan=4 colspan=1>Medium Positive3x LoD3 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.33</td><td rowspan=1 colspan=1>0.48</td><td rowspan=1 colspan=1>82.55 - 84.67</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.65</td><td rowspan=1 colspan=1>0.29</td><td rowspan=1 colspan=1>83.11 - 84.17</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.30</td><td rowspan=1 colspan=1>0.28</td><td rowspan=1 colspan=1>82.70 - 83.77</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.42</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>82.55 - 84.67</td></tr><tr><td rowspan=4 colspan=1>Low Positive1x LoD1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.28</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>82.57 - 83.86</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.66</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>83.02 - 84.37</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.52</td><td rowspan=1 colspan=1>0.39</td><td rowspan=1 colspan=1>82.82 - 84.29</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.48</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>82.57 - 84.37</td></tr><tr><td rowspan=4 colspan=1>High Negative0.1x LoD0.1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.17</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>82.42 - 83.88</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.58</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>82.80 - 84.31</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.37</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>82.51 - 83.92</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.37</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>82.42 - 84.31</td></tr><tr><td rowspan=1 colspan=1>HRV3</td><td rowspan=1 colspan=1>Medium Positive</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.73</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>81.99 - 83.94</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=3>Human RhinovirusA1</td><td rowspan=1 colspan=1>MeanTm</td><td rowspan=1 colspan=1>% CVTm</td><td rowspan=1 colspan=1>ObservedTm Range</td></tr><tr><td rowspan=11 colspan=1></td><td rowspan=3 colspan=2>3x LoD3 TCID50/mL</td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.25</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>82.49 - 83.88</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.89</td><td rowspan=1 colspan=1>0.43</td><td rowspan=1 colspan=1>82.10 - 83.88</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.96</td><td rowspan=1 colspan=1>0.51</td><td rowspan=1 colspan=1>81.99 - 83.94</td></tr><tr><td rowspan=4 colspan=2>Low Positive1x LoD1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.67</td><td rowspan=1 colspan=1>0.49</td><td rowspan=1 colspan=1>81.76 - 83.65</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.24</td><td rowspan=1 colspan=1>0.44</td><td rowspan=1 colspan=1>82.40 - 84.08</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.07</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>82.13 - 83.76</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.98</td><td rowspan=1 colspan=1>0.54</td><td rowspan=1 colspan=1>81.76 - 84.08</td></tr><tr><td rowspan=4 colspan=2>High Negative0.1x LoD0.1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>82.68</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>81.78 - 83.67</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.21</td><td rowspan=1 colspan=1>0.46</td><td rowspan=1 colspan=1>82.18 - 84.37</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>82.97</td><td rowspan=1 colspan=1>0.42</td><td rowspan=1 colspan=1>81.85 - 83.64</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>82.96</td><td rowspan=1 colspan=1>0.52</td><td rowspan=1 colspan=1>81.78 - 84.37</td></tr><tr><td rowspan=12 colspan=1>HRV4</td><td rowspan=4 colspan=2>Medium Positive3x LoD3 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.81</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.28 - 84.98</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.14</td><td rowspan=1 colspan=1>0.34</td><td rowspan=1 colspan=1>83.43 - 84.83</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.80</td><td rowspan=1 colspan=1>0.25</td><td rowspan=1 colspan=1>83.44 - 84.20</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.90</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.28 - 84.98</td></tr><tr><td rowspan=4 colspan=2>Low Positive1x LoD1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.79</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.18 - 84.45</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>84.08</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.55 - 84.61</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>84.04</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>83.35 - 84.93</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.94</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>83.18 - 84.93</td></tr><tr><td rowspan=4 colspan=2>High Negative0.1x LoD0.1 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>83.58</td><td rowspan=1 colspan=1>0.26</td><td rowspan=1 colspan=1>83.11 - 84.05</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>83.88</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>83.34 - 84.49</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>83.86</td><td rowspan=1 colspan=1>0.27</td><td rowspan=1 colspan=1>83.24 - 84.58</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>83.76</td><td rowspan=1 colspan=1>0.33</td><td rowspan=1 colspan=1>83.11 - 84.58</td></tr></table>

Reproducibility Agreement Summary for Influenza A/H1   

<table><tr><td colspan="3" rowspan="1">Influenza A H1N1A/Brisbane/59/07</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Equivocal</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">% Agreement withExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="2" rowspan="4">MediumPositive (3xLoD600 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2%−-100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">94.0% - 100%</td></tr><tr><td colspan="2" rowspan="4">Low Positive(1x LoD)200 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">95.0%</td><td colspan="1" rowspan="1">75.1% - 99.9%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">2/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">62.1% - 96.8%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">56/60</td><td colspan="1" rowspan="1">3/60</td><td colspan="1" rowspan="1">1/60</td><td colspan="1" rowspan="1">93%</td><td colspan="1" rowspan="1">83.8%  98.2%</td></tr><tr><td colspan="2" rowspan="1">High Negativeb</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">0/20</td><td colspan="1" rowspan="1">100%</td><td colspan="1" rowspan="1">83.2% - 100%</td></tr><tr><td colspan="2" rowspan="2">(0.1 x LoD)20 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">17/20</td><td colspan="1" rowspan="1">2/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">85.0%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">14/20</td><td colspan="1" rowspan="1">5/20</td><td colspan="1" rowspan="1">1/20</td><td colspan="1" rowspan="1">70.0%</td><td colspan="1" rowspan="1">45.7% - 88.1%</td></tr><tr><td colspan="2" rowspan="1">Influenza A H1N1A/Brisbane/59/07</td><td colspan="1" rowspan="1">#Positive</td><td colspan="1" rowspan="1">#Equivocal</td><td colspan="1" rowspan="1">#Negative</td><td colspan="1" rowspan="1">% Agreement withExpected Resulta</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">51/60</td><td colspan="1" rowspan="1">7/60</td><td colspan="1" rowspan="1">2/60</td><td colspan="1" rowspan="1">85.0%</td><td colspan="1" rowspan="1">73.4% - 92.9%</td></tr><tr><td colspan="1" rowspan="4">Negative</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">0/180</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">98.0% - 100%</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">0/540</td><td colspan="1" rowspan="1">540/540</td><td colspan="1" rowspan="1">100.0%</td><td colspan="1" rowspan="1">99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

Reproducibility Tm Summary (by assay) for Influenza A/H1   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">Influenza A H1N1A/Brisbane/59/07</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CV Tm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="12">FluApan1</td><td colspan="1" rowspan="4">Moderate Positive3x LoD600 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.27 - 85.12</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.78 - 85.56</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.64</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">83.76 - 85.23</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.76 - 85.56</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD200 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">84.17 - 85.31</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">84.59 - 85.75</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.16 - 85.18</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">84.16 - 85.75</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD20 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.27</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">83.85 - 84.81</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">84.29 - 85.32</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.46</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">83.89 - 85.48</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.48</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.85 - 85.48</td></tr><tr><td colspan="1" rowspan="12">FluApan2</td><td colspan="1" rowspan="4">Moderate Positive3x LoD600 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.42</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">79.78 - 80.63</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.85</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.39 - 81.26</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.31</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">79.89 - 80.72</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.52</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">79.78 - 81.26</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD200 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.36</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">79.99 - 80.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.80</td><td colspan="1" rowspan="1">0.21</td><td colspan="1" rowspan="1">80.42 - 81.15</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.52</td><td colspan="1" rowspan="1">0.22</td><td colspan="1" rowspan="1">80.19 - 80.89</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.57</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">79.99 - 81.15</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD20 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.91</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">79.15 - 80.41</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.49</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">79.67 - 80.83</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.10</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">79.56 - 80.73</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.17</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">79.15 - 80.83</td></tr><tr><td colspan="1" rowspan="4">FluAH1-pan</td><td colspan="1" rowspan="4">Moderate Positive3x LoD600 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.79</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">78.31 - 79.25</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">78.30 - 79.57</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.76</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">77.79 - 79.25</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.91</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">77.67 - 79.57</td></tr><tr><td colspan="1" rowspan="8"></td><td colspan="1" rowspan="4">Low Positive1x LoD200 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.77</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">78.42 - 79.34</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">78.72 - 79.67</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.80</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">78.30 - 79.26</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.93</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">78.30 - 79.67</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD20 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">77.65</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">77.15 - 78.21</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.18</td><td colspan="1" rowspan="1">0.45</td><td colspan="1" rowspan="1">77.47 - 79.26</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">77.93</td><td colspan="1" rowspan="1">0.49</td><td colspan="1" rowspan="1">77.43 - 79.04</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">77.92</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">77.15 - 79.26</td></tr></table>

Reproducibility Agreement Summary for Influenza A/2009 H1   

<table><tr><td rowspan=1 colspan=2>Influenza A 2009 H1N1 A/SwineNY/03/2009</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>#Equivocal</td><td rowspan=1 colspan=1>#Negative</td><td rowspan=1 colspan=1>% Agreementwith ExpectedResulta</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)300 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)100 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negativeb(0.1 x LoD)10 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>1/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>95.0%</td><td rowspan=1 colspan=1>75.1% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>59/60</td><td rowspan=1 colspan=1>1/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>91.1% - 100%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3%  100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \%$ of the time.

Reproducibility Tm Summary (by assay) for Influenza A/2009 H1   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">Influenza A 2009 H1N1A/Swine NY/03/2009</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CV Tm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="3">FluApan1</td><td colspan="1" rowspan="3">Moderate Positive3x LoD</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.67</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.27 - 85.12</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.10</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">84.78 - 85.56</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.64</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">83.76 - 85.23</td></tr><tr><td colspan="1" rowspan="9"></td><td colspan="1" rowspan="1">300 TCID50/mL</td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.80</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.76 - 85.56</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.57</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">84.17 - 85.31</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">84.59 - 85.75</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.68</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.16 - 85.18</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">84.16 - 85.75</td></tr><tr><td colspan="1" rowspan="4">High Negative0.x LoD10 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.27</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">83.85 - 84.81</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">84.75</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">84.29 - 85.32</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.46</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">83.89 - 85.48</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">84.48</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">83.85 - 85.48</td></tr><tr><td colspan="1" rowspan="12">FluApan2</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">80.31 - 80.83</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">81.01</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">80.70 - 81.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.69</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">80.19 -81.02</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.81</td><td colspan="1" rowspan="1">0.32</td><td colspan="1" rowspan="1">80.19 - 81.36</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.39</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">79.87 - 80.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.91</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.20 - 81.24</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.30</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">80.05 - 80.61</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.62</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">79.87 - 81.24</td></tr><tr><td colspan="1" rowspan="4">High Negative0.x LoD10 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">80.43</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.08 - 81.03</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">80.72</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">80.11 - 81.14</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">80.41</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">79.86 - 80.82</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">80.54</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">79.86 - 81.14</td></tr><tr><td colspan="1" rowspan="12">FluAH1 pan</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.87</td><td colspan="1" rowspan="1">0.40</td><td colspan="1" rowspan="1">78.20 - 79.56</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.44</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">78.94 - 79.99</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.62</td><td colspan="1" rowspan="1">0.43</td><td colspan="1" rowspan="1">77.92 - 79.44</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.97</td><td colspan="1" rowspan="1">0.58</td><td colspan="1" rowspan="1">77.92 - 79.99</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.37</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">77.89 - 79.24</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.90</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">78.21 - 79.76</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.16</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">77.83 - 78.69</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.47</td><td colspan="1" rowspan="1">0.51</td><td colspan="1" rowspan="1">77.78 - 79.76</td></tr><tr><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.34</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">77.90 - 79.37</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">78.83</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">77.99 - 79.68</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.08</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">77.68 - 78.51</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.40</td><td colspan="1" rowspan="1">0.53</td><td colspan="1" rowspan="1">77.68 - 79.68</td></tr><tr><td colspan="1" rowspan="8">FluA H1-2009</td><td colspan="1" rowspan="4">Moderate Positive3x LoD300 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.73</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.31 - 79.14</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.20</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.84 - 79.67</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.54</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">77.89 - 78.92</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.81</td><td colspan="1" rowspan="1">0.44</td><td colspan="1" rowspan="1">77.89 - 79.67</td></tr><tr><td colspan="1" rowspan="4">Low Positive1x LoD100 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.64</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">78.10 - 79.14</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.03</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">78.53 - 79.48</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.49</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">78.12 - 78.82</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.71</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">78.10 - 79.48</td></tr><tr><td colspan="1" rowspan="4"></td><td colspan="1" rowspan="4">High Negative0.1x LoD10 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">78.60</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">77.90 - 79.04</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.00</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">78.52 - 79.36</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">78.52</td><td colspan="1" rowspan="1">0.28</td><td colspan="1" rowspan="1">78.10 - 78.93</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">78.70</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">77.90 - 79.36</td></tr></table>

Reproducibility Agreement Summary for Influenza A/H3   

<table><tr><td rowspan=1 colspan=2>Influenza A H3N2A/Wisconsin/67/2005</td><td rowspan=1 colspan=1># Positive</td><td rowspan=1 colspan=1># Equivocal</td><td rowspan=1 colspan=1># Negative</td><td rowspan=1 colspan=1>% Agreement withExpected Resulta</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=4 colspan=1>Medium Positive(3x LoD)15 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>Low Positive(1x LoD)5 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>0/20</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.2%−-100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>60/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>0/60</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.0% - 100%</td></tr><tr><td rowspan=4 colspan=1>High Negativeb0.1x LoD0.5 TCID50/mL</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>11/20</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>3.2% - 37.9%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>12/20</td><td rowspan=1 colspan=1>4/20</td><td rowspan=1 colspan=1>20.0%</td><td rowspan=1 colspan=1>5.7% - 43.7%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>3/20</td><td rowspan=1 colspan=1>8/20</td><td rowspan=1 colspan=1>9/20</td><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>3.2% - 37.9%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>10/60</td><td rowspan=1 colspan=1>31/60</td><td rowspan=1 colspan=1>19/60</td><td rowspan=1 colspan=1>16.7%</td><td rowspan=1 colspan=1>8.3% - 28.5%</td></tr><tr><td rowspan=4 colspan=1>Negative</td><td rowspan=1 colspan=1>Site A</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site B</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>Site C</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>0/180</td><td rowspan=1 colspan=1>180/180</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>98.0% - 100%</td></tr><tr><td rowspan=1 colspan=1>All Sites</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>0/540</td><td rowspan=1 colspan=1>540/540</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.3% - 100%</td></tr></table>

a Expected results for the “Medium Positive”, the “Low Positive”, and the “High Negative” panel members are positive. Expected result for the “Negative” panel member is negative. b “High Negative” samples are targeted to be positive $20 \text{‰}$ of the time

Reproducibility Tm Summary (by assay) for Influenza A/H3   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="2" rowspan="1">Influenza A H3N2A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">MeanTm</td><td colspan="1" rowspan="1">% CVTm</td><td colspan="1" rowspan="1">ObservedTm Range</td></tr><tr><td colspan="1" rowspan="7">FluApan1</td><td colspan="1" rowspan="4">Moderate Positive(3x LoD)15 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">85.33</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.79 - 85.73</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.48</td><td colspan="1" rowspan="1">0.42</td><td colspan="1" rowspan="1">84.91 - 86.27</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">85.06</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">84.50 - 85.40</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.36</td><td colspan="1" rowspan="1">0.38</td><td colspan="1" rowspan="1">84.50 - 86.27</td></tr><tr><td colspan="1" rowspan="3">Low Positive(1x LoD)</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.95</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">84.19 - 86.03</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.31</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">84.79 - 86.02</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.83</td><td colspan="1" rowspan="1">0.26</td><td colspan="1" rowspan="1">84.37 - 85.25</td></tr><tr><td colspan="1" rowspan="5">−</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.03</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">84.19 - 86.03</td></tr><tr><td colspan="1" rowspan="4">High Negative(0.1x LoD)0.5 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">84.91</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">84.22 - 85.60</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">85.24</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">84.78 - 85.77</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">84.86</td><td colspan="1" rowspan="1">0.27</td><td colspan="1" rowspan="1">84.48 - 85.33</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">85.01</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">84.22 - 85.77</td></tr><tr><td colspan="1" rowspan="12">FluApan2</td><td colspan="1" rowspan="4">Moderate Positive(3x LoD)15 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.75</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">79.51 - 79.99</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.94</td><td colspan="1" rowspan="1">0.30</td><td colspan="1" rowspan="1">79.40 - 80.37</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.60</td><td colspan="1" rowspan="1">0.24</td><td colspan="1" rowspan="1">79.14 - 79.86</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">79.81</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">79.14 - 80.37</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)5 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.42</td><td colspan="1" rowspan="1">0.31</td><td colspan="1" rowspan="1">79.00 - 80.30</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.69</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">79.14 - 80.29</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.25</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">78.82 - 79.59</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">79.45</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">78.82 - 80.30</td></tr><tr><td colspan="1" rowspan="4">High Negative(0.1x LoD)0.5 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">79.22</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">78.64 - 79.75</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">79.59</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">79.05 - 79.99</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">79.24</td><td colspan="1" rowspan="1">0.20</td><td colspan="1" rowspan="1">78.84 - 79.54</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">79.35</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">78.64 - 79.99</td></tr><tr><td colspan="1" rowspan="12">FluA H3</td><td colspan="1" rowspan="4">Moderate Positive(3x LoD)15 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.58</td><td colspan="1" rowspan="1">0.35</td><td colspan="1" rowspan="1">81.87 - 83.01</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.89</td><td colspan="1" rowspan="1">0.19</td><td colspan="1" rowspan="1">82.60 - 83.11</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.44</td><td colspan="1" rowspan="1">0.33</td><td colspan="1" rowspan="1">81.98 - 82.81</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">82.62</td><td colspan="1" rowspan="1">0.39</td><td colspan="1" rowspan="1">81.87 - 83.11</td></tr><tr><td colspan="1" rowspan="4">Low Positive(1x LoD)5 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.32</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">81.88 - 82.69</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.69</td><td colspan="1" rowspan="1">0.34</td><td colspan="1" rowspan="1">82.16 - 83.32</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.21</td><td colspan="1" rowspan="1">0.25</td><td colspan="1" rowspan="1">81.76 - 82.73</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">82.39</td><td colspan="1" rowspan="1">0.41</td><td colspan="1" rowspan="1">81.67 - 83.32</td></tr><tr><td colspan="1" rowspan="4">High Negative(0.1x LoD)0.5 TCID50/mL</td><td colspan="1" rowspan="1">Site A</td><td colspan="1" rowspan="1">82.28</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">81.71 - 82.91</td></tr><tr><td colspan="1" rowspan="1">Site B</td><td colspan="1" rowspan="1">82.61</td><td colspan="1" rowspan="1">0.29</td><td colspan="1" rowspan="1">82.17 - 83.05</td></tr><tr><td colspan="1" rowspan="1">Site C</td><td colspan="1" rowspan="1">82.20</td><td colspan="1" rowspan="1">0.23</td><td colspan="1" rowspan="1">81.87 - 82.57</td></tr><tr><td colspan="1" rowspan="1">All Sites</td><td colspan="1" rowspan="1">82.37</td><td colspan="1" rowspan="1">0.36</td><td colspan="1" rowspan="1">81.71 - 83.05</td></tr></table>

# b. Precision/(Repeatability)

The repeatability of the FilmArray RP System results was evaluated by repeated testing of the same 12 specimens. The in-house precision testing was performed at ITI (Site C) over the course of 12 testing days for a total of 48 test results per specimen. On each day all 12 specimens were tested 4 times by two operators on two FilmArray instruments. Results from the first 5 days of testing are included as the Reproducibility results for Site C. All operators were instructed to retest any samples that produced Invalid test results (failed controls or incomplete runs). Valid retest results were recorded as the final result.

Summary of Repeatability Results   

<table><tr><td rowspan=2 colspan=1>Spiked Organism</td><td rowspan=1 colspan=2>Moderate Positive(3x LoD)</td><td rowspan=1 colspan=2>Low Positive(1x LoD)</td><td rowspan=1 colspan=2>High Negative(0.1x LoD)</td></tr><tr><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td><td rowspan=1 colspan=1># Positive /Total</td><td rowspan=1 colspan=1>% PositiveResults</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>34/48</td><td rowspan=1 colspan=1>70.8%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>46/48</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>18/48</td><td rowspan=1 colspan=1>37.5%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>44/48</td><td rowspan=1 colspan=1>91.7%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>33/48</td><td rowspan=1 colspan=1>68.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19/48</td><td rowspan=1 colspan=1>39.6%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>83/96</td><td rowspan=1 colspan=1>86.5%</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>43/48a</td><td rowspan=1 colspan=1>89.6%b</td><td rowspan=1 colspan=1>39/48b</td><td rowspan=1 colspan=1>81.3%b</td></tr><tr><td rowspan=1 colspan=1>Influenza A/2009 H1N1</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48c</td><td rowspan=1 colspan=1>97.9%c</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>7/48d</td><td rowspan=1 colspan=1>14.6%d</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>25/48</td><td rowspan=1 colspan=1>52.1%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>19/48</td><td rowspan=1 colspan=1>39.6%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>47/48</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>27/48</td><td rowspan=1 colspan=1>56.3%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>41/48</td><td rowspan=1 colspan=1>85.4%</td><td rowspan=1 colspan=1>14/48</td><td rowspan=1 colspan=1>29.2%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>48/48</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>26/48</td><td rowspan=1 colspan=1>54.2%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>144/144</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>96/96</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>32/48</td><td rowspan=1 colspan=1>66.7%</td></tr><tr><td rowspan=1 colspan=1>% positive (all analytes)per test level</td><td rowspan=1 colspan=2>1102/110499.8%</td><td rowspan=1 colspan=2>942/96098.1%</td><td rowspan=1 colspan=2>599/96062.4%</td></tr></table>

a The five (5) non-positive results for Influenza A/H1 at LoD include: (1) Negative, (1) Influenza A/H1 equivocal and (3) Influenza A equivocal results. b The nine (9) non-positive results for Influenza $\mathrm { A } / \mathrm { H } 1$ at $0 . 1 \mathbf { x }$ LoD include: (1) Negative, (3) Influenza A (no subtype detected), (1) Influenza $\mathrm { A } / \mathrm { H } 1$ equivocal, and (4) Influenza A equivocal results. c One (1) equivocal Influenza A/2009 H1N1 result at the 0.1x LoD test level. d The 42 non-positive results for Influenza $\mathrm { A } / \mathrm { H } 3$ at $0 . 1 \mathrm { { x } }$ LoD include: (18) Negative, (2) Influenza A (no subtype detected) (15) Influenza A H3 equivocal, and (7) Influenza A equivocal results.

# c. Linearity/assay reportable range:

Not applicable for a qualitative assay

# d. Traceability, Stability, Expected values (controls, calibrators, or methods):

i. Assay Controls

The FilmArray Respiratory Panel (RP) pouch contains two internal control assays:

(1) The RNA Process Control targets an mRNA of the yeast, Schizosaccharomyces pombe. During FilmArray RP pouch manufacture, whole yeast are freeze-dried into the sample injection port of each pouch. When the test specimen is loaded into the pouch, S. pombe is rehydrated and enters the pouch with the specimen. The yeast nucleic acid is extracted, purified and tested simultaneously with nucleic acids from the patient specimen. A positive result for the processing control indicates that all steps in the process (nucleic acid extraction, reverse transcription, PCR, melt, detection, and analysis) are functioning properly.

(2) The second stage PCR (PCR2) control assay detects a synthetic DNA template that is dried into triplicate wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

The RNA Process Control and the PCR2 Control assays are used to Pass or Fail each FilmArray RP pouch run. This combination of control assays monitors each of the critical mechanical and chemical processes that occur in a pouch run, while limiting the possibility of random control assay failures that could contribute to unnecessary pouch failures.

External controls are not provided with the FilmArray RP System. External control mixes were provided to the clinical study sites for daily testing during the sponsor’s prospective clinical study. These are described in the discussion of the clinical trial on page 114 below; however the following is present in the product package insert: “Good laboratory practice recommends running external positive and negative controls regularly. Use viral transport medium as the external Negative Control, and previously characterized positive samples or negative samples spiked with well characterized target organisms as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.”

# ii. Specimen Stability

An analytical study was performed to establish the recommended transport and storage conditions for nasopharyngeal swab (NPS) specimens that will be tested using the Film Array Respiratory Panel System.

For the specimen transport and storage evaluation, both positive and negative specimens were assessed. Positive specimens consisted of simulated NPS (cultured human epithelial cells in VTM) spiked with mixes of targeted organisms at 5x the limit of detection (LoD) Negative data were collected from specimens that had been spiked with a different organism mix, such that results for the organism(s) of interest were expected to be negative. Ten independent specimens were prepared for each organism mix and tested “fresh” or without storage (0 Hour). Additional aliquots were made and two aliquots of 10 specimens per mix were stored at each of the conditions described below. Duplicate aliquots were stored for each specimen to allow for retesting if needed. For each RP organism, 10 positive and 30 negative specimens were evaluated at each storage condition. At least 9/10 positive results were required for each organism at each storage condition. In addition to positive and negative results, the average crossing point $\textstyle ( \mathbf { C p } )$ of amplification curves and melting temperature (Tm) of melt curves were reviewed for each assay at each storage condition. A shift in Tm or Cp from one storage condition to another could indicate that the integrity of the specimen was affected by storage.

Composition of Organism Mixes Spiked into Simulated NPS Specimens for Specimen Transport and Storage Study   

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>1,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>4 TCID50/mL</td><td rowspan=1 colspan=1>20 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=1 colspan=1>1,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50 /mL</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>30 TCID50/mL</td><td rowspan=1 colspan=1>150 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.9 x 106genome copies/mL</td><td rowspan=1 colspan=1>9.5 x 106genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>150,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>5,000 TCID50/mL</td><td rowspan=1 colspan=1>25,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>4,000 genome copies/mL</td><td rowspan=1 colspan=1>20,000 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>4,000 CFU/mL</td><td rowspan=1 colspan=1>20,000 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 4</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>3,000 genome copies/mL</td><td rowspan=1 colspan=1>15,000 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>600 TCID50/mL</td><td rowspan=1 colspan=1>3,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>500 TCID50/mL</td><td rowspan=1 colspan=1>2,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr></table>

Storage Conditions Evaluated for Simulated NPS Specimens   

<table><tr><td rowspan=1 colspan=2>Storage Conditions</td></tr><tr><td rowspan=1 colspan=1>0h</td><td rowspan=1 colspan=1>0 Hour, no storage</td></tr><tr><td rowspan=1 colspan=1>4h</td><td rowspan=1 colspan=1>4 Hours, room temperature (18-30°C)</td></tr><tr><td rowspan=1 colspan=1>3d</td><td rowspan=1 colspan=1>3 Days, refrigerated (2-8°C)</td></tr><tr><td rowspan=1 colspan=1>30d</td><td rowspan=1 colspan=1>30 Days, frozen (&lt; -15°C)</td></tr></table>

False negative results by organism and storage condition are presented in the table below:

False Negative Results by Organism and Storage Condition   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=4>False Negative Results</td></tr><tr><td rowspan=1 colspan=1>0 Hour</td><td rowspan=1 colspan=1>4 Hour (18° - 30C)</td><td rowspan=1 colspan=1>3 Days (2 ° - 8C)</td><td rowspan=1 colspan=1>30 Days (&lt; - 15C )</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>1/10</td><td rowspan=1 colspan=1>1/10</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td><td rowspan=1 colspan=1>0/10</td></tr></table>

In addition, the average crossing point $\displaystyle ( \mathrm { C p } )$ of amplification curves and melting temperature (Tm) of melt curves were also reviewed for each assay at each storage condition. The difference in average $\mathrm { C p } \left( \Delta \mathrm { C p } \right)$ and average Tm ( $\Delta$ Tm) between the test storage condition and the initial (0 Hour) test result was calculated for each assay. When $\Delta$ Cp for all assays was reviewed, no general trend toward later Cps (decreased sensitivity or fewer templates) was observed for any one particular storage condition.

The results of this study support the claim that NPS specimens in Viral Transport Medium (VTM) can be stored for up to 4 hours at room temperature $( 1 8 - 3 0 ^ { \circ } \mathrm { C } )$ , 3 days in the refrigerator $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ or 30 days in the freezer $( < - 1 5 ^ { \circ } \mathrm { C } )$ without affecting the accuracy of FilmArray RP test results.

# iii. Preservation of Vacuum for Pouch Hydration and Sample Loading and Stability of Reagents and Control Material after Hydration and Loading of a Pouch

To maintain pouch integrity and reagent stability, pouches are packaged in an aluminum canister, which is then sealed under vacuum in opaque mylar outer packaging. Ideally, a pouch is hydrated and sample loaded immediately after the pouch packaging is opened, and the loaded pouch is immediately placed in the FilmArray instrument for testing. However, laboratory workflow and instrument availability may result in a pouch being opened before a sample can be loaded, or a pouch to be loaded before an instrument is available for testing.

The FilmArray product literature recommends the loading and testing of pouches immediately or within a limited timeframe (30 minutes for pouch loading and 60 minutes for testing a loaded pouch).

Analytical testing was performed to discover whether the integrity and performance of an opened pouch is affected by longer times outside of the vacuum-sealed packaging prior to loading and testing.

Pouches were removed from outer packaging and exposed to air at ambient temperature for various times up to 24 hours. The preservation of vacuum for pouch hydration and sample loading was evaluated by loading blank simulated NPS (sNPS) samples (no target analytes present) according to standard procedure. The volume of each solution (hydration solution and sample/sample buffer mix) drawn into the pouch was recorded and the loaded pouches were then tested to observe the performance of the internal controls assays. The control results from these same pouches were used to assess reagent stability. Overall, the measures of performance for these tests included the efficiency of pouch hydration and sample loading, visual inspection of pouches, and the results (pass or fail) of internal control assays (RNA Process Control and PCR2 Control).

The stability of reagents and control material after hydration and loading of a pouch was also examined. Pouches were hydrated and loaded with blank sNPS sample (no analytes present) according to standard protocol directly after removal from outer packaging. The loaded pouches were then stored under ambient conditions for various times up to 24 hours. The results of internal control reactions (pass or fail) were assessed to determine whether system performance was affected by storage of a loaded pouch prior to testing. In addition, four (4) replicate pouches were loaded with contrived sample containing live respiratory pathogens (Chlamydophila pneumoniae, Human Metapneumovirus, Influenza A H1N1, Parainfluenza Virus 3, and Respiratory Syncytial Virus) at concentrations both above and below the estimated system LoD. The loaded pouches were stored at ambient temperature for 8 hours prior to testing. The results of both internal control and analyte assays from these 4 pouches were compared to eight (8) control pouches that were loaded with the same contrived sample but tested without storage (immediately after loading).

Each tested handling condition was considered successful if the variables (including mean Cp for the relevant assay) evaluated were equivalent (within 3 cycles for $\mathrm { C p }$ ) to that of the control sample (test at time 0 after pouch opening or loading). Any handling condition that resulted in improper pouch hydration, sample loading, failed control results, or unexpected negative analyte results was repeated to confirm the result. If the result was repeatable, then the integrity and/or stability of the pouch were considered diminished under the handling condition that caused the failure.

The data presented in the table below demonstrated that pouch vacuum was preserved for up to 24 hours after the removal of a pouch from its outer packaging, allowing for proper pouch hydration and sample uptake.

Pouch Testing Stability   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>PouchHydratedProperly?</td><td rowspan=1 colspan=2>Sample LoadedProperly?</td><td rowspan=1 colspan=2>InternalControlsPass?</td><td rowspan=1 colspan=2>Internal Control Cp</td></tr><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>RNA PC</td><td rowspan=1 colspan=1>PCR2</td></tr><tr><td rowspan=6 colspan=1>Remove pouchfrom packaging atvarious times priorto loading</td><td rowspan=1 colspan=1>0 (control)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20.77</td><td rowspan=1 colspan=1>21.43</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>21.73</td><td rowspan=1 colspan=1>19.93</td></tr><tr><td rowspan=1 colspan=1>60 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>22.53</td><td rowspan=1 colspan=1>21.53</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>21.50</td><td rowspan=1 colspan=1>20.33</td></tr><tr><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>21.97</td><td rowspan=1 colspan=1>20.90</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1>—</td><td rowspan=1 colspan=1>21.40</td><td rowspan=1 colspan=1>20.65</td></tr></table>

The data presented in the table below indicated that the stability of the control material and assay reagents is not significantly affected up to 24 hours after pouch hydration and sample loading.

Loaded Pouch Testing Stability   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Internal ControlsPass?</td><td rowspan=1 colspan=2>Internal Control Cp</td></tr><tr><td rowspan=1 colspan=1>Test Condition</td><td rowspan=1 colspan=1>Time</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>RNA PC</td><td rowspan=1 colspan=1>PCR2</td></tr><tr><td rowspan=6 colspan=1>Store a loaded pouch forvarious times prior to testing</td><td rowspan=1 colspan=1>0 (control)</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>20.77</td><td rowspan=1 colspan=1>21.43</td></tr><tr><td rowspan=1 colspan=1>30 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.90</td><td rowspan=1 colspan=1>21.57</td></tr><tr><td rowspan=1 colspan=1>60 minutes</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.67</td><td rowspan=1 colspan=1>20.20</td></tr><tr><td rowspan=1 colspan=1>4 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>23.47</td><td rowspan=1 colspan=1>20.63</td></tr><tr><td rowspan=1 colspan=1>8 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>22.80</td><td rowspan=1 colspan=1>21.23</td></tr><tr><td rowspan=1 colspan=1>24 hours</td><td rowspan=1 colspan=1>X</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>23.40</td><td rowspan=1 colspan=1>21.07</td></tr></table>

The following table lists the number of positive and negative results, an overall $\%$ positive, and the mean $\mathrm { C p }$ for each analyte or assay at each of the two test conditions:

Pouch Testing and Analyte Stability Tested Immediately and at 8 Hours   

<table><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=4>Control(Tested immediately)(n = 8 pouches)</td><td rowspan=1 colspan=4>Test(Tested after 8 hours at room temp.)(n = 4 pouches)</td></tr><tr><td rowspan=1 colspan=1>Organism/Assay</td><td rowspan=1 colspan=1>Multipleof LODTested</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>MeanCp</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>%Positive</td><td rowspan=1 colspan=1>Mean Cp</td></tr><tr><td rowspan=1 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>~18x</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.4</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.3</td></tr><tr><td rowspan=1 colspan=1>Influenza A HI</td><td rowspan=4 colspan=1>~0.2x</td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1>2/8a, b</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>1/4</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Flu A-panl</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>26.9</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>1/4</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td></tr><tr><td rowspan=1 colspan=1>Flu A-pan2</td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1>2/8</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>2/4</td><td rowspan=1 colspan=1>50%</td><td rowspan=1 colspan=1>27.1</td></tr><tr><td rowspan=1 colspan=1>FluA-HI</td><td rowspan=1 colspan=1>6/8</td><td rowspan=1 colspan=1>2/8</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>3/4</td><td rowspan=1 colspan=1>1/4</td><td rowspan=1 colspan=1>75%</td><td rowspan=1 colspan=1>27.1</td></tr><tr><td rowspan=1 colspan=1>HumanMetapneumovirus</td><td rowspan=1 colspan=1>~8x</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.1</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza virus 3</td><td rowspan=1 colspan=1>~2x</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>1/8</td><td rowspan=1 colspan=1>88%</td><td rowspan=1 colspan=1>23.1</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>23.6</td></tr><tr><td rowspan=1 colspan=1>Respiratory SyncytialVirus</td><td rowspan=1 colspan=1>~7x</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18.0</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>18.0</td></tr><tr><td rowspan=1 colspan=1>RNA Process Control</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.2</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.6</td></tr><tr><td rowspan=1 colspan=1>PCR2 Control</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>0/8</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19.8</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>0/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>19.6</td></tr></table>

a One Influenza A (no subtype detected). b One Equivocal Influenza A.

In conclusion, the data supports the conservative recommendations in the FilmArray product package insert for the loading and testing of pouches within a limited timeframe (30 minutes for pouch loading and 60 minutes for testing a loaded pouch).

# iv. Fresh vs. Frozen Study

In order to utilize frozen banked clinical samples to supplement the prospective clinical study data in the evaluation of the FilmArray Respiratory Panel System, an analytical study was conducted to demonstrate that preservation of samples (by freezing at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ does not affect the accuracy of test results compared to freshly collected or freshly prepared samples.

The “fresh vs. frozen” analytical study was carried out as part of a larger reproducibility study using samples that had been stored at $\mathrm { \leq } \mathrm { - } 7 0 ^ { \circ } \mathrm { C }$ for as long as 30 days. The test samples were prepared in bulk and tested immediately after preparation (fresh) prior to storage. The fresh sample tests provided a baseline of performance to which the results from subsequent testing of the frozen samples was compared. A panel of twelve (12) specimens was created by spiking simulated NPS sample matrix with known quantities of live RP analytes. The specimens in the panel were designed so that all RP analytes would be tested at different concentrations. Multiple analytes were spiked into each specimen to simulate the possibility of co-infections with multiple pathogens. Human Rhinovirus (HRV) and Respiratory Syncytial Virus (RSV) were included in more specimens than the other analytes because they were the most common panel members detected in co-infected clinical samples during the clinical evaluation performed by ITI. Immediately after sample preparation (spiking), each fresh specimen pool was screened with the FilmArray RP system to establish ‘baseline’ or expected results. The remaining specimen volume was then aliquoted into several individual use vials, and frozen until the scheduled day of testing (up to 30 days frozen at $\le - 7 0 ^ { \circ } \mathrm { C }$ ). Specimen aliquots were hand-delivered or shipped to external testing sites on dry ice and stored frozen until the day of testing. All specimen aliquots were tested on the same day they were removed from the freezer.

In total, at least 120 samples were tested for each analyte at (1x) or near $( 3 \mathbf { x } )$ the LoD determined in the LoD studies. All contrived samples consisted of live organisms spiked into negative simulated nasopharyngeal swab sample matrix. (Note: The simulated sample matrix consisted of human cells in viral transport media (VTM) and was validated in “Validation of a Simulated NPS Sample Matrix for Use with the FilmArray Respiratory Panel System”. This validation study demonstrated that the sensitivity of the FilmArray RP system was comparable between samples prepared in nasopharyngeal swab (NPS) and simulated NPS sample matrices. Since the natural NPS samples are required to be diluted in VTM before being tested by the FilmArray RP System, the simulated NPS matrix consisting of VTM and epithelial cells is considered to be similar enough to the natural matrix (i.e., NPS in VTM) for the purpose of this study.

The summary results of the study are presented in the table below:

Testing of Fresh vs. Frozen Samples $( \le - 7 0 ^ { \circ } )$   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Medium Positive (3x LoD)</td><td colspan="2" rowspan="1">Low Positive (1x LoD)</td><td colspan="2" rowspan="1">All Test Levels Combined</td></tr><tr><td colspan="1" rowspan="1">#Detected7Total</td><td colspan="1" rowspan="1">% Positive[95% CI]</td><td colspan="1" rowspan="1"># Detected/Ttotal</td><td colspan="1" rowspan="1">% Positive[95% CI]</td><td colspan="1" rowspan="1"># DetectedTotal</td><td colspan="1" rowspan="1">% Positive[95% CI]</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%[91.06% - 99.96%]</td><td colspan="1" rowspan="1">119/120</td><td colspan="1" rowspan="1">99.2%[95.44% - 99.98%]</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%[91.06% - 99.96%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">119/120</td><td colspan="1" rowspan="1">99.2%[95.44% - 99.98%]</td></tr><tr><td colspan="1" rowspan="1">#DetectedITotal</td><td colspan="1" rowspan="1">% Positive[95% CI]</td><td colspan="1" rowspan="1"># Detected/Total</td><td colspan="1" rowspan="1">% Positive[95% CI]</td><td colspan="1" rowspan="1"># DetectedTotal</td><td colspan="1" rowspan="1">% Positive[95% CI]</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%[97.97% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">240/240</td><td colspan="1" rowspan="1">100%[98.47% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">56/60a</td><td colspan="1" rowspan="1">93.3%[83.80% - 98.15%]</td><td colspan="1" rowspan="1">116/120</td><td colspan="1" rowspan="1">96.7%[91.69% - 99.08%]</td></tr><tr><td colspan="1" rowspan="1">Influenza A2009H1N1</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">56/60</td><td colspan="1" rowspan="1">93.3%[83.80 - 98.15%]</td><td colspan="1" rowspan="1">116/120</td><td colspan="1" rowspan="1">96.7%[91.69% - 99.08%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">59/60</td><td colspan="1" rowspan="1">98.3%[91.06% - 99.96%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">119/120</td><td colspan="1" rowspan="1">99.2%[95.44% - 99.98%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">57/60</td><td colspan="1" rowspan="1">95.0%[86.08% - 98.96%]</td><td colspan="1" rowspan="1">117/120</td><td colspan="1" rowspan="1">97.5%[92.87% - 99.48%]</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">60/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">0/60</td><td colspan="1" rowspan="1">100%[94.04% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td></tr><tr><td colspan="1" rowspan="1">Respiratory SyncytialVirus</td><td colspan="1" rowspan="1">180/180</td><td colspan="1" rowspan="1">100%[97.97% - 100%]</td><td colspan="1" rowspan="1">120/120</td><td colspan="1" rowspan="1">100%[96.97% - 100%]</td><td colspan="1" rowspan="1">300/300</td><td colspan="1" rowspan="1">100%[98.78% - 100%]</td></tr></table>

a Three (3) Equivocals are not included as positive results. Samples with Equivocal results were not retested in this study.

The results of this study demonstrated that freezing at $\le - 7 0 ^ { \circ } \mathrm { C }$ for up to 30 days does not alter the performance of the FilmArray RP System in comparison to testing of fresh specimens.

# e. Detection limit:

Analytical studies were carried out to determine the LoD for each FilmArray RP targeted organism.

A preliminary estimate of sensitivity for many RP analytes was determined as part of the “FilmArray RP Melt Detector Tuning” study using limiting dilutions of single organisms spiked into negative NPS specimens. Based on these preliminary results, initial estimates of LoD were set by spiking negative NPS specimens with serial 10-fold dilutions of quantified live virus. Where possible, each stock organism was re-grown, verified, and quantified

$\mathrm { ( T C I D _ { 5 0 } / m L }$ , $\mathrm { C F U / m L }$ , copies/mL, etc.). Coronavirus HKU1 could not be re-grown for verification despite multiple attempts; instead, a clinical specimen containing a high viral load was quantified by a non-FilmArray real-time PCR against a standard curve of synthetic RNA transcript (Coronavirus HKU1) to obtain quantification of the stock material in RNA genome copies $\mathrm { \Delta } / \mathrm { m L }$ , respectively. The identity of the virus contained in the clinical specimen was verified by bi-directional sequencing. The single-spiked specimens were then tested with the FilmArray RP System. The lowest concentration of organism for which all or most replicates were assigned a positive (detected) result by the FilmArray software was selected for additional testing in multi-spiked specimens.

Multiple organisms were combined and tested as a mix (multi-spike; see the table below) in replicate NPS specimens at a single concentration or as a dilution series centered on the estimated LoD. Results from single-spiked and multi-spiked specimens were compared to determine whether the presence of multiple organisms in a specimen would affect the system LoD.

# Composition of Organism Mixes

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>Mix 4</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>Human Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Parainfluenza Virus 1</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td></tr></table>

Both percent detection $\%$ positive) and $\mathrm { C p }$ values were used to compare results for singlespiked and multi- spiked specimen data sets. It was concluded that multi-spiking and singlespiking results were substantially equivalent, since the $\%$ detection was within $2 5 \%$ for the two data sets and/or the mean $\mathrm { C p }$ values were within 3 cycles. (Note: Based on an observed Cp standard deviation for replicate samples of about 1-2 cycles, a 3 cycle difference is a conservative estimate of the expected variation when testing near LoD and does not indicate a significant difference in sensitivity.)

Additional LoD testing and confirmation of LoD for primary strains was then performed with multi-spiked specimens. Coronavirus HKU1 was not included in a multi- spiked mix for this study. Coronavirus HKU1 LoD was confirmed as a single-spiked sample but was shown in subsequent studies to be detected at single-spiked LoD levels when tested in multispiked specimens.

As needed, adjustments to the concentration of individual organisms within a multi-spiked mix were made prior to confirmation of final LoD concentrations. Confirmation of the system LoD for each organism included testing of 20 independent multi-spiked specimens at LoD and an additional 20 specimens spiked 10- fold below LoD (0.1x LoD). For confirmation, at least 19/20 positive results were required for each organism at the LoD concentration and fewer than $9 5 \%$ positive results were required at concentrations below LoD.

Additional sensitivity testing was performed on one or more alternate strains of several RP organisms. These alternate strains represented different major subgroups of common respiratory viruses (Adenovirus, Human Metapneumovirus, Human Rhinovirus and Respiratory Syncytial Virus) and isolates of Influenza A and B from different seasons and geographic locations. Each alternate strain was tested as 10-fold dilutions around the previously determined LoD.

Note: Although this testing was performed in part to demonstrate that the FilmArray RP system can detect various species and strains of an organism with similar sensitivity, it is difficult to make comparative sensitivity evaluations between different organisms and different strains that have been quantified by $\mathrm { T C I D } _ { 5 0 }$ . This quantification method measures the infectivity and cytotoxicity or lethality of an organism in tissue culture and is therefore subject to many influences (strain-to-strain differences in infectivity, viability of the original material, culturing conditions, etc.). Quantification by infectivity assay will not be equivalent between strains or organisms. Also, since the measurand for the FilmArray system is nucleic acid, LoD concentrations established in $\mathrm { T C I D } _ { 5 0 }$ can vary dramatically between targets, but may not actually reflect significant differences in sensitivity of detection as measured by target nucleic acid concentration.

During the early stages of LoD testing, it was discovered that some NPS specimens collected from apparently healthy individuals did in fact contain respiratory pathogens (mostly Rhinovirus). As a result, a simulated NPS (sNPS) sample matrix, consisting of cultured human epithelial cells (HeLa) in Viral Transport Medium, was evaluated for use in LoD confirmation testing and subsequent analytical studies. The sensitivity of the FilmArray RP system with this simulated sample matrix was found to be comparable to a sample matrix comprised of pooled NPS collected from healthy individuals. The simulated NPS sample matrix was used for final LoD confirmations and all subsequent analytical studies.

Summary results for the LoD confirmation studies using primary strains and additional sensitivity testing of alternate strains are presented below by organism:

# Adenovirus

# LoD Confirmation Results for Adenovirus C Serotype 1 (Primary Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Adenovirus C Serotype 1</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(™CID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100.0%</td></tr><tr><td rowspan=1 colspan=1>30(0.1x LoD)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>75%</td></tr></table>

# LoD Testing Results for Adenovirus B Serotype 3 (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Adenovirus B Serotype 3</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCD50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>Single Spike DilutionSeries</td><td rowspan=1 colspan=1>3000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr></table>

# LoD Testing Results for Adenovirus E Serotype 4a (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Adenovirus E Serotype 4a</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>3000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0%</td></tr></table>

# Bordetella pertussis

LoD Testing Results for Bordetella pertussis A639   

<table><tr><td rowspan=1 colspan=7>Bordetella pertussis A639</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(CFU/m)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>MedianCp</td><td rowspan=2 colspan=1>Mean Cp +/-Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation</td><td rowspan=1 colspan=1>4000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>22.0</td><td rowspan=1 colspan=1>22.28± 0.89</td></tr><tr><td rowspan=1 colspan=1>400(0.1x LoD)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>60%</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>25.285 ± 2.09</td></tr></table>

Chlamydophila pneumoniae LoD Testing Results for Chlamydophila pneumoniae TW183   

<table><tr><td rowspan=1 colspan=7>Chlamydophila pneumoniae TW183</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(genome copies/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>MedianCp</td><td rowspan=2 colspan=1>Mean Cp +/-Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Single vs. Multi-Spike</td><td rowspan=1 colspan=1>3000 (single)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>24.7</td><td rowspan=1 colspan=1>24.83± 1.79</td></tr><tr><td rowspan=1 colspan=1>3000 (multi)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>233</td><td rowspan=1 colspan=1>23.44 ± 1.66</td></tr></table>

# Chlamydophila pneumoniae TW183

<table><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(genome copies/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>Median Cp</td><td rowspan=2 colspan=1>Mean Cp +/-Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation</td><td rowspan=1 colspan=1>3000</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95%</td><td rowspan=1 colspan=1>23.4</td><td rowspan=1 colspan=1>23.78± 1.88</td></tr><tr><td rowspan=1 colspan=1>300(0.1x LoD)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>72%</td><td rowspan=1 colspan=1>27.1</td><td rowspan=1 colspan=1>25.65 ± 1.82</td></tr></table>

Coronavirus HKU1 LoD Confirmation Results for Coronavirus HKU1 Clinical Specimen (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Coronavirus HKU1Clinical Specimen</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(genome copies/ml)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Single Spike )</td><td rowspan=1 colspan=1>1.90 x 106</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1.90 x 105(0.1x LoD)</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>70%</td></tr></table>

Coronavirus NL63 LoD Confirmation Results for Coronavirus NL63 NR-470 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Coronavirus NL63 NR-470</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.5(0.1x LoD)</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>80%</td></tr></table>

Coronavirus OC43 LoD Confirmation Results for Coronavirus OC43 VR-759 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Coronavirus OC43 VR-759</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>60(0.1x LoD)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>5%</td></tr></table>

# Coronavirus 229E LoD Confirmation Results for Coronavirus 229E VR-740 (Primary Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Coronavirus 229E VR-740</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Final LoD Confirmation</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>0.4(0.1x LoD)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>80%</td></tr></table>

# Enterovirus

# LoD Confirmation Results for Enterovirus (Echovirus 6) (Primary Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Enterovirus Echovirus 6</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 2)</td><td rowspan=1 colspan=1>30000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>3000(0.1x LoD)</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>90%</td></tr></table>

Human Metapneumovirus LoD Confirmation Results for Human Metapneumovirus (Type A1) (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Human Metapneumovirus (Type A1)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.2 (0.1x LoD)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>45%</td></tr></table>

# LoD Testing Results for Human Metapneumovirus (Type A2) (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Human Metapneumovirus (Type A2)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr></table>

# LoD Testing Results for Human Metapneumovirus (Type B1) (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Human Metapneumovirus (Type B1)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr></table>

# LoD Testing Results for Human Metapneumovirus (Type B2) (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Human Metapneumovirus (Type B2)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0%</td></tr></table>

Human Rhinovirus LoD Confirmation Results for Human Rhinovirus Type 1A (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Human Rhinovirus - Type 1A</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 2)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>0.1 (0.1x LoD)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>15%</td></tr></table>

Influenza A/H1N1 LoD Confirmation Results for Influenza A/H1N1 (A/Brisbane/59/07) (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=6>Influenza A/H1N1 (A/Brisbane/59/07)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>20 (0.1x LoD)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>60%</td></tr></table>

a Influenza A (no subtype detected) result

LoD Confirmation Results for Influenza A/H1N1 (A/New Caledonia/20/99) (Alternate Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=6>Influenza A/H1N1 (A/New Caledonia/20/99)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCD50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr><tr><td rowspan=1 colspan=1>LoD Confirmation</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

Influenza A/2009 H1N1 LoD Confirmation Results for Influenza A/2009 H1N1 (A/Swine/NY/2009/03) (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=6>Influenza A/2009 H1N1 (A/Swine/NY/2009/03)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix 2) 10 (0.1x LoD)</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>10 (0.1x LoD)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>85%</td></tr></table>

a Equivocal Influenza A results

Influenza A/H3N2 LoD Confirmation Results for Influenza A/H3N2 (A/Wisconsin/67/2005) (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=6>Influenza A/H3N2 (A/Wisconsin/67/2005)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix 3) 0.5 (0.1x LoD)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>(Multiple Spike Mix 3) 0.5 (0.1x LoD)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8a</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40%</td></tr></table>

Two Equivocal Influenza A/H3 results, six Influenza A Equivocal results

LoD Confirmation Results for Influenza A/H3N2 (A/Port Chalmers/1/73) (Alternate Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=6>Influenza A/H3N2 (A/Port Chalmers/1/73)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=3># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Equ</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>LoD Confirmation</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

a Influenza A Equivocal b One Influenza A Equivocal and one Influenza A/H3 Equivocal

# Influenza B

# LoD Confirmation Results for Influenza B (B/FL/04/06) (Primary Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Influenza B (B/FL/04/06)</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of SamplesPos       Neg</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix 3)</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>6 (0.1 x LoD)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>65%</td></tr></table>

# LoD Testing Results for Influenza B (B/Taiwan/2/62) (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Influenza B (B/Taiwan/2/62)</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=3 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>LoD Confirmation</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr></table>

# Mycoplasma pneumoniae

# LoD Confirmation Results for Mycoplasma pneumoniae (M129)

<table><tr><td rowspan=1 colspan=7>Mycoplasma pneumoniae M129</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>MedianCp</td><td rowspan=2 colspan=1>Mean Cp +/-Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Single vs. Multi-Spike(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>30 (single)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90%</td><td rowspan=1 colspan=1>20.96</td><td rowspan=1 colspan=1>20.84± 1.99</td></tr><tr><td rowspan=1 colspan=1>30 (multi)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>21.70</td><td rowspan=1 colspan=1>21.85 ± 1.13</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Mycoplasma pneumoniae M129</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCD50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>%Positive</td><td rowspan=2 colspan=1>MedianCp</td><td rowspan=2 colspan=1>Mean Cp +/-Std. Dev.</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix 1)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>20.9</td><td rowspan=1 colspan=1>21.30± 2.18</td></tr><tr><td rowspan=1 colspan=1>3 (0.1x LoD)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>85%</td><td rowspan=1 colspan=1>24.9</td><td rowspan=1 colspan=1>25.24 ± 1.88</td></tr></table>

Parainfluenza Virus 1 LoD Confirmation Results for Parainfluenza Virus 1 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Parainfluenza Virus 1</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 4)</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>50 (0.1 x LoD)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>61%</td></tr></table>

Parainfluenza Virus 2 LoD Confirmation Results for Parainfluenza Virus 2 (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Parainfluenza Virus 2</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoD Confirmation(Multiple Spike Mix 4)</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>1 (0.1x LoD)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>30%</td></tr></table>

# Parainfluenza Virus 3 LoD Confirmation Results for Parainfluenza Virus 3 (Primary Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Parainfluenza Virus 3</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Final LoD Confirmation</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>(Multiple Spike Mix 3)</td><td rowspan=1 colspan=1>1 (0.1x LoD)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>10%</td></tr></table>

Parainfluenza Virus 4 LoD Confirmation Results for Parainfluenza Virus 4a (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Parainfluenza Virus 4a</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCD50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=1 colspan=1>Final LoD Confirmation</td><td rowspan=1 colspan=1>5000</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>(Multiple Spike Mix 2)</td><td rowspan=1 colspan=1>500 (0.1x LoD)</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>65%</td></tr></table>

Respiratory Syncytial Virus A LoD Confirmation Results for Respiratory Syncytial Virus A (Primary Strain for the LoD Study)   

<table><tr><td rowspan=1 colspan=5>Respiratory Syncytial Virus A</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=2 colspan=1>Final LoDConfirmation(Multiple Spike Mix 2) 0.2 (0.1x LoD)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>(Multiple Spike Mix 2) 0.2 (0.1x LoD)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>35%</td></tr></table>

# LoD Confirmation Results for Respiratory Syncytial Virus B (Alternate Strain for the LoD Study)

<table><tr><td rowspan=1 colspan=5>Respiratory Syncytial Virus B</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=2 colspan=1>Spiked Conc.(TCID50/mL)</td><td rowspan=1 colspan=2># of Samples</td><td rowspan=2 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>Neg</td></tr><tr><td rowspan=4 colspan=1>Single SpikeDilution Series</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>75%</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>25%</td></tr><tr><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0%</td></tr></table>

In conclusion, confirmed LoD concentrations for each of the verified and quantified organisms listed are presented below. Two strains are listed for Influenza A/H1N1, Influenza A/H3N2, and Influenza B; the strain in bold is the primary strain:

LoD Concentrations by Analyte   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Concentration</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Serotype 1 (Species C)</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>4000 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>TW183</td><td rowspan=1 colspan=1>3000 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Clinical Specimen #6213</td><td rowspan=1 colspan=1>1.9 x 106 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>NR-470</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>600 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>4 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>Echovirus 6</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>hMPV-16, IA10-2003 (Type A1)</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Type 1A</td><td rowspan=1 colspan=1>1 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>200 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>2,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A 2009 H1N1</td><td rowspan=1 colspan=1>A/SwineNY/03/2009</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=2 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62</td><td rowspan=1 colspan=1>60 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=1>30 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=1>500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>Type 4a</td><td rowspan=1 colspan=1>5,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Type A</td><td rowspan=1 colspan=1>2 TCID50/mL</td></tr></table>

# f. Analytical Reactivity

The analytical inclusivity study was performed to determine whether the FilmArray RP System is able to detect a variety of strains (inclusivity panel) that represent the temporal and geographic diversity of each of RP target organism. This study expanded upon the limit of detection studies by determining whether different strains of the same organism can be detected at similar concentrations.

This study involves testing panels of various strains of organisms spiked into simulated NPS specimens using the FilmArray Respiratory Panel (RP) System. Inclusivity organisms were initially tested near the limit of detection (LoD) for each organism as determined in the LoD study. If a specimen containing a particular strain was positive (detected) at the LoD level, no further testing of that strain was required. If a strain was not detected at the LoD concentration, the strain was retested at the LoD level and 10-fold higher. If necessary, the same approach was followed until a positive result was obtained or the maximum

concentration possible for that strain stock had been tested.

The inclusivity panel represents the evolutionary, temporal, and geographical diversity of the RP analytes. For example, the inclusivity panel includes 17 different Adenovirus serotypes representing species A, B, C, D, E and F, over 20 Influenza A strains (subtypes H3, H1 and $2 0 0 9 \mathrm { H l }$ ) isolated from around the world as many as 77 years ago (1933) and as recently as 2009, and 25 strains of the group that comprised of Human Rhinovirus A and B, Enterovirus, Echovirus, and Coxsackievirus. However, for some analytes, the inclusivity panel is restricted to only 1 or 2 strains because of limited strain diversity and availability.

For inclusivity (reactivity) testing, a total of 133 individual RP target organism strains were tested and accurately identified with the FilmArray RP system. Each of the 131 strains tested in this study were reactive with the FilmArray RP system. The sensitivity of detection for Adenovirus C serotype 6 was 10,000-fold less than the system LoD due to sequence variation (mismatches) within the primers of the second stage PCR reaction. The sensitivity of detection for Adenovirus C serotype 2 was 100-fold less than the system LoD due to sequence variation (mismatches) within the region targeted by the FilmArray RP Adenovirus assay. The limited detection capability of the FilmArray RP system for these Adenovirus serotypes is described in FilmArray RP intended use as follows: “The FilmArray RP detects Adenovirus species $C$ serotype 2 and serotype 6 with reduced sensitivity. It is recommended that specimens found to be negative for Adenovirus after examination using FilmArray RP be confirmed by an alternate method (e.g., FDA cleared molecular test or cell culture).”

Summary tables of Analytical Reactivity (Inclusivity) Testing with the FilmArray RP System are presented below:

Adenovirus Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=12 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=8 colspan=1>B</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7a</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7d2(Iowa/2001)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>7h(Iowa/1999)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>11(Wisconsin/2005)</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>14(Missouri/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>21(Missouri/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>34(Texas/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=1>C</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>2(New York/2004)a</td><td rowspan=1 colspan=1>30,000</td><td rowspan=1 colspan=1>100x</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>6(Colorado/2005)a</td><td rowspan=1 colspan=1>3,000,000</td><td rowspan=1 colspan=1>10,000x</td></tr><tr><td rowspan=1 colspan=1>D</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=2 colspan=1>E</td><td rowspan=1 colspan=1>4a(S Carolina/2004)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>4p3(New Jersey/2005)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>41(Indiana/2004)</td><td rowspan=1 colspan=1>300</td><td rowspan=1 colspan=1>1x</td></tr></table>

a Alignment of the FilmArray RP primers with Adenovirus species C serotype 2 and serotype 6 sequences demonstrated that there are a number of mismatches between the serotype 2 and serotype 6 sequence of the Hexon gene and both outer and inner primers contained in the FilmArray RP pouch.

Bordetella pertussis Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain/Isolate</td><td rowspan=1 colspan=1>Concentration Detected</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=9 colspan=1>Bordetellapertussis</td><td rowspan=1 colspan=1>A639</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>E341</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>FATCC 8467</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5 [17921]ATCC 9340</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>18323 [NCTC 10739]ATCC 9797</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>10-536ATCC 10380</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CNCTC Hp12/63 [623]ATCC 51445</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Tohama IATCC BAA-589 (vaccine strain)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>MN2531(ATCC BAA-1335)</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

Chlamydophila pneumoniae Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain / Isolate</td><td rowspan=1 colspan=1>Concentration Detected</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=4 colspan=1>Chlamydophilapneumoniae</td><td rowspan=1 colspan=1>AR-39ATCC 53592</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CDC/CWL-029VR-1310</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>CM-1ATCC VR-1360</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>TW183ATCC VR-2282</td><td rowspan=1 colspan=1>3000 copies/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

# Coronavirus Inclusivity

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain/ Isolate</td><td rowspan=1 colspan=1>Concentration Detected</td><td rowspan=1 colspan=1>Multiple ofLoD Detected</td></tr><tr><td rowspan=6 colspan=1>Coronavirus</td><td rowspan=1 colspan=1>HKU1</td><td rowspan=1 colspan=1>Clinical Sample #1120</td><td rowspan=1 colspan=1>2.08 x 106 genome copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=1>NL63</td><td rowspan=1 colspan=1>Clinical Sample# 6123(Lot #304842)</td><td rowspan=1 colspan=1>1.41 x 104 TCID50 /mL(~1.9 x 106 ggenome copies/mL)</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Clinical Sample#6213Type B</td><td rowspan=1 colspan=1>1.9 x 106 genome copies/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>NR-470 (BEI)</td><td rowspan=1 colspan=1>50 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>OC43</td><td rowspan=1 colspan=1>ATCC VR-759</td><td rowspan=1 colspan=1>600 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>229E</td><td rowspan=1 colspan=1>ATCC VR-740</td><td rowspan=1 colspan=1>4 TCID50/mL</td><td rowspan=1 colspan=1>1x</td></tr></table>

Enterovirus/Rhinovirus Inclusivity   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration Detected(TCID50/ml)</td><td colspan="1" rowspan="1">Multiple of LoDDetected</td></tr><tr><td colspan="1" rowspan="12">Enterovirus</td><td colspan="1" rowspan="3">A</td><td colspan="1" rowspan="1">Coxsackievirus A10ATCC VR-168</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71ATCC VR-1432</td><td colspan="1" rowspan="1">1:30,000dilution of stock</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="6">B</td><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">9,400</td><td colspan="1" rowspan="1">&lt;1x</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">300,000</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="2">C</td><td colspan="1" rowspan="1">Coxsackievirus A21 /KuykendallATCC VR-850</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24 DN-19ATCC VR-583</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">Db</td><td colspan="1" rowspan="1">Enterovirus 68 (F02-3607 corn)ATCC VR-1197</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="6">HumanRhinovirus</td><td colspan="1" rowspan="6">A</td><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A2 (HGP)ATCC VR-482</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A7 (68-CV11)ATCC VR-1601</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A16 (11757)ATCC VR-283</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">A34 (137-3)ATCC VR-507</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A57 (Ch47) ATCC VR-1600a</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="8">′</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">A77 (130-63)ATCC VR-1187</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">A85 (50-525-CV54)ATCC VR-1195</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="6">B</td><td colspan="1" rowspan="1">B3 (FEB)ATCC VR-483</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B14 (1059)ATCC VR-284</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B17 (33342) ATCC-282a</td><td colspan="1" rowspan="1">100</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="1">B27 (5870) ATCC VR-1137</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">B42 (56822) ATCC VR-338</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">B83 (Baylor 7) ATCC VR-1193</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1x</td></tr></table>

a The 5’UTR sequences of these strains were aligned with the outer and inner PCR primers for the HRV assays to determine whether sequence mismatches may be contributing to the somewhat reduced sensitivity of the FilmArray RP system for these strains. Primers for at least one of the 4 HRV assays (inner primers) aligned very well with published sequence for each strain $\boldsymbol { { 0 - 1 } }$ mismatches in the primer region), suggesting that the observed differences in sensitivity are likely attributable to inconsistencies in quantification of the viral stock rather than the sensitivity of the assay(s) themselves. b Enterovirus species D is also referred to as Human Rhinovirus species C.

Human Metapneumovirus Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=10 colspan=1>HumanMetapneumovirus</td><td rowspan=2 colspan=1>A1</td><td rowspan=1 colspan=1>9(IA3-2002)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>16(IA10-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>A2</td><td rowspan=1 colspan=1>20(IA14-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>27(IA27-2004)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=4 colspan=1>B1</td><td rowspan=1 colspan=1>3(Peru2-2002)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>5(Peru3-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>13(IA7-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>18(IA18-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>B2</td><td rowspan=1 colspan=1>8(Peru6-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>22(IA16-2003)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr></table>

# Influenza A Inclusivity

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=10 colspan=1>Influenza A (H1N1)</td><td rowspan=1 colspan=1>A/Brisbane/59/07</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Solomon Islands/3/2006</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Hawaii/15/01CDC#2001701117</td><td rowspan=1 colspan=1>1:300 dilution of stock</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>A/New Caledonia/20/99</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A1/Denver/1/57ATCC VR-546</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Mal/302/54ATCC VR-98</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A1/FM/1/47ATCC VR-97</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Weiss/43ATCC VR-96</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/PR/8/34ATCC VR-95</td><td rowspan=1 colspan=1>2000</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A/NWS/33ATCC VR-219</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=1>Influenza A (2009H1N1)</td><td rowspan=1 colspan=1>Swine NY/01/2009</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Swine NY/02/2009</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Swine NY/03/2009</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=9 colspan=1>Influenza A (H3N2)</td><td rowspan=1 colspan=1>A/Brisbane/10/07</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Wisconsin/67/2005</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/NewYork/55/2005CDC#2005705561</td><td rowspan=1 colspan=1>1:300,000 dilution of stock</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>A/Victoria/3/75ATCC VR-822</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Port Chalmers/1/73ATCC VR-810</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/Aichi/2/68ATCC VR-547</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>A/Hong Kong/8/68ATCC VR-544</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Alice (vaccine) A/England/42/72ATCC VR-776</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>MRC-2 Recombinant strainATCC VR-777</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1x</td></tr></table>

# Influenza B Inclusivity

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/ml)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=11 colspan=1>Influenza B</td><td rowspan=1 colspan=1>B/FL/04/06Z-0810037CF</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Ohio/01/2005CDC#2005743348</td><td rowspan=1 colspan=1>1:3,000,000 dilution ofstock</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Florida/07/04</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Malaysia/2506/04</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/Hong Kong/5/72ATCC VR-823</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Taiwan/2/62ATCC VR-295</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Maryland/1/59ATC VR-296</td><td rowspan=1 colspan=1>600</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=1 colspan=1>B/GL/1739/54ATCC VR-103</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Allen/45ATCC VR-102</td><td rowspan=1 colspan=1>6,000 EID50/mL</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>B/Lee/40ATCC VR-101</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit RecombinantATCC VR-786</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>1x</td></tr></table>

# Mycoplasma pneumoniae Inclusivity

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Strain / Isolate</td><td colspan="1" rowspan="1">Concentration Detected(TCID, or CCU/mL)</td><td colspan="1" rowspan="1">Multiple ofLoD Detected</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129(Type 1)</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="8">′</td><td colspan="1" rowspan="1">M129-B7ATCC 29342 (Type 1)</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">PI 1428ATCC 29085 (Type 1)</td><td colspan="1" rowspan="1">3000</td><td colspan="1" rowspan="1">100x</td></tr><tr><td colspan="1" rowspan="1">FH Strain [NCTC 10119]ATCC 15531 (Type 2)</td><td colspan="1" rowspan="1">300 CFU/mL</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">[Mac]ATCC 15492 (Type 2)</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">[M52]ATCC 15293</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">10x</td></tr><tr><td colspan="1" rowspan="1">[Bru]ATCC 15377</td><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">1,000x</td></tr><tr><td colspan="1" rowspan="1">Mutant 22ATCC 39505</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">1x</td></tr><tr><td colspan="1" rowspan="1">UMTB 10GATCC 49894</td><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">10x</td></tr></table>

Parainfluenza Virus Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Concentration Detected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=12 colspan=1>Parainfluenza Virus</td><td rowspan=3 colspan=2>PIV1</td><td rowspan=1 colspan=1>Zeptometrix #0810014CF</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>C-35ATCC VR-94</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>C39BEI NR-3226</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=2>PIV2</td><td rowspan=1 colspan=1>Zeptometrix #0810015CF</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>GreerATCC VR-92</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=2>PIV3</td><td rowspan=1 colspan=1>Zeptometrix#0810016CF</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>C-243ATCC VR-93</td><td rowspan=1 colspan=1>500</td><td rowspan=1 colspan=1>50x</td></tr><tr><td rowspan=1 colspan=1>NIH 47885BEI NR-3233</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>10x</td></tr><tr><td rowspan=4 colspan=1>PIV4</td><td rowspan=2 colspan=1>4A</td><td rowspan=1 colspan=1>M25ATCC VR-1378</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix #0810060CF</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=2 colspan=1>4B</td><td rowspan=1 colspan=1>CH-19503ATCC VR-1377</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>Zeptometrix#08010060BCF</td><td rowspan=1 colspan=1>5,000</td><td rowspan=1 colspan=1>1x</td></tr></table>

Respiratory Syncytial Virus Inclusivity   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Type</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>ConcentrationDetected(TCID50/mL)</td><td rowspan=1 colspan=1>Multiple of LoDDetected</td></tr><tr><td rowspan=6 colspan=1>Respiratory SyncytialVirus</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>Zeptometrix#0810040ACF</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/A2ATCC VR-1540</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>A/LongATCC VR-26</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>B/9320ATCC VR-955</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/Wash18537/62ATCC VR-1580</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr><tr><td rowspan=1 colspan=1>B/WV/14617/85ATCC VR-1400</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1x</td></tr></table>

# i. Supplemental Adenovirus Reactivity Information (clinical study data and in silico analyses):

A subset of prospective and retrospective archived clinical specimens that were FilmArray positive for Adenovirus was subjected to PCR and bi-directional sequence analysis. BLAST analysis of the sequence data obtained for a region of the Adenovirus polymerase gene identified adenovirus species B (serotypes 3, $3 { + } 1 1 \mathfrak { p }$ , 7, 16 and 21), species C (serotypes 1, 2, and 5), and species E (serotype 4) in these clinical specimens. Species B, C and E are the most common Adenovirus species associated with respiratory illness. Species A, D, and F (often associated with conjunctivitis and gastroenteritis) were not identified in clinical specimens.

In addition to laboratory testing, bioinformatics resources were also used to predict reactivity of additional Adenovirus species and serotypes with the FilmArray RP System. Simulated reactivity was based on the number and location of mismatches between the target sequence and the assay primer(s). The table below lists the Adenovirus types that were not tested by the FilmArray system either in analytical or clinical testing. Bioinformatics analysis of these Adenovirus sequences indicates that the FilmArray RP system will react with all Adenovirus species and serotypes.

Simulated FilmArray RP Reactivity with Untested Adenovirus Serotypes   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=8 colspan=1>Adenovirus</td><td rowspan=2 colspan=1>A</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>AB330093</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>DQ149610</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=3 colspan=1>B</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>AB330097</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>AB052912</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>DQ149643</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=3 colspan=1>D</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>AB330090</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>DQ149615</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>DQ149616</td><td rowspan=1 colspan=1>Positive</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Species</td><td rowspan=1 colspan=1>Serotype</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArrayAdenovirus Result</td></tr><tr><td rowspan=31 colspan=2>FG</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>DQ149617</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>AB330098</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>DQ149618</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>DQ149619</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>DQ149620</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>DQ149621</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>DQ149622</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>DQ149623</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>DQ149624</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>DQ149625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>DQ149626</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>DQ149627</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>DQ149628</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>DQ149629</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>DQ149630</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>DQ149631</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>37</td><td rowspan=1 colspan=1>DQ149632</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>DQ149633</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>DQ149634</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>DQ149635</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>DQ149636</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>DQ149637</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>DQ149638</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>DQ149639</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>DQ149640</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>AB330129</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>DQ149641</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>51</td><td rowspan=1 colspan=1>DQ149642</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>FJ169625</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>F</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>AB330121</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>DQ923122</td><td rowspan=1 colspan=1>Positive</td></tr></table>

# ii. Supplemental Human Rhinovirus/Enterovirus Reactivity Information (clinical study data and in silico analyses):

In addition to the analytical inclusivity testing, BLAST analysis was performed on sequence data ( $\zeta ^ { \star }$ UTR) obtained from prospective and retrospective archived clinical specimens that were FilmArray positive for Human Rhinovirus/Enterovirus. The following species and subtypes were identified in clinical specimens:

Species and Subtypes Identified in Human Rhinovirus/Enterovirus Clinical Specimens   

<table><tr><td rowspan=1 colspan=1>cYanmaGellowb</td><td></td></tr><tr><td rowspan=1 colspan=1>Enterovirus Species A</td><td rowspan=1 colspan=1>Enterovirus serotype 71Coxsackievirus A2 and A6</td></tr><tr><td rowspan=1 colspan=1>Enterovirus Species B</td><td rowspan=1 colspan=1>Echovirus serotypes 3, 6, 7, 11, 15, 21 and 30Coxsackievirus B1, B3 and A9Enterovirus serotypes 81 and 88</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Species A</td><td rowspan=1 colspan=1>Human Rhinovirus serotypes 1B, 8 ,9 ,10, 13, 19, 21, 22, 23, 28, 30, 32, 34,38, 39, 40, 46, 47, 49, 51, 54, 56, 58, 59, 61, 62, 66, 68, 75, 77, 78, 80, 82,98 and 100</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Species B</td><td rowspan=1 colspan=1>Human Rhinovirus serotypes 27, 69, 83 and 91</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus Species C</td><td rowspan=1 colspan=1>At least 3 individual strains and 12 distinct isolatesa</td></tr></table>

a Human Rhinovirus species C (also known as Enterovirus species D) has not been classified into serotypes.

Simulated reactivity of the FilmArray RP Human Rhinovirus/Enterovirus assays with Enterovirus Species C Poliovirus strains was generated using a bioinformatics approach. Alignment of assay primer sequences with the GenBank sequences indicates the FilmArray RP assay will react with Poliovirus types 1, 2 and 3, giving a Human Rhinovirus/Enterovirus result.

Simulated Reactivity of the FilmArray RP HRV and Entero Assays with Poliovirus Sequences (Enterovirus Species C)   

<table><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBank ID</td><td rowspan=1 colspan=1>Simulated FilmArray Result</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 1 strain CHN8264c/GZ/CHN/2004</td><td rowspan=1 colspan=1>FJ769385</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 2, complete genome</td><td rowspan=1 colspan=1>AY177685</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human poliovirus 3 strain IRA10853, complete genome</td><td rowspan=1 colspan=1>EU684056</td><td rowspan=1 colspan=1>Human Rhinovirus/Enterovirus</td></tr></table>

# iii. Supplemental Reactivity Information for Influenza Strains of Human, Swine and Avian Origin (analytical testing data and in silico analyses):

Additional analytical reactivity testing was carried out with three swine and avian isolates of Influenza A. The results are presented in the following table:

Swine and Avian Influenza A Reactivity Testing   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate / Strain</td><td rowspan=1 colspan=1>Test Concentration FilmArray Result</td><td rowspan=1 colspan=1>Test Concentration FilmArray Result</td></tr><tr><td rowspan=2 colspan=1>Swine</td><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>Influenza A/Swine/1976/31</td><td rowspan=1 colspan=1>~1 x 106 EID50/mL</td><td rowspan=1 colspan=1>Influenza A H1a</td></tr><tr><td rowspan=1 colspan=1>H1N1</td><td rowspan=1 colspan=1>Influenza A/Swine/Iowa/15/30</td><td rowspan=1 colspan=1>∼1 x 107 EID50/mL</td><td rowspan=1 colspan=1>Influenza A H1a</td></tr><tr><td rowspan=1 colspan=1>Avian</td><td rowspan=1 colspan=1>H1N2</td><td rowspan=1 colspan=1>Kilbourne F63A/NWS/34 (HA) x A/Rockefeller Institute/5/57 (NA)</td><td rowspan=1 colspan=1>14.8 ng RNAb</td><td rowspan=1 colspan=1>Influenza A H1</td></tr></table>

a No reactivity was observed with the $2 0 0 9 \mathrm { H I }$ subtyping assay or other FilmArray RP assays. b Purified and quantified RNA from avian influenza culture was obtained from BEl Resources.

Simulated reactivity of the FilmArray RP Influenza assays with additional influenza strains of human, swine and avian origin was generated using a bioinformatics approach. Assay primer sequences were aligned with GenBank sequences corresponding to the appropriate gene targets and reactivity were predicated based on the number and location of mismatches in the targeted region. For each strain, multiple (3) GenBank IDs were evaluated, corresponding to the gene segments targeted by the FilmArray RP assays (matrix (MA), non-structural (NS), and hemagglutinin (HA)). The strains listed in the following table are predicted to react with the pan-influenza A and H1, H3 or $2 0 0 9 \mathrm { H } 1$ subtyping assays as indicated:

Simulated Reactivity of FilmArray Influenza A Assays with Human, Swine, and Avian Influenza Strains   

<table><tr><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">GenBankID</td><td colspan="1" rowspan="1">SimulatedFilmArray Reactivity</td></tr><tr><td colspan="1" rowspan="20">Human</td><td colspan="1" rowspan="3">H1N1</td><td colspan="1" rowspan="3">A/California/UR06-0393/2007(H1N1)</td><td colspan="1" rowspan="1">CY026540</td><td colspan="1" rowspan="3">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">CY026543</td></tr><tr><td colspan="1" rowspan="1">CY026539</td></tr><tr><td colspan="1" rowspan="3">H1N1-2009</td><td colspan="1" rowspan="3">A/Aalborg/INS133/2009(H1N1)</td><td colspan="1" rowspan="1">CY063606</td><td colspan="1" rowspan="3">Influenza A 2009 H1</td></tr><tr><td colspan="1" rowspan="1">CY063610</td></tr><tr><td colspan="1" rowspan="1">CY063607</td></tr><tr><td colspan="1" rowspan="3">H1N2</td><td colspan="1" rowspan="3">A/New York/297/2003(H1N2)</td><td colspan="1" rowspan="1">CY002668</td><td colspan="1" rowspan="3">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">CY002665</td></tr><tr><td colspan="1" rowspan="1">CY002664</td></tr><tr><td colspan="1" rowspan="3">H2N2</td><td colspan="1" rowspan="3">A/Albany/20/1957(H2N2)</td><td colspan="1" rowspan="1">CY022013</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY022014</td></tr><tr><td colspan="1" rowspan="1">CY022017</td></tr><tr><td colspan="1" rowspan="3">H3N2</td><td colspan="1" rowspan="3">A/Boston/38/2008(H3N2)</td><td colspan="1" rowspan="1">CY044581</td><td colspan="1" rowspan="3">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">CY044584</td></tr><tr><td colspan="1" rowspan="1">CY044580</td></tr><tr><td colspan="1" rowspan="5">H5N1</td><td colspan="1" rowspan="3">A/Cambodia/R0405050/2007(H5N1)</td><td colspan="1" rowspan="1">HQ200572</td><td colspan="1" rowspan="5">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">HQ200573</td></tr><tr><td colspan="1" rowspan="1">FJ225472</td></tr><tr><td colspan="1" rowspan="2">A/Hong Kong/486/97(H5N1)</td><td colspan="1" rowspan="1">AF084281</td></tr><tr><td colspan="1" rowspan="1">AF255368</td></tr><tr><td colspan="1" rowspan="13"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">EY</td><td colspan="1" rowspan="1">AF115289</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="3">H7N2</td><td colspan="1" rowspan="3">A/New York/107/2003(H7N2)</td><td colspan="1" rowspan="1">EU587368</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">EU587373</td></tr><tr><td colspan="1" rowspan="1">EU587374</td></tr><tr><td colspan="1" rowspan="3">H7N3</td><td colspan="1" rowspan="3">A/Canada/rv504/2004(H7N3)</td><td colspan="1" rowspan="1">CY015006</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY015007</td></tr><tr><td colspan="1" rowspan="1">CY015010</td></tr><tr><td colspan="1" rowspan="3">H7N7</td><td colspan="1" rowspan="3">A/Netherlands/219/03(H7N7)</td><td colspan="1" rowspan="1">AY340089</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">AY342422</td></tr><tr><td colspan="1" rowspan="1">AY338459</td></tr><tr><td colspan="1" rowspan="3">H9N2</td><td colspan="1" rowspan="3">A/Hong Kong/1073/99(H9N2)</td><td colspan="1" rowspan="1">AJ404626</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">AJ278647</td></tr><tr><td colspan="1" rowspan="1">AJ278649</td></tr><tr><td colspan="1" rowspan="12">Swine</td><td colspan="1" rowspan="3">H1N1</td><td colspan="1" rowspan="3">A/swine/Wisconsin/1/1971(H1N1)</td><td colspan="1" rowspan="1">CY022414</td><td colspan="1" rowspan="3">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">CY022417</td></tr><tr><td colspan="1" rowspan="1">CY022413</td></tr><tr><td colspan="1" rowspan="6">H1N2</td><td colspan="1" rowspan="3">A/swine/Hong Kong/NS857/2001(H1N2)</td><td colspan="1" rowspan="1">GQ229348</td><td colspan="1" rowspan="3">Influenza A H1</td></tr><tr><td colspan="1" rowspan="1">GQ229350</td></tr><tr><td colspan="1" rowspan="1">GQ229347</td></tr><tr><td colspan="1" rowspan="3">A/swine/Sweden/1021/2009(H1N2)</td><td colspan="1" rowspan="1">GQ495135</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">GQ495136</td></tr><tr><td colspan="1" rowspan="1">GQ495132</td></tr><tr><td colspan="1" rowspan="3">H5N1</td><td colspan="1" rowspan="3">A/swine/East Java/UT6010/2007(H5N1)</td><td colspan="1" rowspan="1">HM440124</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">HM440111</td></tr><tr><td colspan="1" rowspan="1">HM440123</td></tr><tr><td colspan="1" rowspan="17">Avian</td><td colspan="1" rowspan="9">H2N2</td><td colspan="1" rowspan="3">A/chicken/New York/13828-3/1995(H2N2)</td><td colspan="1" rowspan="1">CY014822</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY014825</td></tr><tr><td colspan="1" rowspan="1">CY014821</td></tr><tr><td colspan="1" rowspan="3">A/Japan/305/1957(H2N2)</td><td colspan="1" rowspan="1">CY045804</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY014977</td></tr><tr><td colspan="1" rowspan="1">CY014980</td></tr><tr><td colspan="1" rowspan="3">A/Korea/426/1968(H2N2)</td><td colspan="1" rowspan="1">CY031595</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY031596</td></tr><tr><td colspan="1" rowspan="1">CY031599</td></tr><tr><td colspan="1" rowspan="3">H3N1</td><td colspan="1" rowspan="3">A/blue-winged teal/ALB/452/1983(H3N1)</td><td colspan="1" rowspan="1">CY004635</td><td colspan="1" rowspan="3">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">CY004638</td></tr><tr><td colspan="1" rowspan="1">CY005940</td></tr><tr><td colspan="1" rowspan="3">H3N2</td><td colspan="1" rowspan="3">A/American black duck/NorthCarolina/675-075/2004(H3N2)</td><td colspan="1" rowspan="1">GU051135</td><td colspan="1" rowspan="3">Influenza A(no subtype detected)</td></tr><tr><td colspan="1" rowspan="1">GU051136</td></tr><tr><td colspan="1" rowspan="1">GU051137</td></tr><tr><td colspan="1" rowspan="2">H3N5</td><td colspan="1" rowspan="2">A/mallard/Netherlands/2/1999(H3N5)</td><td colspan="1" rowspan="1">CY060261</td><td colspan="1" rowspan="2">Influenza A H3</td></tr><tr><td colspan="1" rowspan="1">CY060264</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArray Reactivity</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>CY060265</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>H3N6</td><td rowspan=3 colspan=1>A/American blackduck/NewBrunswick/25182/2007(H3N6)</td><td rowspan=1 colspan=1>CY047696</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>CY047697</td></tr><tr><td rowspan=1 colspan=1>CY047700</td></tr><tr><td rowspan=3 colspan=1>H3N7</td><td rowspan=3 colspan=1>A/northernshoveler/California/HKWF1367/2007(H3N7)</td><td rowspan=1 colspan=1>CY033372</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>CY033375</td></tr><tr><td rowspan=1 colspan=1>CY033376</td></tr><tr><td rowspan=3 colspan=1>H3N8</td><td rowspan=3 colspan=1>A/Americanblackduck/Washington/699/1978(H3N8)</td><td rowspan=1 colspan=1>GU052299</td><td rowspan=3 colspan=1>Influenza A H3</td></tr><tr><td rowspan=1 colspan=1>GU052302</td></tr><tr><td rowspan=1 colspan=1>GU052300</td></tr><tr><td rowspan=3 colspan=1>H4N6</td><td rowspan=3 colspan=1>A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td><td rowspan=1 colspan=1>CY063977</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>CY063978</td></tr><tr><td rowspan=1 colspan=1>CY063981</td></tr><tr><td rowspan=9 colspan=1>H5N1</td><td rowspan=3 colspan=1>A/rook/Rostov-on-Don/26/2007(H5N1)</td><td rowspan=1 colspan=1>EU814503</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>EU814504</td></tr><tr><td rowspan=1 colspan=1>EU814507</td></tr><tr><td rowspan=3 colspan=1>A/turkey/VA/505477-18/2007(H5N1)</td><td rowspan=1 colspan=1>GU186509</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>GU186510</td></tr><tr><td rowspan=1 colspan=1>GU186513</td></tr><tr><td rowspan=3 colspan=1>A/chicken/Bangladesh/1151-10/2010(H5N1)</td><td rowspan=1 colspan=1>HQ156765</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>HQ156766</td></tr><tr><td rowspan=1 colspan=1>HQ156764</td></tr><tr><td rowspan=3 colspan=1>H5N2</td><td rowspan=3 colspan=1>A/duck/Pennsylvania/10218/1984(H5N2)</td><td rowspan=1 colspan=1>AB295603</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>AB286120</td></tr><tr><td rowspan=1 colspan=1>AB286652</td></tr><tr><td rowspan=3 colspan=1>H5N3</td><td rowspan=3 colspan=1>A/duck/Singapore/F119/3/1997(H5N3)</td><td rowspan=1 colspan=1>GU052802</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>GU052803</td></tr><tr><td rowspan=1 colspan=1>GU052805</td></tr><tr><td rowspan=3 colspan=1>H6N1</td><td rowspan=3 colspan=1>A/duck/PA/486/1969(H6N1)</td><td rowspan=1 colspan=1>EU743286</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>EU743287</td></tr><tr><td rowspan=1 colspan=1>EU743289</td></tr><tr><td rowspan=3 colspan=1>H6N2</td><td rowspan=3 colspan=1>A/mallard/Czech Republic/15902-17K/2009(H6N2)</td><td rowspan=1 colspan=1>HQ244430</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>HQ244433</td></tr><tr><td rowspan=1 colspan=1>HQ244434</td></tr><tr><td rowspan=3 colspan=1>H7N7</td><td rowspan=3 colspan=1>A/mallard/Korea/GH171/2007(H7N7)</td><td rowspan=1 colspan=1>FJ750872</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>FJ959087</td></tr><tr><td rowspan=1 colspan=1>FJ959090</td></tr><tr><td rowspan=3 colspan=1>H9N2</td><td rowspan=3 colspan=1>A/turkey/Wisconsin/1/1966(H9N2)</td><td rowspan=1 colspan=1>CY014663</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>CY014664</td></tr><tr><td rowspan=1 colspan=1>CY014667</td></tr><tr><td rowspan=2 colspan=1>H10N7</td><td rowspan=2 colspan=1>A/chicken/Germany/N/1949(H10N7)</td><td rowspan=1 colspan=1>GQ176136</td><td rowspan=2 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>GQ176135</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>GenBankID</td><td rowspan=1 colspan=1>SimulatedFilmArray Reactivity</td></tr><tr><td rowspan=4 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Y</td><td rowspan=1 colspan=1>GQ176132</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=3 colspan=1>H11N9</td><td rowspan=3 colspan=1>A/duck/Memphis/546/1974(H11N9)</td><td rowspan=1 colspan=1>CY014691</td><td rowspan=3 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>GQ257441</td></tr><tr><td rowspan=1 colspan=1>CY014687</td></tr></table>

# g. Analytical Specificity/Cross-reactivity Evaluation

An analytical exclusivity study was carried out to assess the potential for false positive results due to cross-reactivity between RP assays and other RP or non-RP organisms. RP organisms and non-RP organisms were tested at high concentrations and the non-RP organism exclusivity panel consisted of organisms that are related to, but distinct from, RP target organisms, or that could be present in specimens collected from the intended test population.

Exclusivity (cross-reactivity) testing was performed at Idaho Technology, Inc. and an external testing site. Samples were prepared by spiking RP and non-RP target organisms into NPS (or simulated NPS) sample matrix at a high concentration $( 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viral targets and $1 0 ^ { 6 } \mathrm { C F U / m L }$ for bacterial targets, or the highest concentration possible based on the organism stock).

The RP target exclusivity panel included all of the primary strains used to establish LoD levels for the system plus additional strains of some common organisms such as Influenza A and B. RP target organisms tested at high concentrations were positive only for the appropriate RP assay(s). No cross-reactivity between RP assays was observed.

The following tables list all of the RP organisms tested at high concentration. The test concentration is listed followed by the multiple of the LoD concentration the test concentration represents.

Summary of Cross-Reactivity Testing with FilmArray RP Target Organisms   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Type / Strain / ID</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Multiple ofLoD Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Serotype 1(Species C)</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">333x</td></tr><tr><td colspan="1" rowspan="9">Bordetella pertussis</td><td colspan="1" rowspan="1">E431</td><td colspan="1" rowspan="9">1.00E+06 CFU/ml</td><td colspan="1" rowspan="9">250x</td></tr><tr><td colspan="1" rowspan="1">A639</td></tr><tr><td colspan="1" rowspan="1">ATCC 8467</td></tr><tr><td colspan="1" rowspan="1">ATCC 9797</td></tr><tr><td colspan="1" rowspan="1">ATCC 51445</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-589</td></tr><tr><td colspan="1" rowspan="1">ATCC 9340</td></tr><tr><td colspan="1" rowspan="1">ATCC 10380</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-1335</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">2.42E+05 copies/mL</td><td colspan="1" rowspan="1">81x</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">HKU1 (type B)Clinical specimen</td><td colspan="1" rowspan="1">2.78E+09 copies/mL</td><td colspan="1" rowspan="1">1,463x</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">NL63NR-470</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">OC43ATCC VR-759</td><td colspan="1" rowspan="1">7.30E+04 TCID50/mL</td><td colspan="1" rowspan="1">122x</td></tr><tr><td colspan="1" rowspan="1">229EATCC VR-740</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,418x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">hMPV-16IA10-2003 A1</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">4,085x</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">3.40E+06 TCID50/mL</td><td colspan="1" rowspan="1">113x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">5,670x</td></tr><tr><td colspan="1" rowspan="9">Influenza A H1N1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">500x</td></tr><tr><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">500x</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td><td colspan="1" rowspan="1">5,000x</td></tr><tr><td colspan="1" rowspan="1">A1/FM/1/47</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">70x</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">24x</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">70x</td></tr><tr><td colspan="1" rowspan="1">Influenza A 2009 H1N1</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">4.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">4,000x</td></tr><tr><td colspan="1" rowspan="8">Influenza A H3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1634x</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">940x</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">20,000x</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">20,000x</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">4.70E+03 TCID50/mL</td><td colspan="1" rowspan="1">940x</td></tr><tr><td colspan="1" rowspan="1">A/MRC 2</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,634x</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,634x</td></tr><tr><td colspan="1" rowspan="8">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td><td colspan="1" rowspan="1">278x</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">5.03E+04 TCID50/mL</td><td colspan="1" rowspan="1">838x</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136 x</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">136x</td></tr><tr><td colspan="1" rowspan="1">B/Florida/07/04</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">1,667x</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">95x</td></tr><tr><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">1,667 x</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Type / Strain / ID</td><td rowspan=1 colspan=3>Test Concentration</td><td rowspan=1 colspan=1>Multiple ofLoD Tested</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1>B/HongKong/5/72</td><td rowspan=1 colspan=3>8.17E+03 TCID50/mL</td><td rowspan=1 colspan=1>136x</td></tr><tr><td rowspan=1 colspan=1>B/Brigit</td><td rowspan=1 colspan=3>3.50E+04 TCID50/mL</td><td rowspan=1 colspan=1>583x</td></tr><tr><td rowspan=9 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>M129</td><td rowspan=1 colspan=3>1.88E+05 TCID50/mL</td><td rowspan=9 colspan=1>6,267 xn/a33,333x</td></tr><tr><td rowspan=1 colspan=1>ATCC 15531</td><td rowspan=1 colspan=3>4.27+05 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>ATCC 15293</td><td rowspan=4 colspan=3>1.00E+06 CCU/ml</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 15377</td></tr><tr><td rowspan=1 colspan=1>ATCC 15492</td></tr><tr><td rowspan=1 colspan=1>ATCC 29085</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>ATCC 29342</td><td rowspan=3 colspan=3></td></tr><tr><td rowspan=1 colspan=1>ATCC 39505</td></tr><tr><td rowspan=1 colspan=1>ATCC 49894</td></tr><tr><td rowspan=4 colspan=1>Parainfluenza Virus</td><td rowspan=1 colspan=1>Type 1</td><td rowspan=1 colspan=3>1.39E+04 TCID50/mL</td><td rowspan=1 colspan=1>28 x</td></tr><tr><td rowspan=1 colspan=1>Type 2</td><td rowspan=1 colspan=3>1.67E+04 TCID50/mL</td><td rowspan=1 colspan=1>1,670 x</td></tr><tr><td rowspan=1 colspan=1>Type 3</td><td rowspan=1 colspan=3>1.00E+05 TCID50/mL</td><td rowspan=1 colspan=1>10,000 x</td></tr><tr><td rowspan=1 colspan=1>Type 4a</td><td rowspan=1 colspan=3>5.67E+03 TCID50/mL</td><td rowspan=1 colspan=1>1.13 x</td></tr><tr><td rowspan=2 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=3>1.39E+04 TCID50/mL</td><td rowspan=1 colspan=1>6,950 x</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=3>2.14E+04 TCID50/mL</td><td rowspan=1 colspan=1>10,700 x</td></tr></table>

The non-RP target exclusivity panel consisted of 26 bacteria, 6 viruses, and 1 fungus (Candida albicans). The organisms included in the non-RP exclusivity panel were selected either because they are related to RP target organisms, are clinically relevant (colonize the upper respiratory tract or cause respiratory symptoms), are common skin flora or laboratory contaminants, or are microorganisms for which much of the population may have been infected (Herpes Simplex Virus, etc.). None of the organisms in the non-RP target exclusivity panel tested positive by the FilmArray RP System assays except for one measles virus strain. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

The following table lists all of the non-RP organisms tested at high concentration. The test concentration is listed:

RP Target Exclusivity Organisms Tested   

<table><tr><td>Bacteria/Fungii</td><td>Strain</td><td>Test Concentration</td></tr><tr><td>Bordetella bronchiseptica</td><td>Clinical isolate</td><td>1.00E+06 CFU/mL</td></tr><tr><td>Bordetella holmesii</td><td>F061</td><td>1.00E+06 CFU/mL</td></tr><tr><td>Bordetella parapertussis</td><td>A747</td><td>1.00E+06 CFU/mL</td></tr><tr><td>Candida albicans</td><td>Zeptometrix #0801504</td><td>1.00E+06 CFU/mL</td></tr><tr><td>Chlamydia trachomatis</td><td>D-UW3</td><td>1.00E+06 IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC 14779CDC [NCTC 10681]</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">1.00+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">8.67E+04 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeacheae</td><td colspan="1" rowspan="1">Long Beach 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">Philadelphia Strain</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11- type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114POG211</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 700825</td><td colspan="1" rowspan="1">1.05E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M1027-type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix #0801512</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">8.43E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618T-strain 960</td><td colspan="1" rowspan="1">5.23E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="2">Bocavirus</td><td colspan="1" rowspan="2">Type 1 Clinical Specimen</td><td colspan="1" rowspan="1">3.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">4.67E+07 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169ATCC VR-538</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes Virus</td><td colspan="1" rowspan="1">Simplex Type 1</td><td colspan="1" rowspan="1">1:40 dilution of stock</td></tr><tr><td colspan="1" rowspan="2">Measles Virus (Rubeola)</td><td colspan="1" rowspan="1">Zeptometrix #0810025CFa</td><td colspan="1" rowspan="1">4.20E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">1.00E+05 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix #810079CF</td><td colspan="1" rowspan="1">5.03E+04 TCID50/mL</td></tr></table>

a This stock produced one false positive Adenovirus result. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

Additional analytical exclusivity testing was carried out with either live isolates or purified genomic RNA of avian host influenza A strains with the following results:

Exclusivity Testing with Avian Influenza Live Isolates or Purified Genomic RNA   

<table><tr><td rowspan=1 colspan=1>Host</td><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Isolate/Strain</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>FilmArray Result</td></tr><tr><td rowspan=8 colspan=1>Avian</td><td rowspan=2 colspan=1>H2N2</td><td rowspan=1 colspan=1>A/Japan/305/57</td><td rowspan=1 colspan=1>3.3 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>Kilbourne F38A/Korea/426/68 (HA,NA) x A/Puerto Rico/8/34</td><td rowspan=1 colspan=1>6.3 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N1</td><td rowspan=1 colspan=1>A/Vietnam/1203/2004 R-H5</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N2</td><td rowspan=1 colspan=1>A/duck/Pennsylvania/10218/84</td><td rowspan=1 colspan=1>2.5 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtype detected)</td></tr><tr><td rowspan=1 colspan=1>H5N3</td><td rowspan=1 colspan=1>KilbourneF181A/duck/Singapore/645/97</td><td rowspan=1 colspan=1>247 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>H7N3</td><td rowspan=1 colspan=1>A/Mallard/Netherlands/12/2000</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>H7N2</td><td rowspan=1 colspan=1>A/NewYork/107/2003</td><td rowspan=1 colspan=1>N/Ab</td><td rowspan=1 colspan=1>Influenza A(no subtypedetected)</td></tr><tr><td rowspan=1 colspan=1>H10N7</td><td rowspan=1 colspan=1>A/chicken/Germany/N/49</td><td rowspan=1 colspan=1>68 ng RNAa</td><td rowspan=1 colspan=1>Influenza A(no subtypedetected)</td></tr></table>

a Purified and quantified RNA from avian influenza cultures was obtained from BEI Resources b Stock virus HA titer from C $\mathrm { O C } = 1 2 8$ . Twenty microliters of virus stock tested.

# i. Supplemental Human, Swine, and Avian Host Influenza A Virus Strains Exclusivity Information (in silico analyses):

Laboratory testing of influenza strains was supplemented with in silico predictions of exclusivity using bioinformatics and sequence alignments between FilmArray RP assay primers and GenBank sequences for the strains shown in the table below. Exclusivity was predicated based on the number of and location of mismatches between assay primers and available strain sequences. For each strain, multiple GenBank IDs were evaluated, corresponding to the genes targeted by the FilmArray RP pan-influenza A and HA subtyping assays. All listed strains are predicted to produce an Influenza A (no subtype detected) FilmArray RP result.

Simulated Exclusivity of FilmArray Influenza A Assays with Human, Swine, and Avian Influenza Strains   

<table><tr><td colspan="1" rowspan="1">Host</td><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">GenBankID</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">PredicatedFilarray RPResults</td></tr><tr><td colspan="1" rowspan="21">Human</td><td colspan="1" rowspan="3">H2N2</td><td colspan="1" rowspan="1">CY022013</td><td colspan="1" rowspan="3">A/Albany/20/1957(H2N2)</td><td colspan="1" rowspan="21">Influenza A (nosubtype detected)</td></tr><tr><td colspan="1" rowspan="1">CY022014</td></tr><tr><td colspan="1" rowspan="1">CY022017</td></tr><tr><td colspan="1" rowspan="6">H5N1</td><td colspan="1" rowspan="1">HQ200572</td><td colspan="1" rowspan="3">A/Cambodia/R0405050/2007(H5N1)</td></tr><tr><td colspan="1" rowspan="1">HQ200573</td></tr><tr><td colspan="1" rowspan="1">FJ225472</td></tr><tr><td colspan="1" rowspan="1">AF084281</td><td colspan="1" rowspan="3">A/Hong Kong/486/97(H5N1)</td></tr><tr><td colspan="1" rowspan="1">AF255368</td></tr><tr><td colspan="1" rowspan="1">AF115289</td></tr><tr><td colspan="1" rowspan="3">H7N2</td><td colspan="1" rowspan="1">EU587368</td><td colspan="1" rowspan="3">A/New York/107/2003(H7N2)</td></tr><tr><td colspan="1" rowspan="1">EU587373</td></tr><tr><td colspan="1" rowspan="1">EU587374</td></tr><tr><td colspan="1" rowspan="3">H7N3</td><td colspan="1" rowspan="1">CY015006</td><td colspan="1" rowspan="3">A/Canada/rv504/2004(H7N3)</td></tr><tr><td colspan="1" rowspan="1">CY015007</td></tr><tr><td colspan="1" rowspan="1">CY015010</td></tr><tr><td colspan="1" rowspan="3">H7N7</td><td colspan="1" rowspan="1">AY340089</td><td colspan="1" rowspan="3">A/Netherlands/219/03(H7N7)</td></tr><tr><td colspan="1" rowspan="1">AY342422</td></tr><tr><td colspan="1" rowspan="1">AY338459</td></tr><tr><td colspan="1" rowspan="3">H9N2</td><td colspan="1" rowspan="1">AJ404626</td><td colspan="1" rowspan="3">A/Hong Kong/1073/99(H9N2)</td></tr><tr><td colspan="1" rowspan="1">AJ278647</td></tr><tr><td colspan="1" rowspan="1">AJ278649</td></tr><tr><td colspan="1" rowspan="6">Swine</td><td colspan="1" rowspan="3">H1N2</td><td colspan="1" rowspan="1">GQ495135</td><td colspan="1" rowspan="3">A/swine/Sweden/1021/2009(H1N2)</td><td colspan="1" rowspan="19">Influenza A (nosubtype detected)</td></tr><tr><td colspan="1" rowspan="1">GQ495136</td></tr><tr><td colspan="1" rowspan="1">GQ495132</td></tr><tr><td colspan="1" rowspan="3">H5N1</td><td colspan="1" rowspan="1">HM440124</td><td colspan="1" rowspan="3">A/swine/EastJava/UT6010/2007(H5N1)</td></tr><tr><td colspan="1" rowspan="1">HM440111</td></tr><tr><td colspan="1" rowspan="1">HM440123</td></tr><tr><td colspan="1" rowspan="13">Avian</td><td colspan="1" rowspan="9">H2N2</td><td colspan="1" rowspan="1">CY014822</td><td colspan="1" rowspan="3">A/chicken/New York/13828-3/1995(H2N2)</td></tr><tr><td colspan="1" rowspan="1">CY014825</td></tr><tr><td colspan="1" rowspan="1">CY014821</td></tr><tr><td colspan="1" rowspan="1">CY045804</td><td colspan="1" rowspan="3">A/Japan/305/1957(H2N2)</td></tr><tr><td colspan="1" rowspan="1">CY014977</td></tr><tr><td colspan="1" rowspan="1">CY014980</td></tr><tr><td colspan="1" rowspan="1">CY031595</td><td colspan="1" rowspan="3">A/Korea/426/1968(H2N2)</td></tr><tr><td colspan="1" rowspan="1">CY031596</td></tr><tr><td colspan="1" rowspan="1">CY031599</td></tr><tr><td colspan="1" rowspan="3">H3N2</td><td colspan="1" rowspan="1">GU051135</td><td colspan="1" rowspan="3">A/American black duck/NorthCarolina/675- 075/2004(H3N2)</td></tr><tr><td colspan="1" rowspan="1">GU051136</td></tr><tr><td colspan="1" rowspan="1">GU051137</td></tr><tr><td colspan="1" rowspan="1">H3N6</td><td colspan="1" rowspan="1">CY047696</td><td colspan="1" rowspan="1">A/American black</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">CY047697</td><td colspan="1" rowspan="2">duck/NewBrunswick/25182/2007(H3N6</td></tr><tr><td colspan="1" rowspan="1">CY047700</td></tr><tr><td colspan="1" rowspan="3">H4N6</td><td colspan="1" rowspan="1">CY063977</td><td colspan="1" rowspan="3">A/blue-winged teal/Minnesota/Sg-00043/2007(H4N6)</td></tr><tr><td colspan="1" rowspan="1">CY063978</td></tr><tr><td colspan="1" rowspan="1">CY063981</td></tr><tr><td colspan="1" rowspan="9">H5N1</td><td colspan="1" rowspan="1">EU814503</td><td colspan="1" rowspan="3">A/rook/Rostov-on-Don/26/2007(H5N1)</td></tr><tr><td colspan="1" rowspan="1">EU814504</td></tr><tr><td colspan="1" rowspan="1">EU814507</td></tr><tr><td colspan="1" rowspan="1">GU186509</td><td colspan="1" rowspan="3">A/turkey/VA/505477-18/2007(H5N1)</td></tr><tr><td colspan="1" rowspan="1">GU186510</td></tr><tr><td colspan="1" rowspan="1">GU186513</td></tr><tr><td colspan="1" rowspan="1">HQ156765</td><td colspan="1" rowspan="3">A/chicken/Bangladesh/1151-10/2010(H5N1)</td></tr><tr><td colspan="1" rowspan="1">HQ156766</td></tr><tr><td colspan="1" rowspan="1">HQ156764</td></tr><tr><td colspan="1" rowspan="3">H5N2</td><td colspan="1" rowspan="1">AB295603</td><td colspan="1" rowspan="3">A/duck/Pennsylvania/10218/1984(H5N2</td></tr><tr><td colspan="1" rowspan="1">AB286120</td></tr><tr><td colspan="1" rowspan="1">AB286652</td></tr><tr><td colspan="1" rowspan="3">H5N3</td><td colspan="1" rowspan="1">GU052802</td><td colspan="1" rowspan="3">A/duck/Singapore/F119/3/1997(H5N3)</td></tr><tr><td colspan="1" rowspan="1">GU052803</td></tr><tr><td colspan="1" rowspan="1">GU052805</td></tr><tr><td colspan="1" rowspan="3">H6N1</td><td colspan="1" rowspan="1">EU743286</td><td colspan="1" rowspan="3">A/duck/PA/486/1969(H6N1)</td></tr><tr><td colspan="1" rowspan="1">EU743287</td></tr><tr><td colspan="1" rowspan="1">EU743289</td></tr><tr><td colspan="1" rowspan="3">H6N2</td><td colspan="1" rowspan="1">HQ244430</td><td colspan="1" rowspan="3">A/mallard/Czech Republic/15902-17K/2009(H6N2)</td></tr><tr><td colspan="1" rowspan="1">HQ244433</td></tr><tr><td colspan="1" rowspan="1">HQ244434</td></tr><tr><td colspan="1" rowspan="3">H7N7</td><td colspan="1" rowspan="1">FJ750872</td><td colspan="1" rowspan="3">A/mallard/Korea/GH171/2007(H7N7)</td></tr><tr><td colspan="1" rowspan="1">FJ959087</td></tr><tr><td colspan="1" rowspan="1">FJ959090</td></tr><tr><td colspan="1" rowspan="3">H9N2</td><td colspan="1" rowspan="1">CY014663</td><td colspan="1" rowspan="3">A/turkey/Wisconsin/1/1966(H9N2)</td></tr><tr><td colspan="1" rowspan="1">CY014664</td></tr><tr><td colspan="1" rowspan="1">CY014667</td></tr><tr><td colspan="1" rowspan="3">H10N7</td><td colspan="1" rowspan="1">GQ176136</td><td colspan="1" rowspan="3">A/chicken/Germany/N/1949(H10N7)</td></tr><tr><td colspan="1" rowspan="1">GQ176135</td></tr><tr><td colspan="1" rowspan="1">GQ176132</td></tr><tr><td colspan="1" rowspan="3">H11N9</td><td colspan="1" rowspan="1">CY014691</td><td colspan="1" rowspan="3">A/duck/Memphis/546/1974(H11N9)</td></tr><tr><td colspan="1" rowspan="1">GQ257441</td></tr><tr><td colspan="1" rowspan="1">CY014687</td></tr></table>

# ii. Supplemental SARS Coronavirus Exclusivity Information (in silico analyses)

In silico analysis of the SARS virus sequence (GenBank ID NC_004718.3) against primers for each of the four non-SARS coronavirus assays (CoV HKU1, CoV NL63, CoV OC43, and $\mathrm { C o V } 2 2 9 \mathrm { E } )$ ) was conducted. When non-SARS coronavirus assay primers were aligned with the corresponding target sequence from SARS virus, each primer contained at least 6 mismatches. Based on these alignments, no reactivity was predicted between SARS virus and the four FilmArray RP non-SARS coronavirus assays.

# h. Assay cut-off:

# i. Melt Windows Determination:

Sequence diversity results in a range of melting temperatures (Tms) of the PCR2 melting curves for each of the target organism amplicons generated by the FilmArray Respiratory Panel (RP).

A mathematical model was used to predict the melting temperatures of all known organism variants (within NCBI databases) for each assay. The distribution of predicted Tms was used to establish an initial melt range for each assay. In addition, the overall system variability, including for instrument, chemistry, pouch lot and other sources of variability, was estimated using a synthetic template melting data set generated on various FilmArray instruments using multiple pouch lots. Historical RP Panel melting data were also considered to establish appropriate melt ranges. In particular, the Analytical Inclusivity data and all available patient specimen data from the clinical evaluation sites were considered. These data sets included overall system variability and potentially included additional organism sequence variants not accounted for in the NCBI databases. Furthermore, data collected at the beta sites were used as an overall check and validation for the proposed melting ranges.

For each of the pathogen assays, the lower and upper limits of the melting window was calculated based on the minimum of the model prediction based on published sequence variation, the clinical evaluation data, and the inclusivity data:

Predicted Tm Upper and Lower Limits for Analytes   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Lower Limit</td><td colspan="1" rowspan="1">Upper Limit</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">77.7</td><td colspan="1" rowspan="1">88.5</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">78.9</td><td colspan="1" rowspan="1">83.3</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">86.0</td><td colspan="1" rowspan="1">91.4</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">73.1</td><td colspan="1" rowspan="1">78.3</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">78.4</td><td colspan="1" rowspan="1">82.8</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">79.2</td><td colspan="1" rowspan="1">83.6</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E3</td><td colspan="1" rowspan="1">78.7</td><td colspan="1" rowspan="1">83.8</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 1</td><td colspan="1" rowspan="1">84.5</td><td colspan="1" rowspan="1">91.2</td></tr><tr><td colspan="1" rowspan="1">Enterovirus - 2</td><td colspan="1" rowspan="1">84.4</td><td colspan="1" rowspan="1">91.1</td></tr><tr><td colspan="1" rowspan="1">FluA - Pan1</td><td colspan="1" rowspan="1">81.2</td><td colspan="1" rowspan="1">89.0</td></tr><tr><td colspan="1" rowspan="1">FluA - Pan2</td><td colspan="1" rowspan="1">75.8</td><td colspan="1" rowspan="1">83.4</td></tr><tr><td colspan="1" rowspan="1">FluA - H1 Pan</td><td colspan="1" rowspan="1">74.0</td><td colspan="1" rowspan="1">81.4</td></tr><tr><td colspan="1" rowspan="1">FluA - 2009 H1</td><td colspan="1" rowspan="1">77.0</td><td colspan="1" rowspan="1">81.6</td></tr><tr><td colspan="1" rowspan="1">FluA - H3</td><td colspan="1" rowspan="1">78.7</td><td colspan="1" rowspan="1">85.7</td></tr><tr><td colspan="1" rowspan="1">FluB</td><td colspan="1" rowspan="1">78.6</td><td colspan="1" rowspan="1">83.7</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">73.4</td><td colspan="1" rowspan="1">79.7</td></tr><tr><td colspan="1" rowspan="1">HRV - 1</td><td colspan="1" rowspan="1">80.4</td><td colspan="1" rowspan="1">89.8</td></tr><tr><td colspan="1" rowspan="1">HRV - 2</td><td colspan="1" rowspan="1">79.9</td><td colspan="1" rowspan="1">89.6</td></tr><tr><td colspan="1" rowspan="1">HRV - 3</td><td colspan="1" rowspan="1">79.4</td><td colspan="1" rowspan="1">89.1</td></tr><tr><td colspan="1" rowspan="1">HRV - 4</td><td colspan="1" rowspan="1">79.9</td><td colspan="1" rowspan="1">90.6</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">75.9</td><td colspan="1" rowspan="1">80.0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">76.3</td><td colspan="1" rowspan="1">81.0</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">80.3</td><td colspan="1" rowspan="1">86.3</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">78.7</td><td colspan="1" rowspan="1">83.4</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">74.9</td><td colspan="1" rowspan="1">79.9</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">77.2</td><td colspan="1" rowspan="1">83.3</td></tr></table>

For each of the control assays, the melting windows specified were presented in the following table:

Predicted Tm Upper and Lower Limits for Controls   

<table><tr><td rowspan=1 colspan=1>Control Assay</td><td rowspan=1 colspan=1>Lower Limit</td><td rowspan=1 colspan=1>Upper Limit</td></tr><tr><td rowspan=1 colspan=1>hRNA</td><td rowspan=1 colspan=1>79.5</td><td rowspan=1 colspan=1>83.5</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>73.6</td><td rowspan=1 colspan=1>77.6</td></tr><tr><td rowspan=1 colspan=1>PCR2</td><td rowspan=1 colspan=1>74.5</td><td rowspan=1 colspan=1>78.5</td></tr><tr><td rowspan=1 colspan=1>Yeast RNA</td><td rowspan=1 colspan=1>80.6</td><td rowspan=1 colspan=1>84.6</td></tr></table>

# ii. Melt Windows Validation

The validation of the melting windows was done in two steps. First, the Tms for the data collected at the beta sites were compared to the melting windows proposed. This comparison did not identify any Tm outside of the proposed ranges. Second, as part of the FilmArray RP Melt Detector Tuning, multiple data sets collected by the IVD and R&D groups and data collected at the clinical evaluation sites were re-analyzed using the proposed melting ranges. This validation did not identify any missed calls as a direct result of the window location.

# i. Interfering Substances:

An interference study was carried out to evaluate the influence of potential interfering substances on the accuracy of test results obtained with the FilmArray RP System. Four different organism mixes (refer to the tables below) containing FilmArray RP analytes were spiked into a simulated NPS (sNPS) sample matrix (human epithelial cells in VTM) at 5x their respective LoDs. The 5x LoD organism concentration was chosen to be near the analyte LoD but also to provide consistent results for sample-to-sample comparison.

Mixes for Interference Studies   

<table><tr><td rowspan=1 colspan=1>Mix 1</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>1,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>4 TCID50/mL</td><td rowspan=1 colspan=1>20 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H1</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=1 colspan=1>1,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>2 TCID50/mL</td><td rowspan=1 colspan=1>10 TCID50mL</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>30 TCID50/mL</td><td rowspan=1 colspan=1>150 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 2</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>1.9 x 106 genome copies/mL</td><td rowspan=1 colspan=1>9.5 x 106 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>30,000 TCID50/mL</td><td rowspan=1 colspan=1>150,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>5,000 TCID50/mL</td><td rowspan=1 colspan=1>25,000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>4,000 genome copies/mL</td><td rowspan=1 colspan=1>20,000 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 3</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>4000 CFU/mL</td><td rowspan=1 colspan=1>20,000 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>5 TCID50/mL</td><td rowspan=1 colspan=1>25 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Mix 4</td><td rowspan=1 colspan=1>LoD</td><td rowspan=1 colspan=1>5x LoD</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>3000 genome copies/mL</td><td rowspan=1 colspan=1>15,000 genome copies/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>600 TCID50/mL</td><td rowspan=1 colspan=1>3000 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1 TCID50/mL</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>500 TCID50/mL</td><td rowspan=1 colspan=1>2,500 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>50 TCID50/mL</td></tr></table>

The potentially interfering substances tested in this study included substances that are normally found in the nasopharynx or may be introduced into NPS specimens during specimen collection. Endogenous substances like blood, mucin, human genomic DNA and various infectious microorganisms were evaluated. Exogenous substances that may be introduced into NPS specimens before or during sample collection, such as medication, treatments, or topical applications for treating symptoms associated with respiratory infections, were also included in this study.) In addition, materials used to collect (swabs) or store (viral transport medium) NPS specimens or substances that could be introduced from the clinical laboratory environment (i.e. bleach, etc.) were tested as potential techniquespecific interfering substances. The substances were spiked at a level predicted to be above the concentration of the substance likely to be found in an authentic NPS specimen.

The following tables list the test substances at final concentrations used in this study:

# Endogenous Substances

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EN1</td><td rowspan=1 colspan=1>Blood (with Na Citrate)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>none</td></tr><tr><td rowspan=1 colspan=1>EN2</td><td rowspan=1 colspan=1>Mucin (bovine submaxillary gland, type I-S)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr></table>

Endogenous Substances – Human Genomic DNA   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>Test Concentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>DNA1</td><td rowspan=2 colspan=1>Human genomic DNA 200ng/uL</td><td rowspan=1 colspan=1>None</td><td rowspan=1 colspan=1>20 ng/uL</td><td rowspan=1 colspan=1>none</td></tr><tr><td rowspan=1 colspan=1>DNA2</td><td rowspan=2 colspan=1>Human genomic DNA 200ng/uL</td><td rowspan=1 colspan=1>20 ng/uL</td><td rowspan=1 colspan=1>2 ng/uL</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr><tr><td rowspan=1 colspan=1>DNA3</td><td rowspan=1 colspan=1>2 ng/uL</td><td rowspan=1 colspan=1>0.2 ng/uL</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr></table>

Endogenous Substances – Potentially Interfering Microorganisms   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Organism</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Lot Number</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=1 colspan=1>MO1</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>Zeptometrix#0810040ACF</td><td rowspan=1 colspan=1>304756</td><td rowspan=1 colspan=1>2.8 x 104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO2</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>1A</td><td rowspan=1 colspan=1>305067</td><td rowspan=1 colspan=1>1.1 x 104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO3</td><td rowspan=1 colspan=1>Influenza A /2009 H1N1</td><td rowspan=1 colspan=1>A/Swine/NY/03/2009</td><td rowspan=1 colspan=1>305849</td><td rowspan=1 colspan=1>1.0 x 105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>MO4</td><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>COL</td><td rowspan=1 colspan=1>304535</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MO5</td><td rowspan=1 colspan=1>Neisseria meningitidis</td><td rowspan=1 colspan=1>M1027</td><td rowspan=1 colspan=1>304320</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>MO6</td><td rowspan=1 colspan=1>Corynebacterium diptheriae</td><td rowspan=1 colspan=1>ATCC14779</td><td rowspan=1 colspan=1>305297</td><td rowspan=1 colspan=1>1.0 x 106 CFU/mL</td></tr></table>

# Exogenous Substances

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=1>Test Substance</td><td rowspan=1 colspan=1>TestConcentration</td><td rowspan=1 colspan=1>Solvent</td></tr><tr><td rowspan=1 colspan=1>EX1</td><td rowspan=1 colspan=1>Tobramycin (inhaled/systemic antibiotic)</td><td rowspan=1 colspan=1>0.6 mg/mL</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr><tr><td rowspan=1 colspan=1>EX2</td><td rowspan=1 colspan=1>Mupirocin(active ingredient in anti-bacterial ointment)</td><td rowspan=1 colspan=1>2% w/v</td><td rowspan=1 colspan=1>Reagent Grade Water</td></tr><tr><td rowspan=1 colspan=1>EX4</td><td rowspan=1 colspan=1>Saline Nasal Spray with Preservatives (0.65% NaClPhenylcarbinol Benzalkonium Chloride)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX5</td><td rowspan=1 colspan=1>Nasal Decongestant SprayOxymetazoline HC1 0.05%(also contains Benzalkonium chloride, menthol,eucalyptol, camphor, benzyl alcohol and phosphatebuffers)</td><td rowspan=1 colspan=1>1% v/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX6</td><td rowspan=1 colspan=1>Analgesic ointment (Vicks®VapoRub®)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX7</td><td rowspan=1 colspan=1>Petroleum Jelly(Vaseline®)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>EX8</td><td rowspan=1 colspan=1>Smokeless tobacco(Snuff/Nasal Tobacco)</td><td rowspan=1 colspan=1>1% w/v</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Technique Specific Substances   

<table><tr><td rowspan=1 colspan=1>IS #</td><td rowspan=1 colspan=2>Test Substance</td><td rowspan=1 colspan=1>Test Concentration</td></tr><tr><td rowspan=3 colspan=1>TS1</td><td rowspan=3 colspan=2>Bleach</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>2% v/v</td></tr><tr><td rowspan=1 colspan=1>5% v/v</td></tr><tr><td rowspan=1 colspan=1>TS2</td><td rowspan=1 colspan=2>Disinfecting wipes</td><td rowspan=1 colspan=1>½/4 - ½2 in{2</td></tr><tr><td rowspan=1 colspan=1>TS3</td><td rowspan=1 colspan=2>Ethanol</td><td rowspan=1 colspan=1>7% v/v</td></tr><tr><td rowspan=1 colspan=1>TS4</td><td rowspan=1 colspan=2>DNAzap™M</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=1 colspan=1>TS5</td><td rowspan=1 colspan=2>RNaseOUTTM</td><td rowspan=1 colspan=1>1% v/v</td></tr><tr><td rowspan=4 colspan=1>TS6</td><td rowspan=4 colspan=1>Swabs</td><td rowspan=1 colspan=1>Copan 168C (rayon/twisted aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Copan FloQ (flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Copan 175KS01 (polyester/aluminum shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=1 colspan=1>Millipore 519CS01M (flocked nylon/plastic shaft)</td><td rowspan=1 colspan=1>1 swab</td></tr><tr><td rowspan=5 colspan=1>TS7</td><td rowspan=5 colspan=1>ViralTransportMedia</td><td rowspan=1 colspan=1>Remel M4</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M4-RT</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M5</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Remel M6</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Copan UTM</td><td rowspan=1 colspan=1>100%</td></tr></table>

On each day of testing, one specimen per organism mix was not spiked with any test substance to serve as a positive control. For biological test substances that may contain genetic material (such as blood, mucin, DNA, microorganisms), one negative specimen (blank sNPS sample matrix with no organism mix) was spiked with only the test substance (negative control) to evaluate the potential for false positive results due to the test substance itself. For each endogenous, exogenous, and technique-specific test substance, one specimen per organism mix was spiked with the appropriate amount of test substance. All specimens were tested with the FilmArray RP System on the day they were prepared.

None of the potentially interfering endogenous, exogenous, or technique-specific substances tested in this study were found to affect the accuracy of respiratory pathogen detection in simulated NPS specimens using the FilmArray RP System.

In addition to the analytical interference substances study, the first phase prospective clinical dataset was analyzed for evidence of the effects of potentially interfering medications (prescription and over-the-counter; OTC) administered to or taken by the study population. If clinically significant interference from medications had occurred, it would be expected that the analyte detection rate in the medicated population would be lower than that in the non-medicated population. This effect was not observed in the clinical dataset. One or more analytes was detected by the reference/comparator methods in 118 of 257 $( 4 5 . 9 \% )$ nonmedicated individuals. The FilmArray RP system correctly identified all analytes in 112 $( 9 4 . 9 \% )$ of these individuals. One or more analytes was detected by the reference/comparator methods in 342 of 596 $( 5 7 . 4 \% )$ medicated individuals. The FilmArray

RP system correctly identified all analytes in 331 $( 9 6 . 8 \% )$ of these individuals. The table below lists the medications administered to or taken by the enrolled study population (selfreported and/or recorded from patient charts).

Medications Used by Enrolled Study Population   

<table><tr><td rowspan=1 colspan=1>Acetaminophen</td><td rowspan=1 colspan=1>Clindamycin</td><td rowspan=1 colspan=1>Lansoprazole</td><td rowspan=1 colspan=1>Phenobarbital</td></tr><tr><td rowspan=1 colspan=1>Albuterol</td><td rowspan=1 colspan=1>Co-trimoxazole</td><td rowspan=1 colspan=1>Levalbuterol</td><td rowspan=1 colspan=1>Phenylephrine</td></tr><tr><td rowspan=1 colspan=1>Alendronate</td><td rowspan=1 colspan=1>Cough drops/syrups(variousOTC</td><td rowspan=1 colspan=1>Levothyroxine</td><td rowspan=1 colspan=1>Pilocarpine</td></tr><tr><td rowspan=1 colspan=1>Alprazolam</td><td rowspan=1 colspan=1>Cyproheptadine</td><td rowspan=1 colspan=1>Lisinopril</td><td rowspan=1 colspan=1>Pravachol (Pravastatin)</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin</td><td rowspan=1 colspan=1>Deferasirox</td><td rowspan=1 colspan=1>Loperamide</td><td rowspan=1 colspan=1>Prednisolone</td></tr><tr><td rowspan=1 colspan=1>Amoxicillin Clavulanate</td><td rowspan=1 colspan=1>Dextromethorphan</td><td rowspan=1 colspan=1>Loratadine</td><td rowspan=1 colspan=1>Prednisone</td></tr><tr><td rowspan=1 colspan=1>Azithromycin</td><td rowspan=1 colspan=1>Dextromethorphanhydrobromide</td><td rowspan=1 colspan=1>Lorazepam</td><td rowspan=1 colspan=1>Propranolol</td></tr><tr><td rowspan=1 colspan=1>Baclofen</td><td rowspan=1 colspan=1>Diphenhydramine</td><td rowspan=1 colspan=1>Metoclopramide</td><td rowspan=1 colspan=1>Pseudoephedrine</td></tr><tr><td rowspan=1 colspan=1>Benzonatate</td><td rowspan=1 colspan=1>Doxylamine succinate</td><td rowspan=1 colspan=1>Metolazone</td><td rowspan=1 colspan=1>Ranitidine</td></tr><tr><td rowspan=1 colspan=1>Betamethasone</td><td rowspan=1 colspan=1>Electrolyte solutions(rehydration salts)</td><td rowspan=1 colspan=1>Metoprolol</td><td rowspan=1 colspan=1>Saline NasalDrops/Sprays(variousOTC)</td></tr><tr><td rowspan=1 colspan=1>Brimonidine</td><td rowspan=1 colspan=1>Enoxaparin</td><td rowspan=1 colspan=1>Midazolam</td><td rowspan=1 colspan=1>Salmeterol</td></tr><tr><td rowspan=1 colspan=1>Budesonide</td><td rowspan=1 colspan=1>Eszopiclone</td><td rowspan=1 colspan=1>Montelukast</td><td rowspan=1 colspan=1>Simethicone</td></tr><tr><td rowspan=1 colspan=1>Budesonide/FormoterolFumarate Dihydrate</td><td rowspan=1 colspan=1>Fentanyl</td><td rowspan=1 colspan=1>Naproxen</td><td rowspan=1 colspan=1>Sirolimus</td></tr><tr><td rowspan=1 colspan=1>Bupropion hydrochloride</td><td rowspan=1 colspan=1>Fluticasone</td><td rowspan=1 colspan=1>Nizatidine</td><td rowspan=1 colspan=1>Spironolactone</td></tr><tr><td rowspan=1 colspan=1>Carvedilol</td><td rowspan=1 colspan=1>Furosemide</td><td rowspan=1 colspan=1>Omeprazole</td><td rowspan=1 colspan=1>Temazepam</td></tr><tr><td rowspan=1 colspan=1>Cefdinir</td><td rowspan=1 colspan=1>Gabapentin</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>Tiotropium</td></tr><tr><td rowspan=1 colspan=1>Cefepime</td><td rowspan=1 colspan=1>Gemfibrozil</td><td rowspan=1 colspan=1>Oxybutynin</td><td rowspan=1 colspan=1>Ursodiol</td></tr><tr><td rowspan=1 colspan=1>Ceftriaxone</td><td rowspan=1 colspan=1>Guaifenesin</td><td rowspan=1 colspan=1>Pancrelipase</td><td rowspan=1 colspan=1>Valacyclovir</td></tr><tr><td rowspan=1 colspan=1>Cephrazole</td><td rowspan=1 colspan=1>Hydrocodone</td><td rowspan=1 colspan=1>Pantoprazole</td><td rowspan=1 colspan=1>Vancomycin</td></tr><tr><td rowspan=1 colspan=1>Cetirizine</td><td rowspan=1 colspan=1>Ibuprofen</td><td rowspan=1 colspan=1>Penicillin</td><td rowspan=1 colspan=1>Vitamins/Multivitamins/Minerals (various OTC)</td></tr><tr><td rowspan=1 colspan=1>Chlorpheniramine</td><td rowspan=1 colspan=1>Inderal (Propranolol)</td><td rowspan=1 colspan=1>Pentamidine</td><td rowspan=1 colspan=1>Voriconazole</td></tr><tr><td rowspan=1 colspan=1>Ciprofloxacin</td><td rowspan=1 colspan=1>Isosorbide</td><td rowspan=1 colspan=2></td></tr></table>

# i. FluMist® Nasal Influenza Vaccine (MedImmune, 2009-2010 version)

Simulated nasopharyngeal swab (sNPS) specimens were spiked with dilutions of FluMist nasal influenza vaccine (MedImmune, 2009-2010 version), resulting in final test concentrations of $10 \%$ , $1 \%$ and $0 . 1 \%$ (v/v) FluMist. Each specimen was tested with the FilmArray RP system to determine if the vaccine material would react with any of the FilmArray RP assays. If reactivity was observed at any or all dilutions, additional dilutions were tested until no reactivity was observed.

FluMist® Influenza Vaccine Live, Intranasal; Intranasal Spray, 2009-2010 Formula was provided by the manufacturer (MedImmune). The material was provided as individual sprayers containing a single $0 . 2 \mathrm { m L }$ dose. Each dose contains approximately 106.5-7.5 FFU (fluorescence focus units) each of three live, attenuated influenza virus reassortants

including: A/SouthDakota/6/2007 (H1N1) (A/Brisbane/59/2007-like), A/Uraguay/716/2007 (H3N2) (A/Brisbane/10/2007- like), and B/Brisbane/60/2008. Non-viral components of the vaccine listed in the provided materials include: monosodium glutamate, gelatin, arginine, sucrose, gentamicin sulfate, monobasic potassium phosphate and dibasic potassium phosphate. (Note: For the 2009-2010 respiratory season, a separate 2009 H1N1 variant vaccine was prepared and administered. The 2009 H1N1 variant vaccine material was not provided by the manufacturer for testing with the FilmArray RP system.)

Positive results were obtained for Influenza $\mathrm { A } / \mathrm { H } 1$ , Influenza $\mathrm { A } / \mathrm { H } 3$ , and Influenza B in each of the samples tested at $10 \%$ , $1 \%$ and $0 . 1 \%$ (v/v) FluMist®. Additional dilutions were made and tested until each of the Influenza viruses could no longer be detected by the FilmArray RP system $0 . 0 0 0 1 \%$ for Influenza A H1, $0 . 0 0 0 0 1 \%$ for Influenza A H3 and $0 . 0 0 0 0 0 1 \%$ for Influenza B). There was no reactivity with the Influenza $\operatorname { A } / 2 0 0 9 \operatorname { H } 1 \mathrm { N } 1$ subtyping assay, which specifically detects the pandemic $2 0 0 9 \mathrm { A } / \mathrm { H } 1$ variant and was not included in the $2 0 0 9 { - } 2  { \mathrm { 0 1 0 } }  { \mathrm { F l u M i s t } ^ { \textregistered } }$ formulation.

This reactivity is documented in the FilmArray RP System product package insert as a limitation, i.e., “The performance of the FilmArray RP has not been established in individuals who received influenza vaccine. Recent administration of a nasal influenza vaccine may cause false positive results for Influenza A and/or Influenza $B _ { \cdot }$ .”

# j. Carry-Over Contamination

An analytical carry-over study was carried out to demonstrate that when recommended laboratory practices are followed, there is little risk of false positive results caused by carryover or cross-contamination in the FilmArray Respiratory Panel (RP) System. The data for this study were generated as part the exclusivity testing examining the potential for crossreactivity between Respiratory Panel targets and assays. The majority of testing was performed by new users at a contracted laboratory as part of an early performance evaluation of the FilmArray RP system. Multiple spiked NPS (or simulated NPS) samples containing high levels of organism (approximately $\mathrm { i } 0 ^ { 5 }$ for viruses) were tested over several days using 2 FilmArray instruments. Each sample was prepared and loaded into a RP pouch one at a time in a work area (biosafety cabinet or hood) that was separated from the location of the FilmArray instrument(s), and regular cleaning of the work area and pouch loading station were carried out according to the recommended testing procedures. Consecutive testing of the same organism was discouraged in order to maximize the potential for detecting false positive results due to carryover from a previously tested sample. Over 60 RP target organism spiked samples were tested in a total of 63 valid runs, and at least half of the samples were tested subsequently to a sample containing a different target organism at high concentration. No false positive results were reported ( $100 \%$ negative agreement). The consecutive set-up and testing of samples containing high concentrations of different target organisms demonstrated that these practices are effective in preventing false positive results due to carryover between samples.

Because the PCR2 array in each FilmArray pouch contains over 100 individual PCR reaction wells in close proximity, the possibility of well-to-well carryover was also examined. Individual false positive melt curves could be generated if (1) reaction material (template and rehydrated assay specific primers) carried over into an adjacent well early in second stage PCR cycling, or (2) amplicon from one well entered a neighboring well prior to melt analysis. The 63 valid test results described above (over 6000 reaction wells with no false positive organism or assay results) were examined well-by-well and 4 false positive melt curves were observed as a result of carryover between neighboring wells. Each carryover event produced a single positive reaction well per assay and had no effect on the overall accuracy of the FilmArray RP results. This well-to-well carryover analysis resulted in an overall $\%$ negative agreement of $9 9 . 9 \%$ with the $9 5 \% \mathrm { C I }$ of $9 9 . 9 \%$ to $1 0 0 . 0 \%$ (5342/5346 expected negative wells).

Combined, these results indicated that when proper laboratory practices are followed, the risk of false positive FilmArray RP test results due to sample cross- contamination or wellto-well carryover is insignificant.

# k. Simulated NPS Sample Matrix Validation Study

A nasopharyngeal swab (NPS) is the intended sample type to be used with the FilmArray RP system. A NPS is collected from an individual by inserting a flexible, fine-shafted synthetic swab into the nostril and back to the nasopharynx. After a few seconds, the swab is slowly withdrawn with a rotating motion and the swab is placed into a vial containing approximately $3 ~ \mathrm { m L }$ of viral transport medium (VTM). Once in VTM, vortexing helps to release the material collected by the swab (human cells, viral particles, bacteria, secretions, etc.) into the medium, which is then used as the test sample.

A series of studies was required to demonstrate the analytical performance characteristics of the FilmArray RP system. For these types of studies, the ideal sample matrix is NPS collected from healthy individuals that does not contain nucleic acid from any of the pathogens detected by the panel. Initially, sample matrix for contrived samples was obtained by collecting NPS from donors at Idaho Technology, Inc (ITI). Each donor claimed to be healthy and free of symptoms associated with respiratory infection on the day the NPS was collected. Individual NPS were screened with the FilmArray RP system prior to use in the studies, to confirm that the sample matrix was negative for RP organisms.

This process began in the early fall of 2009, at the onset of the typical respiratory season, and it was found during screening that respiratory pathogens could be detected in some percentage of healthy donor NPS. For example, over 2 months of collection (November – December 2009), about $9 \%$ of the screened NPS were positive for one or more FilmArray RP analytes. Some weeks, the rate was as high as $2 6 \%$ . This pattern continued through the respiratory season with a positivity rate of nearly $12 \%$ from January to March 2010. Furthermore, even after the initial screening for negativity, false positive results were occasionally detected in contrived samples and the false positive organism could often be traced to the sample matrix.

Respiratory Panel Organisms Detected in Healthy Donor NPS Screening   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>#Pos</td><td rowspan=1 colspan=1>#Total</td><td rowspan=1 colspan=1>%Pos</td><td rowspan=1 colspan=1>HRV</td><td rowspan=1 colspan=1>FluA2009H1N1</td><td rowspan=1 colspan=1>CoVHKU1</td><td rowspan=1 colspan=1>CoV0C43</td><td rowspan=1 colspan=1>CoV229E</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>PIV 1</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>Adeno</td></tr><tr><td rowspan=1 colspan=1>Nov-Dec2009</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>234</td><td rowspan=1 colspan=1>8.55%</td><td rowspan=1 colspan=1>11a</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3$</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Jan-Mar2010</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>159</td><td rowspan=1 colspan=1>11.95%</td><td rowspan=1 colspan=1>10b</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4b.c</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Total(% of total)</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>393</td><td rowspan=1 colspan=1>9.9%</td><td rowspan=1 colspan=1>21a,b(5.3%)</td><td rowspan=1 colspan=1>4(1.0%)</td><td rowspan=1 colspan=1>4b.c(1.0%)</td><td rowspan=1 colspan=1>4b.c(1.0%)</td><td rowspan=1 colspan=1>3(0.8%)</td><td rowspan=1 colspan=1>2(0.5%)</td><td rowspan=1 colspan=1>2(0.5%)</td><td rowspan=1 colspan=1>1(0.3%)</td><td rowspan=1 colspan=1>1(0.3%)</td></tr></table>

a 1 sample represents co-infection of Rhinovirus / CoV-OC43 b 1 sample represents co-infection of Rhinovirus / CoV-HKU1 c 1 sample represents co-infection of /cross-reactivity: CoV HKU1/CoV OC43

The NPS being screened were collected from donors at ITI, following an IRB- approved protocol. Under the terms of the protocol, the NPS specimens were de- identified. No personal information was collected from the donors and no test results could be traced back to the donor. As a result, if a NPS was positive for one or more RP organisms during screening, it was not possible to follow-up with the donor to inquire about signs of infection prior to or in the days following the date of donation.

Given the very large number of contrived samples needed for analytical studies, it became apparent that use of a sample matrix of NPS collected from donors would be problematic. Most importantly, the true negativity of the matrix could not be adequately confirmed, which placed the accuracy and integrity of analytical studies at risk. Consequently, the use of a simulated NPS matrix was investigated. The intent was for the simulated NPS to resemble the content of an actual NPS as closely as possible, while ensuring that the simulated NPS was free of respiratory pathogens and did not alter the performance of the system.

Ten-fold dilutions of HeLa cells in VTM were prepared (from a starting concentration of 2 $\mathrm { \Delta x \ 1 0 ^ { 5 } c e l l s / m L }$ and tested with the FilmArray system both fresh and after being stored frozen $\left( - 2 0 ^ { \circ } \mathrm { C } \right)$ for one week. Results were compared to NPS collected from healthy donors and an appropriate cell concentration for the simulated NPS was chosen by comparison of $\mathrm { C p }$ values for the hRNA assay (targets mRNA of the human beta-catenin gene) between the true and artificial sample matrices. Once a human cell concentration was selected, contrived samples containing multiple live respiratory pathogens at or near the estimated system LoD were prepared in both NPS and simulated NPS sample matrices. The organism mixes included the following:

Composition of Organism Mixes   

<table><tr><td colspan="1" rowspan="1">Mix 1</td><td colspan="1" rowspan="1">LoD</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">300 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">4 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">2 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H1</td><td colspan="1" rowspan="1">200 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">300 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mix 3</td><td colspan="1" rowspan="1">LoD</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">4000 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">5 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza A/H3</td><td colspan="1" rowspan="1">5 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">60 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">100 TCID50/mLa</td></tr><tr><td colspan="1" rowspan="1">Mix 4</td><td colspan="1" rowspan="1">LoD</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">2000 genome copies/mL b</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">600 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1 TCID550/mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">500 TCID50mL</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">10 TCID50/mL</td></tr></table>

a The system LoD for Parainfluenza Virus 3 was confirmed at $1 0 \mathrm { T C I D } 5 0 / \mathrm { m L }$ after completion of this study. b The system LoD for $C$ pneumoniae was confirmed at 3000 genome copies $/ \mathrm { m L }$ after completion of this study.

Crossing point $\displaystyle ( \mathrm { C p } )$ values of amplification curves generated for all positive assays were also used to compare results from samples prepared in NPS and simulated NPS sample matrices. (Note: Though the use of $\mathrm { C p }$ values in this study was helpful in making comparisons, since the FilmArray RP is a qualitative rather than quantitative system, and the FilmArray RP uses two-stage nested PCR: Cps are therefore inherently more variable than Cps in traditional real-time PCR systems, the significance of these numbers can not be over-interpreted.). A difference of 3 cycles or less was considered equivalent when comparing mean Cp values between NPS and simulated NPS for the hRNA and organism assays.

All samples tested (NPS and simulated NPS) were negative for all panel organisms and positive amplification and melt curves were recorded for the hRNA assay. Average Cp values for the hRNA assay were calculated for each sample type and plotted for comparison. The Cp values observed for the NPS (20.5 –27.1) were consistent with historical data from NPS screening. The average Cp of the hRNA assay was similar for fresh and frozen simulated NPS dilutions. The average Cp for NPS was most similar to the Cp values obtained with simulated NPS prepared at a concentration of $2 \mathrm { ~ x ~ } 1 0 ^ { 3 }$ cells/mL. Based on this analysis, a concentration of $2 \mathrm { \dot { ~ x } 1 0 ^ { 3 } H e L a c e l l s / m L }$ in VTM was selected as the composition of simulated NPS.

Comparison of results for FilmArray RP assays between NPS and simulated NPS sample matrices is presented below:

Comparison of Results for FilmArray RP Assays between NPS and sNPS Sample Matrices.   

<table><tr><td rowspan=1 colspan=1># Positive/Total</td><td rowspan=1 colspan=1>Adeno</td><td rowspan=1 colspan=1>B.pertussis</td><td rowspan=1 colspan=1>C.pneumoniae</td><td rowspan=1 colspan=1>Cov NL63</td><td rowspan=1 colspan=1>CoV 229E</td><td rowspan=1 colspan=1>CoV OC43</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>HRV</td></tr><tr><td rowspan=1 colspan=1>NPS Control</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>Simulated NPStest</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>4/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>3/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>Δ Mean Cp(SimulatedNPS - NPS)</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.09</td><td rowspan=1 colspan=1>-1.29</td><td rowspan=1 colspan=1>-0.51</td><td rowspan=1 colspan=1>0.84</td><td rowspan=1 colspan=1>0.95</td><td rowspan=1 colspan=1>1.21</td><td rowspan=1 colspan=1>0.09 -0.53a</td></tr><tr><td rowspan=1 colspan=1># Positive/Total</td><td rowspan=1 colspan=1>InfluenzaA H1</td><td rowspan=1 colspan=1>InfluenzaA H3</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>M.pneumoniae</td><td rowspan=1 colspan=1>PIVI</td><td rowspan=1 colspan=1>PIV2</td><td rowspan=1 colspan=1>PIV3</td><td rowspan=4 colspan=1></td></tr><tr><td rowspan=1 colspan=1>NPS Control</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>Simulated NPStest</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>Δ Mean Cp(SimulatedNPS - NPS)</td><td rowspan=1 colspan=1>2.27b</td><td rowspan=1 colspan=1>1.50b</td><td rowspan=1 colspan=1>-0.70</td><td rowspan=1 colspan=1>-0.36</td><td rowspan=1 colspan=1>-0.28</td><td rowspan=1 colspan=1>-0.05</td><td rowspan=1 colspan=1>-1.42</td></tr></table>

a Range of delta mean Cp for the HRV I, HRV2, HRV3, and HRV4 assays. b The delta mean $\mathrm { C p }$ shown is for the FluA-HI and FluA-H3 subtyping assays only. Delta mean $\mathrm { C p }$ was 1.48 for the FluA-panl assay and 1.07 for the FluA-pan2 assay.

The study generated no false positive results, and the results demonstrated that the sensitivity of the FilmArray RP system was comparable between samples prepared in nasopharyngeal swab (NPS) and simulated NPS sample matrices. This sample matrix was then used in analytical performance studies of the FilmArray RP system in place of NPS collected from donors. Prior to use, all freshly prepared simulated NPS were screened with the FilmArray RP system to confirm that the hRNA assay provides the expected Cp values and that the sample matrix was negative for all RP organisms.

# l. Comparator Assays Analytical Validation Studies

In the prospective clinical trial for the FilmArray RP System, the FilmArray results were compared to the results from standard viral culture and fluorescent antibody testing performed by the clinical sites for several of the common viruses (Adenovirus, Influenza A, Influenza B, Parainfluenza Virus 3, and Respiratory Syncytial Virus). The majority of the remaining pathogens required 2 PCR comparator assays which were tested directly from the NPS specimens. The exception to this was Influenza A subtyping, which was tested from viral culture material with 1 assay per virus:

Comparator PCR Testing Scheme   

<table><tr><td colspan="1" rowspan="1">Organism/Virus</td><td colspan="1" rowspan="1">Comparator PCR Tests</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">FluA - H1 subtyping</td><td colspan="1" rowspan="1">1 PCR test on culture</td></tr><tr><td colspan="1" rowspan="1">FluA - H3 subtyping</td><td colspan="1" rowspan="1">1 PCR test on culture</td></tr><tr><td colspan="1" rowspan="1">FluA - 2009 H1N1 subtyping</td><td colspan="1" rowspan="1">1 PCR test on culture</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">2 PCR tests on direct sample</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">1 PCR test on culture</td></tr></table>

Analytically validated PCR followed by bi-directional sequencing assays that target different sequences from those used in the FilmArray pouch were used by ITI as part of a pre-defined composite comparator method for assays in the FilmArray RP system.

ITI relied on its own Research & Development and Regulated Products groups to develop and validate all of the comparator PCR tests used in this study. All assays were designed to provide adequate amplicon length for sequencing while still being able to specifically detect the organisms of interest. Some viruses possess few regions to which a specific PCR assay can be targeted; this limited amplicon length and gene target choice in some instances. However, all comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s).

FilmArray and Comparator Assay Gene Targets   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">FilmArrayTarget</td><td colspan="1" rowspan="1">Comparator AssayTarget 1</td><td colspan="1" rowspan="1">Target 1Chemistry</td><td colspan="1" rowspan="1">Comparator AssayTarget 2</td><td colspan="1" rowspan="1">Target 2Chemistry</td></tr><tr><td colspan="1" rowspan="1">Bordetellapertussis</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">pertussis toxingene</td><td colspan="1" rowspan="1">tcfA gene(Bpss07)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">bhuR gene(Bpss09)</td><td colspan="1" rowspan="1">Tagman</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">ompA gene</td><td colspan="1" rowspan="1">0809 gene(Cpne 0809)</td><td colspan="1" rowspan="1">Taqman(nested)</td><td colspan="1" rowspan="1">gyrB gene(Cpne gyrB)</td><td colspan="1" rowspan="1">Taqman(nested)</td></tr><tr><td colspan="1" rowspan="1">Coronaviruses</td><td colspan="1" rowspan="1">HKU1</td><td colspan="1" rowspan="1">nucleocapsidgene</td><td colspan="1" rowspan="1">Spike gene(CoV HKU1 Sp)</td><td colspan="1" rowspan="1">Taqman(nested)</td><td colspan="1" rowspan="1">polymerase gene(CoV HKU1 P2)</td><td colspan="1" rowspan="1">Taqman(nested)</td></tr><tr><td colspan="1" rowspan="1">Coronaviruses</td><td colspan="1" rowspan="1">NL63</td><td colspan="1" rowspan="1">Nucleoproteingene</td><td colspan="1" rowspan="1">matrix gene(CoV NL63 M )</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">polymerase gene(CoV NL63 P)</td><td colspan="1" rowspan="1">LC green(nested)</td></tr><tr><td colspan="1" rowspan="1">Coronaviruses</td><td colspan="1" rowspan="1">OC43</td><td colspan="1" rowspan="1">Nucleoproteingene</td><td colspan="1" rowspan="1">Hemagglutinin/esterase gene(CoV OC43 HE)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">nucleoprotein genea(CoV OC43 N)</td><td colspan="1" rowspan="1">Tagman</td></tr><tr><td colspan="1" rowspan="1">Coronaviruses</td><td colspan="1" rowspan="1">229E</td><td colspan="1" rowspan="1">Polymerasegene</td><td colspan="1" rowspan="1">Nucleoprotein gene(CoV 229E N)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">polymerase genea(CoV 229E pol)</td><td colspan="1" rowspan="1">Tagman</td></tr><tr><td colspan="1" rowspan="1">Enteroviruses</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5'UTR</td><td colspan="1" rowspan="1">5'UTRa(Entero TQ)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">5'UTRa(Entero LC)</td><td colspan="1" rowspan="1">LC green(nested)</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">nucleocapsidgene</td><td colspan="1" rowspan="1">Matrix gene(HMPV M)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">nucleocapsid genea(HMPV N)</td><td colspan="1" rowspan="1">LC green(nested)</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H1</td><td colspan="1" rowspan="1">hemagglutiningene</td><td colspan="1" rowspan="1">hemagglutinin geneaFluA - H1</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H1 2009</td><td colspan="1" rowspan="1">hemagglutiningene</td><td colspan="1" rowspan="1">hemagglutinin genea(FluA - H1_2009 )</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">H3</td><td colspan="1" rowspan="1">hemagglutiningene</td><td colspan="1" rowspan="1">hemagglutinin genea(FluA - H3 )</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">CARDS toxingene</td><td colspan="1" rowspan="1">gyrB gene(Mpne gyrB)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1">Tuf gene(Mpne tuf)</td><td colspan="1" rowspan="1">Taqman</td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaViruses</td><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">fusion gene</td><td colspan="1" rowspan="1">matrix genePPIV 4)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rhinoviruses</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">5'UTR</td><td colspan="1" rowspan="1">5'UTRa(Rhinovirus TQ)</td><td colspan="1" rowspan="1">Taqman(nested)</td><td colspan="1" rowspan="1">5'UTRa(Rhinovirus LC)</td><td colspan="1" rowspan="1">LC green(nested)</td></tr></table>

a These comparator assays target a different region of the same gene targeted by the FilmArray Respiratory Panel. The assays amplify different amplicon sequences and have no overlap with the FilmArray targets.

Validation studies included analytical sensitivity (LoD) study, analytical reactivity study, and analytical exclusivity study.

The confirmed LoD for each comparator assay is presented in the table below:

LoD for Comparator Assays   

<table><tr><td colspan="1" rowspan="1">Assay</td><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">LoD Concentration</td><td colspan="1" rowspan="1">#Pos/Total</td><td colspan="1" rowspan="1">% Positive</td></tr><tr><td colspan="1" rowspan="1">Bordetella 07 Assay</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">4000 CFU/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">Bordetella 09 Assay</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">4000 CFU/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae gyrBAssay</td><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1">3000 genome copies/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae 0809Assay</td><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1">3000 genome copies /mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 Pol12 Assay</td><td colspan="1" rowspan="1">Coronavirus HKUI</td><td colspan="1" rowspan="1">1.9 E06 genome copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 Spike Assay</td><td colspan="1" rowspan="1">Coronavirus HKUI</td><td colspan="1" rowspan="1">1.9 E+06 genome copies/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 M Assay</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 Pol Assay</td><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV OC43 HE Assay</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">600 TCID50/mL</td><td colspan="1" rowspan="1">20120</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV OC43 N Assay</td><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">600 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">CoV 229E P Assay</td><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">4 TCID50/mL</td><td colspan="1" rowspan="1">19120</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">CoV 229E N Assay</td><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">4 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterovirus TQ Assay</td><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">30,000 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Enterovirus LC Assay</td><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">30,000 TCID50/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">HMPV M Assay</td><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">2 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">HMPV N Assay</td><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">2 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza H1 Assay</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">200 TCID50/mL</td><td colspan="1" rowspan="1">19/20</td><td colspan="1" rowspan="1">95%</td></tr><tr><td colspan="1" rowspan="1">Influenza H1N12009Assay</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Influenza H3N2 Assay</td><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">5 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae gyrBAssay</td><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">30 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae tuf Assay</td><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">30 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">PIV4 Assay</td><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">500 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus TQ Assay</td><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus LC Assay</td><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1 TCID50/mL</td><td colspan="1" rowspan="1">20/20</td><td colspan="1" rowspan="1">100%</td></tr></table>

Analytical reactivity study results are summarized in the table below. All Coronavirus types did not have inclusivity strains available to test

Inclusivity for Comparator Assays   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">LOD(TCID50/ml)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Assay 1</td><td colspan="1" rowspan="1">Assay 2</td></tr><tr><td colspan="1" rowspan="8">Bordetella pertussis</td><td colspan="1" rowspan="8">4000</td><td colspan="1" rowspan="1">EA41</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 8467</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 9797</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 51445</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA-589</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 9340</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 10380</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC BAA 1335</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="3">Chlamydophila pneumoniae</td><td colspan="1" rowspan="3">3000(genome copies)</td><td colspan="1" rowspan="1">VR-1360</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">VR-1310</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 53592</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">Enterovirus</td><td colspan="1" rowspan="9">30000</td><td colspan="1" rowspan="1">Coxsackievirus A10</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A24</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A9</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B3</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus A21/Kuykendall</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">EY</td><td></td><td></td><td></td><td></td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">LOD(TCID50/ml)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Assay 1</td><td colspan="1" rowspan="1">Assay 2</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Echovirus 68 (F02-3607 corn)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="5">Influenza A (H1N1)</td><td colspan="1" rowspan="5">200</td><td colspan="1" rowspan="1">A/New Caledonia/20/99</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="5"></td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">Influenza A (H1N1-2009)</td><td colspan="1" rowspan="2">100</td><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="9"></td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="7">Influenza A (H3N2)</td><td colspan="1" rowspan="7">5</td><td colspan="1" rowspan="1">A/Victoria/3f15</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/f13</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/68Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Alice</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">MRC-2</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">Human Metapneumovirus</td><td colspan="1" rowspan="9">2</td><td colspan="1" rowspan="1">3 (Peru2-2002 BI)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">5 (Peru3-2002 BI)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">8 (Peru6-2002 BI)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">9 (IA3-2002 Al)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">18 (IA18-2003 B2)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">20 (IA14-2003 A2)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">27</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">Human Rhinovirus</td><td colspan="1" rowspan="9">1</td><td colspan="1" rowspan="1">A16 (11757)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A2 (HGP)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A34 (137-3)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A57 (Ch47)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A7 (68-CV 11)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A77 (130-63)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">A85 (50-525-CV54)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Not tested</td></tr><tr><td colspan="1" rowspan="1">B14 (1059)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B27 (5870)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="6">Mycoplasma pneumoniae</td><td colspan="1" rowspan="6">30</td><td colspan="1" rowspan="1">ATCC 15531</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 15293</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 15377</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 15492</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 29085</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">ATCC 49894</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="3">Parainfluenza Virus 4</td><td colspan="1" rowspan="3">500</td><td colspan="1" rowspan="1">Type 4A (M-25)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Type 4B (Zeptometrix#08010060BCF)</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">Type 4B (CH-19503)</td><td colspan="1" rowspan="1">Positive</td></tr></table>

Analytical exclusivity study testing organisms and concentrations tested are presented in the table below. No cross-reactivity was observed for all tested organisms at the tested concentrations.

Analytical Exclusivity for Comparator Assays   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Concentration (TCID50/ml)</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">1.30E+09</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">7.40E+09</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">9.80E+09</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.00E+08</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">2.34E+10</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">3.50E+09</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC14779</td><td colspan="1" rowspan="1">4.20E+09</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">2.12E+09</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">2.60E+06</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">8.30E+09</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11 (type strain)</td><td colspan="1" rowspan="1">6.83E+08</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">2.20E+08</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">3.16E+08</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.99E+09</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">M1027 (type strain)</td><td colspan="1" rowspan="1">1.63E+09</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.05E+10</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus (MRSA)</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">8.40E+09</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">6.20E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">5.54E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.57E+08</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">2.53E+07</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.93E+08</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.04E+09</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.26E+06</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.51E+06</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">KB</td><td colspan="1" rowspan="1">1.02E+08</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">K340</td><td colspan="1" rowspan="1">1.40E+09</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">ATCC VR-740</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">ATCC VR-759</td><td colspan="1" rowspan="1">2.19E+06</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">6123</td><td colspan="1" rowspan="1">1.41E+05</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">NR-470</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.02E+08</td></tr><tr><td colspan="1" rowspan="1">hMPV-16</td><td colspan="1" rowspan="1">IA10-2003, A1</td><td colspan="1" rowspan="1">2.45E+05</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">3.80E+06</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1) var 2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">1.26E+06</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">2.45E+05</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">4.17E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">5.01E+05</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">6.61E+06</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.70E+05</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">4.17E+05</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">3.50E+09</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">7.26E+06</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">5.63E+06</td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">Promega (G3041)</td><td colspan="1" rowspan="1">182ng/ul</td></tr></table>

A retrospective clinical evaluation of the FilmArray RP system was performed at ITI to supplement the prospective evaluation data. In this study, retrospective pre- selected archived specimens were further validated/confirmed by PCR assays followed by sequencing analysis, and then tested with the FilmArray RP system.

Because it is possible that the provided samples had been misidentified, had degraded during storage or contained additional pathogens (such as Rhinovirus) not identified by the source laboratory, the presence of the expected analyte or the absence of the analytes was confirmed using analytically validated “validation” PCR assays for the analytes in the specific panel. These “validation” PCR assays are summarized in the table below:

Retrospective Validation PCR Assaysa   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">FilmArray Target(s)(assay names)</td><td colspan="1" rowspan="1">ValidationAssayGene Target 1(assay name)</td><td colspan="1" rowspan="1">Target 116Chemistr</td><td colspan="1" rowspan="1">Validation AssayGeneTarget 2(assay name)</td><td colspan="1" rowspan="1">Target 2Chemistry</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hexon gene(Adeno)</td><td colspan="1" rowspan="1">polymerase gene(Adpl)</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enteroviruses</td><td colspan="1" rowspan="1">5'UTR(Entero1, Entero2,HRV1, HRV2, HRV3,HRV4)</td><td colspan="1" rowspan="1">5'UTREnteroviruses(EnteroLC)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1">5'UTRRhinoviruses(RhinoTq)c</td><td colspan="1" rowspan="1">Taqman</td></tr><tr><td colspan="1" rowspan="1">InfluenzaA/H1</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="1">matrix gene(FluA Matrix)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1">hemagglutiningene (FluA H1)c</td><td colspan="1" rowspan="1">Taqman</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td><td colspan="1" rowspan="2"></td></tr><tr><td colspan="1" rowspan="1">hemagglutinin gene(FluA-H1-pan)</td></tr><tr><td colspan="1" rowspan="3">InfluenzaA/H12009</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="3">matrix gene(FluA Matrix)c</td><td colspan="1" rowspan="3">LC Green</td><td colspan="1" rowspan="3">hemagglutiningene(FluA H1 09)c</td><td colspan="1" rowspan="3">Taqman</td></tr><tr><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td></tr><tr><td colspan="1" rowspan="1">hemagglutinin gene(FluA-H1-2009)</td></tr><tr><td colspan="1" rowspan="3">InfluenzaA/H3</td><td colspan="1" rowspan="1">matrix gene(FluA-pan1)</td><td colspan="1" rowspan="3">matrix gene(FluA Matrix)c</td><td colspan="1" rowspan="3">LC Green</td><td colspan="1" rowspan="3">hemagglutiningene (FluA H3)c</td><td colspan="1" rowspan="3">Taqman</td></tr><tr><td colspan="1" rowspan="1">NS gene(FluA-pan2)</td></tr><tr><td colspan="1" rowspan="1">hemagglutinin geneFFuA-H3)</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">hemagglutinin gene(FluB)</td><td colspan="1" rowspan="1">matrix gene(FluB)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 1</td><td colspan="1" rowspan="1">hemagglutinin gene(PIV1)</td><td colspan="1" rowspan="1">hemaglutinin gene(PIV1)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 2</td><td colspan="1" rowspan="1">fusion gene(PIV2)</td><td colspan="1" rowspan="1">fusion gene(PIV2)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 3</td><td colspan="1" rowspan="1">fusion gene(PIV3)</td><td colspan="1" rowspan="1">fusion gene(PIV3)c</td><td colspan="1" rowspan="1">LC Green</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ParainfluenzaVirus 4</td><td colspan="1" rowspan="1">fusion gene(PIV4)</td><td colspan="1" rowspan="1">matrix gene(PIV4)</td><td colspan="1" rowspan="1">Taqman</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

a Assays for $B$ . pertussis, C. pneumoniae, M. pneumoniae, Coronavirus OC43, and Coronavirus 229E were identical to assays used in the original prospective study. b All LC Green assays are nested assays. c These validation assays target a different region of the same gene targeted by the FilmArray RP. The assays amplify different amplicon sequences.

Validation studies for these additional “validation” PCR assays also included analytical sensitivity (LoD) study, analytical reactivity study, and analytical exclusivity study.

The confirmed LoD for each “validation” PCR assay is presented in the table below:

# Validation PCR Assays for Additional Assays

<table><tr><td rowspan=1 colspan=1>Assay</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>LoD Conc.</td><td rowspan=1 colspan=1># Pos/Total</td><td rowspan=1 colspan=1>% Positive</td></tr><tr><td rowspan=1 colspan=1>Adeno Polymerase Assay</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>300 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=3 colspan=1>Influenza A Matrix</td><td rowspan=1 colspan=1>Influenza A HINI-Seasonal</td><td rowspan=1 colspan=1>200 TCID50/mL</td><td rowspan=3 colspan=1>19/20</td><td rowspan=3 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Influenza A Hl-2009</td><td rowspan=1 colspan=1>100 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3N2</td><td rowspan=1 colspan=1>5 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B Pan Assay</td><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>60 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 1 Alt Assay</td><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>500 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 2 Alt Assay</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3 Alt Assay</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>10 TCID50/mL</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

Analytical reactivity study results are summarized in the table below:

Analytical Reactivity Studies for Additional Assays   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">LOD (TCID50/mL)</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Assay Results</td></tr><tr><td colspan="1" rowspan="16">Adenovirus</td><td colspan="1" rowspan="16">300</td><td colspan="1" rowspan="1">Adeno KB 3</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 2-1</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 4a</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 4p3</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno KB 5</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 6</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno KB 7A</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 7d/d2</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 7h</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno KB 8</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 11</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 14</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 21a</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno KB 31</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno A-549 34</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Adeno HEK293A 41-1</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">Influenza A</td><td colspan="1" rowspan="2">Flu A H1N1 = 200Flu A 2009 H1N1 =100</td><td colspan="1" rowspan="1">A/New Caledonia/20/9 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/PR/8/34 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="15"></td><td colspan="1" rowspan="15">Flu A H3N2 = 5</td><td colspan="1" rowspan="1">A1/FM/1/47 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/NWS/33 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A1/Denver/1/57 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Solomon Islands/3/2006</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Weiss/43 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Mal/302/54 (H1)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Swine NY/01/2009</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Swine NY/03/2009</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Victoria/3/75 (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Port Chalmers/1/73 (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Aichi/2/68 (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Hong Kong/8/68 (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Alice (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">MRC-2 (H3 recombinant)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A/Brisbane/10/07 (H3)</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="9">Influenza B</td><td colspan="1" rowspan="9">60</td><td colspan="1" rowspan="1">B/Allen/45</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Brigit</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/FL/07/04</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/GL/1739/54</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Hong Kong/5/72</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Lee/40</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Malaysia/2506/04</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Maryland/1/59</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B/Taiwan/2/62</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 1</td><td colspan="1" rowspan="2">500</td><td colspan="1" rowspan="1">C-35</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">C-39</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus </td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">PIV2 Greer</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="2">Parainfluenza Virus 3</td><td colspan="1" rowspan="2">10</td><td colspan="1" rowspan="1">C-243</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">47885 NIH</td><td colspan="1" rowspan="1">Positive</td></tr></table>

Analytical exclusivity study results are summarized in the table below:

Analytical Exclusivity Studies for Additional Assays   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">Conc.(TCID50/ml)</td><td colspan="1" rowspan="1">Adeno</td><td colspan="1" rowspan="1">flu AMatrix</td><td colspan="1" rowspan="1">flu BPan</td><td colspan="1" rowspan="1">PIV 3</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">Clinical isolate</td><td colspan="1" rowspan="1">1.30E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">7.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">9.80E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.00E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">O157:H7</td><td colspan="1" rowspan="1">2.34E+10</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">3,50E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Corynebacteriumdiptheriae</td><td colspan="1" rowspan="1">ATCC14779</td><td colspan="1" rowspan="1">4.20E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">2.12E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">2.60E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">8.30E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11 (type strain)</td><td colspan="1" rowspan="1">6.83E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumtuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">2.20E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">3.16E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.99E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">M1027 (type strain)</td><td colspan="1" rowspan="1">1.63E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.05E+10</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus(MRSA)</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">8.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Staphylococcusepidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">6.20E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">5.54E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7.57E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">2.53E+07</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.93E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">6.04E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.26E+06</td><td colspan="1" rowspan="1">Posa</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.51E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 1</td><td colspan="1" rowspan="1">KB</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">K340</td><td colspan="1" rowspan="1">1.40E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">229E(ATCC VR-740)</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">OC43(ATCC VR-759)</td><td colspan="1" rowspan="1">2.19E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">HKU1 (6123)</td><td colspan="1" rowspan="1">1.41E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Coronavirus</td><td colspan="1" rowspan="1">NL63 (NR-470)</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Echovirus</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.02E+08</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">hMPV-16</td><td colspan="1" rowspan="1">IA10-2003, A1</td><td colspan="1" rowspan="1">2.45E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1)</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">3.80E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H1N1) var2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">1.26E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza A (H3N2)</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">2.45E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">5.01E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">6.61E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Pos</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus</td><td colspan="1" rowspan="1">1A</td><td colspan="1" rowspan="1">1.70E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">A</td><td colspan="1" rowspan="1">4.17E+05</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">3.50E+09</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">7.26E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">5.63E+06</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr><tr><td colspan="1" rowspan="1">Human DNA</td><td colspan="1" rowspan="1">Promega (G3041)</td><td colspan="1" rowspan="1">182ng/ul</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td><td colspan="1" rowspan="1">Neg</td></tr></table>

a This stock produced false positive Adenovirus result. The false positive was found to be caused by Adenovirus contamination of the viral stock and not due to cross-reactivity between the Adenovirus assay and Measles virus.

# 2. Comparison studies:

a. Method comparison with predicate device:

Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

# 3. Clinical studies:

# a. Prospective Clinical Study

The clinical performance of the FilmArray RP system was evaluated during a prospective study at 3 U.S. clinical laboratories. In the first phase of the prospective clinical study, NP swab samples were collected and tested at 3 U.S. clinical laboratories (Sites 1, 2, and 3) during December, 2009 through May, 2010 (2009/2010 respiratory season),. In the second phase of the prospective clinical study, patient enrollment and NP swab sample collection and testing were carried out at 2 of the 3 sites (Sites 2 and 3) for an additional 5 month time period, September, 2010 through January, 2011, (2010/2011 respiratory season), in an effort to achieve increased detection of several low prevalence analytes (i.e., PIV1, PIV2, and PIV4) during the anticipated peak season for PIV1 and PIV2. Clinical study sites were chosen based upon the expertise of the principal investigators, the quality of their personnel and laboratory resources, and the desired geographic/demographic variation of the potential subject populations.

Three geographically separated clinical study sites participated in the clinical evaluation of the FilmArray RP system. The study sites were located in the Southeast (Charleston, SC), the Southwest (Dallas, TX), and the Midwest (Detroit, MI). The size of the population in these locations ranges from approximately 107,000 in Charleston, SC, to approximately 1.2 million in Dallas, TX (source: U.S. Census Bureau). Each study location was representative of the intended use setting (clinical laboratories) and testing was performed by trained clinical laboratory personnel.

The study sites enrolled subjects from diverse demographic groups. Site 1 enrolled subjects from an adult emergency department. Site 2 enrolled primarily children from the emergency department, but also enrolled immunocompromised adults from an outpatient bone marrow transplant clinic (approximately $10 \%$ of the enrolled subjects at this site). Site 3 enrolled subjects from a pediatric emergency department and associated urgent care clinic.

A total of 1144 subjects were initially enrolled in the prospective study. Eight hundred fifty seven (857) subjects were initially enrolled in the 2009/2010 respiratory season and 4 (3 at Site 1 and 1 at Site 3) were withdrawn. Another 287 subjects were enrolled in the 2010/2011 respiratory season and 20 specimens from Site 2 were excluded from data analysis due to improper specimen storage prior to testing. Three (3) specimens from Site 3 were without valid external control mix results acquired (a clinical protocol deviation), and therefore excluded from data analysis.

The following table provides a summary of demographic information for the 1117 subjects that participated in the combined prospective study (2009/2010 respiratory season and 2010/2011 respiratory season), and were included in the data analysis:

Demographic Summary for FilmArray RP Prospective Clinical Study   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>600 (54%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>517 (46%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>724 (65%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>119 (11%)</td></tr><tr><td rowspan=1 colspan=1>22 - 49</td><td rowspan=1 colspan=1>190 (17%)</td></tr><tr><td rowspan=1 colspan=1>≥ 50</td><td rowspan=1 colspan=1>84 (8%)</td></tr></table>

The following table provides a summary of demographic information for the 853 subjects that participated in the first phase of the prospective study (2009/2010 respiratory season), and were included in the data analysis:

Demographic Summary for 2009/2010 Respiratory Season   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>449 (53%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>404 (47%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>484 (57%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>95 (11%)</td></tr><tr><td rowspan=1 colspan=1>22 - 49</td><td rowspan=1 colspan=1>190 (22%)</td></tr><tr><td rowspan=1 colspan=1>≥ 50</td><td rowspan=1 colspan=1>84 (10%)</td></tr></table>

The following table provides a summary of demographic information for the 264 subjects that participated in the second phase of the prospective study (2010/2011 respiratory season), and were included in the data analysis:

Demographic Summary for 2010/2011 Respiratory Season   

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>151 (57%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>113 (43%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>240 (91%)</td></tr><tr><td rowspan=1 colspan=1>6 - 21</td><td rowspan=1 colspan=1>24 (9%)</td></tr><tr><td rowspan=1 colspan=1>22 - 49</td><td rowspan=1 colspan=1>0 (0%)</td></tr><tr><td rowspan=1 colspan=1>≥ 50</td><td rowspan=1 colspan=1>0 (0%)</td></tr></table>

All specimens used in the study were prospectively collected from subjects with signs/symptoms of respiratory infection including but not limited to fever, cough, sore throat, runny/stuffy nose, myalgia, earache, headache, chills, or fatigue. The subject may or may not have had respiratory infection testing ordered by a physician. Written informed consent was acquired from each subject and/or their parent/guardian (if under 18) at the time of enrollment into the prospective study. After informed consent was acquired, the subject was assigned a Volunteer Identification Number (VIN). The VIN was used to deidentify the specimen used for FilmArray testing and to provide data to the sponsor. A key that connected the subject’s identity to their assigned VIN was maintained in a secure location at each site in order to facilitate the tracking of standard of care testing and for monitoring purposes. Access to this key was limited to the study coordinator or laboratory supervisor at each site. At the time of enrollment the following information was recorded on the Case Report Form (CRF): 1) Age and sex; 2) Information about their suspected respiratory infection, i.e. signs and symptoms, date of onset; and 3) Current medications (self-reported and/or collected from medical records).

Two respiratory specimens were collected from each enrolled subject. One specimen (NPS or other standard specimen type, such as nasal aspirate/wash, used by the study site) was used for standard of care testing and for viral culture to detect seven common respiratory viruses (Adenovirus, Flu A, Flu B, PIV1, 2, and 3, and RSV). A frozen aliquot of viral culture material was prepared for comparator PCR/sequencing testing. The second specimen, a NPS in $3 ~ \mathrm { m L }$ of viral transport media (VTM) was used for FilmArray testing according to the Instructions for Use and was aliquoted and frozen for further comparator PCR/sequencing testing. Individual specimens were delinked from all patient identifiers and given a study sample code.

Reference and comparator methods used to assess the performance of the FilmArray RP system include viral culture followed by fluorescent antibody identification for common viruses (Adenovirus, Flu A, Flu B, PIV1, PIV2, PIV3, and RSV), viral culture followed by 1 analytically validated PCR assay with bi-directional sequence confirmation (Flu A subtyping and PIV4), and a composite comparator method of 2 analytically validated PCR assays with bi-directional sequence confirmation (all other panel analytes):

Reference/Comparator Methods Used to Assess FilmArray RP Performance   

<table><tr><td rowspan=1 colspan=1>Organism/Virus</td><td rowspan=1 colspan=1>Reference/Comparator Method(s)</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=7 colspan=1>Culture followed by DFA</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>FluA/H1 subtyping</td><td rowspan=4 colspan=1>Culture + 1 PCR/sequencing test on culture</td></tr><tr><td rowspan=1 colspan=1>FluA/H3 subtyping</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1N1 subtyping</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=10 colspan=1>2 PCR/sequencing tests on direct sample</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC431</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td></tr></table>

Viral culture was performed with fresh specimens using standard methods. Sites 1 and 3   
used R-Mix shell vial culture format (mixed monolayer of human adenocarcinoma A549

cells and mink lung Mv1Lu cells) incubated for up to 72 hours. Site 2 used traditional cell culture format, employing three separately inoculated cell lines (primary rhesus monkey kidney pRhMK cells, human epidermoid carcinoma Hep-2 cells, and human lung fibroblast MRC5 cells) incubated for up to 14 days. Each site confirmed the presence or absence of the viruses (Adenovirus, Flu A, Flu B, PIV1, PIV2, PIV3, and RSV) using a FDA-cleared direct fluorescent antibody method (D3 Ultra Viral Respiratory Screening and ID Kit; Diagnostic Hybrids, Inc.).

Frozen aliquots of viral culture material and NPS specimens were shipped to ITI on a weekly basis for comparator PCR/sequencing testing. PCR testing was performed by ITI research associates in a blinded manner; specimens were labeled only with the subject’s VIN and the FilmArray results were not available to them. A bi-directional sequencing result for each comparator PCR positive result was required to demonstrate sequence identity with the target organism. All comparator PCR assays were designed at ITI to provide adequate amplicon length for sequencing while still being able to detect the organisms of interest. The assays were also designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. PCR plates containing potentially positive amplicons were not opened at ITI; they were sent directly to the contract sequencing laboratory, for ExoSAP clean up and bi-directional sequencing. Sequencing results that met acceptance criteria (i.e., a minimum of $9 9 \%$ accuracy for at least $90 \%$ of the sequence, Phred quality score of 20 or higher) were recorded as positive for the specific analyte.

Performance of the FilmArray RP system detecting Adenovirus, Flu A, Flu B, PIV1, PIV2, PIV3, or RSV, respectively, was compared to viral culture followed by fluorescent antibody identification. “True” Adenovirus, Influenza A, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, or RSV positives, respectively, were considered as any sample that was tested positive for Adenovirus, Influenza A, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, or RSV, respectively, by viral culture followed by DFA testing. “True” Adenovirus, Influenza A, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, or RSV negatives, respectively, were considered as any sample that was tested negative for Adenovirus, Influenza A, Influenza B, Parainfluenza Virus1, Parainfluenza Virus 2, Parainfluenza Virus 3, or RSV, respectively, by viral culture followed by DFA testing.

Performance of the FilmArray RP system detecting Influenza A/H1, A/H3, A/2009 H1N1, or Parainfluenza Virus 4, respectively, was compared to viral culture followed by one analytically validated PCR assay with bi-directional sequence confirmation. The comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. “True” Influenza A/H1, A/H3, A/2009 H1N1 positives, respectively, were considered as any sample that was tested positive for Influenza A by viral culture, and had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Influenza A/H1, A/H3, or A/2009 H1N1 sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” Influenza A/H1, A/H3, or A/2009 H1N1 negatives, respectively, were considered as any sample that was

tested negative for Influenza A by viral culture, or any sample that was tested positive for Influenza A virus by viral culture, but was tested negative by the respective Influenza A subtype specific PCR assay. “True” Parainfluenza Virus 4 positives were considered as any sample for which bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Parainfluenza Virus 4 sequences deposited in the NCBI Genbank database with acceptable alignment E-values was obtained from testing of viral culture material. “True” Parainfluenza Virus 4 negatives were considered as any sample where testing of viral culture material with a Parainfluenza Virus 4 specific PCR assay was negative.

The E-value ranges generated from the clinical trial per organism were presented in the following table:

E-values for Clinical Trial Organisms   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-value High</td></tr><tr><td rowspan=1 colspan=1>FluA/H1 subtyping</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FluA/H3 subtyping</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>FluA/2009 H1N1 subtyping</td><td rowspan=1 colspan=1>8.0E-122</td><td rowspan=1 colspan=1>1.4E-78</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>4.7E-158</td><td rowspan=1 colspan=1>1.9E-150</td></tr></table>

Performance of the FilmArray RP system for detecting Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Enterovirus, Human Metapneumovirus, Human Rhinovirus, or Mycoplasma pneumoniae, respectively, was compared to a predetermined algorithm that used composite comparator methods. The composite reference methods consist of two analytically validated PCR assays followed by bi-directional genetic sequencing. The comparator assays were designed to amplify a different sequence from that amplified by the FilmArray assay(s). None of the comparator PCR assays overlapped any FilmArray amplicon sequence even if the same gene was targeted. “True” Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Enterovirus, Human Metapneumovirus, Human Rhinovirus, or Mycoplasma pneumoniae positives, respectively, were considered as any sample that had bi-directional sequencing data meeting pre-defined quality acceptance criteria that matched Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Enterovirus, Human Metapneumovirus, Human Rhinovirus, or Mycoplasma pneumoniae sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. “True” Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Enterovirus, Human Metapneumovirus, Human Rhinovirus, or Mycoplasma pneumoniae negatives were considered as any sample that was tested negative by both of the comparator PCR assays. The alignment E-value ranges generated from the clinical trial per organism were presented in the following table:

E-values for Clinical Trial Organisms   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-value High</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>2.3E-89</td><td rowspan=1 colspan=1>1.4E-15</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>4.1E-65</td><td rowspan=1 colspan=1>3.4E-52</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1.9E-83</td><td rowspan=1 colspan=1>8.0E-16</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>8.8E-148</td><td rowspan=1 colspan=1>6.5E-70</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>2.0E-143</td><td rowspan=1 colspan=1>8.8E-11</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>4.8E-138</td><td rowspan=1 colspan=1>0.0E+00</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>9.23E-42</td><td rowspan=1 colspan=1>2.72E-12</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>5.8E-77</td><td rowspan=1 colspan=1>2.3E-15</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>4.51E-72</td><td rowspan=1 colspan=1>1.15E-08</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>1.1E-106</td><td rowspan=1 colspan=1>6.4E-34</td></tr></table>

The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower is the E-Value, the more significant is the match. A sequence alignment in GenBank that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone. (http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).

Performance of the FilmArray RP system detecting Adenovirus, Influenza A, Influenza B, RSV, Influenza A/H1, Influenza A/H3, Influenza A/2009 H1N1, Human Rhinovirus, Enterovirus, Coronavirus NL63, Coronavirus HKU1, Human Metapneumovirus, or Parainfluenza Virus 3 were evaluated and established during the first phase of the prospective clinical study (data from a total of 853 subjects from the 2009/2010 respiratory season). Performance of the FilmArray RP system detecting Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus OC43, Coronavirus 229E, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, or Parainfluenza Virus 4 were evaluated during the first and the second phases of the prospective clinical study (combined data of a total of 1117 subjects - a total of 853 subjects from the 2009/2010 respiratory season and a total of 264 subjects from the 2010/2011 respiratory season). During the second phase of the prospective clinical study (2010/2011 respiratory season), only the reference/comparator methods for Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus OC43, Coronavirus 229E, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, or Parainfluenza Virus 4 were performed on all specimens and used to calculate the performance.

The prospective performance data (all sites combined) are presented in the following tables by analyte:

Adenovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>14a</td><td rowspan=1 colspan=1>38</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3b</td><td rowspan=1 colspan=1>812</td><td rowspan=1 colspan=1>815</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>826</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 24/27     88.9%(95%CI: 70.8%-97.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 812/826   98.3%(95%CI: 97.2%-99.1%)</td></tr></table>

a Adenoviruses were identified in 13/14 of these false positive specimens by bidirectional sequence analysis. Ten were identified as Adenovirus group C, two as Adenovirus group B, and one as Adenovirus group E.

b One of three specimens re-tested positive with the FilmArray RP System. Bidirectional sequence analysis identified two false negative specimens as Adenovirus group C and one as Adenovirus group B.

Bordetella pertussisa   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1110</td><td rowspan=1 colspan=1>1110</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>1111</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 6/6          100%(95%CI: 54.1-100%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1110/1111  99.9%(95%CI: 99.5%- 100%)</td></tr></table>

a These results are from the prospective clinical trial specimens; see below for analysis of additional Bordetella pertussis specimens.

Chlamydophila pneumoniae   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1116</td><td rowspan=1 colspan=1>1116</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1116</td><td rowspan=1 colspan=1>11170</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 1/1           100%(95%CI: n/a)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1116/1116  100%(95%CI: 99.7%- 100%)</td></tr></table>

a These results are from the prospective clinical trial specimens; see below for analysis of contrived Chlamydophila pneumoniae specimens.

Coronavirus HKU1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>827</td><td rowspan=1 colspan=1>828</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 23/24       95.8%(95%CI: 78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 827/829   99.8%(95%CI: 99.1%-100%)</td></tr></table>

a CoV-HKU1 was detected in both false positive specimens using an alternate bi-directional sequence analysis assay.

Coronavirus NL63   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>23</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>830</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 23/24      95.8%(95%CI: 78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 829/829 100.0% (95%CI: 99.6%-100%)</td></tr></table>

Coronavirus OC43a   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5b,c</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1098</td><td rowspan=1 colspan=1>1098</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 14/14        100%(95%CI: 76.8%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 1098/1103 99.6%(95%CI: 99.0%-99.9%)</td></tr></table>

a These results are from the prospective clinical specimens; see below for analysis of additional specimens of supplemental Coronavirus OC43 specimens. b CoV-OC43 was detected in 2/5 false positive specimens using an alternate assay with bi-directional sequence analysis. $^ { \mathrm { ~ c ~ } } 2 / 5$ false positives were determined to be cross-reactive products derived from amplification of CoV-HKUI virus with the CoV-OC43 assay primers.

Coronavirus $2 2 9 \mathrm { E } ^ { \mathrm { a } }$   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1103</td><td rowspan=1 colspan=1>1103</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1105</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 12/12       100%(95%CI: 73.5%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 1103/100599.8%(95%CI: 99.4%-100.0%)</td></tr></table>

a These results are from the prospective clinical specimens; see below for analysis of supplemental Coronavirus 229E specimens. b CoV-229E was identified by bi-directional sequence analysis in 1/2 false positive specimens using an alternate assay.

Human Metapneumovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>88</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>94</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>754</td><td rowspan=1 colspan=1>759</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>760</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 88/93      94.6%(95%CI: 87.9%-98.2%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 754//76099.2% (95%CI: 98.3%-99.7%)</td></tr></table>

Human Rhinovirus/Enterovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>190</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>225</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>613</td><td rowspan=1 colspan=1>628</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>205</td><td rowspan=1 colspan=1>648</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 190/205     92.7%(95%CI: 88.2%-95.8%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 613/648    94.6%(95%CI: 92.6%-96.2%)</td></tr></table>

Influenza A   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>843</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 9/10       90.0% (95%CI: 55.5%-99.8%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 841/843   99.8% (95%CI: 99.2%-100.0%)</td></tr></table>

a. Influenza A virus (2009 H1N1 subtype) was detected in 2/2 false positive samples using bi-directional sequence analysis.

Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 0/0        n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 853/853   100.0% (95%CI: 99.6%-100.0%)</td></tr></table>

Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 0/0        n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 853/853  100.0% (95%CI: 99.6%-100.0%)</td></tr></table>

# Influenza A/2009 H1N1

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>3a,b</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>841</td><td rowspan=1 colspan=1>842</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>844</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 8/9        88.9%(95%CI: 51.8%-99.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 841/844   99.6%(95%CI: 99.0%-99.9%)</td></tr></table>

a For one of the Influenza A positive culture samples, Influenza A virus (2009 H1N1 subtype) was not detected by the PCR comparator assay followed by bidirectional sequence analysis. However, Influenza A virus (2009 H1N1 subtype) was detected from the direct sample and confirmed by bi-directional sequence analysis using an alternate assay.

b Influenza A virus (2009 H1N1 subtype) was detected in 2/2 False Positive samples using bi-directional sequence analysis.

Influenza B   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>853</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 0/0        n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 853/853   100.0%(95%CI: 99.6%-100.0%)</td></tr></table>

Mycoplasma pneumoniaea   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1113</td><td rowspan=1 colspan=1>1113</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1113</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 4/4           100%(95%CI: 39.8-100%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1113/1113  100%(95%CI: 99.7%- 100%)</td></tr></table>

a These results are from the prospective clinical specimens; see below for analysis of additional clinical specimens.

Parainfluenza Virus 1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1115</td><td rowspan=1 colspan=1>1115</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1116</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 1/1            100.0% n/a</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1115/1116    99.9%(95%CI: 99.5%-100.0%)</td></tr></table>

a Parainfluenza Virus 1 was identified in this specimen using bi-directional sequence analysis.

Parainfluenza Virus 2   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2a,</td><td rowspan=1 colspan=1>9</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b,c</td><td rowspan=1 colspan=1>1107</td><td rowspan=1 colspan=1>1108</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1109</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 7/8            87.4%(95%CI: 47.4%-99.7%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1107/1109    99.8%(95%CI: 99.4%-100.0%)</td></tr></table>

a Parainfluenza Virus 2 was detected in both False Positive specimens using bidirectional sequence analysis. b Parainfluenza Virus 2 was not detected in this False Negative specimen using PCR/sequencing analysis. c Two adjacent specimens (one False Positive and one False Negative) may have been switched during the viral culture reference method testing as is evidenced by bi-directional sequence analysis of these specimens

Parainfluenza Virus 3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>10a</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>819</td><td rowspan=1 colspan=1>820</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>829</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 23/24         95.8%(95%CI: 78.9%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 819/829       98.8%(95%CI: 97.8%-99.4%)</td></tr></table>

a Parainfluenza Virus 3 were identified in 10/10 False Positive specimens using bi-directional sequence analysis. b Parainfluenza 3Virus was detected in this False Negative specimen when retested using the FilmArray RP system.

Parainfluenza Virus 4   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1a</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1107</td><td rowspan=1 colspan=1>1107</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1108</td><td rowspan=1 colspan=1>1117</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 9/9           100.0%(95%CI: 66.4%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 1107/1108  99.9%(95%CI: 99.5%-100.0%)</td></tr></table>

a Parainfluenza Virus 4 was identified in this False Positive specimen using bidirectional sequence analysis although it was not detected from viral culture.

RSV   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>87a</td><td rowspan=1 colspan=1>139</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>714</td><td rowspan=1 colspan=1>714</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>801</td><td rowspan=1 colspan=1>853</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 52/52      100.0%(95%CI: 93.2%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 714/801    89.1%(95%CI: 86.8%-91.2%)</td></tr></table>

a RSV was detected in 83/87 False Positive specimens using bi-directional sequence analysis.

The prospective performance data (all sites combined) are presented in the following table by organism:

Summary of Prospective Performance for All Analytes   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>Sensitivity</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>Specificity</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>24/27</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>70.8% - 97.7%</td><td rowspan=1 colspan=1>812/826</td><td rowspan=1 colspan=1>98.3%</td><td rowspan=1 colspan=1>97.2% - 99.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A(versus culture alone)</td><td rowspan=1 colspan=1>9/10</td><td rowspan=1 colspan=1>90.0%</td><td rowspan=1 colspan=1>55.5% - 99.8%</td><td rowspan=1 colspan=1>841/843</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.2% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>88.9%</td><td rowspan=1 colspan=1>51.8% - 99.7%</td><td rowspan=1 colspan=1>841/844</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0/0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>853/853</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1115/1116</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>7/8</td><td rowspan=1 colspan=1>87.4%</td><td rowspan=1 colspan=1>47.4% - 97.7%</td><td rowspan=1 colspan=1>1107/1109</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% - 99.9%</td><td rowspan=1 colspan=1>819/829</td><td rowspan=1 colspan=1>98.8%</td><td rowspan=1 colspan=1>97.8% - 99.4%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>66.4% - 100.0%</td><td rowspan=1 colspan=1>1107/1108</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>52/52</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.2% - 100.0%</td><td rowspan=1 colspan=1>714/801</td><td rowspan=1 colspan=1>89.1%</td><td rowspan=1 colspan=1>86.8% - 91.2%</td></tr><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>54.1-100%</td><td rowspan=1 colspan=1>1110/1111</td><td rowspan=1 colspan=1>99.9%</td><td rowspan=1 colspan=1>99.5%-100%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1116/1116</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7% - 100%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% - 99.9%</td><td rowspan=1 colspan=1>827/829</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23/24</td><td rowspan=1 colspan=1>95.8%</td><td rowspan=1 colspan=1>78.9% - 99.9%</td><td rowspan=1 colspan=1>829/829</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>99.6% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>14/14</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>76.8% - 100%</td><td rowspan=1 colspan=1>1098/1103</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>99.0% - 99.9%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>12/12</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>73.5% - 100%</td><td rowspan=1 colspan=1>1103/1105</td><td rowspan=1 colspan=1>99.8%</td><td rowspan=1 colspan=1>99.4% - 100%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>88/93</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>87.9% - 98.2%</td><td rowspan=1 colspan=1>754/760</td><td rowspan=1 colspan=1>99.2%</td><td rowspan=1 colspan=1>98.3% - 99.7%</td></tr><tr><td rowspan=1 colspan=1>HumanRhinovirus/Enterovirus</td><td rowspan=1 colspan=1>190/205</td><td rowspan=1 colspan=1>92.7%</td><td rowspan=1 colspan=1>88.2% - 95.8%</td><td rowspan=1 colspan=1>613/648</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>92.6% - 96.2%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>39.8% - 100%</td><td rowspan=1 colspan=1>1113/1113</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>99.7% 0- 100%</td></tr></table>

# i. Prospective Clinical Study FilmArray Software and Instrument, and Pouch Controls Performance Analysis

A total of 1117 prospective clinical specimens were tested and analyzed during the first phase and the second phases of the prospective clinical evaluation (2009/2010 respiratory season and 2010/2011 respiratory season). Of the 1117 analyzed prospective clinical specimens, $9 4 . 6 \%$ (1057/1117) of these specimens yielded valid results on the first attempt (i.e., first loaded pouch). Invalid results or no results were obtained for the remaining 60 $( 5 . 4 \% )$ specimens (no results for 24 specimens due to incomplete runs; 36 specimens were “invalid” due to Pouch Control failures). Of the 24 incomplete runs, 3 were aborted by users $( 0 . 3 \% )$ ; 6 were due to software-related errors $( 0 . 6 \% )$ and 15 were due to instrument errors $( 1 . 3 \% )$ . Fifty-seven (57) of the 60 initially failed (no results or invalid) specimens yielded valid results after a single retest using a new pouch/sample. The remaining three (3) specimens failed on the second attempt (2 due to failed Pouch Controls, 1 due to an

instrument error), but yielded valid results following a second retest using another pouch/sample.

Software and Instrument Performance – Prospective Clinical Study (First Phase and Second Phases Combined)   

<table><tr><td rowspan=2 colspan=1>TotalSpecimensTested</td><td rowspan=2 colspan=1>TotalCompleted Tests onFirstTPouch</td><td rowspan=2 colspan=1>TotalTests NotCompletedon FirstPouch</td><td rowspan=2 colspan=1>AbortedRuns byUser</td><td rowspan=2 colspan=1>SoftwareError</td><td rowspan=2 colspan=1>Incomplete SoftwareRelated)</td><td rowspan=1 colspan=4>InstrumentError</td></tr><tr><td rowspan=1 colspan=1>ValveControllerErrors</td><td rowspan=1 colspan=1>IncorrectInstallation of aFirmware/Software Update</td><td rowspan=1 colspan=1>CameraCommuni-catiopError.b,c</td><td rowspan=1 colspan=1>InstrumentErorTotal</td></tr><tr><td rowspan=2 colspan=1>1117</td><td rowspan=1 colspan=1>1093</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>2.1%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1.0%</td><td rowspan=1 colspan=1>0.1%</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>1.3%</td></tr></table>

a Known failure mode in the instrument bladder system. A new bladder material was implemented to reduce this error mode. b An additional 18 errors occurred; however, valid results were obtained by restarting the test using the same pouch. c Known failure mode caused by a communication timing error between the FilmArray software and the driver of the instrument camera. This error mode was eliminated in software version 1.1. (Validation study of a total of 872 runs using software version 1.1 demonstrated $0 \%$ error rate caused by this particular error mode.)

Analysis of Pouch Controls - Prospective Clinical Study (First Phase and Second Phase Combined)   

<table><tr><td rowspan=2 colspan=1>TotalCompleted Tests on FirstTPouch</td><td rowspan=2 colspan=1>Total Runswith PouchControlsPassed</td><td rowspan=2 colspan=1>Total Runswith PouchControlsFailed</td><td rowspan=1 colspan=3>Runs with RNA ProcessingControls Failed</td><td rowspan=1 colspan=3>Runs with PCR2 Controls Failed</td></tr><tr><td rowspan=1 colspan=1>Total Runswith RNAProcessL ControlsFailed</td><td rowspan=1 colspan=1>Both RNAprocessand PCR2ControlsFailed</td><td rowspan=1 colspan=1>Only RNAProcessControlsFFaileda</td><td rowspan=1 colspan=1>Total Runswith PCR2ControlsFailed</td><td rowspan=1 colspan=1>Both RNAProcess andPCR2 ControlsFailed</td><td rowspan=1 colspan=1>OnlyPCR2ControlsFailed</td></tr><tr><td rowspan=2 colspan=1>1093</td><td rowspan=1 colspan=1>1057</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>3.3%</td><td rowspan=1 colspan=1>2.7%</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>1.2%</td><td rowspan=1 colspan=1>0.5%</td></tr></table>

a Investigation into the higher than expected rate of RNA process controls revealed that the QC processes used to control the amount of control template was inadequate. The QC process was revised to better control the concentration of control template.

# ii. Prospective Clinical Study FilmArray Pouch Performance Analysis

The FilmArray RP pouch is manufactured to contain a vacuum that draws in the required amount of Hydration Solution and Sample/Buffer Mix when the seals are broken by the cannula-tipped syringes in each of the pouch ports. If either the Hydration Solution or the Sample/Buffer Mix is not drawn into the pouch, the operator is instructed to discard the faulty pouch and obtain a new pouch to test the specimen. Out of 1281 pouches used in the first phase of the prospective clinical evaluation (2009/2010 respiratory season), thirty-seven $( 3 7 / 1 2 8 1 ; 2 . 9 \% )$ failed to draw Hydration Solution or Sample/Buffer Mix. However, the majority (31/37) of these failures occurred early in the evaluation (December, 2009 through early February, 2010). Out of 521 pouches used in the second phase of the prospective clinical study (2010/2011 respiratory season), ten (10) (10/521; $1 . 9 \% )$ failed to draw Hydration Solution or Sample/Buffer Mix. It has been observed that an operator can partially break a port seal, leading to loss of vacuum and failure to draw the solution. It may also be possible for a pouch to be manufactured with a faulty port seal or reduced vacuum. A decrease in the rate of this failure type over time suggested that either pouch manufacturing improved or the operators became more adept at reliably puncturing the port seals.

During the first phase of the prospective clinical evaluation, one operator observed a leak in a pouch when it was removed from the instrument after testing (1/1244 loaded pouches $=$ $0 . 0 8 \%$ failure rate). The pouches are manufactured from durable materials able to withstand the manipulations performed by the instrument; however, an occasional small rupture or tear may occur. Following discovery of the pouch leak, the operator followed recommended decontamination procedures and performed contamination surveillance swab testing of the area surrounding the instrument. No contamination was detected. No pouch leaks were observed in the second phase of the prospective clinical study.

# iii. Prospective Clinical Study External Controls Performance Analysis

Four (4) frozen $( - 7 0 ^ { \circ } \mathrm { C } )$ control mixes were provided to the study sites for daily testing during the first phase of the prospective clinical study (2009/2010 respiratory season). Three control mixes contained pooled NPS specimens spiked with whole virus with some plasmid DNA for hard to acquire organisms. Combined, the 3 mixes covered all panel analytes. A fourth mix was negative for all panel members and only contained pooled NPS:

Daily Control Mixes During First Phase of Prospective Study   

<table><tr><td rowspan=1 colspan=1>Control Mix 1</td><td rowspan=1 colspan=1>Control Mix 2</td><td rowspan=1 colspan=1>Control Mix 3</td><td rowspan=1 colspan=1>Control Mix 4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=7 colspan=1>Negative for allorganisms</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Influenza A/2009 H1N1</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=1>Influenza A/H3</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1></td></tr></table>

The operators were required to complete a valid control mix run (correct results obtained) prior to beginning patient sample testing on each testing day. A total of 300 control mix runs were attempted during the first phase of the prospective clinical study (2009/2010 respiratory season). Seventeen (17) runs did not complete and 6 runs had failed pouch control(s). Of the remaining 277 runs (site 1, 99 runs; site 2, 70 runs; and site 3, 108 runs), 7 (site 1, 1 run; site 2, 2 runs; and site 3, 4 runs) (7/277; $2 . 5 \% )$ did not return the correct organism results either due to the detection of an extra analyte (4/7) and/or the failure to detect one or more spiked analytes (4/7). These failures may be due to low level virus from the NPS donors, introduction of contamination during the preparation or testing of the samples, or improper handling of the frozen aliquots.

Four external controls were used by study sites for daily testing during the second phase of the prospective clinical study (2010/2011 respiratory season). Three of the controls were freeze-dried, room-temperature stable, synthetic RNA mixes that were rehydrated with Hydration Solution prior to being tested. Synthetic RNA mixes were used in the second phase of the clinical study due to several more organisms becoming difficult to obtain in the concentrations necessary for high volume external control testing. Instead of including more plasmids in the control mixes, validated synthetic RNA mixes were used. The RNA mixes are designed to test all assays in the pouch for each of the analytes. Combined, these 3 mixes cover all analyte assays in the pouch. The fourth control consisted of Hydration Solution alone (Negative):

Daily Control Mixes During Second Phase of Prospective Study   

<table><tr><td rowspan=1 colspan=1>Control Mix Alpha</td><td rowspan=1 colspan=1>Control Mix Beta</td><td rowspan=1 colspan=1>Control Mix Gamma</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=9 colspan=1>Negative for allorganisms(HydrationSolution alone)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>Bordetella pertussis</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Coronavirus OC43</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1(Hemagglutinin sequence only)</td><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>Human Metapneumovirus</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Human Rhinovirus</td><td rowspan=1 colspan=1>Influenza A (no subtype)</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>Influenza A H1-2009(Hemagglutinin sequence only)</td><td rowspan=1 colspan=1>Parainfluenza Virus 4</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>Influenza A H3(Hemagglutinin sequence only)</td><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Rhinovirus/Enterovirus</td><td rowspan=1 colspan=2></td></tr></table>

The operators were required to complete a valid control mix run (correct results obtained) prior to beginning patient sample testing on each testing day. A total of 137 control mix runs were attempted during the second phase of the prospective clinical study (2010/2011 respiratory season). Seventeen (17) runs did not complete and 3 had failed pouch control(s). Of the remaining 117 runs (site 2, 61 runs; and site 3, 56 runs), 14 (site 2, 6 runs; and site 3, 8 runs) (14/117; $1 2 . 0 \% )$ did not return the correct organism results either due to the detection of an extra analyte (1/14) and/or the failure to detect one or more spiked analytes (13/14). These failures may be due to introduction of contamination during the preparation or testing of the samples, improper/incomplete rehydration of the control mixes, or introduction of RNases into the synthetic RNA control mixes during rehydration.

# iv. Prospective Clinical Study Environmental Contamination Surveillance Analysis

The FilmArray RP system is a closed system; all steps require no user manipulation. This dramatically reduces the possibility of contamination from environmental material or amplicon during the testing process. However, it is still possible for contaminants to be introduced during pouch loading. To determine if environmental contamination of workstations might interfere with the accuracy of test results, during the first phase of the prospective clinical study, FilmArray operators were instructed to perform contamination surveillance swab testing at least weekly and following any pouch leaks. Forty (40) total swab test runs were attempted; 35 of the runs completed and the controls passed. No contamination was detected in any of the swab tests indicating that the routine cleaning procedures recommended in the clinical trial protocol are sufficient to prevent false positive test results caused by environmental contamination of workstations used for preparing pouches. These routine cleaning procedures were recommended in the package

insert.

# v. Prospective Clinical Study Mixed Infection Analysis

The FilmArray RP system detected a total of 94 mixed infections in the first phase of the prospective clinical evaluation performed from December 2009 to May 2010 (853 tested and analyzed specimens). This represents $1 8 . 0 \%$ of the total positive specimens (94/524). Eighty-seven (87/94; $9 2 . 6 \%$ double infections, $6 ( 6 / 9 4 ; 6 . 3 \% )$ triple infections, and one quadruple infection (1/94; $1 . 1 \% )$ ) were observed. The total number of test results comprising these co- infections was 189. The single most common co-infection $( 2 0 / 9 4 ; 2 1 . 1 \% )$ was Human Rhinovirus/Enterovirus with Respiratory Syncytial Virus. These viruses were the most prevalent in the tested population. Out of the 94 co-infections, 54 contained one or more analytes that had not been detected with the reference/comparator methods, i.e., discrepant co-infection.

Distinct Co-infection Combinations Detected by the FilmArray RP System in the First Phase of the Prospective Clinical Trial (December 2009 to May 2010)   

<table><tr><td colspan="4" rowspan="1">Distinct Co-infection CombinationsDetected by FilmArray RP</td><td colspan="1" rowspan="1">TotalCo-infections</td><td colspan="1" rowspan="1">Number ofDiscrepantCo-infectionsa</td><td colspan="1" rowspan="1">Discrepant Analyte(s)a</td></tr><tr><td colspan="1" rowspan="1">Analyte 1</td><td colspan="1" rowspan="1">Analyte 2</td><td colspan="1" rowspan="1">Analyte 3</td><td colspan="1" rowspan="1">Analyte 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus/B. pertussis</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus (2)</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoV229E</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus; RSV</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">C. pneumoniae</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Adenovirus; hMPV</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">Adenovirus; HRV-Entero</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV3</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">CoV229E</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV-Entero</td></tr><tr><td colspan="1" rowspan="1">CoV229E</td><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV; CoV229E</td></tr><tr><td colspan="1" rowspan="1">CoV229E</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoVHKU1</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">CoVHKU1</td><td colspan="1" rowspan="1">CoVOC43</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">CoVOC43 (2)b</td></tr><tr><td colspan="1" rowspan="1">CoVHKU1</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoVHKU1</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoVHKU1</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">RSV</td></tr><tr><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">HRV-Entero</td></tr><tr><td colspan="1" rowspan="1">CoVNL63</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSV (2)</td></tr><tr><td colspan="1" rowspan="1">CoVOC43</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">RSV (2); CoVOC43 (2)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">hMPV (2); HRV-Entero (1)</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV 3</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">hMPV</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">hMPV (1); RSV (3)</td></tr><tr><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">PIV 1</td></tr><tr><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">PIV 3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">HRV-Entero (3); PIV 3 (3)</td></tr><tr><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">HRV-Entero</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">HRV-Entero (8); RSV (13)</td></tr><tr><td colspan="1" rowspan="1">PIV 4</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Total Co-infections</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">54</td><td colspan="1" rowspan="1">65/200(32.5%)</td></tr><tr><td colspan="4" rowspan="1">Total Double Infections</td><td colspan="1" rowspan="1">87</td><td colspan="1" rowspan="1">47</td><td colspan="1" rowspan="1">55/178 (30.9%)</td></tr><tr><td colspan="4" rowspan="1">Total Triple Infections</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">8/18 (44.4%)</td></tr><tr><td colspan="4" rowspan="1">Total Quadruple Infections</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">2/4 (50%)</td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by FilmArray RP but not detected by the reference/comparator methods or by discrepancy investigation using bi-directional sequence analysis. Sixty-five (65) analytes from 54 of the $9 4 \mathrm { c o }$ -infections were not detected by the reference/comparator methods. Forty-four (44) were investigated using bi-directional sequence analysis with an alternate assay; in 37 cases the investigative assay successfully identified the analyte in the specimen. b Both discrepant CoV43 results were false-positive due to cross-reactivity of the OC43 assay with HKU1 viruses in the specimens

Additional Distinct Co-infection Combinations Detected by Reference/Comparator Methods, But Not Detected by the FilmArray RP System in the First Phase of the Prospective Clinical Trial (December 2009 to May 2010)   

<table><tr><td rowspan=1 colspan=2>Distinct Co-infectionCombinationsa</td><td rowspan=2 colspan=1>Total Co-infections</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infections</td><td rowspan=2 colspan=1>Discrepant Analyte(s)</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td></tr><tr><td rowspan=1 colspan=1>CoV229E</td><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>hMPV (2); HRV/Entero (1)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero</td><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>HRV/Entero (1)</td></tr></table>

a This table includes only distinct co-infections that were detected by the reference/comparator method but not by FilmArray RP; the remaining co-infections detected by the reference/comparator methods are already represented in the table above.

Mixed Infections Detected by FilmArray RP in the First Phase (December 2009 to May 2010) and the Second Phase (September 2010 to January 2011) of the Prospective Clinical Trial and Prevalence of Individual Analytes in Mixed Infections   

<table><tr><td></td><td></td><td rowspan=1 colspan=1>idual Analytes</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>Organism Combinations</td><td rowspan=1 colspan=1>NumberofPositiveSamples</td><td rowspan=1 colspan=1>Percentage ofTotal SamplesTested(n/1117)</td><td rowspan=1 colspan=3>Organism</td><td rowspan=1 colspan=1>Numberof MixedInfectionS</td><td rowspan=1 colspan=1>Prevalence inMixedInfections(n/121)</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + RSV</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>2.40%</td><td rowspan=1 colspan=3>HRV/Entero</td><td rowspan=1 colspan=1>82</td><td rowspan=1 colspan=1>68%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>1.30%</td><td rowspan=1 colspan=3>RSV</td><td rowspan=1 colspan=1>59</td><td rowspan=1 colspan=1>49%</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV3</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0.80%</td><td rowspan=1 colspan=3>Adenovirus</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>20%</td></tr><tr><td rowspan=1 colspan=1>hMPV + HRV/Entero</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0.60%</td><td rowspan=1 colspan=3>hMPV</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>21%</td></tr><tr><td rowspan=1 colspan=1>CoVOC43 + HRV/Entero</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.50%</td><td rowspan=1 colspan=3>PIV3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>15%</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=3>Coronavirus NL63</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>11%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + HRV/Entero</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=3>Coronavirus OC43</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>Coronavirus HKU1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>10%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>PIV4</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4%</td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>Coronavirus 229E</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>B. pertussis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2%</td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.40%</td><td rowspan=1 colspan=3>PIV2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2%</td></tr><tr><td rowspan=1 colspan=1>CoVOC43 + RSV</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>C. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV3</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>Influenza B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0.30%</td><td rowspan=1 colspan=3>M. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + HRV/Entero + PIV3</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.20%</td><td rowspan=1 colspan=3>PIV1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1%</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.20%</td><td rowspan=1 colspan=5></td></tr><tr><td rowspan=2 colspan=1>CoVHKU1 + CoVOC43</td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>0.20%</td><td rowspan=2 colspan=1></td><td rowspan=20 colspan=4></td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=2 colspan=1>HRV/Entero + B. pertussis</td><td rowspan=2 colspan=1>2</td><td rowspan=2 colspan=1>0.10%</td><td rowspan=2 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + B. pertussis +HRV/Entero</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + C. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + COV229E + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoVNL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoVOC43</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>Adenovirus + PIV3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoV229E + CoVNL63 +HRV/Entero + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoV229E + HRV/Entero</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoV229E + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoVHKU1 + HRV/Entero + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoVNL63 + hMPV + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>CoVOC43 + hMPV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td></td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism Combinations</td><td rowspan=1 colspan=1>NumberofPositiveSamples</td><td rowspan=1 colspan=1>Percentage ofTotal SamplesTested(n/1117)</td><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Numberof MixedInfectionS</td><td rowspan=1 colspan=1>Prevalence inMixedInfections(n/121)</td></tr><tr><td rowspan=1 colspan=1>Influenza B + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td><td rowspan=3 colspan=3></td></tr><tr><td rowspan=1 colspan=1>PIV 4 + RSV</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td></tr><tr><td rowspan=1 colspan=1>PIV2 + M. pneumoniae</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.10%</td></tr></table>

# vi. Retrospective Clinical Study: Study 299

A retrospective clinical evaluation of the FilmArray RP system was performed at ITI to supplement the prospective evaluation data. In this study, retrospective pre-selected archived specimens were first validated/confirmed by PCR/sequencing analysis, and then tested with the FilmArray RP system. Samples for this study included 508 retrospective pre-selected archived specimens. Most specimens had previous positive test results for selected organisms of interest from the source sites. Several negative specimens (as determined at the source sites) were also included in the retrospective sample set.

These retrospective archived samples were characterized previously at the source sites using a variety of methods including DSFA, bacterial culture, viral culture followed by DFA, Luminex X-TAG RVP, and Laboratory Developed Test (LDT) PCR assays. Upon arrival at ITI, a 4-digit study number (VIN) was assigned to each sample and a key was created in order to group samples together into panels by organism. The organism panel was randomized such that the users testing the samples with the FilmArray RP were blinded as to the expected test result. Specimens were organized into “test panels” such that specimens positive for a particular analyte could serve as a negative for another analyte, allowing for calculations of negative percent agreement (NPA). Additional negative specimens were also included in each panel. There were 5 different panels, labeled 4 through 8, as indicated in the following table:

Testing Panels for Study 299   

<table><tr><td rowspan=1 colspan=1>Panel #</td><td rowspan=1 colspan=1>Organisms</td><td rowspan=1 colspan=1>Number of Samples</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>Influenza A/H1, A/H3, A/2009, H1N1,Influenza B, and Negatives</td><td rowspan=1 colspan=1>178</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>Parainfluenza Viruses 1, 2, 3, Enterovirus, andNegatives</td><td rowspan=1 colspan=1>148</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Adenovirus, Bordetella pertussis, andNegatives</td><td rowspan=1 colspan=1>74</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>Parainfluenza Virus 4 and Negatives</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Mycoplasma pneumoniae and Negatives</td><td rowspan=1 colspan=1>95</td></tr></table>

Because it is possible that the provided samples had been misidentified, had degraded during storage, or contained additional pathogens (such as Rhinovirus) not identified by the source laboratory, the presence of the expected analyte or the absence of the analytes was confirmed using analytically validated ‘validation’ PCR assays for the analytes in the specific panel. (Note that while some validation assays share the same gene target as the FilmArray RP

panel, they targeted different regions of the gene). Positive PCR reactions were then subjected to bi-directional sequencing for definitive confirmation. ‘True’ organism positives were considered as any sample that had bi- directional sequencing data meeting pre-defined quality acceptance criteria that matched the particular organism sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), with acceptable E-values. The E-value ranges generated from the retrospective clinical trial per organism are presented in the following table:

E-Value Ranges from Study 299   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>E-value Low</td><td rowspan=1 colspan=1>E-Value High</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>6.4E-57</td><td rowspan=1 colspan=1>1.4E-28</td></tr><tr><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>5.2E-91</td><td rowspan=1 colspan=1>9.8E-16</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>9.3E-50</td><td rowspan=1 colspan=1>3.4E-30</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>4.1E-99</td><td rowspan=1 colspan=1>5.1E-35</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>4.9E-118</td><td rowspan=1 colspan=1>9.0E-61</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>9.0E-88</td><td rowspan=1 colspan=1>3.4E-73</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>1.0E-180</td><td rowspan=1 colspan=1>4.9E-50</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniae</td><td rowspan=1 colspan=1>1.83E-90</td><td rowspan=1 colspan=1>2.10E-35</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>3.7E-59</td><td rowspan=1 colspan=1>4.2E-52</td></tr><tr><td rowspan=1 colspan=1>PIV-2</td><td rowspan=1 colspan=1>1.0E-180</td><td rowspan=1 colspan=1>9.7E-81</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>2.7E-69</td><td rowspan=1 colspan=1>1.6E-33</td></tr><tr><td rowspan=1 colspan=1>PIV-4</td><td rowspan=1 colspan=1>1.7E-157</td><td rowspan=1 colspan=1>2.7E-148</td></tr></table>

If this “validation” testing did not confirm the presence of the expected organism in the sample, it was not analyzed further as this result may have reflected that the sample had been improperly labeled or handled. If additional pathogens were identified in the sample (those not identified by the source laboratory), the samples was also omitted from the analysis for that specific analyte. Due to a large number of samples for Bordetella pertussis and Mycoplasma pneumoniae that failed validation, supplemental specimens were tested and additional analyses performed (see separate sections regarding M. pneumoniae Testing and B. pertussis Testing, below). Of the 508 samples, the expected result (positive for reported organism, or negative for presumed negative specimens) was confirmed for 426 of 508 specimens using “validation” PCR assays and sequencing (the identity of 82 specimens could not be confirmed). Data collected from “validation” PCR assays showed that specimens contained a broad range of analyte concentrations for all organisms tested (based on Cp value distribution).

Validated/confirmed specimens were then analyzed on the FilmArray RP. The identity of the specimens was unknown to the FilmArray operators. If a FilmArray error was encountered or if a pouch failed, residual NPS specimen (when available) was used to retest the sample. Of the 426 specimens, 418 were analyzed; 8 specimens were not run due to instrument and software errors or pouch sample control failures. (These are excluded from data analysis due to insufficient volumes to retest these specimens.)

The following table provides a summary of demographic information for the 426 subjects with confirmed samples included in the data analysis of the retrospective study:

Demographic Summary for FilmArray RP Retrospective Clinical Study   

<table><tr><td rowspan=3 colspan=1>Sex</td><td rowspan=1 colspan=1>Female (%)</td><td rowspan=1 colspan=1>107 (25.1%)</td></tr><tr><td rowspan=1 colspan=1>Male (%)</td><td rowspan=1 colspan=1>104 (24.4%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>215 (50.5%)</td></tr><tr><td rowspan=3 colspan=1>Age</td><td rowspan=1 colspan=1>Avg</td><td rowspan=1 colspan=1>17.5</td></tr><tr><td rowspan=1 colspan=1>Min</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>Max</td><td rowspan=1 colspan=1>81.0</td></tr><tr><td rowspan=5 colspan=1>Age Range</td><td rowspan=1 colspan=1>≤5</td><td rowspan=1 colspan=1>110 (25.8%)</td></tr><tr><td rowspan=1 colspan=1>6-21</td><td rowspan=1 colspan=1>43 (10.1%)</td></tr><tr><td rowspan=1 colspan=1>22-49</td><td rowspan=1 colspan=1>26 (6.1%)</td></tr><tr><td rowspan=1 colspan=1>≥50</td><td rowspan=1 colspan=1>32 (7.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknowna</td><td rowspan=1 colspan=1>215 (50.5%)</td></tr><tr><td rowspan=1 colspan=2>Total Specimens</td><td rowspan=1 colspan=1>426</td></tr></table>

Demographic information was not provided for specimens from one source. Because the specimens were provided by a pediatric hospital, it is understood that the age range of specimens was from $< 1$ yrs to 21 yrs.

The retrospective performance data from testing panel $\# 4$ samples (a total of 161 samples: 32 Flu A/H1, 54 A/H3, 35 A/2009 H1, 31 Flu B and 9 Negative) are presented in the following tables by analyte:

Seasonal Influenza A/H1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>127</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>159a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 32/32      100.0% (95%CI: 89.1%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 127/127  100.0% (95%CI: 97.1%-100%)</td></tr></table>

a The initial analysis of samples VIN 4-037 and VIN 4-058 resulted in pouch sample control failures. There was insufficient volume to retest the specimens; therefore these two specimens were excluded from further analysis.

Seasonal Influenza A/H3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>54</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>105</td><td rowspan=1 colspan=1>159a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 54/54      100% (95%CI: 93.4%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 105/105    100% (95%CI: 96.5%-100%)</td></tr></table>

a The initial analysis of samples VIN 4-037 and VIN 4-058 resulted in pouch sample control failures. There was insufficient volume to retest the specimens; therefore these two specimens were excluded from further analysis.

2009 H1N1 Influenza Virus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>125</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>159a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 34/34     100% (95%CI: 89.7%-100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 125/125 100% (95%CI: 97.1%-100.0%)</td></tr></table>

a The initial analysis of samples VIN 4-037 and VIN 4-058 resulted in pouch sample control failures. There was insufficient volume to retest the specimens; therefore these two specimens were excluded from further analysis

Influenza B Virus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>129</td><td rowspan=1 colspan=1>159a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 30/30   100% (95%CI: 88.4%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 129/129 100% (95%CI: 97.2%-100%)</td></tr></table>

a The initial analysis of samples VIN 4-037 and VIN 4-058 resulted in pouch sample control failures. There was insufficient volume to retest the specimens; therefore these two specimens were excluded from further analysis

The retrospective performance data from testing panel #5 samples (a total of 129 samples: 23 Enterovirus, 35 PIV1, 28 PIV2, 36 PIV3 and 7 Negative) are presented in the following tables by analyte:

Enterovirus   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>22</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>91</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>113a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 22/23  95.7% (95%CI: 78.1%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 90/900 100% (95%CI: 96.0%-100%)</td></tr></table>

a It was observed that 16 Parainfluenza Virus positive specimens were unexpectedly positive when tested with the Rhinovirus “validation” PCR. Because there was no previous test for Rhinovirus or Enterovirus for these particular specimens, the Rhinovirus PCR result was not investigated by sequencing and the Rhinovirus or Enterovirus status of these specimens were not included in the FilmArray RP percent agreement analysis for Enterovirus.

Parainfluenza Virus 1   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>34</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>95</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>94</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:−34/35    97.1% (95%CI: 85.1%-99.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:: 94/94   100%    (95%CI: 96.2%-100.0%)</td></tr></table>

Parainfluenza Virus 2   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 28/28   100%  (95%CI: 87.6%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 101/101  100%   (95%CI: 96.4%-100%)</td></tr></table>

Parainfluenza Virus 3   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>36</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>93</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>93</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 36/36    100%  (95%CI: 90.3%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:93/93   100%  (95%CI: 96.1%-100%)</td></tr></table>

The retrospective performance data from testing panel $\# 6$ samples (a total of 74 samples) is presented in the following tables for 28 Adenovirus specimens, $3 3 \ B$ . pertussis specimens, and 13 negatives. (Note: additional $B$ . pertussis specimens obtained outside of Study 299 are discussed in a separate section, Additional $B$ . pertussis Testing, below).

Adenovirusa   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>28</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>55b</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:   27/27 100%   (95%CI: 87.2%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement:  28/28 100.0% (95%CI: 87.7%-100.0%)</td></tr></table>

a One sample referred as adenovirus present could not be confirmed prior to RP testing and is not included in the analysis.

b It was found that two $B _ { \cdot }$ . pertussis positive specimens were positive for Adenovirus (VINs 6-025 & 6-032); 14 unconfirmed $B$ . pertussis samples were also excluded.from the analysis of panel 6. Because there was no previous test for Adenovirus on these particular specimens, the Adenovirus PCR result was not investigated by sequencing and the Adenovirus status of these specimens was not included in the FilmArray RP percent agreement analysis for Adenovirus. Two specimens were lost due to incomplete FilmArray runs resulting from instrument and software errors (VINs 6- 052 and 6-068), and there was insufficient volume of either sample to repeat the test, therefore it was omitted from further analysis. The 28 negative specimens were drawn from the originally negative specimens $( { \mathfrak { n } } = 1 2$ ; one specimen was not analyzed due to FilmArray pouch failure) and $B$ . pertussis positive specimens.

Bordetella pertussisa   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>17</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>39</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>39</td><td rowspan=1 colspan=1>56b</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 17/17    100%  (95%CI: 80.5%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 39/39   100%  (95%CI: 91.0%-100.0%)</td></tr></table>

a Additional $B$ . pertussis specimens separate from Study 299 are discussed on page 126 below. b Of 33 specimens originally obtained as having tested positive for the presence of $B$ pertussis, 14 were not confirmed as positive with reference testing and are not included in the analysis. Of the 19 confirmed positive specimens, two specimens were lost due to incomplete FilmArray runs resulting from instrument and software errors (VINs 6-052 and 6-068, respectively) and there was insufficient volume of either sample to repeat the analysis. These are also omitted from the analysis. The 39 negative specimens were 12 of 13 originally negative specimens $[ { \tt n } = 1 2$ ; one specimen was not analyzed due to FilmArray pouch failure) and 27 adenovirus positive samples used as $B$ . pertussis negative samples. (The unconfirmed adenovirus specimen is also not included in this analysis.)

The retrospective performance data from testing panel #7 samples (a total of 18 samples: 11 PIV4 and 7 Negative) are presented in the following table by analyte:

Parainfluenza Virus 4   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>17a</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 11/11      100%(95%CI: 71.5%-100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 6/6        100% (95%CI: 54.1%-100%)</td></tr></table>

a Two independent tests of sample VIN 7-018 both resulted in pouch sample control failures. There was insufficient volume to retest the specimen again; therefore it was excluded from further analysis.

# vii. Mycoplasma pneumoniae Testing

Sixty-eight (68) specimens from patients with respiratory disease that had tested positive for M. pneumoniae by PCR-testing at outside facilities were obtained as part of Study 299 (tested as Panel 8). Specimens were obtained from three outside sources and analyzed with $2 7 M .$ . pneumoniae negative specimens from the original prospective study. No clinical information regarding patient characteristics from the positive specimens was provided other than that the nasopharyngeal swab specimens were obtained primarily from children.

All 95 specimens underwent confirmation testing prior to FilmArray testing. Confirmation testing was performed by detecting the presence of the $M .$ pneumoniae tuf gene by PCR with bi-directional sequencing; confirmation testing was by a single assay, not by two assays as was performed in the original prospective study). Of the 68 presumed positive specimens, 42 were confirmed as positive with a median Cp value of 33.2, range $2 5 . 0 - 4 0 . 0 $ . All 27 of the negative samples were confirmed as negative.

Panel 8 Confirmation Testing   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Outside LaboratoryResult</td><td rowspan=1 colspan=1>ConfirmationResult</td></tr><tr><td rowspan=1 colspan=1>Mpne detected</td><td rowspan=1 colspan=1>68</td><td rowspan=1 colspan=1>42</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>53a</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>95</td><td rowspan=1 colspan=1>95</td></tr></table>

a Twenty-six of the original 68 presumed positive samples by testing at outside facilities were not confirmed.

The sponsor was subsequently able to obtain an additional twenty (20) presumed positive samples from another outside source and test them along with sixteen (16) negative samples as an additional panel (Panel 9). No clinical information regarding patient characteristics from the positive specimens was provided other than that the nasopharyngeal swab specimens were obtained from ‘mostly children.’ For the negative specimens, $6 3 \%$ were from female donors with a median age of 27 (range 3 – 83 years.).

Confirmation testing of these samples, also performed as above, yielded the following:

Panel 9 Confirmation Testing   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Lab Result</td><td rowspan=1 colspan=1>ConfirmationResult</td></tr><tr><td rowspan=1 colspan=1>Mpne detected</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>16a</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>36</td></tr></table>

a Four of the original 20 presumed positive samples for testing at outside facilities were not confirmed.

Due to the high overall unconfirmed rate (30 of 88 specimens were anticipated to be positive), additional testing was performed on 24 of the 30 unconfirmed specimens with sufficient material available for serial retesting. The first stage of additional testing included repeating the original tuf gene assay with a larger sample volume ( $1 4 \mu \mathrm { L }$ compared to $1 0 ~ \mu \mathrm { L }$ for the original specimen testing); of the 24 specimens tested, 20 remained negative with 4 additional positive specimens confirmed by bi-directional sequencing. The remaining 20 unconfirmed specimens were tested by a second validated PCR assay for GyrB; from this second stage of testing, an additional 4 samples tested positive and were confirmed by bidirectional sequencing. Overall, of the original 30 ‘unconfirmed specimens’, 8 were considered confirmed as positive and 22 negative. All samples were then included in the

The following results are for Panel 8 and Panel 9 M. pneumoniae referred specimens where unconfirmed specimens are considered negative and the 8 specimens confirmed by enhanced testing considered positive:

Overall Results for Testing of M. pneumoniae Specimens (Panels 8 and 9 combined)a   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>&#x27;Enhanced&#x27; Confirmation Testing Resultb</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>54</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>67</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>64b,c</td><td rowspan=1 colspan=1>65</td><td rowspan=1 colspan=1>129</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement:: 54/64    84.4% (95%CI: 73.1% - 94.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 58/65    89.2% (95%CI: 79.1% -95.6 %)</td></tr></table>

Several explanations for the low NPA rate are noted by the sponsor, including the possibility that nucleic acid concentration in the specimens were affected by the freezing of specimens to levels below detection by the ‘confirmation’ assays and that not all specimens referred to as ‘positive’ were true positives (i.e., ‘positive’ specimens from one site were second specimens from patients who had tested positive on a different specimen). The sponsor finds support for the former hypothesis by noting that 5/8 false positive results were the highest Cp values of all positive specimens.

The following table summarizes results if only samples confirmed by the original single assay M. pneumoniae tuf are included:

Overall Results for Testing of M. pneumoniae Originally ‘Confirmed’ Specimens Only (Panel 8 and Panel 9 combined)   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Confirmation Testing Result a</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>52</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>52</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>44</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>96</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 52/56    92.9% (95%CI: 82.7-100%%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 0/40     100.0% (95%CI: 91.2-100 %)</td></tr></table>

a Due to a processing error, most of the residual specimen volume for several specimens was discarded. When these samples were needed for retesting due to run errors, the limited residual volume for these specimens needed to be diluted approximately 3-fold for there to be sufficient material for retesting. Two of the specimens that were negative by FilmArray testing but positive by Reference testing were diluted 3-fold before FilmArray testing.

# viii. Additional Bordetella pertussis Testing

The FilmArray prospective clinical trial identified only six $( 6 ) B$ . pertussis FilmArray positive/reference method confirmed specimens (6/1117). To supplement this limited number of samples, the sponsor obtained 33 additional $B$ . pertussis specimens identified as positive during an investigation of a $B$ . pertussis outbreak as part of Study 299, however, only 19 of these specimens were confirmed as containing $B$ . pertussis by validation testing, i.e., by a composite of two PCR assays with bi-directional sequencing. The results for these 19 specimens (two of these 19 specimens did not yield valid FilmArray results, yielding 17 positive specimens) were compared with results from FilmArray testing, and are described on page 125 above with other results from Study 299.

To address the uncertainty due to the large percentage of samples that could not be verified in the original 33 ‘presumed’ positive specimens (summarized on page 123 above), the sponsor obtained an additional 64 specimens that had tested positive for $B$ . pertussis using PCR targeting the IS481 insertion element (cycle threshold $\textstyle ( \mathrm { C p } )$ range: 12.1 - 34.7) from a 2010- 2011 outbreak investigation at a California facility. These specimens were tested in a blinded fashion with 50 specimens from patients with suspected pertussis but negative for IS481 by PCR testing at a separate outside facility. Average patient age, when available, was 7.2 years, median age 6, and range from 6 months to 91 years.

Confirmation/reference testing was performed on all specimens using the following assays:

Confirmation Assays for B. pertussis Specimens   

<table><tr><td rowspan=1 colspan=1>AssayName</td><td rowspan=1 colspan=1>Gene Target</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>Sequencing</td><td rowspan=1 colspan=1>Specificity</td></tr><tr><td rowspan=1 colspan=1>Bpss07</td><td rowspan=1 colspan=1>tcfA: trachealcolonization factor</td><td rowspan=1 colspan=1>ITI</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>B. pertussis, B. parapertussis, and B.bronchiseptica</td></tr><tr><td rowspan=1 colspan=1>Bpss09</td><td rowspan=1 colspan=1>bhuR: outer membraneprotein</td><td rowspan=1 colspan=1>ITI</td><td rowspan=1 colspan=1>Yes</td><td rowspan=1 colspan=1>B. pertussis, B. parapertussis, and B.bronchiseptica</td></tr><tr><td rowspan=1 colspan=1>IS481</td><td rowspan=1 colspan=1>Insertion element 481</td><td rowspan=1 colspan=1>Publisheda</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>B. pertussis, B. holmesii, and some B.bronchiseptica variants</td></tr><tr><td rowspan=1 colspan=1>ptxS1</td><td rowspan=1 colspan=1>Pertussis toxin gene</td><td rowspan=1 colspan=1>Publisheda</td><td rowspan=1 colspan=1>No</td><td rowspan=1 colspan=1>B. pertussis, B. parapertussis,and B. bronchiseptica</td></tr></table>

a Assays labeled as ‘Published’ were assays developed by the sponsor following published CDC protocols for IS481 and ptxS1. Separate analyses demonstrated similar sensitivity for these assays when compared to the sponsor’s Bspp07 and Bspp09 assays.

Specimens that were negative for all four assays were considered negative for analysis. The following algorithm was used for evaluating specimens positive by at least one assay:

Confirmation Algorithm for Positive Archived Specimens   

<table><tr><td rowspan=1 colspan=1>IS481(PCR only) Result</td><td rowspan=1 colspan=1>ptxS1(PCR only) Result</td><td rowspan=1 colspan=1>Bspp07 + Bspp09Comparator Method(PCR and sequencing)</td><td rowspan=1 colspan=1>Specimen Interpretation</td></tr><tr><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>B. pertussis</td><td rowspan=1 colspan=1>Positive for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Cp less than 35 cycles</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Cp equal to or greater than35 but less than 40 cycles</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Any</td><td rowspan=1 colspan=1>Non-pertussis Bordetella sp.</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Cp equal to or greater than35 but less than 40 cycles</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative for B. pertussis</td></tr></table>

In essence, specimens were identified as positive if sequencing results for either Bspp07 or Bspp09 were positive. Following this algorithm, 54 of the original 64 samples were confirmed as positive. Samples not confirmed as positive included the following:

Confirmatory Test Negative B. pertussis Specimens   

<table><tr><td colspan="1" rowspan="1">VIN</td><td colspan="1" rowspan="1">IS481a</td><td colspan="1" rowspan="1">ptxS1</td><td colspan="1" rowspan="1">Bspp07</td><td colspan="1" rowspan="1">Bspp09</td><td colspan="1" rowspan="1">Result</td></tr><tr><td colspan="1" rowspan="1">5805-3003</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3027</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3060</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negativeb</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3061</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3069</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3081</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3089</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3096c</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3101</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">5805-3113</td><td colspan="1" rowspan="1">Indeterminate</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Negative</td></tr></table>

a According to the published CDC protocol for interpreting $B .$ . pertussis results with the $I S 4 8 1$ and $p t x \mathbf { S } 1$ assays, the $I S 4 8 1$ assay is considered positive if the $\mathrm { C p }$ value is below 35. Any $\mathrm { C p }$ value equal to or greater than 35 but less than 40 cycles considered indeterminate; a value of 40 or greater is considered negative. b Specimen 5805-3060 was positive by PCR for amplification of Bspp07 but this result could not be confirmed by bi-directional sequencing. c Specimens 5805-3081 and 5805-3096 were found to be positive for $B$ . holmesii during subsequent discrepancy analysis investigation.

Confirmation testing also determined two of the originally ‘negative’ specimens to be positive for $B _ { \cdot }$ . pertussis:

Specimens Referred as Negative Identified as Positive on Confirmatory Testing   

<table><tr><td rowspan=1 colspan=1>VIN</td><td rowspan=1 colspan=1>IS481a</td><td rowspan=1 colspan=1>ptxS1</td><td rowspan=1 colspan=1>Bspp07</td><td rowspan=1 colspan=1>Bspp09</td><td rowspan=1 colspan=1>Result</td></tr><tr><td rowspan=1 colspan=1>5805-3054</td><td rowspan=1 colspan=1>Indeterminate</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>5805-3075</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr></table>

During confirmation testing, samples that were not validated were analyzed as negative, i.e., 10 samples originally identified as positive at the outside site were considered negative for the analysis; this is in contrast to the results for Study 299 where samples that did not validate were excluded from subsequent analyses. Final results of confirmation testing were as follows:

Results of Confirmatory Testing   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>Lab Result</td><td rowspan=1 colspan=1>Confirmation Result</td></tr><tr><td rowspan=1 colspan=1>Bper positive</td><td rowspan=1 colspan=1>64</td><td rowspan=1 colspan=1>56a</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>58b</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>114</td><td rowspan=1 colspan=1>114</td></tr></table>

a Two (2) of these 56 specimens were originally identified as negative for $B .$ . pertussis by the source laboratory. b Ten (10) of these 58 specimens were originally identified as positive for B. pertussis by the source laboratory

The following table reports the results of blinded testing of the FilmArray RP using confirmation testing as the reference method:

# Results of Testing B. pertussis Specimensa

<table><tr><td rowspan=2 colspan=1>Film ArrayResult</td><td rowspan=1 colspan=3>Confirmation Testing Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>53</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>55</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>56</td><td rowspan=1 colspan=1>59</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>56c</td><td rowspan=1 colspan=1>58</td><td rowspan=1 colspan=1>114</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 53/56   94.6% (95%CI: 85.1  98.9%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 56/58    96.5%(95%CI:88.1 - 99.6%)</td></tr></table>

a A protocol error during the study inadvertently diluted specimens 1:4 rather than 1:2 as per protocol.

b Specimens 5805-3060 and 5805-3069 had been identified by the source lab as positive for $B _ { \cdot }$ . pertussis were not confirmed with the original validation testing. Both of these specimens were positive for $B$ . pertussis during subsequent discrepancy investigation.

c Specimens 5805-3054 and 5805-3075 had been identified by the source lab as negative for B. pertussis, but unexpectedly were found to be positive during confirmation testing. Both of these specimens were found to be negative for $B$ . pertussis during subsequent discrepancy investigation. The third false negative specimen, 5805-3063, was among the latest Cp values during confirmation testing and may have been affected by the inadvertent additional dilution.

FilmArray Cp and Tm values for $B$ . pertussis samples were as follows:

FilmArray RP data summary for ‘True’ Positive $B .$ . pertussis specimens   

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=1 colspan=2>Cp</td><td rowspan=1 colspan=3>Tm ()</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Detection rangea</td></tr><tr><td rowspan=1 colspan=1>Bper</td><td rowspan=1 colspan=1>14.7</td><td rowspan=1 colspan=1>5 - 23.1</td><td rowspan=1 colspan=1>88.7</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>88.3 - 89.1</td></tr></table>

a Tm limits as defined by the FilmArray software.

The following co-infections were detected during analysis of the $1 1 4 B .$ . pertussis samples:

Infections and Co-infections Detected in the B. pertussis and B. pertussis Control Specimens Tested   

<table><tr><td colspan="1" rowspan="1">FilmArray Result</td><td colspan="1" rowspan="1">N</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + Cov HKU1 + HRV/Entero + Influenza A H3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + Cov NL63 + HRV/Entero</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + CoV OC43</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + CoV229E</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + HRV/Entero</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">B. pertussis +HRV/Entero + PIV3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + PIV3</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">B. pertussis + PIV4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">C. pneumoniae + HRV/Entero</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 + HRV/Entero</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">CoV 229E</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero</td><td colspan="1" rowspan="1">18</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + PIV4</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">M. pneumoniae</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">PIV 1</td><td colspan="1" rowspan="1">4</td></tr><tr><td colspan="1" rowspan="1">PIV1 +PIV4</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">3</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">58</td></tr></table>

# ix. Additional testing of Coronavirus OC43 and Coronavirus 229E Specimens

Additional Coronavirus OC43 $\mathrm { ( n = 2 8 ) }$ ) and 229E $( \mathrm { n } { = } 1 4 )$ ) presumed positive specimens were obtained by the sponsor in addition to 18 negative controls (samples submitted for respiratory testing) and tested as a separate Panel (Panel 10). No clinical information was available other than samples were almost all obtained from children. Confirmatory testing for the Coronavirus OC43 specimens was by testing with the comparator assays for the nucleoprotein gene target that was used during the prospective clinical study; for samples that were not confirmed by the initial comparator assay, testing of the Hemagglutinin/esterase gene was then performed by the same assay used during the prospective studies. (Both assays were followed by bi-directional sequencing.) Coronavirus 229E specimens were initially confirmed by a two stage process using the same assays as during the prospective study. Each specimen was tested for the nucleoprotein gene assay using the same assay that was used during the prospective clinical study, followed by testing of negative specimens for the polymerase gene, also using the same assay used during the prospective study. For five samples testing by the second validation assay was necessary (see below); however there was insufficient specimen volume to perform additional extractions and residual extracted specimen was used to perform the additional testing. Confirmation testing results were as follows:

Coronavirus OC43 Confirmation Testing (Panel 10)   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>LabResult</td><td rowspan=1 colspan=1>ConfirmationResult</td></tr><tr><td rowspan=1 colspan=1>Nucleoprotein</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>24a</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>N/T</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>42</td></tr></table>

a Two of the unconfirmed specimens were positive by PCR testing of both validation assays; however, for samples VIN #5805-2050 and VIN #5805-2055, sequence data could not be generated. These two specimens were considered reference method negative. b The 18 negative samples were not tested by the outside laboratory for Coronavirus OC43.

Coronavirus 229E Confirmation Testing (Panel 10)   

<table><tr><td rowspan=1 colspan=1>Specimen</td><td rowspan=1 colspan=1>LabResult</td><td rowspan=1 colspan=1>ConfirmationResult</td></tr><tr><td rowspan=1 colspan=1>Nucleoprotein</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>13a</td></tr><tr><td rowspan=1 colspan=1>Negativeb</td><td rowspan=1 colspan=1>N/T</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>31</td></tr></table>

a VIN 5805-2010 was PCR negative by the nucleoprotein assay and positive by the polymerase assay; however, sequencing confirmed the presence of CoV 229E for only one of two strands. No sequence data was obtained for the second strand. This specimen was considered negative in the analysis below. b The 18 negative samples were not tested by the outside laboratory for Coronavirus 229E.

Samples for Coronavirus OC43 and Coronavirus 229E were run together as Panel 10; therefore, 36 potential negative samples were available for Coronavirus OC43 testing (of 36 negative specimens, 18 were originally referred as negative, 4 samples were not confirmed for CoV OC43, 1 specimen that was not confirmed for Cov 229E, and 13 confirmed Cov 229E specimens), and 47 negative samples were available for Coronavirus 229E testing (of 47 negative specimens, 18 were originally referred as negative, 4 samples were not confirmed for CoV OC43, 1 specimen was not confirmed for Cov 229E, and 24 confirmed Cov 229E specimens). Specimens referred as positive samples but that could not be confirmed were considered as negative reference specimens for the analyses below. Results are presented in the following tables:

Results for Testing of Coronavirus OC43 (Panel 10)   

<table><tr><td rowspan=2 colspan=1>FilmArrayResult</td><td rowspan=1 colspan=3>Confirmation Testing Result</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>3a</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>36</td><td rowspan=1 colspan=1>60b</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 24/24    100.0% (95%CI: 85.8 - 100%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 33/36   91.7% (95%CI: 77.5 - 98.2%)</td></tr></table>

a Specimens 5805-2050 and 5805-2055 had been identified by the source laboratory as positive for Coronavirus OC43 and were PCR positive using confirmation PCR, but no sequence could be obtained and thus the analyte content could not be confirmed (see above). Specimen 5805-2017 was from a sample that had not been tested by the source lab for Coronaviruses but was negative by confirmation PCR for Coronavirus OC43. A negative result was obtained when re-tested with the FilmArray during discrepancy resolution. b Testing results include the 24 confirmed positive specimens; of the 28 specimens referred, the four unconfirmed specimens were analyzed as reference negative specimens. The 36 reference negative specimens included these 4 specimens, 18 specimens referred as negative, and 14 Coronavirus 229E ‘referred positive’ specimens.

# FilmArray Cp and Tm values for Coronavirus OC43 specimens were as follows:

Coronavirus OC43 Specimen Cp and Tm Values   

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=1 colspan=2>Cp</td><td rowspan=1 colspan=3>Tm ()a</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Detection rangea</td></tr><tr><td rowspan=1 colspan=1>Cov OC43</td><td rowspan=1 colspan=1>11.8</td><td rowspan=1 colspan=1>7.6 - 22.8</td><td rowspan=1 colspan=1>81.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>78.7  83.8</td></tr></table>

a Tm limits as defined by the FilmArray software.

Results for Testing of Coronavirus 229E (Panel 10)a   

<table><tr><td rowspan=2 colspan=1>FilmArrayResults</td><td rowspan=1 colspan=3>Reference</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>2a</td><td rowspan=1 colspan=1>15</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>45</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>47</td><td rowspan=1 colspan=1>60b</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 13/13    100.0% (95%CI: 73.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 45/47   95.7% (95%CI: 85.5% - 99.5%)</td></tr></table>

a Specimen 5805-2010 had been identified by the source laboratory as positive for Coronavirus 229E but the analyte status could not be confirmed (see above). b Testing included the 13 confirmed specimens; the unconfirmed specimen was analyzed as a reference negative specimen. The 47 reference negative specimens included this one specimen, 18 specimens referred as negative, and 32 Coronavirus OC43 positive specimens.

FilmArray Cp and Tm values for Coronavirus 229E specimens were as follows:

# Coronavirus 229E Specimen Cp and Tm Values

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=1 colspan=2>Cp</td><td rowspan=1 colspan=3>Tm (C)</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Detection rangea</td></tr><tr><td rowspan=1 colspan=1>Cov 229E</td><td rowspan=1 colspan=1>15.7</td><td rowspan=1 colspan=1>9.7-21.2</td><td rowspan=1 colspan=1>81.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>78.7 - 83.8</td></tr></table>

a Tm limits as defined by the FilmArray software.

# x. Overall Results of Testing Specimens from Study 299 and Additional Specimens

The retrospective performance data are presented in the following table by analyte:

Results of Testing of Archived Specimens   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=2>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=2>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>27/27</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.2% - 100.0%</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.7% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>B. pertussis,b</td><td rowspan=1 colspan=1>53/56</td><td rowspan=1 colspan=1>94.6%</td><td rowspan=1 colspan=1>85.1 - 98.9%</td><td rowspan=1 colspan=1>56/58</td><td rowspan=1 colspan=1>96.5%</td><td rowspan=1 colspan=1>88.1% - 99.6%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43a,c</td><td rowspan=1 colspan=1>24/24</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>85.8 - 100%</td><td rowspan=1 colspan=1>33/36</td><td rowspan=1 colspan=1>91.7%</td><td rowspan=1 colspan=1>77.5% - 98.2%)</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229Ea,c</td><td rowspan=1 colspan=1>13/13</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>73.1% - 100.0%</td><td rowspan=1 colspan=1>45/47</td><td rowspan=1 colspan=1>95.7</td><td rowspan=1 colspan=1>85.5% - 99.5%</td></tr><tr><td rowspan=1 colspan=1>Enterovirus</td><td rowspan=1 colspan=1>22/23</td><td rowspan=1 colspan=1>95.7%</td><td rowspan=1 colspan=1>78.0% - 99.9%</td><td rowspan=1 colspan=1>90/90</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.0% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>32/32</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.1% - 100.0%</td><td rowspan=1 colspan=1>127/127</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>54/54</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>93.4% - 100.0%</td><td rowspan=1 colspan=1>105/105</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.5% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>34/34</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>89.7% - 100.0%</td><td rowspan=1 colspan=1>125/125</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>Flu B</td><td rowspan=1 colspan=1>30/30</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>88.4% - 100.0%</td><td rowspan=1 colspan=1>129/129</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>97.2% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>M. pneumoniaead</td><td rowspan=1 colspan=1>54/64</td><td rowspan=1 colspan=1>84.4%</td><td rowspan=1 colspan=1>73.1% - 94.0%</td><td rowspan=1 colspan=1>58/65</td><td rowspan=1 colspan=1>89.2.1%</td><td rowspan=1 colspan=1>79.1% - 95.6%</td></tr><tr><td rowspan=1 colspan=1>PIV-1</td><td rowspan=1 colspan=1>34/35</td><td rowspan=1 colspan=1>97.1%</td><td rowspan=1 colspan=1>85.1% - 99.9%</td><td rowspan=1 colspan=1>94/94</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.2% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PIV-2</td><td rowspan=1 colspan=1>28/28</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>87.6% - 100.0%</td><td rowspan=1 colspan=1>101/101</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.4% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PIV-3</td><td rowspan=1 colspan=1>36/36</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>90.3% - 100.0%</td><td rowspan=1 colspan=1>93/93</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>96.1% - 100.0%</td></tr><tr><td rowspan=1 colspan=1>PIV-4</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>71.5% - 100.0%</td><td rowspan=1 colspan=1>6/6</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>54.1% - 100.0%</td></tr></table>

a These results include the outcome of additional testing of referred specimens. b B. pertussis results are only reported for the additional specimens obtained after Study 299 and does not include results of the original 33 specimens from Study 299 where only 19 specimens were confirmed. Confirmed samples only from Study 299, panel 6, demonstrated 17/17 $( 1 0 0 \% )$ sensitivity and 39/39 $( 1 0 0 \% )$ specificity (see above). c Specimens tested outside of Study 299 as Panel 10. $^ { \mathrm { d } } M .$ pneumoniae results include specimens from Study 299 (Panel 8) and the additional 36 specimens tested outside of Study 299 as Panel 9.

# xi. Additional testing of Contrived Chlamydophila pneumoniae specimens

In the original FilmArray prospective study one $C .$ . pneumoniae positive specimen was identified (1/1117) by FilmArray/reference testing; no false negative results were seen. An additional archived specimen was obtained from a source laboratory, and 2 additional specimens tested positive for $C .$ . pneumoniae when being evaluated for other pathogens, yielding a total of four $C .$ . pneumoniae specimens available for testing. Results for testing these four specimens are summarized below:

Results for Additional $C .$ pneumoniae-Confirmed Specimens   

<table><tr><td rowspan=3 colspan=1>SopecimenID#</td><td rowspan=1 colspan=5>FilmArray Respiratory Panel</td><td rowspan=3 colspan=1></td></tr><tr><td rowspan=2 colspan=1>PouchSerialNumber</td><td rowspan=1 colspan=3>Cpne</td><td rowspan=2 colspan=1>FilmArrayCpne Result</td></tr><tr><td rowspan=1 colspan=1>Result</td><td rowspan=1 colspan=1>MedianCp</td><td rowspan=1 colspan=1>MedianTm</td></tr><tr><td rowspan=1 colspan=1>8-052</td><td rowspan=1 colspan=1>00069018</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>23.9</td><td rowspan=1 colspan=1>80.4</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Source laboratory result was positive for M.pneumoniae; tested during archivedspecimen evaluation. (Sample was co-infected with M. pneumoniae and C.pneumoniae.)</td></tr><tr><td rowspan=1 colspan=1>5805-3058</td><td rowspan=1 colspan=1>00117867</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>16.5</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Source laboratory result was negative forB.pertussis; tested during archived specimenevaluation.</td></tr><tr><td rowspan=1 colspan=1>A-Cpne 005 00144044</td><td rowspan=1 colspan=1>A-Cpne 005 00144044</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>8.6</td><td rowspan=1 colspan=1>79.7</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Identified by a source laboratory as positivefor C. pneumoniae and intended for archivedspecimen evaluation.</td></tr><tr><td rowspan=1 colspan=1>3-003</td><td rowspan=1 colspan=1>00014864</td><td rowspan=1 colspan=1>Pos</td><td rowspan=1 colspan=1>14.2</td><td rowspan=1 colspan=1>79.8</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>Identified by ITI comparator assays (twoanalyte specific PCR assays followed by bi-directional sequencing); tested duringprospective clinical evaluation.</td></tr></table>

Due to the limited number of samples available, the sponsor performed an additional study of contrived $C$ pneumoniae specimens using retained negative clinical specimens from the FilmArray prospective study as sample matrix.

Fifty contrived specimens spiked with $C .$ . pneumoniae were developed for testing based on levels of $C .$ . pneumoniae detected in clinical specimens as reported by Kuoppa et al. (2002)1 . The distribution of contrived samples had the following characteristics: minimum value $> 0$ and less than ${ < } 1 5 0 0$ copies/ml, mode 9,000 – 11,000 copies/ml, mean approximately 15,000 copies $/ \mathrm { m l }$ , maximum $> 4 5 { , } 0 0 0$ copies $/ \mathrm { m l }$ , with $70 \%$ of the contrived specimens $< 1 0 { , } 0 0 0$ copies $/ \mathrm { m l }$ . The final distribution of samples was as follows:

C. pneumoniae Spiking Levels for Contrived Specimens   

<table><tr><td colspan="1" rowspan="1">Spiking Level(genome copies/ml)</td><td colspan="1" rowspan="1">Number of Spiked Samples</td></tr><tr><td colspan="1" rowspan="1">0 (negative)</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">600</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">3000</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">6000</td><td colspan="1" rowspan="1">12</td></tr><tr><td colspan="1" rowspan="1">9000</td><td colspan="1" rowspan="1">16</td></tr><tr><td colspan="1" rowspan="1">12000</td><td colspan="1" rowspan="1">8</td></tr><tr><td colspan="1" rowspan="1">15000</td><td colspan="1" rowspan="1">5</td></tr><tr><td colspan="1" rowspan="1">30000</td><td colspan="1" rowspan="1">2</td></tr><tr><td colspan="1" rowspan="1">Total</td><td colspan="1" rowspan="1">100 samples</td></tr></table>

Results of testing of the 100 contrived samples were as follows:

Results for Contrived $C .$ pneumoniae Sample Testing   

<table><tr><td rowspan="2">Film Array Result</td><td colspan="4">Contrived sample</td></tr><tr><td>Positive</td><td colspan="2">Negative</td><td>Total</td></tr><tr><td>Positive</td><td>50</td><td colspan="2">0</td><td>50</td></tr><tr><td>Negative</td><td>0</td><td colspan="2">50</td><td>50</td></tr><tr><td>Total</td><td>50</td><td colspan="2">50</td><td>100</td></tr><tr><td colspan="2">Positive Percent Agreement:</td><td>50/50</td><td>100.0%</td><td>(95%CI: 92.9 - 100.0%)</td></tr><tr><td colspan="2">Negative Percent Agreement:</td><td>50/50</td><td>100.0%</td><td>(95%CI: 92.9 - 100.0%)</td></tr></table>

FilmArray Cp and Tm values for the contrived samples were as follows:

FilmArray Cp and Tm Values for C. pneumoniae Contrived samples   

<table><tr><td rowspan=2 colspan=1>Assay</td><td rowspan=1 colspan=2>Cp</td><td rowspan=1 colspan=3>Tm ()</td></tr><tr><td rowspan=1 colspan=1>Median</td><td rowspan=1 colspan=1>Range</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Detection rangea</td></tr><tr><td rowspan=1 colspan=1>Cpne</td><td rowspan=1 colspan=1>21.8</td><td rowspan=1 colspan=1>19.9 -27.1</td><td rowspan=1 colspan=1>80.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>79.5 - 80.8</td></tr></table>

a Tm limits as defined by the FilmArray software.

To confirm the appropriateness of the spiking levels of $C .$ . pneumoniae contrived samples, an additional study of contrived specimens was conducted comparing the distribution of $\mathrm { C p }$ values with contrived specimens to $\mathrm { C p }$ values observed during investigation of a $C .$ . pneumoniae outbreak.2 In this study, the sponsor generated the $\mathrm { C p }$ values for contrived specimens using an assay reproduced following Thurman et al.3 Results were as follows:

Cp values for Contrived Specimens with Reproduced Assay   

<table><tr><td colspan="1" rowspan="1">Spike Level(genome copies/mL)</td><td colspan="1" rowspan="1">VIN</td><td colspan="1" rowspan="1">Cp Value</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">6125AD-007</td><td colspan="1" rowspan="1">37.0</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">6125AD-010</td><td colspan="1" rowspan="1">38.0</td></tr><tr><td colspan="1" rowspan="1">95</td><td colspan="1" rowspan="1">6125AD-017</td><td colspan="1" rowspan="1">38.2</td></tr><tr><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">6125AD-003</td><td colspan="1" rowspan="1">37.9</td></tr><tr><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">6125AD-009</td><td colspan="1" rowspan="1">38.7</td></tr><tr><td colspan="1" rowspan="1">300</td><td colspan="1" rowspan="1">6125AD-013</td><td colspan="1" rowspan="1">37.5</td></tr><tr><td colspan="1" rowspan="1">950</td><td colspan="1" rowspan="1">6125AD-004</td><td colspan="1" rowspan="1">35.1</td></tr><tr><td colspan="1" rowspan="1">950</td><td colspan="1" rowspan="1">6125AD-008</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">950</td><td colspan="1" rowspan="1">6125AD-012</td><td colspan="1" rowspan="1">37.0</td></tr><tr><td colspan="1" rowspan="1">3,000</td><td colspan="1" rowspan="1">6125AD-001</td><td colspan="1" rowspan="1">34.8</td></tr><tr><td colspan="1" rowspan="1">3,000</td><td colspan="1" rowspan="1">6125AD-002</td><td colspan="1" rowspan="1">35.3</td></tr><tr><td colspan="1" rowspan="1">3,000</td><td colspan="1" rowspan="1">6125AD-018</td><td colspan="1" rowspan="1">35.0</td></tr><tr><td colspan="1" rowspan="1">9,500</td><td colspan="1" rowspan="1">6125AD-006</td><td colspan="1" rowspan="1">33.7</td></tr><tr><td colspan="1" rowspan="1">9,500</td><td colspan="1" rowspan="1">6125AD-014</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">9,500</td><td colspan="1" rowspan="1">6125AD-015</td><td colspan="1" rowspan="1">33.6</td></tr><tr><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">6125AD-005</td><td colspan="1" rowspan="1">31.8</td></tr><tr><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">6125AD-011</td><td colspan="1" rowspan="1">32.5</td></tr><tr><td colspan="1" rowspan="1">30,000</td><td colspan="1" rowspan="1">6125AD-016</td><td colspan="1" rowspan="1">32.8</td></tr></table>

Average Cp values with Standard Deviations (SD)   

<table><tr><td rowspan=2 colspan=1>Spike Level(genome copies/mL)</td><td rowspan=1 colspan=2>Cp values</td><td rowspan=2 colspan=1>Delta</td></tr><tr><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>950</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>-0.6</td></tr><tr><td rowspan=1 colspan=1>3,000</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>+0.1</td></tr><tr><td rowspan=1 colspan=1>9,500</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>-0.7</td></tr><tr><td rowspan=1 colspan=1>30,000</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>-0.1</td></tr></table>

Comparisons of the $\mathrm { C p }$ values generated from this study with values reported by Thurman et al. from clinical specimens demonstrated that the spiking levels of the contrived specimens approximate what may be expected with clinical use.

# 4. Clinical cut-off

Not applicable

# 5. Expected values/Reference ranges

Expected Value (As Determined by the FilmArray RP) Summary by Site for the FilmArray RP First Phase Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="1">Overall (n=853)</td><td colspan="2" rowspan="1">Site 1 (n=275)</td><td colspan="2" rowspan="1">Site 2 (n=333)</td><td colspan="2" rowspan="1">Site 3 (n=245)</td></tr><tr><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">4.5%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">9.0%</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">3.9%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">2.7%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2.7%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">4.1%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">11.0%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.4%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">12.3%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">16.7%</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Entero</td><td colspan="1" rowspan="1">225</td><td colspan="1" rowspan="1">26.4%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">13.1%</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">27.6%</td><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">39.6%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Flu A/H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Flu A/H3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Flu A/2009 H1N1</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">3.9%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">12.7%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">139</td><td colspan="1" rowspan="1">16.3%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">25.8%</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">20.0%</td></tr></table>

Expected Value (As Determined by the FilmArray RP) Summary by Site for the FilmArray RP Second Phase Prospective Clinical Evaluation (September 2010 – January 2011)   

<table><tr><td rowspan=2 colspan=1>Analyte</td><td rowspan=1 colspan=2>Overall</td><td rowspan=1 colspan=2>Site 2 (n = 180)</td><td rowspan=1 colspan=2>Site 3 (n = 84)</td></tr><tr><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11.9%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11.9%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Entero</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>47.3%</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>50.6%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>39.1%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>ParainfluenzaVirus 3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>11.7%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>23.8%</td></tr></table>

Expected Value (as Determined by FilmArray RP) Summary by Age Group for First Phase Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total(Expected Value)</td><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>≥ 50 years</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38 (4.5%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>4 (0.4%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>25 (2.9%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23 (2.7%)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>8 (0.9%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>14 (1.6%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>94 (11.0%)</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Entero</td><td rowspan=1 colspan=1>225 (26.4%)</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Flu A/H1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A/H3</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Flu A/2009 H1N1</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>33 (3.9%)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>8 (0.9%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>139 (16.3%)</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td></tr></table>

Expected Value (As Determined by FilmArray RP) Summary by Age Group for FilmArray RP Second Phase Prospective Clinical Evaluation (September 2010 – January 2011)   

<table><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Total(Expected Value)</td><td colspan="1" rowspan="1">≤ 5 years</td><td colspan="1" rowspan="1">6-21 years</td><td colspan="1" rowspan="1">22-49 years</td><td colspan="1" rowspan="1">≥ 50 years</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">15 (5.7%)</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">3 (1.1%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1 (0.4%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">11 (4.2%)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Human Metapneumovirus</td><td colspan="1" rowspan="1">4 (1.5%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus/Entero</td><td colspan="1" rowspan="1">125 (47.3%)</td><td colspan="1" rowspan="1">118</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">5 (1.9%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Flu A/H1</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Flu A/H3</td><td colspan="1" rowspan="1">5 (1.9%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Flu A/2009 H1N1</td><td colspan="1" rowspan="1">0 (0%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">1 (0.4%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">2 (0.8%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 1</td><td colspan="1" rowspan="1">1 (0.4%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 2</td><td colspan="1" rowspan="1">9 (3.4%)</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 3</td><td colspan="1" rowspan="1">5 (1.9%)</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus 4</td><td colspan="1" rowspan="1">2 (0.7%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">31 (11.7%)</td><td colspan="1" rowspan="1">30</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td></tr></table>

Expected Value (Co-infections as Determined by FilmArray RP) Summary by Age Group for FilmArray RP First Phase Prospective Clinical Evaluation (December 2009 – May 2010)   

<table><tr><td colspan="1" rowspan="1">Co-Infection</td><td colspan="1" rowspan="1">Total(Expected Value)</td><td colspan="1" rowspan="1">≤ 5 Years</td><td colspan="1" rowspan="1">6-21 Years</td><td colspan="1" rowspan="1">22-49 Years</td><td colspan="1" rowspan="1">≥ 50 Years</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + RSV</td><td colspan="1" rowspan="1">21 (2.46%)</td><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + Adenovirus</td><td colspan="1" rowspan="1">8 (0.94%)</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + PIV3</td><td colspan="1" rowspan="1">8 (0.94%)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + hMPV</td><td colspan="1" rowspan="1">7 (0.82%)</td><td colspan="1" rowspan="1">7</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">hMPV + RSV</td><td colspan="1" rowspan="1">4 (0.47%)</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + CoV NL63</td><td colspan="1" rowspan="1">4 (0.47%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 + hMPV</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 + HRV/Entero</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 + RSV</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 + hMPV</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 + RSV</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">hMPV + PIV3</td><td colspan="1" rowspan="1">3 (0.35%)</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + HRV/Entero + PIV3</td><td colspan="1" rowspan="1">2 (0.23%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV OC43 + RSV</td><td colspan="1" rowspan="1">2 (0.23%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 + CoV OC43</td><td colspan="1" rowspan="1">2 (0.23%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">1</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + PIV4</td><td colspan="1" rowspan="1">2 (0.23%)</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + hMPV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + PIV3</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + CoV NL63</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + RSV+CoV 229E</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + HRV/Entero +B.pertussis</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">Adenovirus + C. pneumoniae</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV 229E + RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV 229E + CoV NL63 +HRV/Entero+RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV 229E + HRV/Entero</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1 + HRV/Entero + RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">CoV NL63 + hMPV + RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + B. pertussis</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">HRV/Entero + PIV1</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">hMPV + PIV4</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr><tr><td colspan="1" rowspan="1">PIV4 + RSV</td><td colspan="1" rowspan="1">1 (0.12%)</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td></tr></table>

Expected Value (Co-infections as Determined by FilmArray RP) Summary by Age Group for FilmArray RP Second Phase Prospective Clinical Evaluation (September 2010 – January 2011)   

<table><tr><td rowspan=1 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Total(ExpectedValue)</td><td rowspan=1 colspan=1>≤5 Years</td><td rowspan=1 colspan=1>6-21 Years</td><td rowspan=1 colspan=1>22-49 Years</td><td rowspan=1 colspan=1>≥ 50 Years</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + RSV</td><td rowspan=1 colspan=1>6 (2.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + Adenovirus</td><td rowspan=1 colspan=1>6 (2.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + HRV/Entero</td><td rowspan=1 colspan=1>5 (1.9%)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV2</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV3</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + PIV4</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Adenovirus + CoV OC43</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + hMPV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/Entero + B. pertussis</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>PIV2 + M. pneumoniae</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# N. Instrument Name:

FilmArray Instrument.

# O. System Descriptions:

1. Modes of Operation:

See Section, I. Device Description, 3, The FilmArray Instrument, on page 5 above.

2. Software

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes X or No

3. Specimen Identification User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling Not applicable

5. Calibration Not applicable

6. Quality Control

The FilmArray Respiratory Panel (RP) pouch contains two internal control assays:

a. The RNA Process Control targets an mRNA of the yeast, Schizosaccharomyces pombe. During FilmArray RP pouch manufacture, whole yeast are freeze-dried into the sample injection port of each pouch. When the test specimen is loaded into the pouch, S. pombe is rehydrated and enters the pouch with the specimen. The yeast nucleic acid is extracted, purified and tested simultaneously with nucleic acids from the patient specimen. A positive result for the processing control indicates that all steps in the process (nucleic acid extraction, reverse transcription, PCR, melt, detection, and analysis) are functioning properly.   
b. The second stage PCR (PCR2) control assay detects a synthetic DNA template that is dried into triplicate wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

The RNA Process Control and the PCR2 Control assays are used to Pass or Fail each FilmArray RP pouch run. This combination of control assays monitors each of the critical mechanical and chemical processes that occur in a pouch run, while limiting the possibility of random control assay failures that could contribute to unnecessary pouch failures.

Good laboratory practice recommends running external positive and negative controls regularly. Use viral transport medium as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.

P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.