# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k121003   
B. Purpose for Submission: New device   
C. Measurand: Influenza A and Influenza B RNA from nasopharyngeal swabs

# D. Type of Test:

The PLEX-ID Flu device is a qualitative in vitro assay for identification of influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal) and influenza B from nasopharyngeal swabs collected from patients with signs and symptoms of influenza. The PLEX-ID Flu assay uses RT-PCR amplification followed by analysis using mass spectrometry to evaluate base composition of the PCR products.

E. Applicant: Abbott Molecular Inc.   
F. Proprietary and Established Names: PLEX-ID Flu   
G. Regulatory Information:

1. Regulation section: 866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code: OCC, OEP, OQW, OTA

4. Panel:

Microbiology

# H. Intended Use:

1. Intended use(s):

The PLEX-ID Flu assay is a qualitative nucleic acid in vitro diagnostic test intended for the detection and differentiation of influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal) and influenza B viral nucleic acids in nasopharyngeal swab specimens from patients symptomatic for respiratory tract infection. The PLEX-ID Flu assay is intended for use on the PLEX-ID System (version 1.2) as an aid in the diagnosis of influenza infection in conjunction with clinical and epidemiological information. This assay is not intended to detect influenza C virus.

Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions.

If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $^ { 3 + }$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): Prescription Use

4. Special instrument requirements:

PLEX- $. \mathrm { I D } _ { \mathrm { S P } }$ (Sample Preparation System) PLEX- $\cdot \mathrm { I D } _ { \mathrm { F H } }$ (Fluid Handler System) PLEX- $\cdot \mathrm { I D } _ { \mathrm { T C } }$ (Thermal Cycler) PLEX-ID Analyzer (version 1.2)

# I. Device Description:

The PLEX-ID Flu Assay provides identification of influenza A subtypes and detection of B viruses. It differentiates the 2009 H1N1 influenza A from seasonal human influenza A/H1N1 and A/H3N2 subtypes. The PLEX-ID Flu assay is a qualitative in vitro diagnostic test used for the detection and identification directly from nasopharyngeal

(NP) swab specimens. First, nucleic acids are extracted from the NP specimens collected from patient with signs and symptoms of influenza infection using the PLEX-ID SP. Nucleic acids are then amplified through a reverse transcription polymerase chain reaction (RT-PCR) on the PLEX-ID TC. The RT-PCR products are then desalted and analyzed in a mass spectrometer to determine the base composition of the amplicons. Analysis of base compositions of PCR amplicons from influenza core gene segments (PB1, PB2, PA, M, NS1, NP) are used to provide subspecies identification via comparison to an influenza virus database of expected amplicons. Desalting, mass spectrometry and data analysis are conducted on the PLEX-ID Analyzer (version 1.2). Reported results include an identification of detected influenza A virus subtypes (seasonal H1, 2009 H1N1, and seasonal H3) and detection of influenza B virus.

The following is a list of reagent kits and instruments required to conduct the PLEX-ID Flu Assay:

• PLEX-ID Flu Amplification Kit (List No. 05N21-91)   
• PLEX-ID Flu Control Kit (List No. 05N21-80)   
• Kit 1 of the PLEX-ID Viral RNA Isolation Kit (05N80-10)   
• Kit 2 of the PLEX-ID Viral RNA Isolation Kit (05N80-20)   
• PLEX-ID (Instrument) System consisting of multiple pieces of equipment and software: • PLEX-ID FH • PLEX-ID SP • PLEX-ID TC • PLEX-ID Analyzer (version 1.2) • PLEX-ID Software v1.2   
• PLEX-ID System Reagents • PLEX-ID Analyzer Check Kit • PLEX-ID Clean-Up Reagents • PLEX-ID Clean-Up Microparticles

Table 1: Primer sets in the PLEX ID Flu Assay   

<table><tr><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Primer PairName</td></tr><tr><td rowspan=1 colspan=1>Influenza Virus</td><td rowspan=1 colspan=1>Polymerase (PB1)</td><td rowspan=1 colspan=1>2798</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Nucleoprotein</td><td rowspan=1 colspan=1>1266</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Matrix Protein</td><td rowspan=1 colspan=1>1279</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Polymerase</td><td rowspan=1 colspan=1>1287</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Non-Structural Protein</td><td rowspan=1 colspan=1>2775</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=1>Polymerase</td><td rowspan=1 colspan=1>1259</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus H1N1 (2009)</td><td rowspan=1 colspan=1>Hemagglutinin</td><td rowspan=1 colspan=1>5101</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1N1</td><td rowspan=1 colspan=1>Neuraminidase</td><td rowspan=1 colspan=1>4998</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=1>Polymerase (PB2)</td><td rowspan=1 colspan=1>1261</td></tr></table>

# J. Substantial Equivalence Information:

1. Predicate device name(s): Prodesse ProFlu+ Prodesse ProFAST+

2. Predicate 510(k) number(s): k073029, k101855

3. Comparison with predicate:

Table 2: Predicate comparison   

<table><tr><td rowspan="2">Feature</td><td rowspan="2">Current Application PLEX-ID Flu</td><td colspan="2">Predicate Devices</td></tr><tr><td>Prodesse ProFLU+</td><td>Prodesse ProFAST+</td></tr><tr><td>510(k) Number</td><td>k121003</td><td>k073029</td><td>k101855</td></tr><tr><td>Regulation No. and Product Code</td><td>21 CFR 866.3980 OEP, OCC, OQW, OTA</td><td>21 CFR 866.3980 OCC</td><td>21 CFR 866.3332 OQW</td></tr><tr><td>Feature</td><td>Current Application</td><td>Predicate Devices</td><td>Prodesse ProFAST+</td></tr><tr><td>Intended Use</td><td>PLEX-ID Flu The PLEX-ID Flu assay is a qualitative nucleic acid in vitro</td><td>Prodesse ProFLU+ The ProFluTM+ Assay is a multiplex Real-Time PCR (RT-PCR) in vitro diagnostic test for the rapid and qualitative detection and</td><td>The ProFASTTM+ Assay is a multiplex Real Time RT-PCR in vitro diagnostic test for the qualitative detection and</td></tr><tr><td></td><td>detection and differentiation of influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal) and influenza B viral nucleic acids in nasopharyngeal swab specimens from patients symptomatic for respiratory tract infection. The PLEX-ID Flu assay is intended for use on the PLEX-ID System (version 1.2) as an aid in the diagnosis of influenza infection in conjunction with clinical and epidemiological information. This assay is not intended to detect influenza C virus. Negative results do not preclude influenza virus infection and should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. If infection with a novel influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL 3+ facility is available to receive and culture specimens.</td><td>discrimination of Influenza A Virus, Influenza B Virus, and Respiratory Syncytial Virus (RSV) nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens obtained from symptomatic patients. This test is intended for use to aid in the differential diagnosis of Influenza A, Influenza B and RSV viral infections in humans and is not intended to detect Influenza C. Negative results do not preclude influenza or RSV virus infection and should not be used as the sole basis for treatment or other management decisions. It is recommended that negative RSV results be confirmed by culture. Performance characteristics for Influenza A Virus were established when Influenza A/H3 and A/H1 were the predominant Influenza A viruses in circulation. When other Influenza A viruses are emerging, performance characteristics may vary. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL 3+ facility is available to receive and culture specimens.</td><td>discrimination of seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza viral nucleic acids isolated and purified from nasopharyngeal (NP) swab specimens from human patients with signs and symptoms of respiratory infection in conjunction with clinical and epidemiological risk factors. This assay targets conserved regions of the Hemagglutinin (HA) gene for seasonal Influenza A/H1, seasonal Influenza A/H3 and 2009 H1N1 Influenza Virus, respectively. This assay is not intended to detect Influenza B or Influenza C Viruses. A negative ProFAST+ Assay result is a presumptive negative result for Influenza A. These results should be confirmed by an FDA cleared nucleic acid-based test (NAT) detecting Influenza A. Negative results do not preclude Influenza virus infection and should not be used as the sole basis for treatment or other patient management decisions. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health department for testing. Viral culture should not be attempted in these cases unless a BSL 3+ facility is available to receive and culture specimens.</td></tr><tr><td>Feature</td><td></td><td>Current Application</td><td colspan="2">Predicate Devices</td></tr><tr><td></td><td></td><td>PLEX-ID Flu</td><td>Prodesse ProFLU+</td><td>Prodesse ProFAST+ Qualitative</td></tr><tr><td>Assay Type Analyte Targets</td><td>•</td><td>Qualitative • influenza genus→ Primer pair 2798 targets polymerase PB1 influenza A → Primer Pair 1266 targets nucleoprotein influenza A → Primer Pair 1279 targets matrix protein influenza A → Primer Pair 1287 polymerase A protein</td><td>Qualitative •Influenza A Virus → Matrix Respiratory Syncytial Virus A →Polymerase Respiratory Syncytial Virus B →Polymerase •Influenza B Virus → Non-structural</td><td>Seasonal H1 Influenza A → Hemagglutinin Seasonal H3 Influenza A → Hemagglutinin 2009 H1N1 Influenza Virus → Hemagglutinin</td></tr><tr><td></td><td>Primer Pair 4998 targets neuraminidase, NA influenza B→ Primer Pair</td><td>influenza A H1N1 (2009) → 1261 targets polymerase, PB2</td><td></td><td></td><td></td></tr><tr><td>Input Sample Types Sample</td><td></td><td>nasopharyngeal swab M4R, M4RT, M5, M6; Copan</td><td>nasopharyngeal swab M4, M4RT, M5, M6; Copan UTM; or Becton Dickenson</td><td></td><td>nasopharyngeal swab M4, M4RT, M5, M6;</td></tr><tr><td>Collection Input Sample</td><td>•</td><td>UTM; or Becton Dickenson UVT 300 μL</td><td>UVT 180 μL</td><td></td><td>Copan UTM; or Becton Dickenson UVT 180 μL</td></tr><tr><td>Volume Principles of the Procedure</td><td>•</td><td>Automated sample to extraction plate Automated nucleic acid extraction Automated transfer of sample eluate to PCR Plate</td><td>Automated nucleic acid isolation Manual transfer of sample eluate to PCR Plate Automated PCR amplification Optical detection of stimulated</td><td></td><td>Automated nucleic acid isolation Manual transfer of sample eluate to PCR Plate Automated PCR</td></tr></table>

<table><tr><td>Feature</td><td>Current Application</td><td colspan="2">Predicate Devices</td></tr><tr><td>Instrumentation</td><td>PLEX-ID Flu The PLEX-ID System</td><td>Prodesse ProFLU+</td><td>Prodesse ProFAST+</td></tr><tr><td>Principle System Components</td><td>integrates sample preparation (PLEX-ID SP, PLEX-ID FH) with PCR amplification (PLEX-ID TC) followed by mass spectrometry analysis to generate assay results (PLEX-</td><td>Nucleic Acid isolation instrument Amplification and detection instrument</td><td>Nucleic Acid isolation instrument Amplification and detection instrument</td></tr><tr><td>Detection Procedure</td><td>• Mass Spectrometry analysis • converts mass of PCR products to base composition Base composition signature from multiple PCR products is then used to identify the influenza virus species and subtype</td><td>Optical detection of stimulated • fluorescence. The fluorescence reader monitors real-time fluorescence during every PCR amplification cycle.</td><td>Optical detection of stimulated fluorescence. The fluorescence reader monitors real-time fluorescence during every PCR amplification cycle.</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

Not Applicable

# L. Test Principle:

The PLEX-ID Flu assay is a test for the detection of influenza virus nucleic acid extracted from nasopharyngeal swabs from symptomatic patients. PLEX-ID Viral RNA Isolation reagents are used to adsorb, wash, and elute nucleic acids on a PLEX-IDSP resulting in purity sufficient for a reverse transcription-polymerase chain reaction (RTPCR). The PLEX-IDFH loads eluted nucleic acid from each sample and assay reagents (reverse transcriptase, DNA-binding protein, and DNA polymerase) into an assay plate containing broad-range primers targeting influenza A and B viruses. RT-PCR amplification is performed on the PLEX-IDTC.

After the amplification step, the PCR plate is loaded onto the PLEX-ID Analyzer for PCR product desalting via weak anion exchange with microparticles. After desalting, the PLEX-ID Analyzer uses electrospray ionization mass spectrometry (ESI-MS) to measure the masses of the desalted PCR products. PCR products are composed of the four DNA building blocks, adenine (A), guanine (G), cytosine (C), and thymine (T). The atomic composition and thus, the molecular mass, of each nucleotide is known. The PLEX-ID Analyzer calculates the base composition, the number of As, Gs, Cs, and Ts from the mass of each PCR product. The base composition signatures from all detected segments are compared with a database of base compositions of the known types and subtypes of influenza viruses to identify the viral species present in the sample.

# Sample Extraction

Nucleic acids are isolated and purified from samples and controls during the sample preparation process. The nucleic acid isolation step uses a non-specific capture chemistry. Nucleic acids in the sample are bound to magnetic beads and non-nucleic acid sample material is washed away during multiple wash steps. Purified nucleic acids are then released from the magnetic beads with an elution buffer.

# PCR Plate Setup and Amplification

The PLEX-ID Flu Assay Plate is a sealed, 96-well PCR plate with wells that are pre-filled with PCR components. PCR plate setup takes place on the PLEX-IDFH. The purified nucleic acid solution from each processed sample and assay control, along with PCR amplification reagents (reverse transcriptase, stabilization reagent, and DNA Polymerase), is distributed to 8 wells, each of which contains a unique PCR primer set. PCR amplification takes place on the PLEX-IDTC.

# Desalting of PCR Products

Following PCR amplification, the PCR plate is placed on the PLEX-ID Analyzer, and the PCR product mixtures are desalted using a weak anion exchange method. Desalting, or purification of the amplified product, is a necessary step prior to analysis by mass spectrometry. The desalting process involves binding PCR products to magnetic microparticles, followed by a series of washes. Unconsumed deoxyribonucleoside triphosphates (dNTPs), salts, and other low molecular weight species that could interfere with the subsequent ESI-MS analysis are removed by rinsing the microparticles with solutions containing volatile salts and organic solvents. The final purified/desalted PCR products are eluted with a high $\mathsf { p H }$ buffer solution containing methanol and piperidine imidazole.

# Mass Spectrometry Analysis

The purified nucleic acids from each well are injected into an electrospray ionization time-of-flight mass spectrometer (ESI TOF-MS) that generates a spectral signal based upon the mass-to-charge ratio of each product. The spectral signals for each product are processed to determine its mass. Peptide-based mass standards, which are added during the elution step of the desalting process, are used to bracket the expected mass-to-charge range.

# Conversion of Mass to Base Composition

A software algorithm converts the mass information to base composition using the exact masses of each of the bases that comprise DNA and the known information about the primer sets present in each well. The complementary nature of the forward and reverse strands of each DNA product reduces the number of possible base composition solutions.

# Influenza Virus Identification

The PLEX-ID Analyzer identifies the influenza virus species and subtype in the sample by comparing the base composition signature of the observed products to a database that links base composition signature to influenza virus species and subtype identity.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Reproducibility:

The reproducibility of the PLEX-ID Flu assay was evaluated at 3 clinical sites. Reproducibility was assessed using a nine member panel of contrived samples that included a negative member, low positive panel members targeted near the assay LOD and moderate positive panel members targeted at approximately 2 to 3 times the LOD for influenza B and the 3 influenza A subtypes detected by the assay. Samples were tested in replicates of 4 with one replicate each of the positive and negative control per batch. Two (2) batches were run per day and each was run by a different operator with 2 operators per site. The study was designed to allow for a total of 120 results for each panel member over 5 days of testing. The PLEX-ID Flu Assay demonstrated reproducible performance within and across all three sites for all viral levels tested when compared to the expected outcome. The summary of the results obtained from the evaluation of multisite reproducibility are presented in Table 3.

Table 3: PLEX-ID Flu Assay Multisite Reproducibility   

<table><tr><td colspan="1" rowspan="2">Panel Member Description</td><td colspan="4" rowspan="1">Correct Results/Number Tested(% Agreement)</td></tr><tr><td colspan="1" rowspan="1">Site 1</td><td colspan="1" rowspan="1">Site 2</td><td colspan="1" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total of All Sites(95% CI)</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">120/120 (100.0%)(96.9, 100.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (2009) LowPositive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">39/39a(100.0%)</td><td colspan="1" rowspan="1">38/40(95.0%)</td><td colspan="1" rowspan="1">117/119a (98.3%)(94.1, 99.5)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (2009)Moderate Positive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">120/120 (100.0%)(96.9, 100.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (seasonal)Low Positive</td><td colspan="1" rowspan="1">39/40(97.5%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">39/40(97.5%)</td><td colspan="1" rowspan="1">118/120 (98.3%)(94.1, 99.5)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1N1 (seasonal)Moderate Positive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">120/120 (100%)(96.9, 100.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (seasonal)Low Positive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">39/39a(100.0%)</td><td colspan="1" rowspan="1">119/119a (100.0%)(96.9, 100.0)</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3N2 (seasonal)Moderate Positive</td><td colspan="1" rowspan="1">39/40(97.5%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">119/120 (99.2%)(95.4, 99.9)</td></tr><tr><td colspan="1" rowspan="1">Influenza BLow Positive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">39/39a(100.0%)</td><td colspan="1" rowspan="1">39/40(97.5%)</td><td colspan="1" rowspan="1">118/119a (99.2%)(95.4, 99.9)</td></tr><tr><td colspan="1" rowspan="1">Influenza BModerate Positive</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">40/40(100.0%)</td><td colspan="1" rowspan="1">39/40(97.5%)</td><td colspan="1" rowspan="1">119/120 (99.2%)(95.4, 99.9)</td></tr></table>

# b. Precision

Within laboratory precision of the PLEX-ID Flu Assay was determined by testing all nine members of the reproducibility panel in duplicate in two runs per day for a total of twelve days. Each run contained the assay positive and negative controls for a total of 20 samples per run. Multiple PLEX-ID Flu assay lots were used in the study. The PLEX-ID Flu Assay demonstrated acceptable precision performance over the twelve day test period. Table 4 is a summary of the precision results.

Table 4: PLEX-ID Flu Assay Within Laboratory Precision   

<table><tr><td colspan="1" rowspan="1">Panel Member Description</td><td colspan="1" rowspan="1">Correct Results/Number Tested(% Agreement)</td><td colspan="1" rowspan="1">95%ConfidenceInterval</td></tr><tr><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H1N1 (2009)Low Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H1N1 (2009)Moderate Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H1N1 (seasonal)Low Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H1N1 (seasonal)Moderate Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H3N2 (seasonal)Low Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza A H3N2 (seasonal)Moderate Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza BLow Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr><tr><td colspan="1" rowspan="1">influenza BModerate Positive</td><td colspan="1" rowspan="1">48/48 (100%)</td><td colspan="1" rowspan="1">93%, 100%</td></tr></table>

b. Linearity/assay reportable range: Not Applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Controls provided with the PLEX ID Flu Assay:

Internal Control

Each PCR well contains an Amplification Control. The control consists of an in vitro synthesized RNA molecule specific to the primer pairs used, but distinguishable from known target sequences. The Amplification Control serves as the internal PCR control and provides scaling information for the level of influenza RNA present in the sample. The level of influenza detected is compared to a reporting cut-off. Although Amplification Control product may not be detected in the presence of high levels of influenza, the total amount of Amplification Control and influenza product detected must exceed a validity threshold.

# Positive Controls

A Positive Control, consisting of inactivated, but structurally intact, influenza A and B viruses, is run with the PLEX-ID Flu assay as an indicator of nucleic acid purity and recovery during sample preparation and also as a measure of successful amplification and detection. The control is processed in the same manner as clinical samples. Control material is prepared from highly purified virus particles which have been isolated from cell culture and chemically and enzymatically treated to alter their surface proteins. The resulting viral particles and the nucleic acid are intact, but unable to bind to or penetrate host cells. This control also serves as the process control for the entire assay and must be run with each extraction batch. If the positive control is “negative”, it may be indicative of incomplete extraction, amplification problems, or PLEXID Flu assay reagent degradation. All clinical samples within the run are invalid and must be repeated, starting with sample preparation. If the positive control results in an error code, all clinical samples within the run are invalid and must be repeated, starting with sample preparation.

The negative control for the PLEX-ID Flu assay consists of viral transport media. This control is processed in the same manner as samples and is run in each extraction batch. If the negative control is “positive”, it is indicative of possible contamination by other samples or by amplified product introduced during sample preparation or during preparation of the PLEX-ID Flu Assay Plate. All clinical samples within the run are invalid and must be repeated, starting with sample preparation. If the negative controls results in an error code, all clinical samples within the run are invalid and must be repeated, starting with sample preparation.

# d. Detection limit:

The limit of detection (LOD) for the PLEX-ID Flu assay was determined by testing quantified $\mathrm { ( T C I D _ { 5 0 } / m L ) }$ influenza strains diluted in negative nasopharyngeal swab matrix. Negative sample matrix was prepared from 464 pre-screened negative NP clinical swabs. Twenty confirmation replicates of each of the following were tested: influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal), and influenza B. The LOD was defined as the lowest influenza concentration at which $> 9 5 \%$ of replicates tested positive. The claimed LOD is summarized in the table below.

Table 5: PLEX-ID Flu Limit of Detection   

<table><tr><td rowspan=1 colspan=1>Species/Subtype Tested</td><td rowspan=1 colspan=1>Limit of Detection (TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>2009 (Pandemic) H1N1 Influenza</td><td rowspan=1 colspan=1>1.3 x 10</td></tr><tr><td rowspan=1 colspan=1>Seasonal H1N1 Influenza A</td><td rowspan=1 colspan=1>4.6</td></tr><tr><td rowspan=1 colspan=1>Seasonal H3N2 Influenza A</td><td rowspan=1 colspan=1>3.3x</td></tr><tr><td rowspan=1 colspan=1>Seasonal Influenza B</td><td rowspan=1 colspan=1>15</td></tr></table>

# e. Analytical specificity:

Cross reactivity:

The effect of 34 common respiratory microbes on the PLEX-ID Flu assay was evaluated. Negative samples and samples containing influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal), and influenza B were also analyzed in the presence of the potentially crossreacting microbes. Each potentially cross-reacting bacterial strain was acquired from ATCC or other sources, then re-grown and quantified. Each potentially cross-reacting virus was acquired from and quantified by ZeptoMetrix Corporation. All dilutions were made in Universal Transport

Medium (UTM). Test levels for each common respiratory microbe used in the analysis are summarized in table 6.

This study was conducted in two parts, with each part analyzing two of the target analytes tested with each of potential cross-reactants. For this analysis Influenza A 2009 H1N1 and Influenza B target were analyzed together and Influenza A H1N1 Seasonal and Influenza A H3N2 Seasonal were analyzed together. Each flu combination was tested near the LOD $\mathrm { ( } { \sim } 5 \mathrm { X ) }$ with the indicated level of test organism from the table below spiked into the sample. Each sample was tested in triplicate along with six replicates that only contained the potentially cross-reacting microorganism. The influenza type or subtypes identified and reported in the positive specimens spiked with potential cross reactants matched the expected results. No influenza types or subtypes were detected or reported in the negative specimens spiked with potential cross-reactants. Table 7 summarizes the evaluation of assay performance in the presence of potentially cross-reacting organisms.

Table 6: PLEX-ID Potentially Cross-reacting Organisms and Test Levels   

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Spike Concentration</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella cartarrhalis</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">M. tuberculosis avirulent</td><td colspan="1" rowspan="1">106 genomes/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplamsa pneumoniae</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">105 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">105 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 1</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 7</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC 43</td><td colspan="1" rowspan="1">104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC229E</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus type B</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus type 1A</td><td colspan="1" rowspan="1">104 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 1</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 2</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 3</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus type 1</td><td colspan="1" rowspan="1">105 TCID50/mL</td></tr></table>

Table 7: PLEX-ID Flu Cross-Reactivity   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="5" rowspan="1">Correct Results/Number Tested</td></tr><tr><td colspan="1" rowspan="1">2009(Pandemic)Influenza AH1N1</td><td colspan="1" rowspan="1">SeasonalInfluenzaA H1N1</td><td colspan="1" rowspan="1">SeasonalInfluenzaA H3N2</td><td colspan="1" rowspan="1">InfluenzaB</td><td colspan="1" rowspan="1">Negative</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Chlamydia pneumoniae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium sp.</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus sp.</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">M. tuberculosis avirulent</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitides</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Neisseria sp.</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Adenovirus type 7</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC 43</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human coronavirus OC229E</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Enterovirus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr virus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Measles</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human metapneumovirus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Respiratory syncytial virus type B</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus type 1A</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Human parainfluenza type 3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">Herpes simplex virus type 1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr><tr><td colspan="1" rowspan="1">None</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">6/6</td></tr></table>

# Inclusivity:

Multiple viral strains representing temporal and geographical diversity for each influenza type and subtype were tested with the PLEX-ID Flu assay. The results reported for each strain tested matched the expected result in 3 out of 3 replicates. Table 8 includes the results and concentration used for the analysis of inclusivity.

Table 8: PLEX-ID Flu Inclusivity   

<table><tr><td colspan="1" rowspan="1">Viral Strain</td><td colspan="1" rowspan="1">Concentration</td><td colspan="1" rowspan="1">A(6007) INIHuy</td><td colspan="1" rowspan="1">V zunnaa   (sos)INIH</td><td colspan="1" rowspan="1">V zunaa   (Seoss)HNE2</td><td colspan="1" rowspan="1">V zunaahr</td><td colspan="1" rowspan="1">I zn</td></tr><tr><td colspan="1" rowspan="1">Influenza A Swine H1N1 NY01 (2009 H1N1)</td><td colspan="1" rowspan="1">6.57 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A Swine H1N1 NY02 (2009 H1N1)</td><td colspan="1" rowspan="1">1.06×101 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A Swine H1N1 NY03 (2009 H1N1)</td><td colspan="1" rowspan="1">9.37 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Brisbane/59/07 (Seasonal H1N1)</td><td colspan="1" rowspan="1">1.37×101 TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">'</td></tr><tr><td colspan="1" rowspan="1">Influenza A/NewCal/20/1999 (Seasonal H1N1)</td><td colspan="1" rowspan="1">1.29×103 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Taiwan/42/06 (Seasonal H1N1)</td><td colspan="1" rowspan="1">4.45×10³ TCID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Solomon Islands/03/06 (Seasonal H1N1)</td><td colspan="1" rowspan="1">1.40×101 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/PR/8/34 (Seasonal H1N1)</td><td colspan="1" rowspan="1">4.02x104 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/WS/33 (Seasonal H1N1)</td><td colspan="1" rowspan="1">4.14×102 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/New Jersey/8/76 (Seasonal H1N1)</td><td colspan="1" rowspan="1">1.24×102 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Weiss/43 (Seasonal H1N1)</td><td colspan="1" rowspan="1">3.41×106 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/FM/1/47 (Seasonal H1N1)</td><td colspan="1" rowspan="1">6.52×103 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Mal/302/54 (Seasonal H1N1)</td><td colspan="1" rowspan="1">7.41×102 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Port Chalmers/1/73 (Seasonal H3N2)</td><td colspan="1" rowspan="1">6.15×10³ CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Hong Kong/8/68 (Seasonal H3N2)</td><td colspan="1" rowspan="1">4.12×104CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Influenza A/Victoria/3/75 (Seasonal H3N2)</td><td colspan="1" rowspan="1">2.46×103 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza A/Aichi/2/68 (Seasonal H3N2)</td><td colspan="1" rowspan="1">4.72×10³ CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Lee/40</td><td colspan="1" rowspan="1">3.28×104 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Mass/3/66</td><td colspan="1" rowspan="1">9.41×103 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Taiwan/2/62</td><td colspan="1" rowspan="1">1.65×102 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Hong Kong/5/72</td><td colspan="1" rowspan="1">2.63×104 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/GL/1739/54</td><td colspan="1" rowspan="1">1.56×10³ CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Allen/45</td><td colspan="1" rowspan="1">1.30×102 CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Maryland/1/59</td><td colspan="1" rowspan="1">8.50 CEID50/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Russia/69</td><td colspan="1" rowspan="1">1.24×10³ CEID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Florida/02/06</td><td colspan="1" rowspan="1">8.11×101 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Influenza B/Malaysia/2506/04</td><td colspan="1" rowspan="1">2.07×101 TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr></table>

# f. Assay cut-off:

The PLEX-ID Flu Assay is a qualitative assay. The PLEX-ID analysis software uses fixed values for two parameters, Level and Q-score, for defining the reporting level of positive detections. These parameters are described briefly below. Cutoff values (reporting thresholds) for these two parameters are set for each primer group within a PLEX-ID assay, and detections must meet or exceed these thresholds in order to be reported as positive. The assay protocol file contains the cutoff values for the two parameters for each primer group in the assay.

# Level

PLEX-ID Flu Assay plate reaction wells each contain an Amplification Control, a competitive PCR control whose product is distinguishable from product generated from influenza RNA in the specimen. Since the PLEXID PCR reactions run to saturation, the final levels of influenza and control products are proportional to their initial levels. The level threshold is derived based on the assay specific amplification controls in each well and provides a relative amplification estimate compared to Amplification Control.

# Q-Score

The Q-score is an empirically derived metric calculated by the system software to denote the goodness of fit of the detected signals (base

compositions of PCR-amplified fragments detected by mass spectrometry) to the base count compositions associated with specific analytes in the database. The Q-score integrates primer-dependent parameters such as number of primer pairs producing signature-matched amplification products compared to the number expected for the detected analyte and consistency of amplitudes across multiple primer pairs. The Q-score has a value from 0 to 1.

The cut off values for the PLEX-ID Flu assay on the PLEX-ID System were determined empirically using the data from a set of analytical experiments. Receiver Operating Characteristic (ROC) plots were produced to visualize the sensitivity and specificity for each of the target analytes over a range of possible Q score and levels. From these plots, cutoff values of $> 0 . 9 0$ for the Q-score and $> 2 0$ for the level were determined as the most appropriate for all target analytes. These cutoff values were validated using the positive banked clinical specimens with comparison to an FDA cleared molecular device. Furthermore, the results from the multisite clinical study provided further support for these cutoff values.

# g. Carryover

Carryover between samples was evaluated using high target concentrations $( 1 \times$ $1 0 ^ { 5 }$ viral genomes/mL) of influenza A (A/New Claedonia/20/99) and influenza B (B/Florida/04/2006) alternately loaded with negative samples in the sample rack, representing a worst case checkerboard configuration. This configuration was utilized in the sample prep stage, thus offering a combined measure of potential carryover between adjacent samples at all stages, including sample prep, PCR, and PLEX-ID Analyzer desalting and ESI-TOF MS. Following five independent isolations, no positive results were reported among the 120 negative samples tested, resulting in an observed carryover rate of $0 \%$ (0/120).

2. Comparison studies:

a. Method comparison with predicate device:

Not Applicable $b$ . Matrix comparison:

An evaluation of various viral transport medium tubes for use with the PLEX-ID Flu Assay was conducted. For a viral transport medium to be considered acceptable the PLEX-ID Flu assay specimen storage requirement at 2 to $8 \mathrm { { } ^ { \circ } C }$ and - $7 0 \mathrm { { } ^ { \circ } C }$ had to be met. Copan UTM was used throughout the design verification studies including the specimen stability study so additional testing of this medium was not conducted. Five additional types of transport media were tested fresh, following 4 days storage at $2 { - } 8 \mathrm { { ^ \circ C } }$ , following 1 month storage at $- 7 0 \mathrm { { \circ C } }$ or colder and following 4 months storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ . Vials of each type were pooled, spiked with low concentrations of both influenza A and influenza B viral cultures and

aliquoted for the test conditions. Sets of 20 replicates were tested at each condition. The results, as summarized below for baseline, short term and long term storage conditions in the various transport media types, demonstrated that these additional types of viral transport media were acceptable for use with the PLEX-ID Flu assay.

Table 9: Evaluation of Transport Media   

<table><tr><td rowspan=1 colspan=1>Transport MediaType</td><td rowspan=1 colspan=1>Baseline</td><td rowspan=1 colspan=1>Short-Term (4days at 2-8 °C)</td><td rowspan=1 colspan=1>Long-Term (~4months-70 C</td></tr><tr><td rowspan=1 colspan=1>BD Universal ViralTransport Medium</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MicroTest M4 ViralTransport Medium</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MicroTest M4RTViral TransportMedium</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MicroTest M5 ViralTransport Medium</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr><tr><td rowspan=1 colspan=1>MicroTest M6 ViralTransport Medium</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td><td rowspan=1 colspan=1>20/20 (100%)</td></tr></table>

# 3. Clinical studies:

Clinical performance of the PLEX-ID Flu assay was assessed by testing nasopharyngeal (NP) swab specimens from subjects presenting with symptoms of influenza-like illness.

Prospectively collected clinical specimens were tested at three sites. All specimens were tested with an assay FDA cleared for the detection and discrimination of influenza A virus, influenza B virus, and Respiratory Syncytial virus nucleic acids isolated and purified from NP swab specimens obtained from symptomatic patients (Comparator 1). Specimens with a positive influenza A test result went on for further testing with Comparator 2, an assay FDA cleared for the detection and discrimination of influenza A virus subtypes [influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal)] from nucleic acids isolated and purified from NP swab specimens. Aliquots from the same specimens were tested with the PLEX-ID Flu assay. For each subtype of influenza A [H1N1 (2009), H1N1 (seasonal), H3N2 (seasonal)], the point estimate for positive percent agreement (PPA) and negative percent agreement (NPA) was calculated. PCR followed by bidirectional sequencing was used to test discrepant samples between Comparator 1 and Comparator 2, Comparator 1 vs. PLEX-ID Flu, and Comparator 2 vs. PLEX-ID Flu. The results from bi-directional sequence testing were not used to calculate the positive percent agreement and negative percent agreement of the PLEX-ID Flu assay.

Of the 1282 specimens included in the study, 6 were unresolved by Comparator 1 and were excluded, resulting in 1276 specimens included in the final analysis.

A total of 90 unique specimens were influenza A positive by Comparator 1 and 1186 were negative. All 90 specimens were tested by Comparator 2 to determine influenza A subtypes [influenza A H1N1 (2009), influenza A H1N1 (seasonal), influenza A H3N2 (seasonal)]. From these 90 unique specimens, 6 were negative for all three subtypes by Comparator 2. The remaining 84 specimens generated a total of 88 positive results. Four specimens generated two different subtype results each by Comparator 2. For one of these four specimens, Comparator 2 assay reported both H1N1 and H3N2 while the PLEX-ID Flu assay reported an H1N1 (seasonal) result. For the remaining 3 specimens, Comparator 2 reported both H3N2 and H1N1, whereas the PLEX-ID Flu assay reported H3N2 (seasonal). These 88 positive results comprised 32 influenza A H1N1 (2009), 7 influenza A H1N1 (seasonal) positive, and 49 influenza A H3N2 (seasonal) positive results.

From the 1287 prospective NP swab specimens collected for this study and tested, $9 8 . 7 \%$ (1285/1302) of PLEX-ID assays results generated were valid.

The comparison of the PLEX-ID Flu assay and Comparator 2 results for influenza A H1N1 (2009) are shown in Table 10, influenza A H1N1 (seasonal) in Table 11, and influenza A H3N2 (seasonal) in Table 12.

Table 10 PLEX-ID Flu Influenza A H1N1 (2009) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>5br$</td><td rowspan=1 colspan=1>37</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1239</td><td rowspan=1 colspan=1>1239</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>1244</td><td rowspan=1 colspan=1>1276</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement100.0% (32/32)(89.3%, 100.0%)</td><td rowspan=1 colspan=1>Negative PercentAgreement99.6% (1239/1244)(99.1%, 99.8%)</td><td rowspan=1 colspan=1></td></tr></table>

a Negative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H1N1 (2009). bFive samples were negative for influenza A 2009 H1N1 by Comparator 2; three samples were positive by bi-directional sequence analysis for 2009 H1N1, one had insufficient sample volume for bi-directional sequence analysis, and $2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ nucleic acid was not detected by bi-directional sequence analysis in one sample.

Table 11 PLEX-ID Flu Influenza A H1N1 (Seasonal) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>3b</td><td rowspan=1 colspan=1>1269</td><td rowspan=1 colspan=1>1272</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1269</td><td rowspan=1 colspan=1>1276</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement57.1% (4/7)(25.0%, 84.2%)</td><td rowspan=1 colspan=1>Negative PercentAgreement100.0% (1269/1269)(99.7%, 100.0%)</td><td rowspan=1 colspan=1></td></tr></table>

aNegative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H1N1 (Seasonal). $^ { \mathsf { b } } \mathrm { A } / \mathrm { H } 1$ nucleic acids were not detected by bi-directional sequence analysis in these three samples.

Table 12 PLEX-ID Flu Influenza A H3N2 (Seasonal) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>2b</td><td rowspan=1 colspan=1>50</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>1c</td><td rowspan=1 colspan=1>1225</td><td rowspan=1 colspan=1>1226</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1227</td><td rowspan=1 colspan=1>1276</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement98.0% (48/49)(89.3%, 99.6%)</td><td rowspan=1 colspan=1>Negative PercentAgreement99.8% (1225/1227)(99.4%, 100.0%)</td><td rowspan=1 colspan=1></td></tr></table>

aNegative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H3N2 (Seasonal). bTwo samples were negative for influenza A by Comparator 1 and were not reflexed to Comparator 2 for subtype determination, but the samples were positive by bi-directional sequence analysis for $\mathrm { A } / \mathrm { H } 3$ . $^ \mathrm { c } _ { \mathrm { \ A / H 3 } }$ nucleic acid was not detected by bi-directional sequence analysis in this sample.

A total of 22 positive specimens were obtained for influenza B. The positive percent agreement and negative percent agreement were calculated. The results of the PLEX-ID Flu compared to Comparator 1 for the detection of influenza B are shown in Table 13.

Bi-directional sequence analysis was not performed for discordant influenza B specimens.

Table 13 PLEX-ID Flu Influenza B Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 1Positive</td><td rowspan=1 colspan=1>Comparator 1Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>24</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1252</td><td rowspan=1 colspan=1>1252</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>22</td><td rowspan=1 colspan=1>1254</td><td rowspan=1 colspan=1>1276</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement100.0% (22/22)(85.1%, 100.0%)</td><td rowspan=1 colspan=1>Negative PercentAgreement99.8% (1252/1254)(99.4%, 100.0%)</td><td rowspan=1 colspan=1></td></tr></table>

# Retrospective Specimen Testing

Due to the decreased prevalence of influenza $\mathrm { \bf A } 2 0 0 9 \mathrm { H } 1 \mathrm { N } 1$ and the absence of influenza A seasonal H1N1 and H3N2, the prospectively collected specimen population was supplemented with pre-selected banked specimens that were collected in February 2008, and from January 2009 through March 2009, and left over archived specimens collected from April 2009 through December 2009.

A total of 1378 retrospective specimens were tested. Of these specimens, 1341 were included in the analysis (the remaining 37 were excluded either due to unavailable results for the PLEX-ID Flu assay or Comparator 1 assay, duplicate subjects, or failure to meet subject inclusion criteria).

Of the 1341 retrospective specimens, 634 were influenza A positive and 707 were negative by Comparator 1. All 634 positive specimens were tested to determine influenza A subtype (influenza A 2009 H1N1, influenza A (seasonal) H1N1, influenza A (seasonal) H3N2). Two specimens were unresolved by Comparator 2 and, therefore, excluded, resulting in 1339 total specimens in the analysis. Fiftyfour of the 634 Comparator 1 influenza A positive specimens were negative for all three subtypes by Comparator 2 and were included as Comparator negative specimens. Consequently, 578 positive results were included in the analysis: 548 influenza A H1N1 (2009), 28 influenza A H1N1 (seasonal), and 2 influenza A H3N2 (seasonal).

From the 1378 retrospective NP swab specimens tested in this study, $9 3 . 4 \%$ (1368/1464) of PLEX-ID assays results generated were valid.

The comparison of the PLEX-ID Flu and Comparator 2 results for influenza A H1N1 (2009) are shown in Table 14, influenza A H1N1 (seasonal) in Table 15, and influenza A H3N2 (seasonal) in Table 16.

Table 14 PLEX-ID Flu Influenza A H1N1 (2009) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>538</td><td rowspan=1 colspan=1>45b</td><td rowspan=1 colspan=1>583</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>10c</td><td rowspan=1 colspan=1>746</td><td rowspan=1 colspan=1>756</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>548</td><td rowspan=1 colspan=1>791</td><td rowspan=1 colspan=1>1339</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement98.2% (538/548)(96.7%, 99.0%)</td><td rowspan=1 colspan=1>Negative PercentAgreement94.3% (746/791)(92.5%, 95.7%)</td><td rowspan=1 colspan=1></td></tr></table>

aNegative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H1N1 (2009). $^ { \mathsf { b } } 4 1$ samples were positive by bi-directional sequence analysis for 2009 H1N1, 1 was unresolved, 2 had insufficient sample volume for bi-directional sequence analysis, and H1N1(2009) nucleic acid was not detected in one sample. cA/H1N1 (2009) was detected in 8 samples by bi-directional sequence analysis.

Table 15 PLEX-ID Flu Influenza A H1N1 (Seasonal) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1311</td><td rowspan=1 colspan=1>1312</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1311</td><td rowspan=1 colspan=1>1339</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement96.4% (27/28)(82.3%, 99.4%)</td><td rowspan=1 colspan=1>Negative PercentAgreement100.0% (1311/1311)(99.7%, 100.0%)</td><td rowspan=1 colspan=1>→</td></tr></table>

aNegative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H1N1 (Seasonal). $^ { \mathsf { b } } \mathrm { A } / \mathrm { H } 1$ nucleic acid was not detected by bi-directional sequence analysis in this sample.

Table 16 PLEX-ID Flu Influenza A H3N2 (Seasonal) Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 2Positive</td><td rowspan=1 colspan=1>ComparatoraNegative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1337</td><td rowspan=1 colspan=1>1337</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1337</td><td rowspan=1 colspan=1>1339</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement100.0% (2/2)(34.2%, 100.0%)</td><td rowspan=1 colspan=1>Negative PercentAgreement100.0% (1337/1337)(99.7%, 100.0%)</td><td rowspan=1 colspan=1></td></tr></table>

aNegative for influenza A by Comparator 1 (and therefore untested by Comparator 2) or positive for influenza A by Comparator 1 and negative by Comparator 2 reflex testing for influenza A H3N2 (Seasonal).

A total of 28 positive specimens were obtained for influenza B. The positive percent agreement and negative percent agreement and were calculated. The comparison results of the PLEX-ID Flu and Comparator 1 to detect influenza B are shown in Table 17. Bi-directional sequence analysis was not performed for discordant influenza B specimens.

Table 17 PLEX-ID Flu Influenza B Specimen Testing   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Comparator 1Positive</td><td rowspan=1 colspan=1>Comparator 1Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluPositive</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>27</td></tr><tr><td rowspan=1 colspan=1>PLEX-ID FluNegative</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1312</td><td rowspan=1 colspan=1>1314</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>1313</td><td rowspan=1 colspan=1>1341</td></tr><tr><td rowspan=1 colspan=1>PercentAgreement(95% CI)</td><td rowspan=1 colspan=1>Positive PercentAgreement92.9% (26/28)(77.4%, 98.0%)</td><td rowspan=1 colspan=1>Negative PercentAgreement99.9% (1312/1313)(99.6%, 100.0%)</td><td rowspan=1 colspan=1></td></tr></table>

4. Clinical cut-off:

Not applicable

# 5. Expected values/Reference range:

The clinical performance of the PLEX-ID Flu assay on the PLEX-ID System was established using nasopharyngeal (NP) swab specimens prospectively collected from two U.S. clinical sites and three specimen suppliers from September 2009 through June 2010. A total of 1287 prospective NP swab specimens were collected for the PLEX-ID Flu clinical study. Of the 1287, five specimens were excluded: duplicate specimens were collected from two subjects (the second specimen from each subject was excluded from the analysis), one specimen from a subject who did not meet the inclusion criteria, and two specimens had an invalid PLEX-ID Flu result, for a total of 1282 included specimens. The prevalence of influenza A 2009 H1N1 decreased significantly during the collection period, consequently, there were only 37 positive influenza A 2009 H1N1, 4 positive influenza A H1N1 (seasonal), 50 positive influenza A H3N2 (seasonal), and 24 positive influenza B specimens among the 1282 specimens. Prevalence rates and demographics are presented in Table 18 and Table 19, respectively, for four of the five sites that provided prospectively collected specimens. No demographic information was available from supplier Site 3 $\scriptstyle ( \mathrm { N } = 1 9 9$ ) and one subject from clinical Site 1.

Table 18: Influenza Prevalence Rate - Prospectively Collected Specimens   

<table><tr><td rowspan=1 colspan=6>Number of Positives by the PLEX-ID Flu Assay(Observed Prevalence Rate)</td></tr><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>2009 H1N1</td><td rowspan=1 colspan=1>SeasonalH1N1</td><td rowspan=1 colspan=1>SeasonalH3N2</td><td rowspan=1 colspan=1>Influenza B</td></tr><tr><td rowspan=1 colspan=1>&lt; 2 years</td><td rowspan=1 colspan=1>202</td><td rowspan=1 colspan=1>3 (1.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>2 to 5 years</td><td rowspan=1 colspan=1>46</td><td rowspan=1 colspan=1>1 (2.2%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>6 to 11 years</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>12 to 18 years</td><td rowspan=1 colspan=1>49</td><td rowspan=1 colspan=1>1 (2.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>19 to 64 years</td><td rowspan=1 colspan=1>730</td><td rowspan=1 colspan=1>18 (2.5%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>1 (0.1%)</td></tr><tr><td rowspan=1 colspan=1>&gt; 65 years</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td></tr><tr><td rowspan=1 colspan=1>Unknown*</td><td rowspan=1 colspan=1>200</td><td rowspan=1 colspan=1>14 (7.0%)</td><td rowspan=1 colspan=1>4 (2.0%)</td><td rowspan=1 colspan=1>50 (25.0%)</td><td rowspan=1 colspan=1>23 (11.5%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1282</td><td rowspan=1 colspan=1>37 (2.9%)</td><td rowspan=1 colspan=1>4 (0.3%)</td><td rowspan=1 colspan=1>50 (3.9%)</td><td rowspan=1 colspan=1>24 (1.9%)</td></tr></table>

\* Demographic information was unavailable.

Table 19: Subject Demographics - Prospectively Collected Specimens   

<table><tr><td rowspan=1 colspan=1>Gender/Age</td><td rowspan=1 colspan=1>Number of Subjects (%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>603 (47.0%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>479 (37.4%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>200 (15.6%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1282 (100.0%)</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.