# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k121942 B. Purpose for Submission:

This is a $5 1 0 ( \mathrm { k } )$ application for a new qualitative Real-Time Polymerase Chain Reaction (PCR) assay used with the Cepheid SmartCycler II instrument for the in vitro qualitative detection of adenovirus DNA in nasopharyngeal swabs (NPS) and nasopharyngeal wash/aspirates (NPA) from symptomatic human patients.

# C. Measurand:

Target DNA sequences for a region of the hexon gene of Adenovirus species

# D. Type of Test:

Real-Time Polymerase Chain Reaction assay for the qualitative detection of adenovirus DNA in NPS and NPA samples. A bi-functional fluorescent probe-primer is used together with a reverse primer to amplify a specific target for the analyte and the RNA internal control. Amplification and detection is performed on the Cepheid SmartCycler II with Dx software version 1.7b or 3.0.

# E. Applicant:

Argene SA

F. Proprietary and Established Names: Adenovirus R-gene $^ \mathrm { \textregistered }$ US

# G. Regulatory Information:

1. Regulation section: 21 CFR 866.3980 Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code:

OCC

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

Adenovirus R-gene® US assay is a Real Time PCR in vitro diagnostic test for the rapid and qualitative detection of Adenovirus viral DNA isolated and purified from nasopharyngeal swab or nasopharyngeal wash/aspirate specimens obtained from individuals exhibiting signs and symptoms of acute respiratory infection. The intended use for this test is to aid in the diagnosis of respiratory Adenovirus infections in humans in conjunction with other clinical and laboratory findings. The test detects, but does not differentiate, Adenovirus species (A, B, C, D, E, F and G). Negative results do not preclude Adenovirus infection and should not be used as the sole basis for treatment or other patient management decisions.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements:

To be used with the Cepheid SmartCycler II utilizing Dx software version 1.7b or 3.0

# I. Device Description:

The test is a Taqman-based real-time PCR amplification and detection assay for the detection of human adenovirus DNA in nasopharyngeal swab and nasopharyngeal aspirate/wash specimens. The assay uses the bioMérieux NucliSENS easyMAG nucleic acid extraction system to extract adenovirus DNA from swab or wash specimens obtained from patients with signs and symptoms of acute respiratory infection. Extracted adenovirus DNA is then amplified using the Adenovirus R-gene US assay components. Amplification is performed on the Cepheid SmartCycler II using Dx software versions 1.7b or 3.0.

The Adenovirus R-gene US assay contains the reaction mix $( \mathsf { R } ^ { \mathrm { x } } \mathrm { 1 0 } )$ , internal control (IC2), molecular-grade water (W0), positive control (PC10), and Package Insert.

Adenovirus R-gene US reaction mix contents and probe info:   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Composition</td><td rowspan=1 colspan=1>Quantity/tube</td><td rowspan=1 colspan=1># ofReactions</td><td rowspan=1 colspan=1>Form</td></tr><tr><td rowspan=1 colspan=1>R*10</td><td rowspan=1 colspan=1>Adenovirus andIC2 amplificationpre-mix</td><td rowspan=1 colspan=1>Adenovirus and IC2 primers andprobes, Tris, MgCl2, dNTP&#x27;s, TaqPolymerase, Rox Carboxy xRhodamine, Ammonium Sulfide,</td><td rowspan=1 colspan=1>450 μL</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>Liquid</td></tr><tr><td rowspan=1 colspan=1>IC2</td><td rowspan=1 colspan=1>Internal Control 2</td><td rowspan=1 colspan=1>Native Phix bacteriophage, Phastmedia, Glycerol</td><td rowspan=1 colspan=1>1 mL</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>Liquid</td></tr><tr><td rowspan=1 colspan=1>WO</td><td rowspan=1 colspan=1>Water forextraction(molecular grade)</td><td rowspan=1 colspan=1>Water</td><td rowspan=1 colspan=1>1.8 mL</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Liquid</td></tr><tr><td rowspan=1 colspan=1>PC10</td><td rowspan=1 colspan=1>AdenovirusPositive Control</td><td rowspan=1 colspan=1>Composite plasmid, Tris buffer, yeasttRNA</td><td rowspan=1 colspan=1>300 μL</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>Liquid</td></tr></table>

Materials required but not supplied: Polyester, rayon or nylon tipped nasopharyngeal swabs, Sterile suction catheter (#8) for washes/aspirates specimen, $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes, Sterile filter or positive displacement micropipettor tips, EasyMAG™ System Disposables (Sample Vessels and Tips), Biohit Pipette Tips for use with easyMAG™ System, Greiner Break Four uncoated plates for use with easyMAG™ System, Cepheid $2 5 \mu \mathrm { L }$ PCR reaction tubes, single use latex or similar gloves, bioMérieux NucliSENS $^ \mathrm { \textregistered }$ easyMAG™ reagents (Buffer 1 Cat.#280130, Buffer 2 Cat.#280131, Buffer 3 Cat.#280132, Magnetic Silica Cat. #280133, Lysis Buffer (bottles) Cat.# 280134, Universal Transport Medium from DHI or MicroTest™ M4RT Transport from Remel, Proteinase K Solution, $6 0 0 \mathrm { m A U / m L }$ (Novagen – Merck – 71049-3/71049-4), Molecular grade water, Sterile physiologic buffer, $- 1 8 ^ { \circ } \mathrm { C } / - 2 2 ^ { \circ } \mathrm { C }$ Freezer, bioMérieux NucliSENS $^ \mathrm { \textregistered }$ easyMAG™ System with Software version 2.0, Cepheid SmartCycler II instrument with Dx Software version 1.7b or 3.0, Micropipettors (range between $1 { - } 1 0 \mu \mathrm { L }$ , $1 0 { - } 2 0 0 \mu \mathrm { L }$ and $1 0 0 { - } 1 0 0 0 \mu \mathrm { L }$ ), Mini-centrifuge with adapter for Cepheid Reaction Tubes, Cepheid cooling block, U.V. light, Workstation or Plexiglas screen for samples and premix distribution.

# Assay Procedure:

1- Collect swab specimens from symptomatic patients using a polyester, rayon, nylon tipped swab or wash/aspirate specimens using a sterile suction catheter.

2- Add Internal Control (IC2) to every sample to monitor for the presence of inhibitors or lysi failure.

3- Perform isolation and purification of nucleic acids using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ System (bioMérieux) and the automated Magnetic Extraction Reagents (bioMérieux).

4- Perform real time amplification of extracted samples using the Adenovirus R-gene US kit on the Cepheid SmartCycler II instrument.

5- Interpretation.

# Interpretation of Results:

There are five possible results reported by the SmartCycler II summarized in the table and below:

<table><tr><td rowspan=1 colspan=1>Assay Result</td><td rowspan=1 colspan=1>IC Result</td><td rowspan=1 colspan=1>Warning/ErrorCode</td><td rowspan=1 colspan=1>Adenovirus Result</td><td rowspan=1 colspan=1>Interpretation</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Adenovirus nucleicacid not detected</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Adenovirus nucleicacid detected</td></tr><tr><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Unresolved; PCRinhibition or reagentfailure. Repeattesting with purifiednucleic acid or obtainnew sample.</td></tr><tr><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>3079</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined;error code 3079</td></tr><tr><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>4098</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined;error code 4098</td></tr></table>

Positive: Human adenovirus DNA was detected in the sample   
Negative: Human adenovirus DNA was not detected in the sample   
Unresolved: PCR inhibition or reagent failure. Repeat testing from the purified nucleic acid or collect and test a new sample.   
Invalid: Not determined; error code 4098   
ND: (Not Determined) if the specimen result is unavailable because the run is in progress or an error or warning occurred during the run.

\* Detection of Internal Control in the Cy3 channel is not required for a positive result. High viral load can lead to reduced or absent Internal Control signal.

Error Code 3079 can be observed with positives (Adenovirus Positive Control, Adenovirus positive swab/wash samples) when the fluorescence signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a Ct value $\geq 5$ is reported in the Adenovirus Ct columns, the sample results can be recorded as positive for the specific analyte. There were no 3079 type errors reported among the 1576 prospective clinical study specimens analyzed.

Error Code 4098: An invalid assay run will display Error Code 4098. One of the controls has failed and repeat testing should be done starting from the purified nucleic acid and using a new aliquot of the Positive Control. If repeated results are still invalid, results should not be reported and testing should be repeated from the original sample or a new sample should be collected and tested. There were no 4098 type errors reported among the 1576 prospective clinical study specimens analyzed.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Prodesse $^ \mathrm { \textregistered }$ ProAdeno $\mathbf { T M } +$

2. Predicate 510(k) number(s): k102952

3. Comparison with predicate:

<table><tr><td colspan="1" rowspan="1">Features</td><td colspan="5" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="5" rowspan="1">Adenovirus R-gene US</td><td colspan="1" rowspan="1">Prodesse ProAdenoTM+Assay</td></tr><tr><td colspan="1" rowspan="3">Intended Use</td><td colspan="5" rowspan="3">Adenovirus R-gene US is aReal Time PCR in vitrodiagnostic test for the rapidand qualitative detection ofAdenovirus viral DNAisolated and purified fromnasopharyngeal specimens(swab or wash/aspirate)obtained from individualsexhibiting signs andsymptoms of acuterespiratory infection. Theintended use for this test isto aid in the diagnosis ofAdenovirus infections inhumans.Negative results do notpreclude Adenovirusinfection and should not beused as the sole basis fortreatment or othermanagement decisions.</td><td colspan="1" rowspan="3">The ProAdenoTM+ Assayis a multiplex Real TimePCR in vitro diagnostictest for the qualitativedetection of humanAdenovirus (HAdV) DNAisolated and purified fromnasopharyngeal swabspecimens obtained fromindividuals exhibitingsigns and symptoms ofacute respiratoryinfection. This test isintended for use to aid inthe diagnosis of HAdVinfections in humans inconjunction with otherclinical and laboratoryfindings. The test detects,but does not differentiate,serotypes 1-51.Negative results do notpreclude HAdV infectionand should not be used asthe sole basis fortreatment or other patientmanagement decisions.</td></tr><tr></tr><tr><td colspan="2" rowspan="1">and sl</td><td colspan="2" rowspan="1">shoul</td></tr><tr><td colspan="1" rowspan="1">510(k)</td><td colspan="5" rowspan="1">k121942</td><td colspan="1" rowspan="1">k102952</td></tr><tr><td colspan="1" rowspan="1">Regulation</td><td colspan="5" rowspan="1">21 CFR 866.3980</td><td colspan="1" rowspan="1">21 CFR 866.3980</td></tr><tr><td colspan="1" rowspan="1">Product Code</td><td colspan="5" rowspan="1">OCC</td><td colspan="1" rowspan="1">OCC</td></tr><tr><td colspan="1" rowspan="1">Features</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Adenovirus R-gene US</td><td colspan="1" rowspan="1">Prodesse ProAdenoTM+Assay</td></tr><tr><td colspan="1" rowspan="1">Assay Targets</td><td colspan="1" rowspan="1">DNA from HumanAdenovirus</td><td colspan="1" rowspan="1">DNA from HumanAdenovirus</td></tr><tr><td colspan="1" rowspan="1">Sample Types</td><td colspan="1" rowspan="1">Nasopharyngeal swabs andNasal aspirates/washes</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td></tr><tr><td colspan="1" rowspan="1">Assay Type</td><td colspan="1" rowspan="1">Real-Time PCR</td><td colspan="1" rowspan="1">Real-Time PCR</td></tr><tr><td colspan="1" rowspan="1">Assay Results</td><td colspan="1" rowspan="1">Qualitative</td><td colspan="1" rowspan="1">Qualitative</td></tr><tr><td colspan="1" rowspan="1">Detection</td><td colspan="1" rowspan="1">Instrument-basedfluorogenic target-specifichybridization</td><td colspan="1" rowspan="1">Instrument-basedfluorogenic target-specifichybridization</td></tr><tr><td colspan="1" rowspan="1">Gene Target</td><td colspan="1" rowspan="1">Hexon gene</td><td colspan="1" rowspan="1">Hexon gene</td></tr><tr><td colspan="1" rowspan="1">Collection andTransport Media</td><td colspan="1" rowspan="1">Universal TransportMedium (DHI),MicroTestTM M4RTTransport (Remel)</td><td colspan="1" rowspan="1">M4 and M5 ViralTransport Medium(Remel), UVT (BectonDickinson), UTM (Copan)</td></tr><tr><td colspan="1" rowspan="1">AssayInstrumentation</td><td colspan="1" rowspan="1">Cepheid SmartCycler® IIwith Dx Software</td><td colspan="1" rowspan="1">Cepheid SmartCycler® IISystem</td></tr><tr><td colspan="1" rowspan="1">Nucleic AcidIsolation</td><td colspan="1" rowspan="1">NuclisENSeasyMAGTM(bioMérieux)</td><td colspan="1" rowspan="1">MagnaPure LC System(Roche)NuclisENSeasyMAGTM(bioMérieux)</td></tr><tr><td colspan="1" rowspan="1">Controls Included</td><td colspan="1" rowspan="1">Positive Control plasmidDNANeg control (mol. gradewater)Internal control (IC2) phageparticle</td><td colspan="1" rowspan="1">Adenovirus positive DNAtranscript control andInternal DNA/RNAcontrol</td></tr><tr><td colspan="1" rowspan="1">Results</td><td colspan="1" rowspan="1">PositiveNegativeUnresolved</td><td colspan="1" rowspan="1">PositiveNegativeUnresolved</td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable):

None

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in a single closed-tube reaction. The Adenovirus R-gene US assay enables simultaneous detection of AdV and Internal Control DNA. The whole process is based on two steps: nucleic acid isolation and Real Time PCR amplification/detection. Human respiratory specimens (nasopharyngeal swabs, washes/aspirates) from symptomatic patients are processed initially to isolate and purify viral nucleic acid from the cellular specimen matrix. Each purified nucleic acid sample is added to the $\mathtt { R x l 0 }$ amplification premix (Ready to use, Taq polymerase included). The $\mathrm { R x l 0 }$ amplification premix contains oligonucleotide primers complementary to a fragment of the Hexon gene region coding for the hexagonal capsomers for AdV and targetspecific oligonucleotide probes dual-labeled with a reporter dye and a quencher dye. As the amplification proceeds, the probe anneals specifically to a region of the template between the forward and reverse primers. As primer extension and amplification occurs, the exonuclease activity of the Taq polymerase cleaves the probe separating the reporter dye away from the quencher. This generates an increase in fluorescent signal upon excitation from a LED light source of appropriate wavelength. With each cycle, additional reporter dye molecules are cleaved from their respective probes, yielding increased fluorescence signal. The amount of fluorescence at any given cycle is dependent on the amount of PCR product (amplicons) present at that time. Fluorescent intensity is monitored at each PCR cycle by fluorescent detection modules within the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

The study evaluated the device’s inter-laboratory, inter-assay, and intra-assay reproducibility for high negative (0.01x LoD), low positive (2x LoD), and moderately positive (10x LoD) specimens. The precision (repeatability and reproducibility) of the Adenovirus R-gene US assay was evaluated using human nasal matrix spiked with 3 viral inputs (moderate positive sample, low positive sample, high negative sample). A within-laboratory (repeatability) precision study was performed at an internal site (5 days, 2 operators, 1 instrument, 1 lot of reagent) and a between laboratory (reproducibility) precision study was performed at 3 external sites in the U.S. (5 days, 2 operators, 1 instrument at each site, 1 lot of reagent. Test panels consisted of 14 members (12 Adenovirus positive specimens plus two negative controls). Samples were extracted each day by both operators (5 days $_ { \textrm { X 2 } }$ operators $= 1 0$ tests per sample) and tested in triplicate. For the moderate positive samples, 5 samples were present in the panel for a total of 150 tests (5 samples $\textbf { \scriptsize x } 1 0$ tests per sample x 3 replicates $= 1 5 0$ ) For low positive samples, 4 samples were present in the panel for a total of 120 tests ( 4 samples $\textbf { \scriptsize x } 1 0$ tests x 3 replicates). For high negative samples, there were 3 samples per panel for a total of 90 tests (3 samples $\textbf { \scriptsize x } 1 0$ tests $_ { \textrm { X 3 } }$ replicates). Combined results for all sites and results stratified by site are presented in the tables below.

Reproducibility – Within Laboratory   

<table><tr><td rowspan=1 colspan=1>Sample</td><td rowspan=1 colspan=1>% Agreement</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100% (150/150)</td><td rowspan=1 colspan=1>97.6-100%</td><td rowspan=1 colspan=1>28.73</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>Low Positive</td><td rowspan=1 colspan=1>95% (114/120)</td><td rowspan=1 colspan=1>89.4-98.1%</td><td rowspan=1 colspan=1>32.90</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>9.1</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>100% (90/90)</td><td rowspan=1 colspan=1>96-100%</td><td rowspan=1 colspan=1>24.19*</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.70</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>98.3% (354/360)</td><td rowspan=1 colspan=1>96.4-99.4%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* Mean Ct calculated from Internal Control

Reproducibility – Between Laboratories   

<table><tr><td rowspan=2 colspan=1>Sample</td><td rowspan=1 colspan=3>% Agreement</td><td rowspan=2 colspan=1>Total %Agreementand 95% CI</td><td rowspan=2 colspan=1>Mean Ct</td><td rowspan=2 colspan=1>SD</td><td rowspan=2 colspan=1>%CV</td></tr><tr><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 3</td></tr><tr><td rowspan=1 colspan=1>ModeratePositive</td><td rowspan=1 colspan=1>100%(150/150)</td><td rowspan=1 colspan=1>100%(150/150)</td><td rowspan=1 colspan=1>100%(150/150)</td><td rowspan=1 colspan=1>100%(450/450)99.2-100%</td><td rowspan=1 colspan=1>28.78</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>4.7</td></tr><tr><td rowspan=1 colspan=1>LowPositive</td><td rowspan=1 colspan=1>99.2%(119/120)</td><td rowspan=1 colspan=1>99.2%(119/120)</td><td rowspan=1 colspan=1>97.5%(117/120)</td><td rowspan=1 colspan=1>98.6%(355/360)96.8-99.6%</td><td rowspan=1 colspan=1>32.0</td><td rowspan=1 colspan=1>2.17</td><td rowspan=1 colspan=1>6.8</td></tr><tr><td rowspan=1 colspan=1>HighNegative</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>95.6%(86/90)</td><td rowspan=1 colspan=1>100%(90/90)</td><td rowspan=1 colspan=1>98.5%(266/270)96.2-99.6%</td><td rowspan=1 colspan=1>24.12*</td><td rowspan=1 colspan=1>0.36</td><td rowspan=1 colspan=1>1.5</td></tr><tr><td rowspan=1 colspan=1>TotalAgreement</td><td rowspan=1 colspan=1>99.7%(359/360)</td><td rowspan=1 colspan=1>98.6%(355/360)</td><td rowspan=1 colspan=1>99.2%(357/360)</td><td rowspan=1 colspan=1>99.2%(1071/1080)98.4-99.6%</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* Mean Ct calculated from Internal Control

b. Linearity/assay reportable range: Not applicable   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):

Adenovirus R-gene US Stability and Positive Control Inter-Lot Reproducibility Argene assessed the reproducibility of three different lots of Positive Control (PC) by testing each lot of controls with one lot of reaction mix. The study yielded a total of seven replicates per lot of PC mix. Reproducibility was determined simultaneously with freeze/thaw stability for the PC mix. The table below shows consistent reproducibility between lots and also stability for up to 29 freeze/thaw cycles.

PC Freeze/Thaw Stability and Reproducibility   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=3>PC10 R3424</td><td rowspan=1 colspan=3>PC10 R3642</td><td rowspan=1 colspan=3>PC10 R4070</td></tr><tr><td rowspan=1 colspan=1>Freeze/ThawCycles</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>Ct</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>28.6</td><td rowspan=6 colspan=1>28.43</td><td rowspan=6 colspan=1>0.24</td><td rowspan=1 colspan=1>28.3</td><td rowspan=6 colspan=1>28.43</td><td rowspan=6 colspan=1>0.14</td><td rowspan=1 colspan=1>28.3</td><td rowspan=7 colspan=1>28.26</td><td rowspan=7 colspan=1>0.22</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>28.5</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>28.9</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>28.3</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>28.4</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>28.4</td></tr><tr><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>28.2</td><td rowspan=1 colspan=1>28.6</td><td rowspan=1 colspan=1>28.2</td></tr><tr><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>28.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>27.8</td></tr><tr><td rowspan=1 colspan=1>PC10</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>28.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>IC2W0</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

The table below shows that reaction mix $( \mathsf { R } ^ { \mathrm { x } } \mathrm { 1 0 } )$ and Internal Control are stable for up to 10 freeze/thaw cycles.

Reaction Mix Freeze/Thaw Stability   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Reference1 Thaw</td><td rowspan=1 colspan=2>11 Freeze/Thawcycles</td><td rowspan=1 colspan=2>Delta CT after 11cycles</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Dilution</td><td rowspan=1 colspan=1>AdV530nm</td><td rowspan=1 colspan=1>IC2560nm</td><td rowspan=1 colspan=1>AdV530nm</td><td rowspan=1 colspan=1>IC2560nm</td><td rowspan=1 colspan=1>AdV530nm</td><td rowspan=1 colspan=1>IC2560nm</td></tr><tr><td rowspan=1 colspan=1>AdV12</td><td rowspan=1 colspan=1>10-4</td><td rowspan=1 colspan=1>26.3</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>26.1</td><td rowspan=1 colspan=1>24.5</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>AdV3</td><td rowspan=1 colspan=1>10-7</td><td rowspan=1 colspan=1>29.2</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>29.1</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>AdV11</td><td rowspan=1 colspan=1>10-4</td><td rowspan=1 colspan=1>26.0</td><td rowspan=1 colspan=1>24.8</td><td rowspan=1 colspan=1>25.5</td><td rowspan=1 colspan=1>24.2</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>-0.6</td></tr><tr><td rowspan=1 colspan=1>AdV5</td><td rowspan=1 colspan=1>10-5</td><td rowspan=1 colspan=1>30.0</td><td rowspan=1 colspan=1>24.4</td><td rowspan=1 colspan=1>30.5</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>-0.1</td></tr><tr><td rowspan=1 colspan=1>AdV8</td><td rowspan=1 colspan=1>10-5</td><td rowspan=1 colspan=1>27.2</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>27.0</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>AdV4</td><td rowspan=1 colspan=1>10-6</td><td rowspan=1 colspan=1>31.7</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>AdV40</td><td rowspan=1 colspan=1>10-5</td><td rowspan=1 colspan=1>29.9</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>29.7</td><td rowspan=1 colspan=1>24.1</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>IC2W0</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>24.6</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>PC10</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1>28.0</td><td rowspan=1 colspan=1>neg</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

# Adenovirus R-gene US Specimen Stability

Samples must be stored refrigerated at $+ 2 ^ { \circ } \mathrm { C } / + 8 ^ { \circ } \mathrm { C }$ for up to 24 hours prior to processing or stored at $- 1 8 ^ { \circ } \mathrm { C } / - 2 2 ^ { \circ } \mathrm { C }$ for 4 days or - $- 7 8 ^ { \circ } \mathrm { C } / - 8 2 ^ { \circ } \mathrm { C }$ for longer period.

# Adenovirus R-gene US Quality Control Ranges

Quality control ranges have been established as indicated in the table below. If the controls are not within these parameters, patient results should be considered invalid and the assay repeated. Each laboratory should establish its own Quality Control ranges and frequency of QC testing based on applicable local laws, regulations, and standard good laboratory practice.

Adenovirus R-gene US Expected Control Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>IC2a + W0</td><td rowspan=1 colspan=2>IC2 ^ + sample</td><td rowspan=1 colspan=2>PC10 b</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Detected(+)</td><td rowspan=1 colspan=1>NotDetected (-)</td><td rowspan=1 colspan=1>Detected(+)</td><td rowspan=1 colspan=1>NotDetected (-)</td><td rowspan=1 colspan=1>Detected(+)</td><td rowspan=1 colspan=1>NotDetected(-)</td></tr><tr><td rowspan=1 colspan=1>Cy3</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1>Pass orerror 3079</td><td rowspan=1 colspan=1>Fail or error4098 or n/a*</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>Invalid;error 4098</td><td rowspan=1 colspan=1>Valid</td><td rowspan=1 colspan=1>Positiveor error3079</td><td rowspan=1 colspan=1>Negative orerror 4098 orUnresolved</td><td rowspan=1 colspan=1>Valid orerror3079</td><td rowspan=1 colspan=1>Invalid;error4098</td></tr></table>

a Typical Ct values for the Internal Control (IC2) range between 13.0 and 45.0 cycles b Typical Ct values for the Positive Control range between 26.0 and 33.0 cycles

Error Code 3079 can be observed with positives (Adenovirus Positive Control, Adenovirus positive NP swab samples) when the fluorescence signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a Ct value $\geq 5$ is reported in the Adenovirus Ct columns, the sample results can be recorded as positive for the specific analyte.

\* Detection of Internal Control is not required in the Cy3 channel for a positive Adenovirus result. High viral load can lead to reduced or absent Internal Control signal. Detection of IC2 is required for a valid negative Adenovirus result.

# d. Detection limit

Prior to the LoD determination, a sequence homology study was performed to classify the 60 known Human Adenoviruses into 21 different groups. Homology groups were established according to the number and the position of base mismatches with the best primer (forward and reverse) and the best probe which comprise the Adenovirus Rgene US amplification/detection premix.

One Human Adenovirus (AdV) type was chosen from each group of homologies to represent each of the 21 different sequences. The LoD study was set up so that each virus was tested using 4, 5, or 6 different dilutions near the estimated LoD and 15 replicates per dilution. Each replicate was taken through the DNA extraction step to amplification and detection. A total of 22 LoD’s were determined comprising the 21 selected Human Adenoviruses, plus Adenovirus serotype 2 (a member of adenovirus species C). The homology groups and Adenovirus subtypes/species are displayed in the table below:

<table><tr><td rowspan=1 colspan=1>Homology Group</td><td rowspan=1 colspan=1>HAdV Type</td><td rowspan=1 colspan=1>Species</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>AdV 12</td><td rowspan=3 colspan=1>A</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>AdV 18</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>AdV 31</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>AdV 3</td><td rowspan=2 colspan=1>B1</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>AdV 7</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>AdV 11</td><td rowspan=1 colspan=1>B2</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>AdV 1</td><td rowspan=3 colspan=1>C</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>AdV 2,5</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>AdV 6</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>AdV 39</td><td rowspan=8 colspan=1>D</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>AdV 15</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>AdV 9</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>AdV 8</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>AdV 17</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>AdV 25</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>AdV 28</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>AdV 42</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>AdV 4</td><td rowspan=1 colspan=1>E</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>AdV 40</td><td rowspan=2 colspan=1>F</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>AdV 41</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>AdV 52</td><td rowspan=1 colspan=1>G</td></tr></table>

The Limit of Detection (LoD) was determined for the Adenovirus R-gene US assay by limiting dilution analysis of representative Adenoviruses from each of the 21 homology groups listed above as well as Adenovirus serotype 2. Fifteen replicates of viral dilutions were tested for Adenovirus positivity over a wide range of dilutions that surround the empirical LoD. The LoD was established by selecting the highest dilution of virus which could be detected at least $9 5 \%$ of the time.

Adenovirus R-gene US Limit of Detection Summary   

<table><tr><td rowspan=1 colspan=1>Homology Group</td><td rowspan=1 colspan=1>HAdV Species</td><td rowspan=1 colspan=1>HAdV Type</td><td rowspan=1 colspan=1>LoD 95%TCID50/ml</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=3 colspan=1>A</td><td rowspan=1 colspan=1>HAdV 12</td><td rowspan=1 colspan=1>.000416</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>HAdV 18</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>HAdV 31</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=2 colspan=1>B1</td><td rowspan=1 colspan=1>HAdV 3</td><td rowspan=1 colspan=1>0.00316</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>HAdV 7</td><td rowspan=1 colspan=1>0.0445</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>B2</td><td rowspan=1 colspan=1>HAdV 11</td><td rowspan=1 colspan=1>889</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 1</td><td rowspan=1 colspan=1>0.0209</td></tr><tr><td rowspan=1 colspan=1>8-1</td><td rowspan=2 colspan=1>C</td><td rowspan=1 colspan=1>HAdV 2</td><td rowspan=1 colspan=1>0.00625</td></tr><tr><td rowspan=1 colspan=1>8-2</td><td rowspan=1 colspan=1>HAdV 5</td><td rowspan=1 colspan=1>0.044</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 6</td><td rowspan=1 colspan=1>0.00512</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 39</td><td rowspan=1 colspan=1>0.0158</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 15</td><td rowspan=1 colspan=1>0.316</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 9</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=2 colspan=1>D</td><td rowspan=1 colspan=1>HAdV 8</td><td rowspan=1 colspan=1>0.000812</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>HAdV 17</td><td rowspan=1 colspan=1>158</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 25</td><td rowspan=1 colspan=1>0.05</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 28</td><td rowspan=1 colspan=1>28.1</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>HAdV 42</td><td rowspan=1 colspan=1>0.05</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>E</td><td rowspan=1 colspan=1>HAdV 4</td><td rowspan=1 colspan=1>0.183</td></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=2 colspan=1>F</td><td rowspan=1 colspan=1>HAdV 40</td><td rowspan=1 colspan=1>0.0104</td></tr><tr><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>HAdV 41</td><td rowspan=1 colspan=1>0.158</td></tr><tr><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>G</td><td rowspan=1 colspan=1>HAdV 52</td><td rowspan=1 colspan=1>na*</td></tr></table>

\* HAdV 52 was not available via organism collections and therefore the LoD was determined using plasmid DNA. The LoD was calculated to be 5000 copies $/ \mathrm { m l }$ by qPCR.

# e. Analytical reactivity

# Cross-Reactivity

A panel of 61 potentially cross-reacting microorganisms (34 viral samples and 27 bacterial samples), representing respiratory pathogens or flora commonly present in the nasopharynx, was evaluated for cross-reactivity. Each potential cross-reactant was individually spiked into an Adenovirus negative nasal wash/aspirate at a challenging concentration. Adenovirus positive samples at 100 and 3 times the limit of detection (100x LoD and 2x LoD) were also tested. The Adenovirus R-gene US assay did not cross-react with any of the cross-reactants tested.

Adenovirus R-gene US Cross-Reactivity Results   

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2></td><td rowspan=1 colspan=2>Adenovirus R-gene US</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=2>Concentrationtested</td><td rowspan=1 colspan=1>530nm - ADVCt (cycles)</td><td rowspan=1 colspan=1>560nm - IC2 Ct(cycles)</td></tr><tr><td rowspan=2 colspan=1>Adenovirus 3</td><td rowspan=1 colspan=2>100x LoD</td><td rowspan=1 colspan=1>27.9</td><td rowspan=1 colspan=1>26.6</td></tr><tr><td rowspan=1 colspan=2>2x LoD</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>25.4</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.0</td></tr><tr><td rowspan=1 colspan=1>Epstein Barr Virus</td><td rowspan=1 colspan=2>107 cp/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.5</td></tr><tr><td rowspan=1 colspan=1>BK Virus</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.6</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus 1</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Herpes Simplex Virus 2</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>Varicella Zoster Virus</td><td rowspan=1 colspan=2>103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 6</td><td rowspan=1 colspan=2>&gt;103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 7</td><td rowspan=1 colspan=2>104 cp/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Human Herpes Virus 8</td><td rowspan=1 colspan=2>104 cp/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.6</td></tr><tr><td rowspan=1 colspan=1>Influenza A Virus</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Influenza B Virus</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus A</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.8</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus B</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.3</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus A</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.3</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus B</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Human Bocavirus 1</td><td rowspan=1 colspan=2>106 cp/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=2>106 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 14</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 87</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1B</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Echovirus 25</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.8</td></tr><tr><td rowspan=1 colspan=1>Coxackie B2</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Coxsackie A9</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.6</td></tr><tr><td rowspan=1 colspan=1>Echovirus 9</td><td rowspan=1 colspan=2>105 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Poliovirus S3</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.6</td></tr><tr><td rowspan=1 colspan=1>Echovirus 30</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Parechovirus 1</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.3</td></tr><tr><td rowspan=1 colspan=1>Parechovirus 2</td><td rowspan=1 colspan=2>104 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=2>104 pfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.9</td></tr><tr><td rowspan=1 colspan=1>Mumps</td><td rowspan=1 colspan=2>104 pfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.6</td></tr></table>

<table><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>Adenovirus R-gene US</td></tr><tr><td rowspan=1 colspan=1>Bacteria</td><td rowspan=1 colspan=1>Concentrationtested</td><td rowspan=1 colspan=1>530nm - ADVCt (cycles)</td><td rowspan=1 colspan=1>560nm - IC2 Ct(cycles)</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.0</td></tr><tr><td rowspan=1 colspan=1>Bordetella parapertussis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>106 ccu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>103 TCID50/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Legionella pneumophila</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Bordetella bronchiseptica</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.2</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus epidermidis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.3</td></tr><tr><td rowspan=1 colspan=1>Klebsiella pneumoniae</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.6</td></tr><tr><td rowspan=1 colspan=1>Haemophilus influenzae</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Serratia marcescens</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Proteus mirabilis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.8</td></tr><tr><td rowspan=1 colspan=1>Pseudomonas aeruginosa</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>Stenotrophomonas maltophila</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Citrobacter freundii</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.4</td></tr><tr><td rowspan=1 colspan=1>Citrobacter koseri</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.0</td></tr><tr><td rowspan=1 colspan=1>Enterobacter cloacae</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.1</td></tr><tr><td rowspan=1 colspan=1>Acinobacter baumannii</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>Streptococcus agalactiae</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.5</td></tr><tr><td rowspan=1 colspan=1>Staphylococcus aureus</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.5</td></tr><tr><td rowspan=1 colspan=1>Klebsiella oxytoca</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.8</td></tr><tr><td rowspan=1 colspan=1>Enterobacter kobei</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.5</td></tr><tr><td rowspan=1 colspan=1>Morganella morganii</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>26.0</td></tr><tr><td rowspan=1 colspan=1>Streptococcus constellatus</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.6</td></tr><tr><td rowspan=1 colspan=1>Raoultella ornithinolytica</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.8</td></tr><tr><td rowspan=1 colspan=1>Haemophilus parainfluenza</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.6</td></tr><tr><td rowspan=1 colspan=1>Branhamella catarrhalis</td><td rowspan=1 colspan=1>106 cfu/mL</td><td rowspan=1 colspan=1>negative</td><td rowspan=1 colspan=1>25.8</td></tr></table>

# Reactivity

No additional strains beyond the 22 tested for LoD were tested for reactivity in analytical studies.

# f. Interference studies:

# Interfering Substances

Potentially interfering or inhibitory substances that may be present in nasopharyngeal swabs or interfere with the PCR reaction were evaluated for the viral strains indicated below. To determine the potential interference of endogenous and exogenous chemical PCR inhibitors on Adenovirus detection, clinically relevant amounts of interfering substances were added to nasopharyngeal samples spiked with Adenovirus at $2 \mathbf { x }$ LoD and $1 0 \mathrm { x }$ LoD. A reference sample with no interfering substance was included at both $1 0 \mathrm { x }$ and $2 \mathbf { x }$ LoD.

The following table shows the interfering substances used for this study. Ten percent of the recommended dose was added to the sample unless otherwise stated in the table. The substances consisted of liquid nasal sprays, ingestible pills and lozenges, and endogenous substances.

Interfering Substances   

<table><tr><td rowspan=1 colspan=1>Substance Name</td><td rowspan=1 colspan=1>ActiveIngredient</td><td rowspan=1 colspan=1>ConcentrationTested</td><td rowspan=1 colspan=1>Justification</td></tr><tr><td rowspan=1 colspan=1>Mucin (Bovinesubmaxillaryglandtype I-S)</td><td rowspan=1 colspan=1>Purified mucinprotein</td><td rowspan=1 colspan=1>60 μg/mL</td><td rowspan=1 colspan=1>1000x maximumlevel present inserum</td></tr><tr><td rowspan=1 colspan=1>Blood (human) EDTAanticoagulated</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>2% (v/v)</td><td rowspan=1 colspan=1>Same concentrationtested for similarIVD&#x27;s</td></tr><tr><td rowspan=1 colspan=1>Tamiflu® 45 mg</td><td rowspan=1 colspan=1>Oseltamivir</td><td rowspan=1 colspan=1>7.5 mg/mL</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Afrin nasal spray</td><td rowspan=1 colspan=1>OxymetazolineHC1</td><td rowspan=1 colspan=1>7.45% (v/v)</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Tobrex 0.3%</td><td rowspan=1 colspan=1>Tobramycin</td><td rowspan=1 colspan=1>9% (v/v) = 0.27mg/mL</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Synagis 50 mg</td><td rowspan=1 colspan=1>Palivizumab</td><td rowspan=1 colspan=1>81.8% (v/v) =18 mg</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr><tr><td rowspan=1 colspan=1>Relenza</td><td rowspan=1 colspan=1>Zanamivir</td><td rowspan=1 colspan=1>45% (v/v) = 1mg</td><td rowspan=1 colspan=1>10% of totalrecommended dose</td></tr></table>

Detailed chemical interference results are presented in the following table. Note that the study was performed in two separate runs with half of the potential interferents tested in run 1 and the other half in run 2. “Reference run $1 ^ { \circ }$ is the Adenovirus only control for the first run. “Reference run $2 ^ { \circ }$ is the Adenovirus only control for the second run. The two Adenovirus only reference samples were tested at concentrations of $1 0 \mathrm { x }$ LoD and 2x LoD. The data in the column indicating delta Ct values were calculated by taking the difference in average $\mathrm { C t }$ cycles between the sample containing the potential chemical interferent and the “reference run” associated with that test.

Adenovirus R-gene US Interfering Substances   

<table><tr><td rowspan=1 colspan=1>SubstanceName</td><td rowspan=1 colspan=1>AdV 3Concentration</td><td rowspan=1 colspan=1>AdV 3Ct (avgcycles)</td><td rowspan=1 colspan=1>AdV DeltaCt (cycles)</td><td rowspan=1 colspan=1>InternalControl 2 Ct(cycles)</td><td rowspan=1 colspan=1>InternalControl 2Delta Ct(cycles)</td></tr><tr><td rowspan=2 colspan=1>Referencerun 1</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.77</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>26.17</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>33.90</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>25.97</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=2 colspan=1>Mucin</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>31.77</td><td rowspan=1 colspan=1>1.00</td><td rowspan=1 colspan=1>26.27</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>33.65</td><td rowspan=1 colspan=1>-0.25</td><td rowspan=1 colspan=1>25.97</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=2 colspan=1>Blood</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>32.00</td><td rowspan=1 colspan=1>1.23</td><td rowspan=1 colspan=1>26.23</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>33.63</td><td rowspan=1 colspan=1>-0.27</td><td rowspan=1 colspan=1>25.87</td><td rowspan=1 colspan=1>-0.10</td></tr><tr><td rowspan=2 colspan=1>Tamiflu®</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>33.33</td><td rowspan=1 colspan=1>2.57</td><td rowspan=1 colspan=1>26.90</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>36.90</td><td rowspan=1 colspan=1>3.00</td><td rowspan=1 colspan=1>27.03</td><td rowspan=1 colspan=1>1.07</td></tr><tr><td rowspan=2 colspan=1>Afrin NasalSpray</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>32.30</td><td rowspan=1 colspan=1>1.53</td><td rowspan=1 colspan=1>26.33</td><td rowspan=1 colspan=1>0.17</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.40</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>26.40</td><td rowspan=1 colspan=1>0.43</td></tr><tr><td rowspan=2 colspan=1>Tobrex 0.3%</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>31.53</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>26.27</td><td rowspan=1 colspan=1>0.10</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.27</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>25.97</td><td rowspan=1 colspan=1>0.00</td></tr><tr><td rowspan=2 colspan=1>Synagis 50mg</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>31.55</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>26.13</td><td rowspan=1 colspan=1>0.03</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>33.27</td><td rowspan=1 colspan=1>-0.63</td><td rowspan=1 colspan=1>25.93</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>SubstanceName</td><td rowspan=1 colspan=1>AdV 3Concentration</td><td rowspan=1 colspan=1>AdV 3Ct (avgcycles</td><td rowspan=1 colspan=1>AdV DeltaCt (cycles)</td><td rowspan=1 colspan=1>InternalControl 2 Ct(cycles)</td><td rowspan=1 colspan=1>InternalControl 2Delta Ct(cycles)</td></tr><tr><td rowspan=2 colspan=1>Referencerun 2</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.87</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>25.30</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>32.80</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>25.17</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=2 colspan=1>Relenza</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.50</td><td rowspan=1 colspan=1>-0.37</td><td rowspan=1 colspan=1>25.60</td><td rowspan=1 colspan=1>0.30</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.40</td><td rowspan=1 colspan=1>1.60</td><td rowspan=1 colspan=1>25.13</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=2 colspan=1>FluMist®</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.77</td><td rowspan=1 colspan=1>-0.10</td><td rowspan=1 colspan=1>25.37</td><td rowspan=1 colspan=1>0.07</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.15</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>25.07</td><td rowspan=1 colspan=1>-0.10</td></tr><tr><td rowspan=2 colspan=1>NeoSynephrine®</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>31.50</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>25.83</td><td rowspan=1 colspan=1>0.53</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.10</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>25.57</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=2 colspan=1>WalgreensO riginalAnefrinNasal Spray</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>31.50</td><td rowspan=1 colspan=1>0.63</td><td rowspan=1 colspan=1>26.50</td><td rowspan=1 colspan=1>1.20</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.20</td><td rowspan=1 colspan=1>1.40</td><td rowspan=1 colspan=1>25.67</td><td rowspan=1 colspan=1>0.50</td></tr><tr><td rowspan=2 colspan=1>ZicamHomeopathicNasal Gel</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.70</td><td rowspan=1 colspan=1>-0.17</td><td rowspan=1 colspan=1>26.03</td><td rowspan=1 colspan=1>0.73</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.47</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>25.57</td><td rowspan=1 colspan=1>0.40</td></tr><tr><td rowspan=2 colspan=1>WalgreensSaline NasalSpray</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>32.03</td><td rowspan=1 colspan=1>1.17</td><td rowspan=1 colspan=1>26.13</td><td rowspan=1 colspan=1>0.83</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>35.73</td><td rowspan=1 colspan=1>2.93</td><td rowspan=1 colspan=1>25.63</td><td rowspan=1 colspan=1>0.47</td></tr><tr><td rowspan=2 colspan=1>Chloraseptic SoreThroatLozenges</td><td rowspan=1 colspan=1>10x LoD</td><td rowspan=1 colspan=1>30.37</td><td rowspan=1 colspan=1>-0.50</td><td rowspan=1 colspan=1>25.27</td><td rowspan=1 colspan=1>-0.03</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>34.47</td><td rowspan=1 colspan=1>1.67</td><td rowspan=1 colspan=1>25.80</td><td rowspan=1 colspan=1>0.63</td></tr></table>

# Microbial Interference

To determine the potential interference of common nasal microorganisms on qualitative Adenovirus detection, a panel of microorganisms was spiked into Adenovirus positive NP washes/aspirates containing Adenovirus at two times the limit of detection (2x LoD). Crossing threshold (Ct) values were compared with a reference sample with no additional microorganism.

Potential Interferents Tested for Microbial Interference   

<table><tr><td rowspan=1 colspan=1>Microorganism</td><td rowspan=1 colspan=1>Source</td><td rowspan=1 colspan=1>ConcentrationTested</td></tr><tr><td rowspan=1 colspan=1>Adenovirus 3(reference)</td><td rowspan=1 colspan=1>ATCC VR-3 virus cultured on MRC5 cells</td><td rowspan=1 colspan=1>~2x LoD (0.0066TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>Staphylococcusaureus</td><td rowspan=1 colspan=1>ATCC 33497</td><td rowspan=1 colspan=1>106 cfu/mL</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>ATCC 9797</td><td rowspan=1 colspan=1>106 cfu/mL</td></tr><tr><td rowspan=1 colspan=1>RespiratorySyncytial Virus A</td><td rowspan=1 colspan=1>ATCC VR-26 virus cultured on MRC5 cells</td><td rowspan=1 colspan=1>104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Escherichia coli</td><td rowspan=1 colspan=1>Zeptometrix, ref0801517</td><td rowspan=1 colspan=1>106 cfu/mL</td></tr><tr><td rowspan=1 colspan=1>Cytomegalovirus</td><td rowspan=1 colspan=1>ATCC VR-977</td><td rowspan=1 colspan=1>104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza 3</td><td rowspan=1 colspan=1>ATCC VR-93 virus cultured on MK2 cells</td><td rowspan=1 colspan=1>106 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>A/PR/8/34 ATCC VR-1469 virus culturedon MDCK cells</td><td rowspan=1 colspan=1>105 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>Strain and source not specified; cultured onMDCK cells</td><td rowspan=1 colspan=1>104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Rhinovirus 1B</td><td rowspan=1 colspan=1>ATCC VR-1645 strain B632 virus culturedon MRC5 cells</td><td rowspan=1 colspan=1>10³ TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>Source not specified; cultured on MK2 cells</td><td rowspan=1 colspan=1>104 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Measles</td><td rowspan=1 colspan=1>NCPV ref0809213v strainMvi/Nottingham/GBR/18.04</td><td rowspan=1 colspan=1>103 pfu/mL</td></tr><tr><td rowspan=1 colspan=1>Mumps</td><td rowspan=1 colspan=1>NCPV, ref0809281v strain 20082</td><td rowspan=1 colspan=1>104 pfu/mL</td></tr></table>

As shown in the table below, Adenovirus was consistently detected with no significant change in the Adenovirus Ct count regardless of whether an additional microorganism was present; thus indicating that there was no interference caused by the microorganisms tested.

Adenovirus R-gene US Microbial Interference   

<table><tr><td colspan="1" rowspan="1">Microorganism</td><td colspan="1" rowspan="1">AdV Ct (avgcycles)</td><td colspan="1" rowspan="1">AdV Delta Ct(cycles)</td><td colspan="1" rowspan="1">InternalControl2 Ct(cycles)</td><td colspan="1" rowspan="1">InternalControl 2Delta Ct(cycles)</td></tr><tr><td colspan="1" rowspan="1">AdV 3 2x LoD (Run1)</td><td colspan="1" rowspan="1">32.17</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">26.20</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + E. coli</td><td colspan="1" rowspan="1">31.73</td><td colspan="1" rowspan="1">-0.43</td><td colspan="1" rowspan="1">26.13</td><td colspan="1" rowspan="1">-0.07</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + CMV</td><td colspan="1" rowspan="1">32.33</td><td colspan="1" rowspan="1">0.17</td><td colspan="1" rowspan="1">26.23</td><td colspan="1" rowspan="1">0.03</td></tr><tr><td colspan="1" rowspan="1">AdV 3 +Parainfluenza 3</td><td colspan="1" rowspan="1">31.97</td><td colspan="1" rowspan="1">-0.20</td><td colspan="1" rowspan="1">26.30</td><td colspan="1" rowspan="1">0.10</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + InfluenzaA</td><td colspan="1" rowspan="1">32.23</td><td colspan="1" rowspan="1">0.07</td><td colspan="1" rowspan="1">25.97</td><td colspan="1" rowspan="1">-0.23</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + Influenza B</td><td colspan="1" rowspan="1">33.03</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">25.80</td><td colspan="1" rowspan="1">-0.40</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + Rhinovirus1B</td><td colspan="1" rowspan="1">31.60</td><td colspan="1" rowspan="1">-0.57</td><td colspan="1" rowspan="1">25.80</td><td colspan="1" rowspan="1">-0.40</td></tr><tr><td colspan="1" rowspan="1">AdV 3 +Coronavirus NL63</td><td colspan="1" rowspan="1">32.07</td><td colspan="1" rowspan="1">-0.10</td><td colspan="1" rowspan="1">26.40</td><td colspan="1" rowspan="1">0.20</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + Measlesvirus</td><td colspan="1" rowspan="1">32.90</td><td colspan="1" rowspan="1">0.73</td><td colspan="1" rowspan="1">26.07</td><td colspan="1" rowspan="1">-0.13</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + Mumpsvirus</td><td colspan="1" rowspan="1">32.53</td><td colspan="1" rowspan="1">0.37</td><td colspan="1" rowspan="1">25.97</td><td colspan="1" rowspan="1">-0.23</td></tr><tr><td colspan="1" rowspan="1">AdV 3 2x LoD (Run2)</td><td colspan="1" rowspan="1">34.07</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">25.37</td><td colspan="1" rowspan="1">n/a</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + Staph.aureus</td><td colspan="1" rowspan="1">32.90</td><td colspan="1" rowspan="1">-1.17</td><td colspan="1" rowspan="1">25.37</td><td colspan="1" rowspan="1">0.00</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + B. pertussis</td><td colspan="1" rowspan="1">32.77</td><td colspan="1" rowspan="1">-1.30</td><td colspan="1" rowspan="1">25.53</td><td colspan="1" rowspan="1">0.17</td></tr><tr><td colspan="1" rowspan="1">AdV 3 + RSV A</td><td colspan="1" rowspan="1">34.93</td><td colspan="1" rowspan="1">0.87</td><td colspan="1" rowspan="1">25.67</td><td colspan="1" rowspan="1">0.30</td></tr></table>

# g. Assay cut-off:

The cut-off determination was performed through testing of 218 samples: 98 negative, 67 high negative, and 53 positive. Of the 98 negative samples, all but four were below 20 AFU, four were between 20 and 30 AFU and none were over 30. Of the 67 high negative samples, 31 were below 20 AFU and 36 were between 20 and 30 AFU. Of the 53 positive samples, all were over 40 AFU. No fluorescence values were obtained between 30 and 40 AFU among the 218 samples tested. The threshold default value (30 AFU) of the SmartCycler II was confirmed to be the correct cut-off value for the Adenovirus R-gene US assay.

This cut-off value was further confirmed in a preliminary clinical study, performed at Caen Hospital (France), on 184 clinical samples, prior to US clinical studies. The cutoff was then confirmed by analysis of the U.S. clinical studies results at the 3 sites, New York, Chicago and Memphis. The 30 AFU value was confirmed as the cut-off to achieve the best sensitivity and specificity.

# h. Carryover Contamination:

This study was performed to examine potential carry-over/cross-contamination with the Adenovirus R-gene US assay by testing simulated human adenovirus (HAdV) high positive samples run in series with HAdV negative samples. The high positive samples consisted of negative nasal washes/aspirates spiked with HAdV type 4 at four logs above the Limit of Detection (LoD). The samples were processed and extracted in a high positive/negative alternating fashion on the bioMérieux NucliSENS® easyMAG® extraction system and likewise processed and run on the Cepheid SmartCycler II

instrument in an alternating fashion. Eleven high positive and eleven negative samples and a negative control were extracted per run on the easyMAG for a total of 55 high positive samples and 55 negative samples over 5 extraction runs and 4 amplification runs. When tested with high positive samples, no carry-over contamination was seen for the Adenovirus R-gene US assay.

The data from the study are summarized in the table below.

<table><tr><td rowspan=1 colspan=1>EasyMagNucliSeNsRun</td><td rowspan=1 colspan=1>Run 1</td><td rowspan=1 colspan=2>Run 2</td><td rowspan=1 colspan=2>Run 3</td><td rowspan=1 colspan=1>Run 4</td><td rowspan=1 colspan=2>Run 5</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>SmartCyclerII Run</td><td rowspan=1 colspan=2>Run 1</td><td rowspan=1 colspan=2>Run 2</td><td rowspan=1 colspan=3>Run 3</td><td rowspan=1 colspan=1>Run 4</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>High Positivesample</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=2 colspan=1>55/55(100%)</td></tr><tr><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>19.4</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>19.5</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19.3</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>19.6</td></tr><tr><td rowspan=1 colspan=1>Negativesample</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>4/4</td><td rowspan=1 colspan=1>7/7</td><td rowspan=1 colspan=1>8/8</td><td rowspan=1 colspan=1>3/3</td><td rowspan=1 colspan=1>11/11</td><td rowspan=1 colspan=1>1/1</td><td rowspan=1 colspan=1>10/10</td><td rowspan=1 colspan=1>55/55(100%)</td></tr></table>

2. Comparison studies:

a. Method comparison with predicate device: Not applicable

b. Matrix comparison: Not applicable

c. Transport media compatibility :

Both types of Viral Transport Media (Universal Transport Medium from DHI, MicroTest™ M4RT Transport from Remel) claimed in the package insert were used in the prospective clinical study and were compatible with the Adenovirus R-gene assay.

3. Clinical studies:

Prospective clinical study:

Performance characteristics of the Adenovirus R-gene US assay were determined in a multi-center prospective investigational study employing 3 geographically diverse institutions in the US. The study began with recruitment and testing commencing on September 28, 2010. Specimens used in the study represent excess nasal specimens (NPS and NPA) that were collected from symptomatic men and women of all ages suspected of respiratory infection and submitted for routine care or analysis. Age ranges for patients providing specimens in the study ranged from 1 month to 101 years. The database was frozen on November 21, 2011 after testing 1186 swab and $3 9 5 \mathrm { N P }$ wash/aspirate specimens and this dataset was used in the analysis. Two NP aspirate/wash specimens and three NPS were excluded from the analysis due to

protocol deviations. The Adenovirus R-gene US assay was compared with rapid culture (shell vial) followed by direct fluorescent antibody (DFA) screening and identification using the D3Ultra™ Direct Fluorescent Antibody (DFA) Respiratory Virus screening & ID kit from Diagnostic Hybrids (DHI). Each US site analyzed around 280-650 samples (swabs and washes/aspirates), for a total of 1576 samples. The studies were performed on the remnant of the specimen collected from children and adults after the routine clinical care.

Each sample was tested by Adenovirus R-gene US assay and rapid culture (shell vial) followed by DFA screening and identification. The Argene and DHI test were performed according to the instructions provided by the manufacturer. The shell vial culture was performed following standard laboratory procedures. The chosen sites were hospital virology laboratories experienced in DFA, cell culture and real time PCR. Performances (sensitivity and specificity) of the Adenovirus R-gene US assay were calculated as compared to the method of viral culture followed by DFA staining with DHI test.

Clinical Study Results – Swab Specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Culture Result</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>AdenovirusR-geneUS</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>44</td><td rowspan=1 colspan=1>43a</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>Sensitivity:91.7% (80.0%-97.7%) 95%CI</td></tr><tr><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>4b</td><td rowspan=1 colspan=1>1092</td><td rowspan=1 colspan=1>1096</td><td rowspan=1 colspan=1>Specificity:96.2% (94.9%-97.2%) 95%CI</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>1135</td><td rowspan=1 colspan=1>1183</td><td rowspan=1 colspan=1></td></tr></table>

a 42/43 samples confirmed as Adenovirus positive by quantitative PCR b 1/4 samples confirmed as Adenovirus positive by quantitative PCR

Clinical Study Results – Wash/Aspirate Specimens   

<table><tr><td rowspan=2 colspan=2></td><td rowspan=1 colspan=2>Culture Result</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=2 colspan=1>AdenovirusR-geneUS</td><td rowspan=1 colspan=1>Detected</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>21a</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>Sensitivity: 100%(86.7%-100%)95%CI</td></tr><tr><td rowspan=1 colspan=1>NotDetected</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>351</td><td rowspan=1 colspan=1>Specificity: 94.4%(91.5%-96.5%)95%CI</td></tr><tr><td rowspan=1 colspan=2></td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>372</td><td rowspan=1 colspan=1>393</td><td rowspan=1 colspan=1></td></tr></table>

a 19/21 samples confirmed as Adenovirus positive by quantitative PCR

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

The general demographic data for all eligible prospective specimens are presented in the two following tables:

Prevalence (positives as determined by reference method) and Expected Value (positives as determined by Adenovirus R-gene US assay) observed during the clinical study in swab specimens   

<table><tr><td rowspan=1 colspan=6>Swab Prospective Study</td></tr><tr><td rowspan=1 colspan=1>SubjectAge (years)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>TotalAdeno R-gene USpositive</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Total DHIpositive</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>&lt;2</td><td rowspan=1 colspan=1>519</td><td rowspan=1 colspan=1>45</td><td rowspan=1 colspan=1>8.7%</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>4.8%</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>280</td><td rowspan=1 colspan=1>26</td><td rowspan=1 colspan=1>9.3%</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5.0%</td></tr><tr><td rowspan=1 colspan=1>6-19</td><td rowspan=1 colspan=1>229</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>2.2%</td></tr><tr><td rowspan=1 colspan=1>19-64</td><td rowspan=1 colspan=1>112</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>43</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1183</td><td rowspan=1 colspan=1>87</td><td rowspan=1 colspan=1>7.35%</td><td rowspan=1 colspan=1>48</td><td rowspan=1 colspan=1>4.06%</td></tr></table>

Prevalence (positives as determined by reference method) and Expected Value (positives as determined by Adenovirus R-gene US assay) observed during the clinical study in wash/aspirate specimens   

<table><tr><td rowspan=1 colspan=6>Wash/Aspirate Prospective Study</td></tr><tr><td rowspan=1 colspan=1>SubjectAge (years)</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>TotalAdeno R-gene USpositive</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Total DHIpositive</td><td rowspan=1 colspan=1>ObservedPrevalence</td></tr><tr><td rowspan=1 colspan=1>&lt;2</td><td rowspan=1 colspan=1>275</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>12.0%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>6.2%</td></tr><tr><td rowspan=1 colspan=1>2-5</td><td rowspan=1 colspan=1>57</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>12.3%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.5%</td></tr><tr><td rowspan=1 colspan=1>6-19</td><td rowspan=1 colspan=1>55</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.6%</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3.6%</td></tr><tr><td rowspan=1 colspan=1>19-64</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0.0%</td></tr><tr><td rowspan=1 colspan=1>&gt;65</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>ND</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>393</td><td rowspan=1 colspan=1>42</td><td rowspan=1 colspan=1>10.69%</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>5.34%</td></tr></table>

# N. Instrument Name:

Cepheid SmartCycler II utilizing Dx software version $1 . 7 6$ or 3.0

# O. System Descriptions:

# 1. Modes of Operation:

The bioMérieux NucliSens easyMAG is an automated nucleic acid isolation and purification system that is based upon the silica extraction technology. The easyMAG is capable of processing a total of 24 reactions with variable sample types, sample volumes, and elution volumes within a single run. Nucleic acid is purified within a single cartridge by several steps that include lysis and binding of nucleic acid to high affinity magnetic silica beads, a series of wash steps and heated elution of purified nucleic acid from the silica beads.

The Cepheid SmartCycler $\operatorname { I I } \circled { \mathbb { R } }$ Real Time instrument with Dx software version $1 . 7 6$ or $3 . 0 \mathrm { a }$ is used to perform real time PCR amplification and detection of nucleic acid. The Cepheid SmartCycler $\operatorname { I I } \circled { \mathbb { R } }$ instrument is an integrated nucleic acid amplification and detection instrument system based on Cepheid’s proprietary microprocessor-controlled I-CORE module. For purified DNA samples, the SmartCycler $\operatorname { I I } \circled { \mathbb { R } }$ instrument enables polymerase chain reaction (PCR) for the amplification of DNA, and hybridization of fluorogenic target-specific probes for the detection of the amplified DNA.

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ___X____ or No

3. Specimen Identification: Specimens ID’s are manually entered into the user interface by the user.

4. Specimen Sampling and Handling:

Liquid samples from nasopharyngeal swabs and/or nasal wash/aspirates are manually transferred collected and transferred into tubes for nucleic acid extraction and purification.

5. Calibration: Calibration is not recommended.

6. Quality Control:

The positive control plasmid enables the experiment to be properly validated. This positive control is amplified with the same primers as the viral DNA of any Adenovirus present in patient samples. A bacteriophage Internal Control (IC2) is added to every sample to detect the presence of inhibitors or lysis failure.

# P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered In The “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.