# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: K122189

B. Purpose for Submission: The purpose of this submission is to show that the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay is substantially equivalent to Prodesse’s ProFlu™ Assay and Pro $\mathrm { { h M P V + } }$ Assay. The ProFlu Assay detects RSV RNA while the Pro hMPV $^ +$ assay detects hMPV RNA.

C. Measurand: This assay detects the presence of RSV and hMPV nucleic acid using RT-PCR. Both viruses have negative single-stranded RNA genomes. The RT-PCR primers are developed to bind to conserved regions of L viral polymerase and NS2 genes for RSV and RNA polymerase gene for hMPV.

D. Type of Test: This is a nucleic acid based test using reverse transcription polymerase chain reaction (RT-PCR). The negative sense RNA genomes of RSV and hMPV are reverse transcribed and then amplified during the RT-PCR reaction. The presence of RSV or hMPV RNA is then detected through sequence-specific labeled probe that is cleaved during PCR amplification releasing the fluorescence reporter dye from the quencher dye.

E. Applicant: Quidel Corporation

F. Proprietary and Established Names: Quidel® Molecular $\mathrm { R S V + h M P V }$ Assay

G. Regulatory Information:

1. Regulation section: 21 CFR866.3980, Respiratory viral panel multiplex nucleic acid assay

2. Classification: Class II

3. Product code: OEM, OCC

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

The Quidel Molecular $\mathrm { R S V + h M P V }$ assay is a multiplex Real Time RT-PCR assay for the qualitative detection and identification of respiratory syncytial virus and human metapneumovirus RNA extracted from nasal and nasopharyngeal swabs specimens from patients with signs and symptoms of respiratory infection. This in vitro diagnostic test is intended to aid in the differential diagnosis of respiratory syncytial virus and human metapneumovirus infections in humans in conjunction with clinical and epidemiological risk factors. This test is not intended to differentiate the two subtypes of RSV or the four genetic sub-lineages of hMPV.

Negative results do not preclude RSV infection and/or hMPV infection and should not be used as the sole basis for diagnosis, treatment or other patient management decisions.

2. Indication(s) for use: Same as intended use.

3. Special conditions for use statement(s): Prescription only.

4. Special instrument requirements: Cepheid SmartCycler II or ABI 7500 Fast Dx

I. Device Description: The Quidel Molecular $\mathrm { R S V + h M P V }$ Assay detects viral nucleic acids that have been extracted from a patient sample using the NucliSENS $^ \mathrm { \textregistered }$ easyMAG $^ \mathrm { \textregistered }$ automated extraction platform. An RT-PCR reaction is then performed in a single tube generating amplicons for each of the target viruses present in the sample. This reaction is performed utilizing either the Cepheid SmartCycler® II or the Applied Biosystems 7500 Fast DX. Identification of RSV and hMPV and the PRC occurs by the use of target-specific primers and fluorescent-labeled probes that hybridize to conserved regions in the genomes of RSV and hMPV and the PRC. Results are analyzed by either the Cepheid or ABI software and reported to the user.

# J. Substantial Equivalence Information:

1. Predicate device name(s): ProFlu™ Assay, Pro hMPV $^ +$ Assay 2. Predicate 510(k) number(s): k092500, k082688 3. Comparison with predicate:

<table><tr><td colspan="4" rowspan="1">Subject Device and Comparator Device Comparison</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject DeviceQuidel Molecular RSV+ hMPV Assay</td><td colspan="1" rowspan="1">Predicate DeviceProdesse ProFlu+</td><td colspan="1" rowspan="1">Predicate DeviceProdesse ProhMPV+</td></tr><tr><td colspan="1" rowspan="2">Intended Use</td><td colspan="1" rowspan="2">The Quidel MolecularRSV + hMPV assay is amultiplex Real TimeRT-PCR assay for the invitro qualitativedetection andidentification ofrespiratory syncytialvirus and humanmetapneumovirus viralRNA extracted fromnasal andnasopharyngeal swabsspecimens with signsand symptoms ofrespiratory infection.This in vitro diagnostictest is intended to aid inthe differential diagnosisof respiratory syncytialvirus and humanmetapneumovirusinfections. This test isnot intended todifferentiate the fourgenetic sub-lineages ofhMPV.Negative results do notpreclude RSV or hMPVinfection and should notbe used as the sole basisfor diagnosis, treatmentor other patientmanagement decisions.</td><td colspan="1" rowspan="1">The ProFluTM+ Assay isa multiplex Real-TimePCR (RT-PCR) in vitrodiagnostic test for therapid and qualitativedetection anddiscrimination ofInfluenza A Virus,Influenza B Virus, andRespiratory SyncytialVirus (RSV) nucleicacids isolated andpurified fromnasopharyngeal (NP)swab specimens obtainedfrom symptomaticpatients. This test isintended for use to aid in</td><td colspan="1" rowspan="2">The Pro hMPV+ Assay isa Real Time RT-PCR invitro diagnostic test forthe qualitative detectionof humanMetapneumovirus(hMPV) nucleic acidisolated and purifiedfrom nasopharyngealswab (NP) specimensobtained from individualsexhibiting signs andsymptoms of acuterespiratory infection.This assay targets ahighly conserved regionof the Nucleocapsid geneof hMPV. The detectionof hMPV nucleic acidfrom symptomaticpatients aids in thediagnosis of humanrespiratory hMPVinfection if used inconjunction with otherclinical and laboratoryfindings. This test is notintended to differentiatethe four genetic sub-lineages of hMPV.Negative results do notpreclude hMPV infectionand should not be used asthe sole basis fordiagnosis, treatment orother managementdecisions.</td></tr><tr><td colspan="1" rowspan="1">the differential diagnosisof Influenza A, InfluenzaB and RSV viralinfections in humans andis not intended to detectInfluenza C.Negative results do notpreclude influenza orRSV virus infection andshould not be used as thesole basis for treatmentor other managementdecisions. It isrecommended thatnegative RSV results beconfirmed by culture.Performancecharacteristics forInfluenza A Virus wereestablished whenInfluenza A/H3 and</td></tr><tr><td colspan="5" rowspan="1">Subject Device and Comparator Device Comparison</td></tr><tr><td colspan="1" rowspan="1">Item</td><td colspan="1" rowspan="1">Subject DeviceQuidel Molecular RSV+ hMPV Assay</td><td colspan="1" rowspan="1">Predicate DeviceProdesse ProFlu+</td><td colspan="2" rowspan="2">Predicate DeviceProdesse ProhMPV+</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">A/H1 were thepredominant Influenza Aviruses in circulation.When other Influenza Aviruses are emerging,performancecharacteristics may vary.If infection with a novelInfluenza A virus issuspected based oncurrent clinical andepidemiologicalscreening criteriarecommended by publichealth authorities,specimens should becollected withappropriate infectioncontrol precautions fornovel virulent Influenzaviruses and sent to stateor local healthdepartment for testing.Viral culture should notbe attempted in thesecases unless a BSL 3+facility is available toreceive and culturespecimens.</td></tr><tr><td colspan="1" rowspan="1">Assay Target</td><td colspan="1" rowspan="1">Respiratory syncytialvirus, hMPV</td><td colspan="1" rowspan="1">Influenza A virus,influenza B virus,respiratory syncytialvirus</td><td colspan="2" rowspan="1">hMPV</td></tr><tr><td colspan="1" rowspan="1">SampleTypes</td><td colspan="1" rowspan="1">Nasal swab,nasopharyngeal swab</td><td colspan="1" rowspan="1">Nasopharyngeal swab</td><td></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">ExtractionMethods</td><td colspan="1" rowspan="1">bioMérieux easyMAGAutomated MagneticExtraction Reagents</td><td colspan="1" rowspan="1">Roche MagNA Pure LCTotal Nucleic AcidIsolation Kit or thebioMérieux easyMAG</td><td colspan="2" rowspan="1">Roche MagNA Pure LCTotal Nucleic AcidIsolation Kit or thebioMérieux easyMAG</td></tr></table>

<table><tr><td rowspan=1 colspan=7>Subject Device and Comparator Device Comparison</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Subject DeviceQuidel Molecular RSV+ hMPV Assay</td><td rowspan=1 colspan=4>Predicate DeviceProdesse ProFlu+</td><td rowspan=1 colspan=1>Predicate DeviceProdesse ProhMPV+</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>Automated MagneticExtraction Reagents</td><td rowspan=1 colspan=1>Automated MagneticExtraction Reagents</td></tr><tr><td rowspan=1 colspan=1>AssayMethodology</td><td rowspan=1 colspan=1>PCR-based system fordetecting the presence orabsence of viral RNA inclinical specimens</td><td rowspan=1 colspan=4>PCR-based system fordetecting the presence orabsence of viral RNA inclinical specimens</td><td rowspan=1 colspan=1>PCR-based system fordetecting the presence orabsence of viral RNA inclinical specimens</td></tr><tr><td rowspan=1 colspan=1>DetectionTechniques</td><td rowspan=1 colspan=1>Multiplex assay usingdifferent reporter dyesfor each target</td><td rowspan=1 colspan=4>Multiplex assay usingdifferent reporter dyes foreach target</td><td rowspan=1 colspan=1>Multiplex assay usingdifferent reporter dyesfor each target</td></tr><tr><td rowspan=1 colspan=1>Viral Targets</td><td rowspan=1 colspan=1>RSV: L viralpolymerase and NS2geneshMPV: RNApolymerase gene</td><td rowspan=1 colspan=4>Influenza A: Matrix gene;Influenza B: Non-structural NS1 and NS2</td><td rowspan=1 colspan=1>Nucleocapsid</td></tr><tr><td rowspan=11 colspan=1>LoD</td><td rowspan=2 colspan=1>The analyticalsensitivity (limit ofdetection or LoD) of theQuidel Molecular hMPVassay was determinedusing quantified(TCID50/mL) cultures of2 RSV strains and 4hMPV strains (A1, A2,B1, B2) serially dilutedin negativenasopharyngeal matrix.</td><td rowspan=2 colspan=4>The analytical sensitivity(limit of detection orLoD) of the ProFlu+Assay was determinedusing quantified(TCID50/mL) cultures of4 Influenza A (2 H1N1and 2 H3N2), 2 InfluenzaB, 2 RespiratorySyncytial Virus Type A,and 2 RespiratorySyncytial Virus Type B</td><td rowspan=11 colspan=1>The analytical sensitivity(limit of detection orLoD) of the Pro hMPV+Assay was determinedusing quantified(TCID50/mL) cultures of2 hMPV (subtype A2 andsubtype B2) strainsserially diluted innasopharyngeal clinicalmatrix. Each viral strainwas extracted using theRoche MagNA Pure LCinstrument and tested inreplicates of 20 perconcentration of virus.Analytical sensitivity(LoD) as defined as thelowest concentration atwhich ≥ 95% of allreplicates tested positive,ranged from 102 - 101TCID50/mL.</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Each dilution was</td><td rowspan=1 colspan=2>strains</td><td rowspan=2 colspan=3>strains serially diluted innasopharyngeal clinical</td></tr><tr><td rowspan=1 colspan=1>extracted using the</td><td rowspan=1 colspan=2>nasop</td><td rowspan=3 colspan=3>nasopharyngeal clinicalmatrix. Each viral strainwas extracted using the</td><td rowspan=1 colspan=1>aryngea</td></tr><tr><td rowspan=1 colspan=1>NucliSENS easyMAG</td><td></td></tr><tr><td rowspan=1 colspan=1>System and tested in</td><td></td></tr><tr><td rowspan=1 colspan=1>replicates of 20 perconcentration of virus</td><td rowspan=4 colspan=4>Roche MagNA Pure LCinstrument and tested inreplicates of 20 perconcentration of virus.Analytical sensitivity(LoD), as defined as the</td></tr><tr><td rowspan=1 colspan=1>on both the AppliedBiosystems® 7500 Fast</td></tr><tr><td rowspan=2 colspan=1>Dx platform. Analyticalsensitivity (LoD) is</td></tr><tr><td rowspan=1 colspan=3>(LoD), as defined as the</td></tr><tr><td rowspan=1 colspan=1>defined as the lowestconcentration at which95% of all replicates</td><td rowspan=1 colspan=4>lowest concentration atwhich 95% of allreplicates tested positive,</td></tr></table>

<table><tr><td rowspan=1 colspan=4>Subject Device and Comparator Device Comparison</td></tr><tr><td rowspan=1 colspan=1>Item</td><td rowspan=1 colspan=1>Subject DeviceQuidel Molecular RSV+ hMPV Assay</td><td rowspan=1 colspan=1>Predicate DeviceProdesse ProFlu+</td><td rowspan=1 colspan=1>Predicate DeviceProdesse ProhMPV+</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>tested positive, rangedfrom 10-1 to 101TCID50/mL.</td><td rowspan=1 colspan=1>ranged from 102 to 10-1TCID50/mL.</td><td rowspan=1 colspan=1></td></tr></table>

# K. Standard/Guidance Document Referenced (if applicable): N/A

# L. Test Principle:

This is a nucleic acid based test using reverse transcription polymerase chain reaction (RT-PCR). Viral infection is detected through the use of PCR to amplify and detect viral RNA. The negative sense RNA genomes of RSV and hMPV are reverse-transcribed and then amplified during the RT-PCR reaction. The presence of RSV or hMPV RNA is then detected through sequence-specific labeled probe that is cleaved during PCR amplification releasing the fluorescence reporter dye from the quencher dye.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

# a. Precision/Reproducibility:

Reproducibility studies were performed on both thermal cycler instruments, the Cepheid SmartCycler II and ABI 7500 Fast Dx. The studies were conducted at three separate testing sites where two users tested each virus sample in triplicate over 5 days. The hMPV-A2 or RSV A viruses were tested at High Negative (0.05xLoD), Low Positive, close to the detection limit $2 \mathbf { x }$ LoD) and Medium Positive (5x LoD) concentrations. The viruses were diluted in a negative nasal matrix. Negative and positive controls were included in the study. The positive control was each virus diluted in UTM and the negative controls were UTM only (Negative control) and negative nasal matrix (RSV and hMPV negative). To conduct each test, the contrived samples were extracted using the bioMérieux easyMAG system and tested on either the Cepheid SmartCycler II or the ABI 7500 Fast Dx.

<table><tr><td colspan="11" rowspan="1">Reproducibility Results  Cepheid SmartCycler II</td></tr><tr><td colspan="1" rowspan="2">PanelMember ID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td></tr><tr><td colspan="1" rowspan="1">RSV HighNegative0.05x LoD</td><td colspan="1" rowspan="1">8/30</td><td colspan="1" rowspan="1">44.3</td><td colspan="1" rowspan="1">9.3</td><td colspan="1" rowspan="1">6/30</td><td colspan="1" rowspan="1">47.4</td><td colspan="1" rowspan="1">5.4</td><td colspan="1" rowspan="1">1/30</td><td colspan="1" rowspan="1">48.7</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">15/90</td></tr><tr><td colspan="1" rowspan="2">PanelMember ID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="2">TotalPositiveResults</td></tr><tr><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt**</td><td colspan="1" rowspan="1">%CV</td></tr><tr><td colspan="1" rowspan="1">RSV LowPositive2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">7.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.4</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">RSV MedPositive 5xLoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">5.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.7</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.5</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">RSVNegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">RSVNegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">RSVPositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.0</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.0</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.6</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">hMPV HighNegative0.15x LoD</td><td colspan="1" rowspan="1">12/30</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">8.3</td><td colspan="1" rowspan="1">6/30</td><td colspan="1" rowspan="1">41.8</td><td colspan="1" rowspan="1">6.3</td><td colspan="1" rowspan="1">8/30</td><td colspan="1" rowspan="1">34.9</td><td colspan="1" rowspan="1">6.4</td><td colspan="1" rowspan="1">26/90</td></tr><tr><td colspan="1" rowspan="1">hMPV LowPositive2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">6.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">6.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">31.1</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">hMPV MedPositive 5xLoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.3</td><td colspan="1" rowspan="1">2.0</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.9</td><td colspan="1" rowspan="1">3.2</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">29.8</td><td colspan="1" rowspan="1">3.4</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">hMPVNegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">hMPVNegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">hMPVPositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.3</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">30.8</td><td colspan="1" rowspan="1">2.5</td><td colspan="1" rowspan="1">90/90</td></tr></table>

<table><tr><td colspan="11" rowspan="1">Reproducibility Results  Applied Biosystems 7500 Fast Dx</td></tr><tr><td colspan="1" rowspan="2">PanelMemberID</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td><td colspan="1" rowspan="1">AVECt*</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PositiveResults</td></tr><tr><td colspan="1" rowspan="1">RSV HighNegative0.15x LoD</td><td colspan="1" rowspan="1">14/30</td><td colspan="1" rowspan="1">30.7</td><td colspan="1" rowspan="1">8.3</td><td colspan="1" rowspan="1">3/30</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">2.4</td><td colspan="1" rowspan="1">9/30</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">26/90</td></tr><tr><td colspan="1" rowspan="1">RSV LowPositive</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.7</td><td colspan="1" rowspan="1">7.4</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.8</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">23.4</td><td colspan="1" rowspan="1">4.5</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">Panel</td><td colspan="3" rowspan="1">Site 1</td><td colspan="3" rowspan="1">Site 2</td><td colspan="3" rowspan="1">Site 3</td><td colspan="1" rowspan="1">Total</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">RSV MedPositive 5xLoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.5</td><td colspan="1" rowspan="1">4.0</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">21.8</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">21.1</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">89/89</td></tr><tr><td colspan="1" rowspan="1">RSVNegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">RSVNegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">RSVPositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.7</td><td colspan="1" rowspan="1">6.5</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.0</td><td colspan="1" rowspan="1">8.5</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.6</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">hMPVHighNegative0.15x LoD</td><td colspan="1" rowspan="1">14/30</td><td colspan="1" rowspan="1">24.9</td><td colspan="1" rowspan="1">19.5</td><td colspan="1" rowspan="1">6/30</td><td colspan="1" rowspan="1">29.1</td><td colspan="1" rowspan="1">10.2</td><td colspan="1" rowspan="1">10/30</td><td colspan="1" rowspan="1">25.1</td><td colspan="1" rowspan="1">16.9</td><td colspan="1" rowspan="1">30/90</td></tr><tr><td colspan="1" rowspan="1">hMPVLowPositive2x LoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">21.0</td><td colspan="1" rowspan="1">8.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">21.5</td><td colspan="1" rowspan="1">4.1</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">21.6</td><td colspan="1" rowspan="1">5.8</td><td colspan="1" rowspan="1">90/90</td></tr><tr><td colspan="1" rowspan="1">hMPVMedPositive 5xLoD</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">18.9</td><td colspan="1" rowspan="1">3.3</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.6</td><td colspan="1" rowspan="1">3.0</td><td colspan="1" rowspan="1">29/29</td><td colspan="1" rowspan="1">19.2</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">89/89</td></tr><tr><td colspan="1" rowspan="1">hMPVNegative</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">hMPVNegativeControl</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/30</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">0/90</td></tr><tr><td colspan="1" rowspan="1">hMPVPositiveControl</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.7</td><td colspan="1" rowspan="1">3.9</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">20.0</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">30/30</td><td colspan="1" rowspan="1">19.9</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">90/90</td></tr></table>

· Average Ct based on the average of only positive results. Ct values of 0 were left out of this calculation.

The data reported shows $100 \%$ detection of hMPV and RSV at Low and Medium Positive levels demonstrating good reproducibility of the test. Detection was lower than $50 \%$ for the High Negative samples which is acceptable and typical for nucleic acid tests.

b. Linearity/assay reportable range: N/A   
c. Traceability, Stability, Expected values (controls, calibrators, or methods):   
A Fresh vs. Frozen study was performed using the same contrived specimens in negative clinica

matrix as was used in the Reproducibility Study. Briefly, hMPV-A2 and RSV were diluted into negative clinical matrix at 5x LoD, 2x LoD and $0 . 1 5 \mathrm { x }$ LoD. The viral panel was tested before freezing and then after frozen storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ for 22-56 days; each sample was tested in triplicate. No intermediate time points were tested for the stability studies. Additionally, 99 clinical samples were tested within 72 hours of collection and once again after 7 days at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

<table><tr><td rowspan=3 colspan=2></td><td rowspan=1 colspan=8>Applied Biosystems 7500 Fast Dx Data</td></tr><tr><td rowspan=1 colspan=4>RSV</td><td rowspan=1 colspan=4>hMPV</td></tr><tr><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>0.15xLoD</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>5x LoD</td><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>0.15xLoD</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=3 colspan=1>FreshTesting</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>20.9</td><td rowspan=1 colspan=1>22.7</td><td rowspan=1 colspan=1>29.6</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>18.4</td><td rowspan=1 colspan=1>19.2</td><td rowspan=1 colspan=1>25.0</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>6.8%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>2.6%</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>11.1%</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=3 colspan=1>Post-Freeze</td><td rowspan=1 colspan=1>Average</td><td rowspan=1 colspan=1>21.7</td><td rowspan=1 colspan=1>23.0</td><td rowspan=1 colspan=1>32.4</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>19.5</td><td rowspan=1 colspan=1>21.0</td><td rowspan=1 colspan=1>27.7</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>STDEV</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>1.3%</td><td rowspan=1 colspan=1>1.7%</td><td rowspan=1 colspan=1>5.9%</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>3.1%</td><td rowspan=1 colspan=1>12.4%</td><td rowspan=1 colspan=1>N/A</td></tr></table>

Sample storage at $- 7 0 \mathrm { { } ^ { \circ } C }$ for up to 56 days did not impact the ability of the Quidel Molecular $\mathrm { R S V + h M P V }$ Assay to detect either RSV or hMPV positive near the clinical cut-off. $2 \mathbf { x }$ and ${ 5 } \mathbf { x }$ LoD samples were both detected $100 \%$ (90/90) of the time. There was also no statistical difference in $\mathrm { C t }$ value before and after freezing for the $2 \mathbf { x }$ LoD and ${ 5 } \mathrm { x }$ LoD samples. The data of the 99 clinical samples, not shown here, also did not show any significant change in Ct value over time when samples were stored at $- 7 0 \mathrm { { } ^ { \circ } C }$ .

Stability studies were also performed on rehydrated master mix and extracted RNA samples. hMPV or RSV viruses at $3 \mathbf { x }$ LoD were added to negative sample matrix, extracted for RNA and the RT-PCR reaction was performed; the master mix study was performed in triplicate while the RNA extraction study was performed in quadruplicate. The master mix was stable at room temperature and $2 ^ { \circ }$ to $8 ^ { \circ } \mathrm { C }$ for 2 days and for 5 days at ${ } _ { - 2 0 } { } ^ { \circ } \mathrm { C }$ . Extracted RNA stability studies showed that hMPV and RSV extracted RNA was stable for 6 hours at $2 ^ { \circ }$ to $8 \mathrm { { } ^ { \circ } C }$ , 4 hours at room temperature, one month at $- 2 0 ^ { \circ } \mathrm { C }$ and for 96 hours at $- 2 0 ^ { \circ } \mathrm { C }$ with up to 3 freeze/thaw cycles.

There was no significant change in Ct value during any of the stability studies which supports the stability claims.

# d. Detection limit:

The LoD was determined for hMPV and RSV using each of the 4 genetic sub-lineages and 2 subtypes of each virus respectively. Virus was diluted in pooled negative NP swab clinical matrix. Each virus sub-lineage/subtype was diluted and each dilution was extracted and run 20 times using both the Cepheid SmartCycler II and ABI 7500 Fast Dx, The LoD was determined to be the lowest titer at which the virus was detected greater than $9 5 \%$ of the time (19/20 or 20/20). For hMPV-A2, hMPV-B2, RSV A and RSV B three separate kit lots were tested and a representative titer was chosen that showed the best repeatability. The final LoD titers were reported for each instrument and all future calculations used the LoD for the specific virus and thermal cycler on which an experiment was being conducted.

<table><tr><td rowspan=3 colspan=1>Virus</td><td rowspan=1 colspan=3>ABI 7500 Fast Dx</td><td rowspan=1 colspan=3>Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=1>Lot 1(110411)</td><td rowspan=1 colspan=1>Lot 2(2N0020)</td><td rowspan=1 colspan=1>Lot 3(908842)</td><td rowspan=1 colspan=1>Lot 1(110411)</td><td rowspan=1 colspan=1>Lot 2(2N0020)</td><td rowspan=1 colspan=1>Lot 3(908842)</td></tr><tr><td rowspan=1 colspan=1>LoDTCID50/mL</td><td rowspan=1 colspan=1>LoDTCID50/mL</td><td rowspan=1 colspan=1>LoDTCID50/mL</td><td rowspan=1 colspan=1>LoDTCID50/mL</td><td rowspan=1 colspan=1>LoDTCID50/mL</td><td rowspan=1 colspan=1>LoDTCID50/mL</td></tr><tr><td rowspan=1 colspan=1>hMPV-A1</td><td rowspan=1 colspan=1>1.76E+01</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>2.65E+01</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV-A2</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>1.25E+00</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>4.17E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV-B1</td><td rowspan=1 colspan=1>1.05E+00</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>7.88E+00</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV-B2</td><td rowspan=1 colspan=1>3.21E+00</td><td rowspan=1 colspan=1>3.21E+00</td><td rowspan=1 colspan=1>3.21E+00</td><td rowspan=1 colspan=1>3.21E+00</td><td rowspan=1 colspan=1>9.63E+00</td><td rowspan=1 colspan=1>3.21E+00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>6.29E-01</td><td rowspan=1 colspan=1>5.10E-01</td><td rowspan=1 colspan=1>5.10E-01</td><td rowspan=1 colspan=1>1.89E+00</td><td rowspan=1 colspan=1>1.70E+00</td><td rowspan=1 colspan=1>5.10E-01</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>7.5E-01</td><td rowspan=1 colspan=1>7.5E-01</td><td rowspan=1 colspan=1>7.5E-01</td><td rowspan=1 colspan=1>7.5E-01</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>2.25E+00</td></tr></table>

The reported LoD values for each instrument are as follows:

<table><tr><td rowspan=1 colspan=3>LoD Value Summary for the 7500 Fast Dx &amp; SmartCycler</td></tr><tr><td rowspan=1 colspan=1>Virus</td><td rowspan=1 colspan=1>LoD TCID50/mL7500 Fast Dx</td><td rowspan=1 colspan=1>LoD TCID50/mLSmartCycler</td></tr><tr><td rowspan=1 colspan=1>RSV A</td><td rowspan=1 colspan=1>6.29E-01</td><td rowspan=1 colspan=1>1.89E+00</td></tr><tr><td rowspan=1 colspan=1>RSV B</td><td rowspan=1 colspan=1>7.50E-01</td><td rowspan=1 colspan=1>2.25E+00</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>hMPV-A1</td><td rowspan=1 colspan=1>1.76E+01</td><td rowspan=1 colspan=1>2.65E+01</td></tr><tr><td rowspan=1 colspan=1>hMPV-A2</td><td rowspan=1 colspan=1>4.17E+00</td><td rowspan=1 colspan=1>4.17E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV-B1</td><td rowspan=1 colspan=1>1.05E+00</td><td rowspan=1 colspan=1>7.88E+00</td></tr><tr><td rowspan=1 colspan=1>hMPV-B2</td><td rowspan=1 colspan=1>3.21E+00</td><td rowspan=1 colspan=1>9.63E+00</td></tr></table>

# e. Analytical specificity:

# Interference:

To evaluate the potential interference of other common respiratory pathogens the organisms in the table below were tested. Samples were prepared by adding RSV B or hMPV-B2 and the potential interference organism into negative clinical matrix at 3x LoD. The potential interference organisms were tested at one clinically relevant concentration, for viruses $1 0 ^ { 4 } – 1 0 ^ { 6 }$ $\mathrm { T C I D 5 0 / m L }$ and for bacteria $1 0 ^ { 6 } – 1 0 ^ { 8 } \mathrm { C F U / m L }$ . Additionally RSV and hMPV were tested for interference when both organisms are in the same sample. For the intra test interference RSV B was tested at 33,300x LoD, $3 3 3 \mathrm { x }$ LoD, $1 0 0 \mathrm { x }$ LoD, $3 0 \mathrm { x }$ LoD and $3 \mathbf { x }$ LoD. hMPV-B2 was tested at 10,000x LoD, $1 0 0 \mathrm { x }$ LoD, $3 0 \mathrm { x }$ LoD, 3x LoD. The prepared samples were extracted and run on both the Cepheid SmartCycler II and ABI 7500 Fast Dx.

There was no interference seen between any of the organisms tested, all tests were positive for either RSV or hMPV. A decrease in Ct value was seen in the presence of Neisseria gonorrhoeae, but sequence analysis showed no homology between the test probe sequence and Neisseria gonorrhoeae sequence. At very high levels (at or above 1,000x LoD) RSV may interfere with the detection of hMPV. However, during the clinical trial co-infection was found to be a rare event, and the concentrations used when interference occurred are much higher than typically present in clinical specimens. Therefore, the risk of RSV/hMPV interference is very low.

<table><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Final Concentration</td><td colspan="1" rowspan="1">RSV detection</td><td colspan="1" rowspan="1">hMPVdetection</td></tr><tr><td colspan="1" rowspan="1">hMPV A2</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">hMPV B2</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">RSV A Long</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">RSV B Wash</td><td colspan="1" rowspan="1">3xLoD</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">A/Mexico/4108/2009</td><td colspan="1" rowspan="1">1.4E+07 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">B/Florida/04/2006</td><td colspan="1" rowspan="1">5.24E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus1/Adenoid 71</td><td colspan="1" rowspan="1">5.67E+04 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 2</td><td colspan="1" rowspan="1">2.51E+09 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 3</td><td colspan="1" rowspan="1">1.10E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 4</td><td colspan="1" rowspan="1">9.54E+06TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 5</td><td colspan="1" rowspan="1">4.00E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 7</td><td colspan="1" rowspan="1">1.25E+07 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 11</td><td colspan="1" rowspan="1">3.85E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 14</td><td colspan="1" rowspan="1">8.63E+04 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 22</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 31</td><td colspan="1" rowspan="1">1.25E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Adenovirus 35</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">1.41E+04 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">1.70E+06TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">1.67E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B4</td><td colspan="1" rowspan="1">2.43E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">2.28E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus</td><td colspan="1" rowspan="1">8.76E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Echovirus 7</td><td colspan="1" rowspan="1">5.38E+08 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Echovirus 9</td><td colspan="1" rowspan="1">1.50E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">1.05E+08 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.05E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 71</td><td colspan="1" rowspan="1">2.68E+03 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Enterovirus 70</td><td colspan="1" rowspan="1">1.66E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Epstein Barr Virus</td><td colspan="1" rowspan="1">5,000cp/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">HSV Type 1 MacInytre</td><td colspan="1" rowspan="1">1.95E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Human Rhinovirus</td><td colspan="1" rowspan="1">1.26E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">HSV Type 2 G strain</td><td colspan="1" rowspan="1">3.67E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Measles virus</td><td colspan="1" rowspan="1">1.95E+06 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mumps virus</td><td colspan="1" rowspan="1">2.75E+08 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 1</td><td colspan="1" rowspan="1">2.50E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 2</td><td colspan="1" rowspan="1">2.20E+04 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 3</td><td colspan="1" rowspan="1">9.10E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Type 4</td><td colspan="1" rowspan="1">1.36E+07 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Varicella Zoster Virus</td><td colspan="1" rowspan="1">7.5E+02TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">7.6E+08 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">1.65E+07 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Chlamydophila pneumoniae</td><td colspan="1" rowspan="1">1.0E+3 DNA copies/ul</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">2.10E+05 TCID50/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophila</td><td colspan="1" rowspan="1">1.36E+07 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mycobacteriumintracellulare</td><td colspan="1" rowspan="1">4.30E+06 CFP/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">1.10E+07 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium avium</td><td colspan="1" rowspan="1">3.07E+05 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma pneumoniae</td><td colspan="1" rowspan="1">4.64E+06 CCU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">3.60E+06 CFU/mL</td><td colspan="1" rowspan="1">+$</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">2.71E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">5.20E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Proteus mirabilis</td><td colspan="1" rowspan="1">2.39E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">6.90E+02 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Neisseria menigitidis</td><td colspan="1" rowspan="1">3.60E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Neisseria mucosa</td><td colspan="1" rowspan="1">7.60E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">3.90E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">3.30E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">2.98E+03 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">2.39E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">1.06E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">9.50E+05 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">2.43E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">1.49E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">7.50E+05 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">7.80E+06 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.79E+05 CFU/mL</td><td colspan="1" rowspan="1">+</td><td colspan="1" rowspan="1">+</td></tr></table>

Cross Reactivity:

To evaluate the potential cross reactivity of the test with other common respiratory pathogens, the organisms in the table above were tested. Samples were prepared by adding the potential interference organism into negative clinical matrix at 3x LoD. The potential interference organisms were tested at one clinically relevant concentration, for viruses $1 0 ^ { 4 } – 1 0 ^ { 6 }$ T $\mathrm { \dot { C } I D 5 0 / m L }$ and for bacteria $1 0 ^ { 6 } { - } 1 0 ^ { 8 } \mathrm { C F U / m L }$ (the same titers as in the Interference table above). The prepared samples were extracted and run on both the Cepheid SmartCycler II and ABI 7500 Fast Dx.

There was no cross-reactivity seen between any of the organisms tested, all tests were negative for both RSV or hMPV.

Inclusivity:

Each virus was diluted into negative clinical matrix at 1x-3x LoD. Samples were extracted and tested in triplicate on either the ABI 7500 Fast Dx or Cepheid SmartCycler II. Virus was detected in all positive samples in all but one virus strain, hMPV B1 Peru3. Only 2/3 samples were positive during testing of hMPV B1 Peru 3. This result was probably due to a combination of small differences in LoD among different genetic sub-lineage and titer variability near the LoD. The study adequately shows that the assay can detect the two RSV subtypes and four hMPV genetic sub-lineages.

<table><tr><td rowspan=1 colspan=5>RSV Inclusivity Panel</td></tr><tr><td rowspan=1 colspan=1>Subtype</td><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>TCID50/mL</td><td rowspan=1 colspan=1>( ABI 7500Fast Dx)</td><td rowspan=1 colspan=1>(CepheidSmartCyclerII</td></tr><tr><td rowspan=1 colspan=1>A (NIBSC)</td><td rowspan=1 colspan=1>A-2</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>A</td><td rowspan=1 colspan=1>A-2</td><td rowspan=1 colspan=1>5.67E+00</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>Washington</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>9320</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>B</td><td rowspan=1 colspan=1>CH9318(1B)</td><td rowspan=1 colspan=1>2.25E+00</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Positive</td></tr></table>

<table><tr><td colspan="5" rowspan="1">hMPV Inclusivity Panel</td></tr><tr><td colspan="1" rowspan="1">Subtype</td><td colspan="1" rowspan="1">Strain</td><td colspan="1" rowspan="1">TCID50/mL</td><td colspan="1" rowspan="1">(ABI 7500Fast Dx)</td><td colspan="1" rowspan="1">(CepheidSmartCycler II)</td></tr><tr><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">IA3-2002 G Gene</td><td colspan="1" rowspan="1">7.95E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A1</td><td colspan="1" rowspan="1">IA10-2003</td><td colspan="1" rowspan="1">7.95E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">IA14-2003 G Gene</td><td colspan="1" rowspan="1">1.25E+1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">A2</td><td colspan="1" rowspan="1">Clinical Isolate</td><td colspan="1" rowspan="1">1.25E+1</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">Peru2-2002 G Gene</td><td colspan="1" rowspan="1">2.36E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B1</td><td colspan="1" rowspan="1">Peru3-2003 G Gene</td><td colspan="1" rowspan="1">2.36E+01</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">Peru1-2002 G Gene</td><td colspan="1" rowspan="1">9.63E+00</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">Peru6-2003</td><td colspan="1" rowspan="1">9.63E+00</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">IA18-2003 G Gene</td><td colspan="1" rowspan="1">9.63E+00</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">B2</td><td colspan="1" rowspan="1">IA18-2003G Gene</td><td colspan="1" rowspan="1">9.63E+00</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr><tr><td colspan="1" rowspan="1">hMPV (NIBSC)</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">Positive</td></tr></table>

The Quidel Molecular $\mathrm { R S V + h M P V }$ assay detected $100 \%$ of the RSV and hMPV on both the ABI 7500 Fast Dx and Cepheid SmartCycler $\textsuperscript { \textregistered }$ II platforms.

# f. Assay cut-off:

Threshold: The threshold for the ABI 7500 Fast Dx was established using the background fluorescence from cycle 3-15. The threshold is set to above the background fluorescence, below the plateau and linear regions of the amplification curve, within the exponential phase of the amplification curve. The threshold is designed to maximize sensitivity and specificity. The threshold settings for each analyte were determined individually using positive and negative clinical specimens.

The threshold for the Cepheid SmartCycler II was established using the background subtraction performed by the software algorithm and adjusting the threshold based on amplification curves obtained from positive and negative clinical samples.

Assay cut-off: The Ct cut-off was determined after testing clinical specimens and choosing a Ct value that was higher than the Ct value of the lowest positive concentration of analyte.

<table><tr><td rowspan=1 colspan=3>Summary of Threshold Settings for the Quidel Molecular RSV + hMPV Assay byInstrument</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>ABI 7500 Fast Dx</td><td rowspan=1 colspan=1>SmartCycler</td></tr><tr><td rowspan=1 colspan=1>hMPV</td><td rowspan=1 colspan=1>5.4e+04</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>RSV</td><td rowspan=1 colspan=1>8.0e+04</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>MS2</td><td rowspan=1 colspan=1>2.4e+04</td><td rowspan=1 colspan=1>10</td></tr></table>

<table><tr><td colspan="5" rowspan="1">Recommended Ct Cut-Off Values by Instrument</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">ABI 7500 Fast Dx</td><td colspan="2" rowspan="1">SmartCycler</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">Latest Ct ValueObserved*</td><td colspan="1" rowspan="1">RecommendedCt Cut-Off</td><td colspan="1" rowspan="1">Latest Ct ValueObserved</td><td colspan="1" rowspan="1">Recommended CtCut-Off</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">24.66</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">38.00</td><td colspan="1" rowspan="1">50</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">30.08</td><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">49.70</td><td colspan="1" rowspan="1">50</td></tr></table>

\*These values reflect subtraction of first the 10 cycles

2. Comparison studies:

a. Method comparison with predicate device:

Method comparison was performed against the gold standard of culture for RSV testing and a FDA cleared device for hMPV. The testing description and data are listed in the Clinical studies section.

# $b$ . Matrix comparison:

A transport media study was conducted to show efficacy of 5 different transport media (UTM, M4, M4-RT, M5 and M6). RSV and hMPV were spiked into the media at 3x LoD. Each condition was extracted in triplicate, and each extraction was tested twice. There was not statistical difference in Ct value among the media tested on either the ABI 7500 Fast Dx or the Cepheid SmartCycler II.

3. Clinical studies:

a. Clinical Sensitivity and Specificity:

The clinical study for this device was performed during the 2012 respiratory virus season, January – March, 2012. All 1014 nasal swab and nasopharyngeal swap specimens were collected prospectively. One specimen per patient was collected at four US study sites. A total 414 samples where tested fresh and 600 were frozen and tested at a later date. Reference testing for RSV was DSFA or culture with DFA, performed immediately after collection. A FDA cleared hMPV molecular test was used as the hMPV comparator for the Quidel Molecular RSV $+ \mathrm { h M P V }$ Assay.

The performance of the Quidel assay in detecting RSV and hMPV was acceptable. The Quidel assay detected an additional 21 and 28 positive RSV samples not detected by DSFA and cell culture with DFA respectively. This was confirmed by an FDA-cleared RT-PCR assay. The detection of hMPV was equivalent to a FDA cleared hMPV molecular test. Discrepant samples were not further tested for hMPV.

<table><tr><td rowspan=1 colspan=9>Cepheid Smart Cycler II Combined Site - Respiratory syncytial virus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>DSFA &amp; CellCulturew/DFA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>93.9%</td><td rowspan=1 colspan=1>99.3%</td></tr><tr><td rowspan=2 colspan=1>QM RSV +hMPV</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>137</td><td rowspan=1 colspan=1>21*</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>97.6%</td><td rowspan=1 colspan=1>96.3%</td><td rowspan=1 colspan=1>98.4%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>849</td><td rowspan=1 colspan=1>852</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>870</td><td rowspan=1 colspan=1>1010</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* All originally discordant specimens were positive for RSV by an FDA-cleared RT-PCR assay.

<table><tr><td rowspan=1 colspan=9>Cepheid SmartCycler II Combined Site - Human metapneumovirus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FDA ClearedhMPV MolecularTest</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positivepercentagreement</td><td rowspan=1 colspan=1>96.7%</td><td rowspan=1 colspan=1>92.4%</td><td rowspan=1 colspan=1>98.6%</td></tr><tr><td rowspan=2 colspan=1>QM RSV +hMPV</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>145</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>Negativepercentagreement</td><td rowspan=1 colspan=1>99.6%</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>99.9%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>798</td><td rowspan=1 colspan=1>803</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>801</td><td rowspan=1 colspan=1>951</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

<table><tr><td rowspan=1 colspan=9>ABI 7500 Fast Dx Combined Site - Respiratory syncytial virus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>DSFA &amp; CellCulture w/DFA</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Sensitivity</td><td rowspan=1 colspan=1>98.6%</td><td rowspan=1 colspan=1>94.9%</td><td rowspan=1 colspan=1>99.6%</td></tr><tr><td rowspan=2 colspan=1>QM RSV + hMPV</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>138</td><td rowspan=1 colspan=1>28*</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>Specificity</td><td rowspan=1 colspan=1>96.8%</td><td rowspan=1 colspan=1>95.4%</td><td rowspan=1 colspan=1>97.8%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>840</td><td rowspan=1 colspan=1>842</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>140</td><td rowspan=1 colspan=1>868</td><td rowspan=1 colspan=1>1008</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

\* All originally discordant specimens were positive for RSV by an FDA-cleared RT-PCR assay.

<table><tr><td rowspan=1 colspan=9>ABI 7500 Fast Dx Combined Site - Human metapneumovirus</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>FDA ClearedhMPVMolecular Test</td><td rowspan=1 colspan=1>−</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=2>95% CI</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>Positivepercentagreement</td><td rowspan=1 colspan=1>98.0%</td><td rowspan=1 colspan=1>94.3%</td><td rowspan=1 colspan=1>99.3%</td></tr><tr><td rowspan=2 colspan=1>QM RSV +hMPV</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>147</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>156</td><td rowspan=1 colspan=1>Negativepercentagreement</td><td rowspan=1 colspan=1>98.9%</td><td rowspan=1 colspan=1>97.9%</td><td rowspan=1 colspan=1>99.4%</td></tr><tr><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>790</td><td rowspan=1 colspan=1>793</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>150</td><td rowspan=1 colspan=1>799</td><td rowspan=1 colspan=1>949</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr></table>

4. Clinical cut-off: N/A

# 5. Expected values/Reference range:

The prevalence detected for RSV and hMPV during the 2012 respiratory virus season was approximately $1 6 \%$ and $1 5 \%$ respectively. Prevalence was higher for both viruses in children under 5 years of age. The table below shows the overall prevalence and the prevalence for all sites segregated by age group. The prevalence detected by this device is appropriate for expected values for RSV and hMPV during the winter months.

<table><tr><td rowspan=1 colspan=6>Applied BioSystem 7500 Fast Dx</td><td rowspan=1 colspan=5>Cepheid SmartCycler II</td></tr><tr><td rowspan=1 colspan=1>AgeGroup(years)</td><td rowspan=1 colspan=1>RSVPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>totaltested</td><td rowspan=1 colspan=1>RSVPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>hMPVPositive</td><td rowspan=1 colspan=1>Prevalence</td><td rowspan=1 colspan=1>totaltested</td></tr><tr><td rowspan=1 colspan=1>&lt;1</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>28.3%</td><td rowspan=1 colspan=1>38</td><td rowspan=1 colspan=1>15.0%</td><td rowspan=1 colspan=1>254</td><td rowspan=1 colspan=1>71</td><td rowspan=1 colspan=1>27.5%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>13.6%</td><td rowspan=1 colspan=1>258</td></tr><tr><td rowspan=1 colspan=1>1 to 5</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>19.9%</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>21.5%</td><td rowspan=1 colspan=1>354</td><td rowspan=1 colspan=1>67</td><td rowspan=1 colspan=1>19.0%</td><td rowspan=1 colspan=1>73</td><td rowspan=1 colspan=1>20.7%</td><td rowspan=1 colspan=1>352</td></tr><tr><td rowspan=1 colspan=1>6 to 10</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>12.7%</td><td rowspan=1 colspan=1>134</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>4.5%</td><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>11.9%</td><td rowspan=1 colspan=1>134</td></tr><tr><td rowspan=1 colspan=1>11 to15</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>11.3%</td><td rowspan=1 colspan=1>62</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4.8%</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>62</td></tr><tr><td rowspan=1 colspan=1>16 to21</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>33</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>9.1%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>&gt;21</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>10.3%</td><td rowspan=1 colspan=1>175</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4.6%</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>9.7%</td><td rowspan=1 colspan=1>175</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>166</td><td rowspan=1 colspan=1>16.4%</td><td rowspan=1 colspan=1>157</td><td rowspan=1 colspan=1>15.5%</td><td rowspan=1 colspan=1>1012*</td><td rowspan=1 colspan=1>158</td><td rowspan=1 colspan=1>15.6%</td><td rowspan=1 colspan=1>148</td><td rowspan=1 colspan=1>14.6%</td><td rowspan=1 colspan=1>1014</td></tr></table>

# N. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# O. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.