# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k123620 B. Purpose for Submission:

To add a second Adenovirus assay to the already cleared device k120267, k110764, and k103175 in order to improve detection of Adenovirus and to remove the limitation on the detection of Adenovirus species C, serotypes 2 and 6.

# C. Measurand:

Adenovirus, Bordetella pertussis, Chlamydophila pneumoniae, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Coronavirus 229E, Human Metapneumovirus, Human Rhinovirus/ Enterovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype 2009 H1, Influenza B, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, and Respiratory Syncytial Virus nucleic acid target sequences in nasopharyngeal swabs.

# D. Type of Test:

A multiplexed nucleic acid test intended for use with the FilmArray instrument for the qualitative in vitro detection and identification of multiple respiratory pathogen nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infection.

# E. Applicant:

BioFire Diagnostics, Inc.

# F. Proprietary and Established Names:

Established name: FilmArray $^ \mathrm { \textregistered }$ Respiratory Panel (RP)   
Common Name(s): FilmArray $^ \mathrm { \textregistered }$ Respiratory Panel (RP) FilmArray $^ \mathrm { \textregistered }$ Respiratory Panel (RP) System FilmArray $^ \mathrm { \textregistered }$ Respiratory Panel (FilmArray RP) FilmArray RP Panel

G. Regulatory Information:

1. Regulation section: 21 CFR $\ S$ 866.3980, Respiratory Viral Panel Multiplex Nucleic Acid Assay

2. Classification: Class II

3. Product code: OCC, OEM, OOU, OEP, OTG, OQW, OOI, OZZ, OZY, OZX

4. Panel: Microbiology (83)

# H. Intended Use:

1. Intended use(s):

FilmArray $^ \mathrm { \textregistered }$ Respiratory Panel (RP) is a multiplexed nucleic acid test intended for use with the FilmArray Instrument for the simultaneous qualitative detection and identification of multiple respiratory viral and bacterial nucleic acids in nasopharyngeal swabs (NPS) obtained from individuals suspected of respiratory tract infections. The following organism types and subtypes are identified using the FilmArray RP: Adenovirus, Coronavirus 229E, Coronavirus HKU1, Coronavirus NL63, Coronavirus OC43, Human Metapneumovirus, Influenza A, Influenza A subtype H1, Influenza A subtype H3, Influenza A subtype H1- 2009, Influenza B, Parainfluenza Virus 1, Parainfluenza Virus 2, Parainfluenza Virus 3, Parainfluenza Virus 4, Human Rhinovirus/Enterovirus, Respiratory Syncytial Virus, Bordetella pertussis, Chlamydophila pneumoniae, and Mycoplasma pneumoniae. The detection and identification of specific viral and bacterial nucleic acids from individuals exhibiting signs and symptoms of a respiratory infection aids in the diagnosis of respiratory infection if used in conjunction with other clinical and epidemiological information. The results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. Negative results in the setting of a respiratory illness may be due to infection with pathogens that are not detected by this test or, lower respiratory tract infection that is not detected by a nasopharyngeal swab specimen. Positive results do not rule out coinfection with other organisms: the agent(s) detected by the Film Array RP may not be the definite cause of disease. Additional laboratory testing (e.g. bacterial and viral culture, immunofluorescence, and radiography) may be necessary when evaluating a patient with possible respiratory tract infection.

Due to the small number of positive specimens collected for certain organisms during the prospective clinical study, performance characteristics for Bordetella pertussis, Coronavirus 229E, Coronavirus OC43, Influenza A H1, Influenza A H3, Influenza A H1-2009, Influenza B, Mycoplasma pneumoniae, Parainfluenza Virus 1, Parainfluenza

# Virus 2, and Parainfluenza Virus 4 were established primarily with retrospective clinical specimens. Performance characteristics for Chlamydophila pneumoniae were established primarily using contrived clinical specimens.

Due to the genetic similarity between Human Rhinovirus and Enterovirus, the FilmArray RP cannot reliably differentiate them. A positive FilmArray RP Rhinovirus/Enterovirus result should be followed-up using an alternate method (e.g., cell culture or sequence analysis). The FilmArray RP assay for Coronavirus OC43 may cross-react with some isolates of Coronavirus HKU1. A dual positive result may be due to cross-reactivity or may indicate a co-infection.

Performance characteristics for Influenza A were established when Influenza A H1-2009, A H1, and A H3 were the predominant Influenza A viruses in circulation. Performance of detecting Influenza A may vary if other Influenza A strains are circulating or a novel Influenza A virus emerges. If infection with a novel Influenza A virus is suspected based on current clinical and epidemiological screening criteria recommended by public health authorities, specimens should be collected with appropriate infection control precautions for novel virulent Influenza viruses and sent to state or local health departments for testing. Viral culture should not be attempted in these cases unless a BSL $3 +$ facility is available to receive and culture specimens.

2. Indication(s) for use: Same as Intended Use

3. Special conditions for use statement(s): The device is for prescription use only

4. Special instrument requirements: FilmArray Instrument

# I. Device Description:

The FilmArray RP system consists of three basic components: the FilmArray RP reagent kit, the FilmArray instrument, and the FilmArray software. It was fully described earlier, please refer to FDA cleared 510(k) submissions: k103175, k110764, and k120267. The table below summarizes the organisms causing respiratory tract illness that can be detected simultaneously by the FilmArray RP system.

<table><tr><td>Viral Targets Detected</td><td>Bacterial Targets Detected</td></tr><tr><td>Adenovirus Coronavirus HKU1 Coronavirus NL63 Coronavirus 229E Coronavirus OC43 Human Metapneumovirus Rhinovirus/Enterovirus Influenza A, including subtypes H1, H3 and H1-2009 Influenza B Parainfluenza Virus 1 Parainfluenza Virus 2 Parainfluenza Virus 3 Parainfluenza Virus 4 Respiratory Syncytial Virus</td><td>Bordetella pertussis Chlamydophila pneumonia Mycoplasma pneumoniae</td></tr></table>

Changes to device: New Adenovirus assay primers were designed based on an alignment to complete coding sequences (CDS) from the NCBI nucleotide database for all Adenovirus serotypes. The assays design emphasized detection of the respiratory serotypes of species B, C, and E, including HAdv2 and HAdv6. In total, four new primers were added to the outer (PCR1) multiplex and five inner (PCR2) primers were multiplexed as a single Adenovirus assay (Adeno2) in three wells of the array not currently utilized by the system. No changes were made to the existing panel assay primers.

# J. Substantial Equivalence Information:

1. Predicate device name(s): FilmArray Respiratory Panel (RP) 2. Predicate 510(k) number(s): k103175, k110764, k120267 1. Comparison with predicate:

# Similarities between the Modified Device and the Predicate

<table><tr><td>Element</td><td>New Device: FilmArray Respiratory Panel</td><td>Predicate: FilmArray Respiratory Panel (k103175, k110764 and k120267)</td></tr><tr><td colspan="1" rowspan="1">OrganismsDetected</td><td colspan="1" rowspan="1">Influenza A, Influenza A subtypeH1, Influenza A subtype H3,Influenza A subtype H1-2009,Influenza B, Respiratory SyncytialVirus, Human Metapneumovirus,Adenovirus, Parainfluenza 1,Parainfluenza 2, Parainfluenza virus3,Parainfluenza 4,Rhinovirus/Enterovirus,Coronavirus HKU1, CoronavirusNL63, Coronavirus 229E,Coronavirus OC43, Mycoplasmapneumoniae, Chlamydophilapneumoniae, and Bordetellapertussis.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">RNA/DNA</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Multiplex nucleic acid</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SpecimenTypes</td><td colspan="1" rowspan="1">Nasopharyngeal swabs</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TechnologicalPrinciples</td><td colspan="1" rowspan="1">Nested multiplex RT-PCR followedby high resolution melting analysisto confirm identity of amplifiedproduct.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Instrumentation</td><td colspan="1" rowspan="1">FilmArray Instrument</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Time to result</td><td colspan="1" rowspan="1">About 1 hour</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">TestInterpretation</td><td colspan="1" rowspan="1">Automated test interpretation andreport generation. User cannotaccess raw data.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">SamplePreparationMethod</td><td colspan="1" rowspan="1">Sample Processing is automated inthe FilmArray RP pouch.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">ReagentStorage</td><td colspan="1" rowspan="1">Reagents are stored at roomtemperature.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">Controls</td><td colspan="1" rowspan="1">Two controls are included in eachreagent pouch to control for sampleprocessing and both stages of PCRand melt analysis.</td><td colspan="1" rowspan="1">Same</td></tr><tr><td colspan="1" rowspan="1">UserComplexity</td><td colspan="1" rowspan="1">Moderate/Low</td><td colspan="1" rowspan="1">Same</td></tr></table>

# Differences between the Modified Device and the Predicate

<table><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>Modified Device:FilmArray Respiratory Panel</td><td rowspan=1 colspan=1>Predicate:FilmArray Respiratory Panel(k103175, k110764 andk120267)</td></tr><tr><td rowspan=1 colspan=1>Limit ofDetection forAdenovirus</td><td rowspan=1 colspan=1>100 TCID50/mL</td><td rowspan=1 colspan=1>300 TCID50/mL</td></tr><tr><td rowspan=1 colspan=1>Detection ofAdenovirusSerotypes</td><td rowspan=1 colspan=1>Detects all serotypes with similarsensitivity</td><td rowspan=1 colspan=1>Detects Adenovirus species Cserotype 2 and serotype 6 withreduced sensitivity.</td></tr></table>

# K. Standard/Guidance Document Reference (if applicable):

1. User Protocol for Evaluation of Qualitative Test Performance, Clinical and Laboratory Standards Institute Approved Guideline, EP12-A (August 2002)   
2. Molecular Diagnostic Methods for Infectious Diseases, Clinical and Laboratory Standards Institute Approved Guideline, MM3-A (December 1995)   
3. Interference Testing in Clinical Chemistry, Clinical and Laboratory Standards Institute Approved Guideline EP7-A (December 2002)   
4. Guidance for the Content of Premarket Submissions for Software Contained in Medical Devices, FDA Guidance Document (May 11, 2005)   
5. Establishing Performance Characteristics of In Vitro Diagnostic Devices for Detection or Detection and Differentiation of Influenza Viruses (February 15, 2008)   
6. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Detection and Differentiation of Influenza A Virus Subtypes Using Multiplex Assays (October 9, 2009)   
7. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Respiratory Viral Panel Multiplex Nucleic Acid Assay (October 9, 2009)   
8. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Testing for Human Metapneumovirus (hMPV) Using Nucleic Acid Assays (October 9, 2009)   
9. Guidance for Industry and FDA Staff - Class II Special Controls Guidance Document: Instrumentation for Clinical Multiplex Test Systems (March 10, 2005)   
10. Guidance for Industry, FDA Reviewers and Compliance on Off-The-Shelf Software Use in Medical Devices (September 9, 1999)   
11. Nucleic Acid Based in Vitro Diagnostic Devices for Detection of Microbial Pathogens, FDA Guidance Document (DRAFT: December 8, 2005)   
12. User Protocol for Evaluation of Qualitative Test Performance, National Committee on Clinical Laboratory Standards (NCCLS) Approved Guideline, EP12-A (August 2002)   
13. Guidance for Industry and FDA Staff â€“ Use of Symbols on Labels and in Labeling of In Vitro Diagnostic Devices Intended for Professional Use (November 30, 2004)

# L. Test Principle:

The FilmArray RP System is multiplex nucleic acid test system composed of the FilmArray instrument, the FilmArray software (preinstalled on a laptop computer) and the FilmArray RP pouch. The FilmArray RP pouch contains freeze-dried reagents to perform nucleic acid purification, reverse transcription, and nested, multiplex PCR with DNA melt analysis. A FilmArray test is initiated by loading water and a patient NPS sample mixed with the provided Sample Buffer into the FilmArray RP pouch and placing it in the FilmArray instrument. This process is simplified by the use of a specifically designed pouch loading station. After the pouch is prepared, the FilmArray software guides the user though the steps of placing the pouch in the instrument, scanning the pouch barcode, entering the sample identification, and initiating the run. Please refer to previously cleared submissions k103175, k110764, and k120267 for additional information.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

Please refer to previously FDA-cleared 510(k) Premarket Notifications, k103175, k110764, and k120267 for additional information.

b. Linearity/assay reportable range:

Not applicable.

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

There are no changes to the controls for the modified assay. Please refer to previously FDA-cleared 510(k) Premarket Notifications, k103175, k110764, and k120267.

# d. Detection limits:

The analytical sensitivity or Limit of Detection (LoD) for Adenovirus was determined by testing limiting dilutions of quantified cultures with the modified FilmArray RP. LoD is defined as the lowest concentration at which the analyte is consistently detected (detection in $2 9 5 \%$ of samples tested). Simulated NPS sample matrix (cultured human cells in VTM) was spiked with Adenovirus and 20 replicates were tested at the estimated LoD concentration of $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for each of the four respiratory serotypes (AdVC1, AdVC2, AdVE4 and AdVC6). Adenovirus was detected in all 20 replicates for each serotype and the system LoD for Adenovirus was reduced from $3 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in the original panel to $1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ in the modified panel.

<table><tr><td rowspan=2 colspan=1>Adenovirus Serotype</td><td rowspan=1 colspan=3>Modified FilmArray RP</td><td rowspan=1 colspan=1>Original FilmArrayRP</td></tr><tr><td rowspan=1 colspan=1>LoD(TCID50/mL)</td><td rowspan=1 colspan=1>#Positive</td><td rowspan=1 colspan=1>% Positive</td><td rowspan=1 colspan=1>Estimated LoD(TCID50/mL)</td></tr><tr><td rowspan=1 colspan=1>AdVC1</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>AdVC2</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>30,000</td></tr><tr><td rowspan=1 colspan=1>AdVE4</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>300</td></tr><tr><td rowspan=1 colspan=1>AdVC6</td><td rowspan=1 colspan=1>100</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100.0%</td><td rowspan=1 colspan=1>3,000,000</td></tr></table>

The LoD for all other analytes detected by the FilmArray RP was found to be equivalent between the original and modified panels by testing replicate samples at LoD level as well as bracketing levels.

# e. Analytical reactivity:

The analytical reactivity of the modified FilmArray RP system was evaluated using 33 Adenovirus isolates representing 6 species (A-F) and 22 serotypes. These included both respiratory and non-respiratory adenovirus isolates. The modified FilmArray RP is designed to detect all respiratory species/serotypes of Adenovirus (B, C, and E). Detection of non-respiratory species (A, D, F and G) will vary.

Samples were tested with both the original and modified panels at the Adenovirus LoD established for the modified panel $( 1 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ). When Adenovirus isolate was not detected by the modified panel at LoD, testing was repeated at $1 0 \mathrm { x }$ LoD (1,000 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ . Testing above 10x LoD was not performed.

AdVB55 and AdVC57 are the only respiratory serotypes that were not evaluated for inclusivity as the sponsor could not obtain these strains. Available sequence information for these serotypes indicates a perfect match to FilmArray Adenovirus assay primers and efficient detection at 1x LoD is predicted.

Inclusivity Results for Adenovirus Respiratory Species/Serotypes Tested with the Original and Modified FilmArray RP   

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Test Level</td><td colspan="1" rowspan="1">XLoD</td><td colspan="1" rowspan="1">OriginalFilmArray RP</td><td colspan="1" rowspan="1">ModifiedFilmArray RP</td></tr><tr><td colspan="1" rowspan="6">B</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">Zeptometrix#0810062CF</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">7a</td><td colspan="1" rowspan="1">Zeptometrix#0810021CF</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">7d2</td><td colspan="1" rowspan="1">Iowa/2001</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">7h</td><td colspan="1" rowspan="1">Iowa/1999</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">Wisconsin/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">Missouri/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="1">16</td><td colspan="1" rowspan="1">ATCC VR-17</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">Missouri/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">34</td><td colspan="1" rowspan="1">UIRF-Texas/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">35</td><td colspan="1" rowspan="1">ATCC VR-718</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">50</td><td colspan="1" rowspan="1">ATCC VR-1602</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="12">C</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">Zeptometrix#0810050CF</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected a</td></tr><tr><td colspan="1" rowspan="5">2</td><td colspan="1" rowspan="1">New York/2001</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-846</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#266153</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#266161</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#266213</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">Zeptometrix#0810020CF</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="5">6</td><td colspan="1" rowspan="1">Colorado/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">ATCC VR-6</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#274924</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#274948</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">Clinical isolate#275032</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="2">E</td><td colspan="1" rowspan="1">4a</td><td colspan="1" rowspan="1">SouthCarolina/2004</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">4p3</td><td colspan="1" rowspan="1">New Jersey/2005</td><td colspan="1" rowspan="1">100TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr></table>

a The initial test of AdVC1 with the modified FilmArray RP pouch was invalid due to a control failure. Adenovirus was detected on the retest.

Inclusivity Results for Non-Respiratory Adenovirus Species/Serotypes Tested with the Original and Modified FilmArray RP   

<table><tr><td colspan="1" rowspan="1">Species</td><td colspan="1" rowspan="1">Type</td><td colspan="1" rowspan="1">Isolate</td><td colspan="1" rowspan="1">Test Level</td><td colspan="1" rowspan="1">XLoD</td><td colspan="1" rowspan="1">OriginalFilmArray RP</td><td colspan="1" rowspan="1">ModifiedFilmArray RP</td></tr><tr><td colspan="1" rowspan="3">A</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">ATCC VR-863</td><td colspan="1" rowspan="1">1,000TCID50/mL</td><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">ATCC VR-19</td><td colspan="1" rowspan="1">1,000TCID50/mL</td><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">Zeptometrix#0810073CF</td><td colspan="1" rowspan="1">1,000TCID50/mL</td><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="3">D</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">Zeptometrix#0810069CF</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">20</td><td colspan="1" rowspan="1">Zeptometrix#0810115CF</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="1">37</td><td colspan="1" rowspan="1">Zeptometrix#0810119CF</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected a</td><td colspan="1" rowspan="1">Detected</td></tr><tr><td colspan="1" rowspan="2">F</td><td colspan="1" rowspan="1">40</td><td colspan="1" rowspan="1">Zeptometrix#0810084CF</td><td colspan="1" rowspan="1">1,000TCID50/mL</td><td colspan="1" rowspan="1">10x</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td></tr><tr><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">Indiana/2004</td><td colspan="1" rowspan="1">100 TCID50/mL</td><td colspan="1" rowspan="1">1x</td><td colspan="1" rowspan="1">Detected</td><td colspan="1" rowspan="1">Detected b</td></tr></table>

a The initial test of AdVD37 at 1x LoD in the original FilmArray RP pouch was negative. Adenovirus was detected on the retest. b The initial test of AdVF41 at 1x LoD in the modified FilmArray RP pouch was negative. Adenovirus was detected on the retest.

# f. Analytical specificity:

An analytical exclusivity study with modified FilmArray RP was carried out to assess the potential for false positive results due to cross-reactivity between RP assays and other RP or non-RP organisms. RP organisms and non-RP organisms were tested at high concentrations and the non-RP organism exclusivity panel consisted of organisms that are related to, but distinct from, RP target organisms, or that could be present in specimens collected from the intended test population. Although not observed in this study, the Coronavirus OC43 assay may cross-react with certain strains of Coronavirus HKU1 when present in the sample at high concentrations. A limitation pertaining to this potential for false positive Coronavirus OC43 results due to cross-reactivity is included in the product labeling.

The following table lists all of the RP organisms tested at high concentration.   

<table><tr><td colspan="1" rowspan="1">Analyte</td><td colspan="1" rowspan="1">Type / Strain / ID</td><td colspan="1" rowspan="1">Test Concentration</td><td colspan="1" rowspan="1">Multiple ofLoD Tested</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">AdVC1</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">1,000x</td></tr><tr><td colspan="1" rowspan="4">Coronavirus</td><td colspan="1" rowspan="1">229EATCC VR-740</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,418x</td></tr><tr><td colspan="1" rowspan="1">HKU1Clinical specimen</td><td colspan="1" rowspan="1">1.34E+08 RNA copies/mL</td><td colspan="1" rowspan="1">70 x</td></tr><tr><td colspan="1" rowspan="1">NL63NR-470</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">1,134x</td></tr><tr><td colspan="1" rowspan="1">OC43ATCC VR-759</td><td colspan="1" rowspan="1">7.30E+04 TCID50/mL</td><td colspan="1" rowspan="1">122x</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">Type A1 - hMPV-16IA10-2003</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">4,085x</td></tr><tr><td colspan="1" rowspan="2">Human Rhinovirus /Enterovirus</td><td colspan="1" rowspan="1">Echovirus 6</td><td colspan="1" rowspan="1">3.40E+06 TCID50/mL</td><td colspan="1" rowspan="1">113x</td></tr><tr><td colspan="1" rowspan="1">Rhinovirus 1A</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL</td><td colspan="1" rowspan="1">5,670x</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1N1</td><td colspan="1" rowspan="1">A/Brisbane/59/07</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">500x</td></tr><tr><td colspan="1" rowspan="1">Influenza AH1-2009</td><td colspan="1" rowspan="1">A/SwineNY/03/2009</td><td colspan="1" rowspan="1">8.40E+05 TCID50/mL</td><td colspan="1" rowspan="1">840x</td></tr><tr><td colspan="1" rowspan="1">Influenza AH3N2</td><td colspan="1" rowspan="1">A/Wisconsin/67/2005</td><td colspan="1" rowspan="1">8.17E+03 TCID50/mL</td><td colspan="1" rowspan="1">1634x</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">B/FL/04/06</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td><td colspan="1" rowspan="1">278x</td></tr><tr><td colspan="1" rowspan="4">Parainfluenza Virus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">28x</td></tr><tr><td colspan="1" rowspan="1">Type 2</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td><td colspan="1" rowspan="1">1,670x</td></tr><tr><td colspan="1" rowspan="1">Type 3</td><td colspan="1" rowspan="1">1.00E+05 TCID50/mL</td><td colspan="1" rowspan="1">10,000x</td></tr><tr><td colspan="1" rowspan="1">Type 4a</td><td colspan="1" rowspan="1">5.67E+03 TCID50/mL a</td><td colspan="1" rowspan="1">1.13x</td></tr><tr><td colspan="1" rowspan="1">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">Type A</td><td colspan="1" rowspan="1">1.39E+04 TCID50/mL</td><td colspan="1" rowspan="1">6,950x</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">A639</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td><td colspan="1" rowspan="1">250x</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">TW183</td><td colspan="1" rowspan="1">2.42E+05 copies/mL</td><td colspan="1" rowspan="1">81x</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">M129</td><td colspan="1" rowspan="1">1.88E+05 TCID50/mL</td><td colspan="1" rowspan="1">6,267x</td></tr></table>

a Highest test concentration possible based on the concentration of virus in the stock culture fluid.

The non-RP target exclusivity panel consisted of 26 bacteria, 6 viruses, and 1 fungus (Candida albicans). These organisms were selected based on their relatedness to FilmArray RP organisms, clinical relevance (cause respiratory symptoms or represent nasopharyngeal flora), or high prevalence within the population (e.g. Herpes Simplex Virus). Negative sample matrix was spiked with bacteria or fungi at a concentration of $1 0 ^ { 6 } \mathrm { C F U / m L }$ and viruses at a concentration between $1 0 ^ { 4 } - 1 0 ^ { 5 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ , or the highest concentration possible. The modified FilmArray RP system did not cross-react with the exclusivity panel organisms.

<table><tr><td colspan="1" rowspan="1">Virus</td><td colspan="1" rowspan="1">Strain / Isolate</td><td colspan="1" rowspan="1">Test Concentration</td></tr><tr><td colspan="1" rowspan="1">Bocavirus</td><td colspan="1" rowspan="1">Clinical Specimen</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Cytomegalovirus (CMV)</td><td colspan="1" rowspan="1">AD-169 (VR-538)</td><td colspan="1" rowspan="1">1.67E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Epstein-Barr Virus (EBV)</td><td colspan="1" rowspan="1">B95-8</td><td colspan="1" rowspan="1">1.00E+05 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Herpes Simplex Virus</td><td colspan="1" rowspan="1">Type 1</td><td colspan="1" rowspan="1">1:40 dilution of stock</td></tr><tr><td colspan="1" rowspan="1">Measles Virus</td><td colspan="1" rowspan="1">Edmonston</td><td colspan="1" rowspan="1">1.00E+05 PFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mumps</td><td colspan="1" rowspan="1">Zeptometrix # 0810079CF</td><td colspan="1" rowspan="1">5.03E+04 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Yeast</td><td colspan="1" rowspan="1">Strain / Isolate</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">Zeptometrix #0801504</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bacterium</td><td colspan="1" rowspan="1">Strain / Isolate</td><td colspan="1" rowspan="1">cYanmaGellowb</td></tr><tr><td colspan="1" rowspan="1">Bordetella bronchiseptica</td><td colspan="1" rowspan="1">clinical isolate</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella holmesii</td><td colspan="1" rowspan="1">F061</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Bordetella parapertussis</td><td colspan="1" rowspan="1">A747</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Chlamydia trachomatis</td><td colspan="1" rowspan="1">D-UW3</td><td colspan="1" rowspan="1">1.00E+06 IFU/mL</td></tr><tr><td colspan="1" rowspan="1">Corynebacterium diptheriae</td><td colspan="1" rowspan="1">ATCC14779</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Escherichia coli</td><td colspan="1" rowspan="1">0157:H7</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Haemophilus influenzae</td><td colspan="1" rowspan="1">MinnA</td><td colspan="1" rowspan="1">8.67E+04 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus acidophilus</td><td colspan="1" rowspan="1">Type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Lactobacillus plantarum</td><td colspan="1" rowspan="1">17-5</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella longbeacheae</td><td colspan="1" rowspan="1">Long Beach 4</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella micdadei</td><td colspan="1" rowspan="1">Tatlock</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Legionella pneumophilia</td><td colspan="1" rowspan="1">Philadelphia</td><td colspan="1" rowspan="1">1.00E+06 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Moraxella catarrhalis</td><td colspan="1" rowspan="1">Ne 11 (type strain)</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycobacterium tuberculosis</td><td colspan="1" rowspan="1">H37Ra-1</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma hominis</td><td colspan="1" rowspan="1">ATCC 23114</td><td colspan="1" rowspan="1">1.00E+06 CCU/mL</td></tr><tr><td colspan="1" rowspan="1">Mycoplasma genitalium</td><td colspan="1" rowspan="1">ATCC 33530</td><td colspan="1" rowspan="1">1.00E+06 copies/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria elongata</td><td colspan="1" rowspan="1">type strain</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria gonorrhoeae</td><td colspan="1" rowspan="1">ATCC 700825</td><td colspan="1" rowspan="1">1.05E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Neisseria meningitidis</td><td colspan="1" rowspan="1">M1027 (type strain)</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">Zeptometrix #0801519</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">COL</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">RP62A</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pneumoniae</td><td colspan="1" rowspan="1">type 59</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus pyogenes</td><td colspan="1" rowspan="1">Zeptometrix #0801512</td><td colspan="1" rowspan="1">1.00E+06 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Streptococcus salivarius</td><td colspan="1" rowspan="1">ATCC 13419</td><td colspan="1" rowspan="1">8.43E+05 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">Ureaplasma urealyticum</td><td colspan="1" rowspan="1">ATCC 27618</td><td colspan="1" rowspan="1">5.23E+05 copies/mL</td></tr></table>

# g. Competitive interference:

Interference testing was performed by preparing simulated NPS samples with a combination of two respiratory viruses, with one being Adenovirus. The competing viruses were selected based on the dominant Adenovirus co-infections documented in the FilmArray RP Clinical Evaluation. In total, five different virus combinations (coinfections) were evaluated with each virus tested at a low level (LoD for organism) and at a high or competing level $( \sim 5 , 0 0 0 - 1 0 0 , 0 0 0 \mathrm { T C I D } _ { 5 0 } / \mathrm { m L } )$ ). There was no interference with competing organisms in a sample.

<table><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">LoDVirus</td><td colspan="1" rowspan="1">CompetingVirus</td><td colspan="1" rowspan="1">Original FilmArray RP Result</td><td colspan="1" rowspan="1">Modified FilmArray RP Result</td></tr><tr><td colspan="1" rowspan="1">la</td><td colspan="1" rowspan="1">AdVC1</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">AdenovirusHuman Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">AdenovirusHuman Rhinovirus/Enterovirus</td></tr><tr><td colspan="1" rowspan="1">1b</td><td colspan="1" rowspan="1">HRV</td><td colspan="1" rowspan="1">AdVC1</td><td colspan="1" rowspan="1">AdenovirusHuman Rhinovirus/Enterovirus</td><td colspan="1" rowspan="1">AdenovirusHuman Rhinovirus/Enterovirus</td></tr><tr><td colspan="1" rowspan="1">2a</td><td colspan="1" rowspan="1">AdVC5</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">-Human Metapneumovirus</td><td colspan="1" rowspan="1">AdenovirusHuman Metapneumovirus</td></tr><tr><td colspan="1" rowspan="1">2b</td><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">AdVC5</td><td colspan="1" rowspan="1">AdenovirusHuman Metapneumovirus</td><td colspan="1" rowspan="1">AdenovirusHuman Metapneumovirus</td></tr><tr><td colspan="1" rowspan="1">3a</td><td colspan="1" rowspan="1">AdVC6</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">-Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">AdenovirusRespiratory Syncytial Virus</td></tr><tr><td colspan="1" rowspan="1">3b</td><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">AdVC6</td><td colspan="1" rowspan="1">-Respiratory Syncytial Virus</td><td colspan="1" rowspan="1">AdenovirusRespiratory Syncytial Virus</td></tr><tr><td colspan="1" rowspan="1">4a</td><td colspan="1" rowspan="1">AdVB7h</td><td colspan="1" rowspan="1">AdVE4p3</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">4b</td><td colspan="1" rowspan="1">AdVE4p3</td><td colspan="1" rowspan="1">AdVB7h</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">5a</td><td colspan="1" rowspan="1">AdVC2</td><td colspan="1" rowspan="1">AdVB21</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Adenovirus</td></tr><tr><td colspan="1" rowspan="1">5b</td><td colspan="1" rowspan="1">AdVB21</td><td colspan="1" rowspan="1">AdVC2</td><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">Adenovirus</td></tr></table>

2. Comparison studies:

# Clinical Comparison

To demonstrate performance of the modified FilmArray RP, a comparison study was performed by testing 222 de-identified archived nasopharyngeal swab specimens collected between 2008 and 2011 throughout the U.S. (at least 8 geographically distinct locations) and Scotland (at least 1 location) with both the original FilmArray RP and modified FilmArray RP. A total of 26 Adenovirus specimens were detected by the modified FilmArray RP, of these only 15 were detected by original FilmArray RP. For the other 19 organisms on the panel, performance appeared to be equivalent between the original and modified FilmArray RP versions.

Performance Comparison of the Modified FilmArray RP to the Original Using Archived Specimens   

<table><tr><td colspan="1" rowspan="2">Analyte</td><td colspan="4" rowspan="1">Positive Agreement</td><td colspan="4" rowspan="1">Negative Agreement</td></tr><tr><td colspan="1" rowspan="1">orig +mod+</td><td colspan="1" rowspan="1">orig +mod -</td><td colspan="1" rowspan="1">PPA</td><td colspan="1" rowspan="1">95% CI</td><td colspan="1" rowspan="1">orig -mod -</td><td colspan="1" rowspan="1">orig -mod+</td><td colspan="1" rowspan="1">NPA</td><td colspan="1" rowspan="1">95% CI</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(15/15)</td><td colspan="1" rowspan="1">78.2 - 100%</td><td colspan="1" rowspan="1">196</td><td colspan="1" rowspan="1">11 a</td><td colspan="1" rowspan="1">94.7%(196/207)</td><td colspan="1" rowspan="1">90.7 97.3%</td></tr><tr><td colspan="1" rowspan="1">CoV 229E</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">54.1 - 100%</td><td colspan="1" rowspan="1">216</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(216/216)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">CoV HKU1</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">1000%(8/8)</td><td colspan="1" rowspan="1">63.1 - 100%</td><td colspan="1" rowspan="1">214</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(214/214)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">CoV NL63</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">1 b</td><td colspan="1" rowspan="1">93.8%(15/16)</td><td colspan="1" rowspan="1">69.8 99.8%</td><td colspan="1" rowspan="1">206</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(206/206)</td><td colspan="1" rowspan="1">98.2 - 100%</td></tr><tr><td colspan="1" rowspan="1">CoV OC43</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(13/13)</td><td colspan="1" rowspan="1">75.3 - 100%</td><td colspan="1" rowspan="1">208</td><td colspan="1" rowspan="1">1c</td><td colspan="1" rowspan="1">99.5%(208/209)</td><td colspan="1" rowspan="1">97.4- 100%</td></tr><tr><td colspan="1" rowspan="1">hMPV</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">69.2 - 100%</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">3d</td><td colspan="1" rowspan="1">98.6%(209/212)</td><td colspan="1" rowspan="1">95.9-99.7%</td></tr><tr><td colspan="1" rowspan="1">HRV/EV</td><td colspan="1" rowspan="1">57</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">93.4%(57/61)</td><td colspan="1" rowspan="1">84.0 -98.2%</td><td colspan="1" rowspan="1">158</td><td colspan="1" rowspan="1">3c</td><td colspan="1" rowspan="1">98.1%(158/161)</td><td colspan="1" rowspan="1">94.6-99.6%</td></tr><tr><td colspan="1" rowspan="1">Flu A</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(36/36)</td><td colspan="1" rowspan="1">90.3 - 100%</td><td colspan="1" rowspan="1">184</td><td colspan="1" rowspan="1">1f</td><td colspan="1" rowspan="1">99.5%(184/185)</td><td colspan="1" rowspan="1">97.0 - 100%</td></tr><tr><td colspan="1" rowspan="1">Flu A H1</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(9/9)</td><td colspan="1" rowspan="1">66.4 - 100%</td><td colspan="1" rowspan="1">213</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(213/213)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">Flu A 2009H1</td><td colspan="1" rowspan="1">15</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(15/15)</td><td colspan="1" rowspan="1">78.2 - 100%</td><td colspan="1" rowspan="1">205</td><td colspan="1" rowspan="1">1f</td><td colspan="1" rowspan="1">99.5%(205/206)</td><td colspan="1" rowspan="1">97.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">Flu A H3</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(13/13)</td><td colspan="1" rowspan="1">75.3 - 100%</td><td colspan="1" rowspan="1">209</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(209/209)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">Flu B</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(10/10)</td><td colspan="1" rowspan="1">69.2 - 100%</td><td colspan="1" rowspan="1">212</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(212/212)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">RSV</td><td colspan="1" rowspan="1">21</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(21/21)</td><td colspan="1" rowspan="1">83.9 - 100%</td><td colspan="1" rowspan="1">201</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(201/201)</td><td colspan="1" rowspan="1">98.2 - 100%</td></tr><tr><td colspan="1" rowspan="1">PIV1</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(11/11)</td><td colspan="1" rowspan="1">71.5 - 100%</td><td colspan="1" rowspan="1">211</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(211/211)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">PIV2</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(8/8)</td><td colspan="1" rowspan="1">63.1 - 100%</td><td colspan="1" rowspan="1">214</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(214/214)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">PIV3</td><td colspan="1" rowspan="1">18</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(18/18)</td><td colspan="1" rowspan="1">81.5- 100%</td><td colspan="1" rowspan="1">204</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(204/204)</td><td colspan="1" rowspan="1">98.2 - 100%</td></tr><tr><td colspan="1" rowspan="1">PIV4</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(6/6)</td><td colspan="1" rowspan="1">54.1 - 100%</td><td colspan="1" rowspan="1">214</td><td colspan="1" rowspan="1">2 g</td><td colspan="1" rowspan="1">99.1%(214/216)</td><td colspan="1" rowspan="1">96.7-99.9%</td></tr><tr><td colspan="1" rowspan="1">B. pertussis</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">1h</td><td colspan="1" rowspan="1">96.2%(25/26)</td><td colspan="1" rowspan="1">80.4-99.9%</td><td colspan="1" rowspan="1">196</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(196/196)</td><td colspan="1" rowspan="1">98.1 - 100%</td></tr><tr><td colspan="1" rowspan="1">C.pneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(1/1)</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">221</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(221/221)</td><td colspan="1" rowspan="1">98.3 - 100%</td></tr><tr><td colspan="1" rowspan="1">M.pneumoniae</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">n/a</td><td colspan="1" rowspan="1">222</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">100%(222/222)</td><td colspan="1" rowspan="1">98.4 - 100%</td></tr></table>

orig $=$ original FilmArray RP, mod $=$ modified FilmArray RP, PPA $=$ positive percent agreement, $\mathrm { N P S = }$ negative percent agreement, $\mathrm { C I } =$ confidence interval

a 10/11 of the additional AdV detections by the modified FilmArray RP were confirmed to contain AdV by bi-directional sequence analysis; these AdV were identified by sequencing as AdVC2, AdVC5, AdVC6, AdVE4, and one undetermined serotype. One specimen could not be sequenced due to low analyte levels.

b A single specimen was found to be positive for $\mathrm { C o V N L } 6 3$ when tested with the original RP pouch but not the modified RP pouch. The specimen had previously been identified as positive for $B$ . pertussis and had not been tested for CoV NL63 by the source laboratory. This specimen previously tested using the original pouch was negative for CoV NL63. There was insufficient specimen for discrepancy investigation. Low viral load is suspected to have caused this spurious result.

c The Coronavirus OC43 discrepancy was due to cross-reactivity between the OC43 assay and HKU1 virus that was detected in the RP modified pouch and not in the original RP pouch.

$^ { \mathrm { d } } 2 / 3$ human Metapneumovirus discrepant specimens were confirmed by bi-directional sequence analysis. Low viral load is suspected to have prevented detection by sequencing for the other specimen.

$^ \mathrm { ~ e ~ } 0 / 7$ Human Rhinovirus/Enterovirus discrepant specimens were confirmed by bi-directional sequence analysis. Low viral load is suspected to have prevented detection by sequencing.

$^ \mathrm { f } _ { \mathrm { A } }$ single specimen containing Influenza A H1-2009 provided repeated equivocal results on the original RP pouch, but was detected by the modified pouch.

g Parainfluenza Virus 4 was confirmed in both discrepant specimens by bi-directional sequence analysis.

$B$ . pertussis was confirmed in the discrepant specimen by bi-directional sequence analysis.

To supplement the archived specimen data for low prevalence organisms and to provide additional Adenovirus performance data specifically for AdVC2 and AdVC6, a set of 44 contrived (spiked NPS) specimens (10 AdVC2, 10 AdVC6, $1 0 ~ C .$ . pneumoniae, and $1 4 M .$ . pneumoniae) was tested with both the modified FilmArray RP and the original FilmArray RP. The organism status of each contrived specimen was blinded to the users analyzing the specimens. Modified FilmArray RP detected all 20 Adenovirus-spiked specimens, while original FilmArray RP detected none of the Adenovirus-spiked specimens.

Modified FilmArray RP also detected another Adenovirus in the background of a specimen spiked with $C$ . pneumonia that was later confirmed with bidirectional sequencing. Performance appeared to be equivalent between modified FilmArray RP and original FilmArray RP for $C .$ . pneumoniae and $M .$ pneumoniae.

Comparison data of Modified FilmArray RP to Original for 44 contrived specimens   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=4>Positive Agreement</td><td rowspan=1 colspan=4>Negative Agreement</td></tr><tr><td rowspan=1 colspan=1>orig +mod +</td><td rowspan=1 colspan=1>orig +mod -</td><td rowspan=1 colspan=1>PPA</td><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>orig -mod -</td><td rowspan=1 colspan=1>orig -mod +</td><td rowspan=1 colspan=1>NPA</td><td rowspan=1 colspan=1>95% CI</td></tr><tr><td rowspan=1 colspan=1>Adenovirus(AdVC2 andAdVC6)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>23</td><td rowspan=1 colspan=1>21a</td><td rowspan=1 colspan=1>52.3%(23/44)</td><td rowspan=1 colspan=1>36.7 67.5%</td></tr><tr><td rowspan=1 colspan=1>C.pneumoniae</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>88.9%(8/9)</td><td rowspan=1 colspan=1>51.8-99.7%</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(35/35)</td><td rowspan=1 colspan=1>90.0 - 100%</td></tr><tr><td rowspan=1 colspan=1>M.pneumoniae</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(14/14)</td><td rowspan=1 colspan=1>76.8 - 100%</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>100%(30/30)</td><td rowspan=1 colspan=1>88.4 - 100%</td></tr></table>

orig $=$ original FilmArray RP, mod $=$ modified FilmArray RP, PPA $=$ positive percent agreement, $\mathrm { N P S } =$ negative percent agreement, $\mathrm { C I } =$ confidence interval

a In addition to ten specimens spiked with AdVC6 and ten specimens spiked with AdVC2, modified FilmArray RP also detected an Adenovirus in the background of one specimen that had been spiked with $C$ . pneumoniae. This detection was confirmed by bi-directional sequence analysis to be AdVC2.

The Adenoviruses detected in archived specimens were categorized into serotype groups using bi-directional sequence analysis. Combining the archived and contrived specimen comparison data demonstrates improved detection of AdVC2, AdVC5, AdVC6, and AdVE4 by modified FilmArray RP as compared to the original FilmArray RP.

Adenovirus Serotype Detections by the Modified and the Original FilmArray RP in Archived and Contrived Specimens   

<table><tr><td rowspan=1 colspan=1>Adenovirus(Serotyped by PCR)</td><td rowspan=1 colspan=1>Number of Adenovirus-positive Specimens(as Detected by ModifiedFilmArray RP)</td><td rowspan=1 colspan=1>Detections by OriginalFilmArray RP</td></tr><tr><td rowspan=1 colspan=1>AdVC1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>8/8 (100%)</td></tr><tr><td rowspan=1 colspan=1>AdVC2</td><td rowspan=1 colspan=1>15a</td><td rowspan=1 colspan=1>2/15 (13%)</td></tr><tr><td rowspan=1 colspan=1>AdVC5</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1/2 (50%)</td></tr><tr><td rowspan=1 colspan=1>AdVC6</td><td rowspan=1 colspan=1>16 a</td><td rowspan=1 colspan=1>1/16 (6%)</td></tr><tr><td rowspan=1 colspan=1>AdVB3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 (100%)</td></tr><tr><td rowspan=1 colspan=1>AdVE4</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>2/3 (67%)</td></tr><tr><td rowspan=1 colspan=1>AdV serotype unknown</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0/2 (0%)</td></tr></table>

a Ten AdVC2 specimens and ten AdVC6 specimens were contrived by spiking these viruses into NPS specimens. One AdVC2 was also detected and sequence confirmed in the background of a specimen spiked with C. pneumoniae.

3. Clinical studies:

Clinical performance characteristics of FilmArray RP were established earlier; please refer to the decision summaries of previously cleared submissions k103175, k110764, and k120267 for detailed information.

4. Clinical cut-off: Not applicable

5. Expected values/Reference range:

Expected Value (as Determined by the FilmArray RP) Summary by Site for the First Phase Prospective Clinical Evaluation (December 2009 â€“ May 2010)   

<table><tr><td colspan="1" rowspan="2">Organism</td><td colspan="2" rowspan="1">Overall (n=853)</td><td colspan="2" rowspan="1">Site 1 (n=275)</td><td colspan="2" rowspan="1">Site 2 (n=333)</td><td colspan="2" rowspan="1">Site 3 (n=245)</td></tr><tr><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td></tr><tr><td colspan="1" rowspan="1">Adenovirus</td><td colspan="1" rowspan="1">38</td><td colspan="1" rowspan="1">4.5%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">22</td><td colspan="1" rowspan="1">9.0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H3</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza A H1-2009</td><td colspan="1" rowspan="1">11</td><td colspan="1" rowspan="1">1.3%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">3.6%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Influenza B</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus1</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus2</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus3</td><td colspan="1" rowspan="1">33</td><td colspan="1" rowspan="1">3.9%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">31</td><td colspan="1" rowspan="1">12.7%</td></tr><tr><td colspan="1" rowspan="1">Parainfluenza Virus4</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.2%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.6%</td></tr><tr><td colspan="1" rowspan="1">RespiratorySyncytial Virus</td><td colspan="1" rowspan="1">139</td><td colspan="1" rowspan="1">16.3%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">86</td><td colspan="1" rowspan="1">25.8%</td><td colspan="1" rowspan="1">49</td><td colspan="1" rowspan="1">20.0%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus 229E</td><td colspan="1" rowspan="1">14</td><td colspan="1" rowspan="1">1.6%</td><td colspan="1" rowspan="1">5</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">2.4%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus HKU1</td><td colspan="1" rowspan="1">25</td><td colspan="1" rowspan="1">2.9%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">3.3%</td><td colspan="1" rowspan="1">13</td><td colspan="1" rowspan="1">3.9%</td><td colspan="1" rowspan="1">3</td><td colspan="1" rowspan="1">1.2%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus NL63</td><td colspan="1" rowspan="1">23</td><td colspan="1" rowspan="1">2.7%</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">1.5%</td><td colspan="1" rowspan="1">9</td><td colspan="1" rowspan="1">2.7%</td><td colspan="1" rowspan="1">10</td><td colspan="1" rowspan="1">4.1%</td></tr><tr><td colspan="1" rowspan="1">Coronavirus OC43</td><td colspan="1" rowspan="1">8</td><td colspan="1" rowspan="1">0.9%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">6</td><td colspan="1" rowspan="1">1.8%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr><tr><td colspan="1" rowspan="1">HumanMetapneumovirus</td><td colspan="1" rowspan="1">94</td><td colspan="1" rowspan="1">11.0%</td><td colspan="1" rowspan="1">12</td><td colspan="1" rowspan="1">4.4%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">12.3%</td><td colspan="1" rowspan="1">41</td><td colspan="1" rowspan="1">16.7%</td></tr><tr><td colspan="1" rowspan="1">HumanRhinovirus/Entero</td><td colspan="1" rowspan="1">225</td><td colspan="1" rowspan="1">26.4%</td><td colspan="1" rowspan="1">36</td><td colspan="1" rowspan="1">13.1%</td><td colspan="1" rowspan="1">92</td><td colspan="1" rowspan="1">27.6%</td><td colspan="1" rowspan="1">97</td><td colspan="1" rowspan="1">39.6%</td></tr><tr><td colspan="1" rowspan="1">Bordetella pertussis</td><td colspan="1" rowspan="1">4</td><td colspan="1" rowspan="1">0.4%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.7%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.3%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Chlamydophilapneumoniae</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.1%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">1</td><td colspan="1" rowspan="1">0.4%</td></tr><tr><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">ExpectedValue</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">Expected Value</td><td colspan="1" rowspan="1">Number</td><td colspan="1" rowspan="1">Expected Value</td></tr><tr><td colspan="1" rowspan="1">Mycoplasmapneumoniae</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">2</td><td colspan="1" rowspan="1">0.2%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td><td colspan="1" rowspan="1">0</td><td colspan="1" rowspan="1">0%</td></tr></table>

Expected Value (as Determined by the FilmArray RP) Summary by Site for the Second Phase Prospective Clinical Evaluation (September 2010 â€“ January 2011)   

<table><tr><td rowspan=2 colspan=1>Organism</td><td rowspan=1 colspan=2>Overall (n=264)</td><td rowspan=1 colspan=2>Site 2 (n=180)</td><td rowspan=1 colspan=2>Site 3 (n=84)</td></tr><tr><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td><td rowspan=1 colspan=1>Number</td><td rowspan=1 colspan=1>ExpectedValue</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>5.7%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>6.0%</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3.4%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>2.2%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1.9%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5.7%</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.7%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>11.7%</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>6.1%</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>23.8%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.4%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>4.2%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>11.9%</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1.5%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4.6%</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Entero</td><td rowspan=1 colspan=1>125</td><td rowspan=1 colspan=1>47.3%</td><td rowspan=1 colspan=1>91</td><td rowspan=1 colspan=1>50.6%</td><td rowspan=1 colspan=1>34</td><td rowspan=1 colspan=1>39.1%</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.1%</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1.6%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0%</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0.5%</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1.2%</td></tr></table>

Expected Value (as Determined by FilmArray RP) Summary by Age Group for First Phase Prospective Clinical Evaluation (December 2009 â€“ May 2010)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Total(Expected Value)</td><td rowspan=1 colspan=1>â‰¤ 5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>â‰¥ 50 years</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>38 (4.5%)</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>11 (1.3%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Expected Value (as Determined by FilmArray RP) Summary by Age Group for Second Phase Prospective Clinical Evaluation (September 2010-January 2011)   

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Total(Expected Value)</td><td rowspan=1 colspan=1>â‰¤ 5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>â‰¥ 50 years</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>33 (3.9%)</td><td rowspan=1 colspan=1>31</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>8 (0.9%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>139 (16.3%)</td><td rowspan=1 colspan=1>127</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>14 (1.6%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>25 (2.9%)</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>23 (2.7%)</td><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>8 (0.9%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>2</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>94 (11.0%)</td><td rowspan=1 colspan=1>76</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Entero</td><td rowspan=1 colspan=1>225 (26.4%)</td><td rowspan=1 colspan=1>161</td><td rowspan=1 colspan=1>24</td><td rowspan=1 colspan=1>29</td><td rowspan=1 colspan=1>11</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>4 (0.4%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>1 (0.1%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2 (0.2%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr></table>

<table><tr><td rowspan=1 colspan=1>Organism</td><td rowspan=1 colspan=1>Total(Expected Value)</td><td rowspan=1 colspan=1>â‰¤ 5 years</td><td rowspan=1 colspan=1>6-21 years</td><td rowspan=1 colspan=1>22-49 years</td><td rowspan=1 colspan=1>â‰¥ 50years</td></tr><tr><td rowspan=1 colspan=1>Adenovirus</td><td rowspan=1 colspan=1>15 (5.7%)</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Influenza A</td><td rowspan=1 colspan=1>5 (1.9%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Influenza A H3</td><td rowspan=1 colspan=1>5 (1.9%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Influenza A H1-2009</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Influenza B</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 1</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 2</td><td rowspan=1 colspan=1>9 (3.4%)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 3</td><td rowspan=1 colspan=1>5 (1.9%)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Parainfluenza Virus 4</td><td rowspan=1 colspan=1>2 (0.7%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Respiratory Syncytial Virus</td><td rowspan=1 colspan=1>31 (11.7%)</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Coronavirus 229E</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Coronavirus HKU1</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Coronavirus NL63</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Coronavirus OC43</td><td rowspan=1 colspan=1>11 (4.2%)</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Human Metapneumovirus</td><td rowspan=1 colspan=1>4 (1.5%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Human Rhinovirus/Entero</td><td rowspan=1 colspan=1>125 (47.3%)</td><td rowspan=1 colspan=1>118</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Bordetella pertussis</td><td rowspan=1 colspan=1>3 (1.1%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Chlamydophila pneumoniae</td><td rowspan=1 colspan=1>0 (0%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Mycoplasma pneumoniae</td><td rowspan=1 colspan=1>2 (0.8%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Expected Value (Co-infections as Determined by FilmArray RP) Summary by Age Group for First Phase Prospective Clinical Evaluation (December 2009 â€“ May 2010)   

<table><tr><td rowspan=1 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Total(ExpectedValue)</td><td rowspan=1 colspan=1>â‰¤ 5 Years</td><td rowspan=1 colspan=1>6-21 Years</td><td rowspan=1 colspan=1>22-49 Years</td><td rowspan=1 colspan=1>â‰¥ 50 Years</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + RSV</td><td rowspan=1 colspan=1>21 (2.46%)</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + AdV</td><td rowspan=1 colspan=1>8 (0.94%)</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV3</td><td rowspan=1 colspan=1>8 (0.94%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + hMPV</td><td rowspan=1 colspan=1>7 (0.82%)</td><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>4 (0.47%)</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + CoV NL63</td><td rowspan=1 colspan=1>4 (0.47%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + hMPV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + HRV/EV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + RSV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + hMPV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + RSV</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV3</td><td rowspan=1 colspan=1>3 (0.35%)</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + HRV/EV + PIV3</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + RSV</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + CoV OC43</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV4</td><td rowspan=1 colspan=1>2 (0.23%)</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + hMPV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + PIV3</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + CoV NL63</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + RSV + CoV 229E</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + HRV/EV + B. pertussis</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>AdV + C. pneumoniae</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV 229E + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV 229E + CoV NL63 +HRV/EV+RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV 229E + HRV/EV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV HKU1 + HRV/EV + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>CoV NL63 + hMPV + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + B. pertussis</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV1</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>hMPV + PIV4</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr><tr><td rowspan=1 colspan=1>PIV4 + RSV</td><td rowspan=1 colspan=1>1 (0.12%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td></tr></table>

Expected Value (Co-infections as Determined by FilmArray RP) Summary by Age Group for Second Phase Prospective Clinical Evaluation (September 2010 â€“ January 2011)   

<table><tr><td rowspan=1 colspan=1>Co-Infection</td><td rowspan=1 colspan=1>Total(ExpectedValue</td><td rowspan=1 colspan=1>â‰¤ 5 Years</td><td rowspan=1 colspan=1>6-21 Years</td><td rowspan=1 colspan=1>22-49 Years</td><td rowspan=1 colspan=1>â‰¥ 50 Years</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + RSV</td><td rowspan=1 colspan=1>6 (2.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + AdV</td><td rowspan=1 colspan=1>6 (2.2%)</td><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + HRV/EV</td><td rowspan=1 colspan=1>5 (1.9%)</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV2</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV3</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + PIV4</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>hMPV + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Flu B + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>AdV + CoV OC43</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + hMPV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>CoV OC43 + RSV</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>HRV/EV + B. pertussis</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>PIV2 + M. pneumoniae</td><td rowspan=1 colspan=1>1 (0.4%)</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>n/a</td></tr></table>

# N. Instrument Name:

FilmArray Instrument

# O. System Descriptions:

1. Modes of Operation: See Section, I. Device Description

2. Software: FDA has reviewed applicantâ€™s instrument Hazard Analysis and software development processes for this instrument and for this assay. Yes __ X___ or No

3. Specimen Identification: User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not Applicable

5. Calibration:

Not applicable

6. Quality Control:

The FilmArray Respiratory Panel (RP) pouch contains two internal control assays:

1. The RNA Process Control targets an mRNA of the yeast, Schizosaccharomyces pombe. During FilmArray RP pouch manufacture, whole yeast is freeze-dried into the sample injection port of each pouch. When the test specimen is loaded into the pouch, S. pombe is rehydrated and enters the pouch with the specimen. The yeast nucleic acid is extracted, purified and tested simultaneously with nucleic acids from the patient specimen. A positive result for the processing control indicates that all steps in the process (nucleic acid extraction, reverse transcription, PCR, melt, detection, and analysis) are functioning properly. 2. The second stage PCR (PCR2) control assay detects a synthetic DNA template that is dried into triplicate wells of the array along with the corresponding primers. A positive result indicates that PCR2 was successful.

The RNA Process Control and the PCR2 Control assays are used to Pass or Fail each FilmArray RP pouch run. This combination of control assays monitors each of the critical mechanical and chemical processes that occur in a pouch run, while limiting the possibility of random control assay failures that could contribute to unnecessary pouch failures.

Good laboratory practice recommends running external positive and negative controls regularly. Use viral transport medium as the external Negative Control, and previously characterized positive samples or negative sample spiked with well characterized target organisms as external Positive Controls. External controls should be used in accordance with local, state, federal accrediting organizations, as applicable.

P. O therSupportiveInstrum entPerform anceCharacteristicsDataNotCovered in the â€œPerformance Characteristicsâ€ Section above:

Not Applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.