# 510(k) SUBSTANTIAL EQUIVALENCE DETERMINATION DECISION SUMMARY

A. 510(k) Number: k123274   
B. Purpose for Submission: New device

C. Measurand: Salmonella, Shigella, Campylobacter (C. jejuni and C. coli only, undifferentiated) nucleic acids target sequences and Shiga Toxin 1 $( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes.

# D. Type of Test:

A multiplexed real time PCR in vitro diagnostic test intended for the qualitative detection and differentiation of Salmonella, Shigella, and Campylobacter (C. jejuni and $C .$ . coli only, undifferentiated) nucleic acids and Shiga Toxin 1 $( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes. Shiga toxin producing E. coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2. Nucleic acids are isolated and purified from preserved stool specimens obtained from symptomatic patients exhibiting signs and symptoms of gastroenteritis.

E. Applicant: Gen-Probe Prodesse, Inc.

F. Proprietary and Established Names: Proprietary Name: ProGastro SSCS Assay

Common Name: Nucleic acid amplification assay for Salmonella spp., Shigella spp., Campylobacter jejuni and Campylobacter coli, and Shiga Toxin 1 and 2 genes.

G. Regulatory Information:

<table><tr><td>Product Code</td><td>Classification</td><td>Regulation Section</td><td>Panel</td></tr><tr><td>PCH</td><td>Class II</td><td>21 CFR 866.3990 Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assay</td><td>Microbiology (83)</td></tr><tr><td>PCI</td><td>Class II</td><td>21 CFR 866.3990 Gastrointestinal Microorganism Multiplex Nucleic Acid-Based Assay</td><td>Microbiology (83)</td></tr><tr><td>001</td><td>Class II</td><td>21 CFR 862.2570 Instrumentation for Clinical Multiplex Test Systems</td><td>Clinical Chemistry (75)</td></tr></table>

# H. Intended Use:

1. Intended use:

The Prodesse® ProGastro SSCS Assay is a multiplex real time PCR in vitro diagnostic test for the qualitative detection and differentiation of Salmonella, Shigella, and Campylobacter (C. jejuni and C. coli only, undifferentiated) nucleic acids and Shiga Toxin 1 $( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes. Shiga toxin producing $E .$ . coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2. Nucleic acids are isolated and purified from preserved stool specimens obtained from symptomatic patients exhibiting signs and symptoms of gastroenteritis. This test is intended for use, in conjunction with clinical presentation and epidemiological risk factors, as an aid in the differential diagnosis of Salmonella, Shigella, Campylobacter jejuni/Campylobacter coli, and STEC infections in humans.

The results of this test should not be used as the sole basis for diagnosis, treatment, or other patient management decisions. Positive results do not rule out co-infection with other organisms that are not detected by this test, and may not be the sole or definitive cause of patient illness. Negative ProGastro SSCS Assay results in the setting of clinical illness compatible with gastroenteritis may be due to infection by pathogens that are not detected by this test or non-infectious causes such as ulcerative colitis, irritable bowel syndrome, or Crohn’s disease.

2. Indication for use: Same as Intended Use

3. Special conditions for use statement(s): For prescription use only

4. Special instrument requirements: The bioMerieux NucliSENS easyMAG system is used for nucleic acid isolation from stool samples.

The Cepheid SmartCycler II Real Time instrument with Dx software version 1.7b or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$ is used to perform PCR amplification and detection of nucleic acid.

Several other Prodesse Assays have been cleared for use with the above instrumentation systems: Prodesse ProFlu+ (k110968), Pro hMPV $^ +$ (k082688), ProGastro Cd (k090239), ProParaflu $^ +$ (k091053), ProFAST $^ +$ (k101855) and ProAdeno $^ +$ (k102952).

# I. Device Description:

The ProGastro SSCS Assay enables detection and differentiation of Salmonella, Shigella, Campylobacter (C. jejuni and $C .$ . coli only, undifferentiated) and an Internal Control in the SSC Mix, and Shiga Toxin 1 $( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes and an Internal Control in the STEC Mix. Shiga toxin producing $E$ . coli (STEC) typically harbor one or both genes that encode for Shiga Toxins 1 and 2.

An overview of the procedure is as follows:

1. Collect raw stool specimens from symptomatic patients placed into Cary Blair (CB) or ParaPak C&S Transport Medium (C&S).   
2. Dilute stool preserved in CB or C&S 1:10 in CB or C&S Medium. Add an Internal Control to every sample to monitor for inhibitors present in the specimens.   
3. Perform isolation and purification of nucleic acids using a NucliSENS easyMAG System (bioMérieux) and the Automated Magnetic Extraction Reagents (bioMérieux).   
4. Add purified nucleic acids to the SSC Mix included in the ProGastro SSCS Kit. The SSC Mix contains target-specific oligonucleotide primers and oligonucleotide probes for detection of Salmonella, Shigella, and Campylobacter (C. jejuni and C. coli only). The primers are complementary to highly conserved regions of genetic sequences for these organisms. The probes are dual-labeled with a reporter dye and a quencher (see table below).   
5. Add purified nucleic acids to the STEC Mix included in the ProGastro SSCS Kit. The STEC Mix contains target-specific oligonucleotide primers and oligonucleotide probes for detection of the Shiga Toxins 1 and 2 genes $\boldsymbol { s t x l }$ and stx2). The primers are complementary to highly conserved regions of genetic sequences for these organisms. The probes are dual-labeled with a reporter dye and a quencher (see table below).   
6. Perform amplification of DNA in a Cepheid SmartCycler II instrument. In this process, the probe anneals specifically to the template followed by primer extension and amplification. The ProGastro SSCS Assay is based on Taqman reagent chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification products present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

<table><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=2>Gene Targeted</td><td rowspan=1 colspan=1>ProbeFluorophore</td><td rowspan=1 colspan=1>AbsorbancePeak</td><td rowspan=1 colspan=1>EmissionPeak</td><td rowspan=1 colspan=1>InstrumentChannel</td></tr><tr><td rowspan=2 colspan=1>SSC Mix</td><td rowspan=2 colspan=1>Campylobacter(C. jejuni and C. colionly</td><td rowspan=1 colspan=1>C. jejuni</td><td rowspan=1 colspan=1>C. coli</td><td rowspan=2 colspan=1>FAM</td><td rowspan=2 colspan=1>495 nm</td><td rowspan=2 colspan=1>520 nm</td><td rowspan=2 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>glyA</td><td rowspan=1 colspan=1>cadF</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Salmonella spp.</td><td rowspan=1 colspan=2>orgC</td><td rowspan=1 colspan=1>CAL FluorOrange 560</td><td rowspan=1 colspan=1>538 nm</td><td rowspan=1 colspan=1>559 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Shigella spp.</td><td rowspan=1 colspan=2>ipaH</td><td rowspan=1 colspan=1>CAL FluorRed 610</td><td rowspan=1 colspan=1>590 nm</td><td rowspan=1 colspan=1>610 nm</td><td rowspan=1 colspan=1>Texas Red</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Shiga Toxin 1</td><td rowspan=1 colspan=2>stx1</td><td rowspan=1 colspan=1>CAL FluorOrange 560</td><td rowspan=1 colspan=1>538 nm</td><td rowspan=1 colspan=1>559 nm</td><td rowspan=1 colspan=1>TET</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Shiga Toxin 2</td><td rowspan=1 colspan=2>stx2</td><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>495 nm</td><td rowspan=1 colspan=1>520 nm</td><td rowspan=1 colspan=1>FAM</td></tr><tr><td rowspan=1 colspan=1>SSC Mix andSTEC Mix</td><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=2>NA</td><td rowspan=1 colspan=1>Quasar 670</td><td rowspan=1 colspan=1>647 nm</td><td rowspan=1 colspan=1>667 nm</td><td rowspan=1 colspan=1>Cy5</td></tr></table>

# Materials Provided

ProGastro SSCS Assay Kit (100 Reactions) (Cat. # 303278)   

<table><tr><td rowspan=1 colspan=1>Reagents</td><td rowspan=1 colspan=1>Description</td><td rowspan=1 colspan=1>Quantity/Tube</td><td rowspan=1 colspan=1>CapColor</td><td rowspan=1 colspan=1>Cat. #</td><td rowspan=1 colspan=1>Reactions/ Tube</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>•   Taq DNA polymeraseoligonucleotide primersoligonucleotide probesBuffer containing dNTPsMgCl2 and stabilizers</td><td rowspan=1 colspan=1>1,100 μL</td><td rowspan=1 colspan=1>Brown</td><td rowspan=1 colspan=1>403325</td><td rowspan=1 colspan=1>55(2 tubes provided)</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Taq DNA polymerase•  oligonucleotide primersoligonucleotide probesBuffer containing dNTPs•   MgCl2 and stabilizers</td><td rowspan=1 colspan=1>1,000 μL</td><td rowspan=1 colspan=1>Blue</td><td rowspan=1 colspan=1>403323</td><td rowspan=1 colspan=1>50(2 tubes provided)</td></tr><tr><td rowspan=1 colspan=1>RNase Inhibitor IV</td><td rowspan=1 colspan=1>•  40U/μL</td><td rowspan=1 colspan=1>50 μL</td><td rowspan=1 colspan=1>Lilac</td><td rowspan=1 colspan=1>403326</td><td rowspan=1 colspan=1>100</td></tr><tr><td rowspan=1 colspan=1>SSCS Control</td><td rowspan=1 colspan=1>•  Non-infectious DNA plasmid containing aportion of the targeted genes (Salmonella,Shigella, C. jejuni, stx1, and stx2)</td><td rowspan=1 colspan=1>400 μL</td><td rowspan=1 colspan=1>Red</td><td rowspan=1 colspan=1>403324</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>C. coli Control</td><td rowspan=1 colspan=1>•  Non-infectious DNA plasmid containing aportion of the targeted gene for C. coli only</td><td rowspan=1 colspan=1>400 μL</td><td rowspan=1 colspan=1>Yellow</td><td rowspan=1 colspan=1>403328</td><td rowspan=1 colspan=1>20</td></tr><tr><td rowspan=1 colspan=1>Gastro RNA/DNAInternal Control(GIC)</td><td rowspan=1 colspan=1>•   Non-infectious DNA plasmid•   Non-infectious in vitro transcribed RNA</td><td rowspan=1 colspan=1>30 μL</td><td rowspan=1 colspan=1>Green</td><td rowspan=1 colspan=1>403327</td><td rowspan=1 colspan=1>&gt;100</td></tr></table>

# Materials Required But Not Provided

# Plasticware and consumables

• RNase/DNase-free $1 . 5 \mathrm { m L }$ polypropylene microcentrifuge tubes   
• Sterile RNase/DNase-free filter or positive displacement micropipettor tips   
• Wide bore sterile RNase/DNase-free filter $2 0 0 \mu \mathrm { L }$ micropipettor tips   
• easyMAG System Disposables (Sample Strips and Aspiration Tips)   
• Biohit Pipette Tips for use with easyMAG System   
• Greiner Break Four uncoated plates for use with easyMAG System   
• Cepheid SmartCycler PCR reaction tubes

# Reagents

bioMérieux NucliSENS easyMAG reagents (Buffer 1 Cat. # 280130, Buffer 2 Cat. # 280131,Buffer 3 Cat. # 280132, Magnetic Silica Cat. # 280133, and Lysis

Buffer Cat. #. 280134) Cary Blair Transport Medium (Remel, Inc. Cat. No. R21610, R21617 or R21925), or ParaPak C&S System (Meridian Cat. No. 900612) • Molecular Grade Water (RNase/DNase Free) Extraction Control (recommended, e.g. previously characterized positive sample)

# Equipment

• $\leq - 7 0 ^ { \mathrm { o } } \mathrm { C }$ Freezer   
• bioMérieux NucliSENS easyMAG System with Software version 1.0.1 or 2.0   
• Biohit multi-channel pipettor for use with easyMAG System Cepheid SmartCycler II Real Time Instrument with Dx Software version 1.7b, 3.0a, or 3.0b Micropipettors (range between $1 { - } 1 0 ~ \mu \mathrm { L }$ , $1 0 { - } 2 0 0 ~ \mu \mathrm { L }$ , and $1 0 0 { \cdot } 1 0 0 0 \mu \mathrm { L } )$ Mini-centrifuge with adapter for Cepheid Reaction Tubes   
• Cepheid cooling block   
• Ice/Ice Bucket or $- 2 0 ^ { \circ } \mathrm { C }$ Cold Block   
• Biosafety Cabinet

# Interpretation of Specimen Results

The SmartCycler Dx software automatically determines the specimen results. The interpretation of the assay specimen results is as follows:

<table><tr><td colspan="8" rowspan="1">SSC Mix</td></tr><tr><td colspan="1" rowspan="1">SampleID</td><td colspan="1" rowspan="1">AssayResult</td><td colspan="1" rowspan="1">ICResult</td><td colspan="1" rowspan="1">Warning /Error Code</td><td colspan="1" rowspan="1">Campy.Result</td><td colspan="1" rowspan="1">SalmonellaResult</td><td colspan="1" rowspan="1">ShigellaResult</td><td colspan="1" rowspan="1">Interpretation of Results</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Negative</td><td colspan="1" rowspan="1">Pass</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Campylobacter jejuni and/orCampylobacter coli , Salmonella, andShigella nucleic acids not detected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Campylobacter jejuni and/orCampylobacter coli nucleic aciddetected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Salmonella spp. nucleic acid detected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">Shigella spp. nucleic acid detected</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">C. jejuni and/or C. coli and Salmonellaspp. nucleic acid detected. Dualinfections are not common, repeat testfrom the purified nucleic acid orcollect and test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">C. jejuni and/or C. coli and Shigellaspp. nucleic acid detected. Dualinfections are not common, repeat testfrom the purified nucleic acid orcollect and test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1">→</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">Salmonella spp. and Shigella spp.nucleic acid detected. Dual infectionsare not common, repeat test from thepurified nucleic acid or collect and testa new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Positive</td><td colspan="1" rowspan="1">NA*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">POS</td><td colspan="1" rowspan="1">C. jejuni and/or C. coli, Salmonellaspp., and Shigella spp. nucleic aciddetected. Triple infections are rare,</td></tr><tr><td colspan="1" rowspan="1">SampleID}$</td><td colspan="1" rowspan="1">AssayResult</td><td colspan="1" rowspan="1">ICResult</td><td colspan="1" rowspan="1">Warning /Error Code</td><td colspan="1" rowspan="1">Campy.Result</td><td colspan="1" rowspan="1">SalmonellaResult</td><td colspan="1" rowspan="1">ShigellaResult</td><td colspan="1" rowspan="1">Interpretation of Results</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">repeat test from the purified nucleicacid or collect and test a new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Unresolved</td><td colspan="1" rowspan="1">Fail</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">NEG</td><td colspan="1" rowspan="1">Unresolved - PCR inhibition orreagent failure. Repeat test from thepurified nucleic acid or collect and testa new sample.</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">30792</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Not Determined  error code 3079</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">Invalid</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">40983</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">ND</td><td colspan="1" rowspan="1">Not Determined  error code 4098</td></tr></table>

1 Columns and data not used for interpretation are not included 2 Error Code 3079: Warning/Error Code 3079 is periodically observed with Campy positives (Positive Control, Campy positive stool samples). Warning/Error Code 3079 occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a $C t$ value $\geq I 3$ is reported in the Campy, Salmonella, and/or Shigella Ct columns, the sample results can be recorded as POS for the specific analyte(s). 3 An Invalid assay run will display Error Code 4098 \* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High bacterial load can lead to reduced or absent Internal Control signal.

<table><tr><td rowspan=1 colspan=7>STEC Mix</td></tr><tr><td rowspan=1 colspan=1>SampleID}</td><td rowspan=1 colspan=1>AssayResult</td><td rowspan=1 colspan=1>ICResult</td><td rowspan=1 colspan=1>Warning /Error Code</td><td rowspan=1 colspan=1>stx2 Result</td><td rowspan=1 colspan=1>stx1Result</td><td rowspan=1 colspan=1>Interpretation of Results</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Pass</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>stx1 and stx2 nucleic acids notdetected</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>stx2 nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>stx1 nucleic acid detected</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>POS</td><td rowspan=1 colspan=1>stx1 and stx2 nucleic acid detected.</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Unresolved</td><td rowspan=1 colspan=1>Fail</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>NEG</td><td rowspan=1 colspan=1>Unresolved - PCR inhibition orreagent failure. Repeat test from thep  a new sample.</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>30792</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined - error code 3079</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>Invalid</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>40983</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>ND</td><td rowspan=1 colspan=1>Not Determined  error code 4098</td></tr></table>

1 Columns and data not used for interpretation are not included 2 Error Code 3079: Warning/Error Code 3079 is periodically observed with $S t x 2$ positives (Positive Control, Stx2 positive stool samples). Warning/Error Code $3 0 7 9$ occurs when the fluorescence (RFU) signal is too high. In this case, all results for that sample are reported by the Dx software as ND (Not Determined). If a $C t$ value $\geq I 3$ is reported in the Stx2 or Stx1 Ct columns, the sample results can be recorded as POS for the specific analyte(s). 3 An Invalid assay run will display Error Code 4098 \* Detection of the Internal Control in the Cy5 detection channel is not required for positive result. High bacterial load can lead to reduced or absent Internal Control signal.

# J. Substantial Equivalence Information:

1. Predicate device name(s): Luminex® xTAG™ Gastrointestinal Pathogen Panel (GPP).

2. Predicate K number(s): k121454

3. Comparison with predicate(s):

<table><tr><td rowspan=1 colspan=3>Similarities</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>Prodesse Pr0Gastro SSCS (k123274)</td><td rowspan=1 colspan=1>Luminex xTAG GPP (k121454)</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>Salmonella spp., Shigella spp., Campylobacter(C. jejuni and C. coli), and STEC (stx1 and stx2genes)</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Multiplex nucleic acid</td><td rowspan=1 colspan=1>Same(See below for differences)</td></tr><tr><td rowspan=1 colspan=1>SpecimenTypes</td><td rowspan=1 colspan=1>Stool specimens</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>User Complexity</td><td rowspan=1 colspan=1>High</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>SamplePreparationMethod</td><td rowspan=1 colspan=1>Up front sample processing is required to extractnucleic acids</td><td rowspan=1 colspan=1>Same</td></tr><tr><td rowspan=1 colspan=1>Controls</td><td rowspan=1 colspan=1>Internal control in each sample. External controlprocessed with each batch of samples.</td><td rowspan=1 colspan=1>Same</td></tr></table>

<table><tr><td rowspan=1 colspan=3>Differences</td></tr><tr><td rowspan=1 colspan=1>Element</td><td rowspan=1 colspan=1>Prodesse Pr0Gastro SSCS (k123274)</td><td rowspan=1 colspan=1>Luminex xTAG GPP (k121454)</td></tr><tr><td rowspan=1 colspan=1>OrganismsDetected</td><td rowspan=1 colspan=1>(See above for similarities)</td><td rowspan=1 colspan=1>Can also detect and distinguish C. lari. Inaddition, can detect and distinguishClostridium difficile toxin A/B,Cryptosporidium (C. parvum and C. hominisonly), Escherichia coli (E. coli) O157,Enterotoxigenic E. coli (ETEC) LT/ST,Giardia (G. lamblia only), Norovirus GI/GII,and Rotavirus A.</td></tr><tr><td rowspan=1 colspan=1>Analyte</td><td rowspan=1 colspan=1>DNA</td><td rowspan=1 colspan=1>RNA/DNA</td></tr><tr><td rowspan=1 colspan=1>TechnologicalPrinciples</td><td rowspan=1 colspan=1>Real time multiplex PCR based on the Taqmanreagent chemistry</td><td rowspan=1 colspan=1>Multiplex RT-PCR and bead hybridizationfollowed by Fluorescence-activated sorting oflabeled beads coupled to streptavidin-conjugated biotinylated products</td></tr><tr><td rowspan=1 colspan=1>Instrumentation</td><td rowspan=1 colspan=1>Cepheid SmartCycler II</td><td rowspan=1 colspan=1>PCR Thermocycler and Luminex 100/200system</td></tr><tr><td rowspan=1 colspan=1>Time to result</td><td rowspan=1 colspan=1>Approximately 3 hours</td><td rowspan=1 colspan=1>Approximately 6 to 12 hours</td></tr></table>

# K. Standard/Guidance Documents Referenced (if applicable):

Format for Traditional and Abbreviated 510(k)s - Guidance for Industry and FDA Staff • Guidance for Off-the-Shelf Software Use in Medical Devices; Final Statistical Guidance on Reporting Results from Studies Evaluating Diagnostic Tests; Draft Guidance for Industry and FDA Reviewers Guidance on Informed Consent for In Vitro Diagnostic Device Studies Using Leftover Human Specimens that are Not Individually Identifiable - Guidance for 7Sponsors, Institutional Review Boards, Clinical Investigators and FDA Staff

Draft Guidance on Establishing the Performance Characteristics of In Vitro Diagnostic Devices for the Detection of Clostridium difficile

# L. Test Principle:

The real-time PCR process simultaneously amplifies and detects nucleic acid targets in two separate closed-tube reactions. The ProGastro SSCS Assay enables detection and differentiation of Salmonella, Shigella, Campylobacter (C. jejuni and C. coli only, undifferentiated) and an Internal Control in the SSC Mix (the first tube), and Shiga Toxin 1 $( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes and an Internal Control in the STEC Mix (the second tube), and is based on two processes: nucleic acid isolation and Real Time polymerase chain reaction (PCR) amplification/detection. The Gastro RNA/DNA Internal Control (GIC) is added to each preserved stool specimen from symptomatic patient to monitor for inhibitors present. Nucleic acids are isolated and purified from the stool specimen. Purified nucleic acid is added to the SSC Mix and the STEC Mix. The SSC Mix and STEC Mix contain target specific oligonucleotide primers and oligonucleotide probes dual-labeled with a reporter dye attached to the $_ 5 \cdot$ end and a quencher dye attached to the $_ 3 \cdot$ end. Amplification proceeds during which the primers and probes anneal specifically to the templates (if present) followed by primer extension and amplification. The ProGastro SSCS Assay is based on Taqman chemistry, which utilizes the $5 ' - 3 '$ exonuclease activity of the Taq polymerase to cleave the probe thus separating the reporter dye from the quencher. This generates an increase in fluorescent signal upon excitation from a light source. With each cycle, additional reporter dye molecules are cleaved from their respective probes, further increasing the fluorescent signal. The amount of fluorescence at any given cycle is dependent on the amount of amplification product present at that time. Fluorescent intensity is monitored during each PCR cycle by the real-time instrument.

# M. Performance Characteristics (if/when applicable):

1. Analytical performance:

a. Precision/Reproducibility:

# Inter-Laboratory Reproducibility Study

An Inter-Laboratory Reproducibility study was conducted at three independent clinical laboratories: two external sites and one internal site at Gen-Probe Prodesse Inc. Panels of samples were tested by two operators at each site and each operator tested for a total of five days resulting in 10 runs per Mix per site. Panel samples were generated by dividing and spiking a ProGastro SSCS negative stool pool with combinations of five different bacterial strains detected by the ProGastro SSCS Assay including Salmonella enterica serotype Enteritidis (Salmonella Enteritidis), Shigella sonnei, Campylobacter jejuni, Campylobacter coli, and Shiga Toxin producing $E$ . coli (STEC). The simulated positive samples were spiked at two different concentrations with each spiked sample containing the following combined targets: Salmonella (Sal)/C. jejuni (CJ) or Shigella (Shi)/C. coli (Cc)/STEC. The simulated High Negative samples were spiked at one concentration with each spiked sample containing all five of the targets: Salmonella/C. jejuni/Shigella/C. coli/STEC. Each Reproducibility Panel consisted of three Medium Positives and three Low Positives for each of the two sample group $( \mathrm { n } = 2 \mathrm { ~ x ~ } 6 = 1 2 )$ ), and three High Negatives $( \mathrm { n } { = } 3 )$ for a total of 15 samples per panel. The High Negative concentration for each organism was at 4 logs below each bacterial strain’s LoD. The Low Positive concentration was at $2 \mathbf { x }$ Assay LoD and the Medium Positive concentration was at 1 log above the Assay LoD for all target strains except the C. jejuni and $C .$ . coli strains. C. jejuni Low Positives were tested at 20x LoD and Medium Positives at 2 logs above LoD and the $C .$ . coli Low Positives at $0 . 5 \log$ above $2 \mathbf { x }$ LoD, and Medium Positives at 1.5 logs above LoD. The Campylobacter strains were tested at higher concentrations because Campylobacter is very sensitive to environmental stressors including freezing where it loses viability. In-house studies (QC testing of Positive Matrix Controls and Verification Panels for the Clinical Study and QC testing of spiked stool pools for Reproducibility) have demonstrated decreased amount of Campylobacter DNA (increasing Ct values) over time.

Note: The average Ct value for the C. jejuni (ATCC 33291) Low Positives was 37.3, which is very close to the average Ct value of 38.8 for the same $C .$ . jejuni strain tested at the estimated LoD level in the Analytical Reactivity Study. The average Ct value for the C. coli (ATCC BAA-371) Low Positives was 35.6, which is very close to the average Ct value of 34.4 for the same $C .$ . coli strain tested at the estimated LoD level in the Analytical Reactivity Study. It appears that the effective DNA concentrations of the $C .$ jejuni (ATCC 33291) and $C .$ . coli (ATCC BAA-371) low positive samples tested in the Reproducibility Study are very close to the estimated LoD DNA concentrations for these two strains tested in the Analytical Reactivity Study. Therefore the levels of the $C .$ . jejuni (ATCC 33291) and C. coli (ATCC BAA-371) low positives tested in this reproducibility study are acceptable.

The bacterial strains used in the Inter-Laboratory Reproducibility Study and panel member concentrations are summarized in the table below:

<table><tr><td rowspan=1 colspan=3>Bacterial Strain Information for Inter-Laboratory Reproducibility Study Panel Members</td></tr><tr><td rowspan=1 colspan=1>Panel Member</td><td rowspan=1 colspan=1>Strains</td><td rowspan=1 colspan=1>Panel MemberConcentration</td></tr><tr><td rowspan=2 colspan=1>Sal/CJ MediumPositive</td><td rowspan=1 colspan=1>Salmonella Enteritidis ATCC 6961</td><td rowspan=1 colspan=1>Sal = 1x105 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ATCC 33291</td><td rowspan=1 colspan=1>CJ = 1x105 CFU/mL</td></tr><tr><td rowspan=2 colspan=1>Sal/CJ LowPositive</td><td rowspan=1 colspan=1>Salmonella Enteritidis ATCC 6961</td><td rowspan=1 colspan=1>Sal = 2x104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ATCC 33291</td><td rowspan=1 colspan=1>CJ = 2x104 CFU/mL</td></tr><tr><td rowspan=3 colspan=1>Shi/Cc/STECMedium Positive</td><td rowspan=1 colspan=1>Shigella sonnei ATCC 29029</td><td rowspan=1 colspan=1>Shi = 1x104 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli ATCC BAA-371</td><td rowspan=1 colspan=1>Cc = 1x105.5 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC O111:H8 (stx1+/stx2+), STEC CenterAcc# TW07926, Strain 3215-99</td><td rowspan=1 colspan=1>STEC = 1x105 CFU/mL</td></tr><tr><td rowspan=3 colspan=1>Shi/Cc/STEC LowPositive</td><td rowspan=1 colspan=1>Shigella sonnei ATCC 29029</td><td rowspan=1 colspan=1>Shi = 2x103 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli ATCC BAA-371</td><td rowspan=1 colspan=1>Cc = 2x104.5 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC O111:H8 (stx1+/stx2+), STEC CenterAcc# TW07926, Strain 3215-99</td><td rowspan=1 colspan=1>STEC = 2x104 CFU/mL</td></tr><tr><td rowspan=5 colspan=1>ALL High Negative</td><td rowspan=1 colspan=1>Salmonella Enteritidis ATCC 6961</td><td rowspan=1 colspan=1>Sal = 1x100 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei ATCC 29029</td><td rowspan=1 colspan=1>Shi = 1x10-1 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ATCC 33291</td><td rowspan=1 colspan=1>CJ = 1x10-1 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli ATCC BAA-371</td><td rowspan=1 colspan=1>Cc = 1x100 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC O111:H8 (stx1+/stx2+), STEC CenterAcc# TW07926, Strain 3215-9</td><td rowspan=1 colspan=1>STEC = 1x100 CFU/mL</td></tr></table>

The Panels were stored at $\mathrm { \leq - 7 0 ^ { \circ } C }$ and delivered frozen. One Panel was used for each of the five testing days per operator per site, generating a total of 90 data points per organism target per concentration (i.e., 3   
replicates/organism/concentration/operator/day/site x 2 operators/site x 3 sites x $5 \ \mathrm { d a y s } = 9 0 $ data points/organism/concentration). The samples were extracted each day using the bioMérieux NucliSENS easyMAG and the extracted nucleic acids were tested with the ProGastro SSCS Assay on the Cepheid SmartCycler II. ProGastro SSCS Assay controls (Positive, Positive Matrix, and Negative) were also included with each panel run. Two lots of ProGastro SSCS Assay SSC and STEC Mixes were used by each of the three sites.

A summary of the Reproducibility Study data is presented in the two tables below:

<table><tr><td rowspan=4 colspan=1></td><td rowspan=3 colspan=1>Panel Member ID</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=12>SSC Mix</td></tr><tr><td rowspan=1 colspan=1>d o un</td><td rowspan=1 colspan=1>pin unotd3</td><td rowspan=1 colspan=1>  n</td><td rowspan=1 colspan=1>cottd oC</td><td rowspan=1 colspan=1>ittod cn !C</td><td rowspan=1 colspan=1>  C</td><td rowspan=1 colspan=1>So otGod</td><td rowspan=1 colspan=1>Sme mobo</td><td rowspan=1 colspan=1>S oNeel</td><td rowspan=1 colspan=1>d o pi</td><td rowspan=1 colspan=1>S otd</td><td rowspan=1 colspan=1>S  </td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>20X*LoD</td><td rowspan=1 colspan=1>100X*LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>6X*LoD</td><td rowspan=1 colspan=1>30X*LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td></tr><tr><td rowspan=3 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.5</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>3.5</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.9</td><td rowspan=1 colspan=1>3.7</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/3010%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/301000%</td><td rowspan=1 colspan=1>29/30967%</td><td rowspan=1 colspan=1>30/301000%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.0</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=3 colspan=1>Site 3</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/3000%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/3000%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.7</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>32.8</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>89/90899.9%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-0.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-00.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-0.0%</td><td rowspan=1 colspan=1>95.9%-10.0%</td><td rowspan=1 colspan=1>94.0%-99.8%%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CtValue</td><td rowspan=1 colspan=1>37.3</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>2.8</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>3.2</td><td rowspan=1 colspan=1>3.1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>n/a</td></tr></table>

\* Note: The Campylobacter strains were tested at higher concentrations because Campylobacter is very sensitive to environmental stressors including freezing where it loses viability. However, the average $\mathrm { C t }$ value for the $C .$ . jejuni (ATCC 33291) Low Positives was 37.3, which is very close to the average $\mathrm { C t }$ value of 38.8 for the same $C .$ . jejuni strain tested at the estimated LoD level in the Analytical Reactivity Study. The average Ct value for the $C .$ . coli (ATCC BAA-371) Low Positives was 35.6, which is very close to the average $\mathrm { C t }$ 10 value of

34.4 for the same C. coli strain tested at the estimated LoD level in the Analytical Reactivity Study. Therefore, the effective DNA concentrations of the C. jejuni (ATCC 33291) and C. coli (ATCC BAA-371) low positive samples tested in the Reproducibility Study are very close to the estimated LoD DNA concentrations for these two strains tested in the Analytical Reactivity Study.

<table><tr><td rowspan=3 colspan=1></td><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=1 colspan=6>SSC Mix</td></tr><tr><td rowspan=1 colspan=1>MOT (I x) OOSoe</td><td rowspan=1 colspan=1>n ( x) iese</td><td rowspan=1 colspan=1>S ( x Neel</td><td rowspan=1 colspan=1>MOT (Z xs) SJode</td><td rowspan=1 colspan=1>n ( xs) ameod</td><td rowspan=1 colspan=1>S ( x) ∃sNeel</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td></tr><tr><td rowspan=3 colspan=1>Site 1</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=3 colspan=1>Site 2</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>36.2</td><td rowspan=1 colspan=1>33.6</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>2.5</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=3 colspan=1>Site 3</td><td rowspan=1 colspan=1>Agreement withExpected Result</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td><td rowspan=1 colspan=1>30/30100%</td></tr><tr><td rowspan=1 colspan=1>Mean Ct Value</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>% CV</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=4 colspan=1></td><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td><td rowspan=1 colspan=1>90/90100%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td><td rowspan=1 colspan=1>95.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CtValue</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>33.4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>n/a</td></tr></table>

Analysis of variance (ANOVA) for each source of variation was also performed and the results are presented in the tables below.

<table><tr><td rowspan=1 colspan=17>ProGastro SSCS Assay Inter-laboratory Reproducibility Analysis  SSC Mix</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>Mean Ct</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Salmonella Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>35.86</td><td rowspan=1 colspan=1>0.483</td><td rowspan=1 colspan=1>1.35</td><td rowspan=1 colspan=1>0.189</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>0.180</td><td rowspan=1 colspan=1>0.50</td><td rowspan=1 colspan=1>0.063</td><td rowspan=1 colspan=1>0.18</td><td rowspan=1 colspan=1>0.110</td><td rowspan=1 colspan=1>0.31</td><td rowspan=1 colspan=1>0.047</td><td rowspan=1 colspan=1>0.13</td><td rowspan=1 colspan=1>0.566</td><td rowspan=1 colspan=1>1.58</td></tr><tr><td rowspan=2 colspan=1>Salmonella MediumPositive</td><td rowspan=2 colspan=1>90</td><td rowspan=2 colspan=1>32.98</td><td rowspan=2 colspan=1>0.341</td><td rowspan=2 colspan=1>1.03</td><td rowspan=2 colspan=1>0.000</td><td rowspan=2 colspan=1>0.00</td><td rowspan=2 colspan=1>0.269</td><td rowspan=1 colspan=1>0.81</td><td rowspan=2 colspan=1>0.184</td><td rowspan=2 colspan=1>0.56</td><td rowspan=2 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=2 colspan=1>0.177</td><td rowspan=2 colspan=1>0.54</td><td rowspan=2 colspan=1>0.504</td><td rowspan=2 colspan=1>1.53</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Shigella Low Positive*</td><td rowspan=1 colspan=1>89</td><td rowspan=1 colspan=1>35.47</td><td rowspan=1 colspan=1>0.549</td><td rowspan=1 colspan=1>1.55</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.277</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.272</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.202</td><td rowspan=1 colspan=1>0.57</td><td rowspan=1 colspan=1>0.303</td><td rowspan=1 colspan=1>0.85</td><td rowspan=1 colspan=1>0.764</td><td rowspan=1 colspan=1>2.16</td></tr><tr><td rowspan=1 colspan=1>Shigella Medium Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>33.07</td><td rowspan=1 colspan=1>0.394</td><td rowspan=1 colspan=1>1.19</td><td rowspan=1 colspan=1>0.069</td><td rowspan=1 colspan=1>0.21</td><td rowspan=1 colspan=1>0.272</td><td rowspan=1 colspan=1>0.82</td><td rowspan=1 colspan=1>0.201</td><td rowspan=1 colspan=1>0.61</td><td rowspan=1 colspan=1>0.127</td><td rowspan=1 colspan=1>0.38</td><td rowspan=1 colspan=1>0.155</td><td rowspan=1 colspan=1>0.47</td><td rowspan=1 colspan=1>0.561</td><td rowspan=1 colspan=1>1.70</td></tr><tr><td rowspan=1 colspan=1>C. coli Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>35.64</td><td rowspan=1 colspan=1>0.486</td><td rowspan=1 colspan=1>1.36</td><td rowspan=1 colspan=1>0.714</td><td rowspan=1 colspan=1>2.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.829</td><td rowspan=1 colspan=1>2.33</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.198</td><td rowspan=1 colspan=1>3.36</td></tr><tr><td rowspan=1 colspan=1>C. coli Medium Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>33.28</td><td rowspan=1 colspan=1>0.462</td><td rowspan=1 colspan=1>1.39</td><td rowspan=1 colspan=1>0.577</td><td rowspan=1 colspan=1>1.73</td><td rowspan=1 colspan=1>0.178</td><td rowspan=1 colspan=1>0.53</td><td rowspan=1 colspan=1>0.770</td><td rowspan=1 colspan=1>2.31</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.082</td><td rowspan=1 colspan=1>3.25</td></tr><tr><td rowspan=1 colspan=1>C. jejuni Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>37.30</td><td rowspan=1 colspan=1>0.549</td><td rowspan=1 colspan=1>1.47</td><td rowspan=1 colspan=1>0.734</td><td rowspan=1 colspan=1>1.97</td><td rowspan=1 colspan=1>0.152</td><td rowspan=1 colspan=1>0.41</td><td rowspan=1 colspan=1>0.752</td><td rowspan=1 colspan=1>2.02</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>1.195</td><td rowspan=1 colspan=1>3.20</td></tr><tr><td rowspan=2 colspan=1>C. jejuni Medium Positive</td><td rowspan=2 colspan=1>90</td><td rowspan=2 colspan=1>34.81</td><td rowspan=2 colspan=1>0.381</td><td rowspan=2 colspan=1>1.09</td><td rowspan=2 colspan=1>0.634</td><td rowspan=2 colspan=1>1.82</td><td rowspan=2 colspan=1>0.091</td><td rowspan=1 colspan=1>0.26</td><td rowspan=2 colspan=1>0.694</td><td rowspan=2 colspan=1>1.99</td><td rowspan=2 colspan=1>0.000</td><td rowspan=2 colspan=1>0.00</td><td rowspan=2 colspan=1>0.000</td><td rowspan=2 colspan=1>0.00</td><td rowspan=2 colspan=1>1.018</td><td rowspan=2 colspan=1>2.92</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>High Negative (GIC)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>33.23</td><td rowspan=1 colspan=1>0.622</td><td rowspan=1 colspan=1>1.87</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.054</td><td rowspan=1 colspan=1>0.16</td><td rowspan=1 colspan=1>0.124</td><td rowspan=1 colspan=1>0.37</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.318</td><td rowspan=1 colspan=1>0.96</td><td rowspan=1 colspan=1>0.711</td><td rowspan=1 colspan=1>2.14</td></tr></table>

Note: 1.Variability from some factors may be numerically negative, which can occur if the variability due to those factors is very small. When this occurs, $\mathrm { S D } { = } 0$ and $C V { = } 0 \%$ . \*\*One sample False Negative for Shigella, data not included in analysis.

<table><tr><td rowspan=1 colspan=17>ProGastro SSCS Assay Inter-laboratory Reproducibility Analysis  STEC Mix</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>Within-Run</td><td rowspan=1 colspan=2>Between-Run</td><td rowspan=1 colspan=2>Between-Day</td><td rowspan=1 colspan=2>Between-Operator</td><td rowspan=1 colspan=2>Between-Lot</td><td rowspan=1 colspan=2>Between-Site</td><td rowspan=1 colspan=2>Total</td></tr><tr><td rowspan=1 colspan=1>Panel</td><td rowspan=1 colspan=1>N</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV (%)</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>CV(%)</td></tr><tr><td rowspan=1 colspan=1>Stx1 Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>35.94</td><td rowspan=1 colspan=1>0.699</td><td rowspan=1 colspan=1>1.95</td><td rowspan=1 colspan=1>0.273</td><td rowspan=1 colspan=1>0.76</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.279</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.801</td><td rowspan=1 colspan=1>2.23</td></tr><tr><td rowspan=1 colspan=1>Stx1 Medium Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>33.38</td><td rowspan=1 colspan=1>0.433</td><td rowspan=1 colspan=1>1.30</td><td rowspan=1 colspan=1>0.258</td><td rowspan=1 colspan=1>0.77</td><td rowspan=1 colspan=1>0.040</td><td rowspan=1 colspan=1>0.12</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.261</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.568</td><td rowspan=1 colspan=1>1.70</td></tr><tr><td rowspan=1 colspan=1>Stx2 Low Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>36.67</td><td rowspan=1 colspan=1>0.638</td><td rowspan=1 colspan=1>1.74</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.084</td><td rowspan=1 colspan=1>0.23</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.145</td><td rowspan=1 colspan=1>0.40</td><td rowspan=1 colspan=1>0.659</td><td rowspan=1 colspan=1>1.80</td></tr><tr><td rowspan=1 colspan=1>Stx2 Medium Positive</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>34.46</td><td rowspan=1 colspan=1>0.429</td><td rowspan=1 colspan=1>1.24</td><td rowspan=1 colspan=1>0.268</td><td rowspan=1 colspan=1>0.78</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.212</td><td rowspan=1 colspan=1>0.62</td><td rowspan=1 colspan=1>0.548</td><td rowspan=1 colspan=1>1.59</td></tr><tr><td rowspan=1 colspan=1>High Negative (GIC)</td><td rowspan=1 colspan=1>90</td><td rowspan=1 colspan=1>32.90</td><td rowspan=1 colspan=1>0.423</td><td rowspan=1 colspan=1>1.29</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.000</td><td rowspan=1 colspan=1>0.00</td><td rowspan=1 colspan=1>0.057</td><td rowspan=1 colspan=1>0.17</td><td rowspan=1 colspan=1>0.295</td><td rowspan=1 colspan=1>0.90</td><td rowspan=1 colspan=1>0.519</td><td rowspan=1 colspan=1>1.58</td></tr></table>

Note: 1.Variability from some factors may be numerically negative, which can occur if the variability due to those factors is very small. When this occurs, $\mathrm { S D } { = } 0$ and $C V { = } 0 \%$ .

# Intra-Laboratory Reproducibility Study

An Intra-Laboratory Precision study was conducted internally at Gen-Probe Prodesse Inc. Panels of samples were tested by two operators and each operator tested for a total of 12 days resulting in 24 runs per PCR Mix. Panel samples were generated as described previously in the Inter-Laboratory Reproducibility Study. One Panel was used for each of the 12 testing days per operator. The samples were extracted each day using the bioMérieux NucliSENS easyMAG and the extracted nucleic acids were tested with the ProGastro SSCS Assay on the Cepheid SmartCycler II. ProGastro SSCS Assay controls (Positive, Positive Matrix, and Negative) were also included with each panel run. Two lots of ProGastro SSCS Assay SSC and STEC Mixes were used by each of the operators.

A summary of the Intra-Laboratory Precision Study data is presented in the tables below:   

<table><tr><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=1 colspan=12>SSC Mix</td></tr><tr><td rowspan=1 colspan=1>d o n</td><td rowspan=1 colspan=1>p nod</td><td rowspan=1 colspan=1> h n</td><td rowspan=1 colspan=1>otod o!C</td><td rowspan=1 colspan=1>cood wrmiaC</td><td rowspan=1 colspan=1>i  C</td><td rowspan=1 colspan=1>So oMoe</td><td rowspan=1 colspan=1>Sn mommbosde</td><td rowspan=1 colspan=1>s ohNeet</td><td rowspan=1 colspan=1>itd To pi</td><td rowspan=1 colspan=1>Sm motd</td><td rowspan=1 colspan=1>S  h</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>20X*LoD</td><td rowspan=1 colspan=1>100X*LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>6X*LoD</td><td rowspan=1 colspan=1>30X*LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10XLoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td></tr><tr><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>71/7298.6%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>71/7298.6%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>92.5%-99.8%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>92.5%-99.8%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CtValue</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Overall%CV</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.6</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>n/a</td></tr></table>

\* Note: The Campylobacter strains were tested at higher concentrations because Campylobacter is very sensitive to environmental stressors including freezing where it loses viability. However, the average Ct value for the $C .$ . jejuni (ATCC 33291) Low Positives was 37.3, which is very close to the average $\mathrm { C t }$ value of 38.8 for the same $C .$ . jejuni strain tested at the estimated LoD level in the Analytical Reactivity Study. The average Ct value for the $C .$ . coli (ATCC BAA371) Low Positives was 35.6, which is very close to the average Ct value of 34.4 for the same $C .$ . coli strain tested at the estimated LoD level in the Analytical Reactivity Study. Therefore, the effective DNA concentrations of the $C .$ . jejuni (ATCC 33291) and $C$ . coli (ATCC BAA-371) low positive samples tested in the Precision Study are very close to the estimated LoD DNA concentrations for these two strains tested in the Analytical Reactivity Study.

<table><tr><td rowspan=2 colspan=1>Panel Member ID</td><td rowspan=1 colspan=6>SSC Mix</td></tr><tr><td rowspan=1 colspan=1>SOT( ) SISoiid</td><td rowspan=1 colspan=1>Srp ( x) inmote</td><td rowspan=1 colspan=1> () NEe</td><td rowspan=1 colspan=1>OT (Z ) SOSod</td><td rowspan=1 colspan=1>Srp ( x) anmoid</td><td rowspan=1 colspan=1>S (  e</td></tr><tr><td rowspan=1 colspan=1>Concentration</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td><td rowspan=1 colspan=1>2XLoD</td><td rowspan=1 colspan=1>10X LoD</td><td rowspan=1 colspan=1>0.0001XLoD</td></tr><tr><td rowspan=1 colspan=1>Total Agreementwith ExpectedResult</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td><td rowspan=1 colspan=1>72/72100%</td></tr><tr><td rowspan=1 colspan=1>95% CI</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td><td rowspan=1 colspan=1>94.9%-100.0%</td></tr><tr><td rowspan=1 colspan=1>Overall Mean CtValue</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>34.2</td><td rowspan=1 colspan=1>n/a</td></tr><tr><td rowspan=1 colspan=1>Overall% CV</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>n/a</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>n/a</td></tr></table>

b. Linearity/assay reportable range:

Not applicable, qualitative assay

c. Traceability, Stability, Expected values (controls, calibrators, or methods):

# Assay Controls

The following controls are provided in the ProGastro SSCS Assay kit:

Positive Controls (PCs): The ProGastro SSCS Assay kit contains an SSCS Positive DNA Control and a C. Coli Positive DNA Control. Both PCs are DNA plasmids containing portions of the Assay target genes of interest. The SSCS Control (PC1) plasmid contains approximately 200 bp of target sequence for each of the following organisms or genes: Campylobacter jejuni, Salmonella, Shigella, stx1 and stx2. The SSCS Control is used with both the SSC Mix and the STEC Mix. The C. coli Control (PC2) plasmid contains a 403 bp target sequence specific for Campylobacter coli. The C. Coli Control is used with the SSC Mix only. The PCs do not go through nucleic acid isolation and purification, but are included during set-up of the PCR reaction. The PCs in conjunction with the IC are used to verify reagent and system performance. The SSCS Control (PC1, SAP# 403324) and the $C .$ . coli Control (PC2, SAP# 403328) are sold at $\mathrm { 1 x 1 0 ^ { 4 } c / \mu L }$ and should be used at $1 \mathrm { x } 1 0 ^ { 3 }$ copies $ { ^ \mathrm { / { u L } } }$ .

Because the PC is meant to assess global failure of the assay (reagent, technical, instrument failure, etc.) and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable Ct range was set to 20.0 to 45.0 in the Clinical Trial Protocol for all targets (except Shigella: 20.0 to 37.0).

Of a total of 77 PC1 tested by the ProGastro SSCS Assay during the clinical trial, $100 \%$ (77/77) of these controls gave correct results. The average Ct of the PC1s tested was 31.5 (Min 29.1- Max 34.4), with 1.1 Standard Deviation (STDEV) and $3 . 6 \%$ CV; 27.6 (Min 25.3- Max 30.8), with 1.1 STDEV and $4 . 0 \%$ ; 29.2 (Min 27.2- Max 31.2), with 1.1 STDEV and $3 . 6 \%$ CV; 30.7 (Min 29.6- Max 32.5), with 0.6 STDEV and 1.8 $\%$ CV; and 28.5 (Min 27.5- Max 30.3), with 0.6 STDEV and $2 . 2 \%$ , for the Campylobacter jejuni, Salmonella, Shigella, stx1, and stx2, respectively. The prospective clinical study data validated the pre-determined PC1 Ct acceptance range of 20.0 to 45.0 for Campylobacter jejuni, Salmonella, stx1, and stx2, and 20.0 to 37.0 for Shigella.

Of a total of 77 PC2 tested by the ProGastro SSCS Assay during the clinical trial, $100 \%$ (77/77) of these controls gave correct results. The average Ct of the PC2s tested was 28.9 (Min 26.8- Max 34.2), with 1.6 Standard Deviation (STDEV) and $5 . 6 \%$ CV for Campylobacter coli. The prospective clinical study data validated the predetermined PC2 Ct acceptance range of 20.0 to 45.0 for Campylobacter coli.

Internal Control (IC): A Gastro RNA/DNA Internal Control (GIC) consisting of a pooled non-infectious DNA plasmid and RNA transcript is incorporated into every sample and is carried through all steps of the procedure from nucleic acid isolation and purification through amplification to monitor for inhibitors present in the specimen or reaction tube. The GIC contains a DNA and an RNA portion to allow for one nucleic acid extract to be used with multiple RT-PCR/PCR assays whether the assay detects a DNA or an RNA target. The GIC also serves as a general process control ensuring that each step of the procedure was performed correctly, assay and instrument parameters were set correctly, and that general reagents were working. The GIC should be used at $5 \mathrm { x } 1 0 ^ { 5 } \mathrm { c } / \mu \mathrm { L }$ for the RNA and $1 \mathrm { x } 1 \mathrm { 0 } ^ { 3 } \mathrm { c } / \mu \mathrm { L }$ DNA portion, and sold at ${ 5 { \mathrm { x } } 1 0 } ^ { 7 } { \mathrm { c } } / { \mu \mathrm { L } }$ for the RNA and $1 \mathrm { x } 1 0 ^ { 5 } \mathrm { c / \mu L }$ for the DNA portion.

The IC is meant to assess global failure (reagent or process) and monitor for PCR inhibition. Because the IC is assessing global failure and the customer variance components were to be determined during the clinical trial, a broad Ct range was desired. The acceptable range for the Internal Control (IC) was set to $1 3 - 4 5 \mathrm { C t }$ in the Clinical Trial Protocol. For the SSC Mix, the average Ct of all eligible prospective ProGastro SSCS Assay negative clinical specimens was 33.8 (Min 31.2- Max 44.2), with 1.60 Standard Deviation (STDEV) and $4 . 7 0 \%$ CV. The average Ct of all eligible ProGastro SSCS Assay Negative Control was 34.4 (Min 33.0- Max 36.2), with 0.6 Standard Deviation (STDEV) and $1 . 9 0 \%$ CV. For the STEC Mix, the average $\mathrm { C t }$ of all eligible prospective ProGastro SSCS Assay negative clinical specimens was 33.6 (Min 31.3- Max 42.6), with 1.60 Standard Deviation (STDEV) and $4 . 8 0 \%$ CV. The average Ct of all eligible ProGastro SSCS Assay Negative Control was 34.1 (Min 33.1- Max 36.2), with 0.5 Standard Deviation (STDEV) and $1 . 5 0 \%$ CV. The prospective clinical study data validated the pre-determined IC Ct acceptance range of 13.0 to 45.0.

Final release specifications were established during Test Method Characterization (TMC) and then validated against the appropriate analytical performance characteristics of specificity and total imprecision.

Final release specifications were developed to ensure First Pass Acceptance (FPA) rates $67 \%$ at lot release and customer rejection rate of $\leq 1 \%$ for a lot released at Acceptable Quality Level (AQL) (i.e. $9 5 \%$ probability of QC lot acceptance at lot release). For customer specifications, the maximum allowable customer lot rejection rate at the Rejectable Quality Level (RQL) Limit (i.e. $10 \%$ probability of QC acceptance at lot release) is $1 \%$ . A false valid risk of $\leq 5 \%$ was used to determine the validity specifications.

QC release specification ranges were set using Operator Characteristics (OC) curve analysis. Lot, technician, instrument, run, and replicate variance components were obtained during test method characterization. Observed $\mathrm { C t }$ means were adjusted for stability effects and specifications determined to meet the validity and specification requirements above.

Positive Control and Internal Control Specifications – SSC Mix   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Customer CtSpecification</td><td rowspan=1 colspan=1>Kit Control Final ReleaseSpecifications</td></tr><tr><td rowspan=1 colspan=1>SSCS PC (Campylobacter)</td><td rowspan=1 colspan=1>20 - 45</td><td rowspan=1 colspan=1>29.2 - 35.8</td></tr><tr><td rowspan=1 colspan=1>SSCS PC (Salmonella)</td><td rowspan=1 colspan=1>20 - 45</td><td rowspan=1 colspan=1>25.0 - 31.6</td></tr><tr><td rowspan=1 colspan=1>SSCS PC (Shigella)</td><td rowspan=1 colspan=1>20 - 37</td><td rowspan=1 colspan=1>26.8 - 33.4</td></tr><tr><td rowspan=1 colspan=1>C. coli PC (Campylobacter)</td><td rowspan=1 colspan=1>20 - 45</td><td rowspan=1 colspan=1>26.3 - 32.9</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>13 - 45</td><td rowspan=1 colspan=1>26.9 - 39.9</td></tr></table>

Positive Control and Internal Control Specifications – STEC Mix   

<table><tr><td rowspan=1 colspan=1>Control</td><td rowspan=1 colspan=1>Customer Ct Specification</td><td rowspan=1 colspan=1>Kit Control Final ReleaseSpecifications</td></tr><tr><td rowspan=1 colspan=1>SSCS PC (stx1)</td><td rowspan=1 colspan=1>20 - 45</td><td rowspan=1 colspan=1>24.9 - 31.5</td></tr><tr><td rowspan=1 colspan=1>SSCS PC (stx2)</td><td rowspan=1 colspan=1>20 - 45</td><td rowspan=1 colspan=1>27.2 - 33.8</td></tr><tr><td rowspan=1 colspan=1>Internal Control</td><td rowspan=1 colspan=1>13 - 45</td><td rowspan=1 colspan=1>26.1 - 39.9</td></tr></table>

Negative Control: A Negative Control is not provided with the kit, but is required and described in the ProGastro SSCS Assay package insert. Stool preservation and transport media (SPTM, Para-Pak C&S or Cary-Blair) spiked with the GIC is to be used as the negative control and processed starting from nucleic acid isolation. The negative control serves to monitor for contamination.

A Positive Matrix Control was included in the ProGastro SSCS Clinical Study but is not provided in the ProGastro SSCS Assay kit:

Positive Matrix Control (PMC): The Positive Matrix Control used in the clinical study is a pool of the ProGastro SSCS Assay target organisms (see the table below) diluted in SPTM and spiked into negative stool matrix. The PMC was included with each nucleic acid isolation run performed for the Clinical and Reproducibility/Precision studies. The PMC served to monitor for lysis and during nucleic acid isolation.

Bacterial strains used for the Positive Matrix Control   

<table><tr><td rowspan=1 colspan=1>Strain information</td><td rowspan=1 colspan=1>Final concentration in PMC</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ATCC# 33291</td><td rowspan=1 colspan=1>1 x 104 CFU/mL(082211)1 x 105 CFU/mL (022812)</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli ATCC# BAA-371</td><td rowspan=1 colspan=1>1 x 105 CFU/mL(082211)1 x 106 CFU/mL (022812)</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Typhimurium ATCC# BAA-191</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri ATCC# 12022</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC Strain 97-3250 O26:H11 (stx1+/stx2+)</td><td rowspan=1 colspan=1>1 x 106 CFU/mL</td></tr></table>

A total of 136 runs were attempted in the clinical trial and there were no run failures due to PMC failure.

The sponsor is also recommending the following in the product package insert: “Good laboratory practice recommends including a positive extraction control (e.g. previously characterized positive sample or negative sample spiked with a well characterized ProGastro SSCS Assay target strain) in each nucleic acid isolation run. The extraction control should be treated like a sample during assay performance and analysis.”

# Specimen and Nucleic Acid Stability Study (Fresh vs. Frozen Study)

An analytical study was conducted to determine the effect that storing (refrigerated) and freezing stool preserved in Stool Preservation and Transport Media (SPTM, ParaPak C&S) as well as the effects of freezing the resultant purified nucleic acids have on the performance of the ProGastro SSCS Assay

In this study, contrived samples were generated using separate aliquots from a ProGastro SSCS negative stool pool spiked with one strain of Salmonella, Shigella, Campylobacter jejuni, and Shiga Toxin producing $E .$ . coli (STEC) representing the four bacterial targets of the ProGastro SSCS Assay. Due to the fact that natural fresh positive samples were not available for this study, samples were made at varying concentrations in negative stool matrix preserved in SPTM (ParaPak C&S).

The contrived sample panel was prepared so that approximately $60 \%$ of the samples had analyte levels close to their LoD. Of the samples near LoD, approximately $50 \%$ were tested at LoD and the other half were Moderate Positive $3 \mathrm { \Delta x }$ LoD) samples. The remaining positive samples were tested at 1, 2, and 3 logs above the LoD for each bacterial strain. Therefore, for each of the four strains, ten samples were generated $( \mathrm { n } { = } 4 0 )$ in the following manner: one sample each at 3 logs, 2 logs, and 1 log above LoD, four samples of 3x LoD, and three samples at LoD. See the table below:

<table><tr><td rowspan=1 colspan=6>Bacterial Strains and Concentrations</td></tr><tr><td rowspan=3 colspan=1>Strain</td><td rowspan=1 colspan=5>Concentration (CFU/mL)</td></tr><tr><td rowspan=2 colspan=1>1x LoD (3Replicates)</td><td rowspan=2 colspan=1>3 x LoD (4Replicates)</td><td rowspan=1 colspan=3>1 Replicate each</td></tr><tr><td rowspan=1 colspan=1>1 log aboveLoD</td><td rowspan=1 colspan=1>2 logs aboveLoD</td><td rowspan=1 colspan=1>3 logs aboveoOD</td></tr><tr><td rowspan=1 colspan=1>Salmonella Enteritidis ATCC6961, 1.06x10° CFU/mL</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>3x104CFU/mL</td><td rowspan=1 colspan=1>1x105CFU/mL</td><td rowspan=1 colspan=1>1x106CFU/mL</td><td rowspan=1 colspan=1>1x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri ATCC 12025,3.30x10° CFU/mL</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>3x104CFU/mL</td><td rowspan=1 colspan=1>1x105CFU/mL</td><td rowspan=1 colspan=1>1x106CFU/mL</td><td rowspan=1 colspan=1>1x107CFU/mL</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni ATCC33291, 7.80x107 CFU/mL</td><td rowspan=1 colspan=1>1x103CFU/mL</td><td rowspan=1 colspan=1>3x103CFU/mL</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>1x105CFU/mL</td><td rowspan=1 colspan=1>1x106CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC 0157:HN (stx1+/stx2+),STEC Center Acc# TW07700,1.55x109 CFU/mL</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>3x104CFU/mL</td><td rowspan=1 colspan=1>1x105CFU/mL</td><td rowspan=1 colspan=1>1x106CFU/mL</td><td rowspan=1 colspan=1>1x107CFU/mL</td></tr></table>

After sample generation, a portion of each fresh prepared sample was extracted on the bioMérieux NucliSENS easyMAG. The remaining portions of the samples were stored refrigerated $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for five days (120 hours) and then extracted again as described above. After refrigeration for five days, the remaining portion of each sample were frozen solid at $\mathrm { \leq - 7 0 \mathrm { \mathrm { \mathrm { \mathrm { \Omega } } } C } }$ for a minimum of 24 hours and then thawed and extracted as described above. At each time point, the nucleic acids were run “fresh” (i.e., immediately subsequent to extraction) in a single replicate using one lot of ProGastro SSCS reagents. The remaining nucleic acids (NA) were then frozen at ${ < } . 7 0 { } ^ { \circ } \mathrm { C }$ for a minimum of 24 hours and later thawed on ice and subject to PCR as described above. The data generated was analyzed for possible negative effects of storing and freezing preserved stool samples and freezing purified nucleic acids when used in conjunction with the ProGastro SSCS Assay.

Positive and Negative percent agreements were calculated and compared to the Fresh Stool Sample/Fresh NA (FR ST/FR NA) data. Each bacterial strain and each concentration were required to not test statistically significantly different in terms of positive/negative result (percent agreement) $( \mathrm { p } { < } 0 . 0 5 )$ for all testing points/concentrations to be deemed equivalent. A paired t-test of Ct values compared to FR ST/FR NA was also performed for positive samples at all testing points and analyzed for informational purposes.

First, the number of positive samples was identified at each concentration at each strain for the fresh samples. The remaining testing points were compared to those numbers to determine the percent agreement to the initial control result:

<table><tr><td rowspan=1 colspan=8>Sample and Nucleic Acid Stability Results — % Agreement to the Fresh ST / Fresh NA Data</td></tr><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Concentration</td><td rowspan=1 colspan=6>% Agreement</td></tr><tr><td rowspan=1 colspan=1>FRST/FRNA</td><td rowspan=1 colspan=1>FR ST/FZ NA²</td><td rowspan=1 colspan=1>REF ST/FR NA3</td><td rowspan=1 colspan=1>REF ST/FZ NA4</td><td rowspan=1 colspan=1>FZ ST/FR NA</td><td rowspan=1 colspan=1>FZ ST/FZ NA6</td></tr><tr><td rowspan=6 colspan=1>Skoe</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4 = 100%</td></tr><tr><td rowspan=1 colspan=1>@ LoD</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td></tr><tr><td rowspan=6 colspan=1>Skie</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4= 100%</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4=100%</td></tr><tr><td rowspan=1 colspan=1>@ LoD</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3=100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3= 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td></tr><tr><td rowspan=6 colspan=1>n</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4= 100%</td><td rowspan=1 colspan=1>4/4= 100%</td></tr><tr><td rowspan=1 colspan=1>@ LoD</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3=100%*</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td></tr><tr><td rowspan=6 colspan=1>STC</td><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>2 logs above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>1 log above LoD</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td><td rowspan=1 colspan=1>1/1 = 100%</td></tr><tr><td rowspan=1 colspan=1>3x LoD</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4 = 100%</td><td rowspan=1 colspan=1>4/4=100%</td><td rowspan=1 colspan=1>4/4 = 100%</td></tr><tr><td rowspan=1 colspan=1>@ LoD</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td><td rowspan=1 colspan=1>3/3 = 100%</td></tr><tr><td rowspan=1 colspan=1>Overall</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td><td rowspan=1 colspan=1>10/10 = 100%</td></tr><tr><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1>1.000</td><td rowspan=1 colspan=1>1.000</td></tr></table>

1 FR ST/FR NA: Fresh contrived Stool samples/Freshly extracted nucleic acids 2 FR ST/FZ NA: Fresh contrived Stool samples/Frozen nucleic acids 3 REF ST/FR NA: Refrigerated contrived Stool samples/Freshly extracted nucleic acids 4 REF ST/FZ NA: Refrigerated contrived Stool samples/Frozen nucleic acids $\mathbf { 5 _ { F Z } }$ ST/FR NA: Frozen contrived Stool samples/Fresh nucleic acids ${ } ^ { 6 } { \bf F } { \bf Z }$ ST/FZ NA: Frozen contrived Stool samples/Frozen nucleic acids

The mean Ct for each sample(s) at a particular bacterial concentration was calculated and the difference in mean Ct for the latter five time points to the FR ST/FR NA was calculated and summarized in the tables below:

<table><tr><td colspan="7" rowspan="1">Mean Ct Values</td></tr><tr><td colspan="1" rowspan="1">Sample ID*</td><td colspan="1" rowspan="1">FR ST/FRNA Ct</td><td colspan="1" rowspan="1">FR ST/FZNA Ct</td><td colspan="1" rowspan="1">REF ST/FRNA Ct</td><td colspan="1" rowspan="1">REF ST/FZNA Ct</td><td colspan="1" rowspan="1">FZ ST/FRNA Ct</td><td colspan="1" rowspan="1">FZ ST/FZNA Ct</td></tr><tr><td colspan="1" rowspan="1">Salmonella at LoD</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">37.1</td></tr><tr><td colspan="1" rowspan="1">Salmonella 3X LoD</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">35.0</td><td colspan="1" rowspan="1">34.9</td></tr><tr><td colspan="1" rowspan="1">Salmonella 1 log above LoD</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">33.1</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">33.2</td><td colspan="1" rowspan="1">34.1</td></tr><tr><td colspan="1" rowspan="1">Salmonella 2 logs above LoD</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">29.5</td><td colspan="1" rowspan="1">30.8</td><td colspan="1" rowspan="1">28.7</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">30.0</td></tr><tr><td colspan="1" rowspan="1">Salmonella 3 logs above LoD</td><td colspan="1" rowspan="1">26.3</td><td colspan="1" rowspan="1">26.0</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">25.8</td><td colspan="1" rowspan="1">26.5</td><td colspan="1" rowspan="1">26.2</td></tr><tr><td colspan="1" rowspan="1">Shigella at LoD</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">32.0</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">32.8</td><td colspan="1" rowspan="1">32.2</td></tr><tr><td colspan="1" rowspan="1">Shigella 3X LoD</td><td colspan="1" rowspan="1">30.6</td><td colspan="1" rowspan="1">30.6</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">31.0</td><td colspan="1" rowspan="1">30.7</td></tr><tr><td colspan="1" rowspan="1">Shigella 1 log above LoD</td><td colspan="1" rowspan="1">29.0</td><td colspan="1" rowspan="1">28.8</td><td colspan="1" rowspan="1">29.8</td><td colspan="1" rowspan="1">28.8</td><td colspan="1" rowspan="1">29.4</td><td colspan="1" rowspan="1">29.1</td></tr><tr><td colspan="1" rowspan="1">Shigella 2 logs above LoD</td><td colspan="1" rowspan="1">25.3</td><td colspan="1" rowspan="1">24.4</td><td colspan="1" rowspan="1">26.0</td><td colspan="1" rowspan="1">25.7</td><td colspan="1" rowspan="1">26.1</td><td colspan="1" rowspan="1">25.3</td></tr><tr><td colspan="1" rowspan="1">Shigella 3 logs above LoD</td><td colspan="1" rowspan="1">21.8</td><td colspan="1" rowspan="1">21.5</td><td colspan="1" rowspan="1">22.3</td><td colspan="1" rowspan="1">22.4</td><td colspan="1" rowspan="1">22.6</td><td colspan="1" rowspan="1">22.2</td></tr><tr><td colspan="1" rowspan="1">C. jejuni at LoD</td><td colspan="1" rowspan="1">39.1</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">41.2</td><td colspan="1" rowspan="1">42.4</td></tr><tr><td colspan="1" rowspan="1">C. jejuni 3X LoD</td><td colspan="1" rowspan="1">37.1</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">37.3</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">40.8</td></tr><tr><td colspan="1" rowspan="1">C. jejuni 1 log above LoD</td><td colspan="1" rowspan="1">35.2</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">35.4</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">37.5</td><td colspan="1" rowspan="1">38.2</td></tr><tr><td colspan="1" rowspan="1">C. jejuni 2 logs above LoD</td><td colspan="1" rowspan="1">31.8</td><td colspan="1" rowspan="1">31.6</td><td colspan="1" rowspan="1">32.1</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">33.9</td><td colspan="1" rowspan="1">33.5</td></tr><tr><td colspan="1" rowspan="1">C. jejuni 3 logs above LoD</td><td colspan="1" rowspan="1">28.1</td><td colspan="1" rowspan="1">27.6</td><td colspan="1" rowspan="1">28.3</td><td colspan="1" rowspan="1">27.8</td><td colspan="1" rowspan="1">30.1</td><td colspan="1" rowspan="1">30.1</td></tr><tr><td colspan="1" rowspan="1">STEC at LoD {stx2}</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">38.4</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">37.4</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">38.1</td></tr><tr><td colspan="1" rowspan="1">STEC 3X LoD {stx2}</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">35.8</td></tr><tr><td colspan="1" rowspan="1">STEC 1 log above LoD {stx2}</td><td colspan="1" rowspan="1">34.1</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">33.3</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">33.9</td></tr><tr><td colspan="1" rowspan="1">STEC 2 logs above LoD {stx2}</td><td colspan="1" rowspan="1">31.4</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">31.3</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">31.2</td><td colspan="1" rowspan="1">31.0</td></tr><tr><td colspan="1" rowspan="1">STEC 3 logs above LoD {stx2}</td><td colspan="1" rowspan="1">27.7</td><td colspan="1" rowspan="1">27.6</td><td colspan="1" rowspan="1">27.8</td><td colspan="1" rowspan="1">27.6</td><td colspan="1" rowspan="1">28.0</td><td colspan="1" rowspan="1">27.9</td></tr><tr><td colspan="1" rowspan="1">STEC at LoD {stx1}</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">37.7</td><td colspan="1" rowspan="1">38.6</td><td colspan="1" rowspan="1">38.2</td></tr><tr><td colspan="1" rowspan="1">STEC 3X LoD {stx1}</td><td colspan="1" rowspan="1">36.9</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">35.3</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">35.5</td></tr><tr><td colspan="1" rowspan="1">STEC 1 log above LoD {stx1}</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">34.3</td><td colspan="1" rowspan="1">34.6</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">34.3</td></tr><tr><td colspan="1" rowspan="1">STEC 2 logs above LoD {stx1}</td><td colspan="1" rowspan="1">30.9</td><td colspan="1" rowspan="1">30.4</td><td colspan="1" rowspan="1">31.0</td><td colspan="1" rowspan="1">30.3</td><td colspan="1" rowspan="1">30.7</td><td colspan="1" rowspan="1">30.4</td></tr><tr><td colspan="1" rowspan="1">STEC 3 logs above LoD {stx1}</td><td colspan="1" rowspan="1">27.2</td><td colspan="1" rowspan="1">26.8</td><td colspan="1" rowspan="1">27.1</td><td colspan="1" rowspan="1">26.7</td><td colspan="1" rowspan="1">27.3</td><td colspan="1" rowspan="1">27.0</td></tr></table>

\* Only one replicate tested at each of the 1 log above LoD, 2 logs above LoD, and 3 logs above LoD levels per the study plan.

<table><tr><td rowspan=1 colspan=6>Mean Ct Difference to initial FR ST/FR NA Data</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>FR ST/FZ NA</td><td rowspan=1 colspan=1>REF ST/FR NA</td><td rowspan=1 colspan=1>REF ST/FZ NA</td><td rowspan=1 colspan=1>FZ ST/FR NA</td><td rowspan=1 colspan=1>FZ ST/FZ NA</td></tr><tr><td rowspan=1 colspan=1>Salmonella at LoD</td><td rowspan=1 colspan=1>-1.2</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>-2.1</td><td rowspan=1 colspan=1>-0.8</td><td rowspan=1 colspan=1>-0.1</td></tr><tr><td rowspan=1 colspan=1>Salmonella 3X LoD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>-0.7</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Salmonella 1 log above LoD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>1.0</td></tr><tr><td rowspan=1 colspan=1>Salmonella 2 logs above LoD</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>-0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>Salmonella 3 logs above LoD</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>-0.1</td></tr><tr><td rowspan=1 colspan=1>Shigella at LoD</td><td rowspan=1 colspan=1>-0.4</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>Shigella 3X LoD</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Shigella 1 log above LoD</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.1</td></tr><tr><td rowspan=1 colspan=1>Shigella 2 logs above LoD</td><td rowspan=1 colspan=1>-0.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>Shigella 3 logs above LoD</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>C. jejuni at LoD</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>2.1*</td><td rowspan=1 colspan=1>3.3*</td></tr><tr><td rowspan=1 colspan=1>C. jejuni 3X LoD</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>2.5*</td><td rowspan=1 colspan=1>3.7*</td></tr><tr><td rowspan=1 colspan=1>C. jejuni 1 log above LoD</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>2.3*</td><td rowspan=1 colspan=1>3.0*</td></tr><tr><td rowspan=1 colspan=1>C. jejuni 2 logs above LoD</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>-0.6</td><td rowspan=1 colspan=1>2.1*</td><td rowspan=1 colspan=1>1.7*</td></tr><tr><td rowspan=1 colspan=1>C. jejuni 3 logs above LoD</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>2.0*</td><td rowspan=1 colspan=1>2.0*</td></tr><tr><td rowspan=1 colspan=1>STEC at LoD {stx2}</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.2</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>0.5</td></tr><tr><td rowspan=1 colspan=1>STEC 3X LoD {stx2}</td><td rowspan=1 colspan=1>-0.7</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>-0.4</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>-0.6</td></tr><tr><td rowspan=1 colspan=1>STEC 1 log above LoD {stx2}</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>-0.8</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>STEC 2 logs above LoD {stx2}</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>-0.4</td></tr><tr><td rowspan=1 colspan=1>STEC 3 logs above LoD {stx2}</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.2</td></tr><tr><td rowspan=1 colspan=1>STEC at LoD {stx1}</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>-1.6</td><td rowspan=1 colspan=1>-1.9</td><td rowspan=1 colspan=1>-1.0</td><td rowspan=1 colspan=1>-1.4</td></tr><tr><td rowspan=1 colspan=1>STEC 3X LoD {stx1}</td><td rowspan=1 colspan=1>-1.4</td><td rowspan=1 colspan=1>-0.7</td><td rowspan=1 colspan=1>-1.6</td><td rowspan=1 colspan=1>-0.6</td><td rowspan=1 colspan=1>-1.4</td></tr><tr><td rowspan=1 colspan=1>STEC 1 log above LoD {stx1}</td><td rowspan=1 colspan=1>-0.3</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>-1.0</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>-0.3</td></tr><tr><td rowspan=1 colspan=1>STEC 2 logs above LoD {stx1}</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>-0.6</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>-0.5</td></tr><tr><td rowspan=1 colspan=1>STEC 3 logs above LoD {stx1}</td><td rowspan=1 colspan=1>-0.4</td><td rowspan=1 colspan=1>-0.1</td><td rowspan=1 colspan=1>-0.5</td><td rowspan=1 colspan=1>0.1</td><td rowspan=1 colspan=1>-0.2</td></tr></table>

\*Test points that indicated potential stability issue

A paired t-test was performed on Ct values for each of five “non-fresh” conditions compared to FR ST/FR NA on the samples that tested positive (all samples tested positive at all testing points) for each PCR Mix.

<table><tr><td rowspan=1 colspan=15>Sample and Nucleic Acid Stability Results — Paired t-test to the FR ST/FR NA Testing Point</td></tr><tr><td rowspan=1 colspan=15>Time point  SSC Mix</td></tr><tr><td rowspan=1 colspan=1>FR ST/FR NA</td><td rowspan=1 colspan=3>FR ST/FZ NA</td><td rowspan=1 colspan=5>REF ST/FR NA*</td><td rowspan=1 colspan=2>REF ST/FZ NA</td><td rowspan=1 colspan=2>FZ ST/FR NA*</td><td rowspan=1 colspan=2>FZ ST/FZ NA*</td></tr><tr><td rowspan=2 colspan=1>NA</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=2>Mean CtDifference</td><td rowspan=1 colspan=3>p-value</td><td rowspan=1 colspan=2>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td></tr><tr><td rowspan=1 colspan=1>p=0.07</td><td rowspan=1 colspan=2>-0.2</td><td rowspan=1 colspan=3>p=0.000005</td><td rowspan=1 colspan=2>0.8</td><td rowspan=1 colspan=1>p=0.2</td><td rowspan=1 colspan=1>-0.2</td><td rowspan=1 colspan=1>p=0.0002</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>p=0.0003</td><td rowspan=1 colspan=1>1.2</td></tr><tr><td rowspan=1 colspan=1></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td rowspan=1 colspan=2>Time point  STEC Mix</td><td></td><td></td><td></td><td></td></tr><tr><td rowspan=1 colspan=1>FR ST/FR NA</td><td rowspan=1 colspan=4>FR ST/FZ NA*</td><td rowspan=1 colspan=3>REF ST/FR NA</td><td rowspan=1 colspan=3>REF ST/FZ NA*</td><td rowspan=1 colspan=2>FZ ST/FR NA</td><td rowspan=1 colspan=2>FZ ST/FZ NA*</td></tr><tr><td rowspan=2 colspan=1>NA</td><td rowspan=1 colspan=2>p-value</td><td rowspan=1 colspan=2>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=2>Mean CtDifference</td><td rowspan=1 colspan=2>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td><td rowspan=1 colspan=1>p-value</td><td rowspan=1 colspan=1>Mean CtDifference</td></tr><tr><td rowspan=1 colspan=2>p=0.03</td><td rowspan=1 colspan=2>-0.4</td><td rowspan=1 colspan=1>p=0.3</td><td rowspan=1 colspan=2>-0.2</td><td rowspan=1 colspan=2>p=0.002</td><td rowspan=1 colspan=1>-0.9</td><td rowspan=1 colspan=1>p=0.9</td><td rowspan=1 colspan=1>0.0</td><td rowspan=1 colspan=1>p=0.01</td><td rowspan=1 colspan=1>-0.6</td></tr></table>

The data in the table above show that there are statistically significant differences between the testing points indicated in the boxes with an “\*” and the original testing point (fresh stool and fresh nucleic acid). The “Mean Ct Difference to initial FR ST/FR NA Data” table shows that the largest mean difference is associated with the $C .$ . jejuni samples after they were frozen. A noticeable difference in the performance of simulated Campylobacter samples upon freezing has also been observed by comparing initial QC testing data for the Positive Matrix Control, and the Reproducibility samples. Even though statistically significant differences in Mean Ct do exist for each PCR Mix at different testing points via the paired t-test analysis, the ProGastro SSCS Assay is not a quantitative, but a qualitative assay. The fact that all samples tested positive for their correct target at all testing points meeting the requirement stated in the study plan (i.e., all samples must not test statistically significantly different $( \mathrm { p } { \le } 0 . 0 5 )$ from their fresh testing point in terms of positive/negative result) supports the claim that there are no significant negative effects due to freezing samples/nucleic acids. However, the observed negative effects of freezing samples with regard to Campylobacter detection are included in the “Sample Collection, Handling, and Storage” and the “Warnings and Precautions” sections of the package insert.

In conclusion, the results of this study demonstrate there is no significant negative effect on the ProGastro SSCS Assay in target organism detection due to storing samples at refrigerated temperature $\left( 2 \mathrm { - } 8 ^ { \circ } \mathrm { C } \right)$ for up to five days, samples undergoing a single freeze/thaw, or nucleic acids undergoing a single freeze/thaw. The data does show a marked, statistically significant difference in Mean Ct for the contrived $C .$ . jejuni samples once they are frozen and tested. This freezing effect had also been noted subsequent to performing this study. This phenomenon has also been described in literature in that Campylobacter is very sensitive to environmental stressors including freezing where it loses viability. In addition, the ProGastro SSCS Clinical Study data supports the use of frozen samples as the majority of Prospective Clinical Study samples were stored frozen prior to testing, and the performance of the ProGastro SSCS Assay is acceptable. This study results also support the use of frozen banked retrospective clinical samples in the evaluation of the ProGastro SSCS Assay to supplement the prospective clinical study data.

# Reagent Stability and Freeze/Thaw Study

An Accelerated Stability study concluded that the ProGastro SSCS Assay Mixes (SSC Mix and STEC Mix) and controls (closed and open tubes) can be stored at $\le - 7 0 ^ { \circ } \mathrm { C }$ for up to 18 months. Real time stability studies are currently ongoing at Gen-Probe Prodesse, Inc.

A Freeze/Thaw Study demonstrated that the SSC Mix and the STEC Mix can each undergo up to 10 freeze-thaws. The controls (Positive and Internal Controls) can undergo up to two freeze-thaws. For the controls, the ProGastro SSCS Assay Package Insert specifies that they should not undergo more than one freeze-thaw cycle, although internal studies demonstrated that up to two freeze-thaws of controls would not adversely affect performance.

# d. Detection limit:

To determine and confirm the Limit of Detection (LoD) of the ProGastro SSCS Assay on the Cepheid SmartCycler II, an analytical study was carried out using fresh bacterial cultures for each detection target (Salmonella, Shigella, Campylobacter ( $C .$ . jejuni and C. coli only), and Shiga Toxin producing $E .$ . coli, STEC). Analytical Sensitivity (LoD) is defined as the lowest concentration of bacteria detected $2 9 5 \%$ of the time.

Fresh bacterial cultures were used for both the LoD Determination and the LoD Confirmation studies, as well as plating for $\mathrm { C F U / m L }$ counting. The bacterial strains tested in the studies are outlined in the table below:

<table><tr><td rowspan=1 colspan=2>Limit of Detection Panel Strains</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>Strain ID</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Typhi</td><td rowspan=1 colspan=1>ATCC6539</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Typhimurium</td><td rowspan=1 colspan=1>BAA-1603</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Enteritidis</td><td rowspan=1 colspan=1>BAA-1045</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>ATCC9207</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>ATCC29027</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>ATCC12025</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>ATCC29029</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>BAA-224</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>ATCC43485</td></tr><tr><td rowspan=1 colspan=1>STEC O26:H/NM (stx1-/stx2+)</td><td rowspan=1 colspan=1>TW08569</td></tr><tr><td rowspan=1 colspan=1>STEC O157:H7 (stx1+/stx2-)</td><td rowspan=1 colspan=1>TW00975</td></tr></table>

The LoD Determination portion of this study included freshly cultured bacteria that were serially diluted, spiked into pooled negative stool matrix, and tested minimally at five concentrations: 1 log above, 0.5 log above, at, 0.5 logs below, and 1 log below the preliminary LoD level as predetermined during product development. Each bacterial strain was tested in quintuplicate extractions and real-time PCR reactions using either the SSC or STEC Mix (depending on the strain) for a total of five data points per bacterial concentration. LoD was estimated as the lowest concentration where 5/5

replicates were detected. The same bacterial dilutions were also cultured on the appropriate solid media for CFU/mL counting to enable calculation of LoDs in $\mathrm { C F U / m L }$ of stool and CFU/reaction. The estimated LoD for each strain was then confirmed by the generation of 20 independent samples/data points using the specific spiked stool concentration utilized during the LoD Determination portion of the study. For some of the strains, more than one concentration was included for the confirmation portion of the study, typically the two lowest concentrations that yielded $100 \%$ detection to ensure achievement of $2 9 5 \%$ detection for each strain. Each of the 20 replicates was subject to the entire test system from sample preparation and extraction to PCR with either the SSC or STEC Mix (depending on the strain). All samples were extracted using the bioMérieux NucliSENS easyMAG Instrument. In the event that the initial estimated LoD concentration was not confirmed (i.e., ${ < } 1 9$ replicates were positive), the LoD confirmation was repeated using the next half-log higher concentration. At least $9 5 \%$ (19/20) of the replicates were required to test positive to confirm the LoD for each bacterial target.

The table below outlines the results of the LoD Determination portion of the study. Concentrations in bold in the table below were tested during the LoD Confirmation portion of the study.

<table><tr><td colspan="6" rowspan="1">LoD Determination (Estimation) Results</td></tr><tr><td colspan="1" rowspan="1">Bacterial Strain</td><td colspan="1" rowspan="1">DilutionConc.</td><td colspan="1" rowspan="1">AverageCT</td><td colspan="1" rowspan="1">StandardDeviation</td><td colspan="1" rowspan="1">%CV</td><td colspan="1" rowspan="1">PCRReplicatesDetected</td></tr><tr><td colspan="1" rowspan="5">Salmonella enterica serotypeTyphi*</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">39.6</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">3/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">Salmonella enterica serotypeTyphimurium*</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">37.0</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">38.8</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">4.2</td><td colspan="1" rowspan="1">3/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">40.6</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1/5</td></tr><tr><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">Salmonella enterica serotypeEnteritidis</td><td colspan="1" rowspan="1">10-5</td><td colspan="1" rowspan="1">33.8</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5.5</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">37.2</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">3.1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.1</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">39.1</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="6">Shigella boydii</td><td colspan="1" rowspan="1">10-4.5</td><td colspan="1" rowspan="1">30.3</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5</td><td colspan="1" rowspan="1">31.6</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5.5</td><td colspan="1" rowspan="1">33.6</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">36.3</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">Shigella dysenteriae</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">36.9</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="4">Shigella flexneri</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">34.7</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.3</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2/5</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">Shigella sonnei</td><td colspan="1" rowspan="1">10-5</td><td colspan="1" rowspan="1">32.3</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5.5</td><td colspan="1" rowspan="1">33.4</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">35.8</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">2.1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">Campylobacter jejuni</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">39.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">40.1</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">42.0</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">3/5</td></tr><tr><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">43.6</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">3/5</td></tr><tr><td colspan="1" rowspan="5">Campylobacter coli</td><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">35.7</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">37.8</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7</td><td colspan="1" rowspan="1">39.1</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.7</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-7.5</td><td colspan="1" rowspan="1">40.1</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">2.2</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-8</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/5</td></tr><tr><td colspan="1" rowspan="5">STEC O26:H/NM (stx1-/stx2+)</td><td colspan="1" rowspan="1">10-4.5</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-$</td><td colspan="1" rowspan="1">36.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5.5</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">4/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">38.9</td><td colspan="1" rowspan="1">1.4</td><td colspan="1" rowspan="1">3.6</td><td colspan="1" rowspan="1">2/5</td></tr><tr><td colspan="1" rowspan="5">STEC O157:H7 (stx1+/stx2-)</td><td colspan="1" rowspan="1">10-4.5</td><td colspan="1" rowspan="1">32.7</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5</td><td colspan="1" rowspan="1">34.0</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-5.5</td><td colspan="1" rowspan="1">37.9</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">5/5</td></tr><tr><td colspan="1" rowspan="1">10-6.5</td><td colspan="1" rowspan="1">40.1</td><td colspan="1" rowspan="1">1.2</td><td colspan="1" rowspan="1">2.9</td><td colspan="1" rowspan="1">3/5</td></tr></table>

\* S. Typhi and S. Typhimurium were confirmed at $\%$ log higher dilutions in the LoD Confirmation portion of the study.

The results of the LoD Confirmation portion of the study are shown in the table below.   
Concentrations in bold are the confirmed LoD per strain.

<table><tr><td rowspan=1 colspan=9>LoD Confirmation Results</td></tr><tr><td rowspan=1 colspan=1>Bacterial Strain</td><td rowspan=1 colspan=1>DilutionConc.</td><td rowspan=1 colspan=1>AverageCT</td><td rowspan=1 colspan=1>StandardDeviation</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>MinCT</td><td rowspan=1 colspan=1>MaxCt</td><td rowspan=1 colspan=1>SampleReplicatesDetected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotypeTyphi</td><td rowspan=1 colspan=1>10-5.5</td><td rowspan=1 colspan=1>36.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotypeTyphimurium</td><td rowspan=1 colspan=1>10-5.5</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>36.5</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Salmonella enterica serotypeEnterritidis</td><td rowspan=1 colspan=1>10-6</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.4</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>18/20</td><td rowspan=1 colspan=1>90%</td></tr><tr><td rowspan=1 colspan=1>10-5.5</td><td rowspan=1 colspan=1>35.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>10-6.5</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>35.5</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>6/20</td><td rowspan=1 colspan=1>30%</td></tr><tr><td rowspan=1 colspan=1>10-6</td><td rowspan=1 colspan=1>35.2</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>10-6</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>36.6</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>10-7</td><td rowspan=1 colspan=1>36.1</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>35.0</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>17/20</td><td rowspan=1 colspan=1>85%</td></tr><tr><td rowspan=1 colspan=1>10-6.5</td><td rowspan=1 colspan=1>33.9</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>32.9</td><td rowspan=1 colspan=1>34.8</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>10-6</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>36.8</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>10-6.5</td><td rowspan=1 colspan=1>39.1</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>41.3</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>10-7</td><td rowspan=1 colspan=1>38.9</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>2.1</td><td rowspan=1 colspan=1>37.7</td><td rowspan=1 colspan=1>41.1</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=2 colspan=1>STEC O26:H/NM (stx1-/stx2+)</td><td rowspan=1 colspan=1>10-5.55$</td><td rowspan=1 colspan=1>37.5</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>36.7</td><td rowspan=1 colspan=1>38.6</td><td rowspan=1 colspan=1>19/20</td><td rowspan=1 colspan=1>95%</td></tr><tr><td rowspan=1 colspan=1>10-5r}$</td><td rowspan=1 colspan=1>35.8</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>35.1</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>STEC O157:H7 (stx1+/stx2-)</td><td rowspan=1 colspan=1>10-5.55</td><td rowspan=1 colspan=1>37.0</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>2.6</td><td rowspan=1 colspan=1>34.6</td><td rowspan=1 colspan=1>39.0</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

†One replicate required retesting in duplicate due to an initial result of “Unresolved” (thus generating 2 Ct values for the same sample) and a total of 20 data points.

In conclusion, the LoDs of the ProGastro SSCS Assay calculated as CFU/mL stool and CFU/reaction are summarized in table below:   

<table><tr><td rowspan=1 colspan=3>LoD Summary</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD concentration inCFU/mL stool</td><td rowspan=1 colspan=1>LoD concentration inCFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Typhi</td><td rowspan=1 colspan=1>1.63x103 CFU/mL</td><td rowspan=1 colspan=1>0.74 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotypeTyphimurium</td><td rowspan=1 colspan=1>2.25x104CFU/mL</td><td rowspan=1 colspan=1>10.21 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Salmonella enterica serotype Enteritidis</td><td rowspan=1 colspan=1>2.47x104CFU/mL</td><td rowspan=1 colspan=1>11.21 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Shigella boydii</td><td rowspan=1 colspan=1>6.60x102 CFU/mL</td><td rowspan=1 colspan=1>0.30 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=1>1.03x10³ CFU/mL</td><td rowspan=1 colspan=1>0.47 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Shigella flexneri</td><td rowspan=1 colspan=1>3.11x103CFU/mL</td><td rowspan=1 colspan=1>1.42 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Shigella sonnei</td><td rowspan=1 colspan=1>1.46x103CFU/mL</td><td rowspan=1 colspan=1>0.66 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Campylobacter jejuni</td><td rowspan=1 colspan=1>1.36x103CFU/mL</td><td rowspan=1 colspan=1>0.62 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>Campylobacter coli</td><td rowspan=1 colspan=1>1.99x10³ CFU/mL</td><td rowspan=1 colspan=1>0.91 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>STEC O26:H/NM (stx1-/stx2+)</td><td rowspan=1 colspan=1>9.27x103CFU/mL</td><td rowspan=1 colspan=1>4.21 CFU/reaction</td></tr><tr><td rowspan=1 colspan=1>STEC O157:H7 (stx1+/stx2-)</td><td rowspan=1 colspan=1>1.66x104CFU/mL</td><td rowspan=1 colspan=1>7.55 CFU/reaction</td></tr></table>

# e. Analytical Reactivity:

The Analytical Reactivity study was performed to determine whether the ProGastro SSCS Assay is able to detect a variety of strains (reactivity panel) that represent the genetic diversity of each of the targeted organisms. This study expanded upon the Analytical Sensitivity Study by determining whether different strains of the same organism (Salmonella, Shigella, Campylobacter and Shiga Toxins 1 and 2 $\boldsymbol { s t x l }$ and $s t x 2$ ) producing strains of E. coli) can be detected at similar concentrations, near the assay detection limit.

In addition to the eleven strains used for the LoD Study, the analytical reactivity of the ProGastro SSCS Assay was evaluated with multiple strains of bacteria. The strains were selected to include those isolated primarily from human infections (when available) and various geographical locations in order to incorporate the genetic variation that may be encountered by ProGastro SSCS users. A Limit of Detection (LoD) was estimated for most of the strains during pre-verification studies (Note: Salmonella bongori is not reactive and, thus, did not have a preliminary LoD). The strains used in this study were tested at $2 \mathbf { x }$ the estimated LoD, or at the highest concentration possible for the Salmonella bongori strain. One sample was generated for each strain by spiking cultured and quantified bacteria into aliquots of a ProGastro SSCS negative stool matrix pool. The Gastro RNA/DNA Internal Control (GIC) was added to each sample just prior to extraction on the bioMérieux NucliSENS easyMAG and each resultant nucleic acid sample was tested in triplicate PCR reaction on the Cepheid SmartCycler II (Dx Software Version 3.0).

Mean Cts and standard deviations for reactive strains were calculated and presented in the following table:

<table><tr><td colspan="8" rowspan="1">ProGastro SSCS Analytical Reactivity Study Results</td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">ConcentrationTested*</td><td colspan="3" rowspan="1">SSC Mix</td><td colspan="2" rowspan="1">STEC Mix</td></tr><tr><td colspan="1" rowspan="1">Campy.MeanCt±SD</td><td colspan="1" rowspan="1">Sal.MeanCt±SD</td><td colspan="1" rowspan="1">Shi.MeanCt±SD</td><td colspan="1" rowspan="1">stx2 MeanCt±SD</td><td colspan="1" rowspan="1">stx1 MeanCt±SD</td></tr><tr><td colspan="1" rowspan="1">Salmonella bongori 43975</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">9.25x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NotReactive</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Paratyphi 8759</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.8±0.9</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Typhimurium19585</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.5±0.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Typhimurium14028</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.7±0.6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. TyphimuriumBAA-189</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.9±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. TyphimuriumBAA-191</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.3±0.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. TyphimuriumBAA-215</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.3±0.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Enteritidis 13076</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">32.4±0.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Enteritidis BAA-708</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37.0±0.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">.</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Enteritidis 4931</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.8±0.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Enteritidis 6961</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.7±0.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Newport 6962</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.3±0.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Newport 27869</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x103 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">38.8±0.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Heidelberg 8326</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.7±0.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Javiana BAÅ-1593</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">38.3±0.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.enterica ser. Montevideo BAA-710</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.9±0.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Oranienburg 9239</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.9±0.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Saintpaul 9712</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.1±1.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Muenchen BAA-1674</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.9±1.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">ConcentrationTested*</td><td colspan="3" rowspan="1">SSC Mix</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">STEC Mix</td></tr><tr><td colspan="1" rowspan="1">Campy.MeanCt±SD</td><td colspan="1" rowspan="1">Sal.MeanCt±SD</td><td colspan="1" rowspan="1">Shi.MeanCt±SD</td><td colspan="1" rowspan="1">stx2 MeanCt±SD</td><td colspan="1" rowspan="1">stx1 MeanCt±SD</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Braenderup BAA-664</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.4±0.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Infantis BAA-1675</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.8±0.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Thompson 8391</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.3±0.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Agona BAA-707</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.1±0.7</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Bareilly (clinicalisolate)</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.6±0.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.Enterica ser. Hadar 51956</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.0±0.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.indica (genomic DNA) BAA-1578D5</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">20 fg/μL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37.8±0.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.salamae strain BAA-1582</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.arizonae</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.diarizonae</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Salmonella enterica subsp.houtenae</td><td colspan="1" rowspan="1">Salmonella</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella boydii 25930</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x10³3 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.4±0.4</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella dysenteriae 29026†</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x10³3 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.8±0.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">39.8±0.5</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri 12022</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.0±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri 25875</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32.3±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 29031</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.0±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 9290</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.6±0.6</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 11060</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">32.6±0.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 25931</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x10³ CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.5±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 29030</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">33.4±0.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="2">Strain</td><td colspan="1" rowspan="2">Target</td><td colspan="1" rowspan="2">ConcentrationTested*</td><td colspan="3" rowspan="1">SSC Mix</td><td colspan="2" rowspan="1">STEC Mix</td></tr><tr><td colspan="1" rowspan="1">Campy.MeanCtaSD</td><td colspan="1" rowspan="1">Sal.MeanCt±SD</td><td colspan="1" rowspan="1">Shi.MeanCt±D</td><td colspan="1" rowspan="1">stx2 MeanCt±SD</td><td colspan="1" rowspan="1">stx1 MeanCt±SD</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei 29930</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">33.5±0.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella flexneri 700930</td><td colspan="1" rowspan="1">Shigella</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">33.7±0.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni 29428</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x10³ CFU/mL</td><td colspan="1" rowspan="1">37.7±0.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni 33291</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x10³ CFU/mL</td><td colspan="1" rowspan="1">38.8±2.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni BAA-222</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x103 CFU/mL</td><td colspan="1" rowspan="1">38.8±0.3</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni BAA-223</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x103 CFU/mL</td><td colspan="1" rowspan="1">38.6±0.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni BAA-219</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x103 CFU/mL</td><td colspan="1" rowspan="1">38.8±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.jejuni BAA-220</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x107 cFU/mL</td><td colspan="1" rowspan="1">37.8±0.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni subsp.doylei BAA-1458</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x105CFU/mL</td><td colspan="1" rowspan="1">35.4±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli BAA-370</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">37.8±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli BAA-371</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">34.4±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli BAA-372</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1">36.6±0.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli 33559</td><td colspan="1" rowspan="1">Campylobacter</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1">35.1±0.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli DEC10B (O26:H11)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37.8±0.3</td></tr><tr><td colspan="1" rowspan="1">E. coli 97-3250 (O26:H11)</td><td colspan="1" rowspan="1">stx 1 &amp; stx 2</td><td colspan="1" rowspan="1">2x105CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">38.2±1.4</td><td colspan="1" rowspan="1">37.3±0.5</td></tr><tr><td colspan="1" rowspan="1">E. coli DA-21(045:H/NM)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">35.7±0.5</td></tr><tr><td colspan="1" rowspan="1">E. coli MI03-19 (045:H2)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.5±0.6</td></tr><tr><td colspan="1" rowspan="1">E. coli MT#80 (O103:H2)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">1x105CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37.5±1.3</td></tr><tr><td colspan="1" rowspan="1">E. coli 3215-99 (0111:H8)</td><td colspan="1" rowspan="1">stx 1 &amp; stx 2</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37.7±0.8</td><td colspan="1" rowspan="1">36.6±0.8</td></tr><tr><td colspan="1" rowspan="1">E. coli RD8 (H111:O2)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x105 CcFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.6±0.2</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli 0201 9611 (O111:H11)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">36.8±0.4</td></tr><tr><td colspan="1" rowspan="1">Campy.MeanCt±SD</td><td colspan="1" rowspan="1">Sal.MeanCt±SD</td><td colspan="1" rowspan="1">Shi.MeanCt±SD</td><td colspan="1" rowspan="1">stx2 MeanCt±SD</td><td colspan="1" rowspan="1">stx1 MeanCt±SD</td></tr><tr><td colspan="1" rowspan="1">E. coli DA-5(0121:H19)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">37.3±1.0</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli DA-1(0121:H/NM)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">38.9±1.4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli GS G5578620(0145:H/NM)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">39.0±1.0</td></tr><tr><td colspan="1" rowspan="1">E. coli IH 16 (0145:H/NT)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.3±0.4</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">E. coli 7:85 (0157:H/N)</td><td colspan="1" rowspan="1">stx 1 &amp; stx 2</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.9±1.3</td><td colspan="1" rowspan="1">36.5±0.7</td></tr><tr><td colspan="1" rowspan="1">E. coli 93-111 (0157:H7)</td><td colspan="1" rowspan="1">stx 1 &amp; stx 2</td><td colspan="1" rowspan="1">2x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.4±0.9</td><td colspan="1" rowspan="1">37.7±0.5</td></tr><tr><td colspan="1" rowspan="1">E. coli DA-34 (O157:H/NM)</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">39.8±0.7</td></tr><tr><td colspan="1" rowspan="1">E. coli EDL933 (0157:H7)</td><td colspan="1" rowspan="1">stx 1 &amp; stx 2</td><td colspan="1" rowspan="1">2x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.5±0.5</td><td colspan="1" rowspan="1">35.9±0.6</td></tr><tr><td colspan="1" rowspan="1">E. coli 1:361 (0157:H7)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.8±0.3</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli DA-54 (O157:H/NM)</td><td colspan="1" rowspan="1">stx 2</td><td colspan="1" rowspan="1">2x105 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.0±0.4</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli O104:H4 GenomicDNA‡ (0104:H4)</td><td colspan="1" rowspan="1">stx2</td><td colspan="1" rowspan="1">0.5pg/μL(2.5pg/rxn)</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">34.5±0.1</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">E. coli O118:H16, EK36TW08644</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">36.8±0.5</td></tr><tr><td colspan="1" rowspan="1">E. coli O118:H-, RW2030TW06407</td><td colspan="1" rowspan="1">stx1</td><td colspan="1" rowspan="1">1x104 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">37.4±0.8</td></tr><tr><td colspan="1" rowspan="1">E. coli O118:H16, RW1191TW07879</td><td colspan="1" rowspan="1">stx1 and stx2</td><td colspan="1" rowspan="1">1x104 cFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">35.0±0.2</td><td colspan="1" rowspan="1">33.4±0.5</td></tr></table>

\* If more than one concentration was tested, the lowest concentration that tested positive for 3/3 reactions was reported. † Shigella dysenteriae strains typically contain a Shiga Toxin, therefore this strain tested positive with both the STEC and SSC Mixes. ¥ Genomic DNA was used to test the reactivity of the SSC and STEC Mixes towards the enterohemorragic $E$ . coli (EHEC) strain associated with the outbreak in Europe during the Spring/Summer of 2011.

In silico analysis of a single sequenced strain available for Salmonella enterica subsp. Enterica serotype Schwarzengrund demonstrated $100 \%$ homology of the ProGastro SSCS Assay Salmonella oligonucleotide primers and probes with the Schwarzengrund strain.

In conclusion, all of the strains used for this study with the exception of Salmonella bongori, Salmonella enterica subsp. salamae, Salmonella enterica subsp. arizonae, Salmonella enterica subsp. diarizonae, and Salmonella enterica subsp. houtenae tested positive with the ProGastro SSCS Assay. All these exceptions are documented as non-reactive strains in the ProGastro SSCS Assay Instructions for Use (Package Insert) as limitations.

Shigella dysenteriae strains typically contain a Shiga Toxin, therefore this strain tested positive with both the STEC and SSC Mixes. This is documented in the ProGastro SSCS Assay Instructions for Use (Package Insert) as a limitation: “Shigella dysenteriae strains typically contain a Shiga Toxin, but not always. A positive call of STEC stx1/stx2 may be from either STEC or Shigella dysenteriae”.

f. Analytical Specificity/Cross-reactivity Evaluation:

To determine the analytical specificity of the ProGastro SSCS Assay, an analytical cross-reactivity evaluation was carried out using cultured and titered strains of common gastrointestinal pathogens that are genetically related, cause similar disease states as the ProGastro SSCS Assay target organisms (Salmonella, Shigella, Campylobacter and Shiga Toxin producing $E .$ . coli (STEC)), or are commonly found in stool.

Analytical specificity is defined as an assay’s ability to exclusively identify the target organisms while not cross-reacting with other organisms in a sample. The Analytical Specificity of the ProGastro SSCS Assay was determined with a panel of 54 organisms. Cyclospora cayetanensis was subject to an in silico analysis only because it was not available for “wet” testing.

All the ProGastro SSCS Assay target organisms (Salmonella, Shigella, Campylobacter jejuni, Campylobacter coli, and STEC) in the specificity panel were serially diluted in Stool Preservation and Transport Media (SPTM) and spiked into a ProGastro SSCS negative stool matrix pool at high concentrations of $1 0 ^ { 6 } – 1 0 ^ { 7 } \mathrm { C F U / m L }$ . This was done to test the specificity of each mix for the target organisms and to also demonstrate that the assay is functioning as expected. The remaining members of the Specificity Panel were spiked into negative stool matrix pool at concentrations of 103.5 – 107.5 $\mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ for viral targets and $1 0 ^ { 6 } - { \dot { 8 } } . 8 \mathrm { x } 1 0 ^ { 8 } \mathrm { C F U / m L }$ for bacterial and fungal targets. The Specificity Panel Organisms were not diluted prior to spiking into stool in order to test them at the highest concentration possible. Norovirus was only available in the form of a positive sample (raw stool) obtained from a Public Health Laboratory. This sample was diluted in SPTM according to the manufacturer’s instructions and then processed according to the ProGastro SSCS Instructions for use.

Analytical Specificity Study results are presented in the table below. In silico analysis of the Cyclospora cayetanensis genome showed that each primer and probe included with the mixes had no similarity to the organism.

<table><tr><td colspan="7" rowspan="1">Analytical Specificity Study Results (Mean and Standard Deviation)</td></tr><tr><td colspan="1" rowspan="1">Organism</td><td colspan="1" rowspan="1">Conc. Tested</td><td colspan="1" rowspan="1">Campy.Detection</td><td colspan="1" rowspan="1">SalmonellaDetection</td><td colspan="1" rowspan="1">ShigellaDetection</td><td colspan="1" rowspan="1">stx 1Detection</td><td colspan="1" rowspan="1">stx 2Detection</td></tr><tr><td colspan="7" rowspan="1">Bacteria</td></tr><tr><td colspan="1" rowspan="1">Salmonella Enteritidis</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">29.6±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter jejuni</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">28.3±0.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter coli</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">31.5±0.1</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Shigella sonnei</td><td colspan="1" rowspan="1">1x106 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">27.9±0.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">STEC O157:H7Strain 93-111</td><td colspan="1" rowspan="1">1x107 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">26.7±0.1</td><td colspan="1" rowspan="1">27.8±0.1</td></tr><tr><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.5x107 cFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.5x107 cFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter lari</td><td colspan="1" rowspan="1">2.5x10-2 Dilutionfrom Raw StoolClinical Specimen*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">6.4x107 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacterhyointestinalis</td><td colspan="1" rowspan="1">7.44x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">5.4x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">7.0x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">2.4x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">1.0x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">4.5x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">3.55x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">2.0x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">4.3x106 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">4.2x106 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">4.8x108 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium difficileToxigenic Layola-02 Nap1</td><td colspan="1" rowspan="1">1.5x107 cFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.5x107 cFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli (non-STEC)</td><td colspan="1" rowspan="1">1.5x107 cFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia coli(enteroinvasive)</td><td colspan="1" rowspan="1">2.2x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">21.6±0.1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">2.0x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">8.8x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">5.6x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.14x108 CFU/mL</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">4.2x106 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">3.2x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.80x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">5.6x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">8.6x108 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">1.5x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">3.3x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="4" rowspan="1">Viruses</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">1.0x105.5TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">5.0x104.5(1.58x105)TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">1.0x106.5TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">1.0x107.5TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">1.0x103.5TCID50/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">2.5x10-2 Dilutionfrom Raw StoolClinicalSpecimen**</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="7" rowspan="1">Fungi</td></tr><tr><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.66x107 CFU/mL</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="6" rowspan="1">Parasites</td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Blastocystis hominis JNS</td><td colspan="1" rowspan="1">10-1 Dilution ofstock</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Giardia lamblia (Intestinalis)</td><td colspan="1" rowspan="1">10-1 Dilution ofstock</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">10-1 Dilution ofstock</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Entamoeba histolytica MH-1:IMSS</td><td colspan="1" rowspan="1">10-1 Dilution ofstock</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr></table>

\* Clinical specimen received and tested was stool preserved in stool preservation and transport medium and processed per the ProGastro SSCS Assay Instructions for Use. Sample was culture positive for Campylobacter and was confirmed as $C .$ . lari by 16S genetic sequencing. $^ { \ast \ast }$ Cultured and titered Norovirus was unavailable; nucleic acids from a positive clinical sample (Milwaukee City Public Health Lab Real Time PCR assay with a Ct valu ${ : = } 2 0 . 5$ ) were tested.

In conclusion, the ProGastro SSCS Assay did not react with any of the organisms listed in the table above, other than enteroinvasive Escherichia coli (EIEC). EIEC is genetically very similar to Shigella and was detected by the SSC Mix as positive for Shigella as expected. This cross-reactivity is reported in the Package Insert as a limitation. No cross-reactivity was observed between the two Mixes (SSC and STEC). The ProGastro SSCS Assay demonstrates no cross reactivity with the organisms that are commonly found in stool, genetically related or cause similar disease states as the ProGastro SSCS Assay target organisms.

# g. Assay cut-off:

To establish the cutoff threshold value of all targets of the ProGastro SSCS Assay on the Cepheid SmartCycler II, negative stool samples and spiked stool samples containing varying levels of Salmonella, Shigella, Campylobacter jejuni, Campylobacter coli and Shiga Toxin producing $E$ . coli (STEC, Shiga Toxin 1 and Shiga Toxin 2 genes) were used. The ProGastro SSCS Assay is comprised of two PCR mixes, one used to detect and differentiate Salmonella, Shigella and Campylobacter (C. jejuni and C. coli only, not differentiated) (SSC Mix) while the other detects and differentiates the Shiga Toxin $1 \ : ( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes in bacteria such as Shiga Toxin producing $E$ . coli (STEC Mix). The accuracy of the chosen Relative Fluorescent Units (RFU) threshold for each channel was confirmed with retrospective clinical samples with the objective to obtain $\ge 9 0 \%$ positive and negative agreement to reference methods.

The “cutoff value” represents the signal level, reported in RFU, at which a “positive” reaction rises significantly above the background or baseline of a “negative” reaction. If a sample exceeds the threshold in a detection channel during PCR, the sample is considered positive for that channel. If the sample does not exceed the threshold for a detection channel by the last PCR cycle, the sample is considered negative for that channel.

Sources of variability incorporated into the Cutoff Determination and Confirmation Study included: operators, unique samples, multiple bacterial and control dilutions, extractions and reagent lots. Cutoff values were determined upon completion of the Cutoff Determination Study which included testing of spiked stool samples and controls. The cutoff values were then verified in the Cutoff Confirmation Study against a set of retrospective clinical samples and controls.

# Cutoff Determination Study:

The cutoff value of the ProGastro SSCS Assay was determined with a panel of negative and spiked samples prepared from 82 individual negative stool clinical samples that were tested by each of the two operators $\scriptstyle ( \ n = 1 6 4 )$ . Individual samples (n $= 1 2$ ) were spiked with each ProGastro SSCS target bacteria (Salmonella, Shigella, $C .$ . jejuni, C. coli, stx1 and stx 2) at three logs above the preliminary Limit of Detection (LoD) concentration representing high positive stool samples. Individual samples $( { \mathfrak { n } } =$ 36) were spiked with each of the six ProGastro SSCS targets at two times the

preliminary LoD concentration representing low positive stool samples. Thirty four (34) samples were left un-spiked (negatives).

The high and low positive samples, as well as 30 of the negative samples were spiked with the Gastro Internal Control (GIC). The GIC monitors for PCR inhibition as well as any reagent, procedural or instrumentation failure. Four negative samples were not spiked with the GIC.

Replicates of the SSCS Control and C. coli Control (SSC Mix only) were included with each PCR run to test for global errors (absence of reagents, instrument failure, etc.). The SSCS Control and C. coli Positive Controls did not require nucleic acid isolation and were diluted in molecular grade water just prior to set up of the PCR reactions. Six replicates of each positive control were included during set-up of the PCR reactions ( $C .$ . coli Control only used in conjunction with the SSC Mix). Six Negative Controls (NC) consisting of Stool Preservation and Transport Media (SPTM, Para-Pak C&S) spiked with GIC were also tested. The NC served to monitor for contamination during the testing procedure.

Samples were extracted using the bioMérieux NucliSENS easyMAG. The extracted nucleic acids and the ProGastro SSCS and C. coli Controls, as well as GIC diluted in water were run using two different lots of both the SSC and STEC Mixes contained in the ProGastro SSCS Assay on the Cepheid SmartCycler II.

Initial cutoff values used for the Cutoff Determination Study were based on preliminary testing during product development.

Data analysis of the Cutoff Determination Study included 1) calculation of lower limit of the acceptable range for cutoff threshold (RFU) based on negative samples; and 2) calculation of upper limit of the acceptable range for cutoff threshold (RFU) based on spiked positive samples. For a given detection, a range of possible cutoffs was generated by determining lower and upper boundaries of the threshold, based on final RFUs that would minimize false positives and false negatives. A Receiver Operator Characteristics (ROC) analysis was performed for all of the channels utilized by the ProGastro SSCS Assay’s SSC and STEC Mixes. In addition, population distribution and statistics were analyzed to aid in selecting a cutoff threshold. Histograms of the final RFU values were used to assess the nature of the distributions and the percentile rankings.

Acceptable ranges for the potential cutoff were generated in the following manners:

The distributions of the negatives and positives were first assessed for normality. If the population distributions were normal, the averages and standard deviations of the final RFUs were calculated for all detections. For the negatives, the lower limit of the acceptable range for cutoff threshold is defined as the average final RFU plus three standard deviations, that is, the point at which $9 9 . 7 \%$ of true negatives are expected to fall below assuming normality. Similarly, for the

positives, the upper limit of the acceptable range for cutoff threshold is defined as the average final RFU minus three standard deviations. This is the point at which at least $9 9 . 7 \%$ of true positives are expected to be greater than, assuming normality.

Population statistics were also analyzed for each detection channel. The lower limit of the cutoff threshold range is defined minimally as the $9 9 . 5 \%$ percentile from the quantile analysis for negative samples. This is the RFU value at which at least $9 9 . 5 \%$ of true negatives are expected to be lower than. For the positives, the upper limit of the cutoff threshold range is defined maximally as the $0 . 5 \%$ percentile. This is the RFU value at which at least $9 9 . 5 \%$ of true positives are expected to be greater than.

A ROC curve analysis was generated and assessed for all channels used with the ProGastro SSCS Assay’s SSC and STEC Mixes. ROC curves for the SSC Mix (Campylobacter (FAM), Salmonella (TET), Shigella (TxRed), and GIC (CY5)) and the STEC Mix (stx2 (FAM), stx1 (TET), and GIC (CY5)) cutoffs were generated. The threshold range that yielded the maximum accuracy was identified.

The cutoff threshold ranges determined by statistical analysis were used in conjunction with all data to select the cutoff settings. The following tables present the cutoff ranges and settings chosen from the Cutoff Determination Study for the SSC and STEC Mixes.

Preliminary Cutoff Threshold Values for SSC Mix   

<table><tr><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>EndPt Threshold (RFU)</td></tr><tr><td rowspan=1 colspan=1>Preliminary CutoffUsed*</td><td rowspan=1 colspan=1>Cutoff Range</td><td rowspan=1 colspan=1>Cutoff Value Chosenfrom the CutoffDetermination Study</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>60</td><td rowspan=1 colspan=1>14-77</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>30</td><td rowspan=1 colspan=1>27-357</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>TxR</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>25</td><td rowspan=1 colspan=1>3-47</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Gastro InternalControl</td><td rowspan=1 colspan=1>32</td><td rowspan=1 colspan=1>15-54</td><td rowspan=1 colspan=1>35</td></tr></table>

\*established during assay development

Preliminary Cutoff Threshold Values for STEC Mix   

<table><tr><td rowspan=2 colspan=1>Channel</td><td rowspan=2 colspan=1>Target</td><td rowspan=1 colspan=3>EndPt Threshold (RFU)</td></tr><tr><td rowspan=1 colspan=1>Preliminary CutoffUsed*</td><td rowspan=1 colspan=1>Cutoff Range</td><td rowspan=1 colspan=1>Cutoff Value Chosenfrom the CutoffDetermination Study</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>70</td><td rowspan=1 colspan=1>12-314</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>40</td><td rowspan=1 colspan=1>14-387</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Gastro InternalControl</td><td rowspan=1 colspan=1>35</td><td rowspan=1 colspan=1>3-143</td><td rowspan=1 colspan=1>35</td></tr></table>

\*established during assay development

# Cutoff Confirmation Study:

The Cutoff Confirmation Study goals were to achieve ${ \ge } 9 0 $ Percent Positive and ${ \ge } 9 0 $ Percent Negative Agreement of the ProGastro SSCS Assay using the preliminary cutoff values chosen after the Cutoff Determination Study as compared to the reference method results using retrospective positive and negative clinical stool samples.

Data analysis of the Cutoff Confirmation Study included 1) calculation of percent positive agreement for all retrospective ProGastro SSCS Assay positive clinical samples; and 2) calculation of percent negative agreement for all retrospective ProGastro SSCS Assay negative clinical samples.

The results of the Cutoff Confirmation Study are presented in the table below:

<table><tr><td rowspan=1 colspan=6>Cutoff Confirmation Study Final Results</td></tr><tr><td rowspan=1 colspan=1>Mix</td><td rowspan=1 colspan=1>Detection</td><td rowspan=1 colspan=1>% PositiveAgreement</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td><td rowspan=1 colspan=1>% NegativeAgreement</td><td rowspan=1 colspan=1>95% ConfidenceInterval</td></tr><tr><td rowspan=3 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3-100%</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td><td rowspan=1 colspan=1>98.4%</td><td rowspan=1 colspan=1>91.7-99.7%</td></tr><tr><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3-100%</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>83.9-100%</td><td rowspan=1 colspan=1>100%</td><td rowspan=1 colspan=1>94.3-100%</td></tr></table>

The following tables present the final settings chosen after the Cutoff Confirmation Study for the SSC and STEC Mixes:

Final Cutoff Threshold Values for SSC Mix   

<table><tr><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Final Cutoff Value(EndPt Threshold inRFU) after the CutoffConfirmation Study</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>60</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>35</td></tr><tr><td rowspan=1 colspan=1>TxR</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Gastro InternalControl</td><td rowspan=1 colspan=1>35</td></tr></table>

Final Cutoff Threshold Values for STEC Mix   

<table><tr><td rowspan=1 colspan=1>Channel</td><td rowspan=1 colspan=1>Target</td><td rowspan=1 colspan=1>Final Cutoff Value(EndPt Threshold inRFU) after the CutoffConfirmation Study</td></tr><tr><td rowspan=1 colspan=1>FAM</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>70</td></tr><tr><td rowspan=1 colspan=1>TET</td><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>40</td></tr><tr><td rowspan=1 colspan=1>Cy5</td><td rowspan=1 colspan=1>Gastro InternalControl</td><td rowspan=1 colspan=1>35</td></tr></table>

# Additional Cutoff Validation Studies:

The final cutoff values confirmed in the Cutoff Confirmation Study were further validated during the clinical studies and all subsequent analytical studies.

# h. Internal Control Interference:

Competitive inhibition of the ProGastro SSCS Assay due to the presence of the Gastro RNA/DNA Internal Control (GIC) was assessed in an analytical study. Simulated samples were tested with and without the GIC to determine if the presence of the GIC affected the detection of the Assay target organisms. For this study, one cultured and titered strain each of Salmonella, Shigella, Campylobacter jejuni, Campylobacter coli, and Shiga Toxin producing Escherichia coli (STEC, $s t x I + / s t x 2 + )$ were diluted and individually spiked into a negative stool pool at 2x and at 1 log below each strain’s Limit of Detection (LoD) as estimated in the Analytical Reactivity Study. Shigella was also generated and tested at 2 logs below LoD. All dilution series were extracted in triplicate on the bioMérieux NucliSENS easyMAG.

The strains used for the GIC Interference Study are listed below:

• Salmonella Enteritidis ATCC 6961 (stock concentration $1 . 0 6 { \times } 1 0 ^ { 9 } \mathrm { C F U / m L } )$ $\bullet$ Shigella flexneri ATCC 12022 (stock concentration $9 . 0 { \mathrm { x } } 1 0 ^ { 9 } \mathrm { C F U / m L } )$ • Campylobacter jejuni ATCC 29428 (stock concentration $1 . 1 0 { \mathrm { x } } 1 0 ^ { 8 } \mathrm { C F U / m L ) }$ • Campylobacter coli ATCC BAA-371 (stock concentration $7 . 2 0 { \times } 1 0 8 \mathrm { C F U / m L } )$ • STEC $( s t x I + / s t x 2 + )$ STEC Center Accession# TW07926 (stock concentration $1 . 7 5 \mathrm { x } 1 0 ^ { 9 } \mathrm { C F U / m L } )$

The extracted nucleic acids were tested by PCR in triplicate using the ProGastro SSCS Assay reagents on the Cepheid SmartCyler II. This resulted in a total of nine data points for each strain at each concentration spiked with the GIC and nine data points for each strain at each concentration without the GIC.

For the GIC to be considered “non-interfering”, the following criteria were required to be met:

• The LoD of samples without GIC must be within one log of the samples with GIC. • The mean Cts for concentrations with and without IC must differ by less than 3.3 Cts for all concentrations tested.

All data were analyzed and Mean Cts, Standard Deviation, $\%$ Coefficient of Variance, and the number of positive results were calculated for each strain at each concentration. The Mean Ct difference between the samples with and without GIC at each concentration was also calculated (See the table below).

<table><tr><td rowspan=1 colspan=11>GIC Interference Results</td></tr><tr><td rowspan=2 colspan=1>Strain</td><td rowspan=2 colspan=1>Conc.CFU/mL</td><td rowspan=1 colspan=4>Samples containing GIC</td><td rowspan=1 colspan=3>Samples without GIC</td><td rowspan=1 colspan=1></td><td rowspan=2 colspan=1>Mean CtDifference(GIC - NoGIC)</td></tr><tr><td rowspan=1 colspan=1>No.Positive</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td><td rowspan=1 colspan=1>No.Positive</td><td rowspan=1 colspan=1>MeanCt</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>%CV</td></tr><tr><td rowspan=2 colspan=1>Salmonella EnteritidisATCC 6961 1.06x109CFU/mL</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>1.0</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.7</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>5/9</td><td rowspan=1 colspan=1>39.3</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.2</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=2 colspan=1>Shigella flexneri ATCC12022 9.0x10° CFU/mL</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.1</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>0.0</td></tr><tr><td rowspan=1 colspan=1>1x10^3</td><td rowspan=1 colspan=1>0/9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>0/9</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=2 colspan=1>Campylobacter jejuniATCC 29428 1.10x108CF/mL</td><td rowspan=1 colspan=1>2x103</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>37.2</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>-0.4</td></tr><tr><td rowspan=1 colspan=1>1x102</td><td rowspan=1 colspan=1>7/9</td><td rowspan=1 colspan=1>40.7</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>1.9</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>41.2</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.5</td><td rowspan=1 colspan=1>-0.5</td></tr><tr><td rowspan=3 colspan=1>Campylobacter coliATCC BAA-3717.20x10° CFU/mL</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.0</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>1.1</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>34.4</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.8</td><td rowspan=1 colspan=1>-0.4</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>37.4</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>37.6</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.3</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>1x102</td><td rowspan=1 colspan=1>4/9</td><td rowspan=1 colspan=1>40.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>40.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>2.3</td><td rowspan=1 colspan=1>-0.3</td></tr><tr><td rowspan=2 colspan=1>STEC (stx1+/stx2+)Acc# TW07926 1.75x109CFU/mLstx 2 Detection</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>35.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>0.9</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>36.3</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>-1.0</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>8/9</td><td rowspan=1 colspan=1>39.5</td><td rowspan=1 colspan=1>1.2</td><td rowspan=1 colspan=1>3.0</td><td rowspan=1 colspan=1>6/9</td><td rowspan=1 colspan=1>39.9</td><td rowspan=1 colspan=1>0.7</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>-0.4</td></tr><tr><td rowspan=2 colspan=1>STEC (stx1+/stx2+)Acc# TW07926 1.75x109CFU/mLstx 1 Detection</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>35.4</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.8</td><td rowspan=1 colspan=1>9/9</td><td rowspan=1 colspan=1>35.6</td><td rowspan=1 colspan=1>0.6</td><td rowspan=1 colspan=1>1.7</td><td rowspan=1 colspan=1>-0.2</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>7/9</td><td rowspan=1 colspan=1>39.6</td><td rowspan=1 colspan=1>2.0</td><td rowspan=1 colspan=1>5.0</td><td rowspan=1 colspan=1>6/9</td><td rowspan=1 colspan=1>40.1</td><td rowspan=1 colspan=1>1.4</td><td rowspan=1 colspan=1>3.4</td><td rowspan=1 colspan=1>-0.5</td></tr></table>

No competitive inhibition near the assay LoD was observed for detection of Salmonella, Shigella, Campylobacter jejuni, Campylobacter coli, and STEC $( s t x I + / s t x 2 + )$ in the presence GIC using the ProGastro SSCS Assay. The assay demonstrated equivalent detection of LoD concentrations for the different target strains with Mean $\mathrm { C t }$ differences for each strain and concentration (with and without the GIC) ranging from -1.0 to 0.0.

# i. Interfering Substances:

An analytical study was carried out to evaluate and assess the potential inhibitory effects of exogenous and endogenous substances that may be present in stool samples on the ProGastro SSCS Assay.

This study examined whether a panel of potential inhibitory endogenous and exogenous substances (The Interfering Substances Panel) affects the performance of the ProGastro SSCS Assay. The panel consisted of endogenous substances (blood, mucus, and fecal fats), over the counter and prescription medicines (antacids, laxatives, anti-diarrheal medications, antibiotics, etc.), and miscellaneous substances (such as spermicidal lubricant and moist towelette residue) that could potentially be found in stool samples. Twenty (20) aliquots of negative stool matrix were spiked with a single cultured and titered strain of Campylobacter coli (CC), 20 aliquots were spiked with a single strain of STEC $( s t x I + / s t x 2 + )$ , 20 aliquots were dual-spiked with a single cultured and titered strain of Salmonella (Sal) and a single strain of Campylobacter jejuni (CJ), and 20 aliquots were spiked with a single strain Shigella (Shi) only. All organisms were tested at a final concentration of $2 \mathbf { x }$ Limit of Detection (LoD). Each interfering substance was then spiked into a single aliquot each of CC spiked stool, STEC spiked stool, Sal/CJ spiked stool, and Shi spiked stool. Interfering substances were added at high clinically relevant concentrations. One aliquot from each spiked sample type was not spiked with any interference substance for comparison purposes (control).

The Interfering Substances Panel samples and the controls were extracted on the bioMériuex NucliSENS easyMAG and samples tested in triplicate on a Cepheid Smartcycler II using one lot of ProGastro SSCS reagents.

Interfering Substances results are presented in the table below:

<table><tr><td rowspan=1 colspan=8>Mean/Standard Deviations of Ct Values for All Targets</td></tr><tr><td rowspan=2 colspan=1>Substance #</td><td rowspan=2 colspan=1>Substance BrandName</td><td rowspan=2 colspan=1>C. coliMean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=1 colspan=1>C. jejuniMean ± SD</td><td rowspan=1 colspan=1>SalmonellaMean ± SD</td><td rowspan=1 colspan=1>ShigellaMean ± SD</td><td rowspan=1 colspan=1>stx1Mean ± SD</td><td rowspan=2 colspan=1>stx2Mean ± SD(Mean CtDifference:Substance No substance)</td></tr><tr><td rowspan=1 colspan=1>(Mean CtDifference:Substance No substance)</td><td rowspan=1 colspan=1>(Mean CtDifference:Substance No substance)</td><td rowspan=1 colspan=1>(Mean CtDifference:Substance No substance)</td><td rowspan=1 colspan=1>(Mean CtDifference:Substance No substance)</td></tr><tr><td rowspan=1 colspan=1>NS (NoSubstanceControl</td><td rowspan=1 colspan=1>NA</td><td rowspan=1 colspan=1>34.6 ± 0.4</td><td rowspan=1 colspan=1>37.1 ± 0.6</td><td rowspan=1 colspan=1>35.0 ± 0.2</td><td rowspan=1 colspan=1>34.9 ± 0.3</td><td rowspan=1 colspan=1>35.8 ± 0.4</td><td rowspan=1 colspan=1>36.3 ± 0.9</td></tr><tr><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>MycostatinVaginal Cream</td><td rowspan=1 colspan=1>34.7 ± 0.1(0.1)</td><td rowspan=1 colspan=1>37.4 ± 0.1(0.3)</td><td rowspan=1 colspan=1>34.9 ± 0.4(-0.1)</td><td rowspan=1 colspan=1>35.0 ± 0.3(0.1)</td><td rowspan=1 colspan=1>35.5 ± 0.5(-0.3)</td><td rowspan=1 colspan=1>35.8 ± 0.4(-0.5)</td></tr><tr><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>Cortizone 10</td><td rowspan=1 colspan=1>34.4 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>37.0 ± 0.4(-0.1)</td><td rowspan=1 colspan=1>34.8 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>34.7 ± 0.4(-0.2)</td><td rowspan=1 colspan=1>36.1 ± 0.6(0.3)</td><td rowspan=1 colspan=1>36.0 ± 1.0(-0.3)</td></tr><tr><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>Preparation H</td><td rowspan=1 colspan=1>34.5 ± 0.1(-0.1)</td><td rowspan=1 colspan=1>37.4 ± 0.4(0.3)</td><td rowspan=1 colspan=1>34.9 ± 0.7(-0.1)</td><td rowspan=1 colspan=1>34.7 ± 0.4(-0.2)</td><td rowspan=1 colspan=1>36.4 ± 0.2(0.6)</td><td rowspan=1 colspan=1>36.2 ± 0.5(-0.1)</td></tr><tr><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>TUMS</td><td rowspan=1 colspan=1>34.5 ± 0.2(-0.1)</td><td rowspan=1 colspan=1>36.9 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>35.3 ± 0.2(0.3)</td><td rowspan=1 colspan=1>34.5 ± 0.6(-0.4)</td><td rowspan=1 colspan=1>35.7 ± 0.7(-0.1)</td><td rowspan=1 colspan=1>36.7 ± 0.4(0.4)</td></tr><tr><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>AmphojelSuspension</td><td rowspan=1 colspan=1>34.6 ± 0.2(0.0)</td><td rowspan=1 colspan=1>37.9 ± 0.8(0.8)</td><td rowspan=1 colspan=1>35.5 ± 0.8(0.5)</td><td rowspan=1 colspan=1>34.9 ± 0.1(0.0)</td><td rowspan=1 colspan=1>36.2 ± 0.8(0.4)</td><td rowspan=1 colspan=1>36.4 ± 0.3(0.1)</td></tr><tr><td rowspan=1 colspan=1>6</td><td rowspan=1 colspan=1>Phillips Milk ofMagnesia</td><td rowspan=1 colspan=1>34.0 ± 0.2(-0.6)</td><td rowspan=1 colspan=1>36.9 ± 0.4(-0.2)</td><td rowspan=1 colspan=1>34.5 ± 0.3(-0.5)</td><td rowspan=1 colspan=1>34.9 ± 0.1(0.0)</td><td rowspan=1 colspan=1>35.9 ± 0.4(0.1)</td><td rowspan=1 colspan=1>36.6 ± 0.9(0.3)</td></tr><tr><td rowspan=1 colspan=1>7</td><td rowspan=1 colspan=1>CVS Mineral OilEnema</td><td rowspan=1 colspan=1>34.1 ± 0.2(-0.5)</td><td rowspan=1 colspan=1>36.8 ± 0.2(-0.3)</td><td rowspan=1 colspan=1>35.2 ± 0.1(0.2)</td><td rowspan=1 colspan=1>34.4 ± 0.3(-0.5)</td><td rowspan=1 colspan=1>36.1 ± 0.2(0.3)</td><td rowspan=1 colspan=1>35.9 ± 1.1(-0.4)</td></tr><tr><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>Rowasa</td><td rowspan=1 colspan=1>34.3 ± 0.2(-0.3)</td><td rowspan=1 colspan=1>37.0 ± 0.6(-0.1)</td><td rowspan=1 colspan=1>35.2 ± 0.7(0.2)</td><td rowspan=1 colspan=1>34.5 ± 0.3(-0.4)</td><td rowspan=1 colspan=1>35.5 ± 0.6(-0.3)</td><td rowspan=1 colspan=1>35.8 ± 0.6(-0.5)</td></tr><tr><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>Ortho OptionsContraceptive Gel</td><td rowspan=1 colspan=1>34.3 ± 0.0(-0.3)</td><td rowspan=1 colspan=1>37.0 ± 0.5(-0.1)</td><td rowspan=1 colspan=1>35.5 ± 0.2(0.5)</td><td rowspan=1 colspan=1>34.3 ± 0.3(-0.6)</td><td rowspan=1 colspan=1>36.1 ± 0.4(0.3)</td><td rowspan=1 colspan=1>36.0 ± 0.5(-0.3)</td></tr><tr><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>Imodium</td><td rowspan=1 colspan=1>34.5 ± 0.1(0.1)</td><td rowspan=1 colspan=1>37.4 ± 0.2(0.3)</td><td rowspan=1 colspan=1>34.9 ± 0.3(-0.1)</td><td rowspan=1 colspan=1>34.4 ± 0.2(-0.5)</td><td rowspan=1 colspan=1>35.9 ± 0.3(0.1)</td><td rowspan=1 colspan=1>36.0 ± 0.6(-0.3)</td></tr><tr><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>Pepto Bismol</td><td rowspan=1 colspan=1>34.6 ± 0.1(0.0)</td><td rowspan=1 colspan=1>37.1 ± 0.4(0.0)</td><td rowspan=1 colspan=1>35.1 ± 0.3(0.1)</td><td rowspan=1 colspan=1>35.4 ± 0.6(0.5)</td><td rowspan=1 colspan=1>36.2 ± 0.3(0.4)</td><td rowspan=1 colspan=1>37.1 ± 0.7(0.8)</td></tr><tr><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>Ex-Lax</td><td rowspan=1 colspan=1>34.4 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>37.0 ± 0.2(-0.1)</td><td rowspan=1 colspan=1>34.8 ± 0.7(-0.2)</td><td rowspan=1 colspan=1>35.5 ± 0.2(0.6)</td><td rowspan=1 colspan=1>35.9 ± 0.5(0.1)</td><td rowspan=1 colspan=1>36.0 ± 0.6(-0.3)</td></tr><tr><td rowspan=1 colspan=1>13</td><td rowspan=1 colspan=1>Flagyl</td><td rowspan=1 colspan=1>34.5 ± 0.1(-0.1)</td><td rowspan=1 colspan=1>37.2 ± 0.1(0.1)</td><td rowspan=1 colspan=1>35.1 ± 0.2(0.1)</td><td rowspan=1 colspan=1>34.9 ± 0.5(0.0)</td><td rowspan=1 colspan=1>36.3 ± 0.3(0.5)</td><td rowspan=1 colspan=1>35.9 ± 0.7(-0.4)</td></tr><tr><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>Vancocin</td><td rowspan=1 colspan=1>34.5 ± 0.1(-0.1)</td><td rowspan=1 colspan=1>37.2 ± 0.4(0.1)</td><td rowspan=1 colspan=1>35.5 ± 0.4(0.5)</td><td rowspan=1 colspan=1>34.9 ± 0.2(0.0)</td><td rowspan=1 colspan=1>35.1 ± 0.3(-0.7)</td><td rowspan=1 colspan=1>37.1 ± 1.0(0.8)</td></tr><tr><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>Aleve</td><td rowspan=1 colspan=1>34.9 ± 0.3(0.3)</td><td rowspan=1 colspan=1>37.6 ± 0.6(0.5)</td><td rowspan=1 colspan=1>36.0 ± 1.0(1.0)</td><td rowspan=1 colspan=1>34.7 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>35.7 ± 0.4(-0.1)</td><td rowspan=1 colspan=1>35.8 ± 0.4(-0.5)</td></tr><tr><td rowspan=1 colspan=1>16</td><td rowspan=1 colspan=1>Wet Ones</td><td rowspan=1 colspan=1>34.4 ± 0.3(-0.2)</td><td rowspan=1 colspan=1>37.2 ± 0.2(0.1)</td><td rowspan=1 colspan=1>35.3 ± 0.3(0.3)</td><td rowspan=1 colspan=1>34.9 ± 0.5(0.0)</td><td rowspan=1 colspan=1>36.0 ± 0.2(0.2)</td><td rowspan=1 colspan=1>36.2 ± 0.7(-0.1)</td></tr><tr><td rowspan=1 colspan=1>17</td><td rowspan=1 colspan=1>Palmitic/StearicAcid</td><td rowspan=1 colspan=1>34.8 ± 0.2(0.2)</td><td rowspan=1 colspan=1>37.0 ± 0.4(-0.1)</td><td rowspan=1 colspan=1>35.4 ± 0.4(0.4)</td><td rowspan=1 colspan=1>34.9 ± 0.4(0.0)</td><td rowspan=1 colspan=1>36.9 ± 1.1(1.1)</td><td rowspan=1 colspan=1>35.7 ± 0.5(-0.6)</td></tr><tr><td rowspan=1 colspan=1>18</td><td rowspan=1 colspan=1>Blood</td><td rowspan=1 colspan=1>34.8 ± 0.4(0.2)</td><td rowspan=1 colspan=1>37.4 ± 0.2(0.3)</td><td rowspan=1 colspan=1>35.8 ± 0.6(0.8)</td><td rowspan=1 colspan=1>35.8 ± 0.8(0.9)</td><td rowspan=1 colspan=1>36.6 ± 0.2(0.8)</td><td rowspan=1 colspan=1>36.5 ± 0.5(0.2)</td></tr></table>

<table><tr><td rowspan=1 colspan=10>Mean/Standard Deviations of Ct Values for All Targets</td></tr><tr><td rowspan=3 colspan=1>Substance #</td><td rowspan=3 colspan=1>Substance BrandName</td><td rowspan=3 colspan=1>C. coliMean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=3 colspan=1>C. jejuniMean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=3 colspan=2>SalmonellaMean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=3 colspan=2>ShigellaMean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=3 colspan=1>stx1Mean ± SD(Mean CtDifference:Substance No substance)</td><td rowspan=3 colspan=1>stx2Mean ± SD(Mean CtDifference:Substance No substance)</td></tr><tr><td rowspan=1 colspan=1></td></tr><tr></tr><tr><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>Mucin</td><td rowspan=1 colspan=1>34.4 ± 0.2(-0.2)</td><td rowspan=1 colspan=1>37.1 ± 0.2(0.0)</td><td rowspan=1 colspan=2>36.1 ± 0.5(1.1)</td><td rowspan=1 colspan=2>35.7 ± 0.3(0.8)</td><td rowspan=1 colspan=1>35.9 ± 0.5(0.1)</td><td rowspan=1 colspan=1>36.2 ± 0.9(-0.1)</td></tr></table>

The ProGastro SSCS Assay did not appear to be affected by the presence of any endogenous or exogenous potential PCR inhibitors tested in this study.

# j. Microbial Interference:

An analytical study was conducted to evaluate the ProGastro SSCS Assay for potential interference by microorganisms that are not detected by the assay using clinically relevant concentrations of potentially interfering microorganisms (same as those used in the Analytical Specificity Study).

Potentially interfering microorganisms (37 bacteria, six viruses, four parasites, and one yeast) were added to contrived positive sample groups (described below) at concentrations of $1 0 ^ { 6 } - 1 0 ^ { 8 } \mathrm { C F U / m L }$ (bacteria and fungi) or at a minimum of $2 . 8 5 \mathrm { ~ x ~ }$ $1 0 ^ { 3 } \mathrm { T C I D } _ { 5 0 } / \mathrm { m L }$ (viruses). The contrived positive samples were prepared from a pool of stool matrix that was confirmed negative by the ProGastro SSCS Assay.

The ProGastro SSCS target organisms were combined to form two contrived positive sample groups and tested at $2 \mathbf { x }$ Limit of Detection (LoD). One positive sample duo group contained Salmonella and Campylobacter jejuni, which have been found in coinfections the most frequently. Shigella, Campylobacter coli, and STEC (stx1 and stx2 positive) made up the second positive sample trio group. Each positive duo/trio sample group was aliquoted and spiked with a potential microbial interfering organism at high concentrations. Also, each duo/trio positive sample group was extracted without a microbial interferent to serve as a reference point. Final concentrations of each sample component can be found in the table below. Each sample was tested in replicates of three.

<table><tr><td colspan="5" rowspan="1">Microbial Interference Panel</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Component 1 (Target Duo/Trio)</td><td colspan="1" rowspan="1">Component 1FinalConcentration</td><td colspan="1" rowspan="1">Component 2(From Analytical SpecificitySpiked Stool</td><td colspan="1" rowspan="1">Component 2Final Concentration</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 001</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 002</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 003</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 004</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 005</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">5.76x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 006</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">6.70x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 007</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">4.86x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 008</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">6.3x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 009</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">2.16x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 010</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">9.0x105 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 011</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">4.05x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 012</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">3.20x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 013</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">1.8x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 014</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">3.87x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 015</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">3.78x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 016</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">4.32x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 017</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 018</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 019</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 020</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 021</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia coli Non-STEC</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 022</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia coli(enteroinvasive)</td><td colspan="1" rowspan="1">1.98x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 023</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">1.8x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 024</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">7.92x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 025</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">5.04x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 026</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 027</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.03x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 028</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">3.78x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 029</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">2.88x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 030</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.62x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 031</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 032</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 033</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">5.04x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 034</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">7.74x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 035</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 036</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 037</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 038</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">2.97x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 039</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">2.85x105 TCID5/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 040</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">1.42x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 041</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">2.85x106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 042</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">2.85x107 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 043</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">2.85x103³ TCID5/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 044</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">2.25x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 045</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.49x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 046</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 047</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 048</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 049</td><td colspan="1" rowspan="1">Salmonella Enteritidis &amp;Campylobacter jejuni</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 050</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 051</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 052</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 053</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 054</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">5.76x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 055</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">6.70x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 056</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">4.86x107 CcFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 057</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">6.3x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 058</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">2.16x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 059</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,d STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">9.0x10³ CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 060</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">4.05x106 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 061</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">3.20x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 062</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">1.8x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 063</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">3.87x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 064</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">3.78x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 065</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">4.32x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 066</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 067</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 068</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 069</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 070</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia coli Non-STEC</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 071</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia coli(enteroinvasive)</td><td colspan="1" rowspan="1">1.98x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 072</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">1.8x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 073</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">7.92x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 074</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">5.04x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 075</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,ad STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 076</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">1.03x108 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 077</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">3.78x106 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 078</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Peptostreptococcusanaerobius</td><td colspan="1" rowspan="1">2.88x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 079</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">1.62x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 080</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 081</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 082</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">5.04x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 083</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">7.74x108 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 084</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 085</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">1.35x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 086</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">1.35x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 087</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">2.97x107 cFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 088</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,nd STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">2.85x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 089</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli, and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">1.42x105 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">Sample</td><td colspan="1" rowspan="1">Component 1 (Target Duo/Trio)</td><td colspan="1" rowspan="1">Component 1FinalConcentration</td><td colspan="1" rowspan="1">Component 2(From Analytical SpecificitySpiked Stool)</td><td colspan="1" rowspan="1">Component 2Final Concentration</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 090</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">2.85x106 TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 091</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">2.85x107 TCID5/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 092</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">2.85x10³ TCID50/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 093</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">2.25x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 094</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">1.49x107 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 095</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 096</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 097</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 098</td><td colspan="1" rowspan="1">Shigella sonnei, Campylobacter coli,and STEC</td><td colspan="1" rowspan="1">2X LoD</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">9x10-2 Dilution</td></tr></table>

The Mean Ct and Standard Deviations (SD) for each positive sample, and the Mean Ct difference (calculated by taking the average Ct for the sample containing only the duo/trio targets and subtracting the mean Ct from the sample containing the duo/trio targets with each Microbial Interference Panel Member added) are presented in tables below. A Microbial Interference Panel Member with a Mean Ct difference of over 3.3 is considered interfering.

<table><tr><td colspan="6" rowspan="1">Microbial Interference Ct Averages and Standard Deviations for Positive Samples  SSC Mix</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct −interferent sampleCt)</td><td colspan="1" rowspan="1">Campy/FAMMean Ct±SD</td><td colspan="1" rowspan="1">Sal/TETMean Ct±SD</td><td colspan="1" rowspan="1">Shi/TxRMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 001</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">38.6±0.6</td><td colspan="1" rowspan="1">34.9±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 002</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">FAM: 0.2TET: -0.3</td><td colspan="1" rowspan="1">38.4±0.3</td><td colspan="1" rowspan="1">35.2±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 003</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">FAM: 0.2TET: 0.0</td><td colspan="1" rowspan="1">38.4±0.3</td><td colspan="1" rowspan="1">34.9±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 004</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">FAM: 0.3TET: -0.4</td><td colspan="1" rowspan="1">38.3±0.5</td><td colspan="1" rowspan="1">35.3±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 005</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">FAM: 0.1TET: 0.1</td><td colspan="1" rowspan="1">38.5±0.3</td><td colspan="1" rowspan="1">34.8±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 006</td><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">FAM: 0.5TET: 0.5</td><td colspan="1" rowspan="1">38.1±0.4</td><td colspan="1" rowspan="1">34.4±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 007</td><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">FAM: 1.2TET: 1.0</td><td colspan="1" rowspan="1">37.4±0.7</td><td colspan="1" rowspan="1">33.9±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 008</td><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">FAM: 1.1TET: 0.6</td><td colspan="1" rowspan="1">37.5±0.5</td><td colspan="1" rowspan="1">34.3±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 009</td><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">FAM: -0.1TET: 0.3</td><td colspan="1" rowspan="1">38.7±0.3</td><td colspan="1" rowspan="1">34.6±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 010</td><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">FAM: -1.6TET: 0.4</td><td colspan="1" rowspan="1">40.2±2.2</td><td colspan="1" rowspan="1">34.5±0.7</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct -interferent sampleCt)</td><td colspan="1" rowspan="1">Campy/FAMMean Ct±SD</td><td colspan="1" rowspan="1">Sal/TETMean Ct±SD</td><td colspan="1" rowspan="1">Shi/TxRMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 011</td><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">FAM: 0.6TET: 0.1</td><td colspan="1" rowspan="1">38.0±0.3</td><td colspan="1" rowspan="1">34.8±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 012</td><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">FAM: 0.2TET: 0.1</td><td colspan="1" rowspan="1">38.4±0.5</td><td colspan="1" rowspan="1">34.8±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 013</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">FAM: 0.8TET: 0.1</td><td colspan="1" rowspan="1">37.8±0.4</td><td colspan="1" rowspan="1">35.0±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 014</td><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">FAM: 0.6TET: -0.1</td><td colspan="1" rowspan="1">38.0±0.3</td><td colspan="1" rowspan="1">35.0±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 015</td><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">FAM: 0.4TET: -0.1</td><td colspan="1" rowspan="1">38.2±0.2</td><td colspan="1" rowspan="1">35.0±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 016</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">FAM: -0.4TET: 0.0</td><td colspan="1" rowspan="1">39.0±0.5</td><td colspan="1" rowspan="1">34.9±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 017</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">FAM: -0.4TET: -0.4</td><td colspan="1" rowspan="1">39.0±0.6</td><td colspan="1" rowspan="1">35.3±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 018</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">FAM: -0.7TET: 0.1</td><td colspan="1" rowspan="1">39.3±1.8</td><td colspan="1" rowspan="1">34.8±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 019</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">FAM: 0.3TET: 0.0</td><td colspan="1" rowspan="1">38.3±0.8</td><td colspan="1" rowspan="1">34.9±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 020</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">FAM: 1.4TET: 0.8</td><td colspan="1" rowspan="1">37.2±0.2</td><td colspan="1" rowspan="1">34.1±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 021</td><td colspan="1" rowspan="1">Escherichia coli Non-STEC</td><td colspan="1" rowspan="1">FAM: 0.7TET: 0.2</td><td colspan="1" rowspan="1">37.9±0.5</td><td colspan="1" rowspan="1">34.7±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 022</td><td colspan="1" rowspan="1">Escherichia coli (enteroinvasive)</td><td colspan="1" rowspan="1">FAM: 1.1TET: 0.3</td><td colspan="1" rowspan="1">37.5±0.6</td><td colspan="1" rowspan="1">34.6±0.6</td><td colspan="1" rowspan="1">22.4±0.2*</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 023</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">FAM: 0.9TET: 0.2</td><td colspan="1" rowspan="1">37.7±0.1</td><td colspan="1" rowspan="1">34.7±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 024</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">FAM: 0.2TET: -1.1</td><td colspan="1" rowspan="1">38.4±0.1</td><td colspan="1" rowspan="1">36.0±2.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 025</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">FAM: -0.7TET: -0.1</td><td colspan="1" rowspan="1">39.3±0.6</td><td colspan="1" rowspan="1">35.0±0.8</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 026</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">FAM: -0.3TET: 0.1</td><td colspan="1" rowspan="1">38.9±0.3</td><td colspan="1" rowspan="1">34.8±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 027</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">FAM: 0.6TET: 0.2</td><td colspan="1" rowspan="1">38.0±0.7</td><td colspan="1" rowspan="1">34.7±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 028</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">FAM: 0.4TET: 0.6</td><td colspan="1" rowspan="1">38.2±0.8</td><td colspan="1" rowspan="1">34.3±0.1</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 029</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">FAM: 0.5TET: 0.4</td><td colspan="1" rowspan="1">38.1±0.7</td><td colspan="1" rowspan="1">34.5±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 030</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">FAM: 0.6TET: 0.3</td><td colspan="1" rowspan="1">38.0±0.7</td><td colspan="1" rowspan="1">34.6±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 031</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">FAM: 0.7TET: 0.5</td><td colspan="1" rowspan="1">37.9±0.5</td><td colspan="1" rowspan="1">34.4±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 032</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">FAM: 0.5TET: 0.5</td><td colspan="1" rowspan="1">38.1±0.4</td><td colspan="1" rowspan="1">34.4±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 033</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">FAM: 0.8TET: -0.3</td><td colspan="1" rowspan="1">37.8±0.3</td><td colspan="1" rowspan="1">35.2±0.7</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 034</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">FAM: 0.1TET: -0.1</td><td colspan="1" rowspan="1">38.5±0.3</td><td colspan="1" rowspan="1">35.0±0.1</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 035</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">FAM: 0.3TET: 0.1</td><td colspan="1" rowspan="1">38.3±0.4</td><td colspan="1" rowspan="1">35.0±0.6</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 036</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">FAM: 0.4TET: 0.0</td><td colspan="1" rowspan="1">38.2±0.5</td><td colspan="1" rowspan="1">34.9±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 037</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">FAM: -0.4TET: 0.0</td><td colspan="1" rowspan="1">39.0±0.4</td><td colspan="1" rowspan="1">34.9±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 038</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">FAM: -0.1TET: 0.6</td><td colspan="1" rowspan="1">38.7±0.5</td><td colspan="1" rowspan="1">34.3±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 039</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">FAM: 0.2TET: 0.1</td><td colspan="1" rowspan="1">38.4±0.4</td><td colspan="1" rowspan="1">35.0±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 040</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">FAM: -0.3TET: -0.7</td><td colspan="1" rowspan="1">38.9±0.2</td><td colspan="1" rowspan="1">35.6±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 041</td><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">FAM: -0.8TET: -0.3</td><td colspan="1" rowspan="1">39.4±0.1</td><td colspan="1" rowspan="1">35.2±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 042</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">FAM: 0.0TET: 0.0</td><td colspan="1" rowspan="1">38.6±0.4</td><td colspan="1" rowspan="1">34.9±0.6</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 043</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">FAM: -0.4TET: 0.1</td><td colspan="1" rowspan="1">39.0±0.3</td><td colspan="1" rowspan="1">34.8±0.2</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 044</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">FAM: -0.9TET: 0.3</td><td colspan="1" rowspan="1">39.5±0.4</td><td colspan="1" rowspan="1">34.6±0.1</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 045</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">FAM: -0.2TET: -0.3</td><td colspan="1" rowspan="1">38.8±0.4</td><td colspan="1" rowspan="1">35.2±0.9</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 046</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">FAM: -0.3TET: 0.2</td><td colspan="1" rowspan="1">38.9±0.2</td><td colspan="1" rowspan="1">34.7±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 047</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">FAM: -0.8TET: 0.1</td><td colspan="1" rowspan="1">39.4±0.3</td><td colspan="1" rowspan="1">34.8±0.4</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 048</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">FAM: 0.1TET: 0.5</td><td colspan="1" rowspan="1">38.5±0.6</td><td colspan="1" rowspan="1">34.4±0.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 049</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">FAM: -0.1TET: 0.2</td><td colspan="1" rowspan="1">38.7±0.3</td><td colspan="1" rowspan="1">34.7±0.5</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 050</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.8±0.5</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.9±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 051</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">FAM: 0.1TxR: -0.8</td><td colspan="1" rowspan="1">34.7 ±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.7±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 052</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">FAM: 0.1TxR: 0.1</td><td colspan="1" rowspan="1">34.9±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.8±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 053</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">FAM: 0.2TxR: 0.2</td><td colspan="1" rowspan="1">34.6±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.7±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 054</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">FAM: 0.3TxR: -0.2</td><td colspan="1" rowspan="1">34.5±0.0</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.1±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 055</td><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">FAM: 0.1TxR: -0.6</td><td colspan="1" rowspan="1">34.9±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.5±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 056</td><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">FAM: 0.1TxR: 0.2</td><td colspan="1" rowspan="1">34.7±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.7±0.6</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 057</td><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">FAM: -0.1TxR: 0.3</td><td colspan="1" rowspan="1">34.9±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.2±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 058</td><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">FAM: -0.2TxR: -0.2</td><td colspan="1" rowspan="1">35.0±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.1±0.3</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct -interferent sampleCt)</td><td colspan="1" rowspan="1">Campy/FAMMean Ct±sSD</td><td colspan="1" rowspan="1">Sal/TETMean Ct±SD</td><td colspan="1" rowspan="1">Shi/TxRMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 059</td><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">FAM: 0.4TxR: 0.0</td><td colspan="1" rowspan="1">35.2±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.9±0.8</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 060</td><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">FAM: 0.1TxR: -0.9</td><td colspan="1" rowspan="1">34.7±0.5</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.8±0.9</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 061</td><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">FAM: -0.1TxR: -0.7</td><td colspan="1" rowspan="1">34.9±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.6±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 062</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">FAM: 1.2TxR: 1.4</td><td colspan="1" rowspan="1">36.0±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.3±02</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 063</td><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">FAM: 0.9TxR: -1.5</td><td colspan="1" rowspan="1">35.7±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.4±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 064</td><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">FAM: -0.7TxR: -0.8</td><td colspan="1" rowspan="1">35.5±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.7±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 065</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">FAM: -1.0TxR: -1.4</td><td colspan="1" rowspan="1">35.8±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.3±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 066</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">FAM: 0.6TxR: -1.4</td><td colspan="1" rowspan="1">35.4±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.3±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 067</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">FAM: -0.4TxR: -0.6</td><td colspan="1" rowspan="1">35.2±0.6</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.5±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 068</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">FAM: -0.9TxR: 1.0</td><td colspan="1" rowspan="1">35.7±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.9±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 069</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">FAM: 1.1TxR: -1.4</td><td colspan="1" rowspan="1">35.9±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.3±0.6</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 070</td><td colspan="1" rowspan="1">Escherichia coli Non-STEC</td><td colspan="1" rowspan="1">FAM: 1.0TxR: 1.5</td><td colspan="1" rowspan="1">35.8±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.4±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 071</td><td colspan="1" rowspan="1">Escherichia coli (enteroinvasive)</td><td colspan="1" rowspan="1">FAM: 0.4TxR: 12.2*</td><td colspan="1" rowspan="1">35.2±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">22.7±0.4*</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 072</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">FAM: -1.0TxR: -1.0</td><td colspan="1" rowspan="1">35.8±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.9±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 073</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">FAM: -0.4TxR: -0.1</td><td colspan="1" rowspan="1">35.1±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.0±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 074</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">FAM: 1.3TxR: -1.5</td><td colspan="1" rowspan="1">36.1±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.4±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 075</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">FAM: -0.9TxR: -1.3</td><td colspan="1" rowspan="1">35.7±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.2±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 076</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">FAM: -1.2TxR: 1.2</td><td colspan="1" rowspan="1">36.0±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.1±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 077</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">FAM: -0.6TxR: -0.6</td><td colspan="1" rowspan="1">35.4±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.5±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 078</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">FAM: -1.1TxR: -0.8</td><td colspan="1" rowspan="1">35.9±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.7±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 079</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">FAM: 1.0TxR: -1.2</td><td colspan="1" rowspan="1">35.8±0.5</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.1±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 080</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">FAM: 0.8TxR: -1.1</td><td colspan="1" rowspan="1">35.6±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.0±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 081</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">FAM: -1.3TxR: -1.2</td><td colspan="1" rowspan="1">36.1±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.1±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 082</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">FAM: -0.3TxR: -0.3</td><td colspan="1" rowspan="1">35.1±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.2±0.2</td></tr><tr><td colspan="6" rowspan="1">Microbial Interference Ct Averages and Standard Deviations for Positive Samples - SSC Mix</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct -interferent sampleCt)</td><td colspan="1" rowspan="1">Campy/FAMMean Ct±SD</td><td colspan="1" rowspan="1">Sal/TETMean Ct±SD</td><td colspan="1" rowspan="1">Shi/TxRMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 083</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">FAM: -0.2TxR: -0.3</td><td colspan="1" rowspan="1">35.0±0.6</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.2±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 084</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">FAM: 0.9TxR: -0.9</td><td colspan="1" rowspan="1">35.7±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.8±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 085</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">FAM: -0.7TxR: 0.4</td><td colspan="1" rowspan="1">35.5±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.3±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 086</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">FAM: -0.3TxR: -0.7</td><td colspan="1" rowspan="1">35.1±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.6±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 087</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">FAM: 0.9TxR: -1.3</td><td colspan="1" rowspan="1">35.7±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.2±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 088</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">FAM: -0.3TxR: -1.3</td><td colspan="1" rowspan="1">35.1±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">36.2±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 089</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">FAM: -0.5TxR: 0.4</td><td colspan="1" rowspan="1">35.3±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.3±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 090</td><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">FAM: -0.9TxR: -0.5</td><td colspan="1" rowspan="1">35.7±0.6</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.4±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 091</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">FAM: -0.4TxR: 0.1</td><td colspan="1" rowspan="1">35.2±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.0±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 092</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">FAM: -0.2TxR: -0.3</td><td colspan="1" rowspan="1">35.0±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.2±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 093</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">FAM: -0.3TxR: -0.5</td><td colspan="1" rowspan="1">35.1±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.4±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 094</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">FAM: -0.5TxR: -0.6</td><td colspan="1" rowspan="1">35.3±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.5±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 095</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">FAM: -0.4TxR: -0.1</td><td colspan="1" rowspan="1">35.2±0.3</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.0±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 096</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">FAM: 0.1TxR: -0.9</td><td colspan="1" rowspan="1">34.9±0.4</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.8±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 097</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">FAM: -0.2TxR: -0.8</td><td colspan="1" rowspan="1">35.0±0.1</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">35.7±1.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 098</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">FAM: -0.3TxR: 0.3</td><td colspan="1" rowspan="1">35.1±0.2</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">34.6±0.6</td></tr></table>

\* Sample contains a high concentration of EIEC; therefore the Ct value is very low

Enteroinvasive Escherichia coli (EIEC) is genetically very similar to Shigella and was detected by the SSC Mix as positive for Shigella as expected. This cross-reactivity is reported in the Package Insert as a limitation.

<table><tr><td colspan="5" rowspan="1">Microbial Interference Ct Averages and Standard Deviations for Positive Samples  STEC Mix</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct -interferent sampleCt)</td><td colspan="1" rowspan="1">stx2/FAMMean Ct±SD</td><td colspan="1" rowspan="1">stx1/TETMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 050</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">N/A</td><td colspan="1" rowspan="1">37.1±0.3</td><td colspan="1" rowspan="1">36.3±0.8</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 051</td><td colspan="1" rowspan="1">Aeromonas hydrophila</td><td colspan="1" rowspan="1">stx2: 0.5stx1: 0.7</td><td colspan="1" rowspan="1">36.6±0.3</td><td colspan="1" rowspan="1">35.6±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 052</td><td colspan="1" rowspan="1">Bacillus cereus</td><td colspan="1" rowspan="1">stx2: 0.5stx1: 0.1</td><td colspan="1" rowspan="1">36.6±0.4</td><td colspan="1" rowspan="1">36.2±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 053</td><td colspan="1" rowspan="1">Bacteroides fragilis</td><td colspan="1" rowspan="1">stx2: 1.0stx1: 0.5</td><td colspan="1" rowspan="1">36.1±0.6</td><td colspan="1" rowspan="1">35.8±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 054</td><td colspan="1" rowspan="1">Campylobacter upsaliensis</td><td colspan="1" rowspan="1">stx2: 1.0stx1: 0.1</td><td colspan="1" rowspan="1">36.1±0.5</td><td colspan="1" rowspan="1">36.2±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 055</td><td colspan="1" rowspan="1">Campylobacter hyointestinalis</td><td colspan="1" rowspan="1">stx2: 1.0stx1: 0.6</td><td colspan="1" rowspan="1">36.1±0.3</td><td colspan="1" rowspan="1">35.7±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 056</td><td colspan="1" rowspan="1">Campylobacter fetus</td><td colspan="1" rowspan="1">stx2: 0.8stx1: 0.7</td><td colspan="1" rowspan="1">36.3±0.1</td><td colspan="1" rowspan="1">35.6±0.8</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 057</td><td colspan="1" rowspan="1">Campylobacter helveticus</td><td colspan="1" rowspan="1">stx2: 0.0stx1: 0.6</td><td colspan="1" rowspan="1">37.1±0.8</td><td colspan="1" rowspan="1">35.7±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 058</td><td colspan="1" rowspan="1">Campylobacter gracilis</td><td colspan="1" rowspan="1">stx2: 0.1stx1: 0.8</td><td colspan="1" rowspan="1">37.0±0.5</td><td colspan="1" rowspan="1">35.5±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 059</td><td colspan="1" rowspan="1">Campylobacter concisus</td><td colspan="1" rowspan="1">stx2: 0.7stx1: 0.8</td><td colspan="1" rowspan="1">36.4±0.2</td><td colspan="1" rowspan="1">35.5±1.0</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 060</td><td colspan="1" rowspan="1">Campylobacter curvus</td><td colspan="1" rowspan="1">stx2: -0.4stx1: -0.5</td><td colspan="1" rowspan="1">37.5±2.8</td><td colspan="1" rowspan="1">36.8±1.0</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 061</td><td colspan="1" rowspan="1">Campylobacter sputorum</td><td colspan="1" rowspan="1">stx2: 1.1stx1: 0.8</td><td colspan="1" rowspan="1">36.0±0.2</td><td colspan="1" rowspan="1">35.5±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 062</td><td colspan="1" rowspan="1">Campylobacter rectus</td><td colspan="1" rowspan="1">stx2: 0.9stx1: 1.3</td><td colspan="1" rowspan="1">36.2±0.9</td><td colspan="1" rowspan="1">35.0±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 063</td><td colspan="1" rowspan="1">Campylobacter showae</td><td colspan="1" rowspan="1">stx2: 1.3stx1: 1.0</td><td colspan="1" rowspan="1">35.8±0.2</td><td colspan="1" rowspan="1">35.3±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 064</td><td colspan="1" rowspan="1">Campylobacter mucosalis</td><td colspan="1" rowspan="1">stx2: 1.4stx1: 1.7</td><td colspan="1" rowspan="1">35.7±0.2</td><td colspan="1" rowspan="1">34.6±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 065</td><td colspan="1" rowspan="1">Citrobacter freundii</td><td colspan="1" rowspan="1">stx2: 0.9stx1: 1.4</td><td colspan="1" rowspan="1">36.2±0.4</td><td colspan="1" rowspan="1">34.9±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 066</td><td colspan="1" rowspan="1">Clostridium difficile</td><td colspan="1" rowspan="1">stx2: 0.5stx1: -0.3</td><td colspan="1" rowspan="1">36.6±0.5</td><td colspan="1" rowspan="1">36.6±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 067</td><td colspan="1" rowspan="1">Clostridium perfringens</td><td colspan="1" rowspan="1">stx2: 0.1stx1: 0.3</td><td colspan="1" rowspan="1">37.0±0.2</td><td colspan="1" rowspan="1">36.0±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 068</td><td colspan="1" rowspan="1">Enterobacter cloacae</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.0</td><td colspan="1" rowspan="1">36.5±0.2</td><td colspan="1" rowspan="1">36.3±0.9</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 069</td><td colspan="1" rowspan="1">Enterococcus faecalis</td><td colspan="1" rowspan="1">stx2: 0.9stx1: -1.0</td><td colspan="1" rowspan="1">36.2±0.5</td><td colspan="1" rowspan="1">37.3±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 070</td><td colspan="1" rowspan="1">Escherichia coli Non-STEC</td><td colspan="1" rowspan="1">stx2: 0.7stx1: -0.5</td><td colspan="1" rowspan="1">36.4±0.3</td><td colspan="1" rowspan="1">36.8±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 071</td><td colspan="1" rowspan="1">Escherichia coli (enteroinvasive)</td><td colspan="1" rowspan="1">stx2: -0.1stx1: -0.3</td><td colspan="1" rowspan="1">37.1±0.7</td><td colspan="1" rowspan="1">36.6±0.6</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 072</td><td colspan="1" rowspan="1">Escherichia fergusonii</td><td colspan="1" rowspan="1">stx2: 0.8stx1: 0.0</td><td colspan="1" rowspan="1">36.3±0.3</td><td colspan="1" rowspan="1">36.3±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 073</td><td colspan="1" rowspan="1">Escherichia hermannii</td><td colspan="1" rowspan="1">stx2: 0.7stx1: 0.2</td><td colspan="1" rowspan="1">36.4±0.4</td><td colspan="1" rowspan="1">36.1±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 074</td><td colspan="1" rowspan="1">Helicobacter pylori</td><td colspan="1" rowspan="1">stx2: 1.2stx1: 1.3</td><td colspan="1" rowspan="1">35.9±0.3</td><td colspan="1" rowspan="1">35.0±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 075</td><td colspan="1" rowspan="1">Klebsiella pneumoniae</td><td colspan="1" rowspan="1">stx2: 1.4stx1: 1.1</td><td colspan="1" rowspan="1">35.7±0.4</td><td colspan="1" rowspan="1">35.2±0.8</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 076</td><td colspan="1" rowspan="1">Lactococcus lactis</td><td colspan="1" rowspan="1">stx2: 0.7stx1: 0.8</td><td colspan="1" rowspan="1">36.4±0.1</td><td colspan="1" rowspan="1">35.5±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 077</td><td colspan="1" rowspan="1">Listeria monocytogenes</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.5</td><td colspan="1" rowspan="1">36.5±0.6</td><td colspan="1" rowspan="1">35.8±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 078</td><td colspan="1" rowspan="1">Peptostreptococcus anaerobius</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 1.1</td><td colspan="1" rowspan="1">36.5±0.2</td><td colspan="1" rowspan="1">35.2±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 079</td><td colspan="1" rowspan="1">Plesiomonas shigelloides</td><td colspan="1" rowspan="1">stx2: 0.2stx1: 0.6</td><td colspan="1" rowspan="1">36.9±0.8</td><td colspan="1" rowspan="1">35.7±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 080</td><td colspan="1" rowspan="1">Proteus vulgaris</td><td colspan="1" rowspan="1">stx2: 0.9stx1: -0.1</td><td colspan="1" rowspan="1">36.2±0.6</td><td colspan="1" rowspan="1">36.4±0.8</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 081</td><td colspan="1" rowspan="1">Pseudomonas aeruginosa</td><td colspan="1" rowspan="1">stx2: 0.8stx1: 0.6</td><td colspan="1" rowspan="1">36.3±0.5</td><td colspan="1" rowspan="1">35.7±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 082</td><td colspan="1" rowspan="1">Pseudomonas fluorescens</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.0</td><td colspan="1" rowspan="1">36.5±0.3</td><td colspan="1" rowspan="1">36.3±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 083</td><td colspan="1" rowspan="1">Serratia marcescens</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.4</td><td colspan="1" rowspan="1">36.5±0.5</td><td colspan="1" rowspan="1">35.9±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 084</td><td colspan="1" rowspan="1">Staphylococcus aureus</td><td colspan="1" rowspan="1">stx2: 1.4stx1: 0.4</td><td colspan="1" rowspan="1">35.7±0.3</td><td colspan="1" rowspan="1">35.9±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 085</td><td colspan="1" rowspan="1">Staphylococcus epidermidis</td><td colspan="1" rowspan="1">stx2: 0.5stx1: 0.8</td><td colspan="1" rowspan="1">36.5±0.5</td><td colspan="1" rowspan="1">35.5±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 086</td><td colspan="1" rowspan="1">Vibrio parahaemolyticus</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.1</td><td colspan="1" rowspan="1">36.5±0.8</td><td colspan="1" rowspan="1">36.2±1.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 087</td><td colspan="1" rowspan="1">Yersinia enterocolitica</td><td colspan="1" rowspan="1">stx2: 0.1stx1: 0.4</td><td colspan="1" rowspan="1">37.0±0.3</td><td colspan="1" rowspan="1">35.9±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 088</td><td colspan="1" rowspan="1">Adenovirus Type 40</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.8</td><td colspan="1" rowspan="1">36.5±0.4</td><td colspan="1" rowspan="1">35.5±0.5</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 089</td><td colspan="1" rowspan="1">Adenovirus Type 41</td><td colspan="1" rowspan="1">stx2: 1.2stx1: 0.5</td><td colspan="1" rowspan="1">35.9±0.5</td><td colspan="1" rowspan="1">35.8±0.4</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 090</td><td colspan="1" rowspan="1">Coxsackievirus B5/10/2006</td><td colspan="1" rowspan="1">stx2: 0.4stx1: 0.2</td><td colspan="1" rowspan="1">36.7±0.5</td><td colspan="1" rowspan="1">36.1±0.2</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 091</td><td colspan="1" rowspan="1">Echovirus 11</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.3</td><td colspan="1" rowspan="1">36.5±0.6</td><td colspan="1" rowspan="1">36.0±0.3</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 092</td><td colspan="1" rowspan="1">Rotavirus</td><td colspan="1" rowspan="1">stx2: 0.5stx1: 0.2</td><td colspan="1" rowspan="1">36.6±0.8</td><td colspan="1" rowspan="1">36.1±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 093</td><td colspan="1" rowspan="1">Norovirus</td><td colspan="1" rowspan="1">stx2: 0.8stx1: 0.2</td><td colspan="1" rowspan="1">36.3±0.4</td><td colspan="1" rowspan="1">36.1±0.6</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 094</td><td colspan="1" rowspan="1">Candida albicans</td><td colspan="1" rowspan="1">stx2: 0.9stx1: 0.3</td><td colspan="1" rowspan="1">36.2±0.2</td><td colspan="1" rowspan="1">36.0±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 095</td><td colspan="1" rowspan="1">Blastocystis hominis</td><td colspan="1" rowspan="1">stx2: 1.1stx1: 0.7</td><td colspan="1" rowspan="1">36.0±0.3</td><td colspan="1" rowspan="1">35.6±0.1</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 096</td><td colspan="1" rowspan="1">Giardia lamblia</td><td colspan="1" rowspan="1">stx2: 0.6stx1: 0.4</td><td colspan="1" rowspan="1">36.5±1.1</td><td colspan="1" rowspan="1">35.9±0.7</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 097</td><td colspan="1" rowspan="1">Cryptosporidium parvum</td><td colspan="1" rowspan="1">stx2: 0.6stx1: -0.1</td><td colspan="1" rowspan="1">36.5±0.2</td><td colspan="1" rowspan="1">36.4±0.9</td></tr><tr><td colspan="1" rowspan="1">Sample ID</td><td colspan="1" rowspan="1">MI Panel Member</td><td colspan="1" rowspan="1">CT Difference(Target only Ct −interferent sampleCt)</td><td colspan="1" rowspan="1">stx2/FAMMean Ct±SD</td><td colspan="1" rowspan="1">stx1/TETMean Ct±SD</td></tr><tr><td colspan="1" rowspan="1">SSCS MI 098</td><td colspan="1" rowspan="1">Entamoeba histolytica</td><td colspan="1" rowspan="1">stx2: 1.5stx1: 1.2</td><td colspan="1" rowspan="1">35.6±0.2</td><td colspan="1" rowspan="1">35.1±0.5</td></tr></table>

In conclusion, the presence of high clinically relevant concentrations of potentially interfering microorganisms did not appear to affect the performance of the ProGastro SSCS Assay detecting low concentrations (close to the LoD concentrations) of assay target organisms.

# k. Competitive Interference

To evaluate the effects of Competitive Interference on the ProGastro SSCS Assay’s SSC Mix and STEC Mix when various combinations of the Assay’s target organisms were present in a single sample, an analytical study was carried out.

Competitive Interference may occur when the detection of one target in a multiplex PCR Mix is outcompeted by another target(s) in that same multiplex PCR Mix for essential reaction components (dNTPs, $\bar { \bf M } \bf g ^ { 2 + }$ etc.). Competitive Interference was assessed by generating contrived samples containing a single target analyte (organism) present at a concentration near the Limit of Detection (LoD) with one or more different target analytes at a higher concentration in the same sample. This study utilized one cultured and titered strain each of Salmonella, Shigella, Campylobacter jejuni, and Campylobacter coli for use with the SSC Mix. For the STEC Mix testing, two Shiga Toxin producing Escherichia coli (STEC) strains $( s t x I + / s t x 2 -$ and $s t x I { - } / s t x 2 { + } )$ were used. The bacterial strains were diluted in Stool Preservation and Transport Media (SPTM, ParaPak C&S) and spiked into aliquots of a ProGastro SSCS negative stool matrix pool with one or two of the targets at 3 logs above their particular LoD and a second or third target analyzed close to the LoD depending on the particular sample. (See tables below).

<table><tr><td colspan="4" rowspan="1">Description of Competitive Interference Samples  SSC Mix</td></tr><tr><td colspan="1" rowspan="1">SalmonellaConcentration</td><td colspan="1" rowspan="1">Shigella Concentration</td><td colspan="1" rowspan="1">C. coli Concentration</td><td colspan="1" rowspan="1">C. jejuni Concentration</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3 logs above LoD</td></tr><tr><td colspan="1" rowspan="1">:</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3 logs above LoD</td></tr><tr><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td></tr><tr><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td></tr><tr><td colspan="1" rowspan="1">3 logs above LoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td></tr></table>

<table><tr><td rowspan=1 colspan=2>Description of Competitive Interference Samples  STEC Mix</td></tr><tr><td rowspan=1 colspan=1>Stx1 Concentration</td><td rowspan=1 colspan=1>Stx2 Concentration</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>-</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2x LoD</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>3 logs above LoD</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>3 logs above LoD</td></tr><tr><td rowspan=1 colspan=1>3 logs above LoD</td><td rowspan=1 colspan=1>2x LoD</td></tr></table>

The Competitive Interference Panel samples and the required Negative Controls were extracted on the bioMériuex NucliSENS easyMAG and samples tested in triplicate (Positive and Negative Controls in a single replicate per PCR Run) on a Cepheid Smartcycler II using one lot of ProGastro SSCS reagents.

The results were analyzed and any organism(s) that prevented the detection of any of the target organisms were reported as a competitive interferent of the ProGastro SSCS Assay.

The Competitive Interference summary data stratified by the SSC Mix and the STEC Mix is presented in the two tables below. The tables present a summary of the data generated in the study, highlighting (in yellow) the organism combinations that exhibited competitive interference.

SSC Mix Competitive Interference (Multiple-Infection) Summary   

<table><tr><td colspan="1" rowspan="1">SalmonellaConc.</td><td colspan="1" rowspan="1">ShigellaConc.</td><td colspan="1" rowspan="1">C. coliConc.</td><td colspan="1" rowspan="1">C. jejuniConc.</td><td colspan="1" rowspan="1">SalmonellaDetected/MeanCt</td><td colspan="1" rowspan="1">ShigellaDetected/Mean Ct</td><td colspan="1" rowspan="1">C. coliDetected/Mean Ct</td><td colspan="1" rowspan="1">C. jejuniDetected/Mean Ct</td><td colspan="1" rowspan="1">Avg CtDifference (LowConc. AnalyteAlone - LowConc. Analytew/ High Conc,CompetitiveAnalyte)</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">YES/35.8</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">YES/31.4</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">YES/29.2</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/35.3</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">YES/38.0</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">YES/35.7</td><td colspan="1" rowspan="1">YES/21.5</td><td colspan="1" rowspan="1">YES/25.5</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1">0.1</td></tr><tr><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">YES/40.8</td><td colspan="1" rowspan="1">YES/21.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/27.6</td><td colspan="1" rowspan="1">-5.0</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">-</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/31.6</td><td colspan="1" rowspan="1">YES/24.8</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-0.2</td></tr><tr><td colspan="1" rowspan="1">SalmonellaConc.</td><td colspan="1" rowspan="1">ShigellaConc.</td><td colspan="1" rowspan="1">C. coliConc.</td><td colspan="1" rowspan="1">C. jejuniConc.</td><td colspan="1" rowspan="1">SalmonellaDetected/MeanCt</td><td colspan="1" rowspan="1">ShigellaDetected/Mean Ct</td><td colspan="1" rowspan="1">C. coliDetected/Mean Ct</td><td colspan="1" rowspan="1">C. jejuniDetected/Mean Ct</td><td colspan="1" rowspan="1">Avg CtDifference (LowConc. AnalyteAlone - LowConc. Analytew/ High Conc,CompetitiveAnalyte</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NO (only1 of 3repsdetected)</td><td colspan="1" rowspan="1">−</td><td colspan="1" rowspan="1">YES/25.3</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">10x LoD*</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/32.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/25.4</td><td colspan="1" rowspan="1">-3.5</td></tr><tr><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/25.9</td><td colspan="1" rowspan="1">YES/32.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-0.6</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/21.8</td><td colspan="1" rowspan="1">YES/34.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/22.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/37.7</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/25.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NO</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2 logs aboveLoD**</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/30.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/34.6</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-0.1</td></tr><tr><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">YES/25.7</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">NO</td><td colspan="1" rowspan="1">N/A</td></tr><tr><td colspan="1" rowspan="1">2 logs aboveLoD**</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1">YES/30.2</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/42.8</td><td colspan="1" rowspan="1">-4.4</td></tr><tr><td colspan="1" rowspan="1">3 logs aboveLoD</td><td colspan="1" rowspan="1">2x LoD</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">YES/25.9</td><td colspan="1" rowspan="1">YES/32.0</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">-0.6</td></tr></table>

Bold: Samples that exhibited competitive interference. \*Tested at a higher “low” concentration due to low analyte not detected \*\*Tested at a lower “high” concentration due to low analyte not detected

STEC Mix Competitive Interference (Multiple-Infection) Summary   

<table><tr><td rowspan=1 colspan=1>Stx1Concentration</td><td rowspan=1 colspan=1>Stx2Concentration</td><td rowspan=1 colspan=1>Stx1Detected/MeanCt</td><td rowspan=1 colspan=1>Stx2Detected/MeanCt</td><td rowspan=1 colspan=1>Avg Ct Difference(Low ConcAnalyteAlone - LowConcAnalyte w/Competition)</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>YES/38.7</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>YES/35.3</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>3 logs aboveLoD</td><td rowspan=1 colspan=1>YES/41.2</td><td rowspan=1 colspan=1>YES/25.8</td><td rowspan=1 colspan=1>-2.5</td></tr><tr><td rowspan=1 colspan=1>3 logs aboveLoD</td><td rowspan=1 colspan=1>2x LoD</td><td rowspan=1 colspan=1>YES/27.0</td><td rowspan=1 colspan=1>YES/36.0</td><td rowspan=1 colspan=1>-0.7</td></tr></table>

Competitive Interference was observed for several samples using the ProGastro SSCS Assay’s SSC Mix. In samples where Campylobacter jejuni is present in high concentrations and Shigella is present in low concentrations (near the assay limit of detection), competitive interference may occur hindering the detection of Shigella by the ProGastro SSCS Assay SSC Mix. In samples where Salmonella is present in high concentrations and $C .$ . jejuni or $C .$ . coli are present in low concentrations (near the assay limit of detection), competitive interference may occur hindering detection of these Campylobacter strains by the ProGastro SSCS Assay SSC Mix. These limitations are added to the ProGastro SSCS Assay Instructions for Use.

Competitive Interference for STEC strains was not observed using the ProGastro SSCS Assay’s STEC Mix.

# l. Carry-Over Contamination:

A Carry-Over Contamination Study was conducted to demonstrate the degree of Carry-Over/Cross-Contamination that may occur when High Positive (HP) samples were processed alongside True Negative (TN) samples during nucleic acid extraction on the bioMérieux NucliSENS easyMAG and during subsequent PCR setup of the ProGastro SSCS Assay on the Cepheid SmartCycler II.

This study assessed the level of carry-over contamination with the ProGastro SSCS Assay by testing simulated dual positive Shigella (Shigella flexneri ATCC12022) and Shiga Toxin producing E. Coli (STEC) (E. coli Strain 3215-99 O111:H8, $s t x I { + } / s t x 2 { + } )$ high positive stool samples run in series with negative stool samples over the course of five runs (one run/day). This study used bacterial targets that are typically found at the highest level in clinical samples, which based on development and analytical studies were Shigella and STEC. The High Positive (HP) samples consisted of negative stool matrix spiked with a concentration of 5 logs above the Limit of Detection (LoD) for Shigella and 4 logs above the LoD for STEC. These concentrations were used to represent the earlier Ct range for these organisms as observed in the Cutoff Determination and Confirmation Study. True Negative (TN) samples consisted of negative stool matrix. The samples were processed and extracted in a HP/TN alternating fashion (i.e. checkerboard pattern) on the extraction instrument and likewise processed and run using both the SSC and STEC Mixes on the Cepheid SmartCycler II instrument in an alternating fashion.

Eleven (11) HP, 11 TN samples, and a Negative Control were extracted per run on the easyMAG for a total of $5 5 \mathrm { H P }$ samples and 55 TN samples over five runs. The table below presents the order of samples run on the easyMAG. Sample cartridges were placed in a linear array on the easyMAG instrument such that Cartridge A / Vessel 8 (TN 4) was adjacent to Cartridge B / Vessel 1 (HP 5) and Cartridge B / Vessel 8 (TN 8) was adjacent to Cartridge C / Vessel 1 (HP 9), maintaining the alternating pattern.

Sample Order for Extraction Using the easyMAG   

<table><tr><td rowspan=1 colspan=8>Sample Cartridge A</td></tr><tr><td rowspan=1 colspan=1>Vessel 1</td><td rowspan=1 colspan=1>Vessel 2</td><td rowspan=1 colspan=1>Vessel 3</td><td rowspan=1 colspan=1>Vessel 4</td><td rowspan=1 colspan=1>Vessel 5</td><td rowspan=1 colspan=1>Vessel 6</td><td rowspan=1 colspan=1>Vessel 7</td><td rowspan=1 colspan=1>Vessel 8</td></tr><tr><td rowspan=1 colspan=1>HP 1</td><td rowspan=1 colspan=1>TN 1</td><td rowspan=1 colspan=1>HP 2</td><td rowspan=1 colspan=1>TN 2</td><td rowspan=1 colspan=1>HP 3</td><td rowspan=1 colspan=1>TN 3</td><td rowspan=1 colspan=1>HP 4</td><td rowspan=1 colspan=1>TN 4</td></tr><tr><td rowspan=1 colspan=8>Sample Cartridge B</td></tr><tr><td rowspan=1 colspan=1>Vessel 1</td><td rowspan=1 colspan=1>Vessel 2</td><td rowspan=1 colspan=1>Vessel 3</td><td rowspan=1 colspan=1>Vessel 4</td><td rowspan=1 colspan=1>Vessel 5</td><td rowspan=1 colspan=1>Vessel 6</td><td rowspan=1 colspan=1>Vessel 7</td><td rowspan=1 colspan=1>Vessel 8</td></tr><tr><td rowspan=1 colspan=1>HP 5</td><td rowspan=1 colspan=1>TN 5</td><td rowspan=1 colspan=1>HP 6</td><td rowspan=1 colspan=1>TN 6</td><td rowspan=1 colspan=1>HP 7</td><td rowspan=1 colspan=1>TN 7</td><td rowspan=1 colspan=1>HP 8</td><td rowspan=1 colspan=1>TN 8</td></tr><tr><td rowspan=1 colspan=8>Sample Cartridge C</td></tr><tr><td rowspan=1 colspan=1>Vessel 1</td><td rowspan=1 colspan=1>Vessel 2</td><td rowspan=1 colspan=1>Vessel 3</td><td rowspan=1 colspan=1>Vessel 4</td><td rowspan=1 colspan=1>Vessel 5</td><td rowspan=1 colspan=1>Vessel 6</td><td rowspan=1 colspan=1>Vessel 7</td><td rowspan=1 colspan=1>Vessel 8</td></tr><tr><td rowspan=1 colspan=1>HP 9</td><td rowspan=1 colspan=1>TN 9</td><td rowspan=1 colspan=1>HP 10</td><td rowspan=1 colspan=1>TN 10</td><td rowspan=1 colspan=1>HP 11</td><td rowspan=1 colspan=1>TN 11</td><td rowspan=1 colspan=1>NC</td><td rowspan=1 colspan=1>NA</td></tr></table>

Mean Ct values and standard deviation for the Shigella (TxRed, HP samples, SSC Mix), stx1 (TET, HP samples, STEC Mix), stx2 (FAM, HP samples, STEC Mix), SSC IC (CY5, TN samples) and STEC IC (CY5, TN samples) Cts are reported in the table below:

<table><tr><td rowspan=1 colspan=11>Average Ct Values and Standard Deviation for High Positive and True Negative Samples - Both Mixes*</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=4>SSC Mix</td><td rowspan=1 colspan=6>STEC Mix</td></tr><tr><td rowspan=1 colspan=2>Shigella</td><td rowspan=1 colspan=2>IC (TN)</td><td rowspan=1 colspan=2>stx1</td><td rowspan=1 colspan=2>stx2</td><td rowspan=1 colspan=2>IC (TN)</td></tr><tr><td rowspan=1 colspan=1>Sample ID</td><td rowspan=1 colspan=1>AverageCt Value</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AverageCt Value</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AverageCt Value</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AverageCt Value</td><td rowspan=1 colspan=1>SD</td><td rowspan=1 colspan=1>AverageCt Value</td><td rowspan=1 colspan=1>SD</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>17.5</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>22.9</td><td rowspan=1 colspan=1>0.4</td><td rowspan=1 colspan=1>24.3</td><td rowspan=1 colspan=1>0.3</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>33.0</td><td rowspan=1 colspan=1>0.5</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>32.7</td><td rowspan=1 colspan=1>0.4</td></tr></table>

The number of correct samples is reported in the table below. The results are broken out by Mix (SSC Mix and STEC Mix) and by the Assay as a whole, including the total number and percent correct.

<table><tr><td rowspan=1 colspan=6>Number of Correct Samples after Initial Testing and Retesting</td></tr><tr><td rowspan=2 colspan=1></td><td rowspan=1 colspan=3>Initial Testing</td><td rowspan=1 colspan=2>Retesting (including OriginalData as described)</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1>Combined</td><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Combined</td></tr><tr><td rowspan=1 colspan=1>High Positive</td><td rowspan=1 colspan=1>54/55*(98.2%)</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>109/110(99.1%)</td><td rowspan=1 colspan=1>77/78(98.7%)</td><td rowspan=1 colspan=1>132/133 (99.2%)</td></tr><tr><td rowspan=1 colspan=1>True Negative</td><td rowspan=1 colspan=1>54/55**(98.2%)</td><td rowspan=1 colspan=1>55/55(100%)</td><td rowspan=1 colspan=1>109/110(99.1%)</td><td rowspan=1 colspan=1>77/78(98.7%)</td><td rowspan=1 colspan=1>132/133 (99.2%)</td></tr></table>

\* Sample HP 5 false positive for Campylobacter (SSC Mix), results from repeat testing from nucleic acid in duplicate were negative for Campylobacter \*\*Sample TN 40 false positive for Salmonella (SSC Mix), results from repeat testing from nucleic acid in duplicate were negative for Salmonella

The results of this study demonstrate no evidence of Carry-Over Contamination over a five day course of processing High Positive Shigella and STEC samples alongside True Negative samples using the ProGastro SSCS Assay. $100 \%$ of the True Negative samples were negative for Shigella, stx1 and stx2.

The false positive Campylobacter and Salmonella results in the samples tested indicates that environmental contamination can occur during PCR set up, most likely from the SSCS Positive Control, as these organisms were not used for this study. The following statement is included in the “Limitations” section of the product package insert: “There is a risk of false positive values resulting from cross-contamination by target organisms, their nucleic acids or amplified product, or from non-specific signals in the assay”.

m. Stool Preservation Transport Medium (SPTM) Comparison Study:

An analytical study was carried out to demonstrate equivalent performance of the two stool preservation media, Meridian ParaPak C&S (C&S) and the Remel Cary Blair (CB) Transport Medium, used for collection and transport of stool samples for use with the ProGastro SSCS Assay.

Meridian Para-Pak (C&S) and Remel Cary Blair Transport Medium with Indicator Transport Media (CB) are the two most utilized Stool Preservation and Transport Media (SPTM) in clinical laboratory settings. The intended use of C&S and CB is for the routine collection, transportation, preservation, and culture of clinical stool specimens for bacterial enteric pathogens. The SPTMs have essentially the same ingredients and it was anticipated that there would be minimal differences in assay performance using either of the materials (see the table below).

<table><tr><td rowspan=1 colspan=3>Characteristics of SPTM</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>Meridian ParaPak C&amp;S Cat#900612</td><td rowspan=1 colspan=1>Remel Cary Blair Transport Medium withIndicator Cat#R21610 or R21925</td></tr><tr><td rowspan=1 colspan=1>Components</td><td rowspan=1 colspan=1>AgarPhosphate BufferSodium ChlorideThioglycolic Acid Sodium SaltCalcium ChloridePhenol RedWater</td><td rowspan=1 colspan=1>Sodium ChlorideAgarSodium ThioglycollateDisodium PhosphateCalcium ChloridePhenol Red IndicatorDemineralized water</td></tr><tr><td rowspan=1 colspan=1>pH</td><td rowspan=1 colspan=1>Not stated in the Package Insert</td><td rowspan=1 colspan=1>8.0 ± 0.5 @ 25°</td></tr><tr><td rowspan=1 colspan=1>Storage Temperature</td><td rowspan=1 colspan=1>Room Temperature</td><td rowspan=1 colspan=1>Room Temperature</td></tr><tr><td rowspan=1 colspan=1>Volume per Vial</td><td rowspan=1 colspan=1>15 mL</td><td rowspan=1 colspan=1>15 mL</td></tr></table>

Five cultured and titered bacterial strains, Salmonella, Shigella, Campylobacter jejuni (CJ), Campylobacter coli (CC), and Shiga Toxin producing $E .$ . coli, (STEC), were diluted in PBS and were spiked at 1 log above, at, 1 log below, and 2 logs below the limit of detection (LoD) in each of the two media. The table below contains the strains and concentrations tested in this study:

<table><tr><td colspan="4" rowspan="1">Strains and Concentrations Tested</td></tr><tr><td colspan="1" rowspan="1">Bacterial Strain</td><td colspan="1" rowspan="1">Limit of Detection*</td><td colspan="2" rowspan="1">Concentrations Tested</td></tr><tr><td colspan="1" rowspan="4">Salmonella Enteritidis ATCC 6961</td><td colspan="1" rowspan="4">1x104 CFU/mL</td><td colspan="2" rowspan="1">1x10CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x10</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1x102 CFU/mL</td></tr><tr><td colspan="1" rowspan="4">Shigella sonnei ATCC 29029</td><td colspan="1" rowspan="4">1x10³ CFU/mL</td><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x103</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="3">Campylobacter jejuni ATCC 33291</td><td colspan="1" rowspan="3">1x10³3 CFU/mL</td><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x10^3</td><td colspan="1" rowspan="1">CFU/mL</td></tr><tr><td colspan="2" rowspan="1">1x102 CFU/mL</td></tr><tr><td colspan="3" rowspan="1">Strains and Concentrations Tested</td></tr><tr><td colspan="1" rowspan="1">Bacterial Strain</td><td colspan="1" rowspan="1">Limit of Detection*</td><td colspan="1" rowspan="1">Concentrations Tested</td></tr><tr><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="3"></td><td colspan="1" rowspan="1">1x10 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x10 CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x10-1CFU/mL</td></tr><tr><td colspan="1" rowspan="6">Campylobacter coli ATCC BAA-371</td><td colspan="1" rowspan="6">1x104 CFU/mL</td><td colspan="1" rowspan="1">1x105CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x104CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x103CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x102CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x101CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x100CFU/mL</td></tr><tr><td colspan="1" rowspan="4">STEC O111:H8 (stx1+/stx2+)Strain 3215-99</td><td colspan="1" rowspan="4">1x104 CFU/mL</td><td colspan="1" rowspan="1">1x105CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x104CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x103CFU/mL</td></tr><tr><td colspan="1" rowspan="1">1x102 CFU/mL</td></tr></table>

\*LoD as estimated in stool matrix in the Analytical Reactivity Study Concentrations in bold are additional concentrations that required retesting due to an LoD not being achieved during initial SPTM comparison testing.

Each sample was extracted using the bioMérieux NucliSENS easyMAG in triplicate and run in a single replicate using one lot of ProGastro SSCS reagents on the Cepheid SmartCycler II.

Ct values for the triplicate extractions were averaged and the average, standard deviation, and $\%$ CV are shown in the table below. The LoD estimate for each strain using each media is bolded below in the table.

<table><tr><td colspan="11" rowspan="1">Average Ct, Standard Deviation (SD), and % Coefficient of Variance (CV) for Target Strains*</td></tr><tr><td colspan="1" rowspan="2"></td><td colspan="2" rowspan="2">Conc.CFU/mL</td><td colspan="4" rowspan="1">C&amp;S</td><td colspan="4" rowspan="1">CB</td></tr><tr><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">Avg Ct</td><td colspan="1" rowspan="1">Ct SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">Avg Ct</td><td colspan="1" rowspan="1">Ct SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="4">Salmonella</td><td colspan="2" rowspan="1">1x10</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.4</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="2" rowspan="1">1x104</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">36.1</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">35.6</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="2" rowspan="1">1x103</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">1.9</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">39.0</td><td colspan="1" rowspan="1">1.1</td><td colspan="1" rowspan="1">2.7</td></tr><tr><td colspan="2" rowspan="1">1x102</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="4">Shigella</td><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.2</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.9</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.8</td></tr><tr><td colspan="1" rowspan="1">1x103</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">35.5</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">36.0</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.2</td></tr><tr><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="6">C jejuni</td><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">31.7</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.3</td></tr><tr><td colspan="1" rowspan="1">1x103</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">35.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.8</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">38.3</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">38.2</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.8</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">42.8</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">1.8</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">41.7</td><td colspan="1" rowspan="1">0.7</td><td colspan="1" rowspan="1">1.7</td></tr><tr><td colspan="1" rowspan="1">1x100</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">40.3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">42.7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="1">1x10-1</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">43.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="6">C. coli</td><td colspan="2" rowspan="1">1x105</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">29.6</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">30.1</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.0</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.9</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.4</td></tr><tr><td colspan="1" rowspan="1">1x103</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">36.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">36.8</td><td colspan="1" rowspan="1">0.4</td><td colspan="1" rowspan="1">1.1</td></tr><tr><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">40.0</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.8</td><td colspan="1" rowspan="1">2.0</td></tr><tr><td colspan="1" rowspan="1">1x101</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">40.6</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="2" rowspan="1">1x100</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="1" rowspan="7">STECstx1+/stx2+</td><td colspan="5" rowspan="1"></td><td colspan="5" rowspan="1">stx1 Detection</td></tr><tr><td colspan="2" rowspan="1">1x105</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">32.5</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.5</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.5</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td></tr><tr><td colspan="2" rowspan="1">1x104</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">36.5</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">38.1</td><td colspan="1" rowspan="1">1.7</td><td colspan="1" rowspan="1">4.4</td></tr><tr><td colspan="2" rowspan="1">1x103</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">37.6</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">39.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="2" rowspan="1">1x102</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">0/3</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr><tr><td colspan="3" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="2" rowspan="1">stx2 Detection</td><td colspan="2" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="2" rowspan="1">1x105</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">33.7</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">0.6</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">34.5</td><td colspan="1" rowspan="1">0.3</td><td colspan="1" rowspan="1">0.9</td></tr><tr><td colspan="10" rowspan="1">Average Ct, Standard Deviation (SD), and % Coefficient of Variance (CV) for Target Strains*</td></tr><tr><td colspan="1" rowspan="5"></td><td colspan="1" rowspan="2">Conc.CFU/mL</td><td colspan="4" rowspan="1">C&amp;S</td><td colspan="4" rowspan="1">CB</td></tr><tr><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">Avg Ct</td><td colspan="1" rowspan="1">Ct SD</td><td colspan="1" rowspan="1">% CV</td><td colspan="1" rowspan="1">#Pos</td><td colspan="1" rowspan="1">Avg Ct</td><td colspan="1" rowspan="1">Ct SD</td><td colspan="1" rowspan="1">% CV</td></tr><tr><td colspan="1" rowspan="1">1x104</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">36.6</td><td colspan="1" rowspan="1">1.0</td><td colspan="1" rowspan="1">2.8</td><td colspan="1" rowspan="1">3/3</td><td colspan="1" rowspan="1">38.0</td><td colspan="1" rowspan="1">1.6</td><td colspan="1" rowspan="1">4.1</td></tr><tr><td colspan="1" rowspan="1">1x103</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">38.5</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">39.3</td><td colspan="1" rowspan="1">0.5</td><td colspan="1" rowspan="1">1.3</td></tr><tr><td colspan="1" rowspan="1">1x102</td><td colspan="1" rowspan="1">2/3</td><td colspan="1" rowspan="1">39.4</td><td colspan="1" rowspan="1">0.1</td><td colspan="1" rowspan="1">0.2</td><td colspan="1" rowspan="1">1/3</td><td colspan="1" rowspan="1">40.7</td><td colspan="1" rowspan="1">NA</td><td colspan="1" rowspan="1">NA</td></tr></table>

For the CB to be considered equivalent to C&S:

All three replicates for at least one concentration must test positive and within one log of the lowest concentration in which all 3 replicates of C&S are positive. The average Ct values at all concentrations in which 3 of 3 replicates are positive shall not differ by more than 3.3 Cts from the average Ct of that concentration diluted in C&S. If the sample differs by more than 3.3 Cts the media will not be considered equivalent.

The Mean Ct difference between the two media for the average of each strain at each concentration is shown the table below:

<table><tr><td rowspan=1 colspan=9>Ct Difference and Data Acceptance for CB in Comparison to ParaPak C&amp;S</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=1>LoD Estimate(CFU/mL) in C&amp;S</td><td rowspan=1 colspan=1>LoD Estimate(CFU/mL) in CB</td><td rowspan=1 colspan=4>Concentration(CFU/mL)</td><td rowspan=1 colspan=1>Ct Difference*(CB - C&amp;S)</td><td rowspan=1 colspan=1>Equivalent?</td></tr><tr><td rowspan=4 colspan=1>Salmonella</td><td rowspan=4 colspan=1>1x103</td><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=4>1x1051x101x1031x102</td><td rowspan=1 colspan=1>0.1</td><td rowspan=4 colspan=1>YES</td></tr><tr><td rowspan=1 colspan=1>-0.5</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=4 colspan=1>Shigella</td><td rowspan=4 colspan=1>1x103</td><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=4>1x1041x1031x1021x101</td><td rowspan=1 colspan=1>0.7</td><td rowspan=4 colspan=1>YES</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=6 colspan=1>C jejuni</td><td rowspan=6 colspan=1>1x101</td><td rowspan=6 colspan=1>1x102</td><td rowspan=6 colspan=4>1x1041x1031x1021x1011x1001x10-1</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1>0.3</td></tr><tr><td rowspan=1 colspan=3></td><td rowspan=1 colspan=2>b</td><td rowspan=1 colspan=1>-0.3</td><td></td></tr><tr><td rowspan=1 colspan=1>-0.1</td><td></td></tr><tr><td rowspan=1 colspan=1>NA</td><td></td></tr><tr><td rowspan=1 colspan=1>NA</td><td></td></tr><tr><td rowspan=1 colspan=1>NA</td><td></td></tr><tr><td rowspan=6 colspan=1>C. coli</td><td rowspan=6 colspan=1>1x102</td><td rowspan=6 colspan=1>1x102</td><td rowspan=6 colspan=4>1x1051x1041x1031x1021x1011x100</td><td rowspan=1 colspan=1>0.5</td><td rowspan=6 colspan=1>YES</td></tr><tr><td rowspan=1 colspan=1>0.7</td></tr><tr><td rowspan=1 colspan=1>0.4</td></tr><tr><td rowspan=1 colspan=1>-0.6</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=10 colspan=1>STECstx1+/stx2+</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>stx1 Detection</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=4>1x1051x1041x1031x102</td><td rowspan=1 colspan=1>1.0</td><td rowspan=4 colspan=1>YES</td></tr><tr><td rowspan=1 colspan=1>1.6</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=4>stx2 Detection</td><td rowspan=1 colspan=2></td></tr><tr><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=1>1x104</td><td rowspan=4 colspan=4>1x1051x1041x1031x102</td><td rowspan=1 colspan=1>0.8</td><td rowspan=4 colspan=1>YES</td></tr><tr><td rowspan=1 colspan=1>1.4</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr><tr><td rowspan=1 colspan=1>NA</td></tr></table>

In conclusion, both the CB and C&S media resulted in $100 \%$ detection at the same lowest concentration tested for $C .$ . coli and STEC. Salmonella, Shigella, and $C .$ . jejuni in CB medium resulted in $100 \%$ detection at a concentration 1 log higher than in C&S medium. In addition, none of the organisms revealed a difference in average Ct value of more than 3.3 between the different media at any concentration tested. Therefore, the performance is considered equivalent. Stool samples may be collected in CB and/or C&S for use with the ProGastro SSCS Assay.

# n. Comparator Assays Analytical Validation Studies

The ProGastro SSCS Assay is a Real Time PCR Assay comprised of two Mixes, one used to detect and differentiate Salmonella, Shigella, and Campylobacter (C. jejuni and C. coli only, not differentiated, SSC Mix) while the other Mix detects and differentiates the Shiga Toxin $1 \ ( s t x I )$ and Shiga Toxin 2 $( s t x 2 )$ genes in bacteria such as Shiga Toxin producing $E .$ . coli (STEC Mix). The ProGastro SSCS Assay Clinical Study established performance by comparing ProGastro SSCS results to broth enrichment/EIA for Shiga Toxin producing bacteria. The broth enrichment/EIA Kit (Meridian Biosciences Premier EHEC EIA Kit) does not differentiate between stx1 and stx2 positive STEC samples and as a result, a sequence reference method (SRM) for the detection and differentiation of $s t x l$ and ${ s t x 2 }$ was developed. Any STEC positive sample via one or both testing methods (EIA or ProGastro SSCS STEC Mix) was further tested for stx1 and stx2 using these sequencing assays. The SRM is comprised of two different PCR mixes, one targeting a different area of the stx1 gene than the ProGastro SSCS Assay’s STEC Mix and the second targeting a different area of the stx2 gene than the STEC Mix. Samples yielding a PCR product from one or both of the SRM PCR Mixes were sent to an external facility for bi-directional sequencing.

Analytical Sensitivity, Reactivity, and Specificity studies were conducted to verify the analytical performance of these SRM stx1 and $s t x 2$ sequencing assays.

# Limit of Detection

The limit of detection (LoD) of the STEC SRM was determined using previously titered $\mathrm { ( C F U / m l ) }$ cultures of STEC, including a single strain of STEC O157:H7 $( s t x I + / s t x 2 - )$ for the stx1 SRM Mix and STEC Non-O157 $( s t x I - / s t x 2 + )$ for the stx2 SRM Mix. Two strains of previously cultured and titered bacteria were serially diluted, spiked into a ProGastro SSCS Assay negative stool matrix pool, and tested at three concentrations: 1 log above, at, and 1 log below the tentative LoD. Each dilution series was extracted on the bioMérieux NucliSENS easyMAG in a single replicate and tested in five replicate PCR reactions using the required STEC SRM Mix depending on the strain $. s t x l +$ strain with the stx1 Mix and $s t x 2 +$ strain with the stx2 Mix). PCR products generated with the SRM Mixes were analyzed using the Qiagen QIAxcel capillary electrophoresis instrument. LoD was determined to be the lowest concentration of bacteria detected $2 9 5 \%$ of the time (5/5 replicates). The LoD for each strain was confirmed by the generation of 20 independent extraction data points using the specific spiked stool concentration utilized during the LoD Determination portion of this study. The required spiked sample was used to perform 20 independent extractions (per strain) on the easyMAG with each purified nucleic acid sample run in a single replicate using a single lot of the required STEC SRM Mix depending on the strain. Subsequent to capillary electrophoresis during the LoD Confirmation portion of the study, one representative sample containing the least concentrated product for each strain was confirmed for the correct gene target by bi-directional genetic sequencing and NCBI BLAST Analysis (www.ncbi.nlm.nih.gov/blast/BLAST.cgi). At least $9 5 \%$ of the 20 replicates were required to test positive to confirm the LoD for each bacterial strain.

The results of the LoD estimation portion of the study are presented in the following table. Concentrations that are bolded were used for the subsequent Confirmation portion of the study.

<table><tr><td rowspan=1 colspan=6>SRM Assay LoD Estimation Results</td></tr><tr><td rowspan=1 colspan=1>STECO Antigen: H Type</td><td rowspan=1 colspan=1>STEC CenterAccession#</td><td rowspan=1 colspan=1>StrainName</td><td rowspan=1 colspan=1>stx1/stx2Designation</td><td rowspan=1 colspan=1>ConcentrationTested (CFU/ml)</td><td rowspan=1 colspan=1>ReplicatesDetected</td></tr><tr><td rowspan=3 colspan=1>STEC O157:H7</td><td rowspan=3 colspan=1>TW00975</td><td rowspan=3 colspan=1>2886-75</td><td rowspan=3 colspan=1>(stx1+/stx2-)</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x104</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>3/5</td></tr><tr><td rowspan=3 colspan=1>STEC O26:HNM</td><td rowspan=3 colspan=1>TW08569</td><td rowspan=3 colspan=1>CB7776</td><td rowspan=3 colspan=1>(stx1-/stx2+)</td><td rowspan=1 colspan=1>1x105</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x104</td><td rowspan=1 colspan=1>5/5</td></tr><tr><td rowspan=1 colspan=1>1x103</td><td rowspan=1 colspan=1>1/5</td></tr></table>

The table below summarizes the confirmed LoD for the strains tested.

<table><tr><td rowspan=1 colspan=7>SRM Assay LoD Confirmation Results</td></tr><tr><td rowspan=1 colspan=1>STEC0 Antigen: H Type</td><td rowspan=1 colspan=1>STEC CenterAccession#</td><td rowspan=1 colspan=1>StrainName</td><td rowspan=1 colspan=1>stx1/stx2Designation</td><td rowspan=1 colspan=1>ConcentrationTested(CFU/ml)</td><td rowspan=1 colspan=1>ReplicatesDetected</td><td rowspan=1 colspan=1>%Detected</td></tr><tr><td rowspan=1 colspan=1>STEC O157:H7</td><td rowspan=1 colspan=1>TW00975</td><td rowspan=1 colspan=1>2886-75</td><td rowspan=1 colspan=1>(stx1+/stx2-)</td><td rowspan=1 colspan=1>1x104</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr><tr><td rowspan=1 colspan=1>STEC 026:HNM</td><td rowspan=1 colspan=1>TW08569</td><td rowspan=1 colspan=1>CB7776</td><td rowspan=1 colspan=1>(stx1-/stx2+)</td><td rowspan=1 colspan=1>1x104</td><td rowspan=1 colspan=1>20/20</td><td rowspan=1 colspan=1>100%</td></tr></table>

The LoD comparison between the ProGastro SSCS Assay (STEC Mix) and the STEC SRM is presented in the table below. The LoD for the SRM is comparable to the LoD confirmed for these two strains using the ProGastro SSCS Assay.

<table><tr><td rowspan=1 colspan=6>Comparison of LoD between SRM Assays and ProGastro SSCS Assay</td></tr><tr><td rowspan=1 colspan=1>STECO Antigen: H Type</td><td rowspan=1 colspan=1>STEC CenterAccession#</td><td rowspan=1 colspan=1>StrainName</td><td rowspan=1 colspan=1>stx1/stx2Designation</td><td rowspan=1 colspan=1>LoD STECSRM</td><td rowspan=1 colspan=1>LoD ProGastrosSCS Assay</td></tr><tr><td rowspan=1 colspan=1>STEC O157:H7</td><td rowspan=1 colspan=1>TW00975</td><td rowspan=1 colspan=1>2886-75</td><td rowspan=1 colspan=1>(stx1+/stx2-)</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>1.66 x 104CFU/mL</td></tr><tr><td rowspan=1 colspan=1>STEC O26:HNM</td><td rowspan=1 colspan=1>TW08569</td><td rowspan=1 colspan=1>CB7776</td><td rowspan=1 colspan=1>(stx1-/stx2+)</td><td rowspan=1 colspan=1>1x104CFU/mL</td><td rowspan=1 colspan=1>9.27 x 103CFU/mL</td></tr></table>

# Analytical Reactivity

Each STEC strain analyzed in this study tested positive by the STEC SRM Assay for its respective $s t x$ gene (stx1 and/or stx2, depending on the strain). Shigella dysenteriae ATCC# 29027 was found to be non-reactive with the STEC SRM Assay (both stx1 and $s t x 2$ ) as expected. The following table summarizes the results.

<table><tr><td rowspan=1 colspan=7>SRM Assay Analytical Reactivity Results Summary</td></tr><tr><td rowspan=1 colspan=1>Strain</td><td rowspan=1 colspan=2>ConcentrationTested</td><td rowspan=1 colspan=1>stx1</td><td rowspan=1 colspan=1>stx1 SRM MixBLAST Result</td><td rowspan=1 colspan=1>stx2</td><td rowspan=1 colspan=1>stx2 SRM MixBLAST Result</td></tr><tr><td rowspan=1 colspan=1>STEC DEC10B</td><td rowspan=1 colspan=2>2x10³ CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>97-3250</td><td rowspan=1 colspan=2>2x104 CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>DA-21</td><td rowspan=1 colspan=2>2x104 CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>MI03-19</td><td rowspan=1 colspan=2>2x104CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>STEC MT#80</td><td rowspan=1 colspan=2>2x10^3CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>3215-99</td><td rowspan=1 colspan=2>2x104CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>RD8</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>STEC 0201 9611</td><td rowspan=1 colspan=1>2x103</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>STEC DA-5</td><td rowspan=1 colspan=1>2x103</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>STEC DA-1</td><td rowspan=1 colspan=1>2x103</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>STEC GS G5578620</td><td rowspan=1 colspan=2>2x103CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>IH 16</td><td rowspan=1 colspan=1>2x104</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>STEC 7:85</td><td rowspan=1 colspan=1>2x104(stx1)</td><td rowspan=1 colspan=1>CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA*</td></tr><tr><td rowspan=1 colspan=1>STEC 7:85</td><td rowspan=1 colspan=2>2x103CFU/ml(stx2)</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>NA*</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>STEC 93-111</td><td rowspan=1 colspan=2>2x103CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>DA-34</td><td rowspan=1 colspan=2>2x104CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>STEC EDL933</td><td rowspan=1 colspan=2>2x103CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>1:361</td><td rowspan=1 colspan=2>2x104CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>DA-54</td><td rowspan=1 colspan=2>2x104CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriae</td><td rowspan=1 colspan=2>2x10³ CFU/ml</td><td rowspan=1 colspan=1>+</td><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr><tr><td rowspan=1 colspan=1>Shigella dysenteriaeATCC 29027</td><td rowspan=1 colspan=2>1.03x10³ CFU/ml</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>-</td><td rowspan=1 colspan=1>Negative</td></tr></table>

# Analytical Specificity

The STEC SRM Assay did not react with any of the non-target organisms included in this study, other than Escherichia hermannii. This strain produced a band with interpretable sequencing results with the stx2 Mix which was identified as $E$ . coli by BLAST analysis, but further analysis revealed no positive hits for ${ s t x 2 }$ , so it is not in fact reactive with the STEC SRM assay. The stx1 target strain was detected using the stx1 Mix, but negative with the stx2 Mix. The stx2 target strain was detected using the stx2 Mix, but negative with the stx1 Mix. The STEC SRM Assay demonstrates no cross reactivity with the organisms that are commonly found in stool, genetically related or cause similar disease states as the ProGastro SSCS Assay target organisms. In silico analysis of the Cyclospora cayetanensis genome showed that each primer and probe included with the SRM Mixes had no similarity to the organism.

In conclusion, the STEC SRM demonstrated similar analytical sensitivity (limit of detection, LoD) and reactivity as compared to the ProGastro SSCS Assay STEC Mix. The STEC SRM was also reactive to the same set of STEC used in the Analytical Specificity Study for the ProGastro SSCS Assay STEC Mix. The STEC SRM was not cross-reactive with any common stool organisms or organisms that cause similar disease states as the ProGastro SSCS Assay target organisms. Based on these

analytical studies performed, the STEC SRM was acceptable to use as part of the reference method to differentiate stx1 from $s t x 2$ for positive samples found by the broth enrichment/EIA Kit (Meridian Biosciences Premier EHEC EIA Kit) reference method or the ProGastro SSCS Assay STEC Mix.

2. Comparison studies:

a. Method comparison with predicate device: Not applicable. Refer to the Clinical Studies Section of this document.

b. Matrix comparison: Not applicable

3. Clinical studies:

# Prospective Clinical Study

The prospective clinical performance study was conducted at four U.S. clinical laboratory sites using prospectively collected samples. All four sites enrolled and tested prospective samples that were collected from July 2011 through November 2011. One of the four sites also enrolled and tested approximately 250 prospective samples collected in May 2012 – July 2012. All specimens used in the study meeting the inclusion criteria represented excess remnants of stool specimens that were prospectively collected from symptomatic individuals suspected of gastrointestinal infection, and were submitted for routine care or analysis by each site, and that otherwise would have been discarded.

Demographic details for the patient population included in the prospective study are summarized in the following table.

<table><tr><td rowspan=1 colspan=1>Sex</td><td rowspan=1 colspan=1>Number of Samples SSC Mix</td><td rowspan=1 colspan=1>Number of Samples STEC Mix</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>615/1214 (50.6%)</td><td rowspan=1 colspan=1>615/1214 (50.6%)</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>581/1214 (47.9%)</td><td rowspan=1 colspan=1>581/1214 (47.9%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>18/1214 (1.5%)</td><td rowspan=1 colspan=1>18/1214 (1.5%)</td></tr><tr><td rowspan=1 colspan=1>Age (yrs)</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>378/1214 (31.1%)</td><td rowspan=1 colspan=1>378/1214 (31.1%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>296/1214 (24.4%)</td><td rowspan=1 colspan=1>296/1214 (24.4%)</td></tr><tr><td rowspan=1 colspan=1>19- 64 years</td><td rowspan=1 colspan=1>357/1214 (29.4%)</td><td rowspan=1 colspan=1>357/1214 (29.4%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>164/1214 (13.5%)</td><td rowspan=1 colspan=1>164/1214 (13.5%)</td></tr><tr><td rowspan=1 colspan=1>Unknown</td><td rowspan=1 colspan=1>19/1214 (1.6%)</td><td rowspan=1 colspan=1>19/1214 (1.6%)</td></tr></table>

Leftover stool specimens collected from symptomatic patients were split and one portion was tested using each site’s routine culture/EIA procedures. The second portion of the samples underwent nucleic acid extraction using the bioMérieux NucliSens easyMAG Instrument and reagents. Purified nucleic acids were tested with the ProGastro SSCS Assay using both the SSC and STEC Mixes on the Cepheid SmartCycler II.

Performance of the ProGastro SSCS Assay was assessed and compared to the reference method of culture (Campylobacter, Salmonella, and Shigella) or broth enrichment followed by FDA cleared EIA test (Shiga Toxin producing E. coli, STEC). Samples positive for STEC by broth/EIA and/or the ProGastro SSCS Assay underwent PCR/bi-directional sequencing to confirm the presence of the stx1 and/or stx2 genes. Two PCR/sequencing assays were used that each targeted different regions of the stx1 or stx2 gene than the ProGastro SSCS Assay. “True” STEC positives were considered as any sample that tested positive for STEC by the broth/EIA method, and “True” STEC negatives were considered as any sample that tested negative for STEC by the broth/EIA method. “True” stx1 or “true” $s t x 2$ positives were considered as any sample that tested positive for STEC by the broth/EIA method and by PCR/sequencing. Bidirectional sequencing data was required to meet pre-defined quality acceptance criteria for both the forward and the reverse sequences that matched stx1 or stx2 sequences deposited in the National Center for Biotechnology Information (NCBI) GenBank database (www.ncbi.nlm.nih.gov), respectively, with acceptable E-values. The E-Value from NCBI BLAST Alignment indicates the statistical significance of a given pair-wise alignment and reflects the size of the database and the scoring system used. The lower the E-Value, the more significant the hit is. A sequence alignment that has an E-Value of 1e-3 means that this similarity has a 1 in 1000 chance of occurring by chance alone. (http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=handbook.section.614).

Discrepant results between the ProGastro SSCS Assay and the reference methods were also evaluated using analytically validated PCR/sequencing assays and results are footnoted in the performance tables below.

A total of 1214 patients were initially enrolled in the prospective clinical trial. Sixtyone patient/samples were excluded from the performance calculations due to deviations from the clinical study protocol. Fourteen specimens were excluded for the SSC Mix and 14 were excluded for the STEC Mix from the prospective clinical study data analysis because they remained “Unresolved” after repeat testing with the respective ProGastro SSCS Assay Mix. Unresolved results occur when the sample is negative for all target detections and the Internal Control, indicating potentially PCRinhibited samples. This resulted in a total of 1139 eligible prospective specimens to be included in the prospective clinical study data analysis.

The prospective study performance data (all sites combined) are presented in the following tables stratified by analyte:

Campylobacter (C. jejuni / C. coli), Prospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>13 a</td><td rowspan=1 colspan=1>33</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1106</td><td rowspan=1 colspan=1>1106</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>1119</td><td rowspan=1 colspan=1>1139</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (83.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 98.8% (98.0% - 99.3%)</td></tr></table>

a Six samples were positive for Campylobacter (C. coli / C. jejuni) by bi-directional sequence analysis.

Salmonella, Prospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>10 a</td><td rowspan=1 colspan=1>30</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1b</td><td rowspan=1 colspan=1>1108</td><td rowspan=1 colspan=1>1109</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>21</td><td rowspan=1 colspan=1>1118</td><td rowspan=1 colspan=1>1139</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 95.2% (77.3% - 99.2%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.1% (98.4% - 99.5%)</td></tr></table>

a Ten samples were positive for Salmonella by bi-directional sequence analysis. b Sample was positive for Salmonella by bi-directional sequence analysis.

Shigella, Prospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>6 a</td><td rowspan=1 colspan=1>21</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1118</td><td rowspan=1 colspan=1>1118</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>1124</td><td rowspan=1 colspan=1>1139</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (79.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.5% (98.8% - 99.8%)</td></tr></table>

a Six samples were positive for Shigella by bi-directional sequence analysis.

STEC, Prospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySTEC Mix</td><td rowspan=1 colspan=3>Broth Enrichment/EIA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>9a</td><td rowspan=1 colspan=1>9b</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1121</td><td rowspan=1 colspan=1>1121</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>1130</td><td rowspan=1 colspan=1>1139</td></tr><tr><td rowspan=1 colspan=4>Sensitivity: 100.0% (70.1% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Specificity: 99.2% (98.5% - 99.6%)</td></tr></table>

a Six samples positive for $s t x l$ , one sample positive for $s t x 2$ , and two samples positive for $s t x I$ and $s t x 2$ by bi-directional sequence analysis. b Six samples positive for $s t x I$ and three samples positive for $s t x 2$ by bi-directional sequence analysis.

Stx 1, Prospective Samples (Percent Agreement between ProGastro SSCS and Composite Reference Method for stx1)   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCS AssaySTEC Mix</td><td rowspan=1 colspan=3>Broth Enrichment/EIA and Sequencing forstx1</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>6 a</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>10</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=4>Percent Positive Agreement: 100.0% (67.6% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Percent Negative Agreement: 40.0% (16.8% - 68.7%)</td></tr></table>

a Five samples were positive by bi-directional sequencing for $s t x 1$ , but were negative by Broth Enrichment/EIA.

Stx2, Prospective Samples (Percent Agreement between ProGastro SSCS and Composite Reference Method for stx2)   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCS AssaySTEC Mix</td><td rowspan=1 colspan=3>Broth Enrichment/EIA and Sequencing forstx2</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>3 a</td><td rowspan=1 colspan=1>6</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>15</td><td rowspan=1 colspan=1>18</td></tr><tr><td rowspan=1 colspan=4>Percent Positive Agreement: 100.0% (43.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Percent Negative Agreement: 80.0% (54.8% - 93.0%)</td></tr></table>

a Three samples were positive by bi-directional sequencing for stx2, but were negative by Broth Enrichment/EIA.

# Prospective Clinical Study Mixed Infection Analysis

The ProGastro SSCS Assay detected one mixed infections in the prospective clinical evaluation. This represents $0 . 9 8 \%$ of the total positive specimens (1/102). The one mixed infections sample was double infections and was confirmed by the reference methods.

Distinct Co-infection Combinations Detected by the ProGastro SSCS Assay in the Prospective Clinical Trial   

<table><tr><td rowspan=1 colspan=3>Distinct Co-infection CombinationsDetected by ProGastro SSCS</td><td rowspan=2 colspan=1>Con-C</td><td rowspan=2 colspan=1>Number ofDiscrepantCo-infectionsa</td><td rowspan=2 colspan=1>Discrepant Analyte(s)a</td></tr><tr><td rowspan=1 colspan=1>Analyte 1</td><td rowspan=1 colspan=1>Analyte 2</td><td rowspan=1 colspan=1>Analyte 3</td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>Shigella</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Salmonella</td><td rowspan=1 colspan=1>Campylobacter</td><td rowspan=1 colspan=1>STEC</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Co-infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Double Infections</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=3>Total Triple Infections</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1></td></tr></table>

a A discrepant co-infection or discrepant analyte was defined as one that was detected by the ProGastro SSCS Assay but not

There were no co-infections that were detected by the reference method and not detected by the ProGastro SSCS Assay.

# Retrospective Clinical Study

In addition to the prospective clinical study, two of the four clinical sites also performed testing using retrospective samples that were collected from 2007 - 2011. A total of 105 stool samples were included in the retrospective study.

These samples had been previously determined to be positive or negative by culture and/or Broth Enrichment/EIA. The ProGastro SSCS Assay was compared to the same reference method that was employed for the prospective study to determine positive and negative percent agreement.

Demographic details for this patient population are summarized in the table below:

<table><tr><td rowspan=1 colspan=1>Sex*</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>Female</td><td rowspan=1 colspan=1>24/55 (43.6%)</td></tr><tr><td rowspan=1 colspan=1>Male</td><td rowspan=1 colspan=1>31/55 (56.4%)</td></tr><tr><td rowspan=1 colspan=1>Age</td><td rowspan=1 colspan=1>Number of Subjects</td></tr><tr><td rowspan=1 colspan=1>≤ 5 years</td><td rowspan=1 colspan=1>12/105 (11.4%)</td></tr><tr><td rowspan=1 colspan=1>6 - 18 years</td><td rowspan=1 colspan=1>24/105 (22.9%)</td></tr><tr><td rowspan=1 colspan=1>19- 64 years</td><td rowspan=1 colspan=1>51/105 (48.6%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>18/105 (17.1%)</td></tr></table>

\*For all of the 50 specimens tested from one site the gender was unknown

The retrospective study performance data (both sites combined) are presented in the following tables stratified by analyte:

Campylobacter (C. jejuni / C. coli), Retrospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>27</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>32</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>72</td><td rowspan=1 colspan=1>73</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>28</td><td rowspan=1 colspan=1>77</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=3>Positive Percent Agreement: 96.4% (82.3% - 99.4%)</td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 93.5% (85.7% - 97.2%)</td></tr></table>

Salmonella, Retrospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>3</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>102</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>102</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement y: 100.0% (43.9% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 100.0% (96.4% - 100.0%)</td></tr></table>

Shigella, Retrospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySSC Mix</td><td rowspan=1 colspan=3>Culture</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>4</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>101</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>4</td><td rowspan=1 colspan=1>101</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (51.0% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 100.0% (96.3% - 100.0%)</td></tr></table>

STEC, Retrospective Samples   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCSAssaySTE Mix</td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIA</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>19 a</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>86</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>19</td><td rowspan=1 colspan=1>86</td><td rowspan=1 colspan=1>105</td></tr><tr><td rowspan=1 colspan=4>Positive Percent Agreement: 100.0% (83.2% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Negative Percent Agreement: 100.0% (95.7% - 100.0%)</td></tr></table>

a Five samples positive for $s t x I$ , five samples positive for $s t x 2$ , and nine samples positive for $s t x I$ and $s t x 2$ .

Stx 1, Retrospective Samples (Percent Agreement between ProGastro SSCS and Composite Reference Method for stx1   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCS AssaySTEC Mix</td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIA andSequencing for stx1</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=4>Percent Positive Agreement: 100.0% (78.5% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Percent Negative Agreement: 100.0% (56.6% - 100.0%)</td></tr></table>

Stx2, Prospective Samples (Percent Agreement between ProGastro SSCS and Composite Reference Method for stx2   

<table><tr><td rowspan=2 colspan=1>ProGastro SSCS AssaySTEC Mix</td><td rowspan=1 colspan=3>Culture or Broth Enrichment/EIA andSequencing for stx2</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Total</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>5</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>14</td><td rowspan=1 colspan=1>5</td><td rowspan=1 colspan=1>19</td></tr><tr><td rowspan=1 colspan=4>Percent Positive Agreement: 100.0% (78.5% - 100.0%)</td></tr><tr><td rowspan=1 colspan=4>Percent Negative Agreement: 100.0% (56.6% - 100.0%)</td></tr></table>

# ProGastro SSCS Assay Failure Rate Due to Controls (Prospective and Retrospective Studies Combined)

There were a total of four ProGastro SSCS runs out of 136 (75 SSC and 61 STEC) runs that were invalid, three from the prospective study and one from the retrospective study. Two of the run failures were due to Negative Control contamination. Another invalid run was due to the PC1 failure for STEC and it is hypothesized that the SSCS PC1 and the C. coli PC2 were mistakenly switched in the SmartCyler II. The final

invalid run was due to 3084/3089 error with an SSC run, where the SmartCycler II run is started before the I-CORE Lid has been closed. For each initially invalid run, all samples, the Negative Control, and the PMC (Extraction Control) were re-run using the ProGastro SSCS Assay starting from the purified nucleic acid and repeat test results were valid and used in analysis.

# “Unresolved” Samples (Prospective and Retrospective Studies Combined)

An “Unresolved” result is generated when the Gastro Internal Control (GIC) fails to be detected in a clinical specimen. A failure of the GIC to be detected can occur if inhibitors are present in a sample or due to technical error (e.g., GIC not added prior to nucleic acid extraction). According to the Assay Instructions for Use and the ProGastro SSCS Clinical Study Protocol all “Unresolved” samples were to be retested starting from the purified nucleic acids. (Note: Detection of the GIC in the Cy5 detection channel is not required for reporting negative results for a sample when at least one analyte is detected in the sample in the proper detection channel, since high bacterial load can lead to reduced or absent Internal Control signal.) Samples with a repeat “Unresolved” result were excluded from the final data analysis. Valid repeat negative and positive results were included in the final analysis of the Clinical Study. The following table summarizes the ProGastro SSCS Assay “Unresolved” samples from the prospective and the retrospective study.

<table><tr><td rowspan=1 colspan=7>Summary of &quot;Unresolved&quot; ProGastro SSCS Clinical Trial Samples</td></tr><tr><td rowspan=1 colspan=1>SSC Mix</td><td rowspan=1 colspan=1>Site 1</td><td rowspan=1 colspan=1>Site 2</td><td rowspan=1 colspan=1>Site 4</td><td rowspan=1 colspan=1>Site 5</td><td rowspan=1 colspan=1>Site 6</td><td rowspan=1 colspan=1>Overall</td></tr><tr><td rowspan=1 colspan=1>Number of Samples Run</td><td rowspan=1 colspan=1>345</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1258</td></tr><tr><td rowspan=1 colspan=1>Unresolved upon initial testing</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>25</td></tr><tr><td rowspan=1 colspan=1>Unresolved upon repeat testing</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>12</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>% Unresolved</td><td rowspan=1 colspan=1>0.3%</td><td rowspan=1 colspan=1>3.0%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Negative upon repeat testing</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>8</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>10</td></tr><tr><td rowspan=1 colspan=1>Positive upon repeat testing</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>1</td></tr><tr><td rowspan=1 colspan=1>STEC Mix</td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td><td rowspan=1 colspan=1></td></tr><tr><td rowspan=1 colspan=1>Number of Samples Run</td><td rowspan=1 colspan=1>345</td><td rowspan=1 colspan=1>400</td><td rowspan=1 colspan=1>340</td><td rowspan=1 colspan=1>123</td><td rowspan=1 colspan=1>50</td><td rowspan=1 colspan=1>1258</td></tr><tr><td rowspan=1 colspan=1>Unresolved upon initial testing</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>20</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>26</td></tr><tr><td rowspan=1 colspan=1>Unresolved upon repeat testing</td><td rowspan=1 colspan=1>2</td><td rowspan=1 colspan=1>11</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>1</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>14</td></tr><tr><td rowspan=1 colspan=1>% Unresolved</td><td rowspan=1 colspan=1>0.6%</td><td rowspan=1 colspan=1>2.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>0.8%</td><td rowspan=1 colspan=1>0.0%</td><td rowspan=1 colspan=1>1.1%</td></tr><tr><td rowspan=1 colspan=1>Negative upon repeat testing</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>9</td><td rowspan=1 colspan=1>3</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>12</td></tr><tr><td rowspan=1 colspan=1>Positive upon repeat testing</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>0</td><td rowspan=1 colspan=1>N/A</td><td rowspan=1 colspan=1>0</td></tr></table>

4. Clinical cut-off:

Not applicable

5. Expected values/Reference range:

In the ProGastro SSCS Assay prospective clinical study, a total of 1139 eligible prospective stool specimens were tested at four U.S. clinical laboratories in the United States. Samples were collected July 2011 thru November 2011 and May 2012 thru July 2012.

The number and percentage of Campylobacter, Salmonella, Shigella, and STEC positive cases as determined by the ProGastro SSCS Assay, stratified by age group, are presented in the following table:

<table><tr><td rowspan=1 colspan=1>Age Group</td><td rowspan=1 colspan=1>Total # SSCMix/Total #STEC Mix</td><td rowspan=1 colspan=1>#CampylobacterPositive (ObservedPositivity Rate)</td><td rowspan=1 colspan=1>#SalmonellaPositive(ObservedPositivity Rate)</td><td rowspan=1 colspan=1>#Shigella Positive(Observed PositivityRate)</td><td rowspan=1 colspan=1>#STEC Positive(Observed PositivityRate)</td></tr><tr><td rowspan=1 colspan=1>&lt;2 years</td><td rowspan=1 colspan=1>240/238</td><td rowspan=1 colspan=1>5 (2.1%)</td><td rowspan=1 colspan=1>11 (4.6%)</td><td rowspan=1 colspan=1>2 (0.8%)</td><td rowspan=1 colspan=1>4 (1.7%)</td></tr><tr><td rowspan=1 colspan=1>2-5 years</td><td rowspan=1 colspan=1>149/148</td><td rowspan=1 colspan=1>8 (5.4%)</td><td rowspan=1 colspan=1>6 (4.0%)</td><td rowspan=1 colspan=1>11 (7.4%)</td><td rowspan=1 colspan=1>2 (1.4%)</td></tr><tr><td rowspan=1 colspan=1>6-11 years</td><td rowspan=1 colspan=1>128/130</td><td rowspan=1 colspan=1>2 (1.6%)</td><td rowspan=1 colspan=1>8 (6.3%)</td><td rowspan=1 colspan=1>6 (4.7%)</td><td rowspan=1 colspan=1>1 (0.8%)</td></tr><tr><td rowspan=1 colspan=1>12-18 years</td><td rowspan=1 colspan=1>157/158</td><td rowspan=1 colspan=1>1 (0.6%)</td><td rowspan=1 colspan=1>4 (2.5%)</td><td rowspan=1 colspan=1>2 (1.3%)</td><td rowspan=1 colspan=1>5 (3.2%)</td></tr><tr><td rowspan=1 colspan=1>19-64 years</td><td rowspan=1 colspan=1>306/306</td><td rowspan=1 colspan=1>15 (4.9%)</td><td rowspan=1 colspan=1>1 (0.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3 (1.0%)</td></tr><tr><td rowspan=1 colspan=1>≥ 65 years</td><td rowspan=1 colspan=1>158/158</td><td rowspan=1 colspan=1>2 (1.3%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>0 (0.0%)</td><td rowspan=1 colspan=1>3 (1.9%)</td></tr><tr><td rowspan=1 colspan=1>Total</td><td rowspan=1 colspan=1>1138/1138*</td><td rowspan=1 colspan=1>33 (2.9%)</td><td rowspan=1 colspan=1>30 (2.6%)</td><td rowspan=1 colspan=1>21 (1.8%)</td><td rowspan=1 colspan=1>18 (1.6%)</td></tr></table>

\*One sample did not have age information and therefore not included in the expected values; however, 1139 valid samples were tested during the prospective clinical study.

# N. Instrument Name:

Cepheid SmartCycler II Real Time Instrument with Dx Software version $1 . 7 6$ or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$

NucliSENS® easyMAG™ System (bioMérieux)

# O. System Descriptions:

1. Modes of Operation:

The bioMerieux NucliSENS easyMAG system is used for nucleic acid isolation from stool samples.

The Cepheid SmartCycler II Real Time instrument with Dx software version 1.7b or $3 . 0 { \mathrm { a } } / { \mathrm { b } }$ is used to perform PCR amplification and detection of nucleic acid.

Several other Prodesse Assays have been cleared for use with the above instrumentation systems: Prodesse ProFlu $^ +$ (k110968), Pro hMPV $^ +$ (k082688), ProGastro Cd (k090239), ProParaflu+ (k091053), ProFAST $^ +$ (k101855) and ProAdeno $^ +$ (k102952).

# 2. Software:

FDA has reviewed applicant’s Hazard Analysis and software development processes for this line of product types:

Yes ____X____ or No

3. Specimen Identification:

User enters Patient ID/Sample ID by typing it in.

4. Specimen Sampling and Handling: Not applicable

5. Calibration: Not applicable

6. Quality Control:

<table><tr><td rowspan=1 colspan=1>Control Type</td><td rowspan=1 colspan=1>Used to Monitor</td></tr><tr><td rowspan=1 colspan=1>Positive</td><td rowspan=1 colspan=1>Substantial reagent failure including primer and probe integrity</td></tr><tr><td rowspan=1 colspan=1>Negative</td><td rowspan=1 colspan=1>Reagent and/or environmental contamination</td></tr><tr><td rowspan=1 colspan=1>Extraction</td><td rowspan=1 colspan=1>Failure in lysis and extraction procedure</td></tr><tr><td rowspan=1 colspan=1>Internal</td><td rowspan=1 colspan=1>PCR inhibition in individual samples and Reagent failure or process error</td></tr></table>

# P. Other Supportive Instrument Performance Characteristics Data Not Covered In the “Performance Characteristics” Section above:

Not applicable

# Q. Proposed Labeling:

The labeling is sufficient and it satisfies the requirements of 21 CFR Part 809.10.

# R. Conclusion:

The submitted information in this premarket notification is complete and supports a substantial equivalence decision.